<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/10/24/451740.atom" />
<meta name="HW.pisa" content="biorxiv;451740v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Origins and Evolution of the Global RNA Virome" />
<meta name="DC.Identifier" content="10.1101/451740" />
<meta name="DC.Date" content="2018-10-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.

IMPORTANCE The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses." />
<meta name="DC.Contributor" content="Yuri I. Wolf" />
<meta name="DC.Contributor" content="Darius Kazlauskas" />
<meta name="DC.Contributor" content="Jaime Iranzo" />
<meta name="DC.Contributor" content="Adriana LucÍa-Sanz" />
<meta name="DC.Contributor" content="Jens H. Kuhn" />
<meta name="DC.Contributor" content="Mart Krupovic" />
<meta name="DC.Contributor" content="Valerian V. Dolja" />
<meta name="DC.Contributor" content="Eugene V. Koonin" />
<meta name="article:published_time" content="2018-10-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Origins and Evolution of the Global RNA Virome" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Viruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.&lt;/p&gt;&lt;h3&gt;IMPORTANCE&lt;/h3&gt;
&lt;p&gt;The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;451740v1" />
<meta name="citation_id" content="451740v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/451740v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/451740v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/451740v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740.full.pdf" />
<meta name="citation_doi" content="10.1101/451740" />
<meta name="citation_num_pages" content="80" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="451740" />
<meta name="citation_author" content="Yuri I. Wolf" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health" />
<meta name="citation_author" content="Darius Kazlauskas" />
<meta name="citation_author_institution" content="Vilniaus universitetas biotechnologijos institutas" />
<meta name="citation_author_institution" content="Département de Microbiologie, Institut Pasteur, Paris" />
<meta name="citation_author" content="Jaime Iranzo" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health" />
<meta name="citation_author" content="Adriana LucÍa-Sanz" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health" />
<meta name="citation_author_institution" content="Centro Nacional de Biotecnología" />
<meta name="citation_author" content="Jens H. Kuhn" />
<meta name="citation_author_institution" content="Integrated Research Facility at Fort Detrick, National Institute of Allergy and Infectious Diseases, National Institutes of Health" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7800-6045" />
<meta name="citation_author" content="Mart Krupovic" />
<meta name="citation_author_institution" content="Département de Microbiologie, Institut Pasteur, Paris" />
<meta name="citation_author" content="Valerian V. Dolja" />
<meta name="citation_author_institution" content="Department of Botany and Plant Pathology, Oregon State University" />
<meta name="citation_author_email" content="doljav@oregonstate.edu" />
<meta name="citation_author" content="Eugene V. Koonin" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3943-8299" />
<meta name="citation_reference" content="Bernhardt HS. 2012. The RNA world hypothesis: the worst theory of the early evolution of life (except for all the others)(a). Biol Direct 7: 23." />
<meta name="citation_reference" content="Gilbert W. 1986. Origin of Life the RNA World. Nature 319: 618–618." />
<meta name="citation_reference" content="Nelson JW, Breaker RR. 2017. The lost language of the RNA World. Sci Signal 10: eaam8812." />
<meta name="citation_reference" content="Koonin EV, Senkevich TG, Dolja VV. 2006. The ancient Virus World and evolution of cells. Biol Direct 1: 29." />
<meta name="citation_reference" content="Baltimore D. 1971. Expression of animal virus genomes. Bacteriol Rev 35: 235–241." />
<meta name="citation_reference" content="Baltimore D. 1971. Viral genetic systems. Trans N Y Acad Sci 33: 327–332." />
<meta name="citation_reference" content="Koonin EV, Dolja VV. 1993. Evolution and taxonomy of positive-strand RNA viruses: implications of comparative analysis of amino acid sequences. Crit Rev Biochem Mol Biol 28: 375–430." />
<meta name="citation_reference" content="Koonin EV, Dolja VV. 2013. A virocentric perspective on the evolution of life. Curr Opin Virol 3: 546–557." />
<meta name="citation_reference" content="Ahlquist P. 2006. Parallels among positive-strand RNA viruses, reverse-transcribing viruses and double-stranded RNA viruses. Nat Rev Microbiol 4: 371–382." />
<meta name="citation_reference" content="Reguera J, Gerlach P, Cusack S. 2016. Towards a structural understanding of RNA synthesis by negative strand RNA viral polymerases. Curr Opin Struct Biol 36: 75–84." />
<meta name="citation_reference" content="Dolja VV, Koonin EV. 2018. Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer. Virus Res 244: 36–52." />
<meta name="citation_reference" content="Bolduc B, Shaughnessy DP, Wolf YI, Koonin EV, Roberto FF, Young M. 2012. Identification of novel positive-strand RNA viruses by metagenomic analysis of archaea-dominated Yellowstone hot springs. J Virol 86: 5562–5573." />
<meta name="citation_reference" content="Krishnamurthy SR, Janowski AB, Zhao G, Barouch D, Wang D. 2016. Hyperexpansion of RNA bacteriophage diversity. PLoS Biol 14: e1002409." />
<meta name="citation_reference" content="Shi M, Lin X-D, Tian J-H, Chen L-J, Chen X, Li C-X, Qin X-C, Li J, Cao J-P, Eden J-S, Buchmann J, Wang W, Xu J, Holmes EC, Zhang Y-Z. 2016. Redefining the invertebrate RNA virosphere. Nature 540: 539–543." />
<meta name="citation_reference" content="Koonin EV, Dolja VV, Krupovic M. 2015. Origins and evolution of viruses of eukaryotes: the ultimate modularity. Virology 479–480: 2–25." />
<meta name="citation_reference" content="Dolja VV, Koonin EV. 2011. Common origins and host-dependent diversity of plant and animal viromes. Curr Opin Virol 1: 322–331." />
<meta name="citation_reference" content="Shi M, Zhang Y-Z, Holmes EC. 2018. Meta-transcriptomics and The Evolutionary Biology of RNA Viruses. Virus Res 243: 83–90." />
<meta name="citation_reference" content="Koonin EV, Dolja VV. 2014. Virus world as an evolutionary network of viruses and capsidless selfish elements. Microbiol Mol Biol Rev 78: 278–303." />
<meta name="citation_reference" content="Krupovic M, Koonin EV. 2017. Multiple origins of viral capsid proteins from cellular ancestors. Proc Natl Acad Sci U S A 114: E2401–E2410." />
<meta name="citation_reference" content="Dolja VV, Boyko VP, Agranovsky AA, Koonin EV. 1991. Phylogeny of capsid proteins of rod-shaped and filamentous RNA plant viruses: two families with distinct patterns of sequence and probably structure conservation. Virology 184: 79–86." />
<meta name="citation_reference" content="Koonin EV. 1991. The phylogeny of RNA-dependent RNA polymerases of positive-strand RNA viruses. J Gen Virol 72 (Pt 9): 2197–2206." />
<meta name="citation_reference" content="Xiong Y, Eickbush TH. 1990. Origin and evolution of retroelements based upon their reverse transcriptase sequences. EMBO J 9: 3353–3362." />
<meta name="citation_reference" content="Poch O, Sauvaget I, Delarue M, Tordo N. 1989. Identification of four conserved motifs among the RNA-dependent polymerase encoding elements. EMBO J 8: 3867–3874." />
<meta name="citation_reference" content="Ng KK-S, Arnold JJ, Cameron CE. 2008. Structure-function relationships among RNA-dependent RNA polymerases. Curr Top Microbiol Immunol 320: 137–156." />
<meta name="citation_reference" content="te Velthuis AJW. 2014. Common and unique features of viral RNA-dependent polymerases. Cell Mol Life Sci 71: 4403–4420." />
<meta name="citation_reference" content="Eickbush TH, Jamburuthugoda VK. 2008. The diversity of retrotransposons and the properties of their reverse transcriptases. Virus Res 134: 221–234." />
<meta name="citation_reference" content="Gladyshev EA, Arkhipova IR. 2011. A widespread class of reverse transcriptase-related cellular genes. Proc Natl Acad Sci U S A 108: 20311–20316." />
<meta name="citation_reference" content="Lambowitz AM, Belfort M. 2015. Mobile bacterial group II introns at the crux of eukaryotic evolution. Microbiol Spectr 3: MDNA3–0050–2014." />
<meta name="citation_reference" content="Novikova O, Belfort M. 2017. Mobile group II introns as ancestral eukaryotic elements. Trends Genet 33: 773–783." />
<meta name="citation_reference" content="Krupovic M, Blomberg J, Coffin JM, Dasgupta I, Fan H, Geering AD, Gifford R, Harrach B, Hull R, Johnson W, Kreuze JF, Lindemann D, Llorens C, Lockhart B, Mayer J, Muller E, Olszewski NE, Pappu HR, Pooggin MM, Richert-Poggeler KR, Sabanadzovic S, Sanfacon H, Schoelz JE, Seal S, Stavolone L, Stoye JP, Teycheney P-Y, Tristem M, Koonin EV, Kuhn JH. 2018. Ortervirales: new virus order unifying five families of reverse-transcribing viruses. J Virol 92: e00515–00518." />
<meta name="citation_reference" content="Koonin EV, Wolf YI, Nagasaki K, Dolja VV. 2008. The Big Bang of picorna-like virus evolution antedates the radiation of eukaryotic supergroups. Nat Rev Microbiol 6: 925–939." />
<meta name="citation_reference" content="Koonin EV. 1992. Evolution of double-stranded RNA viruses: a case for polyphyletic origin from different groups of positive-stranded RNA viruses. Semin Virol 3: 327–339." />
<meta name="citation_reference" content="El Omari K, Sutton G, Ravantti JJ, Zhang H, Walter TS, Grimes JM, Bamford DH, Stuart DI, Mancini EJ. 2013. Plate tectonics of virus shell assembly and reorganization in phage Φ8, a distant relative of mammalian reoviruses. Structure 21: 1384–1395." />
<meta name="citation_reference" content="Poranen MM, Bamford DH. 2012. Assembly of large icosahedral double-stranded RNA viruses. Adv Exp Med Biol 726: 379–402." />
<meta name="citation_reference" content="Pflug A, Guilligay D, Reich S, Cusack S. 2014. Structure of influenza A polymerase bound to the viral RNA promoter. Nature 516: 355–360." />
<meta name="citation_reference" content="Goldbach R, Wellink J. 1988. Evolution of plus-strand RNA viruses. Intervirology 29: 260–267." />
<meta name="citation_reference" content="Greninger AL. 2018. A decade of RNA virus metagenomics is (not) enough. Virus Res 244: 218–229." />
<meta name="citation_reference" content="Shi M, Lin X-D, Vasilakis N, Tian J-H, Li C-X, Chen L-J, Eastwood G, Diao X-N, Chen M-H, Chen X, Qin X-C, Widen SG, Wood TG, Tesh RB, Xu J, Holmes EC, Zhang Y-Z. 2016. Divergent viruses discovered in arthropods and vertebrates revise the evolutionary history of the Flaviviridae and related viruses. J Virol 90: 659–669." />
<meta name="citation_reference" content="Li C-X, Shi M, Tian J-H, Lin X-D, Kang Y-J, Chen L-J, Qin X-C, Xu J, Holmes EC, Zhang Y-Z. 2015. Unprecedented genomic diversity of RNA viruses in arthropods reveals the ancestry of negative-sense RNA viruses. Elife 4: e05378." />
<meta name="citation_reference" content="Webster CL, Longdon B, Lewis SH, Obbard DJ. 2016. Twenty-five new viruses associated with the Drosophilidae (Diptera). Evol Bioinform Online 12: 13–25." />
<meta name="citation_reference" content="Fauver JR, Grubaugh ND, Krajacich BJ, Weger-Lucarelli J, Lakin SM, Fakoli LS, 3rd, Bolay FK, Diclaro JW, 2nd, Dabire KR, Foy BD, Brackney DE, Ebel GD, Stenglein MD. 2016. West African Anopheles gambiae mosquitoes harbor a taxonomically diverse virome including new insect-specific flaviviruses, mononegaviruses, and totiviruses. Virology 498: 288–299." />
<meta name="citation_reference" content="Marzano S-YL, Nelson BD, Ajayi-Oyetunde O, Bradley CA, Hughes TJ, Hartman GL, Eastburn DM, Domier LL. 2016. Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens. J Virol 90: 6846–6863." />
<meta name="citation_reference" content="Deakin G, Dobbs E, Bennett JM, Jones IM, Grogan HM, Burton KS. 2017. Multiple viral infections in Agaricus bisporus-Characterisation of 18 unique RNA viruses and 8 ORFans identified by deep sequencing. Sci Rep 7: 2469." />
<meta name="citation_reference" content="Iranzo J, Krupovic M, Koonin EV. 2016. The double-stranded DNA virosphere as a modular hierarchical network of gene sharing. MBio 7: e00978–00916." />
<meta name="citation_reference" content="Iranzo J, Krupovic M, Koonin EV. 2017. A network perspective on the virus world. Commun Integr Biol 10: e1296614." />
<meta name="citation_reference" content="Gorbalenya AE, Pringle FM, Zeddam J-L, Luke BT, Cameron CE, Kalmakoff J, Hanzlik TN, Gordon KHJ, Ward VK. 2002. The palm subdomain-based active site is internally permuted in viral RNA-dependent RNA polymerases of an ancient lineage. J Mol Biol 324: 47–62." />
<meta name="citation_reference" content="King AMQ, Lefkowitz EJ, Mushegian AR, Adams MJ, Dutilh BE, Gorbalenya AE, Harrach B, Harrison RL, Junglen S, Knowles NJ, Kropinski AM, Krupovic M, Kuhn JH, Nibert ML, Rubino L, Sabanadzovic S, Sanfaçon H, Siddell SG, Simmonds P, Varsani A, Zerbini FM, Davison AJ. 2018. Changes to taxonomy and the International Code of Virus Classification and Nomenclature ratified by the International Committee on Taxonomy of Viruses (2018). Arch Virol 163: 2601–2631." />
<meta name="citation_reference" content="Guindon S, Dufayard J-F, Lefort V, Anisimova M, Hordijk W, Gascuel O. 2010. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst Biol 59: 307–321." />
<meta name="citation_reference" content="Akopyants NS, Lye L-F, Dobson DE, Lukeš J, Beverley SM. 2016. A novel bunyavirus-like virus of trypanosomatid protist parasites. Genome Announc 4: e00715–00716." />
<meta name="citation_reference" content="Grybchuk D, Akopyants NS, Kostygov AY, Konovalovas A, Lye L-F, Dobson DE, Zangger H, Fasel N, Butenko A, Frolov AO, Votýpka J, d’Avila-Levy CM, Kulich P, Moravcová J, Plevka P, Rogozin IB, Serva S, Lukeš J, Beverley SM, Yurchenko V. 2018. Viral discovery and diversity in trypanosomatid protozoa with a focus on relatives of the human parasite Leishmania. Proc Natl Acad Sci U S A 115: E506–E515." />
<meta name="citation_reference" content="Cohen O, Ashkenazy H, Belinky F, Huchon D, Pupko T. 2010. GLOOME: gain loss mapping engine. Bioinformatics 26: 2914–2915." />
<meta name="citation_reference" content="Golmohammadi R, Valegård K, Fridborg K, Liljas L. 1993. The refined structure of bacteriophage MS2 at 2.8 A resolution. J Mol Biol 234: 620–639." />
<meta name="citation_reference" content="Hillman BI, Cai G. 2013. The family Narnaviridae: simplest of RNA viruses. Adv Virus Res 86: 149–176." />
<meta name="citation_reference" content="Nibert ML, Vong M, Fugate KK, Debat HJ. 2018. Evidence for contemporary plant mitoviruses. Virology 518: 14–24." />
<meta name="citation_reference" content="Rastgou M, Habibi MK, Izadpanah K, Masenga V, Milne RG, Wolf YI, Koonin EV, Turina M. 2009. Molecular characterization of the plant virus genus Ourmiavirus and evidence of inter-kingdom reassortment of viral genome segments as its possible route of origin. J Gen Virol 90: 2525–2535." />
<meta name="citation_reference" content="Le Gall O, Christian P, Fauquet CM, King AMQ, Knowles NJ, Nakashima N, Stanway G, Gorbalenya AE. 2008. Picornavirales, a proposed order of positive-sense single-stranded RNA viruses with a pseudo-T = 3 virion architecture. Arch Virol 153: 715–727." />
<meta name="citation_reference" content="Culley A. 2018. New insight into the RNA aquatic virosphere via viromics. Virus Research 244: 84–89." />
<meta name="citation_reference" content="Ng TFF, Marine R, Wang C, Simmonds P, Kapusinszky B, Bodhidatta L, Oderinde BS, Wommack KE, Delwart E. 2012. High variety of known and new RNA and DNA viruses of diverse origins in untreated sewage. J Virol 86: 12161–12175." />
<meta name="citation_reference" content="Hause BM, Palinski R, Hesse R, Anderson G. 2016. Highly diverse posaviruses in swine faeces are aquatic in origin. J Gen Virol 97: 1362–1367." />
<meta name="citation_reference" content="Jiang B, Monroe SS, Koonin EV, Stine SE, Glass RI. 1993. RNA sequence of astrovirus: distinctive genomic organization and a putative retrovirus-like ribosomal frameshifting signal that directs the viral replicase synthesis. Proc Natl Acad Sci U S A 90: 10539–10543." />
<meta name="citation_reference" content="Saberi A, Gulyaeva AA, Brubacher JL, Newmark PA, Gorbalenya AE. 2018. A planarian nidovirus expands the limits of RNA genome size. bioRxiv doi: 10.1101/299776:299776." />
<meta name="citation_reference" content="Nagasaki K, Shirai Y, Takao Y, Mizumoto H, Nishida K, Tomaru Y. 2005. Comparison of genome sequences of single-stranded RNA viruses infecting the bivalvekilling dinoflagellate Heterocapsa circularisquama. Appl Environ Microbiol 71: 8888–8894." />
<meta name="citation_reference" content="Koonin EV, Choi GH, Nuss DL, Shapira R, Carrington JC. 1991. Evidence for common ancestry of a chestnut blight hypovirulence-associated double-stranded RNA and a group of positive-strand RNA plant viruses. Proc Natl Acad Sci U S A 88: 10647–10651." />
<meta name="citation_reference" content="Dawe AL, Nuss DL. 2013. Hypovirus molecular biology: from Koch’s postulates to host self-recognition genes that restrict virus transmission. Adv Virus Res 86: 109–147." />
<meta name="citation_reference" content="Nibert ML, Ghabrial SA, Maiss E, Lesker T, Vainio EJ, Jiang D, Suzuki N. 2014. Taxonomic reorganization of family Partitiviridae and other recent progress in partitivirus research. Virus Res 188: 128–141." />
<meta name="citation_reference" content="Tang J, Ochoa WF, Li H, Havens WM, Nibert ML, Ghabrial SA, Baker TS. 2010. Structure of Fusarium poae virus 1 shows conserved and variable elements of partitivirus capsids and evolutionary relationships to picobirnavirus. J Struct Biol 172: 363–371." />
<meta name="citation_reference" content="Duquerroy S, Da Costa B, Henry C, Vigouroux A, Libersou S, Lepault J, Navaza J, Delmas B, Rey FA. 2009. The picobirnavirus crystal structure provides functional insights into virion assembly and cell entry. EMBO J 28: 1655–1665." />
<meta name="citation_reference" content="Nibert ML, Tang J, Xie J, Collier AM, Ghabrial SA, Baker TS, Tao YJ. 2013. 3D structures of fungal partitiviruses. Adv Virus Res 86: 59–85." />
<meta name="citation_reference" content="Luque D, Gómez-Blanco J, Garriga D, Brilot AF, González JM, Havens WM, Carrascosa JL, Trus BL, Verdaguer N, Ghabrial SA, Castón JR. 2014. Cryo-EM near-atomic structure of a dsRNA fungal virus shows ancient structural motifs preserved in the dsRNA viral lineage. Proc Natl Acad Sci U S A 111: 7641–7646." />
<meta name="citation_reference" content="Koga R, Fukuhara T, Nitta T. 1998. Molecular characterization of a single mitochondria-associated double-stranded RNA in the green alga Bryopsis. Plant Mol Biol 36: 717–724." />
<meta name="citation_reference" content="Koga R, Horiuchi H, Fukuhara T. 2003. Double-stranded RNA replicons associated with chloroplasts of a green alga Bryopsis cinicola. Plant Mol Biol 51: 991–999." />
<meta name="citation_reference" content="Koonin EV, Dolja VV. 2006. Evolution of complexity in the viral world: the dawn of a new vision. Virus Res 117: 1–4." />
<meta name="citation_reference" content="Gorbalenya AE, Koonin EV. 1989. Viral proteins containing the purine NTP-binding sequence pattern. Nucleic Acids Res 17: 8413–8440." />
<meta name="citation_reference" content="Snijder EJ, Decroly E, Ziebuhr J. 2016. The nonstructural proteins directing coronavirus RNA synthesis and processing. Adv Virus Res 96: 59–126." />
<meta name="citation_reference" content="Enjuanes L, Zuñiga S, Castaño-Rodriguez C, Gutierrez-Alvarez J, Canton J, Sola I. 2016. Molecular basis of coronavirus virulence and vaccine development. Adv Virus Res 96: 245–286." />
<meta name="citation_reference" content="Sola I, Almazán F, Zúñiga S, Enjuanes L. 2015. Continuous and discontinuous RNA synthesis in coronaviruses. Annu Rev Virol 2: 265–288." />
<meta name="citation_reference" content="Roossinck MJ, Sabanadzovic S, Okada R, Valverde RA. 2011. The remarkable evolutionary history of endornaviruses. J Gen Virol 92: 2674–2678." />
<meta name="citation_reference" content="Li K, Zheng D, Cheng J, Chen T, Fu Y, Jiang D, Xie J. 2016. Characterization of a novel Sclerotinia sclerotiorum RNA virus as the prototype of a new proposed family within the order Tymovirales. Virus Res 219: 92–99." />
<meta name="citation_reference" content="Syller J. 2002. Umbraviruses-the unique plant viruses that do not encode a capsid protein. Acta Microbiol Pol 51: 99–113." />
<meta name="citation_reference" content="Janowski AB, Krishnamurthy SR, Lim ES, Zhao G, Brenchley JM, Barouch DH, Thakwalakwa C, Manary MJ, Holtz LR, Wang D. 2017. Statoviruses, a novel taxon of RNA viruses present in the gastrointestinal tracts of diverse mammals. Virology 504: 36–44." />
<meta name="citation_reference" content="Greninger AL, DeRisi JL. 2015. Draft genome sequence of tombunodavirus UC1. Genome Announc 3: e00655–00615." />
<meta name="citation_reference" content="Dokland T, Walsh M, Mackenzie JM, Khromykh AA, Ee K-H, Wang S. 2004. West Nile virus core protein; tetramer structure and ribbon formation. Structure 12: 1157–1163." />
<meta name="citation_reference" content="Ma L, Jones CT, Groesch TD, Kuhn RJ, Post CB. 2004. Solution structure of dengue virus capsid protein reveals another fold. Proc Natl Acad Sci U S A 101: 3414–3419." />
<meta name="citation_reference" content="Kuhn RJ, Rossmann MG. 2005. Structure and assembly of icosahedral enveloped RNA viruses. Adv Virus Res 64: 263–284." />
<meta name="citation_reference" content="Ahola T, Karlin DG. 2015. Sequence analysis reveals a conserved extension in the capping enzyme of the alphavirus supergroup, and a homologous domain in nodaviruses. Biol Direct 10: 16." />
<meta name="citation_reference" content="Koonin EV. 1993. Computer-assisted identification of a putative methyltransferase domain in NS5 protein of flaviviruses and λ2 protein of reovirus. J Gen Virol 74 (Pt 4): 733–740." />
<meta name="citation_reference" content="Liu L, Dong H, Chen H, Zhang J, Ling H, Li Z, Shi P-Y, Li H. 2010. Flavivirus RNA cap methyltransferase: structure, function, and inhibition. Front Biol (Beijing) 5: 286–303." />
<meta name="citation_reference" content="Dolja VV, Kreuze JF, Valkonen JPT. 2006. Comparative and functional genomics of closteroviruses. Virus Res 117: 38–51." />
<meta name="citation_reference" content="Kobayashi K, Atsumi G, Iwadate Y, Tomita R, Chiba K-I, Akasaka S, Nishihara M, Takahashi H, Yamaoka N, Nishiguchi M, Sekine K-T. 2013. Gentian Kobu-sho-associated virus: a tentative, novel double-stranded RNA virus that is relevant to gentian Kobu-sho syndrome. J Gen Plant Pathol 79: 56–63." />
<meta name="citation_reference" content="Teixeira M, Sela N, Ng J, Casteel CL, Peng H-C, Bekal S, Girke T, Ghanim M, Kaloshian I. 2016. A novel virus from Macrosiphum euphorbiae with similarities to members of the family Flaviviridae. J Gen Virol 97: 1261–1271." />
<meta name="citation_reference" content="Mata CP, Luque D, Gómez-Blanco J, Rodríguez JM, González JM, Suzuki N, Ghabrial SA, Carrascosa JL, Trus BL, Castón JR. 2017. Acquisition of functions on the outer capsid surface during evolution of double-stranded RNA fungal viruses. PLoS Pathog 13: e1006755." />
<meta name="citation_reference" content="Castón JR, Luque D, Gómez-Blanco J, Ghabrial SA. 2013. Chrysovirus structure: repeated helical core as evidence of gene duplication. Adv Virus Res 86: 87–108." />
<meta name="citation_reference" content="Pan J, Dong L, Lin L, Ochoa WF, Sinkovits RS, Havens WM, Nibert ML, Baker TS, Ghabrial SA, Tao YJ. 2009. Atomic structure reveals the unique capsid organization of a dsRNA virus. Proc Natl Acad Sci U S A 106: 4225–4230." />
<meta name="citation_reference" content="Abrescia NGA, Bamford DH, Grimes JM, Stuart DI. 2012. Structure unifies the viral universe. Annu Rev Biochem 81: 795–822." />
<meta name="citation_reference" content="El Omari K, Meier C, Kainov D, Sutton G, Grimes JM, Poranen MM, Bamford DH, Tuma R, Stuart DI, Mancini EJ. 2013. Tracking in atomic detail the functional specializations in viral RecA helicases that occur during evolution. Nucleic Acids Res 41: 9396–9410." />
<meta name="citation_reference" content="Sutton G, Grimes JM, Stuart DI, Roy P. 2007. Bluetongue virus VP4 is an RNA-capping assembly line. Nat Struct Mol Biol 14: 449–451." />
<meta name="citation_reference" content="Yu X, Jiang J, Sun J, Zhou ZH. 2015. A putative ATPase mediates RNA transcription and capping in a dsRNA virus. Elife 4: e07901." />
<meta name="citation_reference" content="Amarasinghe GK, Ceballos NGA, Banyard AC, Basler CF, Bavari S, Bennett AJ, Blasdell KR, Briese T, Bukreyev A, Caì Y, Calisher CH, Lawson CC, Chandran K, Chapman CA, Chiu CY, Choi K-S, Collins PL, Dietzgen RG, Dolja VV, Dolnik O, Domier LL, Dürrwald R, Dye JM, Easton AJ, Ebihara H, Echevarría JE, Fooks AR, Formenty PBH, Fouchier RAM, Freuling CM, Ghedin E, Goldberg TL, Hewson R, Horie M, Hyndman TH, Jiāng D, Kityo R, Kobinger GP, Kondō H, Koonin EV, Krupovic M, Kurath G, Lamb RA, Lee B, Leroy EM, Maes P, Maisner A, Marston DA, Mor SK, Müller T, et al. 2018. Taxonomy of the order Mononegavirales: update 2018. Arch Virol 163: 2283–2294." />
<meta name="citation_reference" content="Kormelink R, Garcia ML, Goodin M, Sasaya T, Haenni A-L. 2011. Negative-strand’ RNA viruses: the plant-infecting counterparts. Virus Res 162: 184–202." />
<meta name="citation_reference" content="Osaki H, Sasaki A, Nomiyama K, Tomioka K. 2016. Multiple virus infection in a single strain of Fusarium poae shown by deep sequencing. Virus Genes 52: 835–847." />
<meta name="citation_reference" content="Bacharach E, Mishra N, Briese T, Zody MC, Kembou Tsofack JE, Zamostiano R, Berkowitz A, Ng J, Nitido A, Corvelo A, Toussaint NC, Abel Nielsen SC, Hornig M, Del Pozo J, Bloom T, Ferguson H, Eldar A, Lipkin WI. 2016. Characterization of a novel orthomyxo-like virus causing mass die-offs of tilapia. MBio 7: e00431–00416." />
<meta name="citation_reference" content="Maes P, Alkhovsky SV, Bío Y, Beer M, Birkhead M, Briese T, Buchmeier MJ, Calisher CH, Charrel RN, Choi IR, Clegg CS, Torre JCdl, Delwart E, DeRisi JL, Bello PLD, Serio FD, Digiaro M, Dolja VV, Drosten C, Druciarek TZ, Du J, Ebihara H, Elbeaino T, Gergerich RC, Gillis AN, Gonzalez J-PJ, Haenni A-L, Hepojoki J, Hetzel U, Hồ T, Hóng N, Jain RK, Vuren PJv, Jin Q, Jonson MG, Junglen S, Keller KE, Kemp A, Kipar A, Kondov NO, Koonin EV, Kormelink R, Korzyukov Y, Krupovic M, Lambert AJ, Laney AG, LeBreton M, Lukashevic IS, Marklewitz M, Markotter W, et al. 2018. Taxonomy of the family Arenaviridae and the order Bunyavirales: update 2018. Arch Virol 163: 2295–2310." />
<meta name="citation_reference" content="Mielke-Ehret N, Mühlbach H-P. 2012. Emaravirus: a novel genus of multipartite, negative strand RNA plant viruses. Viruses 4: 1515–1536." />
<meta name="citation_reference" content="Krupovic M. 2013. Networks of evolutionary interactions underlying the polyphyletic origin of ssDNA viruses. Curr Opin Virol 3: 578–586." />
<meta name="citation_reference" content="Kazlauskas D, Varsani A, Krupovic M. 2018. Pervasive chimerism in the replication-associated proteins of uncultured single-stranded DNA viruses. Viruses 10: 187." />
<meta name="citation_reference" content="Iranzo J, Koonin EV, Prangishvili D, Krupovic M. 2016. Bipartite network analysis of the archaeal virosphere: evolutionary connections between viruses and capsidless mobile elements. J Virol 90: 11043–11055." />
<meta name="citation_reference" content="Krishnamurthy SR, Wang D. 2018. Extensive conservation of prokaryotic ribosomal binding sites in known and novel picobirnaviruses. Virology 516: 108–114." />
<meta name="citation_reference" content="Boros Á, Polgár B, Pankovics P, Fenyvesi H, Engelmann P, Phan TG, Delwart E, Reuter G. 2018. Multiple divergent picobirnaviruses with functional prokaryotic Shine-Dalgarno ribosome binding sites present in cloacal sample of a diarrheic chicken. Virology 525: 62–72." />
<meta name="citation_reference" content="Jinek M, Doudna JA. 2009. A three-dimensional view of the molecular machinery of RNA interference. Nature 457: 405–412." />
<meta name="citation_reference" content="Wu Q, Wang X, Ding S-W. 2010. Viral suppressors of RNA-based viral immunity: host targets. Cell Host Microbe 8: 12–15." />
<meta name="citation_reference" content="Krupovic M, Dolja VV, Koonin EV. 2015. Plant viruses of the Amalgaviridae family evolved via recombination between viruses with double-stranded and negative-strand RNA genomes. Biol Direct 10: 12." />
<meta name="citation_reference" content="Martin RR, Zhou J, Tzanetakis IE. 2011. Blueberry latent virus: an amalgam of the Partitiviridae and Totiviridae. Virus Res 155: 175–180." />
<meta name="citation_reference" content="Sabanadzovic S, Valverde RA, Brown JK, Martin RR, Tzanetakis IE. 2009. Southern tomato virus: the link between the families Totiviridae and Partitiviridae. Virus Res 140: 130–137." />
<meta name="citation_reference" content="Sabanadzovic S, Abou Ghanem-Sabanadzovic N, Valverde RA. 2010. A novel monopartite dsRNA virus from rhododendron. Arch Virol 155: 1859–1863." />
<meta name="citation_reference" content="Mushegian AR, Koonin EV. 1993. Cell-to-cell movement of plant viruses. Insights from amino acid sequence comparisons of movement proteins and from analogies with cellular transport systems. Arch Virol 133: 239–257." />
<meta name="citation_reference" content="Ghabrial SA, Castón JR, Jiang D, Nibert ML, Suzuki N. 2015. 50-plus years of fungal viruses. Virology 479–480: 356–368." />
<meta name="citation_reference" content="Hillman BI, Annisa A, Suzuki N. 2018. Viruses of plant-interacting fungi. Adv Virus Res 100: 99–116." />
<meta name="citation_reference" content="Kotta-Loizou I, Coutts RHA. 2017. Studies on the virome of the entomopathogenic fungus Beauveria bassiana reveal novel dsRNA elements and mild hypervirulence. PLoS Pathog 13: e1006183." />
<meta name="citation_reference" content="Coyle MC, Elya CN, Bronski M, Eisen MB. 2018. Entomophthovirus: an insect-derived iflavirus that infects a behavior manipulating fungal pathogen of dipterans. bioRxiv doi: 10.1101/371526:371526." />
<meta name="citation_reference" content="Waldron FM, Stone GN, Obbard DJ. 2018. Metagenomic sequencing suggests a diversity of RNA interference-like responses to viruses across multicellular eukaryotes. PLoS Genet 14: e1007533." />
<meta name="citation_reference" content="Aiewsakun P, Simmonds P. 2018. The genomic underpinnings of eukaryotic virus taxonomy: creating a sequence-based framework for family-level virus classification. Microbiome 6: 38." />
<meta name="citation_reference" content="Benson DA, Cavanaugh M, Clark K, Karsch-Mizrachi I, Ostell J, Pruitt KD, Sayers EW. 2018. GenBank. Nucleic Acids Res 46: D41–D47." />
<meta name="citation_reference" content="Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ. 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25: 3389–3402." />
<meta name="citation_reference" content="Marchler-Bauer A, Derbyshire MK, Gonzales NR, Lu S, Chitsaz F, Geer LY, Geer RC, He J, Gwadz M, Hurwitz DI, Lanczycki CJ, Lu F, Marchler GH, Song JS, Thanki N, Wang Z, Yamashita RA, Zhang D, Zheng C, Bryant SH. 2015. CDD: NCBI’s conserved domain database. Nucleic Acids Res 43: D222–226." />
<meta name="citation_reference" content="Edgar RC. 2010. Search and clustering orders of magnitude faster than BLAST. Bioinformatics 26: 2460–2461." />
<meta name="citation_reference" content="Edgar RC. 2004. MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics 5: 113." />
<meta name="citation_reference" content="Soding J. 2005. Protein homology detection by HMM-HMM comparison. Bioinformatics 21: 951–960." />
<meta name="citation_reference" content="Sokal RB, Michener CD. 1958. A statistical method for evaluating systematic relationships. Univ Kansas Sci Bull XXXVIII: 1409–1438." />
<meta name="citation_reference" content="Price MN, Dehal PS, Arkin AP. 2010. FastTree 2-approximately maximum-likelihood trees for large alignments. PLoS One 5: e9490." />
<meta name="citation_reference" content="Lemoine F, Domelevo Entfellner J-B, Wilkinson E, Correia D, Davila Felipe M, De Oliveira T, Gascuel O. 2018. Renewing Felsenstein’s phylogenetic bootstrap in the era of big data. Nature 556: 452–456." />
<meta name="citation_reference" content="Mirdita M, von den Driesch L, Galiez C, Martin MJ, Soding J, Steinegger M. 2017. Uniclust databases of clustered and deeply annotated protein sequences and alignments. Nucleic Acids Res 45: D170–D176." />
<meta name="citation_reference" content="Eddy SR. 2011. Accelerated profile HMM searches. PLoS Comput Biol 7: e1002195." />
<meta name="citation_reference" content="Frickey T, Lupas A. 2004. CLANS: a Java application for visualizing protein families based on pairwise similarity. Bioinformatics 20: 3702–3704." />
<meta name="citation_reference" content="Lancichinetti A, Fortunato S. 2012. Consensus clustering in complex networks. Sci Rep 2: 336." />
<meta name="citation_reference" content="Fortunato S, Hric D. 2016. Community detection in networks: A user guide. Physics Rep Rev Sect Physics Lett 659: 1–44." />
<meta name="citation_reference" content="Rosvall M, Bergstrom CT. 2008. Maps of random walks on complex networks reveal community structure. Proc Natl Acad Sci U S A 105: 1118–1123." />
<meta name="citation_reference" content="Lancichinetti A, Radicchi F, Ramasco JJ, Fortunato S. 2011. Finding statistically significant communities in networks. PLoS One 6: e18961." />
<meta name="citation_reference" content="Petitjean C, Makarova KS, Wolf YI, Koonin EV. 2017. Extreme deviations from expected evolutionary rates in archaeal protein families. Genome Biol Evol 9: 2791–2811." />
<meta name="twitter:title" content="Origins and Evolution of the Global RNA Virome" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.

IMPORTANCE The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses." />
<meta name="og-title" property="og:title" content="Origins and Evolution of the Global RNA Virome" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/451740v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.

IMPORTANCE The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-10-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/451740v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/451740v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/451740v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/451740v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/135497" />
    <title>Origins and Evolution of the Global RNA Virome | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DEB6RVyh3kZY4EgP-GC4kFRCe90cQX-iqadF5WZIu4k","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"135497","apath":"\/biorxiv\/early\/2018\/10\/24\/451740.atom","pisa":"biorxiv;451740v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;451740v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/451740v1","title":"Origins and Evolution of the Global RNA Virome","identifier":"node\/135497"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224hog8pm0MhYtiwVWD2RgtjyV\\\\\\\/pQDQKGPzezXrI9tPrGgd9wH7zeT3OkZeL5j3yFT\\\\\\\/15xt6NHzWCn5A0W6mo6akkFO+YsbzcTo4FmVDKB7ybL1fzVGyvJCQHzolsLL9cJn6NXRkva2iYMhA3edDa4OWAGOduM0QJr3+I9iU1+SbCXqWtRi7C4dmCw3vUlp5+I+SWPJWug1rCORPYRtKqpxyHCq13N9lHb83fmheYdhiItU\\\\\\\/YJ\\\\\\\/6tjlQBGqwszv25\\\\\\\/qCqBpZfc94uFatdJfOKi\\\\\\\/NAWj5OydrbMo4miuJHegM4u3nKI7Zk7UYDaOql4SQxiOs45PtOyU\\\\\\\/wGCekZTtVc60B4aYf9OhZNxZohnYYmuUnfu4iFEQt0LUAP2PvkjdYL4vRmQiUK5DYrSv09nJU4Mtru+5Sxz\\\\\\\/9QkatbWTvV6VRhYAzM4f4iqdpvFagTa0SfQggI29PvaCwIi4c1YHic++47Tf\\\\\\\/B5X1Yf65pFqV1838uJadcMWqVEtgC4kvIrz2Q7CCb6KOmEWkpeUKqGS13D98PLOFAVzk90vU3+mr+3DDvdE5By8IGzxr7GDOgRCUdDPei9qqaC4Iz2GVCg9ZBWkccTPasMdfFH0BW4Tt6ewkqIRvFP+nS5Az\\\\\\\/SFO56uS1WSErcUf5OTq9N7Mb51Ht84R2034d2rf4EpkZL666E+lv1vNldoPC+GFb5QKHNfED44hX8RXLGvWXwaGdh\\\\\\\/EQBBb7qHPjgOGYQwn3820am1GMOs6SHj4ptkxDk3PFIa+RSApv3Z2wfIo+cJ\\\\\\\/bYsKuFgKjavgEp0EkPr+5PUGaewbggMpsR1QLAWfJvWN0UzGkDq47SAOjVbq\\\\\\\/Vn+ON2mL3dFNhAThirKmqzrXpzl5un17TptqeXQRxhxNtndIxSAh1nfVQx+H729b4I28ssAveGAY17uSW7lqdxzQnPiWfSAc\\\\\\\/cefNyk60AzS8npZF6djbufLzfa\\\\\\\/jpBSjhH6n1mRQ9ZCiGlWOMyxS8yWKCulCcQmXwnKpC7FWmphIwQztTGa\\\\\\\/ui1Uc57msDvjT6vE\\\\\\\/zdPpLPB0zVn8oCJz+9bP6XwrYZNJKU\\\\\\\/QCF99ME5tEAt2o9jfBkR2JeD3\\\\\\\/RuXe472GyibvVE4qlo1\\\\\\\/ac0oKLiYRUaSSS3AbnoCGtoKU4maryg1qhe2+3OVhdiZra4\\\\\\\/\\\\\\\/BqJk+B2oaMJwW0S6DfFImgq\\\\\\\/8MfsG2Q5KnyjH1E2sdarv4BaGf\\\\\\\/w2oRNbKgxyjOpeFUKhOlu92d0ho28QnDTfkbGxFB14u\\\\\\\/jqO0r6mR33swEcQ6VqEsAvsXhagVyNykvpmhQVNgo6xRCTWmvAlgtHRdEqP8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022OkThgR5PJNUkUmzYnx7SrA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022fc4aa36e569b94a5bb9bd43b433a4b14e3749421041558fcb79859b1a90382e5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/451740v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-135497 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/451740v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lUn1cr5euJJ1SKeIyvSy_kDak5daYU-sByFc8wj2MZs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="135497" id="node135497" data-pisa="biorxiv;451740v1" data-pisa-master="biorxiv;451740" data-apath="/biorxiv/early/2018/10/24/451740.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Origins and Evolution of the Global RNA Virome</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yuri I.</span> <span class="nlm-surname">Wolf</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Darius</span> <span class="nlm-surname">Kazlauskas</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jaime</span> <span class="nlm-surname">Iranzo</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adriana</span> <span class="nlm-surname">LucÍa-Sanz</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7800-6045" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jens H.</span> <span class="nlm-surname">Kuhn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mart</span> <span class="nlm-surname">Krupovic</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Valerian V.</span> <span class="nlm-surname">Dolja</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3943-8299" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Eugene V.</span> <span class="nlm-surname">Koonin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/451740 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node135497" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yuri I. Wolf </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health</span>, Bethesda, Maryland, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuri%2BI.%2BWolf%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolf%20YI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuri%2BI.%2BWolf%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Darius Kazlauskas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Vilniaus universitetas biotechnologijos institutas</span>, Vilnius, <span class='nlm-country'>Lithuania</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Département de Microbiologie, Institut Pasteur, Paris</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darius%2BKazlauskas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kazlauskas%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarius%2BKazlauskas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jaime Iranzo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health</span>, Bethesda, Maryland, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BIranzo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Iranzo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BIranzo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Adriana LucÍa-Sanz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health</span>, Bethesda, Maryland, USA</div><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Centro Nacional de Biotecnología</span>, Madrid, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adriana%2BLuc%C3%8Da-Sanz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luc%C3%8Da-Sanz%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdriana%2BLuc%25C3%258Da-Sanz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jens H. Kuhn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>Integrated Research Facility at Fort Detrick, National Institute of Allergy and Infectious Diseases, National Institutes of Health</span>, Frederick, Maryland, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BH.%2BKuhn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhn%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BH.%2BKuhn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7800-6045" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jens H. Kuhn</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mart Krupovic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Département de Microbiologie, Institut Pasteur, Paris</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mart%2BKrupovic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krupovic%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMart%2BKrupovic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Valerian V. Dolja </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>f</span><span class='nlm-institution'>Department of Botany and Plant Pathology, Oregon State University</span>, Corvallis, Oregon, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valerian%2BV.%2BDolja%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolja%20VV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AValerian%2BV.%2BDolja%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:doljav@oregonstate.edu" class="" data-icon-position="" data-hide-link-title="0">doljav@oregonstate.edu</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Eugene V. Koonin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health</span>, Bethesda, Maryland, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eugene%2BV.%2BKoonin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koonin%20EV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEugene%2BV.%2BKoonin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3943-8299" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eugene V. Koonin</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/451740v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:135497" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:135497/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/451740v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:135497" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:135497/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/451740v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:135497" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:135497/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/451740v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:135497" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:135497/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/451740v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:135497" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:135497/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Viruses with RNA genomes dominate the eukaryotic virome, reaching enormous diversity in animals and plants. The recent advances of metaviromics prompted us to perform a detailed phylogenomic reconstruction of the evolution of the dramatically expanded global RNA virome. The only universal gene among RNA viruses is the RNA-dependent RNA polymerase (RdRp). We developed an iterative computational procedure that alternates the RdRp phylogenetic tree construction with refinement of the underlying multiple sequence alignments. The resulting tree encompasses 4,617 RNA virus RdRps and consists of 5 major branches, 2 of which include positive-sense RNA viruses, 1 is a mix of positive-sense (+) RNA and double-stranded (ds) RNA viruses, and 2 consist of dsRNA and negative-sense (−) RNA viruses, respectively. This tree topology implies that dsRNA viruses evolved from +RNA viruses on at least two independent occasions, whereas -RNA viruses evolved from dsRNA viruses. Reconstruction of RNA virus evolution using the RdRp tree as the scaffold suggests that the last common ancestors of the major branches of +RNA viruses encoded only the RdRp and a single jelly-roll capsid protein. Subsequent evolution involved independent capture of additional genes, particularly, those encoding distinct RNA helicases, enabling replication of larger RNA genomes and facilitating virus genome expression and virus-host interactions. Phylogenomic analysis reveals extensive gene module exchange among diverse viruses and horizontal virus transfer between distantly related hosts. Although the network of evolutionary relationships within the RNA virome is bound to further expand, the present results call for a thorough reevaluation of the RNA virus taxonomy.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>IMPORTANCE</strong> The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than what was attainable previously. This reconstruction reveals the relationships between different Baltimore Classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">Early evolution of life is widely believed to have first involved RNA molecules that functioned both as information storage devices and as catalysts (ribozymes) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Subsequent evolution involved the emergence of DNA, the dedicated genomic material, and proteins, the ultimate operational molecules. RNA molecules remained central for translating information from genes to proteins (mRNA), the functioning of the translation machinery itself (rRNA and tRNA), and for a variety of regulatory functions (various classes of non-coding RNA that are increasingly discovered in all life forms) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Viruses with RNA genomes (“RNA viruses” for short) that do not involve DNA in their genome replication and expression cycles (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>) can be considered the closest extant recapitulation, and possibly, a relic of the primordial RNA world.</p><p id="p-5">RNA viruses comprise 3 of the 7 so-called Baltimore Classes of viruses that differ with respect to the nature of the genome (i.e., the nucleic acid form that is packaged into virions) and correspond to distinct strategies of genome replication and expression: positive-sense (+) RNA viruses, double-stranded (ds) RNA viruses and negative-sense (−) RNA viruses (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). The +RNA viruses use the simplest possible strategy of replication and expression as the same molecule functions as both genome and mRNA (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). Most likely, the first replicators to emerge in the RNA world, after the evolution of translation, resembled +RNA viruses (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). Because the +RNA released from a virion can be directly used for translation to produce viral proteins, virions of +RNA viruses contain only structural proteins, in addition to the genome. In contrast, -RNA and dsRNA viruses package their transcription and replication machineries into their virions because these are necessary to initiate the virus reproduction cycles (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>).</p><p id="p-6">RNA viruses comprise a major part of the global virome. In prokaryotes, the known representation of RNA viruses is narrow. Only one family of +RNA viruses <em>(Leviviridae)</em> and one family of dsRNA viruses <em>(Cystoviridae)</em> are formally recognized, and furthermore, their members have limited host ranges. No-RNA viruses have been isolated from prokaryotes (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Although recent metagenomic studies suggest that genetic diversity and host range of prokaryotic RNA viruses could be substantially underestimated (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>), it appears that the scope of the prokaryotic RNA virome is incomparably less than that of the DNA virome. As discussed previously, potential causes of the vast expansion of the RNA virome in eukaryotes might include the emergence of the compartmentalized cytosol that provided a hospitable, protective environment for RNA replication that is known to be associated with the endoplasmic reticulum and other membrane compartments (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). Conversely, the nuclear envelope could be a barrier that prevents the access of DNA viruses to the host replication and transcription machineries and thus partially relieves the stiff competition that DNA viruses of prokaryotes represent for RNA viruses.</p><p id="p-7">In a sharp contrast, the 3 Baltimore Classes of RNA viruses dominate the eukaryotic virosphere (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Eukaryotes from all major taxa are hosts to RNA viruses, and particularly in plants and invertebrates, these viruses are enormously abundant and diverse (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>-<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). Until recently, the study of RNA viromes had been heavily skewed towards viruses infecting humans, livestock, and agricultural plants. Because of these limitations and biases, the attempts to reconstruct the evolutionary history of RNA viruses were bound to be incomplete. Nonetheless, these studies have yielded important generalizations. A single gene encoding an RNA-dependent RNA polymerase (RdRp) is universal among RNA viruses, including capsid-less RNA replicons but excluding some satellite viruses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). Even within each of the 3 Baltimore Classes, virus genomes do not include fully conserved genes other than those encoding RdRps (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>). However, several hallmark genes are shared by broad ranges of RNA viruses including, most notably, those encoding capsid proteins of icosahedral and helical virions of +RNA viruses (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), and key enzymes involved in virus replication such as distinct helicases and capping enzymes (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>).</p><p id="p-8">The RdRp gene and encoded protein are natural targets of evolutionary analysis because they are the only universal gene (protein) in RNA viruses. However, obtaining strongly supported phylogenies for RdRps is a difficult task due to the extensive sequence divergence, apart from several conserved motifs that are required for polymerase activity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>-<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). RdRps belong to the expansive class of polymerases containing so-called Palm catalytic domains along with the accessory Fingers and Thumb domains (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). In addition to viral RdRps, Palm domain polymerases include reverse transcriptases (RTs) of retroelements and reverse-transcribing viruses, and DNA polymerases that are responsible for genome replication in cellular organisms and diverse DNA viruses. Within the Palm domain class of proteins, RT and the +RNA virus RdRps are significantly similar in sequence and structure, and appear to comprise a monophyletic group (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>-<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). More specifically, the highest similarity is observed between the +RNA virus RdRps and the RTs of group II introns. These introns are wide-spread retrotransposons in prokaryotes that are thought to be ancestral to the RTs of all other retrotransposons as well as retroviruses and pararetroviruses (recently jointly classified as the order <em>Ortervirales)</em> of eukaryotes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>).</p><p id="p-9">A phylogenetic analysis of the +RNA virus RdRps revealed only a distant relationship between the leviviruses and the bulk of the eukaryotic +RNA viruses, leaving the ancestral relationships uncertain (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). The origin of eukaryotic +RNA viruses from their prokaryotic counterparts is an obvious possibility. However, given the dramatically greater prevalence of +RNA viruses among eukaryotes compared to the narrow spread of leviviruses and “levi-like viruses” in bacteria, an alternative scenario has been proposed. Under this scenario, RdRps of the prokaryotic and eukaryotic +RNA viruses independently descended from distinct RTs (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). Among eukaryotic +RNA viruses, phylogenetic analysis of the RdRps strongly supports the existence of picornavirus and alphavirus “supergroups”, which are further validated by additional signature genes (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>). However, both the exact compositions of these supergroups and the evolutionary relationships among many additional groups of viruses remain uncertain. Some RdRp phylogenies suggest a third supergroup combining animal “flavi-like viruses” and plant tombusviruses but this unification is not supported by additional shared genes and thus remains tenuous (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>).</p><p id="p-10">The similarity of RdRps among dsRNA viruses is limited but these RdRps are similar to varying degree to +RNA virus RdRps. Therefore, different groups of dsRNA viruses might have evolved from different +RNA viruses independently, on multiple occasions (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Although it is not entirely clear how prokaryotic dsRNA viruses fit into this concept, evidence of an evolutionary affinity between cystoviruses and reoviruses has been presented (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>).</p><p id="p-11">For a long time, the evolutionary provenance of-RNA virus RdRps remained uncertain due to their low sequence similarity to other RdRps and RTs (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). However, recent protein structure comparisons point to a striking similarity between the RdRps of-RNA orthomyxoviruses and those of +RNA flaviviruses and dsRNA cystoviruses (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). All these findings notwithstanding, the overall evolutionary relationships among the RdRps of +RNA,-RNA, and dsRNA viruses, and RTs remain unresolved. In particular, whether the RdRps of +RNA and-RNA viruses are mono-or polyphyletic, is unclear.</p><p id="p-12">Many deep evolutionary connections between RNA virus groups that originally were thought to be unrelated have been delineated using the results of pre-metagenomic era evolutionary studies. These discoveries culminated in the establishment of RNA virus supergroups (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>).</p><p id="p-13">However, the evolutionary provenance of many other RNA virus groups remained unclear, and so did the relationships between the RNA viruses of the 3 Baltimore Classes and retroelements, and their ultimate origins. The prospects of substantial progress appeared dim because of the extreme sequence divergence among RNA viruses, which could amount to irrevocable loss of evolutionary information.</p><p id="p-14">Recent revolutionary developments in virus metagenomics (metaviromics) dramatically expanded the known diversity of RNA viruses and provided an unprecedented amount of sequence data for informed investigation into RNA virus evolution (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>). The foremost development was the massive expansion of the known invertebrate virome that was achieved primarily through meta-transcriptome sequencing of various holobionts. The subsequent phylogenetic analysis revealed previously unknown lineages of +RNA and-RNA viruses and prompted reconsideration of high rank virus unifications, such as +RNA virus supergroups (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>-<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). The RNA viromes of fungi and prokaryotes also underwent notable expansion albeit not as massive as that of invertebrates (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>).</p><p id="p-15">Here we reexamine the evolutionary relationships among and within the 3 Baltimore Classes of RNA viruses through a comprehensive analysis of the available genomic and metagenomic sequences. In particular, to build a phylogenetic tree of thousands of viral RdRps, we designed an iterative computational procedure that alternates phylogeny construction with refinement of the underlying multiple alignments. Although RNA viruses have relatively short genomes (~3-41 kb), the combined gene repertoire (pangenome) of these viruses includes numerous genes that are shared, to varying degrees, by related subsets of RNA viruses. To obtain further insight into virus evolution, we therefore attempted to reconstruct the history of gain and loss of conserved proteins and domains in different virus lineages. We also investigated evolution of the single jelly-roll capsid protein (SJR-CP), the dominant type of capsid protein among +RNA viruses. Our analysis revealed patterns that are generally congruent with the RdRp phylogeny and provide further insights into the evolution of different branches of RNA viruses. Finally, we analyzed a bipartite network in which RNA virus genomes are connected via nodes representing virus genes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>) to identify distinct modules in the RNA virosphere. The results shed light on the evolution of RNA viruses, revealing, in particular, the monophyly of-RNA viruses and their apparent origin from dsRNA viruses, which seem to have evolved from distinct branches of +RNA viruses on at least two independent occasions.</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><div id="sec-4" class="subsection"><h3>Comprehensive phylogeny of RNA virus RNA-dependent RNA polymerases: overall structure of the tree and the 5 major branches</h3><p id="p-16">Amino acid sequences of RdRps and RTs were collected from the non-redundant National Center for Biotechnology Information (NCBI) database and analyzed using an iterative clustering-alignment-phylogeny procedure (Figure S1; see Materials and Methods for details). This procedure ultimately yielded a single multiple alignment of the complete set of 4,617 virus RdRp sequences (Data set S1) and 1,028 RT sequences organized in 50+2 clusters (50 clusters of RdRps and 2 clusters of RTs; see Materials and Methods for details). This sequence set did not include RdRps of members of the families <em>Birnaviridae</em> or <em>Permutatetraviridae,</em> distinct groups of RNA viruses that encompass a circular permutation within the RdRp Palm domain (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>) and therefore could not be confidently included in the alignment over their entire lengths.</p><p id="p-17">The phylogenetic tree of RdRps and RTs (Data set S2 and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) was assembled from a set of trees that represent three hierarchical levels of relationships. At the lowest level, full complements of sequences from each cluster were used to construct cluster-specific trees. At the intermediate level, up to 15 representatives from each cluster were selected to elucidate supergroup-level phylogeny. At the highest level, up to 5 representatives from each cluster were taken to resolve global relationships (Data set S3A and S4). The final tree (Data set S2) was assembled by replacing the cluster representatives with the trees from the previous steps.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogeny of RNA virus RNA-dependent RNA polymerases (RdRps) and reverse transcriptases (RTs): the main branches (1-5). Each branch represents collapsed sequences of the respective set of RdRps. The 5 main branches discussed in the text are labeled accordingly. The bootstrap support obtained by (numerator) and by (denominator) is shown for each internal branch. LTR, long-terminal repeat." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogeny of RNA virus RNA-dependent RNA polymerases (RdRps) and reverse transcriptases (RTs): the main branches (1-5). Each branch represents collapsed sequences of the respective set of RdRps. The 5 main branches discussed in the text are labeled accordingly. The bootstrap support obtained by (numerator) and by (denominator) is shown for each internal branch. LTR, long-terminal repeat.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F1.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.1" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F1.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure.1</span> <p id="p-18" class="first-child">Phylogeny of RNA virus RNA-dependent RNA polymerases (RdRps) and reverse transcriptases (RTs): the main branches (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>). Each branch represents collapsed sequences of the respective set of RdRps. The 5 main branches discussed in the text are labeled accordingly. The bootstrap support obtained by (numerator) and by (denominator) is shown for each internal branch. LTR, long-terminal repeat.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The large number and immense diversity of the viruses included in our analysis create serious challenges for a systematic, phylogeny-based nomenclature of the identified evolutionary lineages of RNA viruses. Many such lineages consist of viruses newly discovered by metaviromics, are not yet formally classified by the International Committee on Taxonomy of Viruses (ICTV), and therefore, cannot be assigned formal names. For the purpose of the present work, we adopted a semi-arbitrary naming scheme using the following approach. 1) We use taxon names that have been fully accepted by the ICTV as of March 2018 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>) whenever possible. These names are recognizable through their capitalization and italicization and rank-specific suffixes (e.g., <em>-virales</em> for orders, <em>-viridae</em> for families). As is the common practice in virus taxonomy, the officially classified members of each ICTV-approved taxon are referred to via vernaculars (recognizable through their lack of capitalization and italicization). For instance, the members of the ICTV-approved order <em>Bunyavirales</em> are called bunyaviruses, whereas those of the family <em>Tombusviridae</em> are called tombusviruses. However, in this work, both taxon and vernacular terms are to be understood <em>sensu lato:</em> if our analysis indicates certain viruses to be members of or very closely related to an ICTV-established taxon, we consider them members of that taxon despite the lack of current ICTV recognition. As a result, in our analysis, the order <em>Bunyavirales</em> has more members than in the official taxonomy. 2) we use vernacular names in quotation marks for viruses/lineages that are clearly distinct from those covered by the official ICTV framework. Whenever possible, we use names that circulate in the literature (e.g., “hepeliviruses”, “statoviruses”). In the absence of such unofficial names, we name the lineage reminiscent of the next closely related lineage (e.g., “levi-like viruses” are a clearly distinct sister group to <em>Leviviridae</em>/leviviruses). 3) Monophyletic clusters that transcend the currently highest ICTV-accepted rank (i.e., order) are labeled according to terms circulating in the literature (i.e., “alphavirus supergroup”, “flavivirus supergroup”, and “picornavirus supergroup”). 4) Lineages represented by a single virus are labeled with the respective virus name.</p><p id="p-20">Rooting the phylogenetic tree, generated using PhyML (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>), between the RTs and RdRps resulted in a well-resolved topology of RNA viruses which the tree splits into 5 major branches, each including substantial diversity of viruses (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-21">Branch 1: leviviruses and their eukaryotic relatives, namely, “mitoviruses”, “narnaviruses” and “ourmiaviruses” (the latter three terms are placed in quotation marks as our analysis contradicts the current ICTV framework, which considers mitoviruses and narnaviruses members of one family, <em>Narnaviridae,</em> and ourmiaviruses members of a free-floating genus <em>Ourmiavirus);</em></p><p id="p-22">Branch 2 (“picornavirus supergroup”): a large assemblage of +RNA viruses of eukaryotes, in particular, those of the orders <em>Picornavirales</em> and <em>Nidovirales,</em> the families <em>Caliciviridae, Potyviridae, Astroviridae,</em> and <em>Solemoviridae,</em> a lineage of dsRNA viruses including partitiviruses and picobirnaviruses; and several other, smaller groups of +RNA and dsRNA viruses.</p><p id="p-23">Branch 3: a distinct subset of +RNA viruses including the “alphavirus supergroup” along with “flavivirus supergroup”, nodaviruses, and tombusviruses, the “statovirus”, “weivirus”, “yanvirus”, “zhaovirus” groups; and several additional, smaller groups.</p><p id="p-24">Branch 4: dsRNA viruses including cystoviruses, reoviruses, and totivirues, and several additional families.</p><p id="p-25">Branch 5:-RNA viruses.</p><p id="p-26">Each of these 5 major branches of the tree is strongly supported by bootstrap replications (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Assuming the RT rooting of the tree, Branch 1, which consists of leviviruses and their relatives infecting eukaryotes, is a sister group to the rest of RNA viruses; this position is highly robust to the choice of the phylogenetic method and parameters. This tree topology is compatible with the monophyly of the RdRps and, by inference, of RNA viruses and the origin of eukaryotic RNA viruses from a prokaryotic RNA virus ancestor shared with leviviruses. The deeper history remains murky. We have no information on the nature of the common ancestor of retroelements and RNA viruses, let alone whether the ancestor was an RNA virus or a retroelement. However, parsimony considerations suggest that a retroelement ancestor is more likely given that capsids first appeared in the virus part of the tree rather than having been lost in retroelements.</p><p id="p-27">The next split in the tree occurs between Branch 2 and the short stem that formally joins Branches 3, 4, and 5. However, the unification of Branch 3 with Branches 4 and 5 is weakly supported and might not reflect actual common ancestry.</p><p id="p-28">Arguably, the most striking feature of the RNA virus tree topology is the paraphyly of +RNA viruses relative to dsRNA and-RNA viruses. Indeed, according to this phylogeny,-RNA viruses evolved from within dsRNA viruses, whereas dsRNA viruses are polyphyletic (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). One major group of dsRNA viruses that includes partitiviruses and picobirnaviruses is firmly embedded within the +RNA virus Branch 2, whereas another, larger dsRNA virus group including cystoviruses, reoviruses, totiviruses and viruses from several other families comprise the distinct Branch 4 that might be related to +RNA virus Branch 3 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). This placement of the two branches of dsRNA viruses is conceptually compatible with the previous evolutionary scenarios of independent origins from +RNA viruses. However, the presence of a strongly supported branch combining 3 lineages of dsRNA viruses that infect both prokaryotes and eukaryotes suggests a lesser extent of polyphyly in the evolution of dsRNA viruses than originally proposed (<a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>).</p><p id="p-29">An alternative phylogenetic analysis of the same RdRp alignment using RAxML yielded the same 5 main branches, albeit some with weak support (Data set S3B). Furthermore, although the dsRNA viruses are split the same way using RAxML as in the PhyML tree, the nested tree topology, in which Branch 4 (the bulk of dsRNA viruses) is lodged deep within +RNA viruses, and Branch 5 (-RNA viruses) is located inside Branch 4, is not reproduced (Data set S3B). Instead, Branches 4 and 5 are separate and positioned deep in the tree, right above the split between Branch 1 and the rest of the RdRps. Given the poor resolution of the RAxML tree and a strong biological argument, namely, the absence of identified-RNA viruses in prokaryotes or protists (with the exception of the “leishbuviruses” infecting kinetoplastids (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>); see also discussion below), we believe that the tree topology in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> carries more credence than that shown in Data set S3B. Nevertheless, these discrepancies emphasize that utmost caution is due when biological interpretation of deep branching in trees of highly divergent proteins is attempted.</p></div><div id="sec-5" class="subsection"><h3>Evolution of the 5 major branches of RNA viruses and reconstruction of gene gain and loss events</h3><div id="sec-6" class="subsection"><h4>Reconstruction of gene gains and losses</h4><p id="p-30">The RdRp is the only universal protein of RNA viruses. Accordingly, other viral genes must have been gained and/or lost at different stages of evolution. Thus, after performing the phylogenetic analysis of RdRps, we assigned the proteins and domains shared by diverse viruses to the branches of the RdRp tree. Multiple alignments and hidden Markov model (hmm) profiles were constructed for 16,814 proteins and domains encoded by RNA viruses, and a computational pipeline was developed to map these domains on the viral genomes (Data set S5). The resulting patterns of domain presence/absence in the branches of the RdRp tree were used to reconstruct the history of the gains and losses of RNA virus genes (or proteins and domains, for simplicity), both formally, using the ML-based Gloome method (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>), and informally, from parsimony considerations.</p><p id="p-31">These reconstructions reveal a high level of branch-specificity in the evolution of the gene repertoire of RNA viruses. The only protein that is likely to have been gained at the base of the eukaryotic virus subtree (Branches 2 to 5) (that also includes the bacterial cystoviruses) is the single jelly-roll capsid protein (SJR-CP) (Figure S2A). Retroelements lack capsid proteins, and therefore, there is no indication that SJR-CP was present in the hypothetical element that encoded the common ancestor of RdRps and RTs. Furthermore, reconstruction of the evolution of Branch 1 (leviviruses and their relatives) argues against the ancestral status of SJR-CP in this branch.</p></div><div id="sec-7" class="subsection"><h4>Branch 1: leviviruses and their descendants</h4><p id="p-32">The current RdRp tree topology combined with gene gain-loss reconstruction suggests the following evolutionary scenario for Branch 1 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>): a levivirus-like ancestor that, like the extant members of the <em>Leviviridae,</em> possessed a capsid protein unrelated to SJR-CP (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>) gave rise to naked eukaryotic RNA replicons known as “mitoviruses” and “narnaviruses”. These replicons consist of a single RdRp gene (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>) and replicate either in mitochondria or in the cytosol of the host cells, respectively, of fungal and invertebrate hosts (the latter hosts were identified in metaviromic holobiont analyses) (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). Recently, the existence of plant “mitoviruses” has been reported although it is not known whether these viruses reproduce in the mitochondria (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). The “narnavirus” RdRp is also the ancestor of the RdRp of the expanding group of “ourmiaviruses” (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). “Ourmiaviruses” were originally identified in a narrow range of plants, and genomic analysis revealed the chimeric nature of these viruses, with a “narnavirus”-like RdRp but SJR-CPs and movement proteins (MPs) which were apparently acquired from “picorna-like” and “tombus-like” viruses, respectively (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). Use of metaviromics has led to the identification of numerous related viruses associated with invertebrates, many of which encode distinct SJR-CP variants and some of which acquired an RNA helicase (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> and S2) (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>). Thus, the evolution of this branch apparently involved the loss of the structural module of leviviruses, which yielded naked RNA replicons that reproduced in the mitochondria of early eukaryotes. A group of these replicons subsequently escaped to the cytosol which was followed by the reacquisition of unrelated structural modules from distinct lineages of eukaryotic viruses inhabiting the same environment (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). This complex evolutionary scenario emphasizes the key role of modular gene exchange in the evolution of RNA viruses.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Branch 1 of the RNA virus RNA-dependent RNA polymerases (RdRps): leviviruses and their relatives. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7) whereas yellow dot corresponds to a weakly supported branch. (B) Genome maps of a representative set of Branch 1 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CP, capsid protein; MP, movement protein; S3H, superfamily 3 helicase; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2 (see Figure 7)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Branch 1 of the RNA virus RNA-dependent RNA polymerases (RdRps): leviviruses and their relatives. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7) whereas yellow dot corresponds to a weakly supported branch. (B) Genome maps of a representative set of Branch 1 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CP, capsid protein; MP, movement protein; S3H, superfamily 3 helicase; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2 (see Figure 7).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F2.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.2" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F2.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.2</span> <p id="p-33" class="first-child">Branch 1 of the RNA virus RNA-dependent RNA polymerases (RdRps): leviviruses and their relatives. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7) whereas yellow dot corresponds to a weakly supported branch. (B) Genome maps of a representative set of Branch 1 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CP, capsid protein; MP, movement protein; S3H, superfamily 3 helicase; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2 (see <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Branch 2: Picornavirus supergroup</h4><p id="p-34">The expansive Branch 2 generally corresponds to the previously described “picornavirus supergroup” (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>) (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>). Some of the virus groups that were previously considered peripheral members of this supergroup, such as totiviruses and nodaviruses, were relocated to different branches in the present tree (Branches 4 and 3, respectively), whereas the viruses of the order <em>Nidovirales</em> were moved inside Branch 2 from an uncertain position in the tree. Nevertheless, the core of the supergroup remains coherent, suggestive of common ancestry. Within Branch 2, 3 major clades are strongly supported (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>); however, many of the internal branches are less reliable, so that the relative positions of partitivirus-picobirnavirus, potyvirus-astrovirus and nidovirus clades within Branch 2 remain uncertain.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Branch 2 of the RNA virus RNA-dependent RNA polymerases (RdRps): &#x201C;picornavirus supergroup&#x201D; of the +RNA viruses expanded to include nidoviruses and two groups of dsRNA viruses, partitiviruses and picobirnaviruses. (A) Phylogenetic tree of the virus RdRps showing ICTV accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7). Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified; vert., vertebrate. (B) Genome maps of a representative set of Branch 2 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). 3Cpro, 3C chymotrypsin-like protease; CP, capsid protein; E, envelope protein; En, nidoviral uridylate-specific endoribonuclease (NendoU); Exo, 3&#x2019;-to-5&#x2019; exoribonuclease domain; fCP, capsid protein forming filamentous virions; M, membrane protein; MD macro domain; MP, movement protein; MT, ribose-2-O-methyltransferase domain; N, nucleocapsid protein; N7, guanine-N7-methyltransferase; Ppro, papain-like protease; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2; spike, spike protein; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; S3H, superfamily 3 helicase; VP2, virion protein 2; Z, Zn-finger domain; Spro, serine protease; P3, protein 3. Distinct hues of same color (e.g., green for MPs) are used to indicate the case when proteins that share analogous function are not homologous." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Branch 2 of the RNA virus RNA-dependent RNA polymerases (RdRps): &#x201C;picornavirus supergroup&#x201D; of the +RNA viruses expanded to include nidoviruses and two groups of dsRNA viruses, partitiviruses and picobirnaviruses. (A) Phylogenetic tree of the virus RdRps showing ICTV accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7). Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified; vert., vertebrate. (B) Genome maps of a representative set of Branch 2 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). 3C&lt;sup&gt;pro&lt;/sup&gt;, 3C chymotrypsin-like protease; CP, capsid protein; E, envelope protein; En, nidoviral uridylate-specific endoribonuclease (NendoU); Exo, 3&#x2019;-to-5&#x2019; exoribonuclease domain; fCP, capsid protein forming filamentous virions; M, membrane protein; MD macro domain; MP, movement protein; MT, ribose-2-O-methyltransferase domain; N, nucleocapsid protein; N7, guanine-N7-methyltransferase; Ppro, papain-like protease; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2; spike, spike protein; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; S3H, superfamily 3 helicase; VP2, virion protein 2; Z, Zn-finger domain; Spro, serine protease; P3, protein 3. Distinct hues of same color (e.g., green for MPs) are used to indicate the case when proteins that share analogous function are not homologous.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F3.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.3" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F3.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.3</span> <p id="p-35" class="first-child">Branch 2 of the RNA virus RNA-dependent RNA polymerases (RdRps): “picornavirus supergroup” of the +RNA viruses expanded to include nidoviruses and two groups of dsRNA viruses, partitiviruses and picobirnaviruses. (A) Phylogenetic tree of the virus RdRps showing ICTV accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7). Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified; vert., vertebrate. (B) Genome maps of a representative set of Branch 2 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). 3C<sup>pro</sup>, 3C chymotrypsin-like protease; CP, capsid protein; E, envelope protein; En, nidoviral uridylate-specific endoribonuclease (NendoU); Exo, 3’-to-5’ exoribonuclease domain; fCP, capsid protein forming filamentous virions; M, membrane protein; MD macro domain; MP, movement protein; MT, ribose-2-O-methyltransferase domain; N, nucleocapsid protein; N7, guanine-N7-methyltransferase; Ppro, papain-like protease; SJR1 and SJR2, single jelly-roll capsid proteins of type 1 and 2; spike, spike protein; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; S3H, superfamily 3 helicase; VP2, virion protein 2; Z, Zn-finger domain; Spro, serine protease; P3, protein 3. Distinct hues of same color (e.g., green for MPs) are used to indicate the case when proteins that share analogous function are not homologous.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The largest and most coherent of the Branch 2 clades includes the cornerstone of the picornavirus supergroup, the ~826 viruses-strong order <em>Picornavirales</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), expanded with caliciviruses, solinviviruses and a multitude of unclassified viruses infecting invertebrates, vertebrates, fungi, protists and undefined hosts (for viruses discovered by metaviromics) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>-<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). The second largest, deep-branching clade consists of two lineages that include, respectively, +RNA and dsRNA viruses (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). The +RNA virus lineage combines astroviruses and potyviruses, the evolutionary affinity of which is well recognized (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>). The dsRNA lineage includes the members of the families <em>Amalgaviridae, Hypoviridae, Partitiviridae,</em> and <em>Picobirnaviridae,</em> with each of these families greatly expanded by unclassified affiliates. Finally, the ‘middle’ clade is smaller and less diverse: it encompasses nidoviruses, including the longest of all +RNA virus genomes (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>), and solemoviruses with much shorter genomes (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Notably, some members of the family <em>Luteoviridae</em> and Heterocapsa circularisquama RNA virus, the only known alvernavirus (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>), are nested within the solemovirus clade. Given the lack of support beyond the phylogenetic affinity of the RdRps and the dramatic differences in the genomic architectures of nidoviruses and solemoviruses, the possibility that this unification is caused by a tree construction artifact is difficult to rule out (the branch support notwithstanding).</p><p id="p-37">Hypoviruses, a group of fungal capsid-less RNA replicons, have been traditionally viewed as dsRNA viruses. However, comparison of genome architectures and phylogenetic analysis suggested that hypoviruses are derivatives of potyviruses that have lost the capsid protein (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>). In the current RdRp tree, hypoviruses cluster with the dsRNA viruses of the partitivirus-picobirnavirus clade rather than with potyviruses (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Whether this position is an artifact of tree construction or whether hypoviruses actually share the RdRps with dsRNA viruses is unclear.</p><p id="p-38">The partitivirus-picobirnavirus clade within Branch 2 represents a transition to the <em>bona fide</em>dsRNA Baltimore Class (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). Typical partitiviruses and picobirnaviruses have minimalist genomes that consist of two dsRNA segments encapsidated separately into distinct 120-subunit T=1 capsids (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>-<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>). These genome segments encode, respectively, RdRps and CPs that are clearly homologous between the two families. The CPs of the partitivirus-picobirnavirus clade have been suggested to be distantly related to those of other dsRNA viruses that belong to Branch 4 (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>). Notably, this clade also includes some naked RNA replicons that reproduce in algal mitochondria or chloroplasts, use a mitochondrial genetic code and, in terms of lifestyle, resemble “mitoviruses” (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>). By analogy, the origin of the partitivirus-picobirnavirus group from an as-yet undiscovered lineage of prokaryotic RNA viruses seems likely. More specifically, this group of dsRNA viruses could have evolved through reassortment of genomic segments encoding, respectively, a +RNA virus RdRp of Branch 2 (possibly a naked RNA replicon) and a dsRNA virus capsid protein related to those of Branch 4 viruses. The most recently evolved branch of partitiviruses is characterized by larger, 4-6-partite genomes, in contrast to mono-or bipartite genomes in the deeper branches (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">14</a>). This observation emphasizes a major tendency in virus evolution: increase in genome complexity via gradual acquisition of accessory genes (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>).</p><p id="p-39">Apart from the SJR-CP, an apparently ancestral protein that is likely to be a shared derived character (synapomorphy) of Branch 2 is a serine protease that is present in members of the order <em>Picornavirales</em> (with the diagnostic substitution of cysteine for the catalytic serine), members of the potyvirus-astrovirus clade, solemoviruses, alvernavirus, and nidoviruses (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a> and S2B). As demonstrated previously, this viral protease derives from a distinct bacterial protease, probably of mitochondrial origin, which is compatible with an early origin of Branch 2 in eukaryotic evolution (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a>).</p><p id="p-40">The reconstruction of protein gain-loss, together with the comparison of genome architectures in this branch, reveal extensive rearrangements as well as gene and module displacement (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a> and S2B). Branch 2 includes viruses with relatively long genomes and complex gene repertoires (nidoviruses, potyviruses and many members of <em>Picornavirales)</em> along with viruses with much shorter genomes and minimal sets of genes (astroviruses and solemoviruses). Clearly, evolution of Branch 2 viruses involved multiple gene gains. Of special note is the gain of 3 distinct helicases in 3 clades within this branch: superfamily 3 helicases (S3H) in members of <em>Picornavirales,</em> superfamily 2 helicases (S2H) in potyviruses, and superfamily 1 helicases (S1H) in nidoviruses (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3B</a> and S2B). This independent, convergent gain of distinct helicases reflects the trend noticed early in the study of RNA virus evolution, namely, that most viruses with genomes longer than ~6 kb encode helicases, whereas smaller ones do not. This difference conceivably exists because helicase activity is required for the replication of longer RNA genomes (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>). Another notable feature is the change of virion morphology among potyviruses (replacement of ancestral SJR-CP by an unrelated CP forming filamentous virions) and nidoviruses (displacement by a distinct nucleocapsid protein). The dramatic change in virion morphology and mode of genome encapsidation might have been necessitated by the inability of SJR-CP-based icosahedral capsids to accommodate the larger genomes of the ancestral potyviruses and nidoviruses. In addition, nidoviruses gained capping enzymes (CapE; <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3B</a> and S2B) that most likely were acquired independently of the capping enzymes of other RNA viruses. Nidoviruses also gained ribonucleases and other accessory proteins that are involved in genome replication, virulence and other aspects of the infection cycle of these largest known RNA viruses (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>-<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>).</p></div><div id="sec-9" class="subsection"><h4>Branch 3: “Alphavirus supergroup”, “flavivirus supergroup” and the extensive diversity of “tombus-like viruses”</h4><p id="p-41">Branch 3 is the part of the +RNA virus RdRp tree that underwent the most dramatic rearrangements compared to previous versions. This branch consists of two strongly supported clades of +RNA viruses: i) the assemblage that originally was defined as the “alphavirus supergroup” (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">15</a>) joined by several additional groups of viruses; and ii) flaviviruses and related viruses (“flavivirus supergroup”; <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> and 4). In the former clade, the alphavirus supergroup encompasses an enormous diversity of plant, fungal, and animal +RNA viruses, and consists of 3 well-supported lineages: tymoviruses, virgaviruses/alphaviruses/endornaviruses, and hepeviruses/benyviruses, each accompanied by related viruses that often form yet unclassified lineages (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Within the “alphavirus supergroup” alone, the genome lengths range from ~6 to ~20 kb. Despite this length variation, all supergroup members harbor a conserved RNA replication gene module encoding a CapE, S1H, and RdRp; the conservation of this module attests to the monophyly of the supergroup (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Branch 3 of the RNA virus RNA-dependent RNA polymerases (RdRps): Alphavirus superfamily, radiation of related tombusviruses, nodaviruses and unclassified viruses, and flavivirus supergroup. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. (B) Genome maps of a representative set of Branch 3 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). C, nucleocapsid protein; CapE, capping enzyme; CP-Spro, capsid protein-serine protease; E, envelope protein; fCP, divergent copies of the capsid protein forming filamentous virions; Hsp70h, Hsp70 homolog; MP, movement protein; NS, nonstructural protein; nsP2-3, non-structural proteins; Ppro, papain-like protease; prM, precursor of membrane protein; rCP, capsid protein forming rod-shaped virions; RiS, RNA interference suppressor; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; SJR2, single jelly-roll capsid proteins of type 2; Spro, serine protease; vOTU, virus OTU-like protease; NS, nonstructural protein. Distinct hues of same color (e.g., green for MPs) are used to indicate the cases when proteins that share analogous function are not homologous." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Branch 3 of the RNA virus RNA-dependent RNA polymerases (RdRps): Alphavirus superfamily, radiation of related tombusviruses, nodaviruses and unclassified viruses, and flavivirus supergroup. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. (B) Genome maps of a representative set of Branch 3 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). C, nucleocapsid protein; CapE, capping enzyme; CP-Spro, capsid protein-serine protease; E, envelope protein; fCP, divergent copies of the capsid protein forming filamentous virions; Hsp70h, Hsp70 homolog; MP, movement protein; NS, nonstructural protein; nsP2-3, non-structural proteins; Ppro, papain-like protease; prM, precursor of membrane protein; rCP, capsid protein forming rod-shaped virions; RiS, RNA interference suppressor; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; SJR2, single jelly-roll capsid proteins of type 2; Spro, serine protease; vOTU, virus OTU-like protease; NS, nonstructural protein. Distinct hues of same color (e.g., green for MPs) are used to indicate the cases when proteins that share analogous function are not homologous.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F4.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.4" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F4.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.4</span> <p id="p-42" class="first-child">Branch 3 of the RNA virus RNA-dependent RNA polymerases (RdRps): Alphavirus superfamily, radiation of related tombusviruses, nodaviruses and unclassified viruses, and flavivirus supergroup. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. (B) Genome maps of a representative set of Branch 3 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). C, nucleocapsid protein; CapE, capping enzyme; CP-Spro, capsid protein-serine protease; E, envelope protein; fCP, divergent copies of the capsid protein forming filamentous virions; Hsp70h, Hsp70 homolog; MP, movement protein; NS, nonstructural protein; nsP2-3, non-structural proteins; Ppro, papain-like protease; prM, precursor of membrane protein; rCP, capsid protein forming rod-shaped virions; RiS, RNA interference suppressor; S1H, superfamily 1 helicase; S2H, superfamily 2 helicase; SJR2, single jelly-roll capsid proteins of type 2; Spro, serine protease; vOTU, virus OTU-like protease; NS, nonstructural protein. Distinct hues of same color (e.g., green for MPs) are used to indicate the cases when proteins that share analogous function are not homologous.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">In contrast, virion architectures vary dramatically even within each of the three lineages of the “alphavirus supergroup”. The major structural themes include: variants of icosahedral capsids formed by SJR-CP (e.g., bromoviruses, tymoviruses); unrelated icosahedral capsids enveloped in a lipoprotein bilayer (togaviruses); flexuous filamentous capsids formed by a distinct type of CP (alphaflexiviruses, betaflexiviruses, gammaflexiviruses, closteroviruses); and rigid rod-shaped capsids assembled from another distinct CP (benyiviruses, virgaviruses). It was traditionally thought that the latter capsid type is specific to viruses of flowering plants (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>). However, the recent discovery of a virgavirus-like CP in invertebrate viruses (e.g., Beihai charybdis crab virus 1 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>) (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">14</a>) suggests that the emergence of this unique CP fold antedates land colonization by plants by ~100 Mya. Yet another ‘structural’ theme is offered by endornaviruses, naked RNA replicons which, similarly to hypoviruses in Branch 2 (see above), originally were classified as dsRNA viruses. However, endornaviruses possess all the hallmarks of the alphavirus supergroup and clearly are derived from +RNA viruses of this group. They seem to have been mislabeled dsRNA viruses due to the accumulation of dsRNA replication intermediates in infected cells (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>). A parallel loss of the CP genes apparently occurred in deltaflexiviruses which, in RdRp phylogenies, form a sister group to the flexible filamentous gammaflexiviruses (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>), and in umbraviruses that are included in the family <em>Tombusviridae</em>based on the RdRp phylogeny. Notably, unlike most other capsid-less viruses that are vertically inherited, umbraviruses can hijack capsids of co-infecting luteoviruses for aphid transmission (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>).</p><p id="p-44">Within Branch 3, the phylogenetically compact alphavirus supergroup is embedded within the radiation of diverse virus groups including the well-known tombusviruses and nodaviruses, along with several newcomers discovered via metaviromics, such as the “statovirus”, “weivirus”, “yanvirus”, and “zhaovirus” groups (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). Our RdRp analysis revealed remarkable phylogenetic heterogeneity within and among these groups and split “tombus-like viruses” into 5 lineages with distinct evolutionary affinities (groups ‘Uncl. inv.’, and subsets of “tombus-like viruses” and “nodaviruses” in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>). This subdivision is also supported by the analysis of the CPs of these viruses (see section on SJR-CP evolution; <a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7</a>). Therefore, in contrast to the alphavirus supergroup, nodaviruses or flavivirus supergroup, the term ‘tombuslike’ loses its evolutionary and taxonomic coherence. Accordingly, we use the term “tombusviruses” (without quotation marks) only for one lineage that includes the members of the current family <em>Tombusviridae</em> along with a broad variety of related plant and invertebrate holobiont viruses (<a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">14</a>).</p><p id="p-45">The previously suggested, tenuous <em>Flaviviridae-Tombusviridae</em> affinity is gone in the present tree although members of both families belong to the same major Branch 3. Plant tombusviruses (and members of closely related plant virus genera), the only group of “tombus-like viruses” that was available at the time of previous analyses (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>), now form but a small twig deep within the large assemblage we refer to as tombusviruses. Tombusviruses are affiliated with “statoviruses” (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">80</a>) and a subset of unclassified viruses from invertebrate holobionts rather than with flaviviruses (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>). Flaviviruses now form a separate clade within Branch 3, the flavivirus supergroup that includes members of four recognized flaviviral genera <em>(Pegivirus, Hepacivirus, Flavivirus,</em> and <em>Pestivirus),</em> the newly discovered “jingmenviruses” with segmented genomes (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>) and a variety of unclassified, extremely divergent “flavi-like viruses” of animals and plants. This clade is split into two well-supported lineages, one of which includes pegiviruses and hepaciviruses, and the other one consists of the rest of flaviviruses (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>). Flaviviral virions are enveloped, with the envelope proteins forming an external icosahedral shell, whereas the core nucleocapsid is apparently disordered; the evolutionary provenance of the core protein, with its distinct fold, is unclear (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>, <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>). Notably, flaviviral envelope proteins are class II fusion proteins that are closely related to alphavirus envelope E1 proteins (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>). The theme of gene swapping between these distantly related virus groups of Branch 3 is further emphasized by the homology between alphavirus CPs which form icosahedral capsids under the lipid envelopes, and flavivirus non-structural NS3 proteases that share a chymotrypsin-like fold (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">84</a>). Because the RdRp tree topology implies that the alphavirus ancestor is more recent than the ancestor of flaviviruses (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>), such adoption of the NS3 protease for a structural role is suggestive of emerging alphaviruses borrowing their structural module from preexisting flaviviruses (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>).</p><p id="p-46">The hallmark of Branch 3 is the capping enzyme (CapE), which is present in the entire “alphavirus supergroup” and in flaviviruses (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a> and S2B). A highly divergent version of CapE has been identified in nodaviruses (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>) and, in our present analysis, in the additional subset of viruses that grouped with nodaviruses, as well as a few viruses scattered throughout the clade. Formally, CapE is inferred to be ancestral in the entire Branch 3. However, CapEs of “alphavirus supergroup” members, nodaviruses, and flaviviruses are only distantly related to one another, and at least the latter have closer eukaryotic homologs, the FtsJ family methyltransferases (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>, <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>). Furthermore, tombusviruses, statoviruses, yanviruses, zhaoviruses, weiviruses, and members of the <em>Pegivirus-Hepacivirus</em> lineage of flaviviruses lack CapE, putting into question its presence in the ancestor of this branch (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4B</a>, S2B). The most credible evolutionary scenario seems to involve convergent acquisition of CapEs on at least 3 independent occasions, recapitulating the apparent history of helicases in Branch 2 (see above; <a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3B</a> and S2B). The trend of the capture of helicases by +RNA viruses with longer genomes also holds in Branch 3 and includes the acquisition of S1H at the base of the alphavirus supergroup and S2H by the ancestral flavivirus (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4B</a>, S2B).</p><p id="p-47">To an even greater extent than in Branch 2, the apparent routes of virus evolution in Branch 3 involve lineage-specific gene capture resulting in evolution of complex genome architectures (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4B</a>). The most notable cases are closteroviruses and divergent flaviviruses that have genomes of up to 20-26 kb, rivalling coronaviruses in terms of genome length and the complexity of the gene repertoire (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>-<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>).</p><p id="p-48">The lack of genes assigned to the common ancestor of Branch 3 (with the obvious exception of the RdRp) prevents development of a coherent evolutionary scenario for the entire branch. In the case of the clade encompassing the “alphavirus supergroup” and related viruses, a potential common ancestor could be a simple virus that encoded only an RdRp and a SJR-CP, a CP fold most broadly represented in this clade including diverse tombusviruses, nodaviruses and members of <em>Bromoviridae,</em> and <em>Tymoviridae</em> within the “alphavirus supergroup”. Proposing such an ancestor for the flavivirus clade is challenged by the lack of viruses with short and simple genomes among flaviviruses. Indeed, the lengths of the genomes in this clade vary from ~9 kb to ~26 kb, with even the shortest ones encoding at least three of the flavivirus signature genes (serine protease [Spro], S2H, and RdRp). One potential clue, however, is provided by “jingmenviruses” with tetra-partite genomes in which the protease-helicase modules and the RdRps are encoded by separate genome segments; two other segments apparently encode structural proteins of unclear provenance (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">38</a>). This genome architecture could hint at an ancestral flavivirus genome that was assembled from genes borrowed from pre-existing viruses, one of which possessed a divergent “tombus-like virus” RdRp. Although the origins of Branch 3 are murky, major trends in its subsequent evolution clearly included lineage-specific gene capture, starting with helicases and CapEs in the ancestors of the major lineages and followed by diverse genes in smaller groups (<a id="xref-fig-4-12" class="xref-fig" href="#F4">Figure 4B</a>).</p></div><div id="sec-10" class="subsection"><h4>Branch 4: dsRNA viruses</h4><p id="p-49">Branch 4, which joins Branch 3 with weak support, includes the bulk of the dsRNA viruses (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a> and 5). All dsRNA viruses in this branch share a unique virion organization and encode homologous CPs. In particular, the specialized icosahedral capsids of these viruses, involved in transcription and replication of the dsRNA genome inside cells, are constructed from 60 homo-or heterodimers of CP subunits organized on an unusual <em>T=1</em> (also known as pseudo-7=2) lattice (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a>, <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>). The only exception are the chrysoviruses which encode large CPs corresponding to the CP dimers of other dsRNA viruses and form genuine T=1 capsids (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>). Icosahedral capsids of partitiviruses and picobirnaviruses, which encode RdRps belonging to Branch 2, are also constructed from 60 homodimers (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>) and have been suggested to be evolutionarily related to those of the dsRNA viruses from RdRp Branch 4 (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>) despite little structural similarity between the corresponding CPs. Totiviruses, many of which have “minimal” genomes encoding only RdRps and CPs, comprise one of the two major clades in Branch 4, whereas cystoviruses, the only known prokaryotic dsRNA viruses, together with the vast family <em>Reoviridae,</em> which consists of multi-segmented dsRNA viruses infecting diverse eukaryotes, comprise the second clade (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The closer phylogenetic affinity between cystoviruses and reoviruses appears to be corroborated by the fact that the inner T=1 icosahedral capsid is uniquely encased by the outer icosahedral shell constructed on a 7=13 lattice in both families (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). Both cystoviruses and reoviruses appear to have gained many clade-specific genes, in particular, RecA-like packaging ATPases of the former (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a>) and the CapEs of the latter that are only distantly related to CapEs of other RNA viruses and likely were acquired independently (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a>, <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, S2B).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Branch 4 of the RNA virus RNA-dependent RNA polymerases (RdRps): dsRNA viruses of eukaryotes and prokaryotes. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. Green dots represent well-supported branches (&gt;0.7). (B) Genome maps of a representative set of Branch 4 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; iCP, internal capsid protein; NS, non-structural protein; NTPase, nucleotide triphosphatase; oCP, outer capsid protein; P, protein; phytoreoS7, homolog of S7 domain of phytoreoviruses; pHel, packaging helicase; vOTU, virus OTU-like protease; VP, viral protein; The CPs of totiviruses and chrysoviruses are homologous to iCPs of reoviruses and cystoviruses (black rectangles)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Branch 4 of the RNA virus RNA-dependent RNA polymerases (RdRps): dsRNA viruses of eukaryotes and prokaryotes. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. Green dots represent well-supported branches (&gt;0.7). (B) Genome maps of a representative set of Branch 4 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; iCP, internal capsid protein; NS, non-structural protein; NTPase, nucleotide triphosphatase; oCP, outer capsid protein; P, protein; phytoreoS7, homolog of S7 domain of phytoreoviruses; pHel, packaging helicase; vOTU, virus OTU-like protease; VP, viral protein; The CPs of totiviruses and chrysoviruses are homologous to iCPs of reoviruses and cystoviruses (black rectangles).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F5.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.5" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F5.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.5</span> <p id="p-50" class="first-child">Branch 4 of the RNA virus RNA-dependent RNA polymerases (RdRps): dsRNA viruses of eukaryotes and prokaryotes. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); myco., mycoviruses; uncl., unclassified. Green dots represent well-supported branches (&gt;0.7). (B) Genome maps of a representative set of Branch 4 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; iCP, internal capsid protein; NS, non-structural protein; NTPase, nucleotide triphosphatase; oCP, outer capsid protein; P, protein; phytoreoS7, homolog of S7 domain of phytoreoviruses; pHel, packaging helicase; vOTU, virus OTU-like protease; VP, viral protein; The CPs of totiviruses and chrysoviruses are homologous to iCPs of reoviruses and cystoviruses (black rectangles).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h4>Branch 5: -RNA viruses</h4><p id="p-51">Branch 5, the 100% supported lineage combining all-RNA viruses, is lodged within Branch 4 as the sister group of reoviruses, and this position is upheld by two strongly supported internal branches in the RdRp tree (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a> and 6). The-RNA branch splits into 2 strongly supported clades. The first clade encompasses the 348 viruses-strong membership of the order <em>Mononegavirales</em> (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>), along with the members of the distantly related family <em>Aspiviridae</em> (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>), 3 groups of-RNA viruses discovered through metaviromics (“chuviruses”, “qinviruses”, “yueviruses”) (<a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), and a group of unclassified fungal viruses (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>) (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>). In contrast to the members of the <em>Mononegavirales,</em> most of which possess unsegmented genomes, the remainder of this clade is characterized by bi-, tri-, or even tetrasegmented genomes (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). The second clade combines the family <em>Orthomyxoviridae,</em>the genus <em>Tilapinevirus</em> (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>) and the large order <em>Bunyavirales</em> (394 viruses) (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a>). The latter order consists of two branches, one of which is the sister group to the orthomyxovirus/tilapinevirus clade (albeit with weak support). The number of negative-sense or ambisense genome segments in this clade varies from 2-3 in most of bunyaviruses to 8 in viruses of the genus <em>Emaravirus</em> and the orthomyxovirus/tilapinevirus group (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">99</a>, <a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">101</a>, <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a>). A notable acquisition in the first clade is a CapE, whereas members of the second clade share “cap-snatching” endonucleases (En) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Branch 5 of the RNA virus RNA-dependent RNA polymerases (RdRps):-RNA viruses. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); uncl., unclassified. (B) Genome maps of a representative set of Branch 5 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; EN, &#x201C;cap-snatching&#x201D; endonuclease; GP, glycoprotein; GPC; glycoprotein precursor; HA; hemagglutinin; M, matrix protein; MP, movement protein; NA; neuraminidase; NP, nucleoprotein; NS, nonstructural protein; NSM, medium nonstructural protein; NSs, small nonstructural protein; PA, polymerase acidic protein; PB, polymerase basic protein; vOTU; virus OTU-like protease; VP; viral protein; Z, zinc finger protein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Branch 5 of the RNA virus RNA-dependent RNA polymerases (RdRps):-RNA viruses. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); uncl., unclassified. (B) Genome maps of a representative set of Branch 5 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; EN, &#x201C;cap-snatching&#x201D; endonuclease; GP, glycoprotein; GPC; glycoprotein precursor; HA; hemagglutinin; M, matrix protein; MP, movement protein; NA; neuraminidase; NP, nucleoprotein; NS, nonstructural protein; NS&lt;sub&gt;M&lt;/sub&gt;, medium nonstructural protein; NSs, small nonstructural protein; PA, polymerase acidic protein; PB, polymerase basic protein; vOTU; virus OTU-like protease; VP; viral protein; Z, zinc finger protein.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F6.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.6" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F6.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.6</span> <p id="p-52" class="first-child">Branch 5 of the RNA virus RNA-dependent RNA polymerases (RdRps):-RNA viruses. (A) Phylogenetic tree of the virus RdRps showing ICTV-accepted virus taxa and other major groups of viruses. Approximate numbers of distinct virus RdRps present in each branch are shown in parentheses. Symbols to the right of the parentheses summarize the presumed virus host spectrum of a lineage. Green dots represent well-supported branches (&gt;0.7), whereas yellow dots correspond to weakly supported branches. Inv., viruses of invertebrates (many found in holobionts making host assignment uncertain); uncl., unclassified. (B) Genome maps of a representative set of Branch 5 viruses (drawn to scale) showing major, color-coded conserved domains. When a conserved domain comprises only a part of the larger protein, the rest of this protein is shown in light gray. The locations of such domains are approximated (indicated by fuzzy boundaries). CapE, capping enzyme; CP, capsid protein; EN, “cap-snatching” endonuclease; GP, glycoprotein; GPC; glycoprotein precursor; HA; hemagglutinin; M, matrix protein; MP, movement protein; NA; neuraminidase; NP, nucleoprotein; NS, nonstructural protein; NS<sub>M</sub>, medium nonstructural protein; NSs, small nonstructural protein; PA, polymerase acidic protein; PB, polymerase basic protein; vOTU; virus OTU-like protease; VP; viral protein; Z, zinc finger protein.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-12"><h2 class="">Patterns of the single jelly-roll capsid protein evolution</h2><p id="p-53">The SJR-CP is the dominant type of CP among +RNA viruses and is also found in members of one family of dsRNA viruses (<em>Birnaviridae</em>). Structural comparisons indicate that SJR-CPs of RNA viruses form a monophyletic group and likely have been recruited from cellular SJR proteins on a single occasion during the evolution of RNA viruses (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">19</a>). The short length and high divergence of SJR-CPs preclude adequate resolution in phylogenetic analysis; thus, we performed profile-profile sequence comparisons and clustering of all viral SJR-CP sequences in our dataset (see Methods for details). Analysis of the resulting network revealed patterns that are generally congruent with the RdRp phylogeny and provide further insights into the evolution of different branches of RNA viruses.</p><p id="p-54">At conservative P value thresholds (P&lt;1e<sup>−10</sup>), the majority of SJR-CPs segregated into two large clusters both of which contained representatives from RdRp Branch 2. Cluster 1 included the members of <em>Picornavirales, Caliciviridae,</em> and diverse “picorna-like viruses” of invertebrates, whereas cluster 2 consisted of the members of the families <em>Astroviridae, Luteoviridae</em>, and <em>Solemoviridae</em> and “solemo-like viruses” (Figure S3). In addition, cluster 2 contained members of several families from RdRp Branch <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, namely, <em>Tombusviridae</em> (and diverse “tombus-like viruses”), <em>Hepeviridae,</em> a subgroup of <em>Nodaviridae</em> and “statoviruses”.</p><p id="p-55">At less restrictive P value thresholds (P&lt;1e<sup>−03</sup>), all SJR-CPs were interconnected, largely, through making contacts to the core of cluster 2. Only “ourmiaviruses” had stronger affinity to picornaviruses in cluster 1 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). This pattern of connectivity is consistent with the radiation of SJR-CPs from a common ancestor, likely resembling sequences from cluster 2 of Branch 2. This analysis also revealed high CP sequence divergence among members of some families (e.g., <em>Bromoviridae</em>) and numerous cases of apparent CP gene replacement. For instance, the CPs of nodaviruses fall into two groups: one is related to the turreted CPs of tetraviruses and the other is similar to CPs of tombusviruses, mirroring the RdRp phylogeny (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>). At a greater phylogenetic distance, CPs of astroviruses and hepeviruses are closely related despite them being affiliated to Branches 2 and 3, respectively, suggesting CP gene replacement in the ancestor of one of the two families. Given that the CPs of hepeviruses connect to SJR-CPs of other viruses through astroviruses, CP gene replacement most likely occurred in the ancestor of hepeviruses (Figure S3). Notably, the CPs of “zhaoviruses”, “weiviruses”, and “tombus-like” and “solemo-like viruses” (diverse virus assemblage within solemovirus branch, to the exclusion of bona fide <em>Solemoviridae)</em> did not form discrete clusters but rather were affiliated with diverse virus groups, suggesting extensive recombination in these viruses, with multiple CP gene exchanges (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>). In the case of unclassified “narnaviruses” and “ourmiaviruses”, the CP genes apparently have been acquired on more than 3 independent occasions from different groups of viruses, emphasizing the impact of recombination and gene shuffling in the evolution of RNA viruses. Previously, a similar extent of chimerism has been also observed among ssDNA viruses (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a>, <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>), highlighting the evolutionary and functional plasticity of short viral genomes.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence similarity networks of SJR-CPs. Protein sequences were clustered by the pairwise similarity of their hmm profiles using CLANS. Different groups of SJR-CPs are shown as clouds of differentially colored circles, with the corresponding subgroups labeled as indicated in the figure. Edges connect sequences with CLANS P-value of &lt; 1e-03." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence similarity networks of SJR-CPs. Protein sequences were clustered by the pairwise similarity of their hmm profiles using CLANS. Different groups of SJR-CPs are shown as clouds of differentially colored circles, with the corresponding subgroups labeled as indicated in the figure. Edges connect sequences with CLANS P-value of &lt; 1e-03.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F7.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.7" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F7.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.7</span> <p id="p-56" class="first-child">Sequence similarity networks of SJR-CPs. Protein sequences were clustered by the pairwise similarity of their hmm profiles using CLANS. Different groups of SJR-CPs are shown as clouds of differentially colored circles, with the corresponding subgroups labeled as indicated in the figure. Edges connect sequences with CLANS P-value of &lt; 1e-03.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-13"><h2 class="">The modular gene-sharing network of RNA viruses: gene transfer and module shuffling</h2><p id="p-57">The pronounced structural and functional modularity of virus proteomes and pervasive shuffling of the genomic regions encoding distinct protein modules are key features of virus evolution (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-15-10" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>). Therefore, a productive approach to the study of the virosphere that complements phylogenetics is the construction and analysis of networks of gene sharing. Bipartite networks, in which one type of nodes corresponds to genes and the other one to genomes, have been employed to investigate the dsDNA domain of the virosphere (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>). This analysis revealed hierarchical modular organization of the network, with several modules including non-obvious connections between disparate groups of viruses (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a>). Although, for RNA viruses, this type of analysis is less informative due to the small number of proteins encoded in each viral genome, the “pan-proteome” of RNA viruses is large (Data set S5), prompting us to experiment with bipartite gene sharing networks for RNA viruses. The initial search for statistically significant modularity identified 54 distinct modules, most of which included a single virus family (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A, B</a>). Remarkably, the family <em>Reoviridae</em> has been split into 5 modules, highlighting the vast diversity of this family, comparable to that in order-level taxa. Among the exceptions, the most expansive module included the viruses of the order <em>Picornavirales</em> (module 29), together with the family <em>Caliciviridae</em> (module 47), that are linked through the conserved suit of genes including SJR-CP, chymotrypsin-like protease, S3H and the RdRp (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a> and <a id="xref-fig-8-3" class="xref-fig" href="#F8">C</a>). Viruses of the order <em>Bunyavirales</em> were also recovered in a single module that is characterized by the presence of a conserved nucleocapsid (with the exception of the families <em>Nairoviridae</em> and <em>Arenaviridae)</em> and the cap-snatching endonuclease (module 51; <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8A</a> and <a id="xref-fig-8-5" class="xref-fig" href="#F8">C</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bipartite network of gene sharing in RNA viruses. (A) Groups of related viruses were identified as the modules of the bipartite genome-gene network (not shown), whereas connector genes were defined as those genes present in two or more modules with prevalence greater than 65%. The network in A shows viral modules as colored circles (blue, +RNA viruses; green, dsRNA viruses; red,-RNA viruses), linked to the connector genes (black dots) that are present in each module. The size of the circles is proportional to the number of genomes in each module. Shaded ovals indicate statistically significant, 1st-order supermodules that join modules from taxonomically related groups. (B) Taxonomic analysis of the network modules confirms that most modules contain viruses from a single family and that families do not tend to split among modules. (C) High-order supermodules of the RNA virus network, obtained by iteratively applying a community detection algorithm on the bipartite network of (super)modules and connector genes. GP, glycoprotein; GT, guanylyltransferase; MT, methyltransferase; NCP, nucleocapsid; RdRp, RNA-dependent RNA polymerase; SF, superfamily; SJR-CP, single jelly roll capsid protein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bipartite network of gene sharing in RNA viruses. (A) Groups of related viruses were identified as the modules of the bipartite genome-gene network (not shown), whereas connector genes were defined as those genes present in two or more modules with prevalence greater than 65%. The network in A shows viral modules as colored circles (blue, +RNA viruses; green, dsRNA viruses; red,-RNA viruses), linked to the connector genes (black dots) that are present in each module. The size of the circles is proportional to the number of genomes in each module. Shaded ovals indicate statistically significant, 1st-order supermodules that join modules from taxonomically related groups. (B) Taxonomic analysis of the network modules confirms that most modules contain viruses from a single family and that families do not tend to split among modules. (C) High-order supermodules of the RNA virus network, obtained by iteratively applying a community detection algorithm on the bipartite network of (super)modules and connector genes. GP, glycoprotein; GT, guanylyltransferase; MT, methyltransferase; NCP, nucleocapsid; RdRp, RNA-dependent RNA polymerase; SF, superfamily; SJR-CP, single jelly roll capsid protein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F8.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.8" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F8.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.8</span> <p id="p-58" class="first-child">Bipartite network of gene sharing in RNA viruses. (A) Groups of related viruses were identified as the modules of the bipartite genome-gene network (not shown), whereas connector genes were defined as those genes present in two or more modules with prevalence greater than 65%. The network in A shows viral modules as colored circles (blue, +RNA viruses; green, dsRNA viruses; red,-RNA viruses), linked to the connector genes (black dots) that are present in each module. The size of the circles is proportional to the number of genomes in each module. Shaded ovals indicate statistically significant, 1st-order supermodules that join modules from taxonomically related groups. (B) Taxonomic analysis of the network modules confirms that most modules contain viruses from a single family and that families do not tend to split among modules. (C) High-order supermodules of the RNA virus network, obtained by iteratively applying a community detection algorithm on the bipartite network of (super)modules and connector genes. GP, glycoprotein; GT, guanylyltransferase; MT, methyltransferase; NCP, nucleocapsid; RdRp, RNA-dependent RNA polymerase; SF, superfamily; SJR-CP, single jelly roll capsid protein.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">The next stage of the network analysis aims at detecting supermodules that are formed from the primary modules via connecting genes. The supermodules of RNA viruses failed to attain statistical significance due to the small number of shared genes but nevertheless, some notable connections are revealed by this analysis. Specifically, 8 overlapping supermodules were identified (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8C</a>). The largest and, arguably, most remarkable is a supermodule that combines +RNA, dsRNA and-RNA viruses that share the capping enzymes, S1H (with the exception of <em>Reoviridae</em> and <em>Mononegavirales)</em> and additional connector genes (e.g., the OTU family protease) that link some of the constituent modules. The second largest supermodule combines large subsets of viruses from the RdRp Branches 2 and 3 that are connected through the SJR-CP and the chymotrypsin-like protease. Another supermodule encompasses enveloped +RNA viruses of the families <em>Flaviviridae</em> and <em>Togaviridae,</em> and-RNA viruses of the order <em>Bunyavirales</em> (except for <em>Arenaviridae)</em> that share homologous envelope glycoproteins (except for flaviviruses) and class II fusion proteins.</p><p id="p-60">Information from gene sharing is inherently limited for RNA viruses due to the small number of genes in each genome. Nevertheless, the bipartite network analysis reveals prominent “horizontal” connections that are underlain either by actual gene exchange or by parallel acquisition of homologous genes by distinct RNA viruses.</p></div><div class="section" id="sec-14"><h2 class="">Host ranges of RNA viruses: evolutionary implications and horizontal virus transfer</h2><p id="p-61">RNA viruses have been identified in representatives of all major divisions of eukaryotes, whereas in prokaryotes, members of two families of RNA viruses are known to infect only a limited range of hosts (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-15-11" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">31</a>). For Branch 1 in our phylogenetic tree of RdRps, the route of evolution from leviviruses infecting prokaryotes to eukaryotic “ourmiaviruses” of plants and invertebrates is readily traceable and involves a merger between a levivirus-derived naked RNA replicon that eukaryotes most likely inherited from the mitochondrial endosymbiont with the SJR-CP of a eukaryotic “picorna-like virus”. Notably, such a merger seems to have occurred on at least three other independent occasions in Branch 1 because several groups of invertebrate holobiont “narnaviruses” and some “ourmiaviruses” encode distantly related SJR-CPs that apparently were acquired from different groups of plant and animal viruses (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7</a>).</p><p id="p-62">The case of cystoviruses is less clear given that this clade is sandwiched between eukaryotic viruses in Branch 4 and therefore does not seem to be a good candidate for the ancestor of this branch. It appears more likely that the ancestor was a toti-like virus, whereas cystoviruses are derived forms, which implies virus transfer from eukaryotes to prokaryotes. However, an alternative scenario might be considered. No known prokaryotic viruses are classified in Branch 2 but it has been proposed that picobirnaviruses, for which no hosts have been reliably identified, actually are prokaryotic viruses. This proposal is based on the conspicuous conservation of functional, bacterial-type, ribosome-binding sites (Shine-Dalgarno sequences) in picobirnavirus genomes (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a>, <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a>). Should that be the case, viruses of prokaryotes might be lurking among totiviruses as well. Then, Branch 4 would stem from a prokaryotic ancestor avoiding the need to invoke virus transfer from eukaryotes to prokaryotes to explain the origin of the cystoviruses.</p><p id="p-63">We made an attempt to quantify the potential horizontal virus transfer (HVT) events in RNA viruses that represent the 5 major branches of the RdRp tree. The leaves of the tree were labeled with the known hosts, the entropy of the host ranges for each subtree was calculated, and the resulting values were plotted against the distance from the root (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). By design, for all branches, entropy (host diversity) drops from the maximum values at the root to zero at the leaves. All branches show substantial host range diversity such that, for example, at half-distance from root to leaves, all branches, except for Branch 1, retain at least half of the diversity (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>). Furthermore, differences between the branches are substantial, with the highest entropy observed in branches 4 (dsRNA viruses) and 5 (-RNA viruses). With all the caveats due to potential errors and ambiguities in host assignment, this analysis strongly suggests that HVT played an important role in the evolution of all major groups of RNA viruses.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative analysis of the host range diversity of RNA viruses. The entropy of host ranges is plotted against the ultrameterized tree depth for the 5 main branches of the RdRp phylogeny (see Figure 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Quantitative analysis of the host range diversity of RNA viruses. The entropy of host ranges is plotted against the ultrameterized tree depth for the 5 main branches of the RdRp phylogeny (see Figure 1).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F9.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.9" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F9.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.9</span> <p id="p-64" class="first-child">Quantitative analysis of the host range diversity of RNA viruses. The entropy of host ranges is plotted against the ultrameterized tree depth for the 5 main branches of the RdRp phylogeny (see <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-15"><h2 class="">DISCUSSION</h2><div id="sec-16" class="subsection"><h3>RNA virus evolution coming into focus</h3><p id="p-65">This work was prompted by the advances of metaviromics, which have dramatically increased the known diversity of RNA viruses (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>). We reasoned that this expansion of the RNA virosphere could provide for an improved understanding of virus evolution. Although further progress of metaviromics and enhanced phylogenomic methods will undoubtedly change current ideas, we believe that some key aspects of RNA virus evolution are indeed coming into focus.</p><p id="p-66">The expanded diversity of RNA viruses combined with the iterative procedure for phylogenetic analysis allowed us to obtain a tree of all RdRps and the most closely related RTs in which the main branches are strongly supported and thus appear to be reliable (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a>). To our knowledge, the picture of RNA virus evolution emerging from the tree has not been presented previously. The tree seems to clarify the relationships between the 3 Baltimore Classes of RNA viruses by revealing the nested tree structure in which dsRNA viruses evolved, on at least two occasions, from +RNA viruses, whereas -RNA viruses evolved from a distinct group of dsRNA viruses.</p><p id="p-67">The derivation of-RNA viruses from dsRNA viruses is, arguably, the most unexpected outcome of the present analysis, considering the lack of genes (other than the RdRp) shared by these virus classes. Clearly, given that the primary evidence behind the derivation of-RNA viruses from within dsRNA viruses comes from deep phylogeny, extreme caution is due in the interpretation of this observation. However, the pronounced similarity between the 3D structures of the RdRps of the-RNA influenza virus A and bacteriophage ^6 dsRNA cystovirus (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>) is compatible with our findings. Further, because virtually no-RNA viruses are known in prokaryotes or unicellular eukaryotes [with the single exception of “leischbuviruses” in parasitic trypanosomatids (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>) that were likely acquired from the animal hosts of these protists], their later origin from a preexisting group of +RNA or dsRNA viruses appears most likely.</p><p id="p-68">The +RNA to dsRNA to-RNA scenario of RNA virus genome evolution also makes sense in terms of the molecular logic of genome replication-expression strategies. Indeed, +RNA viruses use the simplest genomic strategy and, in all likelihood, represent the primary pool of RNA viruses. The dsRNA viruses, conceivably, evolved when a +RNA virus switched to encapsidating a replicative intermediate (dsRNA) together with the RdRp. Naked replicons similar to “mitoviruses”, hypoviruses, and endornaviruses might have been evolutionary intermediates in this process. This switch does not seem to be as “easy” and common as previously suspected (<a id="xref-ref-15-12" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>) but, nevertheless, appears to have occurred at least twice during the evolution of RNA viruses. The origin of-RNA viruses is the next step during which the plus-strand is discarded from the virions, perhaps simplifying the processes of transcription and replication. Conceivably, the evolution of dsRNA and-RNA viruses, in which transcription and replication of the viral genomes are confined to the interior of virions or nucleocapsid transcription/replication complexes and no dsRNA accumulates in the infected cells, was driven by the advantage of escaping some of the host defense mechanisms, in particular, RNA interference (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a>, <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a>). The membrane-associated replication complexes of +RNA viruses could represent an initial step in this direction (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>).</p><p id="p-69">Obviously, evolution of the RdRp does not equal evolution of viruses: other genes, in particular those encoding capsid and other structural proteins, are crucial for virus reproduction, and these genes often have different histories. The reconstruction of gene gain and loss sheds some light on these aspects of RNA virus evolution. The ancestors of each of the major branches of RNA viruses except for Branch 1 appear to have been simply organized +RNA viruses resembling tombusviruses (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a>). Thus, these types of viruses encoding RdRps and SJR-CPs might have been ancestral to the bulk of eukaryotic RNA viruses (apart from those in Branch 1 that derive directly from prokaryotic leviviruses). The subsequent parallel capture of different helicases enabled evolution of increasingly complex genomes via accumulation of additional genes (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>). Notably, similar levels of complexity, with the complete coding capacity of 20 to 40 kb, were reached independently in 4 branches of the RdRp tree, namely, Branch 2 (coronaviruses), Branch 3 (closteroviruses and flaviviruses), Branch 4 (reoviruses), and Branch 5 (filoviruses and paramyxoviruses) (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>-6).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A general scenario of RNA virus evolution. The figure is a rough scheme of the key steps of RNA virus evolution inferred in this work. The main branches from the phylogenetic tree of the RdRps are denoted 1 to 5 as in Figure 1. Only the RdRp, CP genes and helicase (S1H, S2H and S3H for the helicases of superfamilies 1, 2 and 3, respectively) are shown systematically. The helicases appear to have been captured independently and in parallel in 3 branches of +RNA viruses, facilitating the evolution of larger, more complex genomes. Additional genes, namely, Endo and Exo (for endonuclease and exonuclease, respectively) and Hsp70h (heat shock protein 70 homolog), are shown selectively, to emphasize the increased genome complexity, respectively, in Nidovirales and in Closteroviridae. The virion architecture is shown schematically for each included group of viruses. Icosahedral capsids composed of unelated CPs are shown by different colors (see text for details). The question mark at the hypothetical ancestral eukaryotic RNA virus indicates the uncertainty with regard to the nature of the host (prokaryotic or eukaryotic) of this ancestral form. The block arrow at the bottom shows the time flow and the complexification trend in RNA virus evolution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1169244830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A general scenario of RNA virus evolution. The figure is a rough scheme of the key steps of RNA virus evolution inferred in this work. The main branches from the phylogenetic tree of the RdRps are denoted 1 to 5 as in Figure 1. Only the RdRp, CP genes and helicase (S1H, S2H and S3H for the helicases of superfamilies 1, 2 and 3, respectively) are shown systematically. The helicases appear to have been captured independently and in parallel in 3 branches of +RNA viruses, facilitating the evolution of larger, more complex genomes. Additional genes, namely, Endo and Exo (for endonuclease and exonuclease, respectively) and Hsp70h (heat shock protein 70 homolog), are shown selectively, to emphasize the increased genome complexity, respectively, in &lt;em&gt;Nidovirales&lt;/em&gt; and in &lt;em&gt;Closteroviridae.&lt;/em&gt; The virion architecture is shown schematically for each included group of viruses. Icosahedral capsids composed of unelated CPs are shown by different colors (see text for details). The question mark at the hypothetical ancestral eukaryotic RNA virus indicates the uncertainty with regard to the nature of the host (prokaryotic or eukaryotic) of this ancestral form. The block arrow at the bottom shows the time flow and the complexification trend in RNA virus evolution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure.10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F10.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure.10" src="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F10.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure.10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/24/451740/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure.10</span> <p id="p-70" class="first-child">A general scenario of RNA virus evolution. The figure is a rough scheme of the key steps of RNA virus evolution inferred in this work. The main branches from the phylogenetic tree of the RdRps are denoted 1 to 5 as in <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1</a>. Only the RdRp, CP genes and helicase (S1H, S2H and S3H for the helicases of superfamilies 1, 2 and 3, respectively) are shown systematically. The helicases appear to have been captured independently and in parallel in 3 branches of +RNA viruses, facilitating the evolution of larger, more complex genomes. Additional genes, namely, Endo and Exo (for endonuclease and exonuclease, respectively) and Hsp70h (heat shock protein 70 homolog), are shown selectively, to emphasize the increased genome complexity, respectively, in <em>Nidovirales</em> and in <em>Closteroviridae.</em> The virion architecture is shown schematically for each included group of viruses. Icosahedral capsids composed of unelated CPs are shown by different colors (see text for details). The question mark at the hypothetical ancestral eukaryotic RNA virus indicates the uncertainty with regard to the nature of the host (prokaryotic or eukaryotic) of this ancestral form. The block arrow at the bottom shows the time flow and the complexification trend in RNA virus evolution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-71">Gene exchange and shuffling of gene modules are important factors of RNA virus evolution. For example, it appears that all dsRNA viruses in the two major clades within Branches 2 and 4 share homologous structural modules that combine with distinct RdRps. At least in the case of partiti-picobirnaviruses in Branch 2, the dsRNA virus (“toti-like virus”) CP apparently displaced the ancestral SJR-CP. However, this particular protein structure does not seem to be essential to encapsidate a dsRNA genome: birnaviruses, whose provenance is uncertain due to the permutation in their RdRps, retain SJR-CP, which is most closely related to SJR-CP of nodaviruses and tetraviruses (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">19</a>). An even more striking example of module shuffling is presented by amalgaviruses, dsRNA viruses that group with partitiviruses in the RdRp tree but encode a distant homolog of the nucleocapsid protein of-RNA bunyaviruses (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a>-<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">114</a>). More generally, structural and replication modules have been repeatedly shuffled during the evolution of +RNA viruses. Examples include displacement of the ancestral SJR-CP by a filamentous CP in potyviruses and by a helical nucleocapsid protein in nidoviruses, and multiple cases of displacement with rod-shaped-like CP and unique nucleocapsid proteins in Branch 3. Thus, exchange of genes and gene modules among RNA viruses is pervasive and can cross the boundaries of Baltimore Classes.</p><p id="p-72">Another recurring trend in RNA virus evolution is the loss of the structural module resulting in the emergence of naked RNA replicons such as “narnaviruses” and “mitoviruses” in Branch 1, hypoviruses in Branch 2, and endornaviruses, umbraviruses and deltaflexiviruses in Branch 3 (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>). On some occasions, broad horizontal spread of a gene leads to a major shift in the lifestyle of viruses, such as adaptation of viruses to a new type of hosts. The primary cases in point are the movement proteins (MPs) of plant viruses that are represented in all 5 branches of the RdRp tree and, outside of the RNA part of the virosphere, in plant caulimoviruses and badnaviruses, and ssDNA viruses (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">115</a>).</p></div><div id="sec-17" class="subsection"><h3>The prevalence of HVT and the overall course of RNA virus evolution</h3><p id="p-73">Arguably, the most striking realization brought about by metaviromics is the diverse host range of numerous groups of viruses, even tight ones that occupy positions near the tips of the RdRp tree. Extending early observations on then highly unexpected similarities between viruses of animals and plants, recent metaviromic analyses reveal numerous clusters of indisputably related viruses infecting animals and plants, plants and fungi, and in some cases, animals or plants and protists. The evolutionary relationships between viruses with distinct host ranges are supported not only by the phylogeny of the RdRp, but also by the fact that these viruses share additional conserved domains, such as, for example, SJR1, S3H and 3C-Pro in the case of <em>Picornavirales</em>members infecting protists, plants, fungi, invertebrates and vertebrates (<a id="xref-fig-3-14" class="xref-fig" href="#F3">Fig. 3</a>).</p><p id="p-74">Invertebrates are particularly promiscuous hosts for viruses, often sharing the same virus group with distantly related organisms. Certainly, much caution is due in the interpretation of host range assignments from metaviromics, especially, holobiont studies. Viruses identified in holobiont samples of, say, invertebrates could actually infect protists associated with these animals (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">50</a>) or could represent contamination from fungal, plant or even prokaryotic sources. These uncertainties notwithstanding, the extensive diversity of hosts even within small branches of the RdRp tree is undeniable. A coevolution scenario in which the ancestors of all these viruses originated from the common ancestor of the respective groups of eukaryotes and coevolved with the hosts implies an enormous diversity of RNA viruses in early eukaryotes. This scenario appears to be highly unlikely given the apparent paucity of RNA viruses in the extant protists (although new metaviromic studies might substantially expand the range of protist viruses). The pervasive HVT alternative seems much more plausible, especially given that arthropods, nematodes, and other invertebrates are well known as virus vectors and thus fit the role of RNA virus reservoirs (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-75">In addition to invertebrates that appear to be dominant HVT agents, fungi could also play an important role in HVT within the global RNA virome. Indeed, fungi that are tightly associated with plants and insects often share closely related viruses with these organisms (<a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">116</a>-<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">118</a>). Furthermore, an indisputable case of cross-kingdom transfer of an insect iflavirus to an entomopathogenic fungus has been recently described (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">119</a>).</p><p id="p-76">These findings appear to be best compatible with a grand evolutionary scenario (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10</a>) in which the ancestor of the eukaryotic RNA virome was a levi/narnavirus-like naked RNA replicon that originally reproduced in mitochondria and combined with a host carbohydratebinding SJR protein (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">19</a>) or a preexisting SJR-CP from a DNA virus to form a simple ancestral virus. Given that viruses of Branches 2, 3, and 4 are present in modern protists, it appears likely that these branches emerged in early eukaryotes. However, because of the apparent dominance of the viruses of the RdRp Branch 2 (“picornavirus supergroup” in general and “aquatic picorna-like viruses” clade in particular) in protists, this branch likely diversified first, whereas the diversification of Branches 3 and 4 occurred later, after ancestral protist viruses were transferred to marine invertebrates during the Cambrian explosion. The recent analysis of the viromes of ctenophores, sponges and cnidarians suggests that substantial diversification of RNA viruses occurred already in these deeply branching metazoa (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">120</a>). Invertebrates brought their already highly diverse RNA virome to land at terrestrialization and subsequently inoculated land plants. In land plants, RNA viruses, particularly those of Branch 3, dramatically expanded, in part, perhaps, because of the exclusion of competing large DNA viruses. Finally, it seems plausible that, given the high prevalence of-RNA viruses in metazoa and their virtual absence in protists [with the exception of the recently discovered “leishbuviruses” that likely invaded their parasitic protist hosts via HVT from an animal host (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">50</a>)], these viruses that comprise the RdRp Branch 5 evolved in animals via mixing and matching genes from reovirus-like and flavivirus-like ancestors.</p></div><div id="sec-18" class="subsection"><h3>The impending overhaul of RNA virus taxonomy</h3><p id="p-77">The expansion of the global RNA virome thanks to the advances of metaviromics, combined with the phylogenomics results, seem to call for an overhaul of the current virus taxonomy on multiple levels. Most importantly, creation of a coherent, hierarchical system with multiple taxonomic ranks seems to be imminent. This process has already started with the proposal of a phylum rank for-RNA viruses, for which monophyly is unequivocally supported by the present analysis (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">6</a>). This phylum could consist of two subphyla with multiple classes and orders. At least 4 additional phyla of RNA viruses can be confidently predicted to emerge, including, respectively, the dsRNA viruses of Branch 4, and +RNA viruses of Branches 1, 2, and 3. Each of these phyla will undoubtedly have a rich internal structure. In addition, some of the current families do not seem to be compatible with the expansive RdRp trees present here and in a previous analysis (<a id="xref-ref-14-12" class="xref-bibr" href="#ref-14">14</a>). For instance, the families <em>Coronaviridae, Togaviridae</em> and <em>Rhabdoviridae</em> are likely to be split into two families each.</p><p id="p-78">While the present study was in preparation, a major attempt on a comprehensive virus classification has been published (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">121</a>). This work analyzed a dendrogram that was produced from distance matrices between viruses derived from sequence similarity scores combined with measures of gene composition similarity. Unlike our present analysis, this approach pre-supposes monophyly of each of the Baltimore classes. Furthermore, given that their analysis is based on measures of similarity rather than on phylogenetic analysis proper, this approach is best regarded as producing a phenetic classification of viruses rather that an evolutionary reconstruction as such. Some of the groups delineated by this method, particularly, among +RNA viruses, are closely similar to those reported here. Others, however, are widely different: for instance, the order <em>Mononegavirales</em> does not come across as monophyletic in their dendrograms. We did not attempt a complete comparison; such an exercise could be useful in the future, for better understanding the routes of RNA virus evolution.</p></div></div><div class="section" id="sec-19"><h2 class="">CONCLUDING REMARKS</h2><p id="p-79">Through metaviromics, many aspects of the global RNA virome evolution can be clarified. Certainly, reconstruction of the deepest events in this evolutionary history is bound to remain tentative, especially, because the RdRp is the only universal gene of the RNA viruses, and hence, the only one that can serve as the template for evolutionary reconstructions. At the depth of divergence characteristic of RdRps, the relationship between the major branches in the tree cannot be established with confidence. Nevertheless, monophyly of several expansive groups, in particular the 5 main branches in the RdRp tree, is strongly supported. Because of the stability of these branches, biologically plausible scenarios of evolution emerge under which dsRNA viruses evolved from different groups of +RNA viruses, whereas -RNA viruses evolved from dsRNA viruses.</p><p id="p-80">Evolutionary reconstructions suggest that the last common ancestors of each major lineage of eukaryotic +RNA viruses were simple viruses that encoded only the RdRp and a CP, most likely, of the SJR fold. Subsequent evolution involved independent capture of distinct helicases which apparently facilitate replication of larger, more complex +RNA genomes. The helicase-assisted replication of +RNA genomes created the opportunities for parallel acquisition of additional genes encoding proteins involved in polyprotein processing and virus genome expression, such as proteases and capping enzymes, respectively, and proteins involved in virus-host interactions, such as MPs or RNAi suppressors of plant viruses. In addition to these processes of vertical evolution of RNA viruses, phylogenomic analysis reveals multiple cases of gene module exchange among diverse viruses and pervasive HVT, often between distantly related hosts, such as animals and plants. Together, these processes have shaped a complex network of evolutionary relationships among RNA viruses.</p><p id="p-81">The much anticipated comprehensive exploration of the RNA viromes of prokaryotes and unicellular eukaryotes, such as free-living excavates, chromalveolates, rhizaria, amoebozoa and choanoflagellates, as well as deeply rooted metazoa, will undoubtedly help in developing better supported evolutionary scenarios for each of the 5 major branches of the RNA virus tree. Nevertheless, it is already clear that the current taxonomy of RNA viruses is due for a complete overhaul.</p></div><div class="section" id="sec-20"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-21" class="subsection"><h3>Phylogeny of RNA-dependent RNA polymerases</h3><p id="p-82">Protein sequences belonging to RNA viruses, excluding retroviruses, and unclassified viruses were downloaded from the NCBI GenBank database in April 2017 (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">122</a>). Initial screening for RdRp domains was performed using PSI-BLAST (<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">123</a>)(e-value of 0.01, effective database size of 2×10<sup>8</sup>) with position-specific scoring matrices (PSSMs) produced from the available RdRp alignments. The sources included group-specific alignments for +RNA viruses and dsRNA viruses (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">31</a>) and the 4 PFAM alignments for the -RNA viruses (pfam00602, pfam00946, pfam04196, and pfam06317) from the NCBI conserved domain database (CDD) (<a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">124</a>). Additionally, a set of RTs from group-II introns and non-long terminal repeat (LTR) retrotransposons was extracted from GenBank as an outgroup.</p><p id="p-83">Extracted RdRp footprints were filtered for the fraction of unresolved amino acids (at most 10%) and clustered using UCLUST (<a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">125</a>) with a similarity threshold of 0.9. One representative from each cluster was selected for further analysis. The resulting set contained 4,640 virus RdRps. This set went through several rounds of semi-manual curation whereby sequences were clustered using UCLUST, aligned using MUSCLE (<a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">126</a>), and cross-searched against each other and their parent sequences (often, complete viral polyproteins) using PSI-BLAST and HHSEARCH (<a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">127</a>). Upon the results of these searches, the boundaries of the RdRp domain were expanded or trimmed to improve their compatibility with each other.</p><p id="p-84">The RdRp and RT sequences were subjected to an iterative clustering and aligned procedure, organized as follows: Initially, sequences were clustered using UCLUST with a similarity threshold of 0.5; clustered sequences were aligned using MUSCLE, and singletons were converted to pseudo-alignments consisting of just one sequence. Sites containing more than 67% of gaps were temporarily removed from alignments and pairwise similarity scores were obtained for clusters using HHSEARCH. Scores for a pair of clusters were converted to distances (the <em>d</em><sub>A,B</sub> = −log(<em>s</em><sub>A,B</sub>/min(<em>s</em><sub>A,A</sub>, <em>S</em><sub>B,B</sub>) formula, in which <em>d</em><sub>A,B</sub> is the distance between cluster A and B and <em>S</em><sub>A,B</sub> is the HHSEARCH score for the comparison of these clusters, was used to convert scores s to distances <em>d</em>). The matrix of pairwise distances was used to make an unweighted pair group method with arithmetic (UPGMA) mean (<a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">128</a>). Shallow tips of the tree were used as the guide tree for a progressive pairwise alignment of the clusters at the tree leaves using HHALIGN (<a id="xref-ref-127-2" class="xref-bibr" href="#ref-127">127</a>), resulting in larger clusters. This procedure was reiterative, ultimately resulting in the single alignment of the whole set of 4,640 virus RdRp sequences and 1,028 RT sequences.</p><p id="p-85">During the clustering procedure, 50 virus RdRp clusters, consisting of 1 to 545 sequences, were defined. These clusters represent either well-established groups of related viruses (roughly comparable to the ICTV family rank) or, in case of poorly characterized and unclassified viruses, groups of well-aligned RdRps that are clearly distinct from others. In all cases, uncertainties were treated conservatively, i.e., in favor of placing sequences with questionable relatedness into separate clusters. Additionally, RT sequences were placed into two clusters consisting of group-II intron RTs and non-LTR retrotransposon RTs.</p><p id="p-86">For each cluster consisting of more than two sequences, an approximate maximum likelihood (ML) phylogenetic tree was constructed from the cluster-specific alignment using the FastTree program (<a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">129</a>) (Whelan and Goldman [WAG] evolutionary model with gamma-distributed site rates) with sites, containing more than 50% of gaps removed from the alignment. Trees were rooted using a variant of the mid-point rooting procedure such that the difference between the (weighted) average root-to-tip distances in the subtrees on the opposite sides of the root is minimized.</p><p id="p-87">To resolve the structure of the global relationships, up to five representatives from each cluster were selected using the within-cluster trees to ensure the diversity of the selected sequences. This procedure resulted in a set of 228 virus RdRps and 10 RTs. The alignments of the selected sequences were extracted from the master alignment and filtered for sites containing more than 50% of gaps. A ML phylogenetic tree was reconstructed for the resulting alignment using PhyML (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>) (Le Gascuel [LG] evolutionary model with gamma-distributed site rates and empirical amino acid frequencies; aBayes support values). Another form of branch support, bootstrap support by transfer (BOOSTER) phylogenetic bootstrap (<a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">130</a>), was also used to assess the reliability of the major tree divisions. Alternatively, the same RdRp alignment was used as the input for ML phylogenetic analysis using RAxML (LG evolutionary model with gamma-distributed site rates and empirical amino acid frequencies).</p><p id="p-88">RdRps in the global tree were divided into 5 major branches (supergroups). Up to 15 representatives from each cluster were selected to form supergroup-level alignments. Respective trees were reconstructed from these alignments using the same procedure (PhyML tree with LG evolutionary model with gamma-distributed site rates and empirical amino acid frequencies).</p><p id="p-89">The overall tree was assembled manually by first replacing the supergroup representatives in the global tree with the supergroup trees, and then, replacing the cluster representatives with the cluster trees. The lower-level trees were rooted according to the arrangement of the representatives in the upper-level tree.</p></div><div id="sec-22" class="subsection"><h3>Identification of protein domains</h3><p id="p-90">Protein domains were identified using a representative set of RNA virus genomes, including representative members of all ICTV-approved virus families and unclassified virus groups. This set was annotated manually using sensitive profile-profile comparisons with the HHsuite package (<a id="xref-ref-127-3" class="xref-bibr" href="#ref-127">127</a>), and hmm profiles for annotated proteins or their domains were generated by running one iteration of HHblits against the latest (October 2017) uniclust30 database (<a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">131</a>). Each annotated profile was assigned to a functional category (e.g., “capsid protein_jelly-roll”, “chymotrypsin-like protease”). These profiles were used to annotate the genomes of all viruses included in the RdRp phylogenetic analysis and for which complete (or near-complete) genome sequences were available. Profiles for the latter proteins were generated by running one iteration of Jackhmmer (<a id="xref-ref-132-1" class="xref-bibr" href="#ref-132">132</a>) against UniRef50 database. Protein regions that did not have significant hits were extracted and clustered with cluster analysis of sequences (CLANS) (<a id="xref-ref-133-1" class="xref-bibr" href="#ref-133">133</a>), and groups containing at least three members were identified, annotated (if possible), and added to the manually annotated profile database. The last step of the domain identification procedure was then repeated using the updated RNA virus profile database. Highly similar viruses (having identical domain organization and &gt;94% identical RdRps) were removed from the dataset. In addition, only one representative genome was retained for some members of over-represented species (e.g., <em>Hepacivirus</em> C). Many genomes encoded readthrough proteins (e.g., alphaviruses, tombusviruses, nidoviruses), resulting in domains from the shorter protein contained within the longer readthrough protein. Such redundancies were removed by filtering out the shorter of the two proteins sharing &gt;80% identity. The final set of annotated genomes included 2,839 viruses.</p></div><div id="sec-23" class="subsection"><h3>Reconstruction of the history of gene gain and loss</h3><p id="p-91">The protein domains identified in the 2,839 virus genomes were mapped onto the composite tree of virus RdRps. A set of 500 representatives was chosen among the (mostly complete) genomes. The domain complement data and the respective tree were analyzed using the GLOOME program (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a>). Domain gains and losses were inferred at each tree branch using the difference of posterior probabilities for the domain occurrence in the nodes at the proximal and the distal ends of the branch (difference &gt;0.5 implies gain; difference &lt;-0.5 implies loss).</p></div><div id="sec-24" class="subsection"><h3>Clustering analysis</h3><p id="p-92">The SJR CP network was generated by performing all-against-all comparison of the SJR CP profiles. To generate hmm profiles, SJR CP sequences were extracted from the total proteome of RNA viruses and two iterations of HHblits were performed against the uniprot20_2016_02 database. The resultant profiles were compared to each other with HHsearch (<a id="xref-ref-127-4" class="xref-bibr" href="#ref-127">127</a>). Their similarity P-values were extracted from the result files and used as an input for CLANS program (<a id="xref-ref-133-2" class="xref-bibr" href="#ref-133">133</a>). Clusters were identified using a network-based algorithm implemented in CLANS. Resulting clusters were manually inspected and refined.</p></div><div id="sec-25" class="subsection"><h3>Analysis of bipartite gene-genome networks</h3><p id="p-93">A bipartite network was built to study the patterns of gene sharing among viral genomes (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">106</a>). After removing genomes with less than 2 domains and domains that appear in less than 3 genomes, the network consisted of 2,829 nodes, of which 2,515 correspond to genomes and 314 to domains. Genome and domain nodes were connected by links whenever a domain is present in a genome. A consensus community detection approach was used to identify the modules of the network (<a id="xref-ref-134-1" class="xref-bibr" href="#ref-134">134</a>, <a id="xref-ref-135-1" class="xref-bibr" href="#ref-135">135</a>). First, we ran 500 replicas with Infomap (<a id="xref-ref-136-1" class="xref-bibr" href="#ref-136">136</a>) (bipartite setting, using domains as factors) and built a similarity matrix by assigning to each pair of nodes a similarity value equal to the fraction of replicas in which both nodes were placed in the same module. Then, hierarchical clustering was performed on the similarity matrix, setting the number of clusters equal to the median number of modules obtained in the 500 replicas. The order statistics local optimization method (Oslom) software (<a id="xref-ref-137-1" class="xref-bibr" href="#ref-137">137</a>) was subsequently used to filter significant modules with a p-value threshold equal to 0.05. To detect higher-order (super)modules, we first identified connector domains as those present in at least 2 modules with prevalence greater than 0.65. The 2nd-order network composed of 54 modules and 34 connector domains was searched for 2nd-order modules with Infomap (500 replicas, bipartite setting with connector domains as factors). Due to the small size of the 2nd-order network, consensus community detection did not qualitatively improve the results of the search, and therefore we took the replica with the best Infomap score and skipped hierarchical clustering for this and subsequent steps of the higher-order module search. After assessing statistical significance with Oslom, 4 2nd-order modules were recovered, encompassing 12 of the original modules associated with closely related virus families. A 3rd-order network was built by pooling these 4 2nd-order modules withthe 46 modules that remained unmerged. The 3rd-order network was analyzed in the same manner to obtain the 9 supermodules. None of these supermodules was assessed as significant by Oslom.</p></div><div id="sec-26" class="subsection"><h3>Quantification of virus host range diversity</h3><p id="p-94">Known hosts or, in case of viruses, isolated from holobionts, virus sources were identified for 3,456 viruses. The host taxonomy (to the phylum level) was mapped to the leaves of the combined tree. The tree was ultrametrized and the weights were computed for all leaves as described in (<a id="xref-ref-138-1" class="xref-bibr" href="#ref-138">138</a>). Each internal node of the tree, therefore, defines a set of leaves with assigned hosts; the distribution of (weighted) relative frequencies of hosts can be characterized by its Shannon entropy. Each leaf has one host defined, so the entropy of the host range distribution is 0, whereas at the root the host diversity is maximal. The entropy generally declines along the tree because viruses that belong to relatively shallow branches tend to share their host ranges. The node depth vs node host range entropy data was averaged for each major branch separately using a Gaussian kernel with the bandwidth equal to 10% of the total tree depth.</p></div></div><div class="section" id="sec-27"><h2 class="">SUPPLEMENTAL MATERIAL</h2><p id="p-96">Supplemental Figure S1. Iterative clustering-alignment-phylogeny procedure.</p><p id="p-97">Supplemental Figure S2. (A) Gain and loss of capsid (nucleocapsid) proteins and movement proteins. (B) Gain and loss of key enzymes. Each branch is shown by a triangle which represents collapsed sequences of the respective set of RdRps. The 5 main branches discussed in the text are indicated. The genes (domains) are denoted by distinct shapes shown at the bottom of each panel. Different colors show distinct families of proteins with similar functions (helicases, proteases, capping enzymes) and different hues of the same color show distinct subfamilies of the same family that are likely to have been acauired independently. The symbols at internal nodes denoted the inferred point of origin (gain) of the respective gene. Empty shapes show loss of the respective genes. Shapes placed inside triangles indicate the presence of the respective gene in a subset of the viruses in the respective branch.</p><p id="p-98">Supplemental Figure S3. Sequence similarity networks of SJR-CPs. Protein sequences were clustered by the pairwise similarity of their hmm profiles using CLANS. Different groups of SJR-CPs are shown as clouds of differentially colored circles, with the corresponding subgroups labeled as indicated in the figure. Edges connect sequences with CLANS P-value &lt; 1e-10.</p><p id="p-99">Supplemental Data set S1. List of viruses.</p><p id="p-100">Supplemental Data set S2. Phylogenetic tree for the global set of RdRps; Newick format.</p><p id="p-101">Supplemental Data set S3. Phylogenetic trees for the global set of RdRp representatives; Newick format. A. Reconstructed using PhyML. B. Reconstructed using RAxML.</p><p id="p-102">Supplemental Data set S4. Phylogenetic trees for RdRp representatives; Newick format.</p><p id="p-103">A. Branch 1 representatives.</p><p id="p-104">B. Branch 2 representatives.</p><p id="p-105">C. Branch 3 representatives.</p><p id="p-106">D. Branch 4 representatives.</p><p id="p-107">E. Branch 5 representatives</p><p id="p-108">Supplemental Data set S5. Proteins domains encoded by the virus genomes.</p><p id="p-109">Additional data can be found at the following directory: <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/wolf/suppl/rnavir18">ftp://ftp.ncbi.nlm.nih.gov/pub/wolf/suppl/rnavir18</a></p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-95">The content of this publication does not necessarily reflect the views or policies of the US Department of the Army, the US Department of Defense, or the US Department of Health and Human Services or of the institutions and companies affiliated with the authors. VVD is grateful to Barbara and Simon Gvakharia for their support and inspiration. YIW and EVK are supported through the intramural program of the National Institutes of Health of the USA. DK was partly supported by a Short Term Fellowship from the Federation of European Biochemical Societies (FEBS). MK was supported by l’Agence Nationale de la Recherche (ANR, France) project ENVIRA. This work was funded in part through Battelle Memorial Institute’s prime contract with the US National Institute of Allergy and Infectious Diseases (NIAID) under Contract No. HHSN272200700016I (JHK). We thank Jiro Wada (IRF-Frederick) for help with figure creation.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.1" data-doi="10.1186/1745-6150-7-23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernhardt</span>  <span class="cit-name-given-names">HS</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The RNA world hypothesis: the worst theory of the early evolution of life (except for all the others)(a)</span>. <abbr class="cit-jnl-abbrev">Biol Direct</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DBernhardt%26rft.auinit1%253DH.%2BS.%26rft.volume%253D7%26rft.spage%253D23%26rft.epage%253D23%26rft.atitle%253DThe%2BRNA%2Bworld%2Bhypothesis%253A%2Bthe%2Bworst%2Btheory%2Bof%2Bthe%2Bearly%2Bevolution%2Bof%2Blife%2B%2528except%2Bfor%2Ball%2Bthe%2Bothers%2529a.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-7-23%26rft_id%253Dinfo%253Apmid%252F22793875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-7-23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22793875&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.2" data-doi="10.1038/319618a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W</span></span>. <span class="cit-pub-date">1986</span>. <span class="cit-article-title">Origin of Life the RNA World</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D319%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1038%252F319618a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/319618a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.3" data-doi="10.1126/scisignal.aam8812"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breaker</span>  <span class="cit-name-given-names">RR</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The lost language of the RNA World</span>. <abbr class="cit-jnl-abbrev">Sci Signal</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">eaam8812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DNelson%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D10%26rft.issue%253D483%26rft.spage%253Deaam8812%26rft.epage%253Deaam8812%26rft.atitle%253DThe%2Blost%2Blanguage%2Bof%2Bthe%2BRNA%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.aam8812%26rft_id%253Dinfo%253Apmid%252F28611182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6MTU6IjEwLzQ4My9lYWFtODgxMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.4" data-doi="10.1186/1745-6150-1-29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Senkevich</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">The ancient Virus World and evolution of cells</span>. <abbr class="cit-jnl-abbrev">Biol Direct</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiol%2BDirect%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D1%26rft.spage%253D29%26rft.epage%253D29%26rft.atitle%253DThe%2Bancient%2BVirus%2BWorld%2Band%2Bevolution%2Bof%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-6150-1-29%26rft_id%253Dinfo%253Apmid%252F16984643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-6150-1-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16984643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">1971</span>. <span class="cit-article-title">Expression of animal virus genomes</span>. <abbr class="cit-jnl-abbrev">Bacteriol Rev</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DBaltimore%26rft.auinit1%253DD%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D235%26rft.epage%253D241%26rft.atitle%253DExpression%2Bof%2Banimal%2Bvirus%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F4329869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo0OiJtbWJyIjtzOjU6InJlc2lkIjtzOjg6IjM1LzMvMjM1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.6" data-doi="10.1111/j.2164-0947.1971.tb02600.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">1971</span>. <span class="cit-article-title">Viral genetic systems</span>. <abbr class="cit-jnl-abbrev">Trans N Y Acad Sci</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransactions%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DTransactions%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.aulast%253DBaltimore%26rft.auinit1%253DD.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D327%26rft.epage%253D332%26rft.atitle%253DViral%2Bgenetic%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2164-0947.1971.tb02600.x%26rft_id%253Dinfo%253Apmid%252F4327120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2164-0947.1971.tb02600.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4327120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.7" data-doi="10.3109/10409239309078440"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">Evolution and taxonomy of positive-strand RNA viruses: implications of comparative analysis of amino acid sequences</span>. <abbr class="cit-jnl-abbrev">Crit Rev Biochem Mol Biol</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D375%26rft.epage%253D430%26rft.atitle%253DEvolution%2Band%2Btaxonomy%2Bof%2Bpositive-strand%2BRNA%2Bviruses%253A%2Bimplications%2Bof%2Bcomparative%2Banalysis%2Bof%2Bamino%2Bacid%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.3109%252F10409239309078440%26rft_id%253Dinfo%253Apmid%252F8269709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/10409239309078440&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8269709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MA73700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.8" data-doi="10.1016/j.coviro.2013.06.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A virocentric perspective on the evolution of life</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D3%26rft.spage%253D546%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2013.06.008%26rft_id%253Dinfo%253Apmid%252F23850169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2013.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23850169&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.9" data-doi="10.1038/nrmicro1389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahlquist</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Parallels among positive-strand RNA viruses, reverse-transcribing viruses and double-stranded RNA viruses</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DAhlquist%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D371%26rft.epage%253D382%26rft.atitle%253DParallels%2Bamong%2Bpositive-strand%2BRNA%2Bviruses%252C%2Breverse-transcribing%2Bviruses%2Band%2Bdouble-stranded%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro1389%26rft_id%253Dinfo%253Apmid%252F16582931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro1389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16582931&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236764400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.10" data-doi="10.1016/j.sbi.2016.01.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reguera</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerlach</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cusack</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Towards a structural understanding of RNA synthesis by negative strand RNA viral polymerases</span>. <abbr class="cit-jnl-abbrev">Curr Opin Struct Biol</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.volume%253D36%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2016.01.002%26rft_id%253Dinfo%253Apmid%252F26826467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2016.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26826467&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.11" data-doi="10.1016/j.virusres.2017.10.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">244</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D244%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.12" data-doi="10.1128/JVI.07196-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolduc</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaughnessy</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">YI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberto</span>  <span class="cit-name-given-names">FF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Identification of novel positive-strand RNA viruses by metagenomic analysis of archaea-dominated Yellowstone hot springs</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">5562</span>–<span class="cit-lpage">5573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.07196-11%26rft_id%253Dinfo%253Apmid%252F22379100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xMC81NTYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.13" data-doi="10.1371/journal.pbio.1002409"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krishnamurthy</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janowski</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barouch</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Hyperexpansion of RNA bacteriophage diversity</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1002409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D14%26rft.spage%253De1002409%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002409%26rft_id%253Dinfo%253Apmid%252F27010970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27010970&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.14" data-doi="10.1038/nature20167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X-D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C-X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">X-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eden</span>  <span class="cit-name-given-names">J-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchmann</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y-Z</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Redefining the invertebrate RNA virosphere</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">540</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D540%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20167%26rft_id%253Dinfo%253Apmid%252F27880757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27880757&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Origins and evolution of viruses of eukaryotes: the ultimate modularity</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">479–480</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.16" data-doi="10.1016/j.coviro.2011.09.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Common origins and host-dependent diversity of plant and animal viromes</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">322</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BOpin%2BVirol%26rft.aulast%253DDolja%26rft.auinit1%253DV.%2BV.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D322%26rft.epage%253D331%26rft.atitle%253DCommon%2Borigins%2Band%2Bhost-dependent%2Bdiversity%2Bof%2Bplant%2Band%2B%2Banimal%2Bviromes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2011.09.007%26rft_id%253Dinfo%253Apmid%252F22408703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2011.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22408703&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.17" data-doi="10.1016/j.virusres.2017.10.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y-Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Meta-transcriptomics and The Evolutionary Biology of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">243</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D243%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.18" data-doi="10.1128/MMBR.00049-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Virus world as an evolutionary network of viruses and capsidless selfish elements</span>. <abbr class="cit-jnl-abbrev">Microbiol Mol Biol Rev</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D278%26rft.epage%253D303%26rft.atitle%253DVirus%2BWorld%2Bas%2Ban%2BEvolutionary%2BNetwork%2Bof%2BViruses%2Band%2BCapsidless%2BSelfish%2BElements%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00049-13%26rft_id%253Dinfo%253Apmid%252F24847023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3OC8yLzI3OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.19" data-doi="10.1073/pnas.1621061114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Multiple origins of viral capsid proteins from cellular ancestors</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E2401</span>–<span class="cit-lpage">E2410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1621061114%26rft_id%253Dinfo%253Apmid%252F28265094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzEyL0UyNDAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.20" data-doi="10.1016/0042-6822(91)90823-T"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">VP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agranovsky</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Phylogeny of capsid proteins of rod-shaped and filamentous RNA plant viruses: two families with distinct patterns of sequence and probably structure conservation</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DDolja%26rft.auinit1%253DV.%2BV.%26rft.volume%253D184%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D86%26rft.atitle%253DPhylogeny%2Bof%2Bcapsid%2Bproteins%2Bof%2Brod-shaped%2Band%2Bfilamentous%2BRNA%2Bplant%2Bviruses%253A%2Btwo%2Bfamilies%2Bwith%2Bdistinct%2Bpatterns%2Bof%2Bsequence%2Band%2Bprobably%2Bstructure%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252891%252990823-T%26rft_id%253Dinfo%253Apmid%252F1871982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(91)90823-T&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1871982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GA34500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">The phylogeny of RNA-dependent RNA polymerases of positive-strand RNA viruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol 72 (Pt</abbr> <span class="cit-vol">9</span>): <span class="cit-fpage">2197</span>–<span class="cit-lpage">2206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%2B72%2B%2528Pt%26rft.volume%253D9%26rft.spage%253D2197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eickbush</span>  <span class="cit-name-given-names">TH</span></span>. <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Origin and evolution of retroelements based upon their reverse transcriptase sequences</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3353</span>–<span class="cit-lpage">3362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D9%26rft.spage%253D3353%26rft_id%253Dinfo%253Apmid%252F1698615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1698615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EA35300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poch</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauvaget</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delarue</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tordo</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">1989</span>. <span class="cit-article-title">Identification of four conserved motifs among the RNA-dependent polymerase encoding elements</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3867</span>–<span class="cit-lpage">3874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D8%26rft.spage%253D3867%26rft_id%253Dinfo%253Apmid%252F2555175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2555175&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989CA40000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.24" data-doi="10.1007/978-3-540-75157-1_7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">KK-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Structure-function relationships among RNA-dependent RNA polymerases</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253DNg%26rft.auinit1%253DK.%2BK.%26rft.volume%253D320%26rft.spage%253D137%26rft.epage%253D156%26rft.atitle%253DStructure-function%2Brelationships%2Bamong%2BRNA-dependent%2BRNA%2Bpolymerases.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-540-75157-1_7%26rft_id%253Dinfo%253Apmid%252F18268843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-540-75157-1_7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18268843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253430500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.25" data-doi="10.1007/s00018-014-1695-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">te Velthuis</span>  <span class="cit-name-given-names">AJW</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Common and unique features of viral RNA-dependent polymerases</span>. <abbr class="cit-jnl-abbrev">Cell Mol Life Sci</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">4403</span>–<span class="cit-lpage">4420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMol%2BLife%2BSci%26rft.volume%253D71%26rft.spage%253D4403%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-014-1695-z%26rft_id%253Dinfo%253Apmid%252F25080879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-014-1695-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25080879&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.26" data-doi="10.1016/j.virusres.2007.12.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eickbush</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamburuthugoda</span>  <span class="cit-name-given-names">VK</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">The diversity of retrotransposons and the properties of their reverse transcriptases</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DEickbush%26rft.auinit1%253DT.%2BH.%26rft.volume%253D134%26rft.issue%253D1-2%26rft.spage%253D221%26rft.epage%253D234%26rft.atitle%253DThe%2Bdiversity%2Bof%2Bretrotransposons%2Band%2Bthe%2Bproperties%2Bof%2Btheir%2Breverse%2Btranscriptases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.12.010%26rft_id%253Dinfo%253Apmid%252F18261821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.12.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18261821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256291900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.27" data-doi="10.1073/pnas.1100266108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arkhipova</span>  <span class="cit-name-given-names">IR</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">A widespread class of reverse transcriptase-related cellular genes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">20311</span>–<span class="cit-lpage">20316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100266108%26rft_id%253Dinfo%253Apmid%252F21876125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzUxLzIwMzExIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lambowitz</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belfort</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Mobile bacterial group II introns at the crux of eukaryotic evolution</span>. <abbr class="cit-jnl-abbrev">Microbiol Spectr</abbr> <span class="cit-vol">3</span>: MDNA3–0050–2014.</cite></div><div class="cit-extra"></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.29" data-doi="10.1016/j.tig.2017.07.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Novikova</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belfort</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Mobile group II introns as ancestral eukaryotic elements</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet%26rft.volume%253D33%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.07.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coffin</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dasgupta</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geering</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hull</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kreuze</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindemann</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Llorens</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lockhart</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olszewski</span>  <span class="cit-name-given-names">NE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pappu</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pooggin</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richert-Poggeler</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanfacon</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoelz</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seal</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stavolone</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoye</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teycheney</span>  <span class="cit-name-given-names">P-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tristem</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">JH</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Ortervirales: new virus order unifying five families of reverse-transcribing viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">92</span>: e<span class="cit-fpage">00515</span>–<span class="cit-lpage">00518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D92%26rft.spage%253D00515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.31" data-doi="10.1038/nrmicro2030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">YI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagasaki</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">The Big Bang of picorna-like virus evolution antedates the radiation of eukaryotic supergroups</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D925%26rft.epage%253D939%26rft.atitle%253DThe%2BBig%2BBang%2Bof%2Bpicorna-like%2Bvirus%2Bevolution%2Bantedates%2Bthe%2Bradiation%2Bof%2Beukaryotic%2Bsupergroups.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2030%26rft_id%253Dinfo%253Apmid%252F18997823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18997823&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260903500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Evolution of double-stranded RNA viruses: a case for polyphyletic origin from different groups of positive-stranded RNA viruses</span>. <abbr class="cit-jnl-abbrev">Semin Virol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin%2BVirol%26rft.volume%253D3%26rft.spage%253D327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.33" data-doi="10.1016/j.str.2013.06.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Omari</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ravantti</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bamford</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mancini</span>  <span class="cit-name-given-names">EJ</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Plate tectonics of virus shell assembly and reorganization in phage Φ8, a distant relative of mammalian reoviruses</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D21%26rft.spage%253D1384%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2013.06.017%26rft_id%253Dinfo%253Apmid%252F23891291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2013.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23891291&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.34" data-doi="10.1007/978-1-4614-0980-9_17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poranen</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bamford</span>  <span class="cit-name-given-names">DH</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Assembly of large icosahedral double-stranded RNA viruses</span>. <abbr class="cit-jnl-abbrev">Adv Exp Med Biol</abbr> <span class="cit-vol">726</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DPoranen%26rft.auinit1%253DM.%2BM.%26rft.volume%253D726%26rft.spage%253D379%26rft.epage%253D402%26rft.atitle%253DAssembly%2Bof%2Blarge%2Bicosahedral%2Bdouble-stranded%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4614-0980-9_17%26rft_id%253Dinfo%253Apmid%252F22297523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4614-0980-9_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22297523&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.35" data-doi="10.1038/nature14008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pflug</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guilligay</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cusack</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Structure of influenza A polymerase bound to the viral RNA promoter</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14008%26rft_id%253Dinfo%253Apmid%252F25409142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409142&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000346484800038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.36" data-doi="10.1159/000150054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldbach</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wellink</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">1988</span>. <span class="cit-article-title">Evolution of plus-strand RNA viruses</span>. <abbr class="cit-jnl-abbrev">Intervirology</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntervirology%26rft.stitle%253DIntervirology%26rft.aulast%253DGoldbach%26rft.auinit1%253DR.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D260%26rft.epage%253D267%26rft.atitle%253DEvolution%2Bof%2Bplus-strand%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000150054%26rft_id%253Dinfo%253Apmid%252F3058643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000150054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3058643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.37" data-doi="10.1016/j.virusres.2017.10.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greninger</span>  <span class="cit-name-given-names">AL</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A decade of RNA virus metagenomics is (not) enough</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">244</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D244%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.38" data-doi="10.1128/JVI.02036-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X-D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasilakis</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C-X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eastwood</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diao</span>  <span class="cit-name-given-names">X-N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">X-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Widen</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tesh</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y-Z</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Divergent viruses discovered in arthropods and vertebrates revise the evolutionary history of the <em>Flaviviridae</em> and related viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02036-15%26rft_id%253Dinfo%253Apmid%252F26491167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjkwLzIvNjU5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C-X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X-D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">Y-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">X-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y-Z</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Unprecedented genomic diversity of RNA viruses in arthropods reveals the ancestry of negative-sense RNA viruses</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: e<span class="cit-fpage">05378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253D05378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.40" data-doi="10.4137/EBO.S39454"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longdon</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obbard</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Twenty-five new viruses associated with the Drosophilidae (Diptera)</span>. <abbr class="cit-jnl-abbrev">Evol Bioinform Online</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BBioinform%2BOnline%26rft.volume%253D12%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.4137%252FEBO.S39454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4137/EBO.S39454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fauver</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grubaugh</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krajacich</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weger-Lucarelli</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lakin</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fakoli</span>  <span class="cit-name-given-names">LS</span></span>, 3rd, <span class="cit-auth"><span class="cit-name-surname">Bolay</span>  <span class="cit-name-given-names">FK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diclaro</span>  <span class="cit-name-given-names">JW</span></span>, 2nd, <span class="cit-auth"><span class="cit-name-surname">Dabire</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foy</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brackney</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebel</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">West African <em>Anopheles gambiae</em> mosquitoes harbor a taxonomically diverse virome including new insect-specific flaviviruses, mononegaviruses, and totiviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">498</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D498%26rft.spage%253D288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.42" data-doi="10.1128/JVI.00357-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marzano</span>  <span class="cit-name-given-names">S-YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ajayi-Oyetunde</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eastburn</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domier</span>  <span class="cit-name-given-names">LL</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">6846</span>–<span class="cit-lpage">6863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00357-16%26rft_id%253Dinfo%253Apmid%252F27194764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82ODQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.43" data-doi="10.1038/s41598-017-01592-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deakin</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobbs</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grogan</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">KS</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Multiple viral infections in <em>Agaricus bisporus</em>-Characterisation of 18 unique RNA viruses and 8 ORFans identified by deep sequencing</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D2469%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-01592-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-01592-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iranzo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The double-stranded DNA virosphere as a modular hierarchical network of gene sharing</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">7</span>: e<span class="cit-fpage">00978</span>–<span class="cit-lpage">00916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D7%26rft.spage%253D00978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iranzo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">A network perspective on the virus world</span>. <abbr class="cit-jnl-abbrev">Commun Integr Biol</abbr> <span class="cit-vol">10</span>: e<span class="cit-fpage">1296614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun%2BIntegr%2BBiol%26rft.volume%253D10%26rft.spage%253D1296614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.46" data-doi="10.1016/S0022-2836(02)01033-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeddam</span>  <span class="cit-name-given-names">J-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luke</span>  <span class="cit-name-given-names">BT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalmakoff</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanzlik</span>  <span class="cit-name-given-names">TN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">KHJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">VK</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">The palm subdomain-based active site is internally permuted in viral RNA-dependent RNA polymerases of an ancient lineage</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D324%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D62%26rft.atitle%253DThe%2Bpalm%2Bsubdomain-based%2Bactive%2Bsite%2Bis%2Binternally%2Bpermuted%2Bin%2Bviral%2BRNA-dependent%2BRNA%2Bpolymerases%2Bof%2Ban%2Bancient%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252802%252901033-1%26rft_id%253Dinfo%253Apmid%252F12421558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(02)01033-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12421558&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179530500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.47" data-doi="10.1007/s00705-018-3847-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">AMQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefkowitz</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mushegian</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutilh</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kropinski</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rubino</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanfaçon</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varsani</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zerbini</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Changes to taxonomy and the International Code of Virus Classification and Nomenclature ratified by the International Committee on Taxonomy of Viruses (2018)</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">2601</span>–<span class="cit-lpage">2631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D163%26rft.spage%253D2601%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-018-3847-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-018-3847-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48 in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.48" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49 in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Akopyants</span>  <span class="cit-name-given-names">NS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lye</span>  <span class="cit-name-given-names">L-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lukeš</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beverley</span>  <span class="cit-name-given-names">SM</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">A novel bunyavirus-like virus of trypanosomatid protist parasites</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr> <span class="cit-vol">4</span>: e<span class="cit-fpage">00715</span>–<span class="cit-lpage">00716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc%26rft.volume%253D4%26rft.spage%253D00715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50 in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.50" data-doi="10.1073/pnas.1717806115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grybchuk</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akopyants</span>  <span class="cit-name-given-names">NS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kostygov</span>  <span class="cit-name-given-names">AY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konovalovas</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lye</span>  <span class="cit-name-given-names">L-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zangger</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fasel</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butenko</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frolov</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Votýpka</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">d’Avila-Levy</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulich</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moravcová</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plevka</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">IB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serva</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lukeš</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beverley</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurchenko</span>  <span class="cit-name-given-names">V</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Viral discovery and diversity in trypanosomatid protozoa with a focus on relatives of the human parasite Leishmania</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">E506</span>–<span class="cit-lpage">E515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1717806115%26rft_id%253Dinfo%253Apmid%252F29284754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE1LzMvRTUwNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51 in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.51" data-doi="10.1093/bioinformatics/btq549"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashkenazy</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belinky</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huchon</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pupko</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">GLOOME: gain loss mapping engine</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2914</span>–<span class="cit-lpage">2915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq549%26rft_id%253Dinfo%253Apmid%252F20876605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20876605&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283919800022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52 in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.52" data-doi="10.1006/jmbi.1993.1616"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Golmohammadi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valegård</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fridborg</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liljas</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">The refined structure of bacteriophage MS2 at 2.8 A resolution</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">234</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGolmohammadi%26rft.auinit1%253DR.%26rft.volume%253D234%26rft.issue%253D3%26rft.spage%253D620%26rft.epage%253D639%26rft.atitle%253DThe%2Brefined%2Bstructure%2Bof%2Bbacteriophage%2BMS2%2Bat%2B2.8%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1993.1616%26rft_id%253Dinfo%253Apmid%252F8254664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1993.1616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8254664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MK79900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53 in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.53" data-doi="10.1016/B978-0-12-394315-6.00006-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hillman</span>  <span class="cit-name-given-names">BI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The family Narnaviridae: simplest of RNA viruses</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D86%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-394315-6.00006-4%26rft_id%253Dinfo%253Apmid%252F23498906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-394315-6.00006-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54 in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.54" data-doi="10.1016/j.virol.2018.02.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vong</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fugate</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">HJ</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Evidence for contemporary plant mitoviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">518</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D518%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2018.02.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2018.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55 in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.55" data-doi="10.1099/vir.0.013086-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rastgou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Habibi</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Izadpanah</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masenga</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">YI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turina</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Molecular characterization of the plant virus genus <em>Ourmiavirus</em> and evidence of inter-kingdom reassortment of viral genome segments as its possible route of origin</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">2525</span>–<span class="cit-lpage">2535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.013086-0%26rft_id%253Dinfo%253Apmid%252F19535502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.013086-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19535502&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56 in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.56" data-doi="10.1007/s00705-008-0041-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Gall</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">AMQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakashima</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanway</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Picornavirales, a proposed order of positive-sense single-stranded RNA viruses with a pseudo-T = 3 virion architecture</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DLe%2BGall%26rft.auinit1%253DO.%26rft.volume%253D153%26rft.issue%253D4%26rft.spage%253D715%26rft.epage%253D727%26rft.atitle%253DPicornavirales%252C%2Ba%2Bproposed%2Border%2Bof%2Bpositive-sense%2Bsingle-stranded%2BRNA%2Bviruses%2Bwith%2Ba%2Bpseudo-T%2B%253D%2B3%2Bvirion%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-008-0041-x%26rft_id%253Dinfo%253Apmid%252F18293057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-008-0041-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18293057&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254258600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57 in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.57" data-doi="10.1016/j.virusres.2017.11.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Culley</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">New insight into the RNA aquatic virosphere via viromics</span>. <abbr class="cit-jnl-abbrev">Virus Research</abbr> <span class="cit-vol">244</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft.volume%253D244%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.11.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.58" data-doi="10.1128/JVI.00869-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">TFF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marine</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapusinszky</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodhidatta</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oderinde</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wommack</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">High variety of known and new RNA and DNA viruses of diverse origins in untreated sewage</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">12161</span>–<span class="cit-lpage">12175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00869-12%26rft_id%253Dinfo%253Apmid%252F22933275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMi8xMjE2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59 in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.59" data-doi="10.1099/jgv.0.000461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hause</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palinski</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hesse</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Highly diverse posaviruses in swine faeces are aquatic in origin</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1362</span>–<span class="cit-lpage">1367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D97%26rft.spage%253D1362%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60 in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.60" data-doi="10.1073/pnas.90.22.10539"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monroe</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stine</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">RI</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">RNA sequence of astrovirus: distinctive genomic organization and a putative retrovirus-like ribosomal frameshifting signal that directs the viral replicase synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10539</span>–<span class="cit-lpage">10543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DJiang%26rft.auinit1%253DB.%26rft.volume%253D90%26rft.issue%253D22%26rft.spage%253D10539%26rft.epage%253D10543%26rft.atitle%253DRNA%2Bsequence%2Bof%2Bastrovirus%253A%2Bdistinctive%2Bgenomic%2Borganization%2Band%2Ba%2Bputative%2Bretrovirus-like%2Bribosomal%2Bframeshifting%2Bsignal%2Bthat%2Bdirects%2Bthe%2Bviral%2Breplicase%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.22.10539%26rft_id%253Dinfo%253Apmid%252F8248142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjIvMTA1MzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8yNC80NTE3NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61 in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.61" data-doi="10.1101/299776:299776"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saberi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brubacher</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A planarian nidovirus expands the limits of RNA genome size</span>. <abbr class="cit-jnl-abbrev">bioRxiv doi</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/299776:299776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%2Bdoi%26rft_id%253Dinfo%253Adoi%252F10.1101%252F299776%253A299776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/299776:299776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62 in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.62" data-doi="10.1128/AEM.71.12.8888-8894.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagasaki</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shirai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mizumoto</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomaru</span>  <span class="cit-name-given-names">Y</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Comparison of genome sequences of single-stranded RNA viruses infecting the bivalvekilling dinoflagellate Heterocapsa circularisquama</span>. <abbr class="cit-jnl-abbrev">Appl Environ Microbiol</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">8888</span>–<span class="cit-lpage">8894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DNagasaki%26rft.auinit1%253DK.%26rft.volume%253D71%26rft.issue%253D12%26rft.spage%253D8888%26rft.epage%253D8894%26rft.atitle%253DComparison%2Bof%2BGenome%2BSequences%2Bof%2BSingle-Stranded%2BRNA%2BViruses%2BInfecting%2Bthe%2BBivalve-Killing%2BDinoflagellate%2BHeterocapsa%2Bcircularisquama%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.71.12.8888-8894.2005%26rft_id%253Dinfo%253Apmid%252F16332886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3MS8xMi84ODg4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMjQvNDUxNzQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63 in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.63" data-doi="10.1073/pnas.88.23.10647"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">GH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nuss</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapira</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">JC</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Evidence for common ancestry of a chestnut blight hypovirulence-associated double-stranded RNA and a group of positive-strand RNA plant viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">10647</span>–<span class="cit-lpage">10651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D88%26rft.issue%253D23%26rft.spage%253D10647%26rft.epage%253D10651%26rft.atitle%253DEvidence%2Bfor%2Bcommon%2Bancestry%2Bof%2Ba%2Bchestnut%2Bblight%2Bhypovirulence-associated%2Bdouble-stranded%2BRNA%2Band%2Ba%2Bgroup%2Bof%2Bpositive-strand%2BRNA%2Bplant%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.23.10647%26rft_id%253Dinfo%253Apmid%252F1961731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODgvMjMvMTA2NDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8yNC80NTE3NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64 in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.64" data-doi="10.1016/B978-0-12-394315-6.00005-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nuss</span>  <span class="cit-name-given-names">DL</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Hypovirus molecular biology: from Koch’s postulates to host self-recognition genes that restrict virus transmission</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D86%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-394315-6.00005-2%26rft_id%253Dinfo%253Apmid%252F23498905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-394315-6.00005-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498905&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318253300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65 in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.65" data-doi="10.1016/j.virusres.2014.04.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maiss</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lesker</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vainio</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Taxonomic reorganization of family <em>Partitiviridae</em> and other recent progress in partitivirus research</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D188%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.04.007%26rft_id%253Dinfo%253Apmid%252F24768846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24768846&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66 in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.66" data-doi="10.1016/j.jsb.2010.06.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">WF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Havens</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">TS</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Structure of Fusarium poae virus 1 shows conserved and variable elements of partitivirus capsids and evolutionary relationships to picobirnavirus</span>. <abbr class="cit-jnl-abbrev">J Struct Biol</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DTang%26rft.auinit1%253DJ.%26rft.volume%253D172%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D371%26rft.atitle%253DStructure%2Bof%2BFusarium%2Bpoae%2Bvirus%2B1%2Bshows%2Bconserved%2Band%2Bvariable%2Belements%2Bof%2Bpartitivirus%2Bcapsids%2Band%2Bevolutionary%2Brelationships%2Bto%2Bpicobirnavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2010.06.022%26rft_id%253Dinfo%253Apmid%252F20599510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2010.06.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20599510&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67 in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.67" data-doi="10.1038/emboj.2009.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duquerroy</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Da Costa</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vigouroux</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Libersou</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lepault</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navaza</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delmas</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">FA</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The picobirnavirus crystal structure provides functional insights into virion assembly and cell entry</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DDuquerroy%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D11%26rft.spage%253D1655%26rft.epage%253D1665%26rft.atitle%253DThe%2Bpicobirnavirus%2Bcrystal%2Bstructure%2Bprovides%2Bfunctional%2Binsights%2Binto%2Bvirion%2Bassembly%2Band%2Bcell%2Bentry%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.109%26rft_id%253Dinfo%253Apmid%252F19407816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTEvMTY1NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68 in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.68" data-doi="10.1016/B978-0-12-394315-6.00003-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collier</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">YJ</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">3D structures of fungal partitiviruses</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D86%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-394315-6.00003-9%26rft_id%253Dinfo%253Apmid%252F23498903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-394315-6.00003-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69 in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.69" data-doi="10.1073/pnas.1404330111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luque</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gómez-Blanco</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garriga</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brilot</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Havens</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrascosa</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trus</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdaguer</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castón</span>  <span class="cit-name-given-names">JR</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Cryo-EM near-atomic structure of a dsRNA fungal virus shows ancient structural motifs preserved in the dsRNA viral lineage</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">7641</span>–<span class="cit-lpage">7646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1404330111%26rft_id%253Dinfo%253Apmid%252F24821769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIxLzc2NDEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8yNC80NTE3NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70 in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.70" data-doi="10.1023/A:1005907310553"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koga</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fukuhara</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitta</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Molecular characterization of a single mitochondria-associated double-stranded RNA in the green alga Bryopsis</span>. <abbr class="cit-jnl-abbrev">Plant Mol Biol</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DKoga%26rft.auinit1%253DR.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D717%26rft.epage%253D724%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Ba%2Bsingle%2Bmitochondria-associated%2Bdouble-stranded%2BRNA%2Bin%2Bthe%2Bgreen%2Balga%2BBryopsis.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1005907310553%26rft_id%253Dinfo%253Apmid%252F9526504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1005907310553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9526504&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072341600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71 in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.71" data-doi="10.1023/A:1023003412859"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koga</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fukuhara</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Double-stranded RNA replicons associated with chloroplasts of a green alga <em>Bryopsis cinicola</em></span>. <abbr class="cit-jnl-abbrev">Plant Mol Biol</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DKoga%26rft.auinit1%253DR.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D991%26rft.epage%253D999%26rft.atitle%253DDouble-stranded%2BRNA%2Breplicons%2Bassociated%2Bwith%2Bchloroplasts%2Bof%2Ba%2Bgreen%2Balga%252C%2BBryopsis%2Bcinicola.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1023003412859%26rft_id%253Dinfo%253Apmid%252F12777056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1023003412859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12777056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181944900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72 in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.72" data-doi="10.1016/j.virusres.2006.01.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Evolution of complexity in the viral world: the dawn of a new vision</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DKoonin%26rft.auinit1%253DE.%2BV.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D4%26rft.atitle%253DEvolution%2Bof%2Bcomplexity%2Bin%2Bthe%2Bviral%2Bworld%253A%2Bthe%2Bdawn%2Bof%2Ba%2Bnew%2Bvision.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.018%26rft_id%253Dinfo%253Apmid%252F16497406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16497406&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73 in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.73" data-doi="10.1093/nar/17.21.8413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1989</span>. <span class="cit-article-title">Viral proteins containing the purine NTP-binding sequence pattern</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">8413</span>–<span class="cit-lpage">8440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D17%26rft.issue%253D21%26rft.spage%253D8413%26rft.epage%253D8438%26rft.atitle%253DViral%2Bproteins%2Bcontaining%2Bthe%2Bpurine%2BNTP-binding%2Bsequence%2Bpattern%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.21.8413%26rft_id%253Dinfo%253Apmid%252F2555771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.21.8413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2555771&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AZ79700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74 in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.74" data-doi="10.1016/bs.aivir.2016.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The nonstructural proteins directing coronavirus RNA synthesis and processing</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D96%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2016.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2016.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.75" data-doi="10.1016/bs.aivir.2016.08.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuñiga</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castaño-Rodriguez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gutierrez-Alvarez</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canton</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">I</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Molecular basis of coronavirus virulence and vaccine development</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D96%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2016.08.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2016.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76 in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.76" data-doi="10.1146/annurev-virology-100114-055218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almazán</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zúñiga</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Continuous and discontinuous RNA synthesis in coronaviruses</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D2%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-100114-055218%26rft_id%253Dinfo%253Apmid%252F26958916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-100114-055218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26958916&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77 in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.77" data-doi="10.1099/vir.0.034702-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roossinck</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valverde</span>  <span class="cit-name-given-names">RA</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">The remarkable evolutionary history of endornaviruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2674</span>–<span class="cit-lpage">2678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.034702-0%26rft_id%253Dinfo%253Apmid%252F21775577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.034702-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775577&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78 in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.78" data-doi="10.1016/j.virusres.2015.11.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Characterization of a novel <em>Sclerotinia sclerotiorum</em> RNA virus as the prototype of a new proposed family within the order Tymovirales</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">219</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D219%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.11.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79 in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Syller</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Umbraviruses-the unique plant viruses that do not encode a capsid protein</span>. <abbr class="cit-jnl-abbrev">Acta Microbiol Pol</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Bmicrobiologica%2BPolonica%26rft.stitle%253DActa%2BMicrobiol%2BPol%26rft.aulast%253DSyller%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D2%26rft.spage%253D99%26rft.epage%253D113%26rft.atitle%253DUmbraviruses--the%2Bunique%2Bplant%2Bviruses%2Bthat%2Bdo%2Bnot%2Bencode%2Ba%2Bcapsid%2Bprotein.%26rft_id%253Dinfo%253Apmid%252F12363080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12363080&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80 in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.80" data-doi="10.1016/j.virol.2017.01.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janowski</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krishnamurthy</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenchley</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barouch</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thakwalakwa</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manary</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holtz</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Statoviruses, a novel taxon of RNA viruses present in the gastrointestinal tracts of diverse mammals</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">504</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D504%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.01.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81 in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greninger</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Draft genome sequence of tombunodavirus UC1</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr> <span class="cit-vol">3</span>: e<span class="cit-fpage">00655</span>–<span class="cit-lpage">00615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc%26rft.volume%253D3%26rft.spage%253D00655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82 in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.82" data-doi="10.1016/j.str.2004.04.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dokland</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khromykh</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ee</span>  <span class="cit-name-given-names">K-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">West Nile virus core protein; tetramer structure and ribbon formation</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">1163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DDokland%26rft.auinit1%253DT.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D1157%26rft.epage%253D1163%26rft.atitle%253DWest%2BNile%2Bvirus%2Bcore%2Bprotein%253B%2Btetramer%2Bstructure%2Band%2Bribbon%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2004.04.024%26rft_id%253Dinfo%253Apmid%252F15242592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2004.04.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15242592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83 in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.83" data-doi="10.1073/pnas.0305892101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Groesch</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Post</span>  <span class="cit-name-given-names">CB</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Solution structure of dengue virus capsid protein reveals another fold</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3414</span>–<span class="cit-lpage">3419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0305892101%26rft_id%253Dinfo%253Apmid%252F14993605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0MTQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8yNC80NTE3NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84 in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.84" data-doi="10.1016/S0065-3527(05)64008-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossmann</span>  <span class="cit-name-given-names">MG</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Structure and assembly of icosahedral enveloped RNA viruses</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DKuhn%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D64%26rft.spage%253D263%26rft.epage%253D284%26rft.atitle%253DStructure%2Band%2Bassembly%2Bof%2Bicosahedral%2Benveloped%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252805%252964008-0%26rft_id%253Dinfo%253Apmid%252F16139597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(05)64008-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16139597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85 in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.85" data-doi="10.1186/s13062-015-0050-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahola</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">DG</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Sequence analysis reveals a conserved extension in the capping enzyme of the alphavirus supergroup, and a homologous domain in nodaviruses</span>. <abbr class="cit-jnl-abbrev">Biol Direct</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BDirect%26rft.volume%253D10%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13062-015-0050-0%26rft_id%253Dinfo%253Apmid%252F25886938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13062-015-0050-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886938&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86 in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">Computer-assisted identification of a putative methyltransferase domain in NS5 protein of flaviviruses and λ2 protein of reovirus</span>. <abbr class="cit-jnl-abbrev">J Gen Virol 74 (Pt</abbr> <span class="cit-vol">4</span>): <span class="cit-fpage">733</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%2B74%2B%2528Pt%26rft.volume%253D4%26rft.spage%253D733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87 in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.87" data-doi="10.1007/s11515-010-0660-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">P-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Flavivirus RNA cap methyltransferase: structure, function, and inhibition</span>. <abbr class="cit-jnl-abbrev">Front Biol (Beijing</abbr>) <span class="cit-vol">5</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BBiol%2B%2528Beijing%26rft.volume%253D5%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11515-010-0660-y%26rft_id%253Dinfo%253Apmid%252F21927615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11515-010-0660-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21927615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88 in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.88" data-doi="10.1016/j.virusres.2006.02.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kreuze</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valkonen</span>  <span class="cit-name-given-names">JPT</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Comparative and functional genomics of closteroviruses</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DDolja%26rft.auinit1%253DV.%2BV.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D51%26rft.atitle%253DComparative%2Band%2Bfunctional%2Bgenomics%2Bof%2Bclosteroviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.02.002%26rft_id%253Dinfo%253Apmid%252F16529837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16529837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.89"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atsumi</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iwadate</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">K-I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akasaka</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishihara</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamaoka</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishiguchi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sekine</span>  <span class="cit-name-given-names">K-T</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Gentian Kobu-sho-associated virus: a tentative, novel double-stranded RNA virus that is relevant to gentian Kobu-sho syndrome</span>. <abbr class="cit-jnl-abbrev">J Gen Plant Pathol</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BPlant%2BPathol%26rft.volume%253D79%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90 in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.90" data-doi="10.1099/jgv.0.000414"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sela</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casteel</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">H-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bekal</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Girke</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghanim</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaloshian</span>  <span class="cit-name-given-names">I</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">A novel virus from <em>Macrosiphum euphorbiae</em> with similarities to members of the family Flaviviridae</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D97%26rft.spage%253D1261%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.000414%26rft_id%253Dinfo%253Apmid%252F26822322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/jgv.0.000414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26822322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91 in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mata</span>  <span class="cit-name-given-names">CP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luque</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gómez-Blanco</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrascosa</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trus</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castón</span>  <span class="cit-name-given-names">JR</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Acquisition of functions on the outer capsid surface during evolution of double-stranded RNA fungal viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>: e<span class="cit-fpage">1006755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253D1006755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92 in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.92" data-doi="10.1016/B978-0-12-394315-6.00004-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castón</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luque</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gómez-Blanco</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Chrysovirus structure: repeated helical core as evidence of gene duplication</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D86%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-394315-6.00004-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-394315-6.00004-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93 in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.93" data-doi="10.1073/pnas.0812071106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">WF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinkovits</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Havens</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">YJ</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Atomic structure reveals the unique capsid organization of a dsRNA virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">4225</span>–<span class="cit-lpage">4230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0812071106%26rft_id%253Dinfo%253Apmid%252F19246376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzExLzQyMjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMC8yNC80NTE3NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94 in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.94" data-doi="10.1146/annurev-biochem-060910-095130"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abrescia</span>  <span class="cit-name-given-names">NGA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bamford</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Structure unifies the viral universe</span>. <abbr class="cit-jnl-abbrev">Annu Rev Biochem</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DAbrescia%26rft.auinit1%253DN.%2BG.%26rft.volume%253D81%26rft.spage%253D795%26rft.epage%253D822%26rft.atitle%253DStructure%2Bunifies%2Bthe%2Bviral%2Buniverse.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-060910-095130%26rft_id%253Dinfo%253Apmid%252F22482909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-060910-095130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22482909&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305765500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95 in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.95" data-doi="10.1093/nar/gkt713"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Omari</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kainov</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poranen</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bamford</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuma</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mancini</span>  <span class="cit-name-given-names">EJ</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Tracking in atomic detail the functional specializations in viral RecA helicases that occur during evolution</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">9396</span>–<span class="cit-lpage">9410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt713%26rft_id%253Dinfo%253Apmid%252F23939620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23939620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96 in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.96" data-doi="10.1038/nsmb1225"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Bluetongue virus VP4 is an RNA-capping assembly line</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSutton%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D449%26rft.epage%253D451%26rft.atitle%253DBluetongue%2Bvirus%2BVP4%2Bis%2Ban%2BRNA-capping%2Bassembly%2Bline.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1225%26rft_id%253Dinfo%253Apmid%252F17417654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17417654&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246187400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97 in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">ZH</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A putative ATPase mediates RNA transcription and capping in a dsRNA virus</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>: e<span class="cit-fpage">07901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253D07901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98 in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.98" data-doi="10.1007/s00705-018-3814-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amarasinghe</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ceballos</span>  <span class="cit-name-given-names">NGA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banyard</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Basler</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bavari</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blasdell</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bukreyev</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caì</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calisher</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandran</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dietzgen</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolnik</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domier</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dürrwald</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dye</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Easton</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebihara</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Echevarría</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fooks</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Formenty</span>  <span class="cit-name-given-names">PBH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RAM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freuling</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghedin</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hewson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horie</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyndman</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiāng</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kityo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobinger</span>  <span class="cit-name-given-names">GP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kondō</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurath</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maes</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maisner</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mor</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Taxonomy of the order <em>Mononegavirales</em>: update 2018</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">2283</span>–<span class="cit-lpage">2294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D163%26rft.spage%253D2283%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-018-3814-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-018-3814-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99 in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.99" data-doi="10.1016/j.virusres.2011.09.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kormelink</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodin</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaya</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haenni</span>  <span class="cit-name-given-names">A-L</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Negative-strand’ RNA viruses: the plant-infecting counterparts</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DKormelink%26rft.auinit1%253DR.%26rft.volume%253D162%26rft.issue%253D1-2%26rft.spage%253D184%26rft.epage%253D202%26rft.atitle%253DNegative-strand%2BRNA%2Bviruses%253A%2Bthe%2Bplant-infecting%2Bcounterparts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2011.09.028%26rft_id%253Dinfo%253Apmid%252F21963660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2011.09.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21963660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298124300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100 in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.100" data-doi="10.1007/s11262-016-1379-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Osaki</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nomiyama</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomioka</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Multiple virus infection in a single strain of <em>Fusarium poae</em> shown by deep sequencing</span>. <abbr class="cit-jnl-abbrev">Virus Genes</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BGenes%26rft.volume%253D52%26rft.spage%253D835%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11262-016-1379-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11262-016-1379-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101 in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bacharach</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kembou Tsofack</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zamostiano</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berkowitz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitido</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corvelo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toussaint</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abel Nielsen</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hornig</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Del Pozo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eldar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">WI</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Characterization of a novel orthomyxo-like virus causing mass die-offs of tilapia</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">7</span>: e<span class="cit-fpage">00431</span>–<span class="cit-lpage">00416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D7%26rft.spage%253D00431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102 in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maes</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alkhovsky</span>  <span class="cit-name-given-names">SV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bío</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calisher</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charrel</span>  <span class="cit-name-given-names">RN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">IR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torre</span>  <span class="cit-name-given-names">JCdl</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">PLD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serio</span>  <span class="cit-name-given-names">FD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Digiaro</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druciarek</span>  <span class="cit-name-given-names">TZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebihara</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elbeaino</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gergerich</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">J-PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haenni</span>  <span class="cit-name-given-names">A-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hepojoki</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hetzel</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hồ</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hóng</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vuren</span>  <span class="cit-name-given-names">PJv</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonson</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kipar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kondov</span>  <span class="cit-name-given-names">NO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kormelink</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korzyukov</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laney</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">LeBreton</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lukashevic</span>  <span class="cit-name-given-names">IS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marklewitz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markotter</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Taxonomy of the family</span> <abbr class="cit-jnl-abbrev">Arenaviridae and the order Bunyavirales</abbr>: update 2018. Arch Virol <span class="cit-vol">163</span>: <span class="cit-fpage">2295</span>–<span class="cit-lpage">2310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArenaviridae%2Band%2Bthe%2Border%2BBunyavirales%26rft.volume%253D163%26rft.spage%253D2295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103 in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.103" data-doi="10.3390/v4091515"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mielke-Ehret</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mühlbach</span>  <span class="cit-name-given-names">H-P</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Emaravirus: a novel genus of multipartite, negative strand RNA plant viruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1515</span>–<span class="cit-lpage">1536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DMielke-Ehret%26rft.auinit1%253DN.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1515%26rft.epage%253D1536%26rft.atitle%253DEmaravirus%253A%2Ba%2Bnovel%2Bgenus%2Bof%2Bmultipartite%252C%2Bnegative%2Bstrand%2BRNA%2Bplant%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4091515%26rft_id%253Dinfo%253Apmid%252F23170170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4091515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23170170&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104 in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.104" data-doi="10.1016/j.coviro.2013.06.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Networks of evolutionary interactions underlying the polyphyletic origin of ssDNA viruses</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D3%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2013.06.010%26rft_id%253Dinfo%253Apmid%252F23850154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2013.06.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23850154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105 in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.105" data-doi="10.3390/v10040187"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kazlauskas</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varsani</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Pervasive chimerism in the replication-associated proteins of uncultured single-stranded DNA viruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10040187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v10040187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106 in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.106" data-doi="10.1128/JVI.01622-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iranzo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prangishvili</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Bipartite network analysis of the archaeal virosphere: evolutionary connections between viruses and capsidless mobile elements</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">11043</span>–<span class="cit-lpage">11055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01622-16%26rft_id%253Dinfo%253Apmid%252F27681128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI5MC8yNC8xMTA0MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107 in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.107" data-doi="10.1016/j.virol.2018.01.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krishnamurthy</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Extensive conservation of prokaryotic ribosomal binding sites in known and novel picobirnaviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">516</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D516%26rft.spage%253D108%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2018.01.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2018.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108 in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.108" data-doi="10.1016/j.virol.2018.09.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boros</span>  <span class="cit-name-given-names">Á</span></span>, <span class="cit-auth"><span class="cit-name-surname">Polgár</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pankovics</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenyvesi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engelmann</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phan</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delwart</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Multiple divergent picobirnaviruses with functional prokaryotic Shine-Dalgarno ribosome binding sites present in cloacal sample of a diarrheic chicken</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">525</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D525%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2018.09.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2018.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109 in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.109" data-doi="10.1038/nature07755"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">JA</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">A three-dimensional view of the molecular machinery of RNA interference</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJinek%26rft.auinit1%253DM.%26rft.volume%253D457%26rft.issue%253D7228%26rft.spage%253D405%26rft.epage%253D412%26rft.atitle%253DA%2Bthree-dimensional%2Bview%2Bof%2Bthe%2Bmolecular%2Bmachinery%2Bof%2BRNA%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07755%26rft_id%253Dinfo%253Apmid%252F19158786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19158786&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262519200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110 in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.110" data-doi="10.1016/j.chom.2010.06.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">S-W</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Viral suppressors of RNA-based viral immunity: host targets</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DWu%26rft.auinit1%253DQ.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D15%26rft.atitle%253DViral%2Bsuppressors%2Bof%2BRNA-based%2Bviral%2Bimmunity%253A%2Bhost%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2010.06.009%26rft_id%253Dinfo%253Apmid%252F20638637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2010.06.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20638637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280458600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111 in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.111" data-doi="10.1186/s13062-015-0047-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Plant viruses of the <em>Amalgaviridae</em> family evolved via recombination between viruses with double-stranded and negative-strand RNA genomes</span>. <abbr class="cit-jnl-abbrev">Biol Direct</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BDirect%26rft.volume%253D10%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13062-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25886840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13062-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.112" data-doi="10.1016/j.virusres.2010.09.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tzanetakis</span>  <span class="cit-name-given-names">IE</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Blueberry latent virus: an amalgam of the <em>Partitiviridae and Totiviridae</em></span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DMartin%26rft.auinit1%253DR.%2BR.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D180%26rft.atitle%253DBlueberry%2Blatent%2Bvirus%253A%2Ban%2Bamalgam%2Bof%2Bthe%2BPartitiviridae%2Band%2BTotiviridae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2010.09.020%26rft_id%253Dinfo%253Apmid%252F20888379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2010.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20888379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.113" data-doi="10.1016/j.virusres.2008.11.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valverde</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tzanetakis</span>  <span class="cit-name-given-names">IE</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Southern tomato virus: the link between the families <em>Totiviridae and Partitiviridae</em></span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DSabanadzovic%26rft.auinit1%253DS.%26rft.volume%253D140%26rft.issue%253D1-2%26rft.spage%253D130%26rft.epage%253D137%26rft.atitle%253DSouthern%2Btomato%2Bvirus%253A%2BThe%2Blink%2Bbetween%2Bthe%2Bfamilies%2BTotiviridae%2Band%2BPartitiviridae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2008.11.018%26rft_id%253Dinfo%253Apmid%252F19118586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2008.11.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19118586&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264650600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference 114 in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.114" data-doi="10.1007/s00705-010-0770-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abou Ghanem-Sabanadzovic</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valverde</span>  <span class="cit-name-given-names">RA</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">A novel monopartite dsRNA virus from rhododendron</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DSabanadzovic%26rft.auinit1%253DS.%26rft.volume%253D155%26rft.issue%253D11%26rft.spage%253D1859%26rft.epage%253D1863%26rft.atitle%253DA%2Bnovel%2Bmonopartite%2BdsRNA%2Bvirus%2Bfrom%2Brhododendron.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-010-0770-5%26rft_id%253Dinfo%253Apmid%252F20721591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-010-0770-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20721591&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference 115 in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.115" data-doi="10.1007/BF01313766"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mushegian</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">Cell-to-cell movement of plant viruses. Insights from amino acid sequence comparisons of movement proteins and from analogies with cellular transport systems</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DMushegian%26rft.auinit1%253DA.%2BR.%26rft.volume%253D133%26rft.issue%253D3-4%26rft.spage%253D239%26rft.epage%253D257%26rft.atitle%253DCell-to-cell%2Bmovement%2Bof%2Bplant%2Bviruses.%2BInsights%2Bfrom%2Bamino%2Bacid%2Bsequence%2Bcomparisons%2Bof%2Bmovement%2Bproteins%2Band%2Bfrom%2Banalogies%2Bwith%2Bcellular%2Btransport%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01313766%26rft_id%253Dinfo%253Apmid%252F8257287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01313766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8257287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MJ63300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference 116 in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghabrial</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castón</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">50-plus years of fungal viruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">479–480</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-451740v1.117" data-doi="10.1016/bs.aivir.2017.10.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hillman</span>  <span class="cit-name-given-names">BI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Annisa</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Viruses of plant-interacting fungi</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D100%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2017.10.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2017.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference 118 in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kotta-Loizou</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutts</span>  <span class="cit-name-given-names">RHA</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Studies on the virome of the entomopathogenic fungus <em>Beauveria bassiana</em> reveal novel dsRNA elements and mild hypervirulence</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>: e<span class="cit-fpage">1006183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253D1006183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference 119 in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.119" data-doi="10.1101/371526:371526"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coyle</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elya</span>  <span class="cit-name-given-names">CN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bronski</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">MB</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Entomophthovirus: an insect-derived iflavirus that infects a behavior manipulating fungal pathogen of dipterans</span>. <abbr class="cit-jnl-abbrev">bioRxiv doi</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/371526:371526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%2Bdoi%26rft_id%253Dinfo%253Adoi%252F10.1101%252F371526%253A371526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/371526:371526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference 120 in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waldron</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">GN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obbard</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Metagenomic sequencing suggests a diversity of RNA interference-like responses to viruses across multicellular eukaryotes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">14</span>: e<span class="cit-fpage">1007533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D14%26rft.spage%253D1007533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference 121 in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.121" data-doi="10.1186/s40168-018-0422-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiewsakun</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The genomic underpinnings of eukaryotic virus taxonomy: creating a sequence-based framework for family-level virus classification</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D6%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-018-0422-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-018-0422-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference 122 in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.122" data-doi="10.1093/nar/gkx1094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavanaugh</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karsch-Mizrachi</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostell</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayers</span>  <span class="cit-name-given-names">EW</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">GenBank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D41</span>–<span class="cit-lpage">D47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD41%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference 123 in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.123" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference 124 in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.124" data-doi="10.1093/nar/gku1221"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marchler-Bauer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitsaz</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geer</span>  <span class="cit-name-given-names">LY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geer</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gwadz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurwitz</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanczycki</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchler</span>  <span class="cit-name-given-names">GH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thanki</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamashita</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">SH</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">CDD: NCBI’s conserved domain database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>: D<span class="cit-fpage">222</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D43%26rft.spage%253D222%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference 125 in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.125" data-doi="10.1093/bioinformatics/btq461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">RC</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Search and clustering orders of magnitude faster than BLAST</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2460</span>–<span class="cit-lpage">2461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20709691&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282170000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference 126 in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.126" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">RC</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference 127 in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.127" data-doi="10.1093/bioinformatics/bti125"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Protein homology detection by HMM-HMM comparison</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti125%26rft_id%253Dinfo%253Apmid%252F15531603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15531603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227977800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference 128 in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michener</span>  <span class="cit-name-given-names">CD</span></span>. <span class="cit-pub-date">1958</span>. <span class="cit-article-title">A statistical method for evaluating systematic relationships</span>. <abbr class="cit-jnl-abbrev">Univ Kansas Sci Bull</abbr> <span class="cit-vol">XXXVIII</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUniv%2BKansas%2BSci%2BBull%26rft.volume%253DXXXVIII%26rft.spage%253D1409%26rft.atitle%253DA%2BSTATISTICAL%2BMETHOD%2BFOR%2BEVALUATING%2BSYSTEMATIC%2BRELATIONSHIPS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference 129 in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.129" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dehal</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">AP</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">FastTree 2-approximately maximum-likelihood trees for large alignments</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: e<span class="cit-fpage">9490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253D9490%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference 130 in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.130" data-doi="10.1038/s41586-018-0043-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lemoine</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domelevo Entfellner</span>  <span class="cit-name-given-names">J-B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Correia</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davila Felipe</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Oliveira</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Renewing Felsenstein’s phylogenetic bootstrap in the era of big data</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">556</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0043-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0043-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference 131 in text" id="ref-131">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.131" data-doi="10.1093/nar/gkw1081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mirdita</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">von den Driesch</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galiez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinegger</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Uniclust databases of clustered and deeply annotated protein sequences and alignments</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D170</span>–<span class="cit-lpage">D176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1081%26rft_id%253Dinfo%253Apmid%252F27899574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899574&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-132-1" title="View reference 132 in text" id="ref-132">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.132" data-doi="10.1371/journal.pcbi.1002195"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Accelerated profile HMM searches</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">7</span>: e<span class="cit-fpage">1002195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D7%26rft.spage%253D1002195%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-133-1" title="View reference 133 in text" id="ref-133">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.133" data-doi="10.1093/bioinformatics/bth444"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frickey</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lupas</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">CLANS: a Java application for visualizing protein families based on pairwise similarity</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3702</span>–<span class="cit-lpage">3704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth444%26rft_id%253Dinfo%253Apmid%252F15284097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15284097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225786600061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-134-1" title="View reference 134 in text" id="ref-134">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.134" data-doi="10.1038/srep00336"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lancichinetti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Consensus clustering in complex networks</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DSci%2BRep%26rft.aulast%253DLancichinetti%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.spage%253D336%26rft.epage%253D336%26rft.atitle%253DConsensus%2Bclustering%2Bin%2Bcomplex%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep00336%26rft_id%253Dinfo%253Apmid%252F22468223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep00336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22468223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F24%2F451740.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-135-1" title="View reference 135 in text" id="ref-135">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.135"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hric</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Community detection in networks: A user guide</span>. <abbr class="cit-jnl-abbrev">Physics Rep Rev Sect Physics Lett</abbr> <span class="cit-vol">659</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysics%2BRep%2BRev%2BSect%2BPhysics%2BLett%26rft.volume%253D659%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-136-1" title="View reference 136 in text" id="ref-136">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.136" data-doi="10.1073/pnas.0706851105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosvall</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">CT</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Maps of random walks on complex networks reveal community structure</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1118</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0706851105%26rft_id%253Dinfo%253Apmid%252F18216267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzQvMTExOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEwLzI0LzQ1MTc0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-137-1" title="View reference 137 in text" id="ref-137">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.137"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lancichinetti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radicchi</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramasco</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Finding statistically significant communities in networks</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: e<span class="cit-fpage">18961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D6%26rft.spage%253D18961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-138-1" title="View reference 138 in text" id="ref-138">↵</a><div class="cit ref-cit ref-journal" id="cit-451740v1.138" data-doi="10.1093/gbe/evx189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petitjean</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makarova</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">YI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Extreme deviations from expected evolutionary rates in archaeal protein families</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2791</span>–<span class="cit-lpage">2811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft.volume%253D9%26rft.spage%253D2791%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/451740v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F451740v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/450841v1" title="Modelling cancer progression using Mutual Hazard Networks" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 24, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/451740v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/451740v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/451740v1" class="active" data-icon-position="" data-hide-link-title="0">Origins and Evolution of the Global RNA Virome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/135497" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6IX8mnUNNQIwDYUr066PP-dmR4jUMjMv2xw4xkQOtK4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="135497" id="node135497--2" data-pisa="biorxiv;451740v1" data-pisa-master="biorxiv;451740" data-seqnum="135497" data-apath="/biorxiv/early/2018/10/24/451740.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Origins and Evolution of the Global RNA Virome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yuri I.</span> <span class="nlm-surname">Wolf</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Darius</span> <span class="nlm-surname">Kazlauskas</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jaime</span> <span class="nlm-surname">Iranzo</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adriana</span> <span class="nlm-surname">LucÍa-Sanz</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jens H.</span> <span class="nlm-surname">Kuhn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mart</span> <span class="nlm-surname">Krupovic</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Valerian V.</span> <span class="nlm-surname">Dolja</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Eugene V.</span> <span class="nlm-surname">Koonin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">451740; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/451740 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/451740v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;title=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;title=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;text=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;title=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;t=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;title=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;title=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="135497" id="node135497--3" data-pisa="biorxiv;451740v1" data-pisa-master="biorxiv;451740" data-seqnum="135497" data-apath="/biorxiv/early/2018/10/24/451740.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Origins and Evolution of the Global RNA Virome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yuri I.</span> <span class="nlm-surname">Wolf</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Darius</span> <span class="nlm-surname">Kazlauskas</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jaime</span> <span class="nlm-surname">Iranzo</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Adriana</span> <span class="nlm-surname">LucÍa-Sanz</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jens H.</span> <span class="nlm-surname">Kuhn</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mart</span> <span class="nlm-surname">Krupovic</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Valerian V.</span> <span class="nlm-surname">Dolja</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Eugene V.</span> <span class="nlm-surname">Koonin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">451740; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/451740 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Wolf%2C+Yuri+I.&amp;rft.au=Kazlauskas%2C+Darius&amp;rft.au=Iranzo%2C+Jaime&amp;rft.au=Luc%C3%8Da-Sanz%2C+Adriana&amp;rft.au=Kuhn%2C+Jens+H.&amp;rft.au=Krupovic%2C+Mart&amp;rft.au=Dolja%2C+Valerian+V.&amp;rft.au=Koonin%2C+Eugene+V."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/135497/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/135497/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/135497/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/135497/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/135497/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/135497/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/135497/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/135497/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/135497/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/135497/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/135497/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/135497/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;count=horizontal&amp;via=&amp;text=Origins%20and%20Evolution%20of%20the%20Global%20RNA%20Virome&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/451740v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/451740v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/451740v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
