<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7055873">7055873</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0229601">10.1371/journal.pone.0229601</a></span><span class="publisher-id" title="publisher-id">PONE-D-19-24652</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Influenza</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span>: <span class="subject" title="subject">Influenza Viruses</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span>: <span class="subject" title="subject">Influenza Viruses</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span>: <span class="subject" title="subject">Influenza Viruses</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span>: <span class="subject" title="subject">Influenza Viruses</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Asia</span>: <span class="subject" title="subject">Myanmar</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Disease Control</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vaccines</span>: <span class="subject" title="subject">Viral Vaccines</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Vaccines</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Disease Control</span>: <span class="subject" title="subject">Vaccines</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Mutation</span>: <span class="subject" title="subject">Substitution Mutation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemical Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Amino Acids</span>: <span class="subject" title="subject">Amino Acid Substitution</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Amino Acids</span>: <span class="subject" title="subject">Amino Acid Substitution</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Proteins</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Amino Acids</span>: <span class="subject" title="subject">Amino Acid Substitution</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Control</span>: <span class="subject" title="subject">Antimicrobial Resistance</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pharmacology</span>: <span class="subject" title="subject">Antimicrobial Resistance</span></div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Epidemic of influenza A(H1N1)pdm09 analyzed by full genome sequences and the first case of oseltamivir-resistant strain in Myanmar 2017</div> <div class="alt-title" title="alt-title">
Epidemic of influenza A(H1N1)pdm09 in Myanmar</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3939-4185</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyaw Win</span><span class="given-names" tagx="given-names" title="given-names">Su Mon</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">Reiko</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Win</span><span class="given-names" tagx="given-names" title="given-names">Nay Chi</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lasham</span><span class="given-names" tagx="given-names" title="given-names">Di Ja</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyaw</span><span class="given-names" tagx="given-names" title="given-names">Yadanar</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Nay</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thein</span><span class="given-names" tagx="given-names" title="given-names">Khin Nyo</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chon</span><span class="given-names" tagx="given-names" title="given-names">Irina</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">Takashi</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><a href="#aff006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thein</span><span class="given-names" tagx="given-names" title="given-names">Win</span></span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyaw</span><span class="given-names" tagx="given-names" title="given-names">Latt Latt</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tin</span><span class="given-names" tagx="given-names" title="given-names">Ommar Swe</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saitoh</span><span class="given-names" tagx="given-names" title="given-names">Akihiko</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff008"><sup>8</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">Tsutomu</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff009"><sup>9</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirokawa</span><span class="given-names" tagx="given-names" title="given-names">Chika</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff009"><sup>9</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchida</span><span class="given-names" tagx="given-names" title="given-names">Yuko</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff010"><sup>10</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">Takehiko</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff010"><sup>10</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">Shinji</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff011"><sup>11</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">Takato</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff011"><sup>11</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamata</span><span class="given-names" tagx="given-names" title="given-names">Kazuhiro</span></span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff012"><sup>12</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osada</span><span class="given-names" tagx="given-names" title="given-names">Hidekazu</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff012"><sup>12</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7616-4680</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dapat</span><span class="given-names" tagx="given-names" title="given-names">Clyde</span></span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff013"><sup>13</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">Hisami</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff012"><sup>12</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tin</span><span class="given-names" tagx="given-names" title="given-names">Htay Htay</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff007"><sup>7</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Infectious Diseases Research Center of Niigata University in Myanmar (IDRC), Yangon, Yangon Region, Myanmar</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Division of International Health, Graduate School of Medical and Dental Sciences, Niigata University, Niigata, Niigata, Japan</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Respiratory Medicine Department, Thingangyun Sanpya General Hospital, Yangon, Yangon Region, Myanmar</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Clinical Laboratory, Microbiology Section, Pyinmana General Hospital, Pyinmana Township, Nay Pyi Taw, Myanmar</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Pediatric Ward 1, Yankin Children Hospital, Yangon, Yangon Region, Myanmar</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Department of Microbiology, Infectious diseases and Immunology, Iwate Medical University, Morioka, Iwate, Japan</span></span><span class="citation_author_institution" id="aff007">[7], <span class="addr-line" title="addr-line">National Health Laboratory, Department of Medical Services, Ministry of Health and Sports, Yangon, Yangon Region, Myanmar</span></span><span class="citation_author_institution" id="aff008">[8], <span class="addr-line" title="addr-line">Department of Pediatrics, Graduate School of Medical and Dental Sciences, Niigata University, Niigata, Niigata, Japan</span></span><span class="citation_author_institution" id="aff009">[9], <span class="addr-line" title="addr-line">Division of Virology, Niigata Prefectural Institute of Public Health and Environmental Sciences, Niigata, Niigata, Japan</span></span><span class="citation_author_institution" id="aff010">[10], <span class="addr-line" title="addr-line">Division of Transboundary Animal Disease, National Agriculture and Food Research Organization (NARO), Tsukuba, Ibaraki, Japan</span></span><span class="citation_author_institution" id="aff011">[11], <span class="addr-line" title="addr-line">Laboratory of Influenza Virus Surveillance, Influenza Research Center, National Institute of Infectious Diseases, Sinjuku-ku, Tokyo, Japan</span></span><span class="citation_author_institution" id="aff012">[12], <span class="addr-line" title="addr-line">Institute of Medicine and Dentistry, Niigata University, Niigata, Japan</span></span><span class="citation_author_institution" id="aff013">[13], <span class="addr-line" title="addr-line">Department of Virology, Tohoku University Graduate School of Medicine, Sendai, Miyagi, Japan</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Baochuan</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Defense Threat Reduction Agency, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">sumonkyawwin@gmail.com</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0229601</span><span class="history" title="history"><span class="received" title="received">received: 2019-9-2</span><span class="accepted" title="accepted">accepted: 2020-2-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Kyaw Win et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0229601.pdf" title="self-uri">pone.0229601.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>A community outbreak of human influenza A(H1N1)pdm09 virus strains was observed in Myanmar in 2017. We investigated the circulation patterns, antigenicity, and drug resistance of 2017 influenza A(H1N1)pdm09 viruses from Myanmar and characterized the full genome of influenza virus strains in Myanmar from in-patients and out-patients to assess the pathogenicity of the viruses. Nasopharyngeal swabs were collected from out-patients and in-patients with acute respiratory tract infections in Yangon and Pyinmana City in Myanmar during January-December 2017. A total of 215 out-patients and 18 in-patients infected with A(H1N1)pdm09 were detected by virus isolation and real-time RT-PCR. Among the positive patients, 90.6% were less than 14 years old. Hemagglutination inhibition (HI) antibody titers against A(H1N1)pdm09 viruses in Myanmar were similar to the recommended Japanese influenza vaccine strain for 2017–2018 seasons (A/Singapore/GP1908/2015) and WHO recommended 2017 southern hemisphere vaccine component (A/Michigan/45/2015). Phylogenetic analysis of the hemagglutinin sequence showed that the Myanmar strains belonged to the genetic subclade 6B.1, possessing mutations of S162N and S164T at potential antigenic sites. However, the amino acid mutation at position 222, which may enhance the severity of disease and mortality, was not found. One case with no prior history of oseltamivir treatment possessed H275Y mutated virus in neuraminidase (NA), which confers resistance to oseltamivir and peramivir with elevated IC<sub>50</sub> values. The full genome sequence of Myanmar strains showed no difference between samples from in-patients and out-patients, suggesting no additional viral mutations associated with patient severity. Several amino acid changes were observed in PB2, PB1, and M2 of Myanmar strains when compared to the vaccine strain and other Asian strains. However, no mutations associated with pathogenicity were found in the Myanmar strains, suggesting that viral factors cannot explain the underlying reasons of the massive outbreak in Myanmar. This study reported the first detection of an oseltamivir-resistant influenza virus in Myanmar, highlighting the importance of continuous antiviral monitoring and genetic characterization of the influenza virus in Myanmar.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
the International Development Strategy Program for Infectious Diseases Research (J-GRID) of AMED (Japan Medical Research and Development Organization) (15fm0108009h0001-19fm0108009h0005)</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the manuscript and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the manuscript and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Influenza outbreaks occur mainly during winter in temperate areas of the northern and southern hemispheres; however, influenza outbreaks in tropical areas around the equator may occur at any time of the year [<a href="#pone.0229601.ref001">1</a>]. Influenza is not only responsible for local epidemics annually but also global pandemics. Since the 20<sup>th</sup> century, there have been four flu pandemics: 1918 H1N1 Spanish flu [<a href="#pone.0229601.ref002">2</a>], 1957 H2N2 Asian flu, 1968 H3N2 Hong Kong flu, and 2009 H1N1 swine flu. The 1918 H1N1 Spanish flu, which killed more than 50 million people worldwide, was the most severe pandemic [<a href="#pone.0229601.ref003">3</a>].</p> <p>Influenza virus is a highly infectious respiratory pathogen manifesting a significant threat to global public health [<a href="#pone.0229601.ref001">1</a>]. Influenza A(H1N1)pdm09 virus, which emerged in 2009 and caused a global influenza pandemic, is now a seasonal influenza virus that co-circulates with another seasonal influenza (H3N2) and influenza B viruses. Worldwide circulation of A(H1N1)pdm09 virus has raised concerns about genotypic diversity that enhances virus transmissibility and pathogenicity and affects vaccine efficacy [<a href="#pone.0229601.ref004">4</a>].</p> <p>In 2008, Myanmar established a National Influenza Centre (NIC) at the National Health Laboratory in Yangon, which has been sharing data with the WHO by sending influenza isolates twice annually to the WHO collaborating center for new vaccine development, influenza genotyping, antiviral susceptibility, monitoring Influenza-like illness (ILI), and Severe Acute Respiratory Infection (SARI) surveillance. However, influenza surveillance efforts in Myanmar are limited. In 2017, the ILI and SARI surveillance system in Myanmar was strengthened by placing the sentinel sites approach, which was reinforced by the Ministry of Health and Sports of Myanmar (MoHS) with the support of WHO [<a href="#pone.0229601.ref005">5</a>].</p> <p>In Myanmar, the first pandemic influenza A(H1N1) 2009 case was detected in June 2009, with no reported fatal cases. Since then, it has been co-circulating with other seasonal influenza viruses in the country. Previously, we reported the epidemiology and genetic characterization of influenza virus A and B circulating in Myanmar [<a href="#pone.0229601.ref006">6</a>, <a href="#pone.0229601.ref007">7</a>] and characterized the drug-susceptibility of seasonal and pandemic influenza A(H1N1) viruses in Myanmar in 2008 and 2009 [<a href="#pone.0229601.ref008">8</a>]. Only 16 sporadic cases of A(H1N1)pdm09 were detected in 2009, and genotyping of these viruses showed no mutations in neuraminidase (NA), which indicated susceptibility to oseltamivir [<a href="#pone.0229601.ref008">8</a>].</p> <p>In July 2017, severe cases of influenza A(H1N1)pdm09 were detected in Myanmar, and the MoHS declared a high alert to mitigate the influenza outbreaks in the country. The overall numbers of severe pneumonia cases and fatal cases reached 1198 and 38, respectively [<a href="#pone.0229601.ref009">9</a>]. This 2017 outbreak had a significant impact on the society of Myanmar. Eventually, the outbreak was successfully controlled in October 2017 by implementing and strengthening active influenza surveillance through early and timely detection of cases and the distribution of clinical management guidelines at hospitals (<a href="#pone.0229601.s004">S1 File</a>).</p> <p>In this study, we investigated the circulation patterns, distribution of influenza subtypes, antigenic and genetic characterization of influenza A(H1N1)pdm09 virus strains and assessed their susceptibility to neuraminidase inhibitors (NAIs) in out-patients and in-patients in Myanmar during the 2017 influenza season.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="studypopulation" title="sec"> <div class="title" tagx="title" title="title">
Study population</div> <p>Out-patients with influenza-like illness symptoms, who visited two surveillance site hospitals, Thingangyun Sanpya General Hospital in Yangon and Pyinmana 200 Bedded General Hospital in Pyinmana, and in-patients with acute respiratory infections, who were admitted to Yankin Children Hospital (YKCH) in Yangon, were enrolled in the study between January and December 2017. The inclusion criteria for out-patients were sudden onset of fever (&amp;gt; 37.8 °C) with more than two of the following symptoms: cough, rhinorrhea, myalgia, arthralgia, and diarrhea; the exclusion criteria were suspected cases of digestive tract infections and chronic respiratory infections such as tuberculosis. The inclusion criteria for in-patients were history or measured fever of ≥ 38 °C, cough, onset within the last 10 days, difficulty in breathing with chest in-drawing for children under 5 years of age, and requirement for hospitalization, while the exclusion criteria were TB infection, chronic respiratory diseases, patients with tracheostomy, immunosuppressive status (e.g., HIV infection, chemotherapy), cystic fibrosis, and cancer. Written informed consents were obtained from patients, and clinical information such as name, age, sex, address, date of symptom onset, date of clinic visit, flu vaccination history, antiviral drug medication history, and symptoms were recorded in registration sheets before specimen collection.</p> </div> <div class="samplecollection" title="sec"> <div class="title" tagx="title" title="title">
Sample collection</div> <p>Two nasopharyngeal swabs were taken from each suspected patient. One swab was used for testing with a rapid test kit (Quick Navi-Flu+RSV, Denka Seiken Co. Ltd., Tokyo, Japan) to screen influenza virus at the sample collection sites. The other swab was stored in a viral transport media for further analysis at the National Health Laboratory (NHL), Myanmar, and Division of International Health, Niigata University, Japan.</p> </div> <div class="virusisolation" title="sec"> <div class="title" tagx="title" title="title">
Virus isolation</div> <p>The positive samples for influenza A and B, detected by real-time RT-PCR analysis, were inoculated in confluent Madin-Darby canine kidney (MDCK) cells to isolate the influenza viruses. The cell culture tubes were then incubated at 34 °C with 5% CO<sub>2</sub> and observed daily for 5 days until the second passage to detect the specific cytopathic effect.</p> </div> <div class="rnaextractionandreal-timert-pcr" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction and real-time RT-PCR</div> <p>Viral RNA was extracted from 140 μL of nasopharyngeal swab supernatant of the original clinical sample for initial typing and from viral isolate for subtyping/lineage detection of influenza A or B using QIAamp Viral RNA mini Kit (QIAGEN, Hilden, Germany) following manufacturer’s instructions. Real-time PCR using QuantiTect Probe RT-PCR kit (QIAGEN) was performed according to the protocols provided by the National Institute of Infectious Diseases in Tokyo, Japan [<a href="#pone.0229601.ref010">10</a>] to screen influenza viruses from original clinical specimens for initial typing as Type A or B. Subtyping PCR for A(H1N1)pdm09 or A(H3N2) and differentiation for B/Victoria or B/Yamagata lineages was performed at NHL for the virus isolates using specific primers and probes, respectively [<a href="#pone.0229601.ref010">10</a>].</p> </div> <div class="antigeniccharacterization" title="sec"> <div class="title" tagx="title" title="title">
Antigenic characterization</div> <p>The antigenicity of selected A(H1N1)pdm09 isolates was characterized by hemagglutination inhibition (HI) assay following the WHO manual using guinea pig red blood cells and commercially available vaccine strain antisera for 2017–2018 seasons in Japan (A/Singapore/GP1908/2015, A/Michigan/45/2015-like strain) at the Department of Virology, Niigata Prefectural Institute of Public Health and Environmental Sciences, Niigata, Japan [<a href="#pone.0229601.ref011">11</a>].</p> </div> <div class="cyclingprobereal-timepcr" title="sec"> <div class="title" tagx="title" title="title">
Cycling probe real-time PCR</div> <p>RNA was extracted from A(H1N1)pdm09 positive isolates using Extragen II kits (Toso, Tokyo, Japan). Complementary DNA was synthesized using uni12 primer for generic influenza A amplification, as described previously [<a href="#pone.0229601.ref012">12</a>]. Cycling probe real-time PCR for the mutation on NA gene that confers resistance to oseltamivir was performed on the isolates using fluorescent-labeled chimeric RNA-DNA probes, RNase H, and the commercially available cycling probe real-time PCR kit, CycleavePCRCore kit (TaKaRa Bio Inc., Ohtsu, Japan), at the Division of International Health, Niigata University, Japan. These cycling probes were labeled with either 6-carboxyfluorescein (FAM) or 6-carboxy-X-rhodamine (ROX), and the PCR reaction was prepared according to the manufacturer’s instructions. This assay utilizes single nucleotide polymorphisms (SNP) for detecting oseltamivir-sensitive (H275) and oseltamivir-resistant (H275Y) viruses based on the reaction curves of FAM (H275) or ROX (H275Y) [<a href="#pone.0229601.ref012">12</a>].</p> </div> <div class="neuraminidaseinhibitorssusceptibilityassay" title="sec"> <div class="title" tagx="title" title="title">
Neuraminidase inhibitors susceptibility assay</div> <p>The selected influenza A(H1N1)pdm09 viruses were tested using fluorescence-based inhibition assay (NA inhibition assay) against four kinds of neuraminidase inhibitors, oseltamivir (Sequoia Research Products Ltd., Pangbourne, UK), zanamivir (Sequoia Research Products Ltd.), peramivir (Shionogi &amp;amp; Co., Ltd., Osaka, Japan), and laninamivir (Daiichi Sankyo Co., Ltd., Tokyo, Japan). Susceptibility of viruses from culture supernatants of infected MDCK cells were used to test the inhibitory effect of each NA inhibitor (NAI). Prior to NAI assay, NAIs were diluted to a final concentration range of 0.005–1250 nM. Each influenza virus isolate was diluted to a final concentration of 25,000 fluorescence units to obtain a dilution in the linear range of the NA activity curve. Twenty-five microliters of each diluted NAI was added to each well of a microtiter plate, followed by the addition of 25 μL of each virus dilution [<a href="#pone.0229601.ref013">13</a>]. The plates were incubated at 37 °C for 30 min. Fifty microliters of 2′-(4-Methylumbelliferyl)-α-D-N-acetylneuraminic acid (MUNANA) substrate was added to each well at a final concentration of 25 μM, and then, the plates were incubated at 37 °C for 60 min. The reaction was finally stopped by adding 260 μL of 200 mM sodium carbonate to each well. The fluorescence of the released 4-methyl umbelliferone (4-MUNANA) was measured in a microplate reader TriStar LB942 (Berthold Japan K.K., Tokyo, Japan) using an excitation wavelength of 360 nm and an emission wavelength of 460 nm. Inhibitory effect was expressed as the IC<sub>50</sub> calculated using XLfit software (IDBS, Surrey, UK). The strain was assessed as resistant if the IC<sub>50</sub> value against one of four drugs is 100-fold elevated for influenza A [<a href="#pone.0229601.ref014">14</a>]. Drug susceptibility assay was performed at Niigata University (Niigata, Japan).</p> </div> <div class="fullgenomesequencingusingnext-generationsequencing" title="sec"> <div class="title" tagx="title" title="title">
Full genome sequencing using next-generation sequencing</div> <p>The eight viral RNA segments for the selected Myanmar A(H1N1)pdm09 viruses were sequenced using next-generation sequencing to obtain the whole viral genome. Briefly, a cDNA library was prepared using random hexamers and NEBNext UltraTM RNA Library Prep kit (NEB) according to the manufacturer’s instructions, and sequenced using a MiSeq second-generation sequencer (Illumina) with Reagent Kit v2. The genomic sequences of the isolates were determined using FluGAS software (version 1.0.0, World Fusion, Tokyo, Japan), which mapped the output pair-end reads to reference sequences of each gene segment selected from the Influenza Virus Database in the National Center for Biotechnology Information Search database (NCBI) by the FluGAS algorithm. A consensus sequence was constructed when coverage was higher than three at each site and adopted a single nucleotide in at least 51% or more coverage, whereas mixed-base codes were adopted when multiple bases shared more than 15% out of the total coverage at the site. In this study, all of the consensus sequences subjected for phylogenetic analysis had more than 10 coverage at any site. The variant frequency at position 275 of the amino acid sequence in NA, which confers oseltamivir-resistance, was calculated using Genome Workbench (QIAGEN, Duesseldorf, Germany) software. Next-generation sequencing was performed at the National Institute of Animal Health, Tsukuba city, Japan.</p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p>In addition to Myanmar strains, A(H1N1)pdm09 collected from various prefectures (Okinawa, Nagasaki, Gunma, Shizuoka, Kyoto, Nara, and Hokkaido prefectures) in Japan as part of the influenza investigation conducted by Niigata University during the 2017–2018 season was also sequenced by Sanger Method [<a href="#pone.0229601.ref015">15</a>]. All the sequences used in this study were registered with the Global Initiative on Sharing All Influenza Data (GISAID) (<a href="#pone.0229601.s001">S1</a> and <a href="#pone.0229601.s002">S2</a> Tables). Reference strains such as the influenza vaccine strains for the 2017–2018 Japanese vaccine, A/Singapore/GP1908/2015, and the 2017 southern hemisphere vaccine, A/Michigan/45/2015, as well as strains collected in India during May and August of 2017 (GISAID Isolate ID: EPI_ISL_281596, EPI_ISL_281597, EPI_ISL_281600, EPI_ISL_281601, EPI_ISL_281602, EPI_ISL_281603, EPI_ISL_281604, and EPI_ISL_282867) were downloaded from GISAID. India is specifically included in this analysis because A(H1N1)pdm09 outbreaks due to Clade 6B and 6B.1 viruses were repeatedly reported in India during 2015–2017, thus suggesting the possibility that the Myanmar strains may have originated in India [<a href="#pone.0229601.ref016">16</a>–<a href="#pone.0229601.ref020">20</a>]. Phylogenetic analysis was performed using the Hasegawa-Kishino-Yano model with discrete Gamma distribution (HKY+G) and the General Time Reversible model with discrete Gamma distribution incorporating Invariant sites (GTR + G + I) as the best-fit nucleotide substitution models for the HA and NA datasets, respectively, as implemented in the MEGA 6.0 software (Molecular Evolutionary Genetics Analysis) [<a href="#pone.0229601.ref021">21</a>]. Best fitting trees for the HA and NA genes were constructed by the maximum likelihood method and bootstrap analysis of 1,000 replicates. Deduced amino acid sequences were analyzed, and amino acid changes were compared with A/California/07/2009 (H1N1pdm09) (GISAID Isolate ID: EPI_ISL_227813).</p> <p>The six internal genes of A(H1N1)pdm09 from Myanmar and India were aligned by each segment using BioEdit software. The FASTA files of the multiple alignment of each segment were uploaded to the FluSurver (<a href="https://flusurver.bii.a-star.edu.sg/">https://flusurver.bii.a-star.edu.sg/</a>) to find any amino acid mutations compared to the vaccine strain, A/Michigan/45/2015, a WHO-recommended A(H1N1)pdm09 strain for the Southern hemisphere in 2017 [<a href="#pone.0229601.ref022">22</a>], and to search for mutations that have significant functional or pathological implications.</p> </div> <div class="ethicalstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethical statement</div> <p>This study was approved by the Niigata University Ethical Committee (2533) and the Ethical Review Committee in the Department of Medical Research, Ministry of Health and Sports, Myanmar (016516). Written consent was obtained from all study participants before sample collection.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="numberofsamplesanddetectionofinfluenzavirustypesandsubtypes" title="sec"> <div class="title" tagx="title" title="title">
Number of samples and detection of influenza virus types and subtypes</div> <p>From January to December of 2017, we collected a total of 328 nasopharyngeal swab samples (267 samples from Yangon study site and 61 samples from Pyinmana study site of Myanmar) from out-patients presenting influenza-like illness symptoms and 288 nasopharyngeal swab samples (from Yangon study site of Myanmar) from in-patients presenting acute respiratory tract infections, and tested the samples for the presence of influenza virus. Rapid test kit assay indicated that a total of 280 samples (45.5%) were positive for the influenza virus. All 616 nasopharyngeal swab samples were also screened by real-time PCR assay. Influenza A(H1N1)pdm09 viruses represented the majority of circulating influenza viruses during 2017 (233, 37.8%) (<a href="#pone.0229601.t001">Table 1</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t001">10.1371/journal.pone.0229601.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Number of samples and fraction of influenza virus-positive tests based on real-time PCR analysis in Yangon and Pyinmana study sites, Myanmar, 2017.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" style="background-color:#D9D9D9" rowspan="1" colspan="1" /> <th align="center" colspan="4" style="background-color:#D9D9D9" rowspan="1">Out-patients</th> <th align="center" colspan="2" style="background-color:#D9D9D9" rowspan="1">In-patients</th> <th align="center" colspan="2" rowspan="2" style="background-color:#D9D9D9">Total(N = 616)</th> </tr> <tr> <th align="left" style="background-color:#D9D9D9" rowspan="1" colspan="1" /> <th align="center" colspan="2" style="background-color:#D9D9D9" rowspan="1">Yangon (N = 267)</th> <th align="center" colspan="2" style="background-color:#D9D9D9" rowspan="1">Pyinmana (N = 61)</th> <th align="center" colspan="2" style="background-color:#D9D9D9" rowspan="1">Yangon (N = 288)</th> </tr> <tr> <th align="left" rowspan="1" colspan="1" /> <th align="left" rowspan="1" colspan="1">N</th> <th align="left" rowspan="1" colspan="1">(%)<a href="#t001fn002">*</a></th> <th align="left" rowspan="1" colspan="1">N</th> <th align="left" rowspan="1" colspan="1">(%)<a href="#t001fn002">*</a></th> <th align="left" rowspan="1" colspan="1">N</th> <th align="left" rowspan="1" colspan="1">(%)<a href="#t001fn002">*</a></th> <th align="left" rowspan="1" colspan="1">N</th> <th align="left" rowspan="1" colspan="1">(%)<a href="#t001fn002">*</a></th> </tr> </thead> <tbody> <tr> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1"><b>Rapid Test (+)</b></td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">215</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(80.5)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">46</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(75.4)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">19</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(6.6)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">280</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(45.5)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> Flu A</td> <td align="left" rowspan="1" colspan="1">206</td> <td align="left" rowspan="1" colspan="1">(77.1)</td> <td align="left" rowspan="1" colspan="1">46</td> <td align="left" rowspan="1" colspan="1">(75.4)</td> <td align="left" rowspan="1" colspan="1">19</td> <td align="left" rowspan="1" colspan="1">(6.6)</td> <td align="left" rowspan="1" colspan="1">271</td> <td align="left" rowspan="1" colspan="1">(44.0)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> Flu B</td> <td align="left" rowspan="1" colspan="1">9</td> <td align="left" rowspan="1" colspan="1">(3.4)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">9</td> <td align="left" rowspan="1" colspan="1">(1.5)</td> </tr> <tr> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1"><b>Real Time PCR (+)</b></td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">213</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(79.8)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">48</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(78.7)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">21</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(7.3)</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">282</td> <td align="left" style="background-color:#F2F2F2" rowspan="1" colspan="1">(45.8)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> A(H1N1)pdm09</td> <td align="left" rowspan="1" colspan="1">169</td> <td align="left" rowspan="1" colspan="1">(63.3)</td> <td align="left" rowspan="1" colspan="1">46</td> <td align="left" rowspan="1" colspan="1">(75.4)</td> <td align="left" rowspan="1" colspan="1">18</td> <td align="left" rowspan="1" colspan="1">(6.3)</td> <td align="left" rowspan="1" colspan="1">233</td> <td align="left" rowspan="1" colspan="1">(37.8)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> A(H3N2)</td> <td align="left" rowspan="1" colspan="1">34</td> <td align="left" rowspan="1" colspan="1">(12.7)</td> <td align="left" rowspan="1" colspan="1">2</td> <td align="left" rowspan="1" colspan="1">(3.3)</td> <td align="left" rowspan="1" colspan="1">3</td> <td align="left" rowspan="1" colspan="1">(1)</td> <td align="left" rowspan="1" colspan="1">39</td> <td align="left" rowspan="1" colspan="1">(6.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> B/Victoria</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"> B/Yamagata</td> <td align="left" rowspan="1" colspan="1">10</td> <td align="left" rowspan="1" colspan="1">(3.8)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">(0)</td> <td align="left" rowspan="1" colspan="1">10</td> <td align="left" rowspan="1" colspan="1">(1.6)</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>(+): positive</p> </div> <div class="fn-type-" title=""> <p>* Percentage are shown for detection (influenza type A, B, subtype A and lineage B) on total collected samples</p> </div> </div> </div> </div> <div class="demographicandbaselineclinicalcharacteristicsofinfluenzaa(h1n1)pdm09patients" title="sec"> <div class="title" tagx="title" title="title">
Demographic and baseline clinical characteristics of influenza A(H1N1)pdm09 patients</div> <p>In 2017, the median age of the influenza A(H1N1)pdm09 positive out-patients was 3.9 years in Yangon and 16.7 years in Pyinmana, respectively, and 2.2 years in the in-patient study. The differences detected between the median age of out-patients in Yangon and Pyinmana study sites was because the patients who were included in the Pyinmana study site were widely distributed in various age groups, from infants to adults (3 months to 56 years), and majority of the patients who visited the clinic were children less than 5 years old in Yangon for both in out-patients and in-patients groups (<a href="#pone.0229601.t002">Table 2</a>). All patients enrolled in this study did not receive any influenza vaccines. Overall the most commonly reported presenting symptoms of the influenza A(H1N1)pdm09-positive out-patients and in-patients were fever, cough, and rhinorrhea. Additionally, the majority of hospitalized patients had a history of dyspnea and wheezing on admission, reflecting the severity of the in-patients (<a href="#pone.0229601.t002">Table 2</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t002">10.1371/journal.pone.0229601.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Demographic and baseline clinical characteristics of A(H1N1)pdm09 patients, Myanmar, 2017.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t002g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Variables</th> <th align="left" colspan="2" rowspan="1">Out-patients</th> <th align="left" rowspan="1" colspan="1">In-patients</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">Study site</th> <th align="left" rowspan="1" colspan="1">Yangon (N = 169)</th> <th align="left" rowspan="1" colspan="1">Pyinmana (N = 46)</th> <th align="left" rowspan="1" colspan="1">Yangon (N = 18)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1"><b>Age</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">median [range]</td> <td align="left" rowspan="1" colspan="1">3.9 [0.5–28.0]</td> <td align="left" rowspan="1" colspan="1">16.7 [0.25–56.0]</td> <td align="left" rowspan="1" colspan="1">2.2 [0.3–5.5]</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>Age groups</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">&amp;lt;5 yrs</td> <td align="left" rowspan="1" colspan="1">116 (68.6%)</td> <td align="left" rowspan="1" colspan="1">9 (19.6%)</td> <td align="left" rowspan="1" colspan="1">16 (88.9%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">5–14 yrs</td> <td align="left" rowspan="1" colspan="1">50 (29.6%)</td> <td align="left" rowspan="1" colspan="1">18 (39.1%)</td> <td align="left" rowspan="1" colspan="1">2 (11.1%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">15–29 yrs</td> <td align="left" rowspan="1" colspan="1">3 (1.8%)</td> <td align="left" rowspan="1" colspan="1">11 (23.9%)</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">30–64 yrs</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">8 (17.4%)</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">&amp;gt; = 65 yrs</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>Gender</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>n (%)</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Female</td> <td align="left" rowspan="1" colspan="1">62 (36.7%)</td> <td align="left" rowspan="1" colspan="1">14 (30.4%)</td> <td align="left" rowspan="1" colspan="1">8 (44.4%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Male</td> <td align="left" rowspan="1" colspan="1">107 (63.3%)</td> <td align="left" rowspan="1" colspan="1">32 (69.6%)</td> <td align="left" rowspan="1" colspan="1">10 (55.6%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>Symptoms</b></td> <td align="left" rowspan="1" colspan="1"><b>Presence (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>Presence (%)</b></td> <td align="left" rowspan="1" colspan="1"><b>Presence (%)</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Fever</td> <td align="left" rowspan="1" colspan="1">139 (82.2%)</td> <td align="left" rowspan="1" colspan="1">44 (95.7%)</td> <td align="left" rowspan="1" colspan="1">14 (77.8%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Cough</td> <td align="left" rowspan="1" colspan="1">143 (84.6%)</td> <td align="left" rowspan="1" colspan="1">37 (80.4%)</td> <td align="left" rowspan="1" colspan="1">18 (100%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Wheezing</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">4 (8.7%)</td> <td align="left" rowspan="1" colspan="1">12 (66.7%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Rhinorrhea</td> <td align="left" rowspan="1" colspan="1">130 (76.9%)</td> <td align="left" rowspan="1" colspan="1">43 (75.4%)</td> <td align="left" rowspan="1" colspan="1">13 (72.2%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Dyspnea</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">14 (77.8%)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Headache</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">21 (45.7%)</td> <td align="left" rowspan="1" colspan="1">NA</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Myalgia</td> <td align="left" rowspan="1" colspan="1">2 (1.2%)</td> <td align="left" rowspan="1" colspan="1">28 (60.9%)</td> <td align="left" rowspan="1" colspan="1">NA</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Arthralgia</td> <td align="left" rowspan="1" colspan="1">1 (0.6%)</td> <td align="left" rowspan="1" colspan="1">28 (60.9%)</td> <td align="left" rowspan="1" colspan="1">NA</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Diarrhea</td> <td align="left" rowspan="1" colspan="1">0 (0.0%)</td> <td align="left" rowspan="1" colspan="1">5 (10.9%)</td> <td align="left" rowspan="1" colspan="1">NA</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>n = number of patients that responded by "yes�? or "no�? for a given symptom.</p> </div> <div class="fn-type-" title=""> <p>NA—Not Applicable</p> </div> </div> </div> <p>During 2017, the monthly distribution of influenza virus-positive, as detected by real-time PCR analysis, revealed that the influenza season in Myanmar started between June and October. Influenza A(H1N1)pdm09 was the predominant subtype in the two study sites, Yangon (169/267, 63.3%) and Pyinmana (46/61, 75.4%) (<a href="#pone.0229601.t001">Table 1</a>). The seasonal peak of A(H1N1)pdm09 was observed in July, during the rainy season of Myanmar, in both Yangon and Pyinmana study sites (<a href="#pone.0229601.g001">Fig 1</a>). Influenza A(H3N2) circulated throughout the season (from June to October) at low levels in both Yangon and Pyinmana. Influenza B viruses detected in Yangon belonged to the B/Yamagata lineage and were observed from August to October, while influenza B was not detected in Pyinmana (<a href="#pone.0229601.g001">Fig 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0229601.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Monthly distribution of influenza virus by region in Myanmar during the 2017 influenza season and the location of sample collection sites.</div> <p>The total number of collected samples is shown in the black dashed line. Influenza A(H1N1) pdm09 is shown as red bar; A(H3N2), blue bar; B/Victoria, purple bar and B/Yamagata, green bar. The seasonal peak of A(H1N1)pdm09 was observed in June-July, during the rainy season, and influenza A(H1N1)pdm09 was the predominant subtype in Myanmar during the 2017 influenza season. The map of Myanmar and the green circles represent the two study sites, Yangon and Pyinmana. This map is generated by the software ArcMap 10.0 (ESRI Japan Corporation, Tokyo, Japan). Boundaries are generated by using a world basemap supplied by ESRI Japan Corporation.</p>   </div> </div> <div class="antigeniccharacterizationofinfluenzaa(h1n1)pdm09viruses" title="sec"> <div class="title" tagx="title" title="title">
Antigenic characterization of influenza A(H1N1)pdm09 viruses</div> <p>The twelve selected influenza A(H1N1)pdm09 isolates were characterized by HI assay to determine the antigenicity of the influenza virus strains circulating during the outbreak season using post-infection rabbit antisera raised against the vaccine virus, A/Singapore/GP1908/2015, that was used for the Japanese influenza vaccine strain for the 2017–2018 seasons [<a href="#pone.0229601.ref023">23</a>]. A/Singapore/GP1908/2015 is antigenically similar to A/Michigan/45/2005, a WHO selected influenza vaccine component for the 2017 southern hemisphere [<a href="#pone.0229601.ref022">22</a>, <a href="#pone.0229601.ref024">24</a>]. All twelve test viruses were well recognized by the antiserum raised against the vaccine virus, A/Singapore/GP1908/2015, with titers were 2- and 4-folds higher than the homologous titer (<a href="#pone.0229601.t003">Table 3</a>). Myanmar influenza A(H1N1)pdm09 viruses were antigenically similar to the vaccine virus A/Singapore/GP1908/2015.</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t003" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t003">10.1371/journal.pone.0229601.t003</a><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
Antigenic analysis of selected influenza A(H1N1)pdm09 viruses from Myanmar 2017.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t003g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" colspan="2" rowspan="3">Viruses</th> <th align="left" rowspan="1" colspan="1">Hemagglutination inhibition (HI) titer</th> <th align="left" rowspan="3" colspan="1">Fold-increase</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">Post-infection rabbit antisera</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">A/Singapore/GP1908/2015</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">REFERENCE VIRUSES</td> <td align="left" rowspan="1" colspan="1">examination date</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Singapore/GP1908/2015<a href="#t003fn001">*</a></td> <td align="left" rowspan="1" colspan="1">5 Dec 2017</td> <td align="left" rowspan="1" colspan="1">2560</td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" colspan="2" rowspan="1">TEST VIRUSES</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M007/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M012/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M015/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M023/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M025/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M062/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M064/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M083/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M108/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M109/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M115/2017</td> <td align="left" rowspan="1" colspan="1">5120</td> <td align="left" rowspan="1" colspan="1">2</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">A/Myanmar/17M204/2017</td> <td align="left" rowspan="1" colspan="1">10240</td> <td align="left" rowspan="1" colspan="1">4</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>*</b>Influenza vaccine strain used in Japan for the 2017/18 seasons.</p> </div> </div> </div> </div> <div class="drugresistantinfluenzaa(h1n1)pdm09virus" title="sec"> <div class="title" tagx="title" title="title">
Drug resistant influenza A(H1N1)pdm09 virus</div> <p>Of the 233 influenza A(H1N1)pdm09 viruses from all the study sites, one isolate (0.4%) had the oseltamivir-resistance with H275Y mutation in the NA protein, as analyzed by cycling probe real-time PCR. The isolate originated from an out-patient, who was a 28 years old male with an onset date on 7 August 2017. He visited the medical institution in Pyinmana with a body temperature of 37.2 °C, headache, runny nose, and myalgia. He did not receive oseltamivir treatment prior to and after the medical visit. His precise illness course is unknown, but he recovered without complications. The epidemiological link of the H275Y mutant virus is unknown. Next-generation sequencing analysis revealed that the variant frequency of oseltamivir-resistance histidine to tyrosine amino acid substitution at position 275 in the NA gene of this mutant virus was 99.8% (495 reads/496 total reads). All the other A(H1N1)pdm09 isolates from Yangon and Pyinmana study sites did not show H275Y mutation in NA (<a href="#pone.0229601.t004">Table 4</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t004" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t004">10.1371/journal.pone.0229601.t004</a><span class="label" tagx="label" title="label">Table 4</span> <div class="title" tagx="title" title="title">
Frequency of H275Y mutated A(H1N1)pdm09 virus in 2017 influenza season using cycling probe real-time PCR method.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t004g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" colspan="2" rowspan="1">Location</th> <th align="left" rowspan="1" colspan="1">H275Y (N)</th> <th align="left" rowspan="1" colspan="1">A(H1N1)pdm09 (N)</th> <th align="left" rowspan="1" colspan="1">%</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="2" colspan="1">Out-patients</td> <td align="left" rowspan="1" colspan="1">Yangon</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">169</td> <td align="char" char="." rowspan="1" colspan="1">0.0</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Pyinmana</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">46</td> <td align="char" char="." rowspan="1" colspan="1">2.2</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">In-patients</td> <td align="left" rowspan="1" colspan="1">Yangon</td> <td align="left" rowspan="1" colspan="1">0</td> <td align="left" rowspan="1" colspan="1">18</td> <td align="char" char="." rowspan="1" colspan="1">0.0</td> </tr> <tr> <td align="left" colspan="2" rowspan="1">Total</td> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">233</td> <td align="char" char="." rowspan="1" colspan="1">0.4</td> </tr> </tbody> </table> </div> </div> <p><i>In vitro</i> fluorescence-based NAI assay performed on the selected influenza A(H1N1)pdm09 isolates (12 isolates from out-patients and 12 from in-patients) revealed that the H275Y mutant A(H1N1)pdm09 virus has elevated IC<sub>50</sub> value for oseltamivir (301.5 nM with 198-fold increase) and peramivir (21.9 nM with 274-fold increase) but not for zanamivir (0.58 nM with 0.9-fold difference) and laninamivir (0.72 nM with 2.7-fold difference), when compared to the IC<sub>50</sub> value of the reference drug-sensitive strain A/Perth/265/2009 (275H), indicating resistance to oseltamivir and peramivir with highly reduced inhibition. All of the remaining A(H1N1)pdm09 viruses were sensitive to the four neuraminidase inhibitors with IC<sub>50</sub> values of less than 10-fold difference compared to the reference virus (<a href="#pone.0229601.t005">Table 5</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t005" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t005">10.1371/journal.pone.0229601.t005</a><span class="label" tagx="label" title="label">Table 5</span> <div class="title" tagx="title" title="title">
IC<sub>50</sub> values of influenza A(H1N1)pdm09 viruses in Myanmar, 2017.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t005g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="2" colspan="1">Strain Name</th> <th align="left" rowspan="2" colspan="1">Status</th> <th align="left" rowspan="2" colspan="1">Drug susceptibility related mutation in NA</th> <th align="left" colspan="4" rowspan="1">IC<sub>50</sub><a href="#t005fn002"><sup>a</sup></a> [nM](fold difference)<a href="#t005fn003"><sup>b</sup></a></th> </tr> <tr> <th align="left" rowspan="1" colspan="1">Oseltamivir</th> <th align="left" rowspan="1" colspan="1">Peramivir</th> <th align="left" rowspan="1" colspan="1">Zanamivir</th> <th align="left" rowspan="1" colspan="1">Laninamivir</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M012/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.50 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.11 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M015/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.72 (0.5)</td> <td align="left" rowspan="1" colspan="1">0.12 (1.5)</td> <td align="left" rowspan="1" colspan="1">0.12 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M023/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.50 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.11 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.08 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M025/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.64 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.13 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M062/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.49 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.11 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M064/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.50 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.10 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.08 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M083/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.58 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.12 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M108/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.68 (0.5)</td> <td align="left" rowspan="1" colspan="1">0.09 (1.1)</td> <td align="left" rowspan="1" colspan="1">0.17 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.11 (0.4)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M109/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.41 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.10 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M115/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.56 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.11 (0.2)</td> <td align="left" rowspan="1" colspan="1">0.09 (0.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M204/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.46 (0.3)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.37 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.30 (1.1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M307/2017</td> <td align="left" rowspan="1" colspan="1">Out</td> <td align="left" rowspan="1" colspan="1">H275Y</td> <td align="left" rowspan="1" colspan="1">301.5 (198)</td> <td align="left" rowspan="1" colspan="1">21.9 (274)</td> <td align="left" rowspan="1" colspan="1">0.58 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.72 (2.7)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP001/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.56 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.34 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.32 (1.2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP002/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.55 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.46 (0.7)</td> <td align="left" rowspan="1" colspan="1">0.31 (1.2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP003/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.55 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.46 (0.7)</td> <td align="left" rowspan="1" colspan="1">0.33 (1.2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP004/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.57 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.38 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.29 (1.1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP005/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.64 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.40 (0.7)</td> <td align="left" rowspan="1" colspan="1">0.30 (1.1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP009/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.72 (0.5)</td> <td align="left" rowspan="1" colspan="1">0.10 (1.3)</td> <td align="left" rowspan="1" colspan="1">0.46 (0.7)</td> <td align="left" rowspan="1" colspan="1">0.32 (1.2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP013/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.56 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.05 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.77 (1.2)</td> <td align="left" rowspan="1" colspan="1">0.29 (1.1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP014/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.56 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.09 (1.1)</td> <td align="left" rowspan="1" colspan="1">0.39 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.32 (1.2)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP015/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.59 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.07 (0.9)</td> <td align="left" rowspan="1" colspan="1">0.39 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.28 (1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP018/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.83 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.12 (1.5)</td> <td align="left" rowspan="1" colspan="1">0.45 (0.7)</td> <td align="left" rowspan="1" colspan="1">0.34 (1.3)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP019/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.63 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.38 (0.6)</td> <td align="left" rowspan="1" colspan="1">0.29 (1.1)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP021/2017</td> <td align="left" rowspan="1" colspan="1">In</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">0.61 (0.4)</td> <td align="left" rowspan="1" colspan="1">0.08 (1)</td> <td align="left" rowspan="1" colspan="1">0.32 (0.5)</td> <td align="left" rowspan="1" colspan="1">0.26 (1)</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>"Out�? indicates out-patient and "In�? indicates in-patient in the status</p> </div> <div class="fn-type-" title=""> <p><sup>a</sup> = Generated in fluorescent-based NI assay.</p> </div> <div class="fn-type-" title=""> <p><sup>b</sup> = Compared with the reference strain A/Perth/265/2009 (275H) with IC<sub>50</sub> values for oseltamivir being 1.52nM, peramivir 0.08nM, zanamivir 0.62nM, laninamivir 0.27nM.</p> </div> </div> </div> </div> <div class="geneticcharacterizationofhaandnaproteinsofinfluenzaa(h1n1)pdm09viruses" title="sec"> <div class="title" tagx="title" title="title">
Genetic characterization of HA and NA proteins of influenza A(H1N1)pdm09 viruses</div> <p>The HA and NA genes of influenza A(H1N1)pdm09 viruses in samples selected from 18 out-patients and 12 in-patients, including two cases with fatal outcomes, A/Myanmar/17MP009/2017 and A/Myanmar/17MP015/2017, were genetically characterized. To match the age group with in-patients, test samples of out-patients were randomly selected from children in the same age group (&amp;lt; 5 years old) for better comparison. We failed to sequence six out of 18 viruses from in-patients. All the analyzed Myanmar viruses belonged to genetic clade 6B.1 with amino acid substitutions of S84N, S162N, and I216T in HA [<a href="#pone.0229601.ref024">24</a>] (<a href="#pone.0229601.g002">Fig 2</a>). The majority (26, 87%) of viruses possessed S164T in HA and formed a clade with the Indian 2017 and Japanese 2017–2018 strains. Among them, 13 viruses from out-patients and 11 viruses from in-patients possessed a T314I substitution in HA. It was a unique amino acid substitution common among the Myanmar sequences. Two Myanmar out-patient viruses (A/Myanmar/17M204/2017 and A/Myanmar/17M307/2017) did not belong to these clusters but were closely related to Indian strains in the same year, 2017. Three out-patient viruses (A/Myanmar/17M109/2017, A/Myanmar/17M309/2017, and A/Myanmar/17M310/2017) and one in-patient virus (A/Myanmar/17MP001/2017) shared A215G and S297P substitutions and formed a group with the Japanese strains in 2016 and 2017, having a bootstrap value of over 90%.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0229601.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
HA and NA phylogeny of influenza A(H1N1) 2009 pandemic isolates in Myanmar in 2017.</div> <p>Trees were constructed by the maximum likelihood method using MEGA software (version 6.06). Bootstrap value was determined for 1000 iterations; only values greater than 70% are shown. Myanmar out-patient is shown in red fonts, and in-patient is shown in dark blue fonts. The HA and NA sequences of Myanmar strains belonged to clade 6B.1. Red circle (�?) represents oseltamivir- and peramivir-resistant strain exhibiting NA H275Y substitution. Amino acid changes were based on A/California/07/2009.</p>   </div> <p>In the phylogeny of the NA gene, all the Myanmar viruses possessed V13I, I34V, V264I, and N270K substitutions and belonged to clade 6B.1. Around 60% of Myanmar out-patient viruses and all in-patient viruses shared the R173K amino acid substitution. Among the viruses from this cluster, two out-patient viruses possessed T438A substitution. H275Y mutation on neuraminidase was found in one A(H1N1)pdm09 strain (A/Myanmar/17M307/2017), which was from an out-patient with no prior history of anti-viral treatment. This resistant virus and another sensitive out-patient virus exhibited P93H substitution in NA. Three out-patient viruses and one in-patient virus exhibited the D451G substitution, which formed a small separate cluster as HA (<a href="#pone.0229601.g002">Fig 2</a>).</p> </div> <div class="internalgeneanalysisofinfluenzaa(h1n1)pdm09viruses" title="sec"> <div class="title" tagx="title" title="title">
Internal gene analysis of influenza A(H1N1)pdm09 viruses</div> <p>We further characterized the amino acid substitutions in the remaining six segments of 25 Myanmar influenza A(H1N1)pdm09 strains, 13 out-patients including a case with NA/H275Y mutation and 12 in-patients, using next-generation sequencing. FluSurver research tool (<a href="https://flusurver.bii.a-star.edu.sg/">https://flusurver.bii.a-star.edu.sg/</a>) was used to screen the significant amino acid mutations in the 25 Myanmar and 8 Indian strains collected in 2017. The amino acid sequences of the six segments from the present study showed 99.0–99.8% identity with the reference strain A/Michigan/45/2015.</p> <p>All Myanmar in-patient strains except one possessed the amino acid substitutions (as compared to A/Michigan/45/2015) at positions 66, 299, 398, and 453 in PB2, at positions 12 and 375 in PB1, at position 27 in M2, and at position 65 in NS1. All these mutations were frequently observed (10/13, 77%) in Myanmar out-patient strains. A/Myanmar/17MP009/2017 and A/Myanmar/17MP015/2017 strains from two fatal cases possessed the same amino acid mutations found in out-patients and in-patients who recovered from the disease (<a href="#pone.0229601.t006">Table 6</a>). A number of new mutations were observed in PB2 and PB1 in low frequencies, but none are indicated to have significant effect by FluSurver. Only sporadic mutations were found in PA and M1, NP and there was no common mutation in NS2 (<a href="#pone.0229601.s003">S3 Table</a>). In general, no difference was found between out-patients and in-patients. Differences in four amino acid mutations were found between Myanmar and Indian strains (<a href="#pone.0229601.t006">Table 6</a>); however, these mutations had no known functional difference except the mutation at position 27 in the M2 segment, which confers drug resistance to amantadine [<a href="#pone.0229601.ref025">25</a>]. Of note, the first oseltamivir-resistant strain in Myanmar shared the same amino acid substitutions with the Indian strains (<a href="#pone.0229601.t006">Table 6</a> and <a href="#pone.0229601.s003">S3 Table</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0229601.t006" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0229601.t006">10.1371/journal.pone.0229601.t006</a><span class="label" tagx="label" title="label">Table 6</span> <div class="title" tagx="title" title="title">
Abbreviated list of amino acid substitutions found in internal genes of influenza A(H1N1)pdm09 viruses collected from out-patients and in-patients in Myanmar compared with viruses from India in 2017.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0229601.t006g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">Substitution</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">PB2</th> <th align="left" colspan="3" rowspan="1" /> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">PB1</th> <th align="left" rowspan="1" colspan="1" /> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">M2</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">NS1</th> </tr> <tr> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">Strain name</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">66</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">299</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">398</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">453</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">12</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">375</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">27<a href="#t006fn002"><sup>a</sup></a></th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">65</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">A/Michigan/45/2015</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">I</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">R</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">T</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">P</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">I</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">S</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">V</th> <th align="left" style="background-color:#BFBFBF" rowspan="1" colspan="1">M</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1"><b>In-patients</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP001/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP002/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP003/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP004/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP005/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP009/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP013/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP014/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP015/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP018/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP019/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17MP021/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>Out-patients</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M007/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M012/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M015/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M023/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M025/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M062/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M064/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M083/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M108/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M109/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M115/2017</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1">V</td> <td align="left" rowspan="1" colspan="1">N</td> <td align="left" rowspan="1" colspan="1">A</td> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M204/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/Myanmar/17M307/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>India</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/0217/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/0225/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/0298/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/0402/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/1706/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/7078/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/8161/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">A/India/9358/2017</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">K</td> <td align="left" rowspan="1" colspan="1">I</td> <td align="left" rowspan="1" colspan="1">T</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">V</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Analysis was performed by FluSurver (<a href="https://flusurver.bii.a-star.edu.sg/">https://flusurver.bii.a-star.edu.sg/</a>). A/Michigan/45/2015 was used as a reference automatically. A full list of amino acid substitutions can be found in the <a href="#pone.0229601.s003">S3 Table</a>.</p> </div> <div class="fn-type-" title=""> <p><sup>a</sup>Reported to be related to drug resistance to amantadine.</p> </div> </div> </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>A regional influenza outbreak with the major subtype of A(H1N1)pdm09 occurred in Myanmar from June to October in 2017, and an increased number of severe pneumonia cases were reported [<a href="#pone.0229601.ref009">9</a>]. The viruses involved in this outbreak were genetically very similar to A(H1N1)pdm09 strains circulating in Asia and other countries during the same year. There were no differences in the genetic characteristics of A(H1N1)pdm09 influenza viruses collected from out-patients and in-patients, including cases with fatal outcomes. The antigenicity of the Myanmar viruses was similar to the vaccine strain. In addition, the first case of oseltamivir-resistant virus in Myanmar without prior oseltamivir treatment was detected in our study.</p> <p>In Myanmar, the peak of influenza virus activity coincides with the rainy season. Myanmar has a tropical monsoon climate with three seasons: winter (November—February), summer (March—Mid May), and rainy (Mid May—October) seasons, based on the analysis of pressure, rainfall, and temperature [<a href="#pone.0229601.ref026">26</a>]. The study showed that the circulation of influenza viruses peaked in July, which is consistent with previous studies showing that the influenza season in Myanmar peaked in July or August, during the rainy season [<a href="#pone.0229601.ref006">6</a>–<a href="#pone.0229601.ref008">8</a>]. This observation is also similar to the national surveillance data reported to the regional and global influenza surveillance platform (FluNET), which showed that the regional influenza outbreak started during the rainy season from July (week 29) to September (week 36), and influenza virus activity peaked in August with 369 flu-positive cases during the reported weeks 31–35 in 2017 [<a href="#pone.0229601.ref027">27</a>]. Nationally, an increasing number of influenza cases was notified in July, and the large number of samples for influenza testing became a burden to NIC, which resulted in a restriction on sample handling in our study sites. Therefore, influenza detection was decreased in August at study sites compared to the national influenza peak period.</p> <p>Antigenic analysis using the HI test showed that Myanmar strains in 2017 were similar to the vaccine strain A/Singapore/GP1908/2015, a vaccine component for the Japanese inactivated vaccine for the 2017/2018 season [<a href="#pone.0229601.ref022">22</a>]. A/Singapore/GP1908/2015 belonged to 6B.1 clade and is antigenically similar to A/Michigan/45/2015, a vaccine component for the Southern hemisphere in 2017 [<a href="#pone.0229601.ref024">24</a>, <a href="#pone.0229601.ref028">28</a>] and Northern hemisphere in 2017/2018 [<a href="#pone.0229601.ref029">29</a>]. In this study, the HI results of the isolated viruses that reacted to the antisera raised against the vaccine strain had higher titer by 2–4 fold increase than the homologous titer of the vaccine strain virus. The higher HI titer of test viruses than vaccine viruses is often observed in the circulating strains, and it merely shows an antigenic match with the vaccine strains. Indeed, a recent report from the European Centre for Disease Prevention and Control shows that the HI titer of the circulating viruses was 2–4 fold higher, indicating a match with the vaccine and the reference viruses [<a href="#pone.0229601.ref030">30</a>]. WHO changed the vaccine component from A/California/7/2009 to A/Michigan/45/2015 for the Southern hemisphere vaccine in 2017 because of the increasing number of clade 6B.1 viruses observed at the beginning of 2016. Clade 6B.1 viruses were reported to show similar antigenicity to ferret antisera against A/California/7/2009 when tested by the HI test, but they reacted poorly with pediatric, adult, and older adult pre- and post-vaccination sera [<a href="#pone.0229601.ref022">22</a>]. Thus, WHO changed the vaccine component into A/Michigan/45/2015 from the 2017 southern hemisphere vaccine. In this outbreak, Myanmar experienced severe cases of influenza A/H1N1pdm09 in 2017; however, it turned out that these viruses matched with the vaccine strain.</p> <p>According to the phylogenetic analysis, the 2017 Myanmar influenza A(H1N1)pdm09 strains belonged to clade 6B.1 and were closely related to the strains that circulated in India and other countries during the same period. India reported several outbreaks and severe cases caused by influenza A(H1N1)pdm09 of Clade 6B and 6B.1 during 2015–2017 [<a href="#pone.0229601.ref016">16</a>–<a href="#pone.0229601.ref020">20</a>, <a href="#pone.0229601.ref031">31</a>]. We suspect the possibility of transmission from India or other countries situated to the west of Myanmar because early cases of this outbreak were reported from Chin State, which shares its western border with Bangladesh. The analyzed strains in this study are genetically similar to the vaccine viruses, A/Michigan/45/2015 and A/Singapore/GP1908/2015. It was known that S74R, I295V mutations in HA protein were recently acquired by the virus; this mutation has high frequency of detection (66%) globally [<a href="#pone.0229601.ref017">17</a>]. Moreover, about 32% of global strains had the S164T mutation, and it is reported as the most recent mutation observed among 2017 strains [<a href="#pone.0229601.ref017">17</a>]. Similarly, these mutations were highly detected (87%) in Myanmar strains. In contrast to Myanmar strains, amino acid substitutions A73V, V321I, T508A, and I510T, were reported from Central India sequences in 2017 with low global occurrence [<a href="#pone.0229601.ref017">17</a>].</p> <p>Another known important amino acid change, D222N/G in HA, was not detected in viruses from Myanmar in-patients. This mutation affects virus tropism and enhances the severity of disease and mortality by assisting viral binding to α-2,3 receptors, which are mainly present in the lower respiratory tract with the occurrence of 0.4% frequency at global level [<a href="#pone.0229601.ref032">32</a>–<a href="#pone.0229601.ref034">34</a>]. In NA phylogeny, amino acid substitutions V13I, I34V, and I314M were found in all Myanmar strains. These substitutions were also detected in Indian and Nepalese A(H1N1)pdm09 outbreak strains in early 2015, which suggests the coevolution of HA and NA [<a href="#pone.0229601.ref016">16</a>]. According to our investigation (unpublished data), influenza A(H3N2) and B viruses, which were the predominant circulating strains in Myanmar during 2016, could be a possible reason for this big outbreak of A(H1N1)pdm09 and it may be responsible for the reduced herd immunity against A(H1N1)pdm09, when the new 6B.1 clade virus was introduced in Myanmar in 2017.</p> <p>Genetic characterization of six internal genes (PB2, PB1, PA, NP, MP, and NS) showed that there were no marked genetic changes related to increased pathogenicity. Several complex factors are associated with severe pneumonia: viral factors, such as mutations in the polymerase, HA, PB1-F2, polymerase acidic protein frameshift, and host factors, such as comorbidities, single-nucleotide polymorphism, inflammation, high cytokines and chemokines; bacterial factors, such as function of attachment, replication, and loss of repair can result in secondary bacterial infections [<a href="#pone.0229601.ref035">35</a>]. No significant genetic differences were found between 13 viruses collected from out-patients and 12 viruses from in-patients. It was, thus, suggested that the severity of illness is not caused by the differences of virus strains between in-patients and out-patients but characteristics of influenza A(H1N1)pdm09 causing viral pneumonia [<a href="#pone.0229601.ref035">35</a>–<a href="#pone.0229601.ref037">37</a>]. In this study, internal genes of Indian strains and Myanmar strains were compared. The local circulation of A(H1N1)pdm09 was observed from 2016 to 2017 in India, Nepal, and Bangladesh prior to the outbreaks in Myanmar [<a href="#pone.0229601.ref038">38</a>]. Among the four mutation differences between Myanmar and Indian strains, the only functional significance was at position 27 in the M2 segment that confers resistance to amantadine. Adamantane-resistant influenza variants have been circulating in the world for decades [<a href="#pone.0229601.ref025">25</a>]. M2 gene mutations associated with resistance to amantadine were found in only about 1% of V27A mutations compared to 95% of S31N mutation [<a href="#pone.0229601.ref025">25</a>]. Indeed, all of Myanmar and Indian strains in this study also possessed S31N mutation. It is highly presumable that strains from the two countries shared the same source of origin because Myanmar and Indian strains shared higher percentage of sequence identity. No additional viral genetic changes associated with disease severity were identified so far.</p> <p>Here, we report the identification and detection of a H275Y mutation in NA that confer resistant to oseltamivir from an influenza A(H1N1)pdm09 isolate in community samples of Myanmar, which was detected for the first time in Myanmar; however, its detection rate was low at only 0.5%. In Myanmar, influenza A(H1N1)pdm09 was first detected in 2009 and the oseltamivir-resistant H275Y variant has not been detected in specimens, either from the community or hospitalized patients since then, although all isolates tested showed the S31N mutation in M2 that conferred resistance to amantadine [<a href="#pone.0229601.ref008">8</a>]. Oseltamivir was not previously used for the treatment of influenza in Myanmar, but at the time of the outbreak in 2017, it was supplied by the WHO as an emergency control measure. The drug was administered to the admitted patients with severe cases of influenza infection (<a href="#pone.0229601.s004">S1 File</a>), but the number of prescribed courses for estimating the effect on the emergence of oseltamivir-resistant strain is not available. It was already reported worldwide that oseltamivir-resistant influenza A(H1N1)pdm09 virus infections were detected in community settings with no known exposure to oseltamivir and the resistant virus spread within the community even in the absence of drug-selective pressure [<a href="#pone.0229601.ref039">39</a>, <a href="#pone.0229601.ref040">40</a>]. This fact suggests that there is a possibility of rapid transmission of resistant strains from patients, who were already treated with oseltamivir, in a short period after prescribing the antiviral drug in Myanmar. Moreover, this virus showed high reduced inhibition (HRI) with high fold-increase in IC<sub>50</sub> (198 and 274) to two NAIs, oseltamivir and peramivir. Global analysis on the susceptibility of human influenza viruses to the NAI reported that the most common NA amino acid substitution was H275Y in A(H1N1)pdm09 viruses, which confers HRI by oseltamivir and peramivir [<a href="#pone.0229601.ref041">41</a>] without affecting the susceptibility to zanamivir and laninamivir [<a href="#pone.0229601.ref042">42</a>]. Despite the low prevalence of oseltamivir-resistance according to our study data, it highlights the necessity for continuous monitoring of the influenza virus and close surveillance of antiviral drug resistance in Myanmar.</p> <p>There were no known fatal cases of influenza A(H1N1)pdm09 reported in Myanmar until 2017. Indeed 2 of our pediatric in-patients who underwent drug susceptibility and genetic sequencing were death cases. Moreover, detailed characterization of influenza A(H1N1)pdm09 viruses is very limited [<a href="#pone.0229601.ref008">8</a>] and whole-genome sequencing of Myanmar influenza A(H1N1)pdm09 strains have not been characterized before our study.</p> <p>Nonetheless, this study has several limitations. The samples tested herein were collected from two surveillance sites for out-patients and from one children hospital for in-patients. Due to the limitations of human resources in Myanmar, there was no hospital to collect samples from in-patients and out-patients in the same surveillance site within the study period. Therefore, we decided to select two hospitals for out-patients and one hospital for in-patients. Moreover, influenza-positive sample size of in-patients was quite small compared to out-patients even though we collected 288 in-patients. We detected the first oseltamivir-resistant strain in Myanmar, but the epidemiological link of the patient was unclear. Besides, we could analyze only two out-patients in Pyinmana by NGS. It would be more useful if we could increase the number of samples for sequencing for a more comprehensive analysis. Analysis of NGS data is still ongoing, and the results will be published elsewhere.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In this study, we characterized the influenza A(H1N1)pdm09 strains that circulated in Myanmar during the 2017 outbreak and attempted to identify factors that are responsible for the virulence of the virus, which may cause the severity of influenza among in-patients by comparing with the out-patient strains. Although viral genome features responsible for the differential severity of influenza between in-patients and out-patients could not be thoroughly elucidated, we present the first-time characterization of Myanmar influenza viruses by the full genome sequencing. This study demonstrates the importance of continuing antiviral monitoring, epidemiological investigations, and genetic characterization in the data-limited country like Myanmar.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Details of whole genomes of Myanmar influenza A(H1N1)pdm09 used in this study.</div> <p>All the data are registered to the Global Initiative on Sharing All Influenza Data (GISAID).</p> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0229601.s001.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Details of influenza A(H1N1)pdm09, HA and NA sequences used in the phylogenetic tree in this study.</div> <p>All the data are registered to the Global Initiative on Sharing All Influenza Data (GISAID).</p> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0229601.s002.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
Detail of amino acid substitutions found in PB2, PB1, PA, NP, MP, and NS protein of influenza A(H1N1)pdm09 viruses collected from out-patients and in-patients in Myanmar compared with viruses from India in 2017.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0229601.s003.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 File</span> <div class="title" tagx="title" title="title">
Influenza (H1N1)pdm infection.</div> <p>Management Protocol and Case Report Form. North Okkalapa General Hospital.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0229601.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We are grateful to Dr. Tomomi Ota and the staff of the Chest Medical Unit of Sanpya Hospital, Pyinmana General Hospital, Yankin Children Hospital for sample collection, for the management of the study. We thank Niigata University staff, Mrs. Akemi Watanabe for laboratory assistance, Ms. Saori Nedachi and Ms. Yukie Maruyama for secretarial support and Dr. Yugo Shobugawa for study planning. We also thank Dr. Nobuhiro Takemae in the Ministry of Agriculture, Forestry and Fisheries in Japan for contribution to the analysis, and Drs. Nobuo Nagata, Isamu Sato, Takashi Kawashima, Toshihiro Tanaka, Satoshi Aoki, Naoki, Kodo, Hironori Masaki, Norichika Asoh, Yoshiko Tsuchihashi, Yutaka Shirahige, and Hirotsune Hamabata for collecting influenza viruses in Japan. We would like to thank Editage (<a href="http://www.editage.com">www.editage.com</a>) for English language editing.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0229601.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goraya</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evolution of Influenza A Virus by Mutation and Re-Assortment</span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">8</span>):pii: E1650. <span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms18081650">10.3390/ijms18081650</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28783091">28783091</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mills</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robins</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transmissibility of 1918 pandemic influenza</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">432</span>(<span class="issue" tagx="issue" title="issue">7019</span>):<span class="fpage" tagx="fpage" title="fpage">904</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature03063">10.1038/nature03063</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15602562">15602562</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biggerstaff</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cauchemez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gambhir</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finelli</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">Estimates of the reproduction number for seasonal, pandemic, and zoonotic influenza: a systematic review of the literature</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">480</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-14-480">10.1186/1471-2334-14-480</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25186370">25186370</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Domselaar</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bastien</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaudoin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tyler</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Nationwide molecular surveillance of pandemic H1N1 influenza A virus genomes: Canada, 2009</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e16087</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0016087">10.1371/journal.pone.0016087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21249207">21249207</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Central Epidemiology Unit-Department of Public Health Ministry of Health and Sports. Influenza Like Illness (ILI) and Severe Acute Respiratory Infection (SARI) Surveillance Guideline (Myanmar), 2018. <a href="https://bit.ly/2J9UTrP">https://bit.ly/2J9UTrP</a></span></li> <li tag="ref"><a name="pone.0229601.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dapat</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyaw</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology of human influenza A and B viruses in Myanmar from 2005 to 2007</span>. <span class="source" tagx="source" title="source">Intervirology</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">310</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1159/000237738">10.1159/000237738</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19776616">19776616</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Htwe</span><span class="given-names" tagx="given-names" title="given-names">KTZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dapat</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shobugawa</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hibino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kondo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Phylogeographic analysis of human influenza A and B viruses in Myanmar, 2010–2015</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e0210550</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0210550">10.1371/journal.pone.0210550</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30629691">30629691</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dapat</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyaw</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myint</span><span class="given-names" tagx="given-names" title="given-names">YY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oo</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oo</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Delayed emergence of oseltamivir-resistant seasonal influenza A (H1N1) and pandemic influenza A(H1N1)pdm09 viruses in Myanmar</span>. <span class="source" tagx="source" title="source">Influenza Other Respir Viruses</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">766</span>–<span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/irv.12030">10.1111/irv.12030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23122276">23122276</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Department of Public Health Ministry of Health and Sports Myanmar. Report on Response Activities and Preventive Measures on Seasonal Influenza A (H1N1)pdm09. MoHS report 2017. <a href="https://www.mohs.gov.mm">https://www.mohs.gov.mm</a></span></li> <li tag="ref"><a name="pone.0229601.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Influenza Virus Research Center-National Institute of Infectious Diseases. Laboratory protocols for detection of influenza virus. <a href="https://www.niid.go.jp/niid/images/lab-manual/Influenza2014.pdf">https://www.niid.go.jp/niid/images/lab-manual/Influenza2014.pdf</a></span></li> <li tag="ref"><a name="pone.0229601.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">World Health Organization. Manual for the laboratory diagnosis and virological surveillance of influenza. <a href="https://www.who.int/influenza/gisrs_laboratory/manual_diagnosis_surveillance_influenza/en/">https://www.who.int/influenza/gisrs_laboratory/manual_diagnosis_surveillance_influenza/en/</a></span></li> <li tag="ref"><a name="pone.0229601.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaraket</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishikawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Identification of oseltamivir resistance among pandemic and seasonal influenza A (H1N1) viruses by an His275Tyr genotyping assay using the cycling probe method</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">125</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.01401-10">10.1128/JCM.01401-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21084523">21084523</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaraket</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kondo</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hibino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yagami</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takemae</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Full Genome Characterization of Human Influenza A/H3N2 Isolates from Asian Countries Reveals a Rare Amantadine Resistance-Conferring Mutation and Novel PB1-F2 Polymorphisms</span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">262</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.00262">10.3389/fmicb.2016.00262</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27014195">27014195</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">World Health Organization. Meetings of the WHO working group on surveillance of influenza antiviral susceptibility—Geneva, November 2011 and June 2012. Weekly Epidemiological Record, 87 (‎39)‎: 369–374. <a href="https://apps.who.int/iris/handle/10665/241965">https://apps.who.int/iris/handle/10665/241965</a></span></li> <li tag="ref"><a name="pone.0229601.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chon</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hibino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yagami</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dapat</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Effectiveness of the quadrivalent inactivated influenza vaccine in Japan during the 2015–2016 season: A test-negative case-control study comparing the results by real time PCR, virus isolation</span>. <span class="source" tagx="source" title="source">Vaccine X</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">1</span>:100011. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jvacx.2019.100011">10.1016/j.jvacx.2019.100011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31384733">31384733</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirakura</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujisaki</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishida</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Characterization of influenza A(H1N1)pdm09 viruses isolated from Nepalese and Indian outbreak patients in early 2015</span>. <span class="source" tagx="source" title="source">Influenza Other Respir Viruses</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">399</span>–<span class="lpage" tagx="lpage" title="lpage">403</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/irv.12469">10.1111/irv.12469</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28792671">28792671</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potdar</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barde</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular analysis of influenza A H1N1pdm09 virus circulating in Madhya Pradesh, India in the year 2017</span>. <span class="source" tagx="source" title="source">Virusdisease</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">380</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s13337-018-0474-0">10.1007/s13337-018-0474-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30159375">30159375</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarmah</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borkakoty</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sarma</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hazarika</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakharia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genetic variations of the Hemagglutinin gene of Pandemic Influenza A (H1N1) viruses in Assam, India during 2016</span>. <span class="source" tagx="source" title="source">3 Biotech</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">408</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13205-018-1427-4">10.1007/s13205-018-1427-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30237955">30237955</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mukherjee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nayak</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Satpathi</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chawla-Sarkar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genetic Characterization of Circulating 2015 A(H1N1)pdm09 Influenza Viruses from Eastern India</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">e0168464</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0168464">10.1371/journal.pone.0168464</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27997573">27997573</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tandel</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dash</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parida</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Oseltamivir-resistant influenza A(H1N1)pdm09 virus associated with high case fatality, India 2015</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">90</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">836</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25013">10.1002/jmv.25013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29288584">29288584</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filipski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">2725</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst197">10.1093/molbev/mst197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122">24132122</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Influenza Virus Research Center-National Institute of Infectious Diseases. Influenza 2016/17 season, Japan. Infectious Agents Surveillance Report. <a href="https://www.niid.go.jp/niid/ja/iasr-vol38/7678-idx453.html">https://www.niid.go.jp/niid/ja/iasr-vol38/7678-idx453.html</a></span></li> <li tag="ref"><a name="pone.0229601.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Influenza Virus Research Center-National Institute of Infectious Diseases. Influenza Vaccine Strains for Japan. <a href="https://www.niid.go.jp/niid/ja/vaccine-j/249-vaccine/584-atpcs002.html">https://www.niid.go.jp/niid/ja/vaccine-j/249-vaccine/584-atpcs002.html</a> (Japanese)</span></li> <li tag="ref"><a name="pone.0229601.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Worldwide Influenza Centre-The Francis Crick Institute. February 2017 interim report. <a href="https://www.crick.ac.uk/sites/default/files/2018-07/crick_nh_vcm_report_feb_2017_v2.pdf">https://www.crick.ac.uk/sites/default/files/2018-07/crick_nh_vcm_report_feb_2017_v2.pdf</a></span></li> <li tag="ref"><a name="pone.0229601.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Adamantane-resistant influenza a viruses in the world (1902–2013): frequency and distribution of M2 gene mutations</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e0119115</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0119115">10.1371/journal.pone.0119115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25768797">25768797</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Aung LL ZE, Theingi P, Elvera N, Aung PP, Han TT, et al. Myanmar Climate Report. Department of Meteorology and Hydrology, Myanmar. 2017;9:1–105. <a href="https://www.met.no">https://www.met.no</a>.</span></li> <li tag="ref"><a name="pone.0229601.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">World Health Organization. Influenza Laboratory Surveillance Information, 2017. FluNet, Global Influenza Surveillance and Response System (GISRS). <a href="https://www.who.int/influenza/gisrs_laboratory/flunet/en/">https://www.who.int/influenza/gisrs_laboratory/flunet/en/</a></span></li> <li tag="ref"><a name="pone.0229601.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: World Health Organization</span>. <span class="mixed-article-title" title="mixed-article-title">Recommended composition of influenza virus vaccines for use in the 2017 southern hemisphere influenza season</span><span class="source" tagx="source" title="source">Weekly Epidemiological Record</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">41</span>(<span class="issue" tagx="issue" title="issue">91</span>):<span class="fpage" tagx="fpage" title="fpage">469</span>–<span class="lpage" tagx="lpage" title="lpage">84</span>.</span></li> <li tag="ref"><a name="pone.0229601.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: World Health Organization</span>. <span class="mixed-article-title" title="mixed-article-title">Recommended composition of influenza virus vaccines for use in the 2017–2018 northern hemisphere influenza season</span>. <span class="source" tagx="source" title="source">Weekly Epidemiological Record</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">92</span>):<span class="fpage" tagx="fpage" title="fpage">117</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>.</span></li> <li tag="ref"><a name="pone.0229601.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">European Centre for Disease Prevention and Control. Influenza virus characterisation-summary Europe-October 2019. <a href="https://www.ecdc.europa.eu/sites/default/files/documents/Influenza-characterisation-october-2019.pdf">https://www.ecdc.europa.eu/sites/default/files/documents/Influenza-characterisation-october-2019.pdf</a></span></li> <li tag="ref"><a name="pone.0229601.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jagadesh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krishnan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivadas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arunkumar</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genetic characterization of hemagglutinin (HA) gene of influenza A viruses circulating in Southwest India during 2017 season</span>. <span class="source" tagx="source" title="source">Virus Genes</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">55</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">458</span>–<span class="lpage" tagx="lpage" title="lpage">64</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-019-01675-x">10.1007/s11262-019-01675-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31129786">31129786</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barde</span><span class="given-names" tagx="given-names" title="given-names">PV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bharti</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahare</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The high frequency of non-aspartic acid residues at HA222 in influenza A(H1N1) 2009 pandemic viruses is associated with mortality during the upsurge of 2015: a molecular and epidemiological study from central India</span>. <span class="source" tagx="source" title="source">Epidemiol Infect</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">145</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">2656</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268817001595">10.1017/S0950268817001595</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28768557">28768557</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilander</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rykkvin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudman</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hungnes</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>. <span class="mixed-article-title" title="mixed-article-title">Observed association between the HA1 mutation D222G in the 2009 pandemic influenza A(H1N1) virus and severe clinical outcome, Norway 2009–2010</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">9</span>):pii: 19498. <span class="pub-id"><a href="https://dx.doi.org/10.2807/ese.15.09.19498-en">10.2807/ese.15.09.19498-en</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20214869">20214869</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Childs</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matrosovich</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wharton</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palma</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Altered receptor specificity and cell tropism of D222G hemagglutinin mutants isolated from fatal cases of pandemic A(H1N1) 2009 influenza virus</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">84</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">12069</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01639-10">10.1128/JVI.01639-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20826688">20826688</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kash</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubenberger</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>. <span class="mixed-article-title" title="mixed-article-title">The role of viral, host, and secondary bacterial factors in influenza pathogenesis</span>. <span class="source" tagx="source" title="source">Am J Pathol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">185</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1528</span>–<span class="lpage" tagx="lpage" title="lpage">36</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajpath.2014.08.030">10.1016/j.ajpath.2014.08.030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25747532">25747532</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruuskanen</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lahti</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murdoch</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Viral pneumonia</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">377</span>(<span class="issue" tagx="issue" title="issue">9773</span>):<span class="fpage" tagx="fpage" title="fpage">1264</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(10)61459-6">10.1016/S0140-6736(10)61459-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21435708">21435708</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacIntyre</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chughtai</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnes</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ridda</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seale</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toms</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The role of pneumonia and secondary bacterial infection in fatal and serious outcomes of pandemic influenza a(H1N1)pdm09</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">637</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-018-3548-0">10.1186/s12879-018-3548-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30526505">30526505</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">World Health Organization. Influenza Laboratory Surveillance Information by Global Influenza Surveillance Response System (GISRS). FluNet—CHARTS. <a href="https://www.who.int/influenza/gisrs_laboratory/flunet/charts/en/">https://www.who.int/influenza/gisrs_laboratory/flunet/charts/en/</a></span></li> <li tag="ref"><a name="pone.0229601.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leang</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phuah</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Increased detection in Australia and Singapore of a novel influenza A(H1N1)2009 variant with reduced oseltamivir and zanamivir sensitivity due to a S247N neuraminidase mutation</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">23</span>):pii: 19884. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21679678">21679678</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lackenby</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moran Gilad</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pebody</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miah</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calatayud</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolotin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Continued emergence and changing epidemiology of oseltamivir-resistant influenza A(H1N1)2009 virus, United Kingdom, winter 2010/11</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">5</span>):pii: 19784. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21315056">21315056</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubareva</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Besselaar</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fry</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregory</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global update on the susceptibility of human influenza viruses to neuraminidase inhibitors, 2015–2016</span>. <span class="source" tagx="source" title="source">Antiviral Res</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">146</span>:<span class="fpage" tagx="fpage" title="fpage">12</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2017.08.004">10.1016/j.antiviral.2017.08.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28802866">28802866</a></span></span></li> <li tag="ref"><a name="pone.0229601.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tewawong</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vichiwattana</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korkong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klinfueng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suntronwong</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thongmee</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Evolution of the neuraminidase gene of seasonal influenza A and B viruses in Thailand between 2010 and 2015</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e0175655</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0175655">10.1371/journal.pone.0175655</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28410396">28410396</a></span></span></li> </ul> </div> </div>  </body></html>