<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Negl Trop Dis</span><span class="iso-abbrev" title="iso-abbrev">PLoS Negl Trop Dis</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosntds</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Neglected Tropical Diseases</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1935-2727</span><span class="issn-epub" tagx="issn" title="issn-epub">1935-2735</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5542384">5542384</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pntd.0005694">10.1371/journal.pntd.0005694</a></span><span class="publisher-id" title="publisher-id">PNTD-D-17-00339</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span>: <span class="subject" title="subject">Asia</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Engineering and Technology</span>: <span class="subject" title="subject">Transportation</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Network Analysis</span>: <span class="subject" title="subject">Centrality</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biogeography</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Ecology and Environmental Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biogeography</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Earth Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geography</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biogeography</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Genetics</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Genetics</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Genetics</span>: <span class="subject" title="subject">Phylogeography</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Tropical Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neglected Tropical Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span>: <span class="subject" title="subject">Sequence Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Asia</span>: <span class="subject" title="subject">China</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Increasing airline travel may facilitate co-circulation of multiple dengue virus serotypes in Asia</div> <div class="alt-title" title="alt-title">
Spread of dengue viruses in Asia</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4466-0858</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Huaiyu</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Zhe</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">Nuno Rodrigues</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Jing</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cazelles</span><span class="given-names" tagx="given-names" title="given-names">Bernard</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff004"><sup>4</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Shanqian</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Bo</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Qiqi</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">Oliver G.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff003"><sup>3</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Bing</span></span><span class="role" tagx="role" title="role">Conceptualization</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">State Key Laboratory of Remote Sensing Science, College of Global Change and Earth System Science, Beijing Normal University, Beijing, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Ministry of Education Key Laboratory for Earth System Modeling, Department of Earth System Science, School of Environment, Tsinghua University, Beijing, China</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Department of Zoology, University of Oxford, Oxford, United Kingdom</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Ecologie &amp;amp; Evolution, UMR 7625, UPMC-ENS, Paris, France</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">UMMISCO UMI 209 IRD - UPMC, Bondy, France</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althouse</span><span class="given-names" tagx="given-names" title="given-names">Benjamin</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Institute for Disease Modeling, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">oliver.pybus@zoo.ox.ac.uk</span> (OP); <span class="email" tagx="email" title="email">bingxu@tsinghua.edu.cn</span> (BiX)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-8-8</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017-8</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0005694</span><span class="history" title="history"><span class="received" title="received">received: 2017-3-8</span><span class="accepted" title="accepted">accepted: 2017-6-8</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Tian et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pntd.0005694.pdf" title="self-uri">pntd.0005694.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The incidence of dengue has grown dramatically in recent decades worldwide, especially in Southeast Asia and the Americas with substantial transmission in 2014–2015. Yet the mechanisms underlying the spatio-temporal circulation of dengue virus (DENV) serotypes at large geographical scales remain elusive. Here we investigate the co-circulation in Asia of DENV serotypes 1–3 from 1956 to 2015, using a statistical framework that jointly estimates migration history and quantifies potential predictors of viral spatial diffusion, including socio-economic, air transportation and maritime mobility data. We find that the spread of DENV-1, -2 and -3 lineages in Asia is significantly associated with air traffic. Our analyses suggest the network centrality of air traffic hubs such as Thailand and India contribute to seeding dengue epidemics, whilst China, Cambodia, Indonesia, and Singapore may establish viral diffusion links with multiple countries in Asia. Phylogeographic reconstructions help to explain how growing air transportation networks could influence the dynamics of DENV circulation.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>In the past half century the incidence of dengue fever worldwide has increased 30-fold, with an estimated ~390 million infections per year. The years 2014 and 2015 were characterized by large dengue outbreaks worldwide, which are a threat to public health, especially in Asian countries. Here we use a phylogeographic approach to reconstruct historical virus movement and evaluate multiple potential predictors of the spatial spread of dengue virus (DENV) among Asian countries. We show that air traffic has played a more important role than maritime transport in shaping large-scale geographic dispersal of DENV in Asia. Our study suggests that the patterns of DENV spread result from the interplay between intensity and structure of human mobility through the air transportation network.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Key Research and Development Program of China</span></div><span class="award-id" tagx="award-id" title="award-id">2016YFA0600104</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Bing</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Natural Science Foundation of China</span></div><span class="award-id" tagx="award-id" title="award-id">81673234</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4466-0858</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Huaiyu</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Fundamental Research Funds for the Central Universities</span></div><span class="award-id" tagx="award-id" title="award-id">2015NT06</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4466-0858</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">Huaiyu</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100004440</span><span class="institution" title="institution">Wellcome Trust</span></div> </div><span class="award-id" tagx="award-id" title="award-id">204311/Z/16/Z</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">Nuno Rodrigues</span></span></span></div> <div class="funding-statement" title="funding-statement">
Funding has been provided by the National Natural Science Foundation of China (81673234), <a href="http://www.nsfc.gov.cn/">http://www.nsfc.gov.cn/</a>; Fundamental Research Funds for the Central Universities, <a href="http://www.moe.gov.cn/?National">http://www.moe.gov.cn/?National</a>&amp;lt;<a href="http://www.moe.gov.cn/%EF%BC%9BNational">http://www.moe.gov.cn/%EF%BC%9BNational</a>&amp;gt; Key Research and Development Program of China (2016YFA0600104), <a href="http://www.most.gov.cn/">http://www.most.gov.cn/</a>; Sir Henry Dale Fellowship (Wellcome Trust / Royal Society Grant 204311/Z/16/Z), <a href="https://wellcome.ac.uk/">https://wellcome.ac.uk/</a>. This work was funded by the European Research Council under the European Union’s Seventh Framework Programme (FP7/2007-2013)/European Research Council grant agree- ment number 614725-PATHPHYLODYN ([<a href="https://erc.europa.eu">https://erc.europa.eu</a>]). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All data used in this study can be downloaded from the following organizations: ICAO (International Civil Aviation Organization; http://www.icao.int/Pages/default.aspx), United Nations Conference on Trade and Development (UNCTAD; http://unctadstat.unctad.org/EN/Index.html), UN World Population Prospects database (http://www.un.org/en/development/desa/population/), UN Statistics Division ([https://unstats.un.org&lt;https://unstats.un.org/&gt;).]https://unstats.un.org&lt;https://unstats.un.org/&gt;)." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All data used in this study can be downloaded from the following organizations: ICAO (International Civil Aviation Organization; <a href="http://www.icao.int/Pages/default.aspx">http://www.icao.int/Pages/default.aspx</a>), United Nations Conference on Trade and Development (UNCTAD; <a href="http://unctadstat.unctad.org/EN/Index.html">http://unctadstat.unctad.org/EN/Index.html</a>), UN World Population Prospects database (<a href="http://www.un.org/en/development/desa/population/">http://www.un.org/en/development/desa/population/</a>), UN Statistics Division ([<a href="https://unstats.un.org">https://unstats.un.org</a>&amp;lt;<a href="https://unstats.un.org/">https://unstats.un.org/</a>&amp;gt;).]<a href="https://unstats.un.org">https://unstats.un.org</a>&amp;lt;<a href="https://unstats.un.org/">https://unstats.un.org/</a>&amp;gt;).</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Dengue virus (DENV) is a growing threat to public health, with nearly 390 million infections every year worldwide, of which ~96 million are symptomatic [<a href="#pntd.0005694.ref001">1</a>,<a href="#pntd.0005694.ref002">2</a>]. An estimated 2.5 billion people are at risk of dengue infection [<a href="#pntd.0005694.ref003">3</a>]. Dengue is regarded as the world’s most important mosquito-borne viral disease and is endemic in more than 100 countries [<a href="#pntd.0005694.ref004">4</a>], with most disease burden limited to tropical and subtropical regions [<a href="#pntd.0005694.ref005">5</a>]. However, in 2014 an outbreak of dengue occurred in Japan for the first time in over 70 years. This occurred despite the country’s temperate climate, and viral phylogenetic analysis suggests that the Japanese outbreak resulted from international travel from Southeast Asia [<a href="#pntd.0005694.ref006">6</a>]. Furthermore, the number of reported dengue cases and outbreaks continues to increase. In 2014, a dengue outbreak affected several Asian countries, including China, Thailand, Vietnam and Japan [<a href="#pntd.0005694.ref007">7</a>,<a href="#pntd.0005694.ref008">8</a>].</p> <p>The majority of DENV infections are asymptomatic or cause a mild febrile disease known as dengue fever, while the more severe forms of dengue infection—dengue hemorrhagic fever (DHF) and dengue shock syndrome (DSS)—may be life threating with &amp;gt;20% mortality [<a href="#pntd.0005694.ref009">9</a>]. Epidemiological studies have indicated that DHF often occurs when a dengue-immune person acquires a second infection with a different DENV serotype [<a href="#pntd.0005694.ref010">10</a>,<a href="#pntd.0005694.ref011">11</a>], and it has been hypothesized that DHF/DSS may result from a process of antibody-dependent enhancement [<a href="#pntd.0005694.ref012">12</a>,<a href="#pntd.0005694.ref013">13</a>]. The geographical areas in which transmission of multiple dengue serotypes occurs has grown in recent years and the pattern of co-circulation is conspicuously different from that which prevailed decades ago [<a href="#pntd.0005694.ref014">14</a>,<a href="#pntd.0005694.ref015">15</a>]. Furthermore, a growing number of DHF/DSS cases in the last 50 years, especially in Asia, demonstrates the need for a better understanding of how DENV genetic diversity and transmission jointly shape dengue epidemics.</p> <p>The growing scale of human mobility, particularly through air transportation, underscores an increase of pathogen introductions into geographic areas suitable for transmission, potentially contributing to the emergence and the re-emergence of epidemics [<a href="#pntd.0005694.ref016">16</a>,<a href="#pntd.0005694.ref017">17</a>]. Recent examples include the SARS epidemic, novel influenza A virus strains, the Middle East respiratory syndrome (MERS) coronavirus, and Zika virus [<a href="#pntd.0005694.ref018">18</a>–<a href="#pntd.0005694.ref021">21</a>]. At local scales, endemic circulation of DENV strains can be driven by viral lineage introduction events that eventually lead to lineage replacement [<a href="#pntd.0005694.ref022">22</a>] and viral introductions are expected to increase with human mobility. Together, these observations suggest that increasing human mobility through air-traffic networks may impact the spread of pathogens at large geographical scales. Here we combine Asian socio-economic data, air and maritime transportation network data and viral genetic data to identify key drivers of the spread of DENV serotypes 1, 2 and 3 in Asia, and to reconstruct the patterns of virus circulation among Asian countries over the past half century.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="compilationofgeneticdatasets" title="sec"> <div class="title" tagx="title" title="title">
Compilation of genetic datasets</div> <p>Maximum phylogenetic information would be obtained by studying whole DENV genomes. However, available DENV whole genomes from Asia have insufficient coverage through time and space for reliable analysis, and most available sequences comprise partial or complete coding E gene sequences. In order to generate a data set with both acceptable phylogenetic diversity and spatiotemporal sampling, we used DENV E gene sequences in subsequent analyses. DENV (DENV-1 to DENV-3) envelope (E) gene sequences with known collection dates and locations of sampling in Asia were collected from GenBank. DENV-4 was not included in this study because too few samples were available (only 64 sequences from 11 countries; <a href="#pntd.0005694.g001">Fig 1</a>). The remaining strains comprised a total of 2,202 sequences sampled between 1956 and 2015, from 20 distinct countries or geographic regions (<a href="#pntd.0005694.g001">Fig 1</a>). Sequences were grouped by serotype and aligned separately using MAFFT [<a href="#pntd.0005694.ref023">23</a>]. Recombination was inspected using the methods implemented in RDP3 and SimPlot [<a href="#pntd.0005694.ref024">24</a>]. After removing duplicate and recombinant strains, the final data set contained 1,272 DENV-1 sequences, 628 DENV-2 sequences and 302 DENV-3 sequences. In the complete sequence dataset, some countries, such as Vietnam, Cambodia, Thailand, and Singapore, were over-represented. In order to control for possible bias from uneven sampling, we randomly subsampled the complete sequence datasets by location and sampling time. At most 10 sequences were sampled per country and per year in order to create a more equitable spatio-temporal sampling distribution. After sub-sampling, the total number of sequences analyzed here was 327 for DENV-1, 357 for DENV-2, and 202 for DENV-3, sampled over a total of 59 years (<a href="#pntd.0005694.s001">S1 Fig</a>). Details of the sequences in each data set, including information on the year of isolation, sampling location, and accession numbers, are provided in Supplementary Information (<a href="#pntd.0005694.s007">S1 Table</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Co-circulation of dengue virus serotypes in Asia.</div> <p>(A). Locations of available viral sequences (within regions defined by sampling effort) and airports. Circle areas are proportional to the number of genetic sequences from a particular geographic location and for a given serotype (dark grey: DENV-1, orange: DENV-2, green: DENV-3, and yellow: DENV-4). Grey dots represent the airports for which passenger flux data was used in the analysis. Sequences obtained from patients in Japan, 2014 are indicated by asterisk. (B) The upper panel shows the proportion of sequences from each serotype per year, while the lower panel shows the number of serotypes isolated per year in Asia. Co-occurrence of multiple serotypes in a single year has become increasingly frequent.</p>   </div> </div> <div class="time-scaledphylogenetictreereconstruction" title="sec"> <div class="title" tagx="title" title="title">
Time-scaled phylogenetic tree reconstruction</div> <p>For each serotype subsampled data set, we first estimated the correlation between root-to-tip genetic divergence and sequence sampling dates, using TempEst [<a href="#pntd.0005694.ref025">25</a>]. This preliminary analysis indicated a good temporal signal for all serotypes (<a href="#pntd.0005694.s002">S2 Fig</a>). To reconstruct past population dynamics, we used a coalescent-based Gaussian Markov random field (GMRF) method with the time-aware smoothing parameter [<a href="#pntd.0005694.ref026">26</a>], as implemented in BEAST v1.8.2 [<a href="#pntd.0005694.ref027">27</a>]. A GTR+I+Γ nucleotide substitution model and an uncorrelated lognormal relaxed molecular clock model were used, with a prior distribution for the evolutionary rate parameter set to a Γ distribution with shape = 0.001 and scale = 1000. The BEAGLE library was used to accelerate computation [<a href="#pntd.0005694.ref028">28</a>]. For each serotype, three independent analyses of 150 million generations were performed, sampling parameters and trees every 15,000 generations. Analyses were combined after the removal of a burn-in of 10–20% of the samples and were checked visually in Tracer v.1.5.</p> </div> <div class="estimatingviralmigrationthroughdiscretegeographiclocations" title="sec"> <div class="title" tagx="title" title="title">
Estimating viral migration through discrete geographic locations</div> <p>To reconstruct the spatial dynamics of DENV-1–3, we used a Bayesian Markov chain Monte Carlo (MCMC) phylogeographic discrete approach [<a href="#pntd.0005694.ref029">29</a>,<a href="#pntd.0005694.ref030">30</a>] that estimates the ancestral locations along each branch of a viral phylogeny, as implemented in BEAST v1.8.2 [<a href="#pntd.0005694.ref027">27</a>]. A GTR+ I+Γ nucleotide substitution model was used in this analysis. Three independent MCMC chains were run for 150 million states, sampling every 15,000 states after the removal of 10% burn-in. Maximum clade credibility (MCC) trees were summarized using TreeAnnotator and visualized using SPREAD [<a href="#pntd.0005694.ref031">31</a>].</p> <p>To provide a minimal set of location state changes that provide an adequate description of viral spread, we used Bayesian stochastic search variable selection (BSSVS) [<a href="#pntd.0005694.ref030">30</a>,<a href="#pntd.0005694.ref032">32</a>]. BSSVS uses Bayes factors (BF) and binary indicator variables (I) to identify statistically supported viral lineage movement routes. Values of BF &amp;gt; 6 and I &amp;gt; 0.5 were considered as denoting a significant migration pathway, where BF &amp;gt; 1,000 indicates decisive statistical support, 100 ≤ BF &amp;lt; 1,000 indicates very strong support, 30 ≤ BF &amp;lt; 100 indicates strong support, 10 ≤ BF &amp;lt; 30 indicates substantial support and 6 ≤ BF &amp;lt; 10 indicates support [<a href="#pntd.0005694.ref032">32</a>]. We next estimated the number of expected transitions among location-pairs ("Markov jum�? counts) along the phylogeny branches [<a href="#pntd.0005694.ref033">33</a>], which provided a quantitative measure of successful viral introductions among countries [<a href="#pntd.0005694.ref034">34</a>].</p> </div> <div class="airtransportationnetworkinasia" title="sec"> <div class="title" tagx="title" title="title">
Air transportation network in Asia</div> <p>We investigated the air transportation network in Asia using mobility data from ICAO (International Civil Aviation Organization; <a href="http://www.icao.int/Pages/default.aspx">http://www.icao.int/Pages/default.aspx</a>). This database contains the number of passengers traveling among 373 airports in Asia during the years 1982, and 1992–2012, and included all scheduled flights both for large and small aircraft. Country-level movement of passengers was obtained by aggregating airport-level movement for each country.</p> <p>To obtain insight into temporal changes across the air transportation network in Asia, directed and weighted air flow networks were constructed, with countries as the network nodes. Network edges were weighted by the number of air passengers connecting pairs of countries. Hubs within the Asian air transportation network were also identified using two network topology properties: degree centrality and betweenness centrality. Degree centrality measures the number of edges connected to a node. Here, the degree of a given country refers to the number of airlines linking to it in the airline network. Betweenness centrality depends on the proportion of shortest paths between all pairs of vertices that pass through a given node. Thus the betweenness centrality of a given country is a measure of the extent to which a country lies on routes between other countries in the airline network and is calculated as: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pntd.0005694.e001g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
w</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
s</div> <div class="mtext_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
f</div> <div class="mtext_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
e</div> <div class="mtext_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
≠</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
≠</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(1)</span></div> where <i>σ</i><sub>st</sub> is the total number of shortest paths from node <i>s</i> to node <i>t</i> and <i>σ</i><sub>st</sub> (k) is the number of those paths that pass through <i>k</i>.<p /> </div> <div class="maritimetransportationnetworkinasia" title="sec"> <div class="title" tagx="title" title="title">
Maritime transportation network in Asia</div> <p>To represent the shipping connectivity between pairs of countries, the liner shipping bilateral connectivity index (LSBCI) in Asia from 2008 to 2014 was obtained from United Nations Conference on Trade and Development (UNCTAD; <a href="http://unctadstat.unctad.org/EN/Index.html">http://unctadstat.unctad.org/EN/Index.html</a>), which captures the amount of goods transported between two countries. Container port throughput (CPT) for each Asian country was also obtained from UNCTAD.</p> </div> <div class="identifyingpotentialdeterminantsofdenvlineagemovement" title="sec"> <div class="title" tagx="title" title="title">
Identifying potential determinants of DENV lineage movement</div> <p>To investigate the potential predictors driving DENV-1, DENV-2 and DENV-3 spatial spread, we used a generalized linear model (GLM) extension to Bayesian phylogeographic inference [<a href="#pntd.0005694.ref020">20</a>,<a href="#pntd.0005694.ref035">35</a>]. The phylogeographic GLM method parameterizes the continuous-time Markov chain (CTMC) matrix of among-location lineage migration parameters as a log-linear function of several potential predictors. Each function includes a coefficient <i>β</i> (in log space) and a binary indicator variable <i>δ</i>. BSSVS [<a href="#pntd.0005694.ref030">30</a>] was used to estimate the posterior probability that each predictor is included or excluded from the model. We considered several potential predictors of DENV diffusion. These included log-transformed and standardized measures of demographic and economic data, geographical distances among countries, absolute latitude, DENV sample sizes. Finally, predictors of human mobility from shipping connectivity and air transportation movement were included (averaged values were estimated using data in 1982, 2000, and 2012).</p> <p>Specific details of potential predictors are as follows: (i) average distance between two countries was estimated as the average of pairwise distances between all pairs of airports in the two countries, (ii) absolute latitudes for each country were calculated as the latitudes of their geometric center, (iii) passenger flow represents the number of passengers on fights between each pair of countries, (iv) LSBCI (linear shipping bilateral connectivity index) represents goods transported by the sea between pairs of countries, (v) population sizes and densities for each country in 2000 were obtained from the UN World Population Prospects database (<a href="http://www.un.org/en/development/desa/population/">http://www.un.org/en/development/desa/population/</a>), (vi) gross domestic product (GDP) for each country in 2000 was collected from the UN Statistics Division (<a href="https://unstats.un.org">https://unstats.un.org</a>) and (vii) average temperature and rainfall for each Asian country was extracted from the data set provided by WorldClim [<a href="#pntd.0005694.ref036">36</a>]. In addition, we included sample sizes (number of dengue sequences per country) for origin and destination locations as potential predictor variables, in order to test the impact of heterogeneous sampling.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="evolutionaryhistoryofdenguevirusserotypesinasia" title="sec"> <div class="title" tagx="title" title="title">
Evolutionary history of dengue virus serotypes in Asia</div> <p><a href="#pntd.0005694.g001">Fig 1A</a> depicts the geographic locations of the sequences used in this study, from a total of 20 distinct countries or geographic regions in Asia; most sequences were sampled in Southeast Asia. To reduce potential bias in the reconstruction of spatial spread that may arise from over-sampling particular locations, we subsampled the original data (see <a href="#sec002">Methods</a>). It is important to note that, in recent years, particularly since the 1980s, genetic evidence of multiple dengue virus sequences from a given location has increased (<a href="#pntd.0005694.g001">Fig 1B</a>) in Asian countries (<a href="#pntd.0005694.s003">S3 Fig</a>).</p> <p>Evolutionary analysis of 327 DENV-1 E gene sequences showed that Asian sequences fell into five distinct lineages, genotypes I—V. The 357 DENV-2 E gene sequences were classified into five genotypes: Asia I, Asia II, America/Asia, Cosmopolitan, and the America genotype. Further, the 202 DENV-3 E gene sequences comprised four genotypes, genotypes I and III—V. The estimated evolutionary rates of the E gene of each serotype, under the selected evolutionary model, were 7.50×10<sup>−4</sup> (95% highest posterior density, HPD: 6.73×10−4–8.28×10<sup>−4</sup>) substitutions per site per year (s/s/y) for DENV-1, 8.14×10<sup>−4</sup> (95% HPD: 7.38×10−4–8.94×10<sup>−4</sup>) s/s/y for DENV-2, and 7.96×10<sup>−4</sup> (95% HPD: 7.01×10−4–8.89×10<sup>−4</sup>) s/s/y for DENV-3. The corresponding estimates of the time of the most recent common ancestors (TMRCAs) of DENV-1–3 in Asia were 1911 (95% HPD: 1891–1928), 1901 (95%HPD 1868–1931, after removal of sylvatic lineages) and 1929 (95% HPD: 1916–1941) respectively.</p> </div> <div class="asianairlinenetworkgrowth" title="sec"> <div class="title" tagx="title" title="title">
Asian airline network growth</div> <p>Air transportation has historically exhibited significant growth [<a href="#pntd.0005694.ref037">37</a>]. We found that Asian air transportation grew substantially from 1980 onwards (<a href="#pntd.0005694.g002">Fig 2A</a>). We observe that several countries act as hubs in airline network, e.g. India, Singapore, and Thailand, while other countries exhibit growing network centrality over time, such as China and Malaysia (<a href="#pntd.0005694.g002">Fig 2B</a>). It is interesting to note that the sequences obtained from dengue patients in 2014 in Japan shared a very high identity with a sequence from China, which shares one of the busiest Asian flight routes with Japan (<a href="#pntd.0005694.g002">Fig 2B</a>). These observations prompt us to question the role of air passenger transportation in DENV spread in Asia.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Growing airline networks in Asia.</div> <p>(A) Number of nodes (countries belonging to the Asian air transport network), the number of airlines, and total passenger flow in the Asian air transport network between 1982–2012. Records are not available for the period 1983–1991, so dashed lines represented imputed missing values obtained by linear interpolation. (B) Passenger flux in Asia in 1982, 2000 and 2012. The size of each node corresponds to the degree centrality of the country in the airline network. Color coded lines represent the volume of air passenger flux. Node colors correspond to the countries indicated on the right-hand side of the figure. Black points indicate countries for which no virus sequence was included in the genetic analyses. China shares one of the busiest Asian flight routes with Japan, which is indicated by an asterisk.</p>   </div> </div> <div class="airtravelmaydrivethespatialspreadofdenvinasia" title="sec"> <div class="title" tagx="title" title="title">
Air travel may drive the spatial spread of DENV in Asia</div> <p>To infer the contribution of candidate factors driving DENV diffusion in Asia we used a generalized linear model that simultaneously estimates ancestral geographic reconstruction and identifies the contribution of potential predictors of spatial spread [<a href="#pntd.0005694.ref020">20</a>,<a href="#pntd.0005694.ref035">35</a>]. Our results show that most candidate predictors of virus spread, such as geographic distance and demographic factors, are not significantly associated with viral spread (<a href="#pntd.0005694.g003">Fig 3</a>). However, we do find that air passenger flow is a dominant driver of DENV lineage movement and the inclusion of this factor in the model is supported for all three DENV serotypes. GDP at the location of origin and CPT (container port throughput) at the destination location are negatively associated with DENV-1 lineage movement in Asia, but not for the other two genotypes (<a href="#pntd.0005694.g003">Fig 3</a>). Sample sizes for each location are not associated with viral lineage movement, which suggests that our conclusions are not driven by sampling biases.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Predictors of DENV-1–3 diffusion in Asia.</div> <p>In the top panel bars show <i>δ</i>, an indicator variable that governs the probability of inclusion or exclusion of the predictor in the model. In the lower panel, points and error bars indicate the mean and the 95% Bayesian credible intervals, respectively, of the estimated conditional effect size of the GLM coefficients (<i>β</i>|<i>δ</i> = 1) on a log scale, for each predictor variable. <i>β</i> is the effective size of the predictor variable. GDP: Gross domestic product; LSBCI: linear shipping bilateral connectivity index; CPT: container port throughput.</p>   </div> </div> <div class="viralmigrationthroughdiscretegeographiclocations" title="sec"> <div class="title" tagx="title" title="title">
Viral migration through discrete geographic locations</div> <p>To understand the spatial circulation of DENV in Asia, we reconstructed the past spatial transmission patterns of serotypes DENV-1–3 over the study period, for each country from which sequences were sampled (<a href="#pntd.0005694.g004">Fig 4</a>). A Bayesian stochastic search variable selection procedure was employed to infer a minimum set of location exchange events, while a Markov jump (MJ) analysis was used to quantity viral lineage movement among pairs of locations (see <a href="#sec002">Methods</a>). The phylogeographic analysis in <a href="#pntd.0005694.g005">Fig 5</a> indicates several significant migration links (with BF support &amp;gt; 1000). For DENV-1 these are between Cambodia and Vietnam, Thailand and Laos, and Singapore and China. For DENV-2, the well supported links are between Thailand and Cambodia, China and Indonesia, and Indonesia and Singapore. Finally, for DENV-3 the strongly supported links are between India and Sri Lanka (BF &amp;gt; 800). We also inferred a number of other well supported movements among countries; a full list is provided in <a href="#pntd.0005694.s008">S2 Table</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility trees of E gene of dengue virus serotypes in Asia.</div> <p>(A) Phylogeographic molecular clock phylogeny of DENV-1 (n = 327), (B) DENV-2 (n = 357), (C) DENV-3 (n = 202, after removal of sylvatic lineages). Branches are colored according to most probable geographic location, as inferred using a Bayesian discrete non-reversible phylogeographic approach.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Strongly supported pathways of lineage movement and histograms of the total number of location state transitions of DENV-1–3 in Asia for 1956 to 2015.</div> <p>(A) Strongly supported state transitions, indicating migration of DENV-1–3 lineages among discrete locations. The size of points corresponds to the number of geographic connections. The colored lines represent statistical support for a given viral movement pathway. Only those viral lineage movements supported with a BF &amp;gt; 6 are shown. (B) Number of expected transitions into and out of each country per serotype. Error bars represent 95% BCIs.</p>   </div> <p>We further find that estimated number of viral lineage migrations (including both importations and exportations) for each country is associated with measures of centrality in the air transportation network (Pearson correlation: r = 0.45, <i>P</i> = 0.05, for degree centrality and state transitions; and r = 0.73, <i>P</i> &amp;lt; 0.01, for betweenness centrality and state transitions; <a href="#pntd.0005694.g006">Fig 6A</a>, <a href="#pntd.0005694.s004">S4 Fig</a>). However, countries do not all contribute equally to viral lineage dissemination. An analysis of the number of virus lineage movements and the air transportation network analysis for each country indicates increased lineage virus movements from Thailand, India, and Indonesia to other locations [<a href="#pntd.0005694.ref038">38</a>,<a href="#pntd.0005694.ref039">39</a>], and a trend towards viral lineage importation from other locations for Vietnam and China. Lineage import and export is approximately equal for Cambodia and Singapore (<a href="#pntd.0005694.g006">Fig 6B</a> and <a href="#pntd.0005694.s005">S5 Fig</a>). Given the betweenness centrality of Thailand and India in the regional airline network, these countries may act as net sources of viral lineages, while China, Cambodia, Indonesia and Singapore may establish strong links with multiple countries within the network.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0005694.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Centrality measures of the air passenger network and number of state transitions.</div> <p>(A) Scatterplot of centrality measures for the air travel network based on degree and betweenness. The "degree�? centrality of a given country (node) refers to the number of airlines linking it in the airline network, and the "betweenness�? centrality of a given country measures the extent to which a country lies on routes between other countries in airline network (see <a href="#sec002">Methods</a>). The size of each point corresponds to the number of expected Markov jump transitions (including both importations and exportations). (B) Net Markov jump counts, summed across all 3 DENV serotypes. For each country, we summarize the average net Markov jumps (jumps to—jumps from) and their 95% credible intervals. The estimates are ordered from the lowest to highest number of net jumps.</p>   </div> <p>There has been an increasing frequency of DENV lineage co-occurrence in China over time, concomitant with its increasing centrality in the Asian air transportation network (Figs <a href="#pntd.0005694.g002">2</a> &amp;amp; <a href="#pntd.0005694.g005">5</a>). In contrast, Vietnam, a dengue-endemic country, has a low frequency of viral lineage import and export with other locations (<a href="#pntd.0005694.g006">Fig 6B</a>), likely due to its lower network centrality. However, we find that some non-endemic countries that do not contribute significantly to viral lineage movement across Asia may still maintain high airline passenger flows, such as Japan, South Korea, and countries in Western Asia. This may be because such areas lack a climate suitable for widespread dengue virus transmission. Further we are not able to infer transmission between countries if no viral sequences are available for those locations. The results here do not appear to be driven by heterogeneous sampling; similar results were obtained using downsampled datasets with a maximum of 5 sequences per country per year (<a href="#pntd.0005694.s006">S6 Fig</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Our study provides a temporal description of the patterns of DENV spread across Asia. The spatial dynamics of dengue virus in Asia inferred here suggests that DENV genetic diversity in Asia is dynamic yet spatially structured, with frequent virus lineage movement among countries, and frequent co-circulation of dengue virus lineages. Our analysis suggests that the air transportation network has contributed to the spatial distribution of DENV serotypes in Asia. This can be explained by the movement of viraemic individuals; if the destination and timing of virus introductions coincides with a climatically suitable period then an epidemic can become established in a susceptible recipient population. Further, it is possible that the mobility of viraemic mosquitoes through air transportation networks is non-negligible [<a href="#pntd.0005694.ref040">40</a>]. For example, one study estimated that 8–20 <i>Anopheline</i> mosquitoes were imported into France per flight in one 3-week period in 1994 [<a href="#pntd.0005694.ref041">41</a>]. Previous studies have also shown that mosquito species, including <i>Aedes albopictus</i>, can survive long-haul flights [<a href="#pntd.0005694.ref042">42</a>,<a href="#pntd.0005694.ref043">43</a>]. Implementing disinfection is proving effective in vector control and disease prevention [<a href="#pntd.0005694.ref040">40</a>,<a href="#pntd.0005694.ref044">44</a>], highlighting the importance of air transportation in pathogen importation.</p> <p>The positive coefficient <i>β</i> for the air traffic predictor in our phylogenetic GLM model indicates that DENV lineage dispersal is associated with air traffic; this result is consistent with previous studies of DENV in Brazil, which concluded that air travel of humans and/or mosquitoes is associated with virus lineage movement [<a href="#pntd.0005694.ref022">22</a>]. Notably there is strong statistical support for the inclusion of the air travel predictor variable in the GLM models for all three DENV serotypes in our study. Further, we find that origin GDP and destination CPT (container port throughput) are negatively related to DENV diffusion for DENV-1; this result is possibly linked to better public health infrastructure and vector control programmes in wealthier countries [<a href="#pntd.0005694.ref045">45</a>]. However, there was no support for the inclusion of those two predictors in the analyses of DENV-2–3. This may stem from the larger number of sequences for DENV-1, or from differences in sampling distribution among countries, despite the fact we have used a subsampling procedure in an attempt to mitigate potential bias. Alternatively, observed among-genotype differences may be driven solely by stochastic variation in the process of spatial dissemination; for example, the well-supported migration links inferred using DENV-1–3 sequences were not identical (<a href="#pntd.0005694.g005">Fig 5</a>).</p> <p>Analysis of the air transportation network indicates differences in the roles of air transport nodes in the regional-scale dissemination of DENV lineages. Our data suggest that large transportation hubs account for most of the inferred virus lineage movements within their respective networks. For example, Thailand and India seem to act as central hubs of DENV lineage movement in Asia, demonstrating both inward/outward viral migrations with other countries in the region. In contrast, our results suggests that the low level of network centrality of Vietnam may explain its limited role in regional DENV geographical dissemination. Some countries, such as China and Malaysia, may have had an increasing role in DENV spread through time. We note that dengue is still considered to be an imported disease in mainland China [<a href="#pntd.0005694.ref046">46</a>]. It might be expected that some DENV endemic countries, with climatic suitability for dengue transmission and high betweenness-centrality in air transportation network, contribute to seeding Asian dengue epidemics. However, it has been shown that the risk factors for the persistence and transmission of dengue are complex [<a href="#pntd.0005694.ref047">47</a>], once the socioeconomic conditions were taken into account (as shown in <a href="#pntd.0005694.g003">Fig 3</a>).</p> <p>Previous studies based on simulation models have shown that global warming is likely to increase the area of land with a climate suitable for dengue virus transmission by altering the distribution of <i>Aedes aegypti</i>, the main mosquito vector of dengue [<a href="#pntd.0005694.ref048">48</a>,<a href="#pntd.0005694.ref049">49</a>]. This may result in more countries being at risk of dengue virus via the air transportation network, resulting in the potential for spread across even greater geographic scales. Temperature increases may lead to lengthened mosquito lifespans and shortened extrinsic incubation periods, which may result in more infected mosquitoes for a longer period of time [<a href="#pntd.0005694.ref050">50</a>,<a href="#pntd.0005694.ref051">51</a>].</p> <p>Our study has several limitations. First, we cannot discern the relative contribution of infected humans versus infected mosquitoes in the spread of DENV, which requires more detailed epidemiological data and pathogen genome sequences from both mosquitoes and patients in future studies. However the information provided by the genetic analyses here can be useful to parameterize future spatially-structured transmission models. For example, the structure and travel flux of the airline transport network, as well as the among-country rates of lineage movement from the phylogeographic model, could be used to inform future simulations [<a href="#pntd.0005694.ref020">20</a>]. Second, our findings are based on DENV E genes for Asian countries, as this is the only genome region for which sufficient numbers of sequences are available, yet complete virus genomes would provide better phylogenetic resolution [<a href="#pntd.0005694.ref052">52</a>,<a href="#pntd.0005694.ref053">53</a>]. Further, our analyses were limited to DENV in Asia and do not consider transmission to or from other regions. Finally, the uneven sampling of DENV sequences in Asia, especially in a few countries after 2000 (e.g. Vietnam and Singapore) necessitated the use of sub-sampling analyses that accounted for the number of samples per location. Results obtained from a more representative data set indicates that our main conclusions are robust to sample sizes.</p> <p>Future trends in global mobility could potentially accelerate the appearance and diffusion of DENV worldwide. Prevention and control of dengue epidemics requires a better understanding of its mode of geographical dissemination, especially for countries in the tropics. Our study highlights the importance of developing a multivalent DENV vaccine in order to cope with increasing frequency of DENV serotype co-occurrence. The potential impacts of vaccination on dengue epidemics in Asia should be considered [<a href="#pntd.0005694.ref054">54</a>]. Viral spatial dissemination, together with host age-structure and host-vector interactions are required in mathematical models to inform future vaccine deployment decisions.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Raw data and subsampled data of DENV sequences in each country in each year in Asia.</div> <p>The color spectrum shows the number of sequences from each year and location, from grey (low numbers of sequences) to dark red (high numbers of sequences). The raw data set (left column) contained 1272 DENV-1 sequences, 628 DENV-2 sequences and 302 DENV-3 sequences. After randomly subsampling by location and sampling time (right column) the total number of subsampled sequences analyzed were 327 for DENV-1, 357 for DENV-2, and 202 for DENV-3.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0005694.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Plots of the root-to-tip genetic distance against sampling time for different serotype subsample data sets (DENV-1, DENV-2, and DENV-3).</div> <p>TempEst was used to obtain exploratory regressions based on the maximum likelihood trees.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0005694.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Number of serotypes isolated in each country in each year for the raw data set (i.e. before subsampling).</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0005694.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
Scatter plot of "Markov jump�? counts, betweenness and degree of air flow network for each country.</div> <p>(TIFF)</p>  <div class="media" title="media"><a href="pntd.0005694.s004.tiff">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
Net Markov jump counts across for each DENV serotype.</div> <p>For each country, we summarize the average net Markov jumps (jumps to—jumps from) and their 95% credible intervals. The estimates are reordered from the lowest to highest net jumps.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0005694.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
Histograms of total number of state transitions in each country for each serotype under different subsampling methods.</div> <p>Error bars represent 95% highest posterior density intervals. Left panel: at most 10 sequences were sampled per country per year. Middle panel: at most 5 sequences were sampled per country per year. Right panel: scatter plot of state transitions of the two subsampled data sets.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pntd.0005694.s006.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Subsampled DENV E gene sequences in study.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="pntd.0005694.s007.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Strongly supported DENV migrations estimated from E gene segment.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="pntd.0005694.s008.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Prof. Guodong Liang for his helpful comments and suggestions.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pntd.0005694.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gething</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messina</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farlow</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">The global distribution and burden of dengue</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">496</span>: <span class="fpage" tagx="fpage" title="fpage">504</span>–<span class="lpage" tagx="lpage" title="lpage">507</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nature12060">10.1038/nature12060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563266">23563266</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gething</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messina</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownstein</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Refining the global spatial limits of dengue virus transmission by evidence-based consensus</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e1760</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0001760">10.1371/journal.pntd.0001760</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22880140">22880140</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luz</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanni</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medlock</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paltiel</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galvani</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Dengue vector control strategies in an urban setting: an economic modelling assessment</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">377</span>: <span class="fpage" tagx="fpage" title="fpage">1673</span>–<span class="lpage" tagx="lpage" title="lpage">1680</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0140-6736(11)60246-8">10.1016/S0140-6736(11)60246-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546076">21546076</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzman</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Dengue</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">385</span>: <span class="fpage" tagx="fpage" title="fpage">453</span>–<span class="lpage" tagx="lpage" title="lpage">465</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0140-6736(14)60572-9">10.1016/S0140-6736(14)60572-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25230594">25230594</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raghwani</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hang</span><span class="given-names" tagx="given-names" title="given-names">VT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hien</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Endemic dengue associated with the co-circulation of multiple viral lineages and localized density-dependent transmission</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1002064</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1002064">10.1371/journal.ppat.1002064</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21655108">21655108</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quam</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessions</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamaraj</span><span class="given-names" tagx="given-names" title="given-names">US</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocklöv</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilder-Smith</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Dissecting Japan's dengue outbreak in 2014</span>. <span class="source" tagx="source" title="source">Am J Trop Med Hyg</span><span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">409</span>–<span class="lpage" tagx="lpage" title="lpage">412</span>. <span class="comment">doi: <a href="https://doi.org/10.4269/ajtmh.15-0468">10.4269/ajtmh.15-0468</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26711518">26711518</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Surface water areas significantly impacted 2014 dengue outbreaks in Guangzhou</span>, <span class="source" tagx="source" title="source">China. Environ Res</span><span class="volume" tagx="volume" title="volume">150</span>: <span class="fpage" tagx="fpage" title="fpage">299</span>–<span class="lpage" tagx="lpage" title="lpage">305</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.envres.2016.05.039">10.1016/j.envres.2016.05.039</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27336234">27336234</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tseng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Severe dengue fever outbreak in Taiwan</span>. <span class="source" tagx="source" title="source">Am J Trop Med Hyg</span><span class="volume" tagx="volume" title="volume">94</span>: <span class="fpage" tagx="fpage" title="fpage">193</span>–<span class="lpage" tagx="lpage" title="lpage">197</span>. <span class="comment">doi: <a href="https://doi.org/10.4269/ajtmh.15-0422">10.4269/ajtmh.15-0422</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26572871">26572871</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mongkolsapaya</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dejnirattisai</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X-n</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasanawathana</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tangthawornchaikul</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Original antigenic sin and apoptosis in the pathogenesis of dengue hemorrhagic fever</span>. <span class="source" tagx="source" title="source">Nat Med</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">921</span>–<span class="lpage" tagx="lpage" title="lpage">927</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nm887">10.1038/nm887</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12808447">12808447</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangkawibha</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rojanasuphot</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahandrik</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viriyapongse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jatanasen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">1984</span>) <span class="mixed-article-title" title="mixed-article-title">Risk factors in dengue shock syndrome: a prospective epidemiologic study in Rayong, Thailand. I. The 1980 outbreak</span>. <span class="source" tagx="source" title="source">Am J Epidemiol</span><span class="volume" tagx="volume" title="volume">120</span>: <span class="fpage" tagx="fpage" title="fpage">653</span>–<span class="lpage" tagx="lpage" title="lpage">669</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6496446">6496446</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzmán</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouri</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valdes</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bravo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Epidemiologic studies on Dengue in Santiago de Cuba, 1997</span>. <span class="source" tagx="source" title="source">Am J Epidemiol</span><span class="volume" tagx="volume" title="volume">152</span>: <span class="fpage" tagx="fpage" title="fpage">793</span>–<span class="lpage" tagx="lpage" title="lpage">799</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11085389">11085389</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span> (<span class="year" tagx="year" title="year">1979</span>) <span class="mixed-article-title" title="mixed-article-title">In vivo enhancement of dengue virus infection in rhesus monkeys by passively transferred antibody</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="volume" tagx="volume" title="volume">140</span>: <span class="fpage" tagx="fpage" title="fpage">527</span>–<span class="lpage" tagx="lpage" title="lpage">533</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/117061">117061</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Rourke</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span> (<span class="year" tagx="year" title="year">1977</span>) <span class="mixed-article-title" title="mixed-article-title">Antibody-enhanced dengue virus infection in primate leukocytes</span>. <span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">265</span>: <span class="fpage" tagx="fpage" title="fpage">739</span>–<span class="lpage" tagx="lpage" title="lpage">741</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/404559">404559</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzman</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Artsob</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchy</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Dengue: a continuing global threat</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">S7</span>–<span class="lpage" tagx="lpage" title="lpage">S16</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nrmicro2460">10.1038/nrmicro2460</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21079655">21079655</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twiddy</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Inferring the rate and time-scale of dengue virus evolution</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">122</span>–<span class="lpage" tagx="lpage" title="lpage">129</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519914">12519914</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatem</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Virus evolution and transmission in an ever more connected world</span>. <span class="source" tagx="source" title="source">Proc Biol Sci</span><span class="volume" tagx="volume" title="volume">282</span>: <span class="fpage" tagx="fpage" title="fpage">20142878</span><span class="comment">doi: <a href="https://doi.org/10.1098/rspb.2014.2878">10.1098/rspb.2014.2878</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26702033">26702033</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownstein</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfe</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandl</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Empirical evidence for the effect of airline travel on inter-regional influenza spread in the United States</span>. <span class="source" tagx="source" title="source">PLoS Med</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e401</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pmed.0030401">10.1371/journal.pmed.0030401</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16968115">16968115</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bogoch</span><span class="given-names" tagx="given-names" title="given-names">II</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">German</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Creatore</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Anticipating the international spread of Zika virus from Brazil</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">387</span>: <span class="fpage" tagx="fpage" title="fpage">335</span>–<span class="lpage" tagx="lpage" title="lpage">336</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0140-6736(16)00080-5">10.1016/S0140-6736(16)00080-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26777915">26777915</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helbing</span><span class="given-names" tagx="given-names" title="given-names">D</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">The hidden geometry of complex, network-driven contagion phenomena</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">342</span>: <span class="fpage" tagx="fpage" title="fpage">1337</span>–<span class="lpage" tagx="lpage" title="lpage">1342</span>. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1245200">10.1126/science.1245200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24337289">24337289</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielejec</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Unifying viral genetics and human transportation data to predict the global transmission dynamics of human influenza H3N2</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">e1003932</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1003932">10.1371/journal.ppat.1003932</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24586153">24586153</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arino</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raposo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sears</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Spread of a novel influenza A (H1N1) virus via global airline transportation</span>. <span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">361</span>: <span class="fpage" tagx="fpage" title="fpage">212</span>–<span class="lpage" tagx="lpage" title="lpage">214</span>. <span class="comment">doi: <a href="https://doi.org/10.1056/NEJMc0904559">10.1056/NEJMc0904559</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19564630">19564630</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunes</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">EC</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pantoja</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Air travel is associated with intracontinental spread of dengue virus serotypes 1–3 in Brazil</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e2769</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0002769">10.1371/journal.pntd.0002769</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24743730">24743730</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">772</span>–<span class="lpage" tagx="lpage" title="lpage">780</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lott</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moulton</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">RDP3: a flexible and fast computer program for analyzing recombination</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2462</span>–<span class="lpage" tagx="lpage" title="lpage">2463</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btq467">10.1093/bioinformatics/btq467</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20798170">20798170</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <span class="source" tagx="source" title="source">Virus Evolution</span><span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">vew007</span><span class="comment">doi: <a href="https://doi.org/10.1093/ve/vew007">10.1093/ve/vew007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774300">27774300</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minin</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloomquist</span><span class="given-names" tagx="given-names" title="given-names">EW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Smooth skyride through a rough skyline: Bayesian coalescent-based inference of population dynamics</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1459</span>–<span class="lpage" tagx="lpage" title="lpage">1471</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msn090">10.1093/molbev/msn090</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18408232">18408232</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">1969</span>–<span class="lpage" tagx="lpage" title="lpage">1973</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Many-core algorithms for statistical phylogenetics</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1370</span>–<span class="lpage" tagx="lpage" title="lpage">1376</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btp244">10.1093/bioinformatics/btp244</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19369496">19369496</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnes</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Ancient hybridization and an Irish origin for the modern polar bear matriline</span>. <span class="source" tagx="source" title="source">Curr Biol</span><span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">1251</span>–<span class="lpage" tagx="lpage" title="lpage">1258</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.cub.2011.05.058">10.1016/j.cub.2011.05.058</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21737280">21737280</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogeography finds its roots</span>. <span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e1000520</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pcbi.1000520">10.1371/journal.pcbi.1000520</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19779555">19779555</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielejec</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">SPREAD: spatial phylogenetic reconstruction of evolutionary dynamics</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">2910</span>–<span class="lpage" tagx="lpage" title="lpage">2912</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btr481">10.1093/bioinformatics/btr481</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21911333">21911333</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Boeckel</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Avian influenza H5N1 viral and bird migration networks in Asia</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">172</span>–<span class="lpage" tagx="lpage" title="lpage">177</span>. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.1405216112">10.1073/pnas.1405216112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25535385">25535385</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minin</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Counting labeled transitions in continuous-time Markov models of evolution</span>. <span class="source" tagx="source" title="source">J Math Biol</span><span class="volume" tagx="volume" title="volume">56</span>: <span class="fpage" tagx="fpage" title="fpage">391</span>–<span class="lpage" tagx="lpage" title="lpage">412</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00285-007-0120-8">10.1007/s00285-007-0120-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17874105">17874105</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vincent</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT-Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Spatial dynamics of human-origin H1 influenza A virus in North American swine</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1002077</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1002077">10.1371/journal.ppat.1002077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21695237">21695237</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">368</span>: <span class="fpage" tagx="fpage" title="fpage">20120196</span><span class="comment">doi: <a href="https://doi.org/10.1098/rstb.2012.0196">10.1098/rstb.2012.0196</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382420">23382420</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hijmans</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parra</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <span class="source" tagx="source" title="source">Int J Climatol</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1965</span>–<span class="lpage" tagx="lpage" title="lpage">1978</span>.</span></li> <li tag="ref"><a name="pntd.0005694.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sgouridis</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonnefoy</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansman</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Air transportation in a carbon constrained world: Long-term dynamics of policies and strategies for mitigating the carbon footprint of commercial aviation</span>. <span class="source" tagx="source" title="source">Transp Res Part A Policy Pract</span><span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">1077</span>–<span class="lpage" tagx="lpage" title="lpage">1091</span>.</span></li> <li tag="ref"><a name="pntd.0005694.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of the dynamics of viral infectious disease</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">540</span>–<span class="lpage" tagx="lpage" title="lpage">550</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nrg2583">10.1038/nrg2583</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19564871">19564871</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schreiber</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soh</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Genomic epidemiology of a dengue virus epidemic in urban Singapore</span>. <span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">83</span>: <span class="fpage" tagx="fpage" title="fpage">4163</span>–<span class="lpage" tagx="lpage" title="lpage">4173</span>. <span class="comment">doi: <a href="https://doi.org/10.1128/JVI.02445-08">10.1128/JVI.02445-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19211734">19211734</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatem</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hay</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">Estimating the malaria risk of African mosquito movement by air travel</span>. <span class="source" tagx="source" title="source">Malaria journal</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">57</span><span class="comment">doi: <a href="https://doi.org/10.1186/1475-2875-5-57">10.1186/1475-2875-5-57</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16842613">16842613</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gratz</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cocksedge</span><span class="given-names" tagx="given-names" title="given-names">W</span></span> (<span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">Why aircraft disinsection?</span><span class="source" tagx="source" title="source">Bull World Health Organ</span><span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">995</span>–<span class="lpage" tagx="lpage" title="lpage">1004</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10994283">10994283</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishihara</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">1945</span>) <span class="mixed-article-title" title="mixed-article-title">An experiment on the transportation of vector mosquitoes by aircraft (in japanese)</span>. <span class="source" tagx="source" title="source">Rinsho to Kenkyu</span><span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">44</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>.</span></li> <li tag="ref"><a name="pntd.0005694.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span> (<span class="year" tagx="year" title="year">1987</span>) <span class="mixed-article-title" title="mixed-article-title">Survival of insects in the wheel bays of a Boeing 747B aircraft on flights between tropical and temperate airports</span>. <span class="source" tagx="source" title="source">Bull World Health Organ</span><span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">659</span>–<span class="lpage" tagx="lpage" title="lpage">662</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3501345">3501345</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hutchinson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayoh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindsay</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Risk of airport malaria in the UK</span>. <span class="source" tagx="source" title="source">European Mosquito Bulletin</span><span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">12</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>.</span></li> <li tag="ref"><a name="pntd.0005694.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">W-P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiem</span><span class="given-names" tagx="given-names" title="given-names">VD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuzuki</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Population density, water supply, and the risk of dengue fever in Vietnam: cohort study and spatial analysis</span>. <span class="source" tagx="source" title="source">PLoS Med</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1001082</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pmed.1001082">10.1371/journal.pmed.1001082</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21918642">21918642</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Dengue is still an imported disease in China: a case study in Guangzhou</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">178</span>–<span class="lpage" tagx="lpage" title="lpage">190</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.meegid.2015.03.005">10.1016/j.meegid.2015.03.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25772205">25772205</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lozano-Fuentes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Use of mapping and spatial and space-time modeling approaches in operational control of Aedes aegypti and dengue</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e411</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0000411">10.1371/journal.pntd.0000411</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19399163">19399163</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hales</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Wet</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maindonald</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woodward</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Potential effect of population and climate changes on global distribution of dengue fever: an empirical model</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">360</span>: <span class="fpage" tagx="fpage" title="fpage">830</span>–<span class="lpage" tagx="lpage" title="lpage">834</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0140-6736(02)09964-6">10.1016/S0140-6736(02)09964-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12243917">12243917</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patz</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Focks</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jetten</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span> (<span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Dengue fever epidemic potential as projected by general circulation models of global climate change</span>. <span class="source" tagx="source" title="source">Environ Health Perspect</span><span class="volume" tagx="volume" title="volume">106</span>: <span class="fpage" tagx="fpage" title="fpage">147</span>–<span class="lpage" tagx="lpage" title="lpage">153</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9452414">9452414</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reeves</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reisen</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milby</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span> (<span class="year" tagx="year" title="year">1994</span>) <span class="mixed-article-title" title="mixed-article-title">Potential effect of global warming on mosquito-borne arboviruses</span>. <span class="source" tagx="source" title="source">J Med Entomol</span><span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">323</span>–<span class="lpage" tagx="lpage" title="lpage">332</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8057305">8057305</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiter</span><span class="given-names" tagx="given-names" title="given-names">P</span></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Climate change and mosquito-borne disease</span>. <span class="source" tagx="source" title="source">Environ Health Perspect</span><span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">141</span>–<span class="lpage" tagx="lpage" title="lpage">161</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11250812">11250812</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunes</span><span class="given-names" tagx="given-names" title="given-names">MRT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medeiros</span><span class="given-names" tagx="given-names" title="given-names">DBdA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva de Lima</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Phylogeography of dengue virus serotype 4, Brazil, 2010–2011</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">1858</span>–<span class="lpage" tagx="lpage" title="lpage">1864</span>. <span class="comment">doi: <a href="https://doi.org/10.3201/eid1811.120217">10.3201/eid1811.120217</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23092706">23092706</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernardin</span><span class="given-names" tagx="given-names" title="given-names">FJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Unifying the spatial epidemiology and molecular evolution of emerging epidemics</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">15066</span>–<span class="lpage" tagx="lpage" title="lpage">15071</span>. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.1206598109">10.1073/pnas.1206598109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22927414">22927414</a></span></span></li> <li tag="ref"><a name="pntd.0005694.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coudeville</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baurin</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">L’Azou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">B</span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Potential impact of dengue vaccination: Insights from two large-scale phase III trials with a tetravalent dengue vaccine</span>. <span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">6426</span>–<span class="lpage" tagx="lpage" title="lpage">6435</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.vaccine.2016.08.050">10.1016/j.vaccine.2016.08.050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27601343">27601343</a></span></span></li> </ul> </div> </div>  </body></html>