<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/07/03/284869.atom"/>
  <meta name="HW.pisa" content="biorxiv;284869v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A method for RNA structure prediction shows evidence for structure in lncRNAs"/>
  <meta name="DC.Identifier" content="10.1101/284869"/>
  <meta name="DC.Date" content="2018-07-03"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author’s permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="To compare the secondary structures of RNA molecules we developed the CROSSalign method. CROSSalign is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied CROSSalign to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure CROSSalign accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. CROSSalign can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage &lt;;http://service.tartaglialab.com//new_submission/CROSSalign&gt;;."/>
  <meta name="DC.Contributor" content="Riccardo Delli ponti"/>
  <meta name="DC.Contributor" content="Alexandros Armaos"/>
  <meta name="DC.Contributor" content="Stefanie Marti"/>
  <meta name="DC.Contributor" content="Gian Gaetano Tartaglia"/>
  <meta name="article:published_time" content="2018-07-03"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A method for RNA structure prediction shows evidence for structure in lncRNAs"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;To compare the secondary structures of RNA molecules we developed the &lt;;i&gt;;CROSSalign&lt;;/i&gt;; method. &lt;;i&gt;;CROSSalign&lt;;/i&gt;; is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied &lt;;i&gt;;CROSSalign&lt;;/i&gt;; to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure &lt;;i&gt;;CROSSalign&lt;;/i&gt;; accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. &lt;;i&gt;;CROSSalign&lt;;/i&gt;; can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage http://service.tartaglialab.com//new_submission/CROSSalign.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;284869v2"/>
  <meta name="citation_id" content="284869v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/284869v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/284869v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/284869v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869.full.pdf"/>
  <meta name="citation_doi" content="10.1101/284869"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="284869"/>
  <meta name="citation_author" content="Riccardo Delli ponti"/>
  <meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology"/>
  <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF)"/>
  <meta name="citation_author" content="Alexandros Armaos"/>
  <meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology"/>
  <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF)"/>
  <meta name="citation_author" content="Stefanie Marti"/>
  <meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology"/>
  <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF)"/>
  <meta name="citation_author" content="Gian Gaetano Tartaglia"/>
  <meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology"/>
  <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF)"/>
  <meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA)"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7524-6310"/>
  <meta name="twitter:title" content="A method for RNA structure prediction shows evidence for structure in lncRNAs"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="To compare the secondary structures of RNA molecules we developed the CROSSalign method. CROSSalign is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied CROSSalign to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure CROSSalign accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. CROSSalign can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage &lt;;http://service.tartaglialab.com//new_submission/CROSSalign&gt;;."/>
  <meta name="og-title" property="og:title" content="A method for RNA structure prediction shows evidence for structure in lncRNAs"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/284869v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="To compare the secondary structures of RNA molecules we developed the CROSSalign method. CROSSalign is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied CROSSalign to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure CROSSalign accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. CROSSalign can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage &lt;;http://service.tartaglialab.com//new_submission/CROSSalign&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-07-03"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A method for RNA structure prediction shows evidence for structure in lncRNAs | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-109519 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">To compare the secondary structures of RNA molecules we developed the 
     <em>CROSSalign</em> method. 
     <em>CROSSalign</em> is based on the combination of the Computational Recognition Of Secondary Structure (CROSS) algorithm to predict the RNA secondary structure at single-nucleotide resolution using sequence information, and the Dynamic Time Warping (DTW) method to align profiles of different lengths. We applied 
     <em>CROSSalign</em> to investigate the structural conservation of long non-coding RNAs such as XIST and HOTAIR as well as ssRNA viruses including HIV. In a pool of sequences with the same secondary structure 
     <em>CROSSalign</em> accurately recognizes repeat A of XIST and domain D2 of HOTAIR and outperforms other methods based on covariance modelling. 
     <em>CROSSalign</em> can be applied to perform pair-wise comparisons and is able to find homologues between thousands of matches identifying the exact regions of similarity between profiles of different lengths. The algorithm is freely available at the webpage 
     <a href="http://service.tartaglialab.com//new_submission/CROSSalign">http://service.tartaglialab.com//new_submission/CROSSalign</a>.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">Sequence similarity is often considered the key feature to investigate evolutionary conservation of coding transcripts [1]. Yet, knowledge of secondary structure provides important insights into the biological function of RNAs by allowing the study of physical properties, such as for instance molecular interactions [2]. In most cases, information about the RNA folding complements sequence analysis [3] and is useful to understand mechanisms of action: microRNA precursors, for example, are processed by DGCR8 only if properly folded in hairpin loop structures [4]. Similarly, the architecture of ribosomal RNAs evolves in a self-contained way through conservation of stem loops present in ancient species [5,6], indicating distinct requirements for structural elements.</p>
    <p id="p-4">Long non-coding RNAs (lncRNAs) are regarded as a mystery in terms of sequence and structural conservation [7]. The vast majority of lncRNAs evolve under little or no selective constraints, undergo almost no purifying selection, show poor expression, and do not have often easily identifiable orthologues [7,8]. Indeed, the average sequence homology of evolutionarily conserved lncRNAs is 20% between human and mouse and drops to 5% between human and fish [7]. Thus, primary structure does not provide relevant information to study lncRNA conservation and secondary structure could be used for better characterization. Similarly to lncRNAs, the transcriptomes of single-stranded RNA (ssRNA) viruses retain their fold even if sequences mutate rapidly [9], which indicates that secondary structure investigation could be key to revealing evolutionary properties.</p>
    <p id="p-5">To study the structural conservation of two RNA molecules, we developed the 
     <em>CROSSalign</em> method. 
     <em>CROSSalign</em>, available at our webpages 
     <a href="http://service.tartaglialab.com//new_submission/CROSSalign">http://service.tartaglialab.com//new_submission/CROSSalign</a>, is based on the combination of two methods: 1) Computational Recognition Of Secondary Structure (CROSS), which is an algorithm trained on experimental data to predict RNA secondary structure profiles without sequence length restrictions and at singlenucleotide resolution [10]; 2) the Dynamic Time Warping (DTW) algorithm to assess the similarity of two profiles of different lengths [11]. DTW flexibility allows managing profiles of different lengths without having to sacrifice computational time.
    </p>
    <p id="p-6">We applied 
     <em>CROSSalign</em> on lncRNAs of different species, compared it with covariation models, as well as on ssRNA viruses. 
     <em>CROSSalign</em> is able to find structural homologues among millions of possible matches identifying structural domains with great accuracy.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <p id="p-7">To test the performances and functionality of CROSS combined with DTW (
     <strong>Supplementary Figure. 1</strong> and 
     <strong>2</strong>), we selected a dataset of 22 structures for which crystallographic (exact base pairing between nucleotides) and Selective 2’-hydroxyl acylation analyzed by primer extension (SHAPE; chemical probing of flexible regions used to assess whether a nucleotide is in double- or single-stranded state) data are available [12]. Using DTW, we calculated the structural distances between all possible pairs in the dataset considering crystallographic (dots and parentheses were transformed into binary code) data as well as 1) SHAPE profiles (Area Under the ROC Curve AUC of 0.76, Positive Predictive Value PPV of 0.76 when compared to crystallographic data) and 2) CROSS profiles (AUC 0.72, PPV 0.74 when compared to crystallographic data, see also 
     <a href="http://service.tartaglialab.com/staticfiles/algorithms/cross/documentation.html#5">http://service.tartaglialab.com/staticfiles/algorithms/cross/documentation.html#5</a>).
    </p>
    <p id="p-8">Structural distances between CROSS profiles show higher correlations to the distances between the respective profiles from crystallographic data (Pearson’s correlation of 0.91) than structural distances between SHAPE profiles (
     <strong>
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A, 1B</a>
     </strong>, correlation of 0.50). Moreover, CROSS shows better performances than algorithms such as RNAstructure [13,14] and RNAfold [12] (respective correlations 0.71 and 0.47 with crystals; 
     <strong>Supplementary Figure. 3</strong> and 
     <strong>4</strong>).
    </p>
    <p id="p-9">Sequence similarity analysis (computed with EMBOSS; see 
     <strong>Material and Methods</strong>) shows comparable correlations with structural distances calculated with either CROSS or crystallographic profiles (respectively: 0.80, 0.83, 0.38 with crystallographic, CROSS and SHAPE data). While CROSS and crystallographic profiles identify specific clusters of RNA molecules with low sequence identity and high structural similarity (colored in red, orange and green according to difference confidence thresholds; 
     <strong>
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure. 1C, 1D, 1E</a>
     </strong>), SHAPE data cannot be used to identify these structures.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Validation of the CROSSalign method. (A) Structural distances correlation between CROSS and crystallographic profiles computed on 22 structures (standard-DTW). (B) Correlation between structural distances computed with SHAPE and crystallographic data on the same data (standard-DTW). (C) Correlation between structural distances (crystallographic profiles) and sequence similarity. Clusters of similar structures and different sequences (sequence similarity &lt;; 40%) are highlighted in brown (structural score &lt;; 0.2), orange (structural score &lt;; 0.1) and (red structural score &lt;; 0.05). (D) Correlation between structural distances (CROSS) and sequence similarity. The clusters previously identified for crystallographic data are shown in the plot. (E) Correlation between structural distances (SHAPE) and sequence similarity. In this case the clusters previously identified are disrupted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Validation of the CROSSalign method.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural distances correlation between CROSS and crystallographic profiles computed on 22 structures (standard-DTW). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Correlation between structural distances computed with SHAPE and crystallographic data on the same data (standard-DTW). (&lt;;strong&gt;;C&lt;;/strong&gt;;) Correlation between structural distances (crystallographic profiles) and sequence similarity. Clusters of similar structures and different sequences (sequence similarity &lt;; 40%) are highlighted in brown (structural score &lt;; 0.2), orange (structural score &lt;; 0.1) and (red structural score &lt;; 0.05). (&lt;;strong&gt;;D&lt;;/strong&gt;;) Correlation between structural distances (CROSS) and sequence similarity. The clusters previously identified for crystallographic data are shown in the plot. (&lt;;strong&gt;;E&lt;;/strong&gt;;) Correlation between structural distances (&lt;;strong&gt;;SHAPE&lt;;/strong&gt;;) and sequence similarity. In this case the clusters previously identified are disrupted.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F1.medium.gif" width="440" height="338"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F1.medium.gif" width="440" height="338"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Validation of the CROSSalign method.</span>
      <p id="p-10" class="first-child">(
       <strong>A</strong>) Structural distances correlation between CROSS and crystallographic profiles computed on 22 structures (standard-DTW). (
       <strong>B</strong>) Correlation between structural distances computed with SHAPE and crystallographic data on the same data (standard-DTW). (
       <strong>C</strong>) Correlation between structural distances (crystallographic profiles) and sequence similarity. Clusters of similar structures and different sequences (sequence similarity &lt;; 40%) are highlighted in brown (structural score &lt;; 0.2), orange (structural score &lt;; 0.1) and (red structural score &lt;; 0.05). (
       <strong>D</strong>) Correlation between structural distances (CROSS) and sequence similarity. The clusters previously identified for crystallographic data are shown in the plot. (
       <strong>E</strong>) Correlation between structural distances (
       <strong>SHAPE</strong>) and sequence similarity. In this case the clusters previously identified are disrupted.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">To further test the usefulness of 
     <em>CROSSalign</em>, we compared its output with that of CMfmder [15]. CMfmder is a method to compute multiple sequence alignments that exploits structural motifs for ranking (
     <strong>Materials and Methods, 
      <em>Comparisons with CMfinder</em>
     </strong>). We analyzed the largest multiple sequence alignments reported in the CMfmder test set (cobalamin, intron gp II, s box, lysine and histone 3) and used the minimal structural distance to assign the closest match to each transcript. Selecting equal-size groups (lowest and highest CMfinder scores), we measured 
     <em>CROSSalign</em> performances on CMfinder rankings, achieving an AUC of 0.80 (
     <strong>Supplementary Methods; Supplementary Figure 5A</strong>). We note that 
     <em>CROSSalign</em> has particularly strong performances on the largest dataset: cobalamin (71 sequences of 216nt; AUC of 0.95; 
     <strong>Supplementary Figure 5B</strong>).
    </p>
    <div id="sec-3" class="subsection">
     <h3>Ribosomal RNAs</h3>
     <p id="p-12">Ribosomal RNAs are considered one of the most ancient, structured and conserved classes of RNA molecules [5]. The first step to validate 
      <em>CROSSalign</em> was to search for structural similarities between the 
      <em>Small SubUnit</em> (SSU) of the rRNA of different bacteria (
      <strong>Pseudomonas aeruginosa, Escherichia coli, Bacillus subtilis, Deinococcus radiodurans</strong>). All the rRNAs show significant structural similarity (p-value &lt;;10
      <sup>−5</sup>) with a low structural distance (~0.08; 
      <strong>
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1A</a>
      </strong>). By contrast, shuffling one of the two sequences in 
      <em>CROSSalign</em> predictions results in non-significant scoring (p-values of 0.10 or higher; 
      <strong>
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1A</a>
      </strong>), indicating the importance of the sequence context in our calculations.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/536619/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/536619/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/536619" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Conservation of ribosomal structures.</span>
       <p id="p-13" class="first-child">(
        <strong>A</strong>) Above the diagonal: structural distances and p-values of bacterial SSUs; below the diagonal: structural distances and p-values upon shuffling one of the two bacterial sequences (Genbank/NCBI: 
        <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=J01859.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F07%2F03%2F284869.atom">J01859.1</a>, NR_026078, NR_074411.1, bacillus and NR_102783.2). (
        <strong>B</strong>) Structural distance and p-values of SSUs against E. coli (Genbank/NCBI: 
        <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=M10098.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F07%2F03%2F284869.atom">M10098.1</a>, NR_074375.1 and NR_132222.1). The shuffled sequence of Escherichia coli is used as a negative control. A coding human mRNA randomly selected from ENSEMBL with the same length of E. coli ribosome (ENSG00000002933) is used as an additional control to highlight the exclusive structural similarities between the different SSUs.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-14">
      <em>Pseudomonas aeruginosa and Escherichia coli</em> are the most similar SSUs (structural distance of 0.06; p-value &lt;;10
      <sup>−5</sup>). As the SSU of the rRNAs is thought to have evolved in a self-contained structure where the secondary structure of the ancient species is contained in the other ones [6], we searched the complete SSU of 
      <em>Escherichia coli in the SSU of other species such as Pyrococcus furiosus, Saccharomyces cerevisiae and Homo sapiens</em>. The results show a strong and significant structural similarity, in agreement with the theory of self-contained evolution (
      <strong>
       <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1B</a>
      </strong>). By contrast, comparison with randomized E. coli SSU or H. sapiens mRNA of the same length results in non-significant scores (p-values of 0.10 or higher 
      <strong>
       <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1B</a>
      </strong>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>XIST</h3>
     <p id="p-15">
      <em>XIST</em> is a lncRNA characterized by several repetitive domains showing different structural properties (
      <strong>
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure. 2A</a>
      </strong> and 
      <strong>
       <a id="xref-fig-2-2" class="xref-fig" href="#F2">2B</a>
      </strong>) [16]. The 5’ conserved region, named A-repeat (or RepA), is indispensable for gene-silencing and has been shown to be highly conserved in mammals. In mouse it consists of 7.5 copies (8.5 in humans) of 26-mers separated by U-rich linkers (
      <strong>
       <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>
      </strong>) [17]. In 2015 and 2017, two reports on the 
      <em>XIST</em> A-repeat structure were published [18,19], both making use of experimental techniques to infer 
      <em>XIST</em> structure. The A-repeat structures obtained are similar with strikingly identical stem-loop structures, both emerging from larger RNA bulges of repeats 3, 5, and 6 (for a comparison with CROSS predictions, see our previous manuscript [10]).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Predictions of XIST structure. (A) Secondary structure profile of murine XIST obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (B) Cumulative distribution function (CDF) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the Rep domains of XIST are reported on the curve. (C) ROC curve of CROSSalign to identify reverse engineered sequences with the same structure as RepA and a different sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Predictions of XIST structure.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Secondary structure profile of murine &lt;;em&gt;;XIST&lt;;/em&gt;; obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Cumulative distribution function (&lt;;strong&gt;;CDF&lt;;/strong&gt;;) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the Rep domains of &lt;;em&gt;;XIST&lt;;/em&gt;; are reported on the curve. (&lt;;strong&gt;;C&lt;;/strong&gt;;) ROC curve of &lt;;em&gt;;CROSSalign&lt;;/em&gt;; to identify reverse engineered sequences with the same structure as RepA and a different sequence.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F2.medium.gif" width="440" height="310"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F2.medium.gif" width="440" height="310"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Predictions of XIST structure.</span>
       <p id="p-16" class="first-child">(
        <strong>A</strong>) Secondary structure profile of murine 
        <em>XIST</em> obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (
        <strong>B</strong>) Cumulative distribution function (
        <strong>CDF</strong>) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the Rep domains of 
        <em>XIST</em> are reported on the curve. (
        <strong>C</strong>) ROC curve of 
        <em>CROSSalign</em> to identify reverse engineered sequences with the same structure as RepA and a different sequence.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">To evaluate the ability of 
      <em>CROSSalign</em> to recognize the structural content in RepA, we used 
      <em>RNAinverse</em> from Vienna suite [12] to generate 50 different sequences (average identity of 40%; 
      <strong>Supplementary Table 1</strong>) with the same structure as RepA (
      <strong>Materials and Methods</strong>; 
      <strong>
       <em>Reverse engineering: from structure to sequence</em>
      </strong>). The sequences were then divided into a reference (25 transcripts) and a positive (25 transcripts) set, and we built a list of negative cases by shuffling 25 times the original RepA. We then used 
      <em>CROSSalign</em> to compute all scores for the positive and negative set against the corresponding reference set and used the minimal structural distance to assign the closest match to each transcript. The strong performances obtained highlight the ability of 
      <em>CROSSalign</em> to identify structural similarities regardless of the sequence similarity (AUC of 0.85; 
      <strong>
       <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>
      </strong>).
     </p>
     <p id="p-18">As a further test of the usefulness of 
      <em>CROSSalign</em> we compared the above results with those obtained using 
      <em>CMsearch</em> from the Infernal-1.1.2 package, an algorithm based on a covariance model approach [20]. In this case, 
      <em>CMsearch</em> is not able to identify any match between either the positive or negative list and the reference sets (AUC of 0.5; 
      <strong>Materials and Methods</strong>; 
      <strong>
       <em>Comparisons with CMsearch</em>
      </strong>). These results indicate that 
      <em>CROSSalign</em> is able to recognize structural similarities between non-similar sequences, outperforming covariance-based approaches such as 
      <em>CMsearch</em>.
     </p>
     <p id="p-19">After this first validation step, we used 
      <em>CROSSalign</em> to study structural similarities of 
      <em>XIST</em> domain RepA in 10 different species [21]. Our analysis reveals that primates cluster close to human (
      <strong>Supplementary Figure 6A</strong>) while other species are more distant (
      <strong>Supplementary Figure. 6A</strong> and 
      <strong>7</strong>). By contrast, calculating sequence similarity with respect to human 
      <em>XIST</em> (computed with EMBOSS; see 
      <strong>Materials and Methods</strong>), we could not identify a specific cluster for primates (Supplementary Figure 6B). Thus, our results indicate that secondary structure shows a higher degree of conservation than sequence.
     </p>
     <p id="p-20">We then selected RepA of orangutan and searched for structural similarities within all human intergenic lncRNAs (lincRNAs 8176 sequences; ENSEMBLE 82). 
      <em>XIST</em> was ranked as the best significant match in the pool (structural distance 0.01; p-value &lt;; 10-6) and RepA was correctly identified (predicted coordinates: 328-764; 95% overlap with the query region; 
      <strong>
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>; Supplementary Table 2
      </strong>). Similar results were observed for baboon RepA (best result: 0.032; 86% overlap with the query region) and lemur RepA (best result: 0.075; p-value; 97% overlap with the query region), suggesting a strong structural conservation within primates (
      <strong>
       <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>
      </strong>). By contrast, human and mouse RepA showed a larger distance in terms of both structural and sequence similarity, which is in agreement with previous studies on lncRNA conservation [22].
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural conservation of XIST RepA within primates. (A) Structural distances of Orangutan RepA are computed against all human lincRNAs. The structural distance is calculated using OBE-DTW and plotted against sequence similarity. Orangutan RepA is identified as the best match (colored in red). (B) Structural distances of baboon RepA against all the lincRNAs of human. Baboon RepA is identified as the best match (colored in red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural conservation of XIST RepA within primates.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural distances of Orangutan RepA are computed against all human lincRNAs. The structural distance is calculated using OBE-DTW and plotted against sequence similarity. Orangutan RepA is identified as the best match (colored in red). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Structural distances of baboon RepA against all the lincRNAs of human. Baboon RepA is identified as the best match (colored in red).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F3.medium.gif" width="440" height="216"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F3.medium.gif" width="440" height="216"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Structural conservation of XIST RepA within primates.</span>
       <p id="p-21" class="first-child">(
        <strong>A</strong>) Structural distances of Orangutan RepA are computed against all human lincRNAs. The structural distance is calculated using OBE-DTW and plotted against sequence similarity. Orangutan RepA is identified as the best match (colored in red). (
        <strong>B</strong>) Structural distances of baboon RepA against all the lincRNAs of human. Baboon RepA is identified as the best match (colored in red).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-22">We used 
      <em>CROSSalign</em> to search for the human RepA within all mouse lncRNAs and identified 
      <em>XIST</em> as the 5
      <sup>th</sup> best hit (structural distance 0.085; p-value &lt;; 10
      <sup>−6</sup>; 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). In this case, the position of RepA was not correctly assigned (coordinates: 1030610698; 0% overlap) but the best match falls into the regulatory region of exon 7, and the structural relation between RepA and exon 7 has been reported [23]. Importantly, the correct coordinates of human RepA within mouse 
      <em>XIST</em> rank second in our analysis (structural distance 0.086; p-value &lt;; 10
      <sup>−6</sup>), while the best match is a miRNA-containing the gene Mirg (ENSMUSG00000097391) and the two secondary structure profiles show a strong correlation of 0.92 (
      <strong>
       <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>
      </strong>). Interestingly, even if little information is available onMirg, the transcript is prevalently expressed in the embryo [24]. This result unveils a previously unreported relationship between 
      <em>XIST</em> and Mirg, in which structural and functional homologies can be linked. Intriguingly, also the second best result, Rian (ENSMUSG00000097451), is expressed in embryo, while no information is available on the third and fourth hits (ENSMUSG00000107391 and ENSMUSG00000085312). We note that the five matches here are not listed in the top 20 hits obtained by analysis of sequence similarity (&lt;;34%).
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural similarities between human and mouse XIST RepA. (A) Structural similarities of human RepA against all mouse lincRNAs. The structural distance was calculated using CROSSalign (OBE-DTW) and plotted against sequence similarity. Human RepA is not the best match (5th best hit; colored in red). (B) Structural similarities of human RepA against all mouse lincRNAs using doublestranded nucleotides (nucleotide with CROSS score &lt;; 0 are set to 0). Human RepA is identified as the best match (colored in red), which highlights the importance of the structural content for the regulatory domains of the lncRNAs. (C) Secondary structure profile of human RepA, obtained as optimal path with OBE-DTW, compared with the best match in mouse lincRNAs (Mirg; ENSMUSG00000097391). The two secondary structure profiles show a strong a correlation (0.92)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural similarities between human and mouse XIST RepA.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural similarities of human RepA against all mouse lincRNAs. The structural distance was calculated using &lt;;em&gt;;CROSSalign&lt;;/em&gt;; (OBE-DTW) and plotted against sequence similarity. Human RepA is not the best match (5th best hit; colored in red). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Structural similarities of human RepA against all mouse lincRNAs using doublestranded nucleotides (nucleotide with CROSS score &lt;; 0 are set to 0). Human RepA is identified as the best match (colored in red), which highlights the importance of the structural content for the regulatory domains of the lncRNAs. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Secondary structure profile of human RepA, obtained as optimal path with OBE-DTW, compared with the best match in mouse lincRNAs (Mirg; ENSMUSG00000097391). The two secondary structure profiles show a strong a correlation (0.92).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F4.medium.gif" width="440" height="358"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F4.medium.gif" width="440" height="358"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">Structural similarities between human and mouse XIST RepA.</span>
       <p id="p-23" class="first-child">(
        <strong>A</strong>) Structural similarities of human RepA against all mouse lincRNAs. The structural distance was calculated using 
        <em>CROSSalign</em> (OBE-DTW) and plotted against sequence similarity. Human RepA is not the best match (5th best hit; colored in red). (
        <strong>B</strong>) Structural similarities of human RepA against all mouse lincRNAs using doublestranded nucleotides (nucleotide with CROSS score &lt;; 0 are set to 0). Human RepA is identified as the best match (colored in red), which highlights the importance of the structural content for the regulatory domains of the lncRNAs. (
        <strong>C</strong>) Secondary structure profile of human RepA, obtained as optimal path with OBE-DTW, compared with the best match in mouse lincRNAs (Mirg; ENSMUSG00000097391). The two secondary structure profiles show a strong a correlation (0.92).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-24">Our results suggest that the secondary structure of RepA is conserved among primates, and diverges between human and mouse. However, analyzing the information contained in structured nucleotides (i.e., nucleotides with CROSS score &lt;; 0 are set to 0) we could identify 
      <em>XIST</em> as the best match of human RepA within all mouse lncRNAs (structural distance 0.034; p-value &lt;; 10
      <sup>−6</sup>; 
      <strong>
       <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>
      </strong>). This result indicates that double-stranded regions are more conserved than single-stranded regions. In addition, we note that by sequence identity, 
      <em>XIST</em> ranks as the 14
      <sup>th</sup> hit of human RepA in all mouse lncRNAs, which indicates that methods based on sequence comparison show a significantly lower ability to identify structural homologues.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>HOTAIR</h3>
     <p id="p-25">HOTAIR shows a complex secondary structure, divided into four domains (D1, 2, 3, 4; 
      <strong>
       <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure. 5A</a>
      </strong> and 
      <strong>
       <a id="xref-fig-5-2" class="xref-fig" href="#F5">5B</a>
      </strong>) [25]. Experimentally it has been determined that more than 50% of the nucleotides are involved in base pairing (CROSS achieves an AUC &gt;; 0.80 in predicting its SHAPE profile; 
      <strong>Supplementary Figure. 8A</strong> and 
      <strong>8B</strong>) [26]. The region D2 is highly structured and consists of 15 helices, 11 terminal loops, and 4 junctions (three 5-way junctions and one 3-way junction) [26].
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Predictions of HOTAIR structure. (A) Secondary structure profile of the complete murine HOTAIR obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (B) Cumulative distribution function (CDF) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the D domains of HOTAIR are reported on the curve. (D) ROC curve of CROSSalign to identify reverse engineered sequences with the same structure as D2 and a different sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Predictions of HOTAIR structure.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Secondary structure profile of the complete murine &lt;;em&gt;;HOTAIR&lt;;/em&gt;; obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Cumulative distribution function (&lt;;strong&gt;;CDF&lt;;/strong&gt;;) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the D domains of &lt;;em&gt;;HOTAIR&lt;;/em&gt;; are reported on the curve. (&lt;;strong&gt;;D&lt;;/strong&gt;;) ROC curve of &lt;;em&gt;;CROSSalign&lt;;/em&gt;; to identify reverse engineered sequences with the same structure as D2 and a different sequence.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F5.medium.gif" width="440" height="346"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F5.medium.gif" width="440" height="346"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">Predictions of HOTAIR structure.</span>
       <p id="p-26" class="first-child">(
        <strong>A</strong>) Secondary structure profile of the complete murine 
        <em>HOTAIR</em> obtained using CROSS. Positive regions are to be considered double-stranded, negative regions single-stranded. (
        <strong>B</strong>) Cumulative distribution function (
        <strong>CDF</strong>) of the structural content of all the human lincRNAs predicted by CROSS. The structural contents of the D domains of 
        <em>HOTAIR</em> are reported on the curve. (
        <strong>D</strong>) ROC curve of 
        <em>CROSSalign</em> to identify reverse engineered sequences with the same structure as D2 and a different sequence.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-27">The D2 domain of 
      <em>HOTAIR</em> is predicted by 
      <em>CROSS</em> to be the most structured (
      <strong>
       <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>
      </strong>). We used the same reverse engineering process as for RepA to generate 50 different sequences (average identity of 40%; 
      <strong>Supplementary Table 1</strong>) with the same secondary structure as D2 (
      <strong>Materials and Methods</strong>; 
      <strong>
       <em>Reverse engineering: from structure to sequence</em>
      </strong>). Also in this case, 
      <em>CROSSalign</em> reports good performances (AUC of 0.72; 
      <strong>
       <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>
      </strong>) that are superior to those obtained using the covariance-based approach 
      <em>CMsearch</em> (AUC of 0.70; 
      <strong>Supplementary Figure 9</strong>; 
      <strong>Materials and Methods</strong>; 
      <strong>
       <em>Comparisons with CMsearch</em>
      </strong>) [20].
     </p>
     <p id="p-28">We then selected the D2 domain of HOTAIR to measure its conservation in 10 species [21] using 
      <em>CROSSalign</em> (
      <strong>Supplementary Figure 9A</strong>). As for 
      <em>XIST</em>, the structural distance analysis indicates that primates cluster close to human, and other species are more distant (
      <strong>Supplementary Figure 9B</strong>). Orangutan D2 was then searched for within all human lncRNAs, and HOTAIR was identified as the best match (structural distance 0.032; p-value &lt;; 10
      <sup>−6</sup>) with overlapping coordinates (nucleotides: 666-1191; 78% overlap with the query region; 
      <strong>
       <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>
      </strong>). Searching for mouse D2 within all human lncRNAs, HOTAIR was found as the best (0.092; p-value &lt;; 10-4) and matching position (nucleotides: 284-788; 57% overlap; 
      <strong>
       <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>
      </strong>). These results suggest that D2 secondary structure is not only conserved in primates but also in mouse.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural conservation of HOTAIR D2 in different species. (A) Structural similarities of orangutan D2 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against with the sequence similarity. The D2 of orangutan is identified as the best match (colored in red). (B) Structural similarities of human D2 against all mouse lincRNAs. Human D2 is identified as the best match (colored in red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural conservation of HOTAIR D2 in different species.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural similarities of orangutan D2 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against with the sequence similarity. The D2 of orangutan is identified as the best match (colored in red). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Structural similarities of human D2 against all mouse lincRNAs. Human D2 is identified as the best match (colored in red).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F6.medium.gif" width="440" height="215"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F6.medium.gif" width="440" height="215"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <span class="caption-title">Structural conservation of HOTAIR D2 in different species.</span>
       <p id="p-29" class="first-child">(
        <strong>A</strong>) Structural similarities of orangutan D2 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against with the sequence similarity. The D2 of orangutan is identified as the best match (colored in red). (
        <strong>B</strong>) Structural similarities of human D2 against all mouse lincRNAs. Human D2 is identified as the best match (colored in red).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-30">To further investigate the secondary structure of HOTAIR, we studied the structural conservation of the D4 domain (
      <strong>Supplementary Figure 10</strong>). As opposed to D2, D4 is predicted by CROSS to be poorly structured (
      <strong>
       <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>
      </strong>). Searching for orangutan D4 within all human lncRNAs yields HOTAIR as the best match (structural distance 0.023; p-value &lt;; 10
      <sup>−6</sup>) and the reported sequence position shows a sizeable overlap with the D4 domain in human HOTAIR (predicted coordinates: 1650-2291; overlap of 79%; 
      <strong>
       <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>
      </strong>). By contrast, when searching for mouse D4 within all human lncRNAs, HOTAIR shows poor ranking (1849
      <sup>th</sup>; structural distance 0.104; p-value = 0.061), which indicates little structural homology between human and mouse (
      <strong>
       <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>; Supplementary Figure 10
      </strong>).
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural conservation of HOTAIR D4 in different species. (A) Structural similarities of Orangutan D4 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against sequence similarity. Orangutan D4 is identified as the best match (colored in red). (B) Structural similarities of human against all mouse lincRNAs. Human D4 is not identified as the best match (1849th best hit; colored in red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural conservation of HOTAIR D4 in different species.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural similarities of Orangutan D4 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against sequence similarity. Orangutan D4 is identified as the best match (colored in red). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Structural similarities of human against all mouse lincRNAs. Human D4 is not identified as the best match (1849th best hit; colored in red).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F7.medium.gif" width="440" height="207"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F7.medium.gif" width="440" height="207"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7.</span>
       <span class="caption-title">Structural conservation of HOTAIR D4 in different species.</span>
       <p id="p-31" class="first-child">(
        <strong>A</strong>) Structural similarities of Orangutan D4 against all human lincRNAs. The structural distance was obtained using OBE-DTW and plotted against sequence similarity. Orangutan D4 is identified as the best match (colored in red). (
        <strong>B</strong>) Structural similarities of human against all mouse lincRNAs. Human D4 is not identified as the best match (1849th best hit; colored in red).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>HIV</h3>
     <p id="p-32">HIV is one of the most studied ssRNA viruses with a complex secondary structure [27] that is accurately predicted by CROSS [10] (see also 
      <a href="http://service.tartaglialab.com/static_files/algorithms/cross/documentation.html#4">http://service.tartaglialab.com/static_files/algorithms/cross/documentation.html#4</a>). As other organisms, HIV and ssRNA also contain non-coding regions [28].
     </p>
     <p id="p-33">We divided HIV into 10 non-overlapping regions of ~1000 nucleotides and searched each of them against a database of ssRNA viruses having as host human (292 cases, downloaded from NCBI; 
      <strong>Supplementary Table 3</strong>) to identify structurally similar domains. We found that coronavirus HKU and Simian-Human immunodeficiency SIV have the most significant matches with HIV (structural distance 0.078, p-value &lt;; 10
      <sup>−6</sup> for SIV; structural distance 0.093, p-value &lt;; 10
      <sup>−4</sup> for HKU). This finding is particularly relevant since SIV and HIV share many similarities in terms of pathogenicity and evolution [29]. Indeed, previous studies already reported a similarity in terms of secondary structure between HIV and SIV that is not explained by sequence similarity [30].
     </p>
     <p id="p-34">In addition, we found that the HIV 5’ region is structurally similar to a strain of Ebola virus (Tai Forest; 
      <strong>Supplementary Table 3</strong>). In agreement with this observation, previous studies indicate that HIV and Ebola have the same mechanisms of egress, taking contact with the cellular protein Tsg101 [31]. Moreover, HIV 5’ is the most conserved region in all ssRNA viruses (
      <strong>
       <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>
      </strong>). This result indicates that the secondary structure of this region is not only necessary for HIV encapsidation [32], but is also essential for the activity of other viruses.
     </p>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural analysis of the HIV transcriptome. (A) Structural conservation of HIV genome (divided into 10 not overlapping regions) compared with the complete genome of 292 ssRNA viruses. The region spanning the first 1000nt (including 5[[#x2019]] UTR) is the most conserved among all the viruses. (B) Structural distances of the complete HIV genome against the complete genomes of 4884 HIV strains. Using analysis of primary and secondary structures, we identified four main clusters (red, green, brown and yellow). Red and green boxes indicate strains whose structural difference cannot be identified through sequence analysis, while brown and red boxes as well as green and yellow boxes identify strains with similar structures and different sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1067674853" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural analysis of the HIV transcriptome.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Structural conservation of HIV genome (divided into 10 not overlapping regions) compared with the complete genome of 292 ssRNA viruses. The region spanning the first 1000nt (including 5[[#x2019]] UTR) is the most conserved among all the viruses. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Structural distances of the complete HIV genome against the complete genomes of 4884 HIV strains. Using analysis of primary and secondary structures, we identified four main clusters (red, green, brown and yellow). Red and green boxes indicate strains whose structural difference cannot be identified through sequence analysis, while brown and red boxes as well as green and yellow boxes identify strains with similar structures and different sequences.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F8.medium.gif" width="440" height="288"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F8.medium.gif" width="440" height="288"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/07/03/284869/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 8.</span>
       <span class="caption-title">Structural analysis of the HIV transcriptome.</span>
       <p id="p-35" class="first-child">(
        <strong>A</strong>) Structural conservation of HIV genome (divided into 10 not overlapping regions) compared with the complete genome of 292 ssRNA viruses. The region spanning the first 1000nt (including 5’ UTR) is the most conserved among all the viruses. (
        <strong>B</strong>) Structural distances of the complete HIV genome against the complete genomes of 4884 HIV strains. Using analysis of primary and secondary structures, we identified four main clusters (red, green, brown and yellow). Red and green boxes indicate strains whose structural difference cannot be identified through sequence analysis, while brown and red boxes as well as green and yellow boxes identify strains with similar structures and different sequences.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-36">We also compared structural distances and sequence similarities of all HIV strains (4804; see 
      <strong>Materials and Methods</strong>). We found two clusters (brown and red; 
      <strong>
       <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>
      </strong>) that are similar in terms of structure (~0.06 structural distance; p-value &lt;; 10
      <sup>−6</sup>) and sequence (80%-95% sequence similarity). Other clusters (red and green; 
      <strong>
       <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>
      </strong>) showed significant distance in structure (from 0.06 to 0.09 of structural distance; p-value &lt;; 10
      <sup>−6</sup>) that is not identifiable by sequence similarity (~85% sequence similarity). This result suggests that HIV could have evolved maintaining a similar sequence but different structures, as previously reported in literature [30].
     </p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Discussion</h2>
    <p id="p-37">We developed the 
     <em>CROSSalign</em> method based on the combination of the CROSS algorithm to predict the RNA secondary structure at single-nucleotide resolution [10] and the Dynamic Time Warping (DTW) algorithm to align profiles of different lengths [11]. DTW has been previously applied in different fields, especially pattern recognition and data mining [34,35], but has never been used to investigate structural alignments of RNA molecules. Since CROSS has no sequence length restrictions and shows strong performances on both coding and non-coding RNAs [10] the combination with DTW allows very accurate comparisons of structural profiles. Thermodynamic approaches, such as RNAstructure [14] or RNAfold [12], cannot be directly used for such a task since they are restricted on the sequence length [36].
    </p>
    <p id="p-38">We applied 
     <em>CROSSalign</em> to investigate the structural conservation of lncRNAs in different species and the complete genomes of ssRNA viruses. We found that the algorithm is able to find structural homologues between thousands of matches and correctly identifies the regions of similarity between profiles of different lengths. The results of our analysis reveal a structural conservation between known lncRNA domains including 
     <em>XIST</em> RepA (best hit out of 8176 cases; 95% overlap with the query region) and 
     <em>HOTAIR</em> D2 (best hit out of 8176 cases; 78% overlap with the query region), but also identify structural similarities between regulatory regions of HIV and other ssRNA viruses, opening new questions regarding similar mechanisms mediated by the secondary structure. Importantly, RepA and D2 profiles were accurately recognized in a pool of RNAs designed to have the same structure but different sequences. On these datasets multiple sequence alignments performed with 
     <em>CMsearch</em> showed lower performances, indicating that 
     <em>CROSSalign</em> predictions can be used to complement primary structure comparisons.
    </p>
    <p id="p-39">Our webserver is available at 
     <a href="http://service.tartaglialab.com//new_submission/CROSSalign">http://service.tartaglialab.com//new_submission/CROSSalign</a> (documentation and tutorials are at the webpages 
     <a href="http://service.tartaglialab.com/static_files/algorithms/CROSSalign/documentation.html">http://service.tartaglialab.com/static_files/algorithms/CROSSalign/documentation.html</a> and 
     <a href="http://service.tartaglialab.com/static_files/algorithms/CROSSalign/tutorial.html">http://service.tartaglialab.com/static_files/algorithms/CROSSalign/tutorial.html</a>) and allows to predict structural similarities between two (Standard, OBE, Fragmented modes) or more (Dataset, Custom dataset) RNA molecules. 
     <em>CROSSalign</em> can be interrogated to search for structural similarities between thousands of lncRNA molecules and identifies regions with similar structures using a specific DTW algorithm (open begins and ends OBE).
    </p>
    <p id="p-40">As shown in the examples presented, 
     <em>CROSSalign</em> is a versatile algorithm able to simplify the complex search for structural similarity among RNA molecules and shows great potential for the study of lncRNAs.
    </p>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-9" class="subsection">
     <h3>Prediction of the RNA secondary structure: CROSS</h3>
     <p id="p-41">Secondary structure profiles were generated using CROSS [10]. CROSS models have been previously trained on data from high-throughput experiments (PARS: yeast and human transcriptomes [3,37] and icSHAPE: mouse transcriptome [38]) as well as on low-throughput SHAPE [27] and high-quality NMR/X-ray data [39]. Since each approach has practical limitations and a different range of applicability, we combined the five models into a single algorithm, Global Score, which provides a consensus prediction.</p>
     <p id="p-42">The consensus model 
      <em>Global Score</em> was trained and tested on independent sets of NMR/X-ray structures (11’670 training fragments, 5’475 testing fragments [12,40], see also 
      <a href="https://github.com/stanti/shapebenchmark/tree/master/benchmarkdata">https://github.com/stanti/shapebenchmark/tree/master/benchmarkdata</a>). In the testing phase, single and double-stranded nucleotides were recognized with an AUC of 0.72 and a PPV of 0.74. Comparison with experimental SHAPE data shows similar performances (AUC of 0.76 and PPV of 0.76; see 
      <a href="http://service.tartaglialab.com/staticfiles/algorithms/cross/documentation.html#5">http://service.tartaglialab.com/staticfiles/algorithms/cross/documentation.html#5</a>) and all the details are reported in our previous publication [10]. In addition, as done with experimental SHAPE data, Global Score has been also used as a constraint in RNAstructure [13,14]. On the test set [12], Global Score was shown to increase the PPV of RNAstructure from 0.68 to 0.72, with remarkable improvements in 13 cases (from 0.44 to 0.72) and decreases the PPV in only three cases for which real SHAPE data does not improve performances. Moreover, using the partition function computed with RNAstructure, we previously calculated the AUC for each structure with and without CROSS constraints and observed an improvement from 0.81 to 0.86 when CROSS is integrated in the algorithm. We observed a similar trend using RNAfold [12] (the PPV increases from 0.67 to 0.70 using Global Score and the AUC remains at 0.85).
     </p>
     <p id="p-43">In the present study all the profiles were computed using the 
      <em>Global Score</em> module: nucleotides with a score higher than 0 are predicted to be double-stranded and structured, while nucleotides with a score lower than 0 are single-stranded. Since the algorithm has no sequence length restriction and shows strong performances on both coding and non-coding RNAs [10] it was combined with DTW for pairwise comparison of structural profiles. Thermodynamic approaches, such as RNAstructure [14] 
      <em>or RNAfold</em> [12], could not be directly used for such task since they are restricted on the sequence length.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Comparison of structural profiles: DTW</h3>
     <p id="p-44">To compare two CROSS profiles, we used the Dynamic Time Warping (
      <em>DTW</em>) algorithm available in the R package dtw [11]. The open begin and end (
      <em>OBE-DTW</em>) algorithm was employed to compare profiles of different lengths. Indeed, the standard 
      <em>DTW</em> method imposes the same begins and ends to the two profiles that are compared, while 
      <em>OBE-DTW</em> searches for the profile of shorter length within the other one. Accordingly, we used standard 
      <em>DTW</em> to compare profiles of similar lengths (i.e., one sequence is less than 3 times longer than the other), while 
      <em>OBE-DTW</em> was employed to search for modules within larger profiles (e.g., RepA within the complete 
      <em>XIST</em> sequence; 
      <em>XIST</em> is ~45 times longer than RepA).
     </p>
     <p id="p-45">The structural distance is computed with an asymmetric pattern and using the Manhattan distance, which is optimal for comparing profiles of different lengths. To avoid biases regarding the length of the profiles, the final structural distance is normalized for the lengths of both profiles using the internal function 
      <em>normalizedDistance</em>. We also tested different normalizations of DTW outputs (including normalization by length of the shorter or longer profile) and we found that the normalization based on the lengths of both profiles is optimal. The function 
      <em>index</em> was used to visualize the optimal path and to extract the matching coordinates between the two profiles.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Statistical analysis</h3>
     <p id="p-46">To compute the significance of a specific DTW score, we analyzed the statistical distributions generated using human lncRNAs of different lengths (200, 500, 1000, 5000 nucleotides). 100 molecules for each class were employed to compute the structural distance between the classes. The distributions are set as a reference to compute the p-values in new analyses (
      <strong>Supplementary Table 4</strong>)
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Datasets</h3>
     <p> </p>
     <ul class="list-unord " id="list-1">
      <li id="list-item-1">
       <p id="p-48">lncRNA sequences were downloaded from ENSEMBL 82 using Biomart and specifying lincRNAs, for a total of 4427 sequences for mouse and 8176 for human.</p>
      </li>
      <li id="list-item-2">
       <p id="p-49">The complete viral genomes were downloaded from NCBI selecting ssRNA viruses having as host human or primates (for SIV), for a total of 292 complete genomes.</p>
      </li>
      <li id="list-item-3">
       <p id="p-50">The complete rRNA sequences were downloaded from NCBI.</p>
      </li>
      <li id="list-item-4">
       <p id="p-51">RepA, D2 and D4 were selected from the data publicly available from the work of Rivas et al., 2016 [21]. To keep consistency between the results we tried to select the same species between the two sets of multialignments. When this was not possible we selected similar species (rat and mouse, orangutan and chimpanzee, lemur and sloth).</p>
      </li>
      <li id="list-item-5">
       <p id="p-52">The HIV strains were downloaded from HIV databases (
        <a href="https://www.hiv.lanl.gov/">https://www.hiv.lanl.gov/</a>), selecting only complete genomes for a total of 4804 sequences processable by CROSS.
       </p>
      </li>
     </ul>
     <p> </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Sequence alignment</h3>
     <p id="p-53">To compute the sequence alignments we used the browser version of EMBOSS-needleall, publicly available at 
      <a href="http://www.bioinformatics.nl/cgi-bin/emboss/needleall">http://www.bioinformatics.nl/cgi-bin/emboss/needleall</a>. The tool was used with standard settings to speed up the calculation. The sequence identity was retrieved from the corresponding field from EMBOSS multiple output.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Reverse engineering: from structure to sequence</h3>
     <p id="p-54">To study how sequence similarity is related to structural similarity we created different sequences with the same secondary structure as RepA (
      <em>XIST</em>) and D2 (
      <em>HOTAIR</em>). To generate the reference structure we used 
      <em>RNAfold</em> [41]. We then generated different sequences encoding for the same previously generated structure. For this task we used the command line version 
      <em>RNAinverse</em> from the Vienna suite [12]. The tool was launched using standard parameters to generate 50 sequences for each structure.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Comparisons with CMfinder</h3>
     <p id="p-55">We compared the structural distances provided by 
      <em>CROSSalign</em> with the multiple sequence alignment scores of CMfinder [15]. For this analysis we selected 5 of the most complex datasets (i.e. highest number and longest sequences) from the test set of 
      <em>CMfinder</em> (cobalamin, intron gp II, sbox, lysine and histone 3). To compare the pairwise distances (
      <em>CROSSalign</em>) with the multiple alignment scores (CMfinder) we computed all the distances within the datasets and selected the lowest distance for each transcript. From low-(median) to high-confidence (top and bottom 5%) CMfinder scores, we observed an increase in the performances of 
      <em>CROSSalign</em>, which indicates a good predictive power on the multiple alignment score.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Comparisons with CMsearch</h3>
     <p id="p-56">
      <em>CMsearch</em> is a method used to search a covariance model (CM) against a sequence database and provides a ranked list of the sequences with the most significant matches relative to the CM. Using the CMbuild package we built the CMs using as input the multiple alignments of the two reference sets (RepA and D2). The E-values of the CMs were obtained upon calibration with 
      <em>CMcalibrate</em>. The calibrated CMs were used to search for homologues in the positive and negative sets using the 
      <em>CMsearch</em> approach. By running 
      <em>RNAalifold</em>[42] to build a consensus secondary structure on aligned sequences[43], we did not obtain improvements for both RepA and D2 alignments.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Algorithm Description</h3>
     <div id="sec-18" class="subsection">
      <h4>Input</h4>
      <p>The user should paste one or two RNA sequences in FASTA format into the dedicated form, providing an email address (optional) to receive a notification when the job is completed. The algorithm can be launched in 4 different modes, each of them being a specific variation of the DTW algorithm (
       <strong>Supplementary Figure 1</strong>). 
      </p>
      <ul class="list-unord " id="list-2">
       <li id="list-item-6">
        <p id="p-58">The standard-DTW is recommended for comparing structures of RNAs with similar lengths (i.e., one sequence is less than 3 times longer than the other).</p>
       </li>
       <li id="list-item-7">
        <p id="p-59">
         <em>OBE-DTW</em> (open begins and ends) is a specific mode to search for a shorter profile within a longer one. This is the recommended mode when comparing profiles of very different sizes (i.e., one sequence is more than 3 times longer than the other). Please keep in mind that the sequence in the form of RNA input 1 will be searched for within RNA input 2, so the sequence in RNA input 1 should be shorter than the other.
        </p>
       </li>
       <li id="list-item-8">
        <p id="p-60">The fragmented OBE-DTW is a specific mode for searching for unknown secondary structure domains of one profile within the other. The secondary structure of RNA input 1 will be fragmented with a non-overlapping window of 200 nucleotides [optimal size to search secondary structure domains in large RNAs [44,45]] Each fragment of RNA input 1 will then be searched for within the other sequence. This approach is the recommended mode when the user is not interested in the global similarity between two secondary structure profiles, but wants to search for an unknown domain conserved in both sequences. A minimum length of 600 nucleotides is recommended for fragmentation.</p>
       </li>
       <li id="list-item-9">
        <p id="p-61">The 
         <em>dataset</em> mode allows the user to search for a single sequence within all the lincRNAs of a specific organism. The shorter profile of each pair will be searched for within the larger one following the 
         <em>OBE-DTW</em> procedure. The organisms available are Human, Mouse, Rat, Macaque and Zebrafish. The lincRNAs where downloaded using Biomart (Ensemble 82). New organisms and updated versions will be regularly added.
        </p>
       </li>
      </ul>
      <p> </p>
     </div>
     <div id="sec-19" class="subsection">
      <h4>Output</h4>
      <p id="p-62">We report the 
       <em>CROSSalign</em> score that measures the structural distance between two structures. The closer the score is to 0, the higher the similarity in terms of secondary structure. According to our statistical analysis, RNA molecules with a structural distance of 0.10 or higher are to be considered different in terms of secondary structure (see online 
       <strong>
        <em>Documentation</em>
       </strong>).
      </p>
      <p id="p-63">The main image shows the overall structural similarity of the two profiles employed to calculate 
       <em>CROSSalign</em> score (
       <strong>Supplementary Figure 2A</strong>). On the two axes the user will see the structural profiles obtained with CROSS for the two RNA sequences in input (score &gt;;0 means a double-stranded nucleotide; &lt;;0 single-stranded;). For a better visualization, the profiles are smoothed using a function previously defined [10]
      </p>
      <p id="p-64">The similarity is represented by the red path in the figure, obtained with the index function of the dtw package. The closer the path is to the diagonal, the more similar are the profiles. Vertical or horizontal paths are to be considered gaps, while diagonal paths highlight similar regions of the two profiles.</p>
      <p id="p-65">Since 
       <em>OBE-DTW</em> allows the identification of the optimal starting/ending points of a match, the optimal match is reported in terms of coordinates relative to the larger profile (RNA input 2). The main plot shows the CROSS profiles of the optimal matching region selected by the 
       <em>OBE-DTW</em> algorithm (
       <strong>Supplementary Figure 2B</strong>). In order to keep the gaps introduced by the 
       <em>OBE-DTW</em> algorithm, the two profiles are not smoothed.
      </p>
      <p id="p-66">The 
       <em>fragmented OBE-DTW</em> is a particular form of 
       <em>DTW</em> optimized to search for all the possible structural domains of a particular sequence within another one. The main output is a scrolling table reporting the structural score, the beginning of the match, the end of the match and the p-value (
       <strong>Supplementary Figure. 2C</strong>). All the values are computed with the procedure used for 
       <em>OBE-DTW</em>. The table can also be downloaded as a .txt file. The same output is used for the dataset mode, but in this case the table can only be downloaded.
      </p>
     </div>
    </div>
   </div>
   <div class="section COI-statement" id="sec-20">
    <h2 class="">COMPETING INTERESTS</h2>
    <p id="p-69">The authors declare no competing financial or non-financial interests</p>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">AUTHORS CONTRIBUTIONS</h2>
    <p id="p-70">GGT designed the work, RDP implemented the approach with SM. AA and RDP developed the server. GGT, RDP and SM wrote the paper. All authors reviewed the manuscript.</p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">AVAILABILITY OF DATA</h2>
    <p id="p-72">The code is publicly available under an open source license compliant with Open Source Initiative at 
     <a href="https://github.com/armaos/algorithm-CROSSalign">https://github.com/armaos/algorithm-CROSSalign</a>. The source code is deposited in a DOI-assigning repository 
     <a href="https://doi.org/10.5281/zenodo.1168294">https://doi.org/10.5281/zenodo.1168294</a>.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGMENTS</h2>
    <p id="p-67">We thank Fernando Cid and the members of Tartaglia’s group for useful comments.</p>
    <p id="p-68">The research leading to these results has received funding from the European Union Seventh Framework Programme (FP7/2007-2013), through the European Research Council, under grant agreement RIBOMYLOME_309545 (Gian Gaetano Tartaglia), and from the Spanish Ministry of Economy and Competitiveness (BFU2014-55054-P and BFU2017-86970-P). We also acknowledge support from AGAUR (2014 SGR 00685), the Spanish Ministry of Economy and Competitiveness, ‘Centro de Excelencia Severo Ochoa 20132017’ (SEV-2012-0208). We also thank the CRG fellowship to SM.</p>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
