<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/15/2020.03.13.991083.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.13.991083v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2" />
<meta name="DC.Identifier" content="10.1101/2020.03.13.991083" />
<meta name="DC.Date" content="2020-03-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic." />
<meta name="DC.Contributor" content="Jamie A. Kelly" />
<meta name="DC.Contributor" content="Jonathan D. Dinman" />
<meta name="article:published_time" content="2020-03-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.13.991083v1" />
<meta name="citation_id" content="2020.03.13.991083v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.13.991083" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.13.991083" />
<meta name="citation_author" content="Jamie A. Kelly" />
<meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland" />
<meta name="citation_author" content="Jonathan D. Dinman" />
<meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland" />
<meta name="citation_author_email" content="dinman@umd.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2402-9698" />
<meta name="citation_reference" content="The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2 (2020) Nat. Microbiol. 10.1038/s41564-020-0695-z" />
<meta name="citation_reference" content="Brian, D. A., and Baric, R. S. (2005) Coronavirus Genome Structure and Replication. in Coronavirus Replication and Reverse Genetics, pp. 1–30, Springer-Verlag, Berlin/Heidelberg, 287, 1–30" />
<meta name="citation_reference" content="Dinman, J. D. (2012) Mechanisms and implications of programmed translational frameshifting. Wiley Interdiscip.Rev.RNA. 3, 661–673" />
<meta name="citation_reference" content="Atkins, J. F., Loughran, G., Bhatt, P. R., Firth, A. E., and Baranov, P. V (2016) Ribosomal frameshifting and transcriptional slippage: From genetic steganography and cryptography to adventitious use. Nucleic Acids Res. 44, 7007–7078" />
<meta name="citation_reference" content="Jacks, T., and Varmus, H. E. (1985) Expression of the Rous Sarcoma Virus pol gene by ribosomal frameshifting. Science (80-.). 230, 1237–1242" />
<meta name="citation_reference" content="Brierley, I., Boursnell, M. E., Binns, M. M., Bilimoria, B., Blok, V. C., Brown, T. D., and Inglis, S. C. (1987) An efficient ribosomal frame-shifting signal in the polymerase-encoding region of the coronavirus IBV. EMBO J. 6, 3779–3785" />
<meta name="citation_reference" content="Plant, E. P. P., Rakauskaite, R., Taylor, D. R. R., and Dinman, J. D. D. (2010) Achieving a golden mean: mechanisms by which coronaviruses ensure synthesis of the correct stoichiometric ratios of viral proteins. J.Virol. 84, 4330–4340" />
<meta name="citation_reference" content="Plant, E. P., Sims, A. C., Baric, R. S., Dinman, J. D., and Taylor, D. R. (2013) Altering SARS coronavirus frameshift efficiency affects genomic and subgenomic RNA production. Viruses. 5, 279–94" />
<meta name="citation_reference" content="Rodnina, M. V, Korniy, N., Klimova, M., Karki, P., Peng, B.-Z., Senyushkina, T., Belardinelli, R., Maracci, C., Wohlgemuth, I., Samatova, E., and Peske, F. (2019) Translational recoding: canonical translation mechanisms reinterpreted. Nucleic Acids Res. 10.1093/nar/gkz783" />
<meta name="citation_reference" content="Puglisi, J. D., Wyatt, J. R., and Tinoco Jr., I. (1988) A pseudoknotted RNA oligonucleotide. Nature. 331, 283–286" />
<meta name="citation_reference" content="Plant, E. P., Perez-Alvarado, G. C., Jacobs, J. L., Mukhopadhyay, B., Hennig, M., and Dinman, J. D. (2005) A Three-Stemmed mRNA Pseudoknot in the SARS Coronavirus Frameshift Signal. PLoS.Biol. 3, 1012–1023" />
<meta name="citation_reference" content="Ramos, F. D., Carrasco, M., Doyle, T., and Brierley, I. (2004) Programmed −1 ribosomal frameshifting in the SARS coronavirus. Biochem.Soc.Trans. 32, 1081–1083" />
<meta name="citation_reference" content="Baranov, P. V, Henderson, C. M., Anderson, C. B., Gesteland, R. F., Atkins, J. F., and Howard, M. T. (2005) Programmed ribosomal frameshifting in decoding the SARS-CoV genome. Virology. 332, 498–510" />
<meta name="citation_reference" content="Cho, C.-P., Lin, S.-C., Chou, M.-Y., Hsu, H.-T., and Chang, K.-Y. (2013) Regulation of programmed ribosomal frameshifting by co-translational refolding RNA hairpins. PLoS One. 8, e62283" />
<meta name="citation_reference" content="Harger, J. W., and Dinman, J. D. (2003) An in vivo dual-luciferase assay system for studying translational recoding in the yeast Saccharomyces cerevisiae. RNA. 9, 1019–1024" />
<meta name="citation_reference" content="Jacobs, J. L., and Dinman, J. D. (2004) Systematic analysis of bicistronic reporter assay data. Nucleic Acids Res. 32, e160" />
<meta name="citation_reference" content="Ahn, D. G., Lee, W., Choi, J. K., Kim, S. J., Plant, E. P., Almazan, F., Taylor, D. R., Enjuanes, L., and Oh, J. W. (2011) Interference of ribosomal frameshifting by antisense peptide nucleic acids suppresses SARS coronavirus replication. Antivir. Res. 91, 1–10" />
<meta name="citation_reference" content="Ishimaru, D., Plant, E. P., Sims, A. C., Yount, B. L., Roth, B. M., Eldho, N. V, Pérez-Alvarado, G. C., Armbruster, D. W., Baric, R. S., Dinman, J. D., Taylor, D. R., Hennig, M., and Hennig, M. (2013) RNA dimerization plays a role in ribosomal frameshifting of the SARS coronavirus. Nucleic Acids Res. 41, 2594–608" />
<meta name="citation_reference" content="Park, S. J., Kim, Y. G., and Park, H. J. (2011) Identification of RNA pseudoknot-binding ligand that inhibits the −1 ribosomal frameshifting of SARS-coronavirus by structure-based virtual screening. J.Am.Chem.Soc. 133, 10094–10100" />
<meta name="citation_reference" content="Ritchie, D. B., Soong, J., Sikkema, W. K. A., and Woodside, M. T. (2014) Anti-frameshifting Ligand Reduces the Conformational Plasticity of the SARS Virus Pseudoknot. J. Am. Chem. Soc. 136, 2196–2199" />
<meta name="twitter:title" content="Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic." />
<meta name="og-title" property="og:title" content="Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.13.991083v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.13.991083v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.13.991083v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1197023" />
    <title>Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"s-02yUlBO0kktiUbhwUpNv-bbsVZAWUB_nKUSXA6wDA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1197023","apath":"\/biorxiv\/early\/2020\/03\/15\/2020.03.13.991083.atom","pisa":"biorxiv;2020.03.13.991083v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.13.991083v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.13.991083v1","title":"Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2","identifier":"node\/1197023"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+G+\\\\\\\/4shoozZ1JBd951UziapHNSCEDYbfwtbLLmGV5DUSmGFaFXL76xf70446sve29qOhcpHchnYH3Vxj\\\\\\\/7StBbFQ2tT4kmwWwrn\\\\\\\/FRL1zIekA1n315u+OM58CegwF7Jmoic\\\\\\\/VuECb5Dd50a1uoccSq7fa8bhZyRo6gPpWSRyheWZ5CTh\\\\\\\/k\\\\\\\/yIS+xchJ4jyUHyPkP27AuFev1tURZZhsm81jJhyMk2MTkojlCTDlmw8H7iA85ahfwtGHbf2YcwkDya1l0lFYbsVajlDlPBy99UpkrT7\\\\\\\/s2KLrf5+WokSKv+OOyyDB6qu6WmCavI5xSsaUWqyZKXUNV8qUBabZEVKteIrQXNphVZwPrSb+Wo3fo1FLU8vs31rhaICtZxqEbcWiK3S004at5eTQKJ2zqF4\\\\\\\/4lb5\\\\\\\/1KHRU8jsj5Bl\\\\\\\/3z39tY\\\\\\\/gZxo\\\\\\\/oL\\\\\\\/CjYATIL0YRW+UVQlIxq2OMcYUq9iah2fquNpQ\\\\\\\/cqPZxcS1yZWlgJ5j260NFheDKdqrv+JlnBfYKr\\\\\\\/z26mqIjbdaQ6vZOnoVfNhnNebB+d081H5jbeqWxfRf1TuJt1RobaOJBUHI8mbDilQFJ6zwxQ\\\\\\\/XTj0iDTX8OuAeUAbTgVXoKScwOJBk3cOe1dA6zg6nfSMGUIF5PnUeF8mxXhElWNk6GX3lBmneH+fA\\\\\\\/Vn3tpWemgT6i6PY0Db9mA+SI9xn52hwtjx+dwjDox3BisXHdC7Ti1x5qBvjXlB9z3CgBvNxbhtElVw4z4YzzD3CUQ0dQv5Yxdkwyqi+WXj\\\\\\\/4KnKmd9ivaRQhXB1DhOf80YHvjOhP5EpE1pTb\\\\\\\/qFLjhP2s9q4dQi0mT6VlONPhJYiWzrW+SfkEmFXiio\\\\\\\/hV7Fh7r2BNcnYnmRBL+PTiZX1I2YK+AbG0hcAj5nSHg8hFAslVdusPv8JOOHyA96JCknBeUtBvJ18ajarqXha9hR66doWfx26rf9uZgMMIIWV5LMPnLW5QuSNjLibMGWG\\\\\\\/ruZzvydJueQsAuSFEPlZg97\\\\\\\/7F6hw0lQZ1yz0nm89wwun\\\\\\\/qB6V1z626AHOk+rf5Py59ukctkj8zvUbt2Ei3fxAdwubGjvMw6RB3RAKRVXoItuSW957wSnJYI8C0abLghvbznZtFxGm9Ur2TfC6kt1ifatRK+B4rbsJZVy2A\\\\\\\/FqcH1nvUbuZ5GFbrCRQQU0ekFbW0SKFTkJIXXBTV38nLI+\\\\\\\/EOQvi1L8gdQ4x0cbO2ItzaSyT2dZsU8uDDS+8Vrl2xvTdaCOtgcemC7V3PyQYadg2UtYnKTro38PQ7inFIEaFe2PAN05jxm1YQ5xGHqJICGSYDiKDOaa2BIBFjqyPXKZ1DUNmgcSnBGTKjS1KM\\\u0022,\\\u0022iv\\\u0022:\\\u0022p5hMvWNH7kzvgzz6etNwXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002273a257106a8575150267f430fee30e3376a460f8bc03a81b3fbbd4f533ec5696\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.13.991083v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1197023 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.13.991083v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--5OHwTEL2CwmrAO5cgqTx-Ra37nPhmxJhWnbhkCz9uo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1197023" id="node1197023" data-pisa="biorxiv;2020.03.13.991083v1" data-pisa-master="biorxiv;2020.03.13.991083" data-apath="/biorxiv/early/2020/03/15/2020.03.13.991083.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jamie A.</span> <span class="nlm-surname">Kelly</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2402-9698" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jonathan D.</span> <span class="nlm-surname">Dinman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.13.991083 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1197023" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jamie A. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Cell Biology and Molecular Genetics, University of Maryland</span>, College Park MD 20742</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BA.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJamie%2BA.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan D. Dinman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Cell Biology and Molecular Genetics, University of Maryland</span>, College Park MD 20742</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BD.%2BDinman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dinman%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BD.%2BDinman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2402-9698" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan D. Dinman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dinman@umd.edu" class="" data-icon-position="" data-hide-link-title="0">dinman@umd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.13.991083v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1197023" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1197023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.991083v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1197023" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1197023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.991083v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1197023" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1197023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.991083v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1197023" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1197023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.13.991083v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1197023" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1197023/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic. COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">SARS-CoV2, the etiological agent of COVID-19, is a member of the coronavirus family (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Coronaviruses have (+) ssRNA genomes that harbor two long open reading frames (ORF) which occupy the 5’ ~ two-thirds of the genomic RNA (ORF1 and ORF2), followed by several ORFs that are expressed late in the viral replication cycle from subgenomic RNAs (sgRNAs) (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a></strong>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). In general the immediate early proteins encoded by ORF1 are involved in ablating the host cellular innate immune response, while the early proteins encoded in ORF2 are involved in genome replication and RNA synthesis. These functions include generation the minus-strand replicative intermediate, new plus-strand genomic RNAs, and subgenomic RNAs which mostly encode structural, late proteins. ORF2 is out of frame with respect to ORF1, and all coronaviruses utilize a molecular mechanism called programmed −1 ribosomal frameshifting (−1 PRF) as a means to synthesize the ORF2 encoded proteins (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). −1 PRF is a mechanism in which <em>cis-</em>acting elements in the mRNA direct elongating ribosomes to shift reading frame by 1 base in the 5’ direction [reviewed in (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>). The use of a −1 PRF mechanism for expression of a viral gene was first published in 1985 for the Rous sarcoma virus (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). A −1 PRF mechanism was shown to be required to translate ORF1ab in a coronavirus, Avain Infectious Bronchitis Virus (IBV), two years later (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). In coronaviruses −1 PRF functions as a developmental switch, and mutations and small molecules that alter this process have deleterious effects on virus replication (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural comparison of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF signals. A. CarB. Pairwise analysis of the two &#x2212;1 PRF signals. The attenuator elements and three-stemmed pseudoknot sequences are boxed as indicated. The U UUA AAC slippery site is underlined. C. Structure of the SARS-CoV &#x2212;1 PRF signal (11) is composed of the 5&#x2019; slippery site, a 6 nt spacer, and the three-stemmed pseudoknot stimulatory element. The single base difference in SARS-CoV-2 (red) maps to the short loop linking Stems 2 and 3. D. Comparison of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF attenuator elements. SARS-CoV-2 specific bases are indicated in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-562248863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structural comparison of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF signals.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; Car&lt;strong&gt;B.&lt;/strong&gt; Pairwise analysis of the two &#x2212;1 PRF signals. The attenuator elements and three-stemmed pseudoknot sequences are boxed as indicated. The U UUA AAC slippery site is underlined. &lt;strong&gt;C.&lt;/strong&gt; Structure of the SARS-CoV &#x2212;1 PRF signal (11) is composed of the 5&#x2019; slippery site, a 6 nt spacer, and the three-stemmed pseudoknot stimulatory element. The single base difference in SARS-CoV-2 (red) maps to the short loop linking Stems 2 and 3. &lt;strong&gt;D.&lt;/strong&gt; Comparison of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF attenuator elements. SARS-CoV-2 specific bases are indicated in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F1.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F1.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Structural comparison of the SARS-CoV and SARS-CoV-2 −1 PRF signals.</span><p id="p-4" class="first-child"><strong>A.</strong> Car<strong>B.</strong> Pairwise analysis of the two −1 PRF signals. The attenuator elements and three-stemmed pseudoknot sequences are boxed as indicated. The U UUA AAC slippery site is underlined. <strong>C.</strong> Structure of the SARS-CoV −1 PRF signal (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>) is composed of the 5’ slippery site, a 6 nt spacer, and the three-stemmed pseudoknot stimulatory element. The single base difference in SARS-CoV-2 (red) maps to the short loop linking Stems 2 and 3. <strong>D.</strong> Comparison of the SARS-CoV and SARS-CoV-2 −1 PRF attenuator elements. SARS-CoV-2 specific bases are indicated in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5">The −1 PRF signal can be broken down into three discrete parts: the “slippery site”, a linker region, and a downstream stimulatory region of mRNA secondary structure, typically an mRNA pseudoknot [reviewed in (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>)]. The primary sequence of the slippery site and its placement in relation to the incoming translational reading frame is critical: it must be N NNW WWZ (codons are shown in the incoming or 0-frame), where N is a stretch of three identical nucleotides, W is either AAA or UUU, and Z ≠ G. The linker region is less well-defined, but typically is short (1 – 12 nt long) and is thought to be important for determining the extent of −1 PRF in a virus-specific manner. The function of the downstream secondary structure is to induce elongating ribosomes to pause, a critical step for efficient −1 PRF to occur [reviewed in (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>)]. The generally accepted mechanism of −1 PRF is that the mRNA secondary structure directs elongating ribosomes to pause with its A- and P-site bound aminoacyl- (aa-) and peptidyl-tRNAs are positioned over the slippery site. The sequence of the slippery site allows for re-pairing of the tRNAs to the −1 frame codons after they “simultaneously slip” by one base in the 5’ direction along the mRNA. The subsequent resolution of the downstream mRNA secondary structure allows the ribosome to continue elongation of the nascent polypeptide in the new translational reading frame. The downstream stimulatory elements are most commonly H-type mRNA pseudoknots, so called because they are composed of two co-axially stacked stem-loops where the second stem is formed by base pairing between sequence in the loop of the first-stem loop, and additional downstream sequence (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). The SARS-CoV pseudoknot is more complex because it contains a third, internal stem-loop element (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Mutations affecting this structure decreased rates of −1 PRF, and had deleterious effects on virus propagation, thus suggesting that it may present a target for small molecule therapeutics (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>). In addition, the presence of a hairpin located immediately 5’ of the slippery site has been reported to regulate −1 PRF by attenuating its activity (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Here, we report on the −1 PRF signal from SARS-CoV2. The core −1 PRF signal is nearly identical to the SARS-CoV −1 PRF signal, containing only a single nucleotide difference, a C to A. This change maps to a loop region in the molecule that is not predicted to impact on the structure of the three-stemmed pseudoknot. The attenuator hairpin appears to be less-well conserved. However, functional analyses reveal that both elements promote nearly identical rates of −1 PRF, suggesting that this element has been structurally and functionally conserved.</p></div><div class="section" id="sec-2"><h2 class="">Results and Discussion</h2><div id="sec-3" class="subsection"><h3>Comparative structural analyses of the two −1 PRF signals</h3><p id="p-6">The core of the SARS-CoV −1 PRF signal begins with the U UUA AAC slippery site. This is followed by a 6 nt spacer region, which is then followed by the three-stemmed −1 PRF stimulating mRNA pseudoknot. A second regulatory element, called the attenuator hairpin, is located 5’ of the slippery site. Pairwise analysis of the SARS-CoV and SARS-CoV-2 frameshift signals revealed that the sequence of the attenuator hairpin was less well conserved than the frameshift-stimulating pseudoknot (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a></strong>). The structure of the SARS-CoV −1 PRF signal was previously determined to include a three-stemmed mRNA pseudoknot (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>). Using this as a guide, the single C to A base difference between the core SARS-CoV and SARS-CoV-2 −1 PRF signals maps to a loop that is not predicted to alter the structure of the −1 PRF stimulating element (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>) (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a></strong>). In contrast, the attenuator hairpin contains six differences in the nucleotide sequence between the two viruses, and the SARS-CoV-2 element is predicted to be less stable than its SARS-CoV counterpart (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1D</a></strong>). This suggests that the attenuation activity of this element is not as important as the ability to direct −1 PRF <em>per se</em>.</p></div><div id="sec-4" class="subsection"><h3>Comparative functional analyses of the two −1 PRF signals</h3><p id="p-7">Standard dual-luciferase assays were used to monitor −1 PRF activities of the two −1 PRF signals (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>) in HEK and HeLa cells. For both of the elements, −1 PRF activity was ~20% in HEK (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a></strong>) and ~30% in HeLa (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a></strong>). Amino acid sequence silent coding mutation of the U UUA AAC slippery sites to C CUC AAC (the incoming 0-frame codons are indicated by spaces) ablated −1 PRF activity in both cases, to less than 1% (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a></strong>). These findings support the hypothesis that structure and function of the −1 PRF signals has been conserved between the two viruses. Unfortunately, the dual luciferase reporter pJD2359 which was used to generate the other reporters used in this study does not contain the attenuator sequence. Thus, differences in this element were not able to be assayed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Functional characterization of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF signals. The efficiencies of &#x2212;1 PRF promoted by the wild-type (U UUA AAC) and silent slippery site mutant (C CUC AAC) &#x2212;1 PRF signals were assayed in HEK cells using dual-luciferase assays as previously described (15, 16). Assays were performed as three technical replicates, and were repeated a minimum three times apiece (biological replicates). A. Assays performed in HEK cells. B. Assays performed in HeLa cells. Error bars denote S.E.M." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-562248863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Functional characterization of the SARS-CoV and SARS-CoV-2 &#x2212;1 PRF signals.&lt;/span&gt; The efficiencies of &#x2212;1 PRF promoted by the wild-type (U UUA AAC) and silent slippery site mutant (C CUC AAC) &#x2212;1 PRF signals were assayed in HEK cells using dual-luciferase assays as previously described (15, 16). Assays were performed as three technical replicates, and were repeated a minimum three times apiece (biological replicates). &lt;strong&gt;A.&lt;/strong&gt; Assays performed in HEK cells. &lt;strong&gt;B.&lt;/strong&gt; Assays performed in HeLa cells. Error bars denote S.E.M.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F2.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F2.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.991083/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Functional characterization of the SARS-CoV and SARS-CoV-2 −1 PRF signals.</span><p id="p-8" class="first-child">The efficiencies of −1 PRF promoted by the wild-type (U UUA AAC) and silent slippery site mutant (C CUC AAC) −1 PRF signals were assayed in HEK cells using dual-luciferase assays as previously described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). Assays were performed as three technical replicates, and were repeated a minimum three times apiece (biological replicates). <strong>A.</strong> Assays performed in HEK cells. <strong>B.</strong> Assays performed in HeLa cells. Error bars denote S.E.M.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>A potential target for antiviral interventions</h3><p id="p-9">Mutations that alter rates of −1 PRF efficiency have been shown to be deleterious to replication of SARS-CoV (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>). A subsequent study described a series of antisense peptide nucleic acids that inhibited SARS-CoV mediated −1 PRF and virus replication (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), and mRNA pseudoknot dimerization was subsequently shown to be important for −1 PRF activity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). Separately, a high throughput screening approach was used to identify a small molecule that inhibited SARS-CoV mediated −1 PRF (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Interestingly, this was shown to limit the conformational plasticity of the −1 PRF stimulating pseudoknot, suggesting that targeting the conformational dynamics of pseudoknots may be an effective strategy for anti-viral drug design (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Given the &gt;1 year best-case scenario timeframe for development of a vaccine, similar approaches targeting −1 PRF may be of interest for controlling the SARS-CoV-2 pandemic.</p></div></div><div class="section" id="sec-6"><h2 class="">Experimental Procedures</h2><div id="sec-7" class="subsection"><h3>Identification of the SARS-CoV2 −1 PRF signal and computational methods</h3><p id="p-10">The SARS-CoV-2 −1 PRF signal was identified from the complete genome sequence (NCBI Ref Seq NC_045512.2). The EMBOSS Water pairwise alignment tool was used to identify sequences in the SARS-CoV-2 genome most similar to the SARS-CoV −1 PRF sequence. One hit was reported between bases 13461 and 13547 of SARS-CoV-2 that was 98.9% identical to the original SARS sequence. The SARS-CoV-2 sequence contains a single point mutation from C to A at base 13533. EMBOSS Water was used to generate pairwise alignments between sequences derived from SARS-CoV (GenBank entry <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_004718.3&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom">NC_004718.3</a> begin nt 13361, end nt 13478) and SARS-CoV-2 (Genbank entry <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom">NC_045512.2</a>, begin nt 13431, end nt 13547).</p></div><div id="sec-8" class="subsection"><h3>Plasmid construction and bacterial transformation</h3><p id="p-11">Reporter plasmids for SARS-CoV-2 were generated by oligonucleotide site directed mutagenesis using the NEB Q5 Site directed mutagenesis kit (cat. #E0554S). A single C to A point mutation was introduced into pJD2359 (SARS-CoV pSGDluc reporter plasmid) (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>) at base 1873 corresponding to the point mutation in the SARS-CoV-2 genome. Site directed mutagenesis primers were synthesized and purified by IDT. Products were transformed in to DH5α Escherichia coli cells (NEB) and spread onto LB agar plates containing 50μg/mL carbenicillin. Positive clones were verified by DNA sequencing (Genewiz). Additional frameshift reporter negative controls were constructed using site-directed mutagenesis. Silent mutations disrupting the −1 PRF slippery site from UUU UUA AAC to UUC CUC AAC (named ssM) were made using the NEB site directed mutagenesis kit.</p></div><div id="sec-9" class="subsection"><h3>Cell culture</h3><p id="p-12">Human embryonic kidney (HEK293T/17) (CRL-11268) and HeLa (CCL-2) cells were purchased from the American Type Culture Collection (Manassas, VA). HEK293T cells were maintained in Dulbecco’s modified Eagle medium (DMEM) (Fisher Scientific 10-013-CV) supplemented with 10% fetal bovine serum (FBS) (Fisher Scientific 26140-079), 1% GlutaMAX (35050-061), 1% nonessential amino acids (NEAA) (Fisher Scientific 11140-050), 1% HEPES buffer (Fisher Scientific 15630-030) and 1x Penicillin-streptomycin (Fisher Scientific 15140-122) at 37°C in 5% CO<sub>2</sub>. HeLa cells were maintained in DMEM supplemented with 10% FBS, 1% GlutaMAX and 1x Penicillin-streptomycin at 37°C in 5% CO<sub>2</sub>.</p></div><div id="sec-10" class="subsection"><h3>Plasmid transfection</h3><p id="p-13">HEK293T and HeLa cells were seeded at 4 × 10<sup>4</sup> cells per well into 24-well plates. Cells were transfected 24 hours after seeding with 500ng dual luciferase reporter plasmid using Lipofectamine3000 (Invitrogen L3000015) per manufacturer’s protocol.</p></div><div id="sec-11" class="subsection"><h3>Dual luciferase assays and calculation of −1 frameshifting efficiency</h3><p id="p-14">Frameshifting efficiency of the reporter plasmids were assayed as previously described (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>) using a dual luciferase reporter assay system kit (Promega). 24 hours post transfection, cells were washed with 1x phosphate-buffered saline (PBS) then lysed with 1x passive lysis buffer (E194A, Promega). Cell lysates were assayed in triplicate in a 96-well plate and luciferase activity was quantified using a GloMax microplate luminometer (Promega). Percent frameshift was calculated by averaging the three firefly or <em>Renilla</em> luciferase technical replicate reads per sample then forming a ratio of firefly to <em>Renilla</em> luminescence per sample. Each sample luminescence ratio was compared to a readthrough control set to 100%. The ratio of ratios for each sample is the percent frameshift for the sample. A minimum of three biological replicates were assayed for each sample, each of which were assayed in triplicate.</p></div></div><div class="section" id="sec-12"><h2 class="">Data Availability</h2><p id="p-15">Full datasets of −1 PRF assays are available upon request.</p></div><div class="section" id="sec-13"><h2 class="">Funding</h2><p id="p-17">This work was supported by Defense Threat Reduction Agency Grant HDTRA1-13-1-0005; NIGMS, National Institutes of Health (NIH) Grant R01 GM117177 (to J.D.D.), and NIH Institutional Training Grant 2T32AI051967-06A1 (to J. A. K.). “The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></div><div class="section COI-statement" id="sec-14"><h2 class="">Conflict of interest</h2><p id="p-18">The authors declare that they have no conflicts of interest with the contents of this article.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-16">We wish to thank Kevin Tu for his help with reagent preparation.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.1" data-doi="10.1038/s41564-020-0695-z"><div class="cit-metadata"><cite><span class="cit-article-title">The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span> (<span class="cit-pub-date">2020</span>) <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41564-020-0695-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-020-0695-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41564-020-0695-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.13.991083v1.2" data-doi="10.1007/3-540-26765-4_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brian</span>,  <span class="cit-name-given-names">D. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-chapter-title">Coronavirus Genome Structure and Replication</span>. in <span class="cit-source">Coronavirus Replication and Reverse Genetics</span>, pp. 1–30, <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin/Heidelberg</span>, <span class="cit-vol">287</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCoronavirus%2BReplication%2Band%2BReverse%2BGenetics%26rft.volume%253D287%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252F3-540-26765-4_1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/3-540-26765-4_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Mechanisms and implications of programmed translational frameshifting</span>. <abbr class="cit-jnl-abbrev">Wiley Interdiscip.Rev.RNA</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">661</span>–<span class="cit-lpage">673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.Rev.RNA%26rft.volume%253D3%26rft.spage%253D661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.4" data-doi="10.1093/nar/gkw530"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loughran</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhatt</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Baranov</span>,  <span class="cit-name-given-names">P. V</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Ribosomal frameshifting and transcriptional slippage: From genetic steganography and cryptography to adventitious use</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">7007</span>–<span class="cit-lpage">7078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw530%26rft_id%253Dinfo%253Apmid%252F27436286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27436286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.5" data-doi="10.1126/science.2416054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacks</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Varmus</span>,  <span class="cit-name-given-names">H. E.</span></span> (<span class="cit-pub-date">1985</span>) <span class="cit-article-title">Expression of the Rous Sarcoma Virus pol gene by ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">230</span>, <span class="cit-fpage">1237</span>–<span class="cit-lpage">1242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJacks%26rft.auinit1%253DT%26rft.volume%253D230%26rft.issue%253D4731%26rft.spage%253D1237%26rft.epage%253D1242%26rft.atitle%253DExpression%2Bof%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Bpol%2Bgene%2Bby%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2416054%26rft_id%253Dinfo%253Apmid%252F2416054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzAvNDczMS8xMjM3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMy45OTEwODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boursnell</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binns</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bilimoria</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blok</span>,  <span class="cit-name-given-names">V. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">T. D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">S. C.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">An efficient ribosomal frame-shifting signal in the polymerase-encoding region of the coronavirus IBV</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">3779</span>–<span class="cit-lpage">3785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D6%26rft.spage%253D3779%26rft_id%253Dinfo%253Apmid%252F3428275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3428275&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.7" data-doi="10.1128/JVI.02480-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rakauskaite</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D. R. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D. D.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Achieving a golden mean: mechanisms by which coronaviruses ensure synthesis of the correct stoichiometric ratios of viral proteins</span>. <abbr class="cit-jnl-abbrev">J.Virol.</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">4330</span>–<span class="cit-lpage">4340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.Virol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02480-09%26rft_id%253Dinfo%253Apmid%252F20164235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzkvNDMzMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE1LzIwMjAuMDMuMTMuOTkxMDgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.8" data-doi="10.3390/v5010279"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D. R.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Altering SARS coronavirus frameshift efficiency affects genomic and subgenomic RNA production</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">279</span>–<span class="cit-lpage">94</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D5%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv5010279%26rft_id%253Dinfo%253Apmid%252F23334702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v5010279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23334702&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.9" data-doi="10.1093/nar/gkz783"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M. V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korniy</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klimova</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karki</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">B.-Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Senyushkina</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belardinelli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maracci</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wohlgemuth</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samatova</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Translational recoding: canonical translation mechanisms reinterpreted</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkz783</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.10" data-doi="10.1038/331283a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puglisi</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyatt</span>,  <span class="cit-name-given-names">J. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tinoco</span>   <span class="cit-name-suffix">Jr.</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-article-title">A pseudoknotted RNA oligonucleotide</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">331</span>, <span class="cit-fpage">283</span>–<span class="cit-lpage">286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPuglisi%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D331%26rft.issue%253D6153%26rft.spage%253D283%26rft.epage%253D286%26rft.atitle%253DA%2Bpseudoknotted%2BRNA%2Boligonucleotide.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F331283a0%26rft_id%253Dinfo%253Apmid%252F3336440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/331283a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3336440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988L714000069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perez-Alvarado</span>,  <span class="cit-name-given-names">G. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hennig</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">A Three-Stemmed mRNA Pseudoknot in the SARS Coronavirus Frameshift Signal</span>. <abbr class="cit-jnl-abbrev">PLoS.Biol.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">1012</span>–<span class="cit-lpage">1023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS.Biol.%26rft.volume%253D3%26rft.spage%253D1012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.991083v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramos</span>,  <span class="cit-name-given-names">F. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrasco</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Programmed −1 ribosomal frameshifting in the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Biochem.Soc.Trans.</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1081</span>–<span class="cit-lpage">1083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.Soc.Trans.%26rft.volume%253D32%26rft.spage%253D1081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.13" data-doi="10.1016/j.virol.2004.11.038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baranov</span>,  <span class="cit-name-given-names">P. V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Howard</span>,  <span class="cit-name-given-names">M. T.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Programmed ribosomal frameshifting in decoding the SARS-CoV genome</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-vol">332</span>, <span class="cit-fpage">498</span>–<span class="cit-lpage">510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaranov%26rft.auinit1%253DP.%2BV.%26rft.volume%253D332%26rft.issue%253D2%26rft.spage%253D498%26rft.epage%253D510%26rft.atitle%253DProgrammed%2Bribosomal%2Bframeshifting%2Bin%2Bdecoding%2Bthe%2BSARS-CoV%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.11.038%26rft_id%253Dinfo%253Apmid%252F15680415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.11.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15680415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226918200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.14" data-doi="10.1371/journal.pone.0062283"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cho</span>,  <span class="cit-name-given-names">C.-P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">S.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">M.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>,  <span class="cit-name-given-names">H.-T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">K.-Y.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Regulation of programmed ribosomal frameshifting by co-translational refolding RNA hairpins</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">e62283</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62283%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062283%26rft_id%253Dinfo%253Apmid%252F23638024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23638024&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.15" data-doi="10.1261/rna.5930803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harger</span>,  <span class="cit-name-given-names">J. W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">An in vivo dual-luciferase assay system for studying translational recoding in the yeast Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">1019</span>–<span class="cit-lpage">1024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DHARGER%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D1019%26rft.epage%253D1024%26rft.atitle%253DAn%2Bin%2Bvivo%2Bdual-luciferase%2Bassay%2Bsystem%2Bfor%2Bstudying%2Btranslational%2Brecoding%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5930803%26rft_id%253Dinfo%253Apmid%252F12869712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMDE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMy45OTEwODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.16" data-doi="10.1093/nar/gnh157"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>,  <span class="cit-name-given-names">J. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Systematic analysis of bicistronic reporter assay data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">e160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DJacobs%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D32%26rft.issue%253D20%26rft.spage%253De160%26rft.epage%253De160%26rft.atitle%253DSystematic%2Banalysis%2Bof%2Bbicistronic%2Breporter%2Bassay%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnh157%26rft_id%253Dinfo%253Apmid%252F15561995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gnh157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15561995&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.17" data-doi="10.1016/j.antiviral.2011.04.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahn</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almazan</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Oh</span>,  <span class="cit-name-given-names">J. W.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Interference of ribosomal frameshifting by antisense peptide nucleic acids suppresses SARS coronavirus replication</span>. <abbr class="cit-jnl-abbrev">Antivir. Res.</abbr> <span class="cit-vol">91</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DAhn%26rft.auinit1%253DD.%2BG.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D10%26rft.atitle%253DInterference%2Bof%2Bribosomal%2Bframeshifting%2Bby%2Bantisense%2Bpeptide%2Bnucleic%2Bacids%2Bsuppresses%2BSARS%2Bcoronavirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2011.04.009%26rft_id%253Dinfo%253Apmid%252F21549154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2011.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21549154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292669900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.18" data-doi="10.1093/nar/gks1361"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ishimaru</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>,  <span class="cit-name-given-names">B. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roth</span>,  <span class="cit-name-given-names">B. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eldho</span>,  <span class="cit-name-given-names">N. V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pérez-Alvarado</span>,  <span class="cit-name-given-names">G. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armbruster</span>,  <span class="cit-name-given-names">D. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hennig</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hennig</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">RNA dimerization plays a role in ribosomal frameshifting of the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>, <span class="cit-fpage">2594</span>–<span class="cit-lpage">608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1361%26rft_id%253Dinfo%253Apmid%252F23275571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23275571&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.19" data-doi="10.1021/ja1098325"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">H. J.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Identification of RNA pseudoknot-binding ligand that inhibits the −1 ribosomal frameshifting of SARS-coronavirus by structure-based virtual screening</span>. <abbr class="cit-jnl-abbrev">J.Am.Chem.Soc.</abbr> <span class="cit-vol">133</span>, <span class="cit-fpage">10094</span>–<span class="cit-lpage">10100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DPark%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D133%26rft.issue%253D26%26rft.spage%253D10094%26rft.epage%253D10100%26rft.atitle%253DIdentification%2Bof%2BRNA%2Bpseudoknot-binding%2Bligand%2Bthat%2Binhibits%2Bthe%2B-1%2Bribosomal%2Bframeshifting%2Bof%2BSARS-coronavirus%2Bby%2Bstructure-based%2Bvirtual%2Bscreening.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja1098325%26rft_id%253Dinfo%253Apmid%252F21591761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja1098325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21591761&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.991083.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292715600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.991083v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soong</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sikkema</span>,  <span class="cit-name-given-names">W. K. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Woodside</span>,  <span class="cit-name-given-names">M. T.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Anti-frameshifting Ligand Reduces the Conformational Plasticity of the SARS Virus Pseudoknot</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc.</abbr> <span class="cit-vol">136</span>, <span class="cit-fpage">2196</span>–<span class="cit-lpage">2199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc.%26rft.volume%253D136%26rft.spage%253D2196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.13.991083v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.14.990747v1" title="Connexin 43 hemichannels mediate spatial and temporal disease spread in ALS" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 15, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.13.991083v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.13.991083v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.13.991083v1" class="active" data-icon-position="" data-hide-link-title="0">Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1197023" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HbJWupbHWubiSu_oV_hsNZljM9GFd_pnX1pColyFIaI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1197023" id="node1197023" data-pisa="biorxiv;2020.03.13.991083v1" data-pisa-master="biorxiv;2020.03.13.991083" data-seqnum="1197023" data-apath="/biorxiv/early/2020/03/15/2020.03.13.991083.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jamie A.</span> <span class="nlm-surname">Kelly</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jonathan D.</span> <span class="nlm-surname">Dinman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.991083; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.991083 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;title=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;title=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;text=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;title=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;t=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;title=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;title=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1197023" id="node1197023--2" data-pisa="biorxiv;2020.03.13.991083v1" data-pisa-master="biorxiv;2020.03.13.991083" data-seqnum="1197023" data-apath="/biorxiv/early/2020/03/15/2020.03.13.991083.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural and functional conservation of the programmed -1 ribosomal frameshift signal of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jamie A.</span> <span class="nlm-surname">Kelly</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jonathan D.</span> <span class="nlm-surname">Dinman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.991083; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.991083 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kelly%2C+Jamie+A.&amp;rft.au=Dinman%2C+Jonathan+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1197023/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1197023/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1197023/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1197023/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1197023/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1197023/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1197023/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1197023/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1197023/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1197023/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1197023/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1197023/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;count=horizontal&amp;via=&amp;text=Structural%20and%20functional%20conservation%20of%20the%20programmed%20-1%20ribosomal%20frameshift%20signal%20of%20SARS-CoV-2&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.991083v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.991083v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
