<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Microbiol</span><span class="iso-abbrev" title="iso-abbrev">Front Microbiol</span><span class="publisher-id" title="publisher-id">Front. Microbiol.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Microbiology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1664-302X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6838024">6838024</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fmicb.2019.02501">10.3389/fmicb.2019.02501</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span>: <span class="subject" title="subject">Original Research</span></div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Protein Interactions Network of Hepatitis E Virus RNA and Polymerase With Host Proteins</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanade</span><span class="given-names" tagx="given-names" title="given-names">Gayatri D.</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a><a href="#fn002"><sup>†</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/833497/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pingale</span><span class="given-names" tagx="given-names" title="given-names">Kunal D.</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a><a href="#fn002"><sup>†</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/833500/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karpe</span><span class="given-names" tagx="given-names" title="given-names">Yogesh A.</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/790510/overview" /></span></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Nanobioscience Group, Agharkar Research Institute</span><span class="addr-line" title="addr-line">Pune</span><span class="country" tagx="country" title="country">India</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Savitribai Phule Pune University</span><span class="addr-line" title="addr-line">Pune</span><span class="country" tagx="country" title="country">India</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Anna Kramvis, University of the Witwatersrand, Johannesburg, South Africa</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Milan Surjit, Translational Health Science and Technology Institute (THSTI), India; Siew Pheng Lim, Denka Life Innovation Research (DLIR), Singapore</p> </div> <div class="corresp" title="corresp">
*Correspondence: Yogesh A. Karpe, <span class="email" tagx="email" title="email">yakarpe@aripune.org</span></div> <div class="fn-type-other" title="other"> <p><sup>†</sup>These authors have contributed equally to this work</p> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Virology, a section of the journal Frontiers in Microbiology</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2501</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-13</span><span class="accepted" title="accepted">accepted: 2019-10-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Kanade, Pingale and Karpe</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Host-pathogen interactions are crucial for the successful propagation of pathogens inside the host cell. Knowledge of interactions between host proteins and viral proteins or viral RNA may provide clues for developing novel antiviral strategies. Hepatitis E virus (HEV), a water-borne pathogen that causes acute hepatitis in humans, is responsible for epidemics in developing countries. HEV pathology and molecular biology have been poorly explored due to the lack of efficient culture systems. A contemporary approach, to better understand the viral infection cycle at the molecular level, is the use of system biology tools depicting virus-host interactions. To determine the host proteins which participate in the regulation of HEV replication, we indentified liver cell proteins interacting with HEV RNA at its putative promoter region and those interacting with HEV polymerase (RdRp) protein. We employed affinity chromatography followed by liquid chromatography quadrupole time-of-flight mass spectrometry (LC-QTOF/MS) to identify the interacting host proteins. Protein-protein interaction networks (PPI) were plotted and analyzed using web-based tools. Topological analysis of the network revealed that the constructed network is potentially significant and relevant for viral replication. Gene ontology and pathway enrichment analysis revealed that HEV RNA promoter- and polymerase-interacting host proteins belong to different cellular pathways such as RNA splicing, RNA metabolism, protein processing in endoplasmic reticulum, unfolded protein response, innate immune pathways, secretory vesicle pathway, and glucose metabolism. We showed that hnRNPK and hnRNPA2B1 interact with both HEV putative promoters and HEV RdRp, which suggest that they may have crucial roles in HEV replication. We demonstrated <i>in vitro</i> binding of hnRNPK and hnRNPA2B1 proteins with the HEV targets in the study, assuring the authenticity of the interactions obtained through mass spectrometry. Thus, our study highlights the ability of viruses, such as HEV, to maneuver host systems to create favorable cellular environments for virus propagation. Studying the host-virus interactions can facilitate the identification of antiviral therapeutic strategies and novel targets.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
host-protein interactions</div> <div class="kwd" title="kwd">
protein interactions network</div> <div class="kwd" title="kwd">
system biology</div> <div class="kwd" title="kwd">
gene ontology analysis</div> <div class="kwd" title="kwd">
hepatitis E virus</div> <div class="kwd" title="kwd">
viral RNA</div> <div class="kwd" title="kwd">
viral polymerase</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Hepatitis E virus (HEV) is a hepevirus that is transmitted via contaminated drinking water to cause acute hepatitis in humans. Although prevalence of HEV has been mainly observed in developing countries, its spread has been reported in many industrialized countries across the globe in recent years. In infected adults, mortality rate due to HEV is up to ∼2% while in infected pregnant women, it increases up to 30% (<a href="#B21">Meng, 2010</a>; <a href="#B23">Nan and Zhang, 2016</a>). Due to the lack of efficient culture systems and robust animal models for HEV propagation, molecular mechanisms underlying the HEV lifecycle are not known (<a href="#B12">Himmelsbach et al., 2018</a>). Non-specific treatment with pegylated interferons along with ribavirin is recommended in rare instances for severe cases because specific antiviral drugs or vaccines against HEV are still not available worldwide (<a href="#B44">van de Garde et al., 2017</a>).</p> <p>Hepatitis E virus belongs to the Orthohepevirus genus, and its genome consists of positive sense, single-stranded RNA comprised of three open reading frames (ORFs) (<a href="#B17">Kenney and Meng, 2018</a>). First step of HEV replication cycle is the translation of ORF1 present on the positive sense genomic RNA to form non-structural polyprotein, which consists of functional domains of methyltransferase, protease, helicase and RNA-dependent RNA polymerase (RdRp). Two additional ORFs present in the sub-genomic intermediate RNA, ORF2 and ORF3, encode capsid protein and a small multifunctional phosphoprotein, respectively (<a href="#B17">Kenney and Meng, 2018</a>). HEV replication cycle involves the formation of a negative sense RNA complementary to the positive sense genomic RNA. HEV RdRp recognizes and starts the transcription at the promoter present at the 3′ end of the positive sense strand to form a complementary negative strand RNA. Negative strand RNA bears two putative promoters: one is the genomic promoter (3′ end of negative sense RNA) for the synthesis of positive sense RNA and the other is the sub-genomic promoter for the synthesis of sub-genomic RNA. Transcription at different viral promoters has to be regulated to maintain the correct stoichiometry of positive sense, negative sense and sub-genomic intermediate RNA. Host proteins, binding at viral promoters as components of the viral replicase complex, help in the regulation of molecular switches responsible for maintaining viral RNA stoichiometry, and their temporal synthesis.</p> <p>Viruses bear relatively compact genomes, encoding a limited number of proteins and, therefore, rely on host factors to establish replication in the infected cell. Being obligatory intracellular parasites, viruses have to subvert the biosynthetic pathways of the host cell. Constant interactions between the virus and its host during the process of co-evolution have shaped the anti-viral immune system of the host and, in turn, the capability of viruses to manipulate host control mechanisms to facilitate their propagation (<a href="#B38">Stebbing and Gazzard, 2003</a>; <a href="#B9">Fermin and Tennant, 2018</a>). Classical scientific approaches to understanding the molecular basis of such virus-host interactions involve analysis of individual gene or protein targets and study of their functional significance. However, these approaches have not been sufficient to address the challenges of the host-pathogen interface. System biology tools provide a multidimensional approach for a comprehensive view of the biological system at molecular network levels. High throughput genomic and proteomic studies, such as siRNA and microRNA screens, and microarrays have greatly expanded our understanding of virus-host networks. Advances in several tools for data acquisition, processing, integration and computation provide rapid, and promising strategies for the development of new therapies for infectious diseases (<a href="#B29">Peng et al., 2009</a>; <a href="#B1">Aderem et al., 2011</a>; <a href="#B49">Xue and Miller-Jensen, 2012</a>).</p> <p>Studies on several viruses reveal that viral proteins or viral RNA interact with host proteins to regulate viral replication. Previous studies on the HEV ORF1 and ORF2 interactome showed the involvement of factors associated with different biological processes, such as ubiquitin proteasome system, innate immunity and RNA metabolism (<a href="#B24">Ojha and Lole, 2016</a>; <a href="#B39">Subramani et al., 2018</a>). In another study by Paingankar et al. host factors have been found to interact with the untranslated region on HEV genomic RNA (<a href="#B26">Paingankar and Arankalle, 2015</a>). However, no conclusive studies have been carried out to analyze host factors present in HEV replicase complex. Also, host factors interacting with promoter sequences on HEV negative sense RNA have not been explored so far. We hypothesized that the set of host proteins interacting with HEV polymerase protein and HEV RNA at its promoter region must play crucial roles in tightly regulating the synthesis of viral RNAs. The host factors may form differential replicase complexes along with HEV RNA and polymerase protein at the promoter region. Therefore it was interesting to find out the proteins which bind at both the genomic and sub-genomic promoters on negative sense RNA of HEV and those which bind to only one specific promoter along with the RdRp. Proteins binding at both the promoters may act as primary transcription factors while, the differential proteins may guide RdRp for where to bind and which strand to synthesize at a given time. We thus believe that host factors interacting with HEV polymerase and promoters play crucial roles in regulating the molecular switches in HEV replication.</p> <p>In order to better understand the HEV-host interface, we identified liver cell proteins interacting with the HEV polymerase and HEV putative promoters and generated a protein-protein interaction network. We further utilized a bioinformatics approach to analyze these interaction networks and assess their significance. Our study identified host proteins related to cellular processes like RNA metabolism, unfolded protein response, stress granules, secretory vesicles, endoplasmic reticulum protein processing, and innate immune pathways. HNRNPK and HNRNPA2B1 proteins were found to be interacting with both HEV promoters and HEV RdRp. We demonstrated the <i>in vitro</i> binding of HEV promoters and HEV RdRp with HNRNPK and HNRNPA2B1, confirming the validity of interactions obtained by mass spectrometry.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="virusrepliconandcells" title="sec"> <div class="title" tagx="title" title="title">
Virus Replicon and Cells</div> <p>Infectious replicon of Sar55 strain of genotype 1 of HEV (pSK-HEV2) and a subclone of a human hepatoma cell line Huh7 S10-3 which is permissive for the replication of HEV infectious clone was obtained from Dr. Suzanne U. Emerson, NIH, Bethesda, MD, United States. Cells were maintained in DMEM GlutaMAX (Invitrogen) medium supplemented with 10% fetal bovine serum (FBS) (Invitrogen) and 100 U/ml penicillin and 100 mg/ml streptomycin (Sigma).</p> </div> <div class="constructionofrecombinantplasmids" title="sec"> <div class="title" tagx="title" title="title">
Construction of Recombinant Plasmids</div> <p>Coding sequence of HEV RNA dependent RNA polymerase (RdRp) was amplified from pSK-HEV2 replicon. RdRp coding sequence was cloned in pcDNA 3.1/myc-His (-) mammalian expression vector in such a way that it will be expressed as FLAG tagged RdRp at its N terminal. This clone has been designated as pcDNA_FLAG-RdRp. Primers used for the amplification have been listed in <a href="#T1">Table 1</a>.</p> <div class="table-wrap_UNKNOWN" id="T1" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>List of primers used in the study.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><b>Sr. No.</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Primer name</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Sequence (5′-3′)</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Amplified construct</b></td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">1</td> <td valign="top" align="left" rowspan="1" colspan="1">pTandem-F</td> <td valign="top" align="left" rowspan="1" colspan="1">GCTAGCCAAGCGCTTGGTTAAC</td> <td valign="top" align="left" rowspan="1" colspan="1">pTandem</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">2</td> <td valign="top" align="left" rowspan="1" colspan="1">pTandem-R</td> <td valign="top" align="left" rowspan="1" colspan="1">CCATGGTGGCATATCTCC</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">3</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRpFLAG_pTF</td> <td valign="top" align="left" rowspan="1" colspan="1">TTAAGAAGGAGATATGCCACCATGGCGCCACCATGGACTACAAAG</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRp</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">4</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRpFLAG_pTR</td> <td valign="top" align="left" rowspan="1" colspan="1">TGGTGATGGTGTGTCATTCCACCCGACACAG</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">5</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRpFLAG_ pTJncF</td> <td valign="top" align="left" rowspan="1" colspan="1">TCGGGTGGAATGACACACCATCACCACCATC</td> <td valign="top" align="left" rowspan="1" colspan="1">Junction region</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">6</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRpFLAG_ pTJncR</td> <td valign="top" align="left" rowspan="1" colspan="1">TTTGTTCCATGTTGTTTAAACTTTCAAAGGAAAACCAC</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">7</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPK_F</td> <td valign="top" align="left" rowspan="1" colspan="1">TTTCCTTTGAAAGTTTAAACATGGAAACTGAACAGCCAG</td> <td valign="top" align="left" rowspan="1" colspan="1">hnRNPK</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">8</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPK_R</td> <td valign="top" align="left" rowspan="1" colspan="1">TAACCAAGCGCTTGGCTAGCTTACAGATCCTCTTCTGAGATG</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">9</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPA2B1_F</td> <td valign="top" align="left" rowspan="1" colspan="1">TTTCCTTTGAAAGTTTAAACATGGAGAGAGAAAAGGAACAGTTC</td> <td valign="top" align="left" rowspan="1" colspan="1">hnRNPA2B1</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">10</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPA2B1_R</td> <td valign="top" align="left" rowspan="1" colspan="1">TCTGAGATGAGTTTTTGTTCGTATCGGCTCCTCCCACC</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">11</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRp_F</td> <td valign="top" align="left" rowspan="1" colspan="1">ATCCGAATTCATGGACTACAAAGACGATGACGACAAGGGTGGCGAAATTGGCCACCA</td> <td valign="top" align="left" rowspan="1" colspan="1">pcDNA_FLAG-RdRp</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">12</td> <td valign="top" align="left" rowspan="1" colspan="1">RdRp_R</td> <td valign="top" align="left" rowspan="1" colspan="1">CGGAGGGATCCTCATTCCACCCGACACAGAATTGA</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">13</td> <td valign="top" align="left" rowspan="1" colspan="1">G promoter_F</td> <td valign="top" align="left" rowspan="1" colspan="1">GCAGACCACATATGTGGTCGATGCCATGGA</td> <td valign="top" align="left" rowspan="1" colspan="1">G promoter</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">14</td> <td valign="top" align="left" rowspan="1" colspan="1">G promoter_R</td> <td valign="top" align="left" rowspan="1" colspan="1">GACTACTAATACGACTCACTATAGGGAGAAAAGGCCTAACTACC</td> <td rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">15</td> <td valign="top" align="left" rowspan="1" colspan="1">Sg promoter_F</td> <td valign="top" align="left" rowspan="1" colspan="1">AGTCAGTGAAGCCAGTGCTTGACCTGACAAATTCAATTCTGTGT</td> <td valign="top" align="left" rowspan="1" colspan="1">Sg promoter</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">16</td> <td valign="top" align="left" rowspan="1" colspan="1">Sg promoter_R</td> <td valign="top" align="left" rowspan="1" colspan="1">GACTACTAATACGACTCACTATAGGGCGGGCAGCATAGGCAGAA</td> <td rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> <p>To confirm interaction of HEV RdRp with hnRNPK/hnRNPA2B1, FLAG tagged RdRp and c-Myc tagged hnRNPK or hnRNPA2B1 encoding sequence was cloned in pTandem vector (Clontech) under CMV promoter and IRES, respectively. The constructs have been designated as pTandem_FLAG-RdRp_Myc-hnRNPK or pTandem_FLAG-RdRp_Myc-hnRNPA2B1. Primers used for the cloning are mentioned in the <a href="#T1">Table 1</a>.</p> </div> <div class="preparationofhevpromoterrnabaits" title="sec"> <div class="title" tagx="title" title="title">
Preparation of HEV Promoter RNA Baits</div> <p>Sequences coding for the putative genomic promoter (G promoter: nt 1 to 139 on positive sense RNA) and putative sub-genomic promoter (Sg promoter: nt 5051 to 5200 on positive sense RNA) of HEV genotype 1 were PCR amplified from pSK-HEV2 replicon. The primers for the amplification of template were designed with T7 promoter sequence in such a way that the RNA of anti-sense orientation is generated. Primers used for the amplification have been listed in <a href="#T1">Table 1</a>. PCR products were used as templates for the synthesis of RNAs bearing respective promoter sequences. <i>In vitro</i> RNA was synthesized by using MEGAscript kit (Ambion) following the manufacturer’s instructions. Biotinylated <i>in vitro</i> transcribed RNAs were prepared using 5 mM rATP, 5 mM rGTP, 5 mM rUTP, 4.5 mM rCTP, and 0.5 mM of biotin-14 CTP (Invitrogen) in the rNTP mix for the <i>in vitro</i> transcription reaction. For synthesizing non-biotinylated RNAs of respective regions, total 5 mM rCTP was added instead of biotin-14-CTP. Unincorporated nucleotides were removed by purifying the RNA using phenol-chloroform precipitation method. Purified RNAs were visualized on 2% agarose gel.</p> </div> <div class="rnaaffinitychromatography" title="sec"> <div class="title" tagx="title" title="title">
RNA Affinity Chromatography</div> <p>A total of 2 μg of each of biotinylated RNA corresponding to either HEV putative genomic or sub-genomic promoter were coupled with M280 streptavidin dynabeads (Invitrogen) in the presence of nucleic acid binding and washing buffer (B&amp;amp;W buffer: 10 mM Tris–HCl, pH 7.5, 1 mM EDTA, 2M NaCl) for 15 min at room temperature on a rotator. Before RNA binding step, beads were washed with solution A (DEPC-treated 0.1 M NaOH, 0.05M NaCl) followed by solution B (DEPC treated 0.1 M NaCl) to remove RNase. Huh7 S10-3 cells were harvested at ∼80% confluency in the lysis buffer (10 mM Tris–Cl, pH 7.4, 10 mM KCl, 2 mM MgCl<sub>2</sub>, 0.5% Tritin X-100 with protease inhibitor cocktail). The lysate was prepared by centrifugation at 12000 rpm at 4°C for 20 min. The bound RNA-beads complexes were incubated with Huh7 S10-3 cell lysate pre-cleared with 20 μl beads for 1 h at 4°C. Cell lysate and RNA-beads complexes were mixed and incubated together at 4°C on a rotator for 2 h. Bound complexes were washed with B&amp;amp;W buffer and proteins bound to RNA were eluted in 100 μl elution buffer (50 mM Tris–Cl, pH 7.4, 0.2% SDS, 0.1% Tween 20). Eluted proteins were loaded on 12% SDS PAGE followed by silver staining for visualization of protein bands using ProteoSilver staining kit (Sigma). Eluates from three independent RNA affinity chromatography experiments were pooled together and subjected to protein identification by mass spectrometry.</p> </div> <div class="immunoprecipitation" title="sec"> <div class="title" tagx="title" title="title">
Immunoprecipitation</div> <p>pcDNA_FLAG-RdRp construct was transfected into Huh7 S10-3 cells using Lipofectamine 3000 (Invitrogen) transfection reagent. After 48 h post transfection, cells were harvested and lysed in IP lysis buffer (50 mM Tris–Cl pH 7.4, 150 mM NaCl, 1% IGEPAL and protease inhibitor cocktail). Protein G dynabeads (30 μl; Invitrogen) were used for each immunoprecipitation experiment. Next, 4 μg of either rabbit anti FLAG antibody or a non-specific isotype IgG was incubated with washed dynabeads for 30 min at room temperature on a rotator. The cell lysate was incubated with antibody plus dynabeads complex for 2 h at 4°C on a rotator. Three washes with IP lysis buffer were given, followed by a final wash with PBS. Interacting proteins were eluted in 50 μl of elution buffer (50 Mm Tris–Cl pH 7.4, 0.2% SDS, and 0.1% Tween 20). Eluted proteins were loaded on 12% SDS PAGE followed by silver staining for visualization of protein bands using ProteoSilver staining kit (Sigma). Eluates from three independent immunoprecipitation experiments were pooled together and subjected to protein identification by mass spectrometry.</p> </div> <div class="liquidchromatographyquadrupoletime-of-flightmassspectrometry(lc-qtof/ms)" title="sec"> <div class="title" tagx="title" title="title">
Liquid Chromatography Quadrupole Time-of-Flight Mass Spectrometry (LC-QTOF/MS)</div> <p>Hepatitis E virus RNA-dependent RNA polymerase and RNA interacting proteins isolated by immunoprecipitation and RNA affinity chromatography, respectively, were subjected to liquid chromatography-mass spectrometry (LC-MS) analysis. Total 30 μg of each of the protein samples was acetone precipitated, and the protein pellets were dissolved by adding 10 μl of 8 M urea, and the volume was brought to 15 μl with water. Samples were then reduced by the addition of 1.5 μl of 100 mM DTT and heated at 90°C for 15 min. The sample was cooled and alkylated by adding 1.5 μl of 200 mM IAA and incubated in the dark at RT for 15 min. 82 μl of ABC was added, and proteins were digested by adding 1 μl of 1 mg/ml trypsin protease and incubating at 37°C for 16 h. The reaction was stopped by the addition of 1–2 μl of concentrated TFA. Then peptides were dissolved in 0.1% TFA, 5% ACN in water for MS-analysis. Agilent 1260 infinity HPLC-Chip/MS system is a microfluidic chip-based technology was used for peptide enrichment and separation. Charged peptides from HPLC-Chip system were directly infused into mass-spectrometer for detection. Agilent Mass Hunter software was used for data acquisition and analysis of total ion chromatograms. Protein searches were carried out using Morpheus software. Protein identification was performed with the following criteria: (a) Trypsin digested peptides with 2 missed cleavages allowed, (b) peptide tolerance &amp;lt;10 ppm, (c) &amp;gt;2 unique peptides, (d) FDR &amp;lt;1%. Fasta files for Human Proteome database were downloaded from the UniProt was used for protein searches. Proteins found in respective negative control sample were eliminated from the dataset to remove non-specific interactions.</p> </div> <div class="constructionofthemolecularinteractionnetwork" title="sec"> <div class="title" tagx="title" title="title">
Construction of the Molecular Interaction Network</div> <p>All the experimentally derived data sets were used to generate HEV-host proteins interaction network by using "Cytoscape version 3.6.1�? (<a href="#B36">Shannon et al., 2003</a>). To analyze the interaction among host proteins, IntAct protein interaction database was used. Only interactions confirmed by direct physical binding were considered for plotting inter protein interaction map. Topological parameters and central measures of the network were calculated by using a network analyzer tool in Cytoscape. Human protein-protein interaction analysis was also performed by using STRING database. In all the networks and throughout the study, we have used NCBI gene names to represent the proteins in order to have a consensus in protein accession. Corresponding gene names, protein names, and Uniprot protein identifiers have been listed separately (<a href="#DS1">Supplementary Tables 1a–c</a>).</p> </div> <div class="geneontologyanalysis" title="sec"> <div class="title" tagx="title" title="title">
Gene Ontology Analysis</div> <p>Gene ontology "GO�? analysis was carried out by using web-based tools like Panther (Gene Ontology Consortium’s web tool), Gprofiler, STRING, and Enricher. To analyze the enrichment of specific pathway, KEGG annotation database was used. Gene set and pathway enrichment analysis was validated by Fisher’s exact <i>t</i>-test. To control false discovery rate (FDR) Benjamini and Hochberg multiple test correction was used; <i>p</i>-value ≤ 0.05 was considered significant. The corrected <i>p</i>-value for each GO term has been given in the section "Results.�?</p> </div> <div class="co-immunoprecipitationforvalidationofinteractionsinvitro" title="sec"> <div class="title" tagx="title" title="title">
Co-immunoprecipitation for Validation of Interactions <i>in vitro</i></div> <p>Huh7 S10-3 cells expressing FLAG-RdRp and Myc-hnRNPK/hnRNPA2B1 were harvested at 48 h post transfection and resuspended in IP lysis buffer (50 mM Tris–Cl, pH 7.5, 150 mM NaCl and 1% IGEPAL detergent) for half an hour at 4°C. Cell lysate was prepared by centrifugation at 12000 rpm for 30 min. Clear supernatant was mixed with anti Myc antibody or isotype IgG antibody attached to protein G dynabeads (Invitrogen) for 2 h. Three washes of IP lysis buffer were given to remove non-specific interactions. Interacting RdRp-host cellular protein complexes were eluted by using elution buffer (50 mM Tris–Cl, pH 7.4, 0.2% SDS, 0.1% Tween 20). Eluted proteins were subjected to western blot to confirm RdRp-host protein interaction using anti-FLAG antibody.</p> </div> <div class="rnaimmunoprecipitationandrtpcr" title="sec"> <div class="title" tagx="title" title="title">
RNA Immunoprecipitation and RT PCR</div> <p>Huh7 S10-3 cells were harvested at ∼80% confluency in cold DPBS. Cells were lysed in lysis buffer (10 mM Tris–Cl, pH 7.4, 10 mM KCl, 2 mM MgCl<sub>2</sub>, 0.5% Tritin X-100 with protease inhibitor cocktail) for 20 min at 4°C followed by centrifugation at 12000 rpm at 4°C for 10 min. Clear supernatant was incubated with <i>in vitro</i> transcribed RNAs of either putative genomic or sub-genomic promoters of HEV for 2 h at 4°C on rotator. 25 μl of protein G dynabeads were incubated with 4 μg of anti-hnRNPK antibody produced in rabbit (Gene Tex) or anti-hnRNPA2B1 antibody produced in rabbit (Gene Tex) or a non-specific isotype IgG antibody in antibody binding buffer (PBS + 0.0.2% Tween 20) for 35 min at room temperature. Beads bound antibody complexes were added to previously incubated lysate plus RNA complex for 1 h at 4°C on rotator. Three washes of lysis buffer were given followed by a final wash of PBS. Complexes were eluted with 80 μl of elution buffer. Elutes were processed for total RNA isolation using phenol-chloroform method. The RNA was transcribed into cDNA using reverse transcriptase polymerase chain reaction (RT PCR) with Superscript III first strand synthesis kit followed by PCR amplification using Platinum taq DNA polymerase (Invitrogen) using specific primers for the detection of promoter regions. The PCR product was visualized on 2% Agarose gel.</p> </div> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="identificationofhev-interactinghostproteins" title="sec"> <div class="title" tagx="title" title="title">
Identification of HEV-Interacting Host Proteins</div> <p>To pull down HEV RdRp-binding proteins, the HEV RdRp protein was expressed as a FLAG-tagged recombinant protein in Huh7 S10-3 cells. pcDNA_FLAG-RdRp construct was transfected in the cells, and the expression of RdRp was confirmed at 48 h post-transfection by western blotting using an anti-FLAG antibody (<a href="#F1">Figure 1A</a>). Protein G magnetic beads were used to pull down specific interactions using the anti-FLAG antibody in a classical immunoprecipitation experiment from the RdRp-expressing cell lysate. The eluted complexes were subjected to 12% SDS-PAGE followed by silver staining with the ProteoSilver kit (Sigma) for visualizing protein bands (<a href="#F1">Figure 1B</a>). Mock-transfected Huh7 S10-3 cell lysate was taken as control to detect the non-specific interactions. For the pull down of specific HEV RNA-interacting proteins, biotinylated HEV putative sub-genomic (Sg), or putative genomic promoter (G) RNA was synthesized <i>in vitro</i>. <i>In vitro</i> synthesized Sg RNA includes the recently mapped intragenomic promoter region regulating the Sg RNA transcription which is conserved across all HEV genotypes (<a href="#B7">Ding et al., 2018</a>). Genomic promoter region has not been yet mapped functionally, however, based on the HEV RdRp binding studies reported previously with the 3′UTR of HEV anti-sense RNA, we have designed the putative G promoter RNA (<a href="#B20">Mahilkar et al., 2016</a>). RNA affinity chromatography was performed using streptavidin magnetic beads with Huh7 S10-3 cell lysates incubated with HEV RNAs <i>in vitro</i>, and the elutes were subjected to protein identification. Non-biotin RNAs of sub-genomic and genomic promoters were taken as control. The eluted complexes were subjected to 12% SDS-PAGE followed by silver staining (<a href="#F1">Figures 1C,D</a>). A distinct banding pattern could be observed in specific pull-down experiments as compared to the negative control. We further employed liquid chromatography quadrupole time-of-flight mass spectrometry (LC-QTOF/MS) to identify the interacting proteins. Proteins represented by at least two unique peptides and having less than 1% FDR were considered for analysis Proteins represented in respective negative control data were eliminated. The list of proteins identified to be interacting with HEV is shown in <a href="#T2">Table 2</a> and <a href="#DS1">Supplementary Tables 1a–c</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 1</span></a></div>   <p>Identification of HEV interacting host proteins. <b>(A)</b> pcDNA_FLAG-RdRp construct was transfected into Huh7 S10-3 cells. Post 48 h of transfection cells were harvested and checked for expression of RdRp by western blot using anti-FLAG antibody. <b>(B)</b> Mock-transfected Huh 7 S10-3 cells or cells transfected with pcDNA_FLAG_RdRp (expressing FLAG-tagged RdRp) were harvested after 48 h of transfection. Immunoprecipetation was performed by anti-FLAG antibody or an isotype control antibody. RdRp interacting host proteins were eluted with protein G dynabeads and analyzed on SDS PAGE followed by silver staining. <b>(C)</b> HEV sub-genomic promoter (Sg) RNA interacting cellular proteins were pull down by using RNA affinity chromatography. Biotinylated HEV sub-genomic promoter RNA were immobilized on M280 streptavidin dynabeads. RNA immobilized beads were incubated with cell lysate of Huh7 S10-3. Interacting host proteins were eluted and checked on SDS PAGE followed by silver staining for visualization. Non-biotin RNA of the sub-genomic promoter was taken as control. <b>(D)</b> HEV genomic promoter (G) RNA interacting cellular proteins were pull down by using RNA affinity chromatography. HEV genomic promoter RNA interacting host proteins were eluted and checked on SDS PAGE followed by silver staining. For <b>(B–D)</b> lane 1, protein molecular weight ladder; lane 2, negative control pull-down; lane 3, experimental test pull down.</p>   </div> <div class="table-wrap_UNKNOWN" id="T2" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>List of HEV genomic promoter, sub-genomic promoter, and RdRp interacting host proteins.</p>  <table frame="hsides" rules="groups" cellspacing="5" cellpadding="5"> <thead> <tr> <td valign="top" align="left" rowspan="1" colspan="1"><b>HEV component</b></td> <td valign="top" align="left" rowspan="1" colspan="1"><b>Gene symbol of interacting protein</b></td> </tr> </thead> <tbody> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Sub-genomic promoter</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPU, HNRNPA2B1, HNRNPF, NCL, CKAP4, HNRNPK, DDX3X, ACTA2, and POTEF</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">Genomic promoter</td> <td valign="top" align="left" rowspan="1" colspan="1">TUBA1B, TUBA1C, HSP90AA1, HSP90AB1, TUBB, PKM, CKB, FASN, ENO1, PRDX1, HSP90B1, PDIA6, RPLP2, HSPA5, PHGDH, KHSRP, PCBP2, CCT3, ALB, MYH9, VCP, SERPINH1, SPTBN1, ALDH1A1, VIM, SPTAN1, ATP5F1A, PRKDC, LMNA, HSPA9, PHB, SRSF1, DHX9, TUBB4B, HNRNPH1, HNRNPK, HNRNPA2B1, ACTN4, TUFM, HSPA6, HNRNPU, HIST1H1D, EEF1D, TPM3, H3F3B, HNRNPC, EEF2, RNH1, SFPQ, EEF1G, PCBP1, HNRNPAB, ATP5F1B, and HIST1H2BN</td> </tr> <tr> <td valign="top" align="left" rowspan="1" colspan="1">RNA dependent RNA polymerase</td> <td valign="top" align="left" rowspan="1" colspan="1">HNRNPA2B1, HNRNPK, DLST, HNRNPH1, OGDH, HIST3H2A, ACTBL2, EEF1A1, ACTB, and PFKL</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"><span class="attrib" tagx="attrib" title="attrib"><i>Proteins have been represented as NCBI gene names.</i></span></div> </div> </div> <div class="constructionandanalysisofhev-proteininteractionnetwork" title="sec"> <div class="title" tagx="title" title="title">
Construction and Analysis of HEV-Protein Interaction Network</div> <p>A combined list of proteins interacting with HEV promoters and RdRp was generated, and a protein network named the HEV-host interaction network was constructed using "Cytoscape version 3.6.1�? (<a href="#F2">Figure 2A</a>). A list of total 70 HEV-interacting proteins was generated, amongst which two proteins, HNRNPA2B1 and HNRNPK, were found to be present in all the three data sets. One more protein (HNRNPH) was found to be shared between RdRp and G promoter data, while another protein (HNRNPU) was common between the Sg promoter and G promoter besides HNRNPA2B1 and HNRNPK. However, about 94% of the proteins were found to be specifically interacting with only one of the HEV partners in the study (<a href="#F2">Figure 2B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 2</span></a></div>   <p>Construction of HEV-host interaction network. <b>(A)</b> HEV-host interaction network: Interaction map of HEV sub-genomic promoter (Sg), genomic promoter (G) and RdRp with interacting host proteins constructed in Cytoscape 3.6.1. Proteins were classified on the basis of their protein class by Panther gene ontology tool. The corresponding symbols indicating different protein classes have been mentioned on the figure. <b>(B)</b> Venn diagram comparing HEV interacting host proteins with different HEV components. Blue, yellow and green colors indicate proteins interacting with the sub-genomic promoter, genomic promoter, and RdRp, respectively. Common proteins within the data sets have been indicated in the colored intersections. Proteins have been represented as the respective NCBI gene names.</p>   </div> <p>We then searched for the proteins interacting with other host proteins within our data to plot the protein-protein interaction network. We used advanced search options of IntAct database by providing "direct interactions�? filter to find experimentally proven protein-protein interactions within our data and plotted a second network named HEV-host PPI network using Cytoscape 3.6.1 (<a href="#B25">Orchard et al., 2013</a>). IntAct database reports evidences like pull down, X-ray crystallography, functional assays, etc under experimentally proven criteria of interaction prediction. IntAct tool sources the molecular interaction data from several curated databases like MINT, Uniprot, molecular connections, EMBL-EBI, and DIP. Primary interactions revealed through mass spectrometry in this study have been shown with black edges, while the secondary protein-protein interactions revealed through IntAct have been shown in red (<a href="#F3">Figure 3</a>). We then calculated the topological parameters of the generated PPI network to access its modularity using the network analyzer tool of Cytoscape. It is revealed that 70 nodes representing HEV-host interactions were connected via a total of 141 edges. The average degree centrality, average path length distribution, and the clustering coefficient of the network were observed to be 0.748, 2.365, and 0.348, respectively. The topological parameters were compared with HEV-human protein interaction networks previously reported in literature which validated the significance of HEV-host protein interaction network constructed in our study (<a href="#DS1">Supplementary Table 2</a>). Amongst the proteins present in our data, PFKL, SERPIN, HNRNPH, HSP90AB1, and TUBB have been previously reported to interact with HEV macrodomain (<a href="#B24">Ojha and Lole, 2016</a>). Besides, PCBP1 and EEF1A1 have been reported to interact with HEV ORF2, while DHX9, HNRNPC, and HNRNPK have been reported to interact with HEV non-coding regions on the sense strand (<a href="#B26">Paingankar and Arankalle, 2015</a>; <a href="#B39">Subramani et al., 2018</a>). This confirms the authenticity of HEV-specific interactome found in our study.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 3</span></a></div>   <p>Construction and analysis of HEV-host PPI network. Interaction map of HEV interacting host proteins further interacting with the other proteins of our data. Black edges represent interactions revealed through mass spectrometry reported in this study. Secondary protein-protein interactions among host proteins revealed through literature mining have been indicated in red colored edges. Proteins have been represented as the respective NCBI gene names.</p>   </div> <p>We also used STRING database to generate the inter-protein interaction network from our data (<a href="#F4">Figure 4</a> and <a href="#DS1">Supplementary Table 3</a>) (<a href="#B41">Szklarczyk et al., 2015</a>). This analysis gave us a more intense multidimensional network as, along with the experimentally determined interactions, those predicted from gene fusion; co-expression, homology, and text mining were also considered. The observed number of edges for the network (487) was significantly higher than the expected number of edges (132) for the given number of nodes (67), implying that there are more interactions than expected. The result suggested that the proteins in our data dispaly more interactions than expected for a random set of proteins. Such enrichment indicates that the proteins are at least partially biologically connected as a group, highlighting the significance of the HEV-host network reported in this study. This also indicates the probability of isolating HEV-specific protein complexes gathered at the HEV replication site.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 4</span></a></div>   <p>Analysis of inter protein interaction network using STRING database. Each edge color indicates a different method of protein-protein interaction prediction as indicated below the figure.</p>   </div> </div> <div class="geneontologyannotation" title="sec"> <div class="title" tagx="title" title="title">
Gene Ontology Annotation</div> <p>To investigate the cellular components or pathways that have been enriched in the HEV-host protein interaction network, we performed gene ontology analysis (GO annotation) of the data using various web-based tools. The GO annotation resulted in classification of the proteins based on their functional clusters or GO categories. The enrichment analysis performed using Panther (Gene Ontology Consortium’s web tool) (<a href="#DS1">Supplementary Figures 1A–C, 2</a>), Gprofiler (<a href="#DS1">Supplementary Figure 3</a>), STRING (<a href="#DS1">Supplementary Figure 4</a>), and Enricher was in agreement and showed a significant enrichment of similar GO terms. FDR &amp;lt;0.05 was set as a statistical threshold for GO analysis. Enrichment of proteins in three different categories, namely biological processes, molecular function, and cellular component, was performed using Enricher (<a href="#B6">Chen et al., 2013</a>; <a href="#B18">Kuleshov et al., 2016</a>). Proteins were arranged according to the combined enrichment score, which is a combination of the <i>p</i>-value and z-score calculated by multiplying the two scores. The <i>p</i>-value is the probability of any gene belonging to any set and is calculated by using exact Fisher’s test. The Z-score is calculated by using a modified Fisher’s exact test and assesses the deviation from the expected rank. The combined score provides a compromise between the available methods for multiple test corrections to control the FDRs (<a href="#B18">Kuleshov et al., 2016</a>).</p> <p>Enrichment of GO terms in biological process category revealed the enrichment of proteins related to mRNA splicing (GO: 0000380; <i>p</i>-value: 3.5 × 10<sup>–3</sup>), response to unfolded proteins (GO: 0006986; <i>p</i>-value: 3.6 × 10<sup>–6</sup>), nucleic acid metabolic process (GO: 0090304, <i>p</i>-value: 1.3 × 10<sup>–6</sup>), and regulation of protein processing (GO: 0010954, <i>p</i>-value: 8 × 10<sup>–3</sup>) (<a href="#F5">Figure 5A</a> and <a href="#DS1">Supplementary Table 4A</a>). A vital characteristic of any virus-host interaction is the manipulation of cellular gene expression in order to establish favorable cellular conditions for efficient viral replication. Eukaryotic genes are highly regulated at post-transcriptional stages, such as splicing, export, regulation of translation, subcellular localization and mRNA turnover. Viruses have often been observed to target these RNA processing stages to hijack the host RNA metabolism which is evident from the enrichment of host RNA splicing and nucleic acid metabolism related protein in our data. Such enhancement could also hint at the reliance of HEV on host systems for the regulation of viral RNA translation, transcription, and stability within the host cell (<a href="#B2">Ahlquist et al., 2003</a>). We observed the highest number of proteins having nucleic acid-binding activity in our data, which further confirms the dependence of HEV on the host for viral RNA metabolism. We have incorporated the data of classification of proteins on the basis of protein class in the HEV-host interaction network using different color codes as indicated on the figure (<a href="#F2">Figure 2A</a> and <a href="#DS1">Supplementary Figure 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 5</span></a></div>   <p>Gene ontology analysis of HEV- host interactions based on <b>(A)</b> biological process, <b>(B)</b> molecular function, and <b>(C)</b> cellular component category. <i>Y</i>-axis represents the combined enrichment score computed using Enricher.</p>   </div> <p>We also observed enrichment of stress granule proteins (GO:1903608; <i>p</i>-value: 3.3 × 10<sup>–2</sup>) in our data. Stress granules are dynamic structures that are formed when cellular translational rates decline after external stresses are applied to cells. Viral infection into the host cell has been shown to induce cytoplasmic stress granule formation in the host cells due to the upregulation of stress granule proteins, such as helicase DDX3X and DHX9 (<a href="#B3">Ariumi et al., 2011</a>; <a href="#B28">Pène et al., 2015</a>). As stress granules regulate the cycle of mRNA turnover and gene expression, they happen to be another vital point for viruses to manipulate cellular systems. Stress granules have also been reported to play a role in promoting innate immune responses (<a href="#B32">Reineke and Lloyd, 2013</a>). This is an additional reason why viruses must counteract the effects of such granules for efficient replication by interacting with proteins of stress granules.</p> <p>Virus replication alters the normal metabolic processes of the host by interacting with components of different molecular pathways. Enrichment of GO terms in molecular function category also resulted in enrichment of proteins involved in RNA metabolism processes, which again highlights the need of the virus to interact with proteins of host RNA turnover. The enriched proteins in molecular function category belong to Translation elongation factor activity (GO: 0003746, <i>p</i>-value: 1 × 10<sup>–3</sup>), RNA binding (GO: 0003723, <i>p</i>-value: 3.9 × 10<sup>–25</sup>) and RNA stem loop binding (GO: 0035613, <i>p</i>-value: 3.4 × 10<sup>–2</sup>).</p> <p>MHC protein complex-binding proteins (GO: 0023023, <i>p</i>-value: 2.5 × 10<sup>–3</sup>) have also been observed to be enriched in the molecular function category (<a href="#F5">Figure 5B</a> and <a href="#DS1">Supplementary Table 4B</a>). The anti-viral responses of the host and the invading strategies of the pathogens have evolved concurrently for millions of years. Infecting pathogens have developed several escape strategies to cripple the immune system. Several viruses have evolved with proteins that interfere with antigen presentation and which target both MHC-I and MHC-II antigen processing pathways in order to distort the anti-viral immune response of the host. Several viruses such as HSV, Epstein-Barr virus, Bovine herpes virus and cytomegalovirus have been shown to have evolved strategies to combat MHC-mediated host immunity (<a href="#B50">Yewdell and Bennink, 1999</a>; <a href="#B33">Røder et al., 2008</a>). However, there are no literature reports of HEV interactions with MHC molecules. In the HEV interactome reported in this study, we observed the presence of MHC-interacting proteins, such as HSP90AA1 and PKM, which could be explored further for their ability to alter host immune response in the context of HEV infection.</p> <p>Enrichment analysis of GO terms according to cellular component enhanced the representation of GO terms, such as ribonucleoprotein granules (GO: 0035770, <i>p</i>-value: 4 × 10<sup>–8</sup>), secretory granule lumen (GO: 0034774, <i>p</i>-value: 1 × 10<sup>–9</sup>) and endocytic vesicle lumen (GO: 0071682, <i>p</i>-value: 3.8 × 10<sup>–2</sup>) (<a href="#F5">Figure 5C</a> and <a href="#DS1">Supplementary Table 4C</a>). HEV has been reported to enter liver cells through receptor-mediated endocytosis (<a href="#B16">Kapur et al., 2012</a>). In agreement with this, our data also suggest enrichment of proteins belonging to endocytic vesicle lumen. HEV replication takes place in the cytoplasm. Along with the replicase complex, HEV RdRp has been shown to localize to the endoplasmic reticulum (ER) membrane (<a href="#B31">Rehman et al., 2008</a>). A recent report also concluded that ORF1 polyprotein co-localizes with the markers of ER-Golgi intermediate compartment, suggesting the involvement of secretory pathway during replication (<a href="#B42">Szkolnicka et al., 2019</a>). Previous studies have also indicated that HEV forms membrane-associated particles in the cytoplasm by means of budding into intracellular vesicles. HEV exploits the multivesicular body pathway to release infectious virion particles outside the cell through the cellular exosomal pathway (<a href="#B22">Nagashima et al., 2014</a>).</p> </div> <div class="pathwayenrichmentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Pathway Enrichment Analysis</div> <p>To understand the cellular pathways targeted by HEV, we performed pathway enrichment analysis by using the KEGG functional annotation pathway database through Enricher. The results revealed enrichment of different pathways involving pathogenic <i>Escherichia coli</i> infection, protein processing in endoplasmic reticulum, legionellosis, spliceosome, antigen presentation and processing, gap junction, citrate cycle, IL17 signaling pathways, apoptosis, and phagosome (<a href="#F6">Figure 6A</a> and <a href="#DS1">Supplementary Table 5</a>). Furthermore, the schematic of the entire pathway network for endoplasmic reticulum protein processing and spliceosome pathway was obtained from the KEGG pathway database (<a href="#B14">Kanehisa and Goto, 2000</a>; <a href="#B15">Kanehisa et al., 2019</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 6</span></a></div>   <p>Pathway enrichment analysis. <b>(A)</b> Graph shows the enriched pathways targeted by HEV, analyzed by KEGG functional annotation pathway database. <i>Y</i>-axis represents the combined score computed using Enricher. <b>(B)</b> Schematic representation of "protein processing in endoplasmic reticulum�? pathway (imported from KEGG: map04141). <b>(C)</b> Schematic representation of the spliceosome pathway targeted by HEV host interacting proteins (imported from KEGG: map03040). Proteins interacting with HEV RNA promoter (G or Sg) or polymerase within the entire pathway are shown in red color.</p>   </div> <p>Protein processing in the endoplasmic reticulum pathway was enriched (<i>p</i>-value 1.2 × 10<sup>–5</sup>). Seven proteins of this pathway, namely, CKAP4, HSPA5, HSP90B1, PDIs, HSP70, HSP90, and VCP were found to interact with HEV RNA and polymerase (<a href="#F6">Figure 6B</a>). These factors are involved in proper folding and processing of newly synthesized proteins in the endoplasmic reticulum. As HEV virus replicates on the ER membrane and its polymerase localizes onto the ER, its interaction with proteins residing at the endoplasmic reticulum is apparent during its replication.</p> <p>Proteins belonging to the heat shock protein (HSP) family are well known for their roles as chaperons in protein folding. These proteins have been observed to play similar roles in the maintenance of proper viral protein folding and stabilization (<a href="#B35">Seo et al., 2018</a>). In many viral infections, viruses commandeer vital cellular components, leading to cellular stress. Cellular stress is often represented as unfolded protein response (UPR) in cells (<a href="#B11">Geller et al., 2012</a>; <a href="#B45">Wang et al., 2018</a>). Previous studies have reported that ORF3 and ORF2 of the virus induce UPR and ER stress (<a href="#B40">Surjit et al., 2007</a>; <a href="#B13">John et al., 2011</a>; <a href="#B48">Xu et al., 2014</a>). In our study, we found enrichment of proteins related to processing in the endoplasmic reticulum, and UPR further confirms the dependence of HEV on the host ER machinery.</p> <p>In our analyses, the spliceosome pathway was found to be enriched by <i>p</i>-value of 5.9 × 10<sup>–4</sup>). Different heterogeneous nuclear ribonucleoproteins, such as HNRNPK, HNRNPA2B1, HNRNPH, HNRNPE1, HNRNPE2, HNRNPC and HNRNPU, and other spliceosomal accessory complex proteins, such as HSP73 and SR, interact with HEV (<a href="#F6">Figure 6C</a>). Spliceosomal complex proteins help in the generation of stable RNA structures, while ribonucleoproteins play roles in RNA stability and transport (<a href="#B47">Will and Lührmann, 2011</a>). Previous studies have reported the binding of HNRNPA2B1 to Influenza virus and Dengue virus RNA and explained its role in regulation of viral transcription (<a href="#B27">Paranjape and Harris, 2007</a>; <a href="#B46">Wang et al., 2014</a>). HNRNPK binds to the core protein of Dengue virus, whereas in Sindbis virus infection, it binds to its non-structural protein and sub-genomic RNA to regulate viral replication (<a href="#B5">Chang et al., 2001</a>; <a href="#B4">Burnham et al., 2007</a>; <a href="#B19">LaPointe et al., 2018</a>). HNRNPK binds to hepatitis C virus RNA near the miR-122 binding site to facilitate its replication (<a href="#B8">Fan et al., 2015</a>). In another study, the role played by HNRNPK in HCV virion production is reported to be mediated by viral RNA binding (<a href="#B30">Poenisch et al., 2015</a>). HNRNPK binds to Influenza M1 RNA and regulates its splicing while maintaining appropriate ratio of M2/M1 protein (<a href="#B43">Thompson et al., 2018</a>). HNRNPH binds to the negative regulator of splicing elements in Rous sarcoma virus to regulate the splicing and polyadenylation machinery (<a href="#B10">Fogel and McNally, 2000</a>). A quantitative proteomics study by <a href="#B34">Rogée et al. (2015)</a> evaluating the alteration of host factors during HEV infection in swine revealed an upregulation of HNRNPK in infected livers (<a href="#B34">Rogée et al., 2015</a>). In another study of modulation of host factors during HEV infection in A549 cells, a significant increase in the expression of HNRNPH was observed in response to virus infection (<a href="#B37">Shen et al., 2014</a>). Our study is consistent with these findings as it underlines a putative role of hnRNP proteins in modulating HEV RNA transcription by binding at the promoter site and HEV polymerase.</p> </div> <div class="validationofinteractionsbetweenhnrnpkandhnrnpa2b1withhevpromotersandrdrp" title="sec"> <div class="title" tagx="title" title="title">
Validation of Interactions Between HNRNPK and HNRNPA2B1 With HEV Promoters and RdRp</div> <p>Analysis of HEV-host protein interactions suggested that HNRNPK and HNRNPA2B1 are common factors associating with HEV RNA promoters and polymerase (<a href="#F2">Figures 2A,B</a>). To demonstrate the validity of the interactions obtained through mass spectrometry, we further <i>in vitro</i> validated the interactions between HNRNPK and HNRNPA2B1 with that of HEV promoters and RdRp. In order to confirm the interaction of RdRp with host proteins, coimmunoprecipitation technique was employed. HEV RdRp encoding sequence and the coding sequence of selected host protein were cloned in pTandem vector. pTandem vector was chosen for its additional feature which enables cloning of two different genes in the same construct to increase the co-transfection efficiency. Huh7 S10-3 cells transfected with pTandem_Flag-RdRp_Myc-HNRNPK or pTandem_Flag-RdRp_Myc-HNRNPA2B1 plasmids. Post 48 h of transfection co-immunoprecipitation was performed by using anti myc antibody or isotype IgG antibody. After co-immunoprecipitation, interaction of host factors with RdRp was confirmed by western blot using anti FLAG antibody. We observed that both HNRNPK and HNRPA2B1 specific immunoprecipitation could pull down HEV RdRp (<a href="#F7">Figures 7A,B</a>). Our results confirmed the binding of both of these proteins with HEV RdRp as obtained in the mass spectrometry data.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIGURE 7</span></a></div>   <p>Validation of interactions between HNRNPK and HNRNPA2B1 with HEV promoters and RdRp. <b>(A)</b> pTandem_FLAG-RdRp_Myc-HNRNPK plasmid was transfected in Huh 7 S10-3 cells. 48 h post transfection co-IP was performed with anti c-Myc antibody. Interaction of c-Myc tagged HNRNPK with that of FLAG tagged RdRp was checked with western blot by using anti c-Myc antibody. <b>(B)</b> pTandem_FLAG-RdRp_Myc-HNRNPA2B1 plasmid was transfected in Huh 7 S10-3 cells. 48 h post transfection co-IP was performed with anti c-Myc antibody. Interaction of c-Myc tagged HNRNPA2B1 with that of FLAG tagged RdRp was checked with western blot by using anti c-Myc antibody. <b>(C)</b> Huh 7 S10-3 cell lysate was incubated with HEV G and Sg promoter RNA followed by immunoprecipitation with anti HNRNPK or anti HNRNPA2B1 antibody. RT-PCR was performed to detect HNRNP-bound HEV RNAs in the elutes. Figure shows amplified PCR products on 2% agarose gel.</p>   </div> <p>To further confirm the binding of HNRNPK and HNRNPA2B1 with HEV putative promoters; we employed RNA immunoprecipitation followed by RT PCR. Huh7 S10-3 cell lysate was incubated with G or Sg promoter RNA followed by pull down by anti-HNRNPK or anti-HNRNPA2B1 or isotype IgG antibody. From the eluted complexes, total RNA was isolated and presence of promoter region RNA sequence was checked by RT PCR. We observed that both HNRNPK and HNRNPA2B1 could successfully pull down HEV genomic and sub-genomic promoter RNA as evident by the specific band on agarose gel (<a href="#F7">Figure 7C</a>). Therefore we could successfully demonstrate the <i>in vitro</i> binding of HNRNPK and HNRNPA2B1 with HEV RdRp and HEV promoters as obtained from the mass spectrometry data, further increasing the confidence level of the obtained interactions. Thus, we believe that HNRNPK and HNRNPA2B1 play crucial roles in HEV replication and their functional significance in the context of HEV replication can be further assessed. Taken as a whole, our study reveals the importance of host cellular machinery in HEV lifecycle regulation. Studying host pathways targeted by HEV can facilitate the hunt for putative anti-viral candidates for therapeutic purposes. In conclusion, our study shows that analyzing host-virus interactions through system biology approach can be beneficial in understanding the molecular regulation of viral lifecycle and can put forth set testable hypotheses for future experimental validation.</p> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data Availability Statement</div> <p>All datasets generated/analyzed for this study are included in the article/<a href="#DS1">Supplementary  Material</a>.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>GK, KP, and YK conceived and designed the experiments, analyzed the data, and wrote the manuscript. GK and KP performed the experiments.</p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding.</b> This work was not funded by any specific funder. GK acknowledges Indian Council of Medical Research (ICMR) for senior research fellowship. KP acknowledges Council of Scientific &amp;amp; Industrial Research (CSIR), Government of India for the senior research fellowship.</p> </div> </div> <div class="ack" title="ack"> <p>The authors are thankful to Director, MACS-ARI, Pune for all the support.</p> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.02501/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2019.02501/full#supplementary-material</a></p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="Data_Sheet_1.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aderem</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adkins</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ansong</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galagan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">A systems biology approach to infectious disease research: innovating the pathogen-host research paradigm.</span><span class="source" tagx="source" title="source"><i>mBio</i></span><span class="volume" tagx="volume" title="volume">2</span>:<span class="issue" tagx="issue" title="issue">e00325-310</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00325-310">10.1128/mBio.00325-310</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21285433">21285433</a></span></span></li> <li tag="ref"><a name="B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahlquist</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noueiry</span><span class="given-names" tagx="given-names" title="given-names">A. O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">W. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kushner</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dye</span><span class="given-names" tagx="given-names" title="given-names">B. T.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Host factors in positive-strand RNA virus genome replication.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">8181</span>–<span class="lpage" tagx="lpage" title="lpage">8186</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02501-05">10.1128/JVI.02501-05</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12857886">12857886</a></span></span></li> <li tag="ref"><a name="B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ariumi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuroki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kushima</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osugi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hijikata</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maki</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus hijacks P-body and stress granule components around lipid droplets.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">6882</span>–<span class="lpage" tagx="lpage" title="lpage">6892</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02418-10">10.1128/JVI.02418-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21543503">21543503</a></span></span></li> <li tag="ref"><a name="B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burnham</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Heterogeneous nuclear ribonuclear protein K interacts with <i>Sindbis</i> virus nonstructural proteins and viral subgenomic mRNA.</span><span class="source" tagx="source" title="source"><i>Virology</i></span><span class="volume" tagx="volume" title="volume">367</span><span class="fpage" tagx="fpage" title="fpage">212</span>–<span class="lpage" tagx="lpage" title="lpage">221</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2007.05.008">10.1016/j.virol.2007.05.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17561226">17561226</a></span></span></li> <li tag="ref"><a name="B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luh</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">The heterogeneous nuclear ribonucleoprotein K (hnRNP K) interacts with dengue virus core protein.</span><span class="source" tagx="source" title="source"><i>DNA And Cell Biology</i></span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">569</span>–<span class="lpage" tagx="lpage" title="lpage">577</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1089/104454901317094981">10.1089/104454901317094981</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11747608">11747608</a></span></span></li> <li tag="ref"><a name="B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">E. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kou</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meirelles</span><span class="given-names" tagx="given-names" title="given-names">G. V.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool.</span><span class="source" tagx="source" title="source"><i>BMC Bioinformatics</i></span><span class="volume" tagx="volume" title="volume">14</span>:<span class="issue" tagx="issue" title="issue">128</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-14-128">10.1186/1471-2105-14-128</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23586463">23586463</a></span></span></li> <li tag="ref"><a name="B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nimgaonkar</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archer</span><span class="given-names" tagx="given-names" title="given-names">N. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bram</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heller</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartz</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of the intragenomic promoter controlling hepatitis E virus subgenomic RNA transcription.</span><span class="source" tagx="source" title="source"><i>mBio</i></span><span class="volume" tagx="volume" title="volume">9</span>:<span class="issue" tagx="issue" title="issue">e00769-18</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00769-18">10.1128/mBio.00769-18</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29739903">29739903</a></span></span></li> <li tag="ref"><a name="B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sutandy</span><span class="given-names" tagx="given-names" title="given-names">F. X. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Syu</span><span class="given-names" tagx="given-names" title="given-names">G. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middleton</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">K. Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Heterogeneous ribonucleoprotein K (hnRNP K) binds miR-122, a mature liver-specific MicroRNA required for <i>Hepatitis</i> C virus replication.</span><span class="source" tagx="source" title="source"><i>Mol. Cell. Proteom.</i></span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">2878</span>–<span class="lpage" tagx="lpage" title="lpage">2886</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/mcp.M115.050344">10.1074/mcp.M115.050344</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26330540">26330540</a></span></span></li> <li tag="ref"><a name="B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fermin</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tennant</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Chapter 10 - Host–Virus Interactions: Battles Between Viruses and Their Hosts</i>.</span> Available at: <a href="https://www.sciencedirect.com/science/article/pii/B9780128112571000103">https://www.sciencedirect.com/science/article/pii/B9780128112571000103</a>, <span class="fpage" tagx="fpage" title="fpage">245</span>–<span class="lpage" tagx="lpage" title="lpage">271</span>.</span></li> <li tag="ref"><a name="B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fogel</span><span class="given-names" tagx="given-names" title="given-names">B. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNally</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">A cellular protein, hnRNP H, binds to the negative regulator of splicing element from <i>Rous sarcoma</i> virus.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">275</span><span class="fpage" tagx="fpage" title="fpage">32371</span>–<span class="lpage" tagx="lpage" title="lpage">32378</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.m005000200">10.1074/jbc.m005000200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10934202">10934202</a></span></span></li> <li tag="ref"><a name="B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geller</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taguwa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frydman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Broad action of Hsp90 as a host chaperone required for viral replication.</span><span class="source" tagx="source" title="source"><i>Biochim. Biophys. Acta</i></span><span class="volume" tagx="volume" title="volume">1823</span><span class="fpage" tagx="fpage" title="fpage">698</span>–<span class="lpage" tagx="lpage" title="lpage">706</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbamcr.2011.11.007">10.1016/j.bbamcr.2011.11.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22154817">22154817</a></span></span></li> <li tag="ref"><a name="B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Himmelsbach</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bender</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hildt</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Life cycle and morphogenesis of the hepatitis E virus.</span><span class="source" tagx="source" title="source"><i>Emerg. Microb. Infect.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">196</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41426-018-0198-7">10.1038/s41426-018-0198-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30498191">30498191</a></span></span></li> <li tag="ref"><a name="B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">John</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herchenröder</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pützer</span><span class="given-names" tagx="given-names" title="given-names">B. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaefer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Hepatitis</i> E virus ORF2 protein activates the pro-apoptotic gene CHOP and anti-apoptotic heat shock proteins.</span><span class="source" tagx="source" title="source"><i>PLoS One</i></span><span class="volume" tagx="volume" title="volume">6</span>:<span class="issue" tagx="issue" title="issue">e25378</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0025378">10.1371/journal.pone.0025378</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21966512">21966512</a></span></span></li> <li tag="ref"><a name="B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanehisa</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">KEGG: kyoto encyclopedia of genes and genomes.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">27</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/28.1.27">10.1093/nar/28.1.27</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a></span></span></li> <li tag="ref"><a name="B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanehisa</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furumichi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morishima</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanabe</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">New approach for understanding genome variations in KEGG.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">D590</span>–<span class="lpage" tagx="lpage" title="lpage">D595</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gky962">10.1093/nar/gky962</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30321428">30321428</a></span></span></li> <li tag="ref"><a name="B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kapur</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thakral</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durgapal</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panda</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus enters liver cells through receptor-dependent clathrin-mediated endocytosis.</span><span class="source" tagx="source" title="source"><i>J. Viral. Hepat.</i></span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">436</span>–<span class="lpage" tagx="lpage" title="lpage">448</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2893.2011.01559.x">10.1111/j.1365-2893.2011.01559.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22571906">22571906</a></span></span></li> <li tag="ref"><a name="B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenney</span><span class="given-names" tagx="given-names" title="given-names">S. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">X.-J.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus genome structure and replication strategy.</span><span class="source" tagx="source" title="source"><i>Cold Spring Harb. Perspect. Med.</i></span><span class="volume" tagx="volume" title="volume">8</span>:<span class="issue" tagx="issue" title="issue">a031724</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/cshperspect.a031724">10.1101/cshperspect.a031724</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29530948">29530948</a></span></span></li> <li tag="ref"><a name="B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuleshov</span><span class="given-names" tagx="given-names" title="given-names">M. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rouillard</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez</span><span class="given-names" tagx="given-names" title="given-names">N. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Enrichr: a comprehensive gene set enrichment analysis web server (2016) update.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">W90</span>–<span class="lpage" tagx="lpage" title="lpage">W97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw377">10.1093/nar/gkw377</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27141961">27141961</a></span></span></li> <li tag="ref"><a name="B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LaPointe</span><span class="given-names" tagx="given-names" title="given-names">A. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gebhart</span><span class="given-names" tagx="given-names" title="given-names">N. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meller</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hardy</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sokoloski</span><span class="given-names" tagx="given-names" title="given-names">K. J.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The identification and characterization of <i>Sindbis</i> Virus R 1 NA:Host protein interactions.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">92</span>:<span class="issue" tagx="issue" title="issue">e02171-2117</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02171-2117">10.1128/JVI.02171-2117</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29321325">29321325</a></span></span></li> <li tag="ref"><a name="B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahilkar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paingankar</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lole</span><span class="given-names" tagx="given-names" title="given-names">K. S.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title"><i>Hepatitis</i> E virus RNA-dependent RNA polymerase: RNA template specificities, recruitment and synthesis.</span><span class="source" tagx="source" title="source"><i>J. Gen. Virol.</i></span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">2231</span>–<span class="lpage" tagx="lpage" title="lpage">2242</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000528">10.1099/jgv.0.000528</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27324050">27324050</a></span></span></li> <li tag="ref"><a name="B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">X. J.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus: animal reservoirs and zoonotic risk.</span><span class="source" tagx="source" title="source"><i>Vet. Microbiol.</i></span><span class="volume" tagx="volume" title="volume">140</span><span class="fpage" tagx="fpage" title="fpage">256</span>–<span class="lpage" tagx="lpage" title="lpage">265</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vetmic.2009.03.017">10.1016/j.vetmic.2009.03.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19361937">19361937</a></span></span></li> <li tag="ref"><a name="B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagashima</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jirintai</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanggis Nishizawa</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouki</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus egress depends on the exosomal pathway, with secretory exosomes derived from multivesicular bodies.</span><span class="source" tagx="source" title="source"><i>J. Gen. Virol.</i></span><span class="volume" tagx="volume" title="volume">95</span><span class="fpage" tagx="fpage" title="fpage">2166</span>–<span class="lpage" tagx="lpage" title="lpage">2175</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.066910-0">10.1099/vir.0.066910-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24970738">24970738</a></span></span></li> <li tag="ref"><a name="B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">molecular biology and infection of <i>Hepatitis</i> E virus.</span><span class="source" tagx="source" title="source"><i>Front. Microbiol.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">1419</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2016.01419">10.3389/fmicb.2016.01419</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27656178">27656178</a></span></span></li> <li tag="ref"><a name="B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ojha</span><span class="given-names" tagx="given-names" title="given-names">N. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lole</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus ORF1 encoded non structural protein–host protein interaction network.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="volume" tagx="volume" title="volume">213</span><span class="fpage" tagx="fpage" title="fpage">195</span>–<span class="lpage" tagx="lpage" title="lpage">204</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2015.12.007">10.1016/j.virusres.2015.12.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26689634">26689634</a></span></span></li> <li tag="ref"><a name="B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orchard</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ammari</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aranda</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breuza</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briganti</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">The MIntAct project–IntAct as a common curation platform for 11 molecular interaction databases.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">D358</span>–<span class="lpage" tagx="lpage" title="lpage">D363</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt1115">10.1093/nar/gkt1115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24234451">24234451</a></span></span></li> <li tag="ref"><a name="B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paingankar</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arankalle</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of cellular proteins interactingwith <i>Hepatitis</i> E virus untranslated regions.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="volume" tagx="volume" title="volume">208</span><span class="fpage" tagx="fpage" title="fpage">98</span>–<span class="lpage" tagx="lpage" title="lpage">109</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2015.06.006">10.1016/j.virusres.2015.06.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26087402">26087402</a></span></span></li> <li tag="ref"><a name="B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paranjape</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Box-binding Protein-1 binds to the dengue Virus 3’untranslated region and mediates antiviral effects.</span><span class="source" tagx="source" title="source"><i>J. Biol. Chem.</i></span><span class="volume" tagx="volume" title="volume">282</span><span class="fpage" tagx="fpage" title="fpage">30497</span>–<span class="lpage" tagx="lpage" title="lpage">30508</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.m705755200">10.1074/jbc.m705755200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17726010">17726010</a></span></span></li> <li tag="ref"><a name="B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pène</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sodroski</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">C. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Dynamic interaction of stress granule, DDX3X and IKK-α mediates multiple functions in hepatitis C virus infection.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">5462</span>–<span class="lpage" tagx="lpage" title="lpage">5477</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03197-14">10.1128/JVI.03197-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25740981">25740981</a></span></span></li> <li tag="ref"><a name="B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">E. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">D. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korth</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katze</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Virus-host interactions: from systems biology to translational research.</span><span class="source" tagx="source" title="source"><i>Curr. Opin. Microbiol</i>.</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">432</span>–<span class="lpage" tagx="lpage" title="lpage">438</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2009.06.003">10.1016/j.mib.2009.06.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19576841">19576841</a></span></span></li> <li tag="ref"><a name="B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poenisch</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metz</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blankenburg</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruggieri</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">J. Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rupp</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of HNRNPK as regulator of <i>Hepatitis</i> C virus particle production.</span><span class="source" tagx="source" title="source"><i>PLoS Pathog.</i></span><span class="volume" tagx="volume" title="volume">11</span>:<span class="issue" tagx="issue" title="issue">e1004573</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004573">10.1371/journal.ppat.1004573</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25569684">25569684</a></span></span></li> <li tag="ref"><a name="B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rehman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kapur</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durgapal</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panda</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Subcellular localization of hepatitis E virus (HEV) replicase.</span><span class="source" tagx="source" title="source"><i>Virology</i></span><span class="volume" tagx="volume" title="volume">370</span><span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2007.07.036">10.1016/j.virol.2007.07.036</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17904184">17904184</a></span></span></li> <li tag="ref"><a name="B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reineke</span><span class="given-names" tagx="given-names" title="given-names">L. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Diversion of stress granules and P-bodies during viral infection.</span><span class="source" tagx="source" title="source"><i>Virology</i></span><span class="volume" tagx="volume" title="volume">436</span><span class="fpage" tagx="fpage" title="fpage">255</span>–<span class="lpage" tagx="lpage" title="lpage">267</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2012.11.017">10.1016/j.virol.2012.11.017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23290869">23290869</a></span></span></li> <li tag="ref"><a name="B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Røder</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geironson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bressendorff</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulsson</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Viral proteins interfering with antigen presentation target the major histocompatibility complex class I peptide-loading complex.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">8246</span>–<span class="lpage" tagx="lpage" title="lpage">8252</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00207-08">10.1128/JVI.00207-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18448533">18448533</a></span></span></li> <li tag="ref"><a name="B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogée</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Gall</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chafey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouquet</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cordonnier</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frederici</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Quantitative proteomics identifies host factors modulated during acute hepatitis E virus infection in the swine model.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">129</span>–<span class="lpage" tagx="lpage" title="lpage">143</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02208-14">10.1128/JVI.02208-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25320303">25320303</a></span></span></li> <li tag="ref"><a name="B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seo</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Heat shock protein 70 and heat shock protein 90 synergistically increase hepatitis B viral capsid assembly.</span><span class="source" tagx="source" title="source"><i>Biochem. Biophys. Res. Commun.</i></span><span class="volume" tagx="volume" title="volume">503</span><span class="fpage" tagx="fpage" title="fpage">2892</span>–<span class="lpage" tagx="lpage" title="lpage">2898</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bbrc.2018.08.065">10.1016/j.bbrc.2018.08.065</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30115382">30115382</a></span></span></li> <li tag="ref"><a name="B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shannon</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markiel</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ozier</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baliga</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramage</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span><span class="source" tagx="source" title="source"><i>Genome Res.</i></span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2498</span>–<span class="lpage" tagx="lpage" title="lpage">2504</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a></span></span></li> <li tag="ref"><a name="B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xingli</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ying</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiaobo</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shixing</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Changes in the cellular proteins of A549 infected with <i>Hepatitis</i> E virus by proteomics analysis.</span><span class="source" tagx="source" title="source"><i>BMC Vet. Res.</i></span><span class="volume" tagx="volume" title="volume">10</span>:<span class="issue" tagx="issue" title="issue">188</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12917-014-0188-5">10.1186/s12917-014-0188-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25175408">25175408</a></span></span></li> <li tag="ref"><a name="B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stebbing</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gazzard</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Virus host interactions.</span><span class="source" tagx="source" title="source"><i>Obstet. Gynaecol.</i></span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">103</span>–<span class="lpage" tagx="lpage" title="lpage">106</span>.</span></li> <li tag="ref"><a name="B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramani</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">V. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandal</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pareek</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaushik</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Host-virus protein interaction network reveals the involvement of multiple host processes in the life cycle of hepatitis E virus.</span><span class="source" tagx="source" title="source"><i>mSystems</i></span><span class="volume" tagx="volume" title="volume">3</span>:<span class="issue" tagx="issue" title="issue">e00135-17</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/mSystems.00135-17">10.1128/mSystems.00135-17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29404423">29404423</a></span></span></li> <li tag="ref"><a name="B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Surjit</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jameel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lal</span><span class="given-names" tagx="given-names" title="given-names">S. K.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Cytoplasmic localization of the ORF2 protein of hepatitis E virus is dependent on its ability to undergo retrotranslocation from the endoplasmic reticulum.</span><span class="source" tagx="source" title="source"><i>J. Virol.</i></span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">3339</span>–<span class="lpage" tagx="lpage" title="lpage">3345</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02039-06">10.1128/JVI.02039-06</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17229684">17229684</a></span></span></li> <li tag="ref"><a name="B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szklarczyk</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franceschini</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyder</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forslund</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heller</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huerta-Cepas</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">STRING v10: protein-protein interaction networks, integrated over the tree of life.</span><span class="source" tagx="source" title="source"><i>Nucleic Acids Res.</i></span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">D447</span>–<span class="lpage" tagx="lpage" title="lpage">D452</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku1003">10.1093/nar/gku1003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25352553">25352553</a></span></span></li> <li tag="ref"><a name="B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szkolnicka</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pollán</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da Silva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oechslin</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouttenoire</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moradpour</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span> (<span class="year" tagx="year" title="year">2019</span>). <span class="mixed-article-title" title="mixed-article-title">Recombinant <i>Hepatitis</i> E viruses harboring tags in the ORF1 Protein.</span><span class="source" tagx="source" title="source"><i>J Virol.</i></span><span class="volume" tagx="volume" title="volume">93</span>:<span class="issue" tagx="issue" title="issue">JVI.00459-19</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00459-19">10.1128/JVI.00459-19</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31315997">31315997</a></span></span></li> <li tag="ref"><a name="B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhat</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roytenberg</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindberg</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gazzara</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallory</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Co-regulatory activity of hnRNP K and NS1-BP in influenza and human mRNA splicing.</span><span class="source" tagx="source" title="source"><i>Nat. Commun</i>.</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="issue" tagx="issue" title="issue">2407</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-04779-4">10.1038/s41467-018-04779-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29921878">29921878</a></span></span></li> <li tag="ref"><a name="B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van</span><span class="given-names" tagx="given-names" title="given-names">de Garde MDB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pas</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van</span><span class="given-names" tagx="given-names" title="given-names">Oord GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gama</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de</span><span class="given-names" tagx="given-names" title="given-names">Man RA, et al.</span></span></span> (<span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Interferon-alpha treatment rapidly clears <i>Hepatitis</i> E virus infection in humanized mice.</span><span class="source" tagx="source" title="source"><i>Sci. Rep.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">8267</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-07434-y">10.1038/s41598-017-07434-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28811492">28811492</a></span></span></li> <li tag="ref"><a name="B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moniruzzaman</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shuffle</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lourie</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasnain</span><span class="given-names" tagx="given-names" title="given-names">S. Z.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Immune regulation of the unfolded protein response at the mucosal barrier in viral infection.</span><span class="source" tagx="source" title="source"><i>Clin. Transl. Immunol.</i></span><span class="volume" tagx="volume" title="volume">7</span>:<span class="issue" tagx="issue" title="issue">e1014</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/cti2.1014">10.1002/cti2.1014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29632667">29632667</a></span></span></li> <li tag="ref"><a name="B46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">hnRNP A2/B1 interacts with influenza A viral protein NS1 and inhibits virus replication potentially through suppressing NS1 RNA/protein levels and NS1 mRNA nuclear export.</span><span class="source" tagx="source" title="source"><i>Virology</i></span><span class="volume" tagx="volume" title="volume">449</span><span class="fpage" tagx="fpage" title="fpage">53</span>–<span class="lpage" tagx="lpage" title="lpage">61</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2013.11.009">10.1016/j.virol.2013.11.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24418537">24418537</a></span></span></li> <li tag="ref"><a name="B47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Will</span><span class="given-names" tagx="given-names" title="given-names">C. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lührmann</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Spliceosome structure and function.</span><span class="source" tagx="source" title="source"><i>Cold Spring Harb. Perspect. Biol.</i></span><span class="volume" tagx="volume" title="volume">3</span>:<span class="issue" tagx="issue" title="issue">a003707</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/cshperspect.a003707">10.1101/cshperspect.a003707</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21441581">21441581</a></span></span></li> <li tag="ref"><a name="B48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Open reading frame 3 of genotype 1 hepatitis E virus inhibits nuclear factor-κappa B signaling induced by tumor necrosis factor-α in human A549 lung epithelial cells.</span><span class="source" tagx="source" title="source"><i>PLoS One</i></span><span class="volume" tagx="volume" title="volume">9</span>:<span class="issue" tagx="issue" title="issue">e100787</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0100787">10.1371/journal.pone.0100787</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24959724">24959724</a></span></span></li> <li tag="ref"><a name="B49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller-Jensen</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Systems biology of virus-host signaling network interactions.</span><span class="source" tagx="source" title="source"><i>BMB Rep.</i></span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">213</span>–<span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"><a href="https://dx.doi.org/10.5483/bmbrep.2012.45.4.213">10.5483/bmbrep.2012.45.4.213</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22531130">22531130</a></span></span></li> <li tag="ref"><a name="B50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yewdell</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennink</span><span class="given-names" tagx="given-names" title="given-names">J. R.</span></span></span> (<span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Mechanisms of viral interference with MHC class I antigen processing and presentation.</span><span class="source" tagx="source" title="source"><i>Annu. Rev. Cell. Dev. Biol.</i></span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">579</span>–<span class="lpage" tagx="lpage" title="lpage">606</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.cellbio.15.1.579">10.1146/annurev.cellbio.15.1.579</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10611973">10611973</a></span></span></li> </ul> </div> </div>  </body></html>