<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/embed/graphic-6.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/17/2020.03.15.992883.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.15.992883v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.15.992883"/>
  <meta name="DC.Date" content="2020-03-17"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This article is a US Government work. It is not subject to copyright under 17 USC 105 and is also made available for use under a CC0 license"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="SARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19— within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus’s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the “closed” prefusion S structure, but is accessible in “open” conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses. HIGHLIGHTS High resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD). Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies. Structure of the SARS-COV-2 RBD in complex with antibody CR3022. Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike."/>
  <meta name="DC.Contributor" content="M. Gordon Joyce"/>
  <meta name="DC.Contributor" content="Rajeshwer S. Sankhala"/>
  <meta name="DC.Contributor" content="Wei-Hung Chen"/>
  <meta name="DC.Contributor" content="Misook Choe"/>
  <meta name="DC.Contributor" content="Hongjun Bai"/>
  <meta name="DC.Contributor" content="Agnes Hajduczki"/>
  <meta name="DC.Contributor" content="Lianying Yan"/>
  <meta name="DC.Contributor" content="Spencer L. Sterling"/>
  <meta name="DC.Contributor" content="Caroline E. Peterson"/>
  <meta name="DC.Contributor" content="Ethan C. Green"/>
  <meta name="DC.Contributor" content="Clayton Smith"/>
  <meta name="DC.Contributor" content="Natalia de Val"/>
  <meta name="DC.Contributor" content="Mihret Amare"/>
  <meta name="DC.Contributor" content="Paul Scott"/>
  <meta name="DC.Contributor" content="Eric D. Laing"/>
  <meta name="DC.Contributor" content="Christopher C. Broder"/>
  <meta name="DC.Contributor" content="Morgane Rolland"/>
  <meta name="DC.Contributor" content="Nelson L. Michael"/>
  <meta name="DC.Contributor" content="Kayvon Modjarrad"/>
  <meta name="article:published_time" content="2020-03-17"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;SARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19—within several months of its initial identification. Comparable to the first SARS-CoV, this coronaviruses surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the closed prefusion S structure, but is accessible in open conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.15.992883v1"/>
  <meta name="citation_id" content="2020.03.15.992883v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992883v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992883v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992883v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.15.992883"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.15.992883"/>
  <meta name="citation_author" content="M. Gordon Joyce"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_email" content="gjoyce@eidresearch.org"/>
  <meta name="citation_author_email" content="kayvon.modjarrad.civ@mail.mil"/>
  <meta name="citation_author" content="Rajeshwer S. Sankhala"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Wei-Hung Chen"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Misook Choe"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Hongjun Bai"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Agnes Hajduczki"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Lianying Yan"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Uniformed Services University"/>
  <meta name="citation_author" content="Spencer L. Sterling"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Uniformed Services University"/>
  <meta name="citation_author" content="Caroline E. Peterson"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Ethan C. Green"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Uniformed Services University"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Clayton Smith"/>
  <meta name="citation_author_institution" content="Center for Molecular Microscopy, Center for Cancer Research, National Cancer Institute, National Institutes of Health"/>
  <meta name="citation_author_institution" content="Cancer Research Technology Program, Frederick National Laboratory for Cancer Research, Leidos Biomedical Research Inc."/>
  <meta name="citation_author" content="Natalia de Val"/>
  <meta name="citation_author_institution" content="Center for Molecular Microscopy, Center for Cancer Research, National Cancer Institute, National Institutes of Health"/>
  <meta name="citation_author_institution" content="Cancer Research Technology Program, Frederick National Laboratory for Cancer Research, Leidos Biomedical Research Inc."/>
  <meta name="citation_author" content="Mihret Amare"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Paul Scott"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Eric D. Laing"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Uniformed Services University"/>
  <meta name="citation_author" content="Christopher C. Broder"/>
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Uniformed Services University"/>
  <meta name="citation_author" content="Morgane Rolland"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="U.S. Military HIV Research Program, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Henry M. Jackson Foundation for the Advancement of Military Medicine"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Nelson L. Michael"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author" content="Kayvon Modjarrad"/>
  <meta name="citation_author_institution" content="Emerging Infectious Diseases Branch, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_institution" content="Center for Infectious Diseases Research, Walter Reed Army Institute of Research"/>
  <meta name="citation_author_email" content="gjoyce@eidresearch.org"/>
  <meta name="citation_author_email" content="kayvon.modjarrad.civ@mail.mil"/>
  <meta name="citation_reference" content="Bai, H., Li, Y., Michael, N.L., Robb, M.L., and Rolland, M. (2019). The breadth of HIV-1 neutralizing antibodies depends on the conservation of key sites in their epitopes. PLoS Comput Biol 15, e1007056."/>
  <meta name="citation_reference" content="Bangaru, S., Lang, S., Schotsaert, M., Vanderven, H.A., Zhu, X., Kose, N., Bombardi, R., Finn, J.A., Kent, S.J., Gilchuk, P., et al. (2019). A Site of Vulnerability on the Influenza Virus Hemagglutinin Head Domain Trimer Interface. Cell 177, 1136–1152 e1118."/>
  <meta name="citation_reference" content="Barber, C.B., D.P. Dobkin, and H. Huhdanpaa (1996). The quickhull algorithm for convex hulls. ACM Transactions on Mathematical Software 22, 469–483."/>
  <meta name="citation_reference" content="Chan, Y.P., Yan, L., Feng, Y.R., and Broder, C.C. (2009). Preparation of recombinant viral glycoproteins for novel and therapeutic antibody discovery. Methods in molecular biology 525, 31-58, xiii."/>
  <meta name="citation_reference" content="Ge, X.Y., Li, J.L., Yang, X.L., Chmura, A.A., Zhu, G., Epstein, J.H., Mazet, J.K., Hu, B., Zhang, W., Peng, C., et al. (2013). Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538."/>
  <meta name="citation_reference" content="Hamelryck, T., and Manderick, B. (2003). PDB file parser and structure class implemented in Python. Bioinformatics 19, 2308–2310."/>
  <meta name="citation_reference" content="Hu, B., Zeng, L.P., Yang, X.L., Ge, X.Y., Zhang, W., Li, B., Xie, J.Z., Shen, X.R., Zhang, Y.Z., Wang, N., et al. (2017). Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS pathogens 13, e1006698."/>
  <meta name="citation_reference" content="Letko, M., Marzi, A., and Munster, V. (2020). Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses. Nature microbiology."/>
  <meta name="citation_reference" content="Li, F. (2016). Structure, Function, and Evolution of Coronavirus Spike Proteins. Annual review of virology 3, 237–261."/>
  <meta name="citation_reference" content="Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., Wang, W., Song, H., Huang, B., Zhu, N., et al. (2020). Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574."/>
  <meta name="citation_reference" content="Menachery, V.D., Yount, B.L., Jr., Debbink, K., Agnihothram, S., Gralinski, L.E., Plante, J.A., Graham, R.L., Scobey, T., Ge, X.Y., Donaldson, E.F., et al. (2015). A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence. Nature medicine 21, 1508–1513."/>
  <meta name="citation_reference" content="Munster, V.J., Koopmans, M., van Doremalen, N., van Riel, D., and de Wit, E. (2020). A Novel Coronavirus Emerging in China - Key Questions for Impact Assessment. The New England journal of medicine 382, 692–694."/>
  <meta name="citation_reference" content="ter Meulen, J., van den Brink, E.N., Poon, L.L., Marissen, W.E., Leung, C.S., Cox, F., Cheung, C.Y., Bakker, A.Q., Bogaards, J.A., van Deventer, E., et al. (2006). Human monoclonal antibody combination against SARS coronavirus: synergy and coverage of escape mutants. PLoS Med 3, e237."/>
  <meta name="citation_reference" content="Tian, X., Li, C., Huang, A., Xia, S., Lu, S., Shi, Z., Lu, L., Jiang, S., Yang, Z., Wu, Y., et al. (2020). Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerging microbes &amp;; infections 9, 382–385."/>
  <meta name="citation_reference" content="Tripp, R.A., Haynes, L.M., Moore, D., Anderson, B., Tamin, A., Harcourt, B.H., Jones, L.P., Yilla, M., Babcock, G.J., Greenough, T., et al. (2005). Monoclonal antibodies to SARS-associated coronavirus (SARS-CoV): Identification of neutralizing and antibodies reactive to S, N, M and E viral proteins. Journal of Virological Methods."/>
  <meta name="citation_reference" content="Wang, L., Shi, W., Chappell, J.D., Joyce, M.G., Zhang, Y., Kanekiyo, M., Becker, M.M., van Doremalen, N., Fischer, R., Wang, N., et al. (2018). Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape. Journal of virology 92."/>
  <meta name="citation_reference" content="Wang, L., Shi, W., Joyce, M.G., Modjarrad, K., Zhang, Y., Leung, K., Lees, C.R., Zhou, T., Yassine, H.M., Kanekiyo, M., et al. (2015). Evaluation of candidate vaccine approaches for MERS-CoV. Nature communications 6, 7712."/>
  <meta name="citation_reference" content="West, B.R., Moyer, C.L., King, L.B., Fusco, M.L., Milligan, J.C., Hui, S., and Saphire, E.O. (2018). Structural Basis of Pan-Ebolavirus Neutralization by a Human Antibody against a Conserved, yet Cryptic Epitope. mBio 9."/>
  <meta name="citation_reference" content="Wrapp, D., Wang, N., Corbett, K.S., Goldsmith, J.A., Hsieh, C.L., Abiona, O., Graham, B.S., and McLellan, J.S. (2020). Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science."/>
  <meta name="citation_reference" content="Wu, F., Zhao, S., Yu, B., Chen, Y.M., Wang, W., Song, Z.G., Hu, Y., Tao, Z.W., Tian, J.H., Pei, Y.Y., et al. (2020). A new coronavirus associated with human respiratory disease in China. Nature."/>
  <meta name="citation_reference" content="Yan, R., Zhang, Y., Guo, Y., Xia, L., and Zhou, Q. (2020a). Structural basis for the recognition of the 2019-nCoV by human ACE2. BiorXiv."/>
  <meta name="citation_reference" content="Yan, R., Zhang, Y., Li, Y., Xia, L., Guo, Y., and Zhou, Q. (2020b). Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science."/>
  <meta name="citation_reference" content="Yang, X.L., Hu, B., Wang, B., Wang, M.N., Zhang, Q., Zhang, W., Wu, L.J., Ge, X.Y., Zhang, Y.Z., Daszak, P., et al. (2015). Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus. Journal of virology 90, 3253–3256."/>
  <meta name="citation_reference" content="Zhou, P., Yang, X.L., Wang, X.G., Hu, B., Zhang, L., Zhang, W., Si, H.R., Zhu, Y., Li, B., Huang, C.L., et al. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature."/>
  <meta name="twitter:title" content="A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="SARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19— within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus’s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the “closed” prefusion S structure, but is accessible in “open” conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses. HIGHLIGHTS High resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD). Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies. Structure of the SARS-COV-2 RBD in complex with antibody CR3022. Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike."/>
  <meta name="og-title" property="og:title" content="A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.15.992883v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="SARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19— within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus’s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the “closed” prefusion S structure, but is accessible in “open” conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses. HIGHLIGHTS High resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD). Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies. Structure of the SARS-COV-2 RBD in complex with antibody CR3022. Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-17"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A Cryptic Site of Vulnerability on the Receptor Binding Domain of the SARS-CoV-2 Spike Glycoprotein | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1199868 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">SUMMARY</h2>
    <p id="p-2">SARS-CoV-2 is a zoonotic virus that has caused a pandemic of severe respiratory disease—COVID-19— within several months of its initial identification. Comparable to the first SARS-CoV, this novel coronavirus’s surface Spike (S) glycoprotein mediates cell entry via the human ACE-2 receptor, and, thus, is the principal target for the development of vaccines and immunotherapeutics. Molecular information on the SARS-CoV-2 S glycoprotein remains limited. Here we report the crystal structure of the SARS-CoV-2 S receptor-binding-domain (RBD) at a the highest resolution to date, of 1.95 Å. We identified a set of SARS-reactive monoclonal antibodies with cross-reactivity to SARS-CoV-2 RBD and other betacoronavirus S glycoproteins. One of these antibodies, CR3022, was previously shown to synergize with antibodies that target the ACE-2 binding site on the SARS-CoV RBD and reduce viral escape capacity. We determined the structure of CR3022, in complex with the SARS-CoV-2 RBD, and defined a broadly reactive epitope that is highly conserved across betacoronaviruses. This epitope is inaccessible in the “closed” prefusion S structure, but is accessible in “open” conformations. This first-ever resolution of a human antibody in complex with SARS-CoV-2 and the broad reactivity of this set of antibodies to a conserved betacoronavirus epitope will allow antigenic assessment of vaccine candidates, and provide a framework for accelerated vaccine, immunotherapeutic and diagnostic strategies against SARS-CoV-2 and related betacoronaviruses.</p>
    <div id="sec-1" class="subsection">
     <p id="p-3">
      <strong>HIGHLIGHTS</strong> High resolution structure of the SARS-CoV-2 Receptor-Binding-Domain (RBD).
     </p>
     <p id="p-4">Recognition of the SARS-CoV-2 RBD by SARS-CoV antibodies.</p>
     <p id="p-5">Structure of the SARS-COV-2 RBD in complex with antibody CR3022.</p>
     <p id="p-6">Identification of a cryptic site of vulnerability on the SARS-CoV-2 Spike.</p>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-7">The emergence of SARS-CoV-2 marks the seventh coronavirus to be isolated from humans, and the third to cause a severe disease—named COVID-19—after severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS)(
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Munster et al., 2020</a>). The rapid spread of SARS-CoV-2, and the grave risk it poses to global health, prompted the World Health Organization to declare, on 30 January 2020, the COVID-19 outbreak to be a public health emergency of international concern and on 11 March 2020 to be a pandemic(
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Wu et al., 2020</a>; 
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Zhou et al., 2020</a>). As of 13 March 2020, there have been nearly 140000 SARS-CoV-2 infections and more than 5000 associated deaths reported across at least 100 countries. The rapidly evolving epidemiology of the pandemic and absence of licensed prophylactics or therapeutics for the disease have accelerated the need to elucidate the molecular biology of this novel coronavirus.
    </p>
    <p id="p-8">Although SARS-CoV-2 is a newly identified virus, it shares genetic and morphologic features with others in the 
     <em>Coronaviridae</em> family, particularly those from the Betacoronavirus genus. The genome of the recently isolated SARS-CoV-2 shares 82% nucleotide identity with human SARS-CoV and 89% with bat SARS-like-CoVZXC21 (
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Lu et al., 2020</a>). The spike (S) glycoprotein, in particular, bears significant structural homology with SARS-CoV compared to other coronaviruses such as MERS-CoV. Like SARS-CoV, the surface Spike (S) glycoprotein of SARS-CoV-2 binds the same host receptor, ACE-2, to mediate cell entry (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Letko et al., 2020</a>; 
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Yan et al., 2020a</a>). S—a class I fusion protein—is also a critical determinant of viral host range and tissue tropism and the primary target of the host immune response (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Li, 2016</a>). As such, most coronavirus vaccine candidates are based on S or one of its sub-components. Coronavirus S glycoproteins contain three segments: a large ectodomain, a single-pass transmembrane anchor and a short intracellular tail. The ectodomain consists of a receptor-binding subunit, S1, which contains two sub-domains: one at the N-terminus and the other at the C-terminus. The latter comprises the receptor-binding domain (RBD), which serves the vital function of attaching the virus to the host receptor and triggering a conformational change in the protein that results in fusion with the host cell membrane through the S2 subunit.
    </p>
    <p id="p-9">Recently, the molecular structure of recombinant full-length SARS-CoV-2 Spike protein was solved in a stabilized pre-fusion state, by single particle cryo-Electron Microscopy (cryo-EM), at a resolution of 3.8 Å(
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Wrapp et al., 2020</a>). Despite the comprehensive structural characterization of the spike protein as a whole, movement of the RBD between “up” and “down” conformational states prevented complete modeling of the RBD domains. Subsequent cryo-EM investigations of SARS-CoV-2 provided more detail of RBD, particularly at sites that contact the human ACE-2 receptor (
     <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Yan et al., 2020a</a>). Here, we report the first high resolution—less than 2 Å—SARS-CoV-2 RBD. Additionally, we present the antigenicity of this recombinant RBD, particularly of interest, given the equipoise in the literature regarding the binding affinities of SARS-CoV antibodies for SARS-CoV-2 RBD. Early reports, have described that the human SARS-CoV antibody, CR3022, is able to bind to the SARS-CoV-2 RBD. In the present study, we verify binding, and subsequently solved the structure of SARS-CoV-2 RBD in complex with CR3022 with a novel “cryptic” epitope.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">RESULTS</h2>
    <div id="sec-4" class="subsection">
     <h3>High resolution structure of the SARS-CoV-2 RBD</h3>
     <p id="p-10">The SARS-CoV-2 RBD (residues 313-532), with a C-terminal His-tag, was expressed in 293F cells, and purified by NiNTA affinity, and size-exclusion chromatography. Crystallization condition screening identified 20% Jeffamine D2000, 10% Jeffamine M2005, 0.2 M NaCl, 0.1M MES pH 5.5 for diffraction quality crystal growth. Crystals diffracted to &lt;;1.8 Å in group P 4
      <sub>1</sub> 2
      <sub>1</sub> 2 and to a complete dataset to 1.95 Å that could be scaled and processed (
      <strong>
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
      </strong>). The structure was refined to an R
      <sub>free</sub> of 20% and R
      <sub>work</sub> of 22% with no Ramachandran outliers. S residues 313-532 were clearly interpretable from the electron density map, with a dual conformation of a loop containing residues 484 to 487 clearly visible in the electron density map (
      <strong>
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>
      </strong>). Structure comparison of the unliganded RBD structure presented here, with the stabilized prefusion SARS-CoV-2 Spike (S-2P) molecule structure determined by Cryo-EM (PDB ID: 6VSB) (
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Wrapp et al., 2020</a>) shows high structural similarity, with an RMSD of 0.68, 0.68, and 0.71 for each of the spike protomers. In the structure of the S-2P molecule (S-2P) (
      <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Wrapp et al., 2020</a>) 25, 29 or 49 amino acids (aa) within each protomer RBD are not modeled, including 40% of the ACE-2 receptor binding site as measured by buried surface area (BSA)(
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Yan et al., 2020b</a>). The SARS-CoV RBD-2 compared to liganded (PDB ID: 2AJF) and unliganded (PDB ID: 2GHV) SARS-CoV RBD structures shows high structural similarity, except for residues 473-488 (
      <strong>
       <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>
      </strong>). A chimeric SARS-CoV-2 RBD structure (PDB ID: 6VW1) with 23 aa differences compared to SARS-CoV-2, in complex with human ACE-2 was recently released in the PDB. Comparisons with previously published structures with the SARS-CoV-2 RBD highlight residues 473-488 as an area with significant structural plasticity. In the RBD structure from this study, we observe electron density for two conformations of the 482-486 loop. One of these conformations is highly similar to the ligand bound form of the RBD, while the second conformation would clash with the ACE2 receptor. Both this structural detail of the unliganded RBD and comparison to previously described RBD structures indicates that this area of the RBD is structurally malleable with implications for antibody or small molecule therapeutics design.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1203011/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1203011/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1203011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1</span>
       <p id="p-11" class="first-child">Crystallographic Data Collection and Refinement Statistics</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Crystal structure of the SARS-CoV-2 Receptor Binding Domain (RBD). A The SARS-CoV-2 RBD is shown in ribbon representation, glycan N343 is shown in stick representation, with the N- and C-termini shown as spheres. The Receptor-binding motif (residues 437-507) is colored forest green. B The SARS-CoV-2 RBD structure is overlaid with related structures including the incomplete RBD structure taken from the S-2P trimer structure (PDB ID: 6VSB), SARS-CoV RBD (PDB ID: 2AJF), and a chimeric SARS-CoV-2 RBD structure (PDB ID: 6VW1). C Close-up view of the membrane distal region of the RBD (residues 471-491). D Sequence alignment of SARS-CoV-2 and SARS-CoV RBDs. The RBD secondary structure is displayed above the sequence alignment Residues with significant structural difference &gt;; 2 [[#xC5]] are highlighted in purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Crystal structure of the SARS-CoV-2 Receptor Binding Domain (RBD).&lt;;/span&gt;; &lt;;strong&gt;;A&lt;;/strong&gt;; The SARS-CoV-2 RBD is shown in ribbon representation, glycan N343 is shown in stick representation, with the N- and C-termini shown as spheres. The Receptor-binding motif (residues 437-507) is colored forest green. &lt;;strong&gt;;B&lt;;/strong&gt;; The SARS-CoV-2 RBD structure is overlaid with related structures including the incomplete RBD structure taken from the S-2P trimer structure (PDB ID: 6VSB), SARS-CoV RBD (PDB ID: 2AJF), and a chimeric SARS-CoV-2 RBD structure (PDB ID: 6VW1). &lt;;strong&gt;;C&lt;;/strong&gt;; Close-up view of the membrane distal region of the RBD (residues 471-491). &lt;;strong&gt;;D&lt;;/strong&gt;; Sequence alignment of SARS-CoV-2 and SARS-CoV RBDs. The RBD secondary structure is displayed above the sequence alignment Residues with significant structural difference &gt;; 2 [[#xC5]] are highlighted in purple.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F1.medium.gif" width="440" height="347"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F1.medium.gif" width="440" height="347"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Crystal structure of the SARS-CoV-2 Receptor Binding Domain (RBD).</span>
       <p id="p-12" class="first-child">
        <strong>A</strong> The SARS-CoV-2 RBD is shown in ribbon representation, glycan N343 is shown in stick representation, with the N- and C-termini shown as spheres. The Receptor-binding motif (residues 437-507) is colored forest green. 
        <strong>B</strong> The SARS-CoV-2 RBD structure is overlaid with related structures including the incomplete RBD structure taken from the S-2P trimer structure (PDB ID: 6VSB), SARS-CoV RBD (PDB ID: 2AJF), and a chimeric SARS-CoV-2 RBD structure (PDB ID: 6VW1). 
        <strong>C</strong> Close-up view of the membrane distal region of the RBD (residues 471-491). 
        <strong>D</strong> Sequence alignment of SARS-CoV-2 and SARS-CoV RBDs. The RBD secondary structure is displayed above the sequence alignment Residues with significant structural difference &gt;; 2 Å are highlighted in purple.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Identification of a set of cross-reactive SARS-CoV-2 antibodies</h3>
     <p id="p-13">In an effort to identify antibodies that could bind to SARS-CoV-2, we screened a set of SARS-CoV,(
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Tripp et al., 2005</a>) and MERS CoV(
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Wang et al., 2018</a>; 
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Wang et al., 2015</a>) RBD-reactive antibodies for binding to the SARS-CoV-2 RBD. We demonstrated that the SARS-CoV mouse antibody 240CD (
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Tripp et al., 2005</a>) had nanomolar (nM) affinity for the SARS-CoV-2 RBD and did not significantly block ACE-2 receptor binding (
      <strong>
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>
      </strong>). CR3022—a SARS-CoV neutralizing antibody (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Tian et al., 2020</a>) identified from a human phage-display library (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">ter Meulen et al., 2006</a>)—also bound to SARS-CoV-2 RBD with nM affinity (
      <strong>
       <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>
      </strong>). We assessed competition binding between 240CD and CR3022, and show that these antibodies cross-compete with each other for binding to the SARS-CoV-2 RBD (
      <strong>
       <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C, 2D</a>
      </strong>).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Antigenic characterization of SARS-CoV-2 RBD. A,B, Binding kinetics of 240CD and CR3022 to SARS-CoV-2 RBD measured by biolayer interferometry. Kinetic constants were determined were calculated using a minimum of four dilutions of the RBD and fitted using a 1:1 binding model. C,D, Competition binding of antibodies CR3022 and 240CD to SARS-CoV-2 RBD. CR3022 or control antibody was allowed to bind to SARS-COV-2 prior to binding to 240CD or vice-versa. E SARS-CoV-2 RBD was sequentially bound by antibodies CR3022 or 240CD followed by soluble human ACE2 receptor. F SARS-CoV reactive antibodies were assessed for binding to bat SARS-related CoV Rs4784 and Rs4231 S glycoproteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Antigenic characterization of SARS-CoV-2 RBD.&lt;;/span&gt;; &lt;;strong&gt;;A&lt;;/strong&gt;;,&lt;;strong&gt;;B&lt;;/strong&gt;;, Binding kinetics of 240CD and CR3022 to SARS-CoV-2 RBD measured by biolayer interferometry. Kinetic constants were determined were calculated using a minimum of four dilutions of the RBD and fitted using a 1:1 binding model. &lt;;strong&gt;;C&lt;;/strong&gt;;,&lt;;strong&gt;;D&lt;;/strong&gt;;, Competition binding of antibodies CR3022 and 240CD to SARS-CoV-2 RBD. CR3022 or control antibody was allowed to bind to SARS-COV-2 prior to binding to 240CD or vice-versa. &lt;;strong&gt;;E&lt;;/strong&gt;; SARS-CoV-2 RBD was sequentially bound by antibodies CR3022 or 240CD followed by soluble human ACE2 receptor. &lt;;strong&gt;;F&lt;;/strong&gt;; SARS-CoV reactive antibodies were assessed for binding to bat SARS-related CoV Rs4784 and Rs4231 S glycoproteins.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F2.medium.gif" width="318" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F2.medium.gif" width="318" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Antigenic characterization of SARS-CoV-2 RBD.</span>
       <p id="p-14" class="first-child">
        <strong>A</strong>,
        <strong>B</strong>, Binding kinetics of 240CD and CR3022 to SARS-CoV-2 RBD measured by biolayer interferometry. Kinetic constants were determined were calculated using a minimum of four dilutions of the RBD and fitted using a 1:1 binding model. 
        <strong>C</strong>,
        <strong>D</strong>, Competition binding of antibodies CR3022 and 240CD to SARS-CoV-2 RBD. CR3022 or control antibody was allowed to bind to SARS-COV-2 prior to binding to 240CD or vice-versa. 
        <strong>E</strong> SARS-CoV-2 RBD was sequentially bound by antibodies CR3022 or 240CD followed by soluble human ACE2 receptor. 
        <strong>F</strong> SARS-CoV reactive antibodies were assessed for binding to bat SARS-related CoV Rs4784 and Rs4231 S glycoproteins.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-15">SARS-CoV-2 has a likely zoonotic origin and horseshoe bats have been implicated as natural reservoirs of both SARS-CoV and SARS-CoV-2 (
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Menachery et al., 2015</a>; 
      <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Zhou et al., 2020</a>). As such, we next explored antibody cross-reactivity with the S glycoproteins of two bat SARS-related CoVs: SARSr-CoV Rs4874 (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ge et al., 2013</a>; 
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Yang et al., 2015</a>) and Rs4231 (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Hu et al., 2017</a>), which are closely related to the progenitor of SARS-CoV and retain the ability to utilize human ACE-2. CR3022 was able to recognize a recombinant Spike glycoproteins generated from bat SARSr-CoV Rs4874, while 240CD, and other mouse generated monoclonal antibodies have a mixed recognition phenotype (
      <strong>
       <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2F</a>
      </strong>).
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Crystal structure of antibody CR3022 in complex with SARS-CoV-2 RBD</h3>
     <p id="p-16">The antigenic cross-reactivity of this set of antibodies precipitated an investigation into their molecular recognition determinants. The potential relevance of a human antibody motivated the investigation to prioritize studies of CR3022, for which a sequence was available (
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">ter Meulen et al., 2006</a>). The CR3022 heavy chain is encoded by IGHV5-51*03, contains a 12-aa CDR H3 with 8 V gene-encoded residues altered by somatic hypermutation. CR3022 light chain is encoded by IGKV4-1*01 with 1 V gene-encoded residue, altered by somatic hypermutation, and a 9-aa CDR L3 (
      <strong>Extended Data Figure 2A</strong>). To provide an atomic-level understanding of the structure of the CR3022 antibody, we crystallized the antigen-binding fragment (Fab) of CR3022. Crystals diffracted to 3.2 Å resolution in space group P 2
      <sub>1</sub> (
      <strong>
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>
      </strong>). Overall the structure of the CR3022 Fab revealed a relatively flat antigen-combining site, with the exception of an extended protruding 12-aa CDR L1 loop (
      <strong>
       <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S2A</a>
      </strong>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequence alignment of the receptor binding domain of SARS-CoV and related betacoronaviruses measured by Biolayer Interferometry for reactivity to SARS-CoV antibodies. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Sequence alignment of the receptor binding domain of SARS-CoV and related betacoronaviruses measured by Biolayer Interferometry for reactivity to SARS-CoV antibodies.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F5.medium.gif" width="440" height="206"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F5.medium.gif" width="440" height="206"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. S1.</span>
       <span class="caption-title">Sequence alignment of the receptor binding domain of SARS-CoV and related betacoronaviruses measured by Biolayer Interferometry for reactivity to SARS-CoV antibodies.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CR3022 Fab crystal structure and epitope analysis. A CR3022 is shown in ribbon representation, with CDR loops indicated. Residues which have undergone somatic hypermutation are indicated with green spheres. B Heavy and light chain CDR loops paratope buried surface area. C Buried surface area analysis of the CR3022 epitope on the SARS-CoV-2 RBD. D Antibody binding to SARS-CoV-2 RBD and RBD with a glycan at position 384. RBD is initially loaded onto the HIS probe, followed by incubation with RBD variants." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;CR3022 Fab crystal structure and epitope analysis.&lt;;/span&gt;; &lt;;strong&gt;;A&lt;;/strong&gt;; CR3022 is shown in ribbon representation, with CDR loops indicated. Residues which have undergone somatic hypermutation are indicated with green spheres. &lt;;strong&gt;;B&lt;;/strong&gt;; Heavy and light chain CDR loops paratope buried surface area. &lt;;strong&gt;;C&lt;;/strong&gt;; Buried surface area analysis of the CR3022 epitope on the SARS-CoV-2 RBD. &lt;;strong&gt;;D&lt;;/strong&gt;; Antibody binding to SARS-CoV-2 RBD and RBD with a glycan at position 384. RBD is initially loaded onto the HIS probe, followed by incubation with RBD variants.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F6.medium.gif" width="440" height="276"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F6.medium.gif" width="440" height="276"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. S2.</span>
       <span class="caption-title">CR3022 Fab crystal structure and epitope analysis.</span>
       <p id="p-46" class="first-child">
        <strong>A</strong> CR3022 is shown in ribbon representation, with CDR loops indicated. Residues which have undergone somatic hypermutation are indicated with green spheres. 
        <strong>B</strong> Heavy and light chain CDR loops paratope buried surface area. 
        <strong>C</strong> Buried surface area analysis of the CR3022 epitope on the SARS-CoV-2 RBD. 
        <strong>D</strong> Antibody binding to SARS-CoV-2 RBD and RBD with a glycan at position 384. RBD is initially loaded onto the HIS probe, followed by incubation with RBD variants.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">To determine the structure of CR3022 in complex with the SARS-CoV-2 RBD, we carried out crystallization conditions screening, with crystals of the CR3022-RBD complex forming in 1M Succinic acid, 0.1M Hepes pH 7, 2% PEG MME2000 and determined the crystal structure by X-ray diffraction to 4.25 Å (
      <strong>
       <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>
      </strong>). The complex structure was solved by molecular replacement using the refined CR3022 and SARS-CoV-2 RBD structures as search models and was refined to an R
      <sub>work</sub>/R
      <sub>free</sub> of 0.242/0.292 (
      <strong>
       <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>
      </strong>). CR3022 bound to the RBD at an epitope centered on S glycoprotein residues 377-386 with a total buried surface area of 871 Å (
      <strong>
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, 
       <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S2</a>
      </strong>, and 
      <strong>
       <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1</a>
      </strong>). This region is highly conserved between SARS-CoV and SARS-CoV-2 (
      <strong>
       <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S3</a>
      </strong>). Comparison of the CR3022 epitope site with previously described antibody-complex structures for SARS-CoV, and MERS-CoV indicates that CR3022 describes a novel recognition site (
      <strong>
       <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>
      </strong> and 
      <strong>
       <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figs. S3</a>, 
       <a id="xref-fig-8-1" class="xref-fig" href="#F8">S4</a>
      </strong>). Further sequence analysis of the epitope indicates that this epitope is conserved in betacoronavirus clade 2b, with also some similarity in clade 2d (
      <strong>
       <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. S3</a>
      </strong>). To confirm that this site was also shared with 240CD, we produced an RBD knockout mutant by introducing a glycan sequon at position 384, and by biolayer interferometry show that both CR3022 and 240CD binding to the RBD can be eliminated by the introduction of a glycan at this site (
      <strong>
       <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. S2D</a>
      </strong>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1203009/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1203009/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S1.</span>
       <p id="p-44" class="first-child">Buried surface area of CR3022 in complex with SARS-CoV-2 RBD</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural and sequence analysis of the CR3022 epitope. Analysis of the CR3022 footprint across betacoronaviruses. The CR3022 epitope on SARS-CoV-2 (China.Wuhan.30Dec19.402132) RBD is compared across betacoronaviruses. The epitope is numbered according to the Wuhan reference; the strength of the interaction between the Ab and the spike protein is indicated by the height and color of the histogram bars above the sequence alignment. Sequences are ordered based on their phylogenetic relationships based on a maximum likelihood phylogenetic tree derived from amino acid RBD sequences. The RBD structure is shown in surface representation and depicts mutations between SARS-CoV-1 and SARS-CoV-2 in red; the CR3022 epitope is outlined in white, with contact residues shown in stick representation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Structural and sequence analysis of the CR3022 epitope.&lt;;/span&gt;; Analysis of the CR3022 footprint across betacoronaviruses. The CR3022 epitope on SARS-CoV-2 (China.Wuhan.30Dec19.402132) RBD is compared across betacoronaviruses. The epitope is numbered according to the Wuhan reference; the strength of the interaction between the Ab and the spike protein is indicated by the height and color of the histogram bars above the sequence alignment. Sequences are ordered based on their phylogenetic relationships based on a maximum likelihood phylogenetic tree derived from amino acid RBD sequences. The RBD structure is shown in surface representation and depicts mutations between SARS-CoV-1 and SARS-CoV-2 in red; the CR3022 epitope is outlined in white, with contact residues shown in stick representation.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F7.medium.gif" width="440" height="413"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F7.medium.gif" width="440" height="413"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. S3.</span>
       <span class="caption-title">Structural and sequence analysis of the CR3022 epitope.</span>
       <p id="p-47" class="first-child">Analysis of the CR3022 footprint across betacoronaviruses. The CR3022 epitope on SARS-CoV-2 (China.Wuhan.30Dec19.402132) RBD is compared across betacoronaviruses. The epitope is numbered according to the Wuhan reference; the strength of the interaction between the Ab and the spike protein is indicated by the height and color of the histogram bars above the sequence alignment. Sequences are ordered based on their phylogenetic relationships based on a maximum likelihood phylogenetic tree derived from amino acid RBD sequences. The RBD structure is shown in surface representation and depicts mutations between SARS-CoV-1 and SARS-CoV-2 in red; the CR3022 epitope is outlined in white, with contact residues shown in stick representation.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV-2 and SARS-CoV antibody epitope analysis. A Sequence alignment of SARS-CoV-2 and SARS-CoV RBD. Antibody and ACE2 receptor interacting residues are highlighted on the sequence. B Antibody epitopes and receptor binding sites are outlined on the surface of the SARS-CoV RBD structures. The Sequence differences between SARS-COV-2 and SARS-CoV are indicated by red coloring on the RBD surface. C Antibody epitopes are shown on the SARS-CoV RBD molecule depicted in ribbon representation. The CR3022 epitope is shown on the SARS-CoV-2 RBD molecule." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SARS-CoV-2 and SARS-CoV antibody epitope analysis.&lt;;/span&gt;; &lt;;strong&gt;;A&lt;;/strong&gt;; Sequence alignment of SARS-CoV-2 and SARS-CoV RBD. Antibody and ACE2 receptor interacting residues are highlighted on the sequence. &lt;;strong&gt;;B&lt;;/strong&gt;; Antibody epitopes and receptor binding sites are outlined on the surface of the SARS-CoV RBD structures. The Sequence differences between SARS-COV-2 and SARS-CoV are indicated by red coloring on the RBD surface. &lt;;strong&gt;;C&lt;;/strong&gt;; Antibody epitopes are shown on the SARS-CoV RBD molecule depicted in ribbon representation. The CR3022 epitope is shown on the SARS-CoV-2 RBD molecule.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F8.medium.gif" width="426" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. S4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F8.medium.gif" width="426" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. S4.</span>
       <span class="caption-title">SARS-CoV-2 and SARS-CoV antibody epitope analysis.</span>
       <p id="p-48" class="first-child">
        <strong>A</strong> Sequence alignment of SARS-CoV-2 and SARS-CoV RBD. Antibody and ACE2 receptor interacting residues are highlighted on the sequence. 
        <strong>B</strong> Antibody epitopes and receptor binding sites are outlined on the surface of the SARS-CoV RBD structures. The Sequence differences between SARS-COV-2 and SARS-CoV are indicated by red coloring on the RBD surface. 
        <strong>C</strong> Antibody epitopes are shown on the SARS-CoV RBD molecule depicted in ribbon representation. The CR3022 epitope is shown on the SARS-CoV-2 RBD molecule.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Crystal structure of CR3022 in complex with SARS-CoV-2 RBD. A, RBD and CR3022 are shown in cartoon representation. B Structure of the SARS-CoV-2 RBD shown in surface representation. Residues which differ between SARS-CoV-2 and SARS-CoV are colored red. The CR3022 epitope is outlined in blue, and Thr-430 and Phe-384 located within the epitope are labeled. C The location of antibody CR3022, 230S, 80R, m396 and F26G19 epitopes and ACE-2 binding site on the RBD are outlined on the surface of the SARS-CoV-2 RBD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Crystal structure of CR3022 in complex with SARS-CoV-2 RBD.&lt;;/span&gt;; &lt;;strong&gt;;A&lt;;/strong&gt;;, RBD and CR3022 are shown in cartoon representation. &lt;;strong&gt;;B&lt;;/strong&gt;; Structure of the SARS-CoV-2 RBD shown in surface representation. Residues which differ between SARS-CoV-2 and SARS-CoV are colored red. The CR3022 epitope is outlined in blue, and Thr-430 and Phe-384 located within the epitope are labeled. &lt;;strong&gt;;C&lt;;/strong&gt;; The location of antibody CR3022, 230S, 80R, m396 and F26G19 epitopes and ACE-2 binding site on the RBD are outlined on the surface of the SARS-CoV-2 RBD.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F3.medium.gif" width="440" height="261"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F3.medium.gif" width="440" height="261"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Crystal structure of CR3022 in complex with SARS-CoV-2 RBD.</span>
       <p id="p-18" class="first-child">
        <strong>A</strong>, RBD and CR3022 are shown in cartoon representation. 
        <strong>B</strong> Structure of the SARS-CoV-2 RBD shown in surface representation. Residues which differ between SARS-CoV-2 and SARS-CoV are colored red. The CR3022 epitope is outlined in blue, and Thr-430 and Phe-384 located within the epitope are labeled. 
        <strong>C</strong> The location of antibody CR3022, 230S, 80R, m396 and F26G19 epitopes and ACE-2 binding site on the RBD are outlined on the surface of the SARS-CoV-2 RBD.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Identification of a cryptic site of vulnerability recognized by CR3022</h3>
     <p id="p-19">The epitope conservation within the clade explains the antigenic cross-reactivity with both human SARS-CoV and bat SARS related CoV. To date, there has been extensive structural characterization of the SARS-CoV, and MERS-CoV spike molecule and domains, which provides a framework for understanding the novel SARS-CoV-2 spike molecule (
      <strong>
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S2</a>
      </strong>). In the context of the coronavirus trimeric S glycoproteins, the RBD displays two prototypical conformations either in an “up” or “down” position, with implications for receptor binding and cell entry (
      <strong>
       <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S1</a>
      </strong>). To further analyze these conformations, we modeled the CR3022 binding to the trimeric structures of SARS-CoV-2, SARS-CoV and MERS-CoV. The CR3022 epitope is occluded by adjacent spike protomers when the RBD is in the “down” conformation, but becomes more accessible when the spike is in a more open conformation here multiple RBD molecules are in the “up” conformation (
      <strong>
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>
      </strong>). There is still a clash of the antibody Fc1 region with the NTD from the same protomer, or an RBD from an adjacent protomer when modeled using the static structure.
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1203013/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1203013/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table S2</span>
       <p id="p-45" class="first-child">Protein DataBank files of SARS-CoV, MERS-CoV, and SARS-CoV-2 S structures</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Identification of CR3022 epitope as a [[#x201C]]cryptic[[#x201D]] epitope. Structural alignment of the SARS-CoV-2 RBD-CR3022 complex with the SARS-CoV S-2P structure. A The RBD-CR3022 structure is aligned to the SARS-CoV trimer structure (surface representation; PDB ID: 6CS1), where two RBD molecules are located in the [[#x201C]]up[[#x201D]] conformation. In this static structure, the Fc1 region of CR3022 (ribbon representation) clashes with the NTD of the same protomer. However, the epitope is fully accessible when more than one RBD is in the [[#x201C]]up[[#x201D]] representation. B Biolayer interferometry measurement of CR3022 binding to SARS S proteins with trypsin treatment or ACE2 receptor binding. C CR3022 binding to a serial dilution of SARS S-2P protein following trypsin treatment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-258585156" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Identification of CR3022 epitope as a [[#x201C]]cryptic[[#x201D]] epitope.&lt;;/span&gt;; Structural alignment of the SARS-CoV-2 RBD-CR3022 complex with the SARS-CoV S-2P structure. &lt;;strong&gt;;A&lt;;/strong&gt;; The RBD-CR3022 structure is aligned to the SARS-CoV trimer structure (surface representation; PDB ID: 6CS1), where two RBD molecules are located in the [[#x201C]]up[[#x201D]] conformation. In this static structure, the Fc1 region of CR3022 (ribbon representation) clashes with the NTD of the same protomer. However, the epitope is fully accessible when more than one RBD is in the [[#x201C]]up[[#x201D]] representation. &lt;;strong&gt;;B&lt;;/strong&gt;; Biolayer interferometry measurement of CR3022 binding to SARS S proteins with trypsin treatment or ACE2 receptor binding. &lt;;strong&gt;;C&lt;;/strong&gt;; CR3022 binding to a serial dilution of SARS S-2P protein following trypsin treatment.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F4.medium.gif" width="440" height="336"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F4.medium.gif" width="440" height="336"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/17/2020.03.15.992883/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">Identification of CR3022 epitope as a “cryptic” epitope.</span>
       <p id="p-20" class="first-child">Structural alignment of the SARS-CoV-2 RBD-CR3022 complex with the SARS-CoV S-2P structure. 
        <strong>A</strong> The RBD-CR3022 structure is aligned to the SARS-CoV trimer structure (surface representation; PDB ID: 6CS1), where two RBD molecules are located in the “up” conformation. In this static structure, the Fc1 region of CR3022 (ribbon representation) clashes with the NTD of the same protomer. However, the epitope is fully accessible when more than one RBD is in the “up” representation. 
        <strong>B</strong> Biolayer interferometry measurement of CR3022 binding to SARS S proteins with trypsin treatment or ACE2 receptor binding. 
        <strong>C</strong> CR3022 binding to a serial dilution of SARS S-2P protein following trypsin treatment.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">To understand whether CR3022 could bind to SARS-CoV S glycoproteins, we measured binding to stabilized S-2P or non-stabilized versions of S (
      <strong>
       <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>
      </strong>). We observed robust binding to the non-stabilized S glycoprotein, while binding to SARS S-2P Trimer was low. We then treated the SARS S-2P trimer with trypsin and/or incubation with the ACE2 receptor to assess whether minimal proteolytic action or receptor binding could increase the availability of the “cryptic” CR3022 epitope. Incubation of the stabilized S-2P trimer with human ACE2 did not dramatically affect CR3022 binding, while in contrast, the trypsin treatment of the S-2P protein resulted in increased binding akin to the unstabilized S glycoprotein binding, and the level of binding was titratable, with increasing amounts of S-2P resulting in higher CR3022 binding. Given the prior neutralization and protection studies utilizing CR3022, and its ability to complement potent neutralizing antibodies, it is likely that the CR3022 epitope represents a “cryptic” epitope that becomes exposed during the processes of viral cell entry.
     </p>
     <p id="p-22">In summary, our data represents the most detailed structural information for the SARS-CoV-2 RBD to date and the first structure of the SARS-CoV-2 in complex with a human antibody. The presence of “cryptic” but protective epitopes for influenza (
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bangaru et al., 2019</a>), and Ebola viruses (
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">West et al., 2018</a>), have been previously described. The identification of a novel “cryptic” epitope for betacoronaviruses including SARS-CoV, and SARS-CoV-2 highlight a novel viral vulnerability that can be harnessed in combination with ACE2 receptor site targeting monoclonal antibodies for vaccine and therapeutic countermeasure development.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">METHODS AND MATERIALS</h2>
    <div class="section" id="sec-9">
     <h3>Production of recombinant proteins</h3>
     <p id="p-24">The Shanghai Public Health Clinical Center &amp;; School of Public Health, in collaboration with the Central Hospital of Wuhan, Huazhong University of Science and Technology, the Wuhan Center for Disease Control and Prevention, the National Institute for Communicable Disease Control and Prevention, Chinese Center for Disease Control, and the University of Sydney, Sydney, Australia released the sequence of a coronavirus genome from a case of a respiratory disease from Wuhan on January 10
      <sup>th</sup> available at recombinomics.co/topic/4351-wuhan-coronavirus-2019-ncov-sequences/. The sequence was also deposited in GenBank (accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom">MN908947</a>) and GISAID (&gt;;EPI_ISL_402125). DNA encoding the SARS-Cov-2 RBD (residues 331-527) was synthesized (Genscript) with a C-terminal His6 purification tag and cloned into a CMVR plasmid, and protein was expressed by transient transfection in 293F cells for six days. The SARS-CoV-2 RBD-His protein was purified from cell culture supernatant using a Ni-NTA (Qiagen) affinity column. DNA encoding the S protein ectodomains (residues 1-1194) from bat SARS-related CoV isolates Rs4231 and Rs4874 (ref.(
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Hu et al., 2017</a>)) were synthesized (Genscript) with a C-terminal T4-Foldon domain or C-terminal GCN domain, respectively, followed by factor xA cleavage sites and Strep-Tactin purification tags. Bat SARSr-CoV S genes were cloned into a modified pcDNA3.1 expression plasmid (
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chan et al., 2009</a>). Protein was initially expressed by transient transfection in 293F cells for six days, then serial cloned to select stably expressing cell lines (Yan L., in submission). The Rs4231-T4 and Rs4874-GCN S proteins were purified from cell culture supernatant using a Strep-Tactin affinity column. The oligomeric structure of these S proteins was selected by size exclusion chromatography (GE/AKTA) and trimeric S proteins were confirmed by Native-PAGE. SARS S-2P was produced as previously described, with Strep-Tactin affinity chromatography followed by gel filtration using a 16/60 Superdex-200 purification column. Purification purity for all S glycoproteins was assessed by SDS-PAGE.
     </p>
     <p id="p-25">The sequences of the CR3022 variable regions of the heavy and light chains are available in GenBank under accession numbers 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ168569&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom">DQ168569</a> and 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ168570&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom">DQ168570</a>, respectively (
      <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">ter Meulen et al., 2006</a>). These sequences were synthesized (Genscript) and cloned into CMVR expression vectors (NIH AIDS reagent program) between a murine Ig leader (GenBank 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=DQ407610&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom">DQ407610</a>) and the constant regions of human IgG1 (GenBank AAA02914), Igκ (GenBank AKL91145). Plasmids encoding heavy and light chains were co-transfected into Expi293F cells (ThermoFisher) according to the manufacturer’s instructions. After 5 days, antibodies were purified from cleared culture supernatants with Protein A agarose (ThermoFisher) using standard procedures, buffer exchanged into Phosphate-Buffered Saline (PBS), and quantified using calculated E and A280 measurements.
     </p>
     <p id="p-26">The Fab fragment of antibody CR3022 was prepared by digestion of the full-length IgG using enzyme Lys-C (Roche). The digestion reaction was allowed to proceed for 2.5 hours at 37°C. Digestion was assessed by SDS-PAGE and upon completion, the reaction mixture was passed through protein-G beads (0.5-1 ml beads), 3 times and the final flow through was assessed by SDS-PAGE for purity. The Fab fragment was mixed with purified SARS-CoV-2 RBD, and the complex was allowed to form for 1 hour at room temperature.</p>
    </div>
    <div class="section" id="sec-10">
     <h3>Sequence information</h3>
     <p id="p-27">SARS-CoV-2 RBD (signal peptide is underlined, purification tag in italics)</p>
     <p id="p-28">
      <span class="underline">MDSKGSSQKGSRLLLLLVVSNLLLPQGVVG</span>NITNLCPFGEVFNATRFASVYAWNRKRISNCVAD YSVLYNSASFSTFKCYGVSPTKLNDLCFTNVYADSFVIRGDEVRQIAPGQTGKIADYNYKLPDDF TGCVIAWNSNNLDSKVGGNYNYLYRLFRKSNLKPFERDISTEIYQAGSTPCNGVEGFNCYFPLQS YGFQPTNGVGYQPYRVVVLSFELLHAPATVCG
      <em>PGSHHHHHH</em>
     </p>
     <p id="p-29">CR3022 Heavy chain Fv</p>
     <p id="p-30">EVQLVQSGTEVKKPGESLKISCKGSGYGFITYWIGWVRQMPGKGLEWMGIIYPGDSETRYSPSFQ GQVTISADKSINTAYLQWSSLKASDTAIYYCAGGSGISTPMDVWGQGTTVTVSS</p>
     <p id="p-31">CR3022 Light chain Fv</p>
     <p id="p-32">DIVMTQSPDSLAVSLGERATINCKSSQSVLYSSINKNYLAWYQQKPGQPPKLLIYWASTRESGVP DRFSGSGSGTDFTLTISSLQAEDVAVYYCQQYYSTPYTFGQGTKVEIK</p>
    </div>
    <div class="section" id="sec-11">
     <h3>Cell lines</h3>
     <p id="p-33">Expi293F (ThermoFisher Scientific #A14527), and 293F cell lines were utilized in this study.</p>
    </div>
    <div class="section" id="sec-12">
     <h3>X-ray Crystallography</h3>
     <div class="section" id="sec-13">
      <h4>Crystallization</h4>
      <p id="p-34">SARS-CoV-2 RBD at 10 mg/ml and 5 mg/ml in PBS buffer was screened for crystallization conditions using an Art Robbins Gryphon crystallization robot, 0.2 ul drops, and a set of 1200 crystallization conditions. Crystal drops were observed using a Jan Scientific UVEX-PS with automated UV and brightfield drop imaging robot. Crystals of the SARS-CoV-2 RBD grew after 24 hours in multiple conditions from the Molecular Dimensions MIDAS crystal screen, with diffraction-quality crystals seen in conditions B1, G1, F6, and H10. CR3022 Fab was screened for crystallization at 10.0 mg/ml and 5.0 mg/ml concentrations in PBS. Diffraction quality crystals grew after 48 hours in 0.1M Imidazole pH 6.5, 40% 2-propanol and 15% PEG 8,000. For the complex, CR3022 Fab and SARS-CoV-2 RBD were mixed in 1:1 molar ratio and crystallization drops were set-up at 8.0 and 4.0 mg/ml concentrations in PBS buffer as described above. Crystals grew in a crystallization condition containing 1M Succinic acid, 0.1M HEPES pH 7.0 and 2% PEG MME2000. Both, RBD alone and CR3022 Fab-RBD complex, crystals were harvested and cryo-cooled in their respective crystallization conditions plus 25% glycerol.</p>
     </div>
     <div class="section" id="sec-14">
      <h4>Diffraction data collection and processing</h4>
      <p id="p-35">Single crystals were transferred to mother liquor containing 22% glycerol, and cryo-cooled in liquid nitrogen prior to data collection. Diffraction data for SARS-CoV-2 RBD were collected at Advanced Photon Source (APS), Argonne National Laboratory, NE-CAT ID24-C beamline, and measured using a Dectris Eiger 16M PIXEL detector. Crystals grown in MIDAS condition B1 (20% Jeffamine D2000, 10% Jeffamine M2005, 0.2 M NaCl, 0.1M MES pH 5.5) provided the highest resolution diffraction with spots visible to 1.8 Å. A complete dataset could be processed to 1.95 Å in space group P41212. CR3022 Fab crystals diffracted to 3.3 Å on NE-CAT ID24-C beamline. Diffraction data could be scaled in P21 space group with 99.9% completeness. Diffraction data for CR3022 and SARS-CoV-2 RBD complex were collected on NE-CAT ID24-C beamline at Advanced Photon Source (APS), and measured using a Dectris Eiger 16M PIXEL detector. Diffraction data from multiple crystals were merged and scaled together to achieve a final resolution of 4.2 Å with overall completeness of 82.2%. Data collection statistics are reported in 
       <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>.
      </p>
     </div>
     <div class="section" id="sec-15">
      <h4>Structure solution and refinement</h4>
      <p id="p-36">Phenix xtriage was used to analyze the scaled diffraction data produced from HKL2000 and XDS. Data was analyzed for completeness, Matthew’s coefficient, twinning or pseudo-translational pathology. The structure of the SARS-CoV-2 RBD was determined by molecular replacement using Phaser and a search model of the SARS RBD (PDB ID: 2AJF, molecule C). CR3022 Fab crystal structure was determined by molecular replacement using Coxsackievirus A6 neutralizing antibody 1D5 (PDB ID: 5XS7) as a search model. The CR3022-RBD complex structure was determined by molecular replacement using the refined CR3022 and SARS-CoV-2 RBD structures as search models. Refinement was carried out using Phenix refine with positional, global isotropic B factor refinement, and defined TLS groups, with iterative cycles of manual model building using COOT. Structure quality was assessed with MolProbity. The final refinement statistics for all the structures are reported in 
       <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>. All structure figures were generated using PyMOL (The PyMOL Molecular Graphics System [DeLano Scientific]).
      </p>
     </div>
    </div>
    <div class="section" id="sec-16">
     <h3>Structure comparisons</h3>
     <div class="section" id="sec-17">
      <h4>Weighing epitope sites based on antigen-antibody interactions</h4>
      <p id="p-37">Epitope sites correspond to antigen sites that are in contact with the antibody in the antigen-antibody complex (i.e. all sites that have non-hydrogen atoms within 4 Å of the antibody). For a given epitope site, the weight, which characterizes the interaction between the epitope site and the antibody (improved based on (
       <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bai et al., 2019</a>)), was defined as: 
       <span class="disp-formula" id="disp-formula-1">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/17/2020.03.15.992883/embed/graphic-6.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/17/2020.03.15.992883/embed/graphic-6.gif"/>
         </noscript>
        </span>
       </span>in which, 
       <em>n</em>
       <sub>
        <em>c</em>
       </sub> is the number of contacts with the antibody (i.e. the number of non-hydrogen antibody atoms within 4 Å of the site); 
       <em>n</em>
       <sub>
        <em>nb</em>
       </sub> is the number of neighboring antibody residues; ⟨
       <em>n</em>
       <sub>
        <em>c</em>
       </sub>⟩ is the mean number of contacts 
       <em>n</em>
       <sub>
        <em>c</em>
       </sub> and ⟨
       <em>n</em>
       <sub>
        <em>nb</em>
       </sub>⟩ is the mean number of neighboring antibody residues 
       <em>n</em>
       <sub>
        <em>nb</em>
       </sub> across all epitope sites. A weight of 1.0 is attributed to the average interaction across all epitope sites. Neighboring residue pairs were identified by Delaunay tetrahedralization of side-chain centers of residues (C
       <sub>α</sub> is counted as a side chain atom, pairs further than 8.5 Å were excluded). Quickhull(Barber, 1996) was used for the tetrahedralization and Biopython PDB (
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hamelryck and Manderick, 2003</a>) to handle the protein structure. In the SARS-CoV-2 and SARS-CoV-1 RBD comparison, residues were considered similar for the following residues pairs: RK, RQ, KQ, QE, QN, ED, DN, TS, SA, VI, IL, LM and FY.
      </p>
     </div>
    </div>
    <div class="section" id="sec-18">
     <h3>Biolayer interferometry</h3>
     <p id="p-38">Affinity kinetic interactions between SARS-CoV-2 RBD proteins and antibodies were monitored on an Octet RED96 instrument (FortéBio). After reference subtraction, binding kinetic constants were determined, from at least 4 concentrations of Fab, by fitting the curves to a 1:1 Langmuir binding model using the Data analysis software 9.0 (FortéBio). Antibodies were loaded at 30 ug/ml onto a AHC probe for 120 s followed by baseline incubation for 30-60 s.</p>
     <p id="p-39">To assess antibody competition, either 240CD or CR3022 or a non-specific control antibody CR1-07 was incubated with the SARS-CoV-2 RBD prior to assessment of binding to CR3022 or 240CD. Antibody concentration was 30 ug/ml. To assess binding of human ACE-2 receptor in the presence or absence of antibodies CR3022, or 240CD, RBD was loaded onto a HIS probe. The RBD was then sequentially incubated with either CR3022, 240CD or control antibody CR1-07 prior to incubation with human ACE-2 receptor.</p>
     <p id="p-40">CR3022 was loaded onto an AHC probe for 120s prior to incubation with SARS-CoV S glycoproteins (15 ug/ml) alone or pre-incubated with ACE2 protein. SARS S-2P protein was treated with 0.1% bovine pancreas trypsin for 10 minutes prior to binding to binding measurements. SARS Spike protein was provided by BEI resources, Lot 768P152. Binding of CR3022 was also carried out against a series of concentrations of SARS S-2P which had been treated with 0.1% w/w bovine pancreatic trypsin.</p>
    </div>
   </div>
   <div class="section" id="sec-19">
    <h2 class="">DATA AVAILABILITY</h2>
    <p id="p-41">The associated accession numbers for the coordinates and structure factors reported in this paper are being deposited to the PDB.</p>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">ADDITIONAL RESOURCES</h2>
    <p id="p-42">None</p>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">DECLARATION OF INTERESTS</h2>
    <p id="p-43">No potential conflict of interest was reported by the authors.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-23">This work was supported by funding from the Defense Health Agency, as well as a cooperative agreement (W81XWH-07-2-0067) between the Henry M. Jackson Foundation for the Advancement of Military Medicine, Inc., and the U.S. Department of Defense (DoD). Funding from Biological Defense Research Directorate of the Naval Medical Research Center (HT9404-13-1-0021; Component Project: Soluble Trimeric Filovirus Envelope Glycoproteins) and Defense Threat Reduction Agency (HDTRA1-17-C-0019; Chulalongkorn Luminex Training and Research Preparedness) to CCB. X-ray diffraction data were collected at beamlines ID-24-C and ID-24-E at the Advanced Photon Source, Argonne National Laboratory. The Northeastern Collaborative Access Team (NE-CAT) beamlines are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P41 GM103403) at the Advanced Photon Source, Argonne National Laboratory. Reagents were obtained through BEI Resources, NIAID, NIH: Monoclonal Anti-SARS-CoV S glycoprotein (Similar to 240C), NR-616; (Similar to 341C), NR-617; (Similar to 540C), NR-618. We thank the authors who made their SARS-CoV-2 genome sequences available through GISAID or GenBank. The opinions or assertions contained herein are the private views of the authors, and are not to be construed as official, or as reflecting true views of the Department of the Army or the Department of Defense.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bai</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michael</span>, 
          <span class="cit-name-given-names">N.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Robb</span>, 
          <span class="cit-name-given-names">M.L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rolland</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">The breadth of HIV-1 neutralizing antibodies depends on the conservation of key sites in their epitopes</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">e1007056</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D15%26rft.spage%253De1007056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.2" data-doi="10.1016/j.cell.2019.04.011">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bangaru</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schotsaert</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vanderven</span>, 
          <span class="cit-name-given-names">H.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kose</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bombardi</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Finn</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kent</span>, 
          <span class="cit-name-given-names">S.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gilchuk</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">A Site of Vulnerability on the Influenza Virus Hemagglutinin Head Domain Trimer Interface</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">177</span>, 
         <span class="cit-fpage">1136</span>–
         <span class="cit-lpage">1152</span> e1118.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1136%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.04.011%26rft_id%253Dinfo%253Apmid%252F31100268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.04.011&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31100268&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.15.992883v1.3" data-doi="10.1145/235815.235821">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Barber</span>, 
          <span class="cit-name-given-names">C.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.P.</span>
          <span class="cit-name-surname">Dobkin</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Huhdanpaa</span>
         </span> (
         <span class="cit-pub-date">1996</span>). 
         <span class="cit-article-title">The quickhull algorithm for convex hulls</span>. 
         <abbr class="cit-jnl-abbrev">ACM Transactions on Mathematical Software</abbr>
         <span class="cit-vol">22</span>, 
         <span class="cit-fpage">469</span>–
         <span class="cit-lpage">483</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DACM%2BTransactions%2Bon%2BMathematical%2BSoftware%26rft.volume%253D22%26rft.spage%253D469%26rft_id%253Dinfo%253Adoi%252F10.1145%252F235815.235821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1145/235815.235821&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">Y.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>, 
          <span class="cit-name-given-names">Y.R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Broder</span>, 
          <span class="cit-name-given-names">C.C.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">Preparation of recombinant viral glycoproteins for novel and therapeutic antibody discovery</span>. 
         <abbr class="cit-jnl-abbrev">Methods in molecular biology</abbr>
         <span class="cit-vol">525</span>, 
         <span class="cit-fpage">31</span>-
         <span class="cit-lpage">58</span>, xiii.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%26rft.volume%253D525%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.5" data-doi="10.1038/nature12711">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">J.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chmura</span>, 
          <span class="cit-name-given-names">A.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Epstein</span>, 
          <span class="cit-name-given-names">J.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mazet</span>, 
          <span class="cit-name-given-names">J.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>). 
         <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">503</span>, 
         <span class="cit-fpage">535</span>–
         <span class="cit-lpage">538</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24172901&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000327464200045&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.6" data-doi="10.1093/bioinformatics/btg299">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hamelryck</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Manderick</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> (
         <span class="cit-pub-date">2003</span>). 
         <span class="cit-article-title">PDB file parser and structure class implemented in Python</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">2308</span>–
         <span class="cit-lpage">2310</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHamelryck%26rft.auinit1%253DT.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2308%26rft.epage%253D2310%26rft.atitle%253DPDB%2Bfile%2Bparser%2Band%2Bstructure%2Bclass%2Bimplemented%2Bin%2BPython%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg299%26rft_id%253Dinfo%253Apmid%252F14630660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg299&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14630660&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000186919200017&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.7" data-doi="10.1371/journal.ppat.1006698">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>, 
          <span class="cit-name-given-names">L.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">J.Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>, 
          <span class="cit-name-given-names">X.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>). 
         <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">e1006698</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Letko</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marzi</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Nature microbiology</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. 
         <abbr class="cit-jnl-abbrev">Annual review of virology</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">261</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bvirology%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.10" data-doi="10.1016/S0140-6736(20)302518">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">565</span>–
         <span class="cit-lpage">574</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32007145&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.11" data-doi="10.1038/nm.3985">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Menachery</span>, 
          <span class="cit-name-given-names">V.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yount</span>, 
          <span class="cit-name-given-names">B.L.</span>, 
          <span class="cit-name-suffix">Jr.</span>.
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Debbink</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Agnihothram</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gralinski</span>, 
          <span class="cit-name-given-names">L.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Plante</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>, 
          <span class="cit-name-given-names">R.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scobey</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Donaldson</span>, 
          <span class="cit-name-given-names">E.F.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</span>. 
         <abbr class="cit-jnl-abbrev">Nature medicine</abbr>
         <span class="cit-vol">21</span>, 
         <span class="cit-fpage">1508</span>–
         <span class="cit-lpage">1513</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.volume%253D21%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3985%26rft_id%253Dinfo%253Apmid%252F26552008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nm.3985&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26552008&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.12" data-doi="10.1056/NEJMp2000929">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>, 
          <span class="cit-name-given-names">V.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koopmans</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Doremalen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Riel</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">de Wit</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">A Novel Coronavirus Emerging in China - Key Questions for Impact Assessment</span>. 
         <abbr class="cit-jnl-abbrev">The New England journal of medicine</abbr>
         <span class="cit-vol">382</span>, 
         <span class="cit-fpage">692</span>–
         <span class="cit-lpage">694</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BNew%2BEngland%2Bjournal%2Bof%2Bmedicine%26rft.volume%253D382%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMp2000929%26rft_id%253Dinfo%253Apmid%252F31978293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMp2000929&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31978293&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.13" data-doi="10.1371/journal.pmed.0030237">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">ter Meulen</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van den Brink</span>, 
          <span class="cit-name-given-names">E.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marissen</span>, 
          <span class="cit-name-given-names">W.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>, 
          <span class="cit-name-given-names">C.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cox</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheung</span>, 
          <span class="cit-name-given-names">C.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bakker</span>, 
          <span class="cit-name-given-names">A.Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bogaards</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Deventer</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2006</span>). 
         <span class="cit-article-title">Human monoclonal antibody combination against SARS coronavirus: synergy and coverage of escape mutants</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Med</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">e237</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BMed%26rft.aulast%253DTer%2BMeulen%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De237%26rft.epage%253De237%26rft.atitle%253DHuman%2Bmonoclonal%2Bantibody%2Bcombination%2Bagainst%2BSARS%2Bcoronavirus%253A%2Bsynergy%2Band%2Bcoverage%2Bof%2Bescape%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pmed.0030237%26rft_id%253Dinfo%253Apmid%252F16796401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pmed.0030237&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16796401&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. 
         <abbr class="cit-jnl-abbrev">Emerging microbes &amp;; infections</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">382</span>–
         <span class="cit-lpage">385</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Bmicrobes%2B%2526%2Binfections%26rft.volume%253D9%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tripp</span>, 
          <span class="cit-name-given-names">R.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haynes</span>, 
          <span class="cit-name-given-names">L.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moore</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tamin</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harcourt</span>, 
          <span class="cit-name-given-names">B.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>, 
          <span class="cit-name-given-names">L.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yilla</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Babcock</span>, 
          <span class="cit-name-given-names">G.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greenough</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2005</span>). 
         <span class="cit-article-title">Monoclonal antibodies to SARS-associated coronavirus (SARS-CoV): Identification of neutralizing and antibodies reactive to S, N, M and E viral proteins</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virological Methods</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chappell</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joyce</span>, 
          <span class="cit-name-given-names">M.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kanekiyo</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Becker</span>, 
          <span class="cit-name-given-names">M.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Doremalen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fischer</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">92</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joyce</span>, 
          <span class="cit-name-given-names">M.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Modjarrad</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lees</span>, 
          <span class="cit-name-given-names">C.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yassine</span>, 
          <span class="cit-name-given-names">H.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kanekiyo</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">Evaluation of candidate vaccine approaches for MERS-CoV</span>. 
         <abbr class="cit-jnl-abbrev">Nature communications</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">7712</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D6%26rft.spage%253D7712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">West</span>, 
          <span class="cit-name-given-names">B.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moyer</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">King</span>, 
          <span class="cit-name-given-names">L.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fusco</span>, 
          <span class="cit-name-given-names">M.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Milligan</span>, 
          <span class="cit-name-given-names">J.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Saphire</span>, 
          <span class="cit-name-given-names">E.O.</span>
         </span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">Structural Basis of Pan-Ebolavirus Neutralization by a Human Antibody against a Conserved, yet Cryptic Epitope</span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>
         <span class="cit-vol">9</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corbett</span>, 
          <span class="cit-name-given-names">K.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goldsmith</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsieh</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abiona</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>, 
          <span class="cit-name-given-names">B.S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">McLellan</span>, 
          <span class="cit-name-given-names">J.S.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Y.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">Z.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>, 
          <span class="cit-name-given-names">Z.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">J.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pei</span>, 
          <span class="cit-name-given-names">Y.Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span> (
         <span class="cit-pub-date">2020a</span>). 
         <span class="cit-article-title">Structural basis for the recognition of the 2019-nCoV by human ACE2</span>. 
         <abbr class="cit-jnl-abbrev">BiorXiv</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span> (
         <span class="cit-pub-date">2020b</span>). 
         <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.23" data-doi="10.1128/JVI.02582-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">L.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Daszak</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">90</span>, 
         <span class="cit-fpage">3253</span>–
         <span class="cit-lpage">3256</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft.volume%253D90%26rft.spage%253D3253%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02582-15%26rft_id%253Dinfo%253Apmid%252F26719272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1128/JVI.02582-15&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26719272&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F17%2F2020.03.15.992883.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.15.992883v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>, 
          <span class="cit-name-given-names">H.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
