<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/05/2020.03.02.968818.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.02.968818v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.02.968818"/>
  <meta name="DC.Date" content="2020-03-05"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019."/>
  <meta name="DC.Contributor" content="Jessica E. Manning"/>
  <meta name="DC.Contributor" content="Jennifer A. Bohl"/>
  <meta name="DC.Contributor" content="Sreyngim Lay"/>
  <meta name="DC.Contributor" content="Sophana Chea"/>
  <meta name="DC.Contributor" content="Ly Sovann"/>
  <meta name="DC.Contributor" content="Yi Sengdoeurn"/>
  <meta name="DC.Contributor" content="Seng Heng"/>
  <meta name="DC.Contributor" content="Chan Vuthy"/>
  <meta name="DC.Contributor" content="Katrina Kalantar"/>
  <meta name="DC.Contributor" content="Vida Ahyong"/>
  <meta name="DC.Contributor" content="Michelle Tan"/>
  <meta name="DC.Contributor" content="Jonathan Sheu"/>
  <meta name="DC.Contributor" content="Cristina M. Tato"/>
  <meta name="DC.Contributor" content="Joseph L. DeRisi"/>
  <meta name="DC.Contributor" content="Laurence Baril"/>
  <meta name="DC.Contributor" content="Veasna Duong"/>
  <meta name="DC.Contributor" content="Philippe Dussart"/>
  <meta name="DC.Contributor" content="Erik A. Karlsson"/>
  <meta name="article:published_time" content="2020-03-05"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia9s first case of COVID-2019.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.02.968818v1"/>
  <meta name="citation_id" content="2020.03.02.968818v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.02.968818v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.02.968818v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.02.968818v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.02.968818"/>
  <meta name="citation_num_pages" content="7"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.02.968818"/>
  <meta name="citation_author" content="Jessica E. Manning"/>
  <meta name="citation_author_institution" content="Laboratory of Malaria and Vector Research, US National Institute of Allergy and Infectious Diseases"/>
  <meta name="citation_author_email" content="jessica.manning@nih.gov"/>
  <meta name="citation_author" content="Jennifer A. Bohl"/>
  <meta name="citation_author_institution" content="Laboratory of Malaria and Vector Research, US National Institute of Allergy and Infectious Diseases"/>
  <meta name="citation_author" content="Sreyngim Lay"/>
  <meta name="citation_author_institution" content="Laboratory of Malaria and Vector Research, US National Institute of Allergy and Infectious Diseases"/>
  <meta name="citation_author" content="Sophana Chea"/>
  <meta name="citation_author_institution" content="Laboratory of Malaria and Vector Research, US National Institute of Allergy and Infectious Diseases"/>
  <meta name="citation_author" content="Ly Sovann"/>
  <meta name="citation_author_institution" content="Cambodian Center for Disease Control, Ministry of Health"/>
  <meta name="citation_author" content="Yi Sengdoeurn"/>
  <meta name="citation_author_institution" content="Cambodian Center for Disease Control, Ministry of Health"/>
  <meta name="citation_author" content="Seng Heng"/>
  <meta name="citation_author_institution" content="Cambodian Center for Disease Control, Ministry of Health"/>
  <meta name="citation_author" content="Chan Vuthy"/>
  <meta name="citation_author_institution" content="Cambodian Center for Disease Control, Ministry of Health"/>
  <meta name="citation_author" content="Katrina Kalantar"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Initiative"/>
  <meta name="citation_author" content="Vida Ahyong"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Biohub"/>
  <meta name="citation_author" content="Michelle Tan"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Biohub"/>
  <meta name="citation_author" content="Jonathan Sheu"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Initiative"/>
  <meta name="citation_author" content="Cristina M. Tato"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Biohub"/>
  <meta name="citation_author" content="Joseph L. DeRisi"/>
  <meta name="citation_author_institution" content="Chan Zuckerberg Biohub"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4611-9205"/>
  <meta name="citation_author" content="Laurence Baril"/>
  <meta name="citation_author_institution" content="Institut Pasteur du Cambodge"/>
  <meta name="citation_author" content="Veasna Duong"/>
  <meta name="citation_author_institution" content="Virology Unit, Institut Pasteur du Cambodge"/>
  <meta name="citation_author" content="Philippe Dussart"/>
  <meta name="citation_author_institution" content="Virology Unit, Institut Pasteur du Cambodge"/>
  <meta name="citation_author" content="Erik A. Karlsson"/>
  <meta name="citation_author_institution" content="Virology Unit, Institut Pasteur du Cambodge"/>
  <meta name="citation_author_email" content="ekarlsson@pasteur-kh.org"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6004-5671"/>
  <meta name="citation_reference" content="Novel Coronavirus (2019-nCoV) situation reports. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports."/>
  <meta name="citation_reference" content="Nextstrain / ncov. https://nextstrain.org/ncov."/>
  <meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature (2020) doi: 10.1038/s41586-020-2012-7."/>
  <meta name="citation_reference" content="Li, Q. et al. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia. New England Journal of Medicine 0, null (2020)."/>
  <meta name="citation_reference" content="Munster, V. J., Koopmans, M., van Doremalen, N., van Riel, D. &amp;; de Wit, E. A Novel Coronavirus Emerging in China — Key Questions for Impact Assessment. New England Journal of Medicine 0, null (2020)."/>
  <meta name="citation_reference" content="Deng, X. et al. Metagenomic sequencing with spiked primer enrichment for viral diagnostics and genomic surveillance. Nature Microbiology 1–12 (2020) doi: 10.1038/s41564-019-0637-9."/>
  <meta name="citation_reference" content="Laboratory guidance. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance."/>
  <meta name="citation_reference" content="Gu, W., Miller, S. &amp;; Chiu, C. Y. Clinical Metagenomic Next-Generation Sequencing for Pathogen Detection. Annu Rev Pathol 14, 319–338 (2019)."/>
  <meta name="citation_reference" content="Gire, S. K. et al. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science 345, 1369–1372 (2014)."/>
  <meta name="citation_reference" content="Hoenen, T. et al. Nanopore Sequencing as a Rapidly Deployable Ebola Outbreak Tool. Emerg Infect Dis 22, 331–334 (2016)."/>
  <meta name="citation_reference" content="Armstrong, G. L. et al. Pathogen Genomics in Public Health. N Engl J Med 381, 2569–2580 (2019)."/>
  <meta name="citation_reference" content="Dozmorov, M. G. GitHub Statistics as a Measure of the Impact of Open-Source Bioinformatics Software. Front Bioeng Biotechnol 6, 198 (2018)."/>
  <meta name="twitter:title" content="Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019."/>
  <meta name="og-title" property="og:title" content="Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.02.968818v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-05"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1181735 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia’s first case of COVID-2019.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">The ongoing outbreak of a novel coronavirus in China is now evident in 37 other countries as others employ vigilant screening procedures.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup> As of 24 Feb 2020, 119 SARS-CoV-2 genome sequences from 15 countries are publicly available from over 79,331 confirmed cases.
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup> These viral genomes differ by only 0 to 5 mutations, likely representing a single introduction in the human population from an unknown reservoir or intermediate host, potentially bats.
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup> Subsequently, the virus rapidly spread since December 2019. Currently, incubation period and transmissibility are questions of global concern, but difficult to answer given nonspecific and mild symptoms that could represent a variety of respiratory viruses.
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>
    </p>
    <p id="p-4">Genetic characterization of the virus from geographically diverse patient samples is key to infer the rate of spread. Additionally, rapid and complete results from cases worldwide are important for sequence-dependent countermeasures and accurate diagnostics. However, as the virus reaches resource-limited settings in proximity to the outbreak’s epicenter, such as Cambodia, Laos, and Myanmar, there are often logistical challenges in surveillance, contact tracing, sample collection, and biospecimen transport. Information delay is also compounded by a dearth of sequencing and/or bioinformatics expertise in-country, further postponing analysis and dissemination of pathogen genomic information.</p>
    <p id="p-5">Here, in a rapidly implemented response to the novel coronavirus outbreak, we used an unbiased metagenomics next generation sequencing (mNGS) approach on the index COVID-19 case in Cambodia as a tactical demonstration of an efficient workflow to identify and characterize the novel virus. While unbiased sequencing successfully achieved an unambiguous identification of the virus in the absence of any sequence enrichment, complete coverage of the SARS-CoV-2 genome was subsequently achieved by augmenting the mNGS library preparation via a spiked primer enrichment approach.
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Viral diagnostic methods</h3>
     <p id="p-6">Nasopharyngeal and oropharyngeal swabs (combined into one tube) were collected from a symptomatic patient meeting case definition for possible infection with COVID-19 pneumonia on January 26, 2020. Extraction of viral nucleic acids from clinical sample was performed with a QIAamp Viral RNA Mini Kit (Qiagen #52906) as described by manufacturer. Extracted RNA samples were DNAse-treated and tested via real-time polymerase chain reaction for SARS-CoV-2 using both the Charité Virology and Hong Kong University protocols and confirmed positive on January 27
      <sup>th</sup>, 2020.
      <sup>
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
      </sup>
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>mNGS preparation and analysis</h3>
     <p id="p-7">On January 29, samples were received for cDNA library preparation. Since clinical specimens are obtained from suspected COVID-19 cases as part of the national outbreak response, requirement for informed consent was waived for pathogen identification and characterization. mNGS libraries were prepared from the viral RNA extracted from the sample. cDNA was converted to Illumina libraries using the NEBNext Ultra II DNA Library Prep Kit (E7645) according to the manufacturer’s recommendations. Library size and concentration were determined using the 4150 Tapestation system (Agilent, MA, USA). External RNA Controls 103 Consortium collection (ThermoFisher, 4456740) spike-in controls were used as markers of potential library preparation errors and for input RNA mass calculation. Sample was sequenced on an Illumina iSeq100 instrument using 150 nucleotide paired-end sequencing. A water (“no-template”) control was included in library preparation. Microbial sequences from the sample are located in the National Center for Biotechnology Information (NCBI) Sequence Read Archive.</p>
     <p id="p-8">Raw fastq files were uploaded to the IDseq portal, a cloud-based, open-source bioinformatics platform to identify microbes from metagenomic data (
      <a href="https://idseq.net">https://idseq.net</a>). The filter steps and microbial data are accessible at 
      <a href="http://public.idseq.net/covid-19?utm_source=bioarxiv&amp;;utm_medium=paper&amp;;utm_campaign=covid-19">http://public.idseq.net/covid-19?utm_source=bioarxiv&amp;;utm_medium=paper&amp;;utm_campaign=covid-19</a>. Potential pathogens were distinguished from commensal flora and contaminating microbial sequences from the environment and reagents by establishing a Z-score metric based on a background distribution derived from four non-templated “water-only” control libraries. Data were normalized to unique reads mapped per million input reads for each microbe at both species and genus levels. Taxa with Z-score less than 1, base pair alignment less than 50 base pairs, and reads per million (rpM) less than 10 were removed from analysis. Geneious Prime 2020.0.5 was used for separate de novo assembly and annotation of the draft SARS-CoV-2 genome. In subsequent runs to achieve higher coverage of the viral genome, two additional steps were taken: 1) the subsampling filter of the nonhost reads was removed and 2) targeted enrichment of the SARS-CoV-2 sequence using 73 tiling primers at a 10:1 molar ratio to random primers was adapted from viral genome recovery methods previously described.
      <sup>
       <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>
      </sup>
     </p>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Results</h2>
    <p id="p-9">By February 1, the SARS-CoV-2 genome was correctly identified in the index COVID-19 patient sample using mNGS and globally accessible bioinformatics. An mNGS library was prepared, multiplexed, and sequenced. Notably, the sample generated 4,773,430 reads with 1,903,800 of those being non-host reads. The resulting non-host fastq files were processed using the IDseq platform which utilizes open-source components. When processed on IDseq pipeline version 3.18 with a February 2020 NCBI index, which included 54 SARS-CoV-2 sequences deposited by the international research community, sequences from the index patient aligned to the SARS-CoV-2 reference genome (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, 582 reads, at 100% identity to the reference sequence; NCBI accession 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN985325.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom">MN985325.1</a>). Further analysis resulted in 33.2% coverage of the SARS-CoV-2 reference at an average depth of 2.2x (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Assembly and alignment on Geneious resulted in 50.5% coverage at 2.6x of the SARS-CoV-2 genome (NCBI accession 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom">MN908947.1</a>). Re-sequencing the index case to a total sequencing depth of 8,786,916 single reads resulted in a greater number of SARS-CoV-2 alignments (819 reads) but had no impact on overall coverage – 32.3% coverage breadth at an average depth of 2.7x.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="COVID19 viral genome identification and coverage A) mNGS taxa heatmap results identifying COVID-19 virus; B) Coverage visualization map of COVID19 sequences; C) Coverage visualization map after target enrichment for COVID-19 sequences using spiked primer enrichment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343676696" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;COVID19 viral genome identification and coverage&lt;;/span&gt;; A) mNGS taxa heatmap results identifying COVID-19 virus; B) Coverage visualization map of COVID19 sequences; C) Coverage visualization map after target enrichment for COVID-19 sequences using spiked primer enrichment.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818/F1.medium.gif" width="395" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818/F1.medium.gif" width="395" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/05/2020.03.02.968818/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">COVID19 viral genome identification and coverage</span>
      <p id="p-10" class="first-child">A) mNGS taxa heatmap results identifying COVID-19 virus; B) Coverage visualization map of COVID19 sequences; C) Coverage visualization map after target enrichment for COVID-19 sequences using spiked primer enrichment.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">Using the target enrichment strategy, the overall fold enrichment was 6.4x. In terms of improving genome coverage, particularly at the 5’ end of the SARS-CoV-2 genome, the resulting 3,077 reads mapping to SARS-CoV-2 assembled into a single contig which provided full coverage of the reference genome at an average depth of 14.9x (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>)(NCBI accession 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN988668.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom">MN988668.1</a>).
    </p>
    <p id="p-12">All human SARS-CoV-2 genomes are very similar, including the SARS-CoV-2 genome from the Cambodian case as we would expect given that he transited directly from Wuhan. One SNP was noted at position 25,654 in ORF3a resulting in a valine-to-leucine substitution when compared to NCBI accession 
     <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom">MN908947.1</a>.
    </p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Discussion</h2>
    <p id="p-13">Here, we demonstrate that unbiased mNGS is a feasible and efficient means to detect pathogens in the midst of an outbreak of a novel virus. The index Cambodian SARS-CoV-2 sequence data uploaded to public databases represent one of the only sequences generated in a low-to-middle income country close to the outbreak epicenter. Despite major advances in mNGS technologies and significant decreases in costs associated with sequencing, preparation of sequencing libraries and sufficient bioinformatics capabilities for timely analysis still present a challenge in the developing world.
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>
    </p>
    <p id="p-14">As cases of COVID-19 spread globally in mid-January, initial real-time PCR protocols took into account that the genetic diversity of SARS-CoV-2 in humans and animals was not completely known. Non-specificity combined with variable sample quality can hinder ability to detect the pathogen in question. In this scenario, our metagenomics approach demonstrated a first-pass high coverage of the N gene, closely aligning to diagnostic protocols utilizing both the E and N genes for primary detection. This detection directly increased on-the-ground certainty in diagnostic sensitivity and specificity of available assays.
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
     </sup>
    </p>
    <p id="p-15">mNGS in field settings proved critical in development of countermeasures for the 2014-2016 Ebola virus epidemic in West Africa, just as ongoing sequencing of COVID-19 cases from lesser developed areas of Southeast Asia will contribute to overall understanding of pathogen transmission, origin, and evolution.
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup> While COVID-19 cases are not limited to one remote region, as is often the case with viral hemorrhagic fever outbreaks, sequenced samples from all countries will be important for global disease containment. Instead of being limited to consensus Sanger sequencing, which may not detect high risk quasispecies, metagenomics is a powerful approach to detect variants of both novel and known species as epidemics evolve.
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>
    </p>
    <p id="p-16">The clear limitation in an mNGS approach is that low viral titers or high levels of host material demand greater read depth than may be available on instruments such as the iSeq100. To overcome this barrier, our follow-up steps included a target enrichment of SARS-CoV-2 while keeping the comprehensiveness of our mNGS pipeline intact.
     <sup>
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>
     </sup> For an emerging threat, this strategy offers the flexibility to successfully recover the pathogen genome in question for subsequent phylogenetic analyses without compromising discovery. However, the other key factor in mNGS success is the accessibility to open-access, cloud-based metagenomics bioinformatics pipelines, such as IDseq which automates the process of separating host sequence characterizing the remaining non-host sequences.
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup> As recent as the Ebola epidemic, bioinformatics analyses were still mainly completed in the Global North.
     <sup>
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>
     </sup> This newly available combination – more rugged, deployable sequencers plus user-friendly, globally accessible bioinformatics – represents an opportunity for responders in limited-resource settings; however, further proof-of-principle during outbreaks remains necessary.
    </p>
    <p id="p-17">Overall, agnostic or unbiased metagenomic sequencing capabilities in-country provide the ability to detect and respond to a variety of pathogens, even those that are unanticipated or unknown. Bridging of existing local and global resources for sequencing and analysis allows for better real-time surveillance locally, while also enabling better health pursuits overall, not just during outbreaks. The example described here serves as a call for continued training and infrastructure to support mNGS capacity in developing countries as bioinformatic tools proliferate and the cost of sequencing decreases.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-18">We would like to acknowledge the patient, Cambodian Ministry of Health and Cambodian Center for Disease Control, Rapid Response Team members, the Sihanoukville Provincial Health Department, and the doctors, nurses, and staff of Sihanoukville Hospital. We thank all of the technicians and staff at Institut Pasteur du Cambodge in the Virology Unit involved in this work especially Viseth Srey Horm, Kim Lay Chea, Phalla Y, Songha Tok, Sokhoun Yan, Sarath Sin, and Sonita Kol. We would like to thank Maira Phelps at Chan Zuckerberg Biohub for coordination of primers. The work of Dr. Manning is supported by the Division of Intramural Research at the National Institute of Allergy and Infectious Diseases at the National Institutes of Health and the Bill and Melinda Gates Foundation. The work of Dr. Karlsson is supported, in part, by, by the U.S. Department of Health and Human Services, Office of the Assistant Secretary for Preparedness and Response (Grant No. 1 IDSEP190051-01-00) and through internal funding at Institut Pasteur in Cambodia. The findings and conclusions in this report are those of the authors and do not necessarily represent the official views of the US Department of Health and Human Services.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.02.968818v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Novel Coronavirus (2019-nCoV) situation reports</span>. 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.02.968818v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Nextstrain / ncov</span>. 
         <a href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.3" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr> (
         <span class="cit-pub-date">2020</span>)
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015507&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">0</span>, null (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>, 
          <span class="cit-name-given-names">V. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koopmans</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Doremalen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Riel</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">de Wit</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-article-title">A Novel Coronavirus Emerging in China — Key Questions for Impact Assessment</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">0</span>, null (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.6" data-doi="10.1038/s41564-019-0637-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Metagenomic sequencing with spiked primer enrichment for viral diagnostics and genomic surveillance</span>. 
         <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr>
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">12</span> (
         <span class="cit-pub-date">2020</span>)
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41564-019-0637-9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-019-0637-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41564-019-0637-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.02.968818v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Laboratory guidance</span>. 
         <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.8" data-doi="10.1146/annurev-pathmechdis-012418-012751">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiu</span>, 
          <span class="cit-name-given-names">C. Y.</span>
         </span>
         <span class="cit-article-title">Clinical Metagenomic Next-Generation Sequencing for Pathogen Detection</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Pathol</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">319</span>–
         <span class="cit-lpage">338</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BPathol%26rft.volume%253D14%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-pathmechdis-012418-012751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev-pathmechdis-012418-012751&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.9" data-doi="10.1126/science.1259657">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gire</span>, 
          <span class="cit-name-given-names">S. K.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">345</span>, 
         <span class="cit-fpage">1369</span>–
         <span class="cit-lpage">1372</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDUvMjAyMC4wMy4wMi45Njg4MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.10" data-doi="10.3201/eid2202.151796">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoenen</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Nanopore Sequencing as a Rapidly Deployable Ebola Outbreak Tool</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>
         <span class="cit-vol">22</span>, 
         <span class="cit-fpage">331</span>–
         <span class="cit-lpage">334</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D22%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2202.151796%26rft_id%253Dinfo%253Apmid%252F26812583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid2202.151796&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26812583&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F05%2F2020.03.02.968818.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Armstrong</span>, 
          <span class="cit-name-given-names">G. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Pathogen Genomics in Public Health</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">381</span>, 
         <span class="cit-fpage">2569</span>–
         <span class="cit-lpage">2580</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D381%26rft.spage%253D2569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.02.968818v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dozmorov</span>, 
          <span class="cit-name-given-names">M. G.</span>
         </span>
         <span class="cit-article-title">GitHub Statistics as a Measure of the Impact of Open-Source Bioinformatics Software</span>. 
         <abbr class="cit-jnl-abbrev">Front Bioeng Biotechnol</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">198</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BBioeng%2BBiotechnol%26rft.volume%253D6%26rft.spage%253D198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
