<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/embed/graphic-6.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.03.25.009084.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.25.009084v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" />
<meta name="DC.Identifier" content="10.1101/2020.03.25.009084" />
<meta name="DC.Date" content="2020-04-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="A subset of COVID-19 patients exhibit altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium and olfactory bulb that express cell entry molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find that samples from whole olfactory mucosa in species including mouse and human express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. However, neither olfactory sensory neurons nor olfactory bulb neurons express these genes, which are instead expressed in support cells, stem cells, and perivascular cells. These findings suggest that CoV-2 infection of non-neuronal cell types leads to anosmia and related disturbances in odor perception in COVID-19 patients.

One Sentence Summary Analysis of new and previously published single-cell sequencing datasets reveals that the SARS-CoV2 receptor ACE2 is expressed in olfactory support cells, stem cells and perivascular cells — but not in neurons — suggesting mechanisms through which the COVID-19 syndrome could lead to olfactory dysfunction.

### Competing Interest Statement

DL is an employee of Mars, Inc. None of the other authors have competing interests to declare." />
<meta name="DC.Contributor" content="David H. Brann" />
<meta name="DC.Contributor" content="Tatsuya Tsukahara" />
<meta name="DC.Contributor" content="Caleb Weinreb" />
<meta name="DC.Contributor" content="Marcela Lipovsek" />
<meta name="DC.Contributor" content="Koen Van den Berge" />
<meta name="DC.Contributor" content="Boying Gong" />
<meta name="DC.Contributor" content="Rebecca Chance" />
<meta name="DC.Contributor" content="Iain C. Macaulay" />
<meta name="DC.Contributor" content="Hsin-jung Chou" />
<meta name="DC.Contributor" content="Russell Fletcher" />
<meta name="DC.Contributor" content="Diya Das" />
<meta name="DC.Contributor" content="Kelly Street" />
<meta name="DC.Contributor" content="Hector Roux de Bezieux" />
<meta name="DC.Contributor" content="Yoon-Gi Choi" />
<meta name="DC.Contributor" content="Davide Risso" />
<meta name="DC.Contributor" content="Sandrine Dudoit" />
<meta name="DC.Contributor" content="Elizabeth Purdom" />
<meta name="DC.Contributor" content="Jonathan S. Mill" />
<meta name="DC.Contributor" content="Ralph Abi Hachem" />
<meta name="DC.Contributor" content="Hiroaki Matsunami" />
<meta name="DC.Contributor" content="Darren W. Logan" />
<meta name="DC.Contributor" content="Bradley J. Goldstein" />
<meta name="DC.Contributor" content="Matthew S. Grubb" />
<meta name="DC.Contributor" content="John Ngai" />
<meta name="DC.Contributor" content="Sandeep Robert Datta" />
<meta name="article:published_time" content="2020-04-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A subset of COVID-19 patients exhibit altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium and olfactory bulb that express cell entry molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find that samples from whole olfactory mucosa in species including mouse and human express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. However, neither olfactory sensory neurons nor olfactory bulb neurons express these genes, which are instead expressed in support cells, stem cells, and perivascular cells. These findings suggest that CoV-2 infection of non-neuronal cell types leads to anosmia and related disturbances in odor perception in COVID-19 patients.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.25.009084v3" />
<meta name="citation_id" content="2020.03.25.009084v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.25.009084" />
<meta name="citation_num_pages" content="47" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.25.009084" />
<meta name="citation_author" content="David H. Brann" />
<meta name="citation_author_institution" content="Harvard Medical School Department of Neurobiology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8844-8886" />
<meta name="citation_author" content="Tatsuya Tsukahara" />
<meta name="citation_author_institution" content="Harvard Medical School Department of Neurobiology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6928-8562" />
<meta name="citation_author" content="Caleb Weinreb" />
<meta name="citation_author_institution" content="Harvard Medical School Department of Neurobiology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6100-6084" />
<meta name="citation_author" content="Marcela Lipovsek" />
<meta name="citation_author_institution" content="Centre for Developmental Neurobiology, Institute of Psychiatry, Psychology and Neuroscience (IoPPN), King’s College London" />
<meta name="citation_author" content="Koen Van den Berge" />
<meta name="citation_author_institution" content="Department of Statistics, University of California" />
<meta name="citation_author_institution" content="Department of Applied Mathematics, Computer Science and Statistics, Ghent University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1833-8478" />
<meta name="citation_author" content="Boying Gong" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of California" />
<meta name="citation_author" content="Rebecca Chance" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California" />
<meta name="citation_author" content="Iain C. Macaulay" />
<meta name="citation_author_institution" content="Earlham Institute, Norwich Research Park" />
<meta name="citation_author" content="Hsin-jung Chou" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California" />
<meta name="citation_author" content="Russell Fletcher" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California" />
<meta name="citation_author_institution" content="Surrozen, Inc., South San Francisco" />
<meta name="citation_author" content="Diya Das" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California" />
<meta name="citation_author_institution" content="Genentech, Inc., South San Francisco" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9646-8983" />
<meta name="citation_author" content="Kelly Street" />
<meta name="citation_author_institution" content="Department of Data Sciences, Dana-Farber Cancer Institute" />
<meta name="citation_author_institution" content="Department of Biostatistics, Harvard T.H. Chan School of Public Health" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6379-5013" />
<meta name="citation_author" content="Hector Roux de Bezieux" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of California" />
<meta name="citation_author_institution" content="Center for Computational Biology, University of California" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1489-8339" />
<meta name="citation_author" content="Yoon-Gi Choi" />
<meta name="citation_author_institution" content="QB3 Functional Genomics Laboratory, University of California" />
<meta name="citation_author" content="Davide Risso" />
<meta name="citation_author_institution" content="Department of Statistical Sciences, University of Padova" />
<meta name="citation_author" content="Sandrine Dudoit" />
<meta name="citation_author_institution" content="Department of Statistics, University of California" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of California" />
<meta name="citation_author" content="Elizabeth Purdom" />
<meta name="citation_author_institution" content="Division of Biostatistics, School of Public Health, University of California" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9455-7990" />
<meta name="citation_author" content="Jonathan S. Mill" />
<meta name="citation_author_institution" content="University of Exeter Medical School, College of Medicine &amp; Health, University of Exeter" />
<meta name="citation_author" content="Ralph Abi Hachem" />
<meta name="citation_author_institution" content="Duke University School of Medicine Department of Head and Neck Surgery &amp; Communication Sciences" />
<meta name="citation_author" content="Hiroaki Matsunami" />
<meta name="citation_author_institution" content="Duke University School of Medicine Department of Molecular Genetics and Microbiology, Department of Neurobiology, Duke Institute for Brain Sciences" />
<meta name="citation_author" content="Darren W. Logan" />
<meta name="citation_author_institution" content="Waltham Petcare Science Institute" />
<meta name="citation_author" content="Bradley J. Goldstein" />
<meta name="citation_author_institution" content="Duke University School of Medicine Department of Head and Neck Surgery &amp; Communication Sciences" />
<meta name="citation_author" content="Matthew S. Grubb" />
<meta name="citation_author_institution" content="Centre for Developmental Neurobiology, Institute of Psychiatry, Psychology and Neuroscience (IoPPN), King’s College London" />
<meta name="citation_author" content="John Ngai" />
<meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California" />
<meta name="citation_author_institution" content="QB3 Functional Genomics Laboratory, University of California" />
<meta name="citation_author_institution" content="Helen Wills Neuroscience Institute, University of California" />
<meta name="citation_author" content="Sandeep Robert Datta" />
<meta name="citation_author_institution" content="Harvard Medical School Department of Neurobiology" />
<meta name="citation_author_email" content="srdatta@hms.harvard.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8068-3862" />
<meta name="citation_reference" content="W.-j. Guan, Z.-y. Ni, Y. Hu, W.-h. Liang, C.-q. Ou, J.-x. He, L. Liu, H. Shan, C.-l. Lei, D. S. C. Hui, B. Du, L.-j. Li, G. Zeng, K.-Y. Yuen, R.-c. Chen, C.-l. Tang, T. Wang, P.-y. Chen, J. Xiang, S.-y. Li, J.-l. Wang, Z.-j. Liang, Y.-x. Peng, L. Wei, Y. Liu, Y.-h. Hu, P. Peng, J.-m. Wang, J.-y. Liu, Z. Chen, G. Li, Z.-j. Zheng, S.-q. Qiu, J. Luo, C.-j. Ye, S.-y. Zhu, N.-s. Zhong, C. China Medical Treatment Expert Group for, Clinical Characteristics of Coronavirus Disease 2019 in China. N. Engl. J. Med. EPub ahead of print, (2020)." />
<meta name="citation_reference" content="H. A. Rothan, S. N. Byrareddy, The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak. J. Autoimmun. EPub ahead of print, (2020)." />
<meta name="citation_reference" content="Z. Wu, J. M. McGoogan, Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72 314 Cases From the Chinese Center for Disease Control and Prevention. JAMA EPub ahead of print, (2020)." />
<meta name="citation_reference" content="P. Zhou, X.-L. Yang, X.-G. Wang, B. Hu, L. Zhang, W. Zhang, H.-R. Si, Y. Zhu, B. Li, C.-L. Huang, H.-D. Chen, J. Chen, Y. Luo, H. Guo, R.-D. Jiang, M.-Q. Liu, Y. Chen, X.-R. Shen, X. Wang, X.-S. Zheng, K. Zhao, Q.-J. Chen, F. Deng, L.-L. Liu, B. Yan, F.-X. Zhan, Y.-Y. Wang, G.-F. Xiao, Z.-L. Shi, A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273 (2020)." />
<meta name="citation_reference" content="M. Ceccarelli, M. Berretta, E. Venanzi Rullo, G. Nunnari, B. Cacopardo, Differences and similarities between Severe Acute Respiratory Syndrome (SARS)-CoronaVirus (CoV) and SARS-CoV-2. Would a rose by another name smell as sweet? Eur Rev Med Pharmacol Sci 24, 2781–2783 (2020)." />
<meta name="citation_reference" content="A. Zumla, J. F. W. Chan, E. I. Azhar, D. S. C. Hui, K.-Y. Yuen, Coronaviruses - drug discovery and therapeutic options. Nat Rev Drug Discov 15, 327–347 (2016)." />
<meta name="citation_reference" content="V. M. Corman, D. Muth, D. Niemeyer, C. Drosten, Hosts and Sources of Endemic Human Coronaviruses. Adv. Virus Res. 100, 163–188 (2018)." />
<meta name="citation_reference" content="S. H. R. Bagheri, A. M. Asghari, M. Farhadi, A. R. S. medRxiv, Coincidence of COVID-19 epidemic and olfactory dysfunction outbreak. medRxiv." />
<meta name="citation_reference" content="A. Giacomelli, L. Pezzati, F. Conti, D. Bernacchia, M. Siano, L. Oreni, S. Rusconi, C. Gervasoni, A. L. Ridolfo, G. Rizzardini, S. Antinori, M. Galli, Self-reported olfactory and taste disorders in SARS-CoV-2 patients: a cross-sectional study. Clin. Infect. Dis., (2020)." />
<meta name="citation_reference" content="L. Mao, M. Wang, S. Chen, Q. He, J. Chang, C. H. E. a. o. print, Neurological manifestations of hospitalized patients with COVID-19 in Wuhan, China: a retrospective case series study. medRxiv. medRxiv." />
<meta name="citation_reference" content="R. Wölfel, V. M. Corman, W. Guggemos, M. Seilmaier, S. Zange, M. A. Müller, D. Niemeyer, T. C. Jones, P. Vollmar, C. Rothe, M. Hoelscher, T. Bleicker, S. Brünink, J. Schneider, R. Ehmann, K. Zwirglmaier, C. Drosten, C. Wendtner, Virological assessment of hospitalized patients with COVID-2019. Nature, 1–10 (2020)." />
<meta name="citation_reference" content="J. R. Lechien, C. M. Chiesa-Estomba, D. R. De Siati, M. Horoi, S. D. Le Bon, A. Rodriguez, D. Dequanter, S. Blecic, F. El Afia, L. Distinguin, Y. Chekkoury-Idrissi, S. Hans, I. L. Delgado, C. Calvo-Henriquez, P. Lavigne, C. Falanga, M. R. Barillari, G. Cammaroto, M. Khalife, P. Leich, C. Souchay, C. Rossi, F. Journe, J. Hsieh, M. Edjlali, R. Carlier, L. Ris, A. Lovato, C. De Filippis, F. Coppee, N. Fakhry, T. Ayad, S. Saussez, Olfactory and gustatory dysfunctions as a clinical presentation of mild-to-moderate forms of the coronavirus disease (COVID-19): a multicenter European study. European Archives of Oto-Rhino-Laryngology, (2020)." />
<meta name="citation_reference" content="C. Menni, A. Valdes, M. B. Freydin, S. Ganesh, J. El-Sayed Moustafa, A. Visconti, P. Hysi, R. C. E. Bowyer, M. Mangino, M. Falchi, J. Wolf, C. Steves, T. Spector, Loss of smell and taste in combination with other symptoms is a strong predictor of COVID-19 infection. medRxiv, 1–23 (2020)." />
<meta name="citation_reference" content="C.-S. Hwang, Olfactory neuropathy in severe acute respiratory syndrome: report of A case. Acta Neurol Taiwan 15, 26–28 (2006)." />
<meta name="citation_reference" content="P. Dalton, Olfaction and anosmia in rhinosinusitis. Curr Allergy Asthma Rep 4, 230–236 (2004)." />
<meta name="citation_reference" content="R. L. Doty, Systemic diseases and disorders. Handb Clin Neurol 164, 361–387 (2019)." />
<meta name="citation_reference" content="M. Suzuki, K. Saito, W.-P. Min, C. Vladau, K. Toida, H. Itoh, S. Murakami, Identification of viruses in patients with postviral olfactory dysfunction. Laryngoscope 117, 272–277 (2007)." />
<meta name="citation_reference" content="A. Åkerlund, M. Bende, C. Murphy, Olfactory threshold and nasal mucosal changes in experimentally induced common cold. Acta Otolaryngol. 115, 88–92 (1995)." />
<meta name="citation_reference" content="H. J. Hedrich, The Laboratory Mouse. (Academic Press, 2012), pp. 845." />
<meta name="citation_reference" content="G. K. Reznik, Comparative anatomy, physiology, and function of the upper respiratory tract. Environ Health Perspect 85, 171–176 (1990)." />
<meta name="citation_reference" content="Y. Suzuki, J. Schafer, A. I. Farbman, Phagocytic cells in the rat olfactory epithelium after bulbectomy. Exp. Neurol. 136, 225–233 (1995)." />
<meta name="citation_reference" content="Y. Suzuki, M. Takeda, A. I. Farbman, Supporting cells as phagocytes in the olfactory epithelium after bulbectomy. J. Comp. Neurol. 376, 509–517 (1996)." />
<meta name="citation_reference" content="F. Vogalis, C. C. Hegg, M. T. Lucero, Ionic conductances in sustentacular cells of the mouse olfactory epithelium. J. Physiol. (Lond). 562, 785–799 (2005)." />
<meta name="citation_reference" content="S. Pfister, T. Weber, W. Härtig, C. Schwerdel, R. Elsaesser, I. Knuesel, J.-M. Fritschy, Novel role of cystic fibrosis transmembrane conductance regulator in maintaining adult mouse olfactory neuronal homeostasis. J. Comp. Neurol. 523, 406–430 (2014)." />
<meta name="citation_reference" content="R. Choi, B. J. Goldstein, Olfactory epithelium: Cells, clinical disorders, and insights from an adult stem cell niche. Laryngoscope Investig Otolaryngol 3, 35–42 (2018)." />
<meta name="citation_reference" content="J. E. Schwob, W. Jang, E. H. Holbrook, B. Lin, D. B. Herrick, J. N. Peterson, J. Hewitt Coleman, Stem and progenitor cells of the mammalian olfactory epithelium: Taking poietic license. J. Comp. Neurol. 525, 1034–1054 (2017)." />
<meta name="citation_reference" content="R. B. Fletcher, D. Das, L. Gadye, K. N. Street, A. Baudhuin, A. Wagner, M. B. Cole, Q. Flores, Y. G. Choi, N. Yosef, E. Purdom, S. Dudoit, D. Risso, J. Ngai, Deconstructing Olfactory Stem Cell Trajectories at Single-Cell Resolution. Cell Stem Cell 20, 817–830.e818 (2017)." />
<meta name="citation_reference" content="M. Hoffmann, H. Kleine-Weber, S. Schroeder, N. Krüger, T. Herrler, S. Erichsen, S. Schiergens, G. Herrler, N.-H. Wu, A. Nitsche, M. A. Müller, C. Drosten, S. Pöhlmann, SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell, (2020)." />
<meta name="citation_reference" content="W. Li, M. J. Moore, N. Vasilieva, J. Sui, S. K. Wong, M. A. Berne, M. Somasundaran, J. L. Sullivan, K. Luzuriaga, T. C. Greenough, H. Choe, M. Farzan, Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)." />
<meta name="citation_reference" content="J. Netland, D. K. Meyerholz, S. Moore, M. Cassell, S. Perlman, Severe acute respiratory syndrome coronavirus infection causes neuronal death in the absence of encephalitis in mice transgenic for human ACE2. J. Virol. 82, 7264–7275 (2008)." />
<meta name="citation_reference" content="W. Li, T. C. Greenough, M. J. Moore, N. Vasilieva, M. Somasundaran, J. L. Sullivan, M. Farzan, H. Choe, Efficient replication of severe acute respiratory syndrome coronavirus in mouse cells is limited by murine angiotensin-converting enzyme 2. J. Virol. 78, 11429–11433 (2004)." />
<meta name="citation_reference" content="K. Kuba, Y. Imai, S. Rao, H. Gao, F. Guo, B. Guan, Y. Huan, P. Yang, Y. Zhang, W. Deng, L. Bao, B. Zhang, G. Liu, Z. Wang, M. Chappell, Y. Liu, D. Zheng, A. Leibbrandt, T. Wada, A. S. Slutsky, D. Liu, C. Qin, C. Jiang, J. M. Penninger, A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury. Nat. Med. 11, 875–879 (2005)." />
<meta name="citation_reference" content="S. Bertram, I. Glowacka, M. A. Müller, H. Lavender, K. Gnirss, I. Nehlmeier, D. Niemeyer, Y. He, G. Simmons, C. Drosten, E. J. Soilleux, O. Jahn, I. Steffen, S. Pöhlmann, Cleavage and activation of the severe acute respiratory syndrome coronavirus spike protein by human airway trypsin-like protease. J. Virol. 85, 13363–13372 (2011)." />
<meta name="citation_reference" content="S. Bertram, A. Heurich, H. Lavender, S. Gierer, S. Danisch, P. Perin, J. M. Lucas, P. S. Nelson, S. Pöhlmann, E. J. Soilleux, Influenza and SARS-coronavirus activating proteases TMPRSS2 and HAT are expressed at multiple sites in human respiratory and gastrointestinal tracts. PLoS ONE 7, e35876 (2012)." />
<meta name="citation_reference" content="H. Chu, C.-M. Chan, X. Zhang, Y. Wang, S. Yuan, J. Zhou, R. K.-H. Au-Yeung, K.-H. Sze, D. Yang, H. Shuai, Y. Hou, C. Li, X. Zhao, V. K.-M. Poon, S. P. Leung, M.-L. Yeung, J. Yan, G. Lu, D.-Y. Jin, G. F. Gao, J. F.-W. Chan, K.-Y. Yuen, Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells. Journal of Biological Chemistry 293, 11709–11726 (2018)." />
<meta name="citation_reference" content="W. Sungnak, N. Huang, C. Bécavin, M. Berg, H. C. A. L. B. Network, SARS-CoV-2 Entry Genes Are Most Highly Expressed in Nasal Goblet and Ciliated Cells within Human Airways. arXiv preprint arXiv:2003.06122, (2020)." />
<meta name="citation_reference" content="M. Deprez, L.-E. Zaragosi, M. Truchi, S. R. Garcia, M.-J. Arguel, K. Lebrigand, A. Paquet, D. Pe’er, C.-H. Marquette, S. Leroy, P. Barbry, A single-cell atlas of the human healthy airways. bioRxiv 24, 3000–3049 (2019)." />
<meta name="citation_reference" content="F. A. Vieira Braga, G. Kar, M. Berg, O. A. Carpaij, K. Polanski, L. M. Simon, S. Brouwer, T. Gomes, L. Hesse, J. Jiang, E. S. Fasouli, M. Efremova, R. Vento-Tormo, C. Talavera-López, M. R. Jonker, K. Affleck, S. Palit, P. M. Strzelecka, H. V. Firth, K. T. Mahbubani, A. Cvejic, K. B. Meyer, K. Saeb-Parsy, M. Luinge, C.-A. Brandsma, W. Timens, I. Angelidis, M. Strunz, G. H. Koppelman, A. J. van Oosterhout, H. B. Schiller, F. J. Theis, M. van den Berge, M. C. Nawijn, S. A. Teichmann, A cellular census of human lungs identifies novel cell states in health and in asthma. Nat. Med. 25, 1153–1163 (2019)." />
<meta name="citation_reference" content="L. R. Saraiva, X. Ibarra-Soria, M. Khan, M. Omura, A. Scialdone, P. Mombaerts, J. C. Marioni, D. W. Logan, Hierarchical deconstruction of mouse olfactory sensory neurons: from whole mucosa to single-cell RNA-seq. Sci Rep 5, 18178 (2015)." />
<meta name="citation_reference" content="N. Kanageswaran, M. Demond, M. Nagel, B. S. P. Schreiner, S. Baumgart, P. Scholz, J. Altmüller, C. Becker, J. F. Doerner, H. Conrad, S. Oberland, C. H. Wetzel, E. M. Neuhaus, H. Hatt, G. Gisselmann, Deep Sequencing of the Murine Olfactory Receptor Neuron Transcriptome. PLOS ONE 10, e0113170 (2015)." />
<meta name="citation_reference" content="Bradley M. Colquitt, E. Markenscoff-Papadimitriou, R. Duffié, S. Lomvardas, Dnmt3a Regulates Global Gene Expression in Olfactory Sensory Neurons and Enables Odorant-Induced Transcription. Neuron 83, 823–838 (2014)." />
<meta name="citation_reference" content="L. Gadye, D. Das, M. A. Sanchez, K. Street, A. Baudhuin, A. Wagner, M. B. Cole, Y. G. Choi, N. Yosef, E. Purdom, S. Dudoit, D. Risso, J. Ngai, R. B. Fletcher, Injury Activates Transient Olfactory Stem Cell States with Diverse Lineage Capacities. Cell Stem Cell 21, 775–790.e779 (2017)." />
<meta name="citation_reference" content="L. R. Saraiva, F. Riveros-McKay, M. Mezzavilla, E. H. Abou-Moussa, C. J. Arayata, M. Makhlouf, C. Trimmer, X. Ibarra-Soria, M. Khan, L. van Gerven, M. Jorissen, M. Gibbs, C. O’Flynn, S. McGrane, P. Mombaerts, J. C. Marioni, J. D. Mainland, D. W. Logan, A transcriptomic atlas of mammalian olfactory mucosae reveals an evolutionary influence on food odor detection in humans. Science Advances 5, eaax0396 (2019)." />
<meta name="citation_reference" content="M. A. Durante, S. Kurtenbach, Z. B. Sargi, J. W. Harbour, R. Choi, S. Kurtenbach, G. M. Goss, H. Matsunami, B. J. Goldstein, Single-cell analysis of olfactory neurogenesis and differentiation in adult humans. Nature Neuroscience 23, 323–326 (2020)." />
<meta name="citation_reference" content="A. Zeisel, H. Hochgerner, P. Lönnerberg, A. Johnsson, F. Memic, J. van der Zwan, M. Häring, E. Braun, L. E. Borm, G. La Manno, S. Codeluppi, A. Furlan, K. Lee, N. Skene, K. D. Harris, J. Hjerling-Leffler, E. Arenas, P. Ernfors, U. Marklund, S. Linnarsson, Molecular Architecture of the Mouse Nervous System. Cell 174, 999–1014.e1022 (2018)." />
<meta name="citation_reference" content="L. S. Brown, C. G. Foster, J.-M. Courtney, N. E. King, D. W. Howells, B. A. Sutherland, Pericytes and Neurovascular Function in the Healthy and Diseased Brain. Front Cell Neurosci 13, 282 (2019)." />
<meta name="citation_reference" content="I. Hamming, W. Timens, M. L. C. Bulthuis, A. T. Lely, G. J. Navis, H. van Goor, Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. J. Pathol. 203, 631–637 (2004)." />
<meta name="citation_reference" content="L. W. Plasschaert, R. Žilionis, R. Choo-Wing, V. Savova, J. Knehr, G. Roma, A. M. Klein, A. B. Jaffe, A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte. Nature 560, 377–381 (2018)." />
<meta name="citation_reference" content="C. G. Bihun, D. H. Percy, Morphologic changes in the nasal cavity associated with sialodacryoadenitis virus infection in the Wistar rat. Vet. Pathol. 32, 1–10 (1995)." />
<meta name="citation_reference" content="A. I. Farbman, Olfactory neurogenesis: genetic or environmental controls? Trends Neurosci 13, 362–365 (1990)." />
<meta name="citation_reference" content="K. Sultan-Styne, R. Toledo, C. Walker, A. Kallkopf, C. E. Ribak, K. M. Guthrie, Long-term survival of olfactory sensory neurons after target depletion. J. Comp. Neurol. 515, 696–710 (2009)." />
<meta name="citation_reference" content="K. Bohmwald, N. M. S. Gálvez, M. Ríos, A. M. Kalergis, Neurologic Alterations Due to Respiratory Virus Infections. Front Cell Neurosci 12, 386 (2018)." />
<meta name="citation_reference" content="S. L. Youngentob, J. E. Schwob, S. Saha, G. Manglapus, B. Jubelt, Functional consequences following infection of the olfactory system by intranasal infusion of the olfactory bulb line variant (OBLV) of mouse hepatitis strain JHM. Chemical Senses 26, 953–963 (2001)." />
<meta name="citation_reference" content="E. M. Barnett, S. Perlman, The olfactory nerve and not the trigeminal nerve is the major site of CNS entry for mouse hepatitis virus, strain JHM. Virology 194, 185–191 (1993)." />
<meta name="citation_reference" content="J. E. Schwob, S. Saha, S. L. Youngentob, B. Jubelt, Intranasal inoculation with the olfactory bulb line variant of mouse hepatitis virus causes extensive destruction of the olfactory bulb and accelerated turnover of neurons in the olfactory epithelium of mice. Chemical Senses 26, 937–952 (2001)." />
<meta name="citation_reference" content="E. Hemmila, C. Turbide, M. Olson, S. Jothy, K. V. Holmes, N. Beauchemin, Ceacam1a-/- mice are completely resistant to infection by murine coronavirus mouse hepatitis virus A59. J. Virol. 78, 10156–10165 (2004)." />
<meta name="citation_reference" content="Z. Chen, L. Mi, J. Xu, J. Yu, X. Wang, J. Jiang, J. Xing, P. Shang, A. Qian, Y. Li, P. X. Shaw, J. Wang, S. Duan, J. Ding, C. Fan, Y. Zhang, Y. Yang, X. Yu, Q. Feng, B. Li, X. Yao, Z. Zhang, L. Li, X. Xue, P. Zhu, Function of HAb18G/CD147 in invasion of host cells by severe acute respiratory syndrome coronavirus. J. Infect. Dis. 191, 755–760 (2005)." />
<meta name="citation_reference" content="K. Wang, W. Chen, Y.-S. Zhou, J.-Q. Lian, Z. Zhang, P. Du, L. Gong, Y. Zhang, H.-Y. Cui, J.-J. Geng, B. Wang, X.-X. Sun, C.-F. Wang, X. Yang, P. Lin, Y.-Q. Deng, D. Wei, X.-M. Yang, Y.-M. Zhu, K. Zhang, Z.-H. Zheng, J.-L. Miao, T. Guo, Y. Shi, J. Zhang, L. Fu, Q.-Y. Wang, H. Bian, P. Zhu, Z.-N. Chen, SARS-CoV-2 invades host cells via a novel route: CD147-spike protein. 395, 497–410 (2020)." />
<meta name="citation_reference" content="A. K. Indra, X. Warot, J. Brocard, J. M. Bornert, J. H. Xiao, P. Chambon, D. Metzger, Temporally-controlled site-specific mutagenesis in the basal layer of the epidermis: comparison of the recombinase activity of the tamoxifen-inducible Cre-ER(T) and Cre-ER(T2) recombinases. Nucleic Acids Res 27, 4324–4327 (1999)." />
<meta name="citation_reference" content="S. Srinivas, T. Watanabe, C.-S. Lin, C. M. William, Y. Tanabe, T. M. Jessell, F. Costantini, Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus. BMC Developmental Biology 1, 4 (2001)." />
<meta name="citation_reference" content="M. B. Cole, D. Risso, A. Wagner, D. DeTomaso, J. Ngai, E. Purdom, S. Dudoit, N. Yosef, Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq. Cell Systems 8, 315–328.e318 (2019)." />
<meta name="citation_reference" content="C. S. McGinnis, L. M. Murrow, Z. J. Gartner, DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors. Cell Systems 8, 329–337.e324 (2019)." />
<meta name="citation_reference" content="S. L. Wolock, R. Lopez, A. M. Klein, Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data. Cell Systems 8, 281–291.e289 (2019)." />
<meta name="citation_reference" content="D. Risso, J. Ngai, T. P. Speed, S. Dudoit, Normalization of RNA-seq data using factor analysis of control genes or samples. Nature Biotechnology 32, 896–902 (2014)." />
<meta name="citation_reference" content="D. Risso, L. Purvis, R. B. Fletcher, D. Das, J. Ngai, S. Dudoit, E. Purdom, clusterExperiment and RSEC: A Bioconductor package and framework for clustering of single-cell and other large gene expression datasets. PLOS Computational Biology 14, e1006378 (2018)." />
<meta name="citation_reference" content="C. Weinreb, S. Wolock, A. M. Klein, SPRING: a kinetic interface for visualizing high dimensional single-cell expression data. Bioinformatics 34, 1246–1248 (2018)." />
<meta name="citation_reference" content="F. A. Wolf, P. Angerer, F. J. Theis, SCANPY: large-scale single-cell gene expression data analysis. Genome Biology 19, 15 (2018)." />
<meta name="citation_reference" content="C. Weinreb, A. Rodriguez-Fraticelli, F. D. Camargo, A. M. Klein, Lineage tracing on transcriptional landscapes links state to fate during differentiation. Science 367, eaaw3381 (2020)." />
<meta name="citation_reference" content="P. G. Kehoe, S. Wong, N. Al Mulhim, L. E. Palmer, J. S. Miners, Angiotensin-converting enzyme 2 is reduced in Alzheimer’s disease in association with increasing amyloid-β and tau pathology. Alzheimers Res Ther 8, 50 (2016)." />
<meta name="citation_reference" content="E. Z. Macosko, A. Basu, R. Satija, J. Nemesh, K. Shekhar, M. Goldman, I. Tirosh, A. R. Bialas, N. Kamitaki, E. M. Martersteck, Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets. Cell 161, 1202–1214 (2015)." />
<meta name="citation_reference" content="T. Stuart, A. Butler, P. Hoffman, C. Hafemeister, E. Papalexi, W. M. Mauck III, Y. Hao, M. Stoeckius, P. Smibert, R. Satija, Comprehensive integration of single-cell data. Cell 177, 1888–1902. e1821 (2019)." />
<meta name="citation_reference" content="L. Zappia, A. Oshlack, Clustering trees: a visualization for evaluating clusterings at multiple resolutions. GigaScience 7, (2018)." />
<meta name="citation_reference" content="E. Galliano, E. Franzoni, M. Breton, A. N. Chand, D. J. Byrne, V. N. Murthy, M. S. Grubb, Embryonic and postnatal neurogenesis produce functionally distinct subclasses of dopaminergic neuron. eLife 7, e32373 (2018)." />
<meta name="citation_reference" content="C. M. Hempel, K. Sugino, S. B. Nelson, A manual method for the purification of fluorescently labeled neurons from the mammalian brain. Nat Protoc 2, 2924–2929 (2007)." />
<meta name="citation_reference" content="I. C. Macaulay, M. J. Teng, W. Haerty, P. Kumar, C. P. Ponting, T. Voet, Separation and parallel sequencing of the genomes and transcriptomes of single cells using G&amp;T-seq. Nature Protocols 11, 2081–2103 (2016)." />
<meta name="citation_reference" content="N. L. Bray, H. Pimentel, P. Melsted, L. Pachter, Near-optimal probabilistic RNA-seq quantification. Nature Biotechnology 34, 525–527 (2016)." />
<meta name="citation_reference" content="D. J. McCarthy, K. R. Campbell, A. T. Lun, Q. F. Wills, Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R. Bioinformatics 33, 1179–1186 (2017)." />
<meta name="citation_reference" content="J. Ninkovic, L. Pinto, S. Petricca, A. Lepier, J. Sun, M. A. Rieger, T. Schroeder, A. Cvekl, J. Favor, M. Götz, The transcription factor Pax6 regulates survival of dopaminergic olfactory bulb neurons via crystallin αA. Neuron 68, 682–694 (2010)." />
<meta name="citation_reference" content="K. Shekhar, S. W. Lapan, I. E. Whitney, N. M. Tran, E. Z. Macosko, M. Kowalczyk, X. Adiconis, J. Z. Levin, J. Nemesh, M. Goldman, S. A. McCarroll, C. L. Cepko, A. Regev, J. R. Sanes, Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics. Cell 166, 1308–1323.e1330 (2016)." />
<meta name="citation_reference" content="J. C. Burns, M. C. Kelly, M. Hoa, R. J. Morell, M. W. Kelley, Single-cell RNA-Seq resolves cellular complexity in sensory organs from the neonatal inner ear. Nature Communications 6, 1–16 (2015)." />
<meta name="citation_reference" content="S. McInturff, J. C. Burns, M. W. Kelley, Characterization of spatial and temporal development of Type I and Type II hair cells in the mouse utricle using new cell-type-specific markers. Biology open 7, (2018)." />
<meta name="citation_reference" content="B. R. Shrestha, C. Chia, L. Wu, S. G. Kujawa, M. C. Liberman, L. V. Goodrich, B. R. Shrestha, C. Chia, L. Wu, S. G. Kujawa, M. C. Liberman, L. V. Goodrich, Sensory neuron diversity in the inner ear is shaped by activity sensory neuron diversity. Cell 174, 1229–1246 (2018)." />
<meta name="citation_reference" content="G. La Manno, D. Gyllborg, S. Codeluppi, R. A. Barker, E. Arenas, S. Linnarsson, Molecular diversity of midbrain development in resource molecular diversity of midbrain development in mouse, human and stem cells. Cell 167, 566–580 (2016)." />
<meta name="citation_reference" content="M. S. Cembrowski, M. G. Phillips, S. F. DiLisio, B. C. Shields, J. Winnubst, J. Chandrashekar, E. Bas, N. Spruston, Dissociable Structural and Functional Hippocampal Outputs via Distinct Subiculum Cell Classes. Cell 173, 1280–1292.e1218 (2018)." />
<meta name="citation_reference" content="M. Chevée, J. D. J. Robertson, G. H. Cannon, S. P. Brown, L. A. Goff, Variation in Activity State, Axonal Projection, and Position Define the Transcriptional Identity of Individual Neocortical Projection Neurons. Cell Reports 22, 441–455 (2018)." />
<meta name="citation_reference" content="R. A. Romanov, A. Zeisel, J. Bakker, F. Girach, A. Hellysaz, R. Tomer, A. Alpár, J. Mulder, F. Clotman, E. Keimpema, B. Hsueh, A. K. Crow, H. Martens, C. Schwindling, D. Calvigioni, J. S. Bains, Z. Máté, G. Szabó, Y. Yanagawa, M. D. Zhang, A. Rendeiro, M. Farlik, M. Uhlén, P. Wulff, C. Bock, C. Broberger, K. Deisseroth, T. Hökfelt, S. Linnarsson, T. L. Horvath, T. Harkany, Molecular interrogation of hypothalamic organization reveals distinct dopamine neuronal subtypes. Nature Neuroscience 20, 176–188 (2017)." />
<meta name="citation_reference" content="A. Furlan, G. La Manno, M. Lubke, M. Haring, H. Abdo, H. Hochgerner, J. Kupari, D. Usoskin, M. S. Airaksinen, G. Oliver, S. Linnarsson, P. Ernfors, Visceral motor neuron diversity delineates a cellular basis for nipple- and pilo-erection muscle control. Nature Neuroscience 19, 1331–1340 (2016)." />
<meta name="citation_reference" content="D. Usoskin, A. Furlan, S. Islam, H. Abdo, P. Lönnerberg, D. Lou, J. Hjerling-Leffler, J. Haeggström, O. Kharchenko, P. V. Kharchenko, S. Linnarsson, P. Ernfors, Unbiased classification of sensory neuron types by large-scale single-cell RNA sequencing. Nature Neuroscience 18, 145–153 (2015)." />
<meta name="citation_reference" content="M. Haring, A. Zeisel, H. Hochgerner, P. Rinwa, J. E. T. Jakobsson, P. Lonnerberg, G. La Manno, N. Sharma, L. Borgius, O. Kiehn, M. C. Lagerstrom, S. Linnarsson, P. Ernfors, Neuronal atlas of the dorsal horn defines its architecture and links sensory input to transcriptional cell types. Nat Neurosci 21, 869–880 (2018)." />
<meta name="citation_reference" content="A. T. Lun, K. Bach, J. C. Marioni, Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome biology 17, 75 (2016)." />
<meta name="twitter:title" content="Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="A subset of COVID-19 patients exhibit altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium and olfactory bulb that express cell entry molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find that samples from whole olfactory mucosa in species including mouse and human express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. However, neither olfactory sensory neurons nor olfactory bulb neurons express these genes, which are instead expressed in support cells, stem cells, and perivascular cells. These findings suggest that CoV-2 infection of non-neuronal cell types leads to anosmia and related disturbances in odor perception in COVID-19 patients.

One Sentence Summary Analysis of new and previously published single-cell sequencing datasets reveals that the SARS-CoV2 receptor ACE2 is expressed in olfactory support cells, stem cells and perivascular cells — but not in neurons — suggesting mechanisms through which the COVID-19 syndrome could lead to olfactory dysfunction.

### Competing Interest Statement

DL is an employee of Mars, Inc. None of the other authors have competing interests to declare." />
<meta name="og-title" property="og:title" content="Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="A subset of COVID-19 patients exhibit altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium and olfactory bulb that express cell entry molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find that samples from whole olfactory mucosa in species including mouse and human express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. However, neither olfactory sensory neurons nor olfactory bulb neurons express these genes, which are instead expressed in support cells, stem cells, and perivascular cells. These findings suggest that CoV-2 infection of non-neuronal cell types leads to anosmia and related disturbances in odor perception in COVID-19 patients.

One Sentence Summary Analysis of new and previously published single-cell sequencing datasets reveals that the SARS-CoV2 receptor ACE2 is expressed in olfactory support cells, stem cells and perivascular cells — but not in neurons — suggesting mechanisms through which the COVID-19 syndrome could lead to olfactory dysfunction.

### Competing Interest Statement

DL is an employee of Mars, Inc. None of the other authors have competing interests to declare." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.25.009084v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.25.009084v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.25.009084v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1237315" />
    <title>Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LBPClmBgofmcAmhGq6Ny6AzfcGPUa8c64keXtFOtwx0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1237315","apath":"\/biorxiv\/early\/2020\/04\/09\/2020.03.25.009084.atom","pisa":"biorxiv;2020.03.25.009084v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.25.009084v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.25.009084v3","title":"Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia","identifier":"node\/1237315"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZMZbCEQ3Xpcur0gX+bWOwKQfeXv0ru33K6aj4PE7RT81vQdfVKrLzsQrdTBeTQbTE7rfQh2lHJmvvclQXlp4Ut\\\\\\\/El5mrFeMa0qv7tjLLA9u9U6jG1QkKoZUgQYspFVnnWrrnbB7Cgo12Ptr5q29CbeO0mN2uMonmZIE790xfEJiafgNxV4EgdMc5YZM9Jh+QGRiOVqQGAwUDGfV\\\\\\\/IxUOgxIDhLQ6ZGbw3CW2ZXC5T8ZtQEeP\\\\\\\/+ruq5MaYfCkqg+nguJlRyo61h2U5tX7\\\\\\\/79z5qpMWBRUFg2dIkBBlfVPfh3K11cEjAEsew8eH6OJUQs\\\\\\\/HET8ehqdIF6pEIh+XhdXK0FgzwHP1sB686hO7bAINyduViU+LDYS\\\\\\\/pNf5GKwJgmjckz6jDR1spRRFgZPfdtMmFIlzXdqwMHHyNhT88fDdpL4Mratd+dBksgaFbiLloJSLiRPQzoQbUtY4b+aQKrjCvTYFMdUiULauYijq6NOdkxigXQYXXCE9+me5mflN6AXpNFxpa+ZdMa\\\\\\\/mksxC926kvDTxLWlo\\\\\\\/NyGxc3crSrlQm5\\\\\\\/VpC2HIulDM\\\\\\\/mcoOn3blb0W2EeJwgLLIcN4mxqjAtBCkWmr5CSyM8keoIDG2to8FWJDyuqcQyw03su4z++tU9kBjYsUAOaMMJfanpawXeD+i5pMHZ0B1N82yttHSg0q\\\\\\\/V+0wDbc82bFUwKlgDy+pgtkYD6Y5BvY3VteerHpC23e+TDeeF73fsNKBvxkxBVy+fFNaewaxvIi1iyF0SCkl0p3xsrJLGIkCwnP\\\\\\\/tU8\\\\\\\/bC8uufI7qyCYloNgGbtYmMFiFPoTWYAXpNg+fMMnZESalXyQzCpsljw0euqOu5DB3HhY7XIxnsHK7uX5AKKNgSNZbcsbXUIq7ebJSNLyFONbQdEHPEgshYGgf8lkMmIWimjEB8h9VDsbp6v37S\\\\\\\/xqbHVTi3WMinj6Pz7Y2PzbJb6rgTL5tsqkEAfocIhJxS7eNaE6dBqWyjbHwAERQKq4R4xaceG\\\\\\\/1XSh2iXe68hE5Nt0d+FNg6iakmrSgL1TCZPzKvqsdoSFCWaAoHW5fiqEBBOMrSYkbRHEw51Wp\\\\\\\/sxYeyYyieDm7RDyV3Rj07zo3COB84azoYlUx8vrSQi8Rs3eV5yMXU6IXocmSY5MdCXLr\\\\\\\/26qxnI8Kx265qYbuJPW0ymo+uTDcLI\\\\\\\/9d2ZHdzFqMEAVWNasNRu7DhGgm7s5zMRaRBOMoY4jgy9X1lRSdRmwXGy7KhtmbaMgddhUpkZseMmIkqIHhlUDfsstvZX6zaRocMivnVPvRnpaNNElCBDTqctk0aA7GkY3eY0XFIPfaC4vljOcS19qa4jwPvUYleYT\\\u0022,\\\u0022iv\\\u0022:\\\u0022diS+xPR9ZL+1JZ1RvKWFlA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcfcfb45a94cd317f27d312bdf2e5e65a9cba1266c8407795b18dccc4e5b0db0\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.25.009084v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1237315 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.25.009084v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DRFtQNzYyGbZj9ouGr2kilRwhhS1wpog6MN3fkWhcdQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237315" id="node1237315" data-pisa="biorxiv;2020.03.25.009084v3" data-pisa-master="biorxiv;2020.03.25.009084" data-apath="/biorxiv/early/2020/04/09/2020.03.25.009084.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8844-8886" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">David H.</span> <span class="nlm-surname">Brann</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6928-8562" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Tatsuya</span> <span class="nlm-surname">Tsukahara</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6100-6084" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Weinreb</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Marcela</span> <span class="nlm-surname">Lipovsek</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1833-8478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Koen</span> <span class="nlm-surname">Van den Berge</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Boying</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Chance</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Iain C.</span> <span class="nlm-surname">Macaulay</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Hsin-jung</span> <span class="nlm-surname">Chou</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Russell</span> <span class="nlm-surname">Fletcher</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-9646-8983" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Diya</span> <span class="nlm-surname">Das</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-6379-5013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kelly</span> <span class="nlm-surname">Street</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0002-1489-8339" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Hector Roux</span> <span class="nlm-surname">de Bezieux</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yoon-Gi</span> <span class="nlm-surname">Choi</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Davide</span> <span class="nlm-surname">Risso</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Sandrine</span> <span class="nlm-surname">Dudoit</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0001-9455-7990" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Elizabeth</span> <span class="nlm-surname">Purdom</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Jonathan S.</span> <span class="nlm-surname">Mill</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Ralph Abi</span> <span class="nlm-surname">Hachem</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Hiroaki</span> <span class="nlm-surname">Matsunami</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Darren W.</span> <span class="nlm-surname">Logan</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">Goldstein</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Grubb</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">John</span> <span class="nlm-surname">Ngai</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="24"><a href="http://orcid.org/0000-0002-8068-3862" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sandeep Robert</span> <span class="nlm-surname">Datta</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.25.009084 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1237315" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David H. Brann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Harvard Medical School Department of Neurobiology</span>, Boston MA 02115 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BH.%2BBrann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brann%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BH.%2BBrann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8844-8886" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David H. Brann</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tatsuya Tsukahara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Harvard Medical School Department of Neurobiology</span>, Boston MA 02115 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tatsuya%2BTsukahara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsukahara%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATatsuya%2BTsukahara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6928-8562" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tatsuya Tsukahara</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Caleb Weinreb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Harvard Medical School Department of Neurobiology</span>, Boston MA 02115 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caleb%2BWeinreb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinreb%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACaleb%2BWeinreb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6100-6084" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Caleb Weinreb</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcela Lipovsek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Centre for Developmental Neurobiology, Institute of Psychiatry, Psychology and Neuroscience (IoPPN), King’s College London</span>, London SE1 1UL, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcela%2BLipovsek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipovsek%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcela%2BLipovsek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Koen Van den Berge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Statistics, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Applied Mathematics, Computer Science and Statistics, Ghent University</span>, Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koen%2BVan%2Bden%2BBerge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20den%20Berge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKoen%2BVan%2Bden%2BBerge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1833-8478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Koen Van den Berge</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Boying Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Division of Biostatistics, School of Public Health, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boying%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoying%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rebecca Chance </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Molecular and Cell Biology, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BChance%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chance%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BChance%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Iain C. Macaulay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Earlham Institute, Norwich Research Park</span>, Norwich, NR4 7UZ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BC.%2BMacaulay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macaulay%20IC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIain%2BC.%2BMacaulay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hsin-jung Chou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Molecular and Cell Biology, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hsin-jung%2BChou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chou%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHsin-jung%2BChou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Russell Fletcher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Molecular and Cell Biology, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Surrozen, Inc., South San Francisco</span>, CA 94080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Russell%2BFletcher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fletcher%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARussell%2BFletcher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Diya Das </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Molecular and Cell Biology, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Genentech, Inc., South San Francisco</span>, CA 94080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diya%2BDas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Das%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiya%2BDas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9646-8983" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diya Das</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Kelly Street </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Department of Data Sciences, Dana-Farber Cancer Institute</span>, Boston, MA</div><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>Department of Biostatistics, Harvard T.H. Chan School of Public Health</span>, Boston, MA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelly%2BStreet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Street%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelly%2BStreet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6379-5013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelly Street</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Hector Roux de Bezieux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Division of Biostatistics, School of Public Health, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>12</span><span class='nlm-institution'>Center for Computational Biology, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hector%2BRoux%2Bde%2BBezieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Bezieux%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHector%2BRoux%2Bde%2BBezieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1489-8339" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hector Roux de Bezieux</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Yoon-Gi Choi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>13</span><span class='nlm-institution'>QB3 Functional Genomics Laboratory, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoon-Gi%2BChoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Choi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoon-Gi%2BChoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Davide Risso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>14</span><span class='nlm-institution'>Department of Statistical Sciences, University of Padova</span>, Padova, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davide%2BRisso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Risso%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavide%2BRisso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Sandrine Dudoit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Statistics, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Division of Biostatistics, School of Public Health, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandrine%2BDudoit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudoit%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandrine%2BDudoit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Elizabeth Purdom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Division of Biostatistics, School of Public Health, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BPurdom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Purdom%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BPurdom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9455-7990" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Purdom</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Jonathan S. Mill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>15</span><span class='nlm-institution'>University of Exeter Medical School, College of Medicine &#38; Health, University of Exeter</span>, Exeter EX2 5DW, UK16</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BS.%2BMill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mill%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BS.%2BMill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Ralph Abi Hachem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>16</span><span class='nlm-institution'>Duke University School of Medicine Department of Head and Neck Surgery &#38; Communication Sciences</span>, Durham, NC 27717 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BAbi%2BHachem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hachem%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARalph%2BAbi%2BHachem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Hiroaki Matsunami </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>17</span><span class='nlm-institution'>Duke University School of Medicine Department of Molecular Genetics and Microbiology, Department of Neurobiology, Duke Institute for Brain Sciences</span>, Durham, NC 27717 <span class='nlm-country'>US</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroaki%2BMatsunami%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matsunami%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHiroaki%2BMatsunami%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Darren W. Logan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>18</span><span class='nlm-institution'>Waltham Petcare Science Institute</span>, Leicestershire LE14 4RT, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Darren%2BW.%2BLogan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Logan%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADarren%2BW.%2BLogan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Bradley J. Goldstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>16</span><span class='nlm-institution'>Duke University School of Medicine Department of Head and Neck Surgery &#38; Communication Sciences</span>, Durham, NC 27717 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BJ.%2BGoldstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goldstein%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABradley%2BJ.%2BGoldstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Matthew S. Grubb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Centre for Developmental Neurobiology, Institute of Psychiatry, Psychology and Neuroscience (IoPPN), King’s College London</span>, London SE1 1UL, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BGrubb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grubb%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BGrubb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">John Ngai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Molecular and Cell Biology, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>13</span><span class='nlm-institution'>QB3 Functional Genomics Laboratory, University of California</span>, Berkeley, CA 94720</div><div class='author-affiliation'><span class='nlm-sup'>19</span><span class='nlm-institution'>Helen Wills Neuroscience Institute, University of California</span>, Berkeley, CA 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNgai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ngai%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BNgai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Sandeep Robert Datta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Harvard Medical School Department of Neurobiology</span>, Boston MA 02115 <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandeep%2BRobert%2BDatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Datta%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASandeep%2BRobert%2BDatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8068-3862" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sandeep Robert Datta</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srdatta@hms.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">srdatta@hms.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.25.009084v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.009084v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.009084v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.009084v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.009084v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.25.009084v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1237315" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1237315/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">A subset of COVID-19 patients exhibit altered olfactory function. Here we analyze bulk and single cell RNA-Seq datasets to identify cell types in the olfactory epithelium and olfactory bulb that express cell entry molecules that mediate infection by SARS-CoV-2 (CoV-2), the causal agent in COVID-19. We find that samples from whole olfactory mucosa in species including mouse and human express two key genes involved in CoV-2 entry, ACE2 and TMPRSS2. However, neither olfactory sensory neurons nor olfactory bulb neurons express these genes, which are instead expressed in support cells, stem cells, and perivascular cells. These findings suggest that CoV-2 infection of non-neuronal cell types leads to anosmia and related disturbances in odor perception in COVID-19 patients.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>One Sentence Summary</strong> Analysis of new and previously published single-cell sequencing datasets reveals that the SARS-CoV2 receptor ACE2 is expressed in olfactory support cells, stem cells and perivascular cells — but not in neurons — suggesting mechanisms through which the COVID-19 syndrome could lead to olfactory dysfunction.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-15">SARS-CoV-2 (CoV-2) is a pandemic coronavirus that causes the COVID-19 syndrome, which can include upper respiratory infection (URI) symptoms, severe respiratory distress, acute cardiac injury and death (<em><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></em>). CoV-2 is closely related to other beta-coronaviruses, including the causal agents in pandemic SARS and MERS (SARS-CoV and MERS-CoV, respectively) and endemic viruses typically associated with mild URI syndromes (hCoV-OC43 and hCoV-229E) (<em><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></em>). Clinical reports suggest that infection with CoV-2 is associated with high rates of disturbances in smell and taste perception, including anosmia (<em><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></em>). These observations are consistent with a case report describing a SARS patient with long term anosmia (<em><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></em>) and with data demonstrating that olfactory function is commonly altered after infection with endemic coronaviruses (<em><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></em>). While many viruses induce transient changes in odor perception due to inflammatory responses, in at least some cases COVID-related anosmia has been reported to occur in the absence of significant nasal inflammation or coryzal symptoms (<em><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></em>). This observation suggests that CoV-2 might directly target odor processing mechanisms, although the specific means through which CoV-2 alters odor perception remains unknown.</p><p id="p-16">The nasal epithelium is anatomically divided into a respiratory epithelium (RE) and sensory olfactory epithelium (OE), whose functions and cell types differ. The nasal RE is continuous with the epithelium that lines much of the respiratory tract and is thought to humidify air as it enters the nose; main cell types include basal cells, ciliated cells, secretory cells (including goblet cells), and brush/microvillar cells (<em><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></em>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of nasal respiratory and olfactory epithelium. Schematic of a sagittal view of the human nasal cavity, in which respiratory and olfactory epithelium are colored (left). For each type of epithelium, a schematic of the anatomy and known major cell types are shown (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Schematic of nasal respiratory and olfactory epithelium.&lt;/span&gt; Schematic of a sagittal view of the human nasal cavity, in which respiratory and olfactory epithelium are colored (left). For each type of epithelium, a schematic of the anatomy and known major cell types are shown (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Schematic of nasal respiratory and olfactory epithelium.</span><p id="p-17" class="first-child">Schematic of a sagittal view of the human nasal cavity, in which respiratory and olfactory epithelium are colored (left). For each type of epithelium, a schematic of the anatomy and known major cell types are shown (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">The OE is a laminated structure housing mature olfactory sensory neurons (OSNs), which detect odors via receptors expressed on their dendritic cilia. OSNs are supported by sustentacular cells, which act to structurally support sensory neurons, phagocytose and/or detoxify potentially damaging agents, and maintain local salt and water balance (<em><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></em>); microvillar cells and mucus-secreting Bowman’s gland cells also play important roles in maintaining OE homeostasis and function (<em><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></em>)(<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In addition, the OE contains globose basal cells (GBCs), which are primarily responsible for regenerating OSNs during normal epithelial turnover, and horizontal basal cells (HBCs), which act as reserve stem cells activated upon tissue damage (<em><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>–<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></em>). OSNs elaborate axons that puncture the cribriform plate at the base of the skull and terminate in the olfactory bulb, whose local circuits process olfactory information before sending it to higher brain centers.</p><p id="p-19">CoV-2 — like SARS-CoV — infects cells through interactions between its spike (S) protein and the ACE2 protein on target cells. This interaction requires cleavage of the S protein, likely by the cell surface protease TMPRSS2, although other proteases (such as Cathepsin B and L, CTSB/CTSL) may also be involved (<em><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></em>). Other coronaviruses use different cell surface receptors and proteases to facilitate cellular entry, including DPP4, FURIN and HSPA5 for MERS-CoV, ANPEP for HCoV-229E, TMPRSS11D for SARS-CoV (in addition to ACE2 and TMPRSS2), and ST6GAL1 and ST3GAL4 for HCoV-OC43 and HCoV-HKU1 (<em><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>–<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></em>). It has recently been demonstrated through single cell RNA sequencing analysis (referred to herein as scSeq) that cells from the human upper airway — including nasal RE goblet, basal and ciliated cells —express high levels of ACE2 and TMPRSS2, suggesting that these RE cell types may serve as a viral reservoir during CoV-2 infection (<em><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></em>). However, analyzed samples in that dataset did not include any OSNs or sustentacular cells, suggesting that tissue sampling in these experiments did not access the OE (<em><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></em>). Here we query both new and previously published bulk RNA-Seq and scSeq datasets from the olfactory system for expression of ACE2, TMRPSS2 and other genes implicated in coronavirus entry. We find that non-neuronal cells in the OE and olfactory bulb, including support, stem and perivascular cells, express CoV-2 entry-associated molecules, raising the possibility that infection of these non-neuronal cell types contributes to anosmia in COVID-19 patients.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><p id="p-20">To determine whether genes relevant to CoV-2 entry are expressed in OSNs or other cell types in the OE, we examined published datasets in which RNA-Seq was independently performed on mouse whole olfactory mucosa (WOM) and on purified populations of mature OSNs (<em><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></em>). The CoV-2 receptor Ace2 and the protease Tmprss2 were expressed in WOM, as were the cathepsins Ctsb and Ctsl (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2A-B</a>) (<em><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></em>). However, expression of these genes (with the exception of Ctsb) was much lower and Ace2 expression was nearly absent in purified OSN samples (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, see Legend for counts, see Methods). Genes used for cell entry by other CoVs (except St3gal4) were also expressed in WOM, and de-enriched in purified OSNs. These samples reflected the presence of expected cell types, as the WOM sequencing data included gene expression profiles associated with a wide variety of RE and OE cells, while the purified OSN samples were specifically enriched for markers of mature OSNs. The de-enrichment of Ace2 and Tmprss2 in OSNs relative to WOM was also observed in two other mouse RNA-Seq datasets (<em><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></em>) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 2. Analysis of mouse bulk RNA-Seq and WOM scSeq dataset. (A) Log2-fold change (FC) of gene expression between OSNs and WOM as in Fig. 2A for three bulk RNA-sequencing datasets. MHV, mouse hepatitis virus. Left plot is same as Fig. 2A except for the addition of Ceacam1. (B) CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for cell types in Fig. 2C., in UMAP representation of WOM dataset with normalized expression. Gfap-positive OECs (olfactory ensheathing cells) and Muc5b-positive secretory cells are indicated by asterisks. (C) Gene expression for CoV-related genes including Ace2 and Tmprss2 as well as marker genes for olfactory and RE subtypes are shown normalized by their maximum expression across cell types. Ace2 and Tmprss2 are expressed in WOM respiratory and olfactory cell types, but not in OSNs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 2. Analysis of mouse bulk RNA-Seq and WOM scSeq dataset.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Log&lt;sub&gt;2&lt;/sub&gt;-fold change (FC) of gene expression between OSNs and WOM as in Fig. 2A for three bulk RNA-sequencing datasets. MHV, mouse hepatitis virus. Left plot is same as Fig. 2A except for the addition of Ceacam1. &lt;strong&gt;(B)&lt;/strong&gt; CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for cell types in Fig. 2C., in UMAP representation of WOM dataset with normalized expression. Gfap-positive OECs (olfactory ensheathing cells) and Muc5b-positive secretory cells are indicated by asterisks. &lt;strong&gt;(C)&lt;/strong&gt; Gene expression for CoV-related genes including Ace2 and Tmprss2 as well as marker genes for olfactory and RE subtypes are shown normalized by their maximum expression across cell types. Ace2 and Tmprss2 are expressed in WOM respiratory and olfactory cell types, but not in OSNs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F5.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F5.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S1.</span> <span class="caption-title">Related to <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2</a>. Analysis of mouse bulk RNA-Seq and WOM scSeq dataset.</span><p id="p-96" class="first-child"><strong>(A)</strong> Log<sub>2</sub>-fold change (FC) of gene expression between OSNs and WOM as in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2A</a> for three bulk RNA-sequencing datasets. MHV, mouse hepatitis virus. Left plot is same as <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2A</a> except for the addition of Ceacam1. <strong>(B)</strong> CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for cell types in <a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2C</a>., in UMAP representation of WOM dataset with normalized expression. Gfap-positive OECs (olfactory ensheathing cells) and Muc5b-positive secretory cells are indicated by asterisks. <strong>(C)</strong> Gene expression for CoV-related genes including Ace2 and Tmprss2 as well as marker genes for olfactory and RE subtypes are shown normalized by their maximum expression across cell types. Ace2 and Tmprss2 are expressed in WOM respiratory and olfactory cell types, but not in OSNs.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coronavirus cell entry-related genes are expressed in mouse olfactory mucosa but not in mature OSNs. (A) Expression of coronavirus (CoV)-related genes and cell type markers in mouse olfactory mucosa from Saraiva et al. 2015 (39). Normalized counts for each gene in the whole olfactory mucosa (WOM) and olfactory sensory neurons (OSNs) are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right (CoV-2 and other CoVs: genes involved in the entry of these viruses, other categories: marker genes for specific cell types such as horizontal basal cells (HBC), and sustentacular cells (SUS)). Mean &#xB1; SD normalized counts in OSNs &#x2014; Ace2: 8.6 &#xB1; 4.2, Tmprss2 117.3 &#xB1; 24.7, Ctsb: 38616.7 &#xB1; 1650.2, Ctsl: 1705 &#xB1; 87; same in WOM &#x2014; Ace2: 254.7 &#xB1; 22.5, Tmprss2 2279 &#xB1; 219.6, Ctsb: 22380 &#xB1; 947, Ctsl: 4900 &#xB1; 90.5) (B) Log2-fold change (FC) in mean across-replicate gene expression between OSNs and WOM, calculated for the genes shown in A. (C) UMAP representation of single cell transcriptome data from WOM, colored by cell types. Abbreviations of cell type names: mOSN: mature OSN, iOSN: immature OSN, SUS: Sustentacular cell, MV: Microvillar cell, OEC: Olfactory ensheathing cell, Resp.: Respiratory, RBC: Red Blood Cell. Black dashed box indicates the area shown in E. (D) CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for HBC, SUS, Bowman&#x2019;s gland, and Respiratory cells in UMAP representation. Each point represents an individual cell, and the color represents the normalized expression level for each gene (nUMIs for a given gene divided by the total nUMIs for each cell). Only the cells in the area shown by black dashed box in C are plotted. (E) Percent of cells expressing Ace2 and Tmprss2. Detection was considered positive if any transcripts (UMIs) were expressed for a given gene. (F) UMAP representation of single cell transcriptome data from the HBC lineage dataset colored by identified cell types. HBC* = activated or cycling HBCs. (G) CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for various cell types, in UMAP representation of single cell transcriptomes with normalized expression. (H) Percent of cells expressing Ace2 and Tmprss2. (I) Reanalysis of the HBC lineage dataset in Gadye et al. 2017 (42), showing the percent of cells expressing Ace2 and Tmprss2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Coronavirus cell entry-related genes are expressed in mouse olfactory mucosa but not in mature OSNs.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Expression of coronavirus (CoV)-related genes and cell type markers in mouse olfactory mucosa from Saraiva et al. 2015 (&lt;em&gt;39&lt;/em&gt;). Normalized counts for each gene in the whole olfactory mucosa (WOM) and olfactory sensory neurons (OSNs) are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right (CoV-2 and other CoVs: genes involved in the entry of these viruses, other categories: marker genes for specific cell types such as horizontal basal cells (HBC), and sustentacular cells (SUS)). Mean &#xB1; SD normalized counts in OSNs &#x2014; Ace2: 8.6 &#xB1; 4.2, Tmprss2 117.3 &#xB1; 24.7, Ctsb: 38616.7 &#xB1; 1650.2, Ctsl: 1705 &#xB1; 87; same in WOM &#x2014; Ace2: 254.7 &#xB1; 22.5, Tmprss2 2279 &#xB1; 219.6, Ctsb: 22380 &#xB1; 947, Ctsl: 4900 &#xB1; 90.5) &lt;strong&gt;(B)&lt;/strong&gt; Log&lt;sub&gt;2&lt;/sub&gt;-fold change (FC) in mean across-replicate gene expression between OSNs and WOM, calculated for the genes shown in &lt;strong&gt;A&lt;/strong&gt;. &lt;strong&gt;(C)&lt;/strong&gt; UMAP representation of single cell transcriptome data from WOM, colored by cell types. Abbreviations of cell type names: mOSN: mature OSN, iOSN: immature OSN, SUS: Sustentacular cell, MV: Microvillar cell, OEC: Olfactory ensheathing cell, Resp.: Respiratory, RBC: Red Blood Cell. Black dashed box indicates the area shown in &lt;strong&gt;E&lt;/strong&gt;. &lt;strong&gt;(D)&lt;/strong&gt; CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for HBC, SUS, Bowman&#x2019;s gland, and Respiratory cells in UMAP representation. Each point represents an individual cell, and the color represents the normalized expression level for each gene (nUMIs for a given gene divided by the total nUMIs for each cell). Only the cells in the area shown by black dashed box in &lt;strong&gt;C&lt;/strong&gt; are plotted. &lt;strong&gt;(E)&lt;/strong&gt; Percent of cells expressing Ace2 and Tmprss2. Detection was considered positive if any transcripts (UMIs) were expressed for a given gene. &lt;strong&gt;(F)&lt;/strong&gt; UMAP representation of single cell transcriptome data from the HBC lineage dataset colored by identified cell types. HBC* = activated or cycling HBCs. &lt;strong&gt;(G)&lt;/strong&gt; CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for various cell types, in UMAP representation of single cell transcriptomes with normalized expression. &lt;strong&gt;(H)&lt;/strong&gt; Percent of cells expressing Ace2 and Tmprss2. &lt;strong&gt;(I)&lt;/strong&gt; Reanalysis of the HBC lineage dataset in Gadye et al. 2017 (&lt;em&gt;42&lt;/em&gt;), showing the percent of cells expressing Ace2 and Tmprss2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F2.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F2.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">Coronavirus cell entry-related genes are expressed in mouse olfactory mucosa but not in mature OSNs.</span><p id="p-21" class="first-child"><strong>(A)</strong> Expression of coronavirus (CoV)-related genes and cell type markers in mouse olfactory mucosa from Saraiva et al. 2015 (<em><a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a></em>). Normalized counts for each gene in the whole olfactory mucosa (WOM) and olfactory sensory neurons (OSNs) are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right (CoV-2 and other CoVs: genes involved in the entry of these viruses, other categories: marker genes for specific cell types such as horizontal basal cells (HBC), and sustentacular cells (SUS)). Mean ± SD normalized counts in OSNs — Ace2: 8.6 ± 4.2, Tmprss2 117.3 ± 24.7, Ctsb: 38616.7 ± 1650.2, Ctsl: 1705 ± 87; same in WOM — Ace2: 254.7 ± 22.5, Tmprss2 2279 ± 219.6, Ctsb: 22380 ± 947, Ctsl: 4900 ± 90.5) <strong>(B)</strong> Log<sub>2</sub>-fold change (FC) in mean across-replicate gene expression between OSNs and WOM, calculated for the genes shown in <strong>A</strong>. <strong>(C)</strong> UMAP representation of single cell transcriptome data from WOM, colored by cell types. Abbreviations of cell type names: mOSN: mature OSN, iOSN: immature OSN, SUS: Sustentacular cell, MV: Microvillar cell, OEC: Olfactory ensheathing cell, Resp.: Respiratory, RBC: Red Blood Cell. Black dashed box indicates the area shown in <strong>E</strong>. <strong>(D)</strong> CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for HBC, SUS, Bowman’s gland, and Respiratory cells in UMAP representation. Each point represents an individual cell, and the color represents the normalized expression level for each gene (nUMIs for a given gene divided by the total nUMIs for each cell). Only the cells in the area shown by black dashed box in <strong>C</strong> are plotted. <strong>(E)</strong> Percent of cells expressing Ace2 and Tmprss2. Detection was considered positive if any transcripts (UMIs) were expressed for a given gene. <strong>(F)</strong> UMAP representation of single cell transcriptome data from the HBC lineage dataset colored by identified cell types. HBC* = activated or cycling HBCs. <strong>(G)</strong> CoV-2 related genes Ace2 and Tmprss2, as well as marker genes for various cell types, in UMAP representation of single cell transcriptomes with normalized expression. <strong>(H)</strong> Percent of cells expressing Ace2 and Tmprss2. <strong>(I)</strong> Reanalysis of the HBC lineage dataset in Gadye et al. 2017 (<em><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></em>), showing the percent of cells expressing Ace2 and Tmprss2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The presence of Ace2 and Tmprss2 transcripts in mouse WOM and their (near total) absence in purified OSNs suggest that the molecular components that enable CoV-2 entry into cells are expressed in non-neuronal cell types in the mouse nasal epithelium. To identify the specific cell types that express Ace2 and Tmprss2, we performed scSeq (using the 10x Chromium platform, see Methods) on the WOM; olfactory sensory neurons did not express Ace2 in this de novo analysis (2 of 28769 mature OSNs were positive for Ace2), while expression was observed in Bowman’s gland cells and HBCs in a small percentage of cells (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2C-E</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">S1B-C</a>, see methods). Because WOM scSeq de-enriches certain cell types (e.g., sustentacular cells), to complement these data we employed a previously established lineage tracing protocol to perform scSeq on HBCs and their descendants during injury-induced regeneration (<em><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></em>). This analysis revealed that Ace2 and Tmprss2 are expressed in subsets of sustentacular cells and HBCs, as well as activated HBCs that serve to regenerate the epithelium (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2F-H</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">S2A</a>). To validate these results, we re-analyzed a similar lineage tracing dataset in which identified HBCs and their progeny were subject to Smart-seq2-based deep sequencing (<em><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></em>). In this dataset, Ace2 was detected in more than 0.7% of GBCs, nearly 2% of HBCs and nearly 3% of sustentacular cells but was not detected in OSNs (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figures 2I</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">S2B</a>). Furthermore, larger percentages of HBCs, GBCs and sustentacular cells expressed Tmprss2. Taken together, these bulk RNA-Seq and scSeq mouse data demonstrate that CoV-2 cell entry-related genes are expressed in support and stem cells in the mouse OE.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 2. Analysis of mouse HBC lineage scSeq datasets. (A) HBC lineage data set. (left) Cell type marker genes in UMAP representation of HBC lineage dataset with normalized expression. (right) Same as Fig. S1C but for HBC lineage dataset. HBC*= activated or cycling HBCs. (B) HBC lineage dataset from (42). UMAP representation, (left) colored by HBC lineage subtypes and (right) colored by expression of identified marker genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 2. Analysis of mouse HBC lineage scSeq datasets.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; HBC lineage data set. (left) Cell type marker genes in UMAP representation of HBC lineage dataset with normalized expression. (right) Same as Fig. S1C but for HBC lineage dataset. HBC*= activated or cycling HBCs. &lt;strong&gt;(B)&lt;/strong&gt; HBC lineage dataset from (&lt;em&gt;42&lt;/em&gt;). UMAP representation, (left) colored by HBC lineage subtypes and (right) colored by expression of identified marker genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F6.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F6.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S2.</span> <span class="caption-title">Related to <a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2</a>. Analysis of mouse HBC lineage scSeq datasets.</span><p id="p-97" class="first-child"><strong>(A)</strong> HBC lineage data set. (left) Cell type marker genes in UMAP representation of HBC lineage dataset with normalized expression. (right) Same as <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. S1C</a> but for HBC lineage dataset. HBC*= activated or cycling HBCs. <strong>(B)</strong> HBC lineage dataset from (<em><a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">42</a></em>). UMAP representation, (left) colored by HBC lineage subtypes and (right) colored by expression of identified marker genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To address whether there are similar patterns of gene expression in human OE, we queried previously published bulk WOM RNA-Seq data derived from macaque, marmoset and human (<em><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></em>), and found expression of almost all CoV-entry-related genes in all WOM samples, consistent with our observations in mouse WOM (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S3A</a>). To identify the specific cell types in human OE that express ACE2, we quantified gene expression in scSeq derived from four human nasal biopsy samples recently reported by Durante et al (<em><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></em>). As in the mouse scSeq datasets, neither ACE2 nor TMPRSS2 were detected in mature OSNs, whereas these genes were detected in both sustentacular cells and HBCs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3A-E</a> and <a id="xref-fig-7-2" class="xref-fig" href="#F7">S3B-C</a>). In contrast, genes relevant to cell entry of other CoVs were expressed in OSNs, as well as in other OE cell types. We confirmed the expression of ACE2 proteins via immunohistochemical staining of human olfactory epithelium biopsy tissue, which revealed expression in sustentacular and basal cells, and an absence of ACE2 protein in OSNs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3F</a>). Together, these results demonstrate that sustentacular and olfactory stem cells, but not mature OSNs, are potentially direct targets of CoV-2 in the human OE.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 3. Bulk RNA-seq datasets for primates and analysis of human scSeq data. (A) Expression of genes required for the entry of coronavirus (CoV) and OSN markers in primate olfactory mucosa in the data from Saraiva et al 2019 (43). Human (H. Sap), Macaque (M. Mul) and Marmoset (C. Jal) data are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right. Raw counts were normalized to account for differences in sequencing depth between samples. (B) Additional marker genes, in UMAP representations with normalized expression, for all the cell types shown in Fig. 3A, including Microvillar (MV) Brush-like cells, Bowman&#x2019;s gland, Olfactory and Respiratory HBCs, and other immune and endothelial cell types. CFTR-expressing MV Ionocyte-like cells and SH2D7-expressing MV Brush-like cells are indicated by asterisks. (C) Percent of cells expressing ACE2 and TMPRSS2 in the cell types not shown in Fig. 3D. ACE2 was rarely detected in non-epithelial cell types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 3. Bulk RNA-seq datasets for primates and analysis of human scSeq data.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Expression of genes required for the entry of coronavirus (CoV) and OSN markers in primate olfactory mucosa in the data from Saraiva et al 2019 (&lt;em&gt;43&lt;/em&gt;). Human (H. Sap), Macaque (M. Mul) and Marmoset (C. Jal) data are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right. Raw counts were normalized to account for differences in sequencing depth between samples. &lt;strong&gt;(B)&lt;/strong&gt; Additional marker genes, in UMAP representations with normalized expression, for all the cell types shown in Fig. 3A, including Microvillar (MV) Brush-like cells, Bowman&#x2019;s gland, Olfactory and Respiratory HBCs, and other immune and endothelial cell types. CFTR-expressing MV Ionocyte-like cells and SH2D7-expressing MV Brush-like cells are indicated by asterisks. &lt;strong&gt;(C)&lt;/strong&gt; Percent of cells expressing ACE2 and TMPRSS2 in the cell types not shown in Fig. 3D. ACE2 was rarely detected in non-epithelial cell types.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F7.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F7.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S3.</span> <span class="caption-title">Related to <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3</a>. Bulk RNA-seq datasets for primates and analysis of human scSeq data.</span><p id="p-98" class="first-child"><strong>(A)</strong> Expression of genes required for the entry of coronavirus (CoV) and OSN markers in primate olfactory mucosa in the data from Saraiva et al 2019 (<em><a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">43</a></em>). Human (H. Sap), Macaque (M. Mul) and Marmoset (C. Jal) data are shown. Each circle represents a biological replicate and each color indicates the category of the gene shown on the right. Raw counts were normalized to account for differences in sequencing depth between samples. <strong>(B)</strong> Additional marker genes, in UMAP representations with normalized expression, for all the cell types shown in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3A</a>, including Microvillar (MV) Brush-like cells, Bowman’s gland, Olfactory and Respiratory HBCs, and other immune and endothelial cell types. CFTR-expressing MV Ionocyte-like cells and SH2D7-expressing MV Brush-like cells are indicated by asterisks. <strong>(C)</strong> Percent of cells expressing ACE2 and TMPRSS2 in the cell types not shown in <a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3D</a>. ACE2 was rarely detected in non-epithelial cell types.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coronavirus cell entry-related genes are expressed in human respiratory and olfactory epithelium, but are not detected in human OSNs. (A) UMAP representation of cell types in human nasal biopsy scSeq data from Durante et al. 2020 (44). Each dot represents an individual cell, colored by cell type. (B) Percent of cells expressing ACE2 and TMPRSS2. ACE2 was not detected in any OSNs. Olfactory and respiratory cell types are shown separately. (C) UMAP representations of 865 detected immature (GNG8) and mature (GNG13) OSNs. Neither ACE2 nor TMPRSS2 are detected in either population of OSNs. (D) UMAP representations of all cells with the normalized expression of CoV-2 related genes ACE2 and TMPRSS2, as well as markers of several cell types. ACE2 and TMPRSS2 are expressed in respiratory and olfactory cell types, but not in OSNs. ACE2 and TMPRSS2 are detected in HBC (KRT5) and sustentacular (CYP2A13) cells, as well as other respiratory epithelial cell types, including respiratory ciliated (FOXJ1) cells. (E) Cell type markers and various CoV related genes including ACE2 and TMPRSS2, are expressed in respiratory and olfactory cell types, but not in OSNs. Gene expression for ACE2 and TMPRSS2 as well as marker genes for olfactory and respiratory epithelial subtypes are shown normalized by their maximum expression across cell types. MHV, mouse hepatitis virus. (F) ACE2 immunohistochemistry of human olfactory mucosal biopsy samples. ACE2 protein (green) is detected in sustentacular cells (white arrows) in both 86-year old male sample (left) and 39-year old female sample (middle). Inset in the left image is an enlarged view of the dashed box. ACE2 does not appear to colocalize with OSN marker TUJ1 (magenta). ACE2 is also detected in basal cells (white arrowheads in the middle image). Nuclei were stained with DAPI. Bar = 50 &#xB5;m. (G) Gene expression across cell types and tissues in Durante et al. (top) and Deprez et al. (bottom). Each gene is normalized to its maximum value across all tissues. Gene expression from Durante et al was transformed (see Methods and Figure S4). The tissues correspond to progressive positions along the airway from nasal to distal lung." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Coronavirus cell entry-related genes are expressed in human respiratory and olfactory epithelium, but are not detected in human OSNs.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; UMAP representation of cell types in human nasal biopsy scSeq data from Durante et al. 2020 (&lt;em&gt;44&lt;/em&gt;). Each dot represents an individual cell, colored by cell type. &lt;strong&gt;(B)&lt;/strong&gt; Percent of cells expressing ACE2 and TMPRSS2. ACE2 was not detected in any OSNs. Olfactory and respiratory cell types are shown separately. &lt;strong&gt;(C)&lt;/strong&gt; UMAP representations of 865 detected immature (GNG8) and mature (GNG13) OSNs. Neither ACE2 nor TMPRSS2 are detected in either population of OSNs. &lt;strong&gt;(D)&lt;/strong&gt; UMAP representations of all cells with the normalized expression of CoV-2 related genes ACE2 and TMPRSS2, as well as markers of several cell types. ACE2 and TMPRSS2 are expressed in respiratory and olfactory cell types, but not in OSNs. ACE2 and TMPRSS2 are detected in HBC (KRT5) and sustentacular (CYP2A13) cells, as well as other respiratory epithelial cell types, including respiratory ciliated (FOXJ1) cells. &lt;strong&gt;(E)&lt;/strong&gt; Cell type markers and various CoV related genes including ACE2 and TMPRSS2, are expressed in respiratory and olfactory cell types, but not in OSNs. Gene expression for ACE2 and TMPRSS2 as well as marker genes for olfactory and respiratory epithelial subtypes are shown normalized by their maximum expression across cell types. MHV, mouse hepatitis virus. &lt;strong&gt;(F)&lt;/strong&gt; ACE2 immunohistochemistry of human olfactory mucosal biopsy samples. ACE2 protein (green) is detected in sustentacular cells (white arrows) in both 86-year old male sample (left) and 39-year old female sample (middle). Inset in the left image is an enlarged view of the dashed box. ACE2 does not appear to colocalize with OSN marker TUJ1 (magenta). ACE2 is also detected in basal cells (white arrowheads in the middle image). Nuclei were stained with DAPI. Bar = 50 &#xB5;m. &lt;strong&gt;(G)&lt;/strong&gt; Gene expression across cell types and tissues in Durante et al. (top) and Deprez et al. (bottom). Each gene is normalized to its maximum value across all tissues. Gene expression from Durante et al was transformed (see Methods and Figure S4). The tissues correspond to progressive positions along the airway from nasal to distal lung.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F3.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F3.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Coronavirus cell entry-related genes are expressed in human respiratory and olfactory epithelium, but are not detected in human OSNs.</span><p id="p-24" class="first-child"><strong>(A)</strong> UMAP representation of cell types in human nasal biopsy scSeq data from Durante et al. 2020 (<em><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></em>). Each dot represents an individual cell, colored by cell type. <strong>(B)</strong> Percent of cells expressing ACE2 and TMPRSS2. ACE2 was not detected in any OSNs. Olfactory and respiratory cell types are shown separately. <strong>(C)</strong> UMAP representations of 865 detected immature (GNG8) and mature (GNG13) OSNs. Neither ACE2 nor TMPRSS2 are detected in either population of OSNs. <strong>(D)</strong> UMAP representations of all cells with the normalized expression of CoV-2 related genes ACE2 and TMPRSS2, as well as markers of several cell types. ACE2 and TMPRSS2 are expressed in respiratory and olfactory cell types, but not in OSNs. ACE2 and TMPRSS2 are detected in HBC (KRT5) and sustentacular (CYP2A13) cells, as well as other respiratory epithelial cell types, including respiratory ciliated (FOXJ1) cells. <strong>(E)</strong> Cell type markers and various CoV related genes including ACE2 and TMPRSS2, are expressed in respiratory and olfactory cell types, but not in OSNs. Gene expression for ACE2 and TMPRSS2 as well as marker genes for olfactory and respiratory epithelial subtypes are shown normalized by their maximum expression across cell types. MHV, mouse hepatitis virus. <strong>(F)</strong> ACE2 immunohistochemistry of human olfactory mucosal biopsy samples. ACE2 protein (green) is detected in sustentacular cells (white arrows) in both 86-year old male sample (left) and 39-year old female sample (middle). Inset in the left image is an enlarged view of the dashed box. ACE2 does not appear to colocalize with OSN marker TUJ1 (magenta). ACE2 is also detected in basal cells (white arrowheads in the middle image). Nuclei were stained with DAPI. Bar = 50 µm. <strong>(G)</strong> Gene expression across cell types and tissues in Durante et al. (top) and Deprez et al. (bottom). Each gene is normalized to its maximum value across all tissues. Gene expression from Durante et al was transformed (see Methods and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S4</a>). The tissues correspond to progressive positions along the airway from nasal to distal lung.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 3. Comparing cell types and expression levels across respiratory and olfactory epithelial datasets. (A) Schematic of the mapping strategy used to identify similar cell types across datasets, applied to a toy example. Each cell type from &#x201C;Dataset 1&#x201D; dataset is mapped to cell types from the &#x201C;Dataset 2&#x201D;. From left to right: Each Dataset 1 cell voted on its 5 most similar cells in Dataset 2; the total number of votes cast for each Dataset 2 cell type was quantified; and vote totals were Z-scored against 1000 shuffles where cell type labels were permutated. (B) Mapping was performed bi-directionally between the Deprez and Durante datasets, and the mapping Z-scores in each direction are compared. The set of cell type correspondences with high Z-scores (&gt;25) in both directions are colored red. (C) Mapping Z-scores from one of the two mappings (Deprez -&gt; Durante). (D) The set of cell type correspondences with high bi-directional mappings (both Z-scores &gt; 25; equivalent to the red dots in B). (E) Gene expression in olfactory microvillar cells from the Durante dataset. These cells express classical microvillar genes (ASCL3) as well as marker genes of pulmonary ionocytes (CFTR, FOXI1). (F) Top: average expression for each gene in three cell types compared between the Durante and Deprez datasets in the units of the original papers. Bottom: average expression after a global non-linear transformation of the Durante data. Basal/suprabasal cells = &#x201C;respiratory HBCs&#x201D; from Durante et al., and &#x201C;basal&#x201D; and &#x201C;suprabasal&#x201D; cells from Deprez et al. Secretory cells = &#x201C;early respiratory secretory cells&#x201D; and &#x201C;respiratory secretory cells&#x201D; from Durante et al., and &#x201C;secretory&#x201D; cells from Deprez et al. Mutliciliated cells = &#x201C;respiratory ciliated cells&#x201D; from Durante et al., and &#x201C;multiciliated&#x201D; cells from Deprez et al. (G) Average expression of genes shown in Fig 3G before (left) and after (right) gene-specific rescaling. Each color is one gene. For every color, three dots are shown, corresponding to expression across the three cell types in F. The diagonal line represents unity. Dots close to the diagonal line indicate a close match in transformed gene expression the Durante and Deprez data for the cell types shown in F. (H) Comparison of percent squared error before versus after gene-specific rescaling." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 3. Comparing cell types and expression levels across respiratory and olfactory epithelial datasets.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Schematic of the mapping strategy used to identify similar cell types across datasets, applied to a toy example. Each cell type from &#x201C;Dataset 1&#x201D; dataset is mapped to cell types from the &#x201C;Dataset 2&#x201D;. From left to right: Each Dataset 1 cell voted on its 5 most similar cells in Dataset 2; the total number of votes cast for each Dataset 2 cell type was quantified; and vote totals were Z-scored against 1000 shuffles where cell type labels were permutated. &lt;strong&gt;(B)&lt;/strong&gt; Mapping was performed bi-directionally between the Deprez and Durante datasets, and the mapping Z-scores in each direction are compared. The set of cell type correspondences with high Z-scores (&gt;25) in both directions are colored red. &lt;strong&gt;(C)&lt;/strong&gt; Mapping Z-scores from one of the two mappings (Deprez -&gt; Durante). &lt;strong&gt;(D)&lt;/strong&gt; The set of cell type correspondences with high bi-directional mappings (both Z-scores &gt; 25; equivalent to the red dots in &lt;strong&gt;B&lt;/strong&gt;). &lt;strong&gt;(E)&lt;/strong&gt; Gene expression in olfactory microvillar cells from the Durante dataset. These cells express classical microvillar genes (ASCL3) as well as marker genes of pulmonary ionocytes (CFTR, FOXI1). &lt;strong&gt;(F)&lt;/strong&gt; Top: average expression for each gene in three cell types compared between the Durante and Deprez datasets in the units of the original papers. Bottom: average expression after a global non-linear transformation of the Durante data. Basal/suprabasal cells = &#x201C;respiratory HBCs&#x201D; from Durante et al., and &#x201C;basal&#x201D; and &#x201C;suprabasal&#x201D; cells from Deprez et al. Secretory cells = &#x201C;early respiratory secretory cells&#x201D; and &#x201C;respiratory secretory cells&#x201D; from Durante et al., and &#x201C;secretory&#x201D; cells from Deprez et al. Mutliciliated cells = &#x201C;respiratory ciliated cells&#x201D; from Durante et al., and &#x201C;multiciliated&#x201D; cells from Deprez et al. &lt;strong&gt;(G)&lt;/strong&gt; Average expression of genes shown in Fig 3G before (left) and after (right) gene-specific rescaling. Each color is one gene. For every color, three dots are shown, corresponding to expression across the three cell types in &lt;strong&gt;F&lt;/strong&gt;. The diagonal line represents unity. Dots close to the diagonal line indicate a close match in transformed gene expression the Durante and Deprez data for the cell types shown in &lt;strong&gt;F&lt;/strong&gt;. &lt;strong&gt;(H)&lt;/strong&gt; Comparison of percent squared error before versus after gene-specific rescaling.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F8.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F8.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S4.</span> <span class="caption-title">Related to <a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3</a>. Comparing cell types and expression levels across respiratory and olfactory epithelial datasets.</span><p id="p-99" class="first-child"><strong>(A)</strong> Schematic of the mapping strategy used to identify similar cell types across datasets, applied to a toy example. Each cell type from “Dataset 1” dataset is mapped to cell types from the “Dataset 2”. From left to right: Each Dataset 1 cell voted on its 5 most similar cells in Dataset 2; the total number of votes cast for each Dataset 2 cell type was quantified; and vote totals were Z-scored against 1000 shuffles where cell type labels were permutated. <strong>(B)</strong> Mapping was performed bi-directionally between the Deprez and Durante datasets, and the mapping Z-scores in each direction are compared. The set of cell type correspondences with high Z-scores (&gt;25) in both directions are colored red. <strong>(C)</strong> Mapping Z-scores from one of the two mappings (Deprez -&gt; Durante). <strong>(D)</strong> The set of cell type correspondences with high bi-directional mappings (both Z-scores &gt; 25; equivalent to the red dots in <strong>B</strong>). <strong>(E)</strong> Gene expression in olfactory microvillar cells from the Durante dataset. These cells express classical microvillar genes (ASCL3) as well as marker genes of pulmonary ionocytes (CFTR, FOXI1). <strong>(F)</strong> Top: average expression for each gene in three cell types compared between the Durante and Deprez datasets in the units of the original papers. Bottom: average expression after a global non-linear transformation of the Durante data. Basal/suprabasal cells = “respiratory HBCs” from Durante et al., and “basal” and “suprabasal” cells from Deprez et al. Secretory cells = “early respiratory secretory cells” and “respiratory secretory cells” from Durante et al., and “secretory” cells from Deprez et al. Mutliciliated cells = “respiratory ciliated cells” from Durante et al., and “multiciliated” cells from Deprez et al. <strong>(G)</strong> Average expression of genes shown in <a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig 3G</a> before (left) and after (right) gene-specific rescaling. Each color is one gene. For every color, three dots are shown, corresponding to expression across the three cell types in <strong>F</strong>. The diagonal line represents unity. Dots close to the diagonal line indicate a close match in transformed gene expression the Durante and Deprez data for the cell types shown in <strong>F</strong>. <strong>(H)</strong> Comparison of percent squared error before versus after gene-specific rescaling.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Given that the nasopharynx is a major site of infection for CoV-2 (<em><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></em>), we compared the frequency of ACE2 and TMPRSS2 expression among the cell types in the human RE and OE (<em><a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a></em>). Sustentacular cells exhibited the highest frequency of ACE2 expression in the OE (2.90% of cells) although this frequency was slightly lower than that observed in respiratory ciliated and secretory cells (3.65% and 3.96%, respectively). Although all HBC subtypes expressed ACE2, the frequency of expression of ACE2 was lower in olfactory HBCs (0.84% of cells) compared to respiratory HBCs (1.78% of cells) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). In addition, all other RE cell subtypes showed higher frequencies of ACE2 and TMPRSS2 expression than was apparent in OE cells.</p><p id="p-26">These results demonstrate the presence of key CoV-2 entry-related genes in specific cell types in the OE, but at lower levels of expression than in RE isolated from the nasal mucosa. We wondered whether these lower levels of expression might nonetheless be sufficient for infection of CoV-2. It was recently reported that nasal RE has higher expression of CoV-2 entry genes than RE of the trachea or lungs (<em><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></em>), and we therefore asked where OE fell within this previously established spectrum of expression. To address this question, we developed a two step alignment procedure in which we first sought to identify cell types that were common across the OE and RE, and then leveraged gene expression patterns in these common cell types to normalize gene expression levels across all cell types in the OE and RE (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S4</a>). This approach revealed a correspondences between goblet cells in the RE and Bowman’s gland cells in the OE (96% mapping probability, see Methods), and between pulmonary ionocytes in the RE and a subset of microvillar cells in the OE (99% mapping probability, see Methods); after alignment, human OE sustentacular cells were found to express ACE2 and TMPRSS2 at levels similar to that observed in the remainder of the non-nasal respiratory tract (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3G</a>) (<em><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></em>).</p><p id="p-27">Given the potential for the RE and OE in the nasal cavity to be directly infected with CoV-2, we assessed the expression of Ace2 and other CoV entry genes in the mouse olfactory bulb (OB), which is directly connected to the OSNs in OE via cranial nerve I (CN I); in principle, alterations in bulb function could cause anosmia independently of functional changes in the OE. To do so, we performed scSeq (using DropSeq, see Methods) on the mouse OB, and merged these data with a previously published OB scSeq analysis, yielding a dataset with nearly 50,000 single cells (see Methods) (<em><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></em>). This analysis revealed that Ace2 expression was absent from OB neurons and instead was observed only in vascular pericytes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4A-D</a> and <a id="xref-fig-9-1" class="xref-fig" href="#F9">S5</a>–<a id="xref-fig-10-1" class="xref-fig" href="#F10">6</a>), which are involved in blood pressure regulation, maintenance of the blood-brain barrier, and inflammatory responses (<em><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></em>). Although other potential CoV proteases were expressed in the OB, Tmprss2 was not expressed in the olfactory bulb. We also performed Smart-seq2-based deep sequencing of single OB dopaminergic neurons (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4E</a> and <a id="xref-fig-11-1" class="xref-fig" href="#F11">S7</a>, see Methods), which confirmed the absence of Ace2 and Tmprss2 expression in that cell type. In addition, re-analysis of 10 deeply sequenced datasets from different regions of the nervous system demonstrated that Ace2 expression is absent from neurons, consistent with prior immunostaining results (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure S8</a>)(<em><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></em>). Together these findings suggest that OB neurons are likely not a primary site of infection, but that vascular pericytes may be sensitive to CoV-2 infection in the OB and brain.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 4. Whole OB scSeq data integration and clustering. (A) UMAP visualizations of olfactory bulb cells from Zeisel et al. (45)(left) and a new DropSeq dataset (right). (B) Combined dataset UMAP visualization of clusters of OB cells (left), and table showing number of corresponding cells from each original dataset, cell classes and subtypes (right). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells; AON, anterior olfactory nucleus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 4. Whole OB scSeq data integration and clustering.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; UMAP visualizations of olfactory bulb cells from Zeisel et al. (&lt;em&gt;45&lt;/em&gt;)(left) and a new DropSeq dataset (right). &lt;strong&gt;(B)&lt;/strong&gt; Combined dataset UMAP visualization of clusters of OB cells (left), and table showing number of corresponding cells from each original dataset, cell classes and subtypes (right). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells; AON, anterior olfactory nucleus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F9.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F9.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S5.</span> <span class="caption-title">Related to <a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4</a>. Whole OB scSeq data integration and clustering.</span><p id="p-100" class="first-child"><strong>(A)</strong> UMAP visualizations of olfactory bulb cells from Zeisel et al. (<em><a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">45</a></em>)(left) and a new DropSeq dataset (right). <strong>(B)</strong> Combined dataset UMAP visualization of clusters of OB cells (left), and table showing number of corresponding cells from each original dataset, cell classes and subtypes (right). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells; AON, anterior olfactory nucleus.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 4. Expression of cell type markers in whole OB scSeq data. (A) UMAP visualisations showing expression of markers for neurons (Snap25), inhibitory neurons (Gad1), mitral and tufted cells (Eomes), excitatory neurons from the anterior olfactory nucleus (AON; Htr2c), glomerular layer calretinin positive neurons (Calb2), glomerular layer dopaminergic neurons (Th), vasoactive intestinal peptide (VIP) positive neurons (Vip), immature neurons (Dcx), intermediate progenitor cells (IPCs; Top2a), oligodendrocytes (Mog), oligodendrocyte precursor cells (OPCs; Olig2), astrocytes (Mlc1), olfactory ensheathing cells (OECs; Frzb), perivascular macrophages (Mrc1), microglia (Itgam), vascular cells (Foxc1), pericytes (Kcnj8) and vascular leptomeningeal cells (Dcn). (B) Dot plot showing expression of cell class and subtype markers alongside genes coding for coronavirus entry proteins in whole OB scSeq data. Circle size denotes the percentage of cells in each class expressing the gene; for plotting, minimum expression was set to 0.01% of cells per cluster. Color scale shows mean scaled expression level per cluster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 4. Expression of cell type markers in whole OB scSeq data.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; UMAP visualisations showing expression of markers for neurons (Snap25), inhibitory neurons (Gad1), mitral and tufted cells (Eomes), excitatory neurons from the anterior olfactory nucleus (AON; Htr2c), glomerular layer calretinin positive neurons (Calb2), glomerular layer dopaminergic neurons (Th), vasoactive intestinal peptide (VIP) positive neurons (Vip), immature neurons (Dcx), intermediate progenitor cells (IPCs; Top2a), oligodendrocytes (Mog), oligodendrocyte precursor cells (OPCs; Olig2), astrocytes (Mlc1), olfactory ensheathing cells (OECs; Frzb), perivascular macrophages (Mrc1), microglia (Itgam), vascular cells (Foxc1), pericytes (Kcnj8) and vascular leptomeningeal cells (Dcn). &lt;strong&gt;(B)&lt;/strong&gt; Dot plot showing expression of cell class and subtype markers alongside genes coding for coronavirus entry proteins in whole OB scSeq data. Circle size denotes the percentage of cells in each class expressing the gene; for plotting, minimum expression was set to 0.01% of cells per cluster. Color scale shows mean scaled expression level per cluster.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F10.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F10.medium.gif" width="374" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S6.</span> <span class="caption-title">Related to <a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4</a>. Expression of cell type markers in whole OB scSeq data.</span><p id="p-101" class="first-child"><strong>(A)</strong> UMAP visualisations showing expression of markers for neurons (Snap25), inhibitory neurons (Gad1), mitral and tufted cells (Eomes), excitatory neurons from the anterior olfactory nucleus (AON; Htr2c), glomerular layer calretinin positive neurons (Calb2), glomerular layer dopaminergic neurons (Th), vasoactive intestinal peptide (VIP) positive neurons (Vip), immature neurons (Dcx), intermediate progenitor cells (IPCs; Top2a), oligodendrocytes (Mog), oligodendrocyte precursor cells (OPCs; Olig2), astrocytes (Mlc1), olfactory ensheathing cells (OECs; Frzb), perivascular macrophages (Mrc1), microglia (Itgam), vascular cells (Foxc1), pericytes (Kcnj8) and vascular leptomeningeal cells (Dcn). <strong>(B)</strong> Dot plot showing expression of cell class and subtype markers alongside genes coding for coronavirus entry proteins in whole OB scSeq data. Circle size denotes the percentage of cells in each class expressing the gene; for plotting, minimum expression was set to 0.01% of cells per cluster. Color scale shows mean scaled expression level per cluster.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 4. Sample quality control of manually sorted OB dopaminergic neurons. (A) Sample quality control metrics showing total number of reads per cell (left), total genes per cell (centre-left), percentage of reads per cell mapping to ERCC spike-ins (centre-right) and percentage of reads per cell mapping to mitochondrial genes (right). (B) Per cell library complexity depicted as the cumulative proportion of detected genes. (C) UMAP visualization of manually sorted DA neurons processed on three separate batches of library preparation and sequencing, showing efficient correction of technical batch effects with the scTransform function in Seurat (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 4. Sample quality control of manually sorted OB dopaminergic neurons.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Sample quality control metrics showing total number of reads per cell (left), total genes per cell (centre-left), percentage of reads per cell mapping to ERCC spike-ins (centre-right) and percentage of reads per cell mapping to mitochondrial genes (right). &lt;strong&gt;(B)&lt;/strong&gt; Per cell library complexity depicted as the cumulative proportion of detected genes. &lt;strong&gt;(C)&lt;/strong&gt; UMAP visualization of manually sorted DA neurons processed on three separate batches of library preparation and sequencing, showing efficient correction of technical batch effects with the scTransform function in Seurat (see Methods).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F11.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F11.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S7.</span> <span class="caption-title">Related to <a id="xref-fig-4-11" class="xref-fig" href="#F4">Fig. 4</a>. Sample quality control of manually sorted OB dopaminergic neurons.</span><p id="p-102" class="first-child"><strong>(A)</strong> Sample quality control metrics showing total number of reads per cell (left), total genes per cell (centre-left), percentage of reads per cell mapping to ERCC spike-ins (centre-right) and percentage of reads per cell mapping to mitochondrial genes (right). <strong>(B)</strong> Per cell library complexity depicted as the cumulative proportion of detected genes. <strong>(C)</strong> UMAP visualization of manually sorted DA neurons processed on three separate batches of library preparation and sequencing, showing efficient correction of technical batch effects with the scTransform function in Seurat (see Methods).</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Related to Fig. 4. Expression of genes coding for CoV-2 entry-related proteins in neurons forming part of deeply sequenced scSeq datasets from various brain regions and sensory systems. Violin plots show expression of Ace2 and Tmprss2, alongside Snap25 for reference. Expression values are log2-normalized counts of the number of transcripts for each gene. Each dot represents a single cell." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Related to Fig. 4. Expression of genes coding for CoV-2 entry-related proteins in neurons forming part of deeply sequenced scSeq datasets from various brain regions and sensory systems.&lt;/span&gt; Violin plots show expression of Ace2 and Tmprss2, alongside Snap25 for reference. Expression values are log&lt;sub&gt;2&lt;/sub&gt;-normalized counts of the number of transcripts for each gene. Each dot represents a single cell.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F12.medium.gif" width="440" height="123"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S8." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F12.medium.gif" width="440" height="123"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S8.</span> <span class="caption-title">Related to <a id="xref-fig-4-12" class="xref-fig" href="#F4">Fig. 4</a>. Expression of genes coding for CoV-2 entry-related proteins in neurons forming part of deeply sequenced scSeq datasets from various brain regions and sensory systems.</span><p id="p-103" class="first-child">Violin plots show expression of Ace2 and Tmprss2, alongside Snap25 for reference. Expression values are log<sub>2</sub>-normalized counts of the number of transcripts for each gene. Each dot represents a single cell.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of coronavirus entry genes in mouse olfactory bulb. (A) UMAP visualization of OB scSeq highlighting the main cell classes and subtypes, as observed in two integrated scSeq datasets (see Methods). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells. Cluster information is summarized in Figures S5&#x2013;6. (B) Heatmap showing expression of cell class markers and genes coding for coronavirus entry proteins in mouse olfactory bulb. Color scale shows scaled mean expression level per class, normalized by their maximum expression across cell types. (C) UMAP visualization of the vascular cell cluster showing expression of CoV-2 entry genes (Ace2, left; Tmprss2, center) and Kcnj8, a pericyte marker. Bottom insets, detailed view of the vascular cell cluster containing Ace2-expressing pericytes. Color scale, log-normalized UMI counts. (D) Percent of cells expressing ACE2. &#x201C;Other vascular&#x201D; denotes all vascular cells excluding pericytes. (E) Violin plots showing Log2-normalized expression (Log2(TPM+1)) of coronavirus entry genes and dopaminergic neuron markers in manually sorted and deeply-sequenced single olfactory bulb dopaminergic neurons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-930029797" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Expression of coronavirus entry genes in mouse olfactory bulb.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; UMAP visualization of OB scSeq highlighting the main cell classes and subtypes, as observed in two integrated scSeq datasets (see Methods). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells. Cluster information is summarized in Figures S5&#x2013;6. &lt;strong&gt;(B)&lt;/strong&gt; Heatmap showing expression of cell class markers and genes coding for coronavirus entry proteins in mouse olfactory bulb. Color scale shows scaled mean expression level per class, normalized by their maximum expression across cell types. &lt;strong&gt;(C)&lt;/strong&gt; UMAP visualization of the vascular cell cluster showing expression of CoV-2 entry genes (Ace2, left; Tmprss2, center) and Kcnj8, a pericyte marker. Bottom insets, detailed view of the vascular cell cluster containing Ace2-expressing pericytes. Color scale, log-normalized UMI counts. &lt;strong&gt;(D)&lt;/strong&gt; Percent of cells expressing ACE2. &#x201C;Other vascular&#x201D; denotes all vascular cells excluding pericytes. &lt;strong&gt;(E)&lt;/strong&gt; Violin plots showing Log&lt;sub&gt;2&lt;/sub&gt;-normalized expression (Log&lt;sub&gt;2&lt;/sub&gt;(TPM+1)) of coronavirus entry genes and dopaminergic neuron markers in manually sorted and deeply-sequenced single olfactory bulb dopaminergic neurons.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F4.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F4.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.25.009084/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Expression of coronavirus entry genes in mouse olfactory bulb.</span><p id="p-28" class="first-child"><strong>(A)</strong> UMAP visualization of OB scSeq highlighting the main cell classes and subtypes, as observed in two integrated scSeq datasets (see Methods). VIP, vasoactive intestinal peptide positive neurons; ETCs, external tufted cells; OPCs, oligodendrocyte precursor cells; IPCs, intermediate precursor cells; OECs, olfactory ensheathing cells. Cluster information is summarized in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figures S5</a>–<a id="xref-fig-10-2" class="xref-fig" href="#F10">6</a>. <strong>(B)</strong> Heatmap showing expression of cell class markers and genes coding for coronavirus entry proteins in mouse olfactory bulb. Color scale shows scaled mean expression level per class, normalized by their maximum expression across cell types. <strong>(C)</strong> UMAP visualization of the vascular cell cluster showing expression of CoV-2 entry genes (Ace2, left; Tmprss2, center) and Kcnj8, a pericyte marker. Bottom insets, detailed view of the vascular cell cluster containing Ace2-expressing pericytes. Color scale, log-normalized UMI counts. <strong>(D)</strong> Percent of cells expressing ACE2. “Other vascular” denotes all vascular cells excluding pericytes. <strong>(E)</strong> Violin plots showing Log<sub>2</sub>-normalized expression (Log<sub>2</sub>(TPM+1)) of coronavirus entry genes and dopaminergic neuron markers in manually sorted and deeply-sequenced single olfactory bulb dopaminergic neurons.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-4"><h2 class="">Discussion</h2><p id="p-29">Here we demonstrate that subsets of OE sustentacular cells, HBCs, microvillar cells and Bowman’s gland cells express the CoV-2 receptor ACE2 and the spike protein protease TMPRSS2. Human OE sustentacular cells express these genes at levels comparable to that observed in lung cells. However, we failed to detect ACE2 expression in mature OSNs at either the transcript or protein levels. These observations suggest that CoV-2 does not directly enter OSNs, but instead may target OE support and stem cells. Similarly, neurons in the olfactory bulb fail to express ACE2, whereas vascular pericytes do. Thus primary infection of non-neuronal cell types — rather than sensory or bulb neurons — may be responsible for anosmia and related disturbances in odor perception in COVID-19 patients.</p><p id="p-30">The identification of non-neuronal cell types in the OE and bulb susceptible to CoV-2 infection suggests three possible, non-mutually-exclusive mechanisms for the acute loss of smell reported in COVID-19 patients. First, local infection of support cells and vascular pericytes in the nose and bulb could cause significant inflammatory responses whose downstream effects could alter the function of OSNs or bulb neurons. Second, damage to support cells (which are responsible for local water balance and ionic balance) could indirectly influence signaling from OSNs to the brain; our finding that a subset of microvillar cells are homologues of pulmonary ionocytes — which play a key role in maintaining extracellular chloride gradients in the lung — is consistent with this model (<em><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></em>). Finally, damage to sustentacular cells and Bowman’s gland cells in mouse models leads to diffuse architectural damage to the entire OE, which could abrogate smell perception (<em><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></em>).</p><p id="p-31">Our data suggest that only sparse subsets of support cells and pericytes express ACE2 (although this may reflect undersampling of transcripts by scSeq). Intriguingly, exposure of the rat OE to the SDAV coronavirus causes infection of only a limited number of sustentacular cells, yet over time OSNs become denuded of cilia (presumably foreclosing the possibility of odor transduction) and the global architecture of the OE becomes disrupted (<em><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a></em>). These observations suggest that coronavirus infection of subsets of sustentacular cells may be sufficient to cause a pathophysiological cascade that culminates in damage to OSNs and defects in olfactory function.</p><p id="p-32">The natural history of CoV2-induced anosmia is only now being defined; while recovery of smell has been reported, it remains unclear whether in a subset of patients smell disturbances will be long-lasting or permanent (<em><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></em>). It is possible that infection of stem cells including both resting and activated HBCs — which express ACE2 and TMPRSS2 at lower levels than RE or sustentacular cells — could result in long-lasting olfactory dysfunction. The survival time of mature OSNs varies based upon many factors, but on average in the mouse such neurons survive weeks to short months, rather than days or years (<em><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></em>). Although HBCs in mice appear largely quiescent, recent scSeq data suggests that ongoing neurogenesis may play a larger role than previously appreciated in the human OE (<em><a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">44</a></em>). It is therefore possible that the dual challenge of acute loss of sustentacular cells, together with the inability to effectively renew the OE over time, may cause persistent anosmia.</p><p id="p-33">Many viruses, including coronaviruses, have been shown to propagate from the nasal epithelium past the cribriform plate to infect the olfactory bulb; this form of central infection has been suggested to mediate olfactory deficits, even in the absence of lasting OE damage (<em><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>–<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></em>). The rodent coronavirus MHV passes from the nose to the bulb, even though rodent OSNs do not express CEACAM1, the main MHV receptor (<em><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></em>) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figures S1A/C</a> and <a id="xref-fig-6-3" class="xref-fig" href="#F6">S2A</a>), suggesting that CoVs in the nasal mucosa can reach the brain through mechanisms independent of axonal transport by sensory nerves; in this case, OB neurons express CEACAM1 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4E</a>), which likely supports the ability of MHV to target these neurons and change odor perception. One speculative possibility is that local seeding of the OE with CoV-2-infected cells can result in OSN-independent transfer of virions from the nose to the bulb, perhaps via the vascular supply shared between the OB and the OSN axons that comprise CN I. Although CN I was not directly queried in our datasets, it is reasonable to infer that vascular pericytes in CN I also express ACE2, which suggests a possible route of entry for CoV-2 from the nose into the brain. Given the absence of ACE2 in OB neurons, we speculate that any central olfactory dysfunction in COVID-19 is the secondary consequence of pericyte-mediated vascular inflammation (<em><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a></em>).</p><p id="p-34">We note several caveats that temper our conclusions. Here, we assume that ACE2 protein levels follow the pattern revealed by analysis of mRNA; while our staining in human OE is consistent with this idea, it remains to be rigorously assessed in all cell types in the OE and bulb. Although current data suggest that ACE2 is the most likely receptor for CoV-2 in vivo, it is possible (although it has not yet been demonstrated) that other molecules such as BSG may enable CoV-2 entry independently of ACE2 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figures 3E, 3G</a>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">S1</a>, <a id="xref-fig-6-4" class="xref-fig" href="#F6">S2</a>) (<em><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></em>). We also propose that damage to the olfactory system is either due to primary infection or secondary inflammation; it is possible (although has not yet been demonstrated) that cells infected with CoV-2 can form syncytia with cells that do not express ACE2. Such a mechanism could damage neurons adjacent to infected cells. Definitive identification of the pathophysiological mechanisms underlying COVID-19-mediated anosmia will require additional research. Nonetheless, our identification of cells in the OE and OB expressing molecules known to be involved in CoV-2 entry illuminates a path forward for future studies.</p></div><div class="section" id="sec-5"><h2 class="">Materials and Methods</h2><div id="sec-6" class="subsection"><h3>Mouse bulk RNA-Seq datasets</h3><p id="p-35">Normalized gene expression tables were obtained from previous published datasets (<em><a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a></em>). For the mouse data sets, the means of the replicates from WOM or OSN were used to calculate Log<sub>2</sub> fold changes. For the mouse data from Saraiva et al. and the primate data sets (<em><a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a></em>), the normalized counts of the genes of interest from individual replicates were plotted. Below is a table with detailed sample information.</p><p id="p-36">Sample information for the bulk RNA-seq data analyzed in this study</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242056/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242056/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div></div><div id="sec-7" class="subsection"><h3>WOM and HBC lineage tracing mouse 10x scSeq experiments</h3><div id="sec-8" class="subsection"><h4>Mice</h4><p id="p-37">2 month-old and 18 month-old wild type C57BL/6J mice were obtained from the National Institute on Aging Aged Rodent Colony and used for the WOM experiments; each experimental condition consisted of one male and one female mouse to aid doublet detection. Mice containing the transgenic Krt5-CreER(T2) driver (<em><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></em>) and Rosa26-YFP reporter allele (<em><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></em>) were used for the HBC lineage tracing dataset. All mice were assumed to be of normal immune status. Animals were housed and maintained according to IACUC guidelines.</p></div><div id="sec-9" class="subsection"><h4>Single-Cell RNA Sequencing</h4><p id="p-38">The olfactory epithelium was surgically removed, and the dorsal, sensory portion was dissected and dissociated, as previously described (<em><a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a></em>). For WOM experiments, dissociated cells were subjected to fluorescence-activated cell sorting (FACS) using propidium iodide to identify and select against dead or dying cells; 100,000 cells/sample were collected in 10% FBS. For the HBC lineage tracing experiments Krt5-CreER; Rosa26YFP/YFP mice were injected once with tamoxifen (0.25 mg tamoxifen/g body weight) at P21-23 days of age and sacrificed at 24 hours, 48 hours, 96 hours, 7 days and 14 days post-injury, as previously described (<em><a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></em>). For each experimental time point, YFP+ cells were isolated by FACS based on YFP expression and negative for propidium iodide, a vital dye.</p><p id="p-39">Cells isolated by FACS were subjected to single-cell RNA-seq. Three replicates (defined here as a FACS collection run) per age were analyzed for the WOM experiment; at least two biological replicates were collected for each experimental condition for the HBC lineage tracing experiment. Single cell cDNA libraries from the isolated cells were prepared using the Chromium Single Cell 3’ System according to the manufacturer’s instructions. The WOM preparation employed v3 chemistry with the following modification: the cell suspension was directly added to the reverse transcription master mix, along with the appropriate volume of water to achieve the approximate cell capture target. The HBC lineage tracing experiments were performed using v2 chemistry. The 0.04% weight/volume BSA washing step was omitted to minimize cell loss. Completed libraries were sequenced on Illumina HiSeq4000 to produce paired-end 100nt reads.</p><p id="p-40">Sequence data were processed with the 10X Genomics Cell Ranger pipeline (2.0.0 for v2 chemistry), resulting in the initial starting number before filtering of 60,408 WOM cells and 25,469 HBC lineage traced cells. The scone R/Bioconductor package was used to filter out lowly-expressed genes (fewer than 2 UMI’s in fewer than 5 cells) and low-quality libraries (using the metric_sample_filter function with arguments hard_nreads = 2000, zcut = 4).</p></div><div id="sec-10" class="subsection"><h4>Preliminary Filtering</h4><p id="p-41">Cells with co-expression of male (Ddx3y, Eif2s3y, Kdm5d, and Uty) and female marker genes (Xist) were removed as potential doublets from the WOM dataset. For both datasets, doublet cell detection was performed per sample using DoubletFinder and Scrublet (<em><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></em>). Genes with at least 3 UMIs in at least 5 cells were used for downstream clustering and cell type identification. For the HBC lineage tracing dataset, the Bioconductor package scone was used to pick the top normalization (“none,fq,ruv_k=1,no_bio,batch”), corresponding to full quantile normalization, batch correction and removing one factor of unwanted variation using RUV (<em><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></em>). A range of cluster labels were created by clustering using the partitioning around medoids (PAM) algorithm and hierarchical clustering in the clusterExperiment Bioconductor package (<em><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></em>), with parameters k0s=(10,13,16,19,22,25) and alpha=(NA,0.1,0.2,0.3). Clusters that did not show differential expression were merged (using the function mergeClusters with arguments mergeMethod = ‘adjP’, cutoff = 0.01, and DEMethod = ‘limma’ for the lineage-traced dataset). Initial clustering identified one Macrophage (Msr1+) cluster consisting of 252 cells; upon its removal and restarting from the normalization step a subsequent set of 15 clusters was obtained. These clusters were used to filter out 1515 cells for which no stable clustering could be found (i.e., ‘unassigned’ cells), and four clusters respectively consisting of 31, 29 and 23 and 305 cells. Doublets were identified using DoubletFinder and 271 putative doublets were removed. Inspection of the data in a three-dimensional UMAP embedding identified two groups of cells whose experimentally sampled timepoint did not match their position along the HBC differentiation trajectory, and these additional 219 cells were also removed from subsequent analyses.</p></div><div id="sec-11" class="subsection"><h4>Analysis of CoV-related genes in WOM and HBC lineage datasets</h4><p id="p-42">Analysis of WOM scSeq data were performed in python using the open-source Scanpy software starting from the raw UMI count matrix of the 40179 cells passing the initial filtering and QC criteria described above. UMIs were total-count normalized and scaled by 10,000 (TPT, tag per ten-thousands) and then log-normalized. For each gene, the residuals from linear regression models using the total number of UMIs per cell as predictors were then scaled via z-scoring. PCA was then performed on a set of highly-variable genes (excluding OR genes) calculated using the “highly_variable_genes” function with parameters: min_mean=0.01, max_mean=10, min_disp=0.5. A batch corrected neighborhood graph was constructed by the “bbknn” function with 42 PCs with the parameters: local_connectivity=1.5, and embedding two-dimensions using the UMAP function with default parameters (min_dist = 0.5). Cells were clustered using the neighborhood graph via the Leiden algorithm (resolution = 1.2). Identified clusters were manually merged and annotated based on known marker gene expression. We removed 281 cells containing mixtures of marker genes with no clear gene expression signature. The identified cell types and the number of each of the remaining 39898 cells detected were as follows. 28,769 mOSN: mature OSN; 2,607 iOSN: immature OSN; 859 INP: Immediate Neural Precursor; 623 GBC: Globose Basal Cell; HBC: Horizontal Basal Cell (1,083 Olfactory and 626 Respiratory); 480 SUS: sustentacular cell; 331 BG: Bowman’s gland; MV: Microvillar cell (563 Brush-like and 1,530 Ionocyte-like); 92 OEC: Olfactory Ensheathing Cell; 76 Resp. Secretory cells; 227 Resp. unspecified cells; 172 atypical OSN; 1,757 various immune cells, 103 RBC: Red Blood Cell. TPT gene expression levels were visualized in two-dimensional UMAP plots.</p><p id="p-43">The filtered HBC lineage dataset containing 21722 cells was analyzing in python and processed for visualization using pipelines in SPRING and Scanpy (<em><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></em>). In brief, total counts were normalized to the median total counts for each cell and highly variable genes were selected using the SPRING gene filtering function (“filter_genes”) using parameters (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>). The dimensionality of the data was reduced to 20 using principal components analysis (PCA) and visualized in two-dimensions using the UMAP method with parameters (n_neighbors=20, min_dist=0.5). Clustering was performed using the Leiden algorithm (resolution=1.45) and clusters were merged manually using known marker genes. The identified cell types and number of each type were: 929 mOSN: mature OSN; 2073 iOSN: immature OSN; 786 INP: Immediate Neural Precursor; 755 GBC: Globose Basal Cell; HBC: Horizontal Basal Cell (7782 Olfactory, 5418 Regenerating, and 964 Respiratory); 2666 SUS: sustentacular cell; and 176 Ionocyte-like Microvillar (MV) cell.</p><p id="p-44">Expression of candidate CoV-2-related genes was defined if at least one transcript (UMI) was detected in that cell, and the percent of cells expressing candidate genes was calculated for each cell type. In the WOM dataset Ace2 was only detected in 2 out of 28,769 mature OSNs (0.007 %), and in the HBC lineage dataset, Ace2 was not detected in any OSNs. Furthermore, Ace2 was not detected in immature sensory neurons (GBCs, INPs, or iOSNs) in either dataset.</p></div></div><div id="sec-12" class="subsection"><h3>Mouse HBC lineage Smart-Seq2 dataset</h3><p id="p-45">Single-cell RNA-seq data from HBC-derived cells from Fletcher et al. and Gadye et al (<em><a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a></em>), labeled via <em>Krt5-CreER</em> driver mice, were downloaded from GEO at accession GSE99251 using the file “GSE95601_oeHBCdiff_Cufflinks_eSet_counts_table.txt.gz”. Processing was performed as described above, including total counts normalization and filtering for highly variable genes using the SPRING gene filtering function “filter_genes” with parameters (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). The resulting data were visualized in SPRING and a subset of cells were removed for quality control, including a cluster of cells with low total counts and another with predominantly reads from ERCC spike-in controls. Putative doublets were also identified using Scrublet and removed (6% of cells) (<em><a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a></em>). The resulting data were visualized in SPRING and partitioned using Louvain clustering on the SPRING k-nearest-neighbor graph using the top 40 principal components. Cell type annotation was performed manually using the same set of markers genes listed above. Three clusters were removed for quality control, including one with low total counts and one with predominantly reads from ERCC spike-in controls (likely background), and one with high mitochondrial counts (likely stressed cells). For visualization, and clustering the remaining cells were projected to 15 dimensions using PCA and visualized with UMAP with parameters (n_neighbors=15, min_dist=0.4, alpha=0.5, maxiter=500). Clustering was performed using the Leiden algorithm (resolution=0.4) and cell types were manually annotated using known marker genes.</p><p id="p-46">The filtered dataset of mouse HBC-derived cells contained 1450 cells. The percent of cells expressing each marker gene was calculated as described above. Of the 51 OSNs identified, none of them expressed Ace2, and only 1 out of 194 INPs and iOSNs expressed Ace2. In contrast, Ace2 and Tmprss2 were both detected in HBCs and SUS cells.</p></div><div id="sec-13" class="subsection"><h3>Human nasal scSeq dataset</h3><p id="p-47">Human scSeq data from Durante et al. (<em><a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">44</a></em>) was downloaded from the GEO at accession GSE139522. 10X Genomics mtx files were filtered to remove any cells with fewer than 500 total counts. Additional preprocessing was performed as described above, including total counts normalization and filtering for highly variable genes using the SPRING gene filtering function “filter_genes” with parameters (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">90</a>, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). The resulting data were visualized in SPRING and partitioned using Louvain clustering on the SPRING k-nearest-neighbor graph. Four clusters were removed for quality control, including two with low total counts (likely background) and two with high mitochondrial counts (likely stressed or dying cells). Putative doublets were also identified using Scrublet and removed (7% of cells). The remaining cells were projected to 40 dimensions using PCA. PCA-batch-correction was performed using Patient 4 as a reference, as previously described (<em><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></em>). The filtered data were then re-partitioned using Louvain clustering on the SPRING graph and each cluster was annotated using known marker genes, as described in (<em><a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">44</a></em>). For example, immature and mature OSNs were identified via their expression of GNG8 and GNG13, respectively. HBCs were identified via the expression of KRT5 and TP63 and olfactory HBCs were distinguished from respiratory HBCs via the expression of CXCL14 and MEG3. Identification of SUS cells (CYP2A13, CYP2J2), Bowman’s gland (SOX9, GPX3), and MV ionocytes-like cells (ASCL3, CFTR, FOXI1) was also performed using known marker genes. For visualization, the top 40 principal components were reduced to two dimensions using UMAP with parameters (n_neighbors=15, min_dist=0.4).</p><p id="p-48">The filtered human scSeq dataset contained 33358 cells. Each of the samples contained cells from both the olfactory and respiratory epithelium, although the frequency of OSNs and respiratory cells varied across patients, as previously described (<em><a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">44</a></em>). 295 cells expressed ACE2 and 4953 cells expressed TMPRSS2. Of the 865 identified OSNs, including both immature and mature cells, none of the cells express ACE2 and only 2 (0.23%) expressed TMPRSS2. In contrast, ACE2 was reliably detected in at least 2% and TMPRSS2 was expressed in close to 50% of multiple respiratory epithelial subtypes. The expression of both known cell type markers and known CoV-related genes was also examined across respiratory and olfactory epithelial cell types. For these gene sets, the mean expression in each cell type was calculated and normalized by the maximum across cell types.</p></div><div id="sec-14" class="subsection"><h3>Immunohistochemistry</h3><p id="p-49">Human olfactory mucosa biopsies were obtained via IRB-approved protocol at Duke University School of Medicine, from nasal septum or superior turbinate during endoscopic sinus surgery. Tissue was fixed with 4% paraformaldehyde and cryosectioned at 10 m and sections were processed for immunohistochemistry, as previously described (<em><a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">44</a></em>). Primary antibodies were applied overnight at 4 degrees. Primary antibodies included rabbit anti-ACE2 (Abcam, ab15348; RRID:AB_301861) and mouse Tuj1 against neuron-specific tubulin (BioLegend, 801201; RRID:AB_2313773). Biotinylated secondary antibodies (Vector Labs), avidin-biotinylated horseradish peroxidase kit (Vector) followed by fluorescein tyramide signal amplification (Perkin Elmer) were applied per manufacturer’s instructions. For dual staining, Tuj1 was visualized using AlexaFluor 594 Goat anti-Mouse (Jackson ImmunoResearch, Ca# 115-585-146; RRID:AB_2338881). Sections were counterstained with 4’,6-daimidino-2-phenylindole (DAPI) and coverslips were mounted using Prolong Gold (Invitrogen) for imaging, using a Leica DMi8 microscope system. Images were processed using Fiji ImageJ software (NIH). Scale bars were applied directly from the Leica acquisition software metadata in ImageJ Tools. Unsharp Mask was applied in ImageJ, and brightness/contrast was adjusted globally. The Tuj1 antibody was validated, as previously described (<em><a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">44</a></em>). Anti-ACE2 was raised against a C-terminal synthetic peptide for human ACE2, and was validated by the manufacturer to not cross-react with ACE1 for immunohistochemical labeling of ACE2 in fruit bat nasal tissue as well as in human lower airway. Recombinant human ACE2 abolished labeling with this antibody in a previous study in human tissue, further demonstrating its specificity (<em><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></em>).</p></div><div id="sec-15" class="subsection"><h3>Mapping scSeq datasets to each other</h3><p id="p-50">Data from Deprez et al. (<em><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a></em>) were downloaded from the Human Cell Atlas website (<a href="https://www.genomique.eu/cellbrowser/HCA/">https://www.genomique.eu/cellbrowser/HCA/</a>; “Single-cell atlas of the airway epithelium (Grch38 human genome)”). A subset of these data was combined with a subset of the Durante data for mapping between cell types. For the Deprez data, the subset consisted of samples from the nasal RE that belonged to a cell type with &gt;20 cells, including Basal, Cycling Basal, Suprabasal, Secretory, Mucous Multiciliated cells, Multiciliated, SMS Goblet and Ionocyte. We observed two distinct subpopulations of Basal cells, with one of the two populations distinguished by expression of Cxcl14. The cells in this population were manually identified using SPRING and defined for downstream analysis as a separate cell type annotation called “Basal (Cxcl14+)”. For the Durante data, the subset consisted of cells from cell types that had some putative similarity to cells in the Deprez dataset, including Olfactory HBC, Cycling respiratory HBC, Respiratory HBC, Early respiratory secretory cells, Respiratory secretory cells, Sustentacular cells, Bowman’s gland, Olfactory microvillar cells.</p><p id="p-51">To establish a cell type mapping:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-52">Durante (<em><a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">44</a></em>) and Deprez (<em><a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a></em>) data were combined and gene expression values were linearly scaled so that all cells across datasets had the same total counts. PCA was then performed using highly variable genes (n=1477 genes) and PCA-batch-correction (<em><a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">68</a></em>) with the Durante data as a reference set.</p></li><li id="list-item-2"><p id="p-53">Mapping was then performed bidirectionally between the two datasets. Each cell from “Dataset 1” ‘voted’ for the 5 most similar cells in the “Dataset 2”, using distance in PCA space as the measure of similarity. A table <em>T</em> counting votes across cell types was then computed, where for cell type <em>i</em> in the Dataset 1 and cell type <em>j</em> in the Dataset 2,
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.25.009084/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.25.009084/embed/graphic-6.gif"/></noscript></span>

</span></p><p id="p-54">Thus, if Dataset 1 has <em>N</em> cells, then <em>T</em> would count 5*N votes (∑<em>T</em><sub><em>ij</em></sub> = 5<em>N</em>)</p></li><li id="list-item-3"><p id="p-55">The table of votes <em>T</em> was Z-scored against a null distribution, generated by repeating the procedure above 1000 times with shuffled cell type labels.</p></li></ol><p id="p-56">The resulting Z-scores were similar between the two possible mapping directions (Durante -&gt; Deprez vs. Deprez -&gt; Durante; R=0.87 Pearson correlation of mapping Z-scores). The mapping Z-scores were also highly robust upon varying the number of votes-cast per cell (R&gt;0.98 correlation of mapping Z-scores upon changing the vote numbers to 1 or 50 as opposed to 5). Only cell-type correspondences with a high Z-score in both mapping directions (Z-score &gt; 25) were used for downstream analysis.</p><p id="p-57">To establish a common scale of gene expression between datasets, we restricted to cell type correspondences that were supported both by bioinformatic mapping and shared a nominal cell type designation based on marker genes. These included: Basal/suprabasal cells = “respiratory HBCs” from Durante et al., and “basal” and “suprabasal” cells from Deprez et al. Secretory cells = “early respiratory secretory cells” and “respiratory secretory cells” from Durante et al., and “secretory” cells from Deprez et al. Mutliciliated cells = “respiratory ciliated cells” from Durante et al., and “multiciliated” cells from Deprez et al.</p><p id="p-58">We next sought a transformation of the Durante data so that it would agree with the Deprez data within the corresponding cell types identified above To account for differing normalization strategies applied to each dataset prior to download (log normalization and rescaling with cell-specific factors for Deprez et al. but not for Durante et al.), we used the following ansatz for the transformation, where the pseudocount <em>p</em> is a global latent parameter and the rescaling factors <em>f</em><sub><em>i</em></sub> are fit to each gene separately. In the equation below, <em>T</em> denotes the transformation and <em>e</em><sub><em>ij</em></sub> represents a gene expression value for cell <em>i</em> and gene <em>j</em> in the Durante data:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.25.009084/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.25.009084/embed/graphic-7.gif"/></noscript></span>

</span></p><p id="p-59">The parameter <em>p</em> was fit by maximizing the correlation of average gene expression across all genes between each of the cell type correspondences listed above. The rescaling factors <em>f</em><sub><em>i</em></sub> were then fitted separately for each gene by taking the quotient of average gene expression between the Deprez data and the log-transformed Durante data, again across the cell type correspondences above.</p></div><div id="sec-16" class="subsection"><h3>Juvenile and Adult mouse whole olfactory bulb scRNAseq dataset</h3><div id="sec-17" class="subsection"><h4>Juvenile mouse data</h4><p id="p-60">Single-cell RNAseq data from whole mouse olfactory bulb (<em><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></em>) were downloaded from mousebrain.org/loomfiles_level_L1.html in loom format (l1 olfactory.loom) and converted to a Seurat object. Samples were obtained from juvenile mice (age postnatal day 26-29). This dataset comprises 20514 cells passing cell quality filters, excluding 122 cells identified as potential doublets.</p></div><div id="sec-18" class="subsection"><h4>Tissue dissection and single-cell dissociation</h4><p id="p-61">A new dataset of whole olfactory bulb scSeq was generated from adult male mice (8–12 weeks-old). Briefly, dissected olfactory bulbs (including the accessory olfactory bulb and fractions of the anterior olfactory nucleus) were dissociated in 750 µl of dissociation media (DM: HBSS containing 10mM HEPES, 1 mM MgCl2, 33 mM D-glucose) with 28 U/mL Papain and 386 U/mL DNase I (Worthington). Minced tissue pieces were transferred to a 5 mL round-bottom tube (BD). DM was added to a final volume of 3.3 mL and the tissue was mechanically triturated 5 times with a P1000 pipette tip. After 1-hour incubation with rocking at 37°C, the suspension was triturated with a 10 mL pipette 10 times and 2.3 mL was passed through 40 µm cell strainer (BD). The suspension was then mechanically triturated with a P1000 pipette tip 10 times and 800 µL were filtered on the same strainer. The cell suspension was further triturated with a P200 pipette tip 10 times and filtered. 1 mL of Quench buffer (22 mL of DM, 2.5 mL of protease inhibitor prepared by resuspending 1 vial of protease inhibitor with 32 mL of DM, and 2000U of DNase I) was added to the suspension and centrifuged at 300g for 5 min. Cells were resuspended with 3 mL of Quench buffer and overlaid gently on top of 5 mL of protease inhibitor, then spun down at 70g for 10min. The pellet was resuspended using DM supplemented with 0.04 % BSA, and spun down at 300g for 5 min. Cells were suspended in 400 µL of DM with 0.04 % BSA. All mouse husbandry and experiments were performed following institutional and federal guidelines and approved by Harvard Medical School’s Institutional Animal Care and Use Committee (IACUC).</p></div><div id="sec-19" class="subsection"><h4>Drop-seq experiments</h4><p id="p-62">Drop-seq experiments were performed as previously described (<em><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></em>). Microfluidics devices were obtained from FlowJEM and barcode beads were obtained from chemgenes. Two 15 min Drop-seq runs were collected from a single dissociation preparation obtained from 2 mice. Two such dissociations were performed, giving 4 total replicates.</p></div><div id="sec-20" class="subsection"><h4>Sequencing of Drop-seq samples</h4><p id="p-63">4 replicates of Drop-seq samples were pooled and sequenced across 3 runs on an Illumina NextSeq 500 platform. Paired end reads from the fastq files were trimmed, aligned, and tagged via the Drop-seq tools (1-2.0) pipeline, using STAR (2.4.2a) with genomic indices from Ensembl Release 82. The digital gene expression matrix was generated for 8,000 cells per replicate.</p></div><div id="sec-21" class="subsection"><h4>Preprocessing of Drop-seq samples</h4><p id="p-64">Cells with low numbers of genes (500), low numbers of UMIs (700) or high numbers of UMIs (&gt;10000) were removed (6 % of cells). Potential doublets were identified via Scrublet and removed (3.5 % of cells). Overall, this new dataset comprised 27004 cells.</p></div><div id="sec-22" class="subsection"><h4>Integration of whole olfactory bulb scRNAseq datasets</h4><p id="p-65">Raw UMI counts from juvenile and adult whole olfactory bulb samples were integrated in Seurat (<em><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></em>). Briefly, raw counts were log-normalised separately and the 10000 most variable genes identified by variance stabilizing transformation for each dataset. The 4529 variable genes present in both datasets and the first 30 principal components (PCs) were used as features for identifying the integration anchors. The integrated expression matrix was scaled and dimensionality reduced using PCA. Based on their percentage of explained variance, the first 28 PCs were chosen for UMAP visualisation and clustering.</p><p id="p-66">Graph-based clustering was performed using the Louvain algorithm following the standard Seurat workflow. Cluster stability was analysed with Clustree on a range of resolution values (0.4 to 1.4), with 0.6 yielding the most stable set of clusters (<em><a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></em>). Overall, 26 clusters were identified, the smallest of which contained only 43 cells with gene expression patterns consistent with blood cell types, and was excluded from further visualisation plots. Clustering the two datasets separately yielded similar results. Moreover, the distribution of cells from each dataset across clusters was homogenous (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure S5</a>) and the clusters corresponded previous cell class and subtype annotations (<em><a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a></em>). As previously reported, a small cluster of excitatory neurons (cluster 13) contained neurons from the anterior olfactory nucleus. UMAP visualisations of expression level for cell class and cell type markers, and for genes coding for coronavirus entry proteins, depict log-normalized UMI counts. The heatmap in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a> shows the mean expression level for each cell class, normalised to the maximum mean value. The percentage of cells per cell class expressing Ace2 was defined as the percentage of cells with at least one UMI.</p></div></div><div id="sec-23" class="subsection"><h3>Smart-Seq2 sequencing of manually sorted olfactory bulb dopaminergic neurons</h3><div id="sec-24" class="subsection"><h4>Tissue dissociation and manual cell sorting</h4><p id="p-67">Acute olfactory bulb 300 µm slices were obtained from Dat-Cre/Flox-tdTomato (B6.SJL-Slc6a3<sub>tm1.1(cre) Bkmn/J</sub>, Jax stock 006660 / B6.Cg– Gt(ROSA)26Sor<sub>tm9(CAG-tdTomato)Hze</sub>, Jax stock 007909) P28 mice as previously described (<em><a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></em>). As part of a wider study, at P27 these mice had undergone brief 24 h unilateral naris occlusion via a plastic plug insert (N = 5 mice) or were subjected to a sham control manipulation (N = 5 mice); all observed effects here were independent of these treatment groups. Single cell suspensions were generated using the Neural Tissue Dissociation Kit – Postnatal Neurons (Miltenyi Biotec. Cat no. 130-094-802), following manufacturer’s instructions for manual dissociation, using 3 fired-polished Pasteur pipettes of progressively smaller diameter. After enzymatic and mechanical dissociations, cells were filtered through a 30 µm cell strainer, centrifuged for 10 minutes at 4° C, resuspended in 500 µl of ACSF (in mM: 140 NaCl, 1.25 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 10 HEPES, 25 Glucose, 3 MgCl<sub>2</sub>, 1 CaCl<sub>2</sub>) with channel blockers (0.1 µM TTX, 20 µM CNQX, 50 µM D-APV) and kept on ice to minimise excitotoxicity and cell death.</p><p id="p-68">For manual sorting of fluorescently labelled dopaminergic neurons we adapted a previously described protocol (<em><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></em>). 50 µl of single cell suspension was dispersed on 3.5mm petri dishes (with a Sylgard-covered base) containing 2 ml of ACSF + channel blockers. Dishes were left undisturbed for 15 minutes to allow the cells to sink and settle. Throughout, dishes were kept on a metal plate on top of ice. tdTomato-positive cells were identified by their red fluorescence under a stereoscope. Using a pulled glass capillary pipette attached to a mouthpiece, individual cells were aspirated and transferred to a clean, empty dish containing 2 ml ACSF + channel blockers. The same cell was then transferred to a third clean plate, changing pipettes for every plate change. Finally, each individual cell was transferred to a 0.2 ml PCR tube containing 2 µl of lysis buffer (RLT Plus - Qiagen). The tube was immediately placed on a metal plate sitting on top of dry ice for flash-freezing. Collected cells were stored at −80C until further processing. Positive (more than 10 cells) and negative (sample collection procedure without picking a cell) controls were collected for each sorting session. In total, we collected samples from 10 mice, averaging 50 tdTomato-positive cells collected per session. Overall, less than 2.5 hours elapsed between mouse sacrifice and collection of the last cell in any session.</p></div><div id="sec-25" class="subsection"><h4>Preparation and amplification of full-length cDNA and sequencing libraries</h4><p id="p-69">Samples were processing using a modified version of the Smart-Seq2 protocol(<em><a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">75</a></em>). Briefly, 1 µl of a 1:2,000,000 dilution of ERCC spike-ins (Invitrogen. Cat. no. 4456740) was added to each sample and mRNA was captured using modified oligo-dT biotinylated beads (Dynabeads, Invitrogen). PCR amplification was performed for 22 cycles. Amplified cDNA was cleaned with a 0.8:1 ratio of Ampure-XP beads (Beckman Coulter). cDNAs were quantified on Qubit using HS DNA reagents (Invitrogen) and selected samples were run on a Bioanalyzer HS DNA chip (Agilent) to evaluate size distribution.</p><p id="p-70">For generating the sequencing libraries, individual cDNA samples were normalised to 0.2ng/µl and 1µl was used for one-quarter standard-sized Nextera XT (Illumina) tagmentation reactions, with 12 amplification cycles. Sample indexing was performed using index sets A and D (Illumina). At this point, individual samples were pooled according to their index set. Pooled libraries were cleaned using a 0.6:1 ratio of Ampure beads and quantified on Qubit using HS DNA reagents and with the KAPA Library Quantification Kits for Illumina (Roche). Samples were sequenced on two separate rapid-runs on HiSeq2500 (Illumina), generating 100bp paired-end reads. An additional 5 samples were sequenced on MiSeq (Illumina).</p></div><div id="sec-26" class="subsection"><h4>Full-length cDNA sequencing data processing and analysis</h4><p id="p-71">Paired-end read fastq files were demultiplexed, quality controlled using FastQC (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) and trimmed using Trim Galore (<a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>). Reads were pseudoaligned and quantified using kallisto (<em><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></em>) against a reference transcriptome from Ensembl Release 89 (Gencode Release M17 GRCm38.p6) with sequences corresponding to the ERCC spike-ins and the Cre recombinase and tdT genes added to the index. Transcripts were collapsed into genes using the sumAcrossFeatures function in scater.</p><p id="p-72">Cell level quality control and cell filtering was performed in scater (<em><a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></em>). Cells with &lt;1000 genes, &lt;100,000 reads, &gt;75% reads mapping to ERCC spike-ins, &gt;10% reads mapping to mitochondrial genes or low library complexity were discarded (14% samples). The population of olfactory bulb cells labelled in DAT-tdTomato mice is known to include a minor non-dopaminergic calretinin-positive subgroup (<em><a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a></em>), so calretinin-expressing cells were excluded from all analyses. The scTransform function in Seurat was used to remove technical batch effects.</p></div></div><div id="sec-27" class="subsection"><h3>Expression of CoV-relevent genes in scSeq datasets from various brain regions and sensory systems</h3><p id="p-73">An analysis of single-cell gene expression data from 10 studies was performed to investigate the expression of genes coding for coronavirus entry proteins in neurons from a range of brain regions and sensory systems. Processed gene expression data tables were obtained from scSeq studies that evaluated gene expression in retina (GSE81905) (<em><a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a></em>) inner ear sensory epithelium (GSE115934) (<em><a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></em>) and spiral ganglion (GSE114997) (<em><a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></em>), ventral midbrain (GSE76381) (<em><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></em>), hippocampus (GSE100449) (<em><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a></em>), cortex (GSE107632) (<em><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></em>), hypothalamus (GSE74672) (<em><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></em>), visceral motor neurons (GSE78845) (<em><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></em>), dorsal root ganglia (GSE59739) (<em><a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a></em>) and spinal cord dorsal horn (GSE103840) (<em><a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a></em>). Smart-Seq2 sequencing data from Vsx2-GFP positive cells was used from the retina dataset. A subset of the expression matrix that corresponds to day 0 (i.e. control, undisturbed neurons) was used from the layer VI somatosensory cortex dataset. A subset of the data containing neurons from untreated (control) mice was used from the hypothalamic neuron dataset. From the ventral midbrain dopaminergic neuron dataset, a subset comprising DAT-Cre/tdTomato positive neurons from P28 mice was used. A subset comprising Type I neurons from wild type mice was used from the spiral ganglion dataset. The “unclassified” neurons were excluded from the visceral motor neuron dataset. A subset containing neurons that were collected at room temperature was used from the dorsal root ganglia dataset. Expression data from dorsal horn neurons obtained from C57/bl6 wild type mice, vGat-cre-tdTomato and vGlut2-eGFP mouse lines was used from the spinal cord dataset. Inspection of all datasets for batch effects was performed using the scater package (version 1.10.1) (<em><a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">77</a></em>). Publicly available raw count expression matrices were used for the retina, hippocampus, hypothalamus, midbrain, visceral motor neurons and spinal cord datasets, whereas the normalized expression data was used from the inner ear hair cell datasets. For datasets containing raw counts, normalization was performed for each dataset separately by computing pool-based size factors that are subsequently deconvolved to obtain cell-based size factors using the scran package (version 1.10.2) (<em><a id="xref-ref-90-3" class="xref-bibr" href="#ref-90">90</a></em>). Violin plots were generated in scater.</p></div></div><div class="section" id="sec-28"><h2 class="">Supplementary Materials</h2><p id="p-74">Materials and Methods</p><p id="p-75"><a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. S1</a>. Related to <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2</a>. Analysis of mouse bulk RNA-Seq and WOM scSeq dataset.</p><p id="p-76"><a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. S2</a>. Related to <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2</a>. Analysis of mouse HBC lineage scSeq datasets.</p><p id="p-77"><a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. S3</a>. Related to <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3</a>. Bulk RNA-seq datasets for primates and additional figures of human scSeq data.</p><p id="p-78"><a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. S4</a>. Related to <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3</a>. Comparing cell types and expression levels across respiratory and olfactory epithelial datasets.</p><p id="p-79"><a id="xref-fig-9-4" class="xref-fig" href="#F9">Fig. S5</a>. Related to <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a>. Whole OB scSeq data integration and clustering.</p><p id="p-80"><a id="xref-fig-10-3" class="xref-fig" href="#F10">Fig. S6</a>. Related to <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4</a>. Expression of cell type markers in whole OB scSeq data.</p><p id="p-81"><a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig. S7</a>. Related to <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4</a>. Sample quality control of manually sorted OB dopaminergic neurons.</p><p id="p-82"><a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig. S8</a>. Related to <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4</a>. Expression of genes coding for CoV-2 entry-related proteins in neurons forming part of deeply sequenced scSeq datasets from various brain regions and sensory systems.</p></div><div class="section" id="sec-29"><h2 class="">Funding</h2><p id="p-84">SRD is supported by grants RO11DC016222 and U19 NS112953 from the National Institutes of Health and by the Simons Collaboration on the Global Brain. JN was supported by NIH grant RO1DC007235. DR was supported by Programma per Giovani Ricercatori Rita Levi Montalcini granted by the Italian Ministry of Education, University, and Research. KVdB is a postdoctoral fellow of the Belgian American Educational Foundation (BAEF), and is supported by the Research Foundation Flanders (FWO), grant 1246220N. ML and MG were supported by a Leverhulme Trust Research Grant (RPG-2016-095) and a Consolidator Grant from the European Research Council (725729; FUNCOPLAN). JM is supported by a Medical Research Council grant (K013807) and a Medical Research Council Clinical Infrastructure Award (M008924). ICM is supported by a BBSRC New Investigator Grant (BB/P022073/1) and the BBSRC National Capability in Genomics and Single Cell Analysis at Earlham Institute (BB/CCG1720/1). BJG is supported by grant R01DC016859.</p></div><div class="section" id="sec-30"><h2 class="">Author contributions</h2><p id="p-85">DHB, TT, RC, HC, RF, YC, ML, ICM, RAH and BJG designed and performed experiments. DHB, TT, CW, ML, KvdB, BG, RC, HC, DD, KS, and HRdB performed analysis. DR, SD, EP, JSM, BJG, MSG JN, SRD, designed and supervised experiments. DHB, TT, CW, ML, HM, DWL, BJG, MSG, JN and SRD wrote the manuscript</p></div><div class="section COI-statement" id="sec-31"><h2 class="">Competing interests</h2><p id="p-86">DL is an employee of Mars, Inc. None of the other authors have financial interests related to this manuscript.</p></div><div class="section" id="sec-32"><h2 class="">Data and materials availability</h2><p id="p-87">Reanalyzed datasets are obtained from the URLs listed in supplementary materials. All data is currently being deposited and will be made publicly accessible from the NCBI GEO at accession GSExxxx.</p></div><div class="section" id="sec-33"><h2 class="">Supplementary Materials</h2><div class="section" id="sec-34"><h3>URLs for re-analyzed datasets</h3><p id="p-88"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4680959/bin/srep18178-s2.xls">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4680959/bin/srep18178-s2.xls</a></p><p id="p-89"><a href="https://doi.org/10.1371/journal.pone.0113170.s014">https://doi.org/10.1371/journal.pone.0113170.s014</a></p><p id="p-90"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52464">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52464</a></p><p id="p-91"><a href="https://advances.sciencemag.org/highwire/filestream/217162/field_highwire_adjunct_files/0/aax0396_Data_file_S1.xlsx">https://advances.sciencemag.org/highwire/filestream/217162/field_highwire_adjunct_files/0/aax0396_Data_file_S1.xlsx</a></p><p id="p-92"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139522">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139522</a></p><p id="p-93"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99251">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99251</a></p><p id="p-94"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120199">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120199</a></p><p id="p-95"><a href="https://www.genomique.eu/cellbrowser/HCA/">https://www.genomique.eu/cellbrowser/HCA/</a></p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-83">We thank members of the Datta lab, James Schwob, Bernardo Sabatini, Andreas Schaefer, Kevin Franks, Michael Greenberg and Vanessa Ruta for helpful comments on the manuscript. We thank James Lipscombe and Andres Crespo for technical support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-6">Includes previously unpublished datasets (whole olfactory mucosa scSeq, HBC scSeq, olfactory bulb scSeq, olfactory bulb dopaminergic neuron deep sequencing), ACE2 immunohistochemistry in human tissue, and analysis of SARS-CoV-2 cell entry genes in the olfactory bulb and brain; research groups of John Ngai, Matthew Grubb, Bradley Goldstein, Hiro Matsunami (and colleagues) added as collaborators.</p></li><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-7"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4680959/bin/srep18178-s2.xls">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4680959/bin/srep18178-s2.xls</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-4"><p id="p-8"><a href="https://doi.org/10.1371/journal.pone.0113170.s014">https://doi.org/10.1371/journal.pone.0113170.s014</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-5"><p id="p-9"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52464">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52464</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-6"><p id="p-10"><a href="https://advances.sciencemag.org/highwire/filestream/217162/field_highwire_adjunct_files/0/aax0396_Data_file_S1.xlsx">https://advances.sciencemag.org/highwire/filestream/217162/field_highwire_adjunct_files/0/aax0396_Data_file_S1.xlsx</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-7"><p id="p-11"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139522">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139522</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-8"><p id="p-12"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99251">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE99251</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-9"><p id="p-13"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120199">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120199</a></p></li><li class="fn-dataset fn-group-external-links" id="fn-10"><p id="p-14"><a href="https://www.genomique.eu/cellbrowser/HCA/">https://www.genomique.eu/cellbrowser/HCA/</a></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.-j.</span> <span class="cit-name-surname">Guan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-y.</span> <span class="cit-name-surname">Ni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.-h.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-q.</span> <span class="cit-name-surname">Ou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-x.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-l.</span> <span class="cit-name-surname">Lei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. S. C.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-j.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zeng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-Y.</span> <span class="cit-name-surname">Yuen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.-c.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-l.</span> <span class="cit-name-surname">Tang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.-y.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.-y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-l.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-j.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-x.</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-h.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-m.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-j.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.-q.</span> <span class="cit-name-surname">Qiu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Luo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-j.</span> <span class="cit-name-surname">Ye</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.-y.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.-s.</span> <span class="cit-name-surname">Zhong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">China</span></span> <span class="cit-auth cit-collab">Medical Treatment Expert Group for</span>, <span class="cit-article-title">Clinical Characteristics of Coronavirus Disease 2019 in China</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. EPub ahead of print, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Rothan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Byrareddy</span></span>, <span class="cit-article-title">The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak</span>. <abbr class="cit-jnl-abbrev">J. Autoimmun</abbr>. EPub ahead of print, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">McGoogan</span></span>, <span class="cit-article-title">Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72 314 Cases From the Chinese Center for Disease Control and Prevention</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> EPub ahead of print, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.4" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-L.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-G.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-R.</span> <span class="cit-name-surname">Si</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-L.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-D.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Luo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.-D.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-Q.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-R.</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-S.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.-J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-L.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.-X.</span> <span class="cit-name-surname">Zhan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-Y.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.-F.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-L.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.5" data-doi="10.26355/eurrev_202003_20551"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ceccarelli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Berretta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Venanzi Rullo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Nunnari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cacopardo</span></span>, <span class="cit-article-title">Differences and similarities between Severe Acute Respiratory Syndrome (SARS)-CoronaVirus (CoV) and SARS-CoV-2. Would a rose by another name smell as sweet?</span> <abbr class="cit-jnl-abbrev">Eur Rev Med Pharmacol Sci</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">2781</span>–<span class="cit-lpage">2783</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur%2BRev%2BMed%2BPharmacol%2BSci%26rft.volume%253D24%26rft.spage%253D2781%26rft_id%253Dinfo%253Adoi%252F10.26355%252Feurrev_202003_20551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.26355/eurrev_202003_20551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.6" data-doi="10.1038/nrd.2015.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zumla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F. W.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. I.</span> <span class="cit-name-surname">Azhar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. S. C.</span> <span class="cit-name-surname">Hui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-Y.</span> <span class="cit-name-surname">Yuen</span></span>, <span class="cit-article-title">Coronaviruses - drug discovery and therapeutic options</span>. <abbr class="cit-jnl-abbrev">Nat Rev Drug Discov</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">347</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.volume%253D15%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd.2015.37%26rft_id%253Dinfo%253Apmid%252F26868298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd.2015.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26868298&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.7" data-doi="10.1016/bs.aivir.2018.01.001"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Muth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Niemeyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-article-title">Hosts and Sources of Endemic Human Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Adv. Virus Res</abbr>. <span class="cit-vol">100</span>, <span class="cit-fpage">163</span>–<span class="cit-lpage">188</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BVirus%2BRes%26rft.volume%253D100%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2018.01.001%26rft_id%253Dinfo%253Apmid%252F29551135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2018.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29551135&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. H. R.</span> <span class="cit-name-surname">Bagheri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Asghari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farhadi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R. S.</span> <span class="cit-name-surname">medRxiv</span></span>, <span class="cit-article-title">Coincidence of COVID-19 epidemic and olfactory dysfunction outbreak</span>. <abbr class="cit-jnl-abbrev">medRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Giacomelli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pezzati</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Conti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bernacchia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Siano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Oreni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rusconi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gervasoni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Ridolfo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rizzardini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Antinori</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Galli</span></span>, <span class="cit-article-title">Self-reported olfactory and taste disorders in SARS-CoV-2 patients: a cross-sectional study</span>. <abbr class="cit-jnl-abbrev">Clin. Infect. Dis</abbr>., (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Mao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. H. E. a. o.</span> <span class="cit-name-surname">print</span></span>, <span class="cit-article-title">Neurological manifestations of hospitalized patients with COVID-19 in Wuhan, China: a retrospective case series study. medRxiv</span>. <abbr class="cit-jnl-abbrev">medRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wölfel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Guggemos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seilmaier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zange</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Niemeyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Vollmar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rothe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoelscher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bleicker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brünink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schneider</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ehmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zwirglmaier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wendtner</span></span>, <span class="cit-article-title">Virological assessment of hospitalized patients with COVID-2019</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Lechien</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Chiesa-Estomba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">De Siati</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Horoi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Le Bon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rodriguez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dequanter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Blecic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">El Afia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Distinguin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chekkoury-Idrissi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. L.</span> <span class="cit-name-surname">Delgado</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Calvo-Henriquez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lavigne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Falanga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Barillari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Cammaroto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Khalife</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Leich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Souchay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rossi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Journe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hsieh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Edjlali</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Carlier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lovato</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">De Filippis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Coppee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Fakhry</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ayad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Saussez</span></span>, <span class="cit-article-title">Olfactory and gustatory dysfunctions as a clinical presentation of mild-to-moderate forms of the coronavirus disease (COVID-19): a multicenter European study</span>. <abbr class="cit-jnl-abbrev">European Archives of Oto-Rhino-Laryngology</abbr>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Menni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Valdes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Freydin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ganesh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">El-Sayed Moustafa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Visconti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hysi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. C. E.</span> <span class="cit-name-surname">Bowyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mangino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Falchi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wolf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Steves</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Spector</span></span>, <span class="cit-article-title">Loss of smell and taste in combination with other symptoms is a strong predictor of COVID-19 infection</span>. <abbr class="cit-jnl-abbrev">medRxiv</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.14" data-doi="papers3://publication/uuid/B05A7D42-9FC0-46A4-8483-D0FE90434E53"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.-S.</span> <span class="cit-name-surname">Hwang</span></span>, <span class="cit-article-title">Olfactory neuropathy in severe acute respiratory syndrome: report of A case</span>. <abbr class="cit-jnl-abbrev">Acta Neurol Taiwan</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">26</span>–<span class="cit-lpage">28</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Bneurologica%2BTaiwanica%26rft.stitle%253DActa%2BNeurol%2BTaiwan%26rft.aulast%253DHwang%26rft.auinit1%253DC.%2BS.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D28%26rft.atitle%253DOlfactory%2Bneuropathy%2Bin%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%253A%2Breport%2Bof%2BA%2Bcase.%26rft_id%253Dinfo%253Adoi%252Fpapers3%253A%252F%252Fpublication%252Fuuid%252FB05A7D42-9FC0-46A4-8483-D0FE90434E53%26rft_id%253Dinfo%253Apmid%252F16599281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=papers3://publication/uuid/B05A7D42-9FC0-46A4-8483-D0FE90434E53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16599281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.15" data-doi="10.1007/s11882-004-0031-3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Dalton</span></span>, <span class="cit-article-title">Olfaction and anosmia in rhinosinusitis</span>. <abbr class="cit-jnl-abbrev">Curr Allergy Asthma Rep</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">230</span>–<span class="cit-lpage">236</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Ballergy%2Band%2Basthma%2Breports%26rft.stitle%253DCurr%2BAllergy%2BAsthma%2BRep%26rft.aulast%253DDalton%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D230%26rft.epage%253D236%26rft.atitle%253DOlfaction%2Band%2Banosmia%2Bin%2Brhinosinusitis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11882-004-0031-3%26rft_id%253Dinfo%253Apmid%252F15056406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11882-004-0031-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15056406&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.16" data-doi="10.1016/B978-0-444-63855-7.00021-6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Doty</span></span>, <span class="cit-article-title">Systemic diseases and disorders</span>. <abbr class="cit-jnl-abbrev">Handb Clin Neurol</abbr> <span class="cit-vol">164</span>, <span class="cit-fpage">361</span>–<span class="cit-lpage">387</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHandb%2BClin%2BNeurol%26rft.volume%253D164%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-444-63855-7.00021-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-444-63855-7.00021-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.17" data-doi="10.1097/01.mlg.0000249922.37381.1e"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Suzuki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Saito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.-P.</span> <span class="cit-name-surname">Min</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vladau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Toida</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Itoh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Murakami</span></span>, <span class="cit-article-title">Identification of viruses in patients with postviral olfactory dysfunction</span>. <abbr class="cit-jnl-abbrev">Laryngoscope</abbr> <span class="cit-vol">117</span>, <span class="cit-fpage">272</span>–<span class="cit-lpage">277</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLaryngoscope%26rft.stitle%253DLaryngoscope%26rft.aulast%253DSuzuki%26rft.auinit1%253DM.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D272%26rft.epage%253D277%26rft.atitle%253DIdentification%2Bof%2Bviruses%2Bin%2Bpatients%2Bwith%2Bpostviral%2Bolfactory%2Bdysfunction.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F01.mlg.0000249922.37381.1e%26rft_id%253Dinfo%253Apmid%252F17277621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/01.mlg.0000249922.37381.1e&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17277621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243890000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.18" data-doi="10.3109/00016489509133353"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Åkerlund</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bende</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Murphy</span></span>, <span class="cit-article-title">Olfactory threshold and nasal mucosal changes in experimentally induced common cold</span>. <abbr class="cit-jnl-abbrev">Acta Otolaryngol</abbr>. <span class="cit-vol">115</span>, <span class="cit-fpage">88</span>–<span class="cit-lpage">92</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Boto-laryngologica%26rft.stitle%253DActa%2BOtolaryngol%26rft.aulast%253DAkerlund%26rft.auinit1%253DA.%26rft.volume%253D115%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D92%26rft.atitle%253DOlfactory%2Bthreshold%2Band%2Bnasal%2Bmucosal%2Bchanges%2Bin%2Bexperimentally%2Binduced%2Bcommon%2Bcold.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F00016489509133353%26rft_id%253Dinfo%253Apmid%252F7762392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/00016489509133353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7762392&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.25.009084v3.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Hedrich</span></span>, <span class="cit-source">The Laboratory Mouse</span>. (<span class="cit-publ-name">Academic Press</span>, <span class="cit-pub-date">2012</span>), pp. <span class="cit-fpage">845</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.20" data-doi="10.2307/3430681"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. K.</span> <span class="cit-name-surname">Reznik</span></span>, <span class="cit-article-title">Comparative anatomy, physiology, and function of the upper respiratory tract</span>. <abbr class="cit-jnl-abbrev">Environ Health Perspect</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">171</span>–<span class="cit-lpage">176</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BHealth%2BPerspectives%26rft.stitle%253DEnvironmental%2BHealth%2BPerspectives%26rft.aulast%253DReznik%26rft.auinit1%253DG.%2BK.%26rft.volume%253D85%26rft.spage%253D171%26rft.epage%253D176%26rft.atitle%253DComparative%2Banatomy%252C%2Bphysiology%252C%2Band%2Bfunction%2Bof%2Bthe%2Bupper%2Brespiratory%2Btract.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3430681%26rft_id%253Dinfo%253Apmid%252F2384061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3430681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2384061&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DJ66100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.21" data-doi="10.1006/exnr.1995.1099"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Suzuki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schafer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Farbman</span></span>, <span class="cit-article-title">Phagocytic cells in the rat olfactory epithelium after bulbectomy</span>. <abbr class="cit-jnl-abbrev">Exp. Neurol</abbr>. <span class="cit-vol">136</span>, <span class="cit-fpage">225</span>–<span class="cit-lpage">233</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Bneurology%26rft.stitle%253DExp%2BNeurol%26rft.aulast%253DSuzuki%26rft.auinit1%253DY.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D233%26rft.atitle%253DPhagocytic%2Bcells%2Bin%2Bthe%2Brat%2Bolfactory%2Bepithelium%2Bafter%2Bbulbectomy.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fexnr.1995.1099%26rft_id%253Dinfo%253Apmid%252F7498412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/exnr.1995.1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7498412&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TK25800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.22" data-doi="10.1002/(SICI)1096-9861(19961223)376:4&lt;509::AID-CNE1&gt;3.0.CO;2-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Suzuki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Takeda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Farbman</span></span>, <span class="cit-article-title">Supporting cells as phagocytes in the olfactory epithelium after bulbectomy</span>. <abbr class="cit-jnl-abbrev">J. Comp. Neurol</abbr>. <span class="cit-vol">376</span>, <span class="cit-fpage">509</span>–<span class="cit-lpage">517</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bcomparative%2Bneurology%26rft.stitle%253DJ%2BComp%2BNeurol%26rft.aulast%253DSuzuki%26rft.auinit1%253DY.%26rft.volume%253D376%26rft.issue%253D4%26rft.spage%253D509%26rft.epage%253D517%26rft.atitle%253DSupporting%2Bcells%2Bas%2Bphagocytes%2Bin%2Bthe%2Bolfactory%2Bepithelium%2Bafter%2Bbulbectomy.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-9861%252819961223%2529376%253A4%253C509%253A%253AAID-CNE1%253E3.0.CO%253B2-5%26rft_id%253Dinfo%253Apmid%252F8978466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-9861(19961223)376:4&lt;509::AID-CNE1&gt;3.0.CO;2-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8978466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VZ43300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.23" data-doi="10.1113/jphysiol.2004.079228"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Vogalis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Hegg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Lucero</span></span>, <span class="cit-article-title">Ionic conductances in sustentacular cells of the mouse olfactory epithelium</span>. <abbr class="cit-jnl-abbrev">J. Physiol. (Lond).</abbr> <span class="cit-vol">562</span>, <span class="cit-fpage">785</span>–<span class="cit-lpage">799</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhysiol.%2B%2528Lond%2529.%26rft_id%253Dinfo%253Adoi%252F10.1113%252Fjphysiol.2004.079228%26rft_id%253Dinfo%253Apmid%252F15611020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1113/jphysiol.2004.079228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15611020&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226778700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pfister</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Weber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Härtig</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schwerdel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Elsaesser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Knuesel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Fritschy</span></span>, <span class="cit-article-title">Novel role of cystic fibrosis transmembrane conductance regulator in maintaining adult mouse olfactory neuronal homeostasis</span>. <abbr class="cit-jnl-abbrev">J. Comp. Neurol</abbr>. <span class="cit-vol">523</span>, <span class="cit-fpage">406</span>–<span class="cit-lpage">430</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComp.%2BNeurol%26rft.volume%253D523%26rft.spage%253D406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.25" data-doi="10.1002/lio2.135"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Choi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Goldstein</span></span>, <span class="cit-article-title">Olfactory epithelium: Cells, clinical disorders, and insights from an adult stem cell niche</span>. <abbr class="cit-jnl-abbrev">Laryngoscope Investig Otolaryngol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">35</span>–<span class="cit-lpage">42</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaryngoscope%2BInvestig%2BOtolaryngol%26rft.volume%253D3%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1002%252Flio2.135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/lio2.135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.26" data-doi="10.1002/cne.24105"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Schwob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Jang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Holbrook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Herrick</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Peterson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hewitt Coleman</span></span>, <span class="cit-article-title">Stem and progenitor cells of the mammalian olfactory epithelium: Taking poietic license</span>. <abbr class="cit-jnl-abbrev">J. Comp. Neurol</abbr>. <span class="cit-vol">525</span>, <span class="cit-fpage">1034</span>–<span class="cit-lpage">1054</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComp.%2BNeurol%26rft.volume%253D525%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcne.24105%26rft_id%253Dinfo%253Apmid%252F27560601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cne.24105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27560601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.27" data-doi="10.1016/j.stem.2017.04.003"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Fletcher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Das</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gadye</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. N.</span> <span class="cit-name-surname">Street</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Baudhuin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wagner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Cole</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Flores</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. G.</span> <span class="cit-name-surname">Choi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Yosef</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Purdom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dudoit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Risso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ngai</span></span>, <span class="cit-article-title">Deconstructing Olfactory Stem Cell Trajectories at Single-Cell Resolution</span>. <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">817</span>–<span class="cit-lpage">830</span>.e818 (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D20%26rft.spage%253D817%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2017.04.003%26rft_id%253Dinfo%253Apmid%252F28506465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2017.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28506465&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kleine-Weber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schroeder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Krüger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Herrler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Erichsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schiergens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Herrler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.-H.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nitsche</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pöhlmann</span></span>, <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.29" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vasilieva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Wong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Berne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Somasundaran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Sullivan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Luzuriaga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Greenough</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Choe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farzan</span></span>, <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.30" data-doi="10.1128/JVI.00737-08"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Netland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Meyerholz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cassell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Perlman</span></span>, <span class="cit-article-title">Severe acute respiratory syndrome coronavirus infection causes neuronal death in the absence of encephalitis in mice transgenic for human ACE2</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">82</span>, <span class="cit-fpage">7264</span>–<span class="cit-lpage">7275</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00737-08%26rft_id%253Dinfo%253Apmid%252F18495771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNS83MjY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4yNS4wMDkwODQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.31" data-doi="10.1128/JVI.78.20.11429-11433.2004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. C.</span> <span class="cit-name-surname">Greenough</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vasilieva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Somasundaran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Sullivan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farzan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Choe</span></span>, <span class="cit-article-title">Efficient replication of severe acute respiratory syndrome coronavirus in mouse cells is limited by murine angiotensin-converting enzyme 2</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">11429</span>–<span class="cit-lpage">11433</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D78%26rft.issue%253D20%26rft.spage%253D11429%26rft.epage%253D11433%26rft.atitle%253DEfficient%2BReplication%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2Bin%2BMouse%2BCells%2BIs%2BLimited%2Bby%2BMurine%2BAngiotensin-Converting%2BEnzyme%2B2%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.20.11429-11433.2004%26rft_id%253Dinfo%253Apmid%252F15452268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8yMC8xMTQyOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjUuMDA5MDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.32" data-doi="10.1038/nm1267"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kuba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Imai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Guan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chappell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Leibbrandt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wada</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Slutsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Qin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Penninger</span></span>, <span class="cit-article-title">A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury</span>. <abbr class="cit-jnl-abbrev">Nat. Med</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">875</span>–<span class="cit-lpage">879</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DKuba%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D875%26rft.epage%253D879%26rft.atitle%253DA%2Bcrucial%2Brole%2Bof%2Bangiotensin%2Bconverting%2Benzyme%2B2%2B%2528ACE2%2529%2Bin%2BSARS%2Bcoronavirus-induced%2Blung%2Binjury.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1267%26rft_id%253Dinfo%253Apmid%252F16007097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16007097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230964700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.33" data-doi="10.1128/JVI.05300-11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bertram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Glowacka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lavender</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gnirss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Nehlmeier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Niemeyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Simmons</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Soilleux</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Jahn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Steffen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pöhlmann</span></span>, <span class="cit-article-title">Cleavage and activation of the severe acute respiratory syndrome coronavirus spike protein by human airway trypsin-like protease</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">85</span>, <span class="cit-fpage">13363</span>–<span class="cit-lpage">13372</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05300-11%26rft_id%253Dinfo%253Apmid%252F21994442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yNC8xMzM2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjUuMDA5MDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.34" data-doi="10.1371/journal.pone.0035876"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bertram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Heurich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lavender</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gierer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Danisch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Perin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Lucas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Nelson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pöhlmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Soilleux</span></span>, <span class="cit-article-title">Influenza and SARS-coronavirus activating proteases TMPRSS2 and HAT are expressed at multiple sites in human respiratory and gastrointestinal tracts</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e35876</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBertram%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35876%26rft.epage%253De35876%26rft.atitle%253DInfluenza%2Band%2BSARS-coronavirus%2Bactivating%2Bproteases%2BTMPRSS2%2Band%2BHAT%2Bare%2Bexpressed%2Bat%2Bmultiple%2Bsites%2Bin%2Bhuman%2Brespiratory%2Band%2Bgastrointestinal%2Btracts.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035876%26rft_id%253Dinfo%253Apmid%252F22558251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22558251&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.35" data-doi="10.1074/jbc.RA118.001897"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-M.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. K.-H.</span> <span class="cit-name-surname">Au-Yeung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-H.</span> <span class="cit-name-surname">Sze</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shuai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. K.-M.</span> <span class="cit-name-surname">Poon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Leung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-L.</span> <span class="cit-name-surname">Yeung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.-Y.</span> <span class="cit-name-surname">Jin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Gao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.-W.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.-Y.</span> <span class="cit-name-surname">Yuen</span></span>, <span class="cit-article-title">Middle East respiratory syndrome coronavirus and bat coronavirus HKU9 both can utilize GRP78 for attachment onto host cells</span>. <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-vol">293</span>, <span class="cit-fpage">11709</span>–<span class="cit-lpage">11726</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.RA118.001897%26rft_id%253Dinfo%253Apmid%252F29887526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTMvMzAvMTE3MDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjAzLjI1LjAwOTA4NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.36"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Sungnak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bécavin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Berg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. C. A. L. B.</span> <span class="cit-name-surname">Network</span></span>, <span class="cit-article-title">SARS-CoV-2 Entry Genes Are Most Highly Expressed in Nasal Goblet and Ciliated Cells within Human Airways</span>. <abbr class="cit-jnl-abbrev">arXiv preprint arXiv:2003.06122</abbr>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Deprez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.-E.</span> <span class="cit-name-surname">Zaragosi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Truchi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Garcia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-J.</span> <span class="cit-name-surname">Arguel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lebrigand</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Paquet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pe’er</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-H.</span> <span class="cit-name-surname">Marquette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Leroy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Barbry</span></span>, <span class="cit-article-title">A single-cell atlas of the human healthy airways</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">3000</span>–<span class="cit-lpage">3049</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D24%26rft.spage%253D3000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.38" data-doi="10.1038/S41591-019-0468-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Vieira Braga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Berg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Carpaij</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Polanski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Simon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brouwer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Gomes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hesse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Fasouli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Efremova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vento-Tormo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Talavera-López</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Jonker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Affleck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Palit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Strzelecka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. V.</span> <span class="cit-name-surname">Firth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. T.</span> <span class="cit-name-surname">Mahbubani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cvejic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Meyer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Saeb-Parsy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Luinge</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-A.</span> <span class="cit-name-surname">Brandsma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Timens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Angelidis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Strunz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Koppelman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">van Oosterhout</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. B.</span> <span class="cit-name-surname">Schiller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Theis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">van den Berge</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Nawijn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Teichmann</span></span>, <span class="cit-article-title">A cellular census of human lungs identifies novel cell states in health and in asthma</span>. <abbr class="cit-jnl-abbrev">Nat. Med</abbr>. <span class="cit-vol">25</span>, <span class="cit-fpage">1153</span>–<span class="cit-lpage">1163</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D25%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1038%252FS41591-019-0468-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/S41591-019-0468-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.39" data-doi="10.1038/srep18178"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Saraiva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ibarra-Soria</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Omura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Scialdone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Mombaerts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Marioni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Logan</span></span>, <span class="cit-article-title">Hierarchical deconstruction of mouse olfactory sensory neurons: from whole mucosa to single-cell RNA-seq</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">18178</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D18178%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep18178%26rft_id%253Dinfo%253Apmid%252F26670777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep18178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26670777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.40" data-doi="10.1371/journal.pone.0113170"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kanageswaran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Demond</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nagel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. S. P.</span> <span class="cit-name-surname">Schreiner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Baumgart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Scholz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Altmüller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Becker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Doerner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Conrad</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Oberland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Wetzel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Neuhaus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hatt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gisselmann</span></span>, <span class="cit-article-title">Deep Sequencing of the Murine Olfactory Receptor Neuron Transcriptome</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e0113170</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0113170%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0113170%26rft_id%253Dinfo%253Apmid%252F25590618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0113170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25590618&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.41" data-doi="10.1016/j.neuron.2014.07.013"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Bradley M.</span> <span class="cit-name-surname">Colquitt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Markenscoff-Papadimitriou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Duffié</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lomvardas</span></span>, <span class="cit-article-title">Dnmt3a Regulates Global Gene Expression in Olfactory Sensory Neurons and Enables Odorant-Induced Transcription</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">823</span>–<span class="cit-lpage">838</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D83%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2014.07.013%26rft_id%253Dinfo%253Apmid%252F25123312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2014.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25123312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.42" data-doi="10.1016/j.stem.2017.10.014"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gadye</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Das</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Sanchez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Street</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Baudhuin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wagner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Cole</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. G.</span> <span class="cit-name-surname">Choi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Yosef</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Purdom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dudoit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Risso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ngai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Fletcher</span></span>, <span class="cit-article-title">Injury Activates Transient Olfactory Stem Cell States with Diverse Lineage Capacities</span>. <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">775</span>–<span class="cit-lpage">790</span>.e779 (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D21%26rft.spage%253D775%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2017.10.014%26rft_id%253Dinfo%253Apmid%252F29174333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2017.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29174333&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.43" data-doi="10.1126/sciadv.aax0396"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Saraiva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Riveros-McKay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mezzavilla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Abou-Moussa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Arayata</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Makhlouf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Trimmer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ibarra-Soria</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Khan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">van Gerven</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jorissen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gibbs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">O’Flynn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">McGrane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Mombaerts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Marioni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Mainland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Logan</span></span>, <span class="cit-article-title">A transcriptomic atlas of mammalian olfactory mucosae reveals an evolutionary influence on food odor detection in humans</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">eaax0396</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DSaraiva%26rft.auinit1%253DL.%2BR.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253Deaax0396%26rft.epage%253Deaax0396%26rft.atitle%253DA%2Btranscriptomic%2Batlas%2Bof%2Bmammalian%2Bolfactory%2Bmucosae%2Breveals%2Ban%2Bevolutionary%2Binfluence%2Bon%2Bfood%2Bodor%2Bdetection%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aax0396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNS83L2VhYXgwMzk2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4yNS4wMDkwODQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.44"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Durante</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kurtenbach</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. B.</span> <span class="cit-name-surname">Sargi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Harbour</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Choi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kurtenbach</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Goss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Matsunami</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Goldstein</span></span>, <span class="cit-article-title">Single-cell analysis of olfactory neurogenesis and differentiation in adult humans</span>. <abbr class="cit-jnl-abbrev">Nature Neuroscience</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">326</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D23%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.45" data-doi="10.1016/j.cell.2018.06.021"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zeisel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hochgerner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lönnerberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Johnsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Memic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">van der Zwan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Häring</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Braun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Borm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">La Manno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Codeluppi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Furlan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Skene</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">Harris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hjerling-Leffler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Arenas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ernfors</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Marklund</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-article-title">Molecular Architecture of the Mouse Nervous System</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">174</span>, <span class="cit-fpage">999</span>–<span class="cit-lpage">1014</span>.e1022 (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D174%26rft.spage%253D999%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.06.021%26rft_id%253Dinfo%253Apmid%252F30096314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.06.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30096314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Foster</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Courtney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">King</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Howells</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Sutherland</span></span>, <span class="cit-article-title">Pericytes and Neurovascular Function in the Healthy and Diseased Brain</span>. <abbr class="cit-jnl-abbrev">Front Cell Neurosci</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">282</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BCell%2BNeurosci%26rft.volume%253D13%26rft.spage%253D282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.47" data-doi="10.1002/path.1570"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hamming</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Timens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L. C.</span> <span class="cit-name-surname">Bulthuis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Lely</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Navis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">van Goor</span></span>, <span class="cit-article-title">Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</span>. <abbr class="cit-jnl-abbrev">J. Pathol</abbr>. <span class="cit-vol">203</span>, <span class="cit-fpage">631</span>–<span class="cit-lpage">637</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DHamming%26rft.auinit1%253DI.%26rft.volume%253D203%26rft.issue%253D2%26rft.spage%253D631%26rft.epage%253D637%26rft.atitle%253DTissue%2Bdistribution%2Bof%2BACE2%2Bprotein%252C%2Bthe%2Bfunctional%2Breceptor%2Bfor%2BSARS%2Bcoronavirus.%2BA%2Bfirst%2Bstep%2Bin%2Bunderstanding%2BSARS%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpath.1570%26rft_id%253Dinfo%253Apmid%252F15141377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/path.1570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15141377&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221633700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.48" data-doi="10.1038/s41586-018-0394-6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. W.</span> <span class="cit-name-surname">Plasschaert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Žilionis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Choo-Wing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Savova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Knehr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Roma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Klein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Jaffe</span></span>, <span class="cit-article-title">A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary ionocyte</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">560</span>, <span class="cit-fpage">377</span>–<span class="cit-lpage">381</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D560%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0394-6%26rft_id%253Dinfo%253Apmid%252F30069046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0394-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30069046&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.49" data-doi="10.1177/030098589503200101"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Bihun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Percy</span></span>, <span class="cit-article-title">Morphologic changes in the nasal cavity associated with sialodacryoadenitis virus infection in the Wistar rat</span>. <abbr class="cit-jnl-abbrev">Vet. Pathol</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2BPathology%26rft.stitle%253DVeterinary%2BPathology%26rft.aulast%253DBihun%26rft.auinit1%253DC.%2BG.%2BD.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D10%26rft.atitle%253DMorphologic%2BChanges%2Bin%2Bthe%2BNasal%2BCavity%2BAssociated%2Bwith%2BSialodacryoadenitis%2BVirus%2BInfection%2Bin%2Bthe%2BWistar%2BRat%26rft_id%253Dinfo%253Adoi%252F10.1177%252F030098589503200101%26rft_id%253Dinfo%253Apmid%252F7725592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/030098589503200101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7725592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.50" data-doi="10.1016/0166-2236(90)90017-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Farbman</span></span>, <span class="cit-article-title">Olfactory neurogenesis: genetic or environmental controls?</span> <abbr class="cit-jnl-abbrev">Trends Neurosci</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bneurosciences%26rft.stitle%253DTrends%2BNeurosci%26rft.aulast%253DFarbman%26rft.auinit1%253DA.%2BI.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DOlfactory%2Bneurogenesis%253A%2Bgenetic%2Bor%2Benvironmental%2Bcontrols%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0166-2236%252890%252990017-5%26rft_id%253Dinfo%253Apmid%252F1699323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0166-2236(90)90017-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1699323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DV73500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.51" data-doi="10.1002/cne.22084"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sultan-Styne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Toledo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Walker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kallkopf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Ribak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Guthrie</span></span>, <span class="cit-article-title">Long-term survival of olfactory sensory neurons after target depletion</span>. <abbr class="cit-jnl-abbrev">J. Comp. Neurol</abbr>. <span class="cit-vol">515</span>, <span class="cit-fpage">696</span>–<span class="cit-lpage">710</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bcomparative%2Bneurology%26rft.stitle%253DJ%2BComp%2BNeurol%26rft.aulast%253DSultan-Styne%26rft.auinit1%253DK.%26rft.volume%253D515%26rft.issue%253D6%26rft.spage%253D696%26rft.epage%253D710%26rft.atitle%253DLong-term%2Bsurvival%2Bof%2Bolfactory%2Bsensory%2Bneurons%2Bafter%2Btarget%2Bdepletion.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcne.22084%26rft_id%253Dinfo%253Apmid%252F19496176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cne.22084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19496176&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.52" data-doi="10.3389/fncel.2018.00386"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bohmwald</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M. S.</span> <span class="cit-name-surname">Gálvez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ríos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Kalergis</span></span>, <span class="cit-article-title">Neurologic Alterations Due to Respiratory Virus Infections</span>. <abbr class="cit-jnl-abbrev">Front Cell Neurosci</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">386</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BCell%2BNeurosci%26rft.volume%253D12%26rft.spage%253D386%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffncel.2018.00386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fncel.2018.00386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.53" data-doi="10.1093/chemse/26.8.953"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Youngentob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Schwob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Saha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Manglapus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jubelt</span></span>, <span class="cit-article-title">Functional consequences following infection of the olfactory system by intranasal infusion of the olfactory bulb line variant (OBLV) of mouse hepatitis strain JHM</span>. <abbr class="cit-jnl-abbrev">Chemical Senses</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">953</span>–<span class="cit-lpage">963</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BSenses%26rft.stitle%253DChem%2BSenses%26rft.aulast%253DYoungentob%26rft.auinit1%253DS.%2BL.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D953%26rft.epage%253D963%26rft.atitle%253DFunctional%2BConsequences%2BFollowing%2BInfection%2Bof%2Bthe%2BOlfactory%2BSystem%2Bby%2BIntranasal%2BInfusion%2Bof%2Bthe%2BOlfactory%2BBulb%2BLine%2BVarient%2B%2528OBLV%2529%2Bof%2BMouse%2BHepatitis%2BStrain%2BJHM%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fchemse%252F26.8.953%26rft_id%253Dinfo%253Apmid%252F11595672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/chemse/26.8.953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11595672&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171690400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.54" data-doi="10.1006/viro.1993.1248"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Barnett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Perlman</span></span>, <span class="cit-article-title">The olfactory nerve and not the trigeminal nerve is the major site of CNS entry for mouse hepatitis virus, strain JHM</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">194</span>, <span class="cit-fpage">185</span>–<span class="cit-lpage">191</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBarnett%26rft.auinit1%253DE.%2BM.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D191%26rft.atitle%253DThe%2Bolfactory%2Bnerve%2Band%2Bnot%2Bthe%2Btrigeminal%2Bnerve%2Bis%2Bthe%2Bmajor%2Bsite%2Bof%2BCNS%2Bentry%2Bfor%2Bmouse%2Bhepatitis%2Bvirus%252C%2Bstrain%2BJHM.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1993.1248%26rft_id%253Dinfo%253Apmid%252F8386871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1993.1248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8386871&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KY34400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.55" data-doi="10.1093/chemse/26.8.937"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Schwob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Saha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Youngentob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jubelt</span></span>, <span class="cit-article-title">Intranasal inoculation with the olfactory bulb line variant of mouse hepatitis virus causes extensive destruction of the olfactory bulb and accelerated turnover of neurons in the olfactory epithelium of mice</span>. <abbr class="cit-jnl-abbrev">Chemical Senses</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">937</span>–<span class="cit-lpage">952</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BSenses%26rft.stitle%253DChem%2BSenses%26rft.aulast%253DSchwob%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D937%26rft.epage%253D952%26rft.atitle%253DIntranasal%2BInoculation%2Bwith%2Bthe%2BOlfactory%2BBulb%2BLine%2BVariant%2Bof%2BMouse%2BHepatitis%2BVirus%2BCauses%2BExtensive%2BDestruction%2Bof%2Bthe%2BOlfactory%2BBulb%2Band%2BAccelerated%2BTurnover%2Bof%2BNeurons%2Bin%2Bthe%2BOlfactory%2BEpithelium%2Bof%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fchemse%252F26.8.937%26rft_id%253Dinfo%253Apmid%252F11595671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/chemse/26.8.937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11595671&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.56" data-doi="10.1128/JVI.78.18.10156-10165.2004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hemmila</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Turbide</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Olson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jothy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Holmes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Beauchemin</span></span>, <span class="cit-article-title">Ceacam1a-/- mice are completely resistant to infection by murine coronavirus mouse hepatitis virus A59</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">10156</span>–<span class="cit-lpage">10165</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHemmila%26rft.auinit1%253DE.%26rft.volume%253D78%26rft.issue%253D18%26rft.spage%253D10156%26rft.epage%253D10165%26rft.atitle%253DCeacam1a-%252F-%2BMice%2BAre%2BCompletely%2BResistant%2Bto%2BInfection%2Bby%2BMurine%2BCoronavirus%2BMouse%2BHepatitis%2BVirus%2BA59%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.18.10156-10165.2004%26rft_id%253Dinfo%253Apmid%252F15331748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8xOC8xMDE1NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjUuMDA5MDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.57" data-doi="10.1086/427811"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Mi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Shang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Qian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. X.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Duan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ding</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xue</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-article-title">Function of HAb18G/CD147 in invasion of host cells by severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">J. Infect. Dis</abbr>. <span class="cit-vol">191</span>, <span class="cit-fpage">755</span>–<span class="cit-lpage">760</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F427811%26rft_id%253Dinfo%253Apmid%252F15688292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/427811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15688292&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.58"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-S.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-Q.</span> <span class="cit-name-surname">Lian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-Y.</span> <span class="cit-name-surname">Cui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-J.</span> <span class="cit-name-surname">Geng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-X.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-F.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-Q.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-M.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-M.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-H.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Miao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.-Y.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-N.</span> <span class="cit-name-surname">Chen</span></span>, <abbr class="cit-jnl-abbrev">SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</abbr>. <span class="cit-vol">395</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">410</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSARS-CoV-2%2Binvades%2Bhost%2Bcells%2Bvia%2Ba%2Bnovel%2Broute%253A%2BCD147-spike%2Bprotein%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.59" data-doi="10.1093/nar/27.22.4324"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Indra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Warot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Brocard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Bornert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chambon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Metzger</span></span>, <span class="cit-article-title">Temporally-controlled site-specific mutagenesis in the basal layer of the epidermis: comparison of the recombinase activity of the tamoxifen-inducible Cre-ER(T) and Cre-ER(T2) recombinases</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">4324</span>–<span class="cit-lpage">4327</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.22.4324%26rft_id%253Dinfo%253Apmid%252F10536138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.22.4324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10536138&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083901800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.60"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Srinivas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Watanabe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-S.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">William</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tanabe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Jessell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Costantini</span></span>, <span class="cit-article-title">Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus</span>. <abbr class="cit-jnl-abbrev">BMC Developmental Biology</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">4</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BDevelopmental%2BBiology%26rft.volume%253D1%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.61"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Cole</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Risso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wagner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">DeTomaso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ngai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Purdom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dudoit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Yosef</span></span>, <span class="cit-article-title">Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">315</span>–<span class="cit-lpage">328</span>.e318 (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D8%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">62.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.009084v3.62"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">McGinnis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Murrow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. J.</span> <span class="cit-name-surname">Gartner</span></span>, <span class="cit-article-title">DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">329</span>–<span class="cit-lpage">337</span>.e324 (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D8%26rft.spage%253D329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.63"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Wolock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lopez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Klein</span></span>, <span class="cit-article-title">Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">281</span>–<span class="cit-lpage">291</span>.e289 (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D8%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.64" data-doi="10.1038/nbt.2931"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Risso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ngai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Speed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dudoit</span></span>, <span class="cit-article-title">Normalization of RNA-seq data using factor analysis of control genes or samples</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">896</span>–<span class="cit-lpage">902</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D32%26rft.spage%253D896%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2931%26rft_id%253Dinfo%253Apmid%252F25150836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25150836&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.65" data-doi="10.1371/journal.pcbi.1006378"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Risso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Purvis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Fletcher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Das</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ngai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dudoit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Purdom</span></span>, <span class="cit-article-title">clusterExperiment and RSEC: A Bioconductor package and framework for clustering of single-cell and other large gene expression datasets</span>. <abbr class="cit-jnl-abbrev">PLOS Computational Biology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1006378</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D14%26rft.spage%253D1006378e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.66"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Weinreb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wolock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Klein</span></span>, <span class="cit-article-title">SPRING: a kinetic interface for visualizing high dimensional single-cell expression data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">1246</span>–<span class="cit-lpage">1248</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D1246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.67" data-doi="10.1186/s13059-017-1382-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Wolf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Angerer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Theis</span></span>, <span class="cit-article-title">SCANPY: large-scale single-cell gene expression data analysis</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">15</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D19%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1382-0%26rft_id%253Dinfo%253Apmid%252F29409532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1382-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29409532&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.68" data-doi="10.1126/science.aaw3381"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Weinreb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rodriguez-Fraticelli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. D.</span> <span class="cit-name-surname">Camargo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Klein</span></span>, <span class="cit-article-title">Lineage tracing on transcriptional landscapes links state to fate during differentiation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">367</span>, <span class="cit-fpage">eaaw3381</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw3381%26rft_id%253Dinfo%253Apmid%252F31974159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNjcvNjQ3OS9lYWF3MzM4MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjUuMDA5MDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.69" data-doi="10.1186/s13195-016-0217-7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Kehoe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Al Mulhim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Palmer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Miners</span></span>, <span class="cit-article-title">Angiotensin-converting enzyme 2 is reduced in Alzheimer’s disease in association with increasing amyloid-β and tau pathology</span>. <abbr class="cit-jnl-abbrev">Alzheimers Res Ther</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">50</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlzheimers%2BRes%2BTher%26rft.volume%253D8%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13195-016-0217-7%26rft_id%253Dinfo%253Apmid%252F27884212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13195-016-0217-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27884212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.70" data-doi="10.1016/j.cell.2015.05.002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. Z.</span> <span class="cit-name-surname">Macosko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Basu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Satija</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Nemesh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Shekhar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goldman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Tirosh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Bialas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kamitaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Martersteck</span></span>, <span class="cit-article-title">Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">161</span>, <span class="cit-fpage">1202</span>–<span class="cit-lpage">1214</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1202%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.05.002%26rft_id%253Dinfo%253Apmid%252F26000488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.71" data-doi="10.1016/j.cell.2019.05.031"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Stuart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Butler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hoffman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hafemeister</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Papalexi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Mauck III</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stoeckius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Smibert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Satija</span></span>, <span class="cit-article-title">Comprehensive integration of single-cell data</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">177</span>, <span class="cit-fpage">1888</span>–<span class="cit-lpage">1902</span>. e1821 (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D177%26rft.spage%253D1888%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2019.05.031%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2019.05.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.72"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zappia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Oshlack</span></span>, <span class="cit-article-title">Clustering trees: a visualization for evaluating clusterings at multiple resolutions</span>. <abbr class="cit-jnl-abbrev">GigaScience</abbr> <span class="cit-vol">7</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.73" data-doi="10.7554/eLife.32373"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Galliano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Franzoni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Breton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. N.</span> <span class="cit-name-surname">Chand</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Byrne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. N.</span> <span class="cit-name-surname">Murthy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Grubb</span></span>, <span class="cit-article-title">Embryonic and postnatal neurogenesis produce functionally distinct subclasses of dopaminergic neuron</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e32373</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De32373%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.32373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.32373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.74" data-doi="10.1038/nprot.2007.416"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hempel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sugino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Nelson</span></span>, <span class="cit-article-title">A manual method for the purification of fluorescently labeled neurons from the mammalian brain</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">2924</span>–<span class="cit-lpage">2929</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHempel%26rft.auinit1%253DC.%2BM.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D2924%26rft.epage%253D2929%26rft.atitle%253DA%2Bmanual%2Bmethod%2Bfor%2Bthe%2Bpurification%2Bof%2Bfluorescently%2Blabeled%2Bneurons%2Bfrom%2Bthe%2Bmammalian%2Bbrain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.416%26rft_id%253Dinfo%253Apmid%252F18007629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18007629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253140000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.75"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. C.</span> <span class="cit-name-surname">Macaulay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Teng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Haerty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Ponting</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Voet</span></span>, <span class="cit-article-title">Separation and parallel sequencing of the genomes and transcriptomes of single cells using G&amp;T-seq</span>. <abbr class="cit-jnl-abbrev">Nature Protocols</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">2081</span>–<span class="cit-lpage">2103</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BProtocols%26rft.volume%253D11%26rft.spage%253D2081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.76" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Bray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pimentel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Melsted</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pachter</span></span>, <span class="cit-article-title">Near-optimal probabilistic RNA-seq quantification</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.77" data-doi="10.1093/bioinformatics/btw777"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">McCarthy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Campbell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Lun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q. F.</span> <span class="cit-name-surname">Wills</span></span>, <span class="cit-article-title">Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">1179</span>–<span class="cit-lpage">1186</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D1179%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.78" data-doi="10.1016/j.neuron.2010.09.030"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ninkovic</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pinto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Petricca</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lepier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Rieger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Schroeder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cvekl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Favor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Götz</span></span>, <span class="cit-article-title">The transcription factor Pax6 regulates survival of dopaminergic olfactory bulb neurons via crystallin αA</span>. <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">68</span>, <span class="cit-fpage">682</span>–<span class="cit-lpage">694</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D68%26rft.spage%253D682%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2010.09.030%26rft_id%253Dinfo%253Apmid%252F21092858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2010.09.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21092858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285079500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.79" data-doi="10.1016/j.cell.2016.07.054"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Shekhar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. W.</span> <span class="cit-name-surname">Lapan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. E.</span> <span class="cit-name-surname">Whitney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Tran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. Z.</span> <span class="cit-name-surname">Macosko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kowalczyk</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Adiconis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. Z.</span> <span class="cit-name-surname">Levin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Nemesh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goldman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">McCarroll</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Cepko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Regev</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Sanes</span></span>, <span class="cit-article-title">Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>, <span class="cit-fpage">1308</span>–<span class="cit-lpage">1323</span>.e1330 (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1308%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.07.054%26rft_id%253Dinfo%253Apmid%252F27565351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.07.054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27565351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.80" data-doi="10.1038/ncomms7802"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Burns</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Kelly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Morell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Kelley</span></span>, <span class="cit-article-title">Single-cell RNA-Seq resolves cellular complexity in sensory organs from the neonatal inner ear</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.81"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">McInturff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Burns</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Kelley</span></span>, <span class="cit-article-title">Characterization of spatial and temporal development of Type I and Type II hair cells in the mouse utricle using new cell-type-specific markers</span>. <abbr class="cit-jnl-abbrev">Biology open</abbr> <span class="cit-vol">7</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.82" data-doi="10.1016/j.cell.2018.07.007"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Shrestha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Kujawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Liberman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. V.</span> <span class="cit-name-surname">Goodrich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Shrestha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Kujawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Liberman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. V.</span> <span class="cit-name-surname">Goodrich</span></span>, <span class="cit-article-title">Sensory neuron diversity in the inner ear is shaped by activity sensory neuron diversity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">174</span>, <span class="cit-fpage">1229</span>–<span class="cit-lpage">1246</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D174%26rft.spage%253D1229%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.07.007%26rft_id%253Dinfo%253Apmid%252F30078709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30078709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.83" data-doi="10.1016/j.cell.2016.09.027"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">La Manno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gyllborg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Codeluppi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Barker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Arenas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-article-title">Molecular diversity of midbrain development in resource molecular diversity of midbrain development in mouse, human and stem cells</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">167</span>, <span class="cit-fpage">566</span>–<span class="cit-lpage">580</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D566%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.09.027%26rft_id%253Dinfo%253Apmid%252F27716510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.09.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27716510&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.84" data-doi="10.1016/j.cell.2018.03.031"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Cembrowski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Phillips</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">DiLisio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Shields</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Winnubst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chandrashekar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Spruston</span></span>, <span class="cit-article-title">Dissociable Structural and Functional Hippocampal Outputs via Distinct Subiculum Cell Classes</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">1280</span>–<span class="cit-lpage">1292</span>.e1218 (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1280%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.031%26rft_id%253Dinfo%253Apmid%252F29681453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29681453&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.85"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chevée</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D. J.</span> <span class="cit-name-surname">Robertson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. H.</span> <span class="cit-name-surname">Cannon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Goff</span></span>, <span class="cit-article-title">Variation in Activity State, Axonal Projection, and Position Define the Transcriptional Identity of Individual Neocortical Projection Neurons</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">441</span>–<span class="cit-lpage">455</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D22%26rft.spage%253D441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.86" data-doi="10.1038/nn.4462"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Romanov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zeisel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bakker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Girach</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hellysaz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tomer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Alp</span></span>ár, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mulder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Clotman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Keimpema</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hsueh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Crow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Martens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schwindling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Calvigioni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Bains</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Máté</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Szabó</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yanagawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rendeiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farlik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Uhlén</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wulff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Broberger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Deisseroth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hökfelt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Horvath</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Harkany</span></span>, <span class="cit-article-title">Molecular interrogation of hypothalamic organization reveals distinct dopamine neuronal subtypes</span>. <abbr class="cit-jnl-abbrev">Nature Neuroscience</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">176</span>–<span class="cit-lpage">188</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D20%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4462%26rft_id%253Dinfo%253Apmid%252F27991900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.4462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27991900&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.87" data-doi="10.1038/nn.4376"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Furlan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">La Manno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lubke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Haring</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Abdo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hochgerner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kupari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Usoskin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Airaksinen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Oliver</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ernfors</span></span>, <span class="cit-article-title">Visceral motor neuron diversity delineates a cellular basis for nipple- and pilo-erection muscle control</span>. <abbr class="cit-jnl-abbrev">Nature Neuroscience</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1331</span>–<span class="cit-lpage">1340</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D19%26rft.spage%253D1331%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.4376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.88" data-doi="10.1038/nn.3881"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Usoskin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Furlan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Islam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Abdo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lönnerberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hjerling-Leffler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Haeggströ</span></span>m, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kharchenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. V.</span> <span class="cit-name-surname">Kharchenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ernfors</span></span>, <span class="cit-article-title">Unbiased classification of sensory neuron types by large-scale single-cell RNA sequencing</span>. <abbr class="cit-jnl-abbrev">Nature Neuroscience</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNeuroscience%26rft.volume%253D18%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.3881%26rft_id%253Dinfo%253Apmid%252F25420068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.3881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420068&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.89" data-doi="10.1038/s41593-018-0141-1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Haring</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zeisel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hochgerner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rinwa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E. T.</span> <span class="cit-name-surname">Jakobsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lonnerberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">La Manno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Sharma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Borgius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kiehn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Lagerstrom</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linnarsson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ernfors</span></span>, <span class="cit-article-title">Neuronal atlas of the dorsal horn defines its architecture and links sensory input to transcriptional cell types</span>. <abbr class="cit-jnl-abbrev">Nat Neurosci</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">869</span>–<span class="cit-lpage">880</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BNeurosci%26rft.volume%253D21%26rft.spage%253D869%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-018-0141-1%26rft_id%253Dinfo%253Apmid%252F29686262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41593-018-0141-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29686262&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.009084v3.90" data-doi="10.1186/s13059-016-0947-7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. T.</span> <span class="cit-name-surname">Lun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bach</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Marioni</span></span>, <span class="cit-article-title">Pooling across cells to normalize single-cell RNA sequencing data with many zero counts</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">75</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D17%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0947-7%26rft_id%253Dinfo%253Apmid%252F27122128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0947-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27122128&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.25.009084.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.25.009084v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.28.013789v3" title="Structural analysis of SARS-CoV-2 and predictions of the human interactome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.17.995795v2" title="Scientometric Trends for Coronaviruses and Other Emerging Viral Infections" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.25.009084v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/04/09/2020.03.25.009084.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.25.009084v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.25.009084v3" class="active" data-icon-position="" data-hide-link-title="0">Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1237315" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-la3zXeamBDfNi0wmkqvIg7-tjSQfq7QikfNVLi2ZLeo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237315" id="node1237315--2" data-pisa="biorxiv;2020.03.25.009084v3" data-pisa-master="biorxiv;2020.03.25.009084" data-seqnum="1237315" data-apath="/biorxiv/early/2020/04/09/2020.03.25.009084.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">David H.</span> <span class="nlm-surname">Brann</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Tatsuya</span> <span class="nlm-surname">Tsukahara</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Weinreb</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Marcela</span> <span class="nlm-surname">Lipovsek</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Koen</span> <span class="nlm-surname">Van den Berge</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Boying</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Chance</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Iain C.</span> <span class="nlm-surname">Macaulay</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Hsin-jung</span> <span class="nlm-surname">Chou</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Russell</span> <span class="nlm-surname">Fletcher</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Diya</span> <span class="nlm-surname">Das</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Kelly</span> <span class="nlm-surname">Street</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Hector Roux</span> <span class="nlm-surname">de Bezieux</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yoon-Gi</span> <span class="nlm-surname">Choi</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Davide</span> <span class="nlm-surname">Risso</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Sandrine</span> <span class="nlm-surname">Dudoit</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Elizabeth</span> <span class="nlm-surname">Purdom</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Jonathan S.</span> <span class="nlm-surname">Mill</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Ralph Abi</span> <span class="nlm-surname">Hachem</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Hiroaki</span> <span class="nlm-surname">Matsunami</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Darren W.</span> <span class="nlm-surname">Logan</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">Goldstein</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Grubb</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">John</span> <span class="nlm-surname">Ngai</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Sandeep Robert</span> <span class="nlm-surname">Datta</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.009084; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.009084 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;title=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;title=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;text=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;title=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;t=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;title=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;title=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237315" id="node1237315--3" data-pisa="biorxiv;2020.03.25.009084v3" data-pisa-master="biorxiv;2020.03.25.009084" data-seqnum="1237315" data-apath="/biorxiv/early/2020/04/09/2020.03.25.009084.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">David H.</span> <span class="nlm-surname">Brann</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Tatsuya</span> <span class="nlm-surname">Tsukahara</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Weinreb</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Marcela</span> <span class="nlm-surname">Lipovsek</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Koen</span> <span class="nlm-surname">Van den Berge</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Boying</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Chance</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Iain C.</span> <span class="nlm-surname">Macaulay</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Hsin-jung</span> <span class="nlm-surname">Chou</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Russell</span> <span class="nlm-surname">Fletcher</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Diya</span> <span class="nlm-surname">Das</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Kelly</span> <span class="nlm-surname">Street</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Hector Roux</span> <span class="nlm-surname">de Bezieux</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yoon-Gi</span> <span class="nlm-surname">Choi</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Davide</span> <span class="nlm-surname">Risso</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Sandrine</span> <span class="nlm-surname">Dudoit</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Elizabeth</span> <span class="nlm-surname">Purdom</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Jonathan S.</span> <span class="nlm-surname">Mill</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Ralph Abi</span> <span class="nlm-surname">Hachem</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Hiroaki</span> <span class="nlm-surname">Matsunami</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Darren W.</span> <span class="nlm-surname">Logan</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Bradley J.</span> <span class="nlm-surname">Goldstein</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Grubb</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">John</span> <span class="nlm-surname">Ngai</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Sandeep Robert</span> <span class="nlm-surname">Datta</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.009084; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.009084 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Brann%2C+David+H.&amp;rft.au=Tsukahara%2C+Tatsuya&amp;rft.au=Weinreb%2C+Caleb&amp;rft.au=Lipovsek%2C+Marcela&amp;rft.au=Van+den+Berge%2C+Koen&amp;rft.au=Gong%2C+Boying&amp;rft.au=Chance%2C+Rebecca&amp;rft.au=Macaulay%2C+Iain+C.&amp;rft.au=Chou%2C+Hsin-jung&amp;rft.au=Fletcher%2C+Russell&amp;rft.au=Das%2C+Diya&amp;rft.au=Street%2C+Kelly&amp;rft.au=de+Bezieux%2C+Hector+Roux&amp;rft.au=Choi%2C+Yoon-Gi&amp;rft.au=Risso%2C+Davide&amp;rft.au=Dudoit%2C+Sandrine&amp;rft.au=Purdom%2C+Elizabeth&amp;rft.au=Mill%2C+Jonathan+S.&amp;rft.au=Hachem%2C+Ralph+Abi&amp;rft.au=Matsunami%2C+Hiroaki&amp;rft.au=Logan%2C+Darren+W.&amp;rft.au=Goldstein%2C+Bradley+J.&amp;rft.au=Grubb%2C+Matthew+S.&amp;rft.au=Ngai%2C+John&amp;rft.au=Datta%2C+Sandeep+Robert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1237315/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1237315/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1237315/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1237315/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1237315/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1237315/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1237315/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1237315/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1237315/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1237315/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1237315/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1237315/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;count=horizontal&amp;via=&amp;text=Non-neuronal%20expression%20of%20SARS-CoV-2%20entry%20genes%20in%20the%20olfactory%20system%20suggests%20mechanisms%20underlying%20COVID-19-associated%20anosmia&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.009084v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.009084v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">Neuroscience<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
