<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/31/2020.01.30.926881.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.30.926881v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes" />
<meta name="DC.Identifier" content="10.1101/2020.01.30.926881" />
<meta name="DC.Date" content="2020-01-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Deubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered." />
<meta name="DC.Contributor" content="Wioletta Rut" />
<meta name="DC.Contributor" content="Mikołaj Żmudziński" />
<meta name="DC.Contributor" content="Scott J. Snipas" />
<meta name="DC.Contributor" content="Miklos Bekes" />
<meta name="DC.Contributor" content="Tony T. Huang" />
<meta name="DC.Contributor" content="Marcin Drag" />
<meta name="article:published_time" content="2020-01-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Deubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.30.926881v1" />
<meta name="citation_id" content="2020.01.30.926881v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.30.926881" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.30.926881" />
<meta name="citation_author" content="Wioletta Rut" />
<meta name="citation_author_institution" content="Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology" />
<meta name="citation_author_email" content="wioletta.rut@pwr.edu.pl" />
<meta name="citation_author_email" content="marcin.drag@pwr.edu.pl" />
<meta name="citation_author" content="Mikołaj Żmudziński" />
<meta name="citation_author_institution" content="Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology" />
<meta name="citation_author" content="Scott J. Snipas" />
<meta name="citation_author_institution" content="Sanford Burnham Prebys Medical Discovery Institute, 10901 North Torrey Pines Road" />
<meta name="citation_author" content="Miklos Bekes" />
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Pharmacology, New York University School of Medicine" />
<meta name="citation_author_institution" content="Arvinas, Inc." />
<meta name="citation_author" content="Tony T. Huang" />
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Pharmacology, New York University School of Medicine" />
<meta name="citation_author" content="Marcin Drag" />
<meta name="citation_author_institution" content="Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology" />
<meta name="citation_author_institution" content="Sanford Burnham Prebys Medical Discovery Institute, 10901 North Torrey Pines Road" />
<meta name="citation_author_email" content="wioletta.rut@pwr.edu.pl" />
<meta name="citation_author_email" content="marcin.drag@pwr.edu.pl" />
<meta name="citation_reference" content="Eletr, Z.M. and K.D. Wilkinson, Regulation of proteolysis by human deubiquitinating enzymes. Biochim Biophys Acta, 2014. 1843(1): p. 114–28." />
<meta name="citation_reference" content="Amerik, A.Y. and M. Hochstrasser, Mechanism and function of deubiquitinating enzymes. Biochim Biophys Acta, 2004. 1695(1-3): p. 189–207." />
<meta name="citation_reference" content="Mevissen, T.E., et al., OTU deubiquitinases reveal mechanisms of linkage specificity and enable ubiquitin chain restriction analysis. Cell, 2013. 154(1): p. 169–84." />
<meta name="citation_reference" content="Mielech, A.M., et al., MERS-CoV papain-like protease has deISGylating and deubiquitinating activities. Virology, 2014. 450-451: p. 64–70." />
<meta name="citation_reference" content="Lindner, H.A., et al., The papain-like protease from the severe acute respiratory syndrome coronavirus is a deubiquitinating enzyme. J Virol, 2005. 79(24): p. 15199–208." />
<meta name="citation_reference" content="Li, Y.T., et al., Ubiquitin 7-amino-4-carbamoylmethylcoumarin as an improved fluorogenic substrate for deubiquitinating enzymes. Tetrahedron, 2016. 72(27-28): p. 4085–4090." />
<meta name="citation_reference" content="Dang, L.C., F.D. Melandri, and R.L. Stein, Kinetic and mechanistic studies on the hydrolysis of ubiquitin C-terminal 7-amido-4-methylcoumarin by deubiquitinating enzymes. Biochemistry, 1998. 37(7): p. 1868–1879." />
<meta name="citation_reference" content="Hassiepen, U., et al., A sensitive fluorescence intensity assay for deubiquitinating proteases using ubiquitin-rhodamine 110-glycine as substrate. Analytical Biochemistry, 2007. 371(2): p. 201–207." />
<meta name="citation_reference" content="Love, K.R., et al., Mechanisms, biology and inhibitors of deubiquitinating enzymes. Nature Chemical Biology, 2007. 3(11): p. 697–705." />
<meta name="citation_reference" content="Reyes-Turcu, F.E., K.H. Ventii, and K.D. Wilkinson, Regulation and Cellular Roles of Ubiquitin-Specific Deubiquitinating Enzymes. Annual Review of Biochemistry, 2009. 78: p. 363–397." />
<meta name="citation_reference" content="Borodovsky, A., et al., Chemistry-based functional proteomics reveals novel members of the deubiquitinating enzyme. Chemistry &amp; Biology, 2002. 9(10): p. 1149–1159." />
<meta name="citation_reference" content="Hewings, D.S., et al., Activity-based probes for the ubiquitin conjugation-deconjugation machinery: new chemistries, new tools, and new insights. Febs Journal, 2017. 284(10): p. 1555–1576." />
<meta name="citation_reference" content="Ronau, J.A., J.F. Beckmann, and M. Hochstrasser, Substrate specificity of the ubiquitin and Ubl proteases. Cell Res, 2016. 26(4): p. 441–56." />
<meta name="citation_reference" content="Harrigan, J.A., et al., Deubiquitylating enzymes and drug discovery: emerging opportunities. Nat Rev Drug Discov, 2018. 17(1): p. 57–78." />
<meta name="citation_reference" content="Iphofer, A., et al., Profiling ubiquitin linkage specificities of deubiquitinating enzymes with branched ubiquitin isopeptide probes. Chembiochem, 2012. 13(10): p. 1416–20." />
<meta name="citation_reference" content="Li, G.R., et al., Activity-based diubiquitin probes for elucidating the linkage specificity of deubiquitinating enzymes. Chemical Communications, 2014. 50(2): p. 216–218." />
<meta name="citation_reference" content="Flierman, D., et al., Non-hydrolyzable Diubiquitin Probes Reveal Linkage-Specific Reactivity of Deubiquitylating Enzymes Mediated by S2 Pockets. Cell Chemical Biology, 2016. 23(4): p. 472–482." />
<meta name="citation_reference" content="Bekes, M., et al., SARS hCoV papain-like protease is a unique Lys(48) linkage-specific di-distributive deubiquitinating enzyme. Biochemical Journal, 2015. 468: p. 215–226." />
<meta name="citation_reference" content="Bekes, M., et al., Recognition of Lys48-Linked Di-ubiquitin and Deubiquitinating Activities of the SARS Coronavirus Papain-like Protease. Molecular Cell, 2016. 62(4): p. 572–585." />
<meta name="citation_reference" content="Gjonaj, L., et al., USP7: combining tools towards selectivity. Chemical Communications, 2019. 55(35): p. 5075–5078." />
<meta name="citation_reference" content="Gjonaj, L., et al., Development of a DUB-selective fluorogenic substrate. Chemical Science, 2019. 10(44): p. 10290–10296." />
<meta name="citation_reference" content="Drag, M., et al., Positional-scanning fluorigenic substrate libraries reveal unexpected specificity determinants of DUBs (deubiquitinating enzymes). Biochemical Journal, 2008. 415: p. 367–375." />
<meta name="citation_reference" content="Poreba, M., et al., Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes. Cell Chemical Biology, 2016. 23(8): p. 1023–1035." />
<meta name="citation_reference" content="Poreba, M., et al., Selective imaging of cathepsin L in breast cancer by fluorescent activity-based probes. Chemical Science, 2018. 9(8): p. 2113–2129." />
<meta name="citation_reference" content="Kasperkiewicz, P., et al., Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils. Journal of the American Chemical Society, 2017. 139(29): p. 10115–10125." />
<meta name="citation_reference" content="Rut, W., et al., Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples. Journal of Medicinal Chemistry, 2018. 61(12): p. 5222–5234." />
<meta name="citation_reference" content="Song, Z.W., et al., A novel UCHL3 inhibitor, perifosine, enhances PARP inhibitor cytotoxicity through inhibition of homologous recombination-mediated DNA double strand break repair. Cell Death &amp; Disease, 2019. 10." />
<meta name="citation_reference" content="Song, Z.W., et al., UCHL3 promotes pancreatic cancer progression and chemo-resistance through FOXM1 stabilization. American Journal of Cancer Research, 2019. 9(9): p. 1970–1981." />
<meta name="citation_reference" content="Maly, D.J., et al., Expedient solid-phase synthesis of fluorogenic protease substrates using the 7-amino-4-carbamoylmethylcoumarin (ACC) fluorophore. Journal of Organic Chemistry, 2002. 67(3): p. 910–915." />
<meta name="citation_reference" content="Poreba, M., G.S. Salvesen, and M. Drag, Synthesis of a HyCoSuL peptide substrate library to dissect protease substrate specificity. Nature Protocols, 2017. 12(10): p. 2189–2214." />
<meta name="citation_reference" content="Poreba, M., et al., Positional scanning substrate combinatorial library (PS-SCL) approach to define caspase substrate specificity. Methods Mol Biol, 2014. 1133: p. 41–59." />
<meta name="citation_reference" content="El Oualid, F., et al., Chemical synthesis of ubiquitin, ubiquitin-based probes, and diubiquitin. Angew Chem Int Ed Engl, 2010. 49(52): p. 10149–53." />
<meta name="citation_reference" content="de Jong, A., et al., Ubiquitin-based probes prepared by total synthesis to profile the activity of deubiquitinating enzymes. Chembiochem, 2012. 13(15): p. 2251–8." />
<meta name="citation_reference" content="Liang, J., et al., Chemical synthesis of Ub-AMC via ligation of peptide hydrazides. Science China-Chemistry, 2013. 56(9): p. 1301–1306." />
<meta name="citation_reference" content="Huang, Y.C., et al., Facile synthesis of C-terminal peptide hydrazide and thioester of NY-ESO-1 (A39-A68) from an Fmoc-hydrazine 2-chlorotrityl chloride resin. Tetrahedron, 2014. 70(18): p. 2951–2955." />
<meta name="citation_reference" content="Zheng, J.S., et al., Chemical synthesis of proteins using peptide hydrazides as thioester surrogates. Nat Protoc, 2013. 8(12): p. 2483–95." />
<meta name="citation_reference" content="Vedrenne, E., et al., Dramatic effect of boron-based Lewis acids in cross-metathesis reactions. Synlett, 2005(4): p. 670–672." />
<meta name="citation_reference" content="D’Arcy, P., X. Wang, and S. Linder, Deubiquitinase inhibition as a cancer therapeutic strategy. Pharmacology &amp; Therapeutics, 2015. 147: p. 32–54." />
<meta name="citation_reference" content="Davis, M.I. and A. Simeonov, Ubiquitin-Specific Proteases as Druggable Targets. Drug Target Rev, 2015. 2(3): p. 60–64." />
<meta name="citation_reference" content="Young, M.J., et al., The role of ubiquitin-specific peptidases in cancer progression. J Biomed Sci, 2019. 26(1): p. 42." />
<meta name="citation_reference" content="Love, K.R., et al., Ubiquitin C-terminal electrophiles are activity-based probes for identification and mechanistic study of ubiquitin conjugating machinery. ACS Chem Biol, 2009. 4(4): p. 275–87." />
<meta name="citation_reference" content="Borodovsky, A., et al., Small-molecule inhibitors and probes for ubiquitin- and ubiquitin-like-specific proteases. Chembiochem, 2005. 6(2): p. 287–91." />
<meta name="citation_reference" content="Soboleva, T.A. and R.T. Baker, Deubiquitinating enzymes: Their functions and substrate specificity. Current Protein &amp; Peptide Science, 2004. 5(3): p. 191–200." />
<meta name="twitter:title" content="Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Deubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered." />
<meta name="og-title" property="og:title" content="Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Deubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.30.926881v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.30.926881v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.30.926881v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1127948" />
    <title>Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OhIpm3CIwsAJemeO3ulBkj97Z6YaxtLlNRSv7ORNFfo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1127948","apath":"\/biorxiv\/early\/2020\/01\/31\/2020.01.30.926881.atom","pisa":"biorxiv;2020.01.30.926881v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.30.926881v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.30.926881v1","title":"Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes","identifier":"node\/1127948"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ifZu9+nwPVirk74st3SR0ahfDwx5i7\\\\\\\/JRmk2nYIRbKZaqFvTZWTOr6BRpkeNxgP1RY3vR\\\\\\\/pWUf+yQEqQE67y7eROcbYmDeQWEcFYB0t3\\\\\\\/gtouwgSKN1zS8LVSbk6M0MHMFyXOZvFfZJ6Q8BnETYxao9UTIIe6eTXWC+NWWUTRAG2Xh32evROV7c8jBg9ncsG99VAVmYM9J\\\\\\\/Ajyk8z\\\\\\\/AIQ3iiCGqg3l+2VTemLJQgvss4MWmDv3B9LZF7uMJUk4SN37RlfdFJq7+29yFEwbiraokOXSVjIkM52JBDqDbFuu\\\\\\\/8SPLx1HVzZ0XsxbNCob2LVz6Ok1vjc\\\\\\\/ZpxtD+NmwJlJnqieV6s4UvfgXlC+SUQ8rat4GnwPRre6y9ikzUjpAR91qmaMqbzViRzHDl65sDGsI5ft1\\\\\\\/FcIG8LBNN8I9ciCM8kqaV0le7GWpx00Spy9eomabmLTQsH1FNcQeqJyJln8ZaSq3ZGxwRlIHmM\\\\\\\/gZNT0YV5rk9BBF5Sx68SMVlkGgpZpf7JydvRuu4neU\\\\\\\/re3xfgtCQUb7Y9TstqFm0\\\\\\\/X9FhddokibuWMu7hJf0kjy0\\\\\\\/uD38KGq9a4WsJTZxNNPZVI99hVYzsizurNNDqxDqqBVSyV4O7K8sz3lP1sjeoMJn0j2uZlp\\\\\\\/SVl5rd0uW4FxWLIn4DvWstCFsLC8Mb+NiobnJaZkLUoJQ1dVq2a3dWSBRNLB0K9ZOm1DL94VpAymMtbovbQw4b6ozv308vkuPzZVjSI51VvNhcdkSPB8lD+IUHwni8nONOfjQd\\\\\\\/LtW4dKFgzm9Lq1fr3rJuxktoP+T\\\\\\\/\\\\\\\/Nve9AJCIZoEKKP+3LeufrkvCl84ljPwvcMgCKCLShSqmqEqGCxzySjRPgOBQUyQt27hJRY1IQFkQoTnh+rtzUwdjG1d2J\\\\\\\/pvpEclw8YRUpszDbDyQ14UOn0yez3V7fXmVqE5tWkYQrGOYvlX\\\\\\\/X5x3RGLztwE6y3XjkJ\\\\\\\/wx70oASGEwh+Gr83Mjhvuo7uwqAAc17Lvl4VTXUb\\\\\\\/2OqUPouIUcHkLld0B3vRh\\\\\\\/qfJx+Ry19wyokH\\\\\\\/Z8amkGt8ssopX4GpwmB+6QuyhjJJv8NIC\\\\\\\/isGt3IE1Rg2jthSDDSdHLDHkdu5IlSlyHMksOHVdqkS+MnMA31fXJVRUUCav27EdYTVN3mYcWrskQJRMD6+Am0z1DrCqHYavjQ2cFz4zfp4mk9sXi3usPHnPD0ek3+g0fl1Y61KPHq8TfglFrCrBXg+k2\\\\\\\/MMPIEYlXYEK5bGl9Pv5D+acJxyoZp0HHjgrWuACgpDvXqQlRpYGfrGXOJBc13ZlsaUXE70Cop1xDkN6\\\\\\\/OxJratTxYIiIc0AmeShwpy\\\u0022,\\\u0022iv\\\u0022:\\\u0022MBObI1z5UtzyJaJ6gX4hXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022a44b935abab892d4aea2b0bb604fd396c8fed0214ed549caffd9083ff079a889\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.30.926881v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1127948 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.30.926881v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vJd6T5i0TMoQ1sOONf_OW94IT_B3rGyKo7QO1muGpXE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127948" id="node1127948" data-pisa="biorxiv;2020.01.30.926881v1" data-pisa-master="biorxiv;2020.01.30.926881" data-apath="/biorxiv/early/2020/01/31/2020.01.30.926881.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wioletta</span> <span class="nlm-surname">Rut</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mikołaj</span> <span class="nlm-surname">Żmudziński</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott J.</span> <span class="nlm-surname">Snipas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miklos</span> <span class="nlm-surname">Bekes</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Tony T.</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Marcin</span> <span class="nlm-surname">Drag</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.30.926881 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1127948" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wioletta Rut </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology</span>, Wyb. Wyspianskiego 27, 50-370 Wroclaw, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wioletta%2BRut%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rut%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWioletta%2BRut%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wioletta.rut@pwr.edu.pl" class="" data-icon-position="" data-hide-link-title="0">wioletta.rut@pwr.edu.pl</a>
<a href="mailto:marcin.drag@pwr.edu.pl" class="" data-icon-position="" data-hide-link-title="0">marcin.drag@pwr.edu.pl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mikołaj Żmudziński </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology</span>, Wyb. Wyspianskiego 27, 50-370 Wroclaw, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miko%C5%82aj%2B%C5%BBmudzi%C5%84ski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C5%BBmudzi%C5%84ski%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiko%25C5%2582aj%2B%25C5%25BBmudzi%25C5%2584ski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Scott J. Snipas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Sanford Burnham Prebys Medical Discovery Institute, 10901 North Torrey Pines Road</span>, La Jolla, CA 92037, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BJ.%2BSnipas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snipas%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BJ.%2BSnipas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miklos Bekes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry &#38; Molecular Pharmacology, New York University School of Medicine</span>, New York, NY 10016, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Arvinas, Inc.</span>, 5 Science Park, New Haven, CT, 06511, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miklos%2BBekes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bekes%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiklos%2BBekes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tony T. Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry &#38; Molecular Pharmacology, New York University School of Medicine</span>, New York, NY 10016, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tony%2BT.%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20TT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATony%2BT.%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marcin Drag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology</span>, Wyb. Wyspianskiego 27, 50-370 Wroclaw, <span class='nlm-country'>Poland</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Sanford Burnham Prebys Medical Discovery Institute, 10901 North Torrey Pines Road</span>, La Jolla, CA 92037, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcin%2BDrag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Drag%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcin%2BDrag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wioletta.rut@pwr.edu.pl" class="" data-icon-position="" data-hide-link-title="0">wioletta.rut@pwr.edu.pl</a>
<a href="mailto:marcin.drag@pwr.edu.pl" class="" data-icon-position="" data-hide-link-title="0">marcin.drag@pwr.edu.pl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.30.926881v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.30.926881v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.30.926881v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.30.926881v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.30.926881v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.30.926881v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1127948" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1127948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Deubiquitinating enzymes (DUBs) are responsible for removing ubiquitin (Ub) from its protein conjugates. DUBs have been implicated as attractive therapeutic targets in the treatment of viral diseases, neurodegenerative disorders and cancer. The lack of selective chemical tools for the exploration of these enzymes significantly impairs the determination of their roles in both normal and pathological states. Commercially available fluorogenic substrates are based on the C-terminal Ub motif or contain Ub coupled to a fluorophore (Z-LRGG-AMC, Ub-AMC); therefore, these substrates suffer from lack of selectivity. By using a hybrid combinatorial substrate library (HyCoSuL) and a defined P2 library containing a wide variety of nonproteinogenic amino acids, we established a full substrate specificity profile for two DUBs—MERS PLpro and human UCH-L3. Based on these results, we designed and synthesized Ub-based substrates and activity-based probes (ABPs) containing selected unnatural amino acids located in the C-terminal Ub motif. Biochemical analysis and cell-based experiments confirmed the activity and selectivity of engineered Ub-based substrates and probes. Using this approach, we propose that for any protease that recognizes Ub and Ub-like substrates, a highly active and selective unnatural substrate or probe can be engineered.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-3">Ubiquitin (Ub) conjugation is one of the most important posttranslational modifications and influences the activity, protein-protein interactions, localization and stability of many proteins [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Attachment of Ub to protein substrates is a covalent modification mediated by the E1, E2 and E3 enzymes. However, due to the activity of deubiquitinating enzymes (DUBs), this modification is reversible. DUBs hydrolyze the isopeptide bond between the C-terminal glycine of Ub and its substrate or between Ubs in poly-Ub chains with various topologies [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The human genome encodes approximately 100 DUBs divided into six subclasses based on sequence similarity and mechanism of action. The majority of DUBs (five of the six subclasses) are cysteine proteases, and only one subclass is represented by a small group of metalloproteases. The largest and most structurally diverse subclass of DUBs are Ub-specific proteases (USPs). These enzymes can hydrolyze a peptide/isopeptide bond between the C-terminus of Ub and its substrate, as well as within the poly-Ub chain. Another subclass of DUBs that belongs to cysteine proteases are the Ub carboxy-terminal hydrolase (UCH) enzymes responsible for the hydrolysis of Ub precursors, leading to the formation of Ub monomers. The other subclasses of cysteine proteases are the ovarian tumor (OTU), Machado-Josephine domain (MJD), and motif-interacting with Ub-containing novel DUB family (MINDY) proteases and one metalloprotease subgroup, known as the JAMM/MPN+ DUBs. Members of the OTU family exhibit unique specificity for cleavage of different poly-Ub chain topologies [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. Furthermore, enzymes displaying deubiquitinating activity are encoded in coronavirus genomes, such as Middle East respiratory syndrome papain-like protease (MERS PLpro) [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>] or severe acute respiratory syndrome papain-like protease (SARS PLpro) [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. PLpro enzymes are involved in negative regulation of the innate immune response during viral infection [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>].</p><p id="p-4">Since DUBs are highly specific toward Ub, the most commonly used DUB assay reagents are based on Ub with a C-terminal fluorogenic group (Ub-ACC [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], Ub-AMC [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], Ub-rhodamine [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]) or an electrophilic warhead (Ub-CHO, Ub-VS, Ub-vinyl methyl ester (VME) [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]). Application of these chemical tools in biological studies has led to a better understanding of the mechanisms of action of many DUBs. Ub with a C-terminal aldehyde group has been frequently used in crystallographic studies of DUBs [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Development of Ub-based probes by introducing a detection tag on the N-terminus and a reactive group on the C-terminus of Ub enabled labeling of DUBs in cell lysates and their detection via Western blot analysis. Ub-based probes are powerful tools for characterizing the activity of DUBs, identifying new DUBs, evaluating the selectivity of DUB inhibitors, and determining the functional roles of DUBs in normal and pathological states [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. However, the use of mono-Ub probes precluded studies of DUB activity and linkage specificity. To overcome this limitation, di-Ub probes were developed [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Studies with di-Ub probes revealed differences between the S1-S1’ and S1-S2 selectivity of enzymes from the OTU subclass [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], as well as the Lys48 linkage specificity of SARS PLpro [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. Despite the wide utility of broad-spectrum DUB substrates and probes in biochemical assays, selective chemical tools to study a single DUB (or a narrow subset of DUBs) are needed. Recently, the Ovaa group developed Ub-based probes selective toward USP7 [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] and USP16 [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] by combining structural analysis, modeling and mutational predictions. Selectivity can also be achieved by modifying the Ub C-terminal conserved LRGG motif [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Determination of DUB substrate preferences in the P4-P2 positions using the positional scanning substrate combinatorial library (PS-SCL) approach revealed that DUBs can recognize amino acids other than the canonical LRG in these positions. Furthermore, this study shows that a Ub mutant with C-terminal extension, in which glycine in the P2 position was replaced by valine, was cleaved by UCH-L3 20-50-fold less efficiently than its wild-type analog [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>]. However, this mutation provided selectivity, as the mutant was not recognized by another tested DUB USP5 (IsoT). In the present study, to further improve Ub mutant selectivity, as well as DUB activity, we decided to utilize unnatural amino acids. A similar approach was successfully applied to design selective substrates and activity-based probes (ABPs) for cysteine [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], serine [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] and threonine proteases [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] as short peptide sequences but never incorporated into whole substrate proteins, such as Ub.</p><p id="p-5">In this work, we developed active and selective chemical tools based on Ub scaffolds for DUB investigation by incorporating unnatural amino acids into the conserved C-terminal Ub motif. We selected two DUBs, namely, UCH-L3 and MERS PLpro, to validate our chemical approach. These enzymes represent potential therapeutic targets in drug design [<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. First, we determined full substrate specificity profiles of the selected DUBs at the P4-P2 positions using defined and hybrid combinatorial libraries of tetrapeptide fluorogenic substrates. Substrate preference analysis allowed us to extract key amino acids that could serve as selectivity-providing motifs in Ub-based substrates and ABPs. The designed tetrapeptide fluorogenic substrates containing unnatural amino acids were selective toward UCH-L3 and MERS PLpro but still not efficiently hydrolyzed. To overcome this obstacle, we synthesized Ub derivatives containing a C-terminal epitope substituted with selected unnatural peptide sequences instead of a canonical LRGG motif. Substrates and covalent probes were obtained by conjugation of the fluorescent label (ACC) or Michael-acceptor warhead (VME), respectively (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). Kinetic analysis revealed that the designed substrates were more efficiently hydrolyzed than the reference “wild-type” substrate Ub-ACC. Moreover, the substrates exhibited high selectivity toward UCH-L3 and MERS PLpro. The selectivity of the Ub-based probe was evaluated using purified recombinant DUBs and cell lysates. The results of these studies revealed that engineered B-U1-VME and B-U2-VME probes detected UCH-L3 in cell lysates with high selectivity. With this <em>proof of concept</em>, we propose that this approach can be successfully applied in the design of Ub-based selective chemical tools for other proteases that recognize Ub and Ub-like substrates.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow applied in the design of DUB-selective substrates and ABPs (PDB ID: 1UBQ). " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1146129728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Workflow applied in the design of DUB-selective substrates and ABPs (PDB ID: 1UBQ).&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F1.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F1.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Workflow applied in the design of DUB-selective substrates and ABPs (PDB ID: 1UBQ).</span><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">2. Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Reagents</h3><p id="p-6">All chemical reagents were obtained from commercial suppliers and used without further purification. The reagents used for the solid-phase peptide synthesis (SPPS) were as follows: Rink Amide (RA) resin (particle size 100-200 mesh, loading 0.74 mmol/g), 2-chlorotrityl chloride resin (particle size 100-200 mesh, loading 0.97 mmol/g), all Fmoc-amino acids, <em>O</em>-benzotriazole-<em>N,N,N‘,N‘</em>-tetramethyl-uronium-hexafluoro-phosphate (HBTU), 2-(1-H-7-azabenzotriazol-1-yl)-1,1,3,3-tetramethyluranium hexafluorophosphate (HATU), piperidine, diisopropylcarbodiimide (DICI) and trifluoroacetic acid (TFA), purchased from Iris Bioteh GmbH (Marktredwitz, Germany); Fmoc-Asp(O<em>t</em>Bu)-(Dmb)Gly-OH and Fmoc-Ser(<em>t</em>Bu)-Thr(ψ<sup>Me,Me</sup>pro)-OH (dipeptide building blocks) from AAPPTec (Louisville, KY, USA); 9-fluorenylmethyl carbazate (Fmoc-NH-NH<sub>2</sub>), Tris(2-carboxyethyl)phosphine hydrochloride (TCEP) and 2,2’-azobis[2-(imidazolin-2-yl)propane] (VA-044) from Combi-Blocks (San Diego, CA, USA); 4-mercaptophenylacetic acid (MPAA), guanidine hydrochloride (Gn*HCl), sodium phosphate monobasic monohydrate (NaH<sub>2</sub>PO<sub>4</sub>*H<sub>2</sub>O) and 4.0 M HCl in dioxane; 2-methyl-2-propanethiol (<em>t</em>BuSH) from Sigma-Aldrich (Poznan, Poland); NaNO<sub>2</sub> from Merck Millipore (Warsaw, Poland); anhydrous <em>N</em>-hydroxybenzotriazole (HOBt) from Creosauls Louisville, KY, USA; 2,4,6-collidine (2,4,6-trimethylpyridine), HPLC-grade acetonitrile, and triisopropylsilane (TIPS) from Sigma-Aldrich (Poznan, Poland); and <em>N,N</em>-diisopropylethylamie (DIPEA) from VWR International (Gdansk, Poland). DMF <em>N,N</em>‘-dimethylformamide (DMF), dichloromethane (DCM), methanol (MeOH), diethyl ether (Et<sub>2</sub>O), acetic acid (AcOH), and phosphorus pentoxide (P<sub>2</sub>O<sub>5</sub>), obtained from Avantor (Gliwice, Poland). Individual substrates, Ub-based substrates and probes were purified by HPLC on a Waters M600 solvent delivery module with a Waters M2489 detector system using a semipreparative Wide Pore C8 Discovery column and Jupiter 10 µm C4 300 Å column (250 × 10 mm). The solvent composition was as follows: phase A (water/0.1% TFA) and phase B (acetonitrile/0.1% TFA). The purity of each compound was confirmed with an analytical HPLC system using a Jupiter 10 µm C4 300 Å column (250 × 4.6 mm). The solvent composition was as follows: phase A (water/0.1% TFA) and phase B (acetonitrile/0.1% TFA); gradient, from 5% B to 95% B over a period of 15 or 20 min. The molecular weight of each substrate and ABP was confirmed by high-resolution mass spectrometry on a High-Resolution Mass Spectrometer Waters LCT premier XE with electrospray ionization (ESI) and a time-of-flight (TOF) module.</p></div><div id="sec-4" class="subsection"><h3>Combinatorial substrate library synthesis</h3><p id="p-7">A combinatorial tetrapeptide fluorogenic substrate library was synthesized on a solid support according to published protocols [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. The library consisted of two tetrapeptide sublibraries. Each of the sublibraries was synthesized separately, and the general synthetic procedure is described for the P3 sublibrary as an example. In the first step, Fmoc-ACC-OH (25 mmol, 2.5 eq) was attached to the RA resin (13.5 g) using coupling reagents: HOBt (25 mmol, 2.5 eq) and DICI (25 mmol, 2.5 eq) in DMF. After 24 h, the Fmoc protecting group was removed with 20% piperidine in DMF. In the next step, Fmoc-Gly-OH (25 mmol, 2.5 eq) was coupled using HATU (25 mmol, 2.5 eq) and 2,4,6-collidine (25 mmol, 2.5 eq) in DMF. Then, Fmoc-Gly-OH (25 mmol, 2.5 eq) was attached to the H<sub>2</sub>N-Gly-ACC-resin using HOBt and DICI as coupling reagents (25 mmol, 2.5 eq). After glycine coupling, the Fmoc group was removed (20% piperidine in DMF), and the resin was washed with DCM and MeOH and dried over P<sub>2</sub>O<sub>5</sub>. Then, the dried resin was divided into 138 portions. To each portion of the H<sub>2</sub>N-Gly-Gly-ACC-resin, natural or unnatural amino acids were attached, and the Fmoc protecting group was removed (20% piperidine in DMF). To provide equimolar substitution of each natural amino acid in the P4 position, an isokinetic mixture of Fmoc-protected amino acids was utilized. The last two steps of P3 sublibrary synthesis included N-terminal acetylation (solution of AcOH, HBTU and DIPEA) and cleavage of peptides from the resin using a TFA:H<sub>2</sub>O:TIPS (95%:2.5%:2.5%, %v/v/v) mixture. Finally, the sublibrary was precipitated in Et<sub>2</sub>O, dissolved in a mixture of acetonitrile and water, lyophilized and dissolved in biochemical grade DMSO at a concentration of 20 mM. The obtained sublibrary was used for kinetic studies without further purification. The P4 sublibrary was synthesized in the same manner.</p></div><div id="sec-5" class="subsection"><h3>Synthesis of the defined P2 library and individual substrates</h3><p id="p-8">The defined P2 library (Ac-Leu-Arg-P2-Gly-ACC, where P2 represents 128 natural and unnatural amino acids) and individual tetrapeptide fluorogenic substrates were synthesized on a solid support using the SPPS method as previously described [<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. Each substrate was purified by HPLC and analyzed using analytical HPLC and HRMS. The purity of each compound was ≥95%. The P2 library and individual substrates were dissolved at 10 mM in DMSO and stored at -80°C until use.</p></div><div id="sec-6" class="subsection"><h3>Library and substrate screening</h3><p id="p-9">All screenings were performed using a spectrofluorometer (Molecular Devices Spectramax Gemini XPS) in 96-well plates (Corning). The release of ACC was monitored continuously for 40 min at the appropriate wavelength (λ<sub>ex</sub> = 355 nm, λ<sub>em</sub> = 460 nm). For the assay, 1 µL of substrate in DMSO was used with 99 µL of enzyme. The enzyme was incubated in assay buffer (MERS PLpro: 20 mM Tris, 150 mM NaCl, 5 mM DTT, pH 8.0 [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]; UCH-L3: 50 mM HEPES, 0.5 mM EDTA, 5 mM DTT, pH 7.5 [<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>]) for 30 min at 37°C before addition to the substrates on a plate. The final substrate concentration in each well during the assays was 200 µM for combinatorial P3 and P4 sublibraries, 100 µM for the Ac-Leu-Arg-P2-Gly-ACC library and 10 µM for individual substrates. The enzyme concentration was 2-3 µM for MERS PLpro and 6-20 µM for UCH-L3. Kinetic assays were repeated at least three times, and the results are presented as the mean values with standard deviations. The linear part of each progress curve was used to determine the substrate hydrolysis rate. Substrate specificity profiles were established by setting the highest value of relative fluorescence unit per second (RFU/s) from each library position as 100% and adjusting other values accordingly.</p></div><div id="sec-7" class="subsection"><h3>Synthesis of Ub derivatives</h3><p id="p-10">Currently, different methods can be applied to obtain Ub and poly-Ub derivatives [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>-<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. We modified existing strategies to develop a new, versatile approach that enables the synthesis of Ub-based substrates and ABPs containing unnatural amino acids. We employed this method to synthesize four different Ub-ACC substrates and four corresponding N-terminally biotinylated ABPs with a C-terminal VME electrophile. Each substrate and probe was synthesized by applying a peptide hydrazide ligation strategy. Peptide hydrazide segments were synthesized by SPPS, purified by HPLC and assembled together in a stepwise manner. Then, we performed radical desulfurization and conjugated H<sub>2</sub>N-Gly-ACC to obtain Ub-ACC substrates or H<sub>2</sub>N-Gly-VME to obtain biotin-6-Ahx-Ub-VME ABPs. Herein, we present a general protocol for Ub derivative synthesis based on the example of Ub-ACC. We started the synthesis with Fmoc-hydrazide resin preparation based on a modified published method [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. 2-CTC resin (0.97 mmol/g) was swelled for 20 min in fresh DCM and washed with 3x DCM. Fmoc-NH-NH<sub>2</sub> (4 eq) was dissolved in DMF:DCM (8/1, <em>v/v</em>), and DIPEA (10 eq) was added. The mixture was added to the resin. The reaction was carried out at room temperature. After 1 h, DIPEA (5 eq) was added. Fmoc-NH-NH<sub>2</sub> coupling was carried out for 4 h. The resin was washed with 3x DMF. End-capping of residual active 2-CTC resin was performed for 30 min by adding a capping mixture (DMF/MeOH/DIPEA, 3.5/0.5/0.2, <em>v/v/v</em>). After capping, Fmoc-NH-NH-2-CTC resin was washed with 3x DMF. Ub derivatives were assembled from 3 peptide hydrazide segments: Ub[<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>]-NH-NH<sub>2</sub>, Ub[<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>-45]-(A<sup>28</sup>C)-NH-NH<sub>2</sub> and Ub[46-75]-(A<sup>46</sup>C)-NH-NH<sub>2</sub>. During the synthesis of ABPs, the Biot-6-Ahx-Ub[<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>]-NH-NH<sub>2</sub> segment was used instead of Ub[<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a>]-NH-NH<sub>2</sub>. Naturally occurring Ala residues on the N-termini of Ub[<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>-45]-(A<sup>28</sup>C)-NH-NH<sub>2</sub> and Ub[46-75]-(A<sup>46</sup>C)-NH-NH<sub>2</sub> were exchanged to Cys residues to enable subsequent native chemical ligation (NCL) reactions. Segments were synthesized manually in a straightforward manner using Fmoc-protected amino acids. The synthesis of Ub[46-75]-(A<sup>46</sup>C)-NH-NH<sub>2</sub> required the use of dipeptide building blocks at Asp<sup>52</sup>Gly<sup>53</sup> (Fmoc-Asp(O<em>t</em>Bu)- (Dmb)Gly-OH) and Ser<sup>65</sup>Thr<sup>66</sup> (Fmoc-Ser(<em>t</em>Bu)-Thr(Ψ<sup>Me,Me</sup>pro)-OH), as reported elsewhere [<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>]. Fmoc-based SPPS of peptide hydrazides was performed in glass peptide synthesis vessels. Fmoc-NH-NH-2-CTC resin was swelled for 20 min in DCM, deprotected with 20% piperidine in DMF (two cycles: 5 and 25 min) and washed with 7x DMF. The first amino acid (2.5 eq) was dissolved in DMF with coupling reagents (2.5 eq HATU and 2.5 eq 2,4,6-collidine). The amino acid was preactivated for 30 s and added to the resin. The mixture was gently agitated for 24 h. After coupling, the resin was washed with 3x DMF, and the Fmoc groups were removed as described previously. A solution of the next amino acid (2.5 eq) with HATU (2.5 eq) and 2,4,6-collidine (2.5 eq) in DMF was added to the resin. After complete coupling of the second amino acid, Fmoc deprotection was carried out, and the following amino acids or dipeptide building blocks were coupled with the same coupling-deprotection cycles. The progress of the coupling and Fmoc deprotection of primary amine groups was monitored with the ninhydrin test. If needed, double coupling was performed. Biotin in the Biot-6-Ahx-Ub[<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a>]-NH-NH<sub>2</sub> segment was coupled in DMF/DMSO (1:1, <em>v/v</em>). After the last coupling step, the resin was washed with 3x DMF, 3x DCM, and 3x MeOH and dried over P<sub>2</sub>O<sub>5</sub> overnight. A cleavage cocktail (8 mL, TFA/H<sub>2</sub>O/TIPS, 95/2.5/2.5, %<em>v/v/v</em>) was added to the dry resin. After 2 h, the cleavage mixture was filtered from the resin and collected. Peptides were precipitated with cold Et<sub>2</sub>O and dried. Crude peptide hydrazides were purified by preparative HPLC and lyophilized. NCL reactions were performed based on a published protocol [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Ub[<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a>]-NH-NH<sub>2</sub> (7.13 µmol) was dissolved in ligation buffer (1.5 mL, 6 M Gn*HCl, 0.2 M NaH<sub>2</sub>PO<sub>4</sub>, pH 3.0-3.1) and cooled in an ice-salt bath to - 15°C for 20 min with stirring. Acyl azide generation was performed at -15°C for 20 min by the addition of NaNO<sub>2</sub> (10 eq). Then, 1 mL of a solution of Ub[<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">28</a>-45]-(A<sup>28</sup>C)-NH-NH<sub>2</sub> (1 eq) and MPAA (100 eq) in ligation buffer (pH adjusted to 6.5) was added to Ub[<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">27</a>]-N<sub>3</sub>. The ice-salt bath was removed, and when the reaction mixture reached room temperature, the pH was slowly adjusted to 6.8. The progress of the reaction was monitored by LC-MS. After completion of the reaction, reducing agent (TCEP, 200 µL, 0.1 M in ligation buffer, pH 6.5) was added. The ligation product (Ub[<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">1</a>-45]-(A<sup>28</sup>C)-NH-NH<sub>2</sub>) was purified by semipreparative HPLC and lyophilized. Ligation of Ub[1-45]-(A<sup>28</sup>C)-NH-NH<sub>2</sub> with Ub[46-75]-(A<sup>46</sup>C)-NH-NH<sub>2</sub> was performed in the manner described above. Next, the product of the second ligation (Ub[1-75]- (A<sup>28</sup>C,A<sup>46</sup>C)-NH-NH<sub>2</sub>) was desulfurized based on a previously published protocol [<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>]. Ub[1-75]-(A<sup>28</sup>C,A<sup>46</sup>C)-NH-NH<sub>2</sub> (14.1 mg, 1.39 µmol) was dissolved in desulfurization buffer (1.8 mL, 6 M Gn*HCl, 100 mM Na<sub>2</sub>HPO<sub>4</sub>, 375 mM TCEP, 37.5 mM VA-044, 150 mM tBuSH, pH 6.9). Desulfurization was performed for 4 h at 37°C. The progress of the reaction was monitored by LC-MS. Ub[1-75]-NH-NH<sub>2</sub> was isolated by semipreparative HPLC and lyophilized. The last step of synthesis was H<sub>2</sub>N-Gly-ACC conjugation. TFA*H<sub>2</sub>N-Gly-ACC was synthesized on RA resin (0.74 mmol/g) and used without further purification. Ub[1-75]-NH-NH<sub>2</sub> (6.7 mg) was dissolved in DMF:DMSO (0.75 mL, 2:1, <em>v/v</em>). The mixture was cooled in an ice-salt bath to -20°C for 20 min with stirring. NaNO<sub>2</sub> (10 eq) in DMF:DMSO (50 µL, 2:1, <em>v/v</em>) was added. The pH of the reaction mixture was adjusted to 3.0-4.0 with 4 M HCl in dioxane solution. Acyl azide generation was carried out for 30 min in an ice-salt bath, and the progress of the reaction was monitored by LC-MS. TFA*H<sub>2</sub>N-Gly-ACC (20 eq) in DMF:DMSO (100 µL, 2:1, <em>v/v</em>) and DIPEA (20 eq) were added to the acyl azide, and the pH was adjusted to 8.0-9.0 with DIPEA. The reaction was performed at -20°C for 1 h. The progress of the reaction was monitored by LC-MS. The product was purified by analytical HPLC (purity &gt;95%), lyophilized, and characterized by HRMS. Synthesis of ABPs was performed in the same manner. The warhead synthesis protocol was adopted from a published method [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. tBu-N-allyl carbamate (500 mg, 3.2 µmol) was dissolved in 10 mL of anhydrous toluene. Methyl acrylate (580 µL, 6.4 µmol), dichlorophenylborane (42 µL, 0.32 µmol) and 2<sup>nd</sup> generation Grubbs catalyst (50 mg) were added. The reaction was carried out under reflux at 40°C with stirring overnight. After 12 h, the solvent was removed, and the mixture was purified by column chromatography on silica gel (Hex/EtOAc 5:1). The crude product was obtained in the form of a yellowish oil. <em>t</em>Bu group deprotection was performed by adding TFA/DCM/TIPS (4.2 mL, 3/1/0.2, <em>v/v/v</em>) cleavage mixture for 45 min with stirring. TFA*H<sub>2</sub>N-Gly-VME was then crystallized in cold Et<sub>2</sub>O and stored at -20°C.</p></div><div id="sec-8" class="subsection"><h3>Determination of kinetic parameters for tetrapeptide and Ub-based substrates</h3><p id="p-11">Kinetic studies were carried out in 96-well plates. Wells contained 20 µL of ACC-labeled substrate at eight different concentrations (0.1–75 µM) and 80 µL of enzyme in the same assay buffer as that described above (10 nM–2.5 µM MERS PLpro; 1 nM–6 µM UCH-L3). Substrate hydrolysis was measured for 30 min using the following wavelengths: λ<sub>ex</sub> = 355 nm, λ<sub>em</sub> = 460 nm. Each experiment was repeated at least three times. Kinetic parameters were calculated using GraphPad Prism software with the Michaelis-Menten equation. Due to the precipitation of tetrapeptide substrates at high concentrations, only the specificity constant (k<sub>cat</sub>/K<sub>M</sub>) was determined. When [S<sub>0</sub>]&lt;&lt;K<sub>M,</sub> the plot of v<sub>i</sub> (the initial velocities) versus [S<sub>0</sub>] yields a straight line with slope representing V<sub>max</sub>/K<sub>M</sub>, k<sub>cat</sub>/K<sub>M</sub> = slope/E (E – total enzyme concentration).</p></div><div id="sec-9" class="subsection"><h3>Determination of DUB inhibition by Ub-based probes</h3><p id="p-12">To assess DUB inhibition by biotinylated Ub-based probes, recombinant enzymes (50 nM) in assay buffer were incubated with eight different probe concentrations (0–3 µM) for 30 min at 37°C. Then, Ub-ACC (30 µM) was added to estimate residual DUB activity. The release of ACC was monitored for 30 min at the following wavelengths: λ<sub>ex</sub> = 355 nm and λ<sub>em</sub> = 460 nm.</p></div><div id="sec-10" class="subsection"><h3>DUB labeling in cell lysates</h3><p id="p-13">A-431 cells were cultured in DMEM supplemented with 10% fetal bovine serum, 2 mM L-glutamine, and antibiotics (100 U/mL penicillin, 100 µg/mL streptomycin) in a humidified 5% CO<sub>2</sub> atmosphere at 37°C. Approximately 1 200 000 cells were harvested and washed three times with PBS. The cell pellet was lysed in buffer containing 20 mM Tris, 150 mM NaCl, and 5 mM DTT, pH 8.0, using a sonicator. The cell lysate was centrifuged for 10 min, and the supernatant was collected. Twenty microliters of lysate was incubated with 80 µL of Ub-based probes at different concentrations for 30 min at 37°C. Then, 50 µL of 3x SDS/DTT was added, and the samples were boiled for 5 min at 95°C and resolved on 4-12% Bis-Tris Plus 12-well gels at 30 µL sample/well. Electrophoresis was performed at 200 V for 29 min. Next, the proteins were transferred to a nitrocellulose membrane (0.2 µm, Bio-Rad) for 60 min at 10 V. The membrane was blocked with 2% BSA in Tris-buffered saline with 0.1% (v/v) Tween 20 (TBS-T) for 60 min at RT. Biotinylated Ub-based probes were detected with a fluorescent streptavidin Alexa Fluor 647 conjugate (1:10 000) in TBS-T with 1% BSA, and UCH-L3 was detected with a mouse anti-human monoclonal IgG<sub>1</sub> antibody (1:1000) and fluorescent goat anti-mouse (1:10 000) using an Azure Biosystems Sapphire<sup>™</sup> Biomolecular Imager and Azure Spot Analysis Software.</p></div></div><div class="section" id="sec-11"><h2 class="">3. Results</h2><div id="sec-12" class="subsection"><h3>DUB substrate specificity profile</h3><p id="p-14">To date, the substrate specificity profile of selected DUBs has been determined using only natural amino acids [<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>]. It has been found that these enzymes can also recognize other amino acids at the P4-P2 positions, not only the C-terminal LRGG motif. These results suggest that dissection of the binding pocket architecture of DUBs can lead to the development of new chemical tools for DUB investigation with high specificity. To precisely examine the binding pocket preferences of selected enzymes, we used substrate libraries containing natural and a large number of unnatural amino acids with diverse chemical structures. Since all DUBs recognize leucine and arginine at the P4 and P3 positions, respectively, we synthesized a defined library with a general structure of Ac-Leu-Arg-P2-Gly-ACC (where P2 is a natural or unnatural amino acid) to determine the substrate specificity profile of DUBs at the P2 position. To examine substrate preferences at the P4 and P3 positions, we designed and synthesized a hybrid combinatorial substrate library (HyCoSuL). This library consists of two sublibraries: Ac-P4-Mix-Gly-Gly-ACC and Ac-Mix-P3-Gly-Gly-ACC (where P4 and P3 represent 19 natural and 119 unnatural amino acids, and Mix represents an equimolar mixture of natural amino acids). We incorporated glycine at the P1 and P2 positions because this amino acid is preferred by all DUBs in the S1 and S2 pockets. These two libraries can be used to determine the substrate specificity profile at the P4-P2 positions of all enzymes that display deubiquitinating activity.</p></div><div id="sec-13" class="subsection"><h3>P2 position</h3><p id="p-15">DUBs exhibited very narrow substrate specificity at the P2 position (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a></strong>, see full substrate specificity profile in the supplemental information, <strong>Figure S1</strong>). MERS PLpro recognized only glycine at this position. The S2 pocket of UCH-L3 could accommodate not only glycine as the best hit but also, to a lesser extent, some aliphatic amino acids, such as Ala (47%), Val (46.5%), Abu (33.5%), Nle (25%), 2-Aoc (24%), and Tle (15%), and large hydrophobic amino acids, such as Nle(OBzl) (38%), Glu(OBzl) (31%), and Bpa (25%).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Substrate specificity of UCH-L3 and MERS PLpro. (A) Substrate specificity profiles of MERS PLpro and UCH-L3 presented as heat maps (left panel) and structures of libraries (right panel). Kinetic parameters for tetrapeptide fluorogenic substrates with MERS PLpro (B) and UCH-L3 (C). (D) Tetrapeptide substrate structures: M S8, Ac-Cys(MeBzl)-Phg-Gly-Gly-ACC; M S27, Ac-Tle-Phg-Gly-Gly-ACC; M S28, Ac-Cys(4-MeOBzl)-Phg-Gly-Gly-ACC; U S1, Ac-Cha-Arg-Abu-Gly-ACC; U S2, Ac-D-Arg-Phe(guan)-Ala-Gly-ACC. (E) Rate of tetrapeptide substrate hydrolysis by DUBs ([S]=10 &#xB5;M; MERS PLpro concentration, 2.5 &#xB5;M; UCH-L3 concentration, 6 &#xB5;M); (F, G) Ub-based substrate selectivity ([S]=1 &#xB5;M; MERS PLpro concentration, 5 nM; UCH-L3 concentration, 1 nM)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1146129728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Substrate specificity of UCH-L3 and MERS PLpro.&lt;/span&gt; (A) Substrate specificity profiles of MERS PLpro and UCH-L3 presented as heat maps (left panel) and structures of libraries (right panel). Kinetic parameters for tetrapeptide fluorogenic substrates with MERS PLpro (B) and UCH-L3 (C). (D) Tetrapeptide substrate structures: M S8, Ac-Cys(MeBzl)-Phg-Gly-Gly-ACC; M S27, Ac-Tle-Phg-Gly-Gly-ACC; M S28, Ac-Cys(4-MeOBzl)-Phg-Gly-Gly-ACC; U S1, Ac-Cha-Arg-Abu-Gly-ACC; U S2, Ac-D&lt;em&gt;-&lt;/em&gt;Arg-Phe(guan)-Ala-Gly-ACC. (E) Rate of tetrapeptide substrate hydrolysis by DUBs ([S]=10 &#xB5;M; MERS PLpro concentration, 2.5 &#xB5;M; UCH-L3 concentration, 6 &#xB5;M); (F, G) Ub-based substrate selectivity ([S]=1 &#xB5;M; MERS PLpro concentration, 5 nM; UCH-L3 concentration, 1 nM).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F2.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F2.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Substrate specificity of UCH-L3 and MERS PLpro.</span><p id="p-16" class="first-child">(A) Substrate specificity profiles of MERS PLpro and UCH-L3 presented as heat maps (left panel) and structures of libraries (right panel). Kinetic parameters for tetrapeptide fluorogenic substrates with MERS PLpro (B) and UCH-L3 (C). (D) Tetrapeptide substrate structures: M S8, Ac-Cys(MeBzl)-Phg-Gly-Gly-ACC; M S27, Ac-Tle-Phg-Gly-Gly-ACC; M S28, Ac-Cys(4-MeOBzl)-Phg-Gly-Gly-ACC; U S1, Ac-Cha-Arg-Abu-Gly-ACC; U S2, Ac-D<em>-</em>Arg-Phe(guan)-Ala-Gly-ACC. (E) Rate of tetrapeptide substrate hydrolysis by DUBs ([S]=10 µM; MERS PLpro concentration, 2.5 µM; UCH-L3 concentration, 6 µM); (F, G) Ub-based substrate selectivity ([S]=1 µM; MERS PLpro concentration, 5 nM; UCH-L3 concentration, 1 nM).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>P3 and P4 positions</h3><p id="p-17">UCH-L3 displayed a narrow substrate preference at the P3 position (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, S1</strong>). Positively charged amino acids were preferred by the S3 pocket (Arg (100%), Phe(guan) (81%), and hArg (30%)). Hydrophobic residues were also recognized but with lower affinity (&lt;30%, except Trp(Me) (41%) and Ile (36%)). In contrast, MERS PLpro preferred hydrophobic amino acids at the P3 position over basic residues (Phg (100%), D-hPhe (60%), Cha (54%), Arg (47%), 2-Aoc (44%), hArg (42%), hTyr (42%), and Leu (38.5%); amino acid structures presented in Table S2 supplemental information). The S4 pocket of UCH-L3 could accommodate hydrophobic amino acids (Cha (100%), Met (66.5%), hLeu (54%), 2-Nal (52.5%), Leu (46%), Cys(4-MeOBzl) (44%)) as well as positively charged L- and D-amino acids (D-Arg (64%), <em>D</em>-Lys (40.5%), hArg (39%), Arg (38%)). MERS PLpro favored aliphatic and bulky hydrophobic residues at the P4 position. The best recognized amino acids were Cys(MeBzl) (100%), Cys(4-MeOBzl) (93%), Cys(Bzl) (84%), Phe(4-Cl) (83%), Phe(2-Cl) (80%), Leu (79%), and Ile (75%). Basic and acidic residues were almost not recognized by MERS PLpro at this position.</p></div><div id="sec-15" class="subsection"><h3>Kinetic analysis of tetrapeptide fluorogenic substrates</h3><p id="p-18">To select the optimal peptide sequence for MERS PLpro and UCH-L3, we analyzed substrate specificity preferences at the P4-P2 positions for each enzyme and selected amino acids that were both well recognized and selective toward the tested DUBs. For MERS PLpro, we chose Phg at the P3 position as the best recognized and most selective amino acid. At the P4 position, we selected Cys(MeBzl), Cys(4-MeOBzl) and Tle as the most promising candidates. The S2 pocket of UCH-L3 could accommodate some aliphatic amino acids; thus, we decided to incorporate Ala and Abu residues in substrate sequences. Although these amino acids were less well recognized than glycine, they can have a significant effect on substrate selectivity. At the P3 and P4 positions, we selected certain amino acids as the best hits (P3: Arg, Phe(guan); P4: D<em>-</em>Arg, Cha). After selection of the amino acids, we synthesized ACC-labeled substrates and determined the catalytic efficiency of the enzyme (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B, C, D</a></strong>). Most likely due to steric hindrance, Ac-Tle-Phg-Gly-Gly-ACC (M S27) was better hydrolyzed by MERS PLpro than the reference substrate (Ac-Leu-Arg-Gly-Gly-ACC). Ac-D<em>-</em>Arg-Phe(guan)-Ala-Gly-ACC (U S2) was 3 times better recognized by UCH-L3 than Ac-Leu-Arg-Gly-Gly-ACC. In the next step, we investigated substrate selectivity (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2E</a></strong>). Kinetic analysis revealed that Ac-Tle-Phg-Gly-Gly-ACC was not recognized by UCH-L3, and Ac-D<em>-</em>Arg-Phe(guan)-Ala-Gly-ACC and Ac-Cha-Arg-Abu-Gly-ACC (U S1) were selective toward this protease. We selected these three peptide sequences for further analysis.</p></div><div id="sec-16" class="subsection"><h3>Synthesis and kinetic analysis of fluorescent Ub derivative substrates</h3><p id="p-19">Since tetrapeptide fluorogenic substrates are not efficiently hydrolyzed by DUBs, even after incorporation of unnatural amino acids, the kinetic rates of the substrates were only 2-3 times higher compared to that of the reference Ac-LRGG-ACC substrate. Therefore, we decided to synthesize fluorescent Ub derivatives containing unnatural amino acids on the C-terminal tetrapeptidic epitope. To date, several chemical synthesis methods for Ub derivatives have been reported [<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>-<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. We applied these methods with several modifications for effective synthesis of Ub-based substrates and ABPs comprising unnatural building blocks. In the first step, we divided Ub into three peptide segments and synthesized them separately using Fmoc-based SPPS of peptide hydrazides on 2-chlorotrityl chloride resin. This approach enables (1) efficient synthesis of each peptide segment with good isolated yields and purity; (2) incorporation of unnatural amino acids on the C-terminal Ub motif; and (3) modification of the N-terminus by introducing tags and linkers. In the next step, three peptide segments were assembled in a stepwise manner through NCL. Then, cysteine residues, which were introduced in the Ub sequence to enable NCL reactions, were converted to alanine using a free radical desulfurization reaction. In the last step, a fluorescent tag (ACC) with glycine was attached to the Ub[1-75]-NH-NH<sub>2</sub> C-terminus, and the final product was purified and analyzed by HPLC and HRMS (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme of Ub derivative synthesis. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1146129728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Scheme of Ub derivative synthesis.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F3.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F3.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Scheme of Ub derivative synthesis.</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">Kinetic analysis of fluorescent Ub-based substrates revealed that by incorporation of unnatural amino acids in the Ub C-terminus, selective and active substrates for DUBs can be synthesized (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2F, G</a></strong>). The designed substrates were better hydrolyzed by MERS PLpro (M27-ACC) and UCH-L3 (U1-ACC and U2-ACC) than Ub-ACC, used as a reference substrate (<strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). U2-ACC was not recognized by MERS PLpro. M-ACC was poorly hydrolyzed by UCH-L3 (almost 200 times weaker than Ub-ACC).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1131737/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1131737/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1131737" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-21" class="first-child">Kinetic parameters of Ub-based fluorogenic substrates with UCH-L3 and MERS PLpro.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Design of selective Ub-based probes</h3><p id="p-22">One of the main aims of this work was to design Ub-based probes for highly selective detection of the investigated DUBs. Thus, we converted our Ub-based substrates to probes by incorporating a detection tag on the N-terminus and an irreversible reactive group on the C-terminus. VME was applied as an electrophilic warhead due to its broad reactivity toward DUBs [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. The biotin tag was separated from the peptide sequence by using a 6-aminohexanoic acid linker (6-Ahx). All Ub-based probes were synthesized using the same synthetic strategy as that used for the Ub-based substrates (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a></strong>). The kinetic analysis with recombinant enzymes revealed that B-U2-VME was a potent and selective probe for UCH-L3 (<strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>). B-M27-VME was selective toward MERS PLpro (a much higher probe concentration was needed to inhibit UCH-L3). Furthermore, this probe was more potent toward MERS PLpro than the control probe B-Ub-VME (<strong><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a></strong>). The results were generally consistent with the Ub-based substrate kinetic data; however, B-U2-VME was a more potent probe for UCH-L3 than B-U1-VME.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1131738/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1131738/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1131738" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-23" class="first-child">Kinetic analysis of DUB inhibition by Ub probes. Recombinant enzymes were incubated with various probe concentrations for 30 min at 37°C before DUB activity measurement. The experiment was performed in triplicate, and the average results are presented (SD&lt;10%).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Detection of DUBs in cell lysates</h3><p id="p-24">To assess Ub-based probe selectivity, we incubated cell lysates with various probe concentrations. Since UCH-L3 is expressed in the A-431 cell line, we selected this cell line for DUB labeling experiments. Due to the lack of MERS PLpro expression in human cell lines, we added the recombinant enzyme to cell lysates during incubation with probes. These DUB assays in cell lysates with designed probes revealed that our probes (B-M27-VME, B-U1-VME and B-U2-VME) displayed high selectivity toward MERS PLpro and UCH-L3 compared to the reference probe that labeled many cellular DUBs (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A, B</a></strong>). Labeling of UCH-L3 by our probes in A-431 cell lysate was confirmed by immunoblotting (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a></strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ub-based probe selectivity. DUB labeling in cell lysates by B-Ub-VME and B-M27-VME (A) and B-U1-VME and B-U2-VME (B). (C) Detection of UCH-L3 in A-431 cell lysates using (1) Ub-based probes (using a streptavidin Alexa Fluor 647 conjugate) and (2) a UCH-L3 antibody. A-431 cell lysate was incubated with three different probe concentrations (100, 200 and 400 nM) for 30 min at 37&#xB0;C. &#x3B2;-Actin was used as a loading control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1146129728" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Ub-based probe selectivity.&lt;/span&gt; DUB labeling in cell lysates by B-Ub-VME and B-M27-VME (A) and B-U1-VME and B-U2-VME (B). (C) Detection of UCH-L3 in A-431 cell lysates using (1) Ub-based probes (using a streptavidin Alexa Fluor 647 conjugate) and (2) a UCH-L3 antibody. A-431 cell lysate was incubated with three different probe concentrations (100, 200 and 400 nM) for 30 min at 37&#xB0;C. &#x3B2;-Actin was used as a loading control.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F4.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F4.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/31/2020.01.30.926881/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Ub-based probe selectivity.</span><p id="p-25" class="first-child">DUB labeling in cell lysates by B-Ub-VME and B-M27-VME (A) and B-U1-VME and B-U2-VME (B). (C) Detection of UCH-L3 in A-431 cell lysates using (1) Ub-based probes (using a streptavidin Alexa Fluor 647 conjugate) and (2) a UCH-L3 antibody. A-431 cell lysate was incubated with three different probe concentrations (100, 200 and 400 nM) for 30 min at 37°C. β-Actin was used as a loading control.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-19"><h2 class="">4. Discussion and Conclusions</h2><p id="p-26">DUBs constitute potential therapeutic targets in the treatment of viral diseases, neurodegenerative diseases and cancer [<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. The hottest example here is present influenza outbreak caused by Wuhan virus. Current studies indicate that this coronavirus is similar to SARS and MERS. Provided by us, this technology could be used for fast delivery of a selective 2019-nCoV PLpro DUB substrate and probe, which could effectively be used as a screen/reporter assay to find drugs that can specifically inhibit the 2019-nCoV PLpro DUB by screening infected patient cells without the need to purify DUB from infected patient cells. In classic examples in which DUBs are involved, enzymes belonging to the USP subclass play a key role in many biological processes and pathological conditions, including DNA repair, stabilization/degradation of the p53 protein, and progression of lung, breast, kidney and prostate cancer [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. UCH-L3 is overexpressed in triple-negative breast cancer (TNBC), which correlates with poor prognosis [<a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">27</a>]. This enzyme is also involved in the progression of pancreatic cancer, but little is known about its precise functions [<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">28</a>]. This is due to the lack of selective chemical tools for the study of this enzyme; such tools would enable the determination of the role of this enzyme in normal and pathological states and could offer leading structures and chemical starting points for drug development. DUBs are highly specific toward Ub; thus, the chemical tools most commonly used to investigate DUBs are based on a Ub scaffold. These broad-spectrum tools are valuable in DUB profiling assays. However, substrates and ABPs recognized by one or a narrow subset of DUBs would enable more precise examination of this group of enzymes. To date, Ub-based probe selectivity toward one or a narrow subset of DUBs has been achieved by (1) selecting various C-terminal reactive electrophilic groups [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]; (2) modifying the Ub scaffold length [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]; (3) synthesizing poly-Ub chains with different linkage topologies [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]; and (4) introducing various mutations in the Ub sequence [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. Our previous work showed that Ub-based substrate selectivity can be accomplished by modification of the C-terminal LRGG motif of Ub [<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>]. The introduction of natural amino acids other than canonical LRG at the P4-P2 positions in the Ub derivative provided selectivity but reduced DUB activity by 20-50 times compared to the natural analog.</p><p id="p-27">Based on this finding, we decided to incorporate unnatural amino acids in the C-terminal motif of Ub to improve DUB activity and selectivity toward Ub-based substrates and ABPs. For precise analysis of the DUB binding pocket architecture, we utilized a combinatorial approach and synthesized three tetrapeptide fluorogenic substrate libraries. Due to the low tetrapeptide substrate hydrolysis rate of DUBs, three positions in the combinatorial library (P4 and P3 sublibraries) were fixed, and one contained an equimolar mixture of 19 amino acids. The P2 sublibrary was designed by fixing the P4, P3 and P1 positions as canonical amino acids present in Ub. The P2 position contained 128 natural and unnatural amino acids. This approach allowed us to achieve the highest possible substrate concentration in each position of the sublibrary during screening assays. We did not examine P1 substrate specificity because crystal structures of DUBs showed that this position can be occupied only by a glycine residue [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. Library screening revealed that DUBs possess wide substrate specificity at the P4 and P3 positions. The DUBs recognized some unnatural amino acids even better than leucine and arginine at the P4-P3 positions. These results shed new light on DUB substrate preferences and the architecture of their binding sites. Kinetic analysis of tetrapeptide fluorogenic substrates revealed that the designed substrates with unnatural amino acids were selective and 2-3 times better recognized than the control substrate (Ac-LRGG-ACC); however, these substrates were still very poorly hydrolyzed. To improve DUB activity toward the designed substrates, we decided to synthesize unnatural Ub by incorporating the selected peptide sequences instead of the canonical LRGG sequence at the C-terminus and adding a fluorescent tag. DUBs possess two recognition regions that are required for effective Ub substrate hydrolysis [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The first region (being the secondary binding site or the exosite) interacts with the Ub surface, while the second region is the active center of the DUB, where the C-terminal LRGG motif is bound. Ub binding through the first region (distant from the active site cleft) leads to large conformational changes in DUB binding pockets that are required for effective catalysis of substrates [<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>]. Due to the nature of Ub binding by DUBs, we were expecting that translation from tetrapeptide substrates to full-length Ub sequences containing 2-3 unnatural amino acids on the C-terminus may result in a slight loss of selectivity. Despite the lack of hydrolysis of the M S27 tetrapeptide substrate by UCH-L3, kinetic analysis of the designed Ub-based substrates revealed that the MERS PLpro substrate (M27-ACC) was recognized by UCH-L3, but this recognition was almost 200 times weaker than that of the reference Ub-ACC. The UCH-L3 substrate (U2-ACC) was not hydrolyzed by MERS PLpro even at high concentrations. Furthermore, these substrates were more efficiently hydrolyzed by DUBs than Ub-ACC. Interestingly, U S1, an approximately 3-fold weaker UCH-L3 tetrapeptide substrate than U S2, after conversion to the U1-ACC Ub substrate, had an almost 1.5-fold higher kinetic efficiency constant than the U S2 analog U2-ACC. Nevertheless, both U1-ACC and U2-ACC displayed high selectivity toward UCH-L3 and exhibited kinetic parameters superior to those of Ub-ACC. Both of these features make these substrates very valuable alternatives to standard reagents used in studies of UCH-L3.</p><p id="p-28">One of the most commonly used chemical tools in protease investigation is ABPs. Therefore, we converted our Ub-based fluorogenic substrates to Ub-based probes by replacing the ACC fluorophore with the VME electrophilic group and attaching a biotin tag to the N-terminus. Cell lysate assays confirmed the dramatically high selectivity of these engineered Ub-based probes toward the investigated enzymes, especially when compared to the reference probe B-Ub-VME, which labeled many cellular DUBs. These results indicate that Ub-based selective chemical tools for DUBs can be obtained by introducing unnatural amino acids into the C-terminal Ub motif.</p><p id="p-29">In summary, we demonstrated that our approach can be successfully applied in the design and synthesis of selective mono-Ub substrates and ABPs. Precise analysis of DUB binding pocket architecture using defined and combinatorial libraries with natural and unnatural amino acids allowed us to extract key residues that were introduced into the C-terminal motif of Ub and provided selectivity toward the investigated enzymes. Our findings expand the knowledge of DUBs, as well as the existing ‘toolbox’ of Ub-based biochemical tools to study this group of enzymes. Moreover, the presented chemical approach may be beneficial for the development of a) new tools for the investigation of DUBs from distinct subclasses; b) more selective tools based on poly-Ub chains with different topologies; and c) new selective tools for studies of other Ub-like modifiers (UBLs), e.g., SUMO proteins, ISG15 or Nedd8, and their conjugation/deconjugation biochemical machinery.</p></div><div class="section" id="sec-21"><h2 class="">Competing interest</h2><p id="p-31">The authors declare no competing financial interest.</p></div><div class="section" id="sec-22"><h2 class="">Author contributions</h2><p id="p-32">M.D. conceptualized the study; M. D., W.R. and M.Ż. designed the research; W.R. and M.Ż. performed the research and collected data; S.J.S., M.B. and T.T.H. contributed enzymes; M.D., W.R. and M.Ż. analyzed and interpreted the data and wrote the manuscript; and S.J.S., M.B. and T.T.H. critically revised the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-30">This project was supported by the National Science Center grant 2015/17/N/ST5/03072 (Preludium 9) in Poland (W.R.) and the “TEAM/2017-4/32” project, which is carried out within the TEAM program of the Foundation for Polish Science, cofinanced by the European Union under the European Regional Development Fund (M.D.). W.R. is a beneficiary of a START scholarship from the Foundation for Polish Science. NIH grants GM107257 (T.T.H.), ES025166 (T.T.H.) and F32GM100598 (M.B.) provided additional support for this work.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.1" data-doi="10.1016/j.bbamcr.2013.06.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eletr</span>,  <span class="cit-name-given-names">Z.M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">K.D.</span> <span class="cit-name-surname">Wilkinson</span></span>, <span class="cit-article-title">Regulation of proteolysis by human deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">1843</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">114</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim%2BBiophys%2BActa%26rft.volume%253D1843%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2013.06.027%26rft_id%253Dinfo%253Apmid%252F23845989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2013.06.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845989&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.2" data-doi="10.1016/j.bbamcr.2004.10.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amerik</span>,  <span class="cit-name-given-names">A.Y.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hochstrasser</span></span>, <span class="cit-article-title">Mechanism and function of deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">1695</span>(<span class="cit-issue">1-3</span>): p. <span class="cit-fpage">189</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DAmerik%26rft.auinit1%253DA.%2BY.%26rft.volume%253D1695%26rft.issue%253D1-3%26rft.spage%253D189%26rft.epage%253D207%26rft.atitle%253DMechanism%2Band%2Bfunction%2Bof%2Bdeubiquitinating%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2004.10.003%26rft_id%253Dinfo%253Apmid%252F15571815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2004.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15571815&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226213600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.3" data-doi="10.1016/j.cell.2013.05.046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mevissen</span>,  <span class="cit-name-given-names">T.E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">OTU deubiquitinases reveal mechanisms of linkage specificity and enable ubiquitin chain restriction analysis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">154</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">169</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.046%26rft_id%253Dinfo%253Apmid%252F23827681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23827681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321327900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mielech</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">MERS-CoV papain-like protease has deISGylating and deubiquitinating activities</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">450-451</span>: p. <span class="cit-fpage">64</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D450451%26rft.spage%253D64%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.5" data-doi="10.1128/JVI.79.24.15199-15208.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindner</span>,  <span class="cit-name-given-names">H.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The papain-like protease from the severe acute respiratory syndrome coronavirus is a deubiquitinating enzyme</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">79</span>(<span class="cit-issue">24</span>): p. <span class="cit-fpage">15199</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLindner%26rft.auinit1%253DH.%2BA.%26rft.volume%253D79%26rft.issue%253D24%26rft.spage%253D15199%26rft.epage%253D15208%26rft.atitle%253DThe%2BPapain-Like%2BProtease%2Bfrom%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BIs%2Ba%2BDeubiquitinating%2BEnzyme%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.24.15199-15208.2005%26rft_id%253Dinfo%253Apmid%252F16306591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8yNC8xNTE5OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzMxLzIwMjAuMDEuMzAuOTI2ODgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ubiquitin 7-amino-4-carbamoylmethylcoumarin as an improved fluorogenic substrate for deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Tetrahedron</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">72</span>(<span class="cit-issue">27-28</span>): p. <span class="cit-fpage">4085</span>–<span class="cit-lpage">4090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTetrahedron%26rft.volume%253D72%26rft.spage%253D4085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.7" data-doi="10.1021/bi9723360"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dang</span>,  <span class="cit-name-given-names">L.C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.D.</span> <span class="cit-name-surname">Melandri</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.L.</span> <span class="cit-name-surname">Stein</span></span>, <span class="cit-article-title">Kinetic and mechanistic studies on the hydrolysis of ubiquitin C-terminal 7-amido-4-methylcoumarin by deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, <span class="cit-pub-date">1998</span>. <span class="cit-vol">37</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">1868</span>–<span class="cit-lpage">1879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D37%26rft.spage%253D1868%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi9723360%26rft_id%253Dinfo%253Apmid%252F9485312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi9723360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9485312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072304200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.8" data-doi="10.1016/j.ab.2007.07.034"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hassiepen</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A sensitive fluorescence intensity assay for deubiquitinating proteases using ubiquitin-rhodamine 110-glycine as substrate</span>. <abbr class="cit-jnl-abbrev">Analytical Biochemistry</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">371</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">201</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DHassiepen%26rft.auinit1%253DU.%26rft.volume%253D371%26rft.issue%253D2%26rft.spage%253D201%26rft.epage%253D207%26rft.atitle%253DA%2Bsensitive%2Bfluorescence%2Bintensity%2Bassay%2Bfor%2Bdeubiquitinating%2Bproteases%2Busing%2Bubiquitin-rhodamine110-glycine%2Bas%2Bsubstrate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ab.2007.07.034%26rft_id%253Dinfo%253Apmid%252F17869210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ab.2007.07.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17869210&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">K.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Mechanisms, biology and inhibitors of deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Nature Chemical Biology</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">3</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">697</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BChemical%2BBiology%26rft.volume%253D3%26rft.spage%253D697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.30.926881v1.10" data-doi="10.1146/annurev.biochem.78.082307.091526"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reyes-Turcu</span>,  <span class="cit-name-given-names">F.E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.H.</span> <span class="cit-name-surname">Ventii</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.D.</span> <span class="cit-name-surname">Wilkinson</span></span>, <span class="cit-article-title">Regulation and Cellular Roles of Ubiquitin-Specific Deubiquitinating Enzymes</span>. <abbr class="cit-jnl-abbrev">Annual Review of Biochemistry</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">78</span>: p. <span class="cit-fpage">363</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DReyes-Turcu%26rft.auinit1%253DF.%2BE.%26rft.volume%253D78%26rft.spage%253D363%26rft.epage%253D397%26rft.atitle%253DRegulation%2Band%2Bcellular%2Broles%2Bof%2Bubiquitin-specific%2Bdeubiquitinating%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.78.082307.091526%26rft_id%253Dinfo%253Apmid%252F19489724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.78.082307.091526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19489724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268069200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.11" data-doi="10.1016/S1074-5521(02)00248-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Chemistry-based functional proteomics reveals novel members of the deubiquitinating enzyme</span>. <abbr class="cit-jnl-abbrev">Chemistry &amp; Biology</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">9</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">1149</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemistry%2B%2526%2Bbiology%26rft.stitle%253DChem%2BBiol%26rft.aulast%253DBorodovsky%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D1149%26rft.epage%253D1159%26rft.atitle%253DChemistry-based%2Bfunctional%2Bproteomics%2Breveals%2Bnovel%2Bmembers%2Bof%2Bthe%2Bdeubiquitinating%2Benzyme%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1074-5521%252802%252900248-X%26rft_id%253Dinfo%253Apmid%252F12401499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1074-5521(02)00248-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12401499&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178895200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hewings</span>,  <span class="cit-name-given-names">D.S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Activity-based probes for the ubiquitin conjugation-deconjugation machinery: new chemistries, new tools, and new insights</span>. <abbr class="cit-jnl-abbrev">Febs Journal</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">284</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">1555</span>–<span class="cit-lpage">1576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFebs%2BJournal%26rft.volume%253D284%26rft.spage%253D1555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.13" data-doi="10.1038/cr.2016.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ronau</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.F.</span> <span class="cit-name-surname">Beckmann</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hochstrasser</span></span>, <span class="cit-article-title">Substrate specificity of the ubiquitin and Ubl proteases</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">26</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">441</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D26%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.38%26rft_id%253Dinfo%253Apmid%252F27012468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27012468&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.14" data-doi="10.1038/nrd.2017.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrigan</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Deubiquitylating enzymes and drug discovery: emerging opportunities</span>. <abbr class="cit-jnl-abbrev">Nat Rev Drug Discov</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">57</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.volume%253D17%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd.2017.152%26rft_id%253Dinfo%253Apmid%252F28959952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd.2017.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28959952&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.15" data-doi="10.1002/cbic.201200261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iphofer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Profiling ubiquitin linkage specificities of deubiquitinating enzymes with branched ubiquitin isopeptide probes</span>. <abbr class="cit-jnl-abbrev">Chembiochem</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">13</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">1416</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChembiochem%2B%253A%2B%2Ba%2BEuropean%2Bjournal%2Bof%2Bchemical%2Bbiology%26rft.stitle%253DChembiochem%26rft.aulast%253DIphofer%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D1416%26rft.epage%253D1420%26rft.atitle%253DProfiling%2Bubiquitin%2Blinkage%2Bspecificities%2Bof%2Bdeubiquitinating%2Benzymes%2Bwith%2Bbranched%2Bubiquitin%2Bisopeptide%2Bprobes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcbic.201200261%26rft_id%253Dinfo%253Apmid%252F22689415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cbic.201200261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22689415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Activity-based diubiquitin probes for elucidating the linkage specificity of deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Chemical Communications</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">50</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">216</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BCommunications%26rft.volume%253D50%26rft.spage%253D216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flierman</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Non-hydrolyzable Diubiquitin Probes Reveal Linkage-Specific Reactivity of Deubiquitylating Enzymes Mediated by S2 Pockets</span>. <abbr class="cit-jnl-abbrev">Cell Chemical Biology</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">23</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">472</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BChemical%2BBiology%26rft.volume%253D23%26rft.spage%253D472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.18" data-doi="10.1042/BJ20141170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bekes</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">SARS hCoV papain-like protease is a unique Lys(48) linkage-specific di-distributive deubiquitinating enzyme</span>. <abbr class="cit-jnl-abbrev">Biochemical Journal</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">468</span>: p. <span class="cit-fpage">215</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20141170%26rft_id%253Dinfo%253Apmid%252F25764917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDY4LzIvMjE1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzEvMjAyMC4wMS4zMC45MjY4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.19" data-doi="10.1016/j.molcel.2016.04.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bekes</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Recognition of Lys48-Linked Di-ubiquitin and Deubiquitinating Activities of the SARS Coronavirus Papain-like Protease</span>. <abbr class="cit-jnl-abbrev">Molecular Cell</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">62</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">572</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BCell%26rft.volume%253D62%26rft.spage%253D572%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.04.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.04.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gjonaj</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">USP7: combining tools towards selectivity</span>. <abbr class="cit-jnl-abbrev">Chemical Communications</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">55</span>(<span class="cit-issue">35</span>): p. <span class="cit-fpage">5075</span>–<span class="cit-lpage">5078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BCommunications%26rft.volume%253D55%26rft.spage%253D5075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gjonaj</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Development of a DUB-selective fluorogenic substrate</span>. <abbr class="cit-jnl-abbrev">Chemical Science</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">10</span>(<span class="cit-issue">44</span>): p. <span class="cit-fpage">10290</span>–<span class="cit-lpage">10296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BScience%26rft.volume%253D10%26rft.spage%253D10290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.22" data-doi="10.1042/BJ20080779"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drag</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Positional-scanning fluorigenic substrate libraries reveal unexpected specificity determinants of DUBs (deubiquitinating enzymes)</span>. <abbr class="cit-jnl-abbrev">Biochemical Journal</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">415</span>: p. <span class="cit-fpage">367</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20080779%26rft_id%253Dinfo%253Apmid%252F18601651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDE1LzMvMzY3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMzEvMjAyMC4wMS4zMC45MjY4ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poreba</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes</span>. <abbr class="cit-jnl-abbrev">Cell Chemical Biology</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">23</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">1023</span>–<span class="cit-lpage">1035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BChemical%2BBiology%26rft.volume%253D23%26rft.spage%253D1023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poreba</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Selective imaging of cathepsin L in breast cancer by fluorescent activity-based probes</span>. <abbr class="cit-jnl-abbrev">Chemical Science</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">2113</span>–<span class="cit-lpage">2129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BScience%26rft.volume%253D9%26rft.spage%253D2113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.25" data-doi="10.1021/jacs.7b04394"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kasperkiewicz</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils</span>. <abbr class="cit-jnl-abbrev">Journal of the American Chemical Society</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">139</span>(<span class="cit-issue">29</span>): p. <span class="cit-fpage">10115</span>–<span class="cit-lpage">10125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.volume%253D139%26rft.spage%253D10115%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjacs.7b04394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jacs.7b04394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rut</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span>. <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">61</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">5222</span>–<span class="cit-lpage">5234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedicinal%2BChemistry%26rft.volume%253D61%26rft.spage%253D5222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A novel UCHL3 inhibitor, perifosine, enhances PARP inhibitor cytotoxicity through inhibition of homologous recombination-mediated DNA double strand break repair</span>. <abbr class="cit-jnl-abbrev">Cell Death &amp; Disease</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">UCHL3 promotes pancreatic cancer progression and chemo-resistance through FOXM1 stabilization</span>. <abbr class="cit-jnl-abbrev">American Journal of Cancer Research</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">9</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">1970</span>–<span class="cit-lpage">1981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BCancer%2BResearch%26rft.volume%253D9%26rft.spage%253D1970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.29" data-doi="10.1021/jo016140o"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maly</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Expedient solid-phase synthesis of fluorogenic protease substrates using the 7-amino-4-carbamoylmethylcoumarin (ACC) fluorophore</span>. <abbr class="cit-jnl-abbrev">Journal of Organic Chemistry</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">67</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">910</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Borganic%2Bchemistry%26rft.stitle%253DJ%2BOrg%2BChem%26rft.aulast%253DMaly%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D910%26rft.epage%253D915%26rft.atitle%253DExpedient%2Bsolid-phase%2Bsynthesis%2Bof%2Bfluorogenic%2Bprotease%2Bsubstrates%2Busing%2Bthe%2B7-amino-4-carbamoylmethylcoumarin%2B%2528ACC%2529%2Bfluorophore.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjo016140o%26rft_id%253Dinfo%253Apmid%252F11856036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jo016140o&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11856036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poreba</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.S.</span> <span class="cit-name-surname">Salvesen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Drag</span></span>, <span class="cit-article-title">Synthesis of a HyCoSuL peptide substrate library to dissect protease substrate specificity</span>. <abbr class="cit-jnl-abbrev">Nature Protocols</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">12</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">2189</span>–<span class="cit-lpage">2214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BProtocols%26rft.volume%253D12%26rft.spage%253D2189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.31" data-doi="10.1007/978-1-4939-0357-3_2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poreba</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Positional scanning substrate combinatorial library (PS-SCL) approach to define caspase substrate specificity</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">1133</span>: p. <span class="cit-fpage">41</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1133%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-0357-3_2%26rft_id%253Dinfo%253Apmid%252F24567093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-0357-3_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567093&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.32" data-doi="10.1002/anie.201005995"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Oualid</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Chemical synthesis of ubiquitin, ubiquitin-based probes, and diubiquitin</span>. <abbr class="cit-jnl-abbrev">Angew Chem Int Ed Engl</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">49</span>(<span class="cit-issue">52</span>): p. <span class="cit-fpage">10149</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAngewandte%2BChemie%2B%2528International%2BEdition%2Bin%2BEnglish%2529.%26rft.stitle%253DAngewandte%2BChemie%2B%2528International%2BEdition%2Bin%2BEnglish%2529.%26rft.aulast%253DEl%2BOualid%26rft.auinit1%253DF.%26rft.volume%253D49%26rft.issue%253D52%26rft.spage%253D10149%26rft.epage%253D10153%26rft.atitle%253DChemical%2Bsynthesis%2Bof%2Bubiquitin%252C%2Bubiquitin-based%2Bprobes%252C%2Band%2Bdiubiquitin.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fanie.201005995%26rft_id%253Dinfo%253Apmid%252F21117055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/anie.201005995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21117055&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285422000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.30.926881v1.33" data-doi="10.1002/cbic.201200497"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Jong</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ubiquitin-based probes prepared by total synthesis to profile the activity of deubiquitinating enzymes</span>. <abbr class="cit-jnl-abbrev">Chembiochem</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">13</span>(<span class="cit-issue">15</span>): p. <span class="cit-fpage">2251</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChembiochem%2B%253A%2B%2Ba%2BEuropean%2Bjournal%2Bof%2Bchemical%2Bbiology%26rft.stitle%253DChembiochem%26rft.aulast%253Dde%2BJong%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D15%26rft.spage%253D2251%26rft.epage%253D2258%26rft.atitle%253DUbiquitin-based%2Bprobes%2Bprepared%2Bby%2Btotal%2Bsynthesis%2Bto%2Bprofile%2Bthe%2Bactivity%2Bof%2Bdeubiquitinating%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcbic.201200497%26rft_id%253Dinfo%253Apmid%252F23011887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cbic.201200497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23011887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Chemical synthesis of Ub-AMC via ligation of peptide hydrazides</span>. <abbr class="cit-jnl-abbrev">Science China-Chemistry</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">56</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">1301</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BChina-Chemistry%26rft.volume%253D56%26rft.spage%253D1301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.35" data-doi="10.1016/j.tet.2014.03.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Facile synthesis of C-terminal peptide hydrazide and thioester of NY-ESO-1 (A39-A68) from an Fmoc-hydrazine 2-chlorotrityl chloride resin</span>. <abbr class="cit-jnl-abbrev">Tetrahedron</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">70</span>(<span class="cit-issue">18</span>): p. <span class="cit-fpage">2951</span>–<span class="cit-lpage">2955</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTetrahedron%26rft.volume%253D70%26rft.spage%253D2951%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tet.2014.03.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tet.2014.03.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.36" data-doi="10.1038/nprot.2013.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Chemical synthesis of proteins using peptide hydrazides as thioester surrogates</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">8</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">2483</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D8%26rft.spage%253D2483%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.152%26rft_id%253Dinfo%253Apmid%252F24232250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24232250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vedrenne</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Dramatic effect of boron-based Lewis acids in cross-metathesis reactions</span>. <abbr class="cit-jnl-abbrev">Synlett</abbr>, <span class="cit-pub-date">2005</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">670</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">D’Arcy</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Linder</span></span>, <span class="cit-article-title">Deubiquitinase inhibition as a cancer therapeutic strategy</span>. <abbr class="cit-jnl-abbrev">Pharmacology &amp; Therapeutics</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">147</span>: p. <span class="cit-fpage">32</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacology%2B%2526%2BTherapeutics%26rft.volume%253D147%26rft.spage%253D32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">M.I.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Simeonov</span></span>, <span class="cit-article-title">Ubiquitin-Specific Proteases as Druggable Targets</span>. <abbr class="cit-jnl-abbrev">Drug Target Rev</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">2</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">60</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2BTarget%2BRev%26rft.volume%253D2%26rft.spage%253D60%26rft_id%253Dinfo%253Apmid%252F26835158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26835158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The role of ubiquitin-specific peptidases in cancer progression</span>. <abbr class="cit-jnl-abbrev">J Biomed Sci</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">26</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiomed%2BSci%26rft.volume%253D26%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.41" data-doi="10.1021/cb9000348"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">K.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ubiquitin C-terminal electrophiles are activity-based probes for identification and mechanistic study of ubiquitin conjugating machinery</span>. <abbr class="cit-jnl-abbrev">ACS Chem Biol</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">275</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DACS%2BChem%2BBiol%26rft.aulast%253DLove%26rft.auinit1%253DK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D275%26rft.epage%253D287%26rft.atitle%253DUbiquitin%2BC-terminal%2Belectrophiles%2Bare%2Bactivity-based%2Bprobes%2Bfor%2Bidentification%2Band%2Bmechanistic%2Bstudy%2Bof%2Bubiquitin%2Bconjugating%2Bmachinery.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcb9000348%26rft_id%253Dinfo%253Apmid%252F19256548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cb9000348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19256548&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.42" data-doi="10.1002/cbic.200400236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Small-molecule inhibitors and probes for ubiquitin- and ubiquitin-like-specific proteases</span>. <abbr class="cit-jnl-abbrev">Chembiochem</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">287</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChembiochem%2B%253A%2B%2Ba%2BEuropean%2Bjournal%2Bof%2Bchemical%2Bbiology%26rft.stitle%253DChembiochem%26rft.aulast%253DBorodovsky%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D287%26rft.epage%253D291%26rft.atitle%253DSmall-molecule%2Binhibitors%2Band%2Bprobes%2Bfor%2Bubiquitin-%2Band%2Bubiquitin-like-specific%2Bproteases.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcbic.200400236%26rft_id%253Dinfo%253Apmid%252F15651044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cbic.200400236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15651044&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F31%2F2020.01.30.926881.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.30.926881v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soboleva</span>,  <span class="cit-name-given-names">T.A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.T.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-article-title">Deubiquitinating enzymes: Their functions and substrate specificity</span>. <abbr class="cit-jnl-abbrev">Current Protein &amp; Peptide Science</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">191</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BProtein%2B%2526%2BPeptide%2BScience%26rft.volume%253D5%26rft.spage%253D191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.30.926881v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.30.925917v1" title="MAGGIE: leveraging genetic variation to identify DNA sequence motifs mediating transcription factor binding and function" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.01.29.925040v1" title="Unraveling the Mechanism of a LOV Domain Optogenetic Sensor: A Glutamine Lever Induces Unfolding of the Jα Helix" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 31, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.30.926881v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.01.30.926881v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.30.926881v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.30.926881v1" class="active" data-icon-position="" data-hide-link-title="0">Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1127948" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lYrWGn05GmC0t2hXsN7LwStoIhBhW7jfs8zUazi0ovs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127948" id="node1127948--2" data-pisa="biorxiv;2020.01.30.926881v1" data-pisa-master="biorxiv;2020.01.30.926881" data-seqnum="1127948" data-apath="/biorxiv/early/2020/01/31/2020.01.30.926881.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wioletta</span> <span class="nlm-surname">Rut</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mikołaj</span> <span class="nlm-surname">Żmudziński</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott J.</span> <span class="nlm-surname">Snipas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miklos</span> <span class="nlm-surname">Bekes</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Tony T.</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Marcin</span> <span class="nlm-surname">Drag</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.30.926881; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.30.926881 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;title=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;title=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;text=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;title=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;t=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;title=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;title=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1127948" id="node1127948--3" data-pisa="biorxiv;2020.01.30.926881v1" data-pisa-master="biorxiv;2020.01.30.926881" data-seqnum="1127948" data-apath="/biorxiv/early/2020/01/31/2020.01.30.926881.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Engineered unnatural ubiquitin for optimal detection of deubiquitinating enzymes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wioletta</span> <span class="nlm-surname">Rut</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mikołaj</span> <span class="nlm-surname">Żmudziński</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Scott J.</span> <span class="nlm-surname">Snipas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miklos</span> <span class="nlm-surname">Bekes</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Tony T.</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Marcin</span> <span class="nlm-surname">Drag</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.30.926881; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.30.926881 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rut%2C+Wioletta&amp;rft.au=%C5%BBmudzi%C5%84ski%2C+Miko%C5%82aj&amp;rft.au=Snipas%2C+Scott+J.&amp;rft.au=Bekes%2C+Miklos&amp;rft.au=Huang%2C+Tony+T.&amp;rft.au=Drag%2C+Marcin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1127948/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1127948/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1127948/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1127948/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1127948/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1127948/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1127948/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1127948/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1127948/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1127948/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1127948/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1127948/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;count=horizontal&amp;via=&amp;text=Engineered%20unnatural%20ubiquitin%20for%20optimal%20detection%20of%20deubiquitinating%20enzymes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.30.926881v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.30.926881v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
