<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/29/2020.02.26.964882.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.26.964882v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Structure of Mpro from COVID-19 virus and discovery of its inhibitors"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.26.964882"/>
  <meta name="DC.Date" content="2020-03-29"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="A new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan[1][1]–[4][2]. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus[5][3],[6][4]. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available. [1]: #ref-1 [2]: #ref-4 [3]: #ref-5 [4]: #ref-6"/>
  <meta name="DC.Contributor" content="Zhenming Jin"/>
  <meta name="DC.Contributor" content="Xiaoyu Du"/>
  <meta name="DC.Contributor" content="Yechun Xu"/>
  <meta name="DC.Contributor" content="Yongqiang Deng"/>
  <meta name="DC.Contributor" content="Meiqin Liu"/>
  <meta name="DC.Contributor" content="Yao Zhao"/>
  <meta name="DC.Contributor" content="Bing Zhang"/>
  <meta name="DC.Contributor" content="Xiaofeng Li"/>
  <meta name="DC.Contributor" content="Leike Zhang"/>
  <meta name="DC.Contributor" content="Chao Peng"/>
  <meta name="DC.Contributor" content="Yinkai Duan"/>
  <meta name="DC.Contributor" content="Jing Yu"/>
  <meta name="DC.Contributor" content="Lin Wang"/>
  <meta name="DC.Contributor" content="Kailin Yang"/>
  <meta name="DC.Contributor" content="Fengjiang Liu"/>
  <meta name="DC.Contributor" content="Rendi Jiang"/>
  <meta name="DC.Contributor" content="Xinglou Yang"/>
  <meta name="DC.Contributor" content="Tian You"/>
  <meta name="DC.Contributor" content="Xiaoce Liu"/>
  <meta name="DC.Contributor" content="Xiuna Yang"/>
  <meta name="DC.Contributor" content="Fang Bai"/>
  <meta name="DC.Contributor" content="Hong Liu"/>
  <meta name="DC.Contributor" content="Xiang Liu"/>
  <meta name="DC.Contributor" content="Luke W. Guddat"/>
  <meta name="DC.Contributor" content="Wenqing Xu"/>
  <meta name="DC.Contributor" content="Gengfu Xiao"/>
  <meta name="DC.Contributor" content="Chengfeng Qin"/>
  <meta name="DC.Contributor" content="Zhengli Shi"/>
  <meta name="DC.Contributor" content="Hualiang Jiang"/>
  <meta name="DC.Contributor" content="Zihe Rao"/>
  <meta name="DC.Contributor" content="Haitao Yang"/>
  <meta name="article:published_time" content="2020-03-29"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Structure of Mpro from COVID-19 virus and discovery of its inhibitors"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;A new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.26.964882v3"/>
  <meta name="citation_id" content="2020.02.26.964882v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.964882v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.964882v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.964882v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.26.964882"/>
  <meta name="citation_num_pages" content="34"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.26.964882"/>
  <meta name="citation_author" content="Zhenming Jin"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author_institution" content="Laboratory of Structural Biology, School of Life Sciences and School of Medicine, Tsinghua University"/>
  <meta name="citation_author" content="Xiaoyu Du"/>
  <meta name="citation_author_institution" content="Laboratory of Structural Biology, School of Life Sciences and School of Medicine, Tsinghua University"/>
  <meta name="citation_author" content="Yechun Xu"/>
  <meta name="citation_author_institution" content="Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Yongqiang Deng"/>
  <meta name="citation_author_institution" content="Department of Virology, State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Academy of Military Medical Sciences"/>
  <meta name="citation_author" content="Meiqin Liu"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Yao Zhao"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Bing Zhang"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Xiaofeng Li"/>
  <meta name="citation_author_institution" content="Department of Virology, State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Academy of Military Medical Sciences"/>
  <meta name="citation_author" content="Leike Zhang"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Chao Peng"/>
  <meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Science"/>
  <meta name="citation_author" content="Yinkai Duan"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Jing Yu"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Lin Wang"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Kailin Yang"/>
  <meta name="citation_author_institution" content="Taussig Cancer Center, Cleveland Clinic"/>
  <meta name="citation_author" content="Fengjiang Liu"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Rendi Jiang"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Xinglou Yang"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Tian You"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Xiaoce Liu"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Xiuna Yang"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Fang Bai"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author" content="Hong Liu"/>
  <meta name="citation_author_institution" content="Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Xiang Liu"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Medicinal Chemical Biology, Frontiers Science Center for Cell Response, College of Life Sciences, College of Pharmacy, Nankai University"/>
  <meta name="citation_author" content="Luke W. Guddat"/>
  <meta name="citation_author_institution" content="School of Chemistry and Molecular Biosciences, the University of Queensland"/>
  <meta name="citation_author" content="Wenqing Xu"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author_institution" content="National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai Advanced Research Institute, Chinese Academy of Science"/>
  <meta name="citation_author" content="Gengfu Xiao"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Chengfeng Qin"/>
  <meta name="citation_author_institution" content="Department of Virology, State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Academy of Military Medical Sciences"/>
  <meta name="citation_author" content="Zhengli Shi"/>
  <meta name="citation_author_institution" content="CAS Key Laboratory of Special Pathogens, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Hualiang Jiang"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author_institution" content="Drug Discovery and Design Center, Shanghai Institute of Materia Medica, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Zihe Rao"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author_institution" content="Laboratory of Structural Biology, School of Life Sciences and School of Medicine, Tsinghua University"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Medicinal Chemical Biology, Frontiers Science Center for Cell Response, College of Life Sciences, College of Pharmacy, Nankai University"/>
  <meta name="citation_author" content="Haitao Yang"/>
  <meta name="citation_author_institution" content="Shanghai Institute for Advanced Immunochemical Studies and School of Life Science and Technology, ShanghaiTech University"/>
  <meta name="citation_author_email" content="yanght@shanghaitech.edu.cn"/>
  <meta name="citation_reference" content="Zhu, N. et al. A novel coronavirus from patients with pneumonia in China, 2019. New England Journal of Medicine (2020)."/>
  <meta name="citation_reference" content="Qun Li et al. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia. New England Journal of Medicine (2020)."/>
  <meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, doi:10.1038/s41586-020-2012-7 (2020)."/>
  <meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature, doi:10.1038/s41586-020-2008-3 (2020)."/>
  <meta name="citation_reference" content="Anand, K. et al. Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain. The EMBO Journal 21, 3213–3224 (2002)."/>
  <meta name="citation_reference" content="Yang, H. T. et al. The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor. Proceedings of the National Academy of Sciences of the United States of America 100, 13190–13195, doi:10.1073/pnas.1835675100 (2003)."/>
  <meta name="citation_reference" content="de Wit, E., van Doremalen, N., Falzarano, D. &amp;; Munster, V. J. SARS and MERS: recent insights into emerging coronaviruses. Nature Reviews Microbiology 14, 523–534, doi:10.1038/nrmicro.2016.81 (2016)."/>
  <meta name="citation_reference" content="Hegyi, A. &amp;; Ziebuhr, J. Conservation of substrate specificities among coronavirus main proteases. Journal of general virology 83, 595–599 (2002)."/>
  <meta name="citation_reference" content="Pillaiyar, T., Manickam, M., Namasivayam, V., Hayashi, Y. &amp;; Jung, S. H. An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy. Journal of Medicinal Chemistry 59, 6595–6628, doi:10.1021/acs.jmedchem.5b01461 (2016)."/>
  <meta name="citation_reference" content="Xue, X. Y. et al. Production of authentic SARS-CoV M-pro with enhanced activity: Application as a novel tag-cleavage endopeptidase for protein overproduction. Journal of Molecular Biology 366, 965–975, doi:10.1016/j.jmb.2006.11.073 (2007)."/>
  <meta name="citation_reference" content="Wang, Q. M., Johnson, R. B., Cox, G. A., Villarreal, E. C. &amp;; Loncharich, R. J. A Continuous Colorimetric Assay for Rhinovirus-14 3C Protease Using Peptidep-Nitroanilides as Substrates. Analytical biochemistry 252, 238–245 (1997)."/>
  <meta name="citation_reference" content="Yang, H. T. et al. Design of wide-spectrum inhibitors targeting coronavirus main proteases. Plos Biology 3, 2044–2044, doi:10.1371/journal.pbio.0030428 (2005)."/>
  <meta name="citation_reference" content="Xue, X. Y. et al. Structures of two coronavirus main proteases: Implications for substrate binding and antiviral drug design. Journal of Virology 82, 2515–2527, doi:10.1128/jvi.02114-07 (2008)."/>
  <meta name="citation_reference" content="Ren, Z. L. et al. The newly emerged SARS-Like coronavirus HCoV-EMC also has an “Achilles’ heel”: current effective inhibitor targeting a 3C-like protease. Protein &amp;; Cell 4, 248–250, doi:10.1007/s13238-013-2841-3 (2013)."/>
  <meta name="citation_reference" content="Wang, F., Chen, C., Tan, W., Yang, K. &amp;; Yang, H. Structure of Main Protease from Human Coronavirus NL63: Insights for Wide Spectrum Anti-Coronavirus Drug Design. Sci Rep 6, 22677–22677, doi:10.1038/srep22677 (2016)."/>
  <meta name="citation_reference" content="Zhao, Q. et al. Structure of the main protease from a global infectious human coronavirus, HCoV-HKU1. Journal of virology 82, 8647–8655, doi:10.1128/JVI.00298-08 (2008)."/>
  <meta name="citation_reference" content="Lee, C.-C. et al. Structural basis of inhibition specificities of 3C and 3C-like proteases by zinc-coordinating and peptidomimetic compounds. J Biol Chem 284, 7646–7655, doi:10.1074/jbc.M807947200 (2009)."/>
  <meta name="citation_reference" content="Ma, Q., Xiao, Y., Hilgenfeld, R. Inhibitor for the Main Protease of Coronavirus Hku4 (2012)."/>
  <meta name="citation_reference" content="Wang, F. H. et al. Crystal Structure of Feline Infectious Peritonitis Virus Main Protease in Complex with Synergetic Dual Inhibitors. Journal of Virology 90, 1910–1917, doi:10.1128/jvi.02685-15 (2016)."/>
  <meta name="citation_reference" content="Wang, F. H. et al. Michael Acceptor-Based Peptidomimetic Inhibitor of Main Protease from Porcine Epidemic Diarrhea Virus. Journal of Medicinal Chemistry 60, 3212–3216, doi:10.1021/acs.jmedchem.7b00103 (2017)."/>
  <meta name="citation_reference" content="Cui, W. et al. The crystal structure of main protease from mouse hepatitis virus A59 in complex with an inhibitor. Biochemical and Biophysical Research Communications 511, 794–799, doi:10.1016/j.bbrc.2019.02.105 (2019)."/>
  <meta name="citation_reference" content="Rubin, B. &amp;; Waugh, M. H. Antiphlogistic effects of antiserotonin (SQ 10,643) and aminopyrine in rats versus endotoxin and other agents. Proceedings of the Society for Experimental Biology and Medicine 119, 438–443 (1965)."/>
  <meta name="citation_reference" content="Chen, L. et al. Cinanserin is an inhibitor of the 3C-like proteinase of severe acute respiratory syndrome coronavirus and strongly reduces virus replication in vitro. Journal of virology 79, 7095–7103 (2005)."/>
  <meta name="citation_reference" content="Feng, B. Y. &amp;; Shoichet, B. K. A detergent-based assay for the detection of promiscuous inhibitors. Nature protocols 1, 550–553 (2006)."/>
  <meta name="citation_reference" content="Singh, N. et al. A safe lithium mimetic for bipolar disorder. Nature communications 4, 1–7 (2013)."/>
  <meta name="citation_reference" content="Lynch, E. &amp;; Kil, J. Development of ebselen, a glutathione peroxidase mimic, for the prevention and treatment of noise-induced hearing loss. Seminars in Hearing 30, 047–055 (2009)."/>
  <meta name="citation_reference" content="Kil, J. et al. Safety and efficacy of ebselen for the prevention of noise-induced hearing loss: a randomised, double-blind, placebo-controlled, phase 2 trial. The Lancet 390, 969–979 (2017)."/>
  <meta name="citation_reference" content="Renson, M., Etschenberg, E. &amp;; Winkelmann, J. 2-Phenyl-1, 2-benzisoselenazol-3 (2H)-one containing pharmaceutical preparations and process for the treatment of rheumatic diseases. (1982)."/>
  <meta name="citation_reference" content="Masaki, C. et al. Effects of the potential lithium-mimetic, ebselen, on impulsivity and emotional processing. Psychopharmacology 233, 2655–2661 (2016)."/>
  <meta name="citation_reference" content="Xu, M. et al. Identification of small-molecule inhibitors of Zika virus infection and induced neural cell death via a drug repurposing screen. Nature Medicine 22, 1101–1107, doi:10.1038/nm.4184 (2016)."/>
  <meta name="citation_reference" content="Aulner, N., Danckaert, A., Ihm, J., Shum, D. &amp;; Shorte, S. L. Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases. Trends in parasitology (2019)."/>
  <meta name="citation_reference" content="Anand, K., Ziebuhr, J., Wadhwani, P., Mesters, J. R. &amp;; Hilgenfeld, R. Coronavirus main proteinase (3CL(pro)) structure: Basis for design of anti-SARS drugs. Science 300, 1763–1767, doi:10.1126/science.1085658 (2003)."/>
  <meta name="citation_reference" content="Winter, G. xia2: an expert system for macromolecular crystallography data reduction. Journal of applied crystallography 43, 186–190 (2010)."/>
  <meta name="citation_reference" content="McCoy, A. J. et al. Phaser crystallographic software. Journal of applied crystallography 40, 658–674 (2007)."/>
  <meta name="citation_reference" content="Potterton, L. et al. CCP4i2: the new graphical user interface to the CCP4 program suite. Acta Crystallographica Section D-Structural Biology 74, 68–84, doi:10.1107/s2059798317016035 (2018)."/>
  <meta name="citation_reference" content="Emsley, P., Lohkamp, B., Scott, W. G. &amp;; Cowtan, K. Features and development of Coot. Acta Crystallographica Section D: Biological Crystallography 66, 486–501 (2010)."/>
  <meta name="citation_reference" content="Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. Acta Crystallographica Section D-Structural Biology 68, 352–367, doi:10.1107/s0907444912001308 (2012)."/>
  <meta name="citation_reference" content="Friesner, R. A. et al. Glide: A new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy. Journal of Medicinal Chemistry 47, 1739–1749, doi:10.1021/jm0306430 (2004)."/>
  <meta name="citation_reference" content="Greenwood, J. R., Calkins, D., Sullivan, A. P. &amp;; Shelley, J. C. Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution. Journal of computer-aided molecular design 24, 591–604 (2010)."/>
  <meta name="citation_reference" content="Bai, F. et al. Free energy landscape for the binding process of Huperzine A to acetylcholinesterase. Proceedings of the National Academy of Sciences of the United States of America 110, 4273–4278, doi:10.1073/pnas.1301814110 (2013)."/>
  <meta name="citation_reference" content="Liu, X. F., Jiang, H. L. &amp;; Li, H. L. SHAFTS: A Hybrid Approach for 3D Molecular Similarity Calculation. 1. Method and Assessment of Virtual Screening. Journal of Chemical Information and Modeling 51, 2372–2385, doi:10.1021/ci200060s (2011)."/>
  <meta name="citation_reference" content="Guimaraes, C. R. W. &amp;; Cardozo, M. MM-GB/SA rescoring of docking poses in structure-based lead optimization. Journal of Chemical Information and Modeling 48, 958–970, doi:10.1021/ci800004w (2008)."/>
  <meta name="citation_reference" content="Wang, M. et al. Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro. Cell Research 30, 269–271, doi:10.1038/s41422-020-0282-0 (2020)."/>
  <meta name="twitter:title" content="Structure of Mpro from COVID-19 virus and discovery of its inhibitors"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="A new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan[1][1]–[4][2]. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus[5][3],[6][4]. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available. [1]: #ref-1 [2]: #ref-4 [3]: #ref-5 [4]: #ref-6"/>
  <meta name="og-title" property="og:title" content="Structure of Mpro from COVID-19 virus and discovery of its inhibitors"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.26.964882v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="A new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan[1][1]–[4][2]. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (Mpro). Mpro is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus[5][3],[6][4]. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus Mpro in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of Mpro. Six of these inhibit Mpro with IC50 values ranging from 0.67 to 21.4 μM. Ebselen also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available. [1]: #ref-1 [2]: #ref-4 [3]: #ref-5 [4]: #ref-6"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-29"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Structure of Mpro from COVID-19 virus and discovery of its inhibitors | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1216000 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">SUMMARY</h2>
    <p id="p-4">A new coronavirus (CoV) identified as COVID-19 virus is the etiological agent responsible for the 2019-2020 viral pneumonia outbreak that commenced in Wuhan
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. Currently there is no targeted therapeutics and effective treatment options remain very limited. In order to rapidly discover lead compounds for clinical use, we initiated a program of combined structure-assisted drug design, virtual drug screening and high-throughput screening to identify new drug leads that target the COVID-19 virus main protease (M
     <sup>pro</sup>). M
     <sup>pro</sup> is a key CoV enzyme, which plays a pivotal role in mediating viral replication and transcription, making it an attractive drug target for this virus
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>. Here, we identified a mechanism-based inhibitor, N3, by computer-aided drug design and subsequently determined the crystal structure of COVID-19 virus M
     <sup>pro</sup> in complex with this compound. Next, through a combination of structure-based virtual and high-throughput screening, we assayed over 10,000 compounds including approved drugs, drug candidates in clinical trials, and other pharmacologically active compounds as inhibitors of M
     <sup>pro</sup>. Six of these inhibit M
     <sup>pro</sup> with IC
     <sub>50</sub> values ranging from 0.67 to 21.4 μM. Ebselen also exhibited promising antiviral activity in cell-based assays. Our results demonstrate the efficacy of this screening strategy, which can lead to the rapid discovery of drug leads with clinical potential in response to new infectious diseases where no specific drugs or vaccines are available.
    </p>
   </div>
   <p id="p-5">CoVs infect humans and other animal species, causing a variety of highly prevalent and severe diseases, including Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS)
    <sup>
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
    </sup>. The COVID-19 virus genome is comprised of ∼30,000 nucleotides; its replicase gene encodes two overlapping polyproteins, pp1a and pp1ab, required for viral replication and transcription
    <sup>
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
    </sup>. The functional polypeptides are released from the polyproteins by extensive proteolytic processing, predominantly by a 33.8-kDa main protease (M
    <sup>pro</sup>), also referred to as the 3C-like protease. M
    <sup>pro</sup> digests the polyprotein at no less than 11 conserved sites, starting with the autolytic cleavage of this enzyme itself from pp1a and pp1ab
    <sup>
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
    </sup>. The functional importance of M
    <sup>pro</sup> in the viral life cycle, together with the absence of closely related homologues in humans, identify the M
    <sup>pro</sup> as an attractive target for antiviral drug design
    <sup>
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
    </sup>.
   </p>
   <p id="p-6">To facilitate the rapid discovery of antiviral compounds with clinical potential, we developed a strategy combining structure-assisted drug design, virtual drug screening and high-throughput screening to repurpose existing drugs to target COVID-19 virus M
    <sup>pro</sup>.
   </p>
   <div class="section" id="sec-1">
    <h2 class="">Establishing a high-throughput activity assay</h2>
    <p id="p-7">Recombinant COVID-19 virus M
     <sup>pro</sup> with native N and C termini was expressed in 
     <em>Escherichia coli</em> and subsequently purified (
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Extended Data Fig. 1a, b</a>). The molecular weight of COVID-19 virus M
     <sup>pro</sup> as determined by mass spectroscopy is 33797.0 Da, consistent with its theoretical molecular weight 33796.8 Da. In order to characterize its enzymatic activity and to carry out high-throughput screening of inhibitors, we developed a fluorescence resonance energy transfer (FRET) assay. To do this, a fluorescently labeled substrate, MCA-AVLQ↓SGFR-Lys(Dnp)-Lys-NH
     <sub>2</sub>, derived from the N-terminal auto-cleavage sequence of the viral protease was designed and synthesized for time-dependent kinetic analysis (
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">Extended Data Fig. 1e</a>). The catalytic efficiency (
     <em>k
      <sub>cat</sub>/K
      <sub>m</sub>
     </em>) for COVID-19 virus M
     <sup>pro</sup> was measured to be 28,500 M
     <sup>−1</sup>s
     <sup>−1</sup> which is slightly higher than that for SARS-CoV M
     <sup>pro</sup> (
     <em>k
      <sub>cat</sub>
     </em>/
     <em>K
      <sub>m</sub>
     </em>=26,500 M
     <sup>−1</sup>s
     <sup>−1</sup>)
     <sup>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>, but over 30-fold higher than that of human rhinovirus 3C protease (
     <em>k
      <sub>cat</sub>
     </em>/
     <em>K
      <sub>m</sub>
     </em>=920 M
     <sup>−1</sup>s
     <sup>−1</sup>)
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>.
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The purification of COVID-19 virus Mpro and the inhibitory assay of N3 compound. a, The SDS-PAGE gel of COVID-19 virus Mpro. The first lane: marker; the second lane: Mpro before treating with rhinovirus 3C protease; third lane: Mpro after the cleavage of C-terminal His tag. For gel source data, see Supplementary Figure 1. b, Size-exclusion chromatography profile of Mpro. c, The chemical structure of N3 inhibitor. d, Inhibition mechanism for N3. e, Typical inhibition curves for N3. f, Cytotoxicity assay of N3 on Vero cells, data are shown as mean [[#xB1]] s.e.m., n = 3 biological replicates. The data (a, b, f) are representative of three independent experiments with similar results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The purification of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; and the inhibitory assay of N3 compound.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The SDS-PAGE gel of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;. The first lane: marker; the second lane: M&lt;;sup&gt;;pro&lt;;/sup&gt;; before treating with rhinovirus 3C protease; third lane: M&lt;;sup&gt;;pro&lt;;/sup&gt;; after the cleavage of C-terminal His tag. For gel source data, see Supplementary Figure 1. &lt;;strong&gt;;b,&lt;;/strong&gt;; Size-exclusion chromatography profile of M&lt;;sup&gt;;pro&lt;;/sup&gt;;. &lt;;strong&gt;;c,&lt;;/strong&gt;; The chemical structure of N3 inhibitor. &lt;;strong&gt;;d,&lt;;/strong&gt;; Inhibition mechanism for N3. &lt;;strong&gt;;e,&lt;;/strong&gt;; Typical inhibition curves for N3. &lt;;strong&gt;;f,&lt;;/strong&gt;; Cytotoxicity assay of N3 on Vero cells, data are shown as mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 3 biological replicates. The data (&lt;;strong&gt;;a, b, f&lt;;/strong&gt;;) are representative of three independent experiments with similar results.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F5.medium.gif" width="369" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F5.medium.gif" width="369" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 1</span>
      <span class="caption-title">The purification of COVID-19 virus M
       <sup>pro</sup> and the inhibitory assay of N3 compound.
      </span>
      <p id="p-37" class="first-child">
       <strong>a,</strong> The SDS-PAGE gel of COVID-19 virus M
       <sup>pro</sup>. The first lane: marker; the second lane: M
       <sup>pro</sup> before treating with rhinovirus 3C protease; third lane: M
       <sup>pro</sup> after the cleavage of C-terminal His tag. For gel source data, see Supplementary Figure 1. 
       <strong>b,</strong> Size-exclusion chromatography profile of M
       <sup>pro</sup>. 
       <strong>c,</strong> The chemical structure of N3 inhibitor. 
       <strong>d,</strong> Inhibition mechanism for N3. 
       <strong>e,</strong> Typical inhibition curves for N3. 
       <strong>f,</strong> Cytotoxicity assay of N3 on Vero cells, data are shown as mean ± s.e.m., 
       <em>n</em> = 3 biological replicates. The data (
       <strong>a, b, f</strong>) are representative of three independent experiments with similar results.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">N3 is a potent irreversible inhibitor of COVID-19 virus M
     <sup>pro</sup>
    </h2>
    <p id="p-8">In a previous study, we designed a Michael acceptor inhibitor N3 using computer-aided drug design (CADD) (
     <a id="xref-fig-5-3" class="xref-fig" href="#F5">Extended Data Fig. 1c</a>), which can specifically inhibit multiple CoV M
     <sup>pro</sup>s, including those from SARS-CoV and MERS-CoV
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup>. It also has displayed potent antiviral activity against infectious bronchitis virus in an animal model
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup>. The CC
     <sub>50</sub> of N3 is &gt;;133 μM (
     <a id="xref-fig-5-4" class="xref-fig" href="#F5">Extended Data Fig. 1f</a>). Next, we constructed a homology model for COVID-19 virus M
     <sup>pro</sup> and used molecular docking to see if N3 could target this new CoV M
     <sup>pro</sup>. A docking pose showed that it could fit inside the substrate-binding pocket. To assess the efficacy of N3 for COVID-19 virus M
     <sup>pro</sup>, kinetic analysis was performed. A progress curve showed that it is a time-dependent irreversible inhibitor of this enzyme. Further, the shape of this curve supports the mechanism of two-step irreversible inactivation. The inhibitor first associates with COVID-19 virus M
     <sup>pro</sup> (EI) with a dissociation constant 
     <em>K
      <sub>i</sub>
     </em>; then, a stable covalent bond is formed between N3 and M
     <sup>pro</sup> (E–I). The evaluation of this time-dependent inhibition requires both the equilibrium-binding constant 
     <em>K
      <sub>i</sub>
     </em> (designated as 
     <em>k
      <sub>2</sub>/k
      <sub>1</sub>
     </em>) and the inactivation rate constant for covalent bond formation 
     <em>k</em>
     <sub>3</sub>. However, N3 exhibits very potent inhibition of COVID-19 virus M
     <sup>pro</sup>, such that measurement of 
     <em>K
      <sub>i</sub>
     </em> and 
     <em>k
      <sub>3</sub>
     </em> proved not feasible (
     <a id="xref-fig-5-5" class="xref-fig" href="#F5">Extended Data Fig. 1d, e</a>). When very rapid inactivation occurs, 
     <em>k
      <sub>obs</sub>
     </em>/[I] was utilized to evaluate the inhibition as an approximation of the pseudo second-order rate constant (
     <em>k
      <sub>3</sub>
     </em>/
     <em>K
      <sub>i</sub>
     </em>)
     <sup>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>
     </sup>. The value of 
     <em>k
      <sub>obs</sub>
     </em>/[I] of N3 for COVID-19 virus M
     <sup>pro</sup> was determined to be 11,300±880 M
     <sup>−1</sup>s
     <sup>−1</sup>, suggesting this Michael acceptor has potent inhibition.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">The crystal structure of COVID-19 virus M
     <sup>pro</sup> in complex with N3
    </h2>
    <p id="p-9">In order to elucidate the inhibitory mechanism of this compound, we determined the crystal structure of COVID-19 virus M
     <sup>pro</sup> in complex with N3 to 2.1-Å resolution. The asymmetric unit contains only one polypeptide (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Extended Data Table 1</a>). However, two of these associate to form a dimer by a crystallographic 2-fold symmetry axis (the two molecules are designated protomer A and B) (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1b</a>). All residues (residues 1–306) are visible in electron density maps. Each protomer is composed of three domains (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>). Domains I (residues 8–101) and II (residues 102–184) have an antiparallel β-barrel structure. Domain III (residues 201–303) contains five α-helices arranged into a largely antiparallel globular cluster, and is connected to domain II by means of a long loop region (residues 185–200). COVID-19 virus M
     <sup>pro</sup> has a Cys–His catalytic dyad, and the substrate-binding site is located in a cleft between Domain I and II. These features are similar to those of other M
     <sup>pro</sup>s reported previously
     <sup>
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>–
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
     </sup>. The electron density map shows that N3 binds in the substrate-binding pocket in an extended conformation (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1c</a>, 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Extended Data Fig. 2</a>), with the inhibitor backbone atoms forming an antiparallel sheet with residues 164–168 of the long strand
     <sub>155-168</sub> on one side, and with residues 189–191 of the loop linking domains II and III.
    </p>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The interactions between COVID-19 virus Mpro and N3. a, The Fo-Fc omit map (contour level = 3 [[#x3C3]], shown as the blue mesh). b, Detailed view of the interactions between the inhibitor N3 and COVID-19 virus Mpro. Mpro residues are shown in blue (Protomer A) and salmon (Protomer B); N3 is in green, water is in black. The hydrogen bonds are shown as black dashed lines. The covalent bond between N3 and C145-A is in purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The interactions between COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; and N3.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The &lt;;em&gt;;F&lt;;sub&gt;;o&lt;;/sub&gt;;-F&lt;;sub&gt;;c&lt;;/sub&gt;;&lt;;/em&gt;; omit map (contour level = 3 [[#x3C3]], shown as the blue mesh). &lt;;strong&gt;;b,&lt;;/strong&gt;; Detailed view of the interactions between the inhibitor N3 and COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;. M&lt;;sup&gt;;pro&lt;;/sup&gt;; residues are shown in blue (Protomer A) and salmon (Protomer B); N3 is in green, water is in black. The hydrogen bonds are shown as black dashed lines. The covalent bond between N3 and C145-A is in purple.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F6.medium.gif" width="211" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F6.medium.gif" width="211" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 2</span>
      <span class="caption-title">The interactions between COVID-19 virus M
       <sup>pro</sup> and N3.
      </span>
      <p id="p-38" class="first-child">
       <strong>a,</strong> The 
       <em>F
        <sub>o</sub>-F
        <sub>c</sub>
       </em> omit map (contour level = 3 σ, shown as the blue mesh). 
       <strong>b,</strong> Detailed view of the interactions between the inhibitor N3 and COVID-19 virus M
       <sup>pro</sup>. M
       <sup>pro</sup> residues are shown in blue (Protomer A) and salmon (Protomer B); N3 is in green, water is in black. The hydrogen bonds are shown as black dashed lines. The covalent bond between N3 and C145-A is in purple.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1223694/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1223694/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1223694" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Extended Data Table 1</span>
      <span class="caption-title">Data collection and refinement statistics</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The crystal structure of COVID-19 virus Mpro in complex with N3. a, Cartoon representation of one protomer of the dimeric Mpro-inhibitor complex. b, Surface representation of the homodimer of Mpro. Protomer A is in blue, protomer B is in salmon, N3 is presented as green sticks. c, A zoomed view of the substrate-binding pocket. The key residues forming the binding pocket are shown in sticks, the two waters, assigned as W1 and W2, are shown as red spheres. P1, P1[[#x2019]], P2, P3, P4 and P5 sites of N3 are indicated. Hydrogen bonds that help to lock the inhibitor are shown in black dashed lines. The 2Fo-Fc density map contoured at 1.2 [[#x3C3]] is shown around N3 molecule (blue mesh), C145-A (yellow mesh), and the two waters (blue mesh). d, The C-S covalent bond." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The crystal structure of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; in complex with N3.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; Cartoon representation of one protomer of the dimeric M&lt;;sup&gt;;pro&lt;;/sup&gt;;-inhibitor complex. &lt;;strong&gt;;b,&lt;;/strong&gt;; Surface representation of the homodimer of M&lt;;sup&gt;;pro&lt;;/sup&gt;;. Protomer A is in blue, protomer B is in salmon, N3 is presented as green sticks. &lt;;strong&gt;;c,&lt;;/strong&gt;; A zoomed view of the substrate-binding pocket. The key residues forming the binding pocket are shown in sticks, the two waters, assigned as W1 and W2, are shown as red spheres. P1, P1[[#x2019]], P2, P3, P4 and P5 sites of N3 are indicated. Hydrogen bonds that help to lock the inhibitor are shown in black dashed lines. The 2&lt;;em&gt;;F&lt;;sub&gt;;o&lt;;/sub&gt;;-F&lt;;sub&gt;;c&lt;;/sub&gt;;&lt;;/em&gt;; density map contoured at 1.2 [[#x3C3]] is shown around N3 molecule (blue mesh), C145-A (yellow mesh), and the two waters (blue mesh). &lt;;strong&gt;;d,&lt;;/strong&gt;; The C-S covalent bond.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F1.medium.gif" width="440" height="131"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F1.medium.gif" width="440" height="131"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 1</span>
      <span class="caption-title">The crystal structure of COVID-19 virus M
       <sup>pro</sup> in complex with N3.
      </span>
      <p id="p-10" class="first-child">
       <strong>a,</strong> Cartoon representation of one protomer of the dimeric M
       <sup>pro</sup>-inhibitor complex. 
       <strong>b,</strong> Surface representation of the homodimer of M
       <sup>pro</sup>. Protomer A is in blue, protomer B is in salmon, N3 is presented as green sticks. 
       <strong>c,</strong> A zoomed view of the substrate-binding pocket. The key residues forming the binding pocket are shown in sticks, the two waters, assigned as W1 and W2, are shown as red spheres. P1, P1’, P2, P3, P4 and P5 sites of N3 are indicated. Hydrogen bonds that help to lock the inhibitor are shown in black dashed lines. The 2
       <em>F
        <sub>o</sub>-F
        <sub>c</sub>
       </em> density map contoured at 1.2 σ is shown around N3 molecule (blue mesh), C145-A (yellow mesh), and the two waters (blue mesh). 
       <strong>d,</strong> The C-S covalent bond.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">Here we detail the specific interactions of N3 with M
     <sup>pro</sup> (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1c, d</a>). The electron density shows that the Sγ atom of C145-A forms a covalent bond (1.8-Å) with the Cβ of the vinyl group, confirming that the Michael addition has occurred. The S1 subsite has an absolute requirement for Gln at the P1 position. The side chains of F140-A, N142-A, E166-A, H163-A, H172-A, S1-B (from protomer B), and main chains of F140-A and L141-A are involved in S1 subsite formation, which also includes two ordered water molecules (named W1 and W2). The lactam at P1 inserts into the S1 subsite and forms a hydrogen bond with H163-A. The side chain of Leu at P2 site inserts deeply into the hydrophobic S2 subsite, which consists of the side chains of H41-A, M49-A, Y54-A, M165-A, and the alkyl portion of the side chain of D187-A. The side chain of Val at P3 is solvent-exposed, indicating that this site can tolerate a wide range of functional groups. The side chain of Ala at P4 side is surrounded by the side chains of M165-A, L167-A, F185-A, Q192-A and the main chain of Q189-A, all of which form a small hydrophobic pocket. P5 makes van der Waals contacts with P168-A and the backbone of residues 190–191. The bulky benzyl group extends into the S1’ site, forming van der Waals interactions with T24-A and T25-A. In addition, N3 forms multiple hydrogen bonds with the main chain of the residues in the substrate-binding pocket, which also helps lock the inhibitor inside the substrate-binding pocket.
    </p>
    <p id="p-12">An overlay of the structures of COVID-19 virus M
     <sup>pro</sup>-N3 and SARS-CoV M
     <sup>pro</sup>-N1
     <sup>
      <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>
     </sup> shows that N3 and N1 bind to M
     <sup>pro</sup>s in a similar mode (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>, 
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">Extended Data Fig. 3</a>). The major difference lies in the P1’ site. Compared with the benzyl ester portion of N3 in the COVID-19 virus M
     <sup>pro</sup> structure, the ethyl ester portion in N1 adopts a slightly different conformation. This may be attributed to an ordered water (W1) in SARS-CoV M
     <sup>pro</sup>-N1 structure, which makes a long-distance hydrogen bond to the carboxylate oxygen of the ester and also forms two hydrogen bonds from the backbone NH of G143 and the side chain of N142. In our previous study, we proposed that all the CoV M
     <sup>pro</sup>s share a highly conserved substrate-recognition pocket, which could serve as a drug target for the design of broad-spectrum inhibitors
     <sup>
      <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>
     </sup>. The recent discovery of new CoVs and accumulation of structural data for CoV M
     <sup>pro</sup>s from various species provided the opportunity to further examine this hypothesis. Superposition of the 12 crystal structures of M
     <sup>pro</sup>s
     <sup>
      <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>–
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup> have shown that the most variable regions were the helical domain III and surface loops, but the substrate-binding pockets located in a cleft between domains I and II are still highly conserved among all CoV M
     <sup>pro</sup>s, suggesting the antiviral inhibitors targeting this site should have wide-spectrum anti-CoV activity (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b, c</a>).
    </p>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Comparison of the binding modes between COVID-19 virus Mpro-N3 and SARS-CoV Mpro-N1. a, The chemical structure of N1 inhibitor. b, The binding mode of COVID-19 virus Mpro (blue sticks) with N3 (green sticks). c, The binding mode of SARS-CoV Mpro (grey sticks) with N1 (pink sticks). The hydrogen bonds formed by water (W1) are indicated by the dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Comparison of the binding modes between COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;-N3 and SARS-CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;-N1.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The chemical structure of N1 inhibitor. &lt;;strong&gt;;b,&lt;;/strong&gt;; The binding mode of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; (blue sticks) with N3 (green sticks). &lt;;strong&gt;;c,&lt;;/strong&gt;; The binding mode of SARS-CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;; (grey sticks) with N1 (pink sticks). The hydrogen bonds formed by water (W1) are indicated by the dashed lines.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F7.medium.gif" width="440" height="270"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F7.medium.gif" width="440" height="270"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 3</span>
      <span class="caption-title">Comparison of the binding modes between COVID-19 virus M
       <sup>pro</sup>-N3 and SARS-CoV M
       <sup>pro</sup>-N1.
      </span>
      <p id="p-39" class="first-child">
       <strong>a,</strong> The chemical structure of N1 inhibitor. 
       <strong>b,</strong> The binding mode of COVID-19 virus M
       <sup>pro</sup> (blue sticks) with N3 (green sticks). 
       <strong>c,</strong> The binding mode of SARS-CoV M
       <sup>pro</sup> (grey sticks) with N1 (pink sticks). The hydrogen bonds formed by water (W1) are indicated by the dashed lines.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The substrate-binding pockets of CoV Mpros across different species. a, Comparison of inhibitor binding mode between the structures of COVID-19 virus Mpro-N3 and SARS-CoV Mpro-N1. COVID-19 virus Mpro is shown in marine cartoon; SARS-CoV Mpro in grey; N3 in green sticks; N1 in hot pink. b, Superposition of crystal structures of Mpros (C[[#x3B1]] 1-300) from 12 CoVs, including COVID-19 virus, SARS-CoV, MERS-CoV, HCoV-HKU1, BtCoV-HKU4, MHV-A59, PEDV, FIPV, TGEV, HCoV-NL63, HCoV-229E and IBV The color spectrum represents the root-mean-square deviation (RMSD) of the aligned C[[#x3B1]] atoms. c, Surface presentation of conserved substrate-binding pockets of 12 CoV Mpros. Red: residues are entirely identical among all 12 Mpros; violet: conserved substitution in one CoV Mpro; orange: conserved substitution in more than one CoV Mpros. S1, S2, S4, and S1[[#x2019]] subsites are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The substrate-binding pockets of CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;s across different species.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; Comparison of inhibitor binding mode between the structures of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;-N3 and SARS-CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;-N1. COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; is shown in marine cartoon; SARS-CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;; in grey; N3 in green sticks; N1 in hot pink. &lt;;strong&gt;;b,&lt;;/strong&gt;; Superposition of crystal structures of M&lt;;sup&gt;;pro&lt;;/sup&gt;;s (C[[#x3B1]] 1-300) from 12 CoVs, including COVID-19 virus, SARS-CoV, MERS-CoV, HCoV-HKU1, BtCoV-HKU4, MHV-A59, PEDV, FIPV, TGEV, HCoV-NL63, HCoV-229E and IBV The color spectrum represents the root-mean-square deviation (RMSD) of the aligned C[[#x3B1]] atoms. &lt;;strong&gt;;c,&lt;;/strong&gt;; Surface presentation of conserved substrate-binding pockets of 12 CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;s. Red: residues are entirely identical among all 12 M&lt;;sup&gt;;pro&lt;;/sup&gt;;s; violet: conserved substitution in one CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;; orange: conserved substitution in more than one CoV M&lt;;sup&gt;;pro&lt;;/sup&gt;;s. S1, S2, S4, and S1[[#x2019]] subsites are indicated.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F2.medium.gif" width="440" height="152"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F2.medium.gif" width="440" height="152"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 2</span>
      <span class="caption-title">The substrate-binding pockets of CoV M
       <sup>pro</sup>s across different species.
      </span>
      <p id="p-13" class="first-child">
       <strong>a,</strong> Comparison of inhibitor binding mode between the structures of COVID-19 virus M
       <sup>pro</sup>-N3 and SARS-CoV M
       <sup>pro</sup>-N1. COVID-19 virus M
       <sup>pro</sup> is shown in marine cartoon; SARS-CoV M
       <sup>pro</sup> in grey; N3 in green sticks; N1 in hot pink. 
       <strong>b,</strong> Superposition of crystal structures of M
       <sup>pro</sup>s (Cα 1-300) from 12 CoVs, including COVID-19 virus, SARS-CoV, MERS-CoV, HCoV-HKU1, BtCoV-HKU4, MHV-A59, PEDV, FIPV, TGEV, HCoV-NL63, HCoV-229E and IBV The color spectrum represents the root-mean-square deviation (RMSD) of the aligned Cα atoms. 
       <strong>c,</strong> Surface presentation of conserved substrate-binding pockets of 12 CoV M
       <sup>pro</sup>s. Red: residues are entirely identical among all 12 M
       <sup>pro</sup>s; violet: conserved substitution in one CoV M
       <sup>pro</sup>; orange: conserved substitution in more than one CoV M
       <sup>pro</sup>s. S1, S2, S4, and S1’ subsites are indicated.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Virtual screening</h2>
    <p id="p-14">The structure of COVID-19 virus M
     <sup>pro</sup> in complex with N3 provides a model for identifying lead inhibitors to target COVID-19 virus M
     <sup>pro</sup> through 
     <em>in silico</em> screening. To achieve this, an in-house database of potential binding compounds was docked using Glide (v8.2). The results show that cinanserin fits snugly into the substrate-binding pocket, by interacting with H41 and E166 of M
     <sup>pro</sup> by cation-π. Subsequently we determined this compound has an IC
     <sub>50</sub> value of 125 μM for M
     <sup>pro</sup>. Moreover, cinanserin is a well-characterized serotonin antagonist, which underwent preliminary clinical testing in humans in the 1960s
     <sup>
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
     </sup> and has previously been shown to inhibit SARS-CoV M
     <sup>pro23</sup>. The CC
     <sub>50</sub> of cinanserin is &gt;; 200 μM (
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">Extended Data Fig. 4</a>). Thus, it has potential for optimization as an anti-viral drug lead.
    </p>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cinanserin is an inhibitor for COVID-19 virus Mpro. a, The docking result of cinanserin. The structure of COVID-19 virus Mpro is shown as a white cartoon, cinanserin is shown as cyan balls and sticks, residues predicted to be interacting with cinanserin are shown as sticks. b, Inhibitory activity of cinanserin on Mpro. c, Antiviral activity of cinanserin determined by qRT-PCR. d, Cytotoxicity assay of cinanserin on Vero E6 cells. All data are shown as mean [[#xB1]] s.e.m., n = 3 biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Cinanserin is an inhibitor for COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The docking result of cinanserin. The structure of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; is shown as a white cartoon, cinanserin is shown as cyan balls and sticks, residues predicted to be interacting with cinanserin are shown as sticks. &lt;;strong&gt;;b,&lt;;/strong&gt;; Inhibitory activity of cinanserin on M&lt;;sup&gt;;pro&lt;;/sup&gt;;. &lt;;strong&gt;;c,&lt;;/strong&gt;; Antiviral activity of cinanserin determined by qRT-PCR. &lt;;strong&gt;;d,&lt;;/strong&gt;; Cytotoxicity assay of cinanserin on Vero E6 cells. All data are shown as mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 3 biological replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F8.medium.gif" width="440" height="341"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F8.medium.gif" width="440" height="341"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 4</span>
      <span class="caption-title">Cinanserin is an inhibitor for COVID-19 virus M
       <sup>pro</sup>.
      </span>
      <p id="p-40" class="first-child">
       <strong>a,</strong> The docking result of cinanserin. The structure of COVID-19 virus M
       <sup>pro</sup> is shown as a white cartoon, cinanserin is shown as cyan balls and sticks, residues predicted to be interacting with cinanserin are shown as sticks. 
       <strong>b,</strong> Inhibitory activity of cinanserin on M
       <sup>pro</sup>. 
       <strong>c,</strong> Antiviral activity of cinanserin determined by qRT-PCR. 
       <strong>d,</strong> Cytotoxicity assay of cinanserin on Vero E6 cells. All data are shown as mean ± s.e.m., 
       <em>n</em> = 3 biological replicates.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">High-throughput screening</h2>
    <p id="p-15">Next, we used our FRET assay, to screen a library of ∼10,000 compounds consisting of approved drugs, clinical trial drug candidates and natural products. Primary hits included seven compounds that are either FDA-approved drugs or clinical trial/preclinical drug candidates. We then determined their IC
     <sub>50</sub> values, which are in the range from 0.67 to 21.4 μM (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Amongst them, disulfiram and carmofur are FDA-approved drugs, whilst ebselen, shikonin, tideglusib, PX-12 and TDZD-8 are currently in clinical trials or preclinical studies. Ebselen has the strongest inhibition of M
     <sup>pro</sup> activity with an IC
     <sub>50</sub> of 0.67 μM. However, in a detergent-based assay
     <sup>
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
     </sup>, TDZD-8 was found to be an aggregate-based inhibitor, which might non-specifically inhibit M
     <sup>pro</sup> (
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">Extended Data Fig. 5</a>) and was therefore not considered for further investigation. Next, we set out to identify the potential covalent inhibitors among these compounds through tandem MS/MS analysis. The MS/MS data shows that ebselen, PX-12 and carmofur are all able to covalently bind to C145 of the catalytic dyad in COVID-19 virus M
     <sup>pro</sup>. However, while PX-12 and carmofur completely modified M
     <sup>pro</sup>, ebselen could only partially modify this viral cysteine protease (
     <a id="xref-fig-10-1" class="xref-fig" href="#F10">Extended Data Fig. 6</a>). Since ebselen has even stronger inhibition than the others, there is a possibility that ebselen could also inhibit M
     <sup>pro</sup> through non-covalent binding. It is likely that a portion of the hits identified by screening are covalently bonded to the catalytic cysteine of M
     <sup>pro</sup> through their sulfhydryl groups. In general, such molecules are expected to be promiscuous binders and therefore, as they stand, may have limited potential as drug leads. Since our structural data is based on N3, we investigated if molecular docking could predict how disulfiram, tideglusib and shikonin bind to this protein. In all cases, reasonable docking poses were found, demonstrating that they could fit inside the substrate-binding pocket (
     <a id="xref-fig-11-1" class="xref-fig" href="#F11">Extended Data Fig. 7</a>).
    </p>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The detergent-based assay for drug leads. a-f, The IC50 values determined by in the presence or absence of 0.01% Triton X-100, which showed that detergent did not affect the results. g, Different concentrations of Triton X-100 notably affected IC50 curves for TDZD-8. All data are shown as mean [[#xB1]] s.e.m., n = 3 biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The detergent-based assay for drug leads.&lt;;/span&gt;; &lt;;strong&gt;;a-f,&lt;;/strong&gt;; The IC&lt;;sub&gt;;50&lt;;/sub&gt;; values determined by in the presence or absence of 0.01% Triton X-100, which showed that detergent did not affect the results. &lt;;strong&gt;;g,&lt;;/strong&gt;; Different concentrations of Triton X-100 notably affected IC&lt;;sub&gt;;50&lt;;/sub&gt;; curves for TDZD-8. All data are shown as mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 3 biological replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F9.medium.gif" width="308" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F9.medium.gif" width="308" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 5</span>
      <span class="caption-title">The detergent-based assay for drug leads.</span>
      <p id="p-41" class="first-child">
       <strong>a-f,</strong> The IC
       <sub>50</sub> values determined by in the presence or absence of 0.01% Triton X-100, which showed that detergent did not affect the results. 
       <strong>g,</strong> Different concentrations of Triton X-100 notably affected IC
       <sub>50</sub> curves for TDZD-8. All data are shown as mean ± s.e.m., 
       <em>n</em> = 3 biological replicates.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Tandem MS/MS analysis reveals that ebselen, PX-12 and carmofur are able to covalently bind to C145 of COVID-19 virus Mpro. a, Molecular weight of apo COVID-19 virus Mpro and compounds treated Mpro. The mass shifts ([[#x394]]m) of the proteins indicate that more than one molecular of the compounds can be covalently bonded to one molecular of Mpro. b-e, A higher-energy collisional dissociation (HCD) MS/MS spectrum recorded on the [M+H]2+ ion b, at m/z 787.3852 of the Mpro unmodified peptide TIKGSFLNGSCGSVGF, c, at m/z 998.4152 of the Mpro modified peptide FTIKGSFLNGSCGSVGF harboring a modification (-C13H9NOSe) induced by ebselen on C145, d, at m/z 831.4080 of the Mpro modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C4H8S) induced by PX-12 on C145, e, at m/z 850.9414 of the Mpro modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C7H13NO) induced by carmofur on C145. Predicted b- and y-type ions (not including all) are listed above and below the peptide sequence, respectively. The experiment was performed once." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Tandem MS/MS analysis reveals that ebselen, PX-12 and carmofur are able to covalently bind to C145 of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; Molecular weight of apo COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; and compounds treated M&lt;;sup&gt;;pro&lt;;/sup&gt;;. The mass shifts ([[#x394]]m) of the proteins indicate that more than one molecular of the compounds can be covalently bonded to one molecular of M&lt;;sup&gt;;pro&lt;;/sup&gt;;. &lt;;strong&gt;;b-e,&lt;;/strong&gt;; A higher-energy collisional dissociation (HCD) MS/MS spectrum recorded on the [M+H]&lt;;sup&gt;;2+&lt;;/sup&gt;; ion &lt;;strong&gt;;b,&lt;;/strong&gt;; at m/z 787.3852 of the M&lt;;sup&gt;;pro&lt;;/sup&gt;; unmodified peptide TIKGSFLNGSCGSVGF, &lt;;strong&gt;;c,&lt;;/strong&gt;; at m/z 998.4152 of the M&lt;;sup&gt;;pro&lt;;/sup&gt;; modified peptide FTIKGSFLNGSCGSVGF harboring a modification (-C&lt;;sub&gt;;13&lt;;/sub&gt;;H&lt;;sub&gt;;9&lt;;/sub&gt;;NOSe) induced by ebselen on C145, &lt;;strong&gt;;d,&lt;;/strong&gt;; at m/z 831.4080 of the M&lt;;sup&gt;;pro&lt;;/sup&gt;; modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C&lt;;sub&gt;;4&lt;;/sub&gt;;H&lt;;sub&gt;;8&lt;;/sub&gt;;S) induced by PX-12 on C145, &lt;;strong&gt;;e,&lt;;/strong&gt;; at m/z 850.9414 of the M&lt;;sup&gt;;pro&lt;;/sup&gt;; modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C&lt;;sub&gt;;7&lt;;/sub&gt;;H&lt;;sub&gt;;13&lt;;/sub&gt;;NO) induced by carmofur on C145. Predicted b- and y-type ions (not including all) are listed above and below the peptide sequence, respectively. The experiment was performed once.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F10.medium.gif" width="374" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F10.medium.gif" width="374" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 6</span>
      <span class="caption-title">Tandem MS/MS analysis reveals that ebselen, PX-12 and carmofur are able to covalently bind to C145 of COVID-19 virus M
       <sup>pro</sup>.
      </span>
      <p id="p-42" class="first-child">
       <strong>a,</strong> Molecular weight of apo COVID-19 virus M
       <sup>pro</sup> and compounds treated M
       <sup>pro</sup>. The mass shifts (Δm) of the proteins indicate that more than one molecular of the compounds can be covalently bonded to one molecular of M
       <sup>pro</sup>. 
       <strong>b-e,</strong> A higher-energy collisional dissociation (HCD) MS/MS spectrum recorded on the [M+H]
       <sup>2+</sup> ion 
       <strong>b,</strong> at m/z 787.3852 of the M
       <sup>pro</sup> unmodified peptide TIKGSFLNGSCGSVGF, 
       <strong>c,</strong> at m/z 998.4152 of the M
       <sup>pro</sup> modified peptide FTIKGSFLNGSCGSVGF harboring a modification (-C
       <sub>13</sub>H
       <sub>9</sub>NOSe) induced by ebselen on C145, 
       <strong>d,</strong> at m/z 831.4080 of the M
       <sup>pro</sup> modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C
       <sub>4</sub>H
       <sub>8</sub>S) induced by PX-12 on C145, 
       <strong>e,</strong> at m/z 850.9414 of the M
       <sup>pro</sup> modified peptide TIKGSFLNGSCGSVGF harboring a modification(-C
       <sub>7</sub>H
       <sub>13</sub>NO) induced by carmofur on C145. Predicted b- and y-type ions (not including all) are listed above and below the peptide sequence, respectively. The experiment was performed once.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F11" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Docking Poses of different COVID-19 virus Mpro inhibitors. a, The crystal structure of COVID-19 virus Mpro-N3 complex. b-d, The docking results of three drug leads. Mpro is shown as grey background, inhibitors are in different colors. The inhibitors identified through the high-throughput screening are likely to occupy the same pocket as N3. e, Predicted binding affinities for the drug leads to COVID-19 virus Mpro by using MM-GBSA module integrated in Schr[[#xF6]]dinger." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Docking Poses of different COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; inhibitors.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The crystal structure of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;-N3 complex. &lt;;strong&gt;;b-d,&lt;;/strong&gt;; The docking results of three drug leads. M&lt;;sup&gt;;pro&lt;;/sup&gt;; is shown as grey background, inhibitors are in different colors. The inhibitors identified through the high-throughput screening are likely to occupy the same pocket as N3. &lt;;strong&gt;;e,&lt;;/strong&gt;; Predicted binding affinities for the drug leads to COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; by using MM-GBSA module integrated in Schr[[#xF6]]dinger.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F11.medium.gif" width="440" height="419"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 7" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F11.medium.gif" width="440" height="419"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 7" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 7</span>
      <span class="caption-title">Docking Poses of different COVID-19 virus M
       <sup>pro</sup> inhibitors.
      </span>
      <p id="p-43" class="first-child">
       <strong>a,</strong> The crystal structure of COVID-19 virus M
       <sup>pro</sup>-N3 complex. 
       <strong>b-d,</strong> The docking results of three drug leads. M
       <sup>pro</sup> is shown as grey background, inhibitors are in different colors. The inhibitors identified through the high-throughput screening are likely to occupy the same pocket as N3. 
       <strong>e,</strong> Predicted binding affinities for the drug leads to COVID-19 virus M
       <sup>pro</sup> by using MM-GBSA module integrated in Schrödinger.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Drug leads inhibit the activity of COVID-19 virus Mpro. a-f, The hydrolytic activity of COVID-19 virus Mpro was measured in the presence of varying concentrations of the drug candidates. Dose-response curves for half-maximum inhibitory concentration (IC50) values were determined by nonlinear regression. All data are shown as mean [[#xB1]] s.e.m., n = 3 biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Drug leads inhibit the activity of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;;.&lt;;/span&gt;; &lt;;strong&gt;;a-f,&lt;;/strong&gt;; The hydrolytic activity of COVID-19 virus M&lt;;sup&gt;;pro&lt;;/sup&gt;; was measured in the presence of varying concentrations of the drug candidates. Dose-response curves for half-maximum inhibitory concentration (IC&lt;;sub&gt;;50&lt;;/sub&gt;;) values were determined by nonlinear regression. All data are shown as mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 3 biological replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F3.medium.gif" width="410" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F3.medium.gif" width="410" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 3</span>
      <span class="caption-title">Drug leads inhibit the activity of COVID-19 virus M
       <sup>pro</sup>.
      </span>
      <p id="p-16" class="first-child">
       <strong>a-f,</strong> The hydrolytic activity of COVID-19 virus M
       <sup>pro</sup> was measured in the presence of varying concentrations of the drug candidates. Dose-response curves for half-maximum inhibitory concentration (IC
       <sub>50</sub>) values were determined by nonlinear regression. All data are shown as mean ± s.e.m., 
       <em>n</em> = 3 biological replicates.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Antiviral activity assay</h2>
    <p id="p-17">To further substantiate the enzymatic inhibition results 
     <em>in vitro</em>, we evaluated whether these compounds could prevent viral replication in cell-based assays. As shown in 
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>, quantitative real-time RT-PCR (qRT-PCR) demonstrated that ebselen and N3 showed the strongest antiviral effects among them at a concentration of 10 μM treatment in COVID-19 virus infected Vero cells. A plaque-reduction assay (
     <a id="xref-fig-12-1" class="xref-fig" href="#F12">Extended Data Fig. 8</a>) was performed to further assess the efficacy of these two compounds in protecting cells. Ebselen and N3 displayed inhibition against COVID-19 virus with individual EC50 values of 4.67 μM and 16.77 μM, respectively (
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b, c</a>). The dose-response curves suggest that both of them could be able to penetrate cellular membrane to access their targets. Ebselen is an organoselenium compound with anti-inflammatory, anti-oxidant and cytoprotective properties. This compound has been investigated for the treatment of multiple diseases, such as bipolar disorders
     <sup>
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
     </sup> and hearing loss
     <sup>
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
     </sup>. Ebselen has extremely low cytotoxicity (LD50 in rats &gt;; 4,600 mg/kg, per os)
     <sup>
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
     </sup> and its safety in humans has been evaluated in a number of clinical trials
     <sup>
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,
      <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>,
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
     </sup>. These data strongly suggest the clinical potential of ebselen for CoV treatment. It is also interesting that cinanserin displayed moderate inhibition against COVID-19 virus with an EC50 value of 20.61 μM from qRT-PCR analysis (
     <a id="xref-fig-8-2" class="xref-fig" href="#F8">Extended Data Fig. 4</a>), which is superior to that in the enzymatic inhibition assay, suggesting that cinanserin might have multi-drug targets in preventing viral infection. In further studies, selection and characterization of drug-resistant mutants will help clarify the mode of cinanserin’s action.
    </p>
    <div id="F12" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Images for the plaque-reduction assay (ebselen). As the concentration of the inhibitor (ebselen) increases, there is a significant reduction in the numbers of the plaques by comparison with NC (negative control) and DMSO. Results are shown as representative of four biological replicates. For image source data, see Supplementary Figure 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Images for the plaque-reduction assay (ebselen).&lt;;/span&gt;; As the concentration of the inhibitor (ebselen) increases, there is a significant reduction in the numbers of the plaques by comparison with NC (negative control) and DMSO. Results are shown as representative of four biological replicates. For image source data, see Supplementary Figure 2.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F12.medium.gif" width="440" height="58"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Extended Data Fig. 8" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F12.medium.gif" width="440" height="58"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 8" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Extended Data Fig. 8</span>
      <span class="caption-title">Images for the plaque-reduction assay (ebselen).</span>
      <p id="p-44" class="first-child">As the concentration of the inhibitor (ebselen) increases, there is a significant reduction in the numbers of the plaques by comparison with NC (negative control) and DMSO. Results are shown as representative of four biological replicates. For image source data, see Supplementary Figure 2.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Antiviral activities of the drug leads against COVID-19 virus. a, The quantification of absolute viral RNA copies (per ml) in the supernatant at 72 h post infection (p.i.) determined by qRT-PCR analysis. Data are mean [[#xB1]] s.e.m., n = 3 biological replicates. b, c, Dose-response curves for ebselen and N3 in the plaque-reduction assay, respectively; all data are shown as mean [[#xB1]] s.e.m., n = 4 biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1392771346" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Antiviral activities of the drug leads against COVID-19 virus.&lt;;/span&gt;; &lt;;strong&gt;;a,&lt;;/strong&gt;; The quantification of absolute viral RNA copies (per ml) in the supernatant at 72 h post infection (p.i.) determined by qRT-PCR analysis. Data are mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 3 biological replicates. &lt;;strong&gt;;b, c,&lt;;/strong&gt;; Dose-response curves for ebselen and N3 in the plaque-reduction assay, respectively; all data are shown as mean [[#xB1]] s.e.m., &lt;;em&gt;;n&lt;;/em&gt;; = 4 biological replicates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F4.medium.gif" width="440" height="151"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F4.medium.gif" width="440" height="151"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.02.26.964882/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 4</span>
      <span class="caption-title">Antiviral activities of the drug leads against COVID-19 virus.</span>
      <p id="p-18" class="first-child">
       <strong>a,</strong> The quantification of absolute viral RNA copies (per ml) in the supernatant at 72 h post infection (p.i.) determined by qRT-PCR analysis. Data are mean ± s.e.m., 
       <em>n</em> = 3 biological replicates. 
       <strong>b, c,</strong> Dose-response curves for ebselen and N3 in the plaque-reduction assay, respectively; all data are shown as mean ± s.e.m., 
       <em>n</em> = 4 biological replicates.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Discussion</h2>
    <p id="p-19">Our crystal structural and docking data have shown that the drug leads identified can bind to the substrate-binding pocket of COVID-19 virus M
     <sup>pro</sup>, which is highly conserved among all CoV M
     <sup>pro</sup>s. This strongly supports our hypothesis that development of a single antiviral agent targeting M
     <sup>pro</sup> or in combination with other potential therapies could provide an effective first line of defense against all CoV-associated diseases.
    </p>
    <p id="p-20">In the last twenty years, new infectious agents have emerged to cause epidemics, such as SARS and MERS
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
     </sup>. The timely development of effective antiviral agents for clinical use is extremely challenging because conventional drug development approaches normally take years of investigations and cost billions of dollars. The repurposing of approved pharmaceutical drugs and drug candidates provides an alternative approach to rapidly identify potential drug leads to manage rapidly emerging viral infections. Cell-based phenotypic screening has proven to be valuable
     <sup>
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
     </sup>, but the complexity of this approach is not readily compatible with high-throughput pipelines, and it cannot identify the molecular target or mechanism of action
     <sup>
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>
     </sup>. In this study, the convergence of structure-based 
     <em>ab initio</em> drug design, virtual screening and high-throughput screening proved to be an efficient strategy to find antiviral leads against COVID-19 virus. The methods presented here can greatly assist in the rapid discovery of drug leads with clinical potential in response to new emerging infectious diseases that currently lack specific drugs and vaccines.
    </p>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Methods</h2>
    <div class="section" id="sec-9">
     <h3>Cloning, protein expression and purification of COVID-19 virus M
      <sup>pro</sup>
     </h3>
     <p id="p-21">The full-length gene encoding COVID-19 virus M
      <sup>pro</sup> (NC_045512) was optimized and synthesized for 
      <em>Escherichia coli</em> expression (Genewiz, USA). Cloning strategy for producing authentic viral M
      <sup>pro</sup> has been reported previously
      <sup>
       <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>
      </sup>. The expression plasmid was transformed into 
      <em>Escherichia coli</em> BL21 (DE3) cells and then cultured in Luria Broth medium containing 100 μg/ml ampicillin at 37 °C. When the cells were grown to OD
      <sub>600</sub> of 0.6-0.8, 0.5 mM IPTG was added to the cell culture to induce the expression at 16 °C. After 10 h, the cells were harvested by centrifugation at 3,000
      <em>g</em>. The cell pellets were resuspended in lysis buffer (20 mM Tris-HCl pH 8.0, 300 mM NaCl), lysed by high-pressure homogenization, and then centrifuged at 25,000
      <em>g</em> for 40 min. The supernatant was loaded onto Ni-NTA affinity column (Qiagen, Germany), and washed by the resuspension buffer containing 20 mM imidazole. The His tagged M
      <sup>pro</sup> was eluted by cleavage buffer (50 mM Tris-HCl pH 7.0, 150 mM NaCl) including 300 mM imidazole. Human rhinovirus 3C protease was added to remove the C-terminal His tag. The M
      <sup>pro</sup> was further purified by ion exchange chromatography and size exclusion chromatography. CoV M
      <sup>pro</sup>s exist as a mixture of monomers and dimers in solutions
      <sup>
       <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
      </sup>. The purified M
      <sup>pro</sup> was stored in 50 mM Tris-HCl pH 7.3, 1 mM EDTA.
     </p>
    </div>
    <div class="section" id="sec-10">
     <h3>Crystallization, data collection and structure determination</h3>
     <p id="p-22">COVID-19 virus M
      <sup>pro</sup> was incubated with 10 mM N3 for 30 min and the complex (5 mg/ml) was crystallized by hanging drop vapor diffusion method at 20 °C. The best crystals were grown with well buffer containing 0.1 M MES pH 6.0, 2% polyethylene glycol (PEG) 6000, 3% DMSO, 1 mM DTT. The cryo-protectant solution contained 0.1 M MES pH 6.0, 30% PEG 400.
     </p>
     <p id="p-23">X-ray data were collected on beamline BL17U1 at Shanghai Synchrotron Radiation Facility (SSRF) at 100 K and at a wavelength of 1.07180 Å using an Eiger X 16M image plate detector. Data integration and scaling were performed using the program Xia2
      <sup>
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
      </sup>. The structure was determined by molecular replacement (MR) with the Phaser module
      <sup>
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
      </sup> in CCP4
      <sup>
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>
      </sup> using the SARS-CoV M
      <sup>pro</sup> (PDB ID: 2H2Z) as a search template. The output model from MR was subsequently subjected to iterative cycles of manual model adjustment with Coot
      <sup>
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </sup> and refinement was finished with Phenix
      <sup>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </sup>. The inhibitor N3 was built according to the omit map. The phasing and refinement statistics are summarized in 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Extended Data Table 1</a>. The 
      <em>R
       <sub>work</sub>/R
       <sub>free</sub>
      </em> values are 0.202/0.235, respectively. 97.3% residues are in most favored regions of the Ramachandran plot, and no residues are found in disallowed regions. Coordinates and structure factors for COVID-19 virus M
      <sup>pro</sup> in complex with the inhibitor N3 have been deposited in Protein Data Bank (PDB) with accession number 6LU7. While this work was under review, we solved the complex structure at a higher resolution (1.7 Å). The relevant coordinates and structure factors have been deposited in PDB with accession number 7BQY.
     </p>
    </div>
    <div class="section" id="sec-11">
     <h3>Enzymatic activity and inhibition assays</h3>
     <p id="p-24">The enzyme activity assays have been described previously
      <sup>
       <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>
      </sup>. Briefly, the activity of COVID-19 virus M
      <sup>pro</sup> was measured by a continuous kinetic assay, with the substrate MCA-AVLQSGFR-Lys(Dnp)-Lys-NH2 (GL Biochem, Shanghai), using wavelengths of 320 nm and 405 nm for excitation and emission, respectively. The assay started by immediately mixing 0.2 μM COVID-19 virus M
      <sup>pro</sup> with different concentrations of substrate (2.5-100 μM). Fluorescence intensity was monitored with an EnVision multimode plate reader (Perkin Elmer, USA). Initial rates were obtained by fitting the linear portion of the curves to a straight line. The kinetic parameters 
      <em>K
       <sub>m</sub>
      </em> and 
      <em>k
       <sub>cat</sub>
      </em> were calculated from a double-reciprocal plot. As N3 is a mechanism-based irreversible inhibitor for COVID-19 virus M
      <sup>pro</sup>, 
      <em>k
       <sub>obs</sub>
      </em>/[I] was used as an approximation of the pseudo second-order rate constant to evaluate the inhibition effect of the inhibitor N3
      <sup>
       <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>
      </sup>. In this case, the measurement was carried out with 0.2 μM of enzyme, 20 μM of substrate and inhibitor at 6 different concentrations (0-1 μM).
     </p>
    </div>
    <div class="section" id="sec-12">
     <h3>Virtual screening</h3>
     <p id="p-25">The virtual screening was performed using our in-house database via a workflow application of Glide (v8.2), Maestro (Schrödinger 2019-1a)
      <sup>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
      </sup>. All compounds in the database were considered to be at pH 7.4 ± 0.2 to estimate their protonation state using the program EpiK
      <sup>
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
      </sup>. Their three dimensional conformations were generated by the ligPrep module of Maestro. The structure of COVID-19 virus M
      <sup>pro</sup> (PDB ID: 6LU7) was used to generate receptor grid for docking simulations. The center of active site of the grid was determined according to the position of N3 in the structure. The flexibility of the receptor hydroxyl and thiol groups in side chains of C145, S46 and Y54 were considered. At the very beginning, a relatively fast but raw screening was performed by using the glide standard precision model, and the top 20% of compounds were kept. Finally, the candidate molecules were picked by analysing the predicted binding modes and their scores.
     </p>
    </div>
    <div class="section" id="sec-13">
     <h3>High-throughput drug screen and IC
      <sub>50</sub> measurement
     </h3>
     <p id="p-26">Potential inhibitors against COVID-19 virus M
      <sup>pro</sup> were screened by an enzymatic inhibition assay. When the different compounds were added into the enzymatic reaction mixture, the change of initial rates was calculated to evaluate their inhibitory effect. Five drug libraries, Approved Drug Library (Target Mol, USA), Clinic Compound Library (Target Mol, USA), FDA-approved Drug Library (Selleck, USA), Natural Product Library (Selleck, USA), and Anti-virus Drug Library (Shanghai Institute for Advanced Immunochemical Studies, SIAIS), which includes ∼10,000 compounds, were used. The preliminary screening reaction mixture included 0.2 μM protein, 20 μM substrate and 50 μM compounds. The compounds of interest were defined as those with a percentage of inhibition over 60% compared with the reaction in the absence of inhibitor. IC
      <sub>50</sub> values of seven drug leads were measured using 0.2 μM protein, 20 μM substrate and 11 different inhibitor concentrations. In order to exclude inhibitors possibly acting as aggregators, detergent-based control was performed by adding 0.001% or 0.01% freshly made up Triton X-100 to the reaction at the same time
      <sup>
       <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>
      </sup>. All experimental data was analyzed using GraphPad Prism. All experiments were performed in triplicate.
     </p>
    </div>
    <div class="section" id="sec-14">
     <h3>Molecular docking</h3>
     <p id="p-27">To understand the binding interaction of these molecules with COVID-19 virus M
      <sup>pro</sup>, two different molecular docking methods, i.e., Glide (v8.2)
      <sup>
       <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>
      </sup> and iFitDock
      <sup>
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>
      </sup> were used to predict their binding poses. Then a 3D molecular similarity calculation method, SHAFTS
      <sup>
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>
      </sup>, was used for molecular alignment poses enumeration by matching the critical pharmacophore and volumetric overlay between the N3 molecule within the M
      <sup>pro</sup> structure and the six drug candidates. However, the selenium atom of ebselen could not be treated by any of these above methods, so sulfur was used to replace it in the calculations. Then the obtained optimal superposition of these molecules was used to assess the reasonability of the predicted binding poses from the two docking methods, and only the binding orientations which were consistent among different methods were kept for constructing the initial complexes. Finally, these complexes were further optimized and re-scored by using MM-GBSA module
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>
      </sup> of Schrödinger, and the residues within 5 Å around the ligand were refined.
     </p>
    </div>
    <div class="section" id="sec-15">
     <h3>Antiviral and cytotoxicity assays for compounds from high-throughput screening</h3>
     <p id="p-28">The 
      <em>in vitro</em> antiviral efficacy of the drug candidates on Vero cells were determined by qRT-PCR. About 1×10
      <sup>4</sup> Vero cells were seeded into a 96-well plate and incubated for 20-24 h at 37 °C. All the infection experiments were performed at biosafety level-3 (BSL-3). Cells were pre-treated with the drug candidates (10 μM) for 1 h; the COVID-19 virus (MOI of 0.01) was subsequently added to allow infection for 2 h. Then, the virus-drug mixture was removed and cells were further cultured with fresh drug-containing medium. At 72 h post infection (p.i.), viral RNA (vRNA) was extracted from the culture supernatant using QIAamp viral RNA mini kit (Qiagen, Germany) according to the manufacturer’s recommendation and detected by qRT-PCR assay using the COVID-19 virus-specific primers. Because shikonin showed cellular toxicity at the test concentration, its antiviral activity assay did not further proceed. Viral RNA copies per milliliter were determined using a synthetic RNA fragment to amplify the target region. The linearized plasmid containing S gene of COVID-19 virus was subjected to 
      <em>in vitro</em> transcription. The resulting RNA transcripts were purified and then quantified using spectrophotometry on Nanodrop 2000 (Thermo Fisher Scientific, USA). The purified RNA was diluted 10-fold serially using RNase-free water and was detected using qRT-PCR. Threshold cycle (Ct) values for the known concentrations of the RNA were plotted against the log of the number of genome equivalent copies. The resultant standard curve was used to determine the number of genome equivalents of vRNA in the samples. The determination of the detection limit was based on the lowest level at which vRNA was detected and remained within the range of linearity of a standard curve (Ct value of 38). TaqMan primers for COVID-19 virus are 5’-TCCTGGTGATTCTTCTTCAGG-3’ and 5’-TCTGAGAGAGGGTCAAGTGC-3’ with COVID-19 virus probe 5’-FAM-AGCTGCAGCACCAGCTGTCCA-BHQ1-3’. The cytotoxicity of the tested drugs on Vero cell were determined by MTS cell proliferation assays (Promega, USA). 1×10
      <sup>4</sup> cells were seeded into a 96-well plate and incubated for 20-24 h at 37 °C. After that, the medium was removed, and 100 μl of medium containing decreasing concentrations of antiviral compounds were added to the wells. After 4 days incubation at 37 °C, MTS assays were performed according to manufacturer’s protocols. All experiments were performed in triplicate. Vero cells were obtained from ATCC (American Type Culture Collection) with authentication service. All cell lines were tested negative for mycoplasma contamination. No commonly misidentified cell lines were used.
     </p>
    </div>
    <div class="section" id="sec-16">
     <h3>Antiviral and cytotoxicity assays for cinanserin</h3>
     <p id="p-29">For the antiviral assay, a clinical isolate COVID-19 virus
      <sup>
       <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
      </sup> was propagated in Vero E6 cells, and viral titer was determined as described previously
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>
      </sup>. All the infection experiments were performed at BSL-3. Pre-seeded Vero E6 cells (5×10
      <sup>4</sup> cells/well) were pre-treated with the different concentrations of cinanserin for 1 h and the virus was subsequently added (MOI of 0.05) to allow infection for 2 h. Then, the virus-drug mixture was removed and cells were further cultured with fresh drug containing medium. At 24 h p.i., the cell supernatant was collected and vRNA in supernatant was subjected to qRT-PCR analysis. For cytotoxicity assays, Vero E6 cells were suspended in growth medium in 96-well plates. The next day, appropriate concentrations of cinanserin were added to the medium. After 24 h, the relative numbers of surviving cells were measured by CCK8 (Beyotime, China) assay in accordance with the manufacturer’s instructions. All experiments were performed in triplicate. Vero E6 cells were obtained from ATCC with authentication service. All cell lines were tested negative for mycoplasma contamination. No commonly misidentified cell lines were used.
     </p>
    </div>
    <div class="section" id="sec-17">
     <h3>Plaque-reduction assays</h3>
     <p id="p-30">1×10
      <sup>5</sup> Vero E6 cells were seeded in a 24-well plate and treated with different doses of the inhibitors. All the infection experiments were performed at BSL-3. Inhibitors with different dilution concentrations were mixed with COVID-19 virus (100 PFU), 200 μl mixtures were inoculated onto monolayer Vero E6 cells for 1 h. After removing the supernatant, the plate was washed twice with DMEM medium, cells were incubated with 0.9% agarose containing appropriate concentrations of inhibitors. The overlay was discarded at 4 days p.i. and cells were fixed for 30 min in 4% polyoxymethylene and stained with crystal violet working solution. The plaque forming units were determined. All experiments were performed in four biological replicates.
     </p>
    </div>
    <div class="section" id="sec-18">
     <h3>Intact protein analysis</h3>
     <p id="p-31">2.5 μl of compounds (10 mM in DMSO) were added into 50 μl of COVID-19 virus M
      <sup>pro</sup> (10 mg/ml). The mixtures were kept in room temperature for 30 min. Liquid chromatography-mass spectrometry (LC-MS) analyses were performed in positive-ion mode with a quadrupole-time-of-flight (QTOF) mass spectrometer (Agilent 6550, USA) coupled with a high-performance liquid chromatograph (HPLC, Agilent 1260, USA) for detecting the molecular weight of intact proteins. The samples were eluted from a Phenomenex Jupiter C4 300Å LC Column (2×150 mm, 5 μm) over a 15 min gradient from 5% to 100% acetonitrile containing 0.1% formic acid at a flow rate of 0.5 ml/min. The acquisition method in positive-ion mode with Dual Agilent Jet Stream electrospray voltage used a capillary temperature of 250 °C, a fragmentor of 175 V, a capillary voltage of 3000 V. Mass deconvolution was performed using Agilent MassHunter Qualitative Analysis B.06.00 software with BioConfirm Workflow.
     </p>
    </div>
    <div class="section" id="sec-19">
     <h3>Tandem MS/MS analysis</h3>
     <p id="p-32">The samples were precipitated and redissolved by 8 M urea, and then digested for 16 h at 25 °C by chymotrypsin at an enzyme-to-substrate ratio of 1:50 (w/w). The digested peptides were desalted and loaded onto a homemade 30 cm-long pulled-tip analytical column (ReproSil-Pur C18 AQ 1.9 μm particle size, Dr. Maisch GmbH, 75 μm ID× 360 μm OD) connected to an Easy-nLC1200 UHPLC (Thermo Fisher Scientific, USA) for mass spectrometry analysis. The elution gradient and mobile phase constitution used for peptide separation were as follows: 0-1 min, 4%-8% B; 1-96 min, 8-35% B; 96-104 min, 35-60% B; 105-120min, 60-100% B (mobile phase A: 0.1% formic acid in water; mobile phase B: 0.1% formic acid in 80% acetonitrile) at a flow rate of 300 nl /min. Peptides eluted from the LC column were directly electro-sprayed into the mass spectrometer with the application of a distal 1.8-kV spray voltage. Survey full-scan MS spectra (from m/z 300–1800) were acquired in the Orbitrap analyzer (Q Exactive, Thermo Fisher Scientific, USA) with resolution r = 70,000 at m/z 400. The top 20 MS/MS events were sequentially generated and selected from the full MS spectrum at a 30% normalized collision energy. The dynamic exclusion time was set at 10 seconds. One acquisition cycle includes one full-scan MS spectrum followed by top 20 MS/MS events, sequentially generated on the first to the twentieth most intense ions selected from the full MS spectrum at a 28% normalized collision energy. The acquired MS/MS data were analyzed UniProtKB 
      <em>E.coli</em> database (database released on Nov. 11, 2016) containing nsp5 using Protein Discoverer 2.1. In order to accurately estimate peptide probabilities and false discovery rates (FDR), we used a decoy database containing the reversed sequences of all the proteins appended to the target database. FDR was set at 0.01. Mass tolerance for precursor ions was set at 20 ppm. Chymotrypsin was defined as cleavage enzyme and the maximal number of missed cleavage sites was set at 4. Protein N-terminus acetylation, methionine oxidation and compounds covalent bindings were set as variable modifications. The modified peptides were manually checked and labeled.
     </p>
    </div>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">Author contributions</h2>
    <p id="p-34">Z.R. and H.Y. conceived the project; Z.J., H.J., Z.R., and H.Y. designed the experiments; Z.J., X.D., Y.Duan., J.Y., T.Y., Xiaoce Liu and Xiuna Yang cloned, expressed, purified and crystallized proteins; Z.J., Y.Z., B.Z. and F.L. collected the diffraction data; B.Z. and Xiang Liu solved the crystal structure; Z.J., X.D., Y.Duan. and J.Y. performed enzymatic activity and inhibition assay, high-throughput drug screen and IC
     <sub>50</sub> measurement; L.W. and F.B. performed virtual screening and molecular docking; Y.X., L.Z. and H.L. performed enzymatic inhibition, cell-based antiviral and cytotoxicity assay for cinanserin; Y.Deng. and X. Li performed qRT-PCR analysis and cytotoxicity assay of N3; M.L., R.J. and Xinglou Yang performed plaque-reduction assay; C.P. performed intact protein and tandem MS/MS analyses; Z.J., X.D., Y.X., Y.Deng., C.P., F.B., H.L., Xiang Liu, K.Y., L.G., W.X., G.X., C.Q., Z.S., H.J., Z.R. and H.Y. analyzed and discussed the data; Z.J., X.D., F.B., Xiang Liu, L.G., G.X., C.Q., Z.S., H.J., Z.R. and H.Y wrote the manuscript.
    </p>
   </div>
   <div class="section COI-statement" id="sec-21">
    <h2 class="">Competing interests</h2>
    <p id="p-35">The authors declare no competing interests.</p>
   </div>
   <div class="section" id="sec-22">
    <h2 class="">Data and materials availability</h2>
    <p id="p-36">The PDB accession No. for the coordinates of COVID-19 virus M
     <sup>pro</sup> in complex with N3 is 6LU7 (Deposited: 2020-01-26; Released: 2020-02-05).
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-33">We would like to thank Ying Lei and Juan Kong from High Throughput Platform, staff from Analytical Chemistry Platform at Shanghai Institute for Advanced Immunochemical Studies, for their technical support. We are grateful to National Centre for Protein Science Shanghai and The Molecular and Cell Biology Core Facility of the School of Life Science and Technology, ShanghaiTech University for use of their instrumentation and technical assistance. We thank Prof. Zhijie Liu, and Haixia Su for discussion. We also thank the staff from beamlines BL17U1, BL18U1 and BL19U1 at Shanghai Synchrotron Radiation Facility (China). This work was supported by grants from National Key R&amp;;D Program of China (grant No. 2017YFC0840300 to Z.R.), Project of International Cooperation and Exchanges NSFC (grant No. 81520108019 to Z.R.), Science and Technology Commission of Shanghai Municipality (grant No. 20431900200) and Department of Science and Technology of Guangxi Zhuang Autonomous Region (grant No. 2020AB40007).</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-2">
      <p id="p-2">
       <a class="rev-xref" href="#xref-fn-2-1">↵</a>
       <span class="fn-label">*</span> e-mail: 
       <span class="em-link">
        <span class="em-addr">hljiang{at}simm.ac.cn</span>
       </span>; 
       <span class="em-link">
        <span class="em-addr">raozh{at}mail.tsinghua.edu.cn</span>
       </span>; 
       <span class="em-link">
        <span class="em-addr">yanght{at}shanghaitech.edu.cn</span>
       </span>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Qun</span>
          <span class="cit-name-surname">Li</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.3" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015507&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.4" data-doi="10.1038/s41586-020-2008-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015508&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.5" data-doi="10.1093/emboj/cdf327">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Anand</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain</span>. 
         <abbr class="cit-jnl-abbrev">The EMBO Journal</abbr>
         <span class="cit-vol">21</span>, 
         <span class="cit-fpage">3213</span>–
         <span class="cit-lpage">3224</span> (
         <span class="cit-pub-date">2002</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DAnand%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D3213%26rft.epage%253D3224%26rft.atitle%253DStructure%2Bof%2Bcoronavirus%2Bmain%2Bproteinase%2Breveals%2Bcombination%2Bof%2Ba%2Bchymotrypsin%2Bfold%2Bwith%2Ban%2Bextra%2B%257Balpha%257D-helical%2Bdomain%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf327%26rft_id%253Dinfo%253Apmid%252F12093723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTMvMzIxMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDIuMjYuOTY0ODgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.6" data-doi="10.1073/pnas.1835675100">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">H. T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-vol">100</span>, 
         <span class="cit-fpage">13190</span>–
         <span class="cit-lpage">13195</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1835675100</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1835675100%26rft_id%253Dinfo%253Apmid%252F14585926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIzLzEzMTkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMi4yNi45NjQ4ODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.7" data-doi="10.1038/nrmicro.2016.81">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">de Wit</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Doremalen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Falzarano</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Munster</span>, 
          <span class="cit-name-given-names">V. J.</span>
         </span>
         <span class="cit-article-title">SARS and MERS: recent insights into emerging coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">523</span>–
         <span class="cit-lpage">534</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro.2016.81</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.8" data-doi="10.1099/0022-1317-83-3-595">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hegyi</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ziebuhr</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Conservation of substrate specificities among coronavirus main proteases</span>. 
         <abbr class="cit-jnl-abbrev">Journal of general virology</abbr>
         <span class="cit-vol">83</span>, 
         <span class="cit-fpage">595</span>–
         <span class="cit-lpage">599</span> (
         <span class="cit-pub-date">2002</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DHegyi%26rft.auinit1%253DA.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D595%26rft.epage%253D599%26rft.atitle%253DConservation%2Bof%2Bsubstrate%2Bspecificities%2Bamong%2Bcoronavirus%2Bmain%2Bproteases%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-83-3-595%26rft_id%253Dinfo%253Apmid%252F11842254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/0022-1317-83-3-595&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11842254&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000173983100012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.9" data-doi="10.1021/acs.jmedchem.5b01461">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pillaiyar</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manickam</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Namasivayam</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashi</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>, 
          <span class="cit-name-given-names">S. H.</span>
         </span>
         <span class="cit-article-title">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>
         <span class="cit-vol">59</span>, 
         <span class="cit-fpage">6595</span>–
         <span class="cit-lpage">6628</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1021/acs.jmedchem.5b01461</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedicinal%2BChemistry%26rft.volume%253D59%26rft.spage%253D6595%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jmedchem.5b01461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/acs.jmedchem.5b01461&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.10" data-doi="10.1016/j.jmb.2006.11.073">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xue</span>, 
          <span class="cit-name-given-names">X. Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Production of authentic SARS-CoV M-pro with enhanced activity: Application as a novel tag-cleavage endopeptidase for protein overproduction</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>
         <span class="cit-vol">366</span>, 
         <span class="cit-fpage">965</span>–
         <span class="cit-lpage">975</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2006.11.073</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DXue%26rft.auinit1%253DX.%26rft.volume%253D366%26rft.issue%253D3%26rft.spage%253D965%26rft.epage%253D975%26rft.atitle%253DProduction%2Bof%2Bauthentic%2BSARS-CoV%2BM%2528pro%2529%2Bwith%2Benhanced%2Bactivity%253A%2Bapplication%2Bas%2Ba%2Bnovel%2Btag-cleavage%2Bendopeptidase%2Bfor%2Bprotein%2Boverproduction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.11.073%26rft_id%253Dinfo%253Apmid%252F17189639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.11.073&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17189639&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.11" data-doi="10.1006/abio.1997.2315">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Q. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Johnson</span>, 
          <span class="cit-name-given-names">R. B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cox</span>, 
          <span class="cit-name-given-names">G. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Villarreal</span>, 
          <span class="cit-name-given-names">E. C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Loncharich</span>, 
          <span class="cit-name-given-names">R. J.</span>
         </span>
         <span class="cit-article-title">A Continuous Colorimetric Assay for Rhinovirus-14 3C Protease Using Peptidep-Nitroanilides as Substrates</span>. 
         <abbr class="cit-jnl-abbrev">Analytical biochemistry</abbr>
         <span class="cit-vol">252</span>, 
         <span class="cit-fpage">238</span>–
         <span class="cit-lpage">245</span> (
         <span class="cit-pub-date">1997</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DWang%26rft.auinit1%253DQ.%2BM.%26rft.volume%253D252%26rft.issue%253D2%26rft.spage%253D238%26rft.epage%253D245%26rft.atitle%253DA%2Bcontinuous%2Bcolorimetric%2Bassay%2Bfor%2Brhinovirus-14%2B3C%2Bprotease%2Busing%2Bpeptide%2Bp-nitroanilides%2Bas%2Bsubstrates.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fabio.1997.2315%26rft_id%253Dinfo%253Apmid%252F9344409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/abio.1997.2315&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9344409&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.12" data-doi="10.1371/journal.pbio.0030428">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">H. T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design of wide-spectrum inhibitors targeting coronavirus main proteases</span>. 
         <abbr class="cit-jnl-abbrev">Plos Biology</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">2044</span>–
         <span class="cit-lpage">2044</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pbio.0030428</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlos%2BBiology%26rft.volume%253D3%26rft.spage%253D2044%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030428&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.13" data-doi="10.1128/JVI.02114-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xue</span>, 
          <span class="cit-name-given-names">X. Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structures of two coronavirus main proteases: Implications for substrate binding and antiviral drug design</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">2515</span>–
         <span class="cit-lpage">2527</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02114-07</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02114-07%26rft_id%253Dinfo%253Apmid%252F18094151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzUvMjUxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDIuMjYuOTY0ODgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.14" data-doi="10.1007/s13238-013-2841-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>, 
          <span class="cit-name-given-names">Z. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The newly emerged SARS-Like coronavirus HCoV-EMC also has an “Achilles’ heel”: current effective inhibitor targeting a 3C-like protease</span>. 
         <abbr class="cit-jnl-abbrev">Protein &amp;; Cell</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">248</span>–
         <span class="cit-lpage">250</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/s13238-013-2841-3</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2B%2526%2BCell%26rft.volume%253D4%26rft.spage%253D248%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-013-2841-3%26rft_id%253Dinfo%253Apmid%252F23549610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s13238-013-2841-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23549610&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.15" data-doi="10.1038/srep22677">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">Structure of Main Protease from Human Coronavirus NL63: Insights for Wide Spectrum Anti-Coronavirus Drug Design</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">22677</span>–
         <span class="cit-lpage">22677</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/srep22677</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D22677%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep22677%26rft_id%253Dinfo%253Apmid%252F26948040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/srep22677&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26948040&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.16" data-doi="10.1128/JVI.00298-08">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure of the main protease from a global infectious human coronavirus, HCoV-HKU1</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">8647</span>–
         <span class="cit-lpage">8655</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00298-08</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00298-08%26rft_id%253Dinfo%253Apmid%252F18562531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNy84NjQ3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMi4yNi45NjQ4ODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.17" data-doi="10.1074/jbc.M807947200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">C.-C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis of inhibition specificities of 3C and 3C-like proteases by zinc-coordinating and peptidomimetic compounds</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>
         <span class="cit-vol">284</span>, 
         <span class="cit-fpage">7646</span>–
         <span class="cit-lpage">7655</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M807947200</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M807947200%26rft_id%253Dinfo%253Apmid%252F19144641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODQvMTIvNzY0NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDIuMjYuOTY0ODgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <abbr class="cit-jnl-abbrev">Inhibitor for the Main Protease of Coronavirus Hku4</abbr> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.19" data-doi="10.1128/JVI.02685-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">F. H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Crystal Structure of Feline Infectious Peritonitis Virus Main Protease in Complex with Synergetic Dual Inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">90</span>, 
         <span class="cit-fpage">1910</span>–
         <span class="cit-lpage">1917</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02685-15</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02685-15%26rft_id%253Dinfo%253Apmid%252F26656689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzQvMTkxMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDIuMjYuOTY0ODgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.20" data-doi="10.1021/acs.jmedchem.7b00103">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">F. H.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Michael Acceptor-Based Peptidomimetic Inhibitor of Main Protease from Porcine Epidemic Diarrhea Virus</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>
         <span class="cit-vol">60</span>, 
         <span class="cit-fpage">3212</span>–
         <span class="cit-lpage">3216</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1021/acs.jmedchem.7b00103</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedicinal%2BChemistry%26rft.volume%253D60%26rft.spage%253D3212%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jmedchem.7b00103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/acs.jmedchem.7b00103&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.21" data-doi="10.1016/j.bbrc.2019.02.105">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cui</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The crystal structure of main protease from mouse hepatitis virus A59 in complex with an inhibitor</span>. 
         <abbr class="cit-jnl-abbrev">Biochemical and Biophysical Research Communications</abbr>
         <span class="cit-vol">511</span>, 
         <span class="cit-fpage">794</span>–
         <span class="cit-lpage">799</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbrc.2019.02.105</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2BBiophysical%2BResearch%2BCommunications%26rft.volume%253D511%26rft.spage%253D794%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2019.02.105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2019.02.105&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.22" data-doi="10.3181/00379727-119-30204">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rubin</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Waugh</span>, 
          <span class="cit-name-given-names">M. H.</span>
         </span>
         <span class="cit-article-title">Antiphlogistic effects of antiserotonin (SQ 10,643) and aminopyrine in rats versus endotoxin and other agents</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the Society for Experimental Biology and Medicine</abbr>
         <span class="cit-vol">119</span>, 
         <span class="cit-fpage">438</span>–
         <span class="cit-lpage">443</span> (
         <span class="cit-pub-date">1965</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DRubin%26rft.auinit1%253DB.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D438%26rft.epage%253D443%26rft.atitle%253DAntiphlogistic%2BEffects%2Bof%2BAntiserotonin%2B%2528SQ%2B10%252C643%2529%2Band%2BAminopyrine%2Bin%2BRats%2BVersus%2BEndotoxin%2Band%2BOther%2BAgents%26rft_id%253Dinfo%253Adoi%252F10.3181%252F00379727-119-30204%26rft_id%253Dinfo%253Apmid%252F14328912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3181/00379727-119-30204&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14328912&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.26.964882v3.23" data-doi="10.1128/JVI.79.11.7095-7103.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cinanserin is an inhibitor of the 3C-like proteinase of severe acute respiratory syndrome coronavirus and strongly reduces virus replication in vitro</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">79</span>, 
         <span class="cit-fpage">7095</span>–
         <span class="cit-lpage">7103</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DChen%26rft.auinit1%253DL.%26rft.volume%253D79%26rft.issue%253D11%26rft.spage%253D7095%26rft.epage%253D7103%26rft.atitle%253DCinanserin%2BIs%2Ban%2BInhibitor%2Bof%2Bthe%2B3C-Like%2BProteinase%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2Band%2BStrongly%2BReduces%2BVirus%2BReplication%2BIn%2BVitro%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.11.7095-7103.2005%26rft_id%253Dinfo%253Apmid%252F15890949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xMS83MDk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMi4yNi45NjQ4ODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>, 
          <span class="cit-name-given-names">B. Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shoichet</span>, 
          <span class="cit-name-given-names">B. K.</span>
         </span>
         <span class="cit-article-title">A detergent-based assay for the detection of promiscuous inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Nature protocols</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">550</span>–
         <span class="cit-lpage">553</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D1%26rft.spage%253D550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.25" data-doi="10.1038/ncomms2380">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Singh</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A safe lithium mimetic for bipolar disorder</span>. 
         <abbr class="cit-jnl-abbrev">Nature communications</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">7</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms2380&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lynch</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kil</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Development of ebselen, a glutathione peroxidase mimic, for the prevention and treatment of noise-induced hearing loss</span>. 
         <abbr class="cit-jnl-abbrev">Seminars in Hearing</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">047</span>–
         <span class="cit-lpage">055</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2BHearing%26rft.volume%253D30%26rft.spage%253D047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kil</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Safety and efficacy of ebselen for the prevention of noise-induced hearing loss: a randomised, double-blind, placebo-controlled, phase 2 trial</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr>
         <span class="cit-vol">390</span>, 
         <span class="cit-fpage">969</span>–
         <span class="cit-lpage">979</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D390%26rft.spage%253D969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Renson</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Etschenberg</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Winkelmann</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">2-Phenyl-1, 2-benzisoselenazol-3 (2H)-one containing pharmaceutical preparations and process for the treatment of rheumatic diseases</span>. (
         <span class="cit-pub-date">1982</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Masaki</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Effects of the potential lithium-mimetic, ebselen, on impulsivity and emotional processing</span>. 
         <abbr class="cit-jnl-abbrev">Psychopharmacology</abbr>
         <span class="cit-vol">233</span>, 
         <span class="cit-fpage">2655</span>–
         <span class="cit-lpage">2661</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPsychopharmacology%26rft.volume%253D233%26rft.spage%253D2655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.30" data-doi="10.1038/nm.4184">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Identification of small-molecule inhibitors of Zika virus infection and induced neural cell death via a drug repurposing screen</span>. 
         <abbr class="cit-jnl-abbrev">Nature Medicine</abbr>
         <span class="cit-vol">22</span>, 
         <span class="cit-fpage">1101</span>–
         <span class="cit-lpage">1107</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nm.4184</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMedicine%26rft.volume%253D22%26rft.spage%253D1101%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.4184%26rft_id%253Dinfo%253Apmid%252F27571349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nm.4184&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27571349&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aulner</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Danckaert</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ihm</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shum</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shorte</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>
         <span class="cit-article-title">Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases</span>. 
         <abbr class="cit-jnl-abbrev">Trends in parasitology</abbr> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.32" data-doi="10.1126/science.1085658">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Anand</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ziebuhr</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wadhwani</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mesters</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Coronavirus main proteinase (3CL(pro)) structure: Basis for design of anti-SARS drugs</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">300</span>, 
         <span class="cit-fpage">1763</span>–
         <span class="cit-lpage">1767</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.1085658</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085658%26rft_id%253Dinfo%253Apmid%252F12746549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMi4yNi45NjQ4ODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.33" data-doi="10.1107/S0021889809045701">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Winter</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">xia2: an expert system for macromolecular crystallography data reduction</span>. 
         <abbr class="cit-jnl-abbrev">Journal of applied crystallography</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">186</span>–
         <span class="cit-lpage">190</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bcrystallography%26rft.volume%253D43%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0021889809045701%26rft_id%253Dinfo%253Apmid%252F23793152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0021889809045701&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23793152&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273764900028&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.34" data-doi="10.1107/S0021889807021206">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McCoy</span>, 
          <span class="cit-name-given-names">A. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Phaser crystallographic software</span>. 
         <abbr class="cit-jnl-abbrev">Journal of applied crystallography</abbr>
         <span class="cit-vol">40</span>, 
         <span class="cit-fpage">658</span>–
         <span class="cit-lpage">674</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BCrystallography%26rft.stitle%253DJournal%2Bof%2BApplied%2BCrystallography%26rft.aulast%253DMcCoy%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D40%26rft.issue%253DPt%2B4%26rft.spage%253D658%26rft.epage%253D674%26rft.atitle%253DPhaser%2Bcrystallographic%2Bsoftware.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0021889807021206%26rft_id%253Dinfo%253Apmid%252F19461840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0021889807021206&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19461840&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000248077500003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.35" data-doi="10.1107/s2059798317016035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Potterton</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">CCP4i2: the new graphical user interface to the CCP4 program suite</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallographica Section D-Structural Biology</abbr>
         <span class="cit-vol">74</span>, 
         <span class="cit-fpage">68</span>–
         <span class="cit-lpage">84</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1107/s2059798317016035</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%2BSection%2BD-Structural%2BBiology%26rft.volume%253D74%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1107%252Fs2059798317016035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/s2059798317016035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.36" data-doi="10.1107/S0907444910007493">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Emsley</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lohkamp</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scott</span>, 
          <span class="cit-name-given-names">W. G.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowtan</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-article-title">Features and development of Coot</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallographica Section D: Biological Crystallography</abbr>
         <span class="cit-vol">66</span>, 
         <span class="cit-fpage">486</span>–
         <span class="cit-lpage">501</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DEmsley%26rft.auinit1%253DP.%26rft.volume%253D66%26rft.issue%253DPt%2B4%26rft.spage%253D486%26rft.epage%253D501%26rft.atitle%253DFeatures%2Band%2Bdevelopment%2Bof%2BCoot.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444910007493%26rft_id%253Dinfo%253Apmid%252F20383002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0907444910007493&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20383002&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000275941300018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.37" data-doi="10.1107/s0907444912001308">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Afonine</span>, 
          <span class="cit-name-given-names">P. V.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Towards automated crystallographic structure refinement with phenix.refine</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallographica Section D-Structural Biology</abbr>
         <span class="cit-vol">68</span>, 
         <span class="cit-fpage">352</span>–
         <span class="cit-lpage">367</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1107/s0907444912001308</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%2BSection%2BD-Structural%2BBiology%26rft.volume%253D68%26rft.spage%253D352%26rft_id%253Dinfo%253Adoi%252F10.1107%252Fs0907444912001308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/s0907444912001308&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.38" data-doi="10.1021/jm0306430">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Glide: A new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>
         <span class="cit-vol">47</span>, 
         <span class="cit-fpage">1739</span>–
         <span class="cit-lpage">1749</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1021/jm0306430</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DFriesner%26rft.auinit1%253DR.%2BA.%26rft.volume%253D47%26rft.issue%253D7%26rft.spage%253D1739%26rft.epage%253D1749%26rft.atitle%253DGlide%253A%2Ba%2Bnew%2Bapproach%2Bfor%2Brapid%252C%2Baccurate%2Bdocking%2Band%2Bscoring.%2B1.%2BMethod%2Band%2Bassessment%2Bof%2Bdocking%2Baccuracy.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm0306430%26rft_id%253Dinfo%253Apmid%252F15027865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm0306430&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15027865&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220317600019&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.39" data-doi="10.1007/s10822-010-9349-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greenwood</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Calkins</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sullivan</span>, 
          <span class="cit-name-given-names">A. P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shelley</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span>
         <span class="cit-article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. 
         <abbr class="cit-jnl-abbrev">Journal of computer-aided molecular design</abbr>
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">591</span>–
         <span class="cit-lpage">604</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputer-aided%2Bmolecular%2Bdesign%26rft.stitle%253DJ%2BComput%2BAided%2BMol%2BDes%26rft.aulast%253DGreenwood%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D24%26rft.issue%253D6-7%26rft.spage%253D591%26rft.epage%253D604%26rft.atitle%253DTowards%2Bthe%2Bcomprehensive%252C%2Brapid%252C%2Band%2Baccurate%2Bprediction%2Bof%2Bthe%2Bfavorable%2Btautomeric%2Bstates%2Bof%2Bdrug-like%2Bmolecules%2Bin%2Baqueous%2Bsolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10822-010-9349-1%26rft_id%253Dinfo%253Apmid%252F20354892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s10822-010-9349-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20354892&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.40" data-doi="10.1073/pnas.1301814110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bai</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Free energy landscape for the binding process of Huperzine A to acetylcholinesterase</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-vol">110</span>, 
         <span class="cit-fpage">4273</span>–
         <span class="cit-lpage">4278</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1301814110</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1301814110%26rft_id%253Dinfo%253Apmid%252F23440190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzExLzQyNzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOS8yMDIwLjAyLjI2Ljk2NDg4Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.41" data-doi="10.1021/ci200060s">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">X. F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">H. L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H. L.</span>
         </span>
         <span class="cit-article-title">SHAFTS: A Hybrid Approach for 3D Molecular Similarity Calculation. 1. Method and Assessment of Virtual Screening</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Chemical Information and Modeling</abbr>
         <span class="cit-vol">51</span>, 
         <span class="cit-fpage">2372</span>–
         <span class="cit-lpage">2385</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1021/ci200060s</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DLiu%26rft.auinit1%253DX.%26rft.volume%253D51%26rft.issue%253D9%26rft.spage%253D2372%26rft.epage%253D2385%26rft.atitle%253DSHAFTS%253A%2Ba%2Bhybrid%2Bapproach%2Bfor%2B3D%2Bmolecular%2Bsimilarity%2Bcalculation.%2B1.%2BMethod%2Band%2Bassessment%2Bof%2Bvirtual%2Bscreening.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci200060s%26rft_id%253Dinfo%253Apmid%252F21819157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ci200060s&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21819157&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.42" data-doi="10.1021/ci800004w">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guimaraes</span>, 
          <span class="cit-name-given-names">C. R. W.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cardozo</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">MM-GB/SA rescoring of docking poses in structure-based lead optimization</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Chemical Information and Modeling</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">958</span>–
         <span class="cit-lpage">970</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1021/ci800004w</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DGuimaraes%26rft.auinit1%253DC.%2BR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D958%26rft.epage%253D970%26rft.atitle%253DMM-GB%252FSA%2Brescoring%2Bof%2Bdocking%2Bposes%2Bin%2Bstructure-based%2Blead%2Boptimization.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci800004w%26rft_id%253Dinfo%253Apmid%252F18422307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ci800004w&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18422307&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.26.964882v3.43" data-doi="10.1038/s41422-020-0282-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>. 
         <abbr class="cit-jnl-abbrev">Cell Research</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">269</span>–
         <span class="cit-lpage">271</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41422-020-0282-0</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41422-020-0282-0%26rft_id%253Dinfo%253Apmid%252F32020029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41422-020-0282-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32020029&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.02.26.964882.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
