<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-11.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/29/2020.03.25.006569.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.25.006569v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Codon usage and evolutionary rates of the 2019-nCoV genes" />
<meta name="DC.Identifier" content="10.1101/2020.03.25.006569" />
<meta name="DC.Date" content="2020-03-29" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.

*   MDPI
    :   Multidisciplinary Digital Publishing Institute
    DOAJ
    :   Directory of open access journals
    TLA
    :   Three letter acronym
    LD
    :   linear dichroism" />
<meta name="DC.Contributor" content="Maddalena Dilucca" />
<meta name="DC.Contributor" content="Sergio Forcelloni" />
<meta name="DC.Contributor" content="Athanasia Pavlopoulou" />
<meta name="DC.Contributor" content="Alexandros G. Georgakilas" />
<meta name="DC.Contributor" content="Andrea Giansanti" />
<meta name="article:published_time" content="2020-03-29" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Codon usage and evolutionary rates of the 2019-nCoV genes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.25.006569v2" />
<meta name="citation_id" content="2020.03.25.006569v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.25.006569" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.25.006569" />
<meta name="citation_author" content="Maddalena Dilucca" />
<meta name="citation_author_institution" content="Physics Department, Sapienza University of Rome" />
<meta name="citation_author_institution" content="Liceo Scientifico Statale Augusto Righi" />
<meta name="citation_author_email" content="maddalena.dilucca@gmail.com" />
<meta name="citation_author" content="Sergio Forcelloni" />
<meta name="citation_author_institution" content="Physics Department, Sapienza University of Rome" />
<meta name="citation_author_email" content="maddalena.dilucca@gmail.com" />
<meta name="citation_author" content="Athanasia Pavlopoulou" />
<meta name="citation_author_institution" content="Izmir Biomedicine and Genome Center (IBG)" />
<meta name="citation_author_institution" content="Izmir International Biomedicine and Genome Institute, Dokuz Eylül University" />
<meta name="citation_author" content="Alexandros G. Georgakilas" />
<meta name="citation_author_institution" content="DNA Damage Laboratory, Physics Department, School of Applied Mathematical and Physical Sciences, National Technical University of Athens (NTUA)" />
<meta name="citation_author" content="Andrea Giansanti" />
<meta name="citation_author_institution" content="Physics Department, Sapienza University of Rome" />
<meta name="citation_author_institution" content="INFN Roma1 unit" />
<meta name="citation_author_email" content="maddalena.dilucca@gmail.com" />
<meta name="citation_reference" content="Belalov IS, Lukashev AN, Causes and Implications of Codon Usage Bias in RNA Viruses PLoS One. 2013; 8(2): e56642. Published online 2013, Feb 25. doi: 10.1371/journal.pone.0056642" />
<meta name="citation_reference" content="Cavanagh D. The Coronavirus Surface Glycoprotein, the Coronaviridae Springer (1995), pp. 73–113" />
<meta name="citation_reference" content="Ceraolo C., Giorgi FM. Genomic variance of the 2019-nCoV coronavirus, DOI: 10.1002/jmv.25700" />
<meta name="citation_reference" content="Cao Y, Li L, Feng Z. et al. Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations. Cell Discov 6, 11 (2020). https://doi.org/10.1038/s41421-020-0147-1" />
<meta name="citation_reference" content="Chen Y, Xu Q, Yuan X, Li X. Analysis of the codon usage pattern in Middle East Respiratory Syndrome Coronavirus, Oncotarget, 2017, Vol. 8, (No. 66), pp: 110337–110349" />
<meta name="citation_reference" content="Darriba D” Posada D, Kozlov AM, Stamatakis A” Morel B, Flouri T. ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models., Mol Biol Evol. 2020 Jan 1;37(1):291–294. doi: 10.1093/molbev/msz189" />
<meta name="citation_reference" content="Dilucca M, Cimini G, Giansanti A. Essentiality, conservation, evolutionary pressure and codon bias in bacterial genomes, Gene. 2018 Jul 15;663:178–188. doi: 10.1016/j.gene.2018.04.017. Epub 2018 Apr 18" />
<meta name="citation_reference" content="Forcelloni, S, Giansanti A. Mutations in disordered proteins as early indicators of nucleic acid changes triggering speciation. Sci Rep 10, 4467 (2020). https://doi.org/10.1038/s41598-020-61466-5" />
<meta name="citation_reference" content="Forni D, Cagliani R, Clerici M, Sironi M. Molecular Evolution of Human Coronavirus Genomes. Trends Microbiol. 2017; 25:35–48" />
<meta name="citation_reference" content="Gorbalenya AE, Luis Enjuanes, John Ziebuhr, Eric J. Snijdera. Nidovirales: Evolving the largest RNA virus genome, Volume 117, Issue 1, April 2006, Pages 17–37 https://doi.org/10.1016/j.virusres.2006.01.017" />
<meta name="citation_reference" content="Grantham R, Gautier C, Gouy M, Mercier R and Pavé A. Codon catalog usage and the genome hypothesis, Nucleic Acids Research, vol. 8, no. 1, pp. r49–r62, 1980" />
<meta name="citation_reference" content="Gu W., Zhou T. Analysis of synonimous codon usage in SARS Coronavirus and other viruses in the Nidovirales, Virus Res. 2004 May;101(2):155–61" />
<meta name="citation_reference" content="Jenkins GM and Holmes EC. The extent of codon usage bias in human RNA viruses and its evolutionary origin, Virus Research, vol. 92, no. 1, pp. 1–7, 2003" />
<meta name="citation_reference" content="Ji W, Wang W, Zhao X, Zai J, Li X. Cross-species transmission of the newly identified coronavirus 2019-nCoV. J Med Virol. 2020;92: 433–440. https://doi.org/10.1002/jmv.25682" />
<meta name="citation_reference" content="Kumar S Stecher G, Li M, Knyaz C, Tamura K. MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Mol Biol Evol. 2018 Jun 1;35(6):1547–1549. doi: 10.1093/molbev/msy096" />
<meta name="citation_reference" content="Lia G, Wang H, Wanga S, Xinga G, Zhanga C, Zhanga W. Insights into the genetic and host adaptability of emerging porcine circovirus, VIRULENCE 2018, VOL. 9, NO. 1, 1301–1313 https://doi.org/10.1080/21505594.2018.1492863" />
<meta name="citation_reference" content="Neuman BW, Kiss G, Kunding AH, Bhella D, Baksh MF, Connelly S. A structural analysis of M protein in coronavirus assembly and morphology J. Struct. Biol., 174 (2011), pp. 11–22" />
<meta name="citation_reference" content="Ruch T, Machamer C. The coronavirus E protein: assembly and beyond Viruses, 4 (2012), pp. 363–382" />
<meta name="citation_reference" content="Sheihk. Analysis of preferred codon usage in the coronavirus N genes and their implications for genome evolution and vaccine design, Journal of Virological Methods 277 (2020) 113806" />
<meta name="citation_reference" content="Siddell SG, Ziebuhr J, Snijder EJ. Coronaviruses, Toroviruses, and Arteriviruses. Topley and Wilson’s Microbiology and Microbial Infections (2005)" />
<meta name="citation_reference" content="Sievers F, Higgins DG. Clustal omega. Curr Protoc Bioinformatics. 2014 Dec 12;48:3.13.1-16. doi: 10.1002/0471250953.bi0313s48" />
<meta name="citation_reference" content="Sievers F, Higgins DG. Methods Mol Biol. 2014;1079:105–16. doi: 10.1007/978-1-62703-646-76 Clustal Omega, accurate alignment of very large numbers of sequences" />
<meta name="citation_reference" content="Sueoka N. Directional mutation pressure and neutral molecular evolution. Proc Natl Acad Sci U S A. 1988; 85:2653–2657" />
<meta name="citation_reference" content="Timani KA, Ye L, Ye L, Zhu Y, Wu Z, Gong Z. Cloning, sequencing, expression, and purification of SARS-associated coronavirus nucleocapsid protein for serodiagnosis of SARS, J Clin Virol. 2004 Aug;30(4):309–12." />
<meta name="citation_reference" content="Weinstein JN, Myers TG, O’Connor PM, Friend SH, Fornace AJ Jr, Kohn KW, Fojo T, Bates SE, Rubinstein LV, Anderson NL, Buolamwini JK, van Osdol WW, Monks AP, Scudiero DA, Sausville EA, Zaharevitz DW, Bunow B, Viswanadhan VN, Johnson GS, Wittes RE, Paull KD (1997) An information-intensive approach to the molecular pharmacology of cancer. Science 275(5298):343–9" />
<meta name="citation_reference" content="Woo PC, Huang Y, Lau SK, Yuen KY. Coronavirus genomics and bioinformatics analysis. Viruses 2010 Aug;2(8):1804–20. doi: 10.3390/v2081803. Epub 2010 Aug 24. PMID: 21994708; PMCID: PMC3185738" />
<meta name="citation_reference" content="Woo PC, Wong BH, Huang Y, Lau SK, Yuen KY. Cytosine deamination and selection of CpG suppressed clones are the two major independent biological forces that shape codon usage bias in coronaviruses, Virology 2007, 369, 431–442" />
<meta name="citation_reference" content="Woo PC, Lau Y, Huang Y, Yuen KY. Coronavirus Diversity, Phylogeny and Interspecies Jumping. Experimental Biology and Medicine, 234(10), 1117–1127.(2009). https://doi.org/10.3181/0903-MR-94" />
<meta name="citation_reference" content="Wright F. The ‘effective number of codons’ used in a gene, Gene. 1990 Mar 1;87(1):23–9" />
<meta name="citation_reference" content="Wu A, Peng Y et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China Cell Host Microbe, Cell Host Microbe. 2020 Mar 11;27(3):325–328. doi: 10.1016/j.chom.2020.02.001. Epub 2020 Feb 7" />
<meta name="citation_reference" content="Xia X, DAMBE5: a comprehensive software package for data analysis in molecular biology and evolution., Mol Biol Evol. 2013 Jul;30(7):1720–8. doi: 10.1093/molbev/mst064. Epub 2013 Apr 5" />
<meta name="twitter:title" content="Codon usage and evolutionary rates of the 2019-nCoV genes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.

*   MDPI
    :   Multidisciplinary Digital Publishing Institute
    DOAJ
    :   Directory of open access journals
    TLA
    :   Three letter acronym
    LD
    :   linear dichroism" />
<meta name="og-title" property="og:title" content="Codon usage and evolutionary rates of the 2019-nCoV genes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.

*   MDPI
    :   Multidisciplinary Digital Publishing Institute
    DOAJ
    :   Directory of open access journals
    TLA
    :   Three letter acronym
    LD
    :   linear dichroism" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-29" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.25.006569v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.25.006569v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.25.006569v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1215996" />
    <title>Codon usage and evolutionary rates of the 2019-nCoV genes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DypRHahGfIswWETk9lAi4389yg-uaro4O3157Hq6vjU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1215996","apath":"\/biorxiv\/early\/2020\/03\/29\/2020.03.25.006569.atom","pisa":"biorxiv;2020.03.25.006569v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.25.006569v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.25.006569v2","title":"Codon usage and evolutionary rates of the 2019-nCoV genes","identifier":"node\/1215996"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hXvJowzOms\\\\\\\/qGdJ2OzNnexSOY54iOWGuLYQfXv8sz85CgWDPA+W6z5UritMWZ\\\\\\\/lMOXYSk2MQZ9QVqewJiXCrq9jQShUqDevRiK6h\\\\\\\/cHINIoYwGgFyWejr+8iuuNO2vG2BffFK3FLaXZO2dDzFO\\\\\\\/gE9Vihr2f\\\\\\\/H7eb2R84TRXks2gXc9ZDXLqkZXjmC7yb1vMGkaIud6C6jMlplWgSht96o3UKdugvqxTRsZj\\\\\\\/OeCvV76V7bTdSu2N8gMl3aUAiJdESk3zS8O6L3Ev5ieI0qwy\\\\\\\/hF3jEEk+n3rYIEvRdiwqlDwb6JtgnrkXbj+ViotHz+gLNqrSxvUv0bpDaEMWejvPvPlWWlk9mJwmiLNWLTB9Zq7spKNTFYfc4K6T7uXX7q7WKQfzDYrS57Ncz\\\\\\\/fy2iWcJCJErBJt7FhCrDhRU2M80B9D7mT7A6\\\\\\\/uPBZ1zvyr1dVreucrOxA8ITTJPzfJr9n\\\\\\\/9zg9fTE3FdhzqcE4XsOP234qUeqtFYsksLLUsCO02AO2YoA62ieIdx3mvFAEPlu5XwhK12QKw1qvf60cPDfLJTWe6plVsBWoB9tz0HVBOQ1pLeplmFDTs7kEk+Abp\\\\\\\/ugaEGq3X8y1lfc50fVAKtnpFOfTK5RLmnTyrtkDOcjtntDz0en\\\\\\\/bdE5aV2J0lRGcMtL8xCaHhqqT0hxagG2ckejavbwTqM1glHV9xp24QejEcoTVwvg3KvB5rdBTokmEBa1IJreWFZK57Vux\\\\\\\/+lsFRM3yw9S3d4GS\\\\\\\/cuiv35YfNnU4zij\\\\\\\/3quN6UjMEDkcpd8h+\\\\\\\/HHKxMBTA+DHYElKxJCUcrNP3yuQPTUaJif+Krbb7ny87pZEI2pyphU\\\\\\\/gEkFPxRYl1THvGwFU79d23PYi0PW1KXm8ZdKC5dEDhu68zFULPXCBbT6ocloSV1qAz\\\\\\\/zh8n8tzYMwWu6YA3qv6KjlvVcp4Lcj7tjxeNZRFt5Qo3Cnd6gXZq\\\\\\\/RQTaJEE2dfxHwe5sahhfU2K2kKkkcwkKVz1Wu0pwEDDUfueTBOBREf6O3d3FMD43DS3a7SS26vgLXvcURYXSvMwY7YNhfWQj2KhtgZODIIsEIA+MwkV1umJEMfly+6UM4iVCCKlZHIlJ3Z13vHRfxb6hRMdJ432fvY9fsGCfwlNXtHszzH5m9SyrOa\\\\\\\/czF3eGCFm3e1sbE+YS14Y+A8c3y8gslUr4Imm7JedfiUKsjfURnAIxnZwzLBTuoMgfCkUm7wFb2Qz2BJ7\\\\\\\/yjOEnX\\\\\\\/ubwV1IYgah+6fNtucl4bJ0\\\\\\\/56y7WT3b4ntYeQNXR6IoiTaTh4IvAQexySvZBPZfBow\\\\\\\/pjd6a0UDZeUdQUQzpgnBJbHzRBOIJliCwh\\\u0022,\\\u0022iv\\\u0022:\\\u00227dOGXP1CA\\\\\\\/a6VTUB+4PEzQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228fda0af34cc6c290e0e2af653f2c8fe69b50193979cfa46dc8f6f8b1aa286c51\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.25.006569v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1215996 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.25.006569v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d3-2kDjsynidtoWP6l7m6obHYBdNQ60XvePEVR7rAd8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215996" id="node1215996" data-pisa="biorxiv;2020.03.25.006569v2" data-pisa-master="biorxiv;2020.03.25.006569" data-apath="/biorxiv/early/2020/03/29/2020.03.25.006569.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Codon usage and evolutionary rates of the 2019-nCoV genes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maddalena</span> <span class="nlm-surname">Dilucca</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sergio</span> <span class="nlm-surname">Forcelloni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Athanasia</span> <span class="nlm-surname">Pavlopoulou</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alexandros G.</span> <span class="nlm-surname">Georgakilas</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Giansanti</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.25.006569 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1215996" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maddalena Dilucca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Physics Department, Sapienza University of Rome</span>, Rome, <span class='nlm-country'>Italy</span>; </div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Liceo Scientifico Statale Augusto Righi</span>, Rome, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maddalena%2BDilucca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilucca%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaddalena%2BDilucca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maddalena.dilucca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maddalena.dilucca@gmail.com</a>
<a href="mailto:maddalena.dilucca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maddalena.dilucca@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sergio Forcelloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Physics Department, Sapienza University of Rome</span>, Rome, <span class='nlm-country'>Italy</span>; </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergio%2BForcelloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Forcelloni%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASergio%2BForcelloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maddalena.dilucca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maddalena.dilucca@gmail.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Athanasia Pavlopoulou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Izmir Biomedicine and Genome Center (IBG)</span>, 35340 Balcova, Izmir, <span class='nlm-country'>Turkey</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Izmir International Biomedicine and Genome Institute, Dokuz Eylül University</span>, 35340 Balcova, Izmir, <span class='nlm-country'>Turkey</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Athanasia%2BPavlopoulou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pavlopoulou%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAthanasia%2BPavlopoulou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alexandros G. Georgakilas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>DNA Damage Laboratory, Physics Department, School of Applied Mathematical and Physical Sciences, National Technical University of Athens (NTUA)</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandros%2BG.%2BGeorgakilas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Georgakilas%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandros%2BG.%2BGeorgakilas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrea Giansanti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Physics Department, Sapienza University of Rome</span>, Rome, <span class='nlm-country'>Italy</span>; </div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>INFN Roma1 unit</span>, Rome, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BGiansanti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giansanti%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BGiansanti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maddalena.dilucca@gmail.com" class="" data-icon-position="" data-hide-link-title="0">maddalena.dilucca@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.25.006569v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1215996" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1215996/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006569v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1215996" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1215996/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006569v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1215996" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1215996/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006569v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1215996" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1215996/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.25.006569v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1215996" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1215996/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Severe acute respiratory syndrome coronavirus 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality ranging from 3% to 6%. To better understand the evolution of the newly emerging 2019-nCoV, in this paper, we analyze the codon usage pattern of 2019-nCoV. For this purpose, we compare the codon usage of 2019-nCoV with that of other 30 viruses belonging to the subfamily of orthocoronavirinae. We found that 2019-nCoV shows a rich composition of AT nucleotides that strongly influences its codon usage, which appears to be not optimized to human host. Then, we study the evolutionary pressures influencing the codon usage and evolutionary rates of the sequences of five conserved genes that encode the corresponding proteins (viral replicase, spike, envelope, membrane and nucleocapsid) characteristic of coronaviruses. We found different patterns of both mutational bias and nature selection that affect the codon usage of these genes at different extents. Moreover, we show that the two integral membrane proteins proteins (matrix and envelope) tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, genes encoding nucleocapsid (N), viral replicase and spike proteins are important targets for the development of vaccines and antiviral drugs, tend to evolve faster as compared to other ones. Taken together, our results suggest that the higher evolutionary rate observed for these two genes could represent a major barrier in the development of antiviral therapeutics 2019-nCoV.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-4">The name “coronavirus” is derived from the Greek <em>κ</em>o<em>ρωνα</em>, that mean crown. The first complete genome of coronavirus (mouse hepatitis virus - MHV), a positive sense, single-stranded RNA virus, was first reported in 1949. It belongs to the family Coronaviridae and ranges from 26.4 (ThCoV HKU12) to 31.7 (SW1) kb in length [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], making it the largest genomes among all known RNA viruses, with G + C contents varying from 32% to 43% [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. The coronavirinae family consists of four genera based on their genetic properties, including genus: <em>Alphacoronavirus, Betacoronavirus</em> (subdivided in subgroups A, B, C and D), <em>Gammacoronavirus</em> and <em>Deltacoronavirus</em>. Coronavirus can infect humans and many different animal species, including swine, cattle, horses, camels, cats, dogs, rodents, birds, bats, rabbits, ferrets, mink, snake, and other wildlife animals. To date, 30 CoVs genomes have been identified. Only seven CoVs have been identified that infect humans, including Human CoV-229E (HCoV-229E), Human coronavirus NL63 (HCoV-NL63), HumanCoV-OC43 (HCoV-OC43), Human CoV-HKU1 (HCoV-HKU1), Human SARS related coronavirus (SARSr-CoV), Middle East Respiratory Syndrome (MERS-CoV) and SARS CoV 2 (nCoV) [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p><p id="p-5">Severe acute respiratory syndrome CoV 2 (2019-nCoV), which first broke out in Wuhan (China) in December of 2019, causes a severe acute respiratory illness with a mortality rate from 3% to 6%. The newly sequenced virus genome encodes two open reading frames (ORFs), ORF1a and ORF1ab, the latter encodes replicase polyproteins, and four structural proteins [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]: the spike-surface glycoprotein (protein S), the small envelop protein (protein E), the matrix protein (M), and the nucleocapsid protein (N).</p><p id="p-6">The phenomenon of codon usage bias (CUB) exists in many genomes including RNA genomes and it is actually determined by mutation and selection [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. As it is well-known, the non-random choice of synonymous codons varies among species that are potential host for viruses [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] [<strong>?</strong>]. It is then important to study patterns of common codon usage in coronaviruses because CUB can be related to the driving forces that shape the evolution of small RNA viruses. Mutational bias has been considered as the major determinant of codon usage variation among RNA viruses [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Indeed, RNA viruses show an effective number of codons (ENC) that is quite high (ENC&gt;45) pointing to a quite random codon usage, whereas if one uses the adaptive index CAI then the viral CUB is consistent with that of the host, as observed in the Equine infectious anemia virus (EIAV) or Zaire ebolavirus (ZEBOV) [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. The aim of this study was to carry out a comprehensive analysis of codon usage and composition of coronavirus and explore the possible leading evolutionary determinants of the biases found.</p></div><div class="section" id="sec-2"><h2 class="">2. Results</h2><div id="sec-3" class="subsection"><h3>2.1. Nucleotide composition</h3><p id="p-7">The nucleotide content of 30 coronaviruses coding sequences are calculated. The results reveal that the nucleotide A is the most frequent base and the nucleotide frequencies are in order A &gt;T&gt;G&gt;C [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>]. For nCoV we note a different trend T&gt;A&gt;G&gt;C, but in totally AT% is more used than GC% (See <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The GC content in 2019-nCoV is 0.373 <em>±</em> 0.05.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1222683/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1222683/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1222683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Coronaviruses (CoVs) in dataset.</span><p id="p-8" class="first-child">We report name, abbreviation, NCBI code and bp size for each virus.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1222689/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1222689/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1222689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Codon usage bias of different coronavirus.</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1222680/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1222680/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1222680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-9" class="first-child">Statistics in nCoV.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>2.2. All the sequenced 2019-nCoV genomes share a common codon usage</h3><p id="p-10">We downloaded the protein-coding sequences of 2019-nCoV from GIDAID database, by classifying each n-CoV based on the geographic location where they are sequenced (see tree in A2). For each n-CoV genome, we calculated the relative synonymous codon usage (RSCU), in the form of a 61-component vector. In <a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig A1</a>, we show the heatmap and the associated clustering of these vectors. Despite the high mutation rate that characterizes 2019-nCoV genome [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], the overall codon usage bias among 2019-nCoV strains appears to be similar. Moreover, their associated vectors did not cluster according to geographic location, thus confirming the common origin of these genomes. Therefore, we will consider a unique vector to represent 2019-nCoV in the following analyses.</p></div><div id="sec-5" class="subsection"><h3>2.3. Codon usage of 2019-nCoV genome strains</h3><p id="p-11">We compared the codon usage of 2019-nCoV with that of the other coronavirus genomes. For this purpose, we used the RSCU which is a biologically relevant metric of distance between the codon usage in the protein-coding sequences of these genomes. In <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>, we report the heatmap of the RSCU values associated with the coronavirus. The RSCU values of the majority of the codons scored between 0 and 3.1 (see legend in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Interestingly, analysis of codon bias in coronavirus genomes reveal that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence has a clear origin from other beta-CoV’s and is closer, specifically, to SARSr-CoV, consistent with their phylogenetic relationships (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure A3</a>). In line with previous observations, we show that the mean CpG relative abundance in the coronavirus genomes is markedly suppressed [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Specifically, GGG, GGC, CCG (pyrimidine-CpG) and ACG (purine-CpG) present a very low frequence of occurrence, probably due to the relative tRNA abundance of the host. In fact, in human the abundance of tRNA in codons GGG, GGC, CCG and ACG are respectavely: 0, 0, 4 and 7.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Clustering of the RSCU vectors associated to coronavirus. Analysis of codon bias in coronavirus genomes reveal that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence was closer to SARSr-CoV as in the reconstructed phylogenetic tree shown in Figure A3. Heatmap are drawn with the CIMminer software [25], which uses Euclidean distances and the average linkage algorithm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Clustering of the RSCU vectors associated to coronavirus.&lt;/span&gt; Analysis of codon bias in coronavirus genomes reveal that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence was closer to SARSr-CoV as in the reconstructed phylogenetic tree shown in Figure A3. Heatmap are drawn with the CIMminer software [25], which uses Euclidean distances and the average linkage algorithm.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F1.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F1.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Clustering of the RSCU vectors associated to coronavirus.</span><p id="p-12" class="first-child">Analysis of codon bias in coronavirus genomes reveal that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence was closer to SARSr-CoV as in the reconstructed phylogenetic tree shown in <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure A3</a>. Heatmap are drawn with the CIMminer software [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], which uses Euclidean distances and the average linkage algorithm.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Z-score values. Z-score is calculated for two codon bias indexes ENC and CAI. We show in figure that a lot of coronavirus have a codon usage major than the average value of the family (Z-score 3). For what concerns 2019-nCoV, it presents average CAI and ENC higher than the average too." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Z-score values.&lt;/span&gt; Z-score is calculated for two codon bias indexes ENC and CAI. We show in figure that a lot of coronavirus have a codon usage major than the average value of the family (Z-score 3). For what concerns 2019-nCoV, it presents average CAI and ENC higher than the average too.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F2.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F2.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Z-score values.</span><p id="p-13" class="first-child">Z-score is calculated for two codon bias indexes ENC and CAI. We show in figure that a lot of coronavirus have a codon usage major than the average value of the family (Z-score 3). For what concerns 2019-nCoV, it presents average CAI and ENC higher than the average too.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">In 2019-nCoV, the most frequently used codons are CGU (arginine, 2.34 times) and GGU (glycine, 2.42), whereas the least used codons are GGG (glycine) and UCG (serine). We also note the most frequently used codons for each amino acid, ended in either U or A. [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>]</p></div><div id="sec-6" class="subsection"><h3>2.4. The codon usage of 2019-nCoV is not optimized to the host</h3><p id="p-15">To measure the codon usage bias in the coronavirus genomes, we used the effective number of codons (ENC) value and the Competition Adaptation Index (CAI). In <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a>, we report the ENC and CAI values for all the coronaviruses we considered in this work. To visually enhance the differences among the codon usage of these coronaviruses we calculated Zscore of a single virus in relation to the other members of the CoV subfamily. Interestingly, the ENC value associated to 2019-nCoV (51.9 <em>±</em> 2.59) is lower than the average of all Coronaviruses, meaning that 2019-nCoV uses a broader set of synonymous codons in its coding sequences. We also note that CAI of 2019-nCoV (0.727 <em>±</em> 0.054) is lower than the average one, underscoring that this coronavirus tend to use codons that are not optimized to the host. We calculate similarity index (SiD) to measure the effect of codon usage bias of the host (human) on the 2019-nCoV genes. The SiD values range from 0 to 1, with higher values indicating that the host has a dominant effect on the usage codons. [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] In our case SiD for 2019-nCoV is equal to 0.78.</p></div><div id="sec-7" class="subsection"><h3>2.5. Genes</h3><p id="p-16">The genome of the newly emerging 2019-nCoV consists of a single, positive-stranded RNA, that is approximately 30K nucleotides long. The overall genome is organizated similar to other coronaviruses. Ceraolo et al. performed a cross-species analysis for all proteins encoded by the 2019-nCoV (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>-<a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a> in [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]). The newly sequenced virus genome encodes polyproteins, common to all betacoronavirus, which are further cleaved into the individual structural proteins E, M, N and S, as well the non-structural RdRP [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. So, only five viral genes are classified according to their viral location and studied for each virus because the short length and insufficient codon usage diversity of the other genes might have biased the results. The corresponding gene products including S protein regulates virus attachment to the receptor of the target host cell [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]; the E protein functions to assemble the virions and acts as an ion channel [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], the M protein plays a role in virus assembly and is involved in biosynthesis of new virus particles [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] the N protein forms the ribonucleoprotein complex with the virus RNA [<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>]. Finally, RdRP catalyzes viral RNA syntesis. We calculate for these five proteins the RSCU vectors in each virus of the dataset (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). We show that only for gene E, M and N 2019-nCoV cluster with SARSr-CoV and SARSr-Rh-BatCoV HKU3, consistent with the inferred phylogeny shown in <a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure A3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RSCU vectors of five different genes. Heatmaps confirm that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence is closer to SARS and HCoV 229E, except for gene RdRP and gene S." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;RSCU vectors of five different genes.&lt;/span&gt; Heatmaps confirm that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence is closer to SARS and HCoV 229E, except for gene RdRP and gene S.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F3.medium.gif" width="440" height="392"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F3.medium.gif" width="440" height="392"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">RSCU vectors of five different genes.</span><p id="p-17" class="first-child">Heatmaps confirm that the newly identified coronavirus 2019-nCoV Wuhan-Hu-1 sequence is closer to SARS and HCoV 229E, except for gene RdRP and gene S.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>2.6. The ENC plot analysis of individual genes of 2019-nCoV</h3><p id="p-18">To further investigate which factors account for the low codon usage bias of the different genes in coronavirus, we analyzed the relationship between the ENC value and the percentage of G or C in the third positioncodons (GC3s). In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, we show separately the ENC-plots obtained for the five genes (M, N, S, E, and RdRP), together with the Wright’s theoretical curve, corresponding to the case when the codon usage is only determined by the GC3s [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. We would like to point out that if mutational bias, as quantified by GC-content in the generally neutral 3rd codon position, is the main factor in determining the codon usage among these genes, the corresponding point in the ENC-plot should lie on or just below Wright’s curve. In <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, all distributions lie below the theoretical curve, an indication that not only mutational bias but also natural selection plays a non-negligible role in the codon choice in all genes. This is exemplified by the violinplots in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> of the distances between the genes and Wright’s theoretical curve. Genes N, S, and RdRP are more scattered below the theoretical curve than those of genes M and E, implying that in the latter the codon usage patterns are pretty consistent with the effects of mutational bias. Interestingly, data points corresponding to the gene N, which is the major viral structural component needed to protect and encapsidate the viral RNA, is localized around GC3=0.5 (See <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). This means that the displacement under the curve most likely reflects the selective pressure exerted on this gene. Conservely, all other genes show a displacement towards lower values of GC3-content, thus corroborating our previously mentioned observation that coronaviruses tend to use codons that end with A and U (see <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">section 2.3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ENC-plots of genes M, N, S, E, RdRP. In these plots, each point corresponds to a single gene. The black-dotted lines in all panels are plots of Wright&#x2019;s theoretical curve corresponding to the case of a CUB merely due to mutational bias (no selective pressure). Red dots represent 2019-nCoV genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;ENC-plots of genes M, N, S, E, RdRP.&lt;/span&gt; In these plots, each point corresponds to a single gene. The black-dotted lines in all panels are plots of Wright&#x2019;s theoretical curve corresponding to the case of a CUB merely due to mutational bias (no selective pressure). Red dots represent 2019-nCoV genes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F4.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F4.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">ENC-plots of genes M, N, S, E, RdRP.</span><p id="p-19" class="first-child">In these plots, each point corresponds to a single gene. The black-dotted lines in all panels are plots of Wright’s theoretical curve corresponding to the case of a CUB merely due to mutational bias (no selective pressure). Red dots represent 2019-nCoV genes.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distances from Wright&#x2019;s curve. Box plots are calculated for each gene for values of distance from theoretical Wright&#x2019;s curve. Genes N, S, and RdRP are more scattered below the theoretical curve than those of genes M and E, implying that in the latter the codon usage patterns are pretty consistent with the effects of mutational bias." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Distances from Wright&#x2019;s curve.&lt;/span&gt; Box plots are calculated for each gene for values of distance from theoretical Wright&#x2019;s curve. Genes N, S, and RdRP are more scattered below the theoretical curve than those of genes M and E, implying that in the latter the codon usage patterns are pretty consistent with the effects of mutational bias.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F5.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F5.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Distances from Wright’s curve.</span><p id="p-20" class="first-child">Box plots are calculated for each gene for values of distance from theoretical Wright’s curve. Genes N, S, and RdRP are more scattered below the theoretical curve than those of genes M and E, implying that in the latter the codon usage patterns are pretty consistent with the effects of mutational bias.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neutrality plot of genes M, N, S, E, RdRP. In these plots, each point corresponds to a single gene in a virus. The solid black lines in all panels are the bisectors corresponding to the case of a CUB merely due to mutational bias (no selective pressure). The black-dotted lines are the linear regressions. Red dots represent 2019-nCoV genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Neutrality plot of genes M, N, S, E, RdRP.&lt;/span&gt; In these plots, each point corresponds to a single gene in a virus. The solid black lines in all panels are the bisectors corresponding to the case of a CUB merely due to mutational bias (no selective pressure). The black-dotted lines are the linear regressions. Red dots represent 2019-nCoV genes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F6.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F6.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Neutrality plot of genes M, N, S, E, RdRP.</span><p id="p-21" class="first-child">In these plots, each point corresponds to a single gene in a virus. The solid black lines in all panels are the bisectors corresponding to the case of a CUB merely due to mutational bias (no selective pressure). The black-dotted lines are the linear regressions. Red dots represent 2019-nCoV genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>2.7. Neutrality plot</h3><p id="p-22">A neutrality plot analysis was performed to estimate the role of mutational bias and natural selection in shaping the codon usage patterns of the five genes under investigation. In this plot, the average GC-content in the first and second positions of codons (GC12) is plotted against GC3s, which is considered as a pure mutational parameter. In <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, we report the neutrality plots obtained for genes M, N, S, E, RdRP, together with the best-fit lines and the slopes associated with them.</p><p id="p-23">The rationale to understand the results is that the wider is the deviation between the slope of the regression line and the bisector, the stronger is the action of selective pressure. All correlations are highly significant (Spearman correlation - <em>R</em><sup>2</sup> analysis, p-value&lt;0.0001). By comparing the divergences between the regression lines and the bisectors in each panel, we reveal that the five genes here considered are subject to different balance between natural selection and mutational bias. Specifically, in line with the ENC-plot analyses, the genes S and RdRP present the largest deviations of their regression lines from the bisector lines and, therefore, a stronger action of natural selection. Conversely, the regression line for the gene M is closer to the bisector than the other genes, meaning that the this gene is the least one subject to the action of natural selection. Finally, the genes E and N are intermediate between the previous cases.</p><p id="p-24">It is worth noting that almost all data points are located below the bisector lines, implying a selective tendency for a higher AT content in the first two codon positions than in the third one. In addition, both GC3 and GC12 are lower than 0.5, thus showing a general preference for A and T bases in all three codon positions. Interestingly, data points associated to gene M and E are closer to the bisector lines as compared to gene N, S, and RdRP. This observation means that the GC content in the first two codon positions tend to be in proportion to GC3 in gene M and E, and this partially explains the closeness of these two genes to the Wright theoretical curve in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>.</p></div><div id="sec-10" class="subsection"><h3>2.8. Forsdyke plot</h3><p id="p-25">We analyze the rate of evolution that characterize these genes by confronting the nucleotide sequences of the newly emerging 2019-nCoV and their corrisponding protein sequences with those of other coronaviruses here considered. Each gene of 2019-nCoV was compared to the orthologous gene in one of the 30 coronaviruses to estimate evolutionary divergences. Each pair of homologous genes is represented by a point in the Forsdyke plot [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], which correlates protein divergence with DNA divergence. Each point in Forsdyke plots measures the divergence between pairs of orthologous genes in the two species, as projected along with the phenotypic (protein) and nucleotidic (DNA) axis. Thus, the slope is an estimation of the fraction of DNA mutations that result in amino acid substitutions in the speciation process between two species [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>]. In <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, we show separately for each gene the associated Forsdyke plot.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Forskdyke plots of genes M, N, S, E, RdRP. Phenotype (Protein div) vs. nucleotide (DNA div) sequence divergence between 2019-nCoV and homologous genes in the other coronaviruses. Each point corresponds to an individual gene. In each panel, we report the best-fit line in red, together with the associated values of the slope (m) and the intercept (q) in the legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Forskdyke plots of genes M, N, S, E, RdRP.&lt;/span&gt; Phenotype (Protein div) vs. nucleotide (DNA div) sequence divergence between 2019-nCoV and homologous genes in the other coronaviruses. Each point corresponds to an individual gene. In each panel, we report the best-fit line in red, together with the associated values of the slope (m) and the intercept (q) in the legend.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F7.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F7.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">Forskdyke plots of genes M, N, S, E, RdRP.</span><p id="p-26" class="first-child">Phenotype (Protein div) vs. nucleotide (DNA div) sequence divergence between 2019-nCoV and homologous genes in the other coronaviruses. Each point corresponds to an individual gene. In each panel, we report the best-fit line in red, together with the associated values of the slope (m) and the intercept (q) in the legend.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Overall, protein and DNA sequence divergences are linearly correlated, and these correlations can be expressed through slopes and intercepts of the regression lines.</p><p id="p-28">Genes M and E show quite low slopes, indicating that these proteins tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, the steeper slopes for genes N, RdRP, and S indicate that these genes tend to evolve faster compared to other ones. A plausible explanation for this observation is that protein N due to its immunogenicity, has been frequently used to generate specific antibodies against various animal coronavirus, including in SARS [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. The viral replicase polyprotein that is essential for the replication of viral RNA and, finally, the gene S encodes the protein that is responsible for the “spikes” present on the surface of coronaviruses. Taken together, our results suggest that the higher evolutionary rate observed for these proteins could represent a major obstacle to in the development of a antiviral therapeutics against 2019-nCoV.</p></div></div><div class="section" id="sec-11"><h2 class="">3. Discussion</h2><p id="p-29">To investigate the factors leading to the 2019-nCoV and other viruses related to codon usage patterns, several analytical methods were used in our study. First, the RSCU value of the 2019-nCoV was calculated. Despite the high mutation rate that characterize 2019-nCoV, we do not show differences in codon usage between these genomes. Moreover, their associated vectors did not cluster for geographically positon, thus confirming the common origin of these genomes. The nucleotide composition confirm higher AT% content and low GC% content as it is common in RNA viruses such as Severe Acute Respiratory Syndrome (SARS) [<strong>?</strong>]. The results indicate also that codon usage bias exists and that the 2019-nCoV preferred codons almost all end in U. The codon usage bias was further confirmed by the mean ENC value of 51.9 (a value greater than 45 is considered as a slight codon usage bias due to mutation pressure or nucleotide compositional constraints). The same analysis is effectuated with CAI index, that it is calculated on the with RNA of codons expressed most often. This suggests that the RNA viruses with high ENC values (and low CAI) adapt to the host with various preferred codons. So a low biased codon usage pattern might allow the virus make use of several codons for each amino acid, and might be beneficial for viral replication and translation in the host cells. The ENC-plot analysis reveals that natural selection plays an important role in the codon choice of the five conserved viral genes here considered: RdRP, S, E, M, and N. However, genes N, S, and ORF1ab are more scattered below the theoretical curve than those of genes M and E, implying that in the latters the codon usage is under more strict control by mutational bias. According with this analysis, the genes S and RdRP present the largest deviations of their linear regressions from the bisectors lines in the neutrality plot analysis that implies a stronger action of natural selection. Conversely, the regression line for the gene M is closer to the bisector than the other genes, meaning that this gene is the least one subject to the action of natural selection. Finally, the genes E and N are intermediate between the previous cases.</p><p id="p-30">To analyze the rate of evolution that characterize these genes by confronting the nucleotide sequences we use the Forsdyke plot thar correlates protein divergence with DNA divergence. The two proteins M and E show quite low slopes, indicating that these proteins tend to evolve slowly by accumulating nucleotide mutations on their genes. Conversely, the steeper slopes for gene N and RdRP and the gene S encoding for the S protein, which is responsible for the “spikes” present on the surface of coronaviruses, indicate that these three genes, and therefore their corresponding protein products, tend to evolve faster compared to the other two genes. Findings of the present study could be directed to be useful for developing diagnostic reagents and probes for detecting a range of viruses and isolates in one test and for vaccine development, using the information about codon usage patterns in these genes.</p></div><div class="section" id="sec-12"><h2 class="">4. Materials and Methods</h2><div id="sec-13" class="subsection"><h3>4.1. Sequence data analyzed</h3><p id="p-31">The complete coding genomic sequences of 306 isolates of nCov reported across the world to date, were obtained from GISAID (available at <em><a href="https://www.gisaid.org/epiflu-applications/next-hcov-19-app/">https://www.gisaid.org/epiflu-applications/next-hcov-19-app/</a></em>) and NCBI viruses database accessed as on 17 March 2020.</p><p id="p-32">Then the sequences are selected according to their geographical distribution, the isolation date, and the host species. The complete coding genomic sequences of 30 coronaviruses were downloaded from the National Center for Biotechnological Information (NCBI) (available at <em><a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a></em>). For each virus we consider in following alphabetical order genes: E, M, N, RdRP and S.</p></div><div id="sec-14" class="subsection"><h3>4.2. Nucleotide Composition Analysis</h3><p id="p-33">The diverse nucleotide compositional properties are calculated for the coding sequences of 30 CoV genomes. These compositional properties comprise the frequencies of occurrence of each nucleotide (A, T, G, and C), AU and GC contents, nucleotides G + C at the first (GC1), second (GC2), and third codon positions (GC3). To calculate these values we use an in-house Python script. We calculate, also, mean frequencies of nucleotides G + C at the first and the second positions (GC12).</p></div><div id="sec-15" class="subsection"><h3>4.3. RSCU</h3><p id="p-34"><em>RSCU</em> vectors for all the genomes were computed by using an in-house Python script, following the formula:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-11.gif"/></noscript></span>

</span>
</p><p id="p-35">In the <em>RSCU</em><sub><em>i</em></sub> <em>X</em><sub><em>i</em></sub> is the number of occurrences, in a given genome, of codon i, and the sum in the denominator runs over its <em>n</em><sub><em>i</em></sub> synonymous codons. If the <em>RSCU</em> value for a codon <em>i</em> is equal to 1, this codon was chosen equally and randomly. Codons with <em>RSCU</em> values greater than 1 have positive codon usage bias, while those with value less than 1 have relatively negative codon usage bias [<strong>?</strong>]. <em>RSCU</em> heat maps were drawn with the CIMminer software [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>], which uses Euclidean distances and the average linkage algorithm.</p></div><div id="sec-16" class="subsection"><h3>4.4. Effective Number of Codons Analysis</h3><p id="p-36"><em>ENC</em> is an estimate of the frequency of different codons used in a coding sequence. In general, <em>ENC</em> ranges from 20 (when each aminoacid is coded by just one and the same codon) to 61 (when all synonymous codons are used on an equal footing). Given a sequence of interest, the computation of <em>ENC</em> starts from <em>F</em><sub><em>α</em></sub>, a quantity defined for each family <em>α</em> of synonymous codons (one for each amino acid):
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-12.gif"/></noscript></span>

</span>
where <em>m</em><sub><em>α</em></sub> is the number of different codons in <em>α</em> (each one appearing <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/inline-graphic-1.gif"/></noscript></span></span> times in the sequence) and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/inline-graphic-2.gif"/></noscript></span></span>.</p><p id="p-37"><em>ENC</em> then weights these quantities on a sequence:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-13.gif"/></noscript></span>

</span>
where <em>N</em><sub><em>S</em></sub> is the number of families with one codon only and <em>K</em><sub><em>m</em></sub> is the number of families with degeneracy <em>m</em> (the set of 6 synonymous codons for <em>Leu</em> can be split into one family with degeneracy 2, similar to that of <em>Phe</em>, and one family with degeneracy 4, similar to that, e.g., of <em>Pro</em>).</p><p id="p-38">We have evaluated <em>ENC</em> by using the implementation in <em>DAMBE</em> 5.0 [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>].</p></div><div id="sec-17" class="subsection"><h3>4.5. Codon Adaptation Index</h3><p id="p-39">Codon adaptation index <em>CAI</em> [<strong>?</strong>] is used to quantify the codon usage similarities between the virus and host coding sequences. The principle behind <em>CAI</em> is that codon usage in highly expressed genes can reveal the optimal (<em>i.e</em>., most efficient for translation) codons for each amino acid. Hence, <em>CAI</em> is calculated based on a reference set of highly expressed genes to assess, for each codon <em>i</em>, the relative synonymous codon usages (<em>RSCU</em><sub><em>i</em></sub>) and the relative codon adaptiveness (<em>w</em><sub><em>i</em></sub>):
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-14.gif"/></noscript></span>

</span>
</p><p id="p-40">In the <em>RSCU</em><sub><em>i</em></sub>, <em>X</em><sub><em>i</em></sub> is the number of occurrences of codon <em>i</em> in the genome, and the sum in the denominator runs over the <em>n</em><sub><em>i</em></sub> synonyms of <em>i</em>; <em>RSCU</em>s thus measure codon usage bias within a family of synonymous codons. <em>w</em><sub><em>i</em></sub> is then defined as the usage frequency of codon <em>i</em> compared to that of the optimal codon for the same amino acid encoded by <em>i</em>—<em>i.e</em>., the one which is mostly used in a reference set of highly expressed genes. The <em>CAI</em> for a given gene <em>g</em> is calculated as the geometric mean of the usage frequencies of codons in that gene, normalized to the maximum <em>CAI</em> value possible for a gene with the same amino acid composition:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-15.gif"/></noscript></span>

</span>
where the product runs over the <em>l</em><sub><em>g</em></sub> codons belonging to that gene (except the stop codon).</p><p id="p-41">This index values range from 0 to 1, where the score 1 represents the tendency of a gene to always use the most frequently used synonymous codons in the host. The CAI analysis of these coding sequences are performed using <em>DAMBE</em> 5.0 [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. The synonymous codon usage data of different hostes (human-Homo sapiens and other species) are retrieved from the codon usage database (available at: <a href="http://www.kazusa.or.jp/codon/">http://www.kazusa.or.jp/codon/</a>).</p><p id="p-42">To study the patterns of codon bias in the coronaviruses, we use Z-score values:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-16.gif"/></noscript></span>

</span>
where ⟨<em>ENC</em>⟩ <sub><em>f amily</em></sub> is the average of the ratio within a codon bias index in a family of a virus <em>v</em>, ⟨<em>ENC</em>⟩<sub><em>v</em></sub> and <em>s</em><sub><em>v</em></sub> are the average value of <em>ENC</em> and its standard deviation over the whole virus <em>v</em>, and <em>N</em><sub><em>v</em></sub> is the number of virus in the family (we use the standard deviation of the mean as we are comparing average values).</p><p id="p-43">The same Z-score is evalueted for codon bias index CAI.</p></div><div id="sec-18" class="subsection"><h3>4.6. ENC plot</h3><p id="p-44">ENC-plot analysis was performed to estimate the relative contributions of mutational bias and natural selection in shaping CUB of genes encoding for proteins that are crucial for 2019-nCoV: RdRP, the spike-surface glycoprotein (protein S), the small envelop protein (protein E), the matrix protein (M), and the nucleocapsid protein (N). The ENC-plot is a plot in which the ENC is the ordinate and the GC3 is the abscissa. Depending on the action of mutational bias and natural selection, different cases are discernable. If a gene is not subject to selection, a clear relationship is expected between ENC and GC3 [<strong>?</strong>]:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/29/2020.03.25.006569/embed/graphic-17.gif"/></noscript></span>

</span>
where <em>s</em> represents the value of <em>GC</em><sub>3</sub> [<strong>?</strong>]. Genes for which the codon choice is only constrained by mutational bias are expected to lie on or just below the Wright’s theoretical curve. Alternatively, if a particular gene is subject to selection, it will fall below Wright’s theoretical curve. In this case, the vertical distance between the point and the theoretical curve provides an estimation of the relative extent to which natural selection and mutational bias affect CUB.</p><p id="p-45">To evaluate scatter of dots from theoretical Wright’s curve, we calculate the module of distance and we drawn box plot calculated with an in-house Python script.</p></div><div id="sec-19" class="subsection"><h3>4.7. Neutrality plot</h3><p id="p-46">We used a neutrality plot analysis [<strong>?</strong>] to estimate the relative contribution of natural selection and mutational bias in shaping CUB of five crucial genes in the research field aiming to develop a vacine against 2019-nCoV: M, N, S, RdRP, and E. In this analysis, the GC1 or GC2 values (ordinate) are plotted against the GC3 values (abscissa), and each gene is represented as a single point on this plane. In this case, the three stop codons (TAA, TAG, or TGA) and the three codons for isoleucine (ATT, ATC, and ATA) were excluded in calculation of GC3, and two single codons for methionine (ATG) and tryptophan (TGG) were excluded in all three (GC1, GC2, GC3) (Sueoka 1988). For each gene, we separately performed a Spearman correlation analysis between GC1 and GC2 with the GC3. If the correlation between GC1/2 and GC3 is statistically significant, the slope of the regression line provides a measure of the relative extent to which natural selection and mutational bias affect CUB of these genes (Sueoka 1999). In particular, if the mutational bias is the driving force that shapes CUB, then the corresponding data points should be distributed along the bisector (slope of unity). On the other hand, if natural selection also affects the codon choice of a family of genes, then the corresponding regression line should diverge from the bisector. Thus, the extent of the divergence between the regression line and the bisector indicates that the extent of codon choice due to the natural selection.</p></div><div id="sec-20" class="subsection"><h3>4.8. Forsdyke plot</h3><p id="p-47">To study the evolutionary rates of genes M, N, S, RdRP, and E, we performed an analysis by using our previously defined Forsdyke plot [<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>]. Each gene in 2019-nCoV (taken as a reference) was confronted with the homologous gene in one of the 30 coronaviruses considered in this analysis. Each pair of homologous genes is represented by a point in the Forsdyke plot, which correlates protein divergence with DNA divergence (see Methods in [<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>] for details). The protein sequences were aligned using Biopython. The DNA sequences were then aligned using the protein alignments as templates. Then, DNA divergences and protein divergences were assessed as axplained in Methods in [<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">8</a>] by counting the number of mismatches in each pair of aligned sequences. Thus, each point in Forsdyke plots measures the divergence between pairs of homologous genes in the two species, as projected along with the phenotypic (protein) and nucleotidic (DNA) axis.The first step in each comparison is to compute the regression line between protein vs. DNA sequence divergence in the Forsdyke plot getting values of intercept and slope for each variant of gene. To test whether the regression parameters associated with each variant are different or not, we have followed a protocol founded by F-statistic test, considering a p-value <em>≤</em> 0.05.</p></div><div id="sec-21" class="subsection"><h3>4.9. Phylogenetic analysis</h3><p id="p-48">To explore the evolutionary relationships among the four genera of coronaviruses, phylogenetic analysis of the full-length genomic sequences of the 30 CoVs listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> was performed. The sequences were aligned with the usage of ClustalO [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. The resulting multiple sequence alignment was used to build a phylogenetic tree by employing a maximum-likelihood (ML) method implemented in the software package MEGA version 10. 1 [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. ModelTest-NG [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] was used to select the best-fit evolutionary model of nucleotide substitution, that is, GTR + G + I. Bootstrap analysis (100 pseudo-replicates) was conducted in order to evaluate the statistical significance of the inferred trees.</p></div></div><div class="section" id="sec-22"><h2 class="">Author Contributions</h2><p id="p-49">For research articles with several authors, a short paragraph specifying their individual contributions must be provided. The following statements should be used “Conceptualization, all.; methodology, M.D, S.F.and A.P..; software, M.D and S.F.; validation, all.; formal analysis, M.D, S.F A.P.; investigation, M.D, S.F., A.P; data curation, M.D, S.F., A.P. ; writing–review and editing, all. All authors have read and agreed to the published version of the manuscript.’, please turn to the CRediT taxonomy for the term explanation. Authorship must be limited to those who have contributed substantially to the work reported.</p></div><div class="section" id="sec-23"><h2 class="">Conflicts of Interest</h2><p id="p-50">The authors declare no conflict of interest.</p></div><div class="section app" id="app-1"><h2>Appendix A</h2><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Appendix A.1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F8.medium.gif" width="224" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Appendix A.1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F8.medium.gif" width="224" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Appendix A.1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Appendix A.1.</span> <div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree from GIDAID. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Phylogenetic tree from GIDAID.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F9.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F9.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A2.</span> <span class="caption-title">Phylogenetic tree from GIDAID.</span><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unrooted ML-based tree of the 30 CoV genomic sequences. The four distinct color-coded clades correspond to the respective genera of CoVs. The 2019-nCoV sequence is indicated by a star. The branch lengths depict evolutionary distance. Bootstrap values higher than 50 are shown at the nodes. The scale bar at the lower left denotes the length of nucleotide substitutions per position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1418691957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Unrooted ML-based tree of the 30 CoV genomic sequences.&lt;/span&gt; The four distinct color-coded clades correspond to the respective genera of CoVs. The 2019-nCoV sequence is indicated by a star. The branch lengths depict evolutionary distance. Bootstrap values higher than 50 are shown at the nodes. The scale bar at the lower left denotes the length of nucleotide substitutions per position.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F10.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F10.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.25.006569/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A3.</span> <span class="caption-title">Unrooted ML-based tree of the 30 CoV genomic sequences.</span><p id="p-56" class="first-child">The four distinct color-coded clades correspond to the respective genera of CoVs. The 2019-nCoV sequence is indicated by a star. The branch lengths depict evolutionary distance. Bootstrap values higher than 50 are shown at the nodes. The scale bar at the lower left denotes the length of nucleotide substitutions per position.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section glossary"><h2 class="">Abbreviations</h2><p id="p-51">The following abbreviations are used in this manuscript:</p><dl id="def-list-1"><dt id="def-item-1">MDPI<span class="def-item-dt-sep"></span></dt><dd>Multidisciplinary Digital Publishing Institute<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">DOAJ<span class="def-item-dt-sep"></span></dt><dd>Directory of open access journals<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">TLA<span class="def-item-dt-sep"></span></dt><dd>Three letter acronym<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">LD<span class="def-item-dt-sep"></span></dt><dd>linear dichroism</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.1" data-doi="10.1371/journal.pone.0056642"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belalov</span>  <span class="cit-name-given-names">IS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lukashev</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-article-title">Causes and Implications of Codon Usage Bias in RNA</span> <abbr class="cit-jnl-abbrev">Viruses PLoS One</abbr>. <span class="cit-pub-date">2013</span>; <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e56642</span>. Published online 2013, Feb 25.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0056642</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%2BPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De56642%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0056642%26rft_id%253Dinfo%253Apmid%252F23451064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0056642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23451064&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">The Coronavirus Surface Glycoprotein, the Coronaviridae Springer</abbr> (<span class="cit-pub-date">1995</span>), pp. <span class="cit-fpage">73</span>–<span class="cit-lpage">113</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.3" data-doi="10.1002/jmv.25700"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ceraolo</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giorgi</span>  <span class="cit-name-given-names">FM</span></span>. <abbr class="cit-jnl-abbrev">Genomic variance of the 2019-nCoV coronavirus</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.25700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomic%2Bvariance%2Bof%2Bthe%2B2019-nCoV%2Bcoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.25700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.4" data-doi="10.1038/s41421-020-0147-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations</span>. <abbr class="cit-jnl-abbrev">Cell Discov</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">11</span> (<span class="cit-pub-date">2020</span>). <a href="https://doi.org/10.1038/s41421-020-0147-1">https://doi.org/10.1038/s41421-020-0147-1</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BDiscov%26rft.volume%253D6%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41421-020-0147-1%26rft_id%253Dinfo%253Apmid%252F32133153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41421-020-0147-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32133153&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-article-title">Analysis of the codon usage pattern in Middle East Respiratory Syndrome Coronavirus</span>, <abbr class="cit-jnl-abbrev">Oncotarget</abbr>, <span class="cit-pub-date">2017</span>, Vol. <span class="cit-vol">8</span>, (No. <span class="cit-issue">66</span>), pp: <span class="cit-fpage">110337</span>–<span class="cit-lpage">110349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D8%26rft.spage%253D110337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.6" data-doi="10.1093/molbev/msz189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darriba D” Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kozlov</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stamatakis A” Morel</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flouri</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models</span>., <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Jan</span> 1;<span class="cit-vol">37</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">291</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msz189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006569v2.7" data-doi="10.1016/j.gene.2018.04.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dilucca</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cimini</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giansanti</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Essentiality, conservation, evolutionary pressure and codon bias in bacterial genomes</span>, <abbr class="cit-jnl-abbrev">Gene</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jul</span> 15;<span class="cit-vol">663</span>:<span class="cit-fpage">178</span>–<span class="cit-lpage">188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.gene.2018.04.017</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> Epub 2018 Apr 18</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D663%26rft.spage%253D178%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2018.04.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2018.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forcelloni</span>,  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giansanti</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Mutations in disordered proteins as early indicators of nucleic acid changes triggering speciation</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">4467</span> (<span class="cit-pub-date">2020</span>). <a href="https://doi.org/10.1038/s41598-020-61466-5">https://doi.org/10.1038/s41598-020-61466-5</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D10%26rft.spage%253D4467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.9" data-doi="10.1016/j.tim.2016.09.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cagliani</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clerici</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sironi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Molecular Evolution of Human Coronavirus Genomes</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-vol">25</span>:<span class="cit-fpage">35</span>–<span class="cit-lpage">48</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D25%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2016.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2016.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Luis</span> <span class="cit-name-surname">Enjuanes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Ziebuhr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Eric J.</span> <span class="cit-name-surname">Snijdera</span></span>. <span class="cit-article-title">Nidovirales: Evolving the largest</span> <abbr class="cit-jnl-abbrev">RNA virus genome</abbr>, Volume <span class="cit-vol">117</span>, <span class="cit-issue">Issue 1</span>, <span class="cit-month">April</span> <span class="cit-pub-date">2006</span>, Pages <span class="cit-fpage">17</span>–<span class="cit-lpage">37</span> <a href="https://doi.org/10.1016/j.virusres.2006.01.017">https://doi.org/10.1016/j.virusres.2006.01.017</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2Bvirus%2Bgenome%26rft.volume%253D117%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.11" data-doi="10.1093/nar/8.1.197-c"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pavé</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Codon catalog usage and the genome hypothesis</span>, <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, vol. <span class="cit-vol">8</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">r49</span>–<span class="cit-lpage">r62</span>, <span class="cit-pub-date">1980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253Dr49%26rft.epage%253D62%26rft.atitle%253DCodon%2Bcatalog%2Busage%2Band%2Bthe%2Bgenome%2Bhypothesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.1.197-c%26rft_id%253Dinfo%253Apmid%252F6986610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.1.197-c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6986610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JC46600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006569v2.12" data-doi="10.1016/j.virusres.2004.01.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Analysis of synonimous codon usage in SARS Coronavirus and other viruses in the Nidovirales</span>, <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">May</span>;<span class="cit-vol">101</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">155</span>–<span class="cit-lpage">61</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGu%26rft.auinit1%253DW.%26rft.volume%253D101%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D161%26rft.atitle%253DAnalysis%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2BSARS%2BCoronavirus%2Band%2Bother%2Bviruses%2Bin%2Bthe%2BNidovirales.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2004.01.006%26rft_id%253Dinfo%253Apmid%252F15041183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2004.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15041183&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.13" data-doi="10.1016/S0168-1702(02)00309-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">GM</span></span> and <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>. <span class="cit-article-title">The extent of codon usage bias in human RNA viruses and its evolutionary origin</span>, <abbr class="cit-jnl-abbrev">Virus Research</abbr>, vol. <span class="cit-vol">92</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>, <span class="cit-pub-date">2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DJenkins%26rft.auinit1%253DG.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DThe%2Bextent%2Bof%2Bcodon%2Busage%2Bbias%2Bin%2Bhuman%2BRNA%2Bviruses%2Band%2Bits%2Bevolutionary%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-1702%252802%252900309-X%26rft_id%253Dinfo%253Apmid%252F12606071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-1702(02)00309-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12606071&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181372400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006569v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-article-title">Cross-species transmission of the newly identified coronavirus 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>. <a href="https://doi.org/10.1002/jmv.25682">https://doi.org/10.1002/jmv.25682</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft.volume%253D92%26rft.spage%253D433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.15" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knyaz</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jun</span> 1;<span class="cit-vol">35</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msy096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lia</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wanga</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xinga</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhanga</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhanga</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Insights into the genetic and host adaptability of emerging porcine circovirus</span>, <abbr class="cit-jnl-abbrev">VIRULENCE</abbr> <span class="cit-pub-date">2018</span>, VOL. <span class="cit-vol">9</span>, NO. <span class="cit-issue">1</span>, <span class="cit-fpage">1301</span>–<span class="cit-lpage">1313</span> <a href="https://doi.org/10.1080/21505594.2018.1492863">https://doi.org/10.1080/21505594.2018.1492863</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVIRULENCE%26rft.volume%253D9%26rft.spage%253D1301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunding</span>  <span class="cit-name-given-names">AH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhella</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baksh</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">A structural analysis of M protein in coronavirus assembly and morphology J</span>. <abbr class="cit-jnl-abbrev">Struct. Biol</abbr>., <span class="cit-vol">174</span> (<span class="cit-pub-date">2011</span>), pp. <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStruct.%2BBiol%26rft.volume%253D174%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruch</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machamer</span>  <span class="cit-name-given-names">C.</span></span> <abbr class="cit-jnl-abbrev">The coronavirus E protein: assembly and beyond Viruses</abbr>, <span class="cit-vol">4</span> (<span class="cit-pub-date">2012</span>), pp. <span class="cit-fpage">363</span>–<span class="cit-lpage">382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Bcoronavirus%2BE%2Bprotein%253A%2Bassembly%2Band%2Bbeyond%2BViruses%26rft.volume%253D4%26rft.spage%253D363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.19"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Sheihk</span>. <span class="cit-article-title">Analysis of preferred codon usage in the coronavirus N genes and their implications for genome evolution and vaccine design</span>, <abbr class="cit-jnl-abbrev">Journal of Virological Methods</abbr> <span class="cit-vol">277</span> (<span class="cit-pub-date">2020</span>) <span class="cit-fpage">113806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirological%2BMethods%26rft.volume%253D277%26rft.spage%253D113806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>. <span class="cit-article-title">Coronaviruses, Toroviruses, and Arteriviruses</span>. <abbr class="cit-jnl-abbrev">Topley and Wilson’s Microbiology and Microbial Infections</abbr> (<span class="cit-pub-date">2005</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.21" data-doi="10.1002/0471250953.bi0313s48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">DG</span></span>. <span class="cit-article-title">Clustal omega</span>. <abbr class="cit-jnl-abbrev">Curr Protoc Bioinformatics</abbr>. <span class="cit-pub-date">2014</span> <span class="cit-month">Dec</span> 12;<span class="cit-vol">48</span>:<span class="cit-fpage">3.13.1</span>-<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/0471250953.bi0313s48</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BProtoc%2BBioinformatics%26rft.volume%253D48%26rft.spage%253D3131%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi0313s48%26rft_id%253Dinfo%253Apmid%252F25501942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi0313s48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25501942&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.22" data-doi="10.1007/978-1-62703-646-76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">DG</span></span>. <span class="cit-article-title">Methods Mol Biol</span>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">1079</span>:<span class="cit-fpage">105</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-62703-646-76</span> <abbr class="cit-jnl-abbrev">Clustal Omega, accurate alignment of very large numbers of sequences</abbr></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClustal%2BOmega%252C%2Baccurate%2Balignment%2Bof%2Bvery%2Blarge%2Bnumbers%2Bof%2Bsequences%26rft.volume%253D1079%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-646-76%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-646-76&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006569v2.23" data-doi="10.1073/pnas.85.8.2653"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Directional mutation pressure and neutral molecular evolution</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">1988</span>; <span class="cit-vol">85</span>:<span class="cit-fpage">2653</span>–<span class="cit-lpage">2657</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D85%26rft.issue%253D8%26rft.spage%253D2653%26rft.epage%253D2657%26rft.atitle%253DDirectional%2Bmutation%2Bpressure%2Band%2Bneutral%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.8.2653%26rft_id%253Dinfo%253Apmid%252F3357886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS84LzI2NTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOS8yMDIwLjAzLjI1LjAwNjU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.24" data-doi="10.1016/j.jcv.2004.01.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Timani</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Cloning, sequencing, expression, and purification of SARS-associated coronavirus nucleocapsid protein for serodiagnosis of SARS</span>, <abbr class="cit-jnl-abbrev">J Clin Virol</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Aug</span>;<span class="cit-vol">30</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">309</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.stitle%253DJ%2BClin%2BVirol%26rft.aulast%253DTimani%26rft.auinit1%253DK.%2BA.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D309%26rft.epage%253D312%26rft.atitle%253DCloning%252C%2Bsequencing%252C%2Bexpression%252C%2Band%2Bpurification%2Bof%2BSARS-associated%2Bcoronavirus%2Bnucleocapsid%2Bprotein%2Bfor%2Bserodiagnosis%2Bof%2BSARS.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2004.01.001%26rft_id%253Dinfo%253Apmid%252F15163419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcv.2004.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15163419&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.25" data-doi="10.1126/science.275.5298.343"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weinstein</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friend</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fornace</span>  <span class="cit-name-given-names">AJ</span>   <span class="cit-name-suffix">Jr</span>.</span>, <span class="cit-auth"><span class="cit-name-surname">Kohn</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fojo</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rubinstein</span>  <span class="cit-name-given-names">LV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buolamwini</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Osdol</span>  <span class="cit-name-given-names">WW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monks</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scudiero</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sausville</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaharevitz</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bunow</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viswanadhan</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wittes</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paull</span>  <span class="cit-name-given-names">KD</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">An information-intensive approach to the molecular pharmacology of cancer</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">275</span>(<span class="cit-issue">5298</span>):<span class="cit-fpage">343</span>–<span class="cit-lpage">9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWeinstein%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D275%26rft.issue%253D5298%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DAn%2BInformation-Intensive%2BApproach%2Bto%2Bthe%2BMolecular%2BPharmacology%2Bof%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5298.343%26rft_id%253Dinfo%253Apmid%252F8994024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNTI5OC8zNDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOS8yMDIwLjAzLjI1LjAwNjU2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.26" data-doi="10.3390/v2081803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY</span></span>. <span class="cit-article-title">Coronavirus genomics and bioinformatics analysis</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2010</span> <span class="cit-month">Aug</span>;<span class="cit-vol">2</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1804</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3390/v2081803</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> Epub 2010 Aug 24.<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">21994708</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span><span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID: </span><span class="cit-pub-id cit-pub-id-pmcid">PMC3185738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D1804%26rft.epage%253D1820%26rft.atitle%253DCoronavirus%2Bgenomics%2Band%2Bbioinformatics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv2081803%26rft_id%253Dinfo%253Apmid%252F21994708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v2081803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.27" data-doi="10.1016/j.virol.2007.08.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY</span></span>. <span class="cit-article-title">Cytosine deamination and selection of CpG suppressed clones are the two major independent biological forces that shape codon usage bias in coronaviruses</span>, <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">369</span>, <span class="cit-fpage">431</span>–<span class="cit-lpage">442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D369%26rft.issue%253D2%26rft.spage%253D431%26rft.epage%253D442%26rft.atitle%253DCytosine%2Bdeamination%2Band%2Bselection%2Bof%2BCpG%2Bsuppressed%2Bclones%2Bare%2Bthe%2Btwo%2Bmajor%2Bindependent%2Bbiological%2Bforces%2Bthat%2Bshape%2Bcodon%2Busage%2Bbias%2Bin%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2007.08.010%26rft_id%253Dinfo%253Apmid%252F17881030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2007.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17881030&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.28" data-doi="10.3181/0903-MR-94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY</span></span>. <span class="cit-article-title">Coronavirus Diversity, Phylogeny and Interspecies Jumping</span>. <abbr class="cit-jnl-abbrev">Experimental Biology and Medicine</abbr>, <span class="cit-vol">234</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">1117</span>–<span class="cit-lpage">1127</span>.(<span class="cit-pub-date">2009</span>). <a href="https://doi.org/10.3181/0903-MR-94">https://doi.org/10.3181/0903-MR-94</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%2BY.%26rft.volume%253D234%26rft.issue%253D10%26rft.spage%253D1117%26rft.epage%253D1127%26rft.atitle%253DCoronavirus%2BDiversity%252C%2BPhylogeny%2Band%2BInterspecies%2BJumping%26rft_id%253Dinfo%253Adoi%252F10.3181%252F0903-MR-94%26rft_id%253Dinfo%253Apmid%252F19546349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3181/0903-MR-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.29" data-doi="10.1016/0378-1119(90)90491-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">The ‘effective number of codons’ used in a gene</span>, <abbr class="cit-jnl-abbrev">Gene</abbr>. <span class="cit-pub-date">1990</span> <span class="cit-month">Mar</span> 1;<span class="cit-vol">87</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">23</span>–<span class="cit-lpage">9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DWright%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DThe%2B%2527effective%2Bnumber%2Bof%2Bcodons%2527%2Bused%2Bin%2Ba%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252890%252990491-9%26rft_id%253Dinfo%253Apmid%252F2110097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(90)90491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2110097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DA22500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.30" data-doi="10.1016/j.chom.2020.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China Cell Host Microbe</span>, <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Mar</span> 11;<span class="cit-vol">27</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">325</span>–<span class="cit-lpage">328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2020.02.001</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> Epub 2020 Feb 7</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2020.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006569v2.31" data-doi="10.1093/molbev/mst064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-article-title">DAMBE5: a comprehensive software package for data analysis in molecular biology and evolution</span>., <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Jul</span>;<span class="cit-vol">30</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1720</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mst064</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> Epub 2013 Apr 5</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst064%26rft_id%253Dinfo%253Apmid%252F23564938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23564938&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.25.006569.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.25.006569v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.27.012260v1" title="Extended depth of focus multiphoton microscopy via incoherent pulse splitting" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.27.010926v1" title="CyclinD1 controls development of cerebellar granule cell progenitors through phosphorylation and stabilization of ATOH1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 29, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.25.006569v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.25.006569v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.25.006569v2" class="active" data-icon-position="" data-hide-link-title="0">Codon usage and evolutionary rates of the 2019-nCoV genes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1215996" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2Do-2bXlOqiEh7WFpGH6rH3_tkACRRTDcTe2jji_YnA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215996" id="node1215996--2" data-pisa="biorxiv;2020.03.25.006569v2" data-pisa-master="biorxiv;2020.03.25.006569" data-seqnum="1215996" data-apath="/biorxiv/early/2020/03/29/2020.03.25.006569.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Codon usage and evolutionary rates of the 2019-nCoV genes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maddalena</span> <span class="nlm-surname">Dilucca</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sergio</span> <span class="nlm-surname">Forcelloni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Athanasia</span> <span class="nlm-surname">Pavlopoulou</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alexandros G.</span> <span class="nlm-surname">Georgakilas</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Giansanti</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.006569; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.006569 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;title=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;title=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;text=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;title=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;t=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;title=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;title=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215996" id="node1215996--3" data-pisa="biorxiv;2020.03.25.006569v2" data-pisa-master="biorxiv;2020.03.25.006569" data-seqnum="1215996" data-apath="/biorxiv/early/2020/03/29/2020.03.25.006569.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Codon usage and evolutionary rates of the 2019-nCoV genes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maddalena</span> <span class="nlm-surname">Dilucca</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sergio</span> <span class="nlm-surname">Forcelloni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Athanasia</span> <span class="nlm-surname">Pavlopoulou</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alexandros G.</span> <span class="nlm-surname">Georgakilas</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Giansanti</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.006569; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.006569 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Dilucca%2C+Maddalena&amp;rft.au=Forcelloni%2C+Sergio&amp;rft.au=Pavlopoulou%2C+Athanasia&amp;rft.au=Georgakilas%2C+Alexandros+G.&amp;rft.au=Giansanti%2C+Andrea"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1215996/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1215996/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1215996/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1215996/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1215996/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1215996/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1215996/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1215996/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1215996/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1215996/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1215996/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1215996/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;count=horizontal&amp;via=&amp;text=Codon%20usage%20and%20evolutionary%20rates%20of%20the%202019-nCoV%20genes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006569v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.006569v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
