<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/27/2020.02.27.967885.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.27.967885v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Prediction of receptorome for human-infecting virome"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.27.967885"/>
  <meta name="DC.Date" content="2020-02-28"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome."/>
  <meta name="DC.Contributor" content="Zheng Zhang"/>
  <meta name="DC.Contributor" content="Sifan Ye"/>
  <meta name="DC.Contributor" content="Aiping Wu"/>
  <meta name="DC.Contributor" content="Taijiao Jiang"/>
  <meta name="DC.Contributor" content="Yousong Peng"/>
  <meta name="article:published_time" content="2020-02-28"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Prediction of receptorome for human-infecting virome"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.27.967885v1"/>
  <meta name="citation_id" content="2020.02.27.967885v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967885v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967885v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967885v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.27.967885"/>
  <meta name="citation_num_pages" content="18"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.27.967885"/>
  <meta name="citation_author" content="Zheng Zhang"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan Provincial Key Laboratory of Medical Virology, Hunan University"/>
  <meta name="citation_author" content="Sifan Ye"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan Provincial Key Laboratory of Medical Virology, Hunan University"/>
  <meta name="citation_author" content="Aiping Wu"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College, Beijing 100005, Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Taijiao Jiang"/>
  <meta name="citation_author_institution" content="Center for Systems Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp;; Peking Union Medical College, Beijing 100005, Suzhou Institute of Systems Medicine"/>
  <meta name="citation_author" content="Yousong Peng"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan Provincial Key Laboratory of Medical Virology, Hunan University"/>
  <meta name="citation_author_email" content="pys2013@hnu.edu.cn"/>
  <meta name="citation_reference" content="Baranowski, E., Ruiz-Jarabo, C.M., and Domingo, E. (2001). Evolution of cell recognition by viruses. Science 292, 1102–1105."/>
  <meta name="citation_reference" content="Casasnovas, J.M. (2013). Virus-receptor interactions and receptor-mediated virus entry into host cells. In Structure and Physics of Viruses (Springer), pp. 441–466."/>
  <meta name="citation_reference" content="Chen, C., Liaw, A., and Breiman, L. (2004). Using random forest to learn imbalanced data. University of California, Berkeley 110, 24."/>
  <meta name="citation_reference" content="Csardi, G., and Nepusz, T. (2006). The igraph software package for complex network research. InterJournal, complex systems 1695, 1–9."/>
  <meta name="citation_reference" content="Dimitrov, D.S. (2004). Virus entry: molecular mechanisms and biomedical applications. Nature Reviews Microbiology 2, 109–122."/>
  <meta name="citation_reference" content="Free, R.B., Hazelwood, L.A., and Sibley, D.R. (2009). Identifying novel protein‐protein interactions using co‐immunoprecipitation and mass spectroscopy. Current protocols in neuroscience 46, 5.28. 21–25.28. 14."/>
  <meta name="citation_reference" content="Fu, L., Niu, B., Zhu, Z., Wu, S., and Li, W. (2012). CD-HIT: accelerated for clustering the next-generation sequencing data. Bioinformatics 28, 3150–3152."/>
  <meta name="citation_reference" content="Gupta, R., Jung, E., and Brunak, S. (2004). Prediction of N-glycosylation sites in human proteins."/>
  <meta name="citation_reference" content="Hoffmann, M., Kleine-Weber, H., Krueger, N., Mueller, M.A., Drosten, C., and Pöhlmann, S. (2020). The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. bioRxiv."/>
  <meta name="citation_reference" content="Lasso, G., Mayer, S.V., Winkelmann, E.R., Chu, T., Elliot, O., Patino-Galindo, J.A., Park, K., Rabadan, R., Honig, B., and Shapira, S.D. (2019). A structure-informed atlas of human-virus interactions. Cell 178, 1526–1541. e1516."/>
  <meta name="citation_reference" content="Li, F. (2015). Receptor recognition mechanisms of coronaviruses: a decade of structural studies. Journal of virology 89, 1954–1964."/>
  <meta name="citation_reference" content="Masson, P., Hulo, C., De Castro, E., Bitter, H., Gruenbaum, L., Essioux, L., Bougueleret, L., Xenarios, I., and Le Mercier, P. (2012). ViralZone: recent updates to the virus knowledge resource. Nucleic acids research 41, D579–D583."/>
  <meta name="citation_reference" content="Minor, P., Pipkin, P., Hockley, D., Schild, G., and Almond, J. (1984). Monoclonal antibodies which block cellular receptors of poliovirus. Virus research 1, 203–212."/>
  <meta name="citation_reference" content="Pedregosa, F., Varoquaux, G., Gramfort, A., Michel, V., Thirion, B., Grisel, O., Blondel, M., Prettenhofer, P., Weiss, R., and Dubourg, V. (2011). Scikit-learn: Machine learning in Python. Journal of machine learning research 12, 2825–2830."/>
  <meta name="citation_reference" content="Petryszak, R., Keays, M., Tang, Y.A., Fonseca, N.A., Barrera, E., Burdett, T., Füllgrabe, A., Fuentes, A.M.-P., Jupp, S., and Koskinen, S. (2016). Expression Atlas update—an integrated database of gene and protein expression in humans, animals and plants. Nucleic acids research 44, D746–D752."/>
  <meta name="citation_reference" content="Qi, F., Qian, S., Zhang, S., and Zhang, Z. (2020). Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses. bioRxiv."/>
  <meta name="citation_reference" content="Ryu, W.-S. (2016). Molecular virology of human pathogenic viruses (Academic Press)."/>
  <meta name="citation_reference" content="Szklarczyk, D., Franceschini, A., Wyder, S., Forslund, K., Heller, D., Huerta-Cepas, J., Simonovic, M., Roth, A., Santos, A., and Tsafou, K.P. (2015). STRING v10: protein–protein interaction networks, integrated over the tree of life. Nucleic acids research 43, D447–D452."/>
  <meta name="citation_reference" content="Wang, J.-h. (2002). Protein recognition by cell surface receptors: physiological receptors versus virus interactions. Trends in biochemical sciences 27, 122–126."/>
  <meta name="citation_reference" content="Wu, A., Peng, Y., Huang, B., Ding, X., Wang, X., Niu, P., Meng, J., Zhu, Z., Zhang, Z., and Wang, J. (2020). Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host &amp;; Microbe."/>
  <meta name="citation_reference" content="Yan, C., Duan, G., Wu, F.-X., and Wang, J. (2019). IILLS: predicting virus-receptor interactions based on similarity and semi-supervised learning. BMC bioinformatics 20, 651."/>
  <meta name="citation_reference" content="Zhang, H., Kang, Z., Gong, H., Xu, D., Wang, J., Li, Z., Cui, X., Xiao, J., Meng, T., and Zhou, W. (2020). The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes. BioRxiv."/>
  <meta name="citation_reference" content="Zhang, Z., Zhu, Z., Chen, W., Cai, Z., Xu, B., Tan, Z., Wu, A., Ge, X., Guo, X., and Tan, Z. (2019). Cell membrane proteins with high n-glycosylation, high expression and multiple interaction partners are preferred by mammalian viruses as receptors. Bioinformatics 35, 723–728."/>
  <meta name="citation_reference" content="Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W., Si, H.-R., Zhu, Y., Li, B., and Huang, C.-L. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 1–4."/>
  <meta name="twitter:title" content="Prediction of receptorome for human-infecting virome"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome."/>
  <meta name="og-title" property="og:title" content="Prediction of receptorome for human-infecting virome"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967885v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-28"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Prediction of receptorome for human-infecting virome | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1172358 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">The virus receptor is key for viral infection of host cells. Identification of the virus receptor is still challenging at present. Our previous study has shown that human virus receptor proteins have some unique features including high level of N-glycosylation, high number of interaction partners and high expressions. Here, we built a random-forest model to identify human virus receptorome from human cell membrane proteins with accepted accuracy based on combination of unique features of human virus receptors and protein sequences. A total of 729 human cell membrane proteins were predicted to constitute the receptorome for the human-infecting virome. Combination of the random-forest model with protein-protein interactions between human and viruses predicted in previous studies further predicted receptors for 693 human-infecting viruses, such as the Enterovirus, Norovirus and West Nile virus. Finally, we predicted the candidate alternative receptors for the 2019-nCoV. The study would greatly facilitate identification of receptors for human-infecting virome.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Receptor-binding is the first step for viral infection of host cells. Proteins are considered to be better receptors for viruses due to their higher binding affinity and specificity than carbohydrate and lipid (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baranowski et al., 2001</a>; 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Casasnovas, 2013</a>; 
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dimitrov, 2004</a>; 
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Li, 2015</a>; 
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Wang, 2002</a>). Lots of human virus receptors has been identified. The selection of proteins by viruses as receptors is not a random process. Previous studies have shown that proteins that were abundant in the host cell surface or had relatively low affinity for their natural ligands were preferred by viruses as receptors (
     <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Dimitrov, 2004</a>; 
     <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Wang, 2002</a>). Based on a collection of 119 mammalian virus receptors, our recent study further showed that human virus protein receptors have higher level of N-glycosylation, higher number of interaction partners in the human protein-protein interaction (PPI) network and high expressions in 32 common human tissues compared to other cell membrane proteins (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Zhang et al., 2019</a>). These studies could facilitate identification of human virus receptors.
    </p>
    <p id="p-5">Identifying receptors for viruses is challenging. Currently, several experimental approaches have been developed in identifying virus receptors. The first approach is to select candidate membrane proteins that bind to the virus receptor-binding proteins (RBPs) by affinity purification and mass spectroscopy (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Free et al., 2009</a>; 
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ryu, 2016</a>). The second approach is to select candidate membrane proteins to which the monoclonal antibodies specific could block the virus entry (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Minor et al., 1984</a>; 
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ryu, 2016</a>). The third approach is to identify the virus receptor by functional cloning selection based on the cDNA expression library (
     <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Ryu, 2016</a>). The protein which could enable viral infection of the non-susceptible cell after transfection would be considered as receptor candidate. However, identifying virus receptor is still time-consuming and difficult at present. Besides, limitations on scalability have hampered large-scale identification of viral receptors. The development of computational methods for identifying the human virus receptors is in great need.
    </p>
    <p id="p-6">Previous studies have developed computational models for predicting PPIs between viruses and hosts which could help identification of virus receptors (
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Lasso et al., 2019</a>; 
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Yan et al., 2019</a>). For example, Lasso et al. developed an in silico computational framework (P-HIPSTer) that employs structural information to predict more than 280,000 PPIs between 1,001 human-infecting viruses and humans, and made a series of new findings about human-virus interactions (
     <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Lasso et al., 2019</a>). Numerous PPIs between viral RBPs and human cell membrane proteins could be used to identify virus receptors. Here, we developed a computational model to predict the receptorome of the human-infecting virome based on the features of human virus receptors and protein sequences. Combination of the model with Lasso’s work further predicted receptors for 693 human-infecting viruses. The study would greatly facilitate identification of human virus receptors.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Human viral receptors, human cell membrane proteins and human membrane proteins</h3>
     <p id="p-7">A total of 91 human virus protein receptors were obtained from the viralReceptor database (available at 
      <a href="http://www.computationalbiology.cn:5000/viralReceptor">http://www.computationalbiology.cn:5000/viralReceptor</a>) developed in our previous study (
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Zhang et al., 2019</a>). Human cell membrane proteins and human membrane proteins were obtained from the UniProtKB/Swiss-Prot database on November 24
      <sup>th</sup>, 2017. The human proteins with the words of “cell membrane” and “cell surface” in the field of “Subcellular location” were considered to be human cell membrane proteins. The human proteins with the words of “membrane” in the field of “Subcellular location” were considered to be human membrane proteins. A total of 2,161 human cell membrane proteins and 5,187 human membrane proteins were obtained.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>The protein features and protein sequence features</h3>
     <p id="p-8">The N-glycosylation sites of human proteins were obtained from the UniprotKB/Swiss-Prot database. For those proteins without annotation of the N-glycosylation site in the UniprotKB/Swiss-Prot database, the N-glycosylation sites were predicted with NetNGlyc 1.0 (available at 
      <a href="http://www.cbs.dtu.dk/services/NetNGlyc/">http://www.cbs.dtu.dk/services/NetNGlyc/</a>) (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gupta et al., 2004</a>). The level of N-glycosylation for a protein was defined as the number of N-glycosylation sites per hundred amino acids.
     </p>
     <p id="p-9">To calculate the node degree of human proteins in the human PPI network, firstly, the human PPIs with combined scores greater than 400 were extracted from the STRING database (version 10.5) (
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Szklarczyk et al., 2015</a>) and formed the human PPI network. Then, the node degree was calculated with the functions of 
      <em>degree</em> in the R package igraph (version 1.2.4.2) (
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Csardi and Nepusz, 2006</a>).
     </p>
     <p id="p-10">The expression levels of human genes in 32 common human tissues were obtained from the Expression Atlas database (
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Petryszak et al., 2016</a>) on February 6
      <sup>th</sup>, 2018. Because there were strong correlations between the expressions of genes in different tissues, the principal component analysis (PCA) method was used to reduce the correlations with the function of 
      <em>PCA</em> in the package scikit-learn (version 0.21.3) (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Pedregosa et al., 2011</a>) in Python (version 3.6.7). Only the first principal component was used to measure the expression level of human genes, which explained 95% of the total variance.
     </p>
     <p id="p-11">The frequencies of k-mers with one or two amino acids were calculated for each human protein with a Python script.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Machine learning modeling</h3>
     <p id="p-12">To distinguish human virus receptors from other human cell membrane proteins with machine learning models, the human virus receptors were taken as positive samples, and the human membrane proteins were taken as negative samples after excluding the human cell membrane proteins since the human cell membrane proteins possibly contain virus receptors unidentified yet.</p>
     <p id="p-13">Because not all human proteins were observed in the human PPI network used or had observed expressions in the data used, only the human proteins with all three protein features, i.e., the N-glycosylation level, node degree and expression in common human tissues, were used in the modeling. Besides, the sequence redundancy in both human virus receptor proteins and human membrane proteins were removed using CD-HIT (version 4.8.1) (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fu et al., 2012</a>) at 70% identity level. Finally, a total of 88 human virus receptors and 1,593 human membrane proteins was used in the machine learning modeling.
     </p>
     <p id="p-14">The random-forest (RF) model is an ensemble machine learning technique using multiple decision trees and can handle data with high variance and high bias, while the risk of over-fitting can be significantly reduced by averaging multiple trees. Therefore, we chose the RF model to distinguish human virus receptors from other human cell membrane proteins. Because the number of positive samples, i.e., human viral receptors, was much smaller than that of negative samples, i.e., human membrane proteins, the function of BalanceRandomForest in the package imbalanced-learn (version 0.5.0) (
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen et al., 2004</a>) in Python was used to deal with the imbalanced positive and negative samples with the parameter of n_estimators set to be 100.
     </p>
     <p id="p-15">Five times of five-fold cross-validations were used to evaluate the predictive performances of the RF model with the function of StratifiedKFold in the package scikit-learn in Python. The predictive performances of the RF model were evaluated by the area under receiver operating characteristics curve (AUC), accuracy, sensitivity and specificity.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Validation of the RF model in ranking receptor candidates</h3>
     <p id="p-16">The predicted interactions between human-infecting viruses and human proteins were obtained from the database of P-HIPSTer (available at 
      <a href="http://phipster.org/">http://phipster.org/</a>) (
      <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Lasso et al., 2019</a>) on November 1
      <sup>st</sup>, 2019. A total of 9,395 pairs of interactions between viral RBPs and human cell membrane proteins with the likelihood ratio (LR) ≥100 were extracted for further analysis, which included 718 viral RBPs and 314 human cell membrane proteins. The RBPs of human-infecting viruses were compiled from three sources: the ViralZone database (
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Masson et al., 2012</a>), the UniprotKB database during which viral proteins annotated with GO terms “viral entry into host cell” or “virion attachment the host cell”, and the literatures related to viral RBPs. Viruses of the same viral family were supposed to use the same RBP. For example, all coronaviruses were supposed to use the spike protein as RBP.
     </p>
     <p id="p-17">To evaluate the ability of the RF model in helping identify virus receptors, 25 pairs of experimentally-validated interactions between viral RBPs and receptors, and the predicted PPIs between these viral RBPs and human cell membrane proteins, were extracted from the P-HIPSTer database. For each viral RBP, the predicted RBP-interacting human cell membrane proteins were ranked by either the LR provided in Lasso’s work, or the predicted score provided by the RF model. Then, the ranks of real receptors were analyzed. The rank percentage of the real receptor was calculated by dividing the rank by the number of RBP-interacting human cell membrane proteins.</p>
     <p id="p-18">When ranking the RBP-interacting proteins by the RF model, high sequence similarity between the RBP-interacting proteins and human proteins in the modeling (both the positive and negative samples) may over-estimate the ability of the RF model. To reduce such effect, for each pair of viral RBP and receptor, the predicted RBP-interacting human proteins were clustered with human proteins used in the modeling using CD-HIT at 50% identity level. All proteins which were clustered with RBP-interacting proteins were excluded in the modeling.</p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Results</h2>
    <div id="sec-8" class="subsection">
     <h3>Development of random-forest models for predicting receptorome of human-infecting virome</h3>
     <p id="p-19">Our previous studies have shown that human virus protein receptors have high level of N-glycosylation, high number of interaction partners in the human PPI network, and high expressions in 32 common human tissues (
      <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Zhang et al., 2019</a>). To identify the potential receptors of human-infecting virome, we firstly built a RF model to distinguish the human virus receptor proteins from other human membrane proteins based on these protein features. The RF model built based on individual protein feature achieved an AUC ranging from 0.50 to 0.66 in five-fold cross-validations (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Combination of all three features improved the RF model much, with both the AUC and prediction accuracy equaling to 0.75 (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1176868/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1176868/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-20" class="first-child">The predictive performances of RF models using different sets of features. N-gly, N-glycosylation; PPI, node degree in human PPI network; Expression, expressions in 32 human tissues. Acc, accuracy; Sen, Sensitivity; Spe, Specificity.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">For comparison, we also developed RF models to distinguish the human virus from other human membrane proteins based on protein sequences. The frequencies of k-mers with length of one and two amino acids were used as features in the modeling. The RF model based on combination of 20 amino-acid frequencies achieved an AUC 0.68. When using the frequencies of k-mers with two amino acids as features, only the top N (N=1 to 400) most important features were used in the modeling to reduce over-fitting. Figure S1 showed that the AUC of the RF models increased rapidly as increase of N from 1 to 10. Then, it increased slowly or even decreased when more ten features were used. Therefore, only top ten features were used in the modeling to balance the model performance and complexity. The RF model based on top ten features achieved an AUC of 0.72.</p>
     <p id="p-22">To further improve the model for predicting receptorome of human-infecting virome, we incorporated the protein features and the top ten most important k-mers of two amino acids in the modeling. The RF model achieved an AUC of 0.77. The prediction accuracy, sensitivity and specificity of the model equaled 0.80, 0.73 and 0.80, respectively (
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The model combining both the protein features and k-mers was used for further analysis.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Prediction of receptorome for human-infecting virome</h3>
     <p id="p-23">Based on the RF model, we predicted the receptorome from human cell membrane proteins. A score ranging from 0 to 1 was assigned to each human cell membrane protein. Proteins with high scores are more likely to be virus receptors. A total of 729 proteins with scores greater than 0.5 were considered to constitute the receptorome of human-infecting virome. All human cell membrane proteins and the relevant scores were provided in Table S1.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Prediction of virus-receptor interactions for human-infecting viruses</h3>
     <p id="p-24">Then, we further investigated the prediction of virus-receptor interactions. In previous study, Lasso et al. predicted 282,528 pairs of PPIs between human and 1,001 human-infecting viruses (
      <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Lasso et al., 2019</a>). Based on their study, 9,395 pairs of PPIs between 718 viral RBPs from 693 human-infecting viruses, and 314 human cell membrane proteins were extracted for further analysis (see Table S2). A viral RBP was predicted to interact with 1 to 65 human cell membrane proteins, with a median of 10. For each viral RBP, the RBP-interacting cell membrane proteins were ranked by the score provided by the RF model to select the most likely receptor (Table S2).
     </p>
     <p id="p-25">To validate the ranking by the RF model, 25 pairs of experimentally-validated interactions between viral RBPs and receptors were extracted. For each pair of these interactions, the rank of the real receptor among the predicted RBP-interacting proteins was obtained, and the related rank percentage was calculated (Materials and Methods). Seven real receptors were ranked in top one by the RF model (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Besides, nearly 70% (17/25) of real receptors were ranked in top three. On average, the real receptors had a rank percentage of 0.21 among all RBP-interacting human cell membrane proteins, suggesting that the real receptors would be ranked in the top 21% of all candidates by the RF model.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1176874/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1176874/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-26" class="first-child">The ranks of real virus receptors among the RBP-interacting human cell membrane proteins by LR and RF score. The median rank percentage of real virus receptors among RBP-interacting human cell membrane proteins, and the number of real virus receptors among top one, three and five ranks were summarized at the bottom. 
        <sup>#</sup> The number in brackets referred to those when only considering the 12 pairs of viral RBP-receptor interaction with LRs. * No LR was provided in Lasso’s work since there were resolved complex structures between the RBP and the receptor. MACV, Machupo mammarenavirus; HAdV-A, Human mastadenovirus A; HAdV-C, Human mastadenovirus C; HAdV-E, Human mastadenovirus E;HAdV-D, Human mastadenovirus D; HSV-1, Human alphaherpesvirus 1; HSV-2, Human alphaherpesvirus 2; HIV-1, Human immunodeficiency virus 1.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-27">For comparison, we also ranked the RBP-interacting proteins by the LR provided in Lasso’s work. In 12 pairs of viral RBP-receptor interaction with LRs, no real receptors were ranked in top one and only two real receptors were ranked in top three when ranking RBP-interacting human cell membrane proteins by the LR. On average, the median rank percentage of real receptors was 0.43 when ranking by the LR, while that was 0.15 by the RF model (
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Prediction of candidate alternative receptors for 2019-nCoV</h3>
     <p id="p-28">Previous studies have shown that the ACE2 protein which is the receptor of 2019-nCoV (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hoffmann et al., 2020</a>; 
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Zhou et al., 2020</a>) had a low level of expression in the lung and the upper respiratory tract (
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Qi et al., 2020</a>; 
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Zhang et al., 2020</a>). It is possible that 2019-nCoV has alternative receptors. We investigated the prediction of the alternative receptors for 2019-nCoV. Lasso’s study have predicted PPIs between 28 human cell membrane proteins which were members of receptorome of human-infecting viruses, and the spike protein of two coronaviruses, including Severe Acute Respiratory Syndrome-CoV and Middle East Respiratory Syndrome-CoV. We supposed that the 2019-nCoV is very likely to use these spike-interacting proteins as alternative receptors. These spike-interacting proteins were ranked by the score provided by the RF model (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The expression levels of these spike-interacting proteins in 32 common human tissues were shown in 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Most of them had high expressions than ACE2 in the lung, such as APP, EZR, CD4 and so on.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The predicted alternative receptors (in the left side) of 2019-nCoV and their expressions in 32 human tissues (in the bottom). The predicted alternative receptors were ranked by the RF score. The expression level was measured by transcripts per million (TPM) and was colored according to the legend in the top right. The white referred to no data available. The lung was highlighted by an arrow. The ACE2 was marked by an asterisk." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-76431381" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The predicted alternative receptors (in the left side) of 2019-nCoV and their expressions in 32 human tissues (in the bottom). The predicted alternative receptors were ranked by the RF score. The expression level was measured by transcripts per million (TPM) and was colored according to the legend in the top right. The white referred to no data available. The lung was highlighted by an arrow. The ACE2 was marked by an asterisk.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885/F1.medium.gif" width="440" height="278"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885/F1.medium.gif" width="440" height="278"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.27.967885/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-29" class="first-child">The predicted alternative receptors (in the left side) of 2019-nCoV and their expressions in 32 human tissues (in the bottom). The predicted alternative receptors were ranked by the RF score. The expression level was measured by transcripts per million (TPM) and was colored according to the legend in the top right. The white referred to no data available. The lung was highlighted by an arrow. The ACE2 was marked by an asterisk.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Discussion</h2>
    <p id="p-30">Identification of receptors for human-infecting viruses is critical for understanding the interactions between viruses and human. Our previous studies have shown that human virus receptors have some unique features compared to other cell membrane proteins: high level of N-glycosylation, high number of interaction partners and high expressions. This study further built a RF model for identifying human virus receptors from human cell membrane proteins with accepted accuracy. Based on the RF model, the receptorome for human-infecting virome were predicted which included a total of 729 human cell membrane proteins. They could facilitate identification of human virus receptors.</p>
    <p id="p-31">Previous studies by Lasso et al. developed computational models for predicting PPIs between human-infecting viruses and humans (
     <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Lasso et al., 2019</a>). A variable number of human cell membrane proteins were predicted to interact with viral RBPs. To further select the potential receptors for viruses, both the LR and the RF model could be used to rank the viral RBP-interacting human cell membrane proteins. The RF model was found to rank the real receptors better than the LR in a small validation dataset, suggesting that the RF model may perform better than the LR in selecting the real receptors from the predicted RBP-interacting human cell membrane proteins. Combination of the RF model and the RBP-interacting human cell membrane proteins predicted in Lasso’s work enables prediction of receptors for 693 human-infecting viruses (Table S2). Further efforts are needed to validate these predicted receptors.
    </p>
    <p id="p-32">The 2019-nCoV is responsible for the on-going outbreak of COVID-19 (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Wu et al., 2020</a>; 
     <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Zhou et al., 2020</a>). Previous studies have shown that the virus used ACE2 as the receptor (
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Hoffmann et al., 2020</a>; 
     <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Zhou et al., 2020</a>). However, several studies have found that ACE2 had low expressions in the lung and the upper respiratory tract (
     <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Qi et al., 2020</a>; 
     <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Zhang et al., 2020</a>). It is possibly that the 2019-nCoV may use co-receptor or alternative receptors to facilitate its infection. A previous study investigated the candidate co-receptors of 2019-nCoV, such as ANPEP, ENPEP and DPP4 (
     <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Qi et al., 2020</a>). We further supposed that the 2019-nCoV possibly have alternative receptors. Based on the predicted PPIs between coronaviruses and humans in previous work, we predicted several candidate alternative receptors for 2019-nCoV. Most candidate alternative receptors had higher expressions than ACE2 in the lung. Nevertheless, more efforts are needed to validate these candidate alternative receptors.
    </p>
    <p id="p-33">There are some limitations to this study. Firstly, the number of human virus receptor proteins was much less than that of human membrane proteins in the modeling, which may hinder accurate modeling. The under-sampling method was used to deal with the imbalance problem. Secondly, the performance of the RF model was modest in discriminating the human virus receptor proteins from human membrane proteins. More efforts are needed to improve the model. Thirdly, the RF model can not be used to identify receptors for a specific human-infecting virus. Instead, it predicts the receptorome for the human-infecting virome. Combining the RF model with the model of PPI predictions such as Lasso’s work can help identify virus-receptor interactions.</p>
    <p id="p-34">In conclusion, this study for the first time built a computational model for predicting the receptorome of the human-infecting virome. It could facilitate identification of human virus receptors in either computational or experimental studies.</p>
   </div>
   <div id="sec-14" class="subsection COI-statement">
    <p id="p-36">The authors have declared that no competing interests exist.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-35">This work was supported by the National Key Plan for Scientific Research and Development of China (2016YFD0500300), Hunan Provincial Natural Science Foundation of China (2018JJ3039), and the Chinese Academy of Medical Sciences (2016-I2M-1-005).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.1" data-doi="10.1126/science.1058613">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baranowski</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruiz-Jarabo</span>, 
          <span class="cit-name-given-names">C.M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> (
         <span class="cit-pub-date">2001</span>). 
         <span class="cit-article-title">Evolution of cell recognition by viruses</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">292</span>, 
         <span class="cit-fpage">1102</span>–
         <span class="cit-lpage">1105</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBaranowski%26rft.auinit1%253DE.%26rft.volume%253D292%26rft.issue%253D5519%26rft.spage%253D1102%26rft.epage%253D1105%26rft.atitle%253DEvolution%2Bof%2BCell%2BRecognition%2Bby%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1058613%26rft_id%253Dinfo%253Apmid%252F11352064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUxOS8xMTAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjcvMjAyMC4wMi4yNy45Njc4ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.27.967885v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Casasnovas</span>, 
          <span class="cit-name-given-names">J.M.</span>
         </span> (
         <span class="cit-pub-date">2013</span>). 
         <span class="cit-chapter-title">Virus-receptor interactions and receptor-mediated virus entry into host cells</span>. In 
         <span class="cit-source">Structure and Physics of Viruses</span> (
         <span class="cit-publ-name">Springer</span>), pp. 
         <span class="cit-fpage">441</span>–
         <span class="cit-lpage">466</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liaw</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Breiman</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">Using random forest to learn imbalanced data</span>. 
         <abbr class="cit-jnl-abbrev">University of California, Berkeley</abbr>
         <span class="cit-vol">110</span>, 
         <span class="cit-fpage">24</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DUniversity%2Bof%2BCalifornia%252C%2BBerkeley%26rft.volume%253D110%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.4" data-doi="10.3724/SP.J.1087.2009.02191">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Csardi</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Nepusz</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">2006</span>). 
         <span class="cit-article-title">The igraph software package for complex network research</span>. 
         <abbr class="cit-jnl-abbrev">InterJournal, complex systems</abbr>
         <span class="cit-vol">1695</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInterJournal%252C%2Bcomplex%2Bsystems%26rft.volume%253D1695%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3724%252FSP.J.1087.2009.02191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3724/SP.J.1087.2009.02191&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.5" data-doi="10.1038/nrmicro817">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dimitrov</span>, 
          <span class="cit-name-given-names">D.S.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">Virus entry: molecular mechanisms and biomedical applications</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>
         <span class="cit-vol">2</span>, 
         <span class="cit-fpage">109</span>–
         <span class="cit-lpage">122</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDimitrov%26rft.auinit1%253DD.%2BS.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D122%26rft.atitle%253DVirus%2Bentry%253A%2Bmolecular%2Bmechanisms%2Band%2Bbiomedical%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro817%26rft_id%253Dinfo%253Apmid%252F15043007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrmicro817&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15043007&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220431700014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Free</span>, 
          <span class="cit-name-given-names">R.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hazelwood</span>, 
          <span class="cit-name-given-names">L.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Sibley</span>, 
          <span class="cit-name-given-names">D.R.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">Identifying novel protein‐protein interactions using co‐immunoprecipitation and mass spectroscopy</span>. 
         <abbr class="cit-jnl-abbrev">Current protocols in neuroscience</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">5.28. 21</span>–
         <span class="cit-lpage">25.28. 14</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bprotocols%2Bin%2Bneuroscience%26rft.volume%253D46%26rft.spage%253D52821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.7" data-doi="10.1093/bioinformatics/bts565">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fu</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">CD-HIT: accelerated for clustering the next-generation sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">28</span>, 
         <span class="cit-fpage">3150</span>–
         <span class="cit-lpage">3152</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts565%26rft_id%253Dinfo%253Apmid%252F23060610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts565&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23060610&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000311902700023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Brunak</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <abbr class="cit-jnl-abbrev">Prediction of N-glycosylation sites in human proteins</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffmann</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kleine-Weber</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krueger</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mueller</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Pöhlmann</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lasso</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mayer</span>, 
          <span class="cit-name-given-names">S.V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Winkelmann</span>, 
          <span class="cit-name-given-names">E.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Elliot</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patino-Galindo</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rabadan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Honig</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Shapira</span>, 
          <span class="cit-name-given-names">S.D.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">A structure-informed atlas of human-virus interactions</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">178</span>, 
         <span class="cit-fpage">1526</span>–
         <span class="cit-lpage">1541</span>. e1516.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D178%26rft.spage%253D1526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.11" data-doi="10.1128/JVI.02615-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">Receptor recognition mechanisms of coronaviruses: a decade of structural studies</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">89</span>, 
         <span class="cit-fpage">1954</span>–
         <span class="cit-lpage">1964</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02615-14%26rft_id%253Dinfo%253Apmid%252F25428871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzQvMTk1NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzI3LzIwMjAuMDIuMjcuOTY3ODg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Masson</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hulo</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Castro</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bitter</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gruenbaum</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Essioux</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bougueleret</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xenarios</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Le Mercier</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">ViralZone: recent updates to the virus knowledge resource</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">41</span>, 
         <span class="cit-fpage">D579</span>–
         <span class="cit-lpage">D583</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D41%26rft.spage%253DD579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.13" data-doi="10.1016/0168-1702(84)90039-X">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Minor</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pipkin</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hockley</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schild</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Almond</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> (
         <span class="cit-pub-date">1984</span>). 
         <span class="cit-article-title">Monoclonal antibodies which block cellular receptors of poliovirus</span>. 
         <abbr class="cit-jnl-abbrev">Virus research</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">203</span>–
         <span class="cit-lpage">212</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DMinor%26rft.auinit1%253DP.%2BD.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D203%26rft.epage%253D212%26rft.atitle%253DMonoclonal%2Bantibodies%2Bwhich%2Bblock%2Bcellular%2Breceptors%2Bof%2Bpoliovirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-1702%252884%252990039-X%26rft_id%253Dinfo%253Apmid%252F6099657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0168-1702(84)90039-X&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=6099657&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1984TF93100002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pedregosa</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Varoquaux</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gramfort</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michel</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thirion</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grisel</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blondel</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Prettenhofer</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weiss</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Dubourg</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> (
         <span class="cit-pub-date">2011</span>). 
         <span class="cit-article-title">Scikit-learn: Machine learning in Python</span>. 
         <abbr class="cit-jnl-abbrev">Journal of machine learning research</abbr>
         <span class="cit-vol">12</span>, 
         <span class="cit-fpage">2825</span>–
         <span class="cit-lpage">2830</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmachine%2Blearning%2Bresearch%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.15" data-doi="10.1093/nar/gkv1045">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Petryszak</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Keays</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>, 
          <span class="cit-name-given-names">Y.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fonseca</span>, 
          <span class="cit-name-given-names">N.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barrera</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Burdett</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Füllgrabe</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fuentes</span>, 
          <span class="cit-name-given-names">A.M.-P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jupp</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Koskinen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-article-title">Expression Atlas update—an integrated database of gene and protein expression in humans, animals and plants</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">44</span>, 
         <span class="cit-fpage">D746</span>–
         <span class="cit-lpage">D752</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1045%26rft_id%253Dinfo%253Apmid%252F26481351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv1045&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26481351&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Qi</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qian</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Single cell RNA sequencing of 13 human tissues identify cell types and receptors of human coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.27.967885v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ryu</span>, 
          <span class="cit-name-given-names">W.-S.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-source">Molecular virology of human pathogenic viruses</span> (
         <span class="cit-publ-name">Academic Press</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.18" data-doi="10.1093/nar/gku1003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Szklarczyk</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Franceschini</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wyder</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Forslund</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heller</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huerta-Cepas</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simonovic</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roth</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Santos</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsafou</span>, 
          <span class="cit-name-given-names">K.P.</span>
         </span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">STRING v10: protein–protein interaction networks, integrated over the tree of life</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">D447</span>–
         <span class="cit-lpage">D452</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1003%26rft_id%253Dinfo%253Apmid%252F25352553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gku1003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25352553&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.19" data-doi="10.1016/S0968-0004(01)02038-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.-h.</span>
         </span> (
         <span class="cit-pub-date">2002</span>). 
         <span class="cit-article-title">Protein recognition by cell surface receptors: physiological receptors versus virus interactions</span>. 
         <abbr class="cit-jnl-abbrev">Trends in biochemical sciences</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">122</span>–
         <span class="cit-lpage">126</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D122%26rft.epage%253D126%26rft.atitle%253DProtein%2Brecognition%2Bby%2Bcell%2Bsurface%2Breceptors%253A%2Bphysiological%2Breceptors%2Bversus%2Bvirus%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252801%252902038-2%26rft_id%253Dinfo%253Apmid%252F11893508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0968-0004(01)02038-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11893508&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.27.967885.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000174416700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meng</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host &amp;; Microbe</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duan</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">F.-X.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">IILLS: predicting virus-receptor interactions based on similarity and semi-supervised learning</span>. 
         <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">651</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D20%26rft.spage%253D651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gong</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cui</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meng</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes</span>. 
         <abbr class="cit-jnl-abbrev">BioRxiv</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cai</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ge</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">Cell membrane proteins with high n-glycosylation, high expression and multiple interaction partners are preferred by mammalian viruses as receptors</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">35</span>, 
         <span class="cit-fpage">723</span>–
         <span class="cit-lpage">728</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.27.967885v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">X.-L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.-G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>, 
          <span class="cit-name-given-names">H.-R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.-L.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
