<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/05/17/325316.atom" />
<meta name="HW.pisa" content="biorxiv;325316v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation" />
<meta name="DC.Identifier" content="10.1101/325316" />
<meta name="DC.Date" content="2018-05-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Linear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from Kluyveromyces lactis as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that K2ORF3 encodes an mRNA cap guanine-N7-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N7-methylated caps. Translation of pGKL1,2 transcripts is enhanced in Ism1Δ and pab1Δ strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses." />
<meta name="DC.Contributor" content="Václav Vopálenský" />
<meta name="DC.Contributor" content="Michal Sýkora" />
<meta name="DC.Contributor" content="Tomáš Mašek" />
<meta name="DC.Contributor" content="Martin Pospíšek" />
<meta name="article:published_time" content="2018-05-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Linear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from &lt;i&gt;Kluyveromyces lactis&lt;/i&gt; as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that &lt;i&gt;K2ORF3&lt;/i&gt; encodes an mRNA cap guanine-N&lt;sup&gt;7&lt;/sup&gt;-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N&lt;sup&gt;7&lt;/sup&gt;-methylated caps. Translation of pGKL1,2 transcripts is enhanced in &lt;i&gt;Ism1Δ&lt;/i&gt; and &lt;i&gt;pab1Δ&lt;/i&gt; strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;325316v1" />
<meta name="citation_id" content="325316v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/325316v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/325316v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/325316v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316.full.pdf" />
<meta name="citation_doi" content="10.1101/325316" />
<meta name="citation_num_pages" content="70" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="325316" />
<meta name="citation_author" content="Václav Vopálenský" />
<meta name="citation_author_institution" content="Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague" />
<meta name="citation_author_email" content="martin.pospisek@natur.cuni.cz" />
<meta name="citation_author_email" content="vaclav.vopalensky@natur.cuni.cz" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9490-8911" />
<meta name="citation_author" content="Michal Sýkora" />
<meta name="citation_author_institution" content="Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague" />
<meta name="citation_author" content="Tomáš Mašek" />
<meta name="citation_author_institution" content="Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague" />
<meta name="citation_author" content="Martin Pospíšek" />
<meta name="citation_author_institution" content="Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague" />
<meta name="citation_author_email" content="martin.pospisek@natur.cuni.cz" />
<meta name="citation_author_email" content="vaclav.vopalensky@natur.cuni.cz" />
<meta name="citation_reference" content="Fukuhara, H. (1995) Linear DNA plasmids of yeasts. FEMS Microbiol Lett, 131, 1–9." />
<meta name="citation_reference" content="citation_title=The Mycota II, Genetics and biotechnology;citation_pages=199-217;citation_year=2004" />
<meta name="citation_reference" content="Gunge, N., Tamaru, A., Ozawa, F. and Sakaguchi, K. (1981) Isolation and Characterization of Linear Deoxyribonucleic Acid Plasmids from Kluyveromyces lactis and the Plasmid-Associated Killer Character. J Bacteriol, 145, 382–390." />
<meta name="citation_reference" content="Gunge, N. (1983) Yeast DNA plasmids. Annu Rev Microbiol, 37, 253–276." />
<meta name="citation_reference" content="Tommasino, M., Ricci, S. and Galeotti, C.L. (1988) Genome organization of the killer plasmid pGK12 from Kluyveromyces lactis. Nucleic Acids Res, 16, 5863–5878." />
<meta name="citation_reference" content="Hishinuma, F., Nakamura, K., Hirai, K., Nishizawa, R., Gunge, N. and Maeda, T. (1984) Cloning and nucleotide sequences of the linear DNA killer plasmids from yeast. Nucleic Acids Res, 12, 7581–7597." />
<meta name="citation_reference" content="Stam, J.C., Kwakman, J., Meijer, M. and Stuitje, A.R. (1986) Efficient isolation of the linear DNA killer plasmid of Kluyveromyces lactis: evidence for location and expression in the cytoplasm and characterization of their terminally bound proteins. Nucleic Acids Res, 14, 6871–6884." />
<meta name="citation_reference" content="Stark, M.J. and Boyd, A. (1986) The killer toxin of Kluyveromyces lactis: characterization of the toxin subunits and identification of the genes which encode them. Embo J, 5, 1995–2002." />
<meta name="citation_reference" content="Stark, M.J., Mileham, A.J., Romanos, M.A. and Boyd, A. (1984) Nucleotide sequence and transcription analysis of a linear DNA plasmid associated with the killer character of the yeast Kluyveromyces lactis. Nucleic Acids Res, 12, 6011–6030." />
<meta name="citation_reference" content="Tokunaga, M., Kawamura, A. and Hishinuma, F. (1989) Expression of pGKL killer 28K subunit in Saccharomyces cerevisiae: identification of 28K subunit as a killer protein. Nucleic Acids Res, 17, 3435–3446." />
<meta name="citation_reference" content="Tokunaga, M., Wada, N. and Hishinuma, F. (1987) Expression and identification of immunity determinants on linear DNA killer plasmids pGKL1 and pGKL2 in Kluyveromyces lactis. Nucleic Acids Res, 15, 1031–1046." />
<meta name="citation_reference" content="Fukuhara, H. (1987) The RF1 gene of the killer DNA of yeast may encode a DNA polymerase. Nucleic Acids Res, 15, 10046." />
<meta name="citation_reference" content="Jung, G.H., Leavitt, M.C. and Ito, J. (1987) Yeast killer plasmid pGKL1 encodes a DNA polymerase belonging to the family B DNA polymerases. Nucleic Acids Res, 15, 9088." />
<meta name="citation_reference" content="Schaffrath, R. and Meacock, P.A. (1995) Kluyveromyces lactis killer plasmid pGKL2: molecular analysis of an essential gene, ORF5. Yeast, 11, 615–628." />
<meta name="citation_reference" content="Takeda, M., Hiraishi, H., Takesako, T., Tanase, S. and Gunge, N. (1996) The terminal protein of the linear DNA plasmid pGKL2 shares an N-terminal domain of the plasmid-encoded DNA polymerase. Yeast, 12, 241–246." />
<meta name="citation_reference" content="Larsen, M., Gunge, N. and Meinhardt, F. (1998) Kluyveromyces lactis killer plasmid pGKL2: evidence for a viral-like capping enzyme encoded by ORF3. Plasmid, 40, 243–246." />
<meta name="citation_reference" content="Tiggemann, M., Jeske, S., Larsen, M. and Meinhardt, F. (2001) Kluyveromyces lactis cytoplasmic plasmid pGKL2: heterologous expression of Orf3p and proof of guanylyltransferase and mRNA-triphosphatase activities. Yeast, 18, 815–825." />
<meta name="citation_reference" content="Stark, M.J.R., Boyd, A., Mileham, A.J. and Romanos, M.A. (1990) The Plasmid-encoded Killer System of Kluyveromyces lactis: A Review. Yeast, 6, 1–29." />
<meta name="citation_reference" content="Schaffrath, R. and Meacock, P.A. (2001) An SSB encoded by and operating on linear killer plasmids from Kluyveromyces lactis. Yeast, 18, 1239–1247." />
<meta name="citation_reference" content="Tommasino, M. (1991) Killer System of Kluyveromyces lactis: the Open Reading Frame 10 of the pGK12 Plasmid Encodes a Putative DNA Binding Protein. Yeast, 7, 245–252." />
<meta name="citation_reference" content="McNeel, D.G. and Tamanoi, F. (1991) Terminal region recognition factor 1, a DNA-binding protein recognizing the inverted terminal repeats of the pGKl linear DNA plasmids. Proc Natl Acad Sci U S A, 88, 11398–11402." />
<meta name="citation_reference" content="citation_title=Progress in Botany;citation_author=S. Jeske;citation_author=F. Meinhardt;citation_author=R. Klassen;citation_pages=98-129;citation_volume=68;citation_year=2007;citation_doi=10.1007/978-3-540-36832-8_5" />
<meta name="citation_reference" content="Wilson, D.W. and Meacock, P.A. (1988) Extranuclear gene expression in yeast: evidence for a plasmid-encoded RNA polymerase of unique structure. Nucleic Acids Res, 16, 8097–8112." />
<meta name="citation_reference" content="Schaffrath, R., Meinhardt, F. and Meacock, P.A. (1997) ORF7 of yeast plasmid pGKL2: analysis of gene expression in vivo. Curr Genet, 31, 190–192." />
<meta name="citation_reference" content="Meinhardt, F., Wodara, C., Larsen, M. and Schickel, J. (1994) A Novel-Approach to Express a Heterologous Gene on Kluyveromyces-Lactis Linear Killer Plasmids - Expression of the Bacterial Aph Gene from a Cytoplasmic Promoter Fragment without in-Phase Fusion to the Plasmid Open Reading Frame. Plasmid, 32, 318–327." />
<meta name="citation_reference" content="Schrunder, J., Gunge, N. and Meinhardt, F. (1996) Extranuclear expression of the bacterial xylose isomerase (xylA) and the UDP-glucose dehydrogenase (hasB) genes in yeast with Kluyveromyces lactis linear killer plasmids as vectors. Curr Microbiol, 33, 323–330." />
<meta name="citation_reference" content="Schrunder, J. and Meinhardt, F. (1995) An extranuclear expression system for analysis of cytoplasmic promoters of yeast linear killer plasmids. Plasmid, 33, 139–151." />
<meta name="citation_reference" content="Huang, B., Johansson, M.J. and Bystrom, A.S. (2005) An early step in wobble uridine tRNA modification requires the Elongator complex. Rna, 11, 424–436." />
<meta name="citation_reference" content="Lu, J., Huang, B., Esberg, A., Johansson, M.J. and Bystrom, A.S. (2005) The Kluyveromyces lactis gamma-toxin targets tRNA anticodons. Rna, 11, 1648–1654." />
<meta name="citation_reference" content="Jablonowski, D., Zink, S., Mehlgarten, C., Daum, G. and Schaffrath, R. (2006) tRNAGlu wobble uridine methylation by Trm9 identifies Elongator’s key role for zymocin-induced cell death in yeast. Mol Microbiol, 59, 677–688." />
<meta name="citation_reference" content="Meineke, B., Kast, A., Schwer, B., Meinhardt, F., Shuman, S. and Klassen, R. (2012) A fungal anticodon nuclease ribotoxin exploits a secondary cleavage site to evade tRNA repair. Rna, 18, 1716–1724." />
<meta name="citation_reference" content="Butler, A.R., White, J.H. and Stark, M.J. (1991) Analysis of the response of Saccharomyces cerevisiae cells to Kluyveromyces lactis toxin. J Gen Microbiol, 137, 1749–1757." />
<meta name="citation_reference" content="Kast, A., Klassen, R. and Meinhardt, F. (2014) rRNA fragmentation induced by a yeast killer toxin. Mol Microbiol, 91, 606–617." />
<meta name="citation_reference" content="Furuichi, Y. and Shatkin, A.J. (2000) Viral and cellular mRNA capping: past and prospects. Adv Virus Res, 55, 135–184." />
<meta name="citation_reference" content="Shuman, S. (2001) Structure, mechanism, and evolution of the mRNA capping apparatus. Prog Nucleic Acid Res Mol Biol, 66, 1–40." />
<meta name="citation_reference" content="Shuman, S. (2002) What messenger RNA capping tells us about eukaryotic evolution. Nat Rev Mol Cell Biol, 3, 619–625." />
<meta name="citation_reference" content="Decroly, E., Ferron, F., Lescar, J. and Canard, B. (2012) Conventional and unconventional mechanisms for capping viral mRNA. Nat Rev Microbiol, 10, 51–65." />
<meta name="citation_reference" content="Gilbert, W.V., Zhou, K., Butler, T.K. and Doudna, J.A. (2007) Cap-independent translation is required for starvation-induced differentiation in yeast. Science, 317, 1224–1227." />
<meta name="citation_reference" content="Bergman, N., Moraes, K.C., Anderson, J.R., Zaric, B., Kambach, C., Schneider, R.J., Wilusz, C.J. and Wilusz, J. (2007) Lsm proteins bind and stabilize RNAs containing 5’ poly(A) tracts. Nat Struct Mol Biol, 14, 824–831." />
<meta name="citation_reference" content="citation_title=A Laboratory Guide to RNA: Isolation, Analysis, and Synthesis;citation_pages=43-50;citation_year=1996" />
<meta name="citation_reference" content="Masek, T., Vopalensky, V., Suchomelova, P. and Pospisek, M. (2005) Denaturing RNA electrophoresis in TAE agarose gels. Anal Biochem, 336, 46–50." />
<meta name="citation_reference" content="Gunge, N. and Yamane, C. (1984) Incompatibility of linear DNA killer plasmids pGKL1 and pGKL2 from Kluyveromyces lactis with mitochondrial DNA from Saccharomyces cerevisiae. J Bacteriol, 159, 533–539." />
<meta name="citation_reference" content="Pospisek, M. and Palkova, Z. (1991) Microisolation of Yeast Nucleic-Acids on the Microtitre Plate without Using Lytic Enzymes. Nucleic Acids Research, 19, 5083–5083." />
<meta name="citation_reference" content="Ghosh, A. and Lima, C.D. (2010) Enzymology of RNA cap synthesis. Wiley interdisciplinary reviews. RNA, 1, 152–172." />
<meta name="citation_reference" content="Sonenberg, N. and Hinnebusch, A.G. (2009) Regulation of translation initiation in eukaryotes: mechanisms and biological targets. Cell, 136, 731–745." />
<meta name="citation_reference" content="Both, G.W., Furuichi, Y., Muthukrishnan, S. and Shatkin, A.J. (1975) Ribosome binding to reovirus mRNA in protein synthesis requires 5’ terminal 7-methylguanosine. Cell, 6, 185–195." />
<meta name="citation_reference" content="Higman, M.A. and Niles, E.G. (1994) Location of the S-adenosyl-L-methionine binding region of the vaccinia virus mRNA (guanine-7-)methyltransferase. J Biol Chem, 269, 14982–14987." />
<meta name="citation_reference" content="Mao, X. and Shuman, S. (1994) Intrinsic RNA (guanine-7) methyltransferase activity of the vaccinia virus capping enzyme D1 subunit is stimulated by the D12 subunit. Identification of amino acid residues in the D1 protein required for subunit association and methyl group transfer. J Biol Chem, 269, 24472–24479." />
<meta name="citation_reference" content="Schwer, B., Hausmann, S., Schneider, S. and Shuman, S. (2006) Poxvirus mRNA cap methyltransferase. Bypass of the requirement for the stimulatory subunit by mutations in the catalytic subunit and evidence for intersubunit allostery. J Biol Chem, 281, 18953–18960." />
<meta name="citation_reference" content="Schwer, B. and Shuman, S. (2006) Genetic analysis of poxvirus mRNA cap methyltransferase: suppression of conditional mutations in the stimulatory D12 subunit by second-site mutations in the catalytic D1 subunit. Virology, 352, 145–156." />
<meta name="citation_reference" content="Kyrieleis, O.J., Chang, J., de la Pena, M., Shuman, S. and Cusack, S. (2014) Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus. Structure, 22, 452–465." />
<meta name="citation_reference" content="Luo, Y., Mao, X., Deng, L., Cong, P. and Shuman, S. (1995) The D1 and D12 subunits are both essential for the transcription termination factor activity of vaccinia virus capping enzyme. J Virol, 69, 3852–3856." />
<meta name="citation_reference" content="Yamada-Okabe, T., Doi, R., Shimmi, O., Arisawa, M. and Yamada-Okabe, H. (1998) Isolation and characterization of a human cDNA for mRNA 5’-capping enzyme. Nucleic Acids Res, 26, 1700–1706." />
<meta name="citation_reference" content="Saha, N., Schwer, B. and Shuman, S. (1999) Characterization of human, Schizosaccharomyces pombe, and Candida albicans mRNA cap methyltransferases and complete replacement of the yeast capping apparatus by mammalian enzymes. J Biol Chem, 274, 16553–16562." />
<meta name="citation_reference" content="Schwer, B., Lehman, K., Saha, N. and Shuman, S. (2001) Characterization of the mRNA capping apparatus of Candida albicans. J Biol Chem, 276, 1857–1864." />
<meta name="citation_reference" content="Yamada-Okabe, T., Mio, T., Matsui, M., Kashima, Y., Arisawa, M. and Yamada-Okabe, H. (1998) Isolation and characterization of the Candida albicans gene for mRNA 5’- triphosphatase: association of mRNA 5’-triphosphatase and mRNA 5’-guanylyltransferase activities is essential for the function of mRNA 5’-capping enzyme in vivo. FEBS Lett, 435, 49–54." />
<meta name="citation_reference" content="Pei, Y., Lehman, K., Tian, L. and Shuman, S. (2000) Characterization of Candida albicans RNA triphosphatase and mutational analysis of its active site. Nucleic Acids Res, 28, 1885–1892." />
<meta name="citation_reference" content="Hausmann, S., Vivares, C.P. and Shuman, S. (2002) Characterization of the mRNA capping apparatus of the microsporidian parasite Encephalitozoon cuniculi. J Biol Chem, 277, 96–103." />
<meta name="citation_reference" content="Takagi, T., Walker, A.K., Sawa, C., Diehn, F., Takase, Y., Blackwell, T.K. and Buratowski, S. (2003) The Caenorhabditis elegans mRNA 5’-capping enzyme. In vitro and in vivo characterization. J Biol Chem, 278, 14174–14184." />
<meta name="citation_reference" content="Srinivasan, P., Piano, F. and Shatkin, A.J. (2003) mRNA capping enzyme requirement for Caenorhabditis elegans viability. J Biol Chem, 278, 14168–14173." />
<meta name="citation_reference" content="Ho, C.K., Martins, A. and Shuman, S. (2000) A yeast-based genetic system for functional analysis of viral mRNA capping enzymes. J Virol, 74, 5486–5494." />
<meta name="citation_reference" content="Schwer, B. and Shuman, S. (1996) Multicopy suppressors of temperature-sensitive mutations of yeast mRNA capping enzyme. Gene Expr, 5, 331–344." />
<meta name="citation_reference" content="Schwer, B., Saha, N., Mao, X., Chen, H.W. and Shuman, S. (2000) Structure-function analysis of yeast mRNA cap methyltransferase and high-copy suppression of conditional mutants by AdoMet synthase and the ubiquitin conjugating enzyme Cdc34p. Genetics, 155, 1561–1576." />
<meta name="citation_reference" content="Saha, N., Shuman, S. and Schwer, B. (2003) Yeast-based genetic system for functional analysis of poxvirus mRNA cap methyltransferase. J Virol, 77, 7300–7307." />
<meta name="citation_reference" content="Chen, Y., Tao, J., Sun, Y., Wu, A., Su, C., Gao, G., Cai, H., Qiu, S., Wu, Y., Ahola, T. et al. (2013) Structure-function analysis of severe acute respiratory syndrome coronavirus RNA cap guanine-N7-methyltransferase. J Virol, 87, 6296–6305." />
<meta name="citation_reference" content="Chen, Y., Cai, H., Pan, J., Xiang, N., Tien, P., Ahola, T. and Guo, D. (2009) Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase. Proc Natl Acad Sci U S A, 106, 3484–3489." />
<meta name="citation_reference" content="Kast, A., Voges, R., Schroth, M., Schaffrath, R., Klassen, R. and Meinhardt, F. (2015) Autoselection of cytoplasmic yeast virus like elements encoding toxin/antitoxin systems involves a nuclear barrier for immunity gene expression. PLoS genetics, 11, e1005005." />
<meta name="citation_reference" content="Schaffrath, R., Meinhardt, F. and Meacock, P.A. (1999) Genetic manipulation of Kluyveromyces lactis linear DNA plasmids: gene targeting and plasmid shuffles. FEMS Microbiol Lett, 178, 201–210." />
<meta name="citation_reference" content="Schwer, B., Mao, X. and Shuman, S. (1998) Accelerated mRNA decay in conditional mutants of yeast mRNA capping enzyme. Nucleic Acids Res, 26, 2050–2057." />
<meta name="citation_reference" content="Davison, A.J. and Moss, B. (1989) Structure of vaccinia virus early promoters. J Mol Biol, 210, 749–769." />
<meta name="citation_reference" content="Schmidt, W.M. and Mueller, M.W. (1999) CapSelect: a highly sensitive method for 5’ CAP-dependent enrichment of full-length cDNA in PCR-mediated analysis of mRNAs. Nucleic Acids Res, 27, e31." />
<meta name="citation_reference" content="Ramanathan, A., Robb, G.B. and Chan, S.H. (2016) mRNA capping: biological functions and applications. Nucleic Acids Res, 44, 7511–7526." />
<meta name="citation_reference" content="Mukherjee, C., Patil, D.P., Kennedy, B.A., Bakthavachalu, B., Bundschuh, R. and Schoenberg, D.R. (2012) Identification of cytoplasmic capping targets reveals a role for cap homeostasis in translation and mRNA stability. Cell reports, 2, 674–684." />
<meta name="citation_reference" content="Ross, J. (1995) mRNA stability in mammalian cells. Microbiol Rev, 59, 423–450." />
<meta name="citation_reference" content="Ahn, B.Y. and Moss, B. (1989) Capped poly(A) leaders of variable lengths at the 5’ ends of vaccinia virus late mRNAs. J Virol, 63, 226–232." />
<meta name="citation_reference" content="Schwer, B., Visca, P., Vos, J.C. and Stunnenberg, H.G. (1987) Discontinuous transcription or RNA processing of vaccinia virus late messengers results in a 5’ poly(A) leader. Cell, 50, 163–169." />
<meta name="citation_reference" content="Patel, D.D. and Pickup, D.J. (1987) Messenger RNAs of a strongly-expressed late gene of cowpox virus contain 5’-terminal poly(A) sequences. Embo J, 6, 3787–3794." />
<meta name="citation_reference" content="Gowda, M., Li, H., Alessi, J., Chen, F., Pratt, R. and Wang, G.L. (2006) Robust analysis of 5’- transcript ends (5’-RATE): a novel technique for transcriptome analysis and genome annotation. Nucleic Acids Res, 34, e126." />
<meta name="citation_reference" content="Xiang, S., Cooper-Morgan, A., Jiao, X., Kiledjian, M., Manley, J.L. and Tong, L. (2009) Structure and function of the 5’-3’ exoribonuclease Rat1 and its activating partner Rai1. Nature, 458, 784–788." />
<meta name="citation_reference" content="Jiao, X., Xiang, S., Oh, C., Martin, C.E., Tong, L. and Kiledjian, M. (2010) Identification of a quality-control mechanism for mRNA 5’-end capping. Nature, 467, 608–611." />
<meta name="citation_reference" content="Song, M.G., Bail, S. and Kiledjian, M. (2013) Multiple Nudix family proteins possess mRNA decapping activity. Rna, 19, 390–399." />
<meta name="citation_reference" content="Wang, Z., Jiao, X., Carr-Schmid, A. and Kiledjian, M. (2002) The hDcp2 protein is a mammalian mRNA decapping enzyme. Proc Natl Acad Sci U S A, 99, 12663–12668." />
<meta name="citation_reference" content="citation_title=Microbial Linear Plasmids. Microbiology Monographs;citation_author=2007;citation_pages=187-226;citation_volume=7;citation_year=2007" />
<meta name="citation_reference" content="Schickel, J., Helmig, C. and Meinhardt, F. (1996) Kluyveromyces lactis killer system: analysis of cytoplasmic promoters of the linear plasmids. Nucleic Acids Res, 24, 1879–1886." />
<meta name="citation_reference" content="Gudkov, A.T., Ozerova, M.V., Shiryaev, V.M. and Spirin, A.S. (2005) 5’-poly(A) sequence as an effective leader for translation in eukaryotic cell-free systems. Biotechnology and bioengineering, 91, 468–473." />
<meta name="citation_reference" content="Muthukrishnan, S., Both, G.W., Furuichi, Y. and Shatkin, A.J. (1975) 5’-Terminal 7- methylguanosine in eukaryotic mRNA is required for translation. Nature, 255, 33–37." />
<meta name="citation_reference" content="Sor, F. and Fukuhara, H. (1985) Structure of a linear plasmid of the yeast Kluyveromyces lactis; Compact organization of the killer genome. Current Genetics, 9, 147–155." />
<meta name="citation_reference" content="Santiago, T.C., Bettany, A.J., Purvis, I.J. and Brown, A.J. (1987) Messenger RNA stability in Saccharomyces cerevisiae: the influence of translation and poly(A) tail length. Nucleic Acids Res, 15, 2417–2429." />
<meta name="citation_reference" content="Bernstein, P., Peltz, S.W. and Ross, J. (1989) The poly(A)-poly(A)-binding protein complex is a major determinant of mRNA stability in vitro. Mol Cell Biol, 9, 659–670." />
<meta name="citation_reference" content="Gallie, D.R. (1991) The cap and poly(A) tail function synergistically to regulate mRNA translational efficiency. Genes Dev, 5, 2108–2116." />
<meta name="citation_reference" content="Tanguay, R.L. and Gallie, D.R. (1996) Translational efficiency is regulated by the length of the 3’ untranslated region. Mol Cell Biol, 16, 146–156." />
<meta name="citation_reference" content="Munroe, D. and Jacobson, A. (1990) mRNA poly(A) tail, a 3’ enhancer of translational initiation. Mol Cell Biol, 10, 3441–3455." />
<meta name="citation_reference" content="Tarun, S.Z., Jr. and Sachs, A.B. (1995) A common function for mRNA 5’ and 3’ ends in translation initiation in yeast. Genes Dev, 9, 2997–3007." />
<meta name="citation_reference" content="Giaever, G., Chu, A.M., Ni, L., Connelly, C., Riles, L., Veronneau, S., Dow, S., Lucau-Danila, A., Anderson, K., Andre, B. et al. (2002) Functional profiling of the Saccharomyces cerevisiae genome. Nature, 418, 387–391." />
<meta name="citation_reference" content="Altmann, M., Handschin, C. and Trachsel, H. (1987) mRNA cap-binding protein: cloning of the gene encoding protein synthesis initiation factor eIF-4E from Saccharomyces cerevisiae. Mol Cell Biol, 7, 998–1003." />
<meta name="citation_reference" content="Brenner, C., Nakayama, N., Goebl, M., Tanaka, K., Toh-e, A. and Matsumoto, K. (1988) CDC33 encodes mRNA cap-binding protein eIF-4E of Saccharomyces cerevisiae. Mol Cell Biol, 8, 3556–3559." />
<meta name="citation_reference" content="Gunge, N. and Sakaguchi, K. (1981) Intergeneric transfer of deoxyribonucleic acid killer plasmids, pGKl1 and pGKl2, from Kluyveromyces lactis into Saccharomyces cerevisiae by cell fusion. J Bacteriol, 147, 155–160." />
<meta name="citation_reference" content="Wieczorek, Z., Niedzwiecka-Kornas, A., Chlebicka, L., Jankowska, M., Kiraga, K., Stepinski, J., Dadlez, M., Drabent, R., Darzynkiewicz, E. and Stolarski, R. (1999) Fluorescence studies on association of human translation initiation factor eIF4E with mRNA cap-analogues. Zeitschrift fur Naturforschung. C, Journal of biosciences, 54, 278–284." />
<meta name="citation_reference" content="Tamarkin-Ben-Harush, A., Vasseur, J.J., Debart, F., Ulitsky, I. and Dikstein, R. (2017) Cap-proximal nucleotides via differential eIF4E binding and alternative promoter usage mediate translational response to energy stress. eLife, 6, e21907." />
<meta name="citation_reference" content="Mokrejs, M., Masek, T., Vopalensky, V., Hlubucek, P., Delbos, P. and Pospisek, M. (2010) IRESite—a tool for the examination of viral and cellular internal ribosome entry sites. Nucleic Acids Res, 38, D131–D136." />
<meta name="citation_reference" content="Altmann, M. and Trachsel, H. (1989) Altered mRNA cap recognition activity of initiation factor 4E in the yeast cell cycle division mutant cdc33. Nucleic Acids Res, 17, 5923–5931." />
<meta name="citation_reference" content="Altmann, M., Sonenberg, N. and Trachsel, H. (1989) Translation in Saccharomyces cerevisiae: Initiation Factor 4E-Dependent Cell-Free System. Molecular and Cellular Biology, 9, 4467–4472." />
<meta name="citation_reference" content="Feketova, Z., Masek, T., Vopalensky, V. and Pospisek, M. (2010) Ambiguous decoding of the CUG codon alters the functionality of the Candida albicans translation initiation factor 4E. FEMS Yeast Res, 10, 558–569." />
<meta name="citation_reference" content="Sugisaki, Y., Gunge, N., Sakaguchi, K., Yamasaki, M. and Tamura, G. (1984) Characterization of a novel killer toxin encoded by a double-stranded linear DNA plasmid of Kluyveromyces lactis. Eur J Biochem, 141, 241–245." />
<meta name="citation_reference" content="Zakharov, J.A., Yurchenko, L.V. and Yarovot, B.F. (1969) Cytoduction: The autonomous transfer of cytoplasmic hereditary factors during pairing of yeast cells. Genetika, 5, 136–141." />
<meta name="citation_reference" content="Valis, K., Masek, T., Novotna, D., Pospisek, M. and Janderova, B. (2006) Immunity to killer toxin K1 is connected with the Golgi-to-vacuole protein degradation pathway. Folia Microbiol (Praha), 51, 196–202." />
<meta name="citation_reference" content="Palfree, R.G. and Bussey, H. (1979) Yeast killer toxin: purification and characterisation of the protein toxin from Saccharomyces cerevisiae. Eur J Biochem, 93, 487–493." />
<meta name="citation_reference" content="Partow, S., Siewers, V., Bjorn, S., Nielsen, J. and Maury, J. (2010) Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae. Yeast, 27, 955–964." />
<meta name="citation_reference" content="Shirokikh, N.E. and Spirin, A.S. (2008) Poly(A) leader of eukaryotic mRNA bypasses the dependence of translation on initiation factors. Proc Natl Acad Sci U S A, 105, 10738–10743." />
<meta name="citation_reference" content="Tahiri-Alaoui, A., Zhao, Y., Sadigh, Y., Popplestone, J., Kgosana, L., Smith, L.P. and Nair, V. (2014) Poly(A) binding protein 1 enhances cap-independent translation initiation of neurovirulence factor from avian herpesvirus. PloS one, 9, e114466." />
<meta name="citation_reference" content="Patel, G.P., Ma, S. and Bag, J. (2005) The autoregulatory translational control element of poly(A)-binding protein mRNA forms a heteromeric ribonucleoprotein complex. Nucleic Acids Res, 33, 7074–7089." />
<meta name="citation_reference" content="Melo, E.O., de Melo Neto, O.P. and Martins de Sa, C. (2003) Adenosine-rich elements present in the 5’-untranslated region of PABP mRNA can selectively reduce the abundance and translation of CAT mRNAs in vivo. FEBS Lett, 546, 329–334." />
<meta name="citation_reference" content="Xia, X., MacKay, V., Yao, X., Wu, J., Miura, F., Ito, T. and Morris, D.R. (2011) Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in Saccharomyces cerevisiae. Genetics, 189, 469–478." />
<meta name="citation_reference" content="Sachs, A.B., Bond, M.W. and Kornberg, R.D. (1986) A single gene from yeast for both nuclear and cytoplasmic polyadenylate-binding proteins: domain structure and expression. Cell, 45, 827–835." />
<meta name="citation_reference" content="Kahvejian, A., Svitkin, Y.V., Sukarieh, R., M’Boutchou, M.N. and Sonenberg, N. (2005) Mammalian poly(A)-binding protein is a eukaryotic translation initiation factor, which acts via multiple mechanisms. Genes Dev, 19, 104–113." />
<meta name="citation_reference" content="Costello, J., Castelli, L.M., Rowe, W., Kershaw, C.J., Talavera, D., Mohammad-Qureshi, S.S., Sims, P.F., Grant, C.M., Pavitt, G.D., Hubbard, S.J. et al. (2015) Global mRNA selection mechanisms for translation initiation. Genome Biol, 16, 10." />
<meta name="citation_reference" content="Smith, R.W. and Gray, N.K. (2010) Poly(A)-binding protein (PABP): a common viral target. Biochem J, 426, 1–12." />
<meta name="citation_reference" content="Kobayashi, M., Arias, C., Garabedian, A., Palmenberg, A.C. and Mohr, I. (2012) Site-specific cleavage of the host poly(A) binding protein by the encephalomyocarditis virus 3C proteinase stimulates viral replication. J Virol, 86, 10686–10694." />
<meta name="citation_reference" content="Mangus, D.A., Amrani, N. and Jacobson, A. (1998) Pbp1p, a factor interacting with Saccharomyces cerevisiae poly(A)-binding protein, regulates polyadenylation. Mol Cell Biol, 18, 7383–7396." />
<meta name="citation_reference" content="Sachs, A.B., Davis, R.W. and Kornberg, R.D. (1987) A single domain of yeast poly(A)-binding protein is necessary and sufficient for RNA binding and cell viability. Mol Cell Biol, 7, 3268–3276." />
<meta name="citation_reference" content="Guldener, U., Heck, S., Fielder, T., Beinhauer, J. and Hegemann, J.H. (1996) A new efficient gene disruption cassette for repeated use in budding yeast. Nucleic Acids Res, 24, 2519–2524." />
<meta name="citation_reference" content="Mayes, A.E., Verdone, L., Legrain, P. and Beggs, J.D. (1999) Characterization of Sm-like proteins in yeast and their association with U6 snRNA. Embo J, 18, 4321–4331." />
<meta name="citation_reference" content="Yoshikawa, K., Tanaka, T., Ida, Y., Furusawa, C., Hirasawa, T. and Shimizu, H. (2011) Comprehensive phenotypic analysis of single-gene deletion and overexpression strains of Saccharomyces cerevisiae. Yeast, 28, 349–361." />
<meta name="citation_reference" content="Jungfleisch, J., Blasco-Moreno, B. and Diez, J. (2016) Use of Cellular Decapping Activators by Positive-Strand RNA Viruses. Viruses, 8, 340." />
<meta name="citation_reference" content="Parker, R. (2012) RNA degradation in Saccharomyces cerevisae. Genetics, 191, 671–702." />
<meta name="citation_reference" content="Mulder, J., Robertson, M.E., Seamons, R.A. and Belsham, G.J. (1998) Vaccinia virus protein synthesis has a low requirement for the intact translation initiation factor eIF4F, the cap-binding complex, within infected cells. J Virol, 72, 8813–8819." />
<meta name="citation_reference" content="Ostrowski, L.A., Hall, A.C. and Mekhail, K. (2017) Ataxin-2: From RNA Control to Human Health and Disease. Genes (Basel), 8, 157." />
<meta name="citation_reference" content="Mangus, D.A., Smith, M.M., McSweeney, J.M. and Jacobson, A. (2004) Identification of factors regulating poly(A) tail synthesis and maturation. Mol Cell Biol, 24, 4196–4206." />
<meta name="citation_reference" content="Spohner, S.C., Schaum, V., Quitmann, H. and Czermak, P. (2016) Kluyveromyces lactis: An emerging tool in biotechnology. J Biotechnol, 222, 104–116." />
<meta name="citation_reference" content="Coleman, T.M., Wang, G. and Huang, F. (2004) Superior 5’ homogeneity of RNA from ATP-initiated transcription under the T7 phi 2.5 promoter. Nucleic Acids Res, 32, e14." />
<meta name="citation_reference" content="Altmann, M., Sonenberg, N., and Trachsel, H. (1989). Translation in Saccharomyces cerevisiae: Initiation Factor 4E-Dependent Cell-Free System. Molecular and Cellular Biology 9, 4467–4472." />
<meta name="citation_reference" content="Altmann, M., and Trachsel, H. (1989). Altered mRNA cap recognition activity of initiation factor 4E in the yeast cell cycle division mutant cdc33. Nucleic Acids Res 17, 5923–5931." />
<meta name="citation_reference" content="Coleman, T.M., Wang, G., and Huang, F. (2004). Superior 5’ homogeneity of RNA from ATP-initiated transcription under the T7 phi 2.5 promoter. Nucleic Acids Res 32, e14." />
<meta name="citation_reference" content="Gietz, R.D., and Woods, R.A. (2002). Transformation of yeast by lithium acetate/single-stranded carrier DNA/polyethylene glycol method. Methods Enzymol 350, 87–96." />
<meta name="citation_reference" content="Gueldener, U., Heinisch, J., Koehler, G.J., Voss, D., and Hegemann, J.H. (2002). A second set of loxP marker cassettes for Cre-mediated multiple gene knockouts in budding yeast. Nucleic Acids Res 30, e23." />
<meta name="citation_reference" content="Guldener, U., Heck, S., Fielder, T., Beinhauer, J., and Hegemann, J.H. (1996). A new efficient gene disruption cassette for repeated use in budding yeast. Nucleic Acids Res 24, 2519–2524." />
<meta name="citation_reference" content="Gunge, N., Tamaru, A., Ozawa, F., and Sakaguchi, K. (1981). Isolation and Characterization of Linear Deoxyribonucleic Acid Plasmids from Kluyveromyces lactis and the Plasmid-Associated Killer Character. Journal of Bacteriology 145, 382–390." />
<meta name="citation_reference" content="Gunge, N., and Yamane, C. (1984). Incompatibility of linear DNA killer plasmids pGKL1 and pGKL2 from Kluyveromyces lactis with mitochondrial DNA from Saccharomyces cerevisiae. J Bacteriol 159, 533–539." />
<meta name="citation_reference" content="Piccirillo, C., Khanna, R., and Kiledjian, M. (2003). Functional characterization of the mammalian mRNA decapping enzyme hDcp2. Rna 9, 1138–1147." />
<meta name="citation_reference" content="Schwer, B., Saha, N., Mao, X., Chen, H.W., and Shuman, S. (2000). Structure-Function Analysis of Yeast mRNA Cap Methyltransferase and High-Copy Suppression of Conditional Mutants by AdoMet Synthase and the Ubiquitin Conjugating Enzyme Cdc34p. Genetics 155, 1561–1576." />
<meta name="citation_reference" content="Schwer, B., and Shuman, S. (1996). Multicopy suppressors of temperature-sensitive mutations of yeast mRNA capping enzyme. Gene Expr 5, 331–344." />
<meta name="citation_reference" content="Valis, K., Masek, T., Novotna, D., Pospisek, M., and Janderova, B. (2006). Immunity to killer toxin K1 is connected with the Golgi-to-vacuole protein degradation pathway. Folia Microbiol (Praha) 51, 196–202." />
<meta name="citation_reference" content="Wang, Z., Jiao, X., Carr-Schmid, A., and Kiledjian, M. (2002). The hDcp2 protein is a mammalian mRNA decapping enzyme. Proc Natl Acad Sci U S A 99, 12663–12668." />
<meta name="citation_reference" content="Woods, D.R., and Bevan, E.A. (1968). Studies on Nature of Killer Factor Produced by Saccharomyces Cerevisiae. Journal of General Microbiology 51, 115–126." />
<meta name="citation_reference" content="Xiang, S., Cooper-Morgan, A., Jiao, X., Kiledjian, M., Manley, J.L., and Tong, L. (2009a). Structure and function of the 5’→3’ exoribonuclease Rat1 and its activating partner Rai1. Nature 458, 784–788." />
<meta name="citation_reference" content="Xiang, S., Cooper-Morgan, A., Jiao, X., Kiledjian, M., Manley, J.L., and Tong, L. (2009b). Structure and function of the 5’→3’ exoribonuclease Rati and its activating partner Rail. Nature 458, 784–788." />
<meta name="twitter:title" content="Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Linear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from Kluyveromyces lactis as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that K2ORF3 encodes an mRNA cap guanine-N7-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N7-methylated caps. Translation of pGKL1,2 transcripts is enhanced in Ism1Δ and pab1Δ strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses." />
<meta name="og-title" property="og:title" content="Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/325316v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Linear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from Kluyveromyces lactis as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that K2ORF3 encodes an mRNA cap guanine-N7-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N7-methylated caps. Translation of pGKL1,2 transcripts is enhanced in Ism1Δ and pab1Δ strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-05-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/325316v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/325316v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/325316v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/325316v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/99373" />
    <title>Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"a31ox9VqRYNAj2W_p5__uHITVQblu19QdzJ--NuqgOM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"99373","apath":"\/biorxiv\/early\/2018\/05\/17\/325316.atom","pisa":"biorxiv;325316v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;325316v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/325316v1","title":"Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5\u2019 and 3\u2019 UTRs and suggest a novel mechanism of translation","identifier":"node\/99373"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tkbF9QmGMCwmYukvtiVziOHJHsTWLcS4rL7V5AVlTMGhfIz3un6sC\\\\\\\/6ekVm6QR9h88e4JNtZ\\\\\\\/mDo2UQjSV6M+lNspyGd88JAFkx9bVsoZZ1A\\\\\\\/2xldh+JyU7reuvuf\\\\\\\/kFXDuRLHezJzg6UNQCh7FdSTcoH+CeNnFujFkCNG4meD8NWjONsyCb5AcQ3hioRTCSU7RJpsc3gwuRrdmmNps2X9E2nYu3KILRgJfMSkURTpvQf89G9SuicTRqFd6N3t+Ul0RShp\\\\\\\/CMHJ94Fe8AUvZEyP9ahu5Wb8oDnMqzWG6NlJz5h7TdGTnCZNkrExice5kRQrE9+deWYSvwvx1Ih52xYTe4ikRwZ1\\\\\\\/SAdzBJe8Lw0l8+\\\\\\\/aor+BilYDHXIee7iOWAmZjU0quOIccCtnkQXT9k7FUyz7DqM4srVrz0XTPlBQYkeaUhBawzcUNJ1ppAgG7uJxyWO1t6CPwqLgPD3HT97bGwAKKd0eRvv\\\\\\\/n5XvkoHDfTZgBSzg34QItSTeB4M26vxsaZlgZZOqp4AwSuiSLeNe\\\\\\\/PL6sjJfwY3qpvH6CVzNyYN7TZPG9Q2oN8Dmoxssf6\\\\\\\/+HL9MD16hYHuCk04pPvxyZYGzdcaqOIo9oC3If3Mb3jMTJ\\\\\\\/TVrn0hew9VOWEdJoaaBNZqo8uZ9Zzrlm0ccKO6PcR46z\\\\\\\/IUnHCuVMaZH5PPAkIoHNDL9wYdlGZb9fKlCqOM20qpuZPxdnilEEpMEzdQFOFBzYZwmQ7YtImN2a1XC3PjuDxXSVfohY4Wxqagxqd9b4rm2k7zGK1pG6zXTOSEYAJBK9DJm5MwkDfZLio5Ty40AGq7zusD4d8w4GwJGLHnc9AqKmRfpEJgMmlWYWzkpVpPReEklJASb7o02\\\\\\\/aBC9J\\\\\\\/4zYpK01YNe11KW91UpvZA6C6poTcJzVselRJTlWYeTuSL4LGujzVB0o+nO7RTAhA0OY6g9XOJxEBr\\\\\\\/w5IP9S0D+rqLVr\\\\\\\/7i6wH3RoYMlLgIC0awiWOOKHVWlkhSXQPvuINeg+L75W3sX0\\\\\\\/Cn5wCWfVRaObD4thpqrF\\\\\\\/r6wpB9Bn9AkCp\\\\\\\/iHVQtr41piLSDcplYPy5RkcJnWR+09qqjTYafkc1Q8sVFPWV7obPZT83cVxiPRRzVyqZzGo\\\\\\\/t1pGb4d0VMT4WyjPUGLsTgEOhlvuN\\\\\\\/yz1gWTYHBX1XXX\\\\\\\/x7hG9GSpolQ3aBlgR+gILCk+S8eQQi8lgoEn6TEkIMTVdeSzCZ97ToTTSVBet6MKG8c\\\\\\\/BJJ23m6w3qRhGSz0Nigi+mbK4I7JBmacZDjJf85zK\\\\\\\/8\\\\\\\/GZ0kn9Kh7eo7qDfgf+cZB0lOlQGi5ngDs9tI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022SpUOSKOrWbbGBpoK1HxciA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229f58b5e5d2b96fe214241788aeaaa6a9045d9fb36f8152742d97862c8d2602ea\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/325316v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-99373 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/325316v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6Alm-kAP9FF95XLj37UQdNMITwkr8pFh4YZpO_wQnk8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="99373" id="node99373" data-pisa="biorxiv;325316v1" data-pisa-master="biorxiv;325316" data-apath="/biorxiv/early/2018/05/17/325316.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9490-8911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Václav</span> <span class="nlm-surname">Vopálenský</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Sýkora</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Tomáš</span> <span class="nlm-surname">Mašek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Pospíšek</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/325316 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node99373" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Václav Vopálenský </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague</span>, Vinicna 5, Prague 2, 128 44, <span class='nlm-country'>Czech Republic</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%C3%A1clav%2BVop%C3%A1lensk%C3%BD%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vop%C3%A1lensk%C3%BD%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AV%25C3%25A1clav%2BVop%25C3%25A1lensk%25C3%25BD%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9490-8911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Václav Vopálenský</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.pospisek@natur.cuni.cz" class="" data-icon-position="" data-hide-link-title="0">martin.pospisek@natur.cuni.cz</a>
<a href="mailto:vaclav.vopalensky@natur.cuni.cz" class="" data-icon-position="" data-hide-link-title="0">vaclav.vopalensky@natur.cuni.cz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michal Sýkora </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague</span>, Vinicna 5, Prague 2, 128 44, <span class='nlm-country'>Czech Republic</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BS%C3%BDkora%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=S%C3%BDkora%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BS%25C3%25BDkora%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tomáš Mašek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague</span>, Vinicna 5, Prague 2, 128 44, <span class='nlm-country'>Czech Republic</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tom%C3%A1%C5%A1%2BMa%C5%A1ek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%C5%A1ek%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATom%25C3%25A1%25C5%25A1%2BMa%25C5%25A1ek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Pospíšek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of RNA Biochemistry, Department of Genetics and Microbiology, Faculty of Science, Charles University in Prague</span>, Vinicna 5, Prague 2, 128 44, <span class='nlm-country'>Czech Republic</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BPosp%C3%AD%C5%A1ek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Posp%C3%AD%C5%A1ek%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BPosp%25C3%25AD%25C5%25A1ek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.pospisek@natur.cuni.cz" class="" data-icon-position="" data-hide-link-title="0">martin.pospisek@natur.cuni.cz</a>
<a href="mailto:vaclav.vopalensky@natur.cuni.cz" class="" data-icon-position="" data-hide-link-title="0">vaclav.vopalensky@natur.cuni.cz</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/325316v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:99373" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:99373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/325316v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:99373" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:99373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/325316v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:99373" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:99373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/325316v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:99373" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:99373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/325316v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:99373" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:99373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Linear plasmids with almost identical compact genetic organization have been found in the cytoplasm of yeast species from nine genera. We employed pGKL1,2 plasmids from <em>Kluyveromyces lactis</em> as a model to investigate the previously unstudied transcriptome of yeast cytoplasmic linear plasmids. We performed 5’ and 3’ RACE analysis of all the pGKL1,2 mRNAs and found them not 3’ polyadenylated and containing mostly uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is controlled by the corresponding promoter regions. We refined the description of the pGKL1,2 promoters and found new alternative promoters of several genes. We also provide evidence that <em>K2ORF3</em> encodes an mRNA cap guanine-N<sup>7</sup>-methyltransferase and that 5’ capped pGKL1,2 transcripts contain N<sup>7</sup>-methylated caps. Translation of pGKL1,2 transcripts is enhanced in <em>Ism1Δ</em> and <em>pab1Δ</em> strains and is independent of eIF4E and Pab1 translation factors. We suggested a model of a primitive regulation of pGKL1,2 plasmids gene expression where degree of 5’ mRNA capping, degree of 5’ non-template polyadenylation and presence of negative regulators as PAB1 and Lsm1 play an important role. Our data also suggest a close relationship between linear plasmids and poxviruses.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Linear plasmids have been found in the cytoplasm of yeast species from nine genera, including <em>Kluyveromyces</em>, <em>Debaryomyces</em>, <em>Saccharomyces</em>, <em>Saccharomycopsis</em>, <em>Wingea</em>, and <em>Pichia</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). The overall genetic organization of all yeast cytoplasmic linear plasmids is identical with that of the most studied plasmids, pGKL1 (also termed K1) and pGKL2 (also termed K2), from the yeast <em>Kluyveromyces lactis</em>, where their presence is associated with the killer phenotype (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Killer strains contain 50 to 100 copies of each of the two linear plasmids per cell (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). The pGKL1 and pGKL2 plasmids have compact and extremely AT-rich genomes sized 8874 and 13447 bp, respectively, and carry terminal inverted repeats with proteins covalently attached to their 5’ ends (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>).</p><p id="p-4">The smaller plasmid, pGKL1, contains four open reading frames (ORFs): two of them (<em>K1ORF2</em> and <em>K1ORF4</em>) encode precursors of killer toxin subunits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>); <em>K1ORF3</em> is involved in an immunity phenotype in an unknown manner (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), and <em>K1ORF1</em> codes for the pGKL1 plasmid-specific DNA polymerase and a terminal protein (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Eleven ORFs have been reported in the larger pGKL2 plasmid, which provides vital functions for pGKL1/pGKL2 maintenance in host cell (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Some functions have been attributed to more than half of the proteins encoded by the ORFs carried by the pGKL2 plasmid. <em>K2ORF2</em> codes for a pGKL2-specific DNA polymerase and a terminal protein (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). <em>K2ORF3</em> codes for a virus-like mRNA capping enzyme. A purified K2Orf3 protein (K2Orf3p) displayed 5’- triphosphatase (TPase) and RNA-guanylyltransferase (GTase) activities; however the predicted RNA cap (guanine-N<sup>7</sup>)-methyltransferase (MTase) activity, which is the third crucial activity of conventional mRNA capping complexes, was not detected (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). <em>K2ORF4</em> encodes a putative helicase probably involved in transcription of plasmid-specific mRNAs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>), <em>K2ORF5</em> codes for a single-stranded DNA-binding protein, and <em>K2ORF10</em> encodes a protein bound to the terminal inverted repeats of both pGKL plasmids. All these proteins, with the exception of K2Orf3p and K2Orf4p, are probably involved in pGKL1/2 replication (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). <em>K2ORF6</em> and <em>K2ORF7</em> code for putative subunits of a pGKL1/2-specific RNA polymerase (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). No function has been assigned to the four remaining ORFs (<em>K2ORF1, 8, 9</em>, and <em>11</em>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). All ORFs are likely expressed independently. Each gene is preceded by an upstream conserved sequence (UCS) motif, which is located approximately −30 nucleotides from the putative start codon for pGKL1 UCSs (AT<sup>A</sup>/<sub>C</sub>TGA) or up to −100 nucleotides in the case of pGKL2 UCSs (ATNTGA) (for review see (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>)). Sequences located between an AUG initiation codon and its respective UCS (inclusive) act as promoters in <em>sensu lato</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>–<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). However, detailed mapping of the promoters recognized by the plasmid-specific RNA polymerase has not been performed.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic organization of pGKL plasmids. Known gene functions are indicated. Shades of gray indicate an extent of transcript capping, from black ORFs, with almost all capped transcripts, to white ORFs, with mostly uncapped transcripts. TIR - terminal inverted repeat; &#x2022; - terminal protein; SSB - Single-Stranded Binding protein; TRF1 - Terminal Recognition Factor 1; ? - unknown protein function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic organization of pGKL plasmids. Known gene functions are indicated. Shades of gray indicate an extent of transcript capping, from black ORFs, with almost all capped transcripts, to white ORFs, with mostly uncapped transcripts. TIR - terminal inverted repeat; &#x2022; - terminal protein; SSB - Single-Stranded Binding protein; TRF1 - Terminal Recognition Factor 1; ? - unknown protein function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F1.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F1.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Genetic organization of pGKL plasmids. Known gene functions are indicated. Shades of gray indicate an extent of transcript capping, from black ORFs, with almost all capped transcripts, to white ORFs, with mostly uncapped transcripts. TIR - terminal inverted repeat; • - terminal protein; SSB - Single-Stranded Binding protein; TRF1 - Terminal Recognition Factor 1; ? - unknown protein function.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">In summary, the yeast linear pGKL plasmids represent a model and the most studied example of a unique world of the yeast cytoplasmically localized linear double-stranded DNA plasmids. They code for their own replication and transcription and perhaps also an RNA-modification apparatus, making them remarkably independent of their host cells and evoking thoughts about their similarity to other cytoplasmically localized linear DNA genomes such as viruses belonging to the families <em>Poxviridae</em> or <em>Asfarviridae</em>. These plasmids are therefore also called dsDNA virus-like elements (VLEs) (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>).</p><p id="p-7">The mechanism of action of the <em>K. lactis</em> heterotrimeric killer toxin is well understood. A γ-subunit of the toxin inhibits the growth of sensitive yeast cells by cleaving their tRNAs at the 3’ side of the modified wobble nucleoside 5-methoxycarbonylmethyl-2-thiouridine (mcm5s2U), which leads to the cell cycle arrest in G1 phase (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Readers are referred to references by Kast et al. and Meineke et al. for more information concerning action of the <em>K. lactis</em> toxin and related ribotoxins (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p><p id="p-8">The translation of eukaryotic mRNAs is almost exclusively initiated by the binding of translation initiation factors to the 5’ cap structure. Conventional m<sup>7</sup>G mRNA cap formation requires three principal enzymatically catalyzed steps. During the first step, pyrophosphate is removed from the triphosphate end of the messenger RNA by TPase, GTase then transfers GMP from a GTP molecule to the new diphosphate end, and a methyl moiety is finally transferred from S-adenosyl-L-methionine (SAM) by MTase to the N<sup>7</sup> position of the terminal guanosine. Conventional m<sup>7</sup>G-cap synthesis has been reviewed in (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Whereas enzymatic activities required for canonical m<sup>7</sup>G-cap synthesis are highly conserved among eukaryotes and viruses, the organization of the capping apparatus itself is not conserved (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). The structural and domain organization of the capping apparatus is very diverse, particularly in viruses (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>).</p><p id="p-9">In this work, we focus on the yet uncharacterized transcriptome of the yeast cytoplasmic linear dsDNA plasmids and the possible mechanism of translation initiation used by the plasmid-specific mRNAs. We demonstrate that pGKL mRNAs are not 3’ polyadenylated and that most of them even lack the m<sup>7</sup>G cap at their 5’ ends. Surprisingly, mRNAs transcribed from pGKL plasmids contain poly(A) leaders of variable lengths at their 5’ ends that are not complementary to the plasmid DNA sequence and must thus be synthesized in a template-independent manner. We show that plasmid promoters directly determine the formation of the mRNA 5’ end leaders and their capping, pointing to a possible role of the plasmid RNA polymerase in this unusual phenomenon. We also provide the first evidence that MTase is an essential part of the K2Orf3p enzyme and that capped pGKL mRNAs are methylated at the N<sup>7</sup> position of the terminal guanosine moiety. Using <em>in vitro</em> and <em>in vivo</em> approaches, we demonstrate that translation of plasmid-specific mRNAs is independent of major cellular cap-binding proteins, translation initiation factor 4E (eIF4E), and the Pab1 and Lsm1 proteins that were speculated to be possible translational enhancers and/or stabilizers of cellular and viral mRNAs bearing 5’ poly(A) stretches (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). Our results thus suggest the existence of a novel translation initiation mechanism.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>General Strategy for the Modification of pGKL Plasmids Using Homologous Recombination <em>In Vivo</em></h3><p id="p-10"><em>K. lactis</em> IFO1267 was transformed with a PCR-generated fragment consisting of 5’ and 3’ ends homologous to the part of the pGKL plasmid to be modified, a non-homologous part that introduces mutation/s into a plasmid-specific protein coding sequence, and a gene encoding a resistance marker (typically against G418 or hygromycin B) whose transcription is driven by the UCR of <em>K1ORF1</em> or <em>K1ORF2</em>. This type of construct was prepared by fusion PCR; for the organization of such a construct, see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>. After transformation, yeast cells were plated onto selective agar media, and their DNA content was analyzed for the presence of the modified pGKL plasmid using agarose electrophoresis. Both modified and wild-type target plasmids were usually detected after transformation. Colonies containing both the modified and wild-type target plasmids were selected and cultivated under selective conditions for approximately 60 generations and analyzed using agarose electrophoresis. For subsequent analysis, only colonies containing a modified version of the target plasmid were used. All constructs were verified using colony PCRs and subsequent sequencing of amplified products.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Precise manipulation of pGKL plasmids in vivo revealed the indispensability of the K2Orf3p MTase active site for plasmid maintenance and an essential role of pGKL promoters in mRNA capping and non-template-based 5&#x2019; polyadenylation. (A) General description of the experiment. PCR cassette used for the manipulation of pGKL2 by homologous recombination in vivo consists of regions homologous to the native pGKL2 plasmid (in light green), an antibiotic resistance gene (G418) under the control of the ORF2 promoter from pGKL1 (K1UCR2) and the ORF1 promoter from pGKL1 (K1UCR1), which will artificially control the expression of K2ORF2. Promoters (UCRs) are in blue, and the codons D432, G438 and D440 from the MTase active site that were mutated to Ala codons are indicated by black asterisks. (B) A closer view of the native pGKL2 region subjected to homologous recombination shows a tightly packed plasmid genome. The 3&#x2019; end of the K2ORF3 coding region overlaps the K2ORF2 promoter, 5&#x2019; UTR and first 4 nucleotides of the K2ORF2 coding region. Please note that the pGKL2 plasmids displayed in panels B and C are in the reverse orientation of those in Fig. 1. Shades of gray indicate the level of transcript capping, as in Fig. 1. (C) The resulting plasmid, pRKL2-2, contains a version of K2ORF3 with mutations in the coding region of the MTase active site (white asterisks) and two genes, aminoglycoside 3&#x2019;- phosphotransferase (coding for G418 resistance) and K2ORF2, that are artificially controlled by the pGKL1 promoters K1UCR2 and K1UCR1, respectively. Shades of gray indicate the level of transcript capping, as in Fig. 1. (D) Evidence for the vital role of K2Orf3p MTase in pGKL plasmid maintenance in cells. M - Lambda DNA/Eco130I (StyI) Marker (Fermentas); Lanes 1 and 4 - native pGKL plasmids from K. lactis IFO1267 (pGKL1 (8874 bp) is labeled with an asterisk, and pGKL2 (13447 bp) is labeled with an open arrow). Lane 2 - linear plasmids purified from K. lactis IFO1267 carrying the recombinant pRKL2-2 plasmid (14353 bp) bearing a mutated MTase active site after its cultivation for &#x2248; 400 generations in selective medium containing G418; Lane 3 - linear plasmids purified from K. lactis IFO1267 containing the pRKL2-1 plasmid (14353 bp) after cultivation for &#x2248; 60 generations in selective medium containing G418. Mutations in the coding region of the MTase active site spontaneously reverted to the wild-type in pRKL2-1, leading to rapid loss of the natural pGKL2 plasmid under selective conditions. Analysis of 5&#x2019; ends of pRKL2-2 plasmids revealed that non-template adenosine addition and mRNA cap synthesis are directed by linear plasmid promoters. K2ORF2 controlled by the K1UCR1 promoter produces fewer capped transcripts with a higher level of non-template 5&#x2019; polyadenylation, similar to wild-type K1ORF1 transcripts and in contrast with K2ORF2 transcripts controlled by the natural promoter. The K1UCR2 promoter caused a similar degree of mRNA capping and 5&#x2019; non-template polyadenylation when controlling either its own gene (K1ORF2) or a heterologous bacterial gene coding for aminoglycoside 3&#x2019;-phosphotransferase (G418 resistance). Thirteen to twenty-six independent cDNA clones were analyzed in each experiment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Precise manipulation of pGKL plasmids &lt;em&gt;in vivo&lt;/em&gt; revealed the indispensability of the K2Orf3p MTase active site for plasmid maintenance and an essential role of pGKL promoters in mRNA capping and non-template-based 5&#x2019; polyadenylation. (A) General description of the experiment. PCR cassette used for the manipulation of pGKL2 by homologous recombination &lt;em&gt;in vivo&lt;/em&gt; consists of regions homologous to the native pGKL2 plasmid (in light green), an antibiotic resistance gene (G418) under the control of the &lt;em&gt;ORF2&lt;/em&gt; promoter from pGKL1 (&lt;em&gt;K1UCR2&lt;/em&gt;) and the &lt;em&gt;ORF1&lt;/em&gt; promoter from pGKL1 (&lt;em&gt;K1UCR1&lt;/em&gt;), which will artificially control the expression of &lt;em&gt;K2ORF2&lt;/em&gt;. Promoters (UCRs) are in blue, and the codons D432, G438 and D440 from the MTase active site that were mutated to Ala codons are indicated by black asterisks. (B) A closer view of the native pGKL2 region subjected to homologous recombination shows a tightly packed plasmid genome. The 3&#x2019; end of the &lt;em&gt;K2ORF3&lt;/em&gt; coding region overlaps the &lt;em&gt;K2ORF2&lt;/em&gt; promoter, 5&#x2019; UTR and first 4 nucleotides of the &lt;em&gt;K2ORF2&lt;/em&gt; coding region. Please note that the pGKL2 plasmids displayed in panels B and C are in the reverse orientation of those in Fig. 1. Shades of gray indicate the level of transcript capping, as in Fig. 1. (C) The resulting plasmid, pRKL2-2, contains a version of &lt;em&gt;K2ORF3&lt;/em&gt; with mutations in the coding region of the MTase active site (white asterisks) and two genes, aminoglycoside 3&#x2019;- phosphotransferase (coding for G418 resistance) and &lt;em&gt;K2ORF2&lt;/em&gt;, that are artificially controlled by the pGKL1 promoters &lt;em&gt;K1UCR2&lt;/em&gt; and &lt;em&gt;K1UCR1&lt;/em&gt;, respectively. Shades of gray indicate the level of transcript capping, as in Fig. 1. (D) Evidence for the vital role of K2Orf3p MTase in pGKL plasmid maintenance in cells. M - Lambda DNA/Eco130I (StyI) Marker (Fermentas); Lanes 1 and 4 - native pGKL plasmids from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 (pGKL1 (8874 bp) is labeled with an asterisk, and pGKL2 (13447 bp) is labeled with an open arrow). Lane 2 - linear plasmids purified from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 carrying the recombinant pRKL2-2 plasmid (14353 bp) bearing a mutated MTase active site after its cultivation for &#x2248; 400 generations in selective medium containing G418; Lane 3 - linear plasmids purified from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 containing the pRKL2-1 plasmid (14353 bp) after cultivation for &#x2248; 60 generations in selective medium containing G418. Mutations in the coding region of the MTase active site spontaneously reverted to the wild-type in pRKL2-1, leading to rapid loss of the natural pGKL2 plasmid under selective conditions. Analysis of 5&#x2019; ends of pRKL2-2 plasmids revealed that non-template adenosine addition and mRNA cap synthesis are directed by linear plasmid promoters. &lt;em&gt;K2ORF2&lt;/em&gt; controlled by the &lt;em&gt;K1UCR1&lt;/em&gt; promoter produces fewer capped transcripts with a higher level of non-template 5&#x2019; polyadenylation, similar to wild-type &lt;em&gt;K1ORF1&lt;/em&gt; transcripts and in contrast with &lt;em&gt;K2ORF2&lt;/em&gt; transcripts controlled by the natural promoter. The &lt;em&gt;K1UCR2&lt;/em&gt; promoter caused a similar degree of mRNA capping and 5&#x2019; non-template polyadenylation when controlling either its own gene (&lt;em&gt;K1ORF2&lt;/em&gt;) or a heterologous bacterial gene coding for aminoglycoside 3&#x2019;-phosphotransferase (G418 resistance). Thirteen to twenty-six independent cDNA clones were analyzed in each experiment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F2.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F2.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">Precise manipulation of pGKL plasmids <em>in vivo</em> revealed the indispensability of the K2Orf3p MTase active site for plasmid maintenance and an essential role of pGKL promoters in mRNA capping and non-template-based 5’ polyadenylation.</p><p id="p-12">(A) General description of the experiment. PCR cassette used for the manipulation of pGKL2 by homologous recombination <em>in vivo</em> consists of regions homologous to the native pGKL2 plasmid (in light green), an antibiotic resistance gene (G418) under the control of the <em>ORF2</em> promoter from pGKL1 (<em>K1UCR2</em>) and the <em>ORF1</em> promoter from pGKL1 (<em>K1UCR1</em>), which will artificially control the expression of <em>K2ORF2</em>. Promoters (UCRs) are in blue, and the codons D432, G438 and D440 from the MTase active site that were mutated to Ala codons are indicated by black asterisks.</p><p id="p-13">(B) A closer view of the native pGKL2 region subjected to homologous recombination shows a tightly packed plasmid genome. The 3’ end of the <em>K2ORF3</em> coding region overlaps the <em>K2ORF2</em> promoter, 5’ UTR and first 4 nucleotides of the <em>K2ORF2</em> coding region. Please note that the pGKL2 plasmids displayed in panels B and C are in the reverse orientation of those in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>. Shades of gray indicate the level of transcript capping, as in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>.</p><p id="p-14">(C) The resulting plasmid, pRKL2-2, contains a version of <em>K2ORF3</em> with mutations in the coding region of the MTase active site (white asterisks) and two genes, aminoglycoside 3’- phosphotransferase (coding for G418 resistance) and <em>K2ORF2</em>, that are artificially controlled by the pGKL1 promoters <em>K1UCR2</em> and <em>K1UCR1</em>, respectively. Shades of gray indicate the level of transcript capping, as in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>.</p><p id="p-15">(D) Evidence for the vital role of K2Orf3p MTase in pGKL plasmid maintenance in cells. M - Lambda DNA/Eco130I (StyI) Marker (Fermentas); Lanes 1 and 4 - native pGKL plasmids from <em>K. lactis</em> IFO1267 (pGKL1 (8874 bp) is labeled with an asterisk, and pGKL2 (13447 bp) is labeled with an open arrow). Lane 2 - linear plasmids purified from <em>K. lactis</em> IFO1267 carrying the recombinant pRKL2-2 plasmid (14353 bp) bearing a mutated MTase active site after its cultivation for ≈ 400 generations in selective medium containing G418; Lane 3 - linear plasmids purified from <em>K. lactis</em> IFO1267 containing the pRKL2-1 plasmid (14353 bp) after cultivation for ≈ 60 generations in selective medium containing G418. Mutations in the coding region of the MTase active site spontaneously reverted to the wild-type in pRKL2-1, leading to rapid loss of the natural pGKL2 plasmid under selective conditions.</p><p id="p-16">Analysis of 5’ ends of pRKL2-2 plasmids revealed that non-template adenosine addition and mRNA cap synthesis are directed by linear plasmid promoters. <em>K2ORF2</em> controlled by the <em>K1UCR1</em> promoter produces fewer capped transcripts with a higher level of non-template 5’ polyadenylation, similar to wild-type <em>K1ORF1</em> transcripts and in contrast with <em>K2ORF2</em> transcripts controlled by the natural promoter. The <em>K1UCR2</em> promoter caused a similar degree of mRNA capping and 5’ non-template polyadenylation when controlling either its own gene (<em>K1ORF2</em>) or a heterologous bacterial gene coding for aminoglycoside 3’-phosphotransferase (G418 resistance). Thirteen to twenty-six independent cDNA clones were analyzed in each experiment.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>RNA Purification, Electrophoresis, Reverse Transcription, and 5’ and 3’ RACE</h3><p id="p-17">Total yeast RNA was purified by the hot acidic phenol procedure (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Remaining DNA was removed by a DNA-free Kit (Ambion) according to the manufacturer’s protocol. The quality of RNA was assessed by electrophoresis according to the protocol by Masek et al. (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). For 5’ RACE, RT-PCR was carried out as follows: 0.5 μg of total yeast RNA and 0.15 μg of random primers (Invitrogen) were used for cDNA synthesis using 100 U of SSC III Reverse Transcriptase (Invitrogen) in a 20 μl reaction (50°C for 99 min). After the cDNA purification using the High Pure PCR Product Purification Kit (Roche), 800 U of recombinant TdT (Roche) and 0.5 mM dGTP (Fermentas) were used for cDNA tailing in 1x TdT buffer without CoCl<sub>2</sub> for 30 min at 37°C with subsequent TdT inactivation at 70°C for 10 min. For amplification of cDNA, 2.5 μl of the reaction mixture was used for the following PCR with the universal olig2(dC) anchor primer and an appropriate gene-specific primer. In the case of 3’ RACE, 0.5 μg of total yeast RNA was polycytidinylated using a Poly(A) Tailing Kit (Applied Biosystems) with 2 mM CTP in a total volume of 25 μl for 90 min. Reverse transcription was then performed using oligo(dG)anch2 primer as indicated above. After cDNA purification using the High Pure PCR Product Purification Kit (Roche), 2.5 μl of the purified cDNA was used for a subsequent PCR reaction with the universal primer anch2 and an appropriate gene-specific primer. In both types of RACE experiments, following PCR amplification and electrophoresis, the corresponding fragments were purified from gel using a FastBack DNA minispin kit (Renogen Biolab), cloned into a pCR4-TOPO plasmid using the TOPO strategy and sequenced using the universal T7 promoter primer and/or a T3 primer. All primers used in this study are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S2</a>.</p></div><div id="sec-5" class="subsection"><h3>eIF4E Expression and Purification</h3><p id="p-18">The pGEX4T2::CDC33 plasmid was transformed into the expression strain <em>E. coli</em> BL21(DE3) (Merck). A 500 ml culture was grown at 28°C in 2xTY medium containing ampicillin (100 μg/ml) until the OD<sub>600</sub> reached 0.20. At this point, the culture was cooled to 15°C, cultivated for 4 hours and then induced by 0.1 mM isopropyl β-D-thiogalactopyranoside. Production of recombinant protein was carried out with shaking at 15°C for another 14-16 hours. Lysates were prepared using B-PER Protein Extraction Reagents (Thermo Scientific) according to the manufacturer’s instruction and stored at −70°C for further use.</p></div><div id="sec-6" class="subsection"><h3>mRNA/eIF4E Binding Assay</h3><p id="p-19">On days of binding experiments, bacterial lysate containing GST-eIF4E fusion protein was thawed in an ice bath, centrifuged for 10 min at 15000 g (4°C), and GST-eIF4E was purified using GST-affinity chromatography in a batch setup. In brief, 1 ml of the lysate was incubated for 80 min with 100 μl of glutathione-Sepharose 4 Fast Flow resin (GE Healthcare) at 4°C and washed four times with at least 40 volumes of ice-cold 1x phosphate-buffered saline (PBS). After the last washing step, the glutathione Sepharose binding GST-eIF4E fusion protein was resuspended in 200 μl of buffer I (20 mM HEPES, pH 7.5; 0.1 mM EDTA, pH 8.0; 100 mM KCl; and 1 mM β-mercaptoethanol) and mixed with 3 μg of DNase I-treated total RNA purified from <em>K. lactis</em> IFO1267. Next, 2 μl of the reaction mixture was removed for subsequent analysis using RT-PCR and real-time quantitative PCR (RQ-PCR). The rest of the mixture was incubated for two hours at room temperature and washed six times (with approximately 70 volumes) with buffer I. After the last washing, the glutathione Sepharose binding GST-eIF4E fusion protein and RNA and control samples were subjected to RT-PCR. All buffers used for protein purification contained Complete (EDTA-free) Protease Inhibitor (Roche), and all buffers used for handling RNA contained RNasin Ribonuclease Inhibitor (Promega).</p><p id="p-20">Two microliters of reverse transcriptase reaction (obtained from eIF4E-mRNA <em>in vitro</em> binding assay) were subjected to PCR amplification as follows (5 min at 94°C; then 35 cycles of 30 sec at 94°C; 30 sec at 50°C; 45 sec at 72°C; and finally, 4 min at 72°C) using Taq DNA polymerase (Roche) and <em>HGT1</em> and <em>K2ORF5</em> gene-specific primers (listed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S2</a>).</p><p id="p-21">Two and half microliters of reverse transcriptase reaction (obtained from a reaction mixture before its incubation with glutathione Sepharose binding a GST-eIF4E fusion protein; see above) was subjected to Real-Time PCR amplification (5 min at 94°C; then 55 cycles of 20 sec at 94°C, 30 sec at 50°C, and 45 sec at 72°C; and finally, 4 min at 72°C) using an iCycler iQ Real-Time PCR Detection System (BioRad) and SYBR Green Master Mix (BioRad).</p></div><div id="sec-7" class="subsection"><h3>Transfer of Cytoplasmic Plasmids to Yeast Strains by Incomplete Mating</h3><p id="p-22">The <em>S. cerevisiae</em> strain YAT547 (<em>kar1</em>) containing pGKL plasmids (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>) and isogenic <em>S. cerevisiae</em> ρ<sup>0</sup> strains bearing the wild-type <em>CDC33</em> gene (CWO4ρ<sup>0</sup><em>CDC33</em>wt) and its temperature-sensitive mutations (CWO4p°<em>cdc33-1</em> and CWO4ρ<sup>0</sup><em>cdc33-42</em>, for exact genotypes see <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S3</a>) were cultured separately in YPD medium at 24°C overnight. Cells were resuspended the next day in 200 μl of water, and 5 μl from each of two required strains (approximately 3×10<sup>6</sup> cells) were mixed on the YPD agar plate and further incubated for 5 hours at 24°C. Cells were then scraped to liquid YPD medium, incubated at 24°C with gentle shaking overnight and plated on selective plates. The resulting strains were tested for their auxotrophic markers, mating type and for killer toxin production under both permissive and non-permissive conditions. Strains showing a temperature-sensitive phenotype (when applicable), killer toxin production, growth on SD medium lacking methionine, no visible growth on SD medium lacking uracil and a MATa mating type were used for subsequent analyses.</p></div><div id="sec-8" class="subsection"><h3>pGKL Plasmid Purification and Electrophoresis</h3><p id="p-23">For the analysis of pGKL plasmids, a modified protocol based on that of Pospisek and Palkova (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>) was used. Briefly, cells were grown for three days on a dish containing the appropriate antibiotics, transferred into a microplate well and dried for 2 hours at 45°C. After they had dried completely, cells were resuspended in 40 μl of freshly prepared TESP buffer (20 mM Tris-Cl, pH 8; 50 mM EDTA-NaOH, pH 8; 2 % SDS; and 0.5 mg/ml pronase E) and dried at 37°C overnight. The sample was completely resuspended the next day in 40 μl of 1× DNA loading buffer (Fermentas). Then, 15 μl of the sample were analyzed using agarose electrophoresis (0.5 % agarose, 1 V/cm) for at least 20 hours. After the electrophoresis, the gel was incubated in a solution containing ethidium bromide (0.5 μg/ml) and RNase A (50 μg/ml) for 3 hours.</p></div><div id="sec-9" class="subsection"><h3>Mutagenesis of Predicted K2Orf3p SAM Binding Site Using Homologous Recombination <em>in Vivo</em></h3><p id="p-24">A fusion PCR approach was used to assembly the desired construct for SAM mutagenesis. The first primary PCR product was amplified from a native pGKL2 plasmid using Pfu DNA polymerase (Fermentas) and the primers ORF3_SAM_del_F1 and ORF3_SAMdel_R1. The second primary PCR product was amplified using Pfu DNA polymerase (Fermentas) and the primers KL_orf6C_Flag2F and ORF3_SAMdel_R2 (see <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table S2</a>) from a modified pGKL1 plasmid whose <em>K1ORF2</em> coding sequence had been replaced with G418 (unpublished data). Both primary PCR products, which had defined overlapping ends, were synthesized as follows: 5 min at 95°C; then 30 cycles of 30 sec at 94°C, 30 sec at 56.2°C, and 90 sec at 72°C; and finally, 10 min at 72°C. Then, 1 μl of each PCR mixture was used without purification for a second PCR reaction using Pfu DNA polymerase (Fermentas), the primers ORF3_SAM_del_F1 and ORF3_SAMdel_R2 and following conditions: 5 min at 95°C; then 35 cycles of 30 sec at 94°C, 30 sec at 60.1°C, and 2 min at 72°C; and finally, 10 min at 72°C). The resulting PCR product was purified using agarose electrophoresis and transformed into <em>K. lactis</em> IFO1267. After five hours of cultivation under nonselective conditions, cells were plated onto solid medium containing G418 (250 μg/ml). Site-specific integration of the PCR cassette into a pGKL2 plasmid as well as the subsequent curing of the unmodified pGKL2 plasmid was evaluated by PCR with primers (in_Kan_rev1 and in_ORF3_forw) and by sequencing of a gel-purified PCR fragment that had been amplified with primers (in_Kan_rev1 and K2_ORF3_for_seq).</p></div><div id="sec-10" class="subsection"><h3>Supplemental Experimental Procedures</h3><p id="p-25">Please follow the Supplemental experimental procedures for the rest of the methods including Plasmid construction, Assay of enzyme-GMP complex formation, Removal of mitochondrial DNA by ethidium bromide treatment, Transformation of yeast cells, Killer production in modified CWO4 yeast strains, Assay of killer toxin activity, Statistical analysis, Expression and purification of mRNA decapping enzymes, <em>In vitro</em> processing of mRNAs by decapping enzymes, <em>In vitro</em> synthesis of capped mRNA using the vaccinia virus mRNA capping enzyme, Deletion of <em>PBP1</em>, <em>PAB1</em> and <em>LSM1</em> genes and the full list of primers, strains and plasmids used throughout this study.</p></div></div><div class="section" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Purification of the mRNA Capping Enzyme (K2Orf3p) and Testing its Enzymatic Activities</h3><p id="p-26">The <em>K2ORF3</em> gene, carried by the pGKL2 plasmid, was proposed to code for a mRNA capping enzyme similar to that of the vaccinia virus. Even though K2Orf3p apparently contains evolutionarily conserved domains corresponding to all three principal capping enzymatic activities, TPase, GTase and MTase, only TPase and GTase activities have been experimentally verified (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>). The enzymology of mRNA cap synthesis has been reviewed in (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). A complete N<sup>7</sup>-methylated mRNA cap is essential for regular translation initiation in eukaryotes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). We thus attempted to detect MTase activity of the purified K2Orf3 protein <em>in vitro</em>.</p><p id="p-27">We produced K2Orf3p as N-terminal His-and GST-tagged proteins using an <em>E. coli</em> T7-based system or as an N-terminal His-tagged fusion protein using a baculovirus expression system. We purified the His-and GST-tagged K2Orf3 proteins to homogeneity using appropriate affinity chromatography and tested them for GTase and MTase activities. We readily detected GTase activity of the purified recombinant K2Orf3 proteins in all cases (data not shown); however, their MTase activity was not detected (data not shown). These results are in good agreement with the previously described results of Tiggemann and co-workers (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>), who had shown that K2Orf3p possesses TPase and GTase activities <em>in vitro</em> but not MTase activity. They explained this result by using a C-terminal His-tag for the heterologous production of the K2Orf3p capping enzyme that may have interfered with the C-terminal MTase domain (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>). We produced the putative K2Orf3p capping enzyme N-terminally fused with either His or GST tags but also failed to detect MTase activity using several <em>in vitro</em> approaches.</p><p id="p-28">One possible reason we did not detect MTase activity from K2Orf3p is that the presence of an unknown stimulatory protein is required for complete and <em>in vitro</em>-detectable MTase activity of K2Orf3p, similar to a capping enzyme encoded by the vaccinia virus. The vaccinia virus encodes its own heterodimeric capping enzyme, which is composed of D1 and D12 subunits. D1 contains the domains coding for all three principal enzymatic activities required for conventional m<sup>7</sup>G cap synthesis arranged in an identical order to that of the domains of K2Orf3p. The D12 subunit heterodimerizes with the MTase domain of the D1 subunit and allosterically stimulates its activity (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>–<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). However, the MTase activity of the D1 subunit of the vaccinia virus capping enzyme is detectable even in the absence of the stimulating protein (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>).</p></div><div id="sec-13" class="subsection"><h3>K2Orf3p Does not Complement Conditional Mutations in <em>CEG1</em> and <em>ABD1</em> Genes</h3><p id="p-29"><em>Saccharomyces cerevisiae</em> yeast strains lacking active components of the mRNA capping machinery have been successfully used for studies of heterologous capping enzymes, including those from human (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>), mouse (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>), <em>Candida albicans</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>,<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), <em>Schizosaccharomyces pombe</em> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a>,<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>), <em>Encephalitozoon cuniculi</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>), <em>Caenorhabditis elegans</em> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>) and vaccinia virus (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>). Putative capping enzyme K2Orf3p belongs to the D1-like group of capping enzymes due to its high similarity to the vaccinia virus capping enzyme. We therefore decided to test the ability of K2Orf3p to complement the growth of yeast cells bearing temperature-sensitive mutations in either the cap guanylyltransferase <em>CEG1</em> (GTase, (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>)) or the cap methyltransferase <em>ABD1</em> (MTase, (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>)) genes. We constructed yeast plasmids pYX212::NLS-K2ORF3 and pYX213::NLS-K2ORF3, which had been designed to produce K2Orf3p fused with an SV40 nuclear localization signal under the control of either a strong constitutive triose phosphate isomerase (TPI) promoter or a tightly regulated <em>GAL1</em> promoter, respectively. These plasmids were introduced into the <em>S. cerevisiae</em> strains carrying either <em>ceg1</em><sup>ts</sup> or <em>abd1</em><sup>ts</sup> temperature-sensitive alleles (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a>,<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a>). We detected no growth in any of the resulting yeast strains at the restrictive temperature (data not shown). Multiple reasons can explain such a result. K2Orf3p may need to be artificially forced to interact with the C-terminal domain of RNA polymerase II or may require another co-factor for its proper function. The former approach was used to study an N-terminal fragment of the vaccinia virus capping enzyme D1 (AAs 1-545) in yeast, where it was produced as a protein fused with part of the mouse capping enzyme Mce1 (AAs 211-597). The resulting fusion protein rescued the growth of yeast strains lacking genes coding for a TPase (<em>cet1Δ</em>) and a GTase (<em>ceg1Δ</em>) (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>). Saha et al. used the latter approach to show that the methyltransferase domain of the vaccinia virus capping enzyme, which is composed of the co-expressed catalytic C-terminal part of D1 and stimulatory D12 subunits, can function in lieu of the yeast cap methyltransferase Abd1 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>). A similar approach was also used to test the N<sup>7</sup> methyltransferase activities of viral capping enzymes (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>,<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>). On the other hand, some cap methyltransferases, including Hcm1 (from human), do not require any auxiliary factors to be co-expressed to support yeast growth in lieu of the Abd1 protein (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">54</a>,<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>,<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>). We should notice that the vaccinia virus D1 protein itself displays mild MTase activity (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>), and we complemented the temperature-sensitive phenotype, allowing us to follow the yet unobserved complementation of a partial loss of Abd1 activity in cells. Another, and perhaps the most likely, explanation why K2Orf3p does not complement the mRNA capping machinery in yeast temperature-sensitive mutants is based on the recent observation that expression of AT-rich pGKL genes in yeast nuclei does not lead to functional products due to the frequent premature 3’ polyadenylation of the resulting mRNAs (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>).</p></div><div id="sec-14" class="subsection"><h3>K2Orf3p MTase Active Site Integrity Is Essential for Linear Plasmid Maintenance</h3><p id="p-30">We and others (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>) failed in numerous attempts to determine the guanosine methyltransferase activity of the K2Orf3 protein. We failed even to complement conditional mutations in the gene of yeast <em>ABD1</em>, a nuclear cap methyltransferase, with K2Orf3p targeted to the yeast nucleus. Genomes of pGKL plasmids are extremely tightly packed (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figs. 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">2B, C</a>) (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>). It is unlikely that the non-functional MTase region would not be eliminated from plasmids during evolution. We therefore decided to test whether the intact putative MTase is essential for the maintenance of the pGKL plasmids in <em>K. lactis</em> cells.</p><p id="p-31">We developed a unique fusion PCR strategy, followed with homologous recombination <em>in vivo</em>; this approach enabled us to precisely modify the <em>K2ORF3</em> MTase coding region directly in the cytoplasmically localized pGKL plasmids (for details of the method, see <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2 A-C</a> and Materials and Methods). Using this strategy, we modified a predicted SAM-binding site of the putative <em>K2ORF3</em> MTase domain (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>) by alanine scanning mutagenesis. The homologous integration of the mutational cassette (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2A</a>) into the pGKL2 plasmid resulted in appearance of the new larger plasmid pRKL2 (<span class="underline">R</span>ecombinant <em><span class="underline">K</span></em>. <em><span class="underline">l</span>actis</em> plasmid pGKL<span class="underline">2</span>, 14353 bp) in the yeast cells. This plasmid contained mutated codons encoding alanines at the positions of three amino acids (D432, G438 and D440) conserved in a predicted K2Orf3p SAM-binding site and also included the G418 resistance gene under control of the <em>K1ORF2</em> <span class="underline">U</span>pstream <span class="underline">C</span>ontrol <span class="underline">R</span>egion [UCR; a promoter in <em>sensu lato</em> and a sequence located between the AUG initiation codon and the UCS (including) of the selected ORF] (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2C</a> and <a id="xref-fig-2-6" class="xref-fig" href="#F2">2D</a>). Two smaller plasmids in the cells corresponded to the natural pGKL1 (8874 bp) and pGKL2 (13457 bp) plasmids (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2D</a>, lane 2). Randomly selected <em>K. lactis</em> clones containing all three plasmids were passaged in a selective liquid medium containing G418 (250 μg/ml) for at least 60 generations and plated on agar medium supplemented with G418 (250 μg/ml). Purified plasmids from randomly selected subclones were analyzed by agarose gel electrophoresis. In a few cases, the natural pGKL2 plasmid was lost, and only pGKL1 and recombinant pRKL2 plasmids remained in the yeast cells (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2D</a>, lane 3). In most cases, both pGKL2 and pRKL2, containing the mutated MTase active site, replicated stably together in <em>K. lactis</em> cells passaged under selective pressure of G418 antibiotics for more than 400 generations (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2D</a>). Four hundred generations of the mutated plasmid being favored by selective pressure and cannot outcompete the wild-type plasmid is roughly the limit of the indispensability of any analyzed pGKL ORF in this kind of experiment (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>).</p><p id="p-32">We purified pRKL2 plasmids from both types of yeast clones (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2</a>, lanes 2 and 3) and sequenced the MTase coding regions of their <em>K2ORF3</em> genes. Surprisingly, yeast clones containing pRKL2 and pGKL1 only carried mutations that reversed the K2Orf3p SAM-binding site to its wild-type sequence or contained incomplete mutational cassettes that brought the same result, leaving the SAM-binding site unmodified. We called these former plasmids pRKL2-1 (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2D</a>, lane 3). The pRKL2-2 plasmids, which were prevalent, retained the mutations in the MTase active-site-coding region and required wild-type pGKL2 to be stably maintained in the cell.</p><p id="p-33">These findings clearly indicate the indispensability of the K2Orf3p SAM-binding site for maintaining pGKL plasmids in yeast cells and are consistent with previous reports stating that the entire <em>K2ORF3</em> is essential for the maintenance of pGKL plasmids in yeast cells (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>).</p></div><div id="sec-15" class="subsection"><h3>Linear Plasmid Transcripts are Largely Uncapped and Contain a Non-Template 5’ Poly(A) Leader</h3><p id="p-34">Our findings pointing to the essentiality of the K2Orf3p MTase active site for linear plasmid maintenance in yeast immediately raised the question of whether the pGKL transcripts are capped. We analyzed the 5’ ends of the pGKL transcripts using a modified rapid amplification of 5’ cDNA ends (5’ RACE) approach. Briefly, we purified total RNA from the <em>K. lactis</em> IFO1267 strain, treated it with DNase I, synthesized cDNA using SuperScript III reverse transcriptase and random primers, G-tailed the cDNA using dGTP and terminal deoxynucleotidyl transferase (TdT) and amplified the 5’ regions corresponding to transcripts from individual ORFs by PCR with an anchored oligo(dC) primer and an appropriate gene-specific primer (see primer sequences in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table S2</a>). Reverse transcriptases, SuperScript III including, can overcome the 5’- 5’ triphosphate bond between the 5’ m<sup>7</sup>G cap and the first nucleotide of nascent mRNA transcripts and show whether the mRNA is capped or not (6971). PCR fragments were verified by digestion with restriction endonucleases and cloned into a pCR4-TOPO plasmid. From 20 to 79 independent clones were sequenced and analyzed for each ORF. We used <em>K. lactis ACT</em> transcript (EnsemblFungi Id: KLLA0_D05357g), which codes for actin and is transcribed by the RNA polymerase II from a nuclear gene as a control 5’ capped mRNA.</p><p id="p-35">The 5’ RACE analysis of the pGKL transcripts revealed a unique and surprising composition of 5’ ends. Eukaryotic mRNAs generally contain an m<sup>7</sup>G cap structure at their 5’ ends, which is a prerequisite for their stability and efficient translation initiation (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>–<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>). Even though the pGKL plasmids encode their own putative capping enzyme (K2Orf3p), we found that few pGKL1/2 genes give rise to the frequently 5’-capped transcripts and that most of the transcripts contain a short leader of non-template adenosine nucleotides (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>) at their 5’ ends that could not have been continuously transcribed from the plasmids (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figs. 2E, 3A</a> and C, 7B; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables S1A</a>, S5). We found only two ORFs of the fifteen encoded by the pGKL plasmids, of which at least 90% of the transcripts were 5’ capped. At least 50% of capped transcripts were found in mRNAs corresponding to five ORFs, which is one-third of all ORFs encoded by the pGKL plasmids (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figs. 1, 3C</a>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table S1A</a>). Non-template adenosine nucleotides were less likely added at the 5’ ends of mRNAs transcribed from ORFs whose more than 50% transcripts were capped. Conversely, longer 5’ poly(A) leaders were found in mRNAs transcribed from ORFs that yielded less frequently capped or completely uncapped transcripts (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3C</a>; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table S1A</a>). However, both non-templately added adenosine nucleotides and mRNA caps were frequently found within identical transcripts (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a>). We calculated the number of non-template adenosine nucleotides in the 5’ leaders of the mRNAs of each pGKL ORF as well as the minimal number of template and non-template adenosines found at the 5’ ends of their uncapped transcripts and the fractions of capped mRNAs and mRNAs containing nontemplate 5’ adenosine nucleotides. We also calculated the medians and means of the numbers of non-template adenosine nucleotides added per mRNA molecule from each pGKL ORF. These data are summarized in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="5&#x2019;/3&#x2019; RACE analysis of pGKL transcripts. (A, B) Representative pictures of 5&#x2019; RACE (A) and 3&#x2019; RACE (B) analyses of individual mRNAs transcribed from K1ORF1 encoded by the pGKL1 plasmid. The top sequence corresponds to the template (plasmid) DNA; the cDNA sequences below represent individual clones obtained by the reverse transcription of individual mRNA molecules. In the case of 5&#x2019; RACE (A), sequences homologous to the template (plasmid) DNA are highlighted in red, and non-template 5&#x2019; leaders of the mRNA transcripts are in green. Guanosine nucleotides corresponding to the 5&#x2019; mRNA caps are depicted in black and underlined. K1UCR1, which was also used as a promoter sequence in other experiments, is underlined. K1UCS1 is highlighted in yellow. (B) Sequences obtained by 3&#x2019; RACE. Stop codons are highlighted in gray. For details concerning transcripts of other pGKL ORFs, please see Table S1A and the description therein. (C) Tabular summary of 5&#x2019; RACE experiments. A graphical representation of each experiment is in Table S1A and B. Each ORF is characterized by the following data: 1/ ORF number; 2/ number of 5&#x2019; non-template adenosine nucleotides found in the corresponding mRNAs; 3/ minimal number of adenosine nucleotides (calculated for both template and non-template adenosines) found at 5&#x2019; ends of uncapped transcripts; 4/ percentage of all transcripts containing 5&#x2019; non-template adenosine nucleotides; 5/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 6/ number of sequenced clones used for analyses; the number of capped and 5&#x2019; polyadenylated mRNAs are depicted below the total number; 7/ percentage of all transcripts containing 5&#x2019; mRNA cap structure; 8/ part of the 5&#x2019; UTR complementary to the relevant plasmid genome. Please note that columns 4 and 7 may not sum to one hundred percent because some transcripts contain both 5&#x2019; non-template adenosines and a cap structure. 1 template and non-template; 2 G = capped transcripts; A - 5&#x2019; polyadenylated transcripts; 3 first templated nucleotides transcribed to the mRNA, non-template adenosines were removed; * these adenosines immediately precede the ATG sequence; ** for exact sequences, see Table S1A; transcripts from K. lactis ACT gene, coding for actin, were used as an internal control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;5&#x2019;/3&#x2019; RACE analysis of pGKL transcripts. (A, B) Representative pictures of 5&#x2019; RACE (A) and 3&#x2019; RACE (B) analyses of individual mRNAs transcribed from &lt;em&gt;K1ORF1&lt;/em&gt; encoded by the pGKL1 plasmid. The top sequence corresponds to the template (plasmid) DNA; the cDNA sequences below represent individual clones obtained by the reverse transcription of individual mRNA molecules. In the case of 5&#x2019; RACE (A), sequences homologous to the template (plasmid) DNA are highlighted in red, and non-template 5&#x2019; leaders of the mRNA transcripts are in green. Guanosine nucleotides corresponding to the 5&#x2019; mRNA caps are depicted in black and underlined. &lt;em&gt;K1UCR1&lt;/em&gt;, which was also used as a promoter sequence in other experiments, is underlined. &lt;em&gt;K1UCS1&lt;/em&gt; is highlighted in yellow. (B) Sequences obtained by 3&#x2019; RACE. Stop codons are highlighted in gray. For details concerning transcripts of other pGKL ORFs, please see Table S1A and the description therein. (C) Tabular summary of 5&#x2019; RACE experiments. A graphical representation of each experiment is in Table S1A and B. Each ORF is characterized by the following data: 1/ ORF number; 2/ number of 5&#x2019; non-template adenosine nucleotides found in the corresponding mRNAs; 3/ minimal number of adenosine nucleotides (calculated for both template and non-template adenosines) found at 5&#x2019; ends of uncapped transcripts; 4/ percentage of all transcripts containing 5&#x2019; non-template adenosine nucleotides; 5/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 6/ number of sequenced clones used for analyses; the number of capped and 5&#x2019; polyadenylated mRNAs are depicted below the total number; 7/ percentage of all transcripts containing 5&#x2019; mRNA cap structure; 8/ part of the 5&#x2019; UTR complementary to the relevant plasmid genome. Please note that columns 4 and 7 may not sum to one hundred percent because some transcripts contain both 5&#x2019; non-template adenosines and a cap structure. &lt;sup&gt;1&lt;/sup&gt; template and non-template; &lt;sup&gt;2&lt;/sup&gt; G = capped transcripts; A - 5&#x2019; polyadenylated transcripts; &lt;sup&gt;3&lt;/sup&gt; first templated nucleotides transcribed to the mRNA, non-template adenosines were removed; * these adenosines immediately precede the ATG sequence; ** for exact sequences, see Table S1A; transcripts from &lt;em&gt;K. lactis ACT&lt;/em&gt; gene, coding for actin, were used as an internal control.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F3.medium.gif" width="295" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F3.medium.gif" width="295" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">5’/3’ RACE analysis of pGKL transcripts.</p><p id="p-37">(A, B) Representative pictures of 5’ RACE (<span class="underline">A</span>) and 3’ RACE (<span class="underline">B</span>) analyses of individual mRNAs transcribed from <em>K1ORF1</em> encoded by the pGKL1 plasmid. The top sequence corresponds to the template (plasmid) DNA; the cDNA sequences below represent individual clones obtained by the reverse transcription of individual mRNA molecules. In the case of 5’ RACE (<span class="underline">A</span>), sequences homologous to the template (plasmid) DNA are highlighted in red, and non-template 5’ leaders of the mRNA transcripts are in green. Guanosine nucleotides corresponding to the 5’ mRNA caps are depicted in black and underlined. <em>K1UCR1</em>, which was also used as a promoter sequence in other experiments, is underlined. <em>K1UCS1</em> is highlighted in yellow. (<span class="underline">B</span>) Sequences obtained by 3’ RACE. Stop codons are highlighted in gray. For details concerning transcripts of other pGKL ORFs, please see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table S1A</a> and the description therein.</p><p id="p-38">(C) Tabular summary of 5’ RACE experiments. A graphical representation of each experiment is in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table S1A</a> and <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">B</a>. Each ORF is characterized by the following data: 1/ ORF number; 2/ number of 5’ non-template adenosine nucleotides found in the corresponding mRNAs; 3/ minimal number of adenosine nucleotides (calculated for both template and non-template adenosines) found at 5’ ends of uncapped transcripts; 4/ percentage of all transcripts containing 5’ non-template adenosine nucleotides; 5/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 6/ number of sequenced clones used for analyses; the number of capped and 5’ polyadenylated mRNAs are depicted below the total number; 7/ percentage of all transcripts containing 5’ mRNA cap structure; 8/ part of the 5’ UTR complementary to the relevant plasmid genome. Please note that columns 4 and 7 may not sum to one hundred percent because some transcripts contain both 5’ non-template adenosines and a cap structure. <sup>1</sup> template and non-template; <sup>2</sup> G = capped transcripts; A - 5’ polyadenylated transcripts; <sup>3</sup> first templated nucleotides transcribed to the mRNA, non-template adenosines were removed; * these adenosines immediately precede the ATG sequence; ** for exact sequences, see <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table S1A</a>; transcripts from <em>K. lactis ACT</em> gene, coding for actin, were used as an internal control.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">More than half of all pGKL ORFs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) contained 5’ untranslated regions (UTRs) composed exclusively of adenosine nucleotides (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figs. 3C</a>; <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Tables S1A</a>,S5) regardless of their template or non-template origin, thus resembling adenosine-rich yeast internal ribosome entry sites (IRESs) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>). The transcripts arising from pGKL1 promoters tended to have longer and more uncapped 5’ poly(A) leaders than pGKL2 transcripts (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3C</a>; <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table S1A</a>). Additionally, the 5’ cap is more frequently present on the transcripts containing few or zero non-template adenosine nucleotides at their 5’ ends (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table S1A</a>). The 5’ ends of plasmid mRNAs resemble the organization of intermediate and late vaccinia virus mRNAs (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>,<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>), late cowpox virus mRNAs (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>) and even some plant mRNAs (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>) that also contain non-templated poly(A) leaders at their 5’ ends. However, all these mRNA transcripts are fully capped. Considering the aforementioned examples and combining our findings with the knowledge that the domain organization of the K2Orf3p putative capping enzyme resembles that of the poxviral capping enzyme (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a>) allow us to speculate on the similarity between pGKL plasmids and poxviruses, at least at the level of transcription and posttranscriptional modification.</p><p id="p-40">Our 5’ RACE analyses also identified the three new functional UCS sites and the corresponding transcription start sites of <em>K1ORF4, K2ORF3</em> and <em>K2ORF4</em> (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table S1A</a>, marked as blue boxes).</p></div><div id="sec-16" class="subsection"><h3>Capped pGKL Transcripts Contain an N<sup>7</sup>-Methylguanosine Moiety</h3><p id="p-41">Recent advances in the understanding of mRNA quality control and the function of closely associated cellular mRNA decapping enzymes allowed us to use <em>S. pombe</em> Rai1 and human Dcp2 proteins to directly investigate predicted N<sup>7</sup>-methylation at the 5’ cap guanosine of pGKL mRNAs. Expression plasmids for the production of His-tagged Dcp2 and Rai1 proteins were obtained from Prof. Mike Kiledjian (Rutgers University) and Prof. Liang Tong (Columbia University), respectively. Both proteins were produced in the <em>E. coli</em> BL21(DE3) strain and purified to homogeneity exactly as described by Xiang S. et al. in 2009 (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>). We purified total RNA from the killer <em>K. lactis</em> IFO1267 strain for cap-methylation analysis. The total RNA was DNase I-treated and subsequently incubated with either purified Rai1 decapping enzyme, which removes unmethylated caps only (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>), or purified hDcp2 decapping enzyme, which removes both methylated and unmethylated caps from the 5’ ends of mRNAs (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>,<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>). To control for the proper function of both mRNA decapping enzymes, we used <em>in vitro</em>-transcribed RNA corresponding to the first 579 nt (including 32 nt of the 5’ UTR) of the <em>K. lactis ACT</em> gene, which encodes actin, (EnsemblFungi Id: KLLA0_D05357g) that had been capped using the vaccinia virus capping enzyme in either the presence or absence of SAM and subsequently incubated with Rai1 or hDcp2 decapping enzymes. The workflow of the experiment is illustrated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>. Treatment of <em>K. lactis</em> total RNA with hDcp2 significantly decreased the number of capped pGKL mRNAs represented by the originally highly capped transcripts from <em>K2ORF1</em> and <em>K2ORF2</em> genes, whereas treatment with Rai1 had no such effect. Identical results were obtained for the mRNA coding for actin (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). These results provide strong evidence that capped pGKL transcripts contain regular N<sup>7</sup>-methylated caps, just as the endogenous nuclear transcripts represented by actin mRNA in this experiment. <em>In vitro</em>-transcribed actin mRNA that was capped but not N<sup>7</sup>-methylated (G-actin RNA) served as a control of Rai1 activity. Treatment of G-actin RNA by Rai1 and hDcp2 reduced the number of capped transcripts by 43 and 29%, respectively. Conversely, caps of <em>in vitro</em>-transcribed capped and methylated actin RNA (m<sup>7</sup>G-actin) remained completely unaffected by Rai1 treatment, whereas hDcp2 removed 50% of all the caps (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Guanosine caps at the 5&#x2019; ends of the pGKL mRNAs are N7-methylated. (A) Design of the control experiment. Capping of synthetic actin RNA with an unmethylated 5&#x2019; cap and 5&#x2019; RACE analysis after Rail treatment are depicted. A PCR cassette consisting of a modified T7 promoter (130) and the first 579 nt (including 32 nt of 5&#x2019; UTR) of the K. lactis ACT gene coding sequence was transcribed using T7 RNA polymerase. The resulting uncapped RNA molecules (1) were capped using a vaccinia virus capping enzyme in the absence (2) or presence of SAM (not shown). The RNAs were incubated with either Rail (3) or hDcp2 (not shown). The presence of either methylated (not shown) or unmethylated 5&#x2019; caps after each step was monitored by 5&#x2019; RACE followed by the cloning and sequencing of individual cDNA clones. The graphical representation of the 5&#x2019; RACE results is similar to those in Fig. 3 A. (B) Tabular representation of the 5&#x2019; RACE experiments using either total RNA prepared from K. lactis IFO1267 or synthetic K. lactis actin mRNA capped by a vaccinia virus mRNA capping enzyme in either the presence (m7G-actin) or absence (G-actin) of SAM. RNA preparations were analyzed untreated or treated with Rai1 or hDcp2. Numbers represent the fractions of capped RNAs from all the independent cDNA clones analyzed (numbers in brackets labeled with NAC). Numbers labeled with an asterisk are replicated from Fig. 3C. NAC - number of analyzed clones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Guanosine caps at the 5&#x2019; ends of the pGKL mRNAs are N&lt;sup&gt;7&lt;/sup&gt;-methylated. (A) Design of the control experiment. Capping of synthetic actin RNA with an unmethylated 5&#x2019; cap and 5&#x2019; RACE analysis after Rail treatment are depicted. A PCR cassette consisting of a modified T7 promoter (130) and the first 579 nt (including 32 nt of 5&#x2019; UTR) of the &lt;em&gt;K. lactis ACT&lt;/em&gt; gene coding sequence was transcribed using T7 RNA polymerase. The resulting uncapped RNA molecules (1) were capped using a vaccinia virus capping enzyme in the absence (2) or presence of SAM (not shown). The RNAs were incubated with either Rail (3) or hDcp2 (not shown). The presence of either methylated (not shown) or unmethylated 5&#x2019; caps after each step was monitored by 5&#x2019; RACE followed by the cloning and sequencing of individual cDNA clones. The graphical representation of the 5&#x2019; RACE results is similar to those in Fig. 3 A. (B) Tabular representation of the 5&#x2019; RACE experiments using either total RNA prepared from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 or synthetic &lt;em&gt;K. lactis&lt;/em&gt; actin mRNA capped by a vaccinia virus mRNA capping enzyme in either the presence (m&lt;sup&gt;7&lt;/sup&gt;G-actin) or absence (G-actin) of SAM. RNA preparations were analyzed untreated or treated with Rai1 or hDcp2. Numbers represent the fractions of capped RNAs from all the independent cDNA clones analyzed (numbers in brackets labeled with &lt;sup&gt;NAC&lt;/sup&gt;). Numbers labeled with an asterisk are replicated from Fig. 3C. NAC - number of analyzed clones.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F4.medium.gif" width="390" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F4.medium.gif" width="390" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Guanosine caps at the 5’ ends of the pGKL mRNAs are N<sup>7</sup>-methylated.</p><p id="p-43">(A) Design of the control experiment. Capping of synthetic actin RNA with an unmethylated 5’ cap and 5’ RACE analysis after Rail treatment are depicted. A PCR cassette consisting of a modified T7 promoter (<a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">130</a>) and the first 579 nt (including 32 nt of 5’ UTR) of the <em>K. lactis ACT</em> gene coding sequence was transcribed using T7 RNA polymerase. The resulting uncapped RNA molecules (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>) were capped using a vaccinia virus capping enzyme in the absence (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>) or presence of SAM (not shown). The RNAs were incubated with either Rail (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>) or hDcp2 (not shown). The presence of either methylated (not shown) or unmethylated 5’ caps after each step was monitored by 5’ RACE followed by the cloning and sequencing of individual cDNA clones. The graphical representation of the 5’ RACE results is similar to those in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3 A</a>.</p><p id="p-44">(B) Tabular representation of the 5’ RACE experiments using either total RNA prepared from <em>K. lactis</em> IFO1267 or synthetic <em>K. lactis</em> actin mRNA capped by a vaccinia virus mRNA capping enzyme in either the presence (m<sup>7</sup>G-actin) or absence (G-actin) of SAM. RNA preparations were analyzed untreated or treated with Rai1 or hDcp2. Numbers represent the fractions of capped RNAs from all the independent cDNA clones analyzed (numbers in brackets labeled with <sup>NAC</sup>). Numbers labeled with an asterisk are replicated from <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3C</a>. NAC - number of analyzed clones.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">These results also indirectly strongly support the prediction that K2Orf3p either alone or as a part of a protein complex functions as a regular capping enzyme, exhibiting all three essential enzymatic activities necessary for complete conventional m<sup>7</sup>G cap synthesis.</p></div><div id="sec-17" class="subsection"><h3>The pGKL Promoters Determine 5’ End mRNA Capping and Non-Template Polyadenylation</h3><p id="p-46">Mapping of 5’ mRNA ends allows us to determine putative UCS sites, UCR regions and transcription start sites (TSSs) for all the pGKL1 and pGKL2 ORFs. <em>K1ORF4, K2ORF3</em> and <em>K2ORF4</em> contain two putative UCS sites each. We evaluated the sequences of all these UCS sites and refined their previously reported consensus sequence, ATNTGA (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>), by calculating the occurrences of all 4 nucleotides in the variable N position: A = 50%, G = 11%, C = 22%, and T = 17%. The TSS (+1 nucleotide) is usually located in a short window 17 to 22 nt downstream from the 3’ end of the UCS; the median distance is 19 nt, and the extremes values are 13 and 25 nt. Transcription initiates at the adenosine nucleotide. Two exceptions were present in the minor transcripts of <em>K2ORF2</em> and <em>K2ORF4</em>; they start with G and T, respectively. The minor upstream <em>K2ORF4</em> UCS ends with a guanosine nucleotide (ATATGG), an exception to the consensus sequence, and serves as a non-canonical TSS, initiating with a T that is located only 13 nt downstream from the UCS and thus suggesting a slightly different mode of transcription initiation than in other UCRs. Most of the UTRs are very short regardless of the presence or absence of 5’ non-template leaders. Exceptions to this rule are the <em>K2ORF2</em> UTR (≈ 60 nt), <em>K2ORF3</em><sub>long</sub>UTR (≈ 150 nt), <em>K2ORF3</em><sub>short</sub>UTR (≈ 60 nt) and <em>K2ORF5</em> UTR (≈ 90 nt) (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table S1A</a>). Mapping the 5’ ends of pGKL transcripts helped us to classify genes encoded by pGKL plasmids into two major groups: genes transcribed to frequently 5’-capped mRNAs with low average number of 5’ non-template adenosine nucleotides (<em>K2ORF2, K2ORF3, K2ORF4</em> and <em>K2ORF8</em>) and a larger group of genes producing transcripts less frequently 5’-capped but with a higher degree of 5’ non-template polyadenylation (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3C</a>). UCR sequences may themselves contain signals determining the 5’ end formation of the pGKL transcripts. We took advantage of <em>K. lactis</em> strains containing altered sets of linear plasmids as described in <a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2</a>. We investigated selected transcripts from the <em>K. lactis</em> IFO1267[pRKL2-1] strain (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2C</a> and <a id="xref-fig-2-15" class="xref-fig" href="#F2">D</a>) by 5’ RACE. The pRKL2-1 linear plasmid is a pGKL2 derivative that contains a G418 resistance marker under the control of <em>K1UCR2</em> and a <em>K2ORF2</em> artificially controlled by <em>K1UCR1</em> (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig. 2</a>) Indeed, we found that the presence of a cap at the 5’ end of pRKL2-1 mRNAs, as well as the presence and length of a 5’ poly(A) leader, are directly influenced by the UCR sequence used for expression. Replacement of the wild-type UCR sequence of the <em>K2ORF2</em> gene by the <em>K1ORF1</em> UCR sequence (<em>K1UCR1</em>, <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3A</a>) led to a remarkable switch from nearly fully capped and almost not 5’-polyadenylated transcripts (91.7% capped transcripts, average of added non-template adenosine nucleotides per mRNA molecule ≈ 0) to largely uncapped and 5’-polyadenylated transcripts (7.7% capped transcripts, average of added non-template adenosine nucleotides per mRNA molecule ≈ 5). These values correspond very well to the level of 5’ capping and nontemplate polyadenylation of <em>K1ORF1</em> mRNAs expressed from the pGKL1 plasmid under control of natural <em>K1UCR1</em> (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Fig. 2E</a>). The same trend was also observed for the transcripts of the resistance marker gene G418, transcription of which was controlled by <em>K1UCR2</em>. The majority of these transcripts were uncapped, but all of them were 5’ polyadenylated in exactly the same manner as natural <em>K1ORF2</em> transcripts. The same promoter had been used to drive both G418 marker gene and <em>K1ORF2</em> transcription. Additionally, the average number of nontemplate adenosine residues in the 5’ leader was comparable in both genes (G418 resistance and <em>K1ORF2</em>) controlled by <em>K1UCR2</em>. All these findings strongly support the hypothesis that differences in 5’ end formation of yeast linear plasmid mRNAs are controlled by UCR sequences and are independent of the coding sequence of a gene and its location within the pGKL plasmid system.</p><p id="p-47">Our analyses of the pGKL mRNA ends and results showing the importance of the UCR region for 5’ mRNA end formation explain the surprising results of Schickel et al., who used a glucose dehydrogenase (<em>gdh</em>) reporter gene integrated into yeast cytoplasmic linear plasmids to measure the strength of several pGKL promoters. They showed that <em>K2UCR6</em> controlling the <em>gdh</em> reporter results in almost two times higher expression than when the reporter is controlled by <em>K2UCR10</em> even though these two UCRs have exactly the same length and UCS sequence (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>). We found that 20% of <em>K2ORF6</em> transcripts possess a 5’ cap, whereas only uncapped mRNAs were found among <em>K2ORF10</em> transcripts (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3C</a>, <a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table S1A</a>). If a 5’ cap substantially increases the translation efficiency of its transcript, differences in cap occurrence frequency can account for differences in the translatability of pGKL plasmid mRNAs. A 5’ cap positively affects the translation of mRNAs carrying a 5’ poly(A) leader in a wheat germ extract (WGE) <em>in vitro</em> (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>). Because mRNA capped with a N<sup>7</sup>-unmethylated cap is inefficient in translation initiation (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>), a combination of our results and those of Schickel et al. (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">84</a>) provides more indirect evidence for the N<sup>7</sup> methylation of 5’-capped pGKL mRNAs.</p></div><div id="sec-18" class="subsection"><h3>pGKL mRNAs Do Not Contain 3’ Poly(A) Tails</h3><p id="p-48">The unique 5’ ends of the pGKL mRNAs immediately raised a question about the organization of their 3’ ends. Previously reported indications suggested that pGKL mRNAs were polyadenylated at their 3’ ends (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>), which is reasonable, especially in the case of non-capped pGKL mRNAs, because 3’ polyadenylation of eukaryotic mRNAs plays an important role in their stability (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>,<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>) and the efficiency of their utilization by cellular translation apparatus (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>–<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>). To answer this question, we performed a modified 3’ RACE where a purified and DNase I-treated total yeast RNA was 3’ tailed by an <em>E. coli</em> poly(A) polymerase using CTP instead of ATP. Following reverse transcription with an oligo(dG) anchored primer, PCR amplifications were performed with a universal anchor primer and appropriate gene-specific primers (see primer sequences in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table S2</a>). At least twenty randomly selected 3’ RACE fragments cloned in pCR4-TOPO plasmids were sequenced for each ORF. The 3’ RACE analyses of all 15 pGKL ORFs are exemplified by the results from <em>K1ORF1</em> (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3B</a>; <a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table S1C</a>), <em>K2ORF5</em> and <em>K2ORF10</em> (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table S1C</a>). Surprisingly, we found that pGKL mRNAs do not contain a 3’ poly(A) tail. Transcripts produced from the pGKL plasmids contain heterogeneous 3’ ends, suggesting the existence of more than one independent putative termination signal. Transcripts corresponding to only four ORFs are uniform at their 3’ ends. This group is exemplified by <em>K2ORF10</em> (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table S1C</a>). Four pGKL ORFs gave rise to transcripts that fell into two distinct groups with regard to their 3’ UTR lengths, suggesting two independent transcription terminators. These transcripts are exemplified by those of <em>K2ORF5</em> (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table S1C</a>). Transcripts of the remaining seven pGKL ORFs probably utilize three or more termination sites. Our initial bioinformatic analyses failed to determine the transcription termination signals. We detected only very weak RNA stem loops (ΔG from −1.0 to −4.0) at the 3’ UTR sequences that are probably not able to induce transcription termination.</p></div><div id="sec-19" class="subsection"><h3>Yeast Cap-Binding Protein eIF4E Does Not Bind pGKL mRNAs <em>In Vitro</em></h3><p id="p-49">The presence of a methylguanosine cap at their 5’ end and 3’ polyadenylation are common characteristics of eukaryotic mRNAs. Absence of these structures in the pGKL transcripts should lead to their decreased stability and low translatability (<a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">90</a>,<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>,<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>). However, this behavior clearly does not occur because pGKL genes are regularly expressed. A toxin encoded by the pGKL1 plasmid is secreted in high amounts to the culture medium ((<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>) and next paragraph) despite the transcripts coding for the toxin subunits having the lowest cap occurrence frequency among all of the pGKL transcripts (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3C</a>). To investigate this phenomenon, we designed a series of <em>in vitro</em> and <em>in vivo</em> experiments in which we have demonstrated that killer toxin mRNAs and probably other mRNAs starting with uncapped 5’ poly(A) leaders are translated in a novel cap-independent manner.</p><p id="p-50">Translation initiation factor 4E (eIF4E) plays a crucial role in eukaryotic translation initiation. It recognizes and binds m<sup>7</sup>G cap structures at the 5’ ends of the eukaryotic mRNAs and brings them to the initiating small ribosomal subunits <em>via</em> its interaction with scaffold factor eIF4G. The eIF4E protein is essential in all eukaryotes, and its deletion is also lethal in yeast (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>–<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a>). The structure and function of the cap-binding eIF4E protein from the yeast <em>S. cerevisiae</em> is very well understood. Because pGKL plasmids can be transferred and stably maintained in <em>S. cerevisiae</em> (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>), we decided to test possible interaction between the <em>S. cerevisiae</em> eIF4E cap-binding protein (S.c.-eIF4E) and pGKL mRNAs <em>in vitro</em>. We produced S.c.-eIF4E as an N-terminal GST fusion protein in <em>E. coli</em> and purified it by glutathione-Sepharose affinity chromatography. To test the possible binding of pGKL-specific mRNAs to the yeast eIF4E, we incubated DNase I-treated total RNA purified from <em>K. lactis</em> IFO1267 with the purified GST-S.c.-eIF4E fusion protein bound to the glutathione Sepharose. After 3 hours of incubation at room temperature, the slurry was extensively washed six times with a great excess of buffer I at room temperature and directly used for RT-PCRs with gene-specific primers. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a> clearly shows that pGKL mRNAs, represented by mRNA from <em>K2ORF5</em>, do not bind to yeast eIF4E <em>in vitro</em>, whereas cellular mRNAs, represented by mRNA coding for a high-affinity glucose transporter (<em>HGT1</em>), do. This result is further supported by qPCR analysis that revealed comparable abundances of <em>K2ORF5</em> and <em>HGT1</em> mRNAs in the <em>K. lactis</em> IFO1267 total RNA (data not shown). Interestingly, <em>K2ORF5</em> is not one of the pGKL mRNAs with the low frequency of m<sup>7</sup>G cap occurrence at their 5’ ends. An explanation could be that the strength of eIF4E binding to mRNA is highly influenced by the first proximal nucleotide following the terminal m<sup>7</sup>G, where the association constant <em>K<sub>a</sub></em> is much higher for purines than for pyrimidines and slightly higher for guanosine than for adenosine (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>,<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>). Many similar results have been obtained by other researchers; however, cap analogs were used to determine the <em>K<sub>a</sub></em> and <em>K<sub>d</sub></em> values in most of these studies. Adding more nucleotides after the first proximal nucleotide substantially increased eIF4E binding to the cap while still preserving the differential influence of the first proximal nucleotide on eIF4E binding (<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">99</a>). A short homopolymeric stretch of 5’ adenosine nucleotides thus might destabilize eIF4E binding to the mRNA cap. However, decreased eIF4E binding to the pGKL mRNAs does not substantially affect the performance of those mRNAs in translation, which suggests a novel mechanism of translation initiation independent of eIF4E cap-binding protein.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PGKL mRNAs are not bound to the yeast cap-binding protein eIF4E in vitro. Electrophoretic analysis of semi-quantitative RT-PCR detecting control HGT1 mRNA (upper panel) and K2ORF5 mRNA (lower panel) in samples as follows: Lane 1 - glutathione Sepharose binding GST-eIF4E fusion protein in the presence of K. lactis IFO1267 total mRNA (input); Lane 2 - same as in line 1, but reaction performed without reverse transcriptase (negative control); Lane 3 - supernatant after the first washing step (unbound mRNA); Lanes 4, 5, and 6 - supernatant after the first, third and sixth washing steps, respectively (unbound mRNA); Lane 7 - glutathione Sepharose binding complex GST-eIF4E-mRNA after the sixth washing step (bound mRNA). M - GeneRuler 100 bp DNA Ladder Plus (Thermo Scientific). PCR was performed using Taq DNA polymerase and gene-specific primers listed in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PGKL mRNAs are not bound to the yeast cap-binding protein eIF4E &lt;em&gt;in vitro&lt;/em&gt;. Electrophoretic analysis of semi-quantitative RT-PCR detecting control &lt;em&gt;HGT1&lt;/em&gt; mRNA (upper panel) and &lt;em&gt;K2ORF5&lt;/em&gt; mRNA (lower panel) in samples as follows: Lane 1 - glutathione Sepharose binding GST-eIF4E fusion protein in the presence of &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 total mRNA (input); Lane 2 - same as in line 1, but reaction performed without reverse transcriptase (negative control); Lane 3 - supernatant after the first washing step (unbound mRNA); Lanes 4, 5, and 6 - supernatant after the first, third and sixth washing steps, respectively (unbound mRNA); Lane 7 - glutathione Sepharose binding complex GST-eIF4E-mRNA after the sixth washing step (bound mRNA). M - GeneRuler 100 bp DNA Ladder Plus (Thermo Scientific). PCR was performed using Taq DNA polymerase and gene-specific primers listed in Table S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F5.medium.gif" width="440" height="191"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F5.medium.gif" width="440" height="191"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-51" class="first-child">PGKL mRNAs are not bound to the yeast cap-binding protein eIF4E <em>in vitro</em>.</p><p id="p-52">Electrophoretic analysis of semi-quantitative RT-PCR detecting control <em>HGT1</em> mRNA (upper panel) and <em>K2ORF5</em> mRNA (lower panel) in samples as follows: Lane 1 - glutathione Sepharose binding GST-eIF4E fusion protein in the presence of <em>K. lactis</em> IFO1267 total mRNA (input); Lane 2 - same as in line 1, but reaction performed without reverse transcriptase (negative control); Lane 3 - supernatant after the first washing step (unbound mRNA); Lanes 4, 5, and 6 - supernatant after the first, third and sixth washing steps, respectively (unbound mRNA); Lane 7 - glutathione Sepharose binding complex GST-eIF4E-mRNA after the sixth washing step (bound mRNA). M - GeneRuler 100 bp DNA Ladder Plus (Thermo Scientific). PCR was performed using Taq DNA polymerase and gene-specific primers listed in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table S2</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Killer Toxin Encoded by pGKL Plasmids is Translated by Cap-Independent Pathway</h3><p id="p-53">The lack of eIF4E binding to pGKL mRNAs <em>in vitro</em> prompted us to test whether these mRNAs require a functional cap-dependent pathway for their translation initiation. The reliable determination of whether a specific mRNA is translated cap-independently <em>in vivo</em> is not easy (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>). We employed yeast strains bearing conditional mutations in the <em>CDC33</em> gene, which codes for eIF4E. <em>S. cerevisiae</em> CWO4 strains carrying either a <em>cdc33-1</em> or <em>cdc33-42</em> allele as a sole source of the eIF4E translation initiation factor (kindly obtained from Prof. Michael Altman) show a strong drop in protein synthesis (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>,<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a>) and a remarkable decrease in the number of polysomes (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a>) at an elevated temperature. We decided to transfer pGKL plasmids into these strains and determine whether a reasonable amount of the functional pGKL toxin is produced at 37°C, where Cdc33-1 and Cdc33-42 proteins are not functional and translation initiation is thus impaired (<a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">101</a>,<a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">102</a>). At least three genes (<em>K1ORF2, K1ORF3</em> and <em>K1ORF4</em>) encoded by the pGKL1 plasmid require sufficient expression in the yeast cells to impart killer and immunity phenotypes. The killer toxin itself is a trimeric protein comprising non-identical alpha, beta and gamma subunits that are encoded by <em>K1ORF2</em> and <em>K1ORF4</em> (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1</a>) (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a>). Interestingly, mRNAs transcribed from all three genes belong to the group whose members have the lowest cap occurrence frequency and a high degree of 5’ polyadenylation (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3C</a>). The presence of the killer phenotype under restrictive conditions, when cellular cap-dependent translation initiation is disabled, would thus indicate that mRNAs from at least three pGKL genes can be sufficiently and coordinately expressed in a cap-independent manner.</p><p id="p-54">The artificial transfer of pGKL plasmids naturally occurring in <em>K. lactis</em> into <em>S. cerevisiae</em> is rare and difficult. In addition to other obstacles, the host mitochondria and pGKL plasmids in <em>S. cerevisiae</em> are incompatible, and the recipient strain thus must be devoid of mitochondrial DNA (ρ<sup>0</sup>) (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>). We prepared CWO4 ρ<sup>0</sup> strains solely bearing either wild-type <em>CDC33</em> or its temperature-sensitive <em>cdc33-1</em> and <em>cdc33-42</em> alleles by their long-time cultivation in the presence of ethidium bromide. To facilitate the transfer of pGKL plasmids to the new strains and to preserve their isogenic genetic background, we performed cytoduction using an incomplete mating (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>) of the CWO4 ρ<sup>0</sup> strains with a nuclear fusion-deficient (<em>kar1</em>) yeast strain (<em>S. cerevisiae</em> YAT547) carrying both pGKL1 and pGKL2 plasmids (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a>). We obtained isogenic ρ<sup>0</sup> and [pGKL1/2] <em>S. cerevisiae</em> strains on the CWO4 genetic background that differed only in their <em>CDC33</em> alleles. All these strains contain both pGKL plasmids and display a killer phenotype (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>). We required a gene coding for a yeast single-subunit protein toxin to serve as a control for cap-dependent translation. We chose a thoroughly studied K1 killer toxin encoded by the M1 viral satellite dsRNA from the yeast <em>S. cerevisiae</em>. We prepared the M1 cDNA, cloned a gene coding for the prepro-K1 killer toxin into a yeast shuttle expression plasmid (<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a>) under the control of the strong constitutive TPI promoter (plasmid pYX212::M1) and introduced this expression plasmid into the <em>S. cerevisiae</em> CWO4 ρ<sup>0</sup> strains carrying all three <em>CDC33</em> alleles described above. The resulting strains display K1 killer and immunity phenotypes at the permissive temperature (24°C) (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6B</a>). The experiment depicted in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6</a> shows a comparison of the production of the <em>K. lactis</em> toxin expressed naturally from the pGKL1 plasmid and the K1 toxin expressed artificially from the nuclear plasmid pYX212::M1. The K1 toxin gene was transcribed by RNA polymerase II in the nucleus and thus provided a 5’ m<sup>7</sup>G-capped and 3’ polyadenylated mRNA control. We cultivated all the CWO4 ρ<sup>0</sup> [pGKL1/2] and CWO4 ρ<sup>0</sup> [pYX212::M1] strains in parallel to the exponential phase at the permissive temperature (24°C). All the strains were then harvested and briefly washed three times with an excess of the fresh medium prewarmed to 24°C. Half of each culture was further cultivated at 24°C. The second half was rapidly transferred to fresh medium prewarmed to 37°C and further cultivated at this elevated temperature. At this point, initial samples marked zero were taken for biomass measurements and the determination of toxin activity in the culture medium. Toxin activities were undetected in all samples at time zero and provided clear evidence that all cultures were washed sufficiently and that the toxin activities present in samples taken after 3, 6 and 12 hours of cultivation corresponded to proteins produced only in the course of the experiment (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6</a>). The production of the <em>K. lactis</em> killer toxin in <em>S. cerevisiae</em> strains bearing pGKL plasmids was almost comparable under permissive and restrictive conditions, regardless of whether the wild-type or temperature-sensitive <em>CDC33</em> allele was present (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6A</a>; 37°C). Production of the K1 toxin in the <em>cdc33-1</em> and <em>cdc33-42</em> yeast strains containing the pYX212::M1 plasmid vanished rapidly at 37°C, whereas the wild-type Cdc33 cap-binding protein permitted toxin production at both 24 and 37°C (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6B</a>). The presence of the temperature-sensitive eIF4E proteins in <em>cdc33-1</em> and <em>cdc33-42</em> yeast strains was documented by their restricted growth at 37°C. Both strains containing the wild-type allele of the <em>CDC33</em> gene grew well at both 24 and 37°C (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6</a>). The growth of CWO4 <em>CDC33</em>, ρ<sup>0</sup> [pYX212::M1] was slightly slower at 24°C than at 37°C. The most likely reason is that K1 killer toxin is more active at 24°C than at 37°C, even though we cloned the coding sequence of the K1 super-killer variant that shows enhanced thermal stability (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a>). We previously showed that K1 preprotoxin produced artificially from the nuclear plasmid does not fully support the immunity phenotype of the host cells (<a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">106</a>). Higher K1 toxin stability at 24°C thus probably led to the slightly slower growth. However, both cultures eventually reached comparable biomass, in contrast to the strains bearing conditional <em>cdc33</em><sup>ts</sup> alleles and cultured at a restrictive temperature (37°C), which completed the initiated cell division and ceased growth (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Fig. 6B</a>). The activity of the TPI promoter used to produce K1 toxin from the pYX212::M1 plasmid declines at the end of exponential growth (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a>), and this phenomenon is even more profound in the version of the TPI promoter used in the pYX212 plasmid (unpublished data). The growth of the CWO4 <em>CDC33</em>, ρ<sup>0</sup> [pYX212::M1] strain is slightly faster at 37°C than at 24°C, causing the cells to reach early stationary phase in 8 hours (instead of 12 at 24°C); this difference explains why K1 toxin production in the <em>CDC33</em> wild-type strain ceases in samples grown for 12 hours at 37°C (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Fig. 6B</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Killer toxin encoded by pGKL plasmids is translated by the cap-independent pathway. (A) Production of pGKL killer toxin in S. cerevisiae CWO4&#x3C1;0 strains bearing cytoplasmic pGKL plasmids and different CDC33 alleles. The pGKL toxin-coding mRNAs do not contain 3&#x2019; poly(A) tails and are mostly 5&#x2019; uncapped and 5&#x2019; polyadenylated. (B) Production of K1 killer toxin in control S. cerevisiae CWO4&#x3C1;0 strains containing pYX212::M1 plasmid and different CDC33 alleles. The control mRNA coding for K1 toxin is transcribed in the nucleus by RNA polymerase II. In both cases, the results are shown for S. cerevisiae CWO4&#x3C1;0 strains bearing either the wild-type CDC33 (eIF4E) gene (CDC33wt) or temperature-sensitive mutations in the CDC33 (eIF4E) gene (cdc33-1 and cdc33-42 alleles) grown under both permissive (24&#xB0;C) and non-permissive (37&#xB0;C) conditions. Aliquots were taken at 0, 3, 6 and 12 hours, and the culture medium was filter-sterilized, diluted 10x, and assayed for the presence of the killer toxin activity by an agar well diffusion test using a lawn of S. cerevisiae S6/1 sensitive strain cells. Growth curves of all strains at permissive (blue) and non-permissive (purple) temperatures are shown below each of the corresponding sets of killer toxin experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Killer toxin encoded by pGKL plasmids is translated by the cap-independent pathway. (A) Production of pGKL killer toxin in &lt;em&gt;S. cerevisiae&lt;/em&gt; CWO4&#x3C1;&lt;sup&gt;0&lt;/sup&gt; strains bearing cytoplasmic pGKL plasmids and different &lt;em&gt;CDC33&lt;/em&gt; alleles. The pGKL toxin-coding mRNAs do not contain 3&#x2019; poly(A) tails and are mostly 5&#x2019; uncapped and 5&#x2019; polyadenylated. (B) Production of K1 killer toxin in control &lt;em&gt;S. cerevisiae&lt;/em&gt; CWO4&#x3C1;&lt;sup&gt;0&lt;/sup&gt; strains containing pYX212::M1 plasmid and different &lt;em&gt;CDC33&lt;/em&gt; alleles. The control mRNA coding for K1 toxin is transcribed in the nucleus by RNA polymerase II. In both cases, the results are shown for &lt;em&gt;S. cerevisiae&lt;/em&gt; CWO4&#x3C1;&lt;sup&gt;0&lt;/sup&gt; strains bearing either the wild-type &lt;em&gt;CDC33&lt;/em&gt; (eIF4E) gene (&lt;em&gt;CDC33&lt;/em&gt;wt) or temperature-sensitive mutations in the &lt;em&gt;CDC33&lt;/em&gt; (eIF4E) gene (&lt;em&gt;cdc33-1&lt;/em&gt; and &lt;em&gt;cdc33-42&lt;/em&gt; alleles) grown under both permissive (24&#xB0;C) and non-permissive (37&#xB0;C) conditions. Aliquots were taken at 0, 3, 6 and 12 hours, and the culture medium was filter-sterilized, diluted 10x, and assayed for the presence of the killer toxin activity by an agar well diffusion test using a lawn of &lt;em&gt;S. cerevisiae&lt;/em&gt; S6/1 sensitive strain cells. Growth curves of all strains at permissive (blue) and non-permissive (purple) temperatures are shown below each of the corresponding sets of killer toxin experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F6.medium.gif" width="318" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F6.medium.gif" width="318" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-55" class="first-child">Killer toxin encoded by pGKL plasmids is translated by the cap-independent pathway.</p><p id="p-56">(A) Production of pGKL killer toxin in <em>S. cerevisiae</em> CWO4ρ<sup>0</sup> strains bearing cytoplasmic pGKL plasmids and different <em>CDC33</em> alleles. The pGKL toxin-coding mRNAs do not contain 3’ poly(A) tails and are mostly 5’ uncapped and 5’ polyadenylated.</p><p id="p-57">(B) Production of K1 killer toxin in control <em>S. cerevisiae</em> CWO4ρ<sup>0</sup> strains containing pYX212::M1 plasmid and different <em>CDC33</em> alleles. The control mRNA coding for K1 toxin is transcribed in the nucleus by RNA polymerase II.</p><p id="p-58">In both cases, the results are shown for <em>S. cerevisiae</em> CWO4ρ<sup>0</sup> strains bearing either the wild-type <em>CDC33</em> (eIF4E) gene (<em>CDC33</em>wt) or temperature-sensitive mutations in the <em>CDC33</em> (eIF4E) gene (<em>cdc33-1</em> and <em>cdc33-42</em> alleles) grown under both permissive (24°C) and non-permissive (37°C) conditions. Aliquots were taken at 0, 3, 6 and 12 hours, and the culture medium was filter-sterilized, diluted 10x, and assayed for the presence of the killer toxin activity by an agar well diffusion test using a lawn of <em>S. cerevisiae</em> S6/1 sensitive strain cells. Growth curves of all strains at permissive (blue) and non-permissive (purple) temperatures are shown below each of the corresponding sets of killer toxin experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">The most straightforward explanation of the obtained results is that whereas in this experiment, K1 toxin mRNA is transcribed from the shuttle plasmid under the control of a TPI promoter by the nuclear RNA polymerase II and is thus 5’ capped, 3’ polyadenylated, and translated exclusively by the canonical cap-dependent pathway, pGKL1 toxin mRNAs retain their ability to be translated even when 5’ mRNA cap recognition is severely impaired. The pGKL1 mRNAs are transcribed by the pGKL plasmid-specific RNA polymerase encoded by <em>K2ORF6/7</em> and contain unusual 5’ stretches of non-template adenosine nucleotides rarely linked with the terminal m<sup>7</sup>G caps (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figs. 1</a> and <a id="xref-fig-3-14" class="xref-fig" href="#F3">3C</a>). Our experiments, both <em>in vitro</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a>) and <em>in vivo</em> (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Fig. 6</a>), strongly suggest that these 5’ terminal structures of the pGKL mRNAs support a novel manner of translation initiation independent of the active eIF4E protein. The active translation of 5’ polyadenylated mRNAs, which do not bear 5’ caps and 3’ poly(A) tails, is further supported by our observation that these mRNAs are present in polysomes in both the yeast <em>K. lactis</em> IFO1267 naturally bearing pGKL plasmids and the engineered <em>S. cerevisiae</em> CWO4 ρ<sup>0</sup> [pGKL1/2] strains (data not shown).</p></div><div id="sec-21" class="subsection"><h3>Pab1 and Lsm1 Deletions Support the Translation of pGKL mRNAs</h3><p id="p-60">Poly(A) tracts within the 5’ UTRs and the 5’ poly(A) leaders of the eukaryotic and viral mRNAs have been identified as enhancers of mRNA translation in various examples (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">85</a>,<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a>,<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a>) as well as translational repressors in other cases (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a>,<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">112</a>). Some studies even show that poly(A) stretches within 5’ UTRs may serve as translational enhancers until they reach a certain threshold length. Further increases in the number of consecutive adenosine nucleotides beyond the threshold length lead to translational repression (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">113</a>). Poly(A)-binding protein 1 (PABP1) plays an important role in many of these studies and biological examples. PABP1 is an abundant general translation initiation factor that binds to 3’ poly(A) tails, a common mark of most eukaryotic mRNAs (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">114</a>). In addition, PABP1 serves as a specific regulatory protein and has an important role in mRNA stability and turnover. The importance of PABP1 in cellular translation and other vital processes is underlined by growing evidence that it is targeted by a wide range of viruses aiming to take over host protein synthesis or to escape the cellular antiviral defense (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">115</a>–<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">118</a>).</p><p id="p-61">To investigate the possible effects of yeast PABP1 protein (Pab1) on the expression of pGKL genes, we deleted the <em>PAB1</em> gene from <em>K. lactis</em>. Because no experimental data describing <em>PAB1</em> deletion in the yeast <em>K. lactis</em> were available, we followed a strategy used for the deletion of <em>PAB1</em> in the yeast <em>S. cerevisiae</em> (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">119</a>). Deletion of <em>PAB1</em> gene is lethal in <em>S. cerevisiae</em> (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">94</a>,<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">120</a>) but this lethality can be suppressed by the simultaneous deletion of the gene coding for Pab1p-binding protein (Pbp1) (<a id="xref-ref-119-2" class="xref-bibr" href="#ref-119">119</a>). We disrupted the <em>PBP1</em> gene in the chromosomal DNA of the <em>K. lactis</em> IFO1267 strain using a loxP-G418-loxP system (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">121</a>) and obtained the <em>K. lactis</em> IFO1267 <em>pbp1</em>::G418 strain. The G418 cassette was removed from the genome by an artificially produced Cre-recombinase, providing the strain <em>K. lactis</em> IFO1267 <em>pbp1Δ</em>. We used the same strategy to introduce a <em>PAB1</em> deletion and to obtain the required strain, <em>K. lactis</em> IFO1267 <em>pbp1Δ pab1Δ</em>. Another protein that stabilizes some viral 5’ poly(A) tracts and thus may also affect translation of the respective mRNAs is Lsm1 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>). We disrupted the <em>LSM1</em> gene in the <em>K. lactis</em> IFO1267 genome using a strategy similar to that described for <em>PBP1</em> deletion. The <em>K. lactis pbp1Δ pab1Δ</em> double-mutant strain is viable but exhibits a slow-growth phenotype, similar to the reported behavior in <em>S. cerevisiae</em> (<a id="xref-ref-119-3" class="xref-bibr" href="#ref-119">119</a>).</p><p id="p-62">Deletions of <em>LSM1</em> and <em>PBP1</em> from <em>S. cerevisiae</em> decrease growth rates (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">122</a>,<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">123</a>). In contrast, we detected only minor decrease in the growth rate of the <em>lsm1Δ</em> strain and no difference in the growth rate of the <em>pbp1Δ</em> strain from the growth rate of their parental wild-type strain, <em>K. lactis</em> IFO1267 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7A</a>). Disruption of any of the three genes tested enhanced production of the toxin encoded by the linear cytoplasmic plasmid pGKL1, and the production was most enhanced in the <em>lsm1Δ</em> strain. The increased toxin production by the slow-growing <em>K. lactis</em> IFO1267 <em>pbp1Δ pab1Δ</em> strain was more noticeable when toxin production to the culture medium was quantified as the diameter of the inhibition zone on the lawn of the sensitive yeast strain per production cell (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figs. 7A</a>, S1) The Lsm1 protein is part of the complex of cellular mRNA decapping activators employed by positive-strand viruses to promote their translation and replication (reviewed in (<a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">124</a>)) and might stabilize orthopoxviral mRNAs containing 5’ poly(A) leaders (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>). Despite the many similarities between poxviruses and the apparatuses of pGKL transcription and RNA modification (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>), the simplest explanation for increased pGKL toxin production in the <em>lsm1Δ</em> strain might be the stabilization of pGKL mRNAs in the cell due to prominent involvement of the Lsm1 protein in a cytoplasmic deadenylation-dependent mRNA decay pathway (reviewed in (<a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">125</a>)). We calculated the lengths of the continuous 5’ poly(A) leaders of the pGKL mRNAs while considering both non-template and template-coded adenosine nucleotides. While the maximal length of the 5’ poly(A) leaders can reach 26 nucleotides, their average and median lengths are much lower. The median lengths of 5’ poly(A) leaders in transcripts from only two genes, <em>K1ORF3</em> and <em>K2ORF7</em>, are 12 adenosine nucleotides. Transcripts from <em>K1ORF1, K1ORF2</em> and <em>K2ORF11</em> have 5’ poly(A) leaders with median lengths of 10 adenosine nucleotides. All other pGKL transcripts have shorter 5’ poly(A) leaders (<a id="xref-table-wrap-1-18" class="xref-table" href="#T1">Tables S1A</a> and S5). These results agree very well with the genome-wide study of Xia et al., who showed that the presence of poly(A) stretches in 5’ UTRs in <em>S. cerevisiae</em> transcripts positively correlates with enhanced protein synthesis until the length of the 5’ poly(A) regions reaches at least 12 consecutive adenosine nucleotides (<a id="xref-ref-113-2" class="xref-bibr" href="#ref-113">113</a>), which is also the optimal length for the binding site of yeast Pab1 (<a id="xref-ref-120-2" class="xref-bibr" href="#ref-120">120</a>). Pab1 might function as a negative regulator that can, due to the documented sharp increase in its binding affinity to poly(A) stretches ranging from 8 to 12 adenosine residues in length (<a id="xref-ref-120-3" class="xref-bibr" href="#ref-120">120</a>), further modulate the expression of pGKL genes. Such Pab1 activity was suggested for larger set of yeast genes and can be demonstrated in mRNA coding for cyclin <em>PCL5</em>, which contains the longest yeast 5’ UTR poly(A) region and shows inefficient translation and extremely low ribosomal occupancy (<a id="xref-ref-113-3" class="xref-bibr" href="#ref-113">113</a>). In mammalian cells, PABP1 binds as a part of the negative autoregulatory complex to the adenosine-rich elements within the 5’ UTR of its own mRNA (<a id="xref-ref-111-2" class="xref-bibr" href="#ref-111">111</a>). mRNAs containing long uncapped poly(A) leaders (25 nt long) are efficiently translated in WGE lysates and, in contrast to other mRNAs, do not commonly decrease in translation rate at high mRNA concentrations (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">85</a>). This behavior suggests a lower dependence of such mRNAs on translation factors. Indeed, mRNAs containing long 5’ poly(A) leaders can efficiently enter translation, even in the absence of the otherwise essential translation factors eIF4F, eIF3 and PABP1 (<a id="xref-ref-109-2" class="xref-bibr" href="#ref-109">109</a>). Late poxviral mRNAs containing long 5’ poly(A) leaders have a low requirement for eIF4F and might have a low requirement for PABP1 (<a id="xref-ref-113-4" class="xref-bibr" href="#ref-113">113</a>,<a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">126</a>). Interestingly, pGKL1 mRNAs coding for toxin and immunity phenotypes contain a higher fraction of mRNAs with poly(A) leaders that are at least 12 residues long (<a id="xref-table-wrap-1-19" class="xref-table" href="#T1">Tables S1A</a> and S5). These mRNAs, which perhaps normally have Pab1 bound to their 5’ UTRs, may become unblocked and available for translation in the <em>K. lactis pbp1Δ pab1Δ</em> strain, thus increasing overall pGKL toxin production. The control strain, <em>K. lactis pbp1Δ</em>, showed no growth defects and the lowest enhancement of pGKL toxin production <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7A</a>. Similarly to its human ortholog ataxin-2, yeast Pbp1 protein is involved in a wide range of cellular process; malfunction of ataxin-2 causes serious diseases in humans, including neurodegenerative spinocerebellar ataxia type II (<a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">127</a>). Pbp1 (ataxin-2) functions are particularly associated with mRNA stability and translation, and Pbp1 is a negative regulator of a poly(A) nuclease (<a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">128</a>), which might be a reason for the enhanced production of the pGKL toxin in the <em>pbp1Δ</em> strain. To exclude the possibility of changes in 5’ poly(A) leader length and m<sup>7</sup>G capping of pGKL transcripts in the <em>pbp1Δ pab1Δ</em> double-mutant strain, we analyzed 5’ ends from <em>K1ORF2</em> and <em>K1ORF3</em> mRNAs coding for the toxin α/β-subunit and immunity, respectively, by the 5’ RACE. As clearly seen in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7B</a>, deletion of the <em>PAB1</em> gene does not remarkably affect the structure of the 5’ ends of the pGKL mRNAs.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Translation of the K. lactis killer toxin is enhanced in LSM1 and PAB1 deletion strains. (A) Growth curves and production of the pGKL1 killer toxin from K. lactis IFO1267 (blue), IFO1267 pbp1&#x394; (red), IFO1267 lsm1&#x394; (yellow) and IFO1267 pbp1&#x394; pab1&#x394; (green) strains. All strains were cultivated in YPD medium at 28&#xB0;C for approximately 80 hours, and their OD600 nm and toxin production were monitored. Production of the pGKL1 killer toxin into culture medium was assayed by a well diffusion test on YPD agar plates with a lawn of the S. cerevisiae S6/1 sensitive strain (Fig. S1). Toxin levels in a culture medium in early stationary phase (~51 hours), when all the cultures reached comparable OD600, are depicted. Killer zones were measured using 100x diluted filter-sterilized culture medium, and the average width of the killer zone per cell was calculated for each strain, plotted with standard deviation and statistically analyzed (** p &lt; 0.01, * p &lt; 0.05) (B) 5&#x2019; RACE analysis of pGKL mRNAs from the wild-type and PAB1 deletion strains. 5&#x2019; RACE analysis of K1ORF2 and K1ORF3 mRNAs was performed using total RNA prepared from K. lactis IFO1267 and its double-deletion mutant K. lactis IFO1267 pbp1&#x394; pab1&#x394;. The individual columns represent the following quantities: 1/ percentage of all transcripts containing 5&#x2019; non-template adenosine nucleotides; 2/ percentage of all transcripts containing a 5&#x2019; cap structure; 3/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 4/ the number of 5&#x2019; non-template adenosine nucleotides found in the corresponding mRNAs. Data related to the wild-type strain K. lactis IFO1267 are replicated from Fig. 3C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-531210269" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Translation of the &lt;em&gt;K. lactis&lt;/em&gt; killer toxin is enhanced in &lt;em&gt;LSM1&lt;/em&gt; and &lt;em&gt;PAB1&lt;/em&gt; deletion strains. (A) Growth curves and production of the pGKL1 killer toxin from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 (blue), IFO1267 &lt;em&gt;pbp1&#x394;&lt;/em&gt; (red), IFO1267 &lt;em&gt;lsm1&#x394;&lt;/em&gt; (yellow) and IFO1267 &lt;em&gt;pbp1&#x394; pab1&#x394;&lt;/em&gt; (green) strains. All strains were cultivated in YPD medium at 28&#xB0;C for approximately 80 hours, and their OD&lt;sub&gt;600 nm&lt;/sub&gt; and toxin production were monitored. Production of the pGKL1 killer toxin into culture medium was assayed by a well diffusion test on YPD agar plates with a lawn of the &lt;em&gt;S. cerevisiae&lt;/em&gt; S6/1 sensitive strain (Fig. S1). Toxin levels in a culture medium in early stationary phase (~51 hours), when all the cultures reached comparable OD&lt;sub&gt;600&lt;/sub&gt;, are depicted. Killer zones were measured using 100x diluted filter-sterilized culture medium, and the average width of the killer zone per cell was calculated for each strain, plotted with standard deviation and statistically analyzed (** p &lt; 0.01, * p &lt; 0.05) (B) 5&#x2019; RACE analysis of pGKL mRNAs from the wild-type and &lt;em&gt;PAB1&lt;/em&gt; deletion strains. 5&#x2019; RACE analysis of &lt;em&gt;K1ORF2&lt;/em&gt; and &lt;em&gt;K1ORF3&lt;/em&gt; mRNAs was performed using total RNA prepared from &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 and its double-deletion mutant &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 &lt;em&gt;pbp1&#x394; pab1&#x394;&lt;/em&gt;. The individual columns represent the following quantities: 1/ percentage of all transcripts containing 5&#x2019; non-template adenosine nucleotides; 2/ percentage of all transcripts containing a 5&#x2019; cap structure; 3/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 4/ the number of 5&#x2019; non-template adenosine nucleotides found in the corresponding mRNAs. Data related to the wild-type strain &lt;em&gt;K. lactis&lt;/em&gt; IFO1267 are replicated from Fig. 3C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F7.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F7.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/17/325316/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-63" class="first-child">Translation of the <em>K. lactis</em> killer toxin is enhanced in <em>LSM1</em> and <em>PAB1</em> deletion strains.</p><p id="p-64">(A) Growth curves and production of the pGKL1 killer toxin from <em>K. lactis</em> IFO1267 (blue), IFO1267 <em>pbp1Δ</em> (red), IFO1267 <em>lsm1Δ</em> (yellow) and IFO1267 <em>pbp1Δ pab1Δ</em> (green) strains. All strains were cultivated in YPD medium at 28°C for approximately 80 hours, and their OD<sub>600 nm</sub> and toxin production were monitored. Production of the pGKL1 killer toxin into culture medium was assayed by a well diffusion test on YPD agar plates with a lawn of the <em>S. cerevisiae</em> S6/1 sensitive strain (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. S1</a>). Toxin levels in a culture medium in early stationary phase (~51 hours), when all the cultures reached comparable OD<sub>600</sub>, are depicted. Killer zones were measured using 100x diluted filter-sterilized culture medium, and the average width of the killer zone per cell was calculated for each strain, plotted with standard deviation and statistically analyzed (** p &lt; 0.01, * p &lt; 0.05)</p><p id="p-65">(B) 5’ RACE analysis of pGKL mRNAs from the wild-type and <em>PAB1</em> deletion strains. 5’ RACE analysis of <em>K1ORF2</em> and <em>K1ORF3</em> mRNAs was performed using total RNA prepared from <em>K. lactis</em> IFO1267 and its double-deletion mutant <em>K. lactis</em> IFO1267 <em>pbp1Δ pab1Δ</em>. The individual columns represent the following quantities: 1/ percentage of all transcripts containing 5’ non-template adenosine nucleotides; 2/ percentage of all transcripts containing a 5’ cap structure; 3/ the average number (shown as the mean and median) of non-template adenosine nucleotides per mRNA molecule; 4/ the number of 5’ non-template adenosine nucleotides found in the corresponding mRNAs. Data related to the wild-type strain <em>K. lactis</em> IFO1267 are replicated from <a id="xref-fig-3-15" class="xref-fig" href="#F3">Fig. 3C</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-22"><h2 class="">Conclusions</h2><p id="p-66">We employed pGKL plasmids from the yeast <em>Kluyveromyces lactis</em> as a model to investigate and present here a complex view of the largely unknown transcriptome of yeast cytoplasmic linear plasmids. Although previous experiments reported the binding of pGKL mRNAs to an oligo(dT) column (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">18</a>), we show that the pGKL transcripts are not polyadenylated at their 3’ ends; however, they frequently contain uncapped 5’ poly(A) leaders that are not complementary to the plasmid DNA. The degree of 5’ capping and/or 5’ polyadenylation is specific to each gene and is most likely controlled by short UCRs preceding the corresponding AUG start codon. This transcriptome analysis allowed us to refine the description of the pGKL promoters and revealed new alternative promoters for the <em>K1ORF4, K2ORF3</em> and <em>K2ORF4</em> genes. We also provide evidence that <em>K2ORF3</em> codes for the long sought mRNA cap guanine-N<sup>7</sup> methyltransferase and that capped pGKL transcripts contain N<sup>7</sup>-methylated caps at their 5’ ends. Translation of pGKL transcripts is independent of eIF4E and Pab1 translation factors. With regard to yeast (<a id="xref-ref-113-5" class="xref-bibr" href="#ref-113">113</a>) and mammalian (<a id="xref-ref-111-3" class="xref-bibr" href="#ref-111">111</a>,<a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">112</a>) mRNAs containing long poly(A) stretches in their 5’ UTRs, we suggest that Pab1 serves as a negative regulator of the pGKL mRNAs containing 5’ poly(A) leaders. It is tempting to speculate that differences in the median lengths of the 5’ mRNA poly(A) leaders of each gene (<a id="xref-table-wrap-1-20" class="xref-table" href="#T1">Tables S1A</a>, S5) might reflect the degree of inhibition of the corresponding gene expression by Pab1, which might also be an evolutionary force driving the selection of the appropriate UCRs. We hypothesize that the small and extremely compact genome of pGKL plasmids uses different levels of 5’ capping and 5’ polyadenylation to finely tune the expression of each gene to proper levels to facilitate the formation of functional protein complexes and to keep pGKL gene expression in harmony with the host cell. Some initial evidence supporting this hypothesis comes from comparing our data with that of Schickel et al., as discussed earlier (<a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">84</a>). We also provide data further supporting an evolutionary relationship between a group of the cytoplasmic yeast linear plasmids and poxviruses (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>). This connection is fascinating because members of the <em>Poxviridae</em> family have been found only in vertebrates and arthropods, and some of them cause highly contagious and serious diseases of humans and animals. Parallel studies of yeast cytoplasmic linear plasmids and poxviruses may lead us to the origin of eukaryotic viruses. The pGKL plasmids could also help us better understand basic processes and host cell-virus interactions that are not easily studied in mammalian systems. Last but not least, <em>K. lactis</em> is one of the most important yeast for research and industrial biotechnology (<a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">129</a>), and our results suggest that <em>lsm1Δ</em> strains can greatly improve the production of heterologous proteins from pGKL-based expression plasmids.</p></div><div class="section" id="sec-23"><h2 class="">Funding</h2><p id="p-67">This work was supported by the Czech Science Foundation [grant No. GBP305/12/G034], by the Charles University institutional project [SVV-260426] and by ELIXIR CZ research infrastructure project MEYS [Grant No: LM2015047] including access to computing and storage facilities.</p></div><div class="section" id="sec-25"><h2 class="">Supplemental Experimental Procedures</h2><div class="section" id="sec-26"><h3>Plasmid Construction</h3><p id="p-72">The <em>CDC33</em> (eIF4E) gene was amplified from <em>S. cerevisiae</em> genomic DNA using Pfu DNA polymerase (Fermentas) with eIF4Ef and eIF4Er primers containing NcoI and HindIII restriction sites as follows: 5 min at 95°C; then 25 cycles of 30 sec at 94°C, 30 sec at 55°C, and 2 min at 72°C; and finally, 10 min at 72°C. The PCR cassette was digested and inserted into the NcoI and HindIII restriction sites of pGEX-4T2 (GE Healthcare) to generate a pGEX4T2::CDC33 construct. A pYX212 yeast shuttle plasmid (Ingenius) was digested with EcoRI and XhoI restriction endonucleases. A sequence coding for K1 toxin was excised from a pYX213::M1 (<a id="xref-ref-142-1" class="xref-bibr" href="#ref-142">Valis et al., 2006</a>) plasmid using the same restriction enzymes and ligated into the digested pYX212 plasmid to generate the plasmid pYX212::M1. <em>K2ORF3</em> was amplified from <em>K. lactis</em> pGKL2 plasmid DNA using Pfu DNA polymerase (Fermentas) with the NLS-ORF3 forward and the klorf3_2 reverse primers as follows: 5 min at 95°C; then 5 cycles of 30 sec at 94°C, 30 sec at 55°C, and 2 min at 72°C; and finally, 10 min at 72°C. One microliter of this reaction mixture was used for subsequent PCR amplification using Pfu DNA polymerase with ORF3_NLS_BamF and the klorf3_2 reverse primers as follows: 5 min at 95°C; then 20 cycles of 30 sec at 94°C, 30 sec at 55°C, and 2 min at 72°C; and finally, 10 min at 72°C. The resulting PCR product was purified using agarose electrophoresis and cloned into a pCR4-TOPO plasmid (Invitrogen). An NLS-ORF3 cassette from a pCR4-TOPO plasmid was excised with BamHI and SalI restriction endonucleases and inserted into the corresponding sites of the pYX212 or pYX213 (Ingenius) plasmids to generate pYX212::NLS-ORF3 or pYX213::NLS-ORF3 constructs, respectively. All clones were verified by restriction endonuclease digestion and sequencing. All primers used in this study are listed in <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table S2</a>.</p></div><div class="section" id="sec-27"><h3>Assay of Enzyme-GMP Complex Formation</h3><p id="p-73">A reaction mixture (20 μl) containing 50 mM Tris-HCl (pH 8.0); 2 mM DTT; 20 mM MgCl<sub>2</sub>; [α- <sup>32</sup>P]GTP and purified K2Oof3p was incubated at 30°C for 60 min. The reaction was quenched with SDS-PAGE loading buffer, and the products were resolved by SDS-PAGE. The K2Orf3p-[<sup>32</sup>P]GMP adduct was visualized by autoradiography of the dried gel.</p></div><div class="section" id="sec-28"><h3>Removal of Mitochondrial DNA by Ethidium Bromide Treatment</h3><p id="p-74">Isogenic <em>S. cerevisiae</em> strains bearing the wild-type <em>CDC33</em> gene (CWO4<em>CDC33</em>wt) and its temperature-sensitive mutations (CWO4<em>cdc33-1</em> and CWO4<em>cdc33-42</em>, for exact genotypes see <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table S3</a>) (<a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">Altmann et al., 1989</a>; Altmann and <a id="xref-ref-132-1" class="xref-bibr" href="#ref-132">Trachsel, 1989</a>) were cultivated for two days in synthetic drop-out minimal medium (SD) that lacked leucine and tryptophan (SD-TL) and contained ethidium bromide at a final concentration of 25 μg per ml. Afterwards, these cultures were diluted five thousand times, cultivated again for two days and then diluted and cultivated once more. Loss of mitochondria was verified by the lack of colony growth on medium with glycerol as the sole carbon source and by DAPI staining followed by fluorescence microscopy.</p></div><div class="section" id="sec-29"><h3>Transformation of Yeast Cells</h3><p id="p-75">All yeast transformations with plasmid DNA were performed with the one-step LiCl method (<a id="xref-ref-134-1" class="xref-bibr" href="#ref-134">Gietz and Woods, 2002</a>). The cells were grown in a shaker at 28°C in SD minimal medium lacking selected amino acids or nucleotide bases to ensure plasmid maintenance. Transformation using a PCR fragment for homologous recombination was carried out in the same way with the exception of including a five-hour incubation under nonselective conditions immediately after transformation prior to plating.</p></div><div class="section" id="sec-30"><h3>Killer Production in Modified CWO4 Yeast Strains</h3><p id="p-76">Tested strains (50 ml) were cultured in an appropriate medium at 24°C with shaking. When the A<sub>600</sub> reached 0.9, cells were harvested and washed three times with the same medium preheated to 24°C. One-half of the culture (25 ml) was mixed with 25 ml of fresh medium preheated to 24°C and then cultivated at 24°C for more than 20 hours. The second half of the tested culture (25 ml) was mixed with 25 ml of fresh medium preheated to 52°C and then cultivated at 37°C for more than 20 hours. Aliquots (2 ml) were taken at 0, 3, 6 and 12 hours. Samples were harvested, sterilized using 0.2 μm filters and assayed for the presence of the killer toxin activity in the culture medium by an agar well diffusion assay using <em>S. cerevisiae</em> S6/1 as a sensitive strain.</p></div><div class="section" id="sec-31"><h3>Assay of killer toxin activity</h3><p id="p-77">Filter sterilized culture medium was tested for the presence of the killer toxin activity by the agar well diffusion assay using <em>S. cerevisiae</em> S6/1 as a sensitive strain. Approximately 2×10<sup>5</sup> of sensitive yeast cells were plated onto OSS1 agar plates (5.3% worth agar; 0.7% agar; 1% glucose; 1M sorbitol; 0.002% methylene blue; buffered to pH 4.7 with sodium citrate) for testing of M1 killer toxin activity or onto YPD plates (1% yeast extract; 2% peptone; 2% glucose; 2% of agar) for testing of pGKL1 killer toxin activity. Wells were made with an 8 mm diameter cork borer and 100 μl of filter sterilized culture medium or 100 μl of serial dilution of the filter sterilized culture medium was pipetted into well. Inhibition zones were measured after 48 hours of incubation at 24°C.</p></div><div class="section" id="sec-32"><h3>Statistical analysis</h3><p id="p-78">Results of at least five independent killer toxin activity measurements assayed by the well diffusion method were statistically evaluated using one-way ANOVA followed by post hoc Tukey’s HSD test and further confirmed with Scheffe multiple comparison. The normal distribution of the data was confirmed by Shapiro-Wilk test.</p></div><div class="section" id="sec-33"><h3>Expression and Purification of mRNA Decapping Enzymes</h3><p id="p-79">pET28a::hDcp2 (<a id="xref-ref-143-1" class="xref-bibr" href="#ref-143">Wang et al., 2002</a>) and pET26b::Rai1 (<a id="xref-ref-146-1" class="xref-bibr" href="#ref-146">Xiang et al., 2009b</a>) (generous gifts from M. Kiledjian) expression plasmids were transformed into the <em>E.coli</em> Rosetta<sup>™</sup> expression strain (Novagen). The corresponding mRNA decapping enzymes were expressed and purified according to previously published instructions (<a id="xref-ref-139-1" class="xref-bibr" href="#ref-139">Piccirillo et al., 2003</a>; <a id="xref-ref-146-2" class="xref-bibr" href="#ref-146">Xiang et al., 2009b</a>).</p></div><div class="section" id="sec-34"><h3><em>In Vitro</em> Processing of mRNAs by Decapping Enzymes</h3><p id="p-80">In a 100 μl reaction, 500 ng of total RNA purified from <em>K. lactis</em> IFO1267 or 4.1 pmol of <em>in vitro</em> transcribed and capped RNA were incubated with 0.25 μg of recombinant hDcp2 or 0.25 μg of recombinant Rai1 enzyme in 1x decapping buffer (10 mM Tris-HCl, pH 7.5; 100 mM potassium acetate; 2 mM magnesium acetate; 0.5 mM MnCl<sub>2</sub>; 2 mM dithiothreitol and 0.1 mM spermine) at 37°C for 30 min. After incubation, RNA was purified using an RNA Clean &amp; Concentrator-5 kit (Zymo Research) and analyzed by 5’ RACE.</p></div><div class="section" id="sec-35"><h3><em>In Vitro</em> Synthesis of Capped mRNA</h3><p id="p-81">A DNA template corresponding to the first 579 nt (including 32 nt of 5’ UTR) of the <em>K. lactis ACT1</em> coding sequence (EnsemblFungi Id: KLLA0_D05357g) (<a id="xref-table-wrap-1-21" class="xref-table" href="#T1">Table S1B</a>) was prepared by PCR with the forward primer modT7_KLactin_F (containing a modified T7 promoter sequence (<a id="xref-ref-133-1" class="xref-bibr" href="#ref-133">Coleman et al., 2004</a>)) and the reverse primer actin_KL-rev2 using <em>K. lactis</em> IFO1267 total cDNA as a PCR template as follows: 5 min at 95°C; then 30 cycles of 30 sec at 94°C, 30 sec at 55°C, and 1 min at 72°C; and finally, 10 min at 72°C. PCR products were purified using a High Pure PCR Product Purification Kit (Roche) and <em>in vitro</em>-transcribed using a TranscriptAid T7 High Yield Transcription Kit (Fermentas). Template DNA was removed by a DNA-free<sup>™</sup> Kit (Ambion). Transcribed RNA was purified by an RNA Clean &amp; Concentrator-5 kit (Zymo Research), and 9 pmol of RNA was capped <em>in vitro</em> using the vaccinia virus mRNA capping enzyme (Ambion) in the presence or absence of SAM according to the manufacturer’s instructions and purified using an RNA Clean &amp; Concentrator-5 kit. Part of the sample was collected and analyzed by 5’ RACE to determine the efficiency of the mRNA capping reaction, while the rest of the sample was used for decapping assays with Rai1 or hDcp2 mRNA decapping enzymes.</p></div><div class="section" id="sec-36"><h3><em>PBP1, PAB1</em> and <em>LSM1</em> Gene Deletion</h3><p id="p-82">The <em>PBP1, PAB1</em> and <em>LSM1</em> genes were deleted from the chromosomes of the <em>K. lactis</em> IFO1267 strain using a loxP-G418-loxP cassette (<a id="xref-ref-136-1" class="xref-bibr" href="#ref-136">Guldener et al., 1996</a>). Briefly, the cassette for the deletion of the <em>PBP1</em> gene was amplified from the pUG6 plasmid by PCR with the primers KL_pbp1-del_Rev and KL_pbp1-del_For. The PCR product was resolved by agarose electrophoresis, purified, and transformed into <em>K. lactis</em> IFO1267. The deletion of the <em>PBP1</em> gene was verified by PCR. This modified strain was transformed with the pSH65 plasmid (<a id="xref-ref-135-1" class="xref-bibr" href="#ref-135">Gueldener et al., 2002</a>), incubated for 5 hours in nonselective medium and plated on solid medium supplemented with phleomycin (400 μg/ml). Monocolonies were grown on YPD plates with phleomycin for two days and subsequently tested for their ability to grow on G418 (250 μg/ml) selective plates. The excision of the G418 cassette was verified by PCR for selected colonies that did not grow in the presence of G418. These colonies were cultivated for five days with daily dilution under nonselective conditions (YPD medium only), resulting in the loss of the Cre-containing pSH65 plasmid. The resulting yeast strain, <em>K. lactis</em> IFO1267pbp1Δ, was used for subsequent <em>PAB1</em> deletion using the primers KL_pab1-del_For and KL_pab1-del_Rev in a similar fashion with the exception of the G418 cassette excision. <em>K. lactis</em> IFO1267 was used for <em>LSM1</em> deletion using primers KL_lsm1-del_For and KL_lsm1-del_Rev in a similar fashion with the exception of G418 cassette excision. The PCR reactions were carried out similarly (5 min at 95°C; then 30 cycles of 30 sec at 94°C, 30 sec at 55°C, and 4 min at 68°C; and finally, 10 min at 68°C). The nucleotide sequences of the primers used for the verification of the gene disruption cassettes are summarized in <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table S2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489486/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489486/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489486" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2</span> <p id="p-83" class="first-child">Primers Used in this Study, Related to Experimental Procedures</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489429/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489429/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489491/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489491/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489491" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489530/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489530/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S3.</span> <p id="p-84" class="first-child">Yeast Strains Used in this Study, Related to Experimental Procedures</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489509/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489509/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489462/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489462/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489462" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4.</span> <p id="p-85" class="first-child">Plasmids Used in this Study, Related to Experimental Procedures</p><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489668/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/489668/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489668" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S5.</span> <span class="caption-title">Analyses of the 5’ UTR poly(A) Leaders in pGKL mRNAs; Related to <a id="xref-fig-3-17" class="xref-fig" href="#F3">Figure 3</a></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-68">We thank Petra Studnickova, Vlasta Pelechova and Natalie Suchankova (Charles University, Czech Republic) for their help. We thank Prof. Michael Altman (University of Bern, Switzerland), Prof. Norio Gunge (Sojo University), Prof. Mike Kiledjian (Rutgers University), Prof. Liang Tong (Columbia University), and Prof. Beate Schwer (Weill Cornell Medical College) for yeast strains and plasmids.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.1" data-doi="10.1111/j.1574-6968.1995.tb07745.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fukuhara</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">Linear DNA plasmids of yeasts</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol Lett</abbr>, <span class="cit-vol">131</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BLetters%26rft.stitle%253DFEMS%2BMicrobiol%2BLett%26rft.aulast%253DFukuhara%26rft.auinit1%253DH.%26rft.volume%253D131%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DLinear%2BDNA%2Bplasmids%2Bof%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1995.tb07745.x%26rft_id%253Dinfo%253Apmid%252F7557303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.1995.tb07745.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7557303&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RP78500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-325316v1.2"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tokunaga</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-chapter-title">Linear DNA plasmids and killer system of Kluyveromyces lactis</span>. In: <span class="cit-ed"><span class="cit-name-surname">Kück</span>  <span class="cit-name-given-names">U</span></span> (ed) <span class="cit-source">The Mycota II, Genetics and biotechnology</span> (<span class="cit-edition">2nd ed</span>), <span class="cit-publ-name">Springer-Verlag Berlin</span>, <span class="cit-fpage">199</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamaru</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">1981</span>) <span class="cit-article-title">Isolation and Characterization of Linear Deoxyribonucleic Acid Plasmids from Kluyveromyces lactis and the Plasmid-Associated Killer Character</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr>, <span class="cit-vol">145</span>, <span class="cit-fpage">382</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGunge%26rft.auinit1%253DN%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D382%26rft.epage%253D390%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Blinear%2Bdeoxyribonucleic%2Bacid%2Bplasmids%2Bfrom%2BKluyveromyces%2Blactis%2Band%2Bthe%2Bplasmid-associated%2Bkiller%2Bcharacter.%26rft_id%253Dinfo%253Apmid%252F6257636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTQ1LzEvMzgyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.4" data-doi="10.1146/annurev.mi.37.100183.001345"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">1983</span>) <span class="cit-article-title">Yeast DNA plasmids</span>. <abbr class="cit-jnl-abbrev">Annu Rev Microbiol</abbr>, <span class="cit-vol">37</span>, <span class="cit-fpage">253</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DGunge%26rft.auinit1%253DN.%26rft.volume%253D37%26rft.spage%253D253%26rft.epage%253D276%26rft.atitle%253DYeast%2BDNA%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.37.100183.001345%26rft_id%253Dinfo%253Apmid%252F6357054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.37.100183.001345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6357054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RK48800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.5" data-doi="10.1093/nar/16.13.5863"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tommasino</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ricci</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Galeotti</span>,  <span class="cit-name-given-names">C.L.</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-article-title">Genome organization of the killer plasmid pGK12 from Kluyveromyces lactis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">5863</span>–<span class="cit-lpage">5878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTommasino%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D13%26rft.spage%253D5863%26rft.epage%253D5878%26rft.atitle%253DGenome%2Borganization%2Bof%2Bthe%2Bkiller%2Bplasmid%2BpGK12%2Bfrom%2BKluyveromyces%2Blactis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F16.13.5863%26rft_id%253Dinfo%253Apmid%252F3041369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/16.13.5863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3041369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P228300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.6" data-doi="10.1093/nar/12.19.7581"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hishinuma</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakamura</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirai</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishizawa</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Maeda</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Cloning and nucleotide sequences of the linear DNA killer plasmids from yeast</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">7581</span>–<span class="cit-lpage">7597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHishinuma%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.issue%253D19%26rft.spage%253D7581%26rft.epage%253D7597%26rft.atitle%253DCloning%2Band%2Bnucleotide%2Bsequences%2Bof%2Bthe%2Blinear%2BDNA%2Bkiller%2Bplasmids%2Bfrom%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F12.19.7581%26rft_id%253Dinfo%253Apmid%252F6387625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/12.19.7581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6387625&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TN33600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.7" data-doi="10.1093/nar/14.17.6871"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwakman</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meijer</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stuitje</span>,  <span class="cit-name-given-names">A.R.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">Efficient isolation of the linear DNA killer plasmid of Kluyveromyces lactis: evidence for location and expression in the cytoplasm and characterization of their terminally bound proteins</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">6871</span>–<span class="cit-lpage">6884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStam%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D14%26rft.issue%253D17%26rft.spage%253D6871%26rft.epage%253D6884%26rft.atitle%253DEfficient%2Bisolation%2Bof%2Bthe%2Blinear%2BDNA%2Bkiller%2Bplasmid%2Bof%2BKluyveromyces%2Blactis%253A%2Bevidence%2Bfor%2Blocation%2Band%2Bexpression%2Bin%2Bthe%2Bcytoplasm%2Band%2Bcharacterization%2Bof%2Btheir%2Bterminally%2Bbound%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.17.6871%26rft_id%253Dinfo%253Apmid%252F3763395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/14.17.6871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3763395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986E041300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Boyd</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">The killer toxin of Kluyveromyces lactis: characterization of the toxin subunits and identification of the genes which encode them</span>. <abbr class="cit-jnl-abbrev">Embo J</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">1995</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmbo%2BJ%26rft.volume%253D5%26rft.spage%253D1995%26rft_id%253Dinfo%253Apmid%252F3758030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3758030&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986D534000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.9" data-doi="10.1093/nar/12.15.6011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mileham</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romanos</span>,  <span class="cit-name-given-names">M.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Boyd</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Nucleotide sequence and transcription analysis of a linear DNA plasmid associated with the killer character of the yeast Kluyveromyces lactis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">6011</span>–<span class="cit-lpage">6030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStark%26rft.auinit1%253DM.%2BJ.R.%26rft.volume%253D12%26rft.issue%253D15%26rft.spage%253D6011%26rft.epage%253D6030%26rft.atitle%253DNucleotide%2Bsequence%2Band%2Btranscription%2Banalysis%2Bof%2Ba%2Blinear%2BDNA%2Bplasmid%2Bassociated%2Bwith%2Bthe%2Bkiller%2Bcharacter%2Bof%2Bthe%2Byeast%2BKhuyveromyces%2Blactis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F12.15.6011%26rft_id%253Dinfo%253Apmid%252F6473099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/12.15.6011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6473099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TE24900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.10" data-doi="10.1093/nar/17.9.3435"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tokunaga</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawamura</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hishinuma</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Expression of pGKL killer 28K subunit in Saccharomyces cerevisiae: identification of 28K subunit as a killer protein</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">3435</span>–<span class="cit-lpage">3446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTokunaga%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D3435%26rft.epage%253D3446%26rft.atitle%253DExpression%2Bof%2BpGKL%2Bkiller%2B28K%2Bsubunit%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bidentification%2Bof%2B28K%2Bsubunit%2Bas%2Ba%2Bkiller%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.9.3435%26rft_id%253Dinfo%253Apmid%252F2657659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.9.3435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2657659&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U545200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.11" data-doi="10.1093/nar/15.3.1031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tokunaga</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wada</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hishinuma</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Expression and identification of immunity determinants on linear DNA killer plasmids pGKL1 and pGKL2 in Kluyveromyces lactis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">1031</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTokunaga%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1031%26rft.epage%253D1046%26rft.atitle%253DExpression%2Band%2Bidentification%2Bof%2Bimmunity%2Bdeterminants%2Bon%2Blinear%2BDNA%2Bkiller%2Bplasmids%2BpGKL1%2Band%2BpGKL2%2Bin%2BKluyveromyces%2Blactis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1031%26rft_id%253Dinfo%253Apmid%252F3029695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.3.1031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3029695&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.12" data-doi="10.1093/nar/15.23.10046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fukuhara</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">The RF1 gene of the killer DNA of yeast may encode a DNA polymerase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">10046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DFukuhara%26rft.auinit1%253DH.%26rft.volume%253D15%26rft.issue%253D23%26rft.spage%253D10046%26rft.epage%253D10046%26rft.atitle%253DThe%2BRF1%2Bgene%2Bof%2Bthe%2Bkiller%2BDNA%2Bof%2Byeast%2Bmay%2Bencode%2Ba%2BDNA%2Bpolymerase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.23.10046%26rft_id%253Dinfo%253Apmid%252F3697067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.23.10046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3697067&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L276100032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.13" data-doi="10.1093/nar/15.21.9088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">G.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leavitt</span>,  <span class="cit-name-given-names">M.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ito</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Yeast killer plasmid pGKL1 encodes a DNA polymerase belonging to the family B DNA polymerases</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">9088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DJung%26rft.auinit1%253DG.%26rft.volume%253D15%26rft.issue%253D21%26rft.spage%253D9088%26rft.epage%253D9088%26rft.atitle%253Dyeast%2Bkiller%2Bplasmid%2BpGKL1%2Bencodes%2Ba%2BDNA%2Bpolymerase%2Bbelonging%2Bto%2Bthe%2Bfamily%2BB%2BDNA%2Bpolymerases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.21.9088%26rft_id%253Dinfo%253Apmid%252F3684586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.21.9088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3684586&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K780400043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.14" data-doi="10.1002/yea.320110703"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meacock</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">Kluyveromyces lactis killer plasmid pGKL2: molecular analysis of an essential gene, ORF5</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">615</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DSchaffrath%26rft.auinit1%253DR.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D615%26rft.epage%253D628%26rft.atitle%253DKluyveromyces%2Blactis%2Bkiller%2Bplasmid%2BpGKL2%253A%2Bmolecular%2Banalysis%2Bof%2Ban%2Bessential%2Bgene%252C%2BORF5.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110703%26rft_id%253Dinfo%253Apmid%252F7483835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7483835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RE55300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.15" data-doi="10.1002/(SICI)1097-0061(19960315)12:3&lt;241::AID-YEA907&gt;3.0.CO;2-J"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takeda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hiraishi</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takesako</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanase</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">The terminal protein of the linear DNA plasmid pGKL2 shares an N-terminal domain of the plasmid-encoded DNA polymerase</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">241</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DTakeda%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D246%26rft.atitle%253DThe%2Bterminal%2Bprotein%2Bof%2Bthe%2Blinear%2BDNA%2Bplasmid%2BpGKL2%2Bshares%2Ban%2BN-terminal%2Bdomain%2Bof%2Bthe%2Bplasmid-encoded%2BDNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819960315%252912%253A3%253C241%253A%253AAID-YEA907%253E3.0.CO%253B2-J%26rft_id%253Dinfo%253Apmid%252F8904336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19960315)12:3&lt;241::AID-YEA907&gt;3.0.CO;2-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8904336&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TY30700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.16" data-doi="10.1006/plas.1998.1367"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Kluyveromyces lactis killer plasmid pGKL2: evidence for a viral-like capping enzyme encoded by ORF3</span>. <abbr class="cit-jnl-abbrev">Plasmid</abbr>, <span class="cit-vol">40</span>, <span class="cit-fpage">243</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.stitle%253DPlasmid%26rft.aulast%253DLarsen%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D243%26rft.epage%253D246%26rft.atitle%253DKluyveromyces%2Blactis%2Bkiller%2Bplasmid%2BpGKL2%253A%2Bevidence%2Bfor%2Ba%2Bviral-like%2Bcapping%2Benzyme%2Bencoded%2Bby%2BORF3.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fplas.1998.1367%26rft_id%253Dinfo%253Apmid%252F9806862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/plas.1998.1367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9806862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076923400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.17" data-doi="10.1002/yea.728"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tiggemann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeske</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Kluyveromyces lactis cytoplasmic plasmid pGKL2: heterologous expression of Orf3p and proof of guanylyltransferase and mRNA-triphosphatase activities</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">18</span>, <span class="cit-fpage">815</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DTiggemann%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D815%26rft.epage%253D825%26rft.atitle%253DKluyveromyces%2Blactis%2Bcytoplasmic%2Bplasmid%2BpGKL2%253A%2Bheterologous%2Bexpression%2Bof%2BOrf3p%2Band%2Bproof%2Bof%2Bguanylyltransferase%2Band%2BmRNA-triphosphatase%2Bactivities.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.728%26rft_id%253Dinfo%253Apmid%252F11427964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11427964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.18" data-doi="10.1002/yea.320060102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">M.J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyd</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mileham</span>,  <span class="cit-name-given-names">A.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Romanos</span>,  <span class="cit-name-given-names">M.A.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">The Plasmid-encoded Killer System of Kluyveromyces lactis: A Review</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DStark%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D29%26rft.atitle%253DThe%2Bplasmid-encoded%2Bkiller%2Bsystem%2Bof%2BKluyveromyces%2Blactis%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320060102%26rft_id%253Dinfo%253Apmid%252F2180235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320060102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2180235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CN91600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.19" data-doi="10.1002/yea.773"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meacock</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">An SSB encoded by and operating on linear killer plasmids from Kluyveromyces lactis</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">18</span>, <span class="cit-fpage">1239</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DSchaffrath%26rft.auinit1%253DR.%26rft.volume%253D18%26rft.issue%253D13%26rft.spage%253D1239%26rft.epage%253D1247%26rft.atitle%253DAn%2BSSB%2Bencoded%2Bby%2Band%2Boperating%2Bon%2Blinear%2Bkiller%2Bplasmids%2Bfrom%2BKluyveromyces%2Blactis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.773%26rft_id%253Dinfo%253Apmid%252F11561291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.773&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11561291&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171435400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.20" data-doi="10.1002/yea.320070306"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tommasino</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Killer System of Kluyveromyces lactis: the Open Reading Frame 10 of the pGK12 Plasmid Encodes a Putative DNA Binding Protein</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">245</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DTommasino%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D245%26rft.epage%253D252%26rft.atitle%253DKiller%2Bsystem%2Bof%2BKluyveromyces%2Blactis%253A%2Bthe%2Bopen%2Breading%2Bframe%2B10%2Bof%2Bthe%2BpGK12%2Bplasmid%2Bencodes%2Ba%2Bputative%2BDNA%2Bbinding%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320070306%26rft_id%253Dinfo%253Apmid%252F1882549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320070306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1882549&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FL50300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.21" data-doi="10.1073/pnas.88.24.11398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNeel</span>,  <span class="cit-name-given-names">D.G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tamanoi</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Terminal region recognition factor 1, a DNA-binding protein recognizing the inverted terminal repeats of the pGKl linear DNA plasmids</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-vol">88</span>, <span class="cit-fpage">11398</span>–<span class="cit-lpage">11402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMcNeel%26rft.auinit1%253DD.%2BG.%26rft.volume%253D88%26rft.issue%253D24%26rft.spage%253D11398%26rft.epage%253D11402%26rft.atitle%253DTerminal%2Bregion%2Brecognition%2Bfactor%2B1%252C%2Ba%2BDNA-binding%2Bprotein%2Brecognizing%2Bthe%2Binverted%2Bterminal%2Brepeats%2Bof%2Bthe%2BpGKl%2Blinear%2BDNA%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.24.11398%26rft_id%253Dinfo%253Apmid%252F1763054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODgvMjQvMTEzOTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-325316v1.22" data-doi="10.1007/978-3-540-36832-8_5"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Esser</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Löttge</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Beyschlag</span>,  <span class="cit-name-given-names">W.</span></span> and </li><li><span class="cit-ed"><span class="cit-name-surname">Murata</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Jeske</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2007</span>) In <span class="cit-ed"><span class="cit-name-surname">Esser</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Löttge</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Beyschlag</span>,  <span class="cit-name-given-names">W.</span></span> and <span class="cit-ed"><span class="cit-name-surname">Murata</span>,  <span class="cit-name-given-names">J.</span></span> (eds.), <span class="cit-source">Progress in Botany</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>, Vol. <span class="cit-vol">68</span>, pp. <span class="cit-fpage">98</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2BBotany%26rft.volume%253D68%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-540-36832-8_5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-540-36832-8_5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">D.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meacock</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-article-title">Extranuclear gene expression in yeast: evidence for a plasmid-encoded RNA polymerase of unique structure</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">8097</span>–<span class="cit-lpage">8112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D16%26rft.issue%253D16%26rft.spage%253D8097%26rft.epage%253D8112%26rft.atitle%253DExtranuclear%2Bgene%2Bexpression%2Bin%2Byeast%253A%2Bevidence%2Bfor%2Ba%2Bplasmid-encoded%2BRNA%2Bpolymerase%2Bof%2Bunique%2Bstructure%26rft_id%253Dinfo%253Apmid%252F3138657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3138657&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P964900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.24" data-doi="10.1007/s002940050195"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meacock</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">ORF7 of yeast plasmid pGKL2: analysis of gene expression in vivo</span>. <abbr class="cit-jnl-abbrev">Curr Genet</abbr>, <span class="cit-vol">31</span>, <span class="cit-fpage">190</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DSchaffrath%26rft.auinit1%253DR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D190%26rft.epage%253D192%26rft.atitle%253DORF7%2Bof%2Byeast%2Bplasmid%2BpGKL2%253A%2Banalysis%2Bof%2Bgene%2Bexpression%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050195%26rft_id%253Dinfo%253Apmid%252F9021138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9021138&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.25" data-doi="10.1006/plas.1994.1071"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wodara</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schickel</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-article-title">A Novel-Approach to Express a Heterologous Gene on Kluyveromyces-Lactis Linear Killer Plasmids - Expression of the Bacterial Aph Gene from a Cytoplasmic Promoter Fragment without in-Phase Fusion to the Plasmid Open Reading Frame</span>. <abbr class="cit-jnl-abbrev">Plasmid</abbr>, <span class="cit-vol">32</span>, <span class="cit-fpage">318</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.stitle%253DPlasmid%26rft.aulast%253DMeinhardt%26rft.auinit1%253DF.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D318%26rft.epage%253D327%26rft.atitle%253DA%2Bnovel%2Bapproach%2Bto%2Bexpress%2Ba%2Bheterologous%2Bgene%2Bon%2BKluyveromyces%2Blactis%2Blinear%2Bkiller%2Bplasmids%253A%2Bexpression%2Bof%2Bthe%2Bbacterial%2Baph%2Bgene%2Bfrom%2Ba%2Bcytoplasmic%2Bpromoter%2Bfragment%2Bwithout%2Bin-phase%2Bfusion%2Bto%2Bthe%2Bplasmid%2Bopen%2Breading%2Bframe.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fplas.1994.1071%26rft_id%253Dinfo%253Apmid%252F7899517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/plas.1994.1071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7899517&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PW74500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.26" data-doi="10.1007/s002849900122"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrunder</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Extranuclear expression of the bacterial xylose isomerase (xylA) and the UDP-glucose dehydrogenase (hasB) genes in yeast with Kluyveromyces lactis linear killer plasmids as vectors</span>. <abbr class="cit-jnl-abbrev">Curr Microbiol</abbr>, <span class="cit-vol">33</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bmicrobiology%26rft.stitle%253DCurr%2BMicrobiol%26rft.aulast%253DSchr%25C3%25BCnder%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D5%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DExtranuclear%2Bexpression%2Bof%2Bthe%2Bbacterial%2Bxylose%2Bisomerase%2B%2528xylA%2529%2Band%2Bthe%2BUDP-glucose%2Bdehydrogenase%2B%2528hasB%2529%2Bgenes%2Bin%2Byeast%2Bwith%2BKluyveromyces%2Blactis%2Blinear%2Bkiller%2Bplasmids%2Bas%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002849900122%26rft_id%253Dinfo%253Apmid%252F8875914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002849900122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8875914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.27" data-doi="10.1006/plas.1995.1015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrunder</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">An extranuclear expression system for analysis of cytoplasmic promoters of yeast linear killer plasmids</span>. <abbr class="cit-jnl-abbrev">Plasmid</abbr>, <span class="cit-vol">33</span>, <span class="cit-fpage">139</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.stitle%253DPlasmid%26rft.aulast%253DSchr%25C3%25BCnder%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D151%26rft.atitle%253DAn%2Bextranuclear%2Bexpression%2Bsystem%2Bfor%2Banalysis%2Bof%2Bcytoplasmic%2Bpromoters%2Bof%2Byeast%2Blinear%2Bkiller%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fplas.1995.1015%26rft_id%253Dinfo%253Apmid%252F7597108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/plas.1995.1015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7597108&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QN02200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.28" data-doi="10.1261/rna.7247705"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johansson</span>,  <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bystrom</span>,  <span class="cit-name-given-names">A.S.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">An early step in wobble uridine tRNA modification requires the Elongator complex</span>. <abbr class="cit-jnl-abbrev">Rna</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">424</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DHUANG%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D424%26rft.epage%253D436%26rft.atitle%253DAn%2Bearly%2Bstep%2Bin%2Bwobble%2Buridine%2BtRNA%2Bmodification%2Brequires%2Bthe%2BElongator%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.7247705%26rft_id%253Dinfo%253Apmid%252F15769872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjExLzQvNDI0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.29" data-doi="10.1261/rna.2172105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Esberg</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johansson</span>,  <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bystrom</span>,  <span class="cit-name-given-names">A.S.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">The Kluyveromyces lactis gamma-toxin targets tRNA anticodons</span>. <abbr class="cit-jnl-abbrev">Rna</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">1648</span>–<span class="cit-lpage">1654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DLU%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1648%26rft.epage%253D1654%26rft.atitle%253DThe%2BKluyveromyces%2Blactis%2B%257Bgamma%257D-toxin%2Btargets%2BtRNA%2Banticodons%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2172105%26rft_id%253Dinfo%253Apmid%252F16244131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS8xNjQ4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.30" data-doi="10.1111/j.1365-2958.2005.04972.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jablonowski</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zink</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mehlgarten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daum</span>,  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">tRNAGlu wobble uridine methylation by Trm9 identifies Elongator’s key role for zymocin-induced cell death in yeast</span>. <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr>, <span class="cit-vol">59</span>, <span class="cit-fpage">677</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DJablonowski%26rft.auinit1%253DD.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D677%26rft.epage%253D688%26rft.atitle%253DtRNAGlu%2Bwobble%2Buridine%2Bmethylation%2Bby%2BTrm9%2Bidentifies%2BElongator%2527s%2Bkey%2Brole%2Bfor%2Bzymocin-induced%2Bcell%2Bdeath%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2005.04972.x%26rft_id%253Dinfo%253Apmid%252F16390459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2005.04972.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16390459&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234162900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.31" data-doi="10.1261/rna.034132.112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meineke</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kast</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A fungal anticodon nuclease ribotoxin exploits a secondary cleavage site to evade tRNA repair</span>. <abbr class="cit-jnl-abbrev">Rna</abbr>, <span class="cit-vol">18</span>, <span class="cit-fpage">1716</span>–<span class="cit-lpage">1724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRna%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.034132.112%26rft_id%253Dinfo%253Apmid%252F22836353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTcxNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.32" data-doi="10.1099/00221287-137-7-1749"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">J.H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Analysis of the response of Saccharomyces cerevisiae cells to Kluyveromyces lactis toxin</span>. <abbr class="cit-jnl-abbrev">J Gen Microbiol</abbr>, <span class="cit-vol">137</span>, <span class="cit-fpage">1749</span>–<span class="cit-lpage">1757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DButler%26rft.auinit1%253DA.%2BR.%26rft.volume%253D137%26rft.issue%253D7%26rft.spage%253D1749%26rft.epage%253D1757%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bresponse%2Bof%2BSaccharomyces%2Bcerevisiae%2Bcells%2Bto%2BKluyveromyces%2Blactis%2Btoxin%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-137-7-1749%26rft_id%253Dinfo%253Apmid%252F1955863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-137-7-1749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1955863&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FX60200040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.33" data-doi="10.1111/mmi.12481"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kast</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">rRNA fragmentation induced by a yeast killer toxin</span>. <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr>, <span class="cit-vol">91</span>, <span class="cit-fpage">606</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BMicrobiol%26rft.volume%253D91%26rft.spage%253D606%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12481%26rft_id%253Dinfo%253Apmid%252F24308908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24308908&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.34" data-doi="10.1016/S0065-3527(00)55003-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furuichi</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shatkin</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Viral and cellular mRNA capping: past and prospects</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr>, <span class="cit-vol">55</span>, <span class="cit-fpage">135</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DFuruichi%26rft.auinit1%253DY.%26rft.volume%253D55%26rft.spage%253D135%26rft.epage%253D184%26rft.atitle%253DViral%2Band%2Bcellular%2BmRNA%2Bcapping%253A%2Bpast%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252800%252955003-9%26rft_id%253Dinfo%253Apmid%252F11050942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(00)55003-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11050942&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165695700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.35" data-doi="10.1016/S0079-6603(00)66025-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Structure, mechanism, and evolution of the mRNA capping apparatus</span>. <abbr class="cit-jnl-abbrev">Prog Nucleic Acid Res Mol Biol</abbr>, <span class="cit-vol">66</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bnucleic%2Bacid%2Bresearch%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DProg%2BNucleic%2BAcid%2BRes%2BMol%2BBiol%26rft.aulast%253DShuman%26rft.auinit1%253DS.%26rft.volume%253D66%26rft.spage%253D1%26rft.epage%253D40%26rft.atitle%253DStructure%252C%2Bmechanism%252C%2Band%2Bevolution%2Bof%2Bthe%2BmRNA%2Bcapping%2Bapparatus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0079-6603%252800%252966025-7%26rft_id%253Dinfo%253Apmid%252F11051760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0079-6603(00)66025-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11051760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165500600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.36" data-doi="10.1038/nrm880"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">What messenger RNA capping tells us about eukaryotic evolution</span>. <abbr class="cit-jnl-abbrev">Nat Rev Mol Cell Biol</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">619</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DShuman%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253D619%26rft.epage%253D625%26rft.atitle%253DWhat%2Bmessenger%2BRNA%2Bcapping%2Btells%2Bus%2Babout%2Beukaryotic%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm880%26rft_id%253Dinfo%253Apmid%252F12154373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12154373&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177427600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.37" data-doi="10.1038/nrmicro2675"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lescar</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Canard</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Conventional and unconventional mechanisms for capping viral mRNA</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">51</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDecroly%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D65%26rft.atitle%253DConventional%2Band%2Bunconventional%2Bmechanisms%2Bfor%2Bcapping%2Bviral%2BmRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2675%26rft_id%253Dinfo%253Apmid%252F22138959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.38" data-doi="10.1126/science.1144467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">W.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">T.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Doudna</span>,  <span class="cit-name-given-names">J.A.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Cap-independent translation is required for starvation-induced differentiation in yeast</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">317</span>, <span class="cit-fpage">1224</span>–<span class="cit-lpage">1227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGilbert%26rft.auinit1%253DW.%2BV.%26rft.volume%253D317%26rft.issue%253D5842%26rft.spage%253D1224%26rft.epage%253D1227%26rft.atitle%253DCap-Independent%2BTranslation%2BIs%2BRequired%2Bfor%2BStarvation-Induced%2BDifferentiation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1144467%26rft_id%253Dinfo%253Apmid%252F17761883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTcvNTg0Mi8xMjI0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.39" data-doi="10.1038/nsmb1287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bergman</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moraes</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaric</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kambach</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilusz</span>,  <span class="cit-name-given-names">C.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wilusz</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Lsm proteins bind and stabilize RNAs containing 5’ poly(A) tracts</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">824</span>–<span class="cit-lpage">831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DBergman%26rft.auinit1%253DN.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D824%26rft.epage%253D831%26rft.atitle%253DLsm%2Bproteins%2Bbind%2Band%2Bstabilize%2BRNAs%2Bcontaining%2B5%2527%2Bpoly%2528A%2529%2Btracts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1287%26rft_id%253Dinfo%253Apmid%252F17694069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17694069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249276500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-325316v1.40"><div class="cit-metadata"><ol class="duplicate"><li>In </li><li><span class="cit-ed"><span class="cit-name-surname">Krieg</span>,  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castanotto</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westaway</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rossi</span>,  <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">1996</span>) In <span class="cit-ed"><span class="cit-name-surname">Krieg</span>,  <span class="cit-name-given-names">P. A.</span></span> (ed.), <span class="cit-source">A Laboratory Guide to RNA: Isolation, Analysis, and Synthesis</span>. <span class="cit-publ-name">Wiley-Liss</span>, <span class="cit-publ-loc">New York</span>, pp. <span class="cit-fpage">43</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.41" data-doi="10.1016/j.ab.2004.09.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Masek</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vopalensky</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchomelova</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Denaturing RNA electrophoresis in TAE agarose gels</span>. <abbr class="cit-jnl-abbrev">Anal Biochem</abbr>, <span class="cit-vol">336</span>, <span class="cit-fpage">46</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DMasek%26rft.auinit1%253DT.%26rft.volume%253D336%26rft.issue%253D1%26rft.spage%253D46%26rft.epage%253D50%26rft.atitle%253DDenaturing%2BRNA%2Belectrophoresis%2Bin%2BTAE%2Bagarose%2Bgels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ab.2004.09.010%26rft_id%253Dinfo%253Apmid%252F15582557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ab.2004.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15582557&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225918900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yamane</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Incompatibility of linear DNA killer plasmids pGKL1 and pGKL2 from Kluyveromyces lactis with mitochondrial DNA from Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr>, <span class="cit-vol">159</span>, <span class="cit-fpage">533</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGunge%26rft.auinit1%253DN%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D533%26rft.epage%253D539%26rft.atitle%253DIncompatibility%2Bof%2Blinear%2BDNA%2Bkiller%2Bplasmids%2BpGKL1%2Band%2BpGKL2%2Bfrom%2BKluyveromyces%2Blactis%2Bwith%2Bmitochondrial%2BDNA%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F6378881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNTMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.43" data-doi="10.1093/nar/19.18.5083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Palkova</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Microisolation of Yeast Nucleic-Acids on the Microtitre Plate without Using Lytic Enzymes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">5083</span>–<span class="cit-lpage">5083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPospisek%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D5083%26rft.epage%253D5083%26rft.atitle%253DMicroisolation%2Bof%2Byeast%2Bnucleic%2Bacids%2Bon%2Bthe%2Bmicrotitre%2Bplate%2Bwithout%2Busing%2Blytic%2Benzymes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.18.5083%26rft_id%253Dinfo%253Apmid%252F1923782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/19.18.5083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1923782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GH66000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lima</span>,  <span class="cit-name-given-names">C.D.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Enzymology of RNA cap synthesis. Wiley interdisciplinary reviews</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>, <span class="cit-vol">1</span>, <span class="cit-fpage">152</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.volume%253D1%26rft.spage%253D152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.45" data-doi="10.1016/j.cell.2009.01.042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>,  <span class="cit-name-given-names">A.G.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Regulation of translation initiation in eukaryotes: mechanisms and biological targets</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">136</span>, <span class="cit-fpage">731</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSonenberg%26rft.auinit1%253DN.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D731%26rft.epage%253D745%26rft.atitle%253DRegulation%2Bof%2Btranslation%2Binitiation%2Bin%2Beukaryotes%253A%2Bmechanisms%2Band%2Bbiological%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.042%26rft_id%253Dinfo%253Apmid%252F19239892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19239892&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263688200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.46" data-doi="10.1016/0092-8674(75)90009-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Both</span>,  <span class="cit-name-given-names">G.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furuichi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muthukrishnan</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shatkin</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">1975</span>) <span class="cit-article-title">Ribosome binding to reovirus mRNA in protein synthesis requires 5’ terminal 7-methylguanosine</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">185</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBoth%26rft.auinit1%253DG.%2BW.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D185%26rft.epage%253D195%26rft.atitle%253DRibosome%2Bbinding%2Bto%2Breovirus%2BmRNA%2Bin%2Bprotein%2Bsynthesis%2Brequires%2B5%2527%2Bterminal%2B7-methylguanosine.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252875%252990009-4%26rft_id%253Dinfo%253Apmid%252F1182800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(75)90009-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1182800&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AU26200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Higman</span>,  <span class="cit-name-given-names">M.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Niles</span>,  <span class="cit-name-given-names">E.G.</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-article-title">Location of the S-adenosyl-L-methionine binding region of the vaccinia virus mRNA (guanine-7-)methyltransferase</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">269</span>, <span class="cit-fpage">14982</span>–<span class="cit-lpage">14987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DHigman%26rft.auinit1%253DM.%2BA.%26rft.volume%253D269%26rft.issue%253D21%26rft.spage%253D14982%26rft.epage%253D14987%26rft.atitle%253DLocation%2Bof%2Bthe%2BS-adenosyl-L-methionine%2Bbinding%2Bregion%2Bof%2Bthe%2Bvaccinia%2Bvirus%2BmRNA%2B%2528guanine-7-%2529methyltransferase%26rft_id%253Dinfo%253Apmid%252F8195133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvMjEvMTQ5ODIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-article-title">Intrinsic RNA (guanine-7) methyltransferase activity of the vaccinia virus capping enzyme D1 subunit is stimulated by the D12 subunit. Identification of amino acid residues in the D1 protein required for subunit association and methyl group transfer</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">269</span>, <span class="cit-fpage">24472</span>–<span class="cit-lpage">24479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DMao%26rft.auinit1%253DX.%26rft.volume%253D269%26rft.issue%253D39%26rft.spage%253D24472%26rft.epage%253D24479%26rft.atitle%253DIntrinsic%2BRNA%2B%2528guanine-7%2529%2Bmethyltransferase%2Bactivity%2Bof%2Bthe%2Bvaccinia%2Bvirus%2Bcapping%2Benzyme%2BD1%2Bsubunit%2Bis%2Bstimulated%2Bby%2Bthe%2BD12%2Bsubunit.%2BIdentification%2Bof%2Bamino%2Bacid%2Bresidues%2Bin%2Bthe%2BD1%2Bprotein%2Brequired%2Bfor%2Bsubunit%2Bassociation%2Band%2Bmethyl%2Bgroup%2Btransfer%26rft_id%253Dinfo%253Apmid%252F7929111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvMzkvMjQ0NzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.49" data-doi="10.1074/jbc.M602867200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hausmann</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Poxvirus mRNA cap methyltransferase. Bypass of the requirement for the stimulatory subunit by mutations in the catalytic subunit and evidence for intersubunit allostery</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">281</span>, <span class="cit-fpage">18953</span>–<span class="cit-lpage">18960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M602867200%26rft_id%253Dinfo%253Apmid%252F16707499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjgvMTg5NTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.50" data-doi="10.1016/j.virol.2006.03.050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Genetic analysis of poxvirus mRNA cap methyltransferase: suppression of conditional mutations in the stimulatory D12 subunit by second-site mutations in the catalytic D1 subunit</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-vol">352</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D352%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D156%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bpoxvirus%2BmRNA%2Bcap%2Bmethyltransferase%253A%2Bsuppression%2Bof%2Bconditional%2Bmutations%2Bin%2Bthe%2Bstimulatory%2BD12%2Bsubunit%2Bby%2Bsecond-site%2Bmutations%2Bin%2Bthe%2Bcatalytic%2BD1%2Bsubunit.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.03.050%26rft_id%253Dinfo%253Apmid%252F16716374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.03.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16716374&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.51" data-doi="10.1016/j.str.2013.12.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kyrieleis</span>,  <span class="cit-name-given-names">O.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Pena</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cusack</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>, <span class="cit-vol">22</span>, <span class="cit-fpage">452</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D22%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2013.12.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2013.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cong</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">The D1 and D12 subunits are both essential for the transcription termination factor activity of vaccinia virus capping enzyme</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">69</span>, <span class="cit-fpage">3852</span>–<span class="cit-lpage">3856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLuo%26rft.auinit1%253DY.%26rft.volume%253D69%26rft.issue%253D6%26rft.spage%253D3852%26rft.epage%253D3856%26rft.atitle%253DThe%2BD1%2Band%2BD12%2Bsubunits%2Bare%2Bboth%2Bessential%2Bfor%2Bthe%2Btranscription%2Btermination%2Bfactor%2Bactivity%2Bof%2Bvaccinia%2Bvirus%2Bcapping%2Benzyme%26rft_id%253Dinfo%253Apmid%252F7745734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY5LzYvMzg1MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.53" data-doi="10.1093/nar/26.7.1700"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yamada-Okabe</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimmi</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arisawa</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yamada-Okabe</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Isolation and characterization of a human cDNA for mRNA 5’-capping enzyme</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">1700</span>–<span class="cit-lpage">1706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.7.1700%26rft_id%253Dinfo%253Apmid%252F9512541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/26.7.1700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9512541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073065900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.54" data-doi="10.1074/jbc.274.23.16553"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Characterization of human, Schizosaccharomyces pombe, and Candida albicans mRNA cap methyltransferases and complete replacement of the yeast capping apparatus by mammalian enzymes</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">274</span>, <span class="cit-fpage">16553</span>–<span class="cit-lpage">16562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DSaha%26rft.auinit1%253DN.%26rft.volume%253D274%26rft.issue%253D23%26rft.spage%253D16553%26rft.epage%253D16562%26rft.atitle%253DCharacterization%2Bof%2BHuman%252C%2BSchizosaccharomyces%2Bpombe%252C%2Band%2BCandida%2Balbicans%2BmRNA%2BCap%2BMethyltransferases%2Band%2BComplete%2BReplacement%2Bof%2Bthe%2BYeast%2BCapping%2BApparatus%2Bby%2BMammalian%2BEnzymes%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.23.16553%26rft_id%253Dinfo%253Apmid%252F10347220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMjMvMTY1NTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.55" data-doi="10.1074/jbc.M006072200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lehman</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Characterization of the mRNA capping apparatus of Candida albicans</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">276</span>, <span class="cit-fpage">1857</span>–<span class="cit-lpage">1864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M006072200%26rft_id%253Dinfo%253Apmid%252F11035009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzYvMy8xODU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.56" data-doi="10.1016/S0014-5793(98)01037-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yamada-Okabe</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mio</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsui</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kashima</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arisawa</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yamada-Okabe</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Isolation and characterization of the Candida albicans gene for mRNA 5’- triphosphatase: association of mRNA 5’-triphosphatase and mRNA 5’-guanylyltransferase activities is essential for the function of mRNA 5’-capping enzyme in vivo</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>, <span class="cit-vol">435</span>, <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DYamada-Okabe%26rft.auinit1%253DT.%26rft.volume%253D435%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bthe%2BCandida%2Balbicans%2Bgene%2Bfor%2BmRNA%2B5%2527-triphosphatase%253A%2Bassociation%2Bof%2BmRNA%2B5%2527-triphosphatase%2Band%2BmRNA%2B5%2527-guanylyltransferase%2Bactivities%2Bis%2Bessential%2Bfor%2Bthe%2Bfunction%2Bof%2BmRNA%2B5%2527-capping%2Benzyme%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252898%252901037-0%26rft_id%253Dinfo%253Apmid%252F9755857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(98)01037-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9755857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076048700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.57" data-doi="10.1093/nar/28.9.1885"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lehman</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Characterization of Candida albicans RNA triphosphatase and mutational analysis of its active site</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">28</span>, <span class="cit-fpage">1885</span>–<span class="cit-lpage">1892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPei%26rft.auinit1%253DY.%26rft.volume%253D28%26rft.issue%253D9%26rft.spage%253D1885%26rft.epage%253D1892%26rft.atitle%253DCharacterization%2Bof%2BCandida%2Balbicans%2BRNA%2Btriphosphatase%2Band%2Bmutational%2Banalysis%2Bof%2Bits%2Bactive%2Bsite%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.9.1885%26rft_id%253Dinfo%253Apmid%252F10756187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.9.1885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10756187&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086842200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.58" data-doi="10.1074/jbc.M109649200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hausmann</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vivares</span>,  <span class="cit-name-given-names">C.P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Characterization of the mRNA capping apparatus of the microsporidian parasite Encephalitozoon cuniculi</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">277</span>, <span class="cit-fpage">96</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109649200%26rft_id%253Dinfo%253Apmid%252F11687593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjg6IjI3Ny8xLzk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.59" data-doi="10.1074/jbc.M212101200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takagi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">A.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawa</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diehn</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takase</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blackwell</span>,  <span class="cit-name-given-names">T.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Buratowski</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">The Caenorhabditis elegans mRNA 5’-capping enzyme. In vitro and in vivo characterization</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">278</span>, <span class="cit-fpage">14174</span>–<span class="cit-lpage">14184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M212101200%26rft_id%253Dinfo%253Apmid%252F12576476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMTYvMTQxNzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.60" data-doi="10.1074/jbc.M212102200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piano</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shatkin</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">mRNA capping enzyme requirement for Caenorhabditis elegans viability</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-vol">278</span>, <span class="cit-fpage">14168</span>–<span class="cit-lpage">14173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M212102200%26rft_id%253Dinfo%253Apmid%252F12576475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMTYvMTQxNjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.61" data-doi="10.1128/JVI.74.12.5486-5494.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">C.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martins</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">A yeast-based genetic system for functional analysis of viral mRNA capping enzymes</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">74</span>, <span class="cit-fpage">5486</span>–<span class="cit-lpage">5494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHo%26rft.auinit1%253DC.%2BK.%26rft.volume%253D74%26rft.issue%253D12%26rft.spage%253D5486%26rft.epage%253D5494%26rft.atitle%253DA%2BYeast-Based%2BGenetic%2BSystem%2Bfor%2BFunctional%2BAnalysis%2Bof%2BViral%2BmRNA%2BCapping%2BEnzymes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.12.5486-5494.2000%26rft_id%253Dinfo%253Apmid%252F10823853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMi81NDg2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Multicopy suppressors of temperature-sensitive mutations of yeast mRNA capping enzyme</span>. <abbr class="cit-jnl-abbrev">Gene Expr</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">331</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%2Bexpression%26rft.stitle%253DGene%2BExpr%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D331%26rft.epage%253D344%26rft.atitle%253DMulticopy%2Bsuppressors%2Bof%2Btemperature-sensitive%2Bmutations%2Bof%2Byeast%2BmRNA%2Bcapping%2Benzyme.%26rft_id%253Dinfo%253Apmid%252F8836740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8836740&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UZ15100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Structure-function analysis of yeast mRNA cap methyltransferase and high-copy suppression of conditional mutants by AdoMet synthase and the ubiquitin conjugating enzyme Cdc34p</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">155</span>, <span class="cit-fpage">1561</span>–<span class="cit-lpage">1576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1561%26rft.epage%253D1576%26rft.atitle%253DStructure-Function%2BAnalysis%2Bof%2BYeast%2BmRNA%2BCap%2BMethyltransferase%2Band%2BHigh-Copy%2BSuppression%2Bof%2BConditional%2BMutants%2Bby%2BAdoMet%2BSynthase%2Band%2Bthe%2BUbiquitin%2BConjugating%2BEnzyme%2BCdc34p%26rft_id%253Dinfo%253Apmid%252F10924457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE1NjEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.64" data-doi="10.1128/JVI.77.13.7300-7307.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Yeast-based genetic system for functional analysis of poxvirus mRNA cap methyltransferase</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">77</span>, <span class="cit-fpage">7300</span>–<span class="cit-lpage">7307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSaha%26rft.auinit1%253DN.%26rft.volume%253D77%26rft.issue%253D13%26rft.spage%253D7300%26rft.epage%253D7307%26rft.atitle%253DYeast-Based%2BGenetic%2BSystem%2Bfor%2BFunctional%2BAnalysis%2Bof%2BPoxvirus%2BmRNA%2BCap%2BMethyltransferase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.13.7300-7307.2003%26rft_id%253Dinfo%253Apmid%252F12805428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xMy83MzAwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.65" data-doi="10.1128/JVI.00061-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahola</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Structure-function analysis of severe acute respiratory syndrome coronavirus RNA cap guanine-N7-methyltransferase</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">87</span>, <span class="cit-fpage">6296</span>–<span class="cit-lpage">6305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00061-13%26rft_id%253Dinfo%253Apmid%252F23536667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMS82Mjk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.66" data-doi="10.1073/pnas.0808790106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tien</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahola</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-vol">106</span>, <span class="cit-fpage">3484</span>–<span class="cit-lpage">3489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808790106%26rft_id%253Dinfo%253Apmid%252F19208801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzQ4NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kast</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voges</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schroth</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Autoselection of cytoplasmic yeast virus like elements encoding toxin/antitoxin systems involves a nuclear barrier for immunity gene expression</span>. <abbr class="cit-jnl-abbrev">PLoS genetics</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">e1005005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D11%26rft.spage%253De1005005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.68" data-doi="10.1111/j.1574-6968.1999.tb08678.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaffrath</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meacock</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Genetic manipulation of Kluyveromyces lactis linear DNA plasmids: gene targeting and plasmid shuffles</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol Lett</abbr>, <span class="cit-vol">178</span>, <span class="cit-fpage">201</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BLetters%26rft.stitle%253DFEMS%2BMicrobiol%2BLett%26rft.aulast%253DSchaffrath%26rft.auinit1%253DR.%26rft.volume%253D178%26rft.issue%253D2%26rft.spage%253D201%26rft.epage%253D210%26rft.atitle%253DGenetic%2Bmanipulation%2Bof%2BKluyveromyces%2Blactis%2Blinear%2BDNA%2Bplasmids%253A%2Bgene%2Btargeting%2Band%2Bplasmid%2Bshuffles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1999.tb08678.x%26rft_id%253Dinfo%253Apmid%252F10499269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.1999.tb08678.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10499269&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082450700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.69" data-doi="10.1093/nar/26.9.2050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Accelerated mRNA decay in conditional mutants of yeast mRNA capping enzyme</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">2050</span>–<span class="cit-lpage">2057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.9.2050%26rft_id%253Dinfo%253Apmid%252F9547258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/26.9.2050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9547258&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073435300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">70.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.70" data-doi="10.1016/0022-2836(89)90107-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davison</span>,  <span class="cit-name-given-names">A.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moss</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Structure of vaccinia virus early promoters</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>, <span class="cit-vol">210</span>, <span class="cit-fpage">749</span>–<span class="cit-lpage">769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDavison%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D210%26rft.issue%253D4%26rft.spage%253D749%26rft.epage%253D769%26rft.atitle%253DStructure%2Bof%2Bvaccinia%2Bvirus%2Bearly%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252889%252990107-1%26rft_id%253Dinfo%253Apmid%252F2515286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(89)90107-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2515286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989CH42500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.71" data-doi="10.1093/nar/27.21.e31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">W.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mueller</span>,  <span class="cit-name-given-names">M.W.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">CapSelect: a highly sensitive method for 5’ CAP-dependent enrichment of full-length cDNA in PCR-mediated analysis of mRNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">e31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D27%26rft.issue%253D21%26rft.spage%253De31%26rft.epage%253De31%26rft.atitle%253DCapSelect%253A%2Ba%2Bhighly%2Bsensitive%2Bmethod%2Bfor%2B5%2527%2BCAP-dependent%2Benrichment%2Bof%2Bfull-length%2BcDNA%2Bin%2BPCR-mediated%2Banalysis%2Bof%2BmRNAs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.21.e31%26rft_id%253Dinfo%253Apmid%252F10518626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.21.e31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10518626&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.72" data-doi="10.1093/nar/gkw551"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramanathan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robb</span>,  <span class="cit-name-given-names">G.B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">S.H.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">mRNA capping: biological functions and applications</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">44</span>, <span class="cit-fpage">7511</span>–<span class="cit-lpage">7526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw551%26rft_id%253Dinfo%253Apmid%252F27317694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27317694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patil</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kennedy</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakthavachalu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bundschuh</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schoenberg</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Identification of cytoplasmic capping targets reveals a role for cap homeostasis in translation and mRNA stability</span>. <abbr class="cit-jnl-abbrev">Cell reports</abbr>, <span class="cit-vol">2</span>, <span class="cit-fpage">674</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D2%26rft.spage%253D674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ross</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">mRNA stability in mammalian cells</span>. <abbr class="cit-jnl-abbrev">Microbiol Rev</abbr>, <span class="cit-vol">59</span>, <span class="cit-fpage">423</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DRoss%26rft.auinit1%253DJ.%26rft.volume%253D59%26rft.issue%253D3%26rft.spage%253D423%26rft.epage%253D450%26rft.atitle%253DmRNA%2Bstability%2Bin%2Bmammalian%2Bcells%26rft_id%253Dinfo%253Apmid%252F7565413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1OS8zLzQyMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahn</span>,  <span class="cit-name-given-names">B.Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moss</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Capped poly(A) leaders of variable lengths at the 5’ ends of vaccinia virus late mRNAs</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">63</span>, <span class="cit-fpage">226</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DAhn%26rft.auinit1%253DB%2BY%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D226%26rft.epage%253D232%26rft.atitle%253DCapped%2Bpoly%2528A%2529%2Bleaders%2Bof%2Bvariable%2Blengths%2Bat%2Bthe%2B5%2527%2Bends%2Bof%2Bvaccinia%2Bvirus%2Blate%2BmRNAs.%26rft_id%253Dinfo%253Apmid%252F2462059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjYzLzEvMjI2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.76" data-doi="10.1016/0092-8674(87)90212-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Visca</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vos</span>,  <span class="cit-name-given-names">J.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stunnenberg</span>,  <span class="cit-name-given-names">H.G.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Discontinuous transcription or RNA processing of vaccinia virus late messengers results in a 5’ poly(A) leader</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">50</span>, <span class="cit-fpage">163</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D169%26rft.atitle%253DDiscontinuous%2Btranscription%2Bor%2BRNA%2Bprocessing%2Bof%2Bvaccinia%2Bvirus%2Blate%2Bmessengers%2Bresults%2Bin%2Ba%2B5%2527%2Bpoly%2528A%2529%2Bleader.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252887%252990212-1%26rft_id%253Dinfo%253Apmid%252F3594569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(87)90212-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3594569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J291500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">D.D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pickup</span>,  <span class="cit-name-given-names">D.J.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Messenger RNAs of a strongly-expressed late gene of cowpox virus contain 5’-terminal poly(A) sequences</span>. <abbr class="cit-jnl-abbrev">Embo J</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">3787</span>–<span class="cit-lpage">3794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmbo%2BJ%26rft.volume%253D6%26rft.spage%253D3787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.78" data-doi="10.1093/nar/gkl522"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gowda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alessi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pratt</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.L.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Robust analysis of 5’- transcript ends (5’-RATE): a novel technique for transcriptome analysis and genome annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">34</span>, <span class="cit-fpage">e126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl522%26rft_id%253Dinfo%253Apmid%252F17012272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17012272&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.79" data-doi="10.1038/nature07731"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper-Morgan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manley</span>,  <span class="cit-name-given-names">J.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Structure and function of the 5’-3’ exoribonuclease Rat1 and its activating partner Rai1</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">458</span>, <span class="cit-fpage">784</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXiang%26rft.auinit1%253DS.%26rft.volume%253D458%26rft.issue%253D7239%26rft.spage%253D784%26rft.epage%253D788%26rft.atitle%253DStructure%2Band%2Bfunction%2Bof%2Bthe%2B5%2527--%2526gt%253B3%2527%2Bexoribonuclease%2BRat1%2Band%2Bits%2Bactivating%2Bpartner%2BRai1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07731%26rft_id%253Dinfo%253Apmid%252F19194460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19194460&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265193600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.80" data-doi="10.1038/nature09338"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">C.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Identification of a quality-control mechanism for mRNA 5’-end capping</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">467</span>, <span class="cit-fpage">608</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJiao%26rft.auinit1%253DX.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D608%26rft.epage%253D611%26rft.atitle%253DIdentification%2Bof%2Ba%2Bquality-control%2Bmechanism%2Bfor%2BmRNA%2B5%2527-end%2Bcapping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09338%26rft_id%253Dinfo%253Apmid%252F20802481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20802481&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.81" data-doi="10.1261/rna.037309.112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bail</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Multiple Nudix family proteins possess mRNA decapping activity</span>. <abbr class="cit-jnl-abbrev">Rna</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRna%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.037309.112%26rft_id%253Dinfo%253Apmid%252F23353937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvMzkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.82" data-doi="10.1073/pnas.192445599"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carr-Schmid</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">The hDcp2 protein is a mammalian mRNA decapping enzyme</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-vol">99</span>, <span class="cit-fpage">12663</span>–<span class="cit-lpage">12668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.192445599%26rft_id%253Dinfo%253Apmid%252F12218187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjAvMTI2NjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-book" id="cit-325316v1.83"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> and </li><li><span class="cit-ed"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2007</span>) In <span class="cit-ed"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-ed"><span class="cit-name-surname">Klassen</span>,  <span class="cit-name-given-names">R.</span></span> (eds.), <span class="cit-source">Microbial Linear Plasmids. Microbiology Monographs</span>. <span class="cit-day">12</span> <span class="cit-month">May</span> <span class="cit-auth cit-collab">2007</span> ed. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>, Vol. <span class="cit-vol">7</span>, pp. <span class="cit-fpage">187</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobial%2BLinear%2BPlasmids.%2BMicrobiology%2BMonographs%26rft.volume%253D7%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.84" data-doi="10.1093/nar/24.10.1879"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schickel</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Helmig</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Kluyveromyces lactis killer system: analysis of cytoplasmic promoters of the linear plasmids</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">1879</span>–<span class="cit-lpage">1886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.10.1879%26rft_id%253Dinfo%253Apmid%252F8657569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/24.10.1879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8657569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UN98600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.85" data-doi="10.1002/bit.20525"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gudkov</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozerova</span>,  <span class="cit-name-given-names">M.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiryaev</span>,  <span class="cit-name-given-names">V.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Spirin</span>,  <span class="cit-name-given-names">A.S.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">5’-poly(A) sequence as an effective leader for translation in eukaryotic cell-free systems</span>. <abbr class="cit-jnl-abbrev">Biotechnology and bioengineering</abbr>, <span class="cit-vol">91</span>, <span class="cit-fpage">468</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DGudkov%26rft.auinit1%253DA.%2BT.%26rft.volume%253D91%26rft.issue%253D4%26rft.spage%253D468%26rft.epage%253D473%26rft.atitle%253D5%2527-poly%2528A%2529%2Bsequence%2Bas%2Ban%2Beffective%2Bleader%2Bfor%2Btranslation%2Bin%2Beukaryotic%2Bcell-free%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.20525%26rft_id%253Dinfo%253Apmid%252F15986488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bit.20525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15986488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230915400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.86" data-doi="10.1038/255033a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muthukrishnan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Both</span>,  <span class="cit-name-given-names">G.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furuichi</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shatkin</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">1975</span>) <span class="cit-article-title">5’-Terminal 7- methylguanosine in eukaryotic mRNA is required for translation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">255</span>, <span class="cit-fpage">33</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMuthukrishnan%26rft.auinit1%253DS.%26rft.volume%253D255%26rft.issue%253D5503%26rft.spage%253D33%26rft.epage%253D37%26rft.atitle%253D5%2527-Terminal%2B7-methylguanosine%2Bin%2Beukaryotic%2BmRNA%2Bis%2Brequired%2Bfor%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F255033a0%26rft_id%253Dinfo%253Apmid%252F165427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/255033a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=165427&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W309600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.87" data-doi="10.1007/BF00436963"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sor</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Fukuhara</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1985</span>) <span class="cit-article-title">Structure of a linear plasmid of the yeast Kluyveromyces lactis; Compact organization of the killer genome</span>. <abbr class="cit-jnl-abbrev">Current Genetics</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">147</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BGenetics%26rft.volume%253D9%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00436963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00436963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AJK2900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.88" data-doi="10.1093/nar/15.6.2417"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santiago</span>,  <span class="cit-name-given-names">T.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bettany</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">I.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Messenger RNA stability in Saccharomyces cerevisiae: the influence of translation and poly(A) tail length</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">2417</span>–<span class="cit-lpage">2429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSantiago%26rft.auinit1%253DT.%2BC.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D2417%26rft.epage%253D2429%26rft.atitle%253DMessemger%2BRNA%2Bstability%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bthe%2Binfluence%2Bof%2Btransaltion%2Band%2Bpoly%2B%2528A%2529%2Btail%2Blength%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.6.2417%26rft_id%253Dinfo%253Apmid%252F3550698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.6.2417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3550698&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.89" data-doi="10.1128/MCB.9.2.659"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peltz</span>,  <span class="cit-name-given-names">S.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ross</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">The poly(A)-poly(A)-binding protein complex is a major determinant of mRNA stability in vitro</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">659</span>–<span class="cit-lpage">670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBernstein%26rft.auinit1%253DP%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D659%26rft.epage%253D670%26rft.atitle%253DThe%2Bpoly%2528A%2529-poly%2528A%2529-binding%2Bprotein%2Bcomplex%2Bis%2Ba%2Bmajor%2Bdeterminant%2Bof%2BmRNA%2Bstability%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.9.2.659%26rft_id%253Dinfo%253Apmid%252F2565532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjkvMi82NTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.90" data-doi="10.1101/gad.5.11.2108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gallie</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">The cap and poly(A) tail function synergistically to regulate mRNA translational efficiency</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">2108</span>–<span class="cit-lpage">2116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGallie%26rft.auinit1%253DD%2BR%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253D2108%26rft.epage%253D2116%26rft.atitle%253DThe%2Bcap%2Band%2Bpoly%2528A%2529%2Btail%2Bfunction%2Bsynergistically%2Bto%2Bregulate%2BmRNA%2Btranslational%2Befficiency.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.5.11.2108%26rft_id%253Dinfo%253Apmid%252F1682219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiNS8xMS8yMTA4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.91" data-doi="10.1128/MCB.16.1.146"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanguay</span>,  <span class="cit-name-given-names">R.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gallie</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Translational efficiency is regulated by the length of the 3’ untranslated region</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">146</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTanguay%26rft.auinit1%253DR.%2BL.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D146%26rft.epage%253D156%26rft.atitle%253DTranslational%2Befficiency%2Bis%2Bregulated%2Bby%2Bthe%2Blength%2Bof%2Bthe%2B3%2527%2Buntranslated%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.1.146%26rft_id%253Dinfo%253Apmid%252F8524291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMTQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.92" data-doi="10.1128/MCB.10.7.3441"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Munroe</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">mRNA poly(A) tail, a 3’ enhancer of translational initiation</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">3441</span>–<span class="cit-lpage">3455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMunroe%26rft.auinit1%253DD%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D3441%26rft.epage%253D3455%26rft.atitle%253DmRNA%2Bpoly%2528A%2529%2Btail%252C%2Ba%2B3%2527%2Benhancer%2Bof%2Btranslational%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.7.3441%26rft_id%253Dinfo%253Apmid%252F1972543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzcvMzQ0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.93" data-doi="10.1101/gad.9.23.2997"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tarun</span>,  <span class="cit-name-given-names">S.Z.</span>,   <span class="cit-name-suffix">Jr.</span>.</span> and <span class="cit-auth"><span class="cit-name-surname">Sachs</span>,  <span class="cit-name-given-names">A.B.</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">A common function for mRNA 5’ and 3’ ends in translation initiation in yeast</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">2997</span>–<span class="cit-lpage">3007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTarun%26rft.auinit1%253DS%2BZ%26rft.volume%253D9%26rft.issue%253D23%26rft.spage%253D2997%26rft.epage%253D3007%26rft.atitle%253DA%2Bcommon%2Bfunction%2Bfor%2BmRNA%2B5%2527%2Band%2B3%2527%2Bends%2Bin%2Btranslation%2Binitiation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.23.2997%26rft_id%253Dinfo%253Apmid%252F7498795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8yMy8yOTk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.94" data-doi="10.1038/nature00935"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giaever</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connelly</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riles</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veronneau</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dow</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucau-Danila</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andre</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">418</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">95.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.95" data-doi="10.1128/MCB.7.3.998"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handschin</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Trachsel</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">mRNA cap-binding protein: cloning of the gene encoding protein synthesis initiation factor eIF-4E from Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">998</span>–<span class="cit-lpage">1003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAltmann%26rft.auinit1%253DM%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D998%26rft.epage%253D1003%26rft.atitle%253DmRNA%2Bcap-binding%2Bprotein%253A%2Bcloning%2Bof%2Bthe%2Bgene%2Bencoding%2Bprotein%2Bsynthesis%2Binitiation%2Bfactor%2BeIF-4E%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.3.998%26rft_id%253Dinfo%253Apmid%252F3550438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMy85OTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.96" data-doi="10.1128/MCB.8.8.3556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brenner</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakayama</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goebl</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toh-e</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-article-title">CDC33 encodes mRNA cap-binding protein eIF-4E of Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">3556</span>–<span class="cit-lpage">3559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrenner%26rft.auinit1%253DC%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D3556%26rft.epage%253D3559%26rft.atitle%253DCDC33%2Bencodes%2BmRNA%2Bcap-binding%2Bprotein%2BeIF-4E%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.8.3556%26rft_id%253Dinfo%253Apmid%252F3062383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjgvOC8zNTU2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">1981</span>) <span class="cit-article-title">Intergeneric transfer of deoxyribonucleic acid killer plasmids, pGKl1 and pGKl2, from Kluyveromyces lactis into Saccharomyces cerevisiae by cell fusion</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr>, <span class="cit-vol">147</span>, <span class="cit-fpage">155</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGunge%26rft.auinit1%253DN%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D155%26rft.epage%253D160%26rft.atitle%253DIntergeneric%2Btransfer%2Bof%2Bdeoxyribonucleic%2Bacid%2Bkiller%2Bplasmids%252C%2BpGKl1%2Band%2BpGKl2%252C%2Bfrom%2BKluyveromyces%2Blactis%2Binto%2BSaccharomyces%2Bcerevisiae%2Bby%2Bcell%2Bfusion.%26rft_id%253Dinfo%253Apmid%252F7016841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTQ3LzEvMTU1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.98"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wieczorek</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niedzwiecka-Kornas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chlebicka</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jankowska</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiraga</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stepinski</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dadlez</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drabent</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darzynkiewicz</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stolarski</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Fluorescence studies on association of human translation initiation factor eIF4E with mRNA cap-analogues</span>. <abbr class="cit-jnl-abbrev">Zeitschrift fur Naturforschung. C, Journal of biosciences</abbr>, <span class="cit-vol">54</span>, <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZeitschrift%2Bfur%2BNaturforschung.%2BC%252C%2BJournal%2Bof%2Bbiosciences%26rft.volume%253D54%26rft.spage%253D278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.99" data-doi="10.7554/eLife.21907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamarkin-Ben-Harush</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasseur</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debart</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>,  <span class="cit-name-given-names">I.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Dikstein</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Cap-proximal nucleotides via differential eIF4E binding and alternative promoter usage mediate translational response to energy stress</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">e21907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De21907%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.21907%26rft_id%253Dinfo%253Apmid%252F28177284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.21907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28177284&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.100" data-doi="10.1093/nar/gkp981"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mokrejs</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masek</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vopalensky</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hlubucek</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delbos</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">IRESite—a tool for the examination of viral and cellular internal ribosome entry sites</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">D131</span>–<span class="cit-lpage">D136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp981%26rft_id%253Dinfo%253Apmid%252F19917642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917642&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276399100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.101" data-doi="10.1093/nar/17.15.5923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altmann</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Trachsel</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Altered mRNA cap recognition activity of initiation factor 4E in the yeast cell cycle division mutant cdc33</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">5923</span>–<span class="cit-lpage">5931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAltmann%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D15%26rft.spage%253D5923%26rft.epage%253D5931%26rft.atitle%253DAltered%2BmRNA%2Bcap%2Brecognition%2Bactivity%2Bof%2Binitiation%2Bfactor%2B4E%2Bin%2Bthe%2Byeast%2Bcell%2Bcyde%2Bdivision%2Bmutant%2Bcdc33%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.15.5923%26rft_id%253Dinfo%253Apmid%252F2671936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.15.5923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2671936&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AK53100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.102" data-doi="10.1128/MCB.9.10.4467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Trachsel</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Translation in Saccharomyces cerevisiae: Initiation Factor 4E-Dependent Cell-Free System</span>. <abbr class="cit-jnl-abbrev">Molecular and Cellular Biology</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">4467</span>–<span class="cit-lpage">4472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAltmann%26rft.auinit1%253DM%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D4467%26rft.epage%253D4472%26rft.atitle%253DTranslation%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Binitiation%2Bfactor%2B4E-dependent%2Bcell-free%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.9.10.4467%26rft_id%253Dinfo%253Apmid%252F2685552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvNDQ2NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">103.</span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103. in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.103" data-doi="10.1111/j.1567-1364.2010.00629.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feketova</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masek</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vopalensky</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Ambiguous decoding of the CUG codon alters the functionality of the Candida albicans translation initiation factor 4E</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">558</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2010.00629.x%26rft_id%253Dinfo%253Apmid%252F20491942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2010.00629.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20491942&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sugisaki</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">1984</span>) <span class="cit-article-title">Characterization of a novel killer toxin encoded by a double-stranded linear DNA plasmid of Kluyveromyces lactis</span>. <abbr class="cit-jnl-abbrev">Eur J Biochem</abbr>, <span class="cit-vol">141</span>, <span class="cit-fpage">241</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DSugisaki%26rft.auinit1%253DY%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D241%26rft.epage%253D245%26rft.atitle%253DCharacterization%2Bof%2Ba%2Bnovel%2Bkiller%2Btoxin%2Bencoded%2Bby%2Ba%2Bdouble-stranded%2Blinear%2BDNA%2Bplasmid%2Bof%2BKluyveromyces%2Blactis.%26rft_id%253Dinfo%253Apmid%252F6734597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6734597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zakharov</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurchenko</span>,  <span class="cit-name-given-names">L.V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yarovot</span>,  <span class="cit-name-given-names">B.F.</span></span> (<span class="cit-pub-date">1969</span>) <span class="cit-article-title">Cytoduction: The autonomous transfer of cytoplasmic hereditary factors during pairing of yeast cells</span>. <abbr class="cit-jnl-abbrev">Genetika</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">136</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetika%26rft.volume%253D5%26rft.spage%253D136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valis</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masek</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novotna</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Janderova</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Immunity to killer toxin K1 is connected with the Golgi-to-vacuole protein degradation pathway</span>. <abbr class="cit-jnl-abbrev">Folia Microbiol (Praha)</abbr>, <span class="cit-vol">51</span>, <span class="cit-fpage">196</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFolia%2BMicrobiologica%2B%2528Prague%2529%26rft.stitle%253DFolia%2BMicrobiologica%2B%2528Prague%2529%26rft.aulast%253DValis%26rft.auinit1%253DK.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D196%26rft.epage%253D202%26rft.atitle%253DImmunity%2Bto%2Bkiller%2Btoxin%2BK1%2Bis%2Bconnected%2Bwith%2Bthe%2BGolgi-to-vacuole%2Bprotein%2Bdegradation%2Bpathway.%26rft_id%253Dinfo%253Apmid%252F17004650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17004650&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.107" data-doi="10.1111/j.1432-1033.1979.tb12847.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palfree</span>,  <span class="cit-name-given-names">R.G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bussey</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1979</span>) <span class="cit-article-title">Yeast killer toxin: purification and characterisation of the protein toxin from Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Eur J Biochem</abbr>, <span class="cit-vol">93</span>, <span class="cit-fpage">487</span>–<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DPalfree%26rft.auinit1%253DR.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D493%26rft.atitle%253DYeast%2Bkiller%2Btoxin%253A%2Bpurification%2Band%2Bcharacterisation%2Bof%2Bthe%2Bprotein%2Btoxin%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1432-1033.1979.tb12847.x%26rft_id%253Dinfo%253Apmid%252F33806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1432-1033.1979.tb12847.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=33806&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GK56100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.108" data-doi="10.1002/yea.1806"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Partow</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siewers</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bjorn</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Maury</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Characterization of different promoters for designing a new expression vector in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">955</span>–<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DPartow%26rft.auinit1%253DS.%26rft.volume%253D27%26rft.issue%253D11%26rft.spage%253D955%26rft.epage%253D964%26rft.atitle%253DCharacterization%2Bof%2Bdifferent%2Bpromoters%2Bfor%2Bdesigning%2Ba%2Bnew%2Bexpression%2Bvector%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1806%26rft_id%253Dinfo%253Apmid%252F20625983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20625983&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284006000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.109" data-doi="10.1073/pnas.0804940105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shirokikh</span>,  <span class="cit-name-given-names">N.E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Spirin</span>,  <span class="cit-name-given-names">A.S.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Poly(A) leader of eukaryotic mRNA bypasses the dependence of translation on initiation factors</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-vol">105</span>, <span class="cit-fpage">10738</span>–<span class="cit-lpage">10743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804940105%26rft_id%253Dinfo%253Apmid%252F18658239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzMxLzEwNzM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">110.</span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110. in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tahiri-Alaoui</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadigh</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popplestone</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kgosana</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">L.P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nair</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Poly(A) binding protein 1 enhances cap-independent translation initiation of neurovirulence factor from avian herpesvirus</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">e114466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D9%26rft.spage%253De114466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">111.</span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111. in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.111" data-doi="10.1093/nar/gki1014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">G.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bag</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">The autoregulatory translational control element of poly(A)-binding protein mRNA forms a heteromeric ribonucleoprotein complex</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">33</span>, <span class="cit-fpage">7074</span>–<span class="cit-lpage">7089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPatel%26rft.auinit1%253DG.%2BP.%26rft.volume%253D33%26rft.issue%253D22%26rft.spage%253D7074%26rft.epage%253D7089%26rft.atitle%253DThe%2Bautoregulatory%2Btranslational%2Bcontrol%2Belement%2Bof%2Bpoly%2528A%2529-binding%2Bprotein%2BmRNA%2Bforms%2Ba%2Bheteromeric%2Bribonucleoprotein%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki1014%26rft_id%253Dinfo%253Apmid%252F16356927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki1014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16356927&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234436200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">112.</span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference 112. in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.112" data-doi="10.1016/S0014-5793(03)00620-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melo</span>,  <span class="cit-name-given-names">E.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Melo Neto</span>,  <span class="cit-name-given-names">O.P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Martins de Sa</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Adenosine-rich elements present in the 5’-untranslated region of PABP mRNA can selectively reduce the abundance and translation of CAT mRNAs in vivo</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>, <span class="cit-vol">546</span>, <span class="cit-fpage">329</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DMelo%26rft.auinit1%253DE.%2BO.%26rft.volume%253D546%26rft.issue%253D2-3%26rft.spage%253D329%26rft.epage%253D334%26rft.atitle%253DAdenosine-rich%2Belements%2Bpresent%2Bin%2Bthe%2B5%2527-untranslated%2Bregion%2Bof%2BPABP%2BmRNA%2Bcan%2Bselectively%2Breduce%2Bthe%2Babundance%2Band%2Btranslation%2Bof%2BCAT%2BmRNAs%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252803%252900620-3%26rft_id%253Dinfo%253Apmid%252F12832063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(03)00620-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12832063&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183964000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">113.</span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference 113. in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.113" data-doi="10.1534/genetics.111.132068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacKay</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miura</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Translation initiation: a regulatory role for poly(A) tracts in front of the AUG codon in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">189</span>, <span class="cit-fpage">469</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132068%26rft_id%253Dinfo%253Apmid%252F21840854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNDY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">114.</span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference 114. in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.114" data-doi="10.1016/0092-8674(86)90557-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sachs</span>,  <span class="cit-name-given-names">A.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bond</span>,  <span class="cit-name-given-names">M.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kornberg</span>,  <span class="cit-name-given-names">R.D.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">A single gene from yeast for both nuclear and cytoplasmic polyadenylate-binding proteins: domain structure and expression</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">45</span>, <span class="cit-fpage">827</span>–<span class="cit-lpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSachs%26rft.auinit1%253DA.%2BB.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D827%26rft.epage%253D835%26rft.atitle%253DA%2Bsingle%2Bgene%2Bfrom%2Byeast%2Bfor%2Bboth%2Bnuclear%2Band%2Bcytoplasmic%2Bpolyadenylate-binding%2Bproteins%253A%2Bdomain%2Bstructure%2Band%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990557-X%26rft_id%253Dinfo%253Apmid%252F3518950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90557-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3518950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986C912400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">115.</span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference 115. in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.115" data-doi="10.1101/gad.1262905"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kahvejian</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Svitkin</span>,  <span class="cit-name-given-names">Y.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sukarieh</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">M’Boutchou</span>,  <span class="cit-name-given-names">M.N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Mammalian poly(A)-binding protein is a eukaryotic translation initiation factor, which acts via multiple mechanisms</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">104</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKahvejian%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D104%26rft.epage%253D113%26rft.atitle%253DMammalian%2Bpoly%2528A%2529-binding%2Bprotein%2Bis%2Ba%2Beukaryotic%2Btranslation%2Binitiation%2Bfactor%252C%2Bwhich%2Bacts%2Bvia%2Bmultiple%2Bmechanisms%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1262905%26rft_id%253Dinfo%253Apmid%252F15630022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTkvMS8xMDQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">116.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.116" data-doi="10.1186/s13059-014-0559-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Costello</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castelli</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rowe</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kershaw</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talavera</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohammad-Qureshi</span>,  <span class="cit-name-given-names">S.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">P.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavitt</span>,  <span class="cit-name-given-names">G.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubbard</span>,  <span class="cit-name-given-names">S.J.</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Global mRNA selection mechanisms for translation initiation</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D16%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0559-z%26rft_id%253Dinfo%253Apmid%252F25650959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0559-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25650959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">117.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.117" data-doi="10.1042/BJ20091571"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">R.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gray</span>,  <span class="cit-name-given-names">N.K.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Poly(A)-binding protein (PABP): a common viral target</span>. <abbr class="cit-jnl-abbrev">Biochem J</abbr>, <span class="cit-vol">426</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20091571%26rft_id%253Dinfo%253Apmid%252F20102337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6NzoiNDI2LzEvMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">118.</span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference 118. in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.118" data-doi="10.1128/JVI.00896-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garabedian</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palmenberg</span>,  <span class="cit-name-given-names">A.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mohr</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Site-specific cleavage of the host poly(A) binding protein by the encephalomyocarditis virus 3C proteinase stimulates viral replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">86</span>, <span class="cit-fpage">10686</span>–<span class="cit-lpage">10694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00896-12%26rft_id%253Dinfo%253Apmid%252F22837200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8xOS8xMDY4NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">119.</span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference 119. in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.119" data-doi="10.1128/MCB.18.12.7383"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mangus</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amrani</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Pbp1p, a factor interacting with Saccharomyces cerevisiae poly(A)-binding protein, regulates polyadenylation</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">18</span>, <span class="cit-fpage">7383</span>–<span class="cit-lpage">7396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMangus%26rft.auinit1%253DD.%2BA.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D7383%26rft.epage%253D7396%26rft.atitle%253DPbp1p%252C%2Ba%2BFactor%2BInteracting%2BwithSaccharomyces%2Bcerevisiae%2BPoly%2528A%2529-Binding%2BProtein%252C%2BRegulates%2BPolyadenylation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.12.7383%26rft_id%253Dinfo%253Apmid%252F9819425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi83MzgzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">120.</span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference 120. in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.120" data-doi="10.1128/MCB.7.9.3268"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sachs</span>,  <span class="cit-name-given-names">A.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">R.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kornberg</span>,  <span class="cit-name-given-names">R.D.</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">A single domain of yeast poly(A)-binding protein is necessary and sufficient for RNA binding and cell viability</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">3268</span>–<span class="cit-lpage">3276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSachs%26rft.auinit1%253DA%2BB%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253D3268%26rft.epage%253D3276%26rft.atitle%253DA%2Bsingle%2Bdomain%2Bof%2Byeast%2Bpoly%2528A%2529-binding%2Bprotein%2Bis%2Bnecessary%2Band%2Bsufficient%2Bfor%2BRNA%2Bbinding%2Band%2Bcell%2Bviability.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.9.3268%26rft_id%253Dinfo%253Apmid%252F3313012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjcvOS8zMjY4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">121.</span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference 121. in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.121" data-doi="10.1093/nar/24.13.2519"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guldener</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heck</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielder</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beinhauer</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hegemann</span>,  <span class="cit-name-given-names">J.H.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">A new efficient gene disruption cassette for repeated use in budding yeast</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">2519</span>–<span class="cit-lpage">2524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.13.2519%26rft_id%253Dinfo%253Apmid%252F8692690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/24.13.2519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8692690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UX27000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">122.</span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference 122. in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.122" data-doi="10.1093/emboj/18.15.4321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mayes</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdone</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legrain</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Beggs</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Characterization of Sm-like proteins in yeast and their association with U6 snRNA</span>. <abbr class="cit-jnl-abbrev">Embo J</abbr>, <span class="cit-vol">18</span>, <span class="cit-fpage">4321</span>–<span class="cit-lpage">4331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DMayes%26rft.auinit1%253DA.%2BE.%26rft.volume%253D18%26rft.issue%253D15%26rft.spage%253D4321%26rft.epage%253D4331%26rft.atitle%253DCharacterization%2Bof%2BSm-like%2Bproteins%2Bin%2Byeast%2Band%2Btheir%2Bassociation%2Bwith%2BU6%2BsnRNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.15.4321%26rft_id%253Dinfo%253Apmid%252F10428970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTgvMTUvNDMyMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">123.</span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference 123. in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.123" data-doi="10.1002/yea.1843"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ida</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furusawa</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirasawa</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Shimizu</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Comprehensive phenotypic analysis of single-gene deletion and overexpression strains of Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">28</span>, <span class="cit-fpage">349</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DYoshikawa%26rft.auinit1%253DK.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D349%26rft.epage%253D361%26rft.atitle%253DComprehensive%2Bphenotypic%2Banalysis%2Bof%2Bsingle-gene%2Bdeletion%2Band%2Boverexpression%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1843%26rft_id%253Dinfo%253Apmid%252F21341307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21341307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289953700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">124.</span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference 124. in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.124"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jungfleisch</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blasco-Moreno</span>,  <span class="cit-name-given-names">B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Diez</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Use of Cellular Decapping Activators by Positive-Strand RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D8%26rft.spage%253D340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">125.</span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference 125. in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.125" data-doi="10.1534/genetics.111.137265"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">RNA degradation in Saccharomyces cerevisae</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">191</span>, <span class="cit-fpage">671</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DParker%26rft.auinit1%253DR.%26rft.volume%253D191%26rft.issue%253D3%26rft.spage%253D671%26rft.epage%253D702%26rft.atitle%253DRNA%2BDegradation%2Bin%2BSaccharomyces%2Bcerevisae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137265%26rft_id%253Dinfo%253Apmid%252F22785621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvNjcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">126.</span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference 126. in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mulder</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seamons</span>,  <span class="cit-name-given-names">R.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Belsham</span>,  <span class="cit-name-given-names">G.J.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Vaccinia virus protein synthesis has a low requirement for the intact translation initiation factor eIF4F, the cap-binding complex, within infected cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-vol">72</span>, <span class="cit-fpage">8813</span>–<span class="cit-lpage">8819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMulder%26rft.auinit1%253DJ.%26rft.volume%253D72%26rft.issue%253D11%26rft.spage%253D8813%26rft.epage%253D8819%26rft.atitle%253DVaccinia%2BVirus%2BProtein%2BSynthesis%2BHas%2Ba%2BLow%2BRequirement%2Bfor%2Bthe%2BIntact%2BTranslation%2BInitiation%2BFactor%2BeIF4F%252C%2Bthe%2BCap-Binding%2BComplex%252C%2Bwithin%2BInfected%2BCells%26rft_id%253Dinfo%253Apmid%252F9765426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Mi8xMS84ODEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">127.</span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference 127. in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.127"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ostrowski</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">A.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mekhail</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Ataxin-2: From RNA Control to Human Health and Disease</span>. <abbr class="cit-jnl-abbrev">Genes (Basel)</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2528Basel%2529%26rft.volume%253D8%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">128.</span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference 128. in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.128" data-doi="10.1128/MCB.24.10.4196-4206.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mangus</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McSweeney</span>,  <span class="cit-name-given-names">J.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Identification of factors regulating poly(A) tail synthesis and maturation</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">4196</span>–<span class="cit-lpage">4206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMangus%26rft.auinit1%253DD.%2BA.%26rft.volume%253D24%26rft.issue%253D10%26rft.spage%253D4196%26rft.epage%253D4206%26rft.atitle%253DIdentification%2Bof%2BFactors%2BRegulating%2BPoly%2528A%2529%2BTail%2BSynthesis%2Band%2BMaturation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.10.4196-4206.2004%26rft_id%253Dinfo%253Apmid%252F15121841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMC80MTk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">129.</span><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference 129. in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.129" data-doi="10.1016/j.jbiotec.2016.02.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spohner</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaum</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quitmann</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Czermak</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Kluyveromyces lactis: An emerging tool in biotechnology</span>. <abbr class="cit-jnl-abbrev">J Biotechnol</abbr>, <span class="cit-vol">222</span>, <span class="cit-fpage">104</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiotechnol%26rft.volume%253D222%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2016.02.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2016.02.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">130.</span><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference 130. in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.130" data-doi="10.1093/nar/gnh007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coleman</span>,  <span class="cit-name-given-names">T.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Superior 5’ homogeneity of RNA from ATP-initiated transcription under the T7 phi 2.5 promoter</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">32</span>, <span class="cit-fpage">e14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DColeman%26rft.auinit1%253DT.%2BM.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253De14%26rft.epage%253De14%26rft.atitle%253DSuperior%2B5%2527%2Bhomogeneity%2Bof%2BRNA%2Bfrom%2BATP-initiated%2Btranscription%2Bunder%2Bthe%2BT7%2B%257Bphi%257D2.5%2Bpromoter%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnh007%26rft_id%253Dinfo%253Apmid%252F14744982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gnh007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14744982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">Supplemental References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference  in text" id="ref-131">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.131" data-doi="10.1128/MCB.9.10.4467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Trachsel</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">Translation in Saccharomyces cerevisiae: Initiation Factor 4E-Dependent Cell-Free System</span>. <abbr class="cit-jnl-abbrev">Molecular and Cellular Biology</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">4467</span>–<span class="cit-lpage">4472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAltmann%26rft.auinit1%253DM%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D4467%26rft.epage%253D4472%26rft.atitle%253DTranslation%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Binitiation%2Bfactor%2B4E-dependent%2Bcell-free%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.9.10.4467%26rft_id%253Dinfo%253Apmid%252F2685552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvNDQ2NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzE3LzMyNTMxNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-132-1" title="View reference  in text" id="ref-132">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.132" data-doi="10.1093/nar/17.15.5923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altmann</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Trachsel</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">Altered mRNA cap recognition activity of initiation factor 4E in the yeast cell cycle division mutant cdc33</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">5923</span>–<span class="cit-lpage">5931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAltmann%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D15%26rft.spage%253D5923%26rft.epage%253D5931%26rft.atitle%253DAltered%2BmRNA%2Bcap%2Brecognition%2Bactivity%2Bof%2Binitiation%2Bfactor%2B4E%2Bin%2Bthe%2Byeast%2Bcell%2Bcyde%2Bdivision%2Bmutant%2Bcdc33%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.15.5923%26rft_id%253Dinfo%253Apmid%252F2671936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.15.5923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2671936&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AK53100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-133-1" title="View reference  in text" id="ref-133">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.133" data-doi="10.1093/nar/gnh007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coleman</span>,  <span class="cit-name-given-names">T.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Superior 5’ homogeneity of RNA from ATP-initiated transcription under the T7 phi 2.5 promoter</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">e14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DColeman%26rft.auinit1%253DT.%2BM.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253De14%26rft.epage%253De14%26rft.atitle%253DSuperior%2B5%2527%2Bhomogeneity%2Bof%2BRNA%2Bfrom%2BATP-initiated%2Btranscription%2Bunder%2Bthe%2BT7%2B%257Bphi%257D2.5%2Bpromoter%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnh007%26rft_id%253Dinfo%253Apmid%252F14744982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gnh007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14744982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-134-1" title="View reference  in text" id="ref-134">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.134" data-doi="10.1016/S0076-6879(02)50957-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gietz</span>,  <span class="cit-name-given-names">R.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Transformation of yeast by lithium acetate/single-stranded carrier DNA/polyethylene glycol method</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr> <span class="cit-vol">350</span>, <span class="cit-fpage">87</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DGietz%26rft.auinit1%253DR.%2BD.%26rft.volume%253D350%26rft.spage%253D87%26rft.epage%253D96%26rft.atitle%253DTransformation%2Bof%2Byeast%2Bby%2Blithium%2Bacetate%252Fsingle-stranded%2Bcarrier%2BDNA%252Fpolyethylene%2Bglycol%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252802%252950957-5%26rft_id%253Dinfo%253Apmid%252F12073338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(02)50957-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12073338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176466300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-135-1" title="View reference  in text" id="ref-135">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.135" data-doi="10.1093/nar/30.6.e23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gueldener</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heinisch</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koehler</span>,  <span class="cit-name-given-names">G.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voss</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hegemann</span>,  <span class="cit-name-given-names">J.H.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">A second set of loxP marker cassettes for Cre-mediated multiple gene knockouts in budding yeast</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">e23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGueldener%26rft.auinit1%253DU.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253De23%26rft.epage%253De23%26rft.atitle%253DA%2Bsecond%2Bset%2Bof%2BloxP%2Bmarker%2Bcassettes%2Bfor%2BCre-mediated%2Bmultiple%2Bgene%2Bknockouts%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.6.e23%26rft_id%253Dinfo%253Apmid%252F11884642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.6.e23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11884642&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-136-1" title="View reference  in text" id="ref-136">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.136" data-doi="10.1093/nar/24.13.2519"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guldener</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heck</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielder</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beinhauer</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hegemann</span>,  <span class="cit-name-given-names">J.H.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">A new efficient gene disruption cassette for repeated use in budding yeast</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">2519</span>–<span class="cit-lpage">2524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F24.13.2519%26rft_id%253Dinfo%253Apmid%252F8692690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/24.13.2519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8692690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UX27000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.137"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamaru</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">1981</span>). <span class="cit-article-title">Isolation and Characterization of Linear Deoxyribonucleic Acid Plasmids from Kluyveromyces lactis and the Plasmid-Associated Killer Character</span>. <abbr class="cit-jnl-abbrev">Journal of Bacteriology</abbr> <span class="cit-vol">145</span>, <span class="cit-fpage">382</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGunge%26rft.auinit1%253DN%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D382%26rft.epage%253D390%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Blinear%2Bdeoxyribonucleic%2Bacid%2Bplasmids%2Bfrom%2BKluyveromyces%2Blactis%2Band%2Bthe%2Bplasmid-associated%2Bkiller%2Bcharacter.%26rft_id%253Dinfo%253Apmid%252F6257636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTQ1LzEvMzgyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunge</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yamane</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1984</span>). <span class="cit-article-title">Incompatibility of linear DNA killer plasmids pGKL1 and pGKL2 from Kluyveromyces lactis with mitochondrial DNA from Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-vol">159</span>, <span class="cit-fpage">533</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGunge%26rft.auinit1%253DN%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D533%26rft.epage%253D539%26rft.atitle%253DIncompatibility%2Bof%2Blinear%2BDNA%2Bkiller%2Bplasmids%2BpGKL1%2Band%2BpGKL2%2Bfrom%2BKluyveromyces%2Blactis%2Bwith%2Bmitochondrial%2BDNA%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F6378881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNTMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-139-1" title="View reference  in text" id="ref-139">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.139" data-doi="10.1261/rna.5690503"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Piccirillo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khanna</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Functional characterization of the mammalian mRNA decapping enzyme hDcp2</span>. <abbr class="cit-jnl-abbrev">Rna</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1138</span>–<span class="cit-lpage">1147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DPICCIRILLO%26rft.auinit1%253DC.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D1138%26rft.epage%253D1147%26rft.atitle%253DFunctional%2Bcharacterization%2Bof%2Bthe%2Bmammalian%2BmRNA%2Bdecapping%2Benzyme%2BhDcp2%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5690503%26rft_id%253Dinfo%253Apmid%252F12923261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjkvOS8xMTM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMTcvMzI1MzE2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Structure-Function Analysis of Yeast mRNA Cap Methyltransferase and High-Copy Suppression of Conditional Mutants by AdoMet Synthase and the Ubiquitin Conjugating Enzyme Cdc34p</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>, <span class="cit-fpage">1561</span>–<span class="cit-lpage">1576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D1561%26rft.epage%253D1576%26rft.atitle%253DStructure-Function%2BAnalysis%2Bof%2BYeast%2BmRNA%2BCap%2BMethyltransferase%2Band%2BHigh-Copy%2BSuppression%2Bof%2BConditional%2BMutants%2Bby%2BAdoMet%2BSynthase%2Band%2Bthe%2BUbiquitin%2BConjugating%2BEnzyme%2BCdc34p%26rft_id%253Dinfo%253Apmid%252F10924457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS80LzE1NjEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.141"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwer</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Shuman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Multicopy suppressors of temperature-sensitive mutations of yeast mRNA capping enzyme</span>. <abbr class="cit-jnl-abbrev">Gene Expr</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">331</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%2Bexpression%26rft.stitle%253DGene%2BExpr%26rft.aulast%253DSchwer%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D331%26rft.epage%253D344%26rft.atitle%253DMulticopy%2Bsuppressors%2Bof%2Btemperature-sensitive%2Bmutations%2Bof%2Byeast%2BmRNA%2Bcapping%2Benzyme.%26rft_id%253Dinfo%253Apmid%252F8836740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8836740&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UZ15100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-142-1" title="View reference  in text" id="ref-142">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.142"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valis</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masek</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novotna</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pospisek</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Janderova</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Immunity to killer toxin K1 is connected with the Golgi-to-vacuole protein degradation pathway</span>. <abbr class="cit-jnl-abbrev">Folia Microbiol (Praha)</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">196</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFolia%2BMicrobiologica%2B%2528Prague%2529%26rft.stitle%253DFolia%2BMicrobiologica%2B%2528Prague%2529%26rft.aulast%253DValis%26rft.auinit1%253DK.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D196%26rft.epage%253D202%26rft.atitle%253DImmunity%2Bto%2Bkiller%2Btoxin%2BK1%2Bis%2Bconnected%2Bwith%2Bthe%2BGolgi-to-vacuole%2Bprotein%2Bdegradation%2Bpathway.%26rft_id%253Dinfo%253Apmid%252F17004650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17004650&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-143-1" title="View reference  in text" id="ref-143">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.143" data-doi="10.1073/pnas.192445599"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carr-Schmid</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">The hDcp2 protein is a mammalian mRNA decapping enzyme</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">99</span>, <span class="cit-fpage">12663</span>–<span class="cit-lpage">12668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.192445599%26rft_id%253Dinfo%253Apmid%252F12218187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjAvMTI2NjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8xNy8zMjUzMTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.144" data-doi="10.1099/00221287-51-1-115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">D.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bevan</span>,  <span class="cit-name-given-names">E.A.</span></span> (<span class="cit-pub-date">1968</span>). <span class="cit-article-title">Studies on Nature of Killer Factor Produced by Saccharomyces Cerevisiae</span>. <abbr class="cit-jnl-abbrev">Journal of General Microbiology</abbr> <span class="cit-vol">51</span>, <span class="cit-fpage">115</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DWOODS%26rft.auinit1%253DD.%2BR.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D126%26rft.atitle%253DStudies%2Bon%2Bthe%2BNature%2Bof%2Bthe%2BKiller%2BFactor%2BProduced%2Bby%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-51-1-115%26rft_id%253Dinfo%253Apmid%252F5653223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-51-1-115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5653223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968A941700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-325316v1.145" data-doi="10.1038/nature07731"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper-Morgan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manley</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2009a</span>). <span class="cit-article-title">Structure and function of the 5’→3’ exoribonuclease Rat1 and its activating partner Rai1</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>, <span class="cit-fpage">784</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXiang%26rft.auinit1%253DS.%26rft.volume%253D458%26rft.issue%253D7239%26rft.spage%253D784%26rft.epage%253D788%26rft.atitle%253DStructure%2Band%2Bfunction%2Bof%2Bthe%2B5%2527--%2526gt%253B3%2527%2Bexoribonuclease%2BRat1%2Band%2Bits%2Bactivating%2Bpartner%2BRai1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07731%26rft_id%253Dinfo%253Apmid%252F19194460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19194460&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265193600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-146-1" title="View reference  in text" id="ref-146">↵</a><div class="cit ref-cit ref-journal" id="cit-325316v1.146" data-doi="10.1038/nature07731"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper-Morgan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiledjian</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manley</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2009b</span>). <span class="cit-article-title">Structure and function of the 5’→3’ exoribonuclease Rati and its activating partner Rail</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>, <span class="cit-fpage">784</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXiang%26rft.auinit1%253DS.%26rft.volume%253D458%26rft.issue%253D7239%26rft.spage%253D784%26rft.epage%253D788%26rft.atitle%253DStructure%2Band%2Bfunction%2Bof%2Bthe%2B5%2527--%2526gt%253B3%2527%2Bexoribonuclease%2BRat1%2Band%2Bits%2Bactivating%2Bpartner%2BRai1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07731%26rft_id%253Dinfo%253Apmid%252F19194460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19194460&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F17%2F325316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265193600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/325316v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F325316v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/324566v1" title="Strain-level overlap between infant and hospital fungal microbiomes revealed through de novo assembly of eukaryotic genomes from metagenomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 17, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/325316v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/325316v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/325316v1" class="active" data-icon-position="" data-hide-link-title="0">Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/99373" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-s4lFApbpaHKdGFsmZPZ5C3vBHdk9EDtfP7qYo1D0Sxk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="99373" id="node99373--2" data-pisa="biorxiv;325316v1" data-pisa-master="biorxiv;325316" data-seqnum="99373" data-apath="/biorxiv/early/2018/05/17/325316.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Václav</span> <span class="nlm-surname">Vopálenský</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Sýkora</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Tomáš</span> <span class="nlm-surname">Mašek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Pospíšek</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">325316; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/325316 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/325316v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;title=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;title=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;text=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;title=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;t=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;title=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;title=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="99373" id="node99373--3" data-pisa="biorxiv;325316v1" data-pisa-master="biorxiv;325316" data-seqnum="99373" data-apath="/biorxiv/early/2018/05/17/325316.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Messenger RNAs transcribed from yeast linear cytoplasmic plasmids possess unconventional 5’ and 3’ UTRs and suggest a novel mechanism of translation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Václav</span> <span class="nlm-surname">Vopálenský</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Sýkora</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Tomáš</span> <span class="nlm-surname">Mašek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Pospíšek</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">325316; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/325316 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Vop%C3%A1lensk%C3%BD%2C+V%C3%A1clav&amp;rft.au=S%C3%BDkora%2C+Michal&amp;rft.au=Ma%C5%A1ek%2C+Tom%C3%A1%C5%A1&amp;rft.au=Posp%C3%AD%C5%A1ek%2C+Martin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/99373/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/99373/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/99373/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/99373/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/99373/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/99373/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/99373/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/99373/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/99373/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/99373/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/99373/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/99373/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;count=horizontal&amp;via=&amp;text=Messenger%20RNAs%20transcribed%20from%20yeast%20linear%20cytoplasmic%20plasmids%20possess%20unconventional%205%E2%80%99%20and%203%E2%80%99%20UTRs%20and%20suggest%20a%20novel%20mechanism%20of%20translation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/325316v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/325316v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/325316v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/molecular-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Molecular Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
