<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">G3 (Bethesda)</span><span class="iso-abbrev" title="iso-abbrev">Genetics</span><span class="hwp" title="hwp">G3: Genes, Genomes, Genetics</span><span class="pmc" title="pmc">G3: Genes, Genomes, Genetics</span><span class="publisher-id" title="publisher-id">G3: Genes, Genomes, Genetics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">G3: Genes|Genomes|Genetics</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2160-1836</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Genetics Society of America</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4704709">4704709</a></span><span class="publisher-id" title="publisher-id">GGG_024208</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1534/g3.115.024208">10.1534/g3.115.024208</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Investigations</span></div> <div class="title-group"> <div class="article-title" title="article-title">
The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcala</span><span class="given-names" tagx="given-names" title="given-names">Nicolas</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5961-5064</span><a href="#aff1">*</a><a href="#aff2"><sup>†</sup></a><a href="#cor1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey D.</span></span><a href="#aff3"><sup>‡</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Telenti</span><span class="given-names" tagx="given-names" title="given-names">Amalio</span></span><a href="#aff4"><sup>§</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuilleumier</span><span class="given-names" tagx="given-names" title="given-names">Séverine</span></span><a href="#aff1">*</a><a href="#aff3"><sup>‡</sup></a><a href="#aff5">**</a></span><span class="citation_author_institution" id="aff1">[*], </span><span class="citation_author_institution" id="aff2">[†], </span><span class="citation_author_institution" id="aff3">[‡], </span><span class="citation_author_institution" id="aff4">[§], </span><span class="citation_author_institution" id="aff5">[**], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">1</span>Corresponding author: Department of Biology, Stanford University, California 94305-5020. E-mail: <span class="email" tagx="email" title="email">nalcala@stanford.edu</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2015-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2016-1</span></span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">120</span><span class="history" title="history"><span class="received" title="received">received: 2015-8-01</span><span class="accepted" title="accepted">accepted: 2015-11-02</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="107.pdf" title="self-uri">107.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
admixture</div> <div class="kwd" title="kwd">
migration</div> <div class="kwd" title="kwd">
coalescent</div> <div class="kwd" title="kwd">
site frequency spectrum</div> <div class="kwd" title="kwd">
HIV</div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <p>Past population demographic events leave distinctive signatures in DNA sequence polymorphisms and gene genealogies (<a href="#bib95">Tajima 1989</a>; <a href="#bib86">Simonsen <i>et al.</i> 1995</a>; <a href="#bib80">Schneider and Excoffier 1999</a>; <a href="#bib24">Excoffier 2004</a>; <a href="#bib108">Zeng <i>et al.</i> 2006</a>; <a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>; <a href="#bib62">Naduvilezhath <i>et al.</i> 2011</a>). For example, a population bottleneck increases the variance of coalescence times and reduces the number of variants at low frequency in the site frequency spectrum (SFS) (<a href="#bib95">Tajima 1989</a>). Population expansion creates star-shaped genealogies and increases the mean number of variants at low frequency in the SFS (<a href="#bib95">Tajima 1989</a>; <a href="#bib87">Slatkin 1996</a>). Population sub-division leads to long internal branches in the genealogy and increases the number of fixed variants (<a href="#bib43">Harpending <i>et al.</i> 1998</a>). It also results in "structured�? genealogies in which the means and variances of coalescence times (<i>e.g.</i>, time to the most recent common ancestor) are large, resulting in an excess of intermediate frequency variants in the SFS (<a href="#bib102">Wakeley 1999</a>). Balancing selection leads to a similar structure, and to an excess of variants at intermediate frequency (<a href="#bib95">Tajima 1989</a>; <a href="#bib27">Fay and Wu 2000</a>; <a href="#bib11">Barton and Etheridge 2004</a>; <a href="#bib108">Zeng <i>et al.</i> 2006</a>). Directional selection leads to genealogies with both star shapes (leading to the common ancestor of the selected lineages) and long branches (due to remaining ancestral lineages and/or recombination with the selected haplotype), and generates an excess of variants at both low and high frequency (<a href="#bib10">Barton 1998</a>).</p> <p>Consequently, DNA sequence polymorphisms are increasingly used to infer past demographic events and have successfully reconstructed past histories of many organisms, including humans (<a href="#bib38">Gravel <i>et al.</i> 2011</a>; <a href="#bib71">Reich <i>et al.</i> 2012</a>). Some of the first methods proposed were developed to infer population growth rates (<a href="#bib88">Slatkin and Hudson 1991</a>) or for specific demographic scenarios (<i>e.g.</i>, <a href="#bib65">Nielsen and Wakeley 2001</a>; <a href="#bib44">Hey 2010</a>). Recently developed approaches allow for the inference of a wide range of demographic parameters (<i>e.g.</i>, <a href="#bib93">Strimmer and Pybus 2001</a>; <a href="#bib13">Beaumont <i>et al.</i> 2002</a>; <a href="#bib22">Drummond <i>et al.</i> 2005</a>; <a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>; <a href="#bib26">Excoffier <i>et al.</i> 2013</a>). However, inferring reconnection of diverged populations after isolation remains difficult, as models of past isolation with recent contact are challenging to distinguish from models of ancient population separation with a continuous exchange of migrants. Also, parameter inference requires <i>a priori</i> knowledge of a population split (<a href="#bib44">Hey 2010</a>), and the time at which migration event(s) occurred has thus far been difficult to estimate (<a href="#bib91">Sousa <i>et al.</i> 2011</a>; <a href="#bib92">Strasburg and Rieseberg 2011</a>). <a href="#bib65">Nielsen and Wakeley (2001)</a> first proposed a method to infer time of divergence (IM model), the migration rate between the two populations, as well as the relative sizes of the populations. This approach was extended to account for the divergence of multiple populations (<a href="#bib44">Hey 2010</a>) and temporal reduction of gene flow following an initial population split (<a href="#bib105">Wilkinson-Herbots 2012</a>). Though others have discussed the expected signature of a reconnection event following a past isolation period (<a href="#bib14">Becquet and Przeworski 2009</a>; <a href="#bib90">Sousa and Hey 2013</a>), detailed theoretical formalization is still lacking. Besides this, the use of the aforementioned methods for general inference is also challenging as they are difficult to apply when the number of sampled populations is large (<a href="#bib13">Beaumont <i>et al.</i> 2002</a>; <a href="#bib22">Drummond <i>et al.</i> 2005</a>; <a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>; <a href="#bib57">Lohse <i>et al.</i> 2011</a>; <a href="#bib26">Excoffier <i>et al.</i> 2013</a>), and when multiple sampled time-points are available (<a href="#bib26">Excoffier <i>et al.</i> 2013</a>; <a href="#bib34">Foll <i>et al.</i> 2014</a>). Further, they often rely on a specific evolutionary model, such as the Wright-Fisher model (<i>e.g.</i>, <a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>; <a href="#bib57">Lohse <i>et al.</i> 2011</a>; <a href="#bib26">Excoffier <i>et al.</i> 2013</a>) which might not be representative of, for example, the skewed offspring distributions found for marine species or viruses (<a href="#bib99">Tellier and Lemaire 2014</a>)</p> <p>Past population isolation and subsequent reconnection or secondary contact are common in natural populations and can have a drastic impact on species’ evolutionary histories. Anatomically, modern humans are thought to have admixed with Neanderthal populations after a period of isolation between the African and European continents (<a href="#bib39">Green <i>et al.</i> 2010</a>; <a href="#bib68">Patterson <i>et al.</i> 2012</a>; <a href="#bib78">Sankararaman <i>et al.</i> 2012</a>). Similarly, African and non-African populations of <i>Drosophila melanogaster</i> also experienced a period of isolation, followed by subsequent admixture (<a href="#bib70">Pool <i>et al.</i> 2012</a>). Domesticated species of both plants and animals are isolated from their wild relatives, and often experience subsequent genetic exchange (<i>e.g.</i>, in crops; <a href="#bib23">Ellstrand <i>et al.</i> 1999</a>). Importantly, isolation and subsequent reconnection events are also common features in viral histories, including the Influenza A virus, Human Immunodeficiency Virus (HIV) and Human Cytomegalovirus (HCMV; <a href="#bib73">Renzette <i>et al.</i> 2013</a>), where dependency on the host cell and other features of the life cycle isolate viral populations within hosts, groups of hosts, or host species in-between infections. For example, reassortment between avian and human influenza viruses caused the pandemic outbreaks in 1957 and 1968, and a reassortment of swine, avian and human influenza viruses was responsible for the 2009 pandemic (<a href="#bib46">Hsieh <i>et al.</i> 2006</a>; <a href="#bib36">Garten <i>et al.</i> 2009</a>; <a href="#bib33">Flahault and Zylberman 2010</a>).</p> <p>As in influenza, events of HIV isolation and reconnection are common and can be observed at different spatial and temporal scales. Indeed, tracing the origin of HIV has confirmed that cross-species transmission of SIVs from other primates to humans, acting as sources of HIV, occurred several times over the past 100 years (<a href="#bib56">Liégeois <i>et al.</i> 2014</a>). The virus remained isolated for a very long period and adapted rapidly in humans, which ultimately rose to epidemic levels (<a href="#bib53">Korber <i>et al.</i> 2001</a>; <a href="#bib98">Tebit and Arts 2011</a>). Circulating HIV populations in humans are strongly divergent. Two major types of HIV exist (HIV-1 and HIV-2) and both have diversified into several groups or subtypes (<a href="#bib76">Robertson <i>et al.</i> 2000</a>). The HIV pandemic is mainly caused by HIV-1 Group M, which is composed by divergent (up to 35% sequence divergence) subtypes (named A–D, F–H, J, and K, <a href="#bib37">Gaschen <i>et al.</i> 2002</a>). The HIV epidemic is associated with the worldwide movements of people that allowed HIV-1 subtype colonization events (<a href="#bib98">Tebit and Arts 2011</a>; <a href="#bib41">Hamelaar <i>et al.</i> 2011</a>). In the course of their evolution in human populations, HIV-1 subtypes experienced several large-scale isolation events within both transmission risk groups (<a href="#bib94">Su <i>et al.</i> 2000</a>; <a href="#bib67">Pang <i>et al.</i> 2012</a>; <a href="#bib42">Han <i>et al.</i> 2013</a>; <a href="#bib55">Li <i>et al.</i> 2013</a>) and countries (<a href="#bib15">Bello <i>et al.</i> 2011</a>; <a href="#bib20">Castro-Nallar <i>et al.</i> 2012</a>). Subsequent reconnection then occurred through coinfection, superinfection, and recombination (<a href="#bib98">Tebit and Arts 2011</a>; <a href="#bib100">Vuilleumier and Bonhoeffer 2015</a>). Those reconnection events are the source of emergence of many new recombinant forms which constitute a major challenge for vaccine development and are at the origin of new epidemics (<a href="#bib98">Tebit and Arts 2011</a>; <a href="#bib29">Feng <i>et al.</i> 2013</a>). The multiple colonization events of HIV-1 that occurred in Asia and in South America are representative of this trend (<a href="#bib41">Hamelaar <i>et al.</i> 2011</a>). Indeed, with an estimated 4.8 million people living with HIV as of 2011, Asia is the second most HIV-affected region worldwide (<a href="#bib109">Zhang <i>et al.</i> 2013</a>). Historical epidemics in Asia are associated with transmission events to various risk groups; HIV-1 subtypes B and C were first detected in the mid-1980s and early 1990s, respectively, in the Injecting Drug Users (IDUs) risk group (<a href="#bib58">Lu <i>et al.</i> 2008</a>), and then spread to other populations. Subtype CRF01_AE was first detected in the Men who have Sex with Men (MSM) risk group in the 1990s, then spread to other groups and became the most prevalent HIV-1 subtype in many parts of China. Similarly, the epidemic clade circulating in South America is derived from subtype B viruses that migrated out of Haiti around 1969 (1966–1972) and spread through the world (<a href="#bib15">Bello <i>et al.</i> 2011</a>). Later, a single-founder subtype F1 strain, introduced in Brazil, spread and recombined with local subtype B viruses to form the HIV-1 BF1 and F1 epidemics (<a href="#bib7">Aulicino <i>et al.</i> 2007</a>).</p> <p>The present study identifies specific temporal signatures of reconnection of diverged populations in two broadly used summaries of DNA polymorphism: the distribution of pairwise nucleotide differences (via coalescent theory), and the Site Frequency Spectrum (SFS, via simulation). We also describe how past isolation can influence commonly used SFS-based test statistics (Tajima’s <i>D</i>, Fu and Li’s <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mtext_UNKNOWN">
*</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mtext_UNKNOWN">
*</div> </div> </div></div></span>, Fay and Wu’s <i>H</i>, Zeng <i>et al.</i>’s <i>E</i>), and discuss the specificity and the robustness of the signature relative to other demographic and selective processes. Then, using theoretical results and full genome polymorphism data, we reconstruct the successive invasions of HIV-1 subtypes in China and in South America and analyze the subsequent dynamics of genome composition. We find extensive admixture between HIV-1 subtypes, potentially leading in the long term to homogenization of HIV-1 genomes in these regions.</p> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <p>Using coalescent theory, we first present methods for detecting the signature of population reconnection after isolation on the distribution of pairwise nucleotide differences. Then, in the second section, we use coalescent simulations to derive the signature of population reconnection after isolation on the SFS. The third section highlights the robustness of these signatures to alternative demographic scenarios. Finally, the fourth section presents a data application for these methods: inference of recent HIV-1 evolution in China and in South America.</p> <div class="theoreticalsignature" title="sec"> <div class="title" tagx="title" title="title">
Theoretical signature</div> <p>In our model, we consider <i>d</i> previously connected populations that became isolated at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> and then reconnected at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>. The scaled migration rate <i>M</i> between populations (<i>i.e.</i>, twice the number of migrant genes per population per generation) is the same before and after the isolation period and mutations follow a Poisson process of rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> (<a href="#bib51">Kimura 1969</a>). We used an infinite sites model for the analytical investigations, and for the simulations we considered a finite sites model with a number of sites corresponding to HIV-1 genome size (9719bp, from the reference HXB1 sequence). Our developments are based on the Wright-Fisher model (<a href="#bib32">Fisher 1930</a>; <a href="#bib106">Wright 1931</a>), where each population has a constant size <i>N</i>. For analytical simplicity, we considered that times <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> are scaled in units of <i>N</i> generations, so <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> corresponds to an isolation event that occurred <i>N</i> generations ago. We derived an exact theoretical formula for the distribution of pairwise nucleotide differences in nonrecombining genomic regions (<i>Appendix A</i>), and simulated this distribution using coalescent simulations (software <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
s</div> </div></div></span>; <a href="#bib47">Hudson 2002</a>) in the case of recombining regions. We computed the power to sample sequences from each mode of the distribution using equations A.6a–A.6b (Supporting Information, Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> <p>We generated the SFS through time following isolation and reconnection events using coalescent simulations. We studied the <i>total SFS</i> (considering pooled samples from all populations); in the main text, we present results for equal sample sizes in each populations, and we present results for alternative sampling schemes (Figures B and C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We also studied the joint SFS between pairs of populations. We assessed the temporal changes of the total SFS using the visual test of <a href="#bib63">Nawa and Tajima (2008</a>), which is based on the difference between the expected SFS under neutrality in a population with constant size, and the observed SFS. For each frequency <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
n</div> </div> </div></div></span>, where <i>n</i> is the sample size, the transformation represents <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
i</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ξ</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>, instead of the corresponding number of variants <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ξ</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>. This simple test enables one to easily detect which frequencies present an excess or a deficit of variants. We also derived an optimal test statistic to detect the signature of population reconnection from the SFS (Text A and Figure D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and compute the power of common neutrality tests to detect such events (Figure E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> </div> <div class="robustnessanalysisofthetheoreticalsignature" title="sec"> <div class="title" tagx="title" title="title">
Robustness analysis of the theoretical signature</div> <p>We then analyzed the sensitivity of our results to departures from the infinite sites model (Figure F in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), from the constant population size model (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and from the Wright-Fisher model (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). To account for repeated bottlenecks (<i>e.g.</i>, during transmission events), extinction-recolonization dynamics, and rapid adaptation, we simulated the distribution of pairwise nucleotide differences under a scenario of reconnection of isolated populations under a beta-coalescent model. Beta-coalescent models have recently been proposed to model the genealogies of many organisms with a large variance in the number of offspring per individual (<a href="#bib17">Birkner and Blath 2008</a>; <a href="#bib99">Tellier and Lemaire 2014</a>), in particular viruses (<a href="#bib64">Neher and Hallatschek 2013</a>). We extended the algorithm from Birkner and Blath (<a href="#bib17">Birkner and Blath 2008</a>) – which generates samples under a beta-coalescent at equilibrium – to include an island model of migration (each lineage migrates to another population at rate <i>M</i>) and an isolation period. We assumed that coalescence processes follow the beta-coalescent at all times: before isolation, during isolation and after reconnection. In addition to mutation and migration rates, beta-coalescents rely on a parameter α (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
2</div> </div></div></span>) that gives the probability of multiple lineages coalescing at the same time. We present the results for four different parameter values: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> (equivalent to the classic Kingman coalescent), <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.5</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> [corresponding to the Bolthausen-Sznitman coalescent, used to model strong positive selection (<i>e.g.</i>, <a href="#bib64">Neher and Hallatschek 2013</a>)], and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.5</div> </div></div></span>.</p> <p>We also compared the signature of population reconnection after isolation with that of a closely related model: the Isolation with Migration model [IM (<i>e.g.</i>, <a href="#bib96">Takahata 1995</a>; <a href="#bib101">Wakeley 1996</a>; <a href="#bib77">Rosenberg and Feldman 2002</a>)], using coalescent simulations (Figure I in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> </div> <div class="hivdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
HIV data analysis</div> <p>We used 1646 whole genome HIV-1 samples from subtypes B, C, F1 and CRF01_AE and recombinant forms between these subtypes (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We used the subset of genomes from China sequenced between 2005 and 2009 (297 sequences; Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). For the study in South America (Brazil and Argentina), we used data available between 1999 and 2005 (128 sequences; Table C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Finally, for the worldwide PCA analysis, we used all 1646 genomes available.</p> <p>For the Chinese and South American data, we identified HIV-1 populations (subtype clusters) by Discriminant Analysis of Principal Components (DAPC) (<a href="#bib48">Jombart <i>et al.</i> 2010</a>). Our investigations considered dominant subtypes B, C, CRF01_AE and F1 and their associated recombinants. Each cluster encompassed a dominant subtype and some recombinant forms between subtypes. We found strong support for four groups using the Bayesian Information Criterion (ranging from 2 to 100 PCA axes) that are independent of sampling year (See Figure J in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for a neighbor joining tree of the sequences where the different clusters and sampling years are represented). We then computed the distribution of pairwise nucleotide differences and the total and joint SFS for the major circulating HIV-1 subtypes in China and South America.</p> </div> <div class="computingthetemporalpatternsofgeneticvariationofhivsubtypes" title="sec"> <div class="title" tagx="title" title="title">
Computing the temporal patterns of genetic variation of HIV subtypes</div> <p>We computed the distribution of pairwise nucleotide differences, and the total and joint SFS for the major circulating HIV-1 subtypes in China and South America. The distribution of pairwise nucleotide differences within (within-population pairwise differences) and between clusters (between-population pairwise differences) were generated for each nonrecombinant block between subtypes that we identified with <i>jpHMM</i> (<a href="#bib82">Schultz <i>et al.</i> 2009</a>). In total, 19 and 24 blocks were identified in sequences sampled in China and South America, respectively; the 5%, median and 95% quantiles of block length were 111bp, 220bp and 1333bp, respectively. Detection of bimodality in the distribution, the signature of a past isolation event, was performed by fitting a Gaussian Mixture Model using the Expectation-Maximization algorithm from <a href="#bib60">Meng and Rubin (1993</a>) as implemented in the R package <i>mixtools</i> (<a href="#bib16">Benaglia <i>et al.</i> 2009</a>). Gaussian distributions are used to estimate positions of the mode as distributions are symmetric (the 5% and 95% quantiles of the skewness of all components are -1.05 and 1.05, respectively). The algorithm also provides posterior probabilities of membership into modes, which is used to identify genomes that have bimodal distribution (<i>i.e.</i>, indicating admixture). We analyzed the signatures in the pairwise nucleotide differences by comparing HIV genome signatures with the expected results, assuming either small linked sequences or large recombining sequences.</p> <p>The total and joint SFS were computed using subtype J as an outgroup. Results were robust to the chosen outgroup (Figure K in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Note that the distinction between recombining and nonrecombining sequences does not apply for the SFS, as the expectation (and the maximum likelihood estimate) of the SFS is not affected by recombination: only its variance is affected (<a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>). We tested the significance of the temporal changes of the SFS using a bootstrap test (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data used are from the Los Alamos HIV database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>) (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="bimodalityinpairwisenucleotidedifferencesasasignalofpopulationreconnectionafterisolation" title="sec"> <div class="title" tagx="title" title="title">
Bimodality in pairwise nucleotide differences as a signal of population reconnection after isolation</div> <p>Two distinct signatures of population reconnection after isolation were found in the temporal distribution of pairwise nucleotide differences: bimodality and increased variance of the first mode (<a href="#fig1">Figure 1</a>). Bimodality was observed in these distributions (both within and between populations, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>) only when a population reconnection occurred (<a href="#fig1">Figure 1A</a>). This bimodality reflects the different possible origins of the two sequences sampled: the two sequences either have a recent common ancestor (<i>i.e.</i>, after the reconnection event; first mode in <a href="#fig1">Figure 1A</a>) or an ancient common ancestor (before the isolation event; second mode in <a href="#fig1">Figure 1A</a>). Bimodality was also observed in the distributions of total pairwise nucleotide difference (considering all populations, π) after a reconnection event but, in this case, bimodality was also observed when populations were completely isolated (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>, <a href="#fig1">Figure 1, B and C</a>). To differentiate a connection from an isolation event, bimodality needs to be associated with a temporal increase of the variance of the modes in the distribution (which depends on the recombination rate; see <a href="#fig1">Figure 1</a>). Although testing for increase in variance requires data from different time-points, it has the advantage of not requiring <i>a priori</i> knowledge of the origin of the samples (<i>i.e.</i>, being within or between populations).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Signature of reconnection of previously isolated populations on the distribution of pairwise nucleotide differences as a function of time since the reconnection event <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> (columns), (A) within-<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> and between-population <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>, (B–C) total π pairwise nucleotide differences. In panel A, we consider a sequence of size <i>G</i>, with <i>G</i> larger than the number of segregating sites without recombination; plots in panel A are computed using equation A.5. In panels B and C, we consider a sequence of 9719 bp (length of the reference sequence for HIV-1, HXB1) with various mean numbers of recombination events per generation, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
10</div> </div></div></span>. Nucleotide differences are considered from a sample of 16 sequences and each plot represents the mean distribution over 2000 replicate simulations. Parameters are <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
3</div> </div></div></span> populations, duration of the isolation period <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
6</div> </div></div></span>, scaled migration rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
5</div> </div></div></span>, scaled mutation rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
100</div> </div></div></span>.</p>   </div> <p>Population reconnection after isolation can thus be detected by testing for bimodality and a temporal increase of the variance of the modes. Power to detect bimodality, with high probability (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
95</div> <div class="mtext_UNKNOWN">
%</div> </div></div></span>) (Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), depends on the sample size, the duration of the reconnection period, and the amount of gene flow among populations (scaled migration rate). As shown in Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>, it is easier to detect a reconnection event from the distribution of pairwise nucleotide differences between populations than within populations. Indeed, the second mode in the distribution of pairwise nucleotide differences was larger between population (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>) than that of within population (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span>, <a href="#fig1">Figure 1A</a>). This difference in mode size increases when migration is weak or intermediate (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
≤</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>) and the reconnection event is not recent (Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Ancient reconnection events can be detected even with small sample sizes (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
≃</div> <div class="mn_UNKNOWN">
10</div> </div></div></span>) when migration is moderate or strong (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>; Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). To evaluate the temporal increases in variance of the mode, no further development is required as the Bartlett test (<a href="#bib9">Bartlett 1937</a>) can be used.</p> <p>Thus, we show that distinctive signatures of past isolation and reconnection events can be identified on the distribution of the number of nucleotide differences between pairs of genomes sampled (within, between, or in the total population). We also determine the parameter ranges for which this signature can be detected (sample size, migration rate, and time since reconnection).</p> </div> <div class="thesignatureofpopulationreconnectionafterisolationonthesitefrequencyspectrum(sfs)" title="sec"> <div class="title" tagx="title" title="title">
The signature of population reconnection after isolation on the Site Frequency Spectrum (SFS)</div> <p>When a population is at mutation-drift equilibrium, the distribution of its alleles, or SFS, has a characteristic shape (dotted line <a href="#fig2">Figure 2A</a>). When populations experience demographic changes, departures from this characteristic shape are expected and the size and the position of those changes are informative as to the demographic change. Consistent with the expectation, we found a large excess of variants at intermediate frequencies on the SFS following reconnection events between previously isolated populations (<a href="#fig2">Figure 2</a>). This excess was due to exchanges of variants among populations that were once fixed in one or several populations during isolation. The size and the position of this excess is indicative of the number of populations that have reconnected and of the timing of the reconnection event (<a href="#fig2">Figure 2A</a>), and can be identified by the visual test of equilibrium neutrality proposed by <a href="#bib63">Nawa and Tajima (2008</a>). We predict that in the total SFS (SFS obtained with merged samples from all populations), shortly after a reconnection event one would expect narrow, high peaks of specific variants. These peaks are found at frequencies <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
d</div> </div> </div></div></span>, where <i>d</i> is the number of sampled populations and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
d</div> </div></div></span>, when populations have equal sample sizes. In <a href="#fig2">Figure 2A</a>, peaks can be seen at frequencies <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
d</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
d</div> </div> </div></div></span> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
3</div> </div></div></span>). With unbalanced sample sizes, the number of observed peaks can be higher (up to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> peaks; see Figure B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Signature of a reconnection event between previously isolated populations on (A) the total and (B) joint site frequency spectrum (SFS), as a function of the number of generations since the reconnection event, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>. (A) The SFS representation of <a href="#bib63">Nawa and Tajima (2008</a>) is used; the expected value of a single panmictic population at equilibrium is given by a straight line (dashed line), and deviations from a straight line indicate an excess (in blue) or deficit (in red) of variants at a given frequency. Parameters are <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
3</div> </div></div></span> populations, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
16</div> </div></div></span> sampled genes per population (see Figures B and C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for alternative sampling schemes), with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
100</div> </div></div></span>. Means are over 2000 replicates.</p>   </div> <p>The temporal patterns in the joint SFS (<a href="#fig2">Figure 2B</a>) are also strongly informative as to the occurrence of reconnection events and as to the amount of gene flow among populations since the reconnection event. When populations are isolated, each has its own segregating variants. This translates, in the joint SFS, into variants being restricted to the axes of the joint SFS [classes (20,<i>i</i>), (<i>i</i>,20), with <i>i</i> ranging from 1 to 20 in <a href="#fig2">Figure 2B</a>]. Then, following the reconnection event, gene flow distributes the variants among populations to the point that they occur at similar frequencies in each population and the genetic compositions of populations are fully homogenized. Consequently, in the joint SFS, several variants move progressively away from the axes toward the diagonal of the joint SFS, as expected in a structured population at equilibrium (last column <a href="#fig2">Figure 2B</a>).</p> <p>To robustly detect the excess of variants on the SFS generated by a reconnection event, we derived an optimal test statistic, denoted <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Ω</div> </div> </div></div></span>, using developments provided by <a href="#bib31">Ferretti <i>et al.</i> (2010)</a> and <a href="#bib2">Achaz (2009)</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Although use of this test is restricted to recent reconnection events that follow long periods of isolation, it performs better than commonly used statistics such as Tajima’s <i>D</i> (<a href="#bib95">Tajima 1989</a>), Fu and Li’s <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mtext_UNKNOWN">
*</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mtext_UNKNOWN">
*</div> </div> </div></div></span> (<a href="#bib35">Fu and Li 1993</a>), Fay and Wu’s <i>H</i> (<a href="#bib27">Fay and Wu 2000</a>), Zeng <i>et al.</i>’s <i>E</i> (<a href="#bib108">Zeng <i>et al.</i> 2006</a>). Indeed, even with long isolation periods, those statistics have less than 60% power to detect reconnection events (Figure E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). The results presented in this section apply to both recombining and nonrecombining sequences, as the expected SFS is the same in both cases (<a href="#bib40">Gutenkunst <i>et al.</i> 2009</a>). Also, the power estimates from the tests based on nonrecombining sequences are conservative as they correspond to the case with the highest variance (<a href="#bib2">Achaz 2009</a>) and thus the lowest power.</p> <p>Thus, we show that distinctive signatures of past isolation and reconnection events can be identified in the SFS as well as in the joint SFS. Namely, the SFS has an excess of variants, the size and position of which are indicative of the number of previously isolated populations and of the timing since the reconnection event. The temporal dynamic of the joint SFS is informative as to the level of gene flow between populations.</p> </div> <div class="robustnessofidentifiedsignaturestoalternativedemographicscenarios" title="sec"> <div class="title" tagx="title" title="title">
Robustness of identified signatures to alternative demographic scenarios</div> <p>The signatures of population reconnection identified here, <i>i.e.</i>, bimodality and temporal increase of variance of peaks in the SFS, are robust to population expansion (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), alternative evolutionary and population dynamic models such as repeated bottlenecks, rapid extinction-recolonization dynamics, recurrent positive selection or highly skewed offspring distributions (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Moreover, the signature is distinct from the signature of a closely related model: the Isolation with Migration model (IM) (Figure I in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> <p>The observed signatures are robust even when a massive population expansion occurs concomitantly with the reconnection event (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Such events modify the expected signature only by presenting an excess of low frequency variants compared to what is expected with populations of constant size (as expected from the results of Tajima (<a href="#bib95">Tajima 1989</a>) in a single panmictic population). Replacing the classic Kingman coalescent model (Wright-Fisher model) with a beta-coalescent model does not alter our results for values of the model parameter α (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Beta-coalescent models account for large variance in the number of offspring per individual, as expected under repeated bottlenecks, fast extinction-recolonization dynamics, or recurrent beneficial fixations (<a href="#bib64">Neher and Hallatschek 2013</a>; <a href="#bib99">Tellier and Lemaire 2014</a>). Such variance leads to an excess of low frequency variants and smaller peaks in the total SFS compared to Kingman’s coalescent model; this effect is larger for small α values (light-gray lines Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). However, having an excess of variants at low frequency does not alter the identified signature of reconnection on the total SFS (dynamics of the peaks). Also the temporal dynamics of the signature distinguish this pattern from that of an IM model. Indeed under the IM model, the size of the peaks in the SFS increases with time (Figure I.B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), whereas the signature of a reconnection of isolated populations is characterized by a decrease in peak size with time (<a href="#fig2">Figure 2A</a>).</p> <p>Thus, we show that, even though the signature at specific time-points may be confounded with that of other selective or demographic processes, the temporal signature of past isolation and reconnection identified is unlikely to be confounded, and provides a robust signal.</p> </div> <div class="detectionofpastisolationandcurrentreconnectionofhiv-1subtypes" title="sec"> <div class="title" tagx="title" title="title">
Detection of past isolation and current reconnection of HIV-1 subtypes</div> <p>We applied the conceptual and theoretical approaches via an analysis of the colonization history of different HIV-1 subtypes in Asia and in South America. Using data from the Los Alamos HIV database, we detected a strong signal of reconnection events between previously isolated HIV-1 subtypes in these populations.</p> <p>Past isolation between HIV-1 subtypes in China and South America is signaled by the excess of variants, with peaks at intermediate frequencies in the total SFS (<a href="#fig3">Figure 3, A and B and G and H</a>). These peaks reflect the presence of variants from the sampled population that diverged during isolation and were present at high frequencies in each previously isolated subtype population identified with cluster analysis (see <i>Results</i>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Signatures of HIV-1 subtype reconnection events on the Site Frequency Spectrum (SFS). Results are presented for China (left major panel) and for South America (right major panel). Two time points of HIV-1 sequences sampled are considered in China (Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and in South America (Table C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) (see method section ‘HIV-1 genome analysis’). The total (pooled) SFS is presented in the first row following representation of <a href="#bib63">Nawa and Tajima (2008</a>), the joint SFSs of each pairwise group of HIV subtypes are presented in the second, third, and fourth row and, in the fifth row, the Principal Component Analyses (PCAs) are presented. Subtype J is used as an outgroup (see Figure K in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>, results for alternative outgroups). Original subtypes (subtypes found early in the epidemics in Africa; at the corners) are used to define a triangle in the PCA space. The position of the HIV-1 genomes on the straight edges of the triangle suggests strong admixture between subtypes. For the total and joint SFS, we consider the mean value of 500 random samples of sequences in each subtypes cluster (see Figure J in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for the different clusters). A sample of 16 sequences per subtype cluster is considered in China, and of eight sequences per subtype cluster in South America. The significance of the trends in the dynamics is tested using the optimal test statistic <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Ω</div> </div> </div></div></span> derived in the SI (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p>   </div> <p>The reconnection of HIV-1 subtype populations in China and South America is indicated by the bimodality in the distribution of pairwise nucleotide differences between-populations (considering sequences of small length <a href="#fig4">Figure 4</a>). We found that a large proportion of the genome of HIV-1 displays bimodality (<a href="#fig4">Figure 4</a>). In China, this proportion was initially low but rapidly increased through time. Between 2006 and 2007, the proportion of bimodality increased fourfold (<a href="#fig4">Figure 4</a>), and increased a further 40% between 2008 and 2009. This suggests that the time series analyzed here captures a burst of HIV-1 subtype admixture. The dynamics of genomic admixture between HIV-1 subtypes in China is also highlighted by the temporal changes in the distribution of pairwise nucleotide differences and the SFS. The peaks formed by the excess variants (resulting from population admixture in the total SFS) decreased in size, and the values of optimal test statistics <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Ω</div> </div> </div></div></span> significantly decreased (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We find a significant increase in the variance of the first mode in the distribution of pairwise nucleotide differences between 2005 and 2009 (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.01</div> </div></div></span> two-sided Bartlett test). The joint SFSs for the two time points also indicate that the reconnection event and genome admixture is either recent or that genome admixture is limited but ongoing. Indeed, we can observe the excess of variants along the axes of the joint SFS shifting to the diagonal (<a href="#fig3">Figure 3, C and D</a>). In South America, the proportion of bimodality in HIV-1 samples from 1999 is large (60%) and remains approximately constant from 1999 to 2004, signaling a prior reconnection event. Patterns observed in the joint SFS suggest that HIV-1 subtype genomes are well homogenized overall; further, this process of homogenization seems to have either slowed or ceased. Indeed, no increase of variance in the modes of the distribution of pairwise nucleotide differences was detected (two-sided Bartlett test), and the total SFS shows only small changes between 2001 and 2005, with nonsignificant changes in values of optimal test statistics <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
Ω</div> </div> </div></div></span> (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>(A–B) Proportion of bimodality in pairwise nucleotide differences (between populations) detected in the genomes and (C–D) distribution of total pairwise nucleotide differences (all populations) of HIV-1 sequences for different time points in China (see Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and South America (see Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). As expected under genomic admixture (<a href="#fig1">Figure 1</a>), the proportion of bimodality detected in the pairwise nucleotide differences between populations in HIV-1 genomes increased, and the variance of modes significantly increased with time in China (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.01</div> </div></div></span>, two-sided Bartlett test). We also see a (nonsignificant) trend of temporal changes in South America.</p>   </div> <p>Genome admixture in China and in South America is also visible when the sampled genomes are projected together in a PCA space defined by original subtypes <i>i.e.</i>, which originate from the beginning of the HIV epidemic in Africa (<a href="#fig3">Figure 3, E and F and K and L</a>). Several HIV-1 genomes are present on the straight edges of the triangles defined by the ones at the origin. Temporal PCA patterns also suggest homogenization between subtypes B and C in China and between B and F1 in South America, as they move away from the pure subtypes in PCA space.</p> <p>To provide insights on the generality and global importance of these results, we performed a PCA analysis of worldwide genomes from HIV-1 subtypes B, C, F1, and CRF01_AE (<a href="http://www.hiv.lanl.gov">http://www.hiv.lanl.gov</a>). Again we projected them in the PCA with original subtypes B, C, F1, and CRF01_AE as references (<a href="#fig5">Figure 5</a>). Consistent with the observations for Chinese and South American samples, we observed that HIV-1 genomes are positioned on the straight edges of the triangle formed by original subtypes, positioned at the corners, forming a continuum along the axes rather than discrete clusters within. Very few genomes are localized outside the edges of the triangle defined by the 3 original subtypes. This pattern not only signals that most of the circulating HIV-1 genomes experience constant gene flow from other subtypes but also represents the current spatial distribution of HIV-1 populations. It reflects, for example, the extensive admixture between subtypes B and CRF01_AE in Brazil and China, between B and CRF01_AE in Thailand, and B and F1 in Brazil, Argentina, and Spain.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Projection of worldwide distributed HIV-1 sequences from subtypes B, C, CRF01_AE and F1, and URF and CRF recombinant forms between these subtypes. 1646 genome sequences (see Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). (A) The first two axes of the principal component analysis (PC1 and PC2), and (B) the second and third axes of the principal component analysis (PC2 and PC3) defined by pure subtypes B, C, CRF01_AE and F1 (see method section ‘HIV-1 genome analysis’). The percentage of variance explained by each PC is indicated between parentheses. The position of a HIV-1 genome sequence along the axes of the triangle formed by three pure subtypes reflect admixture (proportion) between the subtypes (<a href="#bib59">Ma and Amos 2012</a>). Standard country codes are used: Argentina (AR), Brazil (BR), Botswana (BW), China (CN), Cyprus (CY), Spain (ES), Japan (JP), Thailand (TH), USA (US), Vietnam (VN), South Africa (ZA). Misrepresented sequences [squared cosine of the PC plane with the observation <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.05</div> </div></div></span>(<a href="#bib1">Abdi and Williams 2010</a>)] were excluded.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We present an analytical framework to understand the evolutionary histories of organisms that are characterized by periods of isolation. Our theoretical investigations identify specific genomic signatures of past population isolation. We show, for example, that the number, size, and dynamics of modes in the distributions of the pairwise nucleotide differences in populations are informative on the existence of past isolation. Our results extend previous work on the distribution of pairwise coalescence times and pairwise nucleotide differences under a model of population split with or without migration (IM model). We show, for example, that under the reconnection model, the distribution of pairwise nucleotide differences has a temporal evolution which is distinct from the one expected under an IM model (<a href="#bib96">Takahata 1995</a>; <a href="#bib101">Wakeley 1996</a>; <a href="#bib77">Rosenberg and Feldman 2002</a>): the distribution is first bimodal and then tends to become unimodal with time. We also found that in the Site Frequency Spectrum (SFS), the presence, the position and the value variants in excess are informative as to the numbers of previously isolated populations, and the joint SFS characterizes the amount of genetic overlap between populations. Signatures identified here are robust to the presence of recombination, extinction-colonization processes, population bottlenecks, and expansion as well as skewed offspring distributions. Our developments add to the current literature in the field as we demonstrate that population reconnection after isolation is not fully captured by commonly used test statistics based on SFS classes of variants at high and/or low frequencies such as <i>D</i>, <i>H</i>, and <i>E</i> (<a href="#bib31">Ferretti <i>et al.</i> 2010</a>). Taken together, this provides critical information on the evolution of the genomic composition of populations.</p> <p>We apply our theory in order to analyze the genome signature of successive colonization of HIV-1 subtypes in Asia and South America. In both regions, we reconstruct the successive colonization of HIV-1 subtypes, and find that they are followed by strong HIV-1 subtype admixture. We find, in these regions, that HIV diversification is followed by a temporal homogenization of HIV-1 genome composition. Our analysis highlights a large degree of admixture between circulating HIV-1 subtype genomes in China and South America, and most probably worldwide. These results suggest that genomic exchanges may indeed be relatively strong, and that HIV-1 is currently in a process of homogenization subsequent to isolation and diversification. HIV genome homogenization may seem to contradict the idea that HIV is undergoing a diversification process (<a href="#bib41">Hamelaar <i>et al.</i> 2011</a>; <a href="#bib98">Tebit and Arts 2011</a>; <a href="#bib29">Feng <i>et al.</i> 2013</a>). However, this homogenization is consistent with observed diversity in the HIV-1 population and the increased number of new HIV recombinant forms described worldwide (<a href="#bib41">Hamelaar <i>et al.</i> 2011</a>; <a href="#bib98">Tebit and Arts 2011</a>; <a href="#bib29">Feng <i>et al.</i> 2013</a>). Indeed, when isolated and diverged populations reconnect, a transient excess of genetic diversity is predicted in the resulting population (<a href="#bib5">Alcala <i>et al.</i> 2013</a>; <a href="#bib4">Alcala and Vuilleumier 2014</a>) as all the genetic material that accumulated during the isolation phase is shared among populations. However, this excess of genetic diversity is transient and with time, the diversity decreases (<a href="#bib5">Alcala <i>et al.</i> 2013</a>). In HIV-1, such processes occur through multiple recombination events among subtypes through coinfection or superinfection and are the source of novel recombinant forms. In the long term, successive recombination events will ultimately lead to the homogenization of the genomic composition of HIV-1. Homogenization might be further enhanced by selective forces imposed by the immune response or treatment acting on the HIV genome (<a href="#bib28">Fellay <i>et al.</i> 2007</a>; <a href="#bib69">Pelak <i>et al.</i> 2011</a>; <a href="#bib21">Chikata <i>et al.</i> 2014</a>). As many world regions are currently being newly colonized by different HIV-1 subtypes, it is expected that new recombinants may still emerge and HIV diversity will further increase. However, as demonstrated here, this is expected to be a transient state toward global homogenization.</p> <p>We find signatures of reconnection events among populations of HIV-1 subtypes that were previously isolated both in China and South America. The signatures on HIV-1 DNA polymorphism suggest reconnection events of three previously isolated HIV-1 populations in both regions. These results align with the known epidemiologic history of HIV-1 in the two regions (<a href="#bib94">Su <i>et al.</i> 2000</a>; <a href="#bib107">Yang <i>et al.</i> 2002</a>; <a href="#bib7">Aulicino <i>et al.</i> 2007</a>; <a href="#bib58">Lu <i>et al.</i> 2008</a>; <a href="#bib97">Takebe <i>et al.</i> 2010</a>; <a href="#bib15">Bello <i>et al.</i> 2011</a>; <a href="#bib6">An <i>et al.</i> 2012</a>; <a href="#bib67">Pang <i>et al.</i> 2012</a>; <a href="#bib42">Han <i>et al.</i> 2013</a>; <a href="#bib29">Feng <i>et al.</i> 2013</a>) and are supported by recently identified recombinants in China (<a href="#bib30">Feng <i>et al.</i> 2014</a>; <a href="#bib103">Wang <i>et al.</i> 2014</a>; <a href="#bib104">Wei <i>et al.</i> 2014</a>; <a href="#bib45">Hsi <i>et al.</i> 2014</a>). To confirm this, we identified the genome sequences that signal the reconnection event. As expected, they reflect first the HIV outbreak in China in 1989: colonization of B and C subtypes (variance of the first peak in the SFS is generated by B and C subtypes). The subsequent colonization of CRF01_AE at the origin of the HIV outbreak in the late 1990s is also signaled in the SFS (a high proportion of CRF01_AE in the highest mode). We obtained similar concordance for the origin of the sequences forming the peaks and the successive subtypes’ colonization events in South America.</p> <p>Our methods also appear robust to high mutation rates and the presence of selection. Indeed, we detect past HIV-1 history despite its extraordinary defining features: HIV has a small genome (about 9.8 kb in length), a short generation time (24 hr; <a href="#bib61">Mohammadi <i>et al.</i> 2013</a>), and a high mutation rate (about 0.002 substitutions/site/year; <a href="#bib52">Korber <i>et al.</i> 2000</a>; <a href="#bib79">Schlub <i>et al.</i> 2014</a>). It is also under strong selective pressure from the immune system (<i>e.g.</i>, <a href="#bib89">Snoeck <i>et al.</i> 2011</a>) and undergoes frequent bottlenecks followed by population expansion [during transmission (<i>e.g.</i>, <a href="#bib50">Keele <i>et al.</i> 2008</a>)]. Nevertheless, the signature observed here in the distributions of pairwise nucleotide difference and on SFS only slightly deviates from theoretical expectations. The robustness of these results may have a simple explanation: a separation of time-scales of the processes involved. Selection and population demographic changes drive divergence of populations during isolation. Migration and recombination during population reconnection allows a rapid – relative to divergence – genome admixture. Both impact genome polymorphism, but the latter does so more drastically and at much shorter time-scales.</p> <p>The evolutionary history of an organism is a powerful resource for understanding its current evolutionary trajectory. The analytical framework developed can be applied to understanding the potential evolutionary pathway of many pathogens and other species that are characterized by periods of isolation. These events are common features of pathogens (<a href="#bib49">Karlsson <i>et al.</i> 2014</a>), but also occur in numerous other species, including humans and domesticated plants and animals (<a href="#bib23">Ellstrand <i>et al.</i> 1999</a>; <a href="#bib54">Lecis <i>et al.</i> 2006</a>; <a href="#bib19">Caicedo <i>et al.</i> 2007</a>; <a href="#bib39">Green <i>et al.</i> 2010</a>; <a href="#bib68">Patterson <i>et al.</i> 2012</a>). Reconnection following a period of isolation is increasingly recognized as an effective mechanism to trigger fast and complex adaptation (<a href="#bib83">Seehausen 2002</a>, <a href="#bib84">2004</a>; <a href="#bib3">Aguilée <i>et al.</i> 2013</a>). Following reconnection, accumulated diversity (from previously isolated populations) is shared, and the resulting high level of genetic diversity may increase the potential for adaptation (<a href="#bib4">Alcala and Vuilleumier 2014</a>). Admixture among divergent populations may also promote complex adaptation by bringing together new functions acquired in different environments that have evolved independently but are compatible with the genome. Such processes have been observed in influenza and in HIV, for example. Indeed, recombination between strains previously isolated in their nonhuman hosts preceded the HIV epidemic in humans (<a href="#bib8">Bailes <i>et al.</i> 2003</a>; <a href="#bib85">Sharp and Hahn 2010</a>; <a href="#bib100">Vuilleumier and Bonhoeffer 2015</a>). Such events are expected to be more frequent in the future due to the increase in human mobility (<a href="#bib49">Karlsson <i>et al.</i> 2014</a>). Finally, our analytical framework could also be applied to evolutionary trajectories of other zoonotic viruses, such as Ebola virus, coronaviruses, hantaviruses, or henipaviruses.</p> <p>It is important to keep in mind that viral population isolation and reconnection events can occur at a much smaller time scale within hosts. There is increasing evidence suggesting that genetic compartmentalization of viral populations occurs in different organs, tissues, or cells, and might play a critical role for virus evolution and more importantly for its control. This has been demonstrated for HIV (<a href="#bib81">Schnell <i>et al.</i> 2011</a>; <a href="#bib18">Buzón <i>et al.</i> 2011</a>) as well as human cytomegalovirus (<a href="#bib72">Renzette <i>et al.</i> 2011</a>, <a href="#bib73">2013</a>, <a href="#bib74">2014</a>, <a href="#bib75">2015</a>). Appling the analytical framework detailed here could provide a powerful tool to dissect the dynamics and genetic composition of virus across different genetic compartments, in addition to those dynamics between hosts.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank the editor Stephen Wright, John Wakeley, and three anonymous reviewers, whose comments and suggestions greatly improved the manuscript. We would also like to thank Kristen Irwin and Nicholas Renzette for careful reading of the manuscript. This project was funded by the Swiss National Science Foundation (SNSF) grants PZ00P3_139421 and 31003A-130065 and PMPDP3_158381 (S.V.), 31003A_149724/1 (A.T.) and an interdisciplinary grant from the Faculty of Biology and Medicine (FBM) of the University of Lausanne (S.V. and A.T.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-supplementary-material" title="supplementary-material"> <p>Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1</a></p> </div> <div class="fn-type-" title=""> <p>Communicating editor: S. I. Wright</p> </div> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Appendix</div> <div class="derivationofthetheoreticalsignatureofpopulationreconnectiononpairwisenucleotidedifferences" title="sec"> <div class="title" tagx="title" title="title">
Derivation of the Theoretical Signature of Population Reconnection on Pairwise Nucleotide Differences</div> <div class="derivationofthedistributionofpairwisecoalescencetimes:" title="sec"> <div class="title" tagx="title" title="title">
Derivation of the distribution of pairwise coalescence times:</div> <p>We here derive the distribution of within- and between-population pairwise coalescence times, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>, assuming a long isolation period (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
3</div> </div></div></span>) and small sequences, <i>i.e.</i>, without recombination. These coalescence times <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> correspond to the time to the most recent common ancestor of two lineages sampled in the same and in different populations, respectively, scaled by the number of genes per population <i>N</i>.</p> <p>The coalescence of two genes under the finite island model can be described by a three states continuous time Markov chain (<a href="#bib66">Notohara 1990</a>): the two genes can be present (1) in the same population, (2) in different populations or (3) be coalesced. The transition probabilities from one state to another depend on the scaled migration rate <i>M</i> between populations and on the number of populations <i>d</i>. Time <i>T</i> is counted in units of <i>N</i> generations. The corresponding Markov chain can be summarized by the following transition rate matrix (<a href="#bib66">Notohara 1990</a>): </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> </div> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
/</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> <div class="mtd_UNKNOWN"> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div><span class="label" tagx="label" title="label">(A.1)</span></div>The probability that two genes were in state (1), (2) or (3) <i>T</i> generations ago (row vector <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div></div></span>), given that they are in state (1), (2) or (3) at the current generation is given by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mn_UNKNOWN">
0</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div></div></span> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></span> at generation 0), where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div></div></span> is a <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
3</div> </div></div></span> matrix. The elements of matrix <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mi_UNKNOWN">
Q</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div></div></span> are denoted <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and represent the probability that a gene in state <i>i</i> at the current generation was in state <i>j T</i> generations ago (<i>cumulative</i> distribution function). Thus, the probability distribution (or probability <i>density</i> function) of within-population and between-population pairwise coalescence times are given by the derivatives of corresponding <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> functions: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
3</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>. Populations are considered isolated when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> and no migration occurs, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> in the transition matrix <b>Q</b>. Populations exchange migrants when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>, thus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> in the transition matrix <b>Q</b>. Consequently, the distributions of within- and between-population pairwise coalescence times, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, following a past isolation event, are: <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
 </div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
 </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
 </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.2a)</span></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN"> </div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
 </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mtext_UNKNOWN">
 </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.2b)</span></div>These distributions have a simple interpretation. In the most recent period, the reconnection period, when <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> correspond to the probability of coalescence of 2 lineages under the finite island model, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>. During the past isolation period, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>, two lineages can only coalesce if they are in the same population after <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> generations of connection (probabilities <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> within- and between-populations, respectively). Their probability of coalescence at generation <i>T</i> in an isolated population is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>. During the older connection period, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>, two genes can coalesce if they were in the same population during isolation (probabilities <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, within- and between-populations, respectively) and if they did not coalesce during isolation (probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>). Their probability of coalescence at generation <i>T</i> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>. Alternatively, they could have coalesced if they were in different populations during isolation (probabilities <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>). Their probability of coalescence at generation <i>T</i> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>.<p /> <p>Functions <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> all depend on the eigenvalues of matrix <b>Q</b>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
+</div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mi_UNKNOWN">
M</div> <div class="mo_UNKNOWN">
−</div> <div class="msqrt_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
Δ</div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
4</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
M</div> </div></div></span>. We obtain: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.3a)</span></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
B</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
B</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
B</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.3b)</span></div>Where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
B</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span>.<p /> </div> <div class="derivationofthedistributionofpairwisenucleotidedifferences:" title="sec"> <div class="title" tagx="title" title="title">
Derivation of the distribution of pairwise nucleotide differences:</div> <p>From equations A.2a and A.2b, we can derive the distributions of within- (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span>) and between-population pairwise nucleotide differences (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>). Given that mutations follow a Poisson process of mean θ, we have <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mn_UNKNOWN">
0</div> <div class="mn_UNKNOWN">
∞</div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mn_UNKNOWN">
0</div> <div class="mn_UNKNOWN">
∞</div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div></div></span>, where <i>k</i> is a positive integer and <i>G</i> is the length of the sequence (under the infinite size model, we assume that <i>G</i> is much larger than the number of segregating site). We can decompose <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> into a sum of smaller integrals (respectively for <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>): </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mn_UNKNOWN">
∞</div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.4)</span></div>Substituting the value of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mi_UNKNOWN">
w</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
T</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> in each term of equation A.4 by the corresponding value in equation A.2a and using the result <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∫</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
T</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="mi_UNKNOWN">
λ</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
T</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
λ</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
λ</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
2</div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mn_UNKNOWN">
1</div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div></div></span>, we obtain: <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
]</div> </div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.5)</span></div>The resulting distributions of pairwise nucleotide differences (from equation A.5) are presented in <a href="#fig1">Figure 1A</a>. Further assuming that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
3</div> </div></div></span> (thus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
≃</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>) leads to: <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.6a)</span></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mi_UNKNOWN">
G</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div></div><span class="label" tagx="label" title="label">(A.6b)</span></div>Terms in equation A.6a (resp. A.6b) have important interpretations in terms of coalescent events. The first term is the contribution of sequences which coalesce during the recent connection period; indeed, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> </div> </div></div></span> (resp. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> </div> </div></div></span>) corresponds to the distribution of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> (resp. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>) under the finite island model at equilibrium, while terms <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> are due to the truncation of the distribution between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span> and infinity. The second term corresponds to sequences that coalesce during the isolation period. Indeed, with probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> (resp. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>), sequences did not coalesce during the reconnection period and were in the same population during isolation; <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div></div></span> correspond to the distribution of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> in an isolated population, and the term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div></div></span> is due to the truncation between 0 and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>. Finally, the third term corresponds to sequences that coalesce during the prior connection period. With probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> (resp. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>), sequences coalesce during the prior connection period (as they did not coalesce during the reconnection period and were in different populations during isolation). Then, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
k</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
θ</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div> </div> </div> </div></div></span> corresponds to the distribution of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> under the finite island model at equilibrium, while terms <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
θ</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> </div> <div class="mstyle_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> <div class="mi_UNKNOWN">
l</div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
θ</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mo_UNKNOWN">
!</div> </div> </div> </div> </div> </div></div></span> are due to the truncation of the distribution between 0 and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div></div></span>.<p /> </div> <div class="derivationofthepowertodetectbimodality:" title="sec"> <div class="title" tagx="title" title="title">
Derivation of the power to detect bimodality:</div> <p>We can derive the power to detect bimodality from the distributions of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> by computing the probability that, in a sample of <i>n</i> independent pairwise differences sequences, at least one is from the first mode and at least one is from the second. Given that the probability for a difference <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
w</div> </div> </div></div></span> to be drawn from the second mode of the distribution is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, the probability that among <i>n</i> differences, at least one is from the first mode and one from the second mode is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
n</div> </div> </div></div></span>. Similarly, the probability to observe a bimodal distribution of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
π</div> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> from a sample of size <i>n</i> is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
q</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mi_UNKNOWN">
n</div> </div> </div></div></span>. These results are used to draw Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>. Note that with <i>n</i> sequences, we can only compute <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> independent pairwise differences, although the total number of pairwise differences we can compute is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span>.</p> </div> </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
Literature Cited</div> <li tag="ref"><a name="bib1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">L. J.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Principal component analysis.</span><span class="source" tagx="source" title="source">Wiley Interdiscip. Rev. Comput. Stat.</span><span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">433</span>–<span class="lpage" tagx="lpage" title="lpage">459</span>.</span></li> <li tag="ref"><a name="bib2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achaz</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Frequency spectrum neutrality tests: one for all and all for one.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">183</span>: <span class="fpage" tagx="fpage" title="fpage">249</span>–<span class="lpage" tagx="lpage" title="lpage">258</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19546320">19546320</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilée</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claessen</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambert</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Adaptive radiation driven by the interplay of eco-evolutionary and landscape dynamics.</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">67</span>: <span class="fpage" tagx="fpage" title="fpage">1291</span>–<span class="lpage" tagx="lpage" title="lpage">1306</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23617909">23617909</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcala</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuilleumier</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Turnover and accumulation of genetic diversity across large time-scale cycles of isolation and connection of populations.</span><span class="source" tagx="source" title="source">Proc. Biol. Sci.</span><span class="volume" tagx="volume" title="volume">281</span>: <span class="fpage" tagx="fpage" title="fpage">20141369</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25253456">25253456</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcala</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streit</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goudet</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuilleumier</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Peak and persistent excess of genetic diversity following an abrupt migration increase.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">193</span>: <span class="fpage" tagx="fpage" title="fpage">953</span>–<span class="lpage" tagx="lpage" title="lpage">971</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23307901">23307901</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Reconstituting the epidemic history of HIV strain crf01_ae among men who have sex with men (msm) in liaoning, northeastern China: Implications for the expanding epidemic among msm in China.</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">86</span>: <span class="fpage" tagx="fpage" title="fpage">12402</span>–<span class="lpage" tagx="lpage" title="lpage">12406</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22933290">22933290</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aulicino</span><span class="given-names" tagx="given-names" title="given-names">P. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bello</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocco</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mangano</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Description of the first full-length HIV type 1 subtype f1 strain in argentina: implications for the origin and dispersion of this subtype in South America.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">1176</span>–<span class="lpage" tagx="lpage" title="lpage">1182</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17961101">17961101</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailes</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibollet-Ruche</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Courgnaud</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peeters</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Hybrid origin of SIV in chimpanzees.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">300</span>: <span class="fpage" tagx="fpage" title="fpage">1713</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12805540">12805540</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlett</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span></span>, <span class="year" tagx="year" title="year">1937</span> <span class="mixed-article-title" title="mixed-article-title">Properties of sufficiency and statistical tests.</span><span class="source" tagx="source" title="source">Proc. R. Soc. Lond. A. Math. Phys. Sci.</span><span class="volume" tagx="volume" title="volume">160</span>: <span class="fpage" tagx="fpage" title="fpage">268</span>–<span class="lpage" tagx="lpage" title="lpage">282</span>.</span></li> <li tag="ref"><a name="bib10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">N. H.</span></span></span>, <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">The effect of hitch-hiking on neutral genealogies.</span><span class="source" tagx="source" title="source">Genet. Res.</span><span class="volume" tagx="volume" title="volume">72</span>: <span class="fpage" tagx="fpage" title="fpage">123</span>–<span class="lpage" tagx="lpage" title="lpage">133</span>.</span></li> <li tag="ref"><a name="bib11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">N. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Etheridge</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span></span>, <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">The effect of selection on genealogies.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">166</span>: <span class="fpage" tagx="fpage" title="fpage">1115</span>–<span class="lpage" tagx="lpage" title="lpage">1131</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15020491">15020491</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaumont</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balding</span><span class="given-names" tagx="given-names" title="given-names">D. J.</span></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Approximate bayesian computation in population genetics.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">162</span>: <span class="fpage" tagx="fpage" title="fpage">2025</span>–<span class="lpage" tagx="lpage" title="lpage">2035</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12524368">12524368</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becquet</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Przeworski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Learning about modes of speciation by computational approaches.</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">63</span>: <span class="fpage" tagx="fpage" title="fpage">2547</span>–<span class="lpage" tagx="lpage" title="lpage">2562</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19228187">19228187</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bello</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soares</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrago</span><span class="given-names" tagx="given-names" title="given-names">C. G.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">The use of bioinformatics for studying HIV evolutionary and epidemiological history in south america.</span><span class="source" tagx="source" title="source">Aids Res. Treat.</span><span class="volume" tagx="volume" title="volume">2011</span>: <span class="fpage" tagx="fpage" title="fpage">154945</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22162803">22162803</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benaglia</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chauveau</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">D. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">D. S.</span></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">mixtools: An r package for analyzing finite mixture models.</span><span class="source" tagx="source" title="source">J. Stat. Softw.</span><span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">29</span>.</span></li> <li tag="ref"><a name="bib17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birkner</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blath</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Computing likelihoods for coalescents with multiple collisions in the infinitely many sites model.</span><span class="source" tagx="source" title="source">J. Math. Biol.</span><span class="volume" tagx="volume" title="volume">57</span>: <span class="fpage" tagx="fpage" title="fpage">435</span>–<span class="lpage" tagx="lpage" title="lpage">465</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18347796">18347796</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buzón</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Codoñer</span><span class="given-names" tagx="given-names" title="given-names">F. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">S. D. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pou</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puertas</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Deep molecular characterization of HIV-1 dynamics under suppressive haart.</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1002314</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22046128">22046128</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caicedo</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boyko</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fledel-Alon</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide patterns of nucleotide polymorphism in domesticated rice.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">1745</span>–<span class="lpage" tagx="lpage" title="lpage">1756</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17907810">17907810</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro-Nallar</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pérez-Losada</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burton</span><span class="given-names" tagx="given-names" title="given-names">G. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">The evolution of HIV: inferences using phylogenetics.</span><span class="source" tagx="source" title="source">Mol. Phylogenet. Evol.</span><span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">777</span>–<span class="lpage" tagx="lpage" title="lpage">792</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22138161">22138161</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chikata</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carlson</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borghan</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naruto</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Host-specific adaptation of HIV-1 subtype b in the japanese population.</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">4764</span>–<span class="lpage" tagx="lpage" title="lpage">4775</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24522911">24522911</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O. G.</span></span></span>, <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences.</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">1185</span>–<span class="lpage" tagx="lpage" title="lpage">1192</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703244">15703244</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellstrand</span><span class="given-names" tagx="given-names" title="given-names">N. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prentice</span><span class="given-names" tagx="given-names" title="given-names">H. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hancock</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span></span>, <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Gene flow and introgression from domesticated plants into their wild relatives.</span><span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">539</span>–<span class="lpage" tagx="lpage" title="lpage">563</span>.</span></li> <li tag="ref"><a name="bib24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Excoffier</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span>, <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Patterns of DNA sequence diversity and genetic structure after a range expansion: lessons from the infinite-island model.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">853</span>–<span class="lpage" tagx="lpage" title="lpage">864</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012760">15012760</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Excoffier</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dupanloup</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huerta-Sánchez</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">V. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foll</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Robust demographic inference from genomic and SNP data.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e1003905</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24204310">24204310</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fay</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C. I.</span></span></span>, <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Hitchhiking under positive darwinian selection.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">155</span>: <span class="fpage" tagx="fpage" title="fpage">1405</span>–<span class="lpage" tagx="lpage" title="lpage">1413</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880498">10880498</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fellay</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shianna</span><span class="given-names" tagx="given-names" title="given-names">K. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colombo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ledergerber</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">A whole-genome association study of major determinants for host control of HIV-1.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">317</span>: <span class="fpage" tagx="fpage" title="fpage">944</span>–<span class="lpage" tagx="lpage" title="lpage">947</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17641165">17641165</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsi</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">The rapidly expanding crf01_ae epidemic in China is driven by multiple lineages of HIV-1 viruses introduced in the 1990s.</span><span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">1793</span>–<span class="lpage" tagx="lpage" title="lpage">1802</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23807275">23807275</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Identification of a novel HIV type 1 circulating recombinant form (crf65_cpx) composed of crf01_ae and subtypes b and c in western Yunnan, China.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">598</span>–<span class="lpage" tagx="lpage" title="lpage">602</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24279591">24279591</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferretti</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez-Enciso</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramos-Onsins</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Optimal neutrality tests based on the frequency spectrum.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">186</span>: <span class="fpage" tagx="fpage" title="fpage">353</span>–<span class="lpage" tagx="lpage" title="lpage">365</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20610408">20610408</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span></span>, <span class="year" tagx="year" title="year">1930</span> <span class="source" tagx="source" title="source">The genetical theory of natural selection</span>, <span class="publisher-name" tagx="publisher-name" title="publisher-name">Clarendon Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>.</span></li> <li tag="ref"><a name="bib33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flahault</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zylberman</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Influenza pandemics: past, present and future challenges.</span><span class="source" tagx="source" title="source">Public Health Rev.</span><span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">319</span>–<span class="lpage" tagx="lpage" title="lpage">340</span>.</span></li> <li tag="ref"><a name="bib34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foll</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shim</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Wfabc: a Wright–Fisher abc-based approach for inferring effective population sizes and selection coefficients from time-sampled data.</span><span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span><span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">87</span>–<span class="lpage" tagx="lpage" title="lpage">98</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24834845">24834845</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Y. X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W. H.</span></span></span>, <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">Statistical tests of neutrality of mutations.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">693</span>–<span class="lpage" tagx="lpage" title="lpage">709</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8454210">8454210</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">C. T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Antigenic and genetic characteristics of swine-origin 2009 a(h1n1) influenza viruses circulating in humans.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">325</span>: <span class="fpage" tagx="fpage" title="fpage">197</span>–<span class="lpage" tagx="lpage" title="lpage">201</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465683">19465683</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaschen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Diversity considerations in HIV-1 vaccine selection.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">296</span>: <span class="fpage" tagx="fpage" title="fpage">2354</span>–<span class="lpage" tagx="lpage" title="lpage">2360</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12089434">12089434</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gravel</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henn</span><span class="given-names" tagx="given-names" title="given-names">B. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutenkunst</span><span class="given-names" tagx="given-names" title="given-names">R. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Indap</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marth</span><span class="given-names" tagx="given-names" title="given-names">G. T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Demographic history and rare allele sharing among human populations.</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">11983</span>–<span class="lpage" tagx="lpage" title="lpage">11988</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21730125">21730125</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briggs</span><span class="given-names" tagx="given-names" title="given-names">A. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maricic</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenzel</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">A draft sequence of the neandertal genome.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">328</span>: <span class="fpage" tagx="fpage" title="fpage">710</span>–<span class="lpage" tagx="lpage" title="lpage">722</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20448178">20448178</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutenkunst</span><span class="given-names" tagx="given-names" title="given-names">R. N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">S. H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustamante</span><span class="given-names" tagx="given-names" title="given-names">C. D.</span></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e1000695</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19851460">19851460</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelaar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gouws</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghys</span><span class="given-names" tagx="given-names" title="given-names">P. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osmanov</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, and <i>WHO-UN AIDS Network for HIV Isolation and Characterisation</i>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Global trends in molecular epidemiology of HIV-1 during 2000–2007.</span><span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">679</span>–<span class="lpage" tagx="lpage" title="lpage">689</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21297424">21297424</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">High prevalence of HIV-1 intersubtype b /c recombinants among injecting drug users in Dehong, China.</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e65337</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23741489">23741489</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harpending</span><span class="given-names" tagx="given-names" title="given-names">H. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batzer</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurven</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jorde</span><span class="given-names" tagx="given-names" title="given-names">L. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Genetic traces of ancient demography.</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">1961</span>–<span class="lpage" tagx="lpage" title="lpage">1967</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9465125">9465125</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hey</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Isolation with migration models for more than two populations.</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">905</span>–<span class="lpage" tagx="lpage" title="lpage">920</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19955477">19955477</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Genome sequence of a novel HIV-1 circulating recombinant form (crf64_bc) identified from Yunnan, China.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">389</span>–<span class="lpage" tagx="lpage" title="lpage">393</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24205972">24205972</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsieh</span><span class="given-names" tagx="given-names" title="given-names">Y.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">T.-Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">P.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">L.-Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Influenza pandemics: past, present and future.</span><span class="source" tagx="source" title="source">J. Formos. Med. Assoc.</span><span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16440064">16440064</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">R. R.</span></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Generating samples under a Wright–Fisher neutral model of genetic variation.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">337</span>–<span class="lpage" tagx="lpage" title="lpage">338</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847089">11847089</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jombart</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devillard</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balloux</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.</span><span class="source" tagx="source" title="source">BMC Genet.</span><span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">94</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20950446">20950446</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlsson</span><span class="given-names" tagx="given-names" title="given-names">E. K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwiatkowski</span><span class="given-names" tagx="given-names" title="given-names">D. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabeti</span><span class="given-names" tagx="given-names" title="given-names">P. C.</span></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Natural selection and infectious disease in human populations.</span><span class="source" tagx="source" title="source">Nat. Rev. Genet.</span><span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">379</span>–<span class="lpage" tagx="lpage" title="lpage">393</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24776769">24776769</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keele</span><span class="given-names" tagx="given-names" title="given-names">B. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giorgi</span><span class="given-names" tagx="given-names" title="given-names">E. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salazar-Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decker</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">K. T.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Identification and characterization of transmitted and early founder virus envelopes in primary HIV-1 infection.</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">105</span>: <span class="fpage" tagx="fpage" title="fpage">7552</span>–<span class="lpage" tagx="lpage" title="lpage">7557</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18490657">18490657</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kimura</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span>, <span class="year" tagx="year" title="year">1969</span> <span class="mixed-article-title" title="mixed-article-title">The number of heterozygous nucleotide sites maintained in a finite population due to steady flux of mutations.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">61</span>: <span class="fpage" tagx="fpage" title="fpage">893</span>–<span class="lpage" tagx="lpage" title="lpage">903</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5364968">5364968</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muldoon</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theiler</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Timing the ancestor of the HIV-1 pandemic strains.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">288</span>: <span class="fpage" tagx="fpage" title="fpage">1789</span>–<span class="lpage" tagx="lpage" title="lpage">1796</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10846155">10846155</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaschen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thakallapally</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kesmir</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary and immunological implications of contemporary HIV-1 variation.</span><span class="source" tagx="source" title="source">Br. Med. Bull.</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>–<span class="lpage" tagx="lpage" title="lpage">42</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11714622">11714622</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lecis</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierpaoli</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biro</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szemethy</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragni</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Bayesian analyses of admixture in wild and domestic cats (<i>Felis silvestris</i>) using linked microsatellite loci.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">119</span>–<span class="lpage" tagx="lpage" title="lpage">131</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16367835">16367835</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ning</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Near full-length genome sequence of a novel HIV type 1 second-generation recombinant form (crf01_ae/crf07_bc) identified among men who have sex with men in jilin, china.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">1604</span>–<span class="lpage" tagx="lpage" title="lpage">1608</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23809010">23809010</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liégeois</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boué</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butel</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mouacha</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Full-length genome analyses of two new simian immunodeficiency virus (SIV) strains from mustached monkeys (<i>c. cephus</i>) in gabon illustrate a complex evolutionary history among the sivmus/mon/gsn lineage.</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">2880</span>–<span class="lpage" tagx="lpage" title="lpage">2898</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25054885">25054885</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">N. H.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">A general method for calculating likelihoods under the coalescent process.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">189</span>: <span class="fpage" tagx="fpage" title="fpage">977</span>–<span class="lpage" tagx="lpage" title="lpage">987</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21900266">21900266</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jia</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">The changing face of HIV in china.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">455</span>: <span class="fpage" tagx="fpage" title="fpage">609</span>–<span class="lpage" tagx="lpage" title="lpage">611</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18833270">18833270</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amos</span><span class="given-names" tagx="given-names" title="given-names">C. I.</span></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Principal components analysis of population admixture.</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e40115</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22808102">22808102</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">X.-L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rubin</span><span class="given-names" tagx="given-names" title="given-names">D. B.</span></span></span>, <span class="year" tagx="year" title="year">1993</span> <span class="mixed-article-title" title="mixed-article-title">Maximum likelihood estimation via the ecm algorithm: A general framework.</span><span class="source" tagx="source" title="source">Biometrika</span><span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">267</span>–<span class="lpage" tagx="lpage" title="lpage">278</span>.</span></li> <li tag="ref"><a name="bib61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohammadi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desfarges</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartha</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joos</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zangger</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">24 hours in the life of HIV-1 in a t cell line.</span><span class="source" tagx="source" title="source">PLoS pathogens</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e1003161</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382686">23382686</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naduvilezhath</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">L. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metzler</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Jaatha: a fast composite-likelihood approach to estimate demographic parameters.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">2709</span>–<span class="lpage" tagx="lpage" title="lpage">2723</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21645157">21645157</a></span></span></li> <li tag="ref"><a name="bib63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawa</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tajima</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span>, <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Simple method for analyzing the pattern of DNA polymorphism and its application to SNP data of human.</span><span class="source" tagx="source" title="source">Genes Genet. Syst.</span><span class="volume" tagx="volume" title="volume">83</span>: <span class="fpage" tagx="fpage" title="fpage">353</span>–<span class="lpage" tagx="lpage" title="lpage">360</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931461">18931461</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">R. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hallatschek</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Genealogies of rapidly adapting populations.</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">110</span>: <span class="fpage" tagx="fpage" title="fpage">437</span>–<span class="lpage" tagx="lpage" title="lpage">442</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23269838">23269838</a></span></span></li> <li tag="ref"><a name="bib65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakeley</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Distinguishing migration from isolation: a markov chain monte carlo approach.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">158</span>: <span class="fpage" tagx="fpage" title="fpage">885</span>–<span class="lpage" tagx="lpage" title="lpage">896</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11404349">11404349</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Notohara</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span>, <span class="year" tagx="year" title="year">1990</span> <span class="mixed-article-title" title="mixed-article-title">The coalescent and the genealogical process in geographically structured population.</span><span class="source" tagx="source" title="source">J. Math. Biol.</span><span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">59</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2277236">2277236</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">Z.-H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Extensive and complex HIV-1 recombination between b′, c and crf01_ae among Idus in south-east Asia.</span><span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">1121</span>–<span class="lpage" tagx="lpage" title="lpage">1129</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22333750">22333750</a></span></span></li> <li tag="ref"><a name="bib68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moorjani</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohland</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Ancient admixture in human history.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">192</span>: <span class="fpage" tagx="fpage" title="fpage">1065</span>–<span class="lpage" tagx="lpage" title="lpage">1093</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22960212">22960212</a></span></span></li> <li tag="ref"><a name="bib69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelak</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Need</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fellay</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shianna</span><span class="given-names" tagx="given-names" title="given-names">K. V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Copy number variation of kir genes influences HIV-1 control.</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e1001208</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140359">22140359</a></span></span></li> <li tag="ref"><a name="bib70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pool</span><span class="given-names" tagx="given-names" title="given-names">J. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbett-Detig</span><span class="given-names" tagx="given-names" title="given-names">R. B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugino</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">K. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardeno</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Population genomics of sub-saharan <i>Drosophila melanogaster</i>: African diversity and non-african admixture.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1003080</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284287">23284287</a></span></span></li> <li tag="ref"><a name="bib71" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reich</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tandon</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazieres</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Reconstructing native american population history.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">488</span>: <span class="fpage" tagx="fpage" title="fpage">370</span>–<span class="lpage" tagx="lpage" title="lpage">374</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22801491">22801491</a></span></span></li> <li tag="ref"><a name="bib72" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renzette</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharjee</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalik</span><span class="given-names" tagx="given-names" title="given-names">T. F.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Extensive genome-wide variability of human cytomegalovirus in congenitally infected infants.</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1001344</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21625576">21625576</a></span></span></li> <li tag="ref"><a name="bib73" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renzette</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharjee</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleiss</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Rapid intrahost evolution of human cytomegalovirus is shaped by demography and positive selection.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e1003735</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086142">24086142</a></span></span></li> <li tag="ref"><a name="bib74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renzette</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">J. D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalik</span><span class="given-names" tagx="given-names" title="given-names">T. F.</span></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Human cytomegalovirus intrahost evolution–a new avenue for understanding and controlling herpesvirus infections.</span><span class="source" tagx="source" title="source">Curr. Opin. Virol.</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">109</span>–<span class="lpage" tagx="lpage" title="lpage">115</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25154343">25154343</a></span></span></li> <li tag="ref"><a name="bib75" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renzette</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pokalyuk</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharjee</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schleiss</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Limits and patterns of cytomegalovirus genomic diversity in humans.</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">E4120</span>–<span class="lpage" tagx="lpage" title="lpage">E4128</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26150505">26150505</a></span></span></li> <li tag="ref"><a name="bib76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradac</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foley</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">HIV-1 nomenclature proposal.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">288</span>: <span class="fpage" tagx="fpage" title="fpage">55</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10766634">10766634</a></span></span></li> <li tag="ref"><a name="bib77" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenberg</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feldman</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">The relationship between coalescence times and population divergence times</span>, pp. <span class="fpage" tagx="fpage" title="fpage">130</span>–<span class="lpage" tagx="lpage" title="lpage">164</span> in <span class="source" tagx="source" title="source">Modern Developments in Theoretical Population Genetics: The legacy of Gustave Malécot</span>, edited by <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veuille</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>.</span></li> <li tag="ref"><a name="bib78" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sankararaman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patterson</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pääbo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reich</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">The date of interbreeding between neandertals and modern humans.</span><span class="source" tagx="source" title="source">PLoS Genet.</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1002947</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23055938">23055938</a></span></span></li> <li tag="ref"><a name="bib79" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlub</span><span class="given-names" tagx="given-names" title="given-names">T. E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grimm</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smyth</span><span class="given-names" tagx="given-names" title="given-names">R. P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cromer</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chopra</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Fifteen to twenty percent of HIV substitution mutations are associated with recombination.</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">88</span>: <span class="fpage" tagx="fpage" title="fpage">3837</span>–<span class="lpage" tagx="lpage" title="lpage">3849</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24453357">24453357</a></span></span></li> <li tag="ref"><a name="bib80" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Excoffier</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span>, <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">152</span>: <span class="fpage" tagx="fpage" title="fpage">1079</span>–<span class="lpage" tagx="lpage" title="lpage">1089</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10388826">10388826</a></span></span></li> <li tag="ref"><a name="bib81" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnell</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spudich</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">R. W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swanstrom</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">HIV-1 replication in the central nervous system occurs in two distinct cell types.</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1002286</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22007152">22007152</a></span></span></li> <li tag="ref"><a name="bib82" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schultz</span><span class="given-names" tagx="given-names" title="given-names">A.-K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bulla</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitner</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">jphmm: improving the reliability of recombination prediction in HIV-1.</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">W647</span>–<span class="lpage" tagx="lpage" title="lpage">W651</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19443440">19443440</a></span></span></li> <li tag="ref"><a name="bib83" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seehausen</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Patterns in fish radiation are compatible with pleistocene desiccation of lake victoria and 14,600 year history for its cichlid species flock.</span><span class="source" tagx="source" title="source">Proc. Biol. Sci.</span><span class="volume" tagx="volume" title="volume">269</span>: <span class="fpage" tagx="fpage" title="fpage">491</span>–<span class="lpage" tagx="lpage" title="lpage">497</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11886641">11886641</a></span></span></li> <li tag="ref"><a name="bib84" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seehausen</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span>, <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Hybridization and adaptive radiation.</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">198</span>–<span class="lpage" tagx="lpage" title="lpage">207</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701254">16701254</a></span></span></li> <li tag="ref"><a name="bib85" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">B. H.</span></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The evolution of HIV-1 and the origin of AIDS.</span><span class="source" tagx="source" title="source">Philos. Trans. R. Soc. Lond. B Biol. Sci.</span><span class="volume" tagx="volume" title="volume">365</span>: <span class="fpage" tagx="fpage" title="fpage">2487</span>–<span class="lpage" tagx="lpage" title="lpage">2494</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20643738">20643738</a></span></span></li> <li tag="ref"><a name="bib86" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">K. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Churchill</span><span class="given-names" tagx="given-names" title="given-names">G. A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aquadro</span><span class="given-names" tagx="given-names" title="given-names">C. F.</span></span></span>, <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">Properties of statistical tests of neutrality for DNA polymorphism data.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">141</span>: <span class="fpage" tagx="fpage" title="fpage">413</span>–<span class="lpage" tagx="lpage" title="lpage">429</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8536987">8536987</a></span></span></li> <li tag="ref"><a name="bib87" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span>, <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Gene genealogies within mutant allelic classes.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">143</span>: <span class="fpage" tagx="fpage" title="fpage">579</span>–<span class="lpage" tagx="lpage" title="lpage">587</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8722806">8722806</a></span></span></li> <li tag="ref"><a name="bib88" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">R. R.</span></span></span>, <span class="year" tagx="year" title="year">1991</span> <span class="mixed-article-title" title="mixed-article-title">Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">129</span>: <span class="fpage" tagx="fpage" title="fpage">555</span>–<span class="lpage" tagx="lpage" title="lpage">562</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1743491">1743491</a></span></span></li> <li tag="ref"><a name="bib89" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snoeck</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fellay</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartha</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douek</span><span class="given-names" tagx="given-names" title="given-names">D. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Telenti</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Mapping of positive selection sites in the HIV-1 genome in the context of RNA and protein structural constraints.</span><span class="source" tagx="source" title="source">Retrovirology</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">87</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22044801">22044801</a></span></span></li> <li tag="ref"><a name="bib90" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hey</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Understanding the origin of species with genome-scale data: modelling gene flow.</span><span class="source" tagx="source" title="source">Nat. Rev. Genet.</span><span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">404</span>–<span class="lpage" tagx="lpage" title="lpage">414</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23657479">23657479</a></span></span></li> <li tag="ref"><a name="bib91" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sousa</span><span class="given-names" tagx="given-names" title="given-names">V. C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grelaud</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hey</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">On the nonidentifiability of migration time estimates in isolation with migration models.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">3956</span>–<span class="lpage" tagx="lpage" title="lpage">3962</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039614">22039614</a></span></span></li> <li tag="ref"><a name="bib92" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strasburg</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">L. H.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Interpreting the estimated timing of migration events between hybridizing species.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">2353</span>–<span class="lpage" tagx="lpage" title="lpage">2366</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21375636">21375636</a></span></span></li> <li tag="ref"><a name="bib93" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strimmer</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O. G.</span></span></span>, <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Exploring the demographic history of DNA sequences using the generalized skyline plot.</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">2298</span>–<span class="lpage" tagx="lpage" title="lpage">2305</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11719579">11719579</a></span></span></li> <li tag="ref"><a name="bib94" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graf</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Briesen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2000</span> <span class="mixed-article-title" title="mixed-article-title">Characterization of a virtually full-length human immunodeficiency virus type 1 genome of a prevalent intersubtype (c/b) recombinant strain in China.</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">74</span>: <span class="fpage" tagx="fpage" title="fpage">11367</span>–<span class="lpage" tagx="lpage" title="lpage">11376</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11070037">11070037</a></span></span></li> <li tag="ref"><a name="bib95" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tajima</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span>, <span class="year" tagx="year" title="year">1989</span> <span class="mixed-article-title" title="mixed-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">123</span>: <span class="fpage" tagx="fpage" title="fpage">585</span>–<span class="lpage" tagx="lpage" title="lpage">595</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255">2513255</a></span></span></li> <li tag="ref"><a name="bib96" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahata</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span>, <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">A genetic perspective on the origin and history of humans.</span><span class="source" tagx="source" title="source">Annu. Rev. Ecol. Syst.</span><span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">343</span>–<span class="lpage" tagx="lpage" title="lpage">372</span>.</span></li> <li tag="ref"><a name="bib97" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takebe</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hase</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uenishi</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Reconstructing the epidemic history of HIV-1 circulating recombinant forms crf07_bc and crf08_bc in east asia: the relevance of genetic diversity and phylodynamics for vaccine strategies.</span><span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">B39</span>–<span class="lpage" tagx="lpage" title="lpage">B44</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20510742">20510742</a></span></span></li> <li tag="ref"><a name="bib98" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tebit</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arts</span><span class="given-names" tagx="given-names" title="given-names">E. J.</span></span></span>, <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Tracking a century of global expansion and evolution of HIV to drive understanding and to combat disease.</span><span class="source" tagx="source" title="source">Lancet Infect. Dis.</span><span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21126914">21126914</a></span></span></li> <li tag="ref"><a name="bib99" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemaire</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Coalescence 2.0: a multiple branching of recent theoretical developments and their applications.</span><span class="source" tagx="source" title="source">Mol. Ecol.</span><span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">2637</span>–<span class="lpage" tagx="lpage" title="lpage">2652</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24750385">24750385</a></span></span></li> <li tag="ref"><a name="bib100" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuilleumier</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Contribution of recombination to the evolutionary history of HIV.</span><span class="source" tagx="source" title="source">Curr. Opin. HIV AIDS</span><span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">77</span>–<span class="lpage" tagx="lpage" title="lpage">127</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25612323">25612323</a></span></span></li> <li tag="ref"><a name="bib101" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakeley</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Pairwise differences under a general model of population subdivision.</span><span class="source" tagx="source" title="source">J. Genet.</span><span class="volume" tagx="volume" title="volume">75</span>: <span class="fpage" tagx="fpage" title="fpage">81</span>–<span class="lpage" tagx="lpage" title="lpage">89</span>.</span></li> <li tag="ref"><a name="bib102" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakeley</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span>, <span class="year" tagx="year" title="year">1999</span> <span class="mixed-article-title" title="mixed-article-title">Nonequilibrium migration in human history.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">153</span>: <span class="fpage" tagx="fpage" title="fpage">1863</span>–<span class="lpage" tagx="lpage" title="lpage">1871</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10581291">10581291</a></span></span></li> <li tag="ref"><a name="bib103" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsi</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Near full-length genome characterization of a new crf01_ae/crf08_bc recombinant transmitted between a heterosexual couple in Guangxi, China.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">484</span>–<span class="lpage" tagx="lpage" title="lpage">488</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24164454">24164454</a></span></span></li> <li tag="ref"><a name="bib104" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">His</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Identification of a novel HIV-1 circulating recombinant form (crf62_bc) in western Yunnan of China.</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">380</span>–<span class="lpage" tagx="lpage" title="lpage">383</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24164474">24164474</a></span></span></li> <li tag="ref"><a name="bib105" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkinson-Herbots</span><span class="given-names" tagx="given-names" title="given-names">H. M.</span></span></span>, <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">The distribution of the coalescence time and the number of pairwise nucleotide differences in a model of population divergence or speciation with an initial period of gene flow.</span><span class="source" tagx="source" title="source">Theor. Popul. Biol.</span><span class="volume" tagx="volume" title="volume">82</span>: <span class="fpage" tagx="fpage" title="fpage">92</span>–<span class="lpage" tagx="lpage" title="lpage">108</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22687581">22687581</a></span></span></li> <li tag="ref"><a name="bib106" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span>, <span class="year" tagx="year" title="year">1931</span> <span class="mixed-article-title" title="mixed-article-title">Evolution in mendelian populations.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">97</span>–<span class="lpage" tagx="lpage" title="lpage">159</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17246615">17246615</a></span></span></li> <li tag="ref"><a name="bib107" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusagawa</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">On-going generation of multiple forms of HIV-1 intersubtype recombinants in the Yunnan province of China.</span><span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">1401</span>–<span class="lpage" tagx="lpage" title="lpage">1407</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12131217">12131217</a></span></span></li> <li tag="ref"><a name="bib108" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Y.-X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C.-I.</span></span></span>, <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Statistical tests for detecting positive selection by utilizing high-frequency variants.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">174</span>: <span class="fpage" tagx="fpage" title="fpage">1431</span>–<span class="lpage" tagx="lpage" title="lpage">1439</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16951063">16951063</a></span></span></li> <li tag="ref"><a name="bib109" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">E. P. F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhuang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span>, <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">HIV prevalence in China: integration of surveillance data and a systematic review.</span><span class="source" tagx="source" title="source">Lancet Infect. Dis.</span><span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">955</span>–<span class="lpage" tagx="lpage" title="lpage">963</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24107261">24107261</a></span></span></li> </ul> </div> </div>  </body></html>