<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/30/846410.atom" />
<meta name="HW.pisa" content="biorxiv;846410v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses" />
<meta name="DC.Identifier" content="10.1101/846410" />
<meta name="DC.Date" content="2019-11-30" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to Pteropus bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by Henipavirus , thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.

Summary sentence Somatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness" />
<meta name="DC.Contributor" content="Noémie Aurine" />
<meta name="DC.Contributor" content="Camille Baquerre" />
<meta name="DC.Contributor" content="Maria Gaudino" />
<meta name="DC.Contributor" content="Christian Jean" />
<meta name="DC.Contributor" content="Claire Dumont" />
<meta name="DC.Contributor" content="Sylvie Rival-Gervier" />
<meta name="DC.Contributor" content="Clémence Kress" />
<meta name="DC.Contributor" content="Branka Horvat" />
<meta name="DC.Contributor" content="Bertrand Pain" />
<meta name="article:published_time" content="2019-11-30" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to &lt;i&gt;Pteropus&lt;/i&gt; bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by &lt;i&gt;Henipavirus&lt;/i&gt;, thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.&lt;/p&gt;&lt;h3&gt;Summary sentence&lt;/h3&gt;
&lt;p&gt;Somatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;846410v1" />
<meta name="citation_id" content="846410v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/846410v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/846410v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/846410v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410.full.pdf" />
<meta name="citation_doi" content="10.1101/846410" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="846410" />
<meta name="citation_author" content="Noémie Aurine" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author_institution" content="CIRI, International Center for Infectiology Research, University of Lyon" />
<meta name="citation_author" content="Camille Baquerre" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author" content="Maria Gaudino" />
<meta name="citation_author_institution" content="CIRI, International Center for Infectiology Research, University of Lyon" />
<meta name="citation_author" content="Christian Jean" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author" content="Claire Dumont" />
<meta name="citation_author_institution" content="CIRI, International Center for Infectiology Research, University of Lyon" />
<meta name="citation_author" content="Sylvie Rival-Gervier" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8266-8095" />
<meta name="citation_author" content="Clémence Kress" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6379-9735" />
<meta name="citation_author" content="Branka Horvat" />
<meta name="citation_author_institution" content="CIRI, International Center for Infectiology Research, University of Lyon" />
<meta name="citation_author_email" content="bertrand.pain@inserm.fr" />
<meta name="citation_author_email" content="branka.horvat@inserm.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0578-7765" />
<meta name="citation_author" content="Bertrand Pain" />
<meta name="citation_author_institution" content="University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute" />
<meta name="citation_author_email" content="bertrand.pain@inserm.fr" />
<meta name="citation_author_email" content="branka.horvat@inserm.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1210-8444" />
<meta name="citation_reference" content="Jones, K. E. et al. Global trends in emerging infectious diseases. Nature 451, 990–993 (2008)." />
<meta name="citation_reference" content="Halpin, K. et al. Pteropid Bats are Confirmed as the Reservoir Hosts of Henipaviruses: A Comprehensive Experimental Study of Virus Transmission. Am. J. Trop. Med. Hyg. 85, 946–951 (2011)." />
<meta name="citation_reference" content="Li, W. et al. Bats Are Natural Reservoirs of SARS-Like Coronaviruses. Science 310, 676–679 (2005)." />
<meta name="citation_reference" content="Mohd, H. A., Al-Tawfiq, J. A. &amp; Memish, Z. A. Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir. Virol. J. 13, (2016)." />
<meta name="citation_reference" content="Wang, L.-F. &amp; Anderson, D. E. Viruses in bats and potential spillover to animals and humans. Curr. Opin. Virol. 34, 79–89 (2019)." />
<meta name="citation_reference" content="Leroy, E. M. et al. Fruit bats as reservoirs of Ebola virus. Nature 438, 575–576 (2005)." />
<meta name="citation_reference" content="Middleton, D. J. et al. Experimental Nipah Virus Infection in Pigs and Cats. J. Comp. Pathol. 126, 124–136 (2002)." />
<meta name="citation_reference" content="Ching, P. K. G. et al. Outbreak of Henipavirus Infection, Philippines, 2014. Emerg. Infect. Dis. 21, 328–331 (2015)." />
<meta name="citation_reference" content="Luby, S. P. et al. Foodborne Transmission of Nipah Virus, Bangladesh. Emerg. Infect. Dis. 12, 1888–1894 (2006)." />
<meta name="citation_reference" content="Nikolay, B. et al. Transmission of Nipah Virus – 14 Years of Investigations in Bangladesh. N. Engl. J. Med. 380, 1804–1814 (2019)." />
<meta name="citation_reference" content="Mathieu, C. &amp; Horvat, B. Henipavirus pathogenesis and antiviral approaches. Expert Rev. Anti Infect. Ther. 13, 343–354 (2015)." />
<meta name="citation_reference" content="Drexler, J. F. et al. Bats host major mammalian paramyxoviruses. Nat. Commun. 3, 796 (2012)." />
<meta name="citation_reference" content="Luby, S. P. The pandemic potential of Nipah virus. Antiviral Res. 100, 38–43 (2013)." />
<meta name="citation_reference" content="Solari, S. &amp; Baker, R. J. Mammal Species of the World: A Taxonomic and Geographic Reference by D. E. Wilson; D. M. Reeder. J. Mammal. 88, 824–830 (2007)." />
<meta name="citation_reference" content="van der Poel, W. H. M., Lina, P. H. C. &amp; Kramps, J. A. Public health awareness of emerging zoonotic viruses of bats: a European perspective. Vector Borne Zoonotic Dis. Larchmt. N 6, 315–324 (2006)." />
<meta name="citation_reference" content="Teeling, E. C. et al. Bat Biology, Genomes, and the Bat1K Project: To Generate Chromosome-Level Genomes for All Living Bat Species. Annu. Rev. Anim. Biosci. 6, 23–46 (2018)." />
<meta name="citation_reference" content="Austad, S. N. Comparative biology of aging. J. Gerontol. A. Biol. Sci. Med. Sci. 64, 199–201 (2009)." />
<meta name="citation_reference" content="Teeling, E. C. et al. A Molecular Phylogeny for Bats Illuminates Biogeography and the Fossil Record. Science 307, 580–584 (2005)." />
<meta name="citation_reference" content="Schountz, T., Baker, M. L., Butler, J. &amp; Munster, V. Immunological Control of Viral Infections in Bats and the Emergence of Viruses Highly Pathogenic to Humans. Front. Immunol. 8, (2017)." />
<meta name="citation_reference" content="Olival, K. J. et al. Host and viral traits predict zoonotic spillover from mammals. Nature 546, 646–650 (2017)." />
<meta name="citation_reference" content="Luis, A. D. et al. A comparison of bats and rodents as reservoirs of zoonotic viruses: are bats special? Proc. R. Soc. B Biol. Sci. 280, (2013)." />
<meta name="citation_reference" content="Enchéry, F. &amp; Horvat, B. Understanding the interaction between henipaviruses and their natural host, fruit bats: Paving the way toward control of highly lethal infection in humans. Int. Rev. Immunol. 36, 108–121 (2017)." />
<meta name="citation_reference" content="Banerjee, A., Misra, V., Schountz, T. &amp; Baker, M. L. Tools to study pathogen-host interactions in bats. Virus Res. 248, 5–12 (2018)." />
<meta name="citation_reference" content="Hare, D., Collins, S., Cuddington, B. &amp; Mossman, K. The Importance of Physiologically Relevant Cell Lines for Studying Virus–Host Interactions. Viruses 8, (2016)." />
<meta name="citation_reference" content="Crameri, G. et al. Establishment, immortalisation and characterisation of pteropid bat cell lines. PloS One 4, e8266 (2009)." />
<meta name="citation_reference" content="Takahashi, K. &amp; Yamanaka, S. Induction of Pluripotent Stem Cells from Mouse Embryonic and Adult Fibroblast Cultures by Defined Factors. Cell 126, 663–676 (2006)." />
<meta name="citation_reference" content="Ogorevc, J., Orehek, S. &amp; Dovč, P. Cellular reprogramming in farm animals: an overview of iPSC generation in the mammalian farm animal species. J. Anim. Sci. Biotechnol. 7, (2016)." />
<meta name="citation_reference" content="Ezashi, T., Yuan, Y. &amp; Roberts, R. M. Pluripotent Stem Cells from Domesticated Mammals. Annu. Rev. Anim. Biosci. 4, 223–253 (2016)." />
<meta name="citation_reference" content="Stanton, M. M. et al. Prospects for the Use of Induced Pluripotent Stem Cells in Animal Conservation and Environmental Protection. STEM CELLS Transl. Med. 8, 7–13 (2019)." />
<meta name="citation_reference" content="Mo, X., Li, N. &amp; Wu, S. Generation and characterization of bat-induced pluripotent stem cells. Theriogenology 82, 283–293 (2014)." />
<meta name="citation_reference" content="Fuet, A. et al. NANOG Is Required for the Long-Term Establishment of Avian Somatic Reprogrammed Cells. Stem Cell Rep. 11, 1272–1286 (2018)." />
<meta name="citation_reference" content="Trusler, O., Huang, Z., Goodwin, J. &amp; Laslett, A. L. Cell surface markers for the identification and study of human naive pluripotent stem cells. Stem Cell Res. 26, 36–43 (2018)." />
<meta name="citation_reference" content="Tosolini, M. et al. Contrasting epigenetic states of heterochromatin in the different types of mouse pluripotent stem cells. Sci. Rep. 8, 5776 (2018)." />
<meta name="citation_reference" content="Kress, C., Montillet, G., Jean, C., Fuet, A. &amp; Pain, B. Chicken embryonic stem cells and primordial germ cells display different heterochromatic histone marks than their mammalian counterparts. Epigenetics Chromatin 9, 5 (2016)." />
<meta name="citation_reference" content="Zalzman, M. et al. Zscan4 regulates telomere elongation and genomic stability in ES cells. Nature 464, 858–863 (2010)." />
<meta name="citation_reference" content="Wu, X. et al. Intrinsic Immunity Shapes Viral Resistance of Stem Cells. Cell 172, 423–438.e25 (2018)." />
<meta name="citation_reference" content="Vercammen, E., Staal, J. &amp; Beyaert, R. Sensing of Viral Infection and Activation of Innate Immunity by Toll-Like Receptor 3. Clin. Microbiol. Rev. 21, 13–25 (2008)." />
<meta name="citation_reference" content="Fortier, M.-E. et al. The viral mimic, polyinosinic:polycytidylic acid, induces fever in rats via an interleukin-1-dependent mechanism. Am. J. Physiol. Regul. Integr. Comp. Physiol. 287, R759–766 (2004)." />
<meta name="citation_reference" content="Young, C. C. W. &amp; Olival, K. J. Optimizing Viral Discovery in Bats. PloS One 11, e0149237 (2016)." />
<meta name="citation_reference" content="Trevisan, M. et al. Modeling Viral Infectious Diseases and Development of Antiviral Therapies Using Human Induced Pluripotent Stem Cell-Derived Systems. Viruses 7, 3835–3856 (2015)." />
<meta name="citation_reference" content="Guo, Y.-L. Utilization of Different Antiviral Mechanisms by Mammalian Embryonic Stem Cells and Differentiated Cells. Immunol. Cell Biol. 95, 17–23 (2017)." />
<meta name="citation_reference" content="Chen, L.-L., Yang, L. &amp; Carmichael, G. G. Molecular basis for an attenuated cytoplasmic dsRNA response in human embryonic stem cells. Cell Cycle 9, 3552–3564 (2010)." />
<meta name="citation_reference" content="Wang, R. et al. Mouse Embryonic Stem Cells Have Underdeveloped Antiviral Mechanisms That Can Be Exploited for the Development of mRNA-Mediated Gene Expression Strategy. Stem Cells Dev. 23, 594–604 (2014)." />
<meta name="citation_reference" content="Giotis, E. S., Montillet, G., Pain, B. &amp; Skinner, M. A. Chicken Embryonic-Stem Cells Are Permissive to Poxvirus Recombinant Vaccine Vectors. Genes 10, (2019)." />
<meta name="citation_reference" content="Hong, X.-X. &amp; Carmichael, G. G. Innate Immunity in Pluripotent Human Cells. J. Biol. Chem. 288, 16196–16205 (2013)." />
<meta name="citation_reference" content="Harada, H. et al. Absence of the type I IFN system in EC cells: transcriptional activator (IRF-1) and repressor (IRF-2) genes are developmentally regulated. Cell 63, 303–312 (1990)." />
<meta name="citation_reference" content="Zhou, P. et al. IRF7 in the Australian black flying fox, Pteropus alecto: evidence for a unique expression pattern and functional conservation. PloS One 9, e103875 (2014)." />
<meta name="citation_reference" content="Li, S., Lu, L.-F., Wang, Z.-X., Chen, D.-D. &amp; Zhang, Y.-A. Fish IRF6 is a positive regulator of IFN expression and involved in both of the MyD88 and TBK1 pathways. Fish Shellfish Immunol. 57, 262–268 (2016)." />
<meta name="citation_reference" content="Ainouze, M. et al. Human papillomavirus type 16 antagonizes IRF6 regulation of IL-1β. PLoS Pathog. 14, e1007158 (2018)." />
<meta name="citation_reference" content="Zhang, G. et al. Comparative analysis of bat genomes provides insight into the evolution of flight and immunity. Science 339, 456–460 (2013)." />
<meta name="citation_reference" content="De La Cruz-Rivera, P. C. et al. The IFN Response in Bats Displays Distinctive IFN-Stimulated Gene Expression Kinetics with Atypical RNASEL Induction. J. Immunol. Baltim. Md 1950 200, 209–217 (2018)." />
<meta name="citation_reference" content="Zhou, P. et al. Contraction of the type I IFN locus and unusual constitutive expression of IFN-α in bats. Proc. Natl. Acad. Sci. U. S. A. 113, 2696–2701 (2016)." />
<meta name="citation_reference" content="Pavlovich, S. S. et al. The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity. Cell 173, 1098–1110.e18 (2018)." />
<meta name="citation_reference" content="Medzhitov, R., Schneider, D. S. &amp; Soares, M. P. Disease tolerance as a defense strategy. Science 335, 936–941 (2012)." />
<meta name="citation_reference" content="Mandl, J. N., Schneider, C., Schneider, D. S. &amp; Baker, M. L. Going to Bat(s) for Studies of Disease Tolerance. Front. Immunol. 9, 2112 (2018)." />
<meta name="citation_reference" content="Aubel, P. &amp; Pain, B. Chicken embryonic stem cells: establishment and characterization. Methods Mol. Biol. Clifton NJ 1074, 137–150 (2013)." />
<meta name="citation_reference" content="Fuet, A. &amp; Pain, B. Chicken Induced Pluripotent Stem Cells: Establishment and Characterization. Methods Mol. Biol. Clifton NJ 1650, 211–228 (2017)." />
<meta name="citation_reference" content="Coronado, D. et al. A short G1 phase is an intrinsic determinant of naïve embryonic stem cell pluripotency. Stem Cell Res. 10, 118–131 (2013)." />
<meta name="citation_reference" content="Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525–527 (2016)." />
<meta name="citation_reference" content="Chen, S., Zhou, Y., Chen, Y. &amp; Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34, i884–i890 (2018)." />
<meta name="citation_reference" content="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, (2014)." />
<meta name="citation_reference" content="Mathieu, C. et al. Lethal Nipah Virus Infection Induces Rapid Overexpression of CXCL10. PLoS ONE 7, (2012)." />
<meta name="citation_reference" content="Zhou, P. et al. Type III IFN Receptor Expression and Functional Characterisation in the Pteropid Bat, Pteropus alecto. PLoS ONE 6, (2011)." />
<meta name="twitter:title" content="Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to Pteropus bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by Henipavirus , thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.

Summary sentence Somatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness" />
<meta name="og-title" property="og:title" content="Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/846410v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to Pteropus bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by Henipavirus , thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.

Summary sentence Somatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-30" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/846410v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/846410v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/846410v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/846410v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1015730" />
    <title>Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vuLfvV8VP9hS6Spw5kkAnZ72nFGnFyC-y6LIxl_P0BY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1015730","apath":"\/biorxiv\/early\/2019\/11\/30\/846410.atom","pisa":"biorxiv;846410v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;846410v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/846410v1","title":"Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses","identifier":"node\/1015730"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rmb6IxvsFQZkA0w83DCJOeitZr4aHSv+e7teYkrlKE2tv+KX+S09P7XimjkluX9a0zG1whMvNnODQi2ZNuD5RsQpxorUgvT8buA2QZSgS2BJwMqy9lZPNNif73OG9ky+XhlhOo0TEGC07fegNb1VJ1LLCta7\\\\\\\/AFXmXQrDkQXlDorlFOynLTNcoYfQT3OxiLC6\\\\\\\/XLQd08pqLoiDW6Wfxvd7K2zz5FBJgKuDZ2rEnZ943XoqLgYimV+tH0D7crgMRtbU0NbtRoEPgSXI6KVFtlJWORwnmmXuLXieTyY3xYcjvfb5nh74e0OWvDCycZ0QUmBhcSB6Ci9HPfTUthawhr8bdSTlokbTiItzq\\\\\\\/u7o5CxAx7L5zlQcPy6aZkpOH\\\\\\\/Aw0aDS0LlLWXN+4IVr6gea7EEp004dy4L0mS+K7jIimA33N97ylrvez3A8INeDXaUb0O1QHJJL\\\\\\\/dnoPrua6tIE\\\\\\\/H2XpNDv1V0TRaUGYHQqCiiBU5SM9pZ3AgSss2\\\\\\\/mJwcsX7SAM4Gq2lhCKEe35wMvQbfwYir5u\\\\\\\/xXHLfZN9iBPOXVFk+nTNNbU5n4uEjoxNTYoRa8f6Zp7tUA9BZc4tBq7sGmr2dDf7WqxVshp07xSFjdK9TTk+bhDcvau4aTfvzAiOwnpIQviVshCOXqdgLOrLEETx5coNmYet1eHq2HKca4uDTzJdvAnQUFMixH2y244YJClNAtW97+RMYyfkfcLUAT5MPuRdR+GaFOh2P9e\\\\\\\/2rGkTk4T7BJaM1+1q1RJ732ZAzJnhjboKIXJr\\\\\\\/tgzVNhkI2KSdPHKfWTjqMz9CT116laXPdDYAKM6ObLv7Lqf42WB0QQqLHviInVN\\\\\\\/l5Gv53uD04GzDQjXlh45SPXAgwOenEmbf3SSxHBiV99s3O6XTuUEqTlhTqfoO75wDzrY20BZlrcel+JAG3EUiSKd3lyaOwmjJaPSXzkjBRaNH3rV5lOswknpGAnzjtjfauiIBP4u2Ha4voRfk\\\\\\\/qzaXFTIdfNtmnIouEN5OQxAOeyaiJw627hSVakbVGGSJJhimRo8SO3SygOS0b1QQUZaAaerwwaGLpY9GHtXGfx84N3iy1lVvpcKcehCmVwcacqlvquy7X48Yo08rztZJD3LNJxaPNAiC2W78cp0vGnHbIpSuTsEaPPbeZud7DDrFXNYhntHR6iOsf5LIoc5CJmbkMw\\\\\\\/blDeV8BGMIK6T22Mss2ybm45TnEsCB1\\\\\\\/OboNS+3MVNdLN04sd8hnOaKxdm+RPTR38uhATLsbk5D9OQYN6MaTaZmOCtdvL49oidWs4F4M7fcvjX+04T+ac0tOSLKtNbykithaRZI0CtM6ScqdHUL9msOy\\\u0022,\\\u0022iv\\\u0022:\\\u0022uVQNE0GOPYgTPiF4051DvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002248aeecf5543916d51300470a7095df891dda780ba5625a964c938628bb08454e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/846410v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1015730 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/846410v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1qv1FJQxL_f8SdtEwK_SThvd8D9sB8wq0YE7hTrHHH4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1015730" id="node1015730" data-pisa="biorxiv;846410v1" data-pisa-master="biorxiv;846410" data-apath="/biorxiv/early/2019/11/30/846410.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Reprogrammed <em>Pteropus</em> Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Noémie</span> <span class="nlm-surname">Aurine</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Camille</span> <span class="nlm-surname">Baquerre</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Gaudino</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Jean</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Dumont</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-8266-8095" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sylvie</span> <span class="nlm-surname">Rival-Gervier</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6379-9735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Clémence</span> <span class="nlm-surname">Kress</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-0578-7765" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Branka</span> <span class="nlm-surname">Horvat</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1210-8444" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bertrand</span> <span class="nlm-surname">Pain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/846410 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1015730" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Noémie Aurine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CIRI, International Center for Infectiology Research, University of Lyon</span>, Université Claude Bernard Lyon 1, INSERM, U1111, CNRS, UMR5308, ENS Lyon, Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=No%C3%A9mie%2BAurine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aurine%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANo%25C3%25A9mie%2BAurine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Camille Baquerre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camille%2BBaquerre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baquerre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamille%2BBaquerre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maria Gaudino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CIRI, International Center for Infectiology Research, University of Lyon</span>, Université Claude Bernard Lyon 1, INSERM, U1111, CNRS, UMR5308, ENS Lyon, Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BGaudino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaudino%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BGaudino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christian Jean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BJean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jean%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BJean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Claire Dumont </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CIRI, International Center for Infectiology Research, University of Lyon</span>, Université Claude Bernard Lyon 1, INSERM, U1111, CNRS, UMR5308, ENS Lyon, Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BDumont%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dumont%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BDumont%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sylvie Rival-Gervier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sylvie%2BRival-Gervier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rival-Gervier%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASylvie%2BRival-Gervier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8266-8095" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sylvie Rival-Gervier</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Clémence Kress </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cl%C3%A9mence%2BKress%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kress%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACl%25C3%25A9mence%2BKress%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6379-9735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Clémence Kress</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Branka Horvat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CIRI, International Center for Infectiology Research, University of Lyon</span>, Université Claude Bernard Lyon 1, INSERM, U1111, CNRS, UMR5308, ENS Lyon, Lyon, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Branka%2BHorvat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Horvat%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABranka%2BHorvat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0578-7765" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Branka Horvat</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bertrand.pain@inserm.fr" class="" data-icon-position="" data-hide-link-title="0">bertrand.pain@inserm.fr</a>
<a href="mailto:branka.horvat@inserm.fr" class="" data-icon-position="" data-hide-link-title="0">branka.horvat@inserm.fr</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Bertrand Pain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>University of Lyon, Université Lyon 1, INSERM, INRA, Stem Cell and Brain Research Institute</span>, U1208, USC1361, Bron, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bertrand%2BPain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pain%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABertrand%2BPain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1210-8444" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bertrand Pain</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bertrand.pain@inserm.fr" class="" data-icon-position="" data-hide-link-title="0">bertrand.pain@inserm.fr</a>
<a href="mailto:branka.horvat@inserm.fr" class="" data-icon-position="" data-hide-link-title="0">branka.horvat@inserm.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/846410v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/846410v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/846410v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/846410v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/846410v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/846410v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1015730" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1015730/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Bats are unique among mammals due to the ability of powered flight and exceptional longevity. They are also asymptomatic hosts for numerous viruses, including recently emerged zoonotic Henipaviruses Nipah and Hendra, which are highly pathogenic for humans and other mammals. Better understanding of how bats control viral infection requires development of relevant permissive cellular experimental models. By applying a somatic reprogramming protocol to <em>Pteropus</em> bat primary cells, using a novel combination of ESRRB, CDX2, and c-MYC transcription factors, we generated bat reprogrammed cells exhibiting stem cell-like characteristics and a neural stem cell-like molecular signature. These cells present a unique interferon-stimulated transcriptomic signature and both produce and respond to interferon type-I, highlighting differences between stem cells from bats and other mammals. In contrast to primary bat cells, these reprogrammed cells are highly susceptible to infection by <em>Henipavirus</em>, thereby enabling isolation of new bat viruses, study of virus-bat interactions, and better understanding of bat biology.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Summary sentence</strong> Somatic reprogramming provides new bat stem cells with unique immune properties and original viral permissivness</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">Emergence of zoonotic diseases is increasing globally; thus the ability to predict and prevent viral epidemics is a major objective for public health organizations. Among all emerging infectious diseases, approximately 60% are of zoonotic origin<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. Bats host a large number of human and animal zoonotic viruses, including Henipaviruses<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup> (Nipah (NiV) and Hendra (HeV)), Coronaviruses (Severe Acute Respiratory Syndrome (SARS-CoV)<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup> and Middle East Respiratory (MERS-CoV))<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>, and Filoviruses (Marburg<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup> and Ebola viruses<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>). These agents are among the most virulent pathogens emerging from animal reservoirs and are capable of infecting a broad range of species, including humans, while remaining asymptomatic in bats. Transmission of NiV from bats to humans occurs either through intermediate host contamination<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> or directly<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> via consumption of fruits or raw date palm juice contaminated with bat saliva or urine. In the case of NiV-Bangladesh isolates, this may be followed by inter-human transmission<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Henipaviruses are negative strand RNA viruses; both NiV and HeV are responsible for outbreaks of respiratory and neurological diseases in Southeast Asia and Australia, respectively, with a fatality rate of 40–100%<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. NiV and Henipa-like viruses have been detected molecularly and/or serologically in <em>Pteropus</em> bats from different Asian and African Countries<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, and worldwide distribution of these bat species poses a threat of future NiV pandemics<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>.</p><p id="p-5">Bats are an ancient and diverse group of animals (the second largest order of mammals after rodents), representing around 20% of all mammalian species<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. Indeed, they are the most abundant and widely distributed non-human mammalian species worldwide<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Bats, the only mammals capable of powered flight, show extraordinary longevity compared with their corporal mass<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Phylogenetic analyses based on molecular data classified bats into two suborders: Yangochiroptera (microbat families) and Yinpterochiroptera (including the non-echolocating <em>Pteropodidae</em> family, which contains the <em>Pteropus genus</em> and the echolocating <em>Rhinolophoidea</em>)<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. A large number of novel bat virus sequences was identified in these two bat suborders, which diverged over 50 million years ago, suggesting a long co-evolutionary history of bats and their viruses that have shaped unique host-pathogen relationships<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. In addition, bats harbor a markedly higher proportion of zoonotic viruses than other mammalian orders<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Although bat serology and viral genome sequencing permit detection of a number of bat viruses, isolating them has been far more challenging. Studying the bat immune system and bat/virus interactions provides valuable insight into the mechanisms underlying successful control of viral infection, and may lead to novel approaches to managing viral spillover and development of new antiviral strategies for humans<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. As infection studies in bats are difficult to implement due to the animal facility constraints and the absence of bat husbandry, it is crucial to establish adequate research tools for comparative <em>in vitro</em> infection, for studying virus-host interactions, and for isolating new bat viruses<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-6">Availability of relevant cell lines is particularly important and presents a critical obstacle for further study of virus-host interactions<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. In humans, the cellular targets of Henipaviruses are endothelial, epithelial, and neural cells. Although several cell lines from <em>Pteropus</em> bats have been established, most are immortalized primary cells (PCs) with fibroblast-like morphology. Utilization of high multiplicities of infection by most studies<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> suggests that these cells show low permissiveness for <em>Henipavirus</em> infection and replication. The use of primary and immortalized cells limits our ability to isolate and study viruses that may show tropism for cell types that are more difficult to establish as a primary culture or to immortalize, thereby emphasizing the need to develop more appropriate <em>Pteropus</em> bat cells with different phenotypes.</p><p id="p-7">Pluripotent stem cells (PSCs) are able to self-renew <em>in vitro</em> and differentiate into derivatives of the three embryonic germ layers: endoderm, ectoderm, and mesoderm. Somatic reprogramming generates induced PSCs (iPSCs) via four pluripotent transcription factors, known as OSKM (a combination of the POU5F1/OCT4, SOX2, KLF4 and c-MYC genes)<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Initially demonstrated in mice, then in humans, rats, and non-human primates, the process has been tested (with variable success) in a large number of mammalian species, including rabbits, sheep, bovines, pigs, horses, dogs, and even in some endangered species<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>; however, despite the importance of bats to ecology and immunovirology, and although certain attempts have been made in that direction<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>, somatic reprogramming of primary bat cells has not been examined rigorously.</p><p id="p-8">Here, we generated <em>Pteropus</em> bat reprogrammed stem cells (RSCs) from PCs using a novel combination of three transcription factors: ESRRB, CDX2 and c-MYC. These cells exhibit typical PSC morphology, are recognized by SSEA-1 and EMA-1 stem-cell-specific antibodies, display PSC-specific chromatin features, and express a set of neural stem cell-associated genes. These reprogrammed cells are much more highly susceptible to virus infection than the parental primary bat cells, making them a new cell type that can be used to isolate novel bat viruses and study host-pathogen interactions. Interestingly, in contrast to stem cells from other mammalian species, bat RSCs are capable of both responding to and producing IFN-I, and present a unique interferon-stimulated gene (ISG) transcriptomic signature, suggesting some fundamental differences between bats and other mammals. This new cell model is an important experimental tool that will improve understanding of the unique properties of bat/virus interactions and will help to decipher novel aspects of bat biology.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3><em>Pteropus</em> bat PCs show limited proliferation <em>in vitro</em></h3><p id="p-9">Explants derived from <em>Pteropus</em> bat trachea (PTC), lung (PTL), and alary membrane skin (PAC) were cultured in gelatin-coated wells either in fibroblast medium (FM) or in ES derived medium (ESM1). Individual cells emerged from these explants at 5, 15, and 30 days after plating, respectively; all cells exhibited a fibroblastic morphology, with an elongated and often flattened shape (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a–c</a>). PLCs entered senescence after 4 to 5 generations, while PACs became senescent after 10–11 generations; by contrast, PTCs continued to proliferate for more than 30 generations (200 days of continuous culture), although there was a slow, progressive and irreversible decrease in the proliferation rate during that time (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1d</a>). These results strongly suggested that primary cultures obtained from different types of bat tissue have a limited life span <em>in vitro</em>, similar to that of other mammalian PC cultures.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of bat reprogrammed stem cells. Bat primary cell cultures (bat PC) derived from trachea (PTC) (a), lung (PLC) (b), and alary membrane (PAC) (c) observed under a light microscope. (d) Comparative growth curves of Pteropus primary cells. Somatic reprogramming of bat PCs with inducible transposons encoding CDX2, c-MYC, and ESRRB genes and culture in EpiStem (EPI) (e) or ESM2 (f) media. (g) Comparative growth curves of bat PC and reprogrammed bat stem cells (bat RSCs). Scale bar, 200 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Generation of bat reprogrammed stem cells.&lt;/span&gt; Bat primary cell cultures (bat PC) derived from trachea (PTC) (a), lung (PLC) (b), and alary membrane (PAC) (c) observed under a light microscope. (d) Comparative growth curves of &lt;em&gt;Pteropus&lt;/em&gt; primary cells. Somatic reprogramming of bat PCs with inducible transposons encoding CDX2, c-MYC, and ESRRB genes and culture in EpiStem (EPI) (e) or ESM2 (f) media. (g) Comparative growth curves of bat PC and reprogrammed bat stem cells (bat RSCs). Scale bar, 200 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F1.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F1.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <span class="caption-title">Generation of bat reprogrammed stem cells.</span><p id="p-10" class="first-child">Bat primary cell cultures (bat PC) derived from trachea (PTC) (a), lung (PLC) (b), and alary membrane (PAC) (c) observed under a light microscope. (d) Comparative growth curves of <em>Pteropus</em> primary cells. Somatic reprogramming of bat PCs with inducible transposons encoding CDX2, c-MYC, and ESRRB genes and culture in EpiStem (EPI) (e) or ESM2 (f) media. (g) Comparative growth curves of bat PC and reprogrammed bat stem cells (bat RSCs). Scale bar, 200 μm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Reprogramming of primary <em>Pteropus</em> bat cells</h3><p id="p-11">We wanted to obtain bat cells with stem cell features and the ability to proliferate long-term but did not want to use immortalizing agents. Therefore, we used a somatic reprograming approach to introduce selected “reprogramming” genes into PCs. Based on the partially annotated genome of <em>P. vampyrus</em>, alignment of protein sequences derived from human and <em>P. vampyrus</em> genes revealed identities ranging from 87–99% between seven major reprogramming human genes, with the noticeable exception of NANOG, which showed only 70% identity (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Table 1</a>). Therefore, we first used Sendai viruses to introduce the human OSKM gene combination into bat PCs in the presence or absence of NANOG, which was expressed through a transposon. Although the genes were delivered and expressed efficiently (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Fig. 1b</a>), no long-term morphological changes were observed up to 100 days post-infection (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplementary Fig. 1a</a>). Delivering the same gene combination using inducible transposon vectors did not provide a better result, although an increase in cellular proliferation was observed, along with transient and partial morphological changes (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Supplementary Fig. 1c, d</a>).</p><p id="p-12">Next, we screened additional transcription factors for their ability to induce somatic reprogramming of bat PCs through delivery of doxycycline-inducible transposons and identified a novel gene combination (ESRRB, CDX2, and c-MYC) as highly efficient at inducing marked and stable morphological changes in transduced bat PCs. When cells were cultured in serum-free EPI medium, foci showing typical stem cell morphology appeared at around 25 days after addition of doxycycline (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1e</a>); however, a longer period (35 days) was required when cells were cultured in serum-containing ESM medium (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1f</a>). Stable proliferation of bat RSCs has continued for more than 110–140 generations, spanning 250–320 days (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1g</a>). The average doubling time in EPI medium was estimated to be around 24 h, and that in ESM medium around 35 h.</p></div><div id="sec-6" class="subsection"><h3>Bat RSCs show stem cell-like characteristics</h3><p id="p-13">Bat RSCs grew in compact colonies in both EPI and ESM medium, with a morphology typical of PSCs: small and round, with a large nucleus, a high nucleo-cytoplasm ratio, and a prominent nucleolus (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>). Electron microscopy confirmed that the morphology and ultrastructure of bat RSCs was very different from that of bat PCs. In particular, chromatin was distributed homogeneously throughout the nucleoplasm, without large zones of electron-dense heterochromatin that are typical of differentiated cells, and regularly observed in the nuclei of bat PCs. The endoplasmic reticulum in bat RSCs was abundant, but not dilated as in bat PCs. (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a>). Cell cycle analysis of bat RSCs revealed a pluripotent stem cell-like profile with a short G2/M phase and a long S phase, unlike bat PCs. Expression of PSC-specific antigens, including stage-specific embryonic antigens (SSEA-1, SSEA-3, and SSEA-4) and an epithelial membrane antigen (EMA)-1<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, was then analyzed by immunostaining and flow cytometry (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2d</a>). Similar to murine ESCs (mESCs), but in contrast to human PSCs (hiPSCs), bat RSCs were positive for SSEA-1 and EMA-1 and negative for SSEA-3 and SSEA-4. The nuclear distribution of some epigenetic marks in RSCs was similar to the pattern observed in some PSCs (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2f</a>). Indeed, the facultative heterochromatin marker H3K27me3 in bat RSCs was distributed within several large foci, rather than being concentrated mainly at the inactivated X chromosome as in PCs. Previously, such numerous prominent H3K27me3 foci were observed in both naïve mouse PSCs33 and in pluripotent and reprogrammed avian cells<sup><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Taken together, these different expression profiles strongly suggest that following reprogramming, bat RSCs share key features with PSCs from other species.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of Pteropus bat reprogrammed stem cells. (a) Morphology of bat RSCs after 170 days of culture in EPI or ESM2 medium. Scale bar, 200 &#x3BC;m. (b) Ultrastructural analysis of reprogrammed cells by electron microscopy. Scale bar, 1 &#x3BC;m. (c) Cell cycle analysis using propidium iodide staining. Expression of markers of pluripotency expressed by bat PCs and bat RSCs, as analyzed by immunostaining (d) and flow cytometry (e) and compared with expression by murine embryonic stem cells (mESC) and human pluripotent stem cells (hiPSC). (f) Histone post-translational modifications in reprogrammed cell nuclei: immunodetection of H3K27me3 and H3K9me3, and DNA counterstaining with TO-PRO-3. Scale bar, 10 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Characterization of &lt;em&gt;Pteropus&lt;/em&gt; bat reprogrammed stem cells.&lt;/span&gt; (a) Morphology of bat RSCs after 170 days of culture in EPI or ESM2 medium. Scale bar, 200 &#x3BC;m. (b) Ultrastructural analysis of reprogrammed cells by electron microscopy. Scale bar, 1 &#x3BC;m. (c) Cell cycle analysis using propidium iodide staining. Expression of markers of pluripotency expressed by bat PCs and bat RSCs, as analyzed by immunostaining (d) and flow cytometry (e) and compared with expression by murine embryonic stem cells (mESC) and human pluripotent stem cells (hiPSC). (f) Histone post-translational modifications in reprogrammed cell nuclei: immunodetection of H3K27me3 and H3K9me3, and DNA counterstaining with TO-PRO-3. Scale bar, 10 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F2.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F2.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <span class="caption-title">Characterization of <em>Pteropus</em> bat reprogrammed stem cells.</span><p id="p-14" class="first-child">(a) Morphology of bat RSCs after 170 days of culture in EPI or ESM2 medium. Scale bar, 200 μm. (b) Ultrastructural analysis of reprogrammed cells by electron microscopy. Scale bar, 1 μm. (c) Cell cycle analysis using propidium iodide staining. Expression of markers of pluripotency expressed by bat PCs and bat RSCs, as analyzed by immunostaining (d) and flow cytometry (e) and compared with expression by murine embryonic stem cells (mESC) and human pluripotent stem cells (hiPSC). (f) Histone post-translational modifications in reprogrammed cell nuclei: immunodetection of H3K27me3 and H3K9me3, and DNA counterstaining with TO-PRO-3. Scale bar, 10 µm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Bat RSC have a specific neural stem cell molecular signature</h3><p id="p-15">To define the full transcriptomic landscape of generated bat RSCs, we performed deep sequencing of RNA isolated from primary and reprogrammed cells. Principal component analysis (PCA) of expressed genes indicated that bat RSCs cultured in either EPI or ESM medium had expression profiles clearly distinct from those of bat PCs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). Furthermore, 3506 genes were differentially expressed between bat PCs and bat RSCs, with a log -fold change &gt; 2 (LFC &gt; 2) and a p-value &gt; 0.05 (Supplementary Table 2). When cultured in either medium during reprograming, bat RSCs did not express key genes related to pluripotency, such as POU5F1/OCT4 and NANOG; noticeable exceptions were SOX2 and ZSCAN4, which play roles in telomere maintenance and long-term-genomic stability in ES cells<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. This suggests that the reprogramming factors introduced into the cells induced a new cell type. String analysis on the first 823 differentially expressed genes (DEGs) with a LFC &gt; 5 between bat PCs and bat RSCs revealed the presence of several major gene clusters (Supplementary Table 3). Further GO analysis of these 823 DEGs indicated that the first three GO terms for biological processes were “nervous system development” (GO:0007399), “generation of neurons” (GO:0048699) and “neurogenesis” (GO:0022008) (Supplementary Table 4). The most important GO terms for molecular function were “inorganic molecular entity transmembrane transporter activity” (GO:0015318), “ion transmembrane transporter activity” (GO:0015075), and “metal ion transmembrane transporter activity” (GO:0046873). The cellular component GO terms included “plasma membrane part” (GO:0044459), “plasma membrane” (GO:0005886), and “cell periphery” (GO:0071944) (Supplementary Table 4). Additional analysis of the main gene clusters revealed that the most important cluster contained 27 members (Cluster 27, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b</a>) and was centered on GRIA1 and GRIK2; this cluster harbored numerous specific neural-associated receptors for neurotransmitters (e.g., GABA and 5-Hydroxytryptamine (serotonin)), illustrating the neural cell-like nature of RSCs. The second most important cluster centered on SOX2 and contained several transcription factors, some of which are specific to PSCs: POU3F3, NKX2-5, GATA4, ZIC2 and ZIC5, CDX2, MYOD1, HNF4A, PRDM14, PIWIL2, TBR1, FOXA1 and FOXA3, PITX3, LIN28B, ELAVL2, and ZSCAN4 (Cluster 23a, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3c</a>). Other clusters contained neural markers and processes, ion channels and membrane bound transporters, and some components participating in the G protein-coupled receptor signaling pathway (Supplementary Table 3). These expression profiles suggest that the reprogramming process gave rise to a new neural-related stem cell type that differs from the initial PCs and shares features with PSCs described in other species. Interestingly, ephrin B2 (EFNB2), a receptor for <em>Henipavirus</em> entry, was among the main DEGs and was expressed at much higher levels in bat RSCs than in bat PCs (Supplementary Table 2). Such an expression profile suggests the possibility of <em>Henipavirus</em> entry into these cells. We tested this in the following experiments.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular features of Pteropus bat reprogrammed stem cells. (a) Principal component analysis (PCA) of RNA-seq results for bat PCs and bat RSCs. Principal Component 1 (PC1, x-axis) represents 91% and PC2 (y-axis) represents 4% of the total variation in the data. (b) Protein network analysis of the first main cluster of 27 genes identified by String analysis of the DEGs identified by RNAseq analysis and centered on GRIA1 and GRIK2 genes (c) Protein network analysis of the second main cluster of 23 genes, centered on the SOX2 gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Molecular features of &lt;em&gt;Pteropus&lt;/em&gt; bat reprogrammed stem cells.&lt;/span&gt; (a) Principal component analysis (PCA) of RNA-seq results for bat PCs and bat RSCs. Principal Component 1 (PC1, x-axis) represents 91% and PC2 (y-axis) represents 4% of the total variation in the data. (b) Protein network analysis of the first main cluster of 27 genes identified by String analysis of the DEGs identified by RNAseq analysis and centered on GRIA1 and GRIK2 genes (c) Protein network analysis of the second main cluster of 23 genes, centered on the SOX2 gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F3.medium.gif" width="162" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F3.medium.gif" width="162" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <span class="caption-title">Molecular features of <em>Pteropus</em> bat reprogrammed stem cells.</span><p id="p-16" class="first-child">(a) Principal component analysis (PCA) of RNA-seq results for bat PCs and bat RSCs. Principal Component 1 (PC1, x-axis) represents 91% and PC2 (y-axis) represents 4% of the total variation in the data. (b) Protein network analysis of the first main cluster of 27 genes identified by String analysis of the DEGs identified by RNAseq analysis and centered on GRIA1 and GRIK2 genes (c) Protein network analysis of the second main cluster of 23 genes, centered on the SOX2 gene.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Bat RSCs are highly permissive for Henipavirus infection</h3><p id="p-17">As <em>Pteropus</em> bats are the recognized natural reservoir of Henipaviruses, we tested the susceptibility of bat PCs and bat RSCs to infection by two human NiV isolates (NiV-Malaysia (NiV-M) and NiV-Bangladesh (NiV-B)), one <em>Pteropus</em> bat NiV isolate (NiV Cambodia (NiV-C)), and one equine HeV isolate. Viral entry into cells was quantified using recombinant vesicular stomatitis virus (VSV), which lacks envelope glycoprotein G, replaced by red fluorescent protein (rVSVΔG-RFP), pseudotyped with surface glycoproteins G and F from each Henipavirus isolate. Bat PCs, bat RSCs, and Vero cells were infected with pseudotyped rVSVΔG-RFP viruses at MOI of 0.01 PFU/cell and the percentage of infected cells was analyzed by flow cytometry 6 h later (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>). In contrast to bat PCs, bat RSCs were susceptible to infection with all four tested viruses. Henipaviruses entered bat RSCs much more readily than bat PCs and, interestingly, more readily than Vero cells, which are commonly used to isolate and propagate <em>Henipavirus</em>. Next, we examined the transcription and replication kinetics of NiV by RT-qPCR and viral titration after infection of cells with virus at a low MOI (0.1 PFU/cell) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b, c</a>). In bat RSCs, NiV RNA synthesis and production of viral particles increased by 4 log10 units between 0 h and 24 h post-infection; no increase was observed in bat PCs and all of the tested strains replicated much better in bat RSC than in bat PSC. Cytopathic effects and formation of giant multinucleated cells are hallmarks of NiV infection; both were readily visible in bat RSCs at 24 h post-infection (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4d</a>) and developed further during the course of infection. These results demonstrate that unlike bat PCs, bat RSCs are highly permissive for <em>Henipavirus</em> infection, thereby suggesting novel opportunities for isolation of new bat viruses.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Susceptibility of Pteropus bat reprogrammed cells to Henipaviruses. (a) Entry of VSV pseudotyped with Henipavirus glycoproteins into bat PC, bat RSC, and Vero cells (MOI = 0.01 PFU/mL). Data were analyzed using One-Way ANOVA, followed by Tukey&#x2019;s multiple comparisons test (**** P &lt; 0.0001; ** P &lt; 0.01; and * P &lt; 0.05). (b&#x2013;d) Bat PCs and bat RSCs were infected with the indicated viruses at a MOI of 0.1 PFU/mL. RT-qPCR analysis of Nipah virus infection kinetics (b) and release of virions into the supernatant (c). Bars represent the means &#xB1; SDs of three independent experiments. (d) Cell cytopathic effects observed under a light microscope at 24 h post-infection. Arrowheads show syncytia. Scale bar, 25 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Susceptibility of &lt;em&gt;Pteropus&lt;/em&gt; bat reprogrammed cells to Henipaviruses.&lt;/span&gt; (a) Entry of VSV pseudotyped with Henipavirus glycoproteins into bat PC, bat RSC, and Vero cells (MOI = 0.01 PFU/mL). Data were analyzed using One-Way ANOVA, followed by Tukey&#x2019;s multiple comparisons test (**** P &lt; 0.0001; ** P &lt; 0.01; and * P &lt; 0.05). (b&#x2013;d) Bat PCs and bat RSCs were infected with the indicated viruses at a MOI of 0.1 PFU/mL. RT-qPCR analysis of Nipah virus infection kinetics (b) and release of virions into the supernatant (c). Bars represent the means &#xB1; SDs of three independent experiments. (d) Cell cytopathic effects observed under a light microscope at 24 h post-infection. Arrowheads show syncytia. Scale bar, 25 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F4.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F4.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <span class="caption-title">Susceptibility of <em>Pteropus</em> bat reprogrammed cells to Henipaviruses.</span><p id="p-18" class="first-child">(a) Entry of VSV pseudotyped with Henipavirus glycoproteins into bat PC, bat RSC, and Vero cells (MOI = 0.01 PFU/mL). Data were analyzed using One-Way ANOVA, followed by Tukey’s multiple comparisons test (**** P &lt; 0.0001; ** P &lt; 0.01; and * P &lt; 0.05). (b–d) Bat PCs and bat RSCs were infected with the indicated viruses at a MOI of 0.1 PFU/mL. RT-qPCR analysis of Nipah virus infection kinetics (b) and release of virions into the supernatant (c). Bars represent the means ± SDs of three independent experiments. (d) Cell cytopathic effects observed under a light microscope at 24 h post-infection. Arrowheads show syncytia. Scale bar, 25 µm.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Innate immune responses of bat RSCs</h3><p id="p-19">As innate immunity is the first line of defense against viral infection, we next analyzed innate responses of bat RSC and compared them with those of bat PCs. First, we performed a comparative analysis of ISG expression in non-stimulated bat PCs and RSCs at the global transcriptomic level. Taking advantage of a consensus list of ISGs<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>, we identified differential expression of 174 ISGs in the two cells types (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5a</a>). Analysis identified two main sets of genes, one expressed at higher levels in bat RSCs (14 genes with a LFC &gt; −2) and another expressed at higher levels in bat PCs (83 genes with a LFC &gt; 2) (Supplementary Table 5). The second set contained a much higher number of genes, suggesting that reprogrammed cells expressed fewer ISGs and that these cells had weaker antiviral capacity. In particular, the interferon-regulated factor 1 (IRF1) gene network was down-regulated in bat RSCs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5b</a>, Supplementary Table 6). This specific intrinsic ISG signature may be one of the mechanisms underlying higher susceptibility of bat RSCs to infection by <em>Henipavirus</em>. Further functional approaches will help to identify which specific genes among those differentially expressed are implicated in bat antiviral responses.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bat RSCs exhibit an original ISG signature (a) Heatmap showing the ISG signature in bat PCs and bat RSCs, illustrating the presence of two main set of genes: one is up-regulated in bat PCs and one (a larger set) is down-regulated in bat RSCs. (b) Cluster of ISGs centered on IRF1 and IRF2 following String analysis of ISGs down-regulated in bat RSCs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Bat RSCs exhibit an original ISG signature&lt;/span&gt; (a) Heatmap showing the ISG signature in bat PCs and bat RSCs, illustrating the presence of two main set of genes: one is up-regulated in bat PCs and one (a larger set) is down-regulated in bat RSCs. (b) Cluster of ISGs centered on IRF1 and IRF2 following String analysis of ISGs down-regulated in bat RSCs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F5.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F5.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5</span> <span class="caption-title">Bat RSCs exhibit an original ISG signature</span><p id="p-20" class="first-child">(a) Heatmap showing the ISG signature in bat PCs and bat RSCs, illustrating the presence of two main set of genes: one is up-regulated in bat PCs and one (a larger set) is down-regulated in bat RSCs. (b) Cluster of ISGs centered on IRF1 and IRF2 following String analysis of ISGs down-regulated in bat RSCs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Bat RSC express IFN-I and ISGs following viral infection and respond to IFN-I stimulation</h3><p id="p-21">Finally, we compared innate immune response of bat RSCs and bat PCs in response to IFN-I. Initially, we analyzed the effects of polyI:C, a synthetic analogue of dsRNA, which binds and activates TLR3 when directly added to the cell culture<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> and is used widely to mimic viral infection<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. Only bat PCs produced both IFNα and IFNβ, as well as MX1, ISG56, and PKR, in response to increasing concentrations of polyI:C; no increase in ISGs expression was observed in stimulated bat RSCs (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6a</a>). Next, we analyzed the effect of infection with a negative strand RNA virus, VSV, which stimulates the TLR3, RIG-I, and MDA5 pathways (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6b</a>). In contrast to stimulation with polyI:C, VSV induced similar responses in both bat RSCs and bat PCs: VSV activated expression of IFNα, IFNβ and ISG56 expression, but not that of MX1 or PKR. Finally, stimulation of bat RSCs and bat PCs with IFN-I, known to activate the intracellular JAK-STAT pathway, induced similar responses in both cell types, characterized by IFN dose-dependent expression of MX1, ISG56, and PKR mRNA (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6c</a>). These results suggest that bat RSCs produce and respond to IFN-I, similar to bat PCs; however, some intracellular signaling pathways, especially those related to the TLR3 pathway, seem to be activated differently.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Innate immune responses of Pteropus bat cells. (a) Dose-dependent induction of bat IFN&#x3B1;, IFN&#x3B2;, Mx1, ISG56, and PKR transcripts in bat PCs and RSCs upon treatment with polyI:C, (b) VSV infection (MOI = 1 PFU/mL) and (c) IFN-u. Cells were stimulated 6 h with either polyI:C or IFN-&#x3B3;, or infected with VSV for 6 or 24 h. Bars represent the means &#xB1; SDs of three independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Innate immune responses of &lt;em&gt;Pteropu&lt;/em&gt;s bat cells.&lt;/span&gt; (a) Dose-dependent induction of bat IFN&#x3B1;, IFN&#x3B2;, Mx1, ISG56, and PKR transcripts in bat PCs and RSCs upon treatment with polyI:C, (b) VSV infection (MOI = 1 PFU/mL) and (c) IFN-u. Cells were stimulated 6 h with either polyI:C or IFN-&#x3B3;, or infected with VSV for 6 or 24 h. Bars represent the means &#xB1; SDs of three independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F6.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F6.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6</span> <span class="caption-title">Innate immune responses of <em>Pteropu</em>s bat cells.</span><p id="p-22" class="first-child">(a) Dose-dependent induction of bat IFNα, IFNβ, Mx1, ISG56, and PKR transcripts in bat PCs and RSCs upon treatment with polyI:C, (b) VSV infection (MOI = 1 PFU/mL) and (c) IFN-u. Cells were stimulated 6 h with either polyI:C or IFN-γ, or infected with VSV for 6 or 24 h. Bars represent the means ± SDs of three independent experiments.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-11"><h2 class="">Discussion</h2><p id="p-23">Efforts to combat deadly zoonotic infections caused by new emerging viruses are limited by lack of knowledge about reservoir host biology and the absence of adequate <em>in vitro</em> models for studying host-pathogen interactions<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup>. Isolation of new viral strains is an important step and is (generally) attempted in established immortalized cell lines such as IFN-I-deficient simian Vero cells. Propagation of viruses in cell lines other than those of the natural host may lead to selection of virus sub-populations that replicate more efficiently in the chosen cell type, leading to accumulation of adaptive mutations that may prevent adequate assessment of host-pathogen interactions<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>. Therefore, further advancement in the field requires establishment of new bat cell models; indeed, development of novel <em>Pteropus</em> cell lines is particularly important for the field of <em>Henipavirus</em> infection research. The high permissiveness of bat RSCs reported herein (even higher than that of Vero cells routinely used for virus isolation and production) provides an important new tool for further studies. This cell model should bridge an existing gap in this field, facilitate optimization of viral discovery in bats<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, and allow further research into bat biology at the cellular and molecular level.</p><p id="p-24">In recent years, somatic reprogramming has become a powerful method for generating PSCs in species from which embryos or tissues are difficult to obtain. Using the original combination of ESSRB, CDX2 and c-MYC transcription factors, we obtained bat RPCs that exhibit stem cell-like properties. Unexpectedly, the OSKM reprograming factors OCT4, SOX2, KLF4, and c-MYC, which are used successfully in other mammalian species<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup>, were inefficient in <em>Pteropus</em> cells, either alone or in combination with NANOG. This underlies the potential distinctiveness of bat cells with respect to this approach. Future somatic reprograming in other bat species will show whether the approach reported herein for <em>Pteropus</em> bat cells can be applied to other species. The reprogramming process presented in this report generated cells with rather unique stem cell characteristics. In addition to expression of PSC-specific antigens and epigenetic markers, these reprogramed bat cells express an original transcriptomic signature comprising numerous neural stem cell markers, as evidenced by GO terms and gene cluster analyses. As neural cells are one of the principal targets of Henipaviruses, this cell profile may favor infection, as observed in our study. In addition, as is currently the case for human stem cells<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>, further differentiation of reprogrammed bat stem cells into neural cell types relevant for viral infection may open new avenues for modeling of viral infectious diseases.</p><p id="p-25">Recent studies of innate immune responses in human and murine stem cells reveal that mammalian ESCs and other types of PSCs do not have a functional IFN-based antiviral mechanism<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>–<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>; this is in contrast to avian ESCs that elicit antiviral responses similar to those of differentiated cells<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. Attenuated innate immune responses by pluripotent human cells correlates with high expression of suppressor of cytokine signaling 1 (SOCS1)<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. However, comparative analysis of RNAseq data in our study revealed low differential expression of SOCS1 in RSCs compared with that in PCs (LFC of −2.6), highlighting an additional distinction between bat and other mammalian cells. Absence of IFN-I expression in murine embryos is attributed to developmental regulation of IRF1 and IRF2 expression, which becomes functional after cell differentiation and is capable of regulating IFN-1 production<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. Similarly, expression of IRF1 and IRF2 is lower in bat RSCs that in bat PCs (Supplementary Table 2), suggesting that another mechanism regulates IFN-I production by bat stem cells. Furthermore, previously observed constitutive expression of IRF7 in bats is thought to allow them to respond more rapidly to infection<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. Although we did not find similar expression of IRF7 in our bat PCs and RSCs, we did (surprisingly) identify increased expression of IRF6 (Supplementary Table 2). While IRF6 is the only member of IRF family not linked to IFN control in mammals, it acts as a positive regulator of IFN expression in fish<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> and inhibits production of the proinflammatory cytokine IL-1β by human cells<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>, suggesting that it may be involved in controlling innate immune responses and inflammation in bats; however, this needs to be further tested in functional studies.</p><p id="p-26">Previous genome analysis of <em>Pteropus alecto</em> and <em>Myotis davidi</em> bats suggests that positive selection of components of the DNA damage checkpoint pathway is associated with changes in the overlap between this pathway and the innate immune system, indicating that evolutionary adaptations important for flight may have secondary effects on bat immunity<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>. Analysis of ISG expression in immortalized <em>P. alecto</em> bat cell lines reveals strong evolutionary conservation, although unstimulated bat cells express ISGs at higher level than their humans counterparts<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. The profile of ISGs expressed in our unstimulated primary and reprogrammed bat cells is different from that expressed by immortalized cell lines, possibly due to absence of immortalization-activated programs, which may have affected ISG expression by bat cell lines in the previous study. Important differences in innate immune signatures between undifferentiated stem cells and their differentiated progeny have been described for several mammalian species; these differences lead to high resistance of stem cells to viral infection<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup>. While we also observed clear differences between primary and reprogrammed bat cells (i.e., lower ISG expression in the latter), these unique cells are capable of both producing IFN-I and replicating viruses, including Henipaviruses; this is in sharp contrast to many other mammalian stem cells, but interestingly, similar to observations in avian stem cells<sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>.</p><p id="p-27">Bats possess peculiarities in terms of immune responses<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>, and our reprogrammed bat cells seemed to follow the same pattern. One of the first lines of defense against viral pathogens is the IFN response; indeed, contraction of the IFN-I locus and constitutive expression of IFNα was observed in <em>P. alecto</em> bats<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. Interestingly, we did not find constitutive expression of IFN-I by bat stem cells or PC lines, although expression of IFNα and IFNβ was inducible in both cell types. In agreement with our results, a previous study did not observe constitutive expression of IFNα in cell lines generated from <em>R. aegyptus</em> bats<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. Taken together, our data suggest that certain differences between in bat immune responses and those of other mammals are present at the stem cell level and may have been established during evolution, thereby shaping development of bat antiviral responses. Early exposure of pregnant bats to viral infections circulating in bat colonies may lead to development of disease tolerance in their progeny<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> and the absence of a strong immune response to viral proteins; this may constitute a part of the antiviral bat defense strategy<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>. As suggested in a recent study of the <em>Rousettus</em> bat genome<sup><a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a></sup>, immune tolerance rather than a heightened antiviral defense strategy may support fitness, despite replication of pathogens, thereby allowing bats to efficiently control infection by viruses that are highly pathogenic to other mammalian species. In conclusion, we obtained reprogrammed bat cells presenting stem cell-like features and high permissiveness for <em>Henipavirus</em> infection. The particular innate immune signature of these cells underlines the distinctiveness of bat species and improves our understanding of the mechanisms underlying bat antiviral responses.</p></div><div class="section" id="sec-12"><h2 class="">Materials and methods</h2><div id="sec-13" class="subsection"><h3><em>Pteropus</em> bat primary cell culture</h3><p id="p-28">Cell cultures of <em>Pteropus</em> bat flying fox (<em>Pteropus giganteus</em>, also known as <em>P. medius</em>) and <em>P. vampyrus</em> (<em>Yinpterochiroptera</em> suborder) were generated from samples collected in Tiergarten Schönbrunn (Vienna, Austria). Lung and trachea samples were obtained a female specimen found naturally dead, and wing-membrane skin biopsies were obtained during the regular veterinary checkup in accordance with national guidelines to minimize stress. Samples were washed with sterile PBS and transferred to Cryo-SFM freezing medium (PromoCell Bioscience, C-29910) on dry ice for shipment from the zoo. To obtain bat primary cell cultures (bat PCs), samples were dissected in a Petri dish and explants derived from the trachea (PTCs), lung (PLCs), and alary membrane (PACs) were cultured at 37°C/7% CO2 in parallel in gelatin-coated wells containing either fibroblast medium (FM) (DMEM/F12 supplemented with 10% fetal bovine serum (FBS), 2 mM L-glutamine, 1000 U/mL penicillin, 1000 U/mL streptomycin) or ESM1 medium (DMEM/F12 supplemented with 10% FBS, 2 mM L-glutamine, 1000 U/mL penicillin, 1000 U/ mL streptomycin, 1% non-essential amino acids (100X), 1 mM sodium pyruvate, 0.1 mM β-mercaptoethanol, 1 ng/mL IL-6, 1 ng/mL IL-6 receptor, 1 ng/mL Mouse Stem Cell Factor, 5 ng/mL insulin-like growth factor-1, and 1000 U/mL leukemia inhibitory factor); this culture protocol is used routinely for avian stem cells<sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>. References to all reagents used for cell culture are listed in Supplementary Table 7.</p></div><div id="sec-14" class="subsection"><h3>Reprogramming vectors</h3><p id="p-29">CDNAs encoding human POU5F1, SOX2, KLF4, NANOG, c-MYC, and CDX2, and mouse ESRRB were cloned, sequenced, and inserted into an inducible pPB transposon backbone. All constructs were generated using the NEBuilder® HiFi DNA Assembly Master Mix system (New England BioLabs, E2621) and sequenced to validate correct cDNA insertion. Viral stocks of the Sendai viral vector CytoTune2.0 kit, expressing the POU5F1/OCT4, SOX2, KLF4, and c-MYC human genes, were purchased from Invitrogen (A16517).</p></div><div id="sec-15" class="subsection"><h3>Generation of reprogrammed <em>Pteropus</em> bat cells</h3><p id="p-30">Bat PCs were seeded in a 6-well plate and infected with viruses containing KLF4-OCT4-SOX2 and c-MYC (5 PFU/cell), and with a virus containing KLF4 (3 PFU/cell), according to the protocol provided by the virus supplier (CytoTune2.0). Cells were passaged 5 days after infection and seeded in two 55-cm<sup>2</sup> flasks. The cells were cultured for 6 days in ESM1 or EPI medium (the medium was changed every 2 days). The EPI medium is serum-free and comprises 50% DMEM/F12 and 50% Neurobasal medium, supplemented with B-27 Supplement, N-2 Supplement, 2 mM L-Glutamine, 1000 U/mL penicillin, 1000 U/mL streptomycin, 1 mM β-mercaptoethanol, 5 ng/mL basic-Fibroblast Growth Factor (b-FGF), and 10 ng/mL human Activin A. In a second approach, bat PCs were modified by electroporation with different combinations of inducible transposons encoding OCT4, SOX2, KLF4, c-MYC, NANOG, ESRBB, and CDX2. Bat PC cells were dissociated and centrifuged at 1200 rpm (300 g) for 5 min at ambient temperature. The cell pellet was rinsed in PBS, centrifuged again, and 1.2×10<sup>6</sup> cells recovered in 120 μL of R resuspension buffer (Neon, Life Technologies) containing 2 μg of pCAGPBase transposase-expressing vector and 4 μg of the reprogramming gene cocktail<sup><a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. 100-μL of this cell-plasmid solution was electroporated (1500 V, 30 ms, 1 pulse) in a 100 μL tip (Neon, Life Technologies, MPK10096) using the Neon system (Life Technologies, MPK5000). After electroporation, the cells were cultured in 3 mL of FM medium in a 6-well plate. The medium from electroporated cells was replaced after 24 h and cells were selected using 5 μg/mL of puromycin and 200 μg/mL of neomycin, depending on the resistance genes carried by the plasmids present in the mixture. The medium containing selection reagents was changed every 2 days for at least 1 week. At the end of the selection process (between 8 and 15 days), the cells were dissociated by 0.05% trypsin-EDTA (Life) and 2×10<sup>5</sup> cells were seeded into a well in a 6-well plate containing 3 mL of ESM1, EPI, or ESM2 medium supplemented with 2 µg/mL of doxycycline. The ESM2 medium comprised DMEM/F12 supplemented with 10% FBS, 2 mM L-glutamine, 1000 U/mL penicillin, 1000 U/ mL streptomycin, 1% non-essential amino acids (100X), 1 mM sodium pyruvate, 0.1 mM β-mercaptoethanol, 5 ng/mL b-FGF, and 10 ng/mL human Activin A. References for all reagents used for cell culture are listed in Supplementary Table 8.</p></div><div id="sec-16" class="subsection"><h3>Characterization of <em>Pteropus</em> bat reprogrammed cells</h3><p id="p-31">Cells were examined by electron microscopy as described<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup>. Exponentially growing cells were subjected to cell cycle analysis as described previously<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Reactivity of cells with antibodies specific for SSEA-1, SSEA-3, SSEA-4, or EMA-1 was tested as previously described<sup><a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a></sup>. The nuclear distribution of histone methylation marks was analyzed by immunofluorescence microscopy as described previously<sup><a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a></sup>. FastQ files were generated by RNAseq sequencing (Eurofins Genomics; <a href="https://www.eurofinsgenomics.eu/">https://www.eurofinsgenomics.eu/</a>) of bat PTCs as a starting material for reprogramming, along with two reprogrammed cell populations, E2 and ES2, which were established by culture in EPI and ESM2 medium, respectively. For analysis, a reference index was created based on the <em>Pteropus vampyrus</em> annotation GCF_000151845.1_Pvam_2.0_rna.fna.gz from the NCBI database, which was updated on January 20, 2018. FastQ files were pre-processed using fastp v0.19.6 and then pseudoaligned using kallisto v0.8.1, with the following parameters: kallisto quant -i pVam2.0 --bias -b 100 -l -s -t 8<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>. DEGs were calculated from abundance.h5 files using standard parameters within DESeq2 v1.24.0<sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>. An ISGs table<sup><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></sup> was crossed with PTCvES DSGs, with an adjusted p-value lower &lt; 0.05 using common gene symbols to generate an ISG heatmap. Raw data and gene tables are available through the GSE134585 datasets. Clustering and GO annotations were performed by String (<a href="https://string-db.org/">https://string-db.org/</a>) and Gene Ontology (<a href="http://geneontology.org/">http://geneontology.org/</a>), using the human Gene symbols as references.</p></div><div id="sec-17" class="subsection"><h3>Pseudotyped virus infection assay</h3><p id="p-32">Henipavirus pseudotyped particles were generated from rVSV-ΔG-RFP, a recombinant VSV in which the G protein envelope has been replaced with RFP (red fluorescent protein), as previously described (Gaudino, submitted). Briefly, the attachment glycoprotein and fusion proteins of NiV/Malaysia, NiV/Bangladesh, NiV/Cambodia, and HeV were cloned from RNA isolated from each virus and inserted into a pCAGGS plasmid vector. For each viral isolate, plasmid vectors coding the glycoproteins were transfected into BSR-T7 cells using the TransIT®-LT1 Transfection Reagent (Mirus Bio, MIR 2300). At 16 h post-transfection, cells were infected with rVSV-ΔG-RFP (0.3 PFU/cell) to produce a pseudotype VSV for each isolate. Supernatants were collected 24 h post-infection and concentrated by ultracentrifugation (28,000 rpm for 2 h at 4°C). Virus stocks were titrated on Vero cells. Pseudotyped viruses were named after the virus providing the surface glycoprotein. To evaluate viral entry into different bat cells, cells cultured in 24-well plates (80% confluent and adherent) were infected at an MOI of 0.01 PFU/cell for 1 h at 37°C. Then, virus-containing medium was removed and cells were washed once with PBS. Finally, fresh medium was added to the cells for 6 h at 37°C. The percentage of infected cells was assessed by measuring RFP fluorescence by flow cytometry (BD LSR Fortessa, BD Biosciences).</p></div><div id="sec-18" class="subsection"><h3>Henipavirus infection and titration</h3><p id="p-33">NiV-Malaysia virus (isolate UMMC1; GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY029767&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom">AY029767</a>) , NiV-Bangladesh virus (isolate SPB200401066, GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY988601&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom">AY988601</a>) , and NiV Cambodia virus (isolate NiV/KHM/CSUR381, GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK801755&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom">MK801755</a>) were generated and titrated on Vero E6 cells at the INSERM Jean Mérieux BSL-4 laboratory in Lyon, France. For each virus, bat cells were plated in 12-well plates and upon reaching 80% of confluence were infected for 1 h at 37°C with virus at 0.1 PFU/cell. Next, virus-containing media were removed and cells were washed once with PBS. Finally, fresh medium was added to cells, followed by incubation at 37°C for 0, 6, 16, 24, or 48 h. Cell morphology was observed under a Zeiss Axiovert 100M microscope and photos were taken at 24 h post-infection using ImageJ software. For each time point, infected cell lysates were prepared using RLT buffer (Qiagen, 79216) prior to RT-qPCR analysis according to a validated BSL-4 procedure. Supernatants were collected and kept at −80°C until titration in a plaque assay on Vero E6 cells. Infections and titrations were performed in a BSL-4 facility at Jean Mérieux (Lyon, France).</p></div><div id="sec-19" class="subsection"><h3>Induction of innate immune responses</h3><p id="p-34">Bat cells were stimulated with a synthetic analogue of dsRNA, polyinosinic-polycytidylic acid, polyI:C (Invitrogen), VSV Indiana strain, or universal IFN-type (a recombinant human IFN-alpha protein hybrid; Pbl Assay Science, 11200-2). Bat cells seeded in 12-well plates at 80% confluence were treated for 6 h at 37°C with either poly IC (0–1000 ng/mL) or universal IFN-type I (0–1000 U/mL). Alternatively, cells were infected with VSV (MOI = 1) for 1 h at 37°C. After viral attachment, cells were washed once with PBS and fresh medium was added. Cells were incubated for 0, 6, or 24 h at 37°C. Infected cell lysates were prepared for RT-qPCR analysis.</p></div><div id="sec-20" class="subsection"><h3>RNA extraction and RT-qPCR analysis</h3><p id="p-35">At the indicated time points, lysates prepared from infected and stimulated cells were collected and RNA extracted using a NucleoSpin RNA kit (Macherey-Nagel, 740955). The yield and purity of the extracted RNA was assessed using a spectrophotometer (DS-11 FX; Denovix). Extracted mRNA was reverse-transcribed using SuperScript™ III Reverse Transcriptase (Invitrogen, 18080). Real time PCR was performed using Platinum SYBR green qPCR SuperMix-UDG kit with ROX reference dye (Thermo Fisher Scientific, 11733038) and a StepOne plus PCR system (Applied Biosystems). Data were analyzed using StepOne v2.3 software (Applied Biosystems) and calculations were done using the 2<sup>ΔΔCT</sup> method. Expression was normalized to that of glyceraldehyde 3-phosphate dehydrogenase (GAPDH), as described previously<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. The primers used are listed in Supplementary Table 8.</p></div><div id="sec-21" class="subsection"><h3>Statistical analysis</h3><p id="p-36">All analyses were performed using GraphPad Prism 8 (GraphPad Software). Data were analyzed using analysis of variance two-way ANOVA, followed by Bonferroni or Tukey’s multiple comparisons test. P values &lt; 0.05 were deemed significant.</p></div></div><div class="section" id="sec-23"><h2 class="">Funding</h2><p id="p-38">The study was supported by LABEX ECOFECT (ANR-11-LABX-0048) of Lyon University, within the program “Investissements d’Avenir” (ANR-11-IDEX-0007), operated by the French National Research Agency (ANR), and by ANR-18-CE11-0014-02 from the Finovi and Aviesan Sino-French agreement on Nipah virus study.</p></div><div class="section" id="sec-24"><h2 class="">Supplementary data</h2><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Attempts to obtain Pteropus bat reprogrammed cells using various delivery systems. (a) The classical OSKM combination was introduced into bat primary cells (PCs) using the non-integrative Sendai virus and cells were observed under a light microscope (scale bar, 200 &#x3BC;m). (b) PCR-based detection of Sendai virus was performed as recommended by the supplier: 1, bat PCs; 2, bat PCs + Sendai; 3, bat PCs + Sendai + NANOG; 4, Control reprogrammed stem cells. (c) Bat PCs were modified by electroporation of inducible transposons encoding OCT4, SOX2, KLF4, and c-MYC. Scale bar, 200 &#x3BC;m. (d) Comparative growth curves of bat PCs modified by electroporation of OSKM and OSKMN." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678651199" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Attempts to obtain &lt;em&gt;Pteropus&lt;/em&gt; bat reprogrammed cells using various delivery systems.&lt;/span&gt; (a) The classical OSKM combination was introduced into bat primary cells (PCs) using the non-integrative Sendai virus and cells were observed under a light microscope (scale bar, 200 &#x3BC;m). (b) PCR-based detection of Sendai virus was performed as recommended by the supplier: 1, bat PCs; 2, bat PCs + Sendai; 3, bat PCs + Sendai + NANOG; 4, Control reprogrammed stem cells. (c) Bat PCs were modified by electroporation of inducible transposons encoding OCT4, SOX2, KLF4, and c-MYC. Scale bar, 200 &#x3BC;m. (d) Comparative growth curves of bat PCs modified by electroporation of OSKM and OSKMN.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F7.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F7.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/30/846410/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1</span> <span class="caption-title">Attempts to obtain <em>Pteropus</em> bat reprogrammed cells using various delivery systems.</span><p id="p-39" class="first-child">(a) The classical OSKM combination was introduced into bat primary cells (PCs) using the non-integrative Sendai virus and cells were observed under a light microscope (scale bar, 200 μm). (b) PCR-based detection of Sendai virus was performed as recommended by the supplier: 1, bat PCs; 2, bat PCs + Sendai; 3, bat PCs + Sendai + NANOG; 4, Control reprogrammed stem cells. (c) Bat PCs were modified by electroporation of inducible transposons encoding OCT4, SOX2, KLF4, and c-MYC. Scale bar, 200 μm. (d) Comparative growth curves of bat PCs modified by electroporation of OSKM and OSKMN.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40"><strong>Table S1 |</strong> Pluripotent-associated genes identified in humans and <em>P. vamp</em>yrus bats.</p><p id="p-41"><strong>Figure 2e |</strong> <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2e</a> higher resolution of FACS analysis</p><p id="p-42"><strong>Table S2</strong> | List of genes expressed by bat PCs (PTCs) and reprogrammed cells cultured in either EPI ECM_E2) or ESM (ECM_ES2) medium after RNAseq analysis</p><p id="p-43"><strong>Table S3 |</strong>List of gene clusters identified after String analysis.</p><p id="p-44"><strong>Table S4 |</strong> Functional enrichment of DEGs identified by RNAseq analysis.</p><p id="p-45"><strong>Table S5 |</strong> List of ISG genes expressed by either bat PC or RSCs.</p><p id="p-46"><strong>Table S6 |</strong> List of ISG genes clustered after String analysis.</p><p id="p-47"><strong>Table S7 |</strong> List of reagents used for cell culture.</p><p id="p-48"><strong>Table S8 |</strong> List of primers used in the study.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1018137/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1018137/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1018137" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1</span> <span class="caption-title">Comparison of protein sequences of pluripotent genes between human and <em>Pteropus vampyrus</em> bat</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-37">The authors thank Drs. Doris Preininger, Anton Weissenbacher, and Tiergarten Schönbrunn (Vienna, Austria) for <em>Pteropus</em> samples. We also thank Drs F. Enchery, K. Dhondt, J. Fouret, and C. Legras-Lachuer for help in initiating and realizing this work.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.1" data-doi="10.1038/nature06536"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">K. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global trends in emerging infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>, <span class="cit-fpage">990</span>–<span class="cit-lpage">993</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D990%26rft.epage%253D993%26rft.atitle%253DGlobal%2Btrends%2Bin%2Bemerging%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06536%26rft_id%253Dinfo%253Apmid%252F18288193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.2" data-doi="10.4269/ajtmh.2011.10-0567"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halpin</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Pteropid Bats are Confirmed as the Reservoir Hosts of Henipaviruses: A Comprehensive Experimental Study of Virus Transmission</span>. <abbr class="cit-jnl-abbrev">Am. J. Trop. Med. Hyg</abbr>. <span class="cit-vol">85</span>, <span class="cit-fpage">946</span>–<span class="cit-lpage">951</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BJournal%2Bof%2BTropical%2BMedicine%2Band%2BHygiene%26rft.stitle%253DAm%2BJ%2BTrop%2BMed%2BHyg%26rft.aulast%253DHalpin%26rft.auinit1%253DK.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D946%26rft.epage%253D951%26rft.atitle%253DPteropid%2BBats%2Bare%2BConfirmed%2Bas%2Bthe%2BReservoir%2BHosts%2Bof%2BHenipaviruses%253A%2BA%2BComprehensive%2BExperimental%2BStudy%2Bof%2BVirus%2BTransmission%26rft_id%253Dinfo%253Adoi%252F10.4269%252Fajtmh.2011.10-0567%26rft_id%253Dinfo%253Apmid%252F22049055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoidHJvcG1lZCI7czo1OiJyZXNpZCI7czo4OiI4NS81Lzk0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzMwLzg0NjQxMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.3" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats Are Natural Reservoirs of SARS-Like Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>, <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mohd</span>,  <span class="cit-name-given-names">H. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Memish</span>,  <span class="cit-name-given-names">Z. A.</span></span> <span class="cit-article-title">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir</span>. <abbr class="cit-jnl-abbrev">Virol. J</abbr>. <span class="cit-vol">13</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.5" data-doi="10.1016/j.coviro.2018.12.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.-F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-article-title">Viruses in bats and potential spillover to animals and humans</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Virol</abbr>. <span class="cit-vol">34</span>, <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BVirol%26rft.volume%253D34%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2018.12.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2018.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.6" data-doi="10.1038/438575a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>,  <span class="cit-name-given-names">E. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Fruit bats as reservoirs of Ebola virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>, <span class="cit-fpage">575</span>–<span class="cit-lpage">576</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLeroy%26rft.auinit1%253DE.%2BM.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DFruit%2Bbats%2Bas%2Breservoirs%2Bof%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F438575a%26rft_id%253Dinfo%253Apmid%252F16319873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/438575a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.7" data-doi="10.1053/jcpa.2001.0532"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Middleton</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Experimental Nipah Virus Infection in Pigs and Cats</span>. <abbr class="cit-jnl-abbrev">J. Comp. Pathol</abbr>. <span class="cit-vol">126</span>, <span class="cit-fpage">124</span>–<span class="cit-lpage">136</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomparative%2Bpathology%26rft.stitle%253DJ%2BComp%2BPathol%26rft.aulast%253DMiddleton%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D126%26rft.issue%253D2-3%26rft.spage%253D124%26rft.epage%253D136%26rft.atitle%253DExperimental%2BNipah%2Bvirus%2Binfection%2Bin%2Bpigs%2Band%2Bcats.%26rft_id%253Dinfo%253Adoi%252F10.1053%252Fjcpa.2001.0532%26rft_id%253Dinfo%253Apmid%252F11945001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1053/jcpa.2001.0532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11945001&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175392600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.8" data-doi="10.3201/eid2102.141433"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ching</span>,  <span class="cit-name-given-names">P. K. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Outbreak of Henipavirus Infection, Philippines, 2014</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">328</span>–<span class="cit-lpage">331</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg.%2BInfect.%2BDis%26rft.volume%253D21%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2102.141433%26rft_id%253Dinfo%253Apmid%252F25626011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2102.141433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25626011&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.9" data-doi="10.3201/eid1212.060732"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luby</span>,  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Foodborne Transmission of Nipah Virus, Bangladesh</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">1888</span>–<span class="cit-lpage">1894</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DLuby%26rft.auinit1%253DS.%2BP.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D1888%26rft.epage%253D1894%26rft.atitle%253DFoodborne%2Btransmission%2Bof%2BNipah%2Bvirus%252C%2BBangladesh.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1212.060732%26rft_id%253Dinfo%253Apmid%252F17326940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1212.060732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17326940&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242301900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nikolay</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Transmission of Nipah Virus – 14 Years of Investigations in Bangladesh</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-vol">380</span>, <span class="cit-fpage">1804</span>–<span class="cit-lpage">1814</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft.volume%253D380%26rft.spage%253D1804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Horvat</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Henipavirus pathogenesis and antiviral approaches</span>. <abbr class="cit-jnl-abbrev">Expert Rev. Anti Infect. Ther</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">343</span>–<span class="cit-lpage">354</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BRev.%2BAnti%2BInfect.%2BTher%26rft.volume%253D13%26rft.spage%253D343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.12" data-doi="10.1038/ncomms1796"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drexler</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats host major mammalian paramyxoviruses</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">796</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DDrexler%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D3%26rft.spage%253D796%26rft.epage%253D796%26rft.atitle%253DBats%2Bhost%2Bmajor%2Bmammalian%2Bparamyxoviruses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1796%26rft_id%253Dinfo%253Apmid%252F22531181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22531181&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.13" data-doi="10.1016/j.antiviral.2013.07.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luby</span>,  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-article-title">The pandemic potential of Nipah virus</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>. <span class="cit-vol">100</span>, <span class="cit-fpage">38</span>–<span class="cit-lpage">43</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D100%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.07.011%26rft_id%253Dinfo%253Apmid%252F23911335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23911335&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328179300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.14" data-doi="10.1644/06-MAMM-R-397R.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Solari</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Mammal Species of the World: A Taxonomic and Geographic Reference by D. E. Wilson; D. M. Reeder</span>. <abbr class="cit-jnl-abbrev">J. Mammal</abbr>. <span class="cit-vol">88</span>, <span class="cit-fpage">824</span>–<span class="cit-lpage">830</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMammal%26rft_id%253Dinfo%253Adoi%252F10.1644%252F06-MAMM-R-397R.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1644/06-MAMM-R-397R.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van der Poel</span>,  <span class="cit-name-given-names">W. H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lina</span>,  <span class="cit-name-given-names">P. H. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kramps</span>,  <span class="cit-name-given-names">J. A.</span></span> <span class="cit-article-title">Public health awareness of emerging zoonotic viruses of bats: a European perspective</span>. <abbr class="cit-jnl-abbrev">Vector Borne Zoonotic Dis. Larchmt. N</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVector%2BBorne%2BZoonotic%2BDis.%2BLarchmt.%2BN%26rft.volume%253D6%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.16" data-doi="10.1146/annurev-animal-022516-022811"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teeling</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bat Biology, Genomes, and the Bat1K Project: To Generate Chromosome-Level Genomes for All Living Bat Species</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">46</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci%26rft.volume%253D6%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-022516-022811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-022516-022811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Austad</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-article-title">Comparative biology of aging</span>. <abbr class="cit-jnl-abbrev">J. Gerontol. A. Biol. Sci. Med. Sci</abbr>. <span class="cit-vol">64</span>, <span class="cit-fpage">199</span>–<span class="cit-lpage">201</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGerontol.%2BA.%2BBiol.%2BSci.%2BMed.%2BSci%26rft.volume%253D64%26rft.spage%253D199%26rft_id%253Dinfo%253Apmid%252F19223603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19223603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264190400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.18" data-doi="10.1126/science.1105113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teeling</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Molecular Phylogeny for Bats Illuminates Biogeography and the Fossil Record</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>, <span class="cit-fpage">580</span>–<span class="cit-lpage">584</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTeeling%26rft.auinit1%253DE.%2BC.%26rft.volume%253D307%26rft.issue%253D5709%26rft.spage%253D580%26rft.epage%253D584%26rft.atitle%253DA%2BMolecular%2BPhylogeny%2Bfor%2BBats%2BIlluminates%2BBiogeography%2Band%2Bthe%2BFossil%2BRecord%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105113%26rft_id%253Dinfo%253Apmid%252F15681385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwOS81ODAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schountz</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Immunological Control of Viral Infections in Bats and the Emergence of Viruses Highly Pathogenic to Humans</span>. <abbr class="cit-jnl-abbrev">Front. Immunol</abbr>. <span class="cit-vol">8</span>, (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.20" data-doi="10.1038/nature22975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>, <span class="cit-fpage">646</span>–<span class="cit-lpage">650</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28636590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luis</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A comparison of bats and rodents as reservoirs of zoonotic viruses: are bats special?</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">280</span>, (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enchéry</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Horvat</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Understanding the interaction between henipaviruses and their natural host, fruit bats: Paving the way toward control of highly lethal infection in humans</span>. <abbr class="cit-jnl-abbrev">Int. Rev. Immunol</abbr>. <span class="cit-vol">36</span>, <span class="cit-fpage">108</span>–<span class="cit-lpage">121</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BRev.%2BImmunol%26rft.volume%253D36%26rft.spage%253D108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Misra</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schountz</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-article-title">Tools to study pathogen-host interactions in bats</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-vol">248</span>, <span class="cit-fpage">5</span>–<span class="cit-lpage">12</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D248%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hare</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuddington</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mossman</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">The Importance of Physiologically Relevant Cell Lines for Studying Virus–Host Interactions</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">8</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.25" data-doi="10.1371/journal.pone.0008266"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crameri</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Establishment, immortalisation and characterisation of pteropid bat cell lines</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e8266</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrameri%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8266%26rft.epage%253De8266%26rft.atitle%253DEstablishment%252C%2Bimmortalisation%2Band%2Bcharacterisation%2Bof%2Bpteropid%2Bbat%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008266%26rft_id%253Dinfo%253Apmid%252F20011515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20011515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.26" data-doi="10.1016/j.cell.2006.07.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yamanaka</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Induction of Pluripotent Stem Cells from Mouse Embryonic and Adult Fibroblast Cultures by Defined Factors</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>, <span class="cit-fpage">663</span>–<span class="cit-lpage">676</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTakahashi%26rft.auinit1%253DK.%26rft.volume%253D126%26rft.issue%253D4%26rft.spage%253D663%26rft.epage%253D676%26rft.atitle%253DInduction%2Bof%2Bpluripotent%2Bstem%2Bcells%2Bfrom%2Bmouse%2Bembryonic%2Band%2Badult%2Bfibroblast%2Bcultures%2Bby%2Bdefined%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.07.024%26rft_id%253Dinfo%253Apmid%252F16904174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.07.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16904174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240276700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ogorevc</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orehek</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dovč</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Cellular reprogramming in farm animals: an overview of iPSC generation in the mammalian farm animal species</span>. <abbr class="cit-jnl-abbrev">J. Anim. Sci. Biotechnol</abbr>. <span class="cit-vol">7</span>, (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-846410v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ezashi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-article-title">Pluripotent Stem Cells from Domesticated Mammals</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci</abbr>. <span class="cit-vol">4</span>, <span class="cit-fpage">223</span>–<span class="cit-lpage">253</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci%26rft.volume%253D4%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stanton</span>,  <span class="cit-name-given-names">M. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Prospects for the Use of Induced Pluripotent Stem Cells in Animal Conservation and Environmental Protection</span>. <abbr class="cit-jnl-abbrev">STEM CELLS Transl. Med</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">7</span>–<span class="cit-lpage">13</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSTEM%2BCELLS%2BTransl.%2BMed%26rft.volume%253D8%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mo</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Generation and characterization of bat-induced pluripotent stem cells</span>. <abbr class="cit-jnl-abbrev">Theriogenology</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">283</span>–<span class="cit-lpage">293</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheriogenology%26rft.volume%253D82%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fuet</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">NANOG Is Required for the Long-Term Establishment of Avian Somatic Reprogrammed Cells</span>. <abbr class="cit-jnl-abbrev">Stem Cell Rep</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">1272</span>–<span class="cit-lpage">1286</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BRep%26rft.volume%253D11%26rft.spage%253D1272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trusler</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodwin</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Laslett</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-article-title">Cell surface markers for the identification and study of human naive pluripotent stem cells</span>. <abbr class="cit-jnl-abbrev">Stem Cell Res</abbr>. <span class="cit-vol">26</span>, <span class="cit-fpage">36</span>–<span class="cit-lpage">43</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BRes%26rft.volume%253D26%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-846410v1.33" data-doi="10.1038/s41598-018-23822-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tosolini</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Contrasting epigenetic states of heterochromatin in the different types of mouse pluripotent stem cells</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">5776</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D8%26rft.spage%253D5776%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-23822-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-23822-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kress</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montillet</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jean</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuet</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pain</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Chicken embryonic stem cells and primordial germ cells display different heterochromatic histone marks than their mammalian counterparts</span>. <abbr class="cit-jnl-abbrev">Epigenetics Chromatin</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">5</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%2BChromatin%26rft.volume%253D9%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.35" data-doi="10.1038/nature08882"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zalzman</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Zscan4 regulates telomere elongation and genomic stability in ES cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>, <span class="cit-fpage">858</span>–<span class="cit-lpage">863</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZalzman%26rft.auinit1%253DM.%26rft.volume%253D464%26rft.issue%253D7290%26rft.spage%253D858%26rft.epage%253D863%26rft.atitle%253DZscan4%2Bregulates%2Btelomere%2Belongation%2Band%2Bgenomic%2Bstability%2Bin%2BES%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08882%26rft_id%253Dinfo%253Apmid%252F20336070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08882&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336070&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276397300030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.36" data-doi="10.1016/j.cell.2017.11.018."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Intrinsic Immunity Shapes Viral Resistance of Stem Cells</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">172</span>, <span class="cit-fpage">423</span>–<span class="cit-lpage">438.e25</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D172%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.11.018.%26rft_id%253Dinfo%253Apmid%252F29249360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.11.018.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29249360&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.37" data-doi="10.1128/CMR.00022-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vercammen</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Staal</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Beyaert</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Sensing of Viral Infection and Activation of Innate Immunity by Toll-Like Receptor 3</span>. <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">13</span>–<span class="cit-lpage">25</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DVercammen%26rft.auinit1%253DE.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D25%26rft.atitle%253DSensing%2Bof%2BViral%2BInfection%2Band%2BActivation%2Bof%2BInnate%2BImmunity%2Bby%2BToll-Like%2BReceptor%2B3%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00022-07%26rft_id%253Dinfo%253Apmid%252F18202435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjc6IjIxLzEvMTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.38" data-doi="10.1152/ajpregu.00293.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fortier</span>,  <span class="cit-name-given-names">M.-E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The viral mimic, polyinosinic:polycytidylic acid, induces fever in rats via an interleukin-1-dependent mechanism</span>. <abbr class="cit-jnl-abbrev">Am. J. Physiol. Regul. Integr. Comp. Physiol</abbr>. <span class="cit-vol">287</span>, <span class="cit-fpage">R759</span>–<span class="cit-lpage">766</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPhysiol.%2BRegul.%2BIntegr.%2BComp.%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajpregu.00293.2004%26rft_id%253Dinfo%253Apmid%252F15205185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajpregu.00293.2004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15205185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223884500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">C. C. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">Optimizing Viral Discovery in Bats</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e0149237</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D11%26rft.spage%253De0149237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.40" data-doi="10.3390/v7072800"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trevisan</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Modeling Viral Infectious Diseases and Development of Antiviral Therapies Using Human Induced Pluripotent Stem Cell-Derived Systems</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">3835</span>–<span class="cit-lpage">3856</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D3835%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7072800%26rft_id%253Dinfo%253Apmid%252F26184286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v7072800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26184286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.-L.</span></span> <span class="cit-article-title">Utilization of Different Antiviral Mechanisms by Mammalian Embryonic Stem Cells and Differentiated Cells</span>. <abbr class="cit-jnl-abbrev">Immunol. Cell Biol</abbr>. <span class="cit-vol">95</span>, <span class="cit-fpage">17</span>–<span class="cit-lpage">23</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunol.%2BCell%2BBiol%26rft.volume%253D95%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-846410v1.42" data-doi="10.4161/cc.9.17.12792"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Carmichael</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">Molecular basis for an attenuated cytoplasmic dsRNA response in human embryonic stem cells</span>. <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">3552</span>–<span class="cit-lpage">3564</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DChen%26rft.auinit1%253DL.%2BL.%26rft.volume%253D9%26rft.issue%253D17%26rft.spage%253D3552%26rft.epage%253D3564%26rft.atitle%253DMolecular%2Bbasis%2Bfor%2Ban%2Battenuated%2Bcytoplasmic%2BdsRNA%2Bresponse%2Bin%2Bhuman%2Bembryonic%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.9.17.12792%26rft_id%253Dinfo%253Apmid%252F20814227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.9.17.12792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20814227&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.43" data-doi="10.1089/scd.2013.0417"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mouse Embryonic Stem Cells Have Underdeveloped Antiviral Mechanisms That Can Be Exploited for the Development of mRNA-Mediated Gene Expression Strategy</span>. <abbr class="cit-jnl-abbrev">Stem Cells Dev</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">594</span>–<span class="cit-lpage">604</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCells%2BDev%26rft.volume%253D23%26rft.spage%253D594%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fscd.2013.0417%26rft_id%253Dinfo%253Apmid%252F24219369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/scd.2013.0417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24219369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giotis</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montillet</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pain</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Skinner</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-article-title">Chicken Embryonic-Stem Cells Are Permissive to Poxvirus Recombinant Vaccine Vectors</span>. <abbr class="cit-jnl-abbrev">Genes</abbr> <span class="cit-vol">10</span>, (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.45" data-doi="10.1074/jbc.M112.435461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">X.-X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Carmichael</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">Innate Immunity in Pluripotent Human Cells</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">288</span>, <span class="cit-fpage">16196</span>–<span class="cit-lpage">16205</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.435461%26rft_id%253Dinfo%253Apmid%252F23599426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMjIvMTYxOTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.46" data-doi="10.1016/0092-8674(90)90163-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harada</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Absence of the type I IFN system in EC cells: transcriptional activator (IRF-1) and repressor (IRF-2) genes are developmentally regulated</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">303</span>–<span class="cit-lpage">312</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHarada%26rft.auinit1%253DH.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D303%26rft.epage%253D312%26rft.atitle%253DAbsence%2Bof%2Bthe%2Btype%2BI%2BIFN%2Bsystem%2Bin%2BEC%2Bcells%253A%2Btranscriptional%2Bactivator%2B%2528IRF-1%2529%2Band%2Brepressor%2B%2528IRF-2%2529%2Bgenes%2Bare%2Bdevelopmentally%2Bregulated.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252890%252990163-9%26rft_id%253Dinfo%253Apmid%252F2208287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(90)90163-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2208287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED64200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.47" data-doi="10.1371/journal.pone.0103875"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">IRF7 in the Australian black flying fox, Pteropus alecto: evidence for a unique expression pattern and functional conservation</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e103875</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D9%26rft.spage%253De103875%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0103875%26rft_id%253Dinfo%253Apmid%252F25100081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0103875&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25100081&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.48" data-doi="10.1016/j.fsi.2016.08.059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">L.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.-X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">D.-D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-A.</span></span> <span class="cit-article-title">Fish IRF6 is a positive regulator of IFN expression and involved in both of the MyD88 and TBK1 pathways</span>. <abbr class="cit-jnl-abbrev">Fish Shellfish Immunol</abbr>. <span class="cit-vol">57</span>, <span class="cit-fpage">262</span>–<span class="cit-lpage">268</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish%2BShellfish%2BImmunol%26rft.volume%253D57%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsi.2016.08.059%26rft_id%253Dinfo%253Apmid%252F27577537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsi.2016.08.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27577537&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ainouze</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human papillomavirus type 16 antagonizes IRF6 regulation of IL-1β</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">14</span>, <span class="cit-fpage">e1007158</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.50" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>, <span class="cit-fpage">456</span>–<span class="cit-lpage">460</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De La Cruz-Rivera</span>,  <span class="cit-name-given-names">P. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The IFN Response in Bats Displays Distinctive IFN-Stimulated Gene Expression Kinetics with Atypical RNASEL Induction</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Baltim. Md 1950</abbr> <span class="cit-vol">200</span>, <span class="cit-fpage">209</span>–<span class="cit-lpage">217</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%2BBaltim.%2BMd%2B1950%26rft.volume%253D200%26rft.spage%253D209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.52" data-doi="10.1073/pnas.1518240113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Contraction of the type I IFN locus and unusual constitutive expression of IFN-α in bats</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">113</span>, <span class="cit-fpage">2696</span>–<span class="cit-lpage">2701</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518240113%26rft_id%253Dinfo%253Apmid%252F26903655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEwLzI2OTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.53" data-doi="10.1016/j.cell.2018.03.070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavlovich</span>,  <span class="cit-name-given-names">S. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">1098</span>–<span class="cit-lpage">1110.e18</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1098%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.54" data-doi="10.1126/science.1214935"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Medzhitov</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">D. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Soares</span>,  <span class="cit-name-given-names">M. P.</span></span> <span class="cit-article-title">Disease tolerance as a defense strategy</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>, <span class="cit-fpage">936</span>–<span class="cit-lpage">941</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMedzhitov%26rft.auinit1%253DR.%26rft.volume%253D335%26rft.issue%253D6071%26rft.spage%253D936%26rft.epage%253D941%26rft.atitle%253DDisease%2BTolerance%2Bas%2Ba%2BDefense%2BStrategy%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214935%26rft_id%253Dinfo%253Apmid%252F22363001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA3MS85MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8zMC84NDY0MTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.55" data-doi="10.3389/fimmu.2018.02112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mandl</span>,  <span class="cit-name-given-names">J. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">D. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-article-title">Going to Bat(s) for Studies of Disease Tolerance</span>. <abbr class="cit-jnl-abbrev">Front. Immunol</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">2112</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BImmunol%26rft.volume%253D9%26rft.spage%253D2112%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2018.02112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2018.02112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aubel</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pain</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Chicken embryonic stem cells: establishment and characterization</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol. Clifton NJ</abbr> <span class="cit-vol">1074</span>, <span class="cit-fpage">137</span>–<span class="cit-lpage">150</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%2BClifton%2BNJ%26rft.volume%253D1074%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fuet</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pain</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Chicken Induced Pluripotent Stem Cells: Establishment and Characterization</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol. Clifton NJ</abbr> <span class="cit-vol">1650</span>, <span class="cit-fpage">211</span>–<span class="cit-lpage">228</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%2BClifton%2BNJ%26rft.volume%253D1650%26rft.spage%253D211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.58" data-doi="10.1016/j.scr.2012.10.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coronado</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A short G1 phase is an intrinsic determinant of naïve embryonic stem cell pluripotency</span>. <abbr class="cit-jnl-abbrev">Stem Cell Res</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">118</span>–<span class="cit-lpage">131</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCell%2BRes%26rft.volume%253D10%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.scr.2012.10.004%26rft_id%253Dinfo%253Apmid%252F23178806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.scr.2012.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23178806&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314483000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.59" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bray</span>,  <span class="cit-name-given-names">N. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pimentel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melsted</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pachter</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Near-optimal probabilistic RNA-seq quantification</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">34</span>, <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.60" data-doi="10.1093/bioinformatics/bty560"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">fastp: an ultra-fast all-in-one FASTQ preprocessor</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">i884</span>–<span class="cit-lpage">i890</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di884%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty560%26rft_id%253Dinfo%253Apmid%252F30423086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30423086&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F30%2F846410.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">M. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anders</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-vol">15</span>, (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-846410v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Lethal Nipah Virus Infection Induces Rapid Overexpression of CXCL10</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>, (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">63.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-846410v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Type III IFN Receptor Expression and Functional Characterisation in the Pteropid Bat, Pteropus alecto</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>, (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/846410v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F846410v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/860536v1" title="An ALS-associated mutation in human FUS reduces neurotransmission from C. elegans motor neurons to muscles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 30, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/846410v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/846410v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/846410v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/846410v1" class="active" data-icon-position="" data-hide-link-title="0">Reprogrammed Pteropus Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1015730" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-55KIKSPCkZl9a3UbrAN8ni3pM5lvIhRhqzY0uFR3iAI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1015730" id="node1015730--2" data-pisa="biorxiv;846410v1" data-pisa-master="biorxiv;846410" data-seqnum="1015730" data-apath="/biorxiv/early/2019/11/30/846410.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Reprogrammed <em>Pteropus</em> Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Noémie</span> <span class="nlm-surname">Aurine</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Camille</span> <span class="nlm-surname">Baquerre</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Gaudino</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Jean</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Dumont</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sylvie</span> <span class="nlm-surname">Rival-Gervier</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Clémence</span> <span class="nlm-surname">Kress</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Branka</span> <span class="nlm-surname">Horvat</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Bertrand</span> <span class="nlm-surname">Pain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">846410; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/846410 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/846410v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;title=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;title=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;text=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;title=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;t=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;title=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;title=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1015730" id="node1015730--3" data-pisa="biorxiv;846410v1" data-pisa-master="biorxiv;846410" data-seqnum="1015730" data-apath="/biorxiv/early/2019/11/30/846410.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Reprogrammed <em>Pteropus</em> Bat Stem Cells Present Distinct Immune Signature and are Highly Permissive for Henipaviruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Noémie</span> <span class="nlm-surname">Aurine</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Camille</span> <span class="nlm-surname">Baquerre</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Gaudino</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Jean</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Claire</span> <span class="nlm-surname">Dumont</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sylvie</span> <span class="nlm-surname">Rival-Gervier</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Clémence</span> <span class="nlm-surname">Kress</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Branka</span> <span class="nlm-surname">Horvat</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Bertrand</span> <span class="nlm-surname">Pain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">846410; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/846410 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Aurine%2C+No%C3%A9mie&amp;rft.au=Baquerre%2C+Camille&amp;rft.au=Gaudino%2C+Maria&amp;rft.au=Jean%2C+Christian&amp;rft.au=Dumont%2C+Claire&amp;rft.au=Rival-Gervier%2C+Sylvie&amp;rft.au=Kress%2C+Cl%C3%A9mence&amp;rft.au=Horvat%2C+Branka&amp;rft.au=Pain%2C+Bertrand"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1015730/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1015730/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1015730/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1015730/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1015730/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1015730/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1015730/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1015730/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1015730/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1015730/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1015730/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1015730/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;count=horizontal&amp;via=&amp;text=Reprogrammed%20Pteropus%20Bat%20Stem%20Cells%20Present%20Distinct%20Immune%20Signature%20and%20are%20Highly%20Permissive%20for%20Henipaviruses&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/846410v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/846410v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/846410v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Cell Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
