<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6855448">6855448</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0224676">10.1371/journal.pone.0224676</a></span><span class="publisher-id" title="publisher-id">PONE-D-19-19136</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Tropical Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neglected Tropical Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Dengue Fever</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Dengue Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Databases</span>: <span class="subject" title="subject">Sequence Databases</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Databases</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span>: <span class="subject" title="subject">Sequence Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Disease Vectors</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Insect Vectors</span>: <span class="subject" title="subject">Mosquitoes</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Species Interactions</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Disease Vectors</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Insect Vectors</span>: <span class="subject" title="subject">Mosquitoes</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Invertebrates</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Arthropoda</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Insects</span>: <span class="subject" title="subject">Mosquitoes</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Asia</span>: <span class="subject" title="subject">China</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Molecular epidemiological characteristics of dengue virus carried by 34 patients in Guangzhou in 2018</div> <div class="alt-title" title="alt-title">
Molecular epidemiological characteristics of dengue virus carried by 34 patients in Guangzhou in 2018</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">Feng</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Huini</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Jieliang</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">Shaofeng</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Pan</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lao</span><span class="given-names" tagx="given-names" title="given-names">Zizhao</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Yaohua</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Lupin</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">Yanni</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Shuangfeng</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Jianguo</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Xiaohong</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Laboratory Animal Center, Guangzhou University of Chinese Medicine, Guangzhou, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Institute of Medical Microbiology, Jinan University, Guangzhou, China</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">School of Pharmaceutical Sciences, Guangzhou University of Chinese Medicine, Guangzhou, China</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">The First Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangzhou, China</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">College of Life Sciences, WuHan university, Wuhan, China</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Mathematical Engineering Academy of Chinese Medicine, Guangzhou University of Chinese Medicine, Guangzhou, China</span></span><span class="citation_author_institution" id="aff007">[7], <span class="addr-line" title="addr-line">Guangzhou eighth People's Hospital, Guangzhou Medical University, Guangzhou, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackard</span><span class="given-names" tagx="given-names" title="given-names">Jason</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Cincinnati College of Medicine, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">lg@gzucm.edu.cn</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-11-11</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0224676</span><span class="history" title="history"><span class="received" title="received">received: 2019-7-8</span><span class="accepted" title="accepted">accepted: 2019-10-19</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Liao et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0224676.pdf" title="self-uri">pone.0224676.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Dengue fever is a major worldwide public health problem that, as estimated by the WHO, causes epidemics in over 100 countries, resulting in hundreds of millions of dengue virus (DENV) infections every year. In China, dengue fever mainly occurs in coastal areas. Recurring dengue outbreaks were reported by Guangdong Province almost every year since the first epidemic in 1978. DENV infections persisted in Guangzhou in consecutive years since 2000, with the dengue epidemic reaching a historical peak in 2014. Because Guangzhou is one of the largest cities for opening up in China, understanding the epidemiological characteristics of dengue fever in the city can hopefully provide a significant basis for developing effective dengue prevention strategies. In this study, a total of 34 DENV strains, including 29 DENV-1 strains and 5 DENV-2 strains, were isolated from a blood samples drawn from patients who were diagnosed with dengue fever by hospitals in Guangzhou during 2018. To explore the epidemiological characteristics of dengue fever, the envelope (E) gene obtained from the isolates was amplified for phylogenetic analysis. The results from the phylogenetic analysis showed that DENV in Guangzhou was mainly imported from Southeast Asian countries. Additionally, propagation paths based on phylogeographical analysis suggested potential local dengue transmission in Guangzhou.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Nature Science Foundation of China</span></div><span class="award-id" tagx="award-id" title="award-id">81803813</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Science &amp;amp; Technology Planning Project of Guangzhou</span></div><span class="award-id" tagx="award-id" title="award-id">No.201804010029</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Science &amp;amp; Technology Planning Project of Guangdong Province Office of Education</span></div><span class="award-id" tagx="award-id" title="award-id">2014GKXM032</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Nature Science Foundation of China</span></div><span class="award-id" tagx="award-id" title="award-id">81973549</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Science &amp;amp; Technology Planning Project of Guangdong Province Office of Education</span></div><span class="award-id" tagx="award-id" title="award-id">2018KCXTD007</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Xiaohong</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Key Research Projects of GZUCM First-class universities and Top Disciplines</span></div><span class="award-id" tagx="award-id" title="award-id">A1-AFD018191A17</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Xiaohong</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">The Open Project of State Key Laboratory of Natural Medicines</span></div><span class="award-id" tagx="award-id" title="award-id">SKLNMKF201906</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2441-2427</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Geng</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported by 81803813, to Li G, from National Nature Science Foundation of China; 201804010029, to Li G, from Science &amp;amp; Technology Planning Project of Guangzhou; SKLNMKF201906, to Li G, from The Open Project of State Key Laboratory of Natural Medicines; A1-AFD018191A17, to Liu XH, from Key Research Projects of GZUCM First-class universities and Top Disciplines; 2018KCXTD007, to Liu XH, from Science &amp;amp; Technology Planning Project of Guangdong Province Office of Education; 2014GKXM032, to Lai XP, from Science &amp;amp; Technology Planning Project of Guangdong Province Office of Education; 81973549, to Li G, from National Nature Science Foundation of China.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Dengue fever, also known as "break-bone fever", is an acute mosquito-borne viral disease caused by dengue virus (DENV) [<a href="#pone.0224676.ref001">1</a>]. Patients infected with DENV can have flu-like symptoms, including a high fever, headache, and vomiting, which generally last for ten days. In a small proportion of cases, the disease develops into dengue hemorrhagic fever (DHF) or dengue shock syndrome (DSS) [<a href="#pone.0224676.ref002">2</a>, <a href="#pone.0224676.ref003">3</a>]. Dengue is principally spread by <i>Aedes aegypti</i> and <i>Aedes albopictus</i> mosquitoes in tropical and subtropical regions where the warm and humid climate indicates favorable mosquito habitat. In other words, dengue fever is also a seasonal infectious disease, and mosquito densities and climatic conditions are strongly associated with the incidence of dengue fever. In recent decades, dengue fever, as a growing threat to human health, has presented a tremendous challenge to health service providers throughout the world [<a href="#pone.0224676.ref004">4</a>]. According to a published study, an estimated 390 million dengue infections occur in over 100 countries every year [<a href="#pone.0224676.ref003">3</a>, <a href="#pone.0224676.ref005">5</a>]. Globally, the vast majority of the world population is exposed to DENV, making dengue one of the most dangerous vector-borne viral diseases worldwide [<a href="#pone.0224676.ref004">4</a>]. Dengue outbreaks have a devastating effect on public health and economic sectors [<a href="#pone.0224676.ref003">3</a>]. In the stricken areas, the disease leads to a slowdown in the local economy as it inflicts a significant health burden on the population, impairs people's quality of life and impedes the development of the tourist industry. With vector management as the primary means to control and prevent DENV transmission, it should be noted that improper use of insecticides is a waste of resources and a source of environmental pollution that threatens both human and environmental health [<a href="#pone.0224676.ref006">6</a>, <a href="#pone.0224676.ref007">7</a>]. So it is important to know the structure of dengue virus.</p> <p>DENV, as a member of the genus <i>Flavivirus</i> in the family <i>Flaviviridae</i>, is an enveloped, single-stranded, positive-sense RNA virus. The DENV genome is approximately 11,000 nucleotides in length and encodes three structural proteins, namely, the capsid (C), premembrane (prM), and envelope (E) proteins, and seven nonstructural (NS) proteins (NS1, NS2A, NS2B, NS3, NS4A, NS4B and NS5) in a single open reading frame (ORF). Specifically, the 5' end of the DENV genome encodes the C/prM/E proteins, while the 3' end encodes the seven NS protein[<a href="#pone.0224676.ref008">8</a>]. There are five DENV serotypes, which are referred to as DENV-1, DENV-2, DENV-3, and DENV-4, and the distinctions between these serotypes are based on their antigenicity. Additionally, different serotypes can be further differentiated into different genotypes[<a href="#pone.0224676.ref009">9</a>].</p> <p>In fact, as early as 1978, a case of imported dengue fever was discovered in Xiamen, after which the Chinese mainland has experienced multiple dengue epidemics in the past few decades [<a href="#pone.0224676.ref010">10</a>]. In addition to the coastal provinces in Southeast China and Taiwan that are mainly affected by the disease, an increasing geographic expansion to the inland is noted as the global climate changes [<a href="#pone.0224676.ref010">10</a>, <a href="#pone.0224676.ref011">11</a>]. Guangdong Province has been declared a severe epidemic area by the Chinese Center for Disease Control and Prevention, as it has been frequently attacked by dengue fever since the first outbreak [<a href="#pone.0224676.ref012">12</a>]. Despite the absence of reported dengue fever cases between 1982 and 1984, 1988 and 1989, and in 1994, 1996, and 1998, the disease has been a constant threat in Guangdong Province [<a href="#pone.0224676.ref013">13</a>]. In Guangzhou, the capital city of Guangdong Province, dengue incidence, regardless of the number of DENV infections, has been reported every year since 2000 [<a href="#pone.0224676.ref014">14</a>]. In 2014, Guangdong Province experienced its worst dengue outbreak on record, during which different DENV serotypes were detected in the DENV-infected patients [<a href="#pone.0224676.ref012">12</a>, <a href="#pone.0224676.ref015">15</a>]. There were over 46000 dengue fever cases notified nationwide throughout the year, including up to 45230 cases and 76 imported cases reported by Guangdong Province, which exceeds the cumulative number of infections between 1990 and 2013 [<a href="#pone.0224676.ref014">14</a>, <a href="#pone.0224676.ref016">16</a>].</p> <p>DENV infections are simply classified as imported and autochthonous cases according to patients' recent travel records. Dengue epidemics refer to continued widespread outbreaks of dengue fever in areas where DENV affects a large population and is transmitted between people by the mosquitoes <i>Aedes aegypti</i> and <i>Aedes albopictus</i>. To date, there are no other DENV foci in mainland China, except for the reported natural focus in Yunnan Province [<a href="#pone.0224676.ref017">17</a>]. As a result of rapid economic growth, thriving tourism, and the greenhouse effect, there were many patients infected with dengue fever in Guangdong Province, and a serious outbreak occurred in 2014. [<a href="#pone.0224676.ref018">18</a>, <a href="#pone.0224676.ref019">19</a>]. The endemicity of dengue in Guangzhou has been heatedly discussed [<a href="#pone.0224676.ref020">20</a>]. A previous study based on phylogenetic analysis suggested the presence of local dengue transmission in Guangzhou [<a href="#pone.0224676.ref010">10</a>]. In the present study, DENV strains were isolated from blood samples drawn from DENV-infected patients who were diagnosed with dengue fever by hospitals in Guangzhou during 2018. We described the epidemiological characteristics of isolated DENV based on phylogenetic reconstruction and analysis using a dataset comprised of DENV sequences downloaded from the GenBank database and those obtained from genome sequencing.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="1.samplecollection" title="sec"> <div class="title" tagx="title" title="title">
1. Sample collection</div> <p>In this study, all patient samples were approved by the First Affiliated Hospital of Guangzhou University of Chinese Medicine and Guangdong Hospital of Traditional Chinese Medicine. All patients had fever &amp;gt; 37.5°C for less than 72 h. Anti-dengue IgM and IgG enzyme-linked immunosorbent assay (ELISA) kits were used to confirm dengue infection [<a href="#pone.0224676.ref021">21</a>]. A total of 170 samples suspected of having dengue fever occurred during the 2018 outbreak. 55 blood samples were determined to be positive samples by hospital diagnosis.</p> <p>Oral informed consent was obtained from all patients who were involved in this study.</p> </div> <div class="2.virusisolation,rnaextractionandserotyping" title="sec"> <div class="title" tagx="title" title="title">
2. Virus isolation, RNA extraction and serotyping</div> <p>Dengue virus was isolated from 55 positive blood samples. To increase virus titer, each patient's serum was inoculated into an Aedes albopictus C6/36 cell line cultured in MEM with 10% FBS (Gibco, Carlsbad, CA, USA) and diluted 1:40 in fresh MEM. Next, the medium was added to C6/36 cells and incubated at 30°C for 2 h. Then, the residual inoculum was replaced with fresh MEM containing 2% FBS; the cells were maintained at 30°C in a humidified atmosphere of 5% CO<sub>2</sub> for 5 to 7 days (until significant cytopathic effects (CPE) occurred). Simultaneously, the supernatants were collected for RNA extraction after centrifugation at 4000 rpm. These samples were stored at -80°C until needed.</p> <p>Viral RNA was extracted from 200 μl of each supernatant using the QIAamp Viral RNA Mini Kit (Qiagen, Germany) as instructed by the manufacturer. Then, the RNA was reverse-transcribed into cDNA by reverse transcriptase at 37°C for 90 min and then at 70°C for 10 min. DENV serotyping was carried out by multiplex RT-PCR [<a href="#pone.0224676.ref022">22</a>].</p> </div> <div class="3.sequencing" title="sec"> <div class="title" tagx="title" title="title">
3. Sequencing</div> <p>Using RNA as templates, RT-PCR was performed to amplify the envelope (E) gene. DENV-1 primers were designed for sequencing based on the DENV-1 standard Hawaii strain (GenBank accession number: KM204119) and DENV-2 strain (GenBank accession number: KM279569) [<a href="#pone.0224676.ref010">10</a>, <a href="#pone.0224676.ref022">22</a>]. E gene sequences of 34 strains were obtained from the patient samples. The primers used in this article are listed in Tables <a href="#pone.0224676.t001">1</a> and <a href="#pone.0224676.t002">2</a>. These sequences were uploaded to the GenBank database, and their IDs, i.e., the GenBank accession numbers of the sequences, are listed in <a href="#pone.0224676.t003">Table 3</a>.</p> <div class="table-wrap_UNKNOWN" id="pone.0224676.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0224676.t001">10.1371/journal.pone.0224676.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Primer sequence, and size of RT-PCR product and generated combination of primers.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0224676.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Virus</th> <th align="left" rowspan="1" colspan="1">Primer</th> <th align="left" rowspan="1" colspan="1">Primer sequence (from 5′ to 3′)</th> <th align="left" rowspan="1" colspan="1">Size of amplicon &amp;amp;Primer combination</th> <th align="left" rowspan="1" colspan="1">Primer position</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">Dcon-F</td> <td align="left" rowspan="1" colspan="1"><tt>AGTTGTTAGTCTACGTGGACCGACA</tt></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">1–25</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV-1</td> <td align="left" rowspan="1" colspan="1">DENV-1-R</td> <td align="left" rowspan="1" colspan="1"><tt>CCCCGTAACACTTTGATCGCTCCATT</tt></td> <td align="left" rowspan="1" colspan="1">342 bp Dcon-F and DEN1-R</td> <td align="left" rowspan="1" colspan="1">317–342</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV-2</td> <td align="left" rowspan="1" colspan="1">DENV-2-R</td> <td align="left" rowspan="1" colspan="1"><tt>CGCCACAAGGGCCATGAACAG</tt></td> <td align="left" rowspan="1" colspan="1">251 bp Dcon-F and DEN2-R</td> <td align="left" rowspan="1" colspan="1">231–251</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><tt>DENV-3</tt></td> <td align="left" rowspan="1" colspan="1"><tt>DENV-3-R</tt></td> <td align="left" rowspan="1" colspan="1"><tt>GCACATGTTGATTCCAGAGGCTGTC</tt></td> <td align="left" rowspan="1" colspan="1">538 bp Dcon-F and DEN3-R</td> <td align="left" rowspan="1" colspan="1">514–538</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV-4</td> <td align="left" rowspan="1" colspan="1">DENV-4-R</td> <td align="left" rowspan="1" colspan="1"><tt>GTTTCCAATCCCATTCCTGAATGTGGTGT</tt></td> <td align="left" rowspan="1" colspan="1">754 bp Dcon-F and DEN4-R</td> <td align="left" rowspan="1" colspan="1">726–754</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN" id="pone.0224676.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0224676.t002">10.1371/journal.pone.0224676.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Primers used for amplifying and sequencing the complete DENV-1and DENV-2 envelope gene.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0224676.t002g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Reaction</th> <th align="left" rowspan="1" colspan="1">Primer name</th> <th align="left" rowspan="1" colspan="1">Sequence (from 5′ to 3′)</th> <th align="left" rowspan="1" colspan="1">Primer position</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="4" colspan="1">Amplification &amp;amp; Sequencing</td> <td align="left" rowspan="1" colspan="1">DENV1-E-F</td> <td align="left" rowspan="1" colspan="1"><tt>TGCCATAGGAACATCCATCAC</tt></td> <td align="left" rowspan="1" colspan="1">863–883</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV1-E-R</td> <td align="left" rowspan="1" colspan="1"><tt>TCCCAATGGCTGCTGATAGTC</tt></td> <td align="left" rowspan="1" colspan="1">2495–2462</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV2-E-F</td> <td align="left" rowspan="1" colspan="1"><tt>AATGGCAGCAATCTTGGCATACACC</tt></td> <td align="left" rowspan="1" colspan="1">747–771</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV2-E-R</td> <td align="left" rowspan="1" colspan="1"><tt>ACTGAGCGGATTCCACAAATGCCCT</tt></td> <td align="left" rowspan="1" colspan="1">2504–2480</td> </tr> </tbody> </table> </div> </div> <div class="table-wrap_UNKNOWN" id="pone.0224676.t003" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0224676.t003">10.1371/journal.pone.0224676.t003</a><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
The 2018 Guangzhou dengue virus sequences information isolated and uploaded in this study.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0224676.t003g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="center" rowspan="1" colspan="1">ID</th> <th align="center" rowspan="1" colspan="1">Accession</th> <th align="center" rowspan="1" colspan="1">Serotype</th> <th align="center" rowspan="1" colspan="1">Location</th> <th align="center" rowspan="1" colspan="1">Collection date</th> <th align="center" rowspan="1" colspan="1">Cluster</th> </tr> </thead> <tbody> <tr> <td align="center" rowspan="1" colspan="1">1–3</td> <td align="center" rowspan="1" colspan="1">MK517719</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/20</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">3–8</td> <td align="center" rowspan="1" colspan="1">MK517720</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/21</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">4–9</td> <td align="center" rowspan="1" colspan="1">MK517721</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/21</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">5–18</td> <td align="center" rowspan="1" colspan="1">MK517735</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/21</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">6–19</td> <td align="center" rowspan="1" colspan="1">MK517740</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/22</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">7–26</td> <td align="center" rowspan="1" colspan="1">MK517738</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/22</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">8–29</td> <td align="center" rowspan="1" colspan="1">MK517744</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/23</td> <td align="center" rowspan="1" colspan="1">Cluster III</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">9–32</td> <td align="center" rowspan="1" colspan="1">MK517739</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/24</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">1–37</td> <td align="center" rowspan="1" colspan="1">MK517743</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/24</td> <td align="center" rowspan="1" colspan="1">Cluster III</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">2–40</td> <td align="center" rowspan="1" colspan="1">MK517750</td> <td align="center" rowspan="1" colspan="1">DENV-2</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/24</td> <td align="center" rowspan="1" colspan="1">Cluster V</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">12–43</td> <td align="center" rowspan="1" colspan="1">MK517746</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/24</td> <td align="center" rowspan="1" colspan="1">Cluster III</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">3–44</td> <td align="center" rowspan="1" colspan="1">MK517752</td> <td align="center" rowspan="1" colspan="1">DENV-2</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/25</td> <td align="center" rowspan="1" colspan="1">Cluster V</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">12–45</td> <td align="center" rowspan="1" colspan="1">MK517747</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/25</td> <td align="center" rowspan="1" colspan="1">Cluster III</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">15–48</td> <td align="center" rowspan="1" colspan="1">MK517734</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/25</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">4–49</td> <td align="center" rowspan="1" colspan="1">MK517748</td> <td align="center" rowspan="1" colspan="1">DENV-2</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/25</td> <td align="center" rowspan="1" colspan="1">Cluster IV</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">17–59</td> <td align="center" rowspan="1" colspan="1">MK517722</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/26</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">18–61</td> <td align="center" rowspan="1" colspan="1">MK517723</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/26</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">19–64</td> <td align="center" rowspan="1" colspan="1">MK517725</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/26</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">20–66</td> <td align="center" rowspan="1" colspan="1">MK517727</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/26</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">21–69</td> <td align="center" rowspan="1" colspan="1">MK517730</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/26</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">22–81</td> <td align="center" rowspan="1" colspan="1">MK517741</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/28</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">23–84</td> <td align="center" rowspan="1" colspan="1">MK517729</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/28</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">24–89</td> <td align="center" rowspan="1" colspan="1">MK517726</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/28</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">25–93</td> <td align="center" rowspan="1" colspan="1">MK517724</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/28</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">26–94</td> <td align="center" rowspan="1" colspan="1">MK517731</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/9/28</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">1–97</td> <td align="center" rowspan="1" colspan="1">MK517733</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/1</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">27–107</td> <td align="center" rowspan="1" colspan="1">MK517742</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/3</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">28–108</td> <td align="center" rowspan="1" colspan="1">MK517745</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/3</td> <td align="center" rowspan="1" colspan="1">Cluster III</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">31–134</td> <td align="center" rowspan="1" colspan="1">MK517728</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/5</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">32–135</td> <td align="center" rowspan="1" colspan="1">MK517732</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/5</td> <td align="center" rowspan="1" colspan="1">Cluster I</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">5–141</td> <td align="center" rowspan="1" colspan="1">MK517751</td> <td align="center" rowspan="1" colspan="1">DENV-2</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/6</td> <td align="center" rowspan="1" colspan="1">Cluster V</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">6–146</td> <td align="center" rowspan="1" colspan="1">MK517749</td> <td align="center" rowspan="1" colspan="1">DENV-2</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/6</td> <td align="center" rowspan="1" colspan="1">Cluster IV</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">3–169</td> <td align="center" rowspan="1" colspan="1">MK517736</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/8</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">4–170</td> <td align="center" rowspan="1" colspan="1">MK517737</td> <td align="center" rowspan="1" colspan="1">DENV-1</td> <td align="center" rowspan="1" colspan="1">Guangzhou</td> <td align="center" rowspan="1" colspan="1">2018/10/8</td> <td align="center" rowspan="1" colspan="1">Cluster II</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>IDs in <a href="#pone.0224676.t001">Table 1</a> are registration numbers when collecting blood samples; the clustering is distinguished according to ML trees.</p> </div> </div> </div> </div> <div class="4.dataset" title="sec"> <div class="title" tagx="title" title="title">
4. Dataset</div> <p>In the dataset, the new isolates and the sequences of DENV-1 and DENV-2 available from the GenBank database were aligned and adjusted using MAFFT v7.308 [<a href="#pone.0224676.ref023">23</a>] and Aliview to remove those without a sampling date or geographic location (The length of the E gene sequence is 1485). Recombination Detection Program (RDP v4.36) was employed in recombination analysis with a wide range of recombination detection methods, including RDP, Chi-maera, BootScan, 3Seq, GENECONV, MaxChi, and SiScan [<a href="#pone.0224676.ref024">24</a>]. To avoid redundancy, CD-HIT-EST (<a href="http://weizhongli-lab.org/cdhit_suite/cgi-bin/index.cgi?cmd=cd-hit-est">http://weizhongli-lab.org/cdhit_suite/cgi-bin/index.cgi?cmd=cd-hit-est</a>) was used to group sequences from outside China into clusters, with the nucleotide identity threshold of 100% [<a href="#pone.0224676.ref025">25</a>]. A single sequence was randomly chosen from each cluster as a representative of all sequences that shared the same sampling date and geographic location.</p> </div> <div class="5.phylogeneticsignalassessmentandnucleotidesubstitutionmodelselection" title="sec"> <div class="title" tagx="title" title="title">
5. Phylogenetic signal assessment and nucleotide substitution model selection</div> <p>The accuracy of phylogenetic inference based on the nucleotide sequence dataset is subject to the saturation level of the phylogenetic signal. Substitution saturation is a crucial factor because a phylogenetic tree becomes meaningless if the aligned sequences in the dataset lose phylogenetic information due to substitution saturation. In the present study, the substitution saturation level of each dataset was assessed with DAMBE and Xia's test method [<a href="#pone.0224676.ref026">26</a>].</p> <p>Models of nucleotide substitution allow for the calculation of probabilities of change between nucleotides along the branches of a phylogenetic tree. On this basis, we used the Akaike information criterion (AIC) to select a best-fit model and created a maximum-likelihood tree (ML tree) with the model. Additionally, a maximum clade credibility (MCC) tree was generated using the Bayesian Information Criterion (BIC) for model selection. The best-fit models of nucleotide substitution were estimated with JModeltest v2.1.7 [<a href="#pone.0224676.ref027">27</a>]. Subsequently, the IQ-TREE [<a href="#pone.0224676.ref028">28</a>] was employed to determine the best-fit models since a massive quantity of sequences were deposited in a global dataset, which was beyond the capacity of JModeltest.</p> </div> <div class="6.phylogenetictreereconstruction" title="sec"> <div class="title" tagx="title" title="title">
6. Phylogenetic tree reconstruction</div> <p>In search of more information about the samples, we set up a global dataset that involved all samples and the DENV-1 and DENV-2 sequences available from the GenBank, except those being removed from the database as redundant sequences. Furthermore, ML trees were constructed with best-fit models in RAxML v8.0.9 [<a href="#pone.0224676.ref029">29</a>], while the reliability of topology was evaluated using bootstrap values derived from 1000 repetitions. The ML trees were visualized with FigTree v1.4.2.</p> </div> <div class="7.phylogeographicanalyses" title="sec"> <div class="title" tagx="title" title="title">
7. Phylogeographic analyses</div> <p>According to the ML trees obtained from the global dataset, the newly isolated sample sequences in the DENV-1 and DENV-2 datasets were divided into different clusters (with the classification of clusters based on bootstrap values and the number of 70% as a cut-off for a "reliable" branch). A sample sequence was selected from each cluster for nucleotide BLAST on the NCBI web server, and 100 DENV sequences were downloaded from each BLAST run. On this basis, the sample sequence of Cluster III was set to match 1000 sequences to gather more biological information. The downloaded sequences and the samples were combined together and divided into four datasets for comparison on the MAFFT web server. Phylogenetic signals of these datasets were detected as described above.</p> <p>Each dataset was analyzed with TempEst v1.5 to investigate its temporal signal and "clocklikeness" of ML phylogenies before applying the assumption of a molecular clock into the phylogenetic analysis [<a href="#pone.0224676.ref030">30</a>]. ML trees generated by RAxML were inputted, and the sample dates of the sequences were defined. After estimation of the best-fitting root, a linear regression was performed on the root-to-tip distances of samples versus the date of the isolate, and significant outliers (sequences) were rejected as they did not fit into the assumption of the molecular clock.</p> <p>The spatial diffusion of the time-scaled genealogy was modeled as a continuous-time Markov chain (CTMC) process with the program BEAST v1.8.3 [<a href="#pone.0224676.ref031">31</a>]. The diffusion process along the phylogenies of the datasets for BLAST was estimated using the Bayesian stochastic search variable selection (BSSVS) procedure. The uncorrelated lognormal relaxed clock (ULRC) method was adopted to produce phylogenetic estimates. In addition, we chose a Bayesian skyline coalescent model to review the demographic history and ran the Bayesian MCMC chain for a bulk number of iterations to ensure process convergence. The BEAGLE package was used to speed up the calculation process [<a href="#pone.0224676.ref032">32</a>]. The effective sample size (ESS) was calculated using TRACER [<a href="#pone.0224676.ref033">33</a>], with all parameters showing ESS values &amp;gt; 200 after burn-in in the initial 10% iterations. The MCC tree was summarized using TreeAnnotator v1.8.2 at the burn-in rate of 10% and was visualized with FigTree v1.4. SPREAD v1.0.6 is used to generate the propagation path map. Import the files generated by the BEAST software into the SPREAD software, set the latitude and longitude of the location, and set the BF value to 3, and then generate the corresponding path map.</p> </div> <div class="8.groupaveragedistancecalculation" title="sec"> <div class="title" tagx="title" title="title">
8. Group average distance calculation</div> <p>We differentiated the DENV-1 and DENV-2 data sets based on genotypes. In order to avoid the impact of old viruses on genetic distance calculation, we only extracted the genotypes of sequence samples from Guangdong Province in the past five years. The MEGA software was used to compare sample sequences of different clusters and compare them to sequences not in the cluster.</p> </div> </div> <div class="result" title="sec"> <div class="title" tagx="title" title="title">
Result</div> <div class="1.sampleinformationandserotypes" title="sec"> <div class="title" tagx="title" title="title">
1. Sample information and serotypes</div> <p>In 2018, a total of 170 blood samples were provided by the First Affiliated Hospital of Guangzhou University of Traditional Chinese Medicine, and 34 strains of dengue viruses were obtained from these blood samples, including 29 strains of DENV-1 and 5 strains of DENV-2. After gel electrophoresis analysis, the RT-PCR products were sequenced and uploaded to the GenBank database to obtain accession numbers. Details are shown in <a href="#pone.0224676.g001">Fig 1</a> and <a href="#pone.0224676.t003">Table 3</a>. The original image of agarose gel electrophoresis is shown in <a href="#pone.0224676.s001">S1 Fig</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Gel electrophoresis analysis of RT-PCR products.</div> <p>According to the results from gel electrophoresis analysis of different samples given in the figure, there are 34 RT-PCR products; the sizes of the amplicons (ID 2–40, 3–44, 4–49, 5–141, and 6–146) are consistent with DENV-2, while the rest are consistent with DENV-1.</p>   </div> </div> <div class="2.phylogeneticsignalassessment" title="sec"> <div class="title" tagx="title" title="title">
2. Phylogenetic signal assessment</div> <p>The substitution saturation level of each dataset of the DENV E gene was assessed with DAMBE. The results of the assessment indicated that no subset experienced full substitution saturation.</p> </div> <div class="3.phylogeneticanalysisandmolecularclocktests" title="sec"> <div class="title" tagx="title" title="title">
3. Phylogenetic analysis and molecular clock tests</div> <p>Obviously, the sample sequences were classified in independent clusters with high bootstrap values (70–100%), indicating robust support for the tree topology. DENV-1 was used to construct 3529 sequences of ML tree, and 3542 of DENV-2. From the ML tree of the global dataset, the sample sequences positive for DENV-1 were divided into three clusters, and the DENV-2 positive sample sequences were grouped into two clusters (for detailed clustering results, see <a href="#pone.0224676.g002">Fig 2</a>, <a href="#pone.0224676.g003">Fig 3</a> and <a href="#pone.0224676.t001">Table 1</a>). Molecular clock tests were performed on the basis of the ML trees. A stable temporal structure was observed as the linear regression on the root-to-tip distances versus the data suggested a relatively high fitting degree, and R<sup>2</sup> had a positive value, which enabled further study of the evolutionary process. The molecular clock detection maps are displayed on the corresponding MCC maps.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood tree for global dataset reconstruction of DENV-1.</div> <p>The maximum likelihood tree result of DENV-1 is shown in the figure. The sample sequence clusters under different branches, and the red blocks labeled in the branches in the figure are clusters of sample sequence clusters. Different color blocks outside the branches represent different genotypes.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood tree for global dataset reconstruction of DENV-2.</div> <p>The maximum likelihood tree result of DENV-2 is shown in the figure. The sample sequence clusters under different branches, and the blue blocks labeled in the branches in the figure are clusters of sample sequence clusters. Different color blocks outside the branches represent different genotypes.</p>   </div> </div> <div class="4.ancestralreconstructionanddiscretepaleogeographyanalysis" title="sec"> <div class="title" tagx="title" title="title">
4. Ancestral reconstruction and discrete paleogeography analysis</div> <p>All sample sequences were clearly clustered into one branch (with the classification of branches based on posterior probabilities and the number of 0.8 and higher as trustworthy). In this study, all sample sequences in the first, second, and third clusters belonged to DENV-1. According to the MCC tree reconstructed from Cluster I, the 17 isolates were clustered in one branch (posterior probability = 0.9985), which is the same as the Thailand and Taiwanese 2015 sequences (posterior probability = 0.9998). Our paleogeography analysis results showed that the sample virus originated in Taiwan. In 2015, the virus affected the region radically and was later imported into Guangzhou between 2015 and 2018. The MCC tree of Cluster II indicated that the sample sequences were clustered in the branch (accession: MG767211) with a sequence obtained from Guangzhou in 2017 (posterior probability = 0.9999), with no other clusters in the same branch. The results from the paleogeography analysis suggested that Guangzhou was the location where the most recent common ancestor was found, while an earlier ancestor appeared in Singapore. The MCC tree of Cluster III suggested that the sample sequences were clustered in the same branch with the Taiwanese 2015 and 2016 sequences, as well as the cluster of the Singaporean 2016 sequences (posterior probability = 0.9992). Through paleogeography analysis, the sample virus might be spread along a propagation path and imported into Guangzhou from Taiwan in 2018 or before. The fourth and fifth clusters belonged to DENV-2. As shown in the MCC tree of Cluster IV, the sample sequences were clustered in the same branch with the Malaysian 2013 sequence (posterior probability = 0.9999), indicating that the most recent ancestor of the sample sequences emerged from Malaysia. According to the propagation path map, the sample virus entered Guangzhou during 2013–2018. The MCC tree of Cluster V suggested that the sample sequences were clustered in the same branch with the Indian 2015 sequences and the Taiwanese 2015 sequence cluster (posterior probability = 0.8465). According to the propagation path, the sample virus was imported into Guangzhou during 2015–2018. It is difficult to infer a highly accurate propagation path as no more sequences are available from the GenBank database. The propagation path map of all sample viruses is included in <a href="#pone.0224676.g004">Fig 4</a>. The MCC trees are shown in Figs <a href="#pone.0224676.g005">5</a>–<a href="#pone.0224676.g009">9</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
The propagation path map of dengue virus in Guangzhou in 2018.</div> <p>The propagation path map is drawn based on the paleogeography analysis. The red curves indicated that the clusters of sample sequences were belonged to DENV-1, while the blue curves belonged to DENV-2, which indicated the propagation path. The time beside the curves indicates the time range in which the virus strain is introduced, and the red flag indicates different locations.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility tree of the sequence of cluster I.</div> <p>A| A root-to-tip analysis was performed in TempEst v1.5. An ML tree was built using the sample dataset to determine the R2 value; R2 &amp;gt; 0 indicates a positive correlation between the dataset and the molecular clock. B| Sequences from different geographic regions are represented by different colors. The location on the branch is the most recent ancestor position, and the value is the posterior probability. The purple branch indicates that Fujian Province is the most recent ancestor of the sequences; the green branch corresponds to Malaysia; the blue branch corresponds to Taiwan Province; and the red branch corresponds to Guangzhou. The dotted line box represents the sample sequence obtained in this experiment.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility tree of the sequence of cluster II.</div> <p>A| A root-to-tip analysis was performed in TempEst v1.5. An ML tree was built using the sample dataset to determine the R2 value; R2 &amp;gt; 0 indicates a positive correlation between the dataset and the molecular clock. B| Sequences from different geographic regions are represented by different colors. The location on the branch is the most recent ancestor position, and the value is the posterior probability. The blue branch indicates that Singapore is the most recent ancestor of the sequences; and the red branch corresponds to Guangzhou. The dotted line box represents the sample sequence obtained in this experiment.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g007"><span class="label" tagx="label" title="label">Fig 7</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility tree of the sequence of cluster III.</div> <p>A| A root-to-tip analysis was performed in TempEst v1.5. An ML tree was built using the sample dataset to determine the R2 value; R2 &amp;gt; 0 indicates a positive correlation between the dataset and the molecular clock. B| Sequences from different geographic regions are represented by different colors. The location on the branch is the most recent ancestor position, and the value is the posterior probability. The green branch indicates that Singapore is the most recent ancestor of the sequences; the blue branch corresponds to Taiwan Province; and the red branch corresponds to Guangzhou. The dotted line box represents the sample sequence obtained in this experiment.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g008"><span class="label" tagx="label" title="label">Fig 8</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility tree of the sequence of cluster IV.</div> <p>A| A root-to-tip analysis was performed in TempEst v1.5. An ML tree was built using the sample dataset to determine the R2 value; R2 &amp;gt; 0 indicates a positive correlation between the dataset and the molecular clock. B| Sequences from different geographic regions are represented by different colors. The location on the branch is the most recent ancestor position, and the value is the posterior probability. The purple branch indicates that Indonesia is the most recent ancestor of the sequences; the blue branch corresponds to Malaysia; the orange branch corresponds to Zhongshan City; the green branch corresponds to East Timor; and the red branch corresponds to Guangzhou. The dotted line box represents the sample sequence obtained in this experiment.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0224676.g009"><span class="label" tagx="label" title="label">Fig 9</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility tree of the sequence of cluster V.</div> <p>A| A root-to-tip analysis was performed in TempEst v1.5. An ML tree was built using the sample dataset to determine the R2 value; R2 &amp;gt; 0 indicates a positive correlation between the dataset and the molecular clock. B| Sequences from different geographic regions are represented by different colors. The location on the branch is the most recent ancestor position, and the value is the posterior probability. The purple branch indicates that Malaysia is the most recent ancestor of the sequences; the green branch corresponds to India; the blue branch corresponds to Taiwan province; and the red branch corresponds to Guangzhou. The dotted line box represents the sample sequence obtained in this experiment.</p>   </div> </div> <div class="5.averagegeneticdistancecomparison" title="sec"> <div class="title" tagx="title" title="title">
5. Average genetic distance comparison</div> <p>The five sample clusters were compared with the genetic distances of the same genotype in clusters in 2014–2018. It can be seen from the table that the sequence of the DENV-2 two-cluster sample collected in this study differs greatly from the sequence of the same genotype. The sequence differences between other sample clusters and the same genotype are small. The data is presented in <a href="#pone.0224676.t004">Table 4</a>.</p> <div class="table-wrap_UNKNOWN" id="pone.0224676.t004" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0224676.t004">10.1371/journal.pone.0224676.t004</a><span class="label" tagx="label" title="label">Table 4</span> <div class="title" tagx="title" title="title">
Comparison of average genetic distances of different clusters.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0224676.t004g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="justify" rowspan="1" colspan="1" /> <th align="left" rowspan="1" colspan="1">Species 1</th> <th align="left" rowspan="1" colspan="1">Species 2</th> <th align="left" rowspan="1" colspan="1">Distance</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="3" colspan="1">DENV 1-Genotype I</td> <td align="left" rowspan="1" colspan="1">others-1</td> <td align="left" rowspan="1" colspan="1">sample2</td> <td align="left" rowspan="1" colspan="1">0.0205329387</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">others-1</td> <td align="left" rowspan="1" colspan="1">sample1</td> <td align="left" rowspan="1" colspan="1">0.0189086126</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">sample2</td> <td align="left" rowspan="1" colspan="1">sample1</td> <td align="left" rowspan="1" colspan="1">0.0202604556</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">DENV 1-Genotype V</td> <td align="left" rowspan="1" colspan="1">others-2</td> <td align="left" rowspan="1" colspan="1">sample3</td> <td align="left" rowspan="1" colspan="1">0.0067526479</td> </tr> <tr> <td align="left" rowspan="3" colspan="1">DENV 2-Cosmopolitan</td> <td align="left" rowspan="1" colspan="1">others-3</td> <td align="left" rowspan="1" colspan="1">sample4</td> <td align="left" rowspan="1" colspan="1">0.0336621574</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">sample4</td> <td align="left" rowspan="1" colspan="1">sample5</td> <td align="left" rowspan="1" colspan="1">0.0670677460</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">sample5</td> <td align="left" rowspan="1" colspan="1">others-3</td> <td align="left" rowspan="1" colspan="1">0.0657505057</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Others in the table indicates the non-sample sequence of the genotype in Guangdong Province from 2014 to 2018.</p> </div> </div> </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In the present study, blood samples were drawn from DENV-infected patients who were diagnosed with dengue fever by hospitals in Guangzhou during 2018, and an analysis was performed on the DENV strains isolated from these blood samples. Through gel electrophoresis analysis, the isolated sequences were classified into two serotypes, namely, DENV-1 and DENV-2. Furthermore, DENV-1 strains were divided into three clusters, and the DENV-2 strains were divided into two clusters according to the ML trees. The ancestral reconstruction analysis showed that the sample sequences largely originated from Indonesia, Malaysia, Singapore, and Taiwan. This suggests that the dengue virus that we are popular in Guangzhou is still based on input. As the capital city of Guangdong Province and the trade center in Southeast China, Guangzhou is exposed to a relatively high risk of the disease, as DENV may enter the city along with imported goods and migrant workers. In addition, DENV continuously flows into China because Thailand, Malaysia, Singapore, and other Southeast Asian countries are popular destinations for Chinese tourists, especially those living in Guangdong Province. Therefore, entry-exit inspection and quarantine should be implemented effectively for dengue prevention and control. For instance, suspected and confirmed cases of DENV infection should be isolated and treated properly to reduce the risk of imported dengue fever. In addition, health education also plays an important role in dengue prevention and control. To reduce the risk of DENV infection, public health authorities should provide the necessary materials for tourists to gain a better understanding of dengue prevention and remind them not to visit an endemic area during epidemic seasons.</p> <p>Interestingly, the results from the traceability analysis showed that the most recent common ancestor of the sequences of cluster II emerged from Guangzhou, and Cluster II and the Guangzhou 2017 isolate are in the same branch without any other foreign sequences. Moreover, they shared a common ancestral lineage to the Malaysian 2014 isolate, the Singaporean 2014 isolate, and the Zhongshan 2015 isolate in other branches. Phylogeographically, there are two possible propagation paths. First, the virus originated in Singapore and became an epidemic in the country before 2014; during the next year, it was imported into Zhongshan, and then in 2017, the disease flowed into Guangzhou from other cities in Guangdong Province; the Guangzhou 2018 isolates came from the virus imported into the city during 2017 as a result of local transmission. Second, the virus emerged from Singapore and was not introduced into Zhongshan until 2015; during 2017 and 2018, the virus was continuously imported into Guangzhou from Singapore or other countries and regions; the sample sequences were not clustered into a branch with the isolates from other countries because no related sequences were available in GenBank or the database had no sufficient patient isolates. Regarding the sources of spread or propagation, some DENV strains were imported from endemic countries and regions and caused dengue fever without further propagation in China; some evolved from imported strains and led to local epidemics during the year; others were localized strains after vertical transmission. Although no substantial evidence was found in this study to prove vertical transmission of DENV in Guangzhou, previous phylogenetic analyses indicated possible local transmission of dengue fever in the city.</p> <p>Since <i>Aedes albopictus</i> is the main medium for DENV transmission, the localization of DENV largely depends on the formation of localized and virus-carrying eggs and the survival of DENV in the eggs and offspring mosquitoes through winter[<a href="#pone.0224676.ref034">34</a>]. A previous study demonstrated that DENV can survive in eggs and spread to offspring mosquitoes. Transovarial transmission of DENV is strongly temperature-related as it determines whether the eggs and young offspring can wait out the winter season. As an example, Yunnan Province has reported the localization of DENV as it has a subtropical monsoon climate and provides a natural habitat for <i>Aedes albopictus</i>. Similar to Yunnan Province, Guangzhou has hot, humid summers and mild, dry winters, which create a favorable environment for the breeding of <i>Aedes</i> mosquitoes. Moreover, as the greenhouse effect continues to warm the planet slowly, overwintering becomes easier for mosquitoes. Peri-urban areas in Guangzhou have plenty of dirty gullies, open spaces, and rented houses that require effective management to improve the living environment. Additionally, local residents who like indoor and outdoor planting, fish farming, and water harvesting also provide places for mosquito breeding. To reduce mosquito breeding sites, effective urban sanitation management should be implemented. For example, stagnant water should be cleared in a timely manner to ensure the smooth operation and maintenance of the city's drainage system; in crowded places, mosquito prevention and control measures should be taken for the public good. Personal hygiene also plays a critical role in dengue prevention. Residents should avoid keeping fish and aquatic plants and regularly clean up stagnant water in their houses. An individual with a fever or other dengue fever-like symptoms should promptly seek medical attention. Hospitals should strive to improve the diagnostic accuracy and efficiency of dengue fever. Suspected DENV-infected patients should be isolated to prevent further transmission. We should pay closer attention to localized DENV in Guangzhou and strictly implement relevant dengue prevention and control measures.</p> <p>Although the results of this study implied that vertical transmission of DENV might exist in Guangzhou, there is no solid evidence supporting the inference as no adequate sequences are available for analysis. The coverage of the sequence has a great impact on the study. Patients with dengue sometimes have negative infections. The patient is not aware that the infection with dengue virus has led to a decrease in the reported sequence, and a small number of viruses have not been sequenced and uploaded to the database. These can all lead to the wrong propagation path. Therefore, local centers for disease control and prevention should work closely with disease control and prevention departments of Southeast Asian countries to observe how DENV is prevalent in these countries, thereby identifying sources of propagation for DENV in China and exploring the epidemiological characteristics of dengue fever to provide a basis for dengue prevention strategies. As the Belt and Road Initiative moves forward, Asian, European and African countries expect to increase transport connections for international trade, strengthen people-to-people exchanges and remove barriers to investment and trade. Given that, the conventional entry-exit inspection and quarantine divided by national administrative regions is falling short, leading to an increased risk of dengue and Zika fever in the countries along the Belt and Road. There is an urgent need to raise public awareness about disease prevention and control, as well as intensified measures against dengue and Zika fever. It is necessary to establish joint prevention and control mechanisms for infectious diseases so that China and other relevant countries can work together to effectively combat infectious diseases and safeguard health security with their technologies and resources.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Original image of agarose gel electrophoresis.</div> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0224676.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
The kit for diagnosing dengue patients is used in hospitals.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="pone.0224676.s002.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
All accession numbers of the DENV-1 sequence used in this study.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="pone.0224676.s003.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
All accession numbers of the DENV-2 sequence used in this study.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="pone.0224676.s004.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The First Affiliated Hospital of Guangzhou University of Chinese Medicine needs to be thanked for their support. The authors also need to be thanked for the contribution of this study.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0224676.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messina</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pigott</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duda</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global spread of dengue virus types: mapping the 70 year history</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">138</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2013.12.011">10.1016/j.tim.2013.12.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24468533">24468533</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilakis</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dengue—quo tu et quo vadis?</span><span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1562</span>–<span class="lpage" tagx="lpage" title="lpage">608</span>. Epub 2011/10/14. <span class="pub-id"><a href="https://dx.doi.org/10.3390/v3091562">10.3390/v3091562</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21994796">21994796</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endy</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">IK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mammen</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Prospective cohort studies of dengue viral transmission and severity of disease</span>. <span class="source" tagx="source" title="source">Curr Top Microbiol Immunol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">338</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>. Epub 2009/10/06. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-3-642-02215-9_1">10.1007/978-3-642-02215-9_1</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19802574">19802574</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global Epidemiology of Dengue Outbreaks in 1990–2015: A Systematic Review and Meta-Analysis</span>. <span class="source" tagx="source" title="source">Front Cell Infect Microbiol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">317</span> Epub 2017/07/28. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fcimb.2017.00317">10.3389/fcimb.2017.00317</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28748176">28748176</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="source" tagx="source" title="source">Dengue: Guidelines for Diagnosis, Treatment, Prevention and Control: New Edition</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">WHO Guidelines Approved by the Guidelines Review Committee</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva</span><span class="year" tagx="year" title="year">2009</span>.</span></li> <li tag="ref"><a name="pone.0224676.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ko</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chao</span><span class="given-names" tagx="given-names" title="given-names">DY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">ZT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Inter- and intra-host sequence diversity reveal the emergence of viral variants during an overwintering epidemic caused by dengue virus serotype 2 in southern Taiwan</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">e0006827</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0006827">10.1371/journal.pntd.0006827</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30286095">30286095</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shepard</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Undurraga</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Betancourt-Cravioto</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzman</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Approaches to refining estimates of global burden and economics of dengue</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e3306</span> Epub 2014/11/21. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0003306">10.1371/journal.pntd.0003306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25412506">25412506</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qing</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Characterization of dengue virus resistance to brequinar in cell culture</span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">54</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">3686</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AAC.00561-10">10.1128/AAC.00561-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20606073">20606073</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Differences in the Transmission of Dengue Fever by Different Serotypes of Dengue Virus</span>. <span class="source" tagx="source" title="source">Vector Borne Zoonotic Dis</span>. <span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/vbz.2019.2477">10.1089/vbz.2019.2477</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31503521">31503521</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology demonstrates that imported and local strains circulated during the 2014 dengue outbreak in Guangzhou, China</span>. <span class="source" tagx="source" title="source">Virol Sin</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">32</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">63</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12250-016-3872-8">10.1007/s12250-016-3872-8</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28120220">28120220</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johansson</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wardrop</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Panhuis</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wesolowski</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Seasonal and interannual risks of dengue introduction from South-East Asia into China, 2005–2015</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e0006743</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0006743">10.1371/journal.pntd.0006743</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30412575">30412575</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Distribution of Different Clades of Seneca Valley Viruses in Guangdong Province, China</span>. <span class="source" tagx="source" title="source">Virol Sin</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">394</span>–<span class="lpage" tagx="lpage" title="lpage">401</span>. Epub 2018/10/18. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s12250-018-0056-8">10.1007/s12250-018-0056-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30328012">30328012</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perkins</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The changing epidemiology of dengue in China, 1990–2014: a descriptive analysis of 25 years of nationwide surveillance data</span>. <span class="source" tagx="source" title="source">BMC Med</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">100</span> Epub 2015/05/01. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12916-015-0336-1">10.1186/s12916-015-0336-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25925417">25925417</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">FC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">WX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">LZ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epidemiological and Virological Characterizations of the 2014 Dengue Outbreak in Guangzhou, China</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">e0156548</span> Epub 2016/06/04. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0156548">10.1371/journal.pone.0156548</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27257804">27257804</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">GQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yakob</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">WY</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Driving Force for 2014 Dengue Outbreak in Guangdong, China</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e0166211</span> Epub 2016/11/20. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0166211">10.1371/journal.pone.0166211</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27861514">27861514</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spear</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marshall</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">The interplay of climate, intervention and imported cases as determinants of the 2014 dengue outbreak in Guangzhou</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">e0005701</span> Epub 2017/06/24. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0005701">10.1371/journal.pntd.0005701</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28640895">28640895</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the viral structural gene of the first dengue virus type 1 outbreak in Xishuangbanna: A border area of China, Burma and Laos</span>. <span class="source" tagx="source" title="source">Int J Infect Dis</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">79</span>:<span class="fpage" tagx="fpage" title="fpage">152</span>–<span class="lpage" tagx="lpage" title="lpage">61</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ijid.2018.11.370">10.1016/j.ijid.2018.11.370</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30528395">30528395</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">MX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hansen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanson-Easey</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue control in the context of climate change: Views from health professionals in different geographic regions of China</span>. <span class="source" tagx="source" title="source">J Infect Public Health</span>. <span class="year" tagx="year" title="year">2018</span> Epub 2019/01/05. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jiph.2018.12.010">10.1016/j.jiph.2018.12.010</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30606474">30606474</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue is still an imported disease in China: a case study in Guangzhou</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">178</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. Epub 2015/03/17. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2015.03.005">10.1016/j.meegid.2015.03.005</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25772205">25772205</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Predicting unprecedented dengue outbreak using imported cases and climatic factors in Guangzhou, 2014</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">e0003808</span> Epub 2015/05/29. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0003808">10.1371/journal.pntd.0003808</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26020627">26020627</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radke</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregory</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kintziger</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauber-Schatz</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunsperger</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallagher</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dengue outbreak in Key West, Florida, USA, 2009</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">135</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2012/01/20. <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1801.110130">10.3201/eid1801.110130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22257471">22257471</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yong</span><span class="given-names" tagx="given-names" title="given-names">YK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thayan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">HT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sekaran</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Rapid detection and serotyping of dengue virus by multiplex RT-PCR and real-time SYBR green RT-PCR</span>. <span class="source" tagx="source" title="source">Singapore Med J</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">48</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">662</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17609830">17609830</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>. <span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">772</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>. Epub 2013/01/19. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golden</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khoosal</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muhire</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">RDP4: Detection and analysis of recombination patterns in virus genomes</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vev003</span> Epub 2015/05/26. <span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev003">10.1093/ve/vev003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774277">27774277</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. <span class="mixed-article-title" title="mixed-article-title">CD-HIT Suite: a web server for clustering and comparing biological sequences</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">680</span>–<span class="lpage" tagx="lpage" title="lpage">2</span>. Epub 2010/01/08. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btq003">10.1093/bioinformatics/btq003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20053844">20053844</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">DAMBE: software package for data analysis in molecular biology and evolution</span>. <span class="source" tagx="source" title="source">J Hered</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">371</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. Epub 2001/09/06. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jhered/92.4.371">10.1093/jhered/92.4.371</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11535656">11535656</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darriba</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taboada</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doallo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. <span class="source" tagx="source" title="source">Nat Methods</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">772</span> Epub 2012/08/01. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.2109">10.1038/nmeth.2109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22847109">22847109</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Haeseler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minh</span><span class="given-names" tagx="given-names" title="given-names">BQ</span></span>. <span class="mixed-article-title" title="mixed-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">32</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">268</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>. Epub 2014/11/06. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu300">10.1093/molbev/msu300</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430">25371430</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1312</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. Epub 2014/01/24. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Max Carvalho</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vew007</span> Epub 2016/10/25. <span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vew007">10.1093/ve/vew007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774300">27774300</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">214</span> Epub 2007/11/13. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayres</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaron</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zwickl</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peter</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holder</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">PO</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="source" tagx="source" title="source">BEAGLE: an application programming interface and high-performance computing library for statistical phylogenetics</span>. <span class="year" tagx="year" title="year">2012</span>.</span></li> <li tag="ref"><a name="pone.0224676.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baele</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">67</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">901</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. Epub 2018/05/03. <span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syy032">10.1093/sysbio/syy032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29718447">29718447</a></span></span></li> <li tag="ref"><a name="pone.0224676.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Survival and replication of dengue-2 virus in diapausing eggs of Aedes albopictus (Diptera: Culicidae)</span>. <span class="source" tagx="source" title="source">J Med Entomol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">44</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">492</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1603/0022-2585(2007)44[492:sarodv]2.0.co;2">10.1603/0022-2585(2007)44[492:sarodv]2.0.co;2</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17547236">17547236</a></span></span></li> </ul> </div> </div>  </body></html>