<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Clin Infect Dis</span><span class="iso-abbrev" title="iso-abbrev">Clin. Infect. Dis</span><span class="publisher-id" title="publisher-id">cid</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1058-4838</span><span class="issn-epub" tagx="issn" title="issn-epub">1537-6591</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">US</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/cid/ciy821">10.1093/cid/ciy821</a></span><span class="publisher-id" title="publisher-id">ciy821</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Articles and Commentaries</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The Effect of Influenza Virus on the Human Oropharyngeal Microbiome</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramos-Sevillano</span><span class="given-names" tagx="given-names" title="given-names">Elisa</span></span><a href="#AF0001">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wade</span><span class="given-names" tagx="given-names" title="given-names">William G</span></span><a href="#AF0002">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mann</span><span class="given-names" tagx="given-names" title="given-names">Alex</span></span><a href="#AF0003">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">Anthony</span></span><a href="#AF0003">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambkin-Williams</span><span class="given-names" tagx="given-names" title="given-names">Robert</span></span><a href="#AF0003">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Killingley</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span><a href="#AF0004">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen-Van-Tam</span><span class="given-names" tagx="given-names" title="given-names">Jonathan S</span></span><a href="#AF0005">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Christoph M</span></span><a href="#AF0001">1</a><a href="#c1" /></span></div><span class="citation_author_institution" id="AF0001">[1], </span><span class="citation_author_institution" id="AF0002">[2], </span><span class="citation_author_institution" id="AF0003">[3], </span><span class="citation_author_institution" id="AF0004">[4], </span><span class="citation_author_institution" id="AF0005">[5], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Correspondence: C. M. Tang, Sir William Dunn School of Pathology, University of Oxford, Park Road, Oxford OX1 3RE, United Kingdom (<span class="email" tagx="email" title="email">christoph.tang@path.ox.ac.uk</span>).</div> </div><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-6-6</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-11-11</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-11-11</span></span><span class="volume" tagx="volume" title="volume">68</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1993</span><span class="lpage" tagx="lpage" title="lpage">2002</span><span class="history" title="history"><span class="received" title="received">received: 2018-6-08</span><span class="accepted" title="accepted">accepted: 2018-10-04</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="ciy821.pdf" title="self-uri">ciy821.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Secondary bacterial infections are an important cause of morbidity and mortality associated with influenza infections. As bacterial disease can be caused by a disturbance of the host microbiome, we examined the impact of influenza on the upper respiratory tract microbiome in a human challenge study.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>The dynamics and ecology of the throat microbiome were examined following an experimental influenza challenge of 52 previously-healthy adult volunteers with influenza A/Wisconsin/67/2005 (H3N2) by intranasal inoculation; 35 healthy control subjects were not subjected to the viral challenge. Serial oropharyngeal samples were taken over a 30-day period, and the V1-V3 region of the bacterial 16S ribosomal RNA sequences were amplified and sequenced to determine the composition of the microbiome. The carriage of pathogens was also detected.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>Of the 52 challenged individuals, 43 developed proven influenza infections, 33 of whom became symptomatic. None of the controls developed influenza, although 22% reported symptoms. The diversity of bacterial communities remained remarkably stable following the acquisition of influenza, with no significant differences over time between individuals with influenza and those in the control group. Influenza infection was not associated with perturbation of the microbiome at the level of phylum or genus. There was no change in colonization rates with <i>Streptococcus pneumoniae</i> or <i>Neisseria meningitidis</i>.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>The throat microbiota is resilient to influenza infection, indicating the robustness of the upper-airway microbiome.</p> </div> </div> <div class="abstract" title="abstract"> <p>Secondary bacterial infections cause significant morbidity following influenza virus infection. We show that influenza A virus infection does not have a major impact on the human pharyngeal microbiome, indicating that microbial communities are robust in this site.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
microbiome</div> <div class="kwd" title="kwd">
influenza</div> <div class="kwd" title="kwd">
upper respiratory tract</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Centers for Disease Control and Prevention</span><span class="named-content" title="named-content">10.13039/100000030</span></div><span class="award-id" tagx="award-id" title="award-id">1U01P000497-01)</span></div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <p>Secondary bacterial infection is a major cause of the mortality and morbidity associated with the influenza virus [<a href="#CIT0001">1</a>]. Bacterial pneumonia caused millions of deaths during influenza epidemics in the 20<sup>th</sup> century [<a href="#CIT0002">2</a>]. For example, 94% of a group of patients who died in the 1918 influenza pandemic had evidence of secondary bacterial pneumonia [<a href="#CIT0003">3</a>], while 28% of fatalities in New York associated with the 2009 H1N1 influenza pandemic had bacterial co-infections, with most diagnosed post-mortem [<a href="#CIT0004">4</a>].</p> <p>Several pathogens cause bacterial infection following influenza. Co-infections with <i>Haemophilus influenzae</i> were so common that it was initially thought to be the cause influenza [<a href="#CIT0005">5</a>]. <i>Streptococcus pneumoniae</i> is the commonest influenza-associated infection [<a href="#CIT0006">6</a>], while cavitatory <i>Staphylococcus aureus</i> pneumonia is a severe influenza complication; secondary pneumonia with methicillin resistant Staphylococcus aureus emerged in the 2009 pandemic [<a href="#CIT0007">7</a>]. There is also an association between influenza A and <i>Neisseria meningitidis</i> disease [<a href="#CIT0008">8</a>].</p> <p>Multiple mechanisms could explain the development of bacterial disease following influenza. The virus is cytolytic and induces epithelial damage, impairment of surfactant production and muco-ciliary clearance [<a href="#CIT0009">9</a>, <a href="#CIT0010">10</a>]. Exposure of the basement membrane and matrix offers binding sites for bacteria [<a href="#CIT0011">11</a>]; glycan receptors can be revealed by viral and bacterial neuraminidases [<a href="#CIT0012">12</a>, <a href="#CIT0013">13</a>]. Additionally, the increased secretion of galectin 1 and galectin 3 can favor the adhesion of <i>S. pneumoniae</i> and <i>N. meningitidis</i> [<a href="#CIT0014">14</a>, <a href="#CIT0015">15</a>]. Murine models indicate that host sialic acid, released by neuraminidases, can also act as a nutrient for pneumococci in the upper airway, promoting bacterial spread [<a href="#CIT0016">16</a>].</p> <p>The human microbiome is a complex community that has co-evolved with its host. A healthy microbiota protects against invasion by pathogenic bacteria through colonization resistance, in which the microbiota restricts the growth or attachment of pathogens, either by competing for nutrients and ecological niches, or by direct antagonism [<a href="#CIT0017">17</a>].</p> <p>Several factors influence the microbiome, including age, smoking, and antibiotic treatment [<a href="#CIT0018">18â€"20</a>]. Furthermore changes in the microbiota are associated with airway disease [<a href="#CIT0021">21</a>]. For example, healthy volunteers were reported to have a more diverse nasopharyngeal microbiome compared with patients suffering from pneumonia [<a href="#CIT0022">22</a>], while a highly diverse community is found in patients with asthma, cystic fibrosis, or Chronic Obstructive Pulmonary Disease [<a href="#CIT0023">23</a>, <a href="#CIT0024">24</a>]. However, little is known about the impact of viral infection on the microbiota. No specific profiles of bacteria in the upper respiratory tract (URT) were associated with any of 7 different viral infections, although an apparent decrease in the carriage of <i>Haemophilus</i> and <i>Neisseria</i> was observed after rhinovirus [<a href="#CIT0025">25</a>, <a href="#CIT0026">26</a>].</p> <p>To further understand secondary bacterial infections, we investigated the impact of the influenza virus on the human nasopharyngeal microbiome. Using 454 pyrosequencing, we characterized the microbiota over a 30-day period in 52 healthy adults challenged with influenza A H3N2 and in 32 non-challenged individuals. This approach allowed us to define changes in the human microbiome following an influenza challenge. Furthermore, we examined whether influenza affected the carriage of pathogens in the upper airways.</p> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="studydesign" title="sec"> <div class="title" tagx="title" title="title">
Study Design</div> <p>The study took place in a quarantine facility with informed consent from volunteers, in accordance with the Declaration of Helsinki and with United Kingdom regulatory and ethical (Institutional Review Board) requirements.</p> <p>Volunteers were screened before study entry. In brief, volunteers were healthy, with no acute or chronic medical condition; were between the ages of 18 and 45; were not living with anyone at risk of influenza complications; and had not had an influenza vaccine in the last 3 years. Blood samples from volunteers were collected before quarantine entry.</p> <p>Subjects were randomly allocated to either the challenge (n = 52) or control group (n = 35) on day -1. On day 0, individuals in the challenge group were inoculated intranasally with 5.5 log<sub>10</sub> TCID50/ml/nostril of influenza strain A/H3N2/Wisconsin/67/2005. The control group members were not subjected to challenge (<a href="#F1">Figure 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Study timeline. Volunteers were screened prior to admission to a quarantine unit on the day before receiving an intranasal challenge of influenza A (on day 0), then were followed for 28 days. The volunteers were kept in quarantine for 6 days post-inoculation. Controls were screened and housed in identical conditions, but were not subjected to a viral challenge. Throat swabs were collected at different time points during the study (ie, on days -1, 3, 6, and 28 post-inoculation).</p>   </div> </div> <div class="clinicalmonitoringandsamplecollection" title="sec"> <div class="title" tagx="title" title="title">
Clinical Monitoring and Sample Collection</div> <p>Volunteers recorded symptoms daily, and vital signs were recorded 3 times daily. Nasopharyngeal swabs, oropharyngeal swabs, and venous blood were collected on days âˆ’1, 3, 6, and 28. Respiratory specimens were analyzed by quantitative polymerase chain reaction (qPCR) and serological specimens were analysed by hemagglutination inhibition (HAI) and microneutralization assays. Laboratory-confirmed influenza was defined as either a 4-fold or greater rise in HAI or microneutralization titres between day -2 and day 28 or a positive nasopharyngeal test by PCR.</p> <p>Volunteers were classified according to symptoms and signs. Influenza-like illness (ILI) was defined, according to the Centers for Disease Control and Prevention definition, as an illness lasting â‰¥24 hours with either a temperature of &amp;gt;37.9Â°C or 2 or more symptoms, at least 1 of which was a respiratory symptom. Volunteers were labeled as symptomatic if they experienced symptoms with no fever; experienced a single symptom; had â‰¥2 symptoms, none of which were respiratory; or had an illness of &amp;lt;24 hours. Therefore, a volunteer could have an ILI or be symptomatic and not be infected with influenza. Individuals were classified according to their symptomatology and tests into 3 groups: the ILI<sup>flu+</sup> group consisted of subjects who received the viral challenge and had an ILI, with evidence of an influenza infection (n = 19); the S<sup>flu+</sup> group consisted of subjects who received the viral challenge and were symptomatic, with evidence of an influenza infection (n = 14); and the ILI/S<sup>flu+</sup> group consisted of individuals belonging to both these groups (n = 33). Those who were not challenged with influenza and remained asymptomatic/influenza-negative formed the AC (asymptomatic control) group (n = 24).</p> </div> <div class="dnaisolationandpyrosequencing" title="sec"> <div class="title" tagx="title" title="title">
DNA Isolation and Pyrosequencing</div> <p>DNA was extracted from swabs using the MoBio Power Soil isolation kit (MoBio laboratories). Barcoded primers were used to amplify ~500 bp from the V1-V3 region of the 16S ribosomal RNA (rRNA) gene, as previously described [<a href="#CIT0027">27</a>]. Amplification conditions were 5 minutes at 95Â°C, then 25 cycles of 95Â°C for 45 seconds, 53Â°C for 45 seconds, and 72Â°C for 1 minute 30 seconds, with a final step of 72Â°C for 15 minutes. The products were analyzed using an Agilent 2100 Bioanalyzer after purification with QIAquick (Qiagen) and were quantified by Quant-iT Picogreen (Invitrogen). The samples were pooled at equimolar concentrations and sequenced using a Roche 454 GS-FLX Titanium platform.</p> </div> <div class="sequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence Analysis</div> <p>The quality of sequences was assessed using FastQC v0.11.3 (Babraham Institute). The pre-processing of sequences was performed with mothur v1.35.1 [<a href="#CIT0028">28</a>]. Next, we used trim.flows to remove sequences with mismatches in the primer sequence or barcode. Sequences were de-noised by shhh.flows, trimmed to remove primers and barcodes, then aligned to the SILVA 16S rRNA reference alignment using trim.seqs; chimeric sequences were eliminated by Uchime [<a href="#CIT0029">29</a>]. Sequences were used to interrogate the Human Oral Microbiome Database (HOMD) v13.2 at the 98.5% identity level using Basic Local Alignment Search Tool (BLAST). met the following criteria: read length &amp;gt;375 nucleotides (nt). (excluding barcode and primers), no ambiguous bases, or homopolymers of &amp;gt;8 nt.</p> <p>A sequence dissimilarity distance of 0.015 was used to cluster sequences into operational taxonomic units (OTUs), using the average neighbor algorithm. Taxonomies were assigned using the HOMD reference dataset. The richness of communities was calculated using the Chao1 and Catchall indices [<a href="#CIT0030">30</a>, <a href="#CIT0031">31</a>] and diversity was estimated with the Simpsonâ€™s inverse [<a href="#CIT0032">32</a>] and Shannon [<a href="#CIT0033">33</a>] indices. Goodâ€™s non-parametric estimator was used to assess the coverage of communities in samples [<a href="#CIT0034">34</a>]. The species diversity was examined by analyzing the beta diversity. Samples were adjusted to 2997 reads per sample by random sampling. Distance matrices were generated with the Jaccard Index and the thetaYC measure of dissimilarity [<a href="#CIT0035">35</a>], and by principal coordinate analysis (PCoA). Dendrograms, representing the relationships between samples, were analyzed by parsimony and UniFrac [<a href="#CIT0036">36</a>].</p> </div> <div class="s.pneumoniaeandn.meningitidiscarriage" title="sec"> <div class="title" tagx="title" title="title"><i>S. pneumoniae</i> and <i>N. meningitidis</i> Carriage</div> <p><i>N. meningitidis</i> was isolated by plating to GC selective medium (Oxoid). A 413 bp fragment-encoding RplF was amplified, sequenced, and identified with PubMLST [<a href="#CIT0037">37</a>]. Meningococcal isolates were characterized by serogrouping, serotyping, and serosubtyping. To detect the pneumococcus, <i>lytA</i> qPCR was performed, as previously described [<a href="#CIT0038">38</a>].</p> </div> <div class="statisticalanalyses" title="sec"> <div class="title" tagx="title" title="title">
Statistical Analyses</div> <p>An analysis of alpha diversity was performed using the Kruskal-Wallis test with Dunnâ€™s correction for multiple comparisons. An analysis of molecular variance was used to evaluate differences in PCoAs [<a href="#CIT0039">39</a>]. Parsimony tests were implemented by mothur [<a href="#CIT0040">40</a>]. The linear discriminant analysis (LDA) effect size was used to examine differences in OTUs in the control/challenged groups [<a href="#CIT0041">41</a>]; the alpha values were set to 0.05 and a LDA score of 2.5 was selected. A 2-tailed Fisher exact test was applied to compare colonization rates.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="infectionandthedevelopmentofsymptomsafterinfluenzachallenge" title="sec"> <div class="title" tagx="title" title="title">
Infection and the Development of Symptoms After Influenza Challenge</div> <p>Although volunteers were screened for antibodies against influenza in advance, 3 individuals in the challenge group and 3 in the control group had serological evidence of a prior H3N2 infection at study entry, so were excluded from subsequent analyses.</p> <p>Influenza infection was confirmed in 43 of the 52 individuals (82%) subjected to the influenza challenge; 33 subjects developed symptoms, including 19 with ILI (ILI<sup>flu+</sup>). An additional 3 subjects were challenged with influenza and became symptomatic, but had no evidence of influenza infection. There were 3 other challenged individuals that remained asymptomatic, with no evidence of influenza.</p> </div> <div class="characterizationofthepharyngealmicrobiome" title="sec"> <div class="title" tagx="title" title="title">
Characterization of the Pharyngeal Microbiome</div> <p>A total of 2505196 sequences were obtained from 87 individuals (52 challenged and 35 controls) on days -1, 3, 6, and 28. DNA extraction failed for 1 sample and amplification was unsuccessful from 4 samples, giving us 343 usable samples. The removal of chimeric sequences yielded 2057548 sequences.</p> <p>Initially, we compared ILI/S<sup>flu+</sup> (n = 33) and AC (n = 24) individuals. To normalize the dataset, we took 2997 sequences from each sample; 11 samples had fewer than this, so were excluded, leaving 213 samples. We identified 2209 OTUs (based on &amp;lt;98.5% of identity, &amp;gt;4 representatives), with 24% of sequences falling into 3 OTUs: OTU1 (accounting for 9.78% of sequences), with predominant member <i>Fusobacterium periodonticum</i>; OTU2 (7.8%), with predominant members <i>S. salivarus</i>/<i>S. vestibularis</i>; and OTU3 (7%), with predominant members <i>S. mitis</i>/<i>S. pneumoniae</i>.</p> </div> <div class="changesinthediversityofnasopharynxmicrobiotabasedonoperationaltaxonomicunitsanalysis" title="sec"> <div class="title" tagx="title" title="title">
Changes in the Diversity of Nasopharynx Microbiota Based on Operational Taxonomic Units Analysis</div> <p>Coverage of bacterial communities was high (mean 97.6% Â± 0.007%), with samples having an average of 161 OTUs (range 63 to 321). The number of OTUs at each time point was not significantly different between groups (asymptomatic controls vs ILI/S<sup>flu+</sup>, <i>P =</i> .4425; <a href="#F2">Figure 2</a>). Furthermore, the number of OTUs remained remarkably constant over time, even in the ILI<sup>flu+</sup> group, with no significant difference compared with asymptomatic controls (<a href="#sup1">Supplementary Figures 1</a> and <a href="#sup1">2</a>; <i>P</i>s = .5543 and .5836, respectively). The Chao 1 index measures the predicted richness of communities, so gives a higher value (median of 262 OTUs; range, 84 to 577 OTUs; <a href="#F2">Figure 2</a>) than the recorded OTUs. The non-parametric estimator Catchall yielded a median of 437 OTUs per sample (range 66 to 2643). There were no significant differences in the richness of the between the ILI/S<sup>flu+</sup> group and asymptomatic control groups (<a href="#F2">Figure 2</a>). Furthermore, there was no significant difference in the richness of microbial communities in symptomatic individuals, even when comparing the ILI/S<sup>flu+</sup> group with the AC group (<a href="#sup1">Supplementary Figures 1</a> and <a href="#sup1">2</a>).</p> <p>We also estimated the diversity of communities using the Shannon index. Again, no difference was detected between the challenged and control groups (<a href="#F2">Figure 2</a>; <a href="#sup1">Supplementary Figure 1</a> and <a href="#sup1">2</a>), even when we analyzed the diversity of the communities in highly-symptomatic ILI subjects and asymptomatic controls (<a href="#sup1">Supplementary Table 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Analysis of the richness of the URT microbiome. The diversity of the microbiome was analyzed by sequencing the V1-3 region of 16S rRNA amplified from samples obtained from the ILI/S<sup>flu+</sup> (white boxes) and asymptomatic control (grey boxes) groups following the influenza A challenge. The box whisker plots extend from the 25<sup>th</sup> to 75<sup>th</sup> percentiles, and the ends of the whiskers show the maximum and minimum values. The line in the middle of the box represents the median and the dots represent the outliers 1.5 greater or lower than the interquartile distance. The analysis was based on: (<i>A</i>) the number of OTUs; (<i>B</i>) Chao1 index; and (<i>C</i>) Shannon diversity index at baseline, 3 (3dpi), 6 (6dpi) and 28 (28dpi) days postâ€"influenza challenge. Abbreviations: AC, asymptomatic control; dpi, days postâ€"influenza challenge; ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic; OTU, operational taxonomic units; URT, upper respiratory tract.</p>   </div> <p>Next, we assessed differences between samples. Dendrograms were generated, highlighting similarities of the membership and structure of communities using Jaccard and thetaYC indices, respectively. At the 4 time points, results for certain individuals clustered together (eg, individual S<sup>flu+</sup> individual 2, ILI<sup>flu+</sup>6, S<sup>flu+</sup>7, S<sup>flu+</sup>10, ILI<sup>flu+</sup>11, S<sup>flu+</sup>12, ILI<sup>flu+</sup>13, S<sup>flu+</sup>17, ILI<sup>flu+</sup>28, AC61, and AC65), indicating that the individual had a stronger influence on the microbiome than did influenza, with samples from 4 time points clustering together (<a href="#sup1">Supplementary Figure 3</a>). There was no significant clustering of OTUs either between groups or over time when we applied a parsimony test. The thetaYC index yielded a complex dendrogram (<a href="#sup1">Supplementary Figure 4</a>). Although there were no large shifts in the communities, minor changes were detected when we applied a parsimony test in the ILI/S<sup>flu+</sup> and AC groups, with slight differences on days 3, 6, and 28 (<i>P</i> &amp;lt; .02, <i>P</i> &amp;lt; .024, and <i>P</i> &amp;lt; .001, respectively). However, these differences were not found after applying a Unifrac unweighted test.</p> <p>The community structure and membership was also visualized by PCoA (<a href="#F3">Figure 3</a>). Small differences were found in the structure of communities in the ILI/S<sup>flu+</sup> and AC groups at day 3 and day 6 (<i>P</i> &amp;lt; .035 and <i>P</i> &amp;lt; .033, respectively; <a href="#F3">Figure 3</a>). Differences were not found in the memberships. The community structure was also examined for each group over time (<a href="#sup1">Supplementary Figure 5</a>); at day 6, the ILI/S<sup>flu+</sup> group showed a statistically significant difference compared to baseline (<i>P</i> &amp;lt; .036), while in the AC group, days 3, 6, and 28 each showed some statistically significant difference compared to baseline (<i>P</i> &amp;lt; .001), indicating that quarantine might have effects on the URT microbiome.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Throat microbiota structure in the ILI/S<sup>flu+</sup> and asymptomatic control groups during the influenza challenge. The PCoA was based on the Thetayc index, comparing the community structures of samples from the ILI/S<sup>flu+</sup> infected group (blue open circles) and asymptomatic control (red open circles) groups at days (<i>A</i>) 3 and (<i>B</i>) 6. The centroid represents the arithmetic mean for each of the groups, each dot represents the microbiota structure profile for each of the samples, while the ellipses represent the 95% of the samples belonging to each group. Abbreviations: ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic; PCoA, principal co-ordinates analysis .</p>   </div> <p>By LDA effect size analysis, the abundance of some OTUs was significantly different between groups on days 3 and 6 post-infection (<a href="#F4">Figure 4</a>). The OTUs which were more abundant within the ILI/S<sup>flu+</sup> group at day 3 included OTU11 (<i>Prevotella melaninogenica</i>), OTU14 (<i>Leptotrichia</i>), OTU38 (Human Oral Taxon 352), and OTU38 (<i>Porphyromonas</i>), while OTU42 (<i>Burkholdiales</i>) and OTU6 (<i>Leptotrichia</i> HOT218) were more abundant within the AC group. At day 6, OTU18 (<i>Fusobacterium necrophorum</i>) and OTU37 (<i>Prevotella</i>) were significantly more abundant in individuals in the ILI/S<sup>flu+</sup> group.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>LEfSe analysis of abundant OTUs in ILI/S<sup><b>flu+</b></sup> and asymptomatic controls. The positive scale indicates the LDA score (Log10) for the most abundant taxa in the ILI/S<sup>flu<b>+</b></sup> group (green bars), while the negative scale represents the LDA scores for the prevalent taxa in the asymptomatic control group on days (<i>A</i>) 3 and (<i>B</i>) 6. Abbreviations: ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic; LDA, Linear Discriminant Analysis; LEfSe, LDA effect size; OTU, operational taxonomic units.</p>   </div> </div> <div class="alterationintheecologyofthepharyngealmicrobiomeduringinfluenza" title="sec"> <div class="title" tagx="title" title="title">
Alteration in the Ecology of the Pharyngeal Microbiome During Influenza</div> <p>Next, we examined the communities by phylum. Of 2057548 sequences, 19416 could not be classified, with the remainder distributed into 11 phyla. A total of 97.1% of the sequences belonged to only 5 phyla: <i>Actinobacteria</i> (7.9%), <i>Fusobacteria</i> (24.2%), <i>Firmicutes</i> (36.5%), <i>Bacteroidetes</i> (14.1%) or <i>Proteobacteria</i> (14.4%). We detected <i>Spirochaetes</i>, <i>Synergistetes</i>, <i>Tenericutes</i>, GN02, and 2 uncultured prokaryotes (TM7 and SR1) each at &amp;lt;1% of the total microbiota.</p> <p>The abundance of each phylum remained remarkably constant following the influenza challenge. There was a significant change in the <i>Bacteroidetes</i> levels, which increased in the ILI/S<sup>flu+</sup> group at day 3 post-infection, as compared to the AC group (<a href="#F5">Figure 5</a>). We also analyzed changes over the time. Within the ILI/S<sup>flu+</sup> group, there was a significant change in the abundance of <i>Actinobacteria</i>, which increased by day 6 compared to baseline (<i>P</i> &amp;lt; .01) and returned to basal levels by day 28 (<a href="#F6">Figure 6</a>). <i>Firmicutes</i> levels also increased between days 6 and 28 post-infection in the control group (<a href="#sup1">Supplementary Figure 6</a>), with an increase in <i>Streptococcus</i> (<a href="#sup1">Supplementary Figure 7</a>) that coincided with individuals returning to the community; this might reflect changes in social behaviors.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Relative abundance of common phyla in the human oropharynx. Comparison of the abundance of each phylum between individuals in the ILI/S<sup>flu<b>+</b></sup> group (+) and the asymptomatic controls (-). The phyla are indicated in each panel. Error bars represent the standard deviation. A non-parametric Kruskal-Wallis test for multiple comparisons was applied to identify the statistically significant differences in relative abundances between groups. *<i>P</i> &amp;lt; .05. Samples are from individuals prior to their entry to quarantine (baseline) and at days 3, 6, and 28 after the challenge, or not (in the control group). Abbreviations: dpi, days postâ€"influenza challenge; ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>Relative abundance of common phyla in the human oropharynx. The abundance of phyla in individuals in the ILI/S<sup>flu<b>+</b></sup> group (+). Phyla are indicated in each panel and error bars represent the standard deviation. A non-parametric Kruskal-Wallis test for multiple comparisons was applied to identify the statistically significant differences in relative abundances between groups. *<i>P</i> &amp;lt; .05; **<i>P</i> &amp;lt; .01. Samples are from individuals prior to their entry to quarantine (baseline) and at days 3, 6, and 28 after the challenge. Abbreviations: dpi, days postâ€"influenza challenge; ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic.</p>   </div> <p>With regard to the genus level, <i>Streptococcus</i> was a core component of the microbiome, accounting for 21.8% of bacteria, with <i>Fusobacterium</i> and <i>Prevotella</i> also abundant (comprising 15.4% and 9.9%, respectively). <i>Neisseria</i>, <i>Haemophilus</i>, and <i>Campylobacter</i> each accounted for 2â€"5% of the microbiota (<a href="#F7">Figure 7</a>). Differences were identified between the ILI/S<sup>flu+</sup> and AC groups (<a href="#F7">Figure 7</a>), with a significant increase in the relative abundance of <i>Prevotella</i> on days 3 and 28 in the challenged group (<i>P</i> &amp;lt; .05) and an increase in <i>Fusobacterium</i> by day 28 in the challenged group (<i>P</i> &amp;lt; .05). Within the ILI/S<sup>flu+</sup> group, a minor increase in <i>Actinomyces</i> was observed at day 6 post-infection (<i>P</i> &amp;lt; .05) and, in parallel, <i>Haemophilus</i> decreased (<i>P</i> &amp;lt; .01; <a href="#F8">Figure 8</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7.</span></a></div>   <p>Relative abundance of the common genera in the human oropharynx. Comparison of the abundance of each phylum in individuals between the ILI/S<sup>flu<b>+</b></sup> group (+) and the asymptomatic controls (-). Genera are indicated in each panel. Error bars represent the standard deviation. A non-parametric Kruskal-Wallis test for multiple comparisons was applied to identify the statistically significant differences in relative abundances between groups. *<i>P</i> &amp;lt; .05. Samples are from individuals prior to their entry to quarantine (baseline) and at days 3, 6, and 28 after the challenge, or not (in the control group). Abbreviations: dpi, days postâ€"influenza challenge; ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8.</span></a></div>   <p>Relative abundance of the common genera in the human oropharynx. The abundance of each genera in individuals within the ILI/S<sup>flu+</sup> group (+). The genera are indicated in each panel. Error bars represent the standard deviation. A non-parametric Kruskal-Wallis test for multiple comparisons was applied to identify the statistically significant differences in relative abundances between groups. *<i>P</i> &amp;lt; .05. Samples are from individuals prior to their entry to quarantine (baseline) and at days 3, 6, and 28 after the challenge. Abbreviations: dpi, days postâ€"influenza challenge; ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic.</p>   </div> </div> <div class="s.pneumoniaeandn.meningitidiscarriageisunalteredduringinfluenza" title="sec"> <div class="title" tagx="title" title="title"><i>S. pneumoniae</i> and <i>N. meningitidis</i> Carriage Is Unaltered During Influenza</div> <p>As 16S sequencing often cannot discriminate different species, we analyzed the carriage of 2 important respiratory pathogens: <i>S. pneumoniae</i> and <i>N. meningitidis</i>. The rate of colonization by encapsulated or non-encapsulated <i>N. meningitidis</i> was similar, with no difference between challenged and control individuals (<a href="#F9">Figure 9</a>). For <i>S. pneumoniae</i>, 16 of 36 influenza-challenged subjects (31%) and 12 of 23 controls (34%) were positive for <i>S. pneumoniae</i>, using qPCR, at day 3 (<i>P</i> = .8163).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 9.</span></a></div>   <p><i>Neisseria meningitidis</i> carriage following the influenza challenge. Meningococcal carriage in (<i>A</i>) the ILI/S<sup>flu+</sup> group and (<i>B</i>) the asymptomatic control group at different times following the challenge. The carriage of non-groupable <i>N. meningitidis</i> was unchanged during influenza infection and ranged between 11.4 and 13.5%, except for control samples from days 6 and 28. There was no change in the carriage of groupable <i>N. meningitidis</i> over time or between groups. Abbreviation: ILI/S<sup>flu+</sup>, subjects who received viral challenge, had laboratory evidence of influenza, and had an influenza-like illness/were symptomatic.</p>   </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>We performed the first prospective characterization of bacterial communities within the human URT during influenza infection. Previous studies have examined changes in the microbiota following challenges with other viruses [<a href="#CIT0025">25</a>, <a href="#CIT0026">26</a>], and found only minor changes in certain bacterial genera. Furthermore, cross-sectional studies have characterized the microbiomes of patients with influenza [<a href="#CIT0042">42</a>]. However we are not aware of a previous study that examined individuals during a defined influenza challenge.</p> <p>We detected slight perturbations in the upper respiratory microbiota following a challenge with influenza A. There was a high level of inter-individual variation in the upper respiratory microbiome, a common feature of human studies [<a href="#CIT0027">27</a>]. We found that the microbiota contains at least 11 phyla, with 5 (<i>Actinobacteria</i>, <i>Firmicutes</i>, <i>Fusobacteria</i>, <i>Bacterioidetes</i>, and <i>Proteobacteria</i>) constituting the core microbiome, as in a previous study [<a href="#CIT0026">26</a>], and with <i>Fusobacteria</i> highly prevalent (accounting for 24.2% of all OTUs) [<a href="#CIT0020">20</a>].</p> <p>The microbiotas were dominated by <i>Streptococcus</i> and <i>Fusobacteria</i>, which include species that can cause influenza-associated pulmonary disease. However, during the acquisition of influenza, the profile of the URT microbiome remained remarkably stable. Although we detected no large shift in the bacterial population at the genus level, there was an increase in the abundance of <i>Prevotella</i> in the ILI/S<sup>flu+</sup> group at days 3 and 28, a key time of secondary bacterial infection; although <i>P. melaninogenica</i> was associated with this increase, little is known about its role in respiratory disease.</p> <p>Overall, our study does not provide evidence that influenza infection has a marked impact in shaping the pharyngeal microbiome. However, certain factors might be responsible for this. Participants were healthy adults, who do not generally suffer secondary bacterial infections; studies in at-risk individuals might reveal more significant changes in the microbiome, although they would pose serious ethical issues. In addition, other effects might become evident in a larger study. Furthermore, 16S rRNAâ€"based methods are limited, lacking species-level resolution for particular genera, including <i>Streptococcus</i> and <i>Neisseria</i>. Even though cohort studies have established a relationship between influenza and the rate of pneumococcal carriage, we did not find differences in pneumococcal carriage following an influenza infection. For the meningococcus, we also examined capsule expression, as this virulence factor is only expressed by a subset of strains [<a href="#CIT0043">43</a>]. However, there was no alteration in the rate of carriage of encapsulated or non-encapsulated <i>N. meningitidis</i> with the advent of an influenza infection. We did not examine swabs for the presence of <i>S. aureus</i>, as this secondary bacteria pathogen follows influenza far less frequently than <i>S. pneumoniae</i> and is associated with severe disease rather than milder infections, such as that represented by our challenge.</p> <p>In conclusion, our findings demonstrate that the upper airway microbiota is not reprogrammed by influenza infection, with only minor perturbations in bacterial communities, similar to recent findings with rhinovirus [<a href="#CIT0044">44</a>]. Further studies are required to examine the cross-kingdom interactions that are responsible for secondary bacterial infection being a key player in the mortality and morbidity associated with influenza.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Data</div> <p>Supplementary materials are available at <i>Clinical Infectious Diseases</i> online. Consisting of data provided by the authors to benefit the reader, the posted materials are not copyedited and are the sole responsibility of the authors, so questions or comments should be addressed to the corresponding author.</p> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">ciy821_suppl_Supplementary-Table-1</span><div class="media" title="media"><a href="ciy821_suppl_supplementary-table-1.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Notes</div> <p><b><i>Disclaimer.</i></b>â€ƒThe views expressed in this manuscript are not necessarily those of the Department of Health, England.</p> <p><b><i>Financial support.</i></b>â€ƒThis work was supported by an award from the Wellcome Trust (to C. M. T.â€™s laboratory). The quarantine challenge study was supported by a Cooperative Grant from US Centers for Disease Control and Prevention (grant number 1U01P000497-01).</p> <p><b><i>Potential conflicts of interest.</i></b>â€ƒAll authors: No reported conflicts. All authors have submitted the ICMJE Form for Disclosure of Potential Conflicts of Interest. J. S. N.-V.-T. is currently on secondment to the Department of Health, England. Conflicts that the editors consider relevant to the content of the manuscript have been disclosed.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CIT0001" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCullers</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Secondary bacterial infections in influenza virus infection pathogenesis</span>. <span class="source" tagx="source" title="source">Curr Top Microbiol Immunol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">385</span>:<span class="fpage" tagx="fpage" title="fpage">327</span>â€"<span class="lpage" tagx="lpage" title="lpage">56</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25027822">25027822</a></span></span></li> <li tag="ref"><a name="CIT0002" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morens</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauci</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="mixed-article-title" title="mixed-article-title">The 1918 influenza pandemic: insights for the 21<sup>st</sup> century</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">195</span>:<span class="fpage" tagx="fpage" title="fpage">1018</span>â€"<span class="lpage" tagx="lpage" title="lpage">28</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17330793">17330793</a></span></span></li> <li tag="ref"><a name="CIT0003" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheng</span><span class="given-names" tagx="given-names" title="given-names">ZM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chertow</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ambroggio</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Autopsy series of 68 cases dying before and during the 1918 influenza pandemic peak</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">108</span>:<span class="fpage" tagx="fpage" title="fpage">16416</span>â€"<span class="lpage" tagx="lpage" title="lpage">21</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21930918">21930918</a></span></span></li> <li tag="ref"><a name="CIT0004" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">EU</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Fatalities associated with the 2009 H1N1 influenza A virus in New York city</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">50</span>:<span class="fpage" tagx="fpage" title="fpage">1498</span>â€"<span class="lpage" tagx="lpage" title="lpage">504</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20420514">20420514</a></span></span></li> <li tag="ref"><a name="CIT0005" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrewes</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laidlaw</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span></span><span class="mixed-article-title" title="mixed-article-title">A virus obtained from influenza patients</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">1933</span>; <span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">66</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="CIT0006" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCullers</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into the interaction between influenza virus and pneumococcus</span>. <span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">571</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16847087">16847087</a></span></span></li> <li tag="ref"><a name="CIT0007" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Community-acquired pneumonia due to pandemic A(H1N1)2009 influenzavirus and methicillin resistant <i>Staphylococcus aureus</i> co-infection</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e8705</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20090931">20090931</a></span></span></li> <li tag="ref"><a name="CIT0008" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cartwright</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuart</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaczmarski</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza A and meningococcal disease</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">1991</span>; <span class="volume" tagx="volume" title="volume">338</span>:<span class="fpage" tagx="fpage" title="fpage">554</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1678811">1678811</a></span></span></li> <li tag="ref"><a name="CIT0009" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakaletz</span><span class="given-names" tagx="given-names" title="given-names">LO</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral potentiation of bacterial superinfection of the respiratory tract</span>. <span class="source" tagx="source" title="source">Trends Microbiol</span><span class="year" tagx="year" title="year">1995</span>; <span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">110</span>â€"<span class="lpage" tagx="lpage" title="lpage">4</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7773588">7773588</a></span></span></li> <li tag="ref"><a name="CIT0010" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakaletz</span><span class="given-names" tagx="given-names" title="given-names">LO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smeesters</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial interactions in the respiratory tract</span>. <span class="source" tagx="source" title="source">Pediatr Infect Dis J</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">S121</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19918134">19918134</a></span></span></li> <li tag="ref"><a name="CIT0011" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCullers</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">The co-pathogenesis of influenza viruses with bacteria in the lung</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">252</span>â€"<span class="lpage" tagx="lpage" title="lpage">62</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24590244">24590244</a></span></span></li> <li tag="ref"><a name="CIT0012" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peltola</span><span class="given-names" tagx="given-names" title="given-names">VT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murti</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCullers</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza virus neuraminidase contributes to secondary bacterial pneumonia</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">192</span>:<span class="fpage" tagx="fpage" title="fpage">249</span>â€"<span class="lpage" tagx="lpage" title="lpage">57</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15962219">15962219</a></span></span></li> <li tag="ref"><a name="CIT0013" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakhnovich</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiser</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neuraminidase expressed by Streptococcus pneumoniae desialylates the lipopolysaccharide of Neisseria meningitidis and Haemophilus influenzae: a paradigm for interbacterial competition among pathogens of the human respiratory tract</span>. <span class="source" tagx="source" title="source">Infect Immun</span><span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">70</span>:<span class="fpage" tagx="fpage" title="fpage">7161</span>â€"<span class="lpage" tagx="lpage" title="lpage">4</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12438402">12438402</a></span></span></li> <li tag="ref"><a name="CIT0014" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nita-Lazar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Desialylation of airway epithelial cells during influenza virus infection enhances pneumococcal adhesion via galectin binding</span>. <span class="source" tagx="source" title="source">Mol Immunol</span><span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">65</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25597246">25597246</a></span></span></li> <li tag="ref"><a name="CIT0015" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quattroni</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucchesi</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Galectin-3 binds Neisseria meningitidis and increases interaction with phagocytic cells</span>. <span class="source" tagx="source" title="source">Cell Microbiol</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">1657</span>â€"<span class="lpage" tagx="lpage" title="lpage">75</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22827322">22827322</a></span></span></li> <li tag="ref"><a name="CIT0016" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siegel</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roche</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weiser</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza promotes pneumococcal growth during coinfection by providing host sialylated substrates as a nutrient source</span>. <span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">16</span>:<span class="fpage" tagx="fpage" title="fpage">55</span>â€"<span class="lpage" tagx="lpage" title="lpage">67</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25011108">25011108</a></span></span></li> <li tag="ref"><a name="CIT0017" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faust</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raes</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microbial interactions: from networks to models</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">538</span>â€"<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22796884">22796884</a></span></span></li> <li tag="ref"><a name="CIT0018" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlson</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Custers-Allen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Disordered microbial communities in the upper respiratory tract of cigarette smokers</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e15216</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21188149">21188149</a></span></span></li> <li tag="ref"><a name="CIT0019" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biesbroek</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keijser</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Seven-valent pneumococcal conjugate vaccine and nasopharyngeal microbiota in healthy children</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">201</span>â€"<span class="lpage" tagx="lpage" title="lpage">10</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24447437">24447437</a></span></span></li> <li tag="ref"><a name="CIT0020" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jakobsson</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jernberg</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersson</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SjÃ¶lund-Karlsson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jansson</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engstrand</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Short-term antibiotic treatment has differing long-term impacts on the human throat and gut microbiome</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e9836</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20352091">20352091</a></span></span></li> <li tag="ref"><a name="CIT0021" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiener-Kronish</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span></span><span class="mixed-article-title" title="mixed-article-title">From microbe to microbiota: considering microbial community composition in infections and airway diseases</span>. <span class="source" tagx="source" title="source">Am J Respir Crit Care Med</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">185</span>:<span class="fpage" tagx="fpage" title="fpage">691</span>â€"<span class="lpage" tagx="lpage" title="lpage">2</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22467799">22467799</a></span></span></li> <li tag="ref"><a name="CIT0022" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Steenhuijsen Piters</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huijskens</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wyllie</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Dysbiosis of upper respiratory tract microbiota in elderly pneumonia patients</span>. <span class="source" tagx="source" title="source">ISME J</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">97</span>â€"<span class="lpage" tagx="lpage" title="lpage">108</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26151645">26151645</a></span></span></li> <li tag="ref"><a name="CIT0023" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schloss</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalikin</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Decade-long bacterial community dynamics in cystic fibrosis airways</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">109</span>:<span class="fpage" tagx="fpage" title="fpage">5809</span>â€"<span class="lpage" tagx="lpage" title="lpage">14</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22451929">22451929</a></span></span></li> <li tag="ref"><a name="CIT0024" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sethi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murphy</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nariya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boushey</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span></span><span class="mixed-article-title" title="mixed-article-title">Airway microbiome dynamics in exacerbations of chronic obstructive pulmonary disease</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">52</span>:<span class="fpage" tagx="fpage" title="fpage">2813</span>â€"<span class="lpage" tagx="lpage" title="lpage">23</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24850358">24850358</a></span></span></li> <li tag="ref"><a name="CIT0025" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yong</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Profiling bacterial community in upper respiratory tracts</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">583</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25391813">25391813</a></span></span></li> <li tag="ref"><a name="CIT0026" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koeppel</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendley</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winther</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sale</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of the nasopharyngeal microbiota in health and during rhinovirus challenge</span>. <span class="source" tagx="source" title="source">Microbiome</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25028608">25028608</a></span></span></li> <li tag="ref"><a name="CIT0027" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kistler</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Booth</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradshaw</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wade</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial community development in experimental gingivitis</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">e71227</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23967169">23967169</a></span></span></li> <li tag="ref"><a name="CIT0028" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schloss</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westcott</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryabin</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities</span>. <span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">75</span>:<span class="fpage" tagx="fpage" title="fpage">7537</span>â€"<span class="lpage" tagx="lpage" title="lpage">41</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464">19801464</a></span></span></li> <li tag="ref"><a name="CIT0029" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clemente</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quince</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">2194</span>â€"<span class="lpage" tagx="lpage" title="lpage">200</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a></span></span></li> <li tag="ref"><a name="CIT0030" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chao</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nonparametric-estimation of the number of classes in a population</span>. <span class="source" tagx="source" title="source">Scand J Stat</span><span class="year" tagx="year" title="year">1984</span>; <span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">265</span>â€"<span class="lpage" tagx="lpage" title="lpage">70</span>.</span></li> <li tag="ref"><a name="CIT0031" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunge</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimating the number of species with CatchAll</span>. <span class="source" tagx="source" title="source">Pac Symp Biocomput</span><span class="year" tagx="year" title="year">2011</span>:<span class="fpage" tagx="fpage" title="fpage">121</span>â€"<span class="lpage" tagx="lpage" title="lpage">30</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21121040">21121040</a></span></span></li> <li tag="ref"><a name="CIT0032" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Measurement of diversity</span>. <span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">1949</span>; <span class="volume" tagx="volume" title="volume">163</span>:<span class="fpage" tagx="fpage" title="fpage">688</span>-688.</span></li> <li tag="ref"><a name="CIT0033" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shannon</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span></span><span class="mixed-article-title" title="mixed-article-title">A mathematical theory of communication</span>. <span class="source" tagx="source" title="source">Bell Syst Tech J</span><span class="year" tagx="year" title="year">1948</span>; <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">379</span>â€"<span class="lpage" tagx="lpage" title="lpage">423</span>.</span></li> <li tag="ref"><a name="CIT0034" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Good</span><span class="given-names" tagx="given-names" title="given-names">IJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">The population frequencies of species and the estimation of population parameters</span>. <span class="source" tagx="source" title="source">Biometrika</span><span class="year" tagx="year" title="year">1953</span>; <span class="volume" tagx="volume" title="volume">40</span>:<span class="fpage" tagx="fpage" title="fpage">237</span>â€"<span class="lpage" tagx="lpage" title="lpage">64</span>.</span></li> <li tag="ref"><a name="CIT0035" /><span class="label" tagx="label" title="label">35.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yue</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clayton</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span></span><span class="mixed-article-title" title="mixed-article-title">A similarity measure based on species proportions</span>. <span class="source" tagx="source" title="source">Commun Stat Theory Methods</span><span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">2123</span>â€"<span class="lpage" tagx="lpage" title="lpage">31</span>.</span></li> <li tag="ref"><a name="CIT0036" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lozupone</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="mixed-article-title" title="mixed-article-title">Species divergence and the measurement of microbial diversity</span>. <span class="source" tagx="source" title="source">FEMS Microbiol Rev</span><span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">557</span>â€"<span class="lpage" tagx="lpage" title="lpage">78</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18435746">18435746</a></span></span></li> <li tag="ref"><a name="CIT0037" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennett</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watkins</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identifying Neisseria species by use of the 50S ribosomal protein L6 (rplF) gene</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">52</span>:<span class="fpage" tagx="fpage" title="fpage">1375</span>â€"<span class="lpage" tagx="lpage" title="lpage">81</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24523465">24523465</a></span></span></li> <li tag="ref"><a name="CIT0038" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">Mda G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tondella</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCaustland</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evaluation and improvement of real-time PCR assays targeting lytA, ply, and psaA genes for detection of pneumococcal DNA</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">45</span>:<span class="fpage" tagx="fpage" title="fpage">2460</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537936">17537936</a></span></span></li> <li tag="ref"><a name="CIT0039" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Excoffier</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smouse</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quattro</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data</span>. <span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1992</span>; <span class="volume" tagx="volume" title="volume">131</span>:<span class="fpage" tagx="fpage" title="fpage">479</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1644282">1644282</a></span></span></li> <li tag="ref"><a name="CIT0040" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddison</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detecting isolation by distance using phylogenies of genes</span>. <span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1990</span>; <span class="volume" tagx="volume" title="volume">126</span>:<span class="fpage" tagx="fpage" title="fpage">249</span>â€"<span class="lpage" tagx="lpage" title="lpage">60</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2227384">2227384</a></span></span></li> <li tag="ref"><a name="CIT0041" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segata</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldron</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Metagenomic biomarker discovery and explanation</span>. <span class="source" tagx="source" title="source">Genome Biol</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">12</span>:<span class="fpage" tagx="fpage" title="fpage">R60</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21702898">21702898</a></span></span></li> <li tag="ref"><a name="CIT0042" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaban</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Links</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardy</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of the upper respiratory tract microbiomes of patients with pandemic H1N1 influenza</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">e69559</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23844261">23844261</a></span></span></li> <li tag="ref"><a name="CIT0043" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kugelberg</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gollan</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrance</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The influence of IS1301 in the capsule biosynthesis locus on meningococcal carriage and disease</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e9413</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20195528">20195528</a></span></span></li> <li tag="ref"><a name="CIT0044" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lehtinen</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hibberd</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MÃ¤nnikkÃ¶</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Nasal microbiota clusters associate with inflammatory response, viral load, and symptom severity in experimental rhinovirus challenge</span>. <span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">11411</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30061588">30061588</a></span></span></li> </ul> </div> </div>  </body></html>