<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="publisher-id" title="publisher-id">978-981-10-4511-0</span><span class="doi" title="doi">10.1007/978-981-10-4511-0</span><span class="nlm-ta" title="nlm-ta">Molecular Diagnostics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Molecular Diagnostics</span><div class="unknown" title="journal-subtitle" tag="journal-subtitle">
Part 2: Clinical, Veterinary, Agrobotanical and Food Safety Applications</div> </div><span class="isbn" tagx="isbn" title="isbn">978-981-10-4510-3</span><span class="isbn" tagx="isbn" title="isbn">978-981-10-4511-0</span></div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7121112">7121112</a></span><span class="publisher-id" title="publisher-id">3</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/978-981-10-4511-0_3">10.1007/978-981-10-4511-0_3</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Virology</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Pelt-Verkuil</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+3131648133776</span><span class="email" tagx="email" title="email">pelt.mdx.quality@ziggo.nl</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Leeuwen</span><span class="given-names" tagx="given-names" title="given-names">W.B.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">leeuwen.van.w@hsleiden.nl</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">te Witt</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="address" title="address"><span class="phone" tagx="phone" title="phone">+3131882353350</span><span class="email" tagx="email" title="email">tewit000@planet.nl</span></span><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.449761.9</span><span class="institution" title="institution">Faculty of Science &amp;amp; Technology, </span><span class="institution" title="institution">University of Applied Sciences Leiden, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.449761.9</span><span class="institution" title="institution">Faculty of Science &amp;amp; Technology, </span><span class="institution" title="institution">University of Applied Sciences Leiden, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boland</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">G.J.Boland@umcutrecht.nl</span></span><a href="#Aff4" /></span><span class="citation_author_institution" id="Aff4">[], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000000090126352</span><span class="institution-id" title="institution-id">grid.7692.a</span><span class="institution" title="institution">Division Laboratories and Pharmacy, Medical Microbiology, </span><span class="institution" title="institution">UMC Utrecht, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-07-07</span></span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">50</span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Molecular techniques have become indispensable in viral diagnostics. Current applications include: (1) The detection of (unknown) viral infections in clinical samples. (2) Quantitative monitoring of viral load. (3) Genotyping of viral strains. (4) Detection of mutations in the viral genome that are associated with viral resistance. Proper sample acquisition and sample transport, as well as accurate DNA or RNA isolation are a prerequisite for reliable test results in molecular diagnostics of viral infections. In-house as well as commercial assays can be used for the amplification of viral DNA or RNA for the detection of viral infections and viral load monitoring. Many virus species consist of several subspecies, genotypes or variants. This molecular variation has to be taken into account when applying molecular diagnostics. More complicated diagnostics for genotyping or the detection of mutations related to therapy failure often rely on sequencing, although for some viral targets commercial assays are available. In this chapter, applications are described in which molecular methods have become the most important form of viral diagnostics. Molecular test results have a direct impact on patient management and as such, results have to be reliable, standardized and reproducible. Therefore, quality control and standardization are important issues!</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Diagnosis</div> <div class="kwd" title="kwd">
Virus detection</div> <div class="kwd" title="kwd">
Variation</div> <div class="kwd" title="kwd">
Genotypes</div> <div class="kwd" title="kwd">
Resistance</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© Springer Nature Singapore Pte Ltd. 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">In the past, viral culturing techniques were used for the detection of viruses in clinical samples. Most of these culture methods have been replaced by molecular detection of viruses. The advantages of molecular detection, when compared to virus culture, are the higher diagnostic sensitivity and the much shorter time-to-result. A limitation of the molecular detection of viruses, when compared to conventional culture, is that the assays will only detect known viruses that were included in the search panel.</p> <p id="Par3">Screening for persisting viruses is usually performed by serological methods. The detection of antibodies is, however, an indirect method to detect an infection and there is usually no correlation with the severity of the disease. Viral load is generally a more accurate indicator of the severity of infection. Molecular techniques for quantitative viral load monitoring are applied to monitor (chronic) infections and to measure the effect of antiviral therapy.</p> <p id="Par4">For some viruses it may be important to detect both the presence of a virus and to determine the genotype or strain and specific resistance-associated mutations.</p> <p id="Par5">In this chapter applications of molecular detection of viral infection, viral load monitoring and more complex and often also less standardized methods, like genotyping and resistance testing in clinical diagnostics, are discussed.</p> </div> <div class="isolationofviralnucleicacidsfromclinicalsamples" title="sec"> <div class="title" tagx="title" title="title">
Isolation of Viral Nucleic Acids from Clinical Samples</div> <p id="Par6">Viral diagnostics are performed on a wide range of clinical materials, including (EDTA) plasma, whole blood, faeces, respiratory or genital swabs, cerebrospinal fluid (liquor), urine, tissue samples or biopsies, broncho-alveolar lavage fluids, etc. Extraction of the viral DNA or RNA is a critical step in the whole procedure of viral diagnostics. For some materials, pre-treatment of the sample is necessary to disrupt the tissue, release the virus or deactivate inhibitory factors. Inhibitory factors for PCR are for instance bile in faeces or haemoglobin in blood samples. Some viruses are not stable and clinical materials have to be processed within several hours to ensure optimal test results. Especially in case of viral load determination, accurate and timely sample processing is essential. This applies for instance to HIV viral load testing in plasma.</p> <p id="Par7">Buffers have been developed for some sample types, e.g. faecal samples, which preserve the nucleic acids present in the sample. For swabs, extraction tubes are available in which the swab can be placed directly to preserve the nucleic acids. In case of tissue samples or biopsies, the tissue has to be disrupted to release viral DNA or RNA e.g. by sonication, freezing, special DNA isolation kits for tissue, etc.</p> <p id="Par8">Extraction of viral nucleic acids can be done manually or automatically. Commercial manufacturers have developed a number of manual extraction kits. These kits vary in the method they use, the time required for extraction, the samples they are suitable for and the costs. In most methods, DNA or RNA is bound to a solid phase of glass fibres or silica, washed, released and precipitated. Manual extraction requires multiple manipulations and with the increasing number of samples, automated extraction is indispensable in diagnostic laboratories. Automated extraction instruments are manufactured by a number of different companies. By automation and standardization, the recovery of DNA and RNA from clinical samples is consistent and reproducible. More samples can be processed within a shorter period of time as compared with manual isolation, depending on the capacity of the instrument.</p> </div> <div class="moleculardiagnosticsofviruses" title="sec"> <div class="title" tagx="title" title="title">
Molecular Diagnostics of Viruses</div> <div class="viraldetectionmethods" title="sec"> <div class="title" tagx="title" title="title">
Viral Detection Methods</div> <p id="Par9">Several molecular techniques are used for the detection of viruses. Widely used are target-amplification methods based on the (reverse transcriptase) polymerase chain reaction ((RT-) PCR). Other target-amplification methods are nucleic acid sequence based amplification (NASBA) and transcription-mediated amplification (TMA). NASBA and TMA are isothermal target-amplification techniques and are described in Chap. 7 of Volume 1.</p> <p id="Par10">Other techniques are based on signal-amplification. One of these techniques is the branched-DNA (bDNA) assay. This technique is based on binding of the denatured target DNA or RNA on sequence-specific capture probes (bound to a micro well) and capture extenders. Second probes, called label extenders, also bind specifically and include a sequence that also binds to pre-amplifier probes. Multiple amplifiers bind to each pre-amplifier probe, amplifying the signal. This technique is also described in Chap. 7 of Volume 1. The advantages of signal amplifying techniques are that these methods are sensitive, specific, obtain a quantitative result and can be automated. However, these techniques are available for only a few targets (like HIV, hepatitis B and C) and are not suitable for development in your own laboratory.</p> <p id="Par11">The most popular technique in many of the virology laboratories is probably real-time PCR (qPCR). Commercially available qPCR based techniques are mainly developed for the most frequently detected targets in diagnostic laboratories. The advantage of qPCR for a molecular diagnostic (virology) laboratory is that the laboratory can develop its own tests for targets for which no CE-marked in vitro diagnostic (IVD) assays are available. qPCR can be used for the qualitative and quantitative detection of viruses. (Multiplex) qPCR can be used for the detection of many different viruses in one assay, for example for the detection of respiratory viruses like influenza virus, para-influenza virus, rhinovirus, respiratory syncytial virus, human metapneumo virus and coronaviruses (respiratory viral panel) in samples from patients with respiratory disease or for the detection of norovirus, rotavirus, enteroviruses and parechovirus (gastrointestinal viral panel) in patients with gastrointestinal disease.</p> <p id="Par12">In recent years attention has been paid to workflow optimization and result processing in order to decrease the number of manual actions, diminish deviations and errors and increase standardization and automation.</p> </div> <div class="primer/probedesign" title="sec"> <div class="title" tagx="title" title="title">
Primer/Probe Design</div> <p id="Par13">The target sequence for the design of primers and probes for the detection of viruses must be unique to the target. Preferably, the PCR primers should be able to identify the target region with high efficiency and specificity (see Vol. 1, Chap. 3). In the development of primers and probes for the detection of viruses, however, the variation of the viral genome has to be taken into account. Especially RNA-viruses, which lack proofreading in the replication of the virus, have high variability. A search in a DNA database, such as the National Center for Biotechnology Information (NCBI) database (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>), may reveal cross-reactivity with other nucleotide sequences. For clinical applications, it is necessary to validate the assay in the clinical material to which it will be applied, for instance faecal samples, to confirm absence of cross reactivity to normal flora in those specific samples. Mutations in the viral genome situated in the binding region of the primers and probe affect the binding efficiency of the primers and probes and consequently influence the efficiency of the amplification reaction. Both the location of the mutations and the number of mutations are important for the binding of the primers and probes and the amplification efficiency.</p> <p id="Par14">For the detection of viruses, primers and probes are preferably designed to anneal to conserved regions of the viral genome. More variable parts of the viral genome usually code for proteins that are targeted by the immune system, such as the envelope proteins or other structural proteins. Obviously, primers and probes designed to target these variable areas will have a greater chance to miss variants.</p> <p id="Par15">When multiple targets must be detected in one clinical sample within one PCR run, primers and probes should be used that bind under similar PCR conditions (temperature, concentration, etc.). The design of the primers and probes is important and self- and cross-hybridization has to be avoided. The more PCR reactions are performed in one-tube, the more difficult this will be. When combining different PCR reactions, these factors must be taken into account and comprehensive testing is mandatory when used for patient diagnostics (see Vol. 1, Chap. 5).</p> <div class="mismatch-toleranceinprimers" title="sec"> <div class="title" tagx="title" title="title">
Mismatch-Tolerance in Primers</div> <p id="Par16">Not much literature is available about the effect of mismatches in primers on amplification-efficiency. Mismatch-tolerance is dependent on the type of mismatch (T/A or G/C), the T<sub>m</sub> of the primer and the percentage G/C compared to A/T. The effect of mutations on qPCR efficiency also depends on the number and location of the mutations.</p> <p id="Par17">Mutations in the target DNA at the 3′-OH end of the primer, the site where the elongation starts, may compromise elongation and PCR efficiency (Vol. 1, Sect. 5.5). This effect of mismatching has been investigated and is directly related to nucleotides involved (Table <a href="#Tab1">3.1</a>). Studies show that mutations at the 3′-part of the primer, where the polymerase starts the elongation, lead to a suboptimal amplification efficiency and an underestimation of the amount of target. Mismatches A·A (A in the primer and A in the target), A·G, G·A, G·G and C·C lead to an underestimation of the amount of the target by a factor of 100. T·C, C·T, or T·T mismatches result in an underestimation by a factor of 10 to 100 of the target and A·C, C·A, T·G and G·T mismatches have nearly no effect. The inhibitory effect on target amplification decreases when the mutation is moved to the 5′-end of the primer and disappears after 5 nucleotides from the 3′-primer end. The effect of mutations in primers and primer binding sites also depends on the G/C content of the primers and the sequence itself (e.g. the G/C content on the 3′-end of the primer) and the annealing temperature. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 3.1</span> </div></p><p>Relative amplification efficiency with 3′—primer terminus mismatch</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2" /> <th align="left" rowspan="2" /> <th align="left" colspan="4">Primer 3′-terminus nucleotide</th> </tr> <tr> <th align="left">T</th> <th align="left">C</th> <th align="left">G</th> <th align="left">A</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="4">Template 3′—nucleotide</td> <td align="left">T</td> <td align="left">0.1–0.01</td> <td align="left">0.1–0.01</td> <td align="left">1.0–0.1</td> <td align="left">1.0</td> </tr> <tr> <td align="left">C</td> <td align="left">0.1–0.01</td> <td align="left">&amp;lt;0.01</td> <td align="left">1.0</td> <td align="left">1.0–0.1</td> </tr> <tr> <td align="left">G</td> <td align="left">1.0–0.1</td> <td align="left">1.0</td> <td align="left">&amp;lt;0.01</td> <td align="left">&amp;lt;0.01</td> </tr> <tr> <td align="left">A</td> <td align="left">1.0</td> <td align="left">1.0–0.1</td> <td align="left">&amp;lt;0.01</td> <td align="left">&amp;lt;0.01</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>The amount of PCR-product is related to the amount of PCR product with a primer with a perfect match</p> </div>  <p /> <p id="Par18">The effect of mutations also depends on the amplification enzyme. When using M-MLV/Taq polymerase for reverse-transcriptase, mutations in the reverse-primer binding site have nearly no effect, probably because of the low optimal temperature (37 ℃) at which M-MLV is active as reverse transcriptase, compared to AMV (optimal temperature 45 ℃). Mutations in the forward-primer binding site seem of less importance when rTth is used as polymerase. Not only the enzymes, but also other factors, like the constituents of the master mix (salt concentration) and hot start, may affect the influence of mutations in the primer binding sites.</p> <p id="Par19">In clinical practice, it is desirable to perform multiple PCR reactions on one sample in one PCR run, either as multiplex or in different wells (singleplex). This means that there is a beforehand established annealing temperature and fixed Mg<sup>2+</sup> concentration. Primers are selected that anneal optimally at this temperature, either by adjusting the length of the primer, by adjusting the sequence or by using nucleotide-analogues (see Vol. 1, Chap. 4).</p> </div> <div class="probedesign" title="sec"> <div class="title" tagx="title" title="title">
Probe Design</div> <p id="Par20">Probe-systems that are mostly used in diagnostics of viral infections with qPCR are hydrolysis probes and hybridisation probes: </p><ul> <li> <p id="Par21">Hydrolysis probes bind to the target sequence during the annealing phase. These probes are designed with a T<sub>m</sub> approx. 10 ℃ higher than the T<sub>m</sub> of the primers. During DNA synthesis, in the elongation phase, the hybridized probe is reached and hydrolysed by the 5′-3′-exonuclease activity of the polymerase. By breaking down the probe, the reporter-fluorochrome (VIC, FAM, TET, etc.) and quencher (TAMRA) are separated and ‘fluorescence resonance energy transfer (FRET)’ (the signal from the reporter) is released (see Vol. 1, Chap. 3). MGB-or LNA-probes are types of hydrolysis probes that are frequently used in qPCR assays for the diagnosis of viral infections (Vol. 1, Chap. 3). When these probes are designed well, they allow up to three mismatches in the target sequence without much loss of signal. Applications where mismatch-tolerance is needed are for instance qPCR for detection of variable viral targets. However, in virology, there are also applications where it is important that molecular variation is detected, for instance in the detection of viral variants with different pathogenicity or in the detection of mutations that are associated with resistance to antiviral medication. So, depending on the purpose of the assay, more or less mismatches can be tolerated.</p> </li> <li> <p id="Par22">Hybridisation probes used in viral diagnostics can be used as molecular ‘beacons’. Molecular ‘beacons’ are long probes with a loop structure, which contains the part of the probe that binds to the target region. Attached to the probe are two G-C rich, complementary regions that create the hairpin structure. One side of the probe is labelled with the fluorescent dye, the other with the quencher. These probes release a fluorescent signal upon hybridization to the target DNA and are not hydrolysed during the elongation phase in the PCR reaction. Molecular ‘beacon’ probes can be designed (T<sub>m</sub> of the loop compared to the T<sub>m</sub> of the stem) to be very sensitive to detect mutations, for example mutations that are associated with viral resistance. Other hybridization probes are double stranded probes containing a long strand (&amp;gt;40 nt) labelled with the reporter fluorochrome and a short strand homologous to the reporter probe (&amp;lt;16 nt), which is labelled with the quencher molecule. During elongation, the reporter probe will preferably bind to the target DNA and the probe with the quencher will be released, thereby releasing the signal. This method allows detection of the signal at much lower temperatures (until 37 ℃) and is much less sensitive to mutations in the target sequence without affecting the PCR efficiency. However, these probes are more difficult to design compared to hydrolysis probes.</p> </li> </ul> <p /> </div> </div> </div> <div class="moleculartechniquesforthedetectionofviralvariation" title="sec"> <div class="title" tagx="title" title="title">
Molecular Techniques for the Detection of Viral Variation</div> <p id="Par23">It is not only important to detect a virus, but often the analysis of genotypic characteristics of the virus is also important. Within one species, there are often strains with different characteristics and different pathogenicity, in need of other treatment strategies. An example is the group of enteroviruses. The genus enterovirus consists of the species echovirus, rhinovirus, poliovirus, enterovirus and coxsackievirus. Furthermore, these species consist of several subspecies and strains. In viral diagnostics, it is important to distinguish, for instance, the pathogenic poliovirus from other species within the genus of enteroviruses, but it may also be relevant to type viruses within the species. This can be done to obtain information on the relationship between different isolates in case of outbreaks, to identify strains with increased virulence or for epidemiological purposes. Another example is human papillomavirus (HPV). HPV is associated with cervical cancer. Within this genus there are high-risk HPV (hrHPV) strains, (e.g. HPV 16, 18, 31, 33, 45, 53, 66 and 68), probably hrHPV strains (e.g. HPV 26) and low-risk HPV strains (e.g. 40, 42, 72, 81 etc.). Distinguishing these (probably) hrHPV strains from the low-risk strains is essential for clinical diagnostics.</p> <p id="Par24">Another application is the molecular detection of resistance to antiviral medication. This field is becoming more important with the development of new antiviral medication and the occurrence of resistance variants of a variety viruses. Examples of viruses for which resistance to antiviral medication is important are HIV, hepatitis B, hepatitis C, herpes simplex virus type I and II and the influenza virus.</p> <p id="Par25">A few techniques are available to detect viral variation. The most frequently used and also the most informative one, is sequencing (part of) the viral genome. Sequencing can be used for several purposes, like genotyping (detection of variation) and the detection of mutations associated with resistance. A disadvantage is, however, that it is also the most time-consuming procedure. With the development of sequencing technology, it has become less expensive and more feasible for diagnostic laboratories (see Vol. 1, Chap. 8).</p> <p id="Par26">Sequences of viral genomes can be uploaded into online databases (e.g. NCBI) to determine the genotype of the virus. For some viruses, tools are available on the internet, e.g. the NCBI genotyping tool, Geno2Pheno or HIV-GRADE, either for genotyping or to quickly determine mutations associated with antiviral resistance. These tools are not available for all viruses. For other viruses, sequences can be uploaded into online databases and compared to viral sequences retrieved from GenBank or the Los Alamos database. Programs like NCBI blast or SeqScape can be used to compare sequences to detect mutations.</p> <p id="Par27">In some cases, information of only a selected part of the genome or of only a few mutations is necessary for subtyping or detection of antiviral resistance. For some applications, for instance genotyping of human papillomavirus (HPV), hepatitis B and hepatitis C, or for the detection of drug-resistant variants for hepatitis B, hybridization assays have been developed (LiPA). These assays consist of a qPCR combined with a hybridization assay (Vol. 1, Chaps. 4 and 5). The hybridization assay is performed on nitrocellulose strips that contain lines loaded with genotype-specific oligonucleotides. To detect drug-resistant variants, the strips contain wild type and mutant-specific oligonucleotide probes. The advantage of this method, compared to sequencing, is that more samples can be processed in the same time and that the evaluation of test results is less time-consuming. However, an obvious disadvantage is that mutations appearing outside the oligonucleotides used in the assay are not detected and newly emerged variants are not distinguished. Furthermore, natural variation of the virus is not visible.</p> <p id="Par28">Sequencing provides more information about the viral genome. Direct sequencing of a PCR product with the Sanger method does not yield information about variants that are present in low percentages in the total viral population. Variants that are present in less than ~25% of the total viral population are not detected. Next generation sequencing methods are more sensitive for the detection of minor viral populations. LiPA-assays also tend to be better in the detection of minority strains when compared to Sanger-sequencing methods.</p> </div> <div class="clinicalapplicationsofmoleculardiagnostics" title="sec"> <div class="title" tagx="title" title="title">
Clinical Applications of Molecular Diagnostics</div> <div class="moleculardetectionofviralinfections" title="sec"> <div class="title" tagx="title" title="title">
Molecular Detection of Viral Infections</div> <p id="Par29">In clinical diagnostics, the question is often: "which virus is causing this disease?�? When using molecular diagnostics to answer this question, the symptoms of the disease and patient characteristics also give directions to the most likely etiological agents. For instance, respiratory diseases can be caused by a number of viruses like influenza, parainfluenza, adenovirus, coronaviruses, rhinoviruses and human metapneumo virus, but in children, the respiratory syncytial virus is also a common cause. Infections of the upper respiratory tract may have different causes than infections of the lower respiratory tract. Additionally, patient-related factors like asthma or other chronic diseases, or medication, may influence the type and seriousness of the infection. Most diagnostic laboratories and diagnostic companies have developed specific respiratory panels to detect the most common causes of respiratory infections. Also for gastroenteritis and meningitis, there are multiple causes and usually a panel of viruses is tested. The most frequent cause of viral meningitis are enteroviruses, but occasionally, viral meningitis can be caused by other viruses like the herpes simplex virus, the mumps virus or arboviruses, which may require other precautions and treatment. Examples of viral causes of diseases are shown in Table <a href="#Tab2">3.2</a>. <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 3.2</span> </div></p><p>Common viral causes of diseases*</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Clinical characteristic</th> <th align="left">Viral causes</th> <th align="left" /> </tr> </thead> <tbody> <tr> <td align="left" rowspan="9">Upper respiratory tract (Mostly viral causes)</td> <td align="left">Rhinovirus (RNA)</td> <td align="left">&amp;gt;100 types, early and late</td> </tr> <tr> <td align="left">Coronavirus (RNA)</td> <td align="left">summer/autumn</td> </tr> <tr> <td align="left">Adenovirus (DNA)</td> <td align="left">5 types, winter and spring season</td> </tr> <tr> <td align="left">RSV** (RNA)</td> <td align="left">&amp;gt;50 types, winter and spring</td> </tr> <tr> <td align="left">Enteroviruses (RNA)</td> <td align="left">2 types, winter season, children</td> </tr> <tr> <td align="left">Influenza (RNA)</td> <td align="left">&amp;gt;90 types, summer and autumn</td> </tr> <tr> <td align="left">Parainfluenza (RNA)</td> <td align="left">Winter and spring</td> </tr> <tr> <td align="left">Human parechovirus (RNA)</td> <td align="left">Autumn, winter and spring 14 types</td> </tr> <tr> <td align="left">Other: EBV/HSV/MERS and SARS coronavirus</td> <td align="left" /> </tr> <tr> <td align="left" rowspan="2">Lower respiratory tract (Mostly bacterial causes, only viral causes mentioned)</td> <td align="left"> <p>Influenza</p> <p>RSV (children)</p> </td> <td align="left" rowspan="2">Lower respiratory infections with viral causes may occur in immune suppressed persons or as superinfections</td> </tr> <tr> <td align="left">Other: coronavirus, rhinoviruses, parainfluenzavirus, adenovirus, VZV</td> </tr> <tr> <td align="left" rowspan="4">Gastroenteritis (also bacterial and parasitic causes)</td> <td align="left">Norovirus (RNA)</td> <td align="left">Winter/outbreaks, mostly adults</td> </tr> <tr> <td align="left">Rotavirus (RNA)</td> <td align="left">Most common in children</td> </tr> <tr> <td align="left">Astrovirus (RNA)</td> <td align="left">Children and adults</td> </tr> <tr> <td align="left">Sapovirus (RNA)</td> <td align="left">Mostly adults</td> </tr> <tr> <td align="left" rowspan="7">Erythema/skin disease</td> <td align="left">Mumps (RNA)</td> <td align="left">Erythema</td> </tr> <tr> <td align="left">Rubella (RNA)</td> <td align="left">Erythema</td> </tr> <tr> <td align="left">Measles (RNA)</td> <td align="left">Erythema, measles</td> </tr> <tr> <td align="left">Parvovirus B19 (RNA)</td> <td align="left">Erythema</td> </tr> <tr> <td align="left">Herpes simplex (DNA)</td> <td align="left">Herpes virus, 2 types, vesicles</td> </tr> <tr> <td align="left">VZV (DNA)</td> <td align="left">Herpes virus, vesicles</td> </tr> <tr> <td align="left">HHV-6 (DNA)</td> <td align="left">Herpes virus, vesicles</td> </tr> <tr> <td align="left" rowspan="6">Hepatitis</td> <td align="left">Hepatitis A (RNA)</td> <td align="left">Acute hepatitis</td> </tr> <tr> <td align="left">Hepatitis B (DNA)</td> <td align="left">Acute and chronic</td> </tr> <tr> <td align="left">Hepatitis C (RNA)</td> <td align="left">Mostly chronic</td> </tr> <tr> <td align="left">Hepatitis D (RNA)</td> <td align="left">Subviral particle, only as coinfection with hepatitis B</td> </tr> <tr> <td align="left">Hepatitis E (RNA)</td> <td align="left">Mostly acute</td> </tr> <tr> <td align="left">Other: EBV, CMV</td> <td align="left">Not specific hepatitis virus but may cause hepatitis occasionally</td> </tr> <tr> <td align="left" rowspan="4">Meningitis</td> <td align="left">Enteroviruses</td> <td align="left" rowspan="4" /> </tr> <tr> <td align="left">Parechovirus</td> </tr> <tr> <td align="left">JC virus</td> </tr> <tr> <td align="left">Other: mumps, measles, poliovirus, HIV, herpesviruses</td> </tr> <tr> <td align="left" rowspan="5">Other viral causes</td> <td align="left">Parvovirus</td> <td align="left">Joint pain</td> </tr> <tr> <td align="left">Dengue virus</td> <td align="left">Travelers, fever and joint pain</td> </tr> <tr> <td align="left">HIV</td> <td align="left">Immunodeficiency, sexual transmission</td> </tr> <tr> <td align="left">Yellow fever virus</td> <td align="left">Travelers</td> </tr> <tr> <td align="left">West-Nile virus</td> <td align="left">Travelers</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*Table may not be complete and changes may occur as new viruses may evolve</p> <p>**Respiratory syncytial virus</p> </div>  <p /> <p id="Par30">The advantage of applying these panels is that all major causes of the infection are included and can be tested in a single assay. A disadvantage can be that the infection of a specific patient is not included in the assay or that the patient may have a variant of a specific virus that is not detected with the assay that is applied.</p> </div> <div class="quantitativedetectionofviruses—viralload" title="sec"> <div class="title" tagx="title" title="title">
Quantitative Detection of Viruses—Viral Load</div> <p id="Par31">Molecular techniques have made it possible to obtain a quantitative or semi quantitative result that can be related to a viral load. To obtain a reliable quantitative test result that can be compared with other test results, the amount (or input) of the clinical sample has to be standardized. Preferably, the test result relates to an international standard to calculate the viral load. Viral load is used to monitor herpes virus reactivation after transplantation and in some chronic viral infections like hepatitis B, hepatitis C and HIV. Since virus detection and viral monitoring have become essential in patient care, it is necessary that these techniques are performed frequently and on a regular basis in molecular diagnostic laboratories. Automation of the workflow from nucleic acid isolation, amplification, detection and processing of the results is essential for laboratories to maintain a high degree of standardization, to ensure that test results are reproducible.</p> <p id="Par32">There are three major reasons for quantitative detection of viral DNA or RNA. The first reason is that the amount of virus (‘viral load’), which is usually reflected in the amount of viral DNA or RNA present, can be an indication of the seriousness of the infection. This was first demonstrated for HIV by using a quantitative HIV-1 bDNA technique. An increasing amount of HIV-1 RNA genomes in plasma was directly correlated with a decrease in CD4+ T-cells, with disease progression and HIV-1 induced mortality. Since these findings were published, HIV-1 viral load monitoring is a major factor in the decision for antiviral treatment.</p> <p id="Par33">Since then, quantitative viral load determination is implemented in patient monitoring for other viral infections as well. Most people are infected with cytomegalovirus and Epstein Barr virus. These viruses usually do not cause problems, but occasionally cause viremia (virus detectable in blood) without symptoms. However, these viruses can cause major problems in patients after stem cell transplantation. Treatment against these viruses is installed when a certain viral load is reached. For EBV, it was shown that a viral load &amp;gt;10<sup>3</sup> copies/ml is predictive for the development of post-transplant lymphoproliferative disease. By starting antiviral therapy when this viral load is reached, but before symptoms appear (pre-emptive therapy), it was shown that the EBV-related mortality significantly reduced in these patients.</p> <p id="Par34">A second reason for quantification of the viral DNA or RNA is that the risk of viral transmission is directly related to the amount of virus present. Vertical transmission (from mother to child during pregnancy or at birth) is an example. Vertical transmission of HIV can be prevented either by decreasing the viral load during pregnancy by antiviral therapy, or by performing a caesarian section. For hepatitis B virus, it is also shown that the risk of vertical transmission is directly related to the viral load of the mother. Infection can be prevented by immediate active and passive vaccination of the newborn. However, the risk of vertical transmission is still present if the mother has a high viral load (&amp;gt;2 × 10<sup>8</sup> IU/ml). In that case, it is also recommended to start antiviral treatment in the last trimester of the pregnancy.</p> <p id="Par35">Alternatively, in case of needle stick incidents, the viral load is directly related to the chance of transmission of the virus. Needle stick accidents is a collective name for incidents in which health care workers accidently come into contact with patient fluids that may be contaminated with hepatitis B, hepatitis C or HIV. Accidental blood contact occurs frequently in hospitals. Measures taken after accidental contact comprise monitoring of viral load in both the victim of the needle stick accident and the patient and/or preventive antiviral treatment. Vice versa, health care workers may also infect patients. Several countries have taken measures to prevent transmission of viral infections to patients. For hepatitis B, several cases of transmission from health care workers to patients are known. Physicians with a high hepatitis B viral load are forbidden to perform medical procedures that pose a risk for patients in most countries.</p> <p id="Par36">A third reason for the quantification of viral DNA or RNA is monitoring the effect of antiviral treatment. The ultimate goal of antiviral treatment is to prevent damage to the host by the infection. Antiviral treatment is installed to reduce the viral load and to prevent this damage. Applications to monitor the effect of antiviral treatment are being used for HIV, hepatitis B, hepatitis C and herpes viruses, like cytomegalovirus and Epstein Barr virus. The primary goal is to decrease the viral load to an undetectable limit. Monitoring of the decrease in viral load directly after starting therapy can have a predictive value for treatment success. For some viruses, the absence of a certain decline in viral load may be an indication to stop or to switch treatment.</p> <p id="Par37">Some conditions influence the reliability of the quantification of a viral load. First, the sampling procedure and the handling of samples may influence the test results. For example, varicella zoster virus is a cell-associated virus. In vesicle fluid, the virus load can be low and may result in a false-negative PCR test result. Scraping of the bottom of the vesicle, where most virus-infected cells are present, may result in a very high viral load. Second, the viral load may not fluctuate too much in relation to the time that is needed to generate a quantitative test result. In acute viral infections, the virus load increases quickly to very high levels and decreases after that. For many acute viral respiratory infections it is not possible to show a relation between the viral load and the seriousness of the respiratory disease. Third, variation in virus load that is caused by the procedure has to be avoided. There are many factors that influence the outcome of the viral load: sample handling and freeze thawing, pipetting deviations, sample viscosity, batch-differences in reagent preparation and PCR-kits, etc. To obtain reliable quantitative test results, these factors have to be avoided.</p> <p id="Par38">Automation in quantitative PCR reduces analytical variation. In some quantitative PCR assays, viral load determinations are calibrated using an international reference strain, which makes comparison of viral loads between laboratories and between assays more reliable.</p> </div> <div class="genotypingandgenotypicdeterminationofviralresistance" title="sec"> <div class="title" tagx="title" title="title">
Genotyping and Genotypic Determination of Viral Resistance</div> <p id="Par39">Due to a combination of sloppy replication, immune pressure and other factors, viruses have evolved into different geno-or subtypes. Clinical features of genotypes may differ and sometimes even influence the choice of therapy. Viral resistance can occur during treatment and determines the course of the infection and the choice of medication. Often, only small variations, or even single nucleotide point mutations (SNP), may have a great impact on the pathogenicity, replicative activity and/or susceptibility for treatment. Mutations are usually represented as the amino acid that is replaced and the position, e.g. Q151 M is a mutation in the reverse transcriptase gene of HIV, in which the glutamine at position 151 in the protein is replaced by a methionine. A few examples are discussed in the following subsections.</p> <div class="hiv-1" title="sec"> <div class="title" tagx="title" title="title">
HIV-1</div> <p id="Par40">Antiviral treatment for HIV consists of a combination of antiviral drugs: </p><ul> <li> <p id="Par41">Nucleotide/nucleoside reverse transcriptase inhibitors (NRTI’s)</p> </li> <li> <p id="Par42">Non-nucleotide reverse transcriptase inhibitors (NNRTI’s)</p> </li> <li> <p id="Par43">Protease inhibitors (PI’s)</p> </li> <li> <p id="Par44">Integrase inhibitors (INSTI’s)</p> </li> <li> <p id="Par45">Fusion inhibitors</p> </li> <li> <p id="Par46">CCR5 co-receptor antagonist (maraviroc)</p> </li> <li> <p id="Par47">Pharmacokinetic booster agents like ritonavir.</p> </li> </ul> <p /> <p id="Par48">Genotypic resistance testing is usually performed on the reverse transcriptase and protease genes of HIV and it is performed in case of suboptimal viral reduction and to assist in the selection of active drugs. Mutations that lead to high-level reduced susceptibility to integrase inhibitors have been described. Pre-treatment resistance testing is advised, due to the risk of transmission of resistant virus strains.</p> <p id="Par49">Some mutations are able to individually induce reduced susceptibility to antiviral drugs, whereas some mutations invoke resistance only in combination with other mutations. For anti-HIV medication, the terms low- or high-genetic barrier are often used to indicate whether resistance can evolve easily or requires more than one mutation. Resistance is usually tested by sequencing part of the genome. Various online tools can assist the laboratory workers with the analysis of genotyping test results. The international AIDS society (USA) maintains a list of resistance-associated mutations (<a href="http://www.iasusa.org/resistance_mutations">http://www.iasusa.org/resistance_mutations</a>, login required) and Stanford University maintains a HIV Drug resistance database (<a href="http://hivdb.stanford.edu">http://hivdb.stanford.edu</a>).</p> </div> <div class="hepatitisb" title="sec"> <div class="title" tagx="title" title="title">
Hepatitis B</div> <p id="Par50">The hepatitis B virus consists of genotypes A through H. The small differences between these genotypes are responsible for the differences in the risk of the development of liver carcinoma and the response to treatment. Treatment of hepatitis B with antiviral medication aims at long-term suppression of the hepatitis B virus and at reducing related morbidity and mortality. Thus far, the differences in genotypes have only been shown to have minor consequences for treatment strategies, but they are of interest for predictions. For instance, hepatitis B genotypes A and B seem to respond relatively well to treatment with Interferon-alpha, compared to, for instance, genotype C and D. A physician tends to preferably treat genotypes A or B with Interferon. Treatment with Interferon aims at improving the antiviral response of the immune system, at reducing viral susceptibility of cells by enhancing Major Histocompatibility Complex (MHC)-antigen expression and at inducing cellular proteins that are important in the natural defense of cells against viruses. This leads to immune control of the chronic hepatitis B infection.</p> <p id="Par51">Direct-acting antivirals (DAA’s, nucleotide-or nucleoside-analogues) that are available for the treatment of an infection with the hepatitis B virus are lamivudine, adefovir, telbivudine, tenofovir and entecavir. One mutation, called the YMDD motive, is associated with resistance to lamivudine. The YMDD motive is a conserved sequence in the RNA-dependent DNA polymerase gene of hepatitis B virus consisting of tyrosine (Y), methionine (M), and two aspartates (DD). Replacement of the methionine by isoleucine (YIDD) or valine (YVDD) leads to resistance. Since the polymerase and hepatitis B surface antigen genes overlap, replacement of nucleotides may also lead to changes in the surface antigen. Therefore, the mutations also have an impact on the production and secretion of the HBsAg. Lamivudine has long been the only treatment option for hepatitis B, but now has been replaced by other more potent antivirals, like entecavir and tenofovir. Entecavir, however, is less effective in patients who have been pretreated with lamivudine and who carry the YMDD variant of hepatitis B. Genotyping and resistance testing of hepatitis B can be performed by sequencing and comparing the sequence to reference strains, but LiPA assays have also been developed for genotyping and for the detection of mutations associated with resistance.</p> </div> <div class="hepatitisc" title="sec"> <div class="title" tagx="title" title="title">
Hepatitis C</div> <p id="Par52">As an alternative to HIV and hepatitis B treatment, which aims at long-term suppression of the virus, the treatment of the hepatitis C virus aims to totally clear the virus from the host. In the past decades, hepatitis C was treated with a combination of interferon-alpha and ribavirin. Nowadays, an increasing number of direct-acting antivirals (DAA’s) are available. Several viral proteins involved in the HCV lifecycle, such as the non-structural (NS) 3/4A serine protease, the NS5B RNA-dependent RNA polymerase (RdRp) and the NS5A protein, are targets for these drugs. NS5A plays a role in the formation of a replication complex of the virus and in the release of new viral particles. The Hepatitis C virus has 7 major genotypes, all with different prevalence and different responses to treatment. Genotypes 1, 2, 3 and 4 are most prevalent in western countries. Genotyping can be performed with a LiPA assay, but sequence-based assays have also been developed. First-generation protease-inhibitors telaprevir and boceprevir are active against hepatitis C genotype 1, but they are less effective and therefore not registered for other genotypes. New NS3-protease-inhibitors are registered for treatment of genotype 1 and 4. Polymerase inhibitors and NS5A inhibitors show less genotypic specificity. Since treatment with one antiviral is often ineffective, combination therapy is always given. The choice of medicine largely depends on the hepatitis C genotype.</p> <p id="Par53">Antiviral resistance does not appear to be a serious problem in treatment with NS5B polymerase-inhibitors. For NS5A-inhibitors, some clinically relevant mutations have been described, like M28T, Q30E/H/R, L31 M/V and Y93H. Drug resistance to protease-inhibitors seems more complex. Q80 K is a mutation that is related to resistance to simeprevir, but only in genotype 1a-infected patients (Fig. <a href="#Fig1">3.1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3.1</span></a></div>   <p>Alignment of a part of HCV protease with reference strain HCV gt 1a N77—translation of nucleotides to amino acids</p>   </div> <p /> <p id="Par54">R155 K/T is a mutation that appears to invoke resistance to almost all protease-inhibitors, including to protease-inhibitors still in development in genotype 1a-infected patients. The D168 V mutation seems to be prevalent mostly in genotype 1b-infected patients and is related to resistance. Mutations associated with resistance may be present before treatment or develop during treatment. Pretreatment sequence analysis and monitoring of response to antiviral treatment will give more information that may be of help in future therapy development.</p> <p id="Par55">Detection of drug-resistant variants is done with sequence-based assays. Sequence variation between the genotypes of hepatitis C is large and makes it difficult to design primers for amplification and sequencing of the protease-, NS5A or polymerase genes of hepatitis C. In Fig. <a href="#Fig2">3.2</a> it is shown that even within a hepatitis C genotype and subtype, there is considerable sequence variation (Fig. <a href="#Fig2">3.2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3.2</span></a></div>   <p>HCV NS5A sequence variation within HCV genotype 1a</p>   </div> <p /> </div> <div class="herpesviruses" title="sec"> <div class="title" tagx="title" title="title">
Herpes Viruses</div> <p id="Par56">Herpes simplex virus 1 and 2 (HSV 1 and 2), varicella zoster virus (VZV) and cytomegalovirus (CMV) belong to the herpes virus family, which can be treated with specific antiviral drugs. Resistance has been described for this family of viruses. Sequencing part of the genome is necessary to detect involved mutations.</p> <p id="Par57">HSV 1 and 2 cause lesions in the mouth that are usually not harmful. However, these lesions can become severe in immune suppressed patients and systemic illness may occur. For HSV 1 and 2, 7–14% of patients that receive treatment with acyclovir, or similar medications like valaciclovir and famciclovir, become resistant. Up to 95% of resistance to acyclovir is associated with mutations in the thymidine-kinase gene (UL23). Viral thymidine-kinase phosphorylates acyclovir, which is a necessary step in the antiviral mechanism of acyclovir, before it can act as a nucleotide analogue to disturb viral DNA synthesis. Acyclovir-resistant variants may still be sensitive to foscarnet. Foscarnet blocks DNA polymerase, thereby inhibiting viral DNA synthesis. Mutations in UL30 (polymerase-gene) are associated with resistance to foscarnet. In both genes, many polymorphisms may be present. Resistance to acyclovir is often associated with insertions or deletions in the thymidine-kinase gene. Also, some single mutations are associated with resistance. Thorough analysis of the gene sequences, comparisons to reference strains and to mutations found in literature are necessary for analysis.</p> <p id="Par58">The same genes (thymidine kinase and polymerase) are involved in resistance of VZV to antiviral therapy. The first infection with varicella zoster virus gives chickenpox. Varicella zoster virus remains latently present in neural axons of the central nervous system and can reactivate, causing shingles. Treatment is similar to herpes simplex virus treatment, with acyclovir or foscarnet. Resistance testing is also performed by sequencing (part of) the thymidine kinase and polymerase genes and comparing to reference sequences.</p> <p id="Par59">First choice of treatment for CMV is ganciclovir. Ganciclovir is a guanosine analogue that exerts its antiviral effect by inhibiting CMV DNA polymerase. Ganciclovir is phosphorylated by a viral phosphotransferase coded on UL97. The CMV DNA polymerase is a product of UL54. Mutations in both genes can lead to resistance. However, mutations in UL97 are significantly more common than mutations in UL54. Mutations in UL54 are associated with higher levels of resistance to ganciclovir or cross-resistance to cidofovir or foscarnet.</p> </div> <div class="influenzavirus" title="sec"> <div class="title" tagx="title" title="title">
Influenza Virus</div> <p id="Par60">Influenza is a member of the family of Orthomyxoviridae, of which the virus species influenza A, influenza B and influenza C can cause our classical ‘flu’. The influenza A species consists of different strains, designated according to the structure of their haemagglutinin (HA) and neuraminidase (NA) genes. These are the most variable viral proteins and are involved in attachment to the target cell and fusion with the endosomal membrane. Influenza viruses have a segmented RNA-genome (8 segments), the NA and HA-coding sequences are on different segments and can be exchanged. HA is encoded on segment nr 4 and NA on segment nr 6. Within influenza A, there are at least 16 HA-encoding segments known, of which segments numbered H1–H5 are known as human variants, and 9 NA-variants. Recombination of influenza variants may lead to new combinations of H- and N-genes (antigenic shift), whereas mutations in the H- and N-genes lead to new variants with slightly different properties (antigenic drift). This antigenic drift causes that for instance two A(H3N2) strains may have slightly different antigenic properties. Antigenic drift of less pathogenic viral strains, however, may lead to more pathogenic strains. Usually, every winter season there are one or a few dominant influenza strains active, often one or two A-strains and one B-strain. Within these strains, different isolates can be distinguished.</p> <p id="Par61">Compared to influenza A, influenza B is genetically less divers, infection is nearly only restricted to humans and influenza B is thought to be less pathogenic. Influenza C is far less common than influenza A and B, but can cause severe illness and is incidentally associated with outbreaks.</p> <p id="Par62">In viral diagnostics, a pan-influenza PCR is used, with primers that detect influenza species A (Matrix gene segment) and a different PCR to detect influenza B. To monitor influenza activity during the winter season and to estimate the effectiveness of the vaccine, viral strains are typed for H and N-gene segments every year. Typing can be performed by reverse transcriptase qPCR with type-specific primers and probes. Some commercial assays are also available. Strain typing is used to predict upcoming strains and to determine the strains that need to be included in the vaccine. The virus mutates from year to year, making existing vaccines ineffective.</p> <p id="Par63">For more specific information, the virus has to be sequenced. The whole genome consists of around 2,800,000 nucleotides. Within influenza strains, different isolates can be distinguished, revealing the highly variable genetic structure of this virus. In patient care, the strain type is related to sensitivity for antiviral treatment. Most influenza A and B strains are sensitive to oseltamivir and zanamivir, two neuraminidase inhibitors. Occasionally, resistant strains are found, sometimes related to long-term treatment of influenza infection in immunocompromised patients. Resistance to oseltamivir has also been reported during the 2007–2008-winter season, due to a mutation on position 275 of the neuraminidase gene, resulting in the substitution of a histidine to a tyrosine (H275Y). Strains that have this mutation, however, are still susceptible to zanamivir.</p> </div> </div> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par64">Molecular diagnostics have become indispensable in the diagnosis of viral infections. qPCR techniques are not only used for the detection of viruses, but also for viral load determination. Whereas chronic viral infections are usually detected by serological methods to test immunity, molecular techniques have become important in the monitoring of chronic viral infections and in the response to antiviral treatment.</p> <p id="Par65">By applying molecular techniques we have learned more about viral variation and the different genotypes that exist within virus species. This variation has clinical impact. For some viruses, specific mutations in the viral genome can be related to resistance to antiviral medication. More rapid and cheaper molecular techniques, like next generation sequencing, will be of aid in exploring this field more extensively.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
Recommended Literature</div> <li tag="ref"><a name="CR1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Annels</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Management of Epstein-Barr Virus (EBV) reactivation after allogeneic stem cell transplantation by simultaneous analysis of EBV DNA load and EBV-specific t cell reconstitution</span><span class="source" tagx="source" title="source">Clin Infect Dis</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">1743</span><span class="lpage" tagx="lpage" title="lpage">1748</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/503838">10.1086/503838</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16705581">16705581</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christopherson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The effects of internal primer-template mismatches on RT-PCR: HIV-1 model studies</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="year" tagx="year" title="year">1997</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">654</span><span class="lpage" tagx="lpage" title="lpage">658</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/25.3.654">10.1093/nar/25.3.654</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9016609">9016609</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espy</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Real-time PCR in clinical microbiology: applications for routine laboratory testing</span><span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">19</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">165</span><span class="lpage" tagx="lpage" title="lpage">256</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/CMR.19.1.165-256.2006">10.1128/CMR.19.1.165-256.2006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418529">16418529</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Large-scale sequencing of human influenza reveals the dynamic nature of viral genome evolution</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">437</span><span class="fpage" tagx="fpage" title="fpage">1162</span><span class="lpage" tagx="lpage" title="lpage">1166</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature04239">10.1038/nature04239</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16208317">16208317</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacKaya</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular detection and typing of influenza viruses: are we ready for an influenza pandemic?</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">194</span><span class="lpage" tagx="lpage" title="lpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2008.02.005">10.1016/j.jcv.2008.02.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18359662">18359662</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schutten</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical utility of viral quantification as a tool for disease monitoring</span><span class="source" tagx="source" title="source">Expert Rev Mol Diagn</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">1</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">153</span><span class="lpage" tagx="lpage" title="lpage">162</span><span class="pub-id"><a href="https://dx.doi.org/10.1586/14737159.1.2.153">10.1586/14737159.1.2.153</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11901810">11901810</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expanded classification of hepatitis C virus into 7 genotypes and 67 subtypes: updated criteria and genotype assignment web resource</span><span class="source" tagx="source" title="source">Hepatology</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">59</span><span class="issue" tagx="issue" title="issue">1</span><span class="fpage" tagx="fpage" title="fpage">318</span><span class="lpage" tagx="lpage" title="lpage">327</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/hep.26744">10.1002/hep.26744</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24115039">24115039</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuyver</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Typing of hepatitis C virus isolates and characterization of new subtypes using a line probe assay</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">74</span><span class="issue" tagx="issue" title="issue">6</span><span class="fpage" tagx="fpage" title="fpage">1093</span><span class="lpage" tagx="lpage" title="lpage">1102</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-74-6-1093">10.1099/0022-1317-74-6-1093</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8389799">8389799</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vercauteren</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandamme</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Algorithms for the interpretation of HIV-1 genotypic drug resistance information</span><span class="source" tagx="source" title="source">Antiviral Res</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">71</span><span class="issue" tagx="issue" title="issue">2–3</span><span class="fpage" tagx="fpage" title="fpage">335</span><span class="lpage" tagx="lpage" title="lpage">342</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2006.05.003">10.1016/j.antiviral.2006.05.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16782210">16782210</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO information for molecular diagnosis of influenza virus—update May 2015. Available at: <a href="http://www.who.int/influenza/gisrs_laboratory/molecular_diagnosis_influenza_virus_humans_update_201403rev201505.pdf%3fua%3d1">http://www.who.int/influenza/gisrs_laboratory/molecular_diagnosis_influenza_virus_humans_update_201403rev201505.pdf?ua=1</a>. Accessed 28 Jul 2015.</span></li> </ul> </div> </div>  </body></html>