<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/01/18/249995.atom" />
<meta name="HW.pisa" content="biorxiv;249995v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences" />
<meta name="DC.Identifier" content="10.1101/249995" />
<meta name="DC.Date" content="2018-01-18" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Motivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.

Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.

Availability: HCAtk and pyHCA are available at &lt;https://github.com/T-B-F/pyHCA&gt; under the CeCILL-C license." />
<meta name="DC.Contributor" content="Tristan Bitard-Feildel" />
<meta name="DC.Contributor" content="Isabelle Callebaut" />
<meta name="article:published_time" content="2018-01-18" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Motivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.&lt;/p&gt;&lt;p&gt;Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.&lt;/p&gt;&lt;p&gt;Availability: HCAtk and pyHCA are available at https://github.com/T-B-F/pyHCA under the CeCILL-C license.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;249995v1" />
<meta name="citation_id" content="249995v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/249995v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/249995v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/249995v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995.full.pdf" />
<meta name="citation_doi" content="10.1101/249995" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="249995" />
<meta name="citation_author" content="Tristan Bitard-Feildel" />
<meta name="citation_author_institution" content="Sorbonne Université, UPMC Université Paris 6" />
<meta name="citation_author_institution" content="CNRS UMR7590, Sorbonne Université, Université Pierre et Marie Curie" />
<meta name="citation_author" content="Isabelle Callebaut" />
<meta name="citation_author_institution" content="CNRS UMR7590, Sorbonne Université, Université Pierre et Marie Curie" />
<meta name="citation_reference" content="Arcus V. L., McKenzie J. L., Robson J., and Cook G. M. The pin-domain ribonucleases and the prokaryotic vapbc toxin-antitoxin array. Protein Engineering Design and Selection, 24(1–2):33–40, Jan 2011." />
<meta name="citation_reference" content="Bateman A., Holden M. T. G., and Yeats C. The g5 domain: a potential n-acetylglucosamine recognition domain involved in biofilm formation. Bioinformatics, 21(8):1301–1303, 2005." />
<meta name="citation_reference" content="Bienkiewicz E. A., Adkins J. N., and Lumb K. J. Functional consequences of preorganized helical structure in the intrinsically disordered cell-cycle inhibitor p27kip1. Biochemistry, 41(3):752–759, 2002." />
<meta name="citation_reference" content="Bitard-Feildel T. and Callebaut I. Exploring the dark foldable proteome by considering hydrophobic amino acids topology. Scientific Reports, 7:41425, 2017." />
<meta name="citation_reference" content="Bitard-Feildel T., Heberlein M., Bornberg-Bauer E., and Callebaut I. Detection of orphan domains in drosophila using \hydrophobic cluster analysis.” Biochimie, 119:244–253, 2015." />
<meta name="citation_reference" content="Callebaut I., Labesse G., Durand P., Poupon A., Canard L., Chomilier J., Henrissat B., and Mornon J. P. Deciphering protein sequence information through hydrophobic cluster analysis (hca): current status and perspectives. Cellular and molecular life sciences, 53(8):621–645, Sep 1997." />
<meta name="citation_reference" content="Durbin R. M., Altshuler D. L., Durbin R. M., Abecasis G. R., Bentley D. R., Chakravarti A., Clark A. G., Collins F. S., De La Vega F. M., Donnelly P., and et al.A map of human genome variation from population-scale sequencing. Nature, 467(7319):1061–1073, Oct 2010." />
<meta name="citation_reference" content="Eudes R., Le Tuan K., Delettré J., Mornon J.-P., and Callebaut I. A generalized analysis of hydrophobic and loop clusters within globular protein sequences. BMC Structural Biology, 7(1):2, 2007." />
<meta name="citation_reference" content="Faure G. and Callebaut I. Comprehensive repertoire of foldable regions within whole genomes. PLoS Computational Biology, 9(10):e1003280, Oct 2013." />
<meta name="citation_reference" content="Faure G. and Callebaut I. Identification of hidden relationships from the coupling of hydrophobic cluster analysis and domain architecture information. Bioinformatics, 29(14):1726–33, Jul 2013." />
<meta name="citation_reference" content="Finn R. D., Coggill P., Eberhardt R. Y., Eddy S. R., Mistry J., Mitchell A. L., Potter S. C., Punta M., Qureshi M., Sangrador-Vegas A., and et al. The pfam protein families database: towards a more sustainable future. Nucleic acids research, 44(D1):D279–85, Jan 2016." />
<meta name="citation_reference" content="Gaboriaud C., Bissery V., Benchetrit T., and Mornon J. Hydrophobic cluster analysis: An efficient new way to compare and analyse amino acid sequences. FEBS Letters, 224(1):149–155, Nov 1987." />
<meta name="citation_reference" content="Galea C. A., Nourse A., Wang Y., Sivakolundu S. G., Heller W. T., and Kriwacki R. W. Role of intrinsic flexibility in signal transduction mediated by the cell cycle regulator, p27 kip1. Journal of molecular biology, 376(3):827–38, 2008." />
<meta name="citation_reference" content="Gruszka D. T., Mendonca C. A. T. F., Paci E., Whelan F., Hawkhead J., Potts J. R., and Clarke J. Disorder drives cooperative folding in a multidomain protein. Proceedings of the National Academy of Sciences of the United States of America, 113(42):11841–11846, 2016." />
<meta name="citation_reference" content="Hayes P. L., Lytle B. L., Volkman B. F., and Peterson F. C. The solution structure of znf593 from homo sapiens reveals a zinc finger in a predominately unstructured protein. Protein Science, 17(3):571–576, 2008." />
<meta name="citation_reference" content="Martin M. W., Newcomb J., Nunes J. J., Bemis J. E., Mcgowan D. C., White R. D., Buchanan J. L., Dimauro E. F., Boucher C., Faust T., and et al.Discovery of novel 2,3-diarylfuro[2,3-b]pyridin-4-amines as potent and selective inhibitors of lck: Synthesis, sar, and pharmacokinetic properties. 17:2299–2304, 2007." />
<meta name="citation_reference" content="Mitchell A., Chang H.-Y., Daugherty L., Fraser M., Hunter S., Lopez R., McAnulla C., McMe-namin C., Nuka G., Pesseat S., and et al. The interpro protein families database: the classification resource after 15 years. Nucleic acids research, 43(Database issue):D213–21, Jan 2015." />
<meta name="citation_reference" content="Oates M. E., Stahlhacke J., Vavoulis D. V., Smithers B., Rackham O. J. L., Sardar A. J., Zaucha J., Thurlby N., Fang H., and Gough J. The superfamily 1.75 database in 2014: a doubling of data. Nucleic acids research, 43(Database issue):D227–33, Jan 2015." />
<meta name="citation_reference" content="Peng Z., Mizianty M. J., Xue B., Kurgan L., and Uversky V. N. More than just tails: intrinsic disorder in histone proteins. Molecular BioSystems, 8(7):1886, 2012." />
<meta name="citation_reference" content="Piovesan D., Tabaro F., Mičetić I., Necci M., Quaglia F., Oldfield C. J., Aspromonte M. C., Davey N. E., Davidović R., Dosztányi Z., and et al.Disprot 7.0: A major update of the database of disordered proteins. Nucleic Acids Research, 45(D1):D219–D227, 2017." />
<meta name="citation_reference" content="Piovesan D., Walsh I., Minervini G., and Tosatto S. C. Fells: fast estimator of latent local structure. Bioinformatics, 33(12):1889–1891, Jun 2017." />
<meta name="citation_reference" content="Polyak K., Lee M., Erdjument-Bromage H., and Koff A. Cloning of p27 kip1, a cyclin-dependent kinase inhibitor and a potential mediator of extracellular antimitogenic signals. Cell, 76:59–66, 1994." />
<meta name="citation_reference" content="Ramakrishnan V., Finch J. T., Graziano V., Lee P. L., and Sweet R. M. Crystal structure of globular domain of histone h5 and its implications for nucleosome binding. Nature, 362(6417):219–223, 1993." />
<meta name="citation_reference" content="Rebehmed J., Quintus F., Mornon J.-P., and Callebaut I. The respective roles of polar/nonpolar binary patterns and amino acid composition in protein regular secondary structures explored exhaustively using hydrophobic cluster analysis. Proteins: Structure, Function, and Bioinformatics, 84(5):624–638, May 2016." />
<meta name="citation_reference" content="Remmert M., Biegert A., Hauser A., and Soding J. Hhblits: lightning-fast iterative protein sequence searching by hmm-hmm alignment. Nature Methods, 9(2):173–175, 2011." />
<meta name="citation_reference" content="Senda T., Saitoh S.-i., and Mitsui Y. Refined crystal structure of recombinant murine interferon-b at 2. 15 åresolution. J. Mol. Biol., 253:187–207, 1995." />
<meta name="citation_reference" content="Stertz S., Reichelt M., Spiegel M., Kuri T., Mart nez-sobrido L., Garc a-sastre A., Weber F., and Kochs G. The intracellular sites of early replication and budding of sars-coronavirus. Virology, 361:304–315, 2007." />
<meta name="citation_reference" content="Tsukazaki T., Chiang T. A., Davison A. F., Attisano L., and Wrana J. L. Sara, a fyve domain protein that recruits smad2 to the tgf-β receptor. Cell, 95(6):779–791, 1998." />
<meta name="citation_reference" content="UniProt Consortium. Uniprot: a hub for protein information. Nucleic acids research, 43(Database issue):D204–12, Jan 2015." />
<meta name="citation_reference" content="Woodcock S., Mornon J. P., and Henrissat B. Detection of secondary structure elements in proteins by hydrophobic cluster analysis. Protein engineering, 5(7):629–35, Oct 1992." />
<meta name="twitter:title" content="HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Motivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.

Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.

Availability: HCAtk and pyHCA are available at &lt;https://github.com/T-B-F/pyHCA&gt; under the CeCILL-C license." />
<meta name="og-title" property="og:title" content="HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/249995v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Motivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.

Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.

Availability: HCAtk and pyHCA are available at &lt;https://github.com/T-B-F/pyHCA&gt; under the CeCILL-C license." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-01-18" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/249995v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/249995v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/249995v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/249995v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/76610" />
    <title>HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WNb2oJoqPhmSuedQ3Eb9EQ-KAc1uJMDoCRVqfHy29y4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"76610","apath":"\/biorxiv\/early\/2018\/01\/18\/249995.atom","pisa":"biorxiv;249995v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;249995v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/249995v1","title":"HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences","identifier":"node\/76610"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tIxuhHC7\\\\\\\/CU2TZMrbUHVFrGMMT7i2IVERY0PQGDmH+CLD+OSLNYbO9VpICGhqTVMeML3Kr27muH2kbhfQbtnEnOXPSez2ZL\\\\\\\/Mfp7Cm9fKx2JKfU8f0nAHxopcxABYuLRQqTuXzOnS8pknIHIUL6X5s9FCzskEmXde4zTyTs2SNrGaFdSVSpzJAlMYOQJWYhib0PDY3qxKujLhT5VLW6o\\\\\\\/9tg77nn20W6qMJ6UqR9CVW5joF4OAmlU8oI5yjAMlXS9MOb\\\\\\\/s1TwdJMpBA6hep8ntlAYwxczE\\\\\\\/kgxbDldragCNWauAddJiUYHBtQJZYrMtl+1kw\\\\\\\/1Oz2zY0jYCeQ+H+vvOYvMWRVx2eKigykFQoAkIva8X7wZI2WbnUMqQocckaKEMyf4Ifi\\\\\\\/hL70i\\\\\\\/g6hcBpEE+n\\\\\\\/SKUvJAuriORiTVDTxFF4ptnqkPuPxRFG\\\\\\\/bl1aL3ufdchmeJLjkzDlkapmfg3KxDlefRW\\\\\\\/gC3D+dMU8+U96NrtZ2TkDjqIOJ++3TfoSu9GtyQgjE0NAPo8n7YUG7W4ZkqJhNHbVI6bVgbKjFJyMXHN94yt5gTFKCReEEJ0N1J\\\\\\\/VgGwvYKn0Sm\\\\\\\/Z38yl0QqOcUbzFoKcKZ\\\\\\\/NSI0O73JIJSAQ9wW59IgagO48GVfK\\\\\\\/acF0gcJtmsobSUCtW0ZxRM3apbdJ8HjFb6+qDqZ936QQRSOGbswKnPj4BeOmh2oacxCqaN0qZet0JjVli4Dz0HT2HE468abhkf5+lbanWrw9Xdmga19DlIrh91U1sE3XZ6GFcDAYL1\\\\\\\/3Cnwhp3bgDC1J9Z4yRGCsA4i+1NgusSa1kJEHw8fE7DqKuid+xgi5SJwhGBoOPloBzYFXACidmrZ8DQ0mUkg\\\\\\\/MzzmBeOBoXbiGPz7qoZEiSBBNl5g8hRZ9MIFCK21WQtkC5HPilSg1j46qqkJvG54PmlBq4o8\\\\\\\/M8YYDKFTTZmPyMYfimAaMtdXeT82l+cU04afOSvQ7sNYdwS07MfFz0gjg\\\\\\\/3vAqgpT8V6wad1ekPpJCr3CAuMlK49txWCK60BbuRAZZkD3NVrYln+Qn8ARER5lBr07rX1F1dj\\\\\\\/KxtjZX2QD7mTR\\\\\\\/lst7QYZUs3A9YOam6YMSuO9Yg7VLRnL0By+Ge+SgVZFOXO07YiMTT7aYUdbc+Hc6f3hW9EPgWg7wT+8U8k0+a+iiA6K\\\\\\\/kkAsbGfbXUFiwvECflkmxx7f6Zl8sXbwmEomx46q2S+q4KjzZP05oeAQrhqI65oKro57jx\\\\\\\/InRx6x9vDse\\\\\\\/S6+AXgBZ9ryA\\\\\\\/cg3r8WgbWoBs4LHfyllE6oeyZvNlnL+9xq+DB+R7WBjHNxh6E=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NYe5iy+JcPKIyZsoSrnTeQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220e694851cfcaf82507ee7991ed9dd7ec39c60e348eadeb5506b415ccbdcb1bc9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/249995v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-76610 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/249995v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SyRkkOxl-OwCqStjQJU-eBlvjx0f4y9Ten1c_BRKfds" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="76610" id="node76610" data-pisa="biorxiv;249995v1" data-pisa-master="biorxiv;249995" data-apath="/biorxiv/early/2018/01/18/249995.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Bitard-Feildel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isabelle</span> <span class="nlm-surname">Callebaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/249995 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node76610" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tristan Bitard-Feildel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Sorbonne Université, UPMC Université Paris 6</span>, CNRS, IBPS, UMR 7238, Laboratoire de Biologie Computationnelle et Quantitative (LCQB), 75005 Paris, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CNRS UMR7590, Sorbonne Université, Université Pierre et Marie Curie</span> – Paris 6 – MNHN – IRD – IUC, Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tristan%2BBitard-Feildel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bitard-Feildel%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATristan%2BBitard-Feildel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Isabelle Callebaut </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>CNRS UMR7590, Sorbonne Université, Université Pierre et Marie Curie</span> – Paris 6 – MNHN – IRD – IUC, Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isabelle%2BCallebaut%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Callebaut%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsabelle%2BCallebaut%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/249995v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:76610" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:76610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/249995v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:76610" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:76610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/249995v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:76610" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:76610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/249995v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:76610" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:76610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/249995v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:76610" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:76610/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Motivation: Detecting protein domains sharing no similarity to known domains, as stored in domain databases, is a challenging problem, particularly for unannotated proteomes, domains emerged recently, fast diverging proteins or domains with intrinsically disordered regions.</p><p id="p-4">Results: We developed pyHCA and HCAtk, a python API and standalone tool gathering together improved versions of previously developed methodologies, with new functionalities. The developed tools can be either used from command line or from a python API.</p><p id="p-5">Availability: HCAtk and pyHCA are available at <a href="https://github.com/T-B-F/pyHCA">https://github.com/T-B-F/pyHCA</a> under the CeCILL-C license.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">The annotation of a protein sequence is very often the first step of many bioinformatics analyses, for instance for studying the function of a gene or the evolution of organisms. Protein domain annotation dominates analyses, describing a protein as a list of blocks corresponding to evolutionary and functional conserved segments. Protein domain families have been extensively compiled through sequence or structure similarity searches and stored in several public databases. These domain databases represent state of the art of our current knowledge of the protein domain universe [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. However, many protein sequences escape, at least partially, domain annotation, particularly in non-model organisms and remain in the so-called dark protein sequence universe [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Classical methodological approaches model protein domain families as Hidden Markov Models (HMMs). However, to that aim, sequences need to be clustered and aligned based on the identification of sequence similarities. Therefore, proteins from organisms distant from the species considered in the model, fast diverging proteins, recently emerged domains and domains containing disordered regions, are less likely to be annotated using this methodology [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Here, in order to provide a comprehensive view of protein domain architectures, we present a standalone software named HCA toolkit (HCAtk), and its associated python API pyHCA. The package is easily installable and extend our previous developed tools making use of the Hydrophobic Cluster Analysis (HCA) of protein sequences [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], with new functionalities. The HCA methodology, based on a two-dimensional representation of protein sequences, highlights clusters of hydrophobic amino acids making up globular domains. More on the HCA methodology can be found in the supplementary materials.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><p id="p-7">Seg-HCA [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>] was developed to automatically delineate potential “foldable” domains within protein sequences and is the core part of our package. Recently, Piovesan et al. [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] implemented an in-house version of Seg-HCA in FELLS, which allows to nicely visualize different properties of a protein sequence. Our new version of Seg-HCA was rewritten for speed and a score is now computed, describing the general composition in hydrophobic clusters of the delineated foldable domains. This score is compared to an empirical distribution computed over 734 disordered protein sequences from DisProt v7 [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] to produce a p-value. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the distributions of scores computed using non redundant sequences of the Protein Data Bank for the set of globular domains and the set of DisProt protein sequences. The resulting p-value can thus be used to evaluate the likelihood of the delineated domains to fold into globular structures. Interestingly, some Seg-HCA domains are reported with a high p-value. A closer inspection revealed these sequences as partially disordered and undergoing possible folding upon binding. A detailed description of scores with some examples is provided in the supplementary material.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HCA score and HCA plot example. Panel A, left, shows the normalized HCA score distribution calculated for protein sequences from DisProt v7 (left, orange { disordered sequences) and from PDB (right, violet - globular domains). The HCA p-vaue assessing the globularity of delineated foldable segment, is computed using the empirical distribution from DisProt sequences. Panel B, right, shows the HCA plots of three BRCT domains from the Pfam family (PF00533). The aligned protein sequences were used as an input and conserved amino acids can be visualized in red (highly conserved) and yellow, in the context of hydrophobic clusters (HC), in order to evaluate the secondary structure conservation, relatively to the HC shapes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;HCA score and HCA plot example.&lt;/span&gt; Panel A, left, shows the normalized HCA score distribution calculated for protein sequences from DisProt v7 (left, orange { disordered sequences) and from PDB (right, violet - globular domains). The HCA p-vaue assessing the globularity of delineated foldable segment, is computed using the empirical distribution from DisProt sequences. Panel B, right, shows the HCA plots of three BRCT domains from the Pfam family (PF00533). The aligned protein sequences were used as an input and conserved amino acids can be visualized in red (highly conserved) and yellow, in the context of hydrophobic clusters (HC), in order to evaluate the secondary structure conservation, relatively to the HC shapes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F1.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F1.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">HCA score and HCA plot example.</span><p id="p-8" class="first-child">Panel A, left, shows the normalized HCA score distribution calculated for protein sequences from DisProt v7 (left, orange { disordered sequences) and from PDB (right, violet - globular domains). The HCA p-vaue assessing the globularity of delineated foldable segment, is computed using the empirical distribution from DisProt sequences. Panel B, right, shows the HCA plots of three BRCT domains from the Pfam family (PF00533). The aligned protein sequences were used as an input and conserved amino acids can be visualized in red (highly conserved) and yellow, in the context of hydrophobic clusters (HC), in order to evaluate the secondary structure conservation, relatively to the HC shapes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The second methodology included in the package is our TREMOLO-HCA software (Traveling through REMOte homoLOgy) [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Using as queries domains delineated using Seg-HCA, remote similarity search is performed against protein sequences from the Uniprot database [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] using HHBlits [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. For each hit, domain arrangement of the Uniprot targets is retrieved from the Interpro database [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The final output allows to directly link unknown protein domains, delineated by Seg-HCA, to existing annotations and to analyze these unknown domains in the context of their domain arrangement. The original tool was based on PSI-Blast and the CDD webserver. The new implementations based on HHBlits and Interpro allows a more sensitive detection of protein sequence remote similarity combined with a larger coverage of the protein domain universe thanks to the multiple sources of annotations integrated in Interpro. Several scripts are also provided to easily parse and query the TREMOLO output and to quickly retrieve protein domains of Interpro overlapping the unknown Seg-HCA domains or to retrieve the whole protein domain architectures associated with the Seg-HCA domains.</p><p id="p-10">Finally, two drawing functionalities were developed. The first functionality allows visualization of hydrophobic clusters of protein sequences, whether these are aligned or not. For each protein provided in a fasta file, an HCA plot is drawn, allowing the quick inspection of the hydrophobic cluster content of a protein sequence, which gives information about its composition in regular secondary structures. A detailed description of the drawing methodology is provided in the Supplementary Material. Moreover, another new functionality was implemented to highlight conservation between aligned protein sequences on their HCA plots. Conserved protein sequence positions can therefore be inspected in the context of their hydrophobic cluster organization (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The second drawing functionality is a new methodology built on the TREMOLO results to easily visualize the known protein domain annotation (from Interpro) and the newly delineated domains in an evolutionary context by using the NCBI taxonomic database. The tree is automatically built by fetching the taxonomic id of the Uniprot target sequences found by TREMOLO thanks to the ete3 python package. The Seg-HCA domains of TREMOLO can then be analyzed in the context of their protein domain arrangement and visualized in terms of taxonomic specificity and domain association (Fig. S1). The HCA toolkit is written in python3 and is provided under the CeCILL-C license agreement. The functions associated with the HCA analyses in the toolkit can also be directly used through a python API and as such can easily be used in other software.</p></div><div class="section" id="sec-3"><h2 class="">Supporting Information</h2><p id="p-11">Supplementary Figure 1 is accessible at <a href="https://github.com/T-B-F/pyHCA/blob/master/img/Supplementary_Fig1.pdf">https://github.com/T-B-F/pyHCA/blob/master/img/Supplementary_Fig1.pdf</a>.</p><div id="sec-4" class="subsection"><h3>HCA methodology, HCA plot and Seg-HCA</h3><p id="p-12">HCA hydrophobic clusters, made of strong hydrophobic amino acids (V, I, L, F, M, Y, W), are different from hydrophobic segments as they can incorporate other, non-hydrophobic residues. This property originates from the use of a two-dimensional alpha-helical net, connecting hydrophobic amino acids separated by up to three non-hydrophobic amino acids (or a proline) [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. Hydrophobic clusters defined in this way (with this hydrophobic alphabet and the connectivity distance associated with the <em>α</em>-helix) have been shown to match at best regular secondary structures (<em>α</em>-helices and <em>β</em>-strands) and to constitute hallmarks of folded domains [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Sequence segments delineated by Seg-HCA, which correspond to regions where a high density in hydrophobic clusters is detected, have been shown to correspond to domains that have the ability to fold, either in an autonomous way or following contact with partners [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>]; these segments are later referred to as HCA domains. The advantage of Seg-HCA for the characterization of the dark proteome is to allow the prediction of these foldable domains from the only information of a single amino acid sequence, without the prior knowledge of homologous sequences.</p><p id="p-13">
<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure S2</a> presents the Hydrophobic Cluster Analysis (HCA) methodology and indicates how are generated the HCA plots shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>. From an original 1D amino acid sequence (panel A), a 2D plot is created (panel D) by rolling the amino acid sequence around an <em>α</em>-helix (panel B) and cutting the helix along the horizontal axis. The helix forms a plane (panel C) on which every line of amino acids corresponds to an helix turn. The plane is duplicated and the hydrophobic clusters are defined by joining contiguous strong hydrophobic amino acids (V, I, L, F, M, Y, W).</p><p id="p-14">Regular secondary structure (RSS) elements can be easily visualized on the 2D plot, mainly corresponding to hydrophobic clusters, which are separated from each other by loops. Vertical hydrophobic clusters mainly correspond to <em>β</em>-strands and horizontal clusters to <em>α</em>-helices. A dictionary of the most current hydrophobic clusters, established from a comprehensive analysis of experimental 3D structures, can be considered for evaluating the main propensities of hydrophobic clusters towards RSS [<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>].</p></div><div id="sec-5" class="subsection"><h3>HCA score</h3><p id="p-15">The HCA score, used to compute a p-value associated with each HCA domain, is defined as follow. Each residue of an HCA segment is associated with a class regarding the residue type and hydrophobicity. Such a residue is either in an hydrophobic cluster and hydrophobic, in an hydrophobic cluster and hydrophilic, or outside an hydrophobic cluster. A value is attributed to each class and the HCA score is computed as follow: with s(i) the function mapping the residue i to each class value.</p><p id="p-16">Therefore, the HCA score scales with the density in hydrophobic clusters and in hydrophobic residues inside the clusters. As the HCA score calculation motivation is to provide an estimation of the globular character, i.e. the foldability of an HCA domain, the value of each of the three classes was optimized to maximize the separation between the distributions of the HCA scores computed on disordered sequences

 from DisProt and on sequences of globular proteins from the PDB. The optimization was performed on a grid search of every combination of integers between [-10; 10]. The best combination was obtained with the values of 10, 9 and -10 for classes of residues in an hydrophobic cluster and hydrophobic, in an hydrophobic cluster and hydrophilic and outside an hydrophobic cluster, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principle of the HCA plot. Panel A, the protein sequence (1D), in which hydrophobic amino acids are represented as white letters, is written on an &#x3B1;-helix, displayed on a cylinder (panel B). This one is cut along the horizontal axis and unrolled, in order to get the full environment of each amino acid, as it exists on the 1D sequence (panel C). Strong hydrophobic amino acids (V, I, L, F, M, Y, W) are encircled and their contours are joined (panel D), forming clusters. Horizontal and vertical clusters are mainly associated with alpha helices and beta strands, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Principle of the HCA plot.&lt;/span&gt; Panel A, the protein sequence (1D), in which hydrophobic amino acids are represented as white letters, is written on an &lt;em&gt;&#x3B1;&lt;/em&gt;-helix, displayed on a cylinder (panel B). This one is cut along the horizontal axis and unrolled, in order to get the full environment of each amino acid, as it exists on the 1D sequence (panel C). Strong hydrophobic amino acids (V, I, L, F, M, Y, W) are encircled and their contours are joined (panel D), forming clusters. Horizontal and vertical clusters are mainly associated with alpha helices and beta strands, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F2.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F2.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <span class="caption-title">Principle of the HCA plot.</span><p id="p-17" class="first-child">Panel A, the protein sequence (1D), in which hydrophobic amino acids are represented as white letters, is written on an <em>α</em>-helix, displayed on a cylinder (panel B). This one is cut along the horizontal axis and unrolled, in order to get the full environment of each amino acid, as it exists on the 1D sequence (panel C). Strong hydrophobic amino acids (V, I, L, F, M, Y, W) are encircled and their contours are joined (panel D), forming clusters. Horizontal and vertical clusters are mainly associated with alpha helices and beta strands, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Finally, a reciprocal inverse Gaussian random distribution was fitted on the DisProt scores distribution and the cumulative density function of this random distribution was used to define the p-value associated to each HCA domain. To avoid problems of scores associated with short HCA domains (≤ 30 residues), and as the limited number of sequences in DisProt makes difficult to adapt the scores to various sequence lengths, the p-values are not reported in these particular cases. This methodological choice is justified as a minimal number of residues is generally necessary for a protein domain to fold into a globular structure.</p></div><div id="sec-6" class="subsection"><h3>Examples of disordered regions with low HCA scores</h3><p id="p-19"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Supplementary Figures 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a> show two examples of protein sequences taken from the left tail of the HCA score distribution of DisProt sequences, displayed in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a> of the main document. For each figure, the HCA plot is drawn on top and the DisProt annotation taken from the DisProt webserver is shown at the bottom. These two sequences have HCA patterns typical of disordered proteins, i.e. proteins having very few hydrophobic residues, often gathered in HCA clusters of small length and spread along the sequence, this one including many proline residues (star symbols). Both proteins regions shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. S3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">S4</a> have low percentages of hydrophobic amino acids (13% and 6%).

<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure S3</a> corresponds to the C-terminal sequence of the human Cyclin-dependent kinase inhibitor p27. p27Kip1 controls eukaryotic cell division through interactions with cyclin-dependent kinases [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] and is known as a flexible protein [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], whose stability is associated with phosphorylation. The C-terminal region of p27 has a high flexibility, which provides the molecular basis for the sequential signal transduction conduit that regulates its own degradation and cell division [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Disordered regions with low HCA score - example n&#xB0; 1. The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 85 to 198 of the human Cyclin-dependent kinase inhibitor p27(Kip1) (Uniprot P46527)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Disordered regions with low HCA score - example n&#xB0; 1.&lt;/span&gt; The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 85 to 198 of the human Cyclin-dependent kinase inhibitor p27(Kip1) (Uniprot P46527).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F3.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F3.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <span class="caption-title">Disordered regions with low HCA score - example n° 1.</span><p id="p-20" class="first-child">The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 85 to 198 of the human Cyclin-dependent kinase inhibitor p27(Kip1) (Uniprot P46527).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">
<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure S4</a> corresponds to the C-terminal sequence of the chicken Histone H5 protein. Histone proteins have well characterized intrinsic disordered regions which are necessary to their biological function [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] and are targets for post-translational modifications recognized by specific readers. Two serine phosphorylation sites have been identified at position 146 and 167. The abundance of lysine and arginine also suggests possible acetylation/methylation sites. On the other hand, the C-terminal domain of chicken Histone H5 has a DNA binding motif [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], whose activity requires a high level of intrinsic flexibility.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Disordered regions with low HCA score - example n&#xB0; 2. The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 87 to 190 of the chicken Histone H5 protein (Uniprot P02259)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Disordered regions with low HCA score - example n&#xB0; 2.&lt;/span&gt; The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 87 to 190 of the chicken Histone H5 protein (Uniprot P02259).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F4.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F4.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4:</span> <span class="caption-title">Disordered regions with low HCA score - example n° 2.</span><p id="p-22" class="first-child">The HCA pattern displayed by the disordered region is typical of non globular regions, with very few clusters and many proline (star symbols). The disordered region corresponds to the sequence segment from amino acids 87 to 190 of the chicken Histone H5 protein (Uniprot P02259).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Examples of disordered regions with high HCA scores</h3><p id="p-23"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figures S5</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">S6</a> show two examples extracted from the right tail of the DisProt HCA score distribution, i.e. proteins with HCA scores similar to the lowest scores of the sequences extracted from PDB (named PDB sequences below). These two examples display HCA patterns including larger hydrophobic clusters (typical of regular secondary structures), as found in globular proteins, but with a slightly lower total content in hydrophobic amino acids (24% for both against 30% expected). The first example concerns a disordered region (amino acids 291 to 352) found in the chicken zing finger FYVE domain-containing protein 9 (UniProt O95405, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. S5</a>). The 3D structure of only the FYVE domain has been experimentally characterized, corresponding to the FYVE zinc finger domain (PF01363) (amino acids 663 to 751), the second domain corresponds, from amino acids 1048 to 1400, to a Pfam domain of unknown function (PF11979). The protein regulates the subcellular localization of SMAD2/SMAD3 by recruiting them to the TGF-beta receptor [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. The HCA pattern displayed by the disordered region is similar to patterns observed for foldable regions, suggesting that this small domain is able to fold, at least under particular conditions. The absence of any clear annotation in the N-terminal part of the protein, including two small regions predicted as disordered, but in which a potential order is detected, suggests the presence of an un-detected domain of unknown function [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Disordered regions with high HCA score - example n&#xB0;1. The HCA pattern displayed by this disordered region is similar to patterns observed for foldable regions. Clusters of hydrophobic residues, whose length are typical of stable regular secondary structures, are separated by regions whose lengths are typical of loops. This disordered region corresponds to an internal segment, from amino acids 291 to 352, of the chicken&#x2019;s zinc finger FYVE domain-containing protein 9 (UniProt O95405), without known function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Disordered regions with high HCA score - example n&#xB0;1.&lt;/span&gt; The HCA pattern displayed by this disordered region is similar to patterns observed for foldable regions. Clusters of hydrophobic residues, whose length are typical of stable regular secondary structures, are separated by regions whose lengths are typical of loops. This disordered region corresponds to an internal segment, from amino acids 291 to 352, of the chicken&#x2019;s zinc finger FYVE domain-containing protein 9 (UniProt O95405), without known function.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F5.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F5.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5:</span> <span class="caption-title">Disordered regions with high HCA score - example n°1.</span><p id="p-24" class="first-child">The HCA pattern displayed by this disordered region is similar to patterns observed for foldable regions. Clusters of hydrophobic residues, whose length are typical of stable regular secondary structures, are separated by regions whose lengths are typical of loops. This disordered region corresponds to an internal segment, from amino acids 291 to 352, of the chicken’s zinc finger FYVE domain-containing protein 9 (UniProt O95405), without known function.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The second example concerns a disordered region (amino acids 82 to 134) found in another member of the zinc finger family (zinc finger protein ZNF593, UniProt O00488, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. S6</a>) which belongs to the human species. The zinc finger domain is central (60 to 86) and flanked by two disordered regions. The protein has a high degree of intrinsic disorder, as revealed by the experimental NMR structure of the full length protein, without truncation of the N- or C-terminal regions [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. No clear function is currently associated with the N- and C-terminal regions of the protein. As for the previous example, the HCA pattern displayed by the disordered region is similar to patterns observed for foldable regions, suggesting that this small domain is able to fold, at least under particular conditions.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Disordered regions with high HCA score - example n&#xB0;2. The HCA pattern displayed by this disordered region is similar to pattern of foldable regions. Clusters of several hydrophobic residues can be seen close together. The disordered region, (82-134) corresponds to the C-terminal of human Zinc finger protein 593 (UniProt O00488)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Disordered regions with high HCA score - example n&#xB0;2.&lt;/span&gt; The HCA pattern displayed by this disordered region is similar to pattern of foldable regions. Clusters of several hydrophobic residues can be seen close together. The disordered region, (82-134) corresponds to the C-terminal of human Zinc finger protein 593 (UniProt O00488).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F6.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F6.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6:</span> <span class="caption-title">Disordered regions with high HCA score - example n°2.</span><p id="p-26" class="first-child">The HCA pattern displayed by this disordered region is similar to pattern of foldable regions. Clusters of several hydrophobic residues can be seen close together. The disordered region, (82-134) corresponds to the C-terminal of human Zinc finger protein 593 (UniProt O00488).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Examples of PDB sequences with high HCA scores</h3><p id="p-27">
<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figures S7</a> and S8 show two examples of HCA plots for sequences extracted from the PDB. <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure S7</a> corresponds to the HCA plot of the Archeoglobus fulgidus VapC ribonuclease (Uniprot O28590, amino acids 1 to 156) whose 3D structure has been solved X-ray crystallography (PDB entry 1W8I). This ribonuclease is involved in a toxin-antitoxin module with toxin activity [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>] and includes one known domain (amino acids 3 to 127 corresponds to the PFAM domain PIN, PF01850). The corresponding structure includes 9 long <em>α</em>-helices with 40% of hydrophobic amino acids. The dense network of HCA clusters visible in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. S7</a> is typical of globular proteins and the long <em>α</em>-helices can be visualized as horizontal clusters on the 2D HCA plot.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PDB sequence with an high HCA score - example n&#xB0;1. This sequence (Uniprot 028590, PDB 1W8I), including amino acids 1 to 156, corresponds a typical globular protein. (&#x3B1;-helix: red rectangle, annotations extract from the experimental 3D structure 1W8I)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PDB sequence with an high HCA score - example n&#xB0;1.&lt;/span&gt; This sequence (Uniprot 028590, PDB 1W8I), including amino acids 1 to 156, corresponds a typical globular protein. (&lt;em&gt;&#x3B1;&lt;/em&gt;-helix: red rectangle, annotations extract from the experimental 3D structure 1W8I).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F7.medium.gif" width="440" height="132"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F7.medium.gif" width="440" height="132"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7:</span> <span class="caption-title">PDB sequence with an high HCA score - example n°1.</span><p id="p-28" class="first-child">This sequence (Uniprot 028590, PDB 1W8I), including amino acids 1 to 156, corresponds a typical globular protein. (<em>α</em>-helix: red rectangle, annotations extract from the experimental 3D structure 1W8I).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">
<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S8</a> is another example of a globular protein HCA plot, i.e. with a high HCA score. The figure shows the HCA plot of the mature mouse interferon beta (Uniprot P01575, amino acids 22 to 181, PDB entry 1WU3 [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]). The protein is made of one domain (Pfam amino acids 27 to 179 (PF00143)), including 5 long <em>α</em>-helices with 42% of hydrophobic residues. As for <a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. S7</a>, the protein contains large hydrophobic clusters, typical of regular secondary structures, separated by loops.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PDB sequence with an high HCA score - example n&#xB0;2. The sequence region (Uniprot P01575, PDB entry 1WU3), including amino acids 22 to 181, corresponds to the PDB structure 1WU3 a typical globular protein. (&#x3B1;-helix: red rectangle, annotations extracted from the experimental 3D structure 1WU3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PDB sequence with an high HCA score - example n&#xB0;2.&lt;/span&gt; The sequence region (Uniprot P01575, PDB entry 1WU3), including amino acids 22 to 181, corresponds to the PDB structure 1WU3 a typical globular protein. (&lt;em&gt;&#x3B1;&lt;/em&gt;-helix: red rectangle, annotations extracted from the experimental 3D structure 1WU3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F8.medium.gif" width="440" height="97"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F8.medium.gif" width="440" height="97"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8:</span> <span class="caption-title">PDB sequence with an high HCA score - example n°2.</span><p id="p-30" class="first-child">The sequence region (Uniprot P01575, PDB entry 1WU3), including amino acids 22 to 181, corresponds to the PDB structure 1WU3 a typical globular protein. (<em>α</em>-helix: red rectangle, annotations extracted from the experimental 3D structure 1WU3).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Examples of PDB sequences with low HCA scores</h3><p id="p-31">
<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figures S9</a> to <a id="xref-fig-10-1" class="xref-fig" href="#F10">S10</a> are two examples of PDB sequence HCA plots with low HCA scores. <a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig. S9</a> corresponds to the N-terminal domain (amino acids 50 to 174) of the nucleoprotein from human SARS coronavirus (Uniprot protein P59595, PDB entry 2OFZ [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]). This nucleoprotein has RNA binding activity, packaging the positive strand of the human SARS coronavirus RNA genome into a helical ribonucleocapsid [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. The RNA binding activity is mediated by the region encompassing amino acids 45 to 181, such binding activity is usually mediated by a high level of flexibility. The full-length protein is made of one or two protein domains according to the Pfam database (PF00937, from 15 to 378) or the SUPERFAMILY database (SSF110304, from 28 to 181 and SSF103068, from 252 to 365). The 3D structure of the first SSF domain is made of four <em>β</em>-strands (from amino acids 61 to 59, 84 to 91, 102 to 113, and 130 to 135) and one small <em>α</em>-helix (50 to 57), with a large number of flexible loops around the <em>β</em>-sheet core (Saikatendu et al., 2007). According to the coverage of the sequence by large loops, this protein domain has a lower percentage of hydrophobic residues (26%), than the regularly admitted of 30% limit, characteristic of globular domains.</p><p id="p-32">
<a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig. S10</a> is another example of PDB protein sequence with low HCA score. The HCA plot represents a sequence segment (from amino acids 500 to 629) of the the Staphylococcus aureaus surface protein G

 (SasG) (Uniprot Q2G2B2 sequence, PDB entry 5DBL). The full-length protein is made of 19 domains. The sequence starts with a signal peptide motif, followed by pairs of G5 domain/E domain (Pfam PF04650, PF17041) and ends by a cell wall anchor domain (PF00746). Amino acids 501 to 548 and 547 to 629 corresponds to a pair of E domain/G5 domain. The G5 domain has only a few conserved amino acids and is supposed to have an adhesive function [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. As assessed by the presence of small clusters and a relatively weak percentage in strong hydrophobic amino acids, approximatively one half of the SasG repetitive regions are intrinsically unfolded in isolation, but fold in the context of neighboring folded G5 domains, highlighting the role of the intrinsically disordered region of the E/G5 domain pair as a key factor for the cooperative folding multidomain protein [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. Once folded, the two domains form an elongated structure, made of small beta strands which correspond on the HCA plot to small clusters. The small <em>β</em>-strands form triplets-stranded <em>β</em>-sheets connected by collagen-like triple helical regions. In this particular case, several threonine are found included in <em>β</em>-beta strand.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PDB sequence with a low HCA score - example n&#xB0;1. The sequence region (Uniprot P5995), amino acids 49 to 174, PDB structure 2OFZ), corresponds to a RNA-binding domain, having large, flexible loops and a few regular secondary structures, constituting a &#x3B2;-sheet core. (&#x3B1;-helix: red rectangle &#x3B2;-strands: yellow arrows, annotation extracted from the experimental 3D structure 2OFZ)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PDB sequence with a low HCA score - example n&#xB0;1.&lt;/span&gt; The sequence region (Uniprot P5995), amino acids 49 to 174, PDB structure 2OFZ), corresponds to a RNA-binding domain, having large, flexible loops and a few regular secondary structures, constituting a &lt;em&gt;&#x3B2;&lt;/em&gt;-sheet core. (&lt;em&gt;&#x3B1;&lt;/em&gt;-helix: red rectangle &lt;em&gt;&#x3B2;&lt;/em&gt;-strands: yellow arrows, annotation extracted from the experimental 3D structure 2OFZ).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F9.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F9.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9:</span> <span class="caption-title">PDB sequence with a low HCA score - example n°1.</span><p id="p-33" class="first-child">The sequence region (Uniprot P5995), amino acids 49 to 174, PDB structure 2OFZ), corresponds to a RNA-binding domain, having large, flexible loops and a few regular secondary structures, constituting a <em>β</em>-sheet core. (<em>α</em>-helix: red rectangle <em>β</em>-strands: yellow arrows, annotation extracted from the experimental 3D structure 2OFZ).</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PDB sequence with low HCA score - example n&#xB0;2. The sequence region, (Uniprot Q2G2B2, amino acids 500 to 629, PDB structure entry 5DBL), corresponds to a pair of E/G5 domains of the S. aureus surface protein G. (&#x3B2;-strand: yellow arrow, annotations extracted from the experimental 3D structure 5DBL)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-650929670" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PDB sequence with low HCA score - example n&#xB0;2.&lt;/span&gt; The sequence region, (Uniprot Q2G2B2, amino acids 500 to 629, PDB structure entry 5DBL), corresponds to a pair of E/G5 domains of the S. aureus surface protein G. (&lt;em&gt;&#x3B2;&lt;/em&gt;-strand: yellow arrow, annotations extracted from the experimental 3D structure 5DBL).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F10.medium.gif" width="440" height="147"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F10.medium.gif" width="440" height="147"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/18/249995/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10:</span> <span class="caption-title">PDB sequence with low HCA score - example n°2.</span><p id="p-34" class="first-child">The sequence region, (Uniprot Q2G2B2, amino acids 500 to 629, PDB structure entry 5DBL), corresponds to a pair of E/G5 domains of the S. aureus surface protein G. (<em>β</em>-strand: yellow arrow, annotations extracted from the experimental 3D structure 5DBL).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-10"><h2 class="">Funding</h2><p id="p-35">This work has been supported by the Agence Nationale de la Recherche (grant number ANR-14-CE10-0021) and the Institut National du Cancer (grant number PLBIO14-299). Conflict of Interest: none declared.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> <span class="em-link"><span class="em-addr">isabelle.callebaut{at}impmc.upmc.fr</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.1" data-doi="10.1093/protein/gzq081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arcus</span>  <span class="cit-name-given-names">V. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robson</span>  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-article-title">The pin-domain ribonucleases and the prokaryotic vapbc toxin-antitoxin array</span>. <abbr class="cit-jnl-abbrev">Protein Engineering Design and Selection</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">1–2</span>):<span class="cit-fpage">33</span>–<span class="cit-lpage">40</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzq081%26rft_id%253Dinfo%253Apmid%252F21036780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/gzq081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21036780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285420400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.2" data-doi="10.1093/bioinformatics/bti206"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">M. T. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yeats</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">The g5 domain: a potential n-acetylglucosamine recognition domain involved in biofilm formation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1301</span>–<span class="cit-lpage">1303</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti206%26rft_id%253Dinfo%253Apmid%252F15598841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15598841&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228401800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.3" data-doi="10.1021/bi015763t"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bienkiewicz</span>  <span class="cit-name-given-names">E. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adkins</span>  <span class="cit-name-given-names">J. N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lumb</span>  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">Functional consequences of preorganized helical structure in the intrinsically disordered cell-cycle inhibitor p27kip1</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, <span class="cit-vol">41</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">752</span>–<span class="cit-lpage">759</span>, <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D41%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi015763t%26rft_id%253Dinfo%253Apmid%252F11790096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi015763t&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11790096&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173374500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bitard-Feildel</span>  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Exploring the dark foldable proteome by considering hydrophobic amino acids topology</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-vol">7</span>:<span class="cit-issue">41425</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.5" data-doi="10.1016/j.biochi.2015.02.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bitard-Feildel</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heberlein</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bornberg-Bauer</span>  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Detection of orphan domains in drosophila using \hydrophobic cluster analysis</span>.” <abbr class="cit-jnl-abbrev">Biochimie</abbr>, <span class="cit-vol">119</span>:<span class="cit-fpage">244</span>–<span class="cit-lpage">253</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D119%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biochi.2015.02.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biochi.2015.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.6" data-doi="10.1007/s000180050082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labesse</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poupon</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chomilier</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mornon</span>  <span class="cit-name-given-names">J. P.</span></span> <span class="cit-article-title">Deciphering protein sequence information through hydrophobic cluster analysis (hca): current status and perspectives</span>. <abbr class="cit-jnl-abbrev">Cellular and molecular life sciences</abbr>, <span class="cit-vol">53</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">621</span>–<span class="cit-lpage">645</span>, <span class="cit-month">Sep</span> <span class="cit-pub-date">1997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DCallebaut%26rft.auinit1%253DI.%26rft.volume%253D53%26rft.issue%253D8%26rft.spage%253D621%26rft.epage%253D645%26rft.atitle%253DDeciphering%2Bprotein%2Bsequence%2Binformation%2Bthrough%2Bhydrophobic%2Bcluster%2Banalysis%2B%2528HCA%2529%253A%2Bcurrent%2Bstatus%2Band%2Bperspectives.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs000180050082%26rft_id%253Dinfo%253Apmid%252F9351466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s000180050082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9351466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YB33200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.7" data-doi="10.1038/nature09534"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De La Vega</span>  <span class="cit-name-given-names">F. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-etal">et al.</span><span class="cit-article-title">A map of human genome variation from population-scale sequencing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">467</span>(<span class="cit-issue">7319</span>):<span class="cit-fpage">1061</span>–<span class="cit-lpage">1073</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDurbin%26rft.auinit1%253DR.%2BM.%26rft.volume%253D467%26rft.issue%253D7319%26rft.spage%253D1061%26rft.epage%253D1073%26rft.atitle%253DA%2Bmap%2Bof%2Bhuman%2Bgenome%2Bvariation%2Bfrom%2Bpopulation-scale%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09534%26rft_id%253Dinfo%253Apmid%252F20981092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20981092&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283548600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eudes</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le Tuan</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delettré</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mornon</span>  <span class="cit-name-given-names">J.-P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">A generalized analysis of hydrophobic and loop clusters within globular protein sequences</span>. <abbr class="cit-jnl-abbrev">BMC Structural Biology</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">2</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BStructural%2BBiology%26rft.volume%253D7%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Comprehensive repertoire of foldable regions within whole genomes</span>. <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1003280</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D9%26rft.spage%253D1003280e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.10" data-doi="10.1093/bioinformatics/btt271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Identification of hidden relationships from the coupling of hydrophobic cluster analysis and domain architecture information</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">14</span>):<span class="cit-fpage">1726</span>–<span class="cit-lpage">33</span>, <span class="cit-month">Jul</span> <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt271%26rft_id%253Dinfo%253Apmid%252F23677940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23677940&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.11" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Punta</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qureshi</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sangrador-Vegas</span>  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-etal">et al.</span> <span class="cit-article-title">The pfam protein families database: towards a more sustainable future</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D279</span>–<span class="cit-lpage">85</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673716&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.12" data-doi="10.1016/0014-5793(87)80439-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaboriaud</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bissery</span>  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benchetrit</span>  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mornon</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Hydrophobic cluster analysis: An efficient new way to compare and analyse amino acid sequences</span>. <abbr class="cit-jnl-abbrev">FEBS Letters</abbr>, <span class="cit-vol">224</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">149</span>–<span class="cit-lpage">155</span>, <span class="cit-month">Nov</span> <span class="cit-pub-date">1987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DGaboriaud%26rft.auinit1%253DC.%26rft.volume%253D224%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D155%26rft.atitle%253DHydrophobic%2Bcluster%2Banalysis%253A%2Ban%2Befficient%2Bnew%2Bway%2Bto%2Bcompare%2Band%2Banalyse%2Bamino%2Bacid%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252887%252980439-8%26rft_id%253Dinfo%253Apmid%252F3678489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(87)80439-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3678489&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L061300030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.13" data-doi="10.1016/j.jmb.2007.12.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galea</span>  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nourse</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sivakolundu</span>  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">W. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kriwacki</span>  <span class="cit-name-given-names">R. W.</span></span> <span class="cit-article-title">Role of intrinsic flexibility in signal transduction mediated by the cell cycle regulator, p27 kip1</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>, <span class="cit-vol">376</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">827</span>–<span class="cit-lpage">38</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGalea%26rft.auinit1%253DC.%2BA.%26rft.volume%253D376%26rft.issue%253D3%26rft.spage%253D827%26rft.epage%253D838%26rft.atitle%253DRole%2Bof%2Bintrinsic%2Bflexibility%2Bin%2Bsignal%2Btransduction%2Bmediated%2Bby%2Bthe%2Bcell%2Bcycle%2Bregulator%252C%2Bp27%2BKip1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2007.12.016%26rft_id%253Dinfo%253Apmid%252F18177895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2007.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18177895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253354900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-confproc" id="cit-249995v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gruszka</span>  <span class="cit-name-given-names">D. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendonca</span>  <span class="cit-name-given-names">C. A. T. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paci</span>  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hawkhead</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potts</span>  <span class="cit-name-given-names">J. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Disorder drives cooperative folding in a multidomain protein</span>. <span class="cit-conf-name">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="cit-vol">113</span>(<span class="cit-issue">42</span>):<span class="cit-fpage">11841</span>–<span class="cit-lpage">11846</span>, <span class="cit-conf-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.15" data-doi="10.1110/ps.073290408"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lytle</span>  <span class="cit-name-given-names">B. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Volkman</span>  <span class="cit-name-given-names">B. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">F. C.</span></span> <span class="cit-article-title">The solution structure of znf593 from homo sapiens reveals a zinc finger in a predominately unstructured protein</span>. <abbr class="cit-jnl-abbrev">Protein Science</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">571</span>–<span class="cit-lpage">576</span>, <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DHayes%26rft.auinit1%253DP.%2BL.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D571%26rft.epage%253D576%26rft.atitle%253DThe%2Bsolution%2Bstructure%2Bof%2BZNF593%2Bfrom%2BHomo%2Bsapiens%2Breveals%2Ba%2Bzinc%2Bfinger%2Bin%2Ba%2Bpredominantly%2Bunstructured%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.073290408%26rft_id%253Dinfo%253Apmid%252F18287285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.073290408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18287285&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newcomb</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bemis</span>  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcgowan</span>  <span class="cit-name-given-names">D. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dimauro</span>  <span class="cit-name-given-names">E. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faust</span>  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-etal">et al.</span><span class="cit-article-title">Discovery of novel 2,3-diarylfuro[2,3-b]pyridin-4-amines as potent and selective inhibitors of lck</span>: <abbr class="cit-jnl-abbrev">Synthesis, sar, and pharmacokinetic properties</abbr>. <span class="cit-vol">17</span>:<span class="cit-fpage">2299</span>–<span class="cit-lpage">2304</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSynthesis%252C%2Bsar%252C%2Band%2Bpharmacokinetic%2Bproperties%26rft.volume%253D17%26rft.spage%253D2299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.17" data-doi="10.1093/nar/gku1243"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAnulla</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMe-namin</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nuka</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pesseat</span>  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-etal">et al.</span> <span class="cit-article-title">The interpro protein families database: the classification resource after 15 years</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D213</span>–<span class="cit-lpage">21</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1243%26rft_id%253Dinfo%253Apmid%252F25428371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.18" data-doi="10.1093/nar/gku1041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oates</span>  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stahlhacke</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vavoulis</span>  <span class="cit-name-given-names">D. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smithers</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rackham</span>  <span class="cit-name-given-names">O. J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sardar</span>  <span class="cit-name-given-names">A. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaucha</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thurlby</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gough</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The superfamily 1.75 database in 2014: a doubling of data</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D227</span>–<span class="cit-lpage">33</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1041%26rft_id%253Dinfo%253Apmid%252F25414345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25414345&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mizianty</span>  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">V. N.</span></span> <span class="cit-article-title">More than just tails: intrinsic disorder in histone proteins</span>. <abbr class="cit-jnl-abbrev">Molecular BioSystems</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1886</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBioSystems%26rft.volume%253D8%26rft.spage%253D1886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.20" data-doi="10.1093/nar/gkw1056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Piovesan</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabaro</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mičetić</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Necci</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quaglia</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oldfield</span>  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aspromonte</span>  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">N. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidović</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dosztányi</span>  <span class="cit-name-given-names">Z.</span></span>, and <span class="cit-etal">et al.</span><span class="cit-article-title">Disprot 7.0: A major update of the database of disordered proteins</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D219</span>–<span class="cit-lpage">D227</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1056%26rft_id%253Dinfo%253Apmid%252F27899601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.21" data-doi="10.1093/bioinformatics/btx085"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Piovesan</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minervini</span>  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tosatto</span>  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-article-title">Fells: fast estimator of latent local structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1889</span>–<span class="cit-lpage">1891</span>, <span class="cit-month">Jun</span> <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D1889%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx085%26rft_id%253Dinfo%253Apmid%252F28186245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28186245&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Polyak</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erdjument-Bromage</span>  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Koff</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Cloning of p27 kip1, a cyclin-dependent kinase inhibitor and a potential mediator of extracellular antimitogenic signals</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">76</span>:<span class="cit-fpage">59</span>–<span class="cit-lpage">66</span>, <span class="cit-pub-date">1994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D76%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.23" data-doi="10.1038/362219a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finch</span>  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graziano</span>  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sweet</span>  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-article-title">Crystal structure of globular domain of histone h5 and its implications for nucleosome binding</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">362</span>(<span class="cit-issue">6417</span>):<span class="cit-fpage">219</span>–<span class="cit-lpage">223</span>, <span class="cit-pub-date">1993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRamakrishnan%26rft.auinit1%253DV.%26rft.volume%253D362%26rft.issue%253D6417%26rft.spage%253D219%26rft.epage%253D223%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bglobular%2Bdomain%2Bof%2Bhistone%2BH5%2Band%2Bits%2Bimplications%2Bfor%2Bnucleosome%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F362219a0%26rft_id%253Dinfo%253Apmid%252F8384699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/362219a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8384699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KT02600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rebehmed</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quintus</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mornon</span>  <span class="cit-name-given-names">J.-P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Callebaut</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">The respective roles of polar/nonpolar binary patterns and amino acid composition in protein regular secondary structures explored exhaustively using hydrophobic cluster analysis</span>. <abbr class="cit-jnl-abbrev">Proteins: Structure, Function, and Bioinformatics</abbr>, <span class="cit-vol">84</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">624</span>–<span class="cit-lpage">638</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStructure%252C%2BFunction%252C%2Band%2BBioinformatics%26rft.volume%253D84%26rft.spage%253D624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Remmert</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Biegert</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Hhblits: lightning-fast iterative protein sequence searching by hmm-hmm alignment</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">173</span>–<span class="cit-lpage">175</span>, <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D9%26rft.spage%253D173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.26" data-doi="10.1006/jmbi.1995.0544"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Senda</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saitoh</span>  <span class="cit-name-given-names">S.-i.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mitsui</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Refined crystal structure of recombinant murine interferon-b at 2. 15 åresolution</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>., <span class="cit-vol">253</span>:<span class="cit-fpage">187</span>–<span class="cit-lpage">207</span>, <span class="cit-pub-date">1995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSenda%26rft.auinit1%253DT.%26rft.volume%253D253%26rft.issue%253D1%26rft.spage%253D187%26rft.epage%253D207%26rft.atitle%253DRefined%2Bcrystal%2Bstructure%2Bof%2Brecombinant%2Bmurine%2Binterferon-beta%2Bat%2B2.15%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1995.0544%26rft_id%253Dinfo%253Apmid%252F7473712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1995.0544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7473712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ24000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.27" data-doi="10.1016/j.virol.2006.11.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stertz</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reichelt</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spiegel</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuri</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mart nez-sobrido</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garc a-sastre</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kochs</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The intracellular sites of early replication and budding of sars-coronavirus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-vol">361</span>:<span class="cit-fpage">304</span>–<span class="cit-lpage">315</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DStertz%26rft.auinit1%253DS.%26rft.volume%253D361%26rft.issue%253D2%26rft.spage%253D304%26rft.epage%253D315%26rft.atitle%253DThe%2Bintracellular%2Bsites%2Bof%2Bearly%2Breplication%2Band%2Bbudding%2Bof%2BSARS-coronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.11.027%26rft_id%253Dinfo%253Apmid%252F17210170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.11.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17210170&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.28" data-doi="10.1016/S0092-8674(00)81701-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsukazaki</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">T. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">A. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Attisano</span>  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wrana</span>  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-article-title">Sara, a fyve domain protein that recruits smad2 to the tgf-<em>β</em> receptor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">95</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">779</span>–<span class="cit-lpage">791</span>, <span class="cit-pub-date">1998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTsukazaki%26rft.auinit1%253DT.%26rft.volume%253D95%26rft.issue%253D6%26rft.spage%253D779%26rft.epage%253D791%26rft.atitle%253DSARA%252C%2Ba%2BFYVE%2Bdomain%2Bprotein%2Bthat%2Brecruits%2BSmad2%2Bto%2Bthe%2BTGFbeta%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981701-8%26rft_id%253Dinfo%253Apmid%252F9865696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81701-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9865696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077498800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.29" data-doi="10.1093/nar/gku989"><div class="cit-metadata"><cite><span class="cit-article-title">UniProt Consortium. Uniprot: a hub for protein information</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D204</span>–<span class="cit-lpage">12</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku989%26rft_id%253Dinfo%253Apmid%252F25348405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25348405&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-249995v1.30" data-doi="10.1093/protein/5.7.629"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodcock</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mornon</span>  <span class="cit-name-given-names">J. P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Detection of secondary structure elements in proteins by hydrophobic cluster analysis</span>. <abbr class="cit-jnl-abbrev">Protein engineering</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">629</span>–<span class="cit-lpage">35</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">1992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DWoodcock%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D629%26rft.epage%253D635%26rft.atitle%253DDetection%2Bof%2Bsecondary%2Bstructure%2Belements%2Bin%2Bproteins%2Bby%2Bhydrophobic%2Bcluster%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252F5.7.629%26rft_id%253Dinfo%253Apmid%252F1480617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/5.7.629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1480617&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F18%2F249995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JY14300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/249995v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F249995v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/249722v1" title="Accounting for Errors in Low Coverage High-Throughput Sequencing Data when Constructing Genetic Maps using Biparental Outcrossed Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/190769v2" title="Evolutionary rate shifts suggest species-specific adaptation events in HIV-1 and SIV" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 18, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/249995v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/249995v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/249995v1" class="active" data-icon-position="" data-hide-link-title="0">HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/76610" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-I8QDIOiHESZUbdPzB7kmb4Jtsjhkdu_Ec4hw0D_L0CI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="76610" id="node76610--2" data-pisa="biorxiv;249995v1" data-pisa-master="biorxiv;249995" data-seqnum="76610" data-apath="/biorxiv/early/2018/01/18/249995.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Bitard-Feildel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isabelle</span> <span class="nlm-surname">Callebaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">249995; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/249995 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/249995v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;title=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;title=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;text=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;title=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;t=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;title=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;title=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="76610" id="node76610--3" data-pisa="biorxiv;249995v1" data-pisa-master="biorxiv;249995" data-seqnum="76610" data-apath="/biorxiv/early/2018/01/18/249995.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Bitard-Feildel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isabelle</span> <span class="nlm-surname">Callebaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">249995; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/249995 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Bitard-Feildel%2C+Tristan&amp;rft.au=Callebaut%2C+Isabelle"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/76610/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/76610/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/76610/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/76610/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/76610/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/76610/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/76610/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/76610/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/76610/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/76610/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/76610/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/76610/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;count=horizontal&amp;via=&amp;text=HCAtk%20and%20pyHCA%3A%20A%20Toolkit%20and%20Python%20API%20for%20the%20Hydrophobic%20Cluster%20Analysis%20of%20Protein%20Sequences&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/249995v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/249995v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/249995v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
