<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7136267">7136267</a></span><span class="publisher-id" title="publisher-id">62663</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-62663-y">10.1038/s41598-020-62663-y</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title"><i>In vitro</i> dual (anticancer and antiviral) activity of the carotenoids produced by haloalkaliphilic archaeon <i>Natrialba</i> sp. M6</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hegazy</span><span class="given-names" tagx="given-names" title="given-names">Ghada E.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abu-Serie</span><span class="given-names" tagx="given-names" title="given-names">Marwa M.</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abo-Elela</span><span class="given-names" tagx="given-names" title="given-names">Gehan M.</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghozlan</span><span class="given-names" tagx="given-names" title="given-names">Hanan</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabry</span><span class="given-names" tagx="given-names" title="given-names">Soraya A.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soliman</span><span class="given-names" tagx="given-names" title="given-names">Nadia A.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">nadiastuttgart@yahoo.com</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdel-Fattah</span><span class="given-names" tagx="given-names" title="given-names">Yasser R.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">yasser1967@yahoo.com</span></span><a href="#Aff4">4</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0404 7762</span><span class="institution-id" title="institution-id">grid.419615.e</span><span class="institution" title="institution">Microbiology Department, Marine Environment, National Institute of Oceanography &amp;amp; Fisheries (NIOF), </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0483 2576</span><span class="institution-id" title="institution-id">grid.420020.4</span><span class="institution" title="institution">Medical Biotechnology Department, Genetic Engineering and Biotechnology Research Institute (GEBRI), City of Scientific Research and Technological Applications (SRTA-City), </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0483 2576</span><span class="institution-id" title="institution-id">grid.420020.4</span><span class="institution" title="institution">Bioprocess Development Department, Genetic Engineering and Biotechnology Research Institute (GEBRI), City of Scientific Research and Technological Applications (SRTA-City), </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">5986</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-20</span><span class="accepted" title="accepted">accepted: 2020-3-16</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Halophilic archaea are a promising natural source of carotenoids. However, little information is available about the biological impacts of these archaeal metabolites. Here, carotenoids of <i>Natrialba</i> sp. M6, which was isolated from Wadi El-Natrun, were produced, purified and identified by Raman spectroscopy, GC-mass spectrometry, and Fourier transform infrared spectroscopy, LC–mass spectrometry and Nuclear magnetic resonance spectroscopy. The C<sub>50</sub> carotenoid bacterioruberin was found to be the predominant compound. Because cancer and viral hepatitis are serious diseases, the anticancer, anti-HCV and anti-HBV potentials of these extracted carotenoids (pigments) were examined for the first time. <i>In vitro</i> results indicated that the caspase-mediated apoptotic anticancer effect of this pigment and its inhibitory efficacy against matrix metalloprotease 9 were significantly higher than those of 5-fluorouracil. Furthermore, the extracted pigment exhibited significantly stronger activity for eliminating HCV and HBV in infected human blood mononuclear cells than currently used drugs. This antiviral activity may be attributed to its inhibitory potential against HCV RNA and HBV DNA polymerases, which thereby suppresses HCV and HBV replication, as indicated by a high viral clearance % in the treated cells. These novel findings suggest that the C<sub>50</sub> carotenoid of <i>Natrialba</i> sp. M6 can be used as an alternative source of natural metabolites that confer potent anticancer and antiviral activities.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Biotechnology</div> <div class="kwd" title="kwd">
Microbiology</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Halophilic archaea (haloarchaea) belong to the family Halobacteriaceae. This family includes a group of microorganisms that are able to live in hypersaline environments with high salt concentrations (up to 4 M), such as solar salterns, salt lakes and salt deposits<sup><a href="#CR1">1</a></sup>. Interestingly, these organisms have received increasing attention due to their ability to produce a plethora of compounds with potential applications in many fields of biotechnology, including salt-tolerant enzymes, biodegradable polyesters, exopolysaccharides, antimicrobial halocins, biosurfactants, and photon-driven retinal protein<sup><a href="#CR2">2</a></sup>. Most haloarchaeal species (e.g., <i>Natrialba</i>) can produce pigments, including carotenoids. <i>Natrialba</i> is an organism in our research that belongs to the Halobacteriaceae family. However, the genus has recently been reassigned to the novel family Natrialbaceae<sup><a href="#CR3">3</a></sup>. <i>Natrialba</i> sp. M6 is an extreme haloalkaliphile that grows at pH 10.0 and 20–25% w/v NaCl and utilizes a wide range of carbohydrate and noncarbohydrate substrates.</p> <p id="Par3">Carotenoids are lipid-soluble pigments that vary in colour between yellow, orange, and red. Carotenoids are classified based on the number of carbons in their backbones into the categories C<sub>30</sub>, C<sub>40</sub> and C<sub>50</sub>. Most carotenoids exist as a C<sub>40</sub> structure in different living organisms, including bacteria, archaea, fungi, algae, and plants<sup><a href="#CR4">4</a></sup>. Meanwhile, haloarchaea can produce C<sub>50</sub> bacterioruberin (BR), a rare carotenoid form that contains four hydroxyl groups<sup><a href="#CR5">5</a></sup>. These pigments are divided into two major groups: xanthophylls (molecules containing oxygen), such as lutein and zeaxanthin, and carotenes (nonoxygenated molecules), such as α-carotene and lycopene<sup><a href="#CR6">6</a></sup>. The presence of carotenoids in the membrane of archaeal cells may help cells adapt to hypersaline environments by acting as a water barrier and allowing ions and oxygen molecules to permeate through the cell membrane. Additionally, BR protects bacteria from damage under intensive light and against free radical- and UV-mediated oxidative DNA damage<sup><a href="#CR7">7</a></sup>. Therefore, these carotenoids can stabilize archaeal cells under high osmotic and oxidative stresses<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>. The biological function of carotenoids is mostly attributed to their antioxidant properties that protect cells from oxidative damage; consequently, they improve human health<sup><a href="#CR10">10</a></sup>.</p> <p id="Par4">However, haloarchaeal carotenoids were first studied in the 1960s<sup><a href="#CR11">11</a></sup>. Few studies during the last half of the 20<sup>th</sup> century focused on haloarchaeal carotenoids compared with carotenoids of other organisms. In addition, little information on the biological properties of haloarchaeal carotenoids is available, and there is an increasing need for new sources of such important bioactive compounds. Hence, after partial extraction of carotenoids produced by the haloalkaliphilic archaeon <i>Natrialba</i> sp. M6, we evaluated their <i>in vitro</i> anticancer activity by investigating their potential for the induction of apoptosis-dependent cell death and for blocking matrix metalloprotease (MMP) 9-dependent cancer progression (angiogenesis and metastasis)<sup><a href="#CR12">12</a></sup>. Additionally, we investigated the antiviral activity of these carotenoids by assessing their impact on hepatitis C virus (HCV) RNA-dependent RNA and hepatitis B virus (HBV) DNA-dependent DNA polymerase-mediated viral replication. This evaluation will open the door for more research on the possible therapeutic applications of carotenoids as natural remedies.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="isolation,screeningandmolecularidentification" title="sec"> <div class="title" tagx="title" title="title">
Isolation, screening and molecular identification</div> <p id="Par5">In this study, 37 microbial isolates were recovered from different sites of water and sediments from El-Hamra Lake, Wadi El-Natrun and have been categorized as archaea and selected for the subsequent work. All isolates were screened for production of some haloalkaliphilic biocatalysts, pigment, biosurfactant and other biochemical testes. Due to distinct characteristics for the isolate coded M6 concerning tested biochemical activities in comparison to other isolates (37); it was selected for completion of this work and subjected to molecular identification. Subsequently, the isolate was identified as <i>Natrialba</i> sp. M6 and kept in GeneBank under accession number (ac: MK063890).</p> </div> <div class="phenotypiccharacterizationandgrowthpatternofthearchaeonnatrialbasp.m6" title="sec"> <div class="title" tagx="title" title="title">
Phenotypic characterization and growth pattern of the archaeon <i>Natrialba</i> sp. M6</div> <p id="Par6"><i>Natrialba</i> sp. M6 (under accession number ac: MK063890) is Gram-negative and forms small, round, smooth, orange-pigmented colonies after one week of incubation at 37 °C <b>(</b>Fig. <a href="#Fig1">1A,B</a><b>)</b>. A scanning electron microscopy (SEM) study revealed rod-shaped cells ranging from 0.613 to 0.925 μm in length and from 0.441 to 0.516 μm in width <b>(</b>Fig. <a href="#Fig1">1C</a><b>)</b>. The growth pattern of <i>Natrialba</i> sp. M6 is presented in Fig. <a href="#Fig1">1D</a>, where a lag phase of 3 days was observed before the log phase began; the log phase extended to the 10<sup>th</sup> day, and then the stationary phase started. Eleven days later, the organism entered the death phase with a gradual reduction in cell dry weight; however, this was associated with an increase in the optical density (OD) <b>(</b>Fig. <a href="#Fig1">1D</a><b>)</b>. This is due to interfering the color of pigment (released outside due to cell death) with turbidity measurements (OD<sub>600nm</sub>) and water evaporation <b>(</b>Fig. <a href="#Fig1">1D</a><b>)</b>. By this way the dry weight is considered an accurate measurable term to indicate the growth rather than OD. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Phenotypic characterization and growth curve of the archaeon <i>Natrialba</i> sp. M6. Orange pigmented colonies of archaeon M6 grown on (<b>A</b>) M5 agar and in (<b>B</b>) M5 broth. (<b>C</b>) Scanning micrograph at a magnification of 20,000×. (<b>D</b>) Growth curve of <i>Natrialba</i> sp. M6 based on optical density (OD) and dry weight measurements.</p>   </div> <p /> </div> <div class="physiologicalcharacterizationofthearchaeonnatrialbasp.m6" title="sec"> <div class="title" tagx="title" title="title">
Physiological characterization of the archaeon <i>Natrialba</i> sp. M6</div> <p id="Par7">After seven days incubation, the growth of <i>Natrialba</i> sp. M6 (g/L) was recorded in reference to NaCl concentration, pH and temperature variation (data not shown). The bacterium failed to grow in absence or presence of 10%, 1.71 M NaCl and the maximum growth (6.9 g) was achieved in presence of 15%, 2.55 M NaCl and declined thereafter. Medium adjusted to pH 7 and 8 supported low growth, which increased gradually with pH increase reaching the maximum value (6 g) at pH 10, while higher pH values (11 and 12) caused a remarkable decrease in growth. The bacterium coded M6 appeared to favor growth in a range of temperature (37–50 °C) and recorded the best (7.5 g) at 45 °C.</p> </div> <div class="optimizationofnatrialbasp.m6pigmentproductionusingbox-behnkendesign" title="sec"> <div class="title" tagx="title" title="title">
Optimization of <i>Natrialba</i> sp. M6 pigment production using Box-Behnken design</div> <p id="Par8">The most significant independent variables tested initially (data not shown) (sodium chloride, X1; pH, X2; inoculum age, X3) were explored at three levels (encoded as 1, 0, and −1). Table <a href="#Tab1">1</a> presents the design matrix, real variable levels and the response of each trial, where the response variable here was the concentration of pigment (g/L) determined by measuring the absorbance of the pigment at 350 nm. In this experiment, the value of the <i>determination coefficient R</i><sup>2</sup> was 0.813 for pigment formation, indicating a high degree of correlation between the experimental and predicted values. To predict the optimal variable values, a second-order polynomial function was fitted to the experimental response results (nonlinear optimization algorithm; Y = 0.79123 + 0.15975X1 − 0.037187X2 − 0.001788X3 + 0.078305X1X2 + 0.149137X1X3 + 0.126206X2X3 − 0.121676X1<sup>2</sup> − 0.145287X2<sup>2</sup> − 0.04965X3<sup>2</sup>). Additionally, Fig. <a href="#Fig2">2</a> shows the simultaneous effects of the three most significant independent factors on each response using three-dimensional graphs generated by Statistica 5.0 software. The presented data <b>(</b>Fig. <a href="#Fig2">2A</a><b>)</b> shows that, a great interaction between pH and NaCl at high values and the maximum level of carotenoid synthesis appeared. However, the plot in Fig. <a href="#Fig2">2B</a> indicates that the curve had high values near the inoculum age axis, with a maximum response near its high values. This indicates that high NaCl and using an old culture were preferable for increased carotenoid synthesis by <i>Natrialba</i> sp. M6, and it reached a maximum along the pH axis, indicating that old cultures with middle values of pH gave the maximum carotenoid yield <b>(</b>Fig. <a href="#Fig2">2C</a><b>)</b>. Thus, the optimal values of the three factors, as obtained from the maximum point of the polynomial model, was recorded as 25 (%w/v) NaCl, pH 10.07 and an inoculum age of 11 days (Table <a href="#Tab2">2</a>), with the predicted product concentration equal to 0.98227 g/L. Table <a href="#Tab2">2</a> presents the optimum formula for maximum pigment production by <i>Natrialba</i> sp. M6 compared to the basal medium. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Box-Behnken factorial experimental design for pigment formation by <i>Natrialba</i> sp. M6.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Trial</th> <th>NaCl</th> <th>pH</th> <th>Inoculum age</th> <th>Response*</th> <th>Pigment conc. g/L</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>0 (20)</td> <td>0 (10)</td> <td>0 (9)</td> <td>1.176</td> <td>0.7867</td> </tr> <tr> <td>2</td> <td>0 (20)</td> <td>0 (10)</td> <td>0 (9)</td> <td>1.183</td> <td>0.7915</td> </tr> <tr> <td>3</td> <td>0 (20)</td> <td>−1 (9)</td> <td>1 (11)</td> <td>0.59</td> <td>0.3885</td> </tr> <tr> <td>4</td> <td>0 (20)</td> <td>1 (11)</td> <td>−1 (7)</td> <td>0.83</td> <td>0.5516</td> </tr> <tr> <td>5</td> <td>−1 (15)</td> <td>−1 (9)</td> <td>0 (9)</td> <td>0.98</td> <td>0.6535</td> </tr> <tr> <td>6</td> <td>0 (20)</td> <td>0 (10)</td> <td>0 (9)</td> <td>1.189</td> <td>0.7955</td> </tr> <tr> <td>7</td> <td>1 (25)</td> <td>0 (10)</td> <td>−1 (7)</td> <td>1.032</td> <td>0.6889</td> </tr> <tr> <td>8</td> <td>0 (20)</td> <td>1 (11)</td> <td>1 (11)</td> <td>1.152</td> <td>0.7704</td> </tr> <tr> <td>9</td> <td>0 (20)</td> <td>−1 (9)</td> <td>−1 (7)</td> <td>1.011</td> <td>0.6746</td> </tr> <tr> <td>10</td> <td>−1 (15)</td> <td>0 (10)</td> <td>1 (11)</td> <td>0.39</td> <td>0.2527</td> </tr> <tr> <td>11</td> <td>−1 (15)</td> <td>0 (10)</td> <td>−1 (7)</td> <td>0.79</td> <td>0.5244</td> </tr> <tr> <td>12</td> <td>1 (25)</td> <td>0 (10)</td> <td>1 (11)</td> <td>1.51</td> <td>1.0136</td> </tr> <tr> <td>13</td> <td>1 (25)</td> <td>1 (11)</td> <td>0 (9)</td> <td>0.83</td> <td>0.5516</td> </tr> <tr> <td>14</td> <td>1 (25)</td> <td>−1 (9)</td> <td>0 (9)</td> <td>1.009</td> <td>0.6732</td> </tr> <tr> <td>15</td> <td>−1 (15)</td> <td>1 (11)</td> <td>0 (9)</td> <td>0.34</td> <td>0.2187</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*Response is the absorbance of the pigments after extraction measured at a wavelength of 350 nm.</p> <p>Coded values: (−1) low level; (+1) high level and (0) middle level.</p> <p>Levels of studied variables presented between brackets are expressed in terms of (%wt/v), value and days for X1, X2 and X3, respectively.</p> </div>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Three-dimensional surface plots showing the relationships between the tested significant variables and pigment formation by <i>Natrialba</i> sp. M6 and the optimal levels of the three factors as obtained from the maximum of the polynomial model. The three figures indicate (<b>A</b>) the interaction between pH and NaCl, (<b>B</b>) the interaction between NaCl and inoculum age and (<b>C</b>) the interaction between pH and inoculum age.</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>The optimum formula for maximum pigment production by <i>Natrialba</i> sp. M6.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Components</th> <th>Basal medium</th> <th>Optimized medium</th> </tr> </thead> <tbody> <tr> <td>Temperature (°C)</td> <td>37</td> <td>45</td> </tr> <tr> <td>pH</td> <td>11</td> <td>10.07</td> </tr> <tr> <td>Casaminoacids (%w/v)</td> <td>0.5</td> <td>0.5</td> </tr> <tr> <td>NaCl (%w/v)</td> <td>20</td> <td>25</td> </tr> <tr> <td>Na<sub>2</sub>CO<sub>3</sub> (%w/v)</td> <td>0.9</td> <td>0.9</td> </tr> <tr> <td>Agitation (rpm)</td> <td>200</td> <td>200</td> </tr> <tr> <td>MgSO<sub>4</sub>.7H<sub>2</sub>O (%w/v)</td> <td>0.02</td> <td>0.02</td> </tr> <tr> <td>KH<sub>2</sub>PO<sub>4</sub> (%w/v)</td> <td>0.1</td> <td>0.1</td> </tr> <tr> <td>Trace element solution* (ml %)</td> <td>0.1</td> <td>0.1</td> </tr> <tr> <td>Old inoculum (days)</td> <td>7</td> <td>11</td> </tr> <tr> <td>Final volume (ml)</td> <td>100</td> <td>100</td> </tr> <tr> <td>Pigment (g/L)</td> <td>0.23</td> <td>0.98226</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*Trace element solution contains (g/L): ZnSO<sub>4</sub>.7H<sub>2</sub>O, 0.1; MnCl<sub>2</sub>.4H<sub>2</sub>O, 0.03; H<sub>3</sub>BO<sub>3</sub>, 0.3; CoCl<sub>2</sub>.6H<sub>2</sub>O, 0.2; CuCl<sub>2</sub>.2H2O, 0.01; NiCl<sub>2</sub>.6H<sub>2</sub>O, 0.02; and Na<sub>2</sub>MoO<sub>4</sub>.H<sub>2</sub>O, 0.03.</p> </div> </div> <p /> </div> <div class="identificationofnatrialbasp.m6pigment" title="sec"> <div class="title" tagx="title" title="title">
Identification of <i>Natrialba</i> sp. M6 pigment</div> <p id="Par9">The cultivation of the archaeon <i>Natrialba</i> sp. M6 was carried out according to our abovementioned optimized conditions by incubation at 45 °C with shaking at 200 rpm for 10 days and with inoculum aged 11 days. Pigments were then extracted by liquid extraction using acetone and methanol. It was found that 3.15 g of biomass cells gave approximately 9.8 mg of pigment after extraction and drying.</p> <p id="Par10">Fourier transform infrared (FTIR) spectroscopy, Raman spectroscopy, gas chromatography-mass spectrometry (GC-MS), liquid chromatography-mass spectrometry (LC-MS) and nuclear magnetic resonance spectroscopy NMR were carried out to elucidate the chemical structure of the partially purified pigment of <i>Natrialba</i> sp. M6 <b>(</b>Fig. <a href="#Fig3">3</a><b>)</b>. The FTIR spectroscopic analyses revealed the presence of various chemical groups in the pigment, as shown in Fig. <a href="#Fig3">3A</a>. The peaks appearing at 543.94 and 1020.38 cm<sup>−1</sup> correspond to C-Br and C-F stretching frequencies, respectively. A band at 1134.18 cm<sup>−1</sup> corresponding to C-OH was also detected. The peak at 1263.42 cm<sup>−1</sup> is assigned to C-O-C. Meanwhile, peaks appearing at 1408.08 and 1454.38 cm<sup>−1</sup> correspond to CH<sub>3</sub> and CH<sub>2</sub> stretching frequencies, respectively. The strong peak at 1637.62 cm<sup>−1</sup> is assigned to C=C alkene stretching, indicating the presence of some aliphatic compounds in this extract. The Raman spectroscopy analysis <b>(</b>Fig. <a href="#Fig3">3B</a><b>)</b> revealed the presence of three regions with two strong and weak signal intensities for orange carotenoids, with a strong band at 1380 cm<sup>−1</sup>, which corresponds to the CH<sub>3</sub> umbrella mode, and a weak band at 1883 cm<sup>−1</sup>, corresponding to C=C bonds. GC-MS analysis of the partially extracted pigment identified each chromatographic peak and a peak corresponding to the major C<sub>40</sub> carotenoid (C<sub>40</sub>H<sub>56</sub>O<sub>10</sub>) nephthoside 1,2′,3′,4′-tetraacetate, as shown in Fig. <a href="#Fig3">3C</a>. In addition, LC-MS analyses revealed the presence of unlike components in the pigment which may be carotenoids <b>(</b>Fig. <a href="#Fig3">3DI,II</a>). Also, the NMR-analysis confirmed the presence of characteristic olefinic proton in aromatic and aliphatic region <b>(</b>Fig. <a href="#Fig3">3E</a><b>)</b> where, CH3 groups were detected in the region 0.8–1.2 ppm of NMR chart. This finding confirmed the former results of the FTIR analyses. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Chemical characterization of the extracted <i>Natrialba</i> sp. M6 pigment (<b>A</b>) FTIR, (<b>B</b>) Raman spectroscopy, (<b>C</b>) GC-MS, (<b>DI</b> &amp;amp; <b>II</b>) LC-MS and (<b>E</b>) <sup>13</sup>C, <sup>1</sup>H NMR analyses.</p>   </div> <p /> </div> <div class="evaluationofthepigmentcompatibilitywithredbloodcells" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of the pigment compatibility with red blood cells</div> <p id="Par11">To evaluate the hemolytic effect of the extracted pigment, it was incubated with human red blood cells (RBCs). Then the released hemoglobin was quantified at its serial concentrations to estimate doses (EC<sub>100</sub> and IC<sub>50</sub>) that did not cause any hemolysis and 50% hemolysis, respectively, relative to the untreated RBCs. We found that 0.792 ± 0.058 mg/ml of the prepared pigment was safe without generating any hemolysis while 4.699 ± 0.0706 mg/ml of it ruptured RBCs by 50%.</p> </div> <div class="invitroanticancerefficacyoftheextractedpigment" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> anticancer efficacy of the extracted pigment</div> <div class="thecytotoxicityofthenatrialbasp.m6pigmentonhumannormalandcancercelllines" title="sec"> <div class="title" tagx="title" title="title">
The cytotoxicity of the <i>Natrialba</i> sp. M6 pigment on human normal and cancer cell lines</div> <p id="Par12">The cytotoxic effect of the studied pigment was determined against normal human lung fibroblast cells (Wi-38) by estimating the pigment concentrations (EC<sub>50</sub> and EC<sub>100</sub>) at 50% and 100% cell viability. Figure <a href="#Fig4">4A</a> illustrates that the EC<sub>50</sub> and EC<sub>100</sub> of the extracted pigment (85.14 ± 2.31 and 31.6 ± 0.96 µg/ml, respectively) were significantly higher than those of a currently used chemotherapy agent (5-fluorouracil, 5-FU, 10.28 ± 0.67 and 3.3 ± 0.48 µg/ml, respectively). This indicates that the extracted pigment was significantly (p &amp;lt; 0.0005) safer than 5-FU towards human normal cells. However, 5-FU caused 50% cell death for Caco-2 (colon cancer line), MCF-7 (breast cancer cell line), HepG-2 (liver cancer line), and HeLa (cervical cancer cell line) cells at a dose (7.17–10.36 µg/ml) less than that (21.18–38.24 µg/ml) of the studied pigment <b>(</b>Fig. <a href="#Fig4">4B</a><b>)</b>. The IC<sub>50</sub> values of 5-FU against human cancer cells were higher than its safe dose (EC<sub>100</sub>), clarifying that the anticancer doses of 5-FU are toxic to normal cells. On the other hand, IC<sub>50</sub> values of pigment for human cancer cell lines did not obviously exceed its EC<sub>100</sub> limit. This was confirmed by the pigment having a higher selectivity index (SI &amp;gt; 2) between normal and cancer cells than 5-FU does (SI &amp;lt; 1.7) <b>(</b>Fig. <a href="#Fig4">4C</a><b>)</b>. Furthermore, Fig. <a href="#Fig4">4D</a> demonstrates the morphological changes (loss of spindle shapes) in pigment-treated human cancer cell lines (Caco-2, MCF-7, HepG-2 and HeLa) without any marked alterations in the shapes of pigment-treated normal cells (wi-38). In contrast, cell damage was observed in both normal and cancer cells treated with 5-FU, as shown in Fig. <a href="#Fig4">4D</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Cytotoxicity effect of the extracted pigments on human normal and cancer cells. (<b>A</b>) EC<sub>50</sub> and EC<sub>100</sub> (µg/ml) of pigment extract and the currently used chemotherapy agent (5-FU) towards the human normal cell line (Wi-38) and (<b>B</b>) their IC<sub>50</sub> (µg/ml) values against four human cancer cell lines. (<b>C</b>) Selectivity index (SI) of pigments against cancer cells compared with that of 5-FU. (<b>D</b>) Variation in the morphology of normal and cancer cells after treatment with the extracted pigments and 5-FU (magnification ×200). All data are expressed as the mean ± SE with significance (unpaired t-test analysis) at p-value &amp;lt; 0.05*, &amp;lt;0.005**, &amp;lt;0.0005***.</p>   </div> <p /> <p id="Par13">Apoptosis-dependent cell death was quantified by flow cytometric analysis of untreated and treated cells after nuclear staining with annexin and propidium iodide (PI). Figure <a href="#Fig5">5A,B</a> shows the flow cytometric analysis of the annexin-stained apoptotic cell population in pigment-treated cancer cells, which was 48.87–53.55%, contrasting the less than 39% population obtained with 5-FU. This suggests that the extracted pigment was more potent as an inducer of apoptosis in all tested cancer cells (Caco-2, MCF-7, HepG-2 and HeLa) than the reference chemotherapy agent 5-FU. For additional confirmation, the induction of apoptosis was also evaluated by dual nuclear staining with acridine orange and ethidium bromide using a fluorescence microscope. Green fluorescence indicates healthy viable cells, while yellow/orange and red fluorescence are signs of early and late apoptotic cells, respectively. As shown in Fig. <a href="#Fig5">5C</a>, pigment-treated cancer cells mainly exhibited yellow, orange and red fluorescence, while 5-FU-treated cancer cells still demonstrated green fluorescence in four cancer cell lines. Moreover, the percentage of caspase 3/7 activation in pigment-treated Caco-2 cells relative to that of untreated cells (76.803%) indicates that this pigment is a more efficient caspase activator than 5-FU (40%, p &amp;lt; 0.005) <b>(</b>Fig. <a href="#Fig5">5D</a><b>)</b>. This supports that this pigment has a greater caspase-induced apoptotic effect than 5-FU on all studied human cancer cell lines. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Apoptotic effect of the extracted pigments on human cancer cells. (<b>A</b>) The apoptosis percentages in pigment- and 5-FU-treated cancer cells after staining with annexin and propidium iodide relative to the percentages in untreated cancer cells with (<b>B</b>) Dot plot flow charts of the untreated and treated cancer cells. (<b>C</b>) Fluorescence images of pigment-treated cancer cells compared with those of 5-FU-treated cancer cells and untreated cancer cells after staining with acridine orange and ethidium bromide and investigation using fluorescence microscopy (green, yellowish-orange, and red fluorescence indicates viable, early apoptotic and late apoptotic cells, respectively) (magnification ×200). (<b>D</b>) The superior potency of pigments relative to 5-FU as caspase 3/7 activators in the treated Caco-2 cells. (<b>E</b>) IC<sub>50</sub> values reflecting the inhibitory effect of pigments and 5-FU on matrix metalloprotease (MMP) 9. All data are expressed as the mean ± SE with significance (unpaired t-test analysis) at p-value &amp;lt; 0.05*, &amp;lt;0.005**, &amp;lt;0.0005***.</p>   </div> <p /> </div> <div class="thepowerfulinhibitorypotentialofthepigmentonmmp-9activity" title="sec"> <div class="title" tagx="title" title="title">
The powerful inhibitory potential of the pigment on MMP-9 activity</div> <p id="Par14">We detected the suppressor effect of the extracted pigment on one of the key proteases (MMP-9) involved in several cancer pathological processes, including invasion, angiogenesis and metastasis. As shown in Fig. <a href="#Fig5">5E</a>, our pigment had a lower IC<sub>50</sub> value than 5-FU for inhibiting MMP-9 (0.484 and 175.12 µg/ml, respectively). This result indicates that this pigment possessed a significantly higher inhibitory impact (p &amp;lt; 0.0005) on MMP-9 than 5-FU. The antiviral activity of this pigment was assessed and compared with that of currently used drugs for HCV and HBV (sofosbuvir and lamivudine, respectively). Before the antiviral activity of the extracted pigment was measured, a safe dose was determined for the host cells of HCV and HBV (peripheral blood mononuclear cells, PBMCs) using the MTT assay. The safe doses (EC<sub>100</sub>) of the extracted pigment and standard antiviral drugs (sofosbuvir and lamivudine) were 436.73 ± 1.74, 1063.9 ± 141.46 and 1083.36 ± 11.15 µg/ml, respectively. The pigments and drugs were separately incubated (96 h) at these estimated doses with HCV- and HBV-infected PBMCs, and then the viral load was quantified by the fully automated Cobas Ampliprep Cobas TaqMan (CAP-CTM) analyser. As illustrated in Fig. <a href="#Fig6">6A</a>, the pigment was able to completely eliminate HCV, and approximately 89.42% of HBV was cleared; in comparison, antiviral drugs cleared 96.14% of HCV (sofosbuvir) and 77.10% of HBV (lamivudine). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Anti-HCV and anti-HBV activity of the extracted pigments. (<b>A</b>) The viral clearance percentages in HCV- and HBV-infected peripheral blood mononuclear cells (PBMCs) after treatment with pigments or standard viral drugs (sofosbuvir and lamivudine for HCV and HBV, respectively) at their safe doses (these percentages were estimated elative to those in untreated infected PBMCs). (<b>B</b>) IC<sub>50</sub> values of pigments and standard viral drugs for inhibition of HCV-NS5B polymerase and DNA-dependent HBV polymerase. All data are expressed as the mean ± SE with significance (unpaired t-test analysis) at p-value &amp;lt; 0.05*, &amp;lt;0.005**, &amp;lt;0.0005***.</p>   </div> <p /> </div> <div class="inhibitorypotentialofthetestedpigmentagainstpolymerase-dependentviralreplication" title="sec"> <div class="title" tagx="title" title="title">
Inhibitory potential of the tested pigment against polymerase-dependent viral replication</div> <p id="Par15">Figure <a href="#Fig6">6B</a> shows that the pigment inhibited HCV RNA- and HBV DNA-dependent polymerases by 50% at 21.65 ± 0.25 and 4.99 ± 0.13 µg/ml, respectively, in comparison with 59.87 ± 4.59 µg/ml (HCV) for sofosbuvir and 1.642 ± 0.49 µg/ml (HBV) for lamivudine. This result suggests that the tested pigment had a higher inhibitory effect on HCV RNA-dependent RNA polymerase (NS5B) than sofosbuvir did (p &amp;lt; 0.05). Meanwhile, this pigment revealed a lower suppressor impact on HBV DNA-dependent DNA polymerase than lamivudine did (p &amp;lt; 0.05).</p> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par16">Haloarchaea, such as <i>Natrialba</i> sp., are good candidates for carotenoid production at a large scale. In addition, they have the advantage of inexpensive production and short culture processing times. Due to the lack of studies on the benefits of the carotenoids produced by haloarchaea <i>Natrialba</i> sp. On human health, more efforts should be made to answer the rising questions about the pharmacological importance of its metabolites. In the current study, maximal carotenoid production was attained using old cultures of <i>Natrialba</i> sp. M6, where the pH was adjusted to 9–11. Pigments are usually produce as a secondary metabolites, thus it is noteworthy that old cultures showed a significant increase in the production of archaeal carotenoid<sup><a href="#CR13">13</a></sup>. Moreover, Rodriguez-Valera <i>et al</i>., reported a variation of pigmentation according to the salt concentration of the medium in the genus Halobacterium<sup><a href="#CR14">14</a></sup>. Thus, NaCl, pH and culture age were further investigated as significant independent variables in a Box-Behnken design to identify the optimum response region for carotenoid biosynthesis in <i>Natrialba</i> sp. M6. Then, a second-order polynomial function was fitted to correlate the relationship between the variables and the responses to them. The results indicated that the optimal levels of the three factors were 25% NaCl, pH 10.07, and 11-day-old culture. Sikkander <i>et al</i>. used a Box-Behnken design to determine the optimum factors affecting carotenoid production by <i>Halorubrum</i> sp. TBZ126, and the estimated coefficients and the corresponding <i>p</i>-values indicated that X1 (temperature), X2 (pH) and X3 (salinity) were significant factors for both cell growth and carotenoid production by this strain<sup><a href="#CR15">15</a></sup>. After extraction, the orange pigment in this study was characterized using FTIR, Raman spectroscopy, GC-MS, LC-MS and NMR methods. An analysis of the resultant peaks identified a carotenoid structure due to the the presence of olefinic proton in aromatic and aliphatic region and compounds with molecular masses close to those of oxygenated C<sub>50</sub> compounds with methyl groups (CH<sub>3</sub>) and conjugated double bonds. In turn, this result supports the notion that the isolated compounds are related to the xanthophyll carotenoid pigments that consist of oxygenated hydrocarbons with alternating single and double bond and exhibit yellow to orange colour. Various pigments, including phytoene, β-carotene, lycopene and salinixanthin carotenoids, are produced by <i>Halobacterium salinarum</i><sup><a href="#CR16">16</a></sup>. According to Torregrosa-Crespo <i>et al</i>., the caretenoids produced by haloarchaeo are Astaxanthin C<sub>40</sub>H<sub>52</sub>O<sub>4</sub>, Lutein C<sub>40</sub>H<sub>56</sub>O<sub>2</sub>, Canthaxanthin C<sub>40</sub>H<sub>52</sub>O<sub>2</sub>, Bacterioruberin C<sub>50</sub>H<sub>76</sub>O<sub>4</sub> and Salinixanthin C<sub>61</sub>H<sub>92</sub>O<sub>9</sub><sup><a href="#CR17">17</a></sup>, but no records for the production of the compound found in this study which contains C<sub>40</sub>H<sub>56</sub>O<sub>10</sub> so far. Therefore, this compound is considered unique and specific for this strain and may be related to carotenoids due to the presence of olefinic proton. In addition Torregrosa-Crespo <i>et al</i>., reported that there are no examples of a large-scale production of these caretenoids from haloarchae.</p> <p id="Par17">In view of the promising characteristics of carotenoids, more attention has recently been given to discovering novel natural safe sources of carotenoids with an effective impact on human diseases (cancer and viral hepatitis). In our study, the concentration of <i>Natrialba</i> sp M6 pigment extract up to 0.79 mg/ml did not demonstrate any hemolysis effect but at 4.699 mg/ml caused hemolysis by 50%. Furthermore, the <i>Natrialba</i> sp. M6 pigment extract was safer than the standard anticancer drug 5-FU against normal human cells, where the EC<sub>100</sub> of the studied pigment was 9-fold that of 5-FU. This extracted pigment also revealed potent antiproliferative and apoptotic activity against colon, breast, liver and cervical cancer cells. Previous study, in line with our findings, reported the antiproliferative effect of haloarchaeal carotenoids of <i>Halogeometricum limi</i> and <i>Haloplanus vescus</i> on the HepG-2 cell line at dose &amp;gt;45 µg/ml<sup><a href="#CR18">18</a></sup>. This indicates that our pigment of <i>Natrialba</i> sp. M6 strain exhibited a higher anticancer effect, causing 50% death in all four cancer cell lines at doses &amp;lt;39 µg/ml, compared to the above-mentioned pigments. Cellular apoptosis occurs via activation of the proteolytic caspase cascade. Caspase 3 is the most important executor of endonuclease-mediated DNA fragmentation, and the formation of apoptotic bodies eventually leads to rapid irreversible cell death. The surface of these apoptotic bodies is characterized by the presence of phosphatidylserine as a ligand of phagocytes. Hence, annexin V (recombinant phosphatidylserine-binding protein) can be used for the quantification of apoptosis<sup><a href="#CR19">19</a></sup>. In the current study, annexin V-stained apoptotic populations were &amp;gt;48% of the tested pigment-treated human cancer cell lines (Caco-2, MCF-7, HepG-2, HeLa), whereas 5-FU-treated cancer cells resulted in significantly smaller annexin V-stained apoptotic ppulations (31.37–38.01%). Moreover, the caspase-dependent apoptotic impact of the extracted pigment for stimulating cancer cell death was declared by their higher relative % of caspase activation than 5-FU <b>(</b>Fig. <a href="#Fig5">5D</a><b>)</b>. Recent literature reported that supernatant metabolites of <i>H. salinarum</i> IBRC M10715 and red pigment-producing HDZK-BYSB107 were able to induce apoptosis in prostate cells via upregulation of caspase 3<sup><a href="#CR20">20</a>,<a href="#CR21">21</a></sup>.</p> <p id="Par18">The prevention of cancer progression is a promising therapeutic strategy against cancer. MMP-9 plays critical roles in cancer pathologies, including tumour angiogenesis, invasion and metastasis and mediates the tumour microenvironment by remodeling the extracellular matrix and altering cell-cell interactions and the cleavage of cell surface proteins<sup><a href="#CR12">12</a></sup>. We have therefore investigated the impact of the extracted pigment on MMP-9 and found that this pigment repressed 50% of MMP-9 activity at less than 0.5 µg/ml significantly more effectively than 5-FU did 50% inhibition at ~175 µg/ml. Resrearchers<sup><a href="#CR22">22</a></sup> found that lycopene inhibited MMP-9 activity in cigarette smoke extract-exposed macrophages by blocking the Ras signalling pathway<sup><a href="#CR22">22</a></sup>. Meanwhile, no previous studies have yet investigated the direct effect of BR on MMP-9 activity.</p> <p id="Par19">The HCV and HBV are the common public types of hepatitis that lead to liver fibrosis, cirrhosis and hepatocellular carcinoma. The prevalence rate of HCV or HBV is above 150 million patients worldwide. The therapeutic target for HCV and HBV is blocking viral replication without any side effects on normal cells, so novel safe natural products are urgently needed<sup><a href="#CR23">23</a></sup>. More interestingly, the present study explored the potent antiviral activity of the extracted carotenoids against HCV and HBV by inhibiting HCV NS5B polymerase and HBV DNA-dependent DNA polymerase which eventually suppresses HCV and HBV replication. The latter effect was indicated by the increased clearance percentage of both viruses after treatment of the infected PBMCs with pigment in comparison with that after treatment with currently used antiviral drugs<sup><a href="#CR24">24</a></sup>. Sun <i>et al</i>., found that prodigiosin (tripyrrole red pigment) inhibited DNA replication of <i>Bombyx mori</i> nucleopolyhedrovirus<sup><a href="#CR24">24</a></sup>. Additionally, Goh <i>et al</i>., illustrated the RNA polymerase (NS5B) inhibition-dependent anti-HCV activity of <i>Monascus</i> orange pigment, which suppresses NS5B activity by 28% at a concentration of 10 µM<sup><a href="#CR25">25</a></sup>. However, to date, no prior research has investigated the anti-HCV and anti-HBV activities of BR.</p> </div> <div class="conclusion" title="conclusion"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par20">We suggest here for the first time that the extracted pigment from <i>Natrialba</i> sp. M6 (ac: MK063890) could be a new component against both cancer and viral hepatitis. This studied pigment not only exhibited caspase-induced apoptotic effects in cancer cells at noncytotoxic doses but also suppressed MMP-9-mediated cancer progression. Furthermore, this pigment was able to inhibit polymerase-dependent HCV and HBV replication. Therefore, the impact of this pigment on the other main mediators of metastatic cancer and viral hepatitis (particularly HCV) will need further investigation using animal models.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="isolationsource,culturemediumandscreening" title="sec"> <div class="title" tagx="title" title="title">
Isolation source, culture medium and screening</div> <p id="Par21">Isolation samples were collected from Egyptian- extreme environment Wadi El-Natrun (salinities ranging from 1.5 to 5 M NaCl, pH values between 8.5 and 11and temperature up to 50 °C. The culture medium used in this study was prepared using distilled water. The pH was adjusted to 11 using Na<sub>2</sub>CO<sub>3</sub>, and the medium was sterilized by autoclaving at 120 °C for 20 min. The composition of the isolation basal medium was as follows (g/l): casamino acids, 5; KH<sub>2</sub>PO<sub>4</sub>, 1; MgSO<sub>4</sub>.7H<sub>2</sub>O, 0.2; NaCl, 200; trace metals, 1 ml; and Na<sub>2</sub>CO<sub>3</sub>, 18. The trace metal solution contained (g/L) ZnSO<sub>4</sub>.7H<sub>2</sub>O, 0.1; MnCl<sub>2</sub>.4H<sub>2</sub>O, 0.03; H<sub>3</sub>BO<sub>3</sub>, 0.3; CoCl<sub>2</sub>.6H<sub>2</sub>O, 0.2; CuCl<sub>2</sub>.2H2O, 0.01; NiCl<sub>2</sub>.6H<sub>2</sub>O, 0.02; and Na<sub>2</sub>MoO<sub>4</sub>.H<sub>2</sub>O, 0.03<sup><a href="#CR26">26</a></sup>. Biochemical screening for the obtained isolates was carried out in respect to different biocatalyst, biosurfactant and pigment production. Thereafter, an extremely haloalkaliphilic archaeon coded as M6 and isolated from El-Hamra Lake, Wadi El-Natrun, Egypt, was employed in this study.</p> </div> <div class="molecularidentification" title="sec"> <div class="title" tagx="title" title="title">
Molecular identification</div> <p id="Par22">For molecular identification, a simplified rapid protocol for preparing DNA from bacterial isolates was used in this study<sup><a href="#CR27">27</a></sup>. This method depends on a rapid disruption of cells from individual colonies picked from an agar medium<sup><a href="#CR28">28</a></sup>. A polymerase chain reaction (PCR) was carried out to amplify the 16S rDNA genes from archaeal genomes using universal primers designed to amplify ~1500 bp of this gene, which were then sequenced, and the BLAST program was used to assess similarity.</p> </div> <div class="microscopicexamination" title="sec"> <div class="title" tagx="title" title="title">
Microscopic examination</div> <p id="Par23">For electron microscopy, a cultured broth sample of the M6 isolate was metalized with a thin gold film using sputtering device 54 (JFC-1100 E, JEOL, USA) for 12 min. SEM was performed with a JSM 5300 scanning electron microscope (JEOL, USA) at 20 kV in the Centre Laboratory, City of Scientific Research and Technological Applications.</p> </div> <div class="preparationofinoculum" title="sec"> <div class="title" tagx="title" title="title">
Preparation of inoculum</div> <p id="Par24">Preculture grown for 7 days until the logarithmic phase (OD<sub>600nm</sub> of 0.99) was used as the inoculum (10%, (v/v)) in all the experiments unless stated.</p> </div> <div class="correlationbetweentheopticaldensityanddryweight" title="sec"> <div class="title" tagx="title" title="title">
Correlation between the optical density and dry weight</div> <p id="Par25">In this experiment, the archaeon (M6) was allowed to grow in 100 ml of the growth medium in a 250 ml conical flask. At time intervals, the OD was measured using a spectrophotometer. For dry weight determination, cultures were centrifuged, and cell pellets were washed twice with saline solution (0.9% (w/v) NaCl isotonic solution), centrifuged and dried at 105 °C until a constant weight<sup><a href="#CR29">29</a></sup>.</p> </div> <div class="effectofnaclconcentration,phandtemperatureonthegrowth" title="sec"> <div class="title" tagx="title" title="title">
Effect of NaCl concentration, pH and temperature on the growth</div> <p id="Par26">In this experiment, the growth of archaeon (M6) was evaluated with respect to variation in NaCl [0.85 M(5%)–5.10 M(30%)], pH (9–12) and temperature (30–50 °C), using dry weight as a measurable indicator for the growth in a fixed time (7 days).</p> </div> <div class="pigmentextractionandquantification" title="sec"> <div class="title" tagx="title" title="title">
Pigment extraction and quantification</div> <p id="Par27">Approximately 50 ml of the culture broth was centrifuged at 10,000 rpm for 20 min at 4 °C. The supernatant was discarded, and 50 ml of distilled solution was added to the pellets and then kept at 4 °C overnight. A mixture of acetone-methanol (7:3 v/v) containing butylhydroxytoluene (BHT) (0.1% as antioxidant) was added to the pellets. Successive extractions were carried out until both solvents and cells were colourless, and they were then again centrifuged. The solvent was evaporated at 45 °C overnight, and the pigment was dissolved in 50 ml of acetone containing 0.1% BHT. Samples were wrapped with aluminium foil to protect them from light. The coloured extraction solution was analysed by scanning the absorbance in the wavelength region of 200–700 nm, and then the pigment was quantified by measuring the OD at the wavelength (λ<sub>350nm</sub>) with the highest absorption and using an absorption coefficient value of 2660<sup><a href="#CR30">30</a></sup>.</p> </div> <div class="optimizationusingresponsesurfacemethodology(rsm)" title="sec"> <div class="title" tagx="title" title="title">
Optimization using response surface methodology (RSM)</div> <p id="Par28">After estimating the relative significance of independent variables, the most significant variables were selected for further determination of their optimal values. For this reason, Box-Behnken design (BBD), which is a response surface methodology, was applied<sup><a href="#CR31">31</a>,<a href="#CR32">32</a></sup>. This optimization process involves three main steps: performing the statistically designed experiments, estimating the coefficients in a mathematical model, predicting the response and checking the adequacy of the model. The three significant variables elucidated for the M6 isolate were NaCl (X1), pH (X2), and culture age (X3). The polynomial equation for the three factors tuning pigment production was as follows: Y = β<sub>0</sub> + β<sub>1</sub>(X<sub>1</sub>) + β<sub>2</sub>(X<sub>2</sub>) + β<sub>3</sub>(X<sub>3</sub>) + β<sub>12</sub>(X<sub>1</sub>X<sub>2</sub>) + β<sub>13</sub>(X<sub>1</sub>X<sub>3</sub>) + β<sub>23</sub>(X<sub>2</sub>X<sub>3</sub>) + β<sub>11</sub>(X<sub>1</sub>)<sup>2</sup> + β<sub>22</sub>(X<sub>2</sub>)<sup>2</sup> + β<sub>33</sub>(X<sub>3</sub>)<sup>2</sup>, where Y is the predicted response (pigment concentration); β<sub>0</sub> is a constant; β<sub>1</sub>, β<sub>2</sub>, β<sub>3</sub> and β<sub>4</sub> are linear coefficients; β<sub>12</sub>, β<sub>13</sub> and β<sub>23</sub> are cross product coefficients; and β<sub>11</sub>, β<sub>22,</sub> β<sub>33</sub> and β<sub>44</sub> are quadratic coefficients. Variable maximal predicted response and coefficient calculations were carried out using Microsoft Excel 2007. The low, middle and high levels of each variable were designated −1, 0 and +1, respectively.</p> </div> <div class="identificationoftheextractedpigment" title="sec"> <div class="title" tagx="title" title="title">
Identification of the extracted pigment</div> <div class="fouriertransforminfrared(ftir)spectroscopyanalysis" title="sec"> <div class="title" tagx="title" title="title">
Fourier transform infrared (FTIR) spectroscopy analysis</div> <p id="Par29">The molecular structure of the lyophilized pigment was partially identified using a Peak Find-Memory-27 spectrophotometer. A mixture of approximately 1 mg of the tested material and 300 mg of pure dry potassium bromide (KBr) was pressed into discs. The measurements obtained infrared spectra between 400 and 4000 cm<sup>−1</sup></p> </div> <div class="ramanspectroscopy" title="sec"> <div class="title" tagx="title" title="title">
Raman spectroscopy</div> <p id="Par30">A Raman Senterra instrument with a multiwavelength capability operating at 785 nm with a power of 50 mW and the wide range of 400–4000 cm<sup>−1</sup> was used to measure the effect of the excitation wavelength on the pigment spectrum. A laser irradiated an object in an optical microscope (laser spot = 2 µm), and the scattered light from the sample was collected by the optics of the microscope passing through holographic filters, a pinhole, and a monochromator to be detected by a charge-coupled device (CCD).</p> </div> <div class="gaschromatography-massspectrometry(gc-ms)analysis" title="sec"> <div class="title" tagx="title" title="title">
Gas chromatography-mass spectrometry (GC-MS) analysis</div> <p id="Par31">This analysis was performed according to a previously reported method<sup><a href="#CR33">33</a></sup> using an Agilent Technologies GC equipped with a mass selective detector, HP-5MS. A 5% phenyl methyl siloxane capillary column with dimensions of 30.0 m × 250 µm × 0.25 µm was used, and helium was used as the carrier gas at 1 ml/min. The column temperature was programmed to initially be 90 °C for 1 min, followed by an increase at 8 °C/min to 205 °C, then 5 °C/min to 240 °C, and then 8 °C/min to 300 °C. The MS instrument was operated at 70 eV. The constituents were identified by a comparison of their mass spectral data with those of standard compounds from the National Institute of Standards and Technology (NIST) Spectral Library.</p> </div> <div class="liquidchromatography–massspectrometrylc-msanalysis" title="sec"> <div class="title" tagx="title" title="title">
Liquid chromatography–mass spectrometry LC-MS analysis</div> <p id="Par32">Separation and mass identification of the natural extract in methanol was done using LC-MS (Shimadzu, Japan). The separation was carried out using isocratic elution system with 100% MeOH (0.1% HCOOH) using C18 column maintained at 25 °C, while PDA detector (200 - 600 nm) was operated for the detection analysis. The separation unit is coupled to MS for mass identification of the separated compounds which was operated on the positive ionization mode in the range 100–2000 Da.</p> </div> <div class="nuclearmagneticresonance(nmr)spectroscopy-measurement" title="sec"> <div class="title" tagx="title" title="title">
Nuclear Magnetic Resonance (NMR) spectroscopy- measurement</div> <p id="Par33"><sup>1</sup>H NMR spectrum was recorded using a Brucker Advance NMR spectrophotometer at 300 MHz (300 K) in DMSO-<i>d</i>6 as solvent. The chemical shifts were reported in ppm (δ) relative to tetramethylsilane (TMS) served as an internal standard (δ = 0 ppm).</p> </div> <div class="assessmentofthehemolyticbehaviouroftheextractedpigment" title="sec"> <div class="title" tagx="title" title="title">
Assessment of the hemolytic behaviour of the extracted pigment</div> <p id="Par34">The haemolytic activity was carried out according to previously reported method<sup><a href="#CR34">34</a></sup>. After collecting human blood in the heparin tube, it was centrifuged then the packed RBCs were harvested, washed and diluted with phosphate buffer saline (PBS). The diluted RBC suspension was mixed with the serial concentrations of the tested pigment. PBS and water were used instead of pigment as negative and positive controls, respectively. After 2 h incubation at 37 °C and centrifugation, the absorbance of the supernatant was measured at 540 nm. The percent of hemolysis was calculated then both doses (EC<sub>100</sub> and IC<sub>50</sub>) that cause 0% and 50% hemolysis, respectively, were estimated using the Graphpad Instat software.</p> </div> </div> <div class="investigationoftheanticanceractivityofthepigmentagainstfourhumancancercelllines" title="sec"> <div class="title" tagx="title" title="title">
Investigation of the anticancer activity of the pigment against four human cancer cell lines</div> <div class="determinationofthecytotoxicityeffectofthepigmentonhumannormalandcancercelllines" title="sec"> <div class="title" tagx="title" title="title">
Determination of the cytotoxicity effect of the pigment on human normal and cancer cell lines</div> <p id="Par35">Normal human lung fibroblast cells (Wi-38) and a colon cancer line (Caco-2) were maintained as adherent cell cultures in Dulbecco’s modified Eagle medium (DMEM) containing 10% foetal bovine serum (FBS). A human liver cancer line (HepG-2), breast cancer line (MCF-7) and cervical cancer line (HeLa) were cultured in Roswell Park Memorial Institute (RPMI) 1640 medium supplemented with 10% FBS (Lonza, USA). Human normal and cancer cells were seeded in 96-well cell culture plates. After cell attachment, serial concentrations of the extracted pigment or FU were added. After 48 h of treatment, all cells were incubated with MTT (10 mg/ml) for 3 h. Then, MTT solution was removed, DMSO was added, and the absorbance was read by a microplate reader at 590 nm<sup><a href="#CR35">35</a></sup>. The effective concentration (EC<sub>50</sub>) and safe dose (EC<sub>100</sub>) that cause 50% and 100% cell viability, respectively, and the half maximal inhibitory concentration (IC<sub>50</sub>) for the growth of the tested cancer cell lines were calculated by GraphPad InStat software. Moreover, the SI, defined as the ratio of the EC<sub>50</sub> for normal cells to that for cancer cell lines, was estimated. Furthermore, cellular morphological changes before and after treatment were examined using a phase-contrast inverted microscope (Olympus, Japan) with a digital camera and CellSense software.</p> </div> <div class="invitroanalysisofapoptoticactivityusingflowcytometryandfluorescencephasecontrastmicroscopy" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> analysis of apoptotic activity using flow cytometry and fluorescence phase contrast microscopy</div> <p id="Par36">The human cancer cell lines Caco-2, MCF-7, HepG2 and HeLa were treated with pigment extract or 5-FU at their IC<sub>50</sub> dose. After 48 h of treatment, cells were harvested by trypsinization, washed with cold PBS and incubated with annexin V-biotin (Sigma, USA) and PI (Sigma, USA) for 15 min in the dark. After washing, the cells were incubated with streptavidin-fluorescein for 15 min, collected by centrifugation and resuspended in PBS. The cell death rates of the annexin- and PI-stained cell populations were detected using flow cytometry with a fluorescein isothiocyanate (FITC) signal detector (FL1) and a phycoerythrin emission signal detector (FL2), respectively. Additionally, the cell death-dependent anticancer activity of the pigment was assayed by two fluorescent nuclear stains (ethidium bromide and acridine orange dyes). The untreated and treated cancer cells were stained with these dual nuclear dyes and examined after 15 min using a fluorescence phase-contrast microscope.</p> </div> <div class="determinationofcaspase3/7activationinpigment-treatedcancercells" title="sec"> <div class="title" tagx="title" title="title">
Determination of caspase 3/7 activation in pigment-treated cancer cells</div> <p id="Par37">Caspase 3/7 activation was measured in untreated and pigment- or 5-FU-treated Caco-2 cells according to the instructions of the apo-ONE® homogeneous caspase-3/7 fluorescence kit (Promega, USA).</p> </div> <div class="assessmentoftheinhibitoryimpactofthestudiedpigmentonmmp-9" title="sec"> <div class="title" tagx="title" title="title">
Assessment of the inhibitory impact of the studied pigment on MMP-9</div> <p id="Par38">The activity of MMP-9 was determined colourimetrically in the presence of serial concentrations of the tested pigment or 5-FU following the manufacturer’s instructions (Abcam, UK). The percentage of inhibition of MMP-9 activity was calculated relative to control MMP-9 activity (in the absence of pigment and 5-FU), and IC<sub>50</sub> values (50% MMP-9 inhibition) were calculated by PRISM 6 (GraphPad Software Inc., CA, USA).</p> </div> </div> <div class="evaluationoftheantiviralpotentialofthepigment" title="sec"> <div class="title" tagx="title" title="title">
Evaluation of the antiviral potential of the pigment</div> <div class="determinationofthecytotoxicityofthepigmentonhumanvirushostpbmcs" title="sec"> <div class="title" tagx="title" title="title">
Determination of the cytotoxicity of the pigment on human virus host PBMCs</div> <p id="Par39">Informed consent was obtained from volunteers before using their blood. Human blood was used according to the Research Ethical Committee (REC) of the Faculty of Medicine (Alexandria University) and the National Health and Medical Research Council policies and the Ministry of Health and Population, Egypt, with approval (0304389). This research has received permission from the Department of Medical Biotechnology (SRTA-City) and the Department of Biochemistry (Faculty of Science, Alexandria University). The HCV and HBV samples were positive for genotypes 4a and D, respectively.</p> <p id="Par40">Human PBMCs were isolated from blood according to the Ficoll-Hypaque density gradient centrifugation method and then counted and resuspended gently in RPMI medium containing 10% FBS to be 1 × 10<sup>6</sup> cells/ml. Approximately 1 × 10<sup>5</sup> mononuclear cells were seeded in 96-well cell culture plates. Then, the cells were incubated with and without serial dilutions of the examined extract or currently used anti-HCV and anti-HBV drugs (sofosbuvir ("Sovaldi") and lamivudine, respectively). After 96 h of incubation in a 5% CO<sub>2</sub> incubator, cell viability was determined using MTT, as described above, to estimate safe doses of the extracted pigment, Sovaldi and lamivudine.</p> </div> <div class="invitroquantificationofanti-hcvandanti-hbvactivityofthepigment" title="sec"> <div class="title" tagx="title" title="title"><i>In vitro</i> quantification of anti-HCV and anti-HBV activity of the pigment</div> <p id="Par41">The viral host cells and human PBMCs (1 × 10<sup>6</sup> cells) were seeded in each well of a 12-well culture plate. All wells except the negative control were incubated overnight with serum infected with either HCV (2.9 × 10<sup>4</sup> copies/ml, genotype 4a) or HBV (1 × 10<sup>4</sup> copies/ml, genotype D) in RPMI-1640 medium in a CO<sub>2</sub> incubator at 37 °C, 5% CO<sub>2</sub> and 95% humidity. Then, the infected medium was replaced with fresh RPMI-1640 medium containing 10% FBS for positive control wells. For treated wells, this infected medium was exchanged with RPMI-1640 medium containing 10% FBS and the tested extract or reference antiviral drugs at their EC<sub>100</sub>. After 96 h, the untreated and treated infected cells were quantitatively analysed for intracellular HCV and HBV using the fully automated Cobas Ampliprep Cobas TaqMan (CAP-CTM) analyser (Roche Diagnostics, USA).</p> </div> <div class="investigationoftheinhibitoryeffectofthepigmentonhcv-ns5bpolymeraseandhbvdna-dependentdnapolymerase" title="sec"> <div class="title" tagx="title" title="title">
Investigation of the inhibitory effect of the pigment on HCV-NS5B polymerase and HBV DNA-dependent DNA polymerase</div> <p id="Par42">The inhibitory effect of the extracted pigment on NS5B polymerase activity was examined following the protocol of<sup><a href="#CR36">36</a></sup>. In brief, the assay is based on incubating an HCV-NS5B (genotype 1b-Con1)-Δ21-His 6 enzyme with heteropolymerically modified (at its 3′-end with dideoxycytidine) RNA template and radiolabelled nucleotide. Incorporation of the radiolabelled nucleotide was quantified in the presence and absence of serial concentrations of the studied pigment or standard anti-HCV drug using the TopCount NXT microplate scintillation counter (Perkin Elmer, Wellesley, MA).</p> <p id="Par43">The inhibitory capacity of pigment on HBV DNA polymerase activity was measured according to the modified method of<sup><a href="#CR37">37</a></sup>. Briefly, the enzymatic reaction contained a mixture of virus suspension, serial concentrations of pigment or lamivudine, KCl, MgCl<sub>2</sub>, Nonidet P-40, Tris-HCl (pH 7.5), β-mercaptoethanol and deoxynucleotides, including 1 µCi of radiolabelled [<sup>3</sup>H-TTP]. Then, the newly synthesized DNA was transferred to filter paper disks and precipitated by 5% trichloroacetic acid to detect the radioactivity using a scintillation counter.</p> <p id="Par44">The inhibition rate for both viral polymerase activities was used to assess the IC<sub>50</sub> values (the concentration of pigment or standard drug that inhibited polymerases by 50%) using GraphPad software.</p> </div> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par45">Data are expressed as the mean ± standard error (SE) of the mean, and statistical significance was estimated by unpaired <i>t</i>-test analysis using the SPSS16 program. The differences were considered statistically significant at <i>p</i>-values &amp;lt; 0.05*, &amp;lt;0.005**, &amp;lt;0.0005***.</p> </div> <div class="datafromthebox-behnkendesign(bbd)experiments" title="sec"> <div class="title" tagx="title" title="title">
Data from the Box-Behnken design (BBD) experiments</div> <p id="Par46">The BBD data were subjected to multiple linear regressions to estimate the <i>t</i>-value, <i>p</i>-value and confidence level. The significance level (<i>p</i>-value) was determined using the <i>t</i>-test. If this probability is sufficiently small, the idea that an effect was caused by varying the level of the variable under examination is accepted. Optimal values of the dedicated response were estimated using the <i>solver</i> function of Microsoft Excel tools. The simultaneous effects of the three most significant independent factors were generated by <i>Statistica</i> 5.0 software. The optimal conditions obtained were verified experimentally and then compared to the data calculated from the model.</p> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors are extremely grateful to the City of Scientific Research and Technological Applications, Alexandria, Egypt, for providing all facilities to complete this work. The authors gratefully thank Dr. Mohamed I.A. Ibrahim for kind help through performing LC/MS spectrometry &amp;amp;NMR spectroscopy, Prof. Dr. Waleed El-Zawawy (NRC) for guidance in understanding the chemistry part and Prof. Dr. Hesham A. El-Enshasy (UTM) for critical thesis revision and valuable comments.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>G.E.H. performed the experimental part of the work and wrote the main manuscript text. M.M.A. performed an <i>in vitro</i> assessment of the biological activities of pigments and interpreted, analysed, wrote and revised the main manuscript text related to these experiments. G.M.A. prepared figures of the work. H.G. performed the analysis. S.A.S. helped in isolate identification, follows up the experimental work &amp;amp; revised the manuscript. N.A.S. revised the manuscript and helped in isolate identification. Y.R.A. interpreted the data, substantively revised the manuscript and suggested the main point of this work. All authors read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data produced during this study are included in this published article.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par47">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mensria</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilera</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hacene</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benammar</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diversity and Bioprospecting of Extremely Halophilic Archaea isolated from Algerian Arid and Semi-Arid Wetland Ecosystems for Halophilic-Active Hydrolytic Enzymes</span><span class="source" tagx="source" title="source">J. Microbiol. Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">207</span><span class="fpage" tagx="fpage" title="fpage">289</span><span class="lpage" tagx="lpage" title="lpage">298</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micres.2017.12.011">10.1016/j.micres.2017.12.011</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margesin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schinner</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Potential of halotolerant and halophilic microorganisms for biotechnology</span><span class="source" tagx="source" title="source">Extremophiles</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">83</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s007920100184">10.1007/s007920100184</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11354458">11354458</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naushad</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenomic analyses and molecular signature for the class Halobacteria and its two major clades: a proposal for divison of the class Halobacteria into an emended order Halobacteriales and two new orders, Haloferacales ord. nov. and Natrialbales ord. nov., containing the novel families Haloferacales fam. Nov. and Natrialbales fam. Nov</span><span class="source" tagx="source" title="source">J. Int. J Syst. Evol. Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">1050</span><span class="lpage" tagx="lpage" title="lpage">1069</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/ijs.0.070136-0">10.1099/ijs.0.070136-0</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Britton, G., Liaaen-Jensen, S., Pfander, H., Mercadante, A. Z. &amp;amp; Egeland, E. S. Carotenoids-handbook. Birkhäuser Verlag, Basel (2004).</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calegari-Santos</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diogo</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fontana</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonfim</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Carotenoid production by halophilic archaea under different culture conditions</span><span class="source" tagx="source" title="source">J. Curr. Microbiol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">641</span><span class="lpage" tagx="lpage" title="lpage">651</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00284-015-0974-8">10.1007/s00284-015-0974-8</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Solid-phase extraction of carotenoids</span><span class="source" tagx="source" title="source">J. Chromatogr. A</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">1216</span><span class="fpage" tagx="fpage" title="fpage">5763</span><span class="lpage" tagx="lpage" title="lpage">5768</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chroma.2009.06.009">10.1016/j.chroma.2009.06.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19539943">19539943</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahmohammadi</span><span class="given-names" tagx="given-names" title="given-names">HR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Protective roles of bacterioruberin and intracellular KCl in the resistance of <i>Halobacterium salinarium</i> against DNA-damaging agents</span><span class="source" tagx="source" title="source">J. Radiat. Res.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">251</span><span class="lpage" tagx="lpage" title="lpage">262</span><span class="pub-id"><a href="https://dx.doi.org/10.1269/jrr.39.251">10.1269/jrr.39.251</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10196780">10196780</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu.</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mass production of C50 carotenoids by <i>Haloferax mediterranei</i> using extruded rice bran and starch under optimal conductivity of brined medium</span><span class="source" tagx="source" title="source">J. Bioprocess. Biosyst. Eng.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">2361</span><span class="lpage" tagx="lpage" title="lpage">2367</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00449-015-1471-y">10.1007/s00449-015-1471-y</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ku</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">NW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influence of nutritive factors on C50 carotenoids production by <i>Haloferax mediterranei</i> ATCC 33500 with two-stage cultivation</span><span class="source" tagx="source" title="source">J. Bioresour. Technol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">6487</span><span class="lpage" tagx="lpage" title="lpage">6493</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.biortech.2010.03.044">10.1016/j.biortech.2010.03.044</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vílchez</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Marine carotenoids: biological functions and commercial applications</span><span class="source" tagx="source" title="source">J. Mar. Drugs.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">319</span><span class="lpage" tagx="lpage" title="lpage">333</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/md9030319">10.3390/md9030319</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacterial carotenoids. XXVI. C50-carotenoids. 2. Bacterioruberin</span><span class="source" tagx="source" title="source">J. Acta Chem. Scand.</span><span class="year" tagx="year" title="year">1967</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">2578</span><span class="lpage" tagx="lpage" title="lpage">2580</span><span class="pub-id"><a href="https://dx.doi.org/10.3891/acta.chem.scand.21-2578">10.3891/acta.chem.scand.21-2578</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">Matrix Metalloproteinase-9 (MMP-9) as a Cancer Biomarker and MMP-9 Biosensors</span><span class="source" tagx="source" title="source">Recent. Adv. Sens.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">3249</span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torregrosa-Grespo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galiana</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinosa</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biocompounds from Haloarchaea and their uses in biotechnology</span><span class="source" tagx="source" title="source">J. Mar. Drugs.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">21</span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Valera</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ventosa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juez</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imhoff</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Variation of environmental features and microbial populations with salt concentrations in a multi-pond saltern</span><span class="source" tagx="source" title="source">Microbiol. Ecol.</span><span class="year" tagx="year" title="year">1985</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">115</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02010483">10.1007/BF02010483</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamidi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdin.</span><span class="given-names" tagx="given-names" title="given-names">ZM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazemyieh.</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hejazi.</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hejazi</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Optimization of total carotenoid production by Halorubrum sp. TBZ126 using response surface methodology</span><span class="source" tagx="source" title="source">J. Microb. Biochem. Technol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="pub-id"><a href="https://dx.doi.org/10.4172/1948-5948.1000158">10.4172/1948-5948.1000158</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Sikkandar, S., <i>et al</i> Halophilic bacteria-A potent source of carotenoids with antioxidant and anticancer potentials. <i>J. Pure Appl. Microbiol</i>. 7, 2825–2830 16 then add 2018 (2013).</span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torregroso-crespo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Exploring the valuable carotenoids for the large scale production by marine microorganis ms</span><span class="source" tagx="source" title="source">Mar. Drugs.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">3</span><span class="lpage" tagx="lpage" title="lpage">25</span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span></span><span class="mixed-article-title" title="mixed-article-title"><i>In Vitro</i> Antioxidant, Antihemolytic, and Anticancer Activity of the Carotenoids from Halophilic Archaea</span><span class="source" tagx="source" title="source">J. Curr. Microbiol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">266</span><span class="lpage" tagx="lpage" title="lpage">271</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00284-017-1374-z">10.1007/s00284-017-1374-z</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elmore</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Apoptosis: a review of programmed cell death</span><span class="source" tagx="source" title="source">J. Toxicol. Pathol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">495</span><span class="lpage" tagx="lpage" title="lpage">516</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/01926230701320337">10.1080/01926230701320337</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Safarpour</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebrahimi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shahzadeh Fazeli</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amoozegar</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Supernatant Metabolites from Halophilic Archaea to Reduce Tumorigenesis in prostate cancer <i>In-vitro</i> and <i>In-vivo</i></span><span class="source" tagx="source" title="source">Iran. J. Pharm. Res.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">241</span><span class="lpage" tagx="lpage" title="lpage">253</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31089359">31089359</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biological potential and mechanism of prodigiosin from <i>Serratia merscens</i> subsp. Lawsoniana in human choriocarcinoma and prostate cancer cell lines</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">11</span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palozza</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A. Modulation of MMP-9 pathway by lycopene in macrophages and fibroblasts exposed to cigarette smoke</span><span class="source" tagx="source" title="source">J. Inflamm. Allergy Drug. Targets.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">36</span><span class="lpage" tagx="lpage" title="lpage">47</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/187152812798889376">10.2174/187152812798889376</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Habashy, N. H. &amp;amp; Abu-Serie, M. M. Major royal-jelly protein 2 and its isoform X1 are two novel safe inhibitors for hepatitis C and B viral entry and replication. <i>Int. J. Biol. Macromol</i>. <b>141</b>, 1072-1087 (2019).</span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Antiviral and specific modes of action of bacterial prodigiosin against Bombyx mori nucleopolyhedrovirus <i>In vitro</i></span><span class="source" tagx="source" title="source">J. Appl. Microbiol. Biotechnol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">3979</span><span class="lpage" tagx="lpage" title="lpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00253-015-7242-5">10.1007/s00253-015-7242-5</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of hepatitis C virus replication by <i>Monascus</i> pigment derivatives that interfere with viral RNA polymerase activity and the mevalonate biosynthesis pathway</span><span class="source" tagx="source" title="source">J. Antimicrob. Chemother.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="lpage" tagx="lpage" title="lpage">58</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jac/dkr432">10.1093/jac/dkr432</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22076990">22076990</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goh.</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeon</span><span class="given-names" tagx="given-names" title="given-names">YJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrow</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielan</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burns</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Osmoadaptive strategies of the archaeon Halococcus hamelinensis isolated from a hypersaline stromatolite environment</span><span class="source" tagx="source" title="source">J. Astrobiology.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">529</span><span class="lpage" tagx="lpage" title="lpage">536</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/ast.2010.0591">10.1089/ast.2010.0591</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quigley</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">A rapid boiling method for the preparation of bacterial plasmids</span><span class="source" tagx="source" title="source">J. Anal. Biochem.</span><span class="year" tagx="year" title="year">1981</span><span class="volume" tagx="volume" title="volume">114</span><span class="fpage" tagx="fpage" title="fpage">193</span><span class="lpage" tagx="lpage" title="lpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0003-2697(81)90473-5">10.1016/0003-2697(81)90473-5</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnscheidt</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KrÜger</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">StrÖmpl</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mau</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Simplified protocols for the preparation of genomic DNA from bacterial cultures</span><span class="source" tagx="source" title="source">J. Mol. Microb. Ecol. Manual.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">1905</span><span class="lpage" tagx="lpage" title="lpage">1919</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widdel</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Theory and measurement of bacterial growth</span><span class="source" tagx="source" title="source">Grundpraktikum Mikrobiologie.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">222</span><span class="lpage" tagx="lpage" title="lpage">234</span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Britton, G. UV/Visible spectroscopy. In: Britton, G., LiaaenJensen, S. &amp;amp; Pfander, H. (eds) Carotenoids: spectroscopy. Birkhäuser Verlag, Basel, 13–62 (1995).</span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Box</span><span class="given-names" tagx="given-names" title="given-names">GEP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behnken</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="mixed-article-title" title="mixed-article-title">Some new three level designs for the study of quantitative variables</span><span class="source" tagx="source" title="source">J. Technmetrics.</span><span class="year" tagx="year" title="year">1960</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00401706.1960.10489912">10.1080/00401706.1960.10489912</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdel-Fattah</span><span class="given-names" tagx="given-names" title="given-names">YR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soliman</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berekaa</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Application of Box-Behnken design for optimization of poly-γ-glutamic acid production by <i>Baciluus licheniformis</i> SAB-26</span><span class="source" tagx="source" title="source">J. Microbiol. Res.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">70</span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jerković</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of summer savory (<i>Satureja hortensis</i> l.) Honey by physico-chemical parameters and chromatographic/spectroscopic techniques (GCFID/MS, HPLC-DAD, UV/VIS and FTIR-ATR)</span><span class="source" tagx="source" title="source">J. Croatica Chemica Acta.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="https://dx.doi.org/10.5562/cca2514">10.5562/cca2514</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Interaction of mesoporous silica nanoparticles with human red blood cell membranes: size and surface effects</span><span class="source" tagx="source" title="source">ACS Nano.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">1366</span><span class="lpage" tagx="lpage" title="lpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/nn103077k">10.1021/nn103077k</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21294526">21294526</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosmann</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid colorimetric assay for cellular growth and survival: Application to proliferation and cytotoxicity assays</span><span class="source" tagx="source" title="source">J. Immunol. Methods.</span><span class="year" tagx="year" title="year">1983</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">55</span><span class="lpage" tagx="lpage" title="lpage">63</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0022-1759(83)90303-4">10.1016/0022-1759(83)90303-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6606682">6606682</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrari</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tumor necrosis factor soluble receptors in patients with various degrees of congestive heart failure</span><span class="source" tagx="source" title="source">Circulation</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">1479</span><span class="lpage" tagx="lpage" title="lpage">1486</span><span class="pub-id"><a href="https://dx.doi.org/10.1161/01.CIR.92.6.1479">10.1161/01.CIR.92.6.1479</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7664430">7664430</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirschman</span><span class="given-names" tagx="given-names" title="given-names">SZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerber</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garfinkel</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">Differential activation of hepatitis B DNA polymerase by detergent and salt</span><span class="source" tagx="source" title="source">J. Med. Virol.</span><span class="year" tagx="year" title="year">1978</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">61</span><span class="lpage" tagx="lpage" title="lpage">76</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.1890020109">10.1002/jmv.1890020109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/681913">681913</a></span></span></li> </ul> </div> </div>  </body></html>