<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5933925">5933925</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29683424">29683424</a></span><span class="publisher-id" title="publisher-id">35962</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.35962">10.7554/eLife.35962</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Microbiology and Infectious Disease</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Stochastic processes constrain the within and between host evolution of influenza virus</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">John T</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9846-8917</span><a href="#aff1">1</a><a href="#fund1" /><a href="#con1" /><a href="#conf1" /><a href="#dataset1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">Robert J</span></span><a href="#aff2">2</a><a href="#fund2" /><a href="#con2" /><a href="#conf1" /><a href="#dataset1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">Emily T</span></span><a href="#aff3">3</a><a href="#con3" /><a href="#conf1" /><a href="#dataset1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">Ryan E</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3546-5935</span><a href="#aff3">3</a><a href="#con4" /><a href="#conf1" /><a href="#dataset1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">Arnold S</span></span><a href="#aff3">3</a><a href="#fund3" /><a href="#fund4" /><a href="#con5" /><a href="#conf1" /><a href="#dataset1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">Adam S</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2906-8335</span><span class="email" tagx="email" title="email">alauring@med.umich.edu</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#fund5" /><a href="#fund6" /><a href="#fund7" /><a href="#con6" /><a href="#conf1" /><a href="#dataset1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Department of Microbiology and Immunology</span><span class="institution" title="institution">University of Michigan</span><span class="addr-line" title="addr-line">Ann Arbor</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Division of Infectious Diseases, Department of Internal Medicine</span><span class="institution" title="institution">University of Michigan</span><span class="addr-line" title="addr-line">Ann Arbor</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Department of Epidemiology</span><span class="institution" title="institution">University of Michigan</span><span class="addr-line" title="addr-line">Ann Arbor</span><span class="country" tagx="country" title="country">United States</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">Richard A</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="aff4">[], <span class="institution" title="institution">University of Basel</span><span class="country" tagx="country" title="country">Switzerland</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2018-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e35962</span><span class="history" title="history"><span class="received" title="received">received: 2018-2-15</span><span class="accepted" title="accepted">accepted: 2018-4-18</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">McCrone et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license">[license_ref] http://creativecommons.org/licenses/by/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-35962.pdf" title="self-uri">elife-35962.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The evolutionary dynamics of influenza virus ultimately derive from processes that take place within and between infected individuals. Here we define influenza virus dynamics in human hosts through sequencing of 249 specimens from 200 individuals collected over 6290 person-seasons of observation. Because these viruses were collected from individuals in a prospective community-based cohort, they are broadly representative of natural infections with seasonal viruses. Consistent with a neutral model of evolution, sequence data from 49 serially sampled individuals illustrated the dynamic turnover of synonymous and nonsynonymous single nucleotide variants and provided little evidence for positive selection of antigenic variants. We also identified 43 genetically-validated transmission pairs in this cohort. Maximum likelihood optimization of multiple transmission models estimated an effective transmission bottleneck of 1–2 genomes. Our data suggest that positive selection is inefficient at the level of the individual host and that stochastic processes dominate the host-level evolution of influenza viruses.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
influenza virus</div> <div class="kwd" title="kwd">
bottleneck</div> <div class="kwd" title="kwd">
transmission</div> <div class="kwd" title="kwd">
diversity</div> <div class="kwd" title="kwd">
evolution</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Virus</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000057</span><span class="institution" title="institution">National Institute of General Medical Sciences</span></div> </div><span class="award-id" tagx="award-id" title="award-id">T32 GM007544</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">John T</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">K08 AI119182</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">Robert J</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000030</span><span class="institution" title="institution">Centers for Disease Control and Prevention</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01 IP00474</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">Arnold S</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 AI097150</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">Arnold S</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000862</span><span class="institution" title="institution">Doris Duke Charitable Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">CSDA 2013105</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">Adam S</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 AI118886</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">Adam S</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100007270</span><span class="institution" title="institution">University of Michigan</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Discovery grant</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">Adam S</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="An analysis of influenza viruses from naturally infected people suggests a tight transmission bottleneck and little positive selection within hosts." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The rapid evolution of influenza viruses has led to reduced vaccine efficacy and the continuing emergence of novel strains. Broadly speaking, evolution is the product of deterministic processes, such as selection, and stochastic processes, such as genetic drift and migration (<a href="#bib21">Kouyos et al., 2006</a>). The global evolution of influenza A virus (IAV) is dominated by the positive selection of novel antigenic variants that subsequently seed annual epidemics in the Northern and Southern hemisphere (<a href="#bib41">Rambaut et al., 2008</a>). These global patterns are contrasted by results from whole genome sequencing of viruses on more local scales, which suggest the importance of stochastic processes such as strain migration and within-clade reassortment (<a href="#bib31">Nelson et al., 2006</a>). While it is clear that influenza’s evolutionary dynamics differ across scales (<a href="#bib30">Nelson and Holmes, 2007</a>; <a href="#bib15">Holmes, 2009</a>), the relative roles of deterministic and stochastic forces in viral evolution within and between acutely infected individuals remain unclear.</p> <p>It is now feasible to efficiently sequence patient-derived isolates at sufficient depth of coverage to define the genetic diversity and dynamics of virus evolution within individual hosts (<a href="#bib19">Kao et al., 2014</a>). Studies of IAV populations in animal and human systems suggest that most intrahost single nucleotide variants (iSNV) are rare and that intrahost populations are subject to strong purifying selection (<a href="#bib42">Rogers et al., 2015</a>; <a href="#bib28">Murcia et al., 2010</a>; <a href="#bib18">Iqbal et al., 2009</a>; <a href="#bib39">Poon et al., 2016</a>; <a href="#bib6">Dinis et al., 2016</a>; <a href="#bib5">Debbink et al., 2017</a>). Positive selection is common in cell culture (<a href="#bib7">Doud et al., 2017</a>; <a href="#bib2">Archetti and Horsfall, 1950</a>; <a href="#bib9">Foll et al., 2014</a>), and has been observed in experimental infections in swine (<a href="#bib17">Illingworth et al., 2014</a>). However, it has only been well documented within human hosts in the extreme cases of drug resistance (<a href="#bib13">Gubareva et al., 2001</a>; <a href="#bib12">Ghedin et al., 2011</a>; <a href="#bib42">Rogers et al., 2015</a>), and long-term infection of immunocompromised individuals (<a href="#bib51">Xue et al., 2017</a>). Indeed, we and others have been unable to identify evidence for positive selection in naturally-occurring, acute human infections (<a href="#bib5">Debbink et al., 2017</a>; <a href="#bib6">Dinis et al., 2016</a>), and its relevance to within-host processes is unclear.</p> <p>Despite limited evidence for positive selection, novel mutations do arise within hosts and some will clearly be positively selected. Their potential for subsequent spread through host populations is heavily dependent on the size of the transmission bottleneck (<a href="#bib1">Alizon et al., 2011</a>; <a href="#bib53">Zwart and Elena, 2015</a>). If the transmission bottleneck is sufficiently wide, low frequency variants can plausibly be transmitted and spread efficiently through host populations (<a href="#bib10">Geoghegan et al., 2016</a>). While experimental infections of ferrets suggest a very narrow transmission bottleneck (<a href="#bib47">Varble et al., 2014</a>; <a href="#bib49">Wilker et al., 2013</a>), studies of equine influenza support a bottleneck wide enough to allow transmission of rare iSNV (<a href="#bib16">Hughes et al., 2012</a>; <a href="#bib28">Murcia et al., 2010</a>). The only available genetic study of influenza virus transmission in humans estimated a remarkably large transmission bottleneck, allowing for transmission of 100–200 genomes (<a href="#bib39">Poon et al., 2016</a>; <a href="#bib46">Sobel Leonard et al., 2017</a>).</p> <p>Here, we use next generation sequencing of within-host influenza virus populations to define the evolutionary dynamics of IAV within and between human hosts. We apply a benchmarked analysis pipeline to identify iSNV and to characterize the genetic diversity of H3N2 and H1N1 populations collected over five post-pandemic seasons from individuals enrolled in a prospective household study of influenza. We find that intrahost populations are dynamic and constrained by genetic drift and purifying selection. In our study, positive selection rarely amplifies a beneficial de novo variant to a frequency greater than 2%. Contrary to what has been previously reported for human influenza transmission (<a href="#bib39">Poon et al., 2016</a>), but consistent with what has been observed in many other viruses with distinct modes of transmission (<a href="#bib53">Zwart and Elena, 2015</a>; <a href="#bib25">McCrone and Lauring, 2018</a>), we identify a very tight effective transmission bottleneck that limits the transmission of low-frequency variants.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p>We used next generation sequencing to characterize influenza virus populations collected from individuals enrolled in the Household Influenza Vaccine Effectiveness (HIVE) study (<a href="#bib27">Monto et al., 2014</a>; <a href="#bib32">Ohmit et al., 2013</a>; <a href="#bib33">Ohmit et al., 2015</a>; <a href="#bib34">Ohmit et al., 2016</a>; <a href="#bib37">Petrie et al., 2013</a>), a community-based cohort that enrolls 213–340 households of 3 or more individuals in Southeastern Michigan each year (<a href="#table1">Table 1</a>). These households are followed prospectively from October to April, with symptom-triggered collection of nasal and throat swab specimens for identification of respiratory viruses by RT-PCR (see Materials and methods). In contrast to case-ascertained studies, which identify households based on an index case who seeks medical care, the HIVE study identifies symptomatic individuals regardless of illness severity. In the first four seasons of the study (2010–2011 through 2013–2014), respiratory specimens were collected 0–7 days after illness onset. Beginning in the 2014–2015 season, each individual provided two samples, a self-collected specimen at the time of symptom onset and a clinic-collected specimen obtained 0–7 days later. Each year, 59–69% of individuals had self-reported or confirmed receipt of that season’s vaccine prior to local circulation of influenza virus.</p> <div class="table-wrap_UNKNOWN" id="table1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.35962.002">10.7554/eLife.35962.002</a><span class="label" tagx="label" title="label">Table 1.</span> <div class="title" tagx="title" title="title">
Influenza viruses over five seasons in a household cohort</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1" /> <th valign="top" rowspan="1" colspan="1">2010–2011</th> <th valign="top" rowspan="1" colspan="1">2011–2012</th> <th valign="top" rowspan="1" colspan="1">2012–2013</th> <th valign="top" rowspan="1" colspan="1">2013–2014</th> <th valign="top" rowspan="1" colspan="1">2014–2015</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">Households</td> <td valign="top" rowspan="1" colspan="1">328</td> <td valign="top" rowspan="1" colspan="1">213</td> <td valign="top" rowspan="1" colspan="1">321</td> <td valign="top" rowspan="1" colspan="1">232</td> <td valign="top" rowspan="1" colspan="1">340</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Participants</td> <td valign="top" rowspan="1" colspan="1">1441</td> <td valign="top" rowspan="1" colspan="1">943</td> <td valign="top" rowspan="1" colspan="1">1426</td> <td valign="top" rowspan="1" colspan="1">1049</td> <td valign="top" rowspan="1" colspan="1">1431</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Vaccinated, n (%)*</td> <td valign="top" rowspan="1" colspan="1">934 (65)</td> <td valign="top" rowspan="1" colspan="1">554 (59)</td> <td valign="top" rowspan="1" colspan="1">942 (66)</td> <td valign="top" rowspan="1" colspan="1">722 (69)</td> <td valign="top" rowspan="1" colspan="1">992 (69)</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">IAV Positive Individuals<sup>†</sup></td> <td valign="top" rowspan="1" colspan="1">86</td> <td valign="top" rowspan="1" colspan="1">23</td> <td valign="top" rowspan="1" colspan="1">69</td> <td valign="top" rowspan="1" colspan="1">48</td> <td valign="top" rowspan="1" colspan="1">166</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">H1N1</td> <td valign="top" rowspan="1" colspan="1">26</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">3</td> <td valign="top" rowspan="1" colspan="1">47</td> <td valign="top" rowspan="1" colspan="1">0</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">H3N2</td> <td valign="top" rowspan="1" colspan="1">58</td> <td valign="top" rowspan="1" colspan="1">22</td> <td valign="top" rowspan="1" colspan="1">66</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">166</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">IAV Positive Households<sup>‡</sup></td> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> <td valign="top" rowspan="1" colspan="1" /> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Two individuals</td> <td valign="top" rowspan="1" colspan="1">13</td> <td valign="top" rowspan="1" colspan="1">2</td> <td valign="top" rowspan="1" colspan="1">9</td> <td valign="top" rowspan="1" colspan="1">7</td> <td valign="top" rowspan="1" colspan="1">23</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Three individuals</td> <td valign="top" rowspan="1" colspan="1">5</td> <td valign="top" rowspan="1" colspan="1">2</td> <td valign="top" rowspan="1" colspan="1">3</td> <td valign="top" rowspan="1" colspan="1">3</td> <td valign="top" rowspan="1" colspan="1">11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Four individuals</td> <td valign="top" rowspan="1" colspan="1">-</td> <td valign="top" rowspan="1" colspan="1">-</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">2</td> <td valign="top" rowspan="1" colspan="1">4</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">High Quality NGS Pairs<sup>§</sup></td> <td valign="top" rowspan="1" colspan="1">4</td> <td valign="top" rowspan="1" colspan="1">1</td> <td valign="top" rowspan="1" colspan="1">2</td> <td valign="top" rowspan="1" colspan="1">6</td> <td valign="top" rowspan="1" colspan="1">39</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>*Self reported or confirmed receipt of vaccine prior to the specified season.</p> <p>†RT-PCR confirmed infection.</p> </div> <div class="fn-type-" title=""> <p>‡Households in which two individuals were positive within 7 days of each other. In cases of trios and quartets, the putative chains could have no pair with onset &amp;gt; 7 days apart.</p> <p>§Samples with &amp;gt; 10<sup>3</sup> genome copies per µl of transport medium, adequate amplification of all eight genomic segments, and average sequencing coverage &amp;gt; 10<sup>3</sup> per nucleotide.</p> </div> </div> </div> <p>Over five seasons and nearly 6290 person-seasons of observation, we identified 77 cases of influenza A/H1N1pdm09 infection and 313 cases of influenza A/H3N2 infection (<a href="#table1">Table 1</a>). Approximately half of the cases (n = 166) were identified in the 2014–2015 season, in which there was an antigenic mismatch between the vaccine and circulating strains (<a href="#bib8">Flannery et al., 2016</a>). All other seasons were antigenically matched. Individuals within a household were considered an epidemiologically linked transmission pair if they were both positive for the same subtype of influenza virus within 7 days of each other. Several households had 3 or four symptomatic cases within this one-week window, suggestive of longer chains of transmission (<a href="#table1">Table 1</a>).</p> <div class="within-hostpopulationshavelowgeneticdiversity" title="sec"> <div class="title" tagx="title" title="title">
Within-host populations have low genetic diversity</div> <p>We processed all specimens for viral load quantification and next generation sequencing. Viral load measurements (genome copies per µl) were used for quality control in variant calling, which we have shown is highly sensitive to input titer (<a href="#bib24">McCrone and Lauring, 2016</a>) (<a href="#fig1">Figure 1A</a>). Accordingly, we report data on 249 high quality specimens from 200 individuals, which had a viral load of &amp;gt;10<sup>3</sup> copies per microliter of transport media, adequate RT-PCR amplification of all eight genomic segments, and an average read coverage of &amp;gt;10<sup>3</sup> across the genome (<a href="#table1">Table 1</a>, <a href="#fig1s1">Figure 1—figure supplement 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.35962.003"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Within-host diversity of IAV populations.</div> <p>(<b>A</b>) Boxplots (median, 25th and 75th percentiles, whiskers extend to most extreme point within median ±1.5 x IQR) of the number of viral genomes per microliter transport media stratified by day post symptom onset. Notches represent the approximate 95% confidence interval of the median. (<b>B</b>) Boxplots (median, 25<sup>th</sup> and 75<sup>th</sup> percentiles, whiskers extend to most extreme point within median ±1.5 x IQR) of the number of iSNV in 249 high quality samples stratified by day post symptom onset. (<b>C</b>) The number of iSNV in each isolate stratified by vaccination status. The red lines indicate the median. (<b>D</b>) Location of all identified iSNV in the influenza A genome. Mutations are colored nonsynonymous (blue) and synonymous (gold) relative to that sample’s consensus sequence. Mutations are considered nonsynonymous if they are nonsynonymous in any known influenza open reading frame. Triangles signify mutations that were found in more than one individual in a given season.</p> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.008">10.7554/eLife.35962.008</a><span class="label" tagx="label" title="label">Figure 1—source data 1.</span> <div class="title" tagx="title" title="title">
Titers and day of sampling for all samples processed from the cohort.</div>  <div class="media" title="media"><a href="elife-35962-fig1-data1.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.009">10.7554/eLife.35962.009</a><span class="label" tagx="label" title="label">Figure 1—source data 2.</span> <div class="title" tagx="title" title="title">
The number of iSNV and day of sampling for samples that qualified for iSNV identification.</div>  <div class="media" title="media"><a href="elife-35962-fig1-data2.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.010">10.7554/eLife.35962.010</a><span class="label" tagx="label" title="label">Figure 1—source data 3.</span> <div class="title" tagx="title" title="title">
The number of iSNV and vaccination status for samples that qualified for iSNV identification.</div>  <div class="media" title="media"><a href="elife-35962-fig1-data3.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.011">10.7554/eLife.35962.011</a><span class="label" tagx="label" title="label">Figure 1—source data 4.</span> <div class="title" tagx="title" title="title">
Location and frequency of iSNV identified in each individual.</div> <p>(The highest quality sample was used when two where available).</p>  <div class="media" title="media"><a href="elife-35962-fig1-data4.csv">LINK</a></div> </div> <p />   </div> <p>We identified intrahost single nucleotide variants (iSNV) using our empirically validated analysis pipeline (<a href="#bib24">McCrone and Lauring, 2016</a>). Our approach relies heavily on the variant caller DeepSNV, which uses a clonal plasmid control to distinguish between true iSNV and errors introduced during sample preparation and/or sequencing (<a href="#bib11">Gerstung et al., 2012</a>). Given the diversity of influenza viruses that circulate locally each season, there were a number of instances in which our patient-derived samples had mutations that were essentially fixed (&amp;gt;0.95 frequency) relative to the clonal control. DeepSNV is unable to estimate an error rate for the control or reference base at these positions. We therefore performed an additional benchmarking experiment to identify a threshold for majority iSNV at which we could correctly infer whether or not the corresponding minor allele was also present (see Materials and methods). We found that we could correctly identify a minor allele at a frequency of ≥2% at such sites. We therefore report data on iSNV present at frequencies between 2% and 98%. As expected, this threshold improved the specificity of our iSNV identification and decreased our sensitivity to detect variants below 5% compared to our initial validation experiment (<a href="#bib24">McCrone and Lauring, 2016</a>), which did not employ a frequency threshold (<a href="#supp1">Supplementary file 1</a>).</p> <p>Consistent with our previous studies of natural infections and those of others, we found that the within-host diversity of seasonal influenza A virus (IAV) populations is low (<a href="#bib6">Dinis et al., 2016</a>; <a href="#bib5">Debbink et al., 2017</a>; <a href="#bib45">Sobel Leonard et al., 2016</a>; <a href="#bib24">McCrone and Lauring, 2016</a>). Two hundred forty-three out of the 249 samples had fewer than 10 minority iSNV (median 2, IQR 1–3). There were six samples with greater than 10 minority iSNV. In 3 of these cases, the frequency distribution of minority iSNVs was bimodal, suggesting that the iSNV were linked and that the samples represented mixed infections. Consistent with this hypothesis, putative genomic haplotypes based on these minority iSNV clustered with distinct isolates on phylogenetic trees (<a href="#fig1s2">Figure 1—figure supplements 2</a> and <a href="#fig1s3">3</a>), and these samples were removed from subsequent analysis. While viral shedding was well correlated with days post symptom onset (<a href="#fig1">Figure 1A</a>), the number of minority iSNV identified was not affected by the day of infection, viral load, subtype, or vaccination status (<a href="#fig1">Figure 1B and C</a>, and <a href="#fig1s4">Figure 1—figure supplement 4</a>). Single nucleotide variants were distributed evenly across the genome (<a href="#fig1">Figure 1D</a>).</p> <p>Minority variants were rarely shared among multiple individuals. Ninety-eight percent of minority iSNV were only found once, 2.3% were found in two individuals, and no minority iSNV were found in three or more individuals. The low level of shared diversity suggests that within-host populations explore distinct regions of sequence space with little evidence for parallel evolution. Of the 12 minority iSNV that were found in multiple individuals (triangles in <a href="#fig1">Figure 1D</a>), three were nonsynonymous and none were present in antigenic epitopes. The vast majority of minority variants were rare (frequency 0.02–0.07) (<a href="#fig2">Figure 2A</a>). The ratio of nonsynonymous to synonymous variants was 0.75, and given the excess of nonsynonymous sites across the genome and within the HA gene, these data suggest significant purifying selection within hosts.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.35962.012"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Within-host dynamics of IAV.</div> <p>(<b>A</b>) Histogram of within-host iSNV frequency in 249 high quality samples. Bin width is 0.05 beginning at 0.02. As in <a href="#fig1">Figure 1</a>, mutations were classified as nonsynonymous (blue) if they were nonsynonymous in any known influenza reading frame. Synonymous mutations are gold. (<b>B</b>) The within-host frequency of nonsynonymous mutations in HA stratified by whether or not they are in known antigenic sites (p=0.46 Wilcoxon rank sum). (<b>C</b>) The global frequency of putative antigenic minority iSNV identified in our cohort that have circulated at frequencies above 5% globally since their time of collection. Each variant is labeled according the H3 numbering scheme. The dashed line indicates when samples were collected. Frequency traces are faded prior to the collection date. (<b>D</b>) Timing of sample collection for 43 paired longitudinal samples relative to day of symptom onset. Of the 49 total, 43 pairs had minority iSNV present in either sample. (<b>E</b>) The change in frequency over time for minority iSNV identified for the paired samples in (<b>A</b>). Nonsynonymous and synonymous iSNV are plotted separately. Mutations are colored according to whether they were detected in both isolates (blue), detected only the first isolate (red), or detected only in the second isolate (yellow). The threshold of detection was 2%. The arrows indicate mutations in known antigenic sites.</p> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.014">10.7554/eLife.35962.014</a><span class="label" tagx="label" title="label">Figure 2—source data 1.</span> <div class="title" tagx="title" title="title">
The frequency and class (nonsynonymous/synonymous) of identified iSNV.</div>  <div class="media" title="media"><a href="elife-35962-fig2-data1.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.015">10.7554/eLife.35962.015</a><span class="label" tagx="label" title="label">Figure 2—source data 2.</span> <div class="title" tagx="title" title="title">
Meta data for nonsynonymous iSNV found in HA.</div>  <div class="media" title="media"><a href="elife-35962-fig2-data2.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.016">10.7554/eLife.35962.016</a><span class="label" tagx="label" title="label">Figure 2—source data 3.</span> <div class="title" tagx="title" title="title">
Frequency and meta data for antigenic iSNV that were also identified at the global level.</div>  <div class="media" title="media"><a href="elife-35962-fig2-data3.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.017">10.7554/eLife.35962.017</a><span class="label" tagx="label" title="label">Figure 2—source data 4.</span> <div class="title" tagx="title" title="title">
Sampling day for within-host sample pairs.</div>  <div class="media" title="media"><a href="elife-35962-fig2-data4.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.018">10.7554/eLife.35962.018</a><span class="label" tagx="label" title="label">Figure 2—source data 5.</span> <div class="title" tagx="title" title="title">
Frequencies of mutations identified in longitudinal sample pairs.</div>  <div class="media" title="media"><a href="elife-35962-fig2-data5.csv">LINK</a></div> </div> <p />   </div> </div> <div class="isnvwithinandoutsideantigenicepitopeshavesimilarfrequencies" title="sec"> <div class="title" tagx="title" title="title">
iSNV within and outside antigenic epitopes have similar frequencies</div> <p>Although the full range of the H3 antigenic sites have not been functionally defined, it is estimated that 131 of the 329 amino acids in HA1 lie in or near these sites (<a href="#bib23">Lee and Chen, 2004</a>). Although we identified 16 minority nonsynonymous iSNV in these regions (<a href="#supp2">Supplementary file 2</a>), mutations in antigenic epitopes were not present at significantly higher within-host frequencies than other nonsynonymous mutations in HA (<a href="#fig2">Figure 2B</a>). Five putative antigenic variants were in positions that have been experimentally shown to differ among antigenically drifted viruses (<a href="#bib44">Smith et al., 2004</a>; <a href="#bib48">Wiley et al., 1981</a>), and two (193S and 189N) lie in the ‘antigenic ridge’ that is a major contributor to antigenic drift (<a href="#bib20">Koel et al., 2013</a>). Five of the iSNV in antigenic epitopes have been detected as minority variants above 5% at the global level since the time of isolation (62G, 128A, 193S, 262N, and 307R) with one (62G) seemingly increasing in global frequency (<a href="#fig2">Figure 2C</a>). An additional within host variant (307R) has dominated the population for the past 12 years (<a href="#bib29">Neher and Bedford, 2015</a>). We identified one putative H1N1 antigenic variant (208K in antigenic site C<sub>a</sub>) (<a href="#bib4">Caton et al., 1982</a>; <a href="#bib50">Xu et al., 2010</a>). In total, putative antigenic variants account for 1.0–2.8% of minority iSNV identified and were found in 3.0–7.1% of infections. None of these iSNV were shared among multiple individuals.</p> </div> <div class="withinhostpopulationsaredynamic" title="sec"> <div class="title" tagx="title" title="title">
Within host populations are dynamic</div> <p>We next examined the changes in within-host virus diversity over time in 49 individuals who provided paired longitudinal samples during the 2014/2015 season (<a href="#fig2">Figure 2D</a>). We found that there was very little change in iSNV frequency in populations sampled twice on the same day (R<sup>2</sup> = 0.982, <a href="#fig2">Figure 2E</a> and <a href="#fig2s1">Figure 2—figure supplement 1A</a>), and the concordance of same day samples suggests that our sampling procedure is reproducible. An analysis of sequence data from replicate libraries indicates that our estimates of iSNV frequency are precise (<a href="#fig2s1">Figure 2—figure supplement 1B</a>). In samples separated by at least a day, only 57% iSNV found in the first sample persisted in the second, above the 2% limit of detection. Additionally, the majority of iSNV (68%) found in the second sample were either new or previously present below the 2% limit of detection. Taken together, these data suggest that the population present in the upper respiratory tract is highly dynamic while maintaining a stable consensus. Of note, 6 iSNV in our longitudinal data set lay within antigenic epitopes (arrows in <a href="#fig2">Figure 2E</a>). Their dynamics are similar to those of other nonsynonymous iSNV and synonymous iSNV, suggesting that most mutations change in frequency due to stochastic as opposed to selective processes. Together with our prior work (<a href="#bib5">Debbink et al., 2017</a>), these data suggest that the positive selection of novel variants within hosts is inefficient and rarely amplifies a newly generated variant to a frequency greater than 2%.</p> </div> <div class="identificationofforty-threetransmissionpairs" title="sec"> <div class="title" tagx="title" title="title">
Identification of forty-three transmission pairs</div> <p>Our within-host data suggest that newly arising iSNV with positive fitness effects are likely to be present at low frequencies (&amp;lt;2%) during an acute infection. The maintenance of these mutations in host populations is therefore highly dependent on the transmission bottleneck. We analyzed virus populations from 85 households with concurrent infections to quantify the level of shared viral diversity and to estimate the size of the IAV transmission bottleneck (<a href="#table1">Table 1</a>). Because epidemiological linkage does not guarantee that concurrent cases constitute a transmission pair (<a href="#bib36">Petrie et al., 2017</a>), we used stringent criteria to eliminate individuals in a household with co-incident community acquisition of distinct viruses. We considered all individuals in a household with symptom onset within a 7 day window to be epidemiologically linked. The donor in each putative pair was defined as the individual with the earlier onset of symptoms. We discarded a transmission event if there were multiple possible donors with the same day of symptom onset. Donor and recipients were not allowed to have symptom onset on the same day, unless the individuals were both index cases for the household. In these six instances, we analyzed the data for both possible donor-recipient directionalities. Based on these criteria, our cohort had 124 putative household transmission events over five seasons (<a href="#table1">Table 1</a>). Of these, 52 pairs had samples of sufficient quality for reliable identification of iSNV from both individuals.</p> <p>We next used sequence data to determine which of these 52 epidemiologically linked pairs represented true household transmission events as opposed to coincident community-acquired infections. We measured the genetic distance between influenza populations from each household pair by L1-norm and compared these distances to those of randomly assigned community pairs within each season (<a href="#fig3">Figure 3A</a>, see also trees in <a href="#fig1s2">Figure 1—figure supplements 2</a> and <a href="#fig1s3">3</a>). While the L1-norm of a pair captures differences between the populations at all levels, in our cohort, it was largely driven by differences at the consensus level. We only considered individuals to be a true transmission pair if they had a genetic distance below the 5th percentile of the community distribution of randomly assigned pairs (<a href="#fig3">Figure 3A</a>). Forty-seven household transmission events met this criterion (<a href="#fig3">Figure 3B</a>). Among these 47 sequence-validated transmission pairs, three had no iSNV in the donor and one additional donor appeared to have a mixed infection. These four transmission events were removed from our bottleneck analysis, as donors without iSNV are uninformative and mixed infections violate model assumptions of site independence (see Materials and methods). We estimated the transmission bottleneck in the remaining 43 high-quality pairs (37 H3N2, 6 H1N1, <a href="#fig3">Figure 3B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.35962.019"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Between host dynamics of IAV.</div> <p>(<b>A</b>) The distribution of pairwise L1-norm distances for household (blue) and randomly-assigned community (gold) pairs. The bar heights are normalized to the height of the highest bar for each given subset (47 for household, 1592 for community). The red line represents the 5<sup>th</sup> percentile of the community distribution. (<b>B</b>) Timing of symptom onset for 52 epidemiologically linked transmission pairs. Days of symptom onset for both donor and recipient individuals are indicated by black dots. Dashed lines represent pairs that were removed due to abnormally high genetic distance between isolates, see (<b>A</b>). (<b>C</b>) The frequency of donor iSNV in both donor and recipient samples. Frequencies below 2% and above 98% were set to 0% and 100% respectively. (<b>D</b>) The presence-absence model fit compared with the observed data. The x-axis represents the frequency of donor iSNV with transmitted iSNV plotted along the top and nontransmitted iSNV plotted along the bottom. The line represents the predicted probability of transmission given by the presence-absence model with a mean bottleneck of 1.68. The shaded regions represent the 95% confidence interval. Black points on the plot represent the probability of transmission estimated as the proportion of iSNV transmitted within a sliding window of width 5% and a step of 1%. The error bars represent the 95% confidence interval and were derived from a binomial distribution as in (<a href="#bib46">Sobel Leonard et al., 2017</a>). Only those windows with more than 5 iSNV are plotted. Blue curve shows the probability of transmission at a given frequency given a bottleneck size of 10 in the presence-absence model. (<b>E</b>) The beta-binomial model fit. Similar to (<b>D</b>), except the predicted outcomes are the based on a beta-binomial model using a mean bottleneck of 1.75. Blue curve shows the probability of transmission at a given frequency given a bottleneck size of 10 in the beta-binomial model.</p> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.021">10.7554/eLife.35962.021</a><span class="label" tagx="label" title="label">Figure 3—source data 1.</span> <div class="title" tagx="title" title="title">
Genetic distance of household and community sample pairs.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data1.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.022">10.7554/eLife.35962.022</a><span class="label" tagx="label" title="label">Figure 3—source data 2.</span> <div class="title" tagx="title" title="title">
Day of onset and meta data for transmission pairs.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data2.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.023">10.7554/eLife.35962.023</a><span class="label" tagx="label" title="label">Figure 3—source data 3.</span> <div class="title" tagx="title" title="title">
Frequencies of iSNV identified in transmission pairs.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data3.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.024">10.7554/eLife.35962.024</a><span class="label" tagx="label" title="label">Figure 3—source data 4.</span> <div class="title" tagx="title" title="title">
The model prediction of the probability of transmission given donor frequency for the presence-absence model.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data4.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.025">10.7554/eLife.35962.025</a><span class="label" tagx="label" title="label">Figure 3—source data 5.</span> <div class="title" tagx="title" title="title">
The frequency of donor iSNV used in fitting transmission models.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data5.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.026">10.7554/eLife.35962.026</a><span class="label" tagx="label" title="label">Figure 3—source data 6.</span> <div class="title" tagx="title" title="title">
The model prediction of the probability of transmission given donor frequency for the beta-binomial model.</div>  <div class="media" title="media"><a href="elife-35962-fig3-data6.csv">LINK</a></div> </div> <p /> <p> </p><div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.027">10.7554/eLife.35962.027</a><span class="label" tagx="label" title="label">Figure 3—source data 7.</span> <div class="title" tagx="title" title="title">
Bottleneck estimates for all isolate pairings in cases where multiple donor or recipient isolates are available.</div> <p>SPECID refers to specimen ID. Those beginning in M were taken at the clinic. Those beginning with HS were isolated at home.</p>  <div class="media" title="media"><a href="elife-35962-fig3-data7.csv">LINK</a></div> </div> <p />   </div> <p>A transmission bottleneck restricts the amount of genetic diversity that is shared by both members of a pair. We found that few minority iSNV where polymorphic in both the donor and recipient populations (<a href="#fig3">Figure 3C</a>). Minority iSNV in the donor were either absent or fixed in the recipient (top and bottom of plot). The lack of shared polymorphic sites (which would lie in the middle of the plot in <a href="#fig3">Figure 3C</a>) suggests a stringent effective bottleneck in which only one allele is passed from donor to recipient.</p> </div> <div class="estimationofthetransmissionbottleneck" title="sec"> <div class="title" tagx="title" title="title">
Estimation of the transmission bottleneck</div> <p>We applied a simple presence-absence model to quantify the effective transmission bottleneck in our cohort. The presence-absence model considers only whether or not a donor allele is present or absent in the recipient sample. Under this model, transmission is a neutral, random sampling process, and the probability of transmission is simply the probability that the iSNV will be included at least once in the sample given its frequency in the donor and the sample size, or bottleneck. We estimated a distinct bottleneck for each transmission pair and assumed these bottlenecks followed a zero-truncated Poisson distribution. Maximum likelihood optimization determined that a mean bottleneck of 1.68 (lambda = 1.15; 0.49–2.14, 95% CI) best described the data. This distribution indicates that the majority of bottlenecks are 1 and that 95% of bottlenecks are expected to be ≤3. One outlier was a single transmission pair with an estimated bottleneck of &amp;gt;200; this pair had two minority iSNV present at very low frequencies in both donor and recipient. There were no apparent differences between H3N2 and H1N1 pairs. The model fit was evaluated by comparing the probability of transmission predicted by the model with that estimated from the data using a sliding window (<a href="#fig3">Figure 3D</a>). The predicted transmission probabilities capture the underlying trend in the data, which suggests that transmission dynamics can be appropriately modeled using a selectively neutral, stringent bottleneck.</p> <p>Because the presence-absence model can underestimate the transmission bottleneck in some circumstances (<a href="#bib46">Sobel Leonard et al., 2017</a>), we also applied a beta binomial model, which Leonard <i>et al.</i> have used to account for the stochastic dynamics of transmitted variants. This model allows for a limited amount of time-independent genetic drift within the recipient (<a href="#bib46">Sobel Leonard et al., 2017</a>), and we modified it to also account for our benchmarked sensitivity for rare variants (<a href="#supp1">Supplementary file 1</a>, Current Pipeline). For all donor-derived iSNV that were absent in the recipient, we estimated the likelihood that these variants were transmitted but either drifted below our level of detection or drifted below 10% and were missed by our variant identification. Despite the relaxed assumptions provided by this modified beta binomial model, maximum likelihood estimation only marginally increased the average bottleneck size (mean 1.75: lambda 1.25; 0.55–2.28, 95% CI) relative to the simpler presence-absence model. We also fit a long-tailed, discrete distribution based on the log-normal. As expected, this analysis resulted in a slightly wider distribution with a mode of 1, a 95<sup>th</sup> percentile of 11 and a 97.5th percentile of 21. Although the beta-binomial again reproduced the trend in the data (<a href="#fig3">Figure 3E</a>), the fit was not better than that of the presence-absence model (AIC 80.2 for beta-binomial compared to 74.5 for the presence-absence model), suggesting the stochastic loss of iSNV in the recipient is not a major contributing factor to the transmission dynamics observed here. As in the presence-absence model, there was no apparent stratification in bottleneck size based on vaccination status, donor or recipient age, or influenza A subtype (<a href="#supp3">Supplementary file 3</a>).</p> <p>Because our bottleneck estimates were much lower than what has previously been reported for human influenza (<a href="#bib39">Poon et al., 2016</a>), we investigated the impact that our simplifying assumptions could have on our results. In particular, the presence-absence model assumes perfect detection of variants in donor and recipient, and it can therefore underestimate the size of a bottleneck in the setting of donor-derived variants that are transmitted but not detected in the recipient. These ‘false negative’ variants can occur when the frequency of an iSNV drifts below the level of detection (e.g. 2% frequency) or when the sensitivity of sequencing is less than perfect for variants at that threshold (e.g. 15% sensitivity for variants at a frequency 2–5%). To determine the impact of sequencing sensitivity and specificity on our bottleneck estimates, we re-called variants using our original pipeline without the 2% frequency cut-off. As shown in <a href="#supp1">Supplementary file 1</a>, this increases the sensitivity of iSNV detection in the 1–5% frequency range, and also the number of false positive variant calls (<a href="#bib24">McCrone and Lauring, 2016</a>). This analysis only slightly increased average transmission bottleneck to 2.11 (lambda = 1.75; 0.87–2.95, 95% CI), and indicates that our results are not biased by the added stringency used in the initial analysis. Interestingly, the inclusion of these low-frequency iSNV reduced model fit and led to an overestimation of the probability of transmitting rare iSNV (<a href="#fig3s1">Figure 3—figure supplement 1</a>). These low-frequency bins are likely dominated by iSNV that were not present at the time of transmission either because they are sequencing artifacts or underwent stochastic extinction in the donor prior to transmission.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We characterized influenza virus dynamics in human hosts through sequencing of 249 specimens from 200 individuals collected over 6290 person-seasons of observation. In our study, we find that acute influenza infections are characterized by low diversity, limited positive selection, and tight transmission bottlenecks. Because we used viruses collected over five influenza seasons from individuals enrolled in a prospective household cohort, these dynamics are likely to be broadly representative of many seasonal influenza infections in temperate regions. Our results are further strengthened by the use of a validated sequence analysis pipeline and models that are robust to the underlying assumptions. The observed within-host dynamics and the tight effective transmission bottleneck suggest that stochastic processes, such as genetic drift, dominate influenza virus evolution at the level of individual hosts. This stands in contrast to the prominent role of positive selection in the global evolution of seasonal influenza.</p> <p>Contrary to previous studies, which have found signatures of adaptive evolution in infected hosts (<a href="#bib13">Gubareva et al., 2001</a>; <a href="#bib42">Rogers et al., 2015</a>; <a href="#bib12">Ghedin et al., 2011</a>; <a href="#bib45">Sobel Leonard et al., 2016</a>), we have found only limited evidence of positive selection during acute infection. Previous reports have relied on infections in which selective pressures are likely to be particularly strong (e.g. due to drug treatment or infection with a poorly adapted virus), or in which the virus has replicated within a host for an extended period of time (<a href="#bib51">Xue et al., 2017</a>). Under these conditions, it is plausible that positively selected alleles reach levels of detection. We suggest that these are important and informative exceptions to dynamics defined here, in which positive selection is rarely strong enough to drive a new mutation to a frequency above 2% over the course of several days. Our findings are consistent with the observed global rates of influenza evolution and with epidemiological studies that have shown limited antigenic selection over the course of local epidemics (<a href="#bib31">Nelson et al., 2006</a>; <a href="#bib3">Bedford et al., 2015</a>).</p> <p>We used both a simple presence-absence model and a more complex beta binomial model to estimate an extremely tight transmission bottleneck. The estimation of a small bottleneck is due to the fact that at most polymorphic sites in the donor, only one allele (iSNV) was found in the recipient. While our methods for variant calling may be more conservative than those used in similar studies, we found that relaxing our variant calling criteria only led to the inclusion of false positive variants and did not significantly inflate our estimates. Furthermore, the beta binomial model accounts for false negative iSNV (i.e. variants that are transmitted but not detected in the donor), which can lead to underestimated transmission bottlenecks (<a href="#bib46">Sobel Leonard et al., 2017</a>). Our formulation of this model incorporates empirically determined sensitivity and specificity metrics to account for both false negative iSNV and false positive iSNV (<a href="#bib24">McCrone and Lauring, 2016</a>). Finally, if rare, undetected, iSNV were shared between linked individuals, we would expect to see transmission of more common iSNV (frequency 5–10%), which we can detect with high sensitivity. In our data, the transmission probability iSNVs &amp;gt; 5% frequency in the donor were also well predicted by small bottleneck size (<a href="#fig3">Figure 3D and E</a>).</p> <p>Although the size of our transmission bottleneck is consistent with estimates obtained for other viruses and in experimental animal models of influenza (<a href="#bib53">Zwart and Elena, 2015</a>; <a href="#bib47">Varble et al., 2014</a>), it differs substantially from the only other study of bottlenecks in natural human infection (<a href="#bib39">Poon et al., 2016</a>; <a href="#bib46">Sobel Leonard et al., 2017</a>). While there are significant differences in the design and demographics of the cohorts, the influenza seasons under study, and sequencing methodology (<a href="#bib22">Kugelman et al., 2017</a>), the bottleneck size estimates are fundamentally driven by the amount of viral diversity shared among individuals in a household. Importantly, and as in Poon et al., we used both epidemiologic linkage and the genetic relatedness of viruses in households to define transmission pairs and to exclude confounding from the observed background diversity in the community. We find that household transmission pairs and randomly assigned community pairs had distinct patterns of shared consensus and minority variant diversity, a pattern that is distinct from the observations of Poon et al. An unexplained aspect of their study is that rare iSNV were frequently shared by randomly selected individuals, and more common ones were not (<a href="#bib39">Poon et al., 2016</a>).</p> <p>Accurately modeling and predicting influenza virus evolution requires a thorough understanding of the virus’ population structure. Some models have assumed a large intrahost population and a relatively loose transmission bottleneck (<a href="#bib10">Geoghegan et al., 2016</a>; <a href="#bib43">Russell et al., 2012</a>; <a href="#bib35">Peck et al., 2015</a>). Here, adaptive iSNV can rapidly rise in frequency and low frequency variants can have a high probability of transmission. In such a model, it would be possible for an emerging virus to become more transmissible or a seasonal virus to evolve resistance to vaccine-induced immunity over a short transmission chain (<a href="#bib14">Herfst et al., 2012</a>; <a href="#bib43">Russell et al., 2012</a>). Although the dynamics of emergent avian influenza and human adapted seasonal viruses likely differ (<a href="#bib38">Petrova and Russell, 2017</a>), our work suggests that fixation of multiple mutations over the course of a single acute infection is unlikely.</p> <p>While it may seem counterintuitive that influenza evolution is dominated by stochasticity on local scales and positive selection on global scales, these models are certainly not in conflict. We have deeply sequenced 332 intrahost populations from 283 individuals collected over more than 11,000 person-seasons of observation and only identified a handful of minority antigenic variants with limited evidence for positive selection (this work and [<a href="#bib5">Debbink et al., 2017</a>]). Importantly, our data suggest that even if selection acts below our level of detection, such rare variants are unlikely to transmit. Given the size of the estimated bottleneck, the probability of transmission is approximately 1.7% for a variant at 1% frequency and 3.3% for a variant at 2% frequency. However, with several million infected individuals each year, even inefficient processes and rare events at the scale of individual hosts are likely to occur at a reasonable frequency on a global scale.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="descriptionofthecohort" title="sec"> <div class="title" tagx="title" title="title">
Description of the cohort</div> <p>The HIVE cohort (<a href="#bib27">Monto et al., 2014</a>; <a href="#bib32">Ohmit et al., 2013</a>, <a href="#bib33">2015</a>, <a href="#bib34">2016</a>; <a href="#bib37">Petrie et al., 2013</a>), established at the UM School of Public Health in 2010, enrolled and followed households of at least three individuals with at least two children &amp;lt; 18 years of age; households were then followed prospectively throughout the year for ascertainment of acute respiratory illnesses. Study participants were queried weekly about the onset of illnesses meeting our standard case definition (two or more of: cough, fever/feverishness, nasal congestion, sore throat, body aches, chills, headache if ≥3 years old; cough, fever/feverishness, nasal congestion/runny nose, trouble breathing, fussiness/irritability, decreased appetite, fatigue in &amp;lt;3 years old), and the symptomatic participants then attended a study visit at the research clinic on site at UM School of Public Health for sample collection. For the 2010–2011 through 2013–2014 seasons, a combined nasal and throat swab (or nasal swab only in children &amp;lt; 3 years of age) was collected at the onsite research clinic by the study team. Beginning with the 2014–2015 seasons, respiratory samples were collected at two time points in each participant meeting the case definition; the first collection was a self- or parent-collected nasal swab collected at illness onset. Subsequently, a combined nasal and throat swab (or nasal swab only in children &amp;lt; 3 years of age) was collected at the onsite research clinic by the study team. Families with very young children (&amp;lt;3 years of age) were followed using home visits by a trained medical assistant.</p> <p>Active illness surveillance and sample collection for cases were conducted October through May and fully captured the influenza season in Southeast Michigan in each of the study years. Data on participant, family and household characteristics, and on high-risk conditions were additionally collected by annual interview and review of each participant’s electronic medical record. In the current cohort, serum specimens were also collected twice yearly during fall (November-December) and spring (May-June) for serologic testing for antibodies against influenza.</p> <p>This study was approved by the Institutional Review Board of the University of Michigan Medical School. Adults provided written informed consent for participation for themselves and their children; children 7–17 years provided oral assent.</p> </div> <div class="identificationofinfluenzavirus" title="sec"> <div class="title" tagx="title" title="title">
Identification of influenza virus</div> <p>Respiratory specimens were processed daily to determine laboratory-confirmed influenza infection. Viral RNA was extracted (Qiagen QIAamp Viral RNA Mini Kit, Germantown, MD) and tested by RT-PCR for universal detection of influenza A and B. Samples with positive results by the universal assay were then subtyped to determine A(H3N2), A(H1N1), A(pH1N1) subtypes and B(Yamagata) and B(Victoria) lineages. We used primers, probes and amplification parameters developed by the Centers for Disease Control and Prevention Influenza Division for use on the ABI 7500 Fast Real-Time PCR System platform. An RNAseP detection step was run for each specimen to confirm specimen quality and successful RNA extraction.</p> </div> <div class="quantificationofviralload" title="sec"> <div class="title" tagx="title" title="title">
Quantification of viral load</div> <p>Quantitative reverse transcription polymerase chain reaction (RT-qPCR) was performed on 5 μl RNA from each sample using CDC RT-PCR primers InfA Forward, InfA Reverse, and InfA probe, which bind to a portion of the influenza M gene (CDC protocol, 28 April 2009). Each reaction contained 5.4 μl nuclease-free water, 0.5 μl each primer/probe, 0.5 μl SuperScript III RT/Platinum Taq mix (Invitrogen111732, Carlsbad, CA) 12.5 μl PCR Master Mix, 0.1 μl ROX, 5 μl RNA. The PCR master mix was thawed and stored at 4°C, 24 hr before reaction set-up. A standard curve relating copy number to Ct value was generated based on 10-fold dilutions of a control plasmid run in duplicate.</p> </div> <div class="illuminalibrarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Illumina library preparation and sequencing</div> <p>We amplified cDNA corresponding to all eight genomic segments from 5 μl of viral RNA using the SuperScript III One-Step RT-PCR Platinum Taq HiFi Kit (Invitrogen 12574). Reactions consisted of 0.5 μl Superscript III Platinum Taq Mix, 12.5 μl 2x reaction buffer, 6 μl DEPC water, and 0.2 μl of 10 μM Uni12/Inf1, 0.3 μl of 10 μM Uni12/Inf3, and 0.5 μl of 10 μM Uni13/Inf1 universal influenza A primers (<a href="#bib52">Zhou et al., 2009</a>). The thermocycler protocol was: 42 ˚C for 60 min then 94 ˚C for 2 min then 5 cycles of 94 ˚C for 30 s, 44 ˚C for 30 s, 68 ˚C for 3 min, then 28 cycles of 94 ˚C for 30 s, 57 ˚C for 30 s, 68 ˚C for 3 min. Amplification of all eight segments was confirmed by gel electrophoresis, and 750 ng of each cDNA mixture were sheared to an average size of 300 to 400 bp using a Covaris (Woburn, MA) S220 focused ultrasonicator. Sequencing libraries were prepared using the NEBNext Ultra DNA library prep kit (NEB E7370L), Agencourt AMPure XP beads (Beckman Coulter A63881, Indianapolis, IN), and NEBNext multiplex oligonucleotides for Illumina (NEB E7600S, Ipswich, MA). The final concentration of each barcoded library was determined by Quanti PicoGreen dsDNA quantification (ThermoFisher Scientific, Waltham, MA), and equal nanomolar concentrations were pooled. Residual primer dimers were removed by gel isolation of a 300–500 bp band, which was purified using a GeneJet Gel Extraction Kit (ThermoFisher Scientific). Purified library pools were sequenced on an Illumina HiSeq 2500 with 2 × 125 nucleotide paired end reads. All raw sequence data have been deposited at the NCBI sequence read archive (BioProject Accession number: PRJNA412631). PCR amplicons derived from an equimolar mixture of eight clonal plasmids, each containing a genomic segment of the circulating strain were processed in similar fashion and sequenced on the same HiSeq flow cell as the appropriate patient derived samples. These clonally derived samples served as internal controls to improve the accuracy of variant identification and control for batch effects that confound sequencing experiments.</p> </div> <div class="identificationofisnv" title="sec"> <div class="title" tagx="title" title="title">
Identification of iSNV</div> <p>Intrahost single nucleotide variants were identified in samples that had greater than 10<sup>3</sup> genomes/μl and an average coverage &amp;gt;1000 x across the genome. Variants were identified using DeepSNV and scripts available at <a href="https://github.com/lauringlab/variant_pipeline">https://github.com/lauringlab/variant_pipeline</a> as described previously (<a href="#bib24">McCrone and Lauring, 2016</a>; copy archived at <a href="https://github.com/elifesciences-publications/variant_pipeline">https://github.com/elifesciences-publications/variant_pipeline</a>) with a few minor and necessary modifications. Briefly, reads were aligned to the reference sequence (H3N2 2010–2011 and 2011–2012: GenBank CY121496-503, H3N2 2012–2013:GenBank KJ942680-8, H3N2 2014–2015: Genbank CY207731-8, H1N1 GenBank: CY121680-8) using Bowtie2 (35). Duplicate reads were then marked and removed using Picard (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>). We identified putative iSNV using DeepSNV. Bases with phred &amp;lt;30 were masked. Minority iSNV (frequency &amp;lt;50%) were then filtered for quality using our empirically determined quality thresholds (p-value&amp;lt;0.01 DeepSNV, average mapping quality &amp;gt;30, average Phred &amp;gt; 35, average read position between 31 and 94). To control for RT-PCR errors in samples with lower input titers, all isolates with titers between 10<sup>3</sup> and 10<sup>5</sup> genomes/μl were processed and sequenced in duplicate. Only iSNV that were found in both replicates were included in down stream analysis. The frequency of the variant in the replicate with higher coverage at the iSNV location was assigned as the frequency of the iSNV. Finally, any SNV with a frequency below 2% was discarded.</p> <p>Given the diversity of the circulating strain in a given season, there were a number of cases in which isolates contained mutations that were essentially fixed (&amp;gt;95%) relative to the plasmid control. Often in these cases, the minor allele in the sample matched the major allele in the plasmid control. We were, therefore, unable to use DeepSNV in estimating the base specific error rate at this site for these minor alleles and required an alternative means of discriminating true and false minority iSNV. To this end we applied stringent quality thresholds to these putative iSNV and implemented a 2% frequency threshold. In order to ensure we were not introducing a large number of false positive iSNV into our analysis, we performed the following experiment. Perth (H3N2) samples were sequenced on the same flow cell as both the Perth and Victoria (H3N2) plasmid controls. Minority iSNV were identified using both plasmid controls. This allowed us to identify rare iSNV at positions in which the plasmid controls differed both with and without the error rates provided by DeepSNV. We found that at a frequency threshold of 2% the methods were nearly identical (NPV of 1, and PPV of 0.94 compared to DeepSNV).</p> </div> <div class="overviewofmodelsusedforestimatingthetransmissionbottleneck" title="sec"> <div class="title" tagx="title" title="title">
Overview of models used for estimating the transmission bottleneck</div> <p>We model transmission as a simple binomial sampling process (<a href="#bib46">Sobel Leonard et al., 2017</a>). In our first model, we assume any transmitted iSNV, no matter the frequency, will be detected in the recipient. In the second, we relax this assumption and account for false negative iSNV in the recipient. To include the variance in the transmission bottlenecks between pairs we use maximum likelihood optimization to fit the average bottleneck size assuming the distribution follows a zero-truncated Poisson distribution.</p> </div> <div class="presence/absencemodel" title="sec"> <div class="title" tagx="title" title="title">
Presence/Absence model</div> <p>The presence/absence model makes several assumptions. We assume perfect detection of all transmitted iSNV in the recipient. For each donor iSNV, we measure only whether or not the variant is present in the recipient. Any iSNV that is not found in the recipient is assumed to have not been transmitted. We also assume the probability of transmission is determined only by the frequency of the iSNV in the donor at the time of sampling (regardless of how much time passes between sampling and transmission). The probability of transmission is simply the probability that the iSNV is included at least once in a sample size equal to the bottleneck. Finally, we assume all genomic sites are independent of one another. For this reason, we discarded the one case where the donor was likely infected by two strains, as the iSNV were certainly linked.</p> <p>Because the presence/absence model is unaware of the frequency of alleles in the recipient we must track both alleles at each donor polymorphic site.</p> <p>Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> be alleles in donor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> at genomic site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>. Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></span> be the probability that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> is the only transmitted allele. There are three possible outcomes for each site. Either only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> is transmitted, only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> is transmitted, or both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> are transmitted. The probability of only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> being transmitted given a bottleneck size of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> is </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> is the frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> in the donor. In other words, this is simply the probability of only drawing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> draws. The probability that only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> is transmitted is similarly defined.<p /> <p>The probability of both alleles being transmitted is given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> are the frequencies of the alleles respectively. This is simply the probability of not picking only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> or only <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
A</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> draws.<p /> <p>This system could easily be extended to cases where there are more than two alleles present at a site; however, that never occurs in our data set.</p> <p>For ease, we will denote the likelihood of observing the data at a polymorphic site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> in each donor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> given the bottleneck size <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></span> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></span> is defined by equation 1 if only one allele is transmitted and equation 2 if two alleles are transmitted.</p> <p>The log likelihood of a bottleneck of size <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> is given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div></div></span> refers to the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>th polymorphic site in the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>th donor. This is the log of the probability of observing the data summed over all polymorphic sites across all donors.<p /> <p>Because the bottleneck size is likely to vary across transmission events, we used maximum likelihood to fit the bottleneck distribution as opposed to fitting a single bottleneck value. Under this model we assumed the bottlenecks were distributed according to a zero-truncated Poisson distribution parameterized by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span>. The likelihood of observing the data given a polymorphic site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> in donor <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span> is </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
∞</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></span> is defined as above, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div></div></span> is the probability of drawing a bottleneck of size <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> from a zero-truncated Poisson distribution with a mean of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
λ</div> </div> </div> </div> </div></div></span>. The sum is across all possible <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> defined on <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
∞</div> <div class="mo_UNKNOWN">
)</div></div></span>. Unless otherwise noted, we only investigated bottleneck sizes up to 100, as initial analyses suggested <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span> is quite small and the probability of drawing a bottleneck size of 100 from a zero-truncated Poisson distribution with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
10</div></div></span> is negligible. We follow this convention whenever this sum appears.<p /> <p>The log likelihood of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span> for the data set is given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mi_UNKNOWN">
n</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
∞</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div></div></div> <p /> </div> <div class="betabinomialmodel" title="sec"> <div class="title" tagx="title" title="title">
Beta binomial model</div> <p>The Beta binomial model is explained in detail in Leonard <i>et al.</i> (Sobel Leonard et al. 2017). It is similar to the presence/absence model in that transmission is modeled as a simple sampling process; however, it relaxes the following assumptions. In this model, the frequencies of transmitted variants are allowed to change between transmission and sampling according a beta distribution. The distribution is not dependent on the amount of time that passes between transmission and sampling, but rather depends on the size of the founding population (here assumed to equal to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>) and the number of variant genomes present in founding population <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span>. Note the frequency in the donor is assumed to be the same between sampling and transmission.</p> <p>The equations below are very similar to those presented by Leonard <i>et al.</i> with one exception. Because we know the sensitivity of our method to detect rare variants based on the expected frequency and the titer, we can include the possibility that iSNV are transmitted but are missed due to poor sensitivity. Because the beta binomial model is aware of the frequency of the iSNV in the recipient, no information is added by tracking both alleles at a genomic site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>.</p> <p>Let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div></div></span> represent the frequency of the minor allele at position <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> in the donor of some transmission pair <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>. Similarly, let <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> </div> </div></div></span> be the frequency of that same allele in the recipient of the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>th transmission pair. Then, as in Leonard <i>et al.</i>, the likelihood of some bottleneck <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> for the data at site <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> in pair <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> where the minor allele is transmitted is given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
a</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
)</div> <div class="mi_UNKNOWN">
p</div> <div class="mo_UNKNOWN">
_</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∣</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div></div></div> <p /> <p>Where p_beta is the probability density function for the beta distribution and p_bin is the probability mass function for the binomial distribution.</p> <p>This is the probability density that the transmitted allele is found in the recipient at a frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> </div> </div></div></span> given that the variant was in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span> genomes in a founding population of size <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> times the probability of drawing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span> variant genomes in a sample size of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> and a variant frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div></div></span>. This is then summed for all possible <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
≤</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>.</p> <p>As in equation 4 the likelihood of a zero truncated Poisson with a mean of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
λ</div> </div> </div> </div> </div></div></span> given this transmitted variants is then given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
λ</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
∞</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div></div></div> <p /> <p>This is simply the likelihood of each <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> weighted by the probability of drawing a bottleneck size of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> from bottleneck distribution.</p> <p>In this model, there are three possible mechanisms for a donor iSNV to not be detected in the recipient. (i) The variant was not transmitted. (ii) The variant was transmitted but is present below our limit of detection (2%). (iii) The variant was transmitted and is present above our limit of detection but represents a false negative in iSNV identification.</p> <p>As in Leonard <i>et al.,</i> the likelihood of scenarios (i) and (ii) for a given <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> are expressed as </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
o</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.02</div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∣</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></div> <p /> <p>Where p_beta_cdf is the cumulative distribution function for the beta distribution. Note that if <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div></div></span> (i.e. the iSNV was not transmitted) then the term reduces to the probability of not drawing the variant in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> draws.</p> <p>The likelihood of the variant being transmitted but not detected in the recipient given a bottleneck of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> is described by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> </div> <div class="mtd_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
0.02</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0.05</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0.1</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
×</div> </div> </div> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN" /> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> </div> <div class="mi_UNKNOWN">
_</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∣</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> </div></div></div> <p /> <p>This is the likelihood of the variant existing in the ranges [0.02,0.05] or [0.05,0.1] given an initial frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> and a bottleneck size of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> multiplied by the expected false negative rate (FNR) given the titer of the recipient and the lower frequency bound. We assumed perfect sensitivity for detection of iSNV present above 10%, rounded recipient titers down to the nearest <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
g</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> </div> <div class="mi_UNKNOWN" /></div></span>titer (e.g. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> </div> </div></div></span>,<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
4</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
5</div> </div> </div></div></span>) and assumed the entire range <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
]</div></div></span> has the same sensitivity as the lower bound.</p> <p>The likelihood of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div></div></span> for iSNV that are not observed in the recipient is then given by summing equations 8 and 9 across all possible <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span>. </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
λ</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
n</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
t</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
∞</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div> <div class="mo_UNKNOWN">
∣</div> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> <p /> <p>The log likelihood of the total dataset is then determined by summing the log of equations 7 and 10 (as applicable) across all polymorphic sites in each donor. (As before here we sum of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
b</div> </div> </div></div></span> within the range <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mn_UNKNOWN">
1,100</div> <div class="mo_UNKNOWN">
]</div></div></span>.)</p> </div> <div class="modelcomparisons" title="sec"> <div class="title" tagx="title" title="title">
Model comparisons</div> <p>In order evaluate the fits of the two transmission models, we calculated the probability of detecting a donor iSNV in the recipient given its frequency in the donor. For each model this reduces to one minus the probability of only detecting the other allele in the recipient summed over all possible bottlenecks. As in fitting the models this infinite sum was approximated by a partial sum up to a bottleneck size of 100. We used a recipient titer of 10<sup>4</sup> genomes/μl to estimate the effect of sequencing sensitivity for the beta binomial model.</p> <p>Annotated computer code for all analyses and for generating the figures can be accessed at <a href="https://github.com/lauringlab/Host_level_IAV_evolution">https://github.com/lauringlab/Host_level_IAV_evolution</a>. (copy archived at <a href="https://github.com/elifesciences-publications/Host_level_IAV_evolution">https://github.com/elifesciences-publications/Host_level_IAV_evolution</a>)</p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000057</span><span class="institution" title="institution">National Institute of General Medical Sciences</span></div> </div><span class="award-id" tagx="award-id" title="award-id">T32 GM007544</span> to John T McCrone.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">K08 AI119182</span> to Robert J Woods.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000030</span><span class="institution" title="institution">Centers for Disease Control and Prevention</span></div> </div><span class="award-id" tagx="award-id" title="award-id">U01 IP00474</span> to Arnold S Monto.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 AI097150</span> to Arnold S Monto.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000862</span><span class="institution" title="institution">Doris Duke Charitable Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">CSDA 2013105</span> to Adam S Lauring.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01 AI118886</span> to Adam S Lauring.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100007270</span><span class="institution" title="institution">University of Michigan</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Discovery grant</span> to Adam S Lauring.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was supported by a Clinician Scientist Development Award from the Doris Duke Charitable Foundation (CSDA 2013105), a University of Michigan Discovery Grant, and R01 AI118886, all to ASL. The HIVE cohort was supported by NIH R01 AI097150 and CDC U01 IP00474 to ASM. JTM was supported by the Michigan Predoctoral Training Program in Genetics (T32GM007544). RJW was supported by K08 AI119182. We thank Katherine Xue for suggestions on analysis of globally circulating variants and Alexey Kondrashov and Aaron King for helpful discussion.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Resources, Data curation, Software, Formal analysis, Investigation, Visualization, Methodology, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Formal analysis, Investigation, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Data curation, Project administration, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Investigation, Methodology, Project administration, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Data curation, Formal analysis, Supervision, Funding acquisition, Investigation, Methodology, Writing—original draft, Project administration, Writing—review and editing.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Ethics</b></div> <div class="fn-type-other" title="other"> <p>Human subjects: This study was approved by the Institutional Review Board of the University of Michigan Medical School. Adults provided written informed consent for participation for themselves and their children; children 7-17 years provided oral assent.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.028">10.7554/eLife.35962.028</a><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
Sensitivity and specificity of variant detection</div>  <div class="media" title="media"><a href="elife-35962-supp1.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.029">10.7554/eLife.35962.029</a><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Nonsynonymous substitutions in HA antigenic sites</div>  <div class="media" title="media"><a href="elife-35962-supp2.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.030">10.7554/eLife.35962.030</a><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
Estimated bottleneck size for individual transmission pairs</div>  <div class="media" title="media"><a href="elife-35962-supp3.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.35962.031">10.7554/eLife.35962.031</a><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-35962-transrepform.docx">LINK</a></div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data generated or analyzed during this study are included in the manuscript and supporting files. Source data files have been provided. All sequence reads have been deposited to NCBI's BioProject under accession number PRJNA412631.</p> <p>The following dataset was generated:</p> <p><span class="related-object" tagx="related-object" title="related-object"><span class="collab'">collab: John T McCrone</span><span class="collab'">collab: Robert J Woods</span><span class="collab'">collab: Emily T Martin</span><span class="collab'">collab: Ryan E Malosh</span><span class="collab'">collab: Arnold S Monto</span><span class="collab'">collab: Adam S Lauring</span><span class="year" tagx="year" title="year">2018</span><span class="source" tagx="source" title="source">Whole genome sequencing of Influenza isolates from a prospective household cohort</span><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA412631">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA412631</a><span class="comment">Publicly available at the NCBI BioProject (accession no: PRJNA412631).</span></span></p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alizon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luciani</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regoes</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Epidemiological and clinical consequences of within-host evolution</span><span class="source" tagx="source" title="source">Trends in Microbiology</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">24</span><span class="lpage" tagx="lpage" title="lpage">32</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2010.09.005">10.1016/j.tim.2010.09.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21055948">21055948</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archetti</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horsfall</span><span class="given-names" tagx="given-names" title="given-names">FL</span></span></span><span class="year" tagx="year" title="year">1950</span><span class="mixed-article-title" title="mixed-article-title">Persistent antigenic variation of influenza A viruses after incomplete neutralization in ovo with heterologous immune serum</span><span class="source" tagx="source" title="source">Journal of Experimental Medicine</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">441</span><span class="lpage" tagx="lpage" title="lpage">462</span><span class="pub-id"><a href="https://dx.doi.org/10.1084/jem.92.5.441">10.1084/jem.92.5.441</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14778924">14778924</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broor</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chadha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunasekaran</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCauley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Odagiri</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potdar</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tashiro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Global circulation patterns of seasonal influenza viruses vary with antigenic drift</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">523</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature14460">10.1038/nature14460</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26053121">26053121</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caton</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownlee</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yewdell</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerhard</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">The antigenic structure of the influenza virus A/PR/8/34 hemagglutinin (H1 subtype)</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">417</span><span class="lpage" tagx="lpage" title="lpage">427</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(82)90135-0">10.1016/0092-8674(82)90135-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6186384">6186384</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debbink</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truscon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mantlo</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Vaccination has minimal impact on the intrahost diversity of H3N2 influenza viruses</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006194</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006194">10.1371/journal.ppat.1006194</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28141862">28141862</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinis</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Florek</span><span class="given-names" tagx="given-names" title="given-names">NW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fatola</span><span class="given-names" tagx="given-names" title="given-names">OO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moncla</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mutschler</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlier</span><span class="given-names" tagx="given-names" title="given-names">OK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meece</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belongia</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedrich</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Deep sequencing reveals potential antigenic variants at low frequencies in influenza a virus-infected humans</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">3355</span><span class="lpage" tagx="lpage" title="lpage">3365</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03248-15">10.1128/JVI.03248-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26739054">26739054</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doud</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensley</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Complete mapping of viral escape from neutralizing antibodies</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">e1006271</span><span class="lpage" tagx="lpage" title="lpage">20</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006271">10.1371/journal.ppat.1006271</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28288189">28288189</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flannery</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zimmerman</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubareva</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowalk</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belongia</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLean</span><span class="given-names" tagx="given-names" title="given-names">HQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaglani</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piedra</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mishin</span><span class="given-names" tagx="given-names" title="given-names">VP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chesnokov</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spencer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thaker</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnes</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foust</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sessions</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fry</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Enhanced genetic characterization of influenza a(h3n2) viruses and vaccine effectiveness by genetic group, 2014-2015</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">1010</span><span class="lpage" tagx="lpage" title="lpage">1019</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiw181">10.1093/infdis/jiw181</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27190176">27190176</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foll</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poh</span><span class="given-names" tagx="given-names" title="given-names">YP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renzette</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrer-Admetlla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bank</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shim</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malaspinas</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ewing</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wegmann</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caffrey</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeldovich</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolon</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kowalik</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schiffer</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finberg</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Influenza virus drug resistance: a time-sampled population genetics perspective</span><span class="source" tagx="source" title="source">PLoS Genetics</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004185</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1004185">10.1371/journal.pgen.1004185</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24586206">24586206</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geoghegan</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senior</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Pathogen population bottlenecks and adaptive landscapes: overcoming the barriers to disease emergence</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">283</span><span class="fpage" tagx="fpage" title="fpage">20160727</span><span class="lpage" tagx="lpage" title="lpage">20160729</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2016.0727">10.1098/rspb.2016.0727</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27581875">27581875</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerstung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beisel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rechsteiner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wild</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schraml</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moch</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beerenwinkel</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Reliable detection of subclonal single-nucleotide variants in tumour cell populations</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">811</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1814">10.1038/ncomms1814</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22549840">22549840</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laplante</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePasse</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santos</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepow</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Porter</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stellrecht</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Operario</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griesemer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halpin</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockwell</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spiro</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">St George</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Deep sequencing reveals mixed infection with 2009 pandemic influenza A (H1N1) virus strains and the emergence of oseltamivir resistance</span><span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">203</span><span class="fpage" tagx="fpage" title="fpage">168</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiq040">10.1093/infdis/jiq040</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21288815">21288815</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubareva</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matrosovich</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soo-Hoo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Selection of influenza virus mutants in experimentally infected volunteers treated with oseltamivir</span><span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">183</span><span class="fpage" tagx="fpage" title="fpage">523</span><span class="lpage" tagx="lpage" title="lpage">531</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/318537">10.1086/318537</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11170976">11170976</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herfst</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrauwen</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linster</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chutinimitkul</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Wit</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorrell</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bestebroer</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rimmelzwaan</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Airborne transmission of influenza A/H5N1 virus between ferrets</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">336</span><span class="fpage" tagx="fpage" title="fpage">1534</span><span class="lpage" tagx="lpage" title="lpage">1541</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1213362">10.1126/science.1213362</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723413">22723413</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">RNA virus genomics: a world of possibilities</span><span class="source" tagx="source" title="source">Journal of Clinical Investigation</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">2488</span><span class="lpage" tagx="lpage" title="lpage">2495</span><span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI38050">10.1172/JCI38050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19729846">19729846</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allen</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baguelin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baillie</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newton</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kellam</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murcia</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Transmission of equine influenza virus during an outbreak is characterized by frequent mixed infections and loose transmission bottlenecks</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003081</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003081">10.1371/journal.ppat.1003081</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23308065">23308065</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Illingworth</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustonen</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Identifying selection in the within-host evolution of influenza using viral sequence data</span><span class="source" tagx="source" title="source">PLoS Computational Biology</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003755</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003755">10.1371/journal.pcbi.1003755</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25080215">25080215</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iqbal</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baillie</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warry</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Essen</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Londt</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brookes</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">IH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCauley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Within-host variation of avian influenza viruses</span><span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">364</span><span class="fpage" tagx="fpage" title="fpage">2739</span><span class="lpage" tagx="lpage" title="lpage">2747</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2009.0088">10.1098/rstb.2009.0088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19687042">19687042</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kao</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murcia</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Supersize me: how whole-genome sequencing and big data are transforming epidemiology</span><span class="source" tagx="source" title="source">Trends in Microbiology</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">282</span><span class="lpage" tagx="lpage" title="lpage">291</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2014.02.011">10.1016/j.tim.2014.02.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24661923">24661923</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koel</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bestebroer</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Vliet</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zondag</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vervaet</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skepner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spronken</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eropkin</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rimmelzwaan</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Substitutions near the receptor binding site determine major antigenic change during influenza virus evolution</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">342</span><span class="fpage" tagx="fpage" title="fpage">976</span><span class="lpage" tagx="lpage" title="lpage">979</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1244730">10.1126/science.1244730</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24264991">24264991</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althaus</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Stochastic or deterministic: what is the effective population size of HIV-1?</span><span class="source" tagx="source" title="source">Trends in Microbiology</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">507</span><span class="lpage" tagx="lpage" title="lpage">511</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2006.10.001">10.1016/j.tim.2006.10.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17049239">17049239</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kugelman</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagle</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reyes</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeffer</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Lockhart</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Error baseline rates of five sample preparation methods used to characterize RNA virus populations</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0171333</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0171333">10.1371/journal.pone.0171333</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28182717">28182717</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Predicting antigenic variants of influenza A/H3N2 viruses</span><span class="source" tagx="source" title="source">Emerging Infectious Diseases</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1385</span><span class="lpage" tagx="lpage" title="lpage">1390</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1008.040107">10.3201/eid1008.040107</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496238">15496238</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Measurements of intrahost viral diversity are extremely sensitive to systematic errors in variant calling</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">6884</span><span class="lpage" tagx="lpage" title="lpage">6895</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00667-16">10.1128/JVI.00667-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27194763">27194763</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Genetic bottlenecks in intraspecies virus transmission</span><span class="source" tagx="source" title="source">Current Opinion in Virology</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">20</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2017.10.008">10.1016/j.coviro.2017.10.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29107838">29107838</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCrone</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span><span class="year" tagx="year" title="year">2018</span>[data-title] Host_level_IAV_evolution<span class="source" tagx="source" title="source">Github</span><a href="https://github.com/lauringlab/Host_level_IAV_evolution">https://github.com/lauringlab/Host_level_IAV_evolution</a></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Frequency of acute respiratory illnesses and circulation of respiratory viruses in households with children over 3 surveillance seasons</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">210</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1799</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiu327">10.1093/infdis/jiu327</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24907381">24907381</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murcia</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baillie</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elton</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jervis</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumford</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newton</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoelzer</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dougan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lennard</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ormond</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moule</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitwham</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCauley</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKinley</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Intra- and interhost evolutionary dynamics of equine influenza virus</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">84</span><span class="fpage" tagx="fpage" title="fpage">6943</span><span class="lpage" tagx="lpage" title="lpage">6954</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00112-10">10.1128/JVI.00112-10</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444896">20444896</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">nextflu: real-time tracking of seasonal influenza virus evolution in humans</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">3546</span><span class="lpage" tagx="lpage" title="lpage">3548</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btv381">10.1093/bioinformatics/btv381</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26115986">26115986</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The evolution of epidemic influenza</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">196</span><span class="lpage" tagx="lpage" title="lpage">205</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg2053">10.1038/nrg2053</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17262054">17262054</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griesemer</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sengamalay</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spiro</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volkov</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubenberger</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Stochastic processes are key determinants of short-term evolution in influenza a virus</span><span class="source" tagx="source" title="source">PLoS Pathogens</span><span class="volume" tagx="volume" title="volume">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e125</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0020125">10.1371/journal.ppat.0020125</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17140286">17140286</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowling</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shay</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Influenza vaccine effectiveness in the community and the household</span><span class="source" tagx="source" title="source">Clinical Infectious Diseases</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">1363</span><span class="lpage" tagx="lpage" title="lpage">1369</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/cit060">10.1093/cid/cit060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23413420">23413420</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fry</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Influenza vaccine effectiveness in households with children during the 2012-2013 season: assessments of prior vaccination and serologic susceptibility</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">211</span><span class="fpage" tagx="fpage" title="fpage">1519</span><span class="lpage" tagx="lpage" title="lpage">1528</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiu650">10.1093/infdis/jiu650</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25416812">25416812</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truscon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaron</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martens</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fry</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Substantial Influenza Vaccine Effectiveness in Households With Children During the 2013-2014 Influenza Season, When 2009 Pandemic Influenza A(H1N1) Virus Predominated</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">213</span><span class="fpage" tagx="fpage" title="fpage">1229</span><span class="lpage" tagx="lpage" title="lpage">1236</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiv563">10.1093/infdis/jiv563</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26597255">26597255</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peck</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Connecting within-host dynamics to the rate of viral molecular evolution</span><span class="source" tagx="source" title="source">Virus Evolution</span><span class="volume" tagx="volume" title="volume">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">vev013</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev013">10.1093/ve/vev013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774285">27774285</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Application of an individual-based transmission hazard model for estimation of influenza vaccine effectiveness in a household cohort</span><span class="source" tagx="source" title="source">American Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">186</span><span class="fpage" tagx="fpage" title="fpage">1380</span><span class="lpage" tagx="lpage" title="lpage">1388</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aje/kwx217">10.1093/aje/kwx217</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28605429">28605429</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrie</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohmit</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowling</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cross</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malosh</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Influenza Transmission in a Cohort of Households with Children: 2010-2011</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e75339</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0075339">10.1371/journal.pone.0075339</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086511">24086511</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrova</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The evolution of seasonal influenza viruses</span><span class="source" tagx="source" title="source">Nature reviews. Microbiology</span><span class="volume" tagx="volume" title="volume">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">517</span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenfeld</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebra</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halpin</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twaddle</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePasse</span><span class="given-names" tagx="given-names" title="given-names">JV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockwell</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowling</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Quantifying influenza virus diversity and transmission in humans</span><span class="source" tagx="source" title="source">Nature Genetics</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">195</span><span class="lpage" tagx="lpage" title="lpage">200</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.3479">10.1038/ng.3479</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26727660">26727660</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">FastTree 2--approximately maximum-likelihood trees for large alignments</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e9490</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubenberger</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The genomic and epidemiological dynamics of human influenza A virus</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">453</span><span class="fpage" tagx="fpage" title="fpage">615</span><span class="lpage" tagx="lpage" title="lpage">619</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature06945">10.1038/nature06945</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18418375">18418375</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rogers</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sebra</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelin</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twaddle</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boivin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Intrahost dynamics of antiviral resistance in influenza A virus reflect complex patterns of segment linkage, reassortment, and natural selection</span><span class="source" tagx="source" title="source">mBio</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e02464-14</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.02464-14">10.1128/mBio.02464-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25852163">25852163</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fonville</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herfst</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Boheemen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linster</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schrauwen</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosterín</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiken</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maher</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The potential for respiratory droplet-transmissible A/H5N1 influenza virus to evolve in a mammalian host</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">336</span><span class="fpage" tagx="fpage" title="fpage">1541</span><span class="lpage" tagx="lpage" title="lpage">1547</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1222526">10.1126/science.1222526</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723414">22723414</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lapedes</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bestebroer</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rimmelzwaan</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Mapping the antigenic and genetic evolution of influenza virus</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">305</span><span class="fpage" tagx="fpage" title="fpage">371</span><span class="lpage" tagx="lpage" title="lpage">376</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1097211">10.1126/science.1097211</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15218094">15218094</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobel Leonard</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McClain</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halpin</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ransier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockwell</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambkin-Williams</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginsburg</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woods</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Deep sequencing of influenza a virus from a human challenge study reveals a selective bottleneck and only limited intrahost genetic diversification</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">11247</span><span class="lpage" tagx="lpage" title="lpage">11258</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01657-16">10.1128/JVI.01657-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27707932">27707932</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobel Leonard</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weissman</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenbaum</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghedin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Transmission bottleneck size estimation from pathogen deep-sequencing data, with an application to human influenza a virus</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">91</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e00171-17</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00171-17">10.1128/JVI.00171-17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28468874">28468874</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varble</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albrecht</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Backes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crumiller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouvier</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sachs</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Sastre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">tenOever</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Influenza A virus transmission bottlenecks are defined by infection route and recipient host</span><span class="source" tagx="source" title="source">Cell Host &amp;amp; Microbe</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">691</span><span class="lpage" tagx="lpage" title="lpage">700</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2014.09.020">10.1016/j.chom.2014.09.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25456074">25456074</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiley</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">Structural identification of the antibody-binding sites of Hong Kong influenza haemagglutinin and their involvement in antigenic variation</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">289</span><span class="fpage" tagx="fpage" title="fpage">373</span><span class="lpage" tagx="lpage" title="lpage">378</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/289373a0">10.1038/289373a0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6162101">6162101</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilker</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinis</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Starrett</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatta</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Connor</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedrich</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Selection on haemagglutinin imposes a bottleneck during mammalian transmission of reassortant H5N1 influenza viruses</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2636</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms3636">10.1038/ncomms3636</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24149915">24149915</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekiert</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hai</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowe</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">IA</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Structural basis of preexisting immunity to the 2009 H1N1 pandemic influenza virus</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">328</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">360</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1186430">10.1126/science.1186430</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20339031">20339031</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens-Ayers</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Englund</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pergam</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boeckh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Parallel evolution of influenza across multiple spatiotemporal scales</span><span class="source" tagx="source" title="source">eLife</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e26875</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.26875">10.7554/eLife.26875</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28653624">28653624</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donnelly</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scholes</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">St George</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatta</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wentworth</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Single-reaction genomic amplification accelerates sequencing and vaccine production for classical and Swine origin human influenza a viruses</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">10309</span><span class="lpage" tagx="lpage" title="lpage">10313</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01109-09">10.1128/JVI.01109-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19605485">19605485</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zwart</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elena</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Matters of size: genetic bottlenecks in virus infection and their potential impact on evolution</span><span class="source" tagx="source" title="source">Annual Review of Virology</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="lpage" tagx="lpage" title="lpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-100114-055135">10.1146/annurev-virology-100114-055135</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26958911">26958911</a></span></span></li> </ul> </div> </div>  </body></html>