<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/10/05/436600.atom" />
<meta name="HW.pisa" content="biorxiv;436600v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer" />
<meta name="DC.Identifier" content="10.1101/436600" />
<meta name="DC.Date" content="2018-10-05" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples." />
<meta name="DC.Contributor" content="Wanda J. Lyon" />
<meta name="DC.Contributor" content="Zachary K. Smith" />
<meta name="DC.Contributor" content="Brian Grier" />
<meta name="DC.Contributor" content="James Baldwin" />
<meta name="DC.Contributor" content="Clarise R. Starr" />
<meta name="article:published_time" content="2018-10-05" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;436600v1" />
<meta name="citation_id" content="436600v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/436600v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/436600v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/436600v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600.full.pdf" />
<meta name="citation_doi" content="10.1101/436600" />
<meta name="citation_num_pages" content="26" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="436600" />
<meta name="citation_author" content="Wanda J. Lyon" />
<meta name="citation_author_institution" content="711th Human Performance Wing, RHXB, Wright Patterson Air Force Base OH" />
<meta name="citation_author" content="Zachary K. Smith" />
<meta name="citation_author_institution" content="711th Human Performance Wing, RHXB, Wright Patterson Air Force Base OH" />
<meta name="citation_author" content="Brian Grier" />
<meta name="citation_author_institution" content="711th Human Performance Wing, RHXB, Wright Patterson Air Force Base OH" />
<meta name="citation_author" content="James Baldwin" />
<meta name="citation_author_institution" content="711th Human Performance Wing, USAFSAM, Wright Patterson Air Force Base" />
<meta name="citation_author" content="Clarise R. Starr" />
<meta name="citation_author_institution" content="711th Human Performance Wing, USAFSAM, Wright Patterson Air Force Base" />
<meta name="citation_reference" content="Baldwin JC, et al. 2010. Upper respiratory virus serotype panel for the pyrosequencer. Wright-Patterson AFB (OH): U.S. Air Force School of Aerospace Medicine; Technical Report AFRL-SA-WP-TR-2012-0002." />
<meta name="citation_reference" content="BaoY, et al. The influenza virus resource at National Center for Biotechnology Information J.Virol 2008; 82(2):596–601." />
<meta name="citation_reference" content="Bertelli, and Greub G. 2013. Rapid bacterial genome sequencing: methods and applications in clinical microbiology. Clin Microbiol Infect. 19(9):803–813." />
<meta name="citation_reference" content="Briese T, et.al. 2009. Genetic detection and characterization of Lujo virus, a new hemorrhagic fever-associated arenavirus from southern Africa. PLoS Pathog. 5(5):e1000455." />
<meta name="citation_reference" content="Brown B, Watson M, Minot S, Rivera M, and Franklin R. 2017. MinION nanopore sequencing of environmental metagenome; a synthetic approach. GiaScience, 1–10 6:" />
<meta name="citation_reference" content="Brown JR, et al. 2015. Astrovirus VA1/HMO-C: an increasingly recognized neurotropic pathogen in immunocompromised patients. Clin Infect Dis. 60(6):881–888." />
<meta name="citation_reference" content="Chan JZ, Pallen MJ, Oppenheim B, and Constantinidou C. 2012. Genome sequencing in clinical microbiology. Nat Biotechnol. 30(11):1068–1071." />
<meta name="citation_reference" content="Deamer D, Akeson M, and Branton D. 2016. Three decades of nanopore sequencing. Nat Biotechnol. 34(5):518–524." />
<meta name="citation_reference" content="De Coster W, Hert SD, Schultz T, Cruts M and Van Broeckhoven C. 2018. Nanopack: visualizing and processing long-read sequencing data. Bioinformatic 34(15):2666–2669." />
<meta name="citation_reference" content="Gire SK, et al. 2014. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science. 345(6202):1369–1372." />
<meta name="citation_reference" content="Greninger AL et al. 2015. Rapid metagenomics identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis. Genom Med, 7:99–112." />
<meta name="citation_reference" content="Hoke CH Jr, Hawksworth A, and Snyder CE Jr. 2012. Initial assessment of impact of adenovirus type 4 and type 7 vaccine on febrile respiratory illness and virus transmission in military basic trainees, March 2012. MSMR. 19(3):2–4." />
<meta name="citation_reference" content="Imamura T, et al. 2011. Enterovirus 68 among children with severe acute respiratory infection, the Philippines. Emerg Infect Dis. 17(8):1430–1435." />
<meta name="citation_reference" content="Forsyth BR, Bloom HH, Johnson KM, and Chanock RM. 1963. Patterns of illness in rhinovirus infections of military personnel. N Engl J Med. 269:602–606." />
<meta name="citation_reference" content="Gray GC, et al. 1999. Respiratory diseases among U.S. military personnel: countering emerging threats. Emerg Infect Dis. 5(3):379–385." />
<meta name="citation_reference" content="Ghedin E, et al. 2005 Large-scale sequencing of human influenza reveals the dynamic nature of viral genome evolution. Nature. 437 (7062):1162–1166." />
<meta name="citation_reference" content="Johns MC, 2009. Preliminary report: febrile acute respiratory disease caused by adenovirus type 14, U.S. Coast Guard Training Center, Cape May New Jersey, March 2009. MSMR. 16(5):10–11." />
<meta name="citation_reference" content="Juul S, et al. 2015. What’s in my pot? Real-time species identification on the MinION. http://biorxiv.org/content/early/2015/11/06/030742" />
<meta name="citation_reference" content="Kaida A, et al. 2011. Enterovirus 68 in children with acute respiratory tract infections, Osaka, Japan. Emerg Infect Dis. 2011; 17(8):1494–1497." />
<meta name="citation_reference" content="Kilianski A, et.al. 2015. Bacterial and viral identification and differentiation by amplicon sequencing on the MinION nanopore sequencer. Gigascience. 4:12." />
<meta name="citation_reference" content="Li H., et al. 2009. 1000 Genome Project Data Processing Subgroup (2009) The Sequence alignment/map (SAM) format and SAMtools. Bioinformatics, 25, 2078–2079." />
<meta name="citation_reference" content="Li H. and Durbin R. 2010. Fast and accurate long-read alignment with Burrows-Wheeler Transform. Bioinformatics, Epub. [PMID: 20080505]" />
<meta name="citation_reference" content="Li L, et al. 2015. Comparing viral metagenomics methods using a highly multiplexed human viral pathogens reagent. J Virol Methods. 213:139–146." />
<meta name="citation_reference" content="Lim ML, Gabele SE, Wallace MR, Gray GC, and Earhart KC. 2004. Upper respiratory tract infections (URI). Mil Med. 169(4):15–16." />
<meta name="citation_reference" content="Loman NJ and Quinlan J. 2014. Poretools: a toolkit for analyzing nanopore sequence data Bioinformatics 1:30(23)3399–3401." />
<meta name="citation_reference" content="Loman NJ, Quick J, Simpson JT. 2015. A complete bacterial genome assembled de novo using only nanopore sequencing data. Nat Methods. 12(8):733–735." />
<meta name="citation_reference" content="Lu J, Breitwieser FP, Thielen P, Salzberg SL. 2017. Bracken: estimating species abundance in metagenomics data. Peer J Computer Science 3:e104, doi:10.7717/peerj-cs.104" />
<meta name="citation_reference" content="Madoui MA, et al. 2015. Genome assembly using Nanopore-guided long and error-free DNA reads. BMC Genomics. 16:327." />
<meta name="citation_reference" content="Miller RR, Montoya V, Gardy JL, Patrick DM, Tang P. 2013. Metagenomics for pathogen detection in public health. Genome Med. 5(9):81." />
<meta name="citation_reference" content="Naccache SN et al. 2015. Diagnosis of neuroinvasive astrovirus infection in an immunocompromised adult with encephalitis by unbiased next-generation sequencing. Clin Infect Dis. 60 (6):919–923." />
<meta name="citation_reference" content="Nakamura S et al. 2009. Direct metagenomic detection of viral pathogens in nasal and fecal specimens using an unbiased high-throughput sequencing approach. PLoS One. 4(1):e4219." />
<meta name="citation_reference" content="Ondov BD, Bergman NH, and Phillippy AM. 2011. Interactive metagenomics visualization in web browser. BMC Bioinformatics 30: 12(1):385" />
<meta name="citation_reference" content="’Shea MK, Pipkin C, Cane PA, and Gray GC. 2007. Respiratory syncytial virus: an important cause of acute respiratory illness among young adults undergoing military training. Influenza Other Respir Viruses. 1 (5-6):193–197." />
<meta name="citation_reference" content="Shin J et al. 2016. Analysis of mouse gut microbiome using full-length 16S rRNA amplicon sequencing. Sci Rep. 2016; 6:29681." />
<meta name="citation_reference" content="Silva CS et al. 2014. Human respiratory coronaviruses detected in patients with influenza-like illness in Arkansas, USA. Virol Mycol. S2. doi:10.4172/2161-0517.S2-004." />
<meta name="citation_reference" content="Van Boheemen S et al. 2012. Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans. MBio. 3(6):e00473–12." />
<meta name="citation_reference" content="Van Den Hoecke S, Verhelst J, Vuylsteke M, and Saelens X. 2015. Analysis of the genetic diversity of influenza A viruses using next-generation DNA sequencing. BMC Genomics. 16:79" />
<meta name="citation_reference" content="Vernikos G, Medini D, Riley DR, Tettelin H. 2015. Ten years of pan-genome analyses. Curr Opin Microbiol. 23:148–154." />
<meta name="citation_reference" content="Quast C, et al. The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools Nucl. Acids Res. 41 (D1): D590–D596." />
<meta name="citation_reference" content="Quick J, Quinlan AR, and Loman NJ. 2014. A reference bacterial genome dataset generated on the MinION™ portable single-molecule nanopore sequencer. Gigascience. 3:22." />
<meta name="citation_reference" content="Quiñones-Mateu ME, Avila S, Reyes-Teran G, and Martinez MA. 2014. Deep sequencing: becoming a critical tool in clinical virology. J Clin Virol. 61(1):9–19." />
<meta name="citation_reference" content="Wood DE, and Salzberg SL 2014. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biology 15:R46." />
<meta name="citation_reference" content="Yu H. 2013. Human infection with avian influenza A H7N9 virus: an assessment of clinical severity. Lancet. 382(9887):138–145." />
<meta name="twitter:title" content="Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples." />
<meta name="og-title" property="og:title" content="Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/436600v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-10-05" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/436600v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/436600v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/436600v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/436600v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/131107" />
    <title>Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JFIICtCxaKFOPPL_2MtYgHTYAO0XzCtHvwSFd5o-vRA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"131107","apath":"\/biorxiv\/early\/2018\/10\/05\/436600.atom","pisa":"biorxiv;436600v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;436600\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;436600\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;436600v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;436600\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;436600\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/436600v1","title":"Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer","identifier":"node\/131107"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nuehAjW0+DQmwSoJ1GtdbciAanFDKe4hHVEHZyq8KQ5IoYoMFGauAfmVhKSJ1eRDkWOVjgurP9hEtG6tbY\\\\\\\/G+QcLdcGoFXhk7ysYE9RZeAItawtytnSFeHVvQbq43rk+x\\\\\\\/yi820Z6\\\\\\\/1VgRYlA3XvUgepiXBXlZje\\\\\\\/D2wASw+a9CiAGhsVdYlej3O0wsSlnFEwsCjsBdad5xujh5JjtWBPydlYhHjFo6hgb5b2pckTXSFJ2eEOrqSzucRM7V6DBoaIKVDelXqAaUscY2L\\\\\\\/wZaG57CLeZ3r5HorEkAp9FsGvVwNpT4nQJf1hnHLYKxtq3qT7p2KH6pjGO\\\\\\\/\\\\\\\/hcPVaaMWBbiarm2qDqKns+o6\\\\\\\/9DiqEI92YtLVKQgxbVPlPQU6TWHvkOI0B95skw1dCIB7a\\\\\\\/GohhK87MqHP2ySxdRXOFqj4+3vekZedES0rqZbLUTnHtFO+6cBtD8pwsbl\\\\\\\/cyVX7S5O6dsj0KxfMvj0Ab2OBNYa5RsGkCV5Ox4p4zYs0aOYMILvOP9pxAAdtMhEMOxqrPN8xr4V+cyniPcgK8ROIByVFmRZHCLhj0WnXwISgsPX21XxnqasteNneM6soOrJvAZerV5E\\\\\\\/on2UpvE\\\\\\\/s2KiVIkGCTjGi3ZUJW8ltKnvHCxD9Pl9OlTDGaSUSXcwt6HwyHDLwtgHXGu\\\\\\\/9fiEweGdYEGVH34zo5pwyhAa1VGxOHJ0gR37ACaXfzAOrKmrKqVD1u2WHc6K\\\\\\\/09ULic\\\\\\\/KT6nXuWXooixbhZ55580dlJsVg+7idbCFywdK+H7VQ\\\\\\\/cHVHXvyqwo386X9xbKCIjAxzqMUIWUcdjyIvQsUSn9VmbJD\\\\\\\/MHHHWoqgXyXJgjCXc9K+rg9p7gC4awnAkeuqx1hCBUMo3fbM99F1GB9r5dRc5LYncJEhrXY31K7YUX2+JBbtTq8CPy8zDhg4uiX3sk3LTCcKTGXCMmxEVHO5BQyYC8d4LskvaHHM0qTiubSo3hrDcVKNmEBIkl3rUW5EEKQF1lyi14ctmi4onvqaY4UjCg1\\\\\\\/ojaHeKL60BB4GfN2e2UjOQ4\\\\\\\/RA56B+7WwTQlX0MlmmrClC4CG3Dr7yygLVSII4eyF3ThM6GOy05Ah8unWfCY5TxU9IKEZKwqmf0QQUbNjxrfW3Ax8d4n92Cmd5Hig1bKaaNekPOqzyxhdIAWzld2BzE1WiloMAJr32OZTQaJErXHCvwZyLCHYGRA4F2KLDuA9Z2nK8a0xoq4RL6HSPn6ehfcPLLq6WbDV6itu6fgRlu+tsB1n5bg1iqLiK\\\\\\\/4xgkInkuxPTFTn1r1joBnU7wzl3NIiVBk+DJGVNr\\\\\\\/ATCsl6B8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022pZ2reuvdrHfPjuTzGg5vGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227f525c6e44ef971b873723f6d16a6ad898c1e957bf453bcd16e482f4be6f113e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/436600v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-131107 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/436600v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7Hx7DGtMQc7m6g8DRSmbmwbvE50hbtIclNFmH3p-pmQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="131107" id="node131107" data-pisa="biorxiv;436600v1" data-pisa-master="biorxiv;436600" data-apath="/biorxiv/early/2018/10/05/436600.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wanda J.</span> <span class="nlm-surname">Lyon</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zachary K.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Grier</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">James</span> <span class="nlm-surname">Baldwin</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Clarise R.</span> <span class="nlm-surname">Starr</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/436600 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node131107" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wanda J. Lyon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-sup'>th</span>, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wanda%2BJ.%2BLyon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lyon%20WJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWanda%2BJ.%2BLyon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zachary K. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-sup'>th</span>, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BK.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20ZK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BK.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brian Grier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-sup'>th</span>, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BGrier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grier%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BGrier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James Baldwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-sup'>th</span>, OH, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BBaldwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baldwin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BBaldwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clarise R. Starr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-sup'>th</span>, OH, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clarise%2BR.%2BStarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Starr%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClarise%2BR.%2BStarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/436600v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/436600v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/436600v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/436600v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/436600v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/436600v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:131107" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:131107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification (WGA), targeted sequencing, and was found to have tremendous potential for field use. The MinION nanopore sequencer was been released to community testers for evaluation using a variety of sequencing applications. The MinION was used to evaluate upper respiratory disease infections using both whole genome amplification and targeted sequencing, and was found to have tremendous potential for field use. In this study, we tested the ability of the MinION nanopore sequencer to accurately identify and differentiate clinical bacterial and viral samples via targeted sequencing and whole genome sequencing. The current nanopore technology has limitations with respect to error rate but has steadily improved with development of new flow cells and kits. Upper respiratory disease organisms were successfully identified and differentiated down to the strain level with 87-98% alignment to our reference genome database. The ability to differentiate strains by amplicon and whole genome sequencing on the MinION was accomplished despite the observed average per 100-base error rate averaged 1.2E-01. This study offers evidence of the utility of sequencing to identify and differentiate both viral and bacterial species present within clinical samples.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Acute respiratory infections (ARI) have been a significant source of disease and nonbattle injury among military forces for centuries. Although mortality from ARIs is low in military populations, the impact of these infections on military readiness in terms of lost work-hours and convalescence is high. Furthermore, novel ARIs have been a periodic source of great morbidity and mortality, such as during the 1918 A/H1N1 (“Spanish”) influenza epidemic, the 2002-2003 severe acute respiratory syndrome (SARS) epidemic in Asia and North America, and the 2009-2010 swine-origin A/H1N1 global pandemic. Periodic outbreaks of adenovirus and similar infections have plagued military training centers in recent years, control of which has been hampered until very recently by the unavailability of an active vaccine [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. Based on historical information dating back to the 1918 influenza pandemic, ARI has also had a major impact on military training and readiness. Supporting the notion that viral infections play a major role in training and readiness, other studies have demonstrated rates of ARI in 80% of trainees, with rates of hospitalization of 20% prior to influenza and adenovirus vaccination programs [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Despite aggressive vaccination programs, ARI has been noted to be the cause of 70% of U.S. Air Force pilot groundings and led to sick call evaluations in 1.5% of all military members seeking care during Operation Desert Storm [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Nonvaccine preventable causes of ARI in military members include respiratory syncytial virus, enteroviruses, and rhinoviruses [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. Although influenza is usually the pathogen that captures the attention of the world, other respiratory pathogens have recently garnered the attention of the military. Adenoviruses cause a wide array of symptoms, from conjunctivitis, gastroenteritis, and mild respiratory infections to life-threatening pneumonias in young adults and blood stream and brain infections in immunocompromised hosts. Enterovirus 68 has recently shown severe respiratory illness in other countries such as the Philippines, the Netherlands, and Japan [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], and the Middle East respiratory syndrome-coronavirus (MERS-CoV) outbreak in the Middle East [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>] along with the H7N9 influenza outbreak in China [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] has put focus on new and emerging pathogens that may be introduced into immunologically naïve populations or recombined into novel forms through human-human or human-animal transmission.</p><p id="p-4">Next-generation sequencing (NGS) technologies are now capable of providing a whole genome sequence or metagenomics for a wide range of organisms [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. NGS is being applied with increasing frequency in clinical microbiology laboratories to detect and characterize pathogens, thereby replacing the currently used Sanger sequencing methods [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>]. Sanger sequencing has been a reliable and robust method that has served clinical laboratories well for over several decades; however, it is labor intensive, slow, and not easily adapted for processing large genomes or large samples sets.</p><p id="p-5">Oxford Nanopore’s MinION may therefore provide us with new opportunities to track infectious disease, for example, rapid sequencing of viral genomes in response to the early phase of influenza pandemics, or for determination of coronavirus virus genotypes during outbreaks. In addition, sequencing will allow for rapid sequencing of viral genomes and provides an opportunity to gain insight into viral genetic drift (single nucleotide polymorphism), emergence of new strains such as SARS-like coronavirus or MERS-CoV, and transmission. To date, a limited number of pathogenic viral studies have been done with the MinION, and all of the published results have focused on sequencing DNA rather than RNA, which is the focus of this effort [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Metagenomic NGS is particularly attractive for surveillance of febrile illness because the approach can broadly detect viruses, bacteria, and parasites in clinical samples [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Although currently limited by a sample-to-answer turnaround time of &gt;24 hours for benchtop sequencers, we report in this study that unbiased pathogen detection using both targeted and WGA NGS which can be realized with actionable results for clinical diagnostics [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] and public health [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] within 10 h.</p><p id="p-6">The characterization of upper respiratory disease (URD) viruses would certainly benefit the use of whole genome sequencing (WGS) and targeted sequencing, providing us with tools for surveillance of highly diverse viral genomes [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>]. Such data can assist in vaccine development, detection of anti-viral drug resistance, or the identification of new emerging re-assorted viral genomes. These advances may help to mitigate the morbidity and mortality of influenza pandemics or seasonal viral epidemics. NGS platforms such as the Illumina MiSeq and Life Technologies Ion Torrent have been extensively used to provide WGS data for viruses, potentially within 2-3 days of the receipt of a sample [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]. Furthermore, although the cost of NGS is decreasing, however it is still cost prohibitive unless large numbers of samples are processed in a multiplexing format. Therefore, there is clearly a place for a portable NGS instrument that can be deployed into the field for surveillance using available field ready kits and sequencing reagents that are comparable to cost of benchtop sequencer. Analysis of sequencing data is accomplished in real time by an internet-connected laptop. Another unique feature of the MinION is its ability to generate very long reads of up to 60 kilobases (kb) [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. The MinION has been successfully used for WGS and targeted sequencing [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>].</p><p id="p-7">The purpose of this study is to examine the efficacy of using the MinION to sequence URD organisms present in clinical nasal washes using targeted amplification, WGA and whole transcriptome amplification (WTA) of low titer URD samples.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Nucleic acid extraction</h3><p id="p-8">Influenza A virus H1N1 (VR1736), parainfluenza virus 1-3 (VR94, VR92, and VR93), coronavirus 229E and OC43 isolates (VR-740 and VR1558), <em>Mycoplasma pneumoniae</em> (ATCC 29342), <em>Haemophilus influenzae</em> virus (51907), and human adenovirus (VR-1603) were obtained from American Type Culture Collection (ATCC) and used as control nucleic acid (NA) for library preparations. RNA/DNA were extracted from 200 μL viral culture supernatants or from clinical nasal washes using the Maxwell total viral nucleic acid kits (Promega), with elution into 50 μL of RT-PCR molecular grade water (Ambion). De-identified positive nasal washes were obtained from clinical laboratory (USAFSAM) included influenza A virus, influenza B virus, parainfluenza viruses 1-3, adenovirus, and coronavirus OC043.</p></div><div id="sec-4" class="subsection"><h3>Targeted RT-PCR and PCR</h3><p id="p-9">Standard targeted PCR was done per using primers listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and the SuperScript III OneStep RT-PCR system (Thermo Fisher Scientific) per the manufacturer’s instructions. Briefly, 10 ng of DNA-free RNA or RNA-free DNA was used for each 50-μL reaction. All PCR reactions were accomplished with 1.0 ul of each primer at 10 uM. RT-PCR thermocycling parameters were as follows: 30 minutes at 50°C, 2 minutes at 95°C, and then 35 cycles of 30 seconds at 95°C, 30 seconds at 52°C, and 75 seconds at 68°C, followed with a final extension at 68°C for 5 minutes. PCR thermocycling was accomplished with AccuStartII PCR Supermix (Quanta Biosciences) with following parameters: 3 minutes at 95°C, and then 25 cycles of 30 seconds at 94°C, 30 seconds at 55°C, and 90 seconds at 72°C, followed with a final extension at 68°C for 3 minutes. DNA or cDNA were cleaned up using 1.5X Agencourt Ampure XP beads as described by the manufacturer (Beckman Coulter). DNA and cDNA were quantified using a Qubit 3.0 fluorometer (Life Technologies) and resuspended in Low TE (Invitrogen).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/609886/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/609886/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/609886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Primers used for targeted RT-PCR</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>WGS preparation</h3><p id="p-10">The Oxford Nanopore Technologies (ONT) MinION Genomic DNA Sequencing kit was used to prepare libraries from WGA that previously amplified using REPLI-g WTA/WGA kits (Qiagen) as described by the manufacturer. WTA was used on clinical nasal washes with low viral titers. RT-PCR or PCR was accomplished as described previously. Briefly, 10 ng of DNA-free RNA or RNA-free DNA was used for each WTA/WGA reaction. Whole genome products were sheared with a Covaris g-tube (Woburn, MA) per the instructions of the manufacturer with fragmentation sizing to approximately 5 kb and the sheared DNA was used in library preparations.</p></div><div id="sec-6" class="subsection"><h3>MinION library preparation</h3><p id="p-11">This work was completed as part of the ONT MinION early access program. During this time the flow cells, flow cell chemistries and library kits changed numerous times during the two-year study. The ONT MinION Sequencing kit that was used to prepare libraries from WGA/WTA are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. DNA or cDNA libraries were sequenced with various flow cells throughout the study including versions R7, R7.3, and Spot-ON 9.4. The subsequent increase in flow cell number indicates improvements in their flow cells, with Spot-On R9.4 flow cell being the latest improvement. In addition, the ONT MinION library reagents were changed as the kit improvements were made during this study (Kit: SQK-MAP0005 for the R7.03 flow cell and Kits: SQK-LSK- 209, SQK- RAD 004 (Rapid) for R9.4 or Spot-ON R9.4 flow cells). Barcoding of the DNA fragments was completed using barcoding kit (EXP-PBC001). Libraries kit used for each of the sequencing libraries are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a>, as well as the amount of starting material and flow cells type. Library preparations were done as described by the manufacturer and were quantified using a Qubit 3.0 fluorometer (Life Technologies).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/609928/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/609928/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/609928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Targeted Sequencing of URD samples using the MiSeq and MinION</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/609794/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/609794/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/609794" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <span class="caption-title">MinION WTA and WGA with Targeted RT-PCR or PCR Amplification</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">In each case, the cDNA/DNA was end-repaired in a 100-μL reaction containing 0.5-2 μg of nucleic acid (depending on the run) in 80 μL water, 10 μL of 10X end-repair buffer (from the NEB Next End Repair Module), and 1 μL of enzyme mix. DNA and cDNA were quantified using a Qubit 3.0 fluorometer (Life Technologies) and resuspended in molecular grade water as the starting material for library preparation per the instruction of the manufacturer. End-repair was performed per the manufacturer’s instructions with the exception that the incubation time was increased to 15 minutes. The resulting blunt-ended DNA was cleaned up using 1.0 × volume of AMPure XP beads (Beckman Coulter) according to the manufacturer’s instructions with the exception that 80% ethanol was used instead of 70%, and the DNA was eluted in 30 μL molecular grade water. Ten μL of adapter mix and 2 μL hairpin (HP) adapter were added to the 30 μL dA-tailed DNA, then 8 μL of molecular grade water followed by the addition of 50 μL Blunt/TA ligase master mix (NEB). The contents were briefly mixed and the reaction was left to proceed at room temperature for 15 minutes. One μL of HP tether was added, mixed briefly, and incubated 15 minutes at room temperature. The sample was cleaned up using 1.0 × by volume AMPure XP beads according to the manufacturer’s instructions with the exception that the DNA was eluted into 25 μL of elution library buffer at 37°C for 10 minutes. The beads were removed by placing the tube into a 1.5-mL magnet and the supernatant was transferred to a low binding 1.5-mL Eppendorf tube and placed on ice until the sequencing flow cell was loaded. Each library was quantified using the Qubit fluorometer, and the total library was within the required range of 150-200 ng prior to proceeding to sequencing.</p></div><div id="sec-7" class="subsection"><h3>MinION sequencing</h3><p id="p-13">A 48-hour sequencing protocol was initiated using the MinION control software, MinKNOW versions 1.1-1.3.24, for flow cells 7.3 and MinKNOW versions 1.4.6.-2.4.5 for R9 flow cells, respectively. Read event data were base-called by the Metrichor agent (version 0.46.1.9, updated to EPI2ME) using workflows WIMP (What In My Pot) 1.2.2 rev 1.5 with appropriate 2D and 1D workflow scripts [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>].</p></div><div id="sec-8" class="subsection"><h3>Illumina MiSeq sequencing</h3><p id="p-14">The targeted RT-PCR amplification was accomplished as described earlier using the same RNA as described previously. The sequencing libraries were prepared using either the Illumina TruSeq Nano or Nextera XT library kits (Illumina) with150-bp paired-end sequencing on the Illumina MiSeq.</p></div><div id="sec-9" class="subsection"><h3>Sequence data analysis</h3><p id="p-15">Read data were extracted from the native HDF5 format (Fast 5) and converted into FASTA using poretools [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] and sequence reads were assess for quality and trimmed with Nanofilt [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] Quality reads with a q-score lower than 7 were omitted. Illumina reads were trimmed with Trimmomatic 0.36 using the following settings: trim reads: Leading: 3, Trailing: 3, and Sliding Window: 4:15. Taxonomic identification was used to verified for each data set by using bioinformatics packages Burrows-Wheeler Alignment MEM (BWA; aligned to reference and Human GR2) and SAM Tools was used to extract statistics and error rate per 100 bases [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Sequence reads were filtered for any human sequence contributions prior to visualization. <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">3</a> listed the analysis tools used for each data set. Kracken and Bracken bioinformatics packages with a k-mer database of viral and bacterial genomes [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>]. All genomes, viral and bacterial were downloaded from NCBI Ref Seq (May, 2015) and were used to build the Kraken database (version 0.10.4) with k=31. Bracken was used for reassign alignments to the species level and visualization was accomplished with Krona [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] using the highest of number of hits (lowest e-value) or Metrichor’s WIMP. Viral and bacterial databases were obtained from NCBI viral genomes resource [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Read percentage identity is defined as 100 * matches/ (matches + deletions + insertions + mismatches). Fraction of reads aligned are defined as (alignment length + insertions - deletions)/ (alignment length + unaligned length - deletions + insertions) [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>].</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Targeted amplicon sequencing using the ONT MinION</h3><p id="p-16">A panel of URD viruses (influenza A virus, influenza B virus, parainfluenza virus 1, parainfluenza virus 2, parainfluenza virus 3, human adenovirus D, coronavirus OC043 and bacterial strains (<em>M. pneumoniae</em> and <em>H. influenzae</em>) were used in this study. The same nucleic acid from each sample was used to generate libraries for both MinION and Miseq. The MiSeq was used verify the Metrichor WIMP identification. All of the organisms listed above were successfully sequenced on the MinION and identified correctly using the WIMP application, uses a databank of reference sequences for viruses, bacteria, and fungi. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows an a representative example of the data analysis for an influenza A H1N1 isolate listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. All viruses and bacteria listed in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">3</a> were correctly identified using WIMP. In addition, BWA and SAM Tools was used to verify that the alignment for each of the isolates listed in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Tables 2</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Target amplification of influenza A virus using the H2N2 primers listed in Table 1 Metrichor WIMP application was used for identification of the Influenza A H3N2 virus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1995865064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Target amplification of influenza A virus using the H2N2 primers listed in Table 1&lt;/span&gt; Metrichor WIMP application was used for identification of the Influenza A H3N2 virus.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Target amplification of influenza A virus using the H2N2 primers listed in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a></span><p id="p-17" class="first-child">Metrichor WIMP application was used for identification of the Influenza A H3N2 virus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Fast 5 files generated by MinKNOW were converted to FASTQs using Poretools then the Fastq files were aligned with Bracken. For example shown in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, the Influenza A analysis had 114 high quality sequence reads and 104 reads (87.3%) were classified as Influenza A (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-19">In addition, MinION sequence identification was further verified by using the same NA extractions for libarary preparation for the Illumina’s Miseq. BWA MEM alignment of the reads were in agreement with Metrichor’s identification for each of the nasal wash samples listed in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>. MinION sequence data for viruses Influenza A H1N1, Influenza B, Parainfluenza 1(PAIV1) and Coronavirus OC043 had a percent identity of 82.6, 98.7, 18.4 and 92.4, respectively (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). Interestingly, there was little difference between the sequencing data quality in terms of alignment error. Influenza A H1N1, Influenza B, Parainfluenza 1(PAIV1) and Coronavirus OC043 had alignment error of 1.15E-01, 1.54E-01, 2.74E-02 and 1.37E-01, respectively (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). The Nanopore kits and flow cells used early in this study perform well in regards to identifying the isolates; however, the number of reads that passed were lower than expected probably due improper amplicons to adapter ratios used in the ligation step.</p></div><div id="sec-12" class="subsection"><h3>MinION Sequencing of WGA /WTA amplified NA and targeted PCR Libraries</h3><p id="p-20">WTA was used with NA extracted from low titer clinical nasal washes and the isolated RNA (1.0 ng/ul) or DNA (1.0 ng/ul) was amplified with Qiagen’s WTA or WGA kits and the resulting amplification products were used as starting material for RT-PCR or PCR prior to library preparations. The individual libraries were barcoded and mixed in equal molar concentrations prior to being loaded onto a R9.4 flow cell. The percent-alignment to reference genomes ranged from 92-96% (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-21">There was minimal sequencing bias seen within the barcoded library pool (data not shown); however, the number of reads were somewhat different among the tested organisms and probably due to an error in quantification of the input library prior to pooling. Future barcoding pooling protocols should include a Tapestation (Agilent) quantification in terms of fragment length, which can be used to calculate of the nM concentration; nevertheless, this experiment demonstrated that the use of Qubit quantifications is sufficient for rapid pooling of multiplex libraries.</p><p id="p-22">Metrichor’s cloud basecalling WIMP application correctly 2D analyzed, demultiplexed sequences and identified the organisms for each virus and bacteria in the library pool (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3F</a>). All of the URD organisms listed in <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> were reevaluated using Bracken to verify Metrichor’s identification and visualized with KRONA (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A-E</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="M. pneumoniae 2D library identification using Metrichor and Kraken/Bracken (A) 2D sequence (template + complement strands) as a function of quality score. (B) Bracken was used to align the reads and top hit abundance reads were visualized with KRONA" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1995865064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&lt;em&gt;M. pneumoniae&lt;/em&gt; 2D library identification using Metrichor and Kraken/Bracken&lt;/span&gt; (A) 2D sequence (template + complement strands) as a function of quality score. (B) Bracken was used to align the reads and top hit abundance reads were visualized with KRONA&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F2.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F2.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title"><em>M. pneumoniae</em> 2D library identification using Metrichor and Kraken/Bracken</span><p id="p-23" class="first-child">(A) 2D sequence (template + complement strands) as a function of quality score. (B) Bracken was used to align the reads and top hit abundance reads were visualized with KRONA</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiplexing libraries of URD organisms (A-E) Barcoded URD reads were aligned with Bracken and visualized with KRONA. (F) Number of reads for each barcode after analysis with the Metrichor application 2D base calling." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1995865064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Multiplexing libraries of URD organisms&lt;/span&gt; (A-E) Barcoded URD reads were aligned with Bracken and visualized with KRONA. (F) Number of reads for each barcode after analysis with the Metrichor application 2D base calling.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F3.medium.gif" width="281" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F3.medium.gif" width="281" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Multiplexing libraries of URD organisms</span><p id="p-24" class="first-child">(A-E) Barcoded URD reads were aligned with Bracken and visualized with KRONA. (F) Number of reads for each barcode after analysis with the Metrichor application 2D base calling.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>1D Rapid Library Preparation</h3><p id="p-25"><em>H. influenzae</em> and Coronavirus 229E libraries were prepared using a rapid 1D library preparation. This approach is suitable for WGA products and purified DNA with long fragments. The average Coronavirus 229E length from WTA was approximately 10 kb and the <em>H. influenzae</em> was 25 kb prior to library preparation. The <em>H. influenzae</em> sequencing run was allowed to run for 17.5 hours The total base yield was 44.85 million and a quality score was centered on 9 (S1). The read length was from 2000 bp and 24050 reads with the majority of the fragments at approximately 5000 bp. <em>H. influenzae</em> was correctly identified by the Metrichor’s WIMP (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) and the read alignment was verified with Bracken and visualization of the output was accomplished with Krona (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Coronavirus 229E was also successful identified using Metrichor’s WIMP application (S2). The coverage was calculated for both the <em>H. influenzae</em> and Coronavirus 229E with the formula C=LN/G, where C = coverage, L = average sequence read length, average Length N = number of reads, and G = the genome size. <em>H. influenzae</em> and Coronavirus 229E genome coverage was approximately 186X and 1700X, respectively.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="H. influenzae was prepared for sequencing using a rapid 1D method (A) H. influenzae read alignment using Metrichor WIMP application. (B) H. influenzae reads were aligned with Bracken and visualized with KRONA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1995865064" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&lt;em&gt;H. influenzae&lt;/em&gt; was prepared for sequencing using a rapid 1D method&lt;/span&gt; (A&lt;em&gt;) H. influenzae&lt;/em&gt; read alignment using Metrichor WIMP application. (B) &lt;em&gt;H. influenzae&lt;/em&gt; reads were aligned with Bracken and visualized with KRONA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F4.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F4.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/10/05/436600/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title"><em>H. influenzae</em> was prepared for sequencing using a rapid 1D method</span><p id="p-26" class="first-child">(A<em>) H. influenzae</em> read alignment using Metrichor WIMP application. (B) <em>H. influenzae</em> reads were aligned with Bracken and visualized with KRONA.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-14"><h2 class="">DISCUSSION</h2><p id="p-27">Sequencing of URD viruses and bacteria was accomplished on the MinION nanopore sequencer using both whole genome and amplicon sequencing. Having the ability to sequence amplicons and whole genomes in a mobile, small-footprint platform is attractive when collecting and analyzing samples in the field, and these qualities are also desirable as sequencing methods move toward environmental and surveillance applications. These methods, data, and results represent a practical and novel application for utilizing MinION nanopore sequencing technology. The data in this study were generated with both R7.3 and 9.4 chemistry flow cells, which accurately identified a panel of URD organisms. As the MinION technology improves, sequencing will generally become cheaper, faster, and more accurate as demonstrated in this study with improvements in identity alignment in the &gt;92% range (<a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">3</a>) as compared to earlier experiments that were in the 30% range [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>].</p><p id="p-28">The discrepancies observed between whole genome followed with RT-PCR and amplicon sequencing were minimal in regards to identity alignment (<a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">3</a>). The base-calling analysis for the MinION nanopore sequencer is an area of active development, as well as flow cell configurations, protocols for kits, and associated kit chemistries. The best quality sequence data are termed 2D reads, while 1D reads are from the template strand. Although the accuracy of MinION sequence data is an area of active development, with recent development of a new nanopore for the flow cells, one should expect increased data quality with each increment in flow cell configuration. Data quality has improved and was demonstrated in this study with each improvement in kit chemistry and flow cell type. This study shows that the platform can be applied for rapid microbial detection with 3-hour run time, which provides sufficient data to generate a reliable identification. As sequencing yield, quality, and turnaround times continue to improve, it is anticipated that nanopore sequencing will challenge mainstays such as PCR for identification of unknown samples. Amplicon and rapid WGS was successfully utilized as an approach for pathogen detection. Low titer viral nasal wash samples were was positively identified after WGA even when samples had high human read counts due to a carry-over human NA contribution. Parainfluenza 1, Parainfluenza 2, Parainfluenza 3 and adenovirus nasal wash samples had 81%, 90%, 95% and 81% of the read count aligning to human references genomes, respectively.</p><p id="p-29">A comparison of the nanopore and short-read sequencing data (Illumina MiSeq) showed that there was agreement in major taxonomic units identified. Therefore, the methodologies described in this study demonstrate that both whole genome and amplicon sequencing will be useful for rapid, accurate, and efficient detection of microbial and viral diversity in clinical samples. In this study, metagenomic sequencing of organisms in clinical samples were successful with increased sequencing depth when WGS was used over that of amplicon sequencing which is beneficial for studying viral genetic rearrangement. In addition, these data also support the utilization of amplicon sequencing over deep sequencing for field applications where surveillance is needed.</p><p id="p-30">The rapid 1D kit, defined as a single strand library composed of template strand, was used to evaluate whole genome amplification (WTA or WGA) of viruses and bacterial species. The library preparation utilizes a transposase to fragment the DNA. Analysis in this study was limited in scope but demonstrates it is functionally a game changer when time is of importance in identifying pathogens in acute infections or during surveillance. This approach is especially beneficial when identification is difficult using conventional bacterial culture techniques and in some cases where organisms are difficult to grow in cultures. Establishing a system for rapid microorganism identification via metagenomic sequencing seems pertinent, especially for field applications.</p><p id="p-31">Although long reads generated from the nanopore sequencer were found to have relatively higher error rates when compared with benchtop sequencing reads. MinION sequencing has been shown to have significant advantages, such as portability, low cost, and real-time detection. These advantages suggest that it is a potential for identification of infectious microbiota community <em>in situ</em>. Nevertheless, both whole genome low titer sample such as nasal washes (e.g. WTA, WGA) and amplicon sequencing are more desirable over culturing and will provide a method for identification of organisms in clinical samples. Throughout this study, it has been demonstrated that a low number of long reads are sufficient for accurate identification of small genomes.</p></div><div class="section" id="sec-15"><h2 class="">CONCLUSIONS</h2><p id="p-32">Recent preliminary reports within the MinION community suggest that the latest version of flow cell chemistry (Spot-On) utilizes more pores per sequence run and generates more data (gigabyte range) when configured with the new nanopore type. Spot-On 9.4 flow cell and flow cell chemistry have improved the amount of errors in sequencing data with reports of &gt;92% sequence alignment and on average &lt;1.2E-01 base error rate per 100 bases. These results confirm the observation by others that the per-base accuracy of MinION data higher that of benchtop DNA sequencing methods (Illumina MiSeq, <a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">3</a>). Targeted sequencing is useful even with higher error rates since gigabytes of data are collected for each run and the fragments are longer (range 800-3 kb) with overlapping sequences which can assist in mitigating error rates.</p><p id="p-33">Having the ability to sequence in a mobile, small-footprint platform is attractive when collecting and analyzing samples in the field, and these qualities are also desirable as sequencing methodology moves toward an even smaller footprint with the ongoing development of a cell phone interface called the SmidgION. These methods, data, and results represent a practical and novel application for the current stage of MinION nanopore sequencing technology and future applications using RNA direct will be a beneficial for rapid identification of RNA viruses. A comparison of the nanopore and short-read sequencing data (Illumina MiSeq) showed that there was agreement in major taxonomic units identified. Therefore, the methodologies described in this study demonstrate that both whole genome and amplicon sequencing will be useful for, accurate, and efficient detection of microbial and viral diversity in clinical samples.</p><div id="sec-16" class="subsection"><h3>Availability of supporting data</h3><p id="p-34">The raw sequence data sets are available at: <a href="https:figshare.com/articles/combined_fastq_zip/7172600">https:figshare.com/articles/combined_fastq_zip/7172600</a></p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-35">This work was supported by the Defense Health Program and the Chief Scientist Office at Wright Patterson Air Force Base, 711<sup>th</sup> Human Performance Wing, Ohio. The authors acknowledge the contribution of Mr. Craig Strapple for performing the bioinformatics analyses.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Upper respiratory virus serotype panel for the pyrosequencer</span>. <abbr class="cit-jnl-abbrev">Wright-Patterson AFB (OH): U.S. Air Force School of Aerospace Medicine; Technical Report AFRL-SA-WP-TR-2012-0002</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bao</span> <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">The influenza virus resource at National Center for Biotechnology Information J.Virol</abbr> <span class="cit-pub-date">2008</span>; <span class="cit-vol">82</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">596</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binfluenza%2Bvirus%2Bresource%2Bat%2BNational%2BCenter%2Bfor%2BBiotechnology%2BInformation%2BJ.Virol%26rft.volume%253D82%26rft.spage%253D596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.3" data-doi="10.1111/1469-0691.12217"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bertelli</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Greub</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Rapid bacterial genome sequencing: methods and applications in clinical microbiology</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr>. <span class="cit-vol">19</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">803</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BMicrobiol%2BInfect%26rft.volume%253D19%26rft.spage%253D803%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1469-0691.12217%26rft_id%253Dinfo%253Apmid%252F23601179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1469-0691.12217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601179&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.4" data-doi="10.1371/journal.ppat.1000455"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Genetic detection and characterization of Lujo virus, a new hemorrhagic fever-associated arenavirus from southern Africa</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e1000455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DBriese%26rft.auinit1%253DT.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000455%26rft.epage%253De1000455%26rft.atitle%253DGenetic%2Bdetection%2Band%2Bcharacterization%2Bof%2BLujo%2Bvirus%252C%2Ba%2Bnew%2Bhemorrhagic%2Bfever-associated%2Barenavirus%2Bfrom%2Bsouthern%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000455%26rft_id%253Dinfo%253Apmid%252F19478873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19478873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minot</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rivera</span>  <span class="cit-name-given-names">M</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">MinION nanopore sequencing of environmental metagenome; a synthetic approach</span>. <abbr class="cit-jnl-abbrev">GiaScience</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> <span class="cit-vol">6</span>:</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGiaScience%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.6" data-doi="10.1093/cid/ciu940"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Astrovirus VA1/HMO-C: an increasingly recognized neurotropic pathogen in immunocompromised patients</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis.</abbr> <span class="cit-vol">60</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">881</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciu940%26rft_id%253Dinfo%253Apmid%252F25572899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciu940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.7" data-doi="10.1038/nbt.2410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">JZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pallen</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oppenheim</span>  <span class="cit-name-given-names">B</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Constantinidou</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Genome sequencing in clinical microbiology</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1068</span>–<span class="cit-lpage">1071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DChan%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D30%26rft.issue%253D11%26rft.spage%253D1068%26rft.epage%253D1071%26rft.atitle%253DGenome%2Bsequencing%2Bin%2Bclinical%2Bmicrobiology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2410%26rft_id%253Dinfo%253Apmid%252F23138305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23138305&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.8" data-doi="10.1038/nbt.3423"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deamer</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akeson</span>  <span class="cit-name-given-names">M</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Branton</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Three decades of nanopore sequencing</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol.</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">518</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.9" data-doi="10.1093/bioinformatics/bty149"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Coster</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hert</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruts</span>  <span class="cit-name-given-names">M</span></span> and <span class="cit-auth"><span class="cit-name-surname">Van Broeckhoven</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Nanopack: visualizing and processing long-read sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatic</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">2666</span>–<span class="cit-lpage">2669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatic%26rft.volume%253D34%26rft.spage%253D2666%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty149%26rft_id%253Dinfo%253Apmid%252F29547981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29547981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.10" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">345</span>(<span class="cit-issue">6202</span>):<span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTAvMDUvNDM2NjAwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greninger</span>  <span class="cit-name-given-names">AL</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Rapid metagenomics identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis</span>. <abbr class="cit-jnl-abbrev">Genom Med</abbr>, <span class="cit-vol">7</span>:<span class="cit-fpage">99</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenom%2BMed%26rft.volume%253D7%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoke</span>  <span class="cit-name-given-names">CH</span>   <span class="cit-name-suffix">Jr</span>.</span>, <span class="cit-auth"><span class="cit-name-surname">Hawksworth</span>  <span class="cit-name-given-names">A</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">CE</span>   <span class="cit-name-suffix">Jr</span>.</span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Initial assessment of impact of adenovirus type 4 and type 7 vaccine on febrile respiratory illness and virus transmission in military basic trainees, March 2012</span>. <abbr class="cit-jnl-abbrev">MSMR.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">2</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMSMR%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D2%26rft.epage%253D6%26rft.atitle%253DInjuries%2Bdue%2Bto%2Bfirearms%2Band%2Bair%2Bguns%2Bamong%2BU.S.%2Bmilitary%2Bmembers%2Bnot%2Bparticipating%2Bin%2Boverseas%2Bcombat%2Boperations%252C%2B2002-2011.%26rft_id%253Dinfo%253Apmid%252F23030773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23030773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.13" data-doi="10.3201/eid1708.101328"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Imamura</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Enterovirus 68 among children with severe acute respiratory infection, the Philippines</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1430</span>–<span class="cit-lpage">1435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DImamura%26rft.auinit1%253DT.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D1430%26rft.epage%253D1435%26rft.atitle%253DEnterovirus%2B68%2Bamong%2Bchildren%2Bwith%2Bsevere%2Bacute%2Brespiratory%2Binfection%252C%2Bthe%2BPhilippines.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1708.101328%26rft_id%253Dinfo%253Apmid%252F21801620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1708.101328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21801620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forsyth</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">KM</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chanock</span>  <span class="cit-name-given-names">RM</span></span>. <span class="cit-pub-date">1963</span>. <span class="cit-article-title">Patterns of illness in rhinovirus infections of military personnel</span>. <abbr class="cit-jnl-abbrev">N Engl J Med.</abbr> <span class="cit-vol">269</span>:<span class="cit-fpage">602</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DForsyth%26rft.auinit1%253DB.%2BR.%26rft.volume%253D269%26rft.spage%253D602%26rft.epage%253D606%26rft.atitle%253DPATTERNS%2BOF%2BILLNESS%2BIN%2BRHINOVIRUS%2BINFECTIONS%2BOF%2BMILITARY%2BPERSONNEL.%26rft_id%253Dinfo%253Apmid%252F14044397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14044397&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.15" data-doi="10.3201/eid0503.990308"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Respiratory diseases among U.S. military personnel: countering emerging threats</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">379</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DGray%26rft.auinit1%253DG.%2BC.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D379%26rft.epage%253D385%26rft.atitle%253DRespiratory%2Bdiseases%2Bamong%2BU.S.%2Bmilitary%2Bpersonnel%253A%2Bcountering%2Bemerging%2Bthreats.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid0503.990308%26rft_id%253Dinfo%253Apmid%252F10341174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid0503.990308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10341174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080827600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.16" data-doi="10.1038/nature04239"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghedin</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2005</span> <span class="cit-article-title">Large-scale sequencing of human influenza reveals the dynamic nature of viral genome evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">437</span> (<span class="cit-issue">7062</span>):<span class="cit-fpage">1162</span>–<span class="cit-lpage">1166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhedin%26rft.auinit1%253DE.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1162%26rft.epage%253D1166%26rft.atitle%253DLarge-scale%2Bsequencing%2Bof%2Bhuman%2Binfluenza%2Breveals%2Bthe%2Bdynamic%2Bnature%2Bof%2Bviral%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04239%26rft_id%253Dinfo%253Apmid%252F16208317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16208317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johns</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Preliminary report: febrile acute respiratory disease caused by adenovirus type 14, U.S. Coast Guard Training Center, Cape May New Jersey, March 2009</span>. <abbr class="cit-jnl-abbrev">MSMR.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">10</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMSMR.%26rft.volume%253D16%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-436600v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Juul</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">What’s in my pot? Real-time species identification on the MinION</span>. <a href="http://biorxiv.org/content/early/2015/11/06/030742">http://biorxiv.org/content/early/2015/11/06/030742</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.19" data-doi="10.3201/eid1708.110028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaida</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Enterovirus 68 in children with acute respiratory tract infections, Osaka, Japan</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis. 2011</abbr>; <span class="cit-vol">17</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1494</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DKaida%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D1494%26rft.epage%253D1497%26rft.atitle%253DEnterovirus%2B68%2Bin%2Bchildren%2Bwith%2Bacute%2Brespiratory%2Btract%2Binfections%252C%2BOsaka%252C%2BJapan.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1708.110028%26rft_id%253Dinfo%253Apmid%252F21801632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1708.110028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21801632&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.20" data-doi="10.1186/s13742-015-0051-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kilianski</span>  <span class="cit-name-given-names">A</span></span>, et.al. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Bacterial and viral identification and differentiation by amplicon sequencing on the MinION nanopore sequencer</span>. <abbr class="cit-jnl-abbrev">Gigascience</abbr>. <span class="cit-vol">4</span>:<span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0051-z%26rft_id%253Dinfo%253Apmid%252F25815165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0051-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25815165&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.21" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>., <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">1000 Genome Project Data Processing Subgroup (2009) The Sequence alignment/map (SAM) format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.22" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>. and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler Transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, Epub.<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> [</span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">20080505</span>]</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080505&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.23" data-doi="10.1016/j.jviromet.2014.12.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Comparing viral metagenomics methods using a highly multiplexed human viral pathogens reagent</span>. <abbr class="cit-jnl-abbrev">J Virol Methods</abbr>. <span class="cit-vol">213</span>:<span class="cit-fpage">139</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2BMethods%26rft.volume%253D213%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2014.12.002%26rft_id%253Dinfo%253Apmid%252F25497414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2014.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25497414&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabele</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">GC</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Earhart</span>  <span class="cit-name-given-names">KC</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Upper respiratory tract infections (URI)</span>. <abbr class="cit-jnl-abbrev">Mil Med.</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">15</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMil%2BMed.%26rft.volume%253D169%26rft.spage%253D15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">NJ</span></span> and <span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2014</span>. <abbr class="cit-jnl-abbrev">Poretools: a toolkit for analyzing nanopore sequence data Bioinformatics 1</abbr>:<span class="cit-vol">30</span>(<span class="cit-issue">23</span>)<span class="cit-fpage">3399</span>–<span class="cit-lpage">3401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoretools%253A%2Ba%2Btoolkit%2Bfor%2Banalyzing%2Bnanopore%2Bsequence%2Bdata%2BBioinformatics%2B1%26rft.volume%253D30%26rft.spage%253D3399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.26" data-doi="10.1038/nmeth.3444"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">JT</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-vol">12</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">733</span>–<span class="cit-lpage">735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D733%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3444%26rft_id%253Dinfo%253Apmid%252F26076426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26076426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.27" data-doi="10.7717/peerj-cs.104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>  <span class="cit-name-given-names">FP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thielen</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Bracken: estimating species abundance in metagenomics data</span>. <abbr class="cit-jnl-abbrev">Peer J Computer Science</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e104</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7717/peerj-cs.104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeer%2BJ%2BComputer%2BScience%26rft.volume%253D3%26rft.spage%253D104e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj-cs.104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj-cs.104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.28" data-doi="10.1186/s12864-015-1519-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madoui</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Genome assembly using Nanopore-guided long and error-free DNA reads</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-vol">16</span>:<span class="cit-fpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1519-z%26rft_id%253Dinfo%253Apmid%252F25927464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1519-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25927464&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.29" data-doi="10.1186/gm485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montoya</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardy</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Metagenomics for pathogen detection in public health</span>. <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D5%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm485%26rft_id%253Dinfo%253Apmid%252F24050114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24050114&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.30" data-doi="10.1093/cid/ciu912"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naccache</span>  <span class="cit-name-given-names">SN</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Diagnosis of neuroinvasive astrovirus infection in an immunocompromised adult with encephalitis by unbiased next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis.</abbr> <span class="cit-vol">60</span> (<span class="cit-issue">6</span>):<span class="cit-fpage">919</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciu912%26rft_id%253Dinfo%253Apmid%252F25572898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciu912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572898&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.31" data-doi="10.1371/journal.pone.0004219"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Direct metagenomic detection of viral pathogens in nasal and fecal specimens using an unbiased high-throughput sequencing approach</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">4</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e4219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNakamura%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253De4219%26rft.epage%253De4219%26rft.atitle%253DDirect%2Bmetagenomic%2Bdetection%2Bof%2Bviral%2Bpathogens%2Bin%2Bnasal%2Band%2Bfecal%2Bspecimens%2Busing%2Ban%2Bunbiased%2Bhigh-throughput%2Bsequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004219%26rft_id%253Dinfo%253Apmid%252F19156205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19156205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.32" data-doi="10.1186/1471-2105-12-385"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ondov</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">NH</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">AM</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Interactive metagenomics visualization in web browser</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics 30</abbr>: <span class="cit-vol">12</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DOndov%26rft.auinit1%253DB.%2BD.%26rft.volume%253D12%26rft.spage%253D385%26rft.epage%253D385%26rft.atitle%253DInteractive%2Bmetagenomic%2Bvisualization%2Bin%2Ba%2BWeb%2Bbrowser.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-385%26rft_id%253Dinfo%253Apmid%252F21961884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21961884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.33" data-doi="10.1111/j.1750-2659.2007.00029.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">’Shea</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pipkin</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cane</span>  <span class="cit-name-given-names">PA</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">GC</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Respiratory syncytial virus: an important cause of acute respiratory illness among young adults undergoing military training</span>. <abbr class="cit-jnl-abbrev">Influenza Other Respir Viruses</abbr>. <span class="cit-vol">1</span> (<span class="cit-issue">5-6</span>):<span class="cit-fpage">193</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfluenza%2BOther%2BRespir%2BViruses%26rft.volume%253D1%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1750-2659.2007.00029.x%26rft_id%253Dinfo%253Apmid%252F18846262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1750-2659.2007.00029.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846262&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.34" data-doi="10.1038/srep29681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Analysis of mouse gut microbiome using full-length 16S rRNA amplicon sequencing</span>. <abbr class="cit-jnl-abbrev">Sci Rep. 2016</abbr>; <span class="cit-vol">6</span>:<span class="cit-fpage">29681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep.%2B2016%26rft.volume%253D6%26rft.spage%253D29681%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep29681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep29681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.35" data-doi="10.4172/2161-0517.S2-004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">CS</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Human respiratory coronaviruses detected in patients with influenza-like illness in Arkansas, USA</span>. <abbr class="cit-jnl-abbrev">Virol Mycol.</abbr> <span class="cit-vol">S2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.4172/2161-0517.S2-004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BMycol.%26rft_id%253Dinfo%253Adoi%252F10.4172%252F2161-0517.S2-004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4172/2161-0517.S2-004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.36" data-doi="10.1128/mBio.00473-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Boheemen</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans</span>. <abbr class="cit-jnl-abbrev">MBio.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">e00473</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio.%26rft.volume%253D3%26rft.spage%253De00473%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00473-12%26rft_id%253Dinfo%253Apmid%252F23170002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00473-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23170002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.37" data-doi="10.1186/s12864-015-1284-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Den Hoecke</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verhelst</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vuylsteke</span>  <span class="cit-name-given-names">M</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saelens</span>  <span class="cit-name-given-names">X</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Analysis of the genetic diversity of influenza A viruses using next-generation DNA sequencing</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-vol">16</span>:<span class="cit-fpage">79</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1284-z%26rft_id%253Dinfo%253Apmid%252F25758772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1284-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25758772&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.38" data-doi="10.1016/j.mib.2014.11.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vernikos</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Medini</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tettelin</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Ten years of pan-genome analyses</span>. <abbr class="cit-jnl-abbrev">Curr Opin Microbiol.</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">148</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BMicrobiol.%26rft.volume%253D23%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2014.11.016%26rft_id%253Dinfo%253Apmid%252F25483351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2014.11.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25483351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quast</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools Nucl</span>. <abbr class="cit-jnl-abbrev">Acids Res.</abbr> <span class="cit-vol">41</span> (<span class="cit-issue">D1</span>): <span class="cit-fpage">D590</span>–<span class="cit-lpage">D596</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-436600v1.40" data-doi="10.1186/2047-217X-3-22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">AR</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">NJ</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">A reference bacterial genome dataset generated on the MinION™ portable single-molecule nanopore sequencer</span>. <abbr class="cit-jnl-abbrev">Gigascience</abbr>. <span class="cit-vol">3</span>:<span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D3%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-3-22%26rft_id%253Dinfo%253Apmid%252F25386338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-3-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25386338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.41" data-doi="10.1016/j.jcv.2014.06.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quiñones-Mateu</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avila</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reyes-Teran</span>  <span class="cit-name-given-names">G</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Deep sequencing: becoming a critical tool in clinical virology</span>. <abbr class="cit-jnl-abbrev">J Clin Virol.</abbr> <span class="cit-vol">61</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">9</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BVirol.%26rft.volume%253D61%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2014.06.013%26rft_id%253Dinfo%253Apmid%252F24998424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcv.2014.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24998424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.42" data-doi="10.1186/gb-2014-15-3-r46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">DE</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">15</span>:<span class="cit-fpage">R46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24580807&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-436600v1.43" data-doi="10.1016/S0140-6736(13)61207-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Human infection with avian influenza A H7N9 virus: an assessment of clinical severity</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-vol">382</span>(<span class="cit-issue">9887</span>):<span class="cit-fpage">138</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D382%26rft.spage%253D138%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252961207-6%26rft_id%253Dinfo%253Apmid%252F23803487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)61207-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23803487&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F10%2F05%2F436600.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322115300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/436600v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F436600v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/436295v1" title="The dynamics of spawning acts by a semelparous fish and its associated energetic expenses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 05, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/436600v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/436600v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/436600v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/436600v1" class="active" data-icon-position="" data-hide-link-title="0">Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/131107" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sGm3iCozoGk0LFdbQOrdfsr-cK7eQRcryDod36L-5bk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="131107" id="node131107--2" data-pisa="biorxiv;436600v1" data-pisa-master="biorxiv;436600" data-seqnum="131107" data-apath="/biorxiv/early/2018/10/05/436600.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wanda J.</span> <span class="nlm-surname">Lyon</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zachary K.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Grier</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">James</span> <span class="nlm-surname">Baldwin</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Clarise R.</span> <span class="nlm-surname">Starr</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">436600; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/436600 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/436600v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;title=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;title=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;text=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;title=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;t=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;title=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;title=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="131107" id="node131107--3" data-pisa="biorxiv;436600v1" data-pisa-master="biorxiv;436600" data-seqnum="131107" data-apath="/biorxiv/early/2018/10/05/436600.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Evaluating an Upper Respiratory Disease Panel on the Portable MinION Sequencer</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Wanda J.</span> <span class="nlm-surname">Lyon</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zachary K.</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Grier</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">James</span> <span class="nlm-surname">Baldwin</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Clarise R.</span> <span class="nlm-surname">Starr</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">436600; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/436600 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Lyon%2C+Wanda+J.&amp;rft.au=Smith%2C+Zachary+K.&amp;rft.au=Grier%2C+Brian&amp;rft.au=Baldwin%2C+James&amp;rft.au=Starr%2C+Clarise+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/131107/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/131107/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/131107/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/131107/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/131107/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/131107/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/131107/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/131107/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/131107/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/131107/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/131107/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/131107/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;count=horizontal&amp;via=&amp;text=Evaluating%20an%20Upper%20Respiratory%20Disease%20Panel%20on%20the%20Portable%20MinION%20Sequencer&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/436600v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/436600v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/436600v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
