<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/embed/graphic-7.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/13/469692.atom" />
<meta name="HW.pisa" content="biorxiv;469692v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-throughput interrogation of programmed ribosomal frameshifting in human cells" />
<meta name="DC.Identifier" content="10.1101/469692" />
<meta name="DC.Date" content="2020-03-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting." />
<meta name="DC.Contributor" content="Martin Mikl" />
<meta name="DC.Contributor" content="Yitzhak Pilpel" />
<meta name="DC.Contributor" content="Eran Segal" />
<meta name="article:published_time" content="2020-03-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="High-throughput interrogation of programmed ribosomal frameshifting in human cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &amp;gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &amp;gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;469692v2" />
<meta name="citation_id" content="469692v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/469692v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/469692v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/469692v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692.full.pdf" />
<meta name="citation_doi" content="10.1101/469692" />
<meta name="citation_num_pages" content="41" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="469692" />
<meta name="citation_author" content="Martin Mikl" />
<meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics, Weizmann Institute of Science" />
<meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, Weizmann Institute of Science" />
<meta name="citation_author_email" content="eran.segal@weizmann.ac.il" />
<meta name="citation_author_email" content="martin.mikl@weizmann.ac.il" />
<meta name="citation_author" content="Yitzhak Pilpel" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, Weizmann Institute of Science" />
<meta name="citation_author" content="Eran Segal" />
<meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics, Weizmann Institute of Science" />
<meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science" />
<meta name="citation_author_email" content="eran.segal@weizmann.ac.il" />
<meta name="citation_author_email" content="martin.mikl@weizmann.ac.il" />
<meta name="citation_reference" content="Advani, V.M., and Dinman, J.D. (2016). Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression. BioEssays News Rev. Mol. Cell. Dev. Biol. 38, 21–26." />
<meta name="citation_reference" content="Baril, M., Dulude, D., Gendron, K., Lemay, G., and Brakier-Gingras, L. (2003). Efficiency of a programmed −1 ribosomal frameshift in the different subtypes of the human immunodeficiency virus type 1 group M. RNA 9, 1246–1253." />
<meta name="citation_reference" content="Belew, A.T., Hepler, N.L., Jacobs, J.L., and Dinman, J.D. (2008). PRFdb: a database of computationally predicted eukaryotic programmed −1 ribosomal frameshift signals. BMC Genomics 9, 339." />
<meta name="citation_reference" content="Belew, A.T., Advani, V.M., and Dinman, J.D. (2011). Endogenous ribosomal frameshift signals operate as mRNA destabilizing elements through at least two molecular pathways in yeast. Nucleic Acids Res. 39, 2799–2808." />
<meta name="citation_reference" content="Belew, A.T., Meskauskas, A., Musalgaonkar, S., Advani, V.M., Sulima, S.O., Kasprzak, W.K., Shapiro, B.A., and Dinman, J.D. (2014). Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway. Nature 512, 265–269." />
<meta name="citation_reference" content="Bidou, L., Stahl, G., Grima, B., Liu, H., Cassan, M., and Rousset, J.P. (1997). In vivo HIV-1 frameshifting efficiency is directly related to the stability of the stem-loop stimulatory signal. RNA N. Y. N 3, 1153–1158." />
<meta name="citation_reference" content="Biswas, P., Jiang, X., Pacchia, A.L., Dougherty, J.P., and Peltz, S.W. (2004). The Human Immunodeficiency Virus Type 1 Ribosomal Frameshifting Site Is an Invariant Sequence Determinant and an Important Target for Antiviral Therapy. J. Virol. 78, 2082–2087." />
<meta name="citation_reference" content="Brakier-Gingras, L., Charbonneau, J., and Butcher, S.E. (2012). Targeting frameshifting in the human immunodeficiency virus. Expert Opin. Ther. Targets 16, 249–258." />
<meta name="citation_reference" content="Brierley, I., and Pennell, S. (2001). Structure and function of the stimulatory RNAs involved in programmed eukaryotic-1 ribosomal frameshifting. Cold Spring Harb. Symp. Quant. Biol. 66, 233–248." />
<meta name="citation_reference" content="Brierley, I., Digard, P., and Inglis, S.C. (1989). Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot. Cell 57, 537–547." />
<meta name="citation_reference" content="Caliskan, N., Katunin, V.I., Belardinelli, R., Peske, F., and Rodnina, M.V. (2014). Programmed −1 frameshifting by kinetic partitioning during impeded translocation. Cell 157, 1619–1631." />
<meta name="citation_reference" content="Caliskan, N., Peske, F., and Rodnina, M.V. (2015). Changed in translation: mRNA recoding by −1 programmed ribosomal frameshifting. Trends Biochem. Sci. 40, 265–274." />
<meta name="citation_reference" content="Caliskan, N., Wohlgemuth, I., Korniy, N., Pearson, M., Peske, F., and Rodnina, M.V. (2017). Conditional Switch between Frameshifting Regimes upon Translation of dnaX mRNA. Mol. Cell 66, 558-567.e4." />
<meta name="citation_reference" content="Chen, T., and Guestrin, C. (2016). XGBoost: A Scalable Tree Boosting System. Proc. 22nd ACM SIGKDD Int. Conf. Knowl. Discov. Data Min. - KDD 16 785–794." />
<meta name="citation_reference" content="Chen, J., Petrov, A., Johansson, M., Tsai, A., O’Leary, S.E., and Puglisi, J.D. (2014). Dynamic pathways of −1 translational frameshifting. Nature 512, 328–332." />
<meta name="citation_reference" content="Choi, J., O’Loughlin, S., Atkins, J.F., and Puglisi, J.D. (2020). The energy landscape of −1 ribosomal frameshifting. Sci. Adv. 6, eaax6969." />
<meta name="citation_reference" content="Clark, M.B., Jänicke, M., Gottesbühren, U., Kleffmann, T., Legge, M., Poole, E.S., and Tate, W.P. (2007). Mammalian Gene PEG10 Expresses Two Reading Frames by High Efficiency –1 Frameshifting in Embryonic-associated Tissues. J. Biol. Chem. 282, 37359–37369." />
<meta name="citation_reference" content="ten Dam, E., Brierley, I., Inglis, S., and Pleij, C. (1994). Identification and analysis of the pseudoknot-containing gag-pro ribosomal frameshift signal of simian retrovirus-1. Nucleic Acids Res. 22, 2304–2310." />
<meta name="citation_reference" content="Dinman, J.D. (2012). Control of gene expression by translational recoding. Adv. Protein Chem. Struct. Biol. 86, 129–149." />
<meta name="citation_reference" content="Dinman, J.D., and Wickner, R.B. (1992). Ribosomal frameshifting efficiency and gag/gag-pol ratio are critical for yeast M1 double-stranded RNA virus propagation. J. Virol. 66, 3669–3676." />
<meta name="citation_reference" content="Du, Z., Holland, J.A., Hansen, M.R., Giedroc, D.P., and Hoffman, D.W. (1997). Base-pairings within the RNA pseudoknot associated with the simian retrovirus-1 gag-pro frameshift site. J. Mol. Biol. 270, 464–470." />
<meta name="citation_reference" content="Felsenstein, K.M., and Goff, S.P. (1988). Expression of the gag-pol fusion protein of Moloney murine leukemia virus without gag protein does not induce virion formation or proteolytic processing. J. Virol. 62, 2179–2182." />
<meta name="citation_reference" content="Firth, A.E., and Atkins, J.F. (2009). A conserved predicted pseudoknot in the NS2A-encoding sequence of West Nile and Japanese encephalitis flaviviruses suggests NS1’ may derive from ribosomal frameshifting. Virol. J. 6, 14." />
<meta name="citation_reference" content="Firth, A.E., and Brierley, I. (2012). Non-canonical translation in RNA viruses. J. Gen. Virol. 93, 1385–1409." />
<meta name="citation_reference" content="Gallant, J.A., and Lindsley, D. (1992). Leftward ribosome frameshifting at a hungry codon. J. Mol. Biol. 223, 31–40." />
<meta name="citation_reference" content="Garcia-Miranda, P., Becker, J.T., Benner, B.E., Blume, A., Sherer, N.M., and Butcher, S.E. (2016). Stability of HIV Frameshift Site RNA Correlates with Frameshift Efficiency and Decreased Virus Infectivity. J. Virol. 90, 6906–6917." />
<meta name="citation_reference" content="Girstmair, H., Saffert, P., Rode, S., Czech, A., Holland, G., Bannert, N., and Ignatova, Z. (2013). Depletion of Cognate Charged Transfer RNA Causes Translational Frameshifting within the Expanded CAG Stretch in Huntingtin. Cell Rep. 3, 148–159." />
<meta name="citation_reference" content="Halma, M.T.J., Ritchie, D.B., Cappellano, T.R., Neupane, K., and Woodside, M.T. (2019). Complex dynamics under tension in a high-efficiency frameshift stimulatory structure. Proc. Natl. Acad. Sci. U. S. A." />
<meta name="citation_reference" content="Hemelaar, J., Gouws, E., Ghys, P.D., and Osmanov, S. (2006). Global and regional distribution of HIV-1 genetic subtypes and recombinants in 2004. AIDS Lond. Engl. 20, W13–23." />
<meta name="citation_reference" content="Hofacker, I.L., Priwitzer, B., and Stadler, P.F. (2004). Prediction of locally stable RNA secondary structures for genome-wide surveys. Bioinforma. Oxf. Engl. 20, 186–190." />
<meta name="citation_reference" content="Hung, M., Patel, P., Davis, S., and Green, S.R. (1998). Importance of ribosomal frameshifting for human immunodeficiency virus type 1 particle assembly and replication. J. Virol. 72, 4819–4824." />
<meta name="citation_reference" content="Ivanov, I.P., and Atkins, J.F. (2007). Ribosomal frameshifting in decoding antizyme mRNAs from yeast and protists to humans: close to 300 cases reveal remarkable diversity despite underlying conservation. Nucleic Acids Res. 35, 1842–1858." />
<meta name="citation_reference" content="Ivanov, I.P., Gesteland, R.F., and Atkins, J.F. (1998). A second mammalian antizyme: conservation of programmed ribosomal frameshifting. Genomics 52, 119–129." />
<meta name="citation_reference" content="Ivanov, I.P., Matsufuji, S., Murakami, Y., Gesteland, R.F., and Atkins, J.F. (2000). Conservation of polyamine regulation by translational frameshifting from yeast to mammals. EMBO J. 19, 1907–1917." />
<meta name="citation_reference" content="Ivanov, I.P., Gesteland, R.F., and Atkins, J.F. (2006). Evolutionary specialization of recoding: Frameshifting in the expression of S. cerevisiae antizyme mRNA is via an atypical antizyme shift site but is still +1. RNA 12, 332–337." />
<meta name="citation_reference" content="Jacks, T., and Varmus, H.E. (1985). Expression of the Rous sarcoma virus pol gene by ribosomal frameshifting. Science 230, 1237–1242." />
<meta name="citation_reference" content="Jacks, T., Power, M.D., Masiarz, F.R., Luciw, P.A., Barr, P.J., and Varmus, H.E. (1988a). Characterization of ribosomal frameshifting in HIV-1 gag-pol expression. Nature 331, 280–283." />
<meta name="citation_reference" content="Jacks, T., Madhani, H.D., Masiarz, F.R., and Varmus, H.E. (1988b). Signals for ribosomal frameshifting in the Rous sarcoma virus gag-pol region. Cell 55, 447–458." />
<meta name="citation_reference" content="Janssen, S., and Giegerich, R. (2015). The RNA shapes studio. Bioinformatics 31, 423–425." />
<meta name="citation_reference" content="Ketteler, R. (2012). On programmed ribosomal frameshifting: the alternative proteomes. Front. Genet. 3, 242." />
<meta name="citation_reference" content="Kleinkauf, R., Houwaart, T., Backofen, R., and Mann, M. (2015a). antaRNA – Multi-objective inverse folding of pseudoknot RNA using ant-colony optimization. BMC Bioinformatics 16, 389." />
<meta name="citation_reference" content="Kleinkauf, R., Mann, M., and Backofen, R. (2015b). antaRNA: ant colony-based RNA sequence design. Bioinformatics 31, 3114–3121." />
<meta name="citation_reference" content="Kollmus, H., Honigman, A., Panet, A., and Hauser, H. (1994). The sequences of and distance between two cis-acting signals determine the efficiency of ribosomal frameshifting in human immunodeficiency virus type 1 and human T-cell leukemia virus type II in vivo. J. Virol. 68, 6087–6091." />
<meta name="citation_reference" content="Korniy, N., Goyal, A., Hoffmann, M., Samatova, E., Peske, F., Pöhlmann, S., and Rodnina, M.V. (2019). Modulation of HIV-1 Gag/Gag-Pol frameshifting by tRNA abundance. Nucleic Acids Res. 47, 5210–5222." />
<meta name="citation_reference" content="Kurian, L., Palanimurugan, R., Gödderz, D., and Dohmen, R.J. (2011). Polyamine sensing by nascent ornithine decarboxylase antizyme stimulates decoding of its mRNA. Nature 477, 490–494." />
<meta name="citation_reference" content="Lin, Z., Gilbert, R.J.C., and Brierley, I. (2012). Spacer-length dependence of programmed −1 or −2 ribosomal frameshifting on a U6A heptamer supports a role for messenger RNA (mRNA) tension in frameshifting. Nucleic Acids Res. 40, 8674–8689." />
<meta name="citation_reference" content="Lorenz, R., Bernhart, S.H., Höner Zu Siederdissen, C., Tafer, H., Flamm, C., Stadler, P.F., and Hofacker, I.L. (2011). ViennaRNA Package 2.0. Algorithms Mol. Biol. AMB 6, 26." />
<meta name="citation_reference" content="Lundberg, S.M., Erion, G., Chen, H., DeGrave, A., Prutkin, J.M., Nair, B., Katz, R., Himmelfarb, J., Bansal, N., and Lee, S.-I. (2020). From local explanations to global understanding with explainable AI for trees. Nat. Mach. Intell. 2, 56–67." />
<meta name="citation_reference" content="Lyon, K., Aguilera, L.U., Morisaki, T., Munsky, B., and Stasevich, T.J. (2019). Live-Cell Single RNA Imaging Reveals Bursts of Translational Frameshifting. Mol. Cell 75, 172-183.e9." />
<meta name="citation_reference" content="Manktelow, E., Shigemoto, K., and Brierley, I. (2005). Characterization of the frameshift signal of Edr, a mammalian example of programmed −1 ribosomal frameshifting. Nucleic Acids Res. 33, 1553–1563." />
<meta name="citation_reference" content="Marcheschi, R.J., Staple, D.W., and Butcher, S.E. (2007). Programmed Ribosomal Frameshifting in SIV is Induced by a Highly Structured RNA Stem-Loop. J. Mol. Biol. 373, 652–663." />
<meta name="citation_reference" content="Marczinke, B., Fisher, R., Vidakovic, M., Bloys, A.J., and Brierley, I. (1998). Secondary structure and mutational analysis of the ribosomal frameshift signal of rous sarcoma virus. J. Mol. Biol. 284, 205–225." />
<meta name="citation_reference" content="Matsufuji, S., Matsufuji, T., Miyazaki, Y., Murakami, Y., Atkins, J.F., Gesteland, R.F., and Hayashi, S. (1995). Autoregulatory frameshifting in decoding mammalian ornithine decarboxylase antizyme. Cell 80, 51–60." />
<meta name="citation_reference" content="Melian, E.B., Hinzman, E., Nagasaki, T., Firth, A.E., Wills, N.M., Nouwens, A.S., Blitvich, B.J., Leung, J., Funk, A., Atkins, J.F., et al. (2010). NS1′ of Flaviviruses in the Japanese Encephalitis Virus Serogroup Is a Product of Ribosomal Frameshifting and Plays a Role in Viral Neuroinvasiveness. J. Virol. 84, 1641–1647." />
<meta name="citation_reference" content="Melian, E.B., Hall-Mendelin, S., Du, F., Owens, N., Bosco-Lauth, A.M., Nagasaki, T., Rudd, S., Brault, A.C., Bowen, R.A., Hall, R.A., et al. (2014). Programmed ribosomal frameshift alters expression of west nile virus genes and facilitates virus replication in birds and mosquitoes. PLoS Pathog. 10, e1004447." />
<meta name="citation_reference" content="Meydan, S., Klepacki, D., Karthikeyan, S., Margus, T., Thomas, P., Jones, J.E., Khan, Y., Briggs, J., Dinman, J.D., Vázquez-Laslop, N., et al. (2017). Programmed Ribosomal Frameshifting Generates a Copper Transporter and a Copper Chaperone from the Same Gene. Mol. Cell 65, 207–219." />
<meta name="citation_reference" content="Mikl, M., Hamburg, A., Pilpel, Y., and Segal, E. (2019). Dissecting splicing decisions and cell-to-cell variability with designed sequence libraries. Nat. Commun. 10, 4572." />
<meta name="citation_reference" content="Moomau, C., Musalgaonkar, S., Khan, Y.A., Jones, J.E., and Dinman, J.D. (2016). Structural and Functional Characterization of Programmed Ribosomal Frameshift Signals in West Nile Virus Strains Reveals High Structural Plasticity Among cis-Acting RNA Elements. J. Biol. Chem. 291, 15788–15795." />
<meta name="citation_reference" content="Moon, S., Byun, Y., and Han, K. (2007). FSDB: a frameshift signal database. Comput. Biol. Chem. 31, 298–302." />
<meta name="citation_reference" content="Mouzakis, K.D., Lang, A.L., Vander Meulen, K.A., Easterday, P.D., and Butcher, S.E. (2013). HIV-1 frameshift efficiency is primarily determined by the stability of base pairs positioned at the mRNA entrance channel of the ribosome. Nucleic Acids Res. 41, 1901–1913." />
<meta name="citation_reference" content="Nam, S.H., Copeland, T.D., Hatanaka, M., and Oroszlan, S. (1993). Characterization of ribosomal frameshifting for expression of pol gene products of human T-cell leukemia virus type I. J. Virol. 67, 196–203." />
<meta name="citation_reference" content="Nikolic, E.I.C., King, L.M., Vidakovic, M., Irigoyen, N., and Brierley, I. (2012). Modulation of ribosomal frameshifting frequency and its effect on the replication of Rous sarcoma virus. J. Virol. 86, 11581–11594." />
<meta name="citation_reference" content="Ono, M., Yasunaga, T., Miyata, T., and Ushikubo, H. (1986). Nucleotide sequence of human endogenous retrovirus genome related to the mouse mammary tumor virus genome. J. Virol. 60, 589–598." />
<meta name="citation_reference" content="Parkin, N.T., Chamorro, M., and Varmus, H.E. (1992). Human immunodeficiency virus type 1 gag-pol frameshifting is dependent on downstream mRNA secondary structure: demonstration by expression in vivo. J. Virol. 66, 5147–5151." />
<meta name="citation_reference" content="Plant, E.P., Pérez-Alvarado, G.C., Jacobs, J.L., Mukhopadhyay, B., Hennig, M., and Dinman, J.D. (2005). A three-stemmed mRNA pseudoknot in the SARS coronavirus frameshift signal. PLoS Biol. 3, e172." />
<meta name="citation_reference" content="Pollom, E., Dang, K.K., Potter, E.L., Gorelick, R.J., Burch, C.L., Weeks, K.M., and Swanstrom, R. (2013). Comparison of SIV and HIV-1 Genomic RNA Structures Reveals Impact of Sequence Evolution on Conserved and Non-Conserved Structural Motifs. PLOS Pathog. 9, e1003294." />
<meta name="citation_reference" content="Ritchie, D.B., Foster, D.A.N., and Woodside, M.T. (2012). Programmed −1 frameshifting efficiency correlates with RNA pseudoknot conformational plasticity, not resistance to mechanical unfolding. Proc. Natl. Acad. Sci. U. S. A. 109, 16167–16172." />
<meta name="citation_reference" content="Ritchie, D.B., Cappellano, T.R., Tittle, C., Rezajooei, N., Rouleau, L., Sikkema, W.K.A., and Woodside, M.T. (2017). Conformational dynamics of the frameshift stimulatory structure in HIV-1. RNA N. Y. N 23, 1376–1384." />
<meta name="citation_reference" content="Rom, E., and Kahana, C. (1994). Polyamines regulate the expression of ornithine decarboxylase antizyme in vitro by inducing ribosomal frame-shifting. Proc. Natl. Acad. Sci. U. S. A. 91, 3959–3963." />
<meta name="citation_reference" content="Staple, D.W., and Butcher, S.E. (2003). Solution structure of the HIV-1 frameshift inducing stem-loop RNA. Nucleic Acids Res. 31, 4326–4331." />
<meta name="citation_reference" content="Su, M.-C., Chang, C.-T., Chu, C.-H., Tsai, C.-H., and Chang, K.-Y. (2005). An atypical RNA pseudoknot stimulator and an upstream attenuation signal for −1 ribosomal frameshifting of SARS coronavirus. Nucleic Acids Res. 33, 4265–4275." />
<meta name="citation_reference" content="Theis, C., Reeder, J., and Giegerich, R. (2008). KnotInFrame: prediction of −1 ribosomal frameshift events. Nucleic Acids Res. 36, 6013–6020." />
<meta name="citation_reference" content="Thiel, V., Ivanov, K.A., Putics, A., Hertzig, T., Schelle, B., Bayer, S., Weissbrich, B., Snijder, E.J., Rabenau, H., Doerr, H.W., et al. (2003). Mechanisms and enzymes involved in SARS coronavirus genome expression. J. Gen. Virol. 84, 2305–2315." />
<meta name="citation_reference" content="Tholstrup, J., Oddershede, L.B., and Sørensen, M.A. (2012). mRNA pseudoknot structures can act as ribosomal roadblocks. Nucleic Acids Res. 40, 303–313." />
<meta name="citation_reference" content="Tosaka, Y., Tanaka, H., Yano, Y., Masai, K., Nozaki, M., Yomogida, K., Otani, S., Nojima, H., and Nishimune, Y. (2000). Identification and characterization of testis specific ornithine decarboxylase antizyme (OAZ-t) gene: expression in haploid germ cells and polyamine-induced frameshifting. Genes Cells Devoted Mol. Cell. Mech. 5, 265–276." />
<meta name="citation_reference" content="Tu, C., Tzeng, T.H., and Bruenn, J.A. (1992). Ribosomal movement impeded at a pseudoknot required for frameshifting. Proc. Natl. Acad. Sci. U. S. A. 89, 8636–8640." />
<meta name="citation_reference" content="Vainberg Slutskin, I., Weingarten-Gabbay, S., Nir, R., Weinberger, A., and Segal, E. (2018). Unraveling the determinants of microRNA mediated regulation using a massively parallel reporter assay. Nat. Commun. 9." />
<meta name="citation_reference" content="Waldman, Y.Y., Tuller, T., Shlomi, T., Sharan, R., and Ruppin, E. (2010). Translation efficiency in humans: tissue specificity, global optimization and differences between developmental stages. Nucleic Acids Res. 38, 2964–2974." />
<meta name="citation_reference" content="Wang, X., Xuan, Y., Han, Y., Ding, X., Ye, K., Yang, F., Gao, P., Goff, S.P., and Gao, G. (2019). Regulation of HIV-1 Gag-Pol Expression by Shiftless, an Inhibitor of Programmed −1 Ribosomal Frameshifting. Cell 176, 625-635.e14." />
<meta name="citation_reference" content="Wang, Y., Wills, N.M., Du, Z., Rangan, A., Atkins, J.F., Gesteland, R.F., and Hoffman, D.W. (2002). Comparative studies of frameshifting and nonframeshifting RNA pseudoknots: a mutational and NMR investigation of pseudoknots derived from the bacteriophage T2 gene 32 mRNA and the retroviral gag-pro frameshift site. RNA N. Y. N 8, 981–996." />
<meta name="citation_reference" content="Weingarten-Gabbay, S., Elias-Kirma, S., Nir, R., Gritsenko, A.A., Stern-Ginossar, N., Yakhini, Z., Weinberger, A., and Segal, E. (2016). Systematic discovery of cap-independent translation sequences in human and viral genomes. Science 351, aad4939." />
<meta name="citation_reference" content="Weingarten-Gabbay, S., Nir, R., Lubliner, S., Sharon, E., Kalma, Y., Weinberger, A., and Segal, E. (2019). Systematic interrogation of human promoters. Genome Res. 29, 171–183." />
<meta name="citation_reference" content="Weiss, R., Lindsley, D., Falahee, B., and Gallant, J. (1988). On the mechanism of ribosomal frameshifting at hungry codons. J. Mol. Biol. 203, 403–410." />
<meta name="citation_reference" content="van Weringh, A., Ragonnet-Cronin, M., Pranckeviciene, E., Pavon-Eternod, M., Kleiman, L., and Xia, X. (2011). HIV-1 modulates the tRNA pool to improve translation efficiency. Mol. Biol. Evol. 28, 1827–1834." />
<meta name="citation_reference" content="Wills, N.M., Moore, B., Hammer, A., Gesteland, R.F., and Atkins, J.F. (2006). A functional −1 ribosomal frameshift signal in the human paraneoplastic Ma3 gene. J. Biol. Chem. 281, 7082–7088." />
<meta name="citation_reference" content="Wu, B., Zhang, H., Sun, R., Peng, S., Cooperman, B.S., Goldman, Y.E., and Chen, C. (2018). Translocation kinetics and structural dynamics of ribosomes are modulated by the conformational plasticity of downstream pseudoknots. Nucleic Acids Res. 46, 9736–9748." />
<meta name="citation_reference" content="Yan, S., Wen, J.-D., Bustamante, C., and Tinoco, I. (2015). Ribosome excursions during mRNA translocation mediate broad branching of frameshift pathways. Cell 160, 870–881." />
<meta name="citation_reference" content="Yordanova, M.M., Wu, C., Andreev, D.E., Sachs, M.S., and Atkins, J.F. (2015). A Nascent Peptide Signal Responsive to Endogenous Levels of Polyamines Acts to Stimulate Regulatory Frameshifting on Antizyme mRNA. J. Biol. Chem. 290, 17863–17878." />
<meta name="twitter:title" content="High-throughput interrogation of programmed ribosomal frameshifting in human cells" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting." />
<meta name="og-title" property="og:title" content="High-throughput interrogation of programmed ribosomal frameshifting in human cells" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/469692v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/469692v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/469692v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/469692v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/469692v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1194046" />
    <title>High-throughput interrogation of programmed ribosomal frameshifting in human cells | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"f2zm-N1qxu0p1oRPu9zDqTnfidUXpgjBPWJcVNGnI8s","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1194046","apath":"\/biorxiv\/early\/2020\/03\/13\/469692.atom","pisa":"biorxiv;469692v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;469692\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;469692\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;469692v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;469692\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;469692\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/469692v2","title":"High-throughput interrogation of programmed ribosomal frameshifting in human cells","identifier":"node\/1194046"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Dgu8ncj5q0YOrsLJaHWSLpYzSFwbg7DzeKzfLqNJzjvOXrALp3OFMZrOIreEqKkdXzQrbWejJ9RWWxQZ3VM+xVZIJn7lw6GZawWGMSREZEWJ+AAx4dp+AoIEXx0UV8FQuPAQBrkF76EPyj\\\\\\\/r5ufgjJJpMfCeSlI76FA5uhzMhiaNfVBsMDXFM4LsXOERY8DAmpcx6d2B4OBESmW7dkvcA2uvwB6OTJkZxOmS28rO9Bilw7JCUOmX5CWyWPdIk+Y4TG1mG35u1quMLX\\\\\\\/P4+MbUyDuHCy9MyFgn\\\\\\\/0JsCvVY7yImXKUNpEzZXvJNssZtEI85ZeLhAN\\\\\\\/3zUAKRzMhTcOnUIxLHpZRBzB0ssmGGBjaTnWSnCuOxwKSApRCGCT3cFc19jdBiz9uKyIWCQAXj3lcWtRZh+En2fFfny1SbK4lFZsfdp+aXb36zDI4O4qM9uvi+Ilgy3QnKgwCGK0zScJjVCrH5lPz1DD9w2HAzaJSuK8FPQlIYpHPDCeL9iEygHy3vUcXjDn42b9V4gXi9xOG21eCcsZDvTmQ2LThDw62LCgBAK55lB7ieoDNTyZi6SnnKa15nHu44MyslRrMAeadBZJhicIf2u00vqjLyhTkmiiTRMSRdpMighk6HFpG2kfZiMKYlFJmG6hYEQTh9HcW55VaY4o3tHEq8DabQ\\\\\\\/03ZTHWtCHOeIKpfxm8Gb0YBbUlIFSd4c\\\\\\\/Ajng\\\\\\\/5\\\\\\\/2ctkGV0Irx14mg6guoalx+2iEk\\\\\\\/1mlwbxbxB3JscriaI\\\\\\\/PzCqmiQDy23pP3yneK81Agea5pYqEcDLC+CVQ5Jy1Mr29r\\\\\\\/mASePnxPmbn1Ckx63TjvmicNjqqbzdb2LWIqJzdEQn3ziMDsDVaRiDBZKlEJ0Y8rFVSyML2LRcA4WxUUzEciv2nmc7X8YVY3fGdzokYuMX8Zll7X2I4jDzg1wuradOoHcvwiNpxc8V9FF7d4CT6\\\\\\\/EqEea6LuKo0sCR1\\\\\\\/L5TVShq91fnjMeJOWUCO5ofxvMTZrLXl5F8\\\\\\\/BOvcsrSIJhDy+g7y9l8XthydPaN73emnSAhnw44KqwFpeP\\\\\\\/Gjan8Ki5+\\\\\\\/GxSbZA8X\\\\\\\/rpPwbMui3YO9k5\\\\\\\/LVZIk5DSoWzWgMPSktafXyBQyzaizGmOJMOk4\\\\\\\/AxdFukywJAo3dYW0jMqNUhkH5Ebhi2TiPWF3JSe6vrZI5Dwy+OgckluiilVrbURFHYvKrTbJCmUnLGdOFYwmlHLnJNw3avJL1jMSdLS9i+6GirT5RzqiB8k1siB+VZjDbUA81yGVKpsi9Ex+uxIxxuUnscq\\\\\\\/gkqPUpzT1+2LvxMS51jNYb2Tp267PuQV6JnRXypzepXhpxXr9xCwUBBVtP\\\u0022,\\\u0022iv\\\u0022:\\\u0022k8+m+qHVEr7pYVdP8WuCaw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228298bb60f7f4aca47ac9a12788fe7059aa44dd3bc380ab2f5388b39bdc4f296a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/469692v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1194046 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/469692v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bcvX1ujJcnhjzaLqvkvGhIdqobv501vQLYbtsWZJbzc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1194046" id="node1194046" data-pisa="biorxiv;469692v2" data-pisa-master="biorxiv;469692" data-apath="/biorxiv/early/2020/03/13/469692.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">High-throughput interrogation of programmed ribosomal frameshifting in human cells</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Mikl</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yitzhak</span> <span class="nlm-surname">Pilpel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Eran</span> <span class="nlm-surname">Segal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/469692 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1194046" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Martin Mikl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Computer Science and Applied Mathematics, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Molecular Cell Biology, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Molecular Genetics, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BMikl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mikl%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BMikl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:eran.segal@weizmann.ac.il" class="" data-icon-position="" data-hide-link-title="0">eran.segal@weizmann.ac.il</a>
<a href="mailto:martin.mikl@weizmann.ac.il" class="" data-icon-position="" data-hide-link-title="0">martin.mikl@weizmann.ac.il</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yitzhak Pilpel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Molecular Genetics, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yitzhak%2BPilpel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pilpel%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYitzhak%2BPilpel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Eran Segal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Computer Science and Applied Mathematics, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Molecular Cell Biology, Weizmann Institute of Science</span>, Rehovot 7610001, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eran%2BSegal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segal%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEran%2BSegal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:eran.segal@weizmann.ac.il" class="" data-icon-position="" data-hide-link-title="0">eran.segal@weizmann.ac.il</a>
<a href="mailto:martin.mikl@weizmann.ac.il" class="" data-icon-position="" data-hide-link-title="0">martin.mikl@weizmann.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/469692v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/469692v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/469692v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/469692v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/469692v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/469692v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1194046" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1194046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-2">Programmed ribosomal frameshifting is the controlled slippage of the translating ribosome to an alternative frame. This tightly regulated process is widely employed by human viruses such as HIV and SARS coronavirus and is critical for their life cycle and virulence. It is also utilized from yeast to human to implement a feedback control mechanism to regulate polyamine levels. Here, we developed a high-throughput, fluorescence-based approach to assess the frameshifting potential of a sequence. We designed and tested &gt;12,000 sequences based on 15 viral and human frameshifting events, allowing us to elucidate the rules governing ribosomal frameshifting in a systematic way and to discover novel regulatory inputs based on amino acid properties and tRNA availability. We assessed the natural variation in HIV gag-pol frameshifting rates by testing &gt;500 clinical isolates and identified subtype-specific differences as well as associations between viral load in patients and the optimality of gag-pol frameshifting rates. We further devised computational models that accurately predict frameshifting potential (up to auROC=0.93) and frameshifting rates (up to Pearson r=0.81) of novel variants, including subtle differences between HIV clinical isolates (r=0.60). Taken together, this systematic approach can contribute to the development of antiviral agents acting on programmed ribosomal frameshifting.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Programmed ribosomal frameshifting (PRF), i.e. controlled slippage of the ribosome, is a mechanism by which two proteins with alternative C termini can be generated from the same mRNA. It allows for an expansion of the proteome but also constitutes an additional regulatory layer to fine-tune gene expression (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Advani and Dinman, 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dinman, 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ketteler, 2012</a>). This mechanism is widespread and indispensable in viruses, which often utilize controlled slippage of the ribosome to an alternative frame to regulate the production of key enzymes, such as in the case of the gag-pol frameshift in HIV and other retroviruses. Utilizing alternative frames increases the amount of genetic information that can be encoded in a given sequence and constitutes another level of gene regulation, which might explain why especially RNA viruses – with their compact genomes and lack of regulation of their genes on the transcriptional level - utilize PRF for crucial regulatory switches (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Firth and Brierley, 2012</a>). The importance of maintaining the stoichiometry between structural proteins encoded by the <em>gag</em> gene and enzymes encoded by the <em>pol</em> gene for viral replicative success make the gag-pol frameshifting event a promising antiviral drug target (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brakier-Gingras et al., 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hung et al., 1998</a>).</p><p id="p-5">Cases of functionally important programmed frameshifting have also been discovered in humans (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Belew et al., 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Clark et al., 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ivanov et al., 1998</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Matsufuji et al., 1995</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rom and Kahana, 1994</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tosaka et al., 2000</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wills et al., 2006</a>). Discovering PRF events in the human genome has been hampered by the limited amenability of PRF to proteome-wide methods due to the generally low abundance of the frameshifted protein relative to the canonical protein or inherent instability of the frameshifting product. Most human cases known to date were found serendipitously or through homologous genes. A striking example of regulatory conservation is the case of ornithine decarboxylase antizyme (OAZ), which is produced through polyamine-stimulated +1 frameshifting and inhibits polyamine production (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Matsufuji et al., 1995</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Rom and Kahana, 1994</a>). This negative feedback loop is used by virtually all organisms from yeast to humans to control polyamine levels (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ivanov et al., 2000</a>), attesting to the evolutionary success of PRF as a regulatory mechanism.</p><p id="p-6">Fameshifting is generally believed to happen at defined positions consisting of a slippery sequence and a downstream roadblock, most commonly a stable secondary RNA structure like a pseudoknot or an extensive stem-loop structure (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Caliskan et al., 2015</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Dinman, 2012</a>). Most presently known −1 slippery sites follow the pattern X XXY YYZ, with the shift happening from codons XXY and YYZ to XXX and YYY. Some known slippery sites in −1 PRF show divergence from this pattern, e.g. in the NSA2 gene of west nile virus with the slippery site sequence UCCUUUU (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Firth and Atkins, 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Melian et al., 2010</a>). At +1 frameshifting sites like OAZ ribosomal translocation happens at a very distinct motif (e.g. UCCUGA). Many case studies have contributed to an understanding of the molecular events happening during frameshifting (e.g. <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Belew et al., 2014</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Caliskan et al., 2015</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kurian et al., 2011</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Ritchie et al., 2017</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tholstrup et al., 2012</a>), but the general, overarching regulatory principles that determine if and to what extent PRF occurs remain largely unknown.</p><p id="p-7">Here, we developed a massively parallel reporter assay that allows for high-throughput quantification of ribosomal frameshifting in human cells. We designed and tested 13,390 oligonucleotides containing rationally designed variants of known frameshifting signals. We systematically deciphered determinants of PRF efficiency across frameshifting events and assayed natural variation in HIV gag-pol frameshifting, providing the first systematic large-scale investigation of ribosomal frameshifting.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>A massively parallel reporter assay for programmed ribosomal frameshifting</h3><p id="p-8">To assay PRF in a comprehensive manner we designed a synthetic oligonucleotide library containing 12809 variants with systematic sequence manipulations of previously reported PRF sites (Supplementary Data 1, “main set”) and 581 sequences of gag-pol frameshifting sites in HIV clinical isolates (collated from <a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>, see Methods for selection criteria of HIV variants) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A massively parallel reporter assay for programmed ribosomal frameshifting. A. Schematics of the library design; for: forward primer, rev: reverse primer, BC: barcode. B. Outline of the experimental pipeline; the plots in the bottom constitute representative bin profiles for variants with no (left), low (middle) or intermediate (right) GFP fluorescence. CD. Mean and 95% CI of &#x2212;1 (C) and +1 (D) PRF signal (% GFP fluorescence) for barcode control groups corresponding to the indicated wild-type PRF sequences (blue), carrying single nucleotide deletions (green) or having a stop codon inserted upstream of the frameshift sequence (red); n between 1 and 32; the shaded area denotes the range of background fluorescence; asterisks denote significant differences between the wild-type and the other groups (single nucleotide deletion and stop-control) combined (Mann-Whitney-U test). E. Comparison of &#x2212;1 and +1 PRF reporter readout (% GFP fluorescence) for previously reported PRF sites (cf. Fig 1CD, Fig S3AC)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;A massively parallel reporter assay for programmed ribosomal frameshifting.&lt;/span&gt; A. Schematics of the library design; for: forward primer, rev: reverse primer, BC: barcode. B. Outline of the experimental pipeline; the plots in the bottom constitute representative bin profiles for variants with no (left), low (middle) or intermediate (right) GFP fluorescence. CD. Mean and 95% CI of &#x2212;1 (C) and +1 (D) PRF signal (% GFP fluorescence) for barcode control groups corresponding to the indicated wild-type PRF sequences (blue), carrying single nucleotide deletions (green) or having a stop codon inserted upstream of the frameshift sequence (red); n between 1 and 32; the shaded area denotes the range of background fluorescence; asterisks denote significant differences between the wild-type and the other groups (single nucleotide deletion and stop-control) combined (Mann-Whitney-U test). E. Comparison of &#x2212;1 and +1 PRF reporter readout (% GFP fluorescence) for previously reported PRF sites (cf. Fig 1CD, Fig S3AC).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F1.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F1.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">A massively parallel reporter assay for programmed ribosomal frameshifting.</span><p id="p-9" class="first-child">A. Schematics of the library design; for: forward primer, rev: reverse primer, BC: barcode. B. Outline of the experimental pipeline; the plots in the bottom constitute representative bin profiles for variants with no (left), low (middle) or intermediate (right) GFP fluorescence. CD. Mean and 95% CI of −1 (C) and +1 (D) PRF signal (% GFP fluorescence) for barcode control groups corresponding to the indicated wild-type PRF sequences (blue), carrying single nucleotide deletions (green) or having a stop codon inserted upstream of the frameshift sequence (red); n between 1 and 32; the shaded area denotes the range of background fluorescence; asterisks denote significant differences between the wild-type and the other groups (single nucleotide deletion and stop-control) combined (Mann-Whitney-U test). E. Comparison of −1 and +1 PRF reporter readout (% GFP fluorescence) for previously reported PRF sites (cf. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1CD</a>, Fig S3AC).</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">The oligonucleotides comprising library-specific common primers, a unique barcode and a 162 nt long variable region containing a potential frameshifting site (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1A</a>, bottom) were synthesized on an Agilent microarray, amplified and cloned in between <em>mcherry</em> and <em>gfp</em> coding sequences such that the <em>gfp</em> coding frame was shifted by +1 or −1 relative to the original, mCherry-encoding frame (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1B</a>). If the corresponding frameshift occurs, GFP is made into protein, and GFP fluorescence intensity thus serves as a measure for frameshifting efficiency. We introduced this construct in the AAVS1 locus in the human K562 cell line using zinc finger nucleases, such that every cell has one frameshifting reporter construct from the library and all the variants have the same genomic environment (Methods). For both, the −1 and +1 reporter libraries, we selected a narrow mCherry-positive population to minimize effects coming from the influence of the variable region on overall expression levels (Fig S1AB, Methods). We sorted this population corresponding to a single integration of the reporter transgene using flow cytometry into 16 bins according to their GFP fluorescence intensity (Fig S1CD) and sequenced genomic DNA from all the bins to determine the distribution of each variant across bins. We previously demonstrated that similar approaches are highly accurate and reproducible (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Mikl et al., 2019</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vainberg Slutskin et al., 2018</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Weingarten-Gabbay et al., 2016</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">2019</a>), and the consistent bin profiles for many barcode control groups with identical variable region (Fig S1E, Methods) corroborate the low technical noise we are able to achieve.</p><p id="p-11">The distribution of mean GFP expression of all library variants showed a clear peak corresponding to background green fluorescence (Fig S2AB). We set the lowest mean GFP fluorescence we observed to 0% and the highest – coming from variants with GFP in frame with mCherry – to 100%. Accordingly, we assigned a percentage to every variant that passed filtering for read number, bin profile and expression levels (by gating for a narrow range of mCherry fluorescence to minimize effects coming from the influence of the variable region on overall expression levels, Methods). This percentage value does not denote the precise rate of frameshifting events, but gives us a meaningful measure of frameshifting efficiencies across PRF events. We thereby obtained measurements for 8972 (67%) and 5922 (44%) sequences cloned into the −1 PRF and +1 PRF reporter, respectively.</p><p id="p-12">To test the possibility that the signal in our assay comes from internal translation initiation downstream of mCherry (which might lead to GFP expression independently of ribosomal frameshifting in between the <em>mCherry</em> and <em>gfp</em> coding region) we cloned our library in a modified reporter construct containing a stop codon after the mCherry coding region (Fig S2C top). GFP fluorescence coming from a frameshifting event would in this case be lost. We subjected this library to the same experimental pipeline and obtained green fluorescence measurements for 10561 variants (79% of the library, 82% and 83% of variants for which we obtained −1 and +1 PRF readouts, resp.; Fig S2C). Green fluorescence was almost exclusively at background levels, showing that the overwhelming majority of tested sequences cannot drive GFP expression on their own.</p><p id="p-13">Another potential source of GFP expression is the removal of cryptic introns in the variable region that would lead to <em>gfp</em> being in frame with <em>mCherry</em> in the resulting mature mRNA. To test if this is a source of false positives in our assay, we performed RNA sequencing on the whole library, covering the entire variable region and its surroundings (Fig S2D). We mapped the full length reads to the library variants to detect splicing events and obtained an RNA readout (at least 100 reads mapped) for 10666 variants (80% of the library, 94% of the variants with readout in the −1 frame and 96% of the variants with readout in the +1 frame). Even without any filtering, only 2.95% of the variants showed gaps in the mapping of at least 20 nt which could represent events of cryptic splicing (Fig S2EF; 0.95% introducing a shift to frame +1 and 0.64% to frame −1). Missing stretches can also come from synthesis or cloning errors (and are therefore present and filtered out on the DNA level. Considering only those cases that had a potential (degenerate, cryptic) donor or acceptor splice site in the area of the gap further reduced the fraction of variants for which we cannot rule out a relevant splicing event to 36 (0.55%; Fig S2EF, red; 0.22% introducing a shift to frame +1 and 0.14% to frame −1). Importantly, even for these cases there was no correlation between the fraction of potentially spliced reads and GFP expression (Pearson r=0.01, p=0.21 for −1 PRF and r=0.02, p=0.09 for +1 PRF), indicating that cryptic splicing events do not constitute a common source for false positive signal in our assay.</p><p id="p-14">During oligonucleotide synthesis errors can occur that can introduce a point mutation or an insertion/deletion. For assaying ribosomal frameshifting, especially the latter type would be detrimental, and therefore we perform full-length DNA sequencing in our assay and consider only reads that exhibit perfect alignment along the length of the variable region covered by sequencing reads (approximately 20 nt in the center of the variable region cannot be covered due to the read length (150 nt from either side)). These synthesis errors, however, also further enrich our synthetic library, as they add additional variants to the collection of sequences tested. Especially single nucleotide deletions or insertions constitute a great internal control, as they differ only in one position from the designed variant. Cases where we obtained enough reads (after filtering, Methods) for the intended variant (Fig S2G top, “wild-type”), a single nucleotide deletion abolishing the signal coming from −1 PRF (Fig S2G middle) and a single nucleotide insertion leading to <em>gfp</em> being in frame (Fig S2G bottom) constitute a proof of principle. They also provide additional evidence that the signal that we measure stems from a (reading frame-sensitive) ribosomal frameshifting event. Looking at the distribution of all single nucleotide deletion and insertion variants from the library reveals the expected peaks at minimal and maximal GFP expression (Fig Fig S2HI), with a small group of variants being misclassified due to the fact that our reads cover only approximately 90% of the variable region.</p><p id="p-15">Based on the distribution of all library variants with a stop codon cassette integrated immediately after the <em>mcherry</em> coding sequence we assigned a noise threshold (95<sup>th</sup> percentile of the stop-vector library distribution, corresponding to 1.3% of the maximal GFP fluorescence; Fig S2C) imposed by autofluorescence of the cells. The contribution of autofluorescence coming from cellular components and metabolites such as NADH and flavins is negligible when methods for signal amplification are used (e.g. in Luciferase assays commonly used for the quantification of PRF). It does, however, constitute a limitation for detection of GFP-fusion proteins made from a single gene copy and present at low copy number in the cell owing to the fact that the PRF event responsible for its expression happens in only a few percent of translation events. This entails a higher detection limit of our assay compared to for example Luciferase assays, which is a necessary compromise to enable high-throughput testing.</p><p id="p-16">Based on the distribution of GFP intensities of variants with a mutation leading to GFP being in frame with mCherry (Fig S2I) we also excluded variants exhibiting a dominant peak with mean GFP fluorescence above 2<sup>12</sup> (25% of maximal GFP fluorescence). This value corresponds to the 5<sup>th</sup> percentile of the distribution of variants with a single nucleotide insertion (above threshold). We thereby aim to rule out biases stemming from DNA frameshifts occurring as synthesis or cloning errors that our mapping strategy might have missed (Methods).</p><p id="p-17">We tested previously reported frameshifting sites (with or without experimental validation, Supplementary Data 1, based on <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Moon et al., 2007</a> and our own survey of the literature) with multiple different barcodes in our assay and could reproducibly detect GFP fluorescence in the expected frame for many of the previously reported PRF events (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig 1CD</a>, Fig S3A-D). For example, the previously reported PRF sites in HIV-1 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jacks et al., 1988a</a>), SARS coronavirus (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Thiel et al., 2003</a>), HERV-K10 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Ono et al., 1986</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wang et al., 2002</a>), simian retrovirus (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">ten Dam et al., 1994</a>), SIVmac239 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Marcheschi et al., 2007</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Pollom et al., 2013</a>), human T cell lymphotropic virus (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Nam et al., 1993</a>) and OAZ (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Matsufuji et al., 1995</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Rom and Kahana, 1994</a>) yielded a fluorescent signal between 1.4% and 18% of maximal GFP fluorescence detected in our assay. These values do not necessarily denote the percentage of translation events, in which frameshifting happens, in absolute terms, and are not directly comparable with percentages reported in other studies.</p><p id="p-18">PRF events tested in our assay typically gave a signal over threshold only in either the −1 or +1 PRF reporter, but not in both (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig 1E</a>). In cases where the native sequence contained stop codons in any frame downstream of the frameshifting site, we also tested variants with all stop codons mutated (Methods). These typically either did not affect or led to a substantially reduced frameshifting signal (Fig S4A). Moving the slippery site by one to three nucleotides typically abolished the signal, e.g. in the case of SARS (Fig S4B, Methods).</p><p id="p-19">Some previously reported PRF events failed to yield a signal above threshold in our assay. Several reasons like sensitivity to the larger context of the reporter construct or a general effect of the tested sequence on expression could be possible explanations. In the case of the PRF site in Rous sarcoma virus (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jacks and Varmus, 1985</a>; Fig S4C), a construct containing 124 nt of sequence downstream to the position of frameshifting was able to induce frameshifting, but shorter versions were not (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jacks et al., 1988b</a>). A sequence motif (termed PK4) at the end of this downstream region was shown to be important for maintaining wild-type frameshifting ability, potentially by creating a pseudoknot-like structure through basepairing with a complementary sequence in the loop of the main stem-loop structure (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Marczinke et al., 1998</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Nikolic et al., 2012</a>). Our construct (including 120 nt of downstream sequence) does contain the PK4 motif, but it is located at the very 3’ end of the native sequence included in the reporter construct. The sequence context of PK4 was shown to be important for proper folding (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Marczinke et al., 1998</a>), and therefore the specific context created by our reporter construct might explain the lack of signal.</p><p id="p-20">We also failed to detect a frameshifting signal for CCR5 (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Belew et al., 2014</a>; Fig S4C). In addition to potential sensitivity to the larger context of the reporter construct and a specific miRNA that is required for maximal frameshifting efficiency (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Belew et al., 2014</a>), the lack of signal might be due to a potential effect of the PRF site on overall expression levels on the RNA or protein level, as has been suggested for cellular frameshifting sites in general (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Advani and Dinman, 2016</a>). We select for a narrow range of mCherry levels and therefore expression levels (Methods, Fig S1AB), which leads to filtering out variants that strongly affect overall expression levels. In the case of CCR5 only 29% of variants with the wild-type sequence passed filtering (Fig S4D; as opposed to 45-97% for other PRF events), which could hint at an effect of the included sequence on expression at the RNA or protein level. Estimating steady state RNA levels did not show systematic differences between PRF events yielding a signal above threshold in our assay and those that failed to do so (Fig S4E), but indicated that specifically RNA from variants containing the PRRSV PRF site was present at reduced levels.</p><p id="p-21">Some of the frameshifting signals tested showed variability between variants with identical variants, but different barcodes (Fig S4C, right). In the case of west nile virus, the majority of sequences tested resulted in a fluorescent signal above threshold, leading to it being classified as “frameshifting” in our assay (p=0.027 for the difference between wild-type and stop control; <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig 1C</a>); in the case of PEG10 (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Clark et al., 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Manktelow et al., 2005</a>) the majority of sequences failed to yield a signal (after filtering). This points at an instability of the signal in this context, and therefore our assay might not be suitable for investigating these specific events. PEG10 is thought to have a relatively extensive downstream region necessary for frameshifting (∼80 nt according to estimates from in vitro testing of truncated regions (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Manktelow et al., 2005</a>)). Our library includes 120 nt of downstream sequence, but the more extensive the downstream sequence, the higher the sensibility to the context and experimental conditions might be. This is in line with our observation for SARS and HERV-K10, for example (see below). In addition, the required context might be larger in the <em>in vivo</em> situation compared to <em>in vitro</em>.</p><p id="p-22">This instability of the signal might be affected by small sequence changes: For some library sequences measurements of a large number of variants with sequence alterations introduced during synthesis and cloning are available. In the case of PEG10, all the synthesis variants containing a specific mutation (C-to-G at position 7 after the slippery site) show a −1 PRF signal above background (Fig S4F). Likewise, a fraction of the designed sequence alterations introduced in the PEG10 context triggered frameshifting (Fig S4G), especially mutations in the upstream region (Fig S4G left). This shows that minimal sequence changes can have far-reaching (and consistent) consequences for the potential of a sequence to yield a frameshifting signal in the context of our high-throughput assay and probably in any experimental setup.</p></div><div id="sec-4" class="subsection"><h3>Slippery sites are optimized for the respective frameshifting event</h3><p id="p-23">While previous investigations of frameshifting focused on individual examples, we aimed to identify commonalities and differences between frameshifting sites. To this end we introduced systematic sequence alterations (such as point mutations, synonymous substitutions, altering and mimicking endogenous secondary structure, etc.; <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig 1A</a>; Methods) in all contexts and measured the effect on frameshifting in large scale using our FACS based assay. Individual PRF events exhibited characteristic sensitivities to sequence alterations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>). The HIV-1 gag-pol PRF site (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Jacks et al., 1988a</a>) was particularly sensitive to changes in or immediately around the slippery site (any sequence change in this region led to a median PRF signal of 25% of the wild-type), while changes in the upstream or downstream region led to PRF signal of – on average – 97% (upstream) and 70% (downstream; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2A</a>, Fig S5A) of the corresponding wild-type. A similar pattern could be observed for SIVmac239. The HERV-K10 (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Ono et al., 1986</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Wang et al., 2002</a>), SARS (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Thiel et al., 2003</a>) and SRV1 (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">ten Dam et al., 1994</a>) PRF sites were highly sensitive to changes in both the slippery site and the downstream region and in the case of HERV-K10 and SRV1 also showed a significant (p=0.0004 and 0.0023, resp.; Wilcoxon signed-rank test) reduction following changes in the upstream region (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2A</a>, Fig S5B). It is noteworthy that the clustering of effects (although done on percent of wild-type PRF signal) resembles the wild-type PRF efficiencies (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig 1C</a>), with the most efficient sites (HERV-K10, SARS and SRV1) showing the highest and broadest sensitivity to sequence alterations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Slippery sites are optimized for the respective frameshifting event. A. Clustered heat map of median percent wild-type frameshifting of variants in which the indicated region (upstream, slippery site (including the preceding codon) and downstream) differs from the native one (n=37-325). B. The median percent wild-type PRF signal of variants in which the indicated position is changed is plotted for the entire length of the variable region; gray box: slippery site. C. Percent wild-type &#x2212;1 PRF signal for variants with a point mutation only in the indicated position of the slippery site (n=6-22). D. Heatmap showing median percent GFP fluorescence conferred by replacing the native slippery site in &#x2212;1 PRF sites tested positive in the assay (Fig 1C) with the indicated combination of slippery site elements. E. Distribution of percent wild-type GFP fluorescence of variants in which the slippery site was replaced by any possible combination of bases following the pattern XXXYYYZ (n=19-80). F. Distribution of normalized reads across GFP (+1 frame) expression bins for variants of the OAZ1 frameshifting site, in which either the first (right) or the second (left) codon is replaced; blue lines: wild-type; green and magenta lines: bin profile for the variant in which the native codon was replaced by the indicated one; grey lines: bin profiles for other codons at this position. G. Percent GFP fluorescence for variants in which the slippery site was replace with all possible combinations of the slippery site pattern found in west nile virus (XYYZZZZ) (n=16-56)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Slippery sites are optimized for the respective frameshifting event.&lt;/span&gt; A. Clustered heat map of median percent wild-type frameshifting of variants in which the indicated region (upstream, slippery site (including the preceding codon) and downstream) differs from the native one (n=37-325). B. The median percent wild-type PRF signal of variants in which the indicated position is changed is plotted for the entire length of the variable region; gray box: slippery site. C. Percent wild-type &#x2212;1 PRF signal for variants with a point mutation only in the indicated position of the slippery site (n=6-22). D. Heatmap showing median percent GFP fluorescence conferred by replacing the native slippery site in &#x2212;1 PRF sites tested positive in the assay (Fig 1C) with the indicated combination of slippery site elements. E. Distribution of percent wild-type GFP fluorescence of variants in which the slippery site was replaced by any possible combination of bases following the pattern XXXYYYZ (n=19-80). F. Distribution of normalized reads across GFP (+1 frame) expression bins for variants of the OAZ1 frameshifting site, in which either the first (right) or the second (left) codon is replaced; blue lines: wild-type; green and magenta lines: bin profile for the variant in which the native codon was replaced by the indicated one; grey lines: bin profiles for other codons at this position. G. Percent GFP fluorescence for variants in which the slippery site was replace with all possible combinations of the slippery site pattern found in west nile virus (XYYZZZZ) (n=16-56).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F2.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F2.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Slippery sites are optimized for the respective frameshifting event.</span><p id="p-24" class="first-child">A. Clustered heat map of median percent wild-type frameshifting of variants in which the indicated region (upstream, slippery site (including the preceding codon) and downstream) differs from the native one (n=37-325). B. The median percent wild-type PRF signal of variants in which the indicated position is changed is plotted for the entire length of the variable region; gray box: slippery site. C. Percent wild-type −1 PRF signal for variants with a point mutation only in the indicated position of the slippery site (n=6-22). D. Heatmap showing median percent GFP fluorescence conferred by replacing the native slippery site in −1 PRF sites tested positive in the assay (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig 1C</a>) with the indicated combination of slippery site elements. E. Distribution of percent wild-type GFP fluorescence of variants in which the slippery site was replaced by any possible combination of bases following the pattern XXXYYYZ (n=19-80). F. Distribution of normalized reads across GFP (+1 frame) expression bins for variants of the OAZ1 frameshifting site, in which either the first (right) or the second (left) codon is replaced; blue lines: wild-type; green and magenta lines: bin profile for the variant in which the native codon was replaced by the indicated one; grey lines: bin profiles for other codons at this position. G. Percent GFP fluorescence for variants in which the slippery site was replace with all possible combinations of the slippery site pattern found in west nile virus (XYYZZZZ) (n=16-56).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Comparing per position the median PRF signal for variants in which this position was changed reveals a sensitivity profile around the PRF site (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2B</a>, Fig S5C), recapitulating these context specific characteristics. Across all −1 PRF events tested positive in our assay, not only mutations at the slippery site (marked in grey) and downstream of it, but also mutations immediately upstream of the slippery site negatively affect PRF efficiency. Other −1 PRF events reported in the literature, but not yielding a signal above background in our assay, tend to show substantially increased signal upon mutations in the areas upstream (PEG10) and downstream (PEG10, CCR5, Rous sarcoma virus) of the slippery site (Fig S5D). Like the specific examples of frameshifting PEG10 variants described above (Fig S4EF), this points at a strong sensitivity of these PRF events to the context in which they are tested.</p><p id="p-26">Most single point mutations within the slippery sequence abrogated frameshifting and resulted in green fluorescence at background levels, both for −1 and +1 frameshifting sites (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2C</a>, Fig S5E), demonstrating that we indeed measure frameshifting at the expected site. Replacing the last position in the canonical slippery site X XXY YYZ had – on average – a less severe effect on PRF efficiency, due to the fact that it is the wobble position in the original frame and therefore has larger flexibility (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2C</a>). In high efficiency PRF sites also this position showed strong and PRF site-specific preferences: In the case of SRV1 (GGGAAAC) replacing the Z position with other options than the endogenous C led to a drastic decrease (U, A) or complete loss (G) of −1 PRF signal (Fig S4E), while in the case of SARS (UUUAAAU) all other nucleotides apart from the endogenous U at the Z position almost completely abolished the signal. To determine whether there is a universally optimal slippery site, we replaced the slippery sequence in our set of −1 PRF events with all possible variations of the pattern XXXYYYZ (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 2D</a>, Fig S4F). This revealed preferences for specific combinations common in known PRF sites (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig 2D</a>), like UUUUUUZ (HIV, SIVmac239) and XXXAAAC (HERV-K10, HTLV, PEG10, SARS), but no universally optimal combination of bases that would lead to maximal frameshifting efficiency and consequently GFP fluorescence across contexts (Fig S5F). Notably, PRF sites showed a high degree of optimization when it comes to the identity of the bases, with any substitution (although preserving the canonical XXXYYYZ pattern) typically leading to a more or less severe drop in PRF signal (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig 2E</a>, Fig S5EF).</p><p id="p-27">Replacing the first or second codon of the OAZ slippery site (UCC UGA) revealed a very limited tolerance to sequence alterations, consistent with earlier results (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Matsufuji et al., 1995</a>). Replacing the second codon (UGA), the only variants leading to a +1 PRF signal over background where the other two stop codons (UAA and UAG; <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig 2F</a>, right), reaching approximately 50% of wild-type frameshifting rates, confirming that a translation termination signal is required for frameshifting (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Rom and Kahana, 1994</a>). At the first codon position, UUU was the only codon leading to +1 PRF signal similar to wild-type (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig 2F</a>, left), maybe due to the creation of a stretch of four Us and thereby facilitating re-pairing of the tRNA in the +1 frame. Indeed, UUU is the second most widely used codon found in that position in the OAZ frameshifting site in other species, e.g. certain funghi and nematodes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ivanov and Atkins, 2007</a>).</p><p id="p-28">Some reported −1 PRF sites deviate from the canonical slippery site pattern (XXXYYYZ). To test the ability of the alternative patterns to induce frameshifting across sequence contexts we replaced the slippery site in a set of PRF events with all combinations of nucleotides following the pattern found in west nile virus (XYYZZZZ, <a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig 2G</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Melian et al., 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Moomau et al., 2016</a>). In general, the ability of alternative slippery site patterns to cause frameshifting was limited to their native sequence contexts. In this context, however, several different nucleotide combinations following the respective pattern were tolerated. The only exception to this rule was human T-lymphotropic virus (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig 2G</a>, endogenous slippery site UUUAAAC), which gave a robust −1 PRF signal corresponding to wild-type levels across many variants following the (west nile virus) pattern XYYZZZZ. In summary, our data suggest that in most cases a slippery site not following the canonical pattern requires a specific sequence context in order to trigger frameshifting, in accordance with earlier results comparing west nile virus strains with canonical or non-canonical slippery sites (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Moomau et al., 2016</a>).</p></div><div id="sec-5" class="subsection"><h3>Downstream sequence and structure affect PRF efficiency in a context-dependent manner</h3><p id="p-29">The region downstream of the slippery site is thought to be crucial for PRF, typically because it creates a roadblock for the translating ribosome by folding into a stable secondary structure like a hairpin or a pseudoknot (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brierley and Pennell, 2001</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brierley et al., 1989</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Parkin et al., 1992</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tu et al., 1992</a>). To get an estimate of the size of the relevant downstream region in different PRF events, we replaced native regions with a constant sequence, leaving stretches of different length after the slippery site unchanged (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>, top; Methods). While in the case of SARS almost the entire downstream region included in our reporter construct (120 nt) was crucial for frameshifting, in the HIV and SIVmac239 PRF sites only a comparatively small region (&lt;24 nt) was required to achieve PRF signal &gt;50% of wild type (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3B</a>). The shortness of the necessary downstream sequence in these cases was striking, but in line with earlier evidence demonstrating that base pairing at the first 3-4 positions of the HIV downstream stem-loop structure shows the strongest association with PRF efficiency (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Mouzakis et al., 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Downstream RNA structure affects PRF efficiency in a context-dependent manner. A. Schematic of downstream sequence manipulations. B. Percent wild-type frameshifting rates of variants in which the native downstream region has been replaced by constant sequences, leaving the indicated number of nucleotides after the slippery site unchanged; n=1-3 per data point. C. Fraction of point mutations in the 40 nt downstream of the slippery site resulting in " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Downstream RNA structure affects PRF efficiency in a context-dependent manner.&lt;/span&gt; A. Schematic of downstream sequence manipulations. B. Percent wild-type frameshifting rates of variants in which the native downstream region has been replaced by constant sequences, leaving the indicated number of nucleotides after the slippery site unchanged; n=1-3 per data point. C. Fraction of point mutations in the 40 nt downstream of the slippery site resulting in &lt;50% of wild-type GFP fluorescence (green) and background fluorescence (&lt;1.3% GFP fluorescence, blue), for &#x2212;1 (top) and +1 (bottom) PRF events. D. The difference in mean % wild-type frameshifting between variants in which the indicated position is predicted to be paired vs unpaired along the variable region; gray box: slippery site. E. Schematic of downstream structure variations. F. Percent &#x2212;1 GFP fluorescence of variants in which the downstream region is replaced by a synthetic sequence predicted to fold into the respective native secondary structure (using Vienna RNA (only stem-loop; light blue) or pKiss (including pseudoknots; dark blue; Methods)); the shaded area denotes the range of background fluorescence. G. % &#x2212;1 GFP fluorescence of variants in which the downstream region is replaced by a synthetic sequence predicted to fold into the HIV (blue) or SRV1 (green) secondary structure (or variations thereof; Methods); the shaded area denotes the range of background fluorescence. H. % GFP fluorescence of variants in which the downstream region carried a single point mutation (left) or was replaced with sequences predicted to fold into variations of PRF inducing structures, plotted against the corresponding wild-type values; p-values (Mann-Whitney-U) for comparisons between groups of low (&lt;4% GFP fluorescence) and high wild-type PRF (&gt;4% GFP fluorescence). IJ. Clustered heatmap of Pearson correlation coefficients between percent &#x2212;1 GFP fluorescence and minimum free energy of regions after the slippery site (computed using Vienna RNAfold on the 40 nt following the slippery site (I) or pKiss on the 120 nt following the slippery site (J)) for groups of variants of the indicated PRF sites; values denote the p-values of the correlations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F3.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F3.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Downstream RNA structure affects PRF efficiency in a context-dependent manner.</span><p id="p-30" class="first-child">A. Schematic of downstream sequence manipulations. B. Percent wild-type frameshifting rates of variants in which the native downstream region has been replaced by constant sequences, leaving the indicated number of nucleotides after the slippery site unchanged; n=1-3 per data point. C. Fraction of point mutations in the 40 nt downstream of the slippery site resulting in &lt;50% of wild-type GFP fluorescence (green) and background fluorescence (&lt;1.3% GFP fluorescence, blue), for −1 (top) and +1 (bottom) PRF events. D. The difference in mean % wild-type frameshifting between variants in which the indicated position is predicted to be paired vs unpaired along the variable region; gray box: slippery site. E. Schematic of downstream structure variations. F. Percent −1 GFP fluorescence of variants in which the downstream region is replaced by a synthetic sequence predicted to fold into the respective native secondary structure (using Vienna RNA (only stem-loop; light blue) or pKiss (including pseudoknots; dark blue; Methods)); the shaded area denotes the range of background fluorescence. G. % −1 GFP fluorescence of variants in which the downstream region is replaced by a synthetic sequence predicted to fold into the HIV (blue) or SRV1 (green) secondary structure (or variations thereof; Methods); the shaded area denotes the range of background fluorescence. H. % GFP fluorescence of variants in which the downstream region carried a single point mutation (left) or was replaced with sequences predicted to fold into variations of PRF inducing structures, plotted against the corresponding wild-type values; p-values (Mann-Whitney-U) for comparisons between groups of low (&lt;4% GFP fluorescence) and high wild-type PRF (&gt;4% GFP fluorescence). IJ. Clustered heatmap of Pearson correlation coefficients between percent −1 GFP fluorescence and minimum free energy of regions after the slippery site (computed using Vienna RNAfold on the 40 nt following the slippery site (I) or pKiss on the 120 nt following the slippery site (J)) for groups of variants of the indicated PRF sites; values denote the p-values of the correlations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">To identify specific positions crucial for PRF we performed scanning mutagenesis of regions ranging from 7 to 36 nt downstream of the slippery site (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3A</a>, bottom). This revealed large differences in the fraction of relevant downstream positions (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3C</a>), ranging from 6% of single point mutations reducing −1 PRF efficiency by more than half in the case of SIVmac239 to 53% in the case of SARS (46% for +1 PRF at the OAZ1 site). The HIV gag-pol PRF site (among others) shows remarkable resilience to point mutations (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3C</a>, Fig S6A), in line with the high degree of genetic variability in HIV. Specific mutations affecting PRF typically disrupt base-pairing (e.g. positions 11, 12, 29 and 30 in HIV and positions 8-13 and 18-26 in SRV1; Fig S6A) and therefore change the (predicted) secondary structure. In the case of HIV only mutations in the middle of the stem are detrimental, suggesting that a reduced stem is still able to drive PRF at wild-type levels, and only forcing a complete change in the structure affects frameshifting.</p><p id="p-32">We tested if we can detect these mutational signatures of secondary structure also on the scale of the entire library and compared the median PRF signal between all variants of a PRF event, in which this position is predicted to be paired or unpaired (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig 3D</a>). Preferences for downstream positions to be paired or unpaired reveal the architecture of structural elements downstream of the PRF site. Groups of frameshifting events show remarkable concordance between these preferences (e.g. HIV and SARS, HERV-K10 and SRV1), but length and position of the optimal downstream secondary structures differ between groups (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig 3D</a>, upper vs. lower panel).</p><p id="p-33">Replacing the endogenous downstream region with elements that have different primary sequence, but are predicted to fold into the same secondary structure (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig 3E</a>, top) abolished frameshifting in most cases, regardless of whether pseudoknots were included or not (Vienna RNA vs. pKiss to determine the structure that served as the input for antaRNA (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kleinkauf et al., 2015a</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2015b</a>); <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig 3F</a>). Introducing variants of the SRV1 and the HIV downstream structure (without preserving the original sequence, <a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig 3E</a>, bottom; Methods) had the ability to trigger frameshifting in some cases (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig 3G</a>), including ones where the wild-type sequence did not yield a frameshifting signal in the context of our assay (CCR5 and PEG10). Strikingly, events associated with higher wild-type frameshifting rates could not be “rescued” by introducing a secondary structure variant from a different PRF event (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig 3H</a>), indicating that more efficient frameshifting sites like SARS seem to be highly optimized, but less tolerant to changes in the type of downstream secondary structure.</p><p id="p-34">Changes in secondary structure (minimum free energy) induced by point mutations downstream of the slippery site showed strong correlation with most, but not all frameshifting events (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig 3I</a> left column, Fig S7 left panels). Surprisingly, this correlation between predicted minimum free energy of secondary structure variants of the native downstream region and GFP fluorescence of −1 PRF reporters was lost when replacing the downstream region with structural variants (more) different from the native sequence (<a id="xref-fig-3-14" class="xref-fig" href="#F3">Fig 3I</a> middle column, Fig S7 middle panels; these structures generally had the potential to induce frameshifting in certain contexts; <a id="xref-fig-3-15" class="xref-fig" href="#F3">Fig 3G</a>). Likewise when testing for correlation across all variants of a PRF site (<a id="xref-fig-3-16" class="xref-fig" href="#F3">Fig 3I</a> right column, Fig S7 right panels), there was no association of lower minimum free energy (MFE) with higher PRF rates.</p><p id="p-35">Many stimulatory downstream sequences are thought to fold into a pseudoknot (e.g. SRV-1 (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">ten Dam et al., 1994</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Du et al., 1997</a>), west nile virus (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Firth and Atkins, 2009</a>) and coronaviruses (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brierley et al., 1989</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Plant et al., 2005</a>)), a more complex folding pattern that is not covered by many secondary structure predictions including the Vienna RNA package (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hofacker et al., 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lorenz et al., 2011</a>). We therefore repeated the analyses using pKiss (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Janssen and Giegerich, 2015</a>), an algorithm that can detect certain classes of pseudoknots (see Supplementary Data 2 for pKiss-predicted structures of library variants with systematic sequence changes in the downstream region). MFE calculated using Vienna RNAfold and pKiss, respectively, show good correlation (Fig S6B). We repeated the above analysis and observed the same pattern as with RNAfold, namely widespread association between secondary structure (low MFE) and higher PRF signal for small deviations from the native sequence (<a id="xref-fig-3-17" class="xref-fig" href="#F3">Fig 3J</a> left column, Fig S8 left panels), but not otherwise (<a id="xref-fig-3-18" class="xref-fig" href="#F3">Fig 3J</a> middle and right column, Fig S8 middle and right panels). This is in line with the fact that we do not see systematic differences between PRF events whose downstream sequence was shown to fold into a hairpin structure (e.g. HIV-1 (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Parkin et al., 1992</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Staple and Butcher, 2003</a>) and SIVmac239 (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Marcheschi et al., 2007</a>)) or a pseudoknot (e.g. SRV-1 (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">ten Dam et al., 1994</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Du et al., 1997</a>) and SARS (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Brierley et al., 1989</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Plant et al., 2005</a>)).</p><p id="p-36">This observation does not imply a (counterintuitive) tendency for more open secondary structure to promote frameshifting. As we do not find evidence for an effect of MFE on overall RNA levels (Fig S7, S8, S9; e.g. through triggering No-Go decay (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Belew et al., 2011</a>)), we suggest that this reflects a strong preference for downstream secondary structure rigidity to be close to that of the wild-type sequence (in particular for SARS, SRV1 and HERV-K10, Fig S7, S8, right panels). Although present secondary structure prediction algorithms might not be sufficiently accurate for this task, especially in the case of pseudoknots, our results support a view according to which not only the structure, but also the sequence downstream of a slippery site is critical for triggering frameshifting.</p></div><div id="sec-6" class="subsection"><h3>Additional regulatory elements contribute to PRF</h3><p id="p-37">The distance between the slippery site and the downstream secondary structure has been shown to be important for efficient frameshifting (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Brierley et al., 1989</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kollmus et al., 1994</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lin et al., 2012</a>), probably to position the paused ribosome at the slippery site (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brierley and Pennell, 2001</a>). To test whether this is generally true or if there are PRF event-specific differences, we introduced between one and nine nucleotides from different constant sequences or deleted between one and six, mutating downstream stop codons as required (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a> top). Most insertions or deletions led to substantially reduced frameshifting signal (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a>), no matter what the context and if the original downstream coding frame was preserved or not, suggesting that the exact position of the downstream regulatory elements is important across PRF events. This is in contrast to an earlier study reporting unchanged or even dramatically increased −1 frameshifting efficiency at the HIV site upon reducing spacer lengths, which the authors attribute to a change in local secondary structure triggered by the specific deletion introduced in their construct (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Mouzakis et al., 2013</a>). Overall, our results are in line with the regulatory elements being in optimal distance from one another and the downstream secondary structure creating a roadblock that causes the ribosome to pause exactly at the slippery site (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Brierley and Pennell, 2001</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Tu et al., 1992</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Additional regulatory elements contribute to PRF. A. Schematic of sequence manipulations upstream and immediately downstream of the slippery site. B. Boxplots of % wild-type frameshifting rates for variants of &#x201C;positive&#x201D; &#x2212;1 PRF sites in which the indicated number of nucleotides were deleted (negative numbers) or inserted (positive numbers, taken from two different constant sequences) after the slippery site (n=4-54). C. % GFP fluorescence for variants of &#x201C;negative&#x201D; &#x2212;1 PRF sites with the sequence upstream of the slippery site being randomly recoded or replaced with sequences predicted to have the indicated secondary structure; the shaded area denotes the range of background fluorescence. D. % wild-type frameshifting rates (+1 for OAZ, blue, and &#x2212;1 for HIV, green) of variants in which the native upstream region has been replaced by constant sequences up to the indicated position relative to the PRF site. E. Boxplot of percent of wild-type frameshifting rates for variants in which the &#x2212;2 or - 1 amino acid relative to the slippery site is replaced with an amino acid from the indicated groups, for codons which maintain the slippery site pattern XXXYYYZ (bottom) or not (top); n=10-115. F. Pearson correlation coefficient (blue) and associated p-values (green) between tAI at the indicated position of the original reading frame and % GFP fluorescence. G. Clustered heat map showing all possible combinations of 3 synthetic slippery sites, 34 synthetic downstream variants and up to 5 synthetic upstream regions (minimal value 1.3%)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Additional regulatory elements contribute to PRF.&lt;/span&gt; A. Schematic of sequence manipulations upstream and immediately downstream of the slippery site. B. Boxplots of % wild-type frameshifting rates for variants of &#x201C;positive&#x201D; &#x2212;1 PRF sites in which the indicated number of nucleotides were deleted (negative numbers) or inserted (positive numbers, taken from two different constant sequences) after the slippery site (n=4-54). C. % GFP fluorescence for variants of &#x201C;negative&#x201D; &#x2212;1 PRF sites with the sequence upstream of the slippery site being randomly recoded or replaced with sequences predicted to have the indicated secondary structure; the shaded area denotes the range of background fluorescence. D. % wild-type frameshifting rates (+1 for OAZ, blue, and &#x2212;1 for HIV, green) of variants in which the native upstream region has been replaced by constant sequences up to the indicated position relative to the PRF site. E. Boxplot of percent of wild-type frameshifting rates for variants in which the &#x2212;2 or - 1 amino acid relative to the slippery site is replaced with an amino acid from the indicated groups, for codons which maintain the slippery site pattern XXXYYYZ (bottom) or not (top); n=10-115. F. Pearson correlation coefficient (blue) and associated p-values (green) between tAI at the indicated position of the original reading frame and % GFP fluorescence. G. Clustered heat map showing all possible combinations of 3 synthetic slippery sites, 34 synthetic downstream variants and up to 5 synthetic upstream regions (minimal value 1.3%).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F4.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F4.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Additional regulatory elements contribute to PRF.</span><p id="p-38" class="first-child">A. Schematic of sequence manipulations upstream and immediately downstream of the slippery site. B. Boxplots of % wild-type frameshifting rates for variants of “positive” −1 PRF sites in which the indicated number of nucleotides were deleted (negative numbers) or inserted (positive numbers, taken from two different constant sequences) after the slippery site (n=4-54). C. % GFP fluorescence for variants of “negative” −1 PRF sites with the sequence upstream of the slippery site being randomly recoded or replaced with sequences predicted to have the indicated secondary structure; the shaded area denotes the range of background fluorescence. D. % wild-type frameshifting rates (+1 for OAZ, blue, and −1 for HIV, green) of variants in which the native upstream region has been replaced by constant sequences up to the indicated position relative to the PRF site. E. Boxplot of percent of wild-type frameshifting rates for variants in which the −2 or - 1 amino acid relative to the slippery site is replaced with an amino acid from the indicated groups, for codons which maintain the slippery site pattern XXXYYYZ (bottom) or not (top); n=10-115. F. Pearson correlation coefficient (blue) and associated p-values (green) between tAI at the indicated position of the original reading frame and % GFP fluorescence. G. Clustered heat map showing all possible combinations of 3 synthetic slippery sites, 34 synthetic downstream variants and up to 5 synthetic upstream regions (minimal value 1.3%).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">While mutations in the slippery site and the downstream region typically have a negative effect on frameshifting efficiency (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig 2A</a>), changes in the upstream region have the potential to lead to higher −1 PRF signal (e.g. PEG10, Fig S5D). This suggests that inhibitory signals upstream of the frameshifting site as found for SARS (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Su et al., 2005</a>) could be a more widespread property of PRF sites, probably creating a balance to ensure that frameshift promoting signals like a rigid downstream secondary structure do not lead to a complete inhibition of translation and potentially degradation of the mRNA. To test for effects of upstream secondary structure on PRF, we replaced the region upstream of the slippery site with a recoded version (encoding the same amino acid sequence), a sequence predicted to fold into the same secondary structure as the wild-type, a sequence lacking any strong predictable secondary structure or a hairpin at the 5’ or 3’ end. Specifically in the case of PEG10, a hairpin at the 5’ end of the upstream region or a lack of secondary structure resulted in a frameshifting signal above threshold (2-8% GFP fluorescence; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4C</a>) as opposed to the wild-type sequence (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig 1C</a>, see our discussion of potential reasons above).</p><p id="p-40">The upstream region of OAZ1 is thought to harbor signals enhancing frameshifting (cf. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ivanov et al., 2006</a>). By testing variants in which increasing portions of the upstream region have been replaced with constant sequences, we find these PRF-promoting signals to be located in the 20 nucleotides before the slippery site (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4D</a>, decrease of PRF signal on average to 31.3% of wild-type, p=0.0077, Wilcoxon signed-rank test, effect on region further upstream not significant). In contrast to the other - 1 PRF sites, HIV frameshifting rates show an increase upon mutation of the corresponding upstream region (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4D</a>, mean increase of PRF signal to 128.2% of wild-type, p=0.013, Wilcoxon signed-rank test, effect on region further upstream not significant), indicating the presence of an inhibitory element in this region.</p><p id="p-41">We expanded our search for properties affecting PRF efficiency and examined the effect of the codons preceding the slippery site. We found that the presence of a charged amino acid immediately upstream of the slippery site significantly reduced frameshifting signal, even when the sequence of the slippery site was unchanged (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig 4E</a>, p&lt;0.0015, Wilcoxon signed-rank test). We examined also the influence of decoding efficiency as measured by the tRNA adaptation index (tAI) on PRF and detected an increasingly negative correlation between tAI and PRF signal in the 0 frame codons leading up to the site of ribosome slippage (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4F</a>). This observation might indicate that progressive slowing down of the ribosome by limited tRNA availability might contribute to stalling at the slippery site and frameshifting.</p><p id="p-42">To not limit our analysis to endogenous frameshifting events, we designed completely synthetic regions corresponding to a slippery site and the upstream and downstream region with different sequence and structural properties. Some fully designed sequences were able to trigger frameshifting and resulted in a −1 PRF signal of up to 11% GFP fluorescence, approximately corresponding to the wild-type frameshifting rate of the most efficient event tested here (SARS coronavirus). We tested all different downstream regions with synthetic slippery sequences resembling common types of −1 PRF sites and found pronounced differences in combinatorial preferences (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig 4G</a>), showing that also fully synthetic frameshifting events exhibit the combinatorial preferences and context-dependent peculiarities observed for native PRF events.</p></div><div id="sec-7" class="subsection"><h3>Prediction of frameshifting potential and efficiency</h3><p id="p-43">Having accurate quantitative measurements for large collections of frameshifting sites, we aimed to predict frameshifting efficiency using machine learning approaches. Based on previous studies and our own findings, we used the identity of slippery site positions, tAI of codons around the frameshifting site, amino acid class, minimum free energy and pairedness of positions downstream of the frameshifting site (see Methods), alone and combined, as features and built computational models based on Gradient Boosting Decision Trees (XGBoost (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen and Guestrin, 2016</a>); <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A</a> and Methods). These models were either estimating whether a sequence has the ability to induce frameshifting (i.e. yield a fluorescent signal above threshold (1.3%, see Figure S2A-C); classification) or quantitatively predicting PRF efficiency (approximated by GFP fluorescence intensity in our assay; regression). We achieved high accuracy (up to an area under the receiver operating characteristic curve (ROC AUC)=0.93 for the classification of a sequence as yielding a PRF signal or not, and Pearson r=0.81 for the comparison between measured and predicted frameshifting signal (% GFP fluorescence)) when training our model on variants of specific frameshifting events and predicting unseen variants from the same event (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5BC</a>). Despite the differences in the regulatory characteristics between individual PRF events, we could predict frameshifting ability (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5D</a>, Fig S10A, ROC AUC=0.89; area under the Precision-Recall-Curve=0.8) and frameshifting signal (% GFP fluorescence; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5E</a>, Pearson r=0.53) also for a pool of all sequence variants tested, no matter whether they yielded a detectable frameshifting signal in our assay or not (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig 5BC</a>, Fig S10A; “-1 PRF events” vs. “all library variants”).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction of frameshifting potential and efficiency. A. Outline of the process of generating features and building and testing predictive models. BC. Prediction scores (area under the receiver operating characteristic curve (&#x201C;ROC AUC&#x201D;) in the case of classification (B) and Pearson correlation coefficients in the case of regression (C)) on held-out test data for the indicated PRF event(s) or the entire library using different feature sets (slip: nucleotide identities in the canonical XXXYYYZ slippery site; tai: tAI scores around the slippery site, aa: amino acid class (&#x201C;unipolar&#x201D;, &#x201C;polar&#x201D;, &#x201C;charged&#x201D;) around the slippery site, dg: MFE of downstream regions, sec: predicted pairedness of downstream positions). D. ROC curve showing performance of a classifier trained on 80% of all designed library variants passing filtering and testing on the remaining 20%. E. For the test set (20% of all designed library variants of all &#x2212;1 PRF events (left) or HIV-1 (right) passing filtering) the measured % GFP fluorescence is plotted against the model prediction (trained on 80% of all library variants of all &#x2212;1 PRF events (left) or HIV-1 (right) passing filtering). F. Prediction scores (ROC AUC) on held-out test data (20%) for the indicated PRF event (columns) using the full set of features and training the model on the training data (80%) from the indicated PRF event (rows)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Prediction of frameshifting potential and efficiency.&lt;/span&gt; A. Outline of the process of generating features and building and testing predictive models. BC. Prediction scores (area under the receiver operating characteristic curve (&#x201C;ROC AUC&#x201D;) in the case of classification (B) and Pearson correlation coefficients in the case of regression (C)) on held-out test data for the indicated PRF event(s) or the entire library using different feature sets (slip: nucleotide identities in the canonical XXXYYYZ slippery site; tai: tAI scores around the slippery site, aa: amino acid class (&#x201C;unipolar&#x201D;, &#x201C;polar&#x201D;, &#x201C;charged&#x201D;) around the slippery site, dg: MFE of downstream regions, sec: predicted pairedness of downstream positions). D. ROC curve showing performance of a classifier trained on 80% of all designed library variants passing filtering and testing on the remaining 20%. E. For the test set (20% of all designed library variants of all &#x2212;1 PRF events (left) or HIV-1 (right) passing filtering) the measured % GFP fluorescence is plotted against the model prediction (trained on 80% of all library variants of all &#x2212;1 PRF events (left) or HIV-1 (right) passing filtering). F. Prediction scores (ROC AUC) on held-out test data (20%) for the indicated PRF event (columns) using the full set of features and training the model on the training data (80%) from the indicated PRF event (rows).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F5.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F5.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Prediction of frameshifting potential and efficiency.</span><p id="p-44" class="first-child">A. Outline of the process of generating features and building and testing predictive models. BC. Prediction scores (area under the receiver operating characteristic curve (“ROC AUC”) in the case of classification (B) and Pearson correlation coefficients in the case of regression (C)) on held-out test data for the indicated PRF event(s) or the entire library using different feature sets (slip: nucleotide identities in the canonical XXXYYYZ slippery site; tai: tAI scores around the slippery site, aa: amino acid class (“unipolar”, “polar”, “charged”) around the slippery site, dg: MFE of downstream regions, sec: predicted pairedness of downstream positions). D. ROC curve showing performance of a classifier trained on 80% of all designed library variants passing filtering and testing on the remaining 20%. E. For the test set (20% of all designed library variants of all −1 PRF events (left) or HIV-1 (right) passing filtering) the measured % GFP fluorescence is plotted against the model prediction (trained on 80% of all library variants of all −1 PRF events (left) or HIV-1 (right) passing filtering). F. Prediction scores (ROC AUC) on held-out test data (20%) for the indicated PRF event (columns) using the full set of features and training the model on the training data (80%) from the indicated PRF event (rows).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">To determine to what extent a model trained on one PRF event could predict frameshifting potential and −1 PRF signal in a different context, we trained our model (based on the full set of features) on sequence variants derived from one event and predicted on variants of another event (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig 5F</a>, Fig S10B). While performance is usually best for variants of the same PRF event, this approach revealed functional similarities that allowed relatively accurate prediction on “foreign” PRF events (ROC AUC up to 0.94, Pearson r up to 0.69). There is considerable overlap with other attempts of functional clustering, e.g. based on sensitivities to mutations (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig 2A</a>), with SRV1 and HERV-K10 PRF sites showing similar behavior and HIV variants yielding good prediction scores for models trained on data from any retroviral PRF event (HIV, SIVmac239 and SRV1; <a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig 5F</a>). The SARS PRF event, however, is not amenable to prediction based on models trained on data from other PRF events. Prediction accuracy also reflects the specific components of particular importance for a PRF event, e.g. the slippery site for HIV (Fig S10C) and the downstream region for SRV1 (Fig S10D). In these cases, frameshifting efficiency can be predicted using models trained on data from almost any other PRF event.</p><p id="p-46">Interpretation of the prediction models provides an additional approach to identify the most important properties of a (potential) PRF site. Here, we used Shapely (SHAP) values for determining the contribution of each feature to the prediction result of every sample (Fig S11A; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lundberg et al., 2020</a>). The features of the different subsets driving the prediction provide additional information about the effect of specific sequence properties on the readout (Fig S11A), e.g. recapitulating earlier observations that XXXCCCZ (“Y=C”) and XXXGGGZ (“Y=G”) slippery sites are disfavored (Fig S11A upper left, cf. <a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig 2D</a>). For a model trained on the MFE of different lengths of downstream region, stronger secondary structure not always led to prediction of higher PRF efficiency (Fig S11A lower left). When taking into account regions of e.g. 60 nt after the slippery site (“dg60”), a lower feature value favored prediction of PRF (Fig S11B).). While the positive effect of strong secondary structure (low MFE) in the first 60 nt on PRF prediction was strongly enhanced if also the downstream region as a whole showed low MFE (Fig S11B), in general we find strong secondary structure of the entire downstream region (120 nt) to be associated with prediction of no PRF (Fig S11A). This could either be explained through an effect on frameshifting efficiency or on RNA stability and overall expression levels. The latter option is unlikely to explain the effect in full as we do not observe a correlation between MFE of the downstream region and steady-state RNA levels (Fig S11C).</p></div><div id="sec-8" class="subsection"><h3>HIV clinical isolates exhibit subtype-specific differences in frameshifting efficiency and associations with viral load in patients</h3><p id="p-47">The HIV gag-pol frameshifting site is arguably one of the most intensely studied examples of PRF and – due to its critical importance for the viral replication cycle – has been repeatedly suggested as an antiviral drug target (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brakier-Gingras et al., 2012</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hung et al., 1998</a>). To assess the natural variation in frameshifting rates in HIV1 we assembled a set of 581 sequences from clinical isolates between 1976 and 2014 (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>, Methods), which differ in the sequence surrounding the frameshifting site, but not in the slippery site itself (Fig S12A). Frameshifting rates of the variants are distributed around the rate observed for the lab strain HXB2 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a>). This constitutes actual differences in frameshifting rates and not only experimental variability, as isolated clones show remarkably good correlation given the small differences in PRF signal we are measuring (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6B</a>). Secondary structure showed the best correlation with frameshifting rates when considering the first 30 nucleotides after the frameshifting site (Fig S12B), matching the region of high sequence conservation (Fig S12A). We grouped the HIV variants based on subtype and found significant differences between the groups (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig 6C</a>, p&lt;8*10<sup>−11</sup>, one-way ANOVA), most notably higher frameshifting rates in subtype C (p&lt;6*10<sup>−12</sup> for the difference between C and B), in contrast to an earlier report (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baril et al., 2003</a>). HIV subtypes show distinct geographical distributions (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hemelaar et al., 2006</a>), and consequently we also observed differences between countries of origin (Fig S12C, p&lt;2*10<sup>−4</sup>), but no change in frameshifting rates over time (Fig S12D, p=0.4).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Testing of PRF sites from HIV clinical isolates reveals subtype specific differences and associations with viral load in patients. A. Distribution of percent GFP fluorescence of all measured HIV variants. B. Percent GFP fluorescence as determined by FACSseq is plotted against the value determined by measuring the corresponding isolated clone by FACS. C. Boxplot showing percent GFP fluorescence of HIV gag-pol PRF variants coming from the indicated subtypes (n=8-112). D. Viral load (HIV titer) is plotted against percent GFP fluorescence determined for the natural gag-pol PRF site variant from the corresponding clinical isolate; vertical line: HIV HXB2 wild-type percent GFP fluorescence. E. Predicted vs measured percent GFP fluorescence for 20% of the HIV variants (held-out test set) with the full feature set except for the identity of slippery site positions (left) or after removing MFE features (middle) or all secondary structure derived features (right) features." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-229389802" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Testing of PRF sites from HIV clinical isolates reveals subtype specific differences and associations with viral load in patients.&lt;/span&gt; A. Distribution of percent GFP fluorescence of all measured HIV variants. B. Percent GFP fluorescence as determined by FACSseq is plotted against the value determined by measuring the corresponding isolated clone by FACS. C. Boxplot showing percent GFP fluorescence of HIV gag-pol PRF variants coming from the indicated subtypes (n=8-112). D. Viral load (HIV titer) is plotted against percent GFP fluorescence determined for the natural gag-pol PRF site variant from the corresponding clinical isolate; vertical line: HIV HXB2 wild-type percent GFP fluorescence. E. Predicted vs measured percent GFP fluorescence for 20% of the HIV variants (held-out test set) with the full feature set except for the identity of slippery site positions (left) or after removing MFE features (middle) or all secondary structure derived features (right) features.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F6.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F6.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/13/469692/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Testing of PRF sites from HIV clinical isolates reveals subtype specific differences and associations with viral load in patients.</span><p id="p-48" class="first-child">A. Distribution of percent GFP fluorescence of all measured HIV variants. B. Percent GFP fluorescence as determined by FACSseq is plotted against the value determined by measuring the corresponding isolated clone by FACS. C. Boxplot showing percent GFP fluorescence of HIV gag-pol PRF variants coming from the indicated subtypes (n=8-112). D. Viral load (HIV titer) is plotted against percent GFP fluorescence determined for the natural gag-pol PRF site variant from the corresponding clinical isolate; vertical line: HIV HXB2 wild-type percent GFP fluorescence. E. Predicted vs measured percent GFP fluorescence for 20% of the HIV variants (held-out test set) with the full feature set except for the identity of slippery site positions (left) or after removing MFE features (middle) or all secondary structure derived features (right) features.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Optimal gag-pol frameshifting rates have been proposed to be critical for virulence (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dinman and Wickner, 1992</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Felsenstein and Goff, 1988</a>). In order to link the frameshifting rate measured in our reporter assay with the replicative success of the corresponding HIV isolate we compared the viral load in patients (where available) to frameshifting rates of the causal HIV isolate (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig 6D</a>). Although viral load is influenced by many factors, we nevertheless observed a clear trend for patients with high viral load to have frameshifting rates close to “wild type” (the lab strain HXB2). These data present optimality of frameshifting rates as a hallmark of HIV infection and as being associated with infectious success, underscoring the potential for drugs altering the efficiency of gag-pol frameshifting.</p><p id="p-50">In order to predict frameshifting efficiency of naturally occurring variants of the HIV gag-pol frameshifting site, we built a model based on the feature sets described above. Despite eliminating the most important feature of the HIV gag-pol PRF site (the slippery sequence) by restricting our analysis to clinical isolates with no sequence alterations in and around the canonical UUUUUUA and despite the narrow range of frameshifting rates (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig 6A</a>), we could accurately predict frameshifting rates of unseen variants (Pearson r=0.6; <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig 6E</a>, left). Interestingly, removing the features based on pairedness of downstream positions and minimum free energy led to similar agreement between measured and predicted values (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig 6E</a>, right). This is in line with the limited importance of the exact downstream structure in comparison to other PRF events tested. Including designed HIV variants in the training set yielded similar prediction accuracy (Fig S12E), indicating that our model can learn the relevant rules from our set of natural variants alone. Taken together, these results show that a model based on data from our assay is sensitive enough to detect even subtle differences between HIV variants and allows prediction of frameshifting rates with an accuracy of clinical relevance.</p></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-51">Here, we combined fluorescent frameshifting reporters with rational design of DNA sequences and high-throughput testing to systematically decipher the rules governing PRF. Our approach aims at a systematic and comparative assessment of the commonalities and peculiarities in the regulation of frameshifting efficiency across PRF events. It is complementary to in-depth analyses of individual frameshifting sites yielding detailed structural or kinetic insights (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Caliskan et al., 2014</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2017</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chen et al., 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Choi et al., 2020</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lyon et al., 2019</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Yan et al., 2015</a>). The high-throughput nature of our approach entails particular limitations. While we show that our assay is highly sensitive to changes in the PRF signal (e.g. in the testing of natural isolates of HIV) and therefore able to identify even subtle effects of sequence alterations on frameshifting, our detection limit is higher than in assays using amplification methods and/or overexpression of the reporter.</p><p id="p-52">Prediction of PRF events has up to now been largely limited to identifying sequences matching the canonical pattern for slippery sites, followed by a downstream secondary structure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Belew et al., 2008</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Theis et al., 2008</a>). To the best of our knowledge, no attempts at quantitatively predicting the effect of sequence variation exist to date. We used our measurements of frameshifting signal of thousands of variants of known PRF sites to build a computational model based on known and novel sequence features affecting frameshifting, leading to accurate prediction of frameshifting potential (up to ROC AUC=0.93) and frameshifting rates (up to Pearson r=0.81). In many cases sequence properties other than slippery site identity or downstream secondary structure rigidity yielded high prediction scores, emphasizing the importance to include additional features such as amino acid properties and tRNA availability in the investigation of PRF regulation.</p><p id="p-53">By controlled sequence and structure manipulations in multiple contexts we aimed to dissect and directly compare the regulatory architecture of 15 frameshifting events, revealing a great diversity in regulatory strategies involving upstream and downstream sequence and structural elements and amino acid properties. We demonstrate a high degree of optimization and specialization of regulatory mechanisms. For example, no universally optimal slippery site exists, but in each of the PRF events tested the native slippery site was found to be optimal given the sequence context.</p><p id="p-54">A correlation between secondary structure rigidity and PRF efficiency has been shown previously (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bidou et al., 1997</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Garcia-Miranda et al., 2016</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wu et al., 2018</a>), while also other properties of secondary structures like their plasticity have been reported to be associated with frameshifting rates (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Halma et al., 2019</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Ritchie et al., 2012</a>). Our results corroborate the functional link between minimum free energy of downstream structures and show that this is true for most PRF events tested. However, this correlation generally depended on the length of the downstream region taken into account, in agreement with the specific association between PRF efficiency and thermodynamic stability of the start of the HIV stem loop structure reported earlier (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Mouzakis et al., 2013</a>). Furthermore, the correlation between PRF signal and MFE only held true when testing small variations of the native sequence and structure; when sequences and structures different from the native one were tested, this correlation was lost.</p><p id="p-55">Slippery site and a downstream stimulatory signal are central – and therefore intensely studied – regulatory elements present in all known PRF events. Here, we expanded the repertoire of regulatory inputs and demonstrated that both tRNA availability as well as amino acid properties can affect PRF efficiencies across native sequence contexts.</p><p id="p-56">Ribosome stalling due to limited tRNA availability has the potential to lead to slippage of the ribosome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gallant and Lindsley, 1992</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Weiss et al., 1988</a>) and contributes to non-programmed frameshifting in huntingtin (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Girstmair et al., 2013</a>). Availability of tRNAs decoding slippery site codons have been shown to influence HIV gag-pol frameshifting efficiency (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Korniy et al., 2019</a>) and HIV-1 itself has the ability to modulate the tRNA pool of the host cell (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">van Weringh et al., 2011</a>). Our observation that limited tRNA availability in the codons leading up to the slippery site progressively contributes to PRF efficiency is in line with these data and expands the notion of an influence of the tRNA pool on frameshifting beyond the slippery site codons and individual PRF events.</p><p id="p-57">In addition, our analyses highlighted amino acid properties around the frameshifting sites as another factor influencing PRF efficiency. Specifically, a charged amino acid at the last position before the slippery site was associated with reduced frameshifting rate. Together with reports of a frameshift-promoting effect of specific amino-acid sequences upstream of the frameshifting site in the case of +1 PRF in fungal OAZ genes (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Yordanova et al., 2015</a>) and the copA gene of Escherichia coli (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Meydan et al., 2017</a>), this observation provides additional evidence for a role of the nascent peptide in regulating ribosomal frameshifting.</p><p id="p-58">In general, our results reveal functional groups of PRF events that exhibit greater similarity in terms of their sensitivities to mutations in regulatory regions and congruence in the preferences for secondary structure. These functional groups also manifest themselves in a higher prediction accuracy within these groups. The groups resemble similarities also according to other criteria, e.g. in their native secondary structure. PRF events, whose downstream stimulatory sequence has been shown to be a stem loop (e.g. HIV-1 (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Parkin et al., 1992</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Staple and Butcher, 2003</a>) and SIVmac239 (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Marcheschi et al., 2007</a>)) tend to be more resilient and less affected by small sequence changes than those with a downstream pseudoknot (e.g. SRV-1 (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">ten Dam et al., 1994</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Du et al., 1997</a>) as well as SARS and other coronaviruses (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Brierley et al., 1989</a>; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Plant et al., 2005</a>)). Typically, these cases exhibit higher fluorescence signal in our assay (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig 1C</a>), but greater sensitivities to even small sequence changes (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Fig 2B</a>, 3C, S4AB, S5A) and requirements for a more extensive downstream region. This might also entail a greater sensitivity to the experimental context and therefore explain the lack of signal (above the noise threshold) for previously reported cases depending on a pseudoknot as frameshift stimulatory signal like Rous sarcoma virus (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jacks and Varmus, 1985</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Marczinke et al., 1998</a>). In other cases, low wild-type rates of frameshifting, requirements for additional trans-acting factors or an effect on overall expression levels might explain the lack of signal in our assay.</p><p id="p-59">Comparing the HIV gag-pol site with other PRF events shows the remarkable robustness and exceptional tolerance for various non-native downstream stimulatory signals. More than in other cases, maintaining the wild-type slippery site UUUUUUA is crucial for frameshifting, and tends to be absolutely invariable across otherwise diverse natural isolates (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Biswas et al., 2004</a>). The quantitative nature of our assay allows us nevertheless to accurately quantify the effect of sequence variation away from the slippery site, revealing sub-type specific differences. Furthermore, we devised a machine learning model that is able to predict the natural variation in HIV frameshifting rates for novel variants (Pearson r=0.60). Precise stoichiometry of frameshifted and non-frameshifted product is crucial for HIV virulence, and the ability to predict frameshifting rates of naturally occurring variants of the gag-pol PRF site therefore has important clinical implications, both in the case of HIV as well as in the case of PRF events in other viruses.</p><p id="p-60">This high-throughput investigation of PRF also constitutes an addition to the experimental toolbox for studying translational recoding and highlights many starting points for further investigations into the regulatory basis and functional importance of frameshifting. In addition to deciphering the effect of cis-regulatory elements, regulation by trans-acting factors like the recently discovered case of the frameshift repressor shiftless (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wang et al., 2019</a>) constitutes another area where a systematic approach like the one presented here can yield novel insights into the mode of action and its specificity. Libraries of frameshifting reporters provide a platform for screening putative modifiers of ribosomal frameshifting, offering a powerful tool to identify ways to generally or selectively control frameshifting and opening new possibilities for interfering with viral replication and for controlling cellular processes depending on translational frameshifting.</p></div><div class="section" id="sec-10"><h2 class="">Methods</h2><div id="sec-11" class="subsection"><h3>Synthetic library design</h3><div id="sec-12" class="subsection"><h4>General design notes</h4><p id="p-61">Oligonucleotides were designed to maintain a constant length of 210 nt. Restriction sites used for cloning were excluded from the design. All the variants were composed of an 18 nt forward primer, 12 nt barcode sequence, 162 nt variable region and 18 nt reverse primer sequences. DNA barcodes were designed to differ from any other barcode in the library in at least 3 nt.</p></div><div id="sec-13" class="subsection"><h4>Selection of frameshifting sites for systematic sequence manipulations</h4><p id="p-62">60 PRF events reported previously or listed in FSDB (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Moon et al., 2007</a>) were included in the library design. Sequences were aligned in a way that the site of frameshifting would be at the same position in all variants (slippery site ending at position 42). In case there were stop codons downstream of the PRF site in any of the three frames, additional versions with stop codons mutated (changed to TGG) were included for testing only. Out of these, 15 were selected for systematic sequence manipulations based on the amount of experimental evidence supporting them, lack of stop codons in the relevant reading frames, biological significance and mechanistic diversity (i.e. covering +1, −1 and −2 frameshifting sites). The HIV and OAZ frameshifting sites were tested beforehand in isolation to confirm that our experimental setup is able to reproduce well established cases of −1 as well as +1 PRF.</p></div><div id="sec-14" class="subsection"><h4>Design of individual rationally designed subsets</h4><p id="p-63">For each of the subsets in the libraries, systematic sequence manipulations were performed on the set of 15 previously reported frameshifting sites. Design of subsets was carried out in Python.</p><div id="sec-15" class="subsection"><h5>Multiple barcode controls</h5><p id="p-64">We added multiple variants to the library that contained the same variable region, but different barcodes, in order to gauge potential effects of the barcode and the technical noise of our assay.</p></div><div id="sec-16" class="subsection"><h5>Slippery site</h5><p id="p-65">We tested all possible variations of the different slippery site patterns found in previously reported frameshifting events (XXXYYYZ, XYYZZZZ, influenza, OAZ (for +1 PRF)). In the case of OAZ, both the first and the second codon in the slippery site UCC UGA were replaced with all other codons. In addition, we created all possible point mutations in −1 PRF slippery sites that would not give rise to a stop codon.</p></div><div id="sec-17" class="subsection"><h5>Immediate downstream region/”spacer”</h5><p id="p-66">We created all possible point mutations in the six nucleotides downstream of PRF slippery sites that would not give rise to a stop codon. In addition, we removed up to 6 bps immediately downstream (in 1 or 3 bp increments, depending on the presence of stop codons in downstream reading frames) or introduced up to 12 bps from five different constant sequences (in 1 or 3 bp increments, depending on the presence of stop codons in downstream reading frames).</p></div><div id="sec-18" class="subsection"><h5>Immediate upstream region</h5><p id="p-67">The first two codons preceding the slippery site (in the original frame) were replaced with all other codons (except stop codons).</p></div><div id="sec-19" class="subsection"><h5>Upstream region</h5><p id="p-68">The upstream region (excluding the last two codons before the slippery site) was randomly recoded five times. Moreover, variants were added for all 15 PRF events in which the same part of the upstream region was replaced by four different constant sequences in increments of six bps, starting from the beginning of the variable region. In addition, we included the following structural variations in the design: We designed sequences 30 nt in length predicted (using ViennaRNA 2.0 (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lorenz et al., 2011</a>) and antaRNA (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Kleinkauf et al., 2015a</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">2015b</a>)) to fold into the respective native structure (but having a different primary sequence or to have no secondary structure, a 9 bp long stem-loop at the 5’ or at the 3’ end (6 sequences per set) and replaced the endogenous region in all 15 PRF events.</p></div><div id="sec-20" class="subsection"><h5>Downstream region</h5><p id="p-69">Variants were added for all 15 PRF events in which the same part of the downstream region was replaced by three different constant sequences (different sets were used for predicted +1 and −1 PRF sites to avoid stop codons in the relevant frames) in increments of six bps, starting from the end of the variable region. For scanning mutagenesis, 30 positions (leaving the first 6 bp after the slippery site unchanged) were replaced with either A or C (G and T were avoided as they often give rise to stop codons in one of the frames). In addition, we included the following structural variations in the design: We used ViennaRNA 2.0 or pKiss (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Janssen and Giegerich, 2015</a>) (which also predicts pseudoknots) to determine the native secondary structure of each of the 15 PRF events. We then created for each PRF event two sets of ten sequences each (using antaRNA) that are predicted to fold into the downstream secondary structure as determined using ViennaRNA and pKiss, respectively. Furthermore we designed variants of the predicted HIV, SRV1 and nsp2F (PRRSV) downstream secondary structure, e.g. by extending or reducing the length of hairpins, and completely synthetic secondary structures and again used antaRNA to create sets of corresponding sequences that we used to replace the native downstream structure in all 15 PRF events.</p></div><div id="sec-21" class="subsection"><h5>Completely synthetic variants</h5><p id="p-70">We designed a set of four upstream regions, three potential slippery sites (resembling previously reported cases of the patterns XXXYYYZ, XYYZZZZ and XXYYYYY), and created all possible combinations with a set of 57 artificial downstream sequences.</p></div><div id="sec-22" class="subsection"><h5>Frameshifts</h5><p id="p-71">We created variants in which the slippery site was moved by one, two or three positions in the 5’ or 3’ direction in order to verify the frame-dependence of the effect.</p></div><div id="sec-23" class="subsection"><h5>Combinatorial variants</h5><p id="p-72">Upstream, slippery site and downstream regions from the whole set of 15 PRF sites were recombined, either preserving the original sequence fully or replacing potential stop codons in all frames with TGG.</p></div></div><div id="sec-24" class="subsection"><h4>gag-pol frameshifting sites from HIV clinical isolates</h4><p id="p-73">We retrieved &gt;30,000 full length sequences of the HIV genome from clinical isolates (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>) and selected gag-pol frameshifting sites which had the canonical sequence around the slippery site (TAATTTTTTA), but differed in at least one position in the 30 nucleotides upstream or the 120 nucleotides downstream. We chose to exclude variability in the slippery site to screen for more subtle differences in frameshifting rates across clinical isolates.</p></div></div></div><div class="section" id="sec-25"><h2 class="">Experimental procedures</h2><div id="sec-26" class="subsection"><h3>K562 cell culture</h3><p id="p-74">K562 cells were acquired from ATCC. Cells were grown in Iscove’s Modified Dulbecco Medium supplemented with 10% fetal bovine serum (SIGMA) and 1% Penicillin-Streptomycin solution (SIGMA). The cells were split when reaching a concentration of ∼10<sup>6</sup> cells/ml. The cells were grown in an incubator at 37°C and 5% CO<sup>2</sup>. Cells were frozen in batches of 4×10<sup>6</sup> cells in growth medium supplemented with 5% DMSO.</p></div><div id="sec-27" class="subsection"><h3>Construction of the master plasmid</h3><p id="p-75">Master plasmids for library insertion were constructed by amplifying parts from existing vectors and cloning the parts sequentially into pZDonor 3.1. The master plasmid contained the EF1alpha promoter, <em>mCherry</em>, a designed multiple cloning site containing restriction sites for library cloning (RsrII and AscI), <em>gfp</em> and the SV40 terminator sequence. In the case of the master plasmid for cloning the stop-vector library, two complementary DNA oligonucleotides (GACTGATAGCTGACTAGTCG and GTCCGACTAGTCAGCTATCAG) were synthesized (IDT), annealed and introduced into the vector described above (cut with RsrII) such that the mCherry-proximal RsrII site was destroyed and the downstream RsrII site (after the inserted stop cassette) was subsequently used for inserting the library.</p></div><div id="sec-28" class="subsection"><h3>Synthetic library cloning</h3><p id="p-76">The cloning steps were performed essentially as described previously (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Vainberg Slutskin et al., 2018</a>). We used Agilent oligo library synthesis technology to produce a pool of 17,809 different fully designed single-stranded 210-oligomers (Agilent Technologies, Santa Clara, CA), which was provided as a single pool of oligonucleotides (10 pmol). The two subsets of this pool corresponding to the designed and native libraries tested here were defined by unique amplification primers. The reverse primer contained a spacer of either 4 or 5 nucleotides to create the offset in the <em>gfp</em> reading frame. The oligo pool was dissolved in 200 μl Tris-ethylene-diamine-tetraacetic acid (Tris-EDTA) and then diluted 1:50 with Tris-EDTA, which was used as template for PCR. We amplified each of the four libraries by performing 8 PCR reactions, each of which contained 19 μl of water, 5 μl of DNA, 10 μl of 5× Herculase II reaction buffer, 5 μl of 2.5 mM deoxynucleotide triphosphate (dNTPs) each, 5 μl of 10 μM forward primer, 5 μl of 10 μM reverse primer, and 1 μl Herculase II fusion DNA polymerase (Agilent Technologies). The parameters for PCR were 95°C for 1 min, 14 cycles of 95°C for 20 s, and 68°C for 1 min, each, and finally one cycle of 68°C for 4 min. The oligonucleotides were amplified using library-specific common primers in the length of 35 nt, which have 18-nt complementary sequence to the single-stranded 210-mers and a tail of 17 nt containing RsrII (forward primer) and AscI (reverse primer) restriction sites. The PCR products were concentrated using Amicon Ultra, 0.5 ml 30K centrifugal filters (Merck Millipore). The concentrated DNA was then purified using a PCR mini-elute purification kit (Qiagen) according to the manufacturer’s protocol. Purified library DNA (540 ng total) was cut with the unique restriction enzymes RsrII and AscI (Fermentas FastDigest) for 2 hours at 37°C in two 40-μl reactions containing 4 μl fast digest (FD) buffer, 1 μl RsrII enzyme, 1 μl AscI enzyme, 18 μl DNA (15 ng/μl) and 16 μl water, followed by heat inactivation for 20 min at 65°C. Digested DNA was separated from smaller fragments and uncut PCR products by electrophoresis on a 2.5% agarose gel stained with GelStar (Cambrex Bio Science Rockland). Fragments were cut from the gel and eluted using electroelution Midi GeBAflex tubes (GeBA, Kfar Hanagid, Israel). Eluted DNA was precipitated using sodium acetate–isopropanol. The master plasmid was cut with RsrII and AscI (Fermentas FastDigest) in a reaction mixture containing 6 μl FD buffer, 3 μl of each enzyme and 3.5 μg of the plasmid in a total volume of 60 μl. After incubation for 2.5 hours at 37°C, 3 μl FD buffer, 3 μl alkaline phosphatase (Fermentas) and 24 μl water were added and the reactions were incubated for an additional 30 mins at 37°C followed by 20 min at 65°C. Digested DNA was purified using a PCR purification kit (Qiagen). The digested plasmid and DNA library were ligated for 30 min at room temperature in a 10 μl reactions, containing 150 ng plasmid and the library in a molar ratio of 1:1, 1 μl CloneDirect 10× ligation buffer, and 1 μl CloneSmart DNA ligase (Lucigen Corporation), followed by heat inactivation for 15 min at 70°C. Ligated DNA was transformed into <em>E. coli</em> 10G electrocompetent cells (Lucigen) divided into aliquots (23 µl each, plus 2 μl of the ligation mix), which were then plated on 4 Luria broth (LB) agar (200 mg/ml amp) 15-cm plates per transformation reaction (25 μl). The rationally designed (12,809 variants) and the native (5000 variants) parts of the library were cloned separately. For the rationally designed and the native library we collected around 10<sup>6</sup> and 4×10<sup>5</sup> colonies, respectively, for each frame the day after transformation by scraping the plates into LB medium. Library-pooled plasmids were purified using a NucleoBond Xtra maxi kit (Macherey Nagel). To ensure that the collected plasmids contain only a single insert of the right size, we performed colony PCR (at least 48 random colonies per library).</p></div><div id="sec-29" class="subsection"><h3>Transfection into K562 cells and genomic integration</h3><p id="p-77">The purified plasmid libraries were transfected into K562 cells and genomically integrated using the Zinc Finger Nuclease (ZFN) system for site-specific integration and the CompoZr® Targeted Integration Kit - AAVS1 (SIGMA). Transfections were carried out using Amaxa® Cell Line Nucleofector® Kit V (LONZA). To ensure library representation we performed 15 nucleofections of the purified plasmid library for each rationally designed library (frame −1 and frame +1) and 5 for each native library. For each nucleofection, 4×10<sup>6</sup> cells were centrifuged and washed twice with 20 ml of Hank’s Balanced Salt Solution (HBSS, SIGMA). Cells were resuspended in 100 μl solution (warmed to room temperature) composed of 82 μl solution V and 19 μl supplement (Amaxa® Cell Line Nucleofector® Kit V). Next, the cells were mixed with 2.75 μg of donor plasmid and 0.6 μg ZFN mRNA (prepared in-house) just prior to transfection. Nucleofection was carried out using program T-16 on the NucleofectorTM device, immediately mixed with ∼0.5 ml of pre-cultured growth medium and transferred to a 6 well plate with additional 1.5 ml of pre-cultured growth medium. A purified plasmid library was also transfected without the addition of ZFN and served as a control to determine when cells lost non-integrated plasmids.</p></div><div id="sec-30" class="subsection"><h3>Sorting the library by FACS</h3><p id="p-78">K562 cells were grown for at least 14 days to ensure that non-integrated plasmid DNA was eliminated. A day prior to sorting, cells were split to ∼0.25×106 cells/ml. On the day of sorting, cells were centrifuged, resuspended in sterile PBS and filtered using cell-strainer capped tubes (Becton Dickinson (BD) Falcon). Sorting was performed with BD FACSAria II SORP (special-order research product) at low sample flow rate and a sorting speed of ∼18000 cells/s. To sort cells that integrated the reporter construct successfully and in a single copy (∼4% of the population), we determined a gate according to mCherry fluorescence so that only mCherry-expressing cells corresponding to a single copy of the construct were sorted (mCherry single population). We collected around 350 cells/variant on average for each library in order to ensure adequate library representation.</p><p id="p-79">Cells sorted for single integration of the transgene were grown for a week before we sorted the population (combined rationally designed and native library, but separately for −1 PRF reporters, +1 PRF reporters and the stop-vector control) into 16 (in the case of the stop-vector 12) bins according to GFP fluorescence, after gating for a narrow range of mCherry expression to avoid effects coming from the influence of the variable region on overall expression level. This resulted in the loss of a significant fraction of variants (as only around 50% of the cells fell into the narrow range of mCherry fluorescence) but ensured that the subsequent sorting by GFP fluorescence was not biased by expression level. Each bin was defined to span a range of GFP fluorescence values such that it contains between 1-10% of the cell population. We collected around 1000 cells/variant on average in order to ensure adequate library representation. Cells from each bin were grown separately for freezing and purification of genomic DNA.</p></div><div id="sec-31" class="subsection"><h3>Genomic DNA purification, amplification and sample preparation</h3><p id="p-80">For each of the bins we purified genomic DNA by centrifuging 5×10<sup>6</sup> cells, washing them with 1 ml PBS and purifying DNA using DNeasy Blood &amp; Tissue Kit (Qiagen) according to the manufacturer’s protocol. In order to maintain the complexity of the library amplified from gDNA, PCR reactions were carried out on a gDNA amount calculated to contain a minimum average of 200 copies of each oligo included in the sample. For each of the 16 bins, we used 15 µg of gDNA as template in a two-step nested PCR. In the first step, three reactions were performed and each reaction contained 5 μg gDNA, 25 μl Kapa Hifi ready mix X2 (KAPA Biosystems), 2.5 μl 10 μM 5′ primer, and 2.5 μl of 10 μM 3′ primer. The parameters for the first PCR were 95°C for 5 min, 18 cycles of 94°C for 30s, 65°C for 30s, and 72°C for 60s, each, and one cycle of 72°C for 5min. In the second PCR step, each reaction contained 2.5 μl of the first PCR product, 25 μl of Kapa Hifi ready mix X2 (KAPA Biosystems), 2.5 μl 10 μM 5′ primer, and 2.5 μl 10 μM 3′ primer. The PCR program was 95°C for 5 min, 24 cycles of 94°C for 30s and 72°C for 30s, each, and one cycle of 72°C for 5min. Specific primers corresponding to the constant region of the plasmid were used. The 5′ primer contained a unique upstream 8-nt bin barcode sequence, and three different barcodes were used for each bin. The 3′ primer was common to all bins. Multiple PCR reaction products of each bin were combined. The concentration of the PCR samples was measured using a monochromator (Tecan i-control), and the samples were mixed in ratios corresponding to their ratio in the population, as defined when sorting the cells into the 16 bins. Sample preparation including gel elution and purification were performed as described above for amplicons from cDNA.</p></div><div id="sec-32" class="subsection"><h3>RNA purification, cDNA synthesis and sample preparation</h3><p id="p-81">For the cell population sorted for single integration of the reporter construct we performed RNA purification by centrifuging 10<sup>7</sup> cells, washing them with PBS, splitting into two tubes and purifying RNA using NucleoSpin RNA II kit (MACHEREY-NAGEL) according to the manufacturer’s protocol. We prepared cDNA in 2 reverse transcription reaction for each replicate using SuperScript® III First-Strand Synthesis System (Thermo Fisher Scientific) with random hexamer primers and 5 μg of input RNA (per reaction) according to the manufacturer protocol. For amplification of the library variants, 3 PCR reactions of 50 ul total volume were performed. Each reaction contained 5 ul cDNA, 25 μl of Kapa Hifi ready mix X2 (KAPA Biosystems), 2.5 μl 10 μM 5′ primer, and 2.5 μl 10 μM 3′ primer. The PCR program was 95°C for 5 min, 20 cycles of 94°C for 30s and 72°C for 30s, each, and one cycle of 72°C for 5min. Specific primers corresponding to the constant region upstream and downstream of the splice sites were used. The PCR products were separated from potential unspecific fragments by electrophoresis on a 1.5% agarose gel stained with EtBr, cut from the gel, and cleaned in 2 steps: gel extraction kit (Qiagen) and SPRI beads (Agencourt AMPure XP). The sample was assessed for size and purity at the Tapestation, using high sensitivity D1K screenTape (Agilent Technologies, Santa Clara, California). We used 20 ng library DNA for library preparation for NGS; specific Illumina adaptors were added, and DNA was amplified using 14 amplification cycles. The sample was reanalyzed using Tapestation.</p><p id="p-82">For RT-PCR analysis we performed RNA purification on 5 million cells from individually constructed and integrated reporter constructs, followed by cDNA synthesis as described above (one reaction). PCR was performed using primers within the mCherry and GFP coding region, yielding a PCR product of about 1000 bp length. 40 cycles of PCR were used to obtain a strong signal even for less prominent bands. PCR products were analyzed on a 1% agarose gel.</p></div></div><div class="section" id="sec-33"><h2 class="">Computational analyses</h2><div id="sec-34" class="subsection"><h3>Mapping next generation sequencing reads</h3><p id="p-83">To unambiguously identify the variant of origin, a unique 12-mer barcode sequence was placed at the 5’ end of each variable region. DNA was sequenced on a NextSeq-500 sequencer. We determined for each read (PE150) its bin barcode and its variant barcode and discarded all the reads that could not be assigned to a bin and a library variant of origin. In a second step, for each variant the mapped reads (paired-end, 150 bases for each mate) were aligned to the designed sequence (in the context of the reporter) and only those ones were counted that mapped without a mismatch over the whole length (disregarding the last 5 bases). Reads showing a mismatch, in particular a single nucleotide deletion or insertion, were quantified separately and if they passed the same threshold applied to correct library variants a PRF readout (% GFP fluorescence) was calculated for them. These additional variants were only used in comparisons with the original designed variant and not incorporated in the general pool of library variants considered for the analyses. For all three libraries (−1 PRF reporters, +1 PRF reporters and the stop-vector control) around 40 milllion successfully mapped reads were used for subsequent analysis. In addition, we sequenced the entire cell population using greater read lengths (PE300) to cover the entire length of the variable region and to obtain a measure of the fraction of reads per variant containing a synthesis or cloning error. Mapping was performed using custom-made Python scripts.</p></div><div id="sec-35" class="subsection"><h3>Computing a measure of frameshifting efficiency (% GFP fluorescence) and related readouts</h3><p id="p-84">We applied a number of filters to the raw sequencing data to reduce experimental noise. First, variants with less than 20 reads (perfect matches along the whole length) mapped across bins were removed. Second, for bins with a read count of less than three, the bin value was set to zero. Third, for each variant we set to zero bins surrounded by zero values as these constituted isolated bins unlikely to come from the actual distribution. Forth, to reduce bias coming from the “open end”-bins at the extreme values, we set their count to match their neighbor’s if it was higher, as these bins are defined as containing the tails of the distribution of variants with no GFP fluorescence and maximal GFP expression, respectively, and peaks in these extreme bins are considered experimental noise. For each variant, we normalized the values across the 16 bins by dividing the filtered read counts for each bin by the sum of all filtered read counts. To determine whether a distribution is unimodal we applied a Savitzky-Golay filter for smoothing the data and removing local maxima introduced by experimental noise. We detected peaks in the smoothed distribution by a simple approach in which a point is considered a maximum peak if it has the maximal value, and was preceded by a value lower by 5%. Variants without detectable peaks (due to the read number not reaching the threshold) were excluded at this stage. Variants with more than one peak were filtered out at the level of analysis. For each bin, we calculated the median of the log2 of GFP fluorescence intensity as measured by FACS for all the cells sorted into that bin and used this as the GFP fluorescence value associated with this bin. For each variant with one peak after smoothing, we calculated the weighted average for the distribution of reads across bins (using unsmoothed read counts normalized for each variant and the GFP fluorescence value for the bins, as described above), resulting in what is referred to in the main text and figures as the “GFP fluorescence” (in log2).</p><p id="p-85">As described in the main text and illustrated in Figure S2A-C, we set the lowest value obtained (the variant with the lowest weighted average of the distribution) to 0% GFP fluorescence and the highest value obtained (the variant with the highest weighted average of the distribution) to 100% GFP fluorescence. Accordingly, a percentage was assigned to each variant based on the GFP fluorescence (log2 of the weighted average of the reads distribution) of a variant as follows:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/13/469692/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/13/469692/embed/graphic-7.gif"/></noscript></span>

</span>
“% wild-type GFP fluorescence” of a variant was calculated as follows:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/13/469692/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/13/469692/embed/graphic-8.gif"/></noscript></span>

</span>
</p></div><div id="sec-36" class="subsection"><h3>Machine learning approaches</h3><p id="p-86">Machine learning procedures were carried out using the python scikit-learn (version 0.18.2) and XGBoost package. Initially, from all duplicated sequences (e.g. barcode control sets), which passed filtering, a single variant was randomly chosen for all subsequent steps to avoid biases resulting from having duplicated sequences. We created sets of variants corresponding either to the full rationally designed library (“full library”), the −1 PRF sites from the main set of events tested positive in our assay combined (“frameshifting events”) or to all variants based on one of these PRF events in isolation. 10% of the variants in the individual sets (20% in case of the HIV variants) were put aside and used only for evaluation of models built using the other 90% (80%). We chose Gradient Boosting Decision Trees (XGBoost (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chen and Guestrin, 2016</a>)) as the prediction algorithm because it can capture non-linear interactions between features, allows combining different types of continuous and categorical features and has proven to be a powerful approach in predicting the effect of regulatory regions (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Mikl et al., 2019</a>; <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Vainberg Slutskin et al., 2018</a>).</p><p id="p-87">For predictions based on secondary structure we used the fold function from the Vienna RNA package 2.0 and extracted both the minimal free energy (“dg”; calculated for different lengths of downstream (10-120 nt) regions) and the predicted pairedness for each position (“sec”; for each position in the downstream region this set of features contains the information whether the position is predicted (Vienna RNAfold) to be paired or not (0 or 1)).</p><p id="p-88">For predictions based on tRNA adaptation index (tAI), we computed (based on <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Waldman et al., 2010</a>) the tAI of codons in the original and −1 frame as well as the (per position) difference between frame - 1 and frame 0 and used the information for 14 starting from the first codon in the slippery site (in the case of the canonical −1 pattern XXXYYYZ, XXY in frame 0 and XXX in frame −1) as features.</p><p id="p-89">For predictions based on amino acid identity, information on the type of amino acids around the slippery site (from position −3 relative to the site of frameshifting until the end of the slippery site) was used (“unpolar”, “polar” or “charged”).</p><p id="p-90">Different hyperparameter settings for learning rate, n_estimators and max_depth were tested in a systematic and combinatorial fashion using 10-fold cross-validation for each set of variants and features independently. Typically around 100 tests were performed and the best set of hyperparameters used for subsequent steps.</p><p id="p-91">At the end, the model was tested by training it on the entire training set (90% of all relevant unique library variants, 80% in the case of HIV clinical isolates) and scoring the correlation (Pearson) of predicted vs. measured values based on the held-out test set (10% of relevant unique library variants from the relevant set, 20% in the case of HIV clinical isolates), which had not been used at any stage during development of the model. This approach was used for each of the features sets in isolation as well as all of the features combined.</p><p id="p-92">For classification, the same pipeline was used. Here, XGBClassifier was trained and the performance scored on the test set, reporting the area under the ROC curve as a metric.</p><p id="p-93">Feature importance and effect on the model was determined using SHAP analysis (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lundberg et al., 2020</a>).</p></div><div id="sec-37" class="subsection"><h3>General data analysis</h3><p id="p-94">For data analysis, we used python 2.7.11 with pandas 0.20.3, numpy 1.13.1, seaborn 0.6, scipy 0.17, scikit-learn 0.18.2 and shap 0.28.5. Confidence intervals were calculated by bootstrapping (1000 iterations).</p></div></div><div class="section" id="sec-39"><h2 class="">Data availability</h2><p id="p-95">All data generated in this study are available in the NCBI gene expression omnibus (GEO) under accession GSE145684.</p></div><div class="section" id="sec-40"><h2 class="">Code availability</h2><p id="p-96">The code used to process the data, generate the figures and train and test the predictive models will be made available as a GitHub repository (<a href="https://github.com/martinmikl/">https://github.com/martinmikl/</a>).</p></div><div class="section" id="sec-41"><h2 class="">Author contributions</h2><p id="p-98">Conceptualization: M.M., Y.P. and E.S.; Methodology, Software and Formal Analysis: M.M.; Investigation: M.M.; Writing: M.M., Y.P. and E.S.; Funding Acquisition: M.M., Y.P. and E.S.; Supervision: Y.P. and E.S.</p><p id="p-99">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-97">The authors thank Adina Weinberger, Orna Dahan, Alexey Gritsenko and Roni Rak for helpful discussions, Ronit Nir, Tali Avnit-Sagi and Maya Lotan-Pompan for technical advice and the Moshe Oren lab for advice and reagents. This work was supported by an EMBO long-term fellowship (to M.M.). E.S. is supported by the Crown Human Genome Center; the Else Kroener Fresenius Foundation; D. L. Schwarz; J. N. Halpern; L. Steinberg; J. Benattar; Aliza Moussaieff; Adelis Foundation; and grants funded by the European Research Council and the Israel Science Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">We now focus exclusively on the extensive comparative analysis of programmed ribosomal frameshifting including the screening of HIV clinical isolates and the prediction of frameshifting potential and frameshifting rates (Figures 1, 2 and 4 in the previous version of the manuscript). We now provide additional controls and readouts (e.g. analysis of RNA expression levels and splicing) and substantially expanded our analyses (new Figures 1-6).</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Advani</span>,  <span class="cit-name-given-names">V.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression. BioEssays News Rev</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Dev. Biol</abbr>. <span class="cit-vol">38</span>, <span class="cit-fpage">21</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BDev.%2BBiol%26rft.volume%253D38%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.2" data-doi="10.1261/rna.5113603"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baril</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dulude</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gendron</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemay</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brakier-Gingras</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Efficiency of a programmed −1 ribosomal frameshift in the different subtypes of the human immunodeficiency virus type 1 group M</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1246</span>–<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DBARIL%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D1246%26rft.epage%253D1253%26rft.atitle%253DEfficiency%2Bof%2Ba%2Bprogrammed%2B-1%2Bribosomal%2Bframeshift%2Bin%2Bthe%2Bdifferent%2Bsubtypes%2Bof%2Bthe%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Bgroup%2BM%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5113603%26rft_id%253Dinfo%253Apmid%252F13130138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjkvMTAvMTI0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.3" data-doi="10.1186/1471-2164-9-339"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hepler</span>,  <span class="cit-name-given-names">N.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">PRFdb: a database of computationally predicted eukaryotic programmed −1 ribosomal frameshift signals</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBelew%26rft.auinit1%253DA.%2BT.%26rft.volume%253D9%26rft.spage%253D339%26rft.epage%253D339%26rft.atitle%253DPRFdb%253A%2Ba%2Bdatabase%2Bof%2Bcomputationally%2Bpredicted%2Beukaryotic%2Bprogrammed%2B-1%2Bribosomal%2Bframeshift%2Bsignals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-339%26rft_id%253Dinfo%253Apmid%252F18637175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18637175&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.4" data-doi="10.1093/nar/gkq1220"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Advani</span>,  <span class="cit-name-given-names">V.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Endogenous ribosomal frameshift signals operate as mRNA destabilizing elements through at least two molecular pathways in yeast</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">39</span>, <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBelew%26rft.auinit1%253DA.%2BT.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D2799%26rft.epage%253D2808%26rft.atitle%253DEndogenous%2Bribosomal%2Bframeshift%2Bsignals%2Boperate%2Bas%2BmRNA%2Bdestabilizing%2Belements%2Bthrough%2Bat%2Bleast%2Btwo%2Bmolecular%2Bpathways%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1220%26rft_id%253Dinfo%253Apmid%252F21109528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21109528&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.5" data-doi="10.1038/nature13429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meskauskas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musalgaonkar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Advani</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sulima</span>,  <span class="cit-name-given-names">S.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kasprzak</span>,  <span class="cit-name-given-names">W.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>,  <span class="cit-name-given-names">B.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13429%26rft_id%253Dinfo%253Apmid%252F25043019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bidou</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stahl</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grima</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cassan</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rousset</span>,  <span class="cit-name-given-names">J.P.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">In vivo HIV-1 frameshifting efficiency is directly related to the stability of the stem-loop stimulatory signal</span>. <abbr class="cit-jnl-abbrev">RNA N. Y. N</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">1153</span>–<span class="cit-lpage">1158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BN.%2BY.%2BN%26rft.volume%253D3%26rft.spage%253D1153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.7" data-doi="10.1128/JVI.78.4.2082-2087.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biswas</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pacchia</span>,  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dougherty</span>,  <span class="cit-name-given-names">J.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Peltz</span>,  <span class="cit-name-given-names">S.W.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">The Human Immunodeficiency Virus Type 1 Ribosomal Frameshifting Site Is an Invariant Sequence Determinant and an Important Target for Antiviral Therapy</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">2082</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBiswas%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D2082%26rft.epage%253D2087%26rft.atitle%253DThe%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%2BRibosomal%2BFrameshifting%2BSite%2BIs%2Ban%2BInvariant%2BSequence%2BDeterminant%2Band%2Ban%2BImportant%2BTarget%2Bfor%2BAntiviral%2BTherapy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.4.2082-2087.2004%26rft_id%253Dinfo%253Apmid%252F14747573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzQvMjA4MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.8" data-doi="10.1517/14728222.2012.665879"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brakier-Gingras</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charbonneau</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Targeting frameshifting in the human immunodeficiency virus</span>. <abbr class="cit-jnl-abbrev">Expert Opin. Ther. Targets</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">249</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2Bopinion%2Bon%2Btherapeutic%2Btargets%26rft.stitle%253DExpert%2BOpin%2BTher%2BTargets%26rft.aulast%253DBrakier-Gingras%26rft.auinit1%253DL.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D258%26rft.atitle%253DTargeting%2Bframeshifting%2Bin%2Bthe%2Bhuman%2Bimmunodeficiency%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1517%252F14728222.2012.665879%26rft_id%253Dinfo%253Apmid%252F22404160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1517/14728222.2012.665879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22404160&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301456700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.9" data-doi="10.1101/sqb.2001.66.233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pennell</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Structure and function of the stimulatory RNAs involved in programmed eukaryotic-1 ribosomal frameshifting. Cold Spring Harb</span>. <abbr class="cit-jnl-abbrev">Symp. Quant. Biol</abbr>. <span class="cit-vol">66</span>, <span class="cit-fpage">233</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DBRIERLEY%26rft.auinit1%253DI.%26rft.volume%253D66%26rft.issue%253D0%26rft.spage%253D233%26rft.epage%253D248%26rft.atitle%253DStructure%2Band%2BFunction%2Bof%2Bthe%2BStimulatory%2BRNAs%2BInvolved%2Bin%2BProgrammed%2BEukaryotic%2B-1%2BRibosomal%2BFrameshifting%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2001.66.233%26rft_id%253Dinfo%253Apmid%252F12762025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjY2LzAvMjMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.10" data-doi="10.1016/0092-8674(89)90124-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Digard</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">S.C.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">57</span>, <span class="cit-fpage">537</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D537%26rft.epage%253D547%26rft.atitle%253DCharacterization%2Bof%2Ban%2Befficient%2Bcoronavirus%2Bribosomal%2Bframeshifting%2Bsignal%253A%2Brequirement%2Bfor%2Ban%2BRNA%2Bpseudoknot.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990124-4%26rft_id%253Dinfo%253Apmid%252F2720781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90124-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2720781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U743600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.11" data-doi="10.1016/j.cell.2014.04.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katunin</span>,  <span class="cit-name-given-names">V.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belardinelli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M.V.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Programmed −1 frameshifting by kinetic partitioning during impeded translocation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>, <span class="cit-fpage">1619</span>–<span class="cit-lpage">1631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1619%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.04.041%26rft_id%253Dinfo%253Apmid%252F24949973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.04.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24949973&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340941900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.12" data-doi="10.1016/j.tibs.2015.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M.V.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Changed in translation: mRNA recoding by −1 programmed ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Trends Biochem. Sci</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci%26rft.volume%253D40%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2015.03.006%26rft_id%253Dinfo%253Apmid%252F25850333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2015.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25850333&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.13" data-doi="10.1016/j.molcel.2017.04.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wohlgemuth</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korniy</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearson</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M.V.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Conditional Switch between Frameshifting Regimes upon Translation of dnaX mRNA</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">558</span>-<span class="cit-lpage">567</span>.e4.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D66%26rft.spage%253D558%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.04.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Guestrin</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">XGBoost: A Scalable Tree Boosting System. Proc. 22nd ACM SIGKDD Int</span>. <abbr class="cit-jnl-abbrev">Conf. Knowl. Discov. Data Min. - KDD</abbr> <span class="cit-vol">16</span> <span class="cit-fpage">785</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConf.%2BKnowl.%2BDiscov.%2BData%2BMin.%2B-%2BKDD%26rft.volume%253D16%26rft.spage%253D785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.15" data-doi="10.1038/nature13428"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johansson</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Leary</span>,  <span class="cit-name-given-names">S.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Puglisi</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Dynamic pathways of −1 translational frameshifting</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>, <span class="cit-fpage">328</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13428%26rft_id%253Dinfo%253Apmid%252F24919156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24919156&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.16" data-doi="10.1126/sciadv.aax6969"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Loughlin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Puglisi</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">The energy landscape of −1 ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Sci. Adv</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">eaax6969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DChoi%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253Deaax6969%26rft.epage%253Deaax6969%26rft.atitle%253DThe%2Benergy%2Blandscape%2Bof%2B-1%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aax6969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNi8xL2VhYXg2OTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.17" data-doi="10.1074/jbc.M705676200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jänicke</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gottesbühren</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleffmann</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legge</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poole</span>,  <span class="cit-name-given-names">E.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tate</span>,  <span class="cit-name-given-names">W.P.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Mammalian Gene PEG10 Expresses Two Reading Frames by High Efficiency –1 Frameshifting in Embryonic-associated Tissues</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">282</span>, <span class="cit-fpage">37359</span>–<span class="cit-lpage">37369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M705676200%26rft_id%253Dinfo%253Apmid%252F17942406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTIvMzczNTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMy80Njk2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.18" data-doi="10.1093/nar/22.12.2304"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">ten Dam</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pleij</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">Identification and analysis of the pseudoknot-containing gag-pro ribosomal frameshift signal of simian retrovirus-1</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">22</span>, <span class="cit-fpage">2304</span>–<span class="cit-lpage">2310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DDam%26rft.auinit1%253DE.%2Bt.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D2304%26rft.epage%253D2310%26rft.atitle%253DIdentification%2Band%2Banalysis%2Bof%2Bthe%2Bpseudoknot-containing%2Bgag-pro%2Bribosomal%2Bframeshift%2Bsignal%2Bof%2Bsimian%2Bretrovirus-1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.12.2304%26rft_id%253Dinfo%253Apmid%252F8036158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.12.2304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8036158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NV93100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.19" data-doi="10.1016/B978-0-12-386497-0.00004-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Control of gene expression by translational recoding</span>. <abbr class="cit-jnl-abbrev">Adv. Protein Chem. Struct. Biol</abbr>. <span class="cit-vol">86</span>, <span class="cit-fpage">129</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAdv%2BProtein%2BChem%2BStruct%2BBiol%26rft.aulast%253DDinman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D86%26rft.spage%253D129%26rft.epage%253D149%26rft.atitle%253DControl%2Bof%2Bgene%2Bexpression%2Bby%2Btranslational%2Brecoding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-386497-0.00004-9%26rft_id%253Dinfo%253Apmid%252F22243583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-386497-0.00004-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243583&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wickner</span>,  <span class="cit-name-given-names">R.B.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">Ribosomal frameshifting efficiency and gag/gag-pol ratio are critical for yeast M1 double-stranded RNA virus propagation</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">66</span>, <span class="cit-fpage">3669</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DDinman%26rft.auinit1%253DJ%2BD%26rft.volume%253D66%26rft.issue%253D6%26rft.spage%253D3669%26rft.epage%253D3676%26rft.atitle%253DRibosomal%2Bframeshifting%2Befficiency%2Band%2Bgag%252Fgag-pol%2Bratio%2Bare%2Bcritical%2Bfor%2Byeast%2BM1%2Bdouble-stranded%2BRNA%2Bvirus%2Bpropagation.%26rft_id%253Dinfo%253Apmid%252F1583726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY2LzYvMzY2OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.21" data-doi="10.1006/jmbi.1997.1127"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hoffman</span>,  <span class="cit-name-given-names">D.W.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Base-pairings within the RNA pseudoknot associated with the simian retrovirus-1 gag-pro frameshift site</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">270</span>, <span class="cit-fpage">464</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDu%26rft.auinit1%253DZ.%26rft.volume%253D270%26rft.issue%253D3%26rft.spage%253D464%26rft.epage%253D470%26rft.atitle%253DBase-pairings%2Bwithin%2Bthe%2BRNA%2Bpseudoknot%2Bassociated%2Bwith%2Bthe%2Bsimian%2Bretrovirus-1%2Bgag-pro%2Bframeshift%2Bsite.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.1127%26rft_id%253Dinfo%253Apmid%252F9237911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.1127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9237911&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XM19300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">K.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goff</span>,  <span class="cit-name-given-names">S.P.</span></span> (<span class="cit-pub-date">1988</span>). <span class="cit-article-title">Expression of the gag-pol fusion protein of Moloney murine leukemia virus without gag protein does not induce virion formation or proteolytic processing</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">62</span>, <span class="cit-fpage">2179</span>–<span class="cit-lpage">2182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFelsenstein%26rft.auinit1%253DK%2BM%26rft.volume%253D62%26rft.issue%253D6%26rft.spage%253D2179%26rft.epage%253D2182%26rft.atitle%253DExpression%2Bof%2Bthe%2Bgag-pol%2Bfusion%2Bprotein%2Bof%2BMoloney%2Bmurine%2Bleukemia%2Bvirus%2Bwithout%2Bgag%2Bprotein%2Bdoes%2Bnot%2Binduce%2Bvirion%2Bformation%2Bor%2Bproteolytic%2Bprocessing.%26rft_id%253Dinfo%253Apmid%252F2452901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjYyLzYvMjE3OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.23" data-doi="10.1186/1743-422X-6-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">A conserved predicted pseudoknot in the NS2A-encoding sequence of West Nile and Japanese encephalitis flaviviruses suggests NS1’ may derive from ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Virol. J</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DFirth%26rft.auinit1%253DA.%2BE.%26rft.volume%253D6%26rft.spage%253D14%26rft.epage%253D14%26rft.atitle%253DA%2Bconserved%2Bpredicted%2Bpseudoknot%2Bin%2Bthe%2BNS2A-encoding%2Bsequence%2Bof%2BWest%2BNile%2Band%2BJapanese%2Bencephalitis%2Bflaviviruses%2Bsuggests%2BNS1%2527%2Bmay%2Bderive%2Bfrom%2Bribosomal%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-6-14%26rft_id%253Dinfo%253Apmid%252F19196463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-6-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19196463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.24" data-doi="10.1099/vir.0.042499-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Non-canonical translation in RNA viruses</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-vol">93</span>, <span class="cit-fpage">1385</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.042499-0%26rft_id%253Dinfo%253Apmid%252F22535777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.042499-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22535777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306348900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.25" data-doi="10.1016/0022-2836(92)90713-T"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gallant</span>,  <span class="cit-name-given-names">J.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lindsley</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">Leftward ribosome frameshifting at a hungry codon</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">223</span>, <span class="cit-fpage">31</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGallant%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D223%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D40%26rft.atitle%253DLeftward%2Bribosome%2Bframeshifting%2Bat%2Ba%2Bhungry%2Bcodon.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252892%252990713-T%26rft_id%253Dinfo%253Apmid%252F1731076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(92)90713-T&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1731076&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992GZ96400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.26" data-doi="10.1128/JVI.00149-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garcia-Miranda</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benner</span>,  <span class="cit-name-given-names">B.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blume</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherer</span>,  <span class="cit-name-given-names">N.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Stability of HIV Frameshift Site RNA Correlates with Frameshift Efficiency and Decreased Virus Infectivity</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">90</span>, <span class="cit-fpage">6906</span>–<span class="cit-lpage">6917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00149-16%26rft_id%253Dinfo%253Apmid%252F27194769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82OTA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.27" data-doi="10.1016/j.celrep.2012.12.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Girstmair</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saffert</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rode</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Czech</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bannert</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ignatova</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Depletion of Cognate Charged Transfer RNA Causes Translational Frameshifting within the Expanded CAG Stretch in Huntingtin</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">148</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D3%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.12.019%26rft_id%253Dinfo%253Apmid%252F23352662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.12.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23352662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321891400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halma</span>,  <span class="cit-name-given-names">M.T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cappellano</span>,  <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neupane</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Woodside</span>,  <span class="cit-name-given-names">M.T.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Complex dynamics under tension in a high-efficiency frameshift stimulatory structure</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hemelaar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gouws</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghys</span>,  <span class="cit-name-given-names">P.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Osmanov</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Global and regional distribution of HIV-1 genetic subtypes and recombinants in 2004</span>. <abbr class="cit-jnl-abbrev">AIDS Lond. Engl</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">W13</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BLond.%2BEngl%26rft.volume%253D20%26rft.spage%253D13W%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Priwitzer</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">P.F.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Prediction of locally stable RNA secondary structures for genome-wide surveys</span>. <abbr class="cit-jnl-abbrev">Bioinforma. Oxf. Engl</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">186</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinforma.%2BOxf.%2BEngl%26rft.volume%253D20%26rft.spage%253D186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.31" data-doi="10.1128/JVI.72.6.4819-4824.1998"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">S.R.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Importance of ribosomal frameshifting for human immunodeficiency virus type 1 particle assembly and replication</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">72</span>, <span class="cit-fpage">4819</span>–<span class="cit-lpage">4824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHung%26rft.auinit1%253DM.%26rft.volume%253D72%26rft.issue%253D6%26rft.spage%253D4819%26rft.epage%253D4824%26rft.atitle%253DImportance%2Bof%2BRibosomal%2BFrameshifting%2Bfor%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%25A0Particle%2BAssembly%2Band%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.72.6.4819-4824.1998%26rft_id%253Dinfo%253Apmid%252F9573247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcyLzYvNDgxOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.32" data-doi="10.1093/nar/gkm035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">I.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Ribosomal frameshifting in decoding antizyme mRNAs from yeast and protists to humans: close to 300 cases reveal remarkable diversity despite underlying conservation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">35</span>, <span class="cit-fpage">1842</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm035%26rft_id%253Dinfo%253Apmid%252F17332016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17332016&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246123600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.33" data-doi="10.1006/geno.1998.5434"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">I.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">A second mammalian antizyme: conservation of programmed ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">52</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DIvanov%26rft.auinit1%253DI.%2BP.%26rft.volume%253D52%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D129%26rft.atitle%253DA%2Bsecond%2Bmammalian%2Bantizyme%253A%2Bconservation%2Bof%2Bprogrammed%2Bribosomal%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fgeno.1998.5434%26rft_id%253Dinfo%253Apmid%252F9782076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/geno.1998.5434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9782076&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076421700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.34" data-doi="10.1093/emboj/19.8.1907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">I.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsufuji</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Conservation of polyamine regulation by translational frameshifting from yeast to mammals</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">1907</span>–<span class="cit-lpage">1917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DIvanov%26rft.auinit1%253DI.%2BP.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D1907%26rft.epage%253D1917%26rft.atitle%253DConservation%2Bof%2Bpolyamine%2Bregulation%2Bby%2Btranslational%2Bframeshifting%2Bfrom%2Byeast%2Bto%2Bmammals%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F19.8.1907%26rft_id%253Dinfo%253Apmid%252F10775274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxOS84LzE5MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMy80Njk2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.35" data-doi="10.1261/rna.2245906"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">I.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Evolutionary specialization of recoding: Frameshifting in the expression of S. cerevisiae antizyme mRNA is via an atypical antizyme shift site but is still +1</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">332</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2245906%26rft_id%253Dinfo%253Apmid%252F16431984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEyLzMvMzMyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.36" data-doi="10.1126/science.2416054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacks</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Varmus</span>,  <span class="cit-name-given-names">H.E.</span></span> (<span class="cit-pub-date">1985</span>). <span class="cit-article-title">Expression of the Rous sarcoma virus pol gene by ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">230</span>, <span class="cit-fpage">1237</span>–<span class="cit-lpage">1242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJacks%26rft.auinit1%253DT%26rft.volume%253D230%26rft.issue%253D4731%26rft.spage%253D1237%26rft.epage%253D1242%26rft.atitle%253DExpression%2Bof%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Bpol%2Bgene%2Bby%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2416054%26rft_id%253Dinfo%253Apmid%252F2416054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzAvNDczMS8xMjM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.37" data-doi="10.1038/331280a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacks</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Power</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masiarz</span>,  <span class="cit-name-given-names">F.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luciw</span>,  <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barr</span>,  <span class="cit-name-given-names">P.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Varmus</span>,  <span class="cit-name-given-names">H.E.</span></span> (<span class="cit-pub-date">1988a</span>). <span class="cit-article-title">Characterization of ribosomal frameshifting in HIV-1 gag-pol expression</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">331</span>, <span class="cit-fpage">280</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJacks%26rft.auinit1%253DT.%26rft.volume%253D331%26rft.issue%253D6153%26rft.spage%253D280%26rft.epage%253D283%26rft.atitle%253DCharacterization%2Bof%2Bribosomal%2Bframeshifting%2Bin%2BHIV-1%2Bgag-pol%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F331280a0%26rft_id%253Dinfo%253Apmid%252F2447506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/331280a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2447506&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988L714000068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.38" data-doi="10.1016/0092-8674(88)90031-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacks</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhani</span>,  <span class="cit-name-given-names">H.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masiarz</span>,  <span class="cit-name-given-names">F.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Varmus</span>,  <span class="cit-name-given-names">H.E.</span></span> (<span class="cit-pub-date">1988b</span>). <span class="cit-article-title">Signals for ribosomal frameshifting in the Rous sarcoma virus gag-pol region</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">55</span>, <span class="cit-fpage">447</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJacks%26rft.auinit1%253DT.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D447%26rft.epage%253D458%26rft.atitle%253DSignals%2Bfor%2Bribosomal%2Bframeshifting%2Bin%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Bgag-pol%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252888%252990031-1%26rft_id%253Dinfo%253Apmid%252F2846182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(88)90031-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2846182&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q826900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.39" data-doi="10.1093/bioinformatics/btu649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janssen</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Giegerich</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">The RNA shapes studio</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">423</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu649%26rft_id%253Dinfo%253Apmid%252F25273103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25273103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.40" data-doi="10.3389/fgene.2012.00242"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ketteler</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">On programmed ribosomal frameshifting: the alternative proteomes</span>. <abbr class="cit-jnl-abbrev">Front. Genet</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DKetteler%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.spage%253D242%26rft.epage%253D242%26rft.atitle%253DOn%2Bprogrammed%2Bribosomal%2Bframeshifting%253A%2Bthe%2Balternative%2Bproteomes.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00242%26rft_id%253Dinfo%253Apmid%252F23181069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23181069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kleinkauf</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Houwaart</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Backofen</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2015a</span>). <span class="cit-article-title">antaRNA – Multi-objective inverse folding of pseudoknot RNA using ant-colony optimization</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.42" data-doi="10.1093/bioinformatics/btv319"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kleinkauf</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Backofen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2015b</span>). <span class="cit-article-title">antaRNA: ant colony-based RNA sequence design</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">3114</span>–<span class="cit-lpage">3121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv319%26rft_id%253Dinfo%253Apmid%252F26023105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26023105&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kollmus</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Honigman</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panet</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hauser</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">The sequences of and distance between two cis-acting signals determine the efficiency of ribosomal frameshifting in human immunodeficiency virus type 1 and human T-cell leukemia virus type II in vivo</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">68</span>, <span class="cit-fpage">6087</span>–<span class="cit-lpage">6091</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKollmus%26rft.auinit1%253DH%26rft.volume%253D68%26rft.issue%253D9%26rft.spage%253D6087%26rft.epage%253D6091%26rft.atitle%253DThe%2Bsequences%2Bof%2Band%2Bdistance%2Bbetween%2Btwo%2Bcis-acting%2Bsignals%2Bdetermine%2Bthe%2Befficiency%2Bof%2Bribosomal%2Bframeshifting%2Bin%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Band%2Bhuman%2BT-cell%2Bleukemia%2Bvirus%2Btype%2BII%2Bin%2Bvivo.%26rft_id%253Dinfo%253Apmid%252F8057488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY4LzkvNjA4NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korniy</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goyal</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samatova</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peske</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pöhlmann</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rodnina</span>,  <span class="cit-name-given-names">M.V.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Modulation of HIV-1 Gag/Gag-Pol frameshifting by tRNA abundance</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">47</span>, <span class="cit-fpage">5210</span>–<span class="cit-lpage">5222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253D5210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.45" data-doi="10.1038/nature10393"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kurian</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palanimurugan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gödderz</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dohmen</span>,  <span class="cit-name-given-names">R.J.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Polyamine sensing by nascent ornithine decarboxylase antizyme stimulates decoding of its mRNA</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>, <span class="cit-fpage">490</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKurian%26rft.auinit1%253DL.%26rft.volume%253D477%26rft.issue%253D7365%26rft.spage%253D490%26rft.epage%253D494%26rft.atitle%253DPolyamine%2Bsensing%2Bby%2Bnascent%2Bornithine%2Bdecarboxylase%2Bantizyme%2Bstimulates%2Bdecoding%2Bof%2Bits%2BmRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10393%26rft_id%253Dinfo%253Apmid%252F21900894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21900894&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295080500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.46" data-doi="10.1093/nar/gks629"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">R.J.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Spacer-length dependence of programmed −1 or −2 ribosomal frameshifting on a U6A heptamer supports a role for messenger RNA (mRNA) tension in frameshifting</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">8674</span>–<span class="cit-lpage">8689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks629%26rft_id%253Dinfo%253Apmid%252F22743270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22743270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309464300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernhart</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Höner Zu Siederdissen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tafer</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flamm</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">P.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>,  <span class="cit-name-given-names">I.L.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">ViennaRNA Package 2.0. Algorithms</span> <abbr class="cit-jnl-abbrev">Mol. Biol. AMB</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BAMB%26rft.volume%253D6%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lundberg</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeGrave</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prutkin</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nair</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katz</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Himmelfarb</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bansal</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.-I.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">From local explanations to global understanding with explainable AI for trees</span>. <abbr class="cit-jnl-abbrev">Nat. Mach. Intell</abbr>. <span class="cit-vol">2</span>, <span class="cit-fpage">56</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMach.%2BIntell%26rft.volume%253D2%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lyon</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aguilera</span>,  <span class="cit-name-given-names">L.U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morisaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munsky</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stasevich</span>,  <span class="cit-name-given-names">T.J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Live-Cell Single RNA Imaging Reveals Bursts of Translational Frameshifting</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">172</span>-<span class="cit-lpage">183</span>.e9.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D75%26rft.spage%253D172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.50" data-doi="10.1093/nar/gki299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manktelow</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shigemoto</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Characterization of the frameshift signal of Edr, a mammalian example of programmed −1 ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">33</span>, <span class="cit-fpage">1553</span>–<span class="cit-lpage">1563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DManktelow%26rft.auinit1%253DE.%26rft.volume%253D33%26rft.issue%253D5%26rft.spage%253D1553%26rft.epage%253D1563%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bframeshift%2Bsignal%2Bof%2BEdr%252C%2Ba%2Bmammalian%2Bexample%2Bof%2Bprogrammed%2B-1%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki299%26rft_id%253Dinfo%253Apmid%252F15767280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15767280&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228080000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.51" data-doi="10.1016/j.jmb.2007.08.033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marcheschi</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Staple</span>,  <span class="cit-name-given-names">D.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Programmed Ribosomal Frameshifting in SIV is Induced by a Highly Structured RNA Stem-Loop</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">373</span>, <span class="cit-fpage">652</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMarcheschi%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D373%26rft.issue%253D3%26rft.spage%253D652%26rft.epage%253D663%26rft.atitle%253DProgrammed%2Bribosomal%2Bframeshifting%2Bin%2BSIV%2Bis%2Binduced%2Bby%2Ba%2Bhighly%2Bstructured%2BRNA%2Bstem-loop.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2007.08.033%26rft_id%253Dinfo%253Apmid%252F17868691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2007.08.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17868691&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250440000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.52" data-doi="10.1006/jmbi.1998.2186"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marczinke</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vidakovic</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloys</span>,  <span class="cit-name-given-names">A.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Secondary structure and mutational analysis of the ribosomal frameshift signal of rous sarcoma virus</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">284</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMarczinke%26rft.auinit1%253DB.%26rft.volume%253D284%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D225%26rft.atitle%253DSecondary%2Bstructure%2Band%2Bmutational%2Banalysis%2Bof%2Bthe%2Bribosomal%2Bframeshift%2Bsignal%2Bof%2Brous%2Bsarcoma%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.2186%26rft_id%253Dinfo%253Apmid%252F9813113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.2186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9813113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077087100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.53" data-doi="10.1016/0092-8674(95)90450-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsufuji</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsufuji</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyazaki</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hayashi</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1995</span>). <span class="cit-article-title">Autoregulatory frameshifting in decoding mammalian ornithine decarboxylase antizyme</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">80</span>, <span class="cit-fpage">51</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMatsufuji%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D60%26rft.atitle%253DAutoregulatory%2Bframeshifting%2Bin%2Bdecoding%2Bmammalian%2Bornithine%2Bdecarboxylase%2Bantizyme.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990450-6%26rft_id%253Dinfo%253Apmid%252F7813017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90450-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7813017&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QB91000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.54" data-doi="10.1128/JVI.01979-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melian</span>,  <span class="cit-name-given-names">E.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinzman</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagasaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wills</span>,  <span class="cit-name-given-names">N.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nouwens</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blitvich</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Funk</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">NS1′ of Flaviviruses in the Japanese Encephalitis Virus Serogroup Is a Product of Ribosomal Frameshifting and Plays a Role in Viral Neuroinvasiveness</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">84</span>, <span class="cit-fpage">1641</span>–<span class="cit-lpage">1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01979-09%26rft_id%253Dinfo%253Apmid%252F19906906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzMvMTY0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.55" data-doi="10.1371/journal.ppat.1004447"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melian</span>,  <span class="cit-name-given-names">E.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall-Mendelin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Owens</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosco-Lauth</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagasaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rudd</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brault</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowen</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Programmed ribosomal frameshift alters expression of west nile virus genes and facilitates virus replication in birds and mosquitoes</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">e1004447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004447%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004447%26rft_id%253Dinfo%253Apmid%252F25375107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375107&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.56" data-doi="10.1016/j.molcel.2016.12.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meydan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klepacki</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karthikeyan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Margus</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briggs</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vázquez-Laslop</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Programmed Ribosomal Frameshifting Generates a Copper Transporter and a Copper Chaperone from the Same Gene</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">65</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D65%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.12.008%26rft_id%253Dinfo%253Apmid%252F28107647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28107647&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mikl</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamburg</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pilpel</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Dissecting splicing decisions and cell-to-cell variability with designed sequence libraries</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">4572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D10%26rft.spage%253D4572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.58" data-doi="10.1074/jbc.M116.735613"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moomau</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musalgaonkar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>,  <span class="cit-name-given-names">Y.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Structural and Functional Characterization of Programmed Ribosomal Frameshift Signals in West Nile Virus Strains Reveals High Structural Plasticity Among cis-Acting RNA Elements</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">291</span>, <span class="cit-fpage">15788</span>–<span class="cit-lpage">15795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M116.735613%26rft_id%253Dinfo%253Apmid%252F27226636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTEvMzAvMTU3ODgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMy80Njk2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.59" data-doi="10.1016/j.compbiolchem.2007.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byun</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">FSDB: a frameshift signal database</span>. <abbr class="cit-jnl-abbrev">Comput. Biol. Chem</abbr>. <span class="cit-vol">31</span>, <span class="cit-fpage">298</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DMoon%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D298%26rft.epage%253D302%26rft.atitle%253DFSDB%253A%2Ba%2Bframeshift%2Bsignal%2Bdatabase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.compbiolchem.2007.05.004%26rft_id%253Dinfo%253Apmid%252F17631420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.compbiolchem.2007.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17631420&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249192700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.60" data-doi="10.1093/nar/gks1254"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mouzakis</span>,  <span class="cit-name-given-names">K.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lang</span>,  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vander Meulen</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Easterday</span>,  <span class="cit-name-given-names">P.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">HIV-1 frameshift efficiency is primarily determined by the stability of base pairs positioned at the mRNA entrance channel of the ribosome</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">41</span>, <span class="cit-fpage">1901</span>–<span class="cit-lpage">1913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1254%26rft_id%253Dinfo%253Apmid%252F23248007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23248007&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316351800050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.61" data-doi="10.1128/JVI.67.1.196-203.1993"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nam</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Copeland</span>,  <span class="cit-name-given-names">T.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatanaka</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Oroszlan</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Characterization of ribosomal frameshifting for expression of pol gene products of human T-cell leukemia virus type I</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">67</span>, <span class="cit-fpage">196</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNam%26rft.auinit1%253DS%2BH%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D196%26rft.epage%253D203%26rft.atitle%253DCharacterization%2Bof%2Bribosomal%2Bframeshifting%2Bfor%2Bexpression%2Bof%2Bpol%2Bgene%2Bproducts%2Bof%2Bhuman%2BT-cell%2Bleukemia%2Bvirus%2Btype%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.67.1.196-203.1993%26rft_id%253Dinfo%253Apmid%252F8416368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjY3LzEvMTk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.62" data-doi="10.1128/JVI.01846-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nikolic</span>,  <span class="cit-name-given-names">E.I.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vidakovic</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Irigoyen</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Modulation of ribosomal frameshifting frequency and its effect on the replication of Rous sarcoma virus</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">86</span>, <span class="cit-fpage">11581</span>–<span class="cit-lpage">11594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01846-12%26rft_id%253Dinfo%253Apmid%252F22896611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMS8xMTU4MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ono</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyata</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ushikubo</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1986</span>). <span class="cit-article-title">Nucleotide sequence of human endogenous retrovirus genome related to the mouse mammary tumor virus genome</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">60</span>, <span class="cit-fpage">589</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DOno%26rft.auinit1%253DM%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D589%26rft.epage%253D598%26rft.atitle%253DNucleotide%2Bsequence%2Bof%2Bhuman%2Bendogenous%2Bretrovirus%2Bgenome%2Brelated%2Bto%2Bthe%2Bmouse%2Bmammary%2Btumor%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F3021993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjYwLzIvNTg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parkin</span>,  <span class="cit-name-given-names">N.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chamorro</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Varmus</span>,  <span class="cit-name-given-names">H.E.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">Human immunodeficiency virus type 1 gag-pol frameshifting is dependent on downstream mRNA secondary structure: demonstration by expression in vivo</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">66</span>, <span class="cit-fpage">5147</span>–<span class="cit-lpage">5151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DParkin%26rft.auinit1%253DN%2BT%26rft.volume%253D66%26rft.issue%253D8%26rft.spage%253D5147%26rft.epage%253D5151%26rft.atitle%253DHuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Bgag-pol%2Bframeshifting%2Bis%2Bdependent%2Bon%2Bdownstream%2BmRNA%2Bsecondary%2Bstructure%253A%2Bdemonstration%2Bby%2Bexpression%2Bin%2Bvivo.%26rft_id%253Dinfo%253Apmid%252F1321294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY2LzgvNTE0NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.65" data-doi="10.1371/journal.pbio.0030172"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plant</span>,  <span class="cit-name-given-names">E.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pérez-Alvarado</span>,  <span class="cit-name-given-names">G.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hennig</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">A three-stemmed mRNA pseudoknot in the SARS coronavirus frameshift signal</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">e172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DPlant%26rft.auinit1%253DE.%2BP.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De172%26rft.epage%253De172%26rft.atitle%253DA%2Bthree-stemmed%2BmRNA%2Bpseudoknot%2Bin%2Bthe%2BSARS%2Bcoronavirus%2Bframeshift%2Bsignal.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030172%26rft_id%253Dinfo%253Apmid%252F15884978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15884978&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.66" data-doi="10.1371/journal.ppat.1003294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pollom</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dang</span>,  <span class="cit-name-given-names">K.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>,  <span class="cit-name-given-names">E.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorelick</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burch</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weeks</span>,  <span class="cit-name-given-names">K.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Swanstrom</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Comparison of SIV and HIV-1 Genomic RNA Structures Reveals Impact of Sequence Evolution on Conserved and Non-Conserved Structural Motifs</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">e1003294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003294%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003294%26rft_id%253Dinfo%253Apmid%252F23593004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593004&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.67" data-doi="10.1073/pnas.1204114109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foster</span>,  <span class="cit-name-given-names">D.A.N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Woodside</span>,  <span class="cit-name-given-names">M.T.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Programmed −1 frameshifting efficiency correlates with RNA pseudoknot conformational plasticity, not resistance to mechanical unfolding</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">109</span>, <span class="cit-fpage">16167</span>–<span class="cit-lpage">16172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1204114109%26rft_id%253Dinfo%253Apmid%252F22988073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQwLzE2MTY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cappellano</span>,  <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tittle</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rezajooei</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouleau</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sikkema</span>,  <span class="cit-name-given-names">W.K.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Woodside</span>,  <span class="cit-name-given-names">M.T.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Conformational dynamics of the frameshift stimulatory structure in HIV-1</span>. <abbr class="cit-jnl-abbrev">RNA N. Y. N</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1376</span>–<span class="cit-lpage">1384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BN.%2BY.%2BN%26rft.volume%253D23%26rft.spage%253D1376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.69" data-doi="10.1073/pnas.91.9.3959"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rom</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kahana</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">Polyamines regulate the expression of ornithine decarboxylase antizyme in vitro by inducing ribosomal frame-shifting</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">91</span>, <span class="cit-fpage">3959</span>–<span class="cit-lpage">3963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRom%26rft.auinit1%253DE.%26rft.volume%253D91%26rft.issue%253D9%26rft.spage%253D3959%26rft.epage%253D3963%26rft.atitle%253DPolyamines%2Bregulate%2Bthe%2Bexpression%2Bof%2Bornithine%2Bdecarboxylase%2Bantizyme%2Bin%2Bvitro%2Bby%2Binducing%2Bribosomal%2Bframe-shifting.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.9.3959%26rft_id%253Dinfo%253Apmid%252F8171019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MS85LzM5NTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMy80Njk2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.70" data-doi="10.1093/nar/gkg654"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Staple</span>,  <span class="cit-name-given-names">D.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Solution structure of the HIV-1 frameshift inducing stem-loop RNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">31</span>, <span class="cit-fpage">4326</span>–<span class="cit-lpage">4331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStaple%26rft.auinit1%253DD.%2BW.%26rft.volume%253D31%26rft.issue%253D15%26rft.spage%253D4326%26rft.epage%253D4331%26rft.atitle%253DSolution%2Bstructure%2Bof%2Bthe%2BHIV-1%2Bframeshift%2Binducing%2Bstem-loop%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg654%26rft_id%253Dinfo%253Apmid%252F12888491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12888491&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184532900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.71" data-doi="10.1093/nar/gki731"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">M.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C.-T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">C.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">C.-H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">K.-Y.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">An atypical RNA pseudoknot stimulator and an upstream attenuation signal for −1 ribosomal frameshifting of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">33</span>, <span class="cit-fpage">4265</span>–<span class="cit-lpage">4275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSu%26rft.auinit1%253DM.-C.%26rft.volume%253D33%26rft.issue%253D13%26rft.spage%253D4265%26rft.epage%253D4275%26rft.atitle%253DAn%2Batypical%2BRNA%2Bpseudoknot%2Bstimulator%2Band%2Ban%2Bupstream%2Battenuation%2Bsignal%2Bfor%2B-1%2Bribosomal%2Bframeshifting%2Bof%2BSARS%2Bcoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki731%26rft_id%253Dinfo%253Apmid%252F16055920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki731&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16055920&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231516900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.72" data-doi="10.1093/nar/gkn578"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Theis</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reeder</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Giegerich</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">KnotInFrame: prediction of −1 ribosomal frameshift events</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">36</span>, <span class="cit-fpage">6013</span>–<span class="cit-lpage">6020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn578%26rft_id%253Dinfo%253Apmid%252F18820303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18820303&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260152900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.73" data-doi="10.1099/vir.0.19424-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thiel</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putics</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schelle</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissbrich</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabenau</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doerr</span>,  <span class="cit-name-given-names">H.W.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-vol">84</span>, <span class="cit-fpage">2305</span>–<span class="cit-lpage">2315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThiel%26rft.auinit1%253DV.%26rft.volume%253D84%26rft.issue%253D9%26rft.spage%253D2305%26rft.epage%253D2315%26rft.atitle%253DMechanisms%2Band%2Benzymes%2Binvolved%2Bin%2BSARS%2Bcoronavirus%2Bgenome%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19424-0%26rft_id%253Dinfo%253Apmid%252F12917450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19424-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12917450&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184993100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.74" data-doi="10.1093/nar/gkr686"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tholstrup</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oddershede</span>,  <span class="cit-name-given-names">L.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sørensen</span>,  <span class="cit-name-given-names">M.A.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">mRNA pseudoknot structures can act as ribosomal roadblocks</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">303</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr686%26rft_id%253Dinfo%253Apmid%252F21908395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21908395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298733500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tosaka</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yano</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masai</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nozaki</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yomogida</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otani</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nojima</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nishimune</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Identification and characterization of testis specific ornithine decarboxylase antizyme (OAZ-t) gene: expression in haploid germ cells and polyamine-induced frameshifting</span>. <abbr class="cit-jnl-abbrev">Genes Cells Devoted Mol. Cell. Mech</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BCells%2BDevoted%2BMol.%2BCell.%2BMech%26rft.volume%253D5%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.76" data-doi="10.1073/pnas.89.18.8636"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tzeng</span>,  <span class="cit-name-given-names">T.H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bruenn</span>,  <span class="cit-name-given-names">J.A.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">Ribosomal movement impeded at a pseudoknot required for frameshifting</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">8636</span>–<span class="cit-lpage">8640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTu%26rft.auinit1%253DC.%26rft.volume%253D89%26rft.issue%253D18%26rft.spage%253D8636%26rft.epage%253D8640%26rft.atitle%253DRibosomal%2Bmovement%2Bimpeded%2Bat%2Ba%2Bpseudoknot%2Brequired%2Bfor%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.18.8636%26rft_id%253Dinfo%253Apmid%252F1528874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMTgvODYzNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vainberg Slutskin</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weingarten-Gabbay</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nir</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinberger</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Unraveling the determinants of microRNA mediated regulation using a massively parallel reporter assay</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.78" data-doi="10.1093/nar/gkq009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waldman</span>,  <span class="cit-name-given-names">Y.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuller</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shlomi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharan</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ruppin</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Translation efficiency in humans: tissue specificity, global optimization and differences between developmental stages</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">38</span>, <span class="cit-fpage">2964</span>–<span class="cit-lpage">2974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq009%26rft_id%253Dinfo%253Apmid%252F20097653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20097653&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277994600025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xuan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goff</span>,  <span class="cit-name-given-names">S.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Regulation of HIV-1 Gag-Pol Expression by Shiftless, an Inhibitor of Programmed −1 Ribosomal Frameshifting</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">625</span>-<span class="cit-lpage">635</span>.e14.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wills</span>,  <span class="cit-name-given-names">N.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rangan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hoffman</span>,  <span class="cit-name-given-names">D.W.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Comparative studies of frameshifting and nonframeshifting RNA pseudoknots: a mutational and NMR investigation of pseudoknots derived from the bacteriophage T2 gene 32 mRNA and the retroviral gag-pro frameshift site</span>. <abbr class="cit-jnl-abbrev">RNA N. Y. N</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">981</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BN.%2BY.%2BN%26rft.volume%253D8%26rft.spage%253D981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.81" data-doi="10.1126/science.aad4939"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weingarten-Gabbay</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elias-Kirma</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nir</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gritsenko</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stern-Ginossar</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yakhini</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinberger</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Systematic discovery of cap-independent translation sequences in human and viral genomes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">351</span>, <span class="cit-fpage">aad4939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWeingarten-Gabbay%26rft.auinit1%253DS.%26rft.volume%253D351%26rft.issue%253D6270%26rft.spage%253Daad4939%26rft.epage%253Daad4939%26rft.atitle%253DSystematic%2Bdiscovery%2Bof%2Bcap-independent%2Btranslation%2Bsequences%2Bin%2Bhuman%2Band%2Bviral%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aad4939%26rft_id%253Dinfo%253Apmid%252F26816383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTEvNjI3MC9hYWQ0OTM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.82" data-doi="10.1101/gr.236075.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weingarten-Gabbay</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nir</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lubliner</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharon</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalma</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinberger</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Systematic interrogation of human promoters</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">29</span>, <span class="cit-fpage">171</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.236075.118%26rft_id%253Dinfo%253Apmid%252F30622120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI5LzIvMTcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTMvNDY5NjkyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.83" data-doi="10.1016/0022-2836(88)90008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindsley</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falahee</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gallant</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1988</span>). <span class="cit-article-title">On the mechanism of ribosomal frameshifting at hungry codons</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">203</span>, <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWeiss%26rft.auinit1%253DR.%26rft.volume%253D203%26rft.issue%253D2%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DOn%2Bthe%2Bmechanism%2Bof%2Bribosomal%2Bframeshifting%2Bat%2Bhungry%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990008-3%26rft_id%253Dinfo%253Apmid%252F3199440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3199440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.84" data-doi="10.1093/molbev/msr005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Weringh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ragonnet-Cronin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pranckeviciene</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavon-Eternod</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleiman</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">HIV-1 modulates the tRNA pool to improve translation efficiency</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">1827</span>–<span class="cit-lpage">1834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr005%26rft_id%253Dinfo%253Apmid%252F21216840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21216840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290814600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.85" data-doi="10.1074/jbc.M511629200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wills</span>,  <span class="cit-name-given-names">N.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hammer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gesteland</span>,  <span class="cit-name-given-names">R.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">A functional −1 ribosomal frameshift signal in the human paraneoplastic Ma3 gene</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">281</span>, <span class="cit-fpage">7082</span>–<span class="cit-lpage">7088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M511629200%26rft_id%253Dinfo%253Apmid%252F16407312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODEvMTEvNzA4MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEzLzQ2OTY5Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.86" data-doi="10.1093/nar/gky636"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooperman</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldman</span>,  <span class="cit-name-given-names">Y.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Translocation kinetics and structural dynamics of ribosomes are modulated by the conformational plasticity of downstream pseudoknots</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">46</span>, <span class="cit-fpage">9736</span>–<span class="cit-lpage">9748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253D9736%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.87" data-doi="10.1016/j.cell.2015.02.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">J.-D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Ribosome excursions during mRNA translocation mediate broad branching of frameshift pathways</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>, <span class="cit-fpage">870</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D870%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.02.003%26rft_id%253Dinfo%253Apmid%252F25703095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25703095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F13%2F469692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-469692v2.88" data-doi="10.1074/jbc.M115.647065"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yordanova</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andreev</span>,  <span class="cit-name-given-names">D.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sachs</span>,  <span class="cit-name-given-names">M.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Atkins</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">A Nascent Peptide Signal Responsive to Endogenous Levels of Polyamines Acts to Stimulate Regulatory Frameshifting on Antizyme mRNA</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">290</span>, <span class="cit-fpage">17863</span>–<span class="cit-lpage">17878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.647065%26rft_id%253Dinfo%253Apmid%252F25998126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMjkvMTc4NjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMy80Njk2OTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/469692v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F469692v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.10.985747v2" title="Cell wall remodeling and vesicle trafficking mediate the root clock in Arabidopsis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 13, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/469692v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/469692v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/469692v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/469692v2" class="active" data-icon-position="" data-hide-link-title="0">High-throughput interrogation of programmed ribosomal frameshifting in human cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1194046" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AcygDNPpbQmyFzX_-x_z1Y3-O_0CBk2Q8iFgrk7Aq2k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1194046" id="node1194046--2" data-pisa="biorxiv;469692v2" data-pisa-master="biorxiv;469692" data-seqnum="1194046" data-apath="/biorxiv/early/2020/03/13/469692.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">High-throughput interrogation of programmed ribosomal frameshifting in human cells</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Mikl</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yitzhak</span> <span class="nlm-surname">Pilpel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Eran</span> <span class="nlm-surname">Segal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">469692; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/469692 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/469692v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;title=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;title=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;text=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;title=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;t=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;title=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;title=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1194046" id="node1194046--3" data-pisa="biorxiv;469692v2" data-pisa-master="biorxiv;469692" data-seqnum="1194046" data-apath="/biorxiv/early/2020/03/13/469692.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">High-throughput interrogation of programmed ribosomal frameshifting in human cells</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Mikl</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yitzhak</span> <span class="nlm-surname">Pilpel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Eran</span> <span class="nlm-surname">Segal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">469692; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/469692 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Mikl%2C+Martin&amp;rft.au=Pilpel%2C+Yitzhak&amp;rft.au=Segal%2C+Eran"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1194046/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1194046/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1194046/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1194046/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1194046/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1194046/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1194046/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1194046/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1194046/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1194046/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1194046/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1194046/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;count=horizontal&amp;via=&amp;text=High-throughput%20interrogation%20of%20programmed%20ribosomal%20frameshifting%20in%20human%20cells&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/469692v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/469692v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/469692v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
