<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/14/402370.atom"/>
  <meta name="HW.pisa" content="biorxiv;402370v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="PathoLive - Real-time pathogen identification from metagenomic Illumina datasets"/>
  <meta name="DC.Identifier" content="10.1101/402370"/>
  <meta name="DC.Date" content="2020-04-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive). ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Simon H. Tausch"/>
  <meta name="DC.Contributor" content="Tobias P. Loka"/>
  <meta name="DC.Contributor" content="Jakob M. Schulze"/>
  <meta name="DC.Contributor" content="Andreas Andrusch"/>
  <meta name="DC.Contributor" content="Jeanette Klenner"/>
  <meta name="DC.Contributor" content="Piotr Wojciech Dabrowski"/>
  <meta name="DC.Contributor" content="Martin S. Lindner"/>
  <meta name="DC.Contributor" content="Andreas Nitsche"/>
  <meta name="DC.Contributor" content="Bernhard Y. Renard"/>
  <meta name="article:published_time" content="2020-04-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="PathoLive - Real-time pathogen identification from metagenomic Illumina datasets"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive).&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;402370v2"/>
  <meta name="citation_id" content="402370v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/402370v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/402370v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/402370v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/14/402370.full.pdf"/>
  <meta name="citation_doi" content="10.1101/402370"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="402370"/>
  <meta name="citation_author" content="Simon H. Tausch"/>
  <meta name="citation_author_institution" content="Federal Institute for Risk Assessment;"/>
  <meta name="citation_author_email" content="misterioes16@gmail.com"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6874-233X"/>
  <meta name="citation_author" content="Tobias P. Loka"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="lokat@rki.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6380-5875"/>
  <meta name="citation_author" content="Jakob M. Schulze"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="jakobschulze@arcor.de"/>
  <meta name="citation_author" content="Andreas Andrusch"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="andruscha@rki.de"/>
  <meta name="citation_author" content="Jeanette Klenner"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="jeanette.klenner@bundeswehr.de"/>
  <meta name="citation_author" content="Piotr Wojciech Dabrowski"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="dabrowskiw@rki.de"/>
  <meta name="citation_author" content="Martin S. Lindner"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="martin@mail-lindner.de"/>
  <meta name="citation_author" content="Andreas Nitsche"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="nitschea@rki.de"/>
  <meta name="citation_author" content="Bernhard Y. Renard"/>
  <meta name="citation_author_institution" content="Robert Koch Institute"/>
  <meta name="citation_author_email" content="renardb@rki.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4589-9809"/>
  <meta name="twitter:title" content="PathoLive - Real-time pathogen identification from metagenomic Illumina datasets"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive). ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="PathoLive - Real-time pathogen identification from metagenomic Illumina datasets"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/402370v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive). ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-14"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>PathoLive - Real-time pathogen identification from metagenomic Illumina datasets | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1244195 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-category-new-results"> 
  <div class="article abstract-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-2">Motivation: Over the past years, NGS has become a crucial workhorse for open-view pathogen diagnostics. Yet, long turnaround times result from using massively parallel high-throughput technologies as the analysis can only be performed after sequencing has finished. The interpretation of results can further be challenged by contaminations, clinically irrelevant sequences, and the sheer amount and complexity of the data. Results: We implemented PathoLive, a real-time diagnostics pipeline for the detection of pathogens from clinical samples hours before sequencing has finished. Based on real-time alignment with HiLive2, mappings are scored with respect to common contaminations, low-entropy areas, and sequences of widespread, non-pathogenic organisms. The results are visualized using an interactive taxonomic tree that provides an easily interpretable overview of the relevance of hits. For a human plasma sample that was spiked in vitro with six pathogenic viruses, all agents were clearly detected after only 40 of 200 sequencing cycles. For a real-world sample from Sudan the results correctly indicated the presence of Crimean-Congo hemorrhagic Fever Virus. In a second real-world dataset from the 2019 SARS-CoV-2 outbreak in Wuhan, we found the presence of a SARS Coronavirus as the most relevant hit without the novel virus reference genome being included in the database. For all samples, clinically irrelevant hits were correctly deemphasized. Our approach is valuable to obtain fast and accurate NGS-based pathogen identifications and correctly prioritize and visualize them based on their clinical significance. Availability: PathoLive is open source and available on GitLab (https://gitlab.com/rki_bioinformatics/PathoLive) and BioConda (conda install -c bioconda patholive).</p>
   </div>
   <h3>Competing Interest Statement</h3>
   <p id="p-3">The authors have declared no competing interest.</p>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
