<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/02/2020.02.04.934232.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.04.934232v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach" />
<meta name="DC.Identifier" content="10.1101/2020.02.04.934232" />
<meta name="DC.Date" content="2020-03-02" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.

Method Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.

Results Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.

Conclusion T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.

*   WHO
    :   World Health Organization
    CoV
    :   Coronaviruses
    MERS-CoV
    :   Middle East Respiratory Syndrome
    SARS-CoV
    :   Acute Respiratory Syndrome
    COVID-19
    :   novel coronavirus
    HCoV-HKU1
    :   Human coronavirus HKU1
    HCoV-OC43
    :   Human coronavirus OC43
    HCoV-NL63
    :   Human coronavirus NL63
    HCoV-229E
    :   Human coronavirus 229E
    ACE2
    :   angiotensin converting enzyme 2
    RBM
    :   receptor-binding motif
    RBM
    :   receptor-binding domain
    vs
    :   versus
    ACT
    :   Artemis Comparison Tool
    ACC
    :   auto cross covariance
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    ANN
    :   Artificial Neural Network
    IEDB
    :   Immune Epitope Database
    IC50
    :   median inhibitory concentrations
    MHC I
    :   Major Histocompatibility complex class I
    MHC II
    :   Major Histocompatibility complex class II
    PDB
    :   Protein database
    MSA
    :   multiple sequence alignment" />
<meta name="DC.Contributor" content="Miyssa I. Abdelmageed" />
<meta name="DC.Contributor" content="Abdelrahman H. Abdelmoneim" />
<meta name="DC.Contributor" content="Mujahed I. Mustafa" />
<meta name="DC.Contributor" content="Nafisa M. Elfadol" />
<meta name="DC.Contributor" content="Naseem S. Murshed" />
<meta name="DC.Contributor" content="Shaza W. Shantier" />
<meta name="DC.Contributor" content="Abdelrafie M. Makhawi" />
<meta name="article:published_time" content="2020-03-02" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Background: New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a public-health emergency of international concern due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach. Method: Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target. Results: Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively. Conclusion: T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.04.934232v2" />
<meta name="citation_id" content="2020.02.04.934232v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.04.934232" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.04.934232" />
<meta name="citation_author" content="Miyssa I. Abdelmageed" />
<meta name="citation_author_institution" content="Faculty of Pharmacy, University of Khartoum" />
<meta name="citation_author" content="Abdelrahman H. Abdelmoneim" />
<meta name="citation_author_institution" content="Faculty of Medicine, Alneelain University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5037-8488" />
<meta name="citation_author" content="Mujahed I. Mustafa" />
<meta name="citation_author_institution" content="Department of Biotechnology, University of Bahri" />
<meta name="citation_author_email" content="mujahedibrahim44@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6893-0536" />
<meta name="citation_author" content="Nafisa M. Elfadol" />
<meta name="citation_author_institution" content="Department of Molecular biology, National University Biomedical Research Institute, National University" />
<meta name="citation_author" content="Naseem S. Murshed" />
<meta name="citation_author_institution" content="Department of Microbiology, International University of Africa" />
<meta name="citation_author" content="Shaza W. Shantier" />
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, Faculty of Pharmacy, University of Khartoum" />
<meta name="citation_author" content="Abdelrafie M. Makhawi" />
<meta name="citation_author_institution" content="Department of Biotechnology, University of Bahri" />
<meta name="citation_reference" content="H. Lu, C. W. Stratton, and Y. W. Tang, “Outbreak of Pneumonia of Unknown Etiology in Wuhan China: the Mystery and the Miracle,” J Med Virol, Jan 16 2020." />
<meta name="citation_reference" content="D. S. Hui, I. A. E, T. A. Madani, F. Ntoumi, R. Kock, O. Dar, et al.,“The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health - The latest 2019 novel coronavirus outbreak in Wuhan, China,” Int J Infect Dis, vol. 91, pp. 264–266, Jan 14 2020." />
<meta name="citation_reference" content="D. Benvenuto, M. Giovannetti, A. Ciccozzi, S. Spoto, S. Angeletti, and M. Ciccozzi, “The 2019-new coronavirus epidemic: evidence for virus evolution,” J Med Virol, Jan 29 2020." />
<meta name="citation_reference" content="“CoV2020”. GISAID EpifluDB. Archivedfrom the original on 12 January 2020. Retrieved February 2020" />
<meta name="citation_reference" content="P. Venkatagopalan, S. M. Daskalova, L. A. Lopez, K. A. Dolezal, and B. G. Hogue, “Coronavirus envelope (E) protein remains at the site of assembly,” Virology, vol. 478, pp. 75–85, Apr 2015." />
<meta name="citation_reference" content="J. L. Nieto-Torres, M. L. Dediego, E. Alvarez, J. M. Jimenez-Guardeno, J. A. Regla-Nava, M. Llorente, et al.,“Subcellular location and topology of severe acute respiratory syndrome coronavirus envelope protein,” Virology, vol. 415, pp. 69–82, Jul 5 2011." />
<meta name="citation_reference" content="K. M. Curtis, B. Yount, and R. S. Baric, “Heterologous gene expression from transmissible gastroenteritis virus replicon particles,” J Virol, vol. 76, pp. 1422–34, Feb 2002." />
<meta name="citation_reference" content="J. Ortego, D. Escors, H. Laude, and L. Enjuanes, “Generation of a replication-competent, propagation-deficient virus vector based on the transmissible gastroenteritis coronavirus genome,” J Virol, vol. 76, pp. 11518–29, Nov 2002." />
<meta name="citation_reference" content="T. R. Ruch and C. E. Machamer, “A single polar residue and distinct membrane topologies impact the function of the infectious bronchitis coronavirus E protein,” PLoS Pathog, vol. 8, p. e1002674, 2012." />
<meta name="citation_reference" content="J. F. Chan, S. Yuan, K. H. Kok, K. K. To, H. Chu, J. Yang, et al.,“A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster,” Lancet, Jan 24 2020." />
<meta name="citation_reference" content="N. Zhang, L. Wang, X. Deng, R. Liang, M. Su, C. He, et al., “Recent advances in the detection of respiratory virus infection in humans,” J Med Virol, Jan 15 2020." />
<meta name="citation_reference" content="J. B. Mahony, A. Petrich, and M. Smieja, “Molecular diagnosis of respiratory virus infections,” Crit Rev Clin Lab Sci, vol. 48, pp. 217–49, Sep-Dec 2011." />
<meta name="citation_reference" content="V. M. Corman, O. Landt, M. Kaiser, R. Molenkamp, A. Meijer, D. K. Chu, et al.,“Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR,” Euro Surveill, vol. 25, Jan 2020." />
<meta name="citation_reference" content="Y. Chen, Q. Liu, and D. Guo, “Emerging coronaviruses: genome structure, replication, and pathogenesis,” J Med Virol, Jan 22 2020" />
<meta name="citation_reference" content="Y. Wan, J. Shang, R. Graham, R. S. Baric, and F. Li, “Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS, J Virol, Jan 29 2020." />
<meta name="citation_reference" content="“Genomic epidemiology of novel coronavirus (nCoV)”. nextstrain.org. Retrieved 11 February 2020." />
<meta name="citation_reference" content="Letko M, Munster V (January 2020). “Functional assessment of cell entry and receptor usage for lineage Bß-coronaviruses, including 2019-nCoV”. bioRxiv 2020.01.22.915660 Check |biorxiv= value (help)." />
<meta name="citation_reference" content="Letko, M, Marzi, et al. (February 2020). “Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses”. Nature Microbiology. doi:10.1038/s41564-020-0688-y. PMID 32094589." />
<meta name="citation_reference" content="El Sahly HM. “Genomic Characterization of the 2019 Novel Coronavirus”. New England Journal of Medicine. Retrieved 9 February 2020." />
<meta name="citation_reference" content="Gralinski LE, Menachery VD (January 2020). “Return of the Coronavirus: 2019-nCoV”. Viruses. 12(2): 135. doi:10.3390/v12020135. PMID 31991541." />
<meta name="citation_reference" content="Lu R, Zhao X, Li J, et al. (January 2020). “Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding”. Lancet. 395(10224):565–574. doi:10.1016/S0140-6736(20)302518. PMID 32007145." />
<meta name="citation_reference" content="H. Lu, “Drug treatment options for the 2019-new coronavirus (2019-nCoV),” Biosci Trends, Jan 28 2020." />
<meta name="citation_reference" content="Y. T. Lo, T. W. Pai, W. K. Wu, and H. T. Chang, “Prediction of conformational epitopes with the use of a knowledge-based energy function and geometrically related neighboring residue characteristics,” BMCBioinformatics, vol. 14 Suppl 4, p. S3, 2013." />
<meta name="citation_reference" content="W. Li, M. D. Joshi, S. Singhania, K. H. Ramsey, and A. K. Murthy, “Peptide Vaccine: Progress and Challenges,” Vaccines (Basel), vol. 2, pp. 515–36, Jul 2 2014." />
<meta name="citation_reference" content="A. W. Purcell, J. McCluskey, and J. Rossjohn, “More than one reason to rethink the use of peptides in vaccine design,” Nat Rev Drug Discov, vol. 6, pp. 404–14, May 2007." />
<meta name="citation_reference" content="N. L. Dudek, P. Perlmutter, M. I. Aguilar, N. P. Croft, and A. W. Purcell, “Epitope discovery and their use in peptide based vaccines,” Curr Pharm Des, vol. 16, pp. 3149–57, 2010." />
<meta name="citation_reference" content="S. Dermime, D. E. Gilham, D. M. Shaw, E. J. Davidson, K. Meziane el, A. Armstrong, et al.,“Vaccine and antibody-directed T cell tumour immunotherapy,” Biochim Biophys Acta, vol. 1704, pp. 11–35, Jul 6 2004." />
<meta name="citation_reference" content="R. H. Meloen, J. P. Langeveld, W. M. Schaaper, and J. W. Slootstra, “Synthetic peptide vaccines: unexpected fulfillment of discarded hope?,” Biologicals, vol. 29, pp. 233–6, Sep-Dec 2001." />
<meta name="citation_reference" content="V. Brusic and N. Petrovsky, “Immunoinformatics and its relevance to understanding human immune disease,” Expert Rev Clin Immunol, vol. 1, pp. 145–57, May 2005." />
<meta name="citation_reference" content="N. Tomar and R. K. De, “Immunoinformatics: a brief review,” Methods Mol Biol, vol. 1184, pp. 23–55, 2014." />
<meta name="citation_reference" content="S. Khalili, A. Jahangiri, H. Borna, K. Ahmadi Zanoos, and J. Amani, “Computational vaccinology and epitope vaccine design by immunoinformatics,” Acta Microbiol Immunol Hung, vol. 61, pp. 285–307, Sep 2014." />
<meta name="citation_reference" content="N. Tomar and R. K. De, “Immunoinformatics: an integrated scenario,” Immunology, vol. 131, pp. 153–68, Oct 2010." />
<meta name="citation_reference" content="N. R. Hegde, S. Gauthami, H. M. Sampath Kumar, and J. Bayry, “The use of databases, data mining and immunoinformatics in vaccinology: where are we?,” Expert Opin Drug Discov, vol. 13, pp. 117–130, Feb 2018." />
<meta name="citation_reference" content="A. A. Bahrami, Z. Payandeh, S. Khalili, A. Zakeri, and M. Bandehpour, “Immunoinformatics: In Silico Approaches and Computational Design of a Multiepitope, Immunogenic Protein,” Int Rev Immunol, vol. 38, pp. 307–322, 2019." />
<meta name="citation_reference" content="T. J. Carver, K. M. Rutherford, M. Berriman, M. A. Rajandream, B. G. Barrell, and J. Parkhill, “ACT: the Artemis Comparison Tool,” Bioinformatics, vol. 21, pp. 3422–3, Aug 15 2005." />
<meta name="citation_reference" content="I. A. Doytchinova and D. R. Flower, “VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines,” BMC Bioinformatics, vol. 8, p. 4, Jan 5 2007." />
<meta name="citation_reference" content="O. O. Oni, A. A. Owoade, and C. A. O. Adeyefa, “Design and evaluation of primer pairs for efficient detection of avian rotavirus,” Trop Anim Health Prod, vol. 50, pp. 267–273, Feb 2018." />
<meta name="citation_reference" content="G. Stecher, K. Tamura, and S. Kumar, “Molecular Evolutionary Genetics Analysis (MEGA) for macOS,” Mol Biol Evol, Jan 6 2020." />
<meta name="citation_reference" content="K. Tamura, J. Dudley, M. Nei, and S. Kumar, “MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0,” Mol Biol Evol, vol. 24, pp. 1596–9, Aug 2007." />
<meta name="citation_reference" content="M. Nielsen, C. Lundegaard, P. Worning, S. L. Lauemoller, K. Lamberth, S. Buus, et al.,“Reliable prediction of T-cell epitopes using neural networks with novel sequence representations,” Protein Sci, vol. 12, pp. 1007–17, May 2003." />
<meta name="citation_reference" content="C. Lundegaard, K. Lamberth, M. Harndahl, S. Buus, O. Lund, and M. Nielsen, “NetMHC-3.0: accurate web accessible predictions of human, mouse and monkey MHC class I affinities for peptides of length 8-11,” Nucleic Acids Res, vol. 36, pp. W509–12, Jul 1 2008." />
<meta name="citation_reference" content="M. Andreatta and M. Nielsen, “Gapped sequence alignment using artificial neural networks: application to the MHC class I system,” Bioinformatics, vol. 32, pp. 511–7, Feb 15 2016." />
<meta name="citation_reference" content="S. Buus, S. L. Lauemoller, P. Worning, C. Kesmir, T. Frimurer, S. Corbet, et al.,“Sensitive quantitative predictions of peptide-MHC binding by a ‘Query by Committee’ artificial neural network approach,” Tissue Antigens, vol. 62, pp. 378–84, Nov 2003." />
<meta name="citation_reference" content="C. Lundegaard, O. Lund, and M. Nielsen, “Accurate approximation method for prediction of class I MHC affinities for peptides of length 8, 10 and 11 using prediction tools trained on 9mers,” Bioinformatics, vol. 24, pp. 1397–8, Jun 1 2008." />
<meta name="citation_reference" content="C. Lundegaard, M. Nielsen, and O. Lund, “The validity of predicted T-cell epitopes,” Trends Biotechnol, vol. 24, pp. 537–8, Dec 2006." />
<meta name="citation_reference" content="S. Southwood, J. Sidney, A. Kondo, M. F. del Guercio, E. Appella, S. Hoffman, et al.,“Several common HLA-DR types share largely overlapping peptide binding repertoires,” J Immunol, vol. 160, pp. 3363–73, Apr 1 1998." />
<meta name="citation_reference" content="A. Patronov and I. Doytchinova, “T-cell epitope vaccine design by immunoinformatics,” Open Biol, vol. 3, p. 120139, Jan 8 2013." />
<meta name="citation_reference" content="H. H. Bui, J. Sidney, K. Dinh, S. Southwood, M. J. Newman, and A. Sette, “Predicting population coverage of T-cell epitope-based diagnostics and vaccines,” BMC Bioinformatics, vol. 7, p. 153, Mar 17 2006." />
<meta name="citation_reference" content="D. A. Benson, M. Cavanaugh, K. Clark, I. Karsch-Mizrachi, D. J. Lipman, J. Ostell, et al.,“GenBank,” Nucleic Acids Res, vol. 45, pp. D37–d42, Jan 4 2017." />
<meta name="citation_reference" content="S. Wang, W. Li, S. Liu, and J. Xu, “RaptorX-Property: a web server for protein structure property prediction,” Nucleic Acids Res, vol. 44, pp. W430–5, Jul 8 2016." />
<meta name="citation_reference" content="E. F. Pettersen, T. D. Goddard, C. C. Huang, G. S. Couch, D. M. Greenblatt, E. C. Meng, et al.,“UCSF Chimera--a visualization system for exploratory research and analysis,” J Comput Chem, vol. 25, pp. 1605–12, Oct 2004." />
<meta name="citation_reference" content="Choo JA, Thong SY, Yap J, van Esch WJ, Raida M, Meijers R, et al. Bioorthogonal cleavage and exchange of major histocompatibility complex ligands by employing azobenzene-containing peptides. Angewandte Chemie (International ed in English). 2014;53(49):13390–4." />
<meta name="citation_reference" content="Guex N, Peitsch MC. SWISS-MODEL and the Swiss-Pdb Viewer: an environment for comparative protein modeling. Electrophoresis. 1997;18(15):2714–23." />
<meta name="citation_reference" content="Morris GM, Huey R, Lindstrom W, Sanner MF, Belew RK, Goodsell DS, et al. AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility. Journal of computational chemistry. 2009;30(16):2785–91." />
<meta name="citation_reference" content="Morris G, S. Goodsell D, S. Halliday R, Huey R, Hart W, Belew R, et al. Automated Docking Using a Lamarckian Genetic Algorithm and Empirical Binding Free Energy Function 1998. 1639–62 p" />
<meta name="citation_reference" content="S. J. Marshall, “Developing countries face double burden of disease,” Bull World Health Organ, vol. 82, p. 556, Jul 2004." />
<meta name="citation_reference" content="A. S. De Groot and R. Rappuoli, “Genome-derived vaccines,” Expert Rev Vaccines, vol. 3, pp. 59–76, Feb 2004." />
<meta name="citation_reference" content="B. Korber, M. LaBute, and K. Yusim, “Immunoinformatics comes of age,” PLoS Comput Biol, vol. 2, p. e71, Jun 30 2006." />
<meta name="citation_reference" content="A. S. Fauci, “Emerging and re-emerging infectious diseases: influenza as a prototype of the host-pathogen balancing act,” Cell, vol. 124, pp. 665–70, Feb 24 2006." />
<meta name="citation_reference" content="S. Chaudhury, E. H. Duncan, T. Atre, S. Dutta, M. D. Spring, W. W. Leitner, et al.,“Combining immunoprofiling with machine learning to assess the effects of adjuvant formulation on human vaccine-induced immunity,” Hum Vaccin Immunother, pp. 1–12, Oct 7 2019." />
<meta name="citation_reference" content="Purcell AW, McCluskey J, Rossjohn J. More than one reason to rethink the use of peptides in vaccine design. Nat Rev Drug Discov 2007; 6 (5): 404–14. PMID: 17473845." />
<meta name="citation_reference" content="S. S. Usmani, R. Kumar, S. Bhalla, V. Kumar, and G. P. S. Raghava, “In Silico Tools and Databases for Designing Peptide-Based Vaccine and Drugs,” Adv Protein Chem Struct Biol, vol. 112, pp. 221–263, 2018." />
<meta name="citation_reference" content="S. K. Dhanda, S. S. Usmani, P. Agrawal, G. Nagpal, A. Gautam, and G. P. S. Raghava, “Novel in silico tools for designing peptide-based subunit vaccines and immunotherapeutics,” Brief Bioinform, vol. 18, pp. 467–478, May 1 2017." />
<meta name="citation_reference" content="Y. Khazaei-Poul, S. Farhadi, S. Ghani, S. A. Ahmadizad, and J. Ranjbari, “Monocyclic Peptides: Types, Synthesis and applications,” Curr Pharm Biotechnol, Jan 20 2020." />
<meta name="citation_reference" content="M. A. Hasan, M. Hossain, and M. J. Alam, “A computational assay to design an epitopebased Peptide vaccine against saint louis encephalitis virus,” Bioinform Biol Insights, vol. 7, pp. 347–55, 2013." />
<meta name="citation_reference" content="S. Chakraborty, R. Chakravorty, M. Ahmed, A. Rahman, T. M. Waise, F. Hassan, et al.,“A computational approach for identification of epitopes in dengue virus envelope protein: a step towards designing a universal dengue vaccine targeting endemic regions,” In Silico Biol, vol. 10, pp. 235–46, 2010." />
<meta name="citation_reference" content="M. Tahir Ul Qamar, A. Bari, M. M. Adeel, A. Maryam, U. A. Ashfaq, X. Du, et al.,“Peptide vaccine against chikungunya virus: immuno-informatics combined with molecular docking approach,” J TranslMed, vol. 16, p. 298, Oct 27 2018." />
<meta name="citation_reference" content="S. S. Twiddy, E. C. Holmes, and A. Rambaut, “Inferring the rate and time-scale of dengue virus evolution,” Mol Biol Evol, vol. 20, pp. 122–9, Jan 2003." />
<meta name="citation_reference" content="J. M. Christie, H. Chapel, R. W. Chapman, and W. M. Rosenberg, “Immune selection and genetic sequence variation in core and envelope regions of hepatitis C virus,” Hepatology, vol. 30, pp. 1037–44, Oct 1999." />
<meta name="citation_reference" content="R. C. Hardison, “Comparative genomics,” PLoSBiol, vol. 1, p. E58, Nov 2003." />
<meta name="citation_reference" content="C. Gilbert, A. Chateigner, L. Ernenwein, V. Barbe, A. Bezier, E. A. Herniou, et al.,“Population genomics supports baculoviruses as vectors of horizontal transfer of insect transposons,” Nat Commun, vol. 5, p. 3348, 2014." />
<meta name="citation_reference" content="D. Schoeman and B. C. Fielding, “Coronavirus envelope protein: current knowledge,” Virol J, vol. 16, p. 69, May 27 2019." />
<meta name="citation_reference" content="E. Mortola and P. Roy, “Efficient assembly and release of SARS coronavirus-like particles by a heterologous expression system,” FEBS Lett, vol. 576, pp. 174–8, Oct 8 2004." />
<meta name="citation_reference" content="R. Sharmin and A. B. Islam, “Conserved antigenic sites between MERS-CoV and Batcoronavirus are revealed through sequence analysis,” Source Code Biol Med, vol. 11, p. 3, 2016." />
<meta name="citation_reference" content="J. Xu, J. Hu, J. Wang, Y. Han, Y. Hu, J. Wen, et al.,“Genome organization of the SARS-CoV,” Genomics Proteomics Bioinformatics, vol. 1, pp. 226–35, Aug 2003." />
<meta name="citation_reference" content="Black M, Trent A, Tirrell M, Olive C. Advances in the design and delivery of peptide subunit vaccines with a focus on toll-like receptor agonists. Expert Rev Vaccines 2010; 9: 157–173." />
<meta name="citation_reference" content="Sesardic D. Synthetic peptide vaccines. J Med Microbiol 1993; 39: 241–242." />
<meta name="citation_reference" content="Breakthrough: Chloroquine phosphate has shown apparentefficacy in treatment of COVID-19 associated pneumonia inclinical studies." />
<meta name="citation_reference" content="Potential Inhibitors Against Papain-like Protease of Novel Coronavirus (COVID-19) from FDA Approved Drugs." />
<meta name="citation_reference" content="J. W. Westerbeck and C. E. Machamer, “A Coronavirus E Protein Is Present in Two Distinct Pools with Different Effects on Assembly and the Secretory Pathway,” J Virol, vol. 89, pp. 9313–23, Sep 2015." />
<meta name="citation_reference" content="M. L. Dediego, L. Pewe, E. Alvarez, M. T. Rejas, S. Perlman, and L. Enjuanes, “Pathogenicity of severe acute respiratory coronavirus deletion mutants in hACE-2 transgenic mice,” Virology, vol. 376, pp. 379–89, Jul 5 2008." />
<meta name="citation_reference" content="J. Netland, M. L. DeDiego, J. Zhao, C. Fett, E. Alvarez, J. L. Nieto-Torres, et al.,“Immunization with an attenuated severe acute respiratory syndrome coronavirus deleted in E protein protects against lethal respiratory disease,” Virology, vol. 399, pp. 120–128, Mar 30 2010." />
<meta name="citation_reference" content="P. B. McCray, Jr., L. Pewe, C. Wohlford-Lenane, M. Hickey, L. Manzel, L. Shi, et al.,“Lethal infection of K18-hACE2 mice infected with severe acute respiratory syndrome coronavirus,” J Virol, vol. 81, pp. 813–21, Jan 2007." />
<meta name="citation_reference" content="J. A. Regla-Nava, J. L. Nieto-Torres, J. M. Jimenez-Guardeno, R. Fernandez-Delgado, C. Fett, C. Castano-Rodriguez, et al.,“Severe acute respiratory syndrome coronaviruses with mutations in the E protein are attenuated and promising vaccine candidates,” J Virol,vol. 89, pp. 3870–87, Apr 2015." />
<meta name="citation_reference" content="J. Zhao, K. Li, C. Wohlford-Lenane, S. S. Agnihothram, C. Fett, J. Zhao, et al.,“Rapid generation of a mouse model for Middle East respiratory syndrome,” Proc Natl Acad Sci USA, vol. 111, pp. 4970–5, Apr 1 2014." />
<meta name="citation_reference" content="X. Guo, Y. Deng, H. Chen, J. Lan, W. Wang, X. Zou, et al.,“Systemic and mucosal immunity in mice elicited by a single immunization with human adenovirus type 5 or 41 vector-based vaccines carrying the spike protein of Middle East respiratory syndrome coronavirus,” Immunology, vol. 145, pp. 476–84, Aug 2015." />
<meta name="twitter:title" content="Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.

Method Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.

Results Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.

Conclusion T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.

*   WHO
    :   World Health Organization
    CoV
    :   Coronaviruses
    MERS-CoV
    :   Middle East Respiratory Syndrome
    SARS-CoV
    :   Acute Respiratory Syndrome
    COVID-19
    :   novel coronavirus
    HCoV-HKU1
    :   Human coronavirus HKU1
    HCoV-OC43
    :   Human coronavirus OC43
    HCoV-NL63
    :   Human coronavirus NL63
    HCoV-229E
    :   Human coronavirus 229E
    ACE2
    :   angiotensin converting enzyme 2
    RBM
    :   receptor-binding motif
    RBM
    :   receptor-binding domain
    vs
    :   versus
    ACT
    :   Artemis Comparison Tool
    ACC
    :   auto cross covariance
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    ANN
    :   Artificial Neural Network
    IEDB
    :   Immune Epitope Database
    IC50
    :   median inhibitory concentrations
    MHC I
    :   Major Histocompatibility complex class I
    MHC II
    :   Major Histocompatibility complex class II
    PDB
    :   Protein database
    MSA
    :   multiple sequence alignment" />
<meta name="og-title" property="og:title" content="Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.

Method Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.

Results Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.

Conclusion T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.

*   WHO
    :   World Health Organization
    CoV
    :   Coronaviruses
    MERS-CoV
    :   Middle East Respiratory Syndrome
    SARS-CoV
    :   Acute Respiratory Syndrome
    COVID-19
    :   novel coronavirus
    HCoV-HKU1
    :   Human coronavirus HKU1
    HCoV-OC43
    :   Human coronavirus OC43
    HCoV-NL63
    :   Human coronavirus NL63
    HCoV-229E
    :   Human coronavirus 229E
    ACE2
    :   angiotensin converting enzyme 2
    RBM
    :   receptor-binding motif
    RBM
    :   receptor-binding domain
    vs
    :   versus
    ACT
    :   Artemis Comparison Tool
    ACC
    :   auto cross covariance
    MEGA
    :   Molecular Evolutionary Genetics Analysis
    ANN
    :   Artificial Neural Network
    IEDB
    :   Immune Epitope Database
    IC50
    :   median inhibitory concentrations
    MHC I
    :   Major Histocompatibility complex class I
    MHC II
    :   Major Histocompatibility complex class II
    PDB
    :   Protein database
    MSA
    :   multiple sequence alignment" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-02" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.04.934232v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.04.934232v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.04.934232v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1177442" />
    <title>Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DJZtc5tNKjQq_xs2cAT2mvYQk8X-HwII-RwtIDc3nbo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1177442","apath":"\/biorxiv\/early\/2020\/03\/02\/2020.02.04.934232.atom","pisa":"biorxiv;2020.02.04.934232v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.04.934232v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.04.934232v2","title":"Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach","identifier":"node\/1177442"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OEdkJC1shhT0KWV9YiOziXtSeRJqk23\\\\\\\/0F5n+aB4MzCXBq+G4VmJmr7qW9\\\\\\\/YUYc3ObmrKJaWQ8e5j\\\\\\\/iLeLi21B9GAm9gdSCr2vIsVKU9WrhP6Lx4S\\\\\\\/Y\\\\\\\/Rc4GgVwYC48TVO3vZttWp5bS2fPi9vKLtxsEdp5MgIgdHhlqWHpMDtS6dwcCd+p0e0K0p0jNqjIZ0eWqKVLOrpwUW2qJbgRaqYopeCmGavT+H9b4KWmB4fFc\\\\\\\/HI5qUH+9+xNckKKTd3cRD6aUV8x0MKtyKPInhn+drNKxJSzihl2rFMWbsEgTH\\\\\\\/t65VN+97CX7gckyWSnthutnTKE+gqKH3MvEaPcmIFev0djMStNb2kkUUTtpNxtbS3cMSvX2xyIO3P3gZHqNfUf1I+BNv7L7V9LDyyTVcoKVH+vEmq9yXzKegvJJCwDL0Jgrv4KWRme\\\\\\\/Twp1eS6F5nOeSBYfM92KlxY7NhWDeJIpxbWU\\\\\\\/ykGpaqlbYjoAbSvdXHhiYVa+fiQEUREiUaSmZRZpLvAJkQKmtf5qbIoWufFt9revS3T3wV12TRGbR4Ui0gpnE5r5qS0m\\\\\\\/6yqIKx+OrLRAv2QspYp+e6W0gp5m1PiPhy8X1btC\\\\\\\/f8I8ZTJuvUJWextw90nHJuSBSMjjNOgPAatkSjjEUGZ7EZCAurfy2BSxHUAXKNDxQd55ayRkgDd3aZp4WtoZq+Nv74Ulaa5wr28QBPh0Vbmt1xPbi160OLnZ\\\\\\\/6CG56M6DVBgGp5goA\\\\\\\/ihj8EFxDED5tDpm5nYRhC2P3Ug1Rkr8nsZzZ2ONGYUTpDg7OwbMriNr6bGlnp\\\\\\\/8cVJ8fac\\\\\\\/LOM+dlFT\\\\\\\/3P6h5dGZ\\\\\\\/QM7UUGvf8nnMjcpy55H0SlFc2gxirSeSVmZ53MZrad9UQjLkPIxAsRJRzBDlFS6Tmd7OHeWk+9Nk2RQ+kz6UG18JfM2AnNmeoke2EkAKLpDG\\\\\\\/ksrb2wMcqGUHLDHS7EKi8BjxlSxUa8jvJkCT8BF58w5T64rirbTW8CCZ0ijoJA4BMP6n+s76wsDssaQqmRJzXKS0eLXN5RRmXOkNDnOZOXbHVSd5\\\\\\\/Xc+t6WC1e794pUcsZ7wMR+M2q1GRSld4\\\\\\\/ss0X740jwziofMss2J4Ru2jZpqQHBFj9MQ3pA57ZSwbyQQD1\\\\\\\/lqMSxUS0B4DAk5Qn46WPhuolhVzH5TLYyNiVq6D+UhI72+pUj0MbwQ8k3elpIIOiX6uXW0zEl\\\\\\\/KwpKX2XrN1OUDBIkohGLZMv3u8oM3nqPcEalmvO3ffJiBHwep05AYkRTU5kq6OAsAbNnRLEXUoF9enMBtY66Z1P\\\\\\\/WSRS1CrLGRt9Dxssg5nvoctrirMghPAJn4X2rz220\\\u0022,\\\u0022iv\\\u0022:\\\u0022xz9f40ieBFGb98zTS9w4dw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e463bcfb8d95079d587cd806edfbadc292cc73e1f9a6ad896546285dd20eae36\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.04.934232v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1177442 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.04.934232v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7ZHAA5IxEoGGcp9YhC-nlhSodWQIioPfGb4VKpPFJZc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177442" id="node1177442" data-pisa="biorxiv;2020.02.04.934232v2" data-pisa-master="biorxiv;2020.02.04.934232" data-apath="/biorxiv/early/2020/03/02/2020.02.04.934232.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Miyssa I.</span> <span class="nlm-surname">Abdelmageed</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5037-8488" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Abdelrahman H.</span> <span class="nlm-surname">Abdelmoneim</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6893-0536" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Mujahed I.</span> <span class="nlm-surname">Mustafa</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nafisa M.</span> <span class="nlm-surname">Elfadol</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Naseem S.</span> <span class="nlm-surname">Murshed</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Shaza W.</span> <span class="nlm-surname">Shantier</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abdelrafie M.</span> <span class="nlm-surname">Makhawi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.04.934232 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1177442" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Miyssa I. Abdelmageed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Faculty of Pharmacy, University of Khartoum</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miyssa%2BI.%2BAbdelmageed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abdelmageed%20MI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiyssa%2BI.%2BAbdelmageed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abdelrahman H. Abdelmoneim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Faculty of Medicine, Alneelain University</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdelrahman%2BH.%2BAbdelmoneim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abdelmoneim%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbdelrahman%2BH.%2BAbdelmoneim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5037-8488" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abdelrahman H. Abdelmoneim</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mujahed I. Mustafa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biotechnology, University of Bahri</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mujahed%2BI.%2BMustafa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mustafa%20MI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMujahed%2BI.%2BMustafa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6893-0536" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mujahed I. Mustafa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mujahedibrahim44@gmail.com" class="" data-icon-position="" data-hide-link-title="0">mujahedibrahim44@gmail.com</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nafisa M. Elfadol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Molecular biology, National University Biomedical Research Institute, National University</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nafisa%2BM.%2BElfadol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Elfadol%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANafisa%2BM.%2BElfadol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Naseem S. Murshed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Microbiology, International University of Africa</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naseem%2BS.%2BMurshed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murshed%20NS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANaseem%2BS.%2BMurshed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Shaza W. Shantier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Pharmaceutical Chemistry, Faculty of Pharmacy, University of Khartoum</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shaza%2BW.%2BShantier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shantier%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShaza%2BW.%2BShantier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Abdelrafie M. Makhawi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biotechnology, University of Bahri</span>, Khartoum, <span class='nlm-country'>Sudan</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdelrafie%2BM.%2BMakhawi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makhawi%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdelrafie%2BM.%2BMakhawi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.04.934232v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1177442" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1177442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.04.934232v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1177442" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1177442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.04.934232v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1177442" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1177442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.04.934232v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1177442" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1177442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.04.934232v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1177442" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1177442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Background</strong> New endemic disease has been spread across Wuhan City, China on December 2019. Within few weeks, the World Health Organization (WHO) announced a novel coronavirus designated as coronavirus disease 2019 (COVID-19). In late January 2020, WHO declared the outbreak of a “public-health emergency of international concern” due to the rapid and increasing spread of the disease worldwide. Currently, there is no vaccine or approved treatment for this emerging infection; thus the objective of this study is to design a multi epitope peptide vaccine against COVID-19 using immunoinformatics approach.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Method</strong> Several techniques facilitating the combination of immunoinformatics approach and comparative genomic approach were used in order to determine the potential peptides for designing the T cell epitopes-based peptide vaccine using the envelope protein of 2019-nCoV as a target.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Results</strong> Extensive mutations, insertion and deletion were discovered with comparative sequencing in COVID-19 strain. Additionally, ten peptides binding to MHC class I and MHC class II were found to be promising candidates for vaccine design with adequate world population coverage of 88.5% and 99.99%, respectively.</p></div><div id="sec-4" class="subsection"><p id="p-5"><strong>Conclusion</strong> T cell epitopes-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target. Nevertheless, the proposed vaccine is rapidly needed to be validated clinically in order to ensure its safety, immunogenic profile and to help on stopping this epidemic before it leads to devastating global outbreaks.</p></div></div><div class="section" id="sec-5"><h2 class="">1. Introduction</h2><p id="p-6">Coronaviruses (CoV) are a large family of zoonotic viruses that cause illness ranging from the common cold to more severe diseases such as Middle East Respiratory Syndrome (MERS-CoV) and severe Acute Respiratory Syndrome (SARS-CoV). In the last decades, six strains of coronaviruses were identified, however in December 2019; a new strain has been spread across Wuhan City, China [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. It was designated as coronavirus disease 2019 (COVID-19) by the World Health Organization (WHO) [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. In late January 2020, WHO declared the outbreak a global pandemic with cases in more than 45 countries where the COVID-19 spreading fast outside China, most significantly in South Korea, Italy and Iran with over 2,924 deaths and 85,212 cases confirmed while 39,537 recovered at 29 February 2020, 06:05 AM (GMT).</p><p id="p-7">COVID-19 is a positive-sense single stranded RNA virus (+ssRNA). Its RNA sequence is approximately 30,000 bases in length [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. It belongs to the subgenus <em>Sarbecovirus</em>, Genus <em>Betacoronavirus</em> within the family <em>Coronaviridae</em>. The corona envelope (E) protein is a small, integral membrane protein involved in several aspects of the virus’ life cycle, such as pathogenesis, envelope formation, assembly and budding; alongside with its interactions with both other CoVs proteins (M, N &amp; S) and host cell proteins (release of infectious particles after budding) [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>].</p><p id="p-8">The infected person characterized with fever, upper or lower respiratory tract symptoms, or diarrhoea, lymphopenia, thrombocytopenia, and increased C-reactive protein and lactate dehydrogenase levels or combination of all these within 3-6 days after exposure. Further molecular diagnosis can be made by Real Time-PCR for genes encoding the internal RNA-dependent RNA polymerase and Spike’s receptor binding domain, which can be confirmed by Sanger sequencing and full genome analysis by NGS, multiplex nucleic acid amplification and microarray-based assays [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]</p><p id="p-9">A phylogenetic tree of the mutation history of a family of viruses is possible to reconstruct with a sufficient number of sequenced genomes. The phylogenetic analysis indicates that COVID-19 is likely originated from bats [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. It also showed that is highly related with at most seven mutations relative to a common ancestor [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>].</p><p id="p-10">The sequence of COVID-19 RBD, together with its RBM that contacts receptor angiotensinconverting enzyme 2 (ACE2), was found similar to that of SARS coronavirus. On January 2020, a group of scientists demonstrated that ACE2 could act as the receptor for COVID-19 [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>].</p><p id="p-11">However, COVID-19 differs from other previous strains in having several critical residues at 2019-nCoV receptor-binding motif (particularly Gln493) which provide advantageous interactions with human ACE2 [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. This difference in affinity possibly explains why the novel coronavirus is more contagious than those other viruses.</p><p id="p-12">At present, there is no vaccine or approved treatment for humans, but Chinese traditional medicine, such ShuFengJieDu Capsules and Lianhuaqingwen Capsule, could be possible treatments for COVID-19. However, there are no clinical trials approving the safety and efficacy for these drugs [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>].</p><p id="p-13">The main concept within all the immunizations is the ability of the vaccine to initiate an immune response in a faster mode than the pathogen itself. Although traditional vaccines, which depend on biochemical trials, induced potent neutralizing and protective responses in the immunized animals but they can be costly, allergenic, time consuming and require in vitro culture of pathogenic viruses leading to serious concern of safety [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Thus the need for safe and efficacious vaccines is highly recommended.</p><p id="p-14">Peptide-based vaccines do not need in vitro culture making them biologically safe, and their selectivity allows accurate activation of immune responses [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. The core mechanism of the peptide vaccines is built on the chemical method to synthesize the recognized B-cell and T-cell epitopes that are immunodominant and can induce specific immune responses. B-cell epitope of a target molecule can be linked with a T-cell epitope to make it immunogenic. The T-cell epitopes are short peptide fragments (8-20 amino acids), whereas the B-cell epitopes can be proteins [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Therefore, in this study, we aimed to design a peptide-based vaccine to predict epitopes from corona envelope (E) protein using immunoinformatics analysis [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>–<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. While rapid further studies are recommended to prove the efficiency of the predicted epitopes as a peptide vaccine against this emerging infection.</p></div><div class="section" id="sec-6"><h2 class="">2. Materials and Methods</h2><p id="p-15">Workflow summarizing the procedures for the epitope-based peptide vaccine prediction is shown in <strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a></strong></p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Descriptive workflow for the epitope-based peptide vaccine prediction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Descriptive workflow for the epitope-based peptide vaccine prediction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F1.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F1.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-16" class="first-child">Descriptive workflow for the epitope-based peptide vaccine prediction.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-7" class="subsection"><h3>2.1. Data retrieval</h3><p id="p-17">Full genebank files of the complete genomes and annotation of COVID-19(NC_04551) SARS-CoV(FJ211859), MESA-CoV(NC_019843), HCoV-HKU1(AY884001), (HCoV-OC43 (KF923903), HCoV-NL63 (NC_005831) and HCoV-229E (KY983587) were retrieved from the National Center of Biotechnology Information (NCBI); while The FASTA format of envelope (E) protein (YP_009724392.1), spike (S)protein (YP_009724390.1), nucleocapsid (N) protein (YP_009724397.2) and membrane (M) protein (YP_009724393.1) of 2019-nCoV and the envelope (E) protein of two Chinese and two American sequences (YP009724392.1, QHQ71975.1, QHO60596.1 and QHN73797.1) were obtained from the NCBI. (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>)</p></div><div id="sec-8" class="subsection"><h3>2.2. The Artemis Comparison Tool (ACT)</h3><p id="p-18">ACT is an in silico analysis software for visualization of comparisons between complete genome sequences and associated annotations [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. It is also applied to identify regions of similarity, rearrangements and insertions at any level from base-pair differences to the whole genome. (<a href="https://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act">https://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act</a>).</p></div><div id="sec-9" class="subsection"><h3>2.3. VaxiJen server</h3><p id="p-19">It is the first server for alignment-independent prediction of protective antigens. It allows antigen classification solely based on the physicochemical properties of proteins without recourse to sequence alignment. It predicts the probability of the antigenicity one or multiple of protein based on auto cross covariance (ACC) transformation of protein sequence. Structural CoV-2019 protein (N,S,E and M) was analyzed by VaxiJen with threshold of 0.4 [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. (<a href="http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html">http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html</a>)</p></div><div id="sec-10" class="subsection"><h3>2.4. BioEdit</h3><p id="p-20">It is a software package proposed to stream a distinct program that can run nearly any sequence operation as well as a few basic alignment investigations. The sequences of E protein were retrieved from UniProt were run in BioEdit to determine the conserved sites through ClustalW in the application settings [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>].</p></div><div id="sec-11" class="subsection"><h3>2.5. The Molecular Evolutionary Genetics Analysis (MEGA)</h3><p id="p-21">MEGA (version 10.1.6) is software for the comparative analysis of molecular sequences. It is used for pairwise and multiple sequences alignment alongside construction and analysis of phylogenetic trees and evolutionary relationships. The gap penalty was 15 for opening and 6.66 for extending the gap for both pairwise and multiple sequences alignment. Bootstrapping of 300 was used in construction of maximum like hood phylogenetic tree [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. (<a href="https://www.megasoftware.net">https://www.megasoftware.net</a>).</p></div><div id="sec-12" class="subsection"><h3>2.6. Prediction of T-cell epitopes</h3><p id="p-22">IEDB tools were used to predict the conserved sequences (10-mersequence) from HLA class I and class II T-cell epitopes by using artificial neural network (ANN) approach [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. Artificial Neural Network (ANN) version 2.2 was chosen as Prediction method as it depends on the median inhibitory concentration (IC50) [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">43</a>–<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. For the binding analysis, all the alleles were carefully chosen, and the length was set at 10 before prediction was done. Analysis of epitopes binding to MHC class I and II molecules was assessed by the IEDB MHC prediction server at (<a href="http://tools.iedb.org/mhci/">http://tools.iedb.org/mhci/</a>) and (<a href="http://tools.iedb.org/mhcii/">http://tools.iedb.org/mhcii/</a>), respectively. All conserved immunodominant peptides binding to MHC I and II molecules at score equal or less than 100 median inhibitory concentrations (IC50) and 1000, respectively were selected for further analysis while epitopes with IC50 greater than 100 were eliminated [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>].</p></div><div id="sec-13" class="subsection"><h3>2.7. Population coverage analysis</h3><p id="p-23">Population coverage for each epitope was carefully determined by the IEDB population coverage calculation tool. Due to the diverse binding sites of epitopes with different HLA allele, the most promising epitope candidates were calculated for population coverage against the whole world, China and Europe population to get and ensure a universal vaccine [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. (<a href="http://tools.iedb.org/population/">http://tools.iedb.org/population/</a>)</p></div><div id="sec-14" class="subsection"><h3>2.8. Tertiary structure (3D) Modeling</h3><p id="p-24">The reference sequence of E protein that has been retrieved from gene bank was used as an input in RaptorX to predict the 3D structure of E protein [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>], the visualization of the obtained 3D protein structure was performed in UCSF Chimera (version1.8) [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>].</p></div><div id="sec-15" class="subsection"><h3>2.9. In silico Molecular Docking</h3><div id="sec-16" class="subsection"><h4>2.9.1. Ligand Preparation</h4><p id="p-25">In order to estimate the binding affinities between the epitopes and molecular structure of MHC I and MHC II, in silico molecular docking were used. Sequences of proposed epitopes were selected from COVID-19 reference sequence using UCSF Chimera 1.10 and saved as a PDB file. The obtained files were then optimized and energy minimized. The HLA-A*02:01 was selected as the macromolecule for docking. Its crystal structure (4UQ3) was downloaded from the RCSB Protein Data Bank (<a href="http://www.rcsb.org/pdb/home/home.do">http://www.rcsb.org/pdb/home/home.do</a>), which was in a complex with an azobenzene-containing peptide [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>].</p><p id="p-26">All water molecules and heteroatoms in the retrieved target file 4UQ3 were then removed. Target structure was further optimized and energy minimized using Swiss PDB viewer V.4.1.0 software[<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>].</p></div><div id="sec-17" class="subsection"><h4>2.9.2. Molecular docking</h4><p id="p-27">Molecular docking was performed using Autodock 4.0 software, based on Lamarckian Genetic Algorithm; which combines energy evaluation through grids of affinity potential to find the suitable binding position for a ligand on a given protein[<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>,<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]Polar hydrogen atoms were added to the protein targets and Kollman united atomic charges were computed. The target’s grid map was calculated and set to 60×60×60 points with grid spacing of 0.375 Å. The grid box was then allocated properly in the target to include the active residue in the center. The genetic algorithm and its run were set to 100.The docking algorithms were set to default. Finally, results were retrieved as binding energies and poses that showed lowest binding energies were visualized using UCSF Chimera.</p></div></div></div><div class="section" id="sec-18"><h2 class="">3. Results</h2><div id="sec-19" class="subsection"><h3>3.1. The Artemis Comparison Tool</h3><p id="p-28">The reference sequence of envelope protein was aligned with HCov-HKU1 reference protein using artemis comparison tool as illustrated in (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Artemis analysis of envelope protein displaying 3 windows, the upper window represents HCov-HKU1 reference sequence and its genes are highlighted in blue starting from orflab gene and ending with N gene. The middle window describes the similarities and the difference between the two genomes. Red lines indicate match between genes from the two genomes blue lines indicates inversion which represents same sequences in the two genomes but they are organized in the opposite direction, and the lower windows represents COVID-19and its genes started from ARFLAB and ends with N genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Artemis analysis of envelope protein displaying 3 windows, the upper window represents HCov-HKU1 reference sequence and its genes are highlighted in blue starting from &lt;em&gt;orflab&lt;/em&gt; gene and ending with &lt;em&gt;N&lt;/em&gt; gene. The middle window describes the similarities and the difference between the two genomes. Red lines indicate match between genes from the two genomes blue lines indicates inversion which represents same sequences in the two genomes but they are organized in the opposite direction, and the lower windows represents COVID-19and its genes started from &lt;em&gt;ARFLAB&lt;/em&gt; and ends with &lt;em&gt;N&lt;/em&gt; genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F2.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F2.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-29" class="first-child">Artemis analysis of envelope protein displaying 3 windows, the upper window represents HCov-HKU1 reference sequence and its genes are highlighted in blue starting from <em>orflab</em> gene and ending with <em>N</em> gene. The middle window describes the similarities and the difference between the two genomes. Red lines indicate match between genes from the two genomes blue lines indicates inversion which represents same sequences in the two genomes but they are organized in the opposite direction, and the lower windows represents COVID-19and its genes started from <em>ARFLAB</em> and ends with <em>N</em> genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>3.2. VaxiJen server</h3><p id="p-30">The mutated proteins were tested for antigenicity using VaxiJen software, where the envelope protein found as the best immunogenic target in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180068/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180068/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-31" class="first-child">VaxiJen overall prediction of probable COVID-19 antigen:</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>3.3. BioEdit</h3><p id="p-32">Sequence alignment of COVID-19envelope protein was done using BioEdit software which shows total conservation across four sequences which were retrieved from China and USA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence alignment of envelope protein of COVID-19 using BioEdit software (total conservation through the 4 strains: 2 form China and 2 from USA)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sequence alignment of envelope protein of COVID-19 using BioEdit software (total conservation through the 4 strains: 2 form China and 2 from USA)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F3.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F3.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-33" class="first-child">Sequence alignment of envelope protein of COVID-19 using BioEdit software (total conservation through the 4 strains: 2 form China and 2 from USA)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>3.4. The Molecular Evolutionary Genetics Analysis</h3><p id="p-34">To study the evolutionary relationship between all the seven strains of coronavirus, a multiple sequence alignment (MSA) was performed using ClustalW by MEGA software. This alignment was used to construct maximum likelihood phylogenetic tree as seen in <strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a></strong>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood phylogenetic tree which describes the evolutionary relationship between the seven strains of coronavirus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Maximum likelihood phylogenetic tree which describes the evolutionary relationship between the seven strains of coronavirus.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F4.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F4.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-35" class="first-child">Maximum likelihood phylogenetic tree which describes the evolutionary relationship between the seven strains of coronavirus.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>3.5. Prediction of T-cell epitopes and population coverage</h3><p id="p-36">IEDB website was used to analyze 2019-nCoV envelope protein for T cell related peptides. Results show ten MHC class I and II associated peptides with high population coverage (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). The most promising peptides were visualized using UCSF Chimera software (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig.6a</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">b</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic diagrams (A) and (B) showing world population coverage of envelope protein of COVID-19 binding to MHC class I and MHC class II molecule respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic diagrams (&lt;strong&gt;A&lt;/strong&gt;) and (&lt;strong&gt;B&lt;/strong&gt;) showing world population coverage of envelope protein of COVID-19 binding to MHC class I and MHC class II molecule respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F5.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F5.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-37" class="first-child">Schematic diagrams (<strong>A</strong>) and (<strong>B</strong>) showing world population coverage of envelope protein of COVID-19 binding to MHC class I and MHC class II molecule respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="3D structures visualized by UCSF Chimera: (A) and (B) showing the most promising peptides in envelope protein of COVID-19 (yellow colored) binding to MHC class I and MHC class II respectively, while (C, D, and E) showing the molecular docking of YVYSRVKNL, LAILTALRL and SLVKPSFYV peptides of coronavirus docked in HLA-A*02:01 respectively" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-694427809" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;3D structures visualized by UCSF Chimera: (&lt;strong&gt;A&lt;/strong&gt;) and (&lt;strong&gt;B&lt;/strong&gt;) showing the most promising peptides in envelope protein of COVID-19 (yellow colored) binding to MHC class I and MHC class II respectively, while (&lt;strong&gt;C, D&lt;/strong&gt;, and &lt;strong&gt;E&lt;/strong&gt;) showing the molecular docking of YVYSRVKNL, LAILTALRL and SLVKPSFYV peptides of coronavirus docked in HLA-A*02:01 respectively&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F6.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F6.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.04.934232/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-38" class="first-child">3D structures visualized by UCSF Chimera: (<strong>A</strong>) and (<strong>B</strong>) showing the most promising peptides in envelope protein of COVID-19 (yellow colored) binding to MHC class I and MHC class II respectively, while (<strong>C, D</strong>, and <strong>E</strong>) showing the molecular docking of YVYSRVKNL, LAILTALRL and SLVKPSFYV peptides of coronavirus docked in HLA-A*02:01 respectively</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180072/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180072/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-39" class="first-child">The most promised MHC class I related peptides in envelope protein based vaccine of COVID-19 along with the predicted World, China, Europe and East Asia coverage:</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180066/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1180066/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-40" class="first-child">The most promised MHC class II related peptides in envelope protein based vaccine of COVID-19 along with the predicted World, China, Europe and East Asia coverage:</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-24"><h2 class="">4. Discussion</h2><p id="p-41">Designing of a novel vaccine is very crucial to defending the rapid endless of global burden of disease [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>–<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]. In the last few decades, biotechnology has advanced rapidly; alongside with the understanding of immunology which assisted the rise of new approaches towards rational vaccines design [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. Peptide-based vaccines are designed to elicit immunity particular pathogens by selectively stimulating antigen specific for B and T cells [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>].Applying the advanced bioinformatics tools and databases, various peptide-based vaccines could be designed where the peptides act as ligands [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>–<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. This approach has been used frequently in Saint Louis encephalitis virus [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>], dengue virus [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>], chikungunya virus [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>] proposing promising peptides for designing vaccines.</p><p id="p-42">The COVID-19 is an RNA virus which tends to mutate more commonly than the DNA viruses [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. These mutations lied on the surface of the protein, which make COVID-19 more superior than other previous strains by inducing its sustainability leaving the immune system in blind spot [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>].</p><p id="p-43">In our present work, different peptides were proposed for the designing of vaccine against COVID-19 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>). In the beginning, the whole genome of COVID-19 was analyzed by comparative genomic approach to determine the potential antigenic target [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. Artemis Comparative Tool (ACT) was used to analyze human Coronavirus (HCov-HKU1) reference sequence vs. Wuhan-Hu-1 COVID-19. Results obtained (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig.2</a>) revealed extensive mutation among the tested genomes. New genes; <em>ORF8</em> and <em>ORF6</em>; were found inserted in COVID-19 which were absent in HCov-HKU1 that might be acquired by the horizontal gene transmission [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>]. High rate of mutation between the two genomes were observed in the region from 20000 bp to the end of the sequence. This region encodes the four major structural proteins in coronavirus which are envelope (E) protein, nucleocapsid (N) protein, membrane (M) protein, and spike (S) protein, all of which are required to produce a structurally complete virus [<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>].</p><p id="p-44">These conserved antigenic sites were revealed in previous studies through sequence alignment between MERS-CoV and Bat-coronavirus [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>] and analyzed in SARS-CoV [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>].</p><p id="p-45">The four proteins were then analyzed by Vaxigen software to test the probability of antigenic proteins. Protein E was found to be the most antigenic gene with the highest probability as shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Literature survey confirmed this result in which protein E was investigated in severe acute respiratory syndrome (SARS) in 2003 and, more recently, Middle-East respiratory syndrome (MERS) [<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">72</a>]. Furthermore, the conservation of this protein against the seven strains was tested and confirmed through the use of BioEdit package tool (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>).</p><p id="p-46">As phylogenetic analysis is very powerful tool for determining the evolutionary relationship between strains. Multiple sequence alignment (MSA) was performed using ClustalW for the seven strains of coronavirus, which are COVID-19(NC_04551), SARS-CoV (FJ211859), MESA-CoV (NC_019843), HCoV-HKU1 (AY884001), HCoV-OC43 (KF923903), HCoV-NL63 (NC_005831) and HCoV-229E (KY983587). The maximum likelihood phylogenetic tree revealed that COVID-19 is found in the same clade of SARS-CoV, thus the two strains are highly related to each other (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>).</p><p id="p-47">The immune response of T cell is considered as a long lasting response compared to B cell, where the antigen can easily escape the antibody memory response [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. Vaccines that effectively generate cell-mediated response are needed to provide protection against the invading pathogen. Moreover the CD8+ T and CD4+ T cell responses play a major role in antiviral immunity [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. Thus designing vaccine against T cell is much more important.</p><p id="p-48">Choosing protein E as the antigenic site, the binding affinity to MHC molecules was then evaluated. The protein reference sequence was submitted to IEDB MHC predication tool. 21 peptides were found to bind MHC class I with different affinities (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), from which ten peptides were selected for vaccine design based on the number of alleles and world population percentage (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a>).Analysis in IEDB MHC II binding prediction tool resulted in prediction of 61 peptides (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), from which ten peptides were selected for vaccine design based on the number of alleles and world population percentage (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig.5</a>). Unfortunately, IEDB did not give any result for B cell epitopes, this might be due to the length of the COVID-19 (75 amino acids).</p><p id="p-49">It is well known that peptides recognized with high number of HLA molecules are potentially inducing immune response. Based on the aforementioned results and taking into consideration the high binding affinity to both MHC classes I and II, conservancy and population coverage, three peptides are strongly proposed to formulate a new vaccine against COVID-19.</p><p id="p-50">These findings were further confirmed by the results obtained for the molecular docking of the proposed peptides and HLA-A*02:01. The formed complex between MHC molecule and the three peptides (YVYSRVKNL, SLVKPSFYV and LAILTALRL) have shown peptide amino-and carboxyl-termini forming one and three hydrogen bonds, respectively at the two ends of a binding groove with MHC residues with least binding energy −13.2 kcal/mol, −11 kcal/mol and −11.3 kcal/mol, respectively (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig.6C, D and E</a>).</p><p id="p-51">Although both flu and anti-HIV drugs are used currently in China for treatment of COVID-19, and chloroquine phosphate, an old drug for treatment of malaria, has recently found to have apparent efficacy and acceptable safety against COVID-19 [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>]; nevertheless more studies are required to standardize these therapies. In addition, there has been some success in the development of mouse models of MERS-CoV and SARS-CoV infection, and candidate vaccines where the envelope (E) protein is mutated or deleted have been described [<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>–<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>]. To best of our knowledge, this is the first study to identify certain peptides in envelope (E) protein as candidates for COVID-19. Accordingly, these epitopes were strongly recommended as promising epitopes vaccine candidate against T cell.</p></div><div class="section" id="sec-25"><h2 class="">5. Conclusion</h2><p id="p-52">Extensive mutations, insertion and deletion were discovered in COVID-19 strain using the comparative sequencing. In addition, a number of MHC class I and II related peptides were found promising candidates. Among which the peptides YVYSRVKNL, SLVKPSFYV and LAILTALRL show high potentiality for vaccine design with adequate world population coverage. T cell epitope-based peptide vaccine was designed for COVID-19 using envelope protein as an immunogenic target; nevertheless, the proposed vaccine rapidly needs to be validated clinically ensuring its safety and immunogenic profile to help on stopping this epidemic before it leads to devastating global outbreaks.</p></div><div class="section" id="sec-26"><h2 class="">Data Availability</h2><p id="p-54">All data underlying the results are available as part of the article, and no additional source data are required.</p></div><div class="section" id="sec-27"><h2 class="">Authors’ contributions</h2><p id="p-55">MIA: Conceptualization, Formal analysis, Investigation, Methodology, Validation, Visualization, Writing– original draft. AHA: Formal analysis, Investigation, Methodology. MIM: Methodology, Writing– original draft, Writing – review and editing. NME: Formal analysis, Methodology, Visualization. NSM: Conceptualization, Resources, Writing – review &amp; editing. SWS: Visualization, Validation, Writing – review and editing, AMM: Data curation, Conceptualization, Project administration, Supervision, Writing - review and editing. All authors have read and approved the final manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgement</h2><p id="p-53">The authors acknowledge the Deanship of Scientific Research at University of Bahri for the supportive cooperation.</p></div><div class="section glossary"><h2 class="">Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">WHO<span class="def-item-dt-sep"></span></dt><dd>World Health Organization<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">CoV<span class="def-item-dt-sep"></span></dt><dd>Coronaviruses<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">MERS-CoV<span class="def-item-dt-sep"></span></dt><dd>Middle East Respiratory Syndrome<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">SARS-CoV<span class="def-item-dt-sep"></span></dt><dd>Acute Respiratory Syndrome<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">COVID-19<span class="def-item-dt-sep"></span></dt><dd>novel coronavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">HCoV-HKU1<span class="def-item-dt-sep"></span></dt><dd>Human coronavirus HKU1<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">HCoV-OC43<span class="def-item-dt-sep"></span></dt><dd>Human coronavirus OC43<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">HCoV-NL63<span class="def-item-dt-sep"></span></dt><dd>Human coronavirus NL63<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">HCoV-229E<span class="def-item-dt-sep"></span></dt><dd>Human coronavirus 229E<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">ACE2<span class="def-item-dt-sep"></span></dt><dd>angiotensin converting enzyme 2<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">RBM<span class="def-item-dt-sep"></span></dt><dd>receptor-binding motif<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">RBM<span class="def-item-dt-sep"></span></dt><dd>receptor-binding domain<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">vs<span class="def-item-dt-sep"></span></dt><dd>versus<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">ACT<span class="def-item-dt-sep"></span></dt><dd>Artemis Comparison Tool<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">ACC<span class="def-item-dt-sep"></span></dt><dd>auto cross covariance<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">MEGA<span class="def-item-dt-sep"></span></dt><dd>Molecular Evolutionary Genetics Analysis<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">ANN<span class="def-item-dt-sep"></span></dt><dd>Artificial Neural Network<span class="def-item-dd-sep"></span></dd><dt id="def-item-18">IEDB<span class="def-item-dt-sep"></span></dt><dd>Immune Epitope Database<span class="def-item-dd-sep"></span></dd><dt id="def-item-19">IC50<span class="def-item-dt-sep"></span></dt><dd>median inhibitory concentrations<span class="def-item-dd-sep"></span></dd><dt id="def-item-20">MHC I<span class="def-item-dt-sep"></span></dt><dd>Major Histocompatibility complex class I<span class="def-item-dd-sep"></span></dd><dt id="def-item-21">MHC II<span class="def-item-dt-sep"></span></dt><dd>Major Histocompatibility complex class II<span class="def-item-dd-sep"></span></dd><dt id="def-item-22">PDB<span class="def-item-dt-sep"></span></dt><dd>Protein database<span class="def-item-dd-sep"></span></dd><dt id="def-item-23">MSA<span class="def-item-dt-sep"></span></dt><dd>multiple sequence alignment</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Stratton</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y. W.</span> <span class="cit-name-surname">Tang</span></span>, “<span class="cit-article-title">Outbreak of Pneumonia of Unknown Etiology in Wuhan China: the Mystery and the Miracle</span>,” <abbr class="cit-jnl-abbrev">J Med Virol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">16</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.2" data-doi="10.1016/j.ijid.2020.01.009"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Hui</span></span>, I. A. E, <span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Madani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ntoumi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Dar</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health - The latest 2019 novel coronavirus outbreak in Wuhan, China</span>,” <abbr class="cit-jnl-abbrev">Int J Infect Dis</abbr>, vol. <span class="cit-vol">91</span>, pp. <span class="cit-fpage">264</span>–<span class="cit-lpage">266</span>, <span class="cit-month">Jan</span> <span class="cit-day">14</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BInfect%2BDis%26rft.volume%253D91%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijid.2020.01.009%26rft_id%253Dinfo%253Apmid%252F31953166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijid.2020.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31953166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Benvenuto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Giovannetti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ciccozzi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Spoto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Angeletti</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ciccozzi</span></span>, “<span class="cit-article-title">The 2019-new coronavirus epidemic: evidence for virus evolution</span>,” <abbr class="cit-jnl-abbrev">J Med Virol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">29</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.4"><div class="cit-metadata"><cite>“<span class="cit-article-title">CoV2020</span>”. <abbr class="cit-jnl-abbrev">GISAID EpifluDB. Archivedfrom the original on 12 January 2020. Retrieved February 2020</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Venkatagopalan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Daskalova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Lopez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Dolezal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Hogue</span></span>, “<span class="cit-article-title">Coronavirus envelope (E) protein remains at the site of assembly</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">478</span>, pp. <span class="cit-fpage">75</span>–<span class="cit-lpage">85</span>, <span class="cit-month">Apr</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D478%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.6" data-doi="10.1016/j.virol.2011.03.029"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Nieto-Torres</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Dediego</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alvarez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Jimenez-Guardeno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Regla-Nava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Llorente</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Subcellular location and topology of severe acute respiratory syndrome coronavirus envelope protein</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">415</span>, pp. <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>, <span class="cit-month">Jul</span> <span class="cit-day">5</span> <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D415%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2011.03.029%26rft_id%253Dinfo%253Apmid%252F21524776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2011.03.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21524776&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.7" data-doi="10.1128/JVI.76.3.1422-1434.2002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Curtis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yount</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Baric</span></span>, “<span class="cit-article-title">Heterologous gene expression from transmissible gastroenteritis virus replicon particles</span>,” <abbr class="cit-jnl-abbrev">J Virol</abbr>, vol. <span class="cit-vol">76</span>, pp. <span class="cit-fpage">1422</span>–<span class="cit-lpage">34</span>, <span class="cit-month">Feb</span> <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCurtis%26rft.auinit1%253DK.%2BM.%26rft.volume%253D76%26rft.issue%253D3%26rft.spage%253D1422%26rft.epage%253D1434%26rft.atitle%253DHeterologous%2BGene%2BExpression%2Bfrom%2BTransmissible%2BGastroenteritis%2BVirus%2BReplicon%2BParticles%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.3.1422-1434.2002%26rft_id%253Dinfo%253Apmid%252F11773416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc2LzMvMTQyMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMDQuOTM0MjMyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.8" data-doi="10.1128/JVI.76.22.11518-11529.2002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ortego</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Escors</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Laude</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Enjuanes</span></span>, “<span class="cit-article-title">Generation of a replication-competent, propagation-deficient virus vector based on the transmissible gastroenteritis coronavirus genome</span>,” <abbr class="cit-jnl-abbrev">J Virol</abbr>, vol. <span class="cit-vol">76</span>, pp. <span class="cit-fpage">11518</span>–<span class="cit-lpage">29</span>, <span class="cit-month">Nov</span> <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DOrtego%26rft.auinit1%253DJ.%26rft.volume%253D76%26rft.issue%253D22%26rft.spage%253D11518%26rft.epage%253D11529%26rft.atitle%253DGeneration%2Bof%2Ba%2BReplication-Competent%252C%2BPropagation-Deficient%2BVirus%2BVector%2BBased%2Bon%2Bthe%2BTransmissible%2BGastroenteritis%2BCoronavirus%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.22.11518-11529.2002%26rft_id%253Dinfo%253Apmid%252F12388713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3Ni8yMi8xMTUxOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMDQuOTM0MjMyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.9" data-doi="10.1371/journal.ppat.1002674"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Ruch</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Machamer</span></span>, “<span class="cit-article-title">A single polar residue and distinct membrane topologies impact the function of the infectious bronchitis coronavirus E protein</span>,” <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>, vol. <span class="cit-vol">8</span>, p. <span class="cit-fpage">e1002674</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRuch%26rft.auinit1%253DT.%2BR.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002674%26rft.epage%253De1002674%26rft.atitle%253DA%2Bsingle%2Bpolar%2Bresidue%2Band%2Bdistinct%2Bmembrane%2Btopologies%2Bimpact%2Bthe%2Bfunction%2Bof%2Bthe%2Binfectious%2Bbronchitis%2Bcoronavirus%2BE%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002674%26rft_id%253Dinfo%253Apmid%252F22570613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Chan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Kok</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">To</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>,” <abbr class="cit-jnl-abbrev">Lancet</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">24</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Su</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Recent advances in the detection of respiratory virus infection in humans</span>,” <abbr class="cit-jnl-abbrev">J Med Virol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">15</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.12" data-doi="10.3109/10408363.2011.640976"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Mahony</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Petrich</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Smieja</span></span>, “<span class="cit-article-title">Molecular diagnosis of respiratory virus infections</span>,” <abbr class="cit-jnl-abbrev">Crit Rev Clin Lab Sci</abbr>, vol. <span class="cit-vol">48</span>, pp. <span class="cit-fpage">217</span>–<span class="cit-lpage">49</span>, <span class="cit-month">Sep-Dec</span> <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bclinical%2Blaboratory%2Bsciences%26rft.stitle%253DCrit%2BRev%2BClin%2BLab%2BSci%26rft.aulast%253DMahony%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D217%26rft.epage%253D249%26rft.atitle%253DMolecular%2Bdiagnosis%2Bof%2Brespiratory%2Bvirus%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F10408363.2011.640976%26rft_id%253Dinfo%253Apmid%252F22185616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/10408363.2011.640976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22185616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Landt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kaiser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Molenkamp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Meijer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>,” <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, vol. <span class="cit-vol">25</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Guo</span></span>, “<span class="cit-article-title">Emerging coronaviruses: genome structure, replication, and pathogenesis</span>,” <abbr class="cit-jnl-abbrev">J Med Virol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">22</span> <span class="cit-pub-date">2020</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Graham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Baric</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, “<span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>, <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">29</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.16"><div class="cit-metadata"><cite>“<span class="cit-article-title">Genomic epidemiology of novel coronavirus (nCoV)</span>”. <abbr class="cit-jnl-abbrev">nextstrain.org</abbr>. Retrieved <span class="cit-day">11</span> <span class="cit-month">February</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letko</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">V</span></span> (<span class="cit-month">January</span> <span class="cit-pub-date">2020</span>). “<span class="cit-article-title">Functional assessment of cell entry and receptor usage for lineage Bß-coronaviruses, including 2019-nCoV</span>”. bioRxiv 2020.01.22.915660 Check |biorxiv= value (help).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.18" data-doi="10.1038/s41564-020-0688-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letko</span>,  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marzi</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-month">February</span> <span class="cit-pub-date">2020</span>). “<span class="cit-article-title">Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</span>”. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41564-020-0688-y</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID </span><span class="cit-pub-id cit-pub-id-pmid">32094589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-020-0688-y%26rft_id%253Dinfo%253Apmid%252F32094589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41564-020-0688-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32094589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">El Sahly</span>  <span class="cit-name-given-names">HM.</span></span> “<span class="cit-article-title">Genomic Characterization of the 2019 Novel Coronavirus</span>”. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. Retrieved <span class="cit-day">9</span> <span class="cit-month">February</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.20" data-doi="10.3390/v12020135"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menachery</span>  <span class="cit-name-given-names">VD</span></span> (<span class="cit-month">January</span> <span class="cit-pub-date">2020</span>). “<span class="cit-article-title">Return of the Coronavirus: 2019-nCoV</span>”. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-vol">12</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v12020135</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID </span><span class="cit-pub-id cit-pub-id-pmid">31991541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D12%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv12020135%26rft_id%253Dinfo%253Apmid%252F31991541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v12020135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31991541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.21" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-month">January</span> <span class="cit-pub-date">2020</span>). “<span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>”. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-vol">395</span>(<span class="cit-issue">10224</span>):<span class="cit-fpage">565</span>–<span class="cit-lpage">574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)302518</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID </span><span class="cit-pub-id cit-pub-id-pmid">32007145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lu</span></span>, “<span class="cit-article-title">Drug treatment options for the 2019-new coronavirus (2019-nCoV)</span>,” <abbr class="cit-jnl-abbrev">Biosci Trends</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">28</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y. T.</span> <span class="cit-name-surname">Lo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. W.</span> <span class="cit-name-surname">Pai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. K.</span> <span class="cit-name-surname">Wu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H. T.</span> <span class="cit-name-surname">Chang</span></span>, “<span class="cit-article-title">Prediction of conformational epitopes with the use of a knowledge-based energy function and geometrically related neighboring residue characteristics</span>,” <abbr class="cit-jnl-abbrev">BMCBioinformatics</abbr>, vol. <span class="cit-vol">14</span> Suppl <span class="cit-issue">4</span>, p. <span class="cit-fpage">S3</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMCBioinformatics%26rft.volume%253D14%26rft.spage%253D3S%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Joshi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Singhania</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Ramsey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Murthy</span></span>, “<span class="cit-article-title">Peptide Vaccine: Progress and Challenges</span>,” <abbr class="cit-jnl-abbrev">Vaccines (Basel)</abbr>, vol. <span class="cit-vol">2</span>, pp. <span class="cit-fpage">515</span>–<span class="cit-lpage">36</span>, <span class="cit-month">Jul</span> <span class="cit-day">2</span> <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccines%2B%2528Basel%2529%26rft.volume%253D2%26rft.spage%253D515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.25" data-doi="10.1038/nrd2224"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Purcell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McCluskey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rossjohn</span></span>, “<span class="cit-article-title">More than one reason to rethink the use of peptides in vaccine design</span>,” <abbr class="cit-jnl-abbrev">Nat Rev Drug Discov</abbr>, vol. <span class="cit-vol">6</span>, pp. <span class="cit-fpage">404</span>–<span class="cit-lpage">14</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BDrug%2Bdiscovery%26rft.stitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.aulast%253DPurcell%26rft.auinit1%253DA.%2BW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D404%26rft.epage%253D414%26rft.atitle%253DMore%2Bthan%2Bone%2Breason%2Bto%2Brethink%2Bthe%2Buse%2Bof%2Bpeptides%2Bin%2Bvaccine%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd2224%26rft_id%253Dinfo%253Apmid%252F17473845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd2224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17473845&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246306700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.26" data-doi="10.2174/138161210793292447"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Dudek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Perlmutter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">Aguilar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">Croft</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Purcell</span></span>, “<span class="cit-article-title">Epitope discovery and their use in peptide based vaccines</span>,” <abbr class="cit-jnl-abbrev">Curr Pharm Des</abbr>, vol. <span class="cit-vol">16</span>, pp. <span class="cit-fpage">3149</span>–<span class="cit-lpage">57</span>, <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bpharmaceutical%2Bdesign%26rft.stitle%253DCurr%2BPharm%2BDes%26rft.aulast%253DDudek%26rft.auinit1%253DN.%2BL.%26rft.volume%253D16%26rft.issue%253D28%26rft.spage%253D3149%26rft.epage%253D3157%26rft.atitle%253DEpitope%2Bdiscovery%2Band%2Btheir%2Buse%2Bin%2Bpeptide%2Bbased%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138161210793292447%26rft_id%253Dinfo%253Apmid%252F20687873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138161210793292447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20687873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dermime</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Gilham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Davidson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Meziane el</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Armstrong</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Vaccine and antibody-directed T cell tumour immunotherapy</span>,” <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr>, vol. <span class="cit-vol">1704</span>, pp. <span class="cit-fpage">11</span>–<span class="cit-lpage">35</span>, <span class="cit-month">Jul</span> <span class="cit-day">6</span> <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DDermime%26rft.auinit1%253DS.%26rft.volume%253D1704%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D35%26rft.atitle%253DVaccine%2Band%2Bantibody-directed%2BT%2Bcell%2Btumour%2Bimmunotherapy.%26rft_id%253Dinfo%253Apmid%252F15238242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15238242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.28" data-doi="10.1006/biol.2001.0298"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Meloen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Langeveld</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Schaaper</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Slootstra</span></span>, “<span class="cit-article-title">Synthetic peptide vaccines: unexpected fulfillment of discarded hope?</span>,” <abbr class="cit-jnl-abbrev">Biologicals</abbr>, vol. <span class="cit-vol">29</span>, pp. <span class="cit-fpage">233</span>–<span class="cit-lpage">6</span>, <span class="cit-month">Sep-Dec</span> <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiologicals%2B%253A%2B%2Bjournal%2Bof%2Bthe%2BInternational%2BAssociation%2Bof%2BBiological%2BStandardization%26rft.stitle%253DBiologicals%26rft.aulast%253DMeloen%26rft.auinit1%253DR.%2BH.%26rft.volume%253D29%26rft.issue%253D3-4%26rft.spage%253D233%26rft.epage%253D236%26rft.atitle%253DSynthetic%2Bpeptide%2Bvaccines%253A%2Bunexpected%2Bfulfillment%2Bof%2Bdiscarded%2Bhope%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fbiol.2001.0298%26rft_id%253Dinfo%253Apmid%252F11851321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/biol.2001.0298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11851321&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173352400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.29" data-doi="10.1586/1744666X.1.1.145"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Brusic</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Petrovsky</span></span>, “<span class="cit-article-title">Immunoinformatics and its relevance to understanding human immune disease</span>,” <abbr class="cit-jnl-abbrev">Expert Rev Clin Immunol</abbr>, vol. <span class="cit-vol">1</span>, pp. <span class="cit-fpage">145</span>–<span class="cit-lpage">57</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DExpert%2BRev%2BClin%2BImmunol%26rft.aulast%253DBrusic%26rft.auinit1%253DV.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D145%26rft.epage%253D157%26rft.atitle%253DImmunoinformatics%2Band%2Bits%2Brelevance%2Bto%2Bunderstanding%2Bhuman%2Bimmune%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1586%252F1744666X.1.1.145%26rft_id%253Dinfo%253Apmid%252F20477662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/1744666X.1.1.145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20477662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Tomar</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">De</span></span>, “<span class="cit-article-title">Immunoinformatics: a brief review</span>,” <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>, vol. <span class="cit-vol">1184</span>, pp. <span class="cit-fpage">23</span>–<span class="cit-lpage">55</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1184%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.31" data-doi="10.1556/AMicr.61.2014.3.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Khalili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jahangiri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Borna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ahmadi Zanoos</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Amani</span></span>, “<span class="cit-article-title">Computational vaccinology and epitope vaccine design by immunoinformatics</span>,” <abbr class="cit-jnl-abbrev">Acta Microbiol Immunol Hung</abbr>, vol. <span class="cit-vol">61</span>, pp. <span class="cit-fpage">285</span>–<span class="cit-lpage">307</span>, <span class="cit-month">Sep</span> <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BMicrobiol%2BImmunol%2BHung%26rft.volume%253D61%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1556%252FAMicr.61.2014.3.4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1556/AMicr.61.2014.3.4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.32"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Tomar</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">De</span></span>, “<span class="cit-article-title">Immunoinformatics: an integrated scenario</span>,” <abbr class="cit-jnl-abbrev">Immunology</abbr>, vol. <span class="cit-vol">131</span>, pp. <span class="cit-fpage">153</span>–<span class="cit-lpage">68</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BImmunology%26rft.stitle%253DSci.%2BImmunol.%26rft.aulast%253DTomar%26rft.auinit1%253DN.%26rft.volume%253D131%26rft.issue%253D2%26rft.spage%253D153%26rft.epage%253D168%26rft.atitle%253DImmunoinformatics%253A%2Ban%2Bintegrated%2Bscenario.%26rft_id%253Dinfo%253Apmid%252F20722763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20722763&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.33"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Hegde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gauthami</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Sampath Kumar</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bayry</span></span>, “<span class="cit-article-title">The use of databases, data mining and immunoinformatics in vaccinology: where are we?</span>,” <abbr class="cit-jnl-abbrev">Expert Opin Drug Discov</abbr>, vol. <span class="cit-vol">13</span>, pp. <span class="cit-fpage">117</span>–<span class="cit-lpage">130</span>, <span class="cit-month">Feb</span> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BOpin%2BDrug%2BDiscov%26rft.volume%253D13%26rft.spage%253D117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Bahrami</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Payandeh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Khalili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zakeri</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bandehpour</span></span>, “<span class="cit-article-title">Immunoinformatics: In Silico Approaches and Computational Design of a Multiepitope, Immunogenic Protein</span>,” <abbr class="cit-jnl-abbrev">Int Rev Immunol</abbr>, vol. <span class="cit-vol">38</span>, pp. <span class="cit-fpage">307</span>–<span class="cit-lpage">322</span>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BRev%2BImmunol%26rft.volume%253D38%26rft.spage%253D307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.35" data-doi="10.1093/bioinformatics/bti553"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Carver</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rutherford</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Berriman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Rajandream</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Barrell</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Parkhill</span></span>, “<span class="cit-article-title">ACT: the Artemis Comparison Tool</span>,” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. <span class="cit-vol">21</span>, pp. <span class="cit-fpage">3422</span>–<span class="cit-lpage">3</span>, <span class="cit-month">Aug</span> <span class="cit-day">15</span> <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti553%26rft_id%253Dinfo%253Apmid%252F15976072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15976072&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231360600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.36" data-doi="10.1186/1471-2105-8-4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Doytchinova</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Flower</span></span>, “<span class="cit-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines</span>,” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, vol. <span class="cit-vol">8</span>, p. <span class="cit-fpage">4</span>, <span class="cit-month">Jan</span> <span class="cit-day">5</span> <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDoytchinova%26rft.auinit1%253DI.%2BA.%26rft.volume%253D8%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DVaxiJen%253A%2Ba%2Bserver%2Bfor%2Bprediction%2Bof%2Bprotective%2Bantigens%252C%2Btumour%2Bantigens%2Band%2Bsubunit%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-4%26rft_id%253Dinfo%253Apmid%252F17207271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17207271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">O. O.</span> <span class="cit-name-surname">Oni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Owoade</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. A. O.</span> <span class="cit-name-surname">Adeyefa</span></span>, “<span class="cit-article-title">Design and evaluation of primer pairs for efficient detection of avian rotavirus</span>,” <abbr class="cit-jnl-abbrev">Trop Anim Health Prod</abbr>, vol. <span class="cit-vol">50</span>, pp. <span class="cit-fpage">267</span>–<span class="cit-lpage">273</span>, <span class="cit-month">Feb</span> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrop%2BAnim%2BHealth%2BProd%26rft.volume%253D50%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.38"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, “<span class="cit-article-title">Molecular Evolutionary Genetics Analysis (MEGA) for macOS</span>,” <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">6</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.39" data-doi="10.1093/molbev/msm092"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dudley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nei</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, “<span class="cit-article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</span>,” <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, vol. <span class="cit-vol">24</span>, pp. <span class="cit-fpage">1596</span>–<span class="cit-lpage">9</span>, <span class="cit-month">Aug</span> <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm092%26rft_id%253Dinfo%253Apmid%252F17488738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17488738&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.40" data-doi="10.1110/ps.0239403"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Worning</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Lauemoller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Reliable prediction of T-cell epitopes using neural networks with novel sequence representations</span>,” <abbr class="cit-jnl-abbrev">Protein Sci</abbr>, vol. <span class="cit-vol">12</span>, pp. <span class="cit-fpage">1007</span>–<span class="cit-lpage">17</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DNielsen%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1007%26rft.epage%253D1017%26rft.atitle%253DReliable%2Bprediction%2Bof%2BT-cell%2Bepitopes%2Busing%2Bneural%2Bnetworks%2Bwith%2Bnovel%2Bsequence%2Brepresentations%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.0239403%26rft_id%253Dinfo%253Apmid%252F12717023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.0239403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12717023&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182486800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.41" data-doi="10.1093/nar/gkn202"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Harndahl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, “<span class="cit-article-title">NetMHC-3.0: accurate web accessible predictions of human, mouse and monkey MHC class I affinities for peptides of length 8-11</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, vol. <span class="cit-vol">36</span>, pp. <span class="cit-fpage">W509</span>–<span class="cit-lpage">12</span>, <span class="cit-month">Jul</span> <span class="cit-day">1</span> <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn202%26rft_id%253Dinfo%253Apmid%252F18463140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18463140&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258142300093&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.42" data-doi="10.1093/bioinformatics/btv639"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Andreatta</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, “<span class="cit-article-title">Gapped sequence alignment using artificial neural networks: application to the MHC class I system</span>,” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. <span class="cit-vol">32</span>, pp. <span class="cit-fpage">511</span>–<span class="cit-lpage">7</span>, <span class="cit-month">Feb</span> <span class="cit-day">15</span> <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv639%26rft_id%253Dinfo%253Apmid%252F26515819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26515819&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.43" data-doi="10.1034/j.1399-0039.2003.00112.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Lauemoller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Worning</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kesmir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Frimurer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Corbet</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Sensitive quantitative predictions of peptide-MHC binding by a ‘Query by Committee’ artificial neural network approach</span>,” <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr>, vol. <span class="cit-vol">62</span>, pp. <span class="cit-fpage">378</span>–<span class="cit-lpage">84</span>, <span class="cit-month">Nov</span> <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DBuus%26rft.auinit1%253DS.%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D378%26rft.epage%253D384%26rft.atitle%253DSensitive%2Bquantitative%2Bpredictions%2Bof%2Bpeptide-MHC%2Bbinding%2Bby%2Ba%2B%2527Query%2Bby%2BCommittee%2527%2Bartificial%2Bneural%2Bnetwork%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1399-0039.2003.00112.x%26rft_id%253Dinfo%253Apmid%252F14617044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1399-0039.2003.00112.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14617044&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185876600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.44" data-doi="10.1093/bioinformatics/btn128"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, “<span class="cit-article-title">Accurate approximation method for prediction of class I MHC affinities for peptides of length 8, 10 and 11 using prediction tools trained on 9mers</span>,” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. <span class="cit-vol">24</span>, pp. <span class="cit-fpage">1397</span>–<span class="cit-lpage">8</span>, <span class="cit-month">Jun</span> <span class="cit-day">1</span> <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn128%26rft_id%253Dinfo%253Apmid%252F18413329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18413329&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.45" data-doi="10.1016/j.tibtech.2006.10.001"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lund</span></span>, “<span class="cit-article-title">The validity of predicted T-cell epitopes</span>,” <abbr class="cit-jnl-abbrev">Trends Biotechnol</abbr>, vol. <span class="cit-vol">24</span>, pp. <span class="cit-fpage">537</span>–<span class="cit-lpage">8</span>, <span class="cit-month">Dec</span> <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DLundegaard%26rft.auinit1%253DC.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D537%26rft.epage%253D538%26rft.atitle%253DThe%2Bvalidity%2Bof%2Bpredicted%2BT-cell%2Bepitopes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2006.10.001%26rft_id%253Dinfo%253Apmid%252F17045685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2006.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17045685&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Southwood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kondo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">del Guercio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Appella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hoffman</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Several common HLA-DR types share largely overlapping peptide binding repertoires</span>,” <abbr class="cit-jnl-abbrev">J Immunol</abbr>, vol. <span class="cit-vol">160</span>, pp. <span class="cit-fpage">3363</span>–<span class="cit-lpage">73</span>, <span class="cit-month">Apr</span> <span class="cit-day">1</span> <span class="cit-pub-date">1998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DSouthwood%26rft.auinit1%253DS.%26rft.volume%253D160%26rft.issue%253D7%26rft.spage%253D3363%26rft.epage%253D3373%26rft.atitle%253DSeveral%2BCommon%2BHLA-DR%2BTypes%2BShare%2BLargely%2BOverlapping%2BPeptide%2BBinding%2BRepertoires%26rft_id%253Dinfo%253Apmid%252F9531296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE2MC83LzMzNjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjA0LjkzNDIzMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.47" data-doi="10.1098/rsob.120139"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Patronov</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Doytchinova</span></span>, “<span class="cit-article-title">T-cell epitope vaccine design by immunoinformatics</span>,” <abbr class="cit-jnl-abbrev">Open Biol</abbr>, vol. <span class="cit-vol">3</span>, p. <span class="cit-fpage">120139</span>, <span class="cit-month">Jan</span> <span class="cit-day">8</span> <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsob.120139%26rft_id%253Dinfo%253Apmid%252F23303307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsob.120139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23303307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.48" data-doi="10.1186/1471-2105-7-153"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Bui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Dinh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Southwood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Newman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, “<span class="cit-article-title">Predicting population coverage of T-cell epitope-based diagnostics and vaccines</span>,” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, vol. <span class="cit-vol">7</span>, p. <span class="cit-fpage">153</span>, <span class="cit-month">Mar</span> <span class="cit-day">17</span> <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBui%26rft.auinit1%253DH.%2BH.%26rft.volume%253D7%26rft.spage%253D153%26rft.epage%253D153%26rft.atitle%253DPredicting%2Bpopulation%2Bcoverage%2Bof%2BT-cell%2Bepitope-based%2Bdiagnostics%2Band%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-153%26rft_id%253Dinfo%253Apmid%252F16545123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16545123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.49" data-doi="10.1093/nar/gkw1070"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Benson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cavanaugh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Clark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Karsch-Mizrachi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ostell</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">GenBank</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, vol. <span class="cit-vol">45</span>, pp. <span class="cit-fpage">D37</span>–<span class="cit-lpage">d42</span>, <span class="cit-month">Jan</span> <span class="cit-day">4</span> <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1070%26rft_id%253Dinfo%253Apmid%252F27899564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.50" data-doi="10.1093/nar/gkw306"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, “<span class="cit-article-title">RaptorX-Property: a web server for protein structure property prediction</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, vol. <span class="cit-vol">44</span>, pp. <span class="cit-fpage">W430</span>–<span class="cit-lpage">5</span>, <span class="cit-month">Jul</span> <span class="cit-day">8</span> <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw306%26rft_id%253Dinfo%253Apmid%252F27112573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27112573&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.51" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. F.</span> <span class="cit-name-surname">Pettersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Goddard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Couch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Greenblatt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span>,” <abbr class="cit-jnl-abbrev">J Comput Chem</abbr>, vol. <span class="cit-vol">25</span>, pp. <span class="cit-fpage">1605</span>–<span class="cit-lpage">12</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.52" data-doi="10.1002/anie.201406295"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Choo</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thong</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Esch</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raida</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meijers</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bioorthogonal cleavage and exchange of major histocompatibility complex ligands by employing azobenzene-containing peptides</span>. <abbr class="cit-jnl-abbrev">Angewandte Chemie (International ed in English)</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">53</span>(<span class="cit-issue">49</span>):<span class="cit-fpage">13390</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAngewandte%2BChemie%2B%2528International%2Bed%2Bin%2BEnglish%2529%26rft.volume%253D53%26rft.spage%253D13390%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fanie.201406295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/anie.201406295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.53" data-doi="10.1002/elps.1150181505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guex</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peitsch</span>  <span class="cit-name-given-names">MC.</span></span> <span class="cit-article-title">SWISS-MODEL and the Swiss-Pdb Viewer: an environment for comparative protein modeling</span>. <abbr class="cit-jnl-abbrev">Electrophoresis</abbr>. <span class="cit-pub-date">1997</span>;<span class="cit-vol">18</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">2714</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.stitle%253DElectrophoresis%26rft.aulast%253DGuex%26rft.auinit1%253DN.%26rft.volume%253D18%26rft.issue%253D15%26rft.spage%253D2714%26rft.epage%253D2723%26rft.atitle%253DSWISS-MODEL%2Band%2Bthe%2BSwiss-PdbViewer%253A%2Ban%2Benvironment%2Bfor%2Bcomparative%2Bprotein%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.1150181505%26rft_id%253Dinfo%253Apmid%252F9504803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.1150181505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9504803&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071948300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.54" data-doi="10.1002/jcc.21256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanner</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodsell</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span>. <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">30</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">2785</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DMorris%26rft.auinit1%253DG.%2BM.%26rft.volume%253D30%26rft.issue%253D16%26rft.spage%253D2785%26rft.epage%253D2791%26rft.atitle%253DAutoDock4%2Band%2BAutoDockTools4%253A%2BAutomated%2Bdocking%2Bwith%2Bselective%2Breceptor%2Bflexibility.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.21256%26rft_id%253Dinfo%253Apmid%252F19399780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.21256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19399780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271112100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G</span></span>, S. <span class="cit-auth"><span class="cit-name-surname">Goodsell</span>  <span class="cit-name-given-names">D</span></span>, S. <span class="cit-auth"><span class="cit-name-surname">Halliday</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Automated Docking Using a Lamarckian Genetic Algorithm and Empirical Binding Free Energy Function</abbr> <span class="cit-pub-date">1998</span>. <span class="cit-fpage">1639</span>–<span class="cit-lpage">62</span> p</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.56"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Marshall</span></span>, “<span class="cit-article-title">Developing countries face double burden of disease</span>,” <abbr class="cit-jnl-abbrev">Bull World Health Organ</abbr>, vol. <span class="cit-vol">82</span>, p. <span class="cit-fpage">556</span>, <span class="cit-month">Jul</span> <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DMarshall%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D82%26rft.issue%253D7%26rft.spage%253D556%26rft.epage%253D556%26rft.atitle%253DDeveloping%2Bcountries%2Bface%2Bdouble%2Bburden%2Bof%2Bdisease.%26rft_id%253Dinfo%253Apmid%252F15500291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15500291&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222722300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.57" data-doi="10.1586/14760584.3.1.59"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">De Groot</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rappuoli</span></span>, “<span class="cit-article-title">Genome-derived vaccines</span>,” <abbr class="cit-jnl-abbrev">Expert Rev Vaccines</abbr>, vol. <span class="cit-vol">3</span>, pp. <span class="cit-fpage">59</span>–<span class="cit-lpage">76</span>, <span class="cit-month">Feb</span> <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2Breview%2Bof%2Bvaccines%26rft.stitle%253DExpert%2BRev%2BVaccines%26rft.aulast%253DDe%2BGroot%26rft.auinit1%253DA.%2BS.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D76%26rft.atitle%253DGenome-derived%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1586%252F14760584.3.1.59%26rft_id%253Dinfo%253Apmid%252F14761244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/14760584.3.1.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14761244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.58" data-doi="10.1371/journal.pcbi.0020071"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Korber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">LaBute</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yusim</span></span>, “<span class="cit-article-title">Immunoinformatics comes of age</span>,” <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>, vol. <span class="cit-vol">2</span>, p. <span class="cit-fpage">e71</span>, <span class="cit-month">Jun</span> <span class="cit-day">30</span> <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DKorber%26rft.auinit1%253DB.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253De71%26rft.epage%253De71%26rft.atitle%253DImmunoinformatics%2Bcomes%2Bof%2Bage.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020071%26rft_id%253Dinfo%253Apmid%252F16846250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16846250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.59" data-doi="10.1016/j.cell.2006.02.010"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Fauci</span></span>, “<span class="cit-article-title">Emerging and re-emerging infectious diseases: influenza as a prototype of the host-pathogen balancing act</span>,” <abbr class="cit-jnl-abbrev">Cell</abbr>, vol. <span class="cit-vol">124</span>, pp. <span class="cit-fpage">665</span>–<span class="cit-lpage">70</span>, <span class="cit-month">Feb</span> <span class="cit-day">24</span> <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFauci%26rft.auinit1%253DA.%2BS.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D665%26rft.epage%253D670%26rft.atitle%253DEmerging%2Band%2Bre-emerging%2Binfectious%2Bdiseases%253A%2Binfluenza%2Bas%2Ba%2Bprototype%2Bof%2Bthe%2Bhost-pathogen%2Bbalancing%2Bact.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.02.010%26rft_id%253Dinfo%253Apmid%252F16497575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16497575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237240900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.60"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chaudhury</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. H.</span> <span class="cit-name-surname">Duncan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Atre</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dutta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Spring</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Leitner</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Combining immunoprofiling with machine learning to assess the effects of adjuvant formulation on human vaccine-induced immunity</span>,” <abbr class="cit-jnl-abbrev">Hum Vaccin Immunother</abbr>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>, <span class="cit-month">Oct</span> <span class="cit-day">7</span> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.61" data-doi="10.1038/nrd2224"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossjohn</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">More than one reason to rethink the use of peptides in vaccine design</span>. <abbr class="cit-jnl-abbrev">Nat Rev Drug Discov</abbr> <span class="cit-pub-date">2007</span>; <span class="cit-vol">6</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">404</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">17473845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BDrug%2Bdiscovery%26rft.stitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.aulast%253DPurcell%26rft.auinit1%253DA.%2BW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D404%26rft.epage%253D414%26rft.atitle%253DMore%2Bthan%2Bone%2Breason%2Bto%2Brethink%2Bthe%2Buse%2Bof%2Bpeptides%2Bin%2Bvaccine%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd2224%26rft_id%253Dinfo%253Apmid%252F17473845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd2224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17473845&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246306700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.62"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Usmani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bhalla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kumar</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. P. S.</span> <span class="cit-name-surname">Raghava</span></span>, “<span class="cit-article-title">In Silico Tools and Databases for Designing Peptide-Based Vaccine and Drugs</span>,” <abbr class="cit-jnl-abbrev">Adv Protein Chem Struct Biol</abbr>, vol. <span class="cit-vol">112</span>, pp. <span class="cit-fpage">221</span>–<span class="cit-lpage">263</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BProtein%2BChem%2BStruct%2BBiol%26rft.volume%253D112%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.63" data-doi="10.1093/bib/bbw025"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Dhanda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Usmani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Agrawal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Nagpal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gautam</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. P. S.</span> <span class="cit-name-surname">Raghava</span></span>, “<span class="cit-article-title">Novel in silico tools for designing peptide-based subunit vaccines and immunotherapeutics</span>,” <abbr class="cit-jnl-abbrev">Brief Bioinform</abbr>, vol. <span class="cit-vol">18</span>, pp. <span class="cit-fpage">467</span>–<span class="cit-lpage">478</span>, <span class="cit-month">May</span> <span class="cit-day">1</span> <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BBioinform%26rft.volume%253D18%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbw025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbw025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.64"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Khazaei-Poul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Farhadi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ghani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Ahmadizad</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ranjbari</span></span>, “<span class="cit-article-title">Monocyclic Peptides: Types, Synthesis and applications</span>,” <abbr class="cit-jnl-abbrev">Curr Pharm Biotechnol</abbr>, <span class="cit-month">Jan</span> <span class="cit-day">20</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.65"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Hasan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hossain</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Alam</span></span>, “<span class="cit-article-title">A computational assay to design an epitopebased Peptide vaccine against saint louis encephalitis virus</span>,” <abbr class="cit-jnl-abbrev">Bioinform Biol Insights</abbr>, vol. <span class="cit-vol">7</span>, pp. <span class="cit-fpage">347</span>–<span class="cit-lpage">55</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinform%2BBiol%2BInsights%26rft.volume%253D7%26rft.spage%253D347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.66"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chakraborty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chakravorty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ahmed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rahman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Waise</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hassan</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">A computational approach for identification of epitopes in dengue virus envelope protein: a step towards designing a universal dengue vaccine targeting endemic regions</span>,” <abbr class="cit-jnl-abbrev">In Silico Biol</abbr>, vol. <span class="cit-vol">10</span>, pp. <span class="cit-fpage">235</span>–<span class="cit-lpage">46</span>, <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2Bsilico%2Bbiology%26rft.stitle%253DIn%2BSilico%2BBiol%26rft.aulast%253DChakraborty%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D235%26rft.epage%253D246%26rft.atitle%253DA%2Bcomputational%2Bapproach%2Bfor%2Bidentification%2Bof%2Bepitopes%2Bin%2Bdengue%2Bvirus%2Benvelope%2Bprotein%253A%2Ba%2Bstep%2Btowards%2Bdesigning%2Ba%2Buniversal%2Bdengue%2Bvaccine%2Btargeting%2Bendemic%2Bregions.%26rft_id%253Dinfo%253Apmid%252F22430357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22430357&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.67"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tahir Ul Qamar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Adeel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Maryam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U. A.</span> <span class="cit-name-surname">Ashfaq</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Peptide vaccine against chikungunya virus: immuno-informatics combined with molecular docking approach</span>,” <abbr class="cit-jnl-abbrev">J TranslMed</abbr>, vol. <span class="cit-vol">16</span>, p. <span class="cit-fpage">298</span>, <span class="cit-month">Oct</span> <span class="cit-day">27</span> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTranslMed%26rft.volume%253D16%26rft.spage%253D298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.68" data-doi="10.1093/molbev/msg010"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Twiddy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rambaut</span></span>, “<span class="cit-article-title">Inferring the rate and time-scale of dengue virus evolution</span>,” <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, vol. <span class="cit-vol">20</span>, pp. <span class="cit-fpage">122</span>–<span class="cit-lpage">9</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DTwiddy%26rft.auinit1%253DS.%2BS.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D122%26rft.epage%253D129%26rft.atitle%253DInferring%2Bthe%2BRate%2Band%2BTime-Scale%2Bof%2BDengue%2BVirus%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg010%26rft_id%253Dinfo%253Apmid%252F12519914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12519914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180243700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.69" data-doi="10.1002/hep.510300403"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Christie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chapel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Chapman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Rosenberg</span></span>, “<span class="cit-article-title">Immune selection and genetic sequence variation in core and envelope regions of hepatitis C virus</span>,” <abbr class="cit-jnl-abbrev">Hepatology</abbr>, vol. <span class="cit-vol">30</span>, pp. <span class="cit-fpage">1037</span>–<span class="cit-lpage">44</span>, <span class="cit-month">Oct</span> <span class="cit-pub-date">1999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHepatology%2B%2528Baltimore%252C%2BMd.%2529%26rft.stitle%253DHepatology%26rft.aulast%253DChristie%26rft.auinit1%253DJ.%2BM.%2BL.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D1037%26rft.epage%253D1044%26rft.atitle%253DImmune%2BSelection%2Band%2BGenetic%2BSequence%2BVariation%2Bin%2BCore%2Band%2BEnvelope%2BRegions%2Bof%2BHepatitis%2BC%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhep.510300403%26rft_id%253Dinfo%253Apmid%252F10498657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/hep.510300403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10498657&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082784600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.70"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Hardison</span></span>, “<span class="cit-article-title">Comparative genomics</span>,” <abbr class="cit-jnl-abbrev">PLoSBiol</abbr>, vol. <span class="cit-vol">1</span>, p. <span class="cit-fpage">E58</span>, <span class="cit-month">Nov</span> <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoSBiol%26rft.volume%253D1%26rft.spage%253D58E%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.71" data-doi="10.1038/ncomms4348"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gilbert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chateigner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ernenwein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Barbe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bezier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Herniou</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Population genomics supports baculoviruses as vectors of horizontal transfer of insect transposons</span>,” <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, vol. <span class="cit-vol">5</span>, p. <span class="cit-fpage">3348</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D5%26rft.spage%253D3348%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4348%26rft_id%253Dinfo%253Apmid%252F24556639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24556639&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.72" data-doi="10.1186/s12985-019-1182-0"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schoeman</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Fielding</span></span>, “<span class="cit-article-title">Coronavirus envelope protein: current knowledge</span>,” <abbr class="cit-jnl-abbrev">Virol J</abbr>, vol. <span class="cit-vol">16</span>, p. <span class="cit-fpage">69</span>, <span class="cit-month">May</span> <span class="cit-day">27</span> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D16%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-019-1182-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12985-019-1182-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.73" data-doi="10.1016/j.febslet.2004.09.009"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Mortola</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Roy</span></span>, “<span class="cit-article-title">Efficient assembly and release of SARS coronavirus-like particles by a heterologous expression system</span>,” <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>, vol. <span class="cit-vol">576</span>, pp. <span class="cit-fpage">174</span>–<span class="cit-lpage">8</span>, <span class="cit-month">Oct</span> <span class="cit-day">8</span> <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DMortola%26rft.auinit1%253DE.%26rft.volume%253D576%26rft.issue%253D1-2%26rft.spage%253D174%26rft.epage%253D178%26rft.atitle%253DEfficient%2Bassembly%2Band%2Brelease%2Bof%2BSARS%2Bcoronavirus-like%2Bparticles%2Bby%2Ba%2Bheterologous%2Bexpression%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2004.09.009%26rft_id%253Dinfo%253Apmid%252F15474033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2004.09.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15474033&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.74"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Sharmin</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Islam</span></span>, “<span class="cit-article-title">Conserved antigenic sites between MERS-CoV and Batcoronavirus are revealed through sequence analysis</span>,” <abbr class="cit-jnl-abbrev">Source Code Biol Med</abbr>, vol. <span class="cit-vol">11</span>, p. <span class="cit-fpage">3</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSource%2BCode%2BBiol%2BMed%26rft.volume%253D11%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.75"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wen</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Genome organization of the SARS-CoV</span>,” <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr>, vol. <span class="cit-vol">1</span>, pp. <span class="cit-fpage">226</span>–<span class="cit-lpage">35</span>, <span class="cit-month">Aug</span> <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%252C%2Bproteomics%2B%2526%2Bbioinformatics%2B%252F%2BBeijing%2BGenomics%2BInstitute%26rft.stitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.aulast%253DXu%26rft.auinit1%253DJ.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D235%26rft.atitle%253DGenome%2Borganization%2Bof%2Bthe%2BSARS-CoV.%26rft_id%253Dinfo%253Apmid%252F15629035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15629035&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trent</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tirrell</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olive</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Advances in the design and delivery of peptide subunit vaccines with a focus on toll-like receptor agonists</span>. <abbr class="cit-jnl-abbrev">Expert Rev Vaccines</abbr> <span class="cit-pub-date">2010</span>; <span class="cit-vol">9</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2Breview%2Bof%2Bvaccines%26rft.stitle%253DExpert%2BRev%2BVaccines%26rft.aulast%253DBlack%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D173%26rft.atitle%253DAdvances%2Bin%2Bthe%2Bdesign%2Band%2Bdelivery%2Bof%2Bpeptide%2Bsubunit%2Bvaccines%2Bwith%2Ba%2Bfocus%2Bon%2Btoll-like%2Breceptor%2Bagonists.%26rft_id%253Dinfo%253Apmid%252F20109027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20109027&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.77" data-doi="10.1099/00222615-39-4-241"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sesardic</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Synthetic peptide vaccines</span>. <abbr class="cit-jnl-abbrev">J Med Microbiol</abbr> <span class="cit-pub-date">1993</span>; <span class="cit-vol">39</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BMicrobiology%26rft.stitle%253DJ%2BMed%2BMicrobiol%26rft.aulast%253DSesardic%26rft.auinit1%253DD.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D241%26rft.epage%253D242%26rft.atitle%253DSynthetic%2Bpeptide%2Bvaccines%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00222615-39-4-241%26rft_id%253Dinfo%253Apmid%252F7692052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00222615-39-4-241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7692052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MB94400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.78"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Breakthrough</span>: <abbr class="cit-jnl-abbrev">Chloroquine phosphate has shown apparentefficacy in treatment of COVID-19 associated pneumonia inclinical studies</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.79"><div class="cit-metadata"><cite><abbr class="cit-jnl-abbrev">Potential Inhibitors Against Papain-like Protease of Novel Coronavirus (COVID-19) from FDA Approved Drugs</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.80" data-doi="10.1128/JVI.01237-15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Westerbeck</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Machamer</span></span>, “<span class="cit-article-title">A Coronavirus E Protein Is Present in Two Distinct Pools with Different Effects on Assembly and the Secretory Pathway</span>,” <abbr class="cit-jnl-abbrev">J Virol</abbr>, vol. <span class="cit-vol">89</span>, pp. <span class="cit-fpage">9313</span>–<span class="cit-lpage">23</span>, <span class="cit-month">Sep</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01237-15%26rft_id%253Dinfo%253Apmid%252F26136577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xOC85MzEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4wNC45MzQyMzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">81.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.81" data-doi="10.1016/j.virol.2008.03.005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Dediego</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pewe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alvarez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Rejas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Perlman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Enjuanes</span></span>, “<span class="cit-article-title">Pathogenicity of severe acute respiratory coronavirus deletion mutants in hACE-2 transgenic mice</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">376</span>, pp. <span class="cit-fpage">379</span>–<span class="cit-lpage">89</span>, <span class="cit-month">Jul</span> <span class="cit-day">5</span> <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DDediego%26rft.auinit1%253DM.%2BL.%26rft.volume%253D376%26rft.issue%253D2%26rft.spage%253D379%26rft.epage%253D389%26rft.atitle%253DPathogenicity%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bcoronavirus%2Bdeletion%2Bmutants%2Bin%2BhACE-2%2Btransgenic%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2008.03.005%26rft_id%253Dinfo%253Apmid%252F18452964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2008.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18452964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">82.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.82" data-doi="10.1016/j.virol.2010.01.004"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Netland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">DeDiego</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alvarez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Nieto-Torres</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Immunization with an attenuated severe acute respiratory syndrome coronavirus deleted in E protein protects against lethal respiratory disease</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">399</span>, pp. <span class="cit-fpage">120</span>–<span class="cit-lpage">128</span>, <span class="cit-month">Mar</span> <span class="cit-day">30</span> <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D399%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.01.004%26rft_id%253Dinfo%253Apmid%252F20110095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.04.934232.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">83.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.83" data-doi="10.1128/JVI.02012-06"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">McCray</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pewe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wohlford-Lenane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hickey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Manzel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Lethal infection of K18-hACE2 mice infected with severe acute respiratory syndrome coronavirus</span>,” <abbr class="cit-jnl-abbrev">J Virol</abbr>, vol. <span class="cit-vol">81</span>, pp. <span class="cit-fpage">813</span>–<span class="cit-lpage">21</span>, <span class="cit-month">Jan</span> <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02012-06%26rft_id%253Dinfo%253Apmid%252F17079315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzIvODEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4wNC45MzQyMzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">84.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.84" data-doi="10.1128/JVI.03566-14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Regla-Nava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Nieto-Torres</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Jimenez-Guardeno</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernandez-Delgado</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Castano-Rodriguez</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Severe acute respiratory syndrome coronaviruses with mutations in the E protein are attenuated and promising vaccine candidates</span>,” <abbr class="cit-jnl-abbrev">J Virol</abbr>,vol. <span class="cit-vol">89</span>, pp. <span class="cit-fpage">3870</span>–<span class="cit-lpage">87</span>, <span class="cit-month">Apr</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03566-14%26rft_id%253Dinfo%253Apmid%252F25609816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzcvMzg3MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMDQuOTM0MjMyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">85.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.04.934232v2.85" data-doi="10.1073/pnas.1323279111"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wohlford-Lenane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Agnihothram</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Rapid generation of a mouse model for Middle East respiratory syndrome</span>,” <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>, vol. <span class="cit-vol">111</span>, pp. <span class="cit-fpage">4970</span>–<span class="cit-lpage">5</span>, <span class="cit-month">Apr</span> <span class="cit-day">1</span> <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323279111%26rft_id%253Dinfo%253Apmid%252F24599590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzEzLzQ5NzAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8wMi8yMDIwLjAyLjA0LjkzNDIzMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.04.934232v2.86"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Deng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zou</span></span>, <span class="cit-etal">et al.</span>,“<span class="cit-article-title">Systemic and mucosal immunity in mice elicited by a single immunization with human adenovirus type 5 or 41 vector-based vaccines carrying the spike protein of Middle East respiratory syndrome coronavirus</span>,” <abbr class="cit-jnl-abbrev">Immunology</abbr>, vol. <span class="cit-vol">145</span>, pp. <span class="cit-fpage">476</span>–<span class="cit-lpage">84</span>, <span class="cit-month">Aug</span> <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D145%26rft.spage%253D476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.04.934232v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.26.966754v2" title="A systematic, label-free method for identifying RNA-associated proteins in vivo provides insights into vertebrate ciliary beating" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/848275v2" title="Toxin-antitoxin gene pairs found in Tn3 family transposons appear to be an integral part of the transposition module" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 02, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.04.934232v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.04.934232v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.04.934232v2" class="active" data-icon-position="" data-hide-link-title="0">Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1177442" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1ZBTzrez_gKHBl05E2pvsYZpNB8iS_on1RsPhAGS3qY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177442" id="node1177442" data-pisa="biorxiv;2020.02.04.934232v2" data-pisa-master="biorxiv;2020.02.04.934232" data-seqnum="1177442" data-apath="/biorxiv/early/2020/03/02/2020.02.04.934232.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Miyssa I.</span> <span class="nlm-surname">Abdelmageed</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Abdelrahman H.</span> <span class="nlm-surname">Abdelmoneim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Mujahed I.</span> <span class="nlm-surname">Mustafa</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nafisa M.</span> <span class="nlm-surname">Elfadol</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Naseem S.</span> <span class="nlm-surname">Murshed</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Shaza W.</span> <span class="nlm-surname">Shantier</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abdelrafie M.</span> <span class="nlm-surname">Makhawi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.04.934232; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.04.934232 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;title=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;title=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;text=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;title=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;t=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;title=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;title=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177442" id="node1177442--2" data-pisa="biorxiv;2020.02.04.934232v2" data-pisa-master="biorxiv;2020.02.04.934232" data-seqnum="1177442" data-apath="/biorxiv/early/2020/03/02/2020.02.04.934232.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Design of multi epitope-based peptide vaccine against E protein of human COVID-19: An immunoinformatics approach</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Miyssa I.</span> <span class="nlm-surname">Abdelmageed</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Abdelrahman H.</span> <span class="nlm-surname">Abdelmoneim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Mujahed I.</span> <span class="nlm-surname">Mustafa</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nafisa M.</span> <span class="nlm-surname">Elfadol</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Naseem S.</span> <span class="nlm-surname">Murshed</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Shaza W.</span> <span class="nlm-surname">Shantier</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Abdelrafie M.</span> <span class="nlm-surname">Makhawi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.04.934232; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.04.934232 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Abdelmageed%2C+Miyssa+I.&amp;rft.au=Abdelmoneim%2C+Abdelrahman+H.&amp;rft.au=Mustafa%2C+Mujahed+I.&amp;rft.au=Elfadol%2C+Nafisa+M.&amp;rft.au=Murshed%2C+Naseem+S.&amp;rft.au=Shantier%2C+Shaza+W.&amp;rft.au=Makhawi%2C+Abdelrafie+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1177442/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1177442/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1177442/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1177442/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1177442/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1177442/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1177442/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1177442/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1177442/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1177442/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1177442/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1177442/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;count=horizontal&amp;via=&amp;text=Design%20of%20multi%20epitope-based%20peptide%20vaccine%20against%20E%20protein%20of%20human%20COVID-19%3A%20An%20immunoinformatics%20approach&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.04.934232v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.04.934232v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
