<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/19/2019.12.18.880849.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.18.880849v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications" />
<meta name="DC.Identifier" content="10.1101/2019.12.18.880849" />
<meta name="DC.Date" content="2019-12-19" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial ( E. coli ) and archaeal ( H. volcanii, P. furiosus ) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism." />
<meta name="DC.Contributor" content="Felix Grünberger" />
<meta name="DC.Contributor" content="Robert Knüppel" />
<meta name="DC.Contributor" content="Michael Jüttner" />
<meta name="DC.Contributor" content="Martin Fenk" />
<meta name="DC.Contributor" content="Andreas Borst" />
<meta name="DC.Contributor" content="Robert Reichelt" />
<meta name="DC.Contributor" content="Winfried Hausner" />
<meta name="DC.Contributor" content="Jörg Soppa" />
<meta name="DC.Contributor" content="Sebastien Ferreira-Cerca" />
<meta name="DC.Contributor" content="Dina Grohmann" />
<meta name="article:published_time" content="2019-12-19" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (&lt;i&gt;E. coli&lt;/i&gt;) and archaeal (&lt;i&gt;H. volcanii, P. furiosus&lt;/i&gt;) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.18.880849v1" />
<meta name="citation_id" content="2019.12.18.880849v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.18.880849" />
<meta name="citation_num_pages" content="40" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.18.880849" />
<meta name="citation_author" content="Felix Grünberger" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &amp; Biochemistry Centre Regensburg, University of Regensburg" />
<meta name="citation_author" content="Robert Knüppel" />
<meta name="citation_author_institution" content="Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg" />
<meta name="citation_author" content="Michael Jüttner" />
<meta name="citation_author_institution" content="Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg" />
<meta name="citation_author" content="Martin Fenk" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &amp; Biochemistry Centre Regensburg, University of Regensburg" />
<meta name="citation_author" content="Andreas Borst" />
<meta name="citation_author_institution" content="Goethe-University, Biocentre, Institute for Molecular Biosciences" />
<meta name="citation_author" content="Robert Reichelt" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &amp; Biochemistry Centre Regensburg, University of Regensburg" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3455-1495" />
<meta name="citation_author" content="Winfried Hausner" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &amp; Biochemistry Centre Regensburg, University of Regensburg" />
<meta name="citation_author" content="Jörg Soppa" />
<meta name="citation_author_institution" content="Goethe-University, Biocentre, Institute for Molecular Biosciences" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6132-5840" />
<meta name="citation_author" content="Sebastien Ferreira-Cerca" />
<meta name="citation_author_institution" content="Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg" />
<meta name="citation_author_email" content="sebastien.ferreira-cerca@ur.de" />
<meta name="citation_author_email" content="dina.grohmann@ur.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0522-843X" />
<meta name="citation_author" content="Dina Grohmann" />
<meta name="citation_author_institution" content="Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &amp; Biochemistry Centre Regensburg, University of Regensburg" />
<meta name="citation_author_email" content="sebastien.ferreira-cerca@ur.de" />
<meta name="citation_author_email" content="dina.grohmann@ur.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0570-2517" />
<meta name="citation_reference" content="Levy, S. E. &amp; Myers, R. M. Advancements in Next-Generation Sequencing. Annual review of genomics and human genetics 17, 95–115; 10.1146/annurev-genom-083115-022413 (2016)." />
<meta name="citation_reference" content="Escobar-Zepeda, A., Vera-Ponce de León, A. &amp; Sanchez-Flores, A. The Road to Metagenomics: From Microbiology to DNA Sequencing Technologies and Bioinformatics. Frontiers in Genetics 6, 348; 10.3389/fgene.2015.00348 (2015)." />
<meta name="citation_reference" content="Wang, Z., Gerstein, M. &amp; Snyder, M. RNA-Seq: a revolutionary tool for transcriptomics. Nature reviews. Genetics 10, 57–63; 10.1038/nrg2484 (2009)." />
<meta name="citation_reference" content="Hör, J., Gorski, S. A. &amp; Vogel, J. Bacterial RNA Biology on a Genome Scale. Molecular cell 70, 785–799; 10.1016/j.molcel.2017.12.023 (2018)." />
<meta name="citation_reference" content="Croucher, N. J. &amp; Thomson, N. R. Studying bacterial transcriptomes using RNA-seq. Current opinion in microbiology 13, 619–624; 10.1016/j.mib.2010.09.009 (2010)." />
<meta name="citation_reference" content="Nowrousian, M. Next-generation sequencing techniques for eukaryotic microorganisms: sequencing-based solutions to biological problems. Eukaryotic cell 9, 1300–1310; 10.1128/EC.00123-10 (2010)." />
<meta name="citation_reference" content="Saliba, A.-E., C Santos, S. &amp; Vogel, J. New RNA-seq approaches for the study of bacterial pathogens. Current opinion in microbiology 35, 78–87; 10.1016/j.mib.2017.01.001 (2017)." />
<meta name="citation_reference" content="Stark, R., Grzelak, M. &amp; Hadfield, J. RNA sequencing: the teenage years. Nature Reviews Genetics; 10.1038/s41576-019-0150-2 (2019)." />
<meta name="citation_reference" content="Tilgner, H. et al. Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events. Nature Biotechnology 33, 736–742; 10.1038/nbt.3242 (2015)." />
<meta name="citation_reference" content="Byrne, A., Cole, C., Volden, R. &amp; Vollmers, C. Realizing the potential of full-length transcriptome sequencing. Philosophical transactions of the Royal Society of London. Series B, Biological sciences 374, 20190097; 10.1098/rstb.2019.0097 (2019)." />
<meta name="citation_reference" content="Eid, J. et al. Real-time DNA sequencing from single polymerase molecules. Science (New York, N.Y.) 323, 133–138; 10.1126/science.1162986 (2009)." />
<meta name="citation_reference" content="Mikheyev, A. S. &amp; Tin, M. M. Y. A first look at the Oxford Nanopore MinION sequencer. Molecular ecology resources 14, 1097–1102; 10.1111/1755-0998.12324 (2014)." />
<meta name="citation_reference" content="Soneson, C. et al. A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes. Nature Communications 10, 1–14; 10.1038/s41467-019-11272-z (2019)." />
<meta name="citation_reference" content="Liu, H. et al. Accurate detection of m6A RNA modifications in native RNA sequences. Nature Communications, 1–9; 10.1101/525741 (2019)." />
<meta name="citation_reference" content="Smith, A. M., Jain, M., Mulroney, L., Garalde, D. R. &amp; Akeson, M. Reading canonical and modified nucleobases in 16S ribosomal RNA using nanopore native RNA sequencing. PLOS ONE 14, 1–15; 10.1371/journal.pone.0216709 (2019)." />
<meta name="citation_reference" content="Workman, R. E. et al. Nanopore native RNA sequencing of a human poly(A) transcriptome. Nature Methods 16, 1297–1305; 10.1038/s41592-019-0617-2 (2019)." />
<meta name="citation_reference" content="Viehweger A, Krautwurst S, Lamkiewicz K, Madhugiri R, Ziebuhr J, Hölzer M, Marz M. Nanopore direct RNA sequencing reveals modification in full-length coronavirus genomes. bioRxiv Genomics, 1–15; 10.1101/483693 (2018)." />
<meta name="citation_reference" content="Keller, M. W. et al. Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome. Scientific Reports 8, 1–8; 10.1038/s41598-018-32615-8 (2018)." />
<meta name="citation_reference" content="Boldogkői, Z., Moldován, N., Balázs, Z., Snyder, M. &amp; Tombácz, D. Long-Read Sequencing – A Powerful Tool in Viral Transcriptome Research. Trends in Microbiology 27, 578–592; 10.1016/j.tim.2019.01.010 (2019)." />
<meta name="citation_reference" content="Viehweger, A. et al. Direct RNA nanopore sequencing of full-length coron-avirus genomes provides novel insights into structural variants and enables modification analysis. bioRxiv, 483693; 10.1101/483693 (2019)." />
<meta name="citation_reference" content="Tombácz, D. et al. Multiple Long-Read Sequencing Survey of Herpes Simplex Virus Dynamic Transcriptome. Frontiers in Genetics 10, 834; 10.3389/fgene.2019.00834 (2019)." />
<meta name="citation_reference" content="Zhao, L. et al. Analysis of Transcriptome and Epitranscriptome in Plants Using PacBio Iso-Seq and Nanopore-Based Direct RNA Sequencing. Frontiers in Genetics 10, 253; 10.3389/fgene.2019.00253 (2019)." />
<meta name="citation_reference" content="Bayega, A. et al. Transcriptome landscape of the developing olive fruit fly embryo delineated by Oxford Nanopore long-read RNA-Seq (2018)." />
<meta name="citation_reference" content="Byrne, A. et al. Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells. Nature Communications 8; 10.1038/ncomms16027 (2017)." />
<meta name="citation_reference" content="Rahimi, K., Venø, M. T., Dupont, D. M. &amp; Kjems, J. Nanopore sequencing of full-length circRNAs in human and mouse brains reveals circRNA-specific exon usage and intron retention (2019)." />
<meta name="citation_reference" content="Dar, D. &amp; Sorek, R. High-resolution RNA 3-ends mapping of bacterial Rho-dependent transcripts. Nucleic Acids Research 46, 6797–6805; 10.1093/nar/gky274 (2018)." />
<meta name="citation_reference" content="Babski, J. et al. Genome-wide identification of transcriptional start sites in the haloarchaeon Haloferax volcanii based on differential RNA-Seq (dRNA-Seq). BMC Genomics 17, 1–19; 10.1186/s12864-016-2920-y (2016)." />
<meta name="citation_reference" content="Laass, S. et al. Characterization of the transcriptome of Haloferax volcanii, grown under four different conditions, with mixed RNA-Seq. PLOS ONE 14, 1–24; 10.1371/journal.pone.0215986 (2019)." />
<meta name="citation_reference" content="Mao, X. et al. Revisiting operons: An analysis of the landscape of transcriptional units in E. coli. BMC Bioinformatics; 10.1186/s12859-015-0805-8 (2015)." />
<meta name="citation_reference" content="Thomason, M. K. et al. Global transcriptional start site mapping using differential RNA sequencing reveals novel antisense RNAs in Escherichia coli. Journal of Bacteriology 197, 18–28; 10.1128/JB.02096-14 (2015)." />
<meta name="citation_reference" content="Grünberger, F. et al. Next Generation DNA-Seq and Differential RNA-Seq Allow Re-annotation of the Pyrococcus furiosus DSM 3638 Genome and Provide Insights Into Archaeal Antisense Transcription. Frontiers in Microbiology; 10.3389/fmicb.2019.01603 (2019)." />
<meta name="citation_reference" content="Stetter, K. O., König, H. &amp; Stackebrandt, E. Pyrodictium gen. nov., a New Genus of Submarine Disc-Shaped Sulphur Reducing Archaebacteria Growing Optimally at 105°C. Systematic and Applied Microbiology; 10.1016/S0723-2020(83)80011-3 (1983)." />
<meta name="citation_reference" content="Allers, T. &amp; Mevarech, M. Archaeal genetics - the third way. Nature Reviews Genetics 6, 58–73; 10.1038/nrg1504 (2005)." />
<meta name="citation_reference" content="Knüppel, R. et al. Insights into the evolutionary conserved regulation of Rio ATPase activity. Nucleic Acids Research 46, 1441–1456; 10.1093/nar/gkx1236 (2018)." />
<meta name="citation_reference" content="Chomczynski, P. &amp; Sacchi, N. Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Analytical biochemistry 162, 156–159; 10.1006/abio.1987.9999 (1987)." />
<meta name="citation_reference" content="Yan, B., Boitano, M., Clark, T. A. &amp; Ettwiller, L. SMRT-Cappable-seq reveals complex operon variants in bacteria. Nature Communications; 10.1038/s41467-018-05997-6 (2018)." />
<meta name="citation_reference" content="Weirather, J. L. et al. Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis. F1000Research 6, 100; 10.12688/f1000research.10571.2 (2017)." />
<meta name="citation_reference" content="Riley, M. et al. Escherichia coli K-12: a cooperatively developed annotation snapshot--2005. Nucleic Acids Research 34, 1–9; 10.1093/nar/gkj405 (2006)." />
<meta name="citation_reference" content="Hartman, A. L. et al. The complete genome sequence of Haloferax volcanii DS2, a model archaeon. PLOS ONE 5, e9605; 10.1371/journal.pone.0009605 (2010)." />
<meta name="citation_reference" content="Li, H. Minimap2: Pairwise alignment for nucleotide sequences. Bioinformatics; 10.1093/bioinformatics/bty191 (2018)." />
<meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079; 10.1093/bioinformatics/btp352 (2009)." />
<meta name="citation_reference" content="Liao, Y., Smyth, G. K. &amp; Shi, W. The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads. Nucleic Acids Research; 10.1093/nar/gkz114 (2019)." />
<meta name="citation_reference" content="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biology 15, 550; 10.1186/s13059-014-0550-8 (2014)." />
<meta name="citation_reference" content="Loman, N. J., Quick, J. &amp; Simpson, J. T. A complete bacterial genome assembled de novo using only nanopore sequencing data. Nature Methods 12, 733–735; 10.1038/nmeth.3444 (2015)." />
<meta name="citation_reference" content="Bailey, T. L. et al. MEME SUITE: tools for motif discovery and searching. Nucleic Acids Research 37, W202–8; 10.1093/nar/gkp335 (2009)." />
<meta name="citation_reference" content="Dar, D., Prasse, D., Schmitz, R. A. &amp; Sorek, R. Widespread formation of alternative 3’ UTR isoforms via transcription termination in archaea. Nature Microbiology 1, 16143; 10.1038/nmicrobiol.2016.143 (2016)." />
<meta name="citation_reference" content="Stoiber, M. et al. De novo Identification of DNA Modifications Enabled by Genome-Guided Nanopore Signal Processing (2016)." />
<meta name="citation_reference" content="Boccaletto, P. et al. MODOMICS: a database of RNA modification pathways. 2017 update. Nucleic Acids Research 46, D303–D307; 10.1093/nar/gkx1030 (2018)." />
<meta name="citation_reference" content="Grosjean, H., Gaspin, C., Marck, C., Decatur, W. A. &amp; Crécy-Lagard, V. de. RNomics and Modomics in the halophilic archaea Haloferax volcanii: Identification of RNA modification genes. BMC Genomics 9, 1–26; 10.1186/1471-2164-9-470 (2008)." />
<meta name="citation_reference" content="Mao, X. et al. DOOR 2.0: Presenting operons and their functions through dynamic and integrated views. Nucleic Acids Research 42; 10.1093/nar/gkt1048 (2014)." />
<meta name="citation_reference" content="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120; 10.1093/bioinformatics/btu170 (2014)." />
<meta name="citation_reference" content="Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nature Methods 9, 357–359; 10.1038/nmeth.1923 (2012)." />
<meta name="citation_reference" content="Bohlin, J., Eldholm, V., Pettersson, J. H. O., Brynildsrud, O. &amp; Snipen, L. The nucleotide composition of microbial genomes indicates differential patterns of selection on core and accessory genomes. BMC Genomics 18, 151; 10.1186/s12864-017-3543-7 (2017)." />
<meta name="citation_reference" content="Wick, R. R., Judd, L. M. &amp; Holt, K. E. Performance of neural network basecalling tools for Oxford Nanopore sequencing. Genome Biology 20, 1–10; 10.1186/s13059-019-1727-y (2019)." />
<meta name="citation_reference" content="Garalde, D. R. et al. Highly parallel direct RNA sequencing on an array of nanopores. Nature Methods 15, 201–206; 10.1038/nmeth.4577 (2018)." />
<meta name="citation_reference" content="Workman, R. E. et al. Nanopore native RNA sequencing of a human poly(A) transcriptome. bioRxiv; 10.1101/459529 (2018)." />
<meta name="citation_reference" content="Parker, M. T. et al. Nanopore direct RNA sequencing maps an Arabidopsis N6 methyladenosine epitranscriptome. bioRxiv; 10.1101/706002 (2019)." />
<meta name="citation_reference" content="Ju, X., Li, D. &amp; Liu, S. Full-length RNA profiling reveals pervasive bidirectional transcription terminators in bacteria. Nature Microbiology; 10.1038/s41564-019-0500-z (2019)." />
<meta name="citation_reference" content="Santangelo, T. J., Cubonová, L.’u., Skinner, K. M. &amp; Reeve, J. N. Archaeal intrinsic transcription termination in vivo. Journal of Bacteriology 191, 7102–7108; 10.1128/JB.00982-09 (2009)." />
<meta name="citation_reference" content="Smith, A. M. et al. Reading canonical and modified nucleotides in 16S ribosomal RNA using nanopore direct RNA sequencing. bioRxiv; 10.1101/132274 (2017)." />
<meta name="citation_reference" content="Wongsurawat, T. et al. Rapid sequencing of multiple RNA viruses in their native form. Frontiers in Microbiology 10, 1–8; 10.3389/fmicb.2019.00260 (2019)." />
<meta name="citation_reference" content="Näther-Schindler, D. J., Schopf, S., Bellack, A., Rachel, R. &amp; Wirth, R. Pyrococcus furiosus flagella: biochemical and transcriptional analyses identify the newly detected flaB0 gene to encode the major flagellin. Frontiers in Microbiology 5, 695; 10.3389/fmicb.2014.00695 (2014)." />
<meta name="citation_reference" content="Lioliou, E. et al. Global regulatory functions of the Staphylococcus aureus endoribonuclease III in gene expression. PLoS Genetics 8, e1002782; 10.1371/journal.pgen.1002782 (2012)." />
<meta name="citation_reference" content="Smith, B. A., Gupta, N., Denny, K. &amp; Culver, G. M. Characterization of 16S rRNA Processing with Pre-30S Subunit Assembly Intermediates from E. coli. Journal of Molecular Biology; 10.1016/j.jmb.2018.04.009 (2018)." />
<meta name="citation_reference" content="Shajani, Z., Sykes, M. T. &amp; Williamson, J. R. Assembly of bacterial ribosomes. Annual review of biochemistry 80, 501–526; 10.1146/annurev-biochem-062608-160432 (2011)." />
<meta name="citation_reference" content="Bechhofer, D. H. &amp; Deutscher, M. P. Bacterial ribonucleases and their roles in RNA metabolism. Critical reviews in biochemistry and molecular biology 54, 242–300; 10.1080/10409238.2019.1651816 (2019)." />
<meta name="citation_reference" content="Klappenbach, J. A., Saxman, P. R., Cole, J. R. &amp; Schmidt, T. M. rrndb: the Ribosomal RNA Operon Copy Number Database. Nucleic Acids Research 29, 181–184; 10.1093/nar/29.1.181 (2001)." />
<meta name="citation_reference" content="Maeda, M., Shimada, T., Ishihama, A. &amp; Semsey, S. Strength and Regulation of Seven rRNA Promoters in Escherichia coli. PLOS ONE 10, e0144697; 10.1371/journal.pone.0144697 (2015)." />
<meta name="citation_reference" content="Hofmann, S. &amp; Miller, O. L. Visualization of ribosomal ribonucleic acid synthesis in a ribonuclease III-Deficient strain of Escherichia coli. Journal of Bacteriology 132, 718–722 (1977)." />
<meta name="citation_reference" content="Jacob, A. I., Köhrer, C., Davies, B. W., RajBhandary, U. L. &amp; Walker, G. C. Conserved bacterial RNase YbeY plays key roles in 70S ribosome quality control and 16S rRNA maturation. Molecular cell 49, 427–438; 10.1016/j.molcel.2012.11.025 (2013)." />
<meta name="citation_reference" content="Sulthana, S. &amp; Deutscher, M. P. Multiple exoribonucleases catalyze maturation of the 3’ terminus of 16S ribosomal RNA (rRNA). The Journal of biological chemistry 288, 12574–12579; 10.1074/jbc.C113.459172 (2013)." />
<meta name="citation_reference" content="Yip, W. S. V., Vincent, N. G. &amp; Baserga, S. J. Ribonucleoproteins in archaeal pre-rRNA processing and modification. Archaea (Vancouver, B.C.) 2013, 614735; 10.1155/2013/614735 (2013)." />
<meta name="citation_reference" content="citation_title=RNA Metabolism and Gene Expression in Archaea;citation_pages=129-158;citation_year=2017" />
<meta name="citation_reference" content="Henras, A. K., Plisson-Chastang, C., O’Donohue, M.-F., Chakraborty, A. &amp; Gleizes, P.-E. An overview of pre-ribosomal RNA processing in eukaryotes. Wiley interdisciplinary reviews. RNA 6, 225–242; 10.1002/wrna.1269 (2015)." />
<meta name="citation_reference" content="Clouet-D’Orval, B. et al. Insights into RNA-processing pathways and associated RNA-degrading enzymes in Archaea. FEMS Microbiology Reviews 42, 579–613; 10.1093/femsre/fuy016 (2018)." />
<meta name="citation_reference" content="Tang, T. H. et al. RNomics in Archaea reveals a further link between splicing of archaeal introns and rRNA processing. Nucleic Acids Research 30, 921–930; 10.1093/nar/30.4.921 (2002)." />
<meta name="citation_reference" content="Russell, A. G., Ebhardt, H. &amp; Dennis, P. P. Substrate requirements for a novel archaeal endonuclease that cleaves within the 5’ external transcribed spacer of Sulfolobus acidocaldarius precursor rRNA. Genetics 152, 1373–1385 (1999)." />
<meta name="citation_reference" content="Jüttner, M. et al. A versatile cis-acting element reporter system to study the function, maturation and stability of ribosomal RNA mutants in archaea. Nucleic Acids Research; 10.1093/nar/gkz1156 (2019)." />
<meta name="citation_reference" content="Danan, M., Schwartz, S., Edelheit, S. &amp; Sorek, R. Transcriptome-wide discovery of circular RNAs in Archaea. Nucleic Acids Research 40, 3131–3142; 10.1093/nar/gkr1009 (2012)." />
<meta name="citation_reference" content="Ferreira-Cerca, S. RNA Metabolism and Gene Expression in Archaea - Chapter6 - Life and Death of Ribosomes in Archaea (2017)." />
<meta name="citation_reference" content="Wongsurawat, T. et al. Decoding the Epitranscriptional Landscape from Native RNA Sequences. bioRxiv 17, 487819; 10.1101/487819 (2018)." />
<meta name="citation_reference" content="O’Farrell, H. C., Pulicherla, N., Desai, P. M. &amp; Rife, J. P. Recognition of a complex substrate by the KsgA/Dim1 family of enzymes has been conserved throughout evolution. RNA (New York, N.Y.) 12, 725–733; 10.1261/rna.2310406 (2006)." />
<meta name="citation_reference" content="Rang, F. J., Kloosterman, W. P. &amp; Ridder, J. de. From squiggle to basepair: Computational approaches for improving nanopore sequencing read accuracy. Genome Biology 19, 1–11; 10.1186/s13059-018-1462-9 (2018)." />
<meta name="citation_reference" content="Strunk, B. S. et al. Ribosome assembly factors prevent premature translation initiation by 40S assembly intermediates. Science (New York, N.Y.) 333, 1449–1453; 10.1126/science.1208245 (2011)." />
<meta name="citation_reference" content="Xu, Z., O’Farrell, H. C., Rife, J. P. &amp; Culver, G. M. A conserved rRNA methyltransferase regulates ribosome biogenesis. Nature structural &amp; molecular biology 15, 534–536; 10.1038/nsmb.1408 (2008)." />
<meta name="citation_reference" content="Lafontaine, D. L., Preiss, T. &amp; Tollervey, D. Yeast 18S rRNA dimethylase Dim1p: a quality control mechanism in ribosome synthesis? Molecular and cellular biology 18, 2360–2370; 10.1128/mcb.18.4.2360 (1998)." />
<meta name="citation_reference" content="Dennis, P. P., Tripp, V., Lui, L., Lowe, T. &amp; Randau, L. C/D box sRNA-guided 2’-O-methylation patterns of archaeal rRNA molecules. BMC Genomics; 10.1186/s12864-015-1839-z (2015)." />
<meta name="citation_reference" content="Gomes-Filho, J. V. &amp; Randau, L. RNA stabilization in hyperthermophilic archaea. Annals of the New York Academy of Sciences 1447, 88–96; 10.1111/nyas.14060 (2019)." />
<meta name="citation_reference" content="Ebright, R. H., Werner, F. &amp; Zhang, X. RNA Polymerase Reaches 60: Transcription Initiation, Elongation, Termination, and Regulation in Prokaryotes. Journal of Molecular Biology 431, 3945–3946; 10.1016/j.jmb.2019.07.026 (2019)." />
<meta name="citation_reference" content="Babski, J. et al. Small regulatory RNAs in Archaea. RNA Biology 11, 484–493; 10.4161/rna.28452 (2014)." />
<meta name="citation_reference" content="Oliva, G., Sahr, T. &amp; Buchrieser, C. Small RNAs, 5’ UTR elements and RNA-binding proteins in intracellular bacteria: impact on metabolism and virulence. FEMS Microbiology Reviews 39, 331–349; 10.1093/femsre/fuv022 (2015)." />
<meta name="citation_reference" content="Ren, G.-X., Guo, X.-P. &amp; Sun, Y.-C. Regulatory 3’ Untranslated Regions of Bacterial mRNAs. Frontiers in Microbiology 8, 1276; 10.3389/fmicb.2017.01276 (2017)." />
<meta name="citation_reference" content="Hirtreiter, A. et al. Spt4/5 stimulates transcription elongation through the RNA polymerase clamp coiled-coil motif. Nucleic Acids Research 38, 4040–4051; 10.1093/nar/gkq135 (2010)." />
<meta name="citation_reference" content="Hirtreiter, A., Grohmann, D. &amp; Werner, F. Molecular mechanisms of RNA polymerase--the F/E (RPB4/7) complex is required for high processivity in vitro. Nucleic Acids Research 38, 585–596; 10.1093/nar/gkp928 (2010)." />
<meta name="citation_reference" content="Berkemer, S. J. et al. Identification of RNA 3’ ends and termination sites in Haloferax volcanii (2019)." />
<meta name="citation_reference" content="Yue, L. et al. aCPSF1 controlled archaeal transcription termination: a prototypical eukaryotic model (2019)." />
<meta name="citation_reference" content="Daum, B. et al. Structure and in situ organisation of the Pyrococcus furiosus archaellum machinery. eLife 6; 10.7554/eLife.27470 (2017)." />
<meta name="citation_reference" content="Deutscher, M. P. Twenty years of bacterial RNases and RNA processing: how we’ve matured. RNA (New York, N.Y.) 21, 597–600; 10.1261/rna.049692.115 (2015)." />
<meta name="citation_reference" content="Venema, J. &amp; Tollervey, D. Processing of pre-ribosomal RNA in Saccharomyces cerevisiae. Yeast (Chichester, England) 11, 1629–1650; 10.1002/yea.320111607 (1995)." />
<meta name="citation_reference" content="Littlefield, J. W. &amp; Dunn, D. B. Natural occurrence of thymine and three methylated adenine bases in several ribonucleic acids. Nature 181, 254–255; 10.1038/181254a0 (1958)." />
<meta name="citation_reference" content="Smith, J. D. &amp; Dunn, D. B. The occurrence of methylated guanines in ribonucleic acids from several sources. The Biochemical journal 72, 294–301; 10.1042/bj0720294 (1959)." />
<meta name="citation_reference" content="Li, S. &amp; Mason, C. E. The pivotal regulatory landscape of RNA modifications. Annual review of genomics and human genetics 15, 127–150; 10.1146/annurev-genom-090413-025405 (2014)." />
<meta name="citation_reference" content="Schwartz, S. &amp; Motorin, Y. Next-generation sequencing technologies for detection of modified nucleotides in RNAs. RNA Biology 14, 1124–1137; 10.1080/15476286.2016.1251543 (2017)." />
<meta name="citation_reference" content="Sas-Chen, A. &amp; Schwartz, S. Misincorporation signatures for detecting modifications in mRNA: Not as simple as it sounds. Methods (San Diego, Calif.) 156, 53–59; 10.1016/j.ymeth.2018.10.011 (2019)." />
<meta name="citation_reference" content="Lorenz, D. A., Sathe, S., Einstein, J. M. &amp; Yeo, G. W. Direct RNA sequencing enables m6A detection in endogenous transcript isoforms at base specific resolution. RNA (New York, N.Y.); 10.1261/rna.072785.119 (2019)." />
<meta name="citation_reference" content="Leger, A. et al. RNA modifications detection by comparative Nanopore direct RNA sequencing (2019)." />
<meta name="citation_reference" content="Sloan, K. E. et al. Tuning the ribosome: The influence of rRNA modification on eukaryotic ribosome biogenesis and function. RNA Biology 14, 1138–1152; 10.1080/15476286.2016.1259781 (2017)." />
<meta name="citation_reference" content="Dar, D. &amp; Sorek, R. Extensive reshaping of bacterial operons by programmed mRNA decay. PLoS Genetics 14, 1–19; 10.1371/journal.pgen.1007354 (2018)." />
<meta name="citation_reference" content="Yu, S.-H., Vogel, J. &amp; Förstner, K. U. ANNOgesic: A Swiss army knife for the RNA-Seq based annotation of bacterial/archaeal genomes (2017)." />
<meta name="citation_reference" content="Freese, N. H., Norris, D. C. &amp; Loraine, A. E. Integrated genome browser: visual analytics platform for genomics. Bioinformatics 32, 2089–2095; 10.1093/bioinformatics/btw069 (2016)." />
<meta name="twitter:title" content="Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial ( E. coli ) and archaeal ( H. volcanii, P. furiosus ) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism." />
<meta name="og-title" property="og:title" content="Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial ( E. coli ) and archaeal ( H. volcanii, P. furiosus ) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-19" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.18.880849v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.18.880849v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.18.880849v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1051835" />
    <title>Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qlzsriEqgjAIYy_ofPDGqrl58Ocm0TFGsxptGMLsw48","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1051835","apath":"\/biorxiv\/early\/2019\/12\/19\/2019.12.18.880849.atom","pisa":"biorxiv;2019.12.18.880849v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.18.880849v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.18.880849v1","title":"Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications","identifier":"node\/1051835"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222LU5h46Uuofq5qW3kNb5dCprH6pw+eQX399sxVRwZBV2f7mCaXyPTR10kDD4SBvNsSKXlVXV0OjlhLrkWmM19mHIlNV5ORLkziC77FaycAHXi+p5b6sX7dpj8LI6SCcPRZyBgGTlsnMOxh9SKAIBMfM4LhqCVICBYLiVXN5b3v1GoPvNAOmpYdR7QUq5J\\\\\\\/JfwDRXYES3tZfF2PsONqmtAB\\\\\\\/CLCQQDUQHkMEvGd8+Ey0ITnrY9LvFxQ5lE9HvPhok\\\\\\\/Cm+po2Jx9R8A0CA7NhP0s8ZVIAF9UPYFGTdBOdhskuZjORfACKoTkrm9QODlUrE7YsMJdTkcGh7IraMp59ahA+vBI1LBynmAzfOyCW4yYSHhuYD0pA7mYNbp54p6SYLp+QOAMnQURsXiZfBHjc0n+0cyjrhxHHNqFuQpBHlbo3fQq0w1NaTyJQGUdAPJ8mmBA0fTyCjgBx+GYDxOx2zzb9VJfUpWe2nPgreg45rMZQ6HQ0+VPHkLuArBe0RU5fLd4Hz0JNbfJZmJHbxiBoBmQARKGCwMessOrNtPuoU9esfXcdPME6fTVAQMe9XdKmlWuiHmx2z8RVyJUkak9unrWAWd46BIasUVvwpwITbiNicRYClS4WNh0mujmEAuv4C2oAi9xxM+4xpHq6cGPIYQ0BnxeCoe4df7fIUJPYae+uBvVOhHCcUEdYPUiv2XqSBigxoE66fWYl63DoyVuPQfsXF6BDkXgFAGtl2O3nPnLyOPDwqiUUpUJRI5VH1TtHZypdtVUc3qLOQXUCulumdJXWBxjCVKFjLN\\\\\\\/OQNOl74PrGU34Xul0SMoCynKVx1nVnepeYh81bnEy01zOIa7gSxEK7T66UrSZZp2mows8qrrKHmfJV2z96b3ijj5iS88aW+TiSXOTuWjyWkbl+glSBbbArqt1qFVNKQQ9fAa5LAWWTXXdVwTewW02EHvogpdNYxCobebhTg7VX+qAfTSA2oDOKiGPbZt42gSRZEt7XxAEgzrptlX62V7OihTBJL\\\\\\\/Knu9zJakz4JKUyqcMW36HNn9VpATuZmy1hocKEwBsGi8ltLu2EmRy6JkT57ayNfKJGhmfcTA3tLBkmB55qOGvIByjiHEYmC1gE4YWvRoTUIubDjxSFlxqukQDc6sa3axLpVNWn+\\\\\\\/lDVIM5icUnIi2y0Um0TkPEM+fmMKMcbFGu9002Tbm6olD8VNMIKM2sLRnQ2OyaxYNQrHdasPJ5ET7JVnXoPLYTfOV6Sq0i1PaD5lA4mA\\\\\\\/8CZd4XSHT6bwrVyhW6iUO0oVUwBUmMwqnH+K77nqg549i2X35sWIbN995l36Ag+88aFIQTX2vXRSWReoJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022VsshjzPVKCV+xrnLk95\\\\\\\/jA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b6eac75a0210a40e5aa6ae10af729b91c0dbbeb9e7447f51a9ebffe140b867f4\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.18.880849v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1051835 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.18.880849v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OFhvCDesN04SgcImqVwyQ2v6H0TOiqVPXTeOL9pYjB4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1051835" id="node1051835" data-pisa="biorxiv;2019.12.18.880849v1" data-pisa-master="biorxiv;2019.12.18.880849" data-apath="/biorxiv/early/2019/12/19/2019.12.18.880849.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Grünberger</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Knüppel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Jüttner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Fenk</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Borst</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3455-1495" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Reichelt</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Winfried</span> <span class="nlm-surname">Hausner</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6132-5840" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jörg</span> <span class="nlm-surname">Soppa</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-0522-843X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sebastien</span> <span class="nlm-surname">Ferreira-Cerca</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-0570-2517" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Grohmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.18.880849 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1051835" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Felix Grünberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &#38; Biochemistry Centre Regensburg, University of Regensburg</span>, Universitätsstraße 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Felix%2BGr%C3%BCnberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gr%C3%BCnberger%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFelix%2BGr%25C3%25BCnberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert Knüppel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg</span>, Universitätsstraße, 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKn%C3%BCppel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kn%C3%BCppel%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BKn%25C3%25BCppel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Jüttner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg</span>, Universitätsstraße, 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ%C3%BCttner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=J%C3%BCttner%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ%25C3%25BCttner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Fenk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &#38; Biochemistry Centre Regensburg, University of Regensburg</span>, Universitätsstraße 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BFenk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fenk%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BFenk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andreas Borst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Goethe-University, Biocentre, Institute for Molecular Biosciences</span>, Max-von-Laue-Str. 9, D-60439 Frankfurt, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BBorst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borst%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BBorst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert Reichelt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &#38; Biochemistry Centre Regensburg, University of Regensburg</span>, Universitätsstraße 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BReichelt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reichelt%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BReichelt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3455-1495" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Reichelt</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Winfried Hausner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &#38; Biochemistry Centre Regensburg, University of Regensburg</span>, Universitätsstraße 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Winfried%2BHausner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hausner%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWinfried%2BHausner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jörg Soppa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Goethe-University, Biocentre, Institute for Molecular Biosciences</span>, Max-von-Laue-Str. 9, D-60439 Frankfurt, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%B6rg%2BSoppa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soppa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25B6rg%2BSoppa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6132-5840" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jörg Soppa</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sebastien Ferreira-Cerca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Biochemistry, Genetics and Microbiology, Biochemistry III, University of Regensburg</span>, Universitätsstraße, 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BFerreira-Cerca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira-Cerca%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASebastien%2BFerreira-Cerca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0522-843X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sebastien Ferreira-Cerca</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sebastien.ferreira-cerca@ur.de" class="" data-icon-position="" data-hide-link-title="0">sebastien.ferreira-cerca@ur.de</a>
<a href="mailto:dina.grohmann@ur.de" class="" data-icon-position="" data-hide-link-title="0">dina.grohmann@ur.de</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Dina Grohmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biochemistry, Genetics and Microbiology, Institute of Microbiology and Archaea Centre, Single-Molecule Biochemistry Lab &#38; Biochemistry Centre Regensburg, University of Regensburg</span>, Universitätsstraße 31, 93053 Regensburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dina%2BGrohmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grohmann%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADina%2BGrohmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0570-2517" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dina Grohmann</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sebastien.ferreira-cerca@ur.de" class="" data-icon-position="" data-hide-link-title="0">sebastien.ferreira-cerca@ur.de</a>
<a href="mailto:dina.grohmann@ur.de" class="" data-icon-position="" data-hide-link-title="0">dina.grohmann@ur.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.18.880849v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.18.880849v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.18.880849v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.18.880849v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.18.880849v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.18.880849v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.18.880849v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1051835" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1051835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The prokaryotic transcriptome is shaped by transcriptional and posttranscriptional events that define the characteristics of an RNA, including transcript boundaries, the base modification status, and processing pathways to yield mature RNAs. Currently, a combination of several specialised short-read sequencing approaches and additional biochemical experiments are required to describe all transcriptomic features. In this study, we present native RNA sequencing of bacterial (<em>E. coli</em>) and archaeal (<em>H. volcanii, P. furiosus</em>) transcriptomes employing the Oxford Nanopore sequencing technology. Based on this approach, we could address multiple transcriptomic characteristics simultaneously with single-molecule resolution. Taking advantage of long RNA reads provided by the Nanopore platform, we could accurately (re-)annotate large transcriptional units and boundaries. Our analysis of transcription termination sites revealed that diverse termination mechanisms are in place in archaea. Moreover, we shed light on the poorly understood rRNA processing pathway in archaea and detected new processing intermediates. One of the key features of native RNA sequencing is that RNA modifications are retained. We could confirm this ability by analysing the well-known KsgA-dependent rRNA methylation sites. Notably, our analysis suggests that rRNA modifications are more abundant in a hyperthermophilic organism.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">In the last decade, next-generation sequencing (NGS) technologies<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> revolutionized the field of microbiology<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, which is not only reflected in the exponential increase in the number of fully sequenced microbial genomes, but also in the detection of microbial diversity in many hitherto inaccessible habitats based on metagenomics. Using transcriptomics, important advances were also possible in the field of RNA biology<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> that shaped our understanding of the transcriptional landscape<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> and RNA-mediated regulatory processes in prokaryotes<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. RNA sequencing (RNA-seq) technologies can be categorized according to their platform-dependent read lengths and necessity of a reverse transcription and amplification steps, to generate cDNA<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. Illumina sequencing yields highly accurate yet short sequencing reads (commonly 100-300 bp). Hence, sequence information is only available in a fragmented form, making full-length transcript- or isoform-detection a challenging task<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Sequencing platforms developed by Pacific Bioscience (PacBio) and Oxford Nanopore Technologies (ONT) solved this issue. Both sequencing methods are <em>bona fide</em> single-molecule sequencing techniques that allow sequencing of long DNAs or RNAs<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. However, the base detection differs significantly between the two methods. PacBio-sequencers rely on fluorescence-based single-molecule detection that identifies bases based on the unique fluorescent signal of each nucleotide during DNA synthesis by a dedicated polymerase<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup>. In contrast, in an ONT sequencer, the DNA or RNA molecule is pushed through a membrane-bound biological pore with the aid of a motor protein that is attached to the pore protein called a nanopore (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>). A change in current is caused by the translocation of the DNA or RNA strand through this nanopore, which serves as a readout signal for the sequencing process. Due to the length of the nanopore (version R9.4), a stretch of approximately five bases contributes to the current signal. Notably, only ONT offers the possibility to directly sequence native RNAs without the need for prior cDNA synthesis and PCR amplification<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. As RNAs are directly “read”, RNA modifications are maintained and first attempts have been made to use ONT sequencing to identify RNA base modifications (e.g. methylations<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>). As ONT-based native RNA-seq holds the capacity to sequence full-length transcripts, to identify RNA base modifications and to detect molecular heterogeneity in a transcriptome, the technology found widespread attention<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Recently, the technology was exploited to sequence viral RNA genomes<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> to gain insights into viral and eukaryotic transcriptomes<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> and to detect RNA isoforms in eukaryotes<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. However, prokaryotic transcriptomes have not been characterized on the genome-wide level by native RNA-seq approaches so far as prokaryotic RNAs lack a poly(A) tail which is required to capture the RNA and feed it into the nanopore.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nanopore-based native RNA sequencing of prokaryotes. a, Key steps of library preparation: (1) native RNA is polyadenylated, which allows library preparation using the direct RNA kit from Oxford Nanopore and sequencing on a MinION device. (2) 3&#x2019; ligation is performed to add the an adapter carrying the motor-protein (red square), which unzips the RNA-cDNA hybrid and pulls the RNA through the Nanopore (detailed description see Supplementary Fig. 1a). b, Data sets for three prokaryotic model organisms (Ecoli: Escherichia coli, Pfu: Pyrococcus furiosus, Hvo: Haloferax volcanii) with (+) and without (-) Terminator 5&#x2019;-Phosphate-Dependent Exonuclease (TEX) treatment were collected and mapped to their respective reference genome. Transcript abundances of genomic features (protein coding genes (CDS): red, 5S rRNA: green, 16S rRNA: purple, 23S rRNA: light-purple) were estimated using featurecounts42." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Nanopore-based native RNA sequencing of prokaryotes.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Key steps of library preparation: (1) native RNA is polyadenylated, which allows library preparation using the direct RNA kit from Oxford Nanopore and sequencing on a MinION device. (2) 3&#x2019; ligation is performed to add the an adapter carrying the motor-protein (red square), which unzips the RNA-cDNA hybrid and pulls the RNA through the Nanopore (detailed description see Supplementary Fig. 1a). &lt;strong&gt;b&lt;/strong&gt;, Data sets for three prokaryotic model organisms (Ecoli: &lt;em&gt;Escherichia coli&lt;/em&gt;, Pfu: &lt;em&gt;Pyrococcus furiosus&lt;/em&gt;, Hvo: &lt;em&gt;Haloferax volcanii&lt;/em&gt;) with (+) and without (-) Terminator 5&#x2019;-Phosphate-Dependent Exonuclease (TEX) treatment were collected and mapped to their respective reference genome. Transcript abundances of genomic features (protein coding genes (CDS): red, 5S rRNA: green, 16S rRNA: purple, 23S rRNA: light-purple) were estimated using featurecounts&lt;sup&gt;42&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F1.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F1.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <span class="caption-title">Nanopore-based native RNA sequencing of prokaryotes.</span><p id="p-5" class="first-child"><strong>a</strong>, Key steps of library preparation: (1) native RNA is polyadenylated, which allows library preparation using the direct RNA kit from Oxford Nanopore and sequencing on a MinION device. (2) 3’ ligation is performed to add the an adapter carrying the motor-protein (red square), which unzips the RNA-cDNA hybrid and pulls the RNA through the Nanopore (detailed description see Supplementary Fig. 1a). <strong>b</strong>, Data sets for three prokaryotic model organisms (Ecoli: <em>Escherichia coli</em>, Pfu: <em>Pyrococcus furiosus</em>, Hvo: <em>Haloferax volcanii</em>) with (+) and without (-) Terminator 5’-Phosphate-Dependent Exonuclease (TEX) treatment were collected and mapped to their respective reference genome. Transcript abundances of genomic features (protein coding genes (CDS): red, 5S rRNA: green, 16S rRNA: purple, 23S rRNA: light-purple) were estimated using featurecounts<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Here, we present a native RNA sequencing study of bacterial and archaeal transcriptomes using Nanopore technology. We employed an experimental workflow that includes the enzymatic polyadenylation of prokaryotic RNA to make them amenable for ONT’s direct RNA sequencing kit. We evaluated the accuracy and reliability of native RNA-seq in comparison to published Illumina-based sequencing studying prokaryotic transcriptomes of bacterial (<em>E. coli</em>) and archaeal (<em>Haloferax volcanii, Pyrococcus furiosus</em>) model organisms<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. The transcriptomic analysis included determination of transcript boundaries, providing, among others, insights into termination mechanisms in archaea. We moreover demonstrate that the long RNA reads gathered on the ONT platform allow reliable transcriptional unit assignment. Strikingly, we gained insights into the so far poorly understood ribosomal RNA (rRNA) maturation pathway in archaea and detected putative new rRNA processing intermediates. As RNA modifications are retained when sequencing native RNAs, we explore rRNA modifications in prokaryotes. Notably, by expanding the approach, our analysis suggests that rRNA modifications are more abundant in an hyperthermophilic organism.</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><div id="sec-3" class="subsection"><h3>Strains and growth conditions</h3><p id="p-7"><em>Escherichia coli</em> K-12 MG1655 cells were grown in LB medium (10 g tryptone, 5 g yeast extract, 10 g NaCl per liter) to an OD<sub>600nm</sub> of 0.5 and harvested by centrifugation at 3,939 x g for 10 min at 4°C.</p><p id="p-8"><em>Pyrococcus furiosus</em> strain DSM 3638 cells were grown anaerobically in 40 ml SME medium<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> supplemented with 40 mM pyruvate, 0.1 % peptone and 0.1 % yeast extract at 95°C to mid-exponential phase and further harvested by centrifugation at 3,939 x g for 45 min at 4°C.</p><p id="p-9">Markerless deletion of <em>Haloferax volcanii</em> KsgA (Hvo_2746) was obtained using the pop-in/pop-out procedure<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. Deletion candidates were verified by Southern blot and PCR analyses. Full characterization of this strain will be described elsewhere (Knüppel and Ferreira-Cerca, <em>in preparation</em>). Wildtype (H26) and Δ<em>ksgA</em> strains were grown in Hv-YPC medium at 42°C under agitation as described previously<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>.</p></div><div id="sec-4" class="subsection"><h3>RNA isolation</h3><p id="p-10"><em>E. coli</em> total RNA was purified using the Monarch<sup>®</sup> Total RNA Miniprep Kit (New England Biolabs) according to manufacturer’s instructions including the recommended on-column DNAse treatment.</p><p id="p-11"><em>P. furiosus</em> cell pellets were lysed by the addition of 1 ml peqGOLD TriFast™ (VWR) followed by shaking for 10 min at room temperature. After adding 0.2 ml 2 M sodium acetate pH 4.0, total RNA was isolated according to the manufacturer’s instructions. Contaminating DNA was removed using the TURBO DNA-free™ Kit (Thermo Fisher Scientific).</p><div id="sec-5" class="subsection"><h4>H. volcanii</h4><p id="p-12"><em>H. volcanii</em> total RNA was purified using the RNA easy kit (Qiagen) according to the manufacturer’s instructions. Alternatively, total RNA was isolated according to the method described by Chomczynski and Sacchi<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>, including a DNA-removal step with RNase-free DNase I (Thermo Scientific).</p><p id="p-13">The Integrity of total RNA from <em>E. coli</em> and <em>P. furiosus</em> was assessed via a Bioanalyzer (Agilent) run using the RNA 6000 Pico Kit (Agilent). To evaluate the extent of remaining buffer and DNA contaminations, the RNA preparation samples were tested using standard spectroscopic measurements (Nanodrop One) and using the Qubit 1X dsDNA HS assay kit (Thermo Fisher Scientific). RNA was quantified using the Qubit RNA HS assay kit.</p></div></div><div id="sec-6" class="subsection"><h3>Primer extension analysis</h3><p id="p-14">5’ends determination of mature 16S and 23S rRNAs from <em>H. volcanii</em> by primer extension was performed as described previously (Knüppel et al, Method in Molecular Biology <em>in press</em>). In brief, reverse transcription was performed with the indicated fluorescently labeled primers (oHv396-DY682: 5’-CCCAATAGCAATGACCTCCG; oHv622-DY782: 5’-GCTCTCGAGCCGAGCTATCCACC) and SuperScript III reverse transcriptase using 1 μg of total RNA as template. The resulting cDNAs and reference dideoxy-chain termination sequencing ladder reactions were separated on a denaturing 14 % TBE-Urea (6 M)-PAGE. Fluorescence signals (700nm and 800nm) were acquired using a Li-COR Odyssey system.</p></div><div id="sec-7" class="subsection"><h3>RNA treatment and poly(A)-tailing</h3><p id="p-15">To prevent secondary structure formation, the RNA was heat incubated at 70°C for 3 min and immediately put on ice before TEX-treatment or poly(A)–tailing of the RNA samples. Partial digestion of RNAs that are not 5’-triphosphorylated (e.g. tRNAs, rRNAs) was achieved by incubation of the RNA with the Terminator 5’-Phosphate-Dependent Exonuclease (TEX, Lucigen). For this purpose, 10 μg of RNA was incubated with 1 unit TEX, 2 μl TEX reaction buffer (Lucigen) and 0.5 μl RiboGuard RNase Inhibitor (Lucigen) in a total volume of 20 μl for 60 minutes at 30°C. The reaction was stopped and the RNA was purified using the RNeasy MinElute Cleanup Kit (Qiagen). For <em>P. furiosus</em> and <em>E. coli</em> RNA samples, control reactions lacking the exonuclease (NOTEX) were treated as described for TEX-containing samples. In the next step, a poly(A)-tail was added using the <em>E. coli</em> poly(A) polymerase (New England Biolabs) following a recently published protocol<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. Briefly, 5 μg RNA, 20 units poly(A) polymerase, 2 μl reaction buffer and 1 mM ATP were incubated for 15 min at 37°C in a total reaction volume of 50 μl. To stop the reaction and to remove the enzyme, the poly(A)-tailed RNA was purified with the RNeasy MinElute Cleanup Kit (Qiagen).</p></div><div id="sec-8" class="subsection"><h3>Direct RNA library preparation and sequencing</h3><p id="p-16">Libraries for Nanopore sequencing were prepared from poly(A)-tailed RNAs according to the SQK-RNA001 Kit protocol (Oxford Nanopore, Version: DRS_9026_v1_revP_15Dec2016) with minor modifications for barcoded libraries (see Supplementary Fig. 1a). In this case, Agencourt AMPure XP magnetic beads (Beckman Coulter) in combination with 1 μl of RiboGuard RNase Inhibitor (Lucigen) were used instead of the recommended Agencourt RNAclean XP beads to purify samples after enzymatic reactions. The total amount of input RNA, the barcoding strategy and the number of flowcells used can be found in Supplementary Table 1. The efficiency of poly(A)-tailing was low. However, this could be compensated with a higher amount of input RNA. We added the ONT control RNA (RCS, yeast) to detect problems that arise from library preparation or sequencing. For the barcoded libraries, the RTA adapter was replaced by custom adapters described in <a href="https://github.com/hyeshik/poreplex">https://github.com/hyeshik/poreplex</a> and reverse transcription (RT) was performed in individual tubes for each library. After RT reactions, cDNA was quantified using the Qubit DNA HS assay kit (Thermo Fisher Scientific) and equimolar amounts of DNA for the multiplexed samples were used in the next step for ligation of the RNA Adapter (RMX) in a single tube. Subsequent reactions were performed according to the protocols recommended by ONT. The libraries were sequenced on a MinION using R9.4 flow cells and subsequently, FAST5 files were generated using the recommended script in MinKNOW.</p></div><div id="sec-9" class="subsection"><h3>Data analysis</h3><div id="sec-10" class="subsection"><h4>Demultiplexing of raw reads, basecalling and quality control of raw reads</h4><p id="p-17">As some bioinformatic tools depend on single-read files we first converted multi-read FAST5 files from the MinKNOW output to single-read FAST5 files using the ont_fast5_api from Oxford Nanopore (<a href="https://github.com/nanoporetech/ont_fast5_api">https://github.com/nanoporetech/ont_fast5_api</a>). To prevent actual good-quality reads from being discarded (this issue was reported previously<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>), we included both failed and passed read folders in the following steps of the analysis. Demultiplexing was done by poreplex (version 0.4, <a href="https://github.com/hyeshik/poreplex">https://github.com/hyeshik/poreplex</a>) with the arguments --trim-adapter, --symlink-fast5, --basecall and --barcoding, to trim off adapter sequences in output FASTQ files, basecall using albacore, create symbolic links to FAST5 files and sort the reads according to their barcodes. However, to ensure consistency between non-multiplexed and multiplexed samples and because of some major improvements in the current basecalling software (guppy), albacore files were not used. Instead demultiplexed FAST5 reads and raw FAST5 reads from non-multiplexed runs were locally basecalled using Guppy (Version 3.0.3) with --reverse_sequence, --hp_correct, -- enable_trimming and --calib_detect turned on. After that, relevant information from the sequencing_summary.txt file in the Guppy output was extracted to analyse properties of raw reads (see Supplementary Fig. 2, see Supplementary Table 1).</p></div><div id="sec-11" class="subsection"><h4>Mapping of reads and quantification</h4><p id="p-18">Files were mapped to reference genomes from <em>Escherichia coli</em> K12 MG1655 (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=U00096.2&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom">U00096.2</a>)<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>, <em>Haloferax volcanii</em> (NCBI Reference Sequence NC_013967)<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> and <em>Pyrococcus furiosus</em> DSM3638<sup><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup> using minimap2 (Release 2.17-r941, <a href="https://github.com/lh3/minimap2">https://github.com/lh3/minimap2</a>)<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. Output alignments in the SAM format were generated with the recommended options for noisy Nanopore Direct RNA-seq (-ax splice, -uf, -k14) and also with (1) -p set to 0.99, to return primary and secondary mappings and (2) with --MD turned on, to include the MD tag for calculating mapping identities. Alignment files were further converted to bam files, sorted and indexed using SAMtools<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Strand-specific wig and bigwig files were finally created using bam2wig (Version 1.5, <a href="https://github.com/MikeAxtell/bam2wig">https://github.com/MikeAxtell/bam2wig</a>). To evaluate the alignments, we first calculated the aligned read length by adding the number of M and I characters in the CIGAR string<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a></sup>. Based on this, the mapping identity was defined as (1-NM/aligned_reads)*100, where NM is the edit distance reported taken from minimap2. Read basecalling and mapping metrics can be found in Supplementary Table 1.</p></div><div id="sec-12" class="subsection"><h4>Gene expression analysis</h4><p id="p-19">For transcript abundance estimation we applied featureCounts (Rsubread 1.32.4) allowing that a read can be assigned to more than one feature (allowMultiOverlap = TRUE) and applying the setting for long reads (isLongRead = TRUE)<sup><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup>. Calculations were performed based on the genome coordinates of genomic feature types (tRNA, rRNA, protein-coding genes). For the abundance comparison to Illumina-sequencing, we applied a regularized log transformation from the DESeq2 package that transforms counts on a log2 scale, normalizing for the library size and minimizing differences between samples with small counts<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> (raw count data for TEX samples in Supplementary Table 2).</p></div><div id="sec-13" class="subsection"><h4>Poly(A) tail analysis</h4><p id="p-20">Poly(A) tail length was estimated by nanopolish following the recommended workflow (Version 0.10.2, <a href="https://nanopolish.readthedocs.io/en/latest/quickstart_polya.html">https://nanopolish.readthedocs.io/en/latest/quickstart_polya.html</a>)<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>.</p></div><div id="sec-14" class="subsection"><h4>Detection of transcriptional units and annotation of transcription start sites and transcription termination sites</h4><p id="p-21">The definition of transcriptional units (TU) and our strategy to detect and annotate them was based on a recent study that re-defined the bioinformatical search for transcriptional units (TU)<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. The TU annotation was performed in a two-step process in the following way: First, TU clusters were defined by collapsing all reads that overlap and fulfill certain criteria that are commented on extensively in the available code for this study (<a href="https://github.com/felixgrunberger/Native_RNAseq_Microbes">https://github.com/felixgrunberger/Native_RNAseq_Microbes</a>). In short, reads were filtered out that did not align protein-coding genes (CDS) or tRNAs, had a mapping identity below 80%, were spliced, were shorter than 50% of the gene body and did not cover either the 5’ or the 3’ untranslated region. The remaining overlapping reads were collapsed in a strand-specific manner and merged.</p><p id="p-22">Finally, the collapsed reads that represent the TU cluster, were split according to the coverage drop at the 3’ region of a gene. This was achieved by calculating the sequencing depth in a window of 20 nt upstream and downstream of the corresponding TTS and applying a deliberately low threshold of 1.5x (higher coverage upstream compared to downstream, see transcriptional unit table in Supplementary Table 5).</p><p id="p-23">TSS were predicted by calculating the median start position of all reads that map to one gene and cover the 5’ part of a CDS. To address the 3’ coverage bias and the underrepresentation of reads that map to the 5’end and also for the 12 missing nucleotides at the TSS in general, all reads starting at least 20 nt downstream of the annotated gene start were included. To not exclude too many reads, the position of TTS were predicted similarly, by also including reads that have end positions starting from 20 nt upstream of a gene end (TSS table in Supplementary Table 3, TTS table in Supplementary Table 4).</p><p id="p-24">For the analysis of prokaryotic promoter elements, the sequences from 58 bases to 11 bases upstream of all uncorrected TSS that are starting a TU were extracted, to identify relevant motifs using MEME with default options except for a custom background file, calculated from intergenic sequences of the respective organism<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. The analysis of terminator sequences was performed comparably by extracting all TTS that are located at the end of a TU and searching for terminators in a sequence window from −45 to +45 from the TTS using MEME and the custom background model. Heatmap analysis of motif positioning was performed by importing MEME FASTA information into R. Metaplots of the nucleotide enrichment analysis (compare<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>) were calculated by comparing the genomic sequences surrounding a TTS in a window from −45 to 45 to randomly selected intergenic positions (subsampled, n = 10000).</p></div><div id="sec-15" class="subsection"><h4>Modified base detection</h4><p>The performance of two different approaches (Tombo vs. error and basecalling properties) for the detection of modified bases was evaluated:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-26">We used Tombo (Version 1.5, <a href="https://nanoporetech.github.io/tombo">https://nanoporetech.github.io/tombo</a>) to identify modified bases based on a comparison to a theoretical distribution (<em>de novo</em> model) and based on the comparison to a reference data set (sample-compare model)<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. Briefly, for <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6a</a> reads mapping to 16S rRNA were preprocessed, resquiggled and the raw signal plotted at a specific genomic coordinate using the respective plotting command (tombo plot genome_locations). In addition, the probability of modified bases was calculated using the detect_modification de_novo command. For <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6b</a> the signals were calculated for both samples (wildtype and deletion mutant) and compared using the <em>control-fast5-basedirs</em> and <em>overplot Boxplot</em> option. For <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig 6c</a> a reference data set was created by sorting the reads mapping to the 16S rRNA based on the position of the read start (before or after gene start), thereby dividing the data set in reads that belong to mature and unprocessed 16S rRNAs. The two read sets were used as input for the sample-compare method. The two different models (<em>de novo</em> and sample-compare using non-mature 16S reads as a reference set) were applied to the complete 16S, base-modification-probabilities extracted from the tombo output and plotted using a custom R script.</p></li><li id="list-item-2"><p id="p-27">For calculating the frequency of correct, deleted and wrong nucleotides at a genomic position the function pileup from the Rsamtools package was used (<a href="https://rdrr.io/bioc/Rsamtools/">https://rdrr.io/bioc/Rsamtools/</a>). Plots were generated using custom R scripts. The results were compared to known modification sites in 16S rRNA for <em>Escherichia coli</em><sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> and <em>Haloferax volcanii</em><sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. Given the high degree of conservation of the modifying enzymes in archaea and because there was no reference set available for <em>Pyrococcus furiosus</em>, the results were compared to the five modified positions that are described for <em>H. volcanii</em> (910, 1352, 1432, 1450/1451 correspond to 938, 1379,1469,1486/1487 in <em>P. furiosus</em>).</p></li></ol></div></div><div id="sec-16" class="subsection"><h3>Public data</h3><p id="p-28">In addition to the in-house generated data, we made use of other published sequencing data sets and data repositories that are described in the following.</p><div id="sec-17" class="subsection"><h4>Transcriptional start sites</h4><p id="p-29">For all three model organisms, global transcriptional start sites were mapped recently using differential RNA sequencing<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a></sup>. Position data were extracted from the Supplementary data of the publications and compared with the TSS described in the ONT data sets given that a start site was found in both data sets.</p></div><div id="sec-18" class="subsection"><h4>Transcriptional termination sites</h4><p id="p-30">So far there are no transcription termination data sets available for <em>H. volcanii</em> and <em>P. furiosus</em>. The 3’ UTR lengths of the <em>E. coli</em> ONT set were compared to TTS predicted based on the Term-Seq method<sup><a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a></sup></p></div><div id="sec-19" class="subsection"><h4>Transcriptional units</h4><p id="p-31">The widely used database DOOR2<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> was used to compare the TU annotation for both archaeal sets. For <em>E. coli</em> a more recent, but also purely bioinformatical prediction, served as a reference set<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>.</p></div><div id="sec-20" class="subsection"><h4>Gene expression comparison</h4><p id="p-32">For <em>P. furiosus</em> gene abundances from ONT data were compared to fragmented RNA sequencing data of mixed growth conditions (conditions, library, sequencing, mapping described in<sup><a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a></sup>), by applying a regularized log transformation as described earlier<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a></sup>. For <em>H. volcanii</em> comparison, raw reads of a mixed RNA sequencing were extracted from the SRA (SRR7811297)<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> trimmed using trimmomatic<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>, (leading:20, trailing:20, slidingwindow:4:20, minlen:12), mapped to the reference genome using bowtie2 (-N 0, -L 26)<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>, converted to sorted bam files using samtools<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup> and compared to ONT data as described for <em>P. furiosus</em>. Illumina RNA sequencing data for <em>E. coli</em> were also extracted from the NCBI (SRP056485, 37°C LB), analysed as described for <em>H. volcanii</em> Illumina data and also compared to the ONT reference data.</p></div></div></div><div class="section" id="sec-21"><h2 class="">Results</h2><div id="sec-22" class="subsection"><h3>Library preparation for Nanopore native RNA sequencing of bacterial and archaeal transcriptomes</h3><p id="p-33">We performed whole-genome native RNA sequencing using the Nanopore sequencing technology (referred to as Nanopore native RNA sequencing in this work) that can be applied to any prokaryotic organism. The key steps of the library preparation are shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>: after enzymatic polyadenylation, the RNA is reverse transcribed to increase sequencing throughput (recommended by ONT)<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>. Subsequently, a motor protein is added that feeds the RNA into the Nanopore. Following this workflow, native RNAs from prokaryotic organisms can be sequenced. Depending on the necessary sequencing depth, the libraries were barcoded using poreplex (<a href="https://github.com/hyeshik/poreplex">https://github.com/hyeshik/poreplex</a>), since this is not yet supported by the official kits and protocols from Oxford Nanopore. To identify primary and processed sites in both mRNA and rRNA, we partially digested 5’-monophosphorylated transcripts using the terminator exonuclease (TEX) and compared them to non-treated samples (see Supplementary Fig. 1a). In contrast to many Illumina sequencing-based approaches that make use of a specialised library preparation design to tackle a well-defined question<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></sup>, we evaluated the potential of native RNA sequencing to analyse multiple transcriptomic features simultaneously including the identification of <em>cis</em>-regulatory elements that govern transcription, the analysis of operon structures and transcriptional boundaries, rRNA processing and 16S rRNA modification patterns (see Supplementary Fig. 1b).</p></div><div id="sec-23" class="subsection"><h3>Sequencing yield and quality control of raw Nanopore reads</h3><p id="p-34">Native RNA sequencing was performed for three prokaryotic organisms: the bacterial model organism <em>Escherichia coli</em>, the halophilic archaeon <em>Haloferax volcanii</em> and the hyperthermophilic archaeon <em>Pyrococcus furiosus</em>. In order to show that native RNA sequencing can be applied to a wide variety of prokaryotic organisms, we specifically chose (i) organisms from the bacterial and archaeal domain of life with <em>P. furiosus</em> and <em>H. volcanii</em> belonging to the Euryarchaeota, (ii) organisms that are classified as mesophilic (<em>E. coli, H. volcanii</em>), hyperthermophilic (<em>P. furiosus</em>), or halophilic organism (<em>H. volcanii</em>) and (iii) organisms that differ significantly in their GC-content (<em>E. coli</em>: 50.8%<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>, <em>H. volcanii</em>: 65%<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>, <em>P. furiosus</em>: 40.8%<sup><a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a></sup>). The prepared libraries were sequenced on a MinION device and reads were collected over 48 hours on R9.4 flow cells (see Supplementary Fig. 2a). The total number of reads in the TEX-treated libraries were sufficient to achieve good coverage of the transcriptome (<em>E</em>. coli: 9.2x, <em>P. furiosus</em>: 15.0x, <em>H. volcanii</em>: 10.3x, see Supplementary Table 1) to perform transcriptional unit annotation and accurate determination of transcript boundaries. Before mapping the reads to the reference genomes, the quality of the sequencing runs was evaluated based on raw read length distribution and quality of reads estimated by Guppy (see Supplementary Fig. 2b,c). To verify that no problems occurred during sequencing or library preparation a poly-adenylated spike-in control (yeast enolase) was used. The control showed a uniform length distribution (median lengths between 1212 and 1306 nucleotides) and a very good read quality (median quality between 10.8 and 12.2) in all samples, therefore, excluding any bias (see Supplementary Fig. 2b,c, Supplementary Table 1). Due to large differences in the composition of the RNAs used to train the basecaller and the sequenced reads, that contain mostly ribosomal RNAs known to harbor base modifications, the overall quality is lower than for the spike-in control in every data set<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup>. To prevent useful reads from being discarded, we decided to also include reads that fall below the standard quality score threshold of 7 and continued with the mapping of the reads to the reference genomes<sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a></sup>.</p></div><div id="sec-24" class="subsection"><h3>Analysis of mapped reads</h3><p id="p-35">An advantage of the long-read Nanopore sequencing technique is that native RNA strands can be sequenced directly as near full-length transcripts<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>. This is also reflected in the sequenced data sets as aligned lengths up to 7864 nt can be observed (see <em>H. volcanii</em> NOTEX sample with a median read length of 1390 nt, Supplementary Fig. 3c). The majority of reads from <em>E. coli</em> and <em>H. volcanii</em> samples mapped to the 23S rRNA (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a>, see Supplementary Fig. 3a). In general, the read identity of CDS-mapping reads is higher than for rRNA mapping reads, but lower than the spike-in control (see Supplementary Fig. 3b,c). It is noteworthy, that a minimum aligned length of about 100 nt appears in the data sets (see Supplementary Fig. 3d) when using the native RNA sequencing protocol and minimap2 as mapping tool<sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>. Unaligned reads had a median read length of 191 nt, in contrast to 572 nt for aligned reads (all data sets combined) suggesting that short reads could not be aligned properly. As small RNAs, CRIPSR-RNAs or tRNAs fall below this threshold, we excluded these RNAs from further analysis. While short transcripts are a problem, longer RNAs are sequenced without loss in quality (see Supplementary Fig. 3e). As the raw read quality correlates with the mapping identity of the reads, problems during sequencing can be live-monitored in MinKNOW and the run can be canceled allowing the loading of a new library (see Supplementary Fig. 3f). Since the subsequent analysis of transcriptional units is heavily dependent on the integrity of the data, we verified the data integrity in the next steps. Sources for bias could emerge (i) from the poly(A)-tailing efficiency that might differ between transcripts and (ii) from the loss of certain transcripts during library preparation or sequencing problems. Comparing the initial amount of RNA with the reverse transcribed DNA, we observed that the enzymatic addition of a 3’ poly(A)-tail is not very efficient (~10%). In general, the addition of poly(A)<sub>20</sub> (length of the reverse transcription adapter) is sufficient to allow for the annealing of the poly(T)-adapter required for reverse transcription and sequencing. This goes in line with the shortest median length we observed for the 5S rRNA (see <em>E. coli</em> TEX sample) (see Supplementary Fig. 4). For most of the transcripts, a tail with 50 to 100 nt was detected. The overall correlation of transcript abundances calculated from sequencing data using Nanopore or Illumina technology was very high given that only mixed conditions could be compared in the case of <em>H. volcanii</em> and <em>P. furiosus</em> (see Supplementary Fig. 5a,b,c, transcript abundance data for TEX samples in Supplementary Table 2).</p></div><div id="sec-25" class="subsection"><h3>Mapping of transcriptional boundaries</h3><div id="sec-26" class="subsection"><h4>Transcription start sites</h4><p id="p-36">Transcription start site (TSS) and transcription termination site (TTS) detection was based on the determination of transcriptional units (TU) (compare material and methods section)<sup><a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></sup>. In total, we found substantial overlap between TTS detected by Illumina d(ifferential) RNA-seq<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>–<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup> and Nanopore native RNA-seq (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>)(positions of TSS derived from ONT TEX-treated samples in Supplementary Table 3). The portion of ONT only and Illumina only is mostly caused by the different algorithms used and the limited sequencing depth in the ONT data sets. Strikingly, the median 5’ untranslated region (UTR) lengths were very similar when data from ONT native and Illumina-based RNA sequencing were compared (<em>E. coli</em>: 68 ONT vs. 62 Illumina; <em>P. furiosus</em>: 23 ONT vs. 13 Illumina; <em>H. volcanii</em>: 1 ONT vs. 0 Illumina). Please note that TSS-mapping based on Nanopore native RNA-seq data must be corrected by 12 nucleotides (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a>). It has been observed previously that about 12 nt are missing at the 5’-end of the sequenced RNAs. This observation can be explained by a lack of control of the RNA translocation speed after the motor protein falls off the 5’ end of the RNA (see Supplementary Figure 6a)<sup><a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>,<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. Promoter analysis confirmed the presence of well-known sequence motifs of bacterial and archaeal promoters<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. This includes the TATA-box and B-recognition element (BRE) characteristic for archaeal promoters and the −10 element in bacterial promoters (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2d</a>). The −35 element in <em>E. coli</em> has been previously shown to be less enriched compared to the −10 site<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>, which might explain why this element cannot be detected in the Nanopore data set. To analyse TSSs in more detail, we compared the 5’ UTR lengths for all genes with predicted TSS in ONT and Illumina data sets (see Supplementary Figure 6b,c,d). The overall correlation between the two techniques was very high (see Supplementary Fig. 6). As expected, the correlation improves with increasing sequencing depth for a gene (&gt;5 reads). While TEX-treatment is a common way of predicting TSS in Illumina sequencing, we observed that it is not necessary for ONT data as very similar TSS are found in both TEX and NOTEX data sets (ρ = 0.86) (see Supplementary Figure 6e).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of transcript boundaries. a, Primary transcription start sites (TSS) were predicted based on Nanopore reads and compared to Illumina d(ifferential) RNA-Seq data from published data sets for E. coli30, P. furiosus31 and H. volcanii27. The total number of all genes with a detected TSS is shown as grey barplots and results from the sum of Nanopore-only predicted TSS and the intersection to the Illumina data. b, Position of TSS is corrected for 12 nucleotides to calculate the length of 5&#x2019; untranslated regions (UTR) in the Nanopore data sets (purple). 5&#x2019; UTRs are compared to d(ifferential) RNA-Seq Illumina data sets (light-green). Median values are indicated by a black bar inside the distribution (compare Supplementary Fig. 6). c, Length of 3&#x2019; UTRs is based on the prediction of transcription termination sites (TTS) and the comparison to annotated gene ends. Distribution of lengths is shown for Nanopore data sets (purple) and compared to a Term-Seq E. coli Illumina data set (light-green)108. d, MEME analysis45 of extracted sequences upstream of Nanopore-predicted TSS reveals bacterial (position &#x2212;10) and archaeal-specific promoter elements (BRE: B-recognition element, recognized by transcription factor B, TATA: TATA-box bound by the TATA-binding protein TBP), therefore validating the positions of predicted TSS. e, Nucleotide enrichment meta analysis was carried out by comparing the genomic sequences surrounding the TTS (&#x2212;45 to +45) to randomly selected intergenic positions of the respective organism (n: 10000) (Terminator motifs in Supplementary Fig. 7)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection of transcript boundaries.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Primary transcription start sites (TSS) were predicted based on Nanopore reads and compared to Illumina d(ifferential) RNA-Seq data from published data sets for &lt;em&gt;E. coli&lt;/em&gt;&lt;sup&gt;30&lt;/sup&gt;, &lt;em&gt;P. furiosus&lt;/em&gt;&lt;sup&gt;31&lt;/sup&gt; and &lt;em&gt;H. volcanii&lt;/em&gt;&lt;sup&gt;27&lt;/sup&gt;. The total number of all genes with a detected TSS is shown as grey barplots and results from the sum of Nanopore-only predicted TSS and the intersection to the Illumina data. &lt;strong&gt;b&lt;/strong&gt;, Position of TSS is corrected for 12 nucleotides to calculate the length of 5&#x2019; untranslated regions (UTR) in the Nanopore data sets (purple). 5&#x2019; UTRs are compared to d(ifferential) RNA-Seq Illumina data sets (light-green). Median values are indicated by a black bar inside the distribution (compare Supplementary Fig. 6). &lt;strong&gt;c&lt;/strong&gt;, Length of 3&#x2019; UTRs is based on the prediction of transcription termination sites (TTS) and the comparison to annotated gene ends. Distribution of lengths is shown for Nanopore data sets (purple) and compared to a Term-Seq &lt;em&gt;E. coli&lt;/em&gt; Illumina data set (light-green)&lt;sup&gt;108&lt;/sup&gt;. &lt;strong&gt;d&lt;/strong&gt;, MEME analysis&lt;sup&gt;45&lt;/sup&gt; of extracted sequences upstream of Nanopore-predicted TSS reveals bacterial (position &#x2212;10) and archaeal-specific promoter elements (BRE: B-recognition element, recognized by transcription factor B, TATA: TATA-box bound by the TATA-binding protein TBP), therefore validating the positions of predicted TSS. &lt;strong&gt;e&lt;/strong&gt;, Nucleotide enrichment meta analysis was carried out by comparing the genomic sequences surrounding the TTS (&#x2212;45 to +45) to randomly selected intergenic positions of the respective organism (n: 10000) (Terminator motifs in Supplementary Fig. 7).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Detection of transcript boundaries.</span><p id="p-37" class="first-child"><strong>a</strong>, Primary transcription start sites (TSS) were predicted based on Nanopore reads and compared to Illumina d(ifferential) RNA-Seq data from published data sets for <em>E. coli</em><sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>, <em>P. furiosus</em><sup><a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">31</a></sup> and <em>H. volcanii</em><sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup>. The total number of all genes with a detected TSS is shown as grey barplots and results from the sum of Nanopore-only predicted TSS and the intersection to the Illumina data. <strong>b</strong>, Position of TSS is corrected for 12 nucleotides to calculate the length of 5’ untranslated regions (UTR) in the Nanopore data sets (purple). 5’ UTRs are compared to d(ifferential) RNA-Seq Illumina data sets (light-green). Median values are indicated by a black bar inside the distribution (compare Supplementary Fig. 6). <strong>c</strong>, Length of 3’ UTRs is based on the prediction of transcription termination sites (TTS) and the comparison to annotated gene ends. Distribution of lengths is shown for Nanopore data sets (purple) and compared to a Term-Seq <em>E. coli</em> Illumina data set (light-green)<sup><a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a></sup>. <strong>d</strong>, MEME analysis<sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup> of extracted sequences upstream of Nanopore-predicted TSS reveals bacterial (position −10) and archaeal-specific promoter elements (BRE: B-recognition element, recognized by transcription factor B, TATA: TATA-box bound by the TATA-binding protein TBP), therefore validating the positions of predicted TSS. <strong>e</strong>, Nucleotide enrichment meta analysis was carried out by comparing the genomic sequences surrounding the TTS (−45 to +45) to randomly selected intergenic positions of the respective organism (n: 10000) (Terminator motifs in Supplementary Fig. 7).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h4>Transcription termination sites</h4><p id="p-38">Native RNA reads are sequenced in the 3’ to 5’ direction, which is a major advantage in the detection of termination sites as any bias introduced after polyadenylation can be excluded. Currently, no reference data sets for the archaeal organisms were available, the distribution of 3’ untranslated regions (3’UTRs) in <em>E. coli</em> ONT data closely resembles the data from a previous Term-seq study<sup><a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">26</a></sup>. Untranslated regions at the 3’end of an annotated transcript are longer and more uniformly distributed in <em>E. coli</em> than in the two archaeal species (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2c</a>). In total, 1321 TTS in <em>E. coli</em>, 856 in <em>P. furiosus</em> and 1461 in <em>H. volcanii</em> were analysed (positions of TTS in TEX-treated samples in Supplementary Table 4). A meta-analysis of all TTS surrounding regions revealed different sequence-dependent termination mechanisms that were confirmed using motif scanning (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2e</a>, see Supplementary Fig. 7a-d). Our data suggest that transcription in <em>P. furiosus</em> is terminated by a double-stretch of Uridines that are distributed over a length of 22 nt, a finding that is in line with the terminator sequences detected by Term-Seq in <em>S. acidocaldarius</em><sup><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a></sup> and similar to the U(8) sequence in <em>Thermococcus kodakarensis</em> by an <em>in vivo</em> reporter assays<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup>. The termination motif found in <em>H. volcanii</em> is a (U)4-sequence and located right before the TTS (see Supplementary Fig. 7d). The motif locations for both <em>Haloferax</em> and <em>Pyrococcus</em> ONT sets suggest that accurate TTS detection of transcripts terminated by poly(U) stretches is currently not possible. We observed that homopolymer sequences are removed during trimming of the reads, which leads to TTS positions that are positioned upstream of the poly(U) signal. Analysing individual transcripts in <em>H. volcanii</em> and <em>P. furiosus</em>, we found that a single transcript can exhibit diverse 3’ ends. This is true for the Pilin transcript in <em>H. volcanii</em> and the Alba transcript in <em>P. furiosus</em>, respectively (Supplementary Fig. 8). Both genes are highly expressed and some transcripts carry extended 3’ UTRs. While the majority of transcripts are terminating at the first poly(U) stretch, a subset of transcripts is substantially longer and terminates at a second poly(U) termination signal (Supplementary Figure 8). Interestingly, homogeneous short poly(U) signals are found both at the canonical termination site and the termination site of the elongated 3’UTR in the case of the Pilin transcript in <em>H. volcanii</em>. In contrast, the Alba gene transcripts in <em>P. furiosus</em> are terminated at both sites via long consecutive poly(U) stretches. Surveying the heterogeneity of the transcripts with an extended 3’UTR, we found a heterogeneous distribution in the length of the transcripts, which possibly indicates a step-wise trimming of the 3’UTR to yield the mature RNAs.</p><p id="p-39">As observed for <em>E. coli</em> termination sequences, Cytosines are enriched over Guanosine adjacent to the TTS in <em>P. furiosus</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2e</a>). Termination motifs detected in the <em>E. coli</em> data set cover both intrinsic (poly(U)) and Rho-dependent termination signatures (see Supplementary Fig. 7a,b)<sup><a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">26</a></sup>. We classified the stem-loop forming motif as Rho-dependent signatures, as these were not enriched in the poly(U) terminated genes. However, the stem-loops could potentially also represent processing or pause sites.</p></div></div><div id="sec-28" class="subsection"><h3>Annotation of large transcriptional units</h3><p id="p-40">Long-read sequencing of native full-length RNAs has the potential to improve and facilitate genome-wide transcriptional unit (TU) annotation, which can be visually explored in a genome browser coverage track (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). For whole-genome analysis, the annotation strategy was based on two major observations: First, during RNA preparation, RNA processing or degradation can occur, which limits the probability of sequencing an RNA in its native form as the percentage of full-length transcripts decreases with expected gene size (see Supplementary Fig. 9a). Secondly, we detected a decrease in coverage from the 3’ to 5’end of the RNA in all RNA classes except for the spike-in control (see Supplementary Fig. 9b), which is a limitation reported in the literature<sup><a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>. Therefore, we assume that not Nanopore sequencing but library preparation causes this problem. Based on this information, we developed a strategy that first collapses all overlapping reads and then splits them according to a significant coverage drop on the 3’ ends (annotation of TUs based on this strategy in Supplementary Table 5). We compared the results to database annotations and found that most of the differences are either caused by the low sequencing depth or by single-unit operons that have been collapsed and are now two-unit operons in the ONT data sets (see Supplementary Fig. 10a,b)<sup><a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup>. Even though limited read availability is a concern in all data sets, many large operons were detected for all organisms (see Supplementary Fig. 10c). In case of limited bioinformatical resources, TUs can be explored visually in a genome browser, which is mostly not possible for Illumina reads (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>, see Supplementary Fig. 11,12). It further allows a quantitative analysis of individual transcripts in relation to other elements of the TU and performs much better than pure bioinformatical prediction or molecular biology methods (RT-PCR) as shown for the archaellum operon in <em>P. furiosus</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>)<sup><a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">50</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. Here, it was possible to (i) detect multiple transcription units forming this cluster, (ii) confirm transcriptional start sites and (iii) to confirm that flaB0, the protein that is referred to as the major archaellin in <em>P. furiosus</em><sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a></sup>, is transcribed in large excess over the other archaellum genes. The largest TU cluster in <em>H. volcanii</em> consists of 25 ribosomal protein genes. Based on the native RNA-seq data, the analysis shows that this operon is split into two transcription units. This shows that the ONT native RNA sequencing method provides the opportunity to correctly annotate transcriptional units thereby outperforming the bioinformatics-only prediction as well as the visual inspection of Illumina coverage (see Supplementary Fig. 11). Besides, we confirmed the complex transcription pattern of the major ribosomal protein gene cluster in <em>E. coli</em> that stretches over more than 10 kB, including the accurate determination of TSS and TTS and a putative cleavage site in the secY gene (see Supplementary Fig. 12)<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcription unit (TU) annotation of the archaellum-operon in P. furiosus. a, Coverage of Nanopore reads is shown in the top panel. TU prediction is performed by detection and linkage of overlapping reads and splitting them according to a 3&#x2019; drop in coverage (see Supplementary Fig. 8). Predicted TUs are drawn with green boxes according to scale. b, Comparison to bioinformatical prediction using the DOOR2 database50. c, Genome annotation with abbreviated gene names, boxed drawn to scale and strand indicated by triangles31. d, Comparison to results from published RT-PCR experiments62. All transcripts detected are drawn by arrows. e, Operon prediction based on mixed Illumina-Seq (coverage in lower panel) and predicted by ANNOgesic31,109. The primary transcription start site (TSS) of the large transcriptional unit is highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Transcription unit (TU) annotation of the archaellum-operon in &lt;em&gt;P. furiosus&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Coverage of Nanopore reads is shown in the top panel. TU prediction is performed by detection and linkage of overlapping reads and splitting them according to a 3&#x2019; drop in coverage (see Supplementary Fig. 8). Predicted TUs are drawn with green boxes according to scale. &lt;strong&gt;b&lt;/strong&gt;, Comparison to bioinformatical prediction using the DOOR2 database&lt;sup&gt;50&lt;/sup&gt;. &lt;strong&gt;c&lt;/strong&gt;, Genome annotation with abbreviated gene names, boxed drawn to scale and strand indicated by triangles&lt;sup&gt;31&lt;/sup&gt;. &lt;strong&gt;d&lt;/strong&gt;, Comparison to results from published RT-PCR experiments&lt;sup&gt;62&lt;/sup&gt;. All transcripts detected are drawn by arrows. &lt;strong&gt;e&lt;/strong&gt;, Operon prediction based on mixed Illumina-Seq (coverage in lower panel) and predicted by ANNOgesic&lt;sup&gt;31,109&lt;/sup&gt;. The primary transcription start site (TSS) of the large transcriptional unit is highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F3.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F3.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <span class="caption-title">Transcription unit (TU) annotation of the archaellum-operon in <em>P. furiosus</em>.</span><p id="p-41" class="first-child"><strong>a</strong>, Coverage of Nanopore reads is shown in the top panel. TU prediction is performed by detection and linkage of overlapping reads and splitting them according to a 3’ drop in coverage (see Supplementary Fig. 8). Predicted TUs are drawn with green boxes according to scale. <strong>b</strong>, Comparison to bioinformatical prediction using the DOOR2 database<sup><a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">50</a></sup>. <strong>c</strong>, Genome annotation with abbreviated gene names, boxed drawn to scale and strand indicated by triangles<sup><a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">31</a></sup>. <strong>d</strong>, Comparison to results from published RT-PCR experiments<sup><a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a></sup>. All transcripts detected are drawn by arrows. <strong>e</strong>, Operon prediction based on mixed Illumina-Seq (coverage in lower panel) and predicted by ANNOgesic<sup><a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a></sup>. The primary transcription start site (TSS) of the large transcriptional unit is highlighted.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Detection and confirmation of rRNA processing in <em>E. coli</em></h3><p id="p-42">Next, we aimed to analyse the multi-step rRNA processing pathway which is the major RNA maturation pathway in any prokaryotic cell. We first focus on the <em>E. coli</em> data set as the processing of bacterial rRNAs is well characterized<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>–<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>. Ribosomal RNA in <em>E. coli</em> is transcribed from 7 independent rDNA operons, containing the mature rRNAs (16S, 23S and 5S rRNAs) and some tRNAs which are intersped by RNA spacers elements<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup>. In agreement with a previous study, transcription of <em>rrnC</em> from two promoters (transcription start sites at −293 and −175) was detected accurately (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a,b</a>)<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection and confirmation of rRNA processing sites in E. coli. a, Transcription of the rDNA locus (rrnC) is starting from two promoters (transcription start sites at &#x2212;293 and &#x2212;175)49. Precursor RNAs are cleaved by RNases (brown) at depicted positions64,65,73,80. Nanopore coverage is shown for TEX (+, blue) and NOTEX (-, blue) samples. b, Single-read analysis of precursor and mature rRNAs. Reads were filtered according to read start and stop positions and are color-coded by mature rRNAs (blue), RNaseIII-processed precursors (brown) and precursor rRNAs starting from the TSS (red). For each category the 40 longest reads were selected and are shown here as single lines. c, Histograms of read start positions and d, read end positions of 16S, 23S and 5S rRNA relative to annotated boundaries of mature rRNAs of TEX (+) and NOTEX (-) samples. Cleavage sites, transcription start sites and transcription boundaries are indicated by dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection and confirmation of rRNA processing sites in &lt;em&gt;E. coli&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Transcription of the rDNA locus (rrnC) is starting from two promoters (transcription start sites at &#x2212;293 and &#x2212;175)&lt;sup&gt;49&lt;/sup&gt;. Precursor RNAs are cleaved by RNases (brown) at depicted positions&lt;sup&gt;64,65,73,80&lt;/sup&gt;. Nanopore coverage is shown for TEX (+, blue) and NOTEX (-, blue) samples. &lt;strong&gt;b&lt;/strong&gt;, Single-read analysis of precursor and mature rRNAs. Reads were filtered according to read start and stop positions and are color-coded by mature rRNAs (blue), RNaseIII-processed precursors (brown) and precursor rRNAs starting from the TSS (red). For each category the 40 longest reads were selected and are shown here as single lines. &lt;strong&gt;c&lt;/strong&gt;, Histograms of read start positions and &lt;strong&gt;d&lt;/strong&gt;, read end positions of 16S, 23S and 5S rRNA relative to annotated boundaries of mature rRNAs of TEX (+) and NOTEX (-) samples. Cleavage sites, transcription start sites and transcription boundaries are indicated by dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F4.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F4.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <span class="caption-title">Detection and confirmation of rRNA processing sites in <em>E. coli</em>.</span><p id="p-43" class="first-child"><strong>a</strong>, Transcription of the rDNA locus (rrnC) is starting from two promoters (transcription start sites at −293 and −175)<sup><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a></sup>. Precursor RNAs are cleaved by RNases (brown) at depicted positions<sup><a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">64</a>,<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>,<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a></sup>. Nanopore coverage is shown for TEX (+, blue) and NOTEX (-, blue) samples. <strong>b</strong>, Single-read analysis of precursor and mature rRNAs. Reads were filtered according to read start and stop positions and are color-coded by mature rRNAs (blue), RNaseIII-processed precursors (brown) and precursor rRNAs starting from the TSS (red). For each category the 40 longest reads were selected and are shown here as single lines. <strong>c</strong>, Histograms of read start positions and <strong>d</strong>, read end positions of 16S, 23S and 5S rRNA relative to annotated boundaries of mature rRNAs of TEX (+) and NOTEX (-) samples. Cleavage sites, transcription start sites and transcription boundaries are indicated by dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The rRNA maturation process, which requires the action of well-defined ribonuclease activities, culminates by the formation of stoichiometric amounts of mature 16S, 23S, and 5S rRNAs<sup><a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">64</a>–<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a></sup>. Unexpectedly, the sequencing efficiency of mature 16S rRNA was lower than the 23S rRNA (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4a</a>). Moreover, although we could properly detect RNA of similar size or longer (see above), the short-lived full rDNA operon transcript detected in RNase III deficient strain<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup>, is not observed using our experimental set-up. In contrast, the downstream pre-rRNAs which are generated by the action of RNAse III were detected (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). Among these intermediates, the 17S pre-rRNA with 115 additional nt at the 5’ end and 33 nt at the 3’ end of the 16S rRNA, was identified (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4b</a>). Additional 16S rRNA processing sites (RNase E: −66, RNase G: 0 5’mature) are also detected (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4 b,c</a>). Recent studies were suggesting that the 3’end of the mature 16S rRNA is generated by 3’-5’ ribonucleases activity and/or endonucleolytic cleavage<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>,<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup>. In addition to the mature 16S rRNA 3’end, we could observe reads of decreasing length spanning from +33 nt (RNaseIII) to 0 (related to 16S rRNA 3’end), that could account for 3’-5’ exonuclease processing activity (or degradation during sample preparation). Together we could accurately identify all the known processing sites at nucleotide resolution in wildtype <em>E. coli</em>.</p></div><div id="sec-30" class="subsection"><h3>Insights into archaeal ribosomal RNA processing</h3><p id="p-45">In comparison to bacteria or eukaryotes, ribosomal RNA processing in archaea is still poorly understood<sup><a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">70</a>,<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>–<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></sup>. Our current knowledge suggests that the primary polycistronic rRNA precursor contains processing stems formed by the 5’-leader and 3’-trailer sequences surrounding the 16S and 23S rRNAs<sup><a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">72</a>,<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>,<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup>. In Euryarchaeota, the 16S and 23S rRNAs are additionally separated by the presence of an internal tRNA. In most archaea, the 16S and 23S rRNA processing stems contain a bulge-helix-bulge motif which is, in the context of intron-containing tRNA, recognized by the splicing endonuclease endA<sup><a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">75</a>–<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></sup>. Similar to intron-containing tRNA maturation, processing at the bulge-helix-bulge motifs results in the ligation of the resulting extremities, thereby generating the archaeal specific circular pre-16S and circular pre-23S rRNAs<sup><a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">76</a>,<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a></sup>. The molecular mechanisms by which the circular pre-rRNA intermediates are further processed into linear mature rRNA are unknown<sup><a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">76</a></sup>.</p><p id="p-46">By analysing single reads, we aimed to confirm and expand our knowledge on the poorly characterized multi-step ribosomal maturation process in <em>P. furiosus</em> and <em>H. volcanii</em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5a,b,c</a>, see Supplementary Fig. 13a,b)<sup><a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-73-5" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">75</a>,<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">80</a></sup>. Due to a more limited number of reads in the multiplexed <em>H. volcanii</em> samples, we have mostly focused on the <em>P. furiosus</em> dataset. As expected, almost all reads are categorized as fully maturated transcripts of the single 16S/23S rRNA cluster that do not contain an extended 5’ or 3’ region (see Supplementary Fig. 13c). Surprisingly, 5’ and 3’end positions did not precisely match the availablle annotations at NCBI (<a href="https://www.ncbi.nlm.nih.gov/genome/">https://www.ncbi.nlm.nih.gov/genome/</a>) or the archaeal genome browser (AGB, <a href="http://archaea.ucsc.edu">http://archaea.ucsc.edu</a>). In contrast, these mature rRNA extremities did match our independent experimental validations by primer extension analysis of the 5’ends of the 16S and 23S rRNAs of <em>H. volcanii</em> (Supplementary Fig. 15f). In addition, the majority of the expected processing sites, notably at the predicted or known bulge-helix-bulge motifs<sup><a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">78</a></sup>, were faithfully observed in <em>P. furiosus</em> and <em>H. volcanii</em> (see Supplementary Fig. 14,15).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ribosomal RNA processing variation in P. furiosus. a, Nanopore coverage of the rDNA locus in P. furiosus containing the tRNA-Ala. Single reads were filtered according to their 5&#x2019; and 3&#x2019; lengths (couting from 16S and 23S rRNA annotation) and are shown in the example read track from a snapshot taken with the integrated genome browser (IGV)110. Besides mature 16S rRNA and 23S rRNA, the remaining reads were classified as (1) a processing variant that entails leading sequence-tRNA-trailing sequence, (2) probably fragmented full-length precursor rRNAs, (3) leading sequence-tRNA, (4) leading sequence-tRNA-23S rRNA and (5) tRNA-23S rRNA (quantification shown in Supplementary Fig. 13c). Illumina coverage from mixed RNA-Seq is shown in the bottom panel31. b, Read classes are shown as processing sketches and color-coded as on the left. c, Maturation steps of the rRNA processing pathway as proposed for Archaeoglobus fulgidus49 with cleavage sites indicated by red arrows (full proposed pathway in Supplementary Fig. 13a). Processing variants found in P. furiosus (4, 5) potentially represent intermediate steps that refine the current rRNA maturation model in archaea (Supplementary Fig. 13b)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Ribosomal RNA processing variation in &lt;em&gt;P. furiosus&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Nanopore coverage of the rDNA locus in &lt;em&gt;P. furiosus&lt;/em&gt; containing the tRNA-Ala. Single reads were filtered according to their 5&#x2019; and 3&#x2019; lengths (couting from 16S and 23S rRNA annotation) and are shown in the example read track from a snapshot taken with the integrated genome browser (IGV)&lt;sup&gt;110&lt;/sup&gt;. Besides mature 16S rRNA and 23S rRNA, the remaining reads were classified as (1) a processing variant that entails leading sequence-tRNA-trailing sequence, (2) probably fragmented full-length precursor rRNAs, (3) leading sequence-tRNA, (4) leading sequence-tRNA-23S rRNA and (5) tRNA-23S rRNA (quantification shown in Supplementary Fig. 13c). Illumina coverage from mixed RNA-Seq is shown in the bottom panel&lt;sup&gt;31&lt;/sup&gt;. &lt;strong&gt;b&lt;/strong&gt;, Read classes are shown as processing sketches and color-coded as on the left. &lt;strong&gt;c&lt;/strong&gt;, Maturation steps of the rRNA processing pathway as proposed for &lt;em&gt;Archaeoglobus fulgidus&lt;/em&gt;&lt;sup&gt;49&lt;/sup&gt; with cleavage sites indicated by red arrows (full proposed pathway in Supplementary Fig. 13a). Processing variants found in &lt;em&gt;P. furiosus&lt;/em&gt; (4, 5) potentially represent intermediate steps that refine the current rRNA maturation model in archaea (Supplementary Fig. 13b).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F5.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F5.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <span class="caption-title">Ribosomal RNA processing variation in <em>P. furiosus</em>.</span><p id="p-47" class="first-child"><strong>a</strong>, Nanopore coverage of the rDNA locus in <em>P. furiosus</em> containing the tRNA-Ala. Single reads were filtered according to their 5’ and 3’ lengths (couting from 16S and 23S rRNA annotation) and are shown in the example read track from a snapshot taken with the integrated genome browser (IGV)<sup><a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a></sup>. Besides mature 16S rRNA and 23S rRNA, the remaining reads were classified as (1) a processing variant that entails leading sequence-tRNA-trailing sequence, (2) probably fragmented full-length precursor rRNAs, (3) leading sequence-tRNA, (4) leading sequence-tRNA-23S rRNA and (5) tRNA-23S rRNA (quantification shown in Supplementary Fig. 13c). Illumina coverage from mixed RNA-Seq is shown in the bottom panel<sup><a id="xref-ref-31-10" class="xref-bibr" href="#ref-31">31</a></sup>. <strong>b</strong>, Read classes are shown as processing sketches and color-coded as on the left. <strong>c</strong>, Maturation steps of the rRNA processing pathway as proposed for <em>Archaeoglobus fulgidus</em><sup><a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">49</a></sup> with cleavage sites indicated by red arrows (full proposed pathway in Supplementary Fig. 13a). Processing variants found in <em>P. furiosus</em> (4, 5) potentially represent intermediate steps that refine the current rRNA maturation model in archaea (Supplementary Fig. 13b).</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">Despite the high sequencing depth of the (pre-)rRNA, we did not detect a full-length precursor consisting of the 16S leading-16S-tRNA-23S-23S trailing elements in <em>P. furiosus and H. volcanii</em>, suggesting that early rRNA processing occurs rapidly in these cells. The remaining reads were grouped according to their 5’ leading and 3’ trailing lengths into five additional categories that are less abundant (with less than 1600 counts in each group): (1) a processing variant that entails the 16S rRNA leading/trailing sequence-tRNA-23S rRNA leading/trailing sequence, (2) probably fragmented full-length precursor rRNAs, (3) 16S rRNA leading/trailing sequence-tRNA, (4) 16S rRNA leading/trailing sequence-tRNA-23S rRNA and (5) tRNA-23S rRNA (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a>, see Supplementary Fig. 13). The early 16S rRNA leading/trailing sequence-tRNA-23S rRNA trailing sequence precursor (1) or 16S rRNA leading/trailing sequence (3) generated by cleavage and reciprocal ligation of the pre-16S and pre-23S rRNAs at the predicted bulge-helix-bulge motifs were detected very accurately, and are reminiscent of previous observations<sup><a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">76</a></sup> (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5</a>, see Supplementary Fig. 13d). Given the number of reads, the direction of ONT sequencing from 3’ to 5’ and the accurate mapping, it is unlikely that the additional putative rRNA precursors (variants 4, 5) carrying the leading sequence in combination with 23S rRNA and tRNA-23S rRNA are all arising from experimental artifact (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5a,b</a>) (see also Discussion). This processing variation could have been overlooked in the past in the analysed archaea, as these are only minor fractions of the transcripts present in the cell or are specific to <em>P. furiosus</em>. Shortened 23S rRNA reads (variants 4, 5) and the overall visible 3’ to 5’ coverage drop can most likely be explained by fragmentation and degradation during library preparation, secondary structures that prevent efficient loading into the nanopore or truncation of RNA reads caused by physical artifacts that have been described before and include problems during RNA translocation or disruptions of the ionic current signal<sup><a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">52</a>,<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">56</a></sup>. Furthermore, it cannot be excluded that non-mature reads are categorized as fragmented reads and will be assigned as mature 16S or 23S rRNA.</p><p id="p-49">Highly enriched positions that differ from the annotation of the mature rRNAs and cannot be assigned to a processing site may represent a wrong annotation of the rRNA locus or currently unknown processing sites in the two archaeal species<sup><a id="xref-ref-31-11" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a></sup>. To provide further examples that show the potential to describe maturation and processing events for prokaryotic rRNA, we sequenced an <em>H. volcanii</em> wildtype strain grown under low salt conditions (see Supplementary Fig. 15b,c)<sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup>. Under these conditions, a 16S rRNA precursor with an extended 5’UTR (position - 121 – 5’bulge) appears that is only enriched in this condition. Quantification by comparison with the NOTEX wildtype set confirms the previous detection of this rRNA variant in a gel-electrophoretic analysis of the rRNA (see Supplementary Fig. 15b)<sup><a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a></sup>.</p></div><div id="sec-31" class="subsection"><h3>Towards mapping of RNA base modifications</h3><p id="p-50">More than 160 types of modified bases have been described in RNAs so far<sup><a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a></sup>. In contrast to other sequencing techniques, Nanopore-based sequencing offers the possibility to detect base modifications directly, as these modifications lead to an electric current signal that differs from the expected theoretical distribution obtained by the unmodified nucleotides sequence<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></sup>. Based on this approach, we benchmarked the potential to detect known modification sites in 16S rRNA. Before expanding the analysis to the full-length mature rRNA, we focused on the dimethylation (m<sup>6</sup><sub>2</sub>A) introduced by the enzyme KsgA at position 1450/1451 in <em>H. volcanii</em><sup><a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup>. Using the <em>de novo</em> model in Tombo, a suite of tools for the identification of modified nucleotides from nanopore sequencing data (updated version from Stoiber et al.<sup><a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup>), the calculated probability of a modification was very high for the stretch of Guanosines adjacent to position 1450 (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6a</a>). Mapping to single-nucleotide resolution is difficult as more than one base contributes to the actual electric current signal in the nanopore<sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>. In the next step, a comparison of a wildtype sample to a deletion mutant of KsgA confirmed the absence of the dimethylation in the deletion strain and confirmed that the m<sup>6</sup><sub>2</sub>A modifications and not the homopolymer-G-stretch are responsible for the altered current signal (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6b</a>). The analysis further revealed a reduced signal variability at non-modified positions between the two samples in comparison to the theoretical distribution, which leads to less false positives in the statistical analysis. As the <em>de novo</em> model has a higher error rate compared to the sample-compare model, we generated two read sets by sorting full-length 16S rRNA transcripts according to their 5’ leading length to mature (n: 2744) and pre-rRNAs (n: 102) in the <em>Haloferax</em> wildtype data set (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6c</a>). The calculated base modification probabilities and electric current signal distributions for this model are very similar to the wildtype/Δ<em>ksgA</em> set, indicating that the two m<sup>6</sup><sub>2</sub>A modifications were not quantitatively introduced in the pre-16S-rRNA reads (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6d</a>). This finding is in line with a proposed “quality control” function of KsgA during late biogenesis of the small ribosomal subunit<sup><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>–<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup> and further highlights the potential to detect the introduction of base modifications at a different stage of rRNA maturation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of modified bases in 16S rRNA based on raw signal comparisons. a, Raw signal of reads (blue squiggles) mapping to 16S rRNA in H. volcanii are compared to the theoretical distribution of native non-modified RNA (grey distribution) using the de novo detection model in tombo in the upper track47. The m62A modification at position 1450/1451 (from 16S start) is indicated by an asterisks in the sequence track. The probability of each base to be modified (in %) is calculated and shown in the lower panel for the selected sequence. b, Position-specific boxplot comparison of signals from sequences surrounding the m62A modification in H. volcanii wildtype (blue) and the &#x394;KsgA mutant. The theoretical distribution of read signal is indicated by a grey distribution curve for every base. The probability is computed based on the comparison of the two samples. c, Strategy for generating a comparison data set: Reads mapping to the 16S rRNA are categorized according to their read start position from the mature annotated rRNA as mature or pre-16S-rRNAs. d, Sample comparison model as shown in panel b based on mature (blue) and pre-rRNAs (red). e, Heatmap for the probability of a base to be modified (blue, light-green, red) shown for the mature 16S rRNA. Probabilities were calculated bases on a de novo (D) model, a sample comparison model (S) using the strategy described in panel c and plotted for already known modification sites (L)49. Three of those modifcations in H. volcanii are shown enlarged in the upper panel. Data are summarized for H. volcanii, f, E. coli48 and g, P. furiosus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1638500777" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection of modified bases in 16S rRNA based on raw signal comparisons.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Raw signal of reads (blue squiggles) mapping to 16S rRNA in &lt;em&gt;H. volcanii&lt;/em&gt; are compared to the theoretical distribution of native non-modified RNA (grey distribution) using the de novo detection model in tombo in the upper track&lt;sup&gt;47&lt;/sup&gt;. The m&lt;sup&gt;6&lt;/sup&gt;&lt;sub&gt;2&lt;/sub&gt;A modification at position 1450/1451 (from 16S start) is indicated by an asterisks in the sequence track. The probability of each base to be modified (in %) is calculated and shown in the lower panel for the selected sequence. &lt;strong&gt;b&lt;/strong&gt;, Position-specific boxplot comparison of signals from sequences surrounding the m&lt;sup&gt;6&lt;/sup&gt;&lt;sub&gt;2&lt;/sub&gt;A modification in &lt;em&gt;H. volcanii&lt;/em&gt; wildtype (blue) and the &#x394;KsgA mutant. The theoretical distribution of read signal is indicated by a grey distribution curve for every base. The probability is computed based on the comparison of the two samples. &lt;strong&gt;c&lt;/strong&gt;, Strategy for generating a comparison data set: Reads mapping to the 16S rRNA are categorized according to their read start position from the mature annotated rRNA as mature or pre-16S-rRNAs. &lt;strong&gt;d&lt;/strong&gt;, Sample comparison model as shown in panel b based on mature (blue) and pre-rRNAs (red). &lt;strong&gt;e&lt;/strong&gt;, Heatmap for the probability of a base to be modified (blue, light-green, red) shown for the mature 16S rRNA. Probabilities were calculated bases on a de novo (D) model, a sample comparison model (S) using the strategy described in panel c and plotted for already known modification sites (L)&lt;sup&gt;49&lt;/sup&gt;. Three of those modifcations in &lt;em&gt;H. volcanii&lt;/em&gt; are shown enlarged in the upper panel. Data are summarized for &lt;em&gt;H. volcanii&lt;/em&gt;, &lt;strong&gt;f&lt;/strong&gt;, &lt;em&gt;E. coli&lt;/em&gt;&lt;sup&gt;48&lt;/sup&gt; and &lt;strong&gt;g&lt;/strong&gt;, &lt;em&gt;P. furiosus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F6.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F6.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/19/2019.12.18.880849/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <span class="caption-title">Detection of modified bases in 16S rRNA based on raw signal comparisons.</span><p id="p-51" class="first-child"><strong>a</strong>, Raw signal of reads (blue squiggles) mapping to 16S rRNA in <em>H. volcanii</em> are compared to the theoretical distribution of native non-modified RNA (grey distribution) using the de novo detection model in tombo in the upper track<sup><a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">47</a></sup>. The m<sup>6</sup><sub>2</sub>A modification at position 1450/1451 (from 16S start) is indicated by an asterisks in the sequence track. The probability of each base to be modified (in %) is calculated and shown in the lower panel for the selected sequence. <strong>b</strong>, Position-specific boxplot comparison of signals from sequences surrounding the m<sup>6</sup><sub>2</sub>A modification in <em>H. volcanii</em> wildtype (blue) and the ΔKsgA mutant. The theoretical distribution of read signal is indicated by a grey distribution curve for every base. The probability is computed based on the comparison of the two samples. <strong>c</strong>, Strategy for generating a comparison data set: Reads mapping to the 16S rRNA are categorized according to their read start position from the mature annotated rRNA as mature or pre-16S-rRNAs. <strong>d</strong>, Sample comparison model as shown in panel b based on mature (blue) and pre-rRNAs (red). <strong>e</strong>, Heatmap for the probability of a base to be modified (blue, light-green, red) shown for the mature 16S rRNA. Probabilities were calculated bases on a de novo (D) model, a sample comparison model (S) using the strategy described in panel c and plotted for already known modification sites (L)<sup><a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">49</a></sup>. Three of those modifcations in <em>H. volcanii</em> are shown enlarged in the upper panel. Data are summarized for <em>H. volcanii</em>, <strong>f</strong>, <em>E. coli</em><sup><a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">48</a></sup> and <strong>g</strong>, <em>P. furiosus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Next, we applied the described <em>de novo</em> and sample-compare model to the full 16S rRNA of all three organisms and compared the results to positions known from literature<sup><a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">49</a></sup>. Applying a probability threshold of 0.5, we found that 60% (<em>de novo</em>) and 1.89% (sample-compare, 28 bases) of the bases in the 16S rRNA are modified in <em>H. volcanii</em> (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Fig. 6e</a>), which emphasizes the better performance of the sample-compare model by the vastly reduced number of false-positivly detected RNA modifications. 10 of the 28 predicted bases (please note that approximately five bases contribute to the signal in the nanopore and that a base single modification therefore influences the current signal at the surrounding bases as well) can be assigned to known base modifications at positions 910, 1430 and 1450/1451<sup><a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">49</a></sup>. Using the same threshold for <em>E. coli</em> (<em>de novo</em>: 50.5%, sample-compare: 0.1%, <a id="xref-fig-6-9" class="xref-fig" href="#F6">Fig. 6f</a>) and <em>P. furiosus</em> (<em>de novo</em>: 64.9 %, sample-compare: 17.7 %, <a id="xref-fig-6-10" class="xref-fig" href="#F6">Fig. 6g</a>), better accuracy of the sample-compare model could be confirmed. Moreover, these data support the long-standing hypothesis that hyperthermophilic organisms might stabilize their rRNAs by a higher degree of RNA modifications<sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>,<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a></sup>.</p><p id="p-53">It is noteworthy that the predicted large number of modifications lead to a loss in read quality. This is for example seen when analysing the <em>Pyrococcus</em> data set, as the quality score is also based on the comparison of detected to theoretical read signal (compare Supplementary Fig. 3c). To look at this observation in more detail, we compared the results from the raw squiggle Tombo approach that is not dependent on mapping of the reads to the mapping properties of the reads. This methodology is based on an approach that was recently used to identify m<sup>6</sup>A RNA modifications from native RNA reads with an accuracy of ~90%<sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a></sup>. After mapping, the frequency for the respective nucleotide at every position in the 16S rRNA to be correct, wrong or deleted was calculated. Homopolymer stretches in the RNA increase the likelihood that the correct number of nucleotides is not correctly identified and this, in turn, leads to an increased number of deleted nucleotides, which is also visible in the <em>Haloferax</em> data sets at the G-stretch adjacent to the m<sup>6</sup><sub>2</sub>A modifications (Supplementary Fig, 16a,b). The overall frequency of wrongly assigned and deleted nucleotides was much higher in <em>P. furiosus</em> (16.4%) than in <em>E. coli</em> (10.7%) and <em>H. volcanii</em> (10.4%) (Supplementary Fig. 16c). Interrogating the full-length 16S rRNA, we found that miscalled positions are equally distributed among the whole transcript (Supplementary Fig. 16d,e,f). Statistical analysis of miscall-frequencies revealed a significant difference between already known modification sites and non-modified positions in <em>Haloferax</em> and <em>Escherichia</em> suggesting that even a very simple model can detect RNA base modifications when wrongly assigned bases considered. The situation is different for <em>Pyrococcus</em>: Native Nanopore data suggest that the rRNA is heavily modified. However, the number, exact positions and chemical identity of these modifications are currently unknown. Hence, an analysis of how often base modifications lead to wrongly identified nucleotides in the 16S rRNA sequence cannot be performed (Supplementary Fig. 16c,g).</p></div></div><div class="section" id="sec-32"><h2 class="">Discussion</h2><p id="p-54">Performing whole-genome native RNA-seq study in prokaryotes provided us with a wealth of information on transcriptional and post-transcriptional processes in <em>E. coli</em> and the archaeal model organisms <em>H. volcanii</em> and <em>P. furiosus</em>. Here, we will mostly discuss new biological insights that emerged from our study. Additionally, we will reflect on the advantages and disadvantages of Nanopore native RNA-seq.</p><div id="sec-33" class="subsection"><h3>Insights into transcriptional processes</h3><p id="p-55">Bacterial and archaeal transcription is an intensely studied molecular process and the mechanisms of basal transcription are well understood<sup><a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a></sup>. Native RNA sequencing allowed us to retrieve accurate information of transcript boundaries on both 5’ and 3’ ends. Our data show that 3’ UTRs length distributions are comparable between <em>E. coli, P. furiosus</em> and <em>H. volcanii</em> with the majority of mRNAs showing a length between 30 - 70 nt. Similar to bacteria, archaea encode large numbers of small non-coding RNAs<sup><a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a></sup>. However, many regulatory events that involve the regulation via small RNAs take place at bacterial 5’ UTRs<sup><a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a></sup>. We and others found that 5’ UTRs are significantly shorter in archaea supporting the idea that post-transcriptional regulation is mediated via the 3’ rather than the 5’ UTR in archaea<sup><a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a></sup>. Additionally, we determined transcription termination sites, which are well analysed for bacterial species but only a few studies focused on archaeal termination mechanisms, especially on the genome-wide level. In both archaeal species, poly(U) stretches were overrepresented at termination sites agreeing well with termination sequences found in <em>Sulfolobus</em> and <em>Methanosarcina</em><sup><a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">46</a></sup>. Interestingly, the majority of TTS found in <em>Pyrococcus</em> transcripts is composed of two U-stretches with at least five consecutive uridine bases while a subclass of <em>Haloferax</em> transcripts is almost exclusively terminated by a single U-stretch with four uridine bases. It has been shown that a five base U-stretch is sufficient to induce termination <em>in vitro</em><sup><a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>,<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a></sup>. Similar observations were described in a recent study by Berkemer et al, which identified a poly(U)4 stretch to be the termination signal in intergenic regions<sup><a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a></sup>. However, the current data set suggests that this short termination signal might be a specific feature for a subclass of <em>Haloferax</em> transcripts resembling the poly(U) termination motif found in <em>E. coli</em>. All other archaeal organisms (<em>P. furiosus</em>, <em>M. mazei</em>, <em>S. acidocaldarius</em>) investigated so far terminate transcription at multiple consecutive poly(U) stretches. Possibly, <em>Haloferax</em> relies on additional termination signals or yet unknown termination factors. A putative candidate is archaeal CPSF1 (aCPSF1), a recently described archaeal termination factor<sup><a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a></sup> that is widespread in archaea. aCPSF1 acts as ribonuclease that cleaves transcripts after a poly(U) stretch to trim transcripts and facilitates transcription termination in <em>Methanococcus maripaludis</em>. The arising 3’ UTR isoforms were detected using Term-seq analysis<sup><a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">96</a></sup>. We also observed heterogeneity in the case of the Pilin and Alba transcripts, respectively, that are distinguished by varying lengths of the 3’ UTR suggesting that aCPSF1 might trim a subset of genes in <em>H. volcanii</em> and <em>P. furiosus</em>. It is noteworthy that 3’ UTR isoforms were also detected in Term-seq studies with <em>Sulfolobus</em> and <em>Methanosarcina</em><sup><a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">46</a></sup>. However, in contrast to the Pilin and Alba transcripts, the 3’ UTR isoforms arise from termination at different sites of a single continues poly(U) stretch suggesting that the isoforms arise from a stochastic termination process of the RNA polymerases at an extended poly(U) stretch at the end of the gene. Taken together, these data demonstrate that a variety of termination mechanisms (that can even co-occur in the same cell) can be found in archaea ranging from stochastic intrinsic termination at extended poly(U) stretches <em>(Pyrococcus, Sulfolobus, Methanosarcina</em>), to abrupt termination at short uridine tracts (<em>H. volcanii)</em> and factor-dependent termination that results in trimming of the 3’UTR (<em>H. volcanii, P. furiosus, M. maripaludis</em>).</p><p id="p-56">In the context of transcription, the long and overlapping native RNA reads helped us to analyse the transcriptional landscape at multigene operons. More specifically, we focused on the archaellum operon as the transcription unit assignment remained ambiguous so far<sup><a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">62</a></sup>. In contrast to bioinformatical and Illumina RNA-seq-based predictions and attempts to unravel the TU via primer extension experiments, we found that the archaellum operon in <em>P. furiosus</em> is transcribed in multiple units with highly diverse abundances. The <em>fla</em>B0 gene encodes the major flagellin protein that forms the filament of the archaellum and therefore, the organism has to produce this protein in large quantities as apparent from the expression level<sup><a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">62</a></sup>. Interestingly, FlaD is expressed at comparably high levels as well as supporting the idea that FlaD is a major constituent of the archaellum in <em>P. furiosus</em>. It has been speculated that FlaD forms the cytosolic ring of the archaellum that anchors the filament in the outer membrane<sup><a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a></sup>. The identity and functional role of FlaD are, however, not known so far.</p></div><div id="sec-34" class="subsection"><h3>Insights into rRNA processing in archaea</h3><p id="p-57">In this study, we have assessed the suitability of native RNA sequencing to obtain information on the rRNA maturation pathway of different prokaryotes. Ribosomal RNA maturation proceeds via the coordinated and defined order of ribonucleases action (exonucleolytic and/or endonucleolytic cleavages) which generate pre-rRNA intermediates with defined premature rRNA sequences<sup><a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">66</a>,<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a>,<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>,<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a></sup>. The establishment of faithfull rRNA maturation maps in model organisms, like <em>E. coli</em>, <em>S. cerevisiae</em> or human cell culture has required numerous analyses over the past decades<sup><a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">66</a>,<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">74</a>,<a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">98</a>,<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">99</a></sup>, and remains a technical challenge. Therefore, methodologies that might accelerate the systematic analysis of rRNA maturation pathways across the tree of life, thereby enabling to unravel the diversity of rRNA maturation strategies need to be established. Beyond the identification of processing sites, the order of the processing events which can be, in part, deduced from co-occurrence analysis of the 5’and 3’extremities is of biological relevance<sup><a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">66</a>,<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">74</a>,<a id="xref-ref-98-3" class="xref-bibr" href="#ref-98">98</a>,<a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">99</a></sup>. Whereas we could confirm and extend our general view on the rRNA maturation pathway in archaea, the 3’-5’ processivity of Nanopore native RNA sequencing observed for rRNA impedes the accurate analysis of pre-rRNA extremities co-segregation (see <a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5</a> and Supplementary Fig. 9b). Nevertheless, we could confirm the presence of processing sites and pre-rRNA intermediates in the different organisms analysed, with the exception of the archaea specific circular-pre-rRNA intermediates<sup><a id="xref-ref-73-6" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-76-5" class="xref-bibr" href="#ref-76">76</a>,<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">78</a>,<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">79</a></sup>, which escape analysis due to the lack of 5’/3’ extremities. Among the identified pre-rRNA intermediates, precursor (4) observed in <em>P. furiosus</em>, which include ligation at the bhb motif of the upstream region of the 16S leader and downstream region of the 16S trailer sequences continuous to the downstream tRNA/23S sequences is of particular interest. The presence of this intermediate suggests that the 16S rRNA bulge-helix-bulge processing occurs prior to internal tRNA and 23S rRNA maturation. This observation is also in agreement with our recent <em>cis</em>-acting element analysis performed in <em>H. volcanii</em><sup><a id="xref-ref-73-7" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-76-6" class="xref-bibr" href="#ref-76">76</a>,<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">79</a></sup>. In fact, based on this analysis we have proposed a model by which 16S rRNA maturation proceeds and is required for the downstream maturation of the internal tRNA and 23S rRNA. Moreover, we have hypothesized that ligation of the 16S rRNA leader/trailer resulting from the 16S rRNA bulge-helix-bulge maturation process generates a putative new pre-rRNA intermediate which we could observe for the first time in <em>Pyrococcus</em> using native RNA sequencing. In addition, the presence of intermediates (1) and (3) support the idea that the maturation of the co-transcribed internal tRNA is inefficient or inhibited and may preferentially occur after processing of the 16S and 23S rRNA bulge-helix-bulge [suggested in<sup><a id="xref-ref-76-7" class="xref-bibr" href="#ref-76">76</a>,<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">78</a></sup>].</p><p id="p-58">The biological relevance of the additionally identified intermediates for rRNA maturation is yet unclear and will need to be appraised in the light of additional functional analysis. In conclusion, despite some intrinsic limitations, direct RNA sequencing can be a useful tool to approach intricated maturation pathway like rRNA maturation, and expand our understanding of RNA maturation in prokaryotes.</p></div><div id="sec-35" class="subsection"><h3>Towards the mapping of rRNA modification patterns</h3><p id="p-59">RNA modifications have been described already in the 50-60s, and has gained significant attention over the last years, under the generic term of epitranscriptome<sup><a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>–<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a></sup>. The high-throughput analysis of these post-transcriptional modifications remains challenging and mostly relies on indirect methods, like primer extension stops analysis or chemical recoding/derivation strategies<sup><a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a>,<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a></sup>. Native RNA sequencing may fill an important gap to systematically analyse RNA modifications on a genome-wide scale. However, global strategies enabling the faithful determination of RNA modification identity and position needs to be developed. Several recent focused analyses have explored different strategies to evaluate the capacity of ONT to accurately detect RNA modifications (e.g. m6A)<sup><a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>,<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>,<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a></sup>. In this study, we focused on the 16S rRNA which is known to carry different types of RNA modifications which are introduced at different positions and different stages of the small ribosomal subunit maturation<sup><a id="xref-ref-73-8" class="xref-bibr" href="#ref-73">73</a>,<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a></sup>. Our study provides evidence that RNA modification detection benefits of the use of unmodified/hypo-modified references (in agreement with recent studies<sup><a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">106</a></sup>). In this study, we have used a sample-compare approach analysing pre-rRNAs, which are expected to contain incomplete modification patterns, in comparison to mature rRNAs, which are expected to harbor a completed modification pattern. To validate our approach we have first focus on the almost universally conserved KsgA-dependent dimethylations of the 16S rRNA<sup><a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">82</a>,<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup>. Using our sample-compare strategy and KsgA deletion strain we could unambiguously provide <em>in vivo</em> evidence that the archaeal KsgA-dependent methylations of the 16S rRNA are completed at a late stage of the small ribosomal subunit biogenesis. This result is in agreement with previous studies done in yeast and bacteria<sup><a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">84</a>–<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">86</a></sup>.</p><p id="p-60">Moreover, our sample-compare approach also suggests an increased amount of rRNA modifications in the hyperthermophile <em>P. furiosus</em>, and a decrease amount of predicted rRNA modifications in halophile <em>H. volcanii</em> in comparison to <em>E. coli</em>. These differential modification patterns across archaea are in good agreement with previous studies and may reflect adaptation to the environmental conditions that these extremophilic archaea encounter<sup><a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">87</a>,<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">88</a></sup>. Unfortunately, the exact nature of these modifications can not be unveiled yet. To facilitate their high-throughput identification, future studies will require to develop and train algorithms improving the identification confidence of diverse RNA modifications.</p></div><div id="sec-36" class="subsection"><h3>Benefits and limitations of Nanopore-based native RNA sequencing</h3><p id="p-61">Taken together, a key advantage of the native RNA-seq approach is that multiple features can be addressed at once distinguishing the technique from the Illumina sequencing technology or biochemical assays. ONT sequencing does not require large scale equipment and is a fast method. Moreover, the method does not necessitate a reverse transcription step or PCR amplification thereby avoiding biases introduced by these enzymes. Due to the limitations of the sequencing read analysis platform, ONT sequencing does not accurately detect small RNAs yet. Additional limitations of the native RNA-seq technique are currently (i) the high amount of input RNA required (2-5 μg) to reach good coverage of the transcriptome without rRNA depletion, (ii) the 3’ bias during RNA sequencing (iii) limited throughput and (iv) limited possibilities for multiplexing. Although ONT sequencing has a comparably low sequencing accuracy, this did not pose a limitation for our analysis. Due to the extraordinary read length and the sensitivity to base modifications, ONT-based native RNA-seq can provide valuable insights into (r)RNA processing, (r)RNA modifications patterns and the transcription of large operons. Strikingly, ONT-based sequencing is a <em>bona fide</em> single-molecule method and hence molecular heterogeneity in the transcriptome can be analysed so that even minor RNA populations can be detected that are inevitably lost in ensemble sequencing approaches.</p></div></div><div class="section" id="sec-38"><h2 class="">Data availability</h2><p id="p-62">Sequencing data sets (gzipped raw FAST5 files) will be deposited in the Sequence Read Archive (SRA).</p></div><div class="section" id="sec-39"><h2 class="">Code availability</h2><p id="p-63">A detailed documentation and code of all essential analysis steps (used tools and custom Rscripts) are available from <a href="https://github.com/felixgrunberger/Native_RNAseq_Microbes">https://github.com/felixgrunberger/Native_RNAseq_Microbes</a>.</p></div><div class="section" id="sec-40"><h2 class="">Author contributions</h2><p id="p-64">F.G. established the nanopore workflow and performed all the bioinformatic analysis. F.G., R.K., M.J., R.R. and A.B. performed RNA extractions. M.F. helped to optimize the RNA treatment protocol. F.G. carried out library preparations and performed sequencing. F.G., R.K., M.J. carried out <em>H. volcanii</em> wildtype/Δ<em>ksgA</em> library preparations and sequencing. R.K. and S.F.-C. generated the KsgA deletion strain. R.K. performed primer extension analysis. F.G., S.F.-C. and D.G. designed the study, analysed and interpreted the data, and wrote the manuscript with the input of all authors. J.S., W.H., S.F.-C. and D.G. supervised the experiments. S.F.-C. and D.G. initiated and supervised the project.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-65">We gratefully acknowledge financial support by the Deutsche Forschungsgemeinschaft within the collaborative research center framework (CRC/SFB960) “RNP biogenesis: assembly of ribosomes and non-ribosomal RNPs and control of their function” [SFB960-TP7 to D.G.] [SFB960-TP-B13 to S.F.-C.]. The work was also supported by the DFG through grant So264/21 to J.S.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3">
<a href="https://github.com/felixgrunberger/Native_RNAseq_Microbes">https://github.com/felixgrunberger/Native_RNAseq_Microbes</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.1" data-doi="10.1146/annurev-genom-083115-022413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Levy</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">R. M</span></span>. <span class="cit-article-title">Advancements in Next-Generation Sequencing</span>. <abbr class="cit-jnl-abbrev">Annual review of genomics and human genetics</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">95</span>–<span class="cit-lpage">115</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genom-083115-022413</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.volume%253D17%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-083115-022413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-083115-022413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.2" data-doi="10.3389/fgene.2015.00348"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Escobar-Zepeda</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vera-Ponce de León</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sanchez-Flores</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">The Road to Metagenomics: From Microbiology to DNA Sequencing Technologies and Bioinformatics</span>. <abbr class="cit-jnl-abbrev">Frontiers in Genetics</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">348</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00348</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D6%26rft.spage%253D348%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.3" data-doi="10.1038/nrg2484"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerstein</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">RNA-Seq: a revolutionary tool for transcriptomics</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Genetics</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">57</span>–<span class="cit-lpage">63</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2484</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D63%26rft.atitle%253DRNA-Seq%253A%2Ba%2Brevolutionary%2Btool%2Bfor%2Btranscriptomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2484%26rft_id%253Dinfo%253Apmid%252F19015660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261866500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.4" data-doi="10.1016/j.molcel.2017.12.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hör</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorski</span>,  <span class="cit-name-given-names">S. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">Bacterial RNA Biology on a Genome Scale</span>. <abbr class="cit-jnl-abbrev">Molecular cell</abbr> <span class="cit-vol">70</span>, <span class="cit-fpage">785</span>–<span class="cit-lpage">799</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2017.12.023</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.volume%253D70%26rft.spage%253D785%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.12.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.12.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.5" data-doi="10.1016/j.mib.2010.09.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Croucher</span>,  <span class="cit-name-given-names">N. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Thomson</span>,  <span class="cit-name-given-names">N. R.</span></span> <span class="cit-article-title">Studying bacterial transcriptomes using RNA-seq</span>. <abbr class="cit-jnl-abbrev">Current opinion in microbiology</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">619</span>–<span class="cit-lpage">624</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mib.2010.09.009</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DCroucher%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D619%26rft.epage%253D624%26rft.atitle%253DStudying%2Bbacterial%2Btranscriptomes%2Busing%2BRNA-seq.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2010.09.009%26rft_id%253Dinfo%253Apmid%252F20888288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2010.09.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20888288&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.6" data-doi="10.1128/EC.00123-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowrousian</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Next-generation sequencing techniques for eukaryotic microorganisms: sequencing-based solutions to biological problems</span>. <abbr class="cit-jnl-abbrev">Eukaryotic cell</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1300</span>–<span class="cit-lpage">1310</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.00123-10</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00123-10%26rft_id%253Dinfo%253Apmid%252F20601439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI5LzkvMTMwMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.7" data-doi="10.1016/j.mib.2017.01.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saliba</span>,  <span class="cit-name-given-names">A.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">C Santos</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">New RNA-seq approaches for the study of bacterial pathogens</span>. <abbr class="cit-jnl-abbrev">Current opinion in microbiology</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">78</span>–<span class="cit-lpage">87</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mib.2017.01.001</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.volume%253D35%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2017.01.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2017.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.8" data-doi="10.1038/s41576-019-0150-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grzelak</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hadfield</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">RNA sequencing: the teenage years</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41576-019-0150-2</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-019-0150-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41576-019-0150-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.9" data-doi="10.1038/nbt.3242"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tilgner</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">736</span>–<span class="cit-lpage">742</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3242</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D33%26rft.spage%253D736%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.10" data-doi="10.1098/rstb.2019.0097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Volden</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vollmers</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-article-title">Realizing the potential of full-length transcriptome sequencing</span>. <abbr class="cit-jnl-abbrev">Philosophical transactions of the Royal Society of London. Series B, Biological sciences</abbr> <span class="cit-vol">374</span>, <span class="cit-fpage">20190097</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2019.0097</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2Btransactions%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon.%2BSeries%2BB%252C%2BBiological%2Bsciences%26rft.volume%253D374%26rft.spage%253D20190097%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2019.0097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2019.0097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.11" data-doi="10.1126/science.1162986"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eid</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Real-time DNA sequencing from single polymerase molecules</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">323</span>, <span class="cit-fpage">133</span>–<span class="cit-lpage">138</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1162986</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162986%26rft_id%253Dinfo%253Apmid%252F19023044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMC8xMzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xOS8yMDE5LjEyLjE4Ljg4MDg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.12" data-doi="10.1111/1755-0998.12324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mikheyev</span>,  <span class="cit-name-given-names">A. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tin</span>,  <span class="cit-name-given-names">M. M. Y</span></span>. <span class="cit-article-title">A first look at the Oxford Nanopore MinION sequencer</span>. <abbr class="cit-jnl-abbrev">Molecular ecology resources</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1097</span>–<span class="cit-lpage">1102</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12324</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Becology%2Bresources%26rft.volume%253D14%26rft.spage%253D1097%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12324%26rft_id%253Dinfo%253Apmid%252F25187008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25187008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.13" data-doi="10.1038/s41467-019-11272-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soneson</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-019-11272-z</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-11272-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-11272-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.14" data-doi="10.1101/525741"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Accurate detection of m6A RNA modifications in native RNA sequences</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/525741</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft_id%253Dinfo%253Adoi%252F10.1101%252F525741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1MjU3NDF2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.15" data-doi="10.1371/journal.pone.0210041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jain</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulroney</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garalde</span>,  <span class="cit-name-given-names">D. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Akeson</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Reading canonical and modified nucleobases in 16S ribosomal RNA using nanopore native RNA sequencing</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0216709</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0210041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0210041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.16" data-doi="10.1038/s41592-019-0617-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Workman</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nanopore native RNA sequencing of a human poly(A) transcriptome</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">1305</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41592-019-0617-2</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41592-019-0617-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41592-019-0617-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.17" data-doi="10.1101/483693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krautwurst</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamkiewicz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hölzer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marz</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Nanopore direct RNA sequencing reveals modification in full-length coronavirus genomes</span>. <abbr class="cit-jnl-abbrev">bioRxiv Genomics</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/483693</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1101%252F483693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0ODM2OTN2NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.18.880849v1.18" data-doi="10.1038/s41598-018-32615-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">M. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-32615-8</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-32615-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-32615-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.19" data-doi="10.1016/j.tim.2019.01.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boldogkői</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moldován</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balázs</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tombácz</span>,  <span class="cit-name-given-names">D</span></span>. <span class="cit-article-title">Long-Read Sequencing – A Powerful Tool in Viral Transcriptome Research</span>. <abbr class="cit-jnl-abbrev">Trends in Microbiology</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">578</span>–<span class="cit-lpage">592</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tim.2019.01.010</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BMicrobiology%26rft.volume%253D27%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2019.01.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2019.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.20" data-doi="10.1101/483693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Direct RNA nanopore sequencing of full-length coron-avirus genomes provides novel insights into structural variants and enables modification analysis</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-fpage">483693</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/483693</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F483693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0ODM2OTN2NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.21" data-doi="10.3389/fgene.2019.00834"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tombácz</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiple Long-Read Sequencing Survey of Herpes Simplex Virus Dynamic Transcriptome</span>. <abbr class="cit-jnl-abbrev">Frontiers in Genetics</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">834</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2019.00834</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D10%26rft.spage%253D834%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2019.00834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2019.00834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.18.880849v1.22" data-doi="10.3389/fgene.2019.00253"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of Transcriptome and Epitranscriptome in Plants Using PacBio Iso-Seq and Nanopore-Based Direct RNA Sequencing</span>. <abbr class="cit-jnl-abbrev">Frontiers in Genetics</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">253</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2019.00253</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D10%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2019.00253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2019.00253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bayega</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Transcriptome landscape of the developing olive fruit fly embryo delineated by Oxford Nanopore long-read RNA-Seq</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.24" data-doi="10.1038/ncomms16027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">8</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms16027</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D8%26rft.spage%253D16027%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms16027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms16027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rahimi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venø</span>,  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dupont</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kjems</span>,  <span class="cit-name-given-names">J</span></span>. <abbr class="cit-jnl-abbrev">Nanopore sequencing of full-length circRNAs in human and mouse brains reveals circRNA-specific exon usage and intron retention</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.26" data-doi="10.1093/nar/gky274"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dar</span>,  <span class="cit-name-given-names">D</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Sorek</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">High-resolution RNA 3-ends mapping of bacterial Rho-dependent transcripts</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">6797</span>–<span class="cit-lpage">6805</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky274</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253D6797%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.27" data-doi="10.1186/s12864-015-2294-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babski</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-wide identification of transcriptional start sites in the haloarchaeon Haloferax volcanii based on differential RNA-Seq (dRNA-Seq)</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2920-y</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2294-6%26rft_id%253Dinfo%253Apmid%252F26818753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2294-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26818753&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.28" data-doi="10.1371/journal.pone.0210041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laass</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of the transcriptome of Haloferax volcanii, grown under four different conditions, with mixed RNA-Seq</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">24</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0215986</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0210041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0210041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.29" data-doi="10.1186/s12859-015-0805-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Revisiting operons: An analysis of the landscape of transcriptional units in E. coli</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0805-8</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0805-8%26rft_id%253Dinfo%253Apmid%252F26538447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0805-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26538447&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.30" data-doi="10.1128/JB.02096-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomason</span>,  <span class="cit-name-given-names">M. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global transcriptional start site mapping using differential RNA sequencing reveals novel antisense RNAs in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Journal of Bacteriology</abbr> <span class="cit-vol">197</span>, <span class="cit-fpage">18</span>–<span class="cit-lpage">28</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.02096-14</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.02096-14%26rft_id%253Dinfo%253Apmid%252F25266388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTk3LzEvMTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xOS8yMDE5LjEyLjE4Ljg4MDg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.31" data-doi="10.3389/fmicb.2019.01603"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grünberger</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Next Generation DNA-Seq and Differential RNA-Seq Allow Re-annotation of the Pyrococcus furiosus DSM 3638 Genome and Provide Insights Into Archaeal Antisense Transcription</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2019.01603</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.01603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.01603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.32" data-doi="10.1016/S0723-2020(83)80011-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stetter</span>,  <span class="cit-name-given-names">K. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">König</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stackebrandt</span>,  <span class="cit-name-given-names">E</span></span>. <span class="cit-article-title">Pyrodictium gen. nov., a New Genus of Submarine Disc-Shaped Sulphur Reducing Archaebacteria Growing Optimally at 105°C</span>. <abbr class="cit-jnl-abbrev">Systematic and Applied Microbiology</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0723-2020(83)80011-3</span> (<span class="cit-pub-date">1983</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.stitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.aulast%253DStetter%26rft.auinit1%253DK.%2BO.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D535%26rft.epage%253D551%26rft.atitle%253DPyrodictium%2Bgen.%2Bnov.%252C%2Ba%2BNew%2BGenus%2Bof%2BSubmarine%2BDisc-Shaped%2BSulphur%2BReducing%2BArchaebacteria%2BGrowing%2BOptimally%2Bat%2B105%25C2%25B0C.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0723-2020%252883%252980011-3%26rft_id%253Dinfo%253Apmid%252F23194811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0723-2020(83)80011-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23194811&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RV78700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.33" data-doi="10.1038/nrg1504"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allers</span>,  <span class="cit-name-given-names">T</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Mevarech</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Archaeal genetics - the third way</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">58</span>–<span class="cit-lpage">73</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1504</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D73%26rft.atitle%253DArchaeal%2Bgenetics%2B-%2Bthe%2Bthird%2Bway.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1504%26rft_id%253Dinfo%253Apmid%252F15630422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1504&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15630422&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226113800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.34" data-doi="10.1093/nar/gkx1236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knüppel</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Insights into the evolutionary conserved regulation of Rio ATPase activity</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">1441</span>–<span class="cit-lpage">1456</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1236</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253D1441%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.35" data-doi="10.1016/0003-2697(87)90021-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chomczynski</span>,  <span class="cit-name-given-names">P</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Sacchi</span>,  <span class="cit-name-given-names">N</span></span>. <span class="cit-article-title">Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction</span>. <abbr class="cit-jnl-abbrev">Analytical biochemistry</abbr> <span class="cit-vol">162</span>, <span class="cit-fpage">156</span>–<span class="cit-lpage">159</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/abio.1987.9999</span> (<span class="cit-pub-date">1987</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DChomczynski%26rft.auinit1%253DP.%26rft.volume%253D162%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D159%26rft.atitle%253DSingle-step%2Bmethod%2Bof%2BRNA%2Bisolation%2Bby%2Bacid%2Bguanidinium%2Bthiocyanate-phenol-chloroform%2Bextraction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2697%252887%252990021-2%26rft_id%253Dinfo%253Apmid%252F2440339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2697(87)90021-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2440339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G774500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.36" data-doi="10.1038/s41467-018-05997-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boitano</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">T. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ettwiller</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-article-title">SMRT-Cappable-seq reveals complex operon variants in bacteria</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-05997-6</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D3676%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-05997-6%26rft_id%253Dinfo%253Apmid%252F30201986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-05997-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30201986&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.37" data-doi="10.12688/f1000research.10571.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weirather</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis</span>. <abbr class="cit-jnl-abbrev">F1000Research</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">100</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.10571.2</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Research%26rft.volume%253D6%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.10571.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.10571.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.38" data-doi="10.1093/nar/gkj405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Riley</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Escherichia coli K-12: a cooperatively developed annotation snapshot--2005</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkj405</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DRiley%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DEscherichia%2Bcoli%2BK-12%253A%2Ba%2Bcooperatively%2Bdeveloped%2Bannotation%2Bsnapshot--2005%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj405%26rft_id%253Dinfo%253Apmid%252F16397293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16397293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234782300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.39" data-doi="10.1371/journal.pone.0009605"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hartman</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The complete genome sequence of Haloferax volcanii DS2, a model archaeon</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e9605</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0009605</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHartman%26rft.auinit1%253DA.%2BL.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9605%26rft.epage%253De9605%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2BHaloferax%2Bvolcanii%2BDS2%252C%2Ba%2Bmodel%2Barchaeon.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009605%26rft_id%253Dinfo%253Apmid%252F20333302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20333302&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.40" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H</span></span>. <span class="cit-article-title">Minimap2: Pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty191</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.41" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.42" data-doi="10.1093/nar/gkz114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smyth</span>,  <span class="cit-name-given-names">G. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W</span></span>. <span class="cit-article-title">The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkz114</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.43" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">M. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anders</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">550</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.44" data-doi="10.1038/nmeth.3444"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loman</span>,  <span class="cit-name-given-names">N. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quick</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">J. T.</span></span> <span class="cit-article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">733</span>–<span class="cit-lpage">735</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3444</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D12%26rft.spage%253D733%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3444%26rft_id%253Dinfo%253Apmid%252F26076426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26076426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.45" data-doi="10.1093/nar/gkp335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">T. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MEME SUITE: tools for motif discovery and searching</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">W202</span>–<span class="cit-lpage">8</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp335</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp335%26rft_id%253Dinfo%253Apmid%252F19458158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19458158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267889100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.46" data-doi="10.1038/nmicrobiol.2016.143"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prasse</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitz</span>,  <span class="cit-name-given-names">R. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sorek</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Widespread formation of alternative 3’ UTR isoforms via transcription termination in archaea</span>. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">16143</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmicrobiol.2016.143</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft.volume%253D1%26rft.spage%253D16143%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmicrobiol.2016.143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmicrobiol.2016.143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stoiber</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">De novo Identification of DNA Modifications Enabled by Genome-Guided Nanopore Signal Processing</abbr> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.48" data-doi="10.1093/nar/gkx1030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boccaletto</span>,  <span class="cit-name-given-names">P</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">MODOMICS: a database of RNA modification pathways. 2017 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">D303</span>–<span class="cit-lpage">D307</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1030</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253DD303%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1030%26rft_id%253Dinfo%253Apmid%252F29106616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29106616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.49" data-doi="10.1186/1471-2164-9-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grosjean</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaspin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marck</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decatur</span>,  <span class="cit-name-given-names">W. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crécy-Lagard</span>,  <span class="cit-name-given-names">V</span></span>. <span class="cit-article-title">de. RNomics and Modomics in the halophilic archaea Haloferax volcanii: Identification of RNA modification genes</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-470</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DXi%26rft.auinit1%253DZ.%26rft.volume%253D9%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bthe%2Binteraction%2Bbetween%2Bthe%2Bendosymbiotic%2Bbacterium%2BWolbachia%2Band%2Bits%2BDrosophila%2Bhost.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-1%26rft_id%253Dinfo%253Apmid%252F18171476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18171476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257872300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.50" data-doi="10.1093/nar/gkt1048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">X</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">DOOR 2.0: Presenting operons and their functions through dynamic and integrated views</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">42</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1048</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1048%26rft_id%253Dinfo%253Apmid%252F24214966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24214966&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800096&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.51" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Usadel</span>,  <span class="cit-name-given-names">B</span></span>. <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.52" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S. L.</span></span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.53" data-doi="10.1186/s12864-017-3543-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bohlin</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eldholm</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pettersson</span>,  <span class="cit-name-given-names">J. H. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brynildsrud</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Snipen</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-article-title">The nucleotide composition of microbial genomes indicates differential patterns of selection on core and accessory genomes</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">151</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3543-7</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3543-7%26rft_id%253Dinfo%253Apmid%252F28187704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3543-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28187704&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.54" data-doi="10.1186/s13059-019-1727-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wick</span>,  <span class="cit-name-given-names">R. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Judd</span>,  <span class="cit-name-given-names">L. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">K. E.</span></span> <span class="cit-article-title">Performance of neural network basecalling tools for Oxford Nanopore sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-019-1727-y</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D20%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1727-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-019-1727-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.55" data-doi="10.1038/nmeth.4577"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garalde</span>,  <span class="cit-name-given-names">D. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">201</span>–<span class="cit-lpage">206</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4577</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4577%26rft_id%253Dinfo%253Apmid%252F29334379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29334379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.56" data-doi="10.1101/459529"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Workman</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nanopore native RNA sequencing of a human poly(A) transcriptome</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/459529</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F459529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0NTk1Mjl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.57" data-doi="10.1101/706002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nanopore direct RNA sequencing maps an Arabidopsis N6 methyladenosine epitranscriptome</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/706002</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F706002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI3MDYwMDJ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.58" data-doi="10.1038/s41564-019-0500-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Full-length RNA profiling reveals pervasive bidirectional transcription terminators in bacteria</span>. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41564-019-0500-z</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-019-0500-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41564-019-0500-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.59" data-doi="10.1128/JB.00982-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santangelo</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cubonová</span>,  <span class="cit-name-given-names">L.’u.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skinner</span>,  <span class="cit-name-given-names">K. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reeve</span>,  <span class="cit-name-given-names">J. N.</span></span> <span class="cit-article-title">Archaeal intrinsic transcription termination in vivo</span>. <abbr class="cit-jnl-abbrev">Journal of Bacteriology</abbr> <span class="cit-vol">191</span>, <span class="cit-fpage">7102</span>–<span class="cit-lpage">7108</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00982-09</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00982-09%26rft_id%253Dinfo%253Apmid%252F19749050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5MS8yMi83MTAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTkvMjAxOS4xMi4xOC44ODA4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.60" data-doi="10.1101/132274"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Reading canonical and modified nucleotides in 16S ribosomal RNA using nanopore direct RNA sequencing</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/132274</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F132274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxMzIyNzR2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.61" data-doi="10.3389/fmicb.2019.00260"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wongsurawat</span>,  <span class="cit-name-given-names">T</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid sequencing of multiple RNA viruses in their native form</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2019.00260</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.00260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.00260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.62" data-doi="10.3389/fmicb.2014.00695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Näther-Schindler</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schopf</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellack</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rachel</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wirth</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Pyrococcus furiosus flagella: biochemical and transcriptional analyses identify the newly detected flaB0 gene to encode the major flagellin</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">695</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2014.00695</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft.volume%253D5%26rft.spage%253D695%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2014.00695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2014.00695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.63" data-doi="10.1371/journal.pgen.1002782"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lioliou</span>,  <span class="cit-name-given-names">E</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Global regulatory functions of the Staphylococcus aureus endoribonuclease III in gene expression</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e1002782</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002782</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLioliou%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002782%26rft.epage%253De1002782%26rft.atitle%253DGlobal%2Bregulatory%2Bfunctions%2Bof%2Bthe%2BStaphylococcus%2Baureus%2Bendoribonuclease%2BIII%2Bin%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002782%26rft_id%253Dinfo%253Apmid%252F22761586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761586&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.64" data-doi="10.1016/j.jmb.2018.04.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gupta</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denny</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Culver</span>,  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-article-title">Characterization of 16S rRNA Processing with Pre-30S Subunit Assembly Intermediates from E. coli</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2018.04.009</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BBiology%26rft.volume%253D430%26rft.spage%253D1745%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2018.04.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2018.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.65" data-doi="10.1146/annurev-biochem-062608-160432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shajani</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sykes</span>,  <span class="cit-name-given-names">M. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Williamson</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-article-title">Assembly of bacterial ribosomes</span>. <abbr class="cit-jnl-abbrev">Annual review of biochemistry</abbr> <span class="cit-vol">80</span>, <span class="cit-fpage">501</span>–<span class="cit-lpage">526</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-biochem-062608-160432</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DShajani%26rft.auinit1%253DZ.%26rft.volume%253D80%26rft.spage%253D501%26rft.epage%253D526%26rft.atitle%253DAssembly%2Bof%2Bbacterial%2Bribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-062608-160432%26rft_id%253Dinfo%253Apmid%252F21529161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-062608-160432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21529161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292895600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.66" data-doi="10.1080/10409238.2019.1651816"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bechhofer</span>,  <span class="cit-name-given-names">D. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Deutscher</span>,  <span class="cit-name-given-names">M. P.</span></span> <span class="cit-article-title">Bacterial ribonucleases and their roles in RNA metabolism</span>. <abbr class="cit-jnl-abbrev">Critical reviews in biochemistry and molecular biology</abbr> <span class="cit-vol">54</span>, <span class="cit-fpage">242</span>–<span class="cit-lpage">300</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10409238.2019.1651816</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bbiochemistry%2Band%2Bmolecular%2Bbiology%26rft.volume%253D54%26rft.spage%253D242%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10409238.2019.1651816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10409238.2019.1651816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.67" data-doi="10.1093/nar/29.1.181"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klappenbach</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saxman</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">T. M.</span></span> <span class="cit-article-title">rrndb: the Ribosomal RNA Operon Copy Number Database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">184</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/29.1.181</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKlappenbach%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D184%26rft.atitle%253Drrndb%253A%2Bthe%2BRibosomal%2BRNA%2BOperon%2BCopy%2BNumber%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.1.181%26rft_id%253Dinfo%253Apmid%252F11125085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.1.181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11125085&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166360300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.68" data-doi="10.1371/journal.pone.0144697"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maeda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimada</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishihama</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Semsey</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Strength and Regulation of Seven rRNA Promoters in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e0144697</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0144697</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0144697%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0144697%26rft_id%253Dinfo%253Apmid%252F26717514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0144697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26717514&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>,  <span class="cit-name-given-names">S</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">O. L.</span></span> <span class="cit-article-title">Visualization of ribosomal ribonucleic acid synthesis in a ribonuclease III-Deficient strain of Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Journal of Bacteriology</abbr> <span class="cit-vol">132</span>, <span class="cit-fpage">718</span>–<span class="cit-lpage">722</span> (<span class="cit-pub-date">1977</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHofmann%26rft.auinit1%253DS%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D718%26rft.epage%253D722%26rft.atitle%253DVisualization%2Bof%2Bribosomal%2Bribonucleic%2Bacid%2Bsynthesis%2Bin%2Ba%2Bribonuclease%2BIII-Deficient%2Bstrain%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Apmid%252F334750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNzE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTkvMjAxOS4xMi4xOC44ODA4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.70" data-doi="10.1016/j.molcel.2012.11.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacob</span>,  <span class="cit-name-given-names">A. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Köhrer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davies</span>,  <span class="cit-name-given-names">B. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">RajBhandary</span>,  <span class="cit-name-given-names">U. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">G. C.</span></span> <span class="cit-article-title">Conserved bacterial RNase YbeY plays key roles in 70S ribosome quality control and 16S rRNA maturation</span>. <abbr class="cit-jnl-abbrev">Molecular cell</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">427</span>–<span class="cit-lpage">438</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2012.11.025</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.volume%253D49%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.11.025%26rft_id%253Dinfo%253Apmid%252F23273979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.11.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23273979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314792500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.71" data-doi="10.1074/jbc.C113.459172"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sulthana</span>,  <span class="cit-name-given-names">S</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Deutscher</span>,  <span class="cit-name-given-names">M. P.</span></span> <span class="cit-article-title">Multiple exoribonucleases catalyze maturation of the 3’ terminus of 16S ribosomal RNA (rRNA)</span>. <abbr class="cit-jnl-abbrev">The Journal of biological chemistry</abbr> <span class="cit-vol">288</span>, <span class="cit-fpage">12574</span>–<span class="cit-lpage">12579</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.C113.459172</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bbiological%2Bchemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C113.459172%26rft_id%253Dinfo%253Apmid%252F23532845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMTgvMTI1NzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xOS8yMDE5LjEyLjE4Ljg4MDg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.72" data-doi="10.1155/2013/614735"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yip</span>,  <span class="cit-name-given-names">W. S. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>,  <span class="cit-name-given-names">N. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baserga</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-article-title">Ribonucleoproteins in archaeal pre-rRNA processing and modification</span>. <abbr class="cit-jnl-abbrev">Archaea (Vancouver, B.C.)</abbr> <span class="cit-vol">2013</span>, <span class="cit-fpage">614735</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2013/614735</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchaea%2B%2528Vancouver%252C%2BB.C.%2529%26rft.volume%253D2013%26rft.spage%253D614735%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2013%252F614735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2013/614735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.18.880849v1.73"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Clouet-d’Orval</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Ferreira-Cerca</span>,  <span class="cit-name-given-names">S.</span></span> in <span class="cit-source">RNA Metabolism and Gene Expression in Archaea</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Clouet-d’Orval</span></span> (<span class="cit-publ-name">Springer International Publishing, Cham</span>, <span class="cit-pub-date">2017</span>), pp. <span class="cit-fpage">129</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.74" data-doi="10.1002/wrna.1269"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henras</span>,  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plisson-Chastang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Donohue</span>,  <span class="cit-name-given-names">M.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gleizes</span>,  <span class="cit-name-given-names">P.-E</span></span>. <span class="cit-article-title">An overview of pre-ribosomal RNA processing in eukaryotes</span>. <abbr class="cit-jnl-abbrev">Wiley interdisciplinary reviews. RNA</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">225</span>–<span class="cit-lpage">242</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wrna.1269</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2Binterdisciplinary%2Breviews.%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1269%26rft_id%253Dinfo%253Apmid%252F25346433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25346433&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.75" data-doi="10.1093/femsre/fuy016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clouet-D’Orval</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Insights into RNA-processing pathways and associated RNA-degrading enzymes in Archaea</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiology Reviews</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">579</span>–<span class="cit-lpage">613</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/femsre/fuy016</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BReviews%26rft.volume%253D42%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsre%252Ffuy016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsre/fuy016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.76" data-doi="10.1093/nar/30.4.921"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">T. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">RNomics in Archaea reveals a further link between splicing of archaeal introns and rRNA processing</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">921</span>–<span class="cit-lpage">930</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/30.4.921</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTang%26rft.auinit1%253DT.%2BH.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D921%26rft.epage%253D930%26rft.atitle%253DRNomics%2Bin%2BArchaea%2Breveals%2Ba%2Bfurther%2Blink%2Bbetween%2Bsplicing%2Bof%2Barchaeal%2Bintrons%2Band%2BrRNA%2Bprocessing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.4.921%26rft_id%253Dinfo%253Apmid%252F11842103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.4.921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11842103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174029600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebhardt</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dennis</span>,  <span class="cit-name-given-names">P. P.</span></span> <span class="cit-article-title">Substrate requirements for a novel archaeal endonuclease that cleaves within the 5’ external transcribed spacer of Sulfolobus acidocaldarius precursor rRNA</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>, <span class="cit-fpage">1373</span>–<span class="cit-lpage">1385</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRussell%26rft.auinit1%253DA.%2BG.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1373%26rft.epage%253D1385%26rft.atitle%253DSubstrate%2BRequirements%2Bfor%2Ba%2BNovel%2BArchaeal%2BEndonuclease%2BThat%2BCleaves%2BWithin%2Bthe%2B5%2527%2BExternal%2BTranscribed%2BSpacer%2Bof%2BSulfolobus%2Bacidocaldarius%2BPrecursor%2BrRNA%26rft_id%253Dinfo%253Apmid%252F10430568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzEzNzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xOS8yMDE5LjEyLjE4Ljg4MDg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.78" data-doi="10.1093/nar/gkz1156"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jüttner</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A versatile cis-acting element reporter system to study the function, maturation and stability of ribosomal RNA mutants in archaea</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkz1156</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz1156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz1156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.79" data-doi="10.1093/nar/gkr1009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edelheit</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sorek</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Transcriptome-wide discovery of circular RNAs in Archaea</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">3131</span>–<span class="cit-lpage">3142</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1009</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1009%26rft_id%253Dinfo%253Apmid%252F22140119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22140119&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303164400034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferreira-Cerca</span>,  <span class="cit-name-given-names">S.</span></span> <abbr class="cit-jnl-abbrev">RNA Metabolism and Gene Expression in Archaea - Chapter6 - Life and Death of Ribosomes in Archaea</abbr> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.81" data-doi="10.1101/487819"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wongsurawat</span>,  <span class="cit-name-given-names">T</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Decoding the Epitranscriptional Landscape from Native RNA Sequences</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">487819</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/487819</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D17%26rft.spage%253D487819%26rft_id%253Dinfo%253Adoi%252F10.1101%252F487819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/487819&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.82" data-doi="10.1261/rna.2310406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Farrell</span>,  <span class="cit-name-given-names">H. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pulicherla</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desai</span>,  <span class="cit-name-given-names">P. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rife</span>,  <span class="cit-name-given-names">J. P.</span></span> <span class="cit-article-title">Recognition of a complex substrate by the KsgA/Dim1 family of enzymes has been conserved throughout evolution</span>. <abbr class="cit-jnl-abbrev">RNA (New York, N.Y.)</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">725</span>–<span class="cit-lpage">733</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.2310406</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2310406%26rft_id%253Dinfo%253Apmid%252F16540698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvNzI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTkvMjAxOS4xMi4xOC44ODA4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.83" data-doi="10.1186/s13059-017-1381-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rang</span>,  <span class="cit-name-given-names">F. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>,  <span class="cit-name-given-names">W. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ridder</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">de. From squiggle to basepair: Computational approaches for improving nanopore sequencing read accuracy</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-018-1462-9</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D19%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1381-1%26rft_id%253Dinfo%253Apmid%252F29301551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1381-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29301551&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.84" data-doi="10.1126/science.1208245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Strunk</span>,  <span class="cit-name-given-names">B. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ribosome assembly factors prevent premature translation initiation by 40S assembly intermediates</span>. <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">333</span>, <span class="cit-fpage">1449</span>–<span class="cit-lpage">1453</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1208245</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1208245%26rft_id%253Dinfo%253Apmid%252F21835981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0OC8xNDQ5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTkvMjAxOS4xMi4xOC44ODA4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.85" data-doi="10.1038/nsmb.1408"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Farrell</span>,  <span class="cit-name-given-names">H. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rife</span>,  <span class="cit-name-given-names">J. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Culver</span>,  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-article-title">A conserved rRNA methyltransferase regulates ribosome biogenesis</span>. <abbr class="cit-jnl-abbrev">Nature structural &amp; molecular biology</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">534</span>–<span class="cit-lpage">536</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.1408</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DXu%26rft.auinit1%253DZ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D534%26rft.epage%253D536%26rft.atitle%253DA%2Bconserved%2BrRNA%2Bmethyltransferase%2Bregulates%2Bribosome%2Bbiogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1408%26rft_id%253Dinfo%253Apmid%252F18391965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18391965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255587800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.86" data-doi="10.1128/MCB.18.4.2360"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lafontaine</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiss</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tollervey</span>,  <span class="cit-name-given-names">D</span></span>. <span class="cit-article-title">Yeast 18S rRNA dimethylase Dim1p: a quality control mechanism in ribosome synthesis?</span> <abbr class="cit-jnl-abbrev">Molecular and cellular biology</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">2360</span>–<span class="cit-lpage">2370</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mcb.18.4.2360</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLafontaine%26rft.auinit1%253DD.%2BL.%2BJ.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D2360%26rft.epage%253D2370%26rft.atitle%253DYeast%2B18S%2BrRNA%2BDimethylase%2BDim1p%253A%2Ba%2BQuality%2BControl%2BMechanism%2Bin%2BRibosome%2BSynthesis%253F%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.4.2360%26rft_id%253Dinfo%253Apmid%252F9528805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE4LzQvMjM2MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.87" data-doi="10.1186/s12864-015-1839-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dennis</span>,  <span class="cit-name-given-names">P. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tripp</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lui</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Randau</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-article-title">C/D box sRNA-guided 2’-O-methylation patterns of archaeal rRNA molecules</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1839-z</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1839-z%26rft_id%253Dinfo%253Apmid%252F26296872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1839-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26296872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.88" data-doi="10.1111/nyas.14060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gomes-Filho</span>,  <span class="cit-name-given-names">J. V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Randau</span>,  <span class="cit-name-given-names">L</span></span>. <span class="cit-article-title">RNA stabilization in hyperthermophilic archaea</span>. <abbr class="cit-jnl-abbrev">Annals of the New York Academy of Sciences</abbr> <span class="cit-vol">1447</span>, <span class="cit-fpage">88</span>–<span class="cit-lpage">96</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nyas.14060</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.volume%253D1447%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.14060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.14060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.89" data-doi="10.1016/j.jmb.2019.07.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ebright</span>,  <span class="cit-name-given-names">R. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Werner</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X</span></span>. <span class="cit-article-title">RNA Polymerase Reaches 60: Transcription Initiation, Elongation, Termination, and Regulation in Prokaryotes</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr> <span class="cit-vol">431</span>, <span class="cit-fpage">3945</span>–<span class="cit-lpage">3946</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2019.07.026</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BBiology%26rft.volume%253D431%26rft.spage%253D3945%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2019.07.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2019.07.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.90" data-doi="10.4161/rna.28452"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babski</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Small regulatory RNAs in Archaea</span>. <abbr class="cit-jnl-abbrev">RNA Biology</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">484</span>–<span class="cit-lpage">493</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/rna.28452</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiology%26rft.volume%253D11%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.28452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.28452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.91" data-doi="10.1093/femsre/fuv022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oliva</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sahr</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Buchrieser</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-article-title">Small RNAs, 5’ UTR elements and RNA-binding proteins in intracellular bacteria: impact on metabolism and virulence</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiology Reviews</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">331</span>–<span class="cit-lpage">349</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/femsre/fuv022</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BReviews%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsre%252Ffuv022%26rft_id%253Dinfo%253Apmid%252F26009640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsre/fuv022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26009640&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.92" data-doi="10.3389/fmicb.2017.01276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">G.-X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">X.-P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Y.-C</span></span>. <span class="cit-article-title">Regulatory 3’ Untranslated Regions of Bacterial mRNAs</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1276</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2017.01276</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft.volume%253D8%26rft.spage%253D1276%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2017.01276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2017.01276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.93" data-doi="10.1093/nar/gkq135"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hirtreiter</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-etal">et al.</span> <span class="cit-article-title">Spt4/5 stimulates transcription elongation through the RNA polymerase clamp coiled-coil motif</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">4040</span>–<span class="cit-lpage">4051</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq135</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq135%26rft_id%253Dinfo%253Apmid%252F20197319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20197319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280563400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.94" data-doi="10.1093/nar/gkp928"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hirtreiter</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grohmann</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Werner</span>,  <span class="cit-name-given-names">F</span></span>. <span class="cit-article-title">Molecular mechanisms of RNA polymerase--the F/E (RPB4/7) complex is required for high processivity in vitro</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">585</span>–<span class="cit-lpage">596</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp928</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp928%26rft_id%253Dinfo%253Apmid%252F19906731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19906731&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274088800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.95"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berkemer</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Identification of RNA 3’ ends and termination sites in Haloferax volcanii</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.96"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yue</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">aCPSF1 controlled archaeal transcription termination: a prototypical eukaryotic model</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.97" data-doi="10.7554/eLife.27470"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Daum</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure and in situ organisation of the Pyrococcus furiosus archaellum machinery</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.27470</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.27470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.27470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.98" data-doi="10.1261/rna.049692.115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deutscher</span>,  <span class="cit-name-given-names">M. P</span></span>. <span class="cit-article-title">Twenty years of bacterial RNases and RNA processing: how we’ve matured</span>. <abbr class="cit-jnl-abbrev">RNA (New York, N.Y.)</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">597</span>–<span class="cit-lpage">600</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.049692.115</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DDeutscher%26rft.auinit1%253DM.%2BP.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D597%26rft.epage%253D600%26rft.atitle%253DTwenty%2Byears%2Bof%2Bbacterial%2BRNases%2Band%2BRNA%2Bprocessing%253A%2Bhow%2Bwe%2527ve%2Bmatured%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.049692.115%26rft_id%253Dinfo%253Apmid%252F25780155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNTk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTkvMjAxOS4xMi4xOC44ODA4NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.99" data-doi="10.1002/yea.320111607"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Venema</span>,  <span class="cit-name-given-names">J</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Tollervey</span>,  <span class="cit-name-given-names">D</span></span>. <span class="cit-article-title">Processing of pre-ribosomal RNA in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Yeast (Chichester, England)</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">1629</span>–<span class="cit-lpage">1650</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.320111607</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVenema%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D16%26rft.spage%253D1629%26rft.epage%253D1650%26rft.atitle%253DProcessing%2Bof%2Bpre-ribosomal%2BRNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320111607%26rft_id%253Dinfo%253Apmid%252F8720068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320111607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8720068&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TQ24100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.100" data-doi="10.1038/181254a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Littlefield</span>,  <span class="cit-name-given-names">J. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dunn</span>,  <span class="cit-name-given-names">D. B.</span></span> <span class="cit-article-title">Natural occurrence of thymine and three methylated adenine bases in several ribonucleic acids</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">181</span>, <span class="cit-fpage">254</span>–<span class="cit-lpage">255</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/181254a0</span> (<span class="cit-pub-date">1958</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLittlefield%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D181%26rft.issue%253D4604%26rft.spage%253D254%26rft.epage%253D255%26rft.atitle%253DNatural%2Boccurrence%2Bof%2Bthymine%2Band%2Bthree%2Bmethylated%2Badenine%2Bbases%2Bin%2Bseveral%2Bribonucleic%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F181254a0%26rft_id%253Dinfo%253Apmid%252F13504142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/181254a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13504142&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1958ZP87500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">101.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.18.880849v1.101" data-doi="10.1042/bj0720294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">J. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dunn</span>,  <span class="cit-name-given-names">D. B.</span></span> <span class="cit-article-title">The occurrence of methylated guanines in ribonucleic acids from several sources</span>. <abbr class="cit-jnl-abbrev">The Biochemical journal</abbr> <span class="cit-vol">72</span>, <span class="cit-fpage">294</span>–<span class="cit-lpage">301</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/bj0720294</span> (<span class="cit-pub-date">1959</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D72%26rft.issue%253D2%26rft.spage%253D294%26rft.epage%253D301%26rft.atitle%253DThe%2Boccurrence%2Bof%2Bmethylated%2Bguanines%2Bin%2Bribonucleic%2Bacids%2Bfrom%2Bseveral%2Bsources%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj0720294%26rft_id%253Dinfo%253Apmid%252F13662298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czoxMDoicHBiaW9jaGVtaiI7czo1OiJyZXNpZCI7czo4OiI3Mi8yLzI5NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE5LzIwMTkuMTIuMTguODgwODQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.102" data-doi="10.1146/annurev-genom-090413-025405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Mason</span>,  <span class="cit-name-given-names">C. E.</span></span> <span class="cit-article-title">The pivotal regulatory landscape of RNA modifications</span>. <abbr class="cit-jnl-abbrev">Annual review of genomics and human genetics</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">127</span>–<span class="cit-lpage">150</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genom-090413-025405</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.volume%253D15%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-090413-025405%26rft_id%253Dinfo%253Apmid%252F24898039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-090413-025405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24898039&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">103.</span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103. in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.103" data-doi="10.1080/15476286.2016.1251543"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">S</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Motorin</span>,  <span class="cit-name-given-names">Y</span></span>. <span class="cit-article-title">Next-generation sequencing technologies for detection of modified nucleotides in RNAs</span>. <abbr class="cit-jnl-abbrev">RNA Biology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1124</span>–<span class="cit-lpage">1137</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15476286.2016.1251543</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiology%26rft.volume%253D14%26rft.spage%253D1124%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1251543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1251543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.104" data-doi="10.1016/j.ymeth.2018.10.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sas-Chen</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Misincorporation signatures for detecting modifications in mRNA: Not as simple as it sounds</span>. <abbr class="cit-jnl-abbrev">Methods (San Diego, Calif.)</abbr> <span class="cit-vol">156</span>, <span class="cit-fpage">53</span>–<span class="cit-lpage">59</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ymeth.2018.10.011</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.volume%253D156%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2018.10.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2018.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.105" data-doi="10.1261/rna.072785.119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sathe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Einstein</span>,  <span class="cit-name-given-names">J. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yeo</span>,  <span class="cit-name-given-names">G. W.</span></span> <span class="cit-article-title">Direct RNA sequencing enables m6A detection in endogenous transcript isoforms at base specific resolution</span>. <abbr class="cit-jnl-abbrev">RNA (New York, N.Y.)</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.072785.119</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.072785.119%26rft_id%253Dinfo%253Apmid%252F31624092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvMTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xOS8yMDE5LjEyLjE4Ljg4MDg0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leger</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">RNA modifications detection by comparative Nanopore direct RNA sequencing</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.107" data-doi="10.1080/15476286.2016.1259781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">K. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Tuning the ribosome: The influence of rRNA modification on eukaryotic ribosome biogenesis and function</span>. <abbr class="cit-jnl-abbrev">RNA Biology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1138</span>–<span class="cit-lpage">1152</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15476286.2016.1259781</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiology%26rft.volume%253D14%26rft.spage%253D1138%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1259781%26rft_id%253Dinfo%253Apmid%252F27911188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1259781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27911188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.108" data-doi="10.1371/journal.pgen.1007741"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dar</span>,  <span class="cit-name-given-names">D</span></span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Sorek</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Extensive reshaping of bacterial operons by programmed mRNA decay</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007354</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007741%26rft_id%253Dinfo%253Apmid%252F29324765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007741&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29324765&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">S.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Förstner</span>,  <span class="cit-name-given-names">K. U.</span></span> <abbr class="cit-jnl-abbrev">ANNOgesic: A Swiss army knife for the RNA-Seq based annotation of bacterial/archaeal genomes</abbr> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">110.</span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110. in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.18.880849v1.110" data-doi="10.1093/bioinformatics/btw069"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Freese</span>,  <span class="cit-name-given-names">N. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Norris</span>,  <span class="cit-name-given-names">D. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Loraine</span>,  <span class="cit-name-given-names">A. E.</span></span> <span class="cit-article-title">Integrated genome browser: visual analytics platform for genomics</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">2089</span>–<span class="cit-lpage">2095</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw069</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw069%26rft_id%253Dinfo%253Apmid%252F27153568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153568&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F19%2F2019.12.18.880849.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.18.880849v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.18.880930v1" title="Cellular state determines the multimodal signaling response of single cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 19, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.18.880849v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.18.880849v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/12/19/2019.12.18.880849.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.18.880849v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.18.880849v1" class="active" data-icon-position="" data-hide-link-title="0">Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1051835" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kx9Tj_UCyD1zKY9qOev4LaN17Wa_i7nF0tgLlmzZYcg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1051835" id="node1051835--2" data-pisa="biorxiv;2019.12.18.880849v1" data-pisa-master="biorxiv;2019.12.18.880849" data-seqnum="1051835" data-apath="/biorxiv/early/2019/12/19/2019.12.18.880849.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Grünberger</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Knüppel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Jüttner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Fenk</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Borst</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Reichelt</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Winfried</span> <span class="nlm-surname">Hausner</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jörg</span> <span class="nlm-surname">Soppa</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sebastien</span> <span class="nlm-surname">Ferreira-Cerca</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Grohmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.18.880849; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.18.880849 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;title=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;title=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;text=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;title=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;t=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;title=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;title=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1051835" id="node1051835--3" data-pisa="biorxiv;2019.12.18.880849v1" data-pisa-master="biorxiv;2019.12.18.880849" data-seqnum="1051835" data-apath="/biorxiv/early/2019/12/19/2019.12.18.880849.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Nanopore-based native RNA sequencing provides insights into prokaryotic transcription, operon structures, rRNA maturation and modifications</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Grünberger</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Knüppel</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Jüttner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Fenk</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Borst</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Robert</span> <span class="nlm-surname">Reichelt</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Winfried</span> <span class="nlm-surname">Hausner</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Jörg</span> <span class="nlm-surname">Soppa</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sebastien</span> <span class="nlm-surname">Ferreira-Cerca</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Grohmann</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.18.880849; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.18.880849 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Gr%C3%BCnberger%2C+Felix&amp;rft.au=Kn%C3%BCppel%2C+Robert&amp;rft.au=J%C3%BCttner%2C+Michael&amp;rft.au=Fenk%2C+Martin&amp;rft.au=Borst%2C+Andreas&amp;rft.au=Reichelt%2C+Robert&amp;rft.au=Hausner%2C+Winfried&amp;rft.au=Soppa%2C+J%C3%B6rg&amp;rft.au=Ferreira-Cerca%2C+Sebastien&amp;rft.au=Grohmann%2C+Dina"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1051835/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1051835/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1051835/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1051835/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1051835/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1051835/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1051835/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1051835/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1051835/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1051835/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1051835/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1051835/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;count=horizontal&amp;via=&amp;text=Nanopore-based%20native%20RNA%20sequencing%20provides%20insights%20into%20prokaryotic%20transcription%2C%20operon%20structures%2C%20rRNA%20maturation%20and%20modifications&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.18.880849v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.18.880849v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
