<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/19/840090.atom" />
<meta name="HW.pisa" content="biorxiv;840090v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality" />
<meta name="DC.Identifier" content="10.1101/840090" />
<meta name="DC.Date" content="2020-01-19" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Rationale Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.

Methods Using high-resolution mass spectrometry, we identified &gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.

Main Results We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.

Conclusions Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children." />
<meta name="DC.Contributor" content="Charles J Sande" />
<meta name="DC.Contributor" content="Jacqueline M Waeni" />
<meta name="DC.Contributor" content="James M Njunge" />
<meta name="DC.Contributor" content="Martin N Mutunga" />
<meta name="DC.Contributor" content="Elijah Gicheru" />
<meta name="DC.Contributor" content="Nelson K Kibinge" />
<meta name="DC.Contributor" content="Agnes Gwela" />
<meta name="article:published_time" content="2020-01-19" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;h3&gt;Rationale&lt;/h3&gt;
&lt;p&gt;Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.&lt;/p&gt;&lt;h3&gt;Methods&lt;/h3&gt;
&lt;p&gt;Using high-resolution mass spectrometry, we identified &amp;gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.&lt;/p&gt;&lt;h3&gt;Main Results&lt;/h3&gt;
&lt;p&gt;We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &amp;gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.&lt;/p&gt;&lt;h3&gt;Conclusions&lt;/h3&gt;
&lt;p&gt;Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;840090v2" />
<meta name="citation_id" content="840090v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/840090v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/840090v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/840090v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090.full.pdf" />
<meta name="citation_doi" content="10.1101/840090" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="840090" />
<meta name="citation_author" content="Charles J Sande" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author_email" content="csande@kemri-wellcome.org" />
<meta name="citation_author" content="Jacqueline M Waeni" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author" content="James M Njunge" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author" content="Martin N Mutunga" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author" content="Elijah Gicheru" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author" content="Nelson K Kibinge" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_author" content="Agnes Gwela" />
<meta name="citation_author_institution" content="KEMRI-Wellcome Trust Research Programme" />
<meta name="citation_reference" content="Causes of severe pneumonia requiring hospital admission in children without HIV infection from Africa and Asia: the PERCH multi-country case-control study. Lancet (London, England) 394, 757–779, doi: 10.1016/s0140-6736(19)30721-4 (2019)." />
<meta name="citation_reference" content="Shi, T. et al. Global, regional, and national disease burden estimates of acute lower respiratory infections due to respiratory syncytial virus in young children in 2015: a systematic review and modelling study. Lancet (London, England), doi: 10.1016/s0140-6736(17)30938-8 (2017)." />
<meta name="citation_reference" content="McNally, L. M. et al. Effect of age, polymicrobial disease, and maternal HIV status on treatment response and cause of severe pneumonia in South African children: a prospective descriptive study. Lancet (London, England) 369, 1440–1451, doi: 10.1016/s0140-6736(07)60670-9 (2007)." />
<meta name="citation_reference" content="Chisti, M. J., Tebruegge, M., La Vincente, S., Graham, S. M. &amp; Duke, T. Pneumonia in severely malnourished children in developing countries - mortality risk, aetiology and validity of WHO clinical signs: a systematic review. Tropical medicine &amp; international health : TM &amp; IH 14, 1173–1189, doi: 10.1111/j.1365-3156.2009.02364.x (2009)." />
<meta name="citation_reference" content="Moisi, J. C. et al. Sensitivity of hospital-based surveillance for severe disease: a geographic information system analysis of access to care in Kilifi district, Kenya. Bulletin of the World Health Organization 89, 102–111, doi: 10.2471/blt.10.080796 (2011)." />
<meta name="citation_reference" content="Mejias, A. et al. Whole blood gene expression profiles to assess pathogenesis and disease severity in infants with respiratory syncytial virus infection. PLoS medicine 10, e1001549, doi: 10.1371/journal.pmed.1001549 (2013)." />
<meta name="citation_reference" content="Johnson, J. E., Gonzales, R. A., Olson, S. J., Wright, P. F. &amp; Graham, B. S. The histopathology of fatal untreated human respiratory syncytial virus infection. Mod Pathol 20, 108–119, doi: 10.1038/modpathol.3800725 (2007)." />
<meta name="citation_reference" content="Jochems, S. P. et al. Novel Analysis of Immune Cells from Nasal Microbiopsy Demonstrates Reliable, Reproducible Data for Immune Populations, and Superior Cytokine Detection Compared to Nasal Wash. PloS one 12, e0169805, doi: 10.1371/journal.pone.0169805 (2017)." />
<meta name="citation_reference" content="Connors, T. J. et al. Developmental Regulation of Effector and Resident Memory T Cell Generation during Pediatric Viral Respiratory Tract Infection. Journal of immunology (Baltimore, Md. : 1950) 201, 432–439, doi: 10.4049/jimmunol.1800396 (2018)." />
<meta name="citation_reference" content="Sande, C. J. et al. Untargeted analysis of the airway proteomes of children with respiratory infections using mass spectrometry based proteomics. Scientific reports 8, 13814, doi: 10.1038/s41598-018-32072-3 (2018)." />
<meta name="citation_reference" content="Sande, C. J. et al. Airway response to respiratory syncytial virus has incidental antibacterial effects. Nature communications 10, 2218, doi: 10.1038/s41467-019-10222-z (2019)." />
<meta name="citation_reference" content="Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nature immunology 18, 583–593, doi: 10.1038/ni.3693 (2017)." />
<meta name="citation_reference" content="Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC bioinformatics 14, 128, doi: 10.1186/1471-2105-14-128 (2013)." />
<meta name="citation_reference" content="Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic acids research 44, W90–97, doi: 10.1093/nar/gkw377 (2016)." />
<meta name="citation_reference" content="Lindemans, C. A. et al. Systemic eosinophil response induced by respiratory syncytial virus. Clinical and experimental immunology 144, 409–417, doi: 10.1111/j.1365-2249.2006.03084.x (2006)." />
<meta name="citation_reference" content="Kim, H. H., Lee, M. H. &amp; Lee, J. S. Eosinophil cationic protein and chemokines in nasopharyngeal secretions of infants with respiratory syncytial virus (RSV) bronchiolitis and non-RSV bronchiolitis. Journal of Korean medical science 22, 37–42, doi: 10.3346/jkms.2007.22.1.37 (2007)." />
<meta name="citation_reference" content="Harrison, A. M., Bonville, C. A., Rosenberg, H. F. &amp; Domachowske, J. B. Respiratory syncytical virus-induced chemokine expression in the lower airways: eosinophil recruitment and degranulation. American journal of respiratory and critical care medicine 159, 1918–1924, doi: 10.1164/ajrccm.159.6.9805083 (1999)." />
<meta name="citation_reference" content="Dimova-Yaneva, D., Russell, D., Main, M., Brooker, R. J. &amp; Helms, P. J. Eosinophil activation and cysteinyl leukotriene production in infants with respiratory syncytial virus bronchiolitis. Clinical and experimental allergy : journal of the British Society for Allergy and Clinical Immunology 34, 555–558, doi: 10.1111/j.1365-2222.2004.1918.x (2004)." />
<meta name="citation_reference" content="Everard, M. L. et al. Analysis of cells obtained by bronchial lavage of infants with respiratory syncytial virus infection. Archives of disease in childhood 71, 428–432 (1994)." />
<meta name="citation_reference" content="McNamara, P. S., Ritson, P., Selby, A., Hart, C. A. &amp; Smyth, R. L. Bronchoalveolar lavage cellularity in infants with severe respiratory syncytial virus bronchiolitis. Archives of disease in childhood 88, 922–926, doi: 10.1136/adc.88.10.922 (2003)." />
<meta name="citation_reference" content="Wang, S. Z. et al. Shedding of L-selectin and PECAM-1 and upregulation of Mac-1 and ICAM-1 on neutrophils in RSV bronchiolitis. The American journal of physiology 275, L983–989, doi: 10.1152/ajplung.1998.275.5.L983 (1998)." />
<meta name="citation_reference" content="Wang, S. Z. et al. The apoptosis of neutrophils is accelerated in respiratory syncytial virus (RSV)-induced bronchiolitis. Clinical and experimental immunology 114, 49–54, doi: 10.1046/j.1365-2249.1998.00681.x (1998)." />
<meta name="citation_reference" content="Kerrin, A. et al. Differential lower airway dendritic cell patterns may reveal distinct endotypes of RSV bronchiolitis. Thorax 72, 620–627, doi: 10.1136/thoraxjnl-2015-207358 (2017)." />
<meta name="citation_reference" content="Bem, R. A. et al. Activation of the granzyme pathway in children with severe respiratory syncytial virus infection. Pediatric research 63, 650–655, doi: 10.1203/PDR.0b013e31816fdc32 (2008)." />
<meta name="citation_reference" content="Gill, M. A. et al. Mobilization of plasmacytoid and myeloid dendritic cells to mucosal sites in children with respiratory syncytial virus and other viral respiratory infections. The Journal of infectious diseases 191, 1105–1115, doi: 10.1086/428589 (2005)." />
<meta name="citation_reference" content="Heidema, J. et al. CD8+ T cell responses in bronchoalveolar lavage fluid and peripheral blood mononuclear cells of infants with severe primary respiratory syncytial virus infections. Journal of immunology (Baltimore, Md. : 1950) 179, 8410–8417 (2007)." />
<meta name="twitter:title" content="In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Rationale Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.

Methods Using high-resolution mass spectrometry, we identified &gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.

Main Results We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.

Conclusions Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children." />
<meta name="og-title" property="og:title" content="In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/840090v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Rationale Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.

Methods Using high-resolution mass spectrometry, we identified &gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.

Main Results We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.

Conclusions Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-19" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/840090v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/840090v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/840090v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/840090v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1112616" />
    <title>In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"f-69B96-NskBC5yQZdz4YDcmQB8-kRHHAY-74Cn2UaY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1112616","apath":"\/biorxiv\/early\/2020\/01\/19\/840090.atom","pisa":"biorxiv;840090v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;840090v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/840090v2","title":"In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality","identifier":"node\/1112616"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c+EVHZkb1rP0RzsJmQpJCbCiADSOOjkRZaTm4mvW8unl\\\\\\\/68+oR4HSjs\\\\\\\/hWbiaYJgc3gor\\\\\\\/gppIgf\\\\\\\/7autgVmNkxTjiUeAlMORqPJkOJwo9YmkuRnz8sRnEvnkaDHyV61TIIYRxGTcP2dcOeCgWVu5TiDEJqS4EjH0JvvlL60+vFZOAlmftllYEhNVE1D+zmPDWUu9v8yLC6okBCb0eAbwWh6fFpjBv7lDwfsDb+e1pZln5e1ZZNdnw\\\\\\\/fanAhQgcJEsbLrTqyp71t1jQk89gOZDiHyaGBo\\\\\\\/ZQKTANkZbsAVFyosgHY1Ch+QkNYtpf5RFQ\\\\\\\/Tbi5A96bfrh4MAiDeRyB8qFDi\\\\\\\/DvHQxqrFDssiFDPASFfVt2XmbXedg969NoyI2b55kPXE5MU4lNcRSGkfAwNqlo2way9KrB+cKcmRY1JIKra0yZ+UdOSZDkfkojy2\\\\\\\/T1f73fl9NT47F5gEMJ1Eo+GJmhMLeyOd5CgvOqBthb7tVrqP\\\\\\\/b54BKbccZt1qzHh8a3cOcXr24\\\\\\\/BfOHMIIRRmzFw32T1a9Tg\\\\\\\/4x9jNFooEF\\\\\\\/qrci3W6sMncYFtiOUFHlLqBD+9NC4DcQlpy\\\\\\\/GXGPKtdzIBznXcMokg6nCU3vDN2a5+6r6moJ1rcvm9OgNofq8TEACuJVou0W0YwntAuve17wIJgL3IAznJ0UsOL1mKnRFE1SPC3nmeOl4Dy9UCqype6+egxegCogYalVlv2wjSs3B000P4KfujveSlfvmhAG35FxFUwMVduwUlhvpQMxDk79n1\\\\\\\/1LUtX\\\\\\\/7hYaikyMEBM6qc2XgSxDt+OclwKf7wCaIe11ntc2SQv3gCcV6oDm\\\\\\\/3cNUAqCJmQ\\\\\\\/im2mCFWCVAKxAIE3AYCxYFi8s3+O0mnH\\\\\\\/b2PzH0htSq9+gbzmQdrl8ErcJrQpW4UtwJZMMUiN4XJt5Mlc7NxVg\\\\\\\/2mGJbR+DTyEdX6ckjK+9JKf+Q1EV6XDLOKXu6N0HhjGYPbaS53UDyyxoEAPooABoGMGjhqGuDrOJmeUZ7RuqKh\\\\\\\/oun6t2t5FBMpS7D0xUhokv+uNpMxu+GpUDV9rk\\\\\\\/QEeUQGay0F3Es+I\\\\\\\/sB9i+lf8XAVAQ7aHE03O9z3TIyoLyr4RqJm67pfDxC49Cq1SMyOgsdXI1fC2xg+2ooZZ98Au99K3iibcJ0p+uKM81aHRpr3HkNf4GVkhQ8yWjKDNFb7Y4ODqRx4TREFYfcvnt8vieFjggVysiIkQTsFm\\\\\\\/\\\\\\\/cocCObTXagw9GonTEIN6bEFKjQ4eK5wy10QmUvX5jd79IG1iHv6xyNHGOLFhQn1zO0ib\\\\\\\/+jQ1cMHIxOvemfroHPxmHhW7PPH8cpZtbxfEDIo7rg4r9Vf\\\u0022,\\\u0022iv\\\u0022:\\\u0022tNvsEwIEFk\\\\\\\/AzTlJk9UctQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002266275a00ee080b4298c0d27810df47d9f35f55905b4a864741b559377fab3ada\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/840090v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1112616 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/840090v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XXJoAh4dmPD09PEVvqyXBcCA5qwgCQ_U4c32j0biTeA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1112616" id="node1112616" data-pisa="biorxiv;840090v2" data-pisa-master="biorxiv;840090" data-apath="/biorxiv/early/2020/01/19/840090.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charles J</span> <span class="nlm-surname">Sande</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jacqueline M</span> <span class="nlm-surname">Waeni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James M</span> <span class="nlm-surname">Njunge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin N</span> <span class="nlm-surname">Mutunga</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Elijah</span> <span class="nlm-surname">Gicheru</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nelson K</span> <span class="nlm-surname">Kibinge</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Agnes</span> <span class="nlm-surname">Gwela</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/840090 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1112616" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles J Sande </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BJ%2BSande%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sande%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BJ%2BSande%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:csande@kemri-wellcome.org" class="" data-icon-position="" data-hide-link-title="0">csande@kemri-wellcome.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jacqueline M Waeni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacqueline%2BM%2BWaeni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waeni%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacqueline%2BM%2BWaeni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James M Njunge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BM%2BNjunge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Njunge%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BM%2BNjunge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin N Mutunga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BN%2BMutunga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mutunga%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BN%2BMutunga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Elijah Gicheru </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elijah%2BGicheru%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gicheru%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElijah%2BGicheru%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nelson K Kibinge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nelson%2BK%2BKibinge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kibinge%20NK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANelson%2BK%2BKibinge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Agnes Gwela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>KEMRI-Wellcome Trust Research Programme</span>, Kilifi, <span class='nlm-country'>Kenya</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agnes%2BGwela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gwela%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAgnes%2BGwela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/840090v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/840090v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/840090v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/840090v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/840090v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/840090v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1112616" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1112616/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Rationale</strong> Pneumonia is a leading cause of mortality in infants and young children. The mechanisms that lead to mortality in these children are poorly understood. Studies of the cellular immunology of the infant airway have traditionally been hindered by the limited sample volumes available from the young, frail children who are admitted to hospital with pneumonia. This is further compounded by the relatively low frequencies of certain immune cell phenotypes that are thought to be critical to the clinical outcome of pneumonia. To address this, we developed a novel in-silico deconvolution method for inferring the frequencies of immune cell phenotypes in the airway of children with different survival outcomes using proteomic data.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Methods</strong> Using high-resolution mass spectrometry, we identified &gt; 1,000 proteins expressed in the airways of children who were admitted to hospital with clinical pneumonia. 61 of these children were discharged from hospital and survived for more than 365 days after discharge, while 19 died during admission. We used machine learning by random forest to derive protein features that could be used to deconvolve immune cell phenotypes in paediatric airway samples. We applied these phenotype-specific signatures to identify airway-resident immune cell phenotypes that were differentially enriched by survival status and validated the findings using a large retrospective pneumonia cohort.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Main Results</strong> We identified immune-cell phenotype classification features for 33 immune cell types. Eosinophil-associated features were significantly elevated in airway samples obtained from pneumonia survivors and were downregulated in children who subsequently died. To confirm these results, we analyzed clinical parameters from &gt;10,000 children who had been admitted with pneumonia in the previous 10 years. The results of this retrospective analysis mirrored airway deconvolution data and showed that survivors had significantly elevated eosinophils at admission compared to fatal pneumonia.</p></div><div id="sec-4" class="subsection"><p id="p-5"><strong>Conclusions</strong> Using a proteomics bioinformatics approach, we identify airway eosinophils as a critical factor for pneumonia survival in infants and young children.</p></div></div><div class="section" id="sec-5"><h2 class="">Introduction</h2><p id="p-6">Pneumonia is a leading cause of paediatric mortality word-wide. A recent study on the global burden of paediatric pneumonia conducted in seven countries found that viruses account for about 61% of all paediatric pneumonia infections, while about 27% of infection were attributed to bacterial pathogens, with RSV accounting for the largest etiological fraction of paediatric pneumonia<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. More than 90% of the deaths that occur due to pneumonia in children under 5, occur in low resource settings, mainly due to the lack of paediatric intensive care facilities<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Very young infants especially those with comorbidities such as HIV and malnutrition have a poor survival prognosis following pneumonia infection. HIV-infected infants who develop a pneumonia infection are up to 10 times more likely to die from the infection than non-HIV infected children<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>, while those with malnutrition are more than 15 times more likely to die after admission<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. The damage to the lungs caused by severe pneumonia appears to persist even after discharge from hospital, with recent estimates showing that post-discharge mortality in African children previously admitted to hospital with pneumonia being eight times greater than those discharged with other diagnoses<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>.</p><p id="p-7">The difference in the immunological response to pneumonia between children who succumb to infection and those who survive it is not clear. An improved understanding of the immunobiology of this elevated post infection mortality risk will be crucial in identifying prognostic biomarkers of poor outcome, which will be critical in guiding care decisions in the first critical hours after admission. Most studies on the mechanisms of severe pneumonia in infants have been done using blood samples<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, and whist these studies have provided significant insights into disease pathology, they might not fully recapitulate the immune response to infection in the airway. In the case of RSV, there has only been one study in the last 60 years that examined the lung samples of children who died from pneumonia. Archived post-mortem lung tissue from three children who died in the pre-intensivist era in the USA in the 1930s and 1940s was evaluated and airway obstruction with necrotic cells, fibrin, mucus, and fluid identified as a prominent feature in RSV lung infection<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Staining for immune cell populations was not technically feasible in these long term archival samples.</p><p id="p-8">Due to the paucity of mechanistic data on the immunological response to pneumonia in the airway there is broad interest in understanding the dynamics of airway-resident immune cells following pneumonia infection in infants. This effort has been hindered by the unsuitability of routine airway sampling techniques for conventional cytometric analyses. Samples from nasopharyngeal washings and naso- and oropharyngeal swabbing are the most common methods of sampling the airways of sick children, and whilst ideal for molecular diagnostics, they are less suitable for phenotyping of airway resident immune cells using conventional cytometry techniques. Sampling of the airway by these methods typically results in limited cell yields and the cells that are recovered are generally highly enriched for granulocytes, complicating detailed characterisation of less abundant phenotypes using traditional flow cytometry-based tools<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. Recent analysis of the cellular composition of upper airway by flow cytometry showed that the typical abundances of critical effector cells like T Cells, B Cells, Mast cells, Dendritic cells and NK cells to be less than 0.5% of all airway cells, while granulocytes were present at a median frequency of &gt;90%<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>.</p><p id="p-9">A potential way to address these shortcomings is to monitor changes in marker proteins that are uniquely or predominantly expressed by specific immune cell phenotypes and to use this information to infer the dynamics of the underlying cell types. We have recently described a high-resolution mass-spectrometry-based proteomics approach for characterizing the total proteome of the infant airway to a depth of more than 1,000 proteins<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. The airway proteome characterized by this technique represents an unbiased snapshot of the underlying cell populations and can be leveraged to infer changes in the frequencies of the contributing cell phenotypes. Here, we describe an in-silico immune cell phenotype deconvolution approach where we use protein markers that are uniquely or predominantly expressed by specific immune cell subsets to infer the dynamics of those phenotypes in airways of children who survived or died from pneumonia. Phenotype-classification features were derived from a previously published data set containing the individual proteomes of purified immune cell populations<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup> (deconvolution data set) and these were then applied to airway proteome data from children with different pneumonia outcomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> contains a graphical description of the study design). Using this information, we identified an eosinophil-related protein signature that was elevated in the airways of children who survived pneumonia but that was downregulated in fatal pneumonia and in well controls. We subsequently validated this association using a large retrospective pneumonia cohort of &gt;10,000 children.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the profile of airway-resident immune cells using flow cytometry and in-silico deconvolution. (A) To characterize the phenotype diversity of airway-resident immune cells, fresh naso- and oropharyngeal flocked swabs were obtained from children admitted to hospital with pneumonia and processed immediately by flow cytometry. Here the gating strategy is shown, where live, singlet cells were initially gated on CD45 expression, followed by fluorescent antibody staining for different cell surface markers. CD3+ and CD19-cells were gated as T-cells, CD66-, CD14+ and HLA-DR-cells were gated as macrophages/monocytes while neutrophils were gated on the basis of CD66 and CD16 double expression. The functional activity of neutrophils was characterized by co-culturing the cells with E.coli particles that were labelled with pHRhodo. Neutrophils that contained phagocytosed particles are shown in green in the overlay dot plot. The bottom represented FSC and SSC plots of samples obtained from 6 different infants and demonstrates a high level of diversity in the immune cell populations present in the airways of children with pneumonia. (B) In silico deconvolution analysis was based on high resolution mass spectrometry data obtained from the airway. Cells were obtained from the naso and oropharyngeal sites using separate swabs, which were both eluted in a common transport media. As shown in A above, these samples contained a broad diversity of immune cells including T-cells, monocytes/macrophages and neutrophils. These cells were isolated from the sample by centrifugation and processed for mass spectrometry analysis using a standard protocol (see methods). (C) In silico phenotype deconvolution was conducted initially by identifying phenotype specific protein features in a previously-published deconvolution data set using random forest feature selection. These features were then applied on airway proteome data set obtained from children admitted to hospital with severe pneumonia, and used to construct a detailed map of airway immunophenotypes and their associations with clinical outcomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309048648" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of the profile of airway-resident immune cells using flow cytometry and in-silico deconvolution. &lt;strong&gt;(A)&lt;/strong&gt; To characterize the phenotype diversity of airway-resident immune cells, fresh naso- and oropharyngeal flocked swabs were obtained from children admitted to hospital with pneumonia and processed immediately by flow cytometry. Here the gating strategy is shown, where live, singlet cells were initially gated on CD45 expression, followed by fluorescent antibody staining for different cell surface markers. CD3+ and CD19-cells were gated as T-cells, CD66-, CD14+ and HLA-DR-cells were gated as macrophages/monocytes while neutrophils were gated on the basis of CD66 and CD16 double expression. The functional activity of neutrophils was characterized by co-culturing the cells with E.coli particles that were labelled with pHRhodo. Neutrophils that contained phagocytosed particles are shown in green in the overlay dot plot. The bottom represented FSC and SSC plots of samples obtained from 6 different infants and demonstrates a high level of diversity in the immune cell populations present in the airways of children with pneumonia. &lt;strong&gt;(B)&lt;/strong&gt; In silico deconvolution analysis was based on high resolution mass spectrometry data obtained from the airway. Cells were obtained from the naso and oropharyngeal sites using separate swabs, which were both eluted in a common transport media. As shown in A above, these samples contained a broad diversity of immune cells including T-cells, monocytes/macrophages and neutrophils. These cells were isolated from the sample by centrifugation and processed for mass spectrometry analysis using a standard protocol (see methods). &lt;strong&gt;(C)&lt;/strong&gt; In silico phenotype deconvolution was conducted initially by identifying phenotype specific protein features in a previously-published deconvolution data set using random forest feature selection. These features were then applied on airway proteome data set obtained from children admitted to hospital with severe pneumonia, and used to construct a detailed map of airway immunophenotypes and their associations with clinical outcomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F1.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F1.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-10" class="first-child">Analysis of the profile of airway-resident immune cells using flow cytometry and in-silico deconvolution. <strong>(A)</strong> To characterize the phenotype diversity of airway-resident immune cells, fresh naso- and oropharyngeal flocked swabs were obtained from children admitted to hospital with pneumonia and processed immediately by flow cytometry. Here the gating strategy is shown, where live, singlet cells were initially gated on CD45 expression, followed by fluorescent antibody staining for different cell surface markers. CD3+ and CD19-cells were gated as T-cells, CD66-, CD14+ and HLA-DR-cells were gated as macrophages/monocytes while neutrophils were gated on the basis of CD66 and CD16 double expression. The functional activity of neutrophils was characterized by co-culturing the cells with E.coli particles that were labelled with pHRhodo. Neutrophils that contained phagocytosed particles are shown in green in the overlay dot plot. The bottom represented FSC and SSC plots of samples obtained from 6 different infants and demonstrates a high level of diversity in the immune cell populations present in the airways of children with pneumonia. <strong>(B)</strong> In silico deconvolution analysis was based on high resolution mass spectrometry data obtained from the airway. Cells were obtained from the naso and oropharyngeal sites using separate swabs, which were both eluted in a common transport media. As shown in A above, these samples contained a broad diversity of immune cells including T-cells, monocytes/macrophages and neutrophils. These cells were isolated from the sample by centrifugation and processed for mass spectrometry analysis using a standard protocol (see methods). <strong>(C)</strong> In silico phenotype deconvolution was conducted initially by identifying phenotype specific protein features in a previously-published deconvolution data set using random forest feature selection. These features were then applied on airway proteome data set obtained from children admitted to hospital with severe pneumonia, and used to construct a detailed map of airway immunophenotypes and their associations with clinical outcomes.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-6"><h2 class="">Results</h2><p id="p-11">To determine whether the deconvolution proteome data set contained sufficient resolution to distinguish individual immune cell phenotypes, we used protein expression levels from the data set to visualise phenotype segregation by nonmetric multidimensional scaling (NMDS). The analysis showed that major immune cell phenotypes including B-cells, T cells, natural killer (NK) cells, dendritic cells(DC), monocytes (MO), basophils, eosinophils and neutrophils could be distinctly segregated on the basis of differential protein expression (<a id="xref-fig-2-1" class="xref-fig" href="#F2">figure 2a</a>). We then set out to identify individual proteins that could be used to accurately distinguish major and sub immune phenotypes (sub phenotypes defined as lower-level hierarchies of the major phenotypes – e.g. plasmacytoid and myeloid DCs within the major DC phenotype) within the deconvolution data set. Using the recursive feature elimination-paired random forest (RF) algorithm, boruta<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>, we identified classification features for 33 major and sub immune cell phenotypes (supplementary table 1). Since only a subset of these feature proteins are likely to be expressed within the specific context of the inflamed airway, we set out to determine which of the features identified by the model were also expressed in the airways of children admitted with pneumonia. For some cell types such as monocytes/macrophages and neutrophils, a substantial proportion (&gt;30%) of the phenotype classification features from the feature selection model were also expressed in the airway, while for others (e.g. NK cells), a lower proportion of the classifiers were identified in the airway (<a id="xref-fig-2-2" class="xref-fig" href="#F2">figures 2B, D, F &amp; H</a>). Supplementary table 1 contains a complete list of all the identified features and their respective expression levels in the infant airway.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Use of proteomics data to segregate immune cell phenotypes. (A) A nonmetric multidimensional scaling (NMDS) plot was used to visualize the separation of immune cell phenotypes at the major phenotype level based on protein expression data from the deconvolution data set. We observed that expression profile of certain cell types including neutrophils, monocytes and basophils, resulted in a clear separation from other immune cell types. To identify specific proteins that could be used to distinguish phenotypes, we used random forest feature selection. Using this approach, we identified protein features for 33 phenotypes that could be used to disaggregate individual phenotypes from a complex mixture (Supplementary table 1). We then examined airway proteome data from children with pneumonia to determine whether any of the proteins identified were detectable in the airway. An example of the results is shown in (B,D,F &amp; H), where the phenotype classification features for NK cells, DCs, Neutrophils and Monocytes are shown labelled on the circular tile plot. The green tiles are the phenotype-specific feature proteins that were identified in the random forest model and that were also detected in the airway. We used enrichment analysis to determine whether the phenotype-classifier proteins identified by the random forest model for specific phenotypes could be used to recapitulate those the phenotypes in an unsupervised phenotype prediction analysis. The clustergrams shown in (C,E,G &amp; I) depict a co-expression matrix of feature proteins and their known expression in different cell types. For 20 of the 33 phenotypes, (including the four examples shown here) the random forest phenotype classification, correctly predicted the original phenotype in the unsupervised prediction analysis. Each row on the clustergram represents a feature protein, while each colored box represents the known expression profile in a particular cell types. Horizonal panels at the bottom indicate enrichment scores from the unsupervised analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309048648" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Use of proteomics data to segregate immune cell phenotypes. &lt;strong&gt;(A)&lt;/strong&gt; A nonmetric multidimensional scaling (NMDS) plot was used to visualize the separation of immune cell phenotypes at the major phenotype level based on protein expression data from the deconvolution data set. We observed that expression profile of certain cell types including neutrophils, monocytes and basophils, resulted in a clear separation from other immune cell types. To identify specific proteins that could be used to distinguish phenotypes, we used random forest feature selection. Using this approach, we identified protein features for 33 phenotypes that could be used to disaggregate individual phenotypes from a complex mixture (Supplementary table 1). We then examined airway proteome data from children with pneumonia to determine whether any of the proteins identified were detectable in the airway. An example of the results is shown in &lt;strong&gt;(B&lt;/strong&gt;,&lt;strong&gt;D&lt;/strong&gt;,&lt;strong&gt;F &amp; H)&lt;/strong&gt;, where the phenotype classification features for NK cells, DCs, Neutrophils and Monocytes are shown labelled on the circular tile plot. The green tiles are the phenotype-specific feature proteins that were identified in the random forest model and that were also detected in the airway. We used enrichment analysis to determine whether the phenotype-classifier proteins identified by the random forest model for specific phenotypes could be used to recapitulate those the phenotypes in an unsupervised phenotype prediction analysis. The clustergrams shown in &lt;strong&gt;(C&lt;/strong&gt;,&lt;strong&gt;E&lt;/strong&gt;,&lt;strong&gt;G &amp; I)&lt;/strong&gt; depict a co-expression matrix of feature proteins and their known expression in different cell types. For 20 of the 33 phenotypes, (including the four examples shown here) the random forest phenotype classification, correctly predicted the original phenotype in the unsupervised prediction analysis. Each row on the clustergram represents a feature protein, while each colored box represents the known expression profile in a particular cell types. Horizonal panels at the bottom indicate enrichment scores from the unsupervised analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F2.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F2.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-12" class="first-child">Use of proteomics data to segregate immune cell phenotypes. <strong>(A)</strong> A nonmetric multidimensional scaling (NMDS) plot was used to visualize the separation of immune cell phenotypes at the major phenotype level based on protein expression data from the deconvolution data set. We observed that expression profile of certain cell types including neutrophils, monocytes and basophils, resulted in a clear separation from other immune cell types. To identify specific proteins that could be used to distinguish phenotypes, we used random forest feature selection. Using this approach, we identified protein features for 33 phenotypes that could be used to disaggregate individual phenotypes from a complex mixture (Supplementary table 1). We then examined airway proteome data from children with pneumonia to determine whether any of the proteins identified were detectable in the airway. An example of the results is shown in <strong>(B</strong>,<strong>D</strong>,<strong>F &amp; H)</strong>, where the phenotype classification features for NK cells, DCs, Neutrophils and Monocytes are shown labelled on the circular tile plot. The green tiles are the phenotype-specific feature proteins that were identified in the random forest model and that were also detected in the airway. We used enrichment analysis to determine whether the phenotype-classifier proteins identified by the random forest model for specific phenotypes could be used to recapitulate those the phenotypes in an unsupervised phenotype prediction analysis. The clustergrams shown in <strong>(C</strong>,<strong>E</strong>,<strong>G &amp; I)</strong> depict a co-expression matrix of feature proteins and their known expression in different cell types. For 20 of the 33 phenotypes, (including the four examples shown here) the random forest phenotype classification, correctly predicted the original phenotype in the unsupervised prediction analysis. Each row on the clustergram represents a feature protein, while each colored box represents the known expression profile in a particular cell types. Horizonal panels at the bottom indicate enrichment scores from the unsupervised analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Next, we undertook a more detailed analysis of the phenotype classification features that were identified by machine learning to determine whether they could be used to recapitulate the phenotypes in an unsupervised prediction analysis. We reasoned that the classification features of a particular phenotype would be broadly related to its functional properties and that when they are subjected to an independent unsupervised enrichment analysis, they would successfully predict the phenotypes from which they were initially derived. Using the unsupervised enrichment platform, enrichR<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>, we found that 20 of the 33 immune cell phenotypes were successfully predicted by their respective feature proteins (examples in <a id="xref-fig-2-3" class="xref-fig" href="#F2">figures 2 C, E, G &amp; I</a>). Subsequent analysis was restricted to these phenotypes. We visualised the performance of individual classification features by comparing their expression levels between different immune cell phenotypes. An example of this analysis is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">figure 3A</a>, where the monocyte classification feature, SERPINB2, is shown as being expressed at substantially higher levels in activated monocytes relative to all other cell phenotypes; the median expression level of SERPINB2 in classical activated monocytes was &gt;10<sup>4</sup> fold higher in monocytes, relative to all other phenotypes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">figure 3B</a>). Of the 64 classification features identified by machine learning for activated monocytes, 17 were detected in the airway samples from children with pneumonia (<a id="xref-fig-3-3" class="xref-fig" href="#F3">figure 3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detailed analysis of the phenotype-specific feature proteins. (A) We used a circular heat map to visualize differential expression of phenotype-specific proteins in a selection of immune cell phenotypes. In this example, the fold difference in protein expression between classical activated monocytes and other immune cell subsets is shown. Each spoke of the wheel represents a single protein, and each segment on a spoke represents the log10 fold difference between protein&#x2019;s expression level in classical activated monocytes and each of the listed cell types. An increase in the intensity of the red hue indicates greater expression in classical activated monocytes, relative to the comparator phenotype. (B) Protein expression across different phenotypes: in this example, the expression level of a feature protein for activated monocytes (SERPINB2 &#x2013; identified from the random forest model) was compared between different phenotypes and was seen to be expressed almost exclusively by monocytes. (C) The airway proteome data from paediatric pneumonia admissions was examined to determine whether any of the phenotype classification features for classical activated monocytes were also detected in the airways of children. Each segment represents a single feature, with proteins that were also detected in the airway depicted in a green hue. An increase in hue intensity, indicates increased airway expression for a particular protein. SERPINB2 (red arrow), was expressed at relatively high levels in the infant airway. (D) The expression level of all monocyte-specific features was plotted for monocytes and other cell types. Each black open circle denotes the expression level of a particular feature in other phenotypes, while red markers indicate their expression in monocytes. The dashed black and red lines indicated median expression of all features in monocytes and alternate cell types respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309048648" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Detailed analysis of the phenotype-specific feature proteins. &lt;strong&gt;(A)&lt;/strong&gt; We used a circular heat map to visualize differential expression of phenotype-specific proteins in a selection of immune cell phenotypes. In this example, the fold difference in protein expression between classical activated monocytes and other immune cell subsets is shown. Each spoke of the wheel represents a single protein, and each segment on a spoke represents the log&lt;sub&gt;10&lt;/sub&gt; fold difference between protein&#x2019;s expression level in classical activated monocytes and each of the listed cell types. An increase in the intensity of the red hue indicates greater expression in classical activated monocytes, relative to the comparator phenotype. &lt;strong&gt;(B)&lt;/strong&gt; Protein expression across different phenotypes: in this example, the expression level of a feature protein for activated monocytes (SERPINB2 &#x2013; identified from the random forest model) was compared between different phenotypes and was seen to be expressed almost exclusively by monocytes. &lt;strong&gt;(C)&lt;/strong&gt; The airway proteome data from paediatric pneumonia admissions was examined to determine whether any of the phenotype classification features for classical activated monocytes were also detected in the airways of children. Each segment represents a single feature, with proteins that were also detected in the airway depicted in a green hue. An increase in hue intensity, indicates increased airway expression for a particular protein. SERPINB2 (red arrow), was expressed at relatively high levels in the infant airway. &lt;strong&gt;(D)&lt;/strong&gt; The expression level of all monocyte-specific features was plotted for monocytes and other cell types. Each black open circle denotes the expression level of a particular feature in other phenotypes, while red markers indicate their expression in monocytes. The dashed black and red lines indicated median expression of all features in monocytes and alternate cell types respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-14" class="first-child">Detailed analysis of the phenotype-specific feature proteins. <strong>(A)</strong> We used a circular heat map to visualize differential expression of phenotype-specific proteins in a selection of immune cell phenotypes. In this example, the fold difference in protein expression between classical activated monocytes and other immune cell subsets is shown. Each spoke of the wheel represents a single protein, and each segment on a spoke represents the log<sub>10</sub> fold difference between protein’s expression level in classical activated monocytes and each of the listed cell types. An increase in the intensity of the red hue indicates greater expression in classical activated monocytes, relative to the comparator phenotype. <strong>(B)</strong> Protein expression across different phenotypes: in this example, the expression level of a feature protein for activated monocytes (SERPINB2 – identified from the random forest model) was compared between different phenotypes and was seen to be expressed almost exclusively by monocytes. <strong>(C)</strong> The airway proteome data from paediatric pneumonia admissions was examined to determine whether any of the phenotype classification features for classical activated monocytes were also detected in the airways of children. Each segment represents a single feature, with proteins that were also detected in the airway depicted in a green hue. An increase in hue intensity, indicates increased airway expression for a particular protein. SERPINB2 (red arrow), was expressed at relatively high levels in the infant airway. <strong>(D)</strong> The expression level of all monocyte-specific features was plotted for monocytes and other cell types. Each black open circle denotes the expression level of a particular feature in other phenotypes, while red markers indicate their expression in monocytes. The dashed black and red lines indicated median expression of all features in monocytes and alternate cell types respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">To enhance the power of the identified features to resolve the constituent cell phenotypes of a complex sample, we generated a phenotype classification profile, in which all the classification features of a particular phenotype were aggregated and their combined expression was plotted relative to all other phenotypes. An example of this analysis is shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">figure 3D</a>, where the combined expression of all monocyte-specific features (plotted in red) was compared to the expression level of the same proteins in all other phenotypes (plotted in gray). The results of this analysis showed that these proteins were expressed at a significantly higher level in monocytes compared to all other phenotypes (p&lt;0.0001). Similar analysis for all other phenotypes is presented in supplementary figure 1. We then used t-SNE dimensional reduction analysis to visualise the segregation of major and sub phenotypes on the basis of the phenotype classification features (<a id="xref-fig-4-1" class="xref-fig" href="#F4">figure 4A</a>). The results of this analysis showed that the feature classifier proteins could clearly distinguish most immune cell types; phenotypes like plasma cells (B.plasma), pDC, mDC, eosinophils, basophils, neutrophils and different monocyte sub phenotypes were clearly distinguishable from the rest of the phenotypes. Some sub-phenotypes such as central memory CD4 T cells (T4.CM) and effector memory CD4 T cells (T4.EM), could not be clearly disaggregated on the basis of feature expression alone.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Performance of all phenotype specific protein features in segregating different immune phenotypes and sub phenotypes was visualized using a t-SNE plot. Most phenotype classification features were able to disaggregate different phenotype hierarchies in the 2D t-SNE space. Neutrophils, eosinophils, monocytes, NK cells, dendritic cells, basophils among others were clearly separated from all other phenotypes. There was less clear segregation for T-cell sub phenotypes, including CD8, CD4 and regulatory T-cell phenotypes. (B) the expression of eosinophil- and neutrophil-specific features was compared between children with different survival outcomes of pneumonia as well as in well controls. Eosinophil features were expressed at significantly higher levels in survivors compared with non-survivors and well controls. In contrast, there was no difference in the expression of neutrophil features between these groups. (C) Feature classifiers for other immune cell phenotypes were also compared by survival status. No significant difference in expression by survival status was observed, with the exception of the CD8 (T8) phenotype, whose features were significantly upregulated in the airways of survivors relative to non-survivors and well controls." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309048648" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A)&lt;/strong&gt; Performance of all phenotype specific protein features in segregating different immune phenotypes and sub phenotypes was visualized using a t-SNE plot. Most phenotype classification features were able to disaggregate different phenotype hierarchies in the 2D t-SNE space. Neutrophils, eosinophils, monocytes, NK cells, dendritic cells, basophils among others were clearly separated from all other phenotypes. There was less clear segregation for T-cell sub phenotypes, including CD8, CD4 and regulatory T-cell phenotypes. &lt;strong&gt;(B)&lt;/strong&gt; the expression of eosinophil- and neutrophil-specific features was compared between children with different survival outcomes of pneumonia as well as in well controls. Eosinophil features were expressed at significantly higher levels in survivors compared with non-survivors and well controls. In contrast, there was no difference in the expression of neutrophil features between these groups. &lt;strong&gt;(C)&lt;/strong&gt; Feature classifiers for other immune cell phenotypes were also compared by survival status. No significant difference in expression by survival status was observed, with the exception of the CD8 (T8) phenotype, whose features were significantly upregulated in the airways of survivors relative to non-survivors and well controls.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F4.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F4.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-16" class="first-child"><strong>(A)</strong> Performance of all phenotype specific protein features in segregating different immune phenotypes and sub phenotypes was visualized using a t-SNE plot. Most phenotype classification features were able to disaggregate different phenotype hierarchies in the 2D t-SNE space. Neutrophils, eosinophils, monocytes, NK cells, dendritic cells, basophils among others were clearly separated from all other phenotypes. There was less clear segregation for T-cell sub phenotypes, including CD8, CD4 and regulatory T-cell phenotypes. <strong>(B)</strong> the expression of eosinophil- and neutrophil-specific features was compared between children with different survival outcomes of pneumonia as well as in well controls. Eosinophil features were expressed at significantly higher levels in survivors compared with non-survivors and well controls. In contrast, there was no difference in the expression of neutrophil features between these groups. <strong>(C)</strong> Feature classifiers for other immune cell phenotypes were also compared by survival status. No significant difference in expression by survival status was observed, with the exception of the CD8 (T8) phenotype, whose features were significantly upregulated in the airways of survivors relative to non-survivors and well controls.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">We then applied the phenotype classification features identified by machine learning to airway proteome data obtained from naso/oropharyngeal samples from children who were admitted to hospital with clinical pneumonia and who either survived and were discharged from hospital(N=61) or ultimately died (N=19) in the course of admission. We also obtained similar data from age-matched well controls, who were sampled from the community (N=10). Using a false discovery rate (FDR) of 5%, we identified &gt;1,000 proteins in the airways of these children. We then compared the expression levels of phenotype-specific feature proteins between these groups. We found that the expression levels of eosinophil-specific classification features were significantly overexpressed in children who survived infection, relative to non-survivors (P&lt;0.0001) and controls (<a id="xref-fig-4-2" class="xref-fig" href="#F4">figure 4B</a>). This was in contrast to neutrophils, where the expression level of phenotype-specific proteins did not vary by survival status (<a id="xref-fig-4-3" class="xref-fig" href="#F4">figure 4B</a>). The expression level of proteins predominantly expressed by naïve CD8 T cells (T8.naive) were also significantly elevated in pneumonia survivors compared to non-survivors (P&lt;0.05). We observed no significant differences for the remaining phenotypes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">figure 4C</a>). To validate these findings, we reviewed the clinical records of &gt;10,000 children who had been admitted to Kilifi County Hospital Kenya over an approximate 10-year period with clinical pneumonia and for whom haematological data (including eosinophil and neutrophil counts) had been collected at admission. These data were stratified by survival status and the difference in eosinophils and neutrophil counts between these groups was tested. We found that eosinophil counts in children who survived pneumonia in the retrospective validation cohort (N=10,859) were significantly greater than those of non-survivors (N=1,604, p=0.0004) – <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, first panel. We then evaluated the risk of mortality in the first 10 days after admission in a subset of children who are at an elevated risk of pneumonia mortality (undernourished children), in order to determine the association between eosinophil counts and pneumonia survival in this group. Eosinophils counts in this group were stratified into two groups on the basis of median counts. Undernourished children who were admitted with pneumonia and with low (below median) eosinophil counts, were significantly more likely to die in the first 10 days of admission, relative to those with high eosinophil counts (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>, second panel). In contrast, no association with mortality was observed when neutrophils were analysed using a similar strategy (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of the phenotype associations with pneumonia mortality using a retrospective pneumonia cohort. (A, first panel) Eosinophils counts were analysed in a retrospective cohort of &gt;10,000 children who had been admitted to hospital with clinical pneumonia and who either survived and were discharged or died during admission. Pneumonia survivors had significantly greater eosinophil counts at admission compared to non-survivors. (A, second panel) In a subset of children who were acutely undernourished at admission (defined as by a mid-upper arm (MUAC) circumference " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309048648" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Validation of the phenotype associations with pneumonia mortality using a retrospective pneumonia cohort. &lt;strong&gt;(A, first panel)&lt;/strong&gt; Eosinophils counts were analysed in a retrospective cohort of &gt;10,000 children who had been admitted to hospital with clinical pneumonia and who either survived and were discharged or died during admission. Pneumonia survivors had significantly greater eosinophil counts at admission compared to non-survivors. &lt;strong&gt;(A, second panel)&lt;/strong&gt; In a subset of children who were acutely undernourished at admission (defined as by a mid-upper arm (MUAC) circumference &lt;11.5cm), children were stratified into two groups on the basis of the median eosinophil count. In the first 10 days after admission, children whose eosinophil counts were below the median were more likely to die relative to those with above-median counts. (B) In contrast, there was no association between neutrophil counts and survival status.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F5.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F5.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/19/840090/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-18" class="first-child">Validation of the phenotype associations with pneumonia mortality using a retrospective pneumonia cohort. <strong>(A, first panel)</strong> Eosinophils counts were analysed in a retrospective cohort of &gt;10,000 children who had been admitted to hospital with clinical pneumonia and who either survived and were discharged or died during admission. Pneumonia survivors had significantly greater eosinophil counts at admission compared to non-survivors. <strong>(A, second panel)</strong> In a subset of children who were acutely undernourished at admission (defined as by a mid-upper arm (MUAC) circumference &lt;11.5cm), children were stratified into two groups on the basis of the median eosinophil count. In the first 10 days after admission, children whose eosinophil counts were below the median were more likely to die relative to those with above-median counts. (B) In contrast, there was no association between neutrophil counts and survival status.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-19">We report on a new method of deconvolving immune cell populations that are resident in the airways of infants and children with different survival outcomes of severe pneumonia. The study mucosal cellular immunity during very severe pneumonia has been hindered by a number of important hurdles including low sample volumes as well as the relatively low abundance of immune cell phenotypes that may be critical in directing the clinical course of pneumonia. In this study, we addressed these problems by using a machine learning approach to identify protein markers that could be used to deconvolve mixed immune cells. We then applied these markers to airway proteome data obtained from children with different survival outcomes of clinical pneumonia. Our results show that protein markers associated with eosinophils are elevated in the airways of survivors and are diminished in children who later died from infection. The airway levels of these eosinophil markers were no different between children who died and well controls, suggesting that the failure to mount an appropriate eosinophil response is a potential mechanism of pneumonia-related mortality, especially in high risk populations such as undernourished children. To validate the findings of the airway proteome analysis, we reviewed the hospitalisation records of &gt;10,000 children who had been admitted to hospital in the previous 10 years with clinical pneumonia. The results of this retrospective analysis confirmed the observations made from analysis of the airway proteome and showed that children who died from pneumonia, had significantly lower blood eosinophil counts relative to those who survived. In contrast, neutrophil levels were not different between survivors and non survivors in both the airway proteome and in the retrospective validation cohort.</p><p id="p-20">Previous studies have shown that eosinophils are activated in the airway shortly after pneumonia infection appear to contribute significantly to airway recovery. An increase in the expression of eosinophil-related markers in children with severe pneumonia has been associated with a reduced requirement for supplemental oxygen<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>, suggesting that these cells are a critical component in the host’s response to infecting pathogens in the airway. Airway-resident eosinophils have not been extensively characterised in previous pneumonia studies, although proteins expressed predominantly by eosinophils including leukotriene C4, eosinophil-derived neurotoxin (RNASE2) and eosinophil cationic protein (ECP) are expressed at high levels during viral pneumonia<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>-<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. Taken together with the results of the current study, there appears to be a clear role for eosinophils in the response to pneumonia infection and that the failure to sufficiently recruit them to the airway may be a significant factor in mortality. A potential limitation of our study is that it was carried out using samples collected from the upper airway and not the lung. While lung samples would have undoubtedly been more informative of responses at the site of disease, the collection of these samples is a highly invasive process and exposes children to substantial additional risk without providing additional diagnostic value above nasopharyngeal or peripheral blood sampling. As a result, these samples are generally only available in settings with paediatric intensive care facilities, and even here, they are generally collected in children with atypically severe disease.</p><p id="p-21">In addition to eosinophils, previous studies have shown infections that cause severe pneumonia such as RSV, trigger a strong cellular response, characterised by the influx of innate immune cells<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. The initial response to infection is characterised by the airway recruitment of neutrophils, which express markers such as CD11b (ITGAM) and neutrophil granule proteins such as neutrophil elastase (ELANE)<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. Other innate immune cells such as NK cells which expressed granzyme B are recruited in the lower airway and can be detected in in the lungs of mechanically-ventilated children with very severe pneumonia<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. In addition to these cells, both myeloid and plasmacytoid dendritic cells are typically recruited into the airways of children with pneumonia in the early stages of infection<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> and exhibit an activated proinflammatory phenotype<sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a></sup>. Adaptive immune cells including CD4+ and CD8+ T cells are present in airway samples for children with viral pneumonia<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. During infection, airway frequencies of granzyme B-secreting, activated CD8 T cells is greater in the airways of children with severe viral pneumonia<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>. In the present study, we found that protein markers of different phenotypes of CD8 T cells were significantly elevated in the airways of children who survived pneumonia relative to those who died or well controls. Although data was not available for validation of these phenotypes in the retrospective cohort, the data suggests a role for these cells in pneumonia survival. Previous studies in mechanically ventilated children showed that the frequency of lung-associated T cells increased as children recovered from infection, suggesting that these cells are an important component of effective local immunity against pneumonia and that deficits in the airways of children with pneumonia is a significant risk factor for mortality. In summary, the present data identifies a new approach for characterising the cellular immune response to pneumonia in the airway and identified critical immune populations that appear to be critical for survival. Future studies should aim to replicate these findings in samples collected from the lower respiratory tract.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>Study site and population</h3><p id="p-22">This study recruited 80 infants and children who were admitted to Kilifi County Hospital with clinical pneumonia, defined using World Health Organisation syndromic criteria. Nasal samples were collected from each child for proteomic analysis. The microbial etiology of pneumonia was determined using both blood cultures and using a 15-target multiplex PCR panel for the detection of respiratory syncytial virus (RSV -A &amp; B), rhinovirus, parainfluenza virus (1, 2, 3 &amp; 4) adenovirus, influenza (A, B &amp; C), coronavirus (OC43 &amp; e229), human metapneumovirus and <em>Mycoplasma pneumoniae</em>. Children with clinical pneumonia signs and a positive diagnostic result from any of these tests were included in the analysis. Children were stratified into the survival group if they were alive for at least 365 days after discharge(n=61) while the mortality group comprised of children who died within 72 hours of admission (n=19). In addition to these groups, we recruited 10 age-matched well controls as a comparator group. Written informed consent was sought from the parents and legal guardians of all children who were sampled in this study. Ethical approval for the conduct of this study was granted by the Kenya Medical Research Institute’s Scientific and ethical research unit (SERU). All study procedures were conducted in accordance with Good Clinical Laboratory Practise (GCLP) standards.</p></div><div id="sec-10" class="subsection"><h3>Analysis of airway proteomes using mass spectrometry</h3><p id="p-23">Naso- and oropharyngeal swab samples were centrifuged at 17,000xg for 10 mins at 4°C to obtain cell pellets which were washed once using PBS and lysed by bead-vortexing for 10 minutes in cell lysis buffer (RLT, Qiagen, Germany). Proteins (as well as DNA and RNA) were then extracted from the lysate using the AllPrep DNA/RNA/Protein Mini Kit (Qiagen, Germany) following manufacturers instructions. The concentration of total protein obtained was determined using the Bradford assay (Bio-Rad, USA). Thirty micrograms (30µg) of total protein from each sample was then reduced with 10mM tris(2-carboxyethyl)phosphine (TCEP, Sigma-Aldrich, USA) at 55°C for 1h and subsequently alkylated with 18mM IAA (Sigma-Aldrich, USA) for 30 minutes at room temperature, while keeping the reaction protected from light. Proteins were precipitated overnight at −20°C with six volumes of pre-chilled (−20°C) acetone (Sigma-Aldrich, USA). The samples were centrifuged at 8,000xg for 10 minutes at 4°C to obtain the protein pellets and supernatants were discarded. The protein pellet was resuspended in 100µl of 50mM Triethylammonium bicarbonate (TEAB, Sigma-Aldrich, USA). Trypsin (Sigma-Aldrich, USA) was added to the protein samples at a trypsin-protein sample ratio of 1:10 and protein digestion was allowed to proceed overnight at 37°C with shaking. The peptide samples were randomly assigned to 10 individual batches: each containing nine patient samples and one pooled control sample. The pooled control sample consisted of a pool of peptides from all patient samples. The peptide samples derived from individual patients were then individually labelled using the TMT10plex mass tag kit (Thermo scientific, USA) according to manufacturer’s instructions, with one isobaric tag being exclusively used to label the pooled control sample. The labelled peptides for each 10plex were subsequently combined to generate 10 individual pools. The labelled peptide pools were desalted using P10 C18 pipette ZipTips (Millipore, USA) according to the manufacturer’s instructions. Eluted peptides were dried in a Speedvac concentrator (Thermo Scientific, USA). Peptides (8 μl) were loaded using a Dionex Ultimate 3000 nano-flow ultra-high-pressure liquid chromatography system (Thermo Scientific, USA) on to a 75µm × 2 cm C18 trap column (Thermo Scientific, USA) and separated on a 75µm × 50 cm C18 reverse-phase analytical column (Thermo Scientific) at heated at 40°C. For LFQ protein quantification; elution was carried out with mobile phase B (80% acetonitrile with 0.1% formic acid) gradient (4 to 30%) over 310 min at a flow rate of 0.25 μl/min. Each LC run was finished by washout with 98% B for 10 min and re-equilibration in 2% B for 30 min. Five blanks of 40 min each were run on the column between each injection comprising of two wash cycles with 90% B and an equilibration phase of 15 min to avoid sample carryover. Peptides were measured using a Q Exactive Orbitrap mass spectrometer (Thermo Scientific, USA) coupled to the chromatography system via a nano-electrospray ion source (Thermo Scientific). On the Q Exactive, the ms^1 settings for peptides were: Resolution, 70000; AGC target, 3e6; maximum IT, 120 ms; scan range, 400-1800 m/z; while the ms^2 settings for fragmentation spectra of peptides were: Resolution, 17000 (35000 for labelled peptides); AGC target, 5e4; maximum IT, 120 ms; isolation window, 1.6 m/z. MS data were acquired by data dependent acquisition where the top 12 (15 for labelled peptides) most intense precursor ions in positive mode were selected for ms^2 Higher-energy C-trap dissociation fragmentation which were subsequently excluded for the next 45 s following fragmentation event. Charge exclusion was set to ignore peptide spectrum matches that were unassigned, singly charged, and those with ≥+8 charges. Raw mass spectrometer files were analysed by MaxQuant software version 1.6.0.1. by searching against the human Uniprot FASTA database (downloaded February 2014) using the Andromeda search engine..</p><p id="p-24">Airway samples were centrifuged at 17,000xg for 10 mins at 4°C to obtain cell pellets which were washed once using PBS and lysed by bead-vortexing for 10 minutes in cell lysis buffer (RLT, Qiagen, Germany). Proteins (as well as DNA and RNA) were then extracted from the lysate using the AllPrep DNA/RNA/Protein Mini Kit (Qiagen, Germany) following manufacturers instructions. The concentration of total protein obtained was determined using the Bradford assay (Bio-Rad, USA). Thirty micrograms (30µg) of total protein from each sample was then reduced with 10mM tris(2-carboxyethyl)phosphine (TCEP, Sigma-Aldrich, USA) at 55°C for 1h and subsequently alkylated with 18mM IAA (Sigma-Aldrich, USA) for 30 minutes at room temperature, while keeping the reaction protected from light. Proteins were precipitated overnight at −20°C with six volumes of pre-chilled (−20°C) acetone (Sigma-Aldrich, USA). The samples were centrifuged at 8,000xg for 10 minutes at 4°C to obtain the protein pellets and supernatants were discarded. The protein pellet was resuspended in 100µl of 50mM Triethylammonium bicarbonate (TEAB, Sigma-Aldrich, USA). Trypsin (Sigma-Aldrich, USA) was added to the protein samples at a trypsin-protein sample ratio of 1:10 and protein digestion was allowed to proceed overnight at 37°C with shaking. The peptide samples were randomly assigned to 10 individual batches: each containing nine patient samples and one pooled control sample. The pooled control sample consisted of a pool of peptides from all patient samples. The peptide samples derived from individual patients were then individually labelled using the TMT10plex mass tag kit (Thermo scientific, USA) according to manufacturer’s instructions, with one isobaric tag being exclusively used to label the pooled control sample. The labelled peptides for each 10plex were subsequently combined to generate 10 individual pools. The labelled peptide pools were desalted using P10 C18 pipette ZipTips (Millipore, USA) according to the manufacturer’s instructions. Eluted peptides were dried in a Speedvac concentrator (Thermo Scientific, USA). Peptides (8 μl) were loaded using a Dionex Ultimate 3000 nano-flow ultra-high-pressure liquid chromatography system (Thermo Scientific, USA) on to a 75µm × 2 cm C18 trap column (Thermo Scientific, USA) and separated on a 75µm × 50 cm C18 reverse-phase analytical column (Thermo Scientific) at heated at 40°C. For LFQ protein quantification; elution was carried out with mobile phase B (80% acetonitrile with 0.1% formic acid) gradient (4 to 30%) over 310 min at a flow rate of 0.25 μl/min. Each LC run was finished by washout with 98% B for 10 min and re-equilibration in 2% B for 30 min. Five blanks of 40 min each were run on the column between each injection comprising of two wash cycles with 90% B and an equilibration phase of 15 min to avoid sample carryover. Peptides were measured using a Q Exactive Orbitrap mass spectrometer (Thermo Scientific, USA) coupled to the chromatography system via a nano-electrospray ion source (Thermo Scientific). On the Q Exactive, the ms^1 settings for peptides were: Resolution, 70000; AGC target, 3e6; maximum IT, 120 ms; scan range, 400-1800 m/z; while the ms^2 settings for fragmentation spectra of peptides were: Resolution, 17000 (35000 for labelled peptides); AGC target, 5e4; maximum IT, 120 ms; isolation window, 1.6 m/z. MS data were acquired by data dependent acquisition where the top 12 (15 for labelled peptides) most intense precursor ions in positive mode were selected for ms^2 Higher-energy C-trap dissociation fragmentation which were subsequently excluded for the next 45 s following fragmentation event. Charge exclusion was set to ignore peptide spectrum matches that were unassigned, singly charged, and those with ≥+8 charges. Raw mass spectrometer files were analysed by <span class="underline">MaxQuant software</span> version 1.6.0.1. by searching against the human Uniprot FASTA database (downloaded February 2014) using the Andromeda search engine.</p></div><div id="sec-11" class="subsection"><h3>Analysis of airway-resident immune cells using flow cytometry</h3><p id="p-25">1ml of nasopharyngeal and oropharyngeal swab samples obtained from children was centrifuged at 17,000xg for 7 minutes, after which 800⧅l of the supernatant was removed and discarded. The remaining 200μl were split into two aliquots of 100μl each. The first aliquot was used for neutrophil phenotyping assays and the other was used for neutrophil phagocytosis assays. 20μl of a pre-constituted cocktail of the following antibodies (from ThermoFisher) was used to label both aliquots – CD45, CD16, CD14, CD3, CD19, HLA-DR, CD66b, CD11b and a Live-dead marker. With the exception of the live/dead marker, all other antibodies were diluted 1:100 in FACS buffer. The live/dead marker was prepared at a 1:1000 dilution in FACS buffer. For the phagocytosis assay tube, 20⧅l of opsonised <em>Escherichia coli</em> (<em>E.coli</em>) was added to the tube (pHrodo Red E. coli BioParticles; ThermoFisher). The bacteria was initially prepared by mixing the <em>E.coli</em> strain with new-born calf sera followed by a 30-minute incubation at 37°C. After this step, both tubes were incubated at 37°C for 35 minutes. After the incubation, 20⧅l of a live-dead marker was added to each tube and incubated for 10 minutes at 37°C. The reaction was stopped by adding 500⧅l of 1X RBC lysis buffer followed by a 5-minute incubation. Cells in each tube were then spun down at 2,700xg for 1 minute and the supernatant discarded. Cells were then washed twice with FACS buffer, after which 350⧅l of FACS flow was added. Cells were then analysed immediately on a BD LSR Fortessa instrument. The following gating strategy was used to detect airway-resident neutrophils: Debris were excluded on the basis of their forward (FSC-A) and side (SSC-A) scatter characteristics, doublets were excluded using FSC-A versus FSC-H and dead cells were excluded using the live-dead marker. Data was analysed using FlowJo software.</p></div><div id="sec-12" class="subsection"><h3>Statistical data analysis</h3><p id="p-26">Data were analyzed in R. deconvolution analysis by random forest classification was done using the Boruta package, with a maximum of 300 iterations. Protein classifiers that whose mean expression in the test phenotype was significantly greater than alternative phenotypes were taken forward for further analysis. T-SNE analysis was carried out using the Rtsne, with the iterations parameter set to a maximum of 300 and a perplexity value of 30. Analysis was carried out in two dimensional space. Cell type enrichment analysis was done one the enrichR platform. The input search term used for enrichment analysis was the RF protein classifier lists derived from random forest classification. All pairwise comparisons between the expression level of RF classifiers between phenotypes and frequency of immune cell types between survival states was done using t-tests on log10 normalised data. The deconvolution data set that was used to identify phenotype-specific protein classifiers was obtained from a previously published paper by Rieckmann et al.<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup></p></div></div><div class="section" id="sec-14"><h2 class="">Data availability</h2><p id="p-27">The proteomics data reported in this paper are available at the ProteomeXchange Consortium database (<span class="underline">Accession numbers:</span> PXD009403).</p></div><div class="section COI-statement" id="sec-15"><h2 class="">Conflicts of interest</h2><p id="p-29">The authors declare that they have no conflicting interests</p></div><div class="section" id="sec-16"><h2 class="">Author contributions</h2><p id="p-30">CJS designed the study</p><p id="p-31">CJS, JMN, MNM, ETG, AG conducted the experiments</p><p id="p-32">CJS,NKK analysed the data</p><p id="p-33">CJS wrote the manuscript</p><p id="p-34">All authors reviewed and approved the manuscript</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-28">This study was supported by fellowship funding to CJS from the Wellcome Trust (WT105882MA). The funder played no role in the conceptualization, design, data collection, analysis, decision to publish, or preparation of the manuscript</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.1" data-doi="10.1016/s0140-6736(19)30721-4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Causes of severe pneumonia requiring hospital admission in children without HIV infection from Africa and Asia: the PERCH multi-country case-control study</span>. <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr> <span class="cit-vol">394</span>, <span class="cit-fpage">757</span>–<span class="cit-lpage">779</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(19)30721-4</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D394%26rft.spage%253D757%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252819%252930721-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(19)30721-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.2" data-doi="10.1016/s0140-6736(17)30938-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global, regional, and national disease burden estimates of acute lower respiratory infections due to respiratory syncytial virus in young children in 2015: a systematic review and modelling study</span>. <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(17)30938-8</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252817%252930938-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(17)30938-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.3" data-doi="10.1016/s0140-6736(07)60670-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNally</span>,  <span class="cit-name-given-names">L. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Effect of age, polymicrobial disease, and maternal HIV status on treatment response and cause of severe pneumonia in South African children: a prospective descriptive study</span>. <abbr class="cit-jnl-abbrev">Lancet (London, England)</abbr> <span class="cit-vol">369</span>, <span class="cit-fpage">1440</span>–<span class="cit-lpage">1451</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/s0140-6736(07)60670-9</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2B%2528London%252C%2BEngland%2529%26rft.volume%253D369%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fs0140-6736%252807%252960670-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/s0140-6736(07)60670-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.4" data-doi="10.1111/j.1365-3156.2009.02364.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chisti</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tebruegge</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">La Vincente</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">S. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Duke</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Pneumonia in severely malnourished children in developing countries - mortality risk, aetiology and validity of WHO clinical signs: a systematic review</span>. <abbr class="cit-jnl-abbrev">Tropical medicine &amp; international health : TM &amp; IH</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1173</span>–<span class="cit-lpage">1189</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-3156.2009.02364.x</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTropical%2Bmedicine%2B%2526%2Binternational%2Bhealth%2B%253A%2B%2BTM%2B%2526%2BIH%26rft.stitle%253DTrop%2BMed%2BInt%2BHealth%26rft.aulast%253DChisti%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D1173%26rft.epage%253D1189%26rft.atitle%253DPneumonia%2Bin%2Bseverely%2Bmalnourished%2Bchildren%2Bin%2Bdeveloping%2Bcountries%2B-%2Bmortality%2Brisk%252C%2Baetiology%2Band%2Bvalidity%2Bof%2BWHO%2Bclinical%2Bsigns%253A%2Ba%2Bsystematic%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3156.2009.02364.x%26rft_id%253Dinfo%253Apmid%252F19772545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3156.2009.02364.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19772545&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.5" data-doi="10.2471/BLT.10.080796"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moisi</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Sensitivity of hospital-based surveillance for severe disease: a geographic information system analysis of access to care in Kilifi district, Kenya</span>. <abbr class="cit-jnl-abbrev">Bulletin of the World Health Organization</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">102</span>–<span class="cit-lpage">111</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.2471/blt.10.080796</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.aulast%253DMoisi%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D111%26rft.atitle%253DSensitivity%2Bof%2Bhospital-based%2Bsurveillance%2Bfor%2Bsevere%2Bdisease%253A%2Ba%2Bgeographic%2Binformation%2Bsystem%2Banalysis%2Bof%2Baccess%2Bto%2Bcare%2Bin%2BKilifi%2Bdistrict%252C%2BKenya.%26rft_id%253Dinfo%253Adoi%252F10.2471%252FBLT.10.080796%26rft_id%253Dinfo%253Apmid%252F21346921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2471/BLT.10.080796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21346921&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287625800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.6" data-doi="10.1371/journal.pmed.1001549"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mejias</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Whole blood gene expression profiles to assess pathogenesis and disease severity in infants with respiratory syncytial virus infection</span>. <abbr class="cit-jnl-abbrev">PLoS medicine</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e1001549</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pmed.1001549</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bmedicine%26rft.volume%253D10%26rft.spage%253De1001549%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pmed.1001549%26rft_id%253Dinfo%253Apmid%252F24265599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pmed.1001549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24265599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.7" data-doi="10.1038/modpathol.3800725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzales</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">P. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B. S.</span></span> <span class="cit-article-title">The histopathology of fatal untreated human respiratory syncytial virus infection</span>. <abbr class="cit-jnl-abbrev">Mod Pathol</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">108</span>–<span class="cit-lpage">119</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/modpathol.3800725</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DModern%2Bpathology%2B%253A%2B%2Ban%2Bofficial%2Bjournal%2Bof%2Bthe%2BUnited%2BStates%2Band%2BCanadian%2BAcademy%2Bof%2BPathology%252C%2BInc%26rft.stitle%253DMod%2BPathol%26rft.aulast%253DJohnson%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D108%26rft.epage%253D119%26rft.atitle%253DThe%2Bhistopathology%2Bof%2Bfatal%2Buntreated%2Bhuman%2Brespiratory%2Bsyncytial%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmodpathol.3800725%26rft_id%253Dinfo%253Apmid%252F17143259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/modpathol.3800725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17143259&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243005000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.8" data-doi="10.1371/journal.pone.0169805"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jochems</span>,  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel Analysis of Immune Cells from Nasal Microbiopsy Demonstrates Reliable, Reproducible Data for Immune Populations, and Superior Cytokine Detection Compared to Nasal Wash</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0169805</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0169805</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D12%26rft.spage%253De0169805%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.9" data-doi="10.4049/jimmunol.1800396"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Connors</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Developmental Regulation of Effector and Resident Memory T Cell Generation during Pediatric Viral Respiratory Tract Infection</span>. <abbr class="cit-jnl-abbrev">Journal of immunology (Baltimore, Md. : 1950)</abbr> <span class="cit-vol">201</span>, <span class="cit-fpage">432</span>–<span class="cit-lpage">439</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.4049/jimmunol.1800396</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bimmunology%2B%2528Baltimore%252C%2BMd.%2B%253A%2B1950%2529%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1800396%26rft_id%253Dinfo%253Apmid%252F29848753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNDMyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTkvODQwMDkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.10" data-doi="10.1038/s41598-018-32072-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sande</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Untargeted analysis of the airway proteomes of children with respiratory infections using mass spectrometry based proteomics</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">13814</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-32072-3</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D13814%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-32072-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-32072-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.11" data-doi="10.1038/s41467-019-10222-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sande</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Airway response to respiratory syncytial virus has incidental antibacterial effects</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">2218</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-019-10222-z</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D10%26rft.spage%253D2218%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-10222-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-10222-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.12" data-doi="10.1038/ni.3693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rieckmann</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Social network architecture of human immune cells unveiled by quantitative proteomics</span>. <abbr class="cit-jnl-abbrev">Nature immunology</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">583</span>–<span class="cit-lpage">593</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/ni.3693</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.volume%253D18%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.3693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kursa</span>,  <span class="cit-name-given-names">M. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rudnicki</span>,  <span class="cit-name-given-names">W. R.</span></span> <span class="cit-article-title">Feature Selection with the Boruta Package</span>. <abbr class="cit-jnl-abbrev">2010</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">13</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.18637/jss.v036.i11</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.14" data-doi="10.1186/1471-2105-14-128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">E. Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">128</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-14-128</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D14%26rft.spage%253D128%26rft.atitle%253DENRICHR%253A%2BINTERACTIVE%2BAND%2BCOLLABORATIVE%2BHTML5%2BGENE%2BLIST%2BENRICHMENT%2BANALYSIS%2BTOOL%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-128%26rft_id%253Dinfo%253Apmid%252F23586463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23586463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.15" data-doi="10.1093/nar/gkw377"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuleshov</span>,  <span class="cit-name-given-names">M. V.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Enrichr: a comprehensive gene set enrichment analysis web server 2016 update</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">W90</span>–<span class="cit-lpage">97</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw377</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw377%26rft_id%253Dinfo%253Apmid%252F27141961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27141961&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.16" data-doi="10.1111/j.1365-2249.2006.03084.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindemans</span>,  <span class="cit-name-given-names">C. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Systemic eosinophil response induced by respiratory syncytial virus</span>. <abbr class="cit-jnl-abbrev">Clinical and experimental immunology</abbr> <span class="cit-vol">144</span>, <span class="cit-fpage">409</span>–<span class="cit-lpage">417</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2249.2006.03084.x</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Band%2Bexperimental%2Bimmunology%26rft.stitle%253DClin%2BExp%2BImmunol%26rft.aulast%253DLindemans%26rft.auinit1%253DC.%2BA.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D409%26rft.epage%253D417%26rft.atitle%253DSystemic%2Beosinophil%2Bresponse%2Binduced%2Bby%2Brespiratory%2Bsyncytial%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2249.2006.03084.x%26rft_id%253Dinfo%253Apmid%252F16734609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2249.2006.03084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16734609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.17" data-doi="10.3346/jkms.2007.22.1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">M. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-article-title">Eosinophil cationic protein and chemokines in nasopharyngeal secretions of infants with respiratory syncytial virus (RSV) bronchiolitis and non-RSV bronchiolitis</span>. <abbr class="cit-jnl-abbrev">Journal of Korean medical science</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">42</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3346/jkms.2007.22.1.37</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BKorean%2Bmedical%2Bscience%26rft.stitle%253DJ%2BKorean%2BMed%2BSci%26rft.aulast%253DKim%26rft.auinit1%253DH.%2BH.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D42%26rft.atitle%253DEosinophil%2Bcationic%2Bprotein%2Band%2Bchemokines%2Bin%2Bnasopharyngeal%2Bsecretions%2Bof%2Binfants%2Bwith%2Brespiratory%2Bsyncytial%2Bvirus%2B%2528RSV%2529%2Bbronchiolitis%2Band%2Bnon-RSV%2Bbronchiolitis.%26rft_id%253Dinfo%253Adoi%252F10.3346%252Fjkms.2007.22.1.37%26rft_id%253Dinfo%253Apmid%252F17297249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3346/jkms.2007.22.1.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17297249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-840090v2.18" data-doi="10.1164/ajrccm.159.6.9805083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonville</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>,  <span class="cit-name-given-names">H. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Domachowske</span>,  <span class="cit-name-given-names">J. B.</span></span> <span class="cit-article-title">Respiratory syncytical virus-induced chemokine expression in the lower airways: eosinophil recruitment and degranulation</span>. <abbr class="cit-jnl-abbrev">American journal of respiratory and critical care medicine</abbr> <span class="cit-vol">159</span>, <span class="cit-fpage">1918</span>–<span class="cit-lpage">1924</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1164/ajrccm.159.6.9805083</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BRespiratory%2Band%2BCritical%2BCare%2BMedicine%26rft.stitle%253DAm.%2BJ.%2BRespir.%2BCrit.%2BCare%2BMed.%26rft.aulast%253DHARRISON%26rft.auinit1%253DA.%25A0M.%26rft.volume%253D159%26rft.issue%253D6%26rft.spage%253D1918%26rft.epage%253D1924%26rft.atitle%253DRespiratory%2BSyncytical%2BVirus-induced%2BChemokine%2BExpression%2Bin%2Bthe%2BLower%2BAirways%2B.%2BEosinophil%2BRecruitment%2Band%2BDegranulation%26rft_id%253Dinfo%253Adoi%252F10.1164%252Fajrccm.159.6.9805083%26rft_id%253Dinfo%253Apmid%252F10351940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1164/ajrccm.159.6.9805083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10351940&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080780000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.19" data-doi="10.1111/j.1365-2222.2004.1918.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dimova-Yaneva</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Main</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooker</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Helms</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-article-title">Eosinophil activation and cysteinyl leukotriene production in infants with respiratory syncytial virus bronchiolitis</span>. <abbr class="cit-jnl-abbrev">Clinical and experimental allergy : journal of the British Society for Allergy and Clinical Immunology</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">555</span>–<span class="cit-lpage">558</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2222.2004.1918.x</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Band%2Bexperimental%2Ballergy%2B%253A%2B%2Bjournal%2Bof%2Bthe%2BBritish%2BSociety%2Bfor%2BAllergy%2Band%2BClinical%2BImmunology%26rft.stitle%253DClin%2BExp%2BAllergy%26rft.aulast%253DDimova-Yaneva%26rft.auinit1%253DD.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D555%26rft.epage%253D558%26rft.atitle%253DEosinophil%2Bactivation%2Band%2Bcysteinyl%2Bleukotriene%2Bproduction%2Bin%2Binfants%2Bwith%2Brespiratory%2Bsyncytial%2Bvirus%2Bbronchiolitis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2222.2004.1918.x%26rft_id%253Dinfo%253Apmid%252F15080807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2222.2004.1918.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15080807&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220857000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.20" data-doi="10.1136/adc.71.5.428"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Everard</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of cells obtained by bronchial lavage of infants with respiratory syncytial virus infection</span>. <abbr class="cit-jnl-abbrev">Archives of disease in childhood</abbr> <span class="cit-vol">71</span>, <span class="cit-fpage">428</span>–<span class="cit-lpage">432</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BDisease%2Bin%2BChildhood%26rft.stitle%253DArch.%2BDis.%2BChild.%26rft.aulast%253DEverard%26rft.auinit1%253DM%2BL%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D428%26rft.epage%253D432%26rft.atitle%253DAnalysis%2Bof%2Bcells%2Bobtained%2Bby%2Bbronchial%2Blavage%2Bof%2Binfants%2Bwith%2Brespiratory%2Bsyncytial%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fadc.71.5.428%26rft_id%253Dinfo%253Apmid%252F7826113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImFyY2hkaXNjaGlsZCI7czo1OiJyZXNpZCI7czo4OiI3MS81LzQyOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE5Lzg0MDA5MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.21" data-doi="10.1136/adc.88.10.922"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNamara</span>,  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritson</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selby</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hart</span>,  <span class="cit-name-given-names">C. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Smyth</span>,  <span class="cit-name-given-names">R. L.</span></span> <span class="cit-article-title">Bronchoalveolar lavage cellularity in infants with severe respiratory syncytial virus bronchiolitis</span>. <abbr class="cit-jnl-abbrev">Archives of disease in childhood</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">922</span>–<span class="cit-lpage">926</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1136/adc.88.10.922</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BDisease%2Bin%2BChildhood%26rft.stitle%253DArch.%2BDis.%2BChild.%26rft.aulast%253DMcNamara%26rft.auinit1%253DP%2BS%26rft.volume%253D88%26rft.issue%253D10%26rft.spage%253D922%26rft.epage%253D926%26rft.atitle%253DBronchoalveolar%2Blavage%2Bcellularity%2Bin%2Binfants%2Bwith%2Bsevere%2Brespiratory%2Bsyncytial%2Bvirus%2Bbronchiolitis%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fadc.88.10.922%26rft_id%253Dinfo%253Apmid%252F14500316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImFyY2hkaXNjaGlsZCI7czo1OiJyZXNpZCI7czo5OiI4OC8xMC85MjIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xOS84NDAwOTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.22" data-doi="10.1152/ajplung.1998.275.5.L983"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S. Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Shedding of L-selectin and PECAM-1 and upregulation of Mac-1 and ICAM-1 on neutrophils in RSV bronchiolitis</span>. <abbr class="cit-jnl-abbrev">The American journal of physiology</abbr> <span class="cit-vol">275</span>, <span class="cit-fpage">L983</span>–<span class="cit-lpage">989</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1152/ajplung.1998.275.5.L983</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2Bjournal%2Bof%2Bphysiology%26rft.volume%253D275%26rft.spage%253DL983%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajplung.1998.275.5.L983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajplung.1998.275.5.L983&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.23" data-doi="10.1046/j.1365-2249.1998.00681.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S. Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The apoptosis of neutrophils is accelerated in respiratory syncytial virus (RSV)-induced bronchiolitis</span>. <abbr class="cit-jnl-abbrev">Clinical and experimental immunology</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1046/j.1365-2249.1998.00681.x</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Band%2Bexperimental%2Bimmunology%26rft.stitle%253DClin%2BExp%2BImmunol%26rft.aulast%253DWang%26rft.auinit1%253DS.%2BZ.%26rft.volume%253D114%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DThe%2Bapoptosis%2Bof%2Bneutrophils%2Bis%2Baccelerated%2Bin%2Brespiratory%2Bsyncytial%2Bvirus%2B%2528RSV%2529-induced%2Bbronchiolitis.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2249.1998.00681.x%26rft_id%253Dinfo%253Apmid%252F9764602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2249.1998.00681.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9764602&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076271900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.24" data-doi="10.1136/thoraxjnl-2015-207358"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kerrin</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential lower airway dendritic cell patterns may reveal distinct endotypes of RSV bronchiolitis</span>. <abbr class="cit-jnl-abbrev">Thorax</abbr> <span class="cit-vol">72</span>, <span class="cit-fpage">620</span>–<span class="cit-lpage">627</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1136/thoraxjnl-2015-207358</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThorax%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fthoraxjnl-2015-207358%26rft_id%253Dinfo%253Apmid%252F27531529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToidGhvcmF4am5sIjtzOjU6InJlc2lkIjtzOjg6IjcyLzcvNjIwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTkvODQwMDkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.25" data-doi="10.1203/PDR.0b013e31816fdc32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bem</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Activation of the granzyme pathway in children with severe respiratory syncytial virus infection</span>. <abbr class="cit-jnl-abbrev">Pediatric research</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">650</span>–<span class="cit-lpage">655</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1203/PDR.0b013e31816fdc32</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPediatric%2BResearch%26rft.stitle%253DPediatr.%2BRes.%26rft.aulast%253DBem%26rft.auinit1%253DR.%2BA.%26rft.volume%253D63%26rft.issue%253D6%26rft.spage%253D650%26rft.epage%253D655%26rft.atitle%253DActivation%2Bof%2Bthe%2Bgranzyme%2Bpathway%2Bin%2Bchildren%2Bwith%2Bsevere%2Brespiratory%2Bsyncytial%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1203%252FPDR.0b013e31816fdc32%26rft_id%253Dinfo%253Apmid%252F18317234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1203/PDR.0b013e31816fdc32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18317234&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256130500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.26" data-doi="10.1086/428589"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gill</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mobilization of plasmacytoid and myeloid dendritic cells to mucosal sites in children with respiratory syncytial virus and other viral respiratory infections</span>. <abbr class="cit-jnl-abbrev">The Journal of infectious diseases</abbr> <span class="cit-vol">191</span>, <span class="cit-fpage">1105</span>–<span class="cit-lpage">1115</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1086/428589</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Binfectious%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1086%252F428589%26rft_id%253Dinfo%253Apmid%252F15747246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/428589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15747246&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F19%2F840090.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227418200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-840090v2.27" data-doi="10.4049/jimmunol.179.12.8410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heidema</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">CD8+ T cell responses in bronchoalveolar lavage fluid and peripheral blood mononuclear cells of infants with severe primary respiratory syncytial virus infections</span>. <abbr class="cit-jnl-abbrev">Journal of immunology (Baltimore, Md. : 1950)</abbr> <span class="cit-vol">179</span>, <span class="cit-fpage">8410</span>–<span class="cit-lpage">8417</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DHeidema%26rft.auinit1%253DJ.%26rft.volume%253D179%26rft.issue%253D12%26rft.spage%253D8410%26rft.epage%253D8417%26rft.atitle%253DCD8%252B%2BT%2BCell%2BResponses%2Bin%2BBronchoalveolar%2BLavage%2BFluid%2Band%2BPeripheral%2BBlood%2BMononuclear%2BCells%2Bof%2BInfants%2Bwith%2BSevere%2BPrimary%2BRespiratory%2BSyncytial%2BVirus%2BInfections%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.179.12.8410%26rft_id%253Dinfo%253Apmid%252F18056387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE3OS8xMi84NDEwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTkvODQwMDkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/840090v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F840090v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/650861v3" title="Prediction of specific TCR-peptide binding from large dictionaries of TCR-peptide pairs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 19, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/840090v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/840090v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/840090v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/840090v2" class="active" data-icon-position="" data-hide-link-title="0">In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1112616" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-D6XFRN2Uskh39QBnY0sWA26Bi5ZLz7Fg1oKu2L89m7Y" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1112616" id="node1112616--2" data-pisa="biorxiv;840090v2" data-pisa-master="biorxiv;840090" data-seqnum="1112616" data-apath="/biorxiv/early/2020/01/19/840090.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charles J</span> <span class="nlm-surname">Sande</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jacqueline M</span> <span class="nlm-surname">Waeni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James M</span> <span class="nlm-surname">Njunge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin N</span> <span class="nlm-surname">Mutunga</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Elijah</span> <span class="nlm-surname">Gicheru</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nelson K</span> <span class="nlm-surname">Kibinge</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Agnes</span> <span class="nlm-surname">Gwela</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">840090; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/840090 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/840090v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;title=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;title=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;text=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;title=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;t=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;title=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;title=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1112616" id="node1112616--3" data-pisa="biorxiv;840090v2" data-pisa-master="biorxiv;840090" data-seqnum="1112616" data-apath="/biorxiv/early/2020/01/19/840090.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">In-silico immune cell deconvolution of the airway proteomes of infants with pneumonia reveals a link between reduced airway eosinophils and an increased risk of mortality</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Charles J</span> <span class="nlm-surname">Sande</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jacqueline M</span> <span class="nlm-surname">Waeni</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James M</span> <span class="nlm-surname">Njunge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin N</span> <span class="nlm-surname">Mutunga</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Elijah</span> <span class="nlm-surname">Gicheru</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nelson K</span> <span class="nlm-surname">Kibinge</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Agnes</span> <span class="nlm-surname">Gwela</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">840090; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/840090 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Sande%2C+Charles+J&amp;rft.au=Waeni%2C+Jacqueline+M&amp;rft.au=Njunge%2C+James+M&amp;rft.au=Mutunga%2C+Martin+N&amp;rft.au=Gicheru%2C+Elijah&amp;rft.au=Kibinge%2C+Nelson+K&amp;rft.au=Gwela%2C+Agnes"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1112616/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1112616/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1112616/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1112616/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1112616/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1112616/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1112616/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1112616/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1112616/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1112616/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1112616/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1112616/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;count=horizontal&amp;via=&amp;text=In-silico%20immune%20cell%20deconvolution%20of%20the%20airway%20proteomes%20of%20infants%20with%20pneumonia%20reveals%20a%20link%20between%20reduced%20airway%20eosinophils%20and%20an%20increased%20risk%20of%20mortality&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/840090v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/840090v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/840090v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
