<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.04.08.029769.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.08.029769v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.08.029769"/>
  <meta name="DC.Date" content="2020-04-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Coronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide[1][1]. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill[2][2]. Studies have shown that dysregulation of the COVID-19 patients’ immune system may lead to inflammatory storm and cause severe illness and even death[3][3],[4][4]. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients[5][5]. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8+ T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment. ### Competing Interest Statement [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5"/>
  <meta name="DC.Contributor" content="Chuang Guo"/>
  <meta name="DC.Contributor" content="Bin Li"/>
  <meta name="DC.Contributor" content="Huan Ma"/>
  <meta name="DC.Contributor" content="Xiaofang Wang"/>
  <meta name="DC.Contributor" content="Pengfei Cai"/>
  <meta name="DC.Contributor" content="Qiaoni Yu"/>
  <meta name="DC.Contributor" content="Lin Zhu"/>
  <meta name="DC.Contributor" content="Liying Jin"/>
  <meta name="DC.Contributor" content="Chen Jiang"/>
  <meta name="DC.Contributor" content="Jingwen Fang"/>
  <meta name="DC.Contributor" content="Qian Liu"/>
  <meta name="DC.Contributor" content="Dandan Zong"/>
  <meta name="DC.Contributor" content="Wen Zhang"/>
  <meta name="DC.Contributor" content="Yichen Lu"/>
  <meta name="DC.Contributor" content="Kun Li"/>
  <meta name="DC.Contributor" content="Xuyuan Gao"/>
  <meta name="DC.Contributor" content="Binqing Fu"/>
  <meta name="DC.Contributor" content="Lianxin Liu"/>
  <meta name="DC.Contributor" content="Xiaoling Ma"/>
  <meta name="DC.Contributor" content="Jianping Weng"/>
  <meta name="DC.Contributor" content="Haiming Wei"/>
  <meta name="DC.Contributor" content="Tengchuan Jin"/>
  <meta name="DC.Contributor" content="Jun Lin"/>
  <meta name="DC.Contributor" content="Kun Qu"/>
  <meta name="article:published_time" content="2020-04-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Coronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill. Studies have shown that dysregulation of the COVID-19 patients9 immune system may lead to inflammatory storm and cause severe illness and even death. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8+ T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.08.029769v1"/>
  <meta name="citation_id" content="2020.04.08.029769v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.08.029769"/>
  <meta name="citation_num_pages" content="22"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.08.029769"/>
  <meta name="citation_author" content="Chuang Guo"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3912-0793"/>
  <meta name="citation_author" content="Bin Li"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Huan Ma"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Xiaofang Wang"/>
  <meta name="citation_author_institution" content="Department of Hepatobiliary Surgery, the First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Pengfei Cai"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Qiaoni Yu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Lin Zhu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Liying Jin"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Chen Jiang"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Jingwen Fang"/>
  <meta name="citation_author_institution" content="HanGene Biotech"/>
  <meta name="citation_author" content="Qian Liu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Dandan Zong"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Wen Zhang"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Yichen Lu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Kun Li"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Xuyuan Gao"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Binqing Fu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_institution" content="CAS Center for Excellence in Molecular Cell Sciences, the CAS Key Laboratory of Innate Immunity and Chronic Disease, University of Science and Technology of China"/>
  <meta name="citation_author" content="Lianxin Liu"/>
  <meta name="citation_author_institution" content="Department of Hepatobiliary Surgery, the First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Xiaoling Ma"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, The First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Jianping Weng"/>
  <meta name="citation_author_institution" content="Department of Endocrinology and Metabolism, The First Affiliated Hospital of USTC, Division of Life Sciences of Medicine, University of Science and Technology of China"/>
  <meta name="citation_author" content="Haiming Wei"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_institution" content="CAS Center for Excellence in Molecular Cell Sciences, the CAS Key Laboratory of Innate Immunity and Chronic Disease, University of Science and Technology of China"/>
  <meta name="citation_author" content="Tengchuan Jin"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_institution" content="CAS Center for Excellence in Molecular Cell Sciences, the CAS Key Laboratory of Innate Immunity and Chronic Disease, University of Science and Technology of China"/>
  <meta name="citation_author_email" content="qukun@ustc.edu.cn"/>
  <meta name="citation_author_email" content="linjun7@ustc.edu.cn"/>
  <meta name="citation_author_email" content="jint@ustc.edu.cn"/>
  <meta name="citation_author" content="Jun Lin"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_institution" content="CAS Center for Excellence in Molecular Cell Sciences, the CAS Key Laboratory of Innate Immunity and Chronic Disease, University of Science and Technology of China"/>
  <meta name="citation_author_email" content="qukun@ustc.edu.cn"/>
  <meta name="citation_author_email" content="linjun7@ustc.edu.cn"/>
  <meta name="citation_author_email" content="jint@ustc.edu.cn"/>
  <meta name="citation_author" content="Kun Qu"/>
  <meta name="citation_author_institution" content="Department of oncology, The First Affiliated Hospital of USTC, Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China"/>
  <meta name="citation_author_institution" content="CAS Center for Excellence in Molecular Cell Sciences, the CAS Key Laboratory of Innate Immunity and Chronic Disease, University of Science and Technology of China"/>
  <meta name="citation_author_email" content="qukun@ustc.edu.cn"/>
  <meta name="citation_author_email" content="linjun7@ustc.edu.cn"/>
  <meta name="citation_author_email" content="jint@ustc.edu.cn"/>
  <meta name="citation_reference" content="WHO. Coronavirus disease 2019 (COVID-19) Situation Report – 72. (2020)."/>
  <meta name="citation_reference" content="Wu, Z. &amp;; McGoogan, J. M. Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72314 Cases From the Chinese Center for Disease Control and Prevention. JAMA, doi: 10.1001/jama.2020.2648 (2020)."/>
  <meta name="citation_reference" content="Mehta, P. et al. COVID-19: consider cytokine storm syndromes and immunosuppression. Lancet 395, 1033–1034, doi: 10.1016/S0140-6736(20)30628-0 (2020)."/>
  <meta name="citation_reference" content="Zhou, Y. et al. Pathogenic T cells and inflammatory monocytes incite inflammatory storm in severe COVID-19 patients. National Science Review, doi: 10.1093/nsr/nwaa041 (2020)."/>
  <meta name="citation_reference" content="Xu, X., Han, Mingfeng, Li, Tiantian, Sun, Wei, Wang, Dongsheng, Fu, Binqing, Zhou, Yonggang, Zheng, Xiaohu, Yang, Yun, Li, Xiuyong, Zhang, Xiaohua, Pan, Aijun, Wei, Haiming. Effective Treatment of Severe COVID-19 Patients with Tocilizumab. [ChinaXiv:202003.00026] (2020)."/>
  <meta name="citation_reference" content="Zumla, A., Hui, D. S., Azhar, E. I., Memish, Z. A. &amp;; Maeurer, M. Reducing mortality from 2019-nCoV: host-directed therapies should be an option. Lancet 395, e35–e36, doi: 10.1016/S0140-6736(20)30305-6 (2020)."/>
  <meta name="citation_reference" content="Chaofu Wang, J. X., Lei Zhao et al. Aveolar Macrophage Activation and Cytokine Storm in the Pathogenesis of Severe COVID-19. PREPRINT (Version 9.1 available at Research Square [+https://doi.org/10.21203/rs.3.rs-19346/v1+] (2020)."/>
  <meta name="citation_reference" content="Li, G. et al. Coronavirus infections and immune responses. J Med Virol 92, 424–432, doi: 10.1002/jmv.25685 (2020)."/>
  <meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506, doi: 10.1016/S0140-6736(20)30183-5 (2020)."/>
  <meta name="citation_reference" content="Kotch, C., Barrett, D. &amp;; Teachey, D. T. Tocilizumab for the treatment of chimeric antigen receptor T cell-induced cytokine release syndrome. Expert Rev Clin Immunol 15, 813–822, doi: 10.1080/1744666X.2019.1629904 (2019)."/>
  <meta name="citation_reference" content="Le, R. Q. et al. FDA Approval Summary: Tocilizumab for Treatment of Chimeric Antigen Receptor T Cell-Induced Severe or Life-Threatening Cytokine Release Syndrome. Oncologist 23, 943–947, doi: 10.1634/theoncologist.2018-0028 (2018)."/>
  <meta name="citation_reference" content="Zheng, G. X. et al. Massively parallel digital transcriptional profiling of single cells. Nat Commun 8, 14049, doi: 10.1038/ncomms14049 (2017)."/>
  <meta name="citation_reference" content="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp;; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol 36, 411–420, doi: 10.1038/nbt.4096 (2018)."/>
  <meta name="citation_reference" content="Norelli, M. et al. Monocyte-derived IL-1 and IL-6 are differentially required for cytokine-release syndrome and neurotoxicity due to CAR T cells. Nat Med 24, 739–748, doi: 10.1038/s41591-018-0036-4 (2018)."/>
  <meta name="citation_reference" content="Aibar, S. et al. SCENIC: single-cell regulatory network inference and clustering. Nat Methods 14, 1083–1086, doi: 10.1038/nmeth.4463 (2017)."/>
  <meta name="citation_reference" content="Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature 563, 347–353, doi: 10.1038/s41586-018-0698-6 (2018)."/>
  <meta name="citation_reference" content="Guan, W. et al. Clinical Correlations of Transcriptional Profile in Patients Infected With Avian Influenza H7N9 Virus. J Infect Dis 218, 1238–1248, doi: 10.1093/infdis/jiy317 (2018)."/>
  <meta name="citation_reference" content="Wang, Z. et al. Recovery from severe H7N9 disease is associated with diverse response mechanisms dominated by CD8(+) T cells. Nat Commun 6, 6833, doi: 10.1038/ncomms7833 (2015)."/>
  <meta name="citation_reference" content="Thevarajan, I. et al. Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19. Nature Medicine, doi: 10.1038/s41591-020-0819-2 (2020)."/>
  <meta name="citation_reference" content="Braciale, T. J., Sun, J. &amp;; Kim, T. S. Regulating the adaptive immune response to respiratory virus infection. Nat Rev Immunol 12, 295–305, doi: 10.1038/nri3166 (2012)."/>
  <meta name="citation_reference" content="Rouse, B. T. &amp;; Sehrawat, S. Immunity and immunopathology to viruses: what decides the outcome? Nat Rev Immunol 10, 514–526, doi: 10.1038/nri2802 (2010)."/>
  <meta name="citation_reference" content="Ahn, S. S. et al. Safety of Tocilizumab in Rheumatoid Arthritis Patients with Resolved Hepatitis B Virus Infection: Data from Real-World Experience. Yonsei Med J 59, 452–456, doi: 10.3349/ymj.2018.59.3.452 (2018)."/>
  <meta name="citation_reference" content="Bersanelli, M. Controversies about COVID-19 and anticancer treatment with immune checkpoint inhibitors. Immunotherapy, doi: 10.2217/imt-2020-0067 (2020)."/>
  <meta name="citation_reference" content="Zhou, Y. et al. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nat Commun 10, 1523, doi: 10.1038/s41467-019-09234-6 (2019)."/>
  <meta name="citation_reference" content="Cline, M. S. et al. Integration of biological networks and gene expression data using Cytoscape. Nat Protoc 2, 2366–2382, doi: 10.1038/nprot.2007.324 (2007)."/>
  <meta name="twitter:title" content="Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Coronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide[1][1]. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill[2][2]. Studies have shown that dysregulation of the COVID-19 patients’ immune system may lead to inflammatory storm and cause severe illness and even death[3][3],[4][4]. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients[5][5]. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8+ T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment. ### Competing Interest Statement [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5"/>
  <meta name="og-title" property="og:title" content="Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.029769v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Coronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide[1][1]. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill[2][2]. Studies have shown that dysregulation of the COVID-19 patients’ immune system may lead to inflammatory storm and cause severe illness and even death[3][3],[4][4]. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients[5][5]. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8+ T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment. ### Competing Interest Statement [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-09"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Tocilizumab treatment in severe COVID-19 patients attenuates the inflammatory storm incited by monocyte centric immune interactions revealed by single-cell analysis | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1237293 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-4">Coronavirus disease 2019 (COVID-19) has caused more than 40,000 deaths worldwide
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. Approximately 14% of patients with COVID-19 experienced severe disease and 5% were critically ill
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. Studies have shown that dysregulation of the COVID-19 patients’ immune system may lead to inflammatory storm and cause severe illness and even death
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. Tocilizumab treatment targeting interleukin 6 receptor has shown inspiring clinical results of severe COVID-19 patients
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>. However, the immune network with Tocilizumab treatment at single cell resolution has not been uncovered. Here, we profiled the single-cell transcriptomes of 13,289 peripheral blood mononuclear cells isolated at three longitudinal stages from two severe COVID-19 patients treated with Tocilizumab. We identified a severe stage-specific monocyte subpopulation and these cells centric immune cell interaction network connected by the inflammatory cytokines and their receptors. The over-activated inflammatory immune response was attenuated after Tocilizumab treatment, yet immune cells including plasma B cells and CD8
     <sup>+</sup> T cells still exhibited an intense humoral and cell-mediated anti-virus immune response in recovered COVID-19 patients. These results provided critical insights into the immunopathogenesis of severe COVID-19 and revealed fundamentals of effectiveness in Tocilizumab treatment.
    </p>
   </div>
   <p id="p-6">As of Apr 1, 2020, WHO reported 40,598 deaths out of 823,626 confirmed cases infected by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and these numbers are still growing rapidly
    <sup>
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
    </sup>. Approximately 14% of patients with COVID-19 experienced severe disease, and 5% were critically ill suffered from 49% fatality rate
    <sup>
     <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
    </sup>, which may be caused by patients’ abnormal immune system activation
    <sup>
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
    </sup>. Hence, there is an urgent need for researchers to understand how the immune system respond to the viral infections at severe stage and thereby provide effective treatment strategies.
   </p>
   <p id="p-7">Studies have shown that the inflammatory storm caused by excessive immune responses was associated with the crucial cause for mortality in COVID-19
    <sup>
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
    </sup>. Plasma concentrations of a series of inflammatory cytokines, such as granulocyte-macrophage colony-stimulating factor (GM-CSF), interleukin (IL)-6
    <sup>4</sup>, tumor necrosis factor α (TNF-α), IL-2, 7, 10 and granulocyte colony-stimulating factor (G-CSF)
    <sup>
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
    </sup> were increased after SARS-CoV-2 infections. Further investigation demonstrated peripheral inflammatory monocytes and pathogenic T cells may incite cytokine storm in severe COVID-19 patients
    <sup>
     <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
    </sup>. To calming inflammatory storm, Tocilizumab, which targeting IL-6 receptors and has proved its effectiveness in the treatment of cytokine release syndrome that is severe or life-threatening
    <sup>
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
    </sup>, was used in the treatment of severe COVID-19. After receiving Tocilizumab, the body temperature of the patients returned to normal after 24 hours. The concentration of oxygen inhalation was significantly decreased on the 5
    <sup>th</sup> day
    <sup>
     <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
    </sup>. However, the immune network arousing the inflammatory storm in severe or recovery stage during Tocilizumab therapy at single cell level has not been uncovered.
   </p>
   <p id="p-8">Here, we profiled the peripheral immune cells of COVID-19 patients using single-cell transcriptome sequencing. We obtained 5 peripheral blood samples from 2 severe COVID-19 patients at 3 consecutive time-points from the severe to recovery stages during Tocilizumab treatment (
    <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a>). Specifically, the blood samples at severe stage were collected within 12 hours of Tocilizumab was given. The blood samples at recovery stage were collected at the 5
    <sup>th</sup> and 7
    <sup>th</sup> day after Tocilizumab treatment. The patients at severe stage had decreased number of lymphocytes, increased percentage of neutrophil and concentrations of C-reaction protein, and increased expression of IL-6 (Supplementary Table 1). Peripheral blood mononuclear cells (PBMCs) were isolated and subjected to single-cell mRNA sequencing (scRNA-seq) using the 10X platform (
    <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>, Supplementary Table 2). After filtering low quality cells, we retained a total of 13,289 single transcriptomes of PBMCs. Of these, 4,364 cells were from severe stage and 8,925 were from patients at recovery stage.
   </p>
   <div id="F1" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="An atlas of peripheral immune cells in severe COVID-19 patients. a, Flowchart depicting the overall design of the study. Blood draws from patient PZ were performed at 2 time points (Day1 and Day5), and PW at 3 time points (Day1, Day5, and Day7). Patients at Day 1 were at severe stage and Day 5 and Day 7 were at recovery stages. Samples were collected within 12 hours of Tocilizumab was given at Day 1. b-d, UMAP representations of single-cell transcriptomes of 13,289 PBMCs. Cells are color-coded by clusters (b), disease stages (c), and the corresponding patient or healthy control (d). Dotted circles represented cell types with &gt;; 5% proportion of PBMCs in (b) and clusters significantly enriched in patients versus control in (c, d). Mono, monocyte; NK, natural killer cells; mDC, myeloid dendritic cells; pDC, plasmacytoid dendritic cells. e, Violin plots of selected marker genes (upper row) for multiple cell subpopulations. The left column are cell subtypes identified by combination of marker genes. f, Bar chart showing the proportion of immune cells from the severe, recovery and healthy stage in each cell subpopulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1829809926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;An atlas of peripheral immune cells in severe COVID-19 patients.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Flowchart depicting the overall design of the study. Blood draws from patient PZ were performed at 2 time points (Day1 and Day5), and PW at 3 time points (Day1, Day5, and Day7). Patients at Day 1 were at severe stage and Day 5 and Day 7 were at recovery stages. Samples were collected within 12 hours of Tocilizumab was given at Day 1. &lt;;strong&gt;;b-d&lt;;/strong&gt;;, UMAP representations of single-cell transcriptomes of 13,289 PBMCs. Cells are color-coded by clusters &lt;;strong&gt;;(b)&lt;;/strong&gt;;, disease stages &lt;;strong&gt;;(c)&lt;;/strong&gt;;, and the corresponding patient or healthy control &lt;;strong&gt;;(d)&lt;;/strong&gt;;. Dotted circles represented cell types with &gt;; 5% proportion of PBMCs in &lt;;strong&gt;;(b)&lt;;/strong&gt;; and clusters significantly enriched in patients versus control in &lt;;strong&gt;;(c, d)&lt;;/strong&gt;;. Mono, monocyte; NK, natural killer cells; mDC, myeloid dendritic cells; pDC, plasmacytoid dendritic cells. &lt;;strong&gt;;e&lt;;/strong&gt;;, Violin plots of selected marker genes (upper row) for multiple cell subpopulations. The left column are cell subtypes identified by combination of marker genes. &lt;;strong&gt;;f&lt;;/strong&gt;;, Bar chart showing the proportion of immune cells from the severe, recovery and healthy stage in each cell subpopulation.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F1.medium.gif" width="282" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F1.medium.gif" width="282" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 1</span>
     <span class="caption-title">An atlas of peripheral immune cells in severe COVID-19 patients.</span>
     <p id="p-9" class="first-child">
      <strong>a</strong>, Flowchart depicting the overall design of the study. Blood draws from patient PZ were performed at 2 time points (Day1 and Day5), and PW at 3 time points (Day1, Day5, and Day7). Patients at Day 1 were at severe stage and Day 5 and Day 7 were at recovery stages. Samples were collected within 12 hours of Tocilizumab was given at Day 1. 
      <strong>b-d</strong>, UMAP representations of single-cell transcriptomes of 13,289 PBMCs. Cells are color-coded by clusters 
      <strong>(b)</strong>, disease stages 
      <strong>(c)</strong>, and the corresponding patient or healthy control 
      <strong>(d)</strong>. Dotted circles represented cell types with &gt;; 5% proportion of PBMCs in 
      <strong>(b)</strong> and clusters significantly enriched in patients versus control in 
      <strong>(c, d)</strong>. Mono, monocyte; NK, natural killer cells; mDC, myeloid dendritic cells; pDC, plasmacytoid dendritic cells. 
      <strong>e</strong>, Violin plots of selected marker genes (upper row) for multiple cell subpopulations. The left column are cell subtypes identified by combination of marker genes. 
      <strong>f</strong>, Bar chart showing the proportion of immune cells from the severe, recovery and healthy stage in each cell subpopulation.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-10">To investigate the heterogeneity and differences of PBMCs between COVID-19 patients and healthy controls, we integrated our COVID-19 single cell transcriptomes with the published single-cell profiles of healthy PBMC from the 10X official website
    <sup>
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
    </sup> and obtained a total of 69,237 cells (See 
    <strong>Methods</strong>) (
    <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b-d</a>). We then applied Seurat
    <sup>
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
    </sup> to normalize and cluster the gene expression matrix, and identified 20 unique cell subsets, which were visualized via uniform manifold approximation and projection (UMAP) (
    <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1b-d</a>). Cell lineages, including monocytes, CD4
    <sup>+</sup> and CD8
    <sup>+</sup> T, γδT, NK, B, plasma B and myeloid dendritic cells (mDC), plasmacytoid dendritic cells (pDC), platelet and CD34
    <sup>+</sup> progenitor cells were identified based on the expression of known marker genes (
    <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1e</a>, Extended Data Fig.1a). With that, we delineated the landscape of circulating immune cells in severe COVID-19 patients.
   </p>
   <p id="p-11">We next explored the distribution of immune cells from the severe, recovery and healthy stage in each cell subpopulation (
    <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1f</a>, Extended Data Fig.1b). We observed that a monocyte subpopulation (cluster 9) existed only in patients at severe stage. Plasma B cells (cluster 11), effector CD8
    <sup>+</sup> T (cluster 6), proliferative MKI67
    <sup>+</sup>CD8
    <sup>+</sup> T cells (cluster 12) and NK cells (cluster 7) were significantly enriched in patients versus control. However, a number of subpopulations, such as γδ T cells (cluster 8), pDCs (cluster 15) and mDCs (cluster 10 and 19), most monocytes (cluster 2, 13 and 14) existed only in patients at recovery stage and healthy controls, indicating that these cell types gradually become normal after the treatment. No significant differences were observed in CD4
    <sup>+</sup> T (cluster 1 and 4), naïve CD8
    <sup>+</sup> T (cluster 3) and B cells (cluster 5) in patients versus control.
   </p>
   <p id="p-12">Monocytes were reported to play a vital role in CAR-T induced cytokine-release syndrome
    <sup>
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
    </sup> and SARS-CoV-2 infection caused inflammatory storm
    <sup>
     <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>
    </sup>, therefore we explored the features and functions of the monocyte from COVID-19 patients. We detected 1,737 monocytes in patients, 927 from severe stage and 810 from recovery stage, and integrated 9,787 monocytes from health control. The UMAP plot displayed two main clouds of monocytes that were clearly segregated (
    <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2a</a>). One monocyte subpopulation (cluster 9) was almost exclusively consisted of cells from severe stage and others (cluster 2, 13, 14, 17) were dominated by cells from the recovery and healthy stages (
    <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2b</a>), suggesting a severe-stage specific monocyte subpopulation.
   </p>
   <div id="F2" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A unique monocyte subpopulation contributes to the inflammatory storm in COVID-19 patients at severe stage. a, UMAP plot showing 4 clusters of CD14+ monocytes and 1 cluster of CD16+ monocyte. Cells are color-coded by clusters. b, Bar plot of the proportion of monocytes in cluster 9 at the severe, recovery and healthy stages. c, UMAP plots showing the expressions of selected cytokines in all monocytes clusters. d, Heatmap of differentially expressed genes (DEGs) in monocytes from pairwise comparison between severe, recovery and healthy stages. e,f, Box plot of the average expressions of genes involved in the signaling pathway [[#x201C]]Acute inflammatory response[[#x201D]] and [[#x201C]]Leukocyte chemotaxis[[#x201D]] in monocytes from severe, recovery and healthy stages. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents P value &lt;; 10[[#x2212]]100, student[[#x2019]]s t-test. g, Heatmap of the area under the curve (AUC) scores of expression regulation by transcription factors (TFs) estimated using SCENIC. Shown are the top differential TFs. h, UMAP plots showing the expressions of genes ETS2, NFIL3 and PHLDA2 in monocytes (top) and the AUC of the estimated regulon activity of the corresponding TFs, indicating the degree of expression regulation of their target genes (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1829809926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;A unique monocyte subpopulation contributes to the inflammatory storm in COVID-19 patients at severe stage.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, UMAP plot showing 4 clusters of CD14&lt;;sup&gt;;+&lt;;/sup&gt;; monocytes and 1 cluster of CD16&lt;;sup&gt;;+&lt;;/sup&gt;; monocyte. Cells are color-coded by clusters. &lt;;strong&gt;;b&lt;;/strong&gt;;, Bar plot of the proportion of monocytes in cluster 9 at the severe, recovery and healthy stages. &lt;;strong&gt;;c&lt;;/strong&gt;;, UMAP plots showing the expressions of selected cytokines in all monocytes clusters. &lt;;strong&gt;;d&lt;;/strong&gt;;, Heatmap of differentially expressed genes (DEGs) in monocytes from pairwise comparison between severe, recovery and healthy stages. &lt;;strong&gt;;e&lt;;/strong&gt;;,&lt;;strong&gt;;f&lt;;/strong&gt;;, Box plot of the average expressions of genes involved in the signaling pathway [[#x201C]]Acute inflammatory response[[#x201D]] and [[#x201C]]Leukocyte chemotaxis[[#x201D]] in monocytes from severe, recovery and healthy stages. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents &lt;;em&gt;;P&lt;;/em&gt;; value &lt;; 10&lt;;sup&gt;;[[#x2212]]100&lt;;/sup&gt;;, student[[#x2019]]s t-test. &lt;;strong&gt;;g&lt;;/strong&gt;;, Heatmap of the area under the curve (AUC) scores of expression regulation by transcription factors (TFs) estimated using SCENIC. Shown are the top differential TFs. &lt;;strong&gt;;h&lt;;/strong&gt;;, UMAP plots showing the expressions of genes &lt;;em&gt;;ETS2, NFIL3&lt;;/em&gt;; and &lt;;em&gt;;PHLDA2&lt;;/em&gt;; in monocytes (top) and the AUC of the estimated regulon activity of the corresponding TFs, indicating the degree of expression regulation of their target genes (bottom).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F2.medium.gif" width="316" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F2.medium.gif" width="316" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 2</span>
     <span class="caption-title">A unique monocyte subpopulation contributes to the inflammatory storm in COVID-19 patients at severe stage.</span>
     <p id="p-13" class="first-child">
      <strong>a</strong>, UMAP plot showing 4 clusters of CD14
      <sup>+</sup> monocytes and 1 cluster of CD16
      <sup>+</sup> monocyte. Cells are color-coded by clusters. 
      <strong>b</strong>, Bar plot of the proportion of monocytes in cluster 9 at the severe, recovery and healthy stages. 
      <strong>c</strong>, UMAP plots showing the expressions of selected cytokines in all monocytes clusters. 
      <strong>d</strong>, Heatmap of differentially expressed genes (DEGs) in monocytes from pairwise comparison between severe, recovery and healthy stages. 
      <strong>e</strong>,
      <strong>f</strong>, Box plot of the average expressions of genes involved in the signaling pathway “Acute inflammatory response” and “Leukocyte chemotaxis” in monocytes from severe, recovery and healthy stages. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents 
      <em>P</em> value &lt;; 10
      <sup>−100</sup>, student’s t-test. 
      <strong>g</strong>, Heatmap of the area under the curve (AUC) scores of expression regulation by transcription factors (TFs) estimated using SCENIC. Shown are the top differential TFs. 
      <strong>h</strong>, UMAP plots showing the expressions of genes 
      <em>ETS2, NFIL3</em> and 
      <em>PHLDA2</em> in monocytes (top) and the AUC of the estimated regulon activity of the corresponding TFs, indicating the degree of expression regulation of their target genes (bottom).
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-14">We then investigated the expressions of several selected inflammatory cytokines and observed that these genes were all significantly enriched in severe stage-specific monocytes (
    <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2c</a>, 
    <em>P</em> &lt;; 0.001, Wilcoxon rank-sum test). To further explore the transcriptional differences among the monocytes’ subtypes, we performed a pairwise comparison of the gene expressions in severe, recovery and healthy stages. We obtained 2,335 differentially expressed genes (DEGs) enriched in each stage, within which reported cytokine storm related genes, such as 
    <em>TNF</em>
    <sup>
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>
    </sup>, 
    <em>IL10</em>
    <sup>
     <em>9</em>
    </sup>, 
    <em>CCL3</em>
    <sup>
     <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>
    </sup> and 
    <em>IL6</em>
    <sup>
     <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a>
    </sup> were found significantly higher expressed in severe stage-specific monocytes (
    <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2d</a>, Supplementary Table 3). In addition, we also discovered a large number of significant and inflammatory related genes that were less reported (
    <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2d</a>, fold change &gt;; 2, 
    <em>P</em> &lt;; 10
    <sup>−3</sup>), including chemokine genes 
    <em>CCL4, CCL20, CXCL2, CXCL8</em> and 
    <em>CXCL9</em>, inflammasome activation associated genes 
    <em>NLRP3</em> and 
    <em>IL1B</em>, and complement pathway genes 
    <em>C1QA, C1QB</em> and 
    <em>C1QC</em> (Extended Data Fig.2a-c, Supplementary Table 4). These results indicated that this monocyte subpopulation may contribute to the inflammatory storm in severe COVID-19 patients.
   </p>
   <p id="p-15">We also observed that genes involved in “acute inflammatory response” and “leukocyte chemotaxis” were significantly decreased at recovery and healthy stage (
    <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig.2e</a>, f, Supplementary Table 5), suggesting that the inflammatory storm caused by this monocyte subpopulation was suppressed after the Tocilizumab treatment.
   </p>
   <p id="p-16">Next, we explored the transcription factors (TFs) that may be involved in the regulation of inflammatory storm in monocytes. We used SCENIC
    <sup>
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
    </sup> and predicted 15 TFs that may regulate genes which were enriched in severe stage-specific monocyte (
    <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 2g</a>). By constructing a gene regulatory network, we found 3 of them, namely ETS2, NFIL3 and PHLDA2 were regulating the cytokine storm relevant genes (Extended Data Fig.2d). In addition, we found the expressions of 
    <em>ETS2, NFIL3</em> and 
    <em>PHLDA2</em> and their target genes were enhanced in severe-specific monocyte subpopulation (
    <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig 2h</a>), suggesting these 3 TFs may regulate the inflammatory storm in monocytes.
   </p>
   <p id="p-17">Given that monocytes in the severe stage may be involved in the regulation of a variety of immune cell types, we used the accumulated ligand/receptor interaction database
    <sup>
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
    </sup> CellPhoneDB (
    <a href="http://www.cellphonedb.org">www.cellphonedb.org</a>) to identify alterations of molecular interactions between monocytes and all the immune cell subsets (Supplementary Table 6). We found 15 pairs of cytokines and their receptors whose interaction were significantly altered in severe versus recovery and healthy stages (
    <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3a</a>). Consistent with previous study
    <sup>
     <a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">4</a>
    </sup>, we found monocytes interacted with CD4
    <sup>+</sup> T cells and plasma B cells in patients at severe stage through the ligand/receptor pairs of IL-6/IL-6R. This interaction, together with many other cytokine storm relevant cell-cell communications were then extensively attenuated after the treatment of Tocilizumab (
    <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3b</a>), suggesting that this drug may functioning by effectively blocking the inflammatory storm in severe COVID-19 patients.
   </p>
   <div id="F3" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The monocyte-centric molecular interactions of peripheral immune cells in COVID-19 patients at severe stage. a, Dot plot of predicted interactions between monocytes and indicated immune cell types in the severe, recovery and healthy stages. P values were measured by circle sizes. The expression levels of the interacted genes were indicated by colors, scales on the right. b, Summary illustration depicting the cytokine/receptor interactions between monocytes and other types of peripheral immune cells in severe, recovery and healthy stages. Bolder lines indicated predicted enriched ligand/receptor interactions between monocytes and other immune cell types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1829809926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The monocyte-centric molecular interactions of peripheral immune cells in COVID-19 patients at severe stage.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, Dot plot of predicted interactions between monocytes and indicated immune cell types in the severe, recovery and healthy stages. &lt;;em&gt;;P&lt;;/em&gt;; values were measured by circle sizes. The expression levels of the interacted genes were indicated by colors, scales on the right. &lt;;strong&gt;;b&lt;;/strong&gt;;, Summary illustration depicting the cytokine/receptor interactions between monocytes and other types of peripheral immune cells in severe, recovery and healthy stages. Bolder lines indicated predicted enriched ligand/receptor interactions between monocytes and other immune cell types.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F3.medium.gif" width="313" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F3.medium.gif" width="313" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 3</span>
     <span class="caption-title">The monocyte-centric molecular interactions of peripheral immune cells in COVID-19 patients at severe stage.</span>
     <p id="p-18" class="first-child">
      <strong>a</strong>, Dot plot of predicted interactions between monocytes and indicated immune cell types in the severe, recovery and healthy stages. 
      <em>P</em> values were measured by circle sizes. The expression levels of the interacted genes were indicated by colors, scales on the right. 
      <strong>b</strong>, Summary illustration depicting the cytokine/receptor interactions between monocytes and other types of peripheral immune cells in severe, recovery and healthy stages. Bolder lines indicated predicted enriched ligand/receptor interactions between monocytes and other immune cell types.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-19">In addition, we also discovered many other ligand/receptor pairs involved in a broader spectrum of immune cell communications enriched at the severe stage. For example, TNF-α and its receptors, which connected monocytes with many other immune cells. Others like IL-1β and its receptor, which connected monocytes with CD8
    <sup>+</sup> T cells. Chemokines, such as CCL4L2, CCL3 and CCL4 and their receptors were also found enriched at severe stage. These cytokines and their receptors may serve as potential drug targets to treat COVID-19 patients at severe stage, and some of their inhibitors are undergoing clinical trials in multiple places around the world (Supplementary Table 7). Collectively, these findings illustrated the molecular basis of cell-cell interactions at the peripheral blood of COVID-19 patients, leading to a better understanding of the mechanisms of inflammatory storm of the disease.
   </p>
   <p id="p-20">Robust multi-factorial immune responses can be elicited against viral infection, such as avian H7N9 disease
    <sup>
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
    </sup>. A recent report has found effective immune responses from a non-severe COVID-19 patient
    <sup>
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
    </sup>. However, it is not clear whether the anti-virus immune response would be affected after Tocilizumab treatment. Therefore, the anti-virus immune responses from the humoral and cell-mediated immunity in COVID-19 patients at severe stage were compared with recovery stage and healthy controls. As expected, we found plasma B cells were barely existing in healthy controls (
    <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>). By contrast, significantly higher proportion of plasma B cells was exclusively increased in both severe and recovery stages (
    <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4a, b</a>), suggesting powerful anti-virus humoral immune responses during Tocilizumab treatment.
   </p>
   <div id="F4" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Enhanced humoral and cell-mediated immunity in severe COVID-19 patients. a, UMAP representations of B and plasma B cell clusters from the severe, recovery and healthy stages. b, Bar plot of the proportions of plasma B cells in B cell lineage from severe, recovery and healthy stages. c, UMAP representations of CD8+ T cell subtypes (left) and the distribution of cells from severe, recovery and healthy stages in each subtype (right). d, Dot plot of the expression of CCR7, PRDM1 and MKI67 in all CD8+ T cell subtypes. e, Heatmap of differentially expressed genes in effector CD8+ T cells from pairwise comparison between the severe, recovery and healthy stages. f, g, Bar plots of GO terms enriched in effector CD8+ T cells from the severe stage (f) or severe and recovery stages (g). h, i, Box plots of the average expressions of genes involved in the signaling pathway [[#x201C]]Leukocyte mediated cytotoxicity[[#x201D]] and [[#x201C]]Positive regulation of cytokine production[[#x201D]] in the effector CD8+ T cells from severe stage, recovery stage and healthy controls. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents P value &lt;; 10[[#x2212]]30." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1829809926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Enhanced humoral and cell-mediated immunity in severe COVID-19 patients.&lt;;/span&gt;; &lt;;strong&gt;;a&lt;;/strong&gt;;, UMAP representations of B and plasma B cell clusters from the severe, recovery and healthy stages. &lt;;strong&gt;;b&lt;;/strong&gt;;, Bar plot of the proportions of plasma B cells in B cell lineage from severe, recovery and healthy stages. &lt;;strong&gt;;c&lt;;/strong&gt;;, UMAP representations of CD8&lt;;sup&gt;;+&lt;;/sup&gt;; T cell subtypes (left) and the distribution of cells from severe, recovery and healthy stages in each subtype (right). &lt;;strong&gt;;d&lt;;/strong&gt;;, Dot plot of the expression of &lt;;em&gt;;CCR7, PRDM1&lt;;/em&gt;; and &lt;;em&gt;;MKI67&lt;;/em&gt;; in all CD8&lt;;sup&gt;;+&lt;;/sup&gt;; T cell subtypes. &lt;;strong&gt;;e&lt;;/strong&gt;;, Heatmap of differentially expressed genes in effector CD8&lt;;sup&gt;;+&lt;;/sup&gt;; T cells from pairwise comparison between the severe, recovery and healthy stages. &lt;;strong&gt;;f, g&lt;;/strong&gt;;, Bar plots of GO terms enriched in effector CD8&lt;;sup&gt;;+&lt;;/sup&gt;; T cells from the severe stage &lt;;strong&gt;;(f)&lt;;/strong&gt;; or severe and recovery stages &lt;;strong&gt;;(g). h, i&lt;;/strong&gt;;, Box plots of the average expressions of genes involved in the signaling pathway [[#x201C]]Leukocyte mediated cytotoxicity[[#x201D]] and [[#x201C]]Positive regulation of cytokine production[[#x201D]] in the effector CD8&lt;;sup&gt;;+&lt;;/sup&gt;; T cells from severe stage, recovery stage and healthy controls. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents &lt;;em&gt;;P&lt;;/em&gt;; value &lt;; 10&lt;;sup&gt;;[[#x2212]]30&lt;;/sup&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F4.medium.gif" width="348" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F4.medium.gif" width="348" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.029769/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 4</span>
     <span class="caption-title">Enhanced humoral and cell-mediated immunity in severe COVID-19 patients.</span>
     <p id="p-21" class="first-child">
      <strong>a</strong>, UMAP representations of B and plasma B cell clusters from the severe, recovery and healthy stages. 
      <strong>b</strong>, Bar plot of the proportions of plasma B cells in B cell lineage from severe, recovery and healthy stages. 
      <strong>c</strong>, UMAP representations of CD8
      <sup>+</sup> T cell subtypes (left) and the distribution of cells from severe, recovery and healthy stages in each subtype (right). 
      <strong>d</strong>, Dot plot of the expression of 
      <em>CCR7, PRDM1</em> and 
      <em>MKI67</em> in all CD8
      <sup>+</sup> T cell subtypes. 
      <strong>e</strong>, Heatmap of differentially expressed genes in effector CD8
      <sup>+</sup> T cells from pairwise comparison between the severe, recovery and healthy stages. 
      <strong>f, g</strong>, Bar plots of GO terms enriched in effector CD8
      <sup>+</sup> T cells from the severe stage 
      <strong>(f)</strong> or severe and recovery stages 
      <strong>(g). h, i</strong>, Box plots of the average expressions of genes involved in the signaling pathway “Leukocyte mediated cytotoxicity” and “Positive regulation of cytokine production” in the effector CD8
      <sup>+</sup> T cells from severe stage, recovery stage and healthy controls. Center line, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; points, outliers; **** represents 
      <em>P</em> value &lt;; 10
      <sup>−30</sup>.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-22">CD8
    <sup>+</sup> T cells are a critical component of cell-mediated immunity against viral infections by killing infected cells and secreting proinflammatory cytokines. To identify the anti-virus immune responses from the cell-mediated immunity during Tocilizumab treatment, we detected 12,121 CD8
    <sup>+</sup> T cells from our analysis. Clustering these cells revealed 3 subtypes: naïve CD8
    <sup>+</sup> T cells (cluster 3), effector CD8
    <sup>+</sup> T cells (cluster 6) and a subset of CD8
    <sup>+</sup> T cells with proliferation characteristics (cluster 12) (
    <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4c, d</a>). Among them, the cells from the severe patients were mainly distributed in the effector CD8
    <sup>+</sup> T cell cluster (
    <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4c, d</a>). We then conducted pairwise comparisons to identify DEGs of CD8
    <sup>+</sup> T cells between the severe, recovery and healthy stages (
    <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4e</a>, Supplementary Table 8). We found that genes enriched in severe stage showed “regulation of cell activation” signatures (
    <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4f</a>, Supplementary Table 9; 
    <em>P</em> &lt;; 10
    <sup>−10</sup>). Meanwhile, genes involved in “leukocyte mediated cytotoxicity” and “positive regulation of cytokine production” were highly enriched in CD8
    <sup>+</sup> T cells from COVID-19 patients at both severe and recovery stage (
    <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4g</a>, Supplementary Table 9; 
    <em>P</em> &lt;; 10
    <sup>−5</sup>). We further detected elevated expression of the 108 and 449 genes involved in these GO terms (
    <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4h, i</a>, Supplementary Table 10). Together, these results demonstrated the critical evidence that robust adaptive immune responses against SARS-CoV-2 infection exist in severe stage and remain after Tocilizumab treatment.
   </p>
   <p id="p-23">The immune system is crucial to fight off viral infection
    <sup>
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
    </sup>. Recent studies have illustrated that monocytes may be the main cause of exacerbation and even death of COVID-19 patients through inflammatory storms
    <sup>
     <a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">4</a>
    </sup>. In this study, we discovered a specific monocyte subpopulation that may lead to the inflammatory storm in patients at severe stage through single-cell mRNA sequencing. By analyzing the monocyte-centric ligand/receptor interactions, we revealed a severe stage-specific landscape of peripheral immune cell communication that may drive the inflammatory storm in COVID-19 patients. With that we obtained a list of cytokine storm relevant ligand/receptors that can serve as candidate drug targets to treat the disease, and provided mechanistic insights of the immunopathogenesis of COVID-19.
   </p>
   <p id="p-24">There are always questions about whether Tocilizumab treatment may affect the antiviral effect of the body
    <sup>
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
    </sup>. Our single cell profiles illustrated a sustained humoral and cell-mediated anti-virus immune response of COVID-19 patients at both severe and recovery stage. For example, the proportion of plasma B cells with antibody-secreting function were keeping at high levels and the cytotoxicity and cytokine production of effector CD8
    <sup>+</sup> T cells were also remained stable in severe COVID-19 patients after Tocilizumab treatment.
   </p>
   <p id="p-25">The distributions of NK cells were significantly different in the two patients (Extended Data Fig.1b), and the analysis of gene expression differences did not enrich significant biological functions, therefore we did not discuss them in depth. The normal functions of other cell types, such as γδT cells and DCs, were almost lost under severe conditions, and the contribution of these cells to the progression of the disease requires further investigation.</p>
   <div class="section" id="sec-1">
    <h2 class="">Methods</h2>
    <div id="sec-2" class="subsection">
     <h3>Human samples</h3>
     <p id="p-26">Peripheral blood samples were obtained from two severe COVID-19 patients. The patient severity was defined by the “Diagnosis and Treatment of COVID-19 (Trial Version 6)” which was released by The General Office of the National Health Commission and the Office of the National Administration of Traditional Chinese Medicine. Patient PZ was defined as a severe patient for his peripheral capillary oxygen saturation (SPO2) &lt;;93%. Patient PZ provided 2 blood samples at severe stage (Day 1) and recovery stage (Day 5). Patient PW was defined as critical ill for respiratory failure, multiple organ dysfunction (MOD) and SPO2 &lt;;93% under high flow oxygen (50 L/min, FIO2 50%). Patient PW provided 3 blood samples at severe stage (Day 1) and recovery stage (Day 5 and Day 7). All samples were collected from the First Affiliated Hospital of University of Science and Technology of China. Before blood draw, informed consent was obtained from each patient. Ethical approvals were obtained from the ethics committee of the First Affiliated Hospital of the University of Science and Technology of China (No. 2020-XG(H)-020).</p>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Cell Isolation</h3>
     <p id="p-27">We collected 2ml peripheral blood each time from the COVID-19 patients. Peripheral blood mononuclear cells (PBMC) were freshly isolated from the whole blood by using a density gradient centrifugation using Ficoll-Paque and cryopreserved for subsequent generation of single-cell RNA library.</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Single-cell RNA-seq</h3>
     <p id="p-28">We generated single-cell transcriptome library following the instructions of single-cell 3’ solution v2 reagent kit (10x Genomics). Briefly, after thawing, washing and counting cells, we loaded the cell suspensions onto a chromium single-cell chip along with partitioning oil, reverse transcription (RT) reagents, and a collection of gel beads that contain 3,500,000 unique 10X Barcodes. After generation of single-cell gel bead-in-emulsions (GEMs), RT was performed using a C1000 Touch™ Thermal Cycler (Bio-Rad). The amplified cDNA was purified with SPRIselect beads (Beckman Coulter). Single-cell libraries were then constructed following fragmentation, end repair, polyA-tailing, adaptor ligation, and size selection based on the manufacturer’s standard parameters. Each sequencing library was generated with unique sample index. Libraries were sequenced on the Illumina NovaSeq 6000 system.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Single-cell RNA-seq data processing</h3>
     <p id="p-29">The raw sequencing data of patients and health donors were processed using Cell Ranger (version 3.1.0) against the GRCh38 human reference genome with default parameters, and data from different patients and disease stages were combined by the Cell Ranger ‘aggr’ function. We are uploading the scRNA-seq data of PBMCs from the 2 severe COVID-19 patients to the Genome Sequence Archive at BIG Data Center and the accession number will be available upon request. We also used the scRNA-seq data of PBMCs from 2 healthy donors, which can be downloaded from the 10X genomics official website. Firstly, we filtered low quality cells using Seurat
      <sup>
       <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>
      </sup> (version 3.1.4). For cells from COVID-19 patients (PZ and PW), we retained cells with detected gene numbers between 500 and 6,000 and mitochondrial UMIs less than 10%. For cells from healthy donors, we retained cells with detected gene numbers between 300 and 5,000 and mitochondrial UMIs less than 10%. Subsequently we normalized gene counts for each cell using the “NormalizeData” function of Seurat with default parameters.
     </p>
     <p id="p-30">In downstream data processing, we used canonical correlation analysis (CCA) and the top 40 canonical components to find the “anchor” cells between patients and healthy controls. We then used the “IntegrateData” function in Seurat to integrate all the cells. We clustered cells based on the integrated dataset using Seurat with parameter “resolution=0.3” and generated 20 clusters. To display cells in a 2-dimensional space, we ran the principal component analysis (PCA) on the integrated dataset and adopted the first 50 principal components for the uniform manifold approximation and projection (UMAP) analysis.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Differential expression analysis</h3>
     <p id="p-31">To search for the differentially expressed genes (DEGs), we first assign the negative elements in the integrated expression matrix to zero. We used Wilcoxon rank-sum test to search for the DEGs between each pair of the 3 stages of cells (i.e. severe stage, recovery stage and healthy control). We applied multiple thresholds to screen for DEGs, including mean fold-change &gt;;2, 
      <em>P</em> value &lt;;0.001, and were detected in &gt;;10% of cells in at least one stage.
     </p>
     <p id="p-32">We defined stage A specific-DEGs as the intersections between the DEGs in stage A versus stage B and the DEGs in stage A versus stage C. We defined stage A and B common-DEGs as the intersections of the DEGs in stage A versus stage C and the DEGs in the stage B versus stage C, minus the DEGs between stage A and B. In this way, we obtained the specific-DEGs for each stage, and the common-DEGs for each pair of the 3 stages. We then uploaded these DEG groups to the Metascape
      <sup>
       <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
      </sup> website (
      <a href="https://metascape.org/gp/index.html#/main/step1">https://metascape.org/gp/index.html#/main/step1</a>), and used the default parameters to perform Gene Ontology (GO) analysis for each stage.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Motif enrichment and regulatory network</h3>
     <p id="p-33">We adopted SCENIC
      <sup>
       <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
      </sup> (version 1.1.2) and RcisTarget database to build the gene regulatory network of CD14
      <sup>+</sup> monocytes. Since the number of CD14
      <sup>+</sup> monocytes from healthy control (N = 9,618) was more than those from the severe and recovery stages (N = 1,607), to balance their contributions in the motif analysis, we randomly sampled 2,000 CD14
      <sup>+</sup> monocytes from the healthy control for calculation. We selected 13,344 genes that were detected in at least 100 monocytes or included in the DEGs of the 3 stages as the input features for SCENIC. With default parameters, SCENIC generated the enrichment scores of 427 motifs. We used the student’s t-test to calculate the 
      <em>P</em> values of these motifs between severe stage and healthy control, and selected severe-specific enriched motifs with fold change &gt;;1.5 and 
      <em>P</em> value &lt;; 10
      <sup>−100</sup>.
     </p>
     <p id="p-34">We then applied the enrichment scores of the severe-specific enriched motifs and the expressions of their targeted genes to Cytoscape
      <sup>
       <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
      </sup> to construct a connection map for the gene regulatory network, as shown in Extended Data Fig. 2b. The thickness of line connecting TFs and target genes represented the weight of regulatory link predicted by SCENIC.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Ligand/receptor interaction analysis</h3>
     <p id="p-35">To identify potential cellular communications between monocytes and other cell types (CD4
      <sup>+</sup> T, CD8
      <sup>+</sup> T, B, plasma B and NK cells), we applied the CellphoneDB
      <sup>
       <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>
      </sup> algorithm to the scRNA-seq profiles from the severe, recovery and healthy stages. CellphoneDB evaluated the impact of a ligand/receptor interactions based on the ligand expression in one cell type and its corresponding receptor expression in another cell type. To identify the enriched ligand/receptor interactions in patients at severe stage, we selected the ligand/receptor interactions with more significant (
      <em>P</em> value &lt;; 0.05) cell-cell interaction pairs in the severe stage than that in the recovery and healthy stages. We also included ligand/receptor pairs which were highly expressed in severe stage.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Data Availability</h3>
     <p id="p-36">We are uploading the scRNA-seq data of PBMCs from the 2 severe COVID-19 patients to the Genome Sequence Archive at BIG Data Center and the accession number will be available upon request. We also used the scRNA-seq data of PBMCs from 2 healthy donors, which can be downloaded from the 10X genomics official website.</p>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Funding</h2>
    <p id="p-37">This work was supported by the National Key R&amp;;D Program of China (2017YFA0102900 to K.Q.), the National Natural Science Foundation of China grants (81788101, 31970858, 31771428, 91940306 and 91640113 to K.Q., 31700796 to C.G. and 81871479 to J.L.), the Fundamental Research Funds for the Central Universities (to K.Q.). We thank the USTC supercomputing center and the School of Life Science Bioinformatics Center for providing supercomputing resources for this project.</p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Author Contributions</h2>
    <p id="p-38">K.Q. conceived and supervised the project; K.Q., C.G. and J.L. designed the experiments; C.G. and J.L. performed the experiments and conducted all the sample preparation for NGS with the help from H.M. and T.C.; B.L. performed the data analysis with the help from P.C., Q.Y., L.Z., L.J., C.J., Q.L., D.Z., W.Z., Y.L., K.L., X.G. and J.F; T.C., X.W., L.L. and X.M. provided COVID-19 blood samples and clinical information. K.Q., C.G., J.L. and B.L. wrote the manuscript with the help of B.F., H.W. and all the other authors.</p>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Competing interests</h2>
    <p id="p-39">Jingwen Fang is the chief executive officer of HanGen Biotech.</p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">Extended Data Figure Legends and Supplementary Tables</h2>
    <p id="p-40">
     <strong>Extended Data Figure 1 | Identification of single-cell subpopulations. a</strong>, UMAP plots showing the expressions of selected marker genes in all identified cells. 
     <strong>b</strong>, Bar chart showing the percentage of cell subpopulations in different clinical stages of patients and healthy controls.
    </p>
    <p id="p-41">
     <strong>Extended Data Figure 2 | Features of monocyte subpopulations. a-c</strong>, Bar plots of enriched GO terms of genes highly expressed in monocytes at severe stage 
     <strong>(a)</strong>, severe and recovery stages 
     <strong>(b)</strong>, and healthy controls 
     <strong>(c). d</strong>, Severe stage specific monocyte regulatory network predicted by SCENIC. Transcription factors were shown in rectangles and their target genes in circles.
    </p>
    <p id="p-42">
     <strong>Supplementary Table 1 | Baseline characteristcs and laboratory findings of COVID-19 patients in this study.</strong>
    </p>
    <p id="p-43">
     <strong>Supplementary Table 2 | Sequencing data quality.</strong>
    </p>
    <p id="p-44">
     <strong>Supplementary Table 3 | DEGs of different stages of CD14 monocytes.</strong>
    </p>
    <p id="p-45">
     <strong>Supplementary Table 4 | GOterms of DEGs of CD14 monocytes.</strong>
    </p>
    <p id="p-46">
     <strong>Supplementary Table 5 | Gene sets of GO terms in 
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2e</a> and 
      <a id="xref-fig-2-10" class="xref-fig" href="#F2">2f</a>.
     </strong>
    </p>
    <p id="p-47">
     <strong>Supplementary Table 6 | Interactions of cytokines and receptors in different stages.</strong>
    </p>
    <p id="p-48">
     <strong>Supplementary Table 7 | Drugs for targeting cytokines or their receptors.</strong>
    </p>
    <p id="p-49">
     <strong>Supplementary Table 8 | DEGs of different stages of effector CD8 T cells.</strong>
    </p>
    <p id="p-50">
     <strong>Supplementary Table 9 | GOterms of DEGs of effector CD8 T cells.</strong>
    </p>
    <p id="p-51">
     <strong>Supplementary Table 10 | Gene sets of GO terms in 
      <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4h</a> and 
      <a id="xref-fig-4-10" class="xref-fig" href="#F4">4i</a>.
     </strong>
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-2">
      <p id="p-2">Contact Information: Kun Qu, Ph.D., Division of Molecular Medicine, Hefei National Laboratory for Physical Sciences at Microscale, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui, 230027, China., Email: 
       <span class="em-link">
        <span class="em-addr">qukun{at}ustc.edu.cn</span>
       </span>, Phone: +86-551-63606257
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">Reference</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>. 
         <abbr class="cit-jnl-abbrev">Coronavirus disease 2019 (COVID-19) Situation Report – 72</abbr>. (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.2" data-doi="10.1001/jama.2020.2648">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">McGoogan</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span>
         <span class="cit-article-title">Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72314 Cases From the Chinese Center for Disease Control and Prevention</span>. 
         <abbr class="cit-jnl-abbrev">JAMA</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.2648</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJAMA%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.2648%26rft_id%253Dinfo%253Apmid%252F32091533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1001/jama.2020.2648&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32091533&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.3" data-doi="10.1016/S0140-6736(20)30628-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mehta</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">COVID-19: consider cytokine storm syndromes and immunosuppression</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">1033</span>–
         <span class="cit-lpage">1034</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30628-0</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D1033%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930628-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30628-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.4" data-doi="10.1093/nsr/nwaa041">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Pathogenic T cells and inflammatory monocytes incite inflammatory storm in severe COVID-19 patients</span>. 
         <abbr class="cit-jnl-abbrev">National Science Review</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nsr/nwaa041</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNational%2BScience%2BReview%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnsr%252Fnwaa041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nsr/nwaa041&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">Mingfeng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Tiantian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">Wei</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Dongsheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fu</span>, 
          <span class="cit-name-given-names">Binqing</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Yonggang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">Xiaohu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">Yun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Xiuyong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Xiaohua</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>, 
          <span class="cit-name-given-names">Aijun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>, 
          <span class="cit-name-given-names">Haiming</span>
         </span>. 
         <abbr class="cit-jnl-abbrev">Effective Treatment of Severe COVID-19 Patients with Tocilizumab</abbr>. [ChinaXiv:202003.00026] (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.6" data-doi="10.1016/S0140-6736(20)30305-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zumla</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>, 
          <span class="cit-name-given-names">D. S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Azhar</span>, 
          <span class="cit-name-given-names">E. I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Memish</span>, 
          <span class="cit-name-given-names">Z. A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Maeurer</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Reducing mortality from 2019-nCoV: host-directed therapies should be an option</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">e35</span>–
         <span class="cit-lpage">e36</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30305-6</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930305-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30305-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.08.029769v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chaofu Wang</span>, 
          <span class="cit-name-given-names">J. X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Lei</span>
          <span class="cit-name-surname">Zhao</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Aveolar Macrophage Activation and Cytokine Storm in the Pathogenesis of Severe COVID-19</span>. 
         <span class="cit-source">PREPRINT (Version</span>
         <span class="cit-vol">9</span>.
         <span class="cit-issue">1</span> available at Research Square [+
         <a href="https://doi.org/10.21203/rs.3.rs-19346/v1">https://doi.org/10.21203/rs.3.rs-19346/v1</a>+] (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.8" data-doi="10.1002/jmv.25685">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Coronavirus infections and immune responses</span>. 
         <abbr class="cit-jnl-abbrev">J Med Virol</abbr>
         <span class="cit-vol">92</span>, 
         <span class="cit-fpage">424</span>–
         <span class="cit-lpage">432</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.25685</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jmv.25685&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.9" data-doi="10.1016/S0140-6736(20)30183-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30183-5</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252F31986264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31986264&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.10" data-doi="10.1080/1744666X.2019.1629904">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kotch</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barrett</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Teachey</span>, 
          <span class="cit-name-given-names">D. T.</span>
         </span>
         <span class="cit-article-title">Tocilizumab for the treatment of chimeric antigen receptor T cell-induced cytokine release syndrome</span>. 
         <abbr class="cit-jnl-abbrev">Expert Rev Clin Immunol</abbr>
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">813</span>–
         <span class="cit-lpage">822</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1080/1744666X.2019.1629904</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DExpert%2BRev%2BClin%2BImmunol%26rft.volume%253D15%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.1080%252F1744666X.2019.1629904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/1744666X.2019.1629904&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.11" data-doi="10.1634/theoncologist.2018-0028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Le</span>, 
          <span class="cit-name-given-names">R. Q.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">FDA Approval Summary: Tocilizumab for Treatment of Chimeric Antigen Receptor T Cell-Induced Severe or Life-Threatening Cytokine Release Syndrome</span>. 
         <abbr class="cit-jnl-abbrev">Oncologist</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">943</span>–
         <span class="cit-lpage">947</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1634/theoncologist.2018-0028</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DOncologist%26rft_id%253Dinfo%253Adoi%252F10.1634%252Ftheoncologist.2018-0028%26rft_id%253Dinfo%253Apmid%252F29622697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTM6InRoZW9uY29sb2dpc3QiO3M6NToicmVzaWQiO3M6ODoiMjMvOC85NDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA4LjAyOTc2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.12" data-doi="10.1038/ncomms14049">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">G. X.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Massively parallel digital transcriptional profiling of single cells</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">14049</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms14049</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D14049%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14049%26rft_id%253Dinfo%253Apmid%252F28091601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms14049&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28091601&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.13" data-doi="10.1038/nbt.4096">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Butler</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffman</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smibert</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Papalexi</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Satija</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Integrating single-cell transcriptomic data across different conditions, technologies, and species</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>
         <span class="cit-vol">36</span>, 
         <span class="cit-fpage">411</span>–
         <span class="cit-lpage">420</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.4096</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D36%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4096%26rft_id%253Dinfo%253Apmid%252F29608179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.4096&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29608179&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.14" data-doi="10.1038/s41591-018-0036-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Norelli</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Monocyte-derived IL-1 and IL-6 are differentially required for cytokine-release syndrome and neurotoxicity due to CAR T cells</span>. 
         <abbr class="cit-jnl-abbrev">Nat Med</abbr>
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">739</span>–
         <span class="cit-lpage">748</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41591-018-0036-4</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D24%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-018-0036-4%26rft_id%253Dinfo%253Apmid%252F29808007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41591-018-0036-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29808007&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.15" data-doi="10.1038/nmeth.4463">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aibar</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SCENIC: single-cell regulatory network inference and clustering</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">1083</span>–
         <span class="cit-lpage">1086</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4463</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D1083%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4463%26rft_id%253Dinfo%253Apmid%252F28991892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4463&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28991892&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.16" data-doi="10.1038/s41586-018-0698-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vento-Tormo</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Single-cell reconstruction of the early maternal-fetal interface in humans</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">563</span>, 
         <span class="cit-fpage">347</span>–
         <span class="cit-lpage">353</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-018-0698-6</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D563%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0698-6%26rft_id%253Dinfo%253Apmid%252F30429548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0698-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30429548&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.17" data-doi="10.1093/infdis/jiy317">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical Correlations of Transcriptional Profile in Patients Infected With Avian Influenza H7N9 Virus</span>. 
         <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>
         <span class="cit-vol">218</span>, 
         <span class="cit-fpage">1238</span>–
         <span class="cit-lpage">1248</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/infdis/jiy317</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D218%26rft.spage%253D1238%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiy317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/infdis/jiy317&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.18" data-doi="10.1038/ncomms7833">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Recovery from severe H7N9 disease is associated with diverse response mechanisms dominated by CD8(+) T cells</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">6833</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms7833</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D6833%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7833%26rft_id%253Dinfo%253Apmid%252F25967273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms7833&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25967273&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.19" data-doi="10.1038/s41591-020-0819-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thevarajan</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19</span>. 
         <abbr class="cit-jnl-abbrev">Nature Medicine</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41591-020-0819-2</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-020-0819-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41591-020-0819-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.20" data-doi="10.1038/nri3166">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Braciale</span>, 
          <span class="cit-name-given-names">T. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">T. S.</span>
         </span>
         <span class="cit-article-title">Regulating the adaptive immune response to respiratory virus infection</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Immunol</abbr>
         <span class="cit-vol">12</span>, 
         <span class="cit-fpage">295</span>–
         <span class="cit-lpage">305</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nri3166</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DBraciale%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D305%26rft.atitle%253DRegulating%2Bthe%2Badaptive%2Bimmune%2Bresponse%2Bto%2Brespiratory%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3166%26rft_id%253Dinfo%253Apmid%252F22402670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nri3166&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22402670&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.21" data-doi="10.1038/nri2802">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rouse</span>, 
          <span class="cit-name-given-names">B. T.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sehrawat</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Immunity and immunopathology to viruses: what decides the outcome?</span>
         <abbr class="cit-jnl-abbrev">Nat Rev Immunol</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">514</span>–
         <span class="cit-lpage">526</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nri2802</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DRouse%26rft.auinit1%253DB.%2BT.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D514%26rft.epage%253D526%26rft.atitle%253DImmunity%2Band%2Bimmunopathology%2Bto%2Bviruses%253A%2Bwhat%2Bdecides%2Bthe%2Boutcome%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri2802%26rft_id%253Dinfo%253Apmid%252F20577268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nri2802&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20577268&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000279151100012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.22" data-doi="10.3349/ymj.2018.59.3.452">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ahn</span>, 
          <span class="cit-name-given-names">S. S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Safety of Tocilizumab in Rheumatoid Arthritis Patients with Resolved Hepatitis B Virus Infection: Data from Real-World Experience</span>. 
         <abbr class="cit-jnl-abbrev">Yonsei Med J</abbr>
         <span class="cit-vol">59</span>, 
         <span class="cit-fpage">452</span>–
         <span class="cit-lpage">456</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.3349/ymj.2018.59.3.452</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DYonsei%2BMed%2BJ%26rft.volume%253D59%26rft.spage%253D452%26rft_id%253Dinfo%253Adoi%252F10.3349%252Fymj.2018.59.3.452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3349/ymj.2018.59.3.452&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.23" data-doi="10.2217/imt-2020-0067">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bersanelli</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Controversies about COVID-19 and anticancer treatment with immune checkpoint inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Immunotherapy</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.2217/imt-2020-0067</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DImmunotherapy%26rft_id%253Dinfo%253Adoi%252F10.2217%252Fimt-2020-0067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2217/imt-2020-0067&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.24" data-doi="10.1038/s41467-019-09234-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Metascape provides a biologist-oriented resource for the analysis of systems-level datasets</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">1523</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-019-09234-6</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-09234-6%26rft_id%253Dinfo%253Apmid%252F30944313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41467-019-09234-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30944313&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.08.029769v1.25" data-doi="10.1038/nprot.2007.324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cline</span>, 
          <span class="cit-name-given-names">M. S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Integration of biological networks and gene expression data using Cytoscape</span>. 
         <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>
         <span class="cit-vol">2</span>, 
         <span class="cit-fpage">2366</span>–
         <span class="cit-lpage">2382</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nprot.2007.324</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DCline%26rft.auinit1%253DM.%2BS.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D2366%26rft.epage%253D2382%26rft.atitle%253DIntegration%2Bof%2Bbiological%2Bnetworks%2Band%2Bgene%2Bexpression%2Bdata%2Busing%2BCytoscape.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.324%26rft_id%253Dinfo%253Apmid%252F17947979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nprot.2007.324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17947979&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.029769.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000253139900010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
