<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.24.005561.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.24.005561v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein" />
<meta name="DC.Identifier" content="10.1101/2020.03.24.005561" />
<meta name="DC.Date" content="2020-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="A novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Besides the binding interface reported in the crystal structures, new plausible binding poses were revealed from all-atom MD simulations. The simulation data helped identify critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 are very important for the tight binding of the complex, the mutants were found to alter the binding modes of the CoV2-RBD to ACE2." />
<meta name="DC.Contributor" content="Cecylia S. Lupala" />
<meta name="DC.Contributor" content="Xuanxuan Li" />
<meta name="DC.Contributor" content="Jian Lei" />
<meta name="DC.Contributor" content="Hong Chen" />
<meta name="DC.Contributor" content="Jianxun Qi" />
<meta name="DC.Contributor" content="Haiguang Liu" />
<meta name="DC.Contributor" content="Xiao-dong Su" />
<meta name="article:published_time" content="2020-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2 as well, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Plausible binding modes between human ACE2 and the RBD were revealed from all-atom MD simulations. The simulation data further revealed critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 of the human ACE2 alter the binding modes of the CoV2-RBD to the ACE2.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.24.005561v2" />
<meta name="citation_id" content="2020.03.24.005561v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.24.005561" />
<meta name="citation_num_pages" content="27" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.24.005561" />
<meta name="citation_author" content="Cecylia S. Lupala" />
<meta name="citation_author_institution" content="Complex Systems Division, Beijing Computational Science Research Center" />
<meta name="citation_author" content="Xuanxuan Li" />
<meta name="citation_author_institution" content="Engingeering Physics Department, Tsinghua University" />
<meta name="citation_author_institution" content="Complex Systems Division, Beijing Computational Science Research Center" />
<meta name="citation_author" content="Jian Lei" />
<meta name="citation_author_institution" content="State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital, Sichuan University" />
<meta name="citation_author" content="Hong Chen" />
<meta name="citation_author_institution" content="School of Life Sciences, State Key Laboratory of Protein and Plant Gene Research and Biomedical Pioneering Innovation Center (BIOPIC), Peking University" />
<meta name="citation_author" content="Jianxun Qi" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of Microbiology, Chinese Academy of Sciences" />
<meta name="citation_author" content="Haiguang Liu" />
<meta name="citation_author_institution" content="Complex Systems Division, Beijing Computational Science Research Center" />
<meta name="citation_author_institution" content="Physics Department, Beijing Normal University" />
<meta name="citation_author_email" content="hgliu@csrc.ac.cn" />
<meta name="citation_author_email" content="xdsu@pku.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7324-6632" />
<meta name="citation_author" content="Xiao-dong Su" />
<meta name="citation_author_institution" content="School of Life Sciences, State Key Laboratory of Protein and Plant Gene Research and Biomedical Pioneering Innovation Center (BIOPIC), Peking University" />
<meta name="citation_author_email" content="hgliu@csrc.ac.cn" />
<meta name="citation_author_email" content="xdsu@pku.edu.cn" />
<meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N. Engl. J. Med. NEJMoa2001017 (2020). doi:10.1056/nejmoa2001017" />
<meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature (2020). doi:10.1038/s41586-020-2008-3" />
<meta name="citation_reference" content="Xu, X. &amp; Chen, P. Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission. Sci. CHINA Life Sci. 1–4 (2020). doi:10.1007/s11427-020-1637-5" />
<meta name="citation_reference" content="Lu, R. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574 (2020)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)." />
<meta name="citation_reference" content="Wong, S. K., Li, W., Moore, M. J., Choe, H. &amp; Farzan, M. A 193-Amino Acid Fragment of the SARS Coronavirus S Protein Efficiently Binds Angiotensinconverting Enzyme 2. J. Biol. Chem. 279, 3197–3201 (2004)." />
<meta name="citation_reference" content="Li, F., Li, W., Farzan, M. &amp; Harrison, S. C. Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science (80-.). 309, 1864–1868 (2005)." />
<meta name="citation_reference" content="Kuba, K., Imai, Y., Ohto-Nakanishi, T. &amp; Penninger, J. M. Trilogy of ACE2: A peptidase in the renin-angiotensin system, a SARS receptor, and a partner for amino acid transporters. Pharmacology and Therapeutics 128, 119–128 (2010)." />
<meta name="citation_reference" content="Bao, L. et al. The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice. bioRxiv 2020.02.07.939389 (2020). doi:10.1101/2020.02.07.939389" />
<meta name="citation_reference" content="Lan, J. et al. Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. bioRxiv 1–20 (2020). doi:10.1101/2020.02.19.956235" />
<meta name="citation_reference" content="Waterhouse, A. et al. SWISS-MODEL: Homology modelling of protein structures and complexes. Nucleic Acids Res. 46, W296–W303 (2018)." />
<meta name="citation_reference" content="Li, W. et al. Efficient Replication of Severe Acute Respiratory Syndrome Coronavirus in Mouse Cells Is Limited by Murine Angiotensin-Converting Enzyme 2. J. Virol. 78, 11429–11433 (2004)." />
<meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R. S. &amp; Li, F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J. Virol. (2020). doi:10.1128/jvi.00127-20" />
<meta name="citation_reference" content="Lee, J. et al. CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field. J. Chem. Theory Comput. 12, 405–413 (2016)." />
<meta name="citation_reference" content="Best, R. B. et al. Optimization of the additive CHARMM all-atom protein force field targeting improved sampling of the backbone φ, ψ and side-chain χ1 and χ2 Dihedral Angles. J. Chem. Theory Comput. 8, 3257–3273 (2012)." />
<meta name="citation_reference" content="Nosé, S. A unified formulation of the constant temperature molecular dynamics methods. J. Chem. Phys. 81, 511–519 (1984)." />
<meta name="citation_reference" content="Hoover, W. G. Canonical dynamics: Equilibrium phase-space distributions. Phys. Rev. A 31, 1695–1697 (1985)." />
<meta name="citation_reference" content="Parrinello, M. &amp; Rahman, A. Polymorphic transitions in single crystals: A new molecular dynamics method. J. Appl. Phys. 52, 7182–7190 (1981)." />
<meta name="citation_reference" content="Abraham, M. J. et al. Gromacs: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 1-2, 19–25 (2015)." />
<meta name="citation_reference" content="Darden, T., York, D. &amp; Pedersen, L. Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems. J. Chem. Phys. 98, 10089–10092 (1993)." />
<meta name="citation_reference" content="Hess, B., Bekker, H., Berendsen, H. J. C. &amp; Fraaije, J. G. E. M. LINCS: A Linear Constraint Solver for molecular simulations. J. Comput. Chem. 18, 1463–1472 (1997)." />
<meta name="citation_reference" content="Kumari, R., Kumar, R. &amp; Lynn, A. G-mmpbsa-A GROMACS tool for high-throughput MM-PBSA calculations. J. Chem. Inf. Model. 54, 1951–1962 (2014)." />
<meta name="citation_reference" content="Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: Visual molecular dynamics. J. Mol. Graph. 14, 33–38 (1996)." />
<meta name="citation_reference" content="Pettersen, E. F. et al. UCSF Chimera - A visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004)." />
<meta name="citation_reference" content="Li, W. et al. Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2. EMBO J. 24, 1634–1643 (2005)." />
<meta name="citation_reference" content="Lavillette, D. et al. Significant redox insensitivity of the functions of the SARS-CoV spike glycoprotein: Comparison with HIV envelope. J. Biol. Chem. 281, 9200–9204 (2006)." />
<meta name="citation_reference" content="Li, F. Structural analysis of major species barriers between humans and palm civets for severe acute respiratory syndrome coronavirus infections. J. Virol. 82, 6984–6991 (2008)." />
<meta name="citation_reference" content="Li, F. Receptor Recognition Mechanisms of Coronaviruses: a Decade of Structural Studies. J. Virol. 89, 1954–1964 (2015)." />
<meta name="citation_reference" content="Walls, A. C. et al. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell 0, (2020)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (80-.). 367, 1260–1263 (2020)." />
<meta name="citation_reference" content="Coronavirus: Hong Kong confirms a second dog is infected. (2020). Available at: https://www.scmp.com/news/hong-kong/health-environment/article/3075993/coronavirus-hong-kong-confirms-second-dog." />
<meta name="citation_reference" content="CASP. CASP and Covid-19. (2020)." />
<meta name="citation_reference" content="Institute for Protein Design. Coronavirus (SARS-CoV-2/COVID-19/nCoV-2019). (2020)." />
<meta name="citation_reference" content="ZhangLab. Structure models of all mature peptides in 2019-nCoV genome by C-I-TASSER. (2020)." />
<meta name="citation_reference" content="AlphaFold Team. Computational predictions of protein structures associated with COVID-19. DeepMind Website (2020)." />
<meta name="twitter:title" content="Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="A novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Besides the binding interface reported in the crystal structures, new plausible binding poses were revealed from all-atom MD simulations. The simulation data helped identify critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 are very important for the tight binding of the complex, the mutants were found to alter the binding modes of the CoV2-RBD to ACE2." />
<meta name="og-title" property="og:title" content="Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="A novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Besides the binding interface reported in the crystal structures, new plausible binding poses were revealed from all-atom MD simulations. The simulation data helped identify critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 are very important for the tight binding of the complex, the mutants were found to alter the binding modes of the CoV2-RBD to ACE2." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.24.005561v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.24.005561v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.24.005561v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1214872" />
    <title>Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"MdJ0Wjt5_oeSz-e5oEtpU6nzKXLWLGY0aMytDb9x9yU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1214872","apath":"\/biorxiv\/early\/2020\/03\/28\/2020.03.24.005561.atom","pisa":"biorxiv;2020.03.24.005561v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.24.005561v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.24.005561v2","title":"Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein","identifier":"node\/1214872"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022swvqNEB\\\\\\\/tPgjAZsRymudSZp+Bcmy8wecKvpeF2LKrX7R\\\\\\\/vfTVfdLM\\\\\\\/948uiqCJevMWjjnNuEn7x7eD1saP9LCASo00ZAZ4TLbza0TB2EPh4uqXj+NVpCNO8PXtfsr8DqzF4CKoNs9VhG+Yk4oSvB+lPTwrq7ghtEAMoHIAlyw0nk3VOjZsVBTo22IETJXqv+PC70IhzuK5XBPXznMFQUDGamMLKJlzPxihJ4gAvqdiDPF8pIvWrkh0u4OK2mFgQQk0rIP+NXx\\\\\\\/uWBpyEzJDOquFKEt+mRdddpQl8p+wapobFO3z8dexeITqwvIm+nMbf5qmvZJB3HA7INg6FWF6zxGp3r1HhGuPmBe2UMd7GMJHGQ8gMnqN8r1TCCQaOEup0EXo2yxFS17EZai531Ow9AoQERbfanpFhXEKCASCreC1oCOSNnQdg4n2sVqmFqyLlxiTCtJpXPu5WNqFHbVdaIW6nptgXyYEnDX+nymi6Y78q6LuqjHmQ+mPhsrun4kB3h+7vI\\\\\\\/ACtsNTC26tcbfJqj3KW2fGQ37jaxxh8hlvjCs06H3jNkIdnm+2b\\\\\\\/zFUcMDdcRltjF7TO6ZZpa8YrNU877UTWYLJng+ExGvzvCIJsUc0ccAoqhLwUONkTa9MAc5PIGqI5hHt3JlbyMQLLjUVrIKGZorxI9jrdB+bteQv54LHn5kLDVcTeYOMFsfIcaOCH2Waavp2AghV39Dvsc7svFliCqcXb4v2mjkTiitZ\\\\\\\/YqAm9z8m\\\\\\\/5x09YOnUou6riC6KkY3\\\\\\\/TZ7sJy2Ko+AluUzGguc62Y5mmq4TK5YOseeHBEA9IYgnkYZj35kpc3tqIaovqmWW2+uBOKUBZZKx1phjscC+NYclJBC\\\\\\\/+8dWoPpilNzMfYBS8GSQREV63beuyLcPTbkRgDCyDAVEgO+IjcyEQW1S1pInQc7QkBKpzcrw83mdNrHQCZgs\\\\\\\/c\\\\\\\/hik39Fta5x0u9YUP0JbUq4U1+xfD1649Cl+I3XrrY4PzHHop4l6XE7uXNyoYOAVtcHhAfkWyiaw2IyMndMba0RJkuBkB07903znuxhnFWie0GeKSfuggOVvuS1UPwOfhRj4khIdUgyWGd2nwzzxWbaEL5WgRDzipQ7++yLSvdzHKWNmG7+4vYyjkSbW29kqS6qw4QACpAyKetLU2s2zKIpvmL1xZVGAuq\\\\\\\/yWCLzrEkxrjUS91hNsYk10WGyeg0SwM6n\\\\\\\/ArajsLTK2nX3oaYBYjQKfykaW09UpbIESqhmzA+c3mJqXpRz7BSBu14FZ7g\\\\\\\/Mvd8LKht6fwCTth6jNMEUwXZCEi6qzsjclvzxTYReKqex4fL45DQDAIiJwKYRCroz2L60d\\\u0022,\\\u0022iv\\\u0022:\\\u0022Mcj4fliobg2U3BbOyfNLiQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222dfa9c5113b9546244e82a86c2962ad61e187446c418c668374423fbf042749c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.24.005561v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1214872 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.24.005561v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MkD7HUA8MlwK8U12K7uDVNmIH94UWXwjWnwFy3UtYh4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1214872" id="node1214872" data-pisa="biorxiv;2020.03.24.005561v2" data-pisa-master="biorxiv;2020.03.24.005561" data-apath="/biorxiv/early/2020/03/28/2020.03.24.005561.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cecylia S.</span> <span class="nlm-surname">Lupala</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xuanxuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jian</span> <span class="nlm-surname">Lei</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Hong</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jianxun</span> <span class="nlm-surname">Qi</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7324-6632" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Haiguang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Xiao-dong</span> <span class="nlm-surname">Su</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.24.005561 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1214872" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cecylia S. Lupala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Complex Systems Division, Beijing Computational Science Research Center</span>, Haidian, Beijing 100193, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecylia%2BS.%2BLupala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lupala%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecylia%2BS.%2BLupala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xuanxuan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Engingeering Physics Department, Tsinghua University</span>, Haidian, Beijing 100084, People’s <span class='nlm-country'>Republic of China</span></div><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Complex Systems Division, Beijing Computational Science Research Center</span>, Haidian, Beijing 100193, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuanxuan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuanxuan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jian Lei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital, Sichuan University</span>, Chengdu, Sichuan 610041, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian%2BLei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lei%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian%2BLei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hong Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>School of Life Sciences, State Key Laboratory of Protein and Plant Gene Research and Biomedical Pioneering Innovation Center (BIOPIC), Peking University</span>, Beijing 100871, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hong%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHong%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jianxun Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>f</span><span class='nlm-institution'>CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of Microbiology, Chinese Academy of Sciences</span>, Beijing, 100101, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianxun%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianxun%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Haiguang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Complex Systems Division, Beijing Computational Science Research Center</span>, Haidian, Beijing 100193, People’s <span class='nlm-country'>Republic of China</span></div><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Physics Department, Beijing Normal University</span>, Haidian, Beijing 100875, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haiguang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaiguang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7324-6632" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haiguang Liu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hgliu@csrc.ac.cn" class="" data-icon-position="" data-hide-link-title="0">hgliu@csrc.ac.cn</a>
<a href="mailto:xdsu@pku.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xdsu@pku.edu.cn</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xiao-dong Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>School of Life Sciences, State Key Laboratory of Protein and Plant Gene Research and Biomedical Pioneering Innovation Center (BIOPIC), Peking University</span>, Beijing 100871, People’s <span class='nlm-country'>Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-dong%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXiao-dong%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hgliu@csrc.ac.cn" class="" data-icon-position="" data-hide-link-title="0">hgliu@csrc.ac.cn</a>
<a href="mailto:xdsu@pku.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xdsu@pku.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.24.005561v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.24.005561v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.24.005561v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.24.005561v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.24.005561v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.24.005561v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1214872" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1214872/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">A novel coronavirus (the SARS-CoV-2) has been identified in January 2020 as the causal pathogen for COVID-19 pneumonia, an outbreak started near the end of 2019 in Wuhan, China. The SARS-CoV-2 was found to be closely related to the SARS-CoV, based on the genomic analysis. The Angiotensin converting enzyme 2 protein (ACE2) utilized by the SARS-CoV as a receptor was found to facilitate the infection of SARS-CoV-2, initiated by the binding of the spike protein to the human ACE2. Using homology modeling and molecular dynamics (MD) simulation methods, we report here the detailed structure of the ACE2 in complex with the receptor binding domain (RBD) of the SARS-CoV-2 spike protein. The predicted model is highly consistent with the experimentally determined complex structures. Besides the binding interface reported in the crystal structures, new plausible binding poses were revealed from all-atom MD simulations. The simulation data helped identify critical residues at the complex interface and provided more details about the interactions between the SARS-CoV-2 RBD and human ACE2. Two mutants mimicking rat ACE2 were modeled to study the mutation effects on RBD binding to ACE2. The simulations showed that the N-terminal helix and the K353 are very important for the tight binding of the complex, the mutants were found to alter the binding modes of the CoV2-RBD to ACE2.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">The outbreak of a new type of severe pneumonia COVID-19 started in December 2019<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> has been going on world-wide, and caused over 15,000 fatalities, infected more than 350,000 individuals globally. Although the earlier infected cases were mainly found in China before March, 2020, particularly in Hubei Province, the confirmed COVID-19 cases have been reported in more than 160 countries or territories by the end of March, 2020, and still increasing rapidly. One urgent desire in coping with this global crisis is to develop or discover drugs that can treat the diseases caused by the novel coronavirus, the SARS-CoV-2 (also known as 2019-nCoV)<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. According to the genome comparative studies, the SARS-CoV-2 belongs to the genus beta-coronavirus, with nucleotide sequence identity of about 96% compared to the closest bat coronavirus RaTG13, about 89% compared to two other bat SARS-like viruses (Bat-SL-CoVZC45 &amp; Bat-SL-CoVZXC21), and 79% compared to the SARS-CoV<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> Furthermore, the SARS-CoV-2 spike protein has a protein sequence identity of 73% for the receptor binding domain (RBD) with the SARS-CoV RBD (denoted as SARS-RBD in the following). The SARS-CoV and SARS-CoV-2 both utilize the human Angiotensin converting enzyme 2 protein (ACE2) to initiate the spike protein binding and facilitate the fusion to host cells<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. The 193-residue RBD of the SARS-CoV spike protein has been found to be sufficient to bind the human ACE2<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Based on this fact, the RBD of SARS-CoV-2 becomes a critical protein target for drug development to treat the COVID-19. When this study was started, neither the crystal structure of the SARS-CoV-2 spike protein nor the RBD segment were determined, so the homology modeling approach was applied to construct the model of the SARS-CoV-2 spike RBD in complex with the human ACE2 binding domain (denoted as CoV2-RBD/ACE2 in the following). Similar approach has been applied to predict the complex structure and estimate the binding energies<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. Because of the high sequence similarity between CoV2-RBD and SARS-RBD, the predicted structure was found to be highly consistent with the resolved crystal structures<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> (see another crystal structure at <a href="http://nmdc.cn/nCoVentry:NMDCS0000001">http://nmdc.cn/nCoVentry:NMDCS0000001</a>, PDBID: 6LZG). These structures laid the foundation for the dynamics investigation of the CoV2-RBD/ACE2 complex using computational simulation method. The predicted CoV2-RBD/ACE2 model was subjected to all-atom molecular dynamics (MD) simulations to study the binding interactions.</p><p id="p-5">Although the crystal structure and the predicted model of the CoV2-RBD/ACE2 complex provide important information about the binding interactions at the molecular interfaces, MD simulations can extend the knowledge to a dynamics regime in a fully solvated environment. The importance of the ACE2 residues was investigated by simulating the complexes with ACE2 mutants, in which partial dissociation from the ACE2 was observed within 500 ns simulations. The control simulations of the SARS-RBD/ACE2 complexes allowed the detailed comparison in receptor binding for the two different types of viruses. The results showed that the wild type CoV2-RBD/ACE2 complex is stable in 500 ns simulations, especially in the well-defined binding interface. On the other hand, the mutations on the helix-1 or K353 of the ACE2 can alter the binding, revealing new binding poses with reduced contacts compared to those in the crystal structures. The analysis of the interaction energy showed that the binding is enhanced by adjusting conformations to form more favorable interactions as the simulation progressed, consistent with the increased hydrogen bonding patterns. Furthermore, the analysis also showed that SARS-RBD and CoV2-RBD have comparable binding affinities to the ACE2, with the former slightly stronger than the latter in terms of molecular mechanics energies. The dynamic information obtained by this study shall be useful in understanding SARS-CoV-2 host interaction and for designing inhibitors to block CoV2-RBD binding.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>The computer model of the SARS-CoV-2 spike RBD in complex with human ACE2</h3><p id="p-6">The spike RBD of SARS-CoV-2 (GenBank: MN908947<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup>) comprises Cys336-Gly526 residues according to the sequence homology analysis with SARS-CoV spike RBD (see Figure S1). The predicted three-dimensional structure model of these residues was obtained with the SWISS-MODEL server<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. This predicted SARS-CoV-2 RBD model was subsequently superimposed into the X-ray structure of SARS-CoV RBD in complex with human ACE2 (PDB code:2AJF, Chain D<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>). Finally, the computer model of SARS-CoV-2 RBD with human ACE2 (CoV2-RBD/ACE2) was obtained for further simulations and analysis.</p></div><div id="sec-4" class="subsection"><h3>Mutant preparation</h3><p id="p-7">Based on the analysis of the predicted model, sequence alignment, and literature survey, two other systems containing mutations in the human ACE2 were prepared and subject to MD simulation studies. The mutant construct is based on the fact that Rat ACE2 markedly diminishes interactions with SARS spike protein<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, and it was proposed that the rat ACE2 likely has reduced binding affinity to the CoV2-RBD<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. To investigate the roles of critical residues on the ACE2, we created two mutants of the human ACE2 (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>): (1) mutant <em>mut_h1</em>, with the ACE2 N-terminal (residue 19-40) mutated to the residues of rat ACE2; and (2) mutant <em>K353H,</em> in which the highly conserved K353 was mutated to histidine (the corresponding amino acid in rat and mouse ACE2 proteins). To focus on the impact of these two binding sites, the rest of the ACE2 were kept to be the same as human ACE2.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1221396/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1221396/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1221396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-8" class="first-child">The ACE2 mutants to mimic the Rat ACE2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Molecular Dynamics Simulation and Analysis</h3><p id="p-9">The predicted model of CoV2-RBD/ACE2 complex was used as the starting models for MD simulations. The spike protein RBD domain is composed of 180 residues (323-502), while the ACE2 protein contains 597 residues (19-615) from the N-terminal domain. The simulation parameterization and equilibration were prepared for complex structures including the mutant systems, using the CHARMM-GUI webserver<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> Each system was solvated in TIP3P water and sodium chloride ions were added to neutralize the systems to a salt concentration of 150 mM. Approximately, each system was composed of about 220,000 atoms that were parametrized with the CHARMM36 force field<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>.</p><p id="p-10">After energy minimization using the steepest descent algorithm, each system was equilibrated at human body temperature 310.15 K, which was maintained by Nose-Hoover scheme with 1.0 ps coupling constant in the NVT ensemble (constant volume and temperature) for 125.0 ps under periodic boundary conditions with harmonic restraint forces applied to the complex molecules (400 kJ mol<sup>-1</sup> nm<sup>-2</sup> on backbone and 40 kJ mol<sup>-1</sup> nm<sup>-1</sup> on the side chains)<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. In the subsequent step, the harmonic restraints were removed and the NPT ensembles were simulated at one atmosphere pressure (10<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup> Pa) and body temperature. The pressure was maintained by isotropic Parrinello-Rahman barostat<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>, with a compressibility of 4.5 □ × □ 10<sup>-5</sup> bar<sup>-1</sup> and coupling time constant of 5.0 □ ps. The simulation trajectories were propagated to 500 nanoseconds using the GROMACS 5.1.2 package<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup></p><p id="p-11">Three independent trajectories starting from random velocities based on Maxwell distributions were simulated for both CoV2-RBD/ACE2 and SARS-RBD/ACE2 complex systems in their wild types. In all simulations, a time step of 2.0 fs was used and the PME (particle mesh Ewald)<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> was applied for long-range electrostatic interactions beyond 12.0 Å. The van der Waals interactions were evaluated within the distance cutoff of 12.0 Å. Hydrogen atoms were constrained using the LINCS algorithm<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>.</p><p id="p-12">The human ACE2 mutants in complex with CoV2-RBD were constructed as described previously. Each mutant complex model was simulated in two independent trajectories. Furthermore, as the crystal structure of the CoV2-RBD/ACE2 complex became available, two additional simulations were carried out using the crystal structure as the starting model to cross-validate the simulation results based on the homology model. Each trajectory was propagated to 500 ns by following the same protocol as the wild type CoV2-RBD/ACE2 complex simulations.</p><p id="p-13">Analyses were carried out with tools in GROMACS (rmsd, rmsf, energy, and pairdist) to examine the system properties, including the overall stability, local residue and general structure fluctuations through the simulations. The <em>g_mmpbsa</em> program<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> was applied to extract the molecular mechanics energy E<sub>MM</sub> (Lennard Jones and electrostatic interactions) between ACE2 and the RBD of spike proteins. VMD and Chimera were applied to analyze the hydrogen bonds, molecular binding interface, water distributions, visualization, and rending model images<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup></p></div></div><div class="section" id="sec-6"><h2 class="">Results</h2><p id="p-14">The homology structure of the CoV2-RBD/ACE2 was compared to both the SARS-RBD/ACE2 crystal structure and the newly resolved crystal structure of the CoV2-RBD/ACE2. The results indicated that the homology model is accurate, especially at the binding interface. The MD simulations further refined the side chain orientations to improve the model quality. The simulation data confirmed the highly stable binding between the CoV2-RBD and the ACE2, in spite of the conformational changes of the ACE2. The relative movement between the CoV2-RBD and the ACE2 mainly exhibited as a swinging motion pivoted at the binding interface. Simulations also revealed the roles of water molecules in the binding of the RBD to the ACE2 receptor. The MD simulation of complex with ACE2 mutants suggested that mutation to the ACE2 helix-1 and the K353 can alter the binding modes and binding affinity.</p><div id="sec-7" class="subsection"><h3>1. Homology modeling and comparison to the SARS-RBD/ACE2 complex</h3><p id="p-15">The predicted CoV2-RBD/ACE2 complex structure is highly similar to the SARS-RBD/ACE2, as shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The RBD domain has an RMSD of 0.99 Å for the aligned residues (1.53 Å for all 174 residue pairs), indicating that the homology model of the CoV2-RBD is in good agreement with the SARS-RBD. For ACE2 residues near the binding interface (within 4.0 Å of the RBD), the RMSD is smaller than 0.43 Å compared to the SARS-RBD/ACE2 complex. The superposed structures revealed that the RBD/ACE2 interfaces are almost identical in two complexes(<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1c</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The predicted structure and the binding interface. (a) Predicted model of the CoV2-RBD/ACE2; (b) Crystal structure of the SARS-RBD/ACE2 (PDB: 2AJF); (c) Superposed models. The binding interface is indicated in the boxed region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The predicted structure and the binding interface.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) Predicted model of the CoV2-RBD/ACE2; (&lt;strong&gt;b&lt;/strong&gt;) Crystal structure of the SARS-RBD/ACE2 (PDB: 2AJF); (&lt;strong&gt;c&lt;/strong&gt;) Superposed models. The binding interface is indicated in the boxed region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F1.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F1.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">The predicted structure and the binding interface.</span><p id="p-16" class="first-child">(<strong>a</strong>) Predicted model of the CoV2-RBD/ACE2; (<strong>b</strong>) Crystal structure of the SARS-RBD/ACE2 (PDB: 2AJF); (<strong>c</strong>) Superposed models. The binding interface is indicated in the boxed region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">In a retrospective comparison, the homology model was superposed to a newly resolved crystal structure (PDB ID: 6LZG; see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2d</a> for a detailed comparison at the interface). The results indicate that the homology model is very accurate, especially for the binding interface. The residues near the CoV2-RBD/ACE2 interface (defined as the combined set of ACE2 residues within 4.0 Å of RBD and the RBD residues within 4.0 Å of the ACE2) exhibited a difference of 0.43 Å RMSD, which is comparable to the difference between the two independently reported crystal models (an RMSD of 0.25 Å for the same comparison). The RMSD is about 0.77 Å for residues in an extended region within 10.0 Å of the binding interface. The RBD domain of the spike protein showed an overall RMSD values less than 1.5 Å, and the ACE2 domain with an RMSD about 2.0 Å between the predicted model and the crystal structures.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative structures extracted from simulation trajectories using clustering analysis. (a) The most populated structure from simulation trajectory#1, with the loop (L67, S477-G485) between &#x3B2;6 and &#x3B2;7 moved towards the ACE2 helix-1, forming additional contacts. (b, c) The most populated structures in the simulations #2 and #3. The complex structures and the binding interfaces are similar to the predicted model and the crystal structure. (d) The comparison of initial (cyan) and representative (pink) structures to the crystal structure (gold) of CoV2-RBD/ACE2. The side chain of K31 (encircled in the red boxes) was refined to the correct position during the MD simulations (right panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Representative structures extracted from simulation trajectories using clustering analysis.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) The most populated structure from simulation trajectory#1, with the loop (L&lt;sub&gt;67&lt;/sub&gt;, S477-G485) between &#x3B2;6 and &#x3B2;7 moved towards the ACE2 helix-1, forming additional contacts. (&lt;strong&gt;b, c&lt;/strong&gt;) The most populated structures in the simulations #2 and #3. The complex structures and the binding interfaces are similar to the predicted model and the crystal structure. (&lt;strong&gt;d&lt;/strong&gt;) The comparison of initial (cyan) and representative (pink) structures to the crystal structure (gold) of CoV2-RBD/ACE2. The side chain of K31 (encircled in the red boxes) was refined to the correct position during the MD simulations (right panel).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-3.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-3.medium.gif" width="440" height="206"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative structures extracted from simulation trajectories using clustering analysis. (a) The most populated structure from simulation trajectory#1, with the loop (L67, S477-G485) between &#x3B2;6 and &#x3B2;7 moved towards the ACE2 helix-1, forming additional contacts. (b, c) The most populated structures in the simulations #2 and #3. The complex structures and the binding interfaces are similar to the predicted model and the crystal structure. (d) The comparison of initial (cyan) and representative (pink) structures to the crystal structure (gold) of CoV2-RBD/ACE2. The side chain of K31 (encircled in the red boxes) was refined to the correct position during the MD simulations (right panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Representative structures extracted from simulation trajectories using clustering analysis.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) The most populated structure from simulation trajectory#1, with the loop (L&lt;sub&gt;67&lt;/sub&gt;, S477-G485) between &#x3B2;6 and &#x3B2;7 moved towards the ACE2 helix-1, forming additional contacts. (&lt;strong&gt;b, c&lt;/strong&gt;) The most populated structures in the simulations #2 and #3. The complex structures and the binding interfaces are similar to the predicted model and the crystal structure. (&lt;strong&gt;d&lt;/strong&gt;) The comparison of initial (cyan) and representative (pink) structures to the crystal structure (gold) of CoV2-RBD/ACE2. The side chain of K31 (encircled in the red boxes) was refined to the correct position during the MD simulations (right panel).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-4.medium.gif" width="440" height="147"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-4.medium.gif" width="440" height="147"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F2/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Representative structures extracted from simulation trajectories using clustering analysis.</span><p id="p-18" class="first-child">(<strong>a</strong>) The most populated structure from simulation trajectory#1, with the loop (L<sub>67</sub>, S477-G485) between β6 and β7 moved towards the ACE2 helix-1, forming additional contacts. (<strong>b, c</strong>) The most populated structures in the simulations #2 and #3. The complex structures and the binding interfaces are similar to the predicted model and the crystal structure. (<strong>d</strong>) The comparison of initial (cyan) and representative (pink) structures to the crystal structure (gold) of CoV2-RBD/ACE2. The side chain of K31 (encircled in the red boxes) was refined to the correct position during the MD simulations (right panel).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>2. The conformational changes of the ACE2 and the CoV2-RBD</h3><p id="p-19">The structural comparison with respect to the starting structure revealed that MD simulations mainly sampled the structures in the conformational space near the starting model. The RMSD with respect to the starting model indicated that the sampled structures were mostly 2.5 Å to 5.0 Å compared to the starting model (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). The largest contribution to the conformational changes in CoV2-RBD is from the loop regions, indicated by the peaks shown in the fluctuation plots (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>). The residues with well-defined secondary structures (mainly β-strands) exhibited very small conformational changes (the RMSD is about 1.0 Å including side chain atoms), while the loop regions have RMSD values between 3.7 Å to 5.4 Å. This is consistent with the residue fluctuations measured with the RMSF (root-mean-square-fluctuation) for the RBD subunit (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a>). For the ACE2 receptor domain, the major conformational change is the opening/closing of the enzymatic active site, which is remote from the RBD binding interface (see Video 1). The CoV2-RBD remained bound to human ACE2, even though substantial conformational changes occurred to the ACE2 and the loop region of the RBD. This observation implies that the binding of CoV2-RBD to the ACE2 is very stable, similar with the case of the SARS-RBD that binds to both open and closed forms of human ACE2<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The dynamics analysis of the simulation trajectory. (a) The root-meansquare-deviation (RMSD) of the simulated structures compared to the initial and representative models. (b) The residue level fluctuations of CoV2-RBD in three simulations. The shaded area corresponds to the CoV2-RBD binding region to the ACE2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The dynamics analysis of the simulation trajectory.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) The root-meansquare-deviation (RMSD) of the simulated structures compared to the initial and representative models. (&lt;strong&gt;b&lt;/strong&gt;) The residue level fluctuations of CoV2-RBD in three simulations. The shaded area corresponds to the CoV2-RBD binding region to the ACE2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F3.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F3.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">The dynamics analysis of the simulation trajectory.</span><p id="p-20" class="first-child">(<strong>a</strong>) The root-meansquare-deviation (RMSD) of the simulated structures compared to the initial and representative models. (<strong>b</strong>) The residue level fluctuations of CoV2-RBD in three simulations. The shaded area corresponds to the CoV2-RBD binding region to the ACE2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Based on the structural similarity, a clustering analysis was carried out to identify the most populated structures, from which the representative model for each trajectory was selected (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). The CoV2-RBD/ACE2 interfaces were highlighted using blue/red colors with stick representations. The RMSD of structures in the simulations were calculated with respect to the these representative models, in order to assess the model convergence and coverage of the simulated conformations (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3a</a>, right panels). The RMSD values indicated that a large portions of the structures sampled by the MD simulations are similar to the representative model (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3a</a>, for trajectory 1, see Figure S2 for the other trajectories). Furthermore, as the simulation progressed, the conformations converged to the representative model obtained from this trajectory. This indicates that the selected structure is plausible and stable. The representative structures for the other two trajectories showed smaller deviations compared to the predicted model (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">2c</a>).</p><p id="p-22">In three simulations of the CoV2-RBD/ACE2 systems, the binding interface was highly stable, exhibiting very small conformational changes, especially for the interfacing residues of the ACE2 protein. The RMSD for the residues at the RBD binding interface is 0.85Å (+/-0.13Å) on average. Side chain atom positions were refined to form more favorable interactions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2d</a>). One outstanding example is the K31 side-chain, which pointed in the wrong orientation in the predicted structure, was quickly refined to the correct orientation, consistent with the crystal structure (right panel of <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2d</a>).</p><p id="p-23">In terms of collective conformational changes, the CoV2-RBD/ACE2 complex showed two interesting movements: (1) the loop (L<sub>67</sub>) between β<sub>6</sub> and β<sub>7</sub> (residues between S477 and G485 in particular) of CoV2-RBD was found to expand its interactions with the N-terminal helix (the helix-1) of the ACE2 (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2a</a>), while it pointed away from the helix-1 in the predicted and the crystal structures (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2d</a>, left); (2) a tilting movement of the RBD relative to the ACE2 was observed, which can be depicted as a swinging motion with the binding interface as the pivot (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> for an illustration). In both predicted and the crystal structures of the CoV2-RBD/ACE2 complex, the L<sub>67</sub> does not form close contacts with the ACE2. The analysis of the crystal packing revealed that this loop participated in the interaction with another asymmetric unit (see Figure S3). Interestingly, the simulation data suggested that the L<sub>67</sub> could move towards the ACE2 and form contacts with the helix-1. This can potentially enhance the binding, as reflected in the change of interaction energies. In the crystal structure, the C480 and C488 of the RBD are crosslinked via a disulfide bond that reduces the flexibility of the L<sub>67</sub> region, limiting its access to the ACE2. On the other hand, it has been reported that the binding of SARS-RBD to ACE2 is insensitive to the redox states of the cysteines to a high extend<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Based on the simulation results, we hypothesize that the reduced form of C480 and C488 can also exist during the virus invasion to host cells, and the reduced cysteines can potentially enhance the binding to ACE2. In the other two simulation trajectories, we found that the L<sub>67</sub> remained in conformations similar to that in the crystal structure and the cysteines (C480 and C488) were close enough for disulfide bond formation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The collective motion of the CoV2-RBD/ACE2 complex. (a) The representative model (blue) from wild type CoV2-RBD/ACE2 simulations superposed to the initial model (silver). The arrows indicate the directions of the collective motion, and the hinge region is colored in red. (b) The same representation for the CoV2-RBD complexed with the ACE2 h1-mut mutant (green), and comparison between the initial model (silver) and the representative model (green) indicates a large tilting movement compared to the wild type system." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The collective motion of the CoV2-RBD/ACE2 complex.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; The representative model (blue) from wild type CoV2-RBD/ACE2 simulations superposed to the initial model (silver). The arrows indicate the directions of the collective motion, and the hinge region is colored in red. &lt;strong&gt;(b)&lt;/strong&gt; The same representation for the CoV2-RBD complexed with the ACE2 h1-mut mutant (green), and comparison between the initial model (silver) and the representative model (green) indicates a large tilting movement compared to the wild type system.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F4.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F4.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">The collective motion of the CoV2-RBD/ACE2 complex.</span><p id="p-24" class="first-child"><strong>(a)</strong> The representative model (blue) from wild type CoV2-RBD/ACE2 simulations superposed to the initial model (silver). The arrows indicate the directions of the collective motion, and the hinge region is colored in red. <strong>(b)</strong> The same representation for the CoV2-RBD complexed with the ACE2 h1-mut mutant (green), and comparison between the initial model (silver) and the representative model (green) indicates a large tilting movement compared to the wild type system.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>3. Simulations revealed detailed binding interface interactions between the CoV2-RBD and the ACE2</h3><p id="p-25">By examining the binding interface of CoV2-RBD and the ACE2, we found the polar and charged residues account for a large fraction, therefore the electrostatic interactions play critical roles for the complex formation. Based on the distances between the two proteins, the key residues at the binding interface were identified and summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for the three representative models (see <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). Majority of these residues are conserved for these three models, except that the model#1 (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2a</a>) has additional contacts to the ACE2 from residues in the L<sub>67</sub> region (highlighted with green color in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). As shown in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>, the L<sub>67</sub> remained in the starting position for the other two representative models (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2b,c</a>). The same simulations were carried out for the SARS-RBD/ACE2 complex, serving as a comparative system. Interestingly, the SARS-RBD counterpart of the L<sub>67</sub> in CoV2-RBD did not form close contacts with the ACE2 in three simulations. It is worthwhile to mention that the sequence identity between CoV2-RBD and SARS-RBD is low in this loop region, suggesting the loop region might be partially responsible for the difference in the receptor binding.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1221395/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1221395/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1221395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Contact residues between the CoV2-RBD and the ACE2.</span><p id="p-26" class="first-child">Green color denotes new interaction not observed in crystal structure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The hydrogen bonds between the CoV2-RBD and ACE2 were extracted using VMD program with default criteria (D-A distance cutoff=3.0 Å and D-H-A angle cutoff=20 degrees, where D,A,H are Donor atom, Acceptor atom, and the Hydrogen atom linked to the Donor atom). The numbers of hydrogen bonds are summarized in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for the simulation trajectory#1. Because of the stringent criteria for hydrogen bonds, the numbers are smaller compared to the reported values in other studies. For a larger distance cutoff of 3.9Å, there are about 8.3 hydrogen bonds on average (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a>). We observed that the number of hydrogen bonds fluctuated over time, and exhibiting an overall increasing trend indicated by the fitted lines (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Similar trends were observed in the other simulations, suggesting that the binding became stronger as the simulation progressed (see Figure S4). Based on the statistics of three simulation trajectories, the CoV2-RBD/ACE2 complex has 2.7 hydrogen bonds between the subunits on average with stringent criteria. In comparison, the SARS-RBD/ACE2 has 3.2 hydrogen bonds on average (see Figure S4 for details). The statistics of hydrogen bonds suggest a slightly weaker binding between the CoV2-RBD and the ACE2, relative to the SARS-RBD/ACE2 complex.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The hydrogen bonds between the CoV2-RBD and the ACE2. (a) The number of hydrogen bonds extracted from the simulation data using Donor-Receptor distance cutoff of 3.0 &#xC5;. (b) Same as (a) except that the cutoff distance was relaxed to 3.9 &#xC5;. The trend lines were obtained by least square fitting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The hydrogen bonds between the CoV2-RBD and the ACE2.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) The number of hydrogen bonds extracted from the simulation data using Donor-Receptor distance cutoff of 3.0 &#xC5;. (&lt;strong&gt;b&lt;/strong&gt;) Same as (a) except that the cutoff distance was relaxed to 3.9 &#xC5;. The trend lines were obtained by least square fitting.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F5.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F5.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">The hydrogen bonds between the CoV2-RBD and the ACE2.</span><p id="p-28" class="first-child">(<strong>a</strong>) The number of hydrogen bonds extracted from the simulation data using Donor-Receptor distance cutoff of 3.0 Å. (<strong>b</strong>) Same as (a) except that the cutoff distance was relaxed to 3.9 Å. The trend lines were obtained by least square fitting.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">It is also noteworthy to point out the important roles of water molecules at the complex interface for CoV2-RBD/ACE2 complex. At any instant time, there are approximately 15 water molecules at the binding interface, simultaneously located within 2.5 Å of both the CoV2-RBD and the ACE2 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). These water molecules can function as bridges by forming hydrogen bonds with the residues from the RBD or the ACE2. The dwelling time of water molecules at the interface can be a few nanoseconds, as revealed by the MD simulations. This results is also consistent with the crystal structure, which has 12 water molecules at the interface (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6c</a>). These discoveries emphasize the role of the water molecules, which desires detailed quantification to understand the interactions between the RBD and the ACE2.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The water molecules at the binding interface. (a) The number of water molecules that bridge the CoV2-RBD and ACE2 found in a simulation trajectory. (b) The water molecules at the interface for the initial structure after equilibration. (c) The water molecules at the interface as observed in the crystal structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The water molecules at the binding interface.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) The number of water molecules that bridge the CoV2-RBD and ACE2 found in a simulation trajectory. (&lt;strong&gt;b&lt;/strong&gt;) The water molecules at the interface for the initial structure after equilibration. (&lt;strong&gt;c&lt;/strong&gt;) The water molecules at the interface as observed in the crystal structure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F6.medium.gif" width="440" height="135"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F6.medium.gif" width="440" height="135"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">The water molecules at the binding interface.</span><p id="p-30" class="first-child">(<strong>a</strong>) The number of water molecules that bridge the CoV2-RBD and ACE2 found in a simulation trajectory. (<strong>b</strong>) The water molecules at the interface for the initial structure after equilibration. (<strong>c</strong>) The water molecules at the interface as observed in the crystal structure.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>4. Simulation of the human ACE2 mutants uncovers new binding modes</h3><p id="p-31">It has been demonstrated that the ACE2 from several mammalian species possess high sequence similarities, yet their binding to the SARS-RBD differs significantly. In particular, the binding of SARS-RBD to the rat ACE2 is much weaker as discovered in experiments<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup> Inspired by these information, two mutants of the CoV2-RBD/ACE2 were constructed: (1) <em>ACE2-mut-h1</em> by mutating N-terminal helix-1 to that of the rat ACE2; (2) <em>ACE2-K353H</em> by mutating K353 to Histidine (the amino acid in wild type Rat ACE2). Two 500-ns MD simulations were carried out for each mutant system. The simulation showed that the mutations in <em>ACE2-mut-h1</em> reduced the interaction between the CoV2-RBD and the helix-1, and the <em>ACE2-K353H</em> showed weaker binding between the CoV2-RBD and the β-hairpin centered at the H353. Using the clustering analysis, the representative structures were identified from each simulation trajectory (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Although the overall topology is very similar to the wild type complex structure, there are pronounced differences. For the <em>ACE2-mut-h1</em> system, the CoV2-RBD tilted further away from the ACE12 helix-1 in one simulation (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7a</a>); and the CoV2-RBD lost its contact with helix-13 (G326 to N330) in another simulation for the <em>ACE2-K353H</em> (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7c</a>). In the wild type ACE2, the K353 is a hydrogen donor, and its mutant H353 cannot form the hydrogen bond with the CoV2-RBD as in the wild type CoV2-RBD/ACE2 complex. The number of contacting residue pairs was significantly reduced in the <em>ACE2-K353H</em> mutant system. This is in line with the report that K353 is more critical than the other residues, as its hydrophobic neighborhood enables this positively charged residue high selectivity to the RBD<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The representative structures for the ACE2-mutants in complex with the CoV2-RBD. (a, b) ACE2-mut-h1, in which the human ACE2 helix-1 (residues 19-40) was mutated to the Rat sequence. The structure in (a) is the same as the green model shown in Figure 4b, which illustrates a pronounced tilting movement. (c, d) ACE2-K353H, where the ACE2 K353 was mutated to a histidine. The CoV2-RBD in (c) lost its contact with the Helix-13 of the ACE2 (residues 498-503, indicated by the black arrow). Each structure was obtained by the clustering analysis of 500ns MD simulation trajectories." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The representative structures for the ACE2-mutants in complex with the CoV2-RBD.&lt;/span&gt; (&lt;strong&gt;a, b&lt;/strong&gt;) &lt;em&gt;ACE2-mut-h1&lt;/em&gt;, in which the human ACE2 helix-1 (residues 19-40) was mutated to the Rat sequence. The structure in (a) is the same as the green model shown in Figure 4b, which illustrates a pronounced tilting movement. (&lt;strong&gt;c, d&lt;/strong&gt;) &lt;em&gt;ACE2-K353H,&lt;/em&gt; where the ACE2 K353 was mutated to a histidine. The CoV2-RBD in (c) lost its contact with the Helix-13 of the ACE2 (residues 498-503, indicated by the black arrow). Each structure was obtained by the clustering analysis of 500ns MD simulation trajectories.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F7.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F7.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">The representative structures for the ACE2-mutants in complex with the CoV2-RBD.</span><p id="p-32" class="first-child">(<strong>a, b</strong>) <em>ACE2-mut-h1</em>, in which the human ACE2 helix-1 (residues 19-40) was mutated to the Rat sequence. The structure in (a) is the same as the green model shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>, which illustrates a pronounced tilting movement. (<strong>c, d</strong>) <em>ACE2-K353H,</em> where the ACE2 K353 was mutated to a histidine. The CoV2-RBD in (c) lost its contact with the Helix-13 of the ACE2 (residues 498-503, indicated by the black arrow). Each structure was obtained by the clustering analysis of 500ns MD simulation trajectories.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The physical interactions between the RBD and the ACE2 were quantified for the simulated structures. We considered the molecular mechanics energy <em>E<sub>MM</sub></em>, which is composed of the van der Waals and the electrostatic interactions. Furthermore, the number of residue contacts <em>(NC)</em> between (RBD and ACE2) was extracted from simulated structures. Both the <em>E<sub>MM</sub></em> and <em>NC</em> indicate that the RBD interactions with the ACE2 are comparable for CoV2 and SARS spike proteins (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). From the simulations, the CoV2-RBD has more close contacts with the ACE2 than the SARS-RBD: the numbers of contacts for CoV2-RBD/ACE2 and SARS-RBD/ACE2 are 32.9 and 30.4 respectively. In terms of interaction energies, CoV2-RBD showed slightly weaker interactions with the ACE2 compared to the SARS-RBD (<em>E<sub>MM</sub></em> values are −1017.1 kJ/mol vs. −1133.4 kJ/mol). We would like to point out that the energy E<sub>MM</sub> is the physical interaction between the RBD and the ACE2, rather than the binding energy, which requires accurately incorporating solvation energy and entropy. Furthermore, the standard deviations of <em>E<sub>MM</sub></em> are 70.2 kJ/mol and 65.5 kJ/mol for the two complexes. Therefore, we infer that the binding affinities are comparable for CoV2-RBD/ACE2 and SARS-RBD/ACE2. The simulations started from the predicted and crystal models yielded very similar results (purple triangles). This is in line with a recent study, in which the authors showed similar binding affinity to human ACE2 for both SARS-CoV-2 and SARS-CoV spike proteins<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> They found the association rate constants <em>k<sub>on</sub></em> to be the same at 1.4×10<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup> M<sup>-1</sup>s<sup>-1</sup>, while the SARS-CoV spike protein showed a faster dissociation, with the rate constant <em>k<sub>off</sub></em> to be 7.1×10<sup>-4</sup> s<sup>-1</sup>, about 4.4 times larger than the SARS-CoV-2 spike protein <em>k<sub>off</sub></em> = 1.6×10<sup>-4</sup> s<sup>-1</sup>. similar <em>k<sub>on</sub></em> values and the equilibrium dissociation constants <em>K<sub>D</sub></em> in nanomolar range were reported in other studies for SARS-CoV-2 spike protein (or RBD) binding to human ACE2<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The interaction energies and the residue contacts between the RBD and the ACE2. The molecular mechanics interactions and the number of contacting residue pairs were obtained from the structures sampled by MD simulations. In the legend, the CoV2 and SARS indicate the sources of the RBD; the WT, K353H, mut_h1 are the abbreviations for different constructs of the human ACE2, corresponding to the wild type, the mut_h1, and the K353H." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-102884178" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The interaction energies and the residue contacts between the RBD and the ACE2.&lt;/span&gt; The molecular mechanics interactions and the number of contacting residue pairs were obtained from the structures sampled by MD simulations. In the legend, the CoV2 and SARS indicate the sources of the RBD; the WT, K353H, mut_h1 are the abbreviations for different constructs of the human ACE2, corresponding to the wild type, the mut_h1, and the K353H.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F8.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F8.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.24.005561/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <span class="caption-title">The interaction energies and the residue contacts between the RBD and the ACE2.</span><p id="p-34" class="first-child">The molecular mechanics interactions and the number of contacting residue pairs were obtained from the structures sampled by MD simulations. In the legend, the CoV2 and SARS indicate the sources of the RBD; the WT, K353H, mut_h1 are the abbreviations for different constructs of the human ACE2, corresponding to the wild type, the mut_h1, and the K353H.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">More interestingly, the mutation impacts were reflected in the <em>E<sub>MM</sub></em> and <em>NC</em> analysis: the <em>ACE2-mut_h1</em> is likely to reduce the binding to the ACE2 due to the tilting movement of CoV2-RBD, making it further from the ACE2 helix-1 (the blue triangle symbol at lower right, see <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7a</a> for the representative structure). In the other simulation trajectory for the CoV2-RBD/ACE2-mut_h1 complex (blue triangle at the left upper corner), the largest <em>NC</em> was observed among all simulations. For simulations of the complex with <em>ACE2-K353H</em> mutants (green diamonds), the number of contacts were both reduced compared to the wild type system. In one simulation, the contacts between the CoV2-RBD and the Helix-13 of the ACE2 were completely lost (see <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7c</a>), consistent with the less favorable interactions reflected on an increase of <em>E<sub>MM</sub></em>. For the SARS-RBD interaction with the <em>ACE2-mut_h1</em>, both simulations revealed fewer contacts compared to the wild type SARS-RBD/ACE2 complex (purple stars in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Although direct observation of dissociation is beyond the reach of all-atom MD simulations, the reduced contacts for mutant ACE2 indicates the weaker binding affinity compared to the wild type human ACE2.</p></div></div><div class="section" id="sec-11"><h2 class="">Discussions and Conclusion</h2><p id="p-36">The homology modeling of the CoV2-RBD/ACE2 complex yielded highly consistent models compared to the crystal structures. All-atom molecular dynamics simulations were carried out to study the dynamic interactions of CoV2-RBD with human ACE2, the results were compared to the SARS-RBD/ACE2 system. The human ACE2 mutants were also constructed to mimic the rat ACE2 to investigate the roles of critical residues, and possible binding modes in other mammals. It is observed that MD simulations improved the structure at the binding interface and strengthened the interactions between the subunits. The structure of the complex interface is highly stable for all simulations of CoV2-RBD/ACE2 complex in the wild type. The loop region between β6 and β7 can potentially form more contacts with the ACE2 as observed in one simulation trajectory. The simulations results also reveal that the interactions between CoV2-RBD and the ACE2 are mediated by water molecules at the interfaces, stressing the necessity of accounting for the explicit water molecules when quantifying the binding affinity.</p><p id="p-37">The interactions between the RBD and the ACE2 were quantified by physical interaction energies (molecular mechanics energy) and the number of contacting residues. The detailed comparison results suggest that the CoV2-RBD and the SARS-RBD bind to human ACE2 with comparable affinity. The comparison between the SARS-RBD/ACE2 and the CoV2-RBD/ACE2 complexes, with the former forms fewer contacts than the latter (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>), yet exhibiting stronger interactions. The decomposition of the <em>E<sub>MM</sub></em> to the van der Waals and the electrostatic interactions revealed that the major difference is attributed to the electrostatic interactions. Furthermore, we compared the major contacting residues and found that the SARS-RBD has two charged residues (R426 and D463) and the CoV2-RBD has only one charged residue (K417) at the complex interface. The polar and hydrophobic residues are comparable in the two RBDs. This is consistent with the statistics of hydrogen bonds at the complex interfaces.</p><p id="p-38">This study was started with a structure predicted using homology modeling method, which later found to be highly consistent with the crystal structure, demonstrating the potentiality of structure prediction and dynamics simulation in revealing molecular details before the availability of high resolution experimental information. Furthermore, the interactions between CoV2-RBD and the ACE2 mutants mimicking rat ACE2 protein were investigated. The results provide valuable information at the atomic level for the reduced binding affinity. The recent report on the SARS-CoV-2 infection to a dog raise the concerns about the COVID-19 transmissions to pet animals. The approach discussed in this study can be applied to analyze the CoV2-RBD interaction with a broader ACE2 variants, including the genetic polymorphism of the ACE2 in human and the ACE2 of other mammalian animals. Preliminary analysis of several ACE2 proteins from mammalian animals using homology modeling and active site prediction methods shows that the binding sites exhibit different features compared to the human ACE2. More thorough and deeper analysis can be carried out using the presented approach.</p><p id="p-39">The high quality homology model allowed us to start the simulation analysis before the high resolution experimental data became available. The protein structure prediction community are making efforts to provide high quality models using advanced modeling and prediction methods<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Based on predicted and experimentally determined structures, molecular dynamics simulations will add more insights to the molecular mechanism and allow better probing properties due to mutation or modification to proteins.</p></div><div class="section" id="sec-13"><h2 class="">Competing interests</h2><p id="p-41">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgement</h2><p id="p-40">The work is supported by Beijing Computational Science Research Center (CSRC) via a director discretionary grant. The authors acknowledge the Beijing Super Cloud Computing Center (BSCC) for providing HPC resources that have contributed to the research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Adding supplementary information.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">Reference</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.1" data-doi="10.1056/nejmoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med. NEJMoa2001017</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/nejmoa2001017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%2BNEJMoa2001017%26rft_id%253Dinfo%253Adoi%252F10.1056%252Fnejmoa2001017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/nejmoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.2" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.3" data-doi="10.1007/s11427-020-1637-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission</span>. <abbr class="cit-jnl-abbrev">Sci. CHINA Life Sci</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s11427-020-1637-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BCHINA%2BLife%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-020-1637-5%26rft_id%253Dinfo%253Apmid%252F32009228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11427-020-1637-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32009228&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.4" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.5" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.6" data-doi="10.1074/jbc.C300520200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">A 193-Amino Acid Fragment of the SARS Coronavirus S Protein Efficiently Binds Angiotensinconverting Enzyme 2</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">279</span>, <span class="cit-fpage">3197</span>–<span class="cit-lpage">3201</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C300520200%26rft_id%253Dinfo%253Apmid%252F14670965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvNS8zMTk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNC4wMDU1NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.7" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-article-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">309</span>, <span class="cit-fpage">1864</span>–<span class="cit-lpage">1868</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNC4wMDU1NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.24.005561v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuba</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohto-Nakanishi</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Penninger</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Trilogy of ACE2: A peptidase in the renin-angiotensin system, a SARS receptor, and a partner for amino acid transporters</span>. <abbr class="cit-jnl-abbrev">Pharmacology and Therapeutics</abbr> <span class="cit-vol">128</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">128</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacology%2Band%2BTherapeutics%26rft.volume%253D128%26rft.spage%253D119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.9" data-doi="10.1101/2020.02.07.939389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> 2020.02.07.939389 (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.07.939389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.07.939389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4wNy45MzkzODl2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjQuMDA1NTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.10" data-doi="10.1101/2020.02.19.956235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lan</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.19.956235</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.19.956235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4xOS45NTYyMzV2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjQuMDA1NTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.11" data-doi="10.1093/nar/gky427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SWISS-MODEL: Homology modelling of protein structures and complexes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">46</span>, <span class="cit-fpage">W296</span>–<span class="cit-lpage">W303</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.12" data-doi="10.1128/JVI.78.20.11429-11433.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Efficient Replication of Severe Acute Respiratory Syndrome Coronavirus in Mouse Cells Is Limited by Murine Angiotensin-Converting Enzyme 2</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">11429</span>–<span class="cit-lpage">11433</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D78%26rft.issue%253D20%26rft.spage%253D11429%26rft.epage%253D11433%26rft.atitle%253DEfficient%2BReplication%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2Bin%2BMouse%2BCells%2BIs%2BLimited%2Bby%2BMurine%2BAngiotensin-Converting%2BEnzyme%2B2%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.20.11429-11433.2004%26rft_id%253Dinfo%253Apmid%252F15452268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8yMC8xMTQyOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjQuMDA1NTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.13" data-doi="10.1128/jvi.00127-20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00127-20</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.00127-20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/jvi.00127-20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.14" data-doi="10.1021/acs.jctc.5b00935"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">405</span>–<span class="cit-lpage">413</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BTheory%2BComput%26rft.volume%253D12%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jctc.5b00935%26rft_id%253Dinfo%253Apmid%252F26631602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.jctc.5b00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26631602&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.15" data-doi="10.1021/ct300400x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Best</span>,  <span class="cit-name-given-names">R. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Optimization of the additive CHARMM all-atom protein force field targeting improved sampling of the backbone φ, ψ and side-chain χ1 and χ2 Dihedral Angles</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">3257</span>–<span class="cit-lpage">3273</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BTheory%2BComput%26rft.volume%253D8%26rft.spage%253D3257%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct300400x%26rft_id%253Dinfo%253Apmid%252F23341755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct300400x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23341755&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.16" data-doi="10.1063/1.447334"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nosé</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">A unified formulation of the constant temperature molecular dynamics methods</span>. <abbr class="cit-jnl-abbrev">J. Chem. Phys</abbr>. <span class="cit-vol">81</span>, <span class="cit-fpage">511</span>–<span class="cit-lpage">519</span> (<span class="cit-pub-date">1984</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D81%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.447334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1063/1.447334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TA66100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.17" data-doi="10.1103/PhysRevA.31.1695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoover</span>,  <span class="cit-name-given-names">W. G.</span></span> <span class="cit-article-title">Canonical dynamics: Equilibrium phase-space distributions</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. A</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">1695</span>–<span class="cit-lpage">1697</span> (<span class="cit-pub-date">1985</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPHYSICAL%2BREVIEW.%2BA%26rft.aulast%253DHoover%26rft.auinit1%253DW.%2BG.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D1695%26rft.epage%253D1697%26rft.atitle%253DCanonical%2Bdynamics%253A%2BEquilibrium%2Bphase-space%2Bdistributions.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevA.31.1695%26rft_id%253Dinfo%253Apmid%252F9895674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevA.31.1695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9895674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ACR3000056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.18" data-doi="10.1063/1.328693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parrinello</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rahman</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Polymorphic transitions in single crystals: A new molecular dynamics method</span>. <abbr class="cit-jnl-abbrev">J. Appl. Phys</abbr>. <span class="cit-vol">52</span>, <span class="cit-fpage">7182</span>–<span class="cit-lpage">7190</span> (<span class="cit-pub-date">1981</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BPhys%26rft.volume%253D52%26rft.spage%253D7182%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.328693%26rft_id%253Dinfo%253Apmid%252F9895674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1063/1.328693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9895674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MT07800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.19" data-doi="10.1016/j.softx.2015.06.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abraham</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Gromacs: High performance molecular simulations through multi-level parallelism from laptops to supercomputers</span>. <abbr class="cit-jnl-abbrev">SoftwareX</abbr> <span class="cit-vol">1-2</span>, <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftwareX%26rft.volume%253D12%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.softx.2015.06.001%26rft_id%253Dinfo%253Apmid%252F27819658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.softx.2015.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27819658&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.20" data-doi="10.1063/1.464397"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darden</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">York</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems</span>. <abbr class="cit-jnl-abbrev">J. Chem. Phys</abbr>. <span class="cit-vol">98</span>, <span class="cit-fpage">10089</span>–<span class="cit-lpage">10092</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D98%26rft.spage%253D10089%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.464397%26rft_id%253Dinfo%253Apmid%252F10.1063%252F1.46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1063/1.464397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10.1063/1.46&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LG10100091&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.21" data-doi="10.1002/(SICI)1096-987X(199709)18:12&lt;1463::AID-JCC4&gt;3.3.CO;2-L"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hess</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bekker</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berendsen</span>,  <span class="cit-name-given-names">H. J. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fraaije</span>,  <span class="cit-name-given-names">J. G. E. M.</span></span> <span class="cit-article-title">LINCS: A Linear Constraint Solver for molecular simulations</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">1463</span>–<span class="cit-lpage">1472</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BChem%26rft.volume%253D18%26rft.spage%253D1463%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.3.CO%253B2-L%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-987X(199709)18:12&lt;1463::AID-JCC4&gt;3.3.CO;2-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XT81100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.22" data-doi="10.1021/ci500020m"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumari</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lynn</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">G-mmpbsa-A GROMACS tool for high-throughput MM-PBSA calculations</span>. <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model</abbr>. <span class="cit-vol">54</span>, <span class="cit-fpage">1951</span>–<span class="cit-lpage">1962</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel%26rft.volume%253D54%26rft.spage%253D1951%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci500020m%26rft_id%253Dinfo%253Apmid%252F24850022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci500020m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24850022&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.23" data-doi="10.1016/0263-7855(96)00018-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Humphrey</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalke</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schulten</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">VMD: Visual molecular dynamics</span>. <abbr class="cit-jnl-abbrev">J. Mol. Graph</abbr>. <span class="cit-vol">14</span>, <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bgraphics%26rft.stitle%253DJ%2BMol%2BGraph%26rft.aulast%253DHumphrey%26rft.auinit1%253DW.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D33-8%252C%2B28%26rft.atitle%253DVMD%253A%2Bvisual%2Bmolecular%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0263-7855%252896%252900018-5%26rft_id%253Dinfo%253Apmid%252F8744570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0263-7855(96)00018-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8744570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH51500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.24" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>,  <span class="cit-name-given-names">E. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">UCSF Chimera - A visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem</abbr>. <span class="cit-vol">25</span>, <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.24.005561.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.25" data-doi="10.1038/sj.emboj.7600640"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>. <span class="cit-vol">24</span>, <span class="cit-fpage">1634</span>–<span class="cit-lpage">1643</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D1634%26rft.epage%253D1643%26rft.atitle%253DReceptor%2Band%2Bviral%2Bdeterminants%2Bof%2BSARS-coronavirus%2Badaptation%2Bto%2Bhuman%2BACE2%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600640%26rft_id%253Dinfo%253Apmid%252F15791205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyNC84LzE2MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI0LjAwNTU2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.26" data-doi="10.1074/jbc.M512529200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lavillette</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Significant redox insensitivity of the functions of the SARS-CoV spike glycoprotein: Comparison with HIV envelope</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">281</span>, <span class="cit-fpage">9200</span>–<span class="cit-lpage">9204</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M512529200%26rft_id%253Dinfo%253Apmid%252F16418166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODEvMTQvOTIwMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjQuMDA1NTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.27" data-doi="10.1128/JVI.00442-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Structural analysis of major species barriers between humans and palm civets for severe acute respiratory syndrome coronavirus infections</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">82</span>, <span class="cit-fpage">6984</span>–<span class="cit-lpage">6991</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00442-08%26rft_id%253Dinfo%253Apmid%252F18448527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNC82OTg0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNC4wMDU1NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.28" data-doi="10.1128/JVI.02615-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Receptor Recognition Mechanisms of Coronaviruses: a Decade of Structural Studies</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">1954</span>–<span class="cit-lpage">1964</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02615-14%26rft_id%253Dinfo%253Apmid%252F25428871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzQvMTk1NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjQuMDA1NTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">0</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.30" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">367</span>, <span class="cit-fpage">1260</span>–<span class="cit-lpage">1263</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNC4wMDU1NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.24.005561v2.31"><div class="cit-metadata"><cite><span class="cit-article-title">Coronavirus: Hong Kong confirms a second dog is infected</span>. (<span class="cit-pub-date">2020</span>). Available at: <a href="https://www.scmp.com/news/hong-kong/health-environment/article/3075993/coronavirus-hong-kong-confirms-second-dog">https://www.scmp.com/news/hong-kong/health-environment/article/3075993/coronavirus-hong-kong-confirms-second-dog</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.32"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CASP</span>. <abbr class="cit-jnl-abbrev">CASP and Covid-19</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.24.005561v2.33"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Institute for Protein Design</span>. <abbr class="cit-jnl-abbrev">Coronavirus (SARS-CoV-2/COVID-19/nCoV-2019)</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.24.005561v2.34"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">ZhangLab</span>. <abbr class="cit-jnl-abbrev">Structure models of all mature peptides in 2019-nCoV genome by C-I-TASSER</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.24.005561v2.35"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">AlphaFold Team</span>. <span class="cit-article-title">Computational predictions of protein structures associated with COVID-19</span>. <abbr class="cit-jnl-abbrev">DeepMind Website</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.24.005561v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/835553v2" title="FastMulRFS: Fast and accurate species tree estimation under generic gene duplication and loss models" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.26.010009v1" title="Large-scale Phenotypic Drug Screen Identifies Neuroprotectants in Zebrafish and Mouse Models of Retinitis Pigmentosa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.24.005561v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.24.005561v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.24.005561v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.24.005561v2" class="active" data-icon-position="" data-hide-link-title="0">Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1214872" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-44zjdzduL9_Pi9ZhG3CLapQBkHf0MlaX6kH-Lgu4R8I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1214872" id="node1214872--2" data-pisa="biorxiv;2020.03.24.005561v2" data-pisa-master="biorxiv;2020.03.24.005561" data-seqnum="1214872" data-apath="/biorxiv/early/2020/03/28/2020.03.24.005561.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cecylia S.</span> <span class="nlm-surname">Lupala</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xuanxuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jian</span> <span class="nlm-surname">Lei</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Hong</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jianxun</span> <span class="nlm-surname">Qi</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Haiguang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Xiao-dong</span> <span class="nlm-surname">Su</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.24.005561; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.24.005561 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;title=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;title=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;text=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;title=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;t=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;title=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;title=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1214872" id="node1214872--3" data-pisa="biorxiv;2020.03.24.005561v2" data-pisa-master="biorxiv;2020.03.24.005561" data-seqnum="1214872" data-apath="/biorxiv/early/2020/03/28/2020.03.24.005561.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Computational simulations reveal the binding dynamics between human ACE2 and the receptor binding domain of SARS-CoV-2 spike protein</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cecylia S.</span> <span class="nlm-surname">Lupala</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xuanxuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jian</span> <span class="nlm-surname">Lei</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Hong</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jianxun</span> <span class="nlm-surname">Qi</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Haiguang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Xiao-dong</span> <span class="nlm-surname">Su</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.24.005561; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.24.005561 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Lupala%2C+Cecylia+S.&amp;rft.au=Li%2C+Xuanxuan&amp;rft.au=Lei%2C+Jian&amp;rft.au=Chen%2C+Hong&amp;rft.au=Qi%2C+Jianxun&amp;rft.au=Liu%2C+Haiguang&amp;rft.au=Su%2C+Xiao-dong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1214872/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1214872/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1214872/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1214872/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1214872/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1214872/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1214872/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1214872/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1214872/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1214872/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1214872/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1214872/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;count=horizontal&amp;via=&amp;text=Computational%20simulations%20reveal%20the%20binding%20dynamics%20between%20human%20ACE2%20and%20the%20receptor%20binding%20domain%20of%20SARS-CoV-2%20spike%20protein&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.24.005561v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.24.005561v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">Biophysics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
