<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Braz J Med Biol Res</span><span class="iso-abbrev" title="iso-abbrev">Braz. J. Med. Biol. Res</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Brazilian Journal of Medical and Biological Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0100-879X</span><span class="issn-epub" tagx="issn" title="issn-epub">1414-431X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">AssociaÃ§Ã£o Brasileira de DivulgaÃ§Ã£o CientÃ­fica</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4075298">4075298</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24676472">24676472</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1590/1414-431X20133249">10.1590/1414-431X20133249</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Clinical Investigation</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Development and validation of a genotype 3 recombinantprotein-based immunoassay for hepatitis E virus serology in swine</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Poel</span><span class="given-names" tagx="given-names" title="given-names">W.H.M.</span></span><a href="#aff1">1</a><a href="#aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavio</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><a href="#aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Goot</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Es</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><a href="#aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><a href="#aff4">4</a></span><span class="citation_author_institution" id="aff1">[1], <span class="country" tagx="country" title="country">The Netherlands</span></span><span class="citation_author_institution" id="aff2">[2], <span class="country" tagx="country" title="country">France</span></span><span class="citation_author_institution" id="aff3">[3], <span class="country" tagx="country" title="country">Spain</span></span><span class="citation_author_institution" id="aff4">[4], <span class="country" tagx="country" title="country">The Netherlands</span></span><span class="citation_author_institution" id="aff5">[5], <span class="country" tagx="country" title="country">United Kingdom</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Correspondence: W.H.M. van der Poel, Central Veterinary Institute, Wageningen University and Research Centre, Edelhertweg 15, 8219 PH Lelystad, The Netherlands, +31-3-2023-8961. E-mail: <span class="email" tagx="email" title="email">wim.vanderpoel@wur.nl</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2014-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2014-4</span></span><span class="volume" tagx="volume" title="volume">47</span><span class="issue" tagx="issue" title="issue">4</span><span class="fpage" tagx="fpage" title="fpage">334</span><span class="lpage" tagx="lpage" title="lpage">339</span><span class="history" title="history"><span class="received" title="received">received: 2013-5-24</span><span class="accepted" title="accepted">accepted: 2013-11-25</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License, which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Hepatitis E virus (HEV) is classified within the family Hepeviridae, genus <i>Hepevirus</i>. HEV genotype 3 (Gt3) infections are endemic in pigs in Western Europe and in North and South America and cause zoonotic infections in humans. Several serological assays to detect HEV antibodies in pigs have been developed, at first mainly based on HEV genotype 1 (Gt1) antigens. To develop a sensitive HEV Gt3 ELISA, a recombinant baculovirus expression product of HEV Gt3 open reading frame-2 was produced and coated onto polystyrene ELISA plates. After incubation of porcine sera, bound HEV antibodies were detected with anti-porcine anti-IgG and anti-IgM conjugates. For primary estimation of sensitivity and specificity of the assay, sets of sera were used from pigs experimentally infected with HEV Gt3. For further validation of the assay and to set the cutoff value, a batch of 1100 pig sera was used. All pig sera were tested using the developed HEV Gt3 assay and two other serologic assays based on HEV Gt1 antigens. Since there is no gold standard available for HEV antibody testing, further validation and a definite setting of the cutoff of the developed HEV Gt3 assay were performed using a statistical approach based on Bayes' theorem. The developed and validated HEV antibody assay showed effective detection of HEV-specific antibodies. This assay can contribute to an improved detection of HEV antibodies and enable more reliable estimates of the prevalence of HEV Gt3 in swine in different regions.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Hepatitis E virus</div> <div class="kwd" title="kwd">
Zoonosis</div> <div class="kwd" title="kwd">
Swine</div> <div class="kwd" title="kwd">
ELISA</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Hepatitis E virus (HEV) is a small nonenveloped RNA virus of approximately 7.2 kb, and has been classified within the family Hepeviridae, genus <i>Hepevirus</i>. Four mammalian genotypes (Gt1-4) were identified in the 1990s, and avian HEV had been identified before that (<a href="#B01">1</a>). More recently, HEVs have also been identified in rats, rabbits, ferrets, bats, and trout (<a href="#B02">2</a>). Gt1 and Gt2 are found exclusively in humans and are responsible for large waterborne epidemics and sporadic cases in endemic regions (Asia, India, Africa, and Mexico) (<a href="#B01">1</a>). Gt1 and Gt2 can be divided further into five (a-e) and two (a and b) subtypes, respectively. In contrast, Gt3 and Gt4 are responsible for sporadic cases in humans and are widespread in animals. Especially for HEV Gt3, it is assumed that zoonotic transmission plays an important role in human infections (<a href="#B03">3</a>). Genotypes 3 and 4 seem more variable and can be divided into 10 (a-j) and 7 (a-g) subgenotypes, respectively (<a href="#B04">4</a>).</p> <p>Despite the considerable genetic variability observed between genotypes, to date only one serotype has been identified. HEV Gt3 infections have been reported to be endemic in pigs in Europe (<a href="#B05">5</a>) and in North and South America (<a href="#B06">6</a>). HEV infections in humans often run a subclinical course but can also cause serious acute hepatitis. The disease is characterized by a self-limiting jaundice of varying severity, which is hard to distinguish from hepatitis of other viral origin, and is often accompanied by nonspecific symptoms such as fever, headache, and pain in the upper abdomen. In pigs naturally infected with HEV Gt3, microscopic lesions of hepatitis are observed, but overt clinical symptoms in pigs have not been reported. Autochthonous HEV Gt3 and Gt4 infections in humans may be zoonotic and may originate from swine reservoirs. For this reason, it is suggested that the study of HEV dynamics in pig populations be done in Europe and the Americas.</p> <p>For swine serology, several HEV antibody assays have been developed, mainly based on HEV Gt1 antigens, in particular open reading frame (ORF)-2 proteins or peptides. HEV Gt3 antibodies have been shown to cross-react with Gt1 antigens (<a href="#B07">7</a>), but, given the high variability of strains, homologous assays may perform better. Since HEV Gt3 is the predominant genotype in pigs in Europe and North America, in this study an HEV antibody assay for pigs was developed based on the HEV Gt3 antigen. Due to the fact that there is no gold standard available, in-house developed or commercially available HEV immunoassays often have not been validated properly. For final validation and a definite choice for cutoff of the developed HEV Gt3 assay in this study, posterior Bayesian statistical inference was used.</p> <p>A total of 1100 serum samples from fattening pigs were tested using the newly developed HEV Gt3 assay and two other HEV immunoassays for comparison. Test results of the three assays were analyzed simultaneously with a latent class model [see Engel et al. (<a href="#B08">8</a>) and references therein]. This statistical model accounts for the lack of a gold standard, i.e., no assay or combination of assays was chosen as a reference.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Material and Methods</div> <div class="hevelisadevelopment" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA development</div> <p>A recombinant expression product of HEV (ORF-2, Gt3, Subtype 3a) was produced in a baculovirus expression system as previously described (<a href="#B09">9</a>,<a href="#B10">10</a>). A synthetic gene encoding 497 amino acids of swine Gt3 HEV ORF-2, Subtype 3a, was engineered using the amino acid sequence published in GenBank AY585859 (Geneart, Germany). The part of the HEV capsid encoded by this gene includes the motifs needed for self-assembly into empty virus-like particles when expressed in Sf9 insect cells. Codon usage and percentage G and C nucleotide content were optimized for expression in Sf9 <i>Spodoptera frugiperda</i> insect cells and a Kozak sequence was introduced upstream of the ATG. The recombinant baculovirus was obtained using the Bac-to-Bac<sup>Â¯</sup> Baculovirus Expression System (Invitrogen, France) according to the manufacturer's instructions. Sf9 insect cells adapted to a serum-free medium (Invitrogen) were grown at 27Â°C in SF900 serum-free medium supplemented with penicillin, streptomycin, and fungizone (Invitrogen). Viral stocks were prepared on Sf9 cells, after two plaque purifications also on Sf9 cells.</p> <p>Protein expression in the culture medium and cell lysate was analyzed by sodium dodecyl sulfate-polyacrylamide gel electrophoresis and Western blotting using negative and positive sera from swine. Sf9 cells were infected with the recombinant baculovirus at a multiplicity of infection of five. The culture medium was harvested 5 days after infection. Cells and debris were removed by centrifugation at 10,000 <i>g</i> for 30 min. The supernatant was then centrifuged at 11,000 <i>g</i> for 3 h in a Beckman SW28 rotor. The resulting pellet was resuspended in 4 mL PBS and held overnight at 4Â°C. The sample was then mixed with 0.44 g/mL CsCl and centrifuged at 15,000 <i>g</i> for 18 h at 4Â°C in a Beckman SW50.1 rotor. Fractions of 0.5 mL were collected from the bottom of each gradient and fraction densities were determined using an Abbe refractometer (Bioblock, France) and stored at a concentration of 0.3 Î¼g/mL.</p> <p>For the test principle, recombinant protein was coated onto polystyrene ELISA plates. Recombinant protein was dissolved 1:1500 in coating buffer (see ELISA procedures), and incubated with the test serum and HEV-specific swine antibodies were detected using anti-porcine anti-IgG and IgM conjugates. Antibodies present in the test serum bound to the antigen during the first incubation, and during the second incubation, the bound antibodies were detected using anti-swine monoclonal antibodies against IgM and against the L-chain conjugated to the enzyme horseradish peroxidase (HRP, see ELISA procedures). Bound conjugate was visualized by adding a substrate chromogen. Wells containing samples negative for HEV antibodies remained colorless. To optimize the ELISA, different concentrations of the purified recombinant protein were tested using a series of swine sera from an experimental infection. Swine sera were tested in different dilutions to establish the optimal dilution for test sera. A primary cutoff of the assay was established using sera from two cesarean-delivered colostrum-deprived, specific pathogen-free pigs. For primary estimation of sensitivity and specificity of the assay, sets of sera were used from pigs experimentally infected with HEV Gt3 (<a href="#B10">10</a>,<a href="#B11">11</a>).</p> <p>All experiments in this study involving the use of animals were approved by the Ethics Committee for Animal Experiments of Wageningen University, in accordance with legislation of The Netherlands and the European Union.</p> </div> <div class="hevelisacomparisonassays" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA comparison assays</div> <p>Two different HEV immunoassays were used for comparison, a commercially available immunoassay and an HEV immunoassay that was developed in-house. The commercially available HEV Ab-ELISA kit (Axiom, Germany) is a double-antigen sandwich ELISA based on a recombinant Burmese HEV Gt1 capsid protein derivative covering the carboxy-terminal amino acid residues 394-606. Due to its test principle, it can detect HEV-specific antibodies independently of the host species and immunoglobulin class. The manufacturer's instructions were strictly adhered to in the assay, including the recommended thresholds for definition of a positive serum. The in-house developed HEV immunoassay is a previously standardized in-house ELISA based on a purified 55-kDa truncated recombinant capsid protein of the Sar-55 strain of human HEV Gt1 (Gt1 assay). Based on its test principles, this immunoassay detects HEV-specific IgG in pig sera. This immunoassay was performed as described by Martin et al. (<a href="#B12">12</a>). The absorbance of each sample was read with an ELISA plate reader with a 450-nm filter. Test samples with absorbances equal to or greater than the cutoff value (0.300) were considered to be positive for anti-HEV IgG. In all immunoassays, sera of HEV naive and HEV infected swine were used as negative and positive controls in each plate.</p> </div> <div class="hevelisavalidation" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA validation</div> <p>For further validation of the assay and to definitely set its cutoff value, a batch of 1100 pig sera was used. The sera were obtained at slaughter from pigs from at least 550 pig farms (maximum two pigs per farm) in The Netherlands. Potential HEV infections in these pigs were of Gt3. To date, only HEV Gt3 infections have been found in pigs in The Netherlands. All pig sera were tested using the developed HEV Gt3 assay and the two comparison serologic assays based on HEV Gt1 antigens.</p> </div> <div class="statisticalevaluationofdiagnostictestaccuracy" title="sec"> <div class="title" tagx="title" title="title">
Statistical evaluation of diagnostic test accuracy</div> <p>The pig sera obtained at slaughter were used for a Bayesian evaluation of the accuracy of the three ELISA tests. In a Bayesian analysis two sources of information are combined: prior information about prevalence, test sensitivity, and test specificity, and new observations, which in this case were the ELISA test results. The prior information attaches a probability to each possible value of a parameter in the model, e.g., an unknown test sensitivity. Consequently, the prior information takes the form of a probability distribution for this sensitivity. This prior distribution is combined with the data into a posterior distribution, which is an up-to-date summary of all available information about the sensitivity. The median of this posterior distribution is presented as an estimate and the 95% range as a Bayesian confidence interval (credible interval) for the sensitivity.</p> <p>The statistical model distinguishes between the test results and the true antibody status of the sera. Because the tests are fallible, observed test results and unobserved true antibody status are not necessarily the same. The test results of the three ELISA tests were analyzed with a latent class model, where the true-antibody status (seropositive or seronegative) represents the latent classes.</p> <p>Data were collected in relatively small batches from different producers. In the statistical model, each batch has its own true prevalence, i.e., probability for a true seropositive, and these different prevalences are assumed to be sampled from a distribution of prevalence values. The probabilistic behavior of the tests, in relation to true antibody status, is modeled according to Engel et al. (<a href="#B08">8</a>). The model accounts for possible dependence between tests for true seropositives and true seronegatives.</p> <p>Prior distributions were based on literature and expert opinion. For assays based on a Gt3 antigen, prior emphasis was on a slightly higher specificity. For assays using more isotypes of antibodies in the conjugate, prior emphasis was on slightly higher sensitivity. The prior median and 95% ranges for sensitivity and specificity are shown in <a href="#t01">Table 1</a>. The prior range for median prevalence was chosen from 0.5 to 0.9. Further (technical) details about model and priors may be found in Supplementary Material.</p> <p> <div class="table-wrap_UNKNOWN" id="t01" orientation="portrait" position="float"> </div></p><div class="graphic" />  <p /> <p>Because batches of sera from the same producer tended to be small, often as small as two sera from the same producer, parameters for prevalence were hard to identify from the data. In order to obtain more reliable inference about the sensitivity and specificity of the tests, it was assumed that dependence between tests, conditional upon true-negative status, was negligible.</p> <p>For the CVI (Central Veterinary Institute) test, binary data were generated at three different thresholds (20,23,25) and three separate analyses were performed, one for each threshold.</p> </div> <div class="hevelisaprocedures" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA procedures</div> <p>For coating onto polystyrene ELISA plates, the optimal dilution of the recombinant baculoprotein stock (0.3 Âµg/mL) was 1:1500 using a coating buffer (0.05 M sodium carbonate buffer, pH 9.65). In the final procedure, 100 ÂµL diluted baculoprotein was dispensed to each well of a polystyrene ELISA plate (Costar 9018, Greiner Bio-one, Alphen a/d Rijn, The Netherlands). Plates were covered with adhesive foil (plate sealer) and incubated overnight at room temperature. Before usage, ELISA plate(s) were washed three times using tap water supplemented with 0.05% Tween 80 in an ELISA plate washer. A volume of 100 ÂµL StabilCoat<sup>Â¯</sup> (Diarect AG, Freiburg, Germany) (diluted 1:1 with PBS) was dispensed to all wells and incubated 30 min at room temperature. After removal of StabilCoat<sup>Â¯</sup>, ELISA plates were dried for 3 h at 37Â°C and stored at 4Â°C until use. In the final test procedure, sera were diluted 1:5 in serum dilution buffer: 10% bovine serum albumin+10% Elk+1% Tergitol (dissolved in PBS) added to low salt buffer (2.5 mL 10% Tween 80 in 500 mL PBS). One hundred microliters of diluted test serum was dispensed to the wells and, after sealing, plates were incubated at room temperature for 1 h. After this first incubation, plates were washed three times using tap water supplemented with 0.05% Tween 80 in an ELISA plate washer. Anti-swine IgM HRP-conjugated monoclonal antibody (CVI 28.4.1.) was diluted 1:2000 and the anti-swine L-chain HRP-conjugated monoclonal antibody (CVI 27.2.1.) was diluted 1:1000 in conjugate buffer (5% fetal calf serum) in high salt buffer (14.5 g NaCl in 500 mL PBS, 2.5 mL 10% Tween 80). For the second incubation, 100 ÂµL of the diluted conjugate solution was dispensed to each well. Plates were covered with a plate sealer and incubated for 1 h at room temperature. Subsequently, plates were washed three times with tap water supplemented with 0.05% Tween 80 using an ELISA plate washer, and 100 ÂµL ready-to-use 3,3â€²,5,5â€²-tetramethylbenzidine substrate was dispensed to each well and incubated for 10 min at room temperature. The reaction was stopped by adding 100 ÂµL 0.5 M sulfuric acid to all wells. The color intensity was measured at 450 nm within 15 min after color development had been stopped.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="hevelisadevelopment" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA development</div> <p>As described in Material and Methods, several protocols were tested to set up an immunoassay based on HEV Gt3 recombinant antigen. Final performance of the developed HEV Gt3 immunoassay was assessed using sera of pigs experimentally infected with HEV Gt3, in comparison with a commercially available HEV Ab-ELISA (Axiom). The results showed slight differences in the kinetics of anti-HEV antibody detection, but seroconversion of HEV-infected pigs was detected with both detection methods (<a href="#f01">Figure 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Hepatitis E virus genotype 3 (Gt3) antibody developments in experimentally HEV Gt3-infected pigs measured by HEV Gt3 ELISA CVI and HEV ELISA Axiom. Test procedures of the developed assay are described in Material and Methods.</div>   </div> </div> <div class="hevelisavalidation" title="sec"> <div class="title" tagx="title" title="title">
HEV ELISA validation</div> <p>The average absorbance values for the set of 1100 pig sera obtained at slaughter and measured using three different HEV antibody assays are reported in <a href="#t02">Table 2</a>.</p> <p> <div class="table-wrap_UNKNOWN" id="t02" orientation="portrait" position="float"> </div></p><div class="graphic" />  <p /> <p>The estimated sensitivity and specificity of each of the three tests are shown in <a href="#t03">Table 3</a>, together with a 95% confidence interval (95%CI, credible interval). These results correspond to a threshold value of 20 for the CVI Gt3 assay. For threshold values of 23 and 25, similar values for estimated sensitivity and specificity were found for the Axiom and Gt1 assays. For a threshold value of 20, the CVI Gt3 assay showed the highest sensitivity. For threshold values of 20, 23, and 25, the CVI Gt3 assay showed similar specificities. For the CVI Gt3 immunoassay, the best results were obtained for the threshold value of 20; therefore, this value was chosen to be used as the primary choice cutoff value for the test.</p> <p> <div class="table-wrap_UNKNOWN" id="t03" orientation="portrait" position="float"> </div></p><div class="graphic" />  <p /> <p>The higher specificity of the Gt1 assay (0.94, 95%CI=0.92-0.96) was apparently achieved at the cost of a modest sensitivity (0.58, 95%CI=0.55-0.62). CVI Gt3 and Axiom had a similar specificity (0.89, 95%CI=0.88-0.90), but the estimated sensitivity of Axiom (0.93, 95%CI=0.90-0.94) was higher than that of CVI Gt3 (0.84, 95%CI=0.81-0.86).</p> <p>The estimated median prevalence was 0.95 (95%CI=0.90-0.97). The estimate for standard deviation <i>Ïƒ</i> is 0.87 (95%CI=0.66-1.03), which implies considerable variation among batches in prevalence: the odds ratio of two batches can be as high as 11, e.g., odds ratio 0.9/(1-0.9) <i>vs</i> 0.45/(1-0.45) for prevalences of 0.90 and 0.45.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>To establish an HEV antibody assay for pigs, where HEV Gt3 is the predominant genotype in Europe and North America, an HEV antibody assay was developed based on the HEV Gt3 antigen. Since there is increasing evidence that human autochthonous hepatitis E cases are linked to swine reservoirs (<a href="#B05">5</a>,<a href="#B13">13</a>), HEV antibody testing in swine is indicated to estimate HEV prevalence. For this purpose, a validated HEV antibody assay for Europe, preferably HEV Gt3 based, is needed. Therefore, a recombinant expression product of HEV (ORF-2, Gt3, Subtype 3a) was chosen as antigen. For sensitivity testing of HEV antibody development in the acute phase of the infection, a conjugate including HEV-specific IgG and IgM was used. The effective performance of this format was tested using a set of serum samples obtained from experimentally infected pigs.</p> <p>Validation of HEV antibody assays is hampered by the fact that a gold standard is not available. To get around this problem, different approaches have been used including Western blot and Bayesian analyses (<a href="#B09">9</a>,<a href="#B14">14</a>). In this study, posterior Bayesian statistical inference was used for final validation and definitive choice of the cutoff value. Through a combined approach of testing, using swine sera obtained from experimental infections and from the field, and comparison of immunoassays using Bayesian evaluation of the accuracy of the assays, an adequate validation of the immunoassay could be established.</p> <p>In the statistical model, it was assumed that there is an interdependency between assays for true seropositives and independence for true seronegatives. Without the last assumption, i.e., allowing for dependence between assays for true seronegatives as well, quite unrealistic results were found for the specificity of all three assays. Quite likely, this is connected to identification problems that influenced the prevalence structure of batches in the data set. Such identification problems are not specific to the statistical approach used in this article, and are potential problems shared by all methods used for diagnostic test evaluation in the absence of a gold standard. For true positives, dependence is likely, since all three immunoassays are based on more or less the same principle. Moreover, based on previous observations, it has been postulated repeatedly that it is likely that there is just one single serotype of mammalian HEV (<a href="#B01">1</a>). Assumed independence for true seronegatives was considered reasonable, because in that case there will not be an immunogenic reaction related to the selected antigens in the assays.</p> <p>To better understand HEV transmission in autochtho-nous HEV infections in humans, HEV surveillance in pigs is needed. Using an accurately validated, sensitive HEV antibody assay based on the predominant antigen will result in more reliable surveillance data. The HEV antibody assay developed and validated in this study is suitable for detection of HEV-specific antibodies in swine reservoirs and can contribute to reliable estimates of HEV prevalence in swine in different regions.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p><a href="http://www.bjournal.com.br/supplementary_material/3249.pdf">Click here to view [pdf]</a>.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>Research was commissioned by the Dutch Ministry of Economic Affairs (Min EZ) and supported by MEDVETNET, Network of Excellence for the Prevention and Control of Foodborne Zoonoses, WP31 ZOOVIRNET, Food Producing Animals as a Potential Source of Emerging Viral Zoonoses.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-other" title="other"> <p>First published online March 21, 2014.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B01" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worm</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Poel</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandstatter</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis E: an overview</span><span class="source" tagx="source" title="source">Microbes Infect</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">657</span><span class="lpage" tagx="lpage" title="lpage">666</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1286-4579(02)01584-8">10.1016/S1286-4579(02)01584-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12048035">12048035</a></span></span></li> <li tag="ref"><a name="B02" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purdy</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic variability and the classification of hepatitis E virus</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">4161</span><span class="lpage" tagx="lpage" title="lpage">4169</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02762-12">10.1128/JVI.02762-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23388713">23388713</a></span></span></li> <li tag="ref"><a name="B03" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christou</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosmidou</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus in the Western world - a pork-related zoonosis</span><span class="source" tagx="source" title="source">Clin Microbiol Infect</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">600</span><span class="lpage" tagx="lpage" title="lpage">604</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/1469-0691.12214">10.1111/1469-0691.12214</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23594199">23594199</a></span></span></li> <li tag="ref"><a name="B04" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hagedorn</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of global hepatitis E virus sequences: genetic diversity, subtypes and zoonosis</span><span class="source" tagx="source" title="source">Rev Med Virol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">5</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/rmv.482">10.1002/rmv.482</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16175650">16175650</a></span></span></li> <li tag="ref"><a name="B05" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Poel</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verschoor</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Heide</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrera</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vivo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kooreman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus sequences in swine related to sequences in humans, The Netherlands</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">970</span><span class="lpage" tagx="lpage" title="lpage">976</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid0706.010608">10.3201/eid0706.010608</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11747723">11747723</a></span></span></li> <li tag="ref"><a name="B06" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos Santos</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Paula</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marchevsky</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinto</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis E virus in swine and effluent samples from slaughterhouses in Brazil</span><span class="source" tagx="source" title="source">Vet Microbiol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">149</span><span class="fpage" tagx="fpage" title="fpage">236</span><span class="lpage" tagx="lpage" title="lpage">241</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vetmic.2010.10.024">10.1016/j.vetmic.2010.10.024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21129863">21129863</a></span></span></li> <li tag="ref"><a name="B07" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerson</span><span class="given-names" tagx="given-names" title="given-names">SU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clemente-Casares</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moiduddin</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arankalle</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torian</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purcell</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Putative neutralization epitopes and broad cross-genotype neutralization of Hepatitis E virus confirmed by a quantitative cell-culture assay</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">697</span><span class="lpage" tagx="lpage" title="lpage">704</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.81545-0">10.1099/vir.0.81545-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16476993">16476993</a></span></span></li> <li tag="ref"><a name="B08" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swildens</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stegeman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buist</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Jong</span><span class="given-names" tagx="given-names" title="given-names">MCM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Estimation of sensitivity and specificity of three conditionally dependent diagnostic tests in the absence of a gold standard</span><span class="source" tagx="source" title="source">J Agric Biol Environ Stat</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">360</span><span class="lpage" tagx="lpage" title="lpage">380</span><span class="pub-id"><a href="https://dx.doi.org/10.1198/108571106X153534">10.1198/108571106X153534</a></span></span></li> <li tag="ref"><a name="B09" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwknegt</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frankena</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rutjes</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wellenberg</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Roda Husman</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der Poel</span><span class="given-names" tagx="given-names" title="given-names">WHM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Basic reproduction ratio for hepatitis E virus transmission in swine by contact-exposure in one-to-one trials</span><span class="source" tagx="source" title="source">Vet Res</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">40</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.1051/vetres:2008017">10.1051/vetres:2008017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18367077">18367077</a></span></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boutrouille</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fablet</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madec</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eloit</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavio</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="mixed-article-title" title="mixed-article-title">The use of Bayesian methods for evaluating the performance of a virus-like particles-based ELISA for serology of hepatitis E virus infection in swine</span><span class="source" tagx="source" title="source">J Virol Methods</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">163</span><span class="fpage" tagx="fpage" title="fpage">329</span><span class="lpage" tagx="lpage" title="lpage">335</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jviromet.2009.10.019">10.1016/j.jviromet.2009.10.019</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19879297">19879297</a></span></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouwknegt</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rutjes</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reusken</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockhofe-Zurwieden</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frankena</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The course of hepatitis E virus infection in pigs after contact-infection and intravenous inoculation</span><span class="source" tagx="source" title="source">BMC Vet Res</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1746-6148-5-7">10.1186/1746-6148-5-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19193209">19193209</a></span></span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segales</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">FF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guenette</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mateu</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Deus</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Association of hepatitis E virus (HEV) and postweaning multisystemic wasting syndrome (PMWS) with lesions of hepatitis in pigs</span><span class="source" tagx="source" title="source">Vet Microbiol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vetmic.2006.12.020">10.1016/j.vetmic.2006.12.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17270366">17270366</a></span></span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouquet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunazzi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eloit</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicand</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Close similarity between sequences of hepatitis E virus recovered from humans and swine, France, 2008-2009</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">2018</span><span class="lpage" tagx="lpage" title="lpage">2025</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1711.110616">10.3201/eid1711.110616</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22099089">22099089</a></span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jimenez de Oya</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galindo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girones</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duizer</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escribano</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saiz</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Serological immunoassay for detection of hepatitis E virus on the basis of genotype 3 open reading frame 2 recombinant proteins produced in <i>Trichoplusia ni</i> larvae</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">47</span><span class="fpage" tagx="fpage" title="fpage">3276</span><span class="lpage" tagx="lpage" title="lpage">3282</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.00750-09">10.1128/JCM.00750-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19656986">19656986</a></span></span></li> </ul> </div> </div>  </body></html>