<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/07/2020.04.05.026336.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.05.026336v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A protocol for adding knowledge to Wikidata, a case report" />
<meta name="DC.Identifier" content="10.1101/2020.04.05.026336" />
<meta name="DC.Date" content="2020-04-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a “commons”. Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability." />
<meta name="DC.Contributor" content="Andra Waagmeester" />
<meta name="DC.Contributor" content="Egon L. Willighagen" />
<meta name="DC.Contributor" content="Andrew I Su" />
<meta name="DC.Contributor" content="Martina Kutmon" />
<meta name="DC.Contributor" content="Jose Emilio Labra Gayo" />
<meta name="DC.Contributor" content="Daniel Fernández-Álvarez" />
<meta name="DC.Contributor" content="Peter J. Schaap" />
<meta name="DC.Contributor" content="Lisa M. Verhagen" />
<meta name="DC.Contributor" content="Jasper J. Koehorst" />
<meta name="article:published_time" content="2020-04-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A protocol for adding knowledge to Wikidata, a case report" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a &quot;commons&quot;. Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.05.026336v1" />
<meta name="citation_id" content="2020.04.05.026336v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.05.026336" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.05.026336" />
<meta name="citation_author" content="Andra Waagmeester" />
<meta name="citation_author_institution" content="Micelio" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9773-4008" />
<meta name="citation_author" content="Egon L. Willighagen" />
<meta name="citation_author_institution" content="Department of Bioinformatics - BiGCaT, NUTRIM, Maastricht University" />
<meta name="citation_author_email" content="egon.willighagen@maastrichtuniversity.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7542-0286" />
<meta name="citation_author" content="Andrew I Su" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9859-4104" />
<meta name="citation_author" content="Martina Kutmon" />
<meta name="citation_author_institution" content="Department of Bioinformatics - BiGCaT, NUTRIM, Maastricht University" />
<meta name="citation_author_institution" content="Maastricht Centre for Systems Biology - MaCSBio, Maastricht University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7699-8191" />
<meta name="citation_author" content="Jose Emilio Labra Gayo" />
<meta name="citation_author_institution" content="WESO Research group, University of Oviedo" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8907-5348" />
<meta name="citation_author" content="Daniel Fernández-Álvarez" />
<meta name="citation_author_institution" content="WESO Research group, University of Oviedo" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8666-7660" />
<meta name="citation_author" content="Peter J. Schaap" />
<meta name="citation_author_institution" content="Department of Agrotechnology and Food Sciences, Laboratory of Systems and Synthetic Biology, Wageningen University &amp; Research" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4346-6084" />
<meta name="citation_author" content="Lisa M. Verhagen" />
<meta name="citation_author_institution" content="Intravacc" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4130-580X" />
<meta name="citation_author" content="Jasper J. Koehorst" />
<meta name="citation_author_institution" content="Department of Agrotechnology and Food Sciences, Laboratory of Systems and Synthetic Biology, Wageningen University &amp; Research" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8172-8981" />
<meta name="citation_reference" content="Watkins J. Preventing a covid-19 pandemic. BMJ. 2020 Feb 28;m810." />
<meta name="citation_reference" content="outbreak.info [Internet]. Available from: https://github.com/SuLab/outbreak.info" />
<meta name="citation_reference" content="Virus Outbreak Data Network (VODAN) [Internet]. Available from:https://www.go-fair.org/implementation-networks/overview/vodan/" />
<meta name="citation_reference" content="CORD-19-on-FHIR -- Semantics for COVID-19 Discovery [Internet]. Available from: https://github.com/fhircat/CORD-19-on-FHIR" />
<meta name="citation_reference" content="COVID-19 Disease Map [Internet]. Available from: https://covid.pages.uni.lu/map_curation" />
<meta name="citation_reference" content="The Coronavirus and Open Science: Our reads and Open use cases [Internet]. 2020. Available from: https://sparceurope.org/coronaopensciencereadsandusecases/" />
<meta name="citation_reference" content="Sondervan J, Bosman J, Kramer B, Brinkman L, Imming M, Versteeg A. The COVID-19 pandemic stresses the societal importance of open science. ScienceGuide [Internet]. 2020 Apr 3; Available from: https://www.scienceguide.nl/2020/04/dire-times-of-covid-19-stress-the-societal-importance-of-open-science/" />
<meta name="citation_reference" content="Sharma M, Scarr S, Kelland K. Speed Science: The risks of swiftly spreading coronavirus research [Internet]. 2020 Feb. Available from: https://graphics.reuters.com/CHINA-HEALTH-RESEARCH/0100B5ES3MG/index.html" />
<meta name="citation_reference" content="Mahase E. Covid-19: six million doses of hydroxychloroquine donated to US despite lack of evidence. BMJ. 2020 Mar 23;m1166." />
<meta name="citation_reference" content="Waagmeester A, Stupp G, Burgstaller-Muehlbacher S, Good BM, Griffith M, Griffith OL, et al. Wikidata as a knowledge graph for the life sciences. eLife. 2020 Mar 17;9:e52614." />
<meta name="citation_reference" content="Vrandečić D, Krötzsch M. Wikidata. Commun ACM. 2014 Sep;57(10):78–85." />
<meta name="citation_reference" content="Burgstaller-Muehlbacher S, Waagmeester A, Mitraka E, Turner J, Putman T, Leong J, et al. Wikidata as a semantic framework for the Gene Wiki initiative. Database. 2016 Mar;2016:baw015.#x002B;." />
<meta name="citation_reference" content="Nielsen FÅ, Mietchen D, Willighagen E. Scholia And Scientometrics With Wikidata. 2017 Sep 8 [cited 2018 Jun 26]; Available from: https://zenodo.org/record/1036595" />
<meta name="citation_reference" content="citation_journal_title=The Semantic Web – ISWC 2014 [Internet]. Springer International Publishing;citation_author=F. Erxleben;citation_author=M. Günther;citation_author=M. Krötzsch;citation_author=J. Mendez;citation_author=D. Vrandečić;citation_title=Introducing Wikidata to the Linked Data Web;citation_pages=50-65;citation_volume=8796;citation_year=2014" />
<meta name="citation_reference" content="Cyganiak R, Lanthaler M, Wood D. RDF 1.1 Concepts and Abstract Syntax. W3C; 2014 Feb." />
<meta name="citation_reference" content="citation_title=The Semantic Web – ISWC 2018 [Internet]. Cham: Springer International Publishing;citation_author=S. Malyshev;citation_author=M. Krötzsch;citation_author=L. González;citation_author=J. Gonsior;citation_author=A. Bielefeldt;citation_title=Getting the Most Out of Wikidata: Semantic Technology Usage in Wikipedia’s Knowledge Graph;citation_pages=376-94;citation_volume=11137;citation_year=2018" />
<meta name="citation_reference" content="Federhen S. The NCBI Taxonomy database. Nucleic Acids Res. 2012 Jan 1;40(D1):D136–43." />
<meta name="citation_reference" content="Brown GR, Hem V, Katz KS, Ovetsky M, Wallin C, Ermolaeva O, et al. Gene: a gene-centered information resource at NCBI. Nucleic Acids Res. 2015 Jan 28;43(D1):D36–42." />
<meta name="citation_reference" content="Consortium TU. UniProt: the universal protein knowledgebase. Nucleic Acids Res. 2017 Jan;45(D1):D158–D169." />
<meta name="citation_reference" content="wwPDB consortium, Burley SK, Berman HM, Bhikadiya C, Bi C, Chen L, et al. Protein Data Bank: the single global archive for 3D macromolecular structure data. Nucleic Acids Res. 2019 Jan 8;47(D1):D520–8." />
<meta name="citation_reference" content="Slenter DN, Kutmon M, Hanspers K, Riutta A, Windsor J, Nunes N, et al. WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research. Nucleic Acids Res. 2018 Jan 4;46(D1):D661–D667." />
<meta name="citation_reference" content="citation_journal_title=The Semantic Web [Internet]. Cham: Springer International Publishing;citation_author=K. Thornton;citation_author=H. Solbrig;citation_author=GS. Stupp;citation_author=JE. Labra Gayo;citation_author=D. Mietchen;citation_author=E. Prud’hommeaux;citation_title=Using Shape Expressions (ShEx) to Share RDF Data Models and to Guide Curation with Rigorous Validation;citation_pages=606-20;citation_volume=11503;citation_year=2019" />
<meta name="citation_reference" content="Prud’hommeaux E, Labra Gayo JE, Solbrig H. Shape expressions: an RDF validation and transformation language. In: Proceedings of the 10th International Conference on Semantic Systems - SEM ‘14 [Internet]. Leipzig, Germany: ACM Press; 2014 [cited 2020 Apr 5]. p. 32–40. Available from: http://dl.acm.org/citation.cfm?doid=2660517.2660523" />
<meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang B, Song J, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med. 2020 Feb 20;382(8):727–33." />
<meta name="citation_reference" content="Berners-Lee T, Hendler J, Lassila O. The Semantic Web. Sci Am. 2001 May;284(5):34–43." />
<meta name="citation_reference" content="Seaborne A, Harris S. SPARQL 1.1 Query Language. W3C; 2013 Mar." />
<meta name="citation_reference" content="W3C OWL Working Group. OWL 2 Web Ontology Language Document Overview [Internet]. 2012 Dec. Available from: https://www.w3.org/TR/owl2-overview/" />
<meta name="citation_reference" content="Berners-Lee T. Linked Data [Internet]. 2006. Available from: https://www.w3.org/DesignIssues/LinkedData.html" />
<meta name="citation_reference" content="Samwald M, Jentzsch A, Bouton C, Kallesøe C, Willighagen E, Hajagos J, et al. Linked open drug data for pharmaceutical research and development. J Cheminformatics. 2011;3(1):19." />
<meta name="citation_reference" content="Hernández D, Hogan A, Krötzsch M. Reifying RDF: What Works Well With Wikidata? In: Proceedings of the 11th International Workshop on Scalable Semantic Web Knowledge Base Systems co-located with 14th International Semantic Web Conference (ISWC 2015) [Internet]. Bethlehem, PA, USA; 2015. (CEUR Workshop Proceedings). Available from: http://ceur-ws.org/Vol-1457/SSWS2015_paper3.pdf" />
<meta name="citation_reference" content="Jose Emilio Labra Gayo. RDFSHape: Online demo implementation of ShEx and SHACL [Internet]. Zenodo; 2018 [cited 2020 Apr 4]. Available from: https://zenodo.org/record/1412128" />
<meta name="citation_reference" content="Pablo Menéndez Suárez, Jose Emilio Labra Labra Gayo. YaShE [Internet]. Zenodo; 2020 [cited 2020 Apr 4]. Available from: https://zenodo.org/record/3739108" />
<meta name="citation_reference" content="Sayers E. E-utilities Quick Start [Internet]. NCBI; 2018. Available from: https://www.ncbi.nlm.nih.gov/books/NBK25500/" />
<meta name="citation_reference" content="Wu C, MacLeod I, Su AI. BioGPS and MyGene.info: organizing online, gene-centric information. Nucleic Acids Res. 2013 Jan 1;41(D1):D561–5." />
<meta name="citation_reference" content="van Iersel MP, Pico AR, Kelder T, Gao J, Ho I, Hanspers K, et al. The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services. BMC Bioinformatics. 2010 Jan;11(1):5+." />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput Biol;citation_author=A. Waagmeester;citation_author=M. Kutmon;citation_author=A. Riutta;citation_author=R. Miller;citation_author=EL. Willighagen;citation_author=CT. Evelo;citation_title=Using the Semantic Web for Rapid Integration of WikiPathways with Other Biological Online Data Resources;citation_pages=e1004989;citation_volume=12;citation_year=2016;citation_issue=6" />
<meta name="twitter:title" content="A protocol for adding knowledge to Wikidata, a case report" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a “commons”. Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability." />
<meta name="og-title" property="og:title" content="A protocol for adding knowledge to Wikidata, a case report" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a “commons”. Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.05.026336v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.05.026336v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.05.026336v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1232125" />
    <title>A protocol for adding knowledge to Wikidata, a case report | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"U80Xxm7mNfCKVh1qza05cbALcfIwwdGIzziXbc7NRwI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1232125","apath":"\/biorxiv\/early\/2020\/04\/07\/2020.04.05.026336.atom","pisa":"biorxiv;2020.04.05.026336v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.05.026336v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.05.026336v1","title":"A protocol for adding knowledge to Wikidata, a case report","identifier":"node\/1232125"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n6y66sDu2opecOcW5DZs4xLp1hx4dFod1FLvKX9WEFrLBnXgLn\\\\\\\/V27sh6XlrEyycKDHSFjEKZSz73xzI1Xp5ZSZ+65+wjh97Zp2bQYfOc91XuPYdYVGLtyN\\\\\\\/ywJq+4NBeReesyM5xxQ5731vql4bPDOCnU9n+\\\\\\\/yUvsWEVTtaA1Jep2yuikyvlQX7URbLsCX9jZ3rHDD+4malBpor\\\\\\\/\\\\\\\/u6eSQ8nZp89YpYLth6ONFwkBu8fFK5ypjCAreMn72c8FKgU22CmIId2qTHXuVzShQlyGePgW06A3caXIPKDTgeyk+U9mGp3bMMBwFKNLF6nhGE3rP0jVC\\\\\\\/JDlNGJAkpGJgEhR8REDYUsN9lmbxMDWELlq\\\\\\\/y0dtei89pAxVBLwP1eWHI2hSXE7RpmT7f\\\\\\\/cFTOndV1w0CcukquZFyn10Bnr1hWnVbRgM2KF5zzJ\\\\\\\/73aiZFEHPMeNxb9FhXQ5kRuOMNv27VI9icmmcNp9UUNc0UPinziiHD3xFhKJI5TsT5JpXD04e\\\\\\\/TAaYC6dYDqtWZzNd\\\\\\\/rjLSj9nSCqfMteO137QCZLK+gbnXJJdp4VIjYOCyiSZi0g7uTITeCY1ACyMD+DGexJkd5PI26tgK0v+huV0mGHVTL1bqXiOxytgoewsFkPG8aIqet0P2D\\\\\\\/KpizZocqpFthDXU11hXu0+t6eOxstx0KKW64\\\\\\\/4rP8W176wrpO2bq4Pq1KTuvwOoUswSiKMhGyoc60KA4jqknVRRDQSMZPOWWRh\\\\\\\/vrU2rbnHq6HswWHPWKhz1M6DEJxV9FKFB5OcxHtJumFf2mGugQTguFPZTwlBp5hFgQxo3wORsl2J+jE+a1WfXDtBumkxtDyXssjo7VRsU8rlQmpJlOz8UntO11di0X1+L3UZWzTmvIjIiAdYhr6GTmBLDHFC7Uq32pGyvEy6oreYQ\\\\\\\/OYQWP0IGfi54Afn8FmI58q6kV14DD0gt6Xc9QMvJh+DEyDHSeAFkO60X+OUCpGKvPmHgkE5Bnj+DR67ft2E0oQyzBa47AFonX70ZmJ\\\\\\\/W\\\\\\\/DLQZ5nik6faQZI6nvxI9EVJOQqYL87aro\\\\\\\/omhLPFx3raqRNuHMVuW6HSWHK6vm0ZqFTP4TRz4ErQNyvkYzW1nrCQcuSc3cti3PPPDv7IslpTWre34enRkLkHz1MeQ4Up\\\\\\\/cVDx0LtirDycTOFyiavZH227GXtUW9xv8SoIhlRPxtgi8btFb9DRyPEy5YlxAHJ1ysXQR\\\\\\\/PXUZCsuPgIp\\\\\\\/pZhuYDcy11tsd9EkxfpGL22u0cn3NXOZ5cd6ukXf5hawGUQr3aHktQyFioipTDCi76X\\\\\\\/ruGPCDymhmhU+Gaa+AbQETCEIlJverhhgakXBC\\\u0022,\\\u0022iv\\\u0022:\\\u0022D5iU9DtUOxSNlfZa3HDTIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002239198f0d04f86271fb2cc5fdd1acb2e8df67b7fcf2527af63bb48100608a7431\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.05.026336v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1232125 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.05.026336v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XR1LgKlLuMfrL_-rLYNHUHQLh8T8qMVynYmO_cQxiVI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232125" id="node1232125" data-pisa="biorxiv;2020.04.05.026336v1" data-pisa-master="biorxiv;2020.04.05.026336" data-apath="/biorxiv/early/2020/04/07/2020.04.05.026336.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A protocol for adding knowledge to Wikidata, a case report</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9773-4008" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andra</span> <span class="nlm-surname">Waagmeester</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7542-0286" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Egon L.</span> <span class="nlm-surname">Willighagen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9859-4104" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew I</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7699-8191" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Martina</span> <span class="nlm-surname">Kutmon</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8907-5348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jose Emilio</span> <span class="nlm-surname">Labra Gayo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-8666-7660" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Fernández-Álvarez</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4346-6084" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Peter J.</span> <span class="nlm-surname">Schaap</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-4130-580X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Lisa M.</span> <span class="nlm-surname">Verhagen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-8172-8981" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jasper J.</span> <span class="nlm-surname">Koehorst</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.05.026336 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1232125" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andra Waagmeester </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Micelio</span>, Antwerpen, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andra%2BWaagmeester%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waagmeester%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndra%2BWaagmeester%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9773-4008" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andra Waagmeester</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Egon L. Willighagen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Bioinformatics - BiGCaT, NUTRIM, Maastricht University</span>, <span class='nlm-country'>The Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Egon%2BL.%2BWillighagen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willighagen%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEgon%2BL.%2BWillighagen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7542-0286" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Egon L. Willighagen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:egon.willighagen@maastrichtuniversity.nl" class="" data-icon-position="" data-hide-link-title="0">egon.willighagen@maastrichtuniversity.nl</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew I Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BI%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20AI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BI%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9859-4104" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew I Su</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martina Kutmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Bioinformatics - BiGCaT, NUTRIM, Maastricht University</span>, <span class='nlm-country'>The Netherlands</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Maastricht Centre for Systems Biology - MaCSBio, Maastricht University</span>, <span class='nlm-country'>The Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BKutmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kutmon%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartina%2BKutmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7699-8191" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martina Kutmon</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jose Emilio Labra Gayo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>WESO Research group, University of Oviedo</span>, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BEmilio%2BLabra%2BGayo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Labra%20Gayo%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BEmilio%2BLabra%2BGayo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8907-5348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Emilio Labra Gayo</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Daniel Fernández-Álvarez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>WESO Research group, University of Oviedo</span>, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BFern%C3%A1ndez-%C3%81lvarez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fern%C3%A1ndez-%C3%81lvarez%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BFern%25C3%25A1ndez-%25C3%2581lvarez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8666-7660" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Fernández-Álvarez</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peter J. Schaap </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Agrotechnology and Food Sciences, Laboratory of Systems and Synthetic Biology, Wageningen University &#38; Research</span>, Wageningen, the <span class='nlm-country'>Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BSchaap%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaap%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BJ.%2BSchaap%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4346-6084" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter J. Schaap</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lisa M. Verhagen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Intravacc</span>, PO Box 450, 3720 AL, Bilthoven, <span class='nlm-country'>The Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BVerhagen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verhagen%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BVerhagen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4130-580X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Verhagen</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jasper J. Koehorst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Agrotechnology and Food Sciences, Laboratory of Systems and Synthetic Biology, Wageningen University &#38; Research</span>, Wageningen, the <span class='nlm-country'>Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jasper%2BJ.%2BKoehorst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Koehorst%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJasper%2BJ.%2BKoehorst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8172-8981" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jasper J. Koehorst</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.05.026336v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1232125" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1232125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026336v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1232125" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1232125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026336v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1232125" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1232125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026336v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1232125" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1232125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.05.026336v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1232125" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1232125/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Pandemics, even more than other scientific questions, require swift integration of knowledge and identifiers. In a setting where there is a large number of loosely connected projects and initiatives, we need a common ground, also known as a “commons”. Wikidata, a public knowledge graph aligned with Wikipedia, is such a commons, but Wikidata may not always have the right schema for the urgent questions. In this paper, we address this problem by showing how a data schema required for the integration can be modelled with entity schemas represented by shape expressions. As a telling example, we describe the process of aligning resources on the genomics of the SARS-CoV-2 virus and related viruses as well as how shape expressions can be defined for Wikidata helping others studying the SARS-CoV-2 pandemic. How this model can be used to make data between various resources interoperable, is demonstrated by integrating data from NCBI Taxonomy, NCBI Genes, UniProt, and WikiPathways. Based on that model, a set of automated applications or bots were written for regular updates of these sources in Wikidata and added to a platform for automatically running these updates. Although this workflow is developed and applied in the context of the SARS-CoV-2 pandemic, it was also applied to other human coronaviruses (MERS, SARS, SARS-CoV-2, Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, Human coronavirus OC4) to demonstrate its broader applicability.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">The COVID-19 pandemic, caused by the SARS-CoV-2 virus, is leading to a burst of swiftly released scientific publications on the matter (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). In response to the pandemic, many research groups have started projects to understand the SARS-CoV-2 virus life cycle and to find solutions. Examples of the numerous projects include outbreak.info (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), VODAN around FAIR data (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>), CORD-19-on-FHIR and #covidpathways (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). Many research papers and preprints get published every week and many call for more Open Science (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). The Dutch universities went a step further and want to make any previously published research openly available, in whatever way related to COVID-19 research (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>).</p><p id="p-4">However, this swift release of research findings comes with an increased number of incorrect interpretations (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>) which can be problematic when new research articles are picked up by main-stream media (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Rapid evaluation of these new research findings and integration with existing resources requires a frictionless access to the underlying research data upon which the findings are based. This requires interoperable data formats and sophisticated integration for these resources. A first step in this integration is knowing if you talk about the same gene or the same proteins. Using an interoperability layer in common, we can link different resources more easily.</p><p id="p-5">The Gene Wiki project has been tearing down the different research silos on genetics, biological processes, related diseases and associated drugs (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). In contrast to legacy databases, where data models follow a relational data schema of connected tables, Wikidata (<a href="https://wikidata.org/">https://wikidata.org/</a>) uses statements to store facts (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). This model of statements aligns well with the RDF triple model of the semantic web and the content of Wikidata is also serialized as Resource Description Framework (RDF) triples (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>), acting as stepping stone for data resources to the semantic web. Through its SPARQL endpoint (<a href="https://query.wikidata.org">https://query.wikidata.org</a>), knowledge captured in Wikidata can be integrated with other nodes in the semantic web, using either mappings between these resources or through federated SPARQL queries (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of a Wikidata item, containing a set of statements which are key-value pairs, with qualifiers and references. Here the item for the ACE2 protein is given containing a statement about its molecular function. This molecular function (peptidyl-dipeptidase activity) contains a reference stating when and where this information was obtained." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Structure of a Wikidata item, containing a set of statements which are key-value pairs, with qualifiers and references. Here the item for the ACE2 protein is given containing a statement about its molecular function. This molecular function (peptidyl-dipeptidase activity) contains a reference stating when and where this information was obtained.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F1.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F1.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 :</span> <p id="p-6" class="first-child">Structure of a Wikidata item, containing a set of statements which are key-value pairs, with qualifiers and references. Here the item for the ACE2 protein is given containing a statement about its molecular function. This molecular function (peptidyl-dipeptidase activity) contains a reference stating when and where this information was obtained.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">The Gene Wiki project aligns novel primary data sources with Wikidata in a cycle of consecutive steps where the data schema of the primary source is aligned with the available Wikidata properties. Once the schema is in place, bots are developed to add and regularly update Wikidata with knowledge from the primary resource under scrutiny.</p><p id="p-8">Automated editing of Wikidata simplifies a lot of things, however, the quality control of that process must be monitored carefully. This requires a clear data schema that allows the various resources to be linked with additional provenance. This schema describes the key concepts required for the integrations of the resources we are interested in: NCBI Taxonomy (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), NCBI Gene (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>), UniProt (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>), the Protein Data Bank (PDB) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), WikiPathways (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>), and PubMed. Therefore, the key elements for which we need a model include viruses, virus strains, virus genes, and virus proteins. The first two provide the link to taxonomies, the models for genes and proteins link to UniProt, PDB, and WikiPathways. These key concepts are also required to annotate research output such as journal articles and datasets related to these topics. Wikidata calls such keywords ‘main subjects’. The introduction of this model and the actual SARS-CoV-2 genes and proteins in Wikidata enables the integration of these resources.</p><p id="p-9">This paper is a case report of a workflow/protocol for data integration and publication. The first step in this approach is to develop the data schema. Within Wikidata, shape expressions (ShEx) are used as the structural schema language to describe and capture schema’s of concepts (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). It shapes the RDF structure by which Wikidata content is made available. These shapes have the advantage that they are easily exchanged and describe linked data models as a knowledge graph. Since the shapes describe the model, they enable discussion, revealing inconsistencies between resources and allow for consistency checks of the content added by automated procedures.</p><p id="p-10">After introducing the model, the process of adding knowledge to Wikidata is described. In this process, the seven human coronaviruses (HCoVs), MERS, SARS, SARS-CoV-2 (causing COVID-19), Human Coronavirus NL63, Human coronavirus 229E, Human coronavirus HKU1, and Human coronavirus OC4 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>), were added to Wikidata. This protocol is finalised by describing how the resulting data schema and data can be applied to support other projects, particularly <span class="underline">#covidpathways</span>.</p><p id="p-11">The semantic web was proposed as a vision of the Web, in which information is given well-defined meaning and better enabling computers and people to work in cooperation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). In order to achieve that goal, several technologies have appeared, like RDF for describing resources (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>), SPARQL to query RDF data (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>) and the Web Ontology Language (OWL) to represent ontologies (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>).</p><p id="p-12">Linked data was later proposed as a set of best practices to share and reuse data on the web (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). The linked data principles can be summarized in four rules that promote the use of uniform resource identifiers (URIs) to name things, which can be looked up to retrieve useful information for humans and for machines using RDF, as well as having links to related resources. These principles have been adopted by several projects, enabling a web of reusable data, known as the linked data cloud (<a href="https://lod-cloud.net/">https://lod-cloud.net/</a>), which has also been applied to life science (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>).</p><p id="p-13">One prominent project is Wikidata, which has become one of the largest collections of open data on the web (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). Wikidata follows the linked data principles offering both HTML and RDF views of every item with their corresponding links to related items, and a SPARQL endpoint called the Wikidata Query Service.</p><p id="p-14">Wikidata’s RDF model offers a reification mechanism which enables to represent information about statements like qualifiers and references (see also <a href="https://www.wikidata.org/wiki/Help:Statements">https://www.wikidata.org/wiki/Help:Statements</a>). For each statement in Wikidata, there is a direct property in the <kbd>wdt</kbd> namespace which indicates the direct value. In addition, the wikidata data model adds other statements for reification purposes that allow to enrich the declarations with references and qualifiers (for a topical treatise, see Ref. (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>)). As an example, item <span class="underline">Q14875321</span>, which represents ACE2 (protein-coding gene in the Homo sapiens species) has a statement specifying that it has a chromosome (<span class="underline">P1057</span>) with value chromosome X (<span class="underline">Q29867336</span>). In RDF Turtle, this can be declared as:</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F2.medium.gif" width="440" height="13"/><noscript><img class="highwire-fragment fragment-image" alt="Figure2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F2.medium.gif" width="440" height="13"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-15">That statement can be reified to add qualifiers and references. For example, a qualifier can state that the genomic assembly (<span class="underline">P659</span>) is GRCh38 (<span class="underline">Q20966585</span>) with a reference declaring that it was stated (<span class="underline">P248</span>) in ensembl Release 99 (<span class="underline">Q83867711</span>). In Turtle, those declarations are represented as (see also <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 2</a>):</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F3.medium.gif" width="440" height="135"/><noscript><img class="highwire-fragment fragment-image" alt="Figure3" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F3.medium.gif" width="440" height="135"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example of RDF data model representing ACE2, created with RDFShape (31)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example of RDF data model representing ACE2, created with RDFShape (31).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F4.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F4.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 :</span> <p id="p-16" class="first-child">Example of RDF data model representing ACE2, created with RDFShape (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Specifying data models with ShEx</h3><p id="p-17">Although the RDF data model is very flexible, specifying an agreed structure for the data allows domain experts to identify the properties and structure of their data facilitating the integration between heterogeneous data sources. Shape expressions were used to provide the right level of abstraction. YaShE, a ShEx editor implemented in JavaScript, was applied to author these shapes (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). This application provides the means to associate labels in the natural language of Wikidata to the corresponding identifiers. The initial entity schema was defined with YaShE as a proof of concept for virus genes and proteins. In parallel, statements already available in Wikidata were used to automatically generate an initial shape for virus strains with sheXer. The statements for virus strains were retrieved with SPARQL from the Wikidata Query Service (WDQS). The generated shape was then further improved through manual curation. The syntax of the shape expressions were continuously validated through YaShE and the Wikidata Entity Schema namespace was used to share and collaboratively update the schema with new properties. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 3</a> gives a visual outline of these steps.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flow diagram for entity schema development and the executable workflow for the virus gene protein bot. A) The workflow of creating the shape expressions. B) The computational workflow of the how information was used from various public resources to populate Wikidata." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flow diagram for entity schema development and the executable workflow for the virus gene protein bot. A) The workflow of creating the shape expressions. B) The computational workflow of the how information was used from various public resources to populate Wikidata.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F5.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F5.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 :</span> <p id="p-18" class="first-child">Flow diagram for entity schema development and the executable workflow for the virus gene protein bot. A) The workflow of creating the shape expressions. B) The computational workflow of the how information was used from various public resources to populate Wikidata.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Populating Wikidata with Human Coronavirus Data</h3><p id="p-19">The second step in our workflow is to add entries for all virus strains, genes and their gene products to Wikidata. This information is spread over different resources. Here, annotations were obtained from NCBI EUtils (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>), Mygene.info (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>), and UniProt, as outlined below. Input to the workflow is the NCBI Taxonomy identifier of a virus under scrutiny. (e.g. <span class="underline">2697049</span> for SARS-CoV-2). The taxon annotations are extracted from NCBI EUtils. The gene and gene product annotations are extracted from mygene.info and the protein annotations are extracted from UniProt using the SPARQL endpoint at (<a href="https://sparql.uniprot.org/">https://sparql.uniprot.org/</a>).</p><p id="p-20">Genomic information from seven human coronaviruses (HCoVs) was obtained from literature including the NCBI Taxonomy identifiers. For six virus strains a reference genome was available and was used to populate Wikidata. For SARS-CoV-1, the NCBI Taxonomy identifier referred to various strains, though no reference strain was available. To overcome this issue, the species taxon for SARS-related coronaviruses (SARSr-CoV) was used instead, following the practices of NCBI Genes and UniProt.</p><div id="sec-5" class="subsection"><h4>NCBI Eutils</h4><p id="p-21">The Entrez Programming Utilities (EUtils) is the application programming interface (API) to the Entrez query and database system at the National Center for Biotechnology Information (NCBI). From this set of services the scientific name of the virus under scrutiny was extracted (e.g. “Severe acute respiratory syndrome coronavirus 2”).</p></div><div id="sec-6" class="subsection"><h4>Mygene.info</h4><p id="p-22">Mygene.info is a web service which provides a REST API that can be used to obtain up-to-data gene annotations. The first step in the process is to get a list of applicable genes for a given virus by providing the NCBI taxon id. The following step is to obtain gene annotations for the individual genes from mygene.info through <a href="http://mygene.info/v3/gene/43740571">http://mygene.info/v3/gene/43740571</a>. This results in the name and a set of applicable identifiers (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 4</a>)</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="JavaScript Object notation output of the mygene,info output for gene with NCBI gene identifier 43740571." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;JavaScript Object notation output of the mygene,info output for gene with NCBI gene identifier 43740571.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F6.medium.gif" width="294" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F6.medium.gif" width="294" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 :</span> <p id="p-23" class="first-child">JavaScript Object notation output of the mygene,info output for gene with NCBI gene identifier 43740571.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>UniProt</h4><p id="p-24">The annotations retrieved from mygene.info also contain protein identifiers such as UniProt, RefSeq and PDB, however their respective names are lacking. To obtain names and mappings to other protein identifiers, RefSeq and UniProt were consulted. Refseq annotations were acquired using the earlier mentioned NCBI EUtils. UniProt Identifiers are acquired using the SPARQL endpoint of UniProt, which is a rich resource for protein annotations provided by the Swiss Bioinformatics Institute. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 5</a> shows the SPARQL query that was applied to acquire the protein annotations.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The UniProt SPARQL query used to obtain additional protein annotations, descriptions and external resources." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The UniProt SPARQL query used to obtain additional protein annotations, descriptions and external resources.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F7.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F7.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 :</span> <p id="p-25" class="first-child">The UniProt SPARQL query used to obtain additional protein annotations, descriptions and external resources.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Wikidataintegrator</h4><p id="p-26">Wikidata integrator is a Python library (<a href="https://github.com/SuLab/WikidataIntegrator">https://github.com/SuLab/WikidataIntegrator</a>) that wraps around the Wikidata API (<a href="https://www.wikidata.org/w/api.php">https://www.wikidata.org/w/api.php</a>) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>). From external resources such as the NCBI, gene and taxonomy statements have been compiled with provenance and assigned to the associated Wikidata items. When an item did not exist (or recognized), it was created. The module compiled a list of statements by parsing the primary sources under scrutiny and extracted what statements already existed on Wikidata. A JavaScript Object Notation (JSON) string was created that resembled the JSON data model used by the Wikidata API. This JSON string was then submitted to the Wikidata API for ingestion.</p></div></div><div id="sec-9" class="subsection"><h3>Data Integration Use Cases / Validation</h3><div id="sec-10" class="subsection"><h4>WikiPathways and BridgeDb</h4><p id="p-27">WikiPathways is a biological pathway database and can visualize the details of interactions between proteins, metabolites, etc. It depends on BridgeDb to map identifiers of external data and knowledge to the identifiers used for the genes, proteins, and metabolites in the pathways (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Furthermore, mappings to Wikidata are required to establish the link of biological entities in pathways and journal articles that have those entities as main topics. Therefore, the virus genes and proteins are required to exist in Wikidata, enabling the interoperability between WikiPathways and Wikidata. Additionally, new virus mapping databases for BridgeDb are created by extracting the new virus gene and protein data, including links between Wikidata, NCBI Gene, RefSeq and UniProt identifiers using a SPARQL query (<a href="https://github.com/bridgedb/Wikidata2Bridgedb">https://github.com/bridgedb/Wikidata2Bridgedb</a>). The mapping databases will be updated regularly and will allow pathway curators to annotate virus genes and proteins in their pathways and provide linkouts on the WikiPathways website.</p><p id="p-28">The COVID-19 related pathways from <span class="underline">WikiPathways COVID-19 Portal</span> are added to Wikidata using the approach previously described (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>). For this, a dedicated repository has been set up to hold the GPML files, the internal WikiPathways file format, The GPML is converted into RDF files with the WikiPathways RDF generator (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>), while the files with author information are manually edited. For getting the most recent GPML files, a custom Bash script was developed (getPathways.sh). The conversion of the GPML to RDF uses the previously published tools for WikiPathways RDF (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>). Here, we adapted the code with unit tests to have one for each GPML file, using some simple template test. These are available in the SARS-CoV-2-WikiPathways branch of <span class="underline">Wikidata2Bridgedb</span>. Based on this earlier generated pathway RDF and using the Wikidataintegrator library, the WikiPathways bot was used to populate Wikidata with additional statements and items. The pathway bot was extended with the capability to link virus proteins to the corresponding pathways which was essential to support the Wikidata resource. These changes can be found in the sars-cov-2-wikipathways-2 branch.</p></div><div id="sec-11" class="subsection"><h4>Scholia</h4><p id="p-29">The second use case is to demonstrate how we can link virus gene and protein information to literature. Here, we used Scholia (<a href="https://tools.wmflabs.org/scholia/">https://tools.wmflabs.org/scholia/</a>) as a central tool (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>). It provides a graphical interface around data in Wikidata, for example literature about a specific coronavirus protein (e.g. Q87917585 for the SARS-CoV-2 spike protein). Scholia uses SPARQL queries to provide information about topics. We annotated literature around the HCoVs with the specific virus strains, the virus genes, and the virus proteins as ‘main topic’.</p></div></div></div><div class="section" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Semantic data landscape</h3><p id="p-30">To align the different sources in Wikidata, a common data schema is needed. We have created a collection of schema’s that represent the structure of the items added to wikidata. Input to the workflow is the NCBI taxon identifier, which is input to mygene.info (see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 3</a>). Taxon information is obtained and added to Wikidata according to a set of linked Entity Schemas (<span class="underline">E170</span>: virus, <span class="underline">E174</span>: strain, <span class="underline">E69</span>: disease). Gene annotations are obtained and added to WIkidata following the Schemas (<span class="underline">E165</span>: virus gene, <span class="underline">E169</span>: virus protein) and protein annotations are obtained and added to Wikidata following the two schemas. The last two schemas are an extension from more generic schemas for proteins (<span class="underline">E167</span>) and genes (<span class="underline">E75</span>).</p></div><div id="sec-14" class="subsection"><h3>Bots</h3><p id="p-31">The bots developed and used in this protocol are adaptations of the bots developed in the Gene Wikiproject. On regular intervals the bots run to update viral gene and protein annotations as well as pathway updates from WikiPathways. For the gene and protein annotations, we have also made a Jupyter Notebook. The bot that synchronises virus, gene and protein information and the Jupyter Notebook are available at <a href="https://github.com/SuLab/Gene_Wiki_SARS-CoV">https://github.com/SuLab/Gene_Wiki_SARS-CoV</a>. The bot that synchronises the WikiPathways pathways with Wikidata was updated from the original version to allow adding proteins annotated with Wikidata identifiers and no longer requires pathways to be part of the Curated Collection. The customized bot source code is available at <a href="https://github.com/SuLab/scheduled-bots/blob/SARS-CoV-2/scheduled_bots/wikipathways/bot.py">https://github.com/SuLab/scheduled-bots/blob/SARS-CoV-2/scheduled_bots/wikipathways/bot.py</a>.</p><p id="p-32">Both bots are managed within a Jenkins environment allowing to schedule the synchronisation procedure at set intervals which can be changed depending on the update speed of the external resources. The Jenkins jobs are available from <a href="http://jenkins.sulab.org/">http://jenkins.sulab.org/</a>.</p></div><div id="sec-15" class="subsection"><h3>Data added</h3><p id="p-33">Using the gene and proteins bots explained in the Methods section, missing genes and proteins have been added for the seven human coronaviruses. The results are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1238582/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1238582/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1238582" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 :</span> <p id="p-34" class="first-child">Summary of the seven human coronaviruses, including taxon identifiers, the Wikidata items, and the number of genes and proteins. The latter two are generated by the SPARQL queries <kbd>geneCount.rq</kbd> and <kbd>proteinCount.rq</kbd> in the Supporting Information.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Use Cases</h3><div id="sec-17" class="subsection"><h4>BridgeDb</h4><p id="p-35">Using the dedicated code to create a BridgeDb identifier mapping database for coronaviruses, mappings were extracted from Wikidata with a SPARQL query for the seven human coronaviruses and the SARS-related viruses. This resulted in a mapping database with 392 mappings between 306 identifiers. This includes 140 Wikidata identifiers, 69 NCBI Gene identifiers, 42 UniProt identifiers, and 55 RefSeq identifiers. The mapping file has been released on the BridgeDb website (<a href="https://bridgedb.github.io/data/gene_database/">https://bridgedb.github.io/data/gene_database/</a>). The mapping database has also been loaded on the BridgeDb webservice at <a href="http://webservice.bridgedb.org/">http://webservice.bridgedb.org/</a> which means it can be used in the next use case: providing links out for WikiPathways.</p></div><div id="sec-18" class="subsection"><h4>WikiPathways</h4><p id="p-36">The WikiPathways project is involved in an international collaboration to curate knowledge about the biological processes around SARS-CoV-2 and COVID-19. The authors have started a pathway specifically about SARS-CoV-2 (<span class="underline">wikipathways:WP4846</span>). To ensure interoperability, WikiPathways uses BridgeDb and taking advantage of the enriched BridgeDb webservice, WikiPathways now links out for HCoV genes and proteins (depending on availability of mappings) to RefSef, NCBI Gene, UniProt, and Scholia (see <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 7</a>). The latter links to the next use case, and provides a link to literature about the virus. It should be noted that for each gene and protein, two Wikidata identifiers with links may be given. In that case, one is for the gene and one for the protein.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the shex schemas and the relations between them. All shapes, properties and items are available from within Wikidata." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the shex schemas and the relations between them. All shapes, properties and items are available from within Wikidata.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F8.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F8.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 :</span> <p id="p-37" class="first-child">Overview of the shex schemas and the relations between them. All shapes, properties and items are available from within Wikidata.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screenshot of SARS-CoV-2 and COVID-19 Pathway in WikiPathways (wikipathways:WP4846) showing the BridgeDb popup box for the ORF3a gene, showing a link out to Scholia via the gene&#x2019;s Wikidata identifier." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Screenshot of SARS-CoV-2 and COVID-19 Pathway in WikiPathways (wikipathways:WP4846) showing the BridgeDb popup box for the ORF3a gene, showing a link out to Scholia via the gene&#x2019;s Wikidata identifier.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F9.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F9.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 :</span> <p id="p-38" class="first-child">Screenshot of SARS-CoV-2 and COVID-19 Pathway in WikiPathways (<span class="underline">wikipathways:WP4846</span>) showing the BridgeDb popup box for the ORF3a gene, showing a link out to Scholia via the gene’s Wikidata identifier.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Scholia</h4><p id="p-39">The WikiPathways use case shows us that literature describes our knowledge about how coronaviruses work at a rather detailed level. Indeed, many articles discuss the genetics, homology of genes and proteins across viruses, or the molecular aspects of how these proteins are created and how they interfere with the biology of the human cell. The biological pathways show these processes, but ultimately the knowledge comes from literature. Wikidata allows us to link literature to specific virus proteins and genes, depending on what the article describes. For this it uses the ‘main subject’ property (<span class="underline">P921</span>). We manually annotated literature with the Wikidata items for specific proteins and genes. We developed two SPARQL queries to count the number of links between genes (<a href="https://w.wiki/Lsp">https://w.wiki/Lsp</a>) and proteins (<a href="https://w.wiki/Lsq">https://w.wiki/Lsq</a>) and the articles that discuss them. Scholia takes advantage of the ‘main subject’ annotation, allowing the creation of “topic” pages for each protein. For example, <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 8</a> shows the topic page of the SARS-CoV-2 spike protein.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screenshot of the Scholia page for the SARS-CoV-2 spike glycoprotein, it shows four articles that specifically discuss this protein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Screenshot of the Scholia page for the SARS-CoV-2 spike glycoprotein, it shows four articles that specifically discuss this protein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F10.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F10.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8 :</span> <p id="p-40" class="first-child">Screenshot of the Scholia page for the SARS-CoV-2 spike glycoprotein, it shows four articles that specifically discuss this protein.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-20"><h2 class="">Discussion</h2><p id="p-41">This paper describes a protocol we developed for our efforts taken to align genetic annotations from reference resources with Wikidata. Numerous annotations are scattered across different sources without any high-level of integration, thereby reducing the reusability of knowledge from different sources. Integration of the annotations from these resources is a complex and time consuming task. Each resource uses different ways to access the data from a user and machine perspective. Making use of these protocols programmatically to access and retrieve data of interest, requires the knowledge of various technologies and procedures to extract the information of interest.</p><p id="p-42">Wikidata provides a solution. It is part of the semantic web, taking advantage of its reification of the Wikidata items as RDF. Data in Wikidata itself is frequently, often almost instantaneously, synchronised with the RDF resource and available through its SPARQL endpoint (<a href="http://query.wikidata.org">http://query.wikidata.org</a>). The modelling process turns out to be an important aspect of this protocol. Wikidata contains numerous entity classes as entities and more than 7000 properties which are ready for (re-)use. However, that also means that one is easily lost. The ShEx schema have helped us develop a clear model, as a social contract between the authors of this paper, as well as documentation for future users.</p><p id="p-43">Using these schemas, it was simpler to validate the correctness of the updated bots to enter data in Wikidata. The bots have been transferred to the Gene Wiki Jenkins platform. This allows the bots to be kept running regularly, pending the ongoing efforts of the coronavirus and COVID-19 research communities. While the work of the bots will continue to need human oversight, potentially to correct errors, it provides a level of scalability and generally reduces the authors from a lot of repetitive work.</p><p id="p-44">One of the risks of using bots, is the possible generation of duplicate items. This can happen to humans as well, but they can apply a wider range of academic skills to resolve these issues. Indeed, in running the bots, duplicate Wikidata items were created, for which an example is shown in <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 9</a>. The Wikidataintegrator library does have functionality to prevent the creation of duplicates by comparing properties, based on used database identifiers. However, if two items have been created using different identifiers, these cannot be easily identified.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of two Wikidata entries for the SARS-CoV-2 M protein. Since this screenshot was made, the entries have been merged in a manually curation process." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-336976072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of two Wikidata entries for the SARS-CoV-2 M protein. Since this screenshot was made, the entries have been merged in a manually curation process.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9 :" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F11.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9 :" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F11.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9 :" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.05.026336/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9 :</span> <p id="p-45" class="first-child">Comparison of two Wikidata entries for the SARS-CoV-2 M protein. Since this screenshot was made, the entries have been merged in a manually curation process.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Close inspection of examples, such as the one in <a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 9</a>, showed that the duplicates were created because there was a lack of overlap between the data to be added and the existing item. The UniProt identifier did not yet resolve, because it was manually extracted from information in a March 27 pre-release (see <a href="http://ftp.uniprot.org/pub/databases/uniprot/pre_release/">ftp.uniprot.org/pub/databases/uniprot/pre_release/</a>). The Pfam identifier pointed to a page that did not contain mappings to other identifiers. In addition, the lack of references to the primary source hampers curators ability to merge duplicate items and expert knowledge was essential to identify the duplication. Fortunately, names used for these RNA viruses only refer to one protein as the membrane protein. Generally, the curator would have to revert to the primary literature to identify the overlap. Statements about ‘encoded by’ to the protein coding genes were found to be helpful as well. Reconciliation might be possible through sequence alignment, which means substantial expert knowledge and skills are required.</p><p id="p-47">This makes reconciliation in Wikidata based on matching labels, descriptions and synonyms, matching statements and captured provenance (qualifiers and references) hazardous, due to different meanings to the same label. A targeted curation query (<em>geneAndProteinCurationQuery.rq</em>, see Supporting Information) was developed to highlight such duplications and manually curated seven duplicate protein entries for SARS-CoV-2 alone. This duplication is common and to be expected, particularly in situations like a pandemic, when many groups contribute to the same effort. In this case, this paper only represents one group contributing to the <em>Wikidata:WikiProject COVID-19</em>.</p><p id="p-48">We also discovered that the virus taxonomy is different from those of mammalian species. For example, there is no clear NCBI taxon identifier for SARS-CoV-1 and after consultation with other projects, we defaulted to using the taxon identifier for the SARS-related CoVs, something that NCBI and UniProt seem to have done as well. Finally, we note that during the two weeks this effort took place, several other resources introduced overviews, including dedicated COVID-19 portals from UniProt (<a href="https://covid-19.uniprot.org/">https://covid-19.uniprot.org/</a>) and the Protein DataBank in Europe (<a href="https://PDBe.org/covid19/">https://PDBe.org/covid19/</a>).</p></div><div class="section" id="sec-21"><h2 class="">Conclusion</h2><p id="p-49">This manuscript presented a protocol to link information from disparate resources, including NCBI Taxonomy, NCBI Gene, UniProt, PubMed, and WikiPathways. Using the existing Wikidata infrastructure, we developed semantic schemas for virus strains, genes, and proteins, bots written in Python to add knowledge on genes and proteins of the seven human coronaviruses and linked them to biological pathways in WikiPathways and to primary literature, visualized in Scholia. We were able to do so in the period of two weeks, using an ad-hoc formed team from existing collaborations, taking advantage of the open nature of the components involved.</p></div><div class="section" id="sec-22"><h2 class="">Funding</h2><p id="p-50"><strong>National Institute of General Medical Sciences (R01 GM089820)</strong></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-51">Andra Waagmeester, Andrew I. Su</p></li></ul><p id="p-52"><strong>Alfred P. Sloan Foundation (grant number G-2019-11458)</strong></p><ul class="list-unord " id="list-2"><li id="list-item-2"><p id="p-53">Egon Willighagen</p></li></ul><p id="p-54"><strong>Spanish Ministry of Economy and Competitiveness (Society challenges: TIN2017-88877-R)</strong></p><ul class="list-unord " id="list-3"><li id="list-item-3"><p id="p-55">Jose Emilio Labra Gayo, Daniel Fernández Álvarez</p></li></ul><p id="p-56"><strong>Netherlands Organisation for Scientific Research funded UNLOCK project (NRGWI.obrug.2018.005)</strong></p><ul class="list-unord " id="list-4"><li id="list-item-4"><p id="p-57">Jasper Koehorst, Peter Schaap</p></li></ul></div><div class="section" id="sec-23"><h2 class="">Supporting Information</h2><div class="section" id="sec-24"><h3>geneAndProteinCurationQuery.rq</h3><span class="preformat dialog">
SELECT DISTINCT ?route ?gene ?geneLabel ?ncbi ?protein
?proteinLabel ?refseq ?uniprot WHERE {
 VALUES ?virus {
  wd:Q82069695 # SARS-CoV-2
 }
 {
  # genes
  ?wdRoot wdt:P703 ?virus ; wdt:P31 wd:Q7187 .
  OPTIONAL {?wdRoot wdt:P351 ?ncbi }
  OPTIONAL {
    ?wdRoot wdt:P688 | ^wdt:P702 ?protein .
    OPTIONAL {?protein wdt:P637 ?refseq }
    OPTIONAL {?protein wdt:P352 ?uniprot }
  }
  BIND (“via gene” AS ?route)
  BIND (?wdRoot AS ?gene)
 }
 UNION
 {
  # proteins
  ?wdRoot wdt:P703 ?virus ; wdt:P31 wd:Q8054 .
  OPTIONAL {
    ?gene wdt:P688 | ^wdt:P702 ?wdRoot
    OPTIONAL { ?gene wdt:P351 ?ncbi }
  }
  OPTIONAL { ?wdRoot wdt:P637 ?refseq }
  OPTIONAL { ?wdRoot wdt:P352 ?uniprot }
  BIND (“via protein” AS ?route)
  BIND (?wdRoot AS ?protein)
 }
 SERVICE wikibase:label {bd:serviceParam wikibase:language
“[AUTO_LANGUAGE],en”. }
} ORDER BY ?ncbi ?wikidata
</span></div><div class="section" id="sec-25"><h3>Gene counts for selected viruses: <strong>geneCount.rq</strong></h3><span class="preformat dialog">
SELECT ?virus ?virusLabel (COUNT(?item) AS ?genes) WHERE {
   VALUES ?virus {
   wd:Q82069695 # SARS-CoV-2
   wd:Q16983360 # HKU1
   wd:Q16991954 # OC43
   wd:Q8351095 # NL63
   wd:Q16983356 # 229E
   wd:Q4902157 # MERS-CoV
   wd:Q278567 # SARS-CoV
 }
   ?item wdt:P31 wd:Q7187 ;
         wdt:P703 ?virus .
SERVICE wikibase:label {bd:serviceParam wikibase:language
“[AUTO_LANGUAGE],en”. }
}
GROUP BY ?virus ?virusLabel
</span></div><div class="section" id="sec-26"><h3>Protein counts for selected viruses: <strong>proteinCount.rq</strong></h3><span class="preformat dialog">
SELECT ?virus ?virusLabel (COUNT(?item) AS ?genes) WHERE {
   VALUES ?virus {
   wd:Q82069695 # SARS-CoV-2
   wd:Q16983360 # HKU1
   wd:Q16991954 # OC43
   wd:Q8351095 # NL63
   wd:Q16983356 # 229E
   wd:Q4902157 # MERS-CoV
   wd:Q278567 # SARS-CoV
 }
   ?item wdt:P31 wd:Q8054 ;
         wdt:P703 ?virus .
SERVICE wikibase:label {bd:serviceParam wikibase:language
“[AUTO_LANGUAGE],en”. }
}
GROUP BY ?virus ?virusLabel
</span></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">The protocol presented in this work would not be as powerful if it was not for the works of many, which ensure that the fruits of our work are immediately interoperable with lots of other efforts, even though that integration was the whole purpose of developing this protocol. We thank the work of the larger Wikidata:WikiProject COVID-19 community in which we operated, Wikidata volunteers in general, and in particular Ralf Stephan, Daniel Mietchen, and Finn Nielsen. Second, the work is partly driven by a collaboration in the WikiPathways COVID-19 project and we particularly thank Anders Riutta for setting up a commit hook to the repository with WikiPathways COVID-19 pathways and the authors of these pathways.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watkins</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Preventing a covid-19 pandemic</span>. <abbr class="cit-jnl-abbrev">BMJ</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Feb</span> 28;<span class="cit-fpage">m810</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.2"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">outbreak.info [Internet]</span>. Available from: <a href="https://github.com/SuLab/outbreak.info">https://github.com/SuLab/outbreak.info</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.3"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Virus Outbreak Data Network (VODAN) [Internet]</span>. Available from:<a href="https://www.go-fair.org/implementation-networks/overview/vodan/">https://www.go-fair.org/implementation-networks/overview/vodan/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.04.05.026336v1.4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CORD-19-on-FHIR -- Semantics for COVID-19 Discovery [Internet]</span>. Available from: <a href="https://github.com/fhircat/CORD-19-on-FHIR">https://github.com/fhircat/CORD-19-on-FHIR</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.5"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">COVID-19</span> <span class="cit-source">Disease Map [Internet]</span>. Available from: <a href="https://covid.pages.uni.lu/map_curation">https://covid.pages.uni.lu/map_curation</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The Coronavirus and Open Science: Our reads and Open use cases [Internet]</span>. <span class="cit-pub-date">2020</span>. Available from: <a href="https://sparceurope.org/coronaopensciencereadsandusecases/">https://sparceurope.org/coronaopensciencereadsandusecases/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sondervan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brinkman</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imming</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Versteeg</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-source">The COVID-19 pandemic stresses the societal importance of open science. ScienceGuide [Internet]</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">Apr</span> 3; Available from: <a href="https://www.scienceguide.nl/2020/04/dire-times-of-covid-19-stress-the-societal-importance-of-open-science/">https://www.scienceguide.nl/2020/04/dire-times-of-covid-19-stress-the-societal-importance-of-open-science/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scarr</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelland</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-source">Speed Science: The risks of swiftly spreading coronavirus research [Internet]</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">Feb</span>. Available from: <a href="https://graphics.reuters.com/CHINA-HEALTH-RESEARCH/0100B5ES3MG/index.html">https://graphics.reuters.com/CHINA-HEALTH-RESEARCH/0100B5ES3MG/index.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mahase</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Covid-19: six million doses of hydroxychloroquine donated to US despite lack of evidence</span>. <abbr class="cit-jnl-abbrev">BMJ</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Mar</span> 23;m1166.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waagmeester</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stupp</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgstaller-Muehlbacher</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">OL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Wikidata as a knowledge graph for the life sciences</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Mar</span> 17;<span class="cit-vol">9</span>:<span class="cit-fpage">e52614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D9%26rft.spage%253De52614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.05.026336v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Krötzsch M.</span> <span class="cit-name-surname">Wikidata</span></span>. <abbr class="cit-jnl-abbrev">Commun ACM</abbr>. <span class="cit-pub-date">2014</span> <span class="cit-month">Sep</span>;<span class="cit-vol">57</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">78</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun%2BACM%26rft.volume%253D57%26rft.spage%253D78%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.05.026336v1.12" data-doi="10.1093/database/baw015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burgstaller-Muehlbacher</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waagmeester</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitraka</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putman</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Wikidata as a semantic framework for the Gene Wiki initiative</span>. <abbr class="cit-jnl-abbrev">Database</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-month">Mar</span>;<span class="cit-vol">2016</span>:<span class="cit-fpage">baw015.</span>#x002B;.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbaw015%26rft_id%253Dinfo%253Apmid%252F26989148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/baw015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26989148&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">FÅ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mietchen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willighagen</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-source">Scholia And Scientometrics With Wikidata</span>. <span class="cit-pub-date">2017</span> <span class="cit-month">Sep</span> 8 [cited 2018 Jun 26]; Available from: <a href="https://zenodo.org/record/1036595">https://zenodo.org/record/1036595</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.14"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Mika</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Tudorache</span>  <span class="cit-name-given-names">T</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Welty</span>  <span class="cit-name-given-names">C</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Knoblock</span>  <span class="cit-name-given-names">C</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D</span></span>,</li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Erxleben</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krötzsch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Introducing Wikidata to the Linked Data Web</span>. In: <span class="cit-ed"><span class="cit-name-surname">Mika</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-ed"><span class="cit-name-surname">Tudorache</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-ed"><span class="cit-name-surname">Welty</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-ed"><span class="cit-name-surname">Knoblock</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-ed"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span>, editors. <abbr class="cit-jnl-abbrev">The Semantic Web – ISWC 2014 [Internet]. Springer International Publishing</abbr>; <span class="cit-pub-date">2014</span>. p. <span class="cit-fpage">50</span>–<span class="cit-lpage">65</span>. (Lecture Notes in Computer Science; vol. <span class="cit-vol">8796</span>). Available from: <a href="http://dx.doi.org/10.1007/978-3-319-11964-9_4">http://dx.doi.org/10.1007/978-3-319-11964-9_4</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BSemantic%2BWeb%2B%253F%2BISWC%2B2014%2B%255BInternet%255D.%2BSpringer%2BInternational%2BPublishing%26rft.volume%253D8796%26rft.spage%253D50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cyganiak</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanthaler</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">RDF 1.1 Concepts and Abstract Syntax. W3C</abbr>; <span class="cit-pub-date">2014</span> <span class="cit-month">Feb</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.16"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Bontcheva</span>  <span class="cit-name-given-names">K</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Suárez-Figueroa</span>  <span class="cit-name-given-names">MC</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Presutti</span>  <span class="cit-name-given-names">V</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Celino</span>  <span class="cit-name-given-names">I</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Sabou</span>  <span class="cit-name-given-names">M</span></span>,</li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Malyshev</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krötzsch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonsior</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bielefeldt</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Getting the Most Out of Wikidata: Semantic Technology Usage in Wikipedia’s Knowledge Graph</span>. In: <span class="cit-ed"><span class="cit-name-surname">Vrandečić</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bontcheva</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-ed"><span class="cit-name-surname">Suárez-Figueroa</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-ed"><span class="cit-name-surname">Presutti</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-ed"><span class="cit-name-surname">Celino</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sabou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span>, editors. <span class="cit-source">The Semantic Web – ISWC 2018 [Internet]. Cham: Springer International Publishing</span>; <span class="cit-pub-date">2018</span> [cited 2020 Apr 4]. p. <span class="cit-fpage">376</span>–<span class="cit-lpage">94</span>. (Lecture Notes in Computer Science; vol. <span class="cit-vol">11137</span>). Available from: <a href="http://link.springer.com/10.1007/978-3-030-00668-6_23">http://link.springer.com/10.1007/978-3-030-00668-6_23</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BSemantic%2BWeb%2B%253F%2BISWC%2B2018%2B%255BInternet%255D.%2BCham%253A%2BSpringer%2BInternational%2BPublishing%26rft.volume%253D11137%26rft.spage%253D376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.17" data-doi="10.1093/nar/gkr1178"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Federhen</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">The NCBI Taxonomy database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">Jan</span> 1;<span class="cit-vol">40</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D136</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1178%26rft_id%253Dinfo%253Apmid%252F22139910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22139910&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.18" data-doi="10.1093/nar/gku1055"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hem</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ovetsky</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallin</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ermolaeva</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Gene: a gene-centered information resource at NCBI</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2015</span> <span class="cit-month">Jan</span> 28;<span class="cit-vol">43</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1055%26rft_id%253Dinfo%253Apmid%252F25355515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25355515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.19" data-doi="10.1093/nar/gkw1099"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Consortium</span>  <span class="cit-name-given-names">TU</span></span>. <span class="cit-article-title">UniProt: the universal protein knowledgebase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2017</span> <span class="cit-month">Jan</span>;<span class="cit-vol">45</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D158</span>–<span class="cit-lpage">D169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1099%26rft_id%253Dinfo%253Apmid%252F27899622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">wwPDB</span> <span class="cit-name-surname">consortium</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burley</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhikadiya</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Protein Data Bank: the single global archive for 3D macromolecular structure data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Jan</span> 8;<span class="cit-vol">47</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D520</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.21" data-doi="10.1093/nar/gkx1064."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slenter</span>  <span class="cit-name-given-names">DN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kutmon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanspers</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riutta</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jan</span> 4;<span class="cit-vol">46</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D661</span>–<span class="cit-lpage">D667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD661%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1064.%26rft_id%253Dinfo%253Apmid%252F29136241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1064.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29136241&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.22"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Hitzler</span>  <span class="cit-name-given-names">P</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Janowicz</span>  <span class="cit-name-given-names">K</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Zaveri</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">AJG</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">V</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solbrig</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stupp</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labra Gayo</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mietchen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prud’hommeaux</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Using Shape Expressions (ShEx) to Share RDF Data Models and to Guide Curation with Rigorous Validation</span>. In: <span class="cit-ed"><span class="cit-name-surname">Hitzler</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fernández</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-ed"><span class="cit-name-surname">Janowicz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-ed"><span class="cit-name-surname">Zaveri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">AJG</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span>, editors. <abbr class="cit-jnl-abbrev">The Semantic Web [Internet]. Cham: Springer International Publishing</abbr>; <span class="cit-pub-date">2019</span> [cited 2020 Apr 4]. p. <span class="cit-fpage">606</span>–<span class="cit-lpage">20</span>. (Lecture Notes in Computer Science; vol. <span class="cit-vol">11503</span>). Available from: <a href="http://link.springer.com/10.1007/978-3-030-21348-0_39">http://link.springer.com/10.1007/978-3-030-21348-0_39</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BSemantic%2BWeb%2B%255BInternet%255D.%2BCham%253A%2BSpringer%2BInternational%2BPublishing%26rft.volume%253D11503%26rft.spage%253D606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.05.026336v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prud’hommeaux</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labra Gayo</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solbrig</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-chapter-title">Shape expressions: an RDF validation and transformation language</span>. In: <span class="cit-source">Proceedings of the 10th International Conference on Semantic Systems - SEM ‘14 [Internet]</span>. <span class="cit-publ-name">Leipzig, Germany: ACM Press</span>; <span class="cit-pub-date">2014</span> [cited 2020 Apr 5]. p. <span class="cit-fpage">32</span>–<span class="cit-lpage">40</span>. Available from: <a href="http://dl.acm.org/citation.cfm?doid=2660517.2660523">http://dl.acm.org/citation.cfm?doid=2660517.2660523</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.24" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">Feb</span> 20;<span class="cit-vol">382</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">727</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.25" data-doi="10.1038/scientificamerican0501-34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berners-Lee</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hendler</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lassila</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">The Semantic Web</span>. <abbr class="cit-jnl-abbrev">Sci Am</abbr>. <span class="cit-pub-date">2001</span> <span class="cit-month">May</span>;<span class="cit-vol">284</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">34</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BAmerican%26rft.stitle%253DScientific%2BAmerican%26rft.aulast%253DStix%26rft.auinit1%253DG.%26rft.volume%253D284%26rft.issue%253D6%26rft.spage%253D34%26rft.epage%253D35%26rft.atitle%253DThe%2Bmice%2Bthat%2Bwarred.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fscientificamerican0501-34%26rft_id%253Dinfo%253Apmid%252F11396337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/scientificamerican0501-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11396337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168819100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seaborne</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">SPARQL 1.1 Query Language</span>. <abbr class="cit-jnl-abbrev">W3C</abbr>; <span class="cit-pub-date">2013</span> <span class="cit-month">Mar</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.27"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">W3C OWL Working Group. OWL 2 Web Ontology Language Document Overview [Internet]</span>. <span class="cit-pub-date">2012</span> <span class="cit-month">Dec</span>. Available from: <a href="https://www.w3.org/TR/owl2-overview/">https://www.w3.org/TR/owl2-overview/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berners-Lee</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-source">Linked Data [Internet]. 2006</span>. Available from: <a href="https://www.w3.org/DesignIssues/LinkedData.html">https://www.w3.org/DesignIssues/LinkedData.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Samwald</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jentzsch</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kallesøe</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willighagen</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hajagos</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Linked open drug data for pharmaceutical research and development</span>. <abbr class="cit-jnl-abbrev">J Cheminformatics</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">3</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BCheminformatics%26rft.volume%253D3%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hernández</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krötzsch</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Reifying RDF: What Works Well With Wikidata?</span> In: <span class="cit-source">Proceedings of the 11th International Workshop on Scalable Semantic Web Knowledge Base Systems co-located with 14th International Semantic Web Conference (ISWC 2015) [Internet]</span>. <span class="cit-publ-loc">Bethlehem, PA, USA</span>; <span class="cit-pub-date">2015</span>. (CEUR Workshop Proceedings). Available from: <a href="http://ceur-ws.org/Vol-1457/SSWS2015_paper3.pdf">http://ceur-ws.org/Vol-1457/SSWS2015_paper3.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.31"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Jose Emilio Labra Gayo</span>. <span class="cit-source">RDFSHape: Online demo implementation of ShEx and SHACL [Internet]. Zenodo; 2018 [cited 2020 Apr 4]</span>. Available from: <a href="https://zenodo.org/record/1412128">https://zenodo.org/record/1412128</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.32"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Pablo Menéndez Suárez</span>, <span class="cit-source">Jose Emilio Labra Labra Gayo. YaShE [Internet]. Zenodo; 2020 [cited 2020 Apr 4]</span>. Available from: <a href="https://zenodo.org/record/3739108">https://zenodo.org/record/3739108</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026336v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sayers</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-source">E-utilities Quick Start [Internet]. NCBI; 2018</span>. Available from: <a href="https://www.ncbi.nlm.nih.gov/books/NBK25500/">https://www.ncbi.nlm.nih.gov/books/NBK25500/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.34" data-doi="10.1093/nar/gks1114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacLeod</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">AI</span></span>. <span class="cit-article-title">BioGPS and MyGene.info: organizing online, gene-centric information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Jan</span> 1;<span class="cit-vol">41</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D561</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1114%26rft_id%253Dinfo%253Apmid%252F23175613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23175613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.05.026336.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300079&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Iersel</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pico</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelder</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanspers</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2010</span> <span class="cit-month">Jan</span>;<span class="cit-vol">11</span>(<span class="cit-issue">1</span>):5+.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026336v1.36"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">CA</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Waagmeester</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kutmon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riutta</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willighagen</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evelo</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Using the Semantic Web for Rapid Integration of WikiPathways with Other Biological Online Data Resources</span>. <span class="cit-ed"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">CA</span></span>, editor. <abbr class="cit-jnl-abbrev">PLOS Comput Biol</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-month">Jun</span> 23;<span class="cit-vol">12</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">e1004989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput%2BBiol%26rft.volume%253D12%26rft.spage%253De1004989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.05.026336v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.06.027540v1" title="The dynamic Nexus: Gap junctions control protein localization and mobility in distinct and surprising ways" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.02.022327v1" title="Dynamic changes in urgency over the course of a decision" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 07, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.05.026336v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.05.026336v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.05.026336v1" class="active" data-icon-position="" data-hide-link-title="0">A protocol for adding knowledge to Wikidata, a case report</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1232125" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yyWjng_zFWGpJ2f1mm_BkdrgvcDnhLzt2csmS0q0cPE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232125" id="node1232125--2" data-pisa="biorxiv;2020.04.05.026336v1" data-pisa-master="biorxiv;2020.04.05.026336" data-seqnum="1232125" data-apath="/biorxiv/early/2020/04/07/2020.04.05.026336.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A protocol for adding knowledge to Wikidata, a case report</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Andra</span> <span class="nlm-surname">Waagmeester</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Egon L.</span> <span class="nlm-surname">Willighagen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew I</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martina</span> <span class="nlm-surname">Kutmon</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jose Emilio</span> <span class="nlm-surname">Labra Gayo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Fernández-Álvarez</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peter J.</span> <span class="nlm-surname">Schaap</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Lisa M.</span> <span class="nlm-surname">Verhagen</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jasper J.</span> <span class="nlm-surname">Koehorst</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.05.026336; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.05.026336 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;title=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;title=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;text=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;title=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;t=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;title=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;title=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232125" id="node1232125--3" data-pisa="biorxiv;2020.04.05.026336v1" data-pisa-master="biorxiv;2020.04.05.026336" data-seqnum="1232125" data-apath="/biorxiv/early/2020/04/07/2020.04.05.026336.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A protocol for adding knowledge to Wikidata, a case report</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Andra</span> <span class="nlm-surname">Waagmeester</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Egon L.</span> <span class="nlm-surname">Willighagen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew I</span> <span class="nlm-surname">Su</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martina</span> <span class="nlm-surname">Kutmon</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jose Emilio</span> <span class="nlm-surname">Labra Gayo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Fernández-Álvarez</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peter J.</span> <span class="nlm-surname">Schaap</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Lisa M.</span> <span class="nlm-surname">Verhagen</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jasper J.</span> <span class="nlm-surname">Koehorst</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.05.026336; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.05.026336 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Waagmeester%2C+Andra&amp;rft.au=Willighagen%2C+Egon+L.&amp;rft.au=Su%2C+Andrew+I&amp;rft.au=Kutmon%2C+Martina&amp;rft.au=Labra+Gayo%2C+Jose+Emilio&amp;rft.au=Fern%C3%A1ndez-%C3%81lvarez%2C+Daniel&amp;rft.au=Schaap%2C+Peter+J.&amp;rft.au=Verhagen%2C+Lisa+M.&amp;rft.au=Koehorst%2C+Jasper+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1232125/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1232125/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1232125/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1232125/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1232125/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1232125/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1232125/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1232125/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1232125/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1232125/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1232125/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1232125/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;count=horizontal&amp;via=&amp;text=A%20protocol%20for%20adding%20knowledge%20to%20Wikidata%2C%20a%20case%20report&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026336v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.05.026336v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
