<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.26.010165.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.26.010165v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies" />
<meta name="DC.Identifier" content="10.1101/2020.03.26.010165" />
<meta name="DC.Date" content="2020-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks." />
<meta name="DC.Contributor" content="Daniel Wrapp" />
<meta name="DC.Contributor" content="Dorien De Vlieger" />
<meta name="DC.Contributor" content="Kizzmekia S. Corbett" />
<meta name="DC.Contributor" content="Gretel M. Torres" />
<meta name="DC.Contributor" content="Wander Van Breedam" />
<meta name="DC.Contributor" content="Kenny Roose" />
<meta name="DC.Contributor" content="Loes van Schie" />
<meta name="DC.Contributor" content="VIB-CMB COVID-19 Response Team" />
<meta name="DC.Contributor" content="Markus Hoffmann" />
<meta name="DC.Contributor" content="Stefan Pöhlmann" />
<meta name="DC.Contributor" content="Barney S. Graham" />
<meta name="DC.Contributor" content="Nico Callewaert" />
<meta name="DC.Contributor" content="Bert Schepens" />
<meta name="DC.Contributor" content="Xavier Saelens" />
<meta name="DC.Contributor" content="Jason S. McLellan" />
<meta name="article:published_time" content="2020-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.26.010165v1" />
<meta name="citation_id" content="2020.03.26.010165v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.26.010165" />
<meta name="citation_num_pages" content="53" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.26.010165" />
<meta name="citation_author" content="Daniel Wrapp" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin" />
<meta name="citation_author" content="Dorien De Vlieger" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author_institution" content="Department of Biomedical Molecular Biology, Ghent University" />
<meta name="citation_author" content="Kizzmekia S. Corbett" />
<meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health" />
<meta name="citation_author" content="Gretel M. Torres" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Geisel School of Medicine at Dartmouth" />
<meta name="citation_author" content="Wander Van Breedam" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author" content="Kenny Roose" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author" content="Loes van Schie" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author" content="VIB-CMB COVID-19 Response Team" />
<meta name="citation_author" content="Markus Hoffmann" />
<meta name="citation_author_institution" content="Infection Biology Unit, German Primate Center – Leibniz Institute for Primate Research" />
<meta name="citation_author" content="Stefan Pöhlmann" />
<meta name="citation_author_institution" content="Infection Biology Unit, German Primate Center – Leibniz Institute for Primate Research" />
<meta name="citation_author_institution" content="Faculty of Biology and Psychology, University Göttingen" />
<meta name="citation_author" content="Barney S. Graham" />
<meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8112-0853" />
<meta name="citation_author" content="Nico Callewaert" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author" content="Bert Schepens" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author_institution" content="Department of Biomedical Molecular Biology, Ghent University" />
<meta name="citation_author_email" content="bert.schepens@vib-ugent.be" />
<meta name="citation_author_email" content="xavier.saelens@vib-ugent.be" />
<meta name="citation_author_email" content="jmclellan@austin.utexas.edu" />
<meta name="citation_author" content="Xavier Saelens" />
<meta name="citation_author_institution" content="VIB Center for Medical Biotechnology" />
<meta name="citation_author_institution" content="Department of Biochemistry and Microbiology, Ghent University" />
<meta name="citation_author_institution" content="Department of Biomedical Molecular Biology, Ghent University" />
<meta name="citation_author_email" content="bert.schepens@vib-ugent.be" />
<meta name="citation_author_email" content="xavier.saelens@vib-ugent.be" />
<meta name="citation_author_email" content="jmclellan@austin.utexas.edu" />
<meta name="citation_author" content="Jason S. McLellan" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin" />
<meta name="citation_author_email" content="bert.schepens@vib-ugent.be" />
<meta name="citation_author_email" content="xavier.saelens@vib-ugent.be" />
<meta name="citation_author_email" content="jmclellan@austin.utexas.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3991-542X" />
<meta name="citation_reference" content="Adams, P.D., Grosse-Kunstleve, R.W., Hung, L.W., Ioerger, T.R., McCoy, A.J., Moriarty, N.W., Read, R.J., Sacchettini, J.C., Sauter, N.K., and Terwilliger, T.C. (2002). PHENIX: building new software for automated crystallographic structure determination. Acta Crystallogr D Biol Crystallogr 58, 1948–1954." />
<meta name="citation_reference" content="Battye, T.G., Kontogiannis, L., Johnson, O., Powell, H.R., and Leslie, A.G. (2011). iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM. Acta Crystallogr D Biol Crystallogr 67, 271–281." />
<meta name="citation_reference" content="Berger Rentsch, M., and Zimmer, G. (2011). A vesicular stomatitis virus replicon-based bioassay for the rapid and sensitive determination of multi-species type I interferon. PLoS One 6, e25858." />
<meta name="citation_reference" content="Bosch, B.J., van der Zee, R., de Haan, C.A., and Rottier, P.J. (2003). The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex. J Virol 77, 8801–8811." />
<meta name="citation_reference" content="Chan, J.F., Yuan, S., Kok, K.H., To, K.K., Chu, H., Yang, J., Xing, F., Liu, J., Yip, C.C., Poon, R.W., et al. (2020). A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet 395, 514–523." />
<meta name="citation_reference" content="Chen, Z., Bao, L., Chen, C., Zou, T., Xue, Y., Li, F., Lv, Q., Gu, S., Gao, X., Cui, S., et al. (2017). Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset. J Infect Dis 215, 1807–1815." />
<meta name="citation_reference" content="Croll, T.I. (2018). ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Acta Crystallogr D Struct Biol 74, 519–530." />
<meta name="citation_reference" content="De Vlieger, D., Ballegeer, M., Rossey, I., Schepens, B., and Saelens, X. (2018). Single-Domain Antibodies and Their Formatting to Combat Viral Infections. Antibodies (Basel) 8." />
<meta name="citation_reference" content="Detalle, L., Stohr, T., Palomo, C., Piedra, P.A., Gilbert, B.E., Mas, V., Millar, A., Power, U.F., Stortelers, C., Allosery, K., et al. (2016). Generation and Characterization of ALX-0171, a Potent Novel Therapeutic Nanobody for the Treatment of Respiratory Syncytial Virus Infection. Antimicrob Agents Chemother 60, 6–13." />
<meta name="citation_reference" content="Dumoulin, M., Conrath, K., Van Meirhaeghe, A., Meersman, F., Heremans, K., Frenken, L.G., Muyldermans, S., Wyns, L., and Matagne, A. (2002). Single-domain antibody fragments with high conformational stability. Protein Sci 11, 500–515." />
<meta name="citation_reference" content="Emsley, P., and Cowtan, K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr D Biol Crystallogr 60, 2126–2132." />
<meta name="citation_reference" content="Evans, P.R., and Murshudov, G.N. (2013). How good are my data and what is the resolution? Acta Crystallogr D Biol Crystallogr 69, 1204–1214." />
<meta name="citation_reference" content="Gaunt, E.R., Hardie, A., Claas, E.C., Simmonds, P., and Templeton, K.E. (2010). Epidemiology and clinical presentations of the four human coronaviruses 229E, HKU1, NL63, and OC43 detected over 3 years using a novel multiplex real-time PCR method. J Clin Microbiol 48, 2940–2947." />
<meta name="citation_reference" content="Ge, X.Y., Li, J.L., Yang, X.L., Chmura, A.A., Zhu, G., Epstein, J.H., Mazet, J.K., Hu, B., Zhang, W., Peng, C., et al. (2013). Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538." />
<meta name="citation_reference" content="Govaert, J., Pellis, M., Deschacht, N., Vincke, C., Conrath, K., Muyldermans, S., and Saerens, D. (2012). Dual beneficial effect of interloop disulfide bond for single domain antibody fragments. J Biol Chem 287, 1970–1979." />
<meta name="citation_reference" content="Gui, M., Song, W., Zhou, H., Xu, J., Chen, S., Xiang, Y., and Wang, X. (2017). Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res 27, 119–129." />
<meta name="citation_reference" content="Hamers-Casterman, C., Atarhouch, T., Muyldermans, S., Robinson, G., Hamers, C., Songa, E.B., Bendahman, N., and Hamers, R. (1993). Naturally occurring antibodies devoid of light chains. Nature 363, 446–448." />
<meta name="citation_reference" content="Hoffmann, M.K.-W., H; Krüger, N; Müller, M; Drosten, C; Pöhlmann, S. (2020). The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. bioRxiv." />
<meta name="citation_reference" content="Huang, C., Wang, Y., Li, X., Ren, L., Zhao, J., Hu, Y., Zhang, L., Fan, G., Xu, J., Gu, X., et al. (2020). Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506." />
<meta name="citation_reference" content="Hwang, W.C., Lin, Y., Santelli, E., Sui, J., Jaroszewski, L., Stec, B., Farzan, M., Marasco, W.A., and Liddington, R.C. (2006). Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. J Biol Chem 281, 34610-34616." />
<meta name="citation_reference" content="Ibanez, L.I., De Filette, M., Hultberg, A., Verrips, T., Temperton, N., Weiss, R.A., Vandevelde, W., Schepens, B., Vanlandschoot, P., and Saelens, X. (2011). Nanobodies with in vitro neutralizing activity protect mice against H5N1 influenza virus infection. J Infect Dis 203, 1063–1072." />
<meta name="citation_reference" content="Jones, H.G., Wrapp, D., Gilman, M.S.A., Battles, M.B., Wang, N., Sacerdote, S., Chuang, G.Y., Kwong, P.D., and McLellan, J.S. (2019). Iterative screen optimization maximizes the efficiency of macromolecular crystallization. Acta Crystallogr F Struct Biol Commun 75, 123–131." />
<meta name="citation_reference" content="Kirchdoerfer, R.N., Cottrell, C.A., Wang, N., Pallesen, J., Yassine, H.M., Turner, H.L., Corbett, K.S., Graham, B.S., McLellan, J.S., and Ward, A.B. (2016). Pre-fusion structure of a human coronavirus spike protein. Nature 531, 118–121." />
<meta name="citation_reference" content="Kirchdoerfer, R.N., Wang, N., Pallesen, J., Wrapp, D., Turner, H.L., Cottrell, C.A., Corbett, K.S., Graham, B.S., McLellan, J.S., and Ward, A.B. (2018). Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis. Scientific reports 8, 15701." />
<meta name="citation_reference" content="Koch, K., Kalusche, S., Torres, J.L., Stanfield, R.L., Danquah, W., Khazanehdari, K., von Briesen, H., Geertsma, E.R., Wilson, I.A., Wernery, U., et al. (2017). Selection of nanobodies with broad neutralizing potential against primary HIV-1 strains using soluble subtype C gp140 envelope trimers. Sci Rep 7, 8390." />
<meta name="citation_reference" content="Ksiazek, T.G., Erdman, D., Goldsmith, C.S., Zaki, S.R., Peret, T., Emery, S., Tong, S., Urbani, C., Comer, J.A., Lim, W., et al. (2003). A novel coronavirus associated with severe acute respiratory syndrome. N Engl J Med 348, 1953–1966." />
<meta name="citation_reference" content="Lan, J.G., J; Yu, J; Shan, S; Zhou, H; Fan, S; Zhang, Q; Shi, X; Wang, Q; Zhang, L; Wang, X. (2020). Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. bioRxiv." />
<meta name="citation_reference" content="Larios Mora, A., Detalle, L., Gallup, J.M., Van Geelen, A., Stohr, T., Duprez, L., and Ackermann, M.R. (2018). Delivery of ALX-0171 by inhalation greatly reduces respiratory syncytial virus disease in newborn lambs. MAbs 10, 778–795." />
<meta name="citation_reference" content="Laursen, N.S., Friesen, R.H.E., Zhu, X., Jongeneelen, M., Blokland, S., Vermond, J., van Eijgen, A., Tang, C., van Diepen, H., Obmolova, G., et al. (2018). Universal protection against influenza infection by a multidomain antibody to influenza hemagglutinin. Science 362, 598–602." />
<meta name="citation_reference" content="Li, F., Li, W., Farzan, M., and Harrison, S.C. (2005). Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science 309, 1864–1868." />
<meta name="citation_reference" content="Li, W., Moore, M.J., Vasilieva, N., Sui, J., Wong, S.K., Berne, M.A., Somasundaran, M., Sullivan, J.L., Luzuriaga, K., Greenough, T.C., et al. (2003). Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454." />
<meta name="citation_reference" content="Li, Y., Wan, Y., Liu, P., Zhao, J., Lu, G., Qi, J., Wang, Q., Lu, X., Wu, Y., Liu, W., et al. (2015). A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein. Cell Res 25, 1237–1249." />
<meta name="citation_reference" content="Lin-Cereghino, J., Wong, W.W., Xiong, S., Giang, W., Luong, L.T., Vu, J., Johnson, S.D., and Lin-Cereghino, G.P. (2005). Condensed protocol for competent cell preparation and transformation of the methylotrophic yeast Pichia pastoris. Biotechniques 38, 44, 46, 48." />
<meta name="citation_reference" content="Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., Wang, W., Song, H., Huang, B., Zhu, N., et al. (2020). Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574." />
<meta name="citation_reference" content="McCoy, A.J. (2007). Solving structures of protein complexes by molecular replacement with Phaser. Acta Crystallogr D Biol Crystallogr 63, 32–41." />
<meta name="citation_reference" content="McLellan, J.S., Chen, M., Joyce, M.G., Sastry, M., Stewart-Jones, G.B., Yang, Y., Zhang, B., Chen, L., Srivatsan, S., Zheng, A., et al. (2013). Structure-based design of a fusion glycoprotein vaccine for respiratory syncytial virus. Science (New York, NY) 342, 592–598." />
<meta name="citation_reference" content="Morin, A., Eisenbraun, B., Key, J., Sanschagrin, P.C., Timony, M.A., Ottaviano, M., and Sliz, P. (2013). Collaboration gets the most out of software. Elife 2, e01456." />
<meta name="citation_reference" content="Pak, J.E., Sharon, C., Satkunarajah, M., Auperin, T.C., Cameron, C.M., Kelvin, D.J., Seetharaman, J., Cochrane, A., Plummer, F.A., Berry, J.D., et al. (2009). Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain. J Mol Biol 388, 815–823." />
<meta name="citation_reference" content="Pallesen, J., Wang, N., Corbett, K.S., Wrapp, D., Kirchdoerfer, R.N., Turner, H.L., Cottrell, C.A., Becker, M.M., Wang, L., Shi, W., et al. (2017). Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proc Natl Acad Sci U S A 114, E7348–E7357." />
<meta name="citation_reference" content="Prabakaran, P., Gan, J., Feng, Y., Zhu, Z., Choudhry, V., Xiao, X., Ji, X., and Dimitrov, D.S. (2006). Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody. J Biol Chem 281, 15829–15836." />
<meta name="citation_reference" content="Raj, V.S., Mou, H., Smits, S.L., Dekkers, D.H., Muller, M.A., Dijkman, R., Muth, D., Demmers, J.A., Zaki, A., Fouchier, R.A., et al. (2013). Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. Nature 495, 251–254." />
<meta name="citation_reference" content="Respaud, R., Vecellio, L., Diot, P., and Heuze-Vourc’h, N. (2015). Nebulization as a delivery method for mAbs in respiratory diseases. Expert Opin Drug Deliv 12, 1027–1039." />
<meta name="citation_reference" content="Rossey, I., Gilman, M.S., Kabeche, S.C., Sedeyn, K., Wrapp, D., Kanekiyo, M., Chen, M., Mas, V., Spitaels, J., Melero, J.A., et al. (2017). Potent single-domain antibodies that arrest respiratory syncytial virus fusion protein in its prefusion state. Nat Commun 8, 14158." />
<meta name="citation_reference" content="Schoonooghe, S., Kaigorodov, V., Zawisza, M., Dumolyn, C., Haustraete, J., Grooten, J., and Mertens, N. (2009). Efficient production of human bivalent and trivalent anti-MUC1 Fab-scFv antibodies in Pichia pastoris. BMC Biotechnol 9, 70." />
<meta name="citation_reference" content="Stalin Raj, V., Okba, N.M.A., Gutierrez-Alvarez, J., Drabek, D., van Dieren, B., Widagdo, W., Lamers, M.M., Widjaja, I., Fernandez-Delgado, R., Sola, I., et al. (2018). Chimeric camel/human heavy-chain antibodies protect against MERS-CoV infection. Sci Adv 4, eaas9667." />
<meta name="citation_reference" content="Tian, X., Li, C., Huang, A., Xia, S., Lu, S., Shi, Z., Lu, L., Jiang, S., Yang, Z., Wu, Y., et al. (2020). Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerg Microbes Infect 9, 382–385." />
<meta name="citation_reference" content="van der Linden, R.H., Frenken, L.G., de Geus, B., Harmsen, M.M., Ruuls, R.C., Stok, W., de Ron, L., Wilson, S., Davis, P., and Verrips, C.T. (1999). Comparison of physical chemical properties of llama VHH antibody fragments and mouse monoclonal antibodies. Biochim Biophys Acta 1431, 37–46." />
<meta name="citation_reference" content="Walls, A.C., Tortorici, M.A., Bosch, B.J., Frenz, B., Rottier, P.J.M., DiMaio, F., Rey, F.A., and Veesler, D. (2016). Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer. Nature 531, 114–117." />
<meta name="citation_reference" content="Walls, A.C., Xiong, X., Park, Y.J., Tortorici, M.A., Snijder, J., Quispe, J., Cameroni, E., Gopal, R., Dai, M., Lanzavecchia, A., et al. (2019). Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 176, 1026–1039 e1015." />
<meta name="citation_reference" content="Walls, A.P. Y-J; Tortorici, MA; Wall, A; McGuire, AT; Veesler, D. (2020). Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell." />
<meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R.S., and Li, F. (2020). Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol." />
<meta name="citation_reference" content="Wang, L., Shi, W., Chappell, J.D., Joyce, M.G., Zhang, Y., Kanekiyo, M., Becker, M.M., van Doremalen, N., Fischer, R., Wang, N., et al. (2018). Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape. J Virol 92." />
<meta name="citation_reference" content="Wang, L., Shi, W., Joyce, M.G., Modjarrad, K., Zhang, Y., Leung, K., Lees, C.R., Zhou, T., Yassine, H.M., Kanekiyo, M., et al. (2015). Evaluation of candidate vaccine approaches for MERS-CoV. Nat Commun 6, 7712." />
<meta name="citation_reference" content="Wang, N., Shi, X., Jiang, L., Zhang, S., Wang, D., Tong, P., Guo, D., Fu, L., Cui, Y., Liu, X., et al. (2013). Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4. Cell Res 23, 986–993." />
<meta name="citation_reference" content="Wang, Q., Qi, J., Yuan, Y., Xuan, Y., Han, P., Wan, Y., Ji, W., Li, Y., Wu, Y., Wang, J., et al. (2014). Bat origins of MERS-CoV supported by bat coronavirus HKU4 usage of human receptor CD26. Cell Host Microbe 16, 328–337." />
<meta name="citation_reference" content="Woo, P.C., Lau, S.K., Huang, Y., and Yuen, K.Y. (2009). Coronavirus diversity, phylogeny and interspecies jumping. Exp Biol Med (Maywood) 234, 1117–1127." />
<meta name="citation_reference" content="Wrapp, D., Wang, N., Corbett, K.S., Goldsmith, J.A., Hsieh, C.L., Abiona, O., Graham, B.S., and McLellan, J.S. (2020). Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (New York, NY)." />
<meta name="citation_reference" content="Yan, R.Z. Y; Li, Y; Xia, L; Guo, Y; Zhou, Q. (2020). Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science." />
<meta name="citation_reference" content="Ying, T., Prabakaran, P., Du, L., Shi, W., Feng, Y., Wang, Y., Wang, L., Li, W., Jiang, S., Dimitrov, D.S., et al. (2015). Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody. Nat Commun 6, 8223." />
<meta name="citation_reference" content="Yu, X., Zhang, S., Jiang, L., Cui, Y., Li, D., Wang, D., Wang, N., Fu, L., Shi, X., Li, Z., et al. (2015). Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27. Sci Rep 5, 13133." />
<meta name="citation_reference" content="Yuan, M W.N., Zhu X, Lee C-CD, So RTY, Lv H, Mok CKP, Wilson IA. (2020). A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV. bioRxiv." />
<meta name="citation_reference" content="Yuan, Y., Cao, D., Zhang, Y., Ma, J., Qi, J., Wang, Q., Lu, G., Wu, Y., Yan, J., Shi, Y., et al. (2017). Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat Commun 8, 15092." />
<meta name="citation_reference" content="Zaki, A.M., van Boheemen, S., Bestebroer, T.M., Osterhaus, A.D., and Fouchier, R.A. (2012). Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med 367, 1814–1820." />
<meta name="citation_reference" content="Zhao, G., He, L., Sun, S., Qiu, H., Tai, W., Chen, J., Li, J., Chen, Y., Guo, Y., Wang, Y., et al. (2018). A Novel Nanobody Targeting Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Receptor-Binding Domain Has Potent Cross-Neutralizing Activity and Protective Efficacy against MERS-CoV. J Virol 92." />
<meta name="citation_reference" content="Zhou, P., Yang, X.L., Wang, X.G., Hu, B., Zhang, L., Zhang, W., Si, H.R., Zhu, Y., Li, B., Huang, C.L., et al. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature." />
<meta name="twitter:title" content="Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks." />
<meta name="og-title" property="og:title" content="Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.26.010165v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.26.010165v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.26.010165v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1215887" />
    <title>Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UzWXGfWs-C87V2oeQdm5wHrAeBYRUl548VqBaksY_1Y","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1215887","apath":"\/biorxiv\/early\/2020\/03\/28\/2020.03.26.010165.atom","pisa":"biorxiv;2020.03.26.010165v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.26.010165v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.26.010165v1","title":"Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies","identifier":"node\/1215887"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sWC+s0FFkgCK\\\\\\\/BdA1HKKNRy9FjFgBizefmCf+FdQ\\\\\\\/nd+lcgK4zbSmzVzk9olGo2nGc0TORa\\\\\\\/MXX+qxCcU7s6wvWHNDOjgwQJ0ZAbqMzZHxAl0q86IvvtZ3GSHZ2JUTq3HOZDqSyVUli9yvMOecH\\\\\\\/GSbSg86KGcRDh2ZyOSqIdsDuozkAM\\\\\\\/RGuUSlv83tWX2mhdEPT+oqZcrRV+GS0OBMa+uwWlfAlAVlW2NA6HSzTWTxzvkfnq1Ehnh8jbB3SlJiUFWysHO6pJFMoPpbgb96pDvWeaPOsIqsWzsz1m7RoG529CCxTA99UDo3PFa2kLg+z3QsbS5ujQ+1MQgmz+OCMqlUoDUdYdyBTLsM92dypg+xtqEHejEMYrQY3JwSg0cUxPXXQFvPRxaxfqVaW2nbWr10BbiZbgXVAg1jdP8QZZN6kqIwPsbuE0Gc+t7DynHvhrSvVGA5Xmq4C2ycK9vJN+paxXWPnDKu9xrMOhPE9cR1dm2+TrIYgjqJoTNUzXHzrnsg7VTsijf2YEFXYQULa5upiQinJdcMQv3mXZH5OtVeSHL+TDm5MvqFyGHIOZr025iyQ0JaKCZstlWgqN5HY8ZfzlhrxL7rQ5xDReNI8eYU37C7QmIvBNCWCHMBf7izH50viodMxtWUXAq7gj3Yk7hD5cEKXYEKzQS+6JH6ATHqKbeOy\\\\\\\/G\\\\\\\/DONq\\\\\\\/3EiEcNGv0fgr0sK8T4N3cMBmWe4u+8VhbV3B9eHySQMe6Sn2KcfzEWmTilHWTVUvkshAMZRC2rtJEOhzVO2U4Bqv\\\\\\\/odM+BSMRlc+8r\\\\\\\/6zA8vfg4gzq3zr5o+ilh4qpvdSzlm3ILAmIygr8xBDrelij5u1\\\\\\\/86BvC1yohuqSZorqoUx4oGNryPuRh0pyAU2lSocSqbs\\\\\\\/mgfSmqNGas2UwDubvjEojzhkDqszOfG1jDYaNo2wFS1kYmTDhTcBeYVfgCNX5O1yY7u3P7MM2tcnlv3B5Z+lerehULXxGsnRW1YNN4I717qsbAujvHZFPSKWI94ocYVZZOdZHTOsj+Gk7MZn6JJqW9b7htc+s8eTDwGbsK\\\\\\\/e8HADBjMuxF1w1LBrlDh74Br1tu\\\\\\\/P6J0TwjGAvhqyvP2X9oKp04yNRilbLVCjpZr29Xtgx4qjOOeapAmx72QQKE4w56yUZA+W1BjFdImhx7cCbB9LzKcOu5jScJGbOra5MWO9hTNp7YUAv1XgKByW+y1dzmZpoBUhn8FMLtuhg4I8gw\\\\\\\/RgRD1wpgZId4qIVAiz12Ntb811lfAVd4odP28zuCk\\\\\\\/xlKArB2YQJWzxecTAGaQT5+9IIlvcUqwOZcJ2ViwuG80qe0jt4mx5sZvUbBGSDi2\\\u0022,\\\u0022iv\\\u0022:\\\u0022+uke5lUk5\\\\\\\/2Fbwc739AGgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222f9ed5dde53cf2ea5278a672488d345e40282b2248362d397d20f102f915a6ec\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.26.010165v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1215887 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.26.010165v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zZQAIifxl9WarwImCZnwVh8Jv90UW5i66wxP4SxvkA4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215887" id="node1215887" data-pisa="biorxiv;2020.03.26.010165v1" data-pisa-master="biorxiv;2020.03.26.010165" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dorien</span> <span class="nlm-surname">De Vlieger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gretel M.</span> <span class="nlm-surname">Torres</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wander</span> <span class="nlm-surname">Van Breedam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kenny</span> <span class="nlm-surname">Roose</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Loes</span> <span class="nlm-surname">van Schie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-collab" data-hwp-id="collab-1">VIB-CMB COVID-19 Response Team</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Markus</span> <span class="nlm-surname">Hoffmann</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Pöhlmann</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-8112-0853" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Nico</span> <span class="nlm-surname">Callewaert</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Bert</span> <span class="nlm-surname">Schepens</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Saelens</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0003-3991-542X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.26.010165 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1215887" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Wrapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biosciences, The University of Texas at Austin</span>, Austin, Texas, USA 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BWrapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wrapp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BWrapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dorien De Vlieger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Biomedical Molecular Biology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorien%2BDe%2BVlieger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Vlieger%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorien%2BDe%2BVlieger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kizzmekia S. Corbett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health</span>, Bethesda, Maryland, USA 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kizzmekia%2BS.%2BCorbett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corbett%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKizzmekia%2BS.%2BCorbett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gretel M. Torres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Microbiology and Immunology, Geisel School of Medicine at Dartmouth</span>, Lebanon, New Hampshire, USA 03756</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gretel%2BM.%2BTorres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Torres%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGretel%2BM.%2BTorres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wander Van Breedam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wander%2BVan%2BBreedam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Breedam%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWander%2BVan%2BBreedam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kenny Roose </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenny%2BRoose%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roose%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenny%2BRoose%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Loes van Schie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Loes%2Bvan%2BSchie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Schie%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALoes%2Bvan%2BSchie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Markus Hoffmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Infection Biology Unit, German Primate Center – Leibniz Institute for Primate Research</span>, 37077 Göttingen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Markus%2BHoffmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffmann%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarkus%2BHoffmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Stefan Pöhlmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Infection Biology Unit, German Primate Center – Leibniz Institute for Primate Research</span>, 37077 Göttingen, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Faculty of Biology and Psychology, University Göttingen</span>, 37077 Göttingen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BP%C3%B6hlmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%B6hlmann%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BP%25C3%25B6hlmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Barney S. Graham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health</span>, Bethesda, Maryland, USA 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barney%2BS.%2BGraham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graham%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarney%2BS.%2BGraham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8112-0853" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barney S. Graham</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Nico Callewaert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nico%2BCallewaert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Callewaert%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANico%2BCallewaert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Bert Schepens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Biomedical Molecular Biology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bert%2BSchepens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schepens%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABert%2BSchepens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bert.schepens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">bert.schepens@vib-ugent.be</a>
<a href="mailto:xavier.saelens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">xavier.saelens@vib-ugent.be</a>
<a href="mailto:jmclellan@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">jmclellan@austin.utexas.edu</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Xavier Saelens </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>VIB Center for Medical Biotechnology</span>, VIB, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biochemistry and Microbiology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Biomedical Molecular Biology, Ghent University</span>, 9052 Ghent, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xavier%2BSaelens%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saelens%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXavier%2BSaelens%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bert.schepens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">bert.schepens@vib-ugent.be</a>
<a href="mailto:xavier.saelens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">xavier.saelens@vib-ugent.be</a>
<a href="mailto:jmclellan@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">jmclellan@austin.utexas.edu</a></span></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jason S. McLellan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biosciences, The University of Texas at Austin</span>, Austin, Texas, USA 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BS.%2BMcLellan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLellan%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BS.%2BMcLellan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3991-542X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason S. McLellan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bert.schepens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">bert.schepens@vib-ugent.be</a>
<a href="mailto:xavier.saelens@vib-ugent.be" class="" data-icon-position="" data-hide-link-title="0">xavier.saelens@vib-ugent.be</a>
<a href="mailto:jmclellan@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">jmclellan@austin.utexas.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.26.010165v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1215887" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1215887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010165v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1215887" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1215887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010165v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1215887" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1215887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010165v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1215887" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1215887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.26.010165v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1215887" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1215887/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">The pathogenic Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV-1) and COVID-19 coronavirus (SARS-CoV-2) have all emerged into the human population with devastating consequences. These viruses make use of a large envelope protein called spike (S) to engage host cell receptors and catalyze membrane fusion. Because of the vital role that these S proteins play, they represent a vulnerable target for the development of therapeutics to combat these highly pathogenic coronaviruses. Here, we describe the isolation and characterization of single-domain antibodies (VHHs) from a llama immunized with prefusion-stabilized coronavirus spikes. These VHHs are capable of potently neutralizing MERS-CoV or SARS-CoV-1 S pseudotyped viruses. The crystal structures of these VHHs bound to their respective viral targets reveal two distinct epitopes, but both VHHs block receptor binding. We also show cross-reactivity between the SARS-CoV-1 S-directed VHH and SARS-CoV-2 S, and demonstrate that this cross-reactive VHH is capable of neutralizing SARS-CoV-2 S pseudotyped viruses as a bivalent human IgG Fc-fusion. These data provide a molecular basis for the neutralization of pathogenic betacoronaviruses by VHHs and suggest that these molecules may serve as useful therapeutics during coronavirus outbreaks.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-4">Coronaviruses are enveloped, positive-sense RNA viruses that are divided into four genera (α, β, γ, δ) and infect a wide variety of host organisms (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Woo et al., 2009</a>). There are at least seven coronaviruses that can cause disease in humans, and four of these viruses (HCoV-HKU1, HCoV-OC43, HCoV-NL63 and HCoV-229E) circulate seasonally throughout the global population, causing mild respiratory disease in most patients (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gaunt et al., 2010</a>). The three remaining viruses, SARS-CoV-1, MERS-CoV and SARS-CoV-2, are zoonotic pathogens that have caused epidemics or pandemics with severe and often fatal symptoms after emerging into the human population (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chan et al., 2020</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang et al., 2020</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ksiazek et al., 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lu et al., 2020</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zaki et al., 2012</a>). For these highly pathogenic betacoronaviruses, prophylactics and therapeutic treatments are needed.</p><p id="p-5">The surfaces of coronaviruses are decorated with a spike glycoprotein (S), a large class I fusion protein (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bosch et al., 2003</a>). The S protein forms a trimeric complex that can be functionally categorized into two distinct subunits, S1 and S2, that are separated by a protease cleavage site. The S1 subunit contains the receptor-binding domain (RBD), which interacts with a proteinaceous host-cell receptor to trigger membrane fusion. The S2 subunit contains the fusion machinery, including the hydrophobic fusion peptide and the α-helical heptad repeats. The functional host cell receptors for SARS-CoV-1 and MERS-CoV are angiotensin converting enzyme 2 (ACE2) and dipeptidyl peptidase 4 (DPP4), respectively (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li et al., 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Raj et al., 2013</a>). The interactions between these receptors and their respective RBDs have been thoroughly characterized, both structurally and biophysically (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li et al., 2005</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang et al., 2013</a>). Recently, it has been reported that SARS-CoV-2 S also makes use of ACE2 as a functional host-cell receptor and several structures of this complex have already been reported (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hoffmann, 2020</a>; Lan, 2020; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wan et al., 2020</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yan, 2020</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhou et al., 2020</a>).</p><p id="p-6">Recent advances in cryo-EM have allowed researchers to determine high-resolution structures of the trimeric spike protein ectodomains and understand how S functions as a macromolecular machine (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kirchdoerfer et al., 2016</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li et al., 2005</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Walls et al., 2016</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wang et al., 2013</a>). Initial cryo-EM characterization of the SARS-CoV-1 spike revealed that the RBDs adopted at least two distinct conformations. In the “up” conformation, the RBDs could be observed jutting out away from the rest of S, such that they could easily engage ACE2 without causing any steric clashes. In the “down” conformation, the RBDs were tightly packed against the top of the S2 subunit, preventing binding by ACE2 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gui et al., 2017</a>). Subsequent experiments have corroborated this phenomenon and similar dynamics have been observed in MERS-CoV S, SARS-CoV-2 S and in alphacoronavirus S proteins (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones et al., 2019</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kirchdoerfer et al., 2018</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pallesen et al., 2017</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Walls, 2020</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wrapp et al., 2020</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yuan et al., 2017</a>). Due to the relatively low abundance of particles that can be observed by cryo-EM with three RBDs in the up conformation, it is thought that this conformation may correspond to an energetically unstable state (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kirchdoerfer et al., 2018</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pallesen et al., 2017</a>). These observations led to the hypothesis that the CoV RBDs may be acting as molecular ratchets, wherein a receptor-binding event would trap the RBD in the less stable up conformation, leading to gradual destabilization until S is finally triggered to initiate membrane fusion. Recent experiments characterizing RBD-directed anti-SARS-CoV-1 antibodies that trap the SARS-CoV-1 RBD in the up conformation and lead to destabilization of the prefusion spike have lent support to this hypothesis (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Walls et al., 2019</a>).</p><p id="p-7">Numerous anti-SARS-CoV-1 RBD and anti-MERS-CoV RBD antibodies have been reported and their mechanisms of neutralization can be attributed to the occlusion of the receptor-binding site and to trapping the RBD in the unstable up conformation, effectively acting as a receptor mimic that triggers a premature transition from the prefusion-to-postfusion conformation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hwang et al., 2006</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Walls et al., 2019</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang et al., 2018</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang et al., 2015</a>). Heavy chain-only antibodies (HCAbs), present in camelids, contain a single variable domain (VHH) instead of two variable domains (VH and VL) that make up the equivalent antigen-binding fragment (Fab) of conventional IgG antibodies (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hamers-Casterman et al., 1993</a>). This single variable domain, in the absence of an effector domain, is referred to as a single-domain antibody, VHH or Nanobody® and typically can acquire affinities and specificities for antigens comparable to conventional antibodies. VHHs can easily be constructed into multivalent formats and are known to have enhanced thermal stability and chemostability compared to most antibodies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Vlieger et al., 2018</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dumoulin et al., 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Govaert et al., 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Laursen et al., 2018</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">van der Linden et al., 1999</a>). Their advantageous biophysical properties have led to the evaluation of several VHHs as therapeutics against common respiratory pathogens, such as respiratory syncytial virus (RSV) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Detalle et al., 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rossey et al., 2017</a>). The use of VHHs as biologics in the context of a respiratory infection is a particularly attractive application, since the highly stable VHHs can be nebulized and administered via an inhaler directly to the site of infection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Respaud et al., 2015</a>). Moreover, due to their stability after prolonged storage, VHHs could be stockpiled as therapeutic treatment options in case of an epidemic. Although therapeutics against MERS-CoV and SARS-CoV-2 are sorely needed, the feasibility of using VHHs for this purpose has not yet been adequately explored. Several MERS-CoV S-directed VHHs have been reported, but their epitopes remain largely undefined, other than being classified as RBD-directed (Stalin <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Raj et al., 2018</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhao et al., 2018</a>).</p><p id="p-8">Here we report the isolation of two potently neutralizing VHHs directed against the SARS-CoV-1 and MERS-CoV RBDs. These VHHs were elicited in response to immunization of a llama with prefusion-stabilized SARS-CoV-1 and MERS-CoV S proteins. We solved the crystal structures of these two VHHs in complex with their respective viral epitopes and determined that their mechanisms of neutralization were occlusion of the receptor binding interface and trapping of the RBDs in the up conformation. We also show that the SARS-CoV-1 RBD-directed VHH exhibits cross-reactivity against the SARS-CoV-2 RBD and is capable of blocking the receptor-binding interface. After engineering this VHH into a bivalent Fc-fusion, we show that this cross-reactive VHH is also capable of potently neutralizing SARS-CoV-2 S pseudoviruses. In addition, we demonstrate that the VHH-Fc fusion can be produced at high yields in an industry-standard CHO cell system, suggesting that it merits further investigation as a potential therapeutic for the ongoing COVID-19 pandemic.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS</h2><div id="sec-3" class="subsection"><h3><strong>Isolation of betacoronavirus S-directed VHHs</strong></h3><p id="p-9">Our initial aim was to isolate VHHs that could potently neutralize MERS-CoV and SARS-CoV-1. Therefore, a llama was sequentially immunized subcutaneously twice with SARS-CoV-1 S protein, twice with MERS-CoV S protein, once again with SARS-CoV-1 S and finally with both SARS-CoV-1 and MERS-CoV S protein (<strong>S. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a></strong>). To obtain VHHs directed against these spike proteins, two consecutive rounds of panning were performed using either SARS-CoV-1 S or MERS-CoV S protein. Positive clones were sequenced and multiple sequence alignment and phylogenetic analysis using the neighbor-joining method revealed that seven unique MERS-CoV S and five unique SARS-CoV-1 S VHHs were isolated (<strong>S. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a></strong>). These VHHs and an irrelevant control (RSV F-VHH, directed against the F protein of human respiratory syncytial virus) were subsequently expressed in <em>Pichia pastoris</em> and purified from the yeast medium (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Rossey et al., 2017</a>). The binding of the purified VHHs to prefusion-stabilized MERS-CoV S and SARS-CoV-1 S was confirmed by ELISA (<strong>S. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a></strong>). Four clones (MERS VHH-55, −12, −34 and −40), obtained after panning on MERS-CoV S protein, bound with high affinity to prefusion stabilized MERS-CoV S, whereas the affinity of VHH-2, −20 and −15 was 100- to 1000-fold lower. Of the five clones isolated after panning on SARS-CoV-1 S protein, three VHH clones (SARS VHH-72, −1 and −6) interacted strongly with prefusion stabilized SARS-CoV-1 S protein.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Epitope determination and biophysical characterization of MERS VHH-55 and SARS VHH-72. A) Reactivity of MERS-CoV and SARS-CoV RBD-directed VHHs against the MERS-CoV and SARS-CoV-1 RBD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control. B) SPR sensorgrams showing binding between the MERS-CoV RBD and MERS VHH-55 (left) and SARS-CoV-1 RBD and SARS VHH-72 (right). Binding curves are colored black and fit of the data to a 1:1 binding model is colored red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Epitope determination and biophysical characterization of MERS VHH-55 and SARS VHH-72.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) Reactivity of MERS-CoV and SARS-CoV RBD-directed VHHs against the MERS-CoV and SARS-CoV-1 RBD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control. &lt;strong&gt;B&lt;/strong&gt;) SPR sensorgrams showing binding between the MERS-CoV RBD and MERS VHH-55 (&lt;em&gt;left&lt;/em&gt;) and SARS-CoV-1 RBD and SARS VHH-72 (&lt;em&gt;right&lt;/em&gt;). Binding curves are colored black and fit of the data to a 1:1 binding model is colored red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Epitope determination and biophysical characterization of MERS VHH-55 and SARS VHH-72.</span><p id="p-10" class="first-child"><strong>A</strong>) Reactivity of MERS-CoV and SARS-CoV RBD-directed VHHs against the MERS-CoV and SARS-CoV-1 RBD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control. <strong>B</strong>) SPR sensorgrams showing binding between the MERS-CoV RBD and MERS VHH-55 (<em>left</em>) and SARS-CoV-1 RBD and SARS VHH-72 (<em>right</em>). Binding curves are colored black and fit of the data to a 1:1 binding model is colored red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>VHHs neutralize coronavirus S pseudotyped viruses</h3><p id="p-11">To assess the antiviral activity of the MERS-CoV and SARS-CoV S-directed VHHs, <em>in vitro</em> neutralization assays using MERS-CoV England1 S and SARS-CoV-1 Urbani S pseudotyped lentiviruses were performed. The high affinity MERS VHH-55, −12, −34 and −40 neutralized MERS-CoV S pseudotyped virus with IC<sub>50</sub> values ranging from 0.014 to 2.9 µg/mL (0.9 nM to 193.3 nM), while no inhibition was observed for the lower affinity MERS-CoV or SARS-CoV-1 specific VHHs (<strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). SARS VHH-72 and −44 were able to neutralize lentiviruses pseudotyped with SARS-CoV-1 S with an IC<sub>50</sub> value of 0.14 (9 nM) and 5.5 µg/mL (355 nM), respectively. No binding was observed for SARS VHH-44 to prefusion stabilized SARS-CoV-1 S protein in the ELISA assay. Sequence analysis revealed that the neutralizing MERS-CoV specific VHHs −12, −40 and −55 have highly similar complementarity-determining regions (CDRs), indicating that they likely bind to the same epitope (<strong>S. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>). In contrast, the CDRs from the SARS-CoV S-specific VHHs −44 and −72 are highly divergent.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD. A) MERS VHH-55 is shown as blue ribbons and the MERS-CoV RBD is shown as a tan-colored molecular surface. The DPP4 binding interface on the MERS-CoV RBD is colored red. B) The structure of DPP4 bound to the MERS-CoV RBD (PDB: 4L72) is aligned to the crystal structure of MERS VHH-55 bound to the MERS-CoV RBD. A single monomer of DPP4 is shown as a red, transparent molecular surface. C) A zoomed-in view of the panel from 2A, with the MERS-CoV RBD now displayed as tan-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots. D) The same view from 2C has been turned by approximately 90&#xB0; to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) MERS VHH-55 is shown as blue ribbons and the MERS-CoV RBD is shown as a tan-colored molecular surface. The DPP4 binding interface on the MERS-CoV RBD is colored red. &lt;strong&gt;B&lt;/strong&gt;) The structure of DPP4 bound to the MERS-CoV RBD (PDB: 4L72) is aligned to the crystal structure of MERS VHH-55 bound to the MERS-CoV RBD. A single monomer of DPP4 is shown as a red, transparent molecular surface. &lt;strong&gt;C&lt;/strong&gt;) A zoomed-in view of the panel from &lt;strong&gt;2A&lt;/strong&gt;, with the MERS-CoV RBD now displayed as tan-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots. &lt;strong&gt;D&lt;/strong&gt;) The same view from &lt;strong&gt;2C&lt;/strong&gt; has been turned by approximately 90&#xB0; to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F2.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F2.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD.</span><p id="p-12" class="first-child"><strong>A</strong>) MERS VHH-55 is shown as blue ribbons and the MERS-CoV RBD is shown as a tan-colored molecular surface. The DPP4 binding interface on the MERS-CoV RBD is colored red. <strong>B</strong>) The structure of DPP4 bound to the MERS-CoV RBD (PDB: 4L72) is aligned to the crystal structure of MERS VHH-55 bound to the MERS-CoV RBD. A single monomer of DPP4 is shown as a red, transparent molecular surface. <strong>C</strong>) A zoomed-in view of the panel from <strong>2A</strong>, with the MERS-CoV RBD now displayed as tan-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots. <strong>D</strong>) The same view from <strong>2C</strong> has been turned by approximately 90° to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen-bonds and salt bridges between MERS VHH-55 and the MERS-CoV RBD are shown as black dots.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1222597/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1222597/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1222597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-13" class="first-child">MERS-CoV and SARS-CoV-1 pseudovirus neutralization data.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Epitope mapping of betacoronavirus S-directed VHHs</h3><p id="p-14">To map the epitopes targeted by the neutralizing VHHs, we tested binding to recombinant MERS-CoV S1, RBD, NTD and SARS-CoV-1 RBD and NTD by ELISA (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A and S</a></strong>. <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>).The MERS-CoV S-specific VHHs strongly bound to MERS-CoV S1 and RBD in a concentration-dependent manner, and no binding to the MERS-CoV NTD was observed. Similarly, strong binding of SARS VHH-72 to the SARS-CoV-1 RBD protein but not the SARS-CoV-1 NTD protein was observed. Again, no binding of SARS VHH-44 to either the SARS-CoV-1 RBD or NTD protein was detected. These data demonstrate that the neutralizing VHHs SARS VHH-72 and MERS VHH-55 target the RBDs. Based on the specificity and potent neutralizing capacity of SARS VHH-72 and MERS VHH-55, we measured the affinities of these VHHs by immobilizing recombinantly expressed VHH to an SPR sensorchip and determined the binding kinetics for their respective RBDs. We found that both of these VHHs bound to their targets with high affinity. SARS VHH-72 bound to its target with an affinity of 1.2 nM and MERS VHH-55 bound to its target with an affinity of 79.2 pM, in part due to a very slow off-rate constant (<em>k</em><sub>d</sub> = 8.2 x10<sup>-5</sup> s<sup>-1</sup>) (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD. A) SARS VHH-72 is shown as dark blue ribbons and the SARS-CoV-1 RBD is shown as a pink-colored molecular surface. The ACE2 binding interface on the SARS-CoV-1 RBD is colored red. B) The structure of ACE2 bound to the SARS-CoV-1 RBD (PDB: 2AJF) is aligned to the crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD. ACE2 is shown as a red, transparent molecular surface. C) A simulated N-linked glycan containing an energy-minimized trimannosyl core (derived from PDB ID: 1HD4) is modeled as red sticks, coming from Asn322 in ACE2. ACE2 is shown as a red molecular surface, the SARS-CoV-1 RBD is shown as pink ribbons and SARS VHH-72 is shown as a dark blue, transparent molecular surface. D) A zoomed-in view of the panel from 3A is shown, with the SARS-CoV-1 RBD now displayed as pink-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen bonds and salt bridges between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots. E) The same view from 3D has been turned by 60&#xB0; to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Interactions between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) SARS VHH-72 is shown as dark blue ribbons and the SARS-CoV-1 RBD is shown as a pink-colored molecular surface. The ACE2 binding interface on the SARS-CoV-1 RBD is colored red. &lt;strong&gt;B&lt;/strong&gt;) The structure of ACE2 bound to the SARS-CoV-1 RBD (PDB: 2AJF) is aligned to the crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD. ACE2 is shown as a red, transparent molecular surface. &lt;strong&gt;C&lt;/strong&gt;) A simulated &lt;em&gt;N&lt;/em&gt;-linked glycan containing an energy-minimized trimannosyl core (derived from PDB ID: 1HD4) is modeled as red sticks, coming from Asn322 in ACE2. ACE2 is shown as a red molecular surface, the SARS-CoV-1 RBD is shown as pink ribbons and SARS VHH-72 is shown as a dark blue, transparent molecular surface. &lt;strong&gt;D&lt;/strong&gt;) A zoomed-in view of the panel from &lt;strong&gt;3A&lt;/strong&gt; is shown, with the SARS-CoV-1 RBD now displayed as pink-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen bonds and salt bridges between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots. &lt;strong&gt;E&lt;/strong&gt;) The same view from &lt;strong&gt;3D&lt;/strong&gt; has been turned by 60&#xB0; to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Interactions between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F3.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F3.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD.</span><p id="p-15" class="first-child"><strong>A</strong>) SARS VHH-72 is shown as dark blue ribbons and the SARS-CoV-1 RBD is shown as a pink-colored molecular surface. The ACE2 binding interface on the SARS-CoV-1 RBD is colored red. <strong>B</strong>) The structure of ACE2 bound to the SARS-CoV-1 RBD (PDB: 2AJF) is aligned to the crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD. ACE2 is shown as a red, transparent molecular surface. <strong>C</strong>) A simulated <em>N</em>-linked glycan containing an energy-minimized trimannosyl core (derived from PDB ID: 1HD4) is modeled as red sticks, coming from Asn322 in ACE2. ACE2 is shown as a red molecular surface, the SARS-CoV-1 RBD is shown as pink ribbons and SARS VHH-72 is shown as a dark blue, transparent molecular surface. <strong>D</strong>) A zoomed-in view of the panel from <strong>3A</strong> is shown, with the SARS-CoV-1 RBD now displayed as pink-colored ribbons. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Hydrogen bonds and salt bridges between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots. <strong>E</strong>) The same view from <strong>3D</strong> has been turned by 60° to show additional contacts. Residues that form interactions are shown as sticks, with nitrogen atoms colored dark blue and oxygen atoms colored red. Interactions between SARS VHH-72 and the SARS-CoV-1 RBD are shown as black dots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Structural basis of VHH interaction with RBDs</h3><p id="p-16">To investigate the molecular determinants that mediate potent neutralization and high-affinity binding by MERS VHH-55, we solved the crystal structure of MERS VHH-55 bound to the MERS-CoV RBD. Crystals grew in space group <em>C</em>222<sub>1</sub> and diffracted X-rays to a resolution of 3.4 Å. After determining a molecular replacement solution and iterative building and refinement, our structure reached an R<sub>work</sub>/R<sub>free</sub> of 21.4%/26.8% (<strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>). The asymmetric unit of this crystal contained eight copies of the MERS VHH-55 + MERS-CoV RBD complex and had a solvent content of ∼58%. The electron density allowed unambiguous definition of the interface between the RBD and VHH, with the three CDRs forming extensive binding contacts with the RBD, burying 716 Å<sup>2</sup> of surface area by pinching the RBD between the CDR2 and CDR3. The CDR3 of MERS VHH-55 is looped over the DPP4-binding interface, occluding DPP4 from productively engaging the MERS-CoV RBD (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A-B</a></strong>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1222595/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1222595/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1222595" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-17" class="first-child">X-ray data collection and refinement statistics</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">There are numerous contacts between the CDRs of MERS VHH-55 and the MERS-CoV RBD, and the majority of these are confined to CDRs 2 and 3 (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C-D</a></strong>). The sole interaction from the MERS VHH-55 CDR1 comes from Asp35 forming a salt bridge with Arg542 from the RBD. CDR2 forms hydrogen bonds using Ser53 and Asp61 to engage RBD residues Asp539 and Gln544. Furthermore, Asn58 from the CDR2 also engages Arg542 from the RBD. Trp99 from the MERS VHH-55 CDR3 forms a salt bridge with Glu513 via the nitrogen from its pyrrole ring. Finally, Glu95 from the MERS VHH-55 CDR3 also forms a salt bridge with Arg542 from the MERS-CoV RBD, suggesting that Arg542 plays a critical role in MERS VHH-55 binding since it is productively engaged by residues from all three CDRs. This arginine has also been implicated in binding to the MERS-CoV receptor DPP4, and has previously been identified as one of the twelve highly conserved amino acids that is crucial for high-affinity receptor engagement (<strong>S. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a></strong>) (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang et al., 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang et al., 2014</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS VHH-72 is cross-reactive against SARS-CoV-2. A) An SPR sensorgram measuring the binding of SARS VHH-72 to the SARS-CoV-2 RBD-SD1. Binding curves are colored black and fit of the data to a 1:1 binding model is colored red. B) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and SARS-CoV-2 are colored green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SARS VHH-72 is cross-reactive against SARS-CoV-2.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) An SPR sensorgram measuring the binding of SARS VHH-72 to the SARS-CoV-2 RBD-SD1. Binding curves are colored black and fit of the data to a 1:1 binding model is colored red. &lt;strong&gt;B&lt;/strong&gt;) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and SARS-CoV-2 are colored green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F4.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F4.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">SARS VHH-72 is cross-reactive against SARS-CoV-2.</span><p id="p-19" class="first-child"><strong>A</strong>) An SPR sensorgram measuring the binding of SARS VHH-72 to the SARS-CoV-2 RBD-SD1. Binding curves are colored black and fit of the data to a 1:1 binding model is colored red. <strong>B</strong>) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and SARS-CoV-2 are colored green.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">In addition to forming a salt bridge with Glu513 from the MERS-CoV RBD, Trp99 of the MERS VHH-55 CDR3 is positioned near a hydrophobic patch formed by Phe506 (<strong>S. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a></strong>). This amino acid exhibits natural sequence variation in several MERS-CoV strains, such that a Leu is occasionally observed at this position. To evaluate the extent to which this substitution may impact MERS VHH-55 binding, we generated a F506L substitution and measured binding by SPR (<strong>S. <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a></strong>). Surprisingly, this substitution resulted in a ∼200-fold reduction in MERS VHH-55 binding affinity. Despite this substantial reduction, the affinity of MERS VHH-55 to MERS-CoV RBD F506L remains high, with a <em>K</em><sub>D</sub> = 16.5 nM. Other than the variability that is observed at position 506 of the MERS-CoV RBD, the rest of the MERS VHH-55 epitope is highly conserved across the 863 strains that are curated in the MERS-CoV Virus Variation database (<strong>S. <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a></strong>). This high degree of epitope conservation suggests that VHH-55 would broadly recognize MERS-CoV strains.</p><p id="p-21">We also sought to discover the molecular determinants of binding between SARS VHH-72 and the SARS-CoV-1 RBD by determining the crystal structure of this complex. Crystals grew in space group <em>P</em>3<sub>1</sub>21 and diffracted X-rays to a resolution of 2.2 Å. We obtained a molecular replacement solution and refined the structure to an R<sub>work</sub>/R<sub>free</sub> of 20.3%/23.6% through iterative building and refinement (<strong><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a></strong>). Our structure reveals that CDRs 2 and 3 contribute to the majority of the 834 Å<sup>2</sup> of buried surface area at the binding interface (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a></strong>). This interface does not, however, overlap with the ACE2 binding interface. Rather, ACE2 would clash with the CDR-distal framework of SARS VHH-72 (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a></strong>). This clash would only be enhanced by the presence of an <em>N</em>-linked glycan at Asn322 of ACE2, which is already located within the VHH framework when the receptor-bound RBD is aligned to the VHH-bound RBD (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a></strong>).</p><p id="p-22">SARS VHH-72 binds to the SARS-CoV-1 RBD by forming an extensive hydrogen-bond network via its CDRs 2 and 3 (<strong>Figure3D-E</strong>). Ser56 from the CDR2 simultaneously forms hydrogen bonds with the peptide backbone of three residues from the SARS-CoV-1 RBD: Leu355, Tyr356 and Ser358. The peptide backbone of Ser358 also forms a hydrogen bond with the backbone of neighboring Thr57 from the CDR2. A salt bridge formed between CDR2 residue Asp61 and RBD residue Arg426 tethers the C-terminal end of the CDR2 to the RBD. The N-terminus of the CDR3 forms a short β-strand that pairs with a β-strand from the SARS-CoV-1 RBD to bridge the interface between these two molecules. This interaction is mediated by backbone hydrogen bonds from CDR3 residues Gly98, Val100 and Val100a to RBD residues Cys366 and Phe364. Glu100c from the CDR3 forms hydrogen bonds with the sidechain hydroxyls from both Ser362 and Tyr494 from the SARS-CoV-1 RBD. The neighboring CDR3 residue also engages in a sidechain-specific interaction by forming a salt bridge between the pyrrole nitrogen of Trp100d and the hydroxyl group from RBD residue Thr363. Asp101 is involved in the most C-terminal interaction from the CDR3 by forming a salt bridge with RBD residue Lys365. The extensive interactions formed between CDRs 2 and 3 of SARS VHH-72 and the SARS-CoV-1 RBD explain the high-affinity binding that we observed between these molecules.</p></div><div id="sec-7" class="subsection"><h3>SARS VHH-72 is cross-reactive against WIV1-CoV and SARS-CoV-2</h3><p id="p-23">Analysis of 10 available SARS-CoV-1 strain sequences revealed a high degree of conservation in the residues that make up the SARS VHH-72 epitope, prompting us to explore the breadth of SARS VHH-72 binding (<strong>S. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a></strong>). WIV1-CoV is a betacoronavirus found in bats that is closely related to SARS-CoV-1 and also utilizes ACE2 as a host-cell receptor (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ge et al., 2013</a>). Due to the relatively high degree of sequence conservation between SARS-CoV and WIV1-CoV, we expressed the WIV1-CoV RBD and measured binding to SARS VHH-72 by SPR (<strong>S. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a></strong>). SARS VHH-72 also exhibits high-affinity binding to the WIV1-CoV RBD (7.4 nM), demonstrating that it is cross-reactive between these two closely related coronaviruses (<strong>S. <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a></strong>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neutralizing mechanisms of MERS VHH-55 and SARS VHH-72. A) The MERS-CoV spike (PDB ID: 5W9H) is shown as a molecular surface, with each monomer colored either white, gray or tan. The tan and white monomers are bound by MERS VHH-55, shown as blue ribbons. The clash between MERS VHH-55 bound to the white monomer and the neighboring tan RBD is highlighted by the red ellipse. B) The SARS-CoV-1 spike (PDB ID: 5X58) is shown as a molecular surface, with each protomer colored either white, gray or pink. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. C) The SARS-CoV-2 spike (PDB ID: 6VXX) is shown as a molecular surface, with each protomer colored either white, gray or green. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. The SARS-CoV-2 trimer appears smaller than SARS-CoV-1 S due to the absence of flexible NTD-distal loops which could not be built during cryo-EM analysis. D) CoV VHHs prevent MERS-CoV RBD, SARS-CoV-1 RBD and SARS-CoV-2 RBD-SD1 from interacting with their receptors. The results of the BLI-based receptor-blocking experiment are shown. The legend lists the immobilized RBDs and the VHHs or receptors that correspond to each curve." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Neutralizing mechanisms of MERS VHH-55 and SARS VHH-72.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) The MERS-CoV spike (PDB ID: 5W9H) is shown as a molecular surface, with each monomer colored either white, gray or tan. The tan and white monomers are bound by MERS VHH-55, shown as blue ribbons. The clash between MERS VHH-55 bound to the white monomer and the neighboring tan RBD is highlighted by the red ellipse. &lt;strong&gt;B&lt;/strong&gt;) The SARS-CoV-1 spike (PDB ID: 5X58) is shown as a molecular surface, with each protomer colored either white, gray or pink. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. &lt;strong&gt;C&lt;/strong&gt;) The SARS-CoV-2 spike (PDB ID: 6VXX) is shown as a molecular surface, with each protomer colored either white, gray or green. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. The SARS-CoV-2 trimer appears smaller than SARS-CoV-1 S due to the absence of flexible NTD-distal loops which could not be built during cryo-EM analysis. &lt;strong&gt;D)&lt;/strong&gt; CoV VHHs prevent MERS-CoV RBD, SARS-CoV-1 RBD and SARS-CoV-2 RBD-SD1 from interacting with their receptors. The results of the BLI-based receptor-blocking experiment are shown. The legend lists the immobilized RBDs and the VHHs or receptors that correspond to each curve.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F5.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F5.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Neutralizing mechanisms of MERS VHH-55 and SARS VHH-72.</span><p id="p-24" class="first-child"><strong>A</strong>) The MERS-CoV spike (PDB ID: 5W9H) is shown as a molecular surface, with each monomer colored either white, gray or tan. The tan and white monomers are bound by MERS VHH-55, shown as blue ribbons. The clash between MERS VHH-55 bound to the white monomer and the neighboring tan RBD is highlighted by the red ellipse. <strong>B</strong>) The SARS-CoV-1 spike (PDB ID: 5X58) is shown as a molecular surface, with each protomer colored either white, gray or pink. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. <strong>C</strong>) The SARS-CoV-2 spike (PDB ID: 6VXX) is shown as a molecular surface, with each protomer colored either white, gray or green. Every monomer is bound by a copy of SARS VHH-72, shown as dark blue ribbons. The clashes between copies of SARS VHH-72 and the two neighboring spike monomers are highlighted by the red circle. The SARS-CoV-2 trimer appears smaller than SARS-CoV-1 S due to the absence of flexible NTD-distal loops which could not be built during cryo-EM analysis. <strong>D)</strong> CoV VHHs prevent MERS-CoV RBD, SARS-CoV-1 RBD and SARS-CoV-2 RBD-SD1 from interacting with their receptors. The results of the BLI-based receptor-blocking experiment are shown. The legend lists the immobilized RBDs and the VHHs or receptors that correspond to each curve.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Based on the high degree of structural homology that has been reported between SARS-CoV-1 S and SARS-CoV-2 S (Walls, 2020; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Wrapp et al., 2020</a>), we also tested SARS VHH-72 for cross-reactivity against the SARS-CoV-2 RBD-SD1 by SPR (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a></strong>). The binding affinity of SARS VHH-72 for the SARS-CoV-2 RBD-SD1 was ∼39 nM. This diminished binding affinity, compared to the binding of SARS VHH-72 to SARS-CoV-1 RBD, can primarily be attributed to an increase in the dissociation rate constant of this interaction (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a></strong>). The only variant residue on the SARS-CoV-1 RBD that makes direct contact with SARS VHH-72 is Arg426, which is Asn439 in the SARS-CoV-2 RBD (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a></strong>).</p></div><div id="sec-8" class="subsection"><h3>VHHs disrupt RBD dynamics and receptor-binding</h3><p id="p-26">As stated previously, the RBDs of MERS-CoV S, SARS-CoV-1 S and SARS-CoV-2 S undergo dynamic conformational rearrangements that alternately mask and present their receptor-binding interfaces and potential neutralizing epitopes to host molecules. By aligning the crystal structures of MERS VHH-55 and SARS VHH-72 bound to their respective targets to the full-length, cryo-EM structures of the MERS-CoV, SARS-CoV-1 and SARS-CoV-2 spike proteins, we can begin to understand how these molecules might function in the context of these dynamic rearrangements. When the MERS-CoV RBDs are all in the down conformation or all in the up conformation, MERS VHH-55 would be able to bind all three of the protomers making up the functional spike trimer without forming any clashes. However, if a down protomer was bound by MERS VHH-55 and the neighboring protomer sampled the up conformation, this RBD would then be trapped in this state by the presence of the neighboring MERS VHH-55 molecule (<strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a></strong>). This conformational trapping would be even more pronounced upon SARS VHH-72 binding to the full-length SARS-CoV-1 S protein or the full-length SARS-CoV-2 S protein.</p><p id="p-27">Due to the binding angle of SARS VHH-72, when a bound SARS-CoV-1 or SARS-CoV-2 RBD samples the down conformation, it would form dramatic clashes with the S2 fusion subunit, regardless of the conformations of the neighboring RBDs (<strong><a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B-C</a></strong>). Therefore, once a single SARS VHH-72 binding event took place, the bound protomer would be trapped in the up conformation until either SARS VHH-72 was released or until the S protein was triggered to undergo the prefusion-to-postfusion transition. Based on the binding angles of MERS VHH-55 and SARS VHH-72, we can conclude that these molecules would likely disrupt the RBD dynamics in the context of a full-length S protein by trapping the up conformation. Because this up conformation is unstable and leads to S protein triggering, it is possible that this conformational trapping may at least partially contribute to the neutralization mechanisms of these VHHs.</p><p id="p-28">Based on our structural analysis, we hypothesized that another mechanism by which both MERS VHH-55 and SARS VHH-72 neutralize their respective viral targets is by blocking the interaction between the RBDs and their host-cell receptors. To test this hypothesis, we performed a BLI-based assay in which the SARS-CoV-1, SARS-CoV-2 and MERS-CoV RBDs were immobilized to biosensor tips, dipped into VHHs and then dipped into wells containing the recombinant, soluble host cell receptors. We found that when tips coated in the MERS-CoV RBD were dipped into MERS VHH55 before being dipped into DPP4, there was no increase in response that could be attributed to receptor binding. When tips coated with the MERS-CoV RBD were dipped into SARS VHH-72 and then DPP4, a robust response signal was observed, as expected. Similar results were observed when the analogous experiments were performed using the SARS-CoV-1 or SARS-CoV-2 RBDs, SARS VHH-72 and ACE2 (<strong><a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a></strong>). These results support our structural analysis that both MERS VHH-55 and SARS VHH-72 are capable of neutralizing their respective viral targets by directly preventing host-cell receptor binding.</p></div><div id="sec-9" class="subsection"><h3>Bivalent SARS VHH-72 is capable of potently neutralizing SARS-CoV-2 S pseudoviruses</h3><p id="p-29">Despite the relatively high-affinity binding that was observed by SPR between SARS VHH-72 and the SARS-CoV-2 RBD, this binding could not be detected by ELISA nor was SARS VHH-72 capable of neutralizing SARS-CoV-2 S VSV pseudoviruses, possibly due to the high off-rate constant, whereas SARS-CoV-1 pseudotypes were readily neutralized (<strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A-D</a></strong>). In an attempt to overcome this rapid dissociation, we engineered two bivalent variants of SARS VHH-72. These included a tail-to-head fusion of two SARS VHH-72 molecules connected by a (GGGGS)<sub>3</sub> linker (VHH-72-VHH-72) and a genetic fusion of SARS VHH-72 to the Fc domain of human IgG1 (VHH-72-Fc) (<strong>S. <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A-C</a></strong>). These bivalent SARS VHH-72 constructs were able to bind to both prefusion SARS-CoV-1 S and SARS-CoV-2 RBD-SD1 as demonstrated by ELISA and by a dose-dependent reduction in the binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells (<strong><a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C-D</a> and S. <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B-C</a></strong>). We could also detect binding of both of these constructs to full length SARS-CoV-1 S and SARS-CoV-2 S expressed on the surface of mammalian cells (<strong>S. <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D-E</a></strong>). Supernatants of HEK 293S cells transiently transfected with VHH-72-Fc exhibited neutralizing activity against both SARS-CoV-1 and SARS-CoV-2 S VSV pseudoviruses in the same assay which showed no such cross-reactive neutralization for monovalent SARS VHH-72 (<strong><a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6E-F</a></strong>). A BLI experiment measuring binding of VHH-72-Fc to immobilized SARS-CoV-2 RBD-SD1 further confirmed that bivalency was able to compensate for the high off-rate constant of the monomer (<strong><a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a></strong>). Furthermore, cross-neutralizing VHH-72-Fc construct reached expression levels of ∼300 mg/L in ExpiCHO cells (<strong><a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a></strong>). Using VHH-72-Fc purified from ExpiCHO cells and a SARS-CoV-2 S pseudotyped VSV with a luciferase reporter, we evaluated the neutralization capacity of VHH-72-Fc and found that it was able to neutralize pseudovirus with an IC<sub>50</sub> of approximately 0.2 μg/mL (<strong><a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a></strong>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bivalency overcomes the high off-rate constant of SARS VHH-72. A) SARS-CoV-1 S and B) SARS-CoV-2 S VSV pseudoviruses were used to evaluate the neutralization capacity of SARS VHH-72. MERS VHH-55 and PBS were included as negative controls. Luciferase activity is reported in counts per second (c.p.s.). NI cells were not infected. C) Binding of bivalent VHHs was tested by ELISA against SARS-CoV-1 S and D) SARS-CoV-2 RBD-SD1. VHH-72-Fc refers to SARS VHH-72 fused to a human IgG1 Fc domain by a GS(GGGGS)2 linker. VHH-72-Fc (S) is the same Fc fusion with a GS, rather than a GS(GGGGS)2, linker. GBP is an irrelevant GFP-binding protein. VHH-72-VHH-72 refers to the tail-to-head construct with two SARS VHH-72 proteins connected by a (GGGGS)3 linker. VHH-23-VHH-23 refers to the two irrelevant VHHs linked via the same (GGGGS)3 linker. E) SARS-CoV-1 S and F) SARS-CoV-2 S pseudoviruses were used to evaluate the neutralization capacity of bivalent VHH-72-Fc. GBP and PBS were included as negative controls. NI cells were not infected." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bivalency overcomes the high off-rate constant of SARS VHH-72.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) SARS-CoV-1 S and &lt;strong&gt;B&lt;/strong&gt;) SARS-CoV-2 S VSV pseudoviruses were used to evaluate the neutralization capacity of SARS VHH-72. MERS VHH-55 and PBS were included as negative controls. Luciferase activity is reported in counts per second (c.p.s.). NI cells were not infected. &lt;strong&gt;C&lt;/strong&gt;) Binding of bivalent VHHs was tested by ELISA against SARS-CoV-1 S and &lt;strong&gt;D&lt;/strong&gt;) SARS-CoV-2 RBD-SD1. VHH-72-Fc refers to SARS VHH-72 fused to a human IgG1 Fc domain by a GS(GGGGS)&lt;sub&gt;2&lt;/sub&gt; linker. VHH-72-Fc (S) is the same Fc fusion with a GS, rather than a GS(GGGGS)&lt;sub&gt;2&lt;/sub&gt;, linker. GBP is an irrelevant GFP-binding protein. VHH-72-VHH-72 refers to the tail-to-head construct with two SARS VHH-72 proteins connected by a (GGGGS)&lt;sub&gt;3&lt;/sub&gt; linker. VHH-23-VHH-23 refers to the two irrelevant VHHs linked via the same (GGGGS)&lt;sub&gt;3&lt;/sub&gt; linker. &lt;strong&gt;E&lt;/strong&gt;) SARS-CoV-1 S and &lt;strong&gt;F&lt;/strong&gt;) SARS-CoV-2 S pseudoviruses were used to evaluate the neutralization capacity of bivalent VHH-72-Fc. GBP and PBS were included as negative controls. NI cells were not infected.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F6.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F6.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <span class="caption-title">Bivalency overcomes the high off-rate constant of SARS VHH-72.</span><p id="p-30" class="first-child"><strong>A</strong>) SARS-CoV-1 S and <strong>B</strong>) SARS-CoV-2 S VSV pseudoviruses were used to evaluate the neutralization capacity of SARS VHH-72. MERS VHH-55 and PBS were included as negative controls. Luciferase activity is reported in counts per second (c.p.s.). NI cells were not infected. <strong>C</strong>) Binding of bivalent VHHs was tested by ELISA against SARS-CoV-1 S and <strong>D</strong>) SARS-CoV-2 RBD-SD1. VHH-72-Fc refers to SARS VHH-72 fused to a human IgG1 Fc domain by a GS(GGGGS)<sub>2</sub> linker. VHH-72-Fc (S) is the same Fc fusion with a GS, rather than a GS(GGGGS)<sub>2</sub>, linker. GBP is an irrelevant GFP-binding protein. VHH-72-VHH-72 refers to the tail-to-head construct with two SARS VHH-72 proteins connected by a (GGGGS)<sub>3</sub> linker. VHH-23-VHH-23 refers to the two irrelevant VHHs linked via the same (GGGGS)<sub>3</sub> linker. <strong>E</strong>) SARS-CoV-1 S and <strong>F</strong>) SARS-CoV-2 S pseudoviruses were used to evaluate the neutralization capacity of bivalent VHH-72-Fc. GBP and PBS were included as negative controls. NI cells were not infected.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="VHH-72-Fc neutralizes SARS-CoV-2 S pseudoviruses. A) BLI sensorgram measuring apparent binding affinity of VHH-72-Fc to immobilized SARS-CoV-2 RBD-SD1. Binding curves are colored black, buffer-only blanks are colored gray and the fit of the data to a 1:1 binding curve is colored red. B) Time course analysis of VHH-72-Fc expression in ExpiCHO cells. Cell culture supernatants of transiently transfected ExpiCHO cells were removed on days 3-7 after transfection (or until cell viability dropped below 75%), as indicated. Two control mAbs were included for comparison, along with the indicated amounts of purified GBP-Fc as a loading control. C) SARS-CoV-2 S pseudotyped VSV neutralization assay. Monolayers of Vero E6 cells were infected with pseudoviruses that had been pre-incubated with the mixtures indicated by the legend. The VHH-72-Fc used in this assay was purified after expression in ExpiCHO cells (n = 4). VHH-23-Fc is an irrelevant control VHH-Fc (n = 3). NI cells were not infected. Luciferase activity is reported in counts per second (c.p.s.) &#xB1; SEM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;VHH-72-Fc neutralizes SARS-CoV-2 S pseudoviruses.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) BLI sensorgram measuring apparent binding affinity of VHH-72-Fc to immobilized SARS-CoV-2 RBD-SD1. Binding curves are colored black, buffer-only blanks are colored gray and the fit of the data to a 1:1 binding curve is colored red. &lt;strong&gt;B&lt;/strong&gt;) Time course analysis of VHH-72-Fc expression in ExpiCHO cells. Cell culture supernatants of transiently transfected ExpiCHO cells were removed on days 3-7 after transfection (or until cell viability dropped below 75%), as indicated. Two control mAbs were included for comparison, along with the indicated amounts of purified GBP-Fc as a loading control. &lt;strong&gt;C&lt;/strong&gt;) SARS-CoV-2 S pseudotyped VSV neutralization assay. Monolayers of Vero E6 cells were infected with pseudoviruses that had been pre-incubated with the mixtures indicated by the legend. The VHH-72-Fc used in this assay was purified after expression in ExpiCHO cells (n = 4). VHH-23-Fc is an irrelevant control VHH-Fc (n = 3). NI cells were not infected. Luciferase activity is reported in counts per second (c.p.s.) &#xB1; SEM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F7.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F7.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <span class="caption-title">VHH-72-Fc neutralizes SARS-CoV-2 S pseudoviruses.</span><p id="p-31" class="first-child"><strong>A</strong>) BLI sensorgram measuring apparent binding affinity of VHH-72-Fc to immobilized SARS-CoV-2 RBD-SD1. Binding curves are colored black, buffer-only blanks are colored gray and the fit of the data to a 1:1 binding curve is colored red. <strong>B</strong>) Time course analysis of VHH-72-Fc expression in ExpiCHO cells. Cell culture supernatants of transiently transfected ExpiCHO cells were removed on days 3-7 after transfection (or until cell viability dropped below 75%), as indicated. Two control mAbs were included for comparison, along with the indicated amounts of purified GBP-Fc as a loading control. <strong>C</strong>) SARS-CoV-2 S pseudotyped VSV neutralization assay. Monolayers of Vero E6 cells were infected with pseudoviruses that had been pre-incubated with the mixtures indicated by the legend. The VHH-72-Fc used in this assay was purified after expression in ExpiCHO cells (n = 4). VHH-23-Fc is an irrelevant control VHH-Fc (n = 3). NI cells were not infected. Luciferase activity is reported in counts per second (c.p.s.) ± SEM.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-10"><h2 class="">DISCUSSION</h2><p id="p-32">Here we report the isolation and characterization of two potently neutralizing single-domain antibodies from a llama immunized with prefusion-stabilized MERS-CoV and SARS-CoV-1 spikes. These VHHs bind to the spike RBDs with high affinity and are capable of neutralizing S pseudotyped viruses <em>in vitro</em>. To our knowledge, the isolation and characterization of SARS-CoV-1 S-directed VHHs have not been described before. Several MERS-CoV S-specific VHHs have been described, all of which have been directed against the RBD. Several of these VHHs have also been reported to block DPP4 binding, much like MERS VHH-55 (Stalin <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Raj et al., 2018</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Zhao et al., 2018</a>). By solving the crystal structures of these newly isolated VHHs in complex with their respective viral targets, we provide detailed insights into epitope binding and their mechanisms of neutralization.</p><p id="p-33">A number of RBD-directed conventional antibodies have been described that are capable of neutralizing SARS-CoV-1 or MERS-CoV. The epitope of MERS VHH-55 overlaps with the epitopes of several of these MERS-CoV RBD-directed antibodies including C2, MCA1, m336, JC57-14, D12, 4C2 and MERS-27 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen et al., 2017</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li et al., 2015</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang et al., 2018</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wang et al., 2015</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ying et al., 2015</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yu et al., 2015</a>) (<strong>S. <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7A</a></strong>). The epitope of SARS VHH-72 does not significantly overlap with the epitopes of any previously described antibodies other than that of the recently described CR3022, which is also capable of binding to the RBDs of both SARS-CoV-1 and SARS-CoV-2 S (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hwang et al., 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pak et al., 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Prabakaran et al., 2006</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Walls et al., 2019</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yuan M, 2020</a>) (<strong>S. <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a></strong>). However, unlike SARS VHH-72, CR3022 does not prevent the binding of ACE2 and it lacks neutralizing activity against SARS-CoV-2 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tian et al., 2020</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Yuan M, 2020</a>). Because SARS VHH-72 binds with nanomolar affinity to a portion of the SARS-CoV-1 S RBD that exhibits low sequence variation, as demonstrated by its cross-reactivity toward the WIV1-CoV and SARS-CoV-2 RBDs, it may broadly bind all S proteins from SARS-CoV-like viruses. We show that by engineering a bivalent VHH-72-Fc construct, we are able to compensate for the relatively high off-rate constant of the monovalent SARS VHH-72. This bivalent molecule expresses well in transiently transfected ExpiCHO cells (∼300 mg/L) and is capable of potently neutralizing SARS-CoV-2 S pseudoviruses <em>in vitro</em>.</p><p id="p-34">Due to the inherent thermostability and chemostability of VHHs, they have been investigated as potential therapeutics against a number of diseases. Several HIV- and influenza-directed VHHs have been reported previously, and there are multiple RSV-directed VHHs that have been evaluated (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Detalle et al., 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ibanez et al., 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Koch et al., 2017</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Rossey et al., 2017</a>). The possibility of administering these molecules via a nebulized spray is particularly attractive in the case of respiratory pathogens because the VHHs could theoretically be inhaled directly to the site of infection in an effort to maximize bioavailability and function (Larios <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mora et al., 2018</a>). Due to the current lack of treatments for MERS, SARS and COVID-19 and the devastating effects associated with pandemic coronavirus outbreaks, both prophylactic and therapeutic interventions are sorely needed. It is our hope that due to their favorable biophysical properties and their potent neutralization capacity, MERS VHH-55, SARS VHH-72 and VHH-72-Fc may serve as both useful reagents for researchers and as potential therapeutic candidates.</p></div><div class="section" id="sec-11"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-36">Conceptualization, D.W., D.D.V., B.S.G., B.S., N.C., X.S., and J.S.M.; Investigation and visualization, D.W., D.D.V., K.S.C., G.M.T., W.V.B., K.R., L.v.S., M.H., S.P., and B.S.; Writing - Original Draft, D.W. and D.D.V.; Writing – Reviewing &amp; Editing, D.W., D.D.V., K.S.C., G.M.T., B.S.G., N.C., B.S., X.S., and J.S.M.; Supervision, B.S.G., N.C., B.S., X.S., and J.S.M.</p></div><div class="section COI-statement" id="sec-12"><h2 class="">DECLARATIONS OF INTEREST</h2><p id="p-37">K.S.C., B.S.G. and J.S.M. are inventors on US patent application no. 62/412,703, entitled “Prefusion Coronavirus Spike Proteins and Their Use”. D.W., K.S.C., B.S.G., and J.S.M. are inventors on US patent application no. 62/972,886, entitled “2019-nCoV Vaccine.” D.W., D.D.V., B.S.G., B.S., X.S., and J.S.M. are inventors on US patent application no. 62/988,610, entitled “Coronavirus Binders”. D.W., N.C., B.S., X.S., and J.S.M. are inventors on US patent application no. 62/991,408, entitled “SARS-CoV-2 Virus Binders”.</p></div><div class="section" id="sec-13"><h2 class="">MATERIALS AND METHODS</h2><div class="section" id="sec-14"><h3>Llama immunization</h3><p id="p-38">A llama, negative for antibodies against MERS-CoV and SARS-CoV-1 S glycoprotein, was subcutaneously immunized with approximately 150 µg recombinant SARS-CoV-1 S-2P protein on days 0, 7, 28 and 150 µg recombinant MERS-CoV S-2P protein on days 14 and 28 and 150 µg of both MERS-CoV S-2P and SARS-CoV-1 S-2P protein on day 35 (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kirchdoerfer et al., 2018</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pallesen et al., 2017</a>). The adjuvant used was Gerbu LQ#3000. Immunizations and handling of the llama were performed according to directive 2010/63/EU of the European parliament for the protection of animals used for scientific purposes and approved by the Ethical Committee for Animal Experiments of the Vrije Universiteit Brussel (permit No. 13-601-1). Blood was collected 5 days after the last immunization for the preparation of lymphocytes. Total RNA from the peripheral blood lymphocytes was extracted and used as template for the first strand cDNA synthesis with oligo dT primer. Using this cDNA, the VHH encoding sequences were amplified by PCR and cloned between the <em>PstI</em> and <em>NotI</em> sites of the phagemid vector pMECS. In the pMECS vector, the VHH encoding sequence is followed by a linker, HA and His<sub>6</sub> tag (AAAYPYDVPDYGSHHHHHH). Electro-competent E.coli TG1 cells were transformed with the recombinant pMECS vector resulting in a VHH library of about 3 x 10<sup>8</sup> independent transformants. The resulting TG1 library stock was then infected with VCS M13 helper phages to obtain a library of VHH-presenting phages.</p></div><div class="section" id="sec-15"><h3>Isolation of MERS- and SARS-CoV-directed VHH-displaying phages</h3><p id="p-39">Phages displaying MERS-CoV-specific VHHs were enriched after 2 rounds of biopanning on 20 μg of immobilized MERS-CoV S-2P protein in one well of a microtiter plate (type II, F96 Maxisorp, Nuc). For each panning round an uncoated well was used as a negative control. The wells were then washed 5 times with phosphate-buffered saline (PBS) + 0.05% Tween 20 and blocked with SEA BLOCK blocking buffer (Thermo Scientific) in the first panning round and 5% milk powder in PBS in the second panning round. About 10<sup>11</sup> phages were added to the coated well and incubated for 1 hour at room temperature. Non-specifically bound phages were removed by washing with PBS + 0.05% Tween 20 (10 times in the first panning round and 15 times in the second panning round). The retained phages were eluted with TEA-solution (14% trimethylamine (Sigma) pH 10) and subsequently neutralized with 1 M Tris-HCl pH 8. The collected phages were amplified in exponentially growing <em>E.coli</em> TG1 cells, infected with VCS M13 helper phages and subsequently purified using PEG 8,000/NaCl precipitation for the next round of selection. Enrichment after each panning round was determined by infecting TG1 cells with 10-fold serial dilutions of the collected phages after which the bacteria were plated on LB agar plates with 100 μg mL<sup>−1</sup> ampicillin and 1% glucose.</p><p id="p-40">Phages displaying SARS-CoV-1 directed VHHs were enriched after 2 rounds of biopanning on 20 μg of SARS-CoV-1 S-2P protein captured with an anti-foldon antibody (generously provided by Dr. Vicente Mas) in one well of a microtiter plate (type II, F96 Maxisorp, Nuc). Before panning phages were first added to DS-Cav1 protein (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McLellan et al., 2013</a>) containing a C-terminal foldon domain, to deplete foldon specific phages. The unbound phages were next added to the coated well. Panning was performed as described above.</p></div><div class="section" id="sec-16"><h3>Periplasmic ELISA screen to select MERS- and SARS-CoV directed VHHs</h3><p id="p-41">After panning, 45 individual colonies of phage infected bacteria isolated after the first panning round on MERS-CoV S-2P or SARS-CoV-1 S-2P protein and 45 individual colonies isolated after the second panning round on MERS-CoV S-2P or SARS-CoV-1 S-2P protein were randomly selected for further analysis by ELISA for the presence of MERS-CoV and SARS-CoV-1 specific VHHs, respectively. The individual colonies were inoculated in 2 mL of terrific broth (TB) medium with 100 µg/mL ampicillin in 24-well deep well plates. After growing individual colonies for 5 hours at 37 °C, isopropyl β-D-1-thiogalactopyranoside (IPTG) (1 mM) was added to induce VHH expression during overnight incubation at 37 °C. To prepare periplasmic extract, the bacterial cells were pelleted and resuspended in 250 μL TES buffer (0.2 M Tris-HCl pH 8, 0.5 mM EDTA, 0.5 M sucrose) and incubated at 4 °C for 30 min. Subsequently 350 μL water was added to induce an osmotic shock. After 1 hour incubation at 4 °C followed by centrifugation, the periplasmic extract was collected.</p><p id="p-42">VHH-containing periplasmic extracts were then tested for binding to either MERS-CoV S-2P or SARS-CoV-1 S-2P protein. Briefly, in the PE-ELISA screen after panning on MERS-CoV S-2P protein, wells of microtiter plates (type II, F96 Maxisorp, Nuc) were coated overnight at 37°C with 100 ng MERS-CoV S-2P (without foldon), MERS-CoV S-2P protein (with foldon) or as negative controls coated with SARS-CoV-1 S-2P protein (with foldon), HCoV-HKU1 S-2P (without foldon), DS-Cav1 (with foldon) or bovine serum albumin (BSA, Sigma-Aldrich). In the PE-ELISA screen after panning on SARS-CoV-1 S protein wells of microtiter plates (type II, F96 Maxisorp, Nuc) were coated with 100 ng SARS-CoV-1 S-2P protein (with foldon), SARS-CoV-1 S-2P protein captured with an anti-foldon antibody (with foldon) or as negative controls coated with MERS-CoV S-2P (without foldon), HCoV-HKU1 S-2P (without foldon), DS-Cav1 (with foldon) or bovine serum albumin (BSA, Sigma-Aldrich). The coated plates were blocked with 5% milk powder in PBS and 50 μL of the periplasmic extract was added to the wells. Bound VHHs were detected with anti-HA (1/2,000, MMS-101P Biolegend) mAb followed by horseradish peroxidase (HRP)-linked anti-mouse IgG (1/2,000, NXA931, GE Healthcare). Periplasmic fractions, for which the OD<sub>450</sub> value of the antigen coated wells were at least two times higher than the OD<sub>450</sub> value of the BSA coated wells, were considered to be specific for the coated antigen and selected for sequencing. The selected clones were grown in 3 mL of LB medium with 100 μg/mL ampicillin. The DNA of the selected colonies was isolated using the QIAprep Spin Miniprep kit (Qiagen) and sequenced using the MP057 primer (5’-TTATGCTTCCGGCTCGTATG-3’).</p></div><div class="section" id="sec-17"><h3>Cloning of MERS- and SARS-CoV directed VHHs into a Pichia pastoris expression vector</h3><p id="p-43">In order to express the MERS- and SARS-CoV VHHs in <em>Pichia pastoris</em>, the VHH encoding sequences were cloned in the pKai61 expression vector (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schoonooghe et al., 2009</a>). In the vector, the VHH sequences contain a C-terminal 6x His-tag, are under the control of the methanol inducible AOX1 promotor and in frame with a modified version of the <em>S.cerevisae</em> α-mating factor prepro signal sequence. The vector contains a Zeocine resistance marker for selection in bacteria as well as in yeast cells. The VHH encoding sequences were amplified by PCR using the following forward and reverse primer (5’-GGCGGGTATCTCTCGAGAAAAGGCAGGTGCAGCTGCAGGAGTCTGGG-3’) and (5’-CTAACTAGTCTAGTGATGGTGATGGTGGTGGCTGGAGACGGTGACCTGG-3’) and cloned between the <em>Xho</em>I and <em>Spe</em>I sites in the pKai61 vector. The vectors were linearized by <em>PmeI</em> and transformed in the <em>Pichia pastoris</em> strain GS115 using the condensed transformation protocol described by Lin-Cereghino <em>et al</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lin-Cereghino et al., 2005</a>). After transformation, the yeast cells were plated on YPD plates (1% (w/v) yeast extract, 2% (w/v) peptone, 2% (w/v) dextrose and 2% (w/v) agar) supplemented with zeocin (100 µg/mL) for selection.</p></div><div class="section" id="sec-18"><h3>Generation of bivalent VHH-constructs for production in <em>Pichia pastoris</em></h3><p id="p-44">To generate bivalent tandem tail-to-head VHH constructs, the VHH sequence was amplified by PCR using the following forward (5’-GGGGTATCTCTCGAGAAAAGGCAGGTGC AGCTGGTGGAGTCTGGG-3’) and reverse (5’-AGACTCCTGCAGCTGCACCTGACT ACCGCCGCCTCCAGATCCACCTCCGCCACTACCGCCTCCGCCGCTGGAGACGGTGAC CTGGG-3’) primers, thereby removing a <em>Pst</em>I site from the beginning of the VHH coding sequence and adding a (GGGGS)<sub>3</sub> linker and the start of the VHH coding sequence with a <em>Pst</em>I site at the end of the sequence. After PCR, the fragment was cloned between the <em>XhoI</em> and <em>SpeI</em> sites in a SARS VHH-72 containing pKai61 vector, thereby generating a homo-bivalent construct. The vector containing this bivalent VHH was linearized and transformed in GS155 <em>Pichia pastoris</em> cells as outlined above.</p></div><div class="section" id="sec-19"><h3>Pichia production and purification MERS- and SARS-CoV directed VHHs</h3><p id="p-45">The transformed <em>Pichia pastoris</em> clones were first expressed in 2 mL cultures. On day 1, 4 clones of each construct were inoculated in 2 mL of YPNG medium (2% pepton, 1% Bacto yeast extract, 1.34% YNB, 0.1 M potassium phosphate pH 6, 0.00004% biotin, 1% glycerol) with 100 μg/mL Zeocin (Life Technologies) and incubated while shaking at 28 °C for 24 hours. The next day, the cells were pelleted by centrifugation and the medium was replaced by YPNM medium (2% pepton, 1% Bacto yeast extract, 1.34% YNB, 0.1 M potassium phosphate pH 6.0, 1% methanol) to induce VHH expression. Cultures were incubated at 28 °C and 50 μL of 50% methanol was added at 16, 24 and 40 h. After 48 h, the yeast cells were pelleted and the supernatant was collected. The presence of soluble VHHs in the supernatants was verified using SDS-PAGE and subsequent Coomassie Blue staining. VHH-containing supernatants of the different clones for each construct were pooled and the VHHs were purified using HisPur<sup>TM</sup> Ni-NTA Spin Plates (88230, Thermo Scientific™). Next, purified VHHs were concentrated on AcroPrep<sup>TM</sup> Advance 96-well filter plates for ultrafiltration 3 kDa cutoff (8033,Pall) and the imidazole-containing elution buffer was exchanged with PBS.</p><p id="p-46">Production was scaled up (50 mL) for the VHHs with neutralizing capacity. Growth and methanol induction conditions and harvesting of medium were similar as mentioned above for the 2 mL cultures. The secreted VHHs in the medium were precipitated by ammonium sulfate (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> precipitation (80% saturation) for 4 h at 4 °C. The insoluble fraction was pelleted by centrifugation at 20,000 g and resuspended in 10 mL binding buffer (20 mM NaH<sub>2</sub>PO<sub>4</sub> pH 7.5, 0.5M NaCl and 20 mM imidazole pH 7.4). The VHHs were purified from the solution using a 1 mL HisTrap HP column (GE Healthcare). To elute the bound VHHs a linear imidazole gradient starting from 20 mM and ending at 500 mM imidazole in binding buffer over a total volume of 20 mL was used. VHH containing fractions were pooled and concentrated and the elution buffer was exchanged with PBS with a Vivaspin column (5 kDa cutoff, GE Healthcare).</p></div><div class="section" id="sec-20"><h3>Enzyme-linked immunosorbent assay</h3><p id="p-47">Wells of microtiter plates (type II, F96 Maxisorp, Nuc) were coated overnight at 4 °C, respectively, with 100 ng recombinant MERS-CoV S-2P protein (with foldon), SARS-CoV-1 S-2P protein (with foldon), MERS-CoV RBD, MERS-CoV NTD, MERS-CoV S1, SARS-CoV-1 RBD, SARS-CoV-1 NTD or Fc-tagged SARS-CoV-2 RBD-SD1. The coated plates were blocked with 5% milk powder in PBS. Dilution series of the VHHs were added to the wells. Binding was detected by incubating the plates sequentially with either mouse anti-Histidine Tag antibody (MCA1396, Abd Serotec) followed horseradish peroxidase (HRP)-linked anti-mouse IgG (1/2000, NXA931, GE Healthcare) or Streptavidin-HRP (554066, BD Biosciences) or by an HRP-linked rabbit anti-camelid VHH monoclonal antibody (A01861-200, GenScript). After washing 50 µL of TMB substrate (Tetramethylbenzidine, BD OptETA) was added to the plates and the reaction was stopped by addition of 50 µL of 1 M H<sub>2</sub>SO4. The absorbance at 450 nM was measured with an iMark Microplate Absorbance Reader (Bio Rad). Curve fitting was performed using nonlinear regression (Graphpad 7.0).</p></div><div class="section" id="sec-21"><h3>CoV pseudovirus neutralization</h3><p id="p-48">Pseudovirus neutralization assay methods have been previously described (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Pallesen et al., 2017</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Wang et al., 2015</a>). Briefly, pseudoviruses expressing spike genes for MERS-CoV England1 (GenBank ID: AFY13307) and SARS-CoV-1 Urbani (GenBank ID: AAP13441.1) were produced by co-transfection of plasmids encoding a luciferase reporter, lentivirus backbone, and spike genes in 293T cells (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Wang et al., 2015</a>). Serial dilutions of VHHs were mixed with pseudoviruses, incubated for 30 min at room temperature, and then added to previously-plated Huh7.5 cells. 72 hours later, cells were lysed, and relative luciferase activity was measured. Percent neutralization was calculated considering uninfected cells as 100% neutralization and cells transduced with only pseudovirus as 0% neutralization. IC<sub>50</sub> titers were determined based on sigmoidal nonlinear regression.</p><p id="p-49">Replication-deficient VSV pseudotyped with MERS-CoV S, SARS-CoV-1 S or SARS-CoV-2 S and coding for GFP or firefly luciferase were generated as described previously (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berger Rentsch and Zimmer, 2011</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hoffmann, 2020</a>). For the VSV pseudotype neutralization experiments, the pseudoviruses were incubated for 30 min at 37 °C with different dilutions of purified VHHs or with dilution series of culture supernatant of 293S cells that had been transfected with plasmids coding for SARS VHH-72 fused to human IgG1 Fc (VHH-72-Fc) or with GFP-binding protein (GBP: a VHH specific for GFP). The incubated pseudoviruses were subsequently added to confluent monolayers of Vero E6 cells. Sixteen hours later, the transduction efficiency was quantified by measuring the firefly luciferase activity in cell lysates using the firefly luciferase substrate of the dual-luciferase reporter assay system (Promega) and a Glowmax plate luminometer (Promega).</p></div><div class="section" id="sec-22"><h3>Mammalian protein expression and purification</h3><p id="p-50">Mammalian expression plasmids encoding for SARS VHH72, MERS VHH55, residues 367-589 of MERS-CoV S (England1 strain), residues 320-502 of SARS-CoV-1 S (Tor2 strain), residues 307-510 of WIV1-CoV S, residues 319-591 of SARS-CoV-2 S, residues 1-281 of SARS-CoV-1 S (Tor2 strain), residues 1-351 of MERS-CoV S (England1 strain), residues 1-751 of MERS-CoV S (England1 strain), residues 1-1190 of SARS-CoV-1 S (Tor2 strain) with K968P and V969P substitutions (SARS-CoV-1 S-2P), residues 1-1291 of MERS-CoV S (England1 strain) with V1060P and L1061P substitutions (MERS S-2P), residues 1-1208 of SARS-CoV-2 S with K986P and V987P substitutions (SARS-CoV-2 S-2P), residues 1-615 of ACE2 and residues 40-766 of DPP4 were transfected into FreeStyle293 cells using polyethylenimine (PEI). All of these plasmids contained N-terminal signal sequences to ensure secretion into the cell supernatant.</p><p id="p-51">Supernatants were harvested and constructs containing C-terminal HRV3C cleavage sites, 8x His-Tags and Twin-Strep-Tags (SARS VHH72, MERS VHH55, MERS-CoV S1, SARS-CoV-1 S-2P, MERS-CoV S-2P, SARS-CoV-2 S-2P, ACE2 and DPP4) were purified using Strep-Tactin resin (IBA). Constructs containing C-terminal HRV3C cleavage sites and Fc-tags (SARS-CoV-1 RBD, MERS-CoV RBD, WIV1-CoV RBD, SARS-CoV-2 RBD-SD1, SARS-CoV-1 NTD, MERS-CoV NTD) were purified using Protein A resin (Pierce). The SARS-CoV-1 RBD, MERS-CoV RBD, WIV1-CoV RBD, SARS-CoV-2 RBD-SD1, SARS VHH-72, MERS VHH-55, MERS-CoV NTD and SARS-CoV-1 NTD were then further purified using a Superdex 75 column (GE Healthcare) in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub>. MERS-CoV S1, SARS-CoV-1 S-2P, MERS-CoV S-2P, ACE2 and DPP4 were further purified using a Superose 6 column (GE Healthcare) in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub>.</p><p id="p-52">HEK 293S cells were transfected with VHH-72-Fc or VHH-72-Fc (S) encoding plasmids using PEI. Briefly, suspension-adapted and serum-free HEK 293S cells were seeded at 3 x 10<sup>6</sup> cells/mL in Freestyle-293 medium (ThermoFisher Scientific). Next, 4.5 µg of pcDNA3.3-VHH72-Fc plasmid DNA was added to the cells and incubated on a shaking platform at 37 °C and 8% CO<sub>2</sub>, for 5 min. Next, 9 µg of PEI was added to the cultures, and cells were further incubated for 5 h, after which an equal culture volume of Ex-Cell-293 (Sigma) was added to the cells. Transfections were incubated for 4 days, after which cells were pelleted (10’, 300g) and supernatants were filtered before further use.</p><p id="p-53">VHH-72-Fc was expressed in ExpiCHO cells (ThermoFisher Scientific), according to the manufacturer’s protocol. Briefly, a 25 mL culture of 6 x10<sup>6</sup> cells/mL, grown at 37 °C and 8% CO<sub>2</sub> was transfected with 20 µg of pcDNA3.3-VHH-72-Fc plasmid DNA using ExpiFectamine CHO reagent. One day after transfection, 150 µL of ExpiCHO enhancer and 4 mL of ExpiCHO feed was added to the cells, and cultures were further incubated at 32 °C and 5% CO<sub>2</sub>. Cells were fed a second time 5 days post-transfection. Cultures were harvested as soon as cell viability dropped below 75%. For purification of the VHH-72-Fc, supernatants were loaded on a 5 mL MabSelect SuRe column (GE Healthcare). Unbound proteins were washed away with McIlvaine buffer pH 7.2, and bound proteins were eluted using McIlvaine buffer pH 3. Immediately after elution, protein-containing fractions were neutralized using a saturated Na<sub>3</sub>PO<sub>4</sub> buffer. These neutralized fractions were then pooled, and loaded onto a HiPrep Desalting column for buffer exchange into storage buffer (25 mM L-Histidine, 125 mM NaCl).</p></div><div class="section" id="sec-23"><h3>Surface plasmon resonance</h3><p id="p-54">His-tagged SARS VHH-72 or MERS VHH-55 was immobilized to a single flow cell of an NTA sensorchip at a level of ∼400 response units (RUs) per cycle using a Biacore X100 (GE Healthcare). The chip was doubly regenerated using 0.35 M EDTA and 0.1 M NaOH followed by 0.5 mM NiCl<sub>2</sub>. Three samples containing only running buffer, composed of 10 mM HEPES pH 8.0, 150 mM NaCl and 0.005% Tween 20, were injected over both ligand and reference flow cells, followed by either SARS-CoV-1 RBD, WIV1-CoV RBD, SARS-CoV-2 RBD-SD1 or MERS-CoV RBD serially diluted from 50-1.56 nM, with a replicate of the 3.1 nM concentration. The resulting data were double-reference subtracted and fit to a 1:1 binding model using the Biacore X100 Evaluation software.</p></div><div class="section" id="sec-24"><h3>Crystallization and data collection</h3><p id="p-55">Plasmids encoding for MERS VHH-55 and residues 367-589 of MERS-CoV S with a C-terminal HRV3C cleavage site and a monomeric human Fc tag were co-transfected into kifunensin-treated FreeStyle 293F cells, as described above. After purifying the cell supernatant with Protein A resin, the immobilized complex was treated with HRV3C protease and Endoglycosidase H to remove both tags and glycans. The complex was then purified using a Superdex 75 column in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub>. The purified complex was then concentrated to 5.0 mg/mL and used to prepare hanging-drop crystallization trays. Crystals grown in 1.0 M Na/K phosphate pH 7.5 were soaked in mother liquor supplemented with 20% ethylene glycol and frozen in liquid nitrogen. Diffraction data were collected to a resolution of 3.40 Å at the SBC beamline 19-ID (APS, Argonne National Laboratory) Plasmids encoding for SARS VHH-72 and residues 320-502 of SARS-CoV-1 S with a C-terminal HRV3C cleavage site and a monomeric human Fc tag were co-transfected into kifunensin-treated FreeStyle 293F cells, as described above. After purifying the cell supernatant with Protein A resin, the immobilized complex was treated with HRV3C protease and Endoglycosidase H to remove both tags and glycans. The processed complex was subjected to size-exclusion chromatography using a Superdex 75 column in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub>. The purified complex was then concentrated to 10.0 mg/mL and used to prepare hanging-drop crystallization trays. Crystals grown in 0.1 M Tris pH 8.5, 0.2 M LiSO<sub>4</sub>, 0.1 M LiCl and 8% PEG 8000 were soaked in mother liquor supplemented with 20% glycerol and frozen in liquid nitrogen. Diffraction data were collected to a resolution of 2.20 Å at the SBC beamline 19-ID (APS, Argonne National Laboratory)</p></div><div class="section" id="sec-25"><h3>Structure determination</h3><p id="p-56">Diffraction data for both complexes were indexed and integrated using iMOSFLM before being scaled in AIMLESS (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Battye et al., 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Evans and Murshudov, 2013</a>). The SARS-CoV-1 RBD+SARS VHH-72 dataset was phased by molecular replacement in PhaserMR using coordinates from PDBs 2AJF and 5F1O as search ensembles (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McCoy, 2007</a>). The MERS-CoV RBD+MERS VHH-55 dataset was also phased by molecular replacement in PhaserMR using coordinates from PDBs 4L72 and 5F1O as search ensembles. The resulting molecular replacement solutions were iteratively rebuilt and refined using Coot, ISOLDE and Phenix (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams et al., 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Croll, 2018</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Emsley and Cowtan, 2004</a>). Crystallographic software packages were curated by SBGrid (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Morin et al., 2013</a>).</p></div><div class="section" id="sec-26"><h3>Biolayer interferometry</h3><p id="p-57">Anti-human capture (AHC) tips (FortéBio) were soaked in running buffer composed of 10 mM HEPES pH 7.5, 150 mM NaCl, 3 mM EDTA, 0.005% Tween 20 and 1 mg/mL BSA for 20 min before being used to capture either Fc-tagged SARS-CoV-1 RBD, Fc-tagged SARS-CoV-2 RBD-SD1 or Fc-tagged MERS-CoV RBD to a level of 0.8 nm in an Octet RED96 (FortéBio). Tips were then dipped into either 100 nM MERS VHH-55 or 100 nM SARS VHH-72. Tips were next dipped into wells containing either 1 μM ACE2 or 100 nM DPP4 supplemented with the nanobody that the tip had already been dipped into to ensure continued saturation. Data were reference-subtracted and aligned to each other in Octet Data Analysis software v11.1 (FortéBio) based on a baseline measurement that was taken before being dipped into the final set of wells that contained either ACE2 or DPP4.</p><p id="p-58">BLI measurements were also performed with VHH-72-Fc fusion produced in HEK 293S cells. SARS-CoV-2 RBD with a mouse IgG1 Fc tag (Sino Biological) was immobilized to an anti-mouse IgG Fc capture (AMC) tip (FortéBio) to a response level of 0.5 nm. Supernatant of non-transfected and VHH-72-Fc transfected HEK293-S cells was applied in a three-fold dilution series in kinetics buffer. Binding was measured at 30 °C, with baseline and dissociation measured in equal dilution of non-transformed HEK293S supernatant in kinetics buffer. Between analyses, biosensors were regenerated by three times 20 s exposure to regeneration buffer (10 mM glycine pH 1.7).</p></div><div class="section" id="sec-27"><h3>Flow cytometry</h3><p id="p-59">Binding of VHH-72-Fc, VHH-72-Fc (S) and monomeric SARS VHH-72 to SARS-CoV-1 and SARS-CoV-2 S was analyzed by flow cytometry using cells transfected with a GFP expression plasmid combined with an expression plasmid for either SARS-CoV-1 or SARS-Cov-2 S. HEK 293S culture media (1/20 diluted in PBS + 0.5%BSA) of VHH-72-Fc and VHH-72-Fc (S) transformants were incubated with transfected cells. Binding of the VHH-72-Fc and VHH-72-Fc (S) to cells was determined by an AF633 conjugated goat anti-human IgG antibody and binding to SARS-CoV-1 or SARS-CoV-2 S was calculated as the mean AF633 fluorescence intensity (MFI) of GFP expressing cells (GFP<sup>+</sup>) divided by the MFI of GFP negative cells (GFP<sup>-</sup>).</p></div><div class="section" id="sec-28"><h3>RBD competition assay on Vero E6 cells</h3><p id="p-60">SARS-CoV-2 RBD fused to murine IgG Fc (Sino Biological) at a final concentration of 0.4 μg/mL was incubated with a dilution series of tail-to-head bivalent VHHs or VHH-Fc fusions and incubated at room temperature for 20 min before an additional 10 min incubation on ice. Vero E6 cells grown at sub-confluency were detached by cell dissociation buffer (Sigma) and trypsin treatment. After washing once with PBS the cells were blocked with 1% BSA in PBS on ice. All remaining steps were also performed on ice. The mixtures containing RBD and tail-to-head bivalent VHHs or VHH-Fc fusions were added to the cells and incubated for one hour.</p><p id="p-61">Subsequently, the cells were washed 3 times with PBS containing 0.5% BSA and stained with an AF647 conjugated donkey anti-mouse IgG antibody (Invitrogen) for 1 hour. Following additional 3 washes with PBS containing 0.5% BSA, the cells were analyzed by flow cytometry using an BD LSRII flow cytometer (BD Biosciences).</p></div></div><div id="sec-29" class="subsection display-objects"><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CoV VHH immunization and panning. A) Schematic depicting the immunization strategy that was used to isolate both SARS-CoV-1 S and MERS-CoV S-directed VHHs from a single llama. The prefusion stabilized SARS-CoV-1 spike is shown in pink and the prefusion stabilized MERS-CoV spike is shown in tan. B) Phylogenetic tree of the isolated MERS-CoV and SARS-CoV S-directed VHHs, based on the neighbor joining method. C) Reactivity of MERS-CoV and SARS-CoV S-directed VHHs against the prefusion stabilized MERS-CoV S and SARS-CoV-1 S protein, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;CoV VHH immunization and panning.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) Schematic depicting the immunization strategy that was used to isolate both SARS-CoV-1 S and MERS-CoV S-directed VHHs from a single llama. The prefusion stabilized SARS-CoV-1 spike is shown in pink and the prefusion stabilized MERS-CoV spike is shown in tan. &lt;strong&gt;B&lt;/strong&gt;) Phylogenetic tree of the isolated MERS-CoV and SARS-CoV S-directed VHHs, based on the neighbor joining method. &lt;strong&gt;C&lt;/strong&gt;) Reactivity of MERS-CoV and SARS-CoV S-directed VHHs against the prefusion stabilized MERS-CoV S and SARS-CoV-1 S protein, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F8.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F8.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 1:</span> <span class="caption-title">CoV VHH immunization and panning.</span><p id="p-62" class="first-child"><strong>A</strong>) Schematic depicting the immunization strategy that was used to isolate both SARS-CoV-1 S and MERS-CoV S-directed VHHs from a single llama. The prefusion stabilized SARS-CoV-1 spike is shown in pink and the prefusion stabilized MERS-CoV spike is shown in tan. <strong>B</strong>) Phylogenetic tree of the isolated MERS-CoV and SARS-CoV S-directed VHHs, based on the neighbor joining method. <strong>C</strong>) Reactivity of MERS-CoV and SARS-CoV S-directed VHHs against the prefusion stabilized MERS-CoV S and SARS-CoV-1 S protein, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence alignment of neutralizing SARS-CoV and MERS-CoV S-directed VHHs. Invariant residues are shown as black dots. The CDRs are shown underneath black lines.." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Sequence alignment of neutralizing SARS-CoV and MERS-CoV S-directed VHHs.&lt;/span&gt; Invariant residues are shown as black dots. The CDRs are shown underneath black lines..&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F9.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F9.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2:</span> <span class="caption-title">Sequence alignment of neutralizing SARS-CoV and MERS-CoV S-directed VHHs.</span><p id="p-63" class="first-child">Invariant residues are shown as black dots. The CDRs are shown underneath black lines..</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lack of binding of MERS-CoV and SARS-CoV directed VHHs to non-RBD epitopes. ELISA data showing binding of the MERS-CoV specific VHHs to the MERS-CoV S1 protein and absence of binding of the MERS-CoV and SARS-CoV specific VHHs against the MERS-CoV NTD and SARS-CoV-1 NTD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Lack of binding of MERS-CoV and SARS-CoV directed VHHs to non-RBD epitopes.&lt;/span&gt; ELISA data showing binding of the MERS-CoV specific VHHs to the MERS-CoV S1 protein and absence of binding of the MERS-CoV and SARS-CoV specific VHHs against the MERS-CoV NTD and SARS-CoV-1 NTD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F10.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F10.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3:</span> <span class="caption-title">Lack of binding of MERS-CoV and SARS-CoV directed VHHs to non-RBD epitopes.</span><p id="p-64" class="first-child">ELISA data showing binding of the MERS-CoV specific VHHs to the MERS-CoV S1 protein and absence of binding of the MERS-CoV and SARS-CoV specific VHHs against the MERS-CoV NTD and SARS-CoV-1 NTD, respectively. A VHH against an irrelevant antigen (F-VHH) was included as a control.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MERS VHH-55 binds to a relatively conserved epitope on the MERS-CoV RBD. A) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 in white ribbons and the MERS-CoV RBD as a multicolored molecular surface. More variable residues are shown in warm colors and more conserved residues are shown in cool colors according to the spectrum (bottom). Sequence alignments and variability mapping was performed using ConSurf. B) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown as ribbons with MERS VHH-55 colored blue and the MERS-CoV RBD colored tan. Phe506 from the MERS-CoV RBD and Trp99 from MERS VHH-55, which are thought to form hydrophobic interactions with one another are shown as sticks surrounded by a transparent molecular surface. C) SPR sensorgram measuring the binding of MERS VHH-55 to the naturally occurring MERS-CoV RBD F506L variant. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MERS VHH-55 binds to a relatively conserved epitope on the MERS-CoV RBD.&lt;/span&gt; A) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 in white ribbons and the MERS-CoV RBD as a multicolored molecular surface. More variable residues are shown in warm colors and more conserved residues are shown in cool colors according to the spectrum (&lt;em&gt;bottom&lt;/em&gt;). Sequence alignments and variability mapping was performed using ConSurf. &lt;strong&gt;B&lt;/strong&gt;) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown as ribbons with MERS VHH-55 colored blue and the MERS-CoV RBD colored tan. Phe506 from the MERS-CoV RBD and Trp99 from MERS VHH-55, which are thought to form hydrophobic interactions with one another are shown as sticks surrounded by a transparent molecular surface. &lt;strong&gt;C&lt;/strong&gt;) SPR sensorgram measuring the binding of MERS VHH-55 to the naturally occurring MERS-CoV RBD F506L variant. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F11.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F11.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 4:</span> <span class="caption-title">MERS VHH-55 binds to a relatively conserved epitope on the MERS-CoV RBD.</span><p id="p-65" class="first-child">A) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 in white ribbons and the MERS-CoV RBD as a multicolored molecular surface. More variable residues are shown in warm colors and more conserved residues are shown in cool colors according to the spectrum (<em>bottom</em>). Sequence alignments and variability mapping was performed using ConSurf. <strong>B</strong>) The crystal structure of MERS VHH-55 bound to the MERS-CoV RBD is shown as ribbons with MERS VHH-55 colored blue and the MERS-CoV RBD colored tan. Phe506 from the MERS-CoV RBD and Trp99 from MERS VHH-55, which are thought to form hydrophobic interactions with one another are shown as sticks surrounded by a transparent molecular surface. <strong>C</strong>) SPR sensorgram measuring the binding of MERS VHH-55 to the naturally occurring MERS-CoV RBD F506L variant. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS VHH-72 binds to a broadly conserved epitope on the SARS-CoV-1 RBD. A) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown, with colors corresponding to those of SFig 4A. B) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and WIV1-CoV are colored teal. C) SPR sensorgram measuring the binding of SARS VHH-72 to the WIV1-CoV RBD. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SARS VHH-72 binds to a broadly conserved epitope on the SARS-CoV-1 RBD.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown, with colors corresponding to those of &lt;strong&gt;S&lt;/strong&gt;Fig 4A. &lt;strong&gt;B&lt;/strong&gt;) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and WIV1-CoV are colored teal. &lt;strong&gt;C&lt;/strong&gt;) SPR sensorgram measuring the binding of SARS VHH-72 to the WIV1-CoV RBD. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F12.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F12.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 5:</span> <span class="caption-title">SARS VHH-72 binds to a broadly conserved epitope on the SARS-CoV-1 RBD.</span><p id="p-66" class="first-child"><strong>A</strong>) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown, with colors corresponding to those of <strong>S</strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4A</a>. <strong>B</strong>) The crystal structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as dark blue ribbons and the RBD as a pink molecular surface. Amino acids that vary between SARS-CoV-1 and WIV1-CoV are colored teal. <strong>C</strong>) SPR sensorgram measuring the binding of SARS VHH-72 to the WIV1-CoV RBD. Binding curves are colored black and the fit of the data to a 1:1 binding model is colored red.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Engineering a functional bivalent VHH construct. A) Flow cytometry measuring the binding of the bivalent SARS VHH-72 tail-to-head fusion (VHH-72-VHH-72) to SARS-CoV-1 or SARS-CoV-2 S expressed on the cell surface. VHH-23-VHH-23, a bivalent tail-to-head fusion of an irrelevant nanobody, was included as a negative control. B) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by VHH-72-VHH-72 in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells was detected by flow cytometry in the presence of the indicated bivalent VHHs (n = 2 except VHH-72-VHH-72 and VHH-23-VHH-23 at 5 &#xB5;g/ml, n = 5). C) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by bivalent VHH-72-Fc fusion proteins in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1-Fc to Vero E6 cells was detected by flow cytometry in the presence of the indicated constructs and amounts (n = 2 except no RBD, n = 4). D) Cell surface binding of SARS VHH-72 to SARS-CoV-1 S. 293T cells were transfected with a GFP expression plasmid together with a SARS-CoV-1 S expression plasmid. Binding of the indicated protein is expressed as the median fluorescent intensity (MFI), measured to detect the His-tagged MERS VHH-55 or SARS VHH-72 or the SARS VHH-72-Fc fusions, of the GFP positive cells divided by the MFI of the GFP negative cells. E) Cell surface binding of SARS VHH-72 to SARS-CoV-2. MFI was calculated using the same equation as S. Figure 6D." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Engineering a functional bivalent VHH construct.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) Flow cytometry measuring the binding of the bivalent SARS VHH-72 tail-to-head fusion (VHH-72-VHH-72) to SARS-CoV-1 or SARS-CoV-2 S expressed on the cell surface. VHH-23-VHH-23, a bivalent tail-to-head fusion of an irrelevant nanobody, was included as a negative control. &lt;strong&gt;B&lt;/strong&gt;) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by VHH-72-VHH-72 in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells was detected by flow cytometry in the presence of the indicated bivalent VHHs (n = 2 except VHH-72-VHH-72 and VHH-23-VHH-23 at 5 &#xB5;g/ml, n = 5). &lt;strong&gt;C&lt;/strong&gt;) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by bivalent VHH-72-Fc fusion proteins in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1-Fc to Vero E6 cells was detected by flow cytometry in the presence of the indicated constructs and amounts (n = 2 except no RBD, n = 4). &lt;strong&gt;D&lt;/strong&gt;) Cell surface binding of SARS VHH-72 to SARS-CoV-1 S. 293T cells were transfected with a GFP expression plasmid together with a SARS-CoV-1 S expression plasmid. Binding of the indicated protein is expressed as the median fluorescent intensity (MFI), measured to detect the His-tagged MERS VHH-55 or SARS VHH-72 or the SARS VHH-72-Fc fusions, of the GFP positive cells divided by the MFI of the GFP negative cells&lt;strong&gt;. E)&lt;/strong&gt; Cell surface binding of SARS VHH-72 to SARS-CoV-2. MFI was calculated using the same equation as &lt;strong&gt;S.&lt;/strong&gt; Figure 6D.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F13.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F13.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 6:</span> <span class="caption-title">Engineering a functional bivalent VHH construct.</span><p id="p-67" class="first-child"><strong>A</strong>) Flow cytometry measuring the binding of the bivalent SARS VHH-72 tail-to-head fusion (VHH-72-VHH-72) to SARS-CoV-1 or SARS-CoV-2 S expressed on the cell surface. VHH-23-VHH-23, a bivalent tail-to-head fusion of an irrelevant nanobody, was included as a negative control. <strong>B</strong>) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by VHH-72-VHH-72 in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells was detected by flow cytometry in the presence of the indicated bivalent VHHs (n = 2 except VHH-72-VHH-72 and VHH-23-VHH-23 at 5 µg/ml, n = 5). <strong>C</strong>) Binding of SARS-CoV-2 RBD-SD1 to Vero E6 cells is prevented by bivalent VHH-72-Fc fusion proteins in a dose-dependent fashion. Binding of SARS-CoV-2 RBD-SD1-Fc to Vero E6 cells was detected by flow cytometry in the presence of the indicated constructs and amounts (n = 2 except no RBD, n = 4). <strong>D</strong>) Cell surface binding of SARS VHH-72 to SARS-CoV-1 S. 293T cells were transfected with a GFP expression plasmid together with a SARS-CoV-1 S expression plasmid. Binding of the indicated protein is expressed as the median fluorescent intensity (MFI), measured to detect the His-tagged MERS VHH-55 or SARS VHH-72 or the SARS VHH-72-Fc fusions, of the GFP positive cells divided by the MFI of the GFP negative cells<strong>. E)</strong> Cell surface binding of SARS VHH-72 to SARS-CoV-2. MFI was calculated using the same equation as <strong>S.</strong> <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6D</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the CoV VHH epitopes with known RBD-directed antibodies. A) The structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 as blue ribbons and the MERS-CoV RBD as a white molecular surface. Epitopes from previously reported crystal structures of the MERS-CoV RBD bound by RBD-directed antibodies are shown as colored patches on the MERS-CoV RBD surface. The LCA60 epitope is shown in yellow, the MERS S4 epitope is shown in green, the overlapping C2/MCA1/m336 epitopes are shown in red and the overlapping JC57-14/D12/4C2/MERS-27 epitopes are shown in purple. B) The structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as cyan ribbons and the SARS-CoV-1 RBD as a white molecular surface. Epitopes from previously reported crystal structures of the SARS-CoV-1 RBD bound by RBD-directed antibodies are shown as colored patches on the SARS-CoV-1 RBD surface. The 80R epitope is shown in blue, the S230 epitope is shown in yellow, and the overlapping m396/F26G19 epitopes are shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-87223373" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Comparison of the CoV VHH epitopes with known RBD-directed antibodies.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;) The structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 as blue ribbons and the MERS-CoV RBD as a white molecular surface. Epitopes from previously reported crystal structures of the MERS-CoV RBD bound by RBD-directed antibodies are shown as colored patches on the MERS-CoV RBD surface. The LCA60 epitope is shown in yellow, the MERS S4 epitope is shown in green, the overlapping C2/MCA1/m336 epitopes are shown in red and the overlapping JC57-14/D12/4C2/MERS-27 epitopes are shown in purple. &lt;strong&gt;B&lt;/strong&gt;) The structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as cyan ribbons and the SARS-CoV-1 RBD as a white molecular surface. Epitopes from previously reported crystal structures of the SARS-CoV-1 RBD bound by RBD-directed antibodies are shown as colored patches on the SARS-CoV-1 RBD surface. The 80R epitope is shown in blue, the S230 epitope is shown in yellow, and the overlapping m396/F26G19 epitopes are shown in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F14.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F14.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010165/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 7:</span> <span class="caption-title">Comparison of the CoV VHH epitopes with known RBD-directed antibodies.</span><p id="p-68" class="first-child"><strong>A</strong>) The structure of MERS VHH-55 bound to the MERS-CoV RBD is shown with MERS VHH-55 as blue ribbons and the MERS-CoV RBD as a white molecular surface. Epitopes from previously reported crystal structures of the MERS-CoV RBD bound by RBD-directed antibodies are shown as colored patches on the MERS-CoV RBD surface. The LCA60 epitope is shown in yellow, the MERS S4 epitope is shown in green, the overlapping C2/MCA1/m336 epitopes are shown in red and the overlapping JC57-14/D12/4C2/MERS-27 epitopes are shown in purple. <strong>B</strong>) The structure of SARS VHH-72 bound to the SARS-CoV-1 RBD is shown with SARS VHH-72 as cyan ribbons and the SARS-CoV-1 RBD as a white molecular surface. Epitopes from previously reported crystal structures of the SARS-CoV-1 RBD bound by RBD-directed antibodies are shown as colored patches on the SARS-CoV-1 RBD surface. The 80R epitope is shown in blue, the S230 epitope is shown in yellow, and the overlapping m396/F26G19 epitopes are shown in red.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-35">We thank members of the McLellan Laboratory for providing helpful comments on the manuscript. We would like to thank Dr. John Ludes-Meyers for assistance with cell transfection and protein production. This work was supported by a National Institutes of Health (NIH)/National Institute of Allergy and Infectious Disease (NIAID) grant R01-AI127521 (to J.S.M.). Research was supported by funding from VIB, Ghent University GOA project to N.C. and X.S., FWO and VLAIO fellowships and research projects to various VIB-CMB COVID-19 response team members. We acknowledge the team of the VIB Nanobody Service Facility for their services. D.D.V. was supported by a FWOsb fellowship, W.V. by the FWO-SBO grant “GlycoDelete”, S.P. by BMBF (Rapid consortium, 01K11723D), and B.S. by FWO-EOS project VIREOS. We are deeply indebted to the VIB-CMB COVID-19 response team members, who volunteered to offer their expertise and agile work under conditions of almost complete lockdown and societal standstill. We thank the support staff of both VIB-IRC and VIB-CMB centers, the members of VIB Discovery Sciences units’ COVID-19 team for rapid and consistent support and input. Argonne is operated by UChicago Argonne, LLC, for the US Department of Energy (DOE), Office of Biological and Environmental Research under Contract DE-AC02-06CH11357.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.1" data-doi="10.1107/S0907444902016657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>,  <span class="cit-name-given-names">P.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grosse-Kunstleve</span>,  <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">L.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ioerger</span>,  <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCoy</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moriarty</span>,  <span class="cit-name-given-names">N.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Read</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacchettini</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauter</span>,  <span class="cit-name-given-names">N.K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Terwilliger</span>,  <span class="cit-name-given-names">T.C</span></span>. (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">1948</span>–<span class="cit-lpage">1954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DAdams%26rft.auinit1%253DP.%2BD.%26rft.volume%253D58%26rft.issue%253DPt%2B11%26rft.spage%253D1948%26rft.epage%253D1954%26rft.atitle%253DPHENIX%253A%2Bbuilding%2Bnew%2Bsoftware%2Bfor%2Bautomated%2Bcrystallographic%2Bstructure%2Bdetermination.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444902016657%26rft_id%253Dinfo%253Apmid%252F12393927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444902016657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12393927&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178745000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.2" data-doi="10.1107/S0907444910048675"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Battye</span>,  <span class="cit-name-given-names">T.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kontogiannis</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">H.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Leslie</span>,  <span class="cit-name-given-names">A.G</span></span>. (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">271</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DBattye%26rft.auinit1%253DT.%2BG.%26rft.volume%253D67%26rft.issue%253DPt%2B4%26rft.spage%253D271%26rft.epage%253D281%26rft.atitle%253DiMOSFLM%253A%2Ba%2Bnew%2Bgraphical%2Binterface%2Bfor%2Bdiffraction-image%2Bprocessing%2Bwith%2BMOSFLM.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444910048675%26rft_id%253Dinfo%253Apmid%252F21460445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444910048675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21460445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288532800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.3" data-doi="10.1371/journal.pone.0025858"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berger Rentsch</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zimmer</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A vesicular stomatitis virus replicon-based bioassay for the rapid and sensitive determination of multi-species type I interferon</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e25858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBerger%2BRentsch%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25858%26rft.epage%253De25858%26rft.atitle%253DA%2Bvesicular%2Bstomatitis%2Bvirus%2Breplicon-based%2Bbioassay%2Bfor%2Bthe%2Brapid%2Band%2Bsensitive%2Bdetermination%2Bof%2Bmulti-species%2Btype%2BI%2Binterferon.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025858%26rft_id%253Dinfo%253Apmid%252F21998709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21998709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.4" data-doi="10.1128/JVI.77.16.8801-8811.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Zee</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>,  <span class="cit-name-given-names">C.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.J.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">77</span>, <span class="cit-fpage">8801</span>–<span class="cit-lpage">8811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBosch%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D77%26rft.issue%253D16%26rft.spage%253D8801%26rft.epage%253D8811%26rft.atitle%253DThe%2BCoronavirus%2BSpike%2BProtein%2BIs%2Ba%2BClass%2BI%2BVirus%2BFusion%2BProtein%253A%2BStructural%2Band%2BFunctional%2BCharacterization%2Bof%2Bthe%2BFusion%2BCore%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.16.8801-8811.2003%26rft_id%253Dinfo%253Apmid%252F12885899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xNi84ODAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.5" data-doi="10.1016/S0140-6736(20)30154-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kok</span>,  <span class="cit-name-given-names">K.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">To</span>,  <span class="cit-name-given-names">K.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xing</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yip</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">514</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D514%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930154-9%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30154-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.6" data-doi="10.1093/infdis/jix209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">1807</span>–<span class="cit-lpage">1815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D215%26rft.spage%253D1807%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjix209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jix209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.7" data-doi="10.1107/S2059798318002425"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Croll</span>,  <span class="cit-name-given-names">T.I</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr D Struct Biol</abbr> <span class="cit-vol">74</span>, <span class="cit-fpage">519</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr%2BD%2BStruct%2BBiol%26rft.volume%253D74%26rft.spage%253D519%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS2059798318002425%26rft_id%253Dinfo%253Apmid%252F29872003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S2059798318002425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29872003&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Vlieger</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballegeer</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossey</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schepens</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saelens</span>,  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Single-Domain Antibodies and Their Formatting to Combat Viral Infections</span>. <abbr class="cit-jnl-abbrev">Antibodies (Basel</abbr>) <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.9" data-doi="10.1128/AAC.01802-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Detalle</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohr</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palomo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piedra</span>,  <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">B.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mas</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Power</span>,  <span class="cit-name-given-names">U.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stortelers</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allosery</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Generation and Characterization of ALX-0171, a Potent Novel Therapeutic Nanobody for the Treatment of Respiratory Syncytial Virus Infection</span>. <abbr class="cit-jnl-abbrev">Antimicrob Agents Chemother</abbr> <span class="cit-vol">60</span>, <span class="cit-fpage">6</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob%2BAgents%2BChemother%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.01802-15%26rft_id%253Dinfo%253Apmid%252F26438495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjY6IjYwLzEvNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjYuMDEwMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.10" data-doi="10.1110/ps.34602"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dumoulin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrath</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Meirhaeghe</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meersman</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heremans</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frenken</span>,  <span class="cit-name-given-names">L.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muyldermans</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyns</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Matagne</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Single-domain antibody fragments with high conformational stability</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">500</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DDumoulin%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D500%26rft.epage%253D515%26rft.atitle%253DSingle-domain%2Bantibody%2Bfragments%2Bwith%2Bhigh%2Bconformational%2Bstability%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.34602%26rft_id%253Dinfo%253Apmid%252F11847273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.34602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847273&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173931900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.11" data-doi="10.1107/S0907444904019158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emsley</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cowtan</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Coot: model-building tools for molecular graphics</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr> <span class="cit-vol">60</span>, <span class="cit-fpage">2126</span>–<span class="cit-lpage">2132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DEmsley%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253DPt%2B12%2BPt%2B1%26rft.spage%253D2126%26rft.epage%253D2132%26rft.atitle%253DCoot%253A%2Bmodel-building%2Btools%2Bfor%2Bmolecular%2Bgraphics.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444904019158%26rft_id%253Dinfo%253Apmid%252F15572765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444904019158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15572765&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225360500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.12" data-doi="10.1107/S0907444913000061"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">P.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Murshudov</span>,  <span class="cit-name-given-names">G.N</span></span>. (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">How good are my data and what is the resolution?</span> <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr> <span class="cit-vol">69</span>, <span class="cit-fpage">1204</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr%2BD%2BBiol%2BCrystallogr%26rft.volume%253D69%26rft.spage%253D1204%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444913000061%26rft_id%253Dinfo%253Apmid%252F23793146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444913000061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793146&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320712800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.13" data-doi="10.1128/JCM.00636-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaunt</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hardie</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claas</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Templeton</span>,  <span class="cit-name-given-names">K.E</span></span>. (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Epidemiology and clinical presentations of the four human coronaviruses 229E, HKU1, NL63, and OC43 detected over 3 years using a novel multiplex real-time PCR method</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr> <span class="cit-vol">48</span>, <span class="cit-fpage">2940</span>–<span class="cit-lpage">2947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00636-10%26rft_id%253Dinfo%253Apmid%252F20554810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQ4LzgvMjk0MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjYuMDEwMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.14" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chmura</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>,  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>,  <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.15" data-doi="10.1074/jbc.M111.242818"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Govaert</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pellis</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deschacht</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincke</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrath</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muyldermans</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saerens</span>,  <span class="cit-name-given-names">D</span></span>. (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Dual beneficial effect of interloop disulfide bond for single domain antibody fragments</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>, <span class="cit-fpage">1970</span>–<span class="cit-lpage">1979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.242818%26rft_id%253Dinfo%253Apmid%252F22128183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODcvMy8xOTcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.16" data-doi="10.1038/cr.2016.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X</span></span>. (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft_id%253Dinfo%253Apmid%252F28008928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28008928&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.17" data-doi="10.1038/363446a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamers-Casterman</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atarhouch</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muyldermans</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamers</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Songa</span>,  <span class="cit-name-given-names">E.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bendahman</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hamers</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Naturally occurring antibodies devoid of light chains</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">363</span>, <span class="cit-fpage">446</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHamers-Casterman%26rft.auinit1%253DC.%26rft.volume%253D363%26rft.issue%253D6428%26rft.spage%253D446%26rft.epage%253D448%26rft.atitle%253DNaturally%2Boccurring%2Bantibodies%2Bdevoid%2Bof%2Blight%2Bchains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F363446a0%26rft_id%253Dinfo%253Apmid%252F8502296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/363446a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8502296&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LE93800057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.K.-W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H</span>; <span class="cit-name-surname">Krüger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N</span>; <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M</span>; <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C</span>; <span class="cit-name-surname">Pöhlmann</span></span>, S. (<span class="cit-pub-date">2020</span>). <abbr class="cit-jnl-abbrev">The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. bioRxiv.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.19" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.20" data-doi="10.1074/jbc.M603275200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">W.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Santelli</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stec</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marasco</span>,  <span class="cit-name-given-names">W.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Liddington</span>,  <span class="cit-name-given-names">R.C</span></span>. (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">281</span>, <span class="cit-fpage">34610</span>-<span class="cit-lpage">34616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.21" data-doi="10.1093/infdis/jiq168"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ibanez</span>,  <span class="cit-name-given-names">L.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Filette</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hultberg</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verrips</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temperton</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandevelde</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schepens</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanlandschoot</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saelens</span>,  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Nanobodies with in vitro neutralizing activity protect mice against H5N1 influenza virus infection</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">203</span>, <span class="cit-fpage">1063</span>–<span class="cit-lpage">1072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiq168%26rft_id%253Dinfo%253Apmid%252F21450996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiq168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21450996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">H.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilman</span>,  <span class="cit-name-given-names">M.S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Battles</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacerdote</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chuang</span>,  <span class="cit-name-given-names">G.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwong</span>,  <span class="cit-name-given-names">P.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J.S</span></span>. (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Iterative screen optimization maximizes the efficiency of macromolecular crystallization</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr F Struct Biol Commun</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">123</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr%2BF%2BStruct%2BBiol%2BCommun%26rft.volume%253D75%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.23" data-doi="10.1038/nature17200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassine</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">H.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A.B</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Pre-fusion structure of a human coronavirus spike protein</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">531</span>, <span class="cit-fpage">118</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17200%26rft_id%253Dinfo%253Apmid%252F26935699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26935699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">H.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A.B</span></span>. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D15701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koch</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalusche</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanfield</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danquah</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khazanehdari</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Briesen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geertsma</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">I.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernery</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Selection of nanobodies with broad neutralizing potential against primary HIV-1 strains using soluble subtype C gp140 envelope trimers</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">8390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D8390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.26" data-doi="10.1056/NEJMoa030781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ksiazek</span>,  <span class="cit-name-given-names">T.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erdman</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>,  <span class="cit-name-given-names">C.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peret</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emery</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urbani</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Comer</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">A novel coronavirus associated with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">1953</span>–<span class="cit-lpage">1966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030781%26rft_id%253Dinfo%253Apmid%252F12690092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690092&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010165v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lan</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J</span>; <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J</span>; <span class="cit-name-surname">Shan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S</span>; <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H</span>; <span class="cit-name-surname">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S</span>; <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q</span>; <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X</span>; <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q</span>; <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L</span>; <span class="cit-name-surname">Wang</span></span>, X. (<span class="cit-pub-date">2020</span>). <abbr class="cit-jnl-abbrev">Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. bioRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.28" data-doi="10.1080/19420862.2018.1470727"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Larios</span> <span class="cit-name-surname">Mora</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span>, <span class="cit-name-surname">Detalle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span>, <span class="cit-name-surname">Gallup</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.M.</span>, <span class="cit-name-surname">Van Geelen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span>, <span class="cit-name-surname">Stohr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span>, <span class="cit-name-surname">Duprez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span>, <span class="cit-name-surname">and Ackermann</span></span>, M.R. (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Delivery of ALX-0171 by inhalation greatly reduces respiratory syncytial virus disease in newborn lambs</span>. <abbr class="cit-jnl-abbrev">MAbs</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">778</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMAbs%26rft.volume%253D10%26rft.spage%253D778%26rft_id%253Dinfo%253Adoi%252F10.1080%252F19420862.2018.1470727%26rft_id%253Dinfo%253Apmid%252F29733750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/19420862.2018.1470727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29733750&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.29" data-doi="10.1126/science.aaq0620"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laursen</span>,  <span class="cit-name-given-names">N.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friesen</span>,  <span class="cit-name-given-names">R.H.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jongeneelen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blokland</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vermond</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Eijgen</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Diepen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obmolova</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Universal protection against influenza infection by a multidomain antibody to influenza hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span>, <span class="cit-fpage">598</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLaursen%26rft.auinit1%253DN.%2BS.%26rft.volume%253D362%26rft.issue%253D6414%26rft.spage%253D598%26rft.epage%253D602%26rft.atitle%253DUniversal%2Bprotection%2Bagainst%2Binfluenza%2Binfection%2Bby%2Ba%2Bmultidomain%2Bantibody%2Bto%2Binfluenza%2Bhemagglutinin%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaq0620%26rft_id%253Dinfo%253Apmid%252F30385580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNC81OTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.30" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S.C</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span>, <span class="cit-fpage">1864</span>–<span class="cit-lpage">1868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.31" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasilieva</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berne</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Somasundaran</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luzuriaga</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenough</span>,  <span class="cit-name-given-names">T.C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.32" data-doi="10.1038/cr.2015.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1237</span>–<span class="cit-lpage">1249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D25%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2015.113%26rft_id%253Dinfo%253Apmid%252F26391698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2015.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26391698&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin-Cereghino</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">W.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luong</span>,  <span class="cit-name-given-names">L.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">S.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lin-Cereghino</span>,  <span class="cit-name-given-names">G.P</span></span>. (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Condensed protocol for competent cell preparation and transformation of the methylotrophic yeast Pichia pastoris</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">38</span>, <span class="cit-issue">44</span>, <span class="cit-fpage">46</span>, 48.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D38%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.34" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.35" data-doi="10.1107/S0907444906045975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCoy</span>,  <span class="cit-name-given-names">A.J</span></span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Solving structures of protein complexes by molecular replacement with Phaser</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">32</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DMcCoy%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D63%26rft.issue%253DPt%2B1%26rft.spage%253D32%26rft.epage%253D41%26rft.atitle%253DSolving%2Bstructures%2Bof%2Bprotein%2Bcomplexes%2Bby%2Bmolecular%2Breplacement%2Bwith%2BPhaser.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444906045975%26rft_id%253Dinfo%253Apmid%252F17164524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444906045975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17164524&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242796000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sastry</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stewart-Jones</span>,  <span class="cit-name-given-names">G.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Srivatsan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Structure-based design of a fusion glycoprotein vaccine for respiratory syncytial virus. Science (New York</span>, <abbr class="cit-jnl-abbrev">NY</abbr>) <span class="cit-vol">342</span>, <span class="cit-fpage">592</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNY%26rft.volume%253D342%26rft.spage%253D592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.37" data-doi="10.7554/eLife.01456"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eisenbraun</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Key</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanschagrin</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timony</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ottaviano</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sliz</span>,  <span class="cit-name-given-names">P</span></span>. (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Collaboration gets the most out of software</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e01456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De01456%26rft.epage%253De01456%26rft.atitle%253DCollaboration%2Bgets%2Bthe%2Bmost%2Bout%2Bof%2Bsoftware%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01456%26rft_id%253Dinfo%253Apmid%252F24040512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.01456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24040512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.38" data-doi="10.1016/j.jmb.2009.03.042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pak</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharon</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satkunarajah</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auperin</span>,  <span class="cit-name-given-names">T.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelvin</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seetharaman</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cochrane</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plummer</span>,  <span class="cit-name-given-names">F.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berry</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">388</span>, <span class="cit-fpage">815</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPak%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D388%26rft.issue%253D4%26rft.spage%253D815%26rft.epage%253D823%26rft.atitle%253DStructural%2Binsights%2Binto%2Bimmune%2Brecognition%2Bof%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2BS%2Bprotein%2Breceptor%2Bbinding%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.03.042%26rft_id%253Dinfo%253Apmid%252F19324051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.03.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19324051&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266302500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.39" data-doi="10.1073/pnas.1707304114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">H.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E7348</span>–<span class="cit-lpage">E7357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1707304114%26rft_id%253Dinfo%253Apmid%252F28807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM1L0U3MzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.40" data-doi="10.1074/jbc.M600697200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choudhry</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ji</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>,  <span class="cit-name-given-names">D.S</span></span>. (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">281</span>, <span class="cit-fpage">15829</span>–<span class="cit-lpage">15836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M600697200%26rft_id%253Dinfo%253Apmid%252F16597622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjMvMTU4MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.41" data-doi="10.1038/nature12005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">V.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smits</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dekkers</span>,  <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dijkman</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Demmers</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>, <span class="cit-fpage">251</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12005%26rft_id%253Dinfo%253Apmid%252F23486063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23486063&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Respaud</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vecellio</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diot</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Heuze-Vourc’h</span>,  <span class="cit-name-given-names">N</span></span>. (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Nebulization as a delivery method for mAbs in respiratory diseases</span>. <abbr class="cit-jnl-abbrev">Expert Opin Drug Deliv</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">1027</span>–<span class="cit-lpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BOpin%2BDrug%2BDeliv%26rft.volume%253D12%26rft.spage%253D1027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.43" data-doi="10.1038/ncomms14158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rossey</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilman</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kabeche</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sedeyn</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanekiyo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mas</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spitaels</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melero</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Potent single-domain antibodies that arrest respiratory syncytial virus fusion protein in its prefusion state</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">14158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D14158%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14158%26rft_id%253Dinfo%253Apmid%252F28194013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms14158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28194013&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.44" data-doi="10.1186/1472-6750-9-70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoonooghe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaigorodov</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zawisza</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumolyn</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haustraete</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grooten</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mertens</span>,  <span class="cit-name-given-names">N</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Efficient production of human bivalent and trivalent anti-MUC1 Fab-scFv antibodies in Pichia pastoris</span>. <abbr class="cit-jnl-abbrev">BMC Biotechnol</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbiotechnology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBiotechnol%26rft.aulast%253DSchoonooghe%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D70%26rft.epage%253D70%26rft.atitle%253DEfficient%2Bproduction%2Bof%2Bhuman%2Bbivalent%2Band%2Btrivalent%2Banti-MUC1%2BFab-scFv%2Bantibodies%2Bin%2BPichia%2Bpastoris.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6750-9-70%26rft_id%253Dinfo%253Apmid%252F19671134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6750-9-70&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19671134&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.45" data-doi="10.1126/sciadv.aas9667"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Stalin</span> <span class="cit-name-surname">Raj</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span>, <span class="cit-name-surname">Okba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.M.A.</span>, <span class="cit-name-surname">Gutierrez-Alvarez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span>, <span class="cit-name-surname">Drabek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span>, <span class="cit-name-surname">van Dieren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span>, <span class="cit-name-surname">Widagdo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span>, <span class="cit-name-surname">Lamers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span>, <span class="cit-name-surname">Widjaja</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span>, <span class="cit-name-surname">Fernandez-Delgado</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span>, <span class="cit-name-surname">Sola</span></span>, I., <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Chimeric camel/human heavy-chain antibodies protect against MERS-CoV infection</span>. <abbr class="cit-jnl-abbrev">Sci Adv</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">eaas9667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DStalin%2BRaj%26rft.auinit1%253DV.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253Deaas9667%26rft.epage%253Deaas9667%26rft.atitle%253DChimeric%2Bcamel%252Fhuman%2Bheavy-chain%2Bantibodies%2Bprotect%2Bagainst%2BMERS-CoV%2Binfection%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aas9667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNC84L2VhYXM5NjY3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">382</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.47" data-doi="10.1016/S0167-4838(99)00030-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van der Linden</span>,  <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frenken</span>,  <span class="cit-name-given-names">L.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Geus</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harmsen</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruuls</span>,  <span class="cit-name-given-names">R.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stok</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Ron</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Verrips</span>,  <span class="cit-name-given-names">C.T.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Comparison of physical chemical properties of llama VHH antibody fragments and mouse monoclonal antibodies</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr> <span class="cit-vol">1431</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253Dvan%2Bder%2BLinden%26rft.auinit1%253DR.%2BH.%26rft.volume%253D1431%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D46%26rft.atitle%253DComparison%2Bof%2Bphysical%2Bchemical%2Bproperties%2Bof%2Bllama%2BVHH%2Bantibody%2Bfragments%2Band%2Bmouse%2Bmonoclonal%2Bantibodies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-4838%252899%252900030-8%26rft_id%253Dinfo%253Apmid%252F10209277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-4838(99)00030-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10209277&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079954400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.48" data-doi="10.1038/nature16988"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frenz</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DiMaio</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rey</span>,  <span class="cit-name-given-names">F.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D</span></span>. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">531</span>, <span class="cit-fpage">114</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16988%26rft_id%253Dinfo%253Apmid%252F26855426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26855426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.49" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quispe</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameroni</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gopal</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">1026</span>–<span class="cit-lpage">1039</span> e1015.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30712865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A.P. Y-J</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">MA</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">A</span></span>; <span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">AT</span></span>; <span class="cit-auth"><span class="cit-name-surname">Veesler</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chappell</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanekiyo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">92</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.53" data-doi="10.1038/ncomms8712"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Modjarrad</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lees</span>,  <span class="cit-name-given-names">C.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassine</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanekiyo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Evaluation of candidate vaccine approaches for MERS-CoV</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">7712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D7712%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms8712%26rft_id%253Dinfo%253Apmid%252F26218507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms8712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26218507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.54" data-doi="10.1038/cr.2013.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">986</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D23%26rft.spage%253D986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.92%26rft_id%253Dinfo%253Apmid%252F23835475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23835475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322574900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.55" data-doi="10.1016/j.chom.2014.08.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xuan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ji</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Bat origins of MERS-CoV supported by bat coronavirus HKU4 usage of human receptor CD26</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">328</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D16%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25211075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2014.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25211075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.56" data-doi="10.3181/0903-MR-94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.Y</span></span>. (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Coronavirus diversity, phylogeny and interspecies jumping</span>. <abbr class="cit-jnl-abbrev">Exp Biol Med (Maywood</abbr>) <span class="cit-vol">234</span>, <span class="cit-fpage">1117</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%2BY.%26rft.volume%253D234%26rft.issue%253D10%26rft.spage%253D1117%26rft.epage%253D1127%26rft.atitle%253DCoronavirus%2BDiversity%252C%2BPhylogeny%2Band%2BInterspecies%2BJumping%26rft_id%253Dinfo%253Adoi%252F10.3181%252F0903-MR-94%26rft_id%253Dinfo%253Apmid%252F19546349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3181/0903-MR-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.26.010165v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abiona</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J.S</span></span>. (<span class="cit-pub-date">2020</span>). <span class="cit-source">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <span class="cit-publ-name">Science</span> (<span class="cit-publ-loc">New York, NY</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.Z. Y</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">L</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. <abbr class="cit-jnl-abbrev">Science</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.59" data-doi="10.1038/ncomms9223"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ying</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>,  <span class="cit-name-given-names">D.S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">8223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D8223%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9223%26rft_id%253Dinfo%253Apmid%252F26370782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">13133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D13133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M W.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C-CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">So</span>  <span class="cit-name-given-names">RTY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mok</span>  <span class="cit-name-given-names">CKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">IA.</span></span> (<span class="cit-pub-date">2020</span>). <abbr class="cit-jnl-abbrev">A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</abbr>. bioRxiv.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.62" data-doi="10.1038/ncomms15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft_id%253Dinfo%253Apmid%252F28393837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28393837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.63" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>,  <span class="cit-name-given-names">T.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>,  <span class="cit-name-given-names">A.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">367</span>, <span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tai</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">A Novel Nanobody Targeting Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Receptor-Binding Domain Has Potent Cross-Neutralizing Activity and Protective Efficacy against MERS-CoV</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">92</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010165v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.26.010165v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.23.004176v2" title="Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.26.009233v1" title="Design of potent membrane fusion inhibitors against SARS-CoV-2, an emerging coronavirus with high fusogenic activity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.26.010165v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.26.010165v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.26.010165v1" class="active" data-icon-position="" data-hide-link-title="0">Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1215887" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EHwP26S1_8XogYR8e-be-8G4jQUIAqlAx9PXYruAQtA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215887" id="node1215887" data-pisa="biorxiv;2020.03.26.010165v1" data-pisa-master="biorxiv;2020.03.26.010165" data-seqnum="1215887" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010165.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dorien</span> <span class="nlm-surname">De Vlieger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gretel M.</span> <span class="nlm-surname">Torres</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wander</span> <span class="nlm-surname">Van Breedam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kenny</span> <span class="nlm-surname">Roose</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Loes</span> <span class="nlm-surname">van Schie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-collab" data-hwp-id="collab-1">VIB-CMB COVID-19 Response Team</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Markus</span> <span class="nlm-surname">Hoffmann</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Pöhlmann</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Nico</span> <span class="nlm-surname">Callewaert</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Bert</span> <span class="nlm-surname">Schepens</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Saelens</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.010165; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.010165 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;title=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;title=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;text=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;title=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;t=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;title=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;title=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215887" id="node1215887--2" data-pisa="biorxiv;2020.03.26.010165v1" data-pisa-master="biorxiv;2020.03.26.010165" data-seqnum="1215887" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010165.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dorien</span> <span class="nlm-surname">De Vlieger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gretel M.</span> <span class="nlm-surname">Torres</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wander</span> <span class="nlm-surname">Van Breedam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kenny</span> <span class="nlm-surname">Roose</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Loes</span> <span class="nlm-surname">van Schie</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-collab" data-hwp-id="collab-1">VIB-CMB COVID-19 Response Team</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Markus</span> <span class="nlm-surname">Hoffmann</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Stefan</span> <span class="nlm-surname">Pöhlmann</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Nico</span> <span class="nlm-surname">Callewaert</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Bert</span> <span class="nlm-surname">Schepens</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Saelens</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.010165; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.010165 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Wrapp%2C+Daniel&amp;rft.au=De+Vlieger%2C+Dorien&amp;rft.au=Corbett%2C+Kizzmekia+S.&amp;rft.au=Torres%2C+Gretel+M.&amp;rft.au=Van+Breedam%2C+Wander&amp;rft.au=Roose%2C+Kenny&amp;rft.au=van+Schie%2C+Loes&amp;rft.au=%2C+&amp;rft.au=Hoffmann%2C+Markus&amp;rft.au=P%C3%B6hlmann%2C+Stefan&amp;rft.au=Graham%2C+Barney+S.&amp;rft.au=Callewaert%2C+Nico&amp;rft.au=Schepens%2C+Bert&amp;rft.au=Saelens%2C+Xavier&amp;rft.au=McLellan%2C+Jason+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1215887/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1215887/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1215887/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1215887/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1215887/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1215887/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1215887/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1215887/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1215887/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1215887/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1215887/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1215887/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;count=horizontal&amp;via=&amp;text=Structural%20Basis%20for%20Potent%20Neutralization%20of%20Betacoronaviruses%20by%20Single-domain%20Camelid%20Antibodies&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010165v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.010165v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
