<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/09/14/769224.atom" />
<meta name="HW.pisa" content="biorxiv;769224v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination" />
<meta name="DC.Identifier" content="10.1101/769224" />
<meta name="DC.Date" content="2019-09-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Using the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.

Significance The availability of a “universal” method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties." />
<meta name="DC.Contributor" content="Hyejeong Kim" />
<meta name="DC.Contributor" content="Victor D. Ellis" />
<meta name="DC.Contributor" content="Andrew Woodman" />
<meta name="DC.Contributor" content="Yan Zhao" />
<meta name="DC.Contributor" content="Jamie J. Arnold" />
<meta name="DC.Contributor" content="Craig E. Cameron" />
<meta name="article:published_time" content="2019-09-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Using the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.&lt;/p&gt;&lt;h3&gt;Significance&lt;/h3&gt;
&lt;p&gt;The availability of a “universal” method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;769224v1" />
<meta name="citation_id" content="769224v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/769224v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/769224v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/769224v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224.full.pdf" />
<meta name="citation_doi" content="10.1101/769224" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="769224" />
<meta name="citation_author" content="Hyejeong Kim" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author" content="Victor D. Ellis" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author" content="Andrew Woodman" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7757-5492" />
<meta name="citation_author" content="Yan Zhao" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author" content="Jamie J. Arnold" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author" content="Craig E. Cameron" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University" />
<meta name="citation_author_email" content="craig.cameron@med.unc.edu" />
<meta name="citation_reference" content="Graci JD, Cameron CE. 2008. Therapeutically targeting RNA viruses via lethal mutagenesis. Future Virol 3:553–566." />
<meta name="citation_reference" content="Perales C, Domingo E. 2016. Antiviral Strategies Based on Lethal Mutagenesis and Error Threshold. Curr Top Microbiol Immunol 392:323–39." />
<meta name="citation_reference" content="Furuta Y, Gowen BB, Takahashi K, Shiraki K, Smee DF, Barnard DL. 2013. Favipiravir (T-705), a novel viral RNA polymerase inhibitor. Antiviral Res 100:446–54." />
<meta name="citation_reference" content="Cameron CE, Moustafa IM, Arnold JJ. 2016. Fidelity of Nucleotide Incorporation by the RNA-Dependent RNA Polymerase from Poliovirus. Enzymes 39:293–323." />
<meta name="citation_reference" content="Lee CA, August A, Arnold JJ, Cameron CE. 2016. Polymerase Mechanism-Based Method of Viral Attenuation. Methods Mol Biol 1349:83–104." />
<meta name="citation_reference" content="Weeks SA, Lee CA, Zhao Y, Smidansky ED, August A, Arnold JJ, Cameron CE. 2012. A Polymerase mechanism-based strategy for viral attenuation and vaccine development. J Biol Chem 287:31618–22." />
<meta name="citation_reference" content="Graham RL, Becker MM, Eckerle LD, Bolles M, Denison MR, Baric RS. 2012. A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease. Nat Med 18:1820–6." />
<meta name="citation_reference" content="Vignuzzi M, Wendt E, Andino R. 2008. Engineering attenuated virus vaccines by controlling replication fidelity. Nat Med 14:154–61." />
<meta name="citation_reference" content="Li C, Wang H, Yuan T, Woodman A, Yang D, Zhou G, Cameron CE, Yu L. 2018. Foot-and-mouth disease virus type O specific mutations determine RNA-dependent RNA polymerase fidelity and virus attenuation. Virology 518:87–94." />
<meta name="citation_reference" content="Korboukh VK, Lee CA, Acevedo A, Vignuzzi M, Xiao Y, Arnold JJ, Hemperly S, Graci JD, August A, Andino R, Cameron CE. 2014. RNA virus population diversity, an optimum for maximal fitness and virulence. J Biol Chem 289:29531–44." />
<meta name="citation_reference" content="Rozen-Gagnon K, Stapleford KA, Mongelli V, Blanc H, Failloux AB, Saleh MC, Vignuzzi M. 2014. Alphavirus mutator variants present host-specific defects and attenuation in mammalian and insect models. PLoS Pathog 10:e1003877." />
<meta name="citation_reference" content="Zeng J, Wang H, Xie X, Li C, Zhou G, Yang D, Yu L. 2014. Ribavirin-resistant variants of foot-and-mouth disease virus: the effect of restricted quasispecies diversity on viral virulence. J Virol 88:4008–20." />
<meta name="citation_reference" content="Gnadig NF, Beaucourt S, Campagnola G, Borderia AV, Sanz-Ramos M, Gong P, Blanc H, Peersen OB, Vignuzzi M. 2012. Coxsackievirus B3 mutator strains are attenuated in vivo. Proc Natl Acad Sci U S A 109:E2294–303." />
<meta name="citation_reference" content="Coffey LL, Beeharry Y, Borderia AV, Blanc H, Vignuzzi M. 2011. Arbovirus high fidelity variant loses fitness in mosquitoes and mice. Proc Natl Acad Sci U S A 108:16038–43." />
<meta name="citation_reference" content="Vignuzzi M, Stone JK, Arnold JJ, Cameron CE, Andino R. 2006. Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population. Nature 439:344–8." />
<meta name="citation_reference" content="Arnold JJ, Vignuzzi M, Stone JK, Andino R, Cameron CE. 2005. Remote site control of an active site fidelity checkpoint in a viral RNA-dependent RNA polymerase. J Biol Chem 280:25706–16." />
<meta name="citation_reference" content="Fitzsimmons WJ, Woods RJ, McCrone JT, Woodman A, Arnold JJ, Yennawar M, Evans R, Cameron CE, Lauring AS. 2018. A speed-fidelity trade-off determines the mutation rate and virulence of an RNA virus. PLoS Biol 16:e2006459." />
<meta name="citation_reference" content="Li C, Wang H, Shi J, Yang D, Zhou G, Chang J, Cameron CE, Woodman A, Yu L. 2019. Senecavirus-Specific Recombination Assays Reveal the Intimate Link between Polymerase Fidelity and RNA Recombination. J Virol." />
<meta name="citation_reference" content="Woodman A, Lee KM, Janissen R, Gong YN, Dekker NH, Shih SR, Cameron CE. 2019. Predicting Intraserotypic Recombination in Enterovirus 71. J Virol 93." />
<meta name="citation_reference" content="Te Velthuis AJW, Long JC, Bauer DLV, Fan RLY, Yen HL, Sharps J, Siegers JY, Killip MJ, French H, Oliva-Martin MJ, Randall RE, de Wit E, van Riel D, Poon LLM, Fodor E. 2018. Mini viral RNAs act as innate immune agonists during influenza virus infection. Nat Microbiol 3:1234–1242." />
<meta name="citation_reference" content="Rawson JMO, Nikolaitchik OA, Keele BF, Pathak VK, Hu WS. 2018. Recombination is required for efficient HIV-1 replication and the maintenance of viral genome integrity. Nucleic Acids Res 46:10535–10545." />
<meta name="citation_reference" content="Woodman A, Arnold JJ, Cameron CE, Evans DJ. 2016. Biochemical and genetic analysis of the role of the viral polymerase in enterovirus recombination. Nucleic Acids Res 44:6883–95." />
<meta name="citation_reference" content="Poirier EZ, Mounce BC, Rozen-Gagnon K, Hooikaas PJ, Stapleford KA, Moratorio G, Vignuzzi M. 2015. Low-Fidelity Polymerases of Alphaviruses Recombine at Higher Rates To Overproduce Defective Interfering Particles. J Virol 90:2446–54." />
<meta name="citation_reference" content="Castro C, Smidansky ED, Arnold JJ, Maksimchuk KR, Moustafa I, Uchida A, Gotte M, Konigsberg W, Cameron CE. 2009. Nucleic acid polymerases use a general acid for nucleotidyl transfer. Nat Struct Mol Biol 16:212–8." />
<meta name="citation_reference" content="Crotty S, Maag D, Arnold JJ, Zhong W, Lau JY, Hong Z, Andino R, Cameron CE. 2000. The broad-spectrum antiviral ribonucleoside ribavirin is an RNA virus mutagen. Nat Med 6:1375–9." />
<meta name="citation_reference" content="Pfeiffer JK, Kirkegaard K. 2003. A single mutation in poliovirus RNA-dependent RNA polymerase confers resistance to mutagenic nucleotide analogs via increased fidelity. Proc Natl Acad Sci U S A 100:7289–94." />
<meta name="citation_reference" content="Severson WE, Schmaljohn CS, Javadian A, Jonsson CB. 2003. Ribavirin causes error catastrophe during Hantaan virus replication. J Virol 77:481–8." />
<meta name="citation_reference" content="Vignuzzi M, Stone JK, Andino R. 2005. Ribavirin and lethal mutagenesis of poliovirus: molecular mechanisms, resistance and biological implications. Virus Res 107:173–81." />
<meta name="citation_reference" content="Sierra M, Airaksinen A, Gonzalez-Lopez C, Agudo R, Arias A, Domingo E. 2007. Foot-and-mouth disease virus mutant with decreased sensitivity to ribavirin: implications for error catastrophe. J Virol 81:2012–24." />
<meta name="citation_reference" content="Arias A, Arnold JJ, Sierra M, Smidansky ED, Domingo E, Cameron CE. 2008. Determinants of RNA-dependent RNA polymerase (in)fidelity revealed by kinetic analysis of the polymerase encoded by a foot-and-mouth disease virus mutant with reduced sensitivity to ribavirin. J Virol 82:12346–55." />
<meta name="citation_reference" content="Banerjee S, Aponte-Diaz D, Yeager C, Sharma SD, Ning G, Oh HS, Han Q, Umeda M, Hara Y, Wang RYL, Cameron CE. 2018. Hijacking of multiple phospholipid biosynthetic pathways and induction of membrane biogenesis by a picornaviral 3CD protein. PLoS Pathog 14:e1007086." />
<meta name="citation_reference" content="Oh HS, Banerjee S, Aponte-Diaz D, Sharma SD, Aligo J, Lodeiro MF, Ning G, Sharma R, Arnold JJ, Cameron CE. 2018. Multiple poliovirus-induced organelles suggested by comparison of spatiotemporal dynamics of membranous structures and phosphoinositides. PLoS Pathog 14:e1007036." />
<meta name="citation_reference" content="Dulin D, Vilfan ID, Berghuis BA, Hage S, Bamford DH, Poranen MM, Depken M, Dekker NH. 2015. Elongation-Competent Pauses Govern the Fidelity of a Viral RNA-Dependent RNA Polymerase. Cell Rep 10:983–992." />
<meta name="citation_reference" content="Lowry K, Woodman A, Cook J, Evans DJ. 2014. Recombination in enteroviruses is a biphasic replicative process involving the generation of greater-than genome length ‘imprecise’ intermediates. PLoS Pathog 10:e1004191." />
<meta name="citation_reference" content="Kumagai A, Ando R, Miyatake H, Greimel P, Kobayashi T, Hirabayashi Y, Shimogori T, Miyawaki A. 2013. A bilirubin-inducible fluorescent protein from eel muscle. Cell 153:1602–11." />
<meta name="citation_reference" content="Arnold JJ, Cameron CE. 1999. Poliovirus RNA-dependent RNA polymerase (3Dpol) is sufficient for template switching in vitro. J Biol Chem 274:2706–16." />
<meta name="citation_reference" content="Kirkegaard K, Baltimore D. 1986. The mechanism of RNA recombination in poliovirus. Cell 47:433–43." />
<meta name="citation_reference" content="Arnold JJ, Cameron CE. 2000. Poliovirus RNA-dependent RNA polymerase (3D(pol)). Assembly of stable, elongation-competent complexes by using a symmetrical primer-template substrate (sym/sub). J Biol Chem 275:5329–36." />
<meta name="citation_reference" content="Peliska JA, Benkovic SJ. 1994. Fidelity of in vitro DNA strand transfer reactions catalyzed by HIV-1 reverse transcriptase. Biochemistry 33:3890–5." />
<meta name="citation_reference" content="Peliska JA, Benkovic SJ. 1992. Mechanism of DNA strand transfer reactions catalyzed by HIV-1 reverse transcriptase. Science 258:1112–8." />
<meta name="citation_reference" content="Peliska JA, Balasubramanian S, Giedroc DP, Benkovic SJ. 1994. Recombinant HIV-1 nucleocapsid protein accelerates HIV-1 reverse transcriptase catalyzed DNA strand transfer reactions and modulates RNase H activity. Biochemistry 33:13817–23." />
<meta name="citation_reference" content="Castro C, Smidansky E, Maksimchuk KR, Arnold JJ, Korneeva VS, Gotte M, Konigsberg W, Cameron CE. 2007. Two proton transfers in the transition state for nucleotidyl transfer catalyzed by RNA-and DNA-dependent RNA and DNA polymerases. Proc Natl Acad Sci U S A 104:4267–72." />
<meta name="citation_reference" content="Parsley TB, Cornell CT, Semler BL. 1999. Modulation of the RNA binding and protein processing activities of poliovirus polypeptide 3CD by the viral RNA polymerase domain. J Biol Chem 274:12867–76." />
<meta name="citation_reference" content="Harris KS, Reddigari SR, Nicklin MJ, Hammerle T, Wimmer E. 1992. Purification and characterization of poliovirus polypeptide 3CD, a proteinase and a precursor for RNA polymerase. J Virol 66:7481–9." />
<meta name="citation_reference" content="Muller HJ. 1964. The Relation of Recombination to Mutational Advance. Mutat Res 106:2–9." />
<meta name="citation_reference" content="Felsenstein J. 1974. The evolutionary advantage of recombination. Genetics 78:737–56." />
<meta name="citation_reference" content="Chao L. 1990. Fitness of RNA virus decreased by Muller’s ratchet. Nature 348:454–5." />
<meta name="citation_reference" content="Kempf BJ, Peersen OB, Barton DJ. 2016. Poliovirus Polymerase Leu420 Facilitates RNA Recombination and Ribavirin Resistance. J Virol 90:8410–21." />
<meta name="citation_reference" content="Kempf BJ, Watkins CL, Peersen OB, Barton DJ. 2019. Picornavirus RNA Recombination Counteracts Error Catastrophe. J Virol 93." />
<meta name="citation_reference" content="Gohara DW, Ha CS, Kumar S, Ghosh B, Arnold JJ, Wisniewski TJ, Cameron CE. 1999. Production of “authentic” poliovirus RNA-dependent RNA polymerase (3D(pol)) by ubiquitin-protease-mediated cleavage in Escherichia coli. Protein Expr Purif 17:128–38." />
<meta name="citation_reference" content="Gohara DW, Crotty S, Arnold JJ, Yoder JD, Andino R, Cameron CE. 2000. Poliovirus RNA-dependent RNA polymerase (3Dpol): structural, biochemical, and biological analysis of conserved structural motifs A and B. J Biol Chem 275:25523–32." />
<meta name="citation_reference" content="Furio V, Moya A, Sanjuan R. 2005. The cost of replication fidelity in an RNA virus. Proc Natl Acad Sci U S A 102:10233–7." />
<meta name="citation_reference" content="Elena SF, Sanjuan R. 2005. Adaptive value of high mutation rates of RNA viruses: separating causes from consequences. J Virol 79:11555–8." />
<meta name="citation_reference" content="Furio V, Moya A, Sanjuan R. 2007. The cost of replication fidelity in human immunodeficiency virus type 1. Proc Biol Sci 274:225–30." />
<meta name="twitter:title" content="RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Using the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.

Significance The availability of a “universal” method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties." />
<meta name="og-title" property="og:title" content="RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/769224v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Using the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.

Significance The availability of a “universal” method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-09-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/769224v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/769224v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/769224v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/769224v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/907200" />
    <title>RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yiXDXbiHCNgQhXmuThs1Ns2IfcgKtjL6FE-WTFDApmU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"907200","apath":"\/biorxiv\/early\/2019\/09\/14\/769224.atom","pisa":"biorxiv;769224v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;769224v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/769224v1","title":"RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination","identifier":"node\/907200"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227n4O4ymkf8tLSgurlCzr7Bi3cGL9zhXKhCNHfkDrzlfqHBKnhqwV4B9V9LATAUcRVbtYigxHe8BMR95n9hyLwVAvElWBmMVEZoUvQE6kKlbaFYb9HaF\\\\\\\/+cEtDg+UqS3kV7cbQQNPrX+NVsssXkEBxWbpPhbu1vedJ0vIgzXe8hrcPTaWHwx5t1akFAttL\\\\\\\/bE2\\\\\\\/jJJvAlCQGMjzeTFiUf4+lBGjHX5mR6jUkKM+PpqBVmOisJ9ybqI4tzTD06rV2ooevY4VNA\\\\\\\/ZKz6oS6U1Yc2P6Qye4CM\\\\\\\/4WM3Dn8whWic9VTzAKeH8ZKwtFzmY\\\\\\\/Q9+Ii2CRd5kILz6Nl0AA\\\\\\\/h\\\\\\\/JT+NgI8Clo9vHc1rLX0YwTIN\\\\\\\/JmfYHnnVWhGQIlca5rCO6emM1eai9MMzSdeGW9BGUhHccvqa2i1qEN5EBp5H9ddhqJyNz+giBuiTBiKqn2OaST\\\\\\\/W+\\\\\\\/bEPaL2N6YXQDg78pJfB+pubs0YZo7qIDiHZlxpsVQ7NLjT3+VH0KV1jp9Zp4fFhmcdY5ZtshyLJcy0dbEEqNYK+KYsUIIihVBQPWzlEh429pN5fm6Jngm5rTdJ96yp9mugjqi+WxjlnzCk+1z0lN+nUD+lxOET7J\\\\\\\/mm5rFu8Mmjngn9hggVpGSYdE1QOJhwa1GmZd+gTFn3Ifdwgqsu\\\\\\\/1kp\\\\\\\/MczjWQTADB1SsLEhaPAw1DEtDXORnjubOubMs7eFs4sBhWL+gAn8lyd368FfxzvI6fSYAZ7+gF9vqJ0YTPqf42DOQFjpkm+JbWNEZWiN\\\\\\\/0DcJNA5vunZf3jvTHLcdKmTMwTiOSzMpNfI+uAd+xcq0rgYURC8oniBgqVo7KHgglUHmr6bZOiMdeEjr4jArzpZZmzA3pnb3oWwhc63o1ETVQ4Ayu+F\\\\\\\/kAlcF9\\\\\\\/xhgivif2TSNGtaPr\\\\\\\/Jeofg1tPHAA0GuHYkQR3Qq2fkYvY7MZA4EBfjEtbAeaeO0Z+ZGZauxwh987OijzmThZUpNdtNVeFpnLYrIXrrNbCwSS7XMAmn\\\\\\\/QS3Uj3lhScgtN7sh16FjjXwRVvkd\\\\\\\/xV3mc\\\\\\\/4XbK3cBPbvgWh2YORQ6dLunmsvlyXjrkqyzUYGO7m1oUg8PfJWfpKiO8jAQKZpG5d2REA\\\\\\\/bkp6k7Bd\\\\\\\/QvGC12F8moGIWX6wJnwTxM1s2tNsOUZXKONNZlSwBtsChqTXa1WulWXA4eSYxtb4ywximQBKME0PQBAamg\\\\\\\/XPmRBj\\\\\\\/kYon72cTYccWqrvWZhOWKWWA72eM6iOgeyb4BBh0rzIBR8yYKlqBZJQzbNMUXnik3EIML5YRN4d\\\\\\\/xXe22xftIuCknK97EiKBzi1gcua6e4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022h8bBMzcFmFW+lQW01IH8UQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002270f607d98c3439b5f39b5e365c4c3eebbbd3a89c7fa278e6904e4f60b95a0734\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/769224v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-907200 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/769224v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C0SO0lUQkYMRL7VKLHxim3PX9J4zppr7_yoLiS-Zytg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="907200" id="node907200" data-pisa="biorxiv;769224v1" data-pisa-master="biorxiv;769224" data-apath="/biorxiv/early/2019/09/14/769224.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hyejeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Victor D.</span> <span class="nlm-surname">Ellis</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7757-5492" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Woodman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jamie J.</span> <span class="nlm-surname">Arnold</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Craig E.</span> <span class="nlm-surname">Cameron</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/769224 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node907200" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hyejeong Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyejeong%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyejeong%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Victor D. Ellis <span class="nlm-suffix">III</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BD.%2BEllis%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EIII%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ellis%20VD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictor%2BD.%2BEllis%2BIII" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Woodman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BWoodman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woodman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BWoodman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7757-5492" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew Woodman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jamie J. Arnold </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BJ.%2BArnold%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arnold%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJamie%2BJ.%2BArnold%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Craig E. Cameron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, The Pennsylvania State University</span>, 201 Althouse Laboratory, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BE.%2BCameron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cameron%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BE.%2BCameron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:craig.cameron@med.unc.edu" class="" data-icon-position="" data-hide-link-title="0">craig.cameron@med.unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/769224v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:907200" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:907200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/769224v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:907200" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:907200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/769224v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:907200" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:907200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/769224v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:907200" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:907200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/769224v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:907200" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:907200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Using the RNA-dependent RNA polymerase (RdRp) from poliovirus (PV) as our model system, we have shown that Lys-359 in motif-D functions as a general acid in the mechanism of nucleotidyl transfer. A K359H (KH) RdRp derivative is slow and faithful relative to wild-type enzyme. In the context of the virus, RdRp-coding sequence evolves, selecting for the following substitutions: I331F (IF, motif-C) and P356S (PS, motif-D). We have evaluated IF-KH, PS-KH, and IF-PS-KH viruses and enzymes. The speed and fidelity of each double mutant are equivalent. Each exhibits a unique recombination phenotype, with IF-KH being competent for copy-choice recombination and PS-KH being competent for forced-copy-choice recombination. Although the IF-PS-KH RdRp exhibits biochemical properties within twofold of wild type, the virus is impaired substantially for recombination in cells. We conclude that there are biochemical properties of the RdRp in addition to speed and fidelity that determine the mechanism and efficiency of recombination. The interwoven nature of speed, fidelity, the undefined property suggested here, and recombination makes it impossible to attribute a single property of the RdRp to fitness. However, the derivatives described here may permit elucidation of the importance of recombination on the fitness of the viral population in a background of constant polymerase speed and fidelity.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Significance</strong> The availability of a “universal” method to create attenuated viruses for use as vaccine strains would permit a rapid response to outbreaks of newly emerging viruses. Targeting RdRp fidelity has emerged as such a universal approach. However, because polymerase fidelity and speed are inextricably linked, the effort to attribute the attenuated phenotype to a single biochemical property of the RdRp may be futile. Here, we show that this circumstance is even more complex. We provide evidence for the existence of a biochemical parameter that combines with fidelity and speed to govern the mechanism and/or efficiency of recombination. We conclude that the field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp instead of debating the importance of individual properties.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-5">In spite of the substantial resources that have been allocated by the National Institute of Allergy and Infectious Diseases and the Centers for Disease Control and Prevention to support prediction of emerging viral pathogens (<a href="https://www.niaid.nih.gov/research/emerging-infectious-diseases-pathogens">https://www.niaid.nih.gov/research/emerging-infectious-diseases-pathogens</a>), viral outbreaks over the past few decades have been caused by viruses for which surveillance was not considered a priority. Rapid response to an outbreak caused by an unexpected viral pathogen requires, minimally, the existence of broad-spectrum, antiviral therapeutics. Prevention requires the availability of vaccines, development of which could take years. Indeed, approved vaccines still do not exist to prevent infections by West Nile virus or severe acute respiratory syndrome (SARS) coronavirus, and these outbreaks occurred more than one decade ago.</p><p id="p-6">Because all RNA viruses encode an RNA-dependent RNA polymerase (RdRp) with conserved function and mechanism, this enzyme has emerged as an attractive target for development of broad-spectrum therapeutics (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>) and a target for function/mechanism-based strategies for viral attenuation (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). One function/mechanism of the RdRp that has been targeted most is that required for faithful incorporation of nucleotides (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>). Changing RdRp fidelity decreases or increases the genetic variation of the viral population, which, in turn, decreases fitness and virulence of the viral population (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>).</p><p id="p-7">Known RdRp variants exhibiting a high-fidelity phenotype can also exhibit a reduced speed of nucleotide incorporation, at least at the biochemical level (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). Replication speed is also a determinant of viral fitness and virulence (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). So, is it the increased fidelity or decreased speed of nucleotide addition that gives rise to the attenuated phenotype? Further complicating the fidelity-versus-speed question are the recent observations that changes to fidelity also have consequence for the efficiency of recombination (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). Increased RdRp fidelity decreases recombination efficiency and vice versa (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). It will likely be impossible to attribute a single biochemical property of the RdRp to biological outcome.</p><p id="p-8">The most extensively studied PV fidelity mutants encode RdRps with amino acid substitutions at sites remote from the active site. Constructing equivalent substitutions conferring equivalent phenotypes in RdRps other than PV is difficult if not impossible. Our laboratory has therefore pursued active-site-based strategies to manipulate the fidelity, speed, and/or recombination efficiency of the RdRp (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). We have shown that a lysine (Lys-359 in PV) present in conserved structural motif D of the RdRp contributes to the efficiency of nucleotidyl transfer (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>). A PV mutant encoding a K359R RdRp is attenuated but elicits a protective immune response in mice that is at least as robust as the immune response elicited by the type 1 Sabin vaccine strain (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>).</p><p id="p-9">In this study, we characterize a second motif-D mutant of PV (K359H). Unlike the K359R RdRp-encoding virus (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>), K359H PV is genetically unstable and acquires mutations encoding two second-site amino acid substitutions after a few passages in cell culture. Together, the two substitutions restore all biochemical properties of the derivative to near wild-type levels. Individually, however, we observe differences in the mechanism (copy-choice vs. forced-copy-choice) and efficiency of recombination by each derivative, although each derivative exhibits equivalent speed and fidelity. We conclude that biochemical properties in addition to speed and fidelity must exist and contribute to both the mechanism and efficiency of recombination. The desire to attribute single, biochemical properties of the viral RdRp to fitness, virulence, and pathogenesis may be futile.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>K359H PV requires two second-site substitutions to restore a “wild-type” growth phenotype</h3><p id="p-10">Most studies of RdRp fidelity have benefited from the selection of derivatives that were either more or less sensitive to a mutagenic nucleoside (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Almost invariably, the derivatives changed residues remote from the catalytic site, thus using an allosteric mechanism to perturb fidelity. Among the most famous of these is the G64S substitution in PV RdRp (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). Many years ago, our laboratory showed that the active site of all polymerases contain a general acid, Lys-359 in the case of PV RdRp, that protonated the pyrophosphate leaving group during nucleotidyl transfer, thereby increasing the catalytic efficiency of RdRp (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>). Substitutions of Lys-359 in PV RdRp give rise to changes in fidelity (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>). The arginine substitution of Lys-359 (K359R RdRp) and the histidine substitution (K359H RdRp) catalyze nucleotidyl transfer at a rate 10-fold lower than wild type (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a>). The impact of the arginine substitution of Lys-359 on PV fidelity and its potential application to vaccine development have been described (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>).</p><p id="p-11">Biological studies of K359H PV were not reported, because this virus was not genetically stable. When PV is rescued from in vitro transcribed RNA, four passages are required for the genetic diversity of the viral population to come to equilibrium (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). During this time for K359H PV, mutations were observed in RdRp-coding sequence that changed the plaque phenotype of the virus. One change was in motif C, I331F, and the other change was in motif D, P356S (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figs. 1A</a></strong> and <strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">1B</a></strong>). In order to compare the impact of these substitutions on PV genome replication without complication of reversion, we engineered the various substitutions into a subgenomic replicon, producing luciferase as an indirect measure of viral RNA produced. Experiments in the presence of guanidine hydrochloride (GuHCl) report on translation of transfected RNA in the absence of replication (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a></strong>). Relative to WT PV, K359H was the most debilitated, with replication failing to reach an end point at 10 h post-transfection (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a></strong>). Each double mutant was markedly better than K359H alone, but the addition of I331F conferred a greater replication advantage than P356S (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a></strong>). The triple mutant replicated even faster but still exhibited a significant reduction in the rate of replication relative to WT (<strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1C</a></strong>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Serial passage of K359H PV in cell culture leads to additional changes to RdRp-coding sequence. (A) Changes in RdRp-coding sequence after serial passage of K359H PV. RNA was isolated from passage 2, converted to cDNA, and amplified by PCR. The electropherogram of the sequenced PCR product is shown, revealing two amino acid changes: I331F (motif C) and P356S (motif D). (B) Location of I331, P356, and K359 in structure of PV RdRp. Palm, fingers, and thumb subdomains are shown. Conserved structural motifs are colored: A, red; B, green; C, yellow; D, blue; E, purple; F, orange; G, black. (C) Replication phenotypes observed for K359H PV and KH-containing PV mutants by using a replicon assay. Replication was monitored using a subgenomic replicon firefly luciferase. Luciferase specific activity is reported in relative light units (RLU) per microgram of total protein in the extract as a function of time post-transfection. Shown is one representative data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Serial passage of K359H PV in cell culture leads to additional changes to RdRp-coding sequence.&lt;/span&gt; (&lt;strong&gt;A&lt;/strong&gt;) Changes in RdRp-coding sequence after serial passage of K359H PV. RNA was isolated from passage 2, converted to cDNA, and amplified by PCR. The electropherogram of the sequenced PCR product is shown, revealing two amino acid changes: I331F (motif C) and P356S (motif D). (&lt;strong&gt;B&lt;/strong&gt;) Location of I331, P356, and K359 in structure of PV RdRp. Palm, fingers, and thumb subdomains are shown. Conserved structural motifs are colored: A, &lt;em&gt;red&lt;/em&gt;; B, &lt;em&gt;green&lt;/em&gt;; C, &lt;em&gt;yellow&lt;/em&gt;; D, &lt;em&gt;blue&lt;/em&gt;; E, &lt;em&gt;purple&lt;/em&gt;; F, &lt;em&gt;orange&lt;/em&gt;; G, &lt;em&gt;black&lt;/em&gt;. (&lt;strong&gt;C&lt;/strong&gt;) Replication phenotypes observed for K359H PV and KH-containing PV mutants by using a replicon assay. Replication was monitored using a subgenomic replicon firefly luciferase. Luciferase specific activity is reported in relative light units (&lt;em&gt;RLU&lt;/em&gt;) per microgram of total protein in the extract as a function of time post-transfection. Shown is one representative data set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F1.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F1.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Serial passage of K359H PV in cell culture leads to additional changes to RdRp-coding sequence.</span><p id="p-12" class="first-child">(<strong>A</strong>) Changes in RdRp-coding sequence after serial passage of K359H PV. RNA was isolated from passage 2, converted to cDNA, and amplified by PCR. The electropherogram of the sequenced PCR product is shown, revealing two amino acid changes: I331F (motif C) and P356S (motif D). (<strong>B</strong>) Location of I331, P356, and K359 in structure of PV RdRp. Palm, fingers, and thumb subdomains are shown. Conserved structural motifs are colored: A, <em>red</em>; B, <em>green</em>; C, <em>yellow</em>; D, <em>blue</em>; E, <em>purple</em>; F, <em>orange</em>; G, <em>black</em>. (<strong>C</strong>) Replication phenotypes observed for K359H PV and KH-containing PV mutants by using a replicon assay. Replication was monitored using a subgenomic replicon firefly luciferase. Luciferase specific activity is reported in relative light units (<em>RLU</em>) per microgram of total protein in the extract as a function of time post-transfection. Shown is one representative data set.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Characterization of the double- and triple-mutant viruses and their polymerases</h3><p id="p-13">At multiplicities of infection of one or higher, the double- and triple-mutant viruses were stable for multiple passages, thus permitting us to characterize the biological properties of these viruses. We have adopted the following nomenclature to refer to the various mutant PVs and their corresponding RdRps: I331F-K359H, IF-KH; P356S-K359H, PS-KH; and I331F-P356S-K359H, IF-PS-KH. Interestingly, the growth properties of the viruses were not as expected based on the experiments with the replicon. Each mutant virus exhibited the same delay relative to WT virus prior to the first detection of infectious virus after four hours post-infection (hpi) (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a></strong>). Thereafter, growth of each virus was far more distinct than observed for the replicon, with PS-KH PV much slower than IF-KH PV and IF-PS-KH PV was substantially faster than both double mutants (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a></strong>). The differences in outcome could reflect the differences in replication efficiency and/or a direct consequence of the substitutions in 3D-coding sequence on virus assembly or spread caused by changes to 3CD protein or 3D-containing precursor protein (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biological and biochemical characterization of variants of K359H PV and the RdRp reveal that both substitutions are required to achieve a near wild-type phenotype. (A) One-step growth analysis. Cells were infected at MOI 10 with the following PVs: WT, IF-KH, PS-KH, and IF-PS-KH. Viral titer (pfu/mL) was plotted as a function of time post-infection. Duplicate infected samples were used for plaque assays. Error bars indicate SEM (n = 2). (B) Specific infectivity. Virus was isolated 24 h post-infection and used for qRT-PCR to determine genomes/mL or plaque assay to determine pfu/mL, with the quotient yielding specific infectivity, genomes/pfu. (C) Ribavirin sensitivity. HeLa cells were infected at a MOI 0.1 with each PV in the presence of 600 &#xB5;M ribavirin. After a 24-h incubation at 37 &#xB0;C, virus was isolated and used for plaque assay. Indicated is the titer of virus recovered in the presence of ribavirin normalized to that recovered in the absence of ribavirin. Solid bar indicates the mean of each virus yield. Error bars indicate SEM (n = 3). (D) Biochemical analysis. The rate constant for incorporation of a single correct nucleotide, ATP, (kpol,corr)) and a single incorrect nucleotide, GTP, (kpol,incorr) by each PV RdRp was performed as previously described (17). Data are reported using one significant figure. The error reported is the standard error from the fit of the data to a single exponential (17)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Biological and biochemical characterization of variants of K359H PV and the RdRp reveal that both substitutions are required to achieve a near wild-type phenotype.&lt;/span&gt; &lt;strong&gt;(A&lt;/strong&gt;) One-step growth analysis. Cells were infected at MOI 10 with the following PVs: WT, IF-KH, PS-KH, and IF-PS-KH. Viral titer (pfu/mL) was plotted as a function of time post-infection. Duplicate infected samples were used for plaque assays. Error bars indicate SEM (&lt;em&gt;n = 2&lt;/em&gt;). &lt;strong&gt;(B)&lt;/strong&gt; Specific infectivity. Virus was isolated 24 h post-infection and used for qRT-PCR to determine genomes/mL or plaque assay to determine pfu/mL, with the quotient yielding specific infectivity, genomes/pfu. &lt;strong&gt;(C)&lt;/strong&gt; Ribavirin sensitivity. HeLa cells were infected at a MOI 0.1 with each PV in the presence of 600 &#xB5;M ribavirin. After a 24-h incubation at 37 &#xB0;C, virus was isolated and used for plaque assay. Indicated is the titer of virus recovered in the presence of ribavirin normalized to that recovered in the absence of ribavirin. Solid bar indicates the mean of each virus yield. Error bars indicate SEM (&lt;em&gt;n = 3&lt;/em&gt;). (&lt;strong&gt;D&lt;/strong&gt;) Biochemical analysis. The rate constant for incorporation of a single correct nucleotide, ATP, (&lt;em&gt;k&lt;/em&gt;&lt;sub&gt;pol,corr&lt;/sub&gt;)) and a single incorrect nucleotide, GTP, (&lt;em&gt;k&lt;/em&gt;&lt;sub&gt;pol,incorr&lt;/sub&gt;) by each PV RdRp was performed as previously described (17). Data are reported using one significant figure. The error reported is the standard error from the fit of the data to a single exponential (17).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F2.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F2.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Biological and biochemical characterization of variants of K359H PV and the RdRp reveal that both substitutions are required to achieve a near wild-type phenotype.</span><p id="p-14" class="first-child"><strong>(A</strong>) One-step growth analysis. Cells were infected at MOI 10 with the following PVs: WT, IF-KH, PS-KH, and IF-PS-KH. Viral titer (pfu/mL) was plotted as a function of time post-infection. Duplicate infected samples were used for plaque assays. Error bars indicate SEM (<em>n = 2</em>). <strong>(B)</strong> Specific infectivity. Virus was isolated 24 h post-infection and used for qRT-PCR to determine genomes/mL or plaque assay to determine pfu/mL, with the quotient yielding specific infectivity, genomes/pfu. <strong>(C)</strong> Ribavirin sensitivity. HeLa cells were infected at a MOI 0.1 with each PV in the presence of 600 µM ribavirin. After a 24-h incubation at 37 °C, virus was isolated and used for plaque assay. Indicated is the titer of virus recovered in the presence of ribavirin normalized to that recovered in the absence of ribavirin. Solid bar indicates the mean of each virus yield. Error bars indicate SEM (<em>n = 3</em>). (<strong>D</strong>) Biochemical analysis. The rate constant for incorporation of a single correct nucleotide, ATP, (<em>k</em><sub>pol,corr</sub>)) and a single incorrect nucleotide, GTP, (<em>k</em><sub>pol,incorr</sub>) by each PV RdRp was performed as previously described (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>). Data are reported using one significant figure. The error reported is the standard error from the fit of the data to a single exponential (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Our studies of RdRp fidelity mutants in cell culture have highlighted the fact that the use of the plaque-forming unit (PFU) as a measure of virus concentration will mask changes in the specific infectivity of the viral RNA (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>). As a result, our experiments generally use genomes instead of PFU as the unit of measure (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>). In doing so, we can use genomes/PFU as a measure of the specific infectivity of the virus and surrogate for virus fitness. The instability of KH PV precluded rigorous, quantitative analysis of this virus, but at least 10-fold more KH PV than WT PV was required to observe a comparable number of plaques. Both substitutions were able to increase the efficiency of plaque formation relative to KH PV (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a></strong>). Indeed, the specific infectivity of IF-KH PV was equivalent to WT. The addition of PS to IF-KH reduced the efficiency of plaque formation; the specific infectivity of IF-PS-KH PV was reduced by twofold relative to IF-KH PV (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a></strong>). Therefore, the exaggerated behavior of PS-KH PV in the PFU-based growth assay relative to the replicon assay likely reflects an additional defect to virus assembly and/or spread.</p><p id="p-16">A change to the specific infectivity of the viral RNA caused by increased mutational load—that is, reduced fidelity of the PS-KH RdRp, would also explain the observed reduction in the specific infectivity of PS-KH PV relative to the other mutant PVs. To test this possibility, we evaluated the sensitivity of each virus population to growth in the presence of ribavirin. We reasoned that the mutagenic activity of ribavirin would exhibit the greatest negative impact on viruses with a mutator phenotype as described previously (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>). Relative to WT, K359H PV exhibited the highest fidelity based on the 2-log difference in sensitivity to ribavirin (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2C</a></strong>). IF-KH and PS-KH PVs also exhibited a higher fidelity than WT PV (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2C</a></strong>). Importantly, both mutants exhibited essentially equivalent fidelity phenotypes (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2C</a></strong>), consistent with the suggestion above that the reduced efficiency of plaque formation observed for PS-KH PV is likely related to impairment of virus assembly and/or spread. Finally, evaluation of IF-PS-KH PV revealed equivalent sensitivity of this mutant to ribavirin as observed for WT PV (<strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2C</a></strong>), suggesting that fidelity had been returned to normal.</p><p id="p-17">We have previously reported biochemical properties of the RdRps for the repertoire of mutant viruses described above (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>). Consistent with these reports, each substitution increases the speed and reduces the fidelity to create a biochemical phenotype on par with that observed for WT (<strong><a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2D</a></strong>).</p></div><div id="sec-6" class="subsection"><h3>Biochemical properties of the RdRp other than speed and fidelity contribute to the efficiency of recombination in cell culture</h3><p id="p-18">Based on the myriad RdRp fidelity mutants that have been reported to date, there appears to be a direct correlation between the rate of nucleotide addition (speed) and the fidelity of nucleotide addition (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). This correlation also extends to recombination efficiency (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). If this is the case, then the ongoing debate of speed versus fidelity as the key determinant of viral fitness will become even more complicated to resolve (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>).</p><p id="p-19">The current state of the art for evaluation of recombination in cell culture is based on the co-transfection of two viral (sub)genomic RNAs incapable of producing infectious virus (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). The donor RNA is a replication-competent, subgenomic RNA that encodes a luciferase reporter instead of the viral capsid (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a></strong>). The acceptor RNA is a replication-incompetent, genomic RNA that has a defective cis-acting replication element, termed oriI (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a></strong>) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). Initiation of replication on the donor followed by a switch to the acceptor at a site after the oriI locus will yield an infectious genome that can be scored by plaque assay (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3A</a></strong>) (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Parameters in addition to RdRp speed and fidelity contribute to recombination efficiency. (A) Schematic of the PV recombination assay used (34). Two RNAs are used: a replication-competent subgenomic RNA lacking capsid-coding sequence (donor RNA); and a replication-incompetent full-length genomic RNA with a defective cis-acting replication element (CRE, indicated by hairpin with defective version indicated by X) in 2C-coding sequence (acceptor RNA). Co-transfection of these RNAs produces infectious virus if recombination occurs. (B) The indicated PV RdRp was engineered into both donor and accepter RNA and co-transfected into a L929 mouse fibroblast cell line. Infectious virus produced by recombination in L929 cells was determined by plaque assay using HeLa cells. Each point shown is an independent experiment reflecting the average of three replicates. When plaques could not be detected, n.d. is indicated. Mean and SEM (n=5) are indicated. (C) Schematic of a modified PV recombination assay using an acceptor RNA expressing the green fluorescent protein, UnaG (35). Recombinants are scored by expression of green fluorescence instead of plaques, thereby increasing the sensitivity. (D) Infectious virus produced by recombination in L929 was scored for infectious virus in HeLa cells 7, 24, and 48 h post-infection for WT and at 48 h only for K359H and PS-KH." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Parameters in addition to RdRp speed and fidelity contribute to recombination efficiency.&lt;/span&gt; &lt;strong&gt;(A&lt;/strong&gt;) Schematic of the PV recombination assay used (34). Two RNAs are used: a replication-competent subgenomic RNA lacking capsid-coding sequence (donor RNA); and a replication-incompetent full-length genomic RNA with a defective cis-acting replication element (CRE, indicated by hairpin with defective version indicated by X) in 2C-coding sequence (acceptor RNA). Co-transfection of these RNAs produces infectious virus if recombination occurs. &lt;strong&gt;(B&lt;/strong&gt;) The indicated PV RdRp was engineered into both donor and accepter RNA and co-transfected into a L929 mouse fibroblast cell line. Infectious virus produced by recombination in L929 cells was determined by plaque assay using HeLa cells. Each point shown is an independent experiment reflecting the average of three replicates. When plaques could not be detected, n.d. is indicated. Mean and SEM (n=5) are indicated. &lt;strong&gt;(C&lt;/strong&gt;) Schematic of a modified PV recombination assay using an acceptor RNA expressing the green fluorescent protein, UnaG (35). Recombinants are scored by expression of green fluorescence instead of plaques, thereby increasing the sensitivity. &lt;strong&gt;(D&lt;/strong&gt;) Infectious virus produced by recombination in L929 was scored for infectious virus in HeLa cells 7, 24, and 48 h post-infection for WT and at 48 h only for K359H and PS-KH.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F3.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F3.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Parameters in addition to RdRp speed and fidelity contribute to recombination efficiency.</span><p id="p-20" class="first-child"><strong>(A</strong>) Schematic of the PV recombination assay used (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>). Two RNAs are used: a replication-competent subgenomic RNA lacking capsid-coding sequence (donor RNA); and a replication-incompetent full-length genomic RNA with a defective cis-acting replication element (CRE, indicated by hairpin with defective version indicated by X) in 2C-coding sequence (acceptor RNA). Co-transfection of these RNAs produces infectious virus if recombination occurs. <strong>(B</strong>) The indicated PV RdRp was engineered into both donor and accepter RNA and co-transfected into a L929 mouse fibroblast cell line. Infectious virus produced by recombination in L929 cells was determined by plaque assay using HeLa cells. Each point shown is an independent experiment reflecting the average of three replicates. When plaques could not be detected, n.d. is indicated. Mean and SEM (n=5) are indicated. <strong>(C</strong>) Schematic of a modified PV recombination assay using an acceptor RNA expressing the green fluorescent protein, UnaG (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Recombinants are scored by expression of green fluorescence instead of plaques, thereby increasing the sensitivity. <strong>(D</strong>) Infectious virus produced by recombination in L929 was scored for infectious virus in HeLa cells 7, 24, and 48 h post-infection for WT and at 48 h only for K359H and PS-KH.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We evaluated our panel of KH-containing PV mutants using this assay, and the outcomes were, in most cases, quite unexpected (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a></strong>). KH PV was unable to produce viable recombinants, as expected for a high-fidelity RdRp (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>). The first surprise was that IF-KH and PS-KH PVs did not exhibit the same phenotypes (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3B</a></strong>). While IF-KH PV produced viable recombinant virus, PS-KH PV did not (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3B</a></strong>). Based on the experiments performed above, these viruses and their polymerases replicate with comparable efficiency and fidelity (<strong><a id="xref-fig-1-7" class="xref-fig" href="#F1">Figs. 1A</a>, <a id="xref-fig-2-10" class="xref-fig" href="#F2">2C</a></strong>, and <strong><a id="xref-fig-2-11" class="xref-fig" href="#F2">2D</a></strong>). Even more surprising, however, was the observation that IF-PS-KH PV was impaired more than one log relative to WT PV in its ability to produce viable recombinants (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3B</a></strong>). The replication efficiency and fidelity of this virus and its polymerase were always within twofold of that observed for WT (<strong><a id="xref-fig-1-8" class="xref-fig" href="#F1">Figs. 1A</a>, <a id="xref-fig-2-12" class="xref-fig" href="#F2">2C</a></strong>, and <strong><a id="xref-fig-2-13" class="xref-fig" href="#F2">2D</a></strong>). Together, these results suggest that there are properties of the PV RdRp important for recombination that are not revealed by our existing biological and biochemical assays.</p><p id="p-22">The one caveat of the recombination assay is that the recombinant viruses produced must be able to form plaques, which may mean that the recombinant virus must spread by a lytic mechanism. It was possible that PS-KH PV was defective for virus assembly and/or spread, at least lytic spread (<strong><a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2B</a></strong>). In order to score for recombinant viruses that spread by either a lytic or non-lytic mechanism, we engineered the acceptor template to encode the UnaG (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>) green fluorescent protein (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3C</a></strong>). The polyprotein was designed such that UnaG protein is released from capsid precursor protein by 3C protease activity. Observation of green cells were visible from a donor-acceptor pair producing WT RdRp as early as 7 hpi, with increases continuing over a 48-h period (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3D</a></strong>). With the sensitivity of this assay and the absence of a requirement for lytic spread, it is clear that PS-KH PV exhibits a recombination defect (<strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3D</a></strong>).</p></div><div id="sec-7" class="subsection"><h3>Poly(rU) polymerase activity as a predictor of the efficiency of copy-choice recombination in cell culture</h3><p id="p-23">Twenty years ago, we showed that template switching was the primary mechanism of product formation when oligo(dT) or oligo(rU) were used to prime poly(rU) RNA synthesis on oligo(rA) or poly(rA) RNA templates (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). Briefly, polymerase engages a primed template, elongates that primer, disengages from the first template at some point during the elongation process to engage a second (acceptor) template and continue RNA synthesis (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a></strong>) (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>). This process occurs reiteratively, yielding products that are much greater than the average length of template used in the reaction (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RdRp derivatives incapable of recombination in cells exhibit defects in copy-choice recombination (template switching) in vitro. (A) Products of reactions using oligo/poly(rA) as template reflect copy-choice recombination, which requires the template-switching activity of the RdRp (36). RdRp engages primed template and initiates RNA synthesis. The RdRp-nascent RNA complex moves to new template from internal positions or as a consequence of reaching the end of template, thus creating greater-than-template-length products. (B) Specific poly(rU) polymerase activity (pmol/min/&#xB5;g) of each RdRp. WT and the indicated PV RdRp variants assessed by using a poly(rU) polymerase activity assay. Error bars represent SEM (n = 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;RdRp derivatives incapable of recombination in cells exhibit defects in copy-choice recombination (template switching) in vitro.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Products of reactions using oligo/poly(rA) as template reflect copy-choice recombination, which requires the template-switching activity of the RdRp (36). RdRp engages primed template and initiates RNA synthesis. The RdRp-nascent RNA complex moves to new template from internal positions or as a consequence of reaching the end of template, thus creating greater-than-template-length products. &lt;strong&gt;(B)&lt;/strong&gt; Specific poly(rU) polymerase activity (pmol/min/&#xB5;g) of each RdRp. WT and the indicated PV RdRp variants assessed by using a poly(rU) polymerase activity assay. Error bars represent SEM (&lt;em&gt;n = 3&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F4.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F4.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">RdRp derivatives incapable of recombination in cells exhibit defects in copy-choice recombination (template switching) in vitro.</span><p id="p-24" class="first-child"><strong>(A)</strong> Products of reactions using oligo/poly(rA) as template reflect copy-choice recombination, which requires the template-switching activity of the RdRp (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a>). RdRp engages primed template and initiates RNA synthesis. The RdRp-nascent RNA complex moves to new template from internal positions or as a consequence of reaching the end of template, thus creating greater-than-template-length products. <strong>(B)</strong> Specific poly(rU) polymerase activity (pmol/min/µg) of each RdRp. WT and the indicated PV RdRp variants assessed by using a poly(rU) polymerase activity assay. Error bars represent SEM (<em>n = 3</em>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The observation that only a subset of the KH mutants were competent for recombination, presumably by a copy-choice mechanism (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), provided the opportunity to determine the extent to which the poly(rU) polymerase activity can predict biological phenotypes. The congruence between the two experiments was remarkable (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a></strong>). Both KH and PS-KH RdRps were impaired for poly(rU) polymerase activity, both IF-KH and IF-PS-KH RdRps exhibited near-WT levels of poly(rU) polymerase activity (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a></strong>). These results further confirm template switching as the primary mechanism of poly(rU) polymerase activity and validate this assay as a screen for identification of RdRps with deficits in template switching.</p></div><div id="sec-8" class="subsection"><h3>Polymerase determinants supporting efficient copy-choice recombination do not overlap completely with determinants supporting efficient forced-copy-choice recombination</h3><p id="p-26">Recently, our laboratory developed an assay for forced-copy-choice recombination (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), inspired by an analogous assay developed to study template switching by the reverse transcriptase from human immunodeficiency virus (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>-<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). The concept of the assay and templates used are presented in <strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a></strong>. The experimental design is diagrammed in <strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a></strong>. Polymerase assembles on the primed template (sym/sub-U). In the presence of only the first nucleotide (ATP), the assembled complexes can be identified as the one-nucleotide-extended product (n+1). Addition of the remaining nucleotides in the absence or presence of the acceptor template will yield a strong-stop RNA product. In the presence of a complementary acceptor RNA, the strong-strop RNA product (donor) will be extended, creating a transfer product.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Properties of the RdRp required for copy-choice recombination in vitro are separable from those required for forced-copy-choice recombination in vitro. (A) The heteropolymeric, symmetrical, primed-template substrate (sym/sub) has been used to establish an assay for forced-copy-choice recombination in vitro (22, 38). RdRp assembles on sym/sub. ATP is added and incorporated to yield a stable elongation complex. It is this elongation complex that is monitored for extension and transfer. Transfer is strictly dependent on the presence of an acceptor RNA with complementarity to the 3&#x2019;-end of donor RNA. (B) Schematic of the experimental design is indicated. Products in boxes are those observed and monitored by denaturing polyacrylamide gel electrophoresis. (C) Reaction products were resolved by electrophoresis and detected by phosphorimaging. The only regions of the gel with bands are shown; these correspond to the sym/sub primer, one-nucleotide-extended primer (n+1), four-nucleotides-extended product (strong stop) and non-templated addition of nucleotides to that product, and the transfer product. (D) Transfer products were quantified and are expressed as a percentage relative to the value observed for WT (The concentrations (&#xB5;M) of n+1 formed for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.22, 0.22, 0.32, 0.36 and 0.37, respectively. The concentrations (&#xB5;M) of transfer product for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.011, 0.00011, 0.0016, 0.0072 and 0.01, respectively. Overall, transfer efficiency ranges from 1% to 5%. Error bars represent SEM (n = 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Properties of the RdRp required for copy-choice recombination in vitro are separable from those required for forced-copy-choice recombination in vitro.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; The heteropolymeric, symmetrical, primed-template substrate (sym/sub) has been used to establish an assay for forced-copy-choice recombination in vitro (22, 38). RdRp assembles on sym/sub. ATP is added and incorporated to yield a stable elongation complex. It is this elongation complex that is monitored for extension and transfer. Transfer is strictly dependent on the presence of an acceptor RNA with complementarity to the 3&#x2019;-end of donor RNA. &lt;strong&gt;(B)&lt;/strong&gt; Schematic of the experimental design is indicated. Products in boxes are those observed and monitored by denaturing polyacrylamide gel electrophoresis. (&lt;strong&gt;C&lt;/strong&gt;) Reaction products were resolved by electrophoresis and detected by phosphorimaging. The only regions of the gel with bands are shown; these correspond to the sym/sub primer, one-nucleotide-extended primer (n+1), four-nucleotides-extended product (strong stop) and non-templated addition of nucleotides to that product, and the transfer product. &lt;strong&gt;(D)&lt;/strong&gt; Transfer products were quantified and are expressed as a percentage relative to the value observed for WT (The concentrations (&#xB5;M) of n+1 formed for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.22, 0.22, 0.32, 0.36 and 0.37, respectively. The concentrations (&#xB5;M) of transfer product for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.011, 0.00011, 0.0016, 0.0072 and 0.01, respectively. Overall, transfer efficiency ranges from 1% to 5%. Error bars represent SEM (n = 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F5.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F5.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Properties of the RdRp required for copy-choice recombination in vitro are separable from those required for forced-copy-choice recombination in vitro.</span><p id="p-27" class="first-child"><strong>(A)</strong> The heteropolymeric, symmetrical, primed-template substrate (sym/sub) has been used to establish an assay for forced-copy-choice recombination in vitro (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>). RdRp assembles on sym/sub. ATP is added and incorporated to yield a stable elongation complex. It is this elongation complex that is monitored for extension and transfer. Transfer is strictly dependent on the presence of an acceptor RNA with complementarity to the 3’-end of donor RNA. <strong>(B)</strong> Schematic of the experimental design is indicated. Products in boxes are those observed and monitored by denaturing polyacrylamide gel electrophoresis. (<strong>C</strong>) Reaction products were resolved by electrophoresis and detected by phosphorimaging. The only regions of the gel with bands are shown; these correspond to the sym/sub primer, one-nucleotide-extended primer (n+1), four-nucleotides-extended product (strong stop) and non-templated addition of nucleotides to that product, and the transfer product. <strong>(D)</strong> Transfer products were quantified and are expressed as a percentage relative to the value observed for WT (The concentrations (µM) of n+1 formed for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.22, 0.22, 0.32, 0.36 and 0.37, respectively. The concentrations (µM) of transfer product for WT, K359H, IF-KH, PS-KH and IF-PS-KH are 0.011, 0.00011, 0.0016, 0.0072 and 0.01, respectively. Overall, transfer efficiency ranges from 1% to 5%. Error bars represent SEM (n = 2).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We evaluated each KH-containing RdRp derivative in this assay. Substrate and product analysis are presented in <strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5C</a></strong>, with the quantitation of the transfer product relative to WT presented in <strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5D</a></strong>. KH RdRp was defective in this assay as well, as transfer product was not detected (<strong><a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5C</a></strong>). Importantly, the failure to transfer was not a reflection of the inability to assemble or produce strong-stop donor RNA (<strong><a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5C</a></strong>). In this assay, PS-KH RdRp outperformed IF-KH RdRp, and PS-KH and IF-PS-KH RdRp were essentially identical in activity.</p><p id="p-29">We conclude that copy-choice and forced-copy-choice recombination use distinct mechanisms, requiring unique biochemical properties of the RdRp.</p></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-30">For more than a decade now, our laboratory and others have published studies asserting connections between viral RdRp fidelity and viral fitness (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>). However, in the few instances in which RdRp derivatives exhibiting perturbed fidelity have been characterized, these derivatives also exhibit changes to the rate of nucleotide addition (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>). Relative to wild-type polymerase, a higher fidelity polymerase is a slower enzyme, and a lower fidelity polymerase is a faster enzyme (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a>). Polymerase speed will clearly determine replication kinetics and therefore can contribute to viral fitness, as has been suggested recently (<a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">17</a>). The most extensively characterized fidelity mutants of PV harbor an RdRp with a substitution located at a remote site that likely causes substantial collateral damage, further confounding the fidelity-versus-speed debate (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>). This study was motivated by the need to understand better the relationship between the biochemical properties of the viral RdRp and viral fitness, virulence, and pathogenesis. We have had in hand for a long time a PV mutant whose speed and fidelity were perturbed by changing an RdRp active-site residue, K359H (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">24</a>). We did not publish this PV mutant until now because of its genetic instability (<strong><a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1A</a></strong>). We realized that this genetic instability might actually represent an opportunity, as the biochemical phenotypes reverted by the second-site suppressors might highlight the biochemical properties driving viral fitness.</p><p id="p-31">Lys-359 protonates the pyrophosphate leaving group during nucleotidyl transfer (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). The K359H substitution will change the efficiency of that protonation event for two reasons. First, the pKa value of histidine is substantially lower than lysine and will therefore not be protonated to the same extent. Second, the distance between the nucleotide phosphates and the imidazole ring of the histidine will be greater than that of the amino group of lysine. It is likely that the distance is more of an issue, because K359R PV is stable and K359R RdRp exhibits the same reduction in catalytic efficiency as K359H RdRp (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">6</a>). The changes of I331F and P356S in motifs C and D, respectively, may serve to move His-359 closer to the nucleotide phosphates.</p><p id="p-32">Each substitution (IF and PS) individually increased the replication efficiency of K359H PV, with the combination of the two substitutions producing an additive outcome (<strong><a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1C</a></strong>). Each substitution also decreased the fidelity of nucleotide addition in cells, as assessed by ribavirin sensitivity, to the same extent; the combination mutant yielded a virus with properties on par with wild type (<strong><a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig. 2C</a></strong>). The observed changes to replication speed and fidelity for the mutant viruses were explained by the biochemical properties of the corresponding RdRps (<strong><a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig. 2D</a></strong>). As observed in the past, both speed and fidelity appear to be correlated.</p><p id="p-33">The substitutions at the active site of the polymerase do not only affect replication efficiency and population diversity but also impact other stages of the lifecycle. The kinetics of virus production for the different mutants did not correspond directly to the kinetics of replication (compare <strong><a id="xref-fig-2-17" class="xref-fig" href="#F2">Fig. 2A</a></strong> to <strong><a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig 1C</a></strong>). Virus production was significantly delayed for PS-KH PV (<strong><a id="xref-fig-2-18" class="xref-fig" href="#F2">Fig. 2A</a></strong>). This issue with virus production manifests as a substantial decrease in the specific infectivity of this virus (<strong><a id="xref-fig-2-19" class="xref-fig" href="#F2">Fig. 2B</a></strong>). One possible explanation for this outcome is that the RdRp is encoded by the 3D region of the viral genome and this region is also a component of the PV 3CD protein. The 3CD protein has well established roles in aspects of the lifecycle before, during, and after replication, including virion morphogenesis (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). Observations such as these highlight the additional level of complexity associated with establishing a cause-and-effect relationship between biochemical properties of the RdRp and fitness, virulence, and/or pathogenesis.</p><p id="p-34">We and others have observed correlations between RdRp fidelity and recombination efficiency, with higher fidelity suppressing recombination and vice versa (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>). An unexpected outcome of this study was the observation that IF-KH and PS-KH PVs exhibit substantially different propensities for recombination in cells (<strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3B</a></strong>), in spite of very similar speed and fidelity phenotypes in cells (<strong><a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1C</a></strong> and <strong><a id="xref-fig-2-20" class="xref-fig" href="#F2">Fig. 2C</a></strong>) and in vitro (<strong><a id="xref-fig-2-21" class="xref-fig" href="#F2">Fig. 2D</a></strong>). PS-KH PV appeared to be completely incapable of supporting recombination in cells (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3B</a></strong>). Because the cell-based recombination assay requires the ability of virus to plaque, it was conceivable that the defect of PS-KH PV was more a reflection of spread than impaired recombination. We established a more sensitive fluorescence-based assay (<strong><a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3C</a></strong>), which also indicated impaired recombination (<strong><a id="xref-fig-3-14" class="xref-fig" href="#F3">Fig. 3D</a></strong>). The impact of the PS substitution on recombination was also quite evident in the context of IF-PS-KH PV. The biological and biochemical properties of this triple mutant are within twofold of wild type (<strong><a id="xref-fig-1-13" class="xref-fig" href="#F1">Figs. 1</a></strong> and <strong><a id="xref-fig-2-22" class="xref-fig" href="#F2">2</a></strong>), but the recombination efficiency of this mutant is down by 30-fold relative to WT PV (<strong><a id="xref-fig-3-15" class="xref-fig" href="#F3">Fig. 3B</a></strong>). Together, these observations support the existence of biochemical properties of the RdRp other than speed and fidelity that are essential for recombination and impaired by changing Pro-356 to Ser. Further characterization of these mutants will be required to identify this undefined biochemical property.</p><p id="p-35">Our laboratory has had a longstanding interest in the mechanism of recombination (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>). We have shown that the high poly(rU) polymerase activity of PV RdRp derives from template switching during elongation, thus mimicking copy-choice recombination (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4A</a></strong>) (<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">36</a>). A second assay that we developed is based on template switching from the end of template, thus mimicking forced-copy-choice recombination (<strong><a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5A</a></strong>) (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">38</a>). Based on the mutants and corresponding RdRp derivatives reported here, high fidelity impairs both types of recombination (see K359H in <strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Figs. 4B</a></strong> and <strong><a id="xref-fig-5-8" class="xref-fig" href="#F5">5D</a></strong>). Interestingly, unique RdRp determinants exist for each mode of recombination. PS-KH RdRp is impaired for copy-choice recombination (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4B</a></strong>); IF-KH is impaired for forced-copy-choice recombination (<strong><a id="xref-fig-5-9" class="xref-fig" href="#F5">Fig. 5D</a></strong>). Impairment of recombination by PS-KH PV in cells is consistent with template switching as the primary mechanism of copy-choice recombination in cells, as demonstrated by Kirkegaard and Baltimore (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>). The observation that IF-KH PV fails to exhibit a wild-type recombination phenotype (<strong><a id="xref-fig-3-16" class="xref-fig" href="#F3">Fig. 3B</a></strong>) may suggest that forced-copy-choice recombination contributes to recombination in cells as well.</p><p id="p-36">The current narrative of the characterized PV mutants impaired for recombination is that recombination acts as a mechanism to purge deleterious mutations in the viral population (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>-<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). Our study reveals several caveats for the interpretation of these previous studies. There are known and unknown biochemical properties of the RdRp that determine recombination efficiency, and there are at least two distinct mechanisms of recombination. Without understanding these complexities for the mutants under investigation, it is difficult to compare one study to another and may explain disparate observations made between laboratories (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>). Further analysis of the panel of mutants reported here may help to resolve the controversy but should illuminate how changes to the mechanism and efficiency of recombination impact viral evolution in a background of constant speed and fidelity.</p><p id="p-37">Viral fitness, virulence, and pathogenesis are determined collectively by the structure, dynamics, and activity of all virus-encoded functions. The conserved structure, dynamics, and mechanism of the viral RdRp makes this enzyme an attractive target for development of attenuated viruses by changing conserved residues capable of perturbing conserved, biochemical function (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">10</a>). As a field, we have focused on RdRp fidelity (<a id="xref-ref-6-10" class="xref-bibr" href="#ref-6">6</a>) but have come to the realization that RdRp speed should not be ignored (<a id="xref-ref-17-11" class="xref-bibr" href="#ref-17">17</a>). This study shows that speed, fidelity, and undefined biochemical properties of the RdRp exist and contribute to both the mechanism and efficiency of recombination (<strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a></strong>). This inextricable connection of the myriad biochemical properties of the RdRp precludes attribution of a single biochemical property to viral fitness, virulence, and pathogenesis (<strong><a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a></strong>). The field will be served best by continued emphasis on discovery of manipulatable functions of the RdRp, and other viral enzymes, instead of debating the importance of individual properties.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The biochemical properties of the RdRp are inextricably linked and collectively determine biological outcomes. PV RdRp is among the most extensively studied enzyme both in cells and in test tubes (4). A vast majority of these studies have emphasized elaboration of mechanisms governing efficient incorporation of nucleotides (speed) with high specificity (fidelity). Whether or not speed is a consequence of fidelity, vice versa, or completely separable is not clear (17, 33, 52-54). However, these two parameters are linked to recombination efficiency. Moreover, the studies reported herein point to the existence of a biochemical property other than speed and fidelity, referred to here as undefined, that contributes to recombination efficiency. Together, the observations reported herein demonstrate that the current state of the art precludes association of a single biochemical property to viral fitness, virulence, and/or pathogenesis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385012906" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The biochemical properties of the RdRp are inextricably linked and collectively determine biological outcomes.&lt;/span&gt; PV RdRp is among the most extensively studied enzyme both in cells and in test tubes (4). A vast majority of these studies have emphasized elaboration of mechanisms governing efficient incorporation of nucleotides (speed) with high specificity (fidelity). Whether or not speed is a consequence of fidelity, vice versa, or completely separable is not clear (17, 33, 52-54). However, these two parameters are linked to recombination efficiency. Moreover, the studies reported herein point to the existence of a biochemical property other than speed and fidelity, referred to here as undefined, that contributes to recombination efficiency. Together, the observations reported herein demonstrate that the current state of the art precludes association of a single biochemical property to viral fitness, virulence, and/or pathogenesis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F6.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F6.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/14/769224/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <span class="caption-title">The biochemical properties of the RdRp are inextricably linked and collectively determine biological outcomes.</span><p id="p-38" class="first-child">PV RdRp is among the most extensively studied enzyme both in cells and in test tubes (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>). A vast majority of these studies have emphasized elaboration of mechanisms governing efficient incorporation of nucleotides (speed) with high specificity (fidelity). Whether or not speed is a consequence of fidelity, vice versa, or completely separable is not clear (<a id="xref-ref-17-12" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>-<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). However, these two parameters are linked to recombination efficiency. Moreover, the studies reported herein point to the existence of a biochemical property other than speed and fidelity, referred to here as undefined, that contributes to recombination efficiency. Together, the observations reported herein demonstrate that the current state of the art precludes association of a single biochemical property to viral fitness, virulence, and/or pathogenesis.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-10"><h2 class="">Materials and Methods</h2><div id="sec-11" class="subsection"><h3>Cells and Viruses</h3><p id="p-39">Adherent monolayers of HeLa and L929 fibroblasts were grown in DMEM/F-12 media. Media was supplemented with 100 U/mL penicillin, 100 µg/mL streptomycin, and 10% heat inactivated (HI)-FBS. All cells were passaged in the presence of trypsin-EDTA. Wild-type and recombinant PV viruses were recovered after transfection of RNA produced <em>in vitro</em> (see below) from full-length cDNA or from the CRE-REP assay parental partners (<a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">22</a>). PV type 1 (Mahoney) was used throughout this study (Genbank accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=V01149.1&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom">V01149.1</a>).</p></div><div id="sec-12" class="subsection"><h3>Plasmids, in vitro transcription, cell transfection, and virus quantification</h3><p id="p-40">All mutations were introduced into the pET26Ub-PV 3D (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>) bacterial expression plasmids using overlap extension PCR. The presence of the desired mutations and the absence of additional mutations were verified by DNA sequencing. The expression plasmid encoding the various mutations was digested and inserted into an intermediate plasmid, pUC18-BglII-EcoRI-3CD (referred as pUC-3CD in (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>), and then the fragment between BglII and ApaI was cloned into the subgenomic replicon, pRLucRA, or viral cDNA, pMovRA or PV1ΔCRE. PV1ΔCRE is a full-length PV type 1 (Mahoney) cDNA bearing 8 synonymous substitutions in the oriI <em>cis</em>-acting replication element (CRE) located in the 2C-coding region and was described previously (<a id="xref-ref-22-11" class="xref-bibr" href="#ref-22">22</a>). In order to create a fluorescence-based assay for recombination in cells, we modified the PV1ΔCRE acceptor RNA to include the coding sequence for UnaG green fluorescent protein. Briefly, UnaG-encoding sequence (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>) carrying a 3C protease cleavage site at its carboxyl terminus was inserted between the IRES and the P1 region of the PV sequence. Translation occurred from the natural poliovirus initiation codon. Proteolytic cleavage and release of the fluorescent protein occurred by normal 3C protease activity. Plasmids encoding PV genomes (full length or subgenomic) were linearized with ApaI. All linearized cDNAs were transcribed <em>in vitro</em> using T7 RNA Polymerase and treated with 2U DNAse Turbo (ThermoFisher) to remove residual DNA template. The RNA transcripts were purified using RNeasy Mini Kit (Qiagen) before spectrophotometric quantification. Purified RNA in RNase-free H<sub>2</sub>O was transfected into either HeLa or L929 fibroblasts using TransMessenger (Qiagen). Virus yield was quantified by plaque assay. Briefly, cells and media were harvested at 2-3 days post-transfection, subjected to three freeze-thaw cycles, and clarified. Supernatant was then used on fresh HeLa cells in 6-well plates; virus infection was allowed to continue for 30 min. Media was then removed, and cells were washed twice with PBS (pH 7.4) washes before a 0.8% (w/v) agarose-media overlay was added. Cells were incubated for 2-3 days and then fixed and stained with crystal violet for virus quantification.</p></div><div id="sec-13" class="subsection"><h3>Luciferase assays</h3><p id="p-41">Subgenomic luciferase assays were performed as described previously (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>).</p></div><div id="sec-14" class="subsection"><h3>Virus sequencing</h3><p id="p-42">Viral RNA was extracted from clarified culture supernatant using a Qiagen RNAeasy Mini kit, reverse transcribed using Superscript II reverse transcriptase (Invitrogen) using an oligo-dT primer according to the manufacturer’s protocol. PCR amplification of the 3D<sup>pol</sup> region in PV used template cDNA and appropriate oligonucleotides as listed in Table 1 by using Phusion high-fidelity DNA polymerase (NEB) according to the manufacturer’s protocol. PCR products were gel purified and sequenced by the Genomics Core Facility of the Pennsylvania State University.</p></div><div id="sec-15" class="subsection"><h3>One-step growth analysis</h3><p id="p-43">HeLa cells in 12-well plates were infected by each virus at a MOI of 10. Following a 30-minute incubation, cells were washed twice with PBS and media was replaced. Virus was harvested at different time-points post infection and the virus yield was quantified by plaque assay.</p></div><div id="sec-16" class="subsection"><h3>Quantitative RT-PCR</h3><p id="p-44">Viral RNA was purified from virus stocks by using QiaAmp viral RNA purification kit (Qiagen) and used for RT-qPCR to determine genome copies. This analysis was performed by the Genomics Core Facility of the Pennsylvania State University. DNAse-treated RNA was reverse-transcribed using the High Capacity cDNA Reverse Transcription kit (Applied Biosystems, Foster City CA) following the protocol provided with the kit. Quantification by real-time qPCR was done with 2X TaqMan Universal PCR Master Mix (Applied Biosystems, Foster City CA) in a volume of 20 μL, with primers 5′-ACCCCTGGTAGCAATCAATATCTTAC-3′(forward) and 5′-TTCTTTACTTCACCGGGTATGTCA-3′ (reverse), and probe 5′-[6-Fam] TGTGCGCTGCCTGAATTTGATGTGA-3′ in a 7300 Real-Time qPCR System (Foster City CA) machine. A standard curve was generated using in vitro transcribed RNA.</p></div><div id="sec-17" class="subsection"><h3>Ribavirin-sensitivity assay</h3><p id="p-45">This assay was described previously (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>). HeLa cells were treated with 600 µM ribavirin for 1 h before infection. Ribavirin-treated cells were then infected at MOI 0.1 with each virus variant. Following infection, the cells were washed with PBS and media was replaced with ribavirin. Infection was allowed for 24 h. Cells and supernatant were subjected to three freeze-thaw cycles. Media was clarified and used for plaque assays. All yields were normalized to an untreated control.</p></div><div id="sec-18" class="subsection"><h3>Cell-based recombination assay</h3><p id="p-46">This assay was developed in the laboratory of David Evans and was used as described previously (<a id="xref-ref-22-12" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a>). L929 fibroblasts were transfected with the PV donor and accepter RNAs, both carrying the same mutations in the RdRp gene. For the fluorescence-based detection of recombinants, the UnaG green fluorescence reporter was encoded by the acceptor RNA. Supernatant at 2∼3 days post-transfection was used to infect HeLa cells. Recombinant viruses were either quantified by plaque assay or by fluorescence imaging (488nm (ex) and 509 nm (em)).</p></div><div id="sec-19" class="subsection"><h3>Polymerase expression and purification</h3><p id="p-47">Purified polymerases used for biochemical analysis were prepared as described previously (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a>).</p></div><div id="sec-20" class="subsection"><h3>Poly(rU) polymerase activity assay</h3><p id="p-48">Reactions contained 50 mM HEPES, pH 7.5, 10 mM 2-mercaptoethanol, 5 mM MgCl<sub>2</sub>, 60 μM ZnCl<sub>2</sub>, dT<sub>15</sub> (2 μM), poly(rA) (100 μM AMP), UTP (500 μM), [α-<sup>32</sup>P]UTP (0.2 μCi/μL), and PV RdRp (0.2 μM). Reactions were initiated by addition of PV RdRp and incubated at 30 °C for 5 min at which time the reactions were quenched by addition of EDTA to a final concentration of 50 mM. Reaction volumes were 50 μL. Products were analyzed by DE81 filter binding, where 10 μL of the quenched reaction was spotted onto DE81 filter paper discs and dried completely. The discs were washed three times for 10 min in 250 mL of 5% dibasic sodium phosphate and rinsed in absolute ethanol. Bound radioactivity was quantitated by liquid scintillation counting in 5 mL of Ecoscint scintillation fluid (National Diagnostics).</p></div><div id="sec-21" class="subsection"><h3>sym/sub-based template switching assay</h3><p id="p-49">The sym/sub assay has been described previously (<a id="xref-ref-22-13" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">36</a>). Elongation complexes were assembled by incubating 5 μM PV RdRp with 1 μM sym/sub RNA primer-template and 500 μM ATP for 5 min (Mix 1). Template-switching reactions were initiated by addition of 60 μM RNA acceptor template and 500 μM CTP, GTP and UTP (Mix 2) and then quenched at various times by addition of 50 mM EDTA. All reactions were performed at 30 °C in 50 mM HEPES, pH 7.5, 10 mM 2-mercaptoethanol, 60 μM ZnCl<sub>2</sub>, and 5 mM MgCl<sub>2</sub>. Products were analyzed by denaturing polyacrylamide gel electrophoresis, visualized using a PhosphorImager, and the transfer products quantified using ImageQuant TL software (GE Healthcare).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-50">This study was supported by a grant (AI45818) from NIAID, NIH to CEC. AW is the recipient of a fellowship (18POST33960071) from the American Heart Association (AHA).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.1" data-doi="10.2217/17460794.3.6.553"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graci</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Therapeutically targeting RNA viruses via lethal mutagenesis</span>. <abbr class="cit-jnl-abbrev">Future Virol</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">553</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFuture%2BVirol%26rft.aulast%253DGraci%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D553%26rft.epage%253D566%26rft.atitle%253DTherapeutically%2Btargeting%2BRNA%2Bviruses%2Bvia%2Blethal%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.2217%252F17460794.3.6.553%26rft_id%253Dinfo%253Apmid%252F19727424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/17460794.3.6.553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19727424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261082000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perales</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Antiviral Strategies Based on Lethal Mutagenesis and Error Threshold</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr> <span class="cit-vol">392</span>:<span class="cit-fpage">323</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.volume%253D392%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.3" data-doi="10.1016/j.antiviral.2013.09.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furuta</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gowen</span>  <span class="cit-name-given-names">BB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiraki</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smee</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnard</span>  <span class="cit-name-given-names">DL</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Favipiravir (T-705), a novel viral RNA polymerase inhibitor</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">446</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D100%26rft.spage%253D446%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.09.015%26rft_id%253Dinfo%253Apmid%252F24084488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.09.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24084488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328234100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.4" data-doi="10.1016/bs.enz.2016.02.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Fidelity of Nucleotide Incorporation by the RNA-Dependent RNA Polymerase from Poliovirus</span>. <abbr class="cit-jnl-abbrev">Enzymes</abbr> <span class="cit-vol">39</span>:<span class="cit-fpage">293</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnzymes%26rft.volume%253D39%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.enz.2016.02.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.enz.2016.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.5" data-doi="10.1007/978-1-4939-3008-1_6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">August</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Polymerase Mechanism-Based Method of Viral Attenuation</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">1349</span>:<span class="cit-fpage">83</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1349%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-3008-1_6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-3008-1_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.6" data-doi="10.1074/jbc.C112.401471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">August</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A Polymerase mechanism-based strategy for viral attenuation and vaccine development</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>:<span class="cit-fpage">31618</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C112.401471%26rft_id%253Dinfo%253Apmid%252F22854962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMzgvMzE2MTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.7" data-doi="10.1038/nm.2972"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolles</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">1820</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D18%26rft.spage%253D1820%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.2972%26rft_id%253Dinfo%253Apmid%252F23142821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.2972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23142821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.8" data-doi="10.1038/nm1726"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wendt</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Engineering attenuated virus vaccines by controlling replication fidelity</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">154</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D154%26rft.epage%253D161%26rft.atitle%253DEngineering%2Battenuated%2Bvirus%2Bvaccines%2Bby%2Bcontrolling%2Breplication%2Bfidelity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1726%26rft_id%253Dinfo%253Apmid%252F18246077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246077&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252946700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Foot-and-mouth disease virus type O specific mutations determine RNA-dependent RNA polymerase fidelity and virus attenuation</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">518</span>:<span class="cit-fpage">87</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D518%26rft.spage%253D87%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.10" data-doi="10.1074/jbc.M114.592303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korboukh</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hemperly</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graci</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">August</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">RNA virus population diversity, an optimum for maximal fitness and virulence</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">289</span>:<span class="cit-fpage">29531</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.592303%26rft_id%253Dinfo%253Apmid%252F25213864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNDMvMjk1MzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.11" data-doi="10.1371/journal.ppat.1003877"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rozen-Gagnon</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mongelli</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Failloux</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saleh</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Alphavirus mutator variants present host-specific defects and attenuation in mammalian and insect models</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">e1003877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1003877%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003877%26rft_id%253Dinfo%253Apmid%252F24453971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453971&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.12" data-doi="10.1128/JVI.03594-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Ribavirin-resistant variants of foot-and-mouth disease virus: the effect of restricted quasispecies diversity on viral virulence</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">4008</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03594-13%26rft_id%253Dinfo%253Apmid%252F24453363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDAwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzE0Lzc2OTIyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.13" data-doi="10.1073/pnas.1204022109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gnadig</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaucourt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnola</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borderia</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanz-Ramos</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Coxsackievirus B3 mutator strains are attenuated in vivo</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">E2294</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1204022109%26rft_id%253Dinfo%253Apmid%252F22853955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0L0UyMjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.14" data-doi="10.1073/pnas.1111650108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beeharry</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borderia</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Arbovirus high fidelity variant loses fitness in mosquitoes and mice</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">108</span>:<span class="cit-fpage">16038</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111650108%26rft_id%253Dinfo%253Apmid%252F21896755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM4LzE2MDM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.15" data-doi="10.1038/nature04388"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">439</span>:<span class="cit-fpage">344</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D439%26rft.issue%253D7074%26rft.spage%253D344%26rft.epage%253D348%26rft.atitle%253DQuasispecies%2Bdiversity%2Bdetermines%2Bpathogenesis%2Bthrough%2Bcooperative%2Binteractions%2Bin%2Ba%2Bviral%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04388%26rft_id%253Dinfo%253Apmid%252F16327776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16327776&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234682100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.16" data-doi="10.1074/jbc.M503444200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Remote site control of an active site fidelity checkpoint in a viral RNA-dependent RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">280</span>:<span class="cit-fpage">25706</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M503444200%26rft_id%253Dinfo%253Apmid%252F15878882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjcvMjU3MDYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.17" data-doi="10.1371/journal.pbio.2006459"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzsimmons</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCrone</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yennawar</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A speed-fidelity trade-off determines the mutation rate and virulence of an RNA virus</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">e2006459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D16%26rft.spage%253De2006459%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2006459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2006459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Senecavirus-Specific Recombination Assays Reveal the Intimate Link between Polymerase Fidelity and RNA Recombination</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janissen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">YN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">NH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shih</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Predicting Intraserotypic Recombination in Enterovirus 71</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">93</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Te Velthuis</span>  <span class="cit-name-given-names">AJW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">DLV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">RLY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharps</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siegers</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Killip</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliva-Martin</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Riel</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fodor</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Mini viral RNAs act as innate immune agonists during influenza virus infection</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">1234</span>–<span class="cit-lpage">1242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D3%26rft.spage%253D1234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.21" data-doi="10.1093/nar/gky910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rawson</span>  <span class="cit-name-given-names">JMO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nikolaitchik</span>  <span class="cit-name-given-names">OA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keele</span>  <span class="cit-name-given-names">BF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">WS</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Recombination is required for efficient HIV-1 replication and the maintenance of viral genome integrity</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">46</span>:<span class="cit-fpage">10535</span>–<span class="cit-lpage">10545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253D10535%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.22" data-doi="10.1093/nar/gkw567"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Biochemical and genetic analysis of the role of the viral polymerase in enterovirus recombination</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">6883</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw567%26rft_id%253Dinfo%253Apmid%252F27317698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27317698&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.23" data-doi="10.1128/JVI.02921-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poirier</span>  <span class="cit-name-given-names">EZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mounce</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozen-Gagnon</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooikaas</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moratorio</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Low-Fidelity Polymerases of Alphaviruses Recombine at Higher Rates To Overproduce Defective Interfering Particles</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">2446</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D90%26rft.spage%253D2446%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02921-15%26rft_id%253Dinfo%253Apmid%252F26676773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/JVI.02921-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26676773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.24" data-doi="10.1038/nsmb.1540"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maksimchuk</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uchida</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gotte</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konigsberg</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Nucleic acid polymerases use a general acid for nucleotidyl transfer</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">212</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DCastro%26rft.auinit1%253DC.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D218%26rft.atitle%253DNucleic%2Bacid%2Bpolymerases%2Buse%2Ba%2Bgeneral%2Bacid%2Bfor%2Bnucleotidyl%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1540%26rft_id%253Dinfo%253Apmid%252F19151724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19151724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263286600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.25" data-doi="10.1038/82191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crotty</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maag</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">The broad-spectrum antiviral ribonucleoside ribavirin is an RNA virus mutagen</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">1375</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DCrotty%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D1375%26rft.epage%253D1379%26rft.atitle%253DThe%2Bbroad-spectrum%2Bantiviral%2Bribonucleoside%2Bribavirin%2Bis%2Ban%2BRNA%2Bvirus%2Bmutagen.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F82191%26rft_id%253Dinfo%253Apmid%252F11100123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/82191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11100123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165704100042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.26" data-doi="10.1073/pnas.1232294100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirkegaard</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A single mutation in poliovirus RNA-dependent RNA polymerase confers resistance to mutagenic nucleotide analogs via increased fidelity</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">7289</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1232294100%26rft_id%253Dinfo%253Apmid%252F12754380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEyLzcyODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.27" data-doi="10.1128/JVI.77.1.481-488.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Severson</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmaljohn</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Javadian</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonsson</span>  <span class="cit-name-given-names">CB</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Ribavirin causes error catastrophe during Hantaan virus replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">77</span>:<span class="cit-fpage">481</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSeverson%26rft.auinit1%253DW.%2BE.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D481%26rft.epage%253D488%26rft.atitle%253DRibavirin%2BCauses%2BError%2BCatastrophe%2Bduring%2BHantaan%2BVirus%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.1.481-488.2003%26rft_id%253Dinfo%253Apmid%252F12477853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijc3LzEvNDgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.28" data-doi="10.1016/j.virusres.2004.11.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Ribavirin and lethal mutagenesis of poliovirus: molecular mechanisms, resistance and biological implications</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">107</span>:<span class="cit-fpage">173</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D181%26rft.atitle%253DRibavirin%2Band%2Blethal%2Bmutagenesis%2Bof%2Bpoliovirus%253A%2Bmolecular%2Bmechanisms%252C%2Bresistance%2Band%2Bbiological%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2004.11.007%26rft_id%253Dinfo%253Apmid%252F15649563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2004.11.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15649563&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226885000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.29" data-doi="10.1128/JVI.01606-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sierra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Airaksinen</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Lopez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agudo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Foot-and-mouth disease virus mutant with decreased sensitivity to ribavirin: implications for error catastrophe</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">2012</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01606-06%26rft_id%253Dinfo%253Apmid%252F17151116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzQvMjAxMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzE0Lzc2OTIyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.30" data-doi="10.1128/JVI.01297-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sierra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Determinants of RNA-dependent RNA polymerase (in)fidelity revealed by kinetic analysis of the polymerase encoded by a foot-and-mouth disease virus mutant with reduced sensitivity to ribavirin</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">12346</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01297-08%26rft_id%253Dinfo%253Apmid%252F18829745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yNC8xMjM0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzE0Lzc2OTIyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aponte-Diaz</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeager</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umeda</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hara</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">RYL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Hijacking of multiple phospholipid biosynthetic pathways and induction of membrane biogenesis by a picornaviral 3CD protein</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">e1007086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aponte-Diaz</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aligo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lodeiro</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Multiple poliovirus-induced organelles suggested by comparison of spatiotemporal dynamics of membranous structures and phosphoinositides</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">e1007036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.33" data-doi="10.1016/j.celrep.2015.01.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dulin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilfan</span>  <span class="cit-name-given-names">ID</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berghuis</span>  <span class="cit-name-given-names">BA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hage</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bamford</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poranen</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Depken</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">NH</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Elongation-Competent Pauses Govern the Fidelity of a Viral RNA-Dependent RNA Polymerase</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">983</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D10%26rft.spage%253D983%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.01.031%26rft_id%253Dinfo%253Apmid%252F25683720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.01.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25683720&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.34" data-doi="10.1371/journal.ppat.1004191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Recombination in enteroviruses is a biphasic replicative process involving the generation of greater-than genome length ‘imprecise’ intermediates</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">e1004191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004191%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004191%26rft_id%253Dinfo%253Apmid%252F24945141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24945141&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.35" data-doi="10.1016/j.cell.2013.05.038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumagai</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ando</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyatake</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greimel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirabayashi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimogori</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyawaki</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A bilirubin-inducible fluorescent protein from eel muscle</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>:<span class="cit-fpage">1602</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D1602%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.038%26rft_id%253Dinfo%253Apmid%252F23768684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23768684&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320722000026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.36" data-doi="10.1074/jbc.274.5.2706"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Poliovirus RNA-dependent RNA polymerase (3Dpol) is sufficient for template switching in vitro</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">274</span>:<span class="cit-fpage">2706</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DArnold%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D274%26rft.issue%253D5%26rft.spage%253D2706%26rft.epage%253D2716%26rft.atitle%253DPoliovirus%2BRNA-dependent%2BRNA%2BPolymerase%2B%25283Dpol%2529%2BIs%2BSufficient%2Bfor%2BTemplate%2BSwitchingin%2BVitro%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.5.2706%26rft_id%253Dinfo%253Apmid%252F9915801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzQvNS8yNzA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.37" data-doi="10.1016/0092-8674(86)90600-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirkegaard</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">The mechanism of RNA recombination in poliovirus</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">47</span>:<span class="cit-fpage">433</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKirkegaard%26rft.auinit1%253DK.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DThe%2Bmechanism%2Bof%2BRNA%2Brecombination%2Bin%2Bpoliovirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990600-8%26rft_id%253Dinfo%253Apmid%252F3021340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90600-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3021340&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986E805900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.38" data-doi="10.1074/jbc.275.8.5329"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Poliovirus RNA-dependent RNA polymerase (3D(pol)). Assembly of stable, elongation-competent complexes by using a symmetrical primer-template substrate (sym/sub</span>). <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">275</span>:<span class="cit-fpage">5329</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DArnold%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D275%26rft.issue%253D8%26rft.spage%253D5329%26rft.epage%253D5336%26rft.atitle%253DPoliovirus%2BRNA-dependent%2BRNA%2BPolymerase%2B%25283Dpol%2529%253A%2BASSEMBLY%2BOF%2BSTABLE%252C%2BELONGATION-COMPETENT%2BCOMPLEXES%2BBY%2BUSING%2BA%2BSYMMETRICAL%2B%2BPRIMER-TEMPLATE%2BSUBSTRATE%2B%2528sym%252Fsub%2529%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.275.8.5329%26rft_id%253Dinfo%253Apmid%252F10681506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzUvOC81MzI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.39" data-doi="10.1021/bi00179a014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peliska</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benkovic</span>  <span class="cit-name-given-names">SJ</span></span>. <span class="cit-pub-date">1994</span>. <span class="cit-article-title">Fidelity of in vitro DNA strand transfer reactions catalyzed by HIV-1 reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">3890</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D33%26rft.spage%253D3890%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00179a014%26rft_id%253Dinfo%253Apmid%252F7511410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00179a014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7511410&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.40" data-doi="10.1126/science.1279806"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peliska</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benkovic</span>  <span class="cit-name-given-names">SJ</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Mechanism of DNA strand transfer reactions catalyzed by HIV-1 reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">258</span>:<span class="cit-fpage">1112</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPeliska%26rft.auinit1%253DJ.%26rft.volume%253D258%26rft.issue%253D5085%26rft.spage%253D1112%26rft.epage%253D1118%26rft.atitle%253DMechanism%2Bof%2BDNA%2Bstrand%2Btransfer%2Breactions%2Bcatalyzed%2Bby%2BHIV-1%2Breverse%2Btranscriptase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1279806%26rft_id%253Dinfo%253Apmid%252F1279806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNTgvNTA4NS8xMTEyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.41" data-doi="10.1021/bi00250a036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peliska</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benkovic</span>  <span class="cit-name-given-names">SJ</span></span>. <span class="cit-pub-date">1994</span>. <span class="cit-article-title">Recombinant HIV-1 nucleocapsid protein accelerates HIV-1 reverse transcriptase catalyzed DNA strand transfer reactions and modulates RNase H activity</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">13817</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D33%26rft.spage%253D13817%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00250a036%26rft_id%253Dinfo%253Apmid%252F7524664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00250a036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7524664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PT48500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.42" data-doi="10.1073/pnas.0608952104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maksimchuk</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korneeva</span>  <span class="cit-name-given-names">VS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gotte</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konigsberg</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Two proton transfers in the transition state for nucleotidyl transfer catalyzed by RNA-and DNA-dependent RNA and DNA polymerases</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">104</span>:<span class="cit-fpage">4267</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608952104%26rft_id%253Dinfo%253Apmid%252F17360513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzExLzQyNjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.43" data-doi="10.1074/jbc.274.18.12867"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parsley</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornell</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Semler</span>  <span class="cit-name-given-names">BL</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Modulation of the RNA binding and protein processing activities of poliovirus polypeptide 3CD by the viral RNA polymerase domain</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">274</span>:<span class="cit-fpage">12867</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DParsley%26rft.auinit1%253DT.%2BB.%26rft.volume%253D274%26rft.issue%253D18%26rft.spage%253D12867%26rft.epage%253D12876%26rft.atitle%253DModulation%2Bof%2Bthe%2BRNA%2BBinding%2Band%2BProtein%2BProcessing%2BActivities%2Bof%2BPoliovirus%2BPolypeptide%2B3CD%2Bby%2Bthe%2BViral%2BRNA%2BPolymerase%2BDomain%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.18.12867%26rft_id%253Dinfo%253Apmid%252F10212275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMTgvMTI4NjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reddigari</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicklin</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hammerle</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Purification and characterization of poliovirus polypeptide 3CD, a proteinase and a precursor for RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">66</span>:<span class="cit-fpage">7481</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHarris%26rft.auinit1%253DK%2BS%26rft.volume%253D66%26rft.issue%253D12%26rft.spage%253D7481%26rft.epage%253D7489%26rft.atitle%253DPurification%2Band%2Bcharacterization%2Bof%2Bpoliovirus%2Bpolypeptide%2B3CD%252C%2Ba%2Bproteinase%2Band%2Ba%2Bprecursor%2Bfor%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Apmid%252F1331532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8xMi83NDgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.45" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">HJ</span></span>. <span class="cit-pub-date">1964</span>. <span class="cit-article-title">The Relation of Recombination to Mutational Advance</span>. <abbr class="cit-jnl-abbrev">Mutat Res</abbr> <span class="cit-vol">106</span>:<span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DMuller%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D106%26rft.spage%253D2%26rft.epage%253D9%26rft.atitle%253DTHE%2BRELATION%2BOF%2BRECOMBINATION%2BTO%2BMUTATIONAL%2BADVANCE.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft_id%253Dinfo%253Apmid%252F14195748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14195748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">1974</span>. <span class="cit-article-title">The evolutionary advantage of recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>:<span class="cit-fpage">737</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.47" data-doi="10.1038/348454a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Fitness of RNA virus decreased by Muller’s ratchet</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">348</span>:<span class="cit-fpage">454</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChao%26rft.auinit1%253DL.%26rft.volume%253D348%26rft.issue%253D6300%26rft.spage%253D454%26rft.epage%253D455%26rft.atitle%253DFitness%2Bof%2BRNA%2Bvirus%2Bdecreased%2Bby%2BMuller%2527s%2Bratchet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F348454a0%26rft_id%253Dinfo%253Apmid%252F2247152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/348454a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2247152&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EK69700066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.48" data-doi="10.1128/JVI.00078-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kempf</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Poliovirus Polymerase Leu420 Facilitates RNA Recombination and Ribavirin Resistance</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">8410</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00078-16%26rft_id%253Dinfo%253Apmid%252F27412593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xOS84NDEwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kempf</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watkins</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Picornavirus RNA Recombination Counteracts Error Catastrophe</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">93</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.50" data-doi="10.1006/prep.1999.1100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gohara</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wisniewski</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Production of “authentic” poliovirus RNA-dependent RNA polymerase (3D(pol)) by ubiquitin-protease-mediated cleavage in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Protein Expr Purif</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">128</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2Bexpression%2Band%2Bpurification%26rft.stitle%253DProtein%2BExpr%2BPurif%26rft.aulast%253DGohara%26rft.auinit1%253DD.%2BW.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D128%26rft.epage%253D138%26rft.atitle%253DProduction%2Bof%2B%2526quot%253Bauthentic%2526quot%253B%2Bpoliovirus%2BRNA-dependent%2BRNA%2Bpolymerase%2B%25283D%2528pol%2529%2529%2Bby%2Bubiquitin-protease-mediated%2Bcleavage%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fprep.1999.1100%26rft_id%253Dinfo%253Apmid%252F10497078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/prep.1999.1100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10497078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083004300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.51" data-doi="10.1074/jbc.M002671200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gohara</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crotty</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoder</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Poliovirus RNA-dependent RNA polymerase (3Dpol): structural, biochemical, and biological analysis of conserved structural motifs A and B</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">275</span>:<span class="cit-fpage">25523</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M002671200%26rft_id%253Dinfo%253Apmid%252F10827187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvMzMvMjU1MjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOS8xNC83NjkyMjQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.52" data-doi="10.1073/pnas.0501062102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furio</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">The cost of replication fidelity in an RNA virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>:<span class="cit-fpage">10233</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501062102%26rft_id%253Dinfo%253Apmid%252F16006529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzI5LzEwMjMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDkvMTQvNzY5MjI0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-769224v1.53" data-doi="10.1128/JVI.79.18.11555-11558.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Adaptive value of high mutation rates of RNA viruses: separating causes from consequences</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">79</span>:<span class="cit-fpage">11555</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D79%26rft.issue%253D18%26rft.spage%253D11555%26rft.epage%253D11558%26rft.atitle%253DAdaptive%2BValue%2Bof%2BHigh%2BMutation%2BRates%2Bof%2BRNA%2BViruses%253A%2BSeparating%2BCauses%2Bfrom%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.18.11555-11558.2005%26rft_id%253Dinfo%253Apmid%252F16140732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8xOC8xMTU1NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA5LzE0Lzc2OTIyNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-769224v1.54" data-doi="10.1098/rspb.2006.3732"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furio</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">The cost of replication fidelity in human immunodeficiency virus type 1</span>. <abbr class="cit-jnl-abbrev">Proc Biol Sci</abbr> <span class="cit-vol">274</span>:<span class="cit-fpage">225</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DFurio%26rft.auinit1%253DV.%26rft.volume%253D274%26rft.issue%253D1607%26rft.spage%253D225%26rft.epage%253D230%26rft.atitle%253DThe%2Bcost%2Bof%2Breplication%2Bfidelity%2Bin%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2006.3732%26rft_id%253Dinfo%253Apmid%252F17148251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2006.3732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17148251&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F09%2F14%2F769224.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242765400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/769224v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F769224v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/769778v1" title="Aggregatibacter actinomycetemcomitans LtxA hijacks endocytic trafficking pathways in human lymphocytes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 14, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/769224v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/769224v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/769224v1" class="active" data-icon-position="" data-hide-link-title="0">RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/907200" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--o9R--lamGzMxmoLqDNQsPboHg0lzbvDuFR9gqTKhcY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="907200" id="node907200--2" data-pisa="biorxiv;769224v1" data-pisa-master="biorxiv;769224" data-seqnum="907200" data-apath="/biorxiv/early/2019/09/14/769224.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hyejeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Victor D.</span> <span class="nlm-surname">Ellis</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Woodman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jamie J.</span> <span class="nlm-surname">Arnold</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Craig E.</span> <span class="nlm-surname">Cameron</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">769224; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/769224 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/769224v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;title=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;title=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;text=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;title=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;t=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;title=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;title=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="907200" id="node907200--3" data-pisa="biorxiv;769224v1" data-pisa-master="biorxiv;769224" data-seqnum="907200" data-apath="/biorxiv/early/2019/09/14/769224.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">RNA-dependent RNA polymerase speed and fidelity are not the only determinants of the mechanism or efficiency of recombination</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hyejeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Victor D.</span> <span class="nlm-surname">Ellis</span> <span class="nlm-suffix">III</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Woodman</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jamie J.</span> <span class="nlm-surname">Arnold</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Craig E.</span> <span class="nlm-surname">Cameron</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">769224; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/769224 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kim%2C+Hyejeong&amp;rft.au=Ellis%2C+Victor+D.&amp;rft.au=Woodman%2C+Andrew&amp;rft.au=Zhao%2C+Yan&amp;rft.au=Arnold%2C+Jamie+J.&amp;rft.au=Cameron%2C+Craig+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/907200/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/907200/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/907200/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/907200/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/907200/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/907200/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/907200/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/907200/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/907200/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/907200/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/907200/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/907200/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;count=horizontal&amp;via=&amp;text=RNA-dependent%20RNA%20polymerase%20speed%20and%20fidelity%20are%20not%20the%20only%20determinants%20of%20the%20mechanism%20or%20efficiency%20of%20recombination&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/769224v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/769224v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/769224v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
