<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/14/2020.03.12.989186.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.12.989186v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase" />
<meta name="DC.Identifier" content="10.1101/2020.03.12.989186" />
<meta name="DC.Date" content="2020-03-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.[1][1] Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.[2][2] Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3’-fluoro-3’-deoxythymidine triphosphate and 3’-azido-3’-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.

 [1]: #ref-1
 [2]: #ref-2" />
<meta name="DC.Contributor" content="Jingyue Ju" />
<meta name="DC.Contributor" content="Xiaoxu Li" />
<meta name="DC.Contributor" content="Shiv Kumar" />
<meta name="DC.Contributor" content="Steffen Jockusch" />
<meta name="DC.Contributor" content="Minchen Chien" />
<meta name="DC.Contributor" content="Chuanjuan Tao" />
<meta name="DC.Contributor" content="Irina Morozova" />
<meta name="DC.Contributor" content="Sergey Kalachikov" />
<meta name="DC.Contributor" content="Robert N. Kirchdoerfer" />
<meta name="DC.Contributor" content="James J. Russo" />
<meta name="article:published_time" content="2020-03-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency. Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2. Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3′-fluoro-3′-deoxythymidine triphosphate and 3′-azido-3′-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.12.989186v1" />
<meta name="citation_id" content="2020.03.12.989186v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.12.989186" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.12.989186" />
<meta name="citation_author" content="Jingyue Ju" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Pharmacology, Columbia University" />
<meta name="citation_author_email" content="dj222@columbia.edu" />
<meta name="citation_author" content="Xiaoxu Li" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Shiv Kumar" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Steffen Jockusch" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemistry, Columbia University" />
<meta name="citation_author" content="Minchen Chien" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Chuanjuan Tao" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Irina Morozova" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Sergey Kalachikov" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_author" content="Robert N. Kirchdoerfer" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Wisconsin-Madison" />
<meta name="citation_author_institution" content="Institute of Molecular Virology, University of Wisconsin-Madison" />
<meta name="citation_author" content="James J. Russo" />
<meta name="citation_author_institution" content="Center for Genome Technology and Biomolecular Engineering, Columbia University" />
<meta name="citation_author_institution" content="Department of Chemical Engineering, Columbia University" />
<meta name="citation_reference" content="Zhu, N. et al., for the China Novel Coronavirus Investigating and Research Team. A novel coronavirus from patients with pneumonia in China, 2019. N Eng J Med 382, 727–733 (2020)." />
<meta name="citation_reference" content="Ju, J., Kumar, S., Li, X., Jockusch, S. &amp; Russo, J. J. Nucleotide analogues as inhibitors of viral polymerases. bioRXiv doi: doi.org/10.1101/2020.01.30.927574 (2020)." />
<meta name="citation_reference" content="Zumla, A., Chan, J. F. W., Azhar, E. I., Hui, D. S. C. &amp; Yuen, K.-Y. Coronaviruses – drug discovery and therapeutic options. Nat Rev | Drug Discovery 15, 327–347 (2016)." />
<meta name="citation_reference" content="Dustin, L. B., Bartolini, B., Capobianchi, M. R. &amp; Pistello, M. Hepatitis C virus: life cycle in cells, infection and host response, and analysis of molecular markers influencing the outcome of infection and response to therapy. Clin Microbiol Infect 22, 826–832 (2016)." />
<meta name="citation_reference" content="te Velthuis, A. J. W. Common and unique features of viral RNA-dependent polymerases. Cell Mol Life Sci 71, 4403–4420 (2014)." />
<meta name="citation_reference" content="Selisko, B., Papageorgiou, N., Ferron, F. &amp; Canard, B. Structural and functional basis of the fidelity of nucleotide selection by Flavivirus RNA-dependent RNA polymerases. Viruses 10, 59 (2018)." />
<meta name="citation_reference" content="McKenna, C. E. et al. Inhibitors of viral nucleic acid polymerases. Pyrophosphate analogues. ACS Symposium Series 401, 1–16. Chapter 1 (1989)." />
<meta name="citation_reference" content="Öberg, B. Rational design of polymerase inhibitors as antiviral drugs. Antiviral Res 71, 90–95 (2006)." />
<meta name="citation_reference" content="Eltahla, A. A., Luciani, F., White, P. A., Lloyd, A. R. &amp; Bull, R. A. Inhibitors of the hepatitis C virus polymerase; mode of action and resistance. Viruses 7, 5206–5224 (2015)." />
<meta name="citation_reference" content="De Clercq, E. &amp; Li, G. Approved antiviral drugs over the past 50 years. Clin Microbiol Rev 29, 695–747 (2016)." />
<meta name="citation_reference" content="Kayali, Z. &amp; Schmidt, W. N. Finally sofosbuvir: an oral anti-HCV drug with wide performance capability. Pharmgenomics Pers Med 7, 387–398 (2014)." />
<meta name="citation_reference" content="Sofia, M. J. et al. Discovery of a β-D-2′-Deoxy-2′-α-fluoro-2′-β-C-methyluridine nucleotide prodrug (PSI-7977) for the treatment of hepatitis C virus. J Med Chem 53, 7202–7218 (2010)." />
<meta name="citation_reference" content="Gitto, S., Gamal, N. &amp; Andreone, P. NS5A inhibitors for the treatment of hepatitis C infection. J Viral Hepatitis 24, 180–186 (2017)." />
<meta name="citation_reference" content="Quezada, E. M. &amp; Kane, C. M. The hepatitis C virus NS5A stimulates NS5B during in vitro RNA synthesis in a template specific manner. Open Biochem J 3, 39–48 (2009)." />
<meta name="citation_reference" content="Holshue, M. L. et al., for the Washington State 2019-nCoV Case Investigation Team. First case of 2019 novel coronavirus in the United States. N Eng J Med 382, 929–936 (2020)." />
<meta name="citation_reference" content="Wang, M. et al. Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro. Cell Research, doi.org/10.1038/s41422-020-0282-0 (2020)." />
<meta name="citation_reference" content="Alanazi, A. S., James, E. &amp; Mehellou, Y. The ProTide prodrug technology: where next? ACS Med Chem Lett 10, 2–5 (2019)." />
<meta name="citation_reference" content="De Clercq, E. &amp; Field, H. J. Antiviral prodrugs – the development of successful prodrug strategies for antiviral chemotherapy. Br J Pharmacol 147, 1–11 (2006)." />
<meta name="citation_reference" content="Roberts, S. K. et al. Robust antiviral activity of R1626, a novel nucleoside analog: A randomized, placebo-controlled study in patients with chronic hepatitis C. Hepatol 48, 398–406 (2008)." />
<meta name="citation_reference" content="Gordon, C. J., Tchesnokov, E. P., Feng, J. Y., Porter, D. P. &amp; Götte, M. The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus. J Biol Chem (2020). doi/10.1074/jbc.AC120.013056." />
<meta name="citation_reference" content="Tchesnokov, E. P., Feng, J. Y., Porter, D. P. &amp; Götte, M. Mechanism of inhibition of Ebola virus RNA-dependent RNA polymerase by Remdesivir. Viruses 11, 326 (1-16) (2019)." />
<meta name="citation_reference" content="Fung, A. et al. Efficiency of incorporation and chain termination determines the inhibition potency of 2’-modified nucleotide analogs against hepatitis C virus polymerase. Antimicrob Agents Chemother 58, 3636–3645 (2014)." />
<meta name="citation_reference" content="Arnold, J. J. et al. Sensitivity of mitochondrial transcription and resistance of RNA polymerase II dependent nuclear transcription to antiviral ribonucleosides. PLoS Pathog 8, e1003030 (2012)." />
<meta name="citation_reference" content="Dutartre, H., Bussetta, C., Boretto, J. &amp; Canard, B. General catalytic deficiency of hepatitis C virus RNA polymerase with an S282T mutation and mutually exclusive resistance towards 2’-modified nucleotide analogues. Antimicrob Agents Chemother 50, 4161–4169 (2006)." />
<meta name="citation_reference" content="Elfiky, A. A., Mahdy, S. M. &amp; Elshemey, W. M. Quantitative structure-activity relationship and molecular docking revealed a potency of anti-hepatitis C virus drugs against human corona viruses. J Med Virol 89, 1040–1047 (2017)." />
<meta name="citation_reference" content="Elfiky, A. A. Anti-HCV, nucleotide inhibitors, repurposing against COVID-19. Life Sci 248, 117477 (2020)." />
<meta name="citation_reference" content="Deval, J., Symons, J. A. &amp; Beigelman, L. Inhibition of viral RNA polymerases by nucleoside and nucleotide analogs: therapeutic applications against positive-strand RNA viruses beyond hepatitis C virus. Curr Opin Virol 9, 1–7 (2014)." />
<meta name="citation_reference" content="Fearns, R. &amp; Deval, J. New antiviral approaches for respiratory syncytial virus and other mononegaviruses: Inhibiting the RNA polymerase. Antiviral Res 134, 63–76 (2016)." />
<meta name="citation_reference" content="Ju, J. et al. Four-color DNA sequencing by synthesis using cleavable fluorescent nucleotide reversible terminators. Proc Natl Acad Sci USA 103, 19635–19640 (2006)." />
<meta name="citation_reference" content="Guo, J. et al. Four-color DNA sequencing with 3’-O-modified nucleotide reversible terminators and chemically cleavable fluorescent dideoxynucleotides. Proc Natl Acad Sci USA 105, 9145–9150 (2008)." />
<meta name="citation_reference" content="Ruparel, H. et al. Design and synthesis of a 3’-O-allyl photocleavable fluorescent nucleotide as a reversible terminator for DNA sequencing by synthesis. Proc Natl Acad Sci USA 102, 5932–5937 (2005)." />
<meta name="citation_reference" content="Tabor, S. &amp; Richardson, C. C. A single residue in DNA polymerases of Escherichia coli DNA polymerase I family is critical for distinguishing between deoxy- and dideoxynucleotides. Proc Natl Acad Sci USA 92, 6339–6343 (1995)." />
<meta name="citation_reference" content="Camerman N., Mastropaolo, D. &amp; Camerman, A. Structure of the anti-human immunodeficiency virus agent 3’-fluoro-3’deoxythymidine and electronic charge calculations for 3’-deoxythymidines. Proc Natl Acad Sci USA 87, 3534–3537 (1990)." />
<meta name="citation_reference" content="Kim, E.-Y., Vrang, L., Öberg, B. &amp; Merigan, T. C. Anti-HIV type 1 activity of 3□-fluoro-3□-deoxythymidine for several different multidrug-resistant mutants. AIDS Res Human Retroviruses 17, 401–407 (2001)." />
<meta name="citation_reference" content="Ghosn, J. et al. Antiviral activity of low-dose alovudine in antiretroviral-experienced patients: results from a 4-week randomized, double-blind, placebo-controlled dose-ranging trial. HIV Med 8, 142–147 (2007)." />
<meta name="citation_reference" content="Mitsuya, H. et al. 3’-Azido-3’-deoxythymidine (BW A509U): an antiviral agent that inhibits the infectivity and cytopathic effect of human T-lymphotropic virus type III/lymphadenopathy-associated virus in vitro. Proc Natl Acad Sci USA 82, 7096–7100 (1985)." />
<meta name="citation_reference" content="Yarchoan, R. et al. Administration of 3’-azido-3’-deoxythymidine, an inhibitor of HTLV-III/LAV replication, to patients with AIDS or AIDS-related complex. Lancet 327, 575–580 (1986)." />
<meta name="citation_reference" content="Mitsuya, H., Yarchoan, R. &amp; Broder, S. Molecular targets for AIDS therapy. Science 249, 1533–1544 (1990)." />
<meta name="citation_reference" content="Furman, P. A. et al. Phosphorylation of 3’-azido-3’-deoxythymidine and selective interaction of the 5’-triphosphate with human immunodeficiency virus reverse transcriptase. Proc Natl Acad Sci USA 83, 8333–8337 (1986)." />
<meta name="citation_reference" content="D’Andrea, G., Brisdelli, F. &amp; Bozzi, A. AZT: An old drug with new perspectives. Curr Clin Pharmacol 3, 20–37 (2008)." />
<meta name="citation_reference" content="Subissi, L. et al. One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities. Proc Natl Acad Sci USA 111, E3900–E3909 (2014)." />
<meta name="citation_reference" content="Kirchdoerfer, R. N. &amp; Ward, A. B. Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors. Nature Commun 10, 2342 (2019)." />
<meta name="citation_reference" content="Minskaia, E. et al. Discovery of an RNA virus 3’→5’ exoribonuclease that is critically involved in coronavirus RNA synthesis. Proc Natl Acad Sci USA 103, 5108–5113 (2006)." />
<meta name="citation_reference" content="Agostini, M. L. et al. Coronavirus susceptibility to the antiviral Remdesivir (GS-5734) is mediated by the viral polymerase and the proofreading exonuclease. Mbio 9, e00221–e00218 (2018)." />
<meta name="citation_reference" content="Ju, J., Li, Z., Edwards, J. R. &amp; Itagaki, Y. Massive Parallel Method for Decoding DNA and RNA. United States Patent 6,664,079 (2003)." />
<meta name="citation_reference" content="Xu, S. et al. In vitro selection of resistance to sofosbuvir in HCV replicons of genotype-1 to −6. Antivir Ther 22, 587–597 (2017)." />
<meta name="citation_reference" content="Ju, J., Li, Z., Edwards, J. R. &amp; Itagaki, Y. Massive parallel method for decoding DNA and RNA, United States Patent 9,868,985 (2018)." />
<meta name="citation_reference" content="Canard, B. &amp; Sarfati, R. S. DNA polymerase fluorescent substrates with reversible 3′-tags. Gene 148, 1–6 (1994)." />
<meta name="citation_reference" content="Ruparel, H. Novel DNA sequencing and genotyping approaches using modified nucleotide analogues. Doctoral Dissertation, Columbia University. 72–112 (2005)." />
<meta name="citation_reference" content="Axelrod, V. D., Vartikyan, R. M., Aivazashvili, V. A. &amp; Beabealashvili, R. S. Specific termination of RNA polymerase synthesis as a method of RNA and DNA sequencing. Nucleic Acids Res 5, 3549–3563 (1978)." />
<meta name="citation_reference" content="Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. J Mol Biol 215, 403–410 (1990)." />
<meta name="twitter:title" content="Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.[1][1] Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.[2][2] Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3’-fluoro-3’-deoxythymidine triphosphate and 3’-azido-3’-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.

 [1]: #ref-1
 [2]: #ref-2" />
<meta name="og-title" property="og:title" content="Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.[1][1] Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.[2][2] Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3’-fluoro-3’-deoxythymidine triphosphate and 3’-azido-3’-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.

 [1]: #ref-1
 [2]: #ref-2" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.12.989186v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.12.989186v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.12.989186v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1196270" />
    <title>Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"cyWuKsEsC26c3UpLKuIk1eGP7faaP8ZdvsyL-XKLemc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1196270","apath":"\/biorxiv\/early\/2020\/03\/14\/2020.03.12.989186.atom","pisa":"biorxiv;2020.03.12.989186v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.12.989186v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.12.989186v1","title":"Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase","identifier":"node\/1196270"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224PwPVoKiuu6AhnjaH2DyCbvamAHKl4FPZcpbj0N1YG+PAQw4FWGAW5IclW9RAWYr\\\\\\\/jIuCXJKz5f6Xy2\\\\\\\/3u9P7+QHFQCpRE2gx\\\\\\\/aaLSvum6JKKIgs+O6fc1j++1fpp2qYRLLFu9JRXJMs3oU6mBGo3p2uq8lnZAF1AVXPzEtzQ2ENwCRfql7m7Vfiht+Er7IN+0xhIBWrYCxybEvVcs+dT5xQpWktOG6b8O6SiSy+XWRcGP9LhE86JWS0dOiyfaV7GPxsV+vVhm6AxrFgIQ2BInxOVGDrGPz+x19kGEaWcv0sADPBqOO9n0BjQuQ8CJ6lGcHbO8fsG\\\\\\\/wh2ZIk4JBUvApQOacKorRj8PAv\\\\\\\/kqUue01DzrDdbqbW0ce23scjhkpfNFxZzzIEqGLYDPzspMwm+Mic8ufd6m\\\\\\\/w1WzhrAcgOn16BA1xZqKo6YJK05u70D9+RWJUfiA8ULarmDlrodze9pSsyEGvpjXONQLtz7k3OI2I4tjySdj6TwvGHwIyErKfmoA16tA0p7Ou\\\\\\\/F1kNR7aRenuCP1G5UESmZsZW54duJLkQVjtvmr3vmh27edq8VNWKQR70vTo0UgQzxnZbPHRjoo7g7g\\\\\\\/vOCd5xMBeRkIF3kKEMFonhumTM0g\\\\\\\/ihGwoFfSjnFJeo3Ii6bRiTMeIn9xtmtKUBiYrBhKiDfrtSNFS58ovYJROx4OReFU2juRtRMm8ZNwbH4wIWjEVym6d+r5YMBh11xjCcajyHey56I+iPXCxys62jgp+Adxg6U\\\\\\\/WB9cw2TeDtT78\\\\\\\/qvlUee7NsdXD7dE2sFORNzRGDcVedZ0BmadJ\\\\\\\/bjJcAvl\\\\\\\/3+u8YmLt2THS\\\\\\\/V2MQhcyBQN7f3oy3APCD8NvZUwLG9+WWhw3GDqjcCLC4eo405Z19N2qTysxTI63ursOZf9JtE0KhzAmuWsdHu0SsRnRK+pCIBAhTFRWY1TaLBTksXdlVRl\\\\\\\/k\\\\\\\/5LOY1Dgj8Mt3yoAgvmaPoexQTp4x2erUPE0q6fqpfkekm4GjNRAzZX2sJQKhjkX6roy02ws7it6SZjx7PfUIersj3Q3m4duZ2xO2t+wnRWmfzlUG\\\\\\\/ci\\\\\\\/m15RVUy1u3oDff+CgIhxWEZfSxNBO6A\\\\\\\/gepUb3l+AJDTFWQhLWRbYXYIw8ff3oYp1nj5Z6fjrYr\\\\\\\/ufHiaY8rhFXrILFGgoayH\\\\\\\/zD7DN2KYs89iAppORThXNlHFV5NW0sxE1QfupSHU7gagzamSh6WXz5kLMexiZpN5axZMVHUFIwiE2CC6SXaDZew1HYEo4CmDN4eCuRpdYL+5yNZw28RmQ\\\\\\\/llMWA085Oc3IDLclH8fZ36VmKIu7X0KbowwmmOOlhwYNFhrl2Qlhg\\\u0022,\\\u0022iv\\\u0022:\\\u00225X9HNJOEf+VYW+m7OAuw4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225efee89528e5b0006bbc1651764b56deb9c906c85e23cf66dc6020f619fb8f77\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.12.989186v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1196270 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.12.989186v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m2JDis3BL7YxwaYxG6hlOFtFB0NVcQyDRwlmlEKmHw4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196270" id="node1196270" data-pisa="biorxiv;2020.03.12.989186v1" data-pisa-master="biorxiv;2020.03.12.989186" data-apath="/biorxiv/early/2020/03/14/2020.03.12.989186.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingyue</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shiv</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Steffen</span> <span class="nlm-surname">Jockusch</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Minchen</span> <span class="nlm-surname">Chien</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chuanjuan</span> <span class="nlm-surname">Tao</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Irina</span> <span class="nlm-surname">Morozova</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Kalachikov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">James J.</span> <span class="nlm-surname">Russo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.12.989186 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1196270" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jingyue Ju </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Pharmacology, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jingyue%2BJu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ju%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJingyue%2BJu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dj222@columbia.edu" class="" data-icon-position="" data-hide-link-title="0">dj222@columbia.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaoxu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shiv Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiv%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiv%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steffen Jockusch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Chemistry, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steffen%2BJockusch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jockusch%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteffen%2BJockusch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Minchen Chien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Minchen%2BChien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chien%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMinchen%2BChien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chuanjuan Tao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuanjuan%2BTao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuanjuan%2BTao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Irina Morozova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irina%2BMorozova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morozova%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrina%2BMorozova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Sergey Kalachikov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergey%2BKalachikov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalachikov%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASergey%2BKalachikov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Robert N. Kirchdoerfer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Biochemistry, University of Wisconsin-Madison</span>, Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Institute of Molecular Virology, University of Wisconsin-Madison</span>, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BN.%2BKirchdoerfer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchdoerfer%20RN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BN.%2BKirchdoerfer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">James J. Russo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genome Technology and Biomolecular Engineering, Columbia University</span>, New York, NY 10027;</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemical Engineering, Columbia University</span>, New York, NY 10027;</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BRusso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Russo%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BRusso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.12.989186v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1196270" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1196270/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.989186v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1196270" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1196270/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.989186v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1196270" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1196270/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.989186v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1196270" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1196270/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.12.989186v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1196270" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1196270/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-2">SARS-CoV-2, a member of the coronavirus family, has caused a global public health emergency.<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup> Here, using model polymerase extension experiments, we demonstrate that the activated triphosphate form of Sofosbuvir is incorporated by low-fidelity polymerases and SARS-CoV RNA-dependent RNA polymerase (RdRp), and blocks further incorporation by these polymerases; the activated triphosphate form of Sofosbuvir is not incorporated by a host-like high-fidelity DNA polymerase. Using the same molecular insight, we selected two other anti-viral agents, Alovudine and AZT (an FDA approved HIV/AIDS drug) for evaluation as inhibitors of SARS-CoV RdRp. We demonstrate the ability of two HIV reverse transcriptase inhibitors, 3’-fluoro-3’-deoxythymidine triphosphate and 3’-azido-3’-deoxythymidine triphosphate (the active triphosphate forms of Alovudine and AZT), to be incorporated by SARS-CoV RdRp where they also terminate further polymerase extension. Given the 98% amino acid similarity of the SARS-CoV and SARS-CoV-2 RdRps, we expect these nucleotide analogues would also inhibit the SARS-CoV-2 polymerase. These results offer guidance to further modify these nucleotide analogues to generate more potent broad-spectrum anti-coronavirus agents.</p></div><div id="sec-1" class="subsection"><p id="p-3">The recent appearance of a new coronaviral infection, COVID-19, in Wuhan, China, and its worldwide spread, has made international headlines. Already more than 3,000 deaths have been ascribed to this virus, and COVID-19 has reached near pandemic status. The virus has been isolated from the lower respiratory tracts of patients with pneumonia, sequenced and visualized by electron microscopy.<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a></sup> The virus, designated SARS-CoV-2, is a new member of the subgenus <em>Sarbecovirus</em>, in the Orthocoronavirinae subfamily, but is distinct from MERS-CoV and SARS-CoV.<sup><a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a></sup> The coronaviruses are single strand RNA viruses, sharing properties with other single-stranded RNA viruses such as hepatitis C virus (HCV), West Nile virus, Marburg virus, HIV virus, Ebola virus, dengue virus, and rhinoviruses. In particular, coronaviruses and HCV are both positive-sense single-strand RNA viruses,<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> and thus have a similar replication mechanism requiring a RNA-dependent RNA polymerase (RdRp).</p><p id="p-4">The coronavirus life cycle has been described.<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup> Briefly, the virus enters the cell by endocytosis, is uncoated, and ORF1a and ORF1b of the positive strand RNA is translated to produce nonstructural protein precursors, including a cysteine protease and a serine protease; these further cleave the precursors to form mature, functional helicase and RNA-dependent RNA polymerase. A replication-transcription complex is then formed, which is responsible for making more copies of the RNA genome via a negative-sense RNA intermediate, as well as the structural and other proteins encoded by the viral genome. The viral RNA is packaged into viral coats in the endoplasmic reticulum-Golgi intermediate complex, after which exocytosis results in release of viral particles for subsequent infectious cycles. Potential inhibitors have been designed to target nearly every stage of this process.<sup><a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a></sup> However, despite decades of research, no effective drug is currently approved to treat serious coronavirus infections such as SARS, MERS, and COVID-19.</p><p id="p-5">One of the most important druggable targets for coronaviruses is the RNA-dependent RNA polymerase (RdRp). This polymerase is highly conserved at the protein level among different positive sense RNA viruses, to which coronaviruses and HCV belong, and shares common structural features in these viruses.<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup> Like RdRps in other viruses, the coronavirus enzyme is highly error-prone,<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> which might increase its ability to accept modified nucleotide analogues. Nucleotide and nucleoside analogues that inhibit polymerases are an important group of anti-viral agents.<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup></p><p id="p-6">Based on our analysis of hepatitis C virus and coronavirus replication, and the molecular structures and activities of viral inhibitors, we previously reasoned that the FDA-approved heptatitis C drug EPCLUSA (Sofosbuvir/Velpatasvir) should inhibit coronaviruses, including SARS-CoV-2.<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a></sup> Sofosbuvir is a pyrimidine nucleotide analogue prodrug with a hydrophobic masked phosphate group enabling it to enter infected eukaryotic cells, and then converted into its active triphosphate form by cellular enzymes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). In this activated form, it inhibits the HCV RNA-dependent RNA polymerase NS5B.<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup> The activated drug (2’-F,Me-UTP) binds in the active site of the RdRp, where it is incorporated into RNA, and due to fluoro and methyl modifications at the 2’ position, inhibits further RNA chain extension, thereby halting RNA replication and stopping viral growth. It acts as an RNA polymerase inhibitor by competing with natural ribonucleotides. Velpatasvir inhibits NS5A, a key protein required for HCV replication. NS5A enhances the function of RNA polymerase NS5B during viral RNA synthesis.<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup></p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conversion of Sofosbuvir to active triphosphate (2&#x2019;-F,Me-UTP) in vivo to inhibit viral polymerases. Adapted from 11." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Conversion of Sofosbuvir to active triphosphate (2&#x2019;-F,Me-UTP) &lt;em&gt;in vivo&lt;/em&gt; to inhibit viral polymerases.&lt;/span&gt; &lt;em&gt;Adapted from&lt;/em&gt; &lt;sup&gt;11&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F1.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F1.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <span class="caption-title">Conversion of Sofosbuvir to active triphosphate (2’-F,Me-UTP) <em>in vivo</em> to inhibit viral polymerases.</span><p id="p-7" class="first-child"><em>Adapted from</em> <sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">There are many other RNA polymerase inhibitors that have been evaluated as antiviral drugs. A related purine nucleotide prodrug, Remdesivir (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2b</a>), was developed by Gilead to treat Ebola virus infections, though not successfully, and is currently in clinical trials for treating COVID-19.<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup> In contrast to Sofosbuvir (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2a</a>), both the 2’- and 3’-OH groups in Remdesivir (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2b</a>) are unmodified, but a cyano group at the 1’ position serves to inhibit the RdRp in the active triphosphate form. In addition to the use of hydrophobic groups to mask the phosphate in the Protide-based prodrug strategy,<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> as with Sofosbuvir and Remdesivir, there are other classes of nucleoside prodrugs including those based on ester derivatives of the ribose hydroxyl groups to enhance cellular delivery.<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup></p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of structures of prodrug viral inhibitors. Top: Prodrug (phosphoramidate) form; Bottom: Active triphosphorylated form." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Comparison of structures of prodrug viral inhibitors.&lt;/span&gt; Top: Prodrug (phosphoramidate) form; Bottom: Active triphosphorylated form.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F2.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F2.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <span class="caption-title">Comparison of structures of prodrug viral inhibitors.</span><p id="p-9" class="first-child">Top: Prodrug (phosphoramidate) form; Bottom: Active triphosphorylated form.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">The replication cycle of HCV<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup> is very similar to that of the coronaviruses.<sup><a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a></sup> Analyzing the structure of the active triphosphate form of Sofosbuvir (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2a</a>) compared to that of Remdesivir (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2b</a>), both of which have already been shown to inhibit the replication of specific RNA viruses (Sofosbuvir for HCV, Remdesivir for SARS-CoV-2), we noted in particular that the 2’-modifications in Sofosbuvir (a fluoro and a methyl group) are substantially smaller than the 1’-cyano group and the 2’-OH group in Remdesivir. The bulky cyano group in close proximity to the 2’-OH may lead to steric hindrance that will impact the incorporation efficiency of the activated form of Remdesivir. Interestingly, it was recently reported that, using the MERS-CoV polymerase, the triphosphate of Remdesivir was preferentially incorporated relative to ATP in solution assays.<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> Nevertheless, it has been shown that the active triphosphate form of Remdesivir does not cause complete polymerase reaction termination and actually delays polymerase termination in Ebola virus and respiratory syncytial virus, likely due to its 1’-cyano group and the free 2’-OH and 3’-OH groups.<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup> Compared to the active form of Sofosbuvir (2’-fluoro-2’-methyl-UTP), two other nucleotide inhibitors with related structures were reviewed: 2’-fluoro-UTP is incorporated by polymerase, but RNA synthesis may continue past the incorporated nucleotide analogue;<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> 2’-C-methyl-UTP has been shown to terminate the reaction catalyzed by HCV RdRp,<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> but proofreading mechanisms can revert this inhibition in mitochondrial DNA-dependent RNA polymerases.<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> Additionally, HCV develops resistance to 2’-C-methyl-UTP due to mutations of the RdRp.<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> A computational study published in 2017<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> considered the ability of various anti-HCV drugs to dock in the active site of SARS and MERS coronavirus RdRps as potential inhibitors. Recently, Elfiky used a computational approach to predict that Sofosbuvir, IDX-184, Ribavirin, and Remidisvir might be potent drugs against COVID-19.<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup></p><p id="p-11">Thus, based on our analysis of the biological pathways of hepatitis C and coronaviruses, the molecular structures and activities of viral inhibitors, model polymerase and SARS-CoV RdRp extension experiments described below, and the efficacy of Sofosbuvir in inhibiting the HCV RdRp, we expect that Sofosbuvir or its modified forms should also inhibit the SARS-CoV-2 polymerase.<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a></sup></p><p id="p-12">The active triphosphate form of Sofosbuvir (2’-F,Me-UTP) was shown to be incorporated by HCV RdRp and prevent any further incorporation by this polymerase.<sup><a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> Other viral polymerases have also been shown to incorporate active forms of various anti-viral prodrugs to inhibit replication.<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> Since, at the time of the preparation of this manuscript, we did not have access to the RdRp from SARS-CoV-2, we first selected two groups of polymerases to test the termination efficiency of the active form of Sofosbuvir, one group with high fidelity behavior with regard to incorporation of modified nucleotide analogues, which one would expect for host cell polymerases, the other group with low fidelity mimicking viral polymerases, as well as the RdRp from SARS-CoV, the virus causing the 2003 and subsequent outbreaks of SARS. Our rationale is that the low fidelity viral-like enzymes would incorporate 2’-F,Me-UTP and stop further replication, while the high fidelity polymerases, typical of host cell polymerases, would not. Experimental proof for termination of the SARS-CoV polymerase catalyzed RNA replication would provide further support for this rationale, indicating that Sofosbuvir or its modified forms will inhibit SARS-CoV-2.</p><p id="p-13">We first carried out DNA polymerase extension reactions with the active form of Sofosbuvir (2’-F,Me-UTP) using Thermo Sequenase as an example of high fidelity, host-like polymerases, and two mutated DNA polymerases which are known to be more promiscuous in their ability to incorporate modified nucleotides, Therminator II and Therminator IX, as examples of viral-like low fidelity enzymes. A DNA template-primer complex, in which the next two available bases were A (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>), was incubated with either 2’-F,Me-UTP (structure shown in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2a</a>), or dTTP as a positive control, in the appropriate polymerase buffer. If the 2’-F,Me-UTP is incorporated and inhibits further incorporation, a single-base primer extension product will be produced. By contrast, dTTP incorporation will result in primer extension by 2 bases. After performing the reactions, we determined the molecular weight of the extension products using MALDI-TOF-mass spectrometry (MALDI-TOF MS).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Incorporation of 2&#x2019;-F,Me-UTP by two low-fidelity polymerases as terminators but not by a high-fidelity polymerase. The sequence of the primer and template used for these extension reactions is shown at the top of the figure. Polymerase extension reactions were performed by incubating the primer and template with 2&#x2019;-F,Me-UTP and the appropriate reaction buffer for the specific enzyme, followed by detection of the reaction products by MALDI-TOF MS. The MS spectra for Therminator II (T2) in (a) and for Therminator IX (T9) in (b) indicates single-base incorporation and termination, while the MS spectrum for Thermo Sequenase (TS) in (c) indicates no incorporation, showing only a primer peak. The accuracy for m/z determination is &#xB1; 10 Da." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Incorporation of 2&#x2019;-F,Me-UTP by two low-fidelity polymerases as terminators but not by a high-fidelity polymerase.&lt;/span&gt; The sequence of the primer and template used for these extension reactions is shown at the top of the figure. Polymerase extension reactions were performed by incubating the primer and template with 2&#x2019;-F,Me-UTP and the appropriate reaction buffer for the specific enzyme, followed by detection of the reaction products by MALDI-TOF MS. The MS spectra for Therminator II (T2) in (a) and for Therminator IX (T9) in (b) indicates single-base incorporation and termination, while the MS spectrum for Thermo Sequenase (TS) in (c) indicates no incorporation, showing only a primer peak. The accuracy for m/z determination is &#xB1; 10 Da.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F3.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F3.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <span class="caption-title">Incorporation of 2’-F,Me-UTP by two low-fidelity polymerases as terminators but not by a high-fidelity polymerase.</span><p id="p-14" class="first-child">The sequence of the primer and template used for these extension reactions is shown at the top of the figure. Polymerase extension reactions were performed by incubating the primer and template with 2’-F,Me-UTP and the appropriate reaction buffer for the specific enzyme, followed by detection of the reaction products by MALDI-TOF MS. The MS spectra for Therminator II (T2) in (a) and for Therminator IX (T9) in (b) indicates single-base incorporation and termination, while the MS spectrum for Thermo Sequenase (TS) in (c) indicates no incorporation, showing only a primer peak. The accuracy for m/z determination is ± 10 Da.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">As seen in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figs. 3a</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3b</a>, when the primer-template complex (sequences shown at top of <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3</a>) and 2’-F,Me-UTP were incubated with the low fidelity 9°N polymerase mutants,<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>–<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> Therminator II (T2) and Therminator IX (T9), we observed single product peaks with molecular weights of 5492 Da and 5488 Da, indicating single base extension in the polymerase reaction. Thus 2’-F,Me-UTP was able to be incorporated and block further nucleotide incorporation. In contrast, when the extension reactions were carried out with high-fidelity Thermo Sequenase DNA polymerase (TS),<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> there was no incorporation, as evidenced by a single primer peak at 5172 Da (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3c</a>). This supports our rationale that Thermo Sequenase, a high fidelity enzyme originally designed for accurate Sanger sequencing, will not incorporate 2’-F,Me-UTP, while a low-fidelity polymerase, such as T2 and T9, will incorporate 2’-F,Me-UTP and stop further nucleotide incorporation. When dTTP was used as a positive control with these three enzymes, incorporation continued past the first A in the template, resulting in a higher molecular weight peak.</p><p id="p-16">These results demonstrate that lower fidelity polymerases will have a high likelihood of incorporating 2’-F,Me-UTP and inhibit viral RNA replication, whereas high fidelity enzymes, more typical of the host DNA and RNA polymerases, will have a low likelihood of being inhibited by 2’-F,Me-UTP. Anti-viral drug design based on this principle may lead to potent viral polymerase inhibitors with fewer side effects. To provide further proof that SARS-CoV-2 RdRp might be inhibited by 2’-F,Me-UTP, we next tested the ability of this molecule to be incorporated into an RNA primer to terminate the reaction catalyzed by the RdRp from SARS-CoV, using an RNA template. As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>, the active triphosphate form of the drug not only was incorporated by the RdRp, but prevented further incorporation, behaving as a terminator in the polymerase reaction.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Incorporation of 2&#x2019;-F,Me-UTP, 3&#x2019;-F-dTTP and 3&#x2019;-N3-dTTP by SARS-CoV RdRp to terminate the polymerase reaction. The sequence of the primer and template used for these extension reactions, which are within the N1 coding sequence of the SARS-CoV-2 genome, is shown at the top of the figure. Polymerase extension reactions were performed by incubating (a) 2&#x2019;-F,Me-UTP, (b) 3&#x2019;-F-dTTP, and (c) 3&#x2019;-N3-dTTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the Methods section. For comparison, data for extension with UTP are presented in Extended Data Fig. 2. The accuracy for m/z determination is &#xB1; 10 Da." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Incorporation of 2&#x2019;-F,Me-UTP, 3&#x2019;-F-dTTP and 3&#x2019;-N&lt;sub&gt;3&lt;/sub&gt;-dTTP by SARS-CoV RdRp to terminate the polymerase reaction.&lt;/span&gt; The sequence of the primer and template used for these extension reactions, which are within the N1 coding sequence of the SARS-CoV-2 genome, is shown at the top of the figure. Polymerase extension reactions were performed by incubating (a) 2&#x2019;-F,Me-UTP, (b) 3&#x2019;-F-dTTP, and (c) 3&#x2019;-N&lt;sub&gt;3&lt;/sub&gt;-dTTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the Methods section. For comparison, data for extension with UTP are presented in Extended Data Fig. 2. The accuracy for m/z determination is &#xB1; 10 Da.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F4.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F4.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <span class="caption-title">Incorporation of 2’-F,Me-UTP, 3’-F-dTTP and 3’-N<sub>3</sub>-dTTP by SARS-CoV RdRp to terminate the polymerase reaction.</span><p id="p-17" class="first-child">The sequence of the primer and template used for these extension reactions, which are within the N1 coding sequence of the SARS-CoV-2 genome, is shown at the top of the figure. Polymerase extension reactions were performed by incubating (a) 2’-F,Me-UTP, (b) 3’-F-dTTP, and (c) 3’-N<sub>3</sub>-dTTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the Methods section. For comparison, data for extension with UTP are presented in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Extended Data Fig. 2</a>. The accuracy for m/z determination is ± 10 Da.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Based on our similar insight related to their molecular structures and previous antiviral activity studies, in comparison with Sofosbuvir, we selected the triphosphate forms of Alovudine (3’-deoxy-3’-fluorothymidine) and azidothymidine (AZT, the first FDA approved drug for HIV/AIDS) for evaluation as inhibitors of the SARS-CoV RdRp. These two molecules share a similar backbone structure (base and ribose) to Sofosbuvir, but have fewer modification sites and less steric hindrance. Furthermore, because these modifications on Alovudine and AZT are on the 3’ carbon in place of the OH group, they directly prevent further incorporation of nucleotides leading to permanent termination of RNA synthesis and replication of the virus.</p><p id="p-19">Alovudine is one of the most potent inhibitors of HIV reverse transcriptase and HIV-1 replication. This promising drug was discontinued after a Phase II trial due to its hematological toxicity.<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> However, subsequent <em>in vitro</em> studies showed Alovudine was very effective at suppressing several nucleoside/nucleotide reverse transcriptase inhibitor (NRTI)-resistant HIV-1 mutants.<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> New clinical studies were then carried out in which low doses of Alovudine were given as supplements to patients showing evidence of infection by NRTI resistant HIV strains and not responding well to their current drug regimen. A 4-week course of 2 mg/day Alovudine reduced viral load significantly, and was relatively well tolerated with no unexpected adverse events.<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup></p><p id="p-20">AZT is another antiretroviral medication which has long been used to prevent and treat HIV/AIDS.<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>–<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> Upon entry into the infected cells, similar to Alovudine, cellular enzymes convert AZT into the effective 5’-triphosphate form (3’-N<sub>3</sub>-dTTP, structure shown in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2d</a>), which competes with dTTP for incorporation into DNA by HIV-reverse transcriptase resulting in termination of HIV’s DNA synthesis.<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> Since the side effects and toxicity of AZT are well understood, novel methodologies have been directed at enhancing AZT plasma levels and its bioavailability in all human organs in order to improve its therapeutic efficacy. Among these possibilities, an AZT prodrug strategy was proposed.<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup></p><p id="p-21">We thus assessed the ability of 3’-N<sub>3</sub>-dTTP and 3’-F-dTTP, the active triphosphate forms of AZT and Alovudine, along with 2’-F,Me-UTP, to be incorporated by SARS-CoV RdRp into an RNA primer and terminate the polymerase reaction.</p><p id="p-22">The RdRp of SARS-CoV, referred to as nsp12, and its two protein cofactors, nsp7 and nsp8, shown to be required for the processive polymerase activity of nsp12, were cloned and purified as described.<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> These three viral gene products have high homology (e.g., 96% identity and 98% similarity for nsp12, with similar homology levels at the amino acid level for nsp7 and nsp8) to the equivalent gene products from SARS-CoV-2, the causative agent of COVID-19. A detailed description of the homologies of nsp7, nsp8 and nsp12 is included in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Extended Data Fig. 1</a> which highlights key functional motifs in nsp12 described by Kirchdoerfer and Ward.<sup><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup> Of these, Motifs A, B, E, F and G are identical in SARS-CoV and SARS-CoV-2 at the amino acid level, and Motifs C and D display only conservative substitutions.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2. Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.51 Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. (A) nsp7; (B) nsp8; (C) nsp12: functional motifs42 are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.5 Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical fingers, palm, and thumb of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2.&lt;/span&gt; Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.&lt;sup&gt;51&lt;/sup&gt; Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. &lt;strong&gt;&lt;em&gt;(A)&lt;/em&gt;&lt;/strong&gt; nsp7; &lt;strong&gt;&lt;em&gt;(B)&lt;/em&gt;&lt;/strong&gt; nsp8; &lt;strong&gt;&lt;em&gt;(C)&lt;/em&gt;&lt;/strong&gt; nsp12: functional motifs&lt;sup&gt;42&lt;/sup&gt; are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.&lt;sup&gt;5&lt;/sup&gt; Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical &lt;em&gt;fingers, palm&lt;/em&gt;, and &lt;em&gt;thumb&lt;/em&gt; of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-5.medium.gif" width="311" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-5.medium.gif" width="311" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2. Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.51 Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. (A) nsp7; (B) nsp8; (C) nsp12: functional motifs42 are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.5 Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical fingers, palm, and thumb of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2.&lt;/span&gt; Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.&lt;sup&gt;51&lt;/sup&gt; Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. &lt;strong&gt;&lt;em&gt;(A)&lt;/em&gt;&lt;/strong&gt; nsp7; &lt;strong&gt;&lt;em&gt;(B)&lt;/em&gt;&lt;/strong&gt; nsp8; &lt;strong&gt;&lt;em&gt;(C)&lt;/em&gt;&lt;/strong&gt; nsp12: functional motifs&lt;sup&gt;42&lt;/sup&gt; are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.&lt;sup&gt;5&lt;/sup&gt; Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical &lt;em&gt;fingers, palm&lt;/em&gt;, and &lt;em&gt;thumb&lt;/em&gt; of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-6.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-6.medium.gif" width="328" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2. Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.51 Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. (A) nsp7; (B) nsp8; (C) nsp12: functional motifs42 are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.5 Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical fingers, palm, and thumb of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2.&lt;/span&gt; Protein sequences are from NCBI Protein Database, accession ID&#x2019;s as indicated. Sequences were aligned with blastp.&lt;sup&gt;51&lt;/sup&gt; Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. &lt;strong&gt;&lt;em&gt;(A)&lt;/em&gt;&lt;/strong&gt; nsp7; &lt;strong&gt;&lt;em&gt;(B)&lt;/em&gt;&lt;/strong&gt; nsp8; &lt;strong&gt;&lt;em&gt;(C)&lt;/em&gt;&lt;/strong&gt; nsp12: functional motifs&lt;sup&gt;42&lt;/sup&gt; are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.&lt;sup&gt;5&lt;/sup&gt; Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical &lt;em&gt;fingers, palm&lt;/em&gt;, and &lt;em&gt;thumb&lt;/em&gt; of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-7.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-7.medium.gif" width="440" height="307"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F5/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 1</span> <span class="caption-title">Protein sequence alignments for nsp7, nsp8, and nsp12: SARS-CoV vs SARS-CoV-2.</span><p id="p-34" class="first-child">Protein sequences are from NCBI Protein Database, accession ID’s as indicated. Sequences were aligned with blastp.<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> Consensus is shown in blue between the query and subject sequences; positions of amino acid substitutions are in red; + indicates conservative amino acid substitutions involving amino acids with close physico-chemical properties. <strong><em>(A)</em></strong> nsp7; <strong><em>(B)</em></strong> nsp8; <strong><em>(C)</em></strong> nsp12: functional motifs<sup><a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">42</a></sup> are shown as colored bars underneath the aligned sequences. Comparison of the polymerase complex components (nsp7, nsp8, and nsp12 proteins) shows that these proteins are very similar in SARS-CoV and SARS-CoV-2. There are no indels in any of the three protein pairs. There is only one amino acid substitution in nsp7 (99% sequence identity); nsp8 has 5 amino acid changes (97% sequence identity), out of which 3 are between amino acids with similar properties. Alignment of the nsp12 pair shows that 898 out of 932 amino acids (96%) are identical between SARS-CoV and SARS-CoV-2. Eighteen of the substitutions are between amino acids with similar physico-chemical properties, therefore the level of similarity is higher, at 98%. Most of the amino acid substitutions (24 out of 34) are located within the N-terminal portion of the nsp12 protein. This region corresponds to the NiRAN domain (nidovirus RdRp-associated nucleotidyltransferase; approximately amino acids 1 through 250) which is also less conservative in other coronaviruses.<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup> Within the next region (the interface domain, aa ~250 through 400), the first 15 amino acid positions have multiple substitutions, but the rest of the interface domain is quite conservative. The region beyond the interface domain, corresponding to the nsp12 C-terminus, contains polymerase functional domains. These domains constitute the canonical <em>fingers, palm</em>, and <em>thumb</em> of the polymerase enzyme and contain several motifs that are conservative among coronaviruses (Motifs A through F). Out of the 34 amino acid substitutions in the nsp12 between SARS-CoV and SARS-CoV-2, only three substitutions are located within these motifs, and all three are between similar amino acids.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">We performed polymerase extension assays with 2’-F,Me-UTP, 3’-F-dTTP, 3’-N<sub>3</sub>-dTTP or UTP following the addition of an pre-annealed RNA template and primer to a pre-assembled mixture of the RdRp (nsp12) and two cofactor proteins (nsp7 and nsp8). The extended primer products from the reaction were subjected to MALDI-TOF-MS analysis. The RNA template and primer, corresponding to the N1 epitope region of the N protein of the SARS-CoV-2 virus, were used for the polymerase assay, and their sequences are indicated at the top of <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>. Because there are two As in a row in the next available positions of the template for RNA polymerase extension downstream of the priming site, if the 2’-F,Me-UTP, 3’-F-dTTP or 3’-N<sub>3</sub>-dTTP are incorporated by the viral RdRp, a single nucleotide analogue will be added to the 3’-end of the primer strand. If they are indeed inhibitors of the polymerase, the extension should stop after this incorporation; further 3’-extension should be prevented. In the case of the UTP control reaction, two UTP’s should be incorporated. As shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Extended Data Fig. 2</a>, this is exactly what we observed. In the MALDI-TOF MS trace in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4a</a>, a peak indicative of the molecular weight of a one-base 2’-F,Me-UTP analogue primer extension product was obtained (7217 Da observed, 7214 Da expected). Similarly, in the trace in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4b</a>, a single extension peak indicative of a single-base extension by 3’-F-dTTP is revealed (7203 Da observed, 7198 Da expected), with no further incorporation. And in the trace in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4c</a>, a single extension peak indicative of a single-base extension by 3’-N<sub>3</sub>-dTTP is seen (7227 Da observed, 7218 Da expected), with no evidence of further incorporation. As a positive control, extension by 2 UTPs occurred (7506 Da observed, 7504 Da expected) in the MALDI-TOF MS trace in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Extended Data Fig. 2</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Incorporation of UTP by SARS-CoV RNA-dependent RNA polymerase. The sequence of the primer and template used for this extension reaction is shown at the top of the figure. Polymerase extension reactions were performed by incubating UTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the methods. The accuracy for m/z determination is &#xB1; 10 Da." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-479494722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Incorporation of UTP by SARS-CoV RNA-dependent RNA polymerase.&lt;/span&gt; The sequence of the primer and template used for this extension reaction is shown at the top of the figure. Polymerase extension reactions were performed by incubating UTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the methods. The accuracy for m/z determination is &#xB1; 10 Da.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F6.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F6.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/14/2020.03.12.989186/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Fig. 2</span> <span class="caption-title">Incorporation of UTP by SARS-CoV RNA-dependent RNA polymerase.</span><p id="p-35" class="first-child">The sequence of the primer and template used for this extension reaction is shown at the top of the figure. Polymerase extension reactions were performed by incubating UTP with pre-assembled SARS-CoV polymerase (nsp12, nsp7 and nsp8), the indicated RNA template and primer, and the appropriate reaction buffer, followed by detection of reaction products by MALDI-TOF MS. The detailed procedure is shown in the methods. The accuracy for m/z determination is ± 10 Da.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">In summary, these results demonstrate that the nucleotide analogues 2’-F,Me-UTP, 3’-F-dTTP and 3’-N<sub>3</sub>-dTTP, are permanent terminators for the SARS-CoV RdRp. Their prodrug versions (Sofosbuvir, 3’-F-5’-<em>O</em>-phosphoramidate dT nucleoside and 3’-N<sub>3</sub>-5’-<em>O</em>-phosphoramidate dT nucleoside) can be readily synthesized using the ProTide prodrug approach, as shown in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2a, c</a> and <a id="xref-fig-2-9" class="xref-fig" href="#F2">d</a>, and can be developed as therapeutics for both SARS and COVID-19.</p><p id="p-25">One factor that has confounded the development of RdRp inhibitors in coronaviruses is the presence of a 3’-exonuclease-based proofreading activity such as that associated with nsp14, a key component of the replication-transcription complex in SARS-CoV,<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> and also encoded in SARS-CoV-2. This exonuclease activity can be overcome with the use of 2’-O-methylated nucleotides.<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a></sup> Importantly, since both Sofosbuvir and AZT are FDA approved drugs, where toxicity tests have already been performed, they can be evaluated quickly in laboratory and clinical settings.</p><p id="p-26">We have recently described<sup><a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a></sup> a strategy to design and synthesize viral polymerase inhibitors, by combining the ProTide Prodrug approach<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup> used in the development of Sofosbuvir with the use of 3’-blocking groups that we have previously built into nucleotide analogues that function as terminators for DNA sequencing by synthesis.<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> We reasoned that (i) the hydrophobic phosphate masking groups will allow entry of the compounds into the virus-infected cells, (ii) the 3’-blocking group on the 3’-OH with either free 2’-OH or modifications at the 2’ position will encourage incorporation of the activated triphosphate analogue by viral polymerases but not host cell polymerases, thus reducing any side effects, and (iii) once incorporated, further extension will be prevented by virtue of the 3’-blocking group, thereby completely inhibiting viral replication and potentially overcoming the development of resistance due to the accumulation of new mutations in the RdRp.<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup></p><p id="p-27">Our design criterion is to identify groups for attachment to the 3’-OH with appropriate structural and chemical properties (e.g., size, shape, rigidity, flexibility, polarity, reactivity [e.g., stability to cellular enzymes]),<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> along with appropriate 2’-substitutions, so that they will be incorporated by the viral RdRp, while minimizing incorporation by the host polymerases. We previously used this chemical and structural principle to select a variety of chemical moieties that block the 3’-OH of the nucleotide analogues as polymerase terminators.<sup><a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup></p><p id="p-28">In conclusion, we demonstrated the capability of more tolerant DNA polymerases, as well as SARS CoV RNA-dependent RNA polymerase, which is nearly identical to the SARS-CoV-2 RdRp responsible for COVID-19, to incorporate 2’-F,Me-UTP, the active form of Sofosbuvir, where it serves to terminate the polymerase reaction. We also showed two other nucleotide triphosphates, 3’-F-dTTP, the active form of Alovudine, and 3’-N<sub>3</sub>-dTTP, the active form of AZT, can also be incorporated and terminate further nucleotide extension by the RdRp in the polymerase reaction, potentially preventing further replication of the virus. Prodrug versions of these compounds and their derivatives therefore can be developed as potent broad-spectrum therapeutics for coronavirus infectious diseases, including SARS, MERS and COVID-19.</p></div><div class="section" id="sec-2"><h2 class="">EXTENDED DATA</h2><div class="section" id="sec-3"><h3>Methods</h3><div class="section" id="sec-4"><h4>Extension reactions with DNA polymerases</h4><p id="p-29">Oligonucleotides were purchased from Integrated DNA Technologies (IDT Inc.). The 20 μl extension reactions consisted of 3 μM DNA template and 5 μM DNA primer (sequences shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3</a>), 10 μM 2’-F,Me-UTP (Sierra Bioresearch) or 10 μM dTTP, 1× Thermo Sequenase buffer or 1× ThermoPol buffer (for Therminator enzymes), and either 10 U Thermo Sequenase (GE Healthcare), 4 U Therminator II or 10 U Therminator IX (New England Biolabs). The 1× Thermo Sequenase buffer consists of 26 mM Tris-HCl, pH 9.5 and 6.5 mM MgCl<sub>2</sub>. The 1× ThermoPol buffer contains 20 mM Tris-HCl, pH 8.8, 10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 10 mM KCl, 2 mM MgSO<sub>4</sub>, and 0.1% Triton X-100. Incubations were performed in a thermal cycler using 15 cycles of 30 sec each at 65°C, 45°C and 65°C. Following desalting using an Oligo Clean &amp; Concentrator (Zymo Research), the samples were subjected to MALDI-TOF-MS (Bruker ultrafleXtreme) analysis, following a previously described method.<sup><a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></sup></p></div><div class="section" id="sec-5"><h4>Extension reactions with RNA-dependent RNA polymerase</h4><p id="p-30">Oligonucleotides were purchased from IDT, Inc. Following a published strategy,<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a></sup> the primer and template (sequences shown in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4</a>) were annealed by heating to 70°C for 10 min and cooling to room temperature in 1× reaction buffer. The RNA polymerase mixture consisting of 2 <em>μ</em>M nsp12 and 6 <em>μ</em>M each of cofactors nsp7 and nsp8 was incubated for 15 min at room temperature in a 1:3:3 ratio in 1× reaction buffer. Then 5 <em>μ</em>l of the annealed template primer solution containing 2 <em>μ</em>M template and 1.7 <em>μ</em>M primer in 1× reaction buffer was added to 10 <em>μ</em>l of the RNA polymerase mixture and incubated for an additional 10 min at room temperature. Finally 5 <em>μ</em>l of a solution containing either 2 mM 2’-F,Me-UTP, 2 mM 3’-F-dTTP or 2 mM UTP in 1× reaction buffer was added, and incubation was carried out for 2 hr at 30°C. The final concentrations of reagents in the 20 <em>μ</em>l extension reactions were 1 <em>μ</em>M nsp12, 3 <em>μ</em>M nsp7, 3 <em>μ</em>M nsp8, 425 nM RNA primer, 500 nM RNA template, either 500 <em>μ</em>M 2’-F,Me-UTP (Sierra Bioresearch), 500 <em>μ</em>M 3’-F-dTTP (Amersham Life Sciences), or 500 <em>μ</em>M 3’-N<sub>3</sub>-dTTP (Amersham Life Sciences), and 1× reaction buffer (10 mM Tris-HCl pH 8, 10 mM KCl, 2 mM MgCl<sub>2</sub> and 1 mM β-mercaptoethanol). [In the experiment with UTP shown in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Extended Data Fig. 2</a>, the final concentrations were 500 nM nsp12, 1.5 <em>μ</em>M nsp7, 1.5 <em>μ</em>M nsp8, 425 nM RNA primer, 250 nM RNA template and 500 <em>μ</em>M UTP (Fisher) and the reaction time was 1 h at 30°C.] Following desalting using an Oligo Clean &amp; Concentrator (Zymo Research), the samples were subjected to MALDI-TOF-MS (Bruker ultrafleXtreme) analysis.</p></div></div></div><div class="section" id="sec-6"><h2 class="">Author Contributions</h2><p id="p-32">J.J. conceived and directed the project; the approaches and assays were designed and conducted by J.J., X.L., S.K., S.J., J.J.R., M.C. and C.T., comparative sequence analysis was performed by I.M. and S.K., and SARS-CoV polymerase and associated proteins nsp12, 7 and 8 were cloned and purified by R.N.K. Data were analyzed by all authors. All authors wrote and reviewed the manuscript.</p></div><div class="section COI-statement" id="sec-7"><h2 class="">Competing Interests</h2><p id="p-33">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">This research is supported by Columbia University, which has filed a patent application on the work described in this manuscript.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span>, <span class="cit-auth cit-collab">for the China Novel Coronavirus Investigating and Research Team</span>. <span class="cit-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Eng J Med</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEng%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.2" data-doi="doi.org/10.1101/2020.01.30.927574"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jockusch</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Russo</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Nucleotide analogues as inhibitors of viral polymerases</span>. <abbr class="cit-jnl-abbrev">bioRXiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">doi.org/10.1101/2020.01.30.927574</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRXiv%26rft_id%253Dinfo%253Adoi%252Fdoi.org%252F10.1101%252F2020.01.30.927574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi.org/10.1101/2020.01.30.927574&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.3" data-doi="10.1038/nrd.2015.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zumla</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azhar</span>,  <span class="cit-name-given-names">E. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">D. S. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.-Y.</span></span> <span class="cit-article-title">Coronaviruses – drug discovery and therapeutic options</span>. <abbr class="cit-jnl-abbrev">Nat Rev | Drug Discovery</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">347</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2B%257C%2BDrug%2BDiscovery%26rft.volume%253D15%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd.2015.37%26rft_id%253Dinfo%253Apmid%252F26868298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrd.2015.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26868298&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dustin</span>,  <span class="cit-name-given-names">L. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartolini</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Capobianchi</span>,  <span class="cit-name-given-names">M. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pistello</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Hepatitis C virus: life cycle in cells, infection and host response, and analysis of molecular markers influencing the outcome of infection and response to therapy</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">826</span>–<span class="cit-lpage">832</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BMicrobiol%2BInfect%26rft.volume%253D22%26rft.spage%253D826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.5" data-doi="10.1007/s00018-014-1695-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">te Velthuis</span>,  <span class="cit-name-given-names">A. J. W.</span></span> <span class="cit-article-title">Common and unique features of viral RNA-dependent polymerases</span>. <abbr class="cit-jnl-abbrev">Cell Mol Life Sci</abbr> <span class="cit-vol">71</span>, <span class="cit-fpage">4403</span>–<span class="cit-lpage">4420</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMol%2BLife%2BSci%26rft.volume%253D71%26rft.spage%253D4403%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-014-1695-z%26rft_id%253Dinfo%253Apmid%252F25080879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-014-1695-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25080879&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.6" data-doi="10.3390/v10020059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Selisko</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papageorgiou</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Canard</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Structural and functional basis of the fidelity of nucleotide selection by Flavivirus RNA-dependent RNA polymerases</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">59</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10020059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v10020059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">C. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Inhibitors of viral nucleic acid polymerases. Pyrophosphate analogues</span>. <abbr class="cit-jnl-abbrev">ACS Symposium Series</abbr> <span class="cit-vol">401</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>. Chapter 1 (<span class="cit-pub-date">1989</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSymposium%2BSeries%26rft.volume%253D401%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.12.989186v1.8" data-doi="10.1016/j.antiviral.2006.05.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Öberg</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Rational design of polymerase inhibitors as antiviral drugs</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">71</span>, <span class="cit-fpage">90</span>–<span class="cit-lpage">95</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DOberg%26rft.auinit1%253DB.%26rft.volume%253D71%26rft.issue%253D2-3%26rft.spage%253D90%26rft.epage%253D95%26rft.atitle%253DRational%2Bdesign%2Bof%2Bpolymerase%2Binhibitors%2Bas%2Bantiviral%2Bdrugs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2006.05.012%26rft_id%253Dinfo%253Apmid%252F16820225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2006.05.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16820225&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240381200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.12.989186v1.9" data-doi="10.3390/v7102868"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eltahla</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luciani</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">A. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bull</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-article-title">Inhibitors of the hepatitis C virus polymerase; mode of action and resistance</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">5206</span>–<span class="cit-lpage">5224</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D5206%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7102868%26rft_id%253Dinfo%253Apmid%252F26426038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v7102868&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426038&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.10" data-doi="10.1128/CMR.00102-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Clercq</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Approved antiviral drugs over the past 50 years</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">695</span>–<span class="cit-lpage">747</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DDe%2BClercq%26rft.auinit1%253DE.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D695%26rft.epage%253D747%26rft.atitle%253DApproved%2BAntiviral%2BDrugs%2Bover%2Bthe%2BPast%2B50%2BYears%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00102-15%26rft_id%253Dinfo%253Apmid%252F27281742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvNjk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTQvMjAyMC4wMy4xMi45ODkxODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kayali</span>,  <span class="cit-name-given-names">Z.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">W. N.</span></span> <span class="cit-article-title">Finally sofosbuvir: an oral anti-HCV drug with wide performance capability</span>. <abbr class="cit-jnl-abbrev">Pharmgenomics Pers Med</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">398</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmgenomics%2BPers%2BMed%26rft.volume%253D7%26rft.spage%253D387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.12" data-doi="10.1021/jm100863x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sofia</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a β-D-2′-Deoxy-2′-α-fluoro-2′-β-C-methyluridine nucleotide prodrug (PSI-7977) for the treatment of hepatitis C virus</span>. <abbr class="cit-jnl-abbrev">J Med Chem</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">7202</span>–<span class="cit-lpage">7218</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DSofia%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D53%26rft.issue%253D19%26rft.spage%253D7202%26rft.epage%253D7218%26rft.atitle%253DDiscovery%2Bof%2Ba%2B%25CE%25B2-d-2%2527-deoxy-2%2527-%25CE%25B1-fluoro-2%2527-%25CE%25B2-C-methyluridine%2Bnucleotide%2Bprodrug%2B%2528PSI-7977%2529%2Bfor%2Bthe%2Btreatment%2Bof%2Bhepatitis%2BC%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm100863x%26rft_id%253Dinfo%253Apmid%252F20845908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jm100863x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20845908&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gitto</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gamal</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Andreone</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">NS5A inhibitors for the treatment of hepatitis C infection</span>. <abbr class="cit-jnl-abbrev">J Viral Hepatitis</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">180</span>–<span class="cit-lpage">186</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BViral%2BHepatitis%26rft.volume%253D24%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.14" data-doi="10.2174/1874091X00903010039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quezada</span>,  <span class="cit-name-given-names">E. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kane</span>,  <span class="cit-name-given-names">C. M.</span></span> <span class="cit-article-title">The hepatitis C virus NS5A stimulates NS5B during in vitro RNA synthesis in a template specific manner</span>. <abbr class="cit-jnl-abbrev">Open Biochem J</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">39</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DOpen%2BBiochem%2BJ%26rft.aulast%253DQuezada%26rft.auinit1%253DE.%2BM.%26rft.volume%253D3%26rft.spage%253D39%26rft.epage%253D48%26rft.atitle%253DThe%2BHepatitis%2BC%2BVirus%2BNS5A%2BStimulates%2BNS5B%2BDuring%2BIn%2BVitro%2BRNA%2BSynthesis%2Bin%2Ba%2BTemplate%2BSpecific%2BManner.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F1874091X00903010039%26rft_id%253Dinfo%253Apmid%252F19590581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/1874091X00903010039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19590581&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holshue</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">for the Washington State 2019-nCoV Case Investigation Team. First case of 2019 novel coronavirus in the United States</span>. <abbr class="cit-jnl-abbrev">N Eng J Med</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">929</span>–<span class="cit-lpage">936</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEng%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.16" data-doi="org/10.1038/s41422-020-0282-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>. <abbr class="cit-jnl-abbrev">Cell Research</abbr>, doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1038/s41422-020-0282-0</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BResearch%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1038%252Fs41422-020-0282-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1038/s41422-020-0282-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alanazi</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mehellou</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">The ProTide prodrug technology: where next?</span> <abbr class="cit-jnl-abbrev">ACS Med Chem Lett</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">2</span>–<span class="cit-lpage">5</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BMed%2BChem%2BLett%26rft.volume%253D10%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.18" data-doi="10.1038/sj.bjp.0706446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Clercq</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Field</span>,  <span class="cit-name-given-names">H. J.</span></span> <span class="cit-article-title">Antiviral prodrugs – the development of successful prodrug strategies for antiviral chemotherapy</span>. <abbr class="cit-jnl-abbrev">Br J Pharmacol</abbr> <span class="cit-vol">147</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBritish%2Bjournal%2Bof%2Bpharmacology%26rft.stitle%253DBr%2BJ%2BPharmacol%26rft.aulast%253DYoung%26rft.auinit1%253DA.%2BA.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DAntiviral%2Bprodrugs%2B-%2Bthe%2Bdevelopment%2Bof%2Bsuccessful%2Bprodrug%2Bstrategies%2Bfor%2Bantiviral%2Bchemotherapy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.bjp.0706446%26rft_id%253Dinfo%253Apmid%252F16284630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.bjp.0706446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16284630&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235151900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.19" data-doi="10.1002/hep.22321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">S. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Robust antiviral activity of R1626, a novel nucleoside analog: A randomized, placebo-controlled study in patients with chronic hepatitis C</span>. <abbr class="cit-jnl-abbrev">Hepatol</abbr> <span class="cit-vol">48</span>, <span class="cit-fpage">398</span>–<span class="cit-lpage">406</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHepatology%2B%2528Baltimore%252C%2BMd.%2529%26rft.stitle%253DHepatology%26rft.aulast%253DRoberts%26rft.auinit1%253DS.%2BK.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D398%26rft.epage%253D406%26rft.atitle%253DRobust%2Bantiviral%2Bactivity%2Bof%2BR1626%252C%2Ba%2Bnovel%2Bnucleoside%2Banalog%253A%2Ba%2Brandomized%252C%2Bplacebo-controlled%2Bstudy%2Bin%2Bpatients%2Bwith%2Bchronic%2Bhepatitis%2BC.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhep.22321%26rft_id%253Dinfo%253Apmid%252F18553458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/hep.22321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18553458&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258247400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.20" data-doi="10.1074/jbc.AC120.013056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tchesnokov</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">J. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">D. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Götte</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> (<span class="cit-pub-date">2020</span>). doi/<span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.AC120.013056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.AC120.013056%26rft_id%253Dinfo%253Apmid%252F32094225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjE0OiJBQzEyMC4wMTMwNTZ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE0LzIwMjAuMDMuMTIuOTg5MTg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tchesnokov</span>,  <span class="cit-name-given-names">E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">J. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">D. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Götte</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Mechanism of inhibition of Ebola virus RNA-dependent RNA polymerase by Remdesivir</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">326</span> (<span class="cit-issue">1-16</span>) (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.22" data-doi="10.1128/AAC.02666-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fung</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Efficiency of incorporation and chain termination determines the inhibition potency of 2’-modified nucleotide analogs against hepatitis C virus polymerase</span>. <abbr class="cit-jnl-abbrev">Antimicrob Agents Chemother</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">3636</span>–<span class="cit-lpage">3645</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob%2BAgents%2BChemother%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.02666-14%26rft_id%253Dinfo%253Apmid%252F24733478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU4LzcvMzYzNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE0LzIwMjAuMDMuMTIuOTg5MTg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.23" data-doi="10.1371/journal.ppat.1003030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Sensitivity of mitochondrial transcription and resistance of RNA polymerase II dependent nuclear transcription to antiviral ribonucleosides</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e1003030</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DArnold%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003030%26rft.epage%253De1003030%26rft.atitle%253DSensitivity%2Bof%2BMitochondrial%2BTranscription%2Band%2BResistance%2Bof%2BRNA%2BPolymerase%2BII%2BDependent%2BNuclear%2BTranscription%2Bto%2BAntiviral%2BRibonucleosides.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003030%26rft_id%253Dinfo%253Apmid%252F23166498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23166498&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.24" data-doi="10.1128/AAC.00433-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dutartre</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bussetta</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boretto</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Canard</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">General catalytic deficiency of hepatitis C virus RNA polymerase with an S282T mutation and mutually exclusive resistance towards 2’-modified nucleotide analogues</span>. <abbr class="cit-jnl-abbrev">Antimicrob Agents Chemother</abbr> <span class="cit-vol">50</span>, <span class="cit-fpage">4161</span>–<span class="cit-lpage">4169</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob%2BAgents%2BChemother%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.00433-06%26rft_id%253Dinfo%253Apmid%252F17000745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiI1MC8xMi80MTYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTQvMjAyMC4wMy4xMi45ODkxODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elfiky</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahdy</span>,  <span class="cit-name-given-names">S. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Elshemey</span>,  <span class="cit-name-given-names">W. M.</span></span> <span class="cit-article-title">Quantitative structure-activity relationship and molecular docking revealed a potency of anti-hepatitis C virus drugs against human corona viruses</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">1040</span>–<span class="cit-lpage">1047</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft.volume%253D89%26rft.spage%253D1040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elfiky</span>,  <span class="cit-name-given-names">A. A.</span></span> <span class="cit-article-title">Anti-HCV, nucleotide inhibitors, repurposing against COVID-19</span>. <abbr class="cit-jnl-abbrev">Life Sci</abbr> <span class="cit-vol">248</span>, <span class="cit-fpage">117477</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLife%2BSci%26rft.volume%253D248%26rft.spage%253D117477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.27" data-doi="10.1016/j.coviro.2014.08.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deval</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Symons</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Beigelman</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Inhibition of viral RNA polymerases by nucleoside and nucleotide analogs: therapeutic applications against positive-strand RNA viruses beyond hepatitis C virus</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2014.08.004%26rft_id%253Dinfo%253Apmid%252F25194816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2014.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25194816&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fearns</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Deval</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">New antiviral approaches for respiratory syncytial virus and other mononegaviruses: Inhibiting the RNA polymerase</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">134</span>, <span class="cit-fpage">63</span>–<span class="cit-lpage">76</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D134%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.29" data-doi="10.1073/pnas.0609513103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Four-color DNA sequencing by synthesis using cleavable fluorescent nucleotide reversible terminators</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">19635</span>–<span class="cit-lpage">19640</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0609513103%26rft_id%253Dinfo%253Apmid%252F17170132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzUyLzE5NjM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTQvMjAyMC4wMy4xMi45ODkxODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.30" data-doi="10.1073/pnas.0804023105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Four-color DNA sequencing with 3’-O-modified nucleotide reversible terminators and chemically cleavable fluorescent dideoxynucleotides</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">105</span>, <span class="cit-fpage">9145</span>–<span class="cit-lpage">9150</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804023105%26rft_id%253Dinfo%253Apmid%252F18591653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkxNDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNC8yMDIwLjAzLjEyLjk4OTE4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.31" data-doi="10.1073/pnas.0501962102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruparel</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Design and synthesis of a 3’-O-allyl photocleavable fluorescent nucleotide as a reversible terminator for DNA sequencing by synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">5932</span>–<span class="cit-lpage">5937</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501962102%26rft_id%253Dinfo%253Apmid%252F15829589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE3LzU5MzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNC8yMDIwLjAzLjEyLjk4OTE4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.32" data-doi="10.1073/pnas.92.14.6339"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tabor</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Richardson</span>,  <span class="cit-name-given-names">C. C.</span></span> <span class="cit-article-title">A single residue in DNA polymerases of Escherichia coli DNA polymerase I family is critical for distinguishing between deoxy- and dideoxynucleotides</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">6339</span>–<span class="cit-lpage">6343</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTabor%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.issue%253D14%26rft.spage%253D6339%26rft.epage%253D6343%26rft.atitle%253DA%2Bsingle%2Bresidue%2Bin%2BDNA%2Bpolymerases%2Bof%2Bthe%2BEscherichia%2Bcoli%2BDNA%2Bpolymerase%2BI%2Bfamily%2Bis%2Bcritical%2Bfor%2Bdistinguishing%2Bbetween%2Bdeoxy-%2Band%2Bdideoxyribonucleotides.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.14.6339%26rft_id%253Dinfo%253Apmid%252F7603992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTIvMTQvNjMzOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE0LzIwMjAuMDMuMTIuOTg5MTg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.33" data-doi="10.1073/pnas.87.9.3534"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Camerman</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mastropaolo</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Camerman</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Structure of the anti-human immunodeficiency virus agent 3’-fluoro-3’deoxythymidine and electronic charge calculations for 3’-deoxythymidines</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">3534</span>–<span class="cit-lpage">3537</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCamerman%26rft.auinit1%253DN.%26rft.volume%253D87%26rft.issue%253D9%26rft.spage%253D3534%26rft.epage%253D3537%26rft.atitle%253DStructure%2Bof%2Bthe%2Banti-human%2Bimmunodeficiency%2Bvirus%2Bagent%2B3%2527-fluoro-3%2527-deoxythymidine%2Band%2Belectronic%2Bcharge%2Bcalculations%2Bfor%2B3%2527-deoxythymidines.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.9.3534%26rft_id%253Dinfo%253Apmid%252F2333298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4Ny85LzM1MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNC8yMDIwLjAzLjEyLjk4OTE4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.34" data-doi="10.1089/088922201750102445"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">E.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vrang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Öberg</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Merigan</span>,  <span class="cit-name-given-names">T. C.</span></span> <span class="cit-article-title">Anti-HIV type 1 activity of 3□-fluoro-3□-deoxythymidine for several different multidrug-resistant mutants</span>. <abbr class="cit-jnl-abbrev">AIDS Res Human Retroviruses</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">401</span>–<span class="cit-lpage">407</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Bresearch%2Band%2Bhuman%2Bretroviruses%26rft.stitle%253DAIDS%2BRes%2BHum%2BRetroviruses%26rft.aulast%253DKim%26rft.auinit1%253DE.%2BY.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D401%26rft.epage%253D407%26rft.atitle%253DAnti-HIV%2Btype%2B1%2Bactivity%2Bof%2B3%2527-fluoro-3%2527-deoxythymidine%2Bfor%2Bseveral%2Bdifferent%2Bmultidrug-resistant%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F088922201750102445%26rft_id%253Dinfo%253Apmid%252F11282008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/088922201750102445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11282008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghosn</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antiviral activity of low-dose alovudine in antiretroviral-experienced patients: results from a 4-week randomized, double-blind, placebo-controlled dose-ranging trial</span>. <abbr class="cit-jnl-abbrev">HIV Med</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">142</span>–<span class="cit-lpage">147</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHIV%2Bmedicine%26rft.stitle%253DHIV%2BMed%26rft.aulast%253DGhosn%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D142%26rft.epage%253D147%26rft.atitle%253DAntiviral%2Bactivity%2Bof%2Blow-dose%2Balovudine%2Bin%2Bantiretroviral-experienced%2Bpatients%253A%2Bresults%2Bfrom%2Ba%2B4-week%2Brandomized%252C%2Bdouble-blind%252C%2Bplacebo-controlled%2Bdose-ranging%2Btrial.%26rft_id%253Dinfo%253Apmid%252F17461857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17461857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.36" data-doi="10.1073/pnas.82.20.7096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitsuya</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">3’-Azido-3’-deoxythymidine (BW A509U): an antiviral agent that inhibits the infectivity and cytopathic effect of human T-lymphotropic virus type III/lymphadenopathy-associated virus in vitro</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">7096</span>–<span class="cit-lpage">7100</span> (<span class="cit-pub-date">1985</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMitsuya%26rft.auinit1%253DH.%26rft.volume%253D82%26rft.issue%253D20%26rft.spage%253D7096%26rft.epage%253D7100%26rft.atitle%253D3%2527-Azido-3%2527-deoxythymidine%2B%2528BW%2BA509U%2529%253A%2Ban%2Bantiviral%2Bagent%2Bthat%2Binhibits%2Bthe%2Binfectivity%2Band%2Bcytopathic%2Beffect%2Bof%2Bhuman%2BT-lymphotropic%2Bvirus%2Btype%2BIII%252Flymphadenopathy-associated%2Bvirus%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.20.7096%26rft_id%253Dinfo%253Apmid%252F2413459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODIvMjAvNzA5NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE0LzIwMjAuMDMuMTIuOTg5MTg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.12.989186v1.37" data-doi="10.1016/S0140-6736(86)92808-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yarchoan</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Administration of 3’-azido-3’-deoxythymidine, an inhibitor of HTLV-III/LAV replication, to patients with AIDS or AIDS-related complex</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">327</span>, <span class="cit-fpage">575</span>–<span class="cit-lpage">580</span> (<span class="cit-pub-date">1986</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D327%26rft.spage%253D575%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252886%252992808-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(86)92808-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.38" data-doi="10.1126/science.1699273"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitsuya</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarchoan</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Broder</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Molecular targets for AIDS therapy</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">249</span>, <span class="cit-fpage">1533</span>–<span class="cit-lpage">1544</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMitsuya%26rft.auinit1%253DH%26rft.volume%253D249%26rft.issue%253D4976%26rft.spage%253D1533%26rft.epage%253D1544%26rft.atitle%253DMolecular%2Btargets%2Bfor%2BAIDS%2Btherapy%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1699273%26rft_id%253Dinfo%253Apmid%252F1699273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNDkvNDk3Ni8xNTMzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTQvMjAyMC4wMy4xMi45ODkxODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.39" data-doi="10.1073/pnas.83.21.8333"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furman</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phosphorylation of 3’-azido-3’-deoxythymidine and selective interaction of the 5’-triphosphate with human immunodeficiency virus reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">8333</span>–<span class="cit-lpage">8337</span> (<span class="cit-pub-date">1986</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFurman%26rft.auinit1%253DP.%2BA.%26rft.volume%253D83%26rft.issue%253D21%26rft.spage%253D8333%26rft.epage%253D8337%26rft.atitle%253DPhosphorylation%2Bof%2B3%2527-azido-3%2527-deoxythymidine%2Band%2Bselective%2Binteraction%2Bof%2Bthe%2B5%2527-triphosphate%2Bwith%2Bhuman%2Bimmunodeficiency%2Bvirus%2Breverse%2Btranscriptase.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.83.21.8333%26rft_id%253Dinfo%253Apmid%252F2430286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODMvMjEvODMzMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE0LzIwMjAuMDMuMTIuOTg5MTg2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.40" data-doi="10.2174/157488408783329913"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">D’Andrea</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brisdelli</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bozzi</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">AZT: An old drug with new perspectives</span>. <abbr class="cit-jnl-abbrev">Curr Clin Pharmacol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">20</span>–<span class="cit-lpage">37</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BClin%2BPharmacol%26rft.aulast%253DD%2527Andrea%26rft.auinit1%253DG.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D37%26rft.atitle%253DAZT%253A%2Ban%2Bold%2Bdrug%2Bwith%2Bnew%2Bperspectives.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F157488408783329913%26rft_id%253Dinfo%253Apmid%252F18690875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/157488408783329913&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18690875&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.41" data-doi="10.1073/pnas.1323705111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subissi</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">E3900</span>–<span class="cit-lpage">E3909</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323705111%26rft_id%253Dinfo%253Apmid%252F25197083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM3L0UzOTAwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTQvMjAyMC4wMy4xMi45ODkxODYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A. B.</span></span> <span class="cit-article-title">Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</span>. <abbr class="cit-jnl-abbrev">Nature Commun</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">2342</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommun%26rft.volume%253D10%26rft.spage%253D2342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.43" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of an RNA virus 3’→5’ exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">5108</span>–<span class="cit-lpage">5113</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNC8yMDIwLjAzLjEyLjk4OTE4Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agostini</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Coronavirus susceptibility to the antiviral Remdesivir (GS-5734) is mediated by the viral polymerase and the proofreading exonuclease</span>. <abbr class="cit-jnl-abbrev">Mbio</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e00221</span>–<span class="cit-lpage">e00218</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMbio%26rft.volume%253D9%26rft.spage%253De00221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Itagaki</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Massive Parallel Method for Decoding DNA and RNA</span>. <abbr class="cit-jnl-abbrev">United States Patent</abbr> <span class="cit-vol">6</span>,<span class="cit-fpage">664,079</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUnited%2BStates%2BPatent%26rft.volume%253D6%26rft.spage%253D664079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title"><em>In vitro</em> selection of resistance to sofosbuvir in HCV replicons of genotype-1 to −6</span>. <abbr class="cit-jnl-abbrev">Antivir Ther</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">587</span>–<span class="cit-lpage">597</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntivir%2BTher%26rft.volume%253D22%26rft.spage%253D587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Itagaki</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Massive parallel method for decoding DNA and RNA</span>, <abbr class="cit-jnl-abbrev">United States Patent</abbr> <span class="cit-vol">9</span>,<span class="cit-fpage">868,985</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUnited%2BStates%2BPatent%26rft.volume%253D9%26rft.spage%253D868985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.48" data-doi="10.1016/0378-1119(94)90226-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Canard</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sarfati</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-article-title">DNA polymerase fluorescent substrates with reversible 3′-tags</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">148</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DCanard%26rft.auinit1%253DB.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D6%26rft.atitle%253DDNA%2Bpolymerase%2Bfluorescent%2Bsubstrates%2Bwith%2Breversible%2B3%2527-tags.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252894%252990226-7%26rft_id%253Dinfo%253Apmid%252F7523248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(94)90226-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7523248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PM49200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.12.989186v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruparel</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-chapter-title">Novel DNA sequencing and genotyping approaches using modified nucleotide analogues</span>. <span class="cit-source">Doctoral Dissertation</span>, <span class="cit-publ-loc">Columbia University</span>. <span class="cit-fpage">72</span>–<span class="cit-lpage">112</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.50" data-doi="10.1093/nar/5.10.3549"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Axelrod</span>,  <span class="cit-name-given-names">V. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vartikyan</span>,  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aivazashvili</span>,  <span class="cit-name-given-names">V. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Beabealashvili</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-article-title">Specific termination of RNA polymerase synthesis as a method of RNA and DNA sequencing</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">3549</span>–<span class="cit-lpage">3563</span> (<span class="cit-pub-date">1978</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DAxelrod%26rft.auinit1%253DV.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D3549%26rft.epage%253D3564%26rft.atitle%253DSpecific%2Btermination%2Bof%2BRNA%2Bpolymerase%2Bsynthesis%2Bas%2Ba%2Bmethod%2Bof%2BRNA%2Band%2BDNA%2Bsequencing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F5.10.3549%26rft_id%253Dinfo%253Apmid%252F724495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/5.10.3549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=724495&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FU93300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">Extended Data Reference</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.989186v1.51" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">E. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F14%2F2020.03.12.989186.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.12.989186v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.13.991570v1" title="A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 14, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.12.989186v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.12.989186v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.12.989186v1" class="active" data-icon-position="" data-hide-link-title="0">Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1196270" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6IxhPKb1LkSWPy24aGsdjGYTB-RTQgpmcVtSWNqpNgc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196270" id="node1196270--2" data-pisa="biorxiv;2020.03.12.989186v1" data-pisa-master="biorxiv;2020.03.12.989186" data-seqnum="1196270" data-apath="/biorxiv/early/2020/03/14/2020.03.12.989186.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingyue</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shiv</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Steffen</span> <span class="nlm-surname">Jockusch</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Minchen</span> <span class="nlm-surname">Chien</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chuanjuan</span> <span class="nlm-surname">Tao</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Irina</span> <span class="nlm-surname">Morozova</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Kalachikov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">James J.</span> <span class="nlm-surname">Russo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.12.989186; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.12.989186 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;title=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;title=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;text=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;title=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;t=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;title=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;title=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196270" id="node1196270--3" data-pisa="biorxiv;2020.03.12.989186v1" data-pisa-master="biorxiv;2020.03.12.989186" data-seqnum="1196270" data-apath="/biorxiv/early/2020/03/14/2020.03.12.989186.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Nucleotide Analogues as Inhibitors of SARS-CoV Polymerase</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jingyue</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaoxu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shiv</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Steffen</span> <span class="nlm-surname">Jockusch</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Minchen</span> <span class="nlm-surname">Chien</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chuanjuan</span> <span class="nlm-surname">Tao</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Irina</span> <span class="nlm-surname">Morozova</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Kalachikov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">James J.</span> <span class="nlm-surname">Russo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.12.989186; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.12.989186 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ju%2C+Jingyue&amp;rft.au=Li%2C+Xiaoxu&amp;rft.au=Kumar%2C+Shiv&amp;rft.au=Jockusch%2C+Steffen&amp;rft.au=Chien%2C+Minchen&amp;rft.au=Tao%2C+Chuanjuan&amp;rft.au=Morozova%2C+Irina&amp;rft.au=Kalachikov%2C+Sergey&amp;rft.au=Kirchdoerfer%2C+Robert+N.&amp;rft.au=Russo%2C+James+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1196270/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1196270/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1196270/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1196270/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1196270/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1196270/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1196270/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1196270/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1196270/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1196270/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1196270/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1196270/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;count=horizontal&amp;via=&amp;text=Nucleotide%20Analogues%20as%20Inhibitors%20of%20SARS-CoV%20Polymerase&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.989186v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.12.989186v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/pharmacology-and-toxicology" class="highlight" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
