<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/20/2020.02.03.932350.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.03.932350v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study" />
<meta name="DC.Identifier" content="10.1101/2020.02.03.932350" />
<meta name="DC.Date" content="2020-02-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus , within Betacoronavirus . Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio , using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification." />
<meta name="DC.Contributor" content="Gurjit S. Randhawa" />
<meta name="DC.Contributor" content="Maximillian P.M. Soltysiak" />
<meta name="DC.Contributor" content="Hadi El Roz" />
<meta name="DC.Contributor" content="Camila P.E. de Souza" />
<meta name="DC.Contributor" content="Kathleen A. Hill" />
<meta name="DC.Contributor" content="Lila Kari" />
<meta name="article:published_time" content="2020-02-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as &lt;i&gt;Sarbecovirus&lt;/i&gt;, within &lt;i&gt;Betacoronavirus&lt;/i&gt;. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, &lt;i&gt;ab initio&lt;/i&gt;, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.03.932350v3" />
<meta name="citation_id" content="2020.02.03.932350v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.03.932350" />
<meta name="citation_num_pages" content="26" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.03.932350" />
<meta name="citation_author" content="Gurjit S. Randhawa" />
<meta name="citation_author_institution" content="Department of Computer Science, The University of Western Ontario" />
<meta name="citation_author_email" content="grandha8@uwo.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1054-125X" />
<meta name="citation_author" content="Maximillian P.M. Soltysiak" />
<meta name="citation_author_institution" content="Department of Biology, The University of Western Ontario" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7495-5203" />
<meta name="citation_author" content="Hadi El Roz" />
<meta name="citation_author_institution" content="Department of Biology, The University of Western Ontario" />
<meta name="citation_author" content="Camila P.E. de Souza" />
<meta name="citation_author_institution" content="Department of Statistical and Actuarial Sciences, The University of Western Ontario" />
<meta name="citation_author" content="Kathleen A. Hill" />
<meta name="citation_author_institution" content="Department of Biology, The University of Western Ontario" />
<meta name="citation_author" content="Lila Kari" />
<meta name="citation_author_institution" content="School of Computer Science, University of Waterloo" />
<meta name="citation_reference" content="citation_title=Virus Taxonomy. Seventh Report of the International Committee on Taxonomy of Viruses;citation_pages=835-849;citation_year=2000" />
<meta name="citation_reference" content="Weiss SR, Navas-Martin S. Coronavirus Pathogenesis and the Emerging Pathogen Severe Acute Respiratory Syndrome Coronavirus. Microbiol. Mol. Biol. 2005; Rev. 69: 635–664." />
<meta name="citation_reference" content="Su S, Wong G, Shi W, Liu J, Lai ACK, Zhou J, et al. Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses. Trends in Microbiology. 2016; 24: 490–502." />
<meta name="citation_reference" content="Cui J, Li F, Shi ZL. Origin and evolution of pathogenic coronaviruses. Nature Reviews Microbiology. 2019; 17: 181–5192." />
<meta name="citation_reference" content="Schoeman D, Fielding BC. Coronavirus envelope protein: Current knowledge. Virology Journal. 2019; 16." />
<meta name="citation_reference" content="de Groot RJ, Baker SC, Baric R, Enjuanes L, Gorbalenya AE, Holmes KV, et al. Family Coronaviridae. In: King AMQ, Adams MJ, Carstens EB, Lefkowitz EJ, editors. Virus taxonomy. Ninth report of the international committee on taxonomy of viruses, Elsevier Academic Press; 2012. pp. 806–828." />
<meta name="citation_reference" content="Woo PCY, Lau SKP, Huang Y, Yuen KY. Coronavirus diversity, phylogeny and interspecies jumping. Experimental Biology and Medicine. 2009; 234: 1117–1127." />
<meta name="citation_reference" content="Wertheim JO, Chu DKW, Peiris JSM, Kosakovsky Pond SL, Poon LLM. A Case for the Ancient Origin of Coronaviruses. J. Virol. 2013; 87: 7039–7045." />
<meta name="citation_reference" content="Luk HKH, Li X, Fung J, Lau SKP, Woo PCY. Molecular epidemiology, evolution and phylogeny of SARS coronavirus. Infection, Genetics and Evolution. 2019; 71: 21–30." />
<meta name="citation_reference" content="Vijaykrishna D, Smith GJD, Zhang JX, Peiris JSM, Chen H, Guan Y. Evolutionary Insights into the Ecology of Coronaviruses. J. Virol. 2007; 81: 4012–4020." />
<meta name="citation_reference" content="Lau SK, Li KS, Tsang AK, Shek CT, Wang M, Choi GK, et al. Recent Transmission of a Novel Alphacoronavirus, Bat Coronavirus HKU10, from Leschenault’s Rousettes to Pomona Leaf-Nosed Bats: First Evidence of Interspecies Transmission of Coronavirus between Bats of Different Suborders. J. Virol. 2012; 86: 11906–11918." />
<meta name="citation_reference" content="Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet. 2020; doi:10.1016/S0140-6736(20)30251-8." />
<meta name="citation_reference" content="Li W, Shi Z, Yu M, Ren W, Smith C, Epstein JH, et al. Bats are natural reservoirs of SARS-like coronaviruses. Science. 2005; 310: 676–679." />
<meta name="citation_reference" content="Duffy S, Shackelton LA, Holmes EC. Rates of evolutionary change in viruses: Patterns and determinants. Nature Reviews Genetics. 2008; 9: 267–276." />
<meta name="citation_reference" content="Jenkins GM, Rambaut A, Pybus OG, Holmes EC. Rates of molecular evolution in RNA viruses: A quantitative phylogenetic analysis. J. Mol. Evol. 2002; 54: 156–165." />
<meta name="citation_reference" content="Nagy PD, Simon AE. New insights into the mechanisms of RNA recombination. Virology. 1997; 235: 1–9." />
<meta name="citation_reference" content="Rowe CL, Fleming JO, Nathan MJ, Sgro JY, Palmenberg AC, Baker SC. Generation of coronavirus spike deletion variants by high-frequency recombination at regions of predicted RNA secondary structure. J. Virol. 1997; 71: 6183–90." />
<meta name="citation_reference" content="Cavanagh D. Coronaviridae: a review of coronaviruses and toroviruses. In: Schmidt A, Wolff MH, Weber O, editors. Coronaviruses with Special Emphasis on First Insights Concerning SARS. Birkhäuser-Verlag, 2005; pp. 1–54." />
<meta name="citation_reference" content="Lai MMC. RNA recombination in animal and plant viruses. Microbiological Reviews. 1992; 56: 61–79." />
<meta name="citation_reference" content="Pasternak AO, Spaan WJM, Snijder EJ. Nidovirus transcription: How to make sense…? Journal of General Virology. 2006; 87: 1403–1421." />
<meta name="citation_reference" content="Drosten C, Günther S, Preiser W, van der Werf S, Brodt HR, Becker S, et al. Identification of a Novel Coronavirus in Patients with Severe Acute Respiratory Syndrome. N. Engl. J. Med. 2003; 348: 1967–1976." />
<meta name="citation_reference" content="Ksiazek TG, Erdman D, Goldsmith CS, Zaki SR, Peret T, Emery S, et al. A Novel Coronavirus Associated with Severe Acute Respiratory Syndrome. N. Engl. J. Med. 2003; 348: 1953–1966." />
<meta name="citation_reference" content="Zaki AM, van Boheemen S, Bestebroer TM, Osterhaus ADME, Fouchier RAM. Isolation of a Novel Coronavirus from a Man with Pneumonia in Saudi Arabia. N. Engl. J. Med. 2012; 367: 1814–1820." />
<meta name="citation_reference" content="Guan Y, Zheng BJ, He YQ, Liu XL, Zhuang ZX, Cheung CL, et al. Isolation and characterization of viruses related to the SARS coronavirus from animals in Southern China. Science. 2003; 302: 276–278." />
<meta name="citation_reference" content="Alagaili AN, Briese T, Mishra N, Kapoor V, Sameroff SC, de Wit E, et al. Middle east respiratory syndrome coronavirus infection in dromedary camels in Saudi Arabia. MBio. 2014; 5." />
<meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang Bo, Song J, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N. Engl. J. Med. 2020; doi:10.1056/NEJMoa2001017." />
<meta name="citation_reference" content="Lu H, Stratton CW, Tang Y. Outbreak of Pneumonia of Unknown Etiology in Wuhan China: the Mystery and the Miracle. J. Med. Virol. 2020; doi:10.1002/jmv.25678." />
<meta name="citation_reference" content="Hui DS, I Azhar E, Madani TA, Ntoumi F, Kock R, Dar O, et al. The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health — The latest 2019 novel coronavirus outbreak in Wuhan, China. International Journal of Infectious Diseases. 2020; 91: 264–266." />
<meta name="citation_reference" content="Liu T, Hu J, Kang M, Lin L, Zhong H, Xiao J, et al. Transmission dynamics of 2019 novel coronavirus (2019-nCoV). BioRxiv [Preprint]. 2020 bioRxiv 919787 [posted 2020 January 25; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.25.919787v1 doi: 10.1101/2020.01.25.919787." />
<meta name="citation_reference" content="Perlman S. Another Decade, Another Coronavirus. N. Engl. J. Med. 2020; doi: 10.1056/NEJMe2001126." />
<meta name="citation_reference" content="Gralinski LE, Menachery VD. Return of the Coronavirus: 2019-nCoV. Viruses. 2020; 12: 135." />
<meta name="citation_reference" content="2019-nCoV Global Cases by Johns Hopkins CSSE. 2020 February 6 [cited 6 February 2020]. In: JHU CSSE website [Internet]. Available from: https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6." />
<meta name="citation_reference" content="Novel Coronavirus(2019-nCoV) Situation Report - 13. 2002 February 02 [cited 02 February 2020]. In: WHO website [Internet]. Available from: https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200202-sitrep-13-ncov-v3.pdf." />
<meta name="citation_reference" content="Chan JFW, Yuan S, Kok KH, To KKW, Chu H, Yang J, et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet. 2020; doi:10.1016/S0140-6736(20)30154-9." />
<meta name="citation_reference" content="Hu B, Zeng LP, Yang XL, Ge XY, Zhang W, Li B, et al. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS Pathog. 2017; 13." />
<meta name="citation_reference" content="Dong N, Yang X, Ye L, Chen K, Chan EWC, Yang M, Chen S. Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China. BioRxiv [Preprint]. 2020 bioRxiv 913368 [posted 2020 January 22; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.20.913368v2 doi:10.1101/2020.01.20.913368." />
<meta name="citation_reference" content="Guo Q, Li M, Wang C, Wang P, Fang Z, Tan J, et al. Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm. BioRxiv [Preprint]. 2020 bioRxiv 914044 [posted 2020 January 22; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.21.914044v2 doi: 10.1101/2020.01.21.914044." />
<meta name="citation_reference" content="Wu F, Zhao S, Yu B, Chen YM, Wang W, Hu Y, et al. Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China. BioRxiv [Preprint]. 2020 bioRxiv 919183 [posted 2020 February 02; cited 2020 February 02]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.24.919183v2 doi:10.1101/2020.01.24.919183." />
<meta name="citation_reference" content="Paraskevis D, Kostaki EG, Magiorkinis G, Panayiotakopoulos G, Tsiodras S. Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event. BioRxiv [Preprint]. 2020 bioRxiv 920249 [posted 2020 January 27; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.26.920249v1 doi: 10.1101/2020.01.26.920249." />
<meta name="citation_reference" content="Ji W, Wang W, Zhao X, Zai J, Li X. Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost cross species transmission from snake to human. J. Med. Virol. 2020; doi:10.1002/jmv.25682." />
<meta name="citation_reference" content="Zhou P, Yang XL, Wang XG, Hu B, Zhang L, Zhang W, et al. Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin. BioRxiv [Preprint]. 2020 bioRxiv 914952 [posted 2020 January 23; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.22.914952v1 doi:10.1101/2020.01.22.914952." />
<meta name="citation_reference" content="Letko M, Munster V. Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV. BioRxiv [Preprint]. 2020 bioRxiv 915660 [posted 2020 January 22; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1 doi:10.1101/2020.01.22.915660." />
<meta name="citation_reference" content="Zhao Y, Zhao Z, Wang Y, Zhou Y, Ma Y, Zuo W. Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCoV. BioRxiv [Preprint]. 2020 bioRxiv 919985 [posted 2020 January 26; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.26.919985v1 doi:10.1101/2020.01.26.919985." />
<meta name="citation_reference" content="Li Y, Zhang J, Wang N, Li H, Shi Y, Gui G, et al. Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening. BioRxiv [Preprint]. 2020 bioRxiv 922922 [posted 2020 January 30; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.28.922922v2 doi:10.1101/2020.01.28.922922." />
<meta name="citation_reference" content="Liu X, Wang XJ. Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines. BioRxiv [Preprint]. 2020 bioRxiv 924100 [posted 2020 January 29; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.29.924100v1 doi:10.1101/2020.01.29.924100." />
<meta name="citation_reference" content="Vinga S, Almeida J. Alignment-free sequence comparison-a review. Bioinformatics. 2003; 19(4): 513–523." />
<meta name="citation_reference" content="Zielezinski A, Vinga S, Almeida J, Karlowski WM. Alignment-free sequence comparison: benefits, applications, and tools. Genome Biology. 2017, 18: 186." />
<meta name="citation_reference" content="Kari L, Hill KA, Sayem AS, Karamichalis R, Bryans N, Davis K, Dattani NS. Mapping the space of genomic signatures. PLoS ONE. 2015; 10: e0119815." />
<meta name="citation_reference" content="Karamichalis R, Kari L, Konstantinidis S, Kopecki S. An investigation into inter- and intragenomic variations of graphic genomic signatures. BMC Bioinformatics. 2015; 16: 246." />
<meta name="citation_reference" content="Solis-Reyes S, Avino M, Poon A. An open-source k-mer based machine learning tool for fast and accurate subtyping of HIV-1 genomes. PLoS ONE. 2018; 13: e0206409." />
<meta name="citation_reference" content="Randhawa GS, Hill KH, Kari L. ML-DSP: Machine Learning with Digital Signal Processing for ultrafast, accurate, and scalable genome classification at all taxonomic levels. BMC Genomics. 2019; 20: 267." />
<meta name="citation_reference" content="Randhawa GS, Hill KH, Kari L. MLDSP-GUI: an alignment-free standalone tool with an interactive graphical user interface for DNA sequence comparison and analysis. Bioinformatics. 2019; btz918." />
<meta name="citation_reference" content="Jeffre HJ. Chaos game representation of gene structure. Nucleic Acids Res. 1990; 18: 2163–2170." />
<meta name="citation_reference" content="Asuero AG, Sayago A, González AG. The correlation coefficient: an overview. Crit Rev Anal Chem. 2006; 36(1): 41–59." />
<meta name="citation_reference" content="Karamichalis R, Kari L. MoDMaps3D: an interactive webtool for the quantification and 3D visualization of interrelationships in a dataset of DNA sequences. Bioinformatics. 2017; 33(19): 3091–3." />
<meta name="citation_reference" content="Kruskal J. Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis. Psychometrika. 1964; 29: 1–27." />
<meta name="citation_reference" content="Carneiro RL, Requião RD, Rossetto S, Domitrovic T, Palhano FL. Codon stabilization coefficient as a metric to gain insights into mRNA stability and codon bias and their relationships with translation. Nucleic acids research. 2019; 47(5): 2216–2228." />
<meta name="citation_reference" content="Karumathil S, Raveendran NT, Ganesh D, Kumar NS, Nair RR, Dirisala VR. Evolution of Synonymous Codon Usage Bias in West African and Central African Strains of Monkeypox Virus. Evolutionary Bioinformatics Online. 2018; 14: doi:10.1177/1176934318761368." />
<meta name="citation_reference" content="Vinogradov AE, Anatskaya OV. DNA helix: the importance of being AT-rich. Mammalian Genome. 2017; 9(10): 455–464." />
<meta name="citation_reference" content="Hollander M, Wolfe DA, Chicken E. Nonparametric statistical methods, 3rd Edition, John Wiley &amp; Sons; 2013." />
<meta name="citation_reference" content="Zhao S, Lin Q, Ran J, Musa SS, Yang G, Wang W, et al. Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak. International Journal of Infectious Diseases [In Press] [received 2020 January 23; revised 2020 January 27; accepted 2020 January 27; cited 2020 February 1], 2020." />
<meta name="citation_reference" content="Shao P, Shan Y. Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model. BioRxiv [Preprint]. 2020 bioRxiv 923169 [posted 2020 January 28; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.28.923169v1 doi:10.1101/2020.01.28.923169." />
<meta name="citation_reference" content="Chen Z, Zhang W, Lu Y, Guo C, Guo Z, Liao C, et al. From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends. BioRxiv [Preprint]. 2020 bioRxiv 919241 [posted 2020 January 27; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.24.919241v3 doi:10.1101/2020.01.24.919241." />
<meta name="citation_reference" content="Hayward JA, Tachedjian M, Cui J, Field H, Holmes EC, Wang L, Tachedjian G. Identification of diverse full-length endogenous betaretroviruses in megabats and microbats. Retrovirology. 2013; 10." />
<meta name="citation_reference" content="Cui J, Tachedjian G, Wang LF. Bats and Rodents Shape Mammalian Retroviral Phylogeny. Sci. Rep. 2015; 5." />
<meta name="citation_reference" content="Hayward JA, Tachedjian M, Cui J, Cheng AZ, Johnson A, Baker ML, et al. Differential evolution of antiretroviral restriction factors in pteropid bats as revealed by APOBEC3 gene complexity. Mol. Biol. Evol. 2018; 35: 1626–1637." />
<meta name="citation_reference" content="Wong A, Li X, Lau S, Woo P. Global Epidemiology of Bat Coronaviruses. Viruses. 2019; 11(2): 174." />
<meta name="citation_reference" content="Yang XL, Hu B, Wang B, Wang MN, Zhang Q, Zhang W, et al. Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus. J. Virol. 2016; 90: 3253–3256." />
<meta name="citation_reference" content="Lau SK, Li KS, Tsang AK, Lam CS, Ahmed S, Chen H, et al. Genetic Characterization of Betacoronavirus Lineage C Viruses in Bats Reveals Marked Sequence Divergence in the Spike Protein of Pipistrellus Bat Coronavirus HKU5 in Japanese Pipistrelle: Implications for the Origin of the Novel Middle East Respiratory Syndrome Coronavirus. J. Virol. 2013; 87: 8638–8650." />
<meta name="citation_reference" content="Lacroix A, Duong V, Hul V, San S, Davun H, Omaliss K, et al. Genetic diversity of coronaviruses in bats in Lao PDR and Cambodia. Infect. Genet. Evol. 2017; 48: 10–18." />
<meta name="citation_reference" content="Drexler JF, Gloza-Rausch F, Glende J, Corman VM, Muth D, Goettsche M, et al. Genomic Characterization of Severe Acute Respiratory Syndrome-Related Coronavirus in European Bats and Classification of Coronaviruses Based on Partial RNA-Dependent RNA Polymerase Gene Sequences. J. Virol. 2010; 84: 11336–11349." />
<meta name="citation_reference" content="Rihtarič D, Hostnik P, Steyer A, Grom J, Toplak I. Identification of SARS-like coronaviruses in horseshoe bats (Rhinolophus hipposideros) in Slovenia. Arch. Virol. 2010; 155: 507–514." />
<meta name="citation_reference" content="He B, Zhang Y, Xu L, Yang W, Yang F, Yun Feng, et al. Identification of Diverse Alphacoronaviruses and Genomic Characterization of a Novel Severe Acute Respiratory Syndrome-Like Coronavirus from Bats in China. J. Virol. 2014; 88: 7070–7082." />
<meta name="citation_reference" content="Wacharapluesadee S, Duengkae P, Rodpan A, Kaewpom T, Maneeorn P, Kanchanasaka B, et al. Diversity of coronavirus in bats from Eastern Thailand Emerging viruses. Virol. J. 2015; 12: 1–7." />
<meta name="citation_reference" content="Tong S, Conrardy C, Ruone S, Kuzmin IV, Guo X, Tao Y, et al. Detection of novel SARS-like and other coronaviruses in bats from Kenya. Emerg. Infect. Dis. 2009; 15: 482–485." />
<meta name="citation_reference" content="Lau SKP, Woo PCY, Li KSM, Huang Y, Tsoi H, Wong BHL, et al. Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats. Proc. Natl. Acad. Sci. 2005; National Academy of Sciences, U. S. A., 102: 14040–14045." />
<meta name="citation_reference" content="Virologists weigh in on novel coronavirus in China’s outbreak. 2020 January 08 [cited 31 January 2020]. In: University of Minnesota [Internet]. Available from: http://www.cidrap.umn.edu/news-perspective/2020/01/virologists-weigh-novel-coronavirus-chinas-outbreak." />
<meta name="citation_reference" content="nCoV’s relationship to bat coronaviruses &amp; recombination signals (no snakes) - no evidence the 2019-nCoV lineage is recombinant. 2020 January 31 [cited 31 January 2020]. In: Virological blog [Internet]. Available from: http://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes-no-evidence-the-2019-nCoV-lineage-is-recombinant/331." />
<meta name="citation_reference" content="Experts: nCoV spread in China’s cities could trigger global epidemic. 2020 January 27 [cited 31 January 2020]. In: University of Minnesota [Internet]. Available from: http://www.cidrap.umn.edu/news-perspective/2020/01/experts-ncov-spread-chinas-cities-could-trigger-global-epidemic." />
<meta name="citation_reference" content="China detects large quantity of novel coronavirus at Wuhan seafood market. 2020 January 27 [cited 31 January 2020]. In: Xinhuanet News [Internet]. Available from: http://www.xinhuanet.com/english/2020-01/27/c_138735677.htm." />
<meta name="citation_reference" content="Takata MA, Gonçalves-Carneiro D, Zang TM, Soll SJ, York A, Blanco-Melo D, Bieniasz PD. CG dinucleotide suppression enables antiviral defence targeting non-self RNA. Nature. 2017; 550(7674): 124–127." />
<meta name="citation_reference" content="Greenbaum BD, Levine AJ, Bhanot G, Rabadan R. Patterns of evolution and host gene mimicry in influenza and other RNA viruses. PLoS Pathogens. 2008; 4(6): doi:10.1371/journal.ppat.1000079." />
<meta name="citation_reference" content="Lobo FP, Mota BEF, Pena SDJ, Azevedo V, Macedo AM, Tauch A, et al. Virus-host coevolution: Common patterns of nucleotide motif usage in Flaviviridae and their hosts. PLoS ONE. 2009; 4(7): 10.1371/journal.pone.0006282." />
<meta name="citation_reference" content="Kindler E, Thiel V. To sense or not to sense viral RNA-essentials of coronavirus innate immune evasion. Current Opinion in Microbiology. 2014; 20: 68–75." />
<meta name="citation_reference" content="Milewska A, Kindler E, Vkovski P, Zeglen S, Ochman M, Thiel V, et al. APOBEC3-mediated restriction of RNA virus replication. Scientific Reports. 2018; 8(1): doi:10.1038/s41598-018-24448-2." />
<meta name="citation_reference" content="Bishop KN, Holmes RK, Sheehy AM, Malim MH. APOBEC-mediated editing of viral RNA. Science. 2004; 305(5684): 645." />
<meta name="citation_reference" content="Pyrc K, Jebbink MF, Berkhout B, Van der Hoek L. Genome structure and transcriptional regulation of human coronavirus NL63. Virology Journal. 2004; 1(1): 7." />
<meta name="citation_reference" content="Berkhout B, Van Hemert F. On the biased nucleotide composition of the human coronavirus RNA genome. Virus Research. 2015; 202: 41–47." />
<meta name="citation_reference" content="Woo PCY, Lau SKP, Huang Y, Yuen KY. Coronavirus diversity, phylogeny and interspecies jumping. Experimental Biology and Medicine. 2009; 234(10): 1117–1127." />
<meta name="citation_reference" content="Woo PCY, Huang Y, Lau SKP, Yuen KY. Coronavirus Genomics and Bioinformatics Analysis. Viruses. 2010; 2(8): 1804–1820." />
<meta name="citation_reference" content="Xue X, Yu H, Yang H, Xue F, Wu Z, Shen W, et al. Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design. J. Virol. 2008; 82: 2515–2527." />
<meta name="citation_reference" content="Anand K, Ziebuhr J, Wadhwani P, Mesters JR, Hilgenfeld R. Coronavirus main proteinase (3CLpro) Structure: Basis for design of anti-SARS drugs. Science. 2003; 300: 1763–1767." />
<meta name="citation_reference" content="Nukoolkarn V, Lee VS, Malaisree M, Aruksakulwong O, Hannongbua S. Molecular dynamic simulations analysis of ritronavir and lopinavir as SARS-CoV 3CLpro inhibitors. J. Theor. Biol. 2008; 254: 861–867." />
<meta name="citation_reference" content="Xu Z, Peng C, Shi Y, Zhu Z, Mu K, Wang X, Zhu W. Nelfinavir was predicted to be a potential inhibitor of 2019-nCoV main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation. BioRxiv [Preprint]. 2020 bioRxiv 921627 [posted 2020 January 28; cited 2020 January 31]. Available from: https://www.biorxiv.org/content/10.1101/2020.01.27.921627v1 doi: 10.1101/2020.01.27.921627." />
<meta name="twitter:title" content="Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus , within Betacoronavirus . Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio , using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification." />
<meta name="og-title" property="og:title" content="Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as Sarbecovirus , within Betacoronavirus . Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, ab initio , using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.03.932350v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.03.932350v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.03.932350v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1160969" />
    <title>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RLZuK2V9sphkgINYRPXs3BcPQX53oDKzc-NrQNaxULE","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1160969","apath":"\/biorxiv\/early\/2020\/02\/20\/2020.02.03.932350.atom","pisa":"biorxiv;2020.02.03.932350v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.03.932350v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.03.932350v3","title":"Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study","identifier":"node\/1160969"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qhoui+dKiMo6g+0DeXavPGe5uNJCENvxo8cojatybMz6D8\\\\\\\/Daw+iBwa35s1rTQKE5B4wPlYbVLXw4R\\\\\\\/YmF8VSZXy1RPB+NzZyfZ\\\\\\\/HWxnpJVtC6goYvnU4WD6yc2voQUAq9JA+XNYxqhzkLKnna26iVi+yVUA4WUjPVrGcHXH20Z+DouJQnsK6dU1hulTU5eJgUIHCtmOq1ZDYPomjLQgA0FVKHNNFNPxSpbnslV+V+F5214mtpFihNY+HxDngwKA2ZoXzrtJ6yTEmkTQgMksC2ZkFMWqvWHp0j3GQC9CUg4+HFFdpol0fd1Lq34XKjDSgE2j0k+4aQSvRbOdyJd9MaYOrGwLPt299P+1cATYQpLW8ZB5GInz7XmTGA0z\\\\\\\/qptcELXN+g5FsQwAtXP6xwA8m96fS9imAA2rMG5Dtm8m8U8MEt+zPGcOxV4kogfTPXA1I6kE+oRUptV5IeAlW9s79Jo7q3OFCj1ZhddVxZFxADgVistOhOO26ZQIL26kDaxCtTYiEbJ4xpz8xaEHCfzpfebLVtuDfZAl5TCK5yeKgLF49dxUyO6Ku\\\\\\\/kTTT4X+nuFnPiRkmuP4F\\\\\\\/PGVSM6DAuwVBJ1qkvFrpDMo+xiLg21bMwKdB\\\\\\\/RLtiJBrXuMWK+pLIO1is2BEvwSC3eKWx9mvtEW3XTib\\\\\\\/mGzs2fu6M6v9VU+2qn1AOrgg5XkjaqXUsg+iRMh2M1Jdz7W3e3qN80N0ImA94folED9PKARRs0QuXoCH93NrOmg3EMgcQjP3n\\\\\\\/oPx2U74svgv625HOMyvV8EbyRVjwzdhvKqrTVLMyUp6oCCW5EGxxF5c6lx2B8i98AynHxV3AIrbX4BmMRPkFDx6Yp9cuibxRggDyMVIMcjSOs4xxXXrxv2ceHJt7IxAMQOeZK62Jr4IlcFp6Lvb6DxAq4\\\\\\\/SHqgTIUVUzxctb8lW56Ptavy8cpUvaVgbc+\\\\\\\/xuzbVybvJnyrcvFCIou2YaG2MDlaIX2vXzxvPuduzs67RcJCXVowneNa\\\\\\\/qDp8WmRZKjXJrcnQx07fpEnhmxwFToMRp9iPXFB5ZZxSML3t3rcVuHYhR9WQIB9MeR+hqLnRNaZ9TCTJ5g+lbNuPvb0oNR\\\\\\\/mEVcufqp2w+PTSfCIod556IoiSxbDvAmMwqomWzG32VHWcZXYnE8GDe+XLaLKKVmGu7d4StW40CHBgCH69aekgFJC5HQl2qOiEOccFTn+4MVGauyapUmQVQpCAwYcvlofmby9wx6EGiCfAjvM8JLAXxnLaF+OyJ2opiEkZ+E8084GMOTDp8Sg08g9cd3esBNIyh0NjBbpneiO8RIomJiWGvbsVf4jP81eGOPCU+h2ut\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z9DOKbga8cD3iOUtzDOm3w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0985ba1564fea743b1ba81708efb91795b0ac155165471ef839128c359b9249\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.03.932350v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1160969 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.03.932350v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C9Ck02uoi28Ru2_0LwoV5gBfMc2RojPRd9QUDI_PHaA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1160969" id="node1160969" data-pisa="biorxiv;2020.02.03.932350v3" data-pisa-master="biorxiv;2020.02.03.932350" data-apath="/biorxiv/early/2020/02/20/2020.02.03.932350.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1054-125X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gurjit S.</span> <span class="nlm-surname">Randhawa</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7495-5203" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Maximillian P.M.</span> <span class="nlm-surname">Soltysiak</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hadi El</span> <span class="nlm-surname">Roz</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Camila P.E.</span> <span class="nlm-surname">de Souza</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Hill</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lila</span> <span class="nlm-surname">Kari</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.03.932350 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1160969" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gurjit S. Randhawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Computer Science, The University of Western Ontario</span>, London, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gurjit%2BS.%2BRandhawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Randhawa%20GS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGurjit%2BS.%2BRandhawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1054-125X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gurjit S. Randhawa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grandha8@uwo.ca" class="" data-icon-position="" data-hide-link-title="0">grandha8@uwo.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maximillian P.M. Soltysiak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, The University of Western Ontario</span>, London, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maximillian%2BP.M.%2BSoltysiak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soltysiak%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaximillian%2BP.M.%2BSoltysiak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7495-5203" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maximillian P.M. Soltysiak</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hadi El Roz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, The University of Western Ontario</span>, London, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hadi%2BEl%2BRoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roz%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHadi%2BEl%2BRoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Camila P.E. de Souza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Statistical and Actuarial Sciences, The University of Western Ontario</span>, London, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Camila%2BP.E.%2Bde%2BSouza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Souza%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACamila%2BP.E.%2Bde%2BSouza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kathleen A. Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, The University of Western Ontario</span>, London, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BA.%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BA.%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lila Kari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>School of Computer Science, University of Waterloo</span>, Waterloo, ON, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lila%2BKari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kari%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALila%2BKari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.03.932350v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.03.932350v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.03.932350v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.03.932350v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.03.932350v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.03.932350v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1160969" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1160969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">As of February 20, 2020, the 2019 novel coronavirus (renamed to COVID-19) spread to 30 countries with 2130 deaths and more than 75500 confirmed cases. COVID-19 is being compared to the infamous SARS coronavirus, which resulted, between November 2002 and July 2003, in 8098 confirmed cases worldwide with a 9.6% death rate and 774 deaths. Though COVID-19 has a death rate of 2.8% as of 20 February, the 75752 confirmed cases in a few weeks (December 8, 2019 to February 20, 2020) are alarming, with cases likely being under-reported given the comparatively longer incubation period. Such outbreaks demand elucidation of taxonomic classification and origin of the virus genomic sequence, for strategic planning, containment, and treatment. This paper identifies an intrinsic COVID-19 genomic signature and uses it together with a machine learning-based alignment-free approach for an ultra-fast, scalable, and highly accurate classification of whole COVID-19 genomes. The proposed method combines supervised machine learning with digital signal processing for genome analyses, augmented by a decision tree approach to the machine learning component, and a Spearman’s rank correlation coefficient analysis for result validation. These tools are used to analyze a large dataset of over 5000 unique viral genomic sequences, totalling 61.8 million bp. Our results support a hypothesis of a bat origin and classify COVID-19 as <em>Sarbecovirus</em>, within <em>Betacoronavirus</em>. Our method achieves high levels of classification accuracy and discovers the most relevant relationships among over 5,000 viral genomes within a few minutes, <em>ab initio</em>, using raw DNA sequence data alone, and without any specialized biological knowledge, training, gene or genome annotations. This suggests that, for novel viral and pathogen genome sequences, this alignment-free whole-genome machine-learning approach can provide a reliable real-time option for taxonomic classification.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Coronaviruses are single-stranded positive-sense RNA viruses that are known to contain some of the largest viral genomes, up to around 32 kbp in length [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. After increases in the number of coronavirus genome sequences available following efforts to investigate the diversity in the wild, the family <em>Coronaviridae</em> now contains four genera (International Committee on Taxonomy of Viruses, [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]). While those species that belong to the genera <em>Alphacoronavirus</em> and <em>Betacoronavirus</em> can infect mammalian hosts, those in <em>Gammacoronavirus</em> and the recently defined <em>Deltacoronavirus</em> mainly infect avian species [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Phylogenetic studies have revealed a complex evolutionary history, with coronaviruses thought to have ancient origins and recent crossover events that can lead to cross-species infection [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Some of the largest sources of diversity for coronaviruses belong to the strains that infect bats and birds, providing a reservoir in wild animals for recombination and mutation that may enable cross-species transmission into other mammals and humans [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>].</p><p id="p-6">Like other RNA viruses, coronavirus genomes are known to have genomic plasticity, and this can be attributed to several major factors. RNA-dependent RNA polymerases (RdRp) have high mutation rates, reaching from 1 in 1000 to 1 in 10000 nucleotides during replication [<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Coronaviruses are also known to use a template switching mechanism which can contribute to high rates of homologous RNA recombination between their viral genomes [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Furthermore, the large size of coronavirus genomes is thought to be able to accommodate mutations to genes [<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>]. These factors help contribute to the plasticity and diversity of coronavirus genomes today.</p><p id="p-7">The highly pathogenic human coronaviruses, Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) belong to lineage B (sub-genus <em>Sarbecovirus)</em> and lineage C (sub-genus <em>Merbecovirus)</em> of <em>Betacoronavirus</em>, respectively [<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Both result from zoonotic transmission to humans and lead to symptoms of viral pneumonia, including fever, breathing difficulties, and more [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Recently, an unidentified pneumonia disease with similar symptoms caused an outbreak in Wuhan and is thought to have started from a local fresh seafood market [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. This was later attributed to a novel coronavirus deemed COVID-19 and represents the third major zoonotic human coronavirus of this century [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. As of February 20, confirmed cases have risen to 75752 globally, with infections reported in 30 countries [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. As a result, the World Health Organization set the risk assessment to “Very High” for China, where the bulk of the cases are contained and “High” for regional and global levels [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Initiatives to identify the source of transmission and possible intermediate animal vectors have commenced since the genome sequence became publicly available.</p><p id="p-8">From analyses employing whole genome to viral protein-based comparisons, the COVID-19 strain is thought to belong to lineage B <em>(Sarbecovirus)</em> of <em>Betacoronavirus.</em> From phylogenetic analysis of the RdRp protein, spike proteins, and full genomes of COVID-19 and other coronaviruses, it was found that COVID-19 is most closely related to two bat SARS-like coronaviruses, <em>bat-SL-CoVZXC21</em> and <em>bat-SL-CoVZC45</em>, found in Chinese horseshoe bats <em>Rhinolophus sinicus</em> [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>–<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. Along with the phylogenetic data, the genome organization of COVID-19 was found to be typical of lineage B <em>(Sarbecovirus) Betacoronaviruses</em> [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. From phylogenetic analysis of full genome alignment and similarity plots, it was found that COVID-19 has the highest similarity to the bat coronavirus <em>RaTG13</em> [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Close associations to bat coronavirus <em>RaTG13</em> and two bat SARS-like CoVs (<em>ZC45</em> and <em>ZXC21</em>) are also supported in alignment-based phylogenetic analyses [<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>]. Within the COVID-19 strains, over 99% sequence similarity and a lack of diversity within these strains suggest a common lineage and source, with support for recent emergence of the human strain [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. There is still ongoing debate whether the COVID-19 strain arose following recombination with previously identified bat and unknown coronaviruses [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] or arose independently as a new lineage to infect humans [<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>]. In combination with the identification that the angiotensin converting enzyme 2 (ACE2) protein is a receptor for COVID-19, as it is for SARS and other <em>Sarbecovirus</em> strains, the hypothesis that COVID-19 originated from bats is deemed very likely [<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>–<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>].</p><p id="p-9">All analyses performed thus far have been alignment-based and rely on the annotations of the viral genes. Though alignment-based methods have been successful in finding sequence similarities, their application can be challenging in many cases [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. It is realistically impossible to analyze thousands of complete genomes using alignment-based methods due to the heavy computation time. Moreover, the alignment demands the sequences to be continuously homologous which is not always the case. Alignment-free methods [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>–<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>] have been proposed in the past as an alternative to address the limitations of the alignment-based methods. Comparative genomics beyond alignment-based approaches have benefited from the computational power of machine learning. Machine learning-based alignment-free methods have also been used successfully for a variety of problems including virus classification [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>–<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">52</a>]. An alignment-free approach [<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>] was proposed for subtype classification of HIV-1 genomes and achieved ~ 97% classification accuracy. MLDSP [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>], with the use of a broad range of <em>1D</em> numerical representations of DNA sequences, has also achieved very high levels of classification accuracy with viruses. Even rapidly evolving, plastic genomes of viruses such as <em>Influenza</em> and <em>Dengue</em> are classified down to the level of strain and subtype, respectively with 100% classification accuracy. MLDSP-GUI [<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">52</a>] provides an option to use 2D Chaos Game Representation (CGR) [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>] as numerical representation of DNA sequences. CGR’s have a longstanding use in species classification with identification of biases in sequence composition [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">52</a>,<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>]. MLDSP-GUI has shown 100% classification accuracy for <em>Flavivirus</em> genus to species classification using <em>2D</em> CGR as numerical representation [<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">52</a>]. MLDSP and MLDSP-GUI have demonstrated the ability to identify the genomic signatures (a species-specific pattern known to be pervasive throughout the genome) with species level accuracy that can be used for sequence (dis)similarity analyses. In this study, we use MLDSP [<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a>] and MLDSP-GUI [<a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">52</a>] with CGR as a numerical representation of DNA sequences to assess the classification of COVID-19 from the perspective of machine learning-based alignment-free whole genome comparison of genomic signatures. Using MLDSP and MLDSP-GUI, we confirm that the COVID-19 belongs to the <em>Betacoronavirus</em>, while its genomic similarity to the sub-genus <em>Sarbecovirus</em> supports a possible bat origin.</p><p>This paper shows how machine learning using intrinsic genomic signatures can provide rapid alignment-free taxonomic classification of novel pathogens. Our method delivers accurate classifications of COVID-19 without <em>a priori</em> biological knowledge, by a simultaneous processing of the geometric space of all relevant viral genomes. The main contributions are:
</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-11">Identifying intrinsic viral genomic signatures, and utilizing them for a real-time and highly accurate machine learning-based classification of novel pathogen sequences, such as COVID-19;</p></li><li id="list-item-2"><p id="p-12">A general-purpose bare-bones approach, which uses raw DNA sequences alone and does not have any requirements for gene or genome annotation;</p></li><li id="list-item-3"><p id="p-13">The use of a “decision tree” approach to supervised machine learning (paralleling taxonomic ranks), for successive refinements of taxonomic classification.</p></li><li id="list-item-4"><p id="p-14">A comprehensive and “in minutes” analysis of a dataset of 5538 unique viral genomic sequences, for a total of 61. 8 million bp analyzed, with high classification accuracy scores at all levels, from the highest to the lowest taxonomic rank;</p></li><li id="list-item-5"><p id="p-15">The use of Spearman’s rank correlation analysis to confirm our results and the relatedness of the COVID-19 sequences to the known genera of the family <em>Coronaviridae</em> and the known sub-genera of the genus <em>Betacoronavirus.</em></p></li></ul></div><div class="section" id="sec-2"><h2 class="">Materials and methods</h2><p id="p-16">The Wuhan seafood market pneumonia virus (COVID-19 virus) isolate Wuhan-Hu-1 complete reference genome of 29903 bp was downloaded from the NCBI database on January 23, 2020. Also, we downloaded all of the available 29 sequences of COVID-19 and the bat <em>Betacoronavirus RaTG13</em> from the GISAID platform and two additional sequences (<em>ba,t-SL-CoVZC45</em>, and <em>bat-SL-CoVZXC21</em>) from the NCBI on January 27, 2019. We downloaded all of the available viral sequences from the Virus-Host DB (14688 sequences available on January 14, 2020). Virus-Host DB covers the sequences from the NCBI RefSeq (release 96, September 9, 2019), and GenBank (release 233.0, August 15, 2019). All sequences shorter than 2000 bp and longer than 50000 bp were ignored to address possible issues arising from sequence length bias.</p><p id="p-17">ML-DSP [<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">51</a>] and MLDSP-GUI [<a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">52</a>] were used as the machine learning-based alignment-free methods for complete genome analyses. As MLDSP-GUI is an extension of the ML-DSP methodology, we will refer to the method hereafter as MLDSP-GUI. Each genomic sequence is mapped into its respective genomic signal (a discrete numeric sequence) using a numerical representation. For this study, we use a two-dimensional <em>k</em>-mer (oligomers of length <em>k</em>) based numerical representation known as Chaos Game Representation (CGR) [<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">53</a>]. The <em>k</em>-mer value 7 is used for all the experiments. The value <em>k</em> = 7 achieved the highest accuracy scores for the HIV-1 subtype classification [<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">50</a>] and this value could be relevant for other virus related analyses. The magnitude spectra are then calculated by applying Discrete Fourier Transform (DFT) to the genomic signals [<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">51</a>]. A pairwise distance matrix is then computed using the Pearson Correlation Coefficient (PCC) [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>] as a distance measure between magnitude spectra. The distance matrix is used to generate the 3D Molecular Distance Maps (MoDMap3D) [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] by applying the classical Multi-Dimensional Scaling (MDS) [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. MoDMap3D represents an estimation of the relationship among sequences based on the genomic distances between the sequences. The feature vectors are constructed from the columns of the distance matrix and are used as an input to train six supervised-learning based classification models (Linear Discriminant, Linear SVM, Quadratic SVM, Fine KNN, Subspace Discriminant, and Subspace KNN) [<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">51</a>]. A 10-fold cross-validation is used to train, and test the classification models and the average of 10 runs is reported as the classification accuracy. The trained machine learning models are then used to test the COVID-19 sequences. The unweighted pair group method with arithmetic mean (UPGMA) phylogenetic tree is also computed using the pairwise distance matrix.</p><p id="p-18">In this paper, MLDSP-GUI is augmented by a decision tree approach to the supervised machine learning component and a Spearman’s rank correlation coefficient analysis for result validation. The decision tree parallels the taxonomic classification levels, and is necessary so as to minimize the number of calls to the supervised classifier module, as well as to maintain a reasonable number of clusters during each supervised training session. For validation of MLDSP-GUI results using CGR as a numerical representation, we use Spearman’s rank correlation coefficient [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>–<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>], as follows. The frequency of each <em>k</em>-mer is calculated in each genome. Due to differences in genome length between species, proportional frequencies are computed by dividing each <em>k</em>-mer frequency by the length of the respective sequence. To determine whether there is a correlation between <em>k</em>-mer frequencies in COVID-19 and specific taxonomic groups, a Spearman’s rank correlation coefficient test is conducted for <em>k</em> = 1 to <em>k</em> = 7.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><p id="p-19"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> provides the details of three datasets Test-1, Test-2, Test-3a and Test-3b used for analyses with MLDSP-GUI. Each dataset’s composition (clusters with number of sequences), the respective sequence length statistics, and results of MLDSP-GUI after applying 10-fold cross-validation as classification accuracy scores are shown. The classification accuracy scores for all six classification models are shown with their average, see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-20">As shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, for the first test (Test-1), we organized the dataset of sequences into 12 clusters (11 families, and Riboviria realm). Only the families with at least 100 sequences were considered. The Riboviria cluster contains all families that belong to the realm Riboviria. For the clusters with more than 500 sequences, we selected 500 sequences at random. Our method can handle all of the available 14668 sequences, but using imbalanced clusters, in regard to the number of sequences, can introduce an unwanted bias. After filtering out the sequences, our pre-processed dataset is left with 3273 sequences organized into 12 clusters <em>(Adenoviridae, Anelloviridae, Caudovirales</em>, <em>Geminiviridae</em>, <em>Genomoviridae</em>, <em>Microviridae</em>, <em>Ortervirales</em>, <em>Papillomaviridae, Parvoviridae, Polydnaviridae, Polyomaviridae</em>, and Riboviria). We used MLDSP-GUI with CGR as the numerical representation at <em>k</em> = 7. The maximum classification accuracy of 94.9% is obtained using the Quadratic SVM model. The respective MoDMap3D is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1(a)</a>. All six classification models trained on 3273 sequences were used to classify (predict the label of) the 29 COVID-19 sequences. All of our machine learning-based models correctly predicted and confirmed the label as Riboviria for all 29 sequences (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MoDMap3D of (a) 3273 viral sequences from Test-1 representing 11 viral families and realm Riboviria, (b) 2779 viral sequences from Test-2 classifying 12 viral families of realm Riboviria, (c) 208 Coronaviridae sequences from Test-3a classified into genera." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MoDMap3D of (a) 3273 viral sequences from Test-1 representing 11 viral families and realm Riboviria, (b) 2779 viral sequences from Test-2 classifying 12 viral families of realm Riboviria, (c) 208 &lt;em&gt;Coronaviridae&lt;/em&gt; sequences from Test-3a classified into genera.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F1.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F1.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig 1.</span> <p id="p-21" class="first-child">MoDMap3D of (a) 3273 viral sequences from Test-1 representing 11 viral families and realm Riboviria, (b) 2779 viral sequences from Test-2 classifying 12 viral families of realm Riboviria, (c) 208 <em>Coronaviridae</em> sequences from Test-3a classified into genera.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Test-1 classified the COVID-19 virus as belonging to the realm Riboviria. The second test (Test-2) is designed to classify COVID-19 among the families of the Riboviria realm. We completed the dataset pre-processing using the same rules as in Test-1 and obtained a dataset of 2779 sequences placed into the 12 families (<em>Betaflexiviridae, Bromoviridae, Caliciviridae, Coronaviridae, Flaviviridae, Peribunyaviridae, Phenuiviridae, Picornaviridae, Potyviridae, Reoviridae, Rhabdoviridae</em>, and <em>Secoviridae</em>), see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. MLDSP-GUI with CGR at <em>k</em> = 7 as the numerical representation was used for the classification of the dataset in Test-2. The maximum classification accuracy of 93.1% is obtained using the Quadratic SVM model. The respective MoDMap3D is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1(b)</a>. All six classification models trained on 2779 sequences were used to classify (predict the label of) the 29 COVID-19 sequences. All of our machine learning-based models predicted the label as <em>Coronaviridae</em> for all 29 sequences (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) with 100% classification accuracy. Test-2 correctly predicted the family of COVID-19 sequences as <em>Coronaviridae.</em> Test-3 performs the genus-level classification.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1164192/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1164192/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1164192" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-23" class="first-child">Classification accuracy scores of viral sequences at different levels of taxonomy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">All classifiers trained on Test-1, Test-2, Test-3a, and Test-3b datasets were used to predict the labels of 29 COVID-19 viral sequences. All classifiers predicted the correct labels for all of the sequences (Riboviria when trained using Test-1, <em>Coronaviridae</em> when trained using Test-2, and <em>Betacoronavirus</em> when trained using Test-3a and Test-3b).</p><p id="p-25">The third test (Test-3a) is designed to classify the COVID-19 sequences at the genus level. We considered 208 <em>Coronaviridae</em> sequences available under four genera <em>(Alphacoronavirus, Betacoronavirus, Deltacoronavirus, Gammacoronavirus)</em> (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). MLDSP-GUI with CGR at <em>k</em> = 7 as the numerical representation was used for the classification of the dataset in Test-3a. The maximum classification accuracy of 98.1% is obtained using the Linear Discriminant model and the respective MoDMap3D is shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1(c)</a>. All six classification models trained on 208 sequences were used to classify (predict the label of) the 29 COVID-19 sequences. All of our machine learning-based models predicted the label as <em>Betacoronavirus</em> for all 29 sequences (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). To verify that the correct prediction is not an artifact of possible bias because of larger <em>Betacoronavirus</em> cluster, we did a secondary Test-3b with cluster size limited to the size of smallest cluster (after removing the <em>Gammacoronavirus</em> because it just had 9 sequences). The maximum classification accuracy of 100% is obtained using the Linear Discriminant model for Test-3b. All six classification models trained on 60 sequences were used to classify the 29 COVID-19 sequences. All of our machine learning-based models predicted the label as <em>Betacoronavirus</em> for all 29 sequences (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). This secondary test showed that the possible bias is not significant enough to have any impact on the classification performance.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1164163/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1164163/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1164163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-26" class="first-child">Predicted taxonomic labels of 29 COVID-19 sequences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Given confirmation that the COVID-19 belongs to the <em>Betacoronavirus</em> genus, there now is a question of its origin and relation to the other viruses of the same genus. To examine this question, we preprocessed our dataset from our third test to keep the sub-clusters of the <em>Betacoronavirus</em> with at least 10 sequences (Test-4). This gives 124 sequences placed into four clusters (<em>Embecovirus</em>, <em>Merbecovirus</em>, <em>Nobecovirus</em>, <em>Sarbecovirus)</em> (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The maximum classification accuracy of 98.4% with CGR at <em>k</em> = 7 as the numerical representation is obtained using the Quadratic SVM model. The respective MoDMap3D is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2(a)</a>. All six classifiers trained on 124 sequences predicted the label as <em>Sarbecovirus</em>, when used to predict the labels of 29 COVID-19 sequences. For Test-5, we added COVID-19 with 29 sequences as the fifth cluster, see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. The maximum classification accuracy of 98.7% with CGR at <em>k</em> = 7 as the numerical representation is obtained using the Subspace Discriminant model. The respective MoDMap3D is shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2(b)</a>. In the MoDMap3D plot from Test-5, COVID-19 sequences are placed in a single distinct cluster, see <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2(b)</a>. As visually suggested by the MoDMap3D (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2(b)</a>), the average inter-cluster distances confirm that the COVID-19 sequences are closest to the <em>Sarbecovirus</em> (average distance 0.0556), followed by <em>Merbecovirus</em> (0.0746), <em>Embecovirus</em> (0.0914), and <em>Nobecovirus</em> (0.0916). The three closest sequences based on the average distances from all COVID-19 sequences are <em>RaTG13</em> (0.0203), <em>bat-SL-CoVZCĄ5</em> (0.0418), and <em>bat-SL-CoVZXC21</em> (0.0428).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MoDMap3D of (a) 124 Betacoronavirus sequences from Test-4 classified into sub-genera, (b) 153 viral sequences from Test-5 classified into 4 sub-genera and COVID-19, (c) 76 viral sequences from Test 6 classified into Sarbecovirus and COVID-19." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;MoDMap3D of (a) 124 &lt;em&gt;Betacoronavirus&lt;/em&gt; sequences from Test-4 classified into sub-genera, (b) 153 viral sequences from Test-5 classified into 4 sub-genera and COVID-19, (c) 76 viral sequences from Test 6 classified into &lt;em&gt;Sarbecovirus&lt;/em&gt; and COVID-19.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F2.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F2.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 2.</span> <p id="p-28" class="first-child">MoDMap3D of (a) 124 <em>Betacoronavirus</em> sequences from Test-4 classified into sub-genera, (b) 153 viral sequences from Test-5 classified into 4 sub-genera and COVID-19, (c) 76 viral sequences from Test 6 classified into <em>Sarbecovirus</em> and COVID-19.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1164178/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1164178/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1164178" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-29" class="first-child">Genus to sub-genus classification accuracy scores of <em>Betacoronavirus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">For Test-6, we classified <em>Sarbecovirus</em> (47 sequences) and COVID-19 (29 sequences) clusters and achieved separation of the two clusters visually apparent in the MoDMap3D, see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2(c)</a>. Quantitatively, using 10-fold cross-validation, all six of our classifiers report 100% classification accuracy. We generated a phylogenetic tree based on all pairwise distances for the dataset in Test-6 that shows the separation of the two clusters and relationships within the clusters (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). As observed in Test-5, the phylogenetic tree shows that the COVID-19 sequences are closer to the bat <em>Betacoronavirus RaTG13</em> sequence collected from a bat host.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The UPGMA phylogenetic tree using the Pearson Correlation Coefficient generated pairwise distance matrix shows COVID-19 (Red) sequences proximal to the bat Betacoronavirus RaTG13 (Blue) and bat SARS-like coronaviruses ZC45/ZXC21 (Green) in a distinct lineage from the rest of Sarbecovirus sequences" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The UPGMA phylogenetic tree using the Pearson Correlation Coefficient generated pairwise distance matrix shows COVID-19 (Red) sequences proximal to the bat &lt;em&gt;Betacoronavirus&lt;/em&gt; RaTG13 (Blue) and bat SARS-like coronaviruses ZC45/ZXC21 (Green) in a distinct lineage from the rest of &lt;em&gt;Sarbecovirus&lt;/em&gt; sequences&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F3.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F3.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 3.</span> <p id="p-31" class="first-child">The UPGMA phylogenetic tree using the Pearson Correlation Coefficient generated pairwise distance matrix shows COVID-19 (Red) sequences proximal to the bat <em>Betacoronavirus</em> RaTG13 (Blue) and bat SARS-like coronaviruses ZC45/ZXC21 (Green) in a distinct lineage from the rest of <em>Sarbecovirus</em> sequences</p><div class="sb-div caption-clear"></div></div></div><p id="p-32"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the Chaos Game Representation (CGR) plots of different sequences from the four different genera (<em>Alphacoronavirus</em>, <em>Betacoronavirus</em>, <em>Deltacoronavirus</em>, <em>Gammacoronavirus)</em> of the family <em>Coronaviridae.</em> The CGR plots visually suggests and the pairwise distances confirm that the genomic signature of the COVID-19 Wuhan-Hu-1 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4(a)</a>) is closer to the genomic signature of the <em>BetaCov-RaTG13</em> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4(b)</a>; distance: 0.0204), followed by the genomic signatures of <em>bat-SL-CoVZCĄδ</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4(c)</a>; distance: 0.0417), <em>bat-SL-CoVZXC21(<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4(d)</a></em>; distance: 0.0428), <em>Alphacoronavirus /DQ811787 PRCV ISU</em>-1 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4(e)</a>; distance: 0.0672), <em>Gammacoronavirus /</em> Infectious bronchitis virus NGA /A116E<em>7/2006/FN</em>430415 (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4(f)</a>; distance: 0.0791), and <em>Deltacoronavirus /</em> PDCoV / USA / Illinois121 /2014/KJ481931 (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4(g)</a>; distance: 0.0851).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chaos Game Representation (CGR) plots at k = 7 of (a) COVID-19 / Wuhan seafood market pneumonia virus isolate Wuhan-Hu-1/MN908947.3, (b) Betacoronavirus / CoV / Bat / Yunnan / RaTG13 /EPI_ISL_402131, (c) Betacoronavirus / Bat SARS-like coronavirus isolate bat-SL-CoVZC45 /MG772933.1, (d) Betacoronavirus / Bat SARS-like coronavirus isolate bat-SL-CoVZXC21 /MG772934.1, (e) Alphacoronavirus /DQ811787 PRCV ISU-1, (f) Gammacoronavirus / Infectious bronchitis virus NGA /A116E7/2006/FN430415, and (g) Deltacoronavirus / PDCoV / USA / Illinois121 /2014/KJ481931. Chaos plot vertices are assigned top left Cytosine, top right Guanine, bottom left Adenine and bottom right Thymine." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chaos Game Representation (CGR) plots at &lt;em&gt;k&lt;/em&gt; = 7 of (a) COVID-19 / Wuhan seafood market pneumonia virus isolate Wuhan-Hu-1/MN908947.3, (b) &lt;em&gt;Betacoronavirus /&lt;/em&gt; CoV / Bat / Yunnan / RaTG13 /EPI_ISL_402131, (c) &lt;em&gt;Betacoronavirus /&lt;/em&gt; Bat SARS-like coronavirus isolate bat-SL-CoVZC45 /MG772933.1, (d) &lt;em&gt;Betacoronavirus /&lt;/em&gt; Bat SARS-like coronavirus isolate bat-SL-CoVZXC21 /MG772934.1, (e) &lt;em&gt;Alphacoronavirus /DQ811787 PRCV ISU&lt;/em&gt;-1, (f) &lt;em&gt;Gammacoronavirus /&lt;/em&gt; Infectious bronchitis virus NGA /A116E7/2006/FN430415, and (g) &lt;em&gt;Deltacoronavirus /&lt;/em&gt; PDCoV / USA / Illinois121 /2014/KJ481931. Chaos plot vertices are assigned top left Cytosine, top right Guanine, bottom left Adenine and bottom right Thymine.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F4.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F4.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 4.</span> <p id="p-33" class="first-child">Chaos Game Representation (CGR) plots at <em>k</em> = 7 of (a) COVID-19 / Wuhan seafood market pneumonia virus isolate Wuhan-Hu-1/MN908947.3, (b) <em>Betacoronavirus /</em> CoV / Bat / Yunnan / RaTG13 /EPI_ISL_402131, (c) <em>Betacoronavirus /</em> Bat SARS-like coronavirus isolate bat-SL-CoVZC45 /MG772933.1, (d) <em>Betacoronavirus /</em> Bat SARS-like coronavirus isolate bat-SL-CoVZXC21 /MG772934.1, (e) <em>Alphacoronavirus /DQ811787 PRCV ISU</em>-1, (f) <em>Gammacoronavirus /</em> Infectious bronchitis virus NGA /A116E7/2006/FN430415, and (g) <em>Deltacoronavirus /</em> PDCoV / USA / Illinois121 /2014/KJ481931. Chaos plot vertices are assigned top left Cytosine, top right Guanine, bottom left Adenine and bottom right Thymine.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The Spearman’s rank correlation coefficient tests were used to further confirm the ML-DSP findings. The first test in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the COVID-19 being compared to the four genera; <em>Alphacoronavirus</em>, <em>Betacoronavirus</em>, <em>Gammacoronavirus</em> and <em>Deltacoronavirus.</em> The COVID-19 showed the highest <em>k</em>-mer frequency correlation to <em>Betacoronavirus</em> at <em>k</em> = 7 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), which is consistent with the ML-DSP results in Test-3 (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). The COVID-19 was then compared to all sub-genera within the <em>Betacoronavirus</em> genus: <em>Embecovirus, Merbecovirus, Nobecovirs</em> and <em>Sarbecovirus</em> seen in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The Spearman’s rank test was again consistent with the ML-DSP results seen in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, as the <em>k</em>-mer frequencies at <em>k</em> = 7 showed the highest correlation to the sub-genus <em>Sarbecovirus</em> (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). These tests confirm the findings in ML-DSP and are consistent with the COVID-19 virus as part of the sub-genus <em>Sarbecovirus.</em></p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hexbin scatterplots of the proportional k-mer (k = 7) frequencies of the COVID-19 sequences versus the four genera: (a) Alphacoronavirus, &#x3C1; = 0.7; (b) Betacoronavirus, &#x3C1; = 0.74; (c) Gammacoronavirus, &#x3C1; = 0.63 and (d) Deltacoronavirus, &#x3C1; = 0.6. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All &#x3C1; values resulted in p-values &lt; 10-5 for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (b) are more strongly correlated and have less deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hexbin scatterplots of the proportional &lt;em&gt;k&lt;/em&gt;-mer (&lt;em&gt;k&lt;/em&gt; = 7) frequencies of the COVID-19 sequences versus the four genera: (a) &lt;em&gt;Alphacoronavirus, &#x3C1;&lt;/em&gt; = 0.7; (b) &lt;em&gt;Betacoronavirus, &#x3C1;&lt;/em&gt; = 0.74; (c) &lt;em&gt;Gammacoronavirus, &#x3C1;&lt;/em&gt; = 0.63 and (d) &lt;em&gt;Deltacoronavirus, &#x3C1;&lt;/em&gt; = 0.6. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All &lt;em&gt;&#x3C1;&lt;/em&gt; values resulted in &lt;em&gt;p&lt;/em&gt;-values &lt; 10&lt;sup&gt;-5&lt;/sup&gt; for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (b) are more strongly correlated and have less deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F5.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F5.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 5.</span> <p id="p-35" class="first-child">Hexbin scatterplots of the proportional <em>k</em>-mer (<em>k</em> = 7) frequencies of the COVID-19 sequences versus the four genera: (a) <em>Alphacoronavirus, ρ</em> = 0.7; (b) <em>Betacoronavirus, ρ</em> = 0.74; (c) <em>Gammacoronavirus, ρ</em> = 0.63 and (d) <em>Deltacoronavirus, ρ</em> = 0.6. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All <em>ρ</em> values resulted in <em>p</em>-values &lt; 10<sup>-5</sup> for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (b) are more strongly correlated and have less deviation.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hexbin scatterplots of the proportional k-mer (k = 7) frequencies of the COVID-19 sequences versus the four sub-genera: (a) Embecovirus, &#x3C1; = 0.59; (b) Merbecovirus, &#x3C1; = 0.64; (c) Nobecovirus, &#x3C1; = 0.54 and (d) Sarbecovirus, &#x3C1; = 0.72. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All &#x3C1; values resulted in p-values &lt; 10-5 for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (d) are more strongly correlated and have less deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hexbin scatterplots of the proportional &lt;em&gt;k&lt;/em&gt;-mer (&lt;em&gt;k&lt;/em&gt; = 7) frequencies of the COVID-19 sequences versus the four sub-genera: (a) &lt;em&gt;Embecovirus, &#x3C1;&lt;/em&gt; = 0.59; (b) &lt;em&gt;Merbecovirus, &#x3C1;&lt;/em&gt; = 0.64; (c) &lt;em&gt;Nobecovirus, &#x3C1;&lt;/em&gt; = 0.54 and (d) &lt;em&gt;Sarbecovirus, &#x3C1;&lt;/em&gt; = 0.72. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All &lt;em&gt;&#x3C1;&lt;/em&gt; values resulted in &lt;em&gt;p&lt;/em&gt;-values &lt; 10&lt;sup&gt;-5&lt;/sup&gt; for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (d) are more strongly correlated and have less deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F6.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F6.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 6.</span> <p id="p-36" class="first-child">Hexbin scatterplots of the proportional <em>k</em>-mer (<em>k</em> = 7) frequencies of the COVID-19 sequences versus the four sub-genera: (a) <em>Embecovirus, ρ</em> = 0.59; (b) <em>Merbecovirus, ρ</em> = 0.64; (c) <em>Nobecovirus, ρ</em> = 0.54 and (d) <em>Sarbecovirus, ρ</em> = 0.72. The color of each hexagonal bin in the plot represents the number of points (in natural logarithm scale) overlapping at that position. All <em>ρ</em> values resulted in <em>p</em>-values &lt; 10<sup>-5</sup> for the correlation test. By visually inspecting each hexbin scatterplot, the degree of correlation is displayed by the variation in spread between the points. Hexagonal points that are closer together and less dispersed as seen in (d) are more strongly correlated and have less deviation.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1164182/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1164182/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1164182" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-37" class="first-child">Spearman’s rank correlation coefficient (<em>ρ</em>) values from <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">6</a>, for which all <em>p</em>-values &lt; 10<sup>-5</sup>. The strongest correlation value was found between <em>Betacoronavirus</em> and <em>Sarbecovirus</em> when using the data sets from Test 3a from <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a> and Test 4 from <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-4"><h2 class="">Discussion</h2><p id="p-38">Prior work elucidating the evolutionary history of the Wuhan COVID-19 virus had suggested an origin from bats prior to zoonotic transmission [<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. Most early cases of individuals infected with COVID-19 had contact with the Huanan South China Seafood Market [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>]. Human-to-human transmission is confirmed, further highlighting the need for continued intervention [<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>–<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]. Still, the early COVID-19 genomes that have been sequenced and uploaded are over 99% similar, suggesting these infections result from a recent cross-species event [<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>].</p><p id="p-39">These prior analyses relied upon alignment-based methods to identify relationships between COVID-19 and other coronaviruses with nucleotide and amino acid sequence similarities. When analyzing the conserved replicase domains of ORF1ab for coronavirus species classification, nearly 94% of amino acid residues were identical to SARS-CoV, yet overall genome similarity was only around 70%, confirming that COVID-19 was genetically different [<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a>]. Within the RdRp region, it was found that another bat coronavirus, <em>RaTG13</em>, was the closest relative to COVID-19 and formed a distinct lineage from other bat SARS-like coronaviruses [<a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>]. Other groups found that two bat SARS-like coronaviruses, <em>bat-SL-CoVZCĄδ</em> and <em>bat-SL-CoVZXC21</em>, were also closely related to COVID-19 [<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">34</a>–<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>]. There is a consensus that these three bat viruses are most similar to COVID-19, however, whether or not COVID-19 arose from a recombination event is still unknown [<a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a>].</p><p id="p-40">Regardless of the stance on recombination, current consensus holds that the hypothesis of COVID-19 originating from bats is highly likely. Bats have been identified as a reservoir of mammalian viruses and cross-species transmission to other mammals, including humans [<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>–<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. Prior to intermediary cross-species infection, the coronaviruses SARS-CoV and MERS-CoV were also thought to have originated in bats [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>,<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>,<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>]. Many novel SARS-like coronaviruses have been discovered in bats across China, and even in European, African and other Asian countries [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>–<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. With widespread geographic coverage, SARS-like coronaviruses have likely been present in bats for a long period of time and novel strains of these coronaviruses can arise through recombination [<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>]. Whether or not COVID-19 was transmitted directly from bats, or from intermediary hosts, is still unknown, and will require identification of COVID-19 in species other than humans, notably from the wet market and surrounding area it is thought to have originated from [<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>]. While bats have been reported to have been sold at the Huanan market, at this time, it is still unknown if there were intermediary hosts involved prior to transmission to humans [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>]. Snakes had been proposed as an intermediary host for COVID-19 based on relative synonymous codon usage bias studies between viruses and their hosts [<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a>], however, this claim has been disputed [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>]. China CDC released information about environmental sampling in the market and indicated that 33 of 585 samples had evidence of COVID-19, with 31 of these positive samples taken from the location where wildlife booths were concentrated, suggesting possible wildlife origin [<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>,<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. Detection of SARS-CoV in Himalyan palm civets and horseshoe bats identified 29 nucleotide sequences that helped trace the origins of SARS-CoV isolates in humans to these intermediary species [<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">76</a>]. Sampling additional animals at the market and wildlife in the surrounding area may help elucidate whether intermediary species were involved or not, as was possible with the SARS-CoV.</p><p id="p-41">Viral outbreaks like COVID-19 demand timely analysis of genomic sequences to guide the research in the right direction. This problem being time-sensitive requires quick sequence similarity comparison against thousands of known sequences to narrow down the candidates of possible origin. Alignment-based methods are known to be time-consuming and can be challenging in cases where homologous sequence continuity cannot be ensured. It is challenging (and sometimes impossible) for alignment-based methods to compare a large number of sequences that are too different in their composition. Alignment-free methods have been used successfully in the past to address the limitations of the alignment-based methods [<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>–<a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">52</a>]. The alignment-free approach is quick and can handle a large number of sequences. Moreover, even the sequences coming from different regions with different compositions can be easily compared quantitatively, with equally meaningful results as when comparing homologous/similar sequences. We use MLDSP-GUI (a variant of MLDSP with additional features), a machine learning-based alignment-free method successfully used in the past for sequence comparisons and analyses [<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">51</a>]. The main advantage alignment-free methodology offers is the ability to analyze large datasets rapidly. In this study we confirm the taxonomy of COVID-19 and, more generally, propose a method to efficiently analyze and classify a novel unclassified DNA sequence against the background of a large dataset. We namely use a “decision tree” approach (paralleling taxonomic ranks), and start with the highest taxonomic level, train the classification models on the available complete genomes, test the novel unknown sequences to predict the label among the labels of the training dataset, move to the next taxonomic level, and repeat the whole process down to the lowest taxonomic label.</p><p id="p-42">Test-1 starts at the highest available level and classifies the viral sequences to the 11 families and Riboviria realm (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). There is only one realm available in the viral taxonomy, so all of the families that belong to the realm Riboviria are placed into a single cluster and a random collection of 500 sequences are selected. No realm is defined for the remaining 11 families. The objective is to train the classification models with the known viral genomes and then predict the labels of the COVID-19 virus sequences. The maximum classification accuracy score of 95% was obtained using the Quadratic SVM model. This test demonstrates that MLDSP-GUI can distinguish between different viral families. The trained models are then used to predict the labels of 29 COVID-19 sequences. As expected, all classification models correctly predict that the COVID-19 sequences belong to the Riboviria realm, see <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>. Test-2 is composed of 12 families from the Riboviria, see <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>, and the goal is to test if MLDSP-GUI is sensitive enough to classify the sequences at the next lower taxonomic level. It should be noted that as we move down the taxonomic levels, sequences become much more similar to one another and the classification problem becomes challenging. MLDSP-GUI is still able to distinguish between the sequences within the Riboviria realm with a maximum classification accuracy of 91.1% obtained using the Linear Discriminant classification model. When COVID-19 sequences are tested using the models trained on Test-2, all of the models correctly predict the COVID-19 sequences as <em>Coronaviridae</em> (<a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). Test-3a moves down another taxonomic level and classifies the <em>Coronaviridae</em> family to four genera <em>(Alphacoronavirus, Betacoronavirus, Deltacoronavirus, Gammacoronavirus)</em>, see <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>. MLDSP-GUI distinguishes sequences at the genus level with a maximum classification accuracy score of 98%, obtained using the Linear Discriminant model. This is a very high accuracy rate considering that no alignment is involved and the sequences are very similar. All trained classification models correctly predict the COVID-19 as <em>Betacoronavirus</em>, see <a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>. Test-3a has <em>Betacoronavirus</em> as the largest cluster and it can be argued that the higher accuracy could be a result of this bias. To avoid bias, we did an additional test removing the smallest cluster <em>Gammacoronavirus</em> and limiting the size of remaining three clusters to the size of the cluster with the minimum number of sequences i.e. 20 with Test-3b. MLDSP-GUI obtains 100% classification accuracy for this additional test and still predicts all of the COVID-19 sequences as <em>Betacoronavirus.</em> These tests confirm that the COVID-19 are from the genus <em>Betacoronavirus.</em></p><p id="p-43">Sequences become very similar at lower taxonomic levels (sub-genera and species). Test-4, Test-5, and Test-6 investigate within the genus <em>Betacoronavirus</em> for sub-genus classification. Test-4 is designed to classify <em>Betacoronavirus</em> into the four sub-genera <em>(Embecovirus, Merbecovirus, Nobecovirus, Sarbecovirus)</em>, see <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>. MLDSP-GUI distinguishes sequences at the sub-genus level with a maximum classification accuracy score of 98.4%, obtained using the Quadratic SVM model. All of the classification models trained on the dataset in Test-4 predicted the label of all 29 COVID-19 sequences as <em>Sarbecovirus</em>. This suggests substantial similarity between COVID-19 and the <em>Sarbecovirus</em> sequences. Test-5 and Test-6 (see <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>) are designed to verify that COVID-19 sequences can be differentiated from the known species in the <em>Betacoronavirus</em> genus. MLDSP-GUI achieved a maximum classification score of 98.7% for Test-5 and 100% for Test-6 using Subspace Discriminant classification model. This shows that although COVID-19 and <em>Sarbecovirus</em> are closer on the basis of genomic similarity (Test-4), they are still distinguishable from known species. Therefore, these results suggest that COVID-19 may represent a genetically distinct species of <em>Sarbecovirus.</em> All COVID-19 virues are visually seen in MoDMap3D generated from Test-5 (see <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2(b)</a>) as a closely packed cluster and it supports a fact that there is 99% similarity among these sequences [<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">31</a>]. The MoDMap3D generated from the Test-5 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2(b)</a>) visually suggests and the average distances from COVID-19 sequences to all other sequences confirm that the COVID-19 sequences are most proximal to the <em>RaTG13</em> (distance: 0.0203), followed by the <em>bat-SL-CoVZCĄ5</em> (0.0418), and <em>bat-SL-CoVZX21</em> (0.0428). To confirm this proximity, a UPGMA phylogenetic tree is computed from the PCC-based pairwise distance matrix of sequences in Test-6, see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. The phylogenetic tree placed the <em>RaTG13</em> sequence closest to the COVID-19 sequences, followed by the <em>bat-SL-CoVZCĄ.5</em> and <em>bat-SL-CoVZX21</em> sequences. This closer proximity represents the smaller genetic distances between these sequences and aligns with the visual sequence relationships shown in the MoDMap3D of <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2(b)</a>.</p><p id="p-44">We further confirm our results regarding the closeness of COVID-19 with the sequences from the <em>Betacoronavirus</em> genus (especially sub-genus <em>Sarbecovirus)</em> by a quantitative analysis based on the Spearman’s rank correlation coefficient tests. Spearman’s rank correlation coefficient [<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>–<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">60</a>] tests were applied to the frequencies of oligonucleotide segments, adjusting for the total number of segments, to measure the degree and statistical significance of correlation between two sets of genomic sequences. Spearman’s p value provides the degree of correlation between the two groups and their <em>k</em>-mer frequencies. The COVID-19 virus was compared to all genera under the <em>Coronaviridae</em> family and the <em>k</em>-mer frequencies showed the strongest correlation to the genus <em>Betacoronavirus</em>, and more specifically <em>Sarbecovirus.</em> The Spearman’s rank tests corroborate that the COVID-19 virus is part of the <em>Sarbecovirus</em> sub-genus, as shown by CGR and ML-DSP. When analyzing sub-genera, it could be hard to classify at lower <em>k</em> values due to the short oligonucleotide frequencies not capturing enough information to highlight the distinctions. Therefore despite the Spearman’s rank correlation coefficient providing results for <em>k</em> = 1 to <em>k</em> = 7, the higher <em>k</em>-mer lengths provided more accurate results, and <em>k</em> = 7 was used.</p><p id="p-45">Attributes of the COVID-19 genomic signature are consistent with previously reported mechanisms of innate immunity operating in bats as a host reservoir for coronaviruses. Vertebrate genomes are known to have an under-representation of CG dinucleotides in their genomes, otherwise known as CG suppression [<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>,<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>]. This feature is thought to have been due to the accumulation of spontaneous deamination mutations of methyl-cytosines over time [<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">81</a>]. As viruses are obligate parasites, evolution of viral genomes is intimately tied to the biology of their hosts [<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>]. As host cells develop strategies such as RNA interference and restriction-modification systems to prevent and limit viral infections, viruses will continue to counteract these strategies [<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">82</a>–<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>]. Dinucleotide composition and biases are pervasive across the genome and make up a part of the organism’s genomic signature [<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">83</a>]. These host genomes have evolutionary pressures that shape the host genomic signature, such as the pressure to eliminate CG dinucleotides within protein coding genes in humans [<a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">82</a>]. Viral genomes have been shown to mimic the same patterns of the hosts, including single-stranded positive-sense RNA viruses, which suggests that many RNA viruses can evolve to mimic the same features of their host’s genes and genomic signature [<a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">81</a>–<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>]. As genomic composition, specifically in mRNA, can be used as a way of discriminating self vs non-self RNA, the viral genomes are likely shaped by the same pressures that influence the host genome [<a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">82</a>]. One such pressure on DNA and RNA is the APOBEC family of enzymes, members of which are known to cause G to A mutations [<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">85</a>–<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>]. While these enzymes primarily work on DNA, it has been demonstrated that these enzymes can also target RNA viral genomes [<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>]. The APOBEC enzymes therefore have RNA editing capability and may help contribute to the innate defence system against various RNA viruses [<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">85</a>]. This could therefore have a direct impact on the genomic signature of RNA viruses. Additional mammalian mechanisms for inhibiting viral RNA have been highlighted for retroviruses with the actions of zinc-finger antiviral protein (ZAP) [<a id="xref-ref-81-4" class="xref-bibr" href="#ref-81">81</a>]. ZAP targets CG dinucleotide sequences, and in vertebrate host cells with the CG suppression in host genomes, this can serve as a mechanism for the distinction of self vs non-self RNA and inhibitory consequences [<a id="xref-ref-81-5" class="xref-bibr" href="#ref-81">81</a>]. Coronaviruses have A/U rich and C/G poor genomes, which over time may have been, in part, a product of cytidine deamination and selection against CG dinucleotides [<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>–<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>]. This is consistent with the fact that bats serve as a reservoir for many coronaviruses and that bats have been observed to have some of the largest and most diverse arrays of APOBEC genes in mammals [<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>]. The Spearman’s rank correlation data and the patterns observed in the CGR images from <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>, of the coronavirus genomes, including COVID-19 identify patterns such as CG underepresentation, also present in vertebrate and, importantly, bat host genomes.</p><p id="p-46">With human-to-human transmission confirmed and concerns for possible asymptomatic transmission, there is a strong need for continued intervention to prevent the spread of the virus [<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">61</a>–<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">63</a>]. Due to the high amino acid similarities between COVID-19 and SARS-CoV main protease essential for viral replication and processing, anticoronaviral drugs targeting this protein and other potential drugs have been identified using virtual docking to the protease for treatment of COVID-19 [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>–<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>]. The human ACE2 receptor has also been identified as the potential receptor for COVID-19 and represents a potential target for treatment [<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>].</p><p id="p-47">MLDSP-GUI is an ultra-fast, alignment-free method as is evidenced by the time-performance of MLDSP-GUI for Test-1 to Test-6 given in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. MLDSP-GUI took just 10.55 seconds to compute a pairwise distance matrix (including reading sequences, computing magnitude spectra using DFT, and calculating the distance matrix using PCC combined) for the Test-1 (largest dataset used in this study with 3273 complete genomes). All of the tests combined (Test-1 to Test-6) are doable in under 10 minutes including the computationally heavy 10-fold cross-validation, and testing of 29 COVID-19 sequences.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time performance of MLDSP-GUI for Test1 to Test-6 (in seconds)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542815415" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time performance of MLDSP-GUI for Test1 to Test-6 (in seconds).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F7.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F7.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/20/2020.02.03.932350/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 7.</span> <p id="p-48" class="first-child">Time performance of MLDSP-GUI for Test1 to Test-6 (in seconds).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">The results of our machine learning-based alignment-free analyses using MLDSP-GUI support the hypothesis of a bat origin for COVID-19 and classify COVID-19 as sub-genus <em>Sarbecovirus</em>, within <em>Betacoronavirus</em>.</p></div><div class="section" id="sec-5"><h2 class="">Conclusion</h2><p id="p-50">This study provides an alignment-free method based on intrinsic genomic signatures that can deliver highly-accurate real-time taxonomic predictions of yet unclassified new sequences, <em>ab initio</em>, using raw DNA sequence data alone and without the need for gene or genome annotation. We use this method to provide evidence for the taxonomic classification of COVID-19 as <em>Sarbecovirus</em>, within <em>Betacoronavirus</em>, as well as quantitative evidence supporting a bat origin hypothesis. Our results are obtained through a comprehensive analysis of over 5000 unique viral sequences, through an alignment-free analysis of their two-dimensional genomic signatures, combined with a “decision tree” use of supervised machine learning and confirmed by Spearman’s rank correlation coefficient analyses. This study suggests that such alignment-free approaches to comparative genomics can be used to complement alignment-based approaches when timely taxonomic classification is of the essence, such as at critical periods during novel viral outbreaks.</p></div><div class="section" id="sec-7"><h2 class="">Supporting information</h2><p id="p-51">S1 Table. Spearman’s rank correlation coefficient (<em>ρ</em>) value for <em>k</em> =1 to <em>k</em> = 7.</p><p id="p-52">S2 Table. Accession IDs of the Sequences downloaded from the GISAID.</p><p id="p-53">S3 Table. Accession IDs of sequences used in Test-1 to Test-6.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">The authors are appreciative of the review of a manuscript draft by Hailie Pavanel.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">nCoV-2019 renamed to COVID-19; updated confirmed cases and deaths reported as of Feb 20; reformatted text for journal submission</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.03.932350v3.1"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Regenmortel</span>  <span class="cit-name-given-names">MV</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">CM</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">DHL</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">EB</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">MK</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Lemon</span>  <span class="cit-name-given-names">SM</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brian</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MMC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laude</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-chapter-title">Coronaviridae</span>. In: <span class="cit-ed"><span class="cit-name-surname">Regenmortel</span>  <span class="cit-name-given-names">MV</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fauquet</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">DHL</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-ed"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lemon</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-etal">et al.</span>, editors. <span class="cit-source">Virus Taxonomy. Seventh Report of the International Committee on Taxonomy of Viruses</span>, <span class="cit-publ-name">Academic Press</span>; <span class="cit-pub-date">2000</span>. pp. <span class="cit-fpage">835</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.2" data-doi="10.1128/MMBR.69.4.635-664.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navas-Martin</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Coronavirus Pathogenesis and the Emerging Pathogen Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol</abbr>. <span class="cit-pub-date">2005</span>; Rev. <span class="cit-vol">69</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DWeiss%26rft.auinit1%253DS.%2BR.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D635%26rft.epage%253D664%26rft.atitle%253DCoronavirus%2BPathogenesis%2Band%2Bthe%2BEmerging%2BPathogen%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.69.4.635-664.2005%26rft_id%253Dinfo%253Apmid%252F16339739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2OS80LzYzNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.3" data-doi="10.1016/j.tim.2016.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">ACK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Trends in Microbiology</abbr>. <span class="cit-pub-date">2016</span>; <span class="cit-vol">24</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BMicrobiology%26rft.volume%253D24%26rft.spage%253D490%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2016.03.003%26rft_id%253Dinfo%253Apmid%252F27012512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2016.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27012512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.4" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">ZL.</span></span> <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-vol">17</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">5192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicrobiology%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoeman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fielding</span>  <span class="cit-name-given-names">BC.</span></span> <span class="cit-article-title">Coronavirus envelope protein: Current knowledge</span>. <abbr class="cit-jnl-abbrev">Virology Journal</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.03.932350v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">KV</span></span>, <span class="cit-etal">et al.</span> <span class="cit-chapter-title">Family Coronaviridae</span>. In: <span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">AMQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefkowitz</span>  <span class="cit-name-given-names">EJ</span></span>, editors. <span class="cit-source">Virus taxonomy. Ninth report of the international committee on taxonomy of viruses</span>, <span class="cit-publ-name">Elsevier Academic Press</span>; <span class="cit-pub-date">2012</span>. pp. <span class="cit-fpage">806</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.7" data-doi="10.3181/0903-MR-94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY.</span></span> <span class="cit-article-title">Coronavirus diversity, phylogeny and interspecies jumping</span>. <abbr class="cit-jnl-abbrev">Experimental Biology and Medicine</abbr>. <span class="cit-pub-date">2009</span>; <span class="cit-vol">234</span>: <span class="cit-fpage">1117</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%2BY.%26rft.volume%253D234%26rft.issue%253D10%26rft.spage%253D1117%26rft.epage%253D1127%26rft.atitle%253DCoronavirus%2BDiversity%252C%2BPhylogeny%2Band%2BInterspecies%2BJumping%26rft_id%253Dinfo%253Adoi%252F10.3181%252F0903-MR-94%26rft_id%253Dinfo%253Apmid%252F19546349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3181/0903-MR-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.8" data-doi="10.1128/JVI.03273-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wertheim</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">DKW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM.</span></span> <span class="cit-article-title">A Case for the Ancient Origin of Coronaviruses</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2013</span>; <span class="cit-vol">87</span>: <span class="cit-fpage">7039</span>–<span class="cit-lpage">7045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03273-12%26rft_id%253Dinfo%253Apmid%252F23596293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMi83MDM5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.9" data-doi="10.1016/j.meegid.2019.03.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luk</span>  <span class="cit-name-given-names">HKH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY.</span></span> <span class="cit-article-title">Molecular epidemiology, evolution and phylogeny of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-vol">71</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D71%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2019.03.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2019.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.10" data-doi="10.1128/JVI.02605-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vijaykrishna</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">GJD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Evolutionary Insights into the Ecology of Coronaviruses</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2007</span>; <span class="cit-vol">81</span>: <span class="cit-fpage">4012</span>–<span class="cit-lpage">4020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02605-06%26rft_id%253Dinfo%253Apmid%252F17267506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzgvNDAxMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.11" data-doi="10.1128/JVI.01305-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shek</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Recent Transmission of a Novel Alphacoronavirus, Bat Coronavirus HKU10, from Leschenault’s Rousettes to Pomona Leaf-Nosed Bats: First Evidence of Interspecies Transmission of Coronavirus between Bats of Different Suborders</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2012</span>; <span class="cit-vol">86</span>: <span class="cit-fpage">11906</span>–<span class="cit-lpage">11918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01305-12%26rft_id%253Dinfo%253Apmid%252F22933277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMS8xMTkwNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.12" data-doi="10.1016/S0140-6736(20)30251-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30251-8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.13" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2005</span>; <span class="cit-vol">310</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yMC8yMDIwLjAyLjAzLjkzMjM1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.14" data-doi="10.1038/nrg2323"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shackelton</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC.</span></span> <span class="cit-article-title">Rates of evolutionary change in viruses: Patterns and determinants</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>. <span class="cit-pub-date">2008</span>; <span class="cit-vol">9</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDuffy%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D267%26rft.epage%253D276%26rft.atitle%253DRates%2Bof%2Bevolutionary%2Bchange%2Bin%2Bviruses%253A%2Bpatterns%2Band%2Bdeterminants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2323%26rft_id%253Dinfo%253Apmid%252F18319742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319742&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.15" data-doi="10.1007/s00239-001-0064-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">OG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC.</span></span> <span class="cit-article-title">Rates of molecular evolution in RNA viruses: A quantitative phylogenetic analysis</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. <span class="cit-pub-date">2002</span>; <span class="cit-vol">54</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DJenkins%26rft.auinit1%253DG.%2BM.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D156%26rft.epage%253D165%26rft.atitle%253DRates%2Bof%2Bmolecular%2Bevolution%2Bin%2BRNA%2Bviruses%253A%2Ba%2Bquantitative%2Bphylogenetic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0064-3%26rft_id%253Dinfo%253Apmid%252F11821909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0064-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11821909&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173522200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.16" data-doi="10.1006/viro.1997.8681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">AE.</span></span> <span class="cit-article-title">New insights into the mechanisms of RNA recombination</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1997</span>; <span class="cit-vol">235</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D235%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DNew%2Binsights%2Binto%2Bthe%2Bmechanisms%2Bof%2BRNA%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1997.8681%26rft_id%253Dinfo%253Apmid%252F9300032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1997.8681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9300032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XT89500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nathan</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgro</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palmenberg</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">SC.</span></span> <span class="cit-article-title">Generation of coronavirus spike deletion variants by high-frequency recombination at regions of predicted RNA secondary structure</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">1997</span>; <span class="cit-vol">71</span>: <span class="cit-fpage">6183</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DRowe%26rft.auinit1%253DC.%2BL.%26rft.volume%253D71%26rft.issue%253D8%26rft.spage%253D6183%26rft.epage%253D6190%26rft.atitle%253DGeneration%2Bof%2Bcoronavirus%2Bspike%2Bdeletion%2Bvariants%2Bby%2Bhigh-frequency%2Brecombination%2Bat%2Bregions%2Bof%2Bpredicted%2BRNA%2Bsecondary%2Bstructure%26rft_id%253Dinfo%253Apmid%252F9223514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzgvNjE4MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.02.03.932350v3.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-chapter-title">Coronaviridae: a review of coronaviruses and toroviruses</span>. In: <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolff</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">O</span></span>, editors. <span class="cit-source">Coronaviruses with Special Emphasis on First Insights Concerning SARS</span>. <span class="cit-publ-name">Birkhäuser-Verlag</span>, <span class="cit-pub-date">2005</span>; pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MMC.</span></span> <span class="cit-article-title">RNA recombination in animal and plant viruses</span>. <abbr class="cit-jnl-abbrev">Microbiological Reviews</abbr>. <span class="cit-pub-date">1992</span>; <span class="cit-vol">56</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D56%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D79%26rft.atitle%253DRNA%2Brecombination%2Bin%2Banimal%2Band%2Bplant%2Bviruses.%26rft_id%253Dinfo%253Apmid%252F1579113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI1Ni8xLzYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.20" data-doi="10.1099/vir.0.81611-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pasternak</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-article-title">Nidovirus transcription: How to make sense</span>…? <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr>. <span class="cit-pub-date">2006</span>; <span class="cit-vol">87</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPasternak%26rft.auinit1%253DA.%2BO.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D1403%26rft.epage%253D1421%26rft.atitle%253DNidovirus%2Btranscription%253A%2Bhow%2Bto%2Bmake%2Bsense...%253F%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.81611-0%26rft_id%253Dinfo%253Apmid%252F16690906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.81611-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16690906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238167400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.21" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiser</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brodt</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of a Novel Coronavirus in Patients with Severe Acute Respiratory Syndrome</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2003</span>; <span class="cit-vol">348</span>: <span class="cit-fpage">1967</span>–<span class="cit-lpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.22" data-doi="10.1056/NEJMoa030781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ksiazek</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erdman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaki</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peret</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus Associated with Severe Acute Respiratory Syndrome</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2003</span>; <span class="cit-vol">348</span>: <span class="cit-fpage">1953</span>–<span class="cit-lpage">1966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030781%26rft_id%253Dinfo%253Apmid%252F12690092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690092&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.23" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">ADME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RAM.</span></span> <span class="cit-article-title">Isolation of a Novel Coronavirus from a Man with Pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2012</span>; <span class="cit-vol">367</span>: <span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.24" data-doi="10.1126/science.1087139"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">YQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">ZX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of viruses related to the SARS coronavirus from animals in Southern China</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2003</span>; <span class="cit-vol">302</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1087139%26rft_id%253Dinfo%253Apmid%252F12958366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDIvNTY0My8yNzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yMC8yMDIwLjAyLjAzLjkzMjM1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alagaili</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briese</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sameroff</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Middle east respiratory syndrome coronavirus infection in dromedary camels in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">MBio</abbr>. <span class="cit-pub-date">2014</span>; <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.26" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Bo</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31978945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.27" data-doi="10.1002/jmv.25678"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stratton</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Outbreak of Pneumonia of Unknown Etiology in Wuhan China: the Mystery and the Miracle</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.25678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.25678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.28" data-doi="10.1016/j.ijid.2020.01.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">I Azhar</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madani</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ntoumi</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kock</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dar</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health — The latest 2019 novel coronavirus outbreak in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">International Journal of Infectious Diseases</abbr>. <span class="cit-pub-date">2020</span>; <span class="cit-vol">91</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BInfectious%2BDiseases%26rft.volume%253D91%26rft.spage%253D264%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijid.2020.01.009%26rft_id%253Dinfo%253Apmid%252F31953166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijid.2020.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31953166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.29" data-doi="10.1101/2020.01.25.919787"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Transmission dynamics of 2019 novel coronavirus (2019-nCoV)</span>. <span class="cit-source">BioRxiv [Preprint]</span>. 2020 bioRxiv 919787 [posted 2020 January 25; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.25.919787v1">https://www.biorxiv.org/content/10.1101/2020.01.25.919787v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.25.919787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.25.919787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNS45MTk3ODd2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.30" data-doi="10.1056/NEJMe2001126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Another Decade, Another Coronavirus</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMe2001126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMe2001126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMe2001126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menachery</span>  <span class="cit-name-given-names">VD.</span></span> <span class="cit-article-title">Return of the Coronavirus: 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2020</span>; <span class="cit-vol">12</span>: 135.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.32"><div class="cit-metadata"><cite><span class="cit-article-title">2019-nCoV Global Cases by Johns Hopkins CSSE</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">February</span> <span class="cit-day">6</span> [cited 6 February 2020]. In: <span class="cit-source">JHU CSSE website [Internet</span>]. Available from: <a href="https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6">https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.33"><div class="cit-metadata"><cite><span class="cit-article-title">Novel Coronavirus(2019-nCoV) Situation Report - 13</span>. <span class="cit-pub-date">2002</span> <span class="cit-month">February</span> <span class="cit-day">02</span> [cited 02 February 2020]. In: <span class="cit-source">WHO website [Internet</span>]. Available from: <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200202-sitrep-13-ncov-v3.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200202-sitrep-13-ncov-v3.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.34" data-doi="10.1016/S0140-6736(20)30154-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">JFW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kok</span>  <span class="cit-name-given-names">KH</span></span>, <span class="cit-auth"><span class="cit-name-surname">To</span>  <span class="cit-name-given-names">KKW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30154-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930154-9%26rft_id%253Dinfo%253Apmid%252F31986261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30154-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31986261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">XY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.36" data-doi="10.1101/2020.01.20.913368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">EWC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 913368 [posted 2020 January 22; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.20.913368v2">https://www.biorxiv.org/content/10.1101/2020.01.20.913368v2</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.20.913368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.20.913368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yMC45MTMzNjh2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.37" data-doi="10.1101/2020.01.21.914044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm</span>. <abbr class="cit-jnl-abbrev">BioRxiv [Preprint]</abbr>. <span class="cit-pub-date">2020</span> bioRxiv 914044 [posted 2020 January 22; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.21.914044v2">https://www.biorxiv.org/content/10.1101/2020.01.21.914044v2</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.21.914044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.21.914044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yMS45MTQwNDR2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.38" data-doi="10.1101/2020.01.24.919183"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">YM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 919183 [posted 2020 February 02; cited 2020 February 02]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.24.919183v2">https://www.biorxiv.org/content/10.1101/2020.01.24.919183v2</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.24.919183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.24.919183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNC45MTkxODN2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.39" data-doi="10.1101/2020.01.26.920249"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paraskevis</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kostaki</span>  <span class="cit-name-given-names">EG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magiorkinis</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panayiotakopoulos</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsiodras</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Full-genome evolutionary analysis of the novel corona virus (2019-nCoV) rejects the hypothesis of emergence as a result of a recent recombination event</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 920249 [posted 2020 January 27; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.26.920249v1">https://www.biorxiv.org/content/10.1101/2020.01.26.920249v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.26.920249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.26.920249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNi45MjAyNDl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.40" data-doi="10.1002/jmv.25682"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-article-title">Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost cross species transmission from snake to human</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.25682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.25682&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.41" data-doi="10.1101/2020.01.22.914952"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">XG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 914952 [posted 2020 January 23; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.22.914952v1">https://www.biorxiv.org/content/10.1101/2020.01.22.914952v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.22.914952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.22.914952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yMi45MTQ5NTJ2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.42" data-doi="10.1101/2020.01.22.915660"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letko</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Functional assessment of cell entry and receptor usage for lineage B <em>β</em>-coronaviruses, including 2019-nCoV</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 915660 [posted 2020 January 22; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1">https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.22.915660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.22.915660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yMi45MTU2NjB2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.43" data-doi="10.1101/2020.01.26.919985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCoV</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 919985 [posted 2020 January 26; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.26.919985v1">https://www.biorxiv.org/content/10.1101/2020.01.26.919985v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.26.919985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.26.919985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNi45MTk5ODV2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.44" data-doi="10.1101/2020.01.28.922922"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gui</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 922922 [posted 2020 January 30; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.28.922922v2">https://www.biorxiv.org/content/10.1101/2020.01.28.922922v2</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.28.922922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.28.922922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yOC45MjI5MjJ2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.45" data-doi="10.1101/2020.01.29.924100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">XJ.</span></span> <span class="cit-article-title">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 924100 [posted 2020 January 29; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.29.924100v1">https://www.biorxiv.org/content/10.1101/2020.01.29.924100v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.29.924100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.29.924100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yOS45MjQxMDB2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.46" data-doi="10.1093/bioinformatics/btg005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vinga</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Alignment-free sequence comparison-a review</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2003</span>; <span class="cit-vol">19</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DVinga%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D513%26rft.epage%253D523%26rft.atitle%253DAlignment-free%2Bsequence%2Bcomparison--a%2Breview%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg005%26rft_id%253Dinfo%253Apmid%252F12611807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12611807&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181410700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.47" data-doi="10.1186/s13059-017-1319-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zielezinski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vinga</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlowski</span>  <span class="cit-name-given-names">WM.</span></span> <span class="cit-article-title">Alignment-free sequence comparison: benefits, applications, and tools</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-pub-date">2017</span>, <span class="cit-vol">18</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D18%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1319-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1319-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.48" data-doi="10.1371/journal.pone.0119815"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayem</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karamichalis</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryans</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dattani</span>  <span class="cit-name-given-names">NS.</span></span> <span class="cit-article-title">Mapping the space of genomic signatures</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-vol">10</span>: <span class="cit-fpage">e0119815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D10%26rft.spage%253De0119815%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0119815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0119815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karamichalis</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konstantinidis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kopecki</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">An investigation into inter- and intragenomic variations of graphic genomic signatures</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-vol">16</span>: <span class="cit-fpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Solis-Reyes</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avino</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">An open-source k-mer based machine learning tool for fast and accurate subtyping of HIV-1 genomes</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">13</span>: <span class="cit-fpage">e0206409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D13%26rft.spage%253De0206409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Randhawa</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">KH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">ML-DSP: Machine Learning with Digital Signal Processing for ultrafast, accurate, and scalable genome classification at all taxonomic levels</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-vol">20</span>: <span class="cit-fpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D20%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Randhawa</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">KH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">MLDSP-GUI: an alignment-free standalone tool with an interactive graphical user interface for DNA sequence comparison and analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-fpage">btz918</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.53" data-doi="10.1093/nar/18.8.2163"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jeffre</span>  <span class="cit-name-given-names">HJ.</span></span> <span class="cit-article-title">Chaos game representation of gene structure</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">1990</span>; <span class="cit-vol">18</span>: <span class="cit-fpage">2163</span>–<span class="cit-lpage">2170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DJeffrey%26rft.auinit1%253DH.J.%26rft.volume%253D18%26rft.issue%253D8%26rft.spage%253D2163%26rft.epage%253D2170%26rft.atitle%253DChaos%2Bgame%2Brepresentation%2Bof%2Bgene%2Bstructure%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.8.2163%26rft_id%253Dinfo%253Apmid%252F2336393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.8.2163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2336393&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DC12800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Asuero</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayago</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">AG.</span></span> <span class="cit-article-title">The correlation coefficient: an overview</span>. <abbr class="cit-jnl-abbrev">Crit Rev Anal Chem</abbr>. <span class="cit-pub-date">2006</span>; <span class="cit-vol">36</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">41</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit%2BRev%2BAnal%2BChem%26rft.volume%253D36%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karamichalis</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kari</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">MoDMaps3D: an interactive webtool for the quantification and 3D visualization of interrelationships in a dataset of DNA sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-vol">33</span>(<span class="cit-issue">19</span>): <span class="cit-fpage">3091</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.56" data-doi="10.1007/BF02289565"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kruskal</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis</span>. <abbr class="cit-jnl-abbrev">Psychometrika</abbr>. <span class="cit-pub-date">1964</span>; <span class="cit-vol">29</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D29%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02289565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02289565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1964CBT8100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.57" data-doi="10.1093/nar/gkz033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Requião</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossetto</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domitrovic</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palhano</span>  <span class="cit-name-given-names">FL.</span></span> <span class="cit-article-title">Codon stabilization coefficient as a metric to gain insights into mRNA stability and codon bias and their relationships with translation</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-vol">47</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2216</span>–<span class="cit-lpage">2228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D47%26rft.spage%253D2216%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">58.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.58" data-doi="10.1177/1176934318761368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karumathil</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raveendran</span>  <span class="cit-name-given-names">NT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ganesh</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">NS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dirisala</span>  <span class="cit-name-given-names">VR.</span></span> <span class="cit-article-title">Evolution of Synonymous Codon Usage Bias in West African and Central African Strains of Monkeypox Virus</span>. <abbr class="cit-jnl-abbrev">Evolutionary Bioinformatics Online</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">14</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1177/1176934318761368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BBioinformatics%2BOnline%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1176934318761368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/1176934318761368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vinogradov</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anatskaya</span>  <span class="cit-name-given-names">OV.</span></span> <span class="cit-article-title">DNA helix: the importance of being AT-rich</span>. <abbr class="cit-jnl-abbrev">Mammalian Genome</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-vol">9</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">455</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2BGenome%26rft.volume%253D9%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.03.932350v3.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hollander</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chicken</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-chapter-title">Nonparametric statistical methods</span>, <span class="cit-edition">3rd</span> Edition, <span class="cit-publ-name">John Wiley &amp; Sons</span>; <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musa</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Preliminary estimation of the basic reproduction number of novel coronavirus (2019-nCoV) in China, from 2019 to 2020: A data-driven analysis in the early phase of the outbreak</span>. <abbr class="cit-jnl-abbrev">International Journal of Infectious Diseases</abbr> [In Press] [received 2020 January 23; revised 2020 January 27; accepted 2020 January 27; cited 2020 February 1], <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">62.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.02.03.932350v3.62" data-doi="10.1101/2020.01.28.923169"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Beware of asymptomatic transmission: Study on 2019-nCoV prevention and control measures based on extended SEIR model</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 923169 [posted 2020 January 28; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.28.923169v1">https://www.biorxiv.org/content/10.1101/2020.01.28.923169v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.28.923169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.28.923169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yOC45MjMxNjl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.63" data-doi="10.1101/2020.01.24.919241"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 919241 [posted 2020 January 27; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.24.919241v3">https://www.biorxiv.org/content/10.1101/2020.01.24.919241v3</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.24.919241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.24.919241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNC45MTkyNDF2NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tachedjian</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tachedjian</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Identification of diverse full-length endogenous betaretroviruses in megabats and microbats</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr>. <span class="cit-pub-date">2013</span>; <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">65.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tachedjian</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">LF.</span></span> <span class="cit-article-title">Bats and Rodents Shape Mammalian Retroviral Phylogeny</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.66" data-doi="10.1093/molbev/msy048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tachedjian</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">AZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential evolution of antiretroviral restriction factors in pteropid bats as revealed by APOBEC3 gene complexity</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">35</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D35%26rft.spage%253D1626%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Global Epidemiology of Bat Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2019</span>; <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.68" data-doi="10.1128/JVI.02582-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and Characterization of a Novel Bat Coronavirus Closely Related to the Direct Progenitor of Severe Acute Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2016</span>; <span class="cit-vol">90</span>: <span class="cit-fpage">3253</span>–<span class="cit-lpage">3256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02582-15%26rft_id%253Dinfo%253Apmid%252F26719272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzYvMzI1MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.69" data-doi="10.1128/JVI.01055-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic Characterization of Betacoronavirus Lineage C Viruses in Bats Reveals Marked Sequence Divergence in the Spike Protein of Pipistrellus Bat Coronavirus HKU5 in Japanese Pipistrelle: Implications for the Origin of the Novel Middle East Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2013</span>; <span class="cit-vol">87</span>: <span class="cit-fpage">8638</span>–<span class="cit-lpage">8650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01055-13%26rft_id%253Dinfo%253Apmid%252F23720729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xNS84NjM4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duong</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hul</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">San</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davun</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omaliss</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic diversity of coronaviruses in bats in Lao PDR and Cambodia</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-vol">48</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol%26rft.volume%253D48%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">71.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.71" data-doi="10.1128/JVI.00650-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drexler</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gloza-Rausch</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glende</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muth</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goettsche</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic Characterization of Severe Acute Respiratory Syndrome-Related Coronavirus in European Bats and Classification of Coronaviruses Based on Partial RNA-Dependent RNA Polymerase Gene Sequences</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2010</span>; <span class="cit-vol">84</span>: <span class="cit-fpage">11336</span>–<span class="cit-lpage">11349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00650-10%26rft_id%253Dinfo%253Apmid%252F20686038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NC8yMS8xMTMzNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.72" data-doi="10.1007/s00705-010-0612-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rihtarič</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hostnik</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steyer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grom</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toplak</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Identification of SARS-like coronaviruses in horseshoe bats (Rhinolophus hipposideros) in Slovenia</span>. <abbr class="cit-jnl-abbrev">Arch. Virol</abbr>. <span class="cit-pub-date">2010</span>; <span class="cit-vol">155</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DRihtaric%26rft.auinit1%253DD.%26rft.volume%253D155%26rft.issue%253D4%26rft.spage%253D507%26rft.epage%253D514%26rft.atitle%253DIdentification%2Bof%2BSARS-like%2Bcoronaviruses%2Bin%2Bhorseshoe%2Bbats%2B%2528Rhinolophus%2Bhipposideros%2529%2Bin%2BSlovenia.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-010-0612-5%26rft_id%253Dinfo%253Apmid%252F20217155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-010-0612-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20217155&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.73" data-doi="10.1128/JVI.00631-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">Feng</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of Diverse Alphacoronaviruses and Genomic Characterization of a Novel Severe Acute Respiratory Syndrome-Like Coronavirus from Bats in China</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2014</span>; <span class="cit-vol">88</span>: <span class="cit-fpage">7070</span>–<span class="cit-lpage">7082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00631-14%26rft_id%253Dinfo%253Apmid%252F24719429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xMi83MDcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">74.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.74" data-doi="10.1186/s12985-014-0235-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wacharapluesadee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duengkae</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodpan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaewpom</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maneeorn</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanchanasaka</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Diversity of coronavirus in bats from Eastern Thailand Emerging viruses</span>. <abbr class="cit-jnl-abbrev">Virol. J</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-vol">12</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol.%2BJ%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-014-0235-7%26rft_id%253Dinfo%253Apmid%252F25591713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12985-014-0235-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25591713&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.75" data-doi="10.3201/eid1503.081013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrardy</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruone</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Detection of novel SARS-like and other coronaviruses in bats from Kenya</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis</abbr>. <span class="cit-pub-date">2009</span>; <span class="cit-vol">15</span>: <span class="cit-fpage">482</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DTong%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D482%26rft.epage%253D485%26rft.atitle%253DDetection%2Bof%2Bnovel%2BSARS-like%2Band%2Bother%2Bcoronaviruses%2Bin%2Bbats%2Bfrom%2BKenya.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1503.081013%26rft_id%253Dinfo%253Apmid%252F19239771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1503.081013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19239771&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263759400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.03.932350v3.76" data-doi="10.1073/pnas.0506735102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsoi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">BHL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-chapter-title">Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</span>. <span class="cit-source">Proc. Natl. Acad. Sci</span>. <span class="cit-pub-date">2005</span>; <span class="cit-publ-name">National Academy of Sciences</span>, <span class="cit-publ-loc">U. S. A</span>., <span class="cit-vol">102</span>: <span class="cit-fpage">14040</span>–<span class="cit-lpage">14045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506735102%26rft_id%253Dinfo%253Apmid%252F16169905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM5LzE0MDQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.77"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Virologists weigh in on novel coronavirus in China’s outbreak</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">January</span> <span class="cit-day">08</span> [cited 31 January 2020]. In: <span class="cit-source">University of Minnesota [Internet]</span>. Available from: <a href="http://www.cidrap.umn.edu/news-perspective/2020/01/virologists-weigh-novel-coronavirus-chinas-outbreak">http://www.cidrap.umn.edu/news-perspective/2020/01/virologists-weigh-novel-coronavirus-chinas-outbreak</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.78"><div class="cit-metadata"><cite><span class="cit-article-title">nCoV’s relationship to bat coronaviruses &amp; recombination signals (no snakes) - no evidence the 2019-nCoV lineage is recombinant</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">January</span> <span class="cit-day">31</span> [cited 31 January 2020]. In: <span class="cit-source">Virological blog [Internet]</span>. Available from: <a href="http://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes-no-evidence-the-2019-nCoV-lineage-is-recombinant/331">http://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes-no-evidence-the-2019-nCoV-lineage-is-recombinant/331</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.79"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Experts: nCoV spread in China’s cities could trigger global epidemic</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">January</span> <span class="cit-day">27</span> [cited 31 January 2020]. In: <span class="cit-source">University of Minnesota [Internet]</span>. Available from: <a href="http://www.cidrap.umn.edu/news-perspective/2020/01/experts-ncov-spread-chinas-cities-could-trigger-global-epidemic">http://www.cidrap.umn.edu/news-perspective/2020/01/experts-ncov-spread-chinas-cities-could-trigger-global-epidemic</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.80"><div class="cit-metadata"><cite><span class="cit-article-title">China detects large quantity of novel coronavirus at Wuhan seafood market</span>. <span class="cit-pub-date">2020</span> <span class="cit-month">January</span> <span class="cit-day">27</span> [cited 31 January 2020]. In: <span class="cit-source">Xinhuanet News [Internet]</span>. Available from: <a href="http://www.xinhuanet.com/english/2020-01/27/c_138735677.htm">http://www.xinhuanet.com/english/2020-01/27/c_138735677.htm</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.81" data-doi="10.1038/nature24039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takata</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonçalves-Carneiro</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco-Melo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD.</span></span> <span class="cit-article-title">CG dinucleotide suppression enables antiviral defence targeting non-self RNA</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2017</span>; <span class="cit-vol">550</span>(<span class="cit-issue">7674</span>): <span class="cit-fpage">124</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D550%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24039%26rft_id%253Dinfo%253Apmid%252F28953888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28953888&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.82" data-doi="10.1371/journal.ppat.1000079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhanot</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabadan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Patterns of evolution and host gene mimicry in influenza and other RNA viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathogens</abbr>. <span class="cit-pub-date">2008</span>; <span class="cit-vol">4</span>(<span class="cit-issue">6</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1000079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DGreenbaum%26rft.auinit1%253DB.%2BD.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De1000079%26rft.epage%253De1000079%26rft.atitle%253DPatterns%2Bof%2Bevolution%2Band%2Bhost%2Bgene%2Bmimicry%2Bin%2Binfluenza%2Band%2Bother%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000079%26rft_id%253Dinfo%253Apmid%252F18535658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18535658&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.83" data-doi="10.1371/journal.pone.0006282"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lobo</span>  <span class="cit-name-given-names">FP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mota</span>  <span class="cit-name-given-names">BEF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pena</span>  <span class="cit-name-given-names">SDJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azevedo</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macedo</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tauch</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Virus-host coevolution: Common patterns of nucleotide motif usage in Flaviviridae and their hosts</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-pub-date">2009</span>; <span class="cit-vol">4</span>(<span class="cit-issue">7</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0006282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLobo%26rft.auinit1%253DF.%2BP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De6282%26rft.epage%253De6282%26rft.atitle%253DVirus-host%2Bcoevolution%253A%2Bcommon%2Bpatterns%2Bof%2Bnucleotide%2Bmotif%2Busage%2Bin%2BFlaviviridae%2Band%2Btheir%2Bhosts.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006282%26rft_id%253Dinfo%253Apmid%252F19617912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19617912&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kindler</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">To sense or not to sense viral RNA-essentials of coronavirus innate immune evasion</span>. <abbr class="cit-jnl-abbrev">Current Opinion in Microbiology</abbr>. <span class="cit-pub-date">2014</span>; <span class="cit-vol">20</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BMicrobiology%26rft.volume%253D20%26rft.spage%253D68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.85" data-doi="10.1038/s41598-018-24448-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milewska</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindler</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vkovski</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">APOBEC3-mediated restriction of RNA virus replication</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">8</span>(<span class="cit-issue">1</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-24448-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-24448-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-24448-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.86" data-doi="10.1126/science.1100658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">KN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheehy</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malim</span>  <span class="cit-name-given-names">MH.</span></span> <span class="cit-article-title">APOBEC-mediated editing of viral RNA</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2004</span>; <span class="cit-vol">305</span>(<span class="cit-issue">5684</span>): <span class="cit-fpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBishop%26rft.auinit1%253DK.%2BN.%26rft.volume%253D305%26rft.issue%253D5684%26rft.spage%253D645%26rft.epage%253D645%26rft.atitle%253DAPOBEC-Mediated%2BEditing%2Bof%2BViral%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1100658%26rft_id%253Dinfo%253Apmid%252F15286366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDUvNTY4NC82NDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yMC8yMDIwLjAyLjAzLjkzMjM1MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pyrc</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jebbink</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berkhout</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van der Hoek</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Genome structure and transcriptional regulation of human coronavirus NL63</span>. <abbr class="cit-jnl-abbrev">Virology Journal</abbr>. <span class="cit-pub-date">2004</span>; <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%2BJournal%26rft.volume%253D1%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.88" data-doi="10.1016/j.virusres.2014.11.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berkhout</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Hemert</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">On the biased nucleotide composition of the human coronavirus RNA genome</span>. <abbr class="cit-jnl-abbrev">Virus Research</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-vol">202</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft.volume%253D202%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">89.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.89" data-doi="10.3181/0903-MR-94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY.</span></span> <span class="cit-auth"><span class="cit-name-surname">Coronavirus</span>  <span class="cit-name-given-names">diversity</span></span>, <span class="cit-article-title">phylogeny and interspecies jumping</span>. <abbr class="cit-jnl-abbrev">Experimental Biology and Medicine</abbr>. <span class="cit-pub-date">2009</span>; <span class="cit-vol">234</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1117</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%2BY.%26rft.volume%253D234%26rft.issue%253D10%26rft.spage%253D1117%26rft.epage%253D1127%26rft.atitle%253DCoronavirus%2BDiversity%252C%2BPhylogeny%2Band%2BInterspecies%2BJumping%26rft_id%253Dinfo%253Adoi%252F10.3181%252F0903-MR-94%26rft_id%253Dinfo%253Apmid%252F19546349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3181/0903-MR-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19546349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.90" data-doi="10.3390/v2081803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">PCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY.</span></span> <span class="cit-article-title">Coronavirus Genomics and Bioinformatics Analysis</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2010</span>; <span class="cit-vol">2</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1804</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253D1804%26rft.epage%253D1820%26rft.atitle%253DCoronavirus%2Bgenomics%2Band%2Bbioinformatics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv2081803%26rft_id%253Dinfo%253Apmid%252F21994708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v2081803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.03.932350v3.91" data-doi="10.1128/JVI.02114-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-pub-date">2008</span>; <span class="cit-vol">82</span>: <span class="cit-fpage">2515</span>–<span class="cit-lpage">2527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02114-07%26rft_id%253Dinfo%253Apmid%252F18094151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzUvMjUxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">92.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.92" data-doi="10.1126/science.1085658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wadhwani</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesters</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Coronavirus main proteinase (3CLpro) Structure: Basis for design of anti-SARS drugs</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2003</span>; <span class="cit-vol">300</span>: <span class="cit-fpage">1763</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085658%26rft_id%253Dinfo%253Apmid%252F12746549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzYzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjAvMjAyMC4wMi4wMy45MzIzNTAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">93.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.03.932350v3.93"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nukoolkarn</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">VS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malaisree</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aruksakulwong</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hannongbua</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Molecular dynamic simulations analysis of ritronavir and lopinavir as SARS-CoV 3CLpro inhibitors</span>. <abbr class="cit-jnl-abbrev">J. Theor. Biol</abbr>. <span class="cit-pub-date">2008</span>; <span class="cit-vol">254</span>: <span class="cit-fpage">861</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DNukoolkarn%26rft.auinit1%253DV.%26rft.volume%253D254%26rft.issue%253D4%26rft.spage%253D861%26rft.epage%253D867%26rft.atitle%253DMolecular%2Bdynamic%2Bsimulations%2Banalysis%2Bof%2Britronavir%2Band%2Blopinavir%2Bas%2BSARS-CoV%2B3CL%2528pro%2529%2Binhibitors.%26rft_id%253Dinfo%253Apmid%252F18706430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18706430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F20%2F2020.02.03.932350.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.03.932350v3.94" data-doi="10.1101/2020.01.27.921627"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Nelfinavir was predicted to be a potential inhibitor of 2019-nCoV main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation</span>. <span class="cit-source">BioRxiv [Preprint]</span>. <span class="cit-pub-date">2020</span> bioRxiv 921627 [posted 2020 January 28; cited 2020 January 31]. Available from: <a href="https://www.biorxiv.org/content/10.1101/2020.01.27.921627v1">https://www.biorxiv.org/content/10.1101/2020.01.27.921627v1</a><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.27.921627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioRxiv%2B%255BPreprint%255D%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.27.921627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNy45MjE2Mjd2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzIwLzIwMjAuMDIuMDMuOTMyMzUwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.03.932350v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.07.897991v2" title="A validation framework for neuroimaging software: the case of population receptive fields" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.01.15.894592v2" title="IRESbase: a Comprehensive Database of Experimentally Validated Internal Ribosome Entry Sites" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 20, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.03.932350v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.03.932350v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.03.932350v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.03.932350v3" class="active" data-icon-position="" data-hide-link-title="0">Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1160969" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-It64u7U7t4o5D-UCPTxZ8k_uOTHcw5rOMl2nx3INVaU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1160969" id="node1160969" data-pisa="biorxiv;2020.02.03.932350v3" data-pisa-master="biorxiv;2020.02.03.932350" data-seqnum="1160969" data-apath="/biorxiv/early/2020/02/20/2020.02.03.932350.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gurjit S.</span> <span class="nlm-surname">Randhawa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Maximillian P.M.</span> <span class="nlm-surname">Soltysiak</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hadi El</span> <span class="nlm-surname">Roz</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Camila P.E.</span> <span class="nlm-surname">de Souza</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Hill</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lila</span> <span class="nlm-surname">Kari</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.03.932350; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.03.932350 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;title=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;title=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;text=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;title=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;t=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;title=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;title=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1160969" id="node1160969--2" data-pisa="biorxiv;2020.02.03.932350v3" data-pisa-master="biorxiv;2020.02.03.932350" data-seqnum="1160969" data-apath="/biorxiv/early/2020/02/20/2020.02.03.932350.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gurjit S.</span> <span class="nlm-surname">Randhawa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Maximillian P.M.</span> <span class="nlm-surname">Soltysiak</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hadi El</span> <span class="nlm-surname">Roz</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Camila P.E.</span> <span class="nlm-surname">de Souza</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Kathleen A.</span> <span class="nlm-surname">Hill</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lila</span> <span class="nlm-surname">Kari</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.03.932350; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.03.932350 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Randhawa%2C+Gurjit+S.&amp;rft.au=Soltysiak%2C+Maximillian+P.M.&amp;rft.au=Roz%2C+Hadi+El&amp;rft.au=de+Souza%2C+Camila+P.E.&amp;rft.au=Hill%2C+Kathleen+A.&amp;rft.au=Kari%2C+Lila"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1160969/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1160969/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1160969/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1160969/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1160969/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1160969/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1160969/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1160969/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1160969/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1160969/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1160969/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1160969/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;count=horizontal&amp;via=&amp;text=Machine%20learning%20using%20intrinsic%20genomic%20signatures%20for%20rapid%20classification%20of%20novel%20pathogens%3A%20COVID-19%20case%20study&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.03.932350v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.03.932350v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
