<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5589256">5589256</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1006528">10.1371/journal.ppat.1006528</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-17-00913</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Opinion</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Genomics</span>: <span class="subject" title="subject">Mammalian Genomics</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genome Annotation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genome Annotation</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Zika Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Zika Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Zika Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Flaviviruses</span>: <span class="subject" title="subject">Zika Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Motif Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Databases</span>: <span class="subject" title="subject">Genomic Databases</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Databases</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Databases</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span>: <span class="subject" title="subject">Comparative Genomics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Comparative Genomics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Genomics</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Genomics</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span>: <span class="subject" title="subject">Genome Evolution</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span>: <span class="subject" title="subject">Genome Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Evolution</span>: <span class="subject" title="subject">Genome Evolution</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Zika genomics urgently need standardized and curated reference sequences</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-0242-7000</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theys</span><span class="given-names" tagx="given-names" title="given-names">Kristof</span></span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3906-758X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libin</span><span class="given-names" tagx="given-names" title="given-names">Pieter</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-8117-9166</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dallmeier</span><span class="given-names" tagx="given-names" title="given-names">Kai</span></span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1937-0506</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pineda-Peña</span><span class="given-names" tagx="given-names" title="given-names">Andrea-Clemencia</span></span><a href="#aff004"><sup>4</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-6594-2766</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandamme</span><span class="given-names" tagx="given-names" title="given-names">Anne-Mieke</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-9433-8752</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuypers</span><span class="given-names" tagx="given-names" title="given-names">Lize</span></span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">Ana B.</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff005"><sup>5</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">KU Leuven – University of Leuven, Department of Microbiology and Immunology, Rega Institute for Medical Research, Clinical and Epidemiological Virology, Leuven, Belgium</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Artificial Intelligence Lab, Department of computer science, Vrije Universiteit Brussel, Brussels, Belgium</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">KU Leuven – University of Leuven, Department of Microbiology and Immunology, Rega Institute for Medical Research, Laboratory of Virology and Chemotherapy, Leuven, Belgium</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Molecular Biology and Immunology Department, Fundación Instituto de Immunología de Colombia (FIDIC), Basic Sciences Department, Universidad del Rosario, Bogotá, Colombia</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Global Health and Tropical Medicine, GHTM, Institute for Hygiene and Tropical Medicine, IHMT, University Nova de Lisboa, UNL, Lisbon, Portugal</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coyne</span><span class="given-names" tagx="given-names" title="given-names">Carolyn B</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Pittsburgh, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">kristof.theys@kuleuven.be</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017-9</span></span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006528</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Theys et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1006528.pdf" title="self-uri">ppat.1006528.pdf</a><div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
KT is funded by a postdoctoral grant of the Fonds Wetenschappelijk Onderzoek – Vlaanderen (FWO). PL is funded by a doctoral grant of the Research Foundation – Flanders (FWO). This work has been funded by FCT through funds to GHTM-UID/Multi/04413/2013. The VIROGENESIS project receives funding from the European Union’s Horizon 2020 research and innovation program under grant agreement No 634650. This research was supported in part by the European Union’s Horizon 2020 research and innovation programs ZikaPLAN (Agreement No 734584) and ZIKAlliance (Agreement No 734548). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <p>Emerging viral epidemics pose a threat to public health globally. The recent spread of the Zika virus (ZIKV) across the Pacific region and the Americas is particularly disturbing, given its association with severe birth malformations. On February 1, 2016, the World Health Organization (WHO) declared ZIKV a Public Health Emergency of International Concern, which prompted a global response to improve our understanding of ZIKV epidemiology and disease manifestations. Intensified ZIKV surveillance and real-time sharing of novel scientific evidence coincided with a scaling up of ZIKV whole genome sequencing for public health purposes and diagnostics [<a href="#ppat.1006528.ref001">1</a>,<a href="#ppat.1006528.ref002">2</a>]. As a result, most human ZIKV genomes available to date originate from the ongoing outbreak. While this proliferation of genomic data offers new opportunities for comparative and evolutionary genomics of ZIKV, we demonstrate that the rapid advance in ZIKV genomics has resulted in inconsistencies that complicate the interpretation, reproducibility, and comparison of findings from and across studies, particularly due to the lack of consensus on a standardized and representative reference genome annotation. ZIKV reference genomes do not match virus strains sampled from the global epidemic or are not adequately annotated at the protein level, and current heterogeneity in study methodology ultimately limits the full potential of ZIKV genomics. In this letter, we address the need for curation and standardized annotation of ZIKV reference genomes in order to guide researchers and clinicians in genomic analyses and the translation of research findings.</p> <p>Reference genomes are used as templates for a wide range of sequence-based applications and should be carefully selected to represent pathogen diversity from a temporal, geographical, and epidemiological perspective. Kuno and Chang (2007) reported the first ZIKV genome with a detailed description and annotation of the coding sequence (CDS) (Genbank accession number [GAN] AY632535). This sequence was based on the MR-766 strain that was isolated in 1947 from a sentinel rhesus monkey at the Zika forest in Uganda [<a href="#ppat.1006528.ref003">3</a>]. The National Center for Biotechnology Information (NCBI) released their reference sequence for ZIKV in 2016, using the AY632535 sequence to guide the annotation of the ZIKV reference genome (GAN NC_012532) [<a href="#ppat.1006528.ref004">4</a>]. With a CDS length of 3,419 amino acids (aa), AY632535 and NC_012532 lack a 4-aa N-glycosylation motif (VNDT) in the envelope region that is present in all human ZIKV genomes sampled during the recent global epidemic, including the candidate reference strain (GAN KX369547) proposed in 2016 for quantification of ZIKV RNA on behalf of the WHO [<a href="#ppat.1006528.ref001">1</a>] (<a href="#ppat.1006528.g001">Fig 1</a>). This loss has been attributed to an artifact of repeated serial passages or adaptation to a mosquito vector, while in vivo restoration was observed upon replication in macaques, and the motif seems to play a role for viral particle structure and transmission [<a href="#ppat.1006528.ref005">5</a>,<a href="#ppat.1006528.ref006">6</a>,<a href="#ppat.1006528.ref007">7</a>,<a href="#ppat.1006528.ref008">8</a>]. The origin and biological role of the deletion is not yet resolved, and site alterations can be seen in both historical and contemporary ZIKV genomes, with 4-aa or 6-aa deletions and a T-to-I substitution, even between different MR-766-derived sequences [<a href="#ppat.1006528.ref005">5</a>,<a href="#ppat.1006528.ref007">7</a>,<a href="#ppat.1006528.ref008">8</a>]. A direct implication of this discrepancy in genome lengths is that the motif is ignored in downstream analyses and that comparative studies report discordant genomic coordinates for a similar observation. The ZIKV reference genome must include the 4-aa motif and be representative for the virus population circulating in the human population. As such, past and future genomic analyses should be updated to harmonize findings. While the WHO reference genome meets these criteria, its sequence (10,769 nt in length) does not cover the full-length ZIKV genome (10,807 nt in length) due to partial 5’ and 3’ untranslated regions (UTRs) (<a href="#ppat.1006528.g001">Fig 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1006528.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Visual representation of the correct genome annotation for the proposed ZIKV reference sequence KJ776791 [<a href="#ppat.1006528.ref010">10</a>].</div> <p>The figure shows the corrected nucleotide positions for the Genbank annotation of KJ776791. This correction was guided by the curated annotation of NC_012532 [<a href="#ppat.1006528.ref004">4</a>]. The upper axis indicates nucleotide positions of proteins in the full-length genome (10,807 nt), including the 5’UTR (length of 107 nt) and 3’UTR (length of 428 nt). The lower axis indicates nucleotide positions of encoded proteins in the CDS (amino acids: 1–3,423; nucleotides: 1–10,269). These positions also apply as the correct annotation of the WHO candidate reference genome KX369547. While the reference sequence we propose encompasses the complete UTR, we note that the lengths of the 5’UTR and 3’UTR of the WHO candidate reference sequence KX369547 are limited to 90 nt and 406 nt, respectively.</p>   </div> <p>More importantly, the reference genome of choice needs to be well annotated to allow consistent identification of genomic regions when comparing viral strains. NCBI staff has manually curated the ZIKV genome sequence reported by Kuno and Chang [<a href="#ppat.1006528.ref003">3</a>,<a href="#ppat.1006528.ref004">4</a>] using the predicted protease cleavage site information from the original study. The value of this validation effort is significant because it corrected the protein lengths reported by Kuno and Chang (<a href="#ppat.1006528.t001">Table 1</a>). However, a similar curation by NCBI of ZIKV genomes that contain the 4-aa motif is lacking, and this results in considerable heterogeneity in reported annotation of ZIKV peptides that can be observed in recent literature and public databases. Firstly, MR-766 genomes with a CDS length of 3,423 aa have been submitted to Genbank without a complete and correct annotation, reducing their value as a reference sequence. For example, the nonhuman GAN KX377335 strain has served as guiding sequence in recent molecular epidemiology studies, but genomic sequences originating from these studies (e.g., GAN KU940224) were submitted to Genbank with a protein annotation that differs significantly from the curated NCBI reference genome [<a href="#ppat.1006528.ref002">2</a>,<a href="#ppat.1006528.ref009">9</a>] (<a href="#ppat.1006528.t001">Table 1</a>). Secondly, the WHO candidate reference strain KX369547 has a complete protein annotation in Genbank, but protein lengths differ markedly from the NCBI reference genome (<a href="#ppat.1006528.t001">Table 1</a>), which affects its current use as reference. Strikingly, protein annotations of ZIKV genomes submitted by studies using KX377335 as the reference sequence propagate the incorrect protein annotations of the human candidate WHO strain (<a href="#ppat.1006528.t001">Table 1</a>). Thirdly, studies have reported ZIKV protein lengths of human strains (GAN KJ776791) that are inconsistent with any of the annotations discussed above, causing more confusion with respect to the genome structure of ZIKV [<a href="#ppat.1006528.ref010">10</a>]. Fourthly, ambiguous UTR lengths are found in ZIKV genomes available in Genbank, without being labeled as partial UTR.</p> <div class="table-wrap_UNKNOWN" id="ppat.1006528.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.ppat.1006528.t001">10.1371/journal.ppat.1006528.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
High level of heterogeneity in reported peptide lengths of ZIKV from genome annotations available in Genbank or protein information reported in original publications.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1006528.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <tbody> <tr> <td align="justify" rowspan="1" colspan="1"><b>Genbank accession number</b></td> <td align="justify" rowspan="1" colspan="1">AY632535 [<a href="#ppat.1006528.ref003">3</a>]</td> <td align="justify" rowspan="1" colspan="1">NC_012532 [<a href="#ppat.1006528.ref004">4</a>]</td> <td align="justify" rowspan="1" colspan="1">KX377335 [<a href="#ppat.1006528.ref009">9</a>]</td> <td align="left" rowspan="1" colspan="1">KU940224 [<a href="#ppat.1006528.ref002">2</a>]</td> <td align="justify" rowspan="1" colspan="1">KX369547 [<a href="#ppat.1006528.ref001">1</a>]</td> <td align="justify" rowspan="1" colspan="1">KJ776791 [<a href="#ppat.1006528.ref010">10</a>]</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>Genbank annotation available</b></td> <td align="justify" rowspan="1" colspan="1">No</td> <td align="justify" rowspan="1" colspan="1">Yes</td> <td align="justify" rowspan="1" colspan="1">No</td> <td align="justify" rowspan="1" colspan="1">Yes</td> <td align="justify" rowspan="1" colspan="1">Yes</td> <td align="justify" rowspan="1" colspan="1">Yes</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>Source of protein information</b></td> <td align="justify" rowspan="1" colspan="1">Original publication</td> <td align="justify" rowspan="1" colspan="1">Genbank</td> <td align="justify" rowspan="1" colspan="1">Original publication</td> <td align="justify" rowspan="1" colspan="1">Genbank</td> <td align="justify" rowspan="1" colspan="1">Genbank</td> <td align="justify" rowspan="1" colspan="1">Original publication</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>Additional information</b></td> <td align="justify" rowspan="1" colspan="1">Sequence derived from MR-766 strain</td> <td align="justify" rowspan="1" colspan="1">Curated AY632535 by NCBI</td> <td align="justify" rowspan="1" colspan="1">Sequence derived from MR-766 strain</td> <td align="justify" rowspan="1" colspan="1">Sequence aligned with KX377335</td> <td align="justify" rowspan="1" colspan="1">WHO reference strain</td> <td align="justify" rowspan="1" colspan="1">Annotation differs from publication (published pre-2015)</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>Source</b></td> <td align="justify" rowspan="1" colspan="1">Monkey</td> <td align="justify" rowspan="1" colspan="1">Monkey</td> <td align="justify" rowspan="1" colspan="1">Monkey</td> <td align="justify" rowspan="1" colspan="1">Human</td> <td align="justify" rowspan="1" colspan="1">Human</td> <td align="justify" rowspan="1" colspan="1">Human</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>CDS length (in aa)</b></td> <td align="justify" rowspan="1" colspan="1">3,419</td> <td align="justify" rowspan="1" colspan="1">3,419</td> <td align="justify" rowspan="1" colspan="1">3,423</td> <td align="left" rowspan="1" colspan="1">3,407</td> <td align="justify" rowspan="1" colspan="1">3,423</td> <td align="justify" rowspan="1" colspan="1">3,424</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>C</b></td> <td align="justify" rowspan="1" colspan="1">122</td> <td align="justify" rowspan="1" colspan="1">122</td> <td align="justify" rowspan="1" colspan="1">122</td> <td align="justify" rowspan="1" colspan="1">109</td> <td align="justify" rowspan="1" colspan="1">125</td> <td align="justify" rowspan="1" colspan="1">105</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>prM</b></td> <td align="justify" rowspan="1" colspan="1">178</td> <td align="justify" rowspan="1" colspan="1">168</td> <td align="justify" rowspan="1" colspan="1">168</td> <td align="justify" rowspan="1" colspan="1">165</td> <td align="justify" rowspan="1" colspan="1">165</td> <td align="justify" rowspan="1" colspan="1">187</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>E</b></td> <td align="justify" rowspan="1" colspan="1">500</td> <td align="justify" rowspan="1" colspan="1">500</td> <td align="justify" rowspan="1" colspan="1">504</td> <td align="justify" rowspan="1" colspan="1">505</td> <td align="justify" rowspan="1" colspan="1">505</td> <td align="justify" rowspan="1" colspan="1">505</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS1</b></td> <td align="justify" rowspan="1" colspan="1">342</td> <td align="justify" rowspan="1" colspan="1">352</td> <td align="justify" rowspan="1" colspan="1">352</td> <td align="justify" rowspan="1" colspan="1">362</td> <td align="justify" rowspan="1" colspan="1">362</td> <td align="justify" rowspan="1" colspan="1">352</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS2A</b></td> <td align="justify" rowspan="1" colspan="1">226</td> <td align="justify" rowspan="1" colspan="1">226</td> <td align="justify" rowspan="1" colspan="1">226</td> <td align="justify" rowspan="1" colspan="1">218</td> <td align="justify" rowspan="1" colspan="1">218</td> <td align="justify" rowspan="1" colspan="1">217</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS2B</b></td> <td align="justify" rowspan="1" colspan="1">130</td> <td align="justify" rowspan="1" colspan="1">130</td> <td align="justify" rowspan="1" colspan="1">130</td> <td align="justify" rowspan="1" colspan="1">144</td> <td align="justify" rowspan="1" colspan="1">144</td> <td align="justify" rowspan="1" colspan="1">139</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS3</b></td> <td align="justify" rowspan="1" colspan="1">617</td> <td align="justify" rowspan="1" colspan="1">617</td> <td align="justify" rowspan="1" colspan="1">617</td> <td align="justify" rowspan="1" colspan="1">603</td> <td align="justify" rowspan="1" colspan="1">603</td> <td align="justify" rowspan="1" colspan="1">619</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS4A</b></td> <td align="justify" rowspan="1" colspan="1">127</td> <td align="justify" rowspan="1" colspan="1">127</td> <td align="justify" rowspan="1" colspan="1">127</td> <td align="justify" rowspan="1" colspan="1">147</td> <td align="justify" rowspan="1" colspan="1">147</td> <td align="justify" rowspan="1" colspan="1">127</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>2K</b></td> <td align="justify" rowspan="1" colspan="1">23</td> <td align="justify" rowspan="1" colspan="1">23</td> <td align="justify" rowspan="1" colspan="1">23</td> <td align="justify" rowspan="1" colspan="1">-</td> <td align="justify" rowspan="1" colspan="1">-</td> <td align="justify" rowspan="1" colspan="1">-</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS4B</b></td> <td align="justify" rowspan="1" colspan="1">252</td> <td align="justify" rowspan="1" colspan="1">251</td> <td align="justify" rowspan="1" colspan="1">251</td> <td align="justify" rowspan="1" colspan="1">502</td> <td align="justify" rowspan="1" colspan="1">502</td> <td align="justify" rowspan="1" colspan="1">255</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1"><b>NS5</b></td> <td align="justify" rowspan="1" colspan="1">902</td> <td align="justify" rowspan="1" colspan="1">903</td> <td align="justify" rowspan="1" colspan="1">903</td> <td align="justify" rowspan="1" colspan="1">652</td> <td align="justify" rowspan="1" colspan="1">652</td> <td align="justify" rowspan="1" colspan="1">904</td> </tr> </tbody> </table> </div> </div> <p>As for any pathogen, an integrated map of ZIKV genomewide diversity and genetic processes that drive the tempo and mode of evolution advances the design of diagnostics, therapeutics, vaccines, and other control strategies. However, an adequate framework for ensuring standardized genome analysis methods and curation of (meta-)data is lacking for ZIKV. The NCBI Reference Sequence Database is widely used as the primary reference for genomics. While multiple sources in the public domain make the efforts to annotate and curate viral genomes adequately, these sources are not yet widely and routinely used. Clinicians and researchers can be misinformed by information submitted to Genbank, and the erroneous translation of study findings can challenge the advance of ZIKV knowledge. Therefore, it is pivotal that a single reference Genbank entry for ZIKV is created with a CDS length of 3,423 aa, complete UTR regions, and a correctly curated genome annotation. Despite its careful selection criteria, we do not support the human WHO candidate reference strain KX369547 as preferred reference sequence for comparative genomics purposes [<a href="#ppat.1006528.ref001">1</a>]. Instead, we propose the use of the ZIKV genome sequence GAN KJ776791 [<a href="#ppat.1006528.ref010">10</a>]. This sequence matches well with temporal, geographical, and epidemiological characteristics of the WHO reference strain, but in addition, KJ776791 has complete UTRs (<a href="#ppat.1006528.g001">Fig 1</a>). The CDSs of KX369547 and KJ776791 are genetically similar, differing only at 8 nucleotide positions and 2 amino acid positions. The widespread distribution of the amino acid substitutions (L42F in the M protein and E146K in the NS1 protein) in the recent epidemic further support the use of KJ776791 over KX369547 (<a href="http://www.nextstrain.org/zika">http://www.nextstrain.org/zika</a>). To this end, the protein annotation of KJ776791 (and of KX369547) needs to be corrected using predicted consensus protease cleavage sites found in the ZIKV open reading frame or, ideally, supported by experimental data [<a href="#ppat.1006528.ref003">3</a>,<a href="#ppat.1006528.ref004">4</a>]. <a href="#ppat.1006528.g001">Fig 1</a> gives a visual representation of our proposed annotation of KJ776791 and of the current WHO reference strain. Detailed information on this corrected annotation can be found at <a href="http://rega.kuleuven.be/cev/reference-sequences/rega-zikv">http://rega.kuleuven.be/cev/reference-sequences/rega-zikv</a>. Furthermore, the selection of additional reliable and curated reference sequences is warranted to overcome current inconsistencies and to establish a set representative for worldwide ZIKV genetic diversity. The scientific community should support such efforts, for example, by using data from the European Virus Archive (<a href="https://www.european-virus-archive.com">https://www.european-virus-archive.com</a>).</p> <p>As time passes, errors propagate in literature and become increasingly difficult to correct. Also, ZIKV genomic data, regardless of any reference potential, should be well curated and given sufficient metadata when made publicly available. Complete ZIKV genomes must cover UTRs and should be indicated accordingly when, in fact, limited to the complete CDS (with partial UTRs) or containing large regions of undetermined nucleotides. In fact, this lack of curation and standardized annotation impairs the accurate use of public data. In conclusion, electronic records can be adapted quickly, but tools for automated curation and verification of sequence data need to be implemented routinely and in a widespread manner. Researchers should also be more vigilant regarding annotation discrepancies.</p> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1006528.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trösemeier</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musso</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blümel</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thézé</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O.G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baylis</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span>, <span class="mixed-article-title" title="mixed-article-title">Genome Sequence of a Candidate World Health Organization Reference Strain of Zika Virus for Nucleic Acid Testing</span>. <span class="source" tagx="source" title="source"><i>Genome Announc</i></span>. <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">e00917</span>–<span class="lpage" tagx="lpage" title="lpage">16</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="comment">doi: <a href="https://doi.org/10.1128/genomeA.00917-16">10.1128/genomeA.00917-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27587826">27587826</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naccache</span><span class="given-names" tagx="given-names" title="given-names">S. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thézé</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sardi</span><span class="given-names" tagx="given-names" title="given-names">S. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somasekar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greninger</span><span class="given-names" tagx="given-names" title="given-names">A. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bandeira</span><span class="given-names" tagx="given-names" title="given-names">A. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campos</span><span class="given-names" tagx="given-names" title="given-names">G.S.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Distinct Zika Virus Lineage in Salvador, Bahia, Brazil</span>. <span class="source" tagx="source" title="source">Emerg. <i>Infect</i>. <i>Dis</i></span>. <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">1788</span>–<span class="lpage" tagx="lpage" title="lpage">1792</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="comment">doi: <a href="https://doi.org/10.3201/eid2210.160663">10.3201/eid2210.160663</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27448188">27448188</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuno</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">G.-J. J</span></span>, <span class="mixed-article-title" title="mixed-article-title">Full-length sequencing and genomic characterization of Bagaza, Kedougou, and Zika viruses</span>. <span class="source" tagx="source" title="source"><i>Arch</i>. <i>Virol</i></span>. <span class="volume" tagx="volume" title="volume">152</span>, <span class="fpage" tagx="fpage" title="fpage">687</span>–<span class="lpage" tagx="lpage" title="lpage">696</span> (<span class="year" tagx="year" title="year">2007</span>). <span class="comment">doi: <a href="https://doi.org/10.1007/s00705-006-0903-z">10.1007/s00705-006-0903-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17195954">17195954</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatcher</span><span class="given-names" tagx="given-names" title="given-names">E. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhdanov</span><span class="given-names" tagx="given-names" title="given-names">S. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blinkova</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nawrocki</span><span class="given-names" tagx="given-names" title="given-names">E. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostapchuck</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schäffer</span><span class="given-names" tagx="given-names" title="given-names">A. A.</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Virus Variation Resource—improved response to emergent viral outbreaks</span>. <span class="source" tagx="source" title="source"><i>Nucleic Acids Res</i></span>. <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">482</span>–<span class="lpage" tagx="lpage" title="lpage">490</span> (<span class="year" tagx="year" title="year">2016</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899638">27899638</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddow</span><span class="given-names" tagx="given-names" title="given-names">A. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuh</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yasuda</span><span class="given-names" tagx="given-names" title="given-names">C. Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasper</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heang</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genetic Characterization of Zika Virus Strains: Geographic Expansion of the Asian Lineage</span>. <span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis</i></span>. <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">e1477</span> (<span class="year" tagx="year" title="year">2012</span>). <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0001477">10.1371/journal.pntd.0001477</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389730">22389730</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sirohi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klose</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierson</span><span class="given-names" tagx="given-names" title="given-names">T. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossmann</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The 3.8Å resolution cryo-EM structure of Zika Virus</span>. <span class="source" tagx="source" title="source"><i>Science</i></span><span class="volume" tagx="volume" title="volume">352</span>, <span class="fpage" tagx="fpage" title="fpage">467</span>–<span class="lpage" tagx="lpage" title="lpage">470</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="comment">doi: <a href="https://doi.org/10.1126/science.aaf5316">10.1126/science.aaf5316</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27033547">27033547</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aliota</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudley</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newman</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohr</span><span class="given-names" tagx="given-names" title="given-names">E. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gellerup</span><span class="given-names" tagx="given-names" title="given-names">D. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitbach</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Heterologous Protection against Asian Zika Virus Challenge in Rhesus Macaques</span>. <span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis</i></span>. <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">e0005168</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0005168">10.1371/journal.pntd.0005168</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27911897">27911897</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faye</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freire</span><span class="given-names" tagx="given-names" title="given-names">C. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iamarino</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faye</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">J. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diallo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution of Zika virus during its emergence in the 20(th) century</span>. <span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis</i></span>. <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e2636</span> (<span class="year" tagx="year" title="year">2014</span>). <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pntd.0002636">10.1371/journal.pntd.0002636</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24421913">24421913</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yun</span><span class="given-names" tagx="given-names" title="given-names">S. I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">B. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frank</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Julander</span><span class="given-names" tagx="given-names" title="given-names">J. G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polejaeva</span><span class="given-names" tagx="given-names" title="given-names">I. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davies</span><span class="given-names" tagx="given-names" title="given-names">C. J.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Complete Genome Sequences of Three Historically Important, Spatiotemporally Distinct, and Genetically Divergent Strains of Zika Virus: MR-766, P6-740, and PRVABC-59</span>. <span class="source" tagx="source" title="source"><i>Genome Announc</i></span>. <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">e00800</span>–<span class="lpage" tagx="lpage" title="lpage">16</span> (<span class="year" tagx="year" title="year">2016</span>). <span class="comment">doi: <a href="https://doi.org/10.1128/genomeA.00800-16">10.1128/genomeA.00800-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27540058">27540058</a></span></span></li> <li tag="ref"><a name="ppat.1006528.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baronti</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piorkowski</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charrel</span><span class="given-names" tagx="given-names" title="given-names">R. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boubis</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leparc-Goffart</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Lamballerie</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>. <span class="mixed-article-title" title="mixed-article-title">Complete coding sequence of Zika virus from a French Polynesia outbreak in 2013</span>. <span class="source" tagx="source" title="source"><i>Genome Announc</i></span>. <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">e00500</span>–<span class="lpage" tagx="lpage" title="lpage">14</span> (<span class="year" tagx="year" title="year">2014</span>). <span class="comment">doi: <a href="https://doi.org/10.1128/genomeA.00500-14">10.1128/genomeA.00500-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24903869">24903869</a></span></span></li> </ul> </div> </div>  </body></html>