<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.04.08.031203.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.08.031203v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin" />
<meta name="DC.Identifier" content="10.1101/2020.04.08.031203" />
<meta name="DC.Date" content="2020-04-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bats are recognized as important reservoirs of viruses deadly to other mammals, including humans. These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals. Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="DC.Contributor" content="Joshua A. Hayward" />
<meta name="DC.Contributor" content="Mary Tachedjian" />
<meta name="DC.Contributor" content="Adam Johnson" />
<meta name="DC.Contributor" content="Tamsin B. Gordon" />
<meta name="DC.Contributor" content="Jie Cui" />
<meta name="DC.Contributor" content="Glenn A. Marsh" />
<meta name="DC.Contributor" content="Michelle L. Baker" />
<meta name="DC.Contributor" content="Lin-Fa Wang" />
<meta name="DC.Contributor" content="Gilda Tachedjian" />
<meta name="article:published_time" content="2020-04-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bats are recognized as important reservoirs of viruses deadly to other mammals, including humans.  These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals.  Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.08.031203v1" />
<meta name="citation_id" content="2020.04.08.031203v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.08.031203" />
<meta name="citation_num_pages" content="46" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.08.031203" />
<meta name="citation_author" content="Joshua A. Hayward" />
<meta name="citation_author_institution" content="Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences" />
<meta name="citation_author_institution" content="Department of Microbiology, Monash University" />
<meta name="citation_author" content="Mary Tachedjian" />
<meta name="citation_author_institution" content="CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit" />
<meta name="citation_author" content="Adam Johnson" />
<meta name="citation_author_institution" content="Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences" />
<meta name="citation_author" content="Tamsin B. Gordon" />
<meta name="citation_author_institution" content="Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences" />
<meta name="citation_author_institution" content="Department of Microbiology, Monash University" />
<meta name="citation_author" content="Jie Cui" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Molecular Virology &amp; Immunology, Institut Pasteur of Shanghai, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center for Biosafety Mega-Science, Chinese Academy of Sciences" />
<meta name="citation_author" content="Glenn A. Marsh" />
<meta name="citation_author_institution" content="CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit" />
<meta name="citation_author" content="Michelle L. Baker" />
<meta name="citation_author_institution" content="CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit" />
<meta name="citation_author" content="Lin-Fa Wang" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke-NUS Medical School" />
<meta name="citation_author" content="Gilda Tachedjian" />
<meta name="citation_author_institution" content="Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences" />
<meta name="citation_author_institution" content="Department of Microbiology, Monash University" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, The University of Melbourne, at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_email" content="gilda.tachedjian@burnet.edu.au" />
<meta name="citation_reference" content="Calisher CH, Childs JE, Field HE, Holmes KV, &amp; Schountz T (2006) Bats: Important reservoir hosts of emerging viruses. Clin. Microbiol. Rev. 19(3):531–545." />
<meta name="citation_reference" content="Smith I &amp; Wang L-F (2013) Bats and their virome: an important source of emerging viruses capable of infecting humans. Curr. Opin. Virol. 3(1):84–91." />
<meta name="citation_reference" content="Amman BR, et al. (2015) A recently discovered pathogenic paramyxovirus, Sosuga virus, is present in Rousettus aegyptiacus fruit bats at multiple locations in Uganda. J. Wildl. Dis. 51(3):774–779." />
<meta name="citation_reference" content="Zhou P, et al. (2020) A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature:1–4." />
<meta name="citation_reference" content="Swanepoel R (1996) Experimental inoculation of plants and animals with Ebola virus. Emerging Infect. Dis. 2(4):321." />
<meta name="citation_reference" content="Williamson MM, et al. (1998) Transmission studies of Hendra virus (equine morbilli-virus) in fruit bats, horses and cats. Aust. Vet. J. 76(12):813–818." />
<meta name="citation_reference" content="Middleton DJ, et al. (2007) Experimental Nipah Virus Infection in Pteropid Bats (Pteropus poliocephalus). J. Comp. Pathol. 136(4):266–272." />
<meta name="citation_reference" content="Watanabe S, et al. (2010) Bat coronaviruses and experimental infection of bats, the Philippines. Emerging Infect. Dis. 16(8):1217–1223." />
<meta name="citation_reference" content="Jones ME, et al. (2015) Experimental inoculation of egyptian rousette bats (Rousettus aegyptiacus) with viruses of the Ebolavirus and Marburgvirus genera. Viruses 7(7):3420–3442." />
<meta name="citation_reference" content="Baker ML, Schountz T, &amp; Wang LF (2013) Antiviral immune responses of bats: A review. Zoonoses Public Hlth:1–13." />
<meta name="citation_reference" content="Zhang G, et al. (2013) Comparative analysis of bat genomes provides insight into the evolution of flight and immunity. Science 339(6118):456–460." />
<meta name="citation_reference" content="Zhou P, et al. (2016) Contraction of the type I IFN locus and unusual constitutive expression of IFN-α in bats. Proc. Natl. Acad. Sci. USA:201518240." />
<meta name="citation_reference" content="Hayward JA, et al. (2018) Differential evolution of antiretroviral restriction factors in pteropid bats as revealed by APOBEC3 gene complexity. Mol. Biol. Evol. 35(7):1626–1637." />
<meta name="citation_reference" content="Pavlovich SS, et al. (2018) The Egyptian rousette genome reveals unexpected features of bat antiviral immunity. Cell 173(5):1098-1110. e1018." />
<meta name="citation_reference" content="Mandl JN, Schneider C, Schneider DS, &amp; Baker ML (2018) Going to bat(s) for studies of disease tolerance. Front. Immunol. 9:2112." />
<meta name="citation_reference" content="Hawkins JA, et al. (2019) A metaanalysis of bat phylogenetics and positive selection based on genomes and transcriptomes from 18 species. Proc. Natl. Acad. Sci. U. S. A. 116(23):11351–11360." />
<meta name="citation_reference" content="Neil SJD, Zang T, &amp; Bieniasz PD (2008) Tetherin inhibits retrovirus release and is antagonized by HIV-1 Vpu. Nature 451(7177):425–430." />
<meta name="citation_reference" content="Sakuma T, Noda T, Urata S, Kawaoka Y, &amp; Yasuda J (2009) Inhibition of Lassa and Marburg virus production by tetherin. J. Virol. 83(5):2382–2385." />
<meta name="citation_reference" content="Hoffmann M, et al. (2019) Tetherin inhibits Nipah virus but not Ebola virus replication in fruit bat cells. J. Virol. 93(3):e01821–01818." />
<meta name="citation_reference" content="Wynne JW &amp; Wang L-F (2013) Bats and viruses: Friend or foe? PLoS Pathog. 9(10):e1003651." />
<meta name="citation_reference" content="Wang SM, Huang KJ, &amp; Wang CT (2019) Severe acute respiratory syndrome coronavirus spike protein counteracts BST2-mediated restriction of virus-like particle release. J. Med. Virol. 91(10):1743–1750." />
<meta name="citation_reference" content="Perez-Caballero D, et al. (2009) Tetherin inhibits HIV-1 release by directly tethering virions to cells. Cell 139(3):499–511." />
<meta name="citation_reference" content="Blanco-Melo D, Venkatesh S, &amp; Bieniasz PD (2016) Origins and evolution of tetherin, an orphan antiviral gene. Cell Host Microbe 20(2):189–201." />
<meta name="citation_reference" content="Cocka LJ &amp; Bates P (2012) Identification of alternatively translated tetherin isoforms with differing antiviral and signaling activities. PLoS Pathog. 8(9):e1002931." />
<meta name="citation_reference" content="Galão Rui P, Le Tortorec A, Pickering S, Kueck T, &amp; Neil Stuart JD (2012) Innate sensing of HIV-1 assembly by tetherin induces NFκB-dependent proinflammatory responses. Cell Host Microbe 12(5):633–644." />
<meta name="citation_reference" content="Tokarev A, et al. (2013) Stimulation of NF-κB activity by the HIV restriction factor BST2. J. Virol. 87(4):2046–2057." />
<meta name="citation_reference" content="Matsuda A, et al. (2003) Large-scale identification and characterization of human genes that activate NF-κB and MAPK signaling pathways. Oncogene 22(21):3307–3318." />
<meta name="citation_reference" content="Ishikawa J, et al. (1995) Molecular cloning and chromosomal mapping of a bone marrow stromal cell surface gene, BST2, that may be involved in pre-B-cell growth. Genomics 26(3):527–534." />
<meta name="citation_reference" content="Liberatore RA &amp; Bieniasz PD (2011) Tetherin is a key effector of the antiretroviral activity of type I interferon in vitro and in vivo. Proc. Natl. Acad. Sci. U. S. A. 108(44):18097–18101." />
<meta name="citation_reference" content="Erikson E, et al. (2011) In vivo expression profile of the antiviral restriction factor and tumor-targeting antigen CD317/BST-2/HM1.24/tetherin in humans. Proc. Natl. Acad. Sci. U. S. A. 108(33):13688–13693." />
<meta name="citation_reference" content="Yang H, et al. (2010) Structural insight into the mechanisms of enveloped virus tethering by tetherin. Proc. Natl. Acad. Sci. U. S. A. 107(43):18428–18432." />
<meta name="citation_reference" content="Hinz A, et al. (2010) Structural basis of HIV-1 tethering to membranes by the BST-2/tetherin ectodomain. Cell Host Microbe 7(4):314–323." />
<meta name="citation_reference" content="Kupzig S, et al. (2003) Bst-2/HM1. 24 is a raft-associated apical membrane protein with an unusual topology. Traffic 4(10):694–709." />
<meta name="citation_reference" content="Rollason R, Korolchuk V, Hamilton C, Schu P, &amp; Banting G (2007) Clathrin-mediated endocytosis of a lipid-raft-associated protein is mediated through a dual tyrosine motif. J. Cell Sci. 120(21):3850–3858." />
<meta name="citation_reference" content="Takeda E, et al. (2012) Identification and functional analysis of three isoforms of bovine BST-2. PLoS One 7(7):e41483." />
<meta name="citation_reference" content="Iwabu Y, et al. (2009) HIV-1 accessory protein Vpu internalizes cell-surface BST-2/tetherin through transmembrane interactions leading to lysosomes. J. Biol. Chem. 284(50):35060–35072." />
<meta name="citation_reference" content="Fukuma A, Abe M, Morikawa Y, Miyazawa T, &amp; Yasuda J (2011) Cloning and characterization of the antiviral activity of feline tetherin/BST-2. PLoS One 6(3)." />
<meta name="citation_reference" content="Ruedi M, et al. (2013) Molecular phylogenetic reconstructions identify East Asia as the cradle for the evolution of the cosmopolitan genus Myotis (Mammalia, Chiroptera). Mol. Phylogen. Evol. 69(3):437–449." />
<meta name="citation_reference" content="Benton MJ &amp; Donoghue PCJ (2007) Paleontological evidence to date the tree of life. Mol. Biol. Evol. 24(1):26–53." />
<meta name="citation_reference" content="Doering TL, Englund PT, &amp; Hart GW (1993) Detection of glycophospholipid anchors on proteins. Curr. Protoc. Mol. Biol. 22(1):17.18.11-17.18.13." />
<meta name="citation_reference" content="Dubé M, et al. (2009) Suppression of tetherin-restricting activity upon human immunodeficiency virus type 1 particle release correlates with localization of Vpu in the trans-Golgi network. J. Virol. 83(9):4574–4590." />
<meta name="citation_reference" content="Dietrich I, et al. (2011) Feline tetherin efficiently restricts release of feline immunodeficiency virus but not spreading of infection. J. Virol. 85(12):5840–5852." />
<meta name="citation_reference" content="Arnaud F, et al. (2010) Interplay between ovine bone marrow stromal cell antigen 2/tetherin and endogenous retroviruses. J. Virol. 84(9):4415–4425." />
<meta name="citation_reference" content="Celestino M, et al. (2012) Feline tetherin is characterized by a short N-terminal region and is counteracted by the feline immunodeficiency virus envelope glycoprotein. J. Virol. 86(12):6688–6700." />
<meta name="citation_reference" content="Schubert HL, et al. (2010) Structural and functional studies on the extracellular domain of BST2/tetherin in reduced and oxidized conformations. Proc. Natl. Acad. Sci. U. S. A. 107(42):17951–17956." />
<meta name="citation_reference" content="Andrew AJ, Miyagi E, Kao S, &amp; Strebel K (2009) The formation of cysteine-linked dimers of BST-2/tetherin is important for inhibition of HIV-1 virus release but not for sensitivity to Vpu. Retrovirology 6(1):80." />
<meta name="citation_reference" content="McNatt MW, et al. (2009) Species-specific activity of HIV-1 Vpu and positive selection of tetherin transmembrane domain variants. PLoS Pathog 5(2):e1000300." />
<meta name="citation_reference" content="Venkatesh S &amp; Bieniasz PD (2013) Mechanism of HIV-1 virion entrapment by tetherin. PLoS Pathog. 9(7):e1003483." />
<meta name="citation_reference" content="Sakuma T, Sakurai A, &amp; Yasuda J (2009) Dimerization of tetherin is not essential for its antiviral activity against Lassa and Marburg viruses. PLoS One 4(9):e6934." />
<meta name="citation_reference" content="Yang Z (2007) PAML 4: Phylogenetic analysis by maximum likelihood. Mol. Biol. Evol. 24(8):1586–1591." />
<meta name="citation_reference" content="Teeling EC, et al. (2005) A molecular phylogeny for bats illuminates biogeography and the fossil record. Science 307(5709):580–584." />
<meta name="citation_reference" content="Tamura K, Stecher G, Peterson D, Filipski A, &amp; Kumar S (2013) MEGA6: Molecular evolutionary genetics analysis version 6.0. Mol. Biol. Evol. 30(12):2725–2729." />
<meta name="citation_reference" content="Krogh A, Larsson B, Von Heijne G, &amp; Sonnhammer EL (2001) Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. J. Mol. Biol. 305(3):567–580." />
<meta name="citation_reference" content="Pierleoni A, Martelli P, &amp; Casadio R (2008) PredGPI: a GPI-anchor predictor. BMC Bioinform. 9(1):1." />
<meta name="citation_reference" content="Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nuc. Acid. Res. 32(5):1792–1797." />
<meta name="citation_reference" content="Cowled C, et al. (2011) Molecular characterisation of Toll-like receptors in the black flying fox Pteropus alecto. Dev. Comp. Immunol. 35(1):7–18." />
<meta name="citation_reference" content="Sambrook J, Fritsch EF, &amp; Maniatis T (1989) Molecular cloning, (Cold Spring Harbor laboratory press, New York)." />
<meta name="citation_reference" content="Brown TA (2002) Synthesis and processing of RNA. Genomes, (Wiley-Liss, Oxford)." />
<meta name="citation_reference" content="Kent WJ (2002) BLAT—the BLAST-like alignment tool. Genome Res. 12(4):656–664." />
<meta name="citation_reference" content="Jouvenet N, et al. (2009) Broad-spectrum inhibition of retroviral and filoviral particle release by tetherin. J. Virol. 83(4):1837–1844." />
<meta name="twitter:title" content="Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bats are recognized as important reservoirs of viruses deadly to other mammals, including humans. These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals. Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-title" property="og:title" content="Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bats are recognized as important reservoirs of viruses deadly to other mammals, including humans. These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals. Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.08.031203v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.08.031203v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.08.031203v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1236501" />
    <title>Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gH7emavOqE5dpGb_Pfk7wCoNPweBEWnaFuiD2fvVOeQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1236501","apath":"\/biorxiv\/early\/2020\/04\/09\/2020.04.08.031203.atom","pisa":"biorxiv;2020.04.08.031203v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.08.031203v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.08.031203v1","title":"Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin","identifier":"node\/1236501"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ojupdY8r3vupWbNfaElC1N476xOwgVoc2KL9PbKgKmc3nuwWDZWvSseAlqKie6E4V9VXNF763uDx0zS7fRSo\\\\\\\/uxSJwBXX0bg\\\\\\\/coEUTvv4DK\\\\\\\/t1pJ9POuHXtohu5VXTX\\\\\\\/OasqSJhN2B7eAXdJdtxdrMr3JTmp1j2\\\\\\\/oo+8NosfSsUeTdk07B\\\\\\\/8GbEHn3Mv6CZ4wftNRG1c2JubnG3cJv1FbTi8KhX6z0z1OICqfJQCZygWJc4hkfFqFgHCL\\\\\\\/EfBtzvkkGon\\\\\\\/ey83O7QX2a7aeOxJFauT4EN5mHJGXLO\\\\\\\/+ywfsheBeMkbgHfsDBBuMaLZd0Jb1gdtVIbAZvqGOdfFWzWo0TEUMXruZ5S5tt6sxYG7\\\\\\\/SzArvZzwv+tcmTTHE59zRzW28NgYKq0vGvVJrZ6oPcwZT2K1mUSy\\\\\\\/S1NAmfrHXZiNyM3RMU7KuCifGK+oaOIs\\\\\\\/iU5N5LQLDq3ZLV6+RR2kL3+uvDY0nKzrvg7FwweJx24LHHvar2dHk1XyCIjjUAJ\\\\\\\/X0AUKHAJDvF1iem\\\\\\\/ZUDQ9V8HxF2Q7CPRy9adozmVh+GIOH8NuJtzi47wN0bYMCzPb5Zv6sxP4IHaXCq+P+HtRjH4jn2TNxsL5kocQXDD+vExE3CuER0592\\\\\\\/s24uTiSVIegJshO4iQyId\\\\\\\/GTGDfHTVzgyA66AdcOXGXu9ube\\\\\\\/vfM4iztIpwErrQ033qZrvkwlYXZ\\\\\\\/nnAAvtwF\\\\\\\/heODwtbJIb\\\\\\\/ciWtU0V3Xu8Nv10+WoeWgaJ18fAAiVAwm6zZp1ieTHLrLPjoF8bur2wo+tXHJoJQZTEZknk53xC6xCTc8W6OBpMGgNoZCDEEqPtc8fKECkpjTL8dFsCs38U6iuLgQZ+InJQ+7SzeW\\\\\\\/3AOlNtO\\\\\\\/S2afcnIRWfR\\\\\\\/Uk1r5HLAlQfBY\\\\\\\/bD9BDaukliDI1NfGfOLTJsJZ9HrL0PMSGR+Q+Xu7FWRgVCz4vK8UzysPbFZlvKuadwvAERwSL4dIRAI9vZLGM9vLutVLrLsU2rhcWq6wuHeRqMIXI3podC0qzJvjBD8KS7WaLTcGAcdus39YTTCNu6XuzFdtDTlw3PsUl74lHstKXfqTClg8KHrXyJhPcj2Iv3uurmNV1zX1JbwuX2Gdv4jNcQ6JJIzKtGh8YKyLdyRm\\\\\\\/Z7UKZSzPtGywh5yIUbgmmbGlI+vMh22URMGJs++0R3aPefRR7dhYSqgwYMCL1vzuR3ZctZTvZz0oTX3iPQVqznf4GmYtPL9cYkkVyeOjzim82+8bEj4Gf4e+MUPxK055dZK0q2b2Jf43P6DqRIWbXTLW1mnwVue9RzXpuhQ9erVgVHANizicuV+5VIc2FmiI2JAXdBsgUoiJio\\\u0022,\\\u0022iv\\\u0022:\\\u0022YA0ZrcQK7dmn7GV3\\\\\\\/yKR5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b59e10b602a7179df7c8816798e4a5d1f5d18cd853faf83d021000eb91e91212\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.08.031203v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1236501 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.08.031203v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MP3Mbp2o7jyMlJ7uJPNDjjoRkX5RQfsysaeesmpTL4I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1236501" id="node1236501" data-pisa="biorxiv;2020.04.08.031203v1" data-pisa-master="biorxiv;2020.04.08.031203" data-apath="/biorxiv/early/2020/04/09/2020.04.08.031203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Joshua A.</span> <span class="nlm-surname">Hayward</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Tachedjian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tamsin B.</span> <span class="nlm-surname">Gordon</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Glenn A.</span> <span class="nlm-surname">Marsh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Michelle L.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Gilda</span> <span class="nlm-surname">Tachedjian</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.08.031203 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1236501" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joshua A. Hayward </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences</span>, Melbourne, VIC 3004, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Microbiology, Monash University</span>, Clayton, VIC 3168, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BA.%2BHayward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayward%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoshua%2BA.%2BHayward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mary Tachedjian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit</span>, Geelong, VIC 3220, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BTachedjian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tachedjian%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BTachedjian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Adam Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences</span>, Melbourne, VIC 3004, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tamsin B. Gordon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences</span>, Melbourne, VIC 3004, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Microbiology, Monash University</span>, Clayton, VIC 3168, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamsin%2BB.%2BGordon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gordon%20TB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATamsin%2BB.%2BGordon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jie Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>CAS Key Laboratory of Molecular Virology &#38; Immunology, Institut Pasteur of Shanghai, Chinese Academy of Sciences</span>, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>Center for Biosafety Mega-Science, Chinese Academy of Sciences</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jie%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJie%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Glenn A. Marsh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit</span>, Geelong, VIC 3220, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Glenn%2BA.%2BMarsh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marsh%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGlenn%2BA.%2BMarsh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michelle L. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>CSIRO, Australian Animal Health Laboratory, Health and Biosecurity Business Unit</span>, Geelong, VIC 3220, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BL.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BL.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lin-Fa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>f</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke-NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin-Fa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALin-Fa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gilda Tachedjian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Health Security Program, Life Sciences Discipline, Burnet Institute, Life Sciences</span>, Melbourne, VIC 3004, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Microbiology, Monash University</span>, Clayton, VIC 3168, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>g</span><span class='nlm-institution'>Department of Microbiology and Immunology, The University of Melbourne, at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, VIC 3000, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilda%2BTachedjian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tachedjian%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGilda%2BTachedjian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gilda.tachedjian@burnet.edu.au" class="" data-icon-position="" data-hide-link-title="0">gilda.tachedjian@burnet.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.08.031203v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031203v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031203v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031203v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031203v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.08.031203v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1236501" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1236501/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Bats are recognized as important reservoirs of viruses deadly to other mammals, including humans. These infections are typically nonpathogenic in bats raising questions about innate immune differences that might exist between bats and other mammals. Tetherin is an important restriction factor which inhibits the release of a diverse range of viruses from host cells, including retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are deadly to humans and transmitted by bats. Here we characterise the tetherin genes from 27 species of bats, revealing that they are under strong selective pressure, and that fruit bats and vesper bats express unique structural variants of the tetherin protein. The expression of two computationally predicted splice isoforms of fruit bat tetherin is verified here, and we identify an additional third unique splice isoform which includes a C-terminal region that is not homologous to known mammalian tetherin variants. We also report that vesper bats possess and express at least five tetherin genes, including structural variants, a greater number than any other mammal reported to date. These findings support the hypothesis of differential antiviral gene evolution in bats relative to other mammals.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Bats are reservoirs of viruses that are highly pathogenic to other mammals, including humans. Viruses such as Hendra, Nipah, Ebola, Marburg, severe acute respiratory syndrome coronaviruses (SARS-CoV-1 and likely also SARS-CoV-2), and the Sosuga virus have crossed species barriers from bats into humans (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). Laboratory studies demonstrate that specific bat species can be infected with Ebola, Marburg, SARS-CoV-1, Hendra, and Nipah viruses without showing clinical symptoms (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Recently, this has led to efforts to elucidate if there are differences observed between the antiviral strategies of bats and other mammals (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). These studies have identified that genes related to immunity in bats are under a significant degree of positive selection, in addition to differences in the copy number and diversity of innate immune genes of bats relative to other mammals (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>-<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>).</p><p id="p-5">Tetherin (BST-2, CD317) is a mammalian restriction factor, an antiviral protein that inhibits the release of diverse enveloped viral particles from the cells in which they are produced (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>-<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). These viruses include retroviruses, coronaviruses, filoviruses, and paramyxoviruses, some of which are transmissible by bats (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). Tetherin is a membrane-associated glycoprotein and its activity is determined by its structure, rather than its primary amino acid sequence (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). Tetherin has been shown to possess a secondary role in immune signaling events by triggering the NF-κB signaling pathway, leading to stimulation of the antiviral interferon response (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>-<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>).</p><p id="p-6">In humans, tetherin is expressed across most cell types, including its BST-2 namesake bone marrow stromal cells, and expression may be upregulated by stimulation with type I interferons (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Tetherin is a dimeric dual-anchor type II membrane protein that contains one protein anchor, a transmembrane domain near its N-terminus, an extracellular coiled-coil domain, and a glycophosphatidylinositol (GPI) lipid anchor, which is attached to its C-terminus as a post-translational modification (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>-<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). Tetherin contains a number of conserved cytosine and asparagine motifs within the extracellular domain, with respective roles in dimerisation and glycosylation, and a dual-tyrosine motif (Y·x·Y) in its cytoplasmic region, which has a role in viral particle endocytosis and immune signaling cascades (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Tetherin is located in lipid rafts at the plasma membrane where many viral particles bud during acquisition of their host membrane-derived viral envelopes (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>). During the viral budding process one anchor remains embedded in the nascent viral envelope while the other remains attached to the plasma membrane, tethering the virion to the cell and preventing its release into the extracellular environment (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>).</p><p id="p-7">Tetherin is common to all mammals, and orthologs which share structural and functional, but not sequence similarity, exist in other vertebrates (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). Most mammals carry only a single tetherin gene; however gene duplication has been observed in sheep, cows, opossums and wallabies (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Furthermore, human tetherin is expressed in two alternative isoforms (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>), the long (l-tetherin) and short (s-tetherin) isoforms, which differ through the truncation of 12 amino acid residues at the N-terminus of l-tetherin. Computational predictions also indicate two additional isoforms, X1 and X2, with an internal, rather than N-terminal difference in amino acid sequences, may also be expressed in fruit bats i.e. megabats of the suborder Yinpterochiroptera, genus <em>Pteropus</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>). The predicted isoform X1 is homologous to human l-tetherin, and the X2 isoform is a splice variant which contains a 7 AA exclusion within the extracellular coiled-coil domain relative to isoform X1. A recent analysis of tetherin in fruit bats has identified that, similar to other mammals, bat tetherin is a functional restriction factor capable of inhibiting the release of the bat-hosted Nipah virus, as well as virus-like particles (VLPs) produced from the Ebola virus and the human immunodeficiency virus (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>).</p><p id="p-8">We hypothesized that given the role of bats as hosts of pathogenic enveloped viruses, and the differences observed in other antiviral genes of their innate immune system (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>), that diversity might also exist in the tetherin genes of bats. Here we report that fruit bats possess a unique structural isoform in addition to the two computational predictions, tetherin isoforms X1 and X2, which were generated through the automated NCBI annotation pipeline from the published <em>P. alecto</em> genome (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). We also find that vesper bats, i.e. microbats of the suborder Yangochiroptera, genus <em>Myotis</em>, possess at least five tetherin genes, two of which contain large structural differences in the extracellular coiled-coil domain. An analysis of the tetherin genes of 27 species of bats reveal that bat tetherin has been subjected to strong positive selection, indicating these flying mammals have experienced evolutionary pressure on this important antiviral gene. Taken together, these findings indicate that bats have undergone tetherin gene expansion and diversification relative to other mammals, supporting the hypothesis of differential antiviral gene evolution in bats.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Bats possess structural homologs of human tetherin</h3><p id="p-9">To advance our understanding of bat tetherin homologs, we mined all available publically accessible sequence read archives representing 26 species of bats through a BLASTn analysis (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The species analysed here represent 8 of the 20 families within the order Chiroptera: three families (Hipposideridae, Pteropodidae, and Rhinolophidae) from within the suborder Yinpterochiroptera, and five families (Emballonuridae, Miniopteridae, Molossidae, Vespertilionidae, and Phyllostomidae) from within the suborder Yangochiroptera. This analysis revealed mRNA sequences representing homologs of human tetherin from 26 bat species, enabling the generation of a consensus bat tetherin sequence from a multiple sequence alignment (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and SI Data 1). The primary sequence lengths range from 177 to 221 AA, compared to 180 AA for human l-tetherin, and of these only 23 residues were conserved in 100% of the bat tetherin sequences analysed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The assembled tetherin sequence for all bats was a homolog of human l-tetherin, except for the Black-bearded Tomb bat (<em>Taphozous melanopogon</em>), which was predicted to possess an equivalent of the human s-tetherin. The predicted <em>T. melanopogon</em> tetherin nucleotide sequence contains two key mutations upstream of its short isoform start site: an ATG to CTG change in the long isoform start codon, and a TAG stop codon in between the two start codon sites, either of which would be sufficient to prevent the production of the long isoform of tetherin. No nucleotide conflicts exist between the 17 reads mapping to this region of the <em>T. melanopogon</em> tetherin sequence, indicating that these mutations are unlikely to represent sequencing or consensus alignment errors.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242976/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242976/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242976" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-10" class="first-child">Sequence accessions with Bioproject and cDNA sources for the prediction and/or confirmation of tetherin homologues</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bat Tetherin structure and sequence diversity. A. The consensus bat tetherin amino acid sequence generated through the multiple sequence alignment of tetherin from 27 bat species (SI Data 1) and compared against human tetherin. Strong positive selection of bat tetherin is revealed by the ratio of non-synonymous to synonymous mutations (Table 2), and is represented as evolutionary hotspots. The YxY dual tyrosine motif, the alternative start site for the short isoform of tetherin, and the GPI anchor attachment site positions are indicated by arrows. The red asterisk (*) indicates the location of the inserted molecular tags in the P. alecto and M. macropus tetherin expression constructs. CD, cytoplasmic domain; TM, transmembrane domain; ED, extracellular domain; GPI, glycophosphatidylinositol; GSP, GPI signal peptide. B. Significant sequence diversity exists among bat tetherin proteins, indicated by the percentage of amino acid sequence conservation at each site of the consensus bat tetherin. Amino acids conserved in all 27 sequences are represented by their letters. Amino acids represented by X are variable and are included to indicate the sequence distance between closely positioned conserved residues. C. The consensus bat tetherin sequence. Amino acid residues represented in &gt; 50% of bat tetherin sequences are indicated with their letter. Positions at which no amino acid residue is represented in &gt; 50% of bat tetherin sequences are indicated with &#x2018;X&#x2019;. The red line indicates the position of the dual tyrosine motif." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bat Tetherin structure and sequence diversity.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. The consensus bat tetherin amino acid sequence generated through the multiple sequence alignment of tetherin from 27 bat species (SI Data 1) and compared against human tetherin. Strong positive selection of bat tetherin is revealed by the ratio of non-synonymous to synonymous mutations (Table 2), and is represented as evolutionary hotspots. The YxY dual tyrosine motif, the alternative start site for the short isoform of tetherin, and the GPI anchor attachment site positions are indicated by arrows. The red asterisk (*) indicates the location of the inserted molecular tags in the &lt;em&gt;P. alecto&lt;/em&gt; and &lt;em&gt;M. macropus&lt;/em&gt; tetherin expression constructs. CD, cytoplasmic domain; TM, transmembrane domain; ED, extracellular domain; GPI, glycophosphatidylinositol; GSP, GPI signal peptide. &lt;strong&gt;B&lt;/strong&gt;. Significant sequence diversity exists among bat tetherin proteins, indicated by the percentage of amino acid sequence conservation at each site of the consensus bat tetherin. Amino acids conserved in all 27 sequences are represented by their letters. Amino acids represented by X are variable and are included to indicate the sequence distance between closely positioned conserved residues. &lt;strong&gt;C&lt;/strong&gt;. The consensus bat tetherin sequence. Amino acid residues represented in &gt; 50% of bat tetherin sequences are indicated with their letter. Positions at which no amino acid residue is represented in &gt; 50% of bat tetherin sequences are indicated with &#x2018;X&#x2019;. The red line indicates the position of the dual tyrosine motif.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F1.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F1.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Bat Tetherin structure and sequence diversity.</span><p id="p-11" class="first-child"><strong>A</strong>. The consensus bat tetherin amino acid sequence generated through the multiple sequence alignment of tetherin from 27 bat species (SI Data 1) and compared against human tetherin. Strong positive selection of bat tetherin is revealed by the ratio of non-synonymous to synonymous mutations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), and is represented as evolutionary hotspots. The YxY dual tyrosine motif, the alternative start site for the short isoform of tetherin, and the GPI anchor attachment site positions are indicated by arrows. The red asterisk (*) indicates the location of the inserted molecular tags in the <em>P. alecto</em> and <em>M. macropus</em> tetherin expression constructs. CD, cytoplasmic domain; TM, transmembrane domain; ED, extracellular domain; GPI, glycophosphatidylinositol; GSP, GPI signal peptide. <strong>B</strong>. Significant sequence diversity exists among bat tetherin proteins, indicated by the percentage of amino acid sequence conservation at each site of the consensus bat tetherin. Amino acids conserved in all 27 sequences are represented by their letters. Amino acids represented by X are variable and are included to indicate the sequence distance between closely positioned conserved residues. <strong>C</strong>. The consensus bat tetherin sequence. Amino acid residues represented in &gt; 50% of bat tetherin sequences are indicated with their letter. Positions at which no amino acid residue is represented in &gt; 50% of bat tetherin sequences are indicated with ‘X’. The red line indicates the position of the dual tyrosine motif.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">The dual-tyrosine Y·x·Y motif, which is critical for mediating viral-particle endocytosis and is involved in immune signaling cascades (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>), is variable among bat species and exists in various combinations of Y|C·x·Y|H (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a> and <a id="xref-fig-1-4" class="xref-fig" href="#F1">1C</a>). All bat species possess at least one tyrosine residue within this motif (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). Conservation of the protein domain organization and key structural motifs of bat tetherin, despite significant amino acid sequence diversity, supports the present understanding of tetherin as a protein whose functions are mediated through structural rather than sequence-mediated interactions (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242983/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242983/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242983" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-13" class="first-child">Evolutionary hotspots in bat tetherin</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Bat tetherin genes are under significant positive selection</h3><p id="p-14">To assess the evolutionary pressure applied during the speciation of bats, a selection test was performed to analyze bat tetherin genes spanning ∼65 mya of Chiropteran history. The analysis revealed that overall, bat tetherin genes have been subjected to positive selection, with an average ratio of non-synonymous (dN) to synonymous (dS) mutations, dN/dS, of 1.125 over the Chiropteran phylogeny, with numerous specific positions subjected to a large degree of positive selection, with dN/dS values of 2.626 - 2.668 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). These sites are predominantly located in the transmembrane domain and the regions of the cytoplasmic and extracellular domains immediately adjacent to the transmembrane domain (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>). This analysis suggests the presence of evolutionary pressure exerted on tetherin, by viral antagonists or countermeasures that target residues in the tetherin, following the speciation of the Chiropteran family. The apparent ubiquity of tetherin among bats, in concert with significant sequence diversity and evidence of strong positive selection pressures operating on each species, supports the notion that tetherin plays a major role in the antiviral repertoire of bats.</p></div><div id="sec-5" class="subsection"><h3>Fruit bats possess unique structural isoforms of the tetherin protein generated through alternative splicing of a single tetherin gene</h3><p id="p-15">The expression of the fruit bat tetherin gene was initially assessed by a BLASTn search within the transcriptome of an Australian fruit bat, the Black flying fox, <em>Pteropus alecto</em>, using the human tetherin amino acid sequence as the search query. We identified 30 contigs matching the query (lowest E-value = 9.21 x 10<sup>−23</sup>). A single <em>P. alecto</em> homolog of tetherin, herein described as isoform A, homologous to human l-tetherin and the predicted X1 isoform (GenBank: XM_006904279), was identified. <em>P. alecto</em> Tetherin isoform A has a low level of primary amino acid sequence conservation (37%) compared to human l-tetherin (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) and other mammalian (28 - 47%) tetherin sequences homologous to human l-tetherin (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Despite the low amino acid sequence identity, the predicted secondary structures and protein domains are largely conserved between bats and other mammals (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), with all tetherin proteins containing the cytoplasmic (CD), transmembrane (TM), extracellular domains (ED), and the post-translationally cleaved GPI-anchor signal peptide (GSP). An exception to this is the tetherin B of sheep, which does not contain the GSP domain.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple sequence alignment (MSA) and protein domains of mammalian tetherin amino acid sequences. A. MSA reveals amino acid residues differing between human and P. alecto sequences which are indicated by the amino acid sequence conservation bar graph below the sequences. B. The protein domains of mammalian tetherin proteins are depicted as overlays of an MSA. Sequence conservation is indicated by the bar graph below the alignment. Protein domains are colour coded: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED (light blue), extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide; * a 7 AA exclusion in P. alecto tetherin isoforms B and C relative to isoform A is indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Multiple sequence alignment (MSA) and protein domains of mammalian tetherin amino acid sequences.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. MSA reveals amino acid residues differing between human and &lt;em&gt;P. alecto&lt;/em&gt; sequences which are indicated by the amino acid sequence conservation bar graph below the sequences. &lt;strong&gt;B&lt;/strong&gt;. The protein domains of mammalian tetherin proteins are depicted as overlays of an MSA. Sequence conservation is indicated by the bar graph below the alignment. Protein domains are colour coded: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED (light blue), extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide; * a 7 AA exclusion in &lt;em&gt;P. alecto&lt;/em&gt; tetherin isoforms B and C relative to isoform A is indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F2.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F2.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Multiple sequence alignment (MSA) and protein domains of mammalian tetherin amino acid sequences.</span><p id="p-16" class="first-child"><strong>A</strong>. MSA reveals amino acid residues differing between human and <em>P. alecto</em> sequences which are indicated by the amino acid sequence conservation bar graph below the sequences. <strong>B</strong>. The protein domains of mammalian tetherin proteins are depicted as overlays of an MSA. Sequence conservation is indicated by the bar graph below the alignment. Protein domains are colour coded: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED (light blue), extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide; * a 7 AA exclusion in <em>P. alecto</em> tetherin isoforms B and C relative to isoform A is indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To verify the expression of bat tetherin isoform A, a cDNA library was prepared from <em>P. alecto</em> spleen tissue, comprising a number of immune cell types, and assessed for the presence of transcripts matching that identified in the <em>P. alecto</em> transcriptome. Primers were designed to flank the open reading frame of tetherin isoform A, and amplicons were generated by PCR. Amplicons were then cloned into plasmid vectors to allow for DNA sequencing. This analysis revealed two additional isoforms of tetherin, isoforms B &amp; C. Isoform B was found to be homologous to the computationally predicted isoform X2 (GenBank: XM_015587122). Both isoforms B and C are predicted to contain structural differences relative to isoform A. They both possess a 7 AA exclusion in the middle of the extracellular domain while isoform C is additionally predicted to contain an alternative C-terminus, without the GSP domain (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). The absence of a GSP domain suggests that it is unlikely that parallel homodimers of isoform C would possess the capacity to tether viral particles to the cell.</p><p id="p-18">Mapping the tetherin cDNA sequences against the <em>P. alecto</em> genome identified a single tetherin gene located on Scaffold KB030270 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), between the MVB12A (multivesicular body subunit 12A) and PLVAP (plasmalemma vesicle associated protein) genes, similar to the human tetherin gene (Viewable in the Ensembl genome browser: <a href="http://www.ensembl.org/Homo_sapiens/Location/View?db=core;g=ENSG00000130303;r=19:17402886-17405683">http://www.ensembl.org/Homo_sapiens/Location/View?db=core;g=ENSG00000130303;r=19:17402886-17405683</a>). <em>P. alecto</em> Isoform B and C use an alternative splice acceptor site for the first intron, resulting in the 7 AA exclusion, while the distinct C-terminus of isoform C is a result of alternative mRNA splicing that incorporates exon 5 while excluding exon 4 used in isoform A and B (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping the tetherin gene to the P. alecto genome. A. The tetherin gene location and neighbouring genes, coloured purple, are depicted. Arrows indicate the orientation of each gene&#x2019;s protein coding domain sequence. Sequence numbers and gene orientations are relative to the beginning of the gene scaffold in the 5&#x2019; to 3&#x2019; direction. B The tetherin mRNA exons are depicted and colour coded: Isoform A, red; Isoform B, blue; Isoform C, green. Exons are mapped to scale against the tetherin gene. Sequence numbers are relative to the beginning of the gene. The 5&#x2019; and 3&#x2019; untranslated regions of the mRNA are coloured black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mapping the tetherin gene to the &lt;em&gt;P. alecto&lt;/em&gt; genome.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. The tetherin gene location and neighbouring genes, coloured purple, are depicted. Arrows indicate the orientation of each gene&#x2019;s protein coding domain sequence. Sequence numbers and gene orientations are relative to the beginning of the gene scaffold in the 5&#x2019; to 3&#x2019; direction. &lt;strong&gt;B&lt;/strong&gt; The tetherin mRNA exons are depicted and colour coded: Isoform A, red; Isoform B, blue; Isoform C, green. Exons are mapped to scale against the tetherin gene. Sequence numbers are relative to the beginning of the gene. The 5&#x2019; and 3&#x2019; untranslated regions of the mRNA are coloured black.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F3.medium.gif" width="440" height="111"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F3.medium.gif" width="440" height="111"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Mapping the tetherin gene to the <em>P. alecto</em> genome.</span><p id="p-19" class="first-child"><strong>A</strong>. The tetherin gene location and neighbouring genes, coloured purple, are depicted. Arrows indicate the orientation of each gene’s protein coding domain sequence. Sequence numbers and gene orientations are relative to the beginning of the gene scaffold in the 5’ to 3’ direction. <strong>B</strong> The tetherin mRNA exons are depicted and colour coded: Isoform A, red; Isoform B, blue; Isoform C, green. Exons are mapped to scale against the tetherin gene. Sequence numbers are relative to the beginning of the gene. The 5’ and 3’ untranslated regions of the mRNA are coloured black.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Vesper bats possess multiple tetherin genes that contain unique structural variations</h3><p id="p-20">To further investigate the diversity of tetherin in non-fruit bat species, we amplified the tetherin nucleotide sequences for the vesper bat species <em>Myotis ricketti</em> and <em>Miniopterus schreibersii</em> using primers designed on the basis of the predicted mRNA sequences (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3A</a>). These primers were used in a PCR analysis of cDNA samples generated from <em>M. ricketti</em> spleen tissue and a <em>M. schreibersii</em> kidney cell line to amplify the tetherin coding domain sequences (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3A</a>). Primers for <em>M. ricketti</em> were also used to analyse cDNA generated from an <em>M. macropus</em> kidney cell line as it was reasoned that given that these species belong to the same genus, primers designed for one might be capable of amplifying tetherin from the other. Following PCR, amplified DNA from <em>M. ricketti, M. macropus</em>, and <em>M. schreibersii</em>, were gel purified, cloned and sequenced.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242980/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242980/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242980" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-21" class="first-child">cDNA amplification and vector sequencing primers for fruit bat and vesper bat tetherins.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">These microbats were found to express various homologs of tetherin that include encoding of unique structural variations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4B</a>). The tetherin homolog of human l-tetherin predicted for <em>M. schreibersii</em> was detected (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Five unique tetherin variants, differing in their encoded primary amino acid sequences (tetherin A – E), were identified for <em>M. macropus</em>, two of which, tetherin A and B, were also detected in <em>M. ricketti</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). These tetherin variants encode three distinct homologs (tetherin A, C, D) of the human l-tetherin, sharing the same protein domains and secondary structures but differing in their primary amino acid sequence (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). Pairwise comparisons (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) reveal that the tetherin variants of <em>M. macropus</em> share between 84.2 – 92.0% amino acid sequence identity with each other, along with 44.6 – 53.5% identity with <em>P. alecto</em> tetherin isoform A, and 37.5 – 43.5% identity with human l-tetherin. The <em>M. ricketti</em> tetherin A and B share 100% amino acid sequence identity with their <em>M. macropus</em> counterparts. The tetherin of <em>M. schreibersii</em> shares between 34.6 and 49.7% sequence identity across all pairs.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple sequence alignment (MSA) and protein domains of fruit bat and vesper bat tetherin amino acid sequences. A. The percentage of sequence conservation is indicated by a bar plot below the MSA. Gap positions are shaded in grey. B. The protein domains of bat tetherin proteins are depicted as overlays of an MSA. Amino acid sequence conservation is indicated by the bar graph below the alignment. Protein domains are indicated and colour coded as follows: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED, (light blue) extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide. * The 7 AA exclusion in P. alecto tetherin isoform B and C relative to isoform A is indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Multiple sequence alignment (MSA) and protein domains of fruit bat and vesper bat tetherin amino acid sequences.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. The percentage of sequence conservation is indicated by a bar plot below the MSA. Gap positions are shaded in grey. &lt;strong&gt;B&lt;/strong&gt;. The protein domains of bat tetherin proteins are depicted as overlays of an MSA. Amino acid sequence conservation is indicated by the bar graph below the alignment. Protein domains are indicated and colour coded as follows: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED, (light blue) extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide. * The 7 AA exclusion in &lt;em&gt;P. alecto&lt;/em&gt; tetherin isoform B and C relative to isoform A is indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F4.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F4.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Multiple sequence alignment (MSA) and protein domains of fruit bat and vesper bat tetherin amino acid sequences.</span><p id="p-23" class="first-child"><strong>A</strong>. The percentage of sequence conservation is indicated by a bar plot below the MSA. Gap positions are shaded in grey. <strong>B</strong>. The protein domains of bat tetherin proteins are depicted as overlays of an MSA. Amino acid sequence conservation is indicated by the bar graph below the alignment. Protein domains are indicated and colour coded as follows: CD (blue), cytoplasmic domain; TM (purple), transmembrane domain; ED, (light blue) extracellular domain; GSP (orange), glycophosphatidylinositol signal peptide. * The 7 AA exclusion in <em>P. alecto</em> tetherin isoform B and C relative to isoform A is indicated.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparisons between human and bat tetherin variants. A. Myotis macropus tetherin A, C, and D. B. M. macropus tetherin B and E analysed separately to A, C and D to account for the inclusion of 60 AA in M. macropus tetherin A, C, and D that is otherwise removed following gap treatment of the alignment of M. macropus B and E. Protein sequences of human and bat tetherins were aligned and gaps were removed. The horizontal axis represents percentage identity between pairs of amino acid sequences, coloured with a white to red gradient indicating increasing similarity as red intensity increases. The vertical axis represents the number of amino acid residue identity differences between sequence pairs, coloured with a white to blue gradient indicating increasing difference as blue intensity increases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Pairwise comparisons between human and bat tetherin variants.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. &lt;em&gt;Myotis macropus&lt;/em&gt; tetherin A, C, and D. &lt;strong&gt;B&lt;/strong&gt;. &lt;em&gt;M. macropus&lt;/em&gt; tetherin B and E analysed separately to A, C and D to account for the inclusion of 60 AA in &lt;em&gt;M. macropus&lt;/em&gt; tetherin A, C, and D that is otherwise removed following gap treatment of the alignment of &lt;em&gt;M. macropus&lt;/em&gt; B and E. Protein sequences of human and bat tetherins were aligned and gaps were removed. The horizontal axis represents percentage identity between pairs of amino acid sequences, coloured with a white to red gradient indicating increasing similarity as red intensity increases. The vertical axis represents the number of amino acid residue identity differences between sequence pairs, coloured with a white to blue gradient indicating increasing difference as blue intensity increases.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F5.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F5.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Pairwise comparisons between human and bat tetherin variants.</span><p id="p-24" class="first-child"><strong>A</strong>. <em>Myotis macropus</em> tetherin A, C, and D. <strong>B</strong>. <em>M. macropus</em> tetherin B and E analysed separately to A, C and D to account for the inclusion of 60 AA in <em>M. macropus</em> tetherin A, C, and D that is otherwise removed following gap treatment of the alignment of <em>M. macropus</em> B and E. Protein sequences of human and bat tetherins were aligned and gaps were removed. The horizontal axis represents percentage identity between pairs of amino acid sequences, coloured with a white to red gradient indicating increasing similarity as red intensity increases. The vertical axis represents the number of amino acid residue identity differences between sequence pairs, coloured with a white to blue gradient indicating increasing difference as blue intensity increases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">An additional splice variant (isoform B) of <em>M. macropus</em> tetherin C was also identified. <em>M. macropus</em> tetherin C isoform B is predicted to have a C-terminal truncation and possesses only the cytoplasmic domain, transmembrane domain, and less than half of the extracellular domain (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). Accordingly, it is predicted to lack the GPI signal peptide required for the post-translational addition of a GPI-anchor, also found for <em>P. alecto</em> tetherin isoform C (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). The absence of a GPI anchor indicates that it is unlikely that homodimers of <em>M. macropus</em> tetherin C isoform B would possess the capacity to tether viral particles to the cell. <em>M. macropus</em> tetherin B and E were found to be structurally unique, possessing a large deletion of ∼60 amino acids in the extracellular coiled-coil domain (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>), compared to the other tetherin homologs, which has been shown to be critical for viral particle restriction (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). This deletion results in the exclusion of the conserved disulphide bond-forming cysteine residues, and the conserved asparagine-linked glycosylation sites, indicating that this form of tetherin is unlikely to form dimers or be glycosylated in the manner of human tetherin (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>). They are however predicted to possess all other tetherin domains including the transmembrane domain and GPI signal peptide, indicating that they may still be capable of tethering viral particles. <em>M. macropus</em> tetherin B and E share 92.1% amino acid sequence identity indicating they are the product of a relatively recent gene duplication (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-26">While there is no available genome for <em>M. macropus</em>, the most closely related publically accessible genome assemblies are for the relatively distantly related <em>M. lucifugus</em> and <em>M. davidii</em>, which diverged from <em>M. macropus</em> approximately 19 and 14 mya, respectively (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). For comparison, humans and chimpanzees diverged approximately 6.5 mya (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). A BLASTn analysis of the <em>M. macropus</em> tetherin nucleotide sequences against the <em>M. davidii</em> and <em>M. lucifugus</em> genomes revealed first that both species were too distantly related to <em>M. macropus</em> to match each <em>M. macropus</em> gene product specific to <em>M. davidii</em> or <em>M. lucifugus</em> gene regions. However, the <em>M. lucifugus</em> genome assembly contained a single gene scaffold (GL430608) which contained matches to all query <em>M. macropus</em> tetherin sequences, comprising seven potential <em>M. lucifugus</em> tetherin genes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>) and indicating the presence of a tetherin gene locus within this scaffold. The <em>M. lucifugus</em> tetherin gene locus was located between the MVB12A and PLVAP genes, as was found to be the case for the single <em>P. alecto</em> tetherin gene (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping the tetherin genes of Myotis macropus to the M. lucifugus genome. Tetherin sequences derived from M. macropus cDNA were mapped against the publically available genome of M. lucifugus (Myoluc2.0 genome assembly). A. The tetherin gene locus of M. lucifugus is contained within scaffold GL430608 between the PLVAP and MVB12A genes. B. M. macropus tetherin A, C, and D mapped against M. lucifugus tetherin region 6. C. M. macropus tetherin B and E mapped against M. lucifugus tetherin region 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mapping the tetherin genes of &lt;em&gt;Myotis macropus&lt;/em&gt; to the &lt;em&gt;M. lucifugus&lt;/em&gt; genome.&lt;/span&gt; Tetherin sequences derived from &lt;em&gt;M. macropus&lt;/em&gt; cDNA were mapped against the publically available genome of &lt;em&gt;M. lucifugus&lt;/em&gt; (Myoluc2.0 genome assembly). &lt;strong&gt;A&lt;/strong&gt;. The tetherin gene locus of &lt;em&gt;M. lucifugus&lt;/em&gt; is contained within scaffold GL430608 between the PLVAP and MVB12A genes. &lt;strong&gt;B&lt;/strong&gt;. &lt;em&gt;M. macropus&lt;/em&gt; tetherin A, C, and D mapped against &lt;em&gt;M. lucifugus&lt;/em&gt; tetherin region 6. &lt;strong&gt;C&lt;/strong&gt;. &lt;em&gt;M. macropus&lt;/em&gt; tetherin B and E mapped against &lt;em&gt;M. lucifugus&lt;/em&gt; tetherin region 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F6.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F6.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Mapping the tetherin genes of <em>Myotis macropus</em> to the <em>M. lucifugus</em> genome.</span><p id="p-27" class="first-child">Tetherin sequences derived from <em>M. macropus</em> cDNA were mapped against the publically available genome of <em>M. lucifugus</em> (Myoluc2.0 genome assembly). <strong>A</strong>. The tetherin gene locus of <em>M. lucifugus</em> is contained within scaffold GL430608 between the PLVAP and MVB12A genes. <strong>B</strong>. <em>M. macropus</em> tetherin A, C, and D mapped against <em>M. lucifugus</em> tetherin region 6. <strong>C</strong>. <em>M. macropus</em> tetherin B and E mapped against <em>M. lucifugus</em> tetherin region 2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The genome assembly of <em>M. davidii</em> also contained significant matches to <em>M. macropus</em> tetherin sequences, however these were spread across seven different gene scaffolds against which no <em>M. macropus</em> tetherin coding domain sequence could be matched in its entirety. For this reason, further genomic analysis of <em>Myotis</em> tetherin genes was confined to the <em>M. lucifugus</em> genome.</p><p id="p-29">The <em>M. lucifugus</em> tetherin gene locus (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>) contains seven potential genes that are defined by the presence of two or more exons mapping to the <em>M. macropus</em> tetherin cDNA. Three exons across tetherin gene regions 1, 2, and 6, possess all of the coding exons required to express homologs to <em>M. macropus</em> tetherin A, C, and D (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). Furthermore, <em>M. lucifugus</em> tetherin gene regions 2, and 3 possess all of the coding exons required to produce homologs to <em>M. macropus</em> tetherin B and E (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>). These data indicate that microbats of the genus <em>Myotis</em> possess a greater number and diversity of tetherin genes than any mammal reported to date.</p></div><div id="sec-7" class="subsection"><h3>Expression and cellular localisation of <em>P. alecto</em> tetherin</h3><p id="p-30">To evaluate the function of bat tetherin proteins, we analysed tetherin from two Australian bats, the fruit bat <em>P. alecto</em> and the vesper bat <em>M. macropus</em>. The coding sequences of the <em>P. alecto</em> tetherin isoforms A, B, and C, and <em>M. macropus</em> tetherin A and B, were cloned into mammalian expression plasmids. While numerous vesper bat tetherins were identified in this study, <em>M. macropus</em> tetherin A and tetherin B were selected for further analysis as tetherin A is a homolog of human l-tetherin and tetherin B was the most structurally unique, since it possessed a large deletion within the extracellular coiled-coil domain compared to tetherin A (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-31">All tetherin coding sequences were modified with the addition of a haemagglutinin (HA) tag sequence (N-SG<strong>YPYDVPDYA</strong>GS-C) to enable antibody-based detection of protein expression. In all cases the HA tag was inserted at the position immediately following the end of the coiled-coil region of the extracellular domain (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>) which is the equivalent location to that which has been previously utilized in the tagging of human tetherin (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">22</a>).</p><p id="p-32">To validate the capacity of each tetherin plasmid construct to express tetherin protein, mammalian HEK293T (293T) cells, which do not express tetherin in the absence of type I interferon stimulation (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>), were transfected with bat tetherin expression constructs. Cell lysates were extracted using a method specific for GPI-anchored proteins (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>) and tetherin expression was evaluated by non-reducing SDS-PAGE and Western blot analysis (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SDS-PAGE and Western blot analysis of the expression of modified bat tetherin constructs. A. Haemagglutinin (HA) tagged P. alecto tetherin isoforms A, B, and C (PA, PB, and PC, respectively). B. HA tagged M. macropus tetherin A and B (MA and MB, respectively). The left side of the membrane is displayed at low exposure while the right side is displayed at high exposure to enable clear images of the bands in each lane. All tetherin proteins were expressed in mammalian 293T cells and cell lysates were collected 24 h after transfection of expression plasmids. Tetherin proteins are present in dimeric (D) and monomeric (M) forms. Protein size scale is expressed in kDa." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SDS-PAGE and Western blot analysis of the expression of modified bat tetherin constructs.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. Haemagglutinin (HA) tagged &lt;em&gt;P. alecto&lt;/em&gt; tetherin isoforms A, B, and C (PA, PB, and PC, respectively). &lt;strong&gt;B&lt;/strong&gt;. HA tagged &lt;em&gt;M. macropus&lt;/em&gt; tetherin A and B (MA and MB, respectively). The left side of the membrane is displayed at low exposure while the right side is displayed at high exposure to enable clear images of the bands in each lane. All tetherin proteins were expressed in mammalian 293T cells and cell lysates were collected 24 h after transfection of expression plasmids. Tetherin proteins are present in dimeric (D) and monomeric (M) forms. Protein size scale is expressed in kDa.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F7.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F7.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">SDS-PAGE and Western blot analysis of the expression of modified bat tetherin constructs.</span><p id="p-33" class="first-child"><strong>A</strong>. Haemagglutinin (HA) tagged <em>P. alecto</em> tetherin isoforms A, B, and C (PA, PB, and PC, respectively). <strong>B</strong>. HA tagged <em>M. macropus</em> tetherin A and B (MA and MB, respectively). The left side of the membrane is displayed at low exposure while the right side is displayed at high exposure to enable clear images of the bands in each lane. All tetherin proteins were expressed in mammalian 293T cells and cell lysates were collected 24 h after transfection of expression plasmids. Tetherin proteins are present in dimeric (D) and monomeric (M) forms. Protein size scale is expressed in kDa.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34"><em>P. alecto</em> tetherin isoforms A and B were detected primarily as broad bands at the expected positions ∼56-75 kDa while isoform C was present as both dimers and monomers (∼32-35 kDa) (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). The presence of broad/multiple bands in close proximity to each other likely reflects variable levels of tetherin glycosylation as previously reported for human tetherin (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">22</a>). <em>M. macropus</em> tetherin A was also detected as a dimer at the expected size of ∼55 kDa while tetherin B was only detected as a monomer with a size of ∼26 kDa (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). These data show that all tetherin constructs are capable of expressing tagged tetherin proteins and that these proteins differ in their capacity to form the expected dimerised form of tetherin. Bat tetherin proteins that are homologous to human l-tetherin dimerise as expected, while the structurally unique tetherin proteins behave differently. In this regard, <em>P. alecto</em> tetherin isoform C generates similar amounts of dimers and monomers, while <em>M. macropus</em> tetherin B appears to exist exclusively as a monomer.</p><p id="p-35">Human tetherin localises to the plasma membrane at sites of virus budding, in addition to membranes of the trans-Golgi network and recycling compartments (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). <em>P. alecto</em> tetherin isoform A displays a similar cellular localisation pattern as human tetherin (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">8C</a>), while isoform C is predominantly localised in the cytoplasm (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8E</a>). <em>P. alecto</em> tetherin isoform B demonstrates a fluorescence pattern with features shared by both isoforms A and B (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8D</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of HA-tagged tetherin proteins. Human and Pteropus alecto tetherin isoforms A, B, and C in HEK293T cells. Tetherin localisation in cells was detected using anti-HA-tag rabbit monoclonal Ig and anti-rabbit AlexaFlour 488 secondary Ig. Nuclei are blue stained with Hoescht. Fixed and permeabilised cells were imaged on Nikon AR1 confocal microscope. HA, haemagglutinin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Expression of HA-tagged tetherin proteins.&lt;/span&gt; Human and &lt;em&gt;Pteropus alecto&lt;/em&gt; tetherin isoforms A, B, and C in HEK293T cells. Tetherin localisation in cells was detected using anti-HA-tag rabbit monoclonal Ig and anti-rabbit AlexaFlour 488 secondary Ig. Nuclei are blue stained with Hoescht. Fixed and permeabilised cells were imaged on Nikon AR1 confocal microscope. HA, haemagglutinin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F8.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F8.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <span class="caption-title">Expression of HA-tagged tetherin proteins.</span><p id="p-36" class="first-child">Human and <em>Pteropus alecto</em> tetherin isoforms A, B, and C in HEK293T cells. Tetherin localisation in cells was detected using anti-HA-tag rabbit monoclonal Ig and anti-rabbit AlexaFlour 488 secondary Ig. Nuclei are blue stained with Hoescht. Fixed and permeabilised cells were imaged on Nikon AR1 confocal microscope. HA, haemagglutinin.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Some, but not all tetherin proteins of <em>P. alecto</em> and <em>M. macropus</em> are capable of restricting the release of HIVΔVpu virus-like particles</h3><p id="p-37">The basic function of <em>P. alecto</em> and <em>M. macropus</em> tetherins to decrease viral particle release was tested against model HIV-1 virus-like particles (VLPs), which are known to be restricted by tetherin from non-human mammals including cats, sheep, and other bat species (<em>Hypsignathus monstrosus</em> and <em>Epomops buettikoferi</em>) (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). 293T cells were transfected with bat tetherin constructs in addition to plasmids expressing HIVΔVpu VLPs that do not express Vpu, an antagonist of human tetherin (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>). <em>P. alecto</em> tetherin isoforms A and B restricted the release of HIVΔVpu VLPs, while isoform C did not (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>). <em>M. macropus</em> tetherin A also restricted HIVΔVpu VLP release from 293T cells, however tetherin B was unable to inhibit HIVΔVpu VLP egress (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>). These data indicate that, with the exception of <em>P. alecto</em> tetherin isoform C and <em>M. macropus</em> tetherin B, tagged bat tetherin proteins are functionally capable of restricting the release of HIV-1 particles from mammalian cells in the absence of Vpu.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bat tetherin restricts the release of HIV-1 virus-like particles (VLPs). A. Pteropus alecto tetherin isoforms A (PA), B (PB), and C (PC), cotransfected with a HIV&#x394;Vpu construct. B. Myotis macropus tetherin A (MA) and tetherin B (MB) cotransfected with a HIV&#x394;Vpu construct. Mammalian 293T cells were cotransfected with 200ng of HIV&#x394;Vpu plasmid expression construct encoding the HIV Gag-Pol polyprotein, which generates HIV-1 VLPs that do not include the tetherin antagonist Vpu, and 0 &#x2013; 200 ng of tetherin plasmid expression vectors. VLPs were harvested at 48 h and concentrated by ultracentrifugation through a sucrose gradient, and VLP and cell lysates were subjected to SDS-PAGE and Western blot analysis. HIV-1 VLPs were detected with a mouse anti-p24 primary antibody and goat anti-mouse Alexa-Fluor 680 fluorophore-conjugated fluorescent secondary antibody. Representative Western blots are shown. The extent of VLP restriction was quantitated by densitometric analysis of Western blots comparing the relative ratios of VLPs present in the cell culture supernatants and cell lysates from N=3 (P. alecto) or N=2 (M. macropus) independent assays. Error bars represent the standard error of the mean. HIV-1, human immunodeficiency virus type 1; SN, cell culture supernatant; CL, cell culture lysate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-771610552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bat tetherin restricts the release of HIV-1 virus-like particles (VLPs).&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;. &lt;em&gt;Pteropus alecto&lt;/em&gt; tetherin isoforms A (PA), B (PB), and C (PC), cotransfected with a HIV&#x394;Vpu construct. &lt;strong&gt;B&lt;/strong&gt;. &lt;em&gt;Myotis macropus&lt;/em&gt; tetherin A (MA) and tetherin B (MB) cotransfected with a HIV&#x394;Vpu construct. Mammalian 293T cells were cotransfected with 200ng of HIV&#x394;Vpu plasmid expression construct encoding the HIV Gag-Pol polyprotein, which generates HIV-1 VLPs that do not include the tetherin antagonist Vpu, and 0 &#x2013; 200 ng of tetherin plasmid expression vectors. VLPs were harvested at 48 h and concentrated by ultracentrifugation through a sucrose gradient, and VLP and cell lysates were subjected to SDS-PAGE and Western blot analysis. HIV-1 VLPs were detected with a mouse anti-p24 primary antibody and goat anti-mouse Alexa-Fluor 680 fluorophore-conjugated fluorescent secondary antibody. Representative Western blots are shown. The extent of VLP restriction was quantitated by densitometric analysis of Western blots comparing the relative ratios of VLPs present in the cell culture supernatants and cell lysates from N=3 (&lt;em&gt;P. alecto&lt;/em&gt;) or N=2 (&lt;em&gt;M. macropus&lt;/em&gt;) independent assays. Error bars represent the standard error of the mean. HIV-1, human immunodeficiency virus type 1; SN, cell culture supernatant; CL, cell culture lysate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F9.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F9.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.08.031203/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9.</span> <span class="caption-title">Bat tetherin restricts the release of HIV-1 virus-like particles (VLPs).</span><p id="p-38" class="first-child"><strong>A</strong>. <em>Pteropus alecto</em> tetherin isoforms A (PA), B (PB), and C (PC), cotransfected with a HIVΔVpu construct. <strong>B</strong>. <em>Myotis macropus</em> tetherin A (MA) and tetherin B (MB) cotransfected with a HIVΔVpu construct. Mammalian 293T cells were cotransfected with 200ng of HIVΔVpu plasmid expression construct encoding the HIV Gag-Pol polyprotein, which generates HIV-1 VLPs that do not include the tetherin antagonist Vpu, and 0 – 200 ng of tetherin plasmid expression vectors. VLPs were harvested at 48 h and concentrated by ultracentrifugation through a sucrose gradient, and VLP and cell lysates were subjected to SDS-PAGE and Western blot analysis. HIV-1 VLPs were detected with a mouse anti-p24 primary antibody and goat anti-mouse Alexa-Fluor 680 fluorophore-conjugated fluorescent secondary antibody. Representative Western blots are shown. The extent of VLP restriction was quantitated by densitometric analysis of Western blots comparing the relative ratios of VLPs present in the cell culture supernatants and cell lysates from N=3 (<em>P. alecto</em>) or N=2 (<em>M. macropus</em>) independent assays. Error bars represent the standard error of the mean. HIV-1, human immunodeficiency virus type 1; SN, cell culture supernatant; CL, cell culture lysate.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-39">To advance our understanding of the differences between the innate antiviral defenses of bats relative to other mammals, we analysed the genes and expressed transcripts of tetherin from diverse bat genera and species within the family Chiroptera. We found that in all but one species, bats possess and express a tetherin protein homologous to the long isoform of human tetherin (l-tetherin). In addition, we found that <em>P. alecto</em> expresses three isoforms from a single tetherin gene, and that vesper bats (genus <em>Myotis</em>) encode five, and possibly as many as seven, distinct tetherin genes.</p><p id="p-40">The one bat species, <em>T. melanopogon</em>, that lacked the human l-tetherin homolog, possessed an equivalent of the short isoform of human tetherin (s-tetherin), an observation also reported for cats (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). In a study of feline tetherin, short isoform-exclusivity was found to improve viral restriction and decrease sensitivity to tetherin antagonism by HIV-1 Vpu when compared to an engineered long isoform version of feline tetherin (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>). Conversely, the feline immunodeficiency virus (FIV), which is adapted to the feline host, is not restricted by either long or short isoforms of feline tetherin (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a>).</p><p id="p-41">In humans, l-tetherin acts as a virus sensor which induces NF-κB signaling in contrast to s-tetherin, which lacks the dual tyrosine motif required for eliciting this innate immune response (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>). The findings presented here suggest that if bat tetherin proteins are confirmed to similarly mediate cytoplasmic domain-mediated immune signaling, then the Black-bearded Tomb bat, which only encodes a homolog of s-tetherin, would be predicted not to mediate this effect.</p><p id="p-42">Bat tetherin amino acid sequences were found to be highly variable. The predicted protein lengths range from 177 to 221 AA (human l-tetherin is 180 AA) and of these, only 23 amino acid residues were found to be conserved in 100% of the 27 bat tetherin sequences analysed (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>). Among these are the structurally important cysteine and asparagine residues which are responsible for tetherin dimerisation and glycosylation, respectively (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). The dual-tyrosine motif, responsible for mediating viral particle endocytosis and immune signaling (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>), was found to exist as a variable Y|C·x·Y|H motif across the bat tetherin variants analysed. All bats maintained at least one of the two tyrosine residues. This observation is significant because mutational studies of human tetherin have demonstrated that the dual tyrosines provide redundancy for both the endocytic and signaling activities, which are maintained so long as either tyrosine is present (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>).</p><p id="p-43">To understand the evolutionary selective pressures on tetherin genes across bat species, a selection test was performed that revealed that bat tetherin genes are under strong positive selection with amino acid positions in and around the transmembrane domain being the region under the strongest selective pressure (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1A</a>). This is in agreement with previous reports that primate tetherin possess multiple sites under positive selection in this same region (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). The driver of positive selection in the case of primate tetherin is antagonism by viral counter-measures including HIV-1 Vpu (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>). Venkatesh et al. (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>) demonstrated that the configuration that tetherin dimers adopt during viral particle retention primarily consists of the GPI-anchor being embedded in the viral envelope and the transmembrane domain remaining attached to the cellular membrane. If this paradigm holds true for bat tetherin, then it follows that the tetherin-cell membrane interface is the major site of tetherin antagonism in bats and it would be reasonable to speculate that the drivers of this selection are viral antagonists analogous in the mode of interaction, if not structure or function, to lentiviral tetherin antagonists.</p><p id="p-44">We amplified tetherin from spleen-derived cDNA of the Australian fruit bat, <em>P. alecto</em>, and confirmed the expression of the two computationally predicted splice variants (isoforms A [X1] and B [X2]), and additionally identified the expression of a third isoform of tetherin, isoform C. Mapped against the <em>P. alecto</em> genome, all three isoforms were found to be derived from the alternative splicing of a single tetherin gene (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). <em>P. alecto</em> tetherin isoform B was found to possess a 7 AA exclusion within the extracellular coiled-coil domain relative to isoform A, while isoform C is predicted to harbor the same 7 AA exclusion and an alternative C-terminus, that lacks the GSP predicted to be present in tetherin isoforms A and B which is necessary for the post-translational addition of the GPI-anchor. This is important because studies of human tetherin have demonstrated that the presence of a GPI-anchor is essential for restricting the release of viral particles (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>). Sheep and cows have been reported to possess a duplication of the tetherin gene (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>). In sheep, the duplicate tetherin, named tetherin B, similarly does not encode a GSP, and studies of sheep tetherin B function reveal that while it is capable of limited restriction of VLPs, it is significantly less potent than sheep tetherin A (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>). The mechanism of its function is unknown and its C-terminal amino acid sequence is entirely dissimilar from that of <em>P. alecto</em> tetherin isoform C. One possible function of additional isoforms of tetherin is to expand the viral target range of bats by undefined mechanisms that are active in the absence of a GPI anchor.</p><p id="p-45">Intriguingly, one of the non-transmembrane domain-associated sites under strong positive selection across tetherin from bats is located in the middle of the extracellular domain. This is the site that distinguishes the <em>P. alecto</em> tetherin isoforms A and B, with the 7 AA exclusion at this location in isoform B. This suggests that the expression of <em>P. alecto</em> tetherin isoform B as an alternative to isoform A, and possibly also <em>M. macropus</em> tetherin B and E, which differ from tetherin A through a large 60 AA deletion in the same region, is to express a form of tetherin that does not include a sequence motif that appears to be a possible site of viral antagonism. This hypothesis may be supported by future experiments aimed at identifying viral antagonists of <em>P. alecto</em> tetherin isoform A that are ineffective against isoform B.</p><p id="p-46">The microbat <em>M. macropus</em> was found to express five different tetherin genes, tetherin A, B, C, D and E, three of which, A, C, and D, encode homologs of human l-tetherin, while <em>M. macropus</em> tetherin B and E possess a large 60 AA deletion within the extracellular coiled-coil domain (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>). Mapping of these genes against the genome assembly of the relatively distantly related <em>M. lucifugus</em> indicates that vesper bats of the genus <em>Myotis</em> may possess as many as seven tetherin genes (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>). The presence of multiple tetherin genes suggests that these bats have expanded and diversified their use of these antiviral restriction factors relative to other mammals, supporting the hypothesis of differential antiviral gene evolution in bats.</p><p id="p-47">The expression of the <em>P. alecto</em> tetherin isoforms A, B, and C, and <em>M. macropus</em> tetherin A and B, reveals differences in their relative capacities to form homodimers under our assay conditions (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>). This observation is notable because the capacity of tetherin to form cysteine-linked dimers is required for the restriction of HIV-1 viral particles (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>) which we chose as the VLPs against which bat tetherin proteins were functionally validated for inhibiting viral particle release. In contrast to restriction of HIV-1, Sakuma et al. (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>) have reported that tetherin dimerisation is not required for the restriction of arenaviruses and filoviruses, suggesting that the need for dimerisation is virus/VLP-dependent.</p><p id="p-48">The bat tetherins that we assessed for function which were most similar to human l-tetherin, <em>P. alecto</em> tetherin isoforms A and B, and <em>M. macropus</em> tetherin A, were all found to predominantly form dimers. This was shown by non-reducing SDS-PAGE size separation and subsequent Western blot analysis in which a detectable signal was observed almost exclusively at the expected size of tetherin dimers (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>). In contrast, <em>P. alecto</em> tetherin isoform C was present in similar amounts in both the dimeric and monomeric forms (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7A</a>). This observation is surprising because <em>P. alecto</em> tetherin isoform C contains all of the conserved cysteine residues and the same extracellular coiled-coil domain as isoforms A and B. The distinct C-terminal region in isoform C is the result of an alternative splicing event that also causes the loss of the final three amino acid residues of the coiled-coil region possessed by isoforms A and B. Additionally, the inserted HA tag is located in the position immediately following then coiled-coil domain. It is possible that the absence of the three terminal amino acid residues in the coiled-coil domain of isoform C relative to isoform A and B accounts for the difference in relative extents of dimerisation. Alternatively, the observed extent of dimerisation of isoform C may be an artifact of the method used to process the samples for analysis and may not be reflective of the extent of dimerisation <em>in situ</em>. It is also important to note that we cannot currently rule out the possibility that the insertion of the HA tag in between the coiled-coil domain and the alternative C-terminal sequence might be responsible for the observed reduction in dimerisation. <em>M. macropus</em> tetherin B was observed exclusively as a monomer (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7B</a>). The monomeric exclusivity of <em>M. macropus</em> tetherin B can be explained by the loss of all conserved cysteine residues necessary for dimerisation within the 60 AA deletion in the extracellular coiled-coil domain of tetherin B (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-49">Given the differing extent of homodimer formation for each bat tetherin and the reports that dimerisation is important for the restriction of some viruses and not required for others (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">46</a>, <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a>), it is likely that these differences might affect the extent by which each tetherin is capable of restricting various VLPs. Tetherin function was validated through an assessment of the capacity of <em>P. alecto</em> tetherin isoforms A, B, and C, and <em>M. macropus</em> tetherins A and B to restrict the release of HIVΔVpu VLPs. <em>P. alecto</em> tetherin isoforms A and B were capable of restricting the release of HIVΔVpu VLPs, in contrast to isoform C which lacked restriction activity (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9A</a>). <em>M. macropus</em> tetherin A restricted HIVΔVpu VLP release, while tetherin B did not (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9B</a>). These findings are consistent with the expected effects of tetherin dimerisation on the restriction of HIVΔVpu VLPs (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">46</a>).</p><p id="p-50">Previous reports on the necessity of the tetherin GPI anchor for inhibition of viral particle release (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>) indicates that the lack of a GPI anchor on isoform C would likely result in an inability to restrict viral egress from the host cell. However, sheep tetherin B, which does not possess a GPI-signal peptide (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>) is capable of limited restriction of betaretroviral VLPs through an unknown mechanism (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">43</a>). This may indicate that isoform C either has an antiviral range that does not include HIVΔVpu VLPs, or that it does not have any antiviral function at all.</p></div><div class="section" id="sec-10"><h2 class="">Conclusions</h2><p id="p-51">Tetherin variants of the fruit bat <em>P. alecto</em> and the vesper bat <em>M. macropus</em> are functionally capable of restricting the release of retroviral HIVΔVpu VLPs in a mammalian cell culture system. These bats also express unique forms of tetherin that do not conform to the standard tetherin protein structure observed in tetherin proteins of other mammals and which were not observed to restrict the release of HIVΔVpu VLPs, raising questions regarding their possible antiviral range, mechanism of action, and <em>in vivo</em> function. Bat tetherin genes are under strong positive selection, indicating an ongoing process of evolution involving tetherin targets and viral antagonists. This evolutionary pressure has resulted in the expansion and diversification of tetherin genes in vesper bats of the genus <em>Myotis</em>, and the emergence of unique splice variants of tetherin in the fruit bat <em>P. alecto</em>, supporting the hypothesis of differential antiviral gene evolution in bats.</p></div><div class="section" id="sec-11"><h2 class="">Methods and Materials</h2><div id="sec-12" class="subsection"><h3>Sequence read archive (SRA) BLAST analysis</h3><p id="p-52">To predict the tetherin sequences of other bat species, the <em>P. alecto</em> tetherin isoform A coding domain nucleotide sequence (549 nt) was used as the search query for a separate BLASTn analysis of each of the publically accessible sequence read archives (SRA) of bat transcriptomes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This approach allowed the individual assembly of a tetherin homolog from each bat species for which a transcriptome sequence read archive was available. The BLASTn analyses were performed using the online SRA Nucleotide BLAST tool (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). The program selection was optimized for ‘somewhat similar sequences’ (BLASTn) and the selected algorithm parameters were as follows: Max target sequences = 1000; Expect threshold = 1.0×10<sup>−10</sup>; Word size = 7; Max matches in a query range = 21; Match/Mismatch scores = 2,-3; Gap costs of existence 5 and extension 2; and no filtering or masking was selected. Matching reads were downloaded and assembled using the Assemble Sequences tool in CLC Genomics Workbench 8.0 (CLC; Qiagen). The consensus sequence of the assembly was then used as the search query for a second SRA BLAST analysis with the same parameters as the first search with the following exceptions: Program selection was optimised for ‘highly similar sequences’ (megablast); Expect threshold = 1.0 x 10<sup>−20</sup>; Word size = 28; Match/Mismatch scores = 1,-2; and linear gap costs of existence and extension. Matching reads were downloaded and assembled in the same manner, and the assembled consensus sequence was used in a third SRA BLAST using the same parameters as the second. This process was iteratively repeated for the assembled consensus sequence of each bat tetherin until it extended through the tetherin coding domain in both directions into the 5’ and 3’ untranslated regions, respectively demarcated by the locations of the start methionine and stop codon.</p></div><div id="sec-13" class="subsection"><h3>Evolutionary selection test</h3><p id="p-53">To determine if evolutionary selective pressures were being applied to bat tetherin, a selection test was performed. To detect the positively selected sites among bat tetherin sequences, a maximum likelihood (ML) phylogeny was generated with CODEML implemented in PAML4 software (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). The input tree was generated based on a pre-existing bat species tree (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>) as well as cytb phylogeny derived from MEGA6 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>) under the GTR+I+G nucleotide substitution model. The coding sequence alignments were fit to the NSsites models allowing (M8; positive selection model) or disallowing (M8a; null model) positive selection. Models were compared using a chi-squared test (degrees of freedom = 2) on twice the difference of likelihood values to derive P-values. The codon frequency model F3×4 was used. In cases where a significant difference (P &lt; 0.01) between M8a versus M8 was detected, the Bayes Empirical Bayes (BEB) analysis was used to identify codons with ω (dN/dS) &gt; 1, reporting values with posterior probability &gt; 0.99 as high significance or 0.95 - 0.99 as moderate significance.</p></div><div id="sec-14" class="subsection"><h3>Identification of protein domains, structures, and motifs</h3><p id="p-54">To confirm the predicted tetherin nucleotide sequences and identify functional domains within bat tetherin protein sequences, translated coding domains were compared against known tetherin amino acid sequences including the human l-tetherin and the <em>P. alecto</em> tetherin isoform A (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). For all tetherin amino acid sequences, secondary structures were determined using the Predict Secondary Structure tool in CLC. The presence of cytoplasmic, transmembrane, and extracellular domains were determined using a hidden Markov model in TMHMM 2.0 (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). GPI signal sequences were determined using the online tool PredGPI (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>) (<a href="http://gpcr2.biocomp.unibo.it/gpipe/index.htm">http://gpcr2.biocomp.unibo.it/gpipe/index.htm</a>). Pairwise comparisons between all tetherin sequences was conducted using the Create Pairwise Comparison tool in CLC. Multiple sequence alignments (MSA) of predicted and known tetherin amino acid and nucleotide sequences were performed using MUSCLE v3.8 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>) with the following parameters: Maximum iterations = 16, Find diagonals = No.</p></div><div id="sec-15" class="subsection"><h3>Transcriptome and contig analysis</h3><p id="p-55">Approval for the use of bat tissue was granted by the Australian Animal Health Laboratories Animal Ethics committee (Protocol AEC1281). The <em>P. alecto</em> transcriptome is accessible through the NCBI Sequence Read Archive (<a href="http://www.ncbi.nlm.nih.gov/Traces/sra/">http://www.ncbi.nlm.nih.gov/Traces/sra/</a>) [SRA: SRP008674].</p><p id="p-56">To identify the homologs of tetherin in <em>P. alecto</em>, the human l-tetherin protein sequence [UniProt: Q10589-1] was used as a query in a tBLASTn analysis of the transcriptome of <em>P. alecto</em>. To identify assembled sequences (contigs) representing mRNA transcripts of interest, local tBLASTn analyses of the transcriptome of <em>P. alecto</em> were conducted with CLC using the following parameters: BLOSUM62 matrix, word size = 3, E-values &lt; 1 x10<sup>−12</sup>, gap costs of existence 11, extension 1, with no filtering of regions of low complexity.</p></div><div id="sec-16" class="subsection"><h3>cDNA analysis</h3><p id="p-57">To amplify tetherin nucleotide sequences from bat cDNA, polymerase chain reaction (PCR) assays were performed using cDNA generated from <em>P. alecto</em> and <em>M. ricketti</em> spleen tissue, and <em>M. macropus</em> and <em>M. schreibersii</em> kidney cell lines, using various combinations of forward and reverse primers (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3A</a>). Primers were designed using the tetherin predictions identified in the contig and SRA analyses. Primers were designed to bind to the 5’ and 3’ untranslated regions of bat cDNA such that the full CDS could be amplified. Bat capture, tissue collection and RNA extraction was conducted as previously reported (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>) with the exception that RNAlater (Ambion) preserved spleen tissue from four male adult bats was pooled before tissue homogenisation and followed with total RNA extraction with the Qiagen RNeasy Mini kit with on-column extraction of genomic DNA with DNase I. Total RNA was reverse transcribed into cDNA with the Qiagen Omniscript reverse transcriptase according to the manufacturer’s protocol with the exception that the reaction contained 100 ng/μl total RNA, 1 μM oligo-dT18 (Qiagen) and 10 μM random hexamers (Promega).</p><p id="p-58">All PCR amplification assays were performed using the Roche FastStart High Fidelity PCR system (Cat # 04738292001) with an annealing temperature gradient of 54°C to 64°C in 2°C increments. Each reaction was made up to a total of 20 μl, containing 1 unit of polymerase, 2 ng of total cDNA, and 8 pmol of each primer. All other parameters for PCR amplification were performed according to the manufacturer’s protocol.</p><p id="p-59">Amplicons from PCR reactions were analysed by agarose gel electrophoresis (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>). Individual DNA bands were physically excised from the gel. DNA was purified from the gel fragments using the Wizard SV Gel and PCR Clean Up kit (Promega, Fitchburg, USA) according to the manufacturer’s protocol.</p><p id="p-60">To further analyse the PCR amplified DNA, each DNA fragment was blunt-end ligated into the pCR2.1-TOPO-TA or pCR-Blunt-II-TOPO plasmid vector (Invitrogen, Waltham, USA). Ligation was performed using the TOPO TA or Zero Blunt TOPO PCR Cloning Kit (Invitrogen) according to the manufacturer’s instructions and plasmids were transformed into Top10 <em>E. coli</em> using a standard heat-shock method (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>). Plasmids were purified from these cultures using a Wizard Plus SV Miniprep DNA Purification kit (Promega). All inserted sequences were confirmed by Sanger sequencing using M13 forward and reverse primers (M13F and M13R; <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3B</a>).</p></div><div id="sec-17" class="subsection"><h3>Genome Mapping</h3><p id="p-61">To identify tetherin genes within the genomes of <em>P. alecto</em> and <em>M. macropus</em>, local BLASTn analyses were performed using CLC. For these analyses the <em>P. alecto</em> and <em>M. macropus</em> tetherin sequences were used as query sequences against the <em>P. alecto</em> genome [NCBI: PRJNA171993] and <em>M. lucifugus</em> genome [NCBI: GCF_000147115]. BLASTn was performed using the following parameters: Word size = 11, E-values &lt; 1 x10<sup>−3</sup>, gap costs of existence 5, extension 2, with no filtering of regions of low complexity. Genes were delineated as beginning and ending at the first and last nucleotide of the contig query sequence. Exons were delineated as consisting of the nucleotide regions within the gene mapping to the tetherin cDNA sequences, bordered by the canonical 5’-AG and 3’-GT dinucleotide motifs (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>). The 5’ and 3’ untranslated regions were defined as the regions upstream of the start methionine and downstream of the stop codon of the CDS, respectively. Contig sequences were mapped against gene scaffolds through a local BLASTn analysis using CLC Genomics Workbench with default settings. The nucleotide sequence preceding the first exon of each gene was assessed for the presence of promoter motifs.</p><p id="p-62">Neighbouring genes were identified by BLAT (BLAST-like alignment tool) analysis (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>) of the bordering nucleotide sequences upstream and downstream of RF genes using the Ensembl genome database BLAT tool (<a href="http://asia.ensembl.org/Multi/Tools/Blast?db=core">http://asia.ensembl.org/Multi/Tools/Blast?db=core</a>).</p></div><div id="sec-18" class="subsection"><h3>Generation of tagged tetherin constructs for expression in mammalian cells</h3><p id="p-63">To enable detection of tetherin expression, the <em>P. alecto</em> tetherin isoforms A, B, and C, and <em>M. macropus</em> tetherin A and B were genetically modified with the insertion of nucleotide sequences encoding the haemaggultinin (HA) antibody-binding epitopes. Tetherin sequences were modified through a 2-step PCR process. PCR reactions were performed using the Roche FastStart HighFidelity PCR kit according to the manufacturer’s recommendations.</p><p id="p-64">To express tetherin proteins in a mammalian cell culture system the tagged tetherin inserts were sub-cloned from the pCR2.1-TOPO (<em>P. alecto</em> tetherin isoforms A, B, and C) and pCR-Blunt-II-TOPO (<em>M. macropus</em> tetherin A and B) vectors into pcDNA3.1 mammalian expression vectors. The HA-tagged tetherin constructs contained terminal enzyme restriction sites. <em>P. alecto</em> tetherin constructs contained <em>Xho</em>I and <em>Xba</em>I sites at their 5’ and 3’ ends, respectively. <em>M. macropus</em> tetherin constructs contained an <em>Eco</em>RI site at each end. These sites, which are also present in the pcDNA3.1 vector, were used for digestion-ligation transfer of the HA-tagged tetherin sequences into pcDNA3.1. The ligation reaction products were transformed into Top10 <em>E. coli</em> and plasmid clones were purified from <em>E. coli</em> colonies using the method described above. All inserted sequences were verified by Sanger sequencing using the pcDNA3.1 vector sequencing primers, T7F forward and BGHR reverse (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3B</a>).</p></div><div id="sec-19" class="subsection"><h3>Expression, extraction, and detection of tetherin in a mammalian cell culture system</h3><p id="p-65">To determine if the tetherin plasmids can express tetherin protein in a mammalian cell culture system, adherent human embryonic kidney 293T (293T) cells (NIH AIDS Research and Reference Reagent Program, Division of AIDS, NIAID, NIH) were transfected with each tetherin construct with protein expression determined by Western blot analysis.</p><p id="p-66">293T cells were maintained in Dulbecco’s modified Eagle medium (DMEM-10, Thermo) enriched with heat-inactivated fetal calf serum (100ml/L; Invitrogen), glutamine (292 μg/ml; Invitrogen), and the antibiotics penicillin (100 units/ml; Invitrogen) and streptomycin (100 units/ml; Invitrogen) (DMEM-10). Cells were incubated at 37°C with 5% CO<sub>2</sub>.</p><p id="p-67">The expression analyses were performed in 6-well plates. Each well was seeded with 3.0×10<sup>5</sup> cells/well in 2 ml of DMEM-10. Cells were transfected when wells had reached 50-60% confluency. The tetherin constructs analysed are listed in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. Each plasmid construct was transfected into duplicate wells at 2 μg/well using the transfection reagent Lipofectamine 2000 (Thermo) according to the manufacturer’s protocol.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242977/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242977/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-68" class="first-child">Primers used for the generation of haemagglutinin-tagged <em>Pteropus alecto</em> and <em>Myotis macropus</em> tetherin expression constructs through a 2-Step PCR process.</p><div class="sb-div caption-clear"></div></div></div><p id="p-69">The following tetherin extraction protocol is adapted from Doering et al. (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>). To extract tetherin from transfected cells the supernatant in each well was discarded and cells were physically resuspended by pipetting in 500 μl of ice-cold Tris-buffered saline (TBS; 50 mM Tris-Cl, 150 mM NaCl, pH 7.5) and transferred into 1.5 ml tubes. To lyse the cells, 100 μl of pre-condensed 2% Triton X-114 (Sigma-Aldrich, St. Louis, USA) was added to the cell suspension followed by incubation on ice for 15 min at 4°C. The duplicate samples were then combined into the same tube and centrifuged at 10,000 x <em>g</em> for 4 min. The supernatant of each sample was transferred to a new tube and warmed to 37°C. Samples were then centrifuged at 10,000 x <em>g</em> for 10 min to separate the samples into two fractions; the upper, detergent-depleted fraction containing hydrophilic, soluble proteins and the lower, detergent-enriched fraction containing hydrophobic, transmembrane- and GPI-anchored proteins, including tetherin. The upper fractions of each sample were carefully aspirated by pipetting and discarded. The protein content of each sample was then purified and concentrated by trichloracetic acid (TCA) precipitation. A 25% volume of 100% (w/v) TCA was added to each sample (i.e. for sample volumes estimated to be 200 μl, 50 μl of TCA was added) and incubated on ice for 10 min. Samples were centrifuged at 20,000 x <em>g</em> for 5 min. The supernatant of each sample was then discarded and the protein pellets were washed with 200 μl of ice cold acetone followed by centrifugation at 20,000 x <em>g</em> for 5 min. The acetone wash was repeated once. The protein pellet was then dried on at 95°C for 5 min. Protein pellets were then resuspended in 20 μl of 4X SDS-PAGE sample buffer (Sodium dodecyl sulphate polyacrylamide gel electrophoresis sample buffer; 40% glycerol, 8% SDS, 240 mM Tris-Cl pH 6.8, 5% β-mercaptoethanol, 0.04% bromophenol blue).</p><p id="p-70">Protein samples were analysed by size-based separation through SDS-PAGE and visualisation using Western blot analysis (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">57</a>). For the SDS-PAGE, 12% acrylamide running gel and 4% stacking gel solutions were used. For the Western blot analysis, the primary antibody solution contained a 1/1000 dilution of a monoclonal rabbit anti-HA antibody (C29F4, Cell Signaling Technology, Danvers, USA) in TBS containing 0.1% Tween-20, and the secondary antibody solution contained a 1/10,000 dilution of a polyclonal goat anti-rabbit Alexa-Fluor 488 fluorophore-conjugate secondary antibody (Thermo), in PBST. To visualise fluorescent antibody-bound tetherin proteins, membranes were scanned using the Odyssey Imaging System (LI-COR Biosciences, Lincoln, USA) at wavelengths of 600 and 800 nm, using the default software settings.</p></div><div id="sec-20" class="subsection"><h3>Fluorescence microscopy</h3><p id="p-71">To visualise tetherin localization within cells, 500 ng of plasmids encoding HA-tagged human l-tetherin and <em>P. alecto</em> tetherin isoforms A, B, and C were transfected into HEK293T cells seeded on glass coverslips as described above. At 48 h post transfection, cells were fixed with 4% paraformaldehyde (Sigma) in PBS for 10 min at room temperature, and then permeablised in 0.2% Triton X-100 in PBS for 5 min at room temperature. Tetherin localisation in cells was detected by staining cells with anti-HA-tag rabbit monoclonal IgG (Thermo) diluted in 0.2% Triton X-100, 3% BSA in PBS for 1 h at room temperature. Cells were subsequently stained with anti-rabbit AlexaFlour 488 secondary IgG (Thermo) diluted in 0.2% Triton X-100, 3% BSA in PBS for 20 min at room temperature. Nuclei are blue stained with Hoechst 33342 (Thermo) according to the manufacturer’s instructions. Coverslips were mounted onto glass slides with ProLong Gold Antifade Mountant (Thermo), then imaged on a Nikon AR1 confocal microscope and analysed with ImageJ (NIH) software.</p></div><div id="sec-21" class="subsection"><h3>Functional validation of tetherin activity</h3><p id="p-72">To assess the effect of bat tetherin on the release of virus-like particles (VLPs), each tagged bat tetherin construct was co-transfected into 293T cells alongside the plasmid pCRV1-NLgagpolΔVpu, which expresses HIV NL4.3 Gag-polΔVpu protein [NCBI: P12497.4], capable of assembling into VLPs, kindly provided by Paul Bieniasz (Aaron Diamond AIDS Research Center, Rockefeller University) (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>).</p><p id="p-73">Transfections were performed as described above, with the following modifications: 12-well plates were seeded with 1.5×10<sup>5</sup> cells/well in a total volume of 1 ml of DMEM-10, DNA-Lipofectamine mixtures for each well contained 1 μl of Lipofectamine, and a total of 400 ng of DNA comprised of 200 ng of VLP plasmid DNA, and 0 – 200 ng of tetherin-encoding plasmid DNA, made up to a total mass of 200 ng with insert-free pcDNA3.1 plasmid DNA. Each sample was prepared in duplicate wells. Cells were incubated at 37°C with 5% CO<sub>2</sub> for a total of 48 h. After 24 h, 1 ml of DMEM-10 was added to each well. Following incubation cell lysates and supernatants were collected for further analysis.</p><p id="p-74">Cell culture supernatants were collected and clarified. Clarified supernatants were layered over 2 ml of a 25% (w/v) sucrose solution in 13.2 ml Thinwall Polypropylene SW41 Ti ultracentrifuge tubes (Beckman-Coulter, Brea, USA) and made up to a total volume of 10 ml with PBS (-). Samples were then centrifuged at 130,000 x <em>g</em> for 2 h using an Optima L-100 XP ultracentrifuge (Beckmann-Coulter). Supernatants were then discarded and VLP pellets were resuspended in 60 μl of NP-40 lysis buffer (Sigma-Aldrich), supplemented with 10 μg/ml of aprotinin, leupeptin, and pepstatin A protease inhibitors (Sigma-Aldrich).</p><p id="p-75">The cells from each well were resuspended in 500 μl of PBS (-) and transferred to 1.5 ml tubes and centrifuged at 200 x <em>g</em> for 5 min. Supernatants were discarded and cell pellets were lysed in 100 μl of NP-40 lysis buffer. Following lysis samples were centrifuged at 20,000 x <em>g</em> for 5 min, supernatants were transferred to a new 1.5 ml tubes and cell debris pellets were discarded.</p><p id="p-76">Cell lysates and cell culture supernatant samples were analysed by SDS-PAGE and Western blot analysis as described above, with the following exceptions: For HIV VLPs, the primary antibody binding was directed against the Gag protein using a solution containing a 1/4000 dilution of a mouse anti-p24 antibody (NIH AIDS Reagent Repository, Germantown, USA) 0.1% Tween-20, in PBS (-). The secondary antibody was a 1/10,000 diluted polyclonal goat anti-mouse Alexa-Fluor 680 fluorophore-conjugate fluorescent secondary antibody (Thermo), in PBST.</p><p id="p-77">The quantity of VLPs released into the supernatant for each sample was calculated using a densitometric analysis of the signal strength of the fluorescent antibodies bound to viral proteins. Densitometry was performed using the Image Studio Lite v5.2 software suite (LI-COR). Each analysis included samples co-transfected with 0 to 200 ng of tetherin plasmid expression vectors. Quantitation was performed by first normalising variation in VLP expression between the samples by determining the ratio of the signal strength of each cell lysate and supernatant sample to the signal strength of the corresponding sample not transfected with tetherin (the 0 ng tetherin sample). Relative differences in the amount of VLPs released into the supernatant was then determined by calculating the ratio of the signal strength of the VLP protein in each normalized supernatant sample to its normalised cell lysate signal.</p></div></div><div class="section" id="sec-23"><h2 class="">Author’s Contributions</h2><p id="p-79">J.A.H., M.T., L-F.W., and G.T. designed the study; J.A.H., J.C., T.B.G. and A.J., performed research and analyzed data; J.A.H., M.T., G.A.M., M.L.B., L-F.W., and G.T. discussed and interpreted the results; J.A.H. wrote the first draft of the paper; J.A.H., and G.T. critically edited the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-78">We thank Paul Bieniasz for providing the pCRV1-NLgagpolΔVpu plasmid construct. We thank the NIH AIDS Research and Reference Reagent program, Division of AIDS, NIAID, NIH and Richard Axel for providing HEK 293T cells. This work was supported by the National Health and Medical Research Council (NHMRC), grant number GNT1121077, awarded to G.T., M.L.B., and G.A.M., and NHMRC Senior Research Fellowship GNT1117748 to G.T. J.A.H. was funded by NHMRC GNT1121077 and a Monash University Vice Chancellor’s Honours-PhD scholarship. L-F.W. is funded by the Singapore National Research Foundation grants NRF2012NRF-CRP001-056 and NRF2016NRF-NSFC002-013. The authors gratefully acknowledge the contribution to this work of the Victorian Operational Infrastructure Support Program received by the Burnet Institute. We thank Paul Bieniasz for the kind provision of the human tetherin expression plasmid used in this study.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.1" data-doi="10.1128/CMR.00017-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calisher</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">KV</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Schountz</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Bats: Important reservoir hosts of emerging viruses</span>. <abbr class="cit-jnl-abbrev">Clin. Microbiol. Rev</abbr>. <span class="cit-vol">19</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">531</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DCalisher%26rft.auinit1%253DC.%2BH.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DBats%253A%2BImportant%2BReservoir%2BHosts%2Bof%2BEmerging%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00017-06%26rft_id%253Dinfo%253Apmid%252F16847084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTMxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.2" data-doi="10.1016/j.coviro.2012.11.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">I</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L-F</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Bats and their virome: an important source of emerging viruses capable of infecting humans</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Virol</abbr>. <span class="cit-vol">3</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">84</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BVirol%26rft.volume%253D3%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2012.11.006%26rft_id%253Dinfo%253Apmid%252F23265969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2012.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23265969&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amman</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A recently discovered pathogenic paramyxovirus, Sosuga virus, is present in <em>Rousettus aegyptiacus</em> fruit bats at multiple locations in Uganda</span>. <abbr class="cit-jnl-abbrev">J. Wildl. Dis</abbr>. <span class="cit-vol">51</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">774</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BWildl.%2BDis%26rft.volume%253D51%26rft.spage%253D774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>) <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>:<span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.5" data-doi="10.3201/eid0204.960407"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Swanepoel</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Experimental inoculation of plants and animals with Ebola virus</span>. <abbr class="cit-jnl-abbrev">Emerging Infect. Dis</abbr>. <span class="cit-vol">2</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DSwanepoel%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DExperimental%2Binoculation%2Bof%2Bplants%2Band%2Banimals%2Bwith%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid0204.960407%26rft_id%253Dinfo%253Apmid%252F8969248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid0204.960407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8969248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WB30100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.6" data-doi="10.1111/j.1751-0813.1998.tb12335.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Transmission studies of Hendra virus (equine morbilli-virus) in fruit bats, horses and cats</span>. <abbr class="cit-jnl-abbrev">Aust. Vet. J</abbr>. <span class="cit-vol">76</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">813</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralian%2BVeterinary%2BJournal%26rft.stitle%253DAustralian%2BVeterinary%2BJournal%26rft.aulast%253DWilliamson%26rft.auinit1%253DM.%2BM.%26rft.volume%253D76%26rft.issue%253D12%26rft.spage%253D813%26rft.epage%253D818%26rft.atitle%253DTransmission%2Bstudies%2Bof%2BHendra%2Bvirus%2B%2528equine%2Bmorbillivirus%2529%2Bin%2Bfruit%2Bbats%252C%2Bhorses%2Band%2Bcats.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1751-0813.1998.tb12335.x%26rft_id%253Dinfo%253Apmid%252F9972433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1751-0813.1998.tb12335.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9972433&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077743700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.7" data-doi="10.1016/j.jcpa.2007.03.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Middleton</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Experimental Nipah Virus Infection in Pteropid Bats (<em>Pteropus poliocephalus</em>)</span>. <abbr class="cit-jnl-abbrev">J. Comp. Pathol</abbr>. <span class="cit-vol">136</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">266</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomparative%2Bpathology%26rft.stitle%253DJ%2BComp%2BPathol%26rft.aulast%253DMiddleton%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D266%26rft.epage%253D272%26rft.atitle%253DExperimental%2BNipah%2Bvirus%2Binfection%2Bin%2Bpteropid%2Bbats%2B%2528Pteropus%2Bpoliocephalus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcpa.2007.03.002%26rft_id%253Dinfo%253Apmid%252F17498518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcpa.2007.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17498518&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247096500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.8" data-doi="10.3201/eid1608.100208"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Bat coronaviruses and experimental infection of bats, the Philippines</span>. <abbr class="cit-jnl-abbrev">Emerging Infect. Dis</abbr>. <span class="cit-vol">16</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1217</span>–<span class="cit-lpage">1223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWatanabe%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D1217%26rft.epage%253D1223%26rft.atitle%253DBat%2Bcoronaviruses%2Band%2Bexperimental%2Binfection%2Bof%2Bbats%252C%2Bthe%2BPhilippines.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1608.100208%26rft_id%253Dinfo%253Apmid%252F20678314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1608.100208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20678314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.9" data-doi="10.3390/v7072779"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Experimental inoculation of egyptian rousette bats (<em>Rousettus aegyptiacus</em>) with viruses of the <em>Ebolavirus</em> and <em>Marburgvirus</em> genera</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">3420</span>–<span class="cit-lpage">3442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D3420%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7072779%26rft_id%253Dinfo%253Apmid%252F26120867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v7072779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26120867&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schountz</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">LF</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Antiviral immune responses of bats: A review</span>. <abbr class="cit-jnl-abbrev">Zoonoses Public Hlth</abbr>:<span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.11" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>(<span class="cit-issue">6118</span>):<span class="cit-fpage">456</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA4LjAzMTIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Contraction of the type I IFN locus and unusual constitutive expression of IFN-α in bats</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr>:<span class="cit-vol">201518240</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.13" data-doi="10.1093/molbev/msy048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Differential evolution of antiretroviral restriction factors in pteropid bats as revealed by <em>APOBEC3</em> gene complexity</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">35</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1626</span>–<span class="cit-lpage">1637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D35%26rft.spage%253D1626%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.14" data-doi="10.1016/j.cell.2018.03.070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavlovich</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The Egyptian rousette genome reveals unexpected features of bat antiviral immunity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1098</span>-<span class="cit-lpage">1110</span>. e1018.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1098%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.15" data-doi="10.3389/fimmu.2018.02112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mandl</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">DS</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">ML</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Going to bat(s) for studies of disease tolerance</span>. <abbr class="cit-jnl-abbrev">Front. Immunol</abbr>. <span class="cit-vol">9</span>:<span class="cit-fpage">2112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BImmunol%26rft.volume%253D9%26rft.spage%253D2112%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffimmu.2018.02112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fimmu.2018.02112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.16" data-doi="10.1073/pnas.1814995116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">A metaanalysis of bat phylogenetics and positive selection based on genomes and transcriptomes from 18 species</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">116</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">11351</span>–<span class="cit-lpage">11360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1814995116%26rft_id%253Dinfo%253Apmid%252F31113885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzIzLzExMzUxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.17" data-doi="10.1038/nature06553"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">SJD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Tetherin inhibits retrovirus release and is antagonized by HIV-1 Vpu</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>(<span class="cit-issue">7177</span>):<span class="cit-fpage">425</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNeil%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D451%26rft.issue%253D7177%26rft.spage%253D425%26rft.epage%253D430%26rft.atitle%253DTetherin%2Binhibits%2Bretrovirus%2Brelease%2Band%2Bis%2Bantagonized%2Bby%2BHIV-1%2BVpu.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06553%26rft_id%253Dinfo%253Apmid%252F18200009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06553&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18200009&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252554100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.18" data-doi="10.1128/JVI.01607-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakuma</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noda</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urata</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaoka</span>  <span class="cit-name-given-names">Y</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Inhibition of Lassa and Marburg virus production by tetherin</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">83</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">2382</span>–<span class="cit-lpage">2385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01607-08%26rft_id%253Dinfo%253Apmid%252F19091864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzUvMjM4MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDguMDMxMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Tetherin inhibits Nipah virus but not Ebola virus replication in fruit bat cells</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">93</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e01821</span>–<span class="cit-lpage">01818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft.volume%253D93%26rft.spage%253De01821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.20" data-doi="10.1371/journal.ppat.1003651"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wynne</span>  <span class="cit-name-given-names">JW</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L-F</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Bats and viruses: Friend or foe?</span> <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">9</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1003651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003651%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003651%26rft_id%253Dinfo%253Apmid%252F24204253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204253&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">KJ</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">CT</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Severe acute respiratory syndrome coronavirus spike protein counteracts BST2-mediated restriction of virus-like particle release</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. <span class="cit-vol">91</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1743</span>–<span class="cit-lpage">1750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirol%26rft.volume%253D91%26rft.spage%253D1743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.22" data-doi="10.1016/j.cell.2009.08.039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perez-Caballero</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Tetherin inhibits HIV-1 release by directly tethering virions to cells</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">139</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPerez-Caballero%26rft.auinit1%253DD.%26rft.volume%253D139%26rft.issue%253D3%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DTetherin%2Binhibits%2BHIV-1%2Brelease%2Bby%2Bdirectly%2Btethering%2Bvirions%2Bto%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.08.039%26rft_id%253Dinfo%253Apmid%252F19879838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.08.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19879838&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271259600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.23" data-doi="10.1016/j.chom.2016.06.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blanco-Melo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">S</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Origins and evolution of tetherin, an orphan antiviral gene</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">189</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D20%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.06.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.24" data-doi="10.1371/journal.ppat.1002931"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cocka</span>  <span class="cit-name-given-names">LJ</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Identification of alternatively translated tetherin isoforms with differing antiviral and signaling activities</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">8</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e1002931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DCocka%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1002931%26rft.epage%253De1002931%26rft.atitle%253DIdentification%2Bof%2Balternatively%2Btranslated%2BTetherin%2Bisoforms%2Bwith%2Bdiffering%2Bantiviral%2Band%2Bsignaling%2Bactivities.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002931%26rft_id%253Dinfo%253Apmid%252F23028328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028328&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.25" data-doi="10.1016/j.chom.2012.10.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galão Rui</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le Tortorec</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickering</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kueck</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Neil Stuart</span>  <span class="cit-name-given-names">JD</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Innate sensing of HIV-1 assembly by tetherin induces NFκB-dependent proinflammatory responses</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">633</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DGalao%26rft.auinit1%253DR.%2BP.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D633%26rft.epage%253D644%26rft.atitle%253DInnate%2BSensing%2Bof%2BHIV-1%2BAssembly%2Bby%2BTetherin%2BInduces%2BNF%25CE%25BAB-Dependent%2BProinflammatory%2BResponses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.10.007%26rft_id%253Dinfo%253Apmid%252F23159053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159053&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311765100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.26" data-doi="10.1128/JVI.02272-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tokarev</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Stimulation of NF-κB activity by the HIV restriction factor BST2</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">87</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">2046</span>–<span class="cit-lpage">2057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02272-12%26rft_id%253Dinfo%253Apmid%252F23221546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg3LzQvMjA0NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDguMDMxMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.27" data-doi="10.1038/sj.onc.1206406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Large-scale identification and characterization of human genes that activate NF-κB and MAPK signaling pathways</span>. <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">3307</span>–<span class="cit-lpage">3318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DMatsuda%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D21%26rft.spage%253D3307%26rft.epage%253D3318%26rft.atitle%253DLarge-scale%2Bidentification%2Band%2Bcharacterization%2Bof%2Bhuman%2Bgenes%2Bthat%2Bactivate%2BNF-kappaB%2Band%2BMAPK%2Bsignaling%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1206406%26rft_id%253Dinfo%253Apmid%252F12761501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1206406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12761501&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183040000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.28" data-doi="10.1016/0888-7543(95)80171-H"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">Molecular cloning and chromosomal mapping of a bone marrow stromal cell surface gene, BST2, that may be involved in pre-B-cell growth</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">527</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DIshikawa%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D527%26rft.epage%253D534%26rft.atitle%253DMolecular%2Bcloning%2Band%2Bchromosomal%2Bmapping%2Bof%2Ba%2Bbone%2Bmarrow%2Bstromal%2Bcell%2Bsurface%2Bgene%252C%2BBST2%252C%2Bthat%2Bmay%2Bbe%2Binvolved%2Bin%2Bpre-B-cell%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252895%252980171-H%26rft_id%253Dinfo%253Apmid%252F7607676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(95)80171-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7607676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QV65800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031203v1.29" data-doi="10.1073/pnas.1113694108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liberatore</span>  <span class="cit-name-given-names">RA</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Tetherin is a key effector of the antiretroviral activity of type I interferon in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">108</span>(<span class="cit-issue">44</span>):<span class="cit-fpage">18097</span>–<span class="cit-lpage">18101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1113694108%26rft_id%253Dinfo%253Apmid%252F22025715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ0LzE4MDk3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.30" data-doi="10.1073/pnas.1101684108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Erikson</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">In vivo expression profile of the antiviral restriction factor and tumor-targeting antigen CD317/BST-2/HM1.24/tetherin in humans</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">108</span>(<span class="cit-issue">33</span>):<span class="cit-fpage">13688</span>–<span class="cit-lpage">13693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1101684108%26rft_id%253Dinfo%253Apmid%252F21808013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMzLzEzNjg4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.31" data-doi="10.1073/pnas.1011485107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Structural insight into the mechanisms of enveloped virus tethering by tetherin</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">107</span>(<span class="cit-issue">43</span>):<span class="cit-fpage">18428</span>–<span class="cit-lpage">18432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011485107%26rft_id%253Dinfo%253Apmid%252F20940320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQzLzE4NDI4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.32" data-doi="10.1016/j.chom.2010.03.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hinz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Structural basis of HIV-1 tethering to membranes by the BST-2/tetherin ectodomain</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">314</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DHinz%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D314%26rft.epage%253D323%26rft.atitle%253DStructural%2Bbasis%2Bof%2BHIV-1%2Btethering%2Bto%2Bmembranes%2Bby%2Bthe%2BBST-2%252Ftetherin%2Bectodomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2010.03.005%26rft_id%253Dinfo%253Apmid%252F20399176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2010.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20399176&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277111400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.33" data-doi="10.1034/j.1600-0854.2003.00129.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kupzig</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Bst-2/HM1. 24 is a raft-associated apical membrane protein with an unusual topology</span>. <abbr class="cit-jnl-abbrev">Traffic</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">694</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTraffic%2B%2528Copenhagen%252C%2BDenmark%2529%26rft.stitle%253DTraffic%26rft.aulast%253DKupzig%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253D694%26rft.epage%253D709%26rft.atitle%253DBst-2%252FHM1.24%2Bis%2Ba%2Braft-associated%2Bapical%2Bmembrane%2Bprotein%2Bwith%2Ban%2Bunusual%2Btopology.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1600-0854.2003.00129.x%26rft_id%253Dinfo%253Apmid%252F12956872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1600-0854.2003.00129.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12956872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185195300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.34" data-doi="10.1242/jcs.003343"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rollason</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korolchuk</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schu</span>  <span class="cit-name-given-names">P</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Banting</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Clathrin-mediated endocytosis of a lipid-raft-associated protein is mediated through a dual tyrosine motif</span>. <abbr class="cit-jnl-abbrev">J. Cell Sci</abbr>. <span class="cit-vol">120</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">3850</span>–<span class="cit-lpage">3858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.003343%26rft_id%253Dinfo%253Apmid%252F17940069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyMC8yMS8zODUwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.35" data-doi="10.1371/journal.pone.0041483"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Identification and functional analysis of three isoforms of bovine BST-2</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e41483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTakeda%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41483%26rft.epage%253De41483%26rft.atitle%253DIdentification%2Band%2Bfunctional%2Banalysis%2Bof%2Bthree%2Bisoforms%2Bof%2Bbovine%2BBST-2.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041483%26rft_id%253Dinfo%253Apmid%252F22911799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22911799&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.36" data-doi="10.1074/jbc.M109.058305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iwabu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">HIV-1 accessory protein Vpu internalizes cell-surface BST-2/tetherin through transmembrane interactions leading to lysosomes</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">284</span>(<span class="cit-issue">50</span>):<span class="cit-fpage">35060</span>–<span class="cit-lpage">35072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.058305%26rft_id%253Dinfo%253Apmid%252F19837671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNTAvMzUwNjAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA4LjAzMTIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fukuma</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morikawa</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyazawa</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Cloning and characterization of the antiviral activity of feline tetherin/BST-2</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">3</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.38" data-doi="10.1016/j.ympev.2013.08.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruedi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Molecular phylogenetic reconstructions identify East Asia as the cradle for the evolution of the cosmopolitan genus Myotis (Mammalia, Chiroptera)</span>. <abbr class="cit-jnl-abbrev">Mol. Phylogen. Evol</abbr>. <span class="cit-vol">69</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">437</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogen.%2BEvol%26rft.volume%253D69%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2013.08.011%26rft_id%253Dinfo%253Apmid%252F23988307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2013.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23988307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.39" data-doi="10.1093/molbev/msl150"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benton</span>  <span class="cit-name-given-names">MJ</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">PCJ</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Paleontological evidence to date the tree of life</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">24</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">26</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl150%26rft_id%253Dinfo%253Apmid%252F17047029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17047029&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243069700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doering</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Englund</span>  <span class="cit-name-given-names">PT</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">GW</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-article-title">Detection of glycophospholipid anchors on proteins</span>. <abbr class="cit-jnl-abbrev">Curr. Protoc. Mol. Biol</abbr>. <span class="cit-vol">22</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">17.18.11</span>-<span class="cit-lpage">17.18.13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BMol.%2BBiol%26rft.volume%253D22%26rft.spage%253D171811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.41" data-doi="10.1128/JVI.01800-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dubé</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Suppression of tetherin-restricting activity upon human immunodeficiency virus type 1 particle release correlates with localization of Vpu in the trans-Golgi network</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">83</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">4574</span>–<span class="cit-lpage">4590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01800-08%26rft_id%253Dinfo%253Apmid%252F19244337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzkvNDU3NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDguMDMxMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.42" data-doi="10.1128/JVI.00071-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Feline tetherin efficiently restricts release of feline immunodeficiency virus but not spreading of infection</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">85</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">5840</span>–<span class="cit-lpage">5852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00071-11%26rft_id%253Dinfo%253Apmid%252F21490095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NS8xMi81ODQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.43" data-doi="10.1128/JVI.00029-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnaud</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Interplay between ovine bone marrow stromal cell antigen 2/tetherin and endogenous retroviruses</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">84</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">4415</span>–<span class="cit-lpage">4425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00029-10%26rft_id%253Dinfo%253Apmid%252F20181686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzkvNDQxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDguMDMxMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.44" data-doi="10.1128/JVI.07037-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Celestino</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Feline tetherin is characterized by a short N-terminal region and is counteracted by the feline immunodeficiency virus envelope glycoprotein</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">86</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">6688</span>–<span class="cit-lpage">6700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.07037-11%26rft_id%253Dinfo%253Apmid%252F22514338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xMi82Njg4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.45" data-doi="10.1073/pnas.1008206107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Structural and functional studies on the extracellular domain of BST2/tetherin in reduced and oxidized conformations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">107</span>(<span class="cit-issue">42</span>):<span class="cit-fpage">17951</span>–<span class="cit-lpage">17956</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1008206107%26rft_id%253Dinfo%253Apmid%252F20880831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQyLzE3OTUxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.46" data-doi="10.1186/1742-4690-6-80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrew</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyagi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">S</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Strebel</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">The formation of cysteine-linked dimers of BST-2/tetherin is important for inhibition of HIV-1 virus release but not for sensitivity to Vpu</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRetrovirology%26rft.aulast%253DAndrew%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D6%26rft.spage%253D80%26rft.epage%253D80%26rft.atitle%253DThe%2Bformation%2Bof%2Bcysteine-linked%2Bdimers%2Bof%2BBST-2%252Ftetherin%2Bis%2Bimportant%2Bfor%2Binhibition%2Bof%2BHIV-1%2Bvirus%2Brelease%2Bbut%2Bnot%2Bfor%2Bsensitivity%2Bto%2BVpu.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-6-80%26rft_id%253Dinfo%253Apmid%252F19737401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-6-80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19737401&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.47" data-doi="10.1371/journal.ppat.1000300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNatt</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Species-specific activity of HIV-1 Vpu and positive selection of tetherin transmembrane domain variants</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e1000300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DMcNatt%26rft.auinit1%253DM.%2BW.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253De1000300%26rft.epage%253De1000300%26rft.atitle%253DSpecies-specific%2Bactivity%2Bof%2BHIV-1%2BVpu%2Band%2Bpositive%2Bselection%2Bof%2Btetherin%2Btransmembrane%2Bdomain%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000300%26rft_id%253Dinfo%253Apmid%252F19214216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19214216&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.48" data-doi="10.1371/journal.ppat.1003483"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">S</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Mechanism of HIV-1 virion entrapment by tetherin</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">9</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e1003483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003483%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003483%26rft_id%253Dinfo%253Apmid%252F23874200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874200&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.49" data-doi="10.1371/journal.pone.0006934"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakuma</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakurai</span>  <span class="cit-name-given-names">A</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yasuda</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Dimerization of tetherin is not essential for its antiviral activity against Lassa and Marburg viruses</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e6934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSakuma%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De6934%26rft.epage%253De6934%26rft.atitle%253DDimerization%2Bof%2Btetherin%2Bis%2Bnot%2Bessential%2Bfor%2Bits%2Bantiviral%2Bactivity%2Bagainst%2BLassa%2Band%2BMarburg%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006934%26rft_id%253Dinfo%253Apmid%252F19742323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19742323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.50" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">PAML 4: Phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">24</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.51" data-doi="10.1126/science.1105113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">A molecular phylogeny for bats illuminates biogeography and the fossil record</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span>(<span class="cit-issue">5709</span>):<span class="cit-fpage">580</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTeeling%26rft.auinit1%253DE.%2BC.%26rft.volume%253D307%26rft.issue%253D5709%26rft.spage%253D580%26rft.epage%253D584%26rft.atitle%253DA%2BMolecular%2BPhylogeny%2Bfor%2BBats%2BIlluminates%2BBiogeography%2Band%2Bthe%2BFossil%2BRecord%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105113%26rft_id%253Dinfo%253Apmid%252F15681385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwOS81ODAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA4LjAzMTIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.52" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">MEGA6: Molecular evolutionary genetics analysis version 6.0</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">30</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.53" data-doi="10.1006/jmbi.2000.4315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Von Heijne</span>  <span class="cit-name-given-names">G</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">EL</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">305</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">567</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrogh%26rft.auinit1%253DA.%26rft.volume%253D305%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D580%26rft.atitle%253DPredicting%2Btransmembrane%2Bprotein%2Btopology%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%253A%2Bapplication%2Bto%2Bcomplete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4315%26rft_id%253Dinfo%253Apmid%252F11152613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167760800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.54" data-doi="10.1186/1471-2105-9-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pierleoni</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martelli</span>  <span class="cit-name-given-names">P</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Casadio</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">PredGPI: a GPI-anchor predictor</span>. <abbr class="cit-jnl-abbrev">BMC Bioinform</abbr>. <span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DFierro%26rft.auinit1%253DA.%2BC.%26rft.volume%253D9%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DEvaluation%2Bof%2Btime%2Bprofile%2Breconstruction%2Bfrom%2Bcomplex%2Btwo-color%2Bmicroarray%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-1%26rft_id%253Dinfo%253Apmid%252F18173834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18173834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.55" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">RC</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nuc. Acid. Res</abbr>. <span class="cit-vol">32</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.56" data-doi="10.1016/j.dci.2010.07.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cowled</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Molecular characterisation of Toll-like receptors in the black flying fox <em>Pteropus alecto</em></span>. <abbr class="cit-jnl-abbrev">Dev. Comp. Immunol</abbr>. <span class="cit-vol">35</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">7</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DCowled%26rft.auinit1%253DC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D18%26rft.atitle%253DMolecular%2Bcharacterisation%2Bof%2BToll-like%2Breceptors%2Bin%2Bthe%2Bblack%2Bflying%2Bfox%2BPteropus%2Balecto.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2010.07.006%26rft_id%253Dinfo%253Apmid%252F20692287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2010.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20692287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.08.031203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.08.031203v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">EF</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-source">Molecular cloning</span>, (<span class="cit-publ-name">Cold Spring Harbor laboratory press</span>, <span class="cit-publ-loc">New York</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.08.031203v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">TA</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-chapter-title">Synthesis and processing of RNA</span>. <span class="cit-source">Genomes</span>, (<span class="cit-publ-name">Wiley-Liss</span>, <span class="cit-publ-loc">Oxford</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.59" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">WJ</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">BLAT—the BLAST-like alignment tool</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">12</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wOC4wMzEyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031203v1.60" data-doi="10.1128/JVI.02211-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jouvenet</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Broad-spectrum inhibition of retroviral and filoviral particle release by tetherin</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">83</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1837</span>–<span class="cit-lpage">1844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02211-08%26rft_id%253Dinfo%253Apmid%252F19036818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzQvMTgzNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDguMDMxMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.08.031203v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.08.032268v1" title="An in vitro model to study the naive human peripheral blood mononuclear cell response to Cryptococcus neoformans infection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.08.032441v1" title="Viral cross-linking and solid-phase purification enables discovery of ribonucleoprotein complexes on incoming RNA virus genomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.08.031203v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.04.08.031203v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.08.031203v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.08.031203v1" class="active" data-icon-position="" data-hide-link-title="0">Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1236501" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-A71m7qyaKaI_wRy41TFsvvx4ibm-SgB9D62E2Y6pQnI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1236501" id="node1236501--2" data-pisa="biorxiv;2020.04.08.031203v1" data-pisa-master="biorxiv;2020.04.08.031203" data-seqnum="1236501" data-apath="/biorxiv/early/2020/04/09/2020.04.08.031203.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Joshua A.</span> <span class="nlm-surname">Hayward</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Tachedjian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tamsin B.</span> <span class="nlm-surname">Gordon</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Glenn A.</span> <span class="nlm-surname">Marsh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Michelle L.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Gilda</span> <span class="nlm-surname">Tachedjian</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.08.031203; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.08.031203 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;title=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;title=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;text=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;title=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;t=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;title=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;title=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1236501" id="node1236501--3" data-pisa="biorxiv;2020.04.08.031203v1" data-pisa-master="biorxiv;2020.04.08.031203" data-seqnum="1236501" data-apath="/biorxiv/early/2020/04/09/2020.04.08.031203.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Bats Possess Unique Variants of the Antiviral Restriction Factor Tetherin</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Joshua A.</span> <span class="nlm-surname">Hayward</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Tachedjian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Johnson</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tamsin B.</span> <span class="nlm-surname">Gordon</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Glenn A.</span> <span class="nlm-surname">Marsh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Michelle L.</span> <span class="nlm-surname">Baker</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Gilda</span> <span class="nlm-surname">Tachedjian</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.08.031203; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.08.031203 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hayward%2C+Joshua+A.&amp;rft.au=Tachedjian%2C+Mary&amp;rft.au=Johnson%2C+Adam&amp;rft.au=Gordon%2C+Tamsin+B.&amp;rft.au=Cui%2C+Jie&amp;rft.au=Marsh%2C+Glenn+A.&amp;rft.au=Baker%2C+Michelle+L.&amp;rft.au=Wang%2C+Lin-Fa&amp;rft.au=Tachedjian%2C+Gilda"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1236501/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1236501/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1236501/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1236501/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1236501/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1236501/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1236501/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1236501/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1236501/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1236501/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1236501/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1236501/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;count=horizontal&amp;via=&amp;text=Bats%20Possess%20Unique%20Variants%20of%20the%20Antiviral%20Restriction%20Factor%20Tetherin&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031203v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.08.031203v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
