<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/17/2020.02.16.946699.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.16.946699v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites" />
<meta name="DC.Identifier" content="10.1101/2020.02.16.946699" />
<meta name="DC.Date" content="2020-02-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum . We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies." />
<meta name="DC.Contributor" content="V Vern Lee" />
<meta name="DC.Contributor" content="Louise M. Judd" />
<meta name="DC.Contributor" content="Aaron R. Jex" />
<meta name="DC.Contributor" content="Kathryn E. Holt" />
<meta name="DC.Contributor" content="Christopher J. Tonkin" />
<meta name="DC.Contributor" content="Stuart A. Ralph" />
<meta name="article:published_time" content="2020-02-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of &lt;i&gt;Toxoplasma gondii&lt;/i&gt; and &lt;i&gt;Plasmodium falciparum&lt;/i&gt;. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.16.946699v1" />
<meta name="citation_id" content="2020.02.16.946699v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.16.946699" />
<meta name="citation_num_pages" content="29" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.16.946699" />
<meta name="citation_author" content="V Vern Lee" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne" />
<meta name="citation_author_institution" content="The Walter and Eliza Hall Institute of Medical Research" />
<meta name="citation_author_email" content="leev1@student.unimelb.edu.au" />
<meta name="citation_author" content="Louise M. Judd" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, Central Clinical School, Monash University" />
<meta name="citation_author" content="Aaron R. Jex" />
<meta name="citation_author_institution" content="The Walter and Eliza Hall Institute of Medical Research" />
<meta name="citation_author_institution" content="Faculty of Veterinary and Agricultural Sciences, The University of Melbourne" />
<meta name="citation_author" content="Kathryn E. Holt" />
<meta name="citation_author_institution" content="Department of Infectious Diseases, Central Clinical School, Monash University" />
<meta name="citation_author_institution" content="London School of Hygiene and Tropical Medicine" />
<meta name="citation_author" content="Christopher J. Tonkin" />
<meta name="citation_author_institution" content="The Walter and Eliza Hall Institute of Medical Research" />
<meta name="citation_author_institution" content="Department of Medical Biology, The University of Melbourne" />
<meta name="citation_author" content="Stuart A. Ralph" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne" />
<meta name="citation_reference" content="Cowman, A.F., et al., Malaria: Biology and Disease. Cell, 2016. 167(3): p. 610–624." />
<meta name="citation_reference" content="Torgerson, P.R. and P. Mastroiacovo, The global burden of congenital toxoplasmosis: a systematic review. Bull World Health Organ, 2013. 91(7): p. 501–8." />
<meta name="citation_reference" content="Furtado, J.M., et al., Toxoplasmosis: A Global Threat. Journal of Global Infectious Diseases, 2011. 3(3): p. 281–284." />
<meta name="citation_reference" content="Horrocks, P., et al., Control of gene expression in Plasmodium falciparum - ten years on. Mol Biochem Parasitol, 2009. 164(1): p. 9–25." />
<meta name="citation_reference" content="Wesseling, J.G., et al., Stage-specific expression and genomic organization of the actin genes of the malaria parasite Plasmodium falciparum. Mol Biochem Parasitol, 1989. 35(2): p. 167–76." />
<meta name="citation_reference" content="Lopez-Barragan, M.J., et al., Directional gene expression and antisense transcripts in sexual and asexual stages of Plasmodium falciparum. BMC Genomics, 2011. 12: p. 587." />
<meta name="citation_reference" content="Le Roch, K.G., et al., Global analysis of transcript and protein levels across the Plasmodium falciparum life cycle. Genome Res, 2004. 14(11): p. 2308–18." />
<meta name="citation_reference" content="Foth, B.J., et al., Quantitative protein expression profiling reveals extensive post-transcriptional regulation and post-translational modifications in schizont-stage malaria parasites. Genome Biol, 2008. 9(12): p. R177." />
<meta name="citation_reference" content="Toenhake, C.G. and R. Bártfai, What functional genomics has taught us about transcriptional regulation in malaria parasites. Briefings in Functional Genomics, 2019." />
<meta name="citation_reference" content="Llora-Batlle, O., E. Tinto-Font, and A. Cortes, Transcriptional variation in malaria parasites: why and how. Brief Funct Genomics, 2019." />
<meta name="citation_reference" content="Yeoh, L.M., et al., Alternative splicing is required for stage differentiation in malaria parasites. Genome Biol, 2019. 20(1): p. 151." />
<meta name="citation_reference" content="Mockenhaupt, S. and E.V. Makeyev, Non-coding functions of alternative pre-mRNA splicing in development. Seminars in cell &amp; developmental biology, 2015. 47-48: p. 32–39." />
<meta name="citation_reference" content="Neverov, A.D., et al., Alternative splicing and protein function. BMC Bioinformatics, 2005. 6(1): p. 266." />
<meta name="citation_reference" content="Kelemen, O., et al., Function of alternative splicing. Gene, 2013. 514(1): p. 1–30." />
<meta name="citation_reference" content="Birzele, F., G. Csaba, and R. Zimmer, Alternative splicing and protein structure evolution. Nucleic acids research, 2008. 36(2): p. 550–558." />
<meta name="citation_reference" content="Wang, E.T., et al., Alternative isoform regulation in human tissue transcriptomes. Nature, 2008. 456(7221): p. 470–6." />
<meta name="citation_reference" content="Licatalosi, D.D. and R.B. Darnell, RNA processing and its regulation: global insights into biological networks. Nat Rev Genet, 2010. 11(1): p. 75–87." />
<meta name="citation_reference" content="Yeoh, L.M., et al., Alternative Splicing in Apicomplexan Parasites. mBio, 2019. 10(1): p. e02866–18." />
<meta name="citation_reference" content="van Dooren, G.G., et al., Processing of an apicoplast leader sequence in Plasmodium falciparum, and the identification of a putative leader cleavage enzyme. J Biol Chem, 2002. 277(26): p. 23612–23619." />
<meta name="citation_reference" content="Pham, J.S., et al., A dual-targeted aminoacyl-tRNA synthetase in Plasmodium falciparum charges cytosolic and apicoplast tRNACys. Biochem J, 2014. 458(3): p. 513–23." />
<meta name="citation_reference" content="Gadalla, N.B., et al., Alternatively spliced transcripts and novel pseudogenes of the Plasmodium falciparum resistance-associated locus pfcrt detected in East African malaria patients. The Journal of antimicrobial chemotherapy, 2015. 70(1): p. 116–123." />
<meta name="citation_reference" content="Jex, A.R., et al., Genome and transcriptome of the porcine whipworm Trichuris suis. Nature Genetics, 2014. 46(7): p. 701–706." />
<meta name="citation_reference" content="Lees, J.G., J.A. Ranea, and C.A. Orengo, Identifying and characterising key alternative splicing events in Drosophila development. BMC genomics, 2015. 16(1): p. 608–608." />
<meta name="citation_reference" content="Goodwin, S., J.D. McPherson, and W.R. McCombie, Coming of age: ten years of next-generation sequencing technologies. Nat Rev Genet, 2016. 17(6): p. 333–51." />
<meta name="citation_reference" content="Salzberg, S.L., et al., GAGE: A critical evaluation of genome assemblies and assembly algorithms. Genome research, 2012. 22(3): p. 557–567." />
<meta name="citation_reference" content="Kozarewa, I., et al., Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes. Nature methods, 2009. 6(4): p. 291–295." />
<meta name="citation_reference" content="Chaudhary, K., et al., Differential localization of alternatively spliced hypoxanthine-xanthine-guanine phosphoribosyltransferase isoforms in Toxoplasma gondii. J Biol Chem, 2005. 280(23): p. 22053–9." />
<meta name="citation_reference" content="Jenjaroenpun, P., et al., Complete genomic and transcriptional landscape analysis using third-generation sequencing: a case study of Saccharomyces cerevisiae CEN.PK113-7D. Nucleic Acids Research, 2018. 46(7): p. e38–e38." />
<meta name="citation_reference" content="Jain, M., et al., Nanopore sequencing and assembly of a human genome with ultra-long reads. Nature Biotechnology, 2018. 36: p. 338." />
<meta name="citation_reference" content="Byrne, A., et al., Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells. Nature Communications, 2017. 8: p. 16027." />
<meta name="citation_reference" content="Simpson, J.T., et al., Detecting DNA cytosine methylation using nanopore sequencing. Nature Methods, 2017. 14: p. 407." />
<meta name="citation_reference" content="Workman, R.E., et al., Nanopore native RNA sequencing of a human poly(A) transcriptome. bioRxiv, 2018: p. 459529." />
<meta name="citation_reference" content="Chappell, L., et al., Refining the transcriptome of the human malaria parasite Plasmodium falciparum using amplification-free RNA-seq. bioRxiv, 2019: p. 852038." />
<meta name="citation_reference" content="Garalde, D.R., et al., Highly parallel direct RNA sequencing on an array of nanopores. Nature Methods, 2018. 15(3): p. 201–206." />
<meta name="citation_reference" content="Viehweger, A., Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis. Genome Research, 2019. 29(9): p. 1545." />
<meta name="citation_reference" content="Sharon, D., et al., A single-molecule long-read survey of the human transcriptome. Nature biotechnology, 2013. 31(11): p. 1009–1014." />
<meta name="citation_reference" content="Soneson, C., et al., A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes. Nature Communications, 2019. 10(1): p. 3359." />
<meta name="citation_reference" content="Li, H., Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics, 2018. 34(18): p. 3094–3100." />
<meta name="citation_reference" content="Shaw, P.J., et al., Estimating mRNA lengths from Plasmodium falciparum genes by Virtual Northern RNA-seq analysis. Int J Parasitol, 2016. 46(1): p. 7–12." />
<meta name="citation_reference" content="Nilsen, T.W. and B.R. Graveley, Expansion of the eukaryotic proteome by alternative splicing. Nature, 2010. 463(7280): p. 457–63." />
<meta name="citation_reference" content="Braunschweig, U., et al., Widespread intron retention in mammals functionally tunes transcriptomes. Genome Res, 2014. 24(11): p. 1774–86." />
<meta name="citation_reference" content="Wong, J.J., et al., Orchestrated intron retention regulates normal granulocyte differentiation. Cell, 2013. 154(3): p. 583–95." />
<meta name="citation_reference" content="Boutz, P.L., A. Bhutkar, and P.A. Sharp, Detained introns are a novel, widespread class of post-transcriptionally spliced introns. Genes &amp; development, 2015. 29(1): p. 63–80." />
<meta name="citation_reference" content="Sakharkar, M.K., V.T. Chow, and P. Kangueane, Distributions of exons and introns in the human genome. In Silico Biol, 2004. 4(4): p. 387–93." />
<meta name="citation_reference" content="Ivashchenko, A.T., V.A. Khailenko, and A. Atambaeva Sh, [Variations of the length of exons and introns in human genome genes]. Genetika, 2009. 45(1): p. 22–9." />
<meta name="citation_reference" content="Lau, Y.L., et al., Deciphering the Draft Genome of Toxoplasma gondii RH Strain. PLoS One, 2016. 11(6): p. e0157901." />
<meta name="citation_reference" content="Hall, N., et al., Sequence of Plasmodium falciparum chromosomes 1, 3-9 and 13. Nature, 2002. 419(6906): p. 527–31." />
<meta name="citation_reference" content="Liao, Y., G.K. Smyth, and W. Shi, featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics, 2014. 30(7): p. 923–30." />
<meta name="citation_reference" content="Gonzalez-Porta, M., et al., Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene. Genome Biol, 2013. 14(7): p. R70." />
<meta name="citation_reference" content="Schmitz, U., et al., Intron retention enhances gene regulatory complexity in vertebrates. Genome biology, 2017. 18(1): p. 216–216." />
<meta name="citation_reference" content="Tang, A.D., et al., Full-length transcript characterization of SF3B1 mutation in chronic lymphocytic leukemia reveals downregulation of retained introns. bioRxiv, 2018: p. 410183." />
<meta name="citation_reference" content="Nagy, E. and L.E. Maquat, A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance. Trends Biochem Sci, 1998. 23(6): p. 198–9." />
<meta name="citation_reference" content="Ge, Y. and B.T. Porse, The functional consequences of intron retention: alternative splicing coupled to NMD as a regulator of gene expression. Bioessays, 2014. 36(3): p. 236–43." />
<meta name="citation_reference" content="Braunschweig, U., et al., Widespread intron retention in mammals functionally tunes transcriptomes. Genome research, 2014. 24(11): p. 1774–1786." />
<meta name="citation_reference" content="Neverov, A.D., et al., Alternative splicing and protein function. BMC bioinformatics, 2005. 6: p. 266–266." />
<meta name="citation_reference" content="Sidik, S.M., et al., A Genome-wide CRISPR Screen in Toxoplasma Identifies Essential Apicomplexan Genes. Cell, 2016. 166(6): p. 1423-1435.e12." />
<meta name="citation_reference" content="Zhang, M., et al., Uncovering the essential genes of the human malaria parasite Plasmodium falciparum by saturation mutagenesis. Science, 2018. 360 (6388)." />
<meta name="citation_reference" content="Pai, A.A., et al., The kinetics of pre-mRNA splicing in the Drosophila genome and the influence of gene architecture. Elife, 2017. 6." />
<meta name="citation_reference" content="Skinner, S.O., et al., Single-cell analysis of transcription kinetics across the cell cycle. Elife, 2016. 5: p. e12175." />
<meta name="citation_reference" content="Tsai, K.-W., et al., Sequence features involved in the mechanism of 3’ splice junction wobbling. BMC molecular biology, 2010. 11: p. 34–34." />
<meta name="citation_reference" content="Jenjaroenpun, P., et al., Complete genomic and transcriptional landscape analysis using third-generation sequencing: a case study of Saccharomyces cerevisiae CEN.PK113-7D. Nucleic Acids Res, 2018. 46(7): p. e38." />
<meta name="citation_reference" content="Moldován, N., et al., Third-generation Sequencing Reveals Extensive Polycistronism and Transcriptional Overlapping in a Baculovirus. Scientific Reports, 2018. 8(1): p. 8604." />
<meta name="citation_reference" content="Dozmorov, M.G., et al., Detrimental effects of duplicate reads and low complexity regions on RNA- and ChIP-seq data. BMC bioinformatics, 2015. 16 Suppl 13(Suppl 13): p. S10–S10." />
<meta name="citation_reference" content="Gardner, M.J., et al., Genome sequence of the human malaria parasite Plasmodium falciparum. Nature, 2002. 419(6906): p. 498–511." />
<meta name="citation_reference" content="Runtuwene, L.R., et al., Nanopore sequencing of drug-resistance-associated genes in malaria parasites, Plasmodium falciparum. Scientific Reports, 2018. 8(1): p. 8286." />
<meta name="citation_reference" content="Loven, J., et al., Revisiting global gene expression analysis. Cell, 2012. 151(3): p. 476–82." />
<meta name="citation_reference" content="Tarazona, S., et al., Differential expression in RNA-seq: a matter of depth. Genome Res, 2011. 21(12): p. 2213–23." />
<meta name="citation_reference" content="Barbosa-Morais, N.L., et al., The Evolutionary Landscape of Alternative Splicing in Vertebrate Species. Science, 2012. 338(6114): p. 1587." />
<meta name="citation_reference" content="Schmitz, U., et al., Intron retention enhances gene regulatory complexity in vertebrates. Genome Biology, 2017. 18(1): p. 216." />
<meta name="citation_reference" content="Kim, E., A. Magen, and G. Ast, Different levels of alternative splicing among eukaryotes. Nucleic acids research, 2007. 35(1): p. 125–131." />
<meta name="citation_reference" content="Lynch, M. and J.S. Conery, The Origins of Genome Complexity. Science, 2003. 302(5649): p. 1401." />
<meta name="citation_reference" content="Chen, L., et al., Correcting for differential transcript coverage reveals a strong relationship between alternative splicing and organism complexity. Molecular biology and evolution, 2014. 31(6): p. 1402–1413." />
<meta name="citation_reference" content="Irimia, M., D. Penny, and S.W. Roy, Coevolution of genomic intron number and splice sites. Trends Genet, 2007. 23(7): p. 321–5." />
<meta name="citation_reference" content="McGuire, A.M., et al., Cross-kingdom patterns of alternative splicing and splice recognition. Genome Biology, 2008. 9(3): p. R50." />
<meta name="citation_reference" content="Sieber, P., et al., Comparative Study on Alternative Splicing in Human Fungal Pathogens Suggests Its Involvement During Host Invasion. Frontiers in Microbiology, 2018. 9(2313)." />
<meta name="citation_reference" content="Grau-Bové, X., I. Ruiz-Trillo, and M. Irimia, Origin of exon skipping-rich transcriptomes in animals driven by evolution of gene architecture. Genome biology, 2018. 19(1): p. 135–135." />
<meta name="citation_reference" content="Kawashima, T., et al., Widespread use of non-productive alternative splice sites in Saccharomyces cerevisiae. PLoS Genet, 2014. 10(4): p. e1004249." />
<meta name="citation_reference" content="Lewis, B.P., R.E. Green, and S.E. Brenner, Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans. Proc Natl Acad Sci U S A, 2003. 100(1): p. 189–92." />
<meta name="citation_reference" content="Sayani, S., et al., Widespread impact of nonsense-mediated mRNA decay on the yeast intronome. Mol Cell, 2008. 31(3): p. 360–70." />
<meta name="citation_reference" content="Plass, M., et al., RNA secondary structure mediates alternative 3’ss selection in Saccharomyces cerevisiae. RNA (New York, N.Y.), 2012. 18(6): p. 1103–1115." />
<meta name="citation_reference" content="Weischenfeldt, J., et al., NMD is essential for hematopoietic stem and progenitor cells and for eliminating by-products of programmed DNA rearrangements. Genes &amp; development, 2008. 22(10): p. 1381–1396." />
<meta name="citation_reference" content="Hwang, J. and L.E. Maquat, Nonsense-mediated mRNA decay (NMD) in animal embryogenesis: to die or not to die, that is the question. Current opinion in genetics &amp; development, 2011. 21(4): p. 422–430." />
<meta name="citation_reference" content="Shi, M., et al., Premature Termination Codons Are Recognized in the Nucleus in A Reading-Frame Dependent Manner. Cell Discov, 2015. 1." />
<meta name="citation_reference" content="Harati, S., J.H. Phan, and M.D. Wang, Investigation of factors affecting RNA-seq gene expression calls. Conference proceedings : … Annual International Conference of the IEEE Engineering in Medicine and Biology Society. IEEE Engineering in Medicine and Biology Society. Annual Conference, 2014. 2014: p. 5232–5235." />
<meta name="citation_reference" content="Zhao, S., et al., Evaluation of two main RNA-seq approaches for gene quantification in clinical RNA sequencing: polyA+ selection versus rRNA depletion. Scientific reports, 2018. 8(1): p. 4781–4781." />
<meta name="citation_reference" content="Ge, Y. and B.T. Porse, The functional consequences of intron retention: Alternative splicing coupled to NMD as a regulator of gene expression. BioEssays, 2014. 36(3): p. 236–243." />
<meta name="citation_reference" content="Zhang, A.Y., et al., A data-driven approach to characterising intron signal in RNAseq data. bioRxiv, 2018: p. 352823." />
<meta name="citation_reference" content="Yap, K. and E.V. Makeyev, Regulation of gene expression in mammalian nervous system through alternative pre-mRNA splicing coupled with RNA quality control mechanisms. Mol Cell Neurosci, 2013. 56: p. 420–8." />
<meta name="citation_reference" content="Wang, M., A.T. Branco, and B. Lemos, The Y Chromosome Modulates Splicing and Sex-Biased Intron Retention Rates in Drosophila. Genetics, 2018. 208(3): p. 1057–1067." />
<meta name="citation_reference" content="Edwards, C.R., et al., A dynamic intron retention program in the mammalian megakaryocyte and erythrocyte lineages. Blood, 2016. 127(17): p. e24." />
<meta name="citation_reference" content="Middleton, R., et al., IRFinder: assessing the impact of intron retention on mammalian gene expression. Genome Biology, 2017. 18(1): p. 51." />
<meta name="citation_reference" content="Nickless, A., J.M. Bailis, and Z. You, Control of gene expression through the nonsense-mediated RNA decay pathway. Cell &amp; bioscience, 2017. 7: p. 26–26." />
<meta name="citation_reference" content="Han, X., et al., Nonsense-mediated mRNA decay: a ‘nonsense’ pathway makes sense in stem cell biology. Nucleic acids research, 2018. 46(3): p. 1038–1051." />
<meta name="citation_reference" content="Vanichkina, D.P., et al., Challenges in defining the role of intron retention in normal biology and disease. Seminars in Cell &amp; Developmental Biology, 2018. 75: p. 40–49." />
<meta name="citation_reference" content="Saudemont, B., et al., The fitness cost of mis-splicing is the main determinant of alternative splicing patterns. Genome biology, 2017. 18(1): p. 208–208." />
<meta name="citation_reference" content="Foth, B.J., et al., Quantitative time-course profiling of parasite and host cell proteins in the human malaria parasite Plasmodium falciparum. Mol Cell Proteomics, 2011. 10(8): p. M110.006411." />
<meta name="citation_reference" content="Aurrecoechea, C., et al., EuPathDB: the eukaryotic pathogen genomics database resource. Nucleic Acids Res, 2017. 45(D1): p. D581–d591." />
<meta name="citation_reference" content="Andrews, S., FastQC A Quality Control tool for High Throughput Sequence Data. 2010: http://www.bioinformatics.babraham.ac.uk/projects/fastqc/." />
<meta name="citation_reference" content="Kim, D., et al., Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nature Biotechnology, 2019. 37(8): p. 907–915." />
<meta name="citation_reference" content="Li, H., et al., The Sequence Alignment/Map format and SAMtools. Bioinformatics, 2009. 25(16): p. 2078–9." />
<meta name="citation_reference" content="Broad Institute, Picard Toolkit. 2019: http://broadinstitute.github.io/picard/." />
<meta name="citation_reference" content="Weirather, J.L., et al., Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis. F1000Research, 2017. 6: p. 100–100." />
<meta name="citation_reference" content="Robinson, J.T., et al., Integrative genomics viewer. Nature biotechnology, 2011. 29(1): p. 24–26." />
<meta name="citation_reference" content="Garalde, D.R., et al., Highly parallel direct RNA sequencing on an array of nanopores. Nat Methods, 2018. 15(3): p. 201–206." />
<meta name="citation_reference" content="Oxford Nanopore Technologies Ltd., Wub. 2016: https://github.com/nanoporetech/wub." />
<meta name="citation_reference" content="Quinlan, A.R. and I.M. Hall, BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics, 2010. 26(6): p. 841–842." />
<meta name="citation_reference" content="An, J., et al., J-Circos: an interactive Circos plotter. Bioinformatics, 2015. 31(9): p. 1463–5." />
<meta name="citation_reference" content="Ramírez, F., et al., deepTools: a flexible platform for exploring deep-sequencing data. Nucleic acids research, 2014. 42 (Web Server issue): p. W187–W191." />
<meta name="citation_reference" content="Blankenberg, D., et al., Dissemination of scientific software with Galaxy ToolShed. Genome Biology, 2014. 15(2): p. 403." />
<meta name="citation_reference" content="Wang, L., S. Wang, and W. Li, RSeQC: quality control of RNA-seq experiments. Bioinformatics, 2012. 28(16): p. 2184–5." />
<meta name="citation_reference" content="Hulsen, T., J. de Vlieg, and W. Alkema, BioVenn – a web application for the comparison and visualization of biological lists using area-proportional Venn diagrams. BMC Genomics, 2008. 9(1): p. 488." />
<meta name="citation_reference" content="Liao, Y., et al., WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Nucleic Acids Res, 2019. 47 (W1): p. W199–w205." />
<meta name="twitter:title" content="Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum . We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies." />
<meta name="og-title" property="og:title" content="Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of Toxoplasma gondii and Plasmodium falciparum . We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.16.946699v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.16.946699v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.16.946699v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1155358" />
    <title>Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"PIWr0BGbvXTlPFI-_ECc3CCjj7K6PQ_KPeUVslN0oAI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1155358","apath":"\/biorxiv\/early\/2020\/02\/17\/2020.02.16.946699.atom","pisa":"biorxiv;2020.02.16.946699v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.16.946699v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.16.946699v1","title":"Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites","identifier":"node\/1155358"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WMENOfFwjVNEW5yC52H+43c8F57hHGmSgvWifw5MRuXASelWjud3EXTHyPh+lvXOd7LJXyPT1PNmt04hPGIO4szUh428p2XkNNV1KdZljMt526MvyzCHkAU8XYxOs4gNG4Jdg2+A3nEVzdvydnlp5SqRsjrjU6wHUnyqp2ahyD1G9j07fe7qlNG4+HI2u2CNz7YZ5AnhYrQkMu4UJGgfVa0EsJiCtiaogj8BofOnN63B+0tzx+BMGfxOsTI0BrhEwXsclLmXFiGvDzBJrBYtmRmdikthriSidSjDPeqlftg+Hwv04SvTABTPwNaZgTd\\\\\\\/pOsEm4Ksipsth+Rwvwk91Vnmcp6s9Q1A7qA9pk0C\\\\\\\/FGlHVbbHUAFMJzqZtVPFttPZI+X1L+XnLU2x7QDhh4DJnb9Na2oTfqb7FbE4Xe5l3WssYMsarAFBh6P3IV519asEZOZgIPKFqPG06QlzMR\\\\\\\/BXPi7+woPjSkVSYbzOoqzy9mXVH1YD8BJaXcftQPw0h69WWvh\\\\\\\/SxieNnZdgt9WBh1W2lEszoU6kBaNugGe6uvZ9W71su1ozfewpubdsibtfKu6ohm\\\\\\\/qxKpEpdbtgRLx6yzJ4xhKjJtCeVju\\\\\\\/PD3YNpI\\\\\\\/df7b7oJCzpmuFccZc98hDdbkJIK4MBnUt8Kz27fdBuQuBL7h8qtNyyKzmNg\\\\\\\/fLxVOfqYx1eTtUNK9RVFtpX9asVw5KO3KNtX5iAIW2vi+gVjoW2ghG3j22Lb62zduUqmOIKrL0s\\\\\\\/RtXtbOlEwNvIHwLrH1pXoOwtQerrLIh3exb\\\\\\\/dimiY7LD7r2U4cz6JOiUV9TmV81Ir9GU3wUWpLvEG2wc1n1HNP8\\\\\\\/6sECRnwCWOhHdDfalaRWiHxWzrmj62ywge4hDfXoqt833OHaOqnV5F8MagIQrYZJJzc+ihTBchcJ16hGqDSM0gcEjIsOKfgwdm7IQGEPyDs8vSaQtjXrnYeq37GagzQgo6ieez8uBacgkBAWiSRAfe7f0nA3vK0SyCsbVKVPJWl2iCfqMhHO9toZYZQ0u03RYffB\\\\\\\/ofOS5OUPTHQSqES9gG2A0knk54CCxpmwb0gRnUz+cjMTVykUpEeunPnOa8G5tr2O3UsBRQl7aYbFc7b9qqqrkgW4xea6H3Z4IksWXdu\\\\\\\/fUIW4V3mbx+bBAtVMQa+vFEhpNAM6DSiUN3B6SK+ZLa6ioiGaa2QsyaKuy42Bsg\\\\\\\/SA4MMcsQZ3eBddQyuw3btHHRJbCFy9n6JbcJs7qZOWqCQmKz0pR1XpwYbEEiW7RUaW5eptnfMDXa5o6ZbwkwGAKzPzEm3ZgK33xp1RWOd0qQJVuULLkFT5W6Ay5xgUHFTfS4l1u\\\u0022,\\\u0022iv\\\u0022:\\\u00224hsgkOh5nKoTIqL2HQND2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f4eadc4064be30924515ee7e3c563d60e94e533f75ef4c34b9799ed2252b4dd8\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.16.946699v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1155358 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.16.946699v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c2t7gVo2_n91go-gfiY429K-01NR8cHS7GoFKuGXsjs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1155358" id="node1155358" data-pisa="biorxiv;2020.02.16.946699v1" data-pisa-master="biorxiv;2020.02.16.946699" data-apath="/biorxiv/early/2020/02/17/2020.02.16.946699.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">V Vern</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louise M.</span> <span class="nlm-surname">Judd</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Aaron R.</span> <span class="nlm-surname">Jex</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kathryn E.</span> <span class="nlm-surname">Holt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Tonkin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Stuart A.</span> <span class="nlm-surname">Ralph</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.16.946699 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1155358" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">V Vern Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne</span>, Melbourne, Victoria, 3010, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Walter and Eliza Hall Institute of Medical Research</span>, Parkville, Melbourne, Victoria 3052, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V%2BVern%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20VV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AV%2BVern%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leev1@student.unimelb.edu.au" class="" data-icon-position="" data-hide-link-title="0">leev1@student.unimelb.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louise M. Judd </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Infectious Diseases, Central Clinical School, Monash University</span>, Melbourne, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louise%2BM.%2BJudd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Judd%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouise%2BM.%2BJudd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron R. Jex </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Walter and Eliza Hall Institute of Medical Research</span>, Parkville, Melbourne, Victoria 3052, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Faculty of Veterinary and Agricultural Sciences, The University of Melbourne</span>, Parkville, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BR.%2BJex%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jex%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BR.%2BJex%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathryn E. Holt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Infectious Diseases, Central Clinical School, Monash University</span>, Melbourne, Victoria, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>London School of Hygiene and Tropical Medicine</span>, London WC1E 7HT, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathryn%2BE.%2BHolt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holt%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathryn%2BE.%2BHolt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christopher J. Tonkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Walter and Eliza Hall Institute of Medical Research</span>, Parkville, Melbourne, Victoria 3052, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Medical Biology, The University of Melbourne</span>, Melbourne, Victoria 3052, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BJ.%2BTonkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tonkin%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BJ.%2BTonkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Stuart A. Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biochemistry and Molecular Biology, Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne</span>, Melbourne, Victoria, 3010, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stuart%2BA.%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStuart%2BA.%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.16.946699v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.16.946699v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.16.946699v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.16.946699v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.16.946699v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.16.946699v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1155358" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1155358/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Alternative splicing is a widespread phenomenon in metazoans by which single genes are able to produce multiple isoforms of the gene product. However, this has been poorly characterised in apicomplexans, a major phylum of some of the most important global parasites. Efforts have been hampered by atypical transcriptomic features, such as the high AT content of Plasmodium RNA, but also the limitations of short read sequencing in deciphering complex splicing events. In this study, we utilised the long read direct RNA sequencing platform developed by Oxford Nanopore Technologies (ONT) to survey the alternative splicing landscape of <em>Toxoplasma gondii</em> and <em>Plasmodium falciparum</em>. We find that while native RNA sequencing has a reduced throughput, it allows us to obtain full-length or near full-length transcripts with comparable quantification to Illumina sequencing. By comparing this data with available gene models, we find widespread alternative splicing, particular intron retention, in these parasites. Most of these transcripts contain premature stop codons, suggesting that in these parasites, alternative splicing represents a pathway to transcriptomic diversity, rather than expanding proteomic diversity. Moreover, alternative splicing rates are comparable between parasites, suggesting a shared splicing machinery, despite notable transcriptomic differences between the parasites. This work highlights a strategy in using long read sequencing to understand splicing events at the whole transcript level, and has implications in future interpretation of RNA-seq studies.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Transcriptomic analyses have been central to insights into the biology and pathogenesis of eukaryotic pathogens. The best-characterised eukaryotic pathogen transcriptomes are those of the phylum Apicomplexa. This phylum includes some of the most important parasites impacting human and veterinary health, such as <em>Plasmodium</em> and <em>Toxoplasma. Plasmodium</em> is the causative agent of malaria, a devastating parasitic disease infecting over 200 million individuals and killing 400,000 each year [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. <em>Toxoplasma</em> causes toxoplasmosis, a widespread zoonoses that primarily impacts immunocompromised, young, and pregnant individuals [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>], and is thought to infect a third of the world’s population [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The pathogenesis of apicomplexan infections is intimately linked to the parasites’ life cycles. The life cycle of most parasitic apicomplexans is complex, involving multiple differentiated forms and hosts, and this requires reprograming of the parasite transcriptome.</p><p id="p-4">Early transcriptomic experiments sought to utilise techniques such as microarrays and Sanger sequencing of cDNA or EST libraries to understand changes in gene expression that define the pathogenesis of the parasites. These studies reveal that the timing of appearance and abundance of individual mRNAs follow developmentally distinct patterns [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], even for many predicted housekeeping genes. For example, the expression of the actin gene family in <em>P. falciparum</em> is developmentally tuned, with actin I primarily transcribed in asexual intraerythrocytic life stages while actin II is primarily present in sexual stage parasites [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Unusually, however, there is a poor correlation between protein and mRNA expression profiles for many genes in parasitic apicomplexans [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. In one experiment, Foth and colleagues found widespread discrepancies between temporal expression patterns of proteins and transcripts in <em>P. falciparum</em> [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Such discrepancies suggest that substantial post-transcriptional regulation occurs within these parasites. Indeed, with the advent of RNA-seq, more recent studies now show that multiple layers of gene expression are required for parasite life progression, through transcriptional, post-transcriptional, and epigenetic control mechanisms [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>-<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>].</p><p id="p-5">RNA splicing provides one such source of co- and post-transcriptional regulation. In this process, introns are removed from the pre-mRNA and the exons retained to form one contiguous molecule that is then translated by the ribosome. However, for complex mRNAs, alternative splicing either of untranslated regions, or the exonic chain, can add additional complexity. Through this process, pre-mRNA species can be differentially spliced, to create multiple distinct mature mRNAs from a single gene. This can alter regulation of the gene, for example by removing small-RNA binding sites [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] or diversify the proteome, as individual genes may encode multiple protein isoforms with altered structure or function [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>].Indeed, proteomic analyses have revealed widespread protein isoforms arising from single genes, corresponding with varying activity, stability, localisation and post-translational modifications [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. With advances in genome and transcript sequencing, it has become apparent over the last decade that alternative splicing of pre-mRNA occurs to a great extent. For example, more than 95% of human genes are alternatively spliced, and many transcript isoforms are specific to tissues or cellular states [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Such observations suggest that RNA diversity is more complex than previously appreciated [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>].</p><p id="p-6">Although alternative splicing appears to play a major (though debated) role in post-transcriptional control in metazoans, the process in less understood in apicomplexans. Studies have identified apicomplexan genes with crucial alternative splicing outcomes [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. For example, alternative splicing is required for attaching a protein trafficking pre-sequence onto two adjacent gene coding sequences [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>], and normal multi-organellar targeting of the <em>P. falciparum</em> cysteinyl tRNA synthetase, which is essential for parasite survival [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Nonetheless, there is little other study of alternative splicing in this phylum. Understanding diversity of parasites transcripts is crucial for drug and vaccine development because certain putative target genes may produce isoforms that escape the intervention. This has been postulated for the chloroquine resistance transporter gene of <em>P. falciparum</em> (<em>Pf</em>CRT) in clinical isolates, though the role of the splice variants remains unclear [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. In other organisms, there is some evidence showing that essential genes are more likely to have alternatively spliced transcripts compared to non-essential genes [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. This has not been explored in apicomplexans but highlights further considerations for investigating drug targets and interventions.</p><p id="p-7">The lack of data for apicomplexan gene isoforms is a major obstacle to dissecting the complexity of transcript outcomes. Traditionally, transcriptomic studies employing RNA-seq have relied on short read technologies such as Illumina, 454 and Ion-torrent [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Despite the power of very high sequencing depth and low error rates, the short reads present a limitation in that simultaneously occurring alternative-splicing events within individual transcripts cannot be unambiguously detected or linked. Previously-developed computational methods for full-length transcript assembly from short read sequencing data are often computationally intensive, and can produce ambiguous or conflicting results between different algorithms [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. In addition, sequencing on cDNA strands amplified by PCR has a propensity to introduce biases in relative transcript abundances and rare isoform identification [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. Hence, it is difficult to draw functional relationships between simultaneous alternative splicing events and observable phenotypes. In apicomplexan parasites, simultaneously occurring alternative splicing events within a specific transcript isoform do occur [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. However, the studies that unearthed these transcript isoforms relied on cDNA probes and reverse transcription PCR, and the wider extent of this phenomenon is unknown.</p><p id="p-8">Recently-developed third generation sequencing platforms, such as those developed by Oxford Nanopore Technologies (ONT) and Pacific Biosciences (PacBio), are capable of producing significantly longer reads at the single-molecule level. These technologies have been used in various applications such as resolving genomic and transcriptional landscapes [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], single cell transcriptome sequencing [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], and DNA or RNA methylation pattern profiling [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. PacBio has recently been used to generate an amplification-free transcriptome from <em>Plasmodium falciparum</em> cDNA, which has helped to elucidate transcriptional start sites and to improve annotation of the 5’and 3’ UTRs [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Unlike most other sequencing platforms, a notable characteristic of ONT sequencing is the ability to directly sequence native RNA [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. With this methodology, each read represents a complete molecular transcript, which could thus significantly resolve weaknesses of amplification-based RNA-seq. In particular, each spliced isoform need only be counted as individual reads, as opposed to complex assignment and assembly of multiple spliced reads. Furthermore, due to differences between DNA and RNA molecules, contaminating DNA sequences cannot be correctly base-called after sequencing and so are easily discarded [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. Recently, several studies have successfully applied single molecule, long-read sequencing to identify a high number of novel transcript isoforms [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. However, these studies have also identified several caveats including a reduced throughput and high error rates.</p><p id="p-9">In this study, we evaluate the ability ONT direct RNA sequencing to characterise the alternative splicing landscape of two parasitic apicomplexans, <em>T. gondii</em> and <em>P. falciparum.</em> Our analyses show that alternative splicing, particularly intron retention, is extensive throughout the transcriptome, with most multi-exon genes having some degree of intron retention, and some genes only rarely producing transcripts with all introns removed. The long reads produced from ONT sequencing showed that most of these alternative splicing events are likely non-productive in protein-coding capacity, but may provide an additional layer of gene expression regulation.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Direct RNA sequencing of <em>T. gondii</em> and <em>P. falciparum</em> allows the detection of full-length transcripts</h3><p id="p-10">We generated ONT sequencing reads of polyA-selected RNA from asynchronous <em>T. gondii</em> (Pru) tachyzoites and <em>P. falciparum</em> (3D7) mixed asexual intraerythrocytic stage parasites. Mixed-stage cultures were used to maximise transcript diversity for these samples. For <em>T. gondii</em>, we obtained a total of 310,813 reads corresponding to about 500 million bases (Mb). For <em>P. falciparum</em>, we obtained a total of 456,098 reads, corresponding to about 300 Mb of data. Although the <em>P. falciparum</em> sample yielded fewer sequenced bases, we estimate the theoretical gene coverage for both parasite samples to be similar at 25-26 fold due to differences in gene number and length. Using minimap2 [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], we successfully mapped 78.90% of the <em>T. gondii</em> reads, and 44.48% of the <em>P. falciparum</em> reads to the parasite genomes. We analysed the quality of the sequencing reads using FASTQC, and found consistently high-quality scores over the length of reads, with no drop-off in quality even at reads of 10 Kb (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1B</a>). This is important because base quality scores generally correlate with read accuracy to the reference sequence. That is indeed the case for our dataset (<a id="xref-fig-6-1" class="xref-fig" href="#F6">supplementary figure S1</a>). We used AlignQC to estimate the base-call error rate of the transcript reads based on aligned segments and found, on average, an error rate of 19-20% for both parasites. The read-length distributions of the mapped and unmapped data (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1A</a>) show that the mapped reads are predominantly longer than the unmapped reads, with some read lengths exceeding 10 kb. As expected, there was a sharp increase in unmapped read counts at the ∼1.35 kb length (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1A</a>) corresponding to yeast enolase 2, a calibration standard added during the library preparation. We calculated the average mapped read lengths to be ∼1.9 k and 1.3k bases for <em>T. gondii</em> and <em>P. falciparum</em> respectively, well within previous range estimates of predicted transcript lengths of both organisms [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>].</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplot shows the correlation between transcript identity (%) and average quality of the reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scatterplot shows the correlation between transcript identity (%) and average quality of the reads.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F6.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F6.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Supplementary Figure S1.</span> <p id="p-49" class="first-child">Scatterplot shows the correlation between transcript identity (%) and average quality of the reads.</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the ONT direct RNA sequencing data from T. gondii and P. falciparum. (A) Scatterplot of read length distribution of mapped (red) and unmapped (blue) reads. (B) Boxplot of quality scores across all bases at each position of the mapped sequencing reads. (C) Bubble scatter heat plots of the fraction of full length transcripts against transcript length. Size and color denotes expression and average coverage respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the ONT direct RNA sequencing data from &lt;em&gt;T. gondii&lt;/em&gt; and &lt;em&gt;P. falciparum&lt;/em&gt;. (A) Scatterplot of read length distribution of mapped (red) and unmapped (blue) reads. (B) Boxplot of quality scores across all bases at each position of the mapped sequencing reads. (C) Bubble scatter heat plots of the fraction of full length transcripts against transcript length. Size and color denotes expression and average coverage respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F1.medium.gif" width="269" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F1.medium.gif" width="269" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">Summary of the ONT direct RNA sequencing data from <em>T. gondii</em> and <em>P. falciparum</em>. (A) Scatterplot of read length distribution of mapped (red) and unmapped (blue) reads. (B) Boxplot of quality scores across all bases at each position of the mapped sequencing reads. (C) Bubble scatter heat plots of the fraction of full length transcripts against transcript length. Size and color denotes expression and average coverage respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">To evaluate the capability of ONT sequencing to generate full-length transcript reads, we re-mapped the reads to the parasites’ annotated transcriptome and calculated the fraction of full-length transcripts per gene. We define full-length transcripts as reads that cover more than 95% of the predicted canonical transcript based on the annotation file. As illustrated in the <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, many transcripts were observed to have full-length reads, more so for the <em>P. falciparum</em> data. In <em>T. gondii</em>, 1117 genes have 75% or more of their corresponding reads that were considered full length. In <em>P. falciparum</em>, this number is 1835 genes. The difference can be attributed to the absence of 5’ and 3’ UTR annotations for <em>P. falciparum</em>, which results in the underestimation of predicted transcript lengths. In both cases, the fraction of mapped reads corresponding to full-length transcripts fell with increasing transcript length, independent of expression levels, which is consistent with read truncation disproportionately affecting the longest reads. To better understand the overall distribution of transcript coverage, we calculated the average coverage of the transcripts per gene. Again, there is a general decrease in transcript coverage with increasing transcript length (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). However, many of the genes retain a high level of transcript coverage, even when there is a low fraction of full length reads. For example, in <em>T. gondii</em>, genes with predicted transcript lengths of 3 kb or longer only had an estimated 12% of their reads that were considered full length on average, even though the average read coverage for those genes is 50.64%. The overall average transcript coverage is calculated to be over 60% in both parasites (<em>T. gondii</em>: 64.73%, <em>P. falciparum</em>: 80.51%). Together, the data indicate a generally high proportion of full- or near full-length transcript reads.</p></div><div id="sec-4" class="subsection"><h3>ONT sequencing is comparable with traditional RNA-seq for quantifying gene expression levels</h3><p id="p-13">To investigate the utility of ONT data to measure transcript abundance, we computed read-count correlations between our ONT dataset and reanalysed published Illumina-based RNA-seq datasets on comparable parasite samples. In theory, ONT RNA sequencing reads directly correspond to complete transcripts and so quantifying the expression of genes can be done by simple counting of the assigned reads. This is dissimilar to traditional short read RNA-Seq which necessitates a further normalisation step (e.g., reads or fragments <em>per kilobase of transcript</em> or <em>transcripts per million</em>) to account for the higher number of reads that would be generated from longer transcripts. For <em>T. gondii</em>, we used Illumina datasets from a closely related strain (ME49) due to a relative lack of comprehensive datasets for the Pru strain. Strikingly, we observe strong positive correlations between the ONT and Illumina datasets (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A &amp; B</a>), regardless of mapping to the transcriptome or genome (Spearman’s rho = 0.81 for transcriptome, 0.87 for genome). For <em>P. falciparum</em> we correlated the mixed stage ONT dataset with individual datasets from three main developmental stages (rings, trophozoites and schizonts), and a final combined dataset. In all cases, we found moderately positive correlations between the datasets. As expected, the correlation is higher in the later stages (<a id="xref-fig-7-1" class="xref-fig" href="#F7">supplementary figure S2</a>; Spearman’s rho = 0.47 rings, 0.57 for trophozoites, 0.63 for schizonts), and the highest with the combined dataset (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A/B</a>; Spearman’s rho = 0.64 for transcriptome, 0.68 for genome), a reflection of mRNA abundance in these different stages.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplot shows the correlation between trancriptome mapped read counts for ONT direct RNA sequencing from mixed stage parasites and Illumina datasets from three developmental time points of P. falciparum. The Spearman correlation coefficient is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatterplot shows the correlation between trancriptome mapped read counts for ONT direct RNA sequencing from mixed stage parasites and Illumina datasets from three developmental time points of &lt;em&gt;P. falciparum&lt;/em&gt;. The Spearman correlation coefficient is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F7.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F7.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S2.</span> <p id="p-50" class="first-child">Scatterplot shows the correlation between trancriptome mapped read counts for ONT direct RNA sequencing from mixed stage parasites and Illumina datasets from three developmental time points of <em>P. falciparum</em>. The Spearman correlation coefficient is shown.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons between ONT direct RNA sequencing and Illumina datasets. (A) Correlation between trancriptome mapped read counts for T. gondii (left) and P. falciparum (right) as presented as a scatterplot. The Spearman correlation coefficient is shown. (B) Circos plots of genome mapped reads. Outer band (grey) represents the reference genome/chromosomes. The red and blue bands represent the genome coverage of ONT direct RNA and Illumina reads respectively. The purple band is the log2 fold change between the two datasets. The scatterplots within the circos plots show the correlation between the genome mapped read bin counts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons between ONT direct RNA sequencing and Illumina datasets. (A) Correlation between trancriptome mapped read counts for &lt;em&gt;T. gondii&lt;/em&gt; (left) and &lt;em&gt;P. falciparum&lt;/em&gt; (right) as presented as a scatterplot. The Spearman correlation coefficient is shown. (B) Circos plots of genome mapped reads. Outer band (grey) represents the reference genome/chromosomes. The red and blue bands represent the genome coverage of ONT direct RNA and Illumina reads respectively. The purple band is the log2 fold change between the two datasets. The scatterplots within the circos plots show the correlation between the genome mapped read bin counts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F2.medium.gif" width="274" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F2.medium.gif" width="274" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-14" class="first-child">Comparisons between ONT direct RNA sequencing and Illumina datasets. (A) Correlation between trancriptome mapped read counts for <em>T. gondii</em> (left) and <em>P. falciparum</em> (right) as presented as a scatterplot. The Spearman correlation coefficient is shown. (B) Circos plots of genome mapped reads. Outer band (grey) represents the reference genome/chromosomes. The red and blue bands represent the genome coverage of ONT direct RNA and Illumina reads respectively. The purple band is the log2 fold change between the two datasets. The scatterplots within the circos plots show the correlation between the genome mapped read bin counts.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">For both parasites, a higher number of gene transcripts were detected in the Illumina datasets than in the ONT data (supplementary table TS1). This is expected, given the greater sequencing depth that was obtained from the Illumina runs. The theoretical average fold coverage from our Illumina data is estimated to be 750 and 1000 times of the <em>T. gondii</em> and <em>P. falciparum</em> genes respectively (compared to 25-26 fold for the ONT data). Notably however, abundant non-mRNA species, particular ribosomal RNA, were detected in the Illumina datasets. This is absent in the ONT datasets, possibly because of differences in polyA RNA purification methodologies or biases due to PCR amplification. We further evaluated the ONT transcriptomes for genome coverage completeness as shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>. The read coverages from the ONT and Illumina reveal no significant bias towards a particular region of the nuclear genome.</p></div><div id="sec-5" class="subsection"><h3>Intron retained transcripts are prevalent and generally non-productive</h3><p id="p-16">A major goal of mature full-length transcript sequencing is the identification of splicing isoforms. Alternative splicing can be broadly classed into four types: intron retention, alternative 3′ splice site selection, alternative 5’ splice site selection, and exon skipping [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Of these, intron retention is the least studied form of alternative splicing despite the numerous studies implicating the significance of the event [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>-<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. This is in part due to the limitations of short read sequencing, but also the relatively long and low-complexity introns in metazoan genomes, which impose limitations on sequencing and assembly. For example, the intronic sequence in the human genome is several magnitudes longer than the length of the exonic sequence [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. In contrast, the compact genomes of <em>Plasmodium</em> and <em>Toxoplasma</em> both have gene models that have similar or longer exon lengths than introns [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>], so reads that span multiple entire introns are quite achievable for these organisms.</p><p id="p-17">To monitor levels of intron retention, we identified junctions and reads that overlapped annotated intronic regions of a gene based on the annotated coordinates using FeatureCounts [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>], and tallied the proportion of reads mapping to that intron to the total reads for the same gene. Proportion scores are represented using the metric <em>percent intron retention</em> (PIR). Based on this analysis, 17.65% of the mapped reads were considered to have intron overlaps for <em>T. gondii</em>, and 4.54% for <em>P. falciparum.</em> We further filtered out junctions without a minimum overlap of six bases to exclude artefacts generated by read errors, and excluded genes that were not supported by a minimum coverage of three reads. The distribution of PIR scores per gene (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) reveals an overall skew towards low proportions of intron-overlapping reads. This is as expected given the propensity for a dominant canonical transcript [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. However, we identify a strikingly high number of genes that retain a high level of intronic regions. Using a threshold of 10% PIR, we identified a total of 3229 genes for <em>T. gondii</em> and 978 genes for <em>P. falciparum</em> that have intronic reads within their transcripts (supplementary table TS2). Moreover, for around 29.82% (963/3229) of the <em>Toxoplasma</em> genes, and 19.63% (192/978) of <em>Plasmodium</em> genes, 50% or more of the reads retain at least one intron. Unusually, there are a considerable number of genes where none of the transcripts appears to have all of their annotated introns removed. We manually investigated these cases further and found major differences between the transcripts and gene model in most cases, suggesting that these highlight genes with incorrectly annotated structure. A couple of examples are outlined in <a id="xref-fig-8-1" class="xref-fig" href="#F8">supplementary figure S3</a>. Most of these genes are annotated as hypothetical proteins, highlighting the potential of ONT sequencing to validate gene models.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IGV snapshots of genes which were flagged as having full intron retention and do not appear to conform with the gene model. Left: T. gondii; Right: P. falciparum." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IGV snapshots of genes which were flagged as having full intron retention and do not appear to conform with the gene model. Left: &lt;em&gt;T. gondii&lt;/em&gt;; Right: &lt;em&gt;P. falciparum.&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F8.medium.gif" width="440" height="114"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F8.medium.gif" width="440" height="114"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S3.</span> <p id="p-51" class="first-child">IGV snapshots of genes which were flagged as having full intron retention and do not appear to conform with the gene model. Left: <em>T. gondii</em>; Right: <em>P. falciparum.</em></p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of intron retention using ONT direct RNA sequencing reads. Levels of intron retention are represented as Percent Intron Retention (PIR). (A) Distribution of intron retention levels over each gene as represented as total counts. (B) Distribution of intron retention levels over each junction as represented as the proportion of each bin count over the sum all intronic junctions. (C) Table of transcript productivity based on intron retention events as analysed using the FLAIR pipeline. Numbers in bracket are the transcript counts (D) Relationship between levels of intron retention and gene expression. Genes were classified into 10 bins of equal number based on expression. The median expression of the genes in each bin is used to represent expression for each bin. Error bars represent the 95% CI. (E) Comparisons of intron retention quantification between ONT and Illumina datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analysis of intron retention using ONT direct RNA sequencing reads. Levels of intron retention are represented as Percent Intron Retention (PIR). (A) Distribution of intron retention levels over each gene as represented as total counts. (B) Distribution of intron retention levels over each junction as represented as the proportion of each bin count over the sum all intronic junctions. (C) Table of transcript productivity based on intron retention events as analysed using the FLAIR pipeline. Numbers in bracket are the transcript counts (D) Relationship between levels of intron retention and gene expression. Genes were classified into 10 bins of equal number based on expression. The median expression of the genes in each bin is used to represent expression for each bin. Error bars represent the 95% CI. (E) Comparisons of intron retention quantification between ONT and Illumina datasets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F3.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F3.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-18" class="first-child">Analysis of intron retention using ONT direct RNA sequencing reads. Levels of intron retention are represented as Percent Intron Retention (PIR). (A) Distribution of intron retention levels over each gene as represented as total counts. (B) Distribution of intron retention levels over each junction as represented as the proportion of each bin count over the sum all intronic junctions. (C) Table of transcript productivity based on intron retention events as analysed using the FLAIR pipeline. Numbers in bracket are the transcript counts (D) Relationship between levels of intron retention and gene expression. Genes were classified into 10 bins of equal number based on expression. The median expression of the genes in each bin is used to represent expression for each bin. Error bars represent the 95% CI. (E) Comparisons of intron retention quantification between ONT and Illumina datasets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The most extreme cases of conflict between the junctions we detect and canonical gene models often highlight potential annotation errors, but there are still a strikingly high number of genes where genuine introns are retained in a high (&gt;50%) proportion of transcripts (<em>T. gondii</em>: 808 genes, <em>P. falciparum</em>: 162 genes). Additionally, the differences between the two parasites are striking. In many organisms, those transcripts with the most introns are those that are more likely to retain at least one or more introns [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. This is partially supported in our analysis, where we observe a higher level of overall intron retention for <em>T. gondii</em> (which has 4.5 introns per gene on average) than <em>P. falciparum</em> (which has only 1.5 introns per gene on average). A possible explanation for this relationship is thus that both organisms have a similar level of intron retention for any given junction, and the higher average intron number in <em>T. gondii</em> genes results in more overall intron retention per gene. To examine this, we calculated PIR scores at the individual junction level, rather than per-gene level, and normalised the count of each PIR value to the proportion of total junctions within each organism. The analysis reveals that after correction for intron number there is virtually no difference in the distribution of intron retention levels between parasites (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). In other words, individual <em>T. gondii</em> junctions are no more likely to experience intron retention than <em>P. falciparum</em> junctions. We further tested whether intron number was the major predictor of intron retention at the gene level by looking at the correlation between the number of introns per gene and levels of intron retention. Interestingly, we only obtained poor or moderate positive correlations in all datasets (<a id="xref-fig-9-1" class="xref-fig" href="#F9">supplementary figure S4</a>; Spearman’s rho = 0.28 for <em>T. gondii</em>, 0.48 for <em>P. falciparum</em>, 0.39 for pooled). This correlation does not significantly improve even when we restrict our analysis to higher (≥ 10 reads) expressed genes. This suggests that while intron number is associated with increased level of intron retention, it is not the main determinant of whether a gene retained at least one intron.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplots show the correlation between intron retention levels and intron number per gene. The Spearman correlation coefficient is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scatterplots show the correlation between intron retention levels and intron number per gene. The Spearman correlation coefficient is shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F9.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F9.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S4.</span> <p id="p-52" class="first-child">Scatterplots show the correlation between intron retention levels and intron number per gene. The Spearman correlation coefficient is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">By taking advantage of the full-length reads made possible by ONT, we are able to predict the protein-coding productivity of the alternate transcripts. We performed productivity analysis on full-length intron-retained reads using the FLAIR [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>] pipeline, which corrects and defines unproductive transcripts as transcripts with a termination codon that is 55 nucleotides or more upstream of the 3’-most splice junction. The rationale for this definition is based on previous evidence suggesting that only premature terminating transcripts following that 55-nucleotide rule mediate an effect on mRNA turnover [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. This is a conservative estimate of productivity as it does not consider intron retention within the 3’-most splice junction. The Flair method identified over 70% of the intron-retained reads to be non-productive for either parasite (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3C</a>), suggesting that the high level of observed intron retention only rarely corresponds to alternative protein products, and that most intron-retaining transcripts may instead be targets for nonsense mediated decay. Intron retention is known to fine-tune protein expression through this pathway in mammalian systems [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. A related prediction from other studies [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>] is that the most highly expressed transcripts should have low levels of intron retention. In our analysis, we do observe a negative relationship between intron retention and gene expression levels (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3D</a>). There is a relatively high variance for this, more so for genes with lower expression levels. This is likely due to the limitation in precision for the lower sequencing depths. For example, intron retention occurring in 10% of transcripts for a given gene will not be precisely measured for a gene for which only five reads are available. We circumvented this by classifying the transcripts into bins of equal read number based on expression and quantifying global intron retention levels within each bin. Again, we observe a negative relationship between intron retention and expression levels (<a id="xref-fig-10-1" class="xref-fig" href="#F10">supplementary figure S5</a>). To investigate the functional significance of this, we further analysed the genes for Gene Ontology (GO) enrichment. Here, we only considered genes with a minimum coverage of 10 reads to increase precision. The analyses reveal the consistent enrichment of genes with functions associated with the ribosome when there are lower levels of intron retention across both parasites (supplementary table TS3). This association has been previously observed in other organisms [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>], though its basis is unknown. We also tested whether intron retention correlated with essentiality based on previous functional genomic screens [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], and found no significant relationships (<a id="xref-fig-11-1" class="xref-fig" href="#F11">supplementary figure S6</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between levels of intron retention and gene expression. Introns were classified into 10 bins of equal read number based on expression (1 = lowest, 10 = highest). Levels of intron retention were calculated as the proportion of intron retained reads over all reads from each bin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between levels of intron retention and gene expression. Introns were classified into 10 bins of equal read number based on expression (1 = lowest, 10 = highest). Levels of intron retention were calculated as the proportion of intron retained reads over all reads from each bin.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F10.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F10.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S5.</span> <p id="p-53" class="first-child">Relationship between levels of intron retention and gene expression. Introns were classified into 10 bins of equal read number based on expression (1 = lowest, 10 = highest). Levels of intron retention were calculated as the proportion of intron retained reads over all reads from each bin.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplots show the relationship between intron retention levels and gene essentiality. Essentiality is represented as mean phenotype and mutagenesis fitness score for T. gondii and P. falciparum respectively" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatterplots show the relationship between intron retention levels and gene essentiality. Essentiality is represented as mean phenotype and mutagenesis fitness score for &lt;em&gt;T. gondii&lt;/em&gt; and &lt;em&gt;P. falciparum&lt;/em&gt; respectively&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F11.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F11.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S6.</span> <p id="p-54" class="first-child">Scatterplots show the relationship between intron retention levels and gene essentiality. Essentiality is represented as mean phenotype and mutagenesis fitness score for <em>T. gondii</em> and <em>P. falciparum</em> respectively</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To validate the identification of intron retention events, we looked at whether retained introns apparent in the ONT data were directly supported by Illumina RNAseq data. We normalised read counts by junction length and only considered intronic data that spanned the full junction. Based on the analysis, 77.88% for <em>T. gondii</em> and 87.37% for <em>P. falciparum</em> of the intronic junction reads flagged from the ONT datasets were supported by Illumina reads. However, we also noted that some alternative splicing events, particularly the lower frequency ones, failed to be captured by ONT sequencing compared to the Illumina dataset (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig 3E</a>). This is again likely due to the limitation in read depth in the ONT dataset. Based on our sequencing of polyA tailed material combined with previous kinetic studies [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], we do not expect the intron retained transcripts to simply be unprocessed transcripts. To confirm this, we looked for evidence that each transcript had at least been partially processed. On average, 92.76% of multi-intron genes identified as having intron retention within their transcripts had at least 1 junction which was canonically spliced in all the transcripts, demonstrating that cannot be attributed to sequencing of pre-mRNA.</p></div><div id="sec-6" class="subsection"><h3>Alternate junction splicing is often proximal and non-productive</h3><p id="p-22">Having previously identified intron-retained, read junctions using an annotated gene model approach, we used RSeQC to identify and quantify the other three classes of alternative splicing read junctions (exon skipping, 5’-, and 3’-splice site change) based on a similar methodology. Levels of alternative spliced junctions are calculated as the proportion of alternate junction over the total junction reads, and are represented using the metric percent spliced (PS). Here, we filtered out junctions unsupported by a minimum coverage of three reads. Using the same threshold as before (≥ 10%), we identified a total of 1138 genes for <em>T. gondii</em>, and 168 genes for <em>P. falciparum</em>, where one or more of their junctions exhibited alternative 5’/3’ splice site selection or exon skipping (supplementary table TS4_1 &amp; 2). Remarkably, these aggregate numbers are lower than those we calculated for intron retention alone. Combining the datasets, intron retention accounts for 60-68% of alternatively spliced genes identified, alternate 5’ junction and 3’ junction splicing for 13-19% and 6-11% respectively, and exon skipping for less than 3% (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). The rest of the junctions flagged in the analysis defy easy categorisation due to major mismatches between the RNAseq data and the annotated gene model. Subsets of genes were also found to have multiple alternative splicing type events within their transcripts as observed in <a id="xref-fig-12-1" class="xref-fig" href="#F12">supplementary figure S7</a>, though there does not appear to be a particular functional trend.</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams display the number of genes with exclusive or overlapping intron retention, alternative 5&#x2019; splicing and alternative 3&#x2019; splicing events." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Venn diagrams display the number of genes with exclusive or overlapping intron retention, alternative 5&#x2019; splicing and alternative 3&#x2019; splicing events.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F12.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F12.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S7.</span> <p id="p-55" class="first-child">Venn diagrams display the number of genes with exclusive or overlapping intron retention, alternative 5’ splicing and alternative 3’ splicing events.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary analysis of alternative splicing from the ONT direct RNA sequencing datasets. Pie charts show the number, proportion and categorisation of genes with alternatively spliced transcripts equaling or exceeding 10% of its total transcript. An example of each event is presented. Red and blue represents sense and anti-sense transcripts respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary analysis of alternative splicing from the ONT direct RNA sequencing datasets. Pie charts show the number, proportion and categorisation of genes with alternatively spliced transcripts equaling or exceeding 10% of its total transcript. An example of each event is presented. Red and blue represents sense and anti-sense transcripts respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F4.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F4.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-23" class="first-child">Summary analysis of alternative splicing from the ONT direct RNA sequencing datasets. Pie charts show the number, proportion and categorisation of genes with alternatively spliced transcripts equaling or exceeding 10% of its total transcript. An example of each event is presented. Red and blue represents sense and anti-sense transcripts respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Having identified junctions subject to alternative splicing, we then quantified what proportion of the transcripts produced at those junctions represented the non-canonical isoform. For alternative splicing involving 5’ or 3’ changes and for intron retention, substantial proportions of isoforms were represented by the alternative transcript, but the median abundance remained below 50% (data shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>). However, whilst exon skipping was a relatively rare event across the genome (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>), for those genes where exon skipping did occur, it represented a higher proportion (approximately two-thirds) of transcript isoforms for those genes than observed for the other forms of alternative splicing (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Levels of each major alternative splice type as represented as Percent Splice (PS). (B) Distribution of alternate 5&#x2019;/3&#x2019; splice positions within 5 bases to the dominant splice site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1335181490" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Levels of each major alternative splice type as represented as Percent Splice (PS). (B) Distribution of alternate 5&#x2019;/3&#x2019; splice positions within 5 bases to the dominant splice site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F5.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F5.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/17/2020.02.16.946699/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-25" class="first-child">(A) Levels of each major alternative splice type as represented as Percent Splice (PS). (B) Distribution of alternate 5’/3’ splice positions within 5 bases to the dominant splice site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To further explore consequences of alternate 5’ or 3’ junction splicing events, we investigated the length distribution of the change in intron length. We found a surprisingly high proportion (∼50%) of alternate 5’/3’ splicing to occur proximal (&lt;6 bases) to the expected canonical site. We graphed the distribution of splicing change positions in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a> and found a substantial spike of splicing changes occurring at the position four bases inside the canonical intron boundary. We tested these junction reads for productivity for a subset of 50 of these “near-miss” alternate splice events, and found all reads to prematurely terminate (unsurprisingly given the necessary frame-shift). This striking over-representation of isoforms departing from the canonical model by specifically four bases has been previously identified in metazoans [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. Very small movements in splice site usage have been described as junction wobbling, and this has been proposed as minor splicing noise, or alternatively, as an additional mechanism of regulation through the NMD pathway [<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">60</a>], although the reason for the specific peak of AS four bases away from the canonical junction is unknown.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-27">Despite the apparent significance of alternative splicing in metazoans, very little is known about the process in apicomplexans. A number of targeted experiments highlight single splicing events and their impact on parasite survival, but global splicing networks have been poorly described. Untargeted RNA-seq experiments have mainly focused on whole transcriptome assembly and/or gene expression. Those studies that do monitor alternative splicing reveal its occurrence in multiple genes and stages, but the extent of these events and their phenotypic significance remains unknown. The lack of a robust methodology in defining transcript isoforms from short read data is a particular challenge in dissecting whole-transcriptome splicing. In this study, we investigated whether ONT direct RNA sequencing could be used to explore the splicing landscape of two apicomplexan parasites- <em>T. gondii</em> and <em>P. falciparum.</em></p><p id="p-28">To our knowledge, ONT direct RNA sequencing has not been previously described in apicomplexans and so our first objective was to evaluate the capability of ONT sequencing in generating sequencing reads from these parasites. We successfully obtained high quality sequencing reads for both parasites that were comparable to that previously described in the literature for other organisms [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]. In particular, we obtained read lengths that exceeded 1kb on average, many of them predicted to represent full-length or near full-length transcripts. Interestingly, although we obtained a higher number of sequencing reads for <em>P. falciparum</em>, the mapping of the reads was suboptimal compared to that of <em>T. gondii</em>. Repetitive DNA sequence motifs are characteristic of many large eukaryotic genomes and this has been known to complicate the mapping of reads that cannot be confidently assigned to these particular regions [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]. In theory, long read sequencing mitigates this problem because long enough reads should unambiguously match a unique site on the genome, irrespective of low complexity or repeat sequences. However, the genome of <em>P. falciparum</em> is particularly AT-rich (∼82%) with numerous regions of extreme low complexity [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Thus, we may expect some reads, particularly the shorter ones, to fail the mapping parameters. Indeed, as indicated above, reads that fail to map to the genome tended to be shorter than reads that do. This is further exacerbated by the high error rate produced from ONT sequencing. Based on previous experiments, the per base error rate of direct RNA sequencing using ONT is 10-20% [<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a>, <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>]. In our dataset, we estimated the error rate to be around 20%. This may have further contributed to the poorer mapping, though we do not expect the high error rate to significantly impact our study because the main analysis is focused on splice connectivity, rather than base sequences. As has happened for ONT DNA sequencing, we are likely to see significant improvements in read and mapping accuracy of RNA sequences as improvements are made to the flow cell and base caller. A study carried out by Runtuwe and colleagues is an elegant example, where ONT DNA sequencing on targeted <em>P. falciparum</em> genes yielded a mapping percentage that improved from 57.86% to 92.46% with improved chemistry of the flow cell, and upgrades to the base-calling algorithms [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>].</p><p id="p-29">The quantification of gene expression is one important goal of RNA-seq. Traditional, short-read sequencing requires the generation and amplification of complementary DNA (cDNA) which can introduce artefacts and biases. Transcriptional amplification or repression is a commonly-overlooked bias [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>], where the levels of global mRNA, rather than specific mRNA, may be variable between different samples. Thus, using a standard amount of total RNA, as is commonly done, can mask actual detection of specific mRNA levels, even after normalisation [<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>]. Direct RNA sequencing allows these caveats to be bypassed because a standard amount of isolated mRNA instead is used as the sequencing material. However, because there is no amplification step, direct RNA sequencing is limited by the amount of mRNA that can be practically obtained and used in the sequencing process. Without sufficient sequencing material, it can be difficult to achieve the high levels of sequencing depth that is needed to analyse gene expression [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>]. In line with the literature, our analysis shows that the current protocol for ONT direct RNA sequencing is comparable to Illumina for quantifying gene expression in the organisms we analysed. It can be noted however that sequencing depth remains the main limitation of ONT sequencing in our study. The reduced throughput and sequencing depth from ONT sequencing compared to Illumina sequencing means that genes or transcript isoforms with low expression may not be captured.</p><p id="p-30">Several previous analyses have reported differences in alternative splicing types and levels among different organisms [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>-<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>]. Notably, the increase in intron number and its retention correlates strongly with multicellular complexity (as defined by numbers of distinct cell types) [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>]. In apicomplexans, the splicing machinery appears to be largely conserved but features of gene structure such as intron number, length and distribution can be highly variable [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. In our study, the difference in intron number between <em>T. gondii</em> and <em>P. falciparum</em> is a relevant example. Despite the differences, we found that alternative splicing for any given junction occurs at similar rates between the two parasites. This further supports the notion that the parasites share similar splicing processes. Intron number of genes is predicted to be positively associated with alternative splicing events [<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>]. This is not simply a stochastic effect but rather related to the general decrease in 5’ splice site strength with increasing intron number in many organisms [<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">73</a>]. As described above however, there is only a weak or moderately positive correlation between intron number and intron retention level of genes in the two parasites studied. Schmitz et al. [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>] previously reported that other features such AT content and splice site entropy are important modulators of intron retention. This may also be the case in our study, given the observation that certain splice junctions are predisposed to retain their intron over others.</p><p id="p-31">In addition to the difference in alternative splicing levels, there are differences in the composition of alternative splicing types between different organisms as reported by Kim and colleagues [<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">70</a>] and McGuire and colleagues [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>]. For example, exon skipping is the predominant form of alternative splicing in metazoans, and intron retention the rarest [<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">70</a>]. Our analysis reveals that the opposite occurs in <em>T. gondii</em> and <em>P. falciparum</em>, with intron retention being the predominant event to occur, and exon skipping the rarest. This composition of alternative splicing type is similar to that observed for plants and fungi [<a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">70</a>, <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a>, <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>], though the reason is unclear. More recent studies find that intron retention has been previously under-detected in metazoans due to methodology limitations or confounding variables, but the high levels of exon skipping has been mostly undisputed [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. Kim and colleagues [<a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">70</a>] speculated that intron retention emerged as the earliest form of alternative splicing, before other mechanisms of complex splicing events evolved. There is some evidence for this, including the apparent shift towards increased exon skipping frequencies in early branching animals [<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">76</a>]. This is associated with the preservation of coding frames, suggesting a role of exon skipping in expanding proteome diversity [<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">76</a>]. In contrast to this, we found that the majority of the splicing events in <em>P. falciparum</em> and <em>T. gondii</em>, particularly intron retention, results in non-productive transcripts. Our results thus indicate that alternative splicing rarely contributes to generating diversity of protein sequence in these parasites, and may relate instead to transcriptomic complexity that impacts protein abundance. If that is true, a previous analysis that showed alternative splicing to be essential for <em>Plasmodium</em> stage differentiation [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>] may possibly be explained by a requirement for modulation of abundance for specific proteins, rather than generation of protein sequences.</p><p id="p-32">Consistent with our observation of alternative splicing playing a minor role in generating true proteome diversity in apicomplexans, many splicing events in other eukaryotes contribute little to the protein isoform repertoire. In particular, many transcripts contain premature termination codons (PTCs), at least in humans and yeast [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>]. Often, PTC transcripts are the result of the retention of intronic sequences that contain PTCs [<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>], but translational frameshifts from active splicing events such as alternative splice site selections have been similarly implicated [<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">78</a>, <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. PTC transcripts are not normally translated but rather targeted for degradation through the nonsense-mediated decay (NMD) pathway [<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>]. This is vital because the transcripts encode altered or truncated proteins which may exhibit deleterious activity [<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>]. Some studies postulate that the predicted alternative splice events are the result of either experimental or transcriptional noise [<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>], or that a substantial portion of such transcripts are contaminating pre-mRNA molecules, and so do not represent true alternative splicing [<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>, <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>]. Nevertheless, many RNA-seq-based analyses operate on the assumption that PTC transcripts are biologically significant or relevant [<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>]. Congruously, studies focusing on mature mRNA isoforms in other organisms suggest that non-productive transcripts mediate an additional layer of post-transcriptional regulation, through downstream RNA processing changes such as mRNA turnover, export, and microRNA silencing [<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>, <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>, <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>]. Alternative splicing in apicomplexans may also play a role in these processes. Strikingly, unique PTC transcript signatures are associated with distinct cell lineages [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>, <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>] in multicellular eukaryotes, which may be analogous to the essential role of alternative splicing observed in stage-differentiation observed in <em>Plasmodium</em> [<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>].</p><p id="p-33">Non-productive transcripts are typically degraded though the NMD pathway, and this has been shown to regulate gene expression at the post-transcriptional level [<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a>]. However, it is difficult to conclusively define the function of the non-productive transcripts without experimentally testing the proteomic fates of these transcripts. In metazoans, non-productive transcripts often highlight genes that were downregulated following a transition of cellular states [<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">91</a>]. Our study is consistent with this, given the observation that the number of non-productive transcripts generally decreased with increasing transcript number. This, in association with NMD, has been shown to be crucial to the maintenance and differentiation of many cell types [<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a>, <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a>]. In contrast, in organisms such <em>Paramecium tetraurelia</em>, non-productive transcripts appear to be the result of splicing error rather than function [<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a>]. Regardless, because gene expression as measured by transcript levels do not necessarily translate to protein expression levels, our findings have potential implications for the interpretation of RNA-seq studies in these parasites. Several studies have already demonstrated the poor correlation between protein and mRNA expression profiles in apicomplexans [<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a>]. Our results highlight that for many genes, raw quantifications of transcript abundance will correlate poorly with the number of copies of productive isoforms, and provides one source of mismatch between transcriptional initiation and protein abundance.</p><p id="p-34">Genome annotation is a crucial element of RNA-seq data analysis. For <em>T. gondii</em> and <em>P. falciparum</em>, the task is a widely accomplished manual effort from experts in the research community. Although genome annotation was not the main focus of the study, the ONT datasets are able to reveal the structure of full-length transcripts. This is crucial in validating gene models. Our data are viewable through the ToxoDB and PlasmoDB web resources [<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a>], and raw data are available at the Sequence Read Archive (SRA) (<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA606986">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA606986</a>) which may aid the research community to further curate and validate the current annotations.</p></div><div class="section" id="sec-8"><h2 class="">Conclusion</h2><p id="p-35">In this study, we have performed the first direct transcriptomic analyses on <em>T. gondii</em> and <em>P. falciparum.</em> We show that ONT direct RNA sequencing enables the quantification of gene expression despite a reduced throughput. In combination with the increased requirement for starting material, this means that the cost and time per bp sequenced remains higher than that of second-generation sequencing platforms. Nevertheless, because ONT direct RNA sequencing enables the detection of full-length transcripts without amplification, the tool remains promising for resolving the limitations of second-generation sequencing.</p><p id="p-36">We demonstrated that alternative splicing is widespread in the two parasites, particularly intron retention. ONT direct RNA sequencing enabled us to determine the productivity of these transcripts without complex computational methodologies, and we show that most the transcripts are premature terminating. This has implications for the quantification of gene expression, as it is highly unlikely for the wealth of transcript diversity that we identified to directly translate to protein isoforms.</p></div><div class="section" id="sec-9"><h2 class="">Methods</h2><div id="sec-10" class="subsection"><h3>Cell culture and RNA extraction</h3><p id="p-37"><em>Toxoplasma gondii</em> tachyzoites (Pru <em>Δku80</em>) were cultured on human foreskin fibroblasts in Dulbecco’s Modified Eagle medium (DME) supplemented with 1% v/v fetal calf serum and 1% v/v Glutamax. Freshly egressed tachyzoites were washed, filter purified (5um) and collected for RNA extraction. <em>P. falciparum</em> (3D7) were cultured in complete media consisting of human erythrocytes (O+, 2% haematocrit), RPMI-HEPES, 5% (w/v) Albumax and 3.6% (w/v) sodium bicarbonate. We collected mixed stage parasite, purified from host RBCs via lysis with 0.05% (w/v) saponin, for RNA extraction. To obtain the 500ng of mRNA recommended for the library preparation, we used TRI Reagent (Sigma) for extraction of total RNA followed by the Dynabeads mRNA Purification Kit for polyadenylated (poly-A) mRNA (Thermo Scientific) purification according to the manufacturer’s protocol. Purity and quantification of mRNA were determined via NanoDrop (Thermo Scientific) and a Qubit RNA HS Assay kit (Thermo Scientific).</p></div><div id="sec-11" class="subsection"><h3>Library prep and Nanopore sequencing</h3><p id="p-38">Libraries for the direct RNA sequencing were generated using the recommended protocol for the SQK-RNA001 kit (Oxford Nanopore Technologies). We loaded and sequenced the libraries on MinION R9.4 flow cells (Oxford Nanopore Technologies) for 48 hours. Base calling was performed concurrent with sequencing using Albacore (v 2.0), which was integrated within the MinION software (MinKNOW, v1.10.23). Only “pass” reads were used for subsequent analyses.</p></div><div id="sec-12" class="subsection"><h3>Mapping and qualitative analysis</h3><p id="p-39">ONT sequencing data was first checked for quality with FastQC (v.0.11.7) [<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a>]. We then utilised Minimap2 (v. 2.1) [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>] to map raw reads to the parasite genome and transcriptome from ToxoDB and PlasmoDB (r. 39), using the recommended preset commands. Intron length thresholds were set at 5000 and 1500 bases for <em>T. gondii</em> and <em>P. falciparum</em> respectively. Previously published Illumina datasets (SRR350746, ERR174301, ERR185969, ERR185970, ERR185971) were mapped using HISAT2 [<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a>] using the preset commands. We checked for mapping quality with Samtools (v.1.7) [<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a>], Picard (v.2.18.2) [<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">101</a>] and AlignQC (v.1.2) [<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">102</a>]. Further qualitative or quantitative analyses and graphical elements were done using the command-line interface and RStudio. We verified and illustrated subsets of mapped reads via IGV [<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">103</a>].</p><p id="p-40">We correlated the ONT sequencing data with the Illumina datasets as previously described [<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">104</a>] using the wub package (v.0.2) [<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">105</a>]. The genome coverage of sequencing datasets were generated using bedtools genome coverage (v2.27) [<a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">106</a>] and visualised via J-Circos (v1) [<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">107</a>]. Log2-fold ratios were calculated using DeepTools bamCompare (v.2.5.1) [<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">108</a>].</p></div><div id="sec-13" class="subsection"><h3>Alternative splicing analysis</h3><p id="p-41">We applied two approaches to analysing alternative splicing. We first identified intron retained junctions and transcripts using featureCounts (v.1.6.2) [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>] on the genome mapped reads. featureCounts matches features specified in an annotation file (gff) to mapped reads. The annotation files used in the analyses were obtained from ToxoDB and PlasmoDB (r. 39), and preprocessed via ToolShed (v.1.0) [<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">109</a>] to specifically extract intron coordinates and gene IDs. We set a minimum threshold requiring mapping to at least six-bases of the intron feature, and a minimum threshold of three reads mapping to the junction/transcript to be considered for further analysis. Percent Intron Retention (PIR) scores were calculated as the proportion of alternative splicing events to the sum of reads for each junction/gene. Productivity of full length transcripts was analysed using the Flair pipeline [<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">51</a>] using default parameters.</p><p id="p-42">For the second approach, we used the junction_annotation.py script of RSeQC (v.2.6.4) [<a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">110</a>] to identify novel or partial-novel junctions from the genome mapped reads based on the unmodified annotation file. Again, we filtered out junctions that had fewer than three supporting reads. The junctions were summarised into a table based on coordinate matching to the 5’ and/or 3’ of the expected canonical junction. We identified alternative 5’/3’ splicing and exon skipping based on the coordinates and strandedness of junctions identified by RSeQC that were either consistent with or conflicted with the annotated junctions. We manually validated the data, matched junctions to available gene IDs, and again calculated the proportion of alternative splicing events to the sum of reads for each junction. Using the final dataset, we re-curated the list of intron-retained junctions to exclude for alternate 5’/3’ splice changes. Proportional Venn diagrams were drawn using BioVenn [<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">111</a>].</p><p id="p-43">Gene set enrichment analyses were carried out by ranking the genes based on their alternative splicing levels and using the first and third quartile of the ranked list as input for GO enrichment analysis via ToxoDB/PlasmoDB based on curated and computed assigned associations. We required the adjusted p-value to be smaller than 0.05 and FDR q-value of less than 0.25. This approach was validated using GSEA via WebGestalt [<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">112</a>].</p></div></div><div class="section" id="sec-15"><h2 class="">Supplementary files</h2><p id="p-44">Supplementary table TS1. An Excel spreadsheet quantifying the expression of genes from Illumina and ONT data.</p><p id="p-45">Supplementary table TS2. An Excel spreadsheet quantifying the levels of intron retention at the gene level.</p><p id="p-46">Supplementary table TS3. An Excel spreadsheet summarising the GO enrichment results for genes with high and low levels of intron retention.</p><p id="p-47">Supplementary table TS4_1. An Excel spreadsheet quantifying categories of alternative splicing at the junction level for <em>T. gondii.</em></p><p id="p-48">Supplementary table TS4_2. An Excel spreadsheet quantifying categories of alternative splicing at the junction level for <em>P. falciparum.</em></p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.1" data-doi="10.1016/j.cell.2016.07.055"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cowman</span>,  <span class="cit-name-given-names">A.F.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Malaria: Biology and Disease</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">167</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">610</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D610%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.07.055%26rft_id%253Dinfo%253Apmid%252F27768886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.07.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27768886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.2" data-doi="10.2471/BLT.12.111732"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Torgerson</span>,  <span class="cit-name-given-names">P.R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Mastroiacovo</span></span>, <span class="cit-article-title">The global burden of congenital toxoplasmosis: a systematic review</span>. <abbr class="cit-jnl-abbrev">Bull World Health Organ</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">91</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">501</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.volume%253D91%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.2471%252FBLT.12.111732%26rft_id%253Dinfo%253Apmid%252F23825877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2471/BLT.12.111732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23825877&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321884400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furtado</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Toxoplasmosis: A Global Threat</span>. <abbr class="cit-jnl-abbrev">Journal of Global Infectious Diseases</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">3</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">281</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGlobal%2BInfectious%2BDiseases%26rft.volume%253D3%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.4" data-doi="10.1016/j.molbiopara.2008.11.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Horrocks</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Control of gene expression in Plasmodium falciparum - ten years on</span>. <abbr class="cit-jnl-abbrev">Mol Biochem Parasitol</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">164</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">9</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DHorrocks%26rft.auinit1%253DP.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D25%26rft.atitle%253DControl%2Bof%2Bgene%2Bexpression%2Bin%2BPlasmodium%2Bfalciparum%2B-%2Bten%2Byears%2Bon.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2008.11.010%26rft_id%253Dinfo%253Apmid%252F19110008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2008.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19110008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263451500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.5" data-doi="10.1016/0166-6851(89)90119-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wesseling</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Stage-specific expression and genomic organization of the actin genes of the malaria parasite Plasmodium falciparum</span>. <abbr class="cit-jnl-abbrev">Mol Biochem Parasitol</abbr>, <span class="cit-pub-date">1989</span>. <span class="cit-vol">35</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">167</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DWesseling%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D167%26rft.epage%253D176%26rft.atitle%253DStage-specific%2Bexpression%2Band%2Bgenomic%2Borganization%2Bof%2Bthe%2Bactin%2Bgenes%2Bof%2Bthe%2Bmalaria%2Bparasite%2BPlasmodium%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0166-6851%252889%252990119-9%26rft_id%253Dinfo%253Apmid%252F2671721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0166-6851(89)90119-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2671721&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.6" data-doi="10.1186/1471-2164-12-587"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lopez-Barragan</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Directional gene expression and antisense transcripts in sexual and asexual stages of Plasmodium falciparum</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">12</span>: p. <span class="cit-fpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLopez-Barragan%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D12%26rft.spage%253D587%26rft.epage%253D587%26rft.atitle%253DDirectional%2Bgene%2Bexpression%2Band%2Bantisense%2Btranscripts%2Bin%2Bsexual%2Band%2Basexual%2Bstages%2Bof%2BPlasmodium%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-587%26rft_id%253Dinfo%253Apmid%252F22129310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22129310&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.7" data-doi="10.1101/gr.2523904"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Roch</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Global analysis of transcript and protein levels across the Plasmodium falciparum life cycle</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">14</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">2308</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLe%2BRoch%26rft.auinit1%253DK.%2BG.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D2308%26rft.epage%253D2318%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Btranscript%2Band%2Bprotein%2Blevels%2Bacross%2Bthe%2BPlasmodium%2Bfalciparum%2Blife%2Bcycle%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2523904%26rft_id%253Dinfo%253Apmid%252F15520293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yMzA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.8" data-doi="10.1186/gb-2008-9-12-r177"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foth</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Quantitative protein expression profiling reveals extensive post-transcriptional regulation and post-translational modifications in schizont-stage malaria parasites</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">R177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DFoth%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253DR177%26rft.epage%253DR177%26rft.atitle%253DQuantitative%2Bprotein%2Bexpression%2Bprofiling%2Breveals%2Bextensive%2Bpost-transcriptional%2Bregulation%2Band%2Bpost-translational%2Bmodifications%2Bin%2Bschizont-stage%2Bmalaria%2Bparasites.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-12-r177%26rft_id%253Dinfo%253Apmid%252F19091060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-12-r177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19091060&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Toenhake</span>,  <span class="cit-name-given-names">C.G.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bártfai</span></span>, <span class="cit-article-title">What functional genomics has taught us about transcriptional regulation in malaria parasites</span>. <abbr class="cit-jnl-abbrev">Briefings in Functional Genomics</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.16.946699v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Llora-Batlle</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tinto-Font</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cortes</span></span>, <span class="cit-article-title">Transcriptional variation in malaria parasites: why and how</span>. <abbr class="cit-jnl-abbrev">Brief Funct Genomics</abbr>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.11" data-doi="10.1186/s13059-019-1756-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeoh</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternative splicing is required for stage differentiation in malaria parasites</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">20</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1756-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-019-1756-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.12" data-doi="10.1016/j.semcdb.2015.10.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mockenhaupt</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E.V.</span> <span class="cit-name-surname">Makeyev</span></span>, <span class="cit-article-title">Non-coding functions of alternative pre-mRNA splicing in development</span>. <abbr class="cit-jnl-abbrev">Seminars in cell &amp; developmental biology</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">47-48</span>: p. <span class="cit-fpage">32</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.volume%253D4748%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2015.10.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2015.10.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.13" data-doi="10.1186/1471-2105-6-266"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neverov</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternative splicing and protein function</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNeverov%26rft.auinit1%253DA.%2BD.%26rft.volume%253D6%26rft.spage%253D266%26rft.epage%253D266%26rft.atitle%253DAlternative%2Bsplicing%2Band%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-266%26rft_id%253Dinfo%253Apmid%252F16274476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16274476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.14" data-doi="10.1016/j.gene.2012.07.083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelemen</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Function of alternative splicing</span>. <abbr class="cit-jnl-abbrev">Gene</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">514</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D514%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2012.07.083%26rft_id%253Dinfo%253Apmid%252F22909801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2012.07.083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22909801&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314321100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.15" data-doi="10.1093/nar/gkm1054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Birzele</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Csaba</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zimmer</span></span>, <span class="cit-article-title">Alternative splicing and protein structure evolution</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">36</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">550</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm1054%26rft_id%253Dinfo%253Apmid%252F18055499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm1054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18055499&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253021600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.16" data-doi="10.1038/nature07509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">E.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternative isoform regulation in human tissue transcriptomes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">456</span>(<span class="cit-issue">7221</span>): p. <span class="cit-fpage">470</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWang%26rft.auinit1%253DE.%2BT.%26rft.volume%253D456%26rft.issue%253D7221%26rft.spage%253D470%26rft.epage%253D476%26rft.atitle%253DAlternative%2Bisoform%2Bregulation%2Bin%2Bhuman%2Btissue%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07509%26rft_id%253Dinfo%253Apmid%252F18978772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978772&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261170500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.17" data-doi="10.1038/nrg2673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Licatalosi</span>,  <span class="cit-name-given-names">D.D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.B.</span> <span class="cit-name-surname">Darnell</span></span>, <span class="cit-article-title">RNA processing and its regulation: global insights into biological networks</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">11</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">75</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLicatalosi%26rft.auinit1%253DD.%2BD.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D87%26rft.atitle%253DRNA%2Bprocessing%2Band%2Bits%2Bregulation%253A%2Bglobal%2Binsights%2Binto%2Bbiological%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2673%26rft_id%253Dinfo%253Apmid%252F20019688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20019688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeoh</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternative Splicing in Apicomplexan Parasites</span>. <abbr class="cit-jnl-abbrev">mBio</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">e02866</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D10%26rft.spage%253De02866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.19" data-doi="10.1074/jbc.M201748200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Dooren</span>,  <span class="cit-name-given-names">G.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Processing of an apicoplast leader sequence in Plasmodium falciparum, and the identification of a putative leader cleavage enzyme</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">277</span>(<span class="cit-issue">26</span>): p. <span class="cit-fpage">23612</span>–<span class="cit-lpage">23619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M201748200%26rft_id%253Dinfo%253Apmid%252F11976331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMjYvMjM2MTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNy8yMDIwLjAyLjE2Ljk0NjY5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.20" data-doi="10.1042/BJ20131451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pham</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A dual-targeted aminoacyl-tRNA synthetase in Plasmodium falciparum charges cytosolic and apicoplast tRNACys</span>. <abbr class="cit-jnl-abbrev">Biochem J</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">458</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">513</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20131451%26rft_id%253Dinfo%253Apmid%252F24428730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDU4LzMvNTEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.21" data-doi="10.1093/jac/dku358"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gadalla</span>,  <span class="cit-name-given-names">N.B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternatively spliced transcripts and novel pseudogenes of the Plasmodium falciparum resistance-associated locus pfcrt detected in East African malaria patients</span>. <abbr class="cit-jnl-abbrev">The Journal of antimicrobial chemotherapy</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">70</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">116</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bantimicrobial%2Bchemotherapy%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdku358%26rft_id%253Dinfo%253Apmid%252F25253286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/dku358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25253286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.22" data-doi="10.1038/ng.3012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jex</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genome and transcriptome of the porcine whipworm Trichuris suis</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">46</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">701</span>–<span class="cit-lpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D46%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3012%26rft_id%253Dinfo%253Apmid%252F24929829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24929829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lees</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.A.</span> <span class="cit-name-surname">Ranea</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.A.</span> <span class="cit-name-surname">Orengo</span></span>, <span class="cit-article-title">Identifying and characterising key alternative splicing events in Drosophila development</span>. <abbr class="cit-jnl-abbrev">BMC genomics</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">608</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D16%26rft.spage%253D608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.24" data-doi="10.1038/nrg.2016.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.D.</span> <span class="cit-name-surname">McPherson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.R.</span> <span class="cit-name-surname">McCombie</span></span>, <span class="cit-article-title">Coming of age: ten years of next-generation sequencing technologies</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">17</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">333</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D17%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.49%26rft_id%253Dinfo%253Apmid%252F27184599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27184599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.25" data-doi="10.1101/gr.131383.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">GAGE: A critical evaluation of genome assemblies and assembly algorithms</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">22</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131383.111%26rft_id%253Dinfo%253Apmid%252F22147368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">291</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D6%26rft.spage%253D291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.27" data-doi="10.1074/jbc.M503178200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Differential localization of alternatively spliced hypoxanthine-xanthine-guanine phosphoribosyltransferase isoforms in Toxoplasma gondii</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">280</span>(<span class="cit-issue">23</span>): p. <span class="cit-fpage">22053</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M503178200%26rft_id%253Dinfo%253Apmid%252F15814612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjMvMjIwNTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNy8yMDIwLjAyLjE2Ljk0NjY5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.28" data-doi="10.1093/nar/gky014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenjaroenpun</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Complete genomic and transcriptional landscape analysis using third-generation sequencing: a case study of Saccharomyces cerevisiae CEN.PK113-7D</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">46</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">e38</span>–<span class="cit-lpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253De38%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.29" data-doi="10.1038/nbt.4060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jain</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">36</span>: p. <span class="cit-fpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D36%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4060%26rft_id%253Dinfo%253Apmid%252F29431738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29431738&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byrne</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">8</span>: p. <span class="cit-fpage">16027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D8%26rft.spage%253D16027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detecting DNA cytosine methylation using nanopore sequencing</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">14</span>: p. <span class="cit-fpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D14%26rft.spage%253D407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Workman</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nanopore native RNA sequencing of a human poly(A) transcriptome</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2018</span>: p. <span class="cit-fpage">459529</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chappell</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Refining the transcriptome of the human malaria parasite Plasmodium falciparum using amplification-free RNA-seq</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2019</span>: p. <span class="cit-fpage">852038</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garalde</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">201</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D15%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.35" data-doi="10.1101/gr.247064.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-article-title">Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">29</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">1545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.247064.118%26rft_id%253Dinfo%253Apmid%252F31439691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMTU0NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE3LzIwMjAuMDIuMTYuOTQ2Njk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.36" data-doi="10.1038/nbt.2705"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharon</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A single-molecule long-read survey of the human transcriptome</span>. <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">31</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.volume%253D31%26rft.spage%253D1009%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2705%26rft_id%253Dinfo%253Apmid%252F24108091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24108091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soneson</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A comprehensive examination of Nanopore native RNA sequencing for characterization of complex transcriptomes</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">3359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D10%26rft.spage%253D3359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.38" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">34</span>(<span class="cit-issue">18</span>): p. <span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shaw</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Estimating mRNA lengths from Plasmodium falciparum genes by Virtual Northern RNA-seq analysis</span>. <abbr class="cit-jnl-abbrev">Int J Parasitol</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">46</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">7</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BParasitol%26rft.volume%253D46%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.40" data-doi="10.1038/nature08909"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nilsen</span>,  <span class="cit-name-given-names">T.W.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B.R.</span> <span class="cit-name-surname">Graveley</span></span>, <span class="cit-article-title">Expansion of the eukaryotic proteome by alternative splicing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">463</span>(<span class="cit-issue">7280</span>): p. <span class="cit-fpage">457</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNilsen%26rft.auinit1%253DT.%2BW.%26rft.volume%253D463%26rft.issue%253D7280%26rft.spage%253D457%26rft.epage%253D463%26rft.atitle%253DExpansion%2Bof%2Bthe%2Beukaryotic%2Bproteome%2Bby%2Balternative%2Bsplicing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08909%26rft_id%253Dinfo%253Apmid%252F20110989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110989&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273981100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.41" data-doi="10.1101/gr.177790.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braunschweig</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Widespread intron retention in mammals functionally tunes transcriptomes</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">24</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">1774</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.177790.114%26rft_id%253Dinfo%253Apmid%252F25258385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS8xNzc0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.42" data-doi="10.1016/j.cell.2013.06.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Orchestrated intron retention regulates normal granulocyte differentiation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">154</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">583</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.052%26rft_id%253Dinfo%253Apmid%252F23911323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23911323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322629900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.43" data-doi="10.1101/gad.247361.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boutz</span>,  <span class="cit-name-given-names">P.L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bhutkar</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.A.</span> <span class="cit-name-surname">Sharp</span></span>, <span class="cit-article-title">Detained introns are a novel, widespread class of post-transcriptionally spliced introns</span>. <abbr class="cit-jnl-abbrev">Genes &amp; development</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">29</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">63</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBoutz%26rft.auinit1%253DP.%2BL.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D80%26rft.atitle%253DDetained%2Bintrons%2Bare%2Ba%2Bnovel%252C%2Bwidespread%2Bclass%2Bof%2Bpost-transcriptionally%2Bspliced%2Bintrons%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.247361.114%26rft_id%253Dinfo%253Apmid%252F25561496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMjkvMS82MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE3LzIwMjAuMDIuMTYuOTQ2Njk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakharkar</span>,  <span class="cit-name-given-names">M.K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.T.</span> <span class="cit-name-surname">Chow</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kangueane</span></span>, <span class="cit-article-title">Distributions of exons and introns in the human genome</span>. <abbr class="cit-jnl-abbrev">In Silico Biol</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">387</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2Bsilico%2Bbiology%26rft.stitle%253DIn%2BSilico%2BBiol%26rft.aulast%253DSakharkar%26rft.auinit1%253DM.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D387%26rft.epage%253D393%26rft.atitle%253DDistributions%2Bof%2Bexons%2Band%2Bintrons%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F15217358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15217358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivashchenko</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.A.</span> <span class="cit-name-surname">Khailenko</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. Atambaeva</span> <span class="cit-name-surname">Sh</span></span>, <span class="cit-article-title">[Variations of the length of exons and introns in human genome genes]</span>. <abbr class="cit-jnl-abbrev">Genetika</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">22</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetika%26rft.stitle%253DGenetika%26rft.aulast%253DIvashchenko%26rft.auinit1%253DA.%2BT.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D22%26rft.epage%253D29%26rft.atitle%253D%255BVariations%2Bof%2Bthe%2Blength%2Bof%2Bexons%2Band%2Bintrons%2Bin%2Bhuman%2Bgenome%2Bgenes%255D%26rft_id%253Dinfo%253Apmid%252F19239093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19239093&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.46" data-doi="10.1371/journal.pone.0157901"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">Y.L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Deciphering the Draft Genome of Toxoplasma gondii RH Strain</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">11</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">e0157901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0157901%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0157901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0157901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.47" data-doi="10.1038/nature01095"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Sequence of Plasmodium falciparum chromosomes 1, 3-9 and 13</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">419</span>(<span class="cit-issue">6906</span>): p. <span class="cit-fpage">527</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHall%26rft.auinit1%253DN.%26rft.volume%253D419%26rft.issue%253D6906%26rft.spage%253D527%26rft.epage%253D531%26rft.atitle%253DSequence%2Bof%2BPlasmodium%2Bfalciparum%2Bchromosomes%2B1%252C%2B3-9%2Band%2B13.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01095%26rft_id%253Dinfo%253Apmid%252F12368867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12368867&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178348400048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.48" data-doi="10.1093/bioinformatics/btt656"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.K.</span> <span class="cit-name-surname">Smyth</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">30</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">923</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt656%26rft_id%253Dinfo%253Apmid%252F24227677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24227677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.49" data-doi="10.1186/gb-2013-14-7-r70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Porta</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">14</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">R70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR70%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-7-r70%26rft_id%253Dinfo%253Apmid%252F23815980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-7-r70&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23815980&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Intron retention enhances gene regulatory complexity in vertebrates</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">216</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D18%26rft.spage%253D216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Full-length transcript characterization of <em>SF3B1</em> mutation in chronic lymphocytic leukemia reveals downregulation of retained introns</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2018</span>: p. <span class="cit-fpage">410183</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.52" data-doi="10.1016/S0968-0004(98)01208-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagy</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">L.E.</span> <span class="cit-name-surname">Maquat</span></span>, <span class="cit-article-title">A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance</span>. <abbr class="cit-jnl-abbrev">Trends Biochem Sci</abbr>, <span class="cit-pub-date">1998</span>. <span class="cit-vol">23</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">198</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DNagy%26rft.auinit1%253DE.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D198%26rft.epage%253D199%26rft.atitle%253DA%2Brule%2Bfor%2Btermination-codon%2Bposition%2Bwithin%2Bintron-containing%2Bgenes%253A%2Bwhen%2Bnonsense%2Baffects%2BRNA%2Babundance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252898%252901208-0%26rft_id%253Dinfo%253Apmid%252F9644970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(98)01208-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9644970&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074238500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.53" data-doi="10.1002/bies.201300156"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B.T.</span> <span class="cit-name-surname">Porse</span></span>, <span class="cit-article-title">The functional consequences of intron retention: alternative splicing coupled to NMD as a regulator of gene expression</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">36</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">236</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioessays%26rft.volume%253D36%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300156%26rft_id%253Dinfo%253Apmid%252F24352796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24352796&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.54" data-doi="10.1101/gr.177790.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braunschweig</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Widespread intron retention in mammals functionally tunes transcriptomes</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">24</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">1774</span>–<span class="cit-lpage">1786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.177790.114%26rft_id%253Dinfo%253Apmid%252F25258385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS8xNzc0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.55" data-doi="10.1186/1471-2105-6-266"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neverov</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Alternative splicing and protein function</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">6</span>: p. <span class="cit-fpage">266</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNeverov%26rft.auinit1%253DA.%2BD.%26rft.volume%253D6%26rft.spage%253D266%26rft.epage%253D266%26rft.atitle%253DAlternative%2Bsplicing%2Band%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-266%26rft_id%253Dinfo%253Apmid%252F16274476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16274476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.56" data-doi="10.1016/j.cell.2016.08.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sidik</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A Genome-wide CRISPR Screen in Toxoplasma Identifies Essential Apicomplexan Genes</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">166</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">1423</span>-<span class="cit-lpage">1435</span>.e12.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1423%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.019%26rft_id%253Dinfo%253Apmid%252F27594426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27594426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Uncovering the essential genes of the human malaria parasite Plasmodium falciparum by saturation mutagenesis</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">360</span> (<span class="cit-issue">6388</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pai</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The kinetics of pre-mRNA splicing in the Drosophila genome and the influence of gene architecture</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.59" data-doi="10.7554/elife.12175"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skinner</span>,  <span class="cit-name-given-names">S.O.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Single-cell analysis of transcription kinetics across the cell cycle</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">5</span>: p. <span class="cit-fpage">e12175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De12175%26rft_id%253Dinfo%253Adoi%252F10.7554%252Felife.12175%26rft_id%253Dinfo%253Apmid%252F26824388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/elife.12175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26824388&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">K.-W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Sequence features involved in the mechanism of 3’ splice junction wobbling</span>. <abbr class="cit-jnl-abbrev">BMC molecular biology</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">11</span>: p. <span class="cit-fpage">34</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%26rft.volume%253D11%26rft.spage%253D34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.61" data-doi="10.1093/nar/gky014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenjaroenpun</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Complete genomic and transcriptional landscape analysis using third-generation sequencing: a case study of Saccharomyces cerevisiae CEN.PK113-7D</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">46</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253De38%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moldován</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Third-generation Sequencing Reveals Extensive Polycistronism and Transcriptional Overlapping in a Baculovirus</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">8604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D8604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.63" data-doi="10.1186/1471-2105-16-S1-S10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dozmorov</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detrimental effects of duplicate reads and low complexity regions on RNA- and ChIP-seq data</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">16</span> Suppl 13(Suppl 13): p. <span class="cit-fpage">S10</span>–<span class="cit-lpage">S10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D16%26rft.spage%253DS10%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-16-S1-S10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-16-S1-S10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.64" data-doi="10.1038/nature01097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gardner</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genome sequence of the human malaria parasite Plasmodium falciparum</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">419</span>(<span class="cit-issue">6906</span>): p. <span class="cit-fpage">498</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGardner%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D419%26rft.issue%253D6906%26rft.spage%253D498%26rft.epage%253D511%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bhuman%2Bmalaria%2Bparasite%2BPlasmodium%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01097%26rft_id%253Dinfo%253Apmid%252F12368864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12368864&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178348400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Runtuwene</span>,  <span class="cit-name-given-names">L.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nanopore sequencing of drug-resistance-associated genes in malaria parasites, Plasmodium falciparum</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">8286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D8286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.66" data-doi="10.1016/j.cell.2012.10.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loven</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Revisiting global gene expression analysis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">151</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">476</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLoven%26rft.auinit1%253DJ.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D476%26rft.epage%253D482%26rft.atitle%253DRevisiting%2Bglobal%2Bgene%2Bexpression%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.10.012%26rft_id%253Dinfo%253Apmid%252F23101621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23101621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310529300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.67" data-doi="10.1101/gr.124321.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tarazona</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Differential expression in RNA-seq: a matter of depth</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">21</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">2213</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.124321.111%26rft_id%253Dinfo%253Apmid%252F21903743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMjEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.68" data-doi="10.1126/science.1230612"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barbosa-Morais</span>,  <span class="cit-name-given-names">N.L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The Evolutionary Landscape of Alternative Splicing in Vertebrate Species</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">338</span>(<span class="cit-issue">6114</span>): p. <span class="cit-fpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarbosa-Morais%26rft.auinit1%253DN.%2BL.%26rft.volume%253D338%26rft.issue%253D6114%26rft.spage%253D1587%26rft.epage%253D1593%26rft.atitle%253DThe%2BEvolutionary%2BLandscape%2Bof%2BAlternative%2BSplicing%2Bin%2BVertebrate%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230612%26rft_id%253Dinfo%253Apmid%252F23258890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzgvNjExNC8xNTg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Intron retention enhances gene regulatory complexity in vertebrates</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D18%26rft.spage%253D216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.70" data-doi="10.1093/nar/gkl924"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Magen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ast</span></span>, <span class="cit-article-title">Different levels of alternative splicing among eukaryotes</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">35</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">125</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl924%26rft_id%253Dinfo%253Apmid%252F17158149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17158149&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243805800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.71" data-doi="10.1126/science.1089370"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.S.</span> <span class="cit-name-surname">Conery</span></span>, <span class="cit-article-title">The Origins of Genome Complexity</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">302</span>(<span class="cit-issue">5649</span>): p. <span class="cit-fpage">1401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D302%26rft.issue%253D5649%26rft.spage%253D1401%26rft.epage%253D1404%26rft.atitle%253DThe%2BOrigins%2Bof%2BGenome%2BComplexity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089370%26rft_id%253Dinfo%253Apmid%252F14631042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDIvNTY0OS8xNDAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTcvMjAyMC4wMi4xNi45NDY2OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.72" data-doi="10.1093/molbev/msu083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Correcting for differential transcript coverage reveals a strong relationship between alternative splicing and organism complexity</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">31</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">1402</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu083%26rft_id%253Dinfo%253Apmid%252F24682283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24682283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.73" data-doi="10.1016/j.tig.2007.04.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Irimia</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Penny</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.W.</span> <span class="cit-name-surname">Roy</span></span>, <span class="cit-article-title">Coevolution of genomic intron number and splice sites</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">23</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">321</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DIrimia%26rft.auinit1%253DM.%26rft.volume%253D23%26rft.issue%253D7%26rft.spage%253D321%26rft.epage%253D325%26rft.atitle%253DCoevolution%2Bof%2Bgenomic%2Bintron%2Bnumber%2Band%2Bsplice%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.04.001%26rft_id%253Dinfo%253Apmid%252F17442445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17442445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248133200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.74" data-doi="10.1186/gb-2008-9-3-r50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Cross-kingdom patterns of alternative splicing and splice recognition</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">R50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMcGuire%26rft.auinit1%253DA.%2BM.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253DR50%26rft.epage%253DR50%26rft.atitle%253DCross-kingdom%2Bpatterns%2Bof%2Balternative%2Bsplicing%2Band%2Bsplice%2Brecognition.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-3-r50%26rft_id%253Dinfo%253Apmid%252F18321378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-3-r50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18321378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sieber</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparative Study on Alternative Splicing in Human Fungal Pathogens Suggests Its Involvement During Host Invasion</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">9</span>(<span class="cit-issue">2313</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.76" data-doi="10.1186/s13059-018-1499-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grau-Bové</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ruiz-Trillo</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Irimia</span></span>, <span class="cit-article-title">Origin of exon skipping-rich transcriptomes in animals driven by evolution of gene architecture</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">19</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">135</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D19%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1499-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-018-1499-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.77" data-doi="10.1371/journal.pgen.1004249"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Widespread use of non-productive alternative splice sites in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">10</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">e1004249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D10%26rft.spage%253De1004249%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004249%26rft_id%253Dinfo%253Apmid%252F24722551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722551&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.78" data-doi="10.1073/pnas.0136770100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lewis</span>,  <span class="cit-name-given-names">B.P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.E.</span> <span class="cit-name-surname">Green</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.E.</span> <span class="cit-name-surname">Brenner</span></span>, <span class="cit-article-title">Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">100</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">189</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0136770100%26rft_id%253Dinfo%253Apmid%252F12502788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDAvMS8xODkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNy8yMDIwLjAyLjE2Ljk0NjY5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.79" data-doi="10.1016/j.molcel.2008.07.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sayani</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Widespread impact of nonsense-mediated mRNA decay on the yeast intronome</span>. <abbr class="cit-jnl-abbrev">Mol Cell</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">31</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">360</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSayani%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D360%26rft.epage%253D370%26rft.atitle%253DWidespread%2Bimpact%2Bof%2Bnonsense-mediated%2BmRNA%2Bdecay%2Bon%2Bthe%2Byeast%2Bintronome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.07.005%26rft_id%253Dinfo%253Apmid%252F18691968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258556500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.16.946699v1.80" data-doi="10.1261/rna.030767.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plass</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-chapter-title">RNA secondary structure mediates alternative 3’ss selection in Saccharomyces cerevisiae</span>. <span class="cit-source">RNA</span> (<span class="cit-publ-loc">New York, N.Y</span>.), <span class="cit-pub-date">2012</span>. <span class="cit-vol">18</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">1103</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.030767.111%26rft_id%253Dinfo%253Apmid%252F22539526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE4LzYvMTEwMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE3LzIwMjAuMDIuMTYuOTQ2Njk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.81" data-doi="10.1101/gad.468808"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weischenfeldt</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">NMD is essential for hematopoietic stem and progenitor cells and for eliminating by-products of programmed DNA rearrangements</span>. <abbr class="cit-jnl-abbrev">Genes &amp; development</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">22</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">1381</span>–<span class="cit-lpage">1396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWeischenfeldt%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D10%26rft.spage%253D1381%26rft.epage%253D1396%26rft.atitle%253DNMD%2Bis%2Bessential%2Bfor%2Bhematopoietic%2Bstem%2Band%2Bprogenitor%2Bcells%2Band%2Bfor%2Beliminating%2Bby-products%2Bof%2Bprogrammed%2BDNA%2Brearrangements%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.468808%26rft_id%253Dinfo%253Apmid%252F18483223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzEwLzEzODEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNy8yMDIwLjAyLjE2Ljk0NjY5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">L.E.</span> <span class="cit-name-surname">Maquat</span></span>, <span class="cit-article-title">Nonsense-mediated mRNA decay (NMD) in animal embryogenesis: to die or not to die, that is the question</span>. <abbr class="cit-jnl-abbrev">Current opinion in genetics &amp; development</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">21</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">422</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.volume%253D21%26rft.spage%253D422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Premature Termination Codons Are Recognized in the Nucleus in A Reading-Frame Dependent Manner</span>. <abbr class="cit-jnl-abbrev">Cell Discov</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harati</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.H.</span> <span class="cit-name-surname">Phan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.D.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-article-title">Investigation of factors affecting RNA-seq gene expression calls</span>. <abbr class="cit-jnl-abbrev">Conference proceedings : … Annual International Conference of the IEEE Engineering in Medicine and Biology Society. IEEE Engineering in Medicine and Biology Society. Annual Conference</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">2014</span>: p. <span class="cit-fpage">5232</span>–<span class="cit-lpage">5235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConference%2Bproceedings%2B%253A%2B%253F%2BAnnual%2BInternational%2BConference%2Bof%2Bthe%2BIEEE%2BEngineering%2Bin%2BMedicine%2Band%2BBiology%2BSociety.%2BIEEE%2BEngineering%2Bin%2BMedicine%2Band%2BBiology%2BSociety.%2BAnnual%2BConference%26rft.volume%253D2014%26rft.spage%253D5232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of two main RNA-seq approaches for gene quantification in clinical RNA sequencing: polyA+ selection versus rRNA depletion</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">4781</span>–<span class="cit-lpage">4781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D4781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.86" data-doi="10.1002/bies.201300156"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">B.T.</span> <span class="cit-name-surname">Porse</span></span>, <span class="cit-article-title">The functional consequences of intron retention: Alternative splicing coupled to NMD as a regulator of gene expression</span>. <abbr class="cit-jnl-abbrev">BioEssays</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">36</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">236</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D36%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300156%26rft_id%253Dinfo%253Apmid%252F24352796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24352796&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">A.Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A data-driven approach to characterising intron signal in RNAseq data</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2018</span>: p. <span class="cit-fpage">352823</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.88" data-doi="10.1016/j.mcn.2013.01.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yap</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E.V.</span> <span class="cit-name-surname">Makeyev</span></span>, <span class="cit-article-title">Regulation of gene expression in mammalian nervous system through alternative pre-mRNA splicing coupled with RNA quality control mechanisms</span>. <abbr class="cit-jnl-abbrev">Mol Cell Neurosci</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">56</span>: p. <span class="cit-fpage">420</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BCell%2BNeurosci%26rft.volume%253D56%26rft.spage%253D420%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mcn.2013.01.003%26rft_id%253Dinfo%253Apmid%252F23357783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mcn.2013.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23357783&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.89" data-doi="10.1534/genetics.117.300637"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.T.</span> <span class="cit-name-surname">Branco</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lemos</span></span>, <span class="cit-article-title">The Y Chromosome Modulates Splicing and Sex-Biased Intron Retention Rates in Drosophila</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">208</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">1057</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300637%26rft_id%253Dinfo%253Apmid%252F29263027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwOC8zLzEwNTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNy8yMDIwLjAyLjE2Ljk0NjY5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.90" data-doi="10.1182/blood-2016-01-692764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">C.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A dynamic intron retention program in the mammalian megakaryocyte and erythrocyte lineages</span>. <abbr class="cit-jnl-abbrev">Blood</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">127</span>(<span class="cit-issue">17</span>): p. <span class="cit-fpage">e24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2016-01-692764%26rft_id%253Dinfo%253Apmid%252F26962124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTI3LzE3L2UyNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE3LzIwMjAuMDIuMTYuOTQ2Njk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.91" data-doi="10.1186/s13059-017-1184-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Middleton</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">IRFinder: assessing the impact of intron retention on mammalian gene expression</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D18%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1184-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1184-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nickless</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.M.</span> <span class="cit-name-surname">Bailis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">You</span></span>, <span class="cit-article-title">Control of gene expression through the nonsense-mediated RNA decay pathway</span>. <abbr class="cit-jnl-abbrev">Cell &amp; bioscience</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">7</span>: p. <span class="cit-fpage">26</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2B%2526%2Bbioscience%26rft.volume%253D7%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.93" data-doi="10.1093/nar/gkx1272"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nonsense-mediated mRNA decay: a ‘nonsense’ pathway makes sense in stem cell biology</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">46</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">1038</span>–<span class="cit-lpage">1051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253D1038%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.94" data-doi="10.1016/j.semcdb.2017.07.030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vanichkina</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Challenges in defining the role of intron retention in normal biology and disease</span>. <abbr class="cit-jnl-abbrev">Seminars in Cell &amp; Developmental Biology</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">75</span>: p. <span class="cit-fpage">40</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2BCell%2B%2526%2BDevelopmental%2BBiology%26rft.volume%253D75%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2017.07.030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2017.07.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.95" data-doi="10.1186/s13059-017-1344-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saudemont</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The fitness cost of mis-splicing is the main determinant of alternative splicing patterns</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">208</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D18%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1344-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1344-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.96"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foth</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Quantitative time-course profiling of parasite and host cell proteins in the human malaria parasite Plasmodium falciparum</span>. <abbr class="cit-jnl-abbrev">Mol Cell Proteomics</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">10</span>(<span class="cit-issue">8</span>): p. M110.006411.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.97" data-doi="10.1093/nar/gkw1105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aurrecoechea</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">EuPathDB: the eukaryotic pathogen genomics database resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">45</span>(<span class="cit-issue">D1</span>): p. <span class="cit-fpage">D581</span>–<span class="cit-lpage">d591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1105%26rft_id%253Dinfo%253Apmid%252F27903906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27903906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.16.946699v1.98"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-source">FastQC A Quality Control tool for High Throughput Sequence Data</span>. <span class="cit-pub-date">2010</span>: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.99" data-doi="10.1038/s41587-019-0201-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">37</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">907</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D37%26rft.spage%253D907%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-019-0201-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41587-019-0201-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.100" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): p. <span class="cit-fpage">2078</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.16.946699v1.101"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Broad</span> <span class="cit-name-surname">Institute</span></span>, <span class="cit-source">Picard Toolkit</span>. <span class="cit-pub-date">2019</span>: <a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">102.</span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference 102. in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weirather</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis</span>. <abbr class="cit-jnl-abbrev">F1000Research</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">6</span>: p. <span class="cit-fpage">100</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Research%26rft.volume%253D6%26rft.spage%253D100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">103.</span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference 103. in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.103" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Integrative genomics viewer</span>. <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">29</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.104" data-doi="10.1038/nmeth. 4577"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garalde</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">15</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">201</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D15%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.%2B4577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth. 4577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.16.946699v1.105"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Oxford Nanopore Technologies Ltd., Wub</span>. <span class="cit-pub-date">2016</span>: <a href="https://github.com/nanoporetech/wub">https://github.com/nanoporetech/wub</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.106" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">A.R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">I.M.</span> <span class="cit-name-surname">Hall</span></span>, <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">26</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.107" data-doi="10.1093/bioinformatics/btu842"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">An</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">J-Circos: an interactive Circos plotter</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">31</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">1463</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu842%26rft_id%253Dinfo%253Apmid%252F25540184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25540184&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.108" data-doi="10.1093/nar/gku365"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">deepTools: a flexible platform for exploring deep-sequencing data</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">42</span> (<span class="cit-issue">Web Server issue</span>): p. <span class="cit-fpage">W187</span>–<span class="cit-lpage">W191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku365%26rft_id%253Dinfo%253Apmid%252F24799436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku365&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24799436&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">109.</span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference 109. in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.109" data-doi="10.1186/gb4161"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Dissemination of scientific software with Galaxy ToolShed</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">15</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D15%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb4161%26rft_id%253Dinfo%253Apmid%252F25001293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb4161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25001293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">110.</span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference 110. in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.110" data-doi="10.1093/bioinformatics/bts356"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">RSeQC: quality control of RNA-seq experiments</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">28</span>(<span class="cit-issue">16</span>): p. <span class="cit-fpage">2184</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts356%26rft_id%253Dinfo%253Apmid%252F22743226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22743226&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307501100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">111.</span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference 111. in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.111" data-doi="10.1186/1471-2164-9-488"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hulsen</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">de Vlieg</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Alkema</span></span>, <span class="cit-article-title">BioVenn – a web application for the comparison and visualization of biological lists using area-proportional Venn diagrams</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHulsen%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.spage%253D488%26rft.epage%253D488%26rft.atitle%253DBioVenn%2B-%2Ba%2Bweb%2Bapplication%2Bfor%2Bthe%2Bcomparison%2Band%2Bvisualization%2Bof%2Bbiological%2Blists%2Busing%2Barea-proportional%2BVenn%2Bdiagrams.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-488%26rft_id%253Dinfo%253Apmid%252F18925949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-488&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18925949&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F17%2F2020.02.16.946699.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">112.</span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference 112. in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.16.946699v1.112" data-doi="10.1093/nar/gkz401"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">47</span> (<span class="cit-issue">W1</span>): p. <span class="cit-fpage">W199</span>–<span class="cit-lpage">w205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DW199%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkz401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.16.946699v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.17.951434v1" title="Induction Of Chronic Stress Reveals An Interplay Of Stress Granules And TDP-43 Pathological Aggregates In Human ALS Fibroblasts And iPSC-Neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.02.16.946244v1" title="Endosidin20 targets cellulose synthase catalytic domain to inhibit cellulose biosynthesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 17, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.16.946699v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.16.946699v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.16.946699v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.16.946699v1" class="active" data-icon-position="" data-hide-link-title="0">Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1155358" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Q2c6kueuQk0tzm1pFNiCsyad2ovG87fUmszfD0uKj2Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1155358" id="node1155358--2" data-pisa="biorxiv;2020.02.16.946699v1" data-pisa-master="biorxiv;2020.02.16.946699" data-seqnum="1155358" data-apath="/biorxiv/early/2020/02/17/2020.02.16.946699.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">V Vern</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louise M.</span> <span class="nlm-surname">Judd</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Aaron R.</span> <span class="nlm-surname">Jex</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kathryn E.</span> <span class="nlm-surname">Holt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Tonkin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Stuart A.</span> <span class="nlm-surname">Ralph</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.16.946699; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.16.946699 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;title=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;title=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;text=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;title=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;t=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;title=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;title=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1155358" id="node1155358--3" data-pisa="biorxiv;2020.02.16.946699v1" data-pisa-master="biorxiv;2020.02.16.946699" data-seqnum="1155358" data-apath="/biorxiv/early/2020/02/17/2020.02.16.946699.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Direct nanopore sequencing of mRNA reveals landscape of transcript isoforms in apicomplexan parasites</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">V Vern</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louise M.</span> <span class="nlm-surname">Judd</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Aaron R.</span> <span class="nlm-surname">Jex</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kathryn E.</span> <span class="nlm-surname">Holt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christopher J.</span> <span class="nlm-surname">Tonkin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Stuart A.</span> <span class="nlm-surname">Ralph</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.16.946699; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.16.946699 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Lee%2C+V+Vern&amp;rft.au=Judd%2C+Louise+M.&amp;rft.au=Jex%2C+Aaron+R.&amp;rft.au=Holt%2C+Kathryn+E.&amp;rft.au=Tonkin%2C+Christopher+J.&amp;rft.au=Ralph%2C+Stuart+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1155358/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1155358/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1155358/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1155358/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1155358/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1155358/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1155358/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1155358/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1155358/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1155358/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1155358/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1155358/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;count=horizontal&amp;via=&amp;text=Direct%20nanopore%20sequencing%20of%20mRNA%20reveals%20landscape%20of%20transcript%20isoforms%20in%20apicomplexan%20parasites&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.946699v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.16.946699v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
