<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.03.29.013342.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.29.013342v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.29.013342"/>
  <meta name="DC.Date" content="2020-04-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX)[1][1]. Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use – (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them. We lay out protocols for dye-based and TaqMan probe-based assays, in order to best compare with ‘gold standard’ reagents. These protocols should form the basis of further modifications that can simplify the assay to the use of overexpressing cells themselves as reagents. Developing dye-based and TaqMan probe-based RT-qPCR assays with RTX ### Competing Interest Statement [1]: #ref-1"/>
  <meta name="DC.Contributor" content="Sanchita Bhadra"/>
  <meta name="DC.Contributor" content="Andre C. Maranhao"/>
  <meta name="DC.Contributor" content="Andrew D. Ellington"/>
  <meta name="article:published_time" content="2020-04-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX). Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use - (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.29.013342v3"/>
  <meta name="citation_id" content="2020.03.29.013342v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.013342v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.013342v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.013342v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.29.013342"/>
  <meta name="citation_num_pages" content="26"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.29.013342"/>
  <meta name="citation_author" content="Sanchita Bhadra"/>
  <meta name="citation_author_institution" content="Center for Systems and Synthetic Biology, University of Texas at Austin, Austin"/>
  <meta name="citation_author" content="Andre C. Maranhao"/>
  <meta name="citation_author_institution" content="Center for Systems and Synthetic Biology, University of Texas at Austin, Austin"/>
  <meta name="citation_author" content="Andrew D. Ellington"/>
  <meta name="citation_author_institution" content="Center for Systems and Synthetic Biology, University of Texas at Austin, Austin"/>
  <meta name="citation_author_email" content="ellingtonlab@gmail.com"/>
  <meta name="citation_reference" content="Ellefson, J. W. et al. Synthetic evolutionary origin of a proofreading reverse transcriptase. Science 352, 1590–1593, doi: 10.1126/science.aaf5409 (2016)."/>
  <meta name="twitter:title" content="A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX)[1][1]. Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use – (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them. We lay out protocols for dye-based and TaqMan probe-based assays, in order to best compare with ‘gold standard’ reagents. These protocols should form the basis of further modifications that can simplify the assay to the use of overexpressing cells themselves as reagents. Developing dye-based and TaqMan probe-based RT-qPCR assays with RTX ### Competing Interest Statement [1]: #ref-1"/>
  <meta name="og-title" property="og:title" content="A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.013342v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX)[1][1]. Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use – (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them. We lay out protocols for dye-based and TaqMan probe-based assays, in order to best compare with ‘gold standard’ reagents. These protocols should form the basis of further modifications that can simplify the assay to the use of overexpressing cells themselves as reagents. Developing dye-based and TaqMan probe-based RT-qPCR assays with RTX ### Competing Interest Statement [1]: #ref-1"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-09"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1237308 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-4">Given the scale of the ongoing COVID-19 pandemic, the need for reliable, scalable testing, and the likelihood of reagent shortages, especially in resource-poor settings, we have developed a RT-qPCR assay that relies on an alternative to conventional viral reverse transcriptases, a thermostable reverse transcriptase / DNA polymerase (RTX)
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. Here we show that RTX performs comparably to the other assays sanctioned by the CDC and validated in kit format. We demonstrate two modes of RTX use – (i) dye-based RT-qPCR assays that require only RTX polymerase, and (ii) TaqMan RT-qPCR assays that use a combination of RTX and Taq DNA polymerases (as the RTX exonuclease does not degrade a TaqMan probe). We also provide straightforward recipes for the purification of this alternative reagent. We anticipate that in low resource or point-of-need settings researchers could obtain the available constructs from Addgene or our lab and begin to develop their own assays, within whatever regulatory framework exists for them.
    </p>
    <p id="p-5">We lay out protocols for dye-based and TaqMan probe-based assays, in order to best compare with ‘gold standard’ reagents. These protocols should form the basis of further modifications that can simplify the assay to the use of overexpressing cells themselves as reagents.</p>
    <p id="p-6">
     <span class="underline">Developing dye-based and TaqMan probe-based RT-qPCR assays with RTX</span>
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">MATERIALS</h2>
    <ul class="list-unord " id="list-1">
     <li id="list-item-1">
      <p id="p-9">10X RTX buffer: 600 mM Tris-HCl, 250 mM (NH
       <sub>4</sub>)
       <sub>2</sub>SO
       <sub>4</sub>, 100 mM KCl, 20 mM MgSO
       <sub>4</sub>, pH 8.4 @25°C
      </p>
     </li>
     <li id="list-item-2">
      <p id="p-10">10X ThermoPol buffer (New England Biolabs): 200 mM Tris-HCl, 100 mM (NH
       <sub>4</sub>)
       <sub>2</sub>SO
       <sub>4</sub>, 100 mM KCl, 20 mM MgSO
       <sub>4</sub>, 1% Triton
       <sup>®</sup> X-100, pH 8.8 @25°C
      </p>
     </li>
     <li id="list-item-3">
      <p id="p-11">6.7 µM Forward PCR primer
       <sup>a</sup>
      </p>
     </li>
     <li id="list-item-4">
      <p id="p-12">6.7 µM Reverse PCR primer
       <sup>a</sup>
      </p>
     </li>
     <li id="list-item-5">
      <p id="p-13">1.7 µM TaqMan probe
       <sup>a</sup>
      </p>
     </li>
     <li id="list-item-6">
      <p id="p-14">20X EvaGreen solution</p>
     </li>
     <li id="list-item-7">
      <p id="p-15">4 mM deoxyribonucleotides (dNTP)</p>
     </li>
     <li id="list-item-8">
      <p id="p-16">5M Betaine</p>
     </li>
     <li id="list-item-9">
      <p id="p-17">20 U/µL SUPERase.In RNase inhibitor</p>
     </li>
     <li id="list-item-10">
      <p id="p-18">5 U/µL Taq DNA polymerase</p>
     </li>
     <li id="list-item-11">
      <p id="p-19">0.2 mg/ml RTX polymerase
       <sup>b</sup>
      </p>
     </li>
     <li id="list-item-12">
      <p id="p-20">Nuclease-free water</p>
     </li>
     <li id="list-item-13">
      <p id="p-21">qPCR tubes or plates
       <sup>c</sup>
      </p>
     </li>
     <li id="list-item-14">
      <p id="p-22">Real-time PCR machine
       <sup>c</sup>
      </p>
     </li>
     <li id="list-item-15">
      <p id="p-23">Cold-block or ice</p>
     </li>
     <li id="list-item-16">
      <p id="p-24">Barrier tips</p>
     </li>
    </ul>
    <ol class="list-alphalower " id="list-2">
     <li id="list-item-17">
      <p id="p-25">Individual primer and probe stocks or pre-made primer-probe mixes for CDC N1, N2 and N3 assays (
       <strong>
        <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
       </strong>) available from Integrated DNA Technologies may be used.
      </p>
     </li>
     <li id="list-item-18">
      <p id="p-26">RTX polymerase with proofreading capability (RTX), and RTX polymerase without proofreading capability (RTX Exo-) have been compared.</p>
     </li>
     <li id="list-item-19">
      <p id="p-27">LightCycler96 real-time PCR machine and 96-well plates with optical plastic film cover designed for use with LightCycler platform were used for these experiments.</p>
     </li>
    </ol>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242072/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1242072/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1242072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <p id="p-28" class="first-child">CDC TaqMan qRT-PCR primers and probes for SARS-CoV-2 (adapted from 
       <a href="https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html">https://www.cdc.gov/coronavirus/2019-ncov/lab/rt-pcr-panel-primer-probes.html</a>).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">RESULTS</h2>
    <div id="sec-3" class="subsection">
     <h3>Dye-based RT-qPCR reaction set up using only RTX DNA polymerase</h3>
     <p id="p-29">EvaGreen dye-containing RT-qPCR reaction set up shown in 
      <strong>
       <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>
      </strong> was used for assembling CDC SARS-CoV-2 N1, N2, and N3 assays. The three assays target three different regions within the nucleocapsid phosphoprotein (N) gene near the 3’-end of the ∼30 kb viral genome. In particular, N1 primers (
      <strong>
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>
      </strong>) amplify a 71 nt long region from position 28287 to 28358 of the genome (GenBank Accession No. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN985325.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.29.013342.atom">MN985325.1</a>). The N2 primers produce a 66 bp amplicon from the genomic position 29164 to 29230. Meanwhile, the N3 primers amplify a 71 nt long region from genomic position 28681 to 28752. All reaction mixes were assembled on a cold-block and directly transferred to a real-time PCR machine programmed to cycle through the steps depicted in 
      <strong>
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>
      </strong>. Melt curve analysis was included after PCR amplification in order to distinguish target amplicons from spurious products.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242066/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242066/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242066" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-30" class="first-child">EvaGreen RT-qPCR reaction mix using only RTX DNA polymerase</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242065/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242065/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242065" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3.</span>
       <p id="p-31" class="first-child">RTX polymerase-based EvaGreen RT-qPCR cycling conditions</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-32">Representative results of SARS-CoV-2 RT-qPCR tests performed using synthetic RNA templates and only RTX or RTX Exo-DNA polymerases are depicted in 
      <strong>
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1</a> and 
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a>
      </strong>.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the [[#x2018]]Tm calling[[#x2019]] analysis in the LightCycler96 software. Average Cq values of all assays are tabulated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the [[#x2018]]Tm calling[[#x2019]] analysis in the LightCycler96 software. Average Cq values of all assays are tabulated.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F1.medium.gif" width="440" height="341"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F1.medium.gif" width="440" height="341"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-33" class="first-child">CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the ‘Tm calling’ analysis in the LightCycler96 software. Average Cq values of all assays are tabulated.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX Exo-DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the [[#x2018]]Tm calling[[#x2019]] analysis in the LightCycler96 software. Average Cq values of all assays are tabulated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX Exo-DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the [[#x2018]]Tm calling[[#x2019]] analysis in the LightCycler96 software. Average Cq values of all assays are tabulated.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F2.medium.gif" width="440" height="329"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F2.medium.gif" width="440" height="329"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-34" class="first-child">CDC SARS-CoV-2 N1, N2, and N3 RT-qPCR assays performed using indicated copies of synthetic RNA and RTX Exo-DNA polymerase. Panels A, B, and C depict N1, N2, and N3 assays measured in real-time using EvaGreen dye. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Panels D, E, and F depict melting peaks of amplicons determined using the ‘Tm calling’ analysis in the LightCycler96 software. Average Cq values of all assays are tabulated.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-35">These results demonstrate that RTX DNA polymerase alone, with or without proofreading capability, can support dye-based RT-qPCR analyses. In our hands the full-length RTX DNA polymerase was slightly better than the Exo-version, especially in the N3 assay.</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>RTX polymerase-based TaqMan qRT-PCR reaction set up</h3>
     <p id="p-36">In order to perform TaqMan RT-qPCR we developed a one-pot two enzyme system comprising RTX and Taq DNA polymerases. TaqMan qRT-PCR reaction set up shown in 
      <strong>
       <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>
      </strong> was used for assembling CDC SARS-CoV-2 N1, N2, and N3 assays using either RTX buffer or ThermoPol buffer. Control reactions lacking RTX polymerase were set up using the same recipe except RTX was replaced with water. All reaction mixes were assembled on a cold-block and directly transferred to a real-time PCR machine programmed to cycle through the steps depicted in 
      <strong>
       <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>
      </strong>.
     </p>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242069/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242069/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242069" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 4.</span>
       <p id="p-37" class="first-child">RTX polymerase-based TaqMan qRT-PCR reaction mix</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242071/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242071/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242071" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 5.</span>
       <p id="p-38" class="first-child">RTX polymerase-based TaqMan qRT-PCR cycling conditions</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-39">Representative results of RTX polymerase-based SARS-CoV-2 TaqMan qRT-PCR tests performed using synthetic RNA templates are depicted in 
      <strong>
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and 
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>
      </strong>. Both RTX and RTX Exo-when combined with Taq DNA polymerase were able to support one-pot TaqMan RT-qPCR assays for all three CDC SARS-CoV-2 assays. Under both buffer conditions tested, 1X ThermoPol versus 1X RTX buffer, RTX polymerase yielded more consistent amplification curves for all three CDC assays compared to RTX Exo-polymerase.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X ThermoPol buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing both RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing only Taq DNA polymerase. Panels G, H, and I depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X ThermoPol buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing both RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing only Taq DNA polymerase. Panels G, H, and I depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F3.medium.gif" width="440" height="319"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F3.medium.gif" width="440" height="319"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-40" class="first-child">CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X ThermoPol buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing both RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing only Taq DNA polymerase. Panels G, H, and I depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X RTX buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X RTX buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F4.medium.gif" width="440" height="298"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F4.medium.gif" width="440" height="298"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-41" class="first-child">CDC SARS-CoV-2 N1, N2, and N3 TaqMan RT-qPCR assays performed in 1X RTX buffer using indicated copies of synthetic RNA and RTX or RTX Exo- and Taq DNA polymerases. Panels A, B, and C depict TaqMan assays containing RTX and Taq DNA polymerases. Panels D, E, and F depict TaqMan assays containing RTX Exo- and Taq DNA polymerases. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-42">To compare the RTX-based TaqMan qRT-PCR with qRT-PCR performed using commercial master mix, duplicate reactions (
      <strong>
       <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>
      </strong>) were set up using TaqPath™ 1-Step RT-qPCR Master Mix, CG (Thermo Fisher Scientific, Waltham, MA, USA). All reactions were assembled on cold block and then incubated at room temperature (25 °C benchtop) for 2 min prior to being loaded on a real-time PCR machine set to cycle through the steps indicated in 
      <strong>
       <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>
      </strong>.
     </p>
     <div id="T6" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242068/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242068/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 6.</span>
       <p id="p-43" class="first-child">CDC N1, N2, N3 TaqMan qRT-PCR assay set up using commercial qRT-PCR master mix</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T7" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242078/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242078/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242078" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 7.</span>
       <p id="p-44" class="first-child">TaqPath™ TaqMan qRT-PCR cycling conditions</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-45">
      <strong>
       <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>
      </strong> depicts typical results from CDC N1, N2, and N3 TaqMan qRT-PCR reactions performed using synthetic RNA and TaqPath™ commercial qRT-PCR mastermix.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="CDC SARS-CoV-2 N1, N2, and N3 TaqMan qRT-PCR assays performed using indicated copies of synthetic RNA and TaqPath[[#x2122]] commercial qRT-PCR mastermix. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;CDC SARS-CoV-2 N1, N2, and N3 TaqMan qRT-PCR assays performed using indicated copies of synthetic RNA and TaqPath[[#x2122]] commercial qRT-PCR mastermix. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F5.medium.gif" width="440" height="180"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F5.medium.gif" width="440" height="180"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <p id="p-46" class="first-child">CDC SARS-CoV-2 N1, N2, and N3 TaqMan qRT-PCR assays performed using indicated copies of synthetic RNA and TaqPath™ commercial qRT-PCR mastermix. Amplification curves from reactions containing 100,000 (black traces), 10,000 (blue traces), 1,000 (red traces), and 100 (pink traces) copies of SARS-CoV-2 synthetic N RNA are depicted. Negative control reactions either contained no templates (gray traces) or contained 100,000 copies of synthetic N RNA from MERS-CoV (green traces). Cq values of all assays are tabulated.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-47">The results with either a viral RT (in TaqPath™ commercial mastermix) or RTX are summarized in 
      <strong>
       <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>
      </strong>. As can be seen, RTX-based TaqMan assays are of comparable sensitivity and specificity to the gold standard TaqPath assay. While all three RTX-based TaqMan assays performed in ThermoPol buffer consistently yielded Cq values, these were somewhat delayed compared to Cq values obtained with the commercial TaqPath™ Master Mix. In contrast, RTX-based TaqMan assays when executed in RTX buffer yielded Cq values that were closer to Cq values obtained with the commercial mastermix.
     </p>
     <div id="T8" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242070/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242070/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242070" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 8.</span>
       <p id="p-48" class="first-child">Comparison of Cq values of RTX-based and TaqPath-based SARS-CoV-2 TaqMan qRT-PCR assays</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="sec-5" class="subsection">
      <h4>Summary of assay performance</h4>
      <p id="p-49">Overall, RTX performs well either as a single enzyme for RT-qPCR with intercalating dyes, such as EvaGreen, or as the RT component of a TaqMan based assay. At the lowest RNA concentrations examined (100 copies), the gold standard TaqPath assay general gave a signal at a Cq value of ca. 32. The more robust full-length RTX on its own actually performed slightly better, with C
       <sub>q</sub> values typically from 29-30 (although the N3 primer set gives higher signal and higher background). When used as a substitute for a viral RT, RTX is slightly less sensitive overall, with Cq values typically from 32-35.
      </p>
     </div>
     <div id="sec-6" class="subsection">
      <h4>Purification of RTX</h4>
      <p id="p-50">For sites wishing to produce their own RTX, the following protocol can be used.</p>
      <div id="T9" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242079/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1242079/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1242079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 7.</span>
        <p id="p-51" class="first-child">Buffers for RTX purification</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="T10" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242076/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1242076/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1242076" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 8.</span>
        <p id="p-52" class="first-child">Buffers for dialysis of RTX</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-53">In brief, a T7-based 
       <em>E. coli</em> expression strain such as BL21(DE3) was transformed with a RTX expression plasmid that contained an carboxyl terminal six histidine tag for purification. On the next day, an individual transformant colony was picked into desired media with appropriate antibiotics (e.g. ampicillin) and 1-2% glucose. This starter culture was grown overnight in shaking incubator at 37°C.
      </p>
      <p id="p-54">On day three, 1 L Superior Broth™ (Athena Enzyme Systems™) or other rich media with appropriate antibiotics in a 4 L Erlenmeyer flask was inoculated with 250-350 μL of the overnight starter culture. Incubate at 37°C, 250 rpm until the culture reaches OD
       <sub>600</sub> of 0.4-0.7 (0.5-0.6 is ideal). When desired OD
       <sub>600</sub> is reached, place flask(s) in 4°C cold room for 30-45 min and set shaking incubator to 18°C. Following cold incubation, induce expression of RTX by adding 1 mL of 1 M IPTG to each 1 L of culture. Place flask(s) in the 18°C shaking incubator for 16-18 hours.
      </p>
      <p id="p-55">Following overnight expression, cells were harvested by centrifuging cultures at 4°C, 5000 xg for 20 min. On ice, resuspend cell pellet in 30 mL cold 
       <strong>Lysis Buffer</strong>. Transfer the resuspended cell pellet to a small 50 mL beaker with clean stir bar and place securely in an ice bath. With moderate stirring, sonicate the sample using 40% amplitude and 1 sec ON / 4 sec OFF for 4 min total sonication time. Centrifuge the resulting lysate at 4°C, 35,000 xg for 30 min. Carefully transfer the supernatant to a clean ultracentrifugation tube, shake in thermomixer at 400 rpm, 85°C for 10 min, and then place on ice for 10 min. Centrifuged the heat-treated lysate at 4°C, 35,000 xg for 30 min. Carefully transfer the supernatant to a clean tube and filter the clarified lysate using a 0.2 μm filter.
      </p>
      <p id="p-56">Perform IMAC column purification in a cold room or allow gravity flow to occur in a refrigerator. Prepare Ni-NTA agarose columns with final column volume (CV) of 1 mL. Apply the clarified lysate from 1 L of expressed culture to a previously equilibrated column and collect the flow-through. Wash column with 20 CV 
       <strong>Equilibration Buffer</strong> and collect the flow-through. Wash column with 5 CV 
       <strong>Wash Buffer</strong> and collect flow-through. Elute RTX from column with 5 mL 
       <strong>Elution Buffer</strong> and transfer to a dialysis cassette with appropriate molecular weight cut-off. Dialyze eluate into 2 L of 
       <strong>Ni-NTA Dialysis Buffer</strong> for 3-4 hours at 4°C. Then dialyze eluate into a second 2 L of 
       <strong>Ni-NTA Dialysis Buffer</strong> overnight at 4°C.
      </p>
      <p id="p-57">Dialyzed eluate was then passed over an equilibrated 5 mL heparin column (HiTrap™ Heparin HP) and eluted along a sodium chloride gradient (100 mM to 2 M NaCl). The peak corresponding to RTX or its variants could be expected between 40-60% Buffer B. RTX fractions were collected, pooled, and dialyzed into 2 L 
       <strong>Heparin Dialysis Buffer</strong> for 3-4 hours. Then dialysis cassettes were transferred into 2 L 
       <strong>Final Dialysis Buffer</strong> overnight where it is expected that the protein sample volume will decrease significantly. Following the final dialysis, protein sample was recovered and protein concentration was determined before storing at −20°C.
      </p>
      <p id="p-58">A similar protocol should work for Taq and other thermostable polymerases, but may require different dialysis and storage buffers.</p>
     </div>
     <div id="sec-7" class="subsection">
      <h4>Summary and opportunities for distribution</h4>
      <p id="p-59">By obtaining a RTX expression vector and purifying the thermostable reverse transcriptase it should prove possible to carry out RT-qPCR reactions with a sensitivity similar to that already observed for approved kits, with few or no false positive results. The sequence information for plasmids for RTX [pET_RTX-6xHis] and its exonuclease deficient variant, RTX(exo-) [pET_RTX(exo -)-6xHis], bearing a C-terminal 6xHis tags can be found in the 
       <strong>Supplemental Information</strong>. These plasmids will be uploaded to Addgene in short order, or can be obtained by directly contacting the Ellington lab, via 
       <span class="em-link">
        <span class="em-addr">a.maranhao{at}utexas.edu</span>
       </span>. The Board of Regents of The University of Texas has licensed IP covering RTX to Promega Corporation.
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Supporting Information</h2>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Plasmid map of pET_RTX-6xHis" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Plasmid map of pET_RTX-6xHis&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F6.medium.gif" width="440" height="410"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F6.medium.gif" width="440" height="410"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S1.</span>
      <p id="p-61" class="first-child">Plasmid map of pET_RTX-6xHis</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T11" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242067/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup last">
         <a href="/highwire/markup/1242067/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S1.</span>
      <p id="p-62" class="first-child">GenBank format of pET_RTX-6xHis</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Plasmid map pET_RTX(exo-)-6xHis" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1634231032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Plasmid map pET_RTX(exo-)-6xHis&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F7.medium.gif" width="440" height="410"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F7.medium.gif" width="440" height="410"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.29.013342/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S2.</span>
      <p id="p-63" class="first-child">Plasmid map pET_RTX(exo-)-6xHis</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T12" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242077/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup last">
         <a href="/highwire/markup/1242077/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S2.</span>
      <p id="p-64" class="first-child">GenBank format of pET_RTX(exo-)-6xHis</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-60">We acknowledge the extraordinary contributions of the Schoggins lab in making synthetic RNA available on short notice for these assays.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-1">
      <p id="p-1">
       <a class="rev-xref" href="#xref-fn-1-1">↵</a>
       <span class="fn-label">1</span> Inquiries about manuscript to 
       <span class="em-link">
        <span class="em-addr">ellingtonlab{at}gmail.com</span>
       </span>
      </p>
     </li>
     <li class="fn-other" id="fn-2">
      <p id="p-2">
       <a class="rev-xref" href="#xref-fn-2-1">↵</a>
       <span class="fn-label">2</span> Inquiries about vectors to 
       <span class="em-link">
        <span class="em-addr">a.maranhao{at}utexas.edu</span>
       </span>
      </p>
     </li>
     <li class="fn-update fn-group-summary-of-updates" id="fn-3">
      <p id="p-8">The final concentration of dNTPs in reaction mixes has been corrected to 400 uM instead of 400 nM.</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.29.013342v3.1" data-doi="10.1126/science.aaf5409">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ellefson</span>, 
          <span class="cit-name-given-names">J. W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Synthetic evolutionary origin of a proofreading reverse transcriptase</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">352</span>, 
         <span class="cit-fpage">1590</span>–
         <span class="cit-lpage">1593</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.aaf5409</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DEllefson%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D352%26rft.issue%253D6293%26rft.spage%253D1590%26rft.epage%253D1593%26rft.atitle%253DSynthetic%2Bevolutionary%2Borigin%2Bof%2Ba%2Bproofreading%2Breverse%2Btranscriptase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf5409%26rft_id%253Dinfo%253Apmid%252F27339990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTIvNjI5My8xNTkwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4yOS4wMTMzNDIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
