<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6294367">6294367</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0209292">10.1371/journal.pone.0209292</a></span><span class="publisher-id" title="publisher-id">PONE-D-18-25421</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Togaviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Alphaviruses</span>: <span class="subject" title="subject">Chikungunya Virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">North America</span>: <span class="subject" title="subject">Mexico</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span>: <span class="subject" title="subject">Sequence Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Heredity</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetic Mapping</span>: <span class="subject" title="subject">Haplotypes</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Nucleotide Sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Nucleotide Sequencing</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Tropical Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neglected Tropical Diseases</span>: <span class="subject" title="subject">Chikungunya Infection</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Chikungunya Infection</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population groupings</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Ethnicities</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Latin American people</span>: <span class="subject" title="subject">Mexican People</span></div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Evolutionary analysis of the Chikungunya virus epidemic in Mexico reveals intra-host mutational hotspots in the E1 protein</div> <div class="alt-title" title="alt-title">
Chikungunya virus evolution in Mexico</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-1289-4457</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muñoz-Medina</span><span class="given-names" tagx="given-names" title="given-names">José Esteban</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Knight</span><span class="given-names" tagx="given-names" title="given-names">Miguel Antonio</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a><a href="#currentaff001"><sup>¤</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Flores</span><span class="given-names" tagx="given-names" title="given-names">Alejandro</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monroy-Muñoz</span><span class="given-names" tagx="given-names" title="given-names">Irma Eloísa</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grande</span><span class="given-names" tagx="given-names" title="given-names">Ricardo</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Esbjörnsson</span><span class="given-names" tagx="given-names" title="given-names">Joakim</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff005"><sup>5</sup></a><a href="#aff006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santacruz-Tinoco</span><span class="given-names" tagx="given-names" title="given-names">Clara Esperanza</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González-Bonilla</span><span class="given-names" tagx="given-names" title="given-names">César Raúl</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff007"><sup>7</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Laboratorio Central de Epidemiología, Centro Médico Nacional "La Raza�?, Instituto Mexicano del Seguro Social, Mexico City, Mexico</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Centro de Investigaciones en Ciencias Microbiológicas, Instituto de Ciencias, Benemérita Universidad Autónoma de Puebla, Puebla, Mexico</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Unidad Universitaria de Secuenciación Masiva y Bioinformática, Instituto de Biotecnología, Universidad Nacional Autónoma de México, Cuernavaca, Morelos, Mexico</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Laboratorio de Genómica, Departamento de Genética y Genómica Humana, Instituto Nacional de Perinatología "Isidro Espinosa de los Reyes�?, Mexico City, México</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Systems Virology, Department of Laboratory Medicine, Lund University, Lund, Sweden</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">NDM Research Building, Nuffield Department Medicine, University of Oxford, Oxford, United Kingdom</span></span><span class="citation_author_institution" id="aff007">[7], <span class="addr-line" title="addr-line">División de Laboratorios de Vigilancia e Investigación Epidemiológica, Instituto Mexicano del Seguro Social, Mexico City, Mexico</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moratorio</span><span class="given-names" tagx="given-names" title="given-names">Gonzalo</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Institut Pasteur Montevideo, URUGUAY</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-current-aff" title="current-aff"><span class="label" tagx="label" title="label">¤</span><p>Current address: Department of Microbiology and Immunology, University of California, San Francisco, California, United States</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">eban10@hotmail.com</span> (JEMM); <span class="email" tagx="email" title="email">miguel.garciaknight@ucsf.edu</span> (MAGK); <span class="email" tagx="email" title="email">alexsf@ibt.unam.mx</span> (ASF)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-12-12</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0209292</span><span class="history" title="history"><span class="received" title="received">received: 2018-8-29</span><span class="accepted" title="accepted">accepted: 2018-12-3</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Muñoz-Medina et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0209292.pdf" title="self-uri">pone.0209292.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The epidemic potential of Chikungunya virus (CHIKV) was recently made evident by its introduction and rapid expansion in the Caribbean and the Americas. We sought to gain a detailed understanding of the dynamics of the epidemic in Mexico, the country with the highest number of confirmed CHIKV cases in the Americas, and to characterise viral evolution at the population and intra-host level. Analysis of the spatiotemporal distribution of 2,739 diagnosed cases in Mexico from December 2014 to December 2015 showed a rapid nationwide expansion of the epidemic with focalisation in the South West of the country. We sequenced the envelope glycoprotein 1 gene (E1) from 25 patients using the Illumina MiSeq platform and report synonymous and non-synonymous consensus mutations. Bayesian phylogenetic analysis using 249 Asian lineage E1 sequences gave updated estimates of nucleotide substitution rates for E1 and time to most recent common ancestor of major lineages. The analysis indicates phylogenetically-related emergent Latin American clusters in South Western Mexico, Nicaragua and Honduras and transmission of American strains in the Pacific islands. Detailed analysis showed that intra-host changes in E1 mainly occurred in two variable regions (E1:189–220 and E1:349–358) in domains II and III, respectively, in residues involved in inter and intra-envelope spike interactions. At the population level, this study sheds light on the introduction and evolutionary dynamics of CHIKV in the Americas. At the intra-host level, this study identifies mutational hotspots of the E1 protein with implications for understanding the relationship between the CHIKV quasispecies, viral fitness and pathogenesis.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100004881</span><span class="institution" title="institution">Instituto Mexicano del Seguro Social</span></div> </div><span class="award-id" tagx="award-id" title="award-id">FIS/IMSS/PROT/G16/1575</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-1289-4457</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muñoz-Medina</span><span class="given-names" tagx="given-names" title="given-names">José Esteban</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010096</span><span class="institution" title="institution">Secretaría de Educación Pública</span></div> </div><span class="award-id" tagx="award-id" title="award-id">103.5/16/10420</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Knight</span><span class="given-names" tagx="given-names" title="given-names">Miguel Antonio</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100007350</span><span class="institution" title="institution">Consejo Nacional de Ciencia y Tecnología</span></div> </div><span class="award-id" tagx="award-id" title="award-id">264598</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Knight</span><span class="given-names" tagx="given-names" title="given-names">Miguel Antonio</span></span></span></div> <div class="funding-statement" title="funding-statement">
This study was supported by the Instituto Mexicano del Seguro Social (FIS/IMSS/PROT/G16/1575 to JEMM), the Secretaría de Educación Pública (103.5/16/10420 to MAGK), the Consejo Nacional de Ciencia y Tecnología (264598 to MAGK). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="We have submitted the Illumina reads to the NCBI. The raw data for all samples were deposited under Bioproject ID PRJNA495608 at the NCBI. Accession numbers: SRX4824662; SRX4824661; SRX4824660; SRX4824659; SRX4824658; SRX4824657; SRX4824656; SRX4824655; SRX4824654; SRX4824649; SRX4824648; SRX4824647; SRX4824646; SRX4824653; SRX4824652; SRX4824651; SRX4824650; SRX4824645; SRX4824644; SRX4824667; SRX4824668; SRX4824665; SRX4824666 and SRX4824664." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>We have submitted the Illumina reads to the NCBI. The raw data for all samples were deposited under Bioproject ID PRJNA495608 at the NCBI. Accession numbers: SRX4824662; SRX4824661; SRX4824660; SRX4824659; SRX4824658; SRX4824657; SRX4824656; SRX4824655; SRX4824654; SRX4824649; SRX4824648; SRX4824647; SRX4824646; SRX4824653; SRX4824652; SRX4824651; SRX4824650; SRX4824645; SRX4824644; SRX4824667; SRX4824668; SRX4824665; SRX4824666 and SRX4824664.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Chikungunya fever is an infectious disease caused by the Chikungunya virus (CHIKV), an arbovirus of the <i>Togaviridae</i> family with a widespread and expanding global distribution. First identified in Tanzania in 1952, CHIKV is principally transmitted, in urban transmission cycles, by <i>Aedes aegypti</i> mosquitoes that mainly inhabit tropical and subtropical regions[<a href="#pone.0209292.ref001">1</a>,<a href="#pone.0209292.ref002">2</a>]. CHIKV has a 11.8kb positive sense RNA genome that encodes four non-structural proteins (nsP1-4) and three main structural proteins (capsid and the spike forming envelope glycoproteins [E1&amp;amp;2])[<a href="#pone.0209292.ref003">3</a>]. Though rarely fatal, CHIKV has a high attack rate and infection can lead to high fever, headache, rash and characteristic severe debilitating arthralgia which can persist. To date, no specific prophylactic or therapeutic interventions exist.</p> <p>Phylogenetic analyses of CHIKV have identified three main lineages: the West African, East Central South African (ECSA) and Asian linage[<a href="#pone.0209292.ref004">4</a>]. In 2004, an epidemic of the ECSA lineage re-emerged in Africa and the Indian Ocean and evolved into a novel Indian Ocean Lineage (IOL)[<a href="#pone.0209292.ref005">5</a>,<a href="#pone.0209292.ref006">6</a>]. Adaptive mutations (e.g. E1:A226V[<a href="#pone.0209292.ref007">7</a>] &amp;amp; E2: L210Q[<a href="#pone.0209292.ref008">8</a>]) facilitating transmission by <i>Ae</i>. <i>albopictus</i> emerged in IOL strains and significant outbreaks occurred in India, South East Asia and Italy. The first autochthonous cases of CHIKV in the Western Hemisphere were reported in Martinique in December 2013; rapid epidemic expansion followed throughout the Caribbean and Latin America leading to over 30,000 laboratory confirmed cases by April 2015[<a href="#pone.0209292.ref009">9</a>]. The epidemic in the Americas is derived from the re-emergent Asian lineage[<a href="#pone.0209292.ref010">10</a>–<a href="#pone.0209292.ref012">12</a>], (though ECSA lineage infections lacking E1:A226V have been identified in Brazil[<a href="#pone.0209292.ref011">11</a>]); a novel Caribbean genotype has been described in isolates from across the Americas containing a transmission-facilitating 3’ untranslated region duplication[<a href="#pone.0209292.ref013">13</a>].</p> <p>The first autochthonous CHIKV case in Mexico was reported in October 2014 in the southern border state of Chiapas[<a href="#pone.0209292.ref012">12</a>]. By November, cases had been confirmed in nine states and the Mexican Institute of Social Security (IMSS, "Instituto Mexicano del Seguro Social�?), which provides health care to ~60% of the population[<a href="#pone.0209292.ref014">14</a>], implemented a confirmatory diagnosis algorithm for CHIKV following guidelines set by the National Institute of Diagnosis and Reference (InDRE, "Instituto de Diagnóstico y Referencia Epidemiológicos�?). Here we analyse the cumulative confirmed cases reported within the IMSS by state from December 2014 to December 2015. We also characterized temporally and geographically representative E1 gene sequences and present an up-to-date phylogenetic analysis of CHIKV lineages, with a focus on the Asian lineage epidemic in the Americas and the Caribbean. Finally, we explore the differing degrees of E1 protein diversity in the viral quasispecies within patients and identify hotspots of intra-patient mutation.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="cumulativecasesofchikvinmexico" title="sec"> <div class="title" tagx="title" title="title">
Cumulative cases of CHIKV in Mexico</div> <p>Out of 5,266 patient serum samples received by the LCE between December 2014 and December 2015, 2,739 samples (52.0%) were CHIKV positive. Univariate analysis indicated that age and geographical region, but not gender, was associated with CHIKV infection. 18–29 years made up the largest age category with positive cases (22.3%) and Mexican states in the south of the country reported most cases (78.4%; <b><a href="#pone.0209292.s003">S1 Table</a></b>).</p> <p>Analysis of the cumulative positive cases in Mexico indicated an initial appearance of the disease in the states of Chiapas and Veracruz from December 2014 to January 2015, followed by cases in states on the Pacific coast including Guerrero, Oaxaca, Colima and the northern state of Sonora (<b><a href="#pone.0209292.g001">Fig 1</a></b>). By mid-2015, infections were detected in central and northern states and in the Yucatan peninsula (<b><a href="#pone.0209292.g001">Fig 1</a></b>) and by December 2015, IMSS clinical sites from all states in Mexico reported confirmed cases.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0209292.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Accumulation of confirmed CHIKV cases per state over time.</div> <p>Political maps of Mexico with state borders shown. Confirmed cases through molecular diagnosis were from patients who attended any of the 35 IMSS clinical centres nationwide. Grey indicates that no cases were reported.</p>   </div> <p>Specimens from 500 patients sampled from January-November 2014 who met the definition for suspected DENV infection but who were found to be DENV-negative, were analysed to assess CHIKV infection. No CHIKV-positive samples were identified prior to December 2014.</p> </div> <div class="novelnonsynonymousmutationinachikve1consensussequence" title="sec"> <div class="title" tagx="title" title="title">
Novel nonsynonymous mutation in a CHIKV E1 consensus sequence</div> <p>To characterise the CHIKV strains circulating in Mexico during the epidemic, we determined the consensus sequence of the E1 glycoprotein-encoding gene from 25 CHIKV-infected patients (<b><a href="#pone.0209292.s004">S2 Table</a></b>) from five Mexican states (five from each state) in the Central/southern region which had most confirmed CHIKV cases by August 2015. Two partial amplicons from the E1 gene were generated (COF1 and CIF2) and deep sequenced. The most abundant sequences from each were overlapped and used to reconstruct a 1044bp region from the E1 gene (E1:270–1314).</p> <p>Amongst all patient-derived consensus sequences, we observed a total of nine nucleotide substitutions resulting in two nonsynonymous mutations (<b><a href="#pone.0209292.t001">Table 1</a></b>). Most of the observed nucleotide substitutions occurred in separate specimens except for A741G and G855A mutations which were observed in six and two specimens, respectively. The nonsynonymous T207M and V291I mutations were observed in specimens from patients from the Pacific states of Colima and Guerrero, respectively.</p> <div class="table-wrap_UNKNOWN" id="pone.0209292.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0209292.t001">10.1371/journal.pone.0209292.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Mutations in partial CHIKV E1 consensus sequences from Mexican patients.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0209292.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="justify" rowspan="1" colspan="1">Nucleotide change</th> <th align="justify" rowspan="1" colspan="1">Codon position</th> <th align="justify" rowspan="1" colspan="1">AA<br /><br />change</th> <th align="justify" rowspan="1" colspan="1">Genomic position<a href="#t001fn001">*</a></th> <th align="justify" rowspan="1" colspan="1">Sample</th> </tr> </thead> <tbody> <tr> <td align="justify" rowspan="1" colspan="1">C384T</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;S</td> <td align="justify" rowspan="1" colspan="1">10,377</td> <td align="justify" rowspan="1" colspan="1">Chiapas_8</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">T516A</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;P</td> <td align="justify" rowspan="1" colspan="1">10,509</td> <td align="justify" rowspan="1" colspan="1">Veracruz_1</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">C620T</td> <td align="justify" rowspan="1" colspan="1">2</td> <td align="justify" rowspan="1" colspan="1">T207M</td> <td align="justify" rowspan="1" colspan="1">10,613</td> <td align="justify" rowspan="1" colspan="1">Colima_13</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">A741G</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;R</td> <td align="justify" rowspan="1" colspan="1">10,738</td> <td align="justify" rowspan="1" colspan="1">Guerrero_3, Guerrero_9, Guerrero_15, Guerrero_16, Guerrero_18 &amp;amp; Colima_23</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">G855A</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;A</td> <td align="justify" rowspan="1" colspan="1">10,848</td> <td align="justify" rowspan="1" colspan="1">Guerrero_9 &amp;amp; Guerrero_15</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">G871A</td> <td align="justify" rowspan="1" colspan="1">1</td> <td align="justify" rowspan="1" colspan="1">V291I</td> <td align="justify" rowspan="1" colspan="1">10,864</td> <td align="justify" rowspan="1" colspan="1">Guerrero_15</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">T1054C</td> <td align="justify" rowspan="1" colspan="1">1</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;L</td> <td align="justify" rowspan="1" colspan="1">11,047</td> <td align="justify" rowspan="1" colspan="1">Guerrero_15</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">A1200G</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;Q</td> <td align="justify" rowspan="1" colspan="1">11,193</td> <td align="justify" rowspan="1" colspan="1">Colima_2</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">T1248C</td> <td align="justify" rowspan="1" colspan="1">3</td> <td align="justify" rowspan="1" colspan="1">Syn&amp;gt;G</td> <td align="justify" rowspan="1" colspan="1">11,241</td> <td align="justify" rowspan="1" colspan="1">Veracruz_20</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><b>*</b>S27 genome; AA, amino acid; Syn, synonymous</p> </div> </div> </div> </div> <div class="evolutionofthemajorchikvlineages" title="sec"> <div class="title" tagx="title" title="title">
Evolution of the major CHIKV lineages</div> <p>To assess the evolutionary relationship between the 25 Mexican strains and CHIKV strains sampled globally, and to provide an updated evaluation of evolution within the Asian lineage using all available E1 sequences, we aligned our sequences with 395 CHIKV strains. Our initial maximum likelihood analysis did not resolve the ECSA lineage as being monophyletic, as has been shown in analyses using whole genomes [<a href="#pone.0209292.ref015">15</a>]. However, the use of a starting tree resulted in a maximum likelihood phylogeny that resolved the three major CHIKV lineages (Asian, ECSA and WAf) with good branch support for all lineages including the IOL and American and Caribbean epidemic sublineage (<b><a href="#pone.0209292.g002">Fig 2</a></b>). As expected, the 25 Mexican sequences all clustered with the epidemic Asian lineage strains from the Americas.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0209292.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood phylogenetic analysis of partial CHIKV E1 gene sequences.</div> <p>A phylogeny of 420 CHIKV strains with an O’nyong-nyong strain used as an outgroup (not shown) built using the General Time Reversible substitution model with a gamma distributed rate variation among sites. Statistical support for major tree nodes is shown as aLRT scores. Epidemic linages are shown in grey and a statistically supported cluster of 6 Mexican strains is marked with an asterisk. The scale bar represents nucleotide substitutions per site. ECSA, East-Central-South Africa; IOL, Indian Ocean Lineage; WAf, West African.</p>   </div> <p>Next, we analysed the major CHIKV lineages shown in <b><a href="#pone.0209292.g002">Fig 2</a></b> separately using a Bayesian coalescence approach to estimate the evolutionary rate and the time to the most recent common ancestor (tMRCA) of each lineage. We initially assessed the evolutionary signal of the alignments using root-to-tip regression estimates (<b><a href="#pone.0209292.s001">S1 Fig</a></b>). We then analysed the Asian all, Asian old, non-IOL ECSA, IOL and WAf datasets, which had strong evolutionary signals (<i>R</i><sup><i>2</i></sup> &amp;gt;0.63 for all datasets); the Asian sequences sampled post 1995 had a poor evolutionary signal (<i>R</i><sup><i>2</i></sup> = 0.15) and were not analysed (<b><a href="#pone.0209292.s001">S1 Fig</a></b>).</p> <p>The evolutionary rate of the entire Asian lineage was similar to the rate of the Asian old dataset (6.71E-4 [95% HPD: 4.92E-4, 8.51E-4] vs 6.52E-4 [95% HPD: 3.01E-4, 9.91E-4] nucleotide substitutions/site/year), indicating that the recent epidemics strains evolve at a similar rate to older Asian lineage strains (<b><a href="#pone.0209292.t002">Table 2</a></b>). The highest evolutionary rate estimate was for the IOL lineage (15.44E-4 [95% HPD: 11.17E-4, 20.28E-4] substitutions/site/year) which was over two-fold higher than the non-IOL ECSA lineage (6.04E-4 [95% HPD: 3.17E-4, 9.40E-4] substitutions/site/year). The lowest mean evolutionary rate estimate was observed for the WAf lineage (4.93E-4 [95% HPD 1.93E-4, 9.38E-4] substitutions/site/year).</p> <div class="table-wrap_UNKNOWN" id="pone.0209292.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0209292.t002">10.1371/journal.pone.0209292.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Rate of nucleotide substitutions and tMRCAs of mayor CHIKV lineages.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0209292.t002g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Dataset</th> <th align="center" rowspan="1" colspan="1">Evolutionary rate<br /><br />Mean nucleotide substitutions/site/year<br /><br />(95% HPD interval)</th> <th align="center" rowspan="1" colspan="1">tMRCA<br /><br />(root height 95% HPD interval)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Asian all</td> <td align="left" rowspan="1" colspan="1">6.71E-4 (4.92E-4-8.51E-4)</td> <td align="left" rowspan="1" colspan="1">07/1957 (01/1955-07/1958)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Asian old</td> <td align="left" rowspan="1" colspan="1">6.52E-4 (3.01E-4-9.91E-4)</td> <td align="left" rowspan="1" colspan="1">08/1957 (12/1954-07/1958)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">ECSA</td> <td align="left" rowspan="1" colspan="1">6.04E-4 (3.17E-4-9.40E-4)</td> <td align="left" rowspan="1" colspan="1">11/1947 (04/1937-02/1953)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">IOL</td> <td align="left" rowspan="1" colspan="1">15.44E-4 (11.17E-4-20.28E-4)</td> <td align="left" rowspan="1" colspan="1">11/2003 (09/2002-07/2004)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">WAf</td> <td align="left" rowspan="1" colspan="1">4.93E-4 (1.93E-4-9.38E-4)</td> <td align="left" rowspan="1" colspan="1">05/1961 (06/1954-07/1964)</td> </tr> </tbody> </table> </div> </div> </div> <div class="phylogenyoftheasianlineage" title="sec"> <div class="title" tagx="title" title="title">
Phylogeny of the Asian lineage</div> <p>To assess the evolution of the Asian lineage in detail, we analysed the MCC tree of all Asian lineage sequences (<b><a href="#pone.0209292.g003">Fig 3A</a></b>). Our analysis largely recapitulates the evolutionary relationships observed in studies using whole genomes sequences[<a href="#pone.0209292.ref004">4</a>,<a href="#pone.0209292.ref015">15</a>,<a href="#pone.0209292.ref016">16</a>]: i) emergence of the Asian lineage in the late 1950s (07/1957 [95% HPD 01/1955-07/1958]; <b><a href="#pone.0209292.t002">Table 2</a></b>); ii) early evolution of Thai and Indian clades and probable extinction of the India clade; iii) introduction of a Thai lineage into Indonesia and the Philippines in the early 1980s (01/1980 [95%HPD: 11/1976-09/1982]); and iv) endemic transmission prior to the emergence of epidemic lineages from South East Asia and Oceania which derived from a common ancestor in the early 2000s (03/2000 [95% HPD: 04/2004-08/1995]).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0209292.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Bayesian phylogenetic analysis of partial CHIKV E1 gene sequences.</div> <p>(A) Bayesian MCC tree of 249 Asian lineage strains calibrated to the sampling date and built using the HKY+G nucleotide substitution model, a lognormal relaxed clock and a Bayesian Skygrid coalescent model. Nodes with posterior probabilities (clade-credibility values) ≥0.9 have 95% HPD intervals for node heights shown. The two main epidemic Asian lineage clades (CL) are also shown. (B) A subtree of the American and Caribbean sublineage showing posterior probabilities and calibrated tMRCA and 95% HPD intervals for statistically supported monophyletic groups. For clarity, node bars with 95% HPD intervals are not shown, only relevant strains are labelled, and the colour code is as in section a).</p>   </div> <p>Studies using whole genomes have additionally identified two major recent monophyletic epidemic clades[<a href="#pone.0209292.ref015">15</a>,<a href="#pone.0209292.ref017">17</a>]. Our phylogeny did not resolve the monophyly of clade I (<b><a href="#pone.0209292.g003">Fig 3A</a></b>). However, clade II was estimated to have originated late in 2010 (09/2010 [95% HPD: 06/2009-10/2011]) in South East Asia where it spread to Pacific Island nations such as Micronesia, Tonga and American Samoa. These sequences are the most closely related to the epidemic sublineage in the Caribbean and Americas, which arose at the start of 2013 (03/2013 [95% HPD: 09/2012-07/2013]; <b><a href="#pone.0209292.g003">Fig 3B</a></b>). Within this sublineage, a well-supported (posterior probability [PP] &amp;gt; 0.9) Mexican cluster (all five specimens form the state of Guerrero and one from Colima [Colima_23]) and two Nicaraguan (accession numbers: KT192722, KT192727, KT192735 and KT192740; and KT192718, KT192724, KT192732, KT192733, KT192736) clusters were found, emerging in late 2014 (11/2014 [95% HPD: 09/2014-01/2015], 11/2014 [95% HPD: 09/2014-11/2014] and 09/2014 [95% HPD: 08/2014-11/2014], respectively; <b><a href="#pone.0209292.g003">Fig 3B</a></b>). Sequences forming dyad clusters were also detected from Mexico (Guerrero_9 and Guerrero_15), Nicaragua/Honduras (KT192731 and KR559488) Colombia/Unknown origin (KR559491 and KR559497) and French Polynesia (KM985619 and KR559473). Interestingly, the former dyad (PP = 0.75) and a third sequence from French Polynesia (KJ939333) suggests possible local transmission of the American and Caribbean sublineage in the Pacific islands. By contrast, the two sequences from Italy (KP252141 and KP252142) and one from Thailand (KT192719), are likely indicative of traveller-associated infection.</p> </div> <div class="intra-patientdiversityofchikv" title="sec"> <div class="title" tagx="title" title="title">
Intra-patient diversity of CHIKV</div> <p>The use of next generation sequencing allowed us to analyse intra-patient variability in the reconstructed E1 gene amplicons (COF1 and CIF2; sequencing yields shown in <b><a href="#pone.0209292.s005">S3 Table</a></b>). After clustering the nucleotide sequences with a minimum 50 amplicons, we clustered the translated products for each nucleotide cluster with 100% identity, obtaining 67 amino acid sequence haplotypes for COF1 and 18 amino acid sequence haplotypes for CIF2. The relative abundance of each haplotype is shown in <b><a href="#pone.0209292.g004">Fig 4</a></b>. A wide degree of heterogeneity in the intra-patient diversity of viral haplotypes was seen, and the number of variants present in COF1 and CIF2 correlated significantly amongst patients (<b><a href="#pone.0209292.s002">S2 Fig</a></b>). Variant number ranged from the presence of a single variant (COF1_1 in patient Veracruz_4, CIF2_1 in eight patients and CIF2_13 in Guerrero_15;) to 56 COF1 and 17 CIF2 variants within a single patient (Oaxaca_25) with diverse degrees of abundance (<b><a href="#pone.0209292.g004">Fig 4</a></b>). We also observed the same dominant haplotypes in both amplicons (COF1_1 and CIF2_1) in all patients except for Colima_13 and Guerrero_15 (<b><a href="#pone.0209292.g004">Fig 4</a></b>), where separate nonsynonymous mutations in the consensus sequence were present at a frequency &amp;gt;50% (<b><a href="#pone.0209292.t001">Table 1</a> and <a href="#pone.0209292.s006">S4 Table</a></b>). In COF1, six haplotypes (COF1_1–6) were highly abundant in most patients, whereas in CIF2, patients either had nine or more variant haplotypes (N = 12) or 1–4 variants (N = 13).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0209292.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Abundance of viral haplotypes in two E1 gene amplicons.</div> <p>Heat maps indicating the relative abundance (red colour intensity) of distinct amino acid viral haplotypes following deep sequencing of two amplicons COF 1 (A) and CIF 2 (B) that span the near full-length CHIKV E1 gene in 25 Mexican patients. Dendrograms represent clustering of patients according to haplotype diversity.</p>   </div> <p>Analysis of the location of E1 amino acid mutations amongst the intra-patient quasispecies indicated that most variation occurred within domains II and III (<b><a href="#pone.0209292.g005">Fig 5</a></b>). Low frequency (0.3%) C96F mutations in the fusion loop were also observed in four patients (<b><a href="#pone.0209292.g005">Fig 5A</a> and <a href="#pone.0209292.s006">S4 Table</a></b>) and the dominant V291I mutation in patient Guerrero_15 mapped to domain I (<b><a href="#pone.0209292.g005">Fig 5A</a></b>). Of note, variation was centred around two regions located between E1 residues 189–220 (hotspot one; 22/38 variation sites) and 349–358 (hotspot two; 8/38 variation sites; <b><a href="#pone.0209292.g005">Fig 5A</a></b>) in domain II and III, respectively. Nine and 13 mutations within hotspots one and two, respectively, occurred in over a third of patients and at average frequencies &amp;gt;1% <b>(<a href="#pone.0209292.s006">S4 Table</a>)</b>. These two regions include residues involved in E1-E1 inter-spike interactions (191–194, 351, 353 and 355) and E2-E1 intra-spike interaction (196–199) as well as residues highly conserved amongst alphaviruses: 193, 198, 202, 204 and 356[<a href="#pone.0209292.ref018">18</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0209292.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Sites of intrapatient mutations in CHIKV E1 protein.</div> <p>(A)Linear diagram of CHIKV E1 protein (adapted from Voss et al[<a href="#pone.0209292.ref018">18</a>] indicating protein domains, amino acid positions (top) and the positions of amino acid mutations (vertical bars) including two mutational hotspots (1 &amp;amp; 2) with the following mutations in order of location: 1 = Y189D, P191T/A/H/R/S, F192V/C/L/S/Y/I, G193D, A194E/G, A195R, R196S/G, P197Q/T, G198V, Q199H/K, F200V/I/C/L, D202E, I203S/T, Q204L/H/P, S205R/G, T207M, E209A, S210R, Y214X, Q218P/H, L219R, V220G; and 2 = N349K, S350A, L352W, Q353K/P, I354S/L/T, S355A/P, F356C/L/S, T358P). Red bars indicate mutations with frequencies &amp;gt;0.01 and that were present in &amp;gt;50% of patients or mutations present in the consensus sequence of a single patient. FL, fusion loop. (B) Structural model of native CHIKV E1 protein indicating the locations of intrapatient point mutations (red).</p>   </div> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We sought to gain insights into CHIKV evolution by analysing the emergent epidemic in Mexico, where over 11,000 confirmed cases were reported in 2015, the highest number in any country that year [<a href="#pone.0209292.ref019">19</a>]. As seen in other non-endemic countries, with the exception of Panama[<a href="#pone.0209292.ref020">20</a>], CHIKV spread rapidly to tropical regions nationwide. The endemicity of the primary vector <i>Ae</i>. <i>aegytpi</i> and an immunologically naïve population were central facilitators of the epidemic; however specific factors leading to the focalisation of cases in the South of the country, particularly along the Pacific Ocean, merit attention. Of note, viruses from this region harboured non-synonymous mutations and formed an emergent cluster, suggesting localised evolution of CHIKV in Mexico. Interestingly, the number of Zika virus (ZIKV) cases during the 2016 epidemic in Mexico was also highest in the Southern states of Guerrero, Veracruz and Yucatán[<a href="#pone.0209292.ref021">21</a>]. Striking similarities in transmission dynamics following CHIKV and ZIKV epidemics in the Americas have been observed[<a href="#pone.0209292.ref022">22</a>]; our data suggest that these similarities are also apparent within Mexico and this may have implications for future arboviral threats as such as Yellow Fever, Mayaro and Oropuche viruses.</p> <p>Local CHIKV transmission was first detected in Mexico in October 2014 amid reports of cases in Guatemala and the United States[<a href="#pone.0209292.ref023">23</a>]. Similarly to Brazil[<a href="#pone.0209292.ref011">11</a>], our analysis supports the introduction of CHIKV shortly before the first autochthonous diagnosed cases, as we found no evidence of CHIKV infection in 500 febrile DENV-negative patients sampled from across the country between January and October 2014. In addition, the tMRCA for the observed Mexican cluster was estimated for November 2014 (September 2014- January 2015 95% HPD). By contrast, ZIKV was reported to be present in Mexican patients 3–10 months prior to the first reports of imported cases[<a href="#pone.0209292.ref024">24</a>] and to have circulated in Brazil up to 12 months prior to its detection[<a href="#pone.0209292.ref025">25</a>]. We suggest that knowledge of ongoing CHIKV transmission in the region and the high rate of symptomatic infections contributed to the early detection of the first cases in Mexico. Active surveillance of emergent arboviral threats, particularly of those with high numbers of asymptomatic cases, is therefore critical for early containment efforts.</p> <p>To assess the evolutionary dynamics of major CHIKV lineages in Mexico and worldwide, we analysed E1 gene sequences. A comparison of phylogenetic trees inferred through ML analyses using whole CHIKV genomes and E1 gene sequences, indicated that the former do not adequately resolve the ECSA lineage[<a href="#pone.0209292.ref004">4</a>]. However, the use of a starting tree in our ML analysis enabled us to resolve all major lineages, including the ECSA lineage. In addition, separate alignments of major lineages indicated robust phylogenetic signals for the datasets analysed. Importantly, our approach captured diversity from Asian lineage viruses lacking whole genome sequences. Our tMRCA estimates for the Asian, IOL and WAf lineages were similar to studies analysing the whole CHIKV genome[<a href="#pone.0209292.ref004">4</a>,<a href="#pone.0209292.ref015">15</a>,<a href="#pone.0209292.ref026">26</a>]. However, our estimates for the ECSA lineage were 10 to 35 years more recent, reflecting the complex phylogenetic structure of these sequences and the limitations of analyses focused on E1 gene sequences for this lineage[<a href="#pone.0209292.ref015">15</a>]. In accordance with a detailed phylogeographic study[<a href="#pone.0209292.ref017">17</a>], we estimated, albeit with wider 95% HPD values, that the American and Caribbean sublineage evolved around March 2013 from closely related South Pacific strains. How CHIKV and ZIKV were initially introduced into the Caribbean and Atlantic coast of America, respectively, from Oceania or South-East Asia without apparent prior transmission to the Western coast of the Americas remains unclear. Identifying the ongoing transmission routes of American lineage CHIKV to Pacific Islands, such as French Polynesia, may provide clues, particularly as these cases appear to be forming epidemiologically linked clusters. By contrast, likely imported cases to Italy and Thailand, seem unrelated to recently reported outbreaks[<a href="#pone.0209292.ref027">27</a>]. Our result also suggest the presence of emerging clusters in Mexico and Nicaragua and epidemiologically-related transmission between Nicaragua and Honduras which, with other emerging clusters in the region, warrant surveillance, in particular when assessing sylvatic transmission emergence in the Americas[<a href="#pone.0209292.ref028">28</a>].</p> <p>Our evolutionary rate estimates amongst Asian datasets (<b><a href="#pone.0209292.t001">Table 1</a>)</b> were similar to what was reported for E1 by Sahadeo <i>et al</i>[<a href="#pone.0209292.ref017">17</a>] (6.0 E-4 [95% HPD: 4.0–8.0 E-4] nucleotide substitution/site/year) and higher than what has been reported for whole genomes[<a href="#pone.0209292.ref004">4</a>,<a href="#pone.0209292.ref011">11</a>,<a href="#pone.0209292.ref015">15</a>]. In accordance with Chen <i>et al</i>[<a href="#pone.0209292.ref015">15</a>], the evolutionary rate of the IOL lineage was over twice that of the ECSA and other lineages -with non-overlapping 95% HPD values- though our estimates were an order of magnitude higher than this report, which was based on the whole genome. The convergent evolution of the adaptive E1:A226V mutation in different IOL clusters[<a href="#pone.0209292.ref007">7</a>] likely contributed to the increased the rate of evolution in E1. More generally, the high evolutionary rate estimates of the IOL lineage may result from frequent sampling during the epidemic and inclusion of variants not purged by purifying selection[<a href="#pone.0209292.ref029">29</a>], though this effect seems to be less marked in the American and Caribbean sublineage in our study but not others[<a href="#pone.0209292.ref017">17</a>].</p> <p>As expected for an arbovirus where evolution is constrained due to replication in dual hosts[<a href="#pone.0209292.ref030">30</a>], variation in the consensus sequences of the E1 gene from 25 patients was limited. We found an undescribed T207M mutation in a single individual that maps to domain II and lies adjacent to an arginine residue involved in E1-E1 inter-spike contacts[<a href="#pone.0209292.ref018">18</a>]. The replacement of a polar amino acid with a hydrophobic residue indicates possible structural changes that should be characterised functionally. We also observed a V291I mutation that was seen in five patients in an outbreak of the IOL lineage in Kerala, India between 2009 and 2013[<a href="#pone.0209292.ref031">31</a>] and in one patient from Nicaragua sampled in December 2014[<a href="#pone.0209292.ref032">32</a>]. V291I maps to domain I and the outer surface of a E1 trimer were it has been suggested to have a limited role in modulating infectivity[<a href="#pone.0209292.ref031">31</a>].</p> <p>The massive parallel sequencing of samples also enabled us to analyse viral diversity in each patient. To assess the entire haplotype of each E1 amplicon (i.e. mutations present in <i>cis</i>) we only included overlapping paired end reads that reconstructed the amplicons and had a sequencing depth of at least 50 overlapping reads. We used this approach to reduce the inclusion of sequencing errors or artefacts, allowing us to explore a real representation of the diversity for each patient. Strikingly, a marked heterogeneity in the level of haplotype diversity at the amino acid level was seen in each patient, with levels consistent over both COF and CIF amplicons. The diversity of viral quasispecies is a critical component of viral fitness[<a href="#pone.0209292.ref033">33</a>], as demonstrated by reduced fitness and pathogenicity of CHIKV replication fidelity variants[<a href="#pone.0209292.ref034">34</a>,<a href="#pone.0209292.ref035">35</a>]. Whether intrahost diversity relates to the clinical outcome of CHIKV and other arboviruses, such as dengue[<a href="#pone.0209292.ref036">36</a>], is a key question. Although our study did not focus on clinical outcomes, most of the primary symptoms reported by patients analysed by sequencing (fever, arthralgia, myalgia and / or headache) were similar to those observed by authors in Mexico and the Island of Reunion [<a href="#pone.0209292.ref037">37</a>–<a href="#pone.0209292.ref038">38</a>], though we observed a greater proportion of arthritis and less haemorrhagic cases <b>(<a href="#pone.0209292.s008">S6 Table</a>)</b>. In addition, a similar proportion of patients showed high levels of haplotype diversity as those reported to have persistent clinical symptoms in the Americas (~ 50%)[<a href="#pone.0209292.ref039">39</a>]. Therefore, in accordance with the variation in clinical outcomes in acute and persistent stages of infection, we agree with the conclusions of previous reports[<a href="#pone.0209292.ref040">40</a>] that the association between interpatient viral diversity and clinic outcome warrants further detailed studies.</p> <p>Of note, the number of non-synonymous intra-host variants observed within the E1 coding sequence was higher than those reported previously in studies assessing the whole CHIKV genome[<a href="#pone.0209292.ref013">13</a>,<a href="#pone.0209292.ref017">17</a>]. The reasons for this discrepancy may be due to the sequencing depth obtained from analysing E1 sequences alone. We suggest that the inclusion of processing artefacts in our analysis (e.g. PCR-generated mutations) is unlikely. For instance, the presence of the same variant in numerous patients and the non-random distribution of variants in discrete mutational hotspots suggest natural variation. Furthermore, we only report minority variants with frequencies &amp;gt;0.1% that represent reads with ≥50 identical nucleotide sequences after clustering at nucleotide level. Individually, most variants occurred at frequencies &amp;lt;10%, suggesting reduced fitness in comparison to the consensus sequence. Interestingly, similar frequencies of minority variants were observed across patients, suggesting a possible evolutionary advantage to their maintenance in the viral swarm[<a href="#pone.0209292.ref041">41</a>]. Alternatively, similar pathways may be used by CHIKV quasispecies to explore sequence space. We also observed a low frequency (0.2%) stop codon mutation (Y214X) in two patients, which may indicate the maintenance of partial or defective genomes[<a href="#pone.0209292.ref042">42</a>], with possible fitness effects. Stop codon mutations leading to deletions have similarly been reported in the Venezuelan Equine Encephalitis virus 6K protein [<a href="#pone.0209292.ref043">43</a>]. The non-random distribution of intra-host mutations in E1 could be driven by several mechanisms. <i>Cis</i> elements that alter template structure and reverse transcriptase fidelity in specific regions of envelope proteins have been demonstrated for HIV[<a href="#pone.0209292.ref044">44</a>]. Alternatively, these sites may be under low selective constraints and be influenced by neutral or positive selection, possibly through immune-selection pressure. To our knowledge, no neutralizing antibodies elicited against CHIKV have been shown to target residues between E1 189–220 and 349–358[<a href="#pone.0209292.ref045">45</a>–<a href="#pone.0209292.ref048">48</a>].</p> <p>To conclude, we report localised CHIKV transmission and evolution in South West Mexico that gave rise to an emergent Mexican cluster which evolved shortly after the introduction of the Asian lineage of CHIKV into the country. Crucially, the characterisation of the intra-host mutant swarms indicates high levels of E1 protein diversity centred in two previously undescribed mutational hotspots in domains II and III which is common to numerous patients. Whether this variation has a role in determining viral fitness during infection should be further explored mechanistically.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p>Human serum specimens were an excess of samples collected during routine passive surveillance activities of the Central Laboratory for Epidemiology (LCE, "Laboratorio Central de Epidemiologia�?), Instituto Mexicano del Seguro Social in Mexico City. All specimens were de-linked from any personal identifiers prior to the commencement of the study.</p> <p>The Study was approved by the Ethics and the Research Committees of the National Committee of Scientific Research of the Instituto Mexicano del Seguro Social with the registration number R-2015-785-096</p> </div> <div class="studydesign" title="sec"> <div class="title" tagx="title" title="title">
Study design</div> <p>From December 2014 to December 2015, serum samples from all suspect CHIKV cases detected through passive surveillance in 35 IMSS medical centres nationwide (located in 32 Mexican states) were submitted for CHIKV confirmatory diagnosis to the Central Laboratory for Epidemiology (LCE, "Laboratorio Central de Epidemiologia�?), IMSS in Mexico City. In accordance with national guidelines, suspect case definition included febrile illness with poly-arthralgia or acute arthritis, plus living in or travelled to, within two weeks of fever onset, an area endemic for <i>Ae</i>. <i>aegypti</i> or <i>Ae</i>. <i>albopictus</i> with confirmed CHIKV cases. Samples were obtained within five days from fever onset. To investigate if CHIKV was present in Mexico prior to October 2014, patients sampled between January and November 2014 principally from Chiapas and surrounding states (Campeche, Oaxaca, Tabasco and Veracruz) who met suspect case definition for dengue virus (DENV) infection (a patient with a non-specified fever who lives or who recently travelled to a region with reported dengue transmission)[<a href="#pone.0209292.ref049">49</a>] but were negative for DENV infection, were submitted for CHIKV diagnosis. Molecular epidemiological studies of CHIKV infection were carried out on 25 patient specimens. To have a broad overview of viral diversity in areas of high transmission, specimens were included that were sampled in Southern Mexico (the region with most confirmed CHIKV cases during the sampling period of the study) and that were sampled at time points spanning December 2014 and August 2015. In addition, inclusion criteria included sample availability for sequencing and Ct values following RT-PCR diagnosis ≤ 32.</p> </div> <div class="denguediagnosis" title="sec"> <div class="title" tagx="title" title="title">
Dengue diagnosis</div> <p>Dengue diagnosis was done according to the algorithms and protocols issued by the InDRE[<a href="#pone.0209292.ref050">50</a>]. In brief, serological assays for Dengue-specific IgM and IgG (Panbio, Korea) and for non-structural protein 1 (NS1; Bio-Rad, California) were done using capture ELISAs. Multiplex reverse-transcription quantitative PCR (qRT-PCR) was done, as described by Chien <i>et al</i>[<a href="#pone.0209292.ref051">51</a>]) (primers in <b><a href="#pone.0209292.s007">S5 Table</a>)</b>, on NS1 positive samples.</p> </div> <div class="chikvdiagnosis" title="sec"> <div class="title" tagx="title" title="title">
CHIKV diagnosis</div> <p>CHIKV diagnosis was done according to guidelines issued by the InDRE[<a href="#pone.0209292.ref049">49</a>] and the Pan-American Health Organization (PAHO)[<a href="#pone.0209292.ref052">52</a>]. Viral RNA was extracted from 200μL of patient serum using the QiAmp Viral RNA Extraction Kit (Qiagen, Hilden, Germany). Forward and reverse primers (CHIKV 6856 and CHIKV 6981, respectively) and a Carboxyfluorescein (FAM)-labelled probe (CHIKV 6919-FAM) were used as described by Lanciotti <i>et al</i>[<a href="#pone.0209292.ref053">53</a>] <b>(<a href="#pone.0209292.s007">S5 Table</a>)</b>. The presence of CHIKV RNA was evaluated using the QuantiTect Probe RT-PCR kit (Qiagen) in a 25μL reaction using 12.5μL of 2x reverse transcription master mix, 0.25μL of QuantiTect RT mix, 0.25μL of each primer (1μM final concentration), probe 0.15μL (0.15μM final concentration), 6.6μL of water and 5μL de RNA. Using the Applied Biosystems 7500 Fast system (Applied Biosystems, Foster City, USA), reverse transcription was carried out at 50°C for 30 mins followed by 95°C for 15 minutes and 45 cycles of 95°C for 15 seconds and 56°C for 1 minute. As stated by the InDRE guidelines[<a href="#pone.0209292.ref049">49</a>], Ct values ≤ 38 in duplicate wells were considered positive.</p> </div> <div class="chikve1sequencingandassembly" title="sec"> <div class="title" tagx="title" title="title">
CHIKV E1 sequencing and assembly</div> <p>A 1044bp region of the CHIKV E1 gene was generated from extracted RNA (as described above) using primers COF1 and COR2 <b>(<a href="#pone.0209292.s007">S5 Table</a>)</b> and the QuantiTect Probe RT-PCR kit (Qiagen) with the conditions described above. For DNA sequencing, two overlapping amplicons (550 base pairs [bp] and 568bp) were then generated using modified primers COF1/CIR1 and CIF2/COR2 (<b><a href="#pone.0209292.s007">S5 Table</a></b>), that include adapter and barcode sequences for the Illumina sequencing. (Illumina, California). Both amplicons obtained for each patient were sequenced on the MiSeq platform (Illumina) with a 600 cycles V3 kit with a paired-end sequencing configuration to obtain 300bp paired end overlapping reads, following the manufacturer’s instructions. For each patient, the paired end reads for each amplicon (COF1 and CIF2 amplicons) were overlapped using FLASH v1.2.7[<a href="#pone.0209292.ref054">54</a>] with default parameters and non-overlapping sequences were discarded. After reconstructing COF1 and CIF2 amplicons, identical nucleotide fragments were clustered, using CD-HIT v4.6.1[<a href="#pone.0209292.ref055">55</a>] with -C 1. The output from the clustering process (.clust file) was transformed to a tab-separated list using the clstr2txt.pl script included in the CD-HIT suite and clusters with at least 50 amplicons, were considered. From each nucleotide cluster, the number of identical sequence amplicons forming the cluster was recorded. The COF1 and CIF2 amplicon sequences with the highest yield were also overlapped (74bp overlap) and aligned to the genome of the African prototype S27 CHIKV strain (accession number AF369024). Following a minimal sequence editing, the consensus sequence for the partial open reading frame for CHIKV E1 gene was identified (1,044bp, genome positions 10,264–11,308). All raw sequencing data was submitted to NCBI Bioproject ID PRJNA495608.</p> </div> <div class="intra-patientvariabilityanalysisofchikve1" title="sec"> <div class="title" tagx="title" title="title">
Intra-patient variability analysis of CHIKV E1</div> <p>For each nucleotide cluster, the representative sequence was translated using the program Transeq from the EMBOSS suite[<a href="#pone.0209292.ref056">56</a>]. The translated representative sequences were also clustered at 100% of amino acid identity using CD-HIT (using -C 1 option). The sequence clusters represented an amino acid haplotype. Using the number of sequences in each nucleotide cluster that was previously recorded, the abundance for each amino acid cluster was calculated and normalized using the sequencing yield from the patient with the lowest yield as the normalization factor. A matrix with the normalized abundance of each haplotype and patient names was generated and visualized using the program Cluster v3.0 and Java TreeView v1.1.6r4, respectively. To calculate individual amino acid mutation frequencies, the number of reads forming all amino acid clusters containing a particular mutation was divided by the total number of reads forming all amino acid clusters in the patient.</p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p>To build the dataset used for the analysis, all available whole CHIKV genomes (334 genomes) plus all available E1 sequences that spanned the 1,044bp consensus fragment under study (384 sequences) were obtained from the NIAID Virus Pathogen Database and Analysis Resource[<a href="#pone.0209292.ref057">57</a>] (ViPR; <a href="http://www.viprbrc.org/">http://www.viprbrc.org/</a>) and aligned together in ClustalW[<a href="#pone.0209292.ref056">56</a>] with the 25 sequences generated in this study. An initial tree was built in PhyML version 3.0[<a href="#pone.0209292.ref058">58</a>] under the General Time Reversible substitution model with a gamma distributed rate variation among sites (GTR+G), as suggested by the Smart Model Selection (SMS) online execution tool (<a href="http://www.atgc-montpellier.fr/sms/">http://www.atgc-montpellier.fr/sms/</a>), with branch support estimated by the Approximate Likelihood Ratio Test (aLRT)-Shimodaira-Hasegawa-like (SH) procedure[<a href="#pone.0209292.ref059">59</a>]. Due to our focus on evolution in the Asian lineage, we built our final dataset using E1 sequences from partial genomes only if they belonged to the Asian lineage (101 strains), as determined by the initial tree (aLRT = 1 for the monophyletic Asian lineage node). Sequences from all other lineages were from whole genomes. In addition, the following strains were removed due to suggestions of laboratory contamination, assembly error or high passage as suggested by the initial tree and previous studies[<a href="#pone.0209292.ref004">4</a>,<a href="#pone.0209292.ref015">15</a>]: Angola/M2022/1962, India/MH4/2000, India/ALSA-1/1986, India/STMWG01/2011, India/STMWG02/2011, Ross and S27. Finally, strains with missing information on sampling year were removed. The final dataset consisted of 420 strains: 12 West African, 17 non-epidemic ECSA, 142 IOL and 249 Asian strains.</p> <p>The maximum liklihood (ML) phylogeny of the final dataset was made in PhyML using a starting tree generated in MEGA version 7.0.14[<a href="#pone.0209292.ref060">60</a>]) with an O’nyong‘yong virus E1 sequence (strain IBH10964) inlcluded as an outgroup. Both starting tree and the PhyML analysis were done using the GTR+G evolutionary model described above and branch support in PhyML was estimated by the aLRT-SH procedure. For Bayesian Markov Chain Monte Carlo (MCMC) phylogenies, CHIKV lineages were analysed seperately in BEAST version 1.8.3[<a href="#pone.0209292.ref061">61</a>]. Strains with missing sampling day were assigned the 15<sup>th</sup> day of the month; missing sampling months and days were assigned as July 2<sup>nd</sup>. The temporal signal of the alignments was evaulated in TempEst v1.5[<a href="#pone.0209292.ref062">62</a>] using ML trees generated in MEGA version 7.0.14 using the methods described above. BEAST analysis was done under the GTR+G nucleotide substitution model or under the Hasegawa-Kishino-Yano nucleotide substitution model with gamma distributed rate variation among sites (HKY+G), a lognormal relaxed clock and a Bayesian Skygrid coalescent model. Effective sample size scores of traces were evaluated in Tracer v1.6. Due to low ESS values under GTR+G (likely because of overparameterisation of the model) all subsequent analyses were done using the HKY+G substitution model. LogCombiner v1.8.3 was used to combine runs of fifty million MCMC states until convergence was reached and a maximum clade credibility (MCC) tree was determined in TreeAnnotator v1.8.3 from the sampling posterior after discarding 10% of states as burn-in. Nodes with posterior probabilities &amp;gt;0.9 were considered significant and evolutionary rates were considered significantly different if the 95% highest posterior density (HPD) intervals did not overlap. MCC trees were edited in FigTree v1.4.2.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Root-to-tip divergence of CHIKV lineage phylogenies.</div> <p>Linear regression of the root-to-tip genetic distances (y axis) against sampling time (x axis) of five dataset alignments with a ‘best fit’ root position. Estimates of the rate of nucleotide substitution (the slope or gradient of the regression), point estimates of the time to most recent common ancestor (TMRCA) and the correlation coefficient (R<sup>2</sup>), are shown. ECSA, East Central Southern Africa lineage; IOL, Indian Ocean lineage; WAf, West African lineage.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Correlation between the number of haplotype variants in two E1 amplicons.</div> <p>Pearson’s correlation analysis between the number of variant haplotypes in each patient in amplicon COF1 and CIF2.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Patient characteristics included in national passive CHIKV surveillance.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Characteristics of 25 CHIKV infected patients from 5 Mexican states.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
The sequencing yield of paired-end reads and the percentage of overlapped reads for E1 amplicons.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Table</span> <div class="title" tagx="title" title="title">
Frequency of intrapatient amino acid mutations in two E1 amplicons and the viral haplotypes in which they occur.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Table</span> <div class="title" tagx="title" title="title">
Primers used for CHIKV and Dengue virus diagnosis and for sequencing of the CHIKV E1 gene.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Table</span> <div class="title" tagx="title" title="title">
Demographic and clinical characteristics of 25 sequenced samples.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0209292.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We would like to thank the Unidad Universitaria de Secuenciación Masiva y Bioinformática (UUSMB) of the Instituto de Biotecnología, UNAM, for advice on DNA sequencing and bioinformatic analysis. The UUSMB is part of the Laboratorio Nacional de Apoyo a las Tecnologías en Ciencias Genómicas which was created and funded by the Programa de Laboratorios Nacionales CONACyT. We also express our gratitude to the working team from the Molecular Biology Department of the Laboratorio Central de Epidemiología for technical assistance. Finally, we would like to thank Dr. Carlos Arias for his feedback and discussion for this manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0209292.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caglioti</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lalle</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castilletti</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carletti</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capobianchi</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chikungunya virus infection: an overview</span>. <span class="source" tagx="source" title="source">New Microbiol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">36</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">211</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23912863">23912863</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ross</span><span class="given-names" tagx="given-names" title="given-names">RW</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Newala epidemic. III. The virus: isolation, pathogenic properties and relationship to the epidemic</span>. <span class="source" tagx="source" title="source">J Hyg (Lond)</span>. <span class="year" tagx="year" title="year">1956</span>;<span class="volume" tagx="volume" title="volume">54</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">177</span>–<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13346078">13346078</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solignat</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gay</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgs</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briant</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devaux</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Replication cycle of chikungunya: a re-emerging arbovirus</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">393</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">183</span>–<span class="lpage" tagx="lpage" title="lpage">97</span>. <a href="https://doi.org/10.1016/j.virol.2009.07.024">10.1016/j.virol.2009.07.024</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19732931">19732931</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volk</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsetsarkin</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia</span><span class="given-names" tagx="given-names" title="given-names">TI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sall</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome-Scale Phylogenetic Analyses of Chikungunya Virus Reveal Independent Emergences of Recent Epidemics and Various Evolutionary Rates</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">84</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">6497</span>–<span class="lpage" tagx="lpage" title="lpage">504</span>. <a href="https://doi.org/10.1128/JVI.01603-09">10.1128/JVI.01603-09</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20410280">20410280</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kariuki Njenga</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nderitu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ledermann</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ndirangu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Logue</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelly</span><span class="given-names" tagx="given-names" title="given-names">CHL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Tracking epidemic Chikungunya virus into the Indian Ocean from East Africa</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2008</span><span class="month" tagx="month" title="month">11</span>;<span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">Pt 11</span>):<span class="fpage" tagx="fpage" title="fpage">2754</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>. <a href="https://doi.org/10.1099/vir.0.2008/005413-0">10.1099/vir.0.2008/005413-0</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931072">18931072</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Arrival of chikungunya virus in the new world: prospects for spread and impact on public health.</span><span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2014</span><span class="month" tagx="month" title="month">6</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">e2921</span><a href="https://doi.org/10.1371/journal.pntd.0002921">10.1371/journal.pntd.0002921</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24967777">24967777</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuffenecker</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iteman</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michault</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murri</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frangeul</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaney</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome microevolution of chikungunya viruses causing the Indian Ocean outbreak</span>. <span class="source" tagx="source" title="source">PLoS Med</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1058</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niyas</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unnikrishnan</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathew</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manakkadan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular characterization of Chikungunya virus isolates from clinical samples and adult Aedes albopictus mosquitoes emerged from larvae from Kerala, South India</span>. <span class="source" tagx="source" title="source">Virol J</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="month" tagx="month" title="month">9</span> 2009):<span class="fpage" tagx="fpage" title="fpage">189</span><a href="https://doi.org/10.1186/1743-422X-7-189">10.1186/1743-422X-7-189</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20704755">20704755</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petersen</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powers</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chikungunya: epidemiology</span>. <span class="source" tagx="source" title="source">F1000Research</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leparc-Goffart</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nougairede</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassadou</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prat</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Lamballerie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>. <span class="mixed-article-title" title="mixed-article-title">Chikungunya in the Americas</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">383</span>:<span class="fpage" tagx="fpage" title="fpage">514</span><a href="https://doi.org/10.1016/S0140-6736(14)60185-9">10.1016/S0140-6736(14)60185-9</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24506907">24506907</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunes</span><span class="given-names" tagx="given-names" title="given-names">MRT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golding</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Emergence and potential for spread of Chikungunya virus in Brazil</span>. <span class="source" tagx="source" title="source">BMC Med</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">102</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25976325">25976325</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz-Quiñonez</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escobar-Escamilla</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortíz-Alcántara</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vázquez-Pichardo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de la Luz Torres-Rodríguez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuñez-León</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Identification of Asian genotype of chikungunya virus isolated in Mexico</span>. <span class="source" tagx="source" title="source">Virus Genes</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <a href="https://doi.org/10.1007/s11262-015-1275-9">10.1007/s11262-015-1275-9</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26781948">26781948</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stapleford</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moratorio</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henningsson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matheus</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enfissi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Whole-Genome Sequencing Analysis from the Chikungunya Virus Caribbean Outbreak Reveals Novel Evolutionary Genomic Elements</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>).</span></li> <li tag="ref"><a name="pone.0209292.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Gobierno de los Estados Unidos Mexicanos-Presidencia de la Republica. Cuarto Informe de Gobierno 2015–2016. Anexo Estadístico. 2016.</span></li> <li tag="ref"><a name="pone.0209292.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puri</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fedorova</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hari</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comprehensive Genome-Scale Phylogenetic Study Provides New Insights on the Global Expansion of Chikungunya Virus</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">90</span>(<span class="issue" tagx="issue" title="issue">23</span>):<span class="fpage" tagx="fpage" title="fpage">JVI.01166</span>–<span class="lpage" tagx="lpage" title="lpage">16</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">K-K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kristy</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sy</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tandoc</span><span class="given-names" tagx="given-names" title="given-names">AO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khoo</span><span class="given-names" tagx="given-names" title="given-names">J-J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sulaiman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Independent Emergence of the Cosmopolitan Asian Chikungunya Virus, Philippines 2012</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahadeo</span><span class="given-names" tagx="given-names" title="given-names">NSD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allicock</span><span class="given-names" tagx="given-names" title="given-names">OM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Salazar</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auguste</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olowokure</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Understanding the evolution and spread of chikungunya virus in the Americas using complete genome sequences</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vex010</span><a href="https://doi.org/10.1093/ve/vex010">10.1093/ve/vex010</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28480053">28480053</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voss</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaney</span><span class="given-names" tagx="given-names" title="given-names">M-C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duquerroy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vonrhein</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girard-Blanc</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crublet</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Glycoprotein organization of Chikungunya virus particles revealed by X-ray crystallography</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">468</span>(<span class="issue" tagx="issue" title="issue">7324</span>):<span class="fpage" tagx="fpage" title="fpage">709</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. <a href="https://doi.org/10.1038/nature09555">10.1038/nature09555</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21124458">21124458</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Pan American Health Organization. No of cases of Chikungunya Fever in the Americas—Cumulative Cases (May 13, 2016) [Internet]. 2016.</span></li> <li tag="ref"><a name="pone.0209292.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrera</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denis</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barahona de Mosca</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cedeño</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Unusual pattern of chikungunya virus epidemic in the Americas, the Panamanian experience</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Secretaria de Salud. "Casos Confirmados de Enfermedad por Virus del Zika�?, Semana Epidemiológica 44 del 2017 [Internet]. 2017.</span></li> <li tag="ref"><a name="pone.0209292.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musso</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao-Lormeau</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zika virus: following the path of dengue and chikungunya?</span><span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">386</span>(<span class="issue" tagx="issue" title="issue">9990</span>):<span class="fpage" tagx="fpage" title="fpage">243</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <a href="https://doi.org/10.1016/S0140-6736(15)61273-9">10.1016/S0140-6736(15)61273-9</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26194519">26194519</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz-Quiñonez</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortiz-Alcántara</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fragoso-Fonseca</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcés-Ayala</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escobar-Escamilla</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vázquez-Pichardo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Complete genome sequences of chikungunya virus strains isolated in Mexico: first detection of imported and autochthonous cases</span>. <span class="source" tagx="source" title="source">Genome Announc</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e00300</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>. <a href="https://doi.org/10.1128/genomeA.00300-15">10.1128/genomeA.00300-15</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25953170">25953170</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz-Quiñonez</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Martínez</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres-Longoria</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vázquez-Pichardo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruz-Ramírez</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramírez-González</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Evidence of the presence of the Zika virus in Mexico since early 2015</span>. <span class="source" tagx="source" title="source">Virus Genes</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">52</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">855</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <a href="https://doi.org/10.1007/s11262-016-1384-0">10.1007/s11262-016-1384-0</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27557815">27557815</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azevedo R do S da</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">MUG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souza</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunha</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika virus in the Americas: Early epidemiological and genetic findings</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">352</span>(<span class="issue" tagx="issue" title="issue">6283</span>):<span class="fpage" tagx="fpage" title="fpage">aaf5036</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrivastava</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gresh</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amedeo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Differing epidemiological dynamics of Chikungunya virus in the Americas during the 2014–2015 epidemic</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venturi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Luca</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fortuna</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elena Remoli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riccardo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Severini</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Detection of a chikungunya outbreak in Central Italy Detection of a chikungunya outbreak in Central</span>. <span class="source" tagx="source" title="source">Euro Surveill</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">39</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lourenço-de-Oliveira</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Failloux</span><span class="given-names" tagx="given-names" title="given-names">A-B</span></span>. <span class="mixed-article-title" title="mixed-article-title">High risk for chikungunya virus to initiate an enzootic sylvatic cycle in the tropical Americas</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">e0005698</span><a href="https://doi.org/10.1371/journal.pntd.0005698">10.1371/journal.pntd.0005698</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28662031">28662031</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>. <span class="mixed-article-title" title="mixed-article-title">The evolution of Ebola virus: Insights from the 2013–2016 epidemic</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">538</span>(<span class="issue" tagx="issue" title="issue">7624</span>):<span class="fpage" tagx="fpage" title="fpage">193</span>–<span class="lpage" tagx="lpage" title="lpage">200</span>. <a href="https://doi.org/10.1038/nature19790">10.1038/nature19790</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27734858">27734858</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forrester</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffey</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Arboviral bottlenecks and challenges to maintaining diversity and fitness during mosquito transmission</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">3991</span>–<span class="lpage" tagx="lpage" title="lpage">4004</span>. <a href="https://doi.org/10.3390/v6103991">10.3390/v6103991</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25341663">25341663</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manakkadan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mudaliar</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joseph</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sivakumar</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Correlation of phylogenetic clade diversification and in vitro infectivity differences among Cosmopolitan genotype strains of Chikungunya virus</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">37</span>:<span class="fpage" tagx="fpage" title="fpage">174</span>–<span class="lpage" tagx="lpage" title="lpage">84</span>. <a href="https://doi.org/10.1016/j.meegid.2015.11.019">10.1016/j.meegid.2015.11.019</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26611825">26611825</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saborio</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gresh</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eswarappa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fire</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chikungunya virus sequences across the first epidemic in Nicaragua, 2014–2015</span>. <span class="source" tagx="source" title="source">Am J Trop Med Hyg</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">94</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">400</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>. <a href="https://doi.org/10.4269/ajtmh.15-0497">10.4269/ajtmh.15-0497</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26643533">26643533</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vignuzzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnold</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andino</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">439</span>(<span class="issue" tagx="issue" title="issue">7074</span>):<span class="fpage" tagx="fpage" title="fpage">344</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <a href="https://doi.org/10.1038/nature04388">10.1038/nature04388</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16327776">16327776</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffey</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beeharry</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borderia</span><span class="given-names" tagx="given-names" title="given-names">a. V</span></span>., <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanc</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vignuzzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Arbovirus high fidelity variant loses fitness in mosquitoes and mice</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">108</span>(<span class="issue" tagx="issue" title="issue">38</span>):<span class="fpage" tagx="fpage" title="fpage">16038</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. <a href="https://doi.org/10.1073/pnas.1111650108">10.1073/pnas.1111650108</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21896755">21896755</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozen-Gagnon</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stapleford</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mongelli</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanc</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Failloux</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saleh</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Alphavirus Mutator Variants Present Host-Specific Defects and Attenuation in Mammalian and Insect Models</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">1</span>).</span></li> <li tag="ref"><a name="pone.0209292.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Roche</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanc</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bordería</span><span class="given-names" tagx="given-names" title="given-names">A V</span></span>., <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henningsson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Increasing clinical severity during a dengue virus type 3 Cuban epidemic: deep sequencing of evolving viral populations</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">90</span>(<span class="month" tagx="month" title="month">2</span>):<span class="fpage" tagx="fpage" title="fpage">JVI.02647</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danis</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Díaz</span><span class="given-names" tagx="given-names" title="given-names">GEE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trujillo</span><span class="given-names" tagx="given-names" title="given-names">MKDC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caballero</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sepulveda</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malo</span><span class="given-names" tagx="given-names" title="given-names">GIR</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Clinical characterization of acute and convalescent illness of confirmed chikungunya cases from Chiapas, S. Mexico: A cross sectional study</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2017</span><span class="month" tagx="month" title="month">10</span>, <span class="volume" tagx="volume" title="volume">12</span> (<span class="issue" tagx="issue" title="issue">10</span>).</span></li> <li tag="ref"><a name="pone.0209292.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cigarroa</span><span class="given-names" tagx="given-names" title="given-names">TN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blitvich</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cetina</span><span class="given-names" tagx="given-names" title="given-names">TRC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talavera</span><span class="given-names" tagx="given-names" title="given-names">ALG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baak</span><span class="given-names" tagx="given-names" title="given-names">BCM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres</span><span class="given-names" tagx="given-names" title="given-names">COM</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chikungunya Virus in febrile humans and aedes aegypri mosquitoes, Yucatan Mexico</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2016</span><span class="month" tagx="month" title="month">10</span>, <span class="volume" tagx="volume" title="volume">22</span> (<span class="issue" tagx="issue" title="issue">10</span>): <span class="fpage" tagx="fpage" title="fpage">1804</span>–<span class="lpage" tagx="lpage" title="lpage">1807</span>. <a href="https://doi.org/10.3201/eid2210.152087">10.3201/eid2210.152087</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27347760">27347760</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Morales</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gil-Restrepo</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramírez-Jaramillo</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya-Arias</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Acevedo-Mendoza</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juan</span><span class="given-names" tagx="given-names" title="given-names">E B-A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Post-chikungunya chronic inflammatory rheumatism: results from a retrospective follow-up study of 283 adult and child cases in La Virginia, Risaralda, Colombia</span>. <span class="source" tagx="source" title="source">F1000Research</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="month" tagx="month" title="month">5</span>):<span class="fpage" tagx="fpage" title="fpage">360</span><span class="comment">doi: <a href="https://doi.org/10.12688/f1000research.8235.2">10.12688/f1000research.8235.2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27081477">27081477</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galán-Huerta</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martínez-Landeros</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado-Gallegos</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caballero-Sosa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malo-García</span><span class="given-names" tagx="given-names" title="given-names">IR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernández-Salas</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular and clinical characterization of Chikungunya virus infections in Southeast Mexico</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sheldon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perales</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitness</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gain</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Equilibrium</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="source" tagx="source" title="source">Viral Quasispecies Evolution</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">76</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">159</span>–<span class="lpage" tagx="lpage" title="lpage">216</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaskov</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buzacott</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thu</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowry</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Long-term transmission of defective RNA viruses in humans and Aedes mosquitoes</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2006</span><span class="month" tagx="month" title="month">1</span><span class="day" tagx="day" title="day">13</span>;<span class="volume" tagx="volume" title="volume">311</span>(<span class="issue" tagx="issue" title="issue">5758</span>):<span class="fpage" tagx="fpage" title="fpage">236</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <a href="https://doi.org/10.1126/science.1115030">10.1126/science.1115030</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16410525">16410525</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forrester</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guerbois</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of Intrahost Variation in Venezuelan Equine Encephalitis Virus Reveals Repeated Deletions in the 6-Kilodalton Protein Gene</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">85</span>(<span class="issue" tagx="issue" title="issue">17</span>):<span class="fpage" tagx="fpage" title="fpage">8709</span>–<span class="lpage" tagx="lpage" title="lpage">17</span>. <a href="https://doi.org/10.1128/JVI.00165-11">10.1128/JVI.00165-11</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21715498">21715498</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geller</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo-Calap</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuevas</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossolillo</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Negroni</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanjuán</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">The external domains of the HIV-1 envelope are a mutational cold spot</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akahata</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holdaway</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pal</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Structural analyses at pseudo atomic resolution of Chikungunya virus and antibodies show mechanisms of neutralization</span>. <span class="source" tagx="source" title="source">Elife</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">2013</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liss</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fox</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimak</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Neutralizing Monoclonal Antibodies Block Chikungunya Virus Entry and Release by Targeting an Epitope Critical to Viral Pathogenesis</span>. <span class="source" tagx="source" title="source">Cell Rep</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">2553</span>–<span class="lpage" tagx="lpage" title="lpage">64</span>. <a href="https://doi.org/10.1016/j.celrep.2015.11.043">10.1016/j.celrep.2015.11.043</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26686638">26686638</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lum</span><span class="given-names" tagx="given-names" title="given-names">F-M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teo</span><span class="given-names" tagx="given-names" title="given-names">T-H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">WWL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kam</span><span class="given-names" tagx="given-names" title="given-names">Y-W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rénia</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">LFP</span></span>. <span class="mixed-article-title" title="mixed-article-title">An essential role of antibodies in the control of Chikungunya virus infection</span>. <span class="source" tagx="source" title="source">J Immunol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">190</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">6295</span>–<span class="lpage" tagx="lpage" title="lpage">302</span>. <a href="https://doi.org/10.4049/jimmunol.1300304">10.4049/jimmunol.1300304</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23670192">23670192</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erasmus</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weaver</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Development of vaccines for chikungunya fever</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">214</span>(<span class="issue" tagx="issue" title="issue">Suppl 5</span>):<span class="fpage" tagx="fpage" title="fpage">S488</span>–<span class="lpage" tagx="lpage" title="lpage">96</span>. <a href="https://doi.org/10.1093/infdis/jiw271">10.1093/infdis/jiw271</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27920179">27920179</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dirección General de Epidemiología I. Lineamientos para la vigilancia epidemiológica y diagnóstico por laboratorio de fiebre chikungunya [Internet]. 2014.</span></li> <li tag="ref"><a name="pone.0209292.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Instituto de Diagnóstico y Referencia Epidemiológicos. Lineamientos para la vigilancia epidemiológica de dengue por laboratorio. México; 2014. Chien L-J, 51.</span></li> <li tag="ref"><a name="pone.0209292.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chien</span><span class="given-names" tagx="given-names" title="given-names">L-J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liao</span><span class="given-names" tagx="given-names" title="given-names">T-L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">P-Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">J-H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">G-JJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Development of real-time reverse transcriptase PCR assays to detect and serotype dengue viruses</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span>. <span class="year" tagx="year" title="year">2006</span><span class="month" tagx="month" title="month">4</span>;<span class="volume" tagx="volume" title="volume">44</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1295</span>–<span class="lpage" tagx="lpage" title="lpage">304</span>. <a href="https://doi.org/10.1128/JCM.44.4.1295-1304.2006">10.1128/JCM.44.4.1295-1304.2006</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597854">16597854</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Organización Panamericana de la Salud. Preparación y respuesta ante la eventual introducción del virus chikungunya en las Américas. Washington, D.C.; 2011.</span></li> <li tag="ref"><a name="pone.0209292.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanciotti</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosoy</span><span class="given-names" tagx="given-names" title="given-names">OL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laven</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panella</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velez</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambert</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Chikungunya virus in US travelers returning from India, 2006</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2007</span><span class="month" tagx="month" title="month">5</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">764</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <a href="https://doi.org/10.3201/eid1305.070015">10.3201/eid1305.070015</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17553261">17553261</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>. <span class="mixed-article-title" title="mixed-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</span><span class="source" tagx="source" title="source">Tanja Mago c</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">21</span>):<span class="fpage" tagx="fpage" title="fpage">2957</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godzik</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2006</span><span class="month" tagx="month" title="month">7</span><span class="day" tagx="day" title="day">1</span>;<span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">1658</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <a href="https://doi.org/10.1093/bioinformatics/btl158">10.1093/bioinformatics/btl158</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699">16731699</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowley</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uludag</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gur</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Squizzato</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The EMBL-EBI bioinformatics web and programmatic tools framework</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">43</span>(<span class="issue" tagx="issue" title="issue">W1</span>):<span class="fpage" tagx="fpage" title="fpage">W580</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <a href="https://doi.org/10.1093/nar/gkv279">10.1093/nar/gkv279</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25845596">25845596</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickett</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadat</span><span class="given-names" tagx="given-names" title="given-names">EL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noronha</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Squires</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunt</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">ViPR: An open bioinformatics database and analysis resource for virology research</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">40</span>(<span class="issue" tagx="issue" title="issue">D1</span>):<span class="fpage" tagx="fpage" title="fpage">593</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.</span></li> <li tag="ref"><a name="pone.0209292.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guindon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufayard</span><span class="given-names" tagx="given-names" title="given-names">J-F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefort</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hordijk</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>. <span class="mixed-article-title" title="mixed-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">2010</span><span class="month" tagx="month" title="month">5</span>;<span class="volume" tagx="volume" title="volume">59</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">307</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. <a href="https://doi.org/10.1093/sysbio/syq010">10.1093/sysbio/syq010</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638">20525638</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>. <span class="mixed-article-title" title="mixed-article-title">Approximate likelihood-ratio test for branches: A fast, accurate, and powerful alternative</span>. <span class="source" tagx="source" title="source">Syst Biol</span>. <span class="year" tagx="year" title="year">2006</span><span class="month" tagx="month" title="month">8</span>;<span class="volume" tagx="volume" title="volume">55</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">539</span>–<span class="lpage" tagx="lpage" title="lpage">52</span>. <a href="https://doi.org/10.1080/10635150600755453">10.1080/10635150600755453</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16785212">16785212</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2016</span><span class="month" tagx="month" title="month">7</span>;<span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1870</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. <a href="https://doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="year" tagx="year" title="year">2007</span><span class="month" tagx="month" title="month">1</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">214</span><a href="https://doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="pone.0209292.ref062" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Max Carvalho</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vew007</span><a href="https://doi.org/10.1093/ve/vew007">10.1093/ve/vew007</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774300">27774300</a></span></span></li> </ul> </div> </div>  </body></html>